
<html lang="en"     class="pb-page"  data-request-id="6a1ac7cf-346e-4d56-b7e9-827d51c9836b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01904;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-6;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Generating Selective Leads for Mer Kinase Inhibitors—Example of a Comprehensive Lead-Generation Strategy" /></meta><meta name="dc.Creator" content="J. Willem M.  Nissink" /></meta><meta name="dc.Creator" content="Sana  Bazzaz" /></meta><meta name="dc.Creator" content="Carolyn  Blackett" /></meta><meta name="dc.Creator" content="Matthew A.  Clark" /></meta><meta name="dc.Creator" content="Olga  Collingwood" /></meta><meta name="dc.Creator" content="Jeremy S.  Disch" /></meta><meta name="dc.Creator" content="Diana  Gikunju" /></meta><meta name="dc.Creator" content="Kristin  Goldberg" /></meta><meta name="dc.Creator" content="John P.  Guilinger" /></meta><meta name="dc.Creator" content="Elizabeth  Hardaker" /></meta><meta name="dc.Creator" content="Edward J.  Hennessy" /></meta><meta name="dc.Creator" content="Rachael  Jetson" /></meta><meta name="dc.Creator" content="Anthony D.  Keefe" /></meta><meta name="dc.Creator" content="William  McCoull" /></meta><meta name="dc.Creator" content="Lindsay  McMurray" /></meta><meta name="dc.Creator" content="Allison  Olszewski" /></meta><meta name="dc.Creator" content="Ross  Overman" /></meta><meta name="dc.Creator" content="Alexander  Pflug" /></meta><meta name="dc.Creator" content="Marian  Preston" /></meta><meta name="dc.Creator" content="Philip B.  Rawlins" /></meta><meta name="dc.Creator" content="Emma  Rivers" /></meta><meta name="dc.Creator" content="Marianne  Schimpl" /></meta><meta name="dc.Creator" content="Paul  Smith" /></meta><meta name="dc.Creator" content="Caroline  Truman" /></meta><meta name="dc.Creator" content="Elizabeth  Underwood" /></meta><meta name="dc.Creator" content="Juli  Warwicker" /></meta><meta name="dc.Creator" content="Jon  Winter-Holt" /></meta><meta name="dc.Creator" content="Simon  Woodcock" /></meta><meta name="dc.Creator" content="Ying  Zhang" /></meta><meta name="dc.Description" content="Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been associated with cancer progression, metastasis, and drug resistance. Their essential function in immune homeostasis has prom..." /></meta><meta name="Description" content="Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been associated with cancer progression, metastasis, and drug resistance. Their essential function in immune homeostasis has prom..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 8, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01904" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01904" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01904" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01904" /></link>
        
    
    

<title>Generating Selective Leads for Mer Kinase Inhibitors—Example of a Comprehensive Lead-Generation Strategy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01904" /></meta><meta property="og:title" content="Generating Selective Leads for Mer Kinase Inhibitors—Example of a Comprehensive Lead-Generation Strategy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0017.jpeg" /></meta><meta property="og:description" content="Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been associated with cancer progression, metastasis, and drug resistance. Their essential function in immune homeostasis has prompted an interest in their role as modulators of antitumor immune response in the tumor microenvironment. Here we illustrate the outcomes of an extensive lead-generation campaign for identification of Mer inhibitors, focusing on the results from concurrent, orthogonal high-throughput screening approaches. Data mining, HT (high-throughput), and DECL (DNA-encoded chemical library) screens offered means to evaluate large numbers of compounds. We discuss campaign strategy and screening outcomes, and exemplify series resulting from prioritization of hits that were identified. Concurrent execution of HT and DECL screening successfully yielded a large number of potent, selective, and novel starting points, covering a range of selectivity profiles across the TAM family members and modes of kinase binding, and offered excellent start points for lead development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01904"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01904">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01904&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01904&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01904&amp;href=/doi/10.1021/acs.jmedchem.0c01904" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3165-3184</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01984" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02033" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Generating Selective Leads for Mer Kinase Inhibitors—Example of a Comprehensive Lead-Generation Strategy</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">J. Willem M. Nissink</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Willem M. Nissink</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c2b5abaeaea7afecacabb1b1abaca982a3b1b6b0a3b8a7aca7a1a3ecaca7b6"><span class="__cf_email__" data-cfemail="5f283633333a327131362c2c3631341f3e2c2b2d3e253a313a3c3e71313a2b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Willem+M.++Nissink">J. Willem M. Nissink</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2572-9140" title="Orcid link">http://orcid.org/0000-0003-2572-9140</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sana Bazzaz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sana Bazzaz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sana++Bazzaz">Sana Bazzaz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carolyn Blackett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carolyn Blackett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carolyn++Blackett">Carolyn Blackett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew A. Clark</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew A. Clark</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+A.++Clark">Matthew A. Clark</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olga Collingwood</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olga Collingwood</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olga++Collingwood">Olga Collingwood</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeremy S. Disch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeremy S. Disch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+S.++Disch">Jeremy S. Disch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Diana Gikunju</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Diana Gikunju</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Diana++Gikunju">Diana Gikunju</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kristin Goldberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kristin Goldberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kristin++Goldberg">Kristin Goldberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John P. Guilinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John P. Guilinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+P.++Guilinger">John P. Guilinger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elizabeth Hardaker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elizabeth Hardaker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++Hardaker">Elizabeth Hardaker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edward J. Hennessy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edward J. Hennessy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, R&D Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edward+J.++Hennessy">Edward J. Hennessy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachael Jetson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachael Jetson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachael++Jetson">Rachael Jetson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony D. Keefe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony D. Keefe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+D.++Keefe">Anthony D. Keefe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William McCoull</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William McCoull</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++McCoull">William McCoull</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2977-1744" title="Orcid link">http://orcid.org/0000-0002-2977-1744</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lindsay McMurray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lindsay McMurray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lindsay++McMurray">Lindsay McMurray</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allison Olszewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allison Olszewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allison++Olszewski">Allison Olszewski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ross Overman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ross Overman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ross++Overman">Ross Overman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexander Pflug</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander Pflug</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander++Pflug">Alexander Pflug</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marian Preston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marian Preston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marian++Preston">Marian Preston</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philip B. Rawlins</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip B. Rawlins</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip+B.++Rawlins">Philip B. Rawlins</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emma Rivers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma Rivers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma++Rivers">Emma Rivers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marianne Schimpl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marianne Schimpl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marianne++Schimpl">Marianne Schimpl</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2284-5250" title="Orcid link">http://orcid.org/0000-0003-2284-5250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Smith">Paul Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Caroline Truman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline Truman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline++Truman">Caroline Truman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elizabeth Underwood</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elizabeth Underwood</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++Underwood">Elizabeth Underwood</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Juli Warwicker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juli Warwicker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juli++Warwicker">Juli Warwicker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jon Winter-Holt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jon Winter-Holt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jon++Winter-Holt">Jon Winter-Holt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon Woodcock</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Woodcock</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Woodcock">Simon Woodcock</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ying Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem, Inc., Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Zhang">Ying Zhang</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01904&amp;href=/doi/10.1021%2Facs.jmedchem.0c01904" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3165–3184</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 8, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 November 2020</li><li><span class="item_label"><b>Published</b> online</span>8 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01904" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01904</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3165%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJ.%2BWillem%2BM.%2BNissink%252C%2BSana%2BBazzaz%252C%2BCarolyn%2BBlackett%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D6%26contentID%3Dacs.jmedchem.0c01904%26title%3DGenerating%2BSelective%2BLeads%2Bfor%2BMer%2BKinase%2BInhibitors%25E2%2580%2594Example%2Bof%2Ba%2BComprehensive%2BLead-Generation%2BStrategy%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3184%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01904"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1538</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01904" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Generating Selective Leads for Mer Kinase Inhibitors—Example of a Comprehensive Lead-Generation Strategy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Willem M. Nissink&quot;},{&quot;first_name&quot;:&quot;Sana&quot;,&quot;last_name&quot;:&quot;Bazzaz&quot;},{&quot;first_name&quot;:&quot;Carolyn&quot;,&quot;last_name&quot;:&quot;Blackett&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;A. Clark&quot;},{&quot;first_name&quot;:&quot;Olga&quot;,&quot;last_name&quot;:&quot;Collingwood&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;S. Disch&quot;},{&quot;first_name&quot;:&quot;Diana&quot;,&quot;last_name&quot;:&quot;Gikunju&quot;},{&quot;first_name&quot;:&quot;Kristin&quot;,&quot;last_name&quot;:&quot;Goldberg&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;P. Guilinger&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;Hardaker&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;J. Hennessy&quot;},{&quot;first_name&quot;:&quot;Rachael&quot;,&quot;last_name&quot;:&quot;Jetson&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;D. Keefe&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;McCoull&quot;},{&quot;first_name&quot;:&quot;Lindsay&quot;,&quot;last_name&quot;:&quot;McMurray&quot;},{&quot;first_name&quot;:&quot;Allison&quot;,&quot;last_name&quot;:&quot;Olszewski&quot;},{&quot;first_name&quot;:&quot;Ross&quot;,&quot;last_name&quot;:&quot;Overman&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;Pflug&quot;},{&quot;first_name&quot;:&quot;Marian&quot;,&quot;last_name&quot;:&quot;Preston&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;B. Rawlins&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;Rivers&quot;},{&quot;first_name&quot;:&quot;Marianne&quot;,&quot;last_name&quot;:&quot;Schimpl&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;Truman&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;Underwood&quot;},{&quot;first_name&quot;:&quot;Juli&quot;,&quot;last_name&quot;:&quot;Warwicker&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;Winter-Holt&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Woodcock&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3165-3184&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01904&quot;},&quot;abstract&quot;:&quot;Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been associated with cancer progression, metastasis, and drug resistance. Their essential function in immune homeostasis has prompted an interest in their role as modulators of antitumor immune response in the tumor microenvironment. Here we illustrate the outcomes of an extensive lead-generation campaign for identification of Mer inhibitors, focusing on the results from concurrent, orthogonal high-throughput screening approaches. Data mining, HT (high-throughput), and DECL (DNA-encoded chemical library) screens offered means to evaluate large numbers of compounds. We discuss campaign strategy and screening outcomes, and exemplify series resulting from prioritization of hits that were identified. Concurrent execution of HT and DECL screening successfully yielded a large number of potent, selective, and novel starting points, covering a range of selectivity profiles across the TAM family members and modes of kinase binding, and offered excelle&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01904&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01904" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01904&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01904" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01904&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01904" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01904&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01904&amp;href=/doi/10.1021/acs.jmedchem.0c01904" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01904" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01904" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01904%26sid%3Dliteratum%253Aachs%26pmid%3D33683117%26genre%3Darticle%26aulast%3DNissink%26date%3D2021%26atitle%3DGenerating%2BSelective%2BLeads%2Bfor%2BMer%2BKinase%2BInhibitors%25E2%2580%2594Example%2Bof%2Ba%2BComprehensive%2BLead-Generation%2BStrategy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D6%26spage%3D3165%26epage%3D3184%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292638" title="MERS">MERS</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/jmcmar.2021.64.issue-6/20210325/jmcmar.2021.64.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been associated with cancer progression, metastasis, and drug resistance. Their essential function in immune homeostasis has prompted an interest in their role as modulators of antitumor immune response in the tumor microenvironment. Here we illustrate the outcomes of an extensive lead-generation campaign for identification of Mer inhibitors, focusing on the results from concurrent, orthogonal high-throughput screening approaches. Data mining, HT (high-throughput), and DECL (DNA-encoded chemical library) screens offered means to evaluate large numbers of compounds. We discuss campaign strategy and screening outcomes, and exemplify series resulting from prioritization of hits that were identified. Concurrent execution of HT and DECL screening successfully yielded a large number of potent, selective, and novel starting points, covering a range of selectivity profiles across the TAM family members and modes of kinase binding, and offered excellent start points for lead development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">TAM kinases (TYRO3, Axl, and Mer) have gained interest in recent years as immunooncologic targets. In this context, most of the focus has been on Mer and Axl, which have been linked to development and polarization of myeloid and dendritic cells. Inhibition of Mer is expected to rekindle immune response in the tumor microenvironment, where Mer signaling is known to suppress inflammatory response. Tumor-associated macrophages are relatively abundant in the tumor microenvironment and are known to play a role in cancer progression and immunosuppression.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Mer is expressed in anti-inflammatory M2 macrophages, where it mediates suppression of anti-inflammatory cytokines and enhances the action of inflammatory repressors.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> This suppression is mediated by Mer’s effects on myeloid cells, and inhibition of Mer promotes a polarization of M2 to M1-type proinflammatory macrophages, which is expected to hamper tumor progression. Mer is known to promote apoptotic cell clearance by a process called efferocytosis,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> which benefits the tumor. Following efferocytosis, macrophages are polarized further to a pro-tumor M2-like phenotype and secrete increased levels of immunosuppressive cytokines. Inhibition of Mer was therefore proposed as a means to halt and revert macrophage polarization and revive immune-mediated tumor suppression.</div><div class="NLM_p">Axl plays a complex role in immune regulation including the activation of T-cells by antigen-presenting dendritic cells. Axl is not an oncogenic driver, but its overexpression has been linked to a wide range of processes including epithelial to mesenchymal transition, tumor angiogenesis, resistance to chemotherapeutic and targeted agents, and decreased antitumor immune response.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Inhibition of Axl is expected to influence T-cell populations in the tumor microenvironment, and consequent release of proinflammatory cytokines. Inhibition of both Mer and Axl is therefore expected to lead to an advantageous and localized inflammatory response, which would enable the immune system to target tumors. Less is known about the third member of the TAM family, Tyro3. It is expressed in the nervous system, and its expression has been noted in a variety of malignancies including colon, breast, lung, and liver cancer. Neural degeneration has been observed in Tyro3 knockout mice<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and it was anticipated that its inhibition may potentially result in undesirable effects, though it must be noted that several compounds in the clinic inhibit Tyro3,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> apparently with limited adverse effect.</div><div class="NLM_p">Our lead-finding strategy to identify suitably selective chemical equity for a lead initialization program is summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. In short, four tracks were explored in parallel. A high-throughput screening effort was initiated internally, and a complementary DNA-encoded chemical library<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> (DECL) screening exercise was performed. In parallel, a data-mining exercise of in-house historical data identified compounds active against Mer from within the AstraZeneca collection, and analysis of literature information identified further compounds. This information was utilized to produce protein–ligand structures of Mer to establish binding modes and inform compound design. This was feasible as, over the years, a large contingent of data points has been accumulated through kinome scans of compounds of interest, most of it inhibition data at a single concentration. It was however anticipated that such compounds would not be very selective, given their origins in legacy kinase projects.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Parallel lead-finding strategies for finding Mer inhibitor leads.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Kinome selectivity profiles were determined for lead compounds. These data, combined with sequence-similarity analyses, produced a short list of potential off-targets. These were divided into two classes: first, given our immune-oncologic focus, similar targets with known immune function can be expected to interfere with or at the least confuse readouts in target validation experiments. For this reason, cross-reactivity in targets like CSF1R (involved in development of macrophages and monocytes), Flt3 (immunosuppressive<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and involved in maturation of dendritic cells<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a>), c-Met,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and Lck<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (both implicated in T-cell genesis), IRAK,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and DDR1/2 (implicated in immune function<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a>) was considered to be less desirable. Second, a set of more commonly avoided targets was considered to be undesirable from a medicinal-chemistry perspective; for example, Kdr inhibition is known to lead to hypertension<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and can give rise to dose-limiting tox; Flt3 and KIT in combination cause myelotoxicity.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Flt3 was selected a representative kinase to use as a routine counterscreen.</div><div class="NLM_p">In parallel to the HTS, a second experimental screen was run in the form of a DECL selection campaign. DECL technology offers a way of screening billions of compounds. It utilizes libraries assembled through combinatorial synthesis, with each molecule covalently attached to a DNA sequence which acts as a unique tag to enable deconvolution of the bound library members after affinity-mediated selections. This technology has emerged as an orthogonal method in the discovery of novel chemical equity for early stage drug discovery and recent reviews have highlighted the successes of the platform, with several DECL-derived molecules being progressed to the clinic.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a></div><div class="NLM_p">Finally, two further approaches were applied but are not described here as they did not lead to usable series, or their development timeline was not expected to coincide with the anticipated screening timelines. A ligand-based virtual screen was performed to explore SAR and find analogues of the initial lead compounds that were identified. However, in those cases where HTS is the key lead-finding approach, investigators will likely wait for output of the latter. Where an HTS is not practicable, virtual screening can deliver valuable leads for target validation, or even leads for development, but in this case, it was superseded by the experimental approaches. A small fragment screen was attempted as well. Because the timeline for delivery of a suitably selective series from a fragment-based design approach was not expected to align with the screening project, a smaller-scale fragment-screening approach was initiated with a focus on larger-sized fragments that would offer potential to bind to the hinge region of the kinase. While hits were found, none offered advantages that could be leveraged in development, and further efforts were deprioritized.</div><div class="NLM_p last">Here, we will discuss the results of data mining, HTS and DECL screening tracks, and outline the triaging efforts that provided us with relevant information to select series of interest. In particular, we will highlight the ability of these approaches to pick up leads across different kinase classes: ATP-competitive series (Type-I inhibitors), ATP-noncompetitive binders in the ATP site (Type-II), and noncompetitive binders binding outwith the ATP pocket (Type-III/IV).</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Data Mining</h3><div class="NLM_p">Suitable lead compounds with known Mer activity were collated from internal data and external literature. These included a number of type-II kinase compounds, one of which, NPS-1034,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> resulted in a Mer structure (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Interestingly, the two chains found in the asymmetric unit of this structure exhibited differential placement of the phenylalanine of the DFG motif (Phe742) in both the “in” and “out” conformations, suggesting a degree of mobility of these residues in the presence of ligand with this particular construct. The αC helix is in the “out” orientation, so the two protein chains exhibit what are typically referred to as type-I<sup>1</sup>/<sub>2</sub> (chain A) and type-II conformations (chain B).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The DFG-out conformer of Chain B is shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Early lead compounds. (b) Biochemical kinase assay inhibition data for data mining hit <b>2</b> (% inhibition at compound concentration of 1 μM) in biochemical binding assays for Mer and key off-targets. (c) Mer protein–ligand crystal structure of <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>). Phe742 is indicated and was found to be in the DFG Phe pocket in chain A, but out of the DFG pocket in chain B (latter shown here). (d) Binding mode of compound <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY">7AVY</a>) with DFG-in conformation highlighted as ball-and-stick. In the binding mode images, the edge of the αC helix can be seen horizontally at the top of the image. The kinase hinge loop sits bottom left, with hydrogen bonding contact shown as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A data mining exercise of equity in our collection was performed, looking specifically for selective leads that would give us potentially an option to work on prior to HTS. For this purpose, we used kinase activity data available in the AstraZeneca database as well as the HTS activity database. Analysis of a large set of single-shot inhibition data revealed one quinazoline derivative of interest, having a suitable margin to a set of off-targets, and a Mer crystal structure was obtained for this compound as well (<b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d).</div><div class="NLM_p">Further exploration of analogues of <b>2</b> led to identification of a series prior to the HTS based on a quinazoline scaffold. A shape-based virtual screen was performed using ROCS<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> to supplement a screening set of analogues, but ultimately, results were limited and hits structurally conservative. Data for three exemplars are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> together with the type-II inhibitor <b>1</b>. Apart from potency, physicochemical properties and <i>in vitro</i> pharmacokinetic properties were used at this stage to assess the quality of the equity at hand.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitor <b>1</b> and Exemplars <b>2</b>–<b>4</b> of Quinazoline Series<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0009.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub> (μL/min/10<sup>6</sup> cells)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="left">8.0</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">210</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">>5</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="char" char="."><4.3</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char="."><3</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">6.9<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char=".">5.0<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="."><3</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">130</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Mean of at least two independent potency measurements unless otherwise stated (Mer, Axl, Tyro3, and Flt3 data, Rapidfire LC–MS assays). Assay standard deviations for potency assays reported in tables can be found in Table SI1, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>. <i>n.d.</i>, not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Value obtained in alternate assay based on GLO readout (assay descriptions for alternate Axl and Flt GLO assays can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>).</p></div></div></div><div class="NLM_p last">A large number of quinazoline exemplars was available in the corporate collection and a subset of these was screened for Mer and Axl potency. A range of diversity of right-hand-side and left-hand-side substitutions was explored with a set of 82 compounds, and <b>3</b> and <b>4</b> exemplify some of the diversity of right-hand-side substituents. Quinazoline compounds generally hit both Mer and Axl, and limited selectivity was seen across the TAM kinases. Selectivity over Flt3 was generally observed for basic substituents off the 6- and 7-positions of the quinazoline. In general, good selectivity was observed across the wider kinome, but a strong bias toward Aurora kinase activity over Mer proved to be unacceptable and the series was deprioritized as a result. The series originated from a historical Aurora kinase project so this was unsurprising. No further efforts were made to optimize the selectivity of this series, as the high-throughput screening output yielded a large number of start points of greater interest.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> HTS</h3><div class="NLM_p">High-throughput screening remains a key lead-generation approach in drug discovery.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We attempted to design a cascade that would help us fulfill two requirements, in addition to finding potent Mer compounds: the necessity to identify selective leads, and the desire to pick up allosteric inhibitors, if such compounds were present in the screening collection. Simple HTS approaches typically look for inhibitory activity with samples tested at a single concentration in a single target. The requirement for selectivity introduces the necessity to measure inhibition in at least two targets but also requires us to do so at more than a single concentration to cover the range of expected primary potencies. Selectivity is a relative property: a compound with primary activity of 1 μM and activity in a second target at 30 μM would have the same selectivity margin as another compound registering activities of 1 nM and 30 nM, respectively. A single-concentration inhibition measure in two targets would not suffice to detect both cases.</div><div class="NLM_p">While the screening collection available to us contained multiple chemical series that were known ATP-competitive kinase inhibitors resulting from historical design programs, we believed that allosteric compounds (ie. Type-III/IV<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>) would be less well represented. The ATP-competitive leads were expected to be reasonably potent, having been synthesized in lead optimization campaigns, while leads for allosteric compounds were expected to be both sparser in collections, and weaker in potency. This complicated running of this kinase-target HTS, as the higher screening concentration required to find allosterics would also significantly increase the hit rate for type-I/II kinase binders that target the ATP site.</div><div class="NLM_p">To be able to fit both the requirement to find allosterics and identify selectivity, a decision was made to run the primary Mer HTS screen at a relatively high primary screening concentration of 10 μM. Use of a high ATP concentration was considered but proved to be incompatible with the assay format. The hit rate was expected to be high based on extrapolation of hit-rate data from historical kinase screens. It was anticipated that this set could be filtered pragmatically in silico, using historical kinase and promiscuity<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> data, with the ultimate aim of selecting a diverse subset for progression (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,c). The subsequent secondary screening step involved screening of the selected set of ∼30 000 actives in the primary target assay and counter-target Flt3 at multiple concentrations to ascertain selectivity. The concentrations used were 10 μM, 1 μM, and 0.1 μM, and this concentration–response range was expected to provide sufficient information to distinguish selectivity margins of one log order or better in a pIC<sub>50</sub> range of approximately 4.5 to 7.5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). Highly potent compounds still pose an issue, as their selectivity margin would not be discernible using the three selected concentrations. However, it was anticipated that the fraction of compounds with high primary Mer potency as well as high Flt3 potency would be relatively small, and their follow-up in downstream concentration–response cascade assays would not impose a large burden in terms of numbers.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cascade design for Mer hit identification. (a) Primary, secondary, and concentration–response screen setups. (b) Schematic representation of the pIC<sub>50</sub> range covered by the three screen concentrations. (c) Reduction of primary-screen hit set. SS single shot (measurement at a compound single concentration).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Outcome</h4><div class="NLM_p">The primary HTS assay run of 1.8 M compounds resulted in a high number of hits, in line with expectation. A total of just over 95 000 hits from the primary screen was processed in-silico and reduced to a set of 31 000. This was achieved by annotating the set according to their likely kinase inhibitor type, based on substructure (likely kinase hinge binder motifs) and experimental kinase activity data, including historical kinase HTS data. Sets were pruned using properties, removing compounds showing anomalously high hit rates across historical HTS assays<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (“frequent hitters”), and filtering out compounds with high lipophilicity (clogP > 5). Precedence was given to equity without reliable kinase-type annotation, as these would be more likely to be different from type-I or type-II, and potentially, more novel (set A, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). Two further sets were taken forward: one contained compounds annotated as type-I/type-II kinase inhibitors (set B, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c) and the remaining set comprised compounds of lesser interest (set C, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The latter set was further subdivided: those that were singleton hits (i.e., having no close structural analogues in the collection) were deprioritized from follow-up, as such hits were considered to be less tractable. Sets B and the remainder of set C were condensed in size significantly by taking forward a structurally diverse subset only using structural clustering followed by selection of a small set of representatives per cluster. The total of 90 583 hits was thereby reduced to a set of 30 188 hits with a physical sample available for secondary screening.</div><div class="NLM_p">Secondary screening yielded inhibition data for Mer and Flt3 at three concentrations. While single-concentration inhibition data can be somewhat noisy, the concentration range of these points enabled identification of compounds with likely Mer/Flt3 selectivity margins of at least 10-fold. More specifically, observation of Mer inhibition greater than 50% combined with a Flt3 inhibition lower than 25% (both at same screening concentration) was determined to predict a selectivity margin better than one log order. A set of 12 000 hits was selected from this round by potency and selectivity margin, and progressed subsequently to concentration–response IC<sub>50</sub> measurements in Mer, Axl, and Flt3 assays.</div><div class="NLM_p">Using the Mer, Axl, and Flt3 concentration–response data, a large number of hit series was identified, covering the range of kinase inhibition types, as well as a range of Mer/Axl/counter-target selectivity profiles. At this point, further data were generated for key exemplars of series to aid ranking and prioritization including lipophilicity, aqueous solubility, human liver microsome intrinsic clearance, and rat hepatocyte intrinsic clearance.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In addition, Mer inhibition was established in an assay with elevated ATP level to check for ATP competitiveness of the compounds. This assay aids with discerning type-I kinase inhibitors (ATP-competitive) from the other modes of action.</div><div class="NLM_p">Subsequent processing focused on identification of series, and this was done using automated means (enrichment analysis<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>) as well as visual inspection of clusters of similar compounds. A subset of 21 HTS series was designated for further follow-up (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) based on potency and selectivity profiles, physicochemical and in vitro PK data available for exemplars, and historical knowledge of series’ origins. The majority of the series were found to be ATP-competitive (type-I), which was expected given the large number of type-I compounds in the collection. One non-ATP-competitive (type-II) series was identified, and one allosteric series emerged (type-III) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). One of the hit series was found to coincide with the type-I<sup>1</sup>/<sub>2</sub> quinazoline series identified prior to the HTS. Representative series members were checked for purity, and, where necessary, repurification or resynthesis was performed followed by retesting.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Mode of Action of Series Identified by HTS and DECL Screening<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">profile</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">type-I DFG-in C-helix in</th><th class="colsep0 rowsep0" align="center">type-I<sup>1</sup>/<sub>2</sub> DFG-in C-helix out</th><th class="colsep0 rowsep0" align="center">type-II DFG-out C-helix out</th><th class="colsep0 rowsep0" align="center">type-III DFG-out C-helix out</th><th class="colsep0 rowsep0" align="center">unknown</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Mer > Axl</td><td class="colsep0 rowsep0" align="left">HTS</td><td class="colsep0 rowsep0" align="left">7 (2)</td><td class="colsep0 rowsep0" align="left">1 (1)</td><td class="colsep0 rowsep0" align="left">1 (1)</td><td class="colsep0 rowsep0" align="left">1 (1)</td><td class="colsep0 rowsep0" align="left">4 (0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DECL</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Mer = Axl</td><td class="colsep0 rowsep0" align="left">HTS</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2 (0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DECL</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Mer < Axl</td><td class="colsep0 rowsep0" align="left">HTS</td><td class="colsep0 rowsep0" align="left">3 (1)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DECL</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup><sup>a</sup></sup><p class="last">Total of 21 HTS series and 3 DECL series were taken forward. The number in parentheses specifies the number of series considered after further prioritization (HTS series). Type designation as described by Roskoski.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></div></div><div class="NLM_p last">Further triaging of the large list of series was performed using a multiobjective approach, focusing on the key aspects we deemed necessary for a suitable lead series: suitable selectivity profile for Mer/Axl kinases and the wider kinome, and magnitude of the observed selectivity; attractive physicochemical properties (or an indication that these would be achievable); origin and historical knowledge of the series; and novelty. Unlike previous analyses, which could be performed largely in-silico, this assessment was more labor-intensive and involved a small team of drug designers. The analysis led to identification of a smaller set of nine series that were progressed for testing of additional structural neighbors, crystallization in Mer and Axl protein constructs, and SAR analysis. In the following section, we will discuss a representative example from each different kinase mode-of-action class in more detail.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Type-I Series</h3><div class="NLM_p">The bulk of the HTS hits found were of type-I, that is, ATP competitive kinase inhibitors. This was not a surprise given the prevalence of such compounds in the AstraZeneca collection, having been made in the course of many historical kinase-targeting projects. Two series with excellent selectivity margins over Flt3 were prioritized, the first of which consisted of a set of bis-aminopyrimidines (BAP series). Exemplars are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. Compounds from this series showed submicromolar Mer potency. Testing in an assay with higher ATP concentration (10-fold) led to a drop-off in activity of about a log order, in line with expectation for ATP-competitive type-I inhibition. Selectivity over Axl was observed to be approximately 10-fold, and the selectivity margin versus Flt3 was greater than 10-fold. X-ray structures revealed that the binding mode for these compounds was consistent, despite differences in their substituents, with the 2-aminopyrimidine providing the kinase hinge-binding functionality. An example binding mode is shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a for <b>5</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Bisaminopyrimidine (BAP) Exemplars with Key Data<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0010.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub> (μL/min/1e6)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char="."><2.4</td><td class="colsep0 rowsep0" align="char" char="."><3.0</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">400</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="char" char="."><4.4</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="char" char=".">190</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char=".">5.9</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">4.5<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char="."><4.1<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">130</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Binding mode of <b>5</b> in Mer (BAP series, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ">7AVZ</a>). (b) Binding mode of <b>9</b> in Mer (purine series, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0">7AW0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Properties for this series are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Mer is generally a bit more potent in this series than Axl. Lipophilicity is clearly on the high side for some, and concomitantly, solubility can be low and intrinsic clearance, high. Kinome selectivity was expected to be reasonable: where data were available, good margins (better than 10-fold) over key counter-targets like Csf1r, Lck, cMet, and AurB were observed.</div><div class="NLM_p">A second series of compounds with a type-I mode of action consists of purine hinge binders and is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Again, Mer potency is somewhat higher than the affinity for Axl, and good selectivity over Flt3 and other counter-targets is observed. Lipophilicity is high for these relatively small compounds, but their size renders them reasonably ligand-efficient, exhibiting potency levels similar to those seen for the BAP series. Intrinsic clearance for these compounds varies but can be high in human liver microsomes and rat hepatocytes. High clearance can be linked to high lipophilicity, so the series was initially progressed with a view to addressing this. Crystal structures were obtained for <b>9</b> and show a binding mode where the amino hydrogen-bond donor and donor purine acceptor bind to the hinge (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Exemplars of Purine Series with Key Data<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0011.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub> (μL/min/1e6)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">6.1<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char=".">4.0<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">190</td><td class="colsep0 rowsep0" align="char" char=".">240</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="char" char="."><4.3</td><td class="colsep0 rowsep0" align="char" char=".">160</td><td class="colsep0 rowsep0" align="char" char=".">>270</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Type-I<sup>1</sup>/<sub>2</sub> and Type-II Series</h3><div class="NLM_p">The series with type-I<sup>1</sup>/<sub>2</sub> mode of action that was identified prior to running the HTS based on historical kinase data was subsequently found again when the HTS was run. This series consisted of a substituted quinazoline hinge binder core with relatively large substituents at the 4- and 6- or 7-positions (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>2</b>–<b>4</b>). The initial compound identified through data mining had a substituent at the 6-position, with a wider range of 7-substituents identified subsequently in the HTS data. High Mer potency was observed, again with greater potency for Mer than for Axl. Despite the type-II-like shape and pharmacophore, ATP competition was apparent from potency data obtained at high ATP concentration for this series, and a structure was obtained with a DFG-in conformation of the protein (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d), in line with the type-I<sup>1</sup>/<sub>2</sub> behavior.</div><div class="NLM_p">Data for a representative type-II series can be found in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The binding mode of <b>11</b> is shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, exhibiting a DFG-out-type conformation in line with a non-ATP-competitive mechanism. An alignment of <b>1</b> is shown for comparison, and it is clear that the hinge binding regions align closely for these (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and that the phenylalanine side chain that resides in the DFG-in pocket is displaced by phenyl and trifluoro groups of <b>1</b> and <b>11</b>, respectively. It can be seen that <b>11</b> extends further toward the αC helix (visible at the top in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). This series exhibited high potency but an unattractive selectivity profile. Though potential advantages of a type-II mode of action were considered (notably, noncompetition against high ATP levels in cell), the size, lipophilicity, solubility, and developability of this series were deemed to be unfavorable and it was deprioritized for those reasons.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Exemplar of Type-II Series<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0012.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub> (μL/min/1e6)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">8.2</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="left"><i>n.d.</i></td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div></div></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of type-II series. Binding mode of <b>11</b> (in green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1">7AW1</a>), and DFG-out binding mode of <b>1</b> (in orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>) for comparison. The edge of the αC helix can be seen horizontally at the top of the image and the kinase hinge loop sits bottom left.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> DNA-Encoded Chemical Library Screen</h3><div class="NLM_p">In addition to the HTS, a DNA-encoded chemical library screen was performed to identify additional start points for lead generation. The campaign was designed to utilize the ability of DECL technology to explore multiple selection experiments, with different target proteins or varying conditions, in parallel. This allowed for evaluation of several Mer constructs, both the long and short forms, along with Axl and three off-target proteins, namely Flt3, Lck, and Kdr. Selection experiments with the addition of ATP along with the Mer constructs were included to classify any resulting binders on the basis of their competitiveness with ATP. In total, the campaign comprised 20 individual selection conditions that were run side-by-side to identify library compounds able to bind both long and short forms of Mer with selectivity over several off-target kinase proteins and distinguish different binding modes (Table SI6, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">The affinity-mediated screening was carried out against a mixture of 46 DNA-encoded small molecule libraries, from both X-Chem’s internal collection and AstraZeneca’s custom libraries, totaling more than 90 billion compounds. Following the screening and sequencing, analysis of the output was carried out to identify enriched features for the Mer binding.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Strongly enriched features were grouped into sets of highly structurally related compounds designated as families. Families were removed if they were also enriched in any of the off-target selections, or were known to bind many targets from historical data, and were prioritized based on physicochemical properties. This led to identification of 67 families of interest across both ATP-competitive and noncompetitive profiles.</div><div class="NLM_p">For each family, the most prevalent building blocks at each cycle of library synthesis for all members were determined (Tables SI7–SI9, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). This provides an overview of the variability within the designated cluster of compounds, that is, an indication of SAR for structurally related active compounds. A representative compound from each family was designed and synthesized. As a general rule, off-DNA compounds were made with a capping methyl group present at the DNA-linker attachment point. The synthesized compounds were screened in Mer, Axl, and Flt3 biochemical assays and based on these results, a subset was taken forward for further profiling with in vitro PK, wider kinase panel screening, and ultimately X-ray crystallography. In this manner, three compounds, <b>12</b>, <b>13</b>, and <b>14</b>, representing three distinct families, designated A, B, and C, respectively, were selected for further study and will be discussed in more detail (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Each compound showed a desired profile of submicromolar Mer activity and a selectivity margin >50 fold against Flt3.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Compounds and Data for Selected off-DNA Exemplars from DECL Screen<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0013.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub>(μL/min/10<sup>6</sup>cells)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">aq. solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">>300</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char="."><4.0<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">260</td><td class="colsep0 rowsep0" align="char" char=".">>300</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char="."><0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Tested <i>n</i> = 1.</p></div></div></div><div class="NLM_p">The exemplar molecule <b>12</b> originated from a 3-cycle library, in which molecules are built up from a core (cycle A, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a), attached to a DNA tag via a linker, using SNAr and subsequent coupling steps (cycles B, C). Compound <b>12</b> showed a binding mode typical of an ATP-competitive (Type I) inhibitor. Its protein–ligand structure is displayed in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Schematic of library member. (b) Binding mode of azaindole <b>12</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The central region of this family A was defined by the diamine building block used in cycle B and included a variety of 5-substituted azaindoles in the terminal cycle of chemistry (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>, Table SI7). Cycle A was a low diversity cycle with only 33 total building blocks included in the library and so limited variation could be observed at this position. While the physicochemical properties of <b>12</b> were suboptimal having high lipophilicity, the compound offered a dual Mer/Axl activity profile with >60 fold selectivity over Flt3. When profiled across a wider panel of kinase targets (131 kinases were tested at a concentration of 1 μM), azaindole <b>12</b> was found to be somewhat promiscuous with >75% inhibition against 19 targets (including Mer and Axl; details in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">A small set of analogues were synthesized based on the types of Cycle C variation observed in the prevalence table (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Pleasingly, all maintained good levels of activity at Mer with pIC<sub>50</sub> > 7.0 and had comparable activity when tested in Axl, retaining the dual Mer/Axl profile. Replacement of the 5-Cl with a methyl group (<b>15</b>) provided the most potent azaindole example with a slightly reduced LogD. Compound <b>16</b>, incorporating a trifluoromethyl group, showed an excellent margin of selectivity over both Tyro3 and Flt3 of >1000-fold.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. SAR Exploration of Azaindole Series<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0014.gif" alt="" id="fx6" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Mer pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Axl pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Tyro3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Flt3 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">rat hepatocyte CL<sub>intrinsic</sub> (μL/min/10<sup>6</sup>cells)</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>intrinsic</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">aq. solubility (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">>270</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char="."><1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char="."><4.0<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="."><4.0</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="char" char=".">130</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char="."><1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char="."><1.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">Tested <i>n</i> = 1.</p></div></div></div><div class="NLM_p">The unsubstituted azaindole <b>17</b> showed no improvement in aqueous solubility, despite a decrease in lipophilicity at a similar level of Mer activity.</div><div class="NLM_p">The second DECL series of interest, family B, came from a 3-cycle library consisting of two rounds of amino acid building blocks followed by capping with a set of carboxylic acids (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). The compound resides in the pocket near the αC helix and displays interactions with the hinge in the ATP pocket region (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The activation loop sits in a DFG-out conformation (i.e., as found in type-II kinase inhibitor binding modes). Its lack of competition with ATP was confirmed by absence of a drop-off in potency in a high ATP Mer assay (Table SI2, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). This was in agreement with the profile of the family members which showed enrichment both in absence and presence of ATP. The kinase selectivity profile across a large set of kinases was found to be excellent for the exemplar <b>13</b> (Table SI3, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). The series was not progressed due to its high molecular weight, peptidic nature and relatively high level of Tyro3 activity.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Schematic of library member. (b) Binding mode of <b>13</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4">7AW4</a>); type-I azaindole <b>12</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>) is aligned to show the location of the ATP pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The final hit compound <b>14</b>, exemplifying family C, came from a 2-cycle library containing a 1,3,4-oxadiazole core (library schematic in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). Family C was the largest of the three investigated, with over 100 members showing a statistically significant enrichment in the DECL selection output. The oxadiazole <b>14</b> was found to exhibit a Type-I<sup>1</sup>/<sub>2</sub> binding mode (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b). The imidazopyridine formed a hydrogen-bonding interaction with the kinase hinge and the terminal chloro-phenyl group extends out toward the αC helix. The TAM kinase activity profile showed a distinct preference for Mer over Axl and Tyro3, as well as an excellent margin over Flt3 (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). Selectivity across a larger set of kinases was found to be excellent (Table SI3, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). To expand the set of compounds for SAR evaluation beyond the representative oxadiazole <b>14</b>, key examples were selected with the use of the prevalence table generated (Table SI9, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>) as well as remining of the DECL screening data to identify any further near neighbors.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Schematic of library member (b) Binding mode of oxadiazole <b>14</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2">7AW2</a>). The kinase hinge is oriented bottom left, and the αC helix is visible horizontally, at the top.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. SAR Results for Oxadiazole Series<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0015.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0016.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">See <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> footnotes for notes.</p></div></div><div></div></div><div class="NLM_p">Compound <b>18</b>, containing a chromane group, was selected for synthesis as it offered an alternative hinge binding motif to test its importance. While the terminal biaryl group conferred high lipophilicity to the molecule, retention of the Mer activity and selectivity over Flt3 was encouraging. The variation of the terminal biaryl group was considered next. Here, a wide range of substituents were found to be enriched within the family, covering biaryls, substituted phenyls and pyridines (see Table SI9, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). A selection was made based on diversity and enrichment and exemplars were progressed for off-DNA synthesis. Their results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>.</div><div class="NLM_p">Introduction of a pyridyl in <b>19</b> resulted in an expected lowering of logD to 3.3 compared to <b>18</b>, although this came with lower Mer potency. LLE ligand efficiency remained at a level similar to the original hit <b>14</b>. Somewhat higher HLM clearance was observed, with solubility increased to 10 μM. Replacement of the chloro-phenyl with an ether linked aliphatic terminal group in <b>20</b> was tolerated, leading to a lowered potency at relatively high lipophilicity, and showing higher levels of clearance than <b>14</b>. Modification of the terminal group to a smaller aliphatic CF<sub>3</sub> group, and changing the central ring to a chloropyridyl (<b>21</b>) yielded a lipophilic ligand efficiency (LLE) of 3.5, similar to those observed for <b>14</b> (LLE 3.5) and <b>19</b> (LLE 3.6). Log <i>D</i> of these compounds was significantly lower at 2.6 and this was accompanied by an improved aqueous solubility. The observed clearance in rat hepatocytes for this compound was low, likely in part as a result of the log <i>D</i> being significantly lower than that of others.</div><div class="NLM_p">A reduction in lipophilicity compared to the original oxadiazole hit was achieved likewise by the introduction of a dimethylamine in compound <b>22</b>, and this change was combined with an <i>ortho</i>-methyl group on the central ring. This lowered the lipophilicity as expected, lowered potency and LLE compared to parent <b>14</b> from 3.5 to 2.9. The final analogue <b>23</b> explored introduction of a benzimidazole system directly attached to the oxadiazole core, which was also tolerated, but in this case, the selectivity over Flt3 was eroded, making this modification less attractive.</div><div class="NLM_p last">Taken together, the series was deemed attractive from a viewpoint of potency, selectivity, and novelty, and early modifications suggested that the high lipophilicity of the parent could be mitigated.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17845" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17845" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We describe the execution of an extensive lead generation campaign for identification of Mer and Axl inhibitors with a range of selectivity profiles and different modes of action. The campaign applied a diverse set of approaches to find leads including data mining, high-throughput screening, and screening of DNA-encoded chemical libraries. Data mining of historical data offered an early start on a series, which gave rise to lead compounds with reasonable selectivity and potency, but this series of quinazolines was ultimately not pursued because of unacceptable off-target activities. Two lead-finding workstreams were started concurrently and their coordinated output provided a plethora of novel options to consider.</div><div class="NLM_p">The first workstream was a Mer HTS campaign and was designed to run at a relatively high screening concentration to be able to pick up weaker but potentially, more novel series. The numerous kinase-targeted compounds in the corporate collection were expected to lead to a high hit rate, and a cascade was designed to filter out and prioritize those compounds with a suitable selectivity profile across primary and off-targets. A large number of structural classes was identified, exhibiting diverse modes of action (type-I/I<sup>1</sup>/<sub>2</sub>/II/III, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Many of the series identified were deprioritized for reasons of off-target activities or were just less favorable start points due to their physicochemical property profile. Remaining series were diverse in terms of kinase mode of inhibition, spanning type-I, I<sup>1</sup>/<sub>2</sub>, II, and III inhibitors, as well as selectivity profiles ranging from dual Mer/Axl inhibitors to more selective variants.</div><div class="NLM_p">A second workstream constituted a DNA-encoded library screen, and provided orthogonality to the HTS efforts both in terms of methodology and compound screening collection. The campaign was designed, like the HTS approach, to prioritize selective and potent series. The DECL format allows for relatively facile ways to screen for different modes of inhibition (e.g., ATP/non-ATP competitive) as well as filter against several off-targets, provided that their relevant protein constructs are available, and this was taken advantage of by deconvoluting against a wider range of off-targets (Flt3, Lck, Kdr) at the stage of series selection. Like with the conventional HTS, several series were identified, but only a small set was progressed. A limited number of their members were selected for off-DNA synthesis utilizing observed screening enrichments, allowing for a quick selection of potentially potent compounds to explore SAR. Their analysis provided rapid insight in the developability of the series. From these DECL series, a novel class of type-I<sup>1</sup>/<sub>2</sub> oxadiazole inhibitors was selected for development alongside the prioritized HTS series. Multiple series resulting from the HTS and DECL efforts were taken forward into lead optimization campaigns and their development will be described separately.</div><div class="NLM_p last">In summary, this lead generation campaign successfully provided a very wide variety of chemical start equity by coordinating the timelines of two key orthogonal approaches, offering a choice of series with a range of selectivity profiles and modes-of-action. The HTS was designed to focus on selective compounds using multiple screening concentrations, in the knowledge that this would require dealing with a large number of kinase-targeting hits. HTS proved to be a reliable technique to identify relevant equity from the corporate collection, and yielded a large number of potential start points, covering different modes of kinase binding. DNA-encoded libraries offer a facile way to supplement these series with additional novelty, and this work led to an additional series being taken forward into development. The HTS and DECL approaches were performed concurrently and timed to deliver their results at the start of the development campaign. The development of key series will be described in more detail elsewhere.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66572" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66572" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Assays</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Mer Biochemical Activity</h4><div class="NLM_p last">The ability of inhibitors to cause inhibition of Mer(6His_R528-M999_Thrombin-Avi-Flag, dephosphorylated) in vitro was assessed using Rapidfire LC–MS method that quantified Axltide (CKKSRGDYMTMQIG-acid, Cambridge Research Biochemicals) peptide phosphorylation levels. Mer (1 nM final) in assay buffer (20 mM HEPES pH 7.5, 0.006% Brij-35, 0.5 mM TCEP, 10 mM, 10 mM Mg(OAc)<sub>2</sub>, 0.02% Pluronic-F127) was added to compound plates using a liquid dispenser; Multidrop Combi. Assay plates were equilibrated for 30 min before the reaction was initiated by addition of Axltide (10 μM final) and ATP (at Km<sub>app</sub>; 80 μM final) substrates in assay buffer. The reaction was allowed to progress for 120 min before being stopped with 0.1% formic acid in water. The Axltide peptide substrate and the phosphorylated Axltide phosphopeptide product levels were determined by mass spectrometry (Rapidfire RF360 and Agilent triple quad mass spectrometer system) using aqueous phase; 0.1% formic acid in water and organic phase; 0.1% formic acid in 90% methanol on type E (C8 Rapidfire cartridge) to elute. Ratios were plotted to generate concentration response profiles and the dose–response curves were fit to the data using the nonlinear regression analysis; 4 parameter logistic smart fit method in the Assay analyzer and Condoseo applications of the Genedata Screener software (Genedata, Inc., Basel, Switzerland).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> AXL Biochemical Activity</h4><div class="NLM_p last">The ability of inhibitors to cause inhibition of GST-AXL(464–885) in vitro was assessed using Rapidfire LC–MS method that quantified Axltide (CKKSRGDYMTMQIG-acid, Cambridge Research Biochemicals) peptide phosphorylation levels. AXL (5 nM final) in assay buffer (20 mM HEPES pH 7.5, 0.006% Brij-35, 0.5 mM TCEP, 10 mM, 10 mM Mg(OAc)<sub>2</sub>, 0.02% Pluronic-F127) was added to compound plates using a liquid dispenser; Multidrop Combi. Assay plates were equilibrated for 30 min before the reaction was initiated by addition of Axltide (10 μM final) and ATP (at Km<sub>app</sub>; 75 μM final) substrates in assay buffer (the assay was also performed without the preincubation step). The reaction was allowed to progress for 180 min before being stopped with 0.1% formic acid in water. The Axltide peptide substrate and the phosphorylated Axltide phosphopeptide product levels were determined by mass spectrometry (Rapidfire RF360 and Agilent triple quad mass spectrometer system) using aqueous phase; 0.1% formic acid in water and organic phase; 0.1% formic acid in 90% methanol on type E (C8 Rapidfire cartridge) to elute. Ratios were plotted to generate concentration response profiles and the dose–response curves were fit to the data using the nonlinear regression analysis; 4 parameter logistic smart fit method in the Assay analyzer and Condoseo applications of the Genedata Screener software (Genedata, Inc., Basel, Switzerland).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Tyro3 Biochemical Activity</h4><div class="NLM_p last">The ability of inhibitors to cause inhibition of GST-TYRO3(453–890) in vitro was assessed using Rapidfire LC–MS method that quantified Axltide (CKKSRGDYMTMQIG-acid, Cambridge Research Biochemicals) peptide phosphorylation levels. TYRO3 (1 nM final) in assay buffer (20 mM HEPES pH 7.5, 0.006% Brij-35, 0.5 mM TCEP, 10 mM, 10 mM Mg(OAc)<sub>2</sub>, 0.02% Pluronic-F127) was added to compound plates using a liquid dispenser; Multidrop Combi. Assay plates were equilibrated for 30 min before initiating the reaction by addition of Axltide (10 μM final) and ATP (at Km<sub>app</sub>; 190 μM final) substrates in assay buffer. The reaction was allowed to progress for 180 min before being stopped with 0.1% formic acid in water. The Axltide peptide substrate and the phosphorylated Axltide phosphopeptide product levels were determined by mass spectrometry (Rapidfire RF360 and Agilent triple quad mass spectrometer system) using aqueous phase; 0.1% formic acid in water and organic phase; 0.1% formic acid in 90% methanol on type E (C8 Rapidfire cartridge) to elute. Ratios were plotted to generate concentration response profiles and the dose–response curves were fit to the data using the nonlinear regression analysis; 4 parameter logistic smart fit method in the Assay analyzer and Condoseo applications of the Genedata Screener software (Genedata, Inc., Basel, Switzerland).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> FLT3 Biochemical Activity</h4><div class="NLM_p">The ability of inhibitors to cause inhibition of GST-TEV-6His-FLT3(H564–S993) in vitro was assessed using Rapidfire LC–MS method that quantified Axltide (CKKSRGDYMTMQIG-acid, Cambridge Research Biochemicals) peptide phosphorylation levels. AXL (5 nM final) in assay buffer (20 mM HEPES pH 7.5, 0.006% Brij-35, 0.5 mM TCEP, 10 mM, 10 mM Mg(OAc)<sub>2</sub>, 0.02% Pluronic-F127) was added to compound plates using a liquid dispenser; Multidrop Combi. Assay plates were equilibrated for 30 min before the reaction was initiated by addition of Axltide (10 μM final) and ATP (at Km<sub>app</sub>; 595 μM final) substrates in assay buffer. The reaction was allowed to progress for 75 min before being stopped with 0.1% formic acid in water. The Axltide peptide substrate and the phosphorylated Axltide phosphopeptide product levels were determined by mass spectrometry (Rapidfire RF360 and Agilent triple quad mass spectrometer system) using aqueous phase; 0.1% formic acid in water and organic phase; 0.1% formic acid in 90% methanol on type E (C8 Rapidfire cartridge) to elute. Ratios were plotted to generate concentration response profiles, and the dose–response curves were fit to the data using the nonlinear regression analysis; 4 parameter logistic smart fit method in the Assay analyzer and Condoseo applications of the Genedata Screener software (Genedata, Inc., Basel, Switzerland).</div><div class="NLM_p last">A small number of Axl and Flt potency data pIC<sub>50</sub>s were generated with alternate assays employing GLO assay technology (as annotated in the tables). Assay descriptions for the GLO assays can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> DNA Encoded Library Screens</h3><div class="NLM_p">All affinity-mediated selections comprised screening of DNA-Encoded Chemical Libraries (DECL) as described by Cuozzo et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> utilizing purified protein reagents (vide infra) as follows at the indicated concentrations: [6His-Mer(R528-M999)-Thrombin-Avi-Flag Activated: 5.7, 2.9, 0.57, and 0.057 μM, 6His-Mer(R528-M999)-Thrombin-Avi-Flag Partially Dephosphorylated: 5.7 and 2.9 μM, 6His-Mer(R528-M999)-Thrombin-Avi-Flag Dephosphorylated: 5.7 and 2.9 μM, 6His-thrombin-Mer(E571-V864): 9.3, 0.93, and 0.093 μM, 6His-thrombin-Mer(E571-V864) Activated: 9.3, 8.35, 0.835, and 0.0835 μM, 6His-TEV-SHM-Kdr(V789–V1356)-LE-Flag: 4.8 μM, GST-Axl(464–885), Carna Biosciences Cat# 08–107:2.3 μM, GST-FLT3(564–993), Carna Biosciences Cat# 08–154:2.2 μM and GST-LCK(1–509), Carna Biosciences Cat# 08–170:2.0 μM]. Library (40 μM) and the indicated proteins were incubated in solution for 1 h in a volume of 60 μL in 1× selection buffer. The 1× Selection buffer consisted of 20 mM HEPES pH 7.2, 8 mM NaOAc, 134 mM KOAc, 4 mM NaCl, 0.8 mM Mg(OAc)<sub>2</sub>, 1 mg/mL sheared salmon sperm DNA, 1 mM TCEP, 5 mM Imidazole, 5 mM MgCl<sub>2</sub>, 0.005% Brij35, 0.01% Pluronic127. For the 6His-Mer(R528-M999)-Thrombin-Avi-Flag Partially Dephosphorylated construct, both of the Mer(E571-V864) constructs and the GST-Axl(464–885) construct, an additional selection condition containing both target and 5 mM ATP+10 mM MgCl<sub>2</sub>, were run in parallel. ATP was preincubated with the target in 1× selection buffer for 0.5 h prior to addition of the DNA-encoded library. For each target, an additional selection condition containing no target was run in parallel. A separate ME200 tip (Phynexus) containing 5 μL of His-Select HF nickel (PTR-92–05–48), Anti-FLAG M2 (PTR-92–05-AP and Sigma Cat# A2220), or glutathione (PTR 92–05–04) affinity matrix was prewashed 3 times in 200 μL of fresh 1× selection buffer for each selection condition. A no target control selection was completed using each of the three affinity matrix tips listed. Kdr and Mer selections using each of the Mer constructs at all the concentrations except 2.9 μM were completed using the His-Select HF nickel affinity matrix tips. Mer selections at 2.9 μM Mer were completed using the Anti-FLAG M2 affinity matrix tips. Lck, Flt3, and Axl selections were all completed using the glutathione affinity matrix tips. Each incubation mixture was separately captured with 20 passages over the appropriate ME200 tip for a total of 0.5 h. The bound protein on the ME200 tip was washed 8 times with 200 μL of fresh 1× selection buffer. Bound library members were eluted by incubating the ME200 tip with 60 μL of fresh 1× selection buffer at 85 °C for 5 min. This eluate was incubated with 20 passages over a fresh, prewashed ME200 tip containing 5 μL of respective affinity matrix to remove any eluted protein. This selection process was run a second time using 30 μL of the eluate of the first selection in place of the input DNA-encoded library and on-DNA controls and using no target, fresh target, or fresh target with competitive inhibitor or ATP as appropriate. The eluate of the second round of selection was PCR amplified in a volume of 200 μL with 5′ and 3′ primers (0.5 μM each) and 1× Platinum PCR Supermix (Invitrogen 11306–016) with 15–25 cycles of [denaturation 94 °C for 30 s, annealing 55 °C for 30 s, and extension 72 °C for 120 s] until the double-stranded amplification products were clearly visible on an ethidium-stained 4% agarose gel. These primers include Illumina READ1 or READ2 sequences as required for sequencing on an Illumina HiSeq 2500. PCR-amplified selection output was then sequenced on an Illumina HiSeq 2500. Sequence reads totaled approximately 2.2 billion over the entirety of the selections. Sequence data were parsed, error-containing sequences were disregarded, amplification duplicates were removed, and building block and chemical scheme encodings were decoded and reported along with associated calculated statistical parameters.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Protein Purification</h4><div class="NLM_p last">Proteins for the DECL work were obtained as follows: GST-Axl (464–885), GST-Flt3 (564–993), and GST-Lck (1–509) were purchased from Carna Biosciences. Mer full intracellular domain (R528-M999) was expressed in <i>S. frugiperda</i> as a fusion protein with the following tags and cleavage sites: GST-TEV-6His-Mer (R528-M999)-thrombin-Avi-Flag. Cells were coinfected with a separate baculovirus encoding PTPN1 to prevent autophosphorylation. Protein was purified via immobilized metal affinity chromatography (IMAC) followed by proteolytic cleavage of the GST tag. Low level phosphorylation was detected, so additional in vitro dephosphorylation was carried out with lambda phosphatase. The kinase domain of Mer was expressed in <i>E. coli</i> with an N-terminal hexahistidine tag (6His-thrombin-Mer (E571-V864)) and purified by IMAC followed by size exclusion chromatography (SEC). Kdr full intracellular domain construct 6His-TEV-Kdr(V789–V1356)-Flag was expressed in <i>S. frugiperda</i> and purified by IMAC and SEC. Activation of Mer and Kdr was carried out by in vitro autophosphorylation followed by SEC. The phosphorylation state of all protein batches was verified by mass spectrometry.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> X-ray Crystallography</h3><div class="NLM_p">Mer protein was produced as described previously<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> by recombinant expression of the kinase domain (E571-V864) in <i>E. coli</i>. Most of the structures reported here were obtained by following a soaking protocol<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and belong to space group C2221, or by cocrystallization of compound with wild-type or K591R/K693R/K702R/K856R protein. For protein construct and crystallization details, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Table SI4</a>. Single crystal X-ray diffraction was measured at Diamond Light Source (beamlines I04 and I04–1) and Soleil (beamlines Proxima I and II). Data processing and structure solution as described by Pflug et al.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> with a summary of relevant conditions are shown in Table SI4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>). Coordinates and experimental data have been deposited with the Protein Data Bank; for PDB codes, see Table SI4 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Synthetic Chemistry</h4><div class="NLM_p">All solvents and chemicals used were reagent grade. Anhydrous solvents THF, DCM, and DMF were purchased from Aldrich. Flash column chromatography was carried out using prepacked silica cartridges (from 4 g up to 330 g) from Redisep or Silicycle and eluted using an Isco Companion system. Purity and characterization of compounds were established by a combination of liquid chromatography–mass spectroscopy (LC–MS), gas chromatography–mass spectroscopy (GC–MS), and NMR analytical techniques and was >95% for all compounds. <sup>1</sup>H NMR spectra were recorded on a Varian INOVA (600 MHz), Varian Gemini 2000 (300 MHz), or Bruker Avance DPX400 (400 MHz) and were determined in CDCl<sub>3</sub>, DMSO-<i>d</i><sub>6</sub>, or MeOH-<i>d</i><sub>4</sub>. Chemical shifts are reported in ppm relative to TMS (0.00 ppm) or solvent peaks as the internal reference. Splitting patterns are indicated as follows: s, singlet; d, doublet; t, triplet; m, multiplet; br, broad peak. Elevated temperatures were used where necessary to sharpen broad NMR peaks due to rotamers and the temperature used is noted for such compounds. Merck precoated TLC plates (silica gel 60 F<sub>254</sub>, 0.25 mm, art. 5715) were used for TLC analysis.</div><div id="sec4_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> <i>N</i>-(3-Fluoro-4-((3-phenyl-1H-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)oxy)phenyl)-2-(4-fluorophenyl)-1,5-dimethyl-3-oxo-2,3-dihydro-1H-pyrazole-4-carboxamide (<b>1</b>)</h5><div class="NLM_p last">Purchased from commercial supplier (Selleckchem). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.69 (s, 3H), 3.35 (s, 3H), 6.28 (d, <i>J</i> = 5.5 Hz, 1H), 7.15–7.22 (m, 1H), 7.24–7.37 (m, 4H), 7.39–7.47 (m, 2H), 7.47–7.55 (m, 2H), 7.59 (d, <i>J</i> = 2.6 Hz, 1H), 7.68 (dd, <i>J</i> = 1.2, 8.3 Hz, 2H), 7.91 (dd, <i>J</i> = 2.4, 13.1 Hz, 1H), 8.08 (d, <i>J</i> = 5.5 Hz, 1H), 10.89 (s, 1H), 12.02 (d, <i>J</i> = 2.2 Hz, 1H); HRMS (ES<sup>+</sup>) for C<sub>31</sub>H<sub>24</sub>N<sub>5</sub>O<sub>3</sub>F<sub>2</sub> (M+H)<sup>+</sup>: calcd 552.1847; found 552.1867.</div></div><div id="sec4_3_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> <i>N</i>-(2-(2-Cyclopropylethoxy)pyrimidin-5-yl)-1,1-diphenylmethanimine</h5><div class="NLM_p last">Tris(dibenzylideneacetone)dipalladium(0) (355 mg, 0.39 mmol) was added to a mixture of 5-bromo-2-(2-cyclopropylethoxy)pyrimidine (2.69 g, 11.1 mmol), diphenylmethanimine (2.94 g, 16.2 mmol), cesium carbonate (5.40 g, 16.6 mmol), and 2,2′-bis(diphenylphosphino)-1,1′-binaphthyl (525 mg, 0.84 mmol) in 1,4-dioxane (20 mL). The resulting mixture was stirred at 100 °C for 4 h under nitrogen. Water (30 mL) was poured into the mixture, and the aqueous layer was extracted with EtOAc. The organic layer was washed by brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford <i>N</i>-(2-(2-cyclopropylethoxy)pyrimidin-5-yl)-1,1-diphenylmethanimine (2.96 g, 78%) as a red oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 0.04–0.13 (2H, m), 0.33–0.48 (2H, m), 0.71–0.78 (1H, m), 1.56 (2H, q), 4.22 (2H, t), 7.17–7.26 (2H, m), 7.39–7.44 (3H, m), 7.48–7.53 (2H, m), 7.54–7.61 (2H, m), 7.68–7.72 (2H, m), 8.03 (2H, s); MS: (ESI) [M + H]<sup>+</sup> 344.</div></div><div id="sec4_3_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 4-((2-(2-Cyclopropylethoxy)pyrimidin-5-yl)amino)-7-methoxyquinazolin-6-ol</h5><div class="NLM_p last">Concentrated hydrochloric acid (0.516 mL, 6.28 mmol) was added to a mixture <i>N</i>-(2-(2-cyclopropylethoxy)pyrimidin-5-yl)-1,1-diphenylmethanimine (708 mg, 2.06 mmol) and 4-chloro-7-methoxyquinazolin-6-ol (478 mg, 2.27 mmol) in tetrahydrofuran (15 mL), and the resulting mixture was stirred at rt for 16 h. The reaction mixture was concentrated in vacuo to afford the crude product. The crude product was purified by flash C18-flash chromatography, elution gradient 0 to 50% MeCN in water (0.1% NH<sub>4</sub>HCO<sub>3</sub>). Pure fractions were evaporated to dryness to afford 4-((2-(2-cyclopropylethoxy)pyrimidin-5-yl)amino)-7-methoxyquinazolin-6-ol (460 mg, 63%) as a orange solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 0.06–0.21 (2 H, m), 0.35–0.50 (2 H, m), 0.72–0.92 (1 H, m), 1.58–1.71 (2 H, m), 3.97 (3 H, s), 4.35 (2 H, t), 7.21 (1 H, s), 7.70 (1 H, s), 8.40 (1 H, s), 8.93 (2 H, s), 9.51 (1 H, s), 9.73 (1 H, s); MS: (ESI) [M + H]<sup>+</sup> 354.</div></div><div id="sec4_3_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> <i>N</i>-(2-(2-Cyclopropylethoxy)pyrimidin-5-yl)-7-methoxy-6-(piperidin-4-ylmethoxy)quinazolin-4-amine (<b>2</b>)</h5><div class="NLM_p last">A solution of triphenylphosphine (118 mg, 0.45 mmol) and di<i>tert</i>-butyl azodicarboxylate (104 mg, 0.45 mmol) in tetrahydrofuran (1 mL) was added to a solution of 4-((2-(2-cyclopropylethoxy)pyrimidin-5-yl)amino)-7-methoxyquinazolin-6-ol (80 mg, 0.23 mmol) and 2-(2-cyclopropylethoxy)pyrimidin-5-amine (35 mg, 0.30 mmol) in tetrahydrofuran (0.5 mL). The resulting solution was stirred at 60 °C for 2 h. The reaction mixture was cooled to rt and concentrated in vacuo to afford the crude product. The crude product was purified by preparative HPLC (Xterra, 19 × 100 mm<sup>2</sup> 5 μm) using decreasingly polar mixtures of water (containing 2 g/L (NH<sub>4</sub>)<sub>2</sub>CO<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-(2-(2-cyclopropylethoxy)pyrimidin-5-yl)-7-methoxy-6-(piperidin-4-ylmethoxy)quinazolin-4-amine (67 mg, 65%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ −0.1–0.05 (m, 2H), 0.26–0.35 (m, 2H), 0.64–0.76 (m, 1H), 1.31–1.46 (m, 2H), 1.53 (q, <i>J</i> = 6.8 Hz, 2H), 1.86 (d, <i>J</i> = 12.7 Hz, 2H), 1.97–2.13 (m, 1H), 2.83 (q, <i>J</i> = 12.3 Hz, 2H), 3.22 (d, <i>J</i> = 12.3 Hz, 2H), 3.86 (s, 3H), 3.90 (d, <i>J</i> = 6.3 Hz, 2H), 4.26 (t, <i>J</i> = 6.7 Hz, 2H), 7.23 (s, 1H), 7.86 (s, 1H), 8.35 (d, <i>J</i> = 10.3 Hz, 1H), 8.74 (s, 2H); HRMS (ES<sup>+</sup>) for C<sub>24</sub>H<sub>31</sub>N<sub>6</sub>O<sub>3</sub> (M+H)<sup>+</sup>: calcd 451.2458; found 451.2471.</div></div><div id="sec4_3_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>N</i>-(5-((7-(3-(Cyclopentylamino)propoxy)-6-methoxyquinazolin-4-yl)amino)pyrimidin-2-yl)-1-methyl-1H-pyrrole-2-carboxamide (<b>3</b>)</h5><div class="NLM_p last">Cyclopentylamine (96 mg, 1.12 mmol) was added to a mixture of <i>N</i>-(5-((7-(3-chloropropoxy)-6-methoxyquinazolin-4-yl)amino)-pyrimidin-2-yl)-1-methyl-1H-pyrrole-2-carboxamide (105 mg, 0.225 mmol), potassium carbonate (32 mg, 0.23 mmol), and tetrabutylammonium iodide (8.30 mg, 0.0225 mmol) in dimethylacetamide (3 mL). The resulting solution was stirred at 100 °C for 3 h. The reaction mixture was cooled to rt and concentrated in vacuo to afford the crude product. The crude product was purified by preparative HPLC using decreasingly polar mixtures of water (containing 0.2% trifluoroacetic acid) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-(5-((7-(3-(cyclopentylamino)propoxy)-6-methoxyquinazolin-4-yl)amino)pyrimidin-2-yl)-1-methyl-1H-pyrrole-2-carboxamide. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.21 (d, <i>J</i> = 18.9 Hz, 4H), 1.58 (s, 4H), 1.72 (s, 2H), 1.98–2.04 (m, 2H), 2.14–2.18 (m, 1H), 3.89 (s, 3H), 4.00 (s, 3H), 4.29 (t, <i>J</i> = 6.1 Hz, 2H), 6.10 (dd, <i>J</i> = 4.0, 2.5 Hz, 1H), 7.03–7.08 (m, 1H), 7.17 (dd, <i>J</i> = 4.1, 1.7 Hz, 1H), 7.28 (s, 1H), 7.86 (s, 1H), 8.38 (s, 1H), 8.51 (s, 1H), 9.07 (s, 2H), 9.75 (s, 1H), 10.48 (s, 1H); HRMS (ES<sup>+</sup>) for C<sub>27</sub>H<sub>33</sub>N<sub>8</sub>O<sub>3</sub> (M+H)<sup>+</sup>: calcd 517.2676; found 517.2679.</div></div><div id="sec4_3_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> <i>N</i>-(2-(3-Cyclopentylpropoxy)pyrimidin-5-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>4</b>)</h5><div class="NLM_p last">A mixture of 4-chloro-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazoline<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (101 mg, 0.301 mmol) and 2-(3-cyclopentylpropoxy)pyrimidin-5-amine (73 mg, 0.331 mmol) in dimethylacetamide (1 mL) was stirred at 80 °C for 5 min. The mixture was removed from the heat, and HCl (4 M in dioxane, 0.15 mL, 0.602 mmol) was added. The resulting solution was stirred at 80 °C for 30 min. The reaction mixture was cooled to rt to afford the crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% MeOH in DCM containing 2% NH<sub>4</sub>OH. Pure fractions were evaporated to dryness to afford <i>N</i>-(2-(3-cyclopentylpropoxy)pyrimidin-5-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (82 mg, 53%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.08 (m, 2H), 1.48 (m, 12H), 1.76 (m, 5H), 1.92 (m, 2H), 2.32 (m, 4H), 2.40 (t, 2H), 3.95 (s, 3H), 4.15 (t, 2H), 4.28 (t, 2H), 7.15 (s, 1H), 7.65 (s, 1H), 8.40 (s, 1H), 8.86 (s, 2H), 9.55 (br s, 1H); HRMS (ES<sup>+</sup>) for C<sub>29</sub>H<sub>41</sub>N<sub>6</sub>O<sub>3</sub> (M+H)<sup>+</sup>: calcd 521.3240; found 521.3256.</div></div><div id="sec4_3_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (<i>R</i>)-N2-(4-(Cyclopropylmethoxy)-3,5-difluorophenyl)-5-(3-methylpiperazin-1-yl)-N4-(tetrahydro-2H-pyran-4-yl)pyrimidine-2,4-diamine (<b>5</b>)</h5><div class="NLM_p last">Compound synthesized as part of library; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ −0.07–0.02 (m, 2H), 0.22–0.32 (m, 2H), 0.89–0.94 (m, 1H), 0.97 (d, <i>J</i> = 6.4 Hz, 3H), 1.45 (ddq, <i>J</i> = 17.5, 11.9, 5.2 Hz, 2H), 1.61 (d, <i>J</i> = 11.5 Hz, 2H), 2.44 (dd, <i>J</i> = 18.8, 7.5 Hz, 1H), 2.57 (d, <i>J</i> = 8.2 Hz, 1H), 2.68–2.8 (m, 2H), 3.05 (s, 2H), 3.17 (t, <i>J</i> = 11.2 Hz, 4H), 3.58 (d, <i>J</i> = 7.2 Hz, 2H), 3.71 (d, <i>J</i> = 8.6 Hz, 2H), 3.88 (dq, <i>J</i> = 11.2, 7.6, 5.7 Hz, 1H), 6.17 (d, <i>J</i> = 8.0 Hz, 1H), 7.24–7.37 (m, 2H), 7.56 (s, 1H), 9.02 (s, 1H) ; HRMS (ES<sup>+</sup>) for C<sub>24</sub>H<sub>33</sub>N<sub>6</sub>O<sub>2</sub>F<sub>2</sub> (M+H)<sup>+</sup>: calcd 465.2633; found 475.2621.</div></div><div id="sec4_3_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 5-Morpholino-N4-phenyl-N2-(4-(piperidin-1-ylsulfonyl)phenyl)pyrimidine-2,4-diamine (<b>6</b>)</h5><div class="NLM_p last">Compound synthesized as part of library; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.34 (s, 2H), 1.54 (s, 5H), 2.84 (d, <i>J</i> = 5.1 Hz, 4H), 2.87–2.92 (m, 4H), 3.72–3.87 (m, 4H), 7.13 (t, <i>J</i> = 7.3 Hz, 1H), 7.33–7.45 (m, 2H), 7.51 (d, <i>J</i> = 8.9 Hz, 2H), 7.74 (d, <i>J</i> = 7.6 Hz, 2H), 7.92 (d, <i>J</i> = 8.9 Hz, 2H), 8.07 (s, 1H), 8.43 (s, 1H), 9.62 (s, 1H); HRMS (ES<sup>+</sup>) for C<sub>25</sub>H<sub>31</sub>N<sub>6</sub>O<sub>3</sub>S (M+H)<sup>+</sup>: calcd 495.2178; found 495.2202.</div></div><div id="sec4_3_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 4-((4-(Benzylamino)-5-(methylsulfonyl)pyrimidin-2-yl)amino)phenol (<b>7</b>)</h5><div class="NLM_p last">Compound synthesized as part of library; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) 3.21 (s, 3H), 4.65 (s, 2H), 6.62 (s, 2H), 7.29 (dd, <i>J</i> = 39.6, 6.8 Hz, 7H), 7.72 (s, 1H), 8.25 (s, 1H), 9.12 (s, 1H), 9.59 (s, 1H) ; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S (M+H)<sup>+</sup>: calcd 371.1178; found 371.1188.</div></div><div id="sec4_3_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Ethyl 4-(cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylate</h5><div class="NLM_p last">Cyclopentylamine (2.12 mL, 21.5 mmol) was added dropwise to a stirred solution of ethyl 4-chloro-2-(methylthio)pyrimidine-5-carboxylate (5.00 g, 21.5 mmol) and triethylamine (3.29 mL, 23.64 mmol) in dioxane (45 mL). The resulting cloudy suspension was stirred at 20 °C for 18 h. The reaction mixture was diluted with EtOAc (25 mL), and washed sequentially with water (20 mL) and saturated brine (20 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution 20% EtOAc in isohexane. Pure fractions were evaporated to dryness to afford ethyl 4-(cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylate<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (5.29 g, 87%) as a yellow gum. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.30 (3H, t), 1.51 (2H, ddd), 1.59–1.76 (4H, m), 1.96–2.10 (2H, m), 2.49 (3H, s), 4.28 (2H, q), 4.42 (1H, dd), 8.19 (1H, d), 8.54 (1H, s); MS: (ESI) [M + H]<sup>+</sup> 282.</div></div><div id="sec4_3_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 4-(Cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylic acid</h5><div class="NLM_p last">A solution of ethyl 4-(cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylate (1.4 g, 4.98 mmol) in ethanol (12 mL) was treated with a solution of lithium hydroxide hydrate (0.401 g, 9.56 mmol) in water (4.00 mL). The reaction mixture was heated to 65 °C and stirred for ∼4 h. The reaction was cooled to ambient temperature and stirred overnight. The reaction mixture was evaporated and the residue taken up in water (∼20 mL). The solution was taken to pH1–2 by dropwise addition of 2 M HCl. The resultant precipitate was collected by filtration, the solid was washed with water before drying, in vacuo, at 60 °C for 2 h to afford 4-(cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylic acid<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (1.24 g, 98%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.42–1.54 (2H, m), 1.55–1.74 (4H, m), 1.97–2.08 (2H, m), 2.48 (3H, s), 4.34–4.44 (1H, m), 8.39 (1H, d), 8.50 (1H, s), 13.13 (1H, br s); MS: (ESI) [M + H]<sup>+</sup> 254.</div></div><div id="sec4_3_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 4-(Cyclopentylamino)-<i>N</i>-methyl-2-(methylthio)pyrimidine-5-carboxamide</h5><div class="NLM_p last">A cooled (ice/water bath) suspension of 4-(cyclopentylamino)-2-(methylthio)pyrimidine-5-carboxylic acid (500 mg, 1.97 mmol) in DCM (10 mL) was treated with <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (0.7 mL, 4.02 mmol) and sulfurous dichloride (0.215 mL, 2.96 mmol). The resulting mixture was stirred in the ice bath for 40 min. A further portion of <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (0.7 mL, 4.02 mmol) and methanamine hydrochloride (160 mg, 2.37 mmol) was added and stirring continued for 3 h. Reaction mixture was treated with water (10 mL) and organic layer collected by gravity elution through a PTFE filter cup. Filtrate was evaporated and residue purified by flash silica chromatography, elution gradient 20 to 100% EtOAc in heptane. Product containing fractions were evaporated to dryness and triturated under heptane. The solid was collected by suction filtration and dried to afford 4-(cyclopentylamino)-<i>N</i>-methyl-2-(methylthio)pyrimidine-5-carboxamide (307 mg, 58%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.39–1.50 (2H, m), 1.53–1.73 (4H, m), 1.93–2.04 (2H, m), 2.46 (3H, s), 2.73 (3H, d), 4.28–4.39 (1H, m), 8.42 (1H, s), 8.47 (1H, d), 9.02 (1H, d); MS: (ESI) [M + H]<sup>+</sup> 267.</div></div><div id="sec4_3_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 4-(Cyclopentylamino)-<i>N</i>-methyl-2-(methylsulfinyl)pyrimidine-5-carboxamide</h5><div class="NLM_p last">3-Chlorobenzoperoxoic acid (226 mg, 1.31 mmol) was added to a cooled (ice/water bath) suspension of 4-(cyclopentylamino)-<i>N</i>-methyl-2-(methylthio)pyrimidine-5-carboxamide (291 mg, 1.09 mmol) in DCM (6.0 mL). An immobile suspension resulted that was diluted with more DCM (6.0 mL). The resultant solution was stirred in the ice bath for 3 h. The reaction mixture was treated with sat. aq. bicarb. (10 mL) and stirred for 20 min. The organic layer was separated by elution through a phase separation cartridge and evaporated. The residue was triturated under isohexane and the solid collected by suction filtration and dried to afford 4-(cyclopentylamino)-<i>N</i>-methyl-2-(methylsulfinyl)pyrimidine-5-carboxamide (255 mg, 83%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.42–1.53 (2H, m), 1.54–1.74 (4H, m), 1.95–2.05 (2H, m), 2.77 (3H, d), 2.82 (3H, s), 4.32–4.41 (1H, m), 8.66 (1H, s), 8.70–8.77 (1H, m), 9.14 (1H, d); MS: (ESI) [M-H]<sup>−</sup> 281.</div></div><div id="sec4_3_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 4-(Cyclopentylamino)-2-((4-methoxy-2-methylphenyl)amino)-<i>N</i>-methylpyrimidine-5-carboxamide (<b>8</b>)</h5><div class="NLM_p last">A solution of 4-(cyclopentylamino)-<i>N</i>-methyl-2-(methylsulfinyl)pyrimidine-5-carboxamide (63 mg, 0.22 mmol) and 4 M HCl in 1,4-dioxane (0.061 mL, 0.25 mmol) in 1,4-dioxane (2 mL) was added to 4-methoxy-2-methylaniline (0.037 g, 0.27 mmol) and stirred at 150 °C for 30 min in the microwave. The reaction mixture was partitioned between saturated Na<sub>2</sub>CO<sub>3</sub> (∼4 mL) and DCM (∼5 mL). The organic layer was isolated and concentrated to dryness. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% MeOH in EtOAc. Product containing fractions were evaporated to dryness to afford 4-(cyclopentylamino)-2-(4-methoxy-2-methylphenylamino)-<i>N</i>-methylpyrimidine-5-carboxamide (46.1 mgs, 57%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.29–1.43 (2H, m), 1.45–1.58 (2H, m), 1.58–1.69 (2H, m), 1.82–1.93 (2H, m), 2.17 (3H, s), 2.69 (3H, d), 3.72 (3H, s), 4.08–4.20 (1H, m), 6.71 (1H, dd), 6.77 (1H, d), 7.31 (1H, d), 8.10 (1H, d), 8.32 (1H, s), 8.42 (1H, s), 8.92 (1H, d); HRMS (ES<sup>+</sup>) for C<sub>19</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> (M+H)<sup>+</sup>: calcd 356.2087; found 356.2099.</div></div><div id="sec4_3_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 2-(Cyclopentyloxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (<b>9</b>)</h5><div class="NLM_p last">2-Chloro-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (150 mg, 0.48 mmol) was added to a mixture of sodium hydroxide (30 mg) in cyclopentanol (8 mL) and the resulted mixture stirred at 80 °C for 2 h. The reaction mixture was cooled to rt, diluted with water and EtOAc. The aqueous layer was extracted with EtOAc and the combined organic layers was washed by brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% MeOH in DCM. Pure fractions were evaporated to dryness to afford 2-(cyclopentyloxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine, which was further purified by recrystallization from methanol to afford 2-(cyclopentyloxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (27 mg, 16%) as a white solid; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.5–2.2 (m, 8H), 2.95 (br s, 3H), 7.50 (q, 1H), 7.75 (br s, 1H), 8.0 (s, 1H); HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub>F<sub>2</sub> (M+H)<sup>+</sup>: calcd 360.1636; found 360.1648.</div></div><div id="sec4_3_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 2-(Cyclobutylmethoxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (<b>10</b>)</h5><div class="NLM_p last">2-Chloro-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (245 mg, 0.792 mmol) was added to a mixture of sodium hydroxide (30 mg) in cyclobutyl methyl alcohol (3 mL) and the resulted mixture stirred at 80 °C for 4 h. The reaction mixture was cooled to rt, diluted with water and EtOAc. The aqueous layer was extracted with EtOAc and the combined organic layers was washed by brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% MeOH in DCM. Pure fractions were evaporated to dryness to afford 2-(cyclobutylmethoxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine, which was further purified by recrystallization from DCM/hexane to afford 2-(cyclobutylmethoxy)-9-(2,6-difluorobenzyl)-<i>N</i>-methyl-9H-purin-6-amine (36 mg, 18%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.7–1.81 (m, 2H), 1.82–1.93 (m, 2H), 2.03 (dtd, <i>J</i> = 14.4, 7.1, 6.4, 4.4 Hz, 2H), 2.61–2.7 (m, 1H), 3.33 (s, 3H), 4.17 (d, <i>J</i> = 7.0 Hz, 2H), 5.34 (s, 2H), 7.04–7.16 (m, 2H), 7.45 (ddd, <i>J</i> = 15.0, 8.4, 6.6 Hz, 1H), 7.66 (s, 1H), 7.94 (s, 1H); HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>OF<sub>2</sub> (M+H)<sup>+</sup>: calcd 360.1636; found 2360.1648.</div></div><div id="sec4_3_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> <i>tert</i>-Butyldimethyl((4-nitro-2-(trifluoromethyl)benzyl)oxy)silane</h5><div class="NLM_p last"><i>tert</i>-Butyldimethylsilyl chloride (0.886 g, 5.88 mmol) was added to a mixture of imidazole (0.616 g, 9.04 mmol) and (4-nitro-2-(trifluoromethyl)phenyl)methanol<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (1.00 g, 4.52 mmol) in DCM (10 mL) at 0 °C under nitrogen. The resulting mixture was stirred at rt for 16 h. The reaction mixture was diluted with DCM (50 mL) and washed sequentially with water and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 20% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford <i>tert</i>-butyldimethyl((4-nitro-2-(trifluoromethyl)benzyl)oxy)silane (1.40 g, 92%) as a yellow oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.58 (dd, <i>J</i> = 8.7, 2.4 Hz, 1H), 8.41 (d, <i>J</i> = 2.4 Hz, 1H), 8.04 (d, <i>J</i> = 8.6 Hz, 1H), 4.96 (t, <i>J</i> = 1.2 Hz, 2H), 0.93 (s, 9H), 0.13 (s, 6H).</div></div><div id="sec4_3_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-3-(trifluoromethyl)aniline</h5><div class="NLM_p last">Iron (9.99 g, 178.89 mmol) was added to a mixture of ammonium chloride (4.78 g, 89.5 mmol) and <i>tert</i>-butyldimethyl((4-nitro-2-(trifluoromethyl)benzyl)oxy)silane (6.00 g, 17.9 mmol) in ethanol (60 mL) and water (20 mL) at rt under nitrogen. The resulting mixture was stirred at 80 °C for 2 h. The reaction mixture was cooled to rt and filtered through Celite. The solvent was removed under reduced pressure and the residue was redissolved in EtOAc (100 mL) and washed sequentially with water and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford 4-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)-3-(trifluoromethyl)aniline (5.20 g, 95%) as yellow oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.28 (d, <i>J</i> = 8.3 Hz, 1H), 6.87 (d, <i>J</i> = 2.4 Hz, 1H), 6.78 (dd, <i>J</i> = 8.3, 2.4 Hz, 1H), 5.49 (s, 2H), 4.67–4.59 (m, 2H), 0.88 (s, 9H), 0.05 (s, 6H); MS: (ESI) [M + H]<sup>+</sup> 306.</div></div><div id="sec4_3_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Phenyl (4-((6-(cyclopropanecarboxamido)pyrimidin-4-yl)oxy)phenyl)carbamate</h5><div class="NLM_p last">Phenyl chloroformate (16.5 g, 105 mmol) was added to a solution of <i>N</i>-[6-(4-aminophenoxy)pyrimidin-4-yl]cyclopropanecarboxamide<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (26.0 g, 96.2 mmol) in tetrahydrofuran (260 mL). The resulting solution was stirred at 60 °C for 1 h. The solids were collected by filtration, and the solid was dried in a vacuum oven to afford phenyl <i>N</i>-[4-[(6-cyclopropaneamidopyrimidin-4-yl)oxy]phenyl]carbamate (24.0 g, 58%) as a yellow solid. <sup>1</sup>H (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.22 (s, 1H), 10.35 (s, 1H), 8.51 (s, 1H), 7.57 (d, 2H), 7.48–7.41 (m, 3H), 7.29–7.23 (m, 5H), 1.78–1.73 (m, 1H), 0.83 (d, 4H); MS: (ESI) [M + H]<sup>+</sup> 391.</div></div><div id="sec4_3_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> <i>N</i>-(6-(4-(3-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-3-(trifluoromethyl)phenyl)ureido)phenoxy)-pyrimidin-4-yl)cyclopropanecarboxamide</h5><div class="NLM_p last">4-[(<i>tert</i>-Butyldimethylsilyl)oxy]methyl-3-(trifluoromethyl)aniline (37.9 g, 124 mmol) and <i><i>N</i></i>,<i>N</i>-diisopropylethylamine (12.0 g, 92.9 mmol) were added to a solution of phenyl-[4-[(6-cyclopropane-amidopyrimidin-4-yl)oxy]phenyl]carbamate (24.0 g, 61.5 mmol) in dimethyl sulfoxide (360 mL). The resulting solution was stirred at 85 °C for 3 h. The reaction was cooled to rt and quenched by the addition of water. The resulting solution was extracted with EtOAc and the organic layers combined and concentrated in vacuo to afford the crude product. The crude product was purified by flash silica chromatography, elution gradient 5% MeOH in DCM. Fractions containing the desired compound were evaporated to dryness to afford of <i>N</i>-[6-(4-[[(4-[[(<i>tert</i>-butyldimethylsilyl)oxy]methyl]-3-(trifluoromethyl)phenyl)carbamoyl]-amino]phenoxy)-pyrimidin-4-yl]cyclopropanecarboxamide (15.0 g, 36%) as a light yellow solid. NMR not obtained for intermediate, confirmed by MS: (ESI) [M + H]<sup>+</sup> 602.</div></div><div id="sec4_3_1_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> <i>N</i>-(6-(4-(3-(4-(Hydroxymethyl)-3-(trifluoromethyl)phenyl)ureido)phenoxy)pyrimidin-4-yl)cyclopropanecarboxamide</h5><div class="NLM_p last"><i>Tetra</i>-<i>n</i>-Butylammonium fluoride (13.4 g, 51.3 mmol) was added to a solution of <i>N</i>-[6-(4-[[(4-[[(<i>tert</i>-butyldimethylsilyl)oxy]methyl]-3-(trifluoromethyl)phenyl)-carbamoyl]amino]phenoxy)pyrimidin-4-yl]cyclopropanecarboxamide (15.0 g, 24.9 mmol) in tetrahydrofuran (120 mL). The resulting solution was stirred at rt for 3 h. The resulting mixture was concentrated in vacuo to afford the crude product. The crude product was purified by flash silica chromatography, elution gradient 2% MeOH in EtOAc. Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-[6-[4-([[4-(hydroxymethyl)-3-(trifluoromethyl)phenyl]-carbamoyl]amino)phenoxy]pyrimidin-4-yl]cyclopropanecarboxamide (12.0 g, 89%) as a light yellow solid. NMR not obtained for intermediate, confirmed by MS: (ESI) [M + H]<sup>+</sup> 488.</div></div><div id="sec4_3_1_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> <i>N</i>-(6-(4-(3-(4-Formyl-3-(trifluoromethyl)phenyl)ureido)phenoxy)pyrimidin-4-yl)cyclopropanecarboxamide</h5><div class="NLM_p last">Pyridinium chlorochromate (10.6 g, 49.2 mmol) and sodium acetate (5.00 g, 61.0 mmol) were added to a solution of <i>N</i>-[6-[4-([[4-(hydroxymethyl)-3-(trifluoromethyl)phenyl]carbamoyl]amino) phenoxy]pyrimidin-4-yl]cyclopropane-carboxamide (12.0 g, 24.6 mmol) in DCM (200 mL). The resulting solution was stirred at rt for 5 h. The resulting mixture was concentrated in vacuo to afford the crude product. The crude product was purified by flash silica chromatography, elution gradient 20% EtOAc in petroleum ether. Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-[6-[4-([[4-formyl-3-(trifluoromethyl)phenyl]-carbamoyl]amino)phenoxy]pyrimidin-4-yl]cyclopropanecarboxamide (8.90 g, 67%) as a light yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 0.84 (br s, 4H), 1.97 (m, 1H), 7.10 (d, 2H), 7.45–7.60 (m, 3H), 7.91 (d, 1H), 8.03 (d, 1H), 8.18 (s, 1H), 8.57 (s, 1H), 9.08 (s, 1H), 9.61 (s, 1H), 10.12 (s, 1H); 11.28 (s, 1H); MS: (ESI) [M + H]<sup>+</sup> 486.</div></div><div id="sec4_3_1_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> <i>N</i>-(6-(4-(3-(4-((5,6-Dihydroimidazo[1,2-<i>a</i>]pyrazin-7(8H)-yl)methyl)-3-(trifluoromethyl)phenyl)ureido)phenoxy)pyrimidin-4-yl)cyclopropanecarboxamide (<b>11</b>)</h5><div class="NLM_p last">A mixture of 1-(4-(6-(cyclopropanecarboxamido)pyrimidin-4-yloxy)phenyl)-3-(4-formyl-3-(trifluoro-methyl)phenyl)urea (48.5 mg, 0.10 mmol), 5,6,7,8-tetrahydroimidazo[1,2-<i>a</i>]pyrazine (18 mg, 0.15 mmol), trifluoroacetic acid (5 drops, cat.) in DCM (2 mL) was stirred at rt for 16 h. Sodium triacetoxyborohydride (64 mg, 0.30 mmol) was then added and the mixture stirred at rt for an additional 5 h. The crude product was purified by preparative HPLC using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-(6-(4-(3-(4-((5,6-dihydroimidazo[1,2-<i>a</i>]pyrazin-7(8H)-yl)methyl)-3-(trifluoromethyl)-phenyl)ureido)phenoxy)pyrimidin-4-yl)cyclopropanecarboxamide (23 mg). Compound obtained as part of library work; no NMR recorded and confirmed by MS: (ESI) [M + H]<sup>+</sup> 593. Purity >95% was confirmed by HPLC.</div></div><div id="sec4_3_1_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2-[1-([[(tert-Butoxy)carbonyl]amino]methyl)-2-azabicyclo[2.1.1]hexan-2-yl]-4-(trifluoromethyl)-1,3-thiazole-5-carboxylic acid</h5><div class="NLM_p last">A solution of ethyl 2-bromo-4-(trifluoromethyl)-1,3-thiazole-5-carboxylate (400 mg, 1.32 mmol), <i>tert</i>-butyl <i>N</i>-[2-azabicyclo[2.1.1]hexan-1-ylmethyl]carbamate (307 mg, 1.45 mmol), potassium carbonate (545 mg, 3.91 mmol) in acetonitrile (20 mL) was stirred for 2 h at 80 °C in inert atmosphere. The reaction mixture was cooled to 20 °C, and the precipitate was removed by filtration and the crude product was purified by prep-HPLC to obtain 2-[1-([[(tert-butoxy)carbonyl]amino]methyl)-2-azabicyclo[2.1.1]hexan-2-yl]-4-(trifluoromethyl)-1,3-thiazole-5-carboxylic acid (0.3 g, 98%) as a yellow solid. MS: [M + H]<sup>+</sup> = 408.</div></div><div id="sec4_3_1_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> <i>tert</i>-Butyl <i>N</i>-([2-[5-(methylcarbamoyl)-4-(trifluoromethyl)-1,3-thiazol-2-yl]-2-azabicyclo-[2.1.1]hexan-1-yl]methyl)carbamate</h5><div class="NLM_p last">A solution of 2-[1-([[(tert-butoxy)carbonyl]amino]methyl)-2-azabicyclo[2.1.1]hexan-2-yl]-4-(trifluoromethyl)-1,3-thiazole-5-carboxylic acid (250 mg, 0.61 mmol), methanamine hydrochloride (50 mg, 0.74 mmol), sodium bicarbonate (155 mg, 1.85 mmol), and HATU (280 mg, 0.74 mmol) in DMF (20 mL) was stirred for 2 h at 25 °C. The reaction mixture was diluted with ethyl acetetate, aq. sodium bicarbonate (20 mL), and extracted with ether (3 × 20 mL). The combined organics were dried over anhydrous sodium sulfate, concentrated, and purified by prep-HPLC to obtain <i>tert</i>-butyl <i>N</i>-([2-[5-(methylcarbamoyl)-4-(trifluoromethyl)-1,3-thiazol-2-yl]-2-azabicyclo[2.1.1]hexan-1-yl]methyl)carbamate (0.23 g, 100%) as a yellow solid. MS: (ESI) [M + H]<sup>+</sup> 421.</div></div><div id="sec4_3_1_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 2-[1-(Aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl]-<i>N</i>-methyl-4-(trifluoromethyl)-1,3-thiazole-5-carboxamide</h5><div class="NLM_p last">A solution of <i>tert</i>-butyl <i>N</i>-([2-[5-(methylcarbamoyl)-4-(trifluoromethyl)-1,3-thiazol-2-yl]-2-azabicyclo[2.1.1]hexan-1-yl]methyl)carbamate (220 mg, 0.52 mmol) and hydrogen chloride (4 M in 1,4-dioxane, 10 mL) in 1,4-dioxane (10 mL) was stirred for 1 h at rt, and the concentrated to obtain 2-[1-(aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl]-<i>N</i>-methyl-4-(trifluoromethyl)-1,3-thiazole-5-carboxamide as a white solid (0.2 g, crude). MS: (ESI) [M + H]<sup>+</sup> 321.</div></div><div id="sec4_3_1_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 2-[1-[([5-Chloro-1H-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]formamido)methyl]-2-azabicyclo[2.1.1]hexan-2-yl]-<i>N</i>-methyl-4-(trifluoromethyl)-1,3-thiazole-5-carboxamide (<b>12</b>)</h5><div class="NLM_p last">A solution of 2-[1-(aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl]-<i>N</i>-methyl-4-(trifluoromethyl)-1,3-thiazole-5-carboxamide (200 mg, 0.62 mmol), 5-chloro-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxylic acid (140 mg, 0.71 mmol), sodium bicarbonate (160 mg, 1.90 mmol), and HATU (280 mg, 0.74 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (10 mL) was stirred for 2 h at rt. The reaction mixture was extracted with ethyl acetate, and the combined organic layers were washed with brine, dried, and concentrated under vacuum. The precipitate was collected by filtration, and the crude product was purified by prep-HPLC to obtain 2-[1-[([5-chloro-1H-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]formamido)methyl]-2-azabicyclo[2.1.1]hexan-2-yl]-<i>N</i>-methyl-4-(trifluoromethyl)-1,3-thiazole-5-carboxamide (15.1 mg, 24%) as a white solid. <sup>1</sup>H NMR (400 MHz, MeOD) δ 1.74 (dd, 2H, <i>J</i> = 5.0, 2.0 Hz), 2.16 (m, 2H), 2.88 (s, 3H), 2.98 (t, 1H, <i>J</i> = 3.2 Hz), 3.48 (s, 2H), 4.25 (s, 2H), 7.90 (s, 1H), 8.27 (d, 1H, <i>J</i> = 2.4 Hz), 8.38 (d, 1H, <i>J</i> = 2.4 Hz). HRMS (ES<sup>+</sup>) for C<sub>20</sub>H<sub>18</sub>N<sub>6</sub>O<sub>2</sub>F<sub>3</sub>SCl (M+H)<sup>+</sup>: calcd 499.0931; found 499.0939.</div></div><div id="sec4_3_1_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> <i>tert</i>-Butyl ((<i>R</i>)-1-(((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)amino)-1-oxo-4-phenylbutan-2-yl)carbamate</h5><div class="NLM_p last">To a solution of (<i>R</i>)-3-amino-<i>N</i>-methyl-3-(4-(trifluoromethyl)phenyl)propanamide (226 mg, 0.81 mmol) in DMF (20 mL) was added HOBT (121 mg, 0.89 mmol), EDCI (171 mg, 0.89), and DIEA (314 mg, 2.43 mmol) at 0 °C. The reaction mixture was stirred for 30 min, and then (<i>R</i>)-2-((tert-butoxycarbonyl)amino)-4-phenylbutanoic acid (200 mg, 0.81 mmol) was added and the mixture was stirred at rt for 3 h. Water (10 mL) was added to the reaction mixture, and the resulting precipitate was collection by filtration and dried to obtain <i>tert</i>-butyl ((<i>R</i>)-1-(((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)amino)-1-oxo-4-phenylbutan-2-yl)carbamate (247 mg, 60%) as a white solid, which was used directly. MS: [M + H]<sup>+</sup> = 508.</div></div><div id="sec4_3_1_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (<i>R</i>)-2-Amino-<i>N</i>-((<i>R</i>)-3-(methylamino)-3-oxo-1-(4- (trifluoromethyl)phenyl)propyl)-4-phenylbutanamide</h5><div class="NLM_p last">A solution of <i>tert</i>-butyl ((<i>R</i>)-1-(((<i>R</i>)-3-(methylamino)-3-oxo-1-(4- (trifluoromethyl)phenyl)propyl)amino)-1-oxo-4-phenylbutan-2-yl)carbamate (247 mg, 0.49 mmol) in HCl/1,4-dioxane solution (4M, 8 mL) was stirred at rt for 1 h. The reaction mixture was concentrated to obtain (<i>R</i>)-2-amino-<i>N</i>-((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)-4-phenylbutanamide (199 mg, 100%) as a white solid, which was used directly. MS: [M + H]<sup>+</sup> = 408.</div></div><div id="sec4_3_1_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 3-Methyl-5-(4-methyl-1,2,3-thiadiazol-5-yl)-<i>N</i>-((<i>R</i>)-1-(((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)amino)-1-oxo-4-phenylbutan-2-yl)isoxazole-4-carboxamide (<b>13</b>)</h5><div class="NLM_p last">A solution of 3-methyl-5-(4-methyl-1,2,3-thiadiazol-5-yl)isoxazole-4-carboxylic acid (33 mg, 0.15 mmol) in SOCl<sub>2</sub> (2 mL) was stirred at 80 °C for 2 h, concentrated, and then diluted with DCM (2 mL). The resultant solution was added to a solution of (<i>R</i>)-2-amino-<i>N</i>-((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)-4-phenylbutanamide (60 mg, 0.15 mml) and triethylamine (31 mg, 0.3 mmol) in DCM (6 mL). The reaction mixture was stirred for 2 h at rt, concentrated, and the residue was purified by column chromatography and then prep-HPLC to obtain 3-methyl-5-(4-methyl-1,2,3-thiadiazol-5-yl)-<i>N</i>-((<i>R</i>)-1-(((<i>R</i>)-3-(methylamino)-3-oxo-1-(4-(trifluoromethyl)phenyl)propyl)amino)-1-oxo-4-phenylbutan-2-yl)isoxazole-4-carboxamide (6 mg, 6%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-d6) δ 8.88 (m, 2H), 8.12–7.43 (m, 5H), 7.40–7.02 (m, 3H), 5.28 (s, 1H), 4.47 (s, 1H), 3.20–2.81 (m, 5H), 2.65–2.55 (m, 3H), 2.47–2.28 (m, 3H), 1.99–1.89 (m, 2H), 1.57 (m, 2H). HRMS (ES<sup>+</sup>) for C<sub>29</sub>H<sub>29</sub>N<sub>6</sub>O<sub>4</sub>F<sub>3</sub>S (M+H)<sup>+</sup>: calcd 615.2001; found 615.2004.</div></div><div id="sec4_3_1_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 5-(4-Bromophenyl)-1,3,4-oxadiazol-2(3H)-one</h5><div class="NLM_p last">Carbonyldiimidazole (11.3 g, 69.8 mmol) was added portionwise to a mixture of 4-bromobenzohydrazide (15.0 g, 69.8 mmol) in DCM (100 mL). The resulting solution was stirred at rt for 16 h. The solution was poured into water to form a white precipitate. The solid was collected by filtration and dried to afford 5-(4-bromophenyl)-1,3,4-oxadiazol-2(3H)-one (16.0 g, 95%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.69–7.80 (4H, m), 12.56 (1H, s); MS: (ESI) [M + H]<sup>+</sup> 241.</div></div><div id="sec4_3_1_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 5-(4-Bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">(Benzotriazol-1-yloxy)tris(dimethylamino)phosphonium hexafluorophosphate) (5.50 g, 12.5 mmol) was added to a solution of 5-(4-bromophenyl)-1,3,4-oxadiazol-2(3H)-one (2.00 g, 8.30 mmol), imidazo[1,2-<i>a</i>]pyridin-6-ylmethanamine hydrochloride (1.52 g, 8.30 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (4.35 mL, 24.9 mmol) in DCM (30 mL) under nitrogen. The resulting mixture was stirred at rt for 6 h. The reaction mixture was diluted with EtOAc (100 mL) and washed with brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0 to 10% MeOH in DCM. Pure fractions were evaporated to dryness to afford 5-(4-bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-1,3,4-oxadiazol-2-amine (1.16 g, 38%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 4.45 (2H, d), 7.26 (1H, d), 7.56 (2H, d), 7.74 (4H, s), 7.95 (1H, s), 8.45 (1H, t), 8.56 (1H, s); MS: (ESI) [M + H]<sup>+</sup> 370.</div></div><div id="sec4_3_1_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 5-(4-Bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">Sodium hydride (60% dispersion in mineral oil, 0.681 g, 28.4 mmol) was added to 5-(4-bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-1,3,4-oxadiazol-2-amine (3.50 g, 9.45 mmol) in tetrahydrofuran (50 mL) under nitrogen. The resulting mixture was stirred at rt for 1 h. Iodomethane (2.68 g, 18.9 mmol) was then added, and the resulting mixture was stirred at rt for 1 h. The reaction mixture was diluted with EtOAc (150 mL) and washed with brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by flash C18-flash chromatography, elution gradient 20 to 100% water in MeOH. Pure fractions were evaporated to dryness to afford 5-(4-bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (3.30 g, 91%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.11 (3H, s), 4.68 (2H, s), 7.24 (1H, d), 7.57 (2H, d), 7.71–7.79 (2H, m), 7.80–7.89 (2H, m), 7.95 (1H, s), 8.62 (1H, s); MS: (ESI) [M + H]<sup>+</sup> 384.</div></div><div id="sec4_3_1_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 5-(2′-Chloro-[1,1′-biphenyl]-4-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>14</b>)</h5><div class="NLM_p last">XPhos Pd G2 (82 mg, 0.10 mmol) was added to a mixture of 5-(4-bromophenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (400 mg, 1.04 mmol), (2-chlorophenyl)boronic acid (244 mg, 1.56 mmol), and potassium carbonate (432 mg, 3.12 mmol) in 1,4-dioxane (4 mL) and water (1 mL) under nitrogen. The resulting mixture was stirred at 80 °C for 2 h. The reaction mixture was cooled to rt, diluted with EtOAc (50 mL), and washed with brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude product. The crude product was purified by preparative HPLC (Puriflash C18, 30 × 150 mm<sup>2</sup> 5 μm), using decreasingly polar mixtures of water (containing 0.1% formic acid) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford 5-(2′-chloro-[1,1′-biphenyl]-4-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (0.145 g, 34%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) 3.13 (3H, s), 4.70 (2H, s), 7.26 (1H, dd), 7.40–7.52 (3H, m), 7.55–7.66 (5H, m), 7.94–8.04 (3H, m), 8.64 (1H, dd); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 35.84, 51.52, 113.89, 117.56, 121.20, 123.95, 125.61, 125.63, 126.10, 128.14, 130.15, 130.45, 130.60, 131.67, 131.88, 133.88, 139.31, 141.13, 144.50, 158.50, 164.72; HRMS (ES<sup>+</sup>) for C<sub>23</sub>H<sub>19</sub>N<sub>5</sub>OCl (M+H)<sup>+</sup>: calcd 416.1278; found 416.1288.</div></div><div id="sec4_3_1_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> <i>N</i>-Methyl-2-(1-((5-methyl-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)-4-(trifluoromethyl)thiazole-5-carboxamide (<b>15</b>)</h5><div class="NLM_p last">2-(1-(Aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl)-<i>N</i>-methyl-4-(trifluoromethyl)thiazole-5-carboxamide (80 mg, 0.25 mmol) was added to 5-methyl-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxylic acid (52.8 mg, 0.30 mmol), 2-(3H-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-yl)-1,1,3,3-tetramethylisouronium hexafluorophosphate(V) (142 mg, 0.37 mmol), and sodium hydrogen carbonate (62.9 mg, 0.75 mmol) in DMF (4 mL). The resulting mixture was stirred at 25 °C for 16 h. The crude product was purified by preparative HPLC (Column: Sunfire Prep C18 OBD Column, 10 μm, 19 × 250 mm<sup>2</sup>; Mobile Phase A: Water (0.1%FA), Mobile Phase B: ACN; Flow rate: 25 mL/min; Gradient: 31% B to 52% B in 7 min; 254/220 nm; Rt: 6.48 min). Fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-methyl-2-(1-((5-methyl-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)-4-(trifluoromethyl)thiazole-5-carboxamide (37.2 mg, 31%) as a white solid. <sup>1</sup>H NMR (DMSO-d6, 400 MHz) δ 1.59 (2H, d), 2.03 (2H, s), 2.38 (3H, s), 2.72 (3H, d), 2.87 (1H, s), 3.45 (2H, s), 4.16 (2H, d), 7.92–8.03 (2H, m), 8.10–8.17 (2H, m), 8.36 (1H, d), 12.03 (1H, s). HRMS for C<sub>21</sub>H<sub>21</sub>N<sub>6</sub>O<sub>2</sub>F<sub>3</sub>S (M+H)<sup>+</sup>: calcd 479.1477; found 479.1474.</div></div><div id="sec4_3_1_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> <i>N</i>-Methyl-4-(trifluoromethyl)-2-(1-((5-(trifluoromethyl)-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)thiazole-5-carboxamide (<b>16</b>)</h5><div class="NLM_p last">2-(1-(Aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl)-<i>N</i>-methyl-4-(trifluoromethyl)thiazole-5-carboxamide (80 mg, 0.25 mmol) was added to 5-(trifluoromethyl)-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxylic acid (69.0 mg, 0.30 mmol), HATU (142 mg, 0.37 mmol), and sodium bicarbonate (62.9 mg, 0.75 mmol) in DMF (4 mL). The resulting mixture was stirred at 25 °C for 16 h. Reaction worked. The crude product was purified by preparative HPLC (Column: XBridge Prep OBD C18 Column 30 × 150 mm<sup>2</sup> 5 μm; Mobile Phase A: Water (10 mmol/L NH<sub>4</sub>HCO<sub>3</sub> + 0.1% NH<sub>3</sub>·H<sub>2</sub>O), Mobile Phase B: ACN; Flow rate: 60 mL/min; Gradient: 30% B to 57% B in 8 min; 254/220 nm; Rt: 7.63 min). The fractions containing the desired compound were evaporated to dryness to afford <i>N</i>-methyl-4-(trifluoromethyl)-2-(1-((5-(trifluoromethyl)-1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)thiazole-5-carboxamide (50.0 mg, 38%) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d6</i>, 400 MHz) δ 1.59 (2H, d), 2.04 (2H, s), 2.72 (3H, d), 2.86 (1H, s), 3.46 (2H, s), 4.19 (2H, d), 8.24–8.39 (3H, m), 8.63–8.73 (2H, m), 12.70 (1H, s). HRMS for C<sub>21</sub>H<sub>18</sub>N<sub>6</sub>O<sub>2</sub>F<sub>6</sub>S (M+H)<sup>+</sup>: calcd 533.1194; found 533.1212.</div></div><div id="sec4_3_1_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 2-(1-((1H-Pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)-<i>N</i>-methyl-4-(trifluoromethyl)thiazole-5-carboxamide (<b>17</b>)</h5><div class="NLM_p last">2-(1-(Aminomethyl)-2-azabicyclo[2.1.1]hexan-2-yl)-<i>N</i>-methyl-4-(trifluoromethyl)thiazole-5-carboxamide (80 mg, 0.25 mmol) was added to 2-(3H-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-yl)-1,1,3,3-tetramethylisouronium hexafluorophosphate(V) (142 mg, 0.37 mmol), 2-(3H-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-yl)-1,1,3,3-tetramethylisouronium hexafluorophosphate(V) (142 mg, 0.37 mmol), and sodium hydrogen carbonate (62.9 mg, 0.75 mmol) in DMF (4 mL). The resulting mixture was stirred at 25 °C for 16 h. The crude product was purified by preparative HPLC (Column: Xselect CSH OBD Column 30 × 150 mm<sup>2</sup> 5 μm; Mobile Phase A: Water (0.05% TFA), Mobile Phase B: ACN; Flow rate: 60 mL/min; Gradient: 18% B to 42% B in 7 min; 254/220 nm; Rt: 5.97 min). Fractions containing the desired compound were evaporated to dryness to afford 2-(1-((1H-pyrrolo[2,3-<i>b</i>]pyridine-3-carboxamido)methyl)-2-azabicyclo[2.1.1]hexan-2-yl)-<i>N</i>-methyl-4-(trifluoromethyl)thiazole-5-carboxamide (33.5 mg, 29%) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d6</i>, 400 MHz) δ 1.59 (2H, m), 2.04 (2H, s), 2.72 (3H, d), 2.86 (1H, s), 3.45 (2H, s), 4.16 (2H, d), 7.17 (1H, m), 8.00–8.09 (2H, m), 8.29 (1H, m), 8.38 (2H, dd), 12.17 (1H, s). HRMS for C<sub>20</sub>H<sub>19</sub>N<sub>6</sub>O<sub>2</sub>F<sub>3</sub>S (M+H)<sup>+</sup>: calcd 465.1321; found 465.1302.</div></div><div id="sec4_3_1_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 3,4-Dihydro-2H-1-benzopyran-6-ylmethanol</h5><div class="NLM_p last">To a solution of 3,4-dihydro-2H-1-benzopyran-6-carbaldehyde (324 mg, 2.00 mmol, 1.0 equiv) in methanol (15 mL) was added NaBH<sub>4</sub> (152 mg, 4.02 mmol, 2.0 equiv). The resulting solution was stirred for 1 h at 0 °C. The reaction was then quenched by the addition of NH<sub>4</sub>Cl (aq.), and the resulting solution was extracted with DCM. The combined organic layers were washed with brine, dried over anhydrous sodium sulfate, and concentrated under vacuum to obtain 3,4-dihydro-2H-1-benzopyran-6-ylmethanol (310 mg, 95%) as a colorless oil. Taken through to the next step without further purification.</div></div><div id="sec4_3_1_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 6-(Chloromethyl)-3,4-dihydro-2H-1-benzopyran</h5><div class="NLM_p last">To a solution of 3,4-dihydro-2H-1-benzopyran-6-ylmethanol (164 mg, 1.00 mmol, 1.00 equiv) in ether (15 mL) was bubbled HCl (gas). The resulting solution was stirred overnight at rt and concentrated under vacuum to obtain 6-(chloromethyl)-3,4-dihydro-2H-1-benzopyran (210 mg, crude) as an orange oil. Taken through to the next step without further purification.</div></div><div id="sec4_3_1_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 5-[4-(2-Chlorophenyl)phenyl]-<i>N</i>-(3,4-dihydro-2H-1-benzopyran-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>18</b>)</h5><div class="NLM_p last">To a solution of NaH (60%) (166 mg, 6.92 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (15 mL) was added 5-[4-(2-chlorophenyl)phenyl]-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (150 mg, 0.52 mmol) followed by 6-(chloromethyl)-3,4-dihydro-2H-1-benzopyran (191 mg, 1.05 mmol). The resulting solution was stirred overnight at rt. The reaction mixture was diluted with DCM, and the organic layer was washed with brine, dried over anhydrous sodium sulfate, and concentrated under vacuum and purified by prep-HPLC to obtain 5-[4-(2-chlorophenyl)phenyl]-<i>N</i>-(3,4-dihydro-2H-1-benzopyran-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (55 mg, 24%) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.03–7.96 (2H, m), 7.63–7.56 (2H, m), 7.53 (1H, dd, <i>J</i> = 6.2, 1.8 Hz), 7.45–7.34 (3H, m), 7.11–7.05 (2H, m), 6.77–6.70 (1H, m), 4.60 (2H, s), 4.19–4.12 (2H, m), 3.09 (3H, s), 2.78 (2H, t, <i>J</i> = 6.4 Hz), 2.08–1.93 (2 H, m); HRMS for C<sub>25</sub>H<sub>23</sub>N<sub>3</sub>O<sub>2</sub>Cl (M+H)<sup>+</sup>: calcd 432.1479; found 432.1497.</div></div><div id="sec4_3_1_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 5-(6-(2-Chlorophenyl)pyridin-3-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">Into a 100 mL round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed 6-(2-chlorophenyl)nicotinic acid (500 mg, 2 mmol), <i>N</i>-methylhydrazinecarbothioamide (224 mg, 2 mmol), and EDCI (1152 mg, 6 mmol) in DMF (10 mL). The solution was stirred at 25 °C for 16 h, and the resulting solution was purified by column chromatography to obtain 5-(6-(2-chlorophenyl)pyridin-3-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (68 mg, 11%) as a white solid. MS: [M + H]<sup>+</sup> = 287.</div></div><div id="sec4_3_1_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 5-(6-(2-Chlorophenyl)pyridin-3-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>19</b>)</h5><div class="NLM_p last">To a solution of 5-(6-(2-chlorophenyl)pyridin-3-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (60 mg, 0.25 mmol) in DMF (3 mL) was added NaH (51 mg, 1.28 mmol) at 0 °C. The resulting solution was stirred for 30 min at rt, and then a solution of 6-(chloromethyl)imidazo[1,2-<i>a</i>]pyridine hydrochloride (78 mg, 0.38 mmol) in DMF (1 mL) was added dropwise. The reaction mixture was stirred for 2 h at rt and then purified by prep-HPLC to obtain 5-(6-(2-chlorophenyl)pyridin-3-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (32 mg, 42%) as white solid. <sup>1</sup>H NMR (400 MHz, DMSO-d6) δ 3.14 (3H, s), 4.72 (2 H, s), 7.28 (1H, dd, <i>J</i> = 9.2, 1.6 Hz), 7.46–7.56 (2H, m), 7.55–7.70 (4 H, m), 7.87 (1H, d, <i>J</i> = 8.4 Hz), 7.97 (1H, s), 8.38 (1H, dd, <i>J</i> = 8.2, 2.2 Hz), 8.65 (1H, s), 9.18–9.24 (1H, m). HRMS for C<sub>22</sub>H<sub>17</sub>N<sub>6</sub>OCl (M+H)<sup>+</sup>: calcd 417.1231; found 417.1231.</div></div><div id="sec4_3_1_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 5-(4-(Cyclopentyloxy)phenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">A solution of 4-(cyclopentyloxy)benzoic acid (206 mg, 1 mmol), <i>N</i>-methylhydrazinecarbothioamide (105 mg, 1 mmol), and EDCI (230 mg, 1.2 mmol) in DMF (3 mL) was stirred for 12 h at rt. The resulting solution was filtered, and the residue was purified by column chromatography to obtain 5-(4-(cyclopentyloxy)phenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (30 mg, 14%) as a light yellow solid. MS: [M + H]<sup>+</sup> = 260.</div></div><div id="sec4_3_1_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 5-(4-(Cyclopentyloxy)phenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>20</b>)</h5><div class="NLM_p last">Under nitrogen atmosphere, a solution of 5-(4-(cyclopentyloxy)phenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (26 mg, 0. One mmol), NaH (20 mg, 0.5 mmol) in DMF (2 mL) was stirred for 30 min at 25 °C. Then 6-(chloromethyl)imidazo[1,2-<i>a</i>]pyridine (34 mg, 0.2 mmol) in DMF (1 mL) was added, and the resulting solution was stirred for 1 h at 25 °C. The reaction mixture was quenched with water, filtered, and the residue was purified by prep-HPLC to obtain 5-(4-(cyclopentyloxy)phenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (22 mg, 55%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.50–1.75 (6H, m), 1.88–1.99 (3 H, m), 3.10–3.15 (3 H, s), 4.75–4.86 (2H, s), 4.86–4.95(1H, t, <i>J</i> = 5.8 Hz), 7.01–7.08 (2H, d, <i>J</i> = 8.8 Hz), 7.75–7.83 (2H, d, <i>J</i> = 8.8 Hz), 7.92–8.03 (2H, m), 8.16–8.21 (1H, s), 8.35–8.36 (1H, s), 8.88–9.01 (1H, s). HRMS for C<sub>22</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub> (M+H)<sup>+</sup>: calcd 390.1930; found 390.1928.</div></div><div id="sec4_3_1_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 5-(3-Chloro-5-(trifluoromethyl)pyridin-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">A solution of 3-chloro-5-(trifluoromethyl)picolinic acid (225 mg, 1 mmol), <i>N</i>-methylhydrazinecarbothioamide (105 mg, 1 mmol), and EDCI (230 mg, 1.2 mmol) in DMF (3 mL) was stirred for 12 h at rt, filtered, and the residue was purified by column chromatography to obtain 5-(3-chloro-5-(trifluoromethyl)pyridin-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (120 mg, 43%) as a light yellow solid. MS: [M + H]<sup>+</sup> = 278.</div></div><div id="sec4_3_1_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 5-(3-Chloro-5-(trifluoromethyl)pyridin-2-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>21</b>)</h5><div class="NLM_p last">In nitrogen atmosphere, a solution of 5-(3-chloro-5-(trifluoromethyl)pyridin-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (100 mg, 0. 36 mmol) and NaH (43 mg, 1.08 mmol) in DMF (2 mL) was stirred for 0.5 h at 25 °C. Then a solution of 6-(chloromethyl)imidazo[1,2-<i>a</i>]pyridine (120 mg, 0.72 mmol) in DMF (1 mL) was added. The resulting solution was stirred for 1 h at 25 °C, then quenched with water, filtered, and residue was purified by prep-HPLC to obtain 5-(3-chloro-5-(trifluoromethyl)pyridin-2-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (9.8 mg, 7%) as a white solid. <sup>1</sup>H NMR (400 MHz, MeOD) δ 3.25–3.27 (3H, s), 4.76–4.85 (2 H, s), 7.44–7.48 (1 H, dd, <i>J</i> = 9.2, 1.2 Hz), 7.58–7.70 (2H, m), 7.87–7.93(1H, s), 8.45–8.48(1H, s), 8.58–8.62 (1H, s), 8.93–8.98 (1H, s). HRMS (ES<sup>+</sup>) for C<sub>17</sub>H<sub>12</sub>N<sub>6</sub>OF<sub>3</sub>Cl (M+H)<sup>+</sup>: calcd 409.0791; found 409.0791.</div></div><div id="sec4_3_1_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 4-(Dimethylamino)-2-methylbenzoyl chloride</h5><div class="NLM_p last">A solution of 4-(dimethylamino)-2-methylbenzoic acid (300 mg, 1.67 mmol) and (COCl)<sub>2</sub> (0.6 mL) in DCM (15 mL) was stirred at 0 °C for 2 h and then concentrated to obtain 4-(dimethylamino)-2-methylbenzoyl chloride (350 mg, crude) as a white solid residue, which was used in the next step without purification.</div></div><div id="sec4_3_1_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 2-(4-(Dimethylamino)-2-methylbenzoyl)-<i>N</i>-methylhydrazinecarbothioamide</h5><div class="NLM_p last">A solution of 4-(dimethylamino)-2-methylbenzoyl chloride (350 mg, 1.17 mmol) and <i>N</i>-methylhydrazinecarbothioamide (280 mg, 2.66 mmol) in pyridine (8 mL) was stirred at 25 °C for 2 h, concentrated, and the residue was purified by reverse phased column chromatography to obtain 2-(4-(dimethylamino)-2-methylbenzoyl)-<i>N</i>-methylhydrazinecarbothioamide (310 mg, 65%) as a white solid. MS: [M + H]<sup>+</sup> = 267.</div></div><div id="sec4_3_1_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 5-(4-(Dimethylamino)-2-methylphenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">A solution of 2-(4-(dimethylamino)-2-methylbenzoyl)-<i>N</i>-methylhydrazinecarbothioamide (310 mg, 1.16 mmol), IBX (325 mg, 1.16 mmol), and triethylamine (234 mg, 2.32 mmol) in DCM (10 mL) was stirred at 0 °C for 3 h, concentrated, and the residue reverse phased column chromatography to obtain 5-(4-(dimethylamino)-2-methylphenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (130 mg, 48%) as a red solid. MS: [M + H]<sup>+</sup> = 233.</div></div><div id="sec4_3_1_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 5-(4-(Dimethylamino)-2-methylphenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine formate (<b>22</b>)</h5><div class="NLM_p last">NaH (112 mg, 2.8 mmol) was added to a solution of 5-(4-(dimethylamino)-2-methylphenyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (130 mg, 0.56 mmol) in DMF (3 mL) at 0 °C. The mixture was stirred at 0 °C for 30 min, then 6-(chloromethyl)imidazo[1,2-<i>a</i>]pyridine hydrochloride (114 mg, 0.56 mmol) was added. The solution was stirred at rt for 1 h, quenched with water, and purified by prep-HPLC to obtain 5-(4-(dimethylamino)-2-methylphenyl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine formate (16.6 mg, 12.7%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.52 (3 H, s), 2.97 (6 H, s), 3.08 (3 H, s), 4.64 (2 H, s), 6.60–6.67 (2 H, m), 7.25 (1 H, d, <i>J</i> = 9.6 Hz), 7.54–7.65 (3 H, m), 7.97 (1 H, s), 8.61 (1 H, s). HRMS for C<sub>20</sub>H<sub>22</sub>N<sub>6</sub>O (M+H)<sup>+</sup>: calcd 363.1933; found 363.1931.</div></div><div id="sec4_3_1_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 2-(1-Ethyl-1H-benzo[d]imidazole-2-carbonyl)-<i>N</i>-methylhydrazinecarbothioamide</h5><div class="NLM_p last">A solution of lithium 1-ethyl-1H-benzo[d]imidazole-2-carboxylate (400 mg, 2.041 mmol), <i>N</i>-methylhydrazinecarbothioamide (215 mg, 2.041 mmol), NaHCO<sub>3</sub> (515 mg, 6.123 mmol), and HATU (931 mg, 2.449 mmol) in DMF (6.0 mL) was stirred at rt for 1 h. The product was purified by column chromatography to obtain 2-(1-ethyl-1H-benzo[d]imidazole-2-carbonyl)-<i>N</i>-methylhydrazinecarbothioamide (294 mg, 52%) as an off-white solid. MS [M + H]<sup>+</sup> = 278.</div></div><div id="sec4_3_1_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 5-(1-Ethyl-1H-benzo[d]imidazol-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine</h5><div class="NLM_p last">IBX (253 mg, 0.903 mmol) was added to a solution of 2-(1-ethyl-1H-benzo[d]imidazole-2-carbonyl)-<i>N</i>-methylhydrazinecarbothioamide (250 mg, 0.903 mmol) and TEA (183 mg, 1.806 mmol) in DCM (4.0 mL) at 0 °C, and then the reaction mixture was stirred at 0 °C for 1 h. The product was extracted with DCM (10 mL × 4), washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and purified by column chromatography to obtain 5-(1-ethyl-1H-benzo[d]imidazol-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (133 mg, 61%) as an off-white solid. MS [M + H]<sup>+</sup> = 244.</div></div><div id="sec4_3_1_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 5-(1-Ethyl-1H-benzo[d]imidazol-2-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (<b>23</b>)</h5><div class="NLM_p last">A solution of 5-(1-ethyl-1H-benzo[d]imidazol-2-yl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (100 mg, 0.412 mmol) and NaH (50 mg, 1.23 mmol) in DMF (4.0 mL) was stirred 0 °C for 30 min, and then 6-(chloromethyl)imidazo[1,2-<i>a</i>]pyridine (69 mg, 0. 412 mmol) was added. The reaction mixture was stirred at rt for 1 h, and then the product was purified by prep-HPLC to obtain 5-(1-ethyl-1H-benzo[d]imidazol-2-yl)-<i>N</i>-(imidazo[1,2-<i>a</i>]pyridin-6-ylmethyl)-<i>N</i>-methyl-1,3,4-oxadiazol-2-amine (19.6 mg, 13%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (3 H, t, <i>J</i> = 7.0 Hz), 3.17 (3 H, s), 4.69–4.80 (4 H, m), 7.24–7.45 (3 H, m), 7.56–7.65 (2 H, m), 7.71–7.81 (2 H, m), 7.96 (1 H, d), 8.61–8.67 (1 H, m). HRMS for C<sub>20</sub>H<sub>19</sub>N<sub>7</sub>O (M+H)<sup>+</sup>: calcd 374.1729; found 374.1729.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01904" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01904?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01904</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Standard deviations of measurements; Mer activity in high-ATP assay; kinase selectivity profiles of <b>12</b>–<b>14</b>; crystallographical details; assay descriptions (Axl, Flt GLO assays); selection conditions used in DECL screening and prevalence tables for families A, B, and C; purity data for compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, CSV file PDB accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY">7AVY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ">7AVZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0">7AW0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1">7AW1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2">7AW2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4">7AW4</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf">jm0c01904_si_001.pdf (1.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_002.csv">jm0c01904_si_002.csv (2.12 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01904" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Willem M. Nissink</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2572-9140" title="Orcid link">http://orcid.org/0000-0003-2572-9140</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e196888d8d848ccf8f889292888f8aa180929593809b848f848280cf8f8495"><span class="__cf_email__" data-cfemail="63140a0f0f060e4d0d0a10100a0d0823021017110219060d0600024d0d0617">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sana Bazzaz</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carolyn Blackett</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew A. Clark</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olga Collingwood</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy S. Disch</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diana Gikunju</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristin Goldberg</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John P. Guilinger</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth Hardaker</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward J. Hennessy</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, R&D Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachael Jetson</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony D. Keefe</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William McCoull</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2977-1744" title="Orcid link">http://orcid.org/0000-0002-2977-1744</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsay McMurray</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allison Olszewski</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ross Overman</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander Pflug</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marian Preston</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip B. Rawlins</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma Rivers</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marianne Schimpl</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2284-5250" title="Orcid link">http://orcid.org/0000-0003-2284-5250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Smith</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline Truman</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth Underwood</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juli Warwicker</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon Winter-Holt</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Woodcock</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Zhang</span> - <span class="hlFld-Affiliation affiliation">X-Chem,
Inc., Waltham, Massachusetts 02453, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): J.W.M.N., C.B., K.G., E.H., W.M., L.M., O.C., R.O., A.P., M.P., P.R., E.R., M.S., E.U., C.T., J.Wa., J.Wi., S.W., and Y.Z. are or were employees of AstraZeneca at the time of writing. Remaining authors are or were employees of XChem Ltd. at the time of writing. Employees and past employees may hold stocks and shares.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4356-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Jason Breed for crystallographic data collection, the beamline scientists at DLS I04 and I04-1 and Soleil Proxima II for technical assistance, Dr. Nikki Carter for assay development, and Dr. D. Barratt for collating the assay procedures. Drs. Dean Brown, Andrew Madin, and Garry Pairaudeau are acknowledged for chemistry discussions and Dr. Kristin Rosner for project support.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54003" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54003" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This paper was published on the Web on March 8, 2021, with inconsistencies in the structures in Figure 2, Table 1, and Tables 3−8.  The corrected version was reposted on March 8, 2021.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">DECL</td><td class="NLM_def"><p class="first last">DNA-Encoded Chemical Library</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">MoA</td><td class="NLM_def"><p class="first last">mode of action</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">(1-[bis(dimethylamino)methylene]-1H-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80417" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80417" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myers, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amend, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Tyro3, Axl and MerTK (TAM receptors): implications for macrophages in the tumor microenvironment</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">94</span>, <span class="refDoi"> DOI: 10.1186/s12943-019-1022-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1186%2Fs12943-019-1022-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=31088471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A280%3ADC%252BB3M7lsleiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=94&author=K.+V.+Myersauthor=S.+R.+Amendauthor=K.+J.+Pienta&title=Targeting+Tyro3%2C+Axl+and+MerTK+%28TAM+receptors%29%3A+implications+for+macrophages+in+the+tumor+microenvironment&doi=10.1186%2Fs12943-019-1022-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Tyro3, Axl and MerTK (TAM receptors): implications for macrophages in the tumor microenvironment</span></div><div class="casAuthors">Myers Kayla V; Pienta Kenneth J; Myers Kayla V; Amend Sarah R; Pienta Kenneth J; Pienta Kenneth J; Pienta Kenneth J</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tumor-associated macrophages are an abundant cell type in the tumor microenvironment.  These macrophages serve as a promising target for treatment of cancer due to their roles in promoting cancer progression and simultaneous immunosuppression.  The TAM receptors (Tyro3, Axl and MerTK) are promising therapeutic targets on tumor-associated macrophages.  The TAM receptors are a family of receptor tyrosine kinases with shared ligands Gas6 and Protein S that skew macrophage polarization towards a pro-tumor M2-like phenotype.  In macrophages, the TAM receptors also promote apoptotic cell clearance, a tumor-promoting process called efferocytosis.  The TAM receptors bind the "eat-me" signal phosphatidylserine on apoptotic cell membranes using Gas6 and Protein S as bridging ligands.  Post-efferocytosis, macrophages are further polarized to a pro-tumor M2-like phenotype and secrete increased levels of immunosuppressive cytokines.  Since M2 polarization and efferocytosis are tumor-promoting processes, the TAM receptors on macrophages serve as exciting targets for cancer therapy.  Current TAM receptor-directed therapies in preclinical development and clinical trials may have anti-cancer effects though impacting macrophage phenotype and function in addition to the cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTmoOYd9tKrj6x48Rj5GnGofW6udTcc2eYYU2e4tpUzS7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M7lsleiuw%253D%253D&md5=98dcbcc47e29f0c18f2a52920900975c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12943-019-1022-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-019-1022-2%26sid%3Dliteratum%253Aachs%26aulast%3DMyers%26aufirst%3DK.%2BV.%26aulast%3DAmend%26aufirst%3DS.%2BR.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26atitle%3DTargeting%2520Tyro3%252C%2520Axl%2520and%2520MerTK%2520%2528TAM%2520receptors%2529%253A%2520implications%2520for%2520macrophages%2520in%2520the%2520tumor%2520microenvironment%26jtitle%3DMol.%2520Cancer%26date%3D2019%26volume%3D18%26spage%3D94%26doi%3D10.1186%2Fs12943-019-1022-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimaudo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pipitone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lori, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raggi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, F.</span></span> <span> </span><span class="NLM_article-title">Role of myeloid-epithelial-reproductive tyrosine kinase and macrophage polarization in the progression of atherosclerotic lesions associated with nonalcoholic fatty liver disease</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">604</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.3389%2Ffphar.2019.00604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=31191323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkt12it7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=604&author=M.+Pastoreauthor=S.+Grimaudoauthor=R.+M.+Pipitoneauthor=G.+Loriauthor=C.+Raggiauthor=S.+Pettaauthor=F.+Marra&title=Role+of+myeloid-epithelial-reproductive+tyrosine+kinase+and+macrophage+polarization+in+the+progression+of+atherosclerotic+lesions+associated+with+nonalcoholic+fatty+liver+disease&doi=10.3389%2Ffphar.2019.00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of myeloid-epithelial-reproductive tyrosine kinase and macrophage polarization in the progression of atherosclerotic lesions associated with nonalcoholic fatty liver disease</span></div><div class="casAuthors">Pastore, Mirella; Grimaudo, Stefania; Pipitone, Rosaria Maria; Lori, Giulia; Raggi, Chiara; Petta, Salvatore; Marra, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">604</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Recent lines of evidence highlight the involvement of myeloid-epithelial-reproductive tyrosine kinase (MerTK) in metabolic disease assocd. with liver damage.  MerTK is mainly expressed in anti-inflammatory M2 macrophages where it mediates transcriptional changes including suppression of proinflammatory cytokines and enhancement of inflammatory repressors.  MerTK is regulated by metabolic pathways through nuclear sensors including LXRs, PPARs, and RXRs, in response to apoptotic bodies or to other sources of cholesterol.  Nonalcoholic fatty liver disease (NAFLD) is one of the most serious public health problems worldwide.  It is a clinicopathol. syndrome closely related to obesity, insulin resistance, and oxidative stress.  It includes a spectrum of conditions ranging from simple steatosis, characterized by hepatic fat accumulation with or without inflammation, to nonalcoholic steatohepatitis (NASH), defined by hepatic fat deposition with hepatocellular damage, inflammation, and accumulating fibrosis.  Several studies support an assocn. between NAFLD and the incidence of cardiovascular diseases including atherosclerosis, a major cause of death worldwide.  This pathol. condition consists in a chronic and progressive inflammatory process in the intimal layer of large- and medium-sized arteries.  The complications of advanced atherosclerosis include chronic or acute ischemic damage in the tissue perfused by the affected artery, leading to cellular death.  By identifying specific targets influencing lipid metab. and cardiovascularrelated diseases, the present review highlights the role of MerTK in NAFLD-assocd. atherosclerotic lesions as a potential innovative therapeutic target.  Therapeutic advantages might derive from the use of compds. selective for nuclear receptors targeting PPARs rather than LXRs regulating macrophage lipid metab. and macrophage mediated inflammation, by favoring the expression of MerTK, which mediates an immunoregulatory action with a redn. in inflammation and in atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJbW7XUoajorVg90H21EOLACvtfcHk0lhnplKyBrD_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkt12it7w%253D&md5=09f520d512ff856620b2f50220614496</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00604%26sid%3Dliteratum%253Aachs%26aulast%3DPastore%26aufirst%3DM.%26aulast%3DGrimaudo%26aufirst%3DS.%26aulast%3DPipitone%26aufirst%3DR.%2BM.%26aulast%3DLori%26aufirst%3DG.%26aulast%3DRaggi%26aufirst%3DC.%26aulast%3DPetta%26aufirst%3DS.%26aulast%3DMarra%26aufirst%3DF.%26atitle%3DRole%2520of%2520myeloid-epithelial-reproductive%2520tyrosine%2520kinase%2520and%2520macrophage%2520polarization%2520in%2520the%2520progression%2520of%2520atherosclerotic%2520lesions%2520associated%2520with%2520nonalcoholic%2520fatty%2520liver%2520disease%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D604%26doi%3D10.3389%2Ffphar.2019.00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. S.</span></span> <span> </span><span class="NLM_article-title">Efferocytosis Signaling in the regulation of macrophage inflammatory responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1394</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1601520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4049%2Fjimmunol.1601520" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=28167649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFWks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=1387-1394&author=M.+R.+Elliottauthor=K.+M.+Kosterauthor=P.+S.+Murphy&title=Efferocytosis+Signaling+in+the+regulation+of+macrophage+inflammatory+responses&doi=10.4049%2Fjimmunol.1601520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Efferocytosis Signaling in the Regulation of Macrophage Inflammatory Responses</span></div><div class="casAuthors">Elliott, Michael R.; Koster, Kyle M.; Murphy, Patrick S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1387-1394</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Since the pioneering work of Elie Metchnikoff and the discovery of cellular immunity, the phagocytic clearance of cellular debris has been considered an integral component of resolving inflammation and restoring function of damaged and infected tissues.  We now know that the phagocytic clearance of dying cells (efferocytosis), particularly by macrophages and other immune phagocytes, has profound consequences on innate and adaptive immune responses in inflamed tissues.  These immunomodulatory effects result from an array of mol. signaling events between macrophages, dying cells, and other tissue-resident cells.  In recent years, many of these mol. pathways have been identified and studied in the context of tissue inflammation, helping us better understand the relationship between efferocytosis and inflammation.  We review specific types of efferocytosis-related signals that can impact macrophage immune responses and discuss their relevance to inflammation-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMMZBczZGz7Vg90H21EOLACvtfcHk0lha_KRfHWR8UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFWks74%253D&md5=f8ef4ac7d54d4d577ba385a8918dc350</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1601520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1601520%26sid%3Dliteratum%253Aachs%26aulast%3DElliott%26aufirst%3DM.%2BR.%26aulast%3DKoster%26aufirst%3DK.%2BM.%26aulast%3DMurphy%26aufirst%3DP.%2BS.%26atitle%3DEfferocytosis%2520Signaling%2520in%2520the%2520regulation%2520of%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2017%26volume%3D198%26spage%3D1387%26epage%3D1394%26doi%3D10.4049%2Fjimmunol.1601520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gay, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byers, L. A.</span></span> <span> </span><span class="NLM_article-title">Giving AXL the axe: targeting AXL in human malignancy</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/bjc.2016.428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2Fbjc.2016.428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=28072762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVSruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2017&pages=415-423&author=C.+M.+Gayauthor=K.+Balajiauthor=L.+A.+Byers&title=Giving+AXL+the+axe%3A+targeting+AXL+in+human+malignancy&doi=10.1038%2Fbjc.2016.428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Giving AXL the axe: targeting AXL in human malignancy</span></div><div class="casAuthors">Gay, Carl M.; Balaji, Kavitha; Byers, Lauren Averett</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">415-423</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase AXL, activated by a complex interaction between its ligand growth arrest-specific protein 6 and phosphatidylserine, regulates various vital cellular processes, including proliferation, survival, motility, and immunol. response.  Although not implicated as an oncogenic driver itself, AXL, a member of the TYRO3, AXL, and MERTK family of receptor tyrosine kinases, is overexpressed in several haematol. and solid malignancies, including acute myeloid leukemia, non-small cell lung cancer, gastric and colorectal adenocarcinomas, and breast and prostate cancers.  In the context of malignancy, evidence suggests that AXL overexpression drives wide-ranging processes, including epithelial to mesenchymal transition, tumor angiogenesis, resistance to chemotherapeutic and targeted agents, and decreased antitumor immune response.  As a result, AXL is an attractive candidate not only as a prognostic biomarker in malignancy but also as a target for anticancer therapies.  Several AXL inhibitors are currently in preclin. and clin. development.  This article reviews the structure, regulation, and function of AXL; the role of AXL in the tumor microenvironment; the development of AXL as a therapeutic target; and areas of ongoing and future investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXM9RJ8TVsLVg90H21EOLACvtfcHk0lha_KRfHWR8UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVSruw%253D%253D&md5=c24668744fe4b017def75b08bac2a863</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2016.428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2016.428%26sid%3Dliteratum%253Aachs%26aulast%3DGay%26aufirst%3DC.%2BM.%26aulast%3DBalaji%26aufirst%3DK.%26aulast%3DByers%26aufirst%3DL.%2BA.%26atitle%3DGiving%2520AXL%2520the%2520axe%253A%2520targeting%2520AXL%2520in%2520human%2520malignancy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2017%26volume%3D116%26spage%3D415%26epage%3D423%26doi%3D10.1038%2Fbjc.2016.428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camenisch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boast, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casagranda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earp, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, G.</span></span> <span> </span><span class="NLM_article-title">Tyro-3 family receptors are essential regulators of mammalian spermatogenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>398</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/19554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2F19554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10227296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADyaK1MXivVegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=398&publication_year=1999&pages=723-728&author=Q.+Luauthor=M.+Goreauthor=Q.+Zhangauthor=T.+Camenischauthor=S.+Boastauthor=F.+Casagrandaauthor=C.+Laiauthor=M.+K.+Skinnerauthor=R.+Kleinauthor=G.+K.+Matsushimaauthor=H.+S.+Earpauthor=S.+P.+Goffauthor=G.+Lemke&title=Tyro-3+family+receptors+are+essential+regulators+of+mammalian+spermatogenesis&doi=10.1038%2F19554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tyro-3 family receptors are essential regulators of mammalian spermatogenesis</span></div><div class="casAuthors">Lu, Qingxian; Gore, Martin; Zhang, Qing; Camenisch, Todd; Boast, Sharon; Casagranda, Franca; Skinner, Michael K.; Klein, Riudiger; Matsushima, Glenn K.; Earp, H. Shelton; Goff, Stephen P.; Lemke, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">398</span>
        (<span class="NLM_cas:issue">6729</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">We have generated and analyzed null mutations in the mouse genes encoding three structurally related receptors with tyrosine kinase activity: Tyro 3, Axl, and Mer.  Mice lacking any single receptor, or any combination of two receptors, are viable and fertile, but male animals that lack all three receptors produce no mature sperm, owing to the progressive death of differentiating germ cells.  This degenerative phenotype appears to result from a failure of the tropic support that is normally provided by Sertoli cells of the seminiferous tubules, whose function depends on testosterone and addnl. factors produced by Leydig cells.  Tyro 3, Axl and Mer are all normally expressed by Sertoli cells during postnatal development, whereas their ligands, Gas6 and protein S, are produced by Leydig cells before sexual maturity, and by both Leydig and Sertoli cells thereafter.  Here we show that the concerted activation of Tyro 3, Axl and Mer in Sertoli cells is crit. to the role that these cells play as nurturers of developing germ cells.  Addnl. observations indicate that these receptors may also be essential for the tropic maintenance of diverse cell types in the mature nervous, immune and reproductive systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY4F6tCv0xxbVg90H21EOLACvtfcHk0lha_KRfHWR8UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVegurk%253D&md5=f254abf74c110fcd3eb1fc773f4b46f7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F19554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F19554%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DGore%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DCamenisch%26aufirst%3DT.%26aulast%3DBoast%26aufirst%3DS.%26aulast%3DCasagranda%26aufirst%3DF.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DSkinner%26aufirst%3DM.%2BK.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DMatsushima%26aufirst%3DG.%2BK.%26aulast%3DEarp%26aufirst%3DH.%2BS.%26aulast%3DGoff%26aufirst%3DS.%2BP.%26aulast%3DLemke%26aufirst%3DG.%26atitle%3DTyro-3%2520family%2520receptors%2520are%2520essential%2520regulators%2520of%2520mammalian%2520spermatogenesis%26jtitle%3DNature%26date%3D1999%26volume%3D398%26spage%3D723%26epage%3D728%26doi%3D10.1038%2F19554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasileiadi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRyckere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. K.</span></span> <span> </span><span class="NLM_article-title">The emerging role of TYRO3 as a therapeutic target in cancer</span>. <i>Cancers (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">474</span>, <span class="refDoi"> DOI: 10.3390/cancers10120474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.3390%2Fcancers10120474" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=474&author=S.+K.+Smartauthor=E.+Vasileiadiauthor=X.+Wangauthor=D.+DeRyckereauthor=D.+K.+Graham&title=The+emerging+role+of+TYRO3+as+a+therapeutic+target+in+cancer&doi=10.3390%2Fcancers10120474"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3390%2Fcancers10120474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10120474%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DS.%2BK.%26aulast%3DVasileiadi%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDeRyckere%26aufirst%3DD.%26aulast%3DGraham%26aufirst%3DD.%2BK.%26atitle%3DThe%2520emerging%2520role%2520of%2520TYRO3%2520as%2520a%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DCancers%2520%2528Basel%2529%26date%3D2018%26volume%3D10%26spage%3D474%26doi%3D10.3390%2Fcancers10120474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janda, K. D.</span></span> <span> </span><span class="NLM_article-title">Synthetic methods for the implementation of encoded combinatorial chemistry</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">9812</span>– <span class="NLM_lpage">9813</span>, <span class="refDoi"> DOI: 10.1021/ja00074a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00074a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVOls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=9812-9813&author=J.+Nielsenauthor=S.+Brennerauthor=K.+D.+Janda&title=Synthetic+methods+for+the+implementation+of+encoded+combinatorial+chemistry&doi=10.1021%2Fja00074a063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic methods for the implementation of encoded combinatorial chemistry</span></div><div class="casAuthors">Nielsen, John; Brenner, Sydney; Janda, Kim D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9812-13</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">There has been a recent renaissance in drug screening with the development of new technologies which allow a large no. of compds. to be simultaneously exposed to a target.  In these "combinatorial libraries", compds. that bind to the target with the highest affinity are selected from the pool of statistical sequences.  Recently, a scheme for encoding combinatorially synthesized libraries has been proposed to surmount a no. of the limitations possessed by existing methods.  Encoded combinatorial chem. combines the large diversity that can be achieved with a chem. library with an encoded genetic tag which can be used for the identification and sequential enrichment of any active component.  The authors have now developed the chem. necessary to implement the conceptual scheme and how a CPG matrix can be appended to allow the parallel synthesis of peptides and their encoding nucleic acid sequences in an alternating, bi-directional manner.  In addn. the authors demonstrate how the same support can be modified to permit a controlled "dendritic" display of the chem. library.  Implementation of this latter regime provides a novel methodol. for controlled multivalent combinatorial ligand display.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbyd3WhXqvG7Vg90H21EOLACvtfcHk0li9-3zd4JgRPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhtVOls7k%253D&md5=ae66399eb84c4a5af0e0a9d76960bb49</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja00074a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00074a063%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DJanda%26aufirst%3DK.%2BD.%26atitle%3DSynthetic%2520methods%2520for%2520the%2520implementation%2520of%2520encoded%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D9812%26epage%3D9813%26doi%3D10.1021%2Fja00074a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Encoded self-assembling chemical libraries</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nbt961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2Fnbt961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=15097996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2004&pages=568-574&author=S.+Melkkoauthor=J.+Scheuermannauthor=C.+E.+Dumelinauthor=D.+Neri&title=Encoded+self-assembling+chemical+libraries&doi=10.1038%2Fnbt961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded self-assembling chemical libraries</span></div><div class="casAuthors">Melkko, Samu; Scheuermann, Joerg; Dumelin, Christoph E.; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">568-574</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The isolation of mols. capable of high-affinity and specific binding to biol. targets is a central problem in chem., biol. and pharmaceutical sciences.  Here we describe the use of encoded self-assembling chem. (ESAC) libraries for the facile identification of mols. that bind macromol. targets.  ESAC technol. uses libraries of org. mols. linked to individual oligonucleotides that mediate the self-assembly of the library and provide a code assocd. with each org. mol.  After panning ESAC libraries on the biomol. target of interest, the 'binding code' of the selected compds. can be 'decoded' by a no. of exptl. techniques (e.g., hybridization on oligonucleotide microarrays).  The potential of this technol. was demonstrated by the affinity maturation (>40-fold) of binding mols. to human serum albumin and bovine carbonic anhydrase, leading to binders with dissocn. consts. in the nanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojeEDxP4G3SLVg90H21EOLACvtfcHk0li9-3zd4JgRPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jsLg%253D&md5=aa09960ffa2b0dd1e4b45a4c5d0ab1a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt961%26sid%3Dliteratum%253Aachs%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DEncoded%2520self-assembling%2520chemical%2520libraries%26jtitle%3DNat.%2520Biotechnol.%26date%3D2004%26volume%3D22%26spage%3D568%26epage%3D574%26doi%3D10.1038%2Fnbt961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whartenby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabresi, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCadden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kardian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardoll, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of FLT3 signaling targets DCs to ameliorate autoimmune disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">16741</span>– <span class="NLM_lpage">16746</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506088102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1073%2Fpnas.0506088102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=16272221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KgsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=16741-16746&author=K.+A.+Whartenbyauthor=P.+A.+Calabresiauthor=E.+McCaddenauthor=B.+Nguyenauthor=D.+Kardianauthor=T.+Wangauthor=C.+Mosseauthor=D.+M.+Pardollauthor=D.+Small&title=Inhibition+of+FLT3+signaling+targets+DCs+to+ameliorate+autoimmune+disease&doi=10.1073%2Fpnas.0506088102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of FLT3 signaling targets DCs to ameliorate autoimmune disease</span></div><div class="casAuthors">Whartenby, Katharine A.; Calabresi, Peter A.; McCadden, Erin; Nguyen, Bao; Kardian, David; Wang, Tianhong; Mosse, Claudio; Pardoll, Drew M.; Small, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">16741-16746</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autoimmune diseases often result from inappropriate or unregulated activation of autoreactive T cells.  Traditional approaches to treatment of autoimmune diseases through immunosuppression have focused on direct inhibition of T cells.  In the present study, the authors examd. the targeted inhibition of antigen-presenting cells as a means to down-regulate immune responses and treat autoimmune disease.  Dendritic cells (DCs) are the central antigen-presenting cells for the initiation of T cell responses, including autoreactive ones.  A large portion of DCs are derived from hematopoietic progenitors that express FLT3 receptor (CD135), and stimulation of the receptor via FLT3 ligand either in vivo or in vitro is known to drive expansion and differentiation of these progenitors toward a DC phenotype.  The authors hypothesized that inhibition of FLT3 signaling would thus produce an inhibition of DC-induced stimulation of T cells, thereby inhibiting autoimmune responses.  To this end, the authors used small-mol. tyrosine kinase inhibitors targeted against FLT3 and examd. the effects on DCs and their role in the promulgation of autoimmune disease.  Results of our studies show that inhibition of FLT3 signaling induces apoptosis in both mouse and human DCs, and thus is a potential target for immune suppression.  Furthermore, targeted inhibition of FLT3 significantly improved the course of established disease in a model for multiple sclerosis, exptl. autoimmune encephalomyelitis, suggesting a potential avenue for treating autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-kLdUEFSK7Vg90H21EOLACvtfcHk0li7oDYw_bXPkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KgsLzL&md5=12e54a144aa8d70af7ec018e1bd086f4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506088102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506088102%26sid%3Dliteratum%253Aachs%26aulast%3DWhartenby%26aufirst%3DK.%2BA.%26aulast%3DCalabresi%26aufirst%3DP.%2BA.%26aulast%3DMcCadden%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DKardian%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DMosse%26aufirst%3DC.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520FLT3%2520signaling%2520targets%2520DCs%2520to%2520ameliorate%2520autoimmune%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D16741%26epage%3D16746%26doi%3D10.1073%2Fpnas.0506088102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karsunky, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, M. G.</span></span> <span> </span><span class="NLM_article-title">Flt3 ligand regulates dendritic cell development from Flt3+ lymphoid and myeloid-committed progenitors to Flt3+ dendritic cells in vivo</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1084/jem.20030323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1084%2Fjem.20030323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=12874263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVyjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2003&pages=305-313&author=H.+Karsunkyauthor=M.+Meradauthor=A.+Cozzioauthor=I.+L.+Weissmanauthor=M.+G.+Manz&title=Flt3+ligand+regulates+dendritic+cell+development+from+Flt3%2B+lymphoid+and+myeloid-committed+progenitors+to+Flt3%2B+dendritic+cells+in+vivo&doi=10.1084%2Fjem.20030323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Flt3 ligand regulates dendritic cell development from Flt3+ lymphoid and myeloid-committed progenitors to Flt3+ dendritic cells in vivo</span></div><div class="casAuthors">Karsunky, Holger; Merad, Miriam; Cozzio, Antonio; Weissman, Irving L.; Manz, Markus G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Stimulation of Flt3 receptor tyrosine kinase through its cognate ligand expands early hematopoietic progenitor and dendritic cells (DCs) in humans and mice.  The exact developmental stages at which hematopoietic progenitors express Flt3, are responsive to its ligand, and subsequently develop to DCs, are not known.  Here the authors show that common lymphoid and common myeloid progenitors, as well as steady state DCs in thymus, spleen, and epidermis, express Flt3.  The receptor is down-regulated once definitive B cell, T cell, and megakaryocyte/erythrocyte commitment occurs, and Flt3 is not detectable on other steady state hematopoietic cell populations.  Upon in vivo Flt3 ligand (Flt3L) administration, Flt3+ progenitor cells and their progeny DCs are expanded, whereas Flt3- downstream progenitors are not, or are only slightly increased.  Transplantation of common lymphoid and common myeloid progenitors and subsequent Flt3L injection increases progeny DCs of both precursor populations.  These findings provide a definitive map of Flt3 expression in the hematopoietic hierarchy and directly demonstrate that Flt3L can drive DC development along both the lymphoid and myeloid developmental pathways from Flt3+ progenitors to Flt3+ DCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKwAXIjMecULVg90H21EOLACvtfcHk0li7oDYw_bXPkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVyjtbk%253D&md5=04de331176e634038283e86ff00d637a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1084%2Fjem.20030323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20030323%26sid%3Dliteratum%253Aachs%26aulast%3DKarsunky%26aufirst%3DH.%26aulast%3DMerad%26aufirst%3DM.%26aulast%3DCozzio%26aufirst%3DA.%26aulast%3DWeissman%26aufirst%3DI.%2BL.%26aulast%3DManz%26aufirst%3DM.%2BG.%26atitle%3DFlt3%2520ligand%2520regulates%2520dendritic%2520cell%2520development%2520from%2520Flt3%252B%2520lymphoid%2520and%2520myeloid-committed%2520progenitors%2520to%2520Flt3%252B%2520dendritic%2520cells%2520in%2520vivo%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2003%26volume%3D198%26spage%3D305%26epage%3D313%26doi%3D10.1084%2Fjem.20030323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komarowska, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbruchel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitzalis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckenridge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marelli-Berg, F. M.</span></span> <span> </span><span class="NLM_article-title">Hepatocyte growth factor receptor c-Met instructs T cell cardiotropism and promotes T cell migration to the heart via autocrine chemokine release</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1087</span>– <span class="NLM_lpage">1099</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2015.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.immuni.2015.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26070483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWjtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=1087-1099&author=I.+Komarowskaauthor=D.+Coeauthor=G.+Wangauthor=R.+Haasauthor=C.+Mauroauthor=M.+Kishoreauthor=D.+Cooperauthor=S.+Nadkarniauthor=H.+Fuauthor=D.+A.+Steinbruchelauthor=C.+Pitzalisauthor=G.+Andersonauthor=P.+Bucyauthor=G.+Lombardiauthor=R.+Breckenridgeauthor=F.+M.+Marelli-Berg&title=Hepatocyte+growth+factor+receptor+c-Met+instructs+T+cell+cardiotropism+and+promotes+T+cell+migration+to+the+heart+via+autocrine+chemokine+release&doi=10.1016%2Fj.immuni.2015.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocyte Growth Factor Receptor c-Met Instructs T Cell Cardiotropism and Promotes T Cell Migration to the Heart via Autocrine Chemokine Release</span></div><div class="casAuthors">Komarowska, Izabela; Coe, David; Wang, Guosu; Haas, Robert; Mauro, Claudio; Kishore, Madhav; Cooper, Dianne; Nadkarni, Suchita; Fu, Hongmei; Steinbruchel, Daniel A.; Pitzalis, Costantino; Anderson, Graham; Bucy, Pat; Lombardi, Giovanna; Breckenridge, Ross; Marelli-Berg, Federica M.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1087-1099</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Effector-T-cell-mediated immunity depends on the efficient localization of antigen-primed lymphocytes to antigen-rich non-lymphoid tissue, which is facilitated by the expression of a unique set of "homing" receptors acquired by memory T cells.  We report that engagement of the hepatocyte growth factor (HGF) receptor c-Met by heart-produced HGF during priming in the lymph nodes instructs T cell cardiotropism, which was assocd. with a specialized homing "signature" (c-Met+CCR4+CXCR3+). c-Met signals facilitated T cell recruitment to the heart via the chemokine receptor CCR5 by inducing autocrine CCR5 ligand release. c-Met triggering was sufficient to support cardiotropic T cell recirculation, while CCR4 and CXCR3 sustained recruitment during heart inflammation.  Transient pharmacol. blockade of c-Met during T cell priming led to enhanced survival of heart, but not skin, allografts assocd. with impaired localization of alloreactive T cells to heart grafts.  These findings suggest c-Met as a target for development of organ-selective immunosuppressive therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLOLQYPIfr4rVg90H21EOLACvtfcHk0lhNna1Dj9QU4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWjtL3F&md5=be1c5b716f7c595889f1ee7a9eb3d186</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DKomarowska%26aufirst%3DI.%26aulast%3DCoe%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHaas%26aufirst%3DR.%26aulast%3DMauro%26aufirst%3DC.%26aulast%3DKishore%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DD.%26aulast%3DNadkarni%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DSteinbruchel%26aufirst%3DD.%2BA.%26aulast%3DPitzalis%26aufirst%3DC.%26aulast%3DAnderson%26aufirst%3DG.%26aulast%3DBucy%26aufirst%3DP.%26aulast%3DLombardi%26aufirst%3DG.%26aulast%3DBreckenridge%26aufirst%3DR.%26aulast%3DMarelli-Berg%26aufirst%3DF.%2BM.%26atitle%3DHepatocyte%2520growth%2520factor%2520receptor%2520c-Met%2520instructs%2520T%2520cell%2520cardiotropism%2520and%2520promotes%2520T%2520cell%2520migration%2520to%2520the%2520heart%2520via%2520autocrine%2520chemokine%2520release%26jtitle%3DImmunity%26date%3D2015%26volume%3D42%26spage%3D1087%26epage%3D1099%26doi%3D10.1016%2Fj.immuni.2015.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhade, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadri, A.</span></span> <span> </span><span class="NLM_article-title">Src kinases central to T-cell receptor signaling regulate TLR-activated innate immune responses from human T cells</span>. <i>Innate Immun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1177/1753425916632305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1177%2F1753425916632305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26888964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWkurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=238-244&author=N.+Sharmaauthor=A.+S.+Akhadeauthor=A.+Qadri&title=Src+kinases+central+to+T-cell+receptor+signaling+regulate+TLR-activated+innate+immune+responses+from+human+T+cells&doi=10.1177%2F1753425916632305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Src kinases central to T-cell receptor signaling regulate TLR-activated innate immune responses from human T cells</span></div><div class="casAuthors">Sharma, Naveen; Akhade, Ajay Suresh; Qadri, Ayub</div><div class="citationInfo"><span class="NLM_cas:title">Innate Immunity</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">238-244</span>CODEN:
                <span class="NLM_cas:coden">IINMCB</span>;
        ISSN:<span class="NLM_cas:issn">1753-4259</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">TLRs have a fundamental role in immunity.  We have recently reported that stimulation of TLR2 and TLR5 in freshly isolated and activated human T cells with microbial ligands without concomitant activation through the TCR brings about secretion of neutrophil chemoattractant, CXCL8, and effector cytokine, IFN-γ, resp.  However, the mechanism of TLR signaling in T cells has not been worked out.  Here, we show that the Src family kinases, p56lck (Lck) and p59fyn (Fyn), which are essential for activation of T cells through the TCR, are also crit. for signal transduction through TLRs in human T cells.  The secretion of CXCL8 following stimulation of the model human T cell line, Jurkat, with the TLR5 ligand, flagellin, was reduced in presence of the Src-kinase inhibitor, PP2 and specific inhibitors of Lck and Fyn.  These inhibitors suppressed generation of activated JNK and p38, which were both required for TLR-induced CXCL8 prodn.  The Lck-deficient deriv. of Jurkat, JCam1.6, responded poorly to TLR2, TLR5 and TLR7 agonists, and did not generate active signaling intermediates.  Lck and Fyn inhibitors also reduced TLR5-induced IFN-γ secretion from the activated T cell phenotype-representing T cell line, HuT78, without modulating JNK and p38 activation.  These results reveal that TCR and TLRs share key proximal signaling regulators in T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqD-A6nTkqZbVg90H21EOLACvtfcHk0lhNna1Dj9QU4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWkurfJ&md5=b9e7e36da131e33c45d583f8cbed7bf3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1177%2F1753425916632305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1753425916632305%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DAkhade%26aufirst%3DA.%2BS.%26aulast%3DQadri%26aufirst%3DA.%26atitle%3DSrc%2520kinases%2520central%2520to%2520T-cell%2520receptor%2520signaling%2520regulate%2520TLR-activated%2520innate%2520immune%2520responses%2520from%2520human%2520T%2520cells%26jtitle%3DInnate%2520Immun.%26date%3D2016%26volume%3D22%26spage%3D238%26epage%3D244%26doi%3D10.1177%2F1753425916632305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawagoe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uematsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span> <span> </span><span class="NLM_article-title">Essential role of IRAK-4 protein and its kinase activity in Toll-like receptor-mediated immune responses but not in TCR signaling</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1084/jem.20061523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1084%2Fjem.20061523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=17485511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFSjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2007&pages=1013-24&author=T.+Kawagoeauthor=S.+Satoauthor=A.+Jungauthor=M.+Yamamotoauthor=K.+Matsuiauthor=H.+Katoauthor=S.+Uematsuauthor=O.+Takeuchiauthor=S.+Akira&title=Essential+role+of+IRAK-4+protein+and+its+kinase+activity+in+Toll-like+receptor-mediated+immune+responses+but+not+in+TCR+signaling&doi=10.1084%2Fjem.20061523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of IRAK-4 protein and its kinase activity in Toll-like receptor-mediated immune responses but not in TCR signaling</span></div><div class="casAuthors">Kawagoe, Tatsukata; Sato, Shintaro; Jung, Andreas; Yamamoto, Masahiro; Matsui, Kosuke; Kato, Hiroki; Uematsu, Satoshi; Takeuchi, Osamu; Akira, Shizuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1013-1024</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin-1 receptor-assocd. kinase 4 (IRAK-4) was reported to be essential for the Toll-like receptor (TLR)- and T cell receptor (TCR)-mediated signaling leading to the activation of nuclear factor κB (NF-κB).  However, the importance of kinase activity of IRAK family members is unclear.  In this study, we investigated the functional role of IRAK-4 activity in vivo by generating mice carrying a knockin mutation (KK213AA) that abrogates its kinase activity.  IRAK-4KN/KN mice were highly resistant to TLR-induced shock response.  The cytokine prodn. in response to TLR ligands was severely impaired in IRAK-4KN/KN as well as IRAK-4-/- macrophages.  The IRAK-4 activity was essential for the activation of signaling pathways leading to mitogen-activated protein kinases.  TLR-induced IRAK-4/IRAK-1-dependent and -independent pathways were involved in early induction of NF-κB-regulated genes in response to TLR ligands such as tumor necrosis factor α and IκBζ.  In contrast to a previous paper, the TCR signaling was not impaired in IRAK-4-/- and IRAK-4KN/KN mice.  Thus, the kinase activity of IRAK-4 is essential for the regulation of TLR-mediated innate immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQDdu13L0FMrVg90H21EOLACvtfcHk0lhZQZckMQkZHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFSjsbY%253D&md5=902905464ef0cba41a3fda070fc982f5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1084%2Fjem.20061523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20061523%26sid%3Dliteratum%253Aachs%26aulast%3DKawagoe%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DMatsui%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DUematsu%26aufirst%3DS.%26aulast%3DTakeuchi%26aufirst%3DO.%26aulast%3DAkira%26aufirst%3DS.%26atitle%3DEssential%2520role%2520of%2520IRAK-4%2520protein%2520and%2520its%2520kinase%2520activity%2520in%2520Toll-like%2520receptor-mediated%2520immune%2520responses%2520but%2520not%2520in%2520TCR%2520signaling%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2007%26volume%3D204%26spage%3D1013%26epage%3D24%26doi%3D10.1084%2Fjem.20061523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leitinger, B.</span></span> <span> </span><span class="NLM_article-title">Discoidin domain receptor functions in physiological and pathological conditions</span>. <i>Int. Rev. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>310</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-800180-6.00002-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2FB978-0-12-800180-6.00002-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=24725424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlymsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2014&pages=39-87&author=B.+Leitinger&title=Discoidin+domain+receptor+functions+in+physiological+and+pathological+conditions&doi=10.1016%2FB978-0-12-800180-6.00002-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discoidin domain receptor functions in physiological and pathological conditions</span></div><div class="casAuthors">Leitinger, Birgit</div><div class="citationInfo"><span class="NLM_cas:title">International Review of Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">310</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-87</span>CODEN:
                <span class="NLM_cas:coden">IRCMAH</span>;
        ISSN:<span class="NLM_cas:issn">1937-6448</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The discoidin domain receptors, DDR1 and DDR2, are nonintegrin collagen receptors that are members of the receptor tyrosine kinase family.  Both DDRs bind a no. of different collagen types and play important roles in embryo development.  Dysregulated DDR function is assocd. with progression of various human diseases, including fibrosis, arthritis, and cancer.  By interacting with key components of the extracellular matrix and displaying distinct activation kinetics, the DDRs form a unique subfamily of receptor tyrosine kinases.  DDR-facilitated cellular functions include cell migration, cell survival, proliferation, and differentiation, as well as remodeling of extracellular matrixes.  This review summarizes the current knowledge of DDR-ligand interactions, DDR-initiated signal pathways and the mol. mechanisms that regulate receptor function.  Also discussed are the roles of DDRs in development and disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaBb00Ee5sVrVg90H21EOLACvtfcHk0lhZQZckMQkZHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlymsb7M&md5=19c591ee1051c6bee5b55b8f55d61877</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-800180-6.00002-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-800180-6.00002-5%26sid%3Dliteratum%253Aachs%26aulast%3DLeitinger%26aufirst%3DB.%26atitle%3DDiscoidin%2520domain%2520receptor%2520functions%2520in%2520physiological%2520and%2520pathological%2520conditions%26jtitle%3DInt.%2520Rev.%2520Cell%2520Mol.%2520Biol.%26date%3D2014%26volume%3D310%26spage%3D39%26epage%3D87%26doi%3D10.1016%2FB978-0-12-800180-6.00002-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuyama, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, T.</span></span> <span> </span><span class="NLM_article-title">Activation of discoidin domain receptor 1 isoform b with collagen up-regulates chemokine production in human macrophages: role of p38 mitogen-activated protein kinase and NF-kappa B</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">2332</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.172.4.2332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4049%2Fjimmunol.172.4.2332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=14764702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslWltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2004&pages=2332-40&author=W.+Matsuyamaauthor=L.+Wangauthor=W.+L.+Farrarauthor=M.+Faureauthor=T.+Yoshimura&title=Activation+of+discoidin+domain+receptor+1+isoform+b+with+collagen+up-regulates+chemokine+production+in+human+macrophages%3A+role+of+p38+mitogen-activated+protein+kinase+and+NF-kappa+B&doi=10.4049%2Fjimmunol.172.4.2332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Discoidin Domain Receptor 1 Isoform b with Collagen Up-Regulates Chemokine Production in Human Macrophages: Role of p38 Mitogen-Activated Protein Kinase and NF-κB</span></div><div class="casAuthors">Matsuyama, Wataru; Wang, Lihua; Farrar, William L.; Faure, Michel; Yoshimura, Teizo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2332-2340</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Macrophages produce an array of proinflammatory mediators at sites of inflammation and contribute to the development of inflammatory responses.  Important roles for cytokines, such as IL-1 or TNF-α, and bacterial products, such as LPS, in this process have been well documented; however, the role for the extracellular matrix proteins, such as collagen, remains unclear.  The authors previously reported that discoidin domain receptor 1 (DDR1), a nonintegrin collagen receptor, is expressed during differentiation of human monocytes into macrophages, and the interaction of the DDR1b isoform with collagen facilitates their differentiation via the p38 mitogen-activated protein kinase (MAPK) pathway.  In this study, the authors report that the interaction of DDR1b with collagen up-regulates the prodn. of IL-8, macrophage inflammatory protein-1α, and monocyte chemoattractant protein-1 in human macrophages in a p38 MAPK- and NF-κB-dependent manner. p38 MAPK was crit. for DDR1b-mediated, increased NF-κB trans-activity, but not for IκB degrdn. or NF-κB nuclear translocation, suggesting a role for p38 MAPK in the modification of NF-κB.  P38 MAPK was crit. for DDR1b-mediated, increased NF-κB trans-activity, but not for IκB degrdn. or NF-κB nuclear translocation, suggesting a role for p38 MAPK in the modification of NF-κB.  DDR1b-mediated IκB degrdn. was mediated through the recruitment of the adaptor protein Shc to the LXNPXY motif of the receptor and the downstream TNFR-assocd. factor 6/NF-κB activator 1 signaling cascade.  Taken together, the authors' study has identified NF-κB as a novel target of DDR1b signaling and provided a novel mechanism by which tissue-infiltrating macrophages produce large amts. of chemokines during the development of inflammatory diseases.  Intervention of DDR1b signaling may be useful to control inflammatory diseases in which these proteins play an important role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0dml00RSS7Vg90H21EOLACvtfcHk0liRCiB1TpcWwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslWltQ%253D%253D&md5=01893aefd9759bd412b6f3907bc13176</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.172.4.2332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.172.4.2332%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuyama%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFarrar%26aufirst%3DW.%2BL.%26aulast%3DFaure%26aufirst%3DM.%26aulast%3DYoshimura%26aufirst%3DT.%26atitle%3DActivation%2520of%2520discoidin%2520domain%2520receptor%25201%2520isoform%2520b%2520with%2520collagen%2520up-regulates%2520chemokine%2520production%2520in%2520human%2520macrophages%253A%2520role%2520of%2520p38%2520mitogen-activated%2520protein%2520kinase%2520and%2520NF-kappa%2520B%26jtitle%3DJ.%2520Immunol.%26date%3D2004%26volume%3D172%26spage%3D2332%26epage%3D40%26doi%3D10.4049%2Fjimmunol.172.4.2332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zioncheck, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vos, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span> <span> </span><span class="NLM_article-title">KDR (VEGF receptor 2) is the major mediator for the hypotensive effect of VEGF</span>. <i>Hypertension</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1161/01.HYP.0000018588.56950.7A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1161%2F01.HYP.0000018588.56950.7A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=12052848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFWgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2002&pages=1095-1100&author=B.+Liauthor=A.+K.+Ogasawaraauthor=R.+Yangauthor=W.+Weiauthor=G.+W.+Heauthor=T.+F.+Zioncheckauthor=S.+Buntingauthor=A.+M.+de+Vosauthor=H.+Jin&title=KDR+%28VEGF+receptor+2%29+is+the+major+mediator+for+the+hypotensive+effect+of+VEGF&doi=10.1161%2F01.HYP.0000018588.56950.7A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">KDR (VEGF receptor 2) is the major mediator for the hypotensive effect of VEGF</span></div><div class="casAuthors">Li, Bing; Ogasawara, Annie K.; Yang, Renhui; Wei, Wei; He, Guo-Wei; Zioncheck, Thomas F.; Bunting, Stuart; De Vos, Abraham M.; Jin, Hongkui</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1095-1100</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) exerts vasodilation-induced hypotension as a major side effect for treatment of ischemic diseases.  VEGF has 2 receptor tyrosine kinases, KDR and Flt-1.  Little is known about which receptor mediates VEGF-induced hypotension.  To elucidate the role of each receptor in mediating hypotension, KDR-selective and Flt-1-selective mutants were used for in vitro and in vivo studies.  The KDR-selective mutant induced vascular endothelial cell proliferation comparable to VEGF, whereas the Flt-1-selective mutant had no effect on proliferation.  I.v. injection of KDR-selective mutant, Flt-selective mutant, or VEGF caused a dose-related decrease in mean arterial pressure in conscious rats.  The hypotensive response to KDR-selective mutant was significantly less than that to VEGF (P<0.01) but was greater than that to Flt-selective mutant (P<0.01).  Similarly, VEGF and KDR-selective mutant induced more potent vasorelaxation than Flt-selective mutant or placenta growth factor that binds Flt-1 only (P<0.01), and the vasorelaxation to KDR-selective mutant was not significantly different at low concns. but less than that to VEGF at high concns.  The results indicate that the vasodilation and hypotensive effect of VEGF may involve both receptors, but KDR is the predominant receptor mediating this effect.  Because KDR-selective mutant induced proliferation and angiogenesis similar to VEGF but was assocd. with 36% attenuation in hypotension, the data suggest that the KDR-selective mutant may represent an alternative treatment for ischemic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4rC3aPHNaC7Vg90H21EOLACvtfcHk0liRCiB1TpcWwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFWgt7w%253D&md5=13dbdf4b1b0c38908aeb211b9713c305</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1161%2F01.HYP.0000018588.56950.7A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.HYP.0000018588.56950.7A%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DOgasawara%26aufirst%3DA.%2BK.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DG.%2BW.%26aulast%3DZioncheck%26aufirst%3DT.%2BF.%26aulast%3DBunting%26aufirst%3DS.%26aulast%3Dde%2BVos%26aufirst%3DA.%2BM.%26aulast%3DJin%26aufirst%3DH.%26atitle%3DKDR%2520%2528VEGF%2520receptor%25202%2529%2520is%2520the%2520major%2520mediator%2520for%2520the%2520hypotensive%2520effect%2520of%2520VEGF%26jtitle%3DHypertension%26date%3D2002%26volume%3D39%26spage%3D1095%26epage%3D1100%26doi%3D10.1161%2F01.HYP.0000018588.56950.7A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramaiah, L.</span>; <span class="NLM_string-name">Bounous, D. I.</span>; <span class="NLM_string-name">Elmore, S. A.</span></span> <span> </span><span class="NLM_article-title">Hematopoietic System</span>. In  <i>Fundamentals of Toxicologic Pathology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallig, M. A.</span>, <span class="NLM_string-name">Haschek, M. W.</span>, <span class="NLM_string-name">Rousseau, C. G.</span>, <span class="NLM_string-name">Bolon, B.</span>, <span class="NLM_string-name">Mahler, B. W.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press, Elsevier Inc.</span>: <span class="NLM_publisher-loc">London</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2FB978-0-12-809841-7.00013-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Ramaiah&author=D.+I.+Bounous&author=S.+A.+Elmoreauthor=M.+A.+Wallig&author=M.+W.+Haschek&author=C.+G.+Rousseau&author=B.+Bolon&author=B.+W.+Mahler&title=Fundamentals+of+Toxicologic+Pathology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-809841-7.00013-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-809841-7.00013-7%26sid%3Dliteratum%253Aachs%26aulast%3DRamaiah%26aufirst%3DL.%26atitle%3DHematopoietic%2520System%26btitle%3DFundamentals%2520of%2520Toxicologic%2520Pathology%26aulast%3DWallig%26aufirst%3DM.%2BA.%26pub%3DAcademic%2520Press%252C%2520Elsevier%2520Inc%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: a selection system based on endowing organic compounds with amplifiable information</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-062917-012550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1146%2Fannurev-biochem-062917-012550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=29328784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=479-502&author=D.+Neriauthor=R.+A.+Lerner&title=DNA-encoded+chemical+libraries%3A+a+selection+system+based+on+endowing+organic+compounds+with+amplifiable+information&doi=10.1146%2Fannurev-biochem-062917-012550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: A Selection System Based on Endowing Organic Compounds with Amplifiable Information</span></div><div class="casAuthors">Neri, Dario; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">479-502</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The discovery of org. ligands that bind specifically to proteins is a central problem in chem., biol., and the biomedical sciences.  The encoding of individual org. mols. with distinctive DNA tags, serving as amplifiable identification bar codes, allows the construction and screening of combinatorial libraries of unprecedented size, thus facilitating the discovery of ligands to many different protein targets.  Fundamentally, one links powers of genetics and chem. synthesis.  After the initial description of DNA-encoded chem. libraries in 1992, several exptl. embodiments of the technol. have been reduced to practice.  This review provides a historical account of important milestones in the development of DNA-encoded chem. libraries, a survey of relevant ongoing research activities, and a glimpse into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAjUib3HfEbVg90H21EOLACvtfcHk0lgS6eBgH_ojqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D&md5=559fefad70a7b9949319ac8ad8066004</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-062917-012550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-062917-012550%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520a%2520selection%2520system%2520based%2520on%2520endowing%2520organic%2520compounds%2520with%2520amplifiable%2520information%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D479%26epage%3D502%26doi%3D10.1146%2Fannurev-biochem-062917-012550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span> <span> </span><span class="NLM_article-title">What do you get from DNA-encoded libraries?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=408-410&author=A.+L.+Satz&title=What+do+you+get+from+DNA-encoded+libraries%3F&doi=10.1021%2Facsmedchemlett.8b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">What Do You Get from DNA-Encoded Libraries?</span></div><div class="casAuthors">Satz, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">408-410</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Use of DNA-encoded libraries (DELs) in the pharmaceutical industry has rapidly increased.  We discuss what to expect when you run a DEL screen and contemplate guidelines for library design.  Addnl., we consider some visionary work and extrapolate to the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqODDlBH-pnArVg90H21EOLACvtfcHk0lgS6eBgH_ojqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslKmt70%253D&md5=0d2d28d00011533ffafd57d0308a1969</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00128%26sid%3Dliteratum%253Aachs%26aulast%3DSatz%26aufirst%3DA.%2BL.%26atitle%3DWhat%2520do%2520you%2520get%2520from%2520DNA-encoded%2520libraries%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D408%26epage%3D410%26doi%3D10.1021%2Facsmedchemlett.8b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel autotaxin inhibitor for the treatment of idiopathic pulmonary fibrosis: a clinical candidate discovered using DNA-encoded chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+autotaxin+inhibitor+for+the+treatment+of+idiopathic+pulmonary+fibrosis%3A+a+clinical+candidate+discovered+using+DNA-encoded+chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0lieYQHi5zy_wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520autotaxin%2520inhibitor%2520for%2520the%2520treatment%2520of%2520idiopathic%2520pulmonary%2520fibrosis%253A%2520a%2520clinical%2520candidate%2520discovered%2520using%2520DNA-encoded%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rho, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span> <span> </span><span class="NLM_article-title">MET and AXL inhibitor NPS-1034 exerts efficacy against lung cancer cells resistant to EGFR kinase inhibitors because of MET or AXL activation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1158%2F0008-5472.CAN-13-1103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=24165158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislKjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=253-262&author=J.+K.+Rhoauthor=Y.+J.+Choiauthor=S.+Y.+Kimauthor=T.+W.+Kimauthor=E.+K.+Choiauthor=S.+J.+Yoonauthor=B.+M.+Parkauthor=E.+Parkauthor=J.+H.+Baeauthor=C.+M.+Choiauthor=J.+C.+Lee&title=MET+and+AXL+inhibitor+NPS-1034+exerts+efficacy+against+lung+cancer+cells+resistant+to+EGFR+kinase+inhibitors+because+of+MET+or+AXL+activation&doi=10.1158%2F0008-5472.CAN-13-1103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">MET and AXL Inhibitor NPS-1034 Exerts Efficacy against Lung Cancer Cells Resistant to EGFR Kinase Inhibitors Because of MET or AXL Activation</span></div><div class="casAuthors">Rho, Jin Kyung; Choi, Yun Jung; Kim, Seon Ye; Kim, Tae Won; Choi, Eun Kyung; Yoon, Seon-Joo; Park, Bu Man; Park, Eunhye; Bae, Jong Hwan; Choi, Chang-Min; Lee, Jae Cheol</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-262</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In non-small cell lung cancer (NSCLC) with epidermal growth factor receptor (EGFR) mutations, acquired resistance to EGFR-tyrosine kinase inhibitors (EGFR-TK1) can occur through a generation of bypass signals such as MET or AXL activation.  In this study, we investigated the antitumor activity of NPS-1034, a newly developed drug that targets both MET and AXL, in NSCLC cells with acquired resistance to gefitinib or erlotinib (HCC827/GR and HCC827/ER, resp.).  Characterization of H820 cells and evaluation of NPS-1034 efficacy in these cells were also performed.  The resistance of HCC827/GR was mediated by MET activation, whereas AXL activation led to resistance in HCC827/ER.  The combination of gefitinib or erlotinib with NPS-1034 synergistically inhibited cell proliferation and induced cell death in both resistant cell lines.  Accordingly, suppression of Akt was noted only in the presence of treatment with both drugs.  NPS-1034 was also effective in xenograft mouse models of HCC827/GR.  Although the H820 cell line was reported previously to have T790M and MET amplification, we discovered that AXL was also activated in this cell line.  There were no antitumor effects of siRNA or inhibitors specific for EGFR or MET, whereas combined treatment with AXL siRNA or NPS-1034 and EGFR-TKIs controlled H820 cells, suggesting that AXL is the main signal responsible for resistance.  In addn., NPS-1034 inhibited cell proliferation as well as ROS1 activity in HCC78 cells with ROS1 rearrangement.  Our results establish the efficacy of NPS-1034 in NSCLC cells rendered resistant to EGFR-TKIs because of MET or AXL activation or ROS1 rearrangement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzVRl7xbUBLrVg90H21EOLACvtfcHk0lgf0k5DWvwbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislKjtA%253D%253D&md5=35929f2dcfd5d1502ba745f367dccd27</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1103%26sid%3Dliteratum%253Aachs%26aulast%3DRho%26aufirst%3DJ.%2BK.%26aulast%3DChoi%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DKim%26aufirst%3DT.%2BW.%26aulast%3DChoi%26aufirst%3DE.%2BK.%26aulast%3DYoon%26aufirst%3DS.%2BJ.%26aulast%3DPark%26aufirst%3DB.%2BM.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DBae%26aufirst%3DJ.%2BH.%26aulast%3DChoi%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26atitle%3DMET%2520and%2520AXL%2520inhibitor%2520NPS-1034%2520exerts%2520efficacy%2520against%2520lung%2520cancer%2520cells%2520resistant%2520to%2520EGFR%2520kinase%2520inhibitors%2520because%2520of%2520MET%2520or%2520AXL%2520activation%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D253%26epage%3D262%26doi%3D10.1158%2F0008-5472.CAN-13-1103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgf0k5DWvwbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lgf0k5DWvwbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenjans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W.</span></span> <span> </span><span class="NLM_article-title">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2015.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.cbpa.2015.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=25909818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=104-110&author=M.+J.+Wigglesworthauthor=D.+C.+Murrayauthor=C.+J.+Blackettauthor=M.+Kossenjansauthor=J.+W.+Nissink&title=Increasing+the+delivery+of+next+generation+therapeutics+from+high+throughput+screening+libraries&doi=10.1016%2Fj.cbpa.2015.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span></div><div class="casAuthors">Wigglesworth, Mark J.; Murray, David C.; Blackett, Carolyn J.; Kossenjans, Michael; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-110</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry has historically relied on high throughput screening as a cornerstone to identify chem. equity for drug discovery projects.  However, with pharmaceutical companies moving through a phase of diminished returns and alternative hit identification strategies proving successful, it is more important than ever to understand how this approach can be used more effectively to increase the delivery of next generation therapeutics from high throughput screening libraries.  There is a wide literature that describes HTS and fragment based screening approaches which offer clear direction on the process for these two distinct activities.  However, few people have considered how best to identify medium to low mol. wt. compds. from large diversity screening sets and increase downstream success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFeNO1FIRBLVg90H21EOLACvtfcHk0ljd1r3gu8k1CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D&md5=4b4bd0e7992796bd208a99dcb099c785</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2015.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DD.%2BC.%26aulast%3DBlackett%26aufirst%3DC.%2BJ.%26aulast%3DKossenjans%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26atitle%3DIncreasing%2520the%2520delivery%2520of%2520next%2520generation%2520therapeutics%2520from%2520high%2520throughput%2520screening%2520libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D26%26spage%3D104%26epage%3D110%26doi%3D10.1016%2Fj.cbpa.2015.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+frequent-hitter+behavior+based+on+historical+high-throughput+screening+data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0ljd1r3gu8k1CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520frequent-hitter%2520behavior%2520based%2520on%2520historical%2520high-throughput%2520screening%2520data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wernevik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergstrom, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noven, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulthe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmgren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stromstedt, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehnstrom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbock, T.</span></span> <span> </span><span class="NLM_article-title">A fully integrated assay panel for early drug metabolism and pharmacokinetics profiling</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1089/adt.2020.970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1089%2Fadt.2020.970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=32407132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=157-179&author=J.+Wernevikauthor=F.+Bergstromauthor=A.+Novenauthor=J.+Hultheauthor=L.+Fredlundauthor=D.+Addisonauthor=J.+Holmgrenauthor=P.+E.+Stromstedtauthor=E.+Rehnstromauthor=T.+Lundbock&title=A+fully+integrated+assay+panel+for+early+drug+metabolism+and+pharmacokinetics+profiling&doi=10.1089%2Fadt.2020.970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Fully Integrated Assay Panel for Early Drug Metabolism and Pharmacokinetics Profiling</span></div><div class="casAuthors">Wernevik, Johan; Bergstroem, Fredrik; Noven, Anna; Hulthe, Johan; Fredlund, Linda; Addison, Dan; Holmgren, Jan; Stroemstedt, Per-Erik; Rehnstroem, Erika; Lundboeck, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">157-179</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Evaluation and optimization of physicochem. and metabolic properties of compds. are a crucial component of the drug development process.  Continuous access to this information during the design-make-test-anal. cycle enables identification of chem. entities with suitable properties for efficient project progression.  In this study, we describe an integrated and automated assay panel (DMPK Wave 1) that informs weekly on lipophilicity, soly., human plasma protein binding, and metabolic stability in rat hepatocytes and human liver microsomes.  All assays are running in 96-well format with ultraperformance liq. chromatog.-mass spectrometry (MS)/MS as read-out.  A streamlined overall workflow has been developed by optimizing all parts of the process, including shipping of compds. between sites, use of fit-for-purpose equipment and information systems, and technol. for compd. requesting, data anal., and reporting.  As a result, lead times can be achieved that well match project demands across sites independently of where compds. are synthesized.  This robust screening strategy is run on a weekly basis and enables optimization of structure-activity relationships in parallel with DMPK properties to allow efficient and informed decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr61vWtfynwTrVg90H21EOLACvtfcHk0liKozn6fYBmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D&md5=9b19876b26b8ddfb7aad414f5cf9c0a1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1089%2Fadt.2020.970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2020.970%26sid%3Dliteratum%253Aachs%26aulast%3DWernevik%26aufirst%3DJ.%26aulast%3DBergstrom%26aufirst%3DF.%26aulast%3DNoven%26aufirst%3DA.%26aulast%3DHulthe%26aufirst%3DJ.%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DAddison%26aufirst%3DD.%26aulast%3DHolmgren%26aufirst%3DJ.%26aulast%3DStromstedt%26aufirst%3DP.%2BE.%26aulast%3DRehnstrom%26aufirst%3DE.%26aulast%3DLundbock%26aufirst%3DT.%26atitle%3DA%2520fully%2520integrated%2520assay%2520panel%2520for%2520early%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520profiling%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2020%26volume%3D18%26spage%3D157%26epage%3D179%26doi%3D10.1089%2Fadt.2020.970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Compound set enrichment: a novel approach to analysis of primary HTS data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2067</span>– <span class="NLM_lpage">2078</span>, <span class="refDoi"> DOI: 10.1021/ci100203e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100203e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=2067-2078&author=T.+Varinauthor=H.+Gublerauthor=C.+N.+Parkerauthor=J.+H.+Zhangauthor=P.+Ramanauthor=P.+Ertlauthor=A.+Schuffenhauer&title=Compound+set+enrichment%3A+a+novel+approach+to+analysis+of+primary+HTS+data&doi=10.1021%2Fci100203e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Compound Set Enrichment: A Novel Approach to Analysis of Primary HTS Data</span></div><div class="casAuthors">Varin, Thibault; Gubler, Hanspeter; Parker, Christian N.; Zhang, Ji-Hu; Raman, Pichai; Ertl, Peter; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2067-2078</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main goal of high-throughput screening (HTS) is to identify active chem. series rather than just individual active compds.  In light of this goal, a new method (called compd. set enrichment) to identify active chem. series from primary screening data is proposed.  The method employs the scaffold tree compd. classification in conjunction with the Kolmogorov-Smirnov statistic to assess the overall activity of a compd. scaffold.  The application of this method to seven PubChem data sets (contg. between 9389 and 263679 mols.) is presented, and the ability of this method to identify compd. classes with only weakly active compds. (potentially latent hits) is demonstrated.  The anal. presented here shows how methods based on an activity cutoff can distort activity information, leading to the incorrect activity assignment of compd. series.  These results suggest that this method might have utility in the rational selection of active classes of compds. (and not just individual active compds.) for followup and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlFSGm_wSXybVg90H21EOLACvtfcHk0liKozn6fYBmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbbN&md5=151077ce174ceaed80e3708e960a57e4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci100203e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100203e%26sid%3Dliteratum%253Aachs%26aulast%3DVarin%26aufirst%3DT.%26aulast%3DGubler%26aufirst%3DH.%26aulast%3DParker%26aufirst%3DC.%2BN.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DCompound%2520set%2520enrichment%253A%2520a%2520novel%2520approach%2520to%2520analysis%2520of%2520primary%2520HTS%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D2067%26epage%3D2078%26doi%3D10.1021%2Fci100203e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimpl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span> <span> </span><span class="NLM_article-title">A-loop interactions in Mer tyrosine kinase give rise to inhibitors with two-step mechanism and long residence time of binding</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">4443</span>– <span class="NLM_lpage">4452</span>, <span class="refDoi"> DOI: 10.1042/BCJ20200735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1042%2FBCJ20200735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=33119085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1amtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2020&pages=4443-4452&author=A.+Pflugauthor=M.+Schimplauthor=J.+W.+M.+Nissinkauthor=R.+C.+Overmanauthor=P.+B.+Rawlinsauthor=C.+Trumanauthor=E.+Underwoodauthor=J.+Warwickerauthor=J.+J.+Winter-Holtauthor=W.+McCoull&title=A-loop+interactions+in+Mer+tyrosine+kinase+give+rise+to+inhibitors+with+two-step+mechanism+and+long+residence+time+of+binding&doi=10.1042%2FBCJ20200735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A-loop interactions in Mer tyrosine kinase give rise to inhibitors with two-step mechanism and long residence time of binding</span></div><div class="casAuthors">Pflug, Alexander; Schimpl, Marianne; Nissink, J. Willem M.; Overman, Ross C.; Rawlins, Philip B.; Truman, Caroline; Underwood, Elizabeth; Warwicker, Juli; Winter-Holt, Jon; McCoull, William</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4443-4452</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The activation loop (A-loop) plays a key role in regulating the catalytic activity of protein kinases.  Phosphorylation in this region enhances the phosphoryl transfer rate of the kinase domain and increases its affinity for ATP.  Furthermore, the A-loop possesses autoinhibitory functions in some kinases, where it collapses onto the protein surface and blocks substrate binding when unphosphorylated.  Due to its flexible nature, the A-loop is usually disordered and untraceable in kinase domain crystal structures.  The resulting lack of structural information is regrettable as it impedes the design of drug A-loop contacts, which have proven favorable in multiple cases.  Here, we characterize the binding with A-loop engagement between type 1.5 kinase inhibitor 'example 172' (EX172) and Mer tyrosine kinase (MerTK).  With the help of crystal structures and binding kinetics, we portray how the recruitment of the A-loop elicits a two-step binding mechanism which results in a drug-target complex characterized by high affinity and long residence time.  In addn., the type 1.5 compd. possesses excellent kinome selectivity and a remarkable preference for the phosphorylated over the dephosphorylated form of MerTK.  We discuss these unique characteristics in the context of known type 1 and type 2 inhibitors and highlight opportunities for future kinase inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDfeb8S10e-bVg90H21EOLACvtfcHk0liKozn6fYBmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1amtLvF&md5=e92a372506add30cfe2b3da8a9e2ded1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBCJ20200735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20200735%26sid%3Dliteratum%253Aachs%26aulast%3DPflug%26aufirst%3DA.%26aulast%3DSchimpl%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DOverman%26aufirst%3DR.%2BC.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%2BJ.%26aulast%3DMcCoull%26aufirst%3DW.%26atitle%3DA-loop%2520interactions%2520in%2520Mer%2520tyrosine%2520kinase%2520give%2520rise%2520to%2520inhibitors%2520with%2520two-step%2520mechanism%2520and%2520long%2520residence%2520time%2520of%2520binding%26jtitle%3DBiochem.%2520J.%26date%3D2020%26volume%3D477%26spage%3D4443%26epage%3D4452%26doi%3D10.1042%2FBCJ20200735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span>; <span class="NLM_string-name">Wilkinson, S. R.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Hall, B. S.</span></span> <i>Compounds, Compositions and Methods for Treatment of Protozoan Infections</i>. International Patent Application <span class="NLM_patent">WO2011066416</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Hu&author=S.+R.+Wilkinson&author=X.+Wu&author=B.+S.+Hall&title=Compounds%2C+Compositions+and+Methods+for+Treatment+of+Protozoan+Infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26jtitle%3DCompounds%252C%2520Compositions%2520and%2520Methods%2520for%2520Treatment%2520of%2520Protozoan%2520Infections%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaten, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swoboda, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inducer of beta cell proliferation identified by high-throughput screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">1669</span>– <span class="NLM_lpage">1672</span>, <span class="refDoi"> DOI: 10.1021/ja309304m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja309304m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=1669-1672&author=W.+Shenauthor=M.+S.+Tremblayauthor=V.+A.+Deshmukhauthor=W.+Wangauthor=C.+M.+Filippiauthor=G.+Harbauthor=Y.+Q.+Zhangauthor=A.+Kamireddyauthor=J.+E.+Baatenauthor=Q.+Jinauthor=T.+Wuauthor=J.+G.+Swobodaauthor=C.+Y.+Choauthor=J.+Liauthor=B.+A.+Laffitteauthor=P.+McNamaraauthor=R.+Glynneauthor=X.+Wuauthor=A.+E.+Hermanauthor=P.+G.+Schultz&title=Small-molecule+inducer+of+beta+cell+proliferation+identified+by+high-throughput+screening&doi=10.1021%2Fja309304m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inducer of β Cell Proliferation Identified by High-Throughput Screening</span></div><div class="casAuthors">Shen, Weijun; Tremblay, Matthew S.; Deshmukh, Vishal A.; Wang, Weidong; Filippi, Christophe M.; Harb, George; Zhang, You-qing; Kamireddy, Anwesh; Baaten, Janine E.; Jin, Qihui; Wu, Tom; Swoboda, Jonathan G.; Peters, Eric C.; Cho, Charles Y.; Li, Jing; Laffitte, Bryan A.; McNamara, Peter; Glynne, Richard; Wu, Xu; Herman, Ann E.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1669-1672</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of factors that promote β cell proliferation could ultimately move type 1 diabetes treatment away from insulin injection therapy and toward a cure.  We have performed high-throughput, cell-based screens using rodent β cell lines to identify mols. that induce proliferation of β cells.  Herein we report the discovery and characterization of WS6, a novel small mol. that promotes β cell proliferation in rodent and human primary islets.  In the RIP-DTA mouse model of β cell ablation, WS6 normalized blood glucose and induced concomitant increases in β cell proliferation and β cell no.  Affinity pulldown and kinase profiling studies implicate Erb3 binding protein-1 and the IκB kinase pathway in the mechanism of action of WS6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcVnec647SN7Vg90H21EOLACvtfcHk0lgrP6_pRjEiEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCgu7w%253D&md5=f7a76c2f8b61e183e171b3343d65543a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja309304m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja309304m%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTremblay%26aufirst%3DM.%2BS.%26aulast%3DDeshmukh%26aufirst%3DV.%2BA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DBaaten%26aufirst%3DJ.%2BE.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DSwoboda%26aufirst%3DJ.%2BG.%26aulast%3DCho%26aufirst%3DC.%2BY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLaffitte%26aufirst%3DB.%2BA.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DSmall-molecule%2520inducer%2520of%2520beta%2520cell%2520proliferation%2520identified%2520by%2520high-throughput%2520screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D1669%26epage%3D1672%26doi%3D10.1021%2Fja309304m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armitage, I.</span>; <span class="NLM_string-name">Bourland, M. E.</span>; <span class="NLM_string-name">Boyle, C. J. S.</span>; <span class="NLM_string-name">Cooper, M. I.</span>; <span class="NLM_string-name">Ferdous, A. J.</span>; <span class="NLM_string-name">Langston, M.</span></span> International Patent Application <span class="NLM_patent">WO2010059239</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Armitage%2C+I.%3B+Bourland%2C+M.+E.%3B+Boyle%2C+C.+J.+S.%3B+Cooper%2C+M.+I.%3B+Ferdous%2C+A.+J.%3B+Langston%2C+M.+International+Patent+Application+WO2010059239%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DArmitage%26aufirst%3DI.%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoGrasso, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">Pyridopyrimidinone Derivatives as potent and selective c-Jun N-terminal kinase (JNK) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1021/ml500474d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500474d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=413-418&author=K.+Zhengauthor=C.+M.+Parkauthor=S.+Iqbalauthor=P.+Hernandezauthor=H.+Parkauthor=P.+V.+LoGrassoauthor=Y.+Feng&title=Pyridopyrimidinone+Derivatives+as+potent+and+selective+c-Jun+N-terminal+kinase+%28JNK%29+inhibitors&doi=10.1021%2Fml500474d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridopyrimidinone derivatives as potent and selective c-Jun N-terminal kinase (JNK) inhibitors</span></div><div class="casAuthors">Zheng, Ke; Park, Chul Min; Iqbal, Sarah; Hernandez, Pamela; Park, HaJeung; LoGrasso, Philip V.; Feng, Yangbo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-418</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of 2-aminopyridopyrimidinone based JNK (c-jun N-terminal kinase) inhibitors were discovered and developed.  Structure-activity relationships (SARs) were systematically developed utilizing biochem. and cell based assays and in vitro and in vivo drug metab. and pharmacokinetic (DMPK) studies.  Through the optimization of lead compd. 1, several potent and selective JNK inhibitors with high oral bioavailability were developed.  (trans)-1-Isopropyl-3-[4-(8-isopropyl-7-oxo-7,8-dihydro-pyrido[2,3-d]pyrimidin-2-yl-amino)-cyclohexyl]urea (I), was a potent JNK3 inhibitor (IC50 = 15 nM), had high selectivity against p38 (IC50 > 10 μM), had high potency in functional cell based assays, and had high stability in human liver microsome (t1/2 = 76 min), a clean CYP-450 inhibition profile, and excellent oral bioavailability (%F = 87).  Moreover, cocrystal structures of (I) and (trans)-1-[4-(8-Cyclopentyl-7-oxo-7,8-dihydro-pyrido[2,3-d]pyrimidin-2-ylamino)-cyclohexyl]-3-isopropyl-urea in JNK3 were solved at 2.0 Å.  These structures elucidated the binding mode (Type-I binding) and can pave the way for further inhibitor design of this pyridopyrimidinone scaffold for JNK inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-GYGot2Yf7bVg90H21EOLACvtfcHk0lj72d-WIg_uvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFCgurc%253D&md5=7bd22deaf4386f2ff02adafba62056cf</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fml500474d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500474d%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DC.%2BM.%26aulast%3DIqbal%26aufirst%3DS.%26aulast%3DHernandez%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLoGrasso%26aufirst%3DP.%2BV.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DPyridopyrimidinone%2520Derivatives%2520as%2520potent%2520and%2520selective%2520c-Jun%2520N-terminal%2520kinase%2520%2528JNK%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D413%26epage%3D418%26doi%3D10.1021%2Fml500474d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F.</span>; <span class="NLM_string-name">Goldstein, D. M.</span>; <span class="NLM_string-name">Han, X.</span>; <span class="NLM_string-name">Park, J.</span>; <span class="NLM_string-name">Sjogren, E. B.</span>; <span class="NLM_string-name">Reuter, D. C.</span>; <span class="NLM_string-name">Hogg, J. H.</span>; <span class="NLM_string-name">Gong, L.</span>; <span class="NLM_string-name">Dillon, M. P.</span>; <span class="NLM_string-name">Eustache, F.</span></span> <i>Pyrimidinyl pyridone inhibitors of kinases</i>. U.S. Patent <span class="NLM_patent">US2009/270389A1</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=F.+Brookfield&author=D.+M.+Goldstein&author=X.+Han&author=J.+Park&author=E.+B.+Sjogren&author=D.+C.+Reuter&author=J.+H.+Hogg&author=L.+Gong&author=M.+P.+Dillon&author=F.+Eustache&title=Pyrimidinyl+pyridone+inhibitors+of+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrookfield%26aufirst%3DF.%26jtitle%3DPyrimidinyl%2520pyridone%2520inhibitors%2520of%2520kinases%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Esther C. Y. Lee, Andrew J. McRiner, Katy E. Georgiadis, Julie Liu, Zooey Wang, Andrew D. Ferguson, Benjamin Levin, Moritz von Rechenberg, Christopher D. Hupp, Michael I. Monteiro, Anthony D. Keefe, Allison Olszewski, Charles J. Eyermann, Paolo Centrella, Yanbin Liu, Shilpi Arora, John W. Cuozzo, Ying Zhang, Matthew A. Clark, Christelle Huguet, <span class="NLM_string-name hlFld-ContribAuthor">Anna Kohlmann</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (10)
                                     , 6730-6744. <a href="https://doi.org/10.1021/acs.jmedchem.0c02271" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02271%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%25252C%252BPotent%252BInhibitors%252Bof%252BHydroxy%252BAcid%252BOxidase%252B1%252B%252528HAO1%252529%252BUsing%252BDNA-Encoded%252BChemical%252BLibrary%252BScreening%26aulast%3DLee%26aufirst%3DEsther%2BC.%2BY.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D31122020%26date%3D06052021%26volume%3D64%26issue%3D10%26spage%3D6730%26epage%3D6744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Y.  Zhang</span>, <span class="hlFld-ContribAuthor ">M.A.  Clark</span>. </span><span class="cited-content_cbyCitation_article-title">Design concepts for DNA-encoded library synthesis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>41 </em>, 116189. <a href="https://doi.org/10.1016/j.bmc.2021.116189" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116189%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Bconcepts%252Bfor%252BDNA-encoded%252Blibrary%252Bsynthesis%26aulast%3DZhang%26aufirst%3DY.%26date%3D2021%26volume%3D41%26spage%3D116189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Parallel lead-finding strategies for finding Mer inhibitor leads.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Early lead compounds. (b) Biochemical kinase assay inhibition data for data mining hit <b>2</b> (% inhibition at compound concentration of 1 μM) in biochemical binding assays for Mer and key off-targets. (c) Mer protein–ligand crystal structure of <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>). Phe742 is indicated and was found to be in the DFG Phe pocket in chain A, but out of the DFG pocket in chain B (latter shown here). (d) Binding mode of compound <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY">7AVY</a>) with DFG-in conformation highlighted as ball-and-stick. In the binding mode images, the edge of the αC helix can be seen horizontally at the top of the image. The kinase hinge loop sits bottom left, with hydrogen bonding contact shown as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cascade design for Mer hit identification. (a) Primary, secondary, and concentration–response screen setups. (b) Schematic representation of the pIC<sub>50</sub> range covered by the three screen concentrations. (c) Reduction of primary-screen hit set. SS single shot (measurement at a compound single concentration).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Binding mode of <b>5</b> in Mer (BAP series, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ">7AVZ</a>). (b) Binding mode of <b>9</b> in Mer (purine series, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0">7AW0</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of type-II series. Binding mode of <b>11</b> (in green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1">7AW1</a>), and DFG-out binding mode of <b>1</b> (in orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>) for comparison. The edge of the αC helix can be seen horizontally at the top of the image and the kinase hinge loop sits bottom left.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Schematic of library member. (b) Binding mode of azaindole <b>12</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Schematic of library member. (b) Binding mode of <b>13</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4">7AW4</a>); type-I azaindole <b>12</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>) is aligned to show the location of the ATP pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/medium/jm0c01904_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Schematic of library member (b) Binding mode of oxadiazole <b>14</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2">7AW2</a>). The kinase hinge is oriented bottom left, and the αC helix is visible horizontally, at the top.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c01904/20210319/images/large/jm0c01904_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01904&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11060" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11060" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myers, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amend, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Tyro3, Axl and MerTK (TAM receptors): implications for macrophages in the tumor microenvironment</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">94</span>, <span class="refDoi"> DOI: 10.1186/s12943-019-1022-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1186%2Fs12943-019-1022-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=31088471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A280%3ADC%252BB3M7lsleiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=94&author=K.+V.+Myersauthor=S.+R.+Amendauthor=K.+J.+Pienta&title=Targeting+Tyro3%2C+Axl+and+MerTK+%28TAM+receptors%29%3A+implications+for+macrophages+in+the+tumor+microenvironment&doi=10.1186%2Fs12943-019-1022-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Tyro3, Axl and MerTK (TAM receptors): implications for macrophages in the tumor microenvironment</span></div><div class="casAuthors">Myers Kayla V; Pienta Kenneth J; Myers Kayla V; Amend Sarah R; Pienta Kenneth J; Pienta Kenneth J; Pienta Kenneth J</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tumor-associated macrophages are an abundant cell type in the tumor microenvironment.  These macrophages serve as a promising target for treatment of cancer due to their roles in promoting cancer progression and simultaneous immunosuppression.  The TAM receptors (Tyro3, Axl and MerTK) are promising therapeutic targets on tumor-associated macrophages.  The TAM receptors are a family of receptor tyrosine kinases with shared ligands Gas6 and Protein S that skew macrophage polarization towards a pro-tumor M2-like phenotype.  In macrophages, the TAM receptors also promote apoptotic cell clearance, a tumor-promoting process called efferocytosis.  The TAM receptors bind the "eat-me" signal phosphatidylserine on apoptotic cell membranes using Gas6 and Protein S as bridging ligands.  Post-efferocytosis, macrophages are further polarized to a pro-tumor M2-like phenotype and secrete increased levels of immunosuppressive cytokines.  Since M2 polarization and efferocytosis are tumor-promoting processes, the TAM receptors on macrophages serve as exciting targets for cancer therapy.  Current TAM receptor-directed therapies in preclinical development and clinical trials may have anti-cancer effects though impacting macrophage phenotype and function in addition to the cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTmoOYd9tKrj6x48Rj5GnGofW6udTcc2eb3daTcsxjpFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M7lsleiuw%253D%253D&md5=98dcbcc47e29f0c18f2a52920900975c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12943-019-1022-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-019-1022-2%26sid%3Dliteratum%253Aachs%26aulast%3DMyers%26aufirst%3DK.%2BV.%26aulast%3DAmend%26aufirst%3DS.%2BR.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26atitle%3DTargeting%2520Tyro3%252C%2520Axl%2520and%2520MerTK%2520%2528TAM%2520receptors%2529%253A%2520implications%2520for%2520macrophages%2520in%2520the%2520tumor%2520microenvironment%26jtitle%3DMol.%2520Cancer%26date%3D2019%26volume%3D18%26spage%3D94%26doi%3D10.1186%2Fs12943-019-1022-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimaudo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pipitone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lori, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raggi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, F.</span></span> <span> </span><span class="NLM_article-title">Role of myeloid-epithelial-reproductive tyrosine kinase and macrophage polarization in the progression of atherosclerotic lesions associated with nonalcoholic fatty liver disease</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">604</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.3389%2Ffphar.2019.00604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=31191323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkt12it7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=604&author=M.+Pastoreauthor=S.+Grimaudoauthor=R.+M.+Pipitoneauthor=G.+Loriauthor=C.+Raggiauthor=S.+Pettaauthor=F.+Marra&title=Role+of+myeloid-epithelial-reproductive+tyrosine+kinase+and+macrophage+polarization+in+the+progression+of+atherosclerotic+lesions+associated+with+nonalcoholic+fatty+liver+disease&doi=10.3389%2Ffphar.2019.00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of myeloid-epithelial-reproductive tyrosine kinase and macrophage polarization in the progression of atherosclerotic lesions associated with nonalcoholic fatty liver disease</span></div><div class="casAuthors">Pastore, Mirella; Grimaudo, Stefania; Pipitone, Rosaria Maria; Lori, Giulia; Raggi, Chiara; Petta, Salvatore; Marra, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">604</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Recent lines of evidence highlight the involvement of myeloid-epithelial-reproductive tyrosine kinase (MerTK) in metabolic disease assocd. with liver damage.  MerTK is mainly expressed in anti-inflammatory M2 macrophages where it mediates transcriptional changes including suppression of proinflammatory cytokines and enhancement of inflammatory repressors.  MerTK is regulated by metabolic pathways through nuclear sensors including LXRs, PPARs, and RXRs, in response to apoptotic bodies or to other sources of cholesterol.  Nonalcoholic fatty liver disease (NAFLD) is one of the most serious public health problems worldwide.  It is a clinicopathol. syndrome closely related to obesity, insulin resistance, and oxidative stress.  It includes a spectrum of conditions ranging from simple steatosis, characterized by hepatic fat accumulation with or without inflammation, to nonalcoholic steatohepatitis (NASH), defined by hepatic fat deposition with hepatocellular damage, inflammation, and accumulating fibrosis.  Several studies support an assocn. between NAFLD and the incidence of cardiovascular diseases including atherosclerosis, a major cause of death worldwide.  This pathol. condition consists in a chronic and progressive inflammatory process in the intimal layer of large- and medium-sized arteries.  The complications of advanced atherosclerosis include chronic or acute ischemic damage in the tissue perfused by the affected artery, leading to cellular death.  By identifying specific targets influencing lipid metab. and cardiovascularrelated diseases, the present review highlights the role of MerTK in NAFLD-assocd. atherosclerotic lesions as a potential innovative therapeutic target.  Therapeutic advantages might derive from the use of compds. selective for nuclear receptors targeting PPARs rather than LXRs regulating macrophage lipid metab. and macrophage mediated inflammation, by favoring the expression of MerTK, which mediates an immunoregulatory action with a redn. in inflammation and in atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJbW7XUoajorVg90H21EOLACvtfcHk0ljx9kN0l6felQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkt12it7w%253D&md5=09f520d512ff856620b2f50220614496</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00604%26sid%3Dliteratum%253Aachs%26aulast%3DPastore%26aufirst%3DM.%26aulast%3DGrimaudo%26aufirst%3DS.%26aulast%3DPipitone%26aufirst%3DR.%2BM.%26aulast%3DLori%26aufirst%3DG.%26aulast%3DRaggi%26aufirst%3DC.%26aulast%3DPetta%26aufirst%3DS.%26aulast%3DMarra%26aufirst%3DF.%26atitle%3DRole%2520of%2520myeloid-epithelial-reproductive%2520tyrosine%2520kinase%2520and%2520macrophage%2520polarization%2520in%2520the%2520progression%2520of%2520atherosclerotic%2520lesions%2520associated%2520with%2520nonalcoholic%2520fatty%2520liver%2520disease%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D604%26doi%3D10.3389%2Ffphar.2019.00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. S.</span></span> <span> </span><span class="NLM_article-title">Efferocytosis Signaling in the regulation of macrophage inflammatory responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1394</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1601520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4049%2Fjimmunol.1601520" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=28167649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFWks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=1387-1394&author=M.+R.+Elliottauthor=K.+M.+Kosterauthor=P.+S.+Murphy&title=Efferocytosis+Signaling+in+the+regulation+of+macrophage+inflammatory+responses&doi=10.4049%2Fjimmunol.1601520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Efferocytosis Signaling in the Regulation of Macrophage Inflammatory Responses</span></div><div class="casAuthors">Elliott, Michael R.; Koster, Kyle M.; Murphy, Patrick S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1387-1394</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Since the pioneering work of Elie Metchnikoff and the discovery of cellular immunity, the phagocytic clearance of cellular debris has been considered an integral component of resolving inflammation and restoring function of damaged and infected tissues.  We now know that the phagocytic clearance of dying cells (efferocytosis), particularly by macrophages and other immune phagocytes, has profound consequences on innate and adaptive immune responses in inflamed tissues.  These immunomodulatory effects result from an array of mol. signaling events between macrophages, dying cells, and other tissue-resident cells.  In recent years, many of these mol. pathways have been identified and studied in the context of tissue inflammation, helping us better understand the relationship between efferocytosis and inflammation.  We review specific types of efferocytosis-related signals that can impact macrophage immune responses and discuss their relevance to inflammation-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMMZBczZGz7Vg90H21EOLACvtfcHk0ljx9kN0l6felQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFWks74%253D&md5=f8ef4ac7d54d4d577ba385a8918dc350</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1601520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1601520%26sid%3Dliteratum%253Aachs%26aulast%3DElliott%26aufirst%3DM.%2BR.%26aulast%3DKoster%26aufirst%3DK.%2BM.%26aulast%3DMurphy%26aufirst%3DP.%2BS.%26atitle%3DEfferocytosis%2520Signaling%2520in%2520the%2520regulation%2520of%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2017%26volume%3D198%26spage%3D1387%26epage%3D1394%26doi%3D10.4049%2Fjimmunol.1601520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gay, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byers, L. A.</span></span> <span> </span><span class="NLM_article-title">Giving AXL the axe: targeting AXL in human malignancy</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/bjc.2016.428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2Fbjc.2016.428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=28072762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVSruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2017&pages=415-423&author=C.+M.+Gayauthor=K.+Balajiauthor=L.+A.+Byers&title=Giving+AXL+the+axe%3A+targeting+AXL+in+human+malignancy&doi=10.1038%2Fbjc.2016.428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Giving AXL the axe: targeting AXL in human malignancy</span></div><div class="casAuthors">Gay, Carl M.; Balaji, Kavitha; Byers, Lauren Averett</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">415-423</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase AXL, activated by a complex interaction between its ligand growth arrest-specific protein 6 and phosphatidylserine, regulates various vital cellular processes, including proliferation, survival, motility, and immunol. response.  Although not implicated as an oncogenic driver itself, AXL, a member of the TYRO3, AXL, and MERTK family of receptor tyrosine kinases, is overexpressed in several haematol. and solid malignancies, including acute myeloid leukemia, non-small cell lung cancer, gastric and colorectal adenocarcinomas, and breast and prostate cancers.  In the context of malignancy, evidence suggests that AXL overexpression drives wide-ranging processes, including epithelial to mesenchymal transition, tumor angiogenesis, resistance to chemotherapeutic and targeted agents, and decreased antitumor immune response.  As a result, AXL is an attractive candidate not only as a prognostic biomarker in malignancy but also as a target for anticancer therapies.  Several AXL inhibitors are currently in preclin. and clin. development.  This article reviews the structure, regulation, and function of AXL; the role of AXL in the tumor microenvironment; the development of AXL as a therapeutic target; and areas of ongoing and future investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXM9RJ8TVsLVg90H21EOLACvtfcHk0lj_XZYOUTe5sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVSruw%253D%253D&md5=c24668744fe4b017def75b08bac2a863</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2016.428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2016.428%26sid%3Dliteratum%253Aachs%26aulast%3DGay%26aufirst%3DC.%2BM.%26aulast%3DBalaji%26aufirst%3DK.%26aulast%3DByers%26aufirst%3DL.%2BA.%26atitle%3DGiving%2520AXL%2520the%2520axe%253A%2520targeting%2520AXL%2520in%2520human%2520malignancy%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2017%26volume%3D116%26spage%3D415%26epage%3D423%26doi%3D10.1038%2Fbjc.2016.428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camenisch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boast, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casagranda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earp, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, G.</span></span> <span> </span><span class="NLM_article-title">Tyro-3 family receptors are essential regulators of mammalian spermatogenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>398</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/19554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2F19554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10227296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADyaK1MXivVegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=398&publication_year=1999&pages=723-728&author=Q.+Luauthor=M.+Goreauthor=Q.+Zhangauthor=T.+Camenischauthor=S.+Boastauthor=F.+Casagrandaauthor=C.+Laiauthor=M.+K.+Skinnerauthor=R.+Kleinauthor=G.+K.+Matsushimaauthor=H.+S.+Earpauthor=S.+P.+Goffauthor=G.+Lemke&title=Tyro-3+family+receptors+are+essential+regulators+of+mammalian+spermatogenesis&doi=10.1038%2F19554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tyro-3 family receptors are essential regulators of mammalian spermatogenesis</span></div><div class="casAuthors">Lu, Qingxian; Gore, Martin; Zhang, Qing; Camenisch, Todd; Boast, Sharon; Casagranda, Franca; Skinner, Michael K.; Klein, Riudiger; Matsushima, Glenn K.; Earp, H. Shelton; Goff, Stephen P.; Lemke, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">398</span>
        (<span class="NLM_cas:issue">6729</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">We have generated and analyzed null mutations in the mouse genes encoding three structurally related receptors with tyrosine kinase activity: Tyro 3, Axl, and Mer.  Mice lacking any single receptor, or any combination of two receptors, are viable and fertile, but male animals that lack all three receptors produce no mature sperm, owing to the progressive death of differentiating germ cells.  This degenerative phenotype appears to result from a failure of the tropic support that is normally provided by Sertoli cells of the seminiferous tubules, whose function depends on testosterone and addnl. factors produced by Leydig cells.  Tyro 3, Axl and Mer are all normally expressed by Sertoli cells during postnatal development, whereas their ligands, Gas6 and protein S, are produced by Leydig cells before sexual maturity, and by both Leydig and Sertoli cells thereafter.  Here we show that the concerted activation of Tyro 3, Axl and Mer in Sertoli cells is crit. to the role that these cells play as nurturers of developing germ cells.  Addnl. observations indicate that these receptors may also be essential for the tropic maintenance of diverse cell types in the mature nervous, immune and reproductive systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY4F6tCv0xxbVg90H21EOLACvtfcHk0lj_XZYOUTe5sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVegurk%253D&md5=f254abf74c110fcd3eb1fc773f4b46f7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F19554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F19554%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DGore%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DCamenisch%26aufirst%3DT.%26aulast%3DBoast%26aufirst%3DS.%26aulast%3DCasagranda%26aufirst%3DF.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DSkinner%26aufirst%3DM.%2BK.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DMatsushima%26aufirst%3DG.%2BK.%26aulast%3DEarp%26aufirst%3DH.%2BS.%26aulast%3DGoff%26aufirst%3DS.%2BP.%26aulast%3DLemke%26aufirst%3DG.%26atitle%3DTyro-3%2520family%2520receptors%2520are%2520essential%2520regulators%2520of%2520mammalian%2520spermatogenesis%26jtitle%3DNature%26date%3D1999%26volume%3D398%26spage%3D723%26epage%3D728%26doi%3D10.1038%2F19554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasileiadi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRyckere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. K.</span></span> <span> </span><span class="NLM_article-title">The emerging role of TYRO3 as a therapeutic target in cancer</span>. <i>Cancers (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">474</span>, <span class="refDoi"> DOI: 10.3390/cancers10120474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.3390%2Fcancers10120474" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=474&author=S.+K.+Smartauthor=E.+Vasileiadiauthor=X.+Wangauthor=D.+DeRyckereauthor=D.+K.+Graham&title=The+emerging+role+of+TYRO3+as+a+therapeutic+target+in+cancer&doi=10.3390%2Fcancers10120474"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3390%2Fcancers10120474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10120474%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DS.%2BK.%26aulast%3DVasileiadi%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDeRyckere%26aufirst%3DD.%26aulast%3DGraham%26aufirst%3DD.%2BK.%26atitle%3DThe%2520emerging%2520role%2520of%2520TYRO3%2520as%2520a%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DCancers%2520%2528Basel%2529%26date%3D2018%26volume%3D10%26spage%3D474%26doi%3D10.3390%2Fcancers10120474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janda, K. D.</span></span> <span> </span><span class="NLM_article-title">Synthetic methods for the implementation of encoded combinatorial chemistry</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">9812</span>– <span class="NLM_lpage">9813</span>, <span class="refDoi"> DOI: 10.1021/ja00074a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00074a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVOls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=9812-9813&author=J.+Nielsenauthor=S.+Brennerauthor=K.+D.+Janda&title=Synthetic+methods+for+the+implementation+of+encoded+combinatorial+chemistry&doi=10.1021%2Fja00074a063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic methods for the implementation of encoded combinatorial chemistry</span></div><div class="casAuthors">Nielsen, John; Brenner, Sydney; Janda, Kim D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9812-13</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">There has been a recent renaissance in drug screening with the development of new technologies which allow a large no. of compds. to be simultaneously exposed to a target.  In these "combinatorial libraries", compds. that bind to the target with the highest affinity are selected from the pool of statistical sequences.  Recently, a scheme for encoding combinatorially synthesized libraries has been proposed to surmount a no. of the limitations possessed by existing methods.  Encoded combinatorial chem. combines the large diversity that can be achieved with a chem. library with an encoded genetic tag which can be used for the identification and sequential enrichment of any active component.  The authors have now developed the chem. necessary to implement the conceptual scheme and how a CPG matrix can be appended to allow the parallel synthesis of peptides and their encoding nucleic acid sequences in an alternating, bi-directional manner.  In addn. the authors demonstrate how the same support can be modified to permit a controlled "dendritic" display of the chem. library.  Implementation of this latter regime provides a novel methodol. for controlled multivalent combinatorial ligand display.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbyd3WhXqvG7Vg90H21EOLACvtfcHk0lhyyrc_1o0vCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhtVOls7k%253D&md5=ae66399eb84c4a5af0e0a9d76960bb49</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja00074a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00074a063%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DJanda%26aufirst%3DK.%2BD.%26atitle%3DSynthetic%2520methods%2520for%2520the%2520implementation%2520of%2520encoded%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D9812%26epage%3D9813%26doi%3D10.1021%2Fja00074a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Encoded self-assembling chemical libraries</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nbt961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1038%2Fnbt961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=15097996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2004&pages=568-574&author=S.+Melkkoauthor=J.+Scheuermannauthor=C.+E.+Dumelinauthor=D.+Neri&title=Encoded+self-assembling+chemical+libraries&doi=10.1038%2Fnbt961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded self-assembling chemical libraries</span></div><div class="casAuthors">Melkko, Samu; Scheuermann, Joerg; Dumelin, Christoph E.; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">568-574</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The isolation of mols. capable of high-affinity and specific binding to biol. targets is a central problem in chem., biol. and pharmaceutical sciences.  Here we describe the use of encoded self-assembling chem. (ESAC) libraries for the facile identification of mols. that bind macromol. targets.  ESAC technol. uses libraries of org. mols. linked to individual oligonucleotides that mediate the self-assembly of the library and provide a code assocd. with each org. mol.  After panning ESAC libraries on the biomol. target of interest, the 'binding code' of the selected compds. can be 'decoded' by a no. of exptl. techniques (e.g., hybridization on oligonucleotide microarrays).  The potential of this technol. was demonstrated by the affinity maturation (>40-fold) of binding mols. to human serum albumin and bovine carbonic anhydrase, leading to binders with dissocn. consts. in the nanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojeEDxP4G3SLVg90H21EOLACvtfcHk0ljgtLgv79HgNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jsLg%253D&md5=aa09960ffa2b0dd1e4b45a4c5d0ab1a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt961%26sid%3Dliteratum%253Aachs%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DEncoded%2520self-assembling%2520chemical%2520libraries%26jtitle%3DNat.%2520Biotechnol.%26date%3D2004%26volume%3D22%26spage%3D568%26epage%3D574%26doi%3D10.1038%2Fnbt961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whartenby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabresi, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCadden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kardian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardoll, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of FLT3 signaling targets DCs to ameliorate autoimmune disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">16741</span>– <span class="NLM_lpage">16746</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506088102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1073%2Fpnas.0506088102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=16272221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KgsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=16741-16746&author=K.+A.+Whartenbyauthor=P.+A.+Calabresiauthor=E.+McCaddenauthor=B.+Nguyenauthor=D.+Kardianauthor=T.+Wangauthor=C.+Mosseauthor=D.+M.+Pardollauthor=D.+Small&title=Inhibition+of+FLT3+signaling+targets+DCs+to+ameliorate+autoimmune+disease&doi=10.1073%2Fpnas.0506088102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of FLT3 signaling targets DCs to ameliorate autoimmune disease</span></div><div class="casAuthors">Whartenby, Katharine A.; Calabresi, Peter A.; McCadden, Erin; Nguyen, Bao; Kardian, David; Wang, Tianhong; Mosse, Claudio; Pardoll, Drew M.; Small, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">16741-16746</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autoimmune diseases often result from inappropriate or unregulated activation of autoreactive T cells.  Traditional approaches to treatment of autoimmune diseases through immunosuppression have focused on direct inhibition of T cells.  In the present study, the authors examd. the targeted inhibition of antigen-presenting cells as a means to down-regulate immune responses and treat autoimmune disease.  Dendritic cells (DCs) are the central antigen-presenting cells for the initiation of T cell responses, including autoreactive ones.  A large portion of DCs are derived from hematopoietic progenitors that express FLT3 receptor (CD135), and stimulation of the receptor via FLT3 ligand either in vivo or in vitro is known to drive expansion and differentiation of these progenitors toward a DC phenotype.  The authors hypothesized that inhibition of FLT3 signaling would thus produce an inhibition of DC-induced stimulation of T cells, thereby inhibiting autoimmune responses.  To this end, the authors used small-mol. tyrosine kinase inhibitors targeted against FLT3 and examd. the effects on DCs and their role in the promulgation of autoimmune disease.  Results of our studies show that inhibition of FLT3 signaling induces apoptosis in both mouse and human DCs, and thus is a potential target for immune suppression.  Furthermore, targeted inhibition of FLT3 significantly improved the course of established disease in a model for multiple sclerosis, exptl. autoimmune encephalomyelitis, suggesting a potential avenue for treating autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-kLdUEFSK7Vg90H21EOLACvtfcHk0ljgtLgv79HgNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KgsLzL&md5=12e54a144aa8d70af7ec018e1bd086f4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506088102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506088102%26sid%3Dliteratum%253Aachs%26aulast%3DWhartenby%26aufirst%3DK.%2BA.%26aulast%3DCalabresi%26aufirst%3DP.%2BA.%26aulast%3DMcCadden%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DKardian%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DMosse%26aufirst%3DC.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DSmall%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520FLT3%2520signaling%2520targets%2520DCs%2520to%2520ameliorate%2520autoimmune%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D16741%26epage%3D16746%26doi%3D10.1073%2Fpnas.0506088102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karsunky, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, M. G.</span></span> <span> </span><span class="NLM_article-title">Flt3 ligand regulates dendritic cell development from Flt3+ lymphoid and myeloid-committed progenitors to Flt3+ dendritic cells in vivo</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1084/jem.20030323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1084%2Fjem.20030323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=12874263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVyjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2003&pages=305-313&author=H.+Karsunkyauthor=M.+Meradauthor=A.+Cozzioauthor=I.+L.+Weissmanauthor=M.+G.+Manz&title=Flt3+ligand+regulates+dendritic+cell+development+from+Flt3%2B+lymphoid+and+myeloid-committed+progenitors+to+Flt3%2B+dendritic+cells+in+vivo&doi=10.1084%2Fjem.20030323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Flt3 ligand regulates dendritic cell development from Flt3+ lymphoid and myeloid-committed progenitors to Flt3+ dendritic cells in vivo</span></div><div class="casAuthors">Karsunky, Holger; Merad, Miriam; Cozzio, Antonio; Weissman, Irving L.; Manz, Markus G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Stimulation of Flt3 receptor tyrosine kinase through its cognate ligand expands early hematopoietic progenitor and dendritic cells (DCs) in humans and mice.  The exact developmental stages at which hematopoietic progenitors express Flt3, are responsive to its ligand, and subsequently develop to DCs, are not known.  Here the authors show that common lymphoid and common myeloid progenitors, as well as steady state DCs in thymus, spleen, and epidermis, express Flt3.  The receptor is down-regulated once definitive B cell, T cell, and megakaryocyte/erythrocyte commitment occurs, and Flt3 is not detectable on other steady state hematopoietic cell populations.  Upon in vivo Flt3 ligand (Flt3L) administration, Flt3+ progenitor cells and their progeny DCs are expanded, whereas Flt3- downstream progenitors are not, or are only slightly increased.  Transplantation of common lymphoid and common myeloid progenitors and subsequent Flt3L injection increases progeny DCs of both precursor populations.  These findings provide a definitive map of Flt3 expression in the hematopoietic hierarchy and directly demonstrate that Flt3L can drive DC development along both the lymphoid and myeloid developmental pathways from Flt3+ progenitors to Flt3+ DCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKwAXIjMecULVg90H21EOLACvtfcHk0ljgtLgv79HgNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVyjtbk%253D&md5=04de331176e634038283e86ff00d637a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1084%2Fjem.20030323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20030323%26sid%3Dliteratum%253Aachs%26aulast%3DKarsunky%26aufirst%3DH.%26aulast%3DMerad%26aufirst%3DM.%26aulast%3DCozzio%26aufirst%3DA.%26aulast%3DWeissman%26aufirst%3DI.%2BL.%26aulast%3DManz%26aufirst%3DM.%2BG.%26atitle%3DFlt3%2520ligand%2520regulates%2520dendritic%2520cell%2520development%2520from%2520Flt3%252B%2520lymphoid%2520and%2520myeloid-committed%2520progenitors%2520to%2520Flt3%252B%2520dendritic%2520cells%2520in%2520vivo%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2003%26volume%3D198%26spage%3D305%26epage%3D313%26doi%3D10.1084%2Fjem.20030323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komarowska, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishore, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbruchel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitzalis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckenridge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marelli-Berg, F. M.</span></span> <span> </span><span class="NLM_article-title">Hepatocyte growth factor receptor c-Met instructs T cell cardiotropism and promotes T cell migration to the heart via autocrine chemokine release</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1087</span>– <span class="NLM_lpage">1099</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2015.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.immuni.2015.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26070483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWjtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=1087-1099&author=I.+Komarowskaauthor=D.+Coeauthor=G.+Wangauthor=R.+Haasauthor=C.+Mauroauthor=M.+Kishoreauthor=D.+Cooperauthor=S.+Nadkarniauthor=H.+Fuauthor=D.+A.+Steinbruchelauthor=C.+Pitzalisauthor=G.+Andersonauthor=P.+Bucyauthor=G.+Lombardiauthor=R.+Breckenridgeauthor=F.+M.+Marelli-Berg&title=Hepatocyte+growth+factor+receptor+c-Met+instructs+T+cell+cardiotropism+and+promotes+T+cell+migration+to+the+heart+via+autocrine+chemokine+release&doi=10.1016%2Fj.immuni.2015.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocyte Growth Factor Receptor c-Met Instructs T Cell Cardiotropism and Promotes T Cell Migration to the Heart via Autocrine Chemokine Release</span></div><div class="casAuthors">Komarowska, Izabela; Coe, David; Wang, Guosu; Haas, Robert; Mauro, Claudio; Kishore, Madhav; Cooper, Dianne; Nadkarni, Suchita; Fu, Hongmei; Steinbruchel, Daniel A.; Pitzalis, Costantino; Anderson, Graham; Bucy, Pat; Lombardi, Giovanna; Breckenridge, Ross; Marelli-Berg, Federica M.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1087-1099</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Effector-T-cell-mediated immunity depends on the efficient localization of antigen-primed lymphocytes to antigen-rich non-lymphoid tissue, which is facilitated by the expression of a unique set of "homing" receptors acquired by memory T cells.  We report that engagement of the hepatocyte growth factor (HGF) receptor c-Met by heart-produced HGF during priming in the lymph nodes instructs T cell cardiotropism, which was assocd. with a specialized homing "signature" (c-Met+CCR4+CXCR3+). c-Met signals facilitated T cell recruitment to the heart via the chemokine receptor CCR5 by inducing autocrine CCR5 ligand release. c-Met triggering was sufficient to support cardiotropic T cell recirculation, while CCR4 and CXCR3 sustained recruitment during heart inflammation.  Transient pharmacol. blockade of c-Met during T cell priming led to enhanced survival of heart, but not skin, allografts assocd. with impaired localization of alloreactive T cells to heart grafts.  These findings suggest c-Met as a target for development of organ-selective immunosuppressive therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLOLQYPIfr4rVg90H21EOLACvtfcHk0lgDaOCSI6EIBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWjtL3F&md5=be1c5b716f7c595889f1ee7a9eb3d186</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DKomarowska%26aufirst%3DI.%26aulast%3DCoe%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHaas%26aufirst%3DR.%26aulast%3DMauro%26aufirst%3DC.%26aulast%3DKishore%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DD.%26aulast%3DNadkarni%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DSteinbruchel%26aufirst%3DD.%2BA.%26aulast%3DPitzalis%26aufirst%3DC.%26aulast%3DAnderson%26aufirst%3DG.%26aulast%3DBucy%26aufirst%3DP.%26aulast%3DLombardi%26aufirst%3DG.%26aulast%3DBreckenridge%26aufirst%3DR.%26aulast%3DMarelli-Berg%26aufirst%3DF.%2BM.%26atitle%3DHepatocyte%2520growth%2520factor%2520receptor%2520c-Met%2520instructs%2520T%2520cell%2520cardiotropism%2520and%2520promotes%2520T%2520cell%2520migration%2520to%2520the%2520heart%2520via%2520autocrine%2520chemokine%2520release%26jtitle%3DImmunity%26date%3D2015%26volume%3D42%26spage%3D1087%26epage%3D1099%26doi%3D10.1016%2Fj.immuni.2015.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhade, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadri, A.</span></span> <span> </span><span class="NLM_article-title">Src kinases central to T-cell receptor signaling regulate TLR-activated innate immune responses from human T cells</span>. <i>Innate Immun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1177/1753425916632305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1177%2F1753425916632305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26888964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWkurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=238-244&author=N.+Sharmaauthor=A.+S.+Akhadeauthor=A.+Qadri&title=Src+kinases+central+to+T-cell+receptor+signaling+regulate+TLR-activated+innate+immune+responses+from+human+T+cells&doi=10.1177%2F1753425916632305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Src kinases central to T-cell receptor signaling regulate TLR-activated innate immune responses from human T cells</span></div><div class="casAuthors">Sharma, Naveen; Akhade, Ajay Suresh; Qadri, Ayub</div><div class="citationInfo"><span class="NLM_cas:title">Innate Immunity</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">238-244</span>CODEN:
                <span class="NLM_cas:coden">IINMCB</span>;
        ISSN:<span class="NLM_cas:issn">1753-4259</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">TLRs have a fundamental role in immunity.  We have recently reported that stimulation of TLR2 and TLR5 in freshly isolated and activated human T cells with microbial ligands without concomitant activation through the TCR brings about secretion of neutrophil chemoattractant, CXCL8, and effector cytokine, IFN-γ, resp.  However, the mechanism of TLR signaling in T cells has not been worked out.  Here, we show that the Src family kinases, p56lck (Lck) and p59fyn (Fyn), which are essential for activation of T cells through the TCR, are also crit. for signal transduction through TLRs in human T cells.  The secretion of CXCL8 following stimulation of the model human T cell line, Jurkat, with the TLR5 ligand, flagellin, was reduced in presence of the Src-kinase inhibitor, PP2 and specific inhibitors of Lck and Fyn.  These inhibitors suppressed generation of activated JNK and p38, which were both required for TLR-induced CXCL8 prodn.  The Lck-deficient deriv. of Jurkat, JCam1.6, responded poorly to TLR2, TLR5 and TLR7 agonists, and did not generate active signaling intermediates.  Lck and Fyn inhibitors also reduced TLR5-induced IFN-γ secretion from the activated T cell phenotype-representing T cell line, HuT78, without modulating JNK and p38 activation.  These results reveal that TCR and TLRs share key proximal signaling regulators in T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqD-A6nTkqZbVg90H21EOLACvtfcHk0lgDaOCSI6EIBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWkurfJ&md5=b9e7e36da131e33c45d583f8cbed7bf3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1177%2F1753425916632305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1753425916632305%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DAkhade%26aufirst%3DA.%2BS.%26aulast%3DQadri%26aufirst%3DA.%26atitle%3DSrc%2520kinases%2520central%2520to%2520T-cell%2520receptor%2520signaling%2520regulate%2520TLR-activated%2520innate%2520immune%2520responses%2520from%2520human%2520T%2520cells%26jtitle%3DInnate%2520Immun.%26date%3D2016%26volume%3D22%26spage%3D238%26epage%3D244%26doi%3D10.1177%2F1753425916632305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawagoe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uematsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span> <span> </span><span class="NLM_article-title">Essential role of IRAK-4 protein and its kinase activity in Toll-like receptor-mediated immune responses but not in TCR signaling</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1084/jem.20061523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1084%2Fjem.20061523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=17485511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFSjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2007&pages=1013-24&author=T.+Kawagoeauthor=S.+Satoauthor=A.+Jungauthor=M.+Yamamotoauthor=K.+Matsuiauthor=H.+Katoauthor=S.+Uematsuauthor=O.+Takeuchiauthor=S.+Akira&title=Essential+role+of+IRAK-4+protein+and+its+kinase+activity+in+Toll-like+receptor-mediated+immune+responses+but+not+in+TCR+signaling&doi=10.1084%2Fjem.20061523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of IRAK-4 protein and its kinase activity in Toll-like receptor-mediated immune responses but not in TCR signaling</span></div><div class="casAuthors">Kawagoe, Tatsukata; Sato, Shintaro; Jung, Andreas; Yamamoto, Masahiro; Matsui, Kosuke; Kato, Hiroki; Uematsu, Satoshi; Takeuchi, Osamu; Akira, Shizuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1013-1024</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin-1 receptor-assocd. kinase 4 (IRAK-4) was reported to be essential for the Toll-like receptor (TLR)- and T cell receptor (TCR)-mediated signaling leading to the activation of nuclear factor κB (NF-κB).  However, the importance of kinase activity of IRAK family members is unclear.  In this study, we investigated the functional role of IRAK-4 activity in vivo by generating mice carrying a knockin mutation (KK213AA) that abrogates its kinase activity.  IRAK-4KN/KN mice were highly resistant to TLR-induced shock response.  The cytokine prodn. in response to TLR ligands was severely impaired in IRAK-4KN/KN as well as IRAK-4-/- macrophages.  The IRAK-4 activity was essential for the activation of signaling pathways leading to mitogen-activated protein kinases.  TLR-induced IRAK-4/IRAK-1-dependent and -independent pathways were involved in early induction of NF-κB-regulated genes in response to TLR ligands such as tumor necrosis factor α and IκBζ.  In contrast to a previous paper, the TCR signaling was not impaired in IRAK-4-/- and IRAK-4KN/KN mice.  Thus, the kinase activity of IRAK-4 is essential for the regulation of TLR-mediated innate immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQDdu13L0FMrVg90H21EOLACvtfcHk0lg6rxKl9M82_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFSjsbY%253D&md5=902905464ef0cba41a3fda070fc982f5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1084%2Fjem.20061523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20061523%26sid%3Dliteratum%253Aachs%26aulast%3DKawagoe%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DMatsui%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DUematsu%26aufirst%3DS.%26aulast%3DTakeuchi%26aufirst%3DO.%26aulast%3DAkira%26aufirst%3DS.%26atitle%3DEssential%2520role%2520of%2520IRAK-4%2520protein%2520and%2520its%2520kinase%2520activity%2520in%2520Toll-like%2520receptor-mediated%2520immune%2520responses%2520but%2520not%2520in%2520TCR%2520signaling%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2007%26volume%3D204%26spage%3D1013%26epage%3D24%26doi%3D10.1084%2Fjem.20061523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leitinger, B.</span></span> <span> </span><span class="NLM_article-title">Discoidin domain receptor functions in physiological and pathological conditions</span>. <i>Int. Rev. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>310</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-800180-6.00002-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2FB978-0-12-800180-6.00002-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=24725424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlymsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2014&pages=39-87&author=B.+Leitinger&title=Discoidin+domain+receptor+functions+in+physiological+and+pathological+conditions&doi=10.1016%2FB978-0-12-800180-6.00002-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discoidin domain receptor functions in physiological and pathological conditions</span></div><div class="casAuthors">Leitinger, Birgit</div><div class="citationInfo"><span class="NLM_cas:title">International Review of Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">310</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-87</span>CODEN:
                <span class="NLM_cas:coden">IRCMAH</span>;
        ISSN:<span class="NLM_cas:issn">1937-6448</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The discoidin domain receptors, DDR1 and DDR2, are nonintegrin collagen receptors that are members of the receptor tyrosine kinase family.  Both DDRs bind a no. of different collagen types and play important roles in embryo development.  Dysregulated DDR function is assocd. with progression of various human diseases, including fibrosis, arthritis, and cancer.  By interacting with key components of the extracellular matrix and displaying distinct activation kinetics, the DDRs form a unique subfamily of receptor tyrosine kinases.  DDR-facilitated cellular functions include cell migration, cell survival, proliferation, and differentiation, as well as remodeling of extracellular matrixes.  This review summarizes the current knowledge of DDR-ligand interactions, DDR-initiated signal pathways and the mol. mechanisms that regulate receptor function.  Also discussed are the roles of DDRs in development and disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaBb00Ee5sVrVg90H21EOLACvtfcHk0lg6rxKl9M82_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlymsb7M&md5=19c591ee1051c6bee5b55b8f55d61877</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-800180-6.00002-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-800180-6.00002-5%26sid%3Dliteratum%253Aachs%26aulast%3DLeitinger%26aufirst%3DB.%26atitle%3DDiscoidin%2520domain%2520receptor%2520functions%2520in%2520physiological%2520and%2520pathological%2520conditions%26jtitle%3DInt.%2520Rev.%2520Cell%2520Mol.%2520Biol.%26date%3D2014%26volume%3D310%26spage%3D39%26epage%3D87%26doi%3D10.1016%2FB978-0-12-800180-6.00002-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuyama, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, T.</span></span> <span> </span><span class="NLM_article-title">Activation of discoidin domain receptor 1 isoform b with collagen up-regulates chemokine production in human macrophages: role of p38 mitogen-activated protein kinase and NF-kappa B</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">2332</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.172.4.2332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4049%2Fjimmunol.172.4.2332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=14764702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslWltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2004&pages=2332-40&author=W.+Matsuyamaauthor=L.+Wangauthor=W.+L.+Farrarauthor=M.+Faureauthor=T.+Yoshimura&title=Activation+of+discoidin+domain+receptor+1+isoform+b+with+collagen+up-regulates+chemokine+production+in+human+macrophages%3A+role+of+p38+mitogen-activated+protein+kinase+and+NF-kappa+B&doi=10.4049%2Fjimmunol.172.4.2332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Discoidin Domain Receptor 1 Isoform b with Collagen Up-Regulates Chemokine Production in Human Macrophages: Role of p38 Mitogen-Activated Protein Kinase and NF-κB</span></div><div class="casAuthors">Matsuyama, Wataru; Wang, Lihua; Farrar, William L.; Faure, Michel; Yoshimura, Teizo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2332-2340</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Macrophages produce an array of proinflammatory mediators at sites of inflammation and contribute to the development of inflammatory responses.  Important roles for cytokines, such as IL-1 or TNF-α, and bacterial products, such as LPS, in this process have been well documented; however, the role for the extracellular matrix proteins, such as collagen, remains unclear.  The authors previously reported that discoidin domain receptor 1 (DDR1), a nonintegrin collagen receptor, is expressed during differentiation of human monocytes into macrophages, and the interaction of the DDR1b isoform with collagen facilitates their differentiation via the p38 mitogen-activated protein kinase (MAPK) pathway.  In this study, the authors report that the interaction of DDR1b with collagen up-regulates the prodn. of IL-8, macrophage inflammatory protein-1α, and monocyte chemoattractant protein-1 in human macrophages in a p38 MAPK- and NF-κB-dependent manner. p38 MAPK was crit. for DDR1b-mediated, increased NF-κB trans-activity, but not for IκB degrdn. or NF-κB nuclear translocation, suggesting a role for p38 MAPK in the modification of NF-κB.  P38 MAPK was crit. for DDR1b-mediated, increased NF-κB trans-activity, but not for IκB degrdn. or NF-κB nuclear translocation, suggesting a role for p38 MAPK in the modification of NF-κB.  DDR1b-mediated IκB degrdn. was mediated through the recruitment of the adaptor protein Shc to the LXNPXY motif of the receptor and the downstream TNFR-assocd. factor 6/NF-κB activator 1 signaling cascade.  Taken together, the authors' study has identified NF-κB as a novel target of DDR1b signaling and provided a novel mechanism by which tissue-infiltrating macrophages produce large amts. of chemokines during the development of inflammatory diseases.  Intervention of DDR1b signaling may be useful to control inflammatory diseases in which these proteins play an important role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0dml00RSS7Vg90H21EOLACvtfcHk0liQzjMOAWPopg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslWltQ%253D%253D&md5=01893aefd9759bd412b6f3907bc13176</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.172.4.2332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.172.4.2332%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuyama%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFarrar%26aufirst%3DW.%2BL.%26aulast%3DFaure%26aufirst%3DM.%26aulast%3DYoshimura%26aufirst%3DT.%26atitle%3DActivation%2520of%2520discoidin%2520domain%2520receptor%25201%2520isoform%2520b%2520with%2520collagen%2520up-regulates%2520chemokine%2520production%2520in%2520human%2520macrophages%253A%2520role%2520of%2520p38%2520mitogen-activated%2520protein%2520kinase%2520and%2520NF-kappa%2520B%26jtitle%3DJ.%2520Immunol.%26date%3D2004%26volume%3D172%26spage%3D2332%26epage%3D40%26doi%3D10.4049%2Fjimmunol.172.4.2332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zioncheck, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vos, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span> <span> </span><span class="NLM_article-title">KDR (VEGF receptor 2) is the major mediator for the hypotensive effect of VEGF</span>. <i>Hypertension</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1161/01.HYP.0000018588.56950.7A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1161%2F01.HYP.0000018588.56950.7A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=12052848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFWgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2002&pages=1095-1100&author=B.+Liauthor=A.+K.+Ogasawaraauthor=R.+Yangauthor=W.+Weiauthor=G.+W.+Heauthor=T.+F.+Zioncheckauthor=S.+Buntingauthor=A.+M.+de+Vosauthor=H.+Jin&title=KDR+%28VEGF+receptor+2%29+is+the+major+mediator+for+the+hypotensive+effect+of+VEGF&doi=10.1161%2F01.HYP.0000018588.56950.7A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">KDR (VEGF receptor 2) is the major mediator for the hypotensive effect of VEGF</span></div><div class="casAuthors">Li, Bing; Ogasawara, Annie K.; Yang, Renhui; Wei, Wei; He, Guo-Wei; Zioncheck, Thomas F.; Bunting, Stuart; De Vos, Abraham M.; Jin, Hongkui</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1095-1100</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) exerts vasodilation-induced hypotension as a major side effect for treatment of ischemic diseases.  VEGF has 2 receptor tyrosine kinases, KDR and Flt-1.  Little is known about which receptor mediates VEGF-induced hypotension.  To elucidate the role of each receptor in mediating hypotension, KDR-selective and Flt-1-selective mutants were used for in vitro and in vivo studies.  The KDR-selective mutant induced vascular endothelial cell proliferation comparable to VEGF, whereas the Flt-1-selective mutant had no effect on proliferation.  I.v. injection of KDR-selective mutant, Flt-selective mutant, or VEGF caused a dose-related decrease in mean arterial pressure in conscious rats.  The hypotensive response to KDR-selective mutant was significantly less than that to VEGF (P<0.01) but was greater than that to Flt-selective mutant (P<0.01).  Similarly, VEGF and KDR-selective mutant induced more potent vasorelaxation than Flt-selective mutant or placenta growth factor that binds Flt-1 only (P<0.01), and the vasorelaxation to KDR-selective mutant was not significantly different at low concns. but less than that to VEGF at high concns.  The results indicate that the vasodilation and hypotensive effect of VEGF may involve both receptors, but KDR is the predominant receptor mediating this effect.  Because KDR-selective mutant induced proliferation and angiogenesis similar to VEGF but was assocd. with 36% attenuation in hypotension, the data suggest that the KDR-selective mutant may represent an alternative treatment for ischemic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4rC3aPHNaC7Vg90H21EOLACvtfcHk0liQzjMOAWPopg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFWgt7w%253D&md5=13dbdf4b1b0c38908aeb211b9713c305</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1161%2F01.HYP.0000018588.56950.7A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.HYP.0000018588.56950.7A%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DOgasawara%26aufirst%3DA.%2BK.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DG.%2BW.%26aulast%3DZioncheck%26aufirst%3DT.%2BF.%26aulast%3DBunting%26aufirst%3DS.%26aulast%3Dde%2BVos%26aufirst%3DA.%2BM.%26aulast%3DJin%26aufirst%3DH.%26atitle%3DKDR%2520%2528VEGF%2520receptor%25202%2529%2520is%2520the%2520major%2520mediator%2520for%2520the%2520hypotensive%2520effect%2520of%2520VEGF%26jtitle%3DHypertension%26date%3D2002%26volume%3D39%26spage%3D1095%26epage%3D1100%26doi%3D10.1161%2F01.HYP.0000018588.56950.7A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramaiah, L.</span>; <span class="NLM_string-name">Bounous, D. I.</span>; <span class="NLM_string-name">Elmore, S. A.</span></span> <span> </span><span class="NLM_article-title">Hematopoietic System</span>. In  <i>Fundamentals of Toxicologic Pathology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallig, M. A.</span>, <span class="NLM_string-name">Haschek, M. W.</span>, <span class="NLM_string-name">Rousseau, C. G.</span>, <span class="NLM_string-name">Bolon, B.</span>, <span class="NLM_string-name">Mahler, B. W.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press, Elsevier Inc.</span>: <span class="NLM_publisher-loc">London</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2FB978-0-12-809841-7.00013-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Ramaiah&author=D.+I.+Bounous&author=S.+A.+Elmoreauthor=M.+A.+Wallig&author=M.+W.+Haschek&author=C.+G.+Rousseau&author=B.+Bolon&author=B.+W.+Mahler&title=Fundamentals+of+Toxicologic+Pathology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-809841-7.00013-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-809841-7.00013-7%26sid%3Dliteratum%253Aachs%26aulast%3DRamaiah%26aufirst%3DL.%26atitle%3DHematopoietic%2520System%26btitle%3DFundamentals%2520of%2520Toxicologic%2520Pathology%26aulast%3DWallig%26aufirst%3DM.%2BA.%26pub%3DAcademic%2520Press%252C%2520Elsevier%2520Inc%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: a selection system based on endowing organic compounds with amplifiable information</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-062917-012550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1146%2Fannurev-biochem-062917-012550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=29328784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=479-502&author=D.+Neriauthor=R.+A.+Lerner&title=DNA-encoded+chemical+libraries%3A+a+selection+system+based+on+endowing+organic+compounds+with+amplifiable+information&doi=10.1146%2Fannurev-biochem-062917-012550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: A Selection System Based on Endowing Organic Compounds with Amplifiable Information</span></div><div class="casAuthors">Neri, Dario; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">479-502</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The discovery of org. ligands that bind specifically to proteins is a central problem in chem., biol., and the biomedical sciences.  The encoding of individual org. mols. with distinctive DNA tags, serving as amplifiable identification bar codes, allows the construction and screening of combinatorial libraries of unprecedented size, thus facilitating the discovery of ligands to many different protein targets.  Fundamentally, one links powers of genetics and chem. synthesis.  After the initial description of DNA-encoded chem. libraries in 1992, several exptl. embodiments of the technol. have been reduced to practice.  This review provides a historical account of important milestones in the development of DNA-encoded chem. libraries, a survey of relevant ongoing research activities, and a glimpse into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAjUib3HfEbVg90H21EOLACvtfcHk0lgE1I0w8lRZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D&md5=559fefad70a7b9949319ac8ad8066004</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-062917-012550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-062917-012550%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520a%2520selection%2520system%2520based%2520on%2520endowing%2520organic%2520compounds%2520with%2520amplifiable%2520information%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D479%26epage%3D502%26doi%3D10.1146%2Fannurev-biochem-062917-012550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span> <span> </span><span class="NLM_article-title">What do you get from DNA-encoded libraries?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=408-410&author=A.+L.+Satz&title=What+do+you+get+from+DNA-encoded+libraries%3F&doi=10.1021%2Facsmedchemlett.8b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">What Do You Get from DNA-Encoded Libraries?</span></div><div class="casAuthors">Satz, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">408-410</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Use of DNA-encoded libraries (DELs) in the pharmaceutical industry has rapidly increased.  We discuss what to expect when you run a DEL screen and contemplate guidelines for library design.  Addnl., we consider some visionary work and extrapolate to the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqODDlBH-pnArVg90H21EOLACvtfcHk0lgE1I0w8lRZxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslKmt70%253D&md5=0d2d28d00011533ffafd57d0308a1969</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00128%26sid%3Dliteratum%253Aachs%26aulast%3DSatz%26aufirst%3DA.%2BL.%26atitle%3DWhat%2520do%2520you%2520get%2520from%2520DNA-encoded%2520libraries%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D408%26epage%3D410%26doi%3D10.1021%2Facsmedchemlett.8b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel autotaxin inhibitor for the treatment of idiopathic pulmonary fibrosis: a clinical candidate discovered using DNA-encoded chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+autotaxin+inhibitor+for+the+treatment+of+idiopathic+pulmonary+fibrosis%3A+a+clinical+candidate+discovered+using+DNA-encoded+chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0lhSShYkLvtkWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520autotaxin%2520inhibitor%2520for%2520the%2520treatment%2520of%2520idiopathic%2520pulmonary%2520fibrosis%253A%2520a%2520clinical%2520candidate%2520discovered%2520using%2520DNA-encoded%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rho, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span> <span> </span><span class="NLM_article-title">MET and AXL inhibitor NPS-1034 exerts efficacy against lung cancer cells resistant to EGFR kinase inhibitors because of MET or AXL activation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1158%2F0008-5472.CAN-13-1103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=24165158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislKjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=253-262&author=J.+K.+Rhoauthor=Y.+J.+Choiauthor=S.+Y.+Kimauthor=T.+W.+Kimauthor=E.+K.+Choiauthor=S.+J.+Yoonauthor=B.+M.+Parkauthor=E.+Parkauthor=J.+H.+Baeauthor=C.+M.+Choiauthor=J.+C.+Lee&title=MET+and+AXL+inhibitor+NPS-1034+exerts+efficacy+against+lung+cancer+cells+resistant+to+EGFR+kinase+inhibitors+because+of+MET+or+AXL+activation&doi=10.1158%2F0008-5472.CAN-13-1103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">MET and AXL Inhibitor NPS-1034 Exerts Efficacy against Lung Cancer Cells Resistant to EGFR Kinase Inhibitors Because of MET or AXL Activation</span></div><div class="casAuthors">Rho, Jin Kyung; Choi, Yun Jung; Kim, Seon Ye; Kim, Tae Won; Choi, Eun Kyung; Yoon, Seon-Joo; Park, Bu Man; Park, Eunhye; Bae, Jong Hwan; Choi, Chang-Min; Lee, Jae Cheol</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-262</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In non-small cell lung cancer (NSCLC) with epidermal growth factor receptor (EGFR) mutations, acquired resistance to EGFR-tyrosine kinase inhibitors (EGFR-TK1) can occur through a generation of bypass signals such as MET or AXL activation.  In this study, we investigated the antitumor activity of NPS-1034, a newly developed drug that targets both MET and AXL, in NSCLC cells with acquired resistance to gefitinib or erlotinib (HCC827/GR and HCC827/ER, resp.).  Characterization of H820 cells and evaluation of NPS-1034 efficacy in these cells were also performed.  The resistance of HCC827/GR was mediated by MET activation, whereas AXL activation led to resistance in HCC827/ER.  The combination of gefitinib or erlotinib with NPS-1034 synergistically inhibited cell proliferation and induced cell death in both resistant cell lines.  Accordingly, suppression of Akt was noted only in the presence of treatment with both drugs.  NPS-1034 was also effective in xenograft mouse models of HCC827/GR.  Although the H820 cell line was reported previously to have T790M and MET amplification, we discovered that AXL was also activated in this cell line.  There were no antitumor effects of siRNA or inhibitors specific for EGFR or MET, whereas combined treatment with AXL siRNA or NPS-1034 and EGFR-TKIs controlled H820 cells, suggesting that AXL is the main signal responsible for resistance.  In addn., NPS-1034 inhibited cell proliferation as well as ROS1 activity in HCC78 cells with ROS1 rearrangement.  Our results establish the efficacy of NPS-1034 in NSCLC cells rendered resistant to EGFR-TKIs because of MET or AXL activation or ROS1 rearrangement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzVRl7xbUBLrVg90H21EOLACvtfcHk0lhSShYkLvtkWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislKjtA%253D%253D&md5=35929f2dcfd5d1502ba745f367dccd27</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1103%26sid%3Dliteratum%253Aachs%26aulast%3DRho%26aufirst%3DJ.%2BK.%26aulast%3DChoi%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DKim%26aufirst%3DT.%2BW.%26aulast%3DChoi%26aufirst%3DE.%2BK.%26aulast%3DYoon%26aufirst%3DS.%2BJ.%26aulast%3DPark%26aufirst%3DB.%2BM.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DBae%26aufirst%3DJ.%2BH.%26aulast%3DChoi%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26atitle%3DMET%2520and%2520AXL%2520inhibitor%2520NPS-1034%2520exerts%2520efficacy%2520against%2520lung%2520cancer%2520cells%2520resistant%2520to%2520EGFR%2520kinase%2520inhibitors%2520because%2520of%2520MET%2520or%2520AXL%2520activation%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D253%26epage%3D262%26doi%3D10.1158%2F0008-5472.CAN-13-1103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lh0NTu2W8tJow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/jm0603365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools&doi=10.1021%2Fjm0603365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lh0NTu2W8tJow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82%26doi%3D10.1021%2Fjm0603365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenjans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W.</span></span> <span> </span><span class="NLM_article-title">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2015.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1016%2Fj.cbpa.2015.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=25909818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=104-110&author=M.+J.+Wigglesworthauthor=D.+C.+Murrayauthor=C.+J.+Blackettauthor=M.+Kossenjansauthor=J.+W.+Nissink&title=Increasing+the+delivery+of+next+generation+therapeutics+from+high+throughput+screening+libraries&doi=10.1016%2Fj.cbpa.2015.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing the delivery of next generation therapeutics from high throughput screening libraries</span></div><div class="casAuthors">Wigglesworth, Mark J.; Murray, David C.; Blackett, Carolyn J.; Kossenjans, Michael; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-110</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry has historically relied on high throughput screening as a cornerstone to identify chem. equity for drug discovery projects.  However, with pharmaceutical companies moving through a phase of diminished returns and alternative hit identification strategies proving successful, it is more important than ever to understand how this approach can be used more effectively to increase the delivery of next generation therapeutics from high throughput screening libraries.  There is a wide literature that describes HTS and fragment based screening approaches which offer clear direction on the process for these two distinct activities.  However, few people have considered how best to identify medium to low mol. wt. compds. from large diversity screening sets and increase downstream success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxFeNO1FIRBLVg90H21EOLACvtfcHk0lim7lARHR3ysg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvV2jsLY%253D&md5=4b4bd0e7992796bd208a99dcb099c785</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2015.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DD.%2BC.%26aulast%3DBlackett%26aufirst%3DC.%2BJ.%26aulast%3DKossenjans%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26atitle%3DIncreasing%2520the%2520delivery%2520of%2520next%2520generation%2520therapeutics%2520from%2520high%2520throughput%2520screening%2520libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D26%26spage%3D104%26epage%3D110%26doi%3D10.1016%2Fj.cbpa.2015.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, S.</span></span> <span> </span><span class="NLM_article-title">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.4155%2Ffmc.14.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=25078133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1113-1126&author=J.+W.+M.+Nissinkauthor=S.+Blackburn&title=Quantification+of+frequent-hitter+behavior+based+on+historical+high-throughput+screening+data&doi=10.4155%2Ffmc.14.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of frequent-hitter behavior based on historical high-throughput screening data</span></div><div class="casAuthors">Nissink, J. Willem M.; Blackburn, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1113-1126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: We mine historical high-throughput data to identify and characterize 'frequent hitters', hits that are potentially false-pos. results.  Background: A key problem in the field of high-throughput screening (HTS) is recognition of frequent hitters, which are false-pos. or otherwise anomalous compds. that tend to crop up across many screens.  Follow-up of such compds. constitutes a waste of resource and decreases efficiency.  Methodol.: We describe a systematic retrospective approach to identify anomalous hitter behavior using historical screening data.  We take into account the uncertainty that arises if not enough screen data are available and extend implementation to target and technol. classes.  Conclusion: Use of the descriptor in analyzing high-throughput screen results frees up resource for follow-up of more likely true hits in the downstream hit-deconvolution cascade, thereby increasing efficiency of screen delivery.  Although effective, historical data bias can affect the annotation, and we exemplify cases where this happened.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXXu5-oAsV7Vg90H21EOLACvtfcHk0lim7lARHR3ysg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyjurvK&md5=f1e15dbd418d1956bd15029265c9075e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.72%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBlackburn%26aufirst%3DS.%26atitle%3DQuantification%2520of%2520frequent-hitter%2520behavior%2520based%2520on%2520historical%2520high-throughput%2520screening%2520data%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1113%26epage%3D1126%26doi%3D10.4155%2Ffmc.14.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wernevik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergstrom, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noven, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulthe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmgren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stromstedt, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehnstrom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbock, T.</span></span> <span> </span><span class="NLM_article-title">A fully integrated assay panel for early drug metabolism and pharmacokinetics profiling</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1089/adt.2020.970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1089%2Fadt.2020.970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=32407132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=157-179&author=J.+Wernevikauthor=F.+Bergstromauthor=A.+Novenauthor=J.+Hultheauthor=L.+Fredlundauthor=D.+Addisonauthor=J.+Holmgrenauthor=P.+E.+Stromstedtauthor=E.+Rehnstromauthor=T.+Lundbock&title=A+fully+integrated+assay+panel+for+early+drug+metabolism+and+pharmacokinetics+profiling&doi=10.1089%2Fadt.2020.970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Fully Integrated Assay Panel for Early Drug Metabolism and Pharmacokinetics Profiling</span></div><div class="casAuthors">Wernevik, Johan; Bergstroem, Fredrik; Noven, Anna; Hulthe, Johan; Fredlund, Linda; Addison, Dan; Holmgren, Jan; Stroemstedt, Per-Erik; Rehnstroem, Erika; Lundboeck, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">157-179</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Evaluation and optimization of physicochem. and metabolic properties of compds. are a crucial component of the drug development process.  Continuous access to this information during the design-make-test-anal. cycle enables identification of chem. entities with suitable properties for efficient project progression.  In this study, we describe an integrated and automated assay panel (DMPK Wave 1) that informs weekly on lipophilicity, soly., human plasma protein binding, and metabolic stability in rat hepatocytes and human liver microsomes.  All assays are running in 96-well format with ultraperformance liq. chromatog.-mass spectrometry (MS)/MS as read-out.  A streamlined overall workflow has been developed by optimizing all parts of the process, including shipping of compds. between sites, use of fit-for-purpose equipment and information systems, and technol. for compd. requesting, data anal., and reporting.  As a result, lead times can be achieved that well match project demands across sites independently of where compds. are synthesized.  This robust screening strategy is run on a weekly basis and enables optimization of structure-activity relationships in parallel with DMPK properties to allow efficient and informed decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr61vWtfynwTrVg90H21EOLACvtfcHk0lim7lARHR3ysg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D&md5=9b19876b26b8ddfb7aad414f5cf9c0a1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1089%2Fadt.2020.970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2020.970%26sid%3Dliteratum%253Aachs%26aulast%3DWernevik%26aufirst%3DJ.%26aulast%3DBergstrom%26aufirst%3DF.%26aulast%3DNoven%26aufirst%3DA.%26aulast%3DHulthe%26aufirst%3DJ.%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DAddison%26aufirst%3DD.%26aulast%3DHolmgren%26aufirst%3DJ.%26aulast%3DStromstedt%26aufirst%3DP.%2BE.%26aulast%3DRehnstrom%26aufirst%3DE.%26aulast%3DLundbock%26aufirst%3DT.%26atitle%3DA%2520fully%2520integrated%2520assay%2520panel%2520for%2520early%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520profiling%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2020%26volume%3D18%26spage%3D157%26epage%3D179%26doi%3D10.1089%2Fadt.2020.970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Compound set enrichment: a novel approach to analysis of primary HTS data</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2067</span>– <span class="NLM_lpage">2078</span>, <span class="refDoi"> DOI: 10.1021/ci100203e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100203e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=2067-2078&author=T.+Varinauthor=H.+Gublerauthor=C.+N.+Parkerauthor=J.+H.+Zhangauthor=P.+Ramanauthor=P.+Ertlauthor=A.+Schuffenhauer&title=Compound+set+enrichment%3A+a+novel+approach+to+analysis+of+primary+HTS+data&doi=10.1021%2Fci100203e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Compound Set Enrichment: A Novel Approach to Analysis of Primary HTS Data</span></div><div class="casAuthors">Varin, Thibault; Gubler, Hanspeter; Parker, Christian N.; Zhang, Ji-Hu; Raman, Pichai; Ertl, Peter; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2067-2078</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main goal of high-throughput screening (HTS) is to identify active chem. series rather than just individual active compds.  In light of this goal, a new method (called compd. set enrichment) to identify active chem. series from primary screening data is proposed.  The method employs the scaffold tree compd. classification in conjunction with the Kolmogorov-Smirnov statistic to assess the overall activity of a compd. scaffold.  The application of this method to seven PubChem data sets (contg. between 9389 and 263679 mols.) is presented, and the ability of this method to identify compd. classes with only weakly active compds. (potentially latent hits) is demonstrated.  The anal. presented here shows how methods based on an activity cutoff can distort activity information, leading to the incorrect activity assignment of compd. series.  These results suggest that this method might have utility in the rational selection of active classes of compds. (and not just individual active compds.) for followup and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlFSGm_wSXybVg90H21EOLACvtfcHk0lgu_n-cms8yIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbbN&md5=151077ce174ceaed80e3708e960a57e4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci100203e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100203e%26sid%3Dliteratum%253Aachs%26aulast%3DVarin%26aufirst%3DT.%26aulast%3DGubler%26aufirst%3DH.%26aulast%3DParker%26aufirst%3DC.%2BN.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DCompound%2520set%2520enrichment%253A%2520a%2520novel%2520approach%2520to%2520analysis%2520of%2520primary%2520HTS%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D2067%26epage%3D2078%26doi%3D10.1021%2Fci100203e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimpl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span> <span> </span><span class="NLM_article-title">A-loop interactions in Mer tyrosine kinase give rise to inhibitors with two-step mechanism and long residence time of binding</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">4443</span>– <span class="NLM_lpage">4452</span>, <span class="refDoi"> DOI: 10.1042/BCJ20200735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=10.1042%2FBCJ20200735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=33119085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1amtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2020&pages=4443-4452&author=A.+Pflugauthor=M.+Schimplauthor=J.+W.+M.+Nissinkauthor=R.+C.+Overmanauthor=P.+B.+Rawlinsauthor=C.+Trumanauthor=E.+Underwoodauthor=J.+Warwickerauthor=J.+J.+Winter-Holtauthor=W.+McCoull&title=A-loop+interactions+in+Mer+tyrosine+kinase+give+rise+to+inhibitors+with+two-step+mechanism+and+long+residence+time+of+binding&doi=10.1042%2FBCJ20200735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A-loop interactions in Mer tyrosine kinase give rise to inhibitors with two-step mechanism and long residence time of binding</span></div><div class="casAuthors">Pflug, Alexander; Schimpl, Marianne; Nissink, J. Willem M.; Overman, Ross C.; Rawlins, Philip B.; Truman, Caroline; Underwood, Elizabeth; Warwicker, Juli; Winter-Holt, Jon; McCoull, William</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4443-4452</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The activation loop (A-loop) plays a key role in regulating the catalytic activity of protein kinases.  Phosphorylation in this region enhances the phosphoryl transfer rate of the kinase domain and increases its affinity for ATP.  Furthermore, the A-loop possesses autoinhibitory functions in some kinases, where it collapses onto the protein surface and blocks substrate binding when unphosphorylated.  Due to its flexible nature, the A-loop is usually disordered and untraceable in kinase domain crystal structures.  The resulting lack of structural information is regrettable as it impedes the design of drug A-loop contacts, which have proven favorable in multiple cases.  Here, we characterize the binding with A-loop engagement between type 1.5 kinase inhibitor 'example 172' (EX172) and Mer tyrosine kinase (MerTK).  With the help of crystal structures and binding kinetics, we portray how the recruitment of the A-loop elicits a two-step binding mechanism which results in a drug-target complex characterized by high affinity and long residence time.  In addn., the type 1.5 compd. possesses excellent kinome selectivity and a remarkable preference for the phosphorylated over the dephosphorylated form of MerTK.  We discuss these unique characteristics in the context of known type 1 and type 2 inhibitors and highlight opportunities for future kinase inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDfeb8S10e-bVg90H21EOLACvtfcHk0lgu_n-cms8yIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1amtLvF&md5=e92a372506add30cfe2b3da8a9e2ded1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBCJ20200735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20200735%26sid%3Dliteratum%253Aachs%26aulast%3DPflug%26aufirst%3DA.%26aulast%3DSchimpl%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DOverman%26aufirst%3DR.%2BC.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%2BJ.%26aulast%3DMcCoull%26aufirst%3DW.%26atitle%3DA-loop%2520interactions%2520in%2520Mer%2520tyrosine%2520kinase%2520give%2520rise%2520to%2520inhibitors%2520with%2520two-step%2520mechanism%2520and%2520long%2520residence%2520time%2520of%2520binding%26jtitle%3DBiochem.%2520J.%26date%3D2020%26volume%3D477%26spage%3D4443%26epage%3D4452%26doi%3D10.1042%2FBCJ20200735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span>; <span class="NLM_string-name">Wilkinson, S. R.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Hall, B. S.</span></span> <i>Compounds, Compositions and Methods for Treatment of Protozoan Infections</i>. International Patent Application <span class="NLM_patent">WO2011066416</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Hu&author=S.+R.+Wilkinson&author=X.+Wu&author=B.+S.+Hall&title=Compounds%2C+Compositions+and+Methods+for+Treatment+of+Protozoan+Infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26jtitle%3DCompounds%252C%2520Compositions%2520and%2520Methods%2520for%2520Treatment%2520of%2520Protozoan%2520Infections%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaten, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swoboda, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inducer of beta cell proliferation identified by high-throughput screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">1669</span>– <span class="NLM_lpage">1672</span>, <span class="refDoi"> DOI: 10.1021/ja309304m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja309304m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=1669-1672&author=W.+Shenauthor=M.+S.+Tremblayauthor=V.+A.+Deshmukhauthor=W.+Wangauthor=C.+M.+Filippiauthor=G.+Harbauthor=Y.+Q.+Zhangauthor=A.+Kamireddyauthor=J.+E.+Baatenauthor=Q.+Jinauthor=T.+Wuauthor=J.+G.+Swobodaauthor=C.+Y.+Choauthor=J.+Liauthor=B.+A.+Laffitteauthor=P.+McNamaraauthor=R.+Glynneauthor=X.+Wuauthor=A.+E.+Hermanauthor=P.+G.+Schultz&title=Small-molecule+inducer+of+beta+cell+proliferation+identified+by+high-throughput+screening&doi=10.1021%2Fja309304m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inducer of β Cell Proliferation Identified by High-Throughput Screening</span></div><div class="casAuthors">Shen, Weijun; Tremblay, Matthew S.; Deshmukh, Vishal A.; Wang, Weidong; Filippi, Christophe M.; Harb, George; Zhang, You-qing; Kamireddy, Anwesh; Baaten, Janine E.; Jin, Qihui; Wu, Tom; Swoboda, Jonathan G.; Peters, Eric C.; Cho, Charles Y.; Li, Jing; Laffitte, Bryan A.; McNamara, Peter; Glynne, Richard; Wu, Xu; Herman, Ann E.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1669-1672</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of factors that promote β cell proliferation could ultimately move type 1 diabetes treatment away from insulin injection therapy and toward a cure.  We have performed high-throughput, cell-based screens using rodent β cell lines to identify mols. that induce proliferation of β cells.  Herein we report the discovery and characterization of WS6, a novel small mol. that promotes β cell proliferation in rodent and human primary islets.  In the RIP-DTA mouse model of β cell ablation, WS6 normalized blood glucose and induced concomitant increases in β cell proliferation and β cell no.  Affinity pulldown and kinase profiling studies implicate Erb3 binding protein-1 and the IκB kinase pathway in the mechanism of action of WS6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcVnec647SN7Vg90H21EOLACvtfcHk0lidxJDJYaYpew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCgu7w%253D&md5=f7a76c2f8b61e183e171b3343d65543a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja309304m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja309304m%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTremblay%26aufirst%3DM.%2BS.%26aulast%3DDeshmukh%26aufirst%3DV.%2BA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DBaaten%26aufirst%3DJ.%2BE.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DSwoboda%26aufirst%3DJ.%2BG.%26aulast%3DCho%26aufirst%3DC.%2BY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLaffitte%26aufirst%3DB.%2BA.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DSmall-molecule%2520inducer%2520of%2520beta%2520cell%2520proliferation%2520identified%2520by%2520high-throughput%2520screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D1669%26epage%3D1672%26doi%3D10.1021%2Fja309304m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armitage, I.</span>; <span class="NLM_string-name">Bourland, M. E.</span>; <span class="NLM_string-name">Boyle, C. J. S.</span>; <span class="NLM_string-name">Cooper, M. I.</span>; <span class="NLM_string-name">Ferdous, A. J.</span>; <span class="NLM_string-name">Langston, M.</span></span> International Patent Application <span class="NLM_patent">WO2010059239</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Armitage%2C+I.%3B+Bourland%2C+M.+E.%3B+Boyle%2C+C.+J.+S.%3B+Cooper%2C+M.+I.%3B+Ferdous%2C+A.+J.%3B+Langston%2C+M.+International+Patent+Application+WO2010059239%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DArmitage%26aufirst%3DI.%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoGrasso, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">Pyridopyrimidinone Derivatives as potent and selective c-Jun N-terminal kinase (JNK) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1021/ml500474d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500474d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=413-418&author=K.+Zhengauthor=C.+M.+Parkauthor=S.+Iqbalauthor=P.+Hernandezauthor=H.+Parkauthor=P.+V.+LoGrassoauthor=Y.+Feng&title=Pyridopyrimidinone+Derivatives+as+potent+and+selective+c-Jun+N-terminal+kinase+%28JNK%29+inhibitors&doi=10.1021%2Fml500474d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridopyrimidinone derivatives as potent and selective c-Jun N-terminal kinase (JNK) inhibitors</span></div><div class="casAuthors">Zheng, Ke; Park, Chul Min; Iqbal, Sarah; Hernandez, Pamela; Park, HaJeung; LoGrasso, Philip V.; Feng, Yangbo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-418</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of 2-aminopyridopyrimidinone based JNK (c-jun N-terminal kinase) inhibitors were discovered and developed.  Structure-activity relationships (SARs) were systematically developed utilizing biochem. and cell based assays and in vitro and in vivo drug metab. and pharmacokinetic (DMPK) studies.  Through the optimization of lead compd. 1, several potent and selective JNK inhibitors with high oral bioavailability were developed.  (trans)-1-Isopropyl-3-[4-(8-isopropyl-7-oxo-7,8-dihydro-pyrido[2,3-d]pyrimidin-2-yl-amino)-cyclohexyl]urea (I), was a potent JNK3 inhibitor (IC50 = 15 nM), had high selectivity against p38 (IC50 > 10 μM), had high potency in functional cell based assays, and had high stability in human liver microsome (t1/2 = 76 min), a clean CYP-450 inhibition profile, and excellent oral bioavailability (%F = 87).  Moreover, cocrystal structures of (I) and (trans)-1-[4-(8-Cyclopentyl-7-oxo-7,8-dihydro-pyrido[2,3-d]pyrimidin-2-ylamino)-cyclohexyl]-3-isopropyl-urea in JNK3 were solved at 2.0 Å.  These structures elucidated the binding mode (Type-I binding) and can pave the way for further inhibitor design of this pyridopyrimidinone scaffold for JNK inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-GYGot2Yf7bVg90H21EOLACvtfcHk0lidxJDJYaYpew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFCgurc%253D&md5=7bd22deaf4386f2ff02adafba62056cf</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fml500474d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500474d%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DC.%2BM.%26aulast%3DIqbal%26aufirst%3DS.%26aulast%3DHernandez%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLoGrasso%26aufirst%3DP.%2BV.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DPyridopyrimidinone%2520Derivatives%2520as%2520potent%2520and%2520selective%2520c-Jun%2520N-terminal%2520kinase%2520%2528JNK%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D413%26epage%3D418%26doi%3D10.1021%2Fml500474d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F.</span>; <span class="NLM_string-name">Goldstein, D. M.</span>; <span class="NLM_string-name">Han, X.</span>; <span class="NLM_string-name">Park, J.</span>; <span class="NLM_string-name">Sjogren, E. B.</span>; <span class="NLM_string-name">Reuter, D. C.</span>; <span class="NLM_string-name">Hogg, J. H.</span>; <span class="NLM_string-name">Gong, L.</span>; <span class="NLM_string-name">Dillon, M. P.</span>; <span class="NLM_string-name">Eustache, F.</span></span> <i>Pyrimidinyl pyridone inhibitors of kinases</i>. U.S. Patent <span class="NLM_patent">US2009/270389A1</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=F.+Brookfield&author=D.+M.+Goldstein&author=X.+Han&author=J.+Park&author=E.+B.+Sjogren&author=D.+C.+Reuter&author=J.+H.+Hogg&author=L.+Gong&author=M.+P.+Dillon&author=F.+Eustache&title=Pyrimidinyl+pyridone+inhibitors+of+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrookfield%26aufirst%3DF.%26jtitle%3DPyrimidinyl%2520pyridone%2520inhibitors%2520of%2520kinases%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX','PDB','7AVX'); return false;">PDB: 7AVX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY','PDB','7AVY'); return false;">PDB: 7AVY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ','PDB','7AVZ'); return false;">PDB: 7AVZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0','PDB','7AW0'); return false;">PDB: 7AW0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1','PDB','7AW1'); return false;">PDB: 7AW1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3','PDB','7AW3'); return false;">PDB: 7AW3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4','PDB','7AW4'); return false;">PDB: 7AW4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2','PDB','7AW2'); return false;">PDB: 7AW2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i83"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01904">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73451"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01904?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01904</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Standard deviations of measurements; Mer activity in high-ATP assay; kinase selectivity profiles of <b>12</b>–<b>14</b>; crystallographical details; assay descriptions (Axl, Flt GLO assays); selection conditions used in DECL screening and prevalence tables for families A, B, and C; purity data for compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, CSV file PDB accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVX">7AVX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVY">7AVY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AVZ">7AVZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW0">7AW0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW1">7AW1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW2">7AW2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW3">7AW3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AW4">7AW4</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_001.pdf">jm0c01904_si_001.pdf (1.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01904/suppl_file/jm0c01904_si_002.csv">jm0c01904_si_002.csv (2.12 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01904%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-6%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01904" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799171e79c32398","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
