
<html lang="en"     class="pb-page"  data-request-id="d7a094cb-8cb0-4a40-bb9d-de1c1976c4bf"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c01000"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Changing for the Better: Discovery of the Highly Potent and Selective CDK9 Inhibitor VIP152 Suitable for Once Weekly Intravenous Dosing for the Treatment of Cancer" /></meta><meta name="dc.Creator" content="Ulrich  Lücking" /></meta><meta name="dc.Creator" content="Dirk  Kosemund" /></meta><meta name="dc.Creator" content="Niels  Böhnke" /></meta><meta name="dc.Creator" content="Philip  Lienau" /></meta><meta name="dc.Creator" content="Gerhard  Siemeister" /></meta><meta name="dc.Creator" content="Karsten  Denner" /></meta><meta name="dc.Creator" content="Rolf  Bohlmann" /></meta><meta name="dc.Creator" content="Hans  Briem" /></meta><meta name="dc.Creator" content="Ildiko  Terebesi" /></meta><meta name="dc.Creator" content="Ulf  Bömer" /></meta><meta name="dc.Creator" content="Martina  Schäfer" /></meta><meta name="dc.Creator" content="Stuart  Ince" /></meta><meta name="dc.Creator" content="Dominik  Mumberg" /></meta><meta name="dc.Creator" content="Arne  Scholz" /></meta><meta name="dc.Creator" content="Raquel  Izumi" /></meta><meta name="dc.Creator" content="Stuart  Hwang" /></meta><meta name="dc.Creator" content="Franz  von Nussbaum" /></meta><meta name="dc.Description" content="Selective inhibition of exclusively transcription-regulating positive transcription elongation factor b/CDK9 is a promising new approach in cancer therapy. Starting from atuveciclib, the first sele..." /></meta><meta name="Description" content="Selective inhibition of exclusively transcription-regulating positive transcription elongation factor b/CDK9 is a promising new approach in cancer therapy. Starting from atuveciclib, the first sele..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 15, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c01000" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c01000" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c01000" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c01000" /></link>
        
    
    

<title>Changing for the Better: Discovery of the Highly Potent and Selective CDK9 Inhibitor VIP152 Suitable for Once Weekly Intravenous Dosing for the Treatment of Cancer | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c01000" /></meta><meta property="og:title" content="Changing for the Better: Discovery of the Highly Potent and Selective CDK9 Inhibitor VIP152 Suitable for Once Weekly Intravenous Dosing for the Treatment of Cancer" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0014.jpeg" /></meta><meta property="og:description" content="Selective inhibition of exclusively transcription-regulating positive transcription elongation factor b/CDK9 is a promising new approach in cancer therapy. Starting from atuveciclib, the first selective CDK9 inhibitor to enter clinical development, lead optimization efforts aimed at identifying intravenously (iv) applicable CDK9 inhibitors with an improved therapeutic index led to the discovery of the highly potent and selective clinical candidate VIP152. The evaluation of various scaffold hops was instrumental in the identification of VIP152, which is characterized by the underexplored benzyl sulfoximine group. VIP152 exhibited the best preclinical overall profile in vitro and in vivo, including high efficacy and good tolerability in xenograft models in mice and rats upon once weekly iv administration. VIP152 has entered clinical trials for the treatment of cancer with promising longterm, durable monotherapy activity in double-hit diffuse large B-cell lymphoma patients." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c01000"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01000">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c01000&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c01000&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c01000&amp;href=/doi/10.1021/acs.jmedchem.1c01000" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00781" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c01159" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Changing for the Better: Discovery of the Highly Potent and Selective CDK9 Inhibitor VIP152 Suitable for Once Weekly Intravenous Dosing for the Treatment of Cancer</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ulrich Lücking</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrich Lücking</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrich++L%C3%BCcking">Ulrich Lücking</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2466-5800" title="Orcid link">https://orcid.org/0000-0003-2466-5800</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dirk Kosemund</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dirk Kosemund</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dirk++Kosemund">Dirk Kosemund</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Niels Böhnke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Niels Böhnke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Niels++B%C3%B6hnke">Niels Böhnke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philip Lienau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip Lienau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip++Lienau">Philip Lienau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gerhard Siemeister</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerhard Siemeister</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerhard++Siemeister">Gerhard Siemeister</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karsten Denner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karsten Denner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karsten++Denner">Karsten Denner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rolf Bohlmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rolf Bohlmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rolf++Bohlmann">Rolf Bohlmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hans Briem</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hans Briem</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hans++Briem">Hans Briem</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ildiko Terebesi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ildiko Terebesi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ildiko++Terebesi">Ildiko Terebesi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulf Bömer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulf Bömer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulf++B%C3%B6mer">Ulf Bömer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martina Schäfer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martina Schäfer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martina++Sch%C3%A4fer">Martina Schäfer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stuart Ince</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stuart Ince</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stuart++Ince">Stuart Ince</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominik Mumberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominik Mumberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominik++Mumberg">Dominik Mumberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Arne Scholz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Arne Scholz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Arne++Scholz">Arne Scholz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raquel Izumi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raquel Izumi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vincerx Pharma, Inc., 260 Sheridan Avenue, Suite 400, Palo Alto, California 94306, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raquel++Izumi">Raquel Izumi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0895-2597" title="Orcid link">https://orcid.org/0000-0002-0895-2597</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stuart Hwang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stuart Hwang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vincerx Pharma, Inc., 260 Sheridan Avenue, Suite 400, Palo Alto, California 94306, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ff8c8b8a9e8d8bd197889e9198bf8996919c9a8d87d19c9092"><span class="__cf_email__" data-cfemail="2d5e59584c5f5903455a4c434a6d5b44434e485f55034e4240">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stuart++Hwang">Stuart Hwang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Franz von Nussbaum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Franz von Nussbaum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceuticals, Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Franz++von+Nussbaum">Franz von Nussbaum</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c01000&amp;href=/doi/10.1021%2Facs.jmedchem.1c01000" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 15, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 June 2021</li><li><span class="item_label"><b>Published</b> online</span>15 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c01000" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c01000</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DUlrich%2BL%25C3%25BCcking%252C%2BDirk%2BKosemund%252C%2BNiels%2BB%25C3%25B6hnke%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c01000%26title%3DChanging%2Bfor%2Bthe%2BBetter%253A%2BDiscovery%2Bof%2Bthe%2BHighly%2BPotent%2Band%2BSelective%2BCDK9%2BInhibitor%2BVIP152%2BSuitable%2Bfor%2BOnce%2BWeekly%2BIntravenous%2BDosing%2Bfor%2Bthe%2BTreatment%2Bof%2BCancer%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D24%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c01000"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">813</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c01000" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Changing for the Better: Discovery of the Highly Potent and Selective CDK9 Inhibitor VIP152 Suitable for Once Weekly Intravenous Dosing for the Treatment of Cancer&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ulrich&quot;,&quot;last_name&quot;:&quot;Lücking&quot;},{&quot;first_name&quot;:&quot;Dirk&quot;,&quot;last_name&quot;:&quot;Kosemund&quot;},{&quot;first_name&quot;:&quot;Niels&quot;,&quot;last_name&quot;:&quot;Böhnke&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;Lienau&quot;},{&quot;first_name&quot;:&quot;Gerhard&quot;,&quot;last_name&quot;:&quot;Siemeister&quot;},{&quot;first_name&quot;:&quot;Karsten&quot;,&quot;last_name&quot;:&quot;Denner&quot;},{&quot;first_name&quot;:&quot;Rolf&quot;,&quot;last_name&quot;:&quot;Bohlmann&quot;},{&quot;first_name&quot;:&quot;Hans&quot;,&quot;last_name&quot;:&quot;Briem&quot;},{&quot;first_name&quot;:&quot;Ildiko&quot;,&quot;last_name&quot;:&quot;Terebesi&quot;},{&quot;first_name&quot;:&quot;Ulf&quot;,&quot;last_name&quot;:&quot;Bömer&quot;},{&quot;first_name&quot;:&quot;Martina&quot;,&quot;last_name&quot;:&quot;Schäfer&quot;},{&quot;first_name&quot;:&quot;Stuart&quot;,&quot;last_name&quot;:&quot;Ince&quot;},{&quot;first_name&quot;:&quot;Dominik&quot;,&quot;last_name&quot;:&quot;Mumberg&quot;},{&quot;first_name&quot;:&quot;Arne&quot;,&quot;last_name&quot;:&quot;Scholz&quot;},{&quot;first_name&quot;:&quot;Raquel&quot;,&quot;last_name&quot;:&quot;Izumi&quot;},{&quot;first_name&quot;:&quot;Stuart&quot;,&quot;last_name&quot;:&quot;Hwang&quot;},{&quot;first_name&quot;:&quot;Franz&quot;,&quot;last_name&quot;:&quot;von Nussbaum&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c01000&quot;},&quot;abstract&quot;:&quot;Selective inhibition of exclusively transcription-regulating positive transcription elongation factor b/CDK9 is a promising new approach in cancer therapy. Starting from atuveciclib, the first selective CDK9 inhibitor to enter clinical development, lead optimization efforts aimed at identifying intravenously (iv) applicable CDK9 inhibitors with an improved therapeutic index led to the discovery of the highly potent and selective clinical candidate VIP152. The evaluation of various scaffold hops was instrumental in the identification of VIP152, which is characterized by the underexplored benzyl sulfoximine group. VIP152 exhibited the best preclinical overall profile in vitro and in vivo, including high efficacy and good tolerability in xenograft models in mice and rats upon once weekly iv administration. VIP152 has entered clinical trials for the treatment of cancer with promising longterm, durable monotherapy activity in double-hit diffuse large B-cell lymphoma patients.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c01000&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c01000" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c01000&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c01000" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c01000&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c01000" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c01000&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c01000&amp;href=/doi/10.1021/acs.jmedchem.1c01000" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c01000" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c01000" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c01000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c01000%26sid%3Dliteratum%253Aachs%26pmid%3D34264057%26genre%3Darticle%26aulast%3DL%25C3%25BCcking%26date%3D2021%26atitle%3DChanging%2Bfor%2Bthe%2BBetter%253A%2BDiscovery%2Bof%2Bthe%2BHighly%2BPotent%2Band%2BSelective%2BCDK9%2BInhibitor%2BVIP152%2BSuitable%2Bfor%2BOnce%2BWeekly%2BIntravenous%2BDosing%2Bfor%2Bthe%2BTreatment%2Bof%2BCancer%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Selective inhibition of exclusively transcription-regulating positive transcription elongation factor b/CDK9 is a promising new approach in cancer therapy. Starting from atuveciclib, the first selective CDK9 inhibitor to enter clinical development, lead optimization efforts aimed at identifying intravenously (iv) applicable CDK9 inhibitors with an improved therapeutic index led to the discovery of the highly potent and selective clinical candidate VIP152. The evaluation of various scaffold hops was instrumental in the identification of VIP152, which is characterized by the underexplored benzyl sulfoximine group. VIP152 exhibited the best preclinical overall profile in vitro and in vivo, including high efficacy and good tolerability in xenograft models in mice and rats upon once weekly iv administration. VIP152 has entered clinical trials for the treatment of cancer with promising longterm, durable monotherapy activity in double-hit diffuse large B-cell lymphoma patients.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The cyclin-dependent kinase (CDK) family consists of closely related serine/threonine kinases with specialized functions related to the regulation of the cell cycle (CDKs 1, 2, 4, and 6) or transcription (CDKs 7, 8, 9, 12, 13, and 19).<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Due to their critical role in cell division and gene transcription, CDKs have been considered strong prospective targets for a new generation of anticancer drugs.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> However, while multi-CDK inhibitors such as alvocidib,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> dinaciclib,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> or roniciclib<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> revealed promising results in xenograft models, they have displayed limited treatment responses and significant adverse events in clinical trials, with the exception of certain hematological malignancies, which have been attributed to their CDK9 inhibitory activity.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> CDK9 associates with cyclins T1, T2a, and T2b to form the heterodimer positive transcription elongation factor b (PTEFb), which activates transcription via phosphorylation of RNA polymerase II.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Overexpression and/or hyperactivity of CDK9 is associated with a variety of human pathological conditions such as cancer, virally induced infectious diseases, and cardiovascular diseases.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> PTEFb-/CDK9-mediated transcription of short-lived antiapoptotic survival proteins and oncogenes such as MCL-1 and MYC plays a critical role in a variety of cancers, especially in several hematological malignancies.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Selective, transient inhibition of CDK9 leads to a rapid depletion of short-lived messenger RNA (mRNA) transcripts of these important survival proteins and oncogenes, providing a promising rationale for cancer therapy. However, the discovery of selective CDK9 inhibitors is challenging due to the high degree of homology among the kinases of the CDK family. So far, four selective CDK9 inhibitors, atuveciclib (BAY 1143572),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> AZD4573,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> KB-0742,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and VIP152 (previously BAY 1251152),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> are reported to have entered clinical trials (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). We previously disclosed the preclinical profile of atuveciclib, the first selective, orally available CDK9 inhibitor to enter clinical development.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In vitro, atuveciclib shows low nanomolar CDK9 inhibitory activity, high selectivity against other CDKs, and antiproliferative activity against various tumor cell lines with submicromolar IC<sub>50</sub> values. Moreover, atuveciclib demonstrated single-agent in vivo efficacy at tolerated doses in various xenograft tumor models upon oral administration. To fully explore future treatment options using selective CDK9 inhibitors, we initiated a follow-up program to identify novel selective CDK9 inhibitors suitable for intravenous (iv) administration in patients. We now outline the discovery of the highly potent and selective CDK9 inhibitor VIP152, which has entered phase 1 clinical trials in patients.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the clinical CDK9 inhibitors atuveciclib, AZD4573, KB-0742, and VIP152.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45106" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45106" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Atuveciclib as a Starting Point for the Identification of the New Lead Structure BAY-332</h3><div class="NLM_p">Atuveciclib demonstrated potent and highly selective CDK9 inhibitor in our biochemical in vitro assay under low adenosine 5′-triphosphate (ATP) conditions (ATP concentration: 10 μM; IC<sub>50</sub> CDK9/CycT1: 13 nM; ratio of IC<sub>50</sub> values CDK2/CDK9: 100, see <named-content content-type="anchor" rid="tbl1" type="simple"></named-content><named-content content-type="anchor" rid="tbl2" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>; and selectivity within the CDK family: >145).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> However, under high ATP conditions (ATP concentration: 2 mM), which reflect the approximate cellular ATP concentration in humans, a significant drop in CDK9 inhibitory activity, by a factor of ca. 30, was recorded (IC<sub>50</sub> CDK9/CycT1: 380 nM). Atuveciclib revealed promising antiproliferative activity against a variety of human cancer cell lines, for example, against A2780 (IC<sub>50</sub>: 380 nM) and MOLM-13 (IC<sub>50</sub>: 310 nM) cells, and high aqueous solubility (<i>S</i><sub>w</sub>, pH 6.5: 479 mg/L). In in vivo pharmacokinetic (PK) studies in rats, atuveciclib showed low blood clearance (CL<sub>b</sub>: 1.1 L/h/kg), a moderate to high volume of distribution (<i>V</i><sub>ss</sub>: 1.0 L/kg), and a short half-life (<i>t</i><sub>1/2</sub>: 0.6 h) (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Moreover, atuveciclib demonstrated efficacy in human xenograft tumor models [e.g., models of acute myeloid leukemia (AML) in mice and rats]. The compound is active upon once daily dosing, as well as upon certain intermittent dosing schedules (3 on/2 off). Due to the promising overall profile in vitro and in vivo, atuveciclib entered clinical trials in patients with advanced cancer and leukemia (NCT01938638 and NCT02345382). In a phase 1 study to determine the safety, tolerability, PKs, maximum tolerated dose (MTD), and preliminary antitumor activity in patients with advanced cancer, neutropenia became the most prominent adverse event upon treatment with atuveciclib.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Lead Optimization Efforts at the Hinge Binding Core and the Corresponding Key In Vitro Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0010.gif" alt="" id="GRAPHIC-d7e408-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0011.gif" alt="" id="gr9" /></img><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Selected Examples of Lead Optimization Efforts at the 2-Position of the 5-Fluoropyrimidine Scaffold and the Corresponding Key In Vitro Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0012.gif" alt="" id="GRAPHIC-d7e416-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0013.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Cells were treated with test compounds for 96 h.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The solid state of the test compounds was not characterized.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Determined by a high-throughput screening method using 1 mM dimethyl sulfoxide (DMSO) stock solutions.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">n.d.: not determined.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Determined by equilibrium shake flask solubility assay from powder.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Properties of the CDK9 Inhibitors Atuveciclib, BAY-332, <b>25</b>, and VIP152</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">atuveciclib</th><th class="colsep0 rowsep0" align="center">BAY-332</th><th class="colsep0 rowsep0" align="center"><b>25</b></th><th class="colsep0 rowsep0" align="center">VIP152</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK9/CycT1; IC<sub>50</sub> [nM] low ATP</td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">selectivity vs CDK2, ratio of low ATP IC<sub>50</sub> values</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="center">54</td><td class="colsep0 rowsep0" align="center">154</td><td class="colsep0 rowsep0" align="center">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK9/CycT1; IC<sub>50</sub> [nM] high ATP</td><td class="colsep0 rowsep0" align="center">380</td><td class="colsep0 rowsep0" align="center">37</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">selectivity vs CDK2, ratio of high ATP IC<sub>50</sub> values</td><td class="colsep0 rowsep0" align="center">53</td><td class="colsep0 rowsep0" align="center">107</td><td class="colsep0 rowsep0" align="center">785</td><td class="colsep0 rowsep0" align="center">1033</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A2780; IC<sub>50</sub> [nM]<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">380</td><td class="colsep0 rowsep0" align="center">57</td><td class="colsep0 rowsep0" align="center">126</td><td class="colsep0 rowsep0" align="center">45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-13; IC<sub>50</sub> [nM]<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">310</td><td class="colsep0 rowsep0" align="center">85</td><td class="colsep0 rowsep0" align="center">80</td><td class="colsep0 rowsep0" align="center">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S</i><sub>w</sub>, pH 6.5 [mg/L]<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">479</td><td class="colsep0 rowsep0" align="center">24</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S</i><sub>w</sub>, pH 4 [mg/L]<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">309</td><td class="colsep0 rowsep0" align="center">30</td><td class="colsep0 rowsep0" align="center">43</td><td class="colsep0 rowsep0" align="center">699</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>P</i><sub>app</sub> A–B [nm/s]</td><td class="colsep0 rowsep0" align="center">35</td><td class="colsep0 rowsep0" align="center">168</td><td class="colsep0 rowsep0" align="center">26</td><td class="colsep0 rowsep0" align="center">166</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">efflux ratio</td><td class="colsep0 rowsep0" align="center">6.1</td><td class="colsep0 rowsep0" align="center">0.61</td><td class="colsep0 rowsep0" align="center">9.3</td><td class="colsep0 rowsep0" align="center">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>b</sub> ratHep [L/h/kg]</td><td class="colsep0 rowsep0" align="center">0.17</td><td class="colsep0 rowsep0" align="center">1.9</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>b</sub> ratLM [L/h/kg]</td><td class="colsep0 rowsep0" align="center">0.15</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>b</sub> rat in vivo, iv [L/h/kg]</td><td class="colsep0 rowsep0" align="center">1.1</td><td class="colsep0 rowsep0" align="center">1.3</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>ss</sub> rat in vivo, iv [L/kg]</td><td class="colsep0 rowsep0" align="center">1.0</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">0.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> rat in vivo, iv [h]</td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="center">1.0</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">blood/plasma ratio (rat)</td><td class="colsep0 rowsep0" align="center">1.1</td><td class="colsep0 rowsep0" align="center">0.88</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">0.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP inhibition [μM]</td><td class="colsep0 rowsep0" align="center">>20</td><td class="colsep0 rowsep0" align="center">2C8: 9.2, 2C19: 15, all others: >20</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">>20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP1A2 induction</td><td class="colsep0 rowsep0" align="center">no (up to 370 μg/L)</td><td class="colsep0 rowsep0" align="center">no (up to 41 μg/L)</td><td class="colsep0 rowsep0" align="center">n.d.<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">no (up to 14 μg/L)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Cells were treated with test compounds for 96 h.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Determined by equilibrium shake flask solubility assay from powder. The solid state of the test compounds was not characterized.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">n.d.: not determined.</p></div></div></div><div class="NLM_p">Since inhibition of MCL-1 induces apoptosis in circulating neutrophils and other white blood cell types,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> permanent on-target inhibition of MCL-1 in circulating white blood cells by daily oral administration of atuveciclib is thought to be the main reason for the observed neutropenia in these patients. To fully explore future treatment options using selective CDK9 inhibitors, we initiated a follow-up program aimed at maximizing the therapeutic index. A highly potent and selective CDK9 inhibitor with a short half-life was envisaged to enable defined periods of acute but sufficient CDK9 inhibition for the desired antitumor effect without reducing the therapeutic window by prolonged transcription inhibition.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Moreover, switching from oral to iv administration was expected to reduce the variation in plasma levels and exposure often associated with oral administration of small-molecule kinase inhibitors<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and thus contribute to the desired increase of the therapeutic index.</div><div class="NLM_p">In preclinical studies, atuveciclib was rated not feasible for iv application in patients. In animal models, the compound revealed only limited efficacy after once weekly iv application. Moreover, the aqueous solubility of atuveciclib is not sufficient to enable the formulation of the corresponding predicted therapeutic human dose of the compound for iv administration in patients. Thus, we set out to identify a new generation of potent and highly selective CDK9 inhibitors suitable for intermittent iv treatment of patients with cancer.</div><div class="NLM_p">According to docking experiments, binding of atuveciclib to the hinge region of CDK9 is mediated by the triazine core and the aniline NH via two hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The methoxyphenyl substituent at the triazine points toward the ribose pocket, whereas the benzylic sulfoximine moiety is directed toward the exit of the ATP binding pocket.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking mode of atuveciclib in complex with CDK9. The compound was docked into a published X-ray complex of CDK9/CycT1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1">3MY1</a>). The aminotriazine moiety can form two hydrogen bonds to the hinge region of the kinase. For the substituted phenyl ring attached to the triazine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Sulfoximines, long neglected in drug discovery, have attracted significant interest in medicinal chemistry recently.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Sulfoximines are aza analogues of sulfones, but their properties differ significantly: the introduction of the nitrogen atom creates asymmetry and offers an additional point for substitution. The nitrogen atom is basic enough to allow metal ion coordination or salt formation. The heteroatoms bound to the sulfur are hydrogen bond acceptors, with NH sulfoximines having dual hydrogen bond donor/acceptor functionality. Moreover, sulfoximines have often exhibited high hydrolytic and metabolic stability, as well as favorable physicochemical properties.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> In the lead optimization approach that led to the identification of atuveciclib, we evaluated a broad variety of functional groups at the molecular position directed to the exit of the binding pocket, but the sulfoximine group offered the best overall profile.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Thus, we decided to keep the sulfoximine moiety at this position initially and rather started to investigate the structural modifications of the hinge binding core to improve the CDK9 inhibitory potency in vitro while maintaining high kinase selectivity. Since relative to sulfoximines, the synthetically less demanding sulfone analogues in general revealed comparable in vitro activity against CDK9 in the triazine inhibitor series; this evaluation was undertaken on the sulfone series. The sulfone analogue of atuveciclib, triazine <b>1</b>, was used as a point of reference (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Switching from the triazine core to pyridine <b>2</b> improved the CDK9 inhibitory activity in vitro under low and high ATP conditions (IC<sub>50</sub> values: 9 and 254 nM) but also reduced the selectivity against CDK2 (ratio of low ATP IC<sub>50</sub> values: 30). The selectivity against cell cycle kinase CDK2 was used as a first indicator for the overall selectivity within the family of CDKs. Introduction of a fluorine substituent at the 5-position of the pyridine core (<b>3</b>) further improved the activity against CDK9 (IC<sub>50</sub> values: 4 and 99 nM), while fluorine at the 3-position (<b>4</b>) reduced the in vitro activity significantly (IC<sub>50</sub> values: 144 and >10,000 nM). However, switching from triazine <b>1</b> to 5-fluoropyrimidine <b>5</b> as the central core substantially improved the CDK9 inhibitory activity under both low and high ATP conditions (IC<sub>50</sub> values: 3 and 7 nM) while slightly improving the selectivity against CDK2 (ratio of low ATP IC<sub>50</sub> values: 77). No further improvement of in vitro potency and selectivity against CDK2 was observed upon the introduction of a chloro (<b>6</b>), cyano (<b>7</b>), or trifluoromethyl (<b>8</b>) substituent at the 5-position.</div><div class="NLM_p">Based on the results from our optimization efforts at the hinge binding core in the sulfone series, the initial lead optimization efforts in the analogous sulfoximine series led to the identification of the new lead compound BAY-332 which is characterized by a 5-fluoropyrimidine scaffold and an additional fluorine atom at the 3-position of the benzylic sulfoximine substituent (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In comparison to atuveciclib, BAY-332<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> demonstrated significantly improved in vitro activity against CDK9 in the biochemical assays, especially under high ATP conditions [IC<sub>50</sub> values: 380 nM (atuveciclib) vs 37 nM (BAY-332), see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>]. The new lead compound also revealed improved selectivity against CDK2 based on the ratio of IC<sub>50</sub> values from the corresponding biochemical in vitro assays under high ATP conditions [ratio: 53 (atuveciclib) vs 107 (BAY-332)]. BAY-332 also exhibited improved antiproliferative potency in vitro, for example, against MOLM-13 cells [IC<sub>50</sub> values: 310 nM (atuveciclib) vs 85 nM (BAY-332)] and A2780 cells [IC<sub>50</sub> values: 380 nM (atuveciclib) vs 57 nM (BAY-332)]. In vitro PK studies with BAY-332 demonstrated moderate metabolic stability in rat hepatocytes (ratHep), resulting in a moderate predicted blood clearance (CL<sub>b</sub>) of 1.9 L/h/kg. In contrast to atuveciclib, we recorded a high permeability coefficient (<i>P</i><sub>app</sub> A–B) of 168 nm/s (atuveciclib: 35 nm/s) and a low efflux ratio of 0.61 (atuveciclib: 6.1) raising no concerns about a possible efflux transporter recognition of BAY-332. However, aqueous solubility at pH 6.5 of BAY-332 was significantly reduced relative to atuveciclib [<i>S</i><sub>w</sub>: 479 mg/L (atuveciclib) vs 24 mg/L (BAY-332)], presumably mainly driven by the switch from the polar triazine to the 5-fluoropyrimidine core. PK studies with BAY-332 in rats in vivo revealed a moderate blood clearance (CL<sub>b</sub>: 1.3 L/h/kg), a high volume of distribution (<i>V</i><sub>ss</sub>: 1.6 L/kg), and an intermediate half-life (<i>t</i><sub>1/2</sub>: 1.0 h). In comparison to BAY-332, its enantiomer (<b>9</b>) revealed very similar in vitro properties, well within the limits of measurement accuracy.</div><div class="NLM_p">In the MOLM-13 human AML model in mice, BAY-332 administered iv either once daily (QD, 15 mg/kg), once every 3 days (Q3D, 35 mg/kg), or once every 5 days (Q5D, 65 mg/kg) demonstrated potent antitumor efficacy with T/C ratios of 0.26 (<i>p</i> = 0.006), 0.26 (<i>p</i> = 0.007), and 0.34 (<i>p</i> = 0.029), respectively (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Additionally, there was good tolerability upon treatment with BAY-332, with a body weight change of less than 10%, but two fatal toxicities upon Q3D and Q5D treatment in these unoptimized dosing schedules. Nevertheless, due to the insufficient solubility properties of BAY-332, formulation of the predicted human dose for once weekly iv treatment of patients was rated not feasible.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antitumor efficacy of BAY-332 in the MOLM-13 human AML model in mice. Tumor growth in mice treated iv with either vehicle or BAY-332 once daily (QD), once every 3 days (Q3D), or once every 5 days (Q5D). Treatments were started 5 days after tumor cell inoculation. Statistical significances were calculated using the mean tumor volumes at the time point when the vehicle group was sacrificed. Asterisks indicate statistical significance relative to the vehicle control (*<i>p</i> < 0.05 and **<i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antitumor Efficacy of BAY-332 and VIP152 in the MOLM-13 Human AML Model in Nude Mice and Athymic Rats</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">animal model</th><th class="colsep0 rowsep0" align="center">treatment</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center">schedule</th><th class="colsep0 rowsep0" align="center">T/C<sub>volume</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">fatal toxicity</th><th class="colsep0 rowsep0" align="center" char=".">maximum body weight loss<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-13 AML xenografts in mice</td><td class="colsep0 rowsep0" align="left">vehicle</td><td class="colsep0 rowsep0" align="center" char=".">0</td><td class="colsep0 rowsep0" align="center">QD</td><td class="colsep0 rowsep0" align="center">1.00</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">BAY-332</td><td class="colsep0 rowsep0" align="center" char=".">15</td><td class="colsep0 rowsep0" align="center">QD</td><td class="colsep0 rowsep0" align="center">0.26**</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">4.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">BAY-332</td><td class="colsep0 rowsep0" align="center" char=".">35</td><td class="colsep0 rowsep0" align="center">Q3D</td><td class="colsep0 rowsep0" align="center">0.26**</td><td class="colsep0 rowsep0" align="center">2/12</td><td class="colsep0 rowsep0" align="center" char=".">5.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">BAY-332</td><td class="colsep0 rowsep0" align="center" char=".">65</td><td class="colsep0 rowsep0" align="center">Q5D</td><td class="colsep0 rowsep0" align="center">0.34*</td><td class="colsep0 rowsep0" align="center">2/12</td><td class="colsep0 rowsep0" align="center" char=".">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-13 AML xenografts in rats</td><td class="colsep0 rowsep0" align="left">vehicle</td><td class="colsep0 rowsep0" align="center" char=".">0</td><td class="colsep0 rowsep0" align="center">Q3D</td><td class="colsep0 rowsep0" align="center">1.00</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">VIP152</td><td class="colsep0 rowsep0" align="center" char=".">3</td><td class="colsep0 rowsep0" align="center">Q3D</td><td class="colsep0 rowsep0" align="center">0.11***</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">VIP152</td><td class="colsep0 rowsep0" align="center" char=".">3.75</td><td class="colsep0 rowsep0" align="center">Q5D</td><td class="colsep0 rowsep0" align="center">0.08***</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">VIP152</td><td class="colsep0 rowsep0" align="center" char=".">4.5</td><td class="colsep0 rowsep0" align="center">QW</td><td class="colsep0 rowsep0" align="center">0.15***</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-13 AML xenografts in mice</td><td class="colsep0 rowsep0" align="left">vehicle</td><td class="colsep0 rowsep0" align="center" char=".">0</td><td class="colsep0 rowsep0" align="center">QD</td><td class="colsep0 rowsep0" align="center">1.00</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">VIP152</td><td class="colsep0 rowsep0" align="center" char=".">12.5</td><td class="colsep0 rowsep0" align="center">QW</td><td class="colsep0 rowsep0" align="center">0.20**</td><td class="colsep0 rowsep0" align="center">0/12</td><td class="colsep0 rowsep0" align="center" char=".">4.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Treatment-to-control (T/C) ratios and statistical significances were calculated using the mean tumor volumes at the time point when the vehicle group was sacrificed. Asterisks indicate the statistical significance relative to the vehicle control (*<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001).</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">The maximum mean body weight loss expressed as a percentage of the starting weight of the animal. Weight loss greater than 20% is considered toxic.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Lead Optimization in the 5-Fluoropyrimidine Series</h3><div class="NLM_p last">To enable formulation of a new generation of CDK9 inhibitors for intermittent iv administration in patients, we set out to further optimize the biochemical and cellular in vitro potency to reduce the required therapeutic dose. Complementarily, we also aimed to improve the solubility properties. In the 5-fluoropyrimidine series, we focused on structural variation of the substituent at the 2-position, which is directed to the exit of the ATP binding pocket. The introduction of polar substituents at this position to increase aqueous solubility was envisaged to be more feasible than for the aromatic substituent at the 4-position pointed toward the ribose pocket. Moreover, in the lead optimization approach to atuveciclib, the structural variation of the ribose pocket substituent had revealed a steep structure–activity relationship with respect to selectivity against CDK2 and in vitro potency.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> presents selected examples of our extensive lead optimization efforts at the 2-position of the pyrimidine scaffold.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Various structural alternatives for the fluorine at the phenyl 3-position were feasible with respect to in vitro potency. Exchange of the fluorine atom for a methoxy substituent (<b>10</b>), for instance, increased the CDK9 inhibitory activity and selectivity against CDK2, though reduced solubility. Introduction of a second fluorine atom at the 4-position (<b>11</b>) did not impact in vitro potency significantly but increased metabolic instability in rat hepatocytes. Use of an <i>N</i>-cyano sulfoximine group in combination with the additional fluorine at the 4-position of the phenyl substituent (<b>12</b>) improved the biochemical and cellular activity significantly but further reduced the metabolic stability. Moreover, the CDK9 inhibitor <b>12</b> revealed very low aqueous solubility. Introduction of an additional nitrogen atom into the aromatic substituent at the 2-position was promising with respect to the CDK9 inhibitory activity, selectivity against CDK2, and aqueous solubility, but pyridine <b>13</b> revealed low permeability (<i>P</i><sub>app</sub> A–B: 23 nm/s) and a high efflux ratio (>10) in the bidirectional Caco-2 permeability assay, raising concerns about transporter recognition. Introduction of a polar hydroxy group into the alkyl substituent of the <i>N</i>-cyano sulfoximine group (<b>14</b>) led to significant improvement of CDK9 inhibitory activity and selectivity against CDK2 over BAY-332. On the other hand, hydroxy derivative <b>14</b> demonstrated only moderate antiproliferative activity against A2780 cells in vitro, low aqueous solubility, and low metabolic stability. Introduction of an NH<sub>2</sub> group into the alkyl chain in combination with an NH sulfoximine group (<b>15</b>) improved solubility significantly (<i>S</i><sub>w</sub>, pH 6.5: 294 mg/L); however, amine <b>15</b> revealed no permeability (<i>P</i><sub>app</sub> A–B: 0 nm/s) through the Caco-2 cell monolayer. The aza analogue of sulfoximine BAY-332, sulfondiimine <b>16</b>, was prepared with the hope for similar potency and selectivity but further improved aqueous solubility. Moreover, no chiral separation of enantiomers is required in the case of achiral sulfondiimine <b>16</b>. In comparison to BAY-332, matched analogue <b>16</b> revealed reduced in vitro activity against CDK9 (IC<sub>50</sub>: 104 nM), as well as reduced antiproliferative activity against A2780 cells (IC<sub>50</sub>: 184 nM), yet similar selectivity against CDK2 (ratio of high ATP IC<sub>50</sub> values: 101) and significantly improved aqueous solubility at pH 6.5 (<i>S</i><sub>w</sub>: 155 mg/L). In the Caco-2 assay, sulfondiimine <b>16</b> demonstrated a reduced permeability coefficient [<i>P</i><sub>app</sub> A–B: 168 nm/s (BAY-332) vs 50 nm/s (<b>16</b>)] and an increased efflux ratio [0.61 (BAY-332) vs 5.6 (<b>16</b>)]. Moreover, sulfoximine BAY-332 and matched sulfondiimine <b>16</b> displayed comparable metabolic stability in rat hepatocytes in vitro. In a subsequent rat PK study in vivo, an increased volume of distribution (<i>V</i><sub>ss</sub>: 2.2 L/kg) and significantly increased blood clearance (CL<sub>b</sub>: 3.3 L/h/kg) as well as a short half-life (<i>t</i><sub>1/2</sub>: 0.6 h) of analogue <b>16</b> relative to BAY-332 were recorded. Evaluation of non-sulfur-based functional groups at the exit of the ATP binding pocket did not lead to the identification of a compound with a suitable overall profile, similar to the findings in the lead optimization approach that led to atuveciclib.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Benzylamine <b>17</b>, for instance, demonstrated reduced in vitro activity in the biochemical and cellular assays compared to BAY-332, as well as low metabolic stability and no improvement of aqueous solubility.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Additional Scaffold Hops Leading to the Clinical Candidate</h3><div class="NLM_p">Since our lead optimization efforts to identify a compound with a superior preclinical overall profile to lead compound BAY-332 proved fruitless in the 5-fluoropyrimidine series, we turned our attention again to the evaluation of additional scaffold modifications. However, this time we also incorporated the neighboring aromatic substituent at the NH position in our design. The idea was to evaluate a shift of the nitrogen of the 5-fluoropyrimidine scaffold, which is not involved in hydrogen bonding to the hinge region of the ATP binding pocket, into the adjacent aromatic ring to yield <i>N</i>-(pyridin-2-yl)pyridin-2-amine <b>31</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). In the triazine series, the same strategy leads to <i>N</i>-(pyridin-2-yl)pyrimidin-4-amine <b>24</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking modes of the 5-fluoropyrimidine BAY-332 and the <i>N</i>-(pyridin-2-yl)pyridin-2-amine VIP152 in complex with CDK9. The compounds were docked into a published X-ray complex of CDK9/CycT1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1">3MY1</a>). BAY-332: the aminopyrimidine moiety can form two hydrogen bonds to the hinge region of the kinase. An additional hydrogen bond is formed by the sulfoximine NH to the backbone carbonyl group of Glu107. For the substituted phenyl ring attached to the pyrimidine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated. VIP152: the central aminopyridine moiety can form two hydrogen bonds to the hinge region of the kinase. An additional hydrogen bond is formed by the sulfoximine NH to the backbone carbonyl group of Glu107. For the substituted phenyl ring attached to the central pyridine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>N</i>-(Pyridin-2-yl)pyrimidin-4-amines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub> (aq), 1,2-dimethoxyethane (DME), 90 °C, 150 min, and 79%; (b) Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 22 h, and 60%; (c) CF<sub>3</sub>C(O)NH<sub>2</sub>, NaO<i>t</i>-Bu, 1,3-dibromo-5,5-dimethylhydantoin, 1,4-dioxane, tetrahydrofuran (THF), 10 °C, and 63%; (d) KMnO<sub>4</sub>, acetone, 50 °C, and crude; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, room temperature (rt), and 15% (over two steps); and (f) chiral preparative HPLC.</p></p></figure><div class="NLM_p last">After a successful synthesis (see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), the enantiomers of <i>N</i>-(pyridin-2-yl)pyrimidin-4-amine <b>24</b> exhibited very similar in vitro properties, much the same as the results previously recorded with the corresponding enantiomers in the triazine and 5-fluoropyrimidine series. The slightly more potent enantiomer <b>25</b> revealed significantly improved CDK9 inhibitory activity (IC<sub>50</sub>: 11 nM) and selectivity against CDK2 under high ATP conditions (ratio of IC<sub>50</sub> values: 785) in comparison to lead structure BAY-332 (IC<sub>50</sub>: 37 nM and ratio of IC<sub>50</sub> values: 107; <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, no increased antiproliferative activity against A2780 cells [IC<sub>50</sub> values: 126 nM (<b>25</b>) vs 57 nM (BAY-332)] and MOLM-13 cells [IC<sub>50</sub> values: 80 nM (<b>25</b>) vs 85 nM (BAY-332)] was recorded. Moreover, 4,6-disubstituted pyrimidine <b>25</b> displayed reduced aqueous solubility at pH 6.5 [<i>S</i><sub>w</sub>: 2 mg/L (<b>25</b>) vs 24 mg/L (BAY-332)], as well as low permeability (<i>P</i><sub>app</sub> A–B: 26 nm/s) and a high efflux ratio (9.3), which raised concern with respect to a potential transporter recognition of <b>25</b>. In contrast, in the corresponding <i>N</i>-(pyridin-2-yl)pyridin-2-amine series, single enantiomer VIP152 revealed a very promising preclinical overall profile.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Preclinical Properties of VIP152</h3><div class="NLM_p"><i>N</i>-(Pyridin-2-yl)pyridin-2-amine VIP152 (see <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) revealed very potent CDK9 inhibitory activity (IC<sub>50</sub>: 3 nM) and high selectivity against the structurally related kinases CDK2 (ratio of IC<sub>50</sub> values: 1033) and CDK7 (ratio of IC<sub>50</sub> values: >5000) in the Bayer kinase panel under high ATP conditions. The high kinase selectivity of VIP152 in vitro, even within the CDK family, was confirmed in the external kinase panels with a selectivity factor of at least 50 against all tested CDKs (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv" class="ext-link">Supporting Information</a>). Outside the CDK family, substantial inhibitory activity with a selectivity factor of less than 50 was only recorded against GSK3α and IRAK1 (DiscoverX <i>K</i><sub>D</sub> values: CDK9, 1.3 nM; GSK3α, 7.4 nM; and IRAK1, 61 nM) (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv" class="ext-link">Supporting Information</a>). In a recent comprehensive survey of CDK inhibitor selectivity in live cells with energy-transfer probes, VIP152 demonstrated the strongest target affinity and selectivity among the tested CDK9 inhibitors.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> VIP152 exhibited very potent antiproliferative activity in vitro, especially against A2780 cells (IC<sub>50</sub>: 45 nM) and MOLM-13 cells (IC<sub>50</sub>: 29 nM). In comparison to VIP152, its enantiomer <b>32</b> revealed very similar in vitro properties, well within the limits of measurement accuracy. However, with multiple batches of enantiomer <b>32</b>, there was a trend toward a slightly reduced activity against CDK9 in the biochemical assay under high ATP conditions (IC<sub>50</sub>: 6 nM) and antiproliferative activity against A2780 cells (IC<sub>50</sub>: 52 nM) and MOLM-13 cells (IC<sub>50</sub>: 42 nM). VIP152 demonstrated low aqueous solubility at pH 6.5 (<i>S</i><sub>w</sub>: 25 mg/L), but in contrast to atuveciclib and BAY-332, <i>N</i>-(pyridin-2-yl)pyridin-2-amine VIP152 exhibited a significant increase in solubility at pH 4 (<i>S</i><sub>w</sub>: 699 mg/L), which is still within the physiologically acceptable range for iv application in humans.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Nevertheless, VIP152 exhibited high Caco-2 permeability (<i>P</i><sub>app</sub> A–B: 166 nm/s) and no efflux (efflux ratio: 1.1) in the Caco-2 assay.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of VIP152<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub> (aq), DME, 100 °C, 4 h, and 95%; (b) Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 5 h, and 94%; (c) CF<sub>3</sub>C(O)NH<sub>2</sub>, NaO<i>t</i>-Bu, 1,3-dibromo-5,5-dimethylhydantoin, 1,4-dioxane, THF, 10 °C, and crude; (d) KMnO<sub>4</sub>, acetone, 50 °C, and crude; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, and 10% (over three steps); and (f) chiral preparative HPLC.</p></p></figure><div class="NLM_p">In an in vivo PK study in rats (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), VIP152 showed low blood clearance (CL<sub>b</sub>: 1.1 L/h/kg), a moderate volume of distribution (<i>V</i><sub>ss</sub>: 0.74 L/kg), and a short to moderate half-life (<i>t</i><sub>1/2</sub>: 1.0 h). Furthermore, VIP152 exhibited a blood/plasma ratio of about 1 and did not show significant inhibition of cytochrome P450 activity, with IC<sub>50</sub> values >20 μM. Induction of CYP1A2 in vitro was not observed.</div><div class="NLM_p">The MOLM-13 xenograft model in rats was used to determine the optimum application frequency regarding the antitumor activity of VIP152. VIP152 administered iv at its respective MTD Q3D (3 mg/kg), Q5D (3.75 mg/kg), or QW (4.5 mg/kg) demonstrated similar potent antitumor efficacies with T/C ratios of 0.11, 0.08, and 0.15, respectively, and excellent tolerability (all <i>p</i> < 0.001; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Antitumor activity upon once weekly iv dosing was then confirmed in a second species using the MOLM-13 xenograft model in mice. Once weekly (QW) administration of VIP152 at 12.5 mg/kg resulted in a T/C ratio of 0.20 (<i>p</i> = 0.001; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) and was well tolerated, as indicated by no significant body weight reduction (body weight change: −5%) and no fatal toxicities throughout the study.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antitumor efficacy of atuveciclib and VIP152 in human AML models in rats and mice. (A) Tumor growth in rats treated iv with either vehicle or VIP152 once every 3 days (Q3D), once every 5 days (Q5D), or once weekly (QW). Treatments were started 6 days after tumor cell inoculation. (B) Tumor growth in mice treated iv with either vehicle or VIP152 once weekly (QW). Treatments were started 5 days after tumor cell inoculation. (C) Tumor growth in rats treated with atuveciclib p.o. daily (15 days) dosing or VIP152 iv once weekly (3 doses). Statistical significances were calculated using the mean tumor volumes at the time point when the vehicle group was sacrificed. Asterisks indicate statistical significance relative to the vehicle control (**<i>p</i> < 0.01 and ***<i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The MV4-11 xenograft model in rats was used to compare the activity of daily oral administration of atuveciclib (12 mg/kg × 15 days) compared with once weekly iv dosing of VIP152 (4.5 mg/kg, Q7D × 3). Treatment with VIP152 leads to complete remission in all animals with no measurable tumor burden for an additional 30 days without treatment compared with regrowth in all of the atuveciclib-treated animals.</div><div class="NLM_p">Since the once weekly application maybe the best regimen to allow for recovery of the treatment-induced acute reduction of circulating neutrophil levels, this application schedule was defined as the target regimen for the clinical development of VIP152.</div><div class="NLM_p last">Due to the very promising overall profile in vitro and in vivo, sulfoximine VIP152 was selected as the development candidate and entered phase 1 clinical trials in patients (NCT02635672 and NCT02745743).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p">The synthesis of racemic <i>N</i>-(pyridin-2-yl)pyrimidin-4-amine <b>24</b> started with the reaction of 4,6-dichloropyrimidine (<b>18</b>) and boronic acid <b>19</b> under Suzuki conditions to afford building block <b>20</b>. A palladium-catalyzed cross-coupling reaction of chloropyrimidine <b>20</b> and commercial pyridin-2-amine <b>21</b> yielded 4,6-disubstituted pyrimidine <b>22</b>. The thioether was first converted into the racemic sulfilimine <b>23</b> using the conditions of Gries and Krüger.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Oxidation and cleavage of the trifluoroacetyl group afforded the desired sulfoximine <b>24</b>, which was separated into the enantiomers by chiral high-performance liquid chromatography (HPLC). The absolute configuration of the single stereoisomers <b>25</b> and <b>26</b> at the sulfur was not determined.</div><div class="NLM_p">The research synthesis of <i>N</i>-(pyridin-2-yl)pyridin-2-amine <b>31</b> was accomplished with a similar synthetic strategy. Suzuki coupling of 2-chloro-5-fluoro-4-iodopyridine (<b>27</b>) and boronic acid <b>19</b> afforded the desired building block <b>28</b> in very high yield. A subsequent palladium-catalyzed cross-coupling reaction of chloropyridine <b>28</b> and commercial pyridin-2-amine <b>21</b> resulted in 2,2′-dipyridylamine <b>29</b> in very high yield, too. Finally, thioether <b>29</b> was converted into the racemic sulfoximine <b>31</b> relying again on the conditions of Gries and Krüger.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Chiral HPLC separation of racemate <b>31</b> yielded VIP152 and its enantiomer <b>32</b>. The stereochemistry at the sulfur of the sulfoximine group of VIP152 was determined by X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Small-molecule X-ray structure of VIP152 (accession code CCDC 2005811).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38672" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38672" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Transient modulation of CDK9 by highly selective inhibitors leads to rapid depletion of short-lived mRNA transcripts of important survival proteins and oncogenes such as MCL-1 and MYC,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> providing a promising rationale for cancer therapy, especially for hematological malignancies. Atuveciclib (BAY 1143572), the first selective, orally available CDK9 inhibitor that entered clinical development, revealed promising results in xenograft models. In a phase 1 clinical trial in patients with advanced cancer, atuveciclib displayed a limited therapeutic index with oral administration with neutropenia being the most prominent adverse event putatively mediated, at least in part, by the on-target inhibition of MCL-1 in circulating white blood cells leading to apoptosis induction in neutrophils. With the aim of improving the therapeutic index of anticancer therapy with CDK9 inhibitors, a follow-up program was initiated to identify a new generation of highly potent and selective CDK9 inhibitors enabling once weekly iv treatment of patients by a more pronounce “oncogenic shock”-like<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> disruption of oncogene transcription, followed by a time without target inhibition needed for the recovery of circulating neutrophils. Our lead optimization efforts started with atuveciclib focused on potency, selectivity, and aqueous solubility which led to the identification of a new generation of CDK9 inhibitors. Sulfoximine containing VIP152 clearly exhibited the most promising preclinical overall profile with respect to potency, selectivity, physicochemical properties, and in vivo PK, as well as in vivo potency. VIP152 is efficacious in human xenograft tumor models of AML in mice and rats upon once weekly iv administration with good tolerability. The evaluation of various scaffold hops and the employment of the underexplored benzyl sulfoximine group were crucial for overcoming hurdles in this project, which highlights the ongoing important role of medicinal chemistry in small-molecule drug discovery.</div><div class="NLM_p last">The discovery of VIP152, in optimizing atuveciclib, demonstrates promising translation of our efforts in early clinical studies.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The improved selectivity on CDK9 starting from removal of the CDK2 activity, ATP independence, and a short-lived, iv strategy resulted in a promising 38% disease control rate in the first 31 patients in the dose escalation phase of VIP152’s first-in-human study. Several patients experienced extended benefit beyond 15 cycles including two patients with double-hit diffuse large B-cell lymphoma (manuscript in prep).<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> This early clinical experience demonstrates that VIP152 is well tolerated with a manageable safety profile for extended periods and opens the door to test the oncogenic shock hypothesis with a once weekly dosing schedule in patients with cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Methods and Materials</h3><div class="NLM_p last">Commercially available reagents and anhydrous solvents were used as supplied, without further purification. All air- and moisture-sensitive reactions were carried out in oven-dried (at 120 °C) glassware under an inert atmosphere of argon. A Biotage Initiator Classic microwave reactor was used for reactions conducted in a microwave oven. Reactions were monitored by thin-layer chromatography (TLC) and ultra-performance liquid chromatography (UPLC) analysis with a Waters Acquity UPLC MS Single Quad system; column: Acquity UPLC BEH C18 1.7 μm, 50 × 2.1 mm; basic conditions: eluent A: H<sub>2</sub>O + 0.2 vol % aq NH<sub>3</sub> (32%), eluent B: MeCN; gradient: 0–1.6 min 1–99% B, 1.6–2.0 min 99% B; flow: 0.8 mL/min; acidic conditions: eluent A: H<sub>2</sub>O + 0.1 vol % formic acid (99%), eluent B: MeCN; gradient: 0–1.6 min 1–99% B, 1.6–2.0 min 99% B; flow: 0.8 mL/min; temperature: 60 °C; and DAD scan: 210–400 nm. Analytical TLC was carried out on aluminum-backed plates coated with a Merck Kieselgel 60 F254, with visualization under UV light at 254 nm. Flash chromatography was carried out using a Biotage Isolera One system with a 200–400 nm variable detector. Preparative HPLC was carried out with a Waters AutoPurification MS Single Quad system; column: Waters XBridge C18 5 μm, 100 × 30 mm; basic conditions: eluent A: H<sub>2</sub>O + 0.2 vol % aq NH<sub>3</sub> (32%), eluent B: MeCN; gradient: 0–0.5 min 5% B, flow: 25 mL/min; 0.51–5.50 min 10–100% B, flow: 70 mL/min; 5.51–6.5 min 100% B, flow: 70 mL/min; acidic conditions: eluent A: H<sub>2</sub>O + 0.1 vol % formic acid (99%), eluent B: MeCN; gradient: 0–0.5 min 5% B, flow: 25 mL/min; 0.51–5.50 min 10–100% B, flow: 70 mL/min; 5.51–6.5 min 100% B, flow: 70 mL/min; temperature: 25 °C; and DAD scan: 210–400 nm. NMR spectra were recorded at rt (22 ± 1 °C), unless otherwise noted, on Bruker Avance III HD spectrometers. <sup>1</sup>H NMR spectra were obtained at 300, 400, 500, or 600 MHz and referenced to the residual solvent signal (7.26 ppm for CDCl<sub>3</sub> and 2.50 ppm for DMSO-<i>d</i><sub>6</sub>). <sup>13</sup>C NMR spectra were obtained at 100 or 150 MHz and also referenced to the residual solvent signal (39.52 ppm for DMSO-<i>d</i><sub>6</sub>). <sup>1</sup>H NMR data are reported as follows: chemical shift (δ) in ppm, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, quin = quintet, br = broad, and m = multiplet), coupling constant(s), and integration. High-resolution mass spectra were recorded on a Xevo G2-XS Tof (Waters) instrument. Low-resolution mass spectra (electrospray ionization, ESI) were obtained via HPLC–MS (ESI) using a Waters Acquity UPLC system equipped with an SQ 3100 Mass Detector; column: Acquity UPLC BEH C18 1.7 μm, 50 × 2.1 mm; eluent A: H<sub>2</sub>O + 0.05% formic acid (99%), eluent B: MeCN + 0.05% formic acid (99%); gradient: 0–0.5 min 5% B, 0.5–2.5 min 5–100% B, 2.5–4.5 min 100% B; total run time: 5 min; and flow: 0.5 mL/min. Optical rotations were recorded on a JASCO P-2000 polarimeter. IR spectra were obtained on a Bruker Vertex 70 FTIR spectrometer (2 cm<sup>–1</sup> resolution, 32 scans, and 400–4000 cm<sup>–1</sup> range) using single-bounce diamond attenuated total reflection (ATR) or KBr techniques. The purity of all target compounds was at least 95%, as determined by LC–MS (for a detailed description of the LC–MS methods, see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_002.pdf" class="ext-link">Supporting Information</a>). Compound names were generated using ICS software.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthetic Procedures</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Compound <b>1</b></h4><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> 1-[(Methylsulfanyl)methyl]-3-nitrobenzene</h5><div class="NLM_p last">Sodium methanethiolate (13.5 g, 192 mmol) was added in two portions to a stirred solution of 1-(chloromethyl)-3-nitrobenzene (30.0 g, 175 mmol) in EtOH (360 mL) at −15 °C. The cold bath was removed, and the mixture was stirred at rt for 3 h. Then, it was diluted with brine and extracted with EtOAc (2×). The combined organic phases were washed with H<sub>2</sub>O, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated to afford the desired product (32.2 g) that was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (m, 1H), 8.11 (m, 1H), 7.66 (m, 1H), 7.50 (m, 1H), 3.75 (s, 2H), 2.01 (s, 3H).</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> 1-[(Methylsulfonyl)methyl]-3-nitrobenzene</h5><div class="NLM_p last"><i>m</i>-Chloroperoxybenzoic acid (77%; 26.9 g, 120 mmol) was added to a stirred solution of 1-[(methylsulfanyl)methyl]-3-nitrobenzene (10.0 g) in dichloromethane (DCM, 1305 mL) at 0 °C. The mixture was stirred at 0 °C for 30 min and then at rt for 2.5 h. Then, it was diluted with H<sub>2</sub>O (300 mL) and NaHCO<sub>3</sub> (11.0 g) was added. The mixture was extracted with DCM (2×). The combined organic phases were filtered using a Whatman filter and concentrated. The residue was purified by flash chromatography (DCM/EtOH 95:5) and finally recrystallized from EtOAc to afford the desired product (6.2 g, 28.9 mmol, and 53% yield over two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.28 (m, 1H), 8.22 (m, 1H), 7.83 (m, 1H), 7.69 (m, 1H), 4.68 (s, 2H), 2.93 (s, 3H).</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> 3-[(Methylsulfonyl)methyl]aniline</h5><div class="NLM_p last">TiCl<sub>3</sub> solution (ca. 15% in ca. 10% HCl, 162 mL) was added to a stirred solution of 1-[(methylsulfonyl)methyl]-3-nitrobenzene (5.1 g, 23.8 mmol) in THF (250 mL) at rt, and the mixture was stirred for 16 h. The pH was increased to 10 by the addition of 1 N NaOH; then, the reaction mixture was extracted with EtOAc (2×). The combined organic phases were washed with brine, filtered using a Whatman filter, and concentrated to afford the desired product (4.5 g) that was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 6.97 (m, 1H), 6.51 (m, 3H), 5.13 (br, 2H), 4.23 (s, 2H), 2.83 (s, 3H).</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> 4-Chloro-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-1,3,5-triazin-2-amine</h5><div class="NLM_p last">DIPEA (3.7 mL, 21.3 mmol) was added to a stirred solution of 2,4-dichloro-1,3,5-triazine (1.60 g, 10.7 mmol) in THF/<i>i</i>-PrOH (1:1, 20 mL) at −40 °C. Then, a suspension of 3-[(methylsulfonyl)methyl]aniline (1.97 g, 10.7 mmol) in THF/<i>i</i>-PrOH (1:1, 10 mL) was added at this temperature. Under stirring, the temperature of the reaction mixture was slowly raised over 3 h to 0 °C. The mixture was concentrated in vacuo to afford the crude product (5.2 g) that was used without further purification.</div></div><div id="sec4_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> 4-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-1,3,5-triazin-2-amine (<b>1</b>)</h5><div class="NLM_p last">A mixture of crude 4-chloro-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-1,3,5-triazin-2-amine (1000 mg), (4-fluoro-2-methoxyphenyl)boronic acid (569 mg, 3.35 mmol; Aldrich), and tetrakis(triphenylphosphine)palladium(0) [Pd(PPh<sub>3</sub>)<sub>4</sub>] (580 mg, 0.50 mmol) in DME (10.3 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (3.4 mL) was degassed using argon. The mixture was stirred under argon for 90 min at 100 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by flash chromatography (DCM to DCM/EtOH 95:5) to afford <b>1</b> (402 mg, 1.03 mmol) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.82 (s, 1H), 7.96 (br s, 1H), 7.69–7.87 (m, 2H), 7.38–7.46 (m, 2H), 7.16 (d, <i>J</i> = 7.53 Hz, 1H), 6.75–6.81 (m, 2H), 4.26 (s, 2H), 3.94 (s, 3H), 2.79 (s, 3H). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub>S, 389.1084; found, 389.1080.</div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Compound <b>2</b></h4><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> 2-Chloro-4-(4-fluoro-2-methoxyphenyl)pyridine</h5><div class="NLM_p last">A mixture of 4-bromo-2-chloropyridine (1 g, 5.20 mmol; Aldrich), (4-fluoro-2-methoxyphenyl)boronic acid (971 mg, 5.72 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (600 mg, 0.52 mmol) in toluene (26 mL) and a solution of K<sub>2</sub>CO<sub>3</sub> (861 mg, 6.23 mmol) in H<sub>2</sub>O (5.2 mL) was degassed using argon. The mixture was stirred under argon for 3 h at 90 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane/EtOAc 4:1) to afford the desired product (210 mg, 0.88 mmol, 17% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.40–8.44 (m, 1H), 7.60–7.63 (m, 1H), 7.54 (dd, <i>J</i> = 1.52, 5.07 Hz, 1H), 7.50 (dd, <i>J</i> = 6.84, 8.62 Hz, 1H), 7.11 (dd, <i>J</i> = 2.53, 11.41 Hz, 1H), 6.92 (dt, <i>J</i> = 2.53, 8.36 Hz, 1H), 3.83 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 238.</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> 4-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}pyridin-2-amine (<b>2</b>)</h5><div class="NLM_p last">A mixture of 3-[(methylsulfonyl)methyl]aniline (123 mg, 0.66 mmol), 2-chloro-4-(4-fluoro-2-methoxyphenyl)pyridine (105 mg, 0.44 mmol), tris(dibenzylideneacetone)dipalladium(0) [Pd<sub>2</sub>(dba)<sub>3</sub>] (48 mg, 0.05 mmol), 2,2′-bis(diphenylphosphino)-1,1′-binaphthyl (BINAP) (102 mg, 0.16 mmol), and sodium <i>tert</i>-butoxide (85 mg, 0.88 mmol) in DMF (4.1 mL) and THF (8.3 mL) was stirred under argon for 4.5 h at 110 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane/EtOAc) to afford <b>2</b> (55 mg, 0.14 mmol, 32% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.16 (s, 1H), 8.15 (d, <i>J</i> = 5.32 Hz, 1H), 7.78 (br d, <i>J</i> = 7.86 Hz, 1H), 7.63 (s, 1H), 7.39 (dd, <i>J</i> = 6.84, 8.36 Hz, 1H), 7.28 (t, <i>J</i> = 7.86 Hz, 1H), 7.07 (dd, <i>J</i> = 2.28, 11.41 Hz, 1H), 6.97 (s, 1H), 6.86–6.95 (m, 2H), 6.85 (dd, <i>J</i> = 1.27, 5.32 Hz, 1H), 4.44 (s, 2H), 3.83 (s, 3H), 2.93 (s, 3H). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.18 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 244.99 Hz, 1C<sub>q</sub>), 157.54 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 155.83 (s, 1C<sub>q</sub>), 147.01 (s, 1CH), 145.95 (s, 1C<sub>q</sub>), 141.94 (s, 1C<sub>q</sub>), 131.17 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1CH), 129.32 (s, 1C<sub>q</sub>), 128.78 (s, 1CH), 123.93 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 3.39 Hz, 1C<sub>q</sub>), 122.99 (s, 1CH), 120.05 (s, 1CH), 117.95 (s, 1CH), 115.35 (s, 1CH), 110.91 (s, 1CH), 107.25 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.19 Hz, 1CH), 100.28 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.86 Hz, 1CH), 59.72 (s, 1CH<sub>2</sub>), 56.17 (s, 1CH<sub>3</sub>), 39.50 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3597 (br s), 3364 (w), 3290 (w), 3009 (w), 2922 (w), 1597 (s), 1529 (m), 1479 (m), 1443 (m), 1379 (m), 1279 (s), 1192 (m), 1113 (s), 1024 (s), 951 (s), 876 (m), 833 (m), 798 (m). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>FN<sub>2</sub>O<sub>3</sub>S, 387.1179; found, 387.1175.</div></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Compound <b>3</b></h4><div id="sec4_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 2,5-Difluoro-4-(4-fluoro-2-methoxyphenyl)pyridine</h5><div class="NLM_p last">A mixture of 2,5-difluoro-4-iodopyridine (500 mg, 2.07 mmol; abcr), (4-fluoro-2-methoxyphenyl)boronic acid (388 mg, 2.28 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (240 mg, 0.21 mmol) in DME (6.2 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (3.1 mL) was degassed using argon. The mixture was stirred under argon for 16 h at 100 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane to hexane/EtOAc 3:2) to afford the desired product (420 mg, 1.76 mmol, 85% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.07 (t, <i>J</i> = 1.32 Hz, 1H), 7.33 (br d, <i>J</i> = 1.51 Hz, 1H), 6.95 (dd, <i>J</i> = 2.64, 4.52 Hz, 1H), 6.71–6.83 (m, 2H), 3.83 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 240.</div></div><div id="sec4_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}pyridin-2-amine (<b>3</b>)</h5><div class="NLM_p last">To a solution of 2,5-difluoro-4-(4-fluoro-2-methoxyphenyl)pyridine (150 mg, 0.63 mmol) and 3-[(methylsulfonyl)methyl]aniline (140 mg, 0.75 mmol) in DMF (5 mL), NaH (60% suspension in mineral oil; 37 mg, 0.94 mmol) was added. The mixture was stirred for 2 h at 70 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by preparative HPLC (basic conditions) to afford <b>3</b> (41 mg, 0.10 mmol, 16% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.11 (d, <i>J</i> = 1.51 Hz, 1H), 7.39–7.48 (m, 2H), 7.30–7.37 (m, 1H), 7.01 (d, <i>J</i> = 7.16 Hz, 1H), 6.82 (d, <i>J</i> = 4.90 Hz, 1H), 6.70–6.79 (m, 2H), 6.51 (s, 1H), 4.22 (s, 2H), 3.83 (s, 3H), 2.78 (s, 3H). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.64 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 245.41 Hz, 1C<sub>q</sub>), 157.92 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 152.41 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 1.69 Hz, 1C<sub>q</sub>), 151.36 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 244.14 Hz, 1C<sub>q</sub>), 147.01 (s, 1CH), 145.95 (s, 1C<sub>q</sub>), 141.82 (s, 1C<sub>q</sub>), 135.00 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 15.68 Hz, 1C<sub>q</sub>), 133.60 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 131.64 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1CH), 129.40 (s, 1C<sub>q</sub>), 128.84 (s, 1CH), 123.10 (s, 1CH), 119.95 (s, 1CH), 118.52 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 3.39 Hz, 1C<sub>q</sub>), 117.59 (s, 1CH), 115.35 (s, 1CH), 112.63 (s, 1CH), 107.12 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.62 Hz, 1CH), 100.16 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 59.67 (s, 1CH<sub>2</sub>), 56.25 (s, 1CH<sub>3</sub>), 39.51 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3370 (w), 3330 (w), 3105 (w), 2924 (w), 1624 (m), 1601 (s), 1483 (s), 1447 (m), 1379 (m), 1298 (s), 1213 (m), 1194 (m), 1153 (m), 1026 (m), 953 (m), 878 (w), 837 (w), 798 (m). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 405. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>F<sub>2</sub>N<sub>2</sub>O<sub>3</sub>S, 405.1085; found, 405.1082.</div></div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Compound <b>4</b></h4><div id="sec4_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 2-Chloro-3-fluoro-4-(4-fluoro-2-methoxyphenyl)pyridine</h5><div class="NLM_p last">A mixture of 2-chloro-3-fluoro-4-iodopyridine [1 g, 3.88 mmol; Tokyo Chemical Industry (TCI)], (4-fluoro-2-methoxyphenyl)boronic acid (660 mg, 3.88 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (449 mg, 0.38 mmol) in DME (10 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (5.8 mL) was degassed using argon. The mixture was stirred under argon for 3.5 h at 100 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane to hexane/EtOAc 1:1) to afford the desired product (569 mg, 2.23 mmol, 57% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.27 (d, <i>J</i> = 5.05 Hz, 1H), 7.45 (t, <i>J</i> = 5.05 Hz, 1H), 7.37 (dd, <i>J</i> = 6.95, 8.46 Hz, 1H), 7.09 (dd, <i>J</i> = 2.53, 11.37 Hz, 1H), 6.90 (dt, <i>J</i> = 2.53, 8.46 Hz, 1H), 3.77 (s, 3H).</div></div><div id="sec4_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 3-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}pyridin-2-amine (<b>4</b>)</h5><div class="NLM_p last">A mixture of 3-[(methylsulfonyl)methyl]aniline (135 mg, 0.73 mmol), 2-chloro-3-fluoro-4-(4-fluoro-2-methoxyphenyl)pyridine (125 mg, 0.49 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (54 mg, 0.06 mmol), BINAP (113 mg, 0.18 mmol), and sodium <i>tert</i>-butoxide (94 mg, 0.98 mmol) in DMF (4.5 mL) and THF (9 mL) was stirred under argon for 2 h at 100 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by preparative HPLC (basic conditions) to afford <b>4</b> (78 mg, 0.19 mmol, 39% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.01 (d, <i>J</i> = 5.05 Hz, 1H), 7.82 (s, 1H), 7.72 (dd, <i>J</i> = 1.64, 7.96 Hz, 1H), 7.38 (t, <i>J</i> = 7.96 Hz, 1H), 7.22–7.28 (m, 1H), 7.06 (d, <i>J</i> = 7.58 Hz, 1H), 6.72–6.80 (m, 4H), 4.27 (s, 2H), 3.83 (s, 3H), 2.80 (s, 3H). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.58 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 244.99 Hz, 1C<sub>q</sub>), 157.87 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 145.09 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1C<sub>q</sub>), 143.94 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 256.43 Hz, 1C<sub>q</sub>), 141.71 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 7.20 Hz, 1CH), 140.92 (s, 1C<sub>q</sub>), 131.65 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1CH), 131.30 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.45 Hz, 1C<sub>q</sub>), 129.19 (s, 1C<sub>q</sub>), 128.55 (s, 1CH), 124.03 (s, 1CH), 121.62 (s, 1CH), 119.28 (s, 1CH), 118.50 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 3.39 Hz, 1C<sub>q</sub>), 116.77 (s, 1CH), 107.08 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.19 Hz, 1CH), 100.10 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.86 Hz, 1CH), 59.77 (s, 1CH<sub>2</sub>), 56.21 (s, 1CH<sub>3</sub>), 39.47 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3379 (m), 3026 (w), 3013 (w), 2920 (w), 1632 (s), 1609 (s), 1599 (s), 1535 (s), 1491 (s), 1437 (s), 1317 (m), 1283 (s), 1236 (m), 1188 (s), 1109 (s), 1030 (s), 984 (m), 951 (s), 893 (m), 835 (s), 762 (m). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>F<sub>2</sub>N<sub>2</sub>O<sub>3</sub>S, 405.1085; found, 405.1085.</div></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Compound <b>5</b></h4><div id="sec4_2_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 2-Chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine</h5><div class="NLM_p last">A mixture of 2,4-dichloro-5-fluoropyrimidine (200 mg, 1.20 mmol; Aldrich), (4-fluoro-2-methoxyphenyl)boronic acid (224 mg, 1.31 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (138 mg, 0.12 mmol) in DME (3.6 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (1.8 mL) was degassed using argon. The mixture was stirred under argon for 16 h at 90 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane/EtOAc 1:1) to afford the desired product (106 mg, 0.41 mmol, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.47 (m, 1H), 7.51 (m, 1H), 6.82 (m, 1H), 6.73 (m, 1H), 3.85 (s, 3H).</div></div><div id="sec4_2_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}pyrimidin-2-amine (<b>5</b>)</h5><div class="NLM_p last">A mixture of 3-[(methylsulfonyl)methyl]aniline (108 mg, 0.58 mmol), 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (150 mg, 0.58 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (96 mg, 0.11 mmol), 4,5-bis(diphenylphosphino)-9,9-dimethylxanthene (Xantphos) (88 mg, 0.15 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (800 mg, 2.46 mmol) in 1,4-dioxane (1.0 mL) was degassed using argon. The mixture was stirred under argon for 150 min at 100 °C. After cooling, the mixture was filtered and the filter was rinsed with DCM and EtOAc. The filtrate was concentrated in vacuo and the residue was purified by preparative HPLC (acidic conditions) to afford <b>5</b> (49 mg, 0.12 mmol, 21% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 9.83 (m, 1H), 8.51 (m, 1H), 7.69 (m, 2H), 7.50 (m, 1H), 7.25 (m, 1H), 7.08 (m, 1H), 6.91 (m, 2H), 4.37 (s, 2H), 3.80 (s, 3H), 2.86 (s, 3H). <sup>13</sup>C NMR (100.65 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.34 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 247.11 Hz, 1C<sub>q</sub>), 158.55 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 156.53 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.54 Hz, 1C<sub>q</sub>), 151.52 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 14.41 Hz, 1C<sub>q</sub>), 150.23 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 250.50 Hz, 1C<sub>q</sub>), 145.72 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 24.58 Hz, 1CH), 140.67 (s, 1C<sub>q</sub>), 132.06 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1CH), 129.31 (s, 1C<sub>q</sub>), 128.67 (s, 1CH), 124.15 (s, 1CH), 120.91 (s, 1CH), 119.05 (m, 1C<sub>q</sub>), 118.49 (s, 1CH), 107.30 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.62 Hz, 1CH), 100.16 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.86 Hz, 1CH), 59.76 (s, 1CH<sub>2</sub>), 56.34 (s, 1CH<sub>3</sub>), 40.46 (s, 1CH<sub>3</sub>). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>2</sub>N<sub>3</sub>O<sub>3</sub>S, 406.1037; found, 406.1041.</div></div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Compound <b>6</b></h4><div id="sec4_2_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 2,5-Dichloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine</h5><div class="NLM_p last">A mixture of 2,4,5-trichloropyrimidine (367 mg, 2 mmol; Combi-Blocks, Inc.), (4-fluoro-2-methoxyphenyl)boronic acid (374 mg, 2.2 mmol), and [1,1′-bis(diphenylphosphino)ferrocene]dichloropalladium(II) [Pd(dppf)Cl<sub>2</sub>] (1:1 complex with DCM; 163 mg, 0.2 mmol; Aldrich) in DME (6.1 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (3 mL) was degassed using argon. The mixture was stirred under argon for 1 h at 90 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by column chromatography (hexane/EtOAc 1:1) to afford the desired product (280 mg, 1.03 mmol, 52% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.60 (s, 1H), 7.34 (dd, <i>J</i> = 6.57, 8.34 Hz, 1H), 6.80 (dt, <i>J</i> = 2.40, 8.27 Hz, 1H), 6.73 (dd, <i>J</i> = 2.27, 10.86 Hz, 1H), 3.82 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 273.</div></div><div id="sec4_2_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 5-Chloro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}pyrimidin-2-amine (<b>6</b>)</h5><div class="NLM_p last">To a mixture of 2,5-dichloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (136 mg, 0.498 mmol) and 3-[(methylsulfonyl)methyl]aniline (92 mg, 0.49 mmol) in <i>n</i>-BuOH, 4 M HCl in 1,4-dioxane (0.125 mL, 0.498 mmol) was added. The mixture was stirred for 65 h at 120 °C and an additional 72 h at 140 °C. Then, the mixture was evaporated to dryness and the residue was taken up in EtOAc and washed with aq NaHCO<sub>3</sub> solution, followed by brine. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The residue was purified by column chromatography (hexane/EtOAc 1:1) to afford <b>6</b> (70 mg, 0.16 mmol, 33% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.42 (s, 1H), 7.79 (d, <i>J</i> = 1.76 Hz, 1H), 7.52–7.58 (m, 1H), 7.29–7.38 (m, 2H), 7.24 (s, 1H), 7.07 (d, <i>J</i> = 7.53 Hz, 1H), 6.79 (dt, <i>J</i> = 2.38, 8.22 Hz, 1H), 6.74 (dd, <i>J</i> = 2.38, 10.67 Hz, 1H), 4.23 (s, 2H), 3.84 (s, 3H), 2.74 (s, 3H). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.90 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 245.84 Hz, 1C<sub>q</sub>), 162.03 (s, 1C<sub>q</sub>), 158.18 (s, 1C<sub>q</sub>), 157.94 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 156.85 (s, 1C<sub>q</sub>), 140.15 (s, 1C<sub>q</sub>), 131.22 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1CH), 129.38 (s, 1C<sub>q</sub>), 128.72 (s, 1CH), 124.63 (s, 1CH), 121.86 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 121.49 (s, 1CH), 119.78 (s, 1C<sub>q</sub>), 119.14 (s, 1CH), 106.99 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 100.04 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 59.69 (s, 1CH<sub>2</sub>), 56.17 (s, 1CH<sub>3</sub>), 39.44 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3337 (s), 3074 (w), 3020 (w), 2924 (w), 1663 (w), 1607 (m), 1593 (m), 1545 (s), 1508 (m), 1445 (s), 1394 (s), 1312 (w), 1281 (s), 1198 (w), 1151 (m), 1024 (w), 953 (s), 880 (m), 849 (m), 822 (m), 789 (m). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 422. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>ClFN<sub>3</sub>O<sub>3</sub>S, 422.0742; found, 422.0741.</div></div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Compound <b>7</b></h4><div id="sec4_2_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>-<i>tert</i>-Butyl-2,4-dichloropyrimidine-5-carboxamide</h5><div class="NLM_p last">To 2,4-dichloropyrimidine-5-carbonyl chloride (5 g, 24 mmol; Combi-Blocks, Inc.) in THF (30 mL), a solution of <i>tert</i>-butylamine (1.82 g, 24.9 mmol) and triethylamine (3.45 mL) in THF (30 mL) was added slowly at −5 °C. The mixture was stirred for 4.5 h at this temperature. After removal of the precipitated salts by filtration, the filtrate was evaporated to dryness. The resulting material was used without further purification.</div></div><div id="sec4_2_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> <i>N</i>-<i>tert</i>-Butyl-2-chloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine-5-carboxamide</h5><div class="NLM_p last">A mixture of <i>N</i>-<i>tert</i>-butyl-2,4-dichloropyrimidine-5-carboxamide (5.73 g, 23.1 mmol), (4-fluoro-2-methoxyphenyl)boronic acid (4.32 g, 25.4 mmol), and [Pd(dppf)Cl<sub>2</sub>] (1:1 complex with DCM; 1.88 g, 2.3 mmol) in DME (70 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (34.6 mL) was degassed using argon. The mixture was stirred under argon for 16 h at 90 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (DCM to DCM/EtOH 95:5 and hexane to hexane/EtOAc 1:1) to afford the desired product (618 mg) as an equimolar mixture with (<i>Z</i>)-3-amino-<i>N</i>-<i>tert</i>-butyl-3-(4-fluoro-2-methoxyphenyl)-2-formylprop-2-enamide.</div></div><div id="sec4_2_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 2-Chloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine-5-carbonitrile</h5><div class="NLM_p last">A mixture of <i>N</i>-<i>tert</i>-butyl-2-chloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine-5-carboxamide (564 mg) and thionyl chloride (10.3 mL) was stirred for 18 h at 80 °C. After cooling, the volatiles were removed under reduced pressure. The residue was purified by column chromatography (hexane to hexane/EtOAc 3:1) to afford the desired product (206 mg, 0.78 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.85 (s, 1H), 7.60 (dd, <i>J</i> = 6.50, 8.57 Hz, 1H), 6.85 (br d, <i>J</i> = 2.07 Hz, 1H), 6.79 (dd, <i>J</i> = 2.17, 10.46 Hz, 1H), 3.93 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 264.</div></div><div id="sec4_2_7_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 4-(4-Fluoro-2-methoxyphenyl)-2-({3-[(methylsulfonyl)methyl]phenyl}amino)pyrimidine-5-carbonitrile (<b>7</b>)</h5><div class="NLM_p last">A mixture of 2-chloro-4-(4-fluoro-2-methoxyphenyl)pyrimidine-5-carbonitrile (80 mg, 0.303 mmol), 3-[(methylsulfonyl)methyl]aniline (56 mg, 0.303 mmol), (XPhos) palladium(II) phenethylamine chloride (XPhos Pd G1, CAS 1028206-56-5) (18.8 mg, 0.023 mmol), 2-dicyclohexylphosphino-2′,4′,6′-triisopropyl-1,1′-biphenyl (XPhos) (10.8 mg, 0.023 mmol), and potassium phosphate (322 mg, 1.52 mmol) in toluene (3.2 mL) and <i>N</i>-methyl-2-pyrrolidone (NMP, 0.8 mL) was degassed using argon. The mixture was stirred under argon for 3 h at 130 °C. After cooling, the mixture was diluted with EtOAc and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by preparative HPLC (acidic conditions) to afford <b>7</b> (7.6 mg, 0.02 mmol, 7% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.65 (s, 1H), 7.81 (s, 1H), 7.63 (br d, <i>J</i> = 3.39 Hz, 2H), 7.50 (dd, <i>J</i> = 6.88, 8.19 Hz, 1H), 7.40 (t, <i>J</i> = 7.91 Hz, 1H), 7.15 (d, <i>J</i> = 7.72 Hz, 1H), 6.74–6.87 (m, 2H), 4.25 (s, 2H), 3.92 (s, 3H), 2.78 (s, 3H). IR (diamond ATR, cm<sup>–1</sup>): 3325 (w), 3313 (w), 3296 (w), 3086 (w), 3020 (w), 2976 (w), 2931 (w), 2224 (m), 1606 (m), 1568 (s), 1441 (s), 1282 (m), 1155 (w), 1117 (w), 1030 (w), 955 (w), 800 (w). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 413.</div></div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Compound <b>8</b></h4><div id="sec4_2_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 4-Chloro-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-5-(trifluoromethyl)pyrimidin-2-amine</h5><div class="NLM_p last">To a solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (527 mg, 2.43 mmol; Combi-Blocks, Inc.) in <i>t</i>-BuOH (8.2 mL) and DCE (8.2 mL), 1 M zinc chloride in diethyl ether (2.43 mL, 2.43 mmol) was added. The resulting mixture was stirred for 1 h at rt. Then, 3-[(methylsulfonyl)methyl]aniline (300 mg, 1.62 mmol) was added, followed by a solution of triethylamine (0.339 mL, 2.43 mmol) in <i>t</i>-BuOH (0.55 mL) and DCE (0.55 mL). After 5 h, the mixture was diluted with DCM and washed with aq citric acid solution–brine, aq NaHCO<sub>3</sub> solution, and brine. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to afford the desired product (662 mg). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.59 (s, 1H), 7.76 (s, 1H), 7.59 (br dd, <i>J</i> = 7.54, 8.85 Hz, 2H), 7.42 (t, <i>J</i> = 7.91 Hz, 1H), 7.19 (d, <i>J</i> = 7.54 Hz, 1H), 4.28 (s, 2H), 2.84 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 366.</div></div><div id="sec4_2_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 4-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-5-(trifluoromethyl)pyrimidin-2-amine (<b>8</b>)</h5><div class="NLM_p last">A mixture of 4-chloro-<i>N</i>-{3-[(methylsulfonyl)methyl]phenyl}-5-(trifluoromethyl)pyrimidin-2-amine (100 mg, 0.27 mmol), (4-fluoro-2-methoxyphenyl)boronic acid (51 mg, 0.3 mmol), and bis(triphenylphosphine)palladium(II) chloride (8 mg, 0.01 mmol) in DME (3.6 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (1.13 mL) was degassed using argon. The mixture was stirred under argon for 96 h at 105 °C. After cooling, the mixture was evaporated to dryness, and the residue was purified by preparative HPLC (acidic conditions) to afford <b>8</b> (14 mg, 0.03 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.72 (s, 1H), 7.83 (s, 1H), 7.57 (dd, <i>J</i> = 1.26, 8.08 Hz, 1H), 7.47 (s, 1H), 7.39 (t, <i>J</i> = 7.83 Hz, 1H), 7.21 (dd, <i>J</i> = 6.32, 8.34 Hz, 1H), 7.13 (d, <i>J</i> = 7.58 Hz, 1H), 6.76 (dt, <i>J</i> = 2.27, 8.21 Hz, 1H), 6.71 (dd, <i>J</i> = 2.27, 10.61 Hz, 1H), 4.24 (s, 2H), 3.77 (s, 3H), 2.74 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.8, 163.6, 160.7, 157.8, 156.8, 139.3, 130.5, 129.5, 128.8, 125.7, 124.1, 122.4, 122.4, 120.1, 113.8, 106.5, 99.7, 59.6, 56.0, 39.4. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>F<sub>4</sub>N<sub>3</sub>O<sub>3</sub>S, 456.1005; found, 456.1002.</div></div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Compound <b>9</b> and BAY-332</h4><div id="sec4_2_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 1-Fluoro-3-[(methylsulfanyl)methyl]-5-nitrobenzene</h5><div class="NLM_p last">Sodium methanethiolate (5.18 g, 73.8 mmol, 1.4 equiv) was added portionwise at 0 °C to a solution of 1-(chloromethyl)-3-fluoro-5-nitrobenzene (10.0 g, 52.8 mmol) in EtOH (110 mL). After stirring for a further 20 min at that temperature, the mixture was warmed to rt and stirred for a further 18 h. The suspension was then poured into dilute aq NaCl solution, and the mixture was extracted with EtOAc. The organic extract was filtered through a Whatman filter, and the filtrate was concentrated under reduced pressure. The residue was taken up in DCM and again concentrated under reduced pressure. The oily residue was taken up in EtOAc and washed with dilute aq NaCl solution, filtered through a Whatman filter, and concentrated under reduced pressure to afford the title compound (9.86 g, 49.0 mmol, 95% purity, 93% yield) as a brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.00 (s, 1H), 7.80–7.85 (m, 1H), 7.42 (dt, <i>J</i> = 8.59, 1.89 Hz, 1H), 3.74 (s, 2H, SCH<sub>2</sub>), 2.03 (s, 3H, SCH<sub>3</sub>). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.8, 163.6, 160.7, 157.8, 156.8, 139.3, 130.5, 129.5, 128.8, 125.7, 124.1, 122.4, 122.4, 120.1, 113.8, 106.5, 99.7, 59.6, 56.0, 39.4.</div></div><div id="sec4_2_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (<i>rac</i>)-1-Fluoro-3-[(methylsulfinyl)methyl]-5-nitrobenzene</h5><div class="NLM_p last">To a solution of 1-fluoro-3-[(methylsulfanyl)methyl]-5-nitrobenzene (10.3 g, 51.3 mmol) in MeCN (120 mL) was added FeCl<sub>3</sub> at rt (241 mg, 1.49 mmol, 3 mol %), and the mixture was stirred for 10 min at that temperature. Then, periodic acid (12.5 g, 54.8 mmol, 1.07 equiv) was added in one portion, and vigorous stirring was continued for 90 min during which the temperature of the mixture was maintained below 30 °C. The resulting suspension was poured into a solution of sodium thiosulfate pentahydrate (71.2 g) in ice–water (652 mL), and the mixture was stirred for 50 min. Then, NaCl was added until saturation, and the mixture was extracted THF (2×). The combined organic extracts were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was stripped off several times with toluene and DCM and was then purified by flash chromatography (silica gel, EtOAc/hexane gradient) to afford the title compound (8.00 g, 36.8 mmol, 92% purity, 72% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 8.04–8.09 (m, 2H), 7.61–7.65 (m, 1H), 4.32 (d, <i>J</i> = 12.81 Hz, 1H, sulfoxide–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.08 (d, <i>J</i> = 12.81 Hz, 1H, sulfoxide–CH<sub>2</sub>, H<sub>B</sub>, AB system), 2.46 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 218.</div></div><div id="sec4_2_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (<i>rac</i>)-2,2,2-Trifluoro-<i>N</i>-[(3-fluoro-5-nitrobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]acetamide</h5><div class="NLM_p last">To a suspension of (<i>rac</i>)-1-fluoro-3-[(methylsulfinyl)methyl]-5-nitrobenzene (8.00 g, 36.8 mmol), 2,2,2-trifluoroacetamide (8.33 g, 73.7 mmol, 2.0 equiv), magnesium oxide (5.94 g, 147 mmol, 4.0 equiv), and dirhodium tetraacetate (0.76 g, 3.6 mmol, 10 mol %) in DCM (1.0 L) was added iodosobenzene diacetate (17.8 g, 55.2 mmol, 1.5 equiv) at rt. The mixture was stirred at rt for 3 days, after which the mixture was filtered and the filtrate was concentrated under reduced pressure. The resulting dark oil was purified by column chromatography (silica gel, EtOAc/hexane gradient) to afford the title compound as a light yellow oil, which crystallized upon standing (10.8 g, 33.0 mmol, 90% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.04–8.16 (m, 2H), 7.55 (dt, <i>J</i> = 7.91, 1.88 Hz, 1H), 4.92 (d, <i>J</i> = 13.94 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.75 (d, <i>J</i> = 13.94 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.32 (s, 3H, sulfoximine–CH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 329.</div></div><div id="sec4_2_9_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>rac</i>)-1-Fluoro-3-[(<i>S</i>-methylsulfonimidoyl)methyl]-5-nitrobenzene</h5><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (22.7 g, 164 mmol, 5.0 equiv) was added at rt to a solution of (<i>rac</i>)-2,2,2-trifluoro-<i>N</i>-[(3-fluoro-5-nitrobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]acetamide (10.8 g, 33.0 mmol) in MeOH (700 mL). After stirring for 1 h at that temperature, the mixture was concentrated at 30 °C to 100 mL. After the addition of brine, the mixture was extracted EtOAc. The organic extract was filtered through a Whatman filter, and the filtrate was concentrated under reduced pressure to afford the crude product (9.36 g), which was directly used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 8.18 (s, 1H), 8.07 (dt, <i>J</i> = 8.84, 2.27 Hz, 1H), 7.76 (d, <i>J</i> = 9.12 Hz, 1H), 4.60 (d, <i>J</i> = 13.14 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.50 (d, <i>J</i> = 13.14 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.87 (s, 1H, NH), 2.79 (s, 3H, sulfoximine–CH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 233.</div></div><div id="sec4_2_9_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (<i>rac</i>)-Ethyl [(3-Fluoro-5-nitrobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate</h5><div class="NLM_p last">(<i>rac</i>)-1-Fluoro-3-[(<i>S</i>-methylsulfonimidoyl)methyl]-5-nitrobenzene (9.36 g) from the previous step was dissolved in pyridine (380 mL) and cooled to 0 °C. Ethyl chloroformate (5.01 mL, 52.4 mmol, ca. 1.3 equiv) was added dropwise at 0 °C to this mixture, after which the mixture was warmed to rt. After stirring for 16 h at that temperature, the mixture was concentrated under reduced pressure. The residue was taken up in EtOAc and washed with brine and then filtered through a Whatman filter. The filtrate was concentrated under reduced pressure and the crude product was purified by flash chromatography (silica gel, EtOAc/hexane) to afford the title compound (7.09 g, 23.3 mmol, 71% yield over two steps). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, 24 °C): δ 8.11 (s, 1H), 8.01 (br d, <i>J</i> = 7.91 Hz, 1H), 7.57 (br d, <i>J</i> = 7.72 Hz, 1H), 4.90 (d, <i>J</i> = 13.94 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.76 (d, <i>J</i> = 13.94 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 4.08–4.22 (q, <i>J</i> = 7.16 Hz, 2H, OC<i>H</i><sub>2</sub>CH<sub>3</sub>), 3.12 (s, 3H, sulfoximine–CH<sub>3</sub>), 1.22–1.35 (t, <i>J</i> = 7.16 Hz, 3H, OCH<sub>2</sub>C<i>H</i><sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 305.</div></div><div id="sec4_2_9_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (<i>rac</i>)-Ethyl [(3-Amino-5-fluorobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate</h5><div class="NLM_p last">TiCl<sub>3</sub> solution (15% in 10% HCl; 158 mL, 186 mmol, 8.0 equiv) was added dropwise at 0 °C to a solution of (<i>rac</i>)-ethyl [(3-fluoro-5-nitrobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate (7.09 g, 23.3 mmol) in THF (300 mL). Then, the mixture was warmed to rt and stirred for 16 h at that temperature. Then, ice was added to the mixture, which was stirred vigorously, after which K<sub>2</sub>CO<sub>3</sub> was added until the evolution of gas stopped. After that, the mixture was basified with 2 N NaOH. The resulting dark mixture was saturated with NaCl and extracted with EtOAc (3 × 500 mL). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure to afford the title compound (6.37 g, quant) as a yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, 27 °C): δ 6.38–6.55 (m, 3H), 4.52–4.64 (m, 2H, sulfoximine–CH<sub>2</sub>), 4.08–4.22 (q, <i>J</i> = 7.16 Hz, 2H, OC<i>H</i><sub>2</sub>CH<sub>3</sub>), 3.91 (br s, 2H, NH<sub>2</sub>), 3.00 (s, 3H, sulfoximine–CH<sub>3</sub>), 1.31 (t, <i>J</i> = 7.16 Hz, 3H, OCH<sub>2</sub>C<i>H</i><sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 275.</div></div><div id="sec4_2_9_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>rac</i>)-Ethyl [(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate</h5><div class="NLM_p last">A mixture of (<i>rac</i>)-ethyl [(3-amino-5-fluorobenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate (436 mg, 1.59 mmol, 1.0 equiv), 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (530 mg, 2.07 mmol, 1.3 equiv), XPhos Pd G1 (98.6 mg, 0.12 mmol, 7.5 mol %), XPhos (56.8 mg, 0.12 mmol, 7.5 mol %), and finely ground potassium phosphate (1.69 g, 7.95 mmol, 5.0 equiv) in toluene (8.5 mL) and NMP (1.3 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 3 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc (2 × 100 mL). The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was purified by silica gel flash chromatography (first chromatography: DCM to DCM/EtOH 93:7 gradient; second chromatography: DCM to DCM/EtOH 95:5 gradient; third chromatography: hexane to hexane/EtOAc 1:1 gradient) to afford the title compound (271 mg, 0.55 mmol, 35% yield) as a beige solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.32–8.35 (m, 1H), 7.69 (br dd, <i>J</i> = 11.24, 1.89 Hz, 1H), 7.47 (t, <i>J</i> = 7.11 Hz, 1H), 7.38 (s, 1H), 7.28–7.34 (m, 1H), 6.72–6.86 (m, 3H), 4.68 (s, 2H, sulfoximine–CH<sub>2</sub>), 4.17 (q, <i>J</i> = 7.07 Hz, 2H, OC<i>H</i><sub>2</sub>CH<sub>3</sub>), 3.87 (s, 3H, OCH<sub>3</sub>), 3.00 (s, 3H, sulfoximine–CH<sub>3</sub>), 1.31 (t, <i>J</i> = 7.07 Hz, 3H, OCH<sub>2</sub>C<i>H</i><sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 495.</div></div><div id="sec4_2_9_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-fluoro-5-[(<i>S</i>-methylsulfonimidoyl)methyl]phenyl}pyrimidin-2-amine [Enantiomer 1 (<b>9</b>) and Enantiomer 2 (BAY-332)]</h5><div class="NLM_p">To a solution of (<i>rac</i>)-ethyl [(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]carbamate (269 mg, 0.54 mmol) in EtOH (5.0 mL) was added at rt a solution of 1.5 M ethanolic sodium ethoxide (1.81 mL, 2.72 mmol, 5.0 equiv) freshly prepared by reacting sodium (350 mg) with EtOH (10 mL). The reaction mixture was heated to 60 °C for 4.5 h, after which the mixture was cooled to rt and diluted with brine. The mixture was extracted with EtOAc, and the combined organic extracts were filtered through a Whatman filter. The filtrate was concentrated under reduced pressure and the crude product was purified by chiral preparative HPLC [Dionex P 580 pump, Gilson 215 liquid handler, Knauer K-2501 UV detector; column: CHIRALPAK IC 5 μm, 250 × 20 mm; eluent: hexane/EtOH 80:20 (v/v); flow: 50 mL/min; rt; solution: 243 mg in 4 mL DCM/MeOH; injection: 8 × 0.5 mL; and detection: UV 280 nm].</div><div class="NLM_p">Enantiomer 1 (<b>9</b>): <i>t</i><sub>R</sub> = 9.15 min. Yield: 85 mg, 0.20 mmol (37%). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 423. [α]<sub>D</sub> +16.6 ± 0.21 (<i>c</i> 1.00, DMSO).</div><div class="NLM_p last">Enantiomer 2 (BAY-322): <i>t</i><sub>R</sub> = 10.36 min. Yield: 91 mg, 0.22 mmol (40%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.35 (d, <i>J</i> = 1.76 Hz, 1H, aniline–NH), 7.76 (d, <i>J</i> = 11.47 Hz, 1H), 7.51 (t, <i>J</i> = 7.26 Hz, 1H), 7.30–7.37 (m, 2H), 6.77–6.87 (m, 3H), 4.35 (d, <i>J</i> = 13.05 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.23 (d, <i>J</i> = 13.05 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.90 (s, 3H, OCH<sub>3</sub>), 2.97 (s, 3H, sulfoximine–CH<sub>3</sub>), 2.73 (br s, 1H, sulfoximine–NH). <sup>13</sup>C NMR (100.56 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.25 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 246.90 Hz, 1C<sub>q</sub>), 162.13 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 238.47 Hz, 1C<sub>q</sub>), 158.73 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.73 Hz, 1C<sub>q</sub>), 156.30 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.68 Hz, 1C<sub>q</sub>), 151.70 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 16.10 Hz, 1C<sub>q</sub>), 150.53 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.50 Hz, 1C<sub>q</sub>), 145.69 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 24.92 Hz, 1CH), 142.29 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 12.27 Hz, 1C<sub>q</sub>), 133.00 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 9.97 Hz, 1C<sub>q</sub>), 132.15 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.35 Hz, 1CH), 118.98–119.10 (m, 1C<sub>q</sub>), 116.73–116.93 (m, 1CH), 110.24 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.62 Hz, 1CH), 107.45 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.85 Hz, 1CH), 104.73 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.45 Hz, 1CH), 100.34 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.07 Hz, 1CH), 62.39 (s, 1CH<sub>2</sub>), 56.46 (s, 1CH<sub>3</sub>), 41.27 (s, 1CH<sub>3</sub>). IR (KBr, cm<sup>–1</sup>): 3428 (br s), 3286 (br s), 3102 (w), 2969 (w), 2923 (w), 1612 (s), 1544 (m), 1434 (s), 1316 (m), 1285 (m), 1213 (m), 1154 (m), 1030 (m), 956 (m), 837 (m), 786 (m). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 423. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S, 423.1103; found, 423.1099. [α]<sub>D</sub> −18.8 ± 0.16 (<i>c</i> 1.00, DMSO).</div></div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Compound <b>10</b></h4><div id="sec4_2_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 1-Methoxy-3-[(<i>Z</i>)-{3-methoxy-5-[(methylsulfanyl)methyl]phenyl}-<i>NNO</i>-azoxy]-5-[(methylsulfanyl)methyl]benzene</h5><div class="NLM_p last">Sodium methanethiolate (2.58 g, 36.8 mmol) was added under stirring in three portions to a solution of 1-(chloromethyl)-3-methoxy-5-nitrobenzene (5.30 g, 26.3 mmol; FCH Group) in EtOH (60 mL) at 0 °C. The cold bath was removed, and the mixture was stirred at rt overnight. Further sodium methanethiolate (0.92 g, 13.1 mmol) was added, and the mixture was stirred overnight. Further sodium methanethiolate (1.66 g, 23.6 mmol) was added, and the mixture was stirred overnight. The mixture was diluted with saturated aq NaCl solution and extracted with EtOAc (2×). The combined organic phases were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by chromatography (hexane to hexane/EtOAc 7:3) to afford the product (2.9 g, 7.66 mmol, 29% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 7.85 (m, 1H), 7.73 (m, 2H), 7.68 (m, 1H), 7.08 (m, 1H), 6.96 (m, 1H), 3.91 (s, 3H, OCH<sub>3</sub>), 3.87 (s, 3H, OCH<sub>3</sub>), 3.73 (s, 2H, SCH<sub>2</sub>), 3.70 (s, 2H, SCH<sub>2</sub>), 2.04 (s, 3H, SCH<sub>3</sub>), 2.03 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 379.</div></div><div id="sec4_2_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 3-Methoxy-5-[(methylsulfanyl)methyl]aniline</h5><div class="NLM_p last">To a solution of 1-methoxy-3-[(<i>Z</i>)-{3-methoxy-5-[(methylsulfanyl)methyl]phenyl}-<i>NNO</i>-azoxy]-5-[(methylsulfanyl)methyl]benzene (2.80 g, 7.40 mmol) in 1,4-dioxane (60 mL) was added iron (5.98 g, 107 mmol, 14 equiv) at rt. After heating to reflux, concd HCl (26 mL) was added dropwise over 3 h to the mixture. Then, the mixture was cooled to rt and stirring was continued for 14 h. After that, the reaction mixture was poured onto ice and the resulting mixture was basified to pH 9 by the addition of 2 N aq NaOH. The mixture was extracted with EtOAc and the organic extract was filtered through a Whatman filter. After concentration of the filtrate under reduced pressure, the crude product was purified by column chromatography (silica gel, hexane/EtOAc gradient) to afford the title compound (1.54 g, quant) as a brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 6.28 (m, 2H), 6.12 (m, 1H), 3.76 (s, 3H, OCH<sub>3</sub>), 3.66 (br, 2H, NH<sub>2</sub>), 3.55 (s, 2H, SCH<sub>2</sub>), 2.02 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 184.</div></div><div id="sec4_2_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-methoxy-5-[(methylsulfanyl)methyl]phenyl}pyrimidin-2-amine</h5><div class="NLM_p last">A mixture of 3-methoxy-5-[(methylsulfanyl)methyl]aniline (436 mg, 1.59 mmol, 1.0 equiv), 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (840 mg, 3.24 mmol, 1.0 equiv), XPhos Pd G1 (203 mg, 0.25 mmol, 7.5 mol %), XPhos (117 mg, 0.25 mmol, 7.5 mol %), and finely ground potassium phosphate (3.48 g, 16.4 mmol, 5.0 equiv) in toluene (19 mL) and NMP (4 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 3 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc. The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was purified by flash chromatography (silica gel, hexane to hexane/EtOAc 7:3 gradient), recrystallized from hexane/EtOAc, and again purified by flash chromatography (silica gel, hexane to hexane/EtOAc 8:2 gradient). The product-containing fractions were concentrated and the precipitate was collected by filtration to afford the title compound (655 mg, 1.62 mmol, 50% yield) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.30 (m, 1H), 7.51 (m, 1H), 7.36 (m, 1H), 7.15 (br, 1H), 7.03 (m, 1H), 6.80 (m, 1H), 6.74 (m, 1H), 6.55 (m, 1H), 3.86 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>), 3.63 (s, 2H, SCH<sub>2</sub>), 2.02 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 404.</div></div><div id="sec4_2_10_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (<i>rac</i>)-[(3-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}-5-methoxybenzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]cyanamide</h5><div class="NLM_p last">To a mixture of 5-fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-methoxy-5-[(methylsulfanyl)methyl]phenyl}pyrimidin-2-amine (638 mg, 1.58 mmol) and cyanamide (133 mg, 3.16 mmol, 2.0 equiv) in DCM (20 mL) was added at 0 °C iodosobenzene diacetate (560 mg, 1.74 mmol, 1.1 equiv). After stirring for 3 h at that temperature, the mixture was concentrated under reduced pressure and the residue was purified by flash chromatography (silica gel, DCM to DCM/EtOH 9:1 gradient) to afford the title compound (410 mg, 0.92 mmol, 58% yield) as a brownish solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.32 (m, 1H), 7.47 (m, 1H), 7.33 (m, 1H), 7.25 (m, 2H), 6.82 (m, 1H), 6.75 (m, 1H), 6.52 (m, 1H), 4.37 (d, <i>J</i> = 12.6 Hz, 1H), 4.13 (d, <i>J</i> = 12.6 Hz, 1H), 3.86 (s, 3H), 3.81 (s, 3H), 2.71 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 444.</div></div><div id="sec4_2_10_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (<i>rac</i>)-[(3-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}-5-methoxybenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]cyanamide</h5><div class="NLM_p last">To a mixture of (<i>rac</i>)-[(3-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}-5-methoxybenzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]cyanamide (394 mg, 0.89 mmol) in acetone (10 mL) was added at rt KMnO<sub>4</sub> (281 mg, 1.78 mmol, 2.0 equiv), and the mixture was heated to 50 °C for 1 h. After cooling to rt, the mixture was concentrated under reduced pressure and purified by flash chromatography (silica gel, DCM to DCM/EtOH 95:5 gradient) to afford the title compound (307 mg, 0.67 mmol, 75% yield) as a light brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 9.92 (m, 1H), 8.53 (m, 1H), 7.53 (m, 2H), 7.34 (m, 1H), 7.09 (m, 1H), 6.92 (m, 1H), 6.62 (m, 1H), 4.87 (m, 2H), 3.80 (s, 3H), 3.70 (s, 3H), 3.31 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 460.</div></div><div id="sec4_2_10_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (<i>rac</i>)-5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-methoxy-5-[(<i>S</i>-methylsulfonimidoyl)methyl]phenyl}pyrimidin-2-amine (<b>10</b>)</h5><div class="NLM_p last">To a solution of (<i>rac</i>)-[(3-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}-5-methoxybenzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]cyanamide (302 mg, 0.66 mmol) in DCM (30 mL) was added at 0 °C trifluoroacetic anhydride (TFAA, 0.28 mL, 1.97 mmol, 3.0 equiv). The mixture was warmed to rt and stirred for 2 h at that temperature. Then, the mixture was concentrated under reduced pressure (at 40 °C), the residue was taken up in MeOH (5 mL), and K<sub>2</sub>CO<sub>3</sub> (454 mg, 3.29 mmol, 5.0 equiv) was added at rt. After stirring for 2 h at rt and 14 h at 4 °C, the mixture was diluted with EtOAc and THF and washed with brine. The organic phase was filtered through a Whatman filter, the filtrate was concentrated under reduced pressure, and the crude product was purified by preparative HPLC (basic conditions) to afford <b>10</b> (27 mg, 0.06 mmol, 9% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.30 (d, <i>J</i> = 2.02 Hz, 1H), 7.49 (t, <i>J</i> = 7.18 Hz, 1H), 7.43–7.46 (m, 1H), 7.25–7.29 (m, 2H), 7.19 (s, 1H), 6.72–6.83 (m, 2H), 6.59–6.63 (m, 1H), 4.32 (d, <i>J</i> = 13.14 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.20 (d, <i>J</i> = 12.88 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.86 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 2.93 (s, 3H, SCH<sub>3</sub>). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.36 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 246.68 Hz, 1C<sub>q</sub>), 159.34 (s, 1C<sub>q</sub>), 158.54 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 156.49 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.54 Hz, 1C<sub>q</sub>), 151.35 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 16.95 Hz, 1C<sub>q</sub>), 150.19 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.35 Hz, 1C<sub>q</sub>), 145.83 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 24.58 Hz, 1CH), 141.52 (s, 1C<sub>q</sub>), 132.03 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1CH), 131.79 (s, 1C<sub>q</sub>), 119.1 (m, 1C<sub>q</sub>), 113.56 (s, 1CH), 109.59 (s, 1CH), 107.30 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 104.10 (s, 1CH), 100.17 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 62.82 (s, 1CH<sub>2</sub>), 56.36 (s, 1CH<sub>3</sub>), 55.00 (s, 1CH<sub>3</sub>), 41.00 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3304 (w), 3256 (w), 3103 (w), 2964 (w), 2943 (w), 1601 (m), 1545 (m), 1454 (m), 1429 (s), 1310 (m), 1150 (m), 1026 (s), 957 (m), 851 (m), 829 (m). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub>S, 435.1303; found, 435.1300.</div></div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Compounds <b>11</b> and <b>12</b></h4><div id="sec4_2_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> (2,3-Difluoro-5-nitrophenyl)methanol</h5><div class="NLM_p last">To a solution of 2,3-difluoro-5-nitrobenzoic acid (9.00 g, 44.3 mmol) in THF (85 mL) was added dropwise at 0 °C 1 M borane in THF (177 mL, 177 mmol, 4.0 equiv). After the addition, the mixture was warmed to rt and stirred for 14 h at that temperature. Then, the mixture was cooled to 0 °C and MeOH was added carefully until evolution of gas ceased. The mixture was then diluted with EtOAc, washed with 1 N aq NaOH and brine, and then dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent under reduced pressure, the crude product was purified by flash chromatography (silica gel, hexane/EtOAc gradient) to afford the title compound as a yellowish oil which crystallized upon standing (8.20 g, 43.4 mmol, 98% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.26 (m, 1H), 8.03 (m, 1H), 4.89 (br, 2H), 2.13 (br, 1H). UPLC (ESI−) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> 188.</div></div><div id="sec4_2_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> (5-Amino-2,3-difluorophenyl)methanol</h5><div class="NLM_p last">TiCl<sub>3</sub> solution (ca. 15% in ca. 10% HCl; 144 mL) was added to a stirred solution of (2,3-difluoro-5-nitrophenyl)methanol (4.00 g, 21.2 mmol) in THF (250 mL) at 0 °C, after which the mixture was warmed to rt and stirred for 16 h at that temperature. Then, the mixture was neutralized by the addition of 1 N NaOH and the resulting dark mass was extracted with EtOAc. The combined organic phases were washed with brine, filtered using a Whatman filter, and concentrated under reduced pressure. The crude product was purified by flash chromatography (silica gel, hexane/EtOAc gradient) to afford the title compound (3.33 g, 20.9 mmol, 99% yield) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 6.47 (m, 1H), 6.42 (m, 1H), 4.69 (br, 2H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 160.</div></div><div id="sec4_2_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 3-(Chloromethyl)-4,5-difluoroaniline</h5><div class="NLM_p last">To a solution of (5-amino-2,3-difluorophenyl)methanol (4.13 g, 26.0 mmol) in DCM (78 mL) and NMP (11 mL) was added at rt thionyl chloride (4.73 mL, 64.9 mmol, 2.5 equiv). After stirring for 14 h at that temperature, the mixture was poured onto a mixture of ice/NaHCO<sub>3</sub>/NaCl and stirred for 2 h. After cessation of the reaction, the mixture was extracted with EtOAc, and the organic extracts were filtered through a Whatman filter and concentrated under reduced pressure to afford the title compound as a crude product which was directly used in the next step.</div></div><div id="sec4_2_11_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 3,4-Difluoro-5-[(methylsulfanyl)methyl]aniline</h5><div class="NLM_p last">Sodium methanethiolate (3.61 g, 51.6 mmol) was added portionwise to a solution of 3-(chloromethyl)-4,5-difluoroaniline from the previous step in EtOH (75 mL) at 0 °C. The mixture was warmed to rt and stirred for 14 h at that temperature. After that, the mixture was diluted with brine and extracted with EtOAc. The organic extracts were filtered through a Whatman filter and concentrated under reduced pressure. The residue was purified by flash chromatography (silica gel, hexane/EtOAc gradient) to afford the title compound (1.27 g, 6.71 mmol, 26% yield over two steps) as a brownish oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 6.36 (m, 2H), 3.62 (br, 4H, SCH<sub>2</sub>, NH<sub>2</sub>), 2.08 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 190.</div></div><div id="sec4_2_11_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> <i>N</i>-{3,4-Difluoro-5-[(methylsulfanyl)methyl]phenyl}-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine</h5><div class="NLM_p last">A mixture of 3,4-difluoro-5-[(methylsulfanyl)methyl]aniline (600 mg, 3.17 mmol, 1.0 equiv), 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (1.06 g, 4.12 mmol, 1.3 equiv), XPhos Pd G1 (197 mg, 0.24 mmol, 7.5 mol %), XPhos (113 mg, 0.24 mmol, 7.5 mol %), and finely ground potassium phosphate (3.37 g, 15.9 mmol, 5.0 equiv) in toluene (21 mL) and NMP (4 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 3 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc. The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was purified by flash chromatography (silica gel, hexane to hexane/EtOAc gradient) to afford the title compound (724 mg, 1.77 mmol, 56% yield) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.30 (d, <i>J</i> = 2.02 Hz, 1H), 7.71 (ddd, <i>J</i> = 12.63, 6.82, 2.78 Hz, 1H), 7.48 (dd, <i>J</i> = 8.46, 6.69 Hz, 1H), 7.07–7.22 (m, 2H), 6.78–6.84 (m, 1H), 6.76 (d, <i>J</i> = 10.85 Hz, 1H), 3.87 (s, 3H, OCH<sub>3</sub>), 3.70 (s, 2H, SCH<sub>2</sub>), 2.06 (s, 3H, SCH<sub>3</sub>).</div></div><div id="sec4_2_11_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> (<i>rac</i>)-[(2,3-Difluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]cyanamide</h5><div class="NLM_p last">To a mixture of <i>N</i>-{3,4-difluoro-5-[(methylsulfanyl)methyl]phenyl}-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (715 mg, 1.75 mmol) and cyanamide (147 mg, 3.49 mmol, 2.0 equiv) in DCM (10 mL) was added at 0 °C iodosobenzene diacetate (619 mg, 1.92 mmol, 1.1 equiv). After stirring for 4 h at that temperature, the mixture was concentrated under reduced pressure and the residue was purified by flash chromatography (silica gel, DCM to DCM/EtOH 8:2 gradient) to afford the title compound (655 mg, 1.46 mmol, 83% yield) as a pinkish solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 1.88 Hz, 1H), 7.89 (ddd, <i>J</i> = 12.53, 7.06, 2.64 Hz, 1H), 7.75 (s, 1H), 7.47 (dd, <i>J</i> = 8.29, 6.78 Hz, 1H), 7.28–7.37 (m, 1H), 6.73–6.87 (m, 2H), 4.33 (m, 2H, SCH<sub>2</sub>), 3.87 (s, 3H, OCH<sub>3</sub>), 2.82 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 450.</div></div><div id="sec4_2_11_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (<i>rac</i>)-[(2,3-Difluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]cyanamide (<b>12</b>)</h5><div class="NLM_p last">To a mixture of (<i>rac</i>)-[(2,3-difluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]cyanamide (645 mg, 1.44 mmol) in acetone was added at rt KMnO<sub>4</sub> (454 mg, 2.87 mmol, 2.0 equiv). The mixture was warmed to 50 °C for 1 h, after which it was cooled to rt and concentrated under reduced pressure. The residue was purified by flash chromatography (silica gel, hexane/EtOAc gradient) to afford <b>12</b> (372 mg, 0.80 mmol, 56% yield) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.35 (d, <i>J</i> = 2.02 Hz, 1H), 7.94 (ddd, <i>J</i> = 12.44, 7.14, 2.65 Hz, 1H), 7.43–7.52 (m, 2H), 7.33 (s, 1H), 6.83 (t, <i>J</i> = 8.11 Hz, 1H), 6.76 (d, <i>J</i> = 10.68 Hz, 1H), 4.65–4.73 (m, 2H, SCH<sub>2</sub>), 3.87 (s, 3H, OCH<sub>3</sub>), 3.12 (s, 3H, SCH<sub>3</sub>). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.44 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 247.11 Hz, 1C<sub>q</sub>), 158.59 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1C<sub>q</sub>), 155.96 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.54 Hz, 1C<sub>q</sub>), 151.68 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 16.11 Hz, 1C<sub>q</sub>), 150.48 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.77 Hz, 1C<sub>q</sub>), 149.15 (dd, <sup>1</sup><i>J</i><sub>CF</sub> = 242.44 Hz, <sup>2</sup><i>J</i><sub>CF</sub> = 13.56 Hz, 1C<sub>q</sub>), 145.91 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.07 Hz, 1CH), 143.43 (dd, <sup>1</sup><i>J</i><sub>CF</sub> = 244.99 Hz, <sup>2</sup><i>J</i><sub>CF</sub> = 13.56 Hz, 1C<sub>q</sub>), 137.32 (dd, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 132.07 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1CH), 118.77 (m, 1C<sub>q</sub>), 117.56 (m, 1CH), 115.74 (m, 1C<sub>q</sub>), 111.99 (s, 1C<sub>q</sub>), 108.30 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 107.38 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 100.25 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 56.35 (s, 1CH<sub>3</sub>), 53.60 (s, 1CH<sub>2</sub>), ∼39.5 (s, overlapped by solvent signals). IR (diamond ATR, cm<sup>–1</sup>): 3292 (w), 3096 (w), 3001 (w), 2924 (w), 2193 (s), 1612 (m), 1593 (m), 1508 (s), 1423 (s), 1242 (s), 1192 (s), 1024 (s), 982 (m), 957 (m), 895 (m), 829 (m), 781 (m). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 465. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>F<sub>4</sub>N<sub>5</sub>O<sub>2</sub>S, 466.0962; found, 466.0958.</div></div><div id="sec4_2_11_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (<i>rac</i>)-<i>N</i>-{3,4-Difluoro-5-[(<i>S</i>-methylsulfonimidoyl)methyl]phenyl}-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (<b>11</b>)</h5><div class="NLM_p last">To a solution of (<i>rac</i>)-[(2,3-difluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)oxido-λ<sup>6</sup>-sulfanylidene]cyanamide (<b>12</b>; 344 mg, 0.74 mmol) in DCM (30 mL) was added at 0 °C TFAA (0.78 mL, 5.54 mmol, 7.5 equiv). The mixture was warmed to rt and stirred for 2 h at that temperature. Then, the mixture was concentrated under reduced pressure (at 40 °C), the residue was taken up in MeOH (5 mL), and K<sub>2</sub>CO<sub>3</sub> (511 mg, 3.70 mmol, 5.0 equiv) was added at rt. After stirring for 2 h at rt, the mixture was diluted with EtOAc and THF and washed with brine. The organic phase was filtered through a Whatman filter, the filtrate was concentrated under reduced pressure, and the crude product was purified by flash chromatography (silica gel, DCM to DCM/EtOH 8:2 gradient) to afford <b>11</b> (300 mg, 0.68 mmol, 92% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.30 (m, 1H), 7.86 (m, 1H), 7.47 (m, 1H), 7.26 (m, 2H), 6.82 (m, 1H), 6.76 (m, 1H), 4.42 (d, <i>J</i> = 13.6 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.35 (d, <i>J</i> = 13.6 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.87 (s, 3H, OCH<sub>3</sub>), 2.96 (s, 3H, SCH<sub>3</sub>), 2.76 (s, 1H, sulfoximine–NH). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.40 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 247.11 Hz, 1C<sub>q</sub>), 158.57 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1C<sub>q</sub>), 156.09 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 151.61 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 15.26 Hz, 1C<sub>q</sub>), 150.4 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.34 Hz, 1C<sub>q</sub>), 149.06 (dd, <sup>1</sup><i>J</i><sub>CF</sub> = 241.60 Hz, <sup>2</sup><i>J</i><sub>CF</sub> = 13.56 Hz, 1C<sub>q</sub>), 145.92 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.01 Hz, 1CH), 143.35 (dd, <sup>1</sup><i>J</i><sub>CF</sub> = 242.87 Hz, <sup>2</sup><i>J</i><sub>CF</sub> = 13.14 Hz, 1C<sub>q</sub>), 136.86 (dd, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 132.05 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1CH), 120.02 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 13.56 Hz, 1C<sub>q</sub>), 118.13 (m, 1C<sub>q</sub>), 117.22 (m, 1CH), 107.34 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 12.04 Hz, 1CH), 107.15 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 100.23 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.28 Hz, 1CH), 56.35 (s, 1CH<sub>3</sub>), 56.06 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 1.27 Hz, 1CH<sub>2</sub>), 41.54 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3300 (br s), 3099 (w), 2982 (w), 2926 (w), 1634 (w), 1610 (m), 1597 (m), 1506 (s), 1431 (s), 1327 (m), 1207 (s), 1151 (m), 1026 (m), 959 (m), 870 (m), 858 (m), 829 (m), 783 (m). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>F<sub>4</sub>N<sub>4</sub>O<sub>2</sub>S, 441.1009; found, 441.1007.</div></div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Compound <b>13</b></h4><div id="sec4_2_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> (5-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methanol</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (2.13 g, 7.89 mmol), (5-aminopyridin-3-yl)methanol (1.29 g, 9.87 mmol; ABCR), XPhos Pd G1 (653 mg, 0.79 mmol), XPhos (376 mg, 0.79 mmol), and potassium phosphate (8.38 g, 39.47 mmol) in toluene (210 mL) and NMP (21 mL) was degassed using argon. The mixture was stirred under argon for 3 h at 130 °C. After cooling, the reaction mixture was partitioned between H<sub>2</sub>O and EtOAc. The organic layer was washed with saturated NaCl, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford an oil. Purification by column chromatography (silica gel, hexane to hexane/EtOAc 1:1, then DCM to DCM/EtOAc to EtOAc/MeOH 8:2) afforded the desired product (3.12 g, 9 mmol, 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.96 (s, 1H), 8.79 (d, <i>J</i> = 2.53 Hz, 1H), 8.60 (d, <i>J</i> = 2.02 Hz, 1H), 8.08–8.14 (m, 2H), 7.55 (dd, <i>J</i> = 6.82, 8.59 Hz, 1H), 7.13 (dd, <i>J</i> = 2.40, 11.49 Hz, 1H), 6.96 (dt, <i>J</i> = 2.27, 8.46 Hz, 1H), 5.27 (t, <i>J</i> = 5.56 Hz, 1H), 4.50 (d, <i>J</i> = 5.56 Hz, 2H), 3.84 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 345.</div></div><div id="sec4_2_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> <i>N</i>-[5-(Chloromethyl)pyridin-3-yl]-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine</h5><div class="NLM_p last">(5-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methanol (3.1 g, 9 mmol) was dissolved in DCM (66 mL) and NMP (6.6 mL). The mixture was cooled to 0 °C and thionyl chloride (1.64 mL, 22.5 mmol) was slowly added. The cold bath was removed, and the mixture was stirred at rt for 2 h. Then, the mixture was diluted with EtOAc, washed with saturated NaHCO<sub>3</sub> solution and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford an oil (3.85 g) that was used without further purification. UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 363.</div></div><div id="sec4_2_12_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{5-[(methylsulfanyl)methyl]pyridin-3-yl}pyrimidin-2-amine</h5><div class="NLM_p last"><i>N</i>-[5-(Chloromethyl)pyridin-3-yl]-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (693 mg, 1.91 mmol) was dissolved in EtOH (100 mL) and sodium methanethiolate (268 mg, 3.82 mmol) was added. The mixture was stirred at rt for 3 h. Then, it was diluted with brine and extracted with EtOAc (2×). The combined organic phases were washed with H<sub>2</sub>O, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated to afford the desired product (740 mg, 1.98 mmol) that was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.98 (s, 1H), 8.76 (d, <i>J</i> = 2.53 Hz, 1H), 8.60 (d, <i>J</i> = 2.02 Hz, 1H), 8.16 (t, <i>J</i> = 2.15 Hz, 1H), 8.07 (d, <i>J</i> = 2.02 Hz, 1H), 7.55 (dd, <i>J</i> = 6.82, 8.59 Hz, 1H), 7.13 (dd, <i>J</i> = 2.40, 11.49 Hz, 1H), 6.96 (dt, <i>J</i> = 2.27, 8.46 Hz, 1H), 3.84 (s, 3H), 3.68 (s, 2H), 1.96 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 375.</div></div><div id="sec4_2_12_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> (<i>rac</i>)-{[(5-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methyl](methyl)-λ<sup>4</sup>-sulfanylidene}cyanamide</h5><div class="NLM_p last">To a cooled solution (0 °C) of 5-fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{5-[(methylsulfanyl)methyl]pyridin-3-yl}pyrimidin-2-amine (730 mg, 1.89 mmol) and cyanamide (159 mg, 3.78 mmol) in DCM (22 mL), iodosobenzene diacetate (670 mg, 2.08 mmol) was added. The mixture was stirred at rt for 2.5 h, then diluted with DCM, and washed with brine and sodium thiosulfate solution (10%). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford a foam. The crude product was purified by column chromatography (EtOAc to EtOAc/MeOH 4:1) to afford the desired product (672 mg, 1.62 mmol, 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.15 (s, 1H), 8.91 (d, <i>J</i> = 2.27 Hz, 1H), 8.60 (d, <i>J</i> = 2.02 Hz, 1H), 8.24 (t, <i>J</i> = 2.15 Hz, 1H), 8.16 (d, <i>J</i> = 2.02 Hz, 1H), 7.58 (dd, <i>J</i> = 6.82, 8.34 Hz, 1H), 7.13 (dd, <i>J</i> = 2.27, 11.37 Hz, 1H), 6.97 (dt, <i>J</i> = 2.53, 8.46 Hz, 1H), 4.26–4.56 (m, 2H), 3.84 (s, 3H), 2.86 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 415.</div></div><div id="sec4_2_12_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> (<i>rac</i>)-{[(5-{[5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methyl](methyl)oxido-λ<sup>6</sup>-sulfanylidene}cyanamide</h5><div class="NLM_p last">To a mixture of (<i>rac</i>)-{[(5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methyl](methyl)-λ<sup>4</sup>-sulfanylidene}cyanamide (650 mg, 1.56 mmol) in acetone (15.6 mL) was added at rt KMnO<sub>4</sub> (496 mg, 3.13 mmol), and the mixture was heated to 50 °C for 1 h. After cooling to rt, the mixture was concentrated under reduced pressure and purified by chromatography (silica gel, DCM to DCM/MeOH 4:1 gradient) to afford the title compound (385 mg, 0.89 mmol, 57% yield) as a colorless foam. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.17 (s, 1H), 8.94 (d, <i>J</i> = 2.53 Hz, 1H), 8.61 (d, <i>J</i> = 2.02 Hz, 1H), 8.31 (t, <i>J</i> = 2.15 Hz, 1H), 8.20 (d, <i>J</i> = 2.02 Hz, 1H), 7.56 (dd, <i>J</i> = 6.82, 8.59 Hz, 1H), 7.13 (dd, <i>J</i> = 2.40, 11.49 Hz, 1H), 6.96 (dt, <i>J</i> = 2.40, 8.40 Hz, 1H), 5.01–5.10 (m, 2H), 3.84 (s, 3H), 3.41 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 431.</div></div><div id="sec4_2_12_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> (<i>rac</i>)-5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{5-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-3-yl}pyrimidin-2-amine (<b>13</b>)</h5><div class="NLM_p last">To a cooled (0 °C) solution of (<i>rac</i>)-{[(5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}pyridin-3-yl)methyl](methyl)oxido-λ<sup>6</sup>-sulfanylidene}cyanamide (112 mg, 0.26 mmol) in DCM (11.3 mL), TFAA (0.11 mL, 0.78 mmol) was added. Then, the mixture was stirred at rt for 2 h. The solvent was evaporated and the residue was dissolved in MeOH (1.8 mL) and K<sub>2</sub>CO<sub>3</sub> (180 mg, 1.3 mmol) was added. After 30 min, the mixture was diluted with THF and partitioned between H<sub>2</sub>O and EtOAc. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude material was purified by chromatography (EtOAc/MeOH 3% to EtOAc/MeOH 40%) to afford <b>13</b> (63 mg, 0.16 mmol, 59% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.20 (s, 1H), 8.95 (br s, 1H), 8.62 (d, <i>J</i> = 1.52 Hz, 1H), 8.21–8.32 (m, 2H), 7.56 (t, <i>J</i> = 7.73 Hz, 1H), 7.14 (dd, <i>J</i> = 2.03, 11.41 Hz, 1H), 6.96 (dt, <i>J</i> = 2.28, 8.36 Hz, 1H), 4.68–4.81 (m, 2H), 3.84 (s, 3H), 3.12 (s, 3H). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.41 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 247.11 Hz, 1C<sub>q</sub>), 158.56 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1C<sub>q</sub>), 156.31 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.54 Hz, 1C<sub>q</sub>), 151.64 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 15.26 Hz, 1C<sub>q</sub>), 150.50 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.35 Hz, 1C<sub>q</sub>), 145.97 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.01 Hz, 1CH), 144.22 (s, 1CH), 140.13 (s, 1CH), 136.85 (s, 1C<sub>q</sub>), 132.12 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.60 Hz, 1CH), 127.07 (s, 1CH), 126.39 (s, 1C<sub>q</sub>), 118.92 (m, 1C<sub>q</sub>), 107.36 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.62 Hz, 1CH), 100.18 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.86 Hz, 1CH), 59.58 (s, 1CH<sub>2</sub>), 56.38 (s, 1CH<sub>3</sub>), 41.12 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3302 (w), 3250 (w), 3125 (w), 2972 (w), 2924 (w), 1682 (m), 1599 (m), 1583 (m), 1543 (s), 1427 (s), 1302 (m), 1281 (s), 1198 (s), 1153 (s), 1030 (s), 953 (m), 881 (m), 837 (s), 785 (s). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>F<sub>2</sub>N<sub>5</sub>O<sub>2</sub>S, 406.1194; found, 406.1145.</div></div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Compound <b>14</b></h4><div id="sec4_2_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 3-[(3-Fluoro-5-nitrobenzyl)sulfanyl]propan-1-ol</h5><div class="NLM_p last">Cs<sub>2</sub>CO<sub>3</sub> (3.1 g, 9.5 mmol) was added to a solution of 1-(chloromethyl)-3-fluoro-5-nitrobenzene (1.5 g, 7.91 mmol) and 3-mercaptopropanol (0.81 mL, 11.1 mmol) in DMF (15 mL). The mixture was stirred at rt for 90 min. Then, it was diluted with H<sub>2</sub>O and extracted with EtOAc (2×). The combined organic phases were washed with LiCl solution (5%), saturated NaHCO<sub>3</sub> solution, and brine and dried (Na<sub>2</sub>SO<sub>4</sub>). The filtrate was concentrated to afford the desired product (2.32 g) that was used without further purification. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.02 (s, 1H), 7.82 (td, <i>J</i> = 2.24, 8.15 Hz, 1H), 7.43 (td, <i>J</i> = 2.05, 8.34 Hz, 1H), 3.79 (s, 2H), 3.74 (t, <i>J</i> = 6.03 Hz, 2H), 2.82 (t, <i>J</i> = 7.06 Hz, 1H), 2.57 (t, <i>J</i> = 7.06 Hz, 2H), 1.77–1.90 (m, 2H). UPLC (ESI−) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> 244.</div></div><div id="sec4_2_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 3-[(3-Amino-5-fluorobenzyl)sulfanyl]propan-1-ol</h5><div class="NLM_p last">TiCl<sub>3</sub> solution (ca. 15% in ca. 10% HCl; 19.4 mL) was added to a stirred solution of 3-[(3-fluoro-5-nitrobenzyl)sulfanyl]propan-1-ol (1.2 g, 4.89 mmol) in THF (50 mL) at rt, and the mixture was stirred for 116 h. The pH was increased to 10 by the addition of 1 N NaOH, and then the mixture was extracted with EtOAc (2×). The combined organic phases were washed with brine, filtered using a Whatman filter, and concentrated. The residue was purified by column chromatography (DCM to DCM/EtOH 95:5) to afford the desired product (685 mg, 3.18 mmol, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 6.40–6.45 (m, 2H), 6.27 (td, <i>J</i> = 2.02, 10.36 Hz, 1H), 3.73 (t, <i>J</i> = 6.06 Hz, 2H), 3.60 (s, 2H), 2.56 (t, <i>J</i> = 7.07 Hz, 2H), 1.82 (quin, <i>J</i> = 6.51 Hz, 2H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 216.</div></div><div id="sec4_2_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 3-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfanyl]propan-1-ol</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (989 mg, 3.85 mmol), 3-[(3-amino-5-fluorobenzyl)sulfanyl]propan-1-ol (638 mg, 2.96 mmol), XPhos Pd G1 (184 mg, 0.22 mmol), XPhos (106 mg, 0.22 mmol), and finely ground potassium phosphate (3.14 g, 14.8 mmol) in toluene (19 mL) and NMP (3.9 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 3 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc. The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was purified by preparative HPLC (acidic conditions) to afford the title compound (846 mg, 1.94 mmol, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (d, <i>J</i> = 2.02 Hz, 1H), 7.59 (td, <i>J</i> = 2.21, 11.24 Hz, 1H), 7.49 (dd, <i>J</i> = 6.57, 8.34 Hz, 1H), 7.29 (br s, 1H), 7.16 (s, 1H), 6.82 (dt, <i>J</i> = 2.40, 8.27 Hz, 1H), 6.76 (dd, <i>J</i> = 2.40, 10.74 Hz, 1H), 6.70 (td, <i>J</i> = 1.77, 9.09 Hz, 1H), 3.87 (s, 3H), 3.71 (t, <i>J</i> = 6.06 Hz, 2H), 3.68 (s, 2H), 2.55 (t, <i>J</i> = 7.07 Hz, 2H), 1.77–1.85 (m, 2H). UPLC (ESI−) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> 434.</div></div><div id="sec4_2_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (<i>rac</i>)-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(3-hydroxypropyl)-λ<sup>4</sup>-sulfanylidene]cyanamide</h5><div class="NLM_p last">To a cooled solution (0 °C) of 3-[(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfanyl]propan-1-ol (774 mg, 1.77 mmol) and cyanamide (150 mg, 3.55 mmol) in DCM (15.4 mL), iodosobenzene diacetate (630 mg, 1.95 mmol) was added. The mixture was stirred at rt for 1 h, then diluted with DCM, and washed with brine and sodium thiosulfate solution (10%). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford a foam. The crude product was purified by column chromatography (DCM to DCM/EtOH 88:12) to afford the desired product (623 mg, 1.33 mmol, 74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.30–8.36 (m, 1H), 7.68–7.75 (m, 1H), 7.43–7.54 (m, 2H), 7.32 (s, 1H), 6.80–6.86 (m, 1H), 6.76 (dd, <i>J</i> = 2.02, 10.86 Hz, 1H), 6.64–6.73 (m, 1H), 4.11–4.36 (m, 2H), 3.87 (s, 3H), 3.74–3.84 (m, 2H), 3.02–3.31 (m, 2H), 2.04–2.13 (m, 2H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 476.</div></div><div id="sec4_2_13_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (<i>rac</i>)-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(3-hydroxypropyl)oxido-λ<sup>6</sup>-sulfanylidene]cyanamide (<b>14</b>)</h5><div class="NLM_p last">(<i>rac</i>)-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(3-hydroxypropyl)-λ<sup>4</sup>-sulfanylidene]cyanamide (534 mg, 1.12 mmol) was dissolved in acetone (11 mL). KMnO<sub>4</sub> (355 mg, 2.25 mmol) was added, and the mixture was stirred at 50 °C for 2 h. Then, additional KMnO<sub>4</sub> (89 mg, 0.56 mmol) was added and stirring was continued for 2 h. The mixture was cooled to rt and filtered. The filter cake was rinsed with acetone and EtOH. The combined organic fractions were concentrated. Purification of the crude material by column chromatography (DCM/EtOH 95:5 to 9:1) afforded 14 (11 mg, 2% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 1.77 Hz, 1H), 7.67–7.73 (m, 1H), 7.45–7.55 (m, 3H), 6.74–6.87 (m, 3H), 4.57 (s, 2H), 3.87 (s, 3H), 3.73 (t, <i>J</i> = 5.56 Hz, 2H), 3.27–3.37 (m, 2H), 2.04–2.16 (m, 2H). IR (diamond ATR, cm<sup>–1</sup>): 3271 (w), 3103 (w), 2920 (w), 2192 (s), 1609 (s), 1587 (s), 1506 (m), 1427 (s), 1331 (w), 1281 (m), 1211 (m), 1151 (m), 1049 (m), 1022 (s), 953 (m), 835 (m), 785 (m). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 492. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>21</sub>F<sub>3</sub>N<sub>5</sub>O<sub>3</sub>S, 492.1318; found, 492.1320.</div></div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Compound <b>15</b></h4><div id="sec4_2_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 3-[(3-Fluoro-5-nitrobenzyl)sulfanyl]propan-1-amine</h5><div class="NLM_p last">Cs<sub>2</sub>CO<sub>3</sub> (8.93 g, 27.4 mmol) was added to a solution of 1-(chloromethyl)-3-fluoro-5-nitrobenzene (2 g, 10.5 mmol) and 3-aminopropane-1-thiol hydrochloride (1.87 g, 14.7 mmol) in DMF (30 mL). The mixture was stirred at rt for 5 h. Then, it was diluted with H<sub>2</sub>O and extracted with EtOAc (2×). The combined organic phases were washed with LiCl solution (5%), saturated NaHCO<sub>3</sub> solution, and brine and dried (Na<sub>2</sub>SO<sub>4</sub>). The filtrate was concentrated to afford the desired product (1.93 g, 7.9 mmol, 74% yield) that was used without further purification.</div></div><div id="sec4_2_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> <i>tert</i>-Butyl {3-[(3-Fluoro-5-nitrobenzyl)sulfanyl]propyl}carbamate</h5><div class="NLM_p last">To a solution of 3-[(3-fluoro-5-nitrobenzyl)sulfanyl]propan-1-amine (1.93 g, 7.9 mmol) in <i>t</i>-BuOH, di-<i>tert</i>-butyl dicarbonate (2.29 g, 10.5 mmol) was added. The reaction mixture was stirred at rt for 3 h, then diluted with EtOAc, and concentrated. Purification of the remaining material by column chromatography (hexanes to hexanes/EtOH 2:1) afforded the desired product (1.84 g, 5.35 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.99–8.02 (m, 1H), 7.82 (td, <i>J</i> = 2.24, 8.15 Hz, 1H), 7.43 (td, <i>J</i> = 1.89, 8.59 Hz, 1H), 4.56 (br s, 1H), 3.77 (s, 2H), 3.20 (br t, <i>J</i> = 6.44 Hz, 2H), 2.47 (t, <i>J</i> = 7.20 Hz, 2H), 1.76 (quin, <i>J</i> = 6.95 Hz, 2H), 1.43 (s, 9H).</div></div><div id="sec4_2_14_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>tert</i>-Butyl {3-[(3-Amino-5-fluorobenzyl)sulfanyl]propyl}carbamate</h5><div class="NLM_p last">A solution of <i>tert</i>-butyl {3-[(3-fluoro-5-nitrobenzyl)sulfanyl]propyl}carbamate (1.33 g, 3.82 mmol) in EtOH (40.5 mL) was degassed with argon. 10% palladium on charcoal (3.86 mmol) was added, and the mixture was stirred at rt under an atmosphere of hydrogen for 18 h. The suspension was filtered using a Whatman filter and concentrated. The resulting residue was purified by column chromatography (hexanes to hexanes/EtOAc 1:1) to afford the desired product (216 mg, 0.69 mmol, 18% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 6.39–6.49 (m, 2H), 6.29 (br d, <i>J</i> = 10.55 Hz, 1H), 4.60 (br s, 1H), 3.89 (br s, 2H), 3.60 (s, 2H), 3.20 (q, <i>J</i> = 6.34 Hz, 2H), 2.47 (t, <i>J</i> = 7.25 Hz, 2H), 1.75 (quin, <i>J</i> = 6.92 Hz, 2H), 1.46 (s, 9H).</div></div><div id="sec4_2_14_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>tert</i>-Butyl {3-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfanyl]propyl}carbamate</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (806 mg, 3.14 mmol), <i>tert</i>-butyl {3-[(3-amino-5-fluorobenzyl)sulfanyl]propyl}carbamate (800 mg, 2.42 mmol), XPhos Pd G1 (150 mg, 0.18 mmol), XPhos (86 mg, 0.18 mmol), and finely ground potassium phosphate (2.56 g, 12.1 mmol) in toluene (15.6 mL) and NMP (3.1 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 2.5 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc. The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was purified by column chromatography (hexanes to hexanes/EtOAc 2:1) to afford the desired product (1.21 g, 2.26 mmol, 93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 1.77 Hz, 1H), 7.65 (br d, <i>J</i> = 11.37 Hz, 1H), 7.52 (dd, <i>J</i> = 6.69, 8.46 Hz, 1H), 7.38 (br s, 1H), 7.17 (br s, 1H), 6.81–6.87 (m, 1H), 6.78 (dd, <i>J</i> = 2.27, 10.86 Hz, 1H), 6.71 (br d, <i>J</i> = 8.84 Hz, 1H), 4.60 (br s, 1H), 3.89 (s, 3H), 3.68 (s, 2H), 3.14–3.23 (m, 2H), 2.46 (t, <i>J</i> = 7.20 Hz, 2H), 1.75 (quin, <i>J</i> = 6.95 Hz, 2H), 1.45 (s, 9H).</div></div><div id="sec4_2_14_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> (<i>rac</i>)-<i>tert</i>-Butyl {3-[<i>S</i>-(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)-<i>N</i>-(trifluoroacetyl)sulfinimidoyl]propyl}carbamate</h5><div class="NLM_p last">Under an atmosphere of argon, a solution of 2,2,2-trifluoroacetamide (316 mg, 2.79 mmol) in THF (1 mL) was added dropwise to a solution of sodium <i>tert</i>-butoxide (179 mg, 1.86 mmol) in THF (1.5 mL), so that the temperature of the mixture remained below 10 °C. Subsequently, a freshly prepared solution of 1,3-dibromo-5,5-dimethylhydantoin (346 mg, 1.21 mmol) in THF (1.5 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. Then, the mixture was stirred for 10 min at 10 °C. Finally, a solution of <i>tert</i>-butyl {3-[(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfanyl]propyl}carbamate (997 mg, 1.86 mmol) in THF (1.5 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. The mixture was stirred for 1 h at 10 °C. The mixture was diluted with toluene (4.0 mL) under cooling, and a solution of sodium sulfite (235 mg, 1.86 mmol) in H<sub>2</sub>O (7.0 mL) was added so that the temperature of the mixture remained below 15 °C. Aq NaCl solution was added, and the mixture was extracted with EtOAc (3×). The combined organic layers were filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (DCM to DCM/EtOH 94:6) to afford the desired product (670 mg, 1.04 mmol, 55% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.36 (br s, 1H), 7.76 (br d, <i>J</i> = 11.49 Hz, 1H), 7.46–7.59 (m, 2H), 6.74–6.91 (m, 2H), 6.67 (br d, <i>J</i> = 7.54 Hz, 1H), 4.83 (br s, 1H), 4.45 (br d, <i>J</i> = 12.81 Hz, 1H), 4.22 (br d, <i>J</i> = 13.00 Hz, 1H), 3.89 (s, 3H), 3.14–3.30 (m, 2H), 2.89–3.03 (m, 2H), 1.91–2.04 (m, 2H), 1.44 (br s, 9H).</div></div><div id="sec4_2_14_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> (<i>rac</i>)-<i>tert</i>-Butyl {3-[<i>S</i>-(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfonimidoyl]propyl}carbamate</h5><div class="NLM_p last">To a solution of (<i>rac</i>)-<i>tert</i>-butyl {3-[<i>S</i>-(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)-<i>N</i>-(trifluoroacetyl)sulfinimidoyl]propyl}carbamate (561 mg, 0.869 mmol) in MeOH (17 mL), aq KOH solution (25%) was added until pH 10.5 was reached. Oxone (451 mg, 0.739 mmol) was added, and the mixture was stirred at a pH of 10–11 for 2 h. Then, the mixture was filtered, neutralized with HCl (pH 6.6), diluted with brine, and extracted with EtOAc. The organic layer was washed with aq sodium thiosulfate solution, filtered using a Whatman filter, and concentrated to afford the desired compound (433 mg, 0.77 mmol, 88% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 1.77 Hz, 1H), 7.81 (td, <i>J</i> = 2.02, 11.37 Hz, 1H), 7.48–7.54 (m, 2H), 6.81–6.87 (m, 1H), 6.76–6.81 (m, 2H), 4.85 (br s, 1H), 4.15–4.34 (m, 2H), 3.89 (s, 3H), 3.27 (q, <i>J</i> = 6.40 Hz, 2H), 3.00–3.13 (m, 2H), 2.10 (td, <i>J</i> = 7.04, 14.46 Hz, 2H), 1.45 (s, 9H).</div></div><div id="sec4_2_14_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> (<i>rac</i>)-<i>N</i>-(3-{[<i>S</i>-(3-Aminopropyl)sulfonimidoyl]methyl}-5-fluorophenyl)-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (<b>15</b>)</h5><div class="NLM_p last">To a solution of (<i>rac</i>)-<i>tert</i>-butyl {3-[<i>S</i>-(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)sulfonimidoyl]propyl}carbamate (27 mg, 0.048 mmol) in DCM (2 mL), trifluoroacetic acid (TFA, 36.7 μL, 0.447 mmol) was added. After 2 h, additional TFA (36.7 μL, 0.447 mmol) was added, and the mixture was stirred for 24 h. Then, the mixture was evaporated to dryness. The residue was purified by preparative HPLC (acidic conditions) to afford <b>15</b> (14.3 mg, 0.03 mmol, 64% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.17 (br s, 1H), 8.61 (s, 1H), 8.45 (br s, 1H), 7.81 (d, <i>J</i> = 12.1 Hz, 1H), 7.56 (t, <i>J</i> = 7.53 Hz, 1H), 7.46 (br s, 1H), 7.14 (d, <i>J</i> = 10.9 Hz, 1H), 6.97 (t, <i>J</i> = 7.53 Hz, 1H), 6.83 (d, <i>J</i> = 8.66 Hz, 1H), 4.26–4.36 (m, 2H), 3.85 (s, 3H), 3.03 (t, <i>J</i> = 7.15 Hz, 2H), 2.82 (br s, 2H), 1.96 (br s, 2H). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub>S, 466.1525; found, 466.1518.</div></div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Compound <b>16</b></h4><div id="sec4_2_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-fluoro-5-[(methylsulfanyl)methyl]phenyl}pyrimidin-2-amine</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (750 mg, 2.92 mmol), 3-fluoro-5-[(methylsulfanyl)methyl]aniline (385 mg, 2.25 mmol), XPhos Pd G1 (139 mg, 0.17 mmol), XPhos (80 mg, 0.17 mmol), and finely ground potassium phosphate (2.39 g, 11.2 mmol) in toluene (15 mL) and NMP (3 mL) was degassed and repressured with argon at rt. The mixture was heated in a sealed vial to 130 °C in a microwave reactor for 3 h. After cooling to rt, brine was added, and the mixture was extracted with EtOAc. The combined organic extracts were filtered through a Whatman filter and the filtrate was concentrated under reduced pressure. The crude product was combined with the crude product from another batch of the same size and purified by column chromatography (hexanes to hexanes/EtOAc 2:1) to afford the desired product (1.62 g, 4.14 mmol, 70% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (d, <i>J</i> = 2.02 Hz, 1H), 7.62 (dt, <i>J</i> = 11.37, 2.15 Hz, 1H), 7.49 (dd, <i>J</i> = 8.46, 6.69 Hz, 1H), 7.22–7.29 (m, 2H), 7.12 (s, 1H), 6.67–6.84 (m, 3H), 3.87 (s, 3H), 3.63 (s, 2H), 2.01 (s, 3H).</div></div><div id="sec4_2_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> (<i>rac</i>)-[(3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]ammonium 2,4,6-Trimethylbenzenesulfonate</h5><div class="NLM_p last">To a solution of ethyl (1<i>E</i>)-<i>N</i>-[(mesitylsulfonyl)oxy]ethanimidoate (1.82 g, 6.37 mmol, 1.0 equiv) in 1,4-dioxane (6.5 mL) was added dropwise at 0 °C perchloric acid (70%, 6.5 mL), and the mixture was stirred for 10 min at that temperature. Cold H<sub>2</sub>O (30 mL) was then added to this mixture, and the resulting emulsion was extracted with DCM (3×). The combined organic extracts were washed with brine and dried over Na<sub>2</sub>SO<sub>4</sub>. The resulting solution of <i>O</i>-(mesitylsulfonyl)hydroxylamine was then added dropwise at 0 °C to a solution of 5-fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-fluoro-5-[(methylsulfanyl)methyl]phenyl}pyrimidin-2-amine (2.50 g, 6.39 mmol) in DCM (6.5 mL). The mixture was allowed to warm to rt overnight and then concentrated under reduced pressure to 5 mL. The precipitate was filtered off, and the residue was rinsed with diethyl ether. The combined filtrate was concentrated under reduced pressure to afford the title compound (3.04 g, 5.00 mmol, 78% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 10.24 (s, 1H), 8.62 (d, <i>J</i> = 2.02 Hz, 1H), 7.85 (dt, <i>J</i> = 12.19, 2.12 Hz, 1H), 7.54 (t, <i>J</i> = 7.32 Hz, 1H), 7.48 (s, 1H), 7.14 (dd, <i>J</i> = 11.37, 2.27 Hz, 1H), 6.97 (td, <i>J</i> = 8.34, 2.27 Hz, 1H), 6.84 (d, <i>J</i> = 8.96 Hz, 1H), 6.73 (s, 2H), 5.96 (s, 2H, S═NH<sub>2</sub><sup>+</sup>), 4.58 (d, <i>J</i> = 12.88 Hz, 1H, SC<i>H</i><sub>A</sub>H<sub>B</sub>, AB system), 4.38 (d, <i>J</i> = 12.88 Hz, 1H, SCH<sub>A</sub><i>H</i><sub>B</sub>, AB system), 3.83 (s, 3H, OCH<sub>3</sub>), 3.03 (s, 3H, SCH<sub>3</sub>), 2.53 (s, 6H, mesitylsulfonate-2-CH<sub>3</sub>, -6-CH<sub>3</sub>), 2.16 (s, 3H, mesitylsulfonate-4-CH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 407 (free amine).</div></div><div id="sec4_2_15_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{3-fluoro-5-[(<i>S</i>-methylsulfonodiimidoyl)methyl]phenyl}pyrimidin-2-amine (<b>16</b>)</h5><div class="NLM_p last">To a solution of (<i>rac</i>)-[(3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}benzyl)(methyl)-λ<sup>4</sup>-sulfanylidene]ammonium 2,4,6-trimethylbenzenesulfonate (300 mg, 0.49 mmol) in anhyd DMF (1.3 mL) was added at 0 °C Na<sub>2</sub>CO<sub>3</sub> (62.9 mg, 0.59 mmol, 1.2 equiv) and NCS (79.3 mg, 0.59 mmol, 1.2 equiv). After stirring for 15 min at 0 °C, hexamethyldisilazane (0.31 mL, 1.48 mmol, 3.0 equiv) was added, and the mixture was warmed to rt. After stirring for a further 18 h, brine was added to the solution, and the mixture was extracted with EtOAc and DCM. The combined organic extracts were filtered through a Whatman filter and concentrated under reduced pressure. The crude product was purified by preparative HPLC [Agilent Prep 1200 system (2 × Prep Pump, DLA, MWD, Prep FC); column: Waters XBridge C18 5 μm, 100 × 30 mm; eluent A: H<sub>2</sub>O + 0.2% NH<sub>3</sub>, eluent B: MeCN; gradient: 0–10 min 25–55% B, 10–12 min 100% B; flow: 60 mL/min; rt; solution: 243 mg in 3.2 mL DMSO; injection: 2 × 1.6 mL; detection: UV 225 nm; <i>t</i><sub>R</sub> = 5.72–6.42 min] to afford <b>16</b> (40.0 mg, 0.09 mmol, 19% yield) as a colorless solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 27 °C): δ 10.04 (s, 1H, aniline–NH), 8.59 (d, <i>J</i> = 2.02 Hz, 1H), 7.73 (dt, <i>J</i> = 12.00, 2.21 Hz, 1H), 7.54 (dd, <i>J</i> = 8.34, 6.82 Hz, 1H), 7.46 (s, 1H), 7.12 (dd, <i>J</i> = 11.37, 2.27 Hz, 1H), 6.95 (td, <i>J</i> = 8.46, 2.53 Hz, 1H), 6.83 (br d, <i>J</i> = 9.60 Hz, 1H), 4.24 (s, 2H, SCH<sub>2</sub>), 3.83 (s, 3H, OCH<sub>3</sub>), 2.80 (s, 3H, SCH<sub>3</sub>). <sup>13</sup>C NMR (100.65 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 164.39 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 247.11 Hz, 1C<sub>q</sub>), 161.83 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 239.48 Hz, 1C<sub>q</sub>), 158.57 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1C<sub>q</sub>), 156.19 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 151.17 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 14.41 Hz, 1C<sub>q</sub>), 150.36 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 251.77 Hz, 1C<sub>q</sub>), 145.88 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.00 Hz, 1CH), 141.88 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 12.29 Hz, 1C<sub>q</sub>), 133.15 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1C<sub>q</sub>), 132.04 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 11.02 Hz, 1CH), 118.88 (m, 1C<sub>q</sub>), 116.97 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 1.70 Hz, 1CH), 110.41 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 107.33 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 104.47 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.70 Hz, 1CH), 100.22 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.85 Hz, 1CH), 64.32 (s, 1CH<sub>2</sub>), 56.35 (s, 1CH<sub>3</sub>), 44.99 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3207 (w), 3190 (w), 3108 (w), 3034 (w), 2971 (w), 2728 (w), 1716 (w), 1599 (m), 1543 (s), 1434 (s), 1317 (m), 1284 (s), 1194 (s), 1155 (s), 1031 (s), 957 (m), 837 (m), 785 (s). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 422. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>F<sub>3</sub>N<sub>5</sub>OS, 422.1263; found, 422.1263.</div></div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Compound <b>17</b></h4><div id="sec4_2_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidine (2.00 g, 7.79 mg) in 7 N methanolic ammonia (20 mL) was heated to 130 °C in a microwave reactor for 2 h. After cooling to rt, the mixture was concentrated under reduced pressure and the residue was purified by flash chromatography (silica gel, hexane to hexane/EtOAc 50:50 gradient) to afford the title compound (1.11 g, 4.68 mmol, 60% yield) as a beige solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.19 (d, <i>J</i> = 1.70 Hz, 1H), 7.43 (dd, <i>J</i> = 8.48, 6.78 Hz, 1H), 6.72–6.84 (m, 2H), 5.02 (br s, 2H, NH<sub>2</sub>), 3.86 (s, 3H, OCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 238.</div></div><div id="sec4_2_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> (3-Fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}phenyl)methanol</h5><div class="NLM_p last">To a mixture of 5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (130 mg, 0.55 mmol, 1.0 equiv), (3-bromo-5-fluorophenyl)methanol (112 mg, 0.55 mmol, 1.0 equiv), and 1,4-dioxane (1.3 mL) in a sealable tube was added at rt Cs<sub>2</sub>CO<sub>3</sub> (214 mg, 0.66 mmol, 1.2 equiv), and the tube was purged several times with argon. Then, Pd<sub>2</sub>(dba)<sub>3</sub> (25.1 mg, 0.027 mmol, 5 mol %) and Xantphos (31.7 mg, 0.055 mmol, 10 mol %) were added, and the tube was purged with argon again. After that, the tube was sealed and heated to 100 °C for 20 h and then cooled to rt upon which further Pd<sub>2</sub>(dba)<sub>3</sub> (25.1 mg, 0.027 mmol, 5 mol %) and Xantphos (31.7 mg, 0.055 mmol, 10 mol %) were added. After heating to 100 °C for a further 22 h, the mixture was cooled to rt and diluted with EtOAc, washed with brine, and filtered through a Whatman filter. The filtrate was concentrated under reduced pressure and purified by flash chromatography (silica gel, hexane to hexane/EtOAc 50:50) and preparative HPLC [Waters AutoPurification system (Pump 254, Sample Manager 2767, CFO, DAD 2996, ELSD 2424, SQD 3100); column: Waters XBridge C18 5 μm, 100 × 30 mm; eluent A: H<sub>2</sub>O + 0.2% NH<sub>3</sub>, eluent B: MeCN; gradient: 0–8 min 10–100% B; flow: 50 mL/min; rt; solution: 149 mg in 3.0 mL DMSO; injection: 3 × 1 mL; detection: DAD scan 210–400 nm; <i>t</i><sub>R</sub> = 5.84–6.00 min] to afford the title compound (26.0 mg, 0.07 mmol, 13% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 1.88 Hz, 1H), 7.63–7.72 (m, 1H), 7.52 (dd, <i>J</i> = 8.48, 6.78 Hz, 1H), 7.26 (br s, 1H), 7.20 (s, 1H), 6.74–6.88 (m, 3H), 4.71 (d, <i>J</i> = 6.03 Hz, 2H, C<i>H</i><sub>2</sub>OH), 3.90 (s, 3H, OCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 362.</div></div><div id="sec4_2_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> <i>N</i>-[3-(Aminomethyl)-5-fluorophenyl]-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-amine (<b>17</b>)</h5><div class="NLM_p last">To a solution of (3-fluoro-5-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyrimidin-2-yl]amino}phenyl)methanol (20 mg, 0.055 mmol) in DMF (0.6 mL) was added dropwise at 0 °C thionyl chloride (10 μL, 0.138 mmol, 2.5 equiv). The mixture was then allowed to warm to rt over 2 h, after which it was diluted with EtOAc (100 mL) and slowly basified in an ice bath by the addition of aq NaHCO<sub>3</sub>/NaCl solution until pH 8. The mixture was then extracted with EtOAc/THF, the combined extracts were filtered through a Whatman filter, and the filtrate was concentrated under reduced pressure. The crude benzyl chloride was dissolved in 7 N methanolic ammonia (7.3 mL) and heated in a sealed tube to 80 °C for 14 h. After cooling to rt, the mixture was concentrated under reduced pressure, and the crude product was purified by preparative HPLC (basic conditions) to afford <b>17</b> (9.8 mg, 0.027 mmol, 50% yield) as a colorless solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.32 (d, <i>J</i> = 1.59 Hz, 1H), 7.61 (br d, <i>J</i> = 11.13 Hz, 1H), 7.49 (t, <i>J</i> = 7.49 Hz, 1H), 7.19–7.24 (m, 1H), 7.12 (s, 1H), 6.81 (t, <i>J</i> = 8.05 Hz, 1H), 6.76 (d, <i>J</i> = 10.97 Hz, 1H), 6.69 (br d, <i>J</i> = 9.22 Hz, 1H), 3.87 (s, 3H, OCH<sub>3</sub>), 3.85 (s, 2H, C<i>H</i><sub>2</sub>NH<sub>2</sub>). ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O, 361.1277; found, 361.1276.</div></div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> Compound <b>24</b>, Enantiomers <b>25</b> and <b>26</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h4><div id="sec4_2_17_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 4-Chloro-6-(4-fluoro-2-methoxyphenyl)pyrimidine (<b>20</b>)</h5><div class="NLM_p last">Under an atmosphere of argon, a mixture of 4,6-dichloropyrimidine (<b>18</b>; 5.00 g, 33.56 mmol), (4-fluoro-2-methoxyphenyl)boronic acid (<b>19</b>; 6.27 g, 36.92 mmol), and [Pd(dppf)Cl<sub>2</sub>] (1:1 complex with DCM; 2.74 g, 3.36 mmol) in 2 M aq K<sub>2</sub>CO<sub>3</sub> (50 mL) and DME (101 mL) was stirred for 150 min at 90 °C. After cooling, the mixture was diluted with EtOAc and washed with dilute aq NaCl solution. The organic layer was filtered using a Whatman filter and concentrated. The residue was purified by chromatography (hexane/EtOAc 5 to 40%) to afford <b>20</b> (6.35 g, 26.61 mmol, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 9.00 (m, 1H), 8.13 (m, 1H), 8.02 (m, 1H), 6.82 (m, 1H), 6.75 (m, 1H), 3.94 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 239.</div></div><div id="sec4_2_17_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 6-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(methylsulfanyl)methyl]pyridin-2-yl}pyrimidin-4-amine (<b>22</b>)</h5><div class="NLM_p last">A mixture of 4-chloro-6-(4-fluoro-2-methoxyphenyl)pyrimidine (<b>20</b>; 475 mg, 1.99 mmol), 4-[(methylsulfanyl)methyl]pyridin-2-amine (<b>21</b>; 614 mg, 3.98 mmol; UkrOrgSynthesis Ltd.), Xantphos (51 mg, 0.09 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (972 mg, 2.99 mmol) in 1,4-dioxane (7.1 mL) was degassed using argon. Pd<sub>2</sub>(dba)<sub>3</sub> (27 mg, 0.03 mmol) was added under argon, and the mixture was stirred in a closed pressure tube for 8 h at 100 °C. After cooling, additional Xantphos (17 mg, 0.03 mmol) and Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol) were added under argon, and the mixture was stirred in a closed pressure tube for 14 h at 100 °C. After cooling, the mixture was diluted with DCM and filtered. The solids were washed with DCM, and the combined filtrates were concentrated. The residue was purified by chromatography (silica gel, DCM/EtOH 95:5) to afford <b>22</b> (424 mg, 1.19 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.86 (m, 1H), 8.26 (m, 1H), 8.04 (m, 1H), 7.94 (m, 1H), 7.82 (s, 1H), 7.76 (br, 1H), 6.96 (m, 1H), 6.77 (m, 2H), 3.92 (s, 3H), 3.67 (s, 2H), 2.06 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 357.</div></div><div id="sec4_2_17_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> (<i>rac</i>)-2,2,2-Trifluoro-<i>N</i>-{[(2-{[6-(4-fluoro-2-methoxyphenyl)pyrimidin-4-yl]amino}pyridin-4-yl)methyl](methyl)-λ<sup>4</sup>-sulfanylidene}acetamide (<b>23</b>)</h5><div class="NLM_p last">Under argon, a solution of 2,2,2-trifluoroacetamide (200 mg, 1.77 mmol) in THF (1.0 mL) was added dropwise to a solution of sodium <i>tert</i>-butoxide (113 mg, 1.18 mmol) in THF (0.8 mL), so that the temperature of the mixture remained below 10 °C. Subsequently, a freshly prepared solution of 1,3-dibromo-5,5-dimethylhydantoin (253 mg, 0.88 mmol) in THF (1.5 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. Then, the mixture was stirred for 10 min at 10 °C. Finally, a solution of 6-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(methylsulfanyl)methyl]pyridin-2-yl}pyrimidin-4-amine (<b>22</b>; 420 mg, 1.18 mmol) in 1,4-dioxane (10.0 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. The mixture was stirred for 75 min at 10 °C. The mixture was diluted with toluene (2.0 mL) under cooling, and a solution of sodium sulfite (149 mg, 1.18 mmol) in H<sub>2</sub>O (2.0 mL) was added so that the temperature of the mixture remained below 15 °C. Aq NaCl solution was added, and the mixture was extracted with EtOAc (3×). The combined organic layers were filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (DCM/EtOH 94:6) to afford <b>23</b> (348 mg, 0.74 mmol, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.90 (s, 1H, pyrimidine-2-H), 8.38 (d, <i>J</i> = 5.02 Hz, 1H), 8.26 (s, 1H, NH), 8.11 (dd, <i>J</i> = 8.78, 7.03 Hz, 1H), 7.72 (s, 1H), 7.64 (s, 1H), 6.95 (dd, <i>J</i> = 5.02, 1.25 Hz, 1H), 6.84 (t, <i>J</i> = 8.10 Hz, 1H), 6.78 (d, <i>J</i> = 10.84 Hz, 1H), 4.52 (d, <i>J</i> = 13.05 Hz, 1H, sulfilimine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.31 (d, <i>J</i> = 13.05 Hz, 1H, sulfilimine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.95 (s, 3H, OCH<sub>3</sub>), 2.74 (s, 3H, sulfilimine–CH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 468.</div></div><div id="sec4_2_17_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> (<i>rac</i>)-6-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyrimidin-4-amine (<b>24</b>)</h5><div class="NLM_p">KMnO<sub>4</sub> (230 mg, 1.46 mmol) was added to a stirred solution of (<i>rac</i>)-2,2,2-trifluoro-<i>N</i>-{[(2-{[6-(4-fluoro-2-methoxyphenyl)pyrimidin-4-yl]amino}pyridin-4-yl)methyl](methyl)-λ<sup>4</sup>-sulfanylidene}acetamide (<b>23</b>; 340 mg, 0.73 mmol) in acetone (3.4 mL) at rt. The mixture was stirred at 50 °C for 5 h, before additional KMnO<sub>4</sub> (115 mg, 0.73 mmol) was added and stirring was continued at 50 °C. After 90 min, additional KMnO<sub>4</sub> (115 mg, 0.73 mmol) was added and stirring was continued at 50 °C. After 90 min, a solution of KMnO<sub>4</sub> (230 mg, 1.46 mmol) in acetone (10 mL) was added and stirring was continued at 50 °C. After 7 h, a solution of KMnO<sub>4</sub> (115 mg, 0.73 mmol) in acetone (10 mL) was added and stirring was continued at 50 °C for 4 h. After cooling, the mixture was filtered, and the residue was washed with acetone. The combined filtrates were concentrated. MeOH (100 mL) was added, and the mixture was filtered. The filtrate was concentrated to afford the crude trifluoroacetamide (180 mg), which was used in the next step.</div><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (29 mg, 0.21 mmol) was added to a solution of crude 2,2,2-trifluoro-<i>N</i>-{[(2-{[6-(4-fluoro-2-methoxyphenyl)pyrimidin-4-yl]amino}pyridin-4-yl)methyl](methyl)oxido-λ<sup>6</sup>-sulfanylidene}acetamide (50 mg) in MeOH (9 mL) at rt. The mixture was stirred for 50 min before H<sub>2</sub>O (100 mL) was added. Then, the mixture was extracted with DCM/<i>i</i>-PrOH (4:1) and EtOAc. The combined organic layers were filtered using a Whatman filter and concentrated. The residue was purified by preparative HPLC (basic conditions) to afford <b>24</b> (11 mg, 0.03 mmol, 15% yield over two steps).</div></div><div id="sec4_2_17_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 6-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyrimidin-4-amine [Enantiomer 1 (<b>25</b>) and Enantiomer 2 (<b>26</b>)]</h5><div class="NLM_p">(<i>rac</i>)-6-(4-Fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyrimidin-4-amine (<b>24</b>; 33 mg) was separated into the enantiomers by chiral preparative HPLC [Sepiatec Prep SFC 100 system; column: CHIRALPAK IA 5 μm, 250 × 20 mm; eluent: CO<sub>2</sub>/MeOH (+0.4% diethylamine) 6:4; flow: 80 mL/min; pressure (outlet): 150 bar; temperature: 40 °C; solution: 33 mg in 2 mL DMF; detection: UV 254 nm].</div><div class="NLM_p">Enantiomer 1 (<b>25</b>): <i>t</i><sub>R</sub> = 2.90–3.50 min. Yield: 12 mg, 0.03 mmol.</div><div class="NLM_p last">Enantiomer 2 (<b>26</b>): <i>t</i><sub>R</sub> = 3.65–4.50 min. Yield: 13 mg, 0.03 mmol. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.27 (s, 1H, aniline–NH), 8.71 (d, <i>J</i> = 1.13 Hz, 1H, pyrimidine-2-H), 8.30–8.32 (m, 1H), 8.28 (d, <i>J</i> = 4.89 Hz, 1H), 8.00 (dd, <i>J</i> = 8.76, 7.25 Hz, 1H), 7.83 (s, 1H), 7.01–7.10 (m, 2H), 6.88 (td, <i>J</i> = 8.43, 2.54 Hz, 1H), 4.39 (s, 2H, sulfoximine–CH<sub>2</sub>), 3.89 (s, 3H, OCH<sub>3</sub>), 2.86 (s, 3H, sulfoximine–CH<sub>3</sub>). UPLC-MS (basic): <i>t</i><sub>R</sub> = 1.14 min; MS (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 388.2. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>FN<sub>5</sub>O<sub>2</sub>S, 388.1244; found, 388.1241.</div></div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Compound <b>31</b>, Enantiomers <b>32</b> and VIP152 (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h4><div id="sec4_2_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 2-Chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyridine (<b>28</b>)</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-iodopyridine (<b>27</b>; 1.00 g, 3.88 mmol, APAC Pharmaceutical, LLC), (4-fluoro-2-methoxyphenyl)boronic acid (<b>19</b>; 660 mg, 3.88 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (449 mg, 0.38 mmol) in DME (10.0 mL) and 2 M aq K<sub>2</sub>CO<sub>3</sub> (5.8 mL) was degassed using argon. The mixture was stirred under an argon atmosphere for 4 h at 100 °C. After cooling, the mixture was diluted with EtOAc and THF and washed with brine. The organic phase was filtered using a Whatman filter and concentrated. The residue was purified by column chromatography (hexane to hexane/EtOAc 50%) to afford <b>28</b> (947 mg, 3.70 mmol, 95% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.27 (m, 1H), 7.33 (m, 1H), 7.24 (m, 1H), 6.75 (m, 2H), 3.83 (s, 3H). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 256.</div></div><div id="sec4_2_18_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> 5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(methylsulfanyl)methyl]pyridin-2-yl}pyridin-2-amine (<b>29</b>)</h5><div class="NLM_p last">A mixture of 2-chloro-5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyridine (<b>28</b>; 400 mg, 1.57 mmol), 4-[(methylsulfanyl)methyl]pyridin-2-amine (<b>21</b>; 483 mg, 3.13 mmol), Xantphos (40 mg, 0.07 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (765 mg, 2.35 mmol) in 1,4-dioxane (6.0 mL) was degassed using argon. Pd<sub>2</sub>(dba)<sub>3</sub> (21 mg, 0.02 mmol) was added under argon, and the mixture was stirred in a closed pressure tube for 5 h at 100 °C. After cooling, the mixture was diluted with aq NaCl solution and extracted with DCM (3×). The combined organic phases were filtered using a Whatman filter and concentrated. The residue was purified by chromatography (hexane to hexane/EtOAc 30%) to afford <b>29</b> (556 mg, 1.48 mmol, 94% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.12–8.19 (m, 2H), 7.61 (d, <i>J</i> = 5.09 Hz, 1H), 7.36–7.46 (m, 2H), 7.24–7.31 (m, 1H), 6.71–6.84 (m, 3H), 3.82 (s, 3H, OCH<sub>3</sub>), 3.61 (s, 2H, SCH<sub>2</sub>), 2.03 (s, 3H, SCH<sub>3</sub>). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 374.</div></div><div id="sec4_2_18_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> (<i>rac</i>)-2,2,2-Trifluoro-<i>N</i>-{[(2-{[5-fluoro-4-(4-fluoro-2-methoxyphenyl)pyridin-2-yl]amino}pyridin-4-yl)methyl](methyl)-λ<sup>4</sup>-sulfanylidene}acetamide (<b>30</b>)</h5><div class="NLM_p last">Under argon, a solution of 2,2,2-trifluoroacetamide (195 mg, 1.73 mmol) in 1,4-dioxane (0.5 mL) was added dropwise to a solution of sodium <i>tert</i>-butoxide (111 mg, 1.15 mmol) in 1,4-dioxane (0.6 mL), so that the temperature of the mixture remained below 10 °C. Subsequently, a freshly prepared solution of 1,3-dibromo-5,5-dimethylhydantoin (247 mg, 0.86 mmol) in 1,4-dioxane (0.6 mL)/THF (1.0 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. Then, the mixture was stirred for 10 min at 10 °C. Finally, a solution of 5-fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(methylsulfanyl)methyl]pyridin-2-yl}pyridin-2-amine (<b>29</b>; 430 mg, 1.15 mmol) in 1,4-dioxane (1.0 mL) was added dropwise to the stirred mixture, so that the temperature of the mixture remained below 10 °C. The mixture was stirred for 60 min at 10 °C. The mixture was diluted with toluene (2.0 mL) under cooling and a solution of sodium sulfite (145 mg, 1.15 mmol) in H<sub>2</sub>O (2.0 mL) was added so that the temperature of the mixture remained below 15 °C. Aq NaCl solution was added, and the mixture was extracted with EtOAc (3×). The combined organic phases were filtered using a Whatman filter and concentrated to afford crude <b>30</b>, which was used without further purification. UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 485.</div></div><div id="sec4_2_18_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> (<i>rac</i>)-5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyridin-2-amine (<b>31</b>)</h5><div class="NLM_p last">Acetone (6.0 mL) and KMnO<sub>4</sub> (814 mg, 5.15 mmol) were added to the residue of the previous step (crude <b>30</b>), and the mixture was stirred at 50 °C for 90 min. Additional KMnO<sub>4</sub> (223 mg, 1.42 mmol) was added and stirring was continued at 50 °C for 4 h. Finally, additional KMnO<sub>4</sub> (305 mg, 1.93 mmol) was added and stirring was continued at 50 °C for 150 min. After cooling, the mixture was filtered, the residue was washed with acetone, and the combined filtrates were concentrated. The residue was dissolved in MeOH (60 mL), K<sub>2</sub>CO<sub>3</sub> (182 mg, 1.32 mmol) was added, and the reaction mixture was stirred for 20 min at rt. The mixture was diluted with aq NaCl solution and extracted with DCM (3×). The combined organic phases were filtered using a Whatman filter and concentrated. The residue was purified by preparative HPLC (basic conditions) to afford <b>31</b> (50 mg, 0.12 mmol, 10% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 27 °C): δ 8.22 (d, <i>J</i> = 5.31 Hz, 1H), 8.13 (d, <i>J</i> = 1.52 Hz, 1H), 7.80 (s, 1H, aniline–NH), 7.46 (d, <i>J</i> = 5.05 Hz, 1H), 7.25–7.30 (m, 2H), 6.92 (d, <i>J</i> = 5.05 Hz, 1H), 6.71–6.82 (m, 2H), 4.36 (d, <i>J</i> = 12.63 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>A</sub>, AB system), 4.25 (d, <i>J</i> = 12.88 Hz, 1H, sulfoximine–CH<sub>2</sub>, H<sub>B</sub>, AB system), 3.83 (s, 3H, OCH<sub>3</sub>), 3.00 (s, 3H, sulfoximine–CH<sub>3</sub>). <sup>13</sup>C NMR (100.67 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.69 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 245.41 Hz, 1C<sub>q</sub>), 157.92 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1C<sub>q</sub>), 154.4 (s, 1C<sub>q</sub>), 151.88 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 254.84 Hz, 1C<sub>q</sub>), 150.82 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.12 Hz, 1C<sub>q</sub>), 147.31 (s, 1CH), 140.63 (s, 1C<sub>q</sub>), 134.74 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 15.26 Hz, 1C<sub>q</sub>), 134.20 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 26.70 Hz, 1CH), 131.59 Hz (d, <sup>3</sup><i>J</i><sub>CF</sub> = 10.17 Hz, 1CH), 118.82 (d, <sup>4</sup><i>J</i><sub>CF</sub> = 2.97 Hz, 1C<sub>q</sub>), 118.04 (s, 1CH), 113.42 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 43.23 Hz, 1CH), 113.18 (s, 1CH), 107.15 (d, <sup>3</sup><i>J</i><sub>CF</sub> = 22.04 Hz, 1CH), 100.16 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 25.86 Hz, 1CH), 67.18 (s, 1CH<sub>2</sub>), 56.25 (s, 1CH<sub>3</sub>), 41.45 (s, 1CH<sub>3</sub>). IR (diamond ATR, cm<sup>–1</sup>): 3371 (w), 3337 (m), 3289 (br s), 3020 (m), 3005 (m), 2918 (m), 2905 (m), 1622 (m), 1607 (s), 1535 (m), 1481 (m), 1416 (m), 1398 (s), 1283 (m), 1248 (m), 1196 (s), 1030 (s), 953 (s), 829 (s). UPLC (ESI+) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> 405.</div></div><div id="sec4_2_18_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> (<i>R</i>)-5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyridin-2-amine (<b>32</b>) and (<i>S</i>)-5-Fluoro-4-(4-fluoro-2-methoxyphenyl)-<i>N</i>-{4-[(<i>S</i>-methylsulfonimidoyl)methyl]pyridin-2-yl}pyridin-2-amine (VIP152)</h5><div class="NLM_p">Separation of the two enantiomers of (<i>rac</i>)-<b>31</b> (3.468 g) via chiral preparative HPLC yielded <b>32</b> and VIP152 [Sepiatec Prep SFC 100 system; column: CHIRALPAK IC 5 μm, 250 × 30 mm; eluent: CO<sub>2</sub>/<i>i</i>-PrOH (+0.4% diethylamine) 7:3; flow: 100 mL/min; pressure (outlet): 150 bar; temperature: 40 °C; solution: 3.468 g in 55 mL DCM/MeOH 2:1; detection: UV 254 nm].</div><div class="NLM_p">(<i>R</i>)-Enantiomer (<b>32</b>): <i>t</i><sub>R</sub> = 7.0–8.1 min. Yield: 1.31 g, 3.24 mmol. [α]<sub>D</sub> −10.5 ± 0.21 (<i>c</i> 1.00, DMSO).</div><div class="NLM_p last">(<i>S</i>)-Enantiomer (VIP152): <i>t</i><sub>R</sub> = 8.5–10.5 min. Yield: 1.32 mg, 3.26 mmol. ESI-HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub>S, 405.1198; found, 405.1196. [α]<sub>D</sub> +11.4 ± 0.13 (<i>c</i> 1.00, DMSO).</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Pharmacology</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Kinase Assays</h4><div class="NLM_p">The inhibitory activities of the compounds against CDK2/CycE and CDK9/CycT1 were measured in TR-FRET-based kinase activity inhibition assays using the biotinylated peptide biotin-Ttds-YISPLKSPYKISEG (C-terminus in amide form, JPT Peptide Technologies) as the substrate. Recombinant fusion proteins of GST and human CDK2 and of GST and human CycE, expressed in insect cells (Sf9) and purified by glutathione Sepharose affinity chromatography, were purchased from ProQinase GmbH (product no. 0050-0055-1). Recombinant full-length His-tagged human CDK9 and CycT1, expressed in insect cells and purified by Ni–NTA affinity chromatography, were purchased from Life Technologies (cat. no. PV4131).</div><div class="NLM_p">For the assays, 50 nL of a 100-fold concentrated solution of the test compound in DMSO was pipetted into a black, low-volume, 1536-well microtiter plate (Greiner Bio-One), 2 μL of enzyme solution in aqueous assay buffer [50 mM Tris HCl pH 8.0, 10 mM MgCl<sub>2</sub>, 1.0 mM dithiothreitol, 0.1 mM sodium orthovanadate, 0.01% (v/v) Nonidet P-40 (Sigma)] was added, and the mixture was incubated for 15 min at 22 °C to allow prebinding of the test compound to the enzyme before the start of the kinase reaction. Then, the kinase reaction was started by the addition of 3 μL of a solution of ATP [final concn: 10 μM (=low) or 2 mM (=high)] and substrate (final concn: 0.75 μM) in assay buffer, and the resulting mixture was incubated for a reaction time of 25 min at 22 °C. The concentrations of the enzymes were adjusted depending on the activity of the enzyme, lot, and ATP concentration and were chosen appropriate to have the assays in the linear range. Typical concentrations were in the range of 10–100 ng/mL for CDK2 and 0.5–1 μg/mL for CDK9. The reaction was stopped by the addition of 5 μL of a solution of TR-FRET detection reagents [0.2 μM streptavidin-XL665 (Cisbio Bioassays), 1 nM anti-RB (pSer807/pSer811)-antibody (BD Pharmingen, # 558389), and 1.2 nM LANCE Eu-W1024 labeled anti-mouse IgG antibody (PerkinElmer, product no. AD0077)] in an aqueous ethylenediaminetetraacetic acid (EDTA) solution [100 mM EDTA, 0.2% (w/v) bovine serum albumin in 100 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES), pH 7.5].</div><div class="NLM_p">The resulting mixture was incubated 1 h at 22 °C to allow the formation of complex between the phosphorylated biotinylated peptide and the detection reagents. Subsequently, the amount of phosphorylated substrate was evaluated by measurement of the resonance energy transfer from the europium chelate to the streptavidin-XL. Therefore, the fluorescence emissions at 620 and 665 nm after excitation at 350 nm were measured with a PHERAstar FS reader (BMG Labtech). The ratio of the emissions at 665 and 620 nm was taken as the measure for the amount of phosphorylated substrate.</div><div class="NLM_p last">The compounds were tested on the same microtiter plate at 11 different concentrations in the range of 20 μM to 0.07 nM (dilution series prepared separately, before the assay, on the level of the 100-fold concentrated solutions in DMSO by serial dilutions) in duplicate values for each concentration. IC<sub>50</sub> values were calculated using Genedata Screener software.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> Merck Millipore CDK Assays</h4><div class="NLM_p last">Assays were performed according to the Merck Millipore KinaseProfiler standard protocols with an ATP concentration of 10 μM.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Proliferation Assays</h4><div class="NLM_p last">A2780 human ovarian tumor cells (# 93112519) were obtained from the European Collection of Authenticated Cell Cultures (ECACC) of Public Health England (Salisbury, UK) and MOLM-13 human AML cells (ACC 554) were obtained from the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSMZ, German Collection of Microorganisms and Cell Cultures, Braunschweig). Authentication of cell lines was conducted at the DSMZ via polymerase chain reaction (PCR)-based DNA profiling of polymorphic short tandem repeats. Cells were propagated under the suggested growth conditions in a humidified 37 °C incubator. Proliferation assays were conducted in 96-well plates at densities of 2500 (A2780) and 5000 (MOLM-13) cells per well in the growth medium containing 10% fetal calf serum (FCS). Cells were treated in quadruplicate with serial dilutions of test compounds for 96 h. Relative cell numbers were quantified by crystal violet staining (A2780)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> or CellTiter-Glo luminescent cell viability assay (Promega) (MOLM-13). IC<sub>50</sub> values (inhibitory concentration at 50% of maximal effect) were determined by means of a four-parameter fit on measurement data, which were normalized to vehicle (DMSO)-treated cells (=100%) and measurement readings taken immediately before compound exposure (=0%).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> PK Studies</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Caco-2 Permeability Assay</h4><div class="NLM_p">Caco-2 cells were seeded at a density of 2.5 × 10<sup>5</sup> cells/well on 24-well insert plates, 0.4 μm pore size, 0.3 cm<sup>2</sup> (CoStar), and grown for 13–15 days in DMEM supplemented with 10% FCS, 1% GlutaMAX (100×, Gibco), 100 U/mL penicillin, 100 μg/mL streptomycin (Gibco), and 1% nonessential amino acids (100×). Cells were maintained at 37 °C in a humidified 5% CO<sub>2</sub> atmosphere. The medium was changed every 2–3 days.</div><div class="NLM_p last">The bidirectional transport assay for the evaluation of Caco-2 permeability was carried out in 24-well insert plates using a robotic system (Tecan). Before the assay was run, the culture medium was replaced by the transport medium (FCS-free HEPES carbonate transport buffer pH 7.2). For the assessment of monolayer integrity, the transepithelial electrical resistance (TEER) was measured. Only monolayers with a TEER of at least 400 Ω·cm<sup>2</sup> were used. Test compounds were predissolved in DMSO and added either to the apical or basolateral compartment at a final concentration of 2 μM. Evaluation was done in triplicate. Before and after incubation for 2 h at 37 °C, the samples were taken from both compartments and analyzed, after precipitation with MeOH, by LC–MS/MS. The apparent permeability coefficient (<i>P</i><sub>app</sub>) was calculated both for the apical to basolateral (A → B) and the basolateral to apical (B → A) direction using the following equation: <i>P</i><sub>app</sub> = (<i>V</i><sub>r</sub>/<i>P</i><sub>0</sub>)(1/<i>S</i>)(<i>P</i><sub>2</sub>/<i>t</i>), where <i>V</i><sub>r</sub> is the volume of medium in the receiver chamber, <i>P</i><sub>0</sub> is the measured peak area of the test compound in the donor chamber at <i>t</i> = 0, <i>S</i> is the surface area of the monolayer, <i>P</i><sub>2</sub> is the measured peak area of the test compound in the acceptor chamber after incubation for 2 h, and <i>t</i> is the incubation time. The efflux ratio basolateral (B) to apical (A) was calculated by dividing <i>P</i><sub>app</sub> B–A by <i>P</i><sub>app</sub> A–B.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> In Vitro Metabolic Stability in Liver Microsomes</h4><div class="NLM_p last">The in vitro metabolic stability of test compounds was determined by incubation at 1 μM in a suspension of liver microsomes in 100 mM phosphate buffer pH 7.4 (NaH<sub>2</sub>PO<sub>4</sub>·H<sub>2</sub>O + Na<sub>2</sub>HPO<sub>4</sub>·2H<sub>2</sub>O) and at a protein concentration of 0.5 mg/mL at 37 °C. The microsomes were activated by adding a cofactor mix containing 8 mM glucose-6-phosphate, 4 mM MgCl<sub>2</sub>, 0.5 mM nicotinamide adenine dinucleotide phosphate, and 1 IU/mL glucose-6-phosphate dehydrogenase in phosphate buffer pH 7.4. The metabolic assay was started shortly afterward by adding the test compound to the incubation at a final volume of 1 mL. The organic solvent in the incubations was limited to ≤0.01% DMSO and ≤1% MeCN. During incubation, the microsomal suspensions were continuously shaken at 580 rpm and aliquots were taken at 2, 8, 16, 30, 45, and 60 min, to which an equal volume of cold MeOH was immediately added. The samples were frozen at −20 °C overnight and after thawing subsequently centrifuged for 15 min at 3000 rpm. The supernatant was analyzed with an Agilent 1200 HPLC system with LC–MS/MS detection. The half-life of a test compound was determined from the concentration–time plot. From the half-life, the intrinsic clearances and the hepatic in vivo blood clearance (CL) and maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the “well-stirred” liver model<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> together with the additional parameter liver blood flow, specific liver weight, and microsomal protein content. The following parameter values were used for rats: liver blood flow: 4.2 L/h/kg; specific liver weight: 32 g/kg body weight; and microsomal protein content: 40 mg/g.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> In Vitro Metabolic Stability in Hepatocytes</h4><div class="NLM_p last">Hepatocytes from Han/Wistar rats were isolated via a two-step perfusion method. After perfusion, the liver was carefully removed from the rat: the liver capsule was opened and the hepatocytes were gently shaken out into a Petri dish with ice-cold Williams’ medium E (WME). The resulting cell suspension was filtered through sterile gauze into 50 mL Falcon tubes and centrifuged at 50<i>g</i> for 3 min at rt. The cell pellet was resuspended in WME (30 mL) and centrifuged twice through a Percoll gradient at 100<i>g</i>. The hepatocytes were washed again with WME and resuspended in the medium containing 5% FCS. Cell viability was determined by trypan blue exclusion. For the metabolic stability assay, liver cells were distributed in WME containing 5% FCS to glass vials at a density of 1.0 × 10<sup>6</sup> vital cells/mL. The test compound was added to a final concentration of 1 μM. During incubation at 37 °C, the hepatocyte suspensions were continuously shaken at 580 rpm, and aliquots were taken at 2, 8, 16, 30, 45, and 90 min, to which an equal volume of cold MeCN was immediately added. The samples were frozen at −20 °C overnight and after thawing subsequently centrifuged for 15 min at 3000 rpm. The supernatant was analyzed with an Agilent 1200 HPLC system with LC–MS/MS detection. The half-life of a test compound was determined from the concentration–time plot. From the half-life, the intrinsic clearances and the hepatic in vivo blood clearance (CL) and maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the “well-stirred” liver model<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> together with the additional parameters liver blood flow, specific liver weight, and amount of liver cells in vivo and in vitro. The same parameters for liver blood flow and specific liver weight as given above were used; liver cells in vivo: 1.1 × 10<sup>8</sup> cells/g liver, liver cells in vitro: 1.0 × 10<sup>6</sup>/mL.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> Plasma Protein Binding</h4><div class="NLM_p last">HT equilibrium dialysis was used to determine the plasma protein binding, as outlined by Banker et al.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> In brief, a semipermeable membrane separated the plasma from the buffer compartment. The test compound was added to the plasma side at a concentration of 3 μM and incubated for 7 h at 37 °C and 5% CO<sub>2</sub> with 99% humidity and moderate shaking. Then, 10 μL of the plasma side was transferred to a deep-well plate containing 90 μL of buffer and 90 μL of the buffer side was added to 10 μL of blank plasma. All samples were precipitated with 400 μL ice-cold MeOH and frozen overnight at −20 °C. After thawing and mixing, the samples were centrifuged for 10 min at 3000 rpm. The supernatant was transferred to a 96-well plate, and LC–MS/MS measurements were undertaken. From the quotient of buffer and plasma concentration, the unbound fraction (<i>f</i><sub>u</sub>) was calculated.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> CYP Inhibition Assay</h4><div class="NLM_p last">The inhibitory potency of test compounds toward cytochrome P450-dependent metabolic pathways was determined in human liver microsomes (pool) by applying individual CYP isoform selective standard probes (CYP1A2, phenacetin; CYP2A6, coumarin; CYP2B6, bupropion; CYP2C8, amodiaquine; CYP2C9, diclofenac; CYP2C19, (<i>S</i>)-mephenytoin; CYP2D6, dextromethorphan; CYP2E1, chlorzoxazone; and CYP3A4, midazolam, testosterone). Reference inhibitors were included as positive controls. Incubation conditions (protein and substrate concentration and incubation time) were optimized with regard to linearity of metabolite formation. Assays were processed in 96-well plates at 37 °C using a Freedom EVO Workstation (Tecan, Crailsheim, Germany). The reactions were stopped by the addition of MeCN (100 μL) containing the respective stable labeled internal standard. Precipitated proteins were removed by centrifugation of the well plate (1800 rcf, 10 min, 10 °C). Metabolite formation was quantified by LC–MS/MS analysis (QTRAP 6500 system, Sciex, Canada), followed by inhibition evaluation and IC<sub>50</sub> calculation.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> CYP Induction Assay</h4><div class="NLM_p last">To evaluate the CYP induction potential in vitro, cultured human hepatocytes from three separate livers were treated once daily for three consecutive days with vehicle control, one of eight concentrations of test compound, or known human CYP inducers (e.g., omeprazole, phenobarbital, and rifampin). After treatment, the cells were incubated in situ with the appropriate marker substrates for the analysis of CYP3A4, CYP2B6, and CYP1A2 activity by LC–MS/MS. Following the in situ incubation, the same hepatocytes from the same treatment groups were harvested for RNA isolation and analyzed by qRT-PCR to assess the effect of test compound on CYP1A2, CYP2B6, and CYP3A4 mRNA expression levels.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> In Vivo PK Studies</h4><div class="NLM_p last">All animal experiments were conducted in accordance with the German Animal Welfare Law and were approved by local authorities. For in vivo PK experiments, test compounds were administered to male Wistar rats intravenously at doses of 0.3–0.5 mg/kg and intragastrically at doses of 0.6–1 mg/kg formulated as solutions using solubilizers such as poly(ethylene glycol) 400 (PEG 400) in well-tolerated amounts. For PK after iv administration, the test compounds were given as iv bolus and blood samples were taken at 2 min, 8 min, 15 min, 30 min, 45 min, 1 h, 2 h, 4 h, 6 h, 8 h, and 24 h after dosing. For PK after intragastral (ig) administration, the test compounds were given ig to fasted rats and blood samples were taken at 5 min, 15 min, 30 min, 45 min, 1 h, 2 h, 4 h, 6 h, 8 h, and 24 h after dosing. Blood was collected into lithium heparin tubes (Monovette, Sarstedt) and centrifuged for 15 min at 3000 rpm. An aliquot of 100 μL from the supernatant (plasma) was taken and precipitated by the addition of cold MeCN (400 μL). The samples were frozen at −20 °C overnight and subsequently thawed and centrifuged at 3000 rpm, 4 °C for 20 min. Aliquots of the supernatants were analyzed using an Agilent 1200 HPLC system with LC–MS/MS detection. PK parameters were based on the plasma concentration and time data and calculated (e.g., using the linear-log trapezoidal rule for AUC estimation) using an Excel-based program. The PK parameters derived from the concentration–time profiles after iv administration are as follows: CL<sub>plasma</sub>, total plasma clearance of test compound (in L/h/kg); CL<sub>blood</sub>, total blood clearance of test compound, and CL<sub>plasma</sub> × <i>C</i><sub>p</sub>/<i>C</i><sub>b</sub> (in L/h/kg) with <i>C</i><sub>p</sub>/<i>C</i><sub>b</sub> being the ratio of concentrations in plasma and blood. The PK parameters calculated from the concentration–time profiles after ig administration are as follows: <i>C</i><sub>max</sub>, maximal plasma concentration (in mg/L); <i>C</i><sub>max,norm</sub>, <i>C</i><sub>max</sub> divided by the administered dose (in kg/L); and <i>T</i><sub>max</sub>, time point at which <i>C</i><sub>max</sub> was observed (in h). The parameters calculated from both iv and ig concentration–time profiles are as follows: AUC<sub>norm</sub>, area under the concentration–time curve from <i>t</i> = 0 to infinity (extrapolated) divided by the administered dose (in kg·h/L); AUC<sub>(0–<i>t</i> last)norm</sub>, area under the concentration–time curve from <i>t</i> = 0 to the last time point for which plasma concentrations could be measured divided by the administered dose (in kg·h/L); <i>t</i><sub>1/2</sub>, apparent half-life (in h); and <i>F</i>, oral bioavailability, AUC<sub>norm</sub> after ig administration divided by AUC<sub>norm</sub> after iv administration (in %).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i126" class="anchor-spacer"></div><h3 class="article-section__title" id="_i126"> Physicochemical Assays</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> Aqueous Solubility of Compound–DMSO Solutions</h4><div class="NLM_p last">Aqueous solubility at pH 6.5 was determined by an orientating HTS method.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Test compounds were applied as 1 mM DMSO solutions. After addition of buffer pH 6.5, the solutions were shaken for 24 h at rt. The undissolved material was removed by filtration. The compound dissolved in the filtrate was quantified by HPLC–MS/MS.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> Equilibrium Shake Flask Solubility Assay</h4><div class="NLM_p last">The thermodynamic solubility of compounds in water was determined by an equilibrium shake flask method.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> A saturated solution of the test compound was prepared, and the solution was mixed for 24 h to ensure that equilibrium was reached. The solution was centrifuged to remove the insoluble fraction, and the concentration of the compound in solution was determined using a standard calibration curve. To prepare the test sample, solid compound (2 mg) was weighed in a 4 mL glass vial. Phosphate buffer pH 6.5 (1 mL) was added, and the suspension was stirred for 24 h at rt. Then, the solution was centrifuged. To prepare the sample for the standard calibration, solid sample (2 mg) was dissolved in MeCN (30 mL). After sonication, the solution was diluted with H<sub>2</sub>O to 50 mL. Sample and standard were quantified by HPLC with UV detection. For each test sample, two injection volumes (5 and 50 μL) in triplicate were made. Three injection volumes (5, 10, and 20 μL) were made for the standard. HPLC conditions: column: XTerra MS C18 2.5 μm, 4.6 × 30 mm; injection volume: sample: 3 × 5 μL and 3 × 50 μL, standard: 5, 10, and 20 μL; flow: 1.5 mL/min; acidic gradient: eluent A: H<sub>2</sub>O + 0.01% TFA, eluent B: MeCN + 0.01% TFA; 0 min 5% B, 0–3 min 65% B linear gradient, 3–5 min 65% B isocratic, 5–6 min 5% B isocratic; and UV detection: wavelength near the absorption maximum (between 200 and 400 nm). The areas of sample and standard injections, as well as the calculation of the solubility values (in mg/L), were determined using Waters Empower 2 FR software.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> log <i>D</i> Measurement</h4><div class="NLM_p last">log <i>D</i> values at pH 7.5 were recorded using an indirect method for determining hydrophobicity constants by reversed-phase HPLC.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> A homologous series of <i>n</i>-alkan-2-ones (C<sub>3</sub>–C<sub>16</sub>, 0.02 M in MeCN) was used for calibration. Test compounds were applied as 0.67 mM DMSO stock solutions in MeCN/H<sub>2</sub>O (1:1). The lipophilicity of compounds was then assessed by comparison to the calibration curve.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> In Vivo Pharmacology Studies</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> MOLM-13 Human AML Models in Mice and Rats</h4><div class="NLM_p">All animal experiments were conducted in accordance with the German Animal Welfare Law and were approved by local authorities. MOLM-13 human AML cells were obtained from the DSMZ. Cell lines were authenticated at the DSMZ via short-tandem repeat DNA fingerprinting before use in experiments to rule out potential cross-contamination among cell lines. All cell lines were tested as being free from mycoplasma contamination using MycoAlert (Lonza) at least every four passages and before a cell line substock was generated.</div><div class="NLM_p">For the AML mouse model, 2 × 10<sup>6</sup> MOLM-13 human AML cells were suspended in 100% Matrigel basement membrane matrix (BD Biosciences) and inoculated subcutaneously to female NMRI <i>nu</i>/<i>nu</i> mice (18–21 g, 5–6 weeks, Taconic M&B). For the AML models in rats, 2 × 10<sup>6</sup> MOLM-13 or MV4-11 human AML cells were suspended in 100% Matrigel and inoculated subcutaneously to female athymic nude rats (120–160 g, 5–6 weeks, Harlan). Animals were stratified into treatment and control groups (<i>n</i> = 12/group) based on primary tumor size. For the MOLM-13 models, treatments were started 5–6 days after tumor cell inoculation when the average tumor sizes were 39–41 and 73 mm<sup>3</sup> for mice and rats, respectively. For the MV4-11 models, treatments were started on day 13 when the tumor area was approximately 40 mm<sup>2</sup>.</div><div class="NLM_p">BAY-332 was administered once daily (QD, 15 mg/kg), once every 3 days (Q3D, 35 mg/kg), or once every 5 days (Q5D, 65 mg/kg). VIP152 was administered once weekly (QW, 12.5 mg/kg) for the MOLM-13 model in mice and Q3D (3 mg/kg), Q5D (3.75 mg/kg), or QW (4.5 mg/kg) for the MOLM-13 model in rats. PEG 400/EtOH/H<sub>2</sub>O (30:10:60) or PEG 400/H<sub>2</sub>O (80:20) was used as the vehicle control. Unless otherwise indicated, all treatments were administered intravenously and were continued until the end of the experiment. Along with body weights, tumors were measured at least twice weekly by caliper, and the volume was calculated using the formula (length × width<sup>2</sup>)/2. Treatment-to-control (T/C) ratios were calculated by dividing the mean tumor volume of the treatment group by the mean tumor volume of the vehicle group at the time point when the vehicle group was sacrificed.</div><div class="NLM_p">All statistical analyses were performed using the statistical programming language R.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Validity of the model assumptions was checked for each fitted statistical model.</div><div class="NLM_p last">Analyses for the MOLM-13 model in mice were performed by fitting a linear mixed effects model on the log-transformed outcome. The model included both a fixed and random intercepts and slopes for each group and subject. The data for the MOLM-13 model in rats were analyzed by group comparisons at the start of the study and on the last day of the vehicle group. Comparisons were performed based on a linear model estimated using weighted least squares that included a separate variance term for each group. Group comparisons were corrected for false positive rate using Dunnett’s method. In all cases, <i>p</i> < 0.05 was considered as being statistically significant.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i132" class="anchor-spacer"></div><h3 class="article-section__title" id="_i132"> Docking</h3><div class="NLM_p last">Docking calculations were performed using Glide<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> in the standard precision mode and default settings. The X-ray complex of CDK9/CycT1 with a benzimidazole inhibitor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1">3MY1</a>) was used.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Prior to the docking procedure, all water molecules and ligands in the protein–ligand complex were removed. Ligands were pre-processed using LigPrep.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Docking poses were selected based on Glide scoring.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133"> X-ray Structure Determination of VIP152</h3><div class="NLM_p last">Single crystals of VIP152 were obtained by slow evaporation of an acetone/H<sub>2</sub>O solution of the compound at rt. A single crystal with dimensions 0.14 × 0.08 × 0.02 mm<sup>3</sup> was mounted on a CryoLoop using inert oil. Data collection was carried out using a Bruker diffractometer equipped with an APEX II CCD area detector, an IμS microsource with Cu Kα radiation, mirrors as a monochromator, and a Kryoflex low-temperature device (<i>T</i> = 110 K). The program APEX II v.2011.8-0 was used for data collection and reduction.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Absorption correction and scaling were performed using SADABS.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The crystal structure solution was achieved using direct methods as implemented in SHELXTL version 6.14<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and visualized using the XP program.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Missing atoms were subsequently located from difference Fourier synthesis and added to the atom list. Least-squares refinement on <i>F</i><sup>2</sup> using all measured intensities was carried out using the program SHELXTL version 6.14.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> All non-hydrogen atoms were refined including anisotropic displacement parameters. Final data collection and structure refinement parameters: λ = 1.54178 Å, <i>T</i> = 110 K, space group = <i>P</i>2(1), <i>a</i> = 16.6442(11) Å, <i>b</i> = 8.5659(6) Å, <i>c</i> = 26.1351(16) Å, β = 99.053(5)°, <i>Z</i> = 8, reflections collected = 59,133, independent reflections = 12,958 (<i>R</i><sub>int</sub> = 0.0468), completeness = 97.8%, data-to-parameter ratio = 12.7, <i>R</i><sub>1</sub> (<i>I</i> > 2σ) = 0.0542, w<i>R</i><sub>2</sub> (all data) = 0.1490, GOF = 1.032, flack parameter = 0.053(9), and largest difference peak and hole = 0.897/–0.278 e Å<sup>3</sup>. CCDC 2005811 (VIP152) contains the supplementary crystallographic data for this paper. The data can be obtained free of charge from The Cambridge Crystallographic Data Centre via <a href="http://www.ccdc.cam.ac.uk/getstructures" class="extLink">www.ccdc.cam.ac.uk/getstructures</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i134"><a href="/doi/suppl/10.1021/acs.jmedchem.1c01000" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68212" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68212" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c01000?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01000</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Kinase selectivity panel data for VIP152 and LC–MS chromatograms for determination of purity (<a href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv">jm1c01000_si_001.csv (3.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_002.pdf">jm1c01000_si_002.pdf (1.12 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c01000" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stuart Hwang</span> - <span class="hlFld-Affiliation affiliation">Vincerx
Pharma, Inc., 260 Sheridan
Avenue, Suite 400, Palo Alto, California 94306, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cfbcbbbaaebdbbe1a7b8aea1a88fb9a6a1acaabdb7e1aca0a2"><span class="__cf_email__" data-cfemail="354641405447411b5d42545b5275435c5b5650474d1b565a58">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrich Lücking</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2466-5800" title="Orcid link">https://orcid.org/0000-0003-2466-5800</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dirk Kosemund</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niels Böhnke</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip Lienau</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerhard Siemeister</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karsten Denner</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rolf Bohlmann</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans Briem</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ildiko Terebesi</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulf Bömer</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martina Schäfer</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stuart Ince</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominik Mumberg</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Arne Scholz</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raquel Izumi</span> - <span class="hlFld-Affiliation affiliation">Vincerx
Pharma, Inc., 260 Sheridan
Avenue, Suite 400, Palo Alto, California 94306, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0895-2597" title="Orcid link">https://orcid.org/0000-0002-0895-2597</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franz von Nussbaum</span> - <span class="hlFld-Affiliation affiliation">Pharmaceuticals,
Research and Development, Bayer Pharma AG, Müllerstr. 178, Berlin 13353, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written with contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): U.L., D.K., N.B., A.S., P.L., G.S., K.D., R.B., H.B., I.T., U.B., M.S., S.I., D.M., and F.v.N. are or have been employees and stockholders of Bayer AG.R.I. and S.H. are employees and stockholders of Vincerx Pharma, Inc.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i139">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank K. Sauvageot-Witzku, R. Golde, A. Glowczewski, C. Pakebusch, N. Gallus, M. Brands, K. Ziegelbauer, K. Prelle, and R. Droschinski for technical support; O. Schenk for HPLC separations; S. Gründemann, G. Depke, M. Menke, and D. Tshitenge for analytical support; and U. Ganzer for the measurement of physicochemical properties. M. Bergmann and K. Greenfield are acknowledged for valuable technical support with the manuscript. Aurexel Life Sciences Ltd. (<a href="http://www.aurexel.com" class="extLink">www.aurexel.com</a>) is acknowledged for editorial support funded by Bayer AG.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ATR</td><td class="NLM_def"><p class="first last">ataxia telangiectasia and Rad3-related protein</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">CL<sub>b</sub></td><td class="NLM_def"><p class="first last">blood clearance</p></td></tr><tr><td class="NLM_term">ECACC</td><td class="NLM_def"><p class="first last">European Collection of Authenticated Cell Cultures</p></td></tr><tr><td class="NLM_term">FCS</td><td class="NLM_def"><p class="first last">fetal calf serum</p></td></tr><tr><td class="NLM_term">MCL-1</td><td class="NLM_def"><p class="first last">induced myeloid leukemia cell differentiation protein</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent permeability coefficient</p></td></tr><tr><td class="NLM_term">RNA Pol II</td><td class="NLM_def"><p class="first last">RNA polymerase II</p></td></tr><tr><td class="NLM_term">PTEFb</td><td class="NLM_def"><p class="first last">positive transcription elongation factor b</p></td></tr><tr><td class="NLM_term">Q3D</td><td class="NLM_def"><p class="first last">every 3 days</p></td></tr><tr><td class="NLM_term">Q5D</td><td class="NLM_def"><p class="first last">every 5 days</p></td></tr><tr><td class="NLM_term">Q7D</td><td class="NLM_def"><p class="first last">every 7 days</p></td></tr><tr><td class="NLM_term">ratHep</td><td class="NLM_def"><p class="first last">rate hepatocytes</p></td></tr><tr><td class="NLM_term"><i>S</i><sub>w</sub></td><td class="NLM_def"><p class="first last">aqueous solubility</p></td></tr><tr><td class="NLM_term">TCI</td><td class="NLM_def"><p class="first last">Tokyo Chemical Industry</p></td></tr><tr><td class="NLM_term">T/C</td><td class="NLM_def"><p class="first last">treatment-to-control</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultra-performance liquid chromatography</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">steady-state volume of distribution</p></td></tr><tr><td class="NLM_term">WME</td><td class="NLM_def"><p class="first last">Williams’ media E</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i141">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26717" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26717" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldis, P.</span></span> <span> </span><span class="NLM_article-title">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3079</span>– <span class="NLM_lpage">3093</span>, <span class="refDoi"> DOI: 10.1242/dev.091744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1242%2Fdev.091744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=23861057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2013&pages=3079-3093&author=S.+Limauthor=P.+Kaldis&title=Cdks%2C+cyclins+and+CKIs%3A+roles+beyond+cell+cycle+regulation&doi=10.1242%2Fdev.091744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span></div><div class="casAuthors">Lim, Shuhui; Kaldis, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3079-3093</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs).  Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle.  In addn. to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metab., stem cell self-renewal, neuronal functions and spermatogenesis.  Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity.  In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5LNcD6NywLVg90H21EOLACvtfcHk0lhhw8J4JtlvCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF&md5=8095fc798a0e7efeab60d36ee414596f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1242%2Fdev.091744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.091744%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DKaldis%26aufirst%3DP.%26atitle%3DCdks%252C%2520cyclins%2520and%2520CKIs%253A%2520roles%2520beyond%2520cell%2520cycle%2520regulation%26jtitle%3DDevelopment%26date%3D2013%26volume%3D140%26spage%3D3079%26epage%3D3093%26doi%3D10.1242%2Fdev.091744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinases</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi"> DOI: 10.1186/gb4184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1186%2Fgb4184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25180339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=122&author=M.+Malumbres&title=Cyclin-dependent+kinases&doi=10.1186%2Fgb4184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinases</span></div><div class="casAuthors">Malumbres Marcos</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">122</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are protein kinases characterized by needing a separate subunit - a cyclin - that provides domains essential for enzymatic activity.  CDKs play important roles in the control of cell division and modulate transcription in response to several extra- and intracellular cues.  The evolutionary expansion of the CDK family in mammals led to the division of CDKs into three cell-cycle-related subfamilies (Cdk1, Cdk4 and Cdk5) and five transcriptional subfamilies (Cdk7, Cdk8, Cdk9, Cdk11 and Cdk20).  Unlike the prototypical Cdc28 kinase of budding yeast, most of these CDKs bind one or a few cyclins, consistent with functional specialization during evolution.  This review summarizes how, although CDKs are traditionally separated into cell-cycle or transcriptional CDKs, these activities are frequently combined in many family members.  Not surprisingly, deregulation of this family of proteins is a hallmark of several diseases, including cancer, and drug-targeted inhibition of specific members has generated very encouraging results in clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQodaug4zJdjgHqQV89nF-2fW6udTcc2ebyZAVlKwcg4Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D&md5=9fd79ec8f495f712bada681b0a0b5afe</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1186%2Fgb4184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb4184%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26atitle%3DCyclin-dependent%2520kinases%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D122%26doi%3D10.1186%2Fgb4184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of transcriptional cyclin-dependent kinases</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/21541264.2018.1539615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2018.1539615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30409083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=118-136&author=M.+D.+Galbraithauthor=H.+Benderauthor=J.+M.+Espinosa&title=Therapeutic+targeting+of+transcriptional+cyclin-dependent+kinases&doi=10.1080%2F21541264.2018.1539615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of transcriptional cyclin-dependent kinases</span></div><div class="casAuthors">Galbraith, Matthew D.; Bender, Heather; Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-136</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  The fact that many cancer types display transcriptional addiction driven by dysregulation of oncogenic enhancers and transcription factors has led to increased interest in a group of protein kinases, known as transcriptional cyclin dependent kinases (tCDKs), as potential therapeutic targets.  Despite early reservations about targeting a process that is essential to healthy cell types, there is now evidence that targeting tCDKs could provide enough therapeutic window to be effective in the clinic.  Here, we discuss recent developments in this field, with an emphasis on highly-selective inhibitors and the challenges to be addressed before these inhibitors could be used for therapeutic purposes.: CAK: CDK-activating kinase;CDK: cyclin-dependent kinase;CMGC group: CDK-, MAPK-, GSK3-, and CLK-like;CTD: C-terminal repeat domain of the RPB1 subunit of RNA polymerase II;DRB: 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole;mCRPC: metastatic castration-resistant prostate cancer;NSCLC: non-small cell lung cancer;P-TEFb: pos. elongation factor b;RNAPII: RNA polymerase II;S2: serine-2 of CTD repeats;S5: serine-5 of CTD repeats;S7: serine-7 of CTD repeats;SEC: super elongation complex;tCDK: transcriptional cyclin-dependent kinase;TNBC: triple-neg. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptgAOjkQMqqbVg90H21EOLACvtfcHk0lhhw8J4JtlvCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP&md5=1700138890acfc08ad2ab2a81f62c476</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1080%2F21541264.2018.1539615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2018.1539615%26sid%3Dliteratum%253Aachs%26aulast%3DGalbraith%26aufirst%3DM.%2BD.%26aulast%3DBender%26aufirst%3DH.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTherapeutic%2520targeting%2520of%2520transcriptional%2520cyclin-dependent%2520kinases%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D118%26epage%3D136%26doi%3D10.1080%2F21541264.2018.1539615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Martínez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lallena, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanfeliciano, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dios, A.</span></span> <span> </span><span class="NLM_article-title">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs: recent advances (2015–2019)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126637</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.bmcl.2019.126637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=31477350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslWjs73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126637&author=C.+S%C3%A1nchez-Mart%C3%ADnezauthor=M.+J.+Lallenaauthor=S.+G.+Sanfelicianoauthor=A.+de+Dios&title=Cyclin+dependent+kinase+%28CDK%29+inhibitors+as+anticancer+drugs%3A+recent+advances+%282015%E2%80%932019%29&doi=10.1016%2Fj.bmcl.2019.126637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs: Recent advances (2015-2019)</span></div><div class="casAuthors">Sanchez-Martinez, Concepcion; Lallena, Maria Jose; Sanfeliciano, Sonia Gutierrez; de Dios, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">126637</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Sustained proliferative capacity and gene dysregulation are hallmarks of cancer.  In mammalian cells, cyclin-dependent kinases (CDKs) control crit. cell cycle checkpoints and key transcriptional events in response to extracellular and intracellular signals leading to proliferation.  Significant clin. activity for the treatment of hormone receptor pos. metastatic breast cancer has been demonstrated by palbociclib, ribociclib and abemaciclib, dual CDK4/6 inhibitors recently FDA-approved.  SY-1365, a CDK7 inhibitor has shown initial encouraging data in phase I for solid tumors treatment.  These results have rejuvenated the CDKs research field.  This review provides an overview of relevant advances on CDK inhibitor research since 2015 to 2019, with special emphasis on transcriptional CDK inhibitors, new emerging strategies such as target protein degrdn. and compds. under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcZ4sKIv0DfrVg90H21EOLACvtfcHk0lgg3F9cAj3maQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslWjs73N&md5=2808e04d5c09daad462133bcf9b9cb3d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126637%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Mart%25C3%25ADnez%26aufirst%3DC.%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DSanfeliciano%26aufirst%3DS.%2BG.%26aulast%3Dde%2BDios%26aufirst%3DA.%26atitle%3DCyclin%2520dependent%2520kinase%2520%2528CDK%2529%2520inhibitors%2520as%2520anticancer%2520drugs%253A%2520recent%2520advances%2520%25282015%25E2%2580%25932019%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126637%26doi%3D10.1016%2Fj.bmcl.2019.126637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The history and future of targeting cyclin-dependent kinases in cancer therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+history+and+future+of+targeting+cyclin-dependent+kinases+in+cancer+therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0lgg3F9cAj3maQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520history%2520and%2520future%2520of%2520targeting%2520cyclin-dependent%2520kinases%2520in%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeidner, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, J. E.</span></span> <span> </span><span class="NLM_article-title">Clinical activity of alvocidib (flavopiridol) in acute myeloid leukemia</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1312</span>– <span class="NLM_lpage">1318</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2015.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.leukres.2015.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=26521988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2015&pages=1312-1318&author=J.+F.+Zeidnerauthor=J.+E.+Karp&title=Clinical+activity+of+alvocidib+%28flavopiridol%29+in+acute+myeloid+leukemia&doi=10.1016%2Fj.leukres.2015.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical activity of alvocidib (flavopiridol) in acute myeloid leukemia</span></div><div class="casAuthors">Zeidner, Joshua F.; Karp, Judith E.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1312-1318</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There have been minimal therapeutic advancements in acute myeloid leukemia (AML) over the past 4 decades and outcomes remain unsatisfactory.  Alvocidib (formerly flavopiridol) is a multi-serine threonine cyclin-dependent kinase inhibitor with demonstrable in vitro and clin. activity in AML when combined in a timed sequential chemotherapy regimen, FLAM (alvocidib followed by cytarabine continuous infusion and mitoxantrone).  FLAM has been evaluated in sequential phase 1 and phase 2 studies in 149 and 256 relapsed/refractory and newly diagnosed non-favorable risk AML patients, resp., with encouraging findings in both patient populations warranting further investigation.  This review highlights the mechanism of action of alvocidib, pre-clin. studies of alvocidib in AML, and the clin. trials evaluating alvocidib alone and in combination with cytotoxic agents (FLAM) in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsjt81Z_94g7Vg90H21EOLACvtfcHk0lgg3F9cAj3maQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgs77N&md5=fc79799812a30d42a31f647f98445b95</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2015.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2015.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DZeidner%26aufirst%3DJ.%2BF.%26aulast%3DKarp%26aufirst%3DJ.%2BE.%26atitle%3DClinical%2520activity%2520of%2520alvocidib%2520%2528flavopiridol%2529%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DLeuk.%2520Res.%26date%3D2015%26volume%3D39%26spage%3D1312%26epage%3D1318%26doi%3D10.1016%2Fj.leukres.2015.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paruch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKittrick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschmeier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgambellone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seghezzi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paradkar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokosz, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of dinaciclib (SCH 727965): a potent and selective inhibitor of cyclin-dependent kinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1021/ml100051d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100051d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=204-208&author=K.+Paruchauthor=M.+P.+Dwyerauthor=C.+Alvarezauthor=C.+Brownauthor=T.-Y.+Chanauthor=R.+J.+Dollauthor=K.+Keertikarauthor=C.+Knutsonauthor=B.+McKittrickauthor=J.+Riveraauthor=R.+Rossmanauthor=G.+Tuckerauthor=T.+Fischmannauthor=A.+Hruzaauthor=V.+Madisonauthor=A.+A.+Nomeirauthor=Y.+Wangauthor=P.+Kirschmeierauthor=E.+Leesauthor=D.+Parryauthor=N.+Sgambelloneauthor=W.+Seghezziauthor=L.+Schultzauthor=F.+Shanahanauthor=D.+Wiswellauthor=X.+Xuauthor=Q.+Zhouauthor=R.+A.+Jamesauthor=V.+M.+Paradkarauthor=H.+Parkauthor=L.+R.+Rokoszauthor=T.+M.+Staufferauthor=T.+J.+Guzi&title=Discovery+of+dinaciclib+%28SCH+727965%29%3A+a+potent+and+selective+inhibitor+of+cyclin-dependent+kinases&doi=10.1021%2Fml100051d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dinaciclib (SCH 727965): A Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Paruch, Kamil; Dwyer, Michael P.; Alvarez, Carmen; Brown, Courtney; Chan, Tin-Yau; Doll, Ronald J.; Keertikar, Kerry; Knutson, Chad; McKittrick, Brian; Rivera, Jocelyn; Rossman, Randall; Tucker, Greg; Fischmann, Thierry; Hruza, Alan; Madison, Vincent; Nomeir, Amin A.; Wang, Yaolin; Kirschmeier, Paul; Lees, Emma; Parry, David; Sgambellone, Nicole; Seghezzi, Wolfgang; Schultz, Lesley; Shanahan, Frances; Wiswell, Derek; Xu, Xiaoying; Zhou, Quiao; James, Ray A.; Paradkar, Vidyadhar M.; Park, Haengsoon; Rokosz, Laura R.; Stauffer, Tara M.; Guzi, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">204-208</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of cyclin-dependent kinases (CDKs) has emerged as an attractive strategy for the development of novel oncol. therapeutics.  Herein is described the utilization of an in vivo screening approach with integrated efficacy and tolerability parameters to identify candidate CDK inhibitors with a suitable balance of activity and tolerability.  This approach has resulted in the identification of SCH 727965, a potent and selective CDK inhibitor that is currently undergoing clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorrwrMq6CW6rVg90H21EOLACvtfcHk0li7hbhcdR2ZnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D&md5=3b982547e064f658748fe4887a208659</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fml100051d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100051d%26sid%3Dliteratum%253Aachs%26aulast%3DParuch%26aufirst%3DK.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DAlvarez%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DT.-Y.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DKnutson%26aufirst%3DC.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DR.%26aulast%3DTucker%26aufirst%3DG.%26aulast%3DFischmann%26aufirst%3DT.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DParry%26aufirst%3DD.%26aulast%3DSgambellone%26aufirst%3DN.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DJames%26aufirst%3DR.%2BA.%26aulast%3DParadkar%26aufirst%3DV.%2BM.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRokosz%26aufirst%3DL.%2BR.%26aulast%3DStauffer%26aufirst%3DT.%2BM.%26aulast%3DGuzi%26aufirst%3DT.%2BJ.%26atitle%3DDiscovery%2520of%2520dinaciclib%2520%2528SCH%2520727965%2529%253A%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520cyclin-dependent%2520kinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D204%26epage%3D208%26doi%3D10.1021%2Fml100051d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jautelat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wengner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span> <span> </span><span class="NLM_article-title">The lab oddity prevails: discovery of pan-CDK inhibitor (<i>R</i>)-<i>S</i>-cyclopropyl-<i>S</i>-(4-{[4-{[(1<i>R</i>,2<i>R</i>)-2-hydroxy-1-methylpropyl]oxy}-5-(trifluoromethyl)pyrimidin-2-yl]amino}phenyl)sulfoximide (BAY 1000394) for the treatment of cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201300096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=23671017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFemsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1067-1085&author=U.+L%C3%BCckingauthor=R.+Jautelatauthor=M.+Kr%C3%BCgerauthor=T.+Brumbyauthor=P.+Lienauauthor=M.+Sch%C3%A4ferauthor=H.+Briemauthor=J.+Schulzeauthor=A.+Hillischauthor=A.+Reichelauthor=A.+M.+Wengnerauthor=G.+Siemeister&title=The+lab+oddity+prevails%3A+discovery+of+pan-CDK+inhibitor+%28R%29-S-cyclopropyl-S-%284-%7B%5B4-%7B%5B%281R%2C2R%29-2-hydroxy-1-methylpropyl%5Doxy%7D-5-%28trifluoromethyl%29pyrimidin-2-yl%5Damino%7Dphenyl%29sulfoximide+%28BAY+1000394%29+for+the+treatment+of+cancer&doi=10.1002%2Fcmdc.201300096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Lab Oddity Prevails: Discovery of Pan-CDK Inhibitor (R)-S-Cyclopropyl-S-(4-{[4-{[(1R,2R)-2-hydroxy-1-methylpropyl]oxy}-5-(trifluoromethyl)pyrimidin-2-yl]amino}phenyl)sulfoximide (BAY 1000394) for the Treatment of Cancer</span></div><div class="casAuthors">Luecking, Ulrich; Jautelat, Rolf; Krueger, Martin; Brumby, Thomas; Lienau, Philip; Schaefer, Martina; Briem, Hans; Schulze, Julia; Hillisch, Alexander; Reichel, Andreas; Wengner, Antje Margret; Siemeister, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1067-1085</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Lead optimization of a high-throughput screening hit led to the rapid identification of aminopyrimidine ZK 304709 (I), a multitargeted CDK and VEGF-R inhibitor that displayed a promising preclin. profile.  Nevertheless, I failed in phase I studies due to dose-limited absorption and high inter-patient variability, which was attributed to limited aq. soly. and off-target activity against carbonic anhydrases.  Further lead optimization efforts to address the off-target activity profile finally resulted in the introduction of a sulfoximine group, which is still a rather unusual approach in medicinal chem.  However, the sulfoximine series of compds. quickly revealed very interesting properties, culminating in the identification of the nanomolar pan-CDK inhibitor BAY 1000394 (II), which is currently being investigated in phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMw3zOAyohGbVg90H21EOLACvtfcHk0li7hbhcdR2ZnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFemsbo%253D&md5=28b7bc5cb05381908f19d099460d925e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300096%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26aulast%3DJautelat%26aufirst%3DR.%26aulast%3DKr%25C3%25BCger%26aufirst%3DM.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DWengner%26aufirst%3DA.%2BM.%26aulast%3DSiemeister%26aufirst%3DG.%26atitle%3DThe%2520lab%2520oddity%2520prevails%253A%2520discovery%2520of%2520pan-CDK%2520inhibitor%2520%2528R%2529-S-cyclopropyl-S-%25284-%257B%255B4-%257B%255B%25281R%252C2R%2529-2-hydroxy-1-methylpropyl%255Doxy%257D-5-%2528trifluoromethyl%2529pyrimidin-2-yl%255Damino%257Dphenyl%2529sulfoximide%2520%2528BAY%25201000394%2529%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1067%26epage%3D1085%26doi%3D10.1002%2Fcmdc.201300096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boffo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span> <span> </span><span class="NLM_article-title">CDK9 inhibitors in acute myeloid leukemia</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/s13046-018-0704-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1186%2Fs13046-018-0704-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=29471852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Srs7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=36&author=S.+Boffoauthor=A.+Damatoauthor=L.+Alfanoauthor=A.+Giordano&title=CDK9+inhibitors+in+acute+myeloid+leukemia&doi=10.1186%2Fs13046-018-0704-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 inhibitors in acute myeloid leukemia</span></div><div class="casAuthors">Boffo, Silvia; Damato, Angela; Alfano, Luigi; Giordano, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36/1-36/10</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Current treatment for acute myeloid leukemia (AML) is less than optimal, but increased understanding of disease pathobiol. and genomics has led to clin. investigation of novel targeted therapies and rational combinations.  Targeting the cyclin-dependent kinase 9 (CDK9) pathway, which is dysregulated in AML, is an attractive approach.  Inhibition of CDK9 leads to downregulation of cell survival genes regulated by super enhancers such as MCL-1, MYC, and cyclin D1.  As CDK9 inhibitors are nonselective, predictive biomarkers that may help identify patients most likely to respond to CDK9 inhibitors are now being utilized, with the goal of improving efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfFmjuJasS_rVg90H21EOLACvtfcHk0lhvObRIgZYjxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Srs7%252FF&md5=5244fd46c159ded1287507c5d8d1f6ad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fs13046-018-0704-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-018-0704-8%26sid%3Dliteratum%253Aachs%26aulast%3DBoffo%26aufirst%3DS.%26aulast%3DDamato%26aufirst%3DA.%26aulast%3DAlfano%26aufirst%3DL.%26aulast%3DGiordano%26aufirst%3DA.%26atitle%3DCDK9%2520inhibitors%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D37%26spage%3D36%26doi%3D10.1186%2Fs13046-018-0704-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, K.</span></span> <span> </span><span class="NLM_article-title">P-TEFb as a promising therapeutic target</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">838</span>, <span class="refDoi"> DOI: 10.3390/molecules25040838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.3390%2Fmolecules25040838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVCntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=838&author=K.+Fujinaga&title=P-TEFb+as+a+promising+therapeutic+target&doi=10.3390%2Fmolecules25040838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-TEFb as a promising therapeutic target</span></div><div class="casAuthors">Fujinaga, Koh</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">838</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The pos. transcription elongation factor b (P-TEFb) was first identified as a general factor that stimulates transcription elongation by RNA polymerase II (RNAPII), but soon afterwards it turned out to be an essential cellular co-factor of human immunodeficiency virus (HIV) transcription mediated by viral Tat proteins.  Studies on the mechanisms of Tat-dependent HIV transcription have led to radical advances in our knowledge regarding the mechanism of eukaryotic transcription, including the discoveries that P-TEFb-mediated elongation control of cellular transcription is a main regulatory step of gene expression in eukaryotes, and deregulation of P-TEFb activity plays crit. roles in many human diseases and conditions in addn. to HIV/AIDS.  P-TEFb is now recognized as an attractive and promising therapeutic target for inflammation/autoimmune diseases, cardiac hypertrophy, cancer, infectious diseases, etc.  In this review article, I will summarize our knowledge about basic P-TEFb functions, the regulatory mechanism of P-TEFb-dependent transcription, P-TEFb's involvement in biol. processes and diseases, and current approaches to manipulating P-TEFb functions for the treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqaUQwr1s_bVg90H21EOLACvtfcHk0lhvObRIgZYjxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVCntbg%253D&md5=c37aba63dd1e05f3aadf873f239a9e4a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25040838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25040838%26sid%3Dliteratum%253Aachs%26aulast%3DFujinaga%26aufirst%3DK.%26atitle%3DP-TEFb%2520as%2520a%2520promising%2520therapeutic%2520target%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D838%26doi%3D10.3390%2Fmolecules25040838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paparidis, N. F. d. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durvale, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canduri, F.</span></span> <span> </span><span class="NLM_article-title">The emerging picture of CDK9/P-TEFb: more than 20 years of advances since PITALRE</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1039/c6mb00387g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1039%2FC6MB00387G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27833949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BC2snltVOlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=246-276&author=N.+F.+d.+S.+Paparidisauthor=M.+C.+Durvaleauthor=F.+Canduri&title=The+emerging+picture+of+CDK9%2FP-TEFb%3A+more+than+20+years+of+advances+since+PITALRE&doi=10.1039%2Fc6mb00387g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging picture of CDK9/P-TEFb: more than 20 years of advances since PITALRE</span></div><div class="casAuthors">Paparidis Nikolas Ferreira Dos Santos; Canduri Fernanda; Durvale Maxwell Castro</div><div class="citationInfo"><span class="NLM_cas:title">Molecular bioSystems</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-276</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CDK9 is a prominent member of the transcriptional CDKs subfamily, a group of kinases whose function is to control the primary steps of mRNA synthesis and processing by eukaryotic RNA polymerase II.  As a cyclin-dependent kinase, CDK9 activation in vivo depends upon its association with T-type cyclins to assemble the positive transcription elongation factor (P-TEFb).  Although CDK9/P-TEFb phosphorylates the C-terminal domain of RNAP II in the same positions targeted by CDK7 (TFIIH) and CDK8 (Mediator), the former does not participate in the transcription initiation, but rather plays a unique role by driving the polymerase to productive elongation.  In addition to RNAP II CTD, the negative transcription elongation factors DSIF and NELF also represent major CDK9 substrates, whose phosphorylation is required to overcome the proximal pause of the polymerase.  CDK9 is recruited to specific genes through proteins that interact with both P-TEFb and distinct elements in DNA, RNA or chromatin, where it modulates the activity of individual RNAP II transcription complexes.  The regulation of CDK9 function is an intricate network that includes post-translational modifications (phosphorylation/dephosphorylation and acetylation/deacetylation of key residues) as well as the association of P-TEFb with various proteins that can stimulate or inhibit its kinase activity.  Several cases of CDK9 deregulation have been linked to important human diseases, including various types of cancer and also AIDS (due to its essential role in HIV replication).  Not only HIV, but also many other human viruses have been shown to depend strongly on CDK9 activity to be transcribed within host cells.  This review summarizes the main advances made on CDK9/P-TEFb field in more than 20 years, introducing the structural, functional and genetic aspects that have been elucidated ever since.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJulYHhsNcnUmhnSqMnvzDfW6udTcc2eZy4Gx2EAw7gLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snltVOlsA%253D%253D&md5=eb1290cacb40f5c4c7158199fe513b14</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FC6MB00387G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00387G%26sid%3Dliteratum%253Aachs%26aulast%3DPaparidis%26aufirst%3DN.%2BF.%2Bd.%2BS.%26aulast%3DDurvale%26aufirst%3DM.%2BC.%26aulast%3DCanduri%26aufirst%3DF.%26atitle%3DThe%2520emerging%2520picture%2520of%2520CDK9%252FP-TEFb%253A%2520more%2520than%252020%2520years%2520of%2520advances%2520since%2520PITALRE%26jtitle%3DMol.%2520BioSyst.%26date%3D2017%26volume%3D13%26spage%3D246%26epage%3D276%26doi%3D10.1039%2Fc6mb00387g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystof, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fürst, R.</span></span> <span> </span><span class="NLM_article-title">Perspective of cyclin-dependent kinase 9 (CDK9) as a drug target</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2883</span>– <span class="NLM_lpage">2890</span>, <span class="refDoi"> DOI: 10.2174/138161212800672750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.2174%2F138161212800672750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=22571657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2883-2890&author=V.+Krystofauthor=S.+Baumliauthor=R.+F%C3%BCrst&title=Perspective+of+cyclin-dependent+kinase+9+%28CDK9%29+as+a+drug+target&doi=10.2174%2F138161212800672750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective of cyclin-dependent kinase 9 (CDK9) as a drug target</span></div><div class="casAuthors">Krystof, Vladimir; Baumli, Sonja; Furst, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2883-2890</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review,.  The deregulation of cyclin-dependent kinases (CDKs) has been assocd. with many cancer types and has evoked an interest in chem. inhibitors with possible therapeutic benefit.  While most known inhibitors display broad selectivity toward multiple CDKs, recent work highlights CDK9 as the crit. target responsible for the anticancer activity of clin. evaluated drugs.  Here, the authors discuss recent findings provided by structural biologists that may allow further development of highly specific inhibitors of CDK9 toward applications in cancer therapy.  The authors also highlight the role of CDK9 in inflammatory processes and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppF9ic2C9FM7Vg90H21EOLACvtfcHk0litWBCAWKq18g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrw%253D&md5=27c49385a4c0fab8b0ce6bfbef49cc5b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F138161212800672750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672750%26sid%3Dliteratum%253Aachs%26aulast%3DKrystof%26aufirst%3DV.%26aulast%3DBaumli%26aufirst%3DS.%26aulast%3DF%25C3%25BCrst%26aufirst%3DR.%26atitle%3DPerspective%2520of%2520cyclin-dependent%2520kinase%25209%2520%2528CDK9%2529%2520as%2520a%2520drug%2520target%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D2883%26epage%3D2890%26doi%3D10.2174%2F138161212800672750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anshabo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, H.</span></span> <span> </span><span class="NLM_article-title">CDK9: A comprehensive review of its biology, and its role as a potential target for anti-cancer agents</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">678559</span>, <span class="refDoi"> DOI: 10.3389/fonc.2021.678559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.3389%2Ffonc.2021.678559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=34041038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BB2c%252FpsFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=678559&author=A.+T.+Anshaboauthor=R.+Milneauthor=S.+Wangauthor=H.+Albrecht&title=CDK9%3A+A+comprehensive+review+of+its+biology%2C+and+its+role+as+a+potential+target+for+anti-cancer+agents&doi=10.3389%2Ffonc.2021.678559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9: A Comprehensive Review of Its Biology, and Its Role as a Potential Target for Anti-Cancer Agents</span></div><div class="casAuthors">Anshabo Abel Tesfaye; Milne Robert; Wang Shudong; Albrecht Hugo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">678559</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are proteins pivotal to a wide range of cellular functions, most importantly cell division and transcription, and their dysregulations have been implicated as prominent drivers of tumorigenesis.  Besides the well-established role of cell cycle CDKs in cancer, the involvement of transcriptional CDKs has been confirmed more recently.  Most cancers overtly employ CDKs that serve as key regulators of transcription (e.g., CDK9) for a continuous production of short-lived gene products that maintain their survival.  As such, dysregulation of the CDK9 pathway has been observed in various hematological and solid malignancies, making it a valuable anticancer target.  This therapeutic potential has been utilized for the discovery of CDK9 inhibitors, some of which have entered human clinical trials.  This review provides a comprehensive discussion on the structure and biology of CDK9, its role in solid and hematological cancers, and an updated review of the available inhibitors currently being investigated in preclinical and clinical settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFCtCGjAqYpHthwrmtH4OBfW6udTcc2eY7U7fvVKjSabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB2c%252FpsFOisw%253D%253D&md5=38defff76f3815e77ba2c5b2e0ea5821</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2021.678559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2021.678559%26sid%3Dliteratum%253Aachs%26aulast%3DAnshabo%26aufirst%3DA.%2BT.%26aulast%3DMilne%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DH.%26atitle%3DCDK9%253A%2520A%2520comprehensive%2520review%2520of%2520its%2520biology%252C%2520and%2520its%2520role%2520as%2520a%2520potential%2520target%2520for%2520anti-cancer%2520agents%26jtitle%3DFront.%2520Oncol.%26date%3D2021%26volume%3D11%26spage%3D678559%26doi%3D10.3389%2Ffonc.2021.678559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of transcriptional cyclin-dependent kinases</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/21541264.2018.1539615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2018.1539615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30409083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=118-136&author=M.+D.+Galbraithauthor=H.+Benderauthor=J.+M.+Espinosa&title=Therapeutic+targeting+of+transcriptional+cyclin-dependent+kinases&doi=10.1080%2F21541264.2018.1539615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of transcriptional cyclin-dependent kinases</span></div><div class="casAuthors">Galbraith, Matthew D.; Bender, Heather; Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-136</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  The fact that many cancer types display transcriptional addiction driven by dysregulation of oncogenic enhancers and transcription factors has led to increased interest in a group of protein kinases, known as transcriptional cyclin dependent kinases (tCDKs), as potential therapeutic targets.  Despite early reservations about targeting a process that is essential to healthy cell types, there is now evidence that targeting tCDKs could provide enough therapeutic window to be effective in the clinic.  Here, we discuss recent developments in this field, with an emphasis on highly-selective inhibitors and the challenges to be addressed before these inhibitors could be used for therapeutic purposes.: CAK: CDK-activating kinase;CDK: cyclin-dependent kinase;CMGC group: CDK-, MAPK-, GSK3-, and CLK-like;CTD: C-terminal repeat domain of the RPB1 subunit of RNA polymerase II;DRB: 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole;mCRPC: metastatic castration-resistant prostate cancer;NSCLC: non-small cell lung cancer;P-TEFb: pos. elongation factor b;RNAPII: RNA polymerase II;S2: serine-2 of CTD repeats;S5: serine-5 of CTD repeats;S7: serine-7 of CTD repeats;SEC: super elongation complex;tCDK: transcriptional cyclin-dependent kinase;TNBC: triple-neg. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptgAOjkQMqqbVg90H21EOLACvtfcHk0litWBCAWKq18g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP&md5=1700138890acfc08ad2ab2a81f62c476</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F21541264.2018.1539615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2018.1539615%26sid%3Dliteratum%253Aachs%26aulast%3DGalbraith%26aufirst%3DM.%2BD.%26aulast%3DBender%26aufirst%3DH.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTherapeutic%2520targeting%2520of%2520transcriptional%2520cyclin-dependent%2520kinases%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D118%26epage%3D136%26doi%3D10.1080%2F21541264.2018.1539615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eyvazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hejazi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahroba, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamiri, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarhriz, V.</span></span> <span> </span><span class="NLM_article-title">CDK9 as an appealing target for therapeutic interventions</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.2174/1389450119666181026152221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.2174%2F1389450119666181026152221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30362418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=453-464&author=S.+Eyvaziauthor=M.+S.+Hejaziauthor=H.+Kahrobaauthor=M.+Abasiauthor=R.+E.+Zamiriauthor=V.+Tarhriz&title=CDK9+as+an+appealing+target+for+therapeutic+interventions&doi=10.2174%2F1389450119666181026152221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 as an Appealing Target for Therapeutic Interventions</span></div><div class="casAuthors">Eyvazi, Shirin; Hejazi, Mohammad Saeid; Kahroba, Homan; Abasi, Mozghan; Zamiri, Reza Eghdam; Tarhriz, Vahideh</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-464</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Cyclin Dependent Kinase 9 (CDK9) as a serine/threonine kinase belongs to a great no. of CDKs.  CDK9 is the main core of PTEF-b complex and phosphorylates RNA polymerase (RNAP) II besides other transcription factors which regulate gene transcription elongation in numerous physiol. processes.  Multi-functional nature of CDK9 in diverse cellular pathways proposes that it is as an appealing target.  In this review, we summarized the recent findings on the mol. interaction of CDK9 with crit. participant mols. to modulate their activity in various diseases.  Furthermore, the presented review provides a rationale supporting the use of CDK9 as a therapeutic target in clin. developments for crucial diseases; particularly cancers will be reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqT3jDCORzE7Vg90H21EOLACvtfcHk0litWBCAWKq18g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCrsrw%253D&md5=caf0ba95246b3ab6e124bccfe17db682</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F1389450119666181026152221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389450119666181026152221%26sid%3Dliteratum%253Aachs%26aulast%3DEyvazi%26aufirst%3DS.%26aulast%3DHejazi%26aufirst%3DM.%2BS.%26aulast%3DKahroba%26aufirst%3DH.%26aulast%3DAbasi%26aufirst%3DM.%26aulast%3DZamiri%26aufirst%3DR.%2BE.%26aulast%3DTarhriz%26aufirst%3DV.%26atitle%3DCDK9%2520as%2520an%2520appealing%2520target%2520for%2520therapeutic%2520interventions%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2019%26volume%3D20%26spage%3D453%26epage%3D464%26doi%3D10.2174%2F1389450119666181026152221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional CDKs in the spotlight</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1080/21541264.2019.1597479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2019.1597479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30946639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=45-46&author=J.+M.+Espinosa&title=Transcriptional+CDKs+in+the+spotlight&doi=10.1080%2F21541264.2019.1597479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional CDKs in the spotlight</span></div><div class="casAuthors">Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">45-46</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">This article focuses on key regulators of the transcription cycle: the transcriptional cyclin-dependent kinases or tCDKs.  CDKs are a distinct class of serine-threonine protein kinases that share a core set of features, including the requirement of a cyclin partner and phosphorylation of their 'activating T-loops' by a CDK-activating kinase (CAK).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6oQtEx4nRbVg90H21EOLACvtfcHk0lhSTFri13ueDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslejtL8%253D&md5=ddce2c9a8e03c0c020d45ff32200ebdd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F21541264.2019.1597479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2019.1597479%26sid%3Dliteratum%253Aachs%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTranscriptional%2520CDKs%2520in%2520the%2520spotlight%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D45%26epage%3D46%26doi%3D10.1080%2F21541264.2019.1597479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosemund, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terebesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prelle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bömer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rühter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span> <span> </span><span class="NLM_article-title">Identification of atuveciclib (BAY 1143572), the first highly selective, clinical PTEFb/CDK9 inhibitor for the treatment of cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1793</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201700447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=28961375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GltLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1776-1793&author=U.+L%C3%BCckingauthor=A.+Scholzauthor=P.+Lienauauthor=G.+Siemeisterauthor=D.+Kosemundauthor=R.+Bohlmannauthor=H.+Briemauthor=I.+Terebesiauthor=K.+Meyerauthor=K.+Prelleauthor=K.+Dennerauthor=U.+B%C3%B6merauthor=M.+Sch%C3%A4ferauthor=K.+Eisauthor=R.+Valenciaauthor=S.+Inceauthor=F.+von+Nussbaumauthor=D.+Mumbergauthor=K.+Ziegelbauerauthor=B.+Kleblauthor=A.+Choidasauthor=P.+Nussbaumerauthor=M.+Baumannauthor=C.+Schultz-Fademrechtauthor=G.+R%C3%BChterauthor=J.+Eickhoffauthor=M.+Brands&title=Identification+of+atuveciclib+%28BAY+1143572%29%2C+the+first+highly+selective%2C+clinical+PTEFb%2FCDK9+inhibitor+for+the+treatment+of+cancer&doi=10.1002%2Fcmdc.201700447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Atuveciclib (BAY 1143572), the First Highly Selective, Clinical PTEFb/CDK9 Inhibitor for the Treatment of Cancer</span></div><div class="casAuthors">Luecking, Ulrich; Scholz, Arne; Lienau, Philip; Siemeister, Gerhard; Kosemund, Dirk; Bohlmann, Rolf; Briem, Hans; Terebesi, Ildiko; Meyer, Kirstin; Prelle, Katja; Denner, Karsten; Boemer, Ulf; Schaefer, Martina; Eis, Knut; Valencia, Ray; Ince, Stuart; von Nussbaum, Franz; Mumberg, Dominik; Ziegelbauer, Karl; Klebl, Bert; Choidas, Axel; Nussbaumer, Peter; Baumann, Matthias; Schultz-Fademrecht, Carsten; Ruehter, Gerd; Eickhoff, Jan; Brands, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">1776-1793</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Selective inhibition of exclusively transcription-regulating PTEFb/CDK9 is a promising new approach in cancer therapy.  Starting from lead compd. BAY-958, lead optimization efforts strictly focusing on kinase selectivity, physicochem. and DMPK properties finally led to the identification of the orally available clin. candidate atuveciclib (BAY 1143572).  Structurally characterized by an unusual benzyl sulfoximine group, BAY 1143572 exhibited the best overall profile in vitro and in vivo, including high efficacy and good tolerability in xenograft models in mice and rats.  BAY 1143572 is the first potent and highly selective PTEFb/CDK9 inhibitor to enter clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqypgkxBuypa7Vg90H21EOLACvtfcHk0lhSTFri13ueDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GltLvJ&md5=04fb5dae9956246889e1313c87339990</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700447%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DKosemund%26aufirst%3DD.%26aulast%3DBohlmann%26aufirst%3DR.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DTerebesi%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DK.%26aulast%3DPrelle%26aufirst%3DK.%26aulast%3DDenner%26aufirst%3DK.%26aulast%3DB%25C3%25B6mer%26aufirst%3DU.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DM.%26aulast%3DEis%26aufirst%3DK.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DInce%26aufirst%3DS.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DZiegelbauer%26aufirst%3DK.%26aulast%3DKlebl%26aufirst%3DB.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DR%25C3%25BChter%26aufirst%3DG.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DBrands%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520atuveciclib%2520%2528BAY%25201143572%2529%252C%2520the%2520first%2520highly%2520selective%252C%2520clinical%2520PTEFb%252FCDK9%2520inhibitor%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1776%26epage%3D1793%26doi%3D10.1002%2Fcmdc.201700447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criscione, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop-Damkov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roamio Franklin, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chilamakuri, C. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeh, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span> <span> </span><span class="NLM_article-title">AZD4573 is a highly selective CDK9 inhibitor that suppresses MCL-1 and induces apoptosis in hematologic cancer cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-19-1853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1078-0432.CCR-19-1853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=31699827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslylsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=922-934&author=J.+Cidadoauthor=S.+Boikoauthor=T.+Proiaauthor=D.+Fergusonauthor=S.+W.+Criscioneauthor=M.+San+Martinauthor=P.+Pop-Damkovauthor=N.+Suauthor=V.+N.+Roamio+Franklinauthor=C.+S.+R.+Chilamakuriauthor=C.+S.+D%E2%80%99Santosauthor=W.+Shaoauthor=J.+C.+Saehauthor=R.+Kochauthor=D.+M.+Weinstockauthor=M.+Zindaauthor=S.+E.+Fawellauthor=L.+Drew&title=AZD4573+is+a+highly+selective+CDK9+inhibitor+that+suppresses+MCL-1+and+induces+apoptosis+in+hematologic+cancer+cells&doi=10.1158%2F1078-0432.ccr-19-1853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4573 is a highly selective CDK9 inhibitor that suppresses MCL-1 and induces apoptosis in hematologic cancer cells</span></div><div class="casAuthors">Cidado, Justin; Boiko, Scott; Proia, Theresa; Ferguson, Douglas; Criscione, Steven W.; Martin, Maryann San; Pop-Damkov, Petar; Su, Nancy; Franklin, Valar Nila Roamio; Chilamakuri, Chandra Sekhar Reddy; D'Santos, Clive S.; Shao, Wenlin; Saeh, Jamal C.; Koch, Raphael; Weinstock, David M.; Zinda, Michael; Fawell, Stephen E.; Drew, Lisa</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">922-934</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">CDK9 is a transcriptional regulator and potential therapeutic target for many cancers.  Multiple nonselective CDK9 inhibitors have progressed clin. but were limited by a narrow therapeutic window.  This work describes a novel, potent, and highly selective CDK9 inhibitor, AZD4573.  The antitumor activity of AZD4573 was detd. across broad cancer cell line panels in vitro as well as cell line- and patient-derived xenograft models in vivo.  Multiple approaches, including integrated transcriptomic and proteomic analyses, loss-of-function pathway interrogation, and pharmacol. comparisons, were employed to further understand the major mechanism driving AZD4573 activity and to establish an exposure/effect relationship.  AZD4573 is a highly selective and potent CDK9 inhibitor.  It demonstrated rapid induction of apoptosis and subsequent cell death broadly across hematol. cancer models in vitro, and MCL-1 depletion in a dose- and time-dependent manner was identified as a major mechanism through which AZD4573 induces cell death in tumor cells.  This understanding of the mechanism, exposure, and antitumor activity of AZD4573 facilitated development of a robust pharmacokinetic/PD/efficacy model used to inform the clin. trial design.  Selective targeting of CDK9 enables the indirect inhibition of MCL-1, providing a therapeutic option for MCL-1-dependent diseases.  Accordingly, AZD4573 is currently being evaluated in a phase I clin. trial for patients with hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp213MDHb0Y7rVg90H21EOLACvtfcHk0ljOZ4o2NhzZ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslylsLrK&md5=18be82de651b9be77a95f3d7c423b6aa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-19-1853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-19-1853%26sid%3Dliteratum%253Aachs%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DProia%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DCriscione%26aufirst%3DS.%2BW.%26aulast%3DSan%2BMartin%26aufirst%3DM.%26aulast%3DPop-Damkov%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DRoamio%2BFranklin%26aufirst%3DV.%2BN.%26aulast%3DChilamakuri%26aufirst%3DC.%2BS.%2BR.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%2BS.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DSaeh%26aufirst%3DJ.%2BC.%26aulast%3DKoch%26aufirst%3DR.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DDrew%26aufirst%3DL.%26atitle%3DAZD4573%2520is%2520a%2520highly%2520selective%2520CDK9%2520inhibitor%2520that%2520suppresses%2520MCL-1%2520and%2520induces%2520apoptosis%2520in%2520hematologic%2520cancer%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D26%26spage%3D922%26epage%3D934%26doi%3D10.1158%2F1078-0432.ccr-19-1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leifer, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagannathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koren, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urgiles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikochik, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupnow, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilfong, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischofberger, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischoff, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottardis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balk, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">Modulating androgen receptor-driven transcription in prostate cancer with selective CDK9 inhibitors</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.chembiol.2020.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=33086052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFGrt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2021&pages=134-147&author=A.+Richtersauthor=S.+K.+Doyleauthor=D.+B.+Freemanauthor=C.+Leeauthor=B.+S.+Leiferauthor=S.+Jagannathanauthor=F.+Kabingerauthor=J.+V.+Korenauthor=N.+B.+Struntzauthor=J.+Urgilesauthor=R.+A.+Staggauthor=B.+H.+Curtinauthor=D.+Chatterjeeauthor=S.+Matheaauthor=P.+J.+Mikochikauthor=T.+D.+Hopkinsauthor=H.+Gaoauthor=J.+R.+Branchauthor=H.+Xinauthor=L.+Westoverauthor=G.+C.+Bignanauthor=B.+A.+Rupnowauthor=K.+L.+Karlinauthor=C.+M.+Olsonauthor=T.+F.+Westbrookauthor=J.+Vaccaauthor=C.+M.+Wilfongauthor=B.+W.+Trotterauthor=D.+C.+Saffranauthor=N.+Bischofbergerauthor=S.+Knappauthor=J.+W.+Russoauthor=I.+Hicksonauthor=J.+R.+Bischoffauthor=M.+M.+Gottardisauthor=S.+P.+Balkauthor=C.+Y.+Linauthor=M.+S.+Popauthor=A.+N.+Koehler&title=Modulating+androgen+receptor-driven+transcription+in+prostate+cancer+with+selective+CDK9+inhibitors&doi=10.1016%2Fj.chembiol.2020.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating Androgen Receptor-Driven Transcription in Prostate Cancer with Selective CDK9 Inhibitors</span></div><div class="casAuthors">Richters, Andre; Doyle, Shelby K.; Freeman, David B.; Lee, Christina; Leifer, Becky S.; Jagannathan, Sajjeev; Kabinger, Florian; Koren, Jost Vrabic; Struntz, Nicholas B.; Urgiles, Julie; Stagg, Ryan A.; Curtin, Brice H.; Chatterjee, Deep; Mathea, Sebastian; Mikochik, Peter J.; Hopkins, Tamara D.; Gao, Hua; Branch, Jonathan; Xin, Hong; Westover, Lori; Bignan, Gilles C.; Rupnow, Brent A.; Karlin, Kristen L.; Olson, Calla M.; Westbrook, Thomas F.; Vacca, Joseph; Wilfong, Chris M.; Trotter, B. Wesley; Saffran, Douglas C.; Bischofberger, Norbert; Knapp, Stefan; Russo, Joshua W.; Hickson, Ian; Bischoff, James R.; Gottardis, Marco M.; Balk, Steven P.; Lin, Charles Y.; Pop, Marius S.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-147.e14</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Castration-resistant prostate cancers (CRPCs) lose sensitivity to androgen-deprivation therapies but frequently remain dependent on oncogenic transcription driven by the androgen receptor (AR) and its splice variants.  To discover modulators of AR-variant activity, we used a lysate-based small-mol. microarray assay and identified KI-ARv-03 as an AR-variant complex binder that reduces AR-driven transcription and proliferation in prostate cancer cells.  We deduced KI-ARv-03 to be a potent, selective inhibitor of CDK9, an important cofactor for AR, MYC, and other oncogenic transcription factors.  Further optimization resulted in KB-0742, an orally bioavailable, selective CDK9 inhibitor with potent anti-tumor activity in CRPC models.  In 22Rv1 cells, KB-0742 rapidly downregulates nascent transcription, preferentially depleting short half-life transcripts and AR-driven oncogenic programs.  In vivo, oral administration of KB-0742 significantly reduced tumor growth in CRPC, supporting CDK9 inhibition as a promising therapeutic strategy to target AR dependence in CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2omjR3_R4bVg90H21EOLACvtfcHk0ljOZ4o2NhzZ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFGrt7vL&md5=e117c22d5cf06feba7d9ff0235b3860e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DDoyle%26aufirst%3DS.%2BK.%26aulast%3DFreeman%26aufirst%3DD.%2BB.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DLeifer%26aufirst%3DB.%2BS.%26aulast%3DJagannathan%26aufirst%3DS.%26aulast%3DKabinger%26aufirst%3DF.%26aulast%3DKoren%26aufirst%3DJ.%2BV.%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DUrgiles%26aufirst%3DJ.%26aulast%3DStagg%26aufirst%3DR.%2BA.%26aulast%3DCurtin%26aufirst%3DB.%2BH.%26aulast%3DChatterjee%26aufirst%3DD.%26aulast%3DMathea%26aufirst%3DS.%26aulast%3DMikochik%26aufirst%3DP.%2BJ.%26aulast%3DHopkins%26aufirst%3DT.%2BD.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DBranch%26aufirst%3DJ.%2BR.%26aulast%3DXin%26aufirst%3DH.%26aulast%3DWestover%26aufirst%3DL.%26aulast%3DBignan%26aufirst%3DG.%2BC.%26aulast%3DRupnow%26aufirst%3DB.%2BA.%26aulast%3DKarlin%26aufirst%3DK.%2BL.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DWestbrook%26aufirst%3DT.%2BF.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DWilfong%26aufirst%3DC.%2BM.%26aulast%3DTrotter%26aufirst%3DB.%2BW.%26aulast%3DSaffran%26aufirst%3DD.%2BC.%26aulast%3DBischofberger%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DJ.%2BW.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26aulast%3DGottardis%26aufirst%3DM.%2BM.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DPop%26aufirst%3DM.%2BS.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DModulating%2520androgen%2520receptor-driven%2520transcription%2520in%2520prostate%2520cancer%2520with%2520selective%2520CDK9%2520inhibitors%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D134%26epage%3D147%26doi%3D10.1016%2Fj.chembiol.2020.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosemund, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terebesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prelle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, F. v.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span> <span> </span><span class="NLM_article-title">Identification of potent and highly selective PTEFb inhibitor BAY 1251152 for the treatment of cancer: from p.o. to i.v. application via scaffold hops</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">984</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1538-7445.AM2017-984" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=984&author=U.+T.+Lueckingauthor=A.+Scholzauthor=D.+Kosemundauthor=R.+Bohlmannauthor=H.+Briemauthor=P.+Lienauauthor=G.+Siemeisterauthor=I.+Terebesiauthor=K.+Meyerauthor=K.+Prelleauthor=R.+Valenciaauthor=S.+Inceauthor=F.+v.+Nussbaumauthor=D.+Mumbergauthor=K.+Ziegelbauerauthor=M.+Brands&title=Identification+of+potent+and+highly+selective+PTEFb+inhibitor+BAY+1251152+for+the+treatment+of+cancer%3A+from+p.o.+to+i.v.+application+via+scaffold+hops&doi=10.1158%2F1538-7445.AM2017-984"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-984%26sid%3Dliteratum%253Aachs%26aulast%3DLuecking%26aufirst%3DU.%2BT.%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DKosemund%26aufirst%3DD.%26aulast%3DBohlmann%26aufirst%3DR.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DTerebesi%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DK.%26aulast%3DPrelle%26aufirst%3DK.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DInce%26aufirst%3DS.%26aulast%3DNussbaum%26aufirst%3DF.%2Bv.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DZiegelbauer%26aufirst%3DK.%26aulast%3DBrands%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520potent%2520and%2520highly%2520selective%2520PTEFb%2520inhibitor%2520BAY%25201251152%2520for%2520the%2520treatment%2520of%2520cancer%253A%2520from%2520p.o.%2520to%2520i.v.%2520application%2520via%2520scaffold%2520hops%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D984%26doi%3D10.1158%2F1538-7445.AM2017-984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Bayer
AG</span>. <span> </span><span class="NLM_article-title">An open-label Phase I dose-escalation
study to characterize
the safety, tolerability, pharmacokinetics, and maximum tolerated
dose of BAY 1143572 given in a once-daily or an intermittent dosing
schedule in subjects with advanced malignancies; Clinical Trial Results
Synopsis, Study no. 16519 (NCT01938638)</span>, October 9,
2017. <a href="https://clinicaltrials.bayer.com/study/?id=16519" class="extLink">https://clinicaltrials.bayer.com/study/?id=16519</a> (accessed
September 4, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bayer%0AAG.+An+open-label+Phase+I+dose-escalation%0Astudy+to+characterize%0Athe+safety%2C+tolerability%2C+pharmacokinetics%2C+and+maximum+tolerated%0Adose+of+BAY+1143572+given+in+a+once-daily+or+an+intermittent+dosing%0Aschedule+in+subjects+with+advanced+malignancies%3B+Clinical+Trial+Results%0ASynopsis%2C+Study+no.+16519+%28NCT01938638%29%2C+October+9%2C%0A2017.+https%3A%2F%2Fclinicaltrials.bayer.com%2Fstudy%2F%3Fid%3D16519+%28accessed%0ASeptember+4%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAn%2520open-label%2520Phase%2520I%2520dose-escalation%250Astudy%2520to%2520characterize%250Athe%2520safety%252C%2520tolerability%252C%2520pharmacokinetics%252C%2520and%2520maximum%2520tolerated%250Adose%2520of%2520BAY%25201143572%2520given%2520in%2520a%2520once-daily%2520or%2520an%2520intermittent%2520dosing%250Aschedule%2520in%2520subjects%2520with%2520advanced%2520malignancies%253B%2520Clinical%2520Trial%2520Results%250ASynopsis%252C%2520Study%2520no.%252016519%2520%2528NCT01938638%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caraher, E.</span></span> <span> </span><span class="NLM_article-title">Mcl-1 is vital for neutrophil survival</span>. <i>Immunol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1007/s12026-015-8655-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1007%2Fs12026-015-8655-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25929430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFChu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=225-233&author=M.+P.+Murphyauthor=E.+Caraher&title=Mcl-1+is+vital+for+neutrophil+survival&doi=10.1007%2Fs12026-015-8655-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mcl-1 is vital for neutrophil survival</span></div><div class="casAuthors">Murphy, Mark P.; Caraher, Emma</div><div class="citationInfo"><span class="NLM_cas:title">Immunologic Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">IMRSEB</span>;
        ISSN:<span class="NLM_cas:issn">0257-277X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Upon entry to the systemic circulation, neutrophils exhibit a short mean time to cell death.  The viability of most cell types in a steady state is preserved by the interplay of the Bcl-2 family of proteins, wherein the anti-apoptotic members inhibit the action of their pro-apoptotic counterparts.  Neutrophils, however, display absent or severely reduced expression of several anti-apoptotic Bcl-2 family proteins.  Hence, they rely on the expression of Mcl-1, an anti-apoptotic member of the Bcl-2 family, for survival.  This protein is uniquely short-lived relative to related proteins and its loss likely ppts. the induction of apoptosis in neutrophils.  This review describes the role of Mcl-1 in the neutrophil in the context of apoptosis and highlights the proteins' importance to the cell.  We also address neutrophil apoptosis in the broader context of the cells' response to pathogens, focussing particularly on the strategies used by pathogens to manipulate the apoptotic pathway to their own ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBYHR4tH7JybVg90H21EOLACvtfcHk0lgl_HNaYs83vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFChu7c%253D&md5=83f9c2d5c842b353b4642ebbc9e5333f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs12026-015-8655-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12026-015-8655-z%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DM.%2BP.%26aulast%3DCaraher%26aufirst%3DE.%26atitle%3DMcl-1%2520is%2520vital%2520for%2520neutrophil%2520survival%26jtitle%3DImmunol.%2520Res.%26date%3D2015%26volume%3D62%26spage%3D225%26epage%3D233%26doi%3D10.1007%2Fs12026-015-8655-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span> <span> </span><span class="NLM_article-title">Cyclin dependent kinase 9 inhibitors for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8667</span>– <span class="NLM_lpage">8684</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00150</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00150" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslGjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8667-8684&author=Y.+A.+Sonawaneauthor=M.+A.+Taylorauthor=J.+V.+Napoleonauthor=S.+Ranaauthor=J.+I.+Contrerasauthor=A.+Natarajan&title=Cyclin+dependent+kinase+9+inhibitors+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.6b00150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin Dependent Kinase 9 Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Sonawane, Yogesh A.; Taylor, Margaret A.; Napoleon, John Victor; Rana, Sandeep; Contreras, Jacob I.; Natarajan, Amarnath</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8667-8684</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin dependent kinase (CDK) inhibitors have been the topic of intense research for nearly 2 decades due to their widely varied and crit. functions within the cell.  Recently CDK9 has emerged as a druggable target for the development of cancer therapeutics.  CDK9 plays a crucial role in transcription regulation; specifically, CDK9 mediated transcriptional regulation of short-lived antiapoptotic proteins is crit. for the survival of transformed cells.  Focused chem. libraries based on a plethora of scaffolds have resulted in mixed success with regard to the development of selective CDK9 inhibitors.  Here we review the regulation of CDK9, its cellular functions, and common core structures used to target CDK9, along with their selectivity profile and efficacy in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIqU9uSujdrVg90H21EOLACvtfcHk0lgl_HNaYs83vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslGjsLo%253D&md5=0910a61f9c915002ff1073b22b5e20d6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00150%26sid%3Dliteratum%253Aachs%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DNapoleon%26aufirst%3DJ.%2BV.%26aulast%3DRana%26aufirst%3DS.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DNatarajan%26aufirst%3DA.%26atitle%3DCyclin%2520dependent%2520kinase%25209%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8667%26epage%3D8684%26doi%3D10.1021%2Facs.jmedchem.6b00150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvanitis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsher, D. W.</span></span> <span> </span><span class="NLM_article-title">Sustained loss of a neoplastic phenotype by brief inactivation of MYC</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1126/science.1071489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1126%2Fscience.1071489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=12098700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFGqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=102-104&author=M.+Jainauthor=C.+Arvanitisauthor=K.+Chuauthor=W.+Deweyauthor=E.+Leonhardtauthor=M.+Trinhauthor=C.+D.+Sundbergauthor=J.+M.+Bishopauthor=D.+W.+Felsher&title=Sustained+loss+of+a+neoplastic+phenotype+by+brief+inactivation+of+MYC&doi=10.1126%2Fscience.1071489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained loss of a neoplastic phenotype by brief inactivation of MYC</span></div><div class="casAuthors">Jain, Meenakshi; Arvanitis, Constadina; Chu, Kenneth; Dewey, William; Leonhardt, Edith; Trinh, Maxine; Sundberg, Christopher D.; Bishop, J. Michael; Felsher, Dean W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5578</span>),
    <span class="NLM_cas:pages">102-104</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. inactivation of oncogenes is being investigated as a possible therapeutic strategy for cancer.  One potential drawback is that cessation of such therapy may allow reactivation of the oncogene and tumor regrowth.  We used a conditional transgenic mouse model for MYC-induced tumorigenesis to demonstrate that brief inactivation of MYC results in the sustained regression of tumors and the differentiation of osteogenic sarcoma cells into mature osteocytes.  Subsequent reactivation of MYC did not restore the cells' malignant properties but instead induced apoptosis.  Thus, brief MYC inactivation appears to cause epigenetic changes in tumor cells that render them insensitive to MYC-induced tumorigenesis.  These results raise the possibility that transient inactivation of MYC may be an effective therapy for certain cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGKf2LqctHbVg90H21EOLACvtfcHk0lgl_HNaYs83vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFGqsr4%253D&md5=6082c792da67fea09d498eebc9cf8eae</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.1071489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1071489%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DM.%26aulast%3DArvanitis%26aufirst%3DC.%26aulast%3DChu%26aufirst%3DK.%26aulast%3DDewey%26aufirst%3DW.%26aulast%3DLeonhardt%26aufirst%3DE.%26aulast%3DTrinh%26aufirst%3DM.%26aulast%3DSundberg%26aufirst%3DC.%2BD.%26aulast%3DBishop%26aufirst%3DJ.%2BM.%26aulast%3DFelsher%26aufirst%3DD.%2BW.%26atitle%3DSustained%2520loss%2520of%2520a%2520neoplastic%2520phenotype%2520by%2520brief%2520inactivation%2520of%2520MYC%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D102%26epage%3D104%26doi%3D10.1126%2Fscience.1071489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbrink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuijen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijnen, J. H.</span></span> <span> </span><span class="NLM_article-title">Variability in bioavailability of small molecular tyrosine kinase inhibitors</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2015.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.ctrv.2015.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25818541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFentrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2015&pages=412-422&author=M.+Herbrinkauthor=B.+Nuijenauthor=J.+H.+M.+Schellensauthor=J.+H.+Beijnen&title=Variability+in+bioavailability+of+small+molecular+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.ctrv.2015.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Variability in bioavailability of small molecular tyrosine kinase inhibitors</span></div><div class="casAuthors">Herbrink, Maikel; Nuijen, Bastiaan; Schellens, Jan H. M.; Beijnen, Jos H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">412-422</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small mol. tyrosine kinase inhibitors (smTKIs) are in the center of the very quickly expanding area of personalized chemotherapy and oral applicability thereof.  The no. of drugs in this class is rapidly growing, with twenty current approvals by both the European Medicines Agency (EMA) and the Food and Drug Administration (FDA).  The drugs are, however, generally characterized by a poor oral, and thus variable, bioavailability.  This results in significant variation in plasma levels and exposure.  The cause is a complex interplay of factors, including poor aq. soly., issued permeability, membrane transport and enzymic metab.  Addnl., food and drug-drug interactions can play a significant role.  The issues related with an impaired bioavailability generally receive little attention.  To the best of our knowledge, this article is the first to provide an overview of the factors that det. the bioavailability of the smTKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu0wd0bHjmSbVg90H21EOLACvtfcHk0li3_7f2qUZWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFentrw%253D&md5=50fc8e53b9b0f8f4bbe16abe0e320e2d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2015.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2015.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DHerbrink%26aufirst%3DM.%26aulast%3DNuijen%26aufirst%3DB.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26atitle%3DVariability%2520in%2520bioavailability%2520of%2520small%2520molecular%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2015%26volume%3D41%26spage%3D412%26epage%3D422%26doi%3D10.1016%2Fj.ctrv.2015.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Sulfoximines: a neglected opportunity in medicinal chemistry</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">9399</span>– <span class="NLM_lpage">9408</span>, <span class="refDoi"> DOI: 10.1002/anie.201302209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fanie.201302209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CrsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=9399-9408&author=U.+L%C3%BCcking&title=Sulfoximines%3A+a+neglected+opportunity+in+medicinal+chemistry&doi=10.1002%2Fanie.201302209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfoximines: A Neglected Opportunity in Medicinal Chemistry</span></div><div class="casAuthors">Luecking, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">9399-9408</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Innovation has frequently been described as the key to drug discovery.  However, in the daily routine, medicinal chemists often tend to stick to the functional groups and structural elements they know and love.  Blockbuster cancer drug Velcade (bortezomib), for example, was rejected by more than 50 companies, supposedly because of its unusual boronic acid function (as often repeated: "only a moron would put boron in a drug!").  Similarly, in the discovery process of the pan-CDK inhibitor BAY 1000394, the unconventional proposal to introduce a sulfoximine group into the lead series also led to sneers and raised eyebrows, since sulfoximines have seldom been used in medicinal chem.  However, it was the introduction of the sulfoximine group that finally allowed the fundamental issues of the project to be overcome, culminating in the identification of the clin. sulfoximine pan-CDK inhibitor BAY 1000394.  This Minireview provides an overview of a widely neglected opportunity in medicinal chem.-the sulfoximine group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopcpeepAANfbVg90H21EOLACvtfcHk0li3_7f2qUZWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CrsrvI&md5=6850d8d77b1141d9d9344410822c00c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201302209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201302209%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DSulfoximines%253A%2520a%2520neglected%2520opportunity%2520in%2520medicinal%2520chemistry%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D9399%26epage%3D9408%26doi%3D10.1002%2Fanie.201302209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Neglected sulfur(VI) pharmacophores in drug discovery: exploration of novel chemical space by the interplay of drug design and method development</span>. <i>Org. Chem. Front.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1319</span>– <span class="NLM_lpage">1324</span>, <span class="refDoi"> DOI: 10.1039/c8qo01233d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1039%2FC8QO01233D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslemtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=1319-1324&author=U.+L%C3%BCcking&title=Neglected+sulfur%28VI%29+pharmacophores+in+drug+discovery%3A+exploration+of+novel+chemical+space+by+the+interplay+of+drug+design+and+method+development&doi=10.1039%2Fc8qo01233d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Neglected sulfur(VI) pharmacophores in drug discovery: exploration of novel chemical space by the interplay of drug design and method development</span></div><div class="casAuthors">Luecking, U.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Chemistry Frontiers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1319-1324</span>CODEN:
                <span class="NLM_cas:coden">OCFRA8</span>;
        ISSN:<span class="NLM_cas:issn">2052-4129</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Historically, sulfoximines, sulfondiimines and sulfonimidamides have been neglected pharmacophores in drug discovery even though they offer very interesting properties.  This highlight shares the key learnings of various lead optimization approaches at Bayer AG that have successfully utilized these neglected sulfur(VI) functional groups to deliver multiple clin. candidates.  In this context, the key synthetic methods utilized for the synthetic prepn. of these unusual compds. will be outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcX6XF4X_NJbVg90H21EOLACvtfcHk0li3_7f2qUZWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslemtL8%253D&md5=eae9b5efca8d68f3d8ab46e8fe29be99</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC8QO01233D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8QO01233D%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DNeglected%2520sulfur%2528VI%2529%2520pharmacophores%2520in%2520drug%2520discovery%253A%2520exploration%2520of%2520novel%2520chemical%2520space%2520by%2520the%2520interplay%2520of%2520drug%2520design%2520and%2520method%2520development%26jtitle%3DOrg.%2520Chem.%2520Front.%26date%3D2019%26volume%3D6%26spage%3D1319%26epage%3D1324%26doi%3D10.1039%2Fc8qo01233d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frings, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnamm, C.</span></span> <span> </span><span class="NLM_article-title">Sulfoximines from a medicinal chemist’s perspective: physicochemical and in vitro parameters relevant for drug discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.09.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.ejmech.2016.09.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27821325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVahsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2017&pages=225-245&author=M.+Fringsauthor=C.+Bolmauthor=A.+Blumauthor=C.+Gnamm&title=Sulfoximines+from+a+medicinal+chemist%E2%80%99s+perspective%3A+physicochemical+and+in+vitro+parameters+relevant+for+drug+discovery&doi=10.1016%2Fj.ejmech.2016.09.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfoximines from a Medicinal Chemist's Perspective: Physicochemical and in vitro Parameters Relevant for Drug Discovery</span></div><div class="casAuthors">Frings, Marcus; Bolm, Carsten; Blum, Andreas; Gnamm, Christian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">225-245</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Sulfoximines, sulfondiimides and sulfonimidamides are fascinating but not yet fully explored variants of the common sulfone or sulfonamide motif.  In this study, we report the physicochem. and in vitro properties of sulfoximines and compare them with related analogs and isosteres.  Furthermore, we present a matched mol. pair anal. of compds. from drug discovery projects within Boehringer Ingelheim.  We demonstrate that the sulfoximine moiety is a chem. stable, comparatively polar and weakly basic functional group, often leading to favorable aq. soly., permeability and metabolic stability.  Moreover, their addnl. vectors at nitrogen enable simple chem. modifications and thus facilitate exploration and fine-tuning of the mol. properties.  We conclude that sulfoximines and their congeners do not exhibit any intrinsic flaw but significantly enrich the toolbox of medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu0FlXXURKybVg90H21EOLACvtfcHk0li3_7f2qUZWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVahsbfK&md5=8a7939f22425f46ad9eb87fcddd29099</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.09.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.09.091%26sid%3Dliteratum%253Aachs%26aulast%3DFrings%26aufirst%3DM.%26aulast%3DBolm%26aufirst%3DC.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DGnamm%26aufirst%3DC.%26atitle%3DSulfoximines%2520from%2520a%2520medicinal%2520chemist%25E2%2580%2599s%2520perspective%253A%2520physicochemical%2520and%2520in%2520vitro%2520parameters%2520relevant%2520for%2520drug%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D126%26spage%3D225%26epage%3D245%26doi%3D10.1016%2Fj.ejmech.2016.09.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirvent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Novel pieces for the emerging picture of sulfoximines in drug discovery: synthesis and evaluation of sulfoximine analogues of marketed drugs and advanced clinical candidates</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201700044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=28221724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1Sltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=487-501&author=J.+A.+Sirventauthor=U.+L%C3%BCcking&title=Novel+pieces+for+the+emerging+picture+of+sulfoximines+in+drug+discovery%3A+synthesis+and+evaluation+of+sulfoximine+analogues+of+marketed+drugs+and+advanced+clinical+candidates&doi=10.1002%2Fcmdc.201700044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Pieces for the Emerging Picture of Sulfoximines in Drug Discovery: Synthesis and Evaluation of Sulfoximine Analogues of Marketed Drugs and Advanced Clinical Candidates</span></div><div class="casAuthors">Sirvent, Juan Alberto; Luecking, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">487-501</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Sulfoximines have gained considerable recognition as an important structural motif in drug discovery of late.  In particular, the clin. kinase inhibitors for the treatment of cancer, roniciclib (pan-CDK inhibitor), BAY 1143572 (P-TEFb inhibitor), and AZD 6738 (ATR inhibitor), have recently drawn considerable attention.  While the interest in this underrepresented functional group in drug discovery is clearly on the rise, there remains an incomplete understanding of the medicinal-chem.-relevant properties of sulfoximines.  Herein we report the synthesis and in vitro characterization of a variety of sulfoximine analogs of marketed drugs and advanced clin. candidates to gain a better understanding of this neglected functional group and its potential in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-5kfVM3I45LVg90H21EOLACvtfcHk0liBe16M7CDZAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1Sltbo%253D&md5=5b4589d7af32c8984d4fd3027c31c168</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700044%26sid%3Dliteratum%253Aachs%26aulast%3DSirvent%26aufirst%3DJ.%2BA.%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DNovel%2520pieces%2520for%2520the%2520emerging%2520picture%2520of%2520sulfoximines%2520in%2520drug%2520discovery%253A%2520synthesis%2520and%2520evaluation%2520of%2520sulfoximine%2520analogues%2520of%2520marketed%2520drugs%2520and%2520advanced%2520clinical%2520candidates%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D487%26epage%3D501%26doi%3D10.1002%2Fcmdc.201700044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="note"><p class="first last">BAY-332 is a single stereoisomer but the absolute configuration at the sulfoximine sulfur was not determined.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="note"><p class="first last">For additional examples, see:</p></div><div class="NLM_citation" id="cit31a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span>; <span class="NLM_string-name">Kosemund, D.</span>; <span class="NLM_string-name">Scholz, A.</span>; <span class="NLM_string-name">Lienau, P.</span>; <span class="NLM_string-name">Siemeister, G.</span>; <span class="NLM_string-name">Bömer, U.</span>; <span class="NLM_string-name">Bohlmann, R.</span></span> <span> </span><span class="NLM_article-title">Disubstituted 5-fluoro pyrimidine derivatives containing a sulfoximine group</span>. <span class="NLM_patent">WO 2013037894 A1</span>, March 21, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=U.+L%C3%BCcking&author=D.+Kosemund&author=A.+Scholz&author=P.+Lienau&author=G.+Siemeister&author=U.+B%C3%B6mer&author=R.+Bohlmann&title=Disubstituted+5-fluoro+pyrimidine+derivatives+containing+a+sulfoximine+group"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DDisubstituted%25205-fluoro%2520pyrimidine%2520derivatives%2520containing%2520a%2520sulfoximine%2520group%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span>; <span class="NLM_string-name">Kosemund, D.</span>; <span class="NLM_string-name">Scholz, A.</span>; <span class="NLM_string-name">Lienau, P.</span>; <span class="NLM_string-name">Siemeister, G.</span>; <span class="NLM_string-name">Bömer, U.</span>; <span class="NLM_string-name">Bohlmann, R.</span></span> <span> </span><span class="NLM_article-title">Disubstituted 5-fluoro-pyrimidines</span>. <span class="NLM_patent">WO 2013037896 A1</span>, March 21, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=U.+L%C3%BCcking&author=D.+Kosemund&author=A.+Scholz&author=P.+Lienau&author=G.+Siemeister&author=U.+B%C3%B6mer&author=R.+Bohlmann&title=Disubstituted+5-fluoro-pyrimidines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DDisubstituted%25205-fluoro-pyrimidines%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span>For the
assay description, see:<span class="NLM_contrib-group"><span class="NLM_string-name">Onofrey, T.</span>; <span class="NLM_string-name">Kazan, G.</span></span> <i>Performance and Correlation of a 96-Well High Throughput Screening Method to Determine Aqueous Drug Solubility</i>; <span class="NLM_publisher-name">Millipore Corporation</span>: <span class="NLM_publisher-loc">Billerica, MA, USA</span>, <span class="NLM_year">2003</span>; Application Note Lit. No. AN1731EN00.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=T.+Onofrey&author=G.+Kazan&title=Performance+and+Correlation+of+a+96-Well+High+Throughput+Screening+Method+to+Determine+Aqueous+Drug+Solubility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3DOnofrey%26aufirst%3DT.%26jtitle%3DPerformance%2520and%2520Correlation%2520of%2520a%252096-Well%2520High%2520Throughput%2520Screening%2520Method%2520to%2520Determine%2520Aqueous%2520Drug%2520Solubility%26pub%3DMillipore%2520Corporation%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantifying CDK inhibitor selectivity in live cells</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2743</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-16559-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1038%2Fs41467-020-16559-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=32488087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtV2kt7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2743&author=C.+I.+Wellsauthor=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=M.+R.+Ingoldauthor=J.+E.+Pickettauthor=D.+H.+Drewryauthor=K.+M.+Pughauthor=M.+K.+Schwinnauthor=B.+Hwangauthor=H.+Zegzoutiauthor=K.+V.+M.+Huberauthor=M.+Congauthor=P.+L.+Meisenheimerauthor=T.+M.+Willsonauthor=M.+B.+Robers&title=Quantifying+CDK+inhibitor+selectivity+in+live+cells&doi=10.1038%2Fs41467-020-16559-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying CDK inhibitor selectivity in live cells</span></div><div class="casAuthors">Wells, Carrow I.; Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Ingold, Morgan R.; Pickett, Julie E.; Drewry, David H.; Pugh, Kathryn M.; Schwinn, Marie K.; Hwang, Byounghoon; Zegzouti, Hicham; Huber, Kilian V. M.; Cong, Mei; Meisenheimer, Poncho L.; Willson, Timothy M.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2743</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Concerted multidisciplinary efforts have led to the development of Cyclin-Dependent Kinase inhibitors (CDKi's) as small mol. drugs and chem. probes of intracellular CDK function.  However, conflicting data has been reported on the inhibitory potency of CDKi's and a systematic characterization of affinity and selectivity against intracellular CDKs is lacking.  We have developed a panel of cell-permeable energy transfer probes to quantify target occupancy for all 21 human CDKs in live cells, and present a comprehensive evaluation of intracellular isoenzyme potency and selectivity for a collection of 46 clin.-advanced CDKi's and tool mols.  We obsd. unexpected intracellular activity profiles for a no. of CDKi's, offering avenues for repurposing of highly potent mols. as probes for previously unreported targets.  Overall, we provide a broadly applicable method for evaluating the selectivity of CDK inhibitors in living cells, and present a refined set of tool mols. to study CDK function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCCek7iI4hSbVg90H21EOLACvtfcHk0liBe16M7CDZAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtV2kt7bJ&md5=16101d5b781e62fb093864a2303f7d2e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16559-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16559-0%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DPickett%26aufirst%3DJ.%2BE.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DPugh%26aufirst%3DK.%2BM.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DHwang%26aufirst%3DB.%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantifying%2520CDK%2520inhibitor%2520selectivity%2520in%2520live%2520cells%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D2743%26doi%3D10.1038%2Fs41467-020-16559-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roethlisberger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahler, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altenburger, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappenberger, A.</span></span> <span> </span><span class="NLM_article-title">If euhydric and isotonic do not work, what are acceptable pH and osmolality for parenteral drug dosage forms?</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1016/j.xphs.2016.09.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.xphs.2016.09.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27889072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2017&pages=446-456&author=D.+Roethlisbergerauthor=H.-C.+Mahlerauthor=U.+Altenburgerauthor=A.+Pappenberger&title=If+euhydric+and+isotonic+do+not+work%2C+what+are+acceptable+pH+and+osmolality+for+parenteral+drug+dosage+forms%3F&doi=10.1016%2Fj.xphs.2016.09.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">If Euhydric and Isotonic Do Not Work, What Are Acceptable pH and Osmolality for Parenteral Drug Dosage Forms?</span></div><div class="casAuthors">Roethlisberger, Dieter; Mahler, Hanns-Christian; Altenburger, Ulrike; Pappenberger, Astrid</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">446-456</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Parenteral products should aim toward being isotonic and euhydric (physiol. pH).  Yet, due to other considerations, this goal is often not reasonable or doable.  There are no clear allowable ranges related to pH and osmolality, and thus, the objective of this review was to provide a better understanding of acceptable formulation pH, buffer strength, and osmolality taking into account the administration route (i.e., i.m., i.v., s.c.) and administration technique (i.e., bolus, push, infusion).  This evaluation was based on 3 different approaches: conventional, exptl., and parametric.  The conventional way of defining formulation limits was based on std. pH and osmolality ranges.  Exptl. detn. of titratable acidity or in vitro hemolysis testing provided addnl. drug product information.  Finally, the parametric approach was based on the calcn. of theor. values such as (1) the maximal vol. of injection which cannot shift the blood's pH or its molarity out of the physiol. range and (b) a diln. ratio at the injection site and by verifying that threshold values are not exceeded.  The combination of all 3 approaches can support the definition of acceptable pH, buffer strength, and osmolality of formulations and thus may reduce the risk of failure during preclin. and clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocRU3wZe7uALVg90H21EOLACvtfcHk0liTem4FYJaA2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7bP&md5=4fde5a41732655312a086b10df403c21</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.xphs.2016.09.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.xphs.2016.09.034%26sid%3Dliteratum%253Aachs%26aulast%3DRoethlisberger%26aufirst%3DD.%26aulast%3DMahler%26aufirst%3DH.-C.%26aulast%3DAltenburger%26aufirst%3DU.%26aulast%3DPappenberger%26aufirst%3DA.%26atitle%3DIf%2520euhydric%2520and%2520isotonic%2520do%2520not%2520work%252C%2520what%2520are%2520acceptable%2520pH%2520and%2520osmolality%2520for%2520parenteral%2520drug%2520dosage%2520forms%253F%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D106%26spage%3D446%26epage%3D456%26doi%3D10.1016%2Fj.xphs.2016.09.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gries, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, J.</span></span> <span> </span><span class="NLM_article-title">A practical approach to N-(trifluoroacetyl)sulfilimines</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1834</span>, <span class="refDoi"> DOI: 10.1055/s-0034-1378328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1055%2Fs-0034-1378328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=1831-1834&author=J.+Griesauthor=J.+Kr%C3%BCger&title=A+practical+approach+to+N-%28trifluoroacetyl%29sulfilimines&doi=10.1055%2Fs-0034-1378328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A practical approach to N-(trifluoroacetyl)sulfilimines</span></div><div class="casAuthors">Gries, Joerg; Krueger, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">Synlett</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1831-1834</span>CODEN:
                <span class="NLM_cas:coden">SYNLES</span>;
        ISSN:<span class="NLM_cas:issn">0936-5214</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A simple and one-pot transition metal-free synthesis of N-(trifluoroacetyl)-protected sulfilimines I [R1 = 4-O2NC6H4, 4-H3CC6H4, 4-ClC6H4, etc.; R2 = CH3, cyclopropyl] via reaction of sulfides and trifluoroacetamide is described.  The salient features includes the use of safe, low-cost, com. available reagents, mild reaction conditions and a broad substrate scope.  Resoln. of sulfoximine racemate via diastereoselective salt formation is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6rGp6lF6RGLVg90H21EOLACvtfcHk0liTem4FYJaA2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLfK&md5=48d236271e0a87874837b36233d1efa2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1055%2Fs-0034-1378328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0034-1378328%26sid%3Dliteratum%253Aachs%26aulast%3DGries%26aufirst%3DJ.%26aulast%3DKr%25C3%25BCger%26aufirst%3DJ.%26atitle%3DA%2520practical%2520approach%2520to%2520N-%2528trifluoroacetyl%2529sulfilimines%26jtitle%3DSynlett%26date%3D2014%26volume%3D25%26spage%3D1831%26epage%3D1834%26doi%3D10.1055%2Fs-0034-1378328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellerich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellinghaus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serve, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span> <span> </span><span class="NLM_article-title">BAY 1143572, a first-in-class, highly selective, potent and orally available inhibitor of PTEFb/CDK9 currently in Phase I, shows convincing anti-tumor activity in preclinical models of acute myeloid leukemia (AML)</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3022</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2016-3022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1538-7445.AM2016-3022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3022&author=A.+Scholzauthor=T.+Oellerichauthor=A.+Hussainauthor=S.+Lindnerauthor=U.+Lueckingauthor=A.+O.+Walterauthor=P.+Ellinghausauthor=R.+Valenciaauthor=F.+von+Nussbaumauthor=D.+Mumbergauthor=M.+Brandsauthor=S.+Inceauthor=H.+Serveauthor=K.+Ziegelbauer&title=BAY+1143572%2C+a+first-in-class%2C+highly+selective%2C+potent+and+orally+available+inhibitor+of+PTEFb%2FCDK9+currently+in+Phase+I%2C+shows+convincing+anti-tumor+activity+in+preclinical+models+of+acute+myeloid+leukemia+%28AML%29&doi=10.1158%2F1538-7445.AM2016-3022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2016-3022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2016-3022%26sid%3Dliteratum%253Aachs%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DOellerich%26aufirst%3DT.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DLindner%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEllinghaus%26aufirst%3DP.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DBrands%26aufirst%3DM.%26aulast%3DInce%26aufirst%3DS.%26aulast%3DServe%26aufirst%3DH.%26aulast%3DZiegelbauer%26aufirst%3DK.%26atitle%3DBAY%25201143572%252C%2520a%2520first-in-class%252C%2520highly%2520selective%252C%2520potent%2520and%2520orally%2520available%2520inhibitor%2520of%2520PTEFb%252FCDK9%2520currently%2520in%2520Phase%2520I%252C%2520shows%2520convincing%2520anti-tumor%2520activity%2520in%2520preclinical%2520models%2520of%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3022%26doi%3D10.1158%2F1538-7445.AM2016-3022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordoba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lettieri, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boni, V.</span></span> <span> </span><span class="NLM_article-title">Phase I dose escalation study of the first-in-class selective PTEFb inhibitor BAY 1251152 in patients with advanced cancer: Novel target validation and early evidence of clinical activity</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2507</span>, <span class="refDoi"> DOI: 10.1200/jco.2018.36.15_suppl.2507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1200%2FJCO.2018.36.15_suppl.2507" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=2507&author=J.+R.+Diamondauthor=V.+Morenoauthor=E.+A.+Limauthor=R.+Cordobaauthor=C.+Caiauthor=S.+J.+Inceauthor=J.+T.+Lettieriauthor=C.+Merzauthor=R.+Valenciaauthor=V.+Boni&title=Phase+I+dose+escalation+study+of+the+first-in-class+selective+PTEFb+inhibitor+BAY+1251152+in+patients+with+advanced+cancer%3A+Novel+target+validation+and+early+evidence+of+clinical+activity&doi=10.1200%2Fjco.2018.36.15_suppl.2507"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.2507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.2507%26sid%3Dliteratum%253Aachs%26aulast%3DDiamond%26aufirst%3DJ.%2BR.%26aulast%3DMoreno%26aufirst%3DV.%26aulast%3DLim%26aufirst%3DE.%2BA.%26aulast%3DCordoba%26aufirst%3DR.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DInce%26aufirst%3DS.%2BJ.%26aulast%3DLettieri%26aufirst%3DJ.%2BT.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DBoni%26aufirst%3DV.%26atitle%3DPhase%2520I%2520dose%2520escalation%2520study%2520of%2520the%2520first-in-class%2520selective%2520PTEFb%2520inhibitor%2520BAY%25201251152%2520in%2520patients%2520with%2520advanced%2520cancer%253A%2520Novel%2520target%2520validation%2520and%2520early%2520evidence%2520of%2520clinical%2520activity%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D2507%26doi%3D10.1200%2Fjco.2018.36.15_suppl.2507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordoba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morillo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdy, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boix, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genvresse, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowakowski, G. S.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy of VIP152, a CDK9 inhibitor, in patients with double-hit lymphoma (DHL)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">7538</span>, <span class="refDoi"> DOI: 10.1200/jco.2021.39.15_suppl.7538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1200%2FJCO.2021.39.15_suppl.7538" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=7538&author=V.+Morenoauthor=R.+Cordobaauthor=D.+Morilloauthor=J.+R.+Diamondauthor=A.+H.+Hamdyauthor=R.+Izumiauthor=C.+Merzauthor=O.+Boixauthor=I.+Genvresseauthor=G.+S.+Nowakowski&title=Safety+and+efficacy+of+VIP152%2C+a+CDK9+inhibitor%2C+in+patients+with+double-hit+lymphoma+%28DHL%29&doi=10.1200%2Fjco.2021.39.15_suppl.7538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1200%2FJCO.2021.39.15_suppl.7538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2021.39.15_suppl.7538%26sid%3Dliteratum%253Aachs%26aulast%3DMoreno%26aufirst%3DV.%26aulast%3DCordoba%26aufirst%3DR.%26aulast%3DMorillo%26aufirst%3DD.%26aulast%3DDiamond%26aufirst%3DJ.%2BR.%26aulast%3DHamdy%26aufirst%3DA.%2BH.%26aulast%3DIzumi%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DBoix%26aufirst%3DO.%26aulast%3DGenvresse%26aufirst%3DI.%26aulast%3DNowakowski%26aufirst%3DG.%2BS.%26atitle%3DSafety%2520and%2520efficacy%2520of%2520VIP152%252C%2520a%2520CDK9%2520inhibitor%252C%2520in%2520patients%2520with%2520double-hit%2520lymphoma%2520%2528DHL%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2021%26volume%3D39%26spage%3D7538%26doi%3D10.1200%2Fjco.2021.39.15_suppl.7538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kueng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eppenberger, U.</span></span> <span> </span><span class="NLM_article-title">Quantification of cells cultured on 96-well plates</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90710-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2F0003-2697%2889%2990710-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=2604040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADyaK3c7gt1yqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=1989&pages=16-19&author=W.+Kuengauthor=E.+Silberauthor=U.+Eppenberger&title=Quantification+of+cells+cultured+on+96-well+plates&doi=10.1016%2F0003-2697%2889%2990710-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of cells cultured on 96-well plates</span></div><div class="casAuthors">Kueng W; Silber E; Eppenberger U</div><div class="citationInfo"><span class="NLM_cas:title">Analytical biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-9</span>
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">The method for cell number measurement in monolayer cultures by crystal violet staining published recently by Gillies et al. (R.  G.  Gillies, N.  Didier, M.  Denton (1986) Anal.  Biochem. 159, 109-113) was modified and significantly improved.  The procedure was adapted for use in 96-well plates since the method is inherently very sensitive.  Modifications allowed fast and complete solubilization of dye adsorbed by cell nuclei during staining.  Since light absorption of the unstained or destained cell layers is negligible, cell number measurements can be performed in the respective wells.  Due to these features, multiple assays may be carried out rapidly using standard 96-well plate readers.  In addition, it is shown that the sensitivity of the assay can be varied and easily controlled by choosing the appropriate pH during the staining procedure.  This increases the flexibility of the method making it useful for determining cell density of a wide range of different cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDlNEnzvDQwoqdXsLcGTEUfW6udTcc2ebUfR967eLJQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3c7gt1yqtg%253D%253D&md5=407acd99c7de5f4ed856249e8f668d6c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990710-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990710-0%26sid%3Dliteratum%253Aachs%26aulast%3DKueng%26aufirst%3DW.%26aulast%3DSilber%26aufirst%3DE.%26aulast%3DEppenberger%26aufirst%3DU.%26atitle%3DQuantification%2520of%2520cells%2520cultured%2520on%252096-well%2520plates%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D182%26spage%3D16%26epage%3D19%26doi%3D10.1016%2F0003-2697%2889%2990710-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowland, M.</span></span> <span> </span><span class="NLM_article-title">Hepatic clearance of drugs. I. Theoretical considerations of a “well-stirred” model and a “parallel tube” model. Influence of hepatic blood flow, plasma and blood cell binding, and the hepatocellular enzymatic activity on hepatic drug clearance</span>. <i>J. Pharmacokinet. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1007/bf01059688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1007%2FBF01059688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=599411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADyaE1cXhtFGjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1977&pages=625-653&author=K.+S.+Pangauthor=M.+Rowland&title=Hepatic+clearance+of+drugs.+I.+Theoretical+considerations+of+a+%E2%80%9Cwell-stirred%E2%80%9D+model+and+a+%E2%80%9Cparallel+tube%E2%80%9D+model.+Influence+of+hepatic+blood+flow%2C+plasma+and+blood+cell+binding%2C+and+the+hepatocellular+enzymatic+activity+on+hepatic+drug+clearance&doi=10.1007%2Fbf01059688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatic clearance of drugs.  I.  Theoretical considerations of a "well-stirred" model and a "parallel tube" model.  Influence of hepatic blood flow, plasma and blood cell binding, and the hepatocellular enzymatic activity on hepatic drug clearance</span></div><div class="casAuthors">Pang, K. Sandy; Rowland, Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacokinetics and Biopharmaceutics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">625-53</span>CODEN:
                <span class="NLM_cas:coden">JPBPBJ</span>;
        ISSN:<span class="NLM_cas:issn">0090-466X</span>.
    </div><div class="casAbstract">Two commonly used models of hepatic drug clearance were examd.  The well-stirred model (model I) views the liver as a well-stirred compartment with concn. of drug in the liver in equil. with that in the emergent blood.  The parallel tube model (model II) regards the liver as a series of parallel tubes with enzymes distributed evenly around the tubes and the concn. of drug declining along the length of the tube.  Both models were examd. under steady-state considerations in the absence of diffusional limitations (i.e., cell membranes not limiting the movement of drug mols.).  Equations involving the determinants of hepatic drug clearance (hepatic blood flow, fraction of drug in blood unbound, and the hepatocellular enzymic activity) and various pharmacokinetic parameters were derived.  Similarities and differences between the models were explored.  Although both models predict similar hepatic drug clearances under a variety of conditions, marked differences between them become apparent in their predictions of the influence of changes in the determinants of drug clearance on various pharmacokinetic parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDpZaENZqfCbVg90H21EOLACvtfcHk0ljkEEIsAOrjRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhtFGjtLs%253D&md5=6a9e9bf46d9ce3571d8da77495cc9e19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2FBF01059688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01059688%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DK.%2BS.%26aulast%3DRowland%26aufirst%3DM.%26atitle%3DHepatic%2520clearance%2520of%2520drugs.%2520I.%2520Theoretical%2520considerations%2520of%2520a%2520%25E2%2580%259Cwell-stirred%25E2%2580%259D%2520model%2520and%2520a%2520%25E2%2580%259Cparallel%2520tube%25E2%2580%259D%2520model.%2520Influence%2520of%2520hepatic%2520blood%2520flow%252C%2520plasma%2520and%2520blood%2520cell%2520binding%252C%2520and%2520the%2520hepatocellular%2520enzymatic%2520activity%2520on%2520hepatic%2520drug%2520clearance%26jtitle%3DJ.%2520Pharmacokinet.%2520Biopharm.%26date%3D1977%26volume%3D5%26spage%3D625%26epage%3D653%26doi%3D10.1007%2Fbf01059688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a 96-well equilibrium dialysis apparatus for measuring plasma protein binding</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1002/jps.10332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fjps.10332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=12712416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1ehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=967-974&author=M.+J.+Bankerauthor=T.+H.+Clarkauthor=J.+A.+Williams&title=Development+and+validation+of+a+96-well+equilibrium+dialysis+apparatus+for+measuring+plasma+protein+binding&doi=10.1002%2Fjps.10332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a 96-well equilibrium dialysis apparatus for measuring plasma protein binding</span></div><div class="casAuthors">Banker, Michael J.; Clark, Tracey H.; Williams, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">967-974</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A 96-well equil. dialysis block was designed and constructed that is compatible with most std. 96-well format lab. supplies and instruments.  The unique design of the dialysis app. allows one to dispense and aspirate from either or both the sample and dialyzate sides from the top of the app., which is not possible with systems currently on the market.  This feature permits the investigator to analyze a large no. of samples, time points, or replicates in the same expt.  The novel alignment of the dialysis membrane vertically in the well maximizes the surface-to-vol. ratio, eliminates problems assocd. with trapped air pockets, and allows one to add or remove samples independently or all at once.  Furthermore, the design of the app. allows both the sample and dialyzate sides of the dialysis well to be accessible by robotic systems, so assays can be readily automated.  Teflon construction is used to minimize nonspecific binding of test samples to the app.  The device is reusable, easily assembled, and can be shaken in controlled temp. environments to decrease the time required to reach equil. as well as facilitate dissoln. of test compds.  Plasma protein binding values obtained for 10 diverse compds. using std. dialysis equipment and the 96-well dialysis block validates this method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9qyX_9tHHrVg90H21EOLACvtfcHk0ljkEEIsAOrjRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1ehsrs%253D&md5=0948f0a11373c57be06be9e0d604ad40</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjps.10332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10332%26sid%3Dliteratum%253Aachs%26aulast%3DBanker%26aufirst%3DM.%2BJ.%26aulast%3DClark%26aufirst%3DT.%2BH.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%252096-well%2520equilibrium%2520dialysis%2520apparatus%2520for%2520measuring%2520plasma%2520protein%2520binding%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26spage%3D967%26epage%3D974%26doi%3D10.1002%2Fjps.10332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span>See: <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span>; <span class="NLM_string-name">Di, L.</span></span> <i>Drug-like Properties: Concepts, Structure Design and Methods</i>; <span class="NLM_publisher-name">Elsevier Inc./Academic Press</span>: <span class="NLM_publisher-loc">Burlington, MA, USA</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">286</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2FB978-012369520-8.50026-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=276-286&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FB978-012369520-8.50026-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-012369520-8.50026-7%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%26pub%3DElsevier%2520Inc.%252FAcademic%2520Press%26date%3D2008%26spage%3D276%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenz, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brent, D. A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive method for determining hydrophobicity constants by reversed-phase high-performance liquid chromatography</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1021/jm00118a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00118a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADyaL1cXltlGlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1923-1933&author=D.+J.+Minickauthor=J.+H.+Frenzauthor=M.+A.+Patrickauthor=D.+A.+Brent&title=A+comprehensive+method+for+determining+hydrophobicity+constants+by+reversed-phase+high-performance+liquid+chromatography&doi=10.1021%2Fjm00118a010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive method for determining hydrophobicity constants by reversed-phase high-performance liquid chromatography</span></div><div class="casAuthors">Minick, Douglas J.; Frenz, John H.; Patrick, Mark A.; Brent, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1923-33</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A method for detg. hydrophobicity consts. for small, org. mols. by reversed-phase liq. chromatog. (RPLC) is presented.  The method uses capacity ratios measured at a no. of different compns. of MeOH to obtain derived values, denoted log K'w, upon which a new scale of hydrophobicity consts. can be developed.  This scale eliminates potential problems such as peak inversion that hamper RPLC methods by using isocratic data to est. hydrophobicity.  The differential H bonded effect obsd. in most correlations of RPLC data with logarithms of octanol-water partition coeffs. (log Po/w) for compds. of opposite net H bonding capabilities (noncongeners) was minimized by adding trace quantities of n-decylamine and 1-octanol to the eluent and using an octyl-modified silica gel stationary phase.  Values of log K'w are largely column-independent as long as the hydrophobic properties of columns are similar.  The correlation of log k'w values with the logarithms of bovine serum albumin binding consts. (log 1/C) is statistically indistinguishable from the correlation of log 1/C with log Po/w, indicating that this data models log 1/C as well as log Po/w for these compds.  Addnl., the chromatog. system is automatable and thus capable of higher sample throughput than measurements of log Po/w by the shake-flask method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP9rXFRIjKoLVg90H21EOLACvtfcHk0lgPPJbH5UArjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXltlGlt7s%253D&md5=ce6a632baa977f92c51f3341f92e303f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm00118a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00118a010%26sid%3Dliteratum%253Aachs%26aulast%3DMinick%26aufirst%3DD.%2BJ.%26aulast%3DFrenz%26aufirst%3DJ.%2BH.%26aulast%3DPatrick%26aufirst%3DM.%2BA.%26aulast%3DBrent%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520method%2520for%2520determining%2520hydrophobicity%2520constants%2520by%2520reversed-phase%2520high-performance%2520liquid%2520chromatography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1923%26epage%3D1933%26doi%3D10.1021%2Fjm00118a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span> </span><span class="NLM_article-title">R: A language and environment for statistical
computing, R version
3.6.3, The R Project for Statistical Computing</span>; <span class="NLM_publisher-name">The R Foundation</span>. <span class="NLM_year">2020</span>, available from: <a href="https://www.r-project.org/" class="extLink">https://www.r-project.org/</a> (accessed July 13, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%3A+A+language+and+environment+for+statistical%0Acomputing%2C+R+version%0A3.6.3%2C+The+R+Project+for+Statistical+Computing%3B+The+R+Foundation.+2020%2C+available+from%3A+https%3A%2F%2Fwww.r-project.org%2F+%28accessed+July+13%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DR%253A%2520A%2520language%2520and%2520environment%2520for%2520statistical%250Acomputing%252C%2520R%2520version%250A3.6.3%252C%2520The%2520R%2520Project%2520for%2520Statistical%2520Computing%26pub%3DThe%2520R%2520Foundation%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span> <i>Glide</i>, Version 5.7; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+Version+5.7%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baumli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. N.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds form the basis for selective P-TEFb inhibition by DRB</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.chembiol.2010.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=20851342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=931-936&author=S.+Baumliauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Halogen+bonds+form+the+basis+for+selective+P-TEFb+inhibition+by+DRB&doi=10.1016%2Fj.chembiol.2010.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonds Form the Basis for Selective P-TEFb Inhibition by DRB</span></div><div class="casAuthors">Baumli, Sonja; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">931-936</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Cdk9, the kinase of the pos. transcription elongation factor b, is required for processive transcription elongation by RNA polymerase II.  Cdk9 inhibition contributes to the anticancer activity of many Cdk inhibitors under clin. investigation and hence there is interest in selective Cdk9 inhibitors.  DRB (5,6-dichlorobenzimidazone-1-β-D-ribofuranoside) is a commonly used reagent for Cdk9 inhibition in cell biol. studies.  The crystal structures of Cdk9 and Cdk2 in complex with DRB reported here describe the mol. basis for the DRB selectivity toward Cdk9.  The DRB chlorine atoms form halogen bonds that are specific for the Cdk9 kinase hinge region.  Kinetic and thermodn. expts. validate the structural findings and implicate the C-terminal residues of Cdk9 in contributing to the affinity for DRB.  These results open the possibility to exploit halogen atoms in inhibitor design to specifically target Cdk9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMj3v7EG7-1LVg90H21EOLACvtfcHk0lgPPJbH5UArjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyit7fN&md5=88686dcae3c6943da8a05e463d4fd419</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DBaumli%26aufirst%3DS.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DHalogen%2520bonds%2520form%2520the%2520basis%2520for%2520selective%2520P-TEFb%2520inhibition%2520by%2520DRB%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D931%26epage%3D936%26doi%3D10.1016%2Fj.chembiol.2010.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span> <i>LigPrep</i>, Version 2.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+Version+2.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span> <i>APEX II</i>, Version 2011.8-0; <span class="NLM_publisher-name">Bruker
AXS Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+APEX+II%2C+Version+2011.8-0%3B+Bruker%0AAXS+Inc.%3A+Madison%2C+WI%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAPEX%2520II%26pub%3DBruker%250AAXS%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst-Irmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrick, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalke, D.</span></span> <span> </span><span class="NLM_article-title">Comparison of silver and molybdenum microfocus X-ray sources for single-crystal structure determination</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1107/s1600576714022985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1107%2FS1600576714022985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=26089746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=3-10&author=L.+Krauseauthor=R.+Herbst-Irmerauthor=G.+M.+Sheldrickauthor=D.+Stalke&title=Comparison+of+silver+and+molybdenum+microfocus+X-ray+sources+for+single-crystal+structure+determination&doi=10.1107%2Fs1600576714022985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of silver and molybdenum microfocus X-ray sources for single-crystal structure determination</span></div><div class="casAuthors">Krause, Lennard; Herbst-Irmer, Regine; Sheldrick, George M.; Stalke, Dietmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-10</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">1600-5767</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The quality of diffraction data obtained using silver and molybdenum microsources has been compared for six model compds. with a wide range of absorption factors.  The expts. were performed on two 30 W air-cooled Incoatec IμS microfocus sources with multilayer optics mounted on a Bruker D8 goniometer with a SMART APEX II CCD detector.  All data were analyzed, processed and refined using std. Bruker software.  The results show that Ag Kα radiation can be beneficial when heavy elements are involved.  A numerical absorption correction based on the positions and indexes of the crystal faces is shown to be of limited use for the highly focused microsource beams, presumably because the assumption that the crystal is completely bathed in a (top-hat profile) beam of uniform intensity is no longer valid.  Fortunately the empirical corrections implemented in SADABS, although originally intended as a correction for absorption, also correct rather well for the variations in the effective vol. of the crystal irradiated.  In three of the cases studied (two Ag and one Mo) the final SHELXL R1 against all data after application of empirical corrections implemented in SADABS was below 1%.  Since such corrections are designed to optimize the agreement of the intensities of equiv. reflections with different paths through the crystal but the same Bragg 2θ angles, a further correction is required for the 2θ dependence of the absorption.  For this, SADABS uses the transmission factor of a spherical crystal with a user-defined value of μr (where μ is the linear absorption coeff. and r is the effective radius of the crystal); the best results are obtained when r is biased towards the smallest crystal dimension.  The results presented here suggest that the IUCr publication requirement that a numerical absorption correction must be applied for strongly absorbing crystals is in need of revision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPP3imKqKd17Vg90H21EOLACvtfcHk0lgPPJbH5UArjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOmtrc%253D&md5=7a9a2f675fff97a9c18ba4e38bdc57bb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS1600576714022985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600576714022985%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DL.%26aulast%3DHerbst-Irmer%26aufirst%3DR.%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26aulast%3DStalke%26aufirst%3DD.%26atitle%3DComparison%2520of%2520silver%2520and%2520molybdenum%2520microfocus%2520X-ray%2520sources%2520for%2520single-crystal%2520structure%2520determination%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2015%26volume%3D48%26spage%3D3%26epage%3D10%26doi%3D10.1107%2Fs1600576714022985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span> <i>SHELXTL</i>, Version 6.14; <span class="NLM_publisher-name">Bruker AXS
Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, USA</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SHELXTL%2C+Version+6.14%3B+Bruker+AXS%0AInc.%3A+Madison%2C+WI%2C+USA%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSHELXTL%26pub%3DBruker%2520AXS%250AInc%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':[],'ref31':['cit31a','cit31b'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the clinical CDK9 inhibitors atuveciclib, AZD4573, KB-0742, and VIP152.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking mode of atuveciclib in complex with CDK9. The compound was docked into a published X-ray complex of CDK9/CycT1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1">3MY1</a>). The aminotriazine moiety can form two hydrogen bonds to the hinge region of the kinase. For the substituted phenyl ring attached to the triazine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antitumor efficacy of BAY-332 in the MOLM-13 human AML model in mice. Tumor growth in mice treated iv with either vehicle or BAY-332 once daily (QD), once every 3 days (Q3D), or once every 5 days (Q5D). Treatments were started 5 days after tumor cell inoculation. Statistical significances were calculated using the mean tumor volumes at the time point when the vehicle group was sacrificed. Asterisks indicate statistical significance relative to the vehicle control (*<i>p</i> < 0.05 and **<i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking modes of the 5-fluoropyrimidine BAY-332 and the <i>N</i>-(pyridin-2-yl)pyridin-2-amine VIP152 in complex with CDK9. The compounds were docked into a published X-ray complex of CDK9/CycT1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1">3MY1</a>). BAY-332: the aminopyrimidine moiety can form two hydrogen bonds to the hinge region of the kinase. An additional hydrogen bond is formed by the sulfoximine NH to the backbone carbonyl group of Glu107. For the substituted phenyl ring attached to the pyrimidine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated. VIP152: the central aminopyridine moiety can form two hydrogen bonds to the hinge region of the kinase. An additional hydrogen bond is formed by the sulfoximine NH to the backbone carbonyl group of Glu107. For the substituted phenyl ring attached to the central pyridine, a π-stacking interaction with Phe103 and a weak hydrogen bond from the <i>para</i>-fluoro substituent to Lys48 may be postulated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>N</i>-(Pyridin-2-yl)pyrimidin-4-amines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub> (aq), 1,2-dimethoxyethane (DME), 90 °C, 150 min, and 79%; (b) Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 22 h, and 60%; (c) CF<sub>3</sub>C(O)NH<sub>2</sub>, NaO<i>t</i>-Bu, 1,3-dibromo-5,5-dimethylhydantoin, 1,4-dioxane, tetrahydrofuran (THF), 10 °C, and 63%; (d) KMnO<sub>4</sub>, acetone, 50 °C, and crude; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, room temperature (rt), and 15% (over two steps); and (f) chiral preparative HPLC.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of VIP152<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub> (aq), DME, 100 °C, 4 h, and 95%; (b) Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 5 h, and 94%; (c) CF<sub>3</sub>C(O)NH<sub>2</sub>, NaO<i>t</i>-Bu, 1,3-dibromo-5,5-dimethylhydantoin, 1,4-dioxane, THF, 10 °C, and crude; (d) KMnO<sub>4</sub>, acetone, 50 °C, and crude; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, and 10% (over three steps); and (f) chiral preparative HPLC.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antitumor efficacy of atuveciclib and VIP152 in human AML models in rats and mice. (A) Tumor growth in rats treated iv with either vehicle or VIP152 once every 3 days (Q3D), once every 5 days (Q5D), or once weekly (QW). Treatments were started 6 days after tumor cell inoculation. (B) Tumor growth in mice treated iv with either vehicle or VIP152 once weekly (QW). Treatments were started 5 days after tumor cell inoculation. (C) Tumor growth in rats treated with atuveciclib p.o. daily (15 days) dosing or VIP152 iv once weekly (3 doses). Statistical significances were calculated using the mean tumor volumes at the time point when the vehicle group was sacrificed. Asterisks indicate statistical significance relative to the vehicle control (**<i>p</i> < 0.01 and ***<i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/medium/jm1c01000_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Small-molecule X-ray structure of VIP152 (accession code CCDC 2005811).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c01000/20210715/images/large/jm1c01000_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c01000&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i141">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldis, P.</span></span> <span> </span><span class="NLM_article-title">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3079</span>– <span class="NLM_lpage">3093</span>, <span class="refDoi"> DOI: 10.1242/dev.091744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1242%2Fdev.091744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=23861057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2013&pages=3079-3093&author=S.+Limauthor=P.+Kaldis&title=Cdks%2C+cyclins+and+CKIs%3A+roles+beyond+cell+cycle+regulation&doi=10.1242%2Fdev.091744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span></div><div class="casAuthors">Lim, Shuhui; Kaldis, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3079-3093</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs).  Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle.  In addn. to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metab., stem cell self-renewal, neuronal functions and spermatogenesis.  Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity.  In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5LNcD6NywLVg90H21EOLACvtfcHk0liI6SHsi0CRLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF&md5=8095fc798a0e7efeab60d36ee414596f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1242%2Fdev.091744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.091744%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DKaldis%26aufirst%3DP.%26atitle%3DCdks%252C%2520cyclins%2520and%2520CKIs%253A%2520roles%2520beyond%2520cell%2520cycle%2520regulation%26jtitle%3DDevelopment%26date%3D2013%26volume%3D140%26spage%3D3079%26epage%3D3093%26doi%3D10.1242%2Fdev.091744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinases</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi"> DOI: 10.1186/gb4184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1186%2Fgb4184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25180339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=122&author=M.+Malumbres&title=Cyclin-dependent+kinases&doi=10.1186%2Fgb4184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinases</span></div><div class="casAuthors">Malumbres Marcos</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">122</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are protein kinases characterized by needing a separate subunit - a cyclin - that provides domains essential for enzymatic activity.  CDKs play important roles in the control of cell division and modulate transcription in response to several extra- and intracellular cues.  The evolutionary expansion of the CDK family in mammals led to the division of CDKs into three cell-cycle-related subfamilies (Cdk1, Cdk4 and Cdk5) and five transcriptional subfamilies (Cdk7, Cdk8, Cdk9, Cdk11 and Cdk20).  Unlike the prototypical Cdc28 kinase of budding yeast, most of these CDKs bind one or a few cyclins, consistent with functional specialization during evolution.  This review summarizes how, although CDKs are traditionally separated into cell-cycle or transcriptional CDKs, these activities are frequently combined in many family members.  Not surprisingly, deregulation of this family of proteins is a hallmark of several diseases, including cancer, and drug-targeted inhibition of specific members has generated very encouraging results in clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQodaug4zJdjgHqQV89nF-2fW6udTcc2eY4aSvA3db0bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D&md5=9fd79ec8f495f712bada681b0a0b5afe</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1186%2Fgb4184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb4184%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26atitle%3DCyclin-dependent%2520kinases%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D122%26doi%3D10.1186%2Fgb4184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of transcriptional cyclin-dependent kinases</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/21541264.2018.1539615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2018.1539615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30409083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=118-136&author=M.+D.+Galbraithauthor=H.+Benderauthor=J.+M.+Espinosa&title=Therapeutic+targeting+of+transcriptional+cyclin-dependent+kinases&doi=10.1080%2F21541264.2018.1539615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of transcriptional cyclin-dependent kinases</span></div><div class="casAuthors">Galbraith, Matthew D.; Bender, Heather; Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-136</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  The fact that many cancer types display transcriptional addiction driven by dysregulation of oncogenic enhancers and transcription factors has led to increased interest in a group of protein kinases, known as transcriptional cyclin dependent kinases (tCDKs), as potential therapeutic targets.  Despite early reservations about targeting a process that is essential to healthy cell types, there is now evidence that targeting tCDKs could provide enough therapeutic window to be effective in the clinic.  Here, we discuss recent developments in this field, with an emphasis on highly-selective inhibitors and the challenges to be addressed before these inhibitors could be used for therapeutic purposes.: CAK: CDK-activating kinase;CDK: cyclin-dependent kinase;CMGC group: CDK-, MAPK-, GSK3-, and CLK-like;CTD: C-terminal repeat domain of the RPB1 subunit of RNA polymerase II;DRB: 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole;mCRPC: metastatic castration-resistant prostate cancer;NSCLC: non-small cell lung cancer;P-TEFb: pos. elongation factor b;RNAPII: RNA polymerase II;S2: serine-2 of CTD repeats;S5: serine-5 of CTD repeats;S7: serine-7 of CTD repeats;SEC: super elongation complex;tCDK: transcriptional cyclin-dependent kinase;TNBC: triple-neg. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptgAOjkQMqqbVg90H21EOLACvtfcHk0liL7YucYanxKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP&md5=1700138890acfc08ad2ab2a81f62c476</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1080%2F21541264.2018.1539615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2018.1539615%26sid%3Dliteratum%253Aachs%26aulast%3DGalbraith%26aufirst%3DM.%2BD.%26aulast%3DBender%26aufirst%3DH.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTherapeutic%2520targeting%2520of%2520transcriptional%2520cyclin-dependent%2520kinases%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D118%26epage%3D136%26doi%3D10.1080%2F21541264.2018.1539615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Martínez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lallena, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanfeliciano, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dios, A.</span></span> <span> </span><span class="NLM_article-title">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs: recent advances (2015–2019)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126637</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.bmcl.2019.126637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=31477350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslWjs73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126637&author=C.+S%C3%A1nchez-Mart%C3%ADnezauthor=M.+J.+Lallenaauthor=S.+G.+Sanfelicianoauthor=A.+de+Dios&title=Cyclin+dependent+kinase+%28CDK%29+inhibitors+as+anticancer+drugs%3A+recent+advances+%282015%E2%80%932019%29&doi=10.1016%2Fj.bmcl.2019.126637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin dependent kinase (CDK) inhibitors as anticancer drugs: Recent advances (2015-2019)</span></div><div class="casAuthors">Sanchez-Martinez, Concepcion; Lallena, Maria Jose; Sanfeliciano, Sonia Gutierrez; de Dios, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">126637</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Sustained proliferative capacity and gene dysregulation are hallmarks of cancer.  In mammalian cells, cyclin-dependent kinases (CDKs) control crit. cell cycle checkpoints and key transcriptional events in response to extracellular and intracellular signals leading to proliferation.  Significant clin. activity for the treatment of hormone receptor pos. metastatic breast cancer has been demonstrated by palbociclib, ribociclib and abemaciclib, dual CDK4/6 inhibitors recently FDA-approved.  SY-1365, a CDK7 inhibitor has shown initial encouraging data in phase I for solid tumors treatment.  These results have rejuvenated the CDKs research field.  This review provides an overview of relevant advances on CDK inhibitor research since 2015 to 2019, with special emphasis on transcriptional CDK inhibitors, new emerging strategies such as target protein degrdn. and compds. under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcZ4sKIv0DfrVg90H21EOLACvtfcHk0liL7YucYanxKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslWjs73N&md5=2808e04d5c09daad462133bcf9b9cb3d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126637%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-Mart%25C3%25ADnez%26aufirst%3DC.%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DSanfeliciano%26aufirst%3DS.%2BG.%26aulast%3Dde%2BDios%26aufirst%3DA.%26atitle%3DCyclin%2520dependent%2520kinase%2520%2528CDK%2529%2520inhibitors%2520as%2520anticancer%2520drugs%253A%2520recent%2520advances%2520%25282015%25E2%2580%25932019%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126637%26doi%3D10.1016%2Fj.bmcl.2019.126637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The history and future of targeting cyclin-dependent kinases in cancer therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+history+and+future+of+targeting+cyclin-dependent+kinases+in+cancer+therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0liL7YucYanxKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520history%2520and%2520future%2520of%2520targeting%2520cyclin-dependent%2520kinases%2520in%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeidner, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, J. E.</span></span> <span> </span><span class="NLM_article-title">Clinical activity of alvocidib (flavopiridol) in acute myeloid leukemia</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1312</span>– <span class="NLM_lpage">1318</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2015.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.leukres.2015.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=26521988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2015&pages=1312-1318&author=J.+F.+Zeidnerauthor=J.+E.+Karp&title=Clinical+activity+of+alvocidib+%28flavopiridol%29+in+acute+myeloid+leukemia&doi=10.1016%2Fj.leukres.2015.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical activity of alvocidib (flavopiridol) in acute myeloid leukemia</span></div><div class="casAuthors">Zeidner, Joshua F.; Karp, Judith E.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1312-1318</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There have been minimal therapeutic advancements in acute myeloid leukemia (AML) over the past 4 decades and outcomes remain unsatisfactory.  Alvocidib (formerly flavopiridol) is a multi-serine threonine cyclin-dependent kinase inhibitor with demonstrable in vitro and clin. activity in AML when combined in a timed sequential chemotherapy regimen, FLAM (alvocidib followed by cytarabine continuous infusion and mitoxantrone).  FLAM has been evaluated in sequential phase 1 and phase 2 studies in 149 and 256 relapsed/refractory and newly diagnosed non-favorable risk AML patients, resp., with encouraging findings in both patient populations warranting further investigation.  This review highlights the mechanism of action of alvocidib, pre-clin. studies of alvocidib in AML, and the clin. trials evaluating alvocidib alone and in combination with cytotoxic agents (FLAM) in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsjt81Z_94g7Vg90H21EOLACvtfcHk0liL7YucYanxKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgs77N&md5=fc79799812a30d42a31f647f98445b95</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2015.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2015.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DZeidner%26aufirst%3DJ.%2BF.%26aulast%3DKarp%26aufirst%3DJ.%2BE.%26atitle%3DClinical%2520activity%2520of%2520alvocidib%2520%2528flavopiridol%2529%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DLeuk.%2520Res.%26date%3D2015%26volume%3D39%26spage%3D1312%26epage%3D1318%26doi%3D10.1016%2Fj.leukres.2015.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paruch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKittrick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschmeier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgambellone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seghezzi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paradkar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokosz, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of dinaciclib (SCH 727965): a potent and selective inhibitor of cyclin-dependent kinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1021/ml100051d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100051d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=204-208&author=K.+Paruchauthor=M.+P.+Dwyerauthor=C.+Alvarezauthor=C.+Brownauthor=T.-Y.+Chanauthor=R.+J.+Dollauthor=K.+Keertikarauthor=C.+Knutsonauthor=B.+McKittrickauthor=J.+Riveraauthor=R.+Rossmanauthor=G.+Tuckerauthor=T.+Fischmannauthor=A.+Hruzaauthor=V.+Madisonauthor=A.+A.+Nomeirauthor=Y.+Wangauthor=P.+Kirschmeierauthor=E.+Leesauthor=D.+Parryauthor=N.+Sgambelloneauthor=W.+Seghezziauthor=L.+Schultzauthor=F.+Shanahanauthor=D.+Wiswellauthor=X.+Xuauthor=Q.+Zhouauthor=R.+A.+Jamesauthor=V.+M.+Paradkarauthor=H.+Parkauthor=L.+R.+Rokoszauthor=T.+M.+Staufferauthor=T.+J.+Guzi&title=Discovery+of+dinaciclib+%28SCH+727965%29%3A+a+potent+and+selective+inhibitor+of+cyclin-dependent+kinases&doi=10.1021%2Fml100051d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dinaciclib (SCH 727965): A Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Paruch, Kamil; Dwyer, Michael P.; Alvarez, Carmen; Brown, Courtney; Chan, Tin-Yau; Doll, Ronald J.; Keertikar, Kerry; Knutson, Chad; McKittrick, Brian; Rivera, Jocelyn; Rossman, Randall; Tucker, Greg; Fischmann, Thierry; Hruza, Alan; Madison, Vincent; Nomeir, Amin A.; Wang, Yaolin; Kirschmeier, Paul; Lees, Emma; Parry, David; Sgambellone, Nicole; Seghezzi, Wolfgang; Schultz, Lesley; Shanahan, Frances; Wiswell, Derek; Xu, Xiaoying; Zhou, Quiao; James, Ray A.; Paradkar, Vidyadhar M.; Park, Haengsoon; Rokosz, Laura R.; Stauffer, Tara M.; Guzi, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">204-208</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of cyclin-dependent kinases (CDKs) has emerged as an attractive strategy for the development of novel oncol. therapeutics.  Herein is described the utilization of an in vivo screening approach with integrated efficacy and tolerability parameters to identify candidate CDK inhibitors with a suitable balance of activity and tolerability.  This approach has resulted in the identification of SCH 727965, a potent and selective CDK inhibitor that is currently undergoing clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorrwrMq6CW6rVg90H21EOLACvtfcHk0lh1CyrmncLvaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D&md5=3b982547e064f658748fe4887a208659</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fml100051d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100051d%26sid%3Dliteratum%253Aachs%26aulast%3DParuch%26aufirst%3DK.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DAlvarez%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DT.-Y.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DKnutson%26aufirst%3DC.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DR.%26aulast%3DTucker%26aufirst%3DG.%26aulast%3DFischmann%26aufirst%3DT.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DParry%26aufirst%3DD.%26aulast%3DSgambellone%26aufirst%3DN.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DJames%26aufirst%3DR.%2BA.%26aulast%3DParadkar%26aufirst%3DV.%2BM.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRokosz%26aufirst%3DL.%2BR.%26aulast%3DStauffer%26aufirst%3DT.%2BM.%26aulast%3DGuzi%26aufirst%3DT.%2BJ.%26atitle%3DDiscovery%2520of%2520dinaciclib%2520%2528SCH%2520727965%2529%253A%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520cyclin-dependent%2520kinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D204%26epage%3D208%26doi%3D10.1021%2Fml100051d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jautelat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wengner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span> <span> </span><span class="NLM_article-title">The lab oddity prevails: discovery of pan-CDK inhibitor (<i>R</i>)-<i>S</i>-cyclopropyl-<i>S</i>-(4-{[4-{[(1<i>R</i>,2<i>R</i>)-2-hydroxy-1-methylpropyl]oxy}-5-(trifluoromethyl)pyrimidin-2-yl]amino}phenyl)sulfoximide (BAY 1000394) for the treatment of cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201300096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=23671017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFemsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1067-1085&author=U.+L%C3%BCckingauthor=R.+Jautelatauthor=M.+Kr%C3%BCgerauthor=T.+Brumbyauthor=P.+Lienauauthor=M.+Sch%C3%A4ferauthor=H.+Briemauthor=J.+Schulzeauthor=A.+Hillischauthor=A.+Reichelauthor=A.+M.+Wengnerauthor=G.+Siemeister&title=The+lab+oddity+prevails%3A+discovery+of+pan-CDK+inhibitor+%28R%29-S-cyclopropyl-S-%284-%7B%5B4-%7B%5B%281R%2C2R%29-2-hydroxy-1-methylpropyl%5Doxy%7D-5-%28trifluoromethyl%29pyrimidin-2-yl%5Damino%7Dphenyl%29sulfoximide+%28BAY+1000394%29+for+the+treatment+of+cancer&doi=10.1002%2Fcmdc.201300096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Lab Oddity Prevails: Discovery of Pan-CDK Inhibitor (R)-S-Cyclopropyl-S-(4-{[4-{[(1R,2R)-2-hydroxy-1-methylpropyl]oxy}-5-(trifluoromethyl)pyrimidin-2-yl]amino}phenyl)sulfoximide (BAY 1000394) for the Treatment of Cancer</span></div><div class="casAuthors">Luecking, Ulrich; Jautelat, Rolf; Krueger, Martin; Brumby, Thomas; Lienau, Philip; Schaefer, Martina; Briem, Hans; Schulze, Julia; Hillisch, Alexander; Reichel, Andreas; Wengner, Antje Margret; Siemeister, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1067-1085</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Lead optimization of a high-throughput screening hit led to the rapid identification of aminopyrimidine ZK 304709 (I), a multitargeted CDK and VEGF-R inhibitor that displayed a promising preclin. profile.  Nevertheless, I failed in phase I studies due to dose-limited absorption and high inter-patient variability, which was attributed to limited aq. soly. and off-target activity against carbonic anhydrases.  Further lead optimization efforts to address the off-target activity profile finally resulted in the introduction of a sulfoximine group, which is still a rather unusual approach in medicinal chem.  However, the sulfoximine series of compds. quickly revealed very interesting properties, culminating in the identification of the nanomolar pan-CDK inhibitor BAY 1000394 (II), which is currently being investigated in phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMw3zOAyohGbVg90H21EOLACvtfcHk0lh1CyrmncLvaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFemsbo%253D&md5=28b7bc5cb05381908f19d099460d925e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300096%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26aulast%3DJautelat%26aufirst%3DR.%26aulast%3DKr%25C3%25BCger%26aufirst%3DM.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DWengner%26aufirst%3DA.%2BM.%26aulast%3DSiemeister%26aufirst%3DG.%26atitle%3DThe%2520lab%2520oddity%2520prevails%253A%2520discovery%2520of%2520pan-CDK%2520inhibitor%2520%2528R%2529-S-cyclopropyl-S-%25284-%257B%255B4-%257B%255B%25281R%252C2R%2529-2-hydroxy-1-methylpropyl%255Doxy%257D-5-%2528trifluoromethyl%2529pyrimidin-2-yl%255Damino%257Dphenyl%2529sulfoximide%2520%2528BAY%25201000394%2529%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1067%26epage%3D1085%26doi%3D10.1002%2Fcmdc.201300096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boffo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span> <span> </span><span class="NLM_article-title">CDK9 inhibitors in acute myeloid leukemia</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/s13046-018-0704-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1186%2Fs13046-018-0704-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=29471852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Srs7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=36&author=S.+Boffoauthor=A.+Damatoauthor=L.+Alfanoauthor=A.+Giordano&title=CDK9+inhibitors+in+acute+myeloid+leukemia&doi=10.1186%2Fs13046-018-0704-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 inhibitors in acute myeloid leukemia</span></div><div class="casAuthors">Boffo, Silvia; Damato, Angela; Alfano, Luigi; Giordano, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36/1-36/10</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Current treatment for acute myeloid leukemia (AML) is less than optimal, but increased understanding of disease pathobiol. and genomics has led to clin. investigation of novel targeted therapies and rational combinations.  Targeting the cyclin-dependent kinase 9 (CDK9) pathway, which is dysregulated in AML, is an attractive approach.  Inhibition of CDK9 leads to downregulation of cell survival genes regulated by super enhancers such as MCL-1, MYC, and cyclin D1.  As CDK9 inhibitors are nonselective, predictive biomarkers that may help identify patients most likely to respond to CDK9 inhibitors are now being utilized, with the goal of improving efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfFmjuJasS_rVg90H21EOLACvtfcHk0lig67rD24Ix8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Srs7%252FF&md5=5244fd46c159ded1287507c5d8d1f6ad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fs13046-018-0704-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-018-0704-8%26sid%3Dliteratum%253Aachs%26aulast%3DBoffo%26aufirst%3DS.%26aulast%3DDamato%26aufirst%3DA.%26aulast%3DAlfano%26aufirst%3DL.%26aulast%3DGiordano%26aufirst%3DA.%26atitle%3DCDK9%2520inhibitors%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D37%26spage%3D36%26doi%3D10.1186%2Fs13046-018-0704-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, K.</span></span> <span> </span><span class="NLM_article-title">P-TEFb as a promising therapeutic target</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">838</span>, <span class="refDoi"> DOI: 10.3390/molecules25040838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.3390%2Fmolecules25040838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVCntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=838&author=K.+Fujinaga&title=P-TEFb+as+a+promising+therapeutic+target&doi=10.3390%2Fmolecules25040838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-TEFb as a promising therapeutic target</span></div><div class="casAuthors">Fujinaga, Koh</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">838</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The pos. transcription elongation factor b (P-TEFb) was first identified as a general factor that stimulates transcription elongation by RNA polymerase II (RNAPII), but soon afterwards it turned out to be an essential cellular co-factor of human immunodeficiency virus (HIV) transcription mediated by viral Tat proteins.  Studies on the mechanisms of Tat-dependent HIV transcription have led to radical advances in our knowledge regarding the mechanism of eukaryotic transcription, including the discoveries that P-TEFb-mediated elongation control of cellular transcription is a main regulatory step of gene expression in eukaryotes, and deregulation of P-TEFb activity plays crit. roles in many human diseases and conditions in addn. to HIV/AIDS.  P-TEFb is now recognized as an attractive and promising therapeutic target for inflammation/autoimmune diseases, cardiac hypertrophy, cancer, infectious diseases, etc.  In this review article, I will summarize our knowledge about basic P-TEFb functions, the regulatory mechanism of P-TEFb-dependent transcription, P-TEFb's involvement in biol. processes and diseases, and current approaches to manipulating P-TEFb functions for the treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqaUQwr1s_bVg90H21EOLACvtfcHk0lig67rD24Ix8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVCntbg%253D&md5=c37aba63dd1e05f3aadf873f239a9e4a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25040838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25040838%26sid%3Dliteratum%253Aachs%26aulast%3DFujinaga%26aufirst%3DK.%26atitle%3DP-TEFb%2520as%2520a%2520promising%2520therapeutic%2520target%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D838%26doi%3D10.3390%2Fmolecules25040838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paparidis, N. F. d. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durvale, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canduri, F.</span></span> <span> </span><span class="NLM_article-title">The emerging picture of CDK9/P-TEFb: more than 20 years of advances since PITALRE</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1039/c6mb00387g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1039%2FC6MB00387G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27833949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BC2snltVOlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=246-276&author=N.+F.+d.+S.+Paparidisauthor=M.+C.+Durvaleauthor=F.+Canduri&title=The+emerging+picture+of+CDK9%2FP-TEFb%3A+more+than+20+years+of+advances+since+PITALRE&doi=10.1039%2Fc6mb00387g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging picture of CDK9/P-TEFb: more than 20 years of advances since PITALRE</span></div><div class="casAuthors">Paparidis Nikolas Ferreira Dos Santos; Canduri Fernanda; Durvale Maxwell Castro</div><div class="citationInfo"><span class="NLM_cas:title">Molecular bioSystems</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-276</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CDK9 is a prominent member of the transcriptional CDKs subfamily, a group of kinases whose function is to control the primary steps of mRNA synthesis and processing by eukaryotic RNA polymerase II.  As a cyclin-dependent kinase, CDK9 activation in vivo depends upon its association with T-type cyclins to assemble the positive transcription elongation factor (P-TEFb).  Although CDK9/P-TEFb phosphorylates the C-terminal domain of RNAP II in the same positions targeted by CDK7 (TFIIH) and CDK8 (Mediator), the former does not participate in the transcription initiation, but rather plays a unique role by driving the polymerase to productive elongation.  In addition to RNAP II CTD, the negative transcription elongation factors DSIF and NELF also represent major CDK9 substrates, whose phosphorylation is required to overcome the proximal pause of the polymerase.  CDK9 is recruited to specific genes through proteins that interact with both P-TEFb and distinct elements in DNA, RNA or chromatin, where it modulates the activity of individual RNAP II transcription complexes.  The regulation of CDK9 function is an intricate network that includes post-translational modifications (phosphorylation/dephosphorylation and acetylation/deacetylation of key residues) as well as the association of P-TEFb with various proteins that can stimulate or inhibit its kinase activity.  Several cases of CDK9 deregulation have been linked to important human diseases, including various types of cancer and also AIDS (due to its essential role in HIV replication).  Not only HIV, but also many other human viruses have been shown to depend strongly on CDK9 activity to be transcribed within host cells.  This review summarizes the main advances made on CDK9/P-TEFb field in more than 20 years, introducing the structural, functional and genetic aspects that have been elucidated ever since.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJulYHhsNcnUmhnSqMnvzDfW6udTcc2eajcqtQKsH6s7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snltVOlsA%253D%253D&md5=eb1290cacb40f5c4c7158199fe513b14</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2FC6MB00387G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00387G%26sid%3Dliteratum%253Aachs%26aulast%3DPaparidis%26aufirst%3DN.%2BF.%2Bd.%2BS.%26aulast%3DDurvale%26aufirst%3DM.%2BC.%26aulast%3DCanduri%26aufirst%3DF.%26atitle%3DThe%2520emerging%2520picture%2520of%2520CDK9%252FP-TEFb%253A%2520more%2520than%252020%2520years%2520of%2520advances%2520since%2520PITALRE%26jtitle%3DMol.%2520BioSyst.%26date%3D2017%26volume%3D13%26spage%3D246%26epage%3D276%26doi%3D10.1039%2Fc6mb00387g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystof, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fürst, R.</span></span> <span> </span><span class="NLM_article-title">Perspective of cyclin-dependent kinase 9 (CDK9) as a drug target</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2883</span>– <span class="NLM_lpage">2890</span>, <span class="refDoi"> DOI: 10.2174/138161212800672750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.2174%2F138161212800672750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=22571657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2883-2890&author=V.+Krystofauthor=S.+Baumliauthor=R.+F%C3%BCrst&title=Perspective+of+cyclin-dependent+kinase+9+%28CDK9%29+as+a+drug+target&doi=10.2174%2F138161212800672750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective of cyclin-dependent kinase 9 (CDK9) as a drug target</span></div><div class="casAuthors">Krystof, Vladimir; Baumli, Sonja; Furst, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2883-2890</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review,.  The deregulation of cyclin-dependent kinases (CDKs) has been assocd. with many cancer types and has evoked an interest in chem. inhibitors with possible therapeutic benefit.  While most known inhibitors display broad selectivity toward multiple CDKs, recent work highlights CDK9 as the crit. target responsible for the anticancer activity of clin. evaluated drugs.  Here, the authors discuss recent findings provided by structural biologists that may allow further development of highly specific inhibitors of CDK9 toward applications in cancer therapy.  The authors also highlight the role of CDK9 in inflammatory processes and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppF9ic2C9FM7Vg90H21EOLACvtfcHk0lgmBBcCf_kS-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrw%253D&md5=27c49385a4c0fab8b0ce6bfbef49cc5b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F138161212800672750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672750%26sid%3Dliteratum%253Aachs%26aulast%3DKrystof%26aufirst%3DV.%26aulast%3DBaumli%26aufirst%3DS.%26aulast%3DF%25C3%25BCrst%26aufirst%3DR.%26atitle%3DPerspective%2520of%2520cyclin-dependent%2520kinase%25209%2520%2528CDK9%2529%2520as%2520a%2520drug%2520target%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D2883%26epage%3D2890%26doi%3D10.2174%2F138161212800672750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anshabo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, H.</span></span> <span> </span><span class="NLM_article-title">CDK9: A comprehensive review of its biology, and its role as a potential target for anti-cancer agents</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">678559</span>, <span class="refDoi"> DOI: 10.3389/fonc.2021.678559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.3389%2Ffonc.2021.678559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=34041038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADC%252BB2c%252FpsFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=678559&author=A.+T.+Anshaboauthor=R.+Milneauthor=S.+Wangauthor=H.+Albrecht&title=CDK9%3A+A+comprehensive+review+of+its+biology%2C+and+its+role+as+a+potential+target+for+anti-cancer+agents&doi=10.3389%2Ffonc.2021.678559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9: A Comprehensive Review of Its Biology, and Its Role as a Potential Target for Anti-Cancer Agents</span></div><div class="casAuthors">Anshabo Abel Tesfaye; Milne Robert; Wang Shudong; Albrecht Hugo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">678559</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are proteins pivotal to a wide range of cellular functions, most importantly cell division and transcription, and their dysregulations have been implicated as prominent drivers of tumorigenesis.  Besides the well-established role of cell cycle CDKs in cancer, the involvement of transcriptional CDKs has been confirmed more recently.  Most cancers overtly employ CDKs that serve as key regulators of transcription (e.g., CDK9) for a continuous production of short-lived gene products that maintain their survival.  As such, dysregulation of the CDK9 pathway has been observed in various hematological and solid malignancies, making it a valuable anticancer target.  This therapeutic potential has been utilized for the discovery of CDK9 inhibitors, some of which have entered human clinical trials.  This review provides a comprehensive discussion on the structure and biology of CDK9, its role in solid and hematological cancers, and an updated review of the available inhibitors currently being investigated in preclinical and clinical settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFCtCGjAqYpHthwrmtH4OBfW6udTcc2ea9RdoK9hn027ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB2c%252FpsFOisw%253D%253D&md5=38defff76f3815e77ba2c5b2e0ea5821</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2021.678559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2021.678559%26sid%3Dliteratum%253Aachs%26aulast%3DAnshabo%26aufirst%3DA.%2BT.%26aulast%3DMilne%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DH.%26atitle%3DCDK9%253A%2520A%2520comprehensive%2520review%2520of%2520its%2520biology%252C%2520and%2520its%2520role%2520as%2520a%2520potential%2520target%2520for%2520anti-cancer%2520agents%26jtitle%3DFront.%2520Oncol.%26date%3D2021%26volume%3D11%26spage%3D678559%26doi%3D10.3389%2Ffonc.2021.678559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of transcriptional cyclin-dependent kinases</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/21541264.2018.1539615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2018.1539615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30409083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=118-136&author=M.+D.+Galbraithauthor=H.+Benderauthor=J.+M.+Espinosa&title=Therapeutic+targeting+of+transcriptional+cyclin-dependent+kinases&doi=10.1080%2F21541264.2018.1539615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of transcriptional cyclin-dependent kinases</span></div><div class="casAuthors">Galbraith, Matthew D.; Bender, Heather; Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-136</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  The fact that many cancer types display transcriptional addiction driven by dysregulation of oncogenic enhancers and transcription factors has led to increased interest in a group of protein kinases, known as transcriptional cyclin dependent kinases (tCDKs), as potential therapeutic targets.  Despite early reservations about targeting a process that is essential to healthy cell types, there is now evidence that targeting tCDKs could provide enough therapeutic window to be effective in the clinic.  Here, we discuss recent developments in this field, with an emphasis on highly-selective inhibitors and the challenges to be addressed before these inhibitors could be used for therapeutic purposes.: CAK: CDK-activating kinase;CDK: cyclin-dependent kinase;CMGC group: CDK-, MAPK-, GSK3-, and CLK-like;CTD: C-terminal repeat domain of the RPB1 subunit of RNA polymerase II;DRB: 5,6-dichloro-1-β-D-ribofuranosylbenzimidazole;mCRPC: metastatic castration-resistant prostate cancer;NSCLC: non-small cell lung cancer;P-TEFb: pos. elongation factor b;RNAPII: RNA polymerase II;S2: serine-2 of CTD repeats;S5: serine-5 of CTD repeats;S7: serine-7 of CTD repeats;SEC: super elongation complex;tCDK: transcriptional cyclin-dependent kinase;TNBC: triple-neg. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptgAOjkQMqqbVg90H21EOLACvtfcHk0lgmBBcCf_kS-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKjs7vP&md5=1700138890acfc08ad2ab2a81f62c476</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F21541264.2018.1539615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2018.1539615%26sid%3Dliteratum%253Aachs%26aulast%3DGalbraith%26aufirst%3DM.%2BD.%26aulast%3DBender%26aufirst%3DH.%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTherapeutic%2520targeting%2520of%2520transcriptional%2520cyclin-dependent%2520kinases%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D118%26epage%3D136%26doi%3D10.1080%2F21541264.2018.1539615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eyvazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hejazi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahroba, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamiri, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarhriz, V.</span></span> <span> </span><span class="NLM_article-title">CDK9 as an appealing target for therapeutic interventions</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.2174/1389450119666181026152221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.2174%2F1389450119666181026152221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30362418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=453-464&author=S.+Eyvaziauthor=M.+S.+Hejaziauthor=H.+Kahrobaauthor=M.+Abasiauthor=R.+E.+Zamiriauthor=V.+Tarhriz&title=CDK9+as+an+appealing+target+for+therapeutic+interventions&doi=10.2174%2F1389450119666181026152221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 as an Appealing Target for Therapeutic Interventions</span></div><div class="casAuthors">Eyvazi, Shirin; Hejazi, Mohammad Saeid; Kahroba, Homan; Abasi, Mozghan; Zamiri, Reza Eghdam; Tarhriz, Vahideh</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-464</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Cyclin Dependent Kinase 9 (CDK9) as a serine/threonine kinase belongs to a great no. of CDKs.  CDK9 is the main core of PTEF-b complex and phosphorylates RNA polymerase (RNAP) II besides other transcription factors which regulate gene transcription elongation in numerous physiol. processes.  Multi-functional nature of CDK9 in diverse cellular pathways proposes that it is as an appealing target.  In this review, we summarized the recent findings on the mol. interaction of CDK9 with crit. participant mols. to modulate their activity in various diseases.  Furthermore, the presented review provides a rationale supporting the use of CDK9 as a therapeutic target in clin. developments for crucial diseases; particularly cancers will be reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqT3jDCORzE7Vg90H21EOLACvtfcHk0lgmBBcCf_kS-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCrsrw%253D&md5=caf0ba95246b3ab6e124bccfe17db682</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F1389450119666181026152221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389450119666181026152221%26sid%3Dliteratum%253Aachs%26aulast%3DEyvazi%26aufirst%3DS.%26aulast%3DHejazi%26aufirst%3DM.%2BS.%26aulast%3DKahroba%26aufirst%3DH.%26aulast%3DAbasi%26aufirst%3DM.%26aulast%3DZamiri%26aufirst%3DR.%2BE.%26aulast%3DTarhriz%26aufirst%3DV.%26atitle%3DCDK9%2520as%2520an%2520appealing%2520target%2520for%2520therapeutic%2520interventions%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2019%26volume%3D20%26spage%3D453%26epage%3D464%26doi%3D10.2174%2F1389450119666181026152221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, J. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional CDKs in the spotlight</span>. <i>Transcription</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1080/21541264.2019.1597479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1080%2F21541264.2019.1597479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=30946639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=45-46&author=J.+M.+Espinosa&title=Transcriptional+CDKs+in+the+spotlight&doi=10.1080%2F21541264.2019.1597479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional CDKs in the spotlight</span></div><div class="casAuthors">Espinosa, Joaquin M.</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">45-46</span>CODEN:
                <span class="NLM_cas:coden">TRANH2</span>;
        ISSN:<span class="NLM_cas:issn">2154-1272</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">This article focuses on key regulators of the transcription cycle: the transcriptional cyclin-dependent kinases or tCDKs.  CDKs are a distinct class of serine-threonine protein kinases that share a core set of features, including the requirement of a cyclin partner and phosphorylation of their 'activating T-loops' by a CDK-activating kinase (CAK).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6oQtEx4nRbVg90H21EOLACvtfcHk0lgAKg1tpQP__Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslejtL8%253D&md5=ddce2c9a8e03c0c020d45ff32200ebdd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F21541264.2019.1597479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21541264.2019.1597479%26sid%3Dliteratum%253Aachs%26aulast%3DEspinosa%26aufirst%3DJ.%2BM.%26atitle%3DTranscriptional%2520CDKs%2520in%2520the%2520spotlight%26jtitle%3DTranscription%26date%3D2019%26volume%3D10%26spage%3D45%26epage%3D46%26doi%3D10.1080%2F21541264.2019.1597479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosemund, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terebesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prelle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bömer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choidas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rühter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span> <span> </span><span class="NLM_article-title">Identification of atuveciclib (BAY 1143572), the first highly selective, clinical PTEFb/CDK9 inhibitor for the treatment of cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1793</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201700447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=28961375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GltLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1776-1793&author=U.+L%C3%BCckingauthor=A.+Scholzauthor=P.+Lienauauthor=G.+Siemeisterauthor=D.+Kosemundauthor=R.+Bohlmannauthor=H.+Briemauthor=I.+Terebesiauthor=K.+Meyerauthor=K.+Prelleauthor=K.+Dennerauthor=U.+B%C3%B6merauthor=M.+Sch%C3%A4ferauthor=K.+Eisauthor=R.+Valenciaauthor=S.+Inceauthor=F.+von+Nussbaumauthor=D.+Mumbergauthor=K.+Ziegelbauerauthor=B.+Kleblauthor=A.+Choidasauthor=P.+Nussbaumerauthor=M.+Baumannauthor=C.+Schultz-Fademrechtauthor=G.+R%C3%BChterauthor=J.+Eickhoffauthor=M.+Brands&title=Identification+of+atuveciclib+%28BAY+1143572%29%2C+the+first+highly+selective%2C+clinical+PTEFb%2FCDK9+inhibitor+for+the+treatment+of+cancer&doi=10.1002%2Fcmdc.201700447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Atuveciclib (BAY 1143572), the First Highly Selective, Clinical PTEFb/CDK9 Inhibitor for the Treatment of Cancer</span></div><div class="casAuthors">Luecking, Ulrich; Scholz, Arne; Lienau, Philip; Siemeister, Gerhard; Kosemund, Dirk; Bohlmann, Rolf; Briem, Hans; Terebesi, Ildiko; Meyer, Kirstin; Prelle, Katja; Denner, Karsten; Boemer, Ulf; Schaefer, Martina; Eis, Knut; Valencia, Ray; Ince, Stuart; von Nussbaum, Franz; Mumberg, Dominik; Ziegelbauer, Karl; Klebl, Bert; Choidas, Axel; Nussbaumer, Peter; Baumann, Matthias; Schultz-Fademrecht, Carsten; Ruehter, Gerd; Eickhoff, Jan; Brands, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">1776-1793</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Selective inhibition of exclusively transcription-regulating PTEFb/CDK9 is a promising new approach in cancer therapy.  Starting from lead compd. BAY-958, lead optimization efforts strictly focusing on kinase selectivity, physicochem. and DMPK properties finally led to the identification of the orally available clin. candidate atuveciclib (BAY 1143572).  Structurally characterized by an unusual benzyl sulfoximine group, BAY 1143572 exhibited the best overall profile in vitro and in vivo, including high efficacy and good tolerability in xenograft models in mice and rats.  BAY 1143572 is the first potent and highly selective PTEFb/CDK9 inhibitor to enter clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqypgkxBuypa7Vg90H21EOLACvtfcHk0lgAKg1tpQP__Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GltLvJ&md5=04fb5dae9956246889e1313c87339990</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700447%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DKosemund%26aufirst%3DD.%26aulast%3DBohlmann%26aufirst%3DR.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DTerebesi%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DK.%26aulast%3DPrelle%26aufirst%3DK.%26aulast%3DDenner%26aufirst%3DK.%26aulast%3DB%25C3%25B6mer%26aufirst%3DU.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DM.%26aulast%3DEis%26aufirst%3DK.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DInce%26aufirst%3DS.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DZiegelbauer%26aufirst%3DK.%26aulast%3DKlebl%26aufirst%3DB.%26aulast%3DChoidas%26aufirst%3DA.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DR%25C3%25BChter%26aufirst%3DG.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DBrands%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520atuveciclib%2520%2528BAY%25201143572%2529%252C%2520the%2520first%2520highly%2520selective%252C%2520clinical%2520PTEFb%252FCDK9%2520inhibitor%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1776%26epage%3D1793%26doi%3D10.1002%2Fcmdc.201700447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criscione, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop-Damkov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roamio Franklin, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chilamakuri, C. S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeh, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span> <span> </span><span class="NLM_article-title">AZD4573 is a highly selective CDK9 inhibitor that suppresses MCL-1 and induces apoptosis in hematologic cancer cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-19-1853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1078-0432.CCR-19-1853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=31699827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslylsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=922-934&author=J.+Cidadoauthor=S.+Boikoauthor=T.+Proiaauthor=D.+Fergusonauthor=S.+W.+Criscioneauthor=M.+San+Martinauthor=P.+Pop-Damkovauthor=N.+Suauthor=V.+N.+Roamio+Franklinauthor=C.+S.+R.+Chilamakuriauthor=C.+S.+D%E2%80%99Santosauthor=W.+Shaoauthor=J.+C.+Saehauthor=R.+Kochauthor=D.+M.+Weinstockauthor=M.+Zindaauthor=S.+E.+Fawellauthor=L.+Drew&title=AZD4573+is+a+highly+selective+CDK9+inhibitor+that+suppresses+MCL-1+and+induces+apoptosis+in+hematologic+cancer+cells&doi=10.1158%2F1078-0432.ccr-19-1853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4573 is a highly selective CDK9 inhibitor that suppresses MCL-1 and induces apoptosis in hematologic cancer cells</span></div><div class="casAuthors">Cidado, Justin; Boiko, Scott; Proia, Theresa; Ferguson, Douglas; Criscione, Steven W.; Martin, Maryann San; Pop-Damkov, Petar; Su, Nancy; Franklin, Valar Nila Roamio; Chilamakuri, Chandra Sekhar Reddy; D'Santos, Clive S.; Shao, Wenlin; Saeh, Jamal C.; Koch, Raphael; Weinstock, David M.; Zinda, Michael; Fawell, Stephen E.; Drew, Lisa</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">922-934</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">CDK9 is a transcriptional regulator and potential therapeutic target for many cancers.  Multiple nonselective CDK9 inhibitors have progressed clin. but were limited by a narrow therapeutic window.  This work describes a novel, potent, and highly selective CDK9 inhibitor, AZD4573.  The antitumor activity of AZD4573 was detd. across broad cancer cell line panels in vitro as well as cell line- and patient-derived xenograft models in vivo.  Multiple approaches, including integrated transcriptomic and proteomic analyses, loss-of-function pathway interrogation, and pharmacol. comparisons, were employed to further understand the major mechanism driving AZD4573 activity and to establish an exposure/effect relationship.  AZD4573 is a highly selective and potent CDK9 inhibitor.  It demonstrated rapid induction of apoptosis and subsequent cell death broadly across hematol. cancer models in vitro, and MCL-1 depletion in a dose- and time-dependent manner was identified as a major mechanism through which AZD4573 induces cell death in tumor cells.  This understanding of the mechanism, exposure, and antitumor activity of AZD4573 facilitated development of a robust pharmacokinetic/PD/efficacy model used to inform the clin. trial design.  Selective targeting of CDK9 enables the indirect inhibition of MCL-1, providing a therapeutic option for MCL-1-dependent diseases.  Accordingly, AZD4573 is currently being evaluated in a phase I clin. trial for patients with hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp213MDHb0Y7rVg90H21EOLACvtfcHk0lgAKg1tpQP__Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslylsLrK&md5=18be82de651b9be77a95f3d7c423b6aa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-19-1853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-19-1853%26sid%3Dliteratum%253Aachs%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DProia%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DCriscione%26aufirst%3DS.%2BW.%26aulast%3DSan%2BMartin%26aufirst%3DM.%26aulast%3DPop-Damkov%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DRoamio%2BFranklin%26aufirst%3DV.%2BN.%26aulast%3DChilamakuri%26aufirst%3DC.%2BS.%2BR.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%2BS.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DSaeh%26aufirst%3DJ.%2BC.%26aulast%3DKoch%26aufirst%3DR.%26aulast%3DWeinstock%26aufirst%3DD.%2BM.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DDrew%26aufirst%3DL.%26atitle%3DAZD4573%2520is%2520a%2520highly%2520selective%2520CDK9%2520inhibitor%2520that%2520suppresses%2520MCL-1%2520and%2520induces%2520apoptosis%2520in%2520hematologic%2520cancer%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D26%26spage%3D922%26epage%3D934%26doi%3D10.1158%2F1078-0432.ccr-19-1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leifer, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagannathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koren, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urgiles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikochik, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupnow, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilfong, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischofberger, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischoff, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottardis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balk, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">Modulating androgen receptor-driven transcription in prostate cancer with selective CDK9 inhibitors</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.chembiol.2020.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=33086052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFGrt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2021&pages=134-147&author=A.+Richtersauthor=S.+K.+Doyleauthor=D.+B.+Freemanauthor=C.+Leeauthor=B.+S.+Leiferauthor=S.+Jagannathanauthor=F.+Kabingerauthor=J.+V.+Korenauthor=N.+B.+Struntzauthor=J.+Urgilesauthor=R.+A.+Staggauthor=B.+H.+Curtinauthor=D.+Chatterjeeauthor=S.+Matheaauthor=P.+J.+Mikochikauthor=T.+D.+Hopkinsauthor=H.+Gaoauthor=J.+R.+Branchauthor=H.+Xinauthor=L.+Westoverauthor=G.+C.+Bignanauthor=B.+A.+Rupnowauthor=K.+L.+Karlinauthor=C.+M.+Olsonauthor=T.+F.+Westbrookauthor=J.+Vaccaauthor=C.+M.+Wilfongauthor=B.+W.+Trotterauthor=D.+C.+Saffranauthor=N.+Bischofbergerauthor=S.+Knappauthor=J.+W.+Russoauthor=I.+Hicksonauthor=J.+R.+Bischoffauthor=M.+M.+Gottardisauthor=S.+P.+Balkauthor=C.+Y.+Linauthor=M.+S.+Popauthor=A.+N.+Koehler&title=Modulating+androgen+receptor-driven+transcription+in+prostate+cancer+with+selective+CDK9+inhibitors&doi=10.1016%2Fj.chembiol.2020.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating Androgen Receptor-Driven Transcription in Prostate Cancer with Selective CDK9 Inhibitors</span></div><div class="casAuthors">Richters, Andre; Doyle, Shelby K.; Freeman, David B.; Lee, Christina; Leifer, Becky S.; Jagannathan, Sajjeev; Kabinger, Florian; Koren, Jost Vrabic; Struntz, Nicholas B.; Urgiles, Julie; Stagg, Ryan A.; Curtin, Brice H.; Chatterjee, Deep; Mathea, Sebastian; Mikochik, Peter J.; Hopkins, Tamara D.; Gao, Hua; Branch, Jonathan; Xin, Hong; Westover, Lori; Bignan, Gilles C.; Rupnow, Brent A.; Karlin, Kristen L.; Olson, Calla M.; Westbrook, Thomas F.; Vacca, Joseph; Wilfong, Chris M.; Trotter, B. Wesley; Saffran, Douglas C.; Bischofberger, Norbert; Knapp, Stefan; Russo, Joshua W.; Hickson, Ian; Bischoff, James R.; Gottardis, Marco M.; Balk, Steven P.; Lin, Charles Y.; Pop, Marius S.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-147.e14</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Castration-resistant prostate cancers (CRPCs) lose sensitivity to androgen-deprivation therapies but frequently remain dependent on oncogenic transcription driven by the androgen receptor (AR) and its splice variants.  To discover modulators of AR-variant activity, we used a lysate-based small-mol. microarray assay and identified KI-ARv-03 as an AR-variant complex binder that reduces AR-driven transcription and proliferation in prostate cancer cells.  We deduced KI-ARv-03 to be a potent, selective inhibitor of CDK9, an important cofactor for AR, MYC, and other oncogenic transcription factors.  Further optimization resulted in KB-0742, an orally bioavailable, selective CDK9 inhibitor with potent anti-tumor activity in CRPC models.  In 22Rv1 cells, KB-0742 rapidly downregulates nascent transcription, preferentially depleting short half-life transcripts and AR-driven oncogenic programs.  In vivo, oral administration of KB-0742 significantly reduced tumor growth in CRPC, supporting CDK9 inhibition as a promising therapeutic strategy to target AR dependence in CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2omjR3_R4bVg90H21EOLACvtfcHk0lhGSwO4GGGkXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFGrt7vL&md5=e117c22d5cf06feba7d9ff0235b3860e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DDoyle%26aufirst%3DS.%2BK.%26aulast%3DFreeman%26aufirst%3DD.%2BB.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DLeifer%26aufirst%3DB.%2BS.%26aulast%3DJagannathan%26aufirst%3DS.%26aulast%3DKabinger%26aufirst%3DF.%26aulast%3DKoren%26aufirst%3DJ.%2BV.%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DUrgiles%26aufirst%3DJ.%26aulast%3DStagg%26aufirst%3DR.%2BA.%26aulast%3DCurtin%26aufirst%3DB.%2BH.%26aulast%3DChatterjee%26aufirst%3DD.%26aulast%3DMathea%26aufirst%3DS.%26aulast%3DMikochik%26aufirst%3DP.%2BJ.%26aulast%3DHopkins%26aufirst%3DT.%2BD.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DBranch%26aufirst%3DJ.%2BR.%26aulast%3DXin%26aufirst%3DH.%26aulast%3DWestover%26aufirst%3DL.%26aulast%3DBignan%26aufirst%3DG.%2BC.%26aulast%3DRupnow%26aufirst%3DB.%2BA.%26aulast%3DKarlin%26aufirst%3DK.%2BL.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DWestbrook%26aufirst%3DT.%2BF.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DWilfong%26aufirst%3DC.%2BM.%26aulast%3DTrotter%26aufirst%3DB.%2BW.%26aulast%3DSaffran%26aufirst%3DD.%2BC.%26aulast%3DBischofberger%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DJ.%2BW.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26aulast%3DGottardis%26aufirst%3DM.%2BM.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DPop%26aufirst%3DM.%2BS.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DModulating%2520androgen%2520receptor-driven%2520transcription%2520in%2520prostate%2520cancer%2520with%2520selective%2520CDK9%2520inhibitors%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D134%26epage%3D147%26doi%3D10.1016%2Fj.chembiol.2020.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosemund, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terebesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prelle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, F. v.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span> <span> </span><span class="NLM_article-title">Identification of potent and highly selective PTEFb inhibitor BAY 1251152 for the treatment of cancer: from p.o. to i.v. application via scaffold hops</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">984</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1538-7445.AM2017-984" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=984&author=U.+T.+Lueckingauthor=A.+Scholzauthor=D.+Kosemundauthor=R.+Bohlmannauthor=H.+Briemauthor=P.+Lienauauthor=G.+Siemeisterauthor=I.+Terebesiauthor=K.+Meyerauthor=K.+Prelleauthor=R.+Valenciaauthor=S.+Inceauthor=F.+v.+Nussbaumauthor=D.+Mumbergauthor=K.+Ziegelbauerauthor=M.+Brands&title=Identification+of+potent+and+highly+selective+PTEFb+inhibitor+BAY+1251152+for+the+treatment+of+cancer%3A+from+p.o.+to+i.v.+application+via+scaffold+hops&doi=10.1158%2F1538-7445.AM2017-984"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-984%26sid%3Dliteratum%253Aachs%26aulast%3DLuecking%26aufirst%3DU.%2BT.%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DKosemund%26aufirst%3DD.%26aulast%3DBohlmann%26aufirst%3DR.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DTerebesi%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DK.%26aulast%3DPrelle%26aufirst%3DK.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DInce%26aufirst%3DS.%26aulast%3DNussbaum%26aufirst%3DF.%2Bv.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DZiegelbauer%26aufirst%3DK.%26aulast%3DBrands%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520potent%2520and%2520highly%2520selective%2520PTEFb%2520inhibitor%2520BAY%25201251152%2520for%2520the%2520treatment%2520of%2520cancer%253A%2520from%2520p.o.%2520to%2520i.v.%2520application%2520via%2520scaffold%2520hops%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D984%26doi%3D10.1158%2F1538-7445.AM2017-984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Bayer
AG</span>. <span> </span><span class="NLM_article-title">An open-label Phase I dose-escalation
study to characterize
the safety, tolerability, pharmacokinetics, and maximum tolerated
dose of BAY 1143572 given in a once-daily or an intermittent dosing
schedule in subjects with advanced malignancies; Clinical Trial Results
Synopsis, Study no. 16519 (NCT01938638)</span>, October 9,
2017. <a href="https://clinicaltrials.bayer.com/study/?id=16519" class="extLink">https://clinicaltrials.bayer.com/study/?id=16519</a> (accessed
September 4, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bayer%0AAG.+An+open-label+Phase+I+dose-escalation%0Astudy+to+characterize%0Athe+safety%2C+tolerability%2C+pharmacokinetics%2C+and+maximum+tolerated%0Adose+of+BAY+1143572+given+in+a+once-daily+or+an+intermittent+dosing%0Aschedule+in+subjects+with+advanced+malignancies%3B+Clinical+Trial+Results%0ASynopsis%2C+Study+no.+16519+%28NCT01938638%29%2C+October+9%2C%0A2017.+https%3A%2F%2Fclinicaltrials.bayer.com%2Fstudy%2F%3Fid%3D16519+%28accessed%0ASeptember+4%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAn%2520open-label%2520Phase%2520I%2520dose-escalation%250Astudy%2520to%2520characterize%250Athe%2520safety%252C%2520tolerability%252C%2520pharmacokinetics%252C%2520and%2520maximum%2520tolerated%250Adose%2520of%2520BAY%25201143572%2520given%2520in%2520a%2520once-daily%2520or%2520an%2520intermittent%2520dosing%250Aschedule%2520in%2520subjects%2520with%2520advanced%2520malignancies%253B%2520Clinical%2520Trial%2520Results%250ASynopsis%252C%2520Study%2520no.%252016519%2520%2528NCT01938638%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caraher, E.</span></span> <span> </span><span class="NLM_article-title">Mcl-1 is vital for neutrophil survival</span>. <i>Immunol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1007/s12026-015-8655-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1007%2Fs12026-015-8655-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25929430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFChu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=225-233&author=M.+P.+Murphyauthor=E.+Caraher&title=Mcl-1+is+vital+for+neutrophil+survival&doi=10.1007%2Fs12026-015-8655-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mcl-1 is vital for neutrophil survival</span></div><div class="casAuthors">Murphy, Mark P.; Caraher, Emma</div><div class="citationInfo"><span class="NLM_cas:title">Immunologic Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">IMRSEB</span>;
        ISSN:<span class="NLM_cas:issn">0257-277X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Upon entry to the systemic circulation, neutrophils exhibit a short mean time to cell death.  The viability of most cell types in a steady state is preserved by the interplay of the Bcl-2 family of proteins, wherein the anti-apoptotic members inhibit the action of their pro-apoptotic counterparts.  Neutrophils, however, display absent or severely reduced expression of several anti-apoptotic Bcl-2 family proteins.  Hence, they rely on the expression of Mcl-1, an anti-apoptotic member of the Bcl-2 family, for survival.  This protein is uniquely short-lived relative to related proteins and its loss likely ppts. the induction of apoptosis in neutrophils.  This review describes the role of Mcl-1 in the neutrophil in the context of apoptosis and highlights the proteins' importance to the cell.  We also address neutrophil apoptosis in the broader context of the cells' response to pathogens, focussing particularly on the strategies used by pathogens to manipulate the apoptotic pathway to their own ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBYHR4tH7JybVg90H21EOLACvtfcHk0lhLCUnA60LBAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFChu7c%253D&md5=83f9c2d5c842b353b4642ebbc9e5333f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs12026-015-8655-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12026-015-8655-z%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DM.%2BP.%26aulast%3DCaraher%26aufirst%3DE.%26atitle%3DMcl-1%2520is%2520vital%2520for%2520neutrophil%2520survival%26jtitle%3DImmunol.%2520Res.%26date%3D2015%26volume%3D62%26spage%3D225%26epage%3D233%26doi%3D10.1007%2Fs12026-015-8655-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span> <span> </span><span class="NLM_article-title">Cyclin dependent kinase 9 inhibitors for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8667</span>– <span class="NLM_lpage">8684</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00150</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00150" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslGjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8667-8684&author=Y.+A.+Sonawaneauthor=M.+A.+Taylorauthor=J.+V.+Napoleonauthor=S.+Ranaauthor=J.+I.+Contrerasauthor=A.+Natarajan&title=Cyclin+dependent+kinase+9+inhibitors+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.6b00150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin Dependent Kinase 9 Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Sonawane, Yogesh A.; Taylor, Margaret A.; Napoleon, John Victor; Rana, Sandeep; Contreras, Jacob I.; Natarajan, Amarnath</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8667-8684</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin dependent kinase (CDK) inhibitors have been the topic of intense research for nearly 2 decades due to their widely varied and crit. functions within the cell.  Recently CDK9 has emerged as a druggable target for the development of cancer therapeutics.  CDK9 plays a crucial role in transcription regulation; specifically, CDK9 mediated transcriptional regulation of short-lived antiapoptotic proteins is crit. for the survival of transformed cells.  Focused chem. libraries based on a plethora of scaffolds have resulted in mixed success with regard to the development of selective CDK9 inhibitors.  Here we review the regulation of CDK9, its cellular functions, and common core structures used to target CDK9, along with their selectivity profile and efficacy in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIqU9uSujdrVg90H21EOLACvtfcHk0lhLCUnA60LBAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslGjsLo%253D&md5=0910a61f9c915002ff1073b22b5e20d6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00150%26sid%3Dliteratum%253Aachs%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DNapoleon%26aufirst%3DJ.%2BV.%26aulast%3DRana%26aufirst%3DS.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DNatarajan%26aufirst%3DA.%26atitle%3DCyclin%2520dependent%2520kinase%25209%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8667%26epage%3D8684%26doi%3D10.1021%2Facs.jmedchem.6b00150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvanitis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsher, D. W.</span></span> <span> </span><span class="NLM_article-title">Sustained loss of a neoplastic phenotype by brief inactivation of MYC</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1126/science.1071489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1126%2Fscience.1071489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=12098700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFGqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=102-104&author=M.+Jainauthor=C.+Arvanitisauthor=K.+Chuauthor=W.+Deweyauthor=E.+Leonhardtauthor=M.+Trinhauthor=C.+D.+Sundbergauthor=J.+M.+Bishopauthor=D.+W.+Felsher&title=Sustained+loss+of+a+neoplastic+phenotype+by+brief+inactivation+of+MYC&doi=10.1126%2Fscience.1071489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained loss of a neoplastic phenotype by brief inactivation of MYC</span></div><div class="casAuthors">Jain, Meenakshi; Arvanitis, Constadina; Chu, Kenneth; Dewey, William; Leonhardt, Edith; Trinh, Maxine; Sundberg, Christopher D.; Bishop, J. Michael; Felsher, Dean W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5578</span>),
    <span class="NLM_cas:pages">102-104</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. inactivation of oncogenes is being investigated as a possible therapeutic strategy for cancer.  One potential drawback is that cessation of such therapy may allow reactivation of the oncogene and tumor regrowth.  We used a conditional transgenic mouse model for MYC-induced tumorigenesis to demonstrate that brief inactivation of MYC results in the sustained regression of tumors and the differentiation of osteogenic sarcoma cells into mature osteocytes.  Subsequent reactivation of MYC did not restore the cells' malignant properties but instead induced apoptosis.  Thus, brief MYC inactivation appears to cause epigenetic changes in tumor cells that render them insensitive to MYC-induced tumorigenesis.  These results raise the possibility that transient inactivation of MYC may be an effective therapy for certain cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGKf2LqctHbVg90H21EOLACvtfcHk0lhLCUnA60LBAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFGqsr4%253D&md5=6082c792da67fea09d498eebc9cf8eae</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.1071489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1071489%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DM.%26aulast%3DArvanitis%26aufirst%3DC.%26aulast%3DChu%26aufirst%3DK.%26aulast%3DDewey%26aufirst%3DW.%26aulast%3DLeonhardt%26aufirst%3DE.%26aulast%3DTrinh%26aufirst%3DM.%26aulast%3DSundberg%26aufirst%3DC.%2BD.%26aulast%3DBishop%26aufirst%3DJ.%2BM.%26aulast%3DFelsher%26aufirst%3DD.%2BW.%26atitle%3DSustained%2520loss%2520of%2520a%2520neoplastic%2520phenotype%2520by%2520brief%2520inactivation%2520of%2520MYC%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D102%26epage%3D104%26doi%3D10.1126%2Fscience.1071489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbrink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuijen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijnen, J. H.</span></span> <span> </span><span class="NLM_article-title">Variability in bioavailability of small molecular tyrosine kinase inhibitors</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2015.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.ctrv.2015.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=25818541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFentrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2015&pages=412-422&author=M.+Herbrinkauthor=B.+Nuijenauthor=J.+H.+M.+Schellensauthor=J.+H.+Beijnen&title=Variability+in+bioavailability+of+small+molecular+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.ctrv.2015.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Variability in bioavailability of small molecular tyrosine kinase inhibitors</span></div><div class="casAuthors">Herbrink, Maikel; Nuijen, Bastiaan; Schellens, Jan H. M.; Beijnen, Jos H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">412-422</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small mol. tyrosine kinase inhibitors (smTKIs) are in the center of the very quickly expanding area of personalized chemotherapy and oral applicability thereof.  The no. of drugs in this class is rapidly growing, with twenty current approvals by both the European Medicines Agency (EMA) and the Food and Drug Administration (FDA).  The drugs are, however, generally characterized by a poor oral, and thus variable, bioavailability.  This results in significant variation in plasma levels and exposure.  The cause is a complex interplay of factors, including poor aq. soly., issued permeability, membrane transport and enzymic metab.  Addnl., food and drug-drug interactions can play a significant role.  The issues related with an impaired bioavailability generally receive little attention.  To the best of our knowledge, this article is the first to provide an overview of the factors that det. the bioavailability of the smTKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu0wd0bHjmSbVg90H21EOLACvtfcHk0lhLCUnA60LBAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFentrw%253D&md5=50fc8e53b9b0f8f4bbe16abe0e320e2d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2015.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2015.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DHerbrink%26aufirst%3DM.%26aulast%3DNuijen%26aufirst%3DB.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26atitle%3DVariability%2520in%2520bioavailability%2520of%2520small%2520molecular%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2015%26volume%3D41%26spage%3D412%26epage%3D422%26doi%3D10.1016%2Fj.ctrv.2015.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Sulfoximines: a neglected opportunity in medicinal chemistry</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">9399</span>– <span class="NLM_lpage">9408</span>, <span class="refDoi"> DOI: 10.1002/anie.201302209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fanie.201302209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CrsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=9399-9408&author=U.+L%C3%BCcking&title=Sulfoximines%3A+a+neglected+opportunity+in+medicinal+chemistry&doi=10.1002%2Fanie.201302209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfoximines: A Neglected Opportunity in Medicinal Chemistry</span></div><div class="casAuthors">Luecking, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">9399-9408</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Innovation has frequently been described as the key to drug discovery.  However, in the daily routine, medicinal chemists often tend to stick to the functional groups and structural elements they know and love.  Blockbuster cancer drug Velcade (bortezomib), for example, was rejected by more than 50 companies, supposedly because of its unusual boronic acid function (as often repeated: "only a moron would put boron in a drug!").  Similarly, in the discovery process of the pan-CDK inhibitor BAY 1000394, the unconventional proposal to introduce a sulfoximine group into the lead series also led to sneers and raised eyebrows, since sulfoximines have seldom been used in medicinal chem.  However, it was the introduction of the sulfoximine group that finally allowed the fundamental issues of the project to be overcome, culminating in the identification of the clin. sulfoximine pan-CDK inhibitor BAY 1000394.  This Minireview provides an overview of a widely neglected opportunity in medicinal chem.-the sulfoximine group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopcpeepAANfbVg90H21EOLACvtfcHk0lgY4djCUSUBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CrsrvI&md5=6850d8d77b1141d9d9344410822c00c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201302209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201302209%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DSulfoximines%253A%2520a%2520neglected%2520opportunity%2520in%2520medicinal%2520chemistry%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D9399%26epage%3D9408%26doi%3D10.1002%2Fanie.201302209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Neglected sulfur(VI) pharmacophores in drug discovery: exploration of novel chemical space by the interplay of drug design and method development</span>. <i>Org. Chem. Front.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1319</span>– <span class="NLM_lpage">1324</span>, <span class="refDoi"> DOI: 10.1039/c8qo01233d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1039%2FC8QO01233D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslemtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=1319-1324&author=U.+L%C3%BCcking&title=Neglected+sulfur%28VI%29+pharmacophores+in+drug+discovery%3A+exploration+of+novel+chemical+space+by+the+interplay+of+drug+design+and+method+development&doi=10.1039%2Fc8qo01233d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Neglected sulfur(VI) pharmacophores in drug discovery: exploration of novel chemical space by the interplay of drug design and method development</span></div><div class="casAuthors">Luecking, U.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Chemistry Frontiers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1319-1324</span>CODEN:
                <span class="NLM_cas:coden">OCFRA8</span>;
        ISSN:<span class="NLM_cas:issn">2052-4129</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Historically, sulfoximines, sulfondiimines and sulfonimidamides have been neglected pharmacophores in drug discovery even though they offer very interesting properties.  This highlight shares the key learnings of various lead optimization approaches at Bayer AG that have successfully utilized these neglected sulfur(VI) functional groups to deliver multiple clin. candidates.  In this context, the key synthetic methods utilized for the synthetic prepn. of these unusual compds. will be outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcX6XF4X_NJbVg90H21EOLACvtfcHk0lgY4djCUSUBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslemtL8%253D&md5=eae9b5efca8d68f3d8ab46e8fe29be99</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC8QO01233D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8QO01233D%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DNeglected%2520sulfur%2528VI%2529%2520pharmacophores%2520in%2520drug%2520discovery%253A%2520exploration%2520of%2520novel%2520chemical%2520space%2520by%2520the%2520interplay%2520of%2520drug%2520design%2520and%2520method%2520development%26jtitle%3DOrg.%2520Chem.%2520Front.%26date%3D2019%26volume%3D6%26spage%3D1319%26epage%3D1324%26doi%3D10.1039%2Fc8qo01233d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frings, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnamm, C.</span></span> <span> </span><span class="NLM_article-title">Sulfoximines from a medicinal chemist’s perspective: physicochemical and in vitro parameters relevant for drug discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.09.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.ejmech.2016.09.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27821325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVahsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2017&pages=225-245&author=M.+Fringsauthor=C.+Bolmauthor=A.+Blumauthor=C.+Gnamm&title=Sulfoximines+from+a+medicinal+chemist%E2%80%99s+perspective%3A+physicochemical+and+in+vitro+parameters+relevant+for+drug+discovery&doi=10.1016%2Fj.ejmech.2016.09.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfoximines from a Medicinal Chemist's Perspective: Physicochemical and in vitro Parameters Relevant for Drug Discovery</span></div><div class="casAuthors">Frings, Marcus; Bolm, Carsten; Blum, Andreas; Gnamm, Christian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">225-245</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Sulfoximines, sulfondiimides and sulfonimidamides are fascinating but not yet fully explored variants of the common sulfone or sulfonamide motif.  In this study, we report the physicochem. and in vitro properties of sulfoximines and compare them with related analogs and isosteres.  Furthermore, we present a matched mol. pair anal. of compds. from drug discovery projects within Boehringer Ingelheim.  We demonstrate that the sulfoximine moiety is a chem. stable, comparatively polar and weakly basic functional group, often leading to favorable aq. soly., permeability and metabolic stability.  Moreover, their addnl. vectors at nitrogen enable simple chem. modifications and thus facilitate exploration and fine-tuning of the mol. properties.  We conclude that sulfoximines and their congeners do not exhibit any intrinsic flaw but significantly enrich the toolbox of medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu0FlXXURKybVg90H21EOLACvtfcHk0lgY4djCUSUBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVahsbfK&md5=8a7939f22425f46ad9eb87fcddd29099</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.09.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.09.091%26sid%3Dliteratum%253Aachs%26aulast%3DFrings%26aufirst%3DM.%26aulast%3DBolm%26aufirst%3DC.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DGnamm%26aufirst%3DC.%26atitle%3DSulfoximines%2520from%2520a%2520medicinal%2520chemist%25E2%2580%2599s%2520perspective%253A%2520physicochemical%2520and%2520in%2520vitro%2520parameters%2520relevant%2520for%2520drug%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D126%26spage%3D225%26epage%3D245%26doi%3D10.1016%2Fj.ejmech.2016.09.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirvent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span></span> <span> </span><span class="NLM_article-title">Novel pieces for the emerging picture of sulfoximines in drug discovery: synthesis and evaluation of sulfoximine analogues of marketed drugs and advanced clinical candidates</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fcmdc.201700044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=28221724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1Sltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=487-501&author=J.+A.+Sirventauthor=U.+L%C3%BCcking&title=Novel+pieces+for+the+emerging+picture+of+sulfoximines+in+drug+discovery%3A+synthesis+and+evaluation+of+sulfoximine+analogues+of+marketed+drugs+and+advanced+clinical+candidates&doi=10.1002%2Fcmdc.201700044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Pieces for the Emerging Picture of Sulfoximines in Drug Discovery: Synthesis and Evaluation of Sulfoximine Analogues of Marketed Drugs and Advanced Clinical Candidates</span></div><div class="casAuthors">Sirvent, Juan Alberto; Luecking, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">487-501</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Sulfoximines have gained considerable recognition as an important structural motif in drug discovery of late.  In particular, the clin. kinase inhibitors for the treatment of cancer, roniciclib (pan-CDK inhibitor), BAY 1143572 (P-TEFb inhibitor), and AZD 6738 (ATR inhibitor), have recently drawn considerable attention.  While the interest in this underrepresented functional group in drug discovery is clearly on the rise, there remains an incomplete understanding of the medicinal-chem.-relevant properties of sulfoximines.  Herein we report the synthesis and in vitro characterization of a variety of sulfoximine analogs of marketed drugs and advanced clin. candidates to gain a better understanding of this neglected functional group and its potential in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-5kfVM3I45LVg90H21EOLACvtfcHk0li0x0If6zf7dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1Sltbo%253D&md5=5b4589d7af32c8984d4fd3027c31c168</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700044%26sid%3Dliteratum%253Aachs%26aulast%3DSirvent%26aufirst%3DJ.%2BA.%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DNovel%2520pieces%2520for%2520the%2520emerging%2520picture%2520of%2520sulfoximines%2520in%2520drug%2520discovery%253A%2520synthesis%2520and%2520evaluation%2520of%2520sulfoximine%2520analogues%2520of%2520marketed%2520drugs%2520and%2520advanced%2520clinical%2520candidates%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D487%26epage%3D501%26doi%3D10.1002%2Fcmdc.201700044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="note"><p class="first last">BAY-332 is a single stereoisomer but the absolute configuration at the sulfoximine sulfur was not determined.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="note"><p class="first last">For additional examples, see:</p></div><div class="NLM_citation" id="rightTab-cit31a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span>; <span class="NLM_string-name">Kosemund, D.</span>; <span class="NLM_string-name">Scholz, A.</span>; <span class="NLM_string-name">Lienau, P.</span>; <span class="NLM_string-name">Siemeister, G.</span>; <span class="NLM_string-name">Bömer, U.</span>; <span class="NLM_string-name">Bohlmann, R.</span></span> <span> </span><span class="NLM_article-title">Disubstituted 5-fluoro pyrimidine derivatives containing a sulfoximine group</span>. <span class="NLM_patent">WO 2013037894 A1</span>, March 21, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=U.+L%C3%BCcking&author=D.+Kosemund&author=A.+Scholz&author=P.+Lienau&author=G.+Siemeister&author=U.+B%C3%B6mer&author=R.+Bohlmann&title=Disubstituted+5-fluoro+pyrimidine+derivatives+containing+a+sulfoximine+group"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DDisubstituted%25205-fluoro%2520pyrimidine%2520derivatives%2520containing%2520a%2520sulfoximine%2520group%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Lücking, U.</span>; <span class="NLM_string-name">Kosemund, D.</span>; <span class="NLM_string-name">Scholz, A.</span>; <span class="NLM_string-name">Lienau, P.</span>; <span class="NLM_string-name">Siemeister, G.</span>; <span class="NLM_string-name">Bömer, U.</span>; <span class="NLM_string-name">Bohlmann, R.</span></span> <span> </span><span class="NLM_article-title">Disubstituted 5-fluoro-pyrimidines</span>. <span class="NLM_patent">WO 2013037896 A1</span>, March 21, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=U.+L%C3%BCcking&author=D.+Kosemund&author=A.+Scholz&author=P.+Lienau&author=G.+Siemeister&author=U.+B%C3%B6mer&author=R.+Bohlmann&title=Disubstituted+5-fluoro-pyrimidines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BCcking%26aufirst%3DU.%26atitle%3DDisubstituted%25205-fluoro-pyrimidines%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span>For the
assay description, see:<span class="NLM_contrib-group"><span class="NLM_string-name">Onofrey, T.</span>; <span class="NLM_string-name">Kazan, G.</span></span> <i>Performance and Correlation of a 96-Well High Throughput Screening Method to Determine Aqueous Drug Solubility</i>; <span class="NLM_publisher-name">Millipore Corporation</span>: <span class="NLM_publisher-loc">Billerica, MA, USA</span>, <span class="NLM_year">2003</span>; Application Note Lit. No. AN1731EN00.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=T.+Onofrey&author=G.+Kazan&title=Performance+and+Correlation+of+a+96-Well+High+Throughput+Screening+Method+to+Determine+Aqueous+Drug+Solubility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3DOnofrey%26aufirst%3DT.%26jtitle%3DPerformance%2520and%2520Correlation%2520of%2520a%252096-Well%2520High%2520Throughput%2520Screening%2520Method%2520to%2520Determine%2520Aqueous%2520Drug%2520Solubility%26pub%3DMillipore%2520Corporation%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantifying CDK inhibitor selectivity in live cells</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2743</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-16559-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1038%2Fs41467-020-16559-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=32488087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtV2kt7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2743&author=C.+I.+Wellsauthor=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=M.+R.+Ingoldauthor=J.+E.+Pickettauthor=D.+H.+Drewryauthor=K.+M.+Pughauthor=M.+K.+Schwinnauthor=B.+Hwangauthor=H.+Zegzoutiauthor=K.+V.+M.+Huberauthor=M.+Congauthor=P.+L.+Meisenheimerauthor=T.+M.+Willsonauthor=M.+B.+Robers&title=Quantifying+CDK+inhibitor+selectivity+in+live+cells&doi=10.1038%2Fs41467-020-16559-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying CDK inhibitor selectivity in live cells</span></div><div class="casAuthors">Wells, Carrow I.; Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Ingold, Morgan R.; Pickett, Julie E.; Drewry, David H.; Pugh, Kathryn M.; Schwinn, Marie K.; Hwang, Byounghoon; Zegzouti, Hicham; Huber, Kilian V. M.; Cong, Mei; Meisenheimer, Poncho L.; Willson, Timothy M.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2743</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Concerted multidisciplinary efforts have led to the development of Cyclin-Dependent Kinase inhibitors (CDKi's) as small mol. drugs and chem. probes of intracellular CDK function.  However, conflicting data has been reported on the inhibitory potency of CDKi's and a systematic characterization of affinity and selectivity against intracellular CDKs is lacking.  We have developed a panel of cell-permeable energy transfer probes to quantify target occupancy for all 21 human CDKs in live cells, and present a comprehensive evaluation of intracellular isoenzyme potency and selectivity for a collection of 46 clin.-advanced CDKi's and tool mols.  We obsd. unexpected intracellular activity profiles for a no. of CDKi's, offering avenues for repurposing of highly potent mols. as probes for previously unreported targets.  Overall, we provide a broadly applicable method for evaluating the selectivity of CDK inhibitors in living cells, and present a refined set of tool mols. to study CDK function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCCek7iI4hSbVg90H21EOLACvtfcHk0li0x0If6zf7dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtV2kt7bJ&md5=16101d5b781e62fb093864a2303f7d2e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16559-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16559-0%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DPickett%26aufirst%3DJ.%2BE.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DPugh%26aufirst%3DK.%2BM.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DHwang%26aufirst%3DB.%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantifying%2520CDK%2520inhibitor%2520selectivity%2520in%2520live%2520cells%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D2743%26doi%3D10.1038%2Fs41467-020-16559-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roethlisberger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahler, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altenburger, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappenberger, A.</span></span> <span> </span><span class="NLM_article-title">If euhydric and isotonic do not work, what are acceptable pH and osmolality for parenteral drug dosage forms?</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1016/j.xphs.2016.09.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.xphs.2016.09.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=27889072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2017&pages=446-456&author=D.+Roethlisbergerauthor=H.-C.+Mahlerauthor=U.+Altenburgerauthor=A.+Pappenberger&title=If+euhydric+and+isotonic+do+not+work%2C+what+are+acceptable+pH+and+osmolality+for+parenteral+drug+dosage+forms%3F&doi=10.1016%2Fj.xphs.2016.09.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">If Euhydric and Isotonic Do Not Work, What Are Acceptable pH and Osmolality for Parenteral Drug Dosage Forms?</span></div><div class="casAuthors">Roethlisberger, Dieter; Mahler, Hanns-Christian; Altenburger, Ulrike; Pappenberger, Astrid</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">446-456</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Parenteral products should aim toward being isotonic and euhydric (physiol. pH).  Yet, due to other considerations, this goal is often not reasonable or doable.  There are no clear allowable ranges related to pH and osmolality, and thus, the objective of this review was to provide a better understanding of acceptable formulation pH, buffer strength, and osmolality taking into account the administration route (i.e., i.m., i.v., s.c.) and administration technique (i.e., bolus, push, infusion).  This evaluation was based on 3 different approaches: conventional, exptl., and parametric.  The conventional way of defining formulation limits was based on std. pH and osmolality ranges.  Exptl. detn. of titratable acidity or in vitro hemolysis testing provided addnl. drug product information.  Finally, the parametric approach was based on the calcn. of theor. values such as (1) the maximal vol. of injection which cannot shift the blood's pH or its molarity out of the physiol. range and (b) a diln. ratio at the injection site and by verifying that threshold values are not exceeded.  The combination of all 3 approaches can support the definition of acceptable pH, buffer strength, and osmolality of formulations and thus may reduce the risk of failure during preclin. and clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocRU3wZe7uALVg90H21EOLACvtfcHk0li0x0If6zf7dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7bP&md5=4fde5a41732655312a086b10df403c21</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.xphs.2016.09.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.xphs.2016.09.034%26sid%3Dliteratum%253Aachs%26aulast%3DRoethlisberger%26aufirst%3DD.%26aulast%3DMahler%26aufirst%3DH.-C.%26aulast%3DAltenburger%26aufirst%3DU.%26aulast%3DPappenberger%26aufirst%3DA.%26atitle%3DIf%2520euhydric%2520and%2520isotonic%2520do%2520not%2520work%252C%2520what%2520are%2520acceptable%2520pH%2520and%2520osmolality%2520for%2520parenteral%2520drug%2520dosage%2520forms%253F%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D106%26spage%3D446%26epage%3D456%26doi%3D10.1016%2Fj.xphs.2016.09.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gries, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, J.</span></span> <span> </span><span class="NLM_article-title">A practical approach to N-(trifluoroacetyl)sulfilimines</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1834</span>, <span class="refDoi"> DOI: 10.1055/s-0034-1378328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1055%2Fs-0034-1378328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=1831-1834&author=J.+Griesauthor=J.+Kr%C3%BCger&title=A+practical+approach+to+N-%28trifluoroacetyl%29sulfilimines&doi=10.1055%2Fs-0034-1378328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A practical approach to N-(trifluoroacetyl)sulfilimines</span></div><div class="casAuthors">Gries, Joerg; Krueger, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">Synlett</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1831-1834</span>CODEN:
                <span class="NLM_cas:coden">SYNLES</span>;
        ISSN:<span class="NLM_cas:issn">0936-5214</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A simple and one-pot transition metal-free synthesis of N-(trifluoroacetyl)-protected sulfilimines I [R1 = 4-O2NC6H4, 4-H3CC6H4, 4-ClC6H4, etc.; R2 = CH3, cyclopropyl] via reaction of sulfides and trifluoroacetamide is described.  The salient features includes the use of safe, low-cost, com. available reagents, mild reaction conditions and a broad substrate scope.  Resoln. of sulfoximine racemate via diastereoselective salt formation is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6rGp6lF6RGLVg90H21EOLACvtfcHk0lj5Z2ViGK67LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjsLfK&md5=48d236271e0a87874837b36233d1efa2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1055%2Fs-0034-1378328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0034-1378328%26sid%3Dliteratum%253Aachs%26aulast%3DGries%26aufirst%3DJ.%26aulast%3DKr%25C3%25BCger%26aufirst%3DJ.%26atitle%3DA%2520practical%2520approach%2520to%2520N-%2528trifluoroacetyl%2529sulfilimines%26jtitle%3DSynlett%26date%3D2014%26volume%3D25%26spage%3D1831%26epage%3D1834%26doi%3D10.1055%2Fs-0034-1378328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellerich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellinghaus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mumberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serve, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelbauer, K.</span></span> <span> </span><span class="NLM_article-title">BAY 1143572, a first-in-class, highly selective, potent and orally available inhibitor of PTEFb/CDK9 currently in Phase I, shows convincing anti-tumor activity in preclinical models of acute myeloid leukemia (AML)</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3022</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2016-3022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1158%2F1538-7445.AM2016-3022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3022&author=A.+Scholzauthor=T.+Oellerichauthor=A.+Hussainauthor=S.+Lindnerauthor=U.+Lueckingauthor=A.+O.+Walterauthor=P.+Ellinghausauthor=R.+Valenciaauthor=F.+von+Nussbaumauthor=D.+Mumbergauthor=M.+Brandsauthor=S.+Inceauthor=H.+Serveauthor=K.+Ziegelbauer&title=BAY+1143572%2C+a+first-in-class%2C+highly+selective%2C+potent+and+orally+available+inhibitor+of+PTEFb%2FCDK9+currently+in+Phase+I%2C+shows+convincing+anti-tumor+activity+in+preclinical+models+of+acute+myeloid+leukemia+%28AML%29&doi=10.1158%2F1538-7445.AM2016-3022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2016-3022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2016-3022%26sid%3Dliteratum%253Aachs%26aulast%3DScholz%26aufirst%3DA.%26aulast%3DOellerich%26aufirst%3DT.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DLindner%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEllinghaus%26aufirst%3DP.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DMumberg%26aufirst%3DD.%26aulast%3DBrands%26aufirst%3DM.%26aulast%3DInce%26aufirst%3DS.%26aulast%3DServe%26aufirst%3DH.%26aulast%3DZiegelbauer%26aufirst%3DK.%26atitle%3DBAY%25201143572%252C%2520a%2520first-in-class%252C%2520highly%2520selective%252C%2520potent%2520and%2520orally%2520available%2520inhibitor%2520of%2520PTEFb%252FCDK9%2520currently%2520in%2520Phase%2520I%252C%2520shows%2520convincing%2520anti-tumor%2520activity%2520in%2520preclinical%2520models%2520of%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3022%26doi%3D10.1158%2F1538-7445.AM2016-3022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordoba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ince, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lettieri, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boni, V.</span></span> <span> </span><span class="NLM_article-title">Phase I dose escalation study of the first-in-class selective PTEFb inhibitor BAY 1251152 in patients with advanced cancer: Novel target validation and early evidence of clinical activity</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2507</span>, <span class="refDoi"> DOI: 10.1200/jco.2018.36.15_suppl.2507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1200%2FJCO.2018.36.15_suppl.2507" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=2507&author=J.+R.+Diamondauthor=V.+Morenoauthor=E.+A.+Limauthor=R.+Cordobaauthor=C.+Caiauthor=S.+J.+Inceauthor=J.+T.+Lettieriauthor=C.+Merzauthor=R.+Valenciaauthor=V.+Boni&title=Phase+I+dose+escalation+study+of+the+first-in-class+selective+PTEFb+inhibitor+BAY+1251152+in+patients+with+advanced+cancer%3A+Novel+target+validation+and+early+evidence+of+clinical+activity&doi=10.1200%2Fjco.2018.36.15_suppl.2507"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.2507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.2507%26sid%3Dliteratum%253Aachs%26aulast%3DDiamond%26aufirst%3DJ.%2BR.%26aulast%3DMoreno%26aufirst%3DV.%26aulast%3DLim%26aufirst%3DE.%2BA.%26aulast%3DCordoba%26aufirst%3DR.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DInce%26aufirst%3DS.%2BJ.%26aulast%3DLettieri%26aufirst%3DJ.%2BT.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DValencia%26aufirst%3DR.%26aulast%3DBoni%26aufirst%3DV.%26atitle%3DPhase%2520I%2520dose%2520escalation%2520study%2520of%2520the%2520first-in-class%2520selective%2520PTEFb%2520inhibitor%2520BAY%25201251152%2520in%2520patients%2520with%2520advanced%2520cancer%253A%2520Novel%2520target%2520validation%2520and%2520early%2520evidence%2520of%2520clinical%2520activity%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D2507%26doi%3D10.1200%2Fjco.2018.36.15_suppl.2507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordoba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morillo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdy, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boix, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genvresse, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowakowski, G. S.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy of VIP152, a CDK9 inhibitor, in patients with double-hit lymphoma (DHL)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">7538</span>, <span class="refDoi"> DOI: 10.1200/jco.2021.39.15_suppl.7538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1200%2FJCO.2021.39.15_suppl.7538" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=7538&author=V.+Morenoauthor=R.+Cordobaauthor=D.+Morilloauthor=J.+R.+Diamondauthor=A.+H.+Hamdyauthor=R.+Izumiauthor=C.+Merzauthor=O.+Boixauthor=I.+Genvresseauthor=G.+S.+Nowakowski&title=Safety+and+efficacy+of+VIP152%2C+a+CDK9+inhibitor%2C+in+patients+with+double-hit+lymphoma+%28DHL%29&doi=10.1200%2Fjco.2021.39.15_suppl.7538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1200%2FJCO.2021.39.15_suppl.7538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2021.39.15_suppl.7538%26sid%3Dliteratum%253Aachs%26aulast%3DMoreno%26aufirst%3DV.%26aulast%3DCordoba%26aufirst%3DR.%26aulast%3DMorillo%26aufirst%3DD.%26aulast%3DDiamond%26aufirst%3DJ.%2BR.%26aulast%3DHamdy%26aufirst%3DA.%2BH.%26aulast%3DIzumi%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DC.%26aulast%3DBoix%26aufirst%3DO.%26aulast%3DGenvresse%26aufirst%3DI.%26aulast%3DNowakowski%26aufirst%3DG.%2BS.%26atitle%3DSafety%2520and%2520efficacy%2520of%2520VIP152%252C%2520a%2520CDK9%2520inhibitor%252C%2520in%2520patients%2520with%2520double-hit%2520lymphoma%2520%2528DHL%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2021%26volume%3D39%26spage%3D7538%26doi%3D10.1200%2Fjco.2021.39.15_suppl.7538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kueng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eppenberger, U.</span></span> <span> </span><span class="NLM_article-title">Quantification of cells cultured on 96-well plates</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90710-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2F0003-2697%2889%2990710-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=2604040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A280%3ADyaK3c7gt1yqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=1989&pages=16-19&author=W.+Kuengauthor=E.+Silberauthor=U.+Eppenberger&title=Quantification+of+cells+cultured+on+96-well+plates&doi=10.1016%2F0003-2697%2889%2990710-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of cells cultured on 96-well plates</span></div><div class="casAuthors">Kueng W; Silber E; Eppenberger U</div><div class="citationInfo"><span class="NLM_cas:title">Analytical biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-9</span>
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">The method for cell number measurement in monolayer cultures by crystal violet staining published recently by Gillies et al. (R.  G.  Gillies, N.  Didier, M.  Denton (1986) Anal.  Biochem. 159, 109-113) was modified and significantly improved.  The procedure was adapted for use in 96-well plates since the method is inherently very sensitive.  Modifications allowed fast and complete solubilization of dye adsorbed by cell nuclei during staining.  Since light absorption of the unstained or destained cell layers is negligible, cell number measurements can be performed in the respective wells.  Due to these features, multiple assays may be carried out rapidly using standard 96-well plate readers.  In addition, it is shown that the sensitivity of the assay can be varied and easily controlled by choosing the appropriate pH during the staining procedure.  This increases the flexibility of the method making it useful for determining cell density of a wide range of different cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDlNEnzvDQwoqdXsLcGTEUfW6udTcc2eYTa1WB5wc7lbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3c7gt1yqtg%253D%253D&md5=407acd99c7de5f4ed856249e8f668d6c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990710-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990710-0%26sid%3Dliteratum%253Aachs%26aulast%3DKueng%26aufirst%3DW.%26aulast%3DSilber%26aufirst%3DE.%26aulast%3DEppenberger%26aufirst%3DU.%26atitle%3DQuantification%2520of%2520cells%2520cultured%2520on%252096-well%2520plates%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D182%26spage%3D16%26epage%3D19%26doi%3D10.1016%2F0003-2697%2889%2990710-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowland, M.</span></span> <span> </span><span class="NLM_article-title">Hepatic clearance of drugs. I. Theoretical considerations of a “well-stirred” model and a “parallel tube” model. Influence of hepatic blood flow, plasma and blood cell binding, and the hepatocellular enzymatic activity on hepatic drug clearance</span>. <i>J. Pharmacokinet. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1007/bf01059688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1007%2FBF01059688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=599411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADyaE1cXhtFGjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1977&pages=625-653&author=K.+S.+Pangauthor=M.+Rowland&title=Hepatic+clearance+of+drugs.+I.+Theoretical+considerations+of+a+%E2%80%9Cwell-stirred%E2%80%9D+model+and+a+%E2%80%9Cparallel+tube%E2%80%9D+model.+Influence+of+hepatic+blood+flow%2C+plasma+and+blood+cell+binding%2C+and+the+hepatocellular+enzymatic+activity+on+hepatic+drug+clearance&doi=10.1007%2Fbf01059688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatic clearance of drugs.  I.  Theoretical considerations of a "well-stirred" model and a "parallel tube" model.  Influence of hepatic blood flow, plasma and blood cell binding, and the hepatocellular enzymatic activity on hepatic drug clearance</span></div><div class="casAuthors">Pang, K. Sandy; Rowland, Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacokinetics and Biopharmaceutics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">625-53</span>CODEN:
                <span class="NLM_cas:coden">JPBPBJ</span>;
        ISSN:<span class="NLM_cas:issn">0090-466X</span>.
    </div><div class="casAbstract">Two commonly used models of hepatic drug clearance were examd.  The well-stirred model (model I) views the liver as a well-stirred compartment with concn. of drug in the liver in equil. with that in the emergent blood.  The parallel tube model (model II) regards the liver as a series of parallel tubes with enzymes distributed evenly around the tubes and the concn. of drug declining along the length of the tube.  Both models were examd. under steady-state considerations in the absence of diffusional limitations (i.e., cell membranes not limiting the movement of drug mols.).  Equations involving the determinants of hepatic drug clearance (hepatic blood flow, fraction of drug in blood unbound, and the hepatocellular enzymic activity) and various pharmacokinetic parameters were derived.  Similarities and differences between the models were explored.  Although both models predict similar hepatic drug clearances under a variety of conditions, marked differences between them become apparent in their predictions of the influence of changes in the determinants of drug clearance on various pharmacokinetic parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDpZaENZqfCbVg90H21EOLACvtfcHk0liMTDQmCYuKJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhtFGjtLs%253D&md5=6a9e9bf46d9ce3571d8da77495cc9e19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2FBF01059688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01059688%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DK.%2BS.%26aulast%3DRowland%26aufirst%3DM.%26atitle%3DHepatic%2520clearance%2520of%2520drugs.%2520I.%2520Theoretical%2520considerations%2520of%2520a%2520%25E2%2580%259Cwell-stirred%25E2%2580%259D%2520model%2520and%2520a%2520%25E2%2580%259Cparallel%2520tube%25E2%2580%259D%2520model.%2520Influence%2520of%2520hepatic%2520blood%2520flow%252C%2520plasma%2520and%2520blood%2520cell%2520binding%252C%2520and%2520the%2520hepatocellular%2520enzymatic%2520activity%2520on%2520hepatic%2520drug%2520clearance%26jtitle%3DJ.%2520Pharmacokinet.%2520Biopharm.%26date%3D1977%26volume%3D5%26spage%3D625%26epage%3D653%26doi%3D10.1007%2Fbf01059688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a 96-well equilibrium dialysis apparatus for measuring plasma protein binding</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1002/jps.10332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1002%2Fjps.10332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=12712416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1ehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=967-974&author=M.+J.+Bankerauthor=T.+H.+Clarkauthor=J.+A.+Williams&title=Development+and+validation+of+a+96-well+equilibrium+dialysis+apparatus+for+measuring+plasma+protein+binding&doi=10.1002%2Fjps.10332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a 96-well equilibrium dialysis apparatus for measuring plasma protein binding</span></div><div class="casAuthors">Banker, Michael J.; Clark, Tracey H.; Williams, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">967-974</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A 96-well equil. dialysis block was designed and constructed that is compatible with most std. 96-well format lab. supplies and instruments.  The unique design of the dialysis app. allows one to dispense and aspirate from either or both the sample and dialyzate sides from the top of the app., which is not possible with systems currently on the market.  This feature permits the investigator to analyze a large no. of samples, time points, or replicates in the same expt.  The novel alignment of the dialysis membrane vertically in the well maximizes the surface-to-vol. ratio, eliminates problems assocd. with trapped air pockets, and allows one to add or remove samples independently or all at once.  Furthermore, the design of the app. allows both the sample and dialyzate sides of the dialysis well to be accessible by robotic systems, so assays can be readily automated.  Teflon construction is used to minimize nonspecific binding of test samples to the app.  The device is reusable, easily assembled, and can be shaken in controlled temp. environments to decrease the time required to reach equil. as well as facilitate dissoln. of test compds.  Plasma protein binding values obtained for 10 diverse compds. using std. dialysis equipment and the 96-well dialysis block validates this method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9qyX_9tHHrVg90H21EOLACvtfcHk0liMTDQmCYuKJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1ehsrs%253D&md5=0948f0a11373c57be06be9e0d604ad40</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjps.10332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10332%26sid%3Dliteratum%253Aachs%26aulast%3DBanker%26aufirst%3DM.%2BJ.%26aulast%3DClark%26aufirst%3DT.%2BH.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%252096-well%2520equilibrium%2520dialysis%2520apparatus%2520for%2520measuring%2520plasma%2520protein%2520binding%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26spage%3D967%26epage%3D974%26doi%3D10.1002%2Fjps.10332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span>See: <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span>; <span class="NLM_string-name">Di, L.</span></span> <i>Drug-like Properties: Concepts, Structure Design and Methods</i>; <span class="NLM_publisher-name">Elsevier Inc./Academic Press</span>: <span class="NLM_publisher-loc">Burlington, MA, USA</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">286</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2FB978-012369520-8.50026-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=276-286&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FB978-012369520-8.50026-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-012369520-8.50026-7%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%26pub%3DElsevier%2520Inc.%252FAcademic%2520Press%26date%3D2008%26spage%3D276%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenz, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brent, D. A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive method for determining hydrophobicity constants by reversed-phase high-performance liquid chromatography</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1021/jm00118a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00118a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADyaL1cXltlGlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1923-1933&author=D.+J.+Minickauthor=J.+H.+Frenzauthor=M.+A.+Patrickauthor=D.+A.+Brent&title=A+comprehensive+method+for+determining+hydrophobicity+constants+by+reversed-phase+high-performance+liquid+chromatography&doi=10.1021%2Fjm00118a010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive method for determining hydrophobicity constants by reversed-phase high-performance liquid chromatography</span></div><div class="casAuthors">Minick, Douglas J.; Frenz, John H.; Patrick, Mark A.; Brent, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1923-33</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A method for detg. hydrophobicity consts. for small, org. mols. by reversed-phase liq. chromatog. (RPLC) is presented.  The method uses capacity ratios measured at a no. of different compns. of MeOH to obtain derived values, denoted log K'w, upon which a new scale of hydrophobicity consts. can be developed.  This scale eliminates potential problems such as peak inversion that hamper RPLC methods by using isocratic data to est. hydrophobicity.  The differential H bonded effect obsd. in most correlations of RPLC data with logarithms of octanol-water partition coeffs. (log Po/w) for compds. of opposite net H bonding capabilities (noncongeners) was minimized by adding trace quantities of n-decylamine and 1-octanol to the eluent and using an octyl-modified silica gel stationary phase.  Values of log K'w are largely column-independent as long as the hydrophobic properties of columns are similar.  The correlation of log k'w values with the logarithms of bovine serum albumin binding consts. (log 1/C) is statistically indistinguishable from the correlation of log 1/C with log Po/w, indicating that this data models log 1/C as well as log Po/w for these compds.  Addnl., the chromatog. system is automatable and thus capable of higher sample throughput than measurements of log Po/w by the shake-flask method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP9rXFRIjKoLVg90H21EOLACvtfcHk0lgpOO4bechNEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXltlGlt7s%253D&md5=ce6a632baa977f92c51f3341f92e303f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm00118a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00118a010%26sid%3Dliteratum%253Aachs%26aulast%3DMinick%26aufirst%3DD.%2BJ.%26aulast%3DFrenz%26aufirst%3DJ.%2BH.%26aulast%3DPatrick%26aufirst%3DM.%2BA.%26aulast%3DBrent%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520method%2520for%2520determining%2520hydrophobicity%2520constants%2520by%2520reversed-phase%2520high-performance%2520liquid%2520chromatography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1923%26epage%3D1933%26doi%3D10.1021%2Fjm00118a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span> </span><span class="NLM_article-title">R: A language and environment for statistical
computing, R version
3.6.3, The R Project for Statistical Computing</span>; <span class="NLM_publisher-name">The R Foundation</span>. <span class="NLM_year">2020</span>, available from: <a href="https://www.r-project.org/" class="extLink">https://www.r-project.org/</a> (accessed July 13, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%3A+A+language+and+environment+for+statistical%0Acomputing%2C+R+version%0A3.6.3%2C+The+R+Project+for+Statistical+Computing%3B+The+R+Foundation.+2020%2C+available+from%3A+https%3A%2F%2Fwww.r-project.org%2F+%28accessed+July+13%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DR%253A%2520A%2520language%2520and%2520environment%2520for%2520statistical%250Acomputing%252C%2520R%2520version%250A3.6.3%252C%2520The%2520R%2520Project%2520for%2520Statistical%2520Computing%26pub%3DThe%2520R%2520Foundation%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span> <i>Glide</i>, Version 5.7; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+Version+5.7%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baumli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. N.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds form the basis for selective P-TEFb inhibition by DRB</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1016%2Fj.chembiol.2010.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=20851342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=931-936&author=S.+Baumliauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Halogen+bonds+form+the+basis+for+selective+P-TEFb+inhibition+by+DRB&doi=10.1016%2Fj.chembiol.2010.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonds Form the Basis for Selective P-TEFb Inhibition by DRB</span></div><div class="casAuthors">Baumli, Sonja; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">931-936</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Cdk9, the kinase of the pos. transcription elongation factor b, is required for processive transcription elongation by RNA polymerase II.  Cdk9 inhibition contributes to the anticancer activity of many Cdk inhibitors under clin. investigation and hence there is interest in selective Cdk9 inhibitors.  DRB (5,6-dichlorobenzimidazone-1-β-D-ribofuranoside) is a commonly used reagent for Cdk9 inhibition in cell biol. studies.  The crystal structures of Cdk9 and Cdk2 in complex with DRB reported here describe the mol. basis for the DRB selectivity toward Cdk9.  The DRB chlorine atoms form halogen bonds that are specific for the Cdk9 kinase hinge region.  Kinetic and thermodn. expts. validate the structural findings and implicate the C-terminal residues of Cdk9 in contributing to the affinity for DRB.  These results open the possibility to exploit halogen atoms in inhibitor design to specifically target Cdk9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMj3v7EG7-1LVg90H21EOLACvtfcHk0lgpOO4bechNEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyit7fN&md5=88686dcae3c6943da8a05e463d4fd419</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DBaumli%26aufirst%3DS.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DHalogen%2520bonds%2520form%2520the%2520basis%2520for%2520selective%2520P-TEFb%2520inhibition%2520by%2520DRB%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D931%26epage%3D936%26doi%3D10.1016%2Fj.chembiol.2010.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span> <i>LigPrep</i>, Version 2.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+Version+2.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span> <i>APEX II</i>, Version 2011.8-0; <span class="NLM_publisher-name">Bruker
AXS Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, USA</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+APEX+II%2C+Version+2011.8-0%3B+Bruker%0AAXS+Inc.%3A+Madison%2C+WI%2C+USA%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAPEX%2520II%26pub%3DBruker%250AAXS%2520Inc%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst-Irmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrick, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalke, D.</span></span> <span> </span><span class="NLM_article-title">Comparison of silver and molybdenum microfocus X-ray sources for single-crystal structure determination</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1107/s1600576714022985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=10.1107%2FS1600576714022985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=26089746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=3-10&author=L.+Krauseauthor=R.+Herbst-Irmerauthor=G.+M.+Sheldrickauthor=D.+Stalke&title=Comparison+of+silver+and+molybdenum+microfocus+X-ray+sources+for+single-crystal+structure+determination&doi=10.1107%2Fs1600576714022985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of silver and molybdenum microfocus X-ray sources for single-crystal structure determination</span></div><div class="casAuthors">Krause, Lennard; Herbst-Irmer, Regine; Sheldrick, George M.; Stalke, Dietmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-10</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">1600-5767</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The quality of diffraction data obtained using silver and molybdenum microsources has been compared for six model compds. with a wide range of absorption factors.  The expts. were performed on two 30 W air-cooled Incoatec IμS microfocus sources with multilayer optics mounted on a Bruker D8 goniometer with a SMART APEX II CCD detector.  All data were analyzed, processed and refined using std. Bruker software.  The results show that Ag Kα radiation can be beneficial when heavy elements are involved.  A numerical absorption correction based on the positions and indexes of the crystal faces is shown to be of limited use for the highly focused microsource beams, presumably because the assumption that the crystal is completely bathed in a (top-hat profile) beam of uniform intensity is no longer valid.  Fortunately the empirical corrections implemented in SADABS, although originally intended as a correction for absorption, also correct rather well for the variations in the effective vol. of the crystal irradiated.  In three of the cases studied (two Ag and one Mo) the final SHELXL R1 against all data after application of empirical corrections implemented in SADABS was below 1%.  Since such corrections are designed to optimize the agreement of the intensities of equiv. reflections with different paths through the crystal but the same Bragg 2θ angles, a further correction is required for the 2θ dependence of the absorption.  For this, SADABS uses the transmission factor of a spherical crystal with a user-defined value of μr (where μ is the linear absorption coeff. and r is the effective radius of the crystal); the best results are obtained when r is biased towards the smallest crystal dimension.  The results presented here suggest that the IUCr publication requirement that a numerical absorption correction must be applied for strongly absorbing crystals is in need of revision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPP3imKqKd17Vg90H21EOLACvtfcHk0lgpOO4bechNEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOmtrc%253D&md5=7a9a2f675fff97a9c18ba4e38bdc57bb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS1600576714022985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600576714022985%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DL.%26aulast%3DHerbst-Irmer%26aufirst%3DR.%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26aulast%3DStalke%26aufirst%3DD.%26atitle%3DComparison%2520of%2520silver%2520and%2520molybdenum%2520microfocus%2520X-ray%2520sources%2520for%2520single-crystal%2520structure%2520determination%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2015%26volume%3D48%26spage%3D3%26epage%3D10%26doi%3D10.1107%2Fs1600576714022985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span> <i>SHELXTL</i>, Version 6.14; <span class="NLM_publisher-name">Bruker AXS
Inc.</span>: <span class="NLM_publisher-loc">Madison, WI, USA</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SHELXTL%2C+Version+6.14%3B+Bruker+AXS%0AInc.%3A+Madison%2C+WI%2C+USA%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSHELXTL%26pub%3DBruker%2520AXS%250AInc%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MY1','PDB','3MY1'); return false;">PDB: 3MY1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i134"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c01000">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14542"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c01000?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01000</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv">CSV</a>)</p></li><li><p class="inline">Kinase selectivity panel data for VIP152 and LC–MS chromatograms for determination of purity (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_001.csv">jm1c01000_si_001.csv (3.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c01000/suppl_file/jm1c01000_si_002.pdf">jm1c01000_si_002.pdf (1.12 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c01000&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c01000" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c01000" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993083392a3c28","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
