
<html lang="en"     class="pb-page"  data-request-id="613ce549-0813-4b76-a255-81fba0489ddc"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c01215"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment-Based Discovery of Novel Non-Hydroxamate LpxC Inhibitors with Antibacterial Activity" /></meta><meta name="dc.Creator" content="Yousuke  Yamada" /></meta><meta name="dc.Creator" content="Hajime  Takashima" /></meta><meta name="dc.Creator" content="David Lee  Walmsley" /></meta><meta name="dc.Creator" content="Fumihito  Ushiyama" /></meta><meta name="dc.Creator" content="Yohei  Matsuda" /></meta><meta name="dc.Creator" content="Harumi  Kanazawa" /></meta><meta name="dc.Creator" content="Toru  Yamaguchi-Sasaki" /></meta><meta name="dc.Creator" content="Nozomi  Tanaka-Yamamoto" /></meta><meta name="dc.Creator" content="Junya  Yamagishi" /></meta><meta name="dc.Creator" content="Risa  Kurimoto-Tsuruta" /></meta><meta name="dc.Creator" content="Yuya  Ogata" /></meta><meta name="dc.Creator" content="Norikazu  Ohtake" /></meta><meta name="dc.Creator" content="Hayley  Angove" /></meta><meta name="dc.Creator" content="Lisa  Baker" /></meta><meta name="dc.Creator" content="Richard  Harris" /></meta><meta name="dc.Creator" content="Alba  Macias" /></meta><meta name="dc.Creator" content="Alan  Robertson" /></meta><meta name="dc.Creator" content="Allan  Surgenor" /></meta><meta name="dc.Creator" content="Hayato  Watanabe" /></meta><meta name="dc.Creator" content="Koichiro  Nakano" /></meta><meta name="dc.Creator" content="Masashi  Mima" /></meta><meta name="dc.Creator" content="Kunihiko  Iwamoto" /></meta><meta name="dc.Creator" content="Atsushi  Okada" /></meta><meta name="dc.Creator" content="Iichiro  Takata" /></meta><meta name="dc.Creator" content="Kosuke  Hitaka" /></meta><meta name="dc.Creator" content="Akihiro  Tanaka" /></meta><meta name="dc.Creator" content="Kiyoko  Fujita" /></meta><meta name="dc.Creator" content="Hiroyuki  Sugiyama" /></meta><meta name="dc.Creator" content="Roderick E.  Hubbard" /></meta><meta name="dc.Description" content="UDP-3-O-acyl-N-acetylglucosamine deacetylase (LpxC) is a zinc metalloenzyme that catalyzes the first committed step in the biosynthesis of Lipid A, an essential component of the cell envelope of Gr..." /></meta><meta name="Description" content="UDP-3-O-acyl-N-acetylglucosamine deacetylase (LpxC) is a zinc metalloenzyme that catalyzes the first committed step in the biosynthesis of Lipid A, an essential component of the cell envelope of Gr..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 19, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01215" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01215" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01215" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01215" /></link>
        
    
    

<title>Fragment-Based Discovery of Novel Non-Hydroxamate LpxC Inhibitors with Antibacterial Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01215" /></meta><meta property="og:title" content="Fragment-Based Discovery of Novel Non-Hydroxamate LpxC Inhibitors with Antibacterial Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0026.jpeg" /></meta><meta property="og:description" content="UDP-3-O-acyl-N-acetylglucosamine deacetylase (LpxC) is a zinc metalloenzyme that catalyzes the first committed step in the biosynthesis of Lipid A, an essential component of the cell envelope of Gram-negative bacteria. The most advanced, disclosed LpxC inhibitors showing antibacterial activity coordinate zinc through a hydroxamate moiety with concerns about binding to other metalloenzymes. Here, we describe the discovery, optimization, and efficacy of two series of compounds derived from fragments with differing modes of zinc chelation. A series was evolved from a fragment where a glycine moiety complexes zinc, which achieved low nanomolar potency in an enzyme functional assay but poor antibacterial activity on cell cultures. A second series was based on a fragment that chelated zinc through an imidazole moiety. Structure-guided design led to a 2-(1S-hydroxyethyl)-imidazole derivative exhibiting low nanomolar inhibition of LpxC and a minimum inhibitory concentration (MIC) of 4 μg/mL against Pseudomonas aeruginosa, which is little affected by the presence of albumin." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01215"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01215">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01215&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01215&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01215&amp;href=/doi/10.1021/acs.jmedchem.0c01215" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14805-14820</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01265" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01243" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Based Discovery of Novel Non-Hydroxamate LpxC Inhibitors with Antibacterial Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yousuke Yamada</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yousuke Yamada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a9d0c684d0c8c4c8cdc8e9ddc8c0dac1c687cac687c3d9"><span class="__cf_email__" data-cfemail="fc8593d1859d919d989dbc889d958f9493d29f93d2968c">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yousuke++Yamada">Yousuke Yamada</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hajime Takashima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hajime Takashima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hajime++Takashima">Hajime Takashima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Lee Walmsley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Lee Walmsley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+Lee++Walmsley">David Lee Walmsley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fumihito Ushiyama</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fumihito Ushiyama</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fumihito++Ushiyama">Fumihito Ushiyama</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yohei Matsuda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yohei Matsuda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yohei++Matsuda">Yohei Matsuda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Harumi Kanazawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harumi Kanazawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harumi++Kanazawa">Harumi Kanazawa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Toru Yamaguchi-Sasaki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Toru Yamaguchi-Sasaki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Toru++Yamaguchi-Sasaki">Toru Yamaguchi-Sasaki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nozomi Tanaka-Yamamoto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nozomi Tanaka-Yamamoto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nozomi++Tanaka-Yamamoto">Nozomi Tanaka-Yamamoto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junya Yamagishi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junya Yamagishi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junya++Yamagishi">Junya Yamagishi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Risa Kurimoto-Tsuruta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Risa Kurimoto-Tsuruta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Risa++Kurimoto-Tsuruta">Risa Kurimoto-Tsuruta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuya Ogata</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuya Ogata</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuya++Ogata">Yuya Ogata</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Norikazu Ohtake</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Norikazu Ohtake</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Norikazu++Ohtake">Norikazu Ohtake</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hayley Angove</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hayley Angove</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hayley++Angove">Hayley Angove</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lisa Baker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lisa Baker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++Baker">Lisa Baker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Harris</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Harris</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Harris">Richard Harris</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alba Macias</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alba Macias</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alba++Macias">Alba Macias</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alan Robertson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alan Robertson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alan++Robertson">Alan Robertson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allan Surgenor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allan Surgenor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allan++Surgenor">Allan Surgenor</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hayato Watanabe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hayato Watanabe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hayato++Watanabe">Hayato Watanabe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Koichiro Nakano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Koichiro Nakano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Koichiro++Nakano">Koichiro Nakano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masashi Mima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masashi Mima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masashi++Mima">Masashi Mima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kunihiko Iwamoto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kunihiko Iwamoto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kunihiko++Iwamoto">Kunihiko Iwamoto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Atsushi Okada</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Atsushi Okada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Atsushi++Okada">Atsushi Okada</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iichiro Takata</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iichiro Takata</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iichiro++Takata">Iichiro Takata</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kosuke Hitaka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kosuke Hitaka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kosuke++Hitaka">Kosuke Hitaka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Akihiro Tanaka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Akihiro Tanaka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Akihiro++Tanaka">Akihiro Tanaka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kiyoko Fujita</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kiyoko Fujita</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kiyoko++Fujita">Kiyoko Fujita</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hiroyuki Sugiyama</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hiroyuki Sugiyama</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Taisho Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hiroyuki++Sugiyama">Hiroyuki Sugiyama</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Roderick E. Hubbard</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roderick E. Hubbard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a9db87c1dccbcbc8dbcde9dfccdbc7c8c5c0da87cac6c4"><span class="__cf_email__" data-cfemail="5c2e7234293e3e3d2e381c2a392e323d30352f723f3331">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roderick+E.++Hubbard">Roderick E. Hubbard</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01215&amp;href=/doi/10.1021%2Facs.jmedchem.0c01215" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14805–14820</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 19, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 July 2020</li><li><span class="item_label"><b>Published</b> online</span>19 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01215" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01215</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01215"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2439</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01215" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Based Discovery of Novel Non-Hydroxamate LpxC Inhibitors with Antibacterial Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yousuke&quot;,&quot;last_name&quot;:&quot;Yamada&quot;},{&quot;first_name&quot;:&quot;Hajime&quot;,&quot;last_name&quot;:&quot;Takashima&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Lee Walmsley&quot;},{&quot;first_name&quot;:&quot;Fumihito&quot;,&quot;last_name&quot;:&quot;Ushiyama&quot;},{&quot;first_name&quot;:&quot;Yohei&quot;,&quot;last_name&quot;:&quot;Matsuda&quot;},{&quot;first_name&quot;:&quot;Harumi&quot;,&quot;last_name&quot;:&quot;Kanazawa&quot;},{&quot;first_name&quot;:&quot;Toru&quot;,&quot;last_name&quot;:&quot;Yamaguchi-Sasaki&quot;},{&quot;first_name&quot;:&quot;Nozomi&quot;,&quot;last_name&quot;:&quot;Tanaka-Yamamoto&quot;},{&quot;first_name&quot;:&quot;Junya&quot;,&quot;last_name&quot;:&quot;Yamagishi&quot;},{&quot;first_name&quot;:&quot;Risa&quot;,&quot;last_name&quot;:&quot;Kurimoto-Tsuruta&quot;},{&quot;first_name&quot;:&quot;Yuya&quot;,&quot;last_name&quot;:&quot;Ogata&quot;},{&quot;first_name&quot;:&quot;Norikazu&quot;,&quot;last_name&quot;:&quot;Ohtake&quot;},{&quot;first_name&quot;:&quot;Hayley&quot;,&quot;last_name&quot;:&quot;Angove&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;Baker&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Harris&quot;},{&quot;first_name&quot;:&quot;Alba&quot;,&quot;last_name&quot;:&quot;Macias&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;Robertson&quot;},{&quot;first_name&quot;:&quot;Allan&quot;,&quot;last_name&quot;:&quot;Surgenor&quot;},{&quot;first_name&quot;:&quot;Hayato&quot;,&quot;last_name&quot;:&quot;Watanabe&quot;},{&quot;first_name&quot;:&quot;Koichiro&quot;,&quot;last_name&quot;:&quot;Nakano&quot;},{&quot;first_name&quot;:&quot;Masashi&quot;,&quot;last_name&quot;:&quot;Mima&quot;},{&quot;first_name&quot;:&quot;Kunihiko&quot;,&quot;last_name&quot;:&quot;Iwamoto&quot;},{&quot;first_name&quot;:&quot;Atsushi&quot;,&quot;last_name&quot;:&quot;Okada&quot;},{&quot;first_name&quot;:&quot;Iichiro&quot;,&quot;last_name&quot;:&quot;Takata&quot;},{&quot;first_name&quot;:&quot;Kosuke&quot;,&quot;last_name&quot;:&quot;Hitaka&quot;},{&quot;first_name&quot;:&quot;Akihiro&quot;,&quot;last_name&quot;:&quot;Tanaka&quot;},{&quot;first_name&quot;:&quot;Kiyoko&quot;,&quot;last_name&quot;:&quot;Fujita&quot;},{&quot;first_name&quot;:&quot;Hiroyuki&quot;,&quot;last_name&quot;:&quot;Sugiyama&quot;},{&quot;first_name&quot;:&quot;Roderick&quot;,&quot;last_name&quot;:&quot;E. Hubbard&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14805-14820&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01215&quot;},&quot;abstract&quot;:&quot;UDP-3-O-acyl-N-acetylglucosamine deacetylase (LpxC) is a zinc metalloenzyme that catalyzes the first committed step in the biosynthesis of Lipid A, an essential component of the cell envelope of Gram-negative bacteria. The most advanced, disclosed LpxC inhibitors showing antibacterial activity coordinate zinc through a hydroxamate moiety with concerns about binding to other metalloenzymes. Here, we describe the discovery, optimization, and efficacy of two series of compounds derived from fragments with differing modes of zinc chelation. A series was evolved from a fragment where a glycine moiety complexes zinc, which achieved low nanomolar potency in an enzyme functional assay but poor antibacterial activity on cell cultures. A second series was based on a fragment that chelated zinc through an imidazole moiety. Structure-guided design led to a 2-(1S-hydroxyethyl)-imidazole derivative exhibiting low nanomolar inhibition of LpxC and a minimum inhibitory concentration (MIC) of 4 μg/mL against Pseudomonas aerugi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01215&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01215" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01215&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01215" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01215&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01215" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01215&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01215&amp;href=/doi/10.1021/acs.jmedchem.0c01215" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01215" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01215" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01215%26sid%3Dliteratum%253Aachs%26pmid%3D33210531%26genre%3Darticle%26aulast%3DYamada%26date%3D2020%26atitle%3DFragment-Based%2BDiscovery%2Bof%2BNovel%2BNon-Hydroxamate%2BLpxC%2BInhibitors%2Bwith%2BAntibacterial%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14805%26epage%3D14820%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291399" title="Zinc">Zinc</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292253" title="Imidazoles">Imidazoles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">UDP-3-<i>O</i>-acyl-<i>N</i>-acetylglucosamine deacetylase (LpxC) is a zinc metalloenzyme that catalyzes the first committed step in the biosynthesis of Lipid A, an essential component of the cell envelope of Gram-negative bacteria. The most advanced, disclosed LpxC inhibitors showing antibacterial activity coordinate zinc through a hydroxamate moiety with concerns about binding to other metalloenzymes. Here, we describe the discovery, optimization, and efficacy of two series of compounds derived from fragments with differing modes of zinc chelation. A series was evolved from a fragment where a glycine moiety complexes zinc, which achieved low nanomolar potency in an enzyme functional assay but poor antibacterial activity on cell cultures. A second series was based on a fragment that chelated zinc through an imidazole moiety. Structure-guided design led to a 2-(1<i>S</i>-hydroxyethyl)-imidazole derivative exhibiting low nanomolar inhibition of LpxC and a minimum inhibitory concentration (MIC) of 4 μg/mL against <i>Pseudomonas aeruginosa</i>, which is little affected by the presence of albumin.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28170" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28170" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">There has been growing concern over the emergence of bacterial resistance to existing therapeutics<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and the challenges of discovering new antibiotics.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> An attractive target for the development of novel therapeutic agents is the bacterial enzyme UDP-3-<i>O</i>-acyl-<i>N</i>-acetylglucosamine deacetylase (LpxC).<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> The enzyme is essential to most Gram-negative pathogens,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> catalyzing the removal of an <i>N</i>-acetyl group from UDP-3-<i>O</i>-acyl-<i>N</i>-acetylglucosamine, which forms the core of Lipid A, the first committed step in biosynthesis of the lipopolysaccharide (LPS), which is an essential component of the bacterial cell wall (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). There are no mammalian homologues, and several series of inhibitors (summarized in ref<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a>) have been developed (representatives in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The most advanced disclosed compounds contain a hydroxamate functionality that complexes the zinc ion at the core of the active site of the metalloenzyme. One of these compounds (ACHN-975<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a>) entered clinical trials but was discontinued due to side effects not thought to be target-related.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Although other properties of ACHN-975 may be important, the hydroxamate moiety has been linked to mutagenicity in HDAC inhibitors<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and also to be responsible for off-target effects due to the lack of specificity in many matrix metalloprotease (MMP) inhibitors.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> There is therefore interest in identifying LpxC inhibitors containing non-hydroxamate metal binding groups. Hydroxypyrone-containing compounds such as <b>2</b> have been discussed<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and are proposed to form a bidentate complex with the zinc ion (patent WO 2015/085238 from Forge Therapeutics), and further examples of this motif have been disclosed in pyrone compounds such as <b>3</b> (Example 49 in patent WO 2017/083434 from Forge Therapeutics). However, no details of the pyrone-containing compounds have yet been published. Compound <b>2</b> is referred to as LpxCi-1 in ref;<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> this should not be confused with a hydroxamate-containing compound named LpxCi-1 in the Achaogen patent WO 2011/005355.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. LpxC catalyzes the first committed step of LPS biosynthesis (shown is the substrate for <i>Pseudomonas aeruginosa</i>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative LpxC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The published crystal structure of <b>1</b> (PF-5081090) in the active site of LpxC from <i>Pseudomonas aeruginosa</i> (PaLpxC)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> illustrates key features of inhibitor binding as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The protein coordinates the catalytic zinc ion through side chains of D241, H78, and H237 with a trigonal pyramidal geometry. In <b>1</b>, the hydroxamate makes a bidentate interaction to complete a trigonal bipyramidal geometry at the zinc ion, and the pendant sulfone occupies the pocket that binds a phosphate of UDP (see the structure with the PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MDT">4MDT</a><a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>) adjacent to K238. The pyridone ring makes only hydrophobic interactions with the protein binding site, acting as a linker that connects the core zinc-chelating unit to groups that mimic the acyl chain in the natural substrate that binds in a hydrophobic tunnel that leads to the solvent. These features of the binding site are exploited by most published inhibitors: (a) a zinc-chelating motif, (b) a polar group occupying the UDP binding pocket, (c) a linker that is mainly hydrophobic but sometimes makes at least one hydrogen bond interaction, (d) an extended hydrophobic moiety that binds in the tunnel, and (e) often addition of groups out into the solvent at the end of the tunnel to modulate compound properties. Crystal structures of different compounds bound to LpxC from various organisms show variation in the extent and nature of the tunnel, which leads to varying specificity for some compounds for different bacterial species.<a onclick="showRef(event, 'ref3 ref14'); return false;" href="javascript:void(0);" class="ref ref3 ref14">(3,14)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Detail of the crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UPG">5UPG</a>) of <b>1</b> (PF-5081090) binding to the active site of PaLpxC. Unless otherwise stated, in all structural figures, compounds are shown by sticks with orange carbons, the zinc ion as a van der Waals sphere, and protein amino acids (side chains only) in ball and stick. Except for (b) and 4(a), all other figures of compounds binding to LpxC are presented with the same view and convention. (a) Inhibitor binding site, with the inside of the solvent surface in gray and the outside in yellow. (b) Detail of inhibitor and protein binding to a zinc ion, showing just the hydroxamate moiety of the ligand, the NE2 atoms of the imidazole of histidines, and one oxygen of the side chain of aspartic acid. Note that <b>1</b> also binds in a second site, described as the F181 site later in this manuscript.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Here, we report the fragment-based discovery, optimization, and characterization of a potent, non-hydroxamate-containing inhibitor of PaLpxC, which shows efficacy in various measures of antibacterial activity.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Establishing a Structure-Based Discovery Platform</h3><div class="NLM_p">Two constructs of the enzyme were used in the work reported here. A construct for residues 1 to 303 with a C-terminal his-tag (hereafter PaLpxC (1–303-His)) was used for the functional assay. A construct for residues 1 to 299 (based on the published structure, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UPG">5UPG</a>) was generated with the surface mutation C40S to reduce protein aggregation (hereafter PaLpxC) and used for all structural and biophysical measurements, including the binding assay. Initial attempts to express and purify N-terminal poly-histidine-tagged PaLpxC produced insoluble aggregates. Removing the tag with proteolytic cleavage did not improve the solubility. Expression of untagged PaLpxC with a reduced induction time resulted in a protein that was more soluble but did not generate crystals. Analysis by <sup>1</sup>H–<sup>15</sup>N HSQC NMR of a <sup>15</sup>N-labeled sample of PaLpxC (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S1a</a>) showed fewer amide peaks than expected, suggesting that the protein is partially unfolded; adding zinc improved the <sup>1</sup>H–<sup>15</sup>N HSQC NMR (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S1b</a>), but the protein still did not form crystals. Thermal shift analysis (TSA) suggested there was still heterogeneity; maintaining zinc in the expression medium resulted in a protein with improved behavior in TSA (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S2</a>) and that did generate crystals. This protein was used in all subsequent experiments.</div><div class="NLM_p last">A binding assay and a functional assay were used to characterize inhibition by compounds. The binding assay used fluorescence polarization (FP) to measure the displacement of a novel fluorescently labeled hydroxamate compound <b>5</b> (see SI) from PaLpxC. This probe resulted from careful optimization of the labeling position and the fluorophore to minimize artifacts from slow binding kinetics probably due to the architecture of the active site of the enzyme. The fluorophore is attached to the sulfone, which occupies the UDP binding pocket and extends out into the solvent. Two different functional assays were used. For the imidazole series, the functional assay measured activity by the reaction of fluorescamine with the free amino group generated on the substrate by LpxC. The glycine series compounds gave interference in the fluorescamine assay, so a liquid chromatography mass spectrometry (LCMS) assay was developed to measure the amount of the product formed. For all assays, compound inhibition or binding is reported as IC<sub>50</sub> values and ligand efficiency (LE) as −RT ln IC<sub>50</sub>/heavy atom count.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hit Identification</h3><div class="NLM_p">The Vernalis fragment library<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> of 1152 compounds was screened against PaLpxC using three ligand-observed NMR experiments (STD, water-LOGSY, and CPMG)<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> in pools of six compounds. A binding signal was observed for 252 compounds in one or more NMR experiments. Putative hits were retested as singletons and for competition with either <b>1</b> or <b>4</b>, chosen to occupy more of the UDP binding pocket (see the modeled structure in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S3</a>). Sixteen fragments were competitive with <b>1</b> (four of which were positive in all three NMR experiments). Representative hits are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> (all listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Table S1</a>). The remaining noncompetitive fragment hits were assessed for competition with <b>4</b>, resulting in a further 12 confirmed fragment hits, 4 in all three NMR experiments (structures not shown). These 28 fragment hits were assessed in the FP binding assay. Most bound with lower affinity than the nominal IC<sub>50</sub> threshold of 2.5 mM, but some showed clear inhibition in the functional assay such as <b>6</b> with an IC<sub>50</sub> of 41.9 μM with a similar IC<sub>50</sub> in the FP assay.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, crystal structures of representative fragments bound to PaLpxC revealed that molecules that were competitive with the hydroxamate tool compounds in the NMR experiments bound to two sites–coordinated to the zinc (<b>6</b>) or in the tunnel (<b>7</b>). The tunnel is collapsed somewhat when small zinc-coordinating compounds are bound but expands when compounds are in the tunnel. Some fragments (such as <b>8</b>) that are noncompetitive with the hydroxamate tool compounds were found to bind to a site distant from the active site (more than 14 Å through the protein to the zinc ion away from the substrate binding site). This site is preformed in all crystal structures and was also observed in the published crystal structure of <b>1</b> (PF-5081090) bound to PaLpxC. The binding site is provided by side chains from several hydrophobic residues (F152 and F176) with F181 at the entrance. Such F181 site compounds were not active in the functional assay and not pursued further.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fragments are found that bind to different sites on LpxC. (a). Schematic of the protein structure, with F181 in ball and stick, with ligand in CPK for the crystal structures of a zinc-coordinating glycine fragment (<b>6</b>, yellow), a tunnel binder (<b>7</b>, brown), and binding to the F181 site (<b>8</b>, green). A second copy (not shown) of <b>8</b> binds in the tunnel. (b) Detail of the binding of glycine fragment (<b>6</b>) with just the main chain of M62 shown (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for the legend). (c). Representative fragment hits.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Two zinc-chelating fragments were selected for investigation, a glycine fragment <b>6</b> and an imidazole fragment <b>9</b>, both of which were competitive with <b>1</b>. In the TSA, <b>6</b> and <b>9</b> stabilize PaLpxC by 4.0 °C and 0.78 °C, respectively, at 2 mM compound as an additional confirmation of binding.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Fragment to Hit for the Glycine Series</h3><div class="NLM_p">As seen in the crystal structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b), the phenyl ring of the aniline of <b>6</b> forms the linker toward the entrance to the tunnel and is sandwiched between hydrophobic amino acid side chains (L18 and A214), while the NH of the aniline makes a water-bridged hydrogen bond to the main chain of M62 (the side chain of this residue is disordered in some of the crystal structures). The zinc-chelating amine group (N–Zn<sup>2+</sup> distance of ∼2.1Å in all glycine series crystal structures) of the amino acid also coordinates to H264 and E77. As expected, modification of that amine group to either dimethylamino, methylamino, hydroxyl, or morpholino led to loss of both binding and enzyme inhibitory activity (data not shown).</div><div class="NLM_p">We first focused on modification of the amino acid side chain of <b>6</b> to optimize interactions in the substrate binding site. Inspection of the crystal structure predicted that the activity would be improved in a D-Ser derivative, which introduces a hydroxyl group to interact with the side chain of T190. This was confirmed in <b>10</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S4a</a>) with an additional interaction seen in the crystal structure with D241 that also chelates the zinc ion. As predicted, the L-Ser derivative <b>11</b> showed a significant decrease in activity; however, the replacement of the hydroxyl of D-Ser with an amino group <b>12</b> was less successful. This could be for two reasons. First, the calculated pKa of the chelating NH<sub>2</sub> is reduced from 6.5 in <b>10</b> (hydroxyl) to 4.4 in <b>12</b> (amino). The pKa is a measure of the strength of bonding of H<sup>+</sup> to the NH<sub>2</sub> of the amino acid; this can be taken as a surrogate for the chelating strength of the NH<sub>2</sub> to the positively charged zinc ion, so a decrease in pKa will mean a decrease in chelating strength.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The other reason for the reduction in binding could be due to changes in hydrogen bonding. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S4a</a>, the hydroxyl in <b>10</b> donates one hydrogen bond (D241) and accepts one hydrogen bond (T190). Changing the hydroxyl for a positively charged nitrogen (NH<sub>3</sub><sup>+</sup>) with three polar hydrogen donors would require T190 to rotate and become an acceptor and still leave one buried unsatisfied hydrogen bond donor.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Glycine Series: SAR for Interacting with Zinc and the Substrate Binding Site</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0015.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0016.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_p">Modeling from the co-crystallized structure of <b>10</b> suggested that a sulfonyl group linked by two carbons but not by one carbon could interact with K238, similar to what is seen for <b>1</b>. The increase in potency was confirmed in <b>14</b> (compared to <b>13</b>) but not improved by replacing the sulfone with a sulfonamide in <b>15</b>, perhaps because the chelating ability of the primary amino group was reduced (pKa = 6.2 in <b>14</b>, but decreased to pKa = 5.5 in <b>15</b>) or perhaps again because of a slight movement of side chains altering the detail of hydrogen bonding. Further modeling suggested that the sulfone could be replaced by a carboxylic acid with a 1-carbon linker to make additional interactions with H264. This carboxylic acid derivative <b>16</b> retained potency, and the predicted binding modes of <b>14</b> and <b>16</b> to PaLpxC were confirmed in crystal structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S4b,c</a>). However, these modifications degraded ligand efficiency, so subsequent SAR fixed the D-Ser as the zinc-chelating moiety and explored extension into the hydrophobic tunnel region.</div><div class="NLM_p">The bound structure of <b>6</b> showed limited opportunities for further interactions from the aniline. Replacing the trifluoromethoxy group with extended ethers could not provide a good vector into the tunnel region. In addition, the amide NH interacts with the protein only through a water-bridged hydrogen bond. We modeled substituted piperidines and pyrrolidines incorporating additional sp3 carbons to explore the vectors toward the tunnel and also reduce the number of polar H-bond donors. Replacing the aniline with 4-benzyl piperidine (as in <b>17</b>) provides a linker with the benzyl group in the entrance of the tunnel with a synthetically accessible vector into the tunnel. Various hydrophobic groups (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) were appended to the para position of the benzyl, which successfully increased potency with retained ligand efficiency. The length and rigidity of the linker between aromatic groups had an effect on activity (as seen comparing <b>18</b>, <b>19</b>, and <b>20</b>), emphasizing the importance of achieving the correct placement in the hydrophobic tunnel.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Glycine Series: SAR for Extension into the Hydrophobic Tunnel</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0017.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0018.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">Introducing an alkyne linker places the second phenyl ring centrally into the hydrophobic tunnel formed by L18, A214, A206, and I197, and addition of a para-hydroxymethyl (in <b>21</b>) on this second ring extends into the solvent at the exit of the tunnel. This further improved enzyme activity with some loss of ligand efficiency. Although <b>21</b> inhibited the enzyme with an IC<sub>50</sub> of 20 nM, this did not translate into antibacterial activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Addition of a sulfonyl group (as in <b>14</b> to interact with K238) further improved the enzyme activity to single digit nanomolar (<b>22</b>). The crystal structure of <b>22</b> bound to PaLpxC (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) confirmed the designed interactions with the protein and the placement of the aromatic rings in the tunnel. The geometry of zinc binding is essentially conserved across the series (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figures S4</a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>b), with a trigonal bipyramidal coordination of the zinc ion, with two oxygens (OD of D241 and ligand carbonyl O) defining the <i>z-</i>axis. However, this most potent of the glycine series showed only weak antibacterial activity in the presence of phenylalanine-arginine β-naphthylamide (PAβN) (an efflux pump inhibitor), so the series was not taken further.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a). Detail of the crystal structure of <b>22</b> binding to the active site of PaLpxC(1-299). See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used. (b). Different view showing zinc coordination to the selected ligand and protein side-chain atoms showing CD1, NE2, and CD2 of the imidazole of histidines and the carboxylic acid groups of the glutamic and aspartic acids.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Glycine Series: Antibacterial Activity</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0019.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">MIC (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">functional IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>E. coli</i>+ PAβN</th><th class="colsep0 rowsep0" align="center"><i>E. coli</i><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><i>P. aeruginosa</i>+ PAβN</th><th class="colsep0 rowsep0" align="center"><i>P. aeruginosa</i><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">0.0221</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="left">>32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>SO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">0.00603</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="left">64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Racemic mixture.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last"><i>Escherichia coli</i> ATCC25922.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last"><i>Pseudomonas aeruginosa</i> ATCC27853.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Fragment to Hit for the Imidazole Series</h3><div class="NLM_p">The initial imidazole fragment hit <b>9</b> bound weakly to PaLpxC. Exploration of the activity of closely related (near neighbor) compounds established that addition of chlorine to the phenyl moiety (<b>23</b>) enhanced activity. The replacement of the hydroxyl by amino or methoxy groups did not lead to improvement (data not shown), but removal of the hydroxyl (<b>24</b>) and the oxygen linker (<b>25</b>) was beneficial (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) with a much-improved ligand efficiency. Reducing the length of the 4-carbon aliphatic linker between imidazole and chlorophenyl to 2-carbons as in <b>26</b> had an impact on activity; 3-carbons retained the same ligand efficiency (<b>27</b>), and this was the first imidazole compound to give a crystal structure (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S5</a>). This confirmed that the imidazole is a monodentate chelator of the zinc ion, placing the aliphatic chain and chlorophenyl toward the hydrophobic tunnel.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Imidazole Series: Improving Zinc Chelation<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0020.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0021.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">* denotes racemic mixture.</p></div></div><div></div></div><div class="NLM_p">The first optimization was of the imidazole to improve chelation by conversion into a bidentate zinc binder. The crystal structure (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Figure S5</a>) suggested that extending from the 2 position of the imidazole would be effective and 2-hydroxymethyl (<b>28</b>) retained ligand efficiency, which was further enhanced in <b>29</b>, with a 2-(1-hydroxyethyl) group. The co-crystal structure of <b>29</b> with PaLpxC (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) confirmed that the compound forms a bidentate complex with trigonal bipyramidal geometry to the zinc through the hydroxyl oxygen atom and an imidazole nitrogen (see also <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b for details of coordination) to give an optimized hit compound. The structure also suggested that the <i>S</i> enantiomer of the 2-(1-hydroxyethyl) group is the most potent.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0007.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Detail of the crystal structure of <b>29</b> binding to the active site of PaLpxC. See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">This exploration of the SAR of the initial imidazole fragment hit illustrates two important features of fragment-based discovery.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> First, the fragment library typically consists of a small number of compounds, selected on physicochemical properties and some measure of chemical diversity.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> As with any screening approach, the hope is that the library contains something that binds well enough to be characterized. It is then important to optimize the features of the fragment that are making the key interaction with the target binding site, before embarking on large-scale optimization of druglike properties. The second feature is that fragments can provide a diversity of start points for optimization.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> In this study, the zinc-chelating groups of the glycine and imidazole series bind so that the remainder of the compound provides different vectors and opportunities for reaching into the linker and tunnel regions, reflected in the different amino acid groups that are contacted.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Hit to Lead for the Imidazole Series</h3><div class="NLM_p">The co-crystal structure of <b>29</b> bound to PaLpxC (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) suggested how to extend the compound into the hydrophobic tunnel of LpxC with moieties similar to those in the glycine series and other LpxC inhibitors. The exploration of phenyl rings and lengths of a linker (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) showed that, as for the glycine series, the enzyme inhibitory activity tended to improve with elongation of the aromatic portion of the molecule. The ligand efficiency for the placement of a biphenyl moiety into the tunnel was optimal for a 1-carbon (<b>32</b>) or a 3-carbon (<b>34</b>) rather than a 2-carbon linker (<b>34</b>), similar to what is observed for a 2-carbon vs 3-carbon linker to chlorophenyl, comparing <b>30</b> to <b>29</b> and <b>31</b>. This feature was retained with further extension through the hydrophobic tunnel with a Ph-alkyne-Ph moiety (compare <b>36</b> with <b>35</b> and <b>37</b>) with <b>35</b> and <b>37</b> showing submicromolar functional activity. Although co-crystal structures of this subset of compounds bound to PaLpxC could not be obtained (poor compound behavior), modeling suggested that the variation in the linker length affected the vector between the hydrophobic tunnel binding group and the zinc-chelating groups with a consequent effect on activity. A 1-carbon linker was therefore chosen as optimal for ligand efficiency.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Imidazole Series: SAR for Extension into the Hydrophobic Tunnel<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0022.gif" alt="" id="fx8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0023.gif" alt="" id="fx9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">* denotes racemic mixture.</p></div></div><div></div></div><div class="NLM_p">Compound <b>35</b> showed weak antibacterial activity against <i>E. coli</i> only in the presence of PAβN (data not shown). This was probably associated with the high lipophilicity of the compound, and so the next step in optimization was to reduce the lipophilicity by introduction of a heterocycle ring in place of the first phenyl ring attached to the linker. Modeling also suggested that a 5-membered hetero-aromatic ring would favorably adjust the vector between the hydrophobic tunnel and zinc atom binding groups. A number of different heterocycles such as variously substituted 1,3-oxazoles, pyrazoles, triazoles, oxadiazoles, and thiazoles were explored (data not shown), but replacement with a 1,2-oxazole was particularly effective, reducing the cLogP (0.9 for <b>38</b> compared to 2.6 for <b>32</b>) but also improving the enzyme inhibitor activity by a factor of 4 with a significant improvement in ligand efficiency in both binding and functional assays. This jump in potency can be explained by inspection of the co-crystal structure of <b>38</b>, which shows that the five-membered ring creates a turn in the ligand. This change in shape matches the binding site of PaLpxC (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), and the ligand makes an additional interaction with H19 of the enzyme. The angle and distance to the NE2 atom of H19 are not consistent with a strong hydrogen bond. We noted that the orientation of this histidine does vary between structures depending on the opportunities for alternate hydrogen bonding interactions, notably between H19 and M62 with small adjustments to the main chain conformation.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0008.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Detail of the crystal structure of <b>38</b> binding to the active site of PaLpxC. The carbonyl of M62 is alone; otherwise, see the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>39</b> is a compound with sub-50 nM functional activity, which combines the features learned from the various series to be important for binding, affinity, and antibacterial activity—a 2-(1<i>S</i>-hydroxyethyl)imidazole moiety for zinc chelation, 1,2-oxazole for interactions in the linker region, and alkyne to provide a suitable vector to place a phenyl group centrally in the hydrophobic tunnel.</div><div class="NLM_p">A series of compounds were synthesized based on <b>39</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), varying a solubilizing group on the phenyl, which as seen in <b>22</b> of the glycine series extends through the end of the tunnel into the solvent to give compounds with low nanomolar inhibition of the enzyme activity and with compound properties that provide antibacterial activity.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Imidazole Series: Antibacterial Activity for Various Gram-Negative Bacteria with a variation in the Solubilizing Group<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0024.gif" alt="" id="fx10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0025.gif" alt="" id="fx11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>rac</i> = racemic mixture.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">1: Functional enzyme activity; 2: <i>E. coli</i> ATCC25922; 3: <i>P. aeruginosa</i> ATCC27853; 4: <i>K. pneumoniae</i> ATCC13883; 5: <i>P. aeruginosa</i> 5567.</p></div></div><div></div></div><div class="NLM_p">Analysis of <b>41</b> and <b>42</b> after chiral separation of the racemic <b>40</b> confirmed that the <i>S</i> enantiomer is more potent (>100× enzyme inhibition). The <i>S</i> enantiomer shows antibacterial activity against a range of Gram-negative bacteria, though the most interesting activity against <i>P. aeruginosa</i> is abolished in the presence of human serum albumin (HSA). Further exploration of the solubilizing group produced compounds <b>44</b> to <b>46</b>, which show similar levels of enzyme inhibition but with more potent antibacterial activity. Compound <b>46</b> has the most potent profile of antibacterial activity across a range of bacterial species, but <b>43</b> (the structure in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) shows the least influence of HSA.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0009.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Detail of the crystal structure of <b>43</b> binding to the active site of PaLpxC. Clipped, semitransparent solvent accessible surface, yellow outside and gray inside. See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for other conventions used. (b) Different view showing zinc coordination to the selected ligand and protein side-chain atoms as described in the legend to <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>(b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Selectivity of Lead Glycine (<b>22</b>) and Imidazole (<b>43</b>) Compounds</h3><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a> lists the inhibition measured for the lead compound of each series and a representative hydroxamate compound (BB-78485<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>) against a small panel of matrix metalloprotease enzymes, MMP-2, MMP-3, and MMP-9. The results illustrate that the structure-guided design of each series has generated compounds with selectivity for LpxC over some representative zinc metalloenzymes, in contrast to previously described hydroxamate LpxC inhibitors.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Inhibition of MMP Enzymes<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">MMP-2</th><th class="colsep0 rowsep0" align="center">MMP-3</th><th class="colsep0 rowsep0" align="center">MMP-9</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BB-78485</td><td class="colsep0 rowsep0" align="left">0.0175</td><td class="colsep0 rowsep0" align="left">0.416</td><td class="colsep0 rowsep0" align="left">0.0972</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ilomastat (MMP inhibitor)</td><td class="colsep0 rowsep0" align="left">0.000422</td><td class="colsep0 rowsep0" align="left">0.0150</td><td class="colsep0 rowsep0" align="left">0.000177</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Values are IC<sub>50</sub> (μM).</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of <b>22</b> and <b>43</b></h3><div class="NLM_p">The synthesis of the advanced compounds <b>22</b> and <b>43</b> illustrates the general scheme for synthesis of the glycine and imidazole series, respectively. The experimental details are provided in the Experimental Section, and the schemes and synthetic details for all tested compounds are provided in the Supporting Information (SI). These compounds have previously been described in the patents WO 2018/216822, WO 2018/216823, and WO2020/105660. The purity of all compounds was measured by LC, and the identity of compounds was confirmed by MS. The purity of all compounds was > 95%, except for compounds <b>10</b> and <b>15</b>, which were 92% pure. LC and mass spectrometric traces for the six key compounds <b>10</b>, <b>25</b>, <b>29</b>, <b>38</b>, <b>41</b>, and <b>43</b> are provided in SI.</div><div class="NLM_p">The synthesis of compound <b>22</b> is outlined in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> shows the synthesis of the amino acid portion of the molecule, intermediate <b>53</b>. The Aldol reaction using lithium diisopropylamide and 2-methylsulfanylacetaldehyde gave the intermediate thioether as a racemic mixture of diastereomers <b>49a</b> and <b>49b</b>. The diastereomers were separated via normal-phase silica gel column chromatography to give the <i>rac</i>-(<i>R</i>*,<i>R</i>*) diastereomer <b>49a</b> and the <i>rac</i>-(<i>R</i>*,<i>S</i>*) diastereomer <b>49b</b> in a 1:1.3 ratio. The relative stereochemistry was determined by X-ray crystallography of ethyl <i>rac</i>-(<i>R</i>*,<i>S</i>*)-2-(dibenzylamino)-3-hydroxy-4-(methylsulfonyl)butanoate (<b>50b)</b>. The <i>rac</i>-(<i>R</i>*,<i>R</i>*) diastereomer <b>49a</b> was subsequently oxidized to the sulfone <b>50a</b> using <i>m</i>-chloroperbenzoic acid.</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>rac</i>-(<i>R</i>*,<i>R</i>*) Acid as the Sodium Salt (<b>53</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N,N</i>-diisopropylethylamine (DIPEA), <i>n-</i>BuLi, 2-methylsulfanylacetaldehyde, tetrahydrofuran (THF), −78 °C to RT, 30 min, chromatography EtOAc/<i>n</i>-hexane, diastereoisomer <i>rac</i>-(<i>R</i>*,<i>R</i>*) <b>49a</b>, 15%, diastereoisomer <i>rac</i>-(<i>R</i>*,<i>S</i>*) <b>49b</b>, 25%; (b) <i>m</i>-chloroperbenzoic acid, CHCl<sub>3</sub>, 0 °C to RT, 16 h, 47%; (c) hydrogen, palladium hydroxide on carbon, EtOH, RT, 16 h, quant; (d) di-<i>tert</i>-butyl dicarbonate ((BOC)<sub>2</sub>O), K<sub>2</sub>CO<sub>3</sub>, THF, EtOH, RT, 3 h, quant; (e) NaOH (10 M aq), EtOH, RT, 3 h, 73%.</p></p></figure><figure id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 2-Amino-3-hydroxy-4-(methanesulfonyl)butan-1-one (<b>22</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-ethynylbenzenemethanol, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, triethylamine (TEA), dimethylformamide (DMF), 50 °C, 30 min, 40%; (b) 1,4-dioxane, 4<i>N</i> HCl in 1,4-dioxane, RT, 3 h, quant; (c) compound <b>53</b>, <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HATU), DIPEA, DMF, RT, 3 h, 96%; (d) formic acid, RT, 2 h, 35%.</p></p></figure><div class="NLM_p">Deprotection of the amine by hydrogenation was followed by <i>tert</i>-butoxycarbonyl (boc)-protection and hydrolysis of the ethyl ester to afford the intermediate acid <b>53</b>. <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows the preparation of the acetylenic portion of the compound and convergent amide coupling with intermediate <b>53</b> to make compound <b>22</b>. Sonogashira coupling with the triflate <b>54</b> (synthesis described in WO 2018/216823) followed by deprotection afforded the intermediate piperidine <b>56</b> using HCl in 1,4-dioxane. Amide coupling was performed using HATU, and the final boc-deprotection was facilitated by formic acid.</div><div class="NLM_p">The synthesis of compound <b>43</b> is outlined in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a>–<a class="ref internalNav" href="#sch5" aria-label="5">5</a>.</div><figure id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0012.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>60</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-iodophenol, NaH, DMF, 100 °C, 16 h, 75%; (b) iron powder, ammonium chloride, EtOH, Water, 70 °C, 15 h, 88%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> shows the preparation of intermediate <b>60</b> via nucleophilic displacement of the tosylate with 4-iodophenoxide, followed by reduction of the nitro group to the primary amine.</div><div class="NLM_p"><a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> shows the synthesis of chiral imidazole intermediate <b>62</b>. Condensation cyclization of (2<i>S</i>)-2-tetrahydropyran-2-yloxypropanal <b>61</b> with ammonia and glyoxal afforded the tetrahydropyran-protected intermediate that was subsequently used in the synthesis of several example compounds. <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a> shows the synthesis of compound <b>43</b> starting from commercial iodo-1,2-oxazole ester <b>63</b>. Sonogashira coupling installed the triisopropylsilyl (TIPS)-protected terminal acetylene group, and a borohydride reduction of the ester afforded the intermediate alcohol <b>65</b>. The convergent coupling step with imidazole <b>62</b> was achieved under Mitsunobu conditions. Following on from deprotection of the TIPS group to form the terminal acetylene using TBAF, a second Sonogashira coupling was performed with compound <b>60</b>. The final deprotection of the tetrahydropyranyl (THP)-protected alcohol and acetonide was performed using <i>p</i>-toluenesulfonic acid to obtain compound <b>43</b>.</div><figure id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Imidazole Compound <b>62</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 28% aqueous ammonia, glyoxal, MeOH, RT, 16 h, quant.</p></p></figure><figure id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0014.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethynyltri(propan-2-yl)silane, TEA, Pd(PPh<sub>3</sub>)<sub>4</sub>, CuI, DMF, 80 °C, 1 h, 69%; (b) NaBH<sub>4</sub>, EtOH, 0 °C, 4 h, 89%; (c) <b>62</b>, 3-(dimethylcarbamoylimino)-1,1-dimethylurea (TMAD), tributylphosphine, THF, RT, 2 h, 93%; (d) tetra-<i>n</i>-butylammonium fluoride (TBAF), THF, 0 °C, 1 h, 82%; (e) <b>60</b>, TEA, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, DMF, 80 °C, 0.5 h, 55%; (f) <i>p</i>-TsOH, MeOH, RT, 1 h, 87%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Fragment-based discovery methods have identified two non-hydroxamate series of compounds that are selective inhibitors of the zinc metalloenzyme, LpxC. Optimization of both series was guided by structure-based design to modify zinc chelation, make interactions with the substrate binding cleft, and introduce improved compound properties. The 2-(1<i>S</i>-hydroxyethyl)imidazole <b>43</b> is an advanced lead compound for which further optimization and <i>in vivo</i> efficacy will be described in a future publication.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All chemicals were used directly as received from commercial suppliers. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on JNM-ECA600 (600 MHz for <sup>1</sup>H, JEOL), JNM-ECA500 (500 MHz for <sup>1</sup>H and 125 MHz for <sup>13</sup>C, JEOL) or AVANCE III HD 400 (400 MHz for <sup>1</sup>H, BRUKER). Chemical shifts for <sup>1</sup>H and <sup>13</sup>C NMR are reported in parts per million (ppm) referenced to the residual solvent peak (dimethyl sulfoxide (DMSO)-<i>d</i><sub><i>6</i></sub>, δ = 2.50 ppm; CDCl<sub>3</sub>, δ = 7.27 ppm; CD<sub>3</sub>OD, δ = 3.30 ppm). Abbreviations used to describe the peak signals in <sup>1</sup>H NMR data are s = singlet, br = broad, d = doublet, dd = double doublet, dt = double triplet, t = triplet, q = quartet, quin = quintet, m = multiplet. All final compounds were purified to >95% by reverse-phase high-performance liquid chromatography (HPLC) or normal-phase silica gel column chromatography. The purity was assessed by reverse-phase UPLC with a gradient of 5–98% acetonitrile in water (with either an acid or a base modifier) and monitored by UV absorption at 210–450 nm.</div><div class="NLM_p">Unless otherwise described, the following conditions were used for compound synthesis, purification, and analysis. A microwave synthesizer Initiator<sup>+</sup> (Biotage) was used for the microwave reaction conditions. OH-type silica gel column chromatography was performed using SNAP Ultra (Biotage) or REVELERIS 40 μm (Grace), and amino-type silica gel chromatography was performed using SNAP Cartridge ISOLUTE Flash-NH<sub>2</sub> (Biotage) or Grace REVELERIS Amino 40 μm (Grace). The prep-HPLC purifications were performed using Agilent 1260 Infinity or Agilent 6130 (ionization method: electron spray ionization (ESI)), and Agilent 385-ELSD was used when the ELSD detector was attached.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-Amino-3-hydroxy-1-{4-[(4-{[4-(hydroxymethyl)phenyl] ethynyl}phenyl)methyl]piperidin-1-yl}-4-(methanesulfonyl)butan-1-one (<b>22</b>)</h3><div class="NLM_p last">Step 1.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Ethyl <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-(Dibenzylamino)-3-hydroxy-4-(methylthio)butanoate (49a) and Ethyl <i>rac</i>-(<i>R</i>*,<i>S</i>*)-2-(Dibenzylamino)-3-hydroxy-4-(methylthio)butanoate (<b>49b</b>)</h3><div class="NLM_p last">To a stirred solution of diisopropylamine (4.7 mL, 65 mmol) in THF (0.21 L) was added 1.6 mol/L <i>n</i>-butyllithium in hexane (20 mL, 32 mmol) at −78 °C. The mixture was allowed to warm to 0 °C, stirred for 30 min, then cooled to −78 °C, and <b>48</b> (6 g, 21 mmol) in THF was added into the mixture. After stirring for 30 min, 2-methylsulfanylacetaldehyde (2.3 g, 25 mmol) in THF was added into the mixture. The reaction mixture was allowed to warm to room temperature, stirred for further 3 h, quenched with a saturated aqueous sodium bicarbonate, and extracted with CHCl<sub>3</sub> three times. The organic layer was passed through a phase separator and concentrated in vacuo onto ISOLUTE HM-N. The residue was purified, and diastereomers were separated with OH-type silica gel column chromatography (8–31% EtOAc in <i>n</i>-hexane) to afford <i>rac</i>-(<i>R</i>*,<i>R</i>*) diastereomer (1.2 g, 15% yield) <b>49a</b> as yellow oil and <i>rac</i>-(<i>R</i>*,<i>S</i>*) diastereomer (2.0 g, 25% yield) <b>49b</b> as a yellow oil. The relative stereochemistry of both diastereomers was determined by X-ray crystallography of ethyl <i>rac</i>-(<i>R</i>*,<i>S</i>*)-2-(dibenzylamino)-3-hydroxy-4-(methylsulfonyl)butanoate <b>49b</b>. <i>rac</i>-(<i>R</i>*,<i>R</i>*) diastereomer <b>49a</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20–7.35 (m, 10H), 4.24–4.44 (m, 2H), 4.06–4.16 (m, 1H), 3.92 (d, <i>J</i> = 13.57 Hz, 2H), 3.48 (d, <i>J</i> = 13.57 Hz, 2H), 3.30 (d, <i>J</i> = 9.05 Hz, 1H), 3.02 (dd, <i>J</i> = 2.87, 14.12 Hz, 1H), 2.81 (d, <i>J</i> = 3.67 Hz, 1H), 2.30 (dd, <i>J</i> = 9.11, 14.12 Hz, 1H), 2.03 (s, 3H), 1.42 (t, <i>J</i> = 7.15 Hz, 3H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 374 [M + H]<sup>+</sup>. <i>rac</i>-(<i>R</i>*,<i>S</i>*) diastereomer <b>49b</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22–7.39 (m, 10H), 4.22–4.40 (m, 2H), 4.10–4.20 (m, 1H), 4.02 (d, <i>J</i> = 13.20 Hz, 2H), 3.66 (s, 1H), 3.53 (d, <i>J</i> = 13.20 Hz, 2H), 3.39 (d, <i>J</i> = 9.17 Hz, 1H), 2.63 (dd, <i>J</i> = 2.81, 13.94 Hz, 1H), 2.48 (dd, <i>J</i> = 6.85, 14.06 Hz, 1H), 2.02 (s, 3H), 1.40 (t, <i>J</i> = 7.15 Hz, 3H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 374 [M + H]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Ethyl <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-(Dibenzylamino)-3-hydroxy-4-(methylsulfonyl)butanoate (<b>50a</b>)</h3><div class="NLM_p last">To a stirred solution of <b>49a</b> (1.2 g, 3 mmol) in CHCl<sub>3</sub> (10 mL) was added <i>m</i>-chloroperbenzoic acid (2.0 g, 11.6 mmol) at 0 °C. The reaction mixture was stirred at room temperature for 16 h, quenched with a saturated aqueous sodium thiosulfate, and extracted with CHCl<sub>3</sub> twice. The organic layer was passed through a phase separator and concentrated in vacuo onto ISOLUTE HM-N. The residue was purified with OH-type silica gel column chromatography (8–100% EtOAc in <i>n</i>-hexane) to give the title compound (0.58 g, 47% yield) as a colorless powder. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26–7.40 (m, 10H), 4.58–4.66 (m, 1H), 4.28–4.48 (m, 2H), 3.87 (d, <i>J</i> = 13.08 Hz, 2H), 3.55 (br d, <i>J</i> = 15.04 Hz, 1H), 3.46 (d, <i>J</i> = 12.96 Hz, 2H), 3.21 (d, <i>J</i> = 9.90 Hz, 1H), 3.07–3.15 (m, 1H), 2.93 (s, 3H), 2.62 (dd, <i>J</i> = 10.39, 15.04 Hz, 1H), 1.46 (t, <i>J</i> = 7.15 Hz, 3H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 406 [M + H]<sup>+</sup>. LCMS condition: C. RT = 0.847 min.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Ethyl <i>rac</i>-(<i>R</i>*,<i>S</i>*)-2-(Dibenzylamino)-3-hydroxy-4-(methylsulfonyl)butanoate (<b>50b</b>)</h3><div class="NLM_p last">To a stirred solution of <b>49b</b> (0.12 g, 0.32 mmol) in acetonitrile (10 mL) and water (5 mL) was added oxone (0.22 g, 0.35 mmol) at 0 °C. The reaction mixture was stirred at room temperature for 16 h, quenched with a saturated aqueous sodium thiosulfate, and extracted with CHCl<sub>3</sub> twice. The organic layer was passed through a phase separator and concentrated in vacuo. The residue was purified with OH-type silica gel column chromatography (0–100% EtOAc in <i>n</i>-hexane) to give the title compound (80 mg, 61% yield) as a colorless powder. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.24–7.39 (m, 10H), 4.26–4.47 (m, 3H), 4.02 (br d, <i>J</i> = 13.08 Hz, 2H), 3.57 (br d, <i>J</i> = 13.20 Hz, 2H), 3.27–3.41 (m, 1H), 3.09 (dd, <i>J</i> = 8.50, 14.86 Hz, 1H), 2.94–3.02 (m, 1H), 2.91 (s, 3H), 1.42 (t, <i>J</i> = 7.09 Hz, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 406 [M + H]<sup>+</sup>. X-ray: A single crystal was grown from ethanol. The crystal details were as follows: triclinic space group <i>P</i>1̅, <i>a</i> = 11.9376(2) Å, <i>b</i> = 13.4893(3) Å, <i>c</i> = 15.7344(3) Å, α = 69.216(5) °, β = 83.333(6) °; γ = 88.323(6) °; <i>R</i> and <i>w<i>R</i></i> values were 0.0931 and 0.3597, respectively.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Ethyl <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-Amino-3-hydroxy-4-(methylsulfonyl)butanoate (<b>51</b>)</h3><div class="NLM_p last">To a solution of <b>50a</b> (0.40 g, 1 mmol) in EtOH (9.9 mL) was added 10% palladium hydroxide on carbon (50 mg, catalytic). The mixture was stirred at room temperature for 16 h under a hydrogen atmosphere. The reaction mixture was filtered, and the filtrate was concentrated in vacuo to give crude <b>51</b> (0.24 g, quant.) as oil, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.41 (ddd, <i>J</i> = 2.14, 5.50, 9.48 Hz, 1H), 4.22–4.29 (m, 1H), 3.72 (q, <i>J</i> = 6.97 Hz, 1H), 3.64 (d, <i>J</i> = 5.50 Hz, 1H), 3.27 (dd, <i>J</i> = 9.90, 15.04 Hz, 1H), 3.19 (br d, <i>J</i> = 14.70 Hz, 1H), 3.04 (s, 3H), 1.32 (t, <i>J</i> = 7.15 Hz, 3H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 226 [M + H]<sup>+</sup>. LCMS condition: B. RT = 0.258 min.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Ethyl <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-((<i>tert</i>-Butoxycarbonyl)amino)-3-hydroxy-4-(methylsulfonyl) butanoate (<b>52</b>)</h3><div class="NLM_p last">To a stirred solution of <b>51</b> (0.17 g, 0.75 mmol as the crude material) in THF (3.8 mL) and EtOH (1.5 mL) were added (BOC)<sub>2</sub>O (0.20 g, 0.92 mmol) and potassium carbonate (0.27 g, 1.95 mmol). After stirring for 3 h at room temperature, the reaction mixture was quenched with water and extracted with CHCl<sub>3</sub> three times. The organic layer was concentrated in vacuo onto ISOLUTE HM-N. The residue was purified with OH-type silica gel column chromatography (30–100% EtOAc in <i>n</i>-hexane) to give <b>52</b> (0.27 g, quant.) as colorless oil. MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 348 [M + Na]<sup>+</sup>. LCMS condition: B. RT = 0.635 min.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Sodium <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-((<i>tert</i>-Butoxycarbonyl)amino)-3-hydroxy-4-(methylsulfonyl) butanoate (<b>53</b>)</h3><div class="NLM_p last">To a solution of <b>52</b> (0.28 g) in EtOH (2.9 mL) was added 10 mol/L aqueous sodium hydroxide (0.26 mL, 2.6 mmol). After stirring for 3 h at room temperature, the resulting precipitate was collected by filtration and air-dried to afford the title compound (0.20 g, 73% yield) as a colorless solid. MS (ESI<sup>–</sup>): <i>m</i>/<i>z</i> 296 [M – H]<sup>−</sup>. LCMS condition: A. RT = 0.879 min.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>tert</i>-Butyl 4-(4-((4-(Hydroxymethyl)phenyl)ethynyl)benzyl)piperidine-1-carboxylate (<b>55</b>)</h3><div class="NLM_p last">A mixture of <b>54</b> (0.81 g, 1.9 mmol), (4-ethynylphenyl)methanol (0.76 g, 5.7 mmol), CuI (36 mg, 0.19 mmol), Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (0.13 g, 0.19 mmol), and Et<sub>3</sub>N (10 mL, 72 mmol) in DMF (10 mL) was stirred under a nitrogen atmosphere at 60 °C for 7 h. After cooling to RT, the mixture was passed through the Celite and washed with EtOAc. The filtrate was concentrated under reduced pressure. The residue was purified with OH-type silica gel chromatography (7–30% EtOAc/<i>n</i>-hexane) to give the title compound (0.16 g, 20% yield) as a beige solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.52 (d, <i>J</i> = 8.07 Hz, 2H), 7.45 (d, <i>J</i> = 8.07 Hz, 2H), 7.35 (d, <i>J</i> = 8.07 Hz, 2H), 7.12 (d, <i>J</i> = 8.07 Hz, 2H), 4.72 (br s, 2H), 3.95–4.18 (m, 2H), 2.58–2.70 (m, 2H), 2.55 (d, <i>J</i> = 6.97 Hz, 2H), 1.58–1.70 (m, 3H), 1.45 (s, 9H), 1.07–1.22 (m, 2H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 428 [M + Na]<sup>+</sup>. LCMS condition: C. RT = 1.024 min.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (4-((4-(Piperidin-4-ylmethyl)phenyl)ethynyl)phenyl)methanol Hydrochloride (<b>56</b>)</h3><div class="NLM_p">To a solution of <b>55</b> (0.14 g, 0.35 mmol) in 1,4-dioxane (4 mL) was added 4 mol/L HCl in 1,4-dioxane (8 mL) at RT. The reaction mixture was stirred at RT for 4 h and concentrated in vacuo to give the title compound (0.13 g, quant.) as a pale-yellow solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.49 (d, <i>J</i> = 8.07 Hz, 2H), 7.44 (d, <i>J</i> = 8.07 Hz, 2H), 7.36 (d, <i>J</i> = 8.07 Hz, 2H), 7.20 (d, <i>J</i> = 8.07 Hz, 2H), 5.27 (br t, <i>J</i> = 4.58 Hz, 1H), 4.52 (br d, <i>J</i> = 4.28 Hz, 2H), 2.87 (br d, <i>J</i> = 12.23 Hz, 2H), 2.32–2.41 (m, 2H), 1.39–1.62 (m, 3H), 1.03 (dq, <i>J</i> = 3.91, 11.98 Hz, 2H), (two proton signals overlap with the DMSO solvent peak). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 306 [M + H]<sup>+</sup>. LCMS condition: B. Retention time (RT) = 0.543 min.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>rac</i>-(<i>R</i>*,<i>R</i>*)-<i>tert</i>-Butyl (3-Hydroxy-1-(4-(4-((4-(hydroxymethyl)phenyl) ethynyl)benzyl)piperidin-1-yl)-4-(methylsulfonyl)-1-oxobutan-2-yl)carbamate (<b>57</b>)</h3><div class="NLM_p last">To a stirred solution of <b>56</b> (34 mg, 0.1 mmol) in DMF (2.0 mL), <b>53</b> (35 mg, 0.11 mmol), HATU (57 mg, 0.15 mmol), and DIPEA (52 μL, 0.54 mmol) were added. The mixture was stirred for 3 h at RT. The reaction mixture was purified via prep-HPLC to give the title compound (56 mg, 96% yield) as an amorphous solid. MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 607 [M + Na]<sup>+</sup>. LCMS condition: C. RT = 0.786 min.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>rac</i>-(<i>R</i>*,<i>R</i>*)-2-Amino-3-hydroxy-1-{4-[(4-{[4-(hydroxymethyl)phenyl]ethynyl} phenyl)methyl]piperidin-1-yl}-4-(methanesulfonyl)butan-1-one (<b>22</b>)</h3><div class="NLM_p last">A solution of <b>57</b> (53 mg, 0.9 mmol) in formic acid (1.5 mL, 40 mmol excess) was stirred for 2 h at RT. The reaction mixture was diluted with CHCl<sub>3</sub> and concentrated in vacuo. To the residue was added 7 mol/L ammonia in MeOH (2 mL, excess), and the mixture was stirred for 10 min at RT. The reaction mixture was concentrated in vacuo and purified with prep-HPLC to give the title compound (11 mg, 35% yield) as a colorless powder. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 7.47 (d, <i>J</i> = 8.23 Hz, 2H), 7.43 (d, <i>J</i> = 7.89 Hz, 2H), 7.36 (d, <i>J</i> = 8.23 Hz, 2H), 7.20 (br d, <i>J</i> = 7.89 Hz, 2H), 4.62 (s, 2H), 4.42–4.57 (m, 3H), 3.92–4.00 (m, 1H), 3.36–3.47 (m, 1H), 3.11–3.23 (m, 1H), 2.94–3.10 (m, 4H), 2.53–2.77 (m, 3H), 1.84–1.98 (m, 1H), 1.75 (br t, <i>J</i> = 12.60 Hz, 2H), 1.09–1.33 (m, 2H). <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 163.2, 162.9, 143.4, 142.1, 141.9, 132.6, 132.6, 130.6, 130.5, 128.1, 123.7, 123.6, 122.6, 122.5, 90.1, 90.0, 66.4, 66.4, 64.9, 57.1, 57.0, 56.7, 47.4, 47.2, 44.2, 43.6, 43.4, 43.3, 39.2, 39.0, 33.7, 33.7, 33.0, 32.8, 31.1 (35 carbon signals are observed because of the rotational isomer). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 485 [M + H]<sup>+</sup>. high-resolution mass spectrometry (HRMS) (ESI<sup>+</sup>/APCI) calcd. for [C<sub>26</sub>H<sub>32</sub>N<sub>2</sub>O<sub>5</sub>S + H]<sup>+</sup>: 485.2105; found: 485.2089. LCMS condition: B. RT = 0.590 min.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-Amino-2-[(4-{[3-({2-[(1<i>S</i>)-1-hydroxyethyl]-1<i>H</i>-imidazol-1-yl}methyl)-1,2-oxazol-5-yl]ethynyl}phenoxy)methyl]propane-1,3-diol (<b>43</b>)</h3><div class="NLM_p last">Step 1.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 5-[(4-Iodophenoxy)methyl]-2,2-dimethyl-5-nitro-1,3-dioxane (<b>59</b>)</h3><div class="NLM_p last">To a solution of 4-iodophenol (2.1 g, 9.6 mmol) in DMF (96 mL) was added NaH (0.50 g, 12 mmol) and stirred for 1.5 h at room temperature. To the reaction mixture was added <b>58</b> (3.6 g, 11 mmol) and stirred for an additional 16 h at 100 °C. The resulting mixture was diluted with EtOAc and washed with water. The organic layer was dried over MgSO<sub>4</sub> and concentrated in vacuo. The residue was triturated with diethyl ether and collected by filtration to afford the title compound (2.8 g, 75% yield) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.57 (d, <i>J</i> = 8.93 Hz, 2H), 6.67 (d, <i>J</i> = 8.93 Hz, 2H), 4.45 (d, <i>J</i> = 12.72 Hz, 2H), 4.42 (s, 2H), 4.15 (d, <i>J</i> = 12.59 Hz, 2H), 1.46 (s, 3H), 1.43 (s, 3H).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 5-[(4-Iodophenoxy)methyl]-2,2-dimethyl-1,3-dioxan-5-amine (<b>60</b>)</h3><div class="NLM_p last">To a solution of <b>59</b> (0.20 g, 0.51 mmol) in EtOH (2.0 mL) and water (1.0 mL) were added ammonium chloride (41 mg, 0.77 mmol) and iron powder (0.17 g, 3.1 mmol). After being stirred for 15 h at 70 °C, the reaction mixture was filtered through Celite. The filtrate was concentrated in vacuo. The residue was diluted with EtOAc and washed with saturated sodium bicarbonate. The organic layer was dried over MgSO<sub>4</sub> and concentrated in vacuo. The residue was purified with amino-type silica gel column chromatography (10–40% EtOAc in <i>n</i>-hexane) to afford the title compound (0.16 g, 88% yield) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.56 (d, <i>J</i> = 8.80 Hz, 2H), 6.71 (d, <i>J</i> = 8.80 Hz, 2H), 3.96 (s, 2H), 3.89 (d, <i>J</i> = 11.62 Hz, 2H), 3.63 (d, <i>J</i> = 11.62 Hz, 2H), 1.47 (s, 3H), 1.42 (s, 3H).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Ethyl 5-{[Tri(propan-2-yl)silyl]ethynyl}-1,2-oxazole-3-carboxylate (<b>64</b>)</h3><div class="NLM_p last">A round–bottomed flask was charged with <b>63</b> (18 g, 67 mmol), ethynyltri(propan-2-yl)silane (12 g, 67 mmol), DMF (52 mL), and TEA (73 mL, 1 mol, in excess). The flask was evacuated and backfilled with nitrogen, followed by addition of Pd(PPh<sub>3</sub>)<sub>4</sub> (2.4 g, 2 mmol) and CuI (0.80 g, 4.2 mmol). After being stirred for 1 h at 80 °C, the mixture was poured into a half-saturated aqueous ammonium chloride and extracted three times with EtOAc-hexane (1:1). The combined organic layer was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo. The residue was purified with OH-type silica gel column chromatography (0–15% EtOAc in <i>n</i>-hexane) to give the title compound (12 g, 69% yield) as a pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.81 (s, 1H), 4.45 (q, <i>J</i> = 7.1 Hz, 2H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H), 1.09–1.23 (m, 21H). MS (ESI<sup>+</sup>/APCI): <i>m</i>/<i>z</i> 322 [M + H]<sup>+</sup>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (5-{[Tri(propan-2-yl)silyl]ethynyl}-1,2-oxazol-3-yl)methanol (<b>65</b>)</h3><div class="NLM_p last">To a cooled (0 °C) solution of <b>64</b> (10 g, 32 mmol) in EtOH (53 mL) and THF (103 mL) was added NaBH<sub>4</sub> (2.4 g, 64 mmol) in portions while keeping the internal temperature below 5 °C. After stirring for 3 h at 0 °C, half-saturated aqueous ammonium chloride was added to the reaction mixture while keeping the internal temperature below 15 °C. The mixture was stirred for 30 min at room temperature and concentrated under reduced pressure, and the resulting aqueous solution was extracted three times with CHCl<sub>3</sub>. The combined organic layer was washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. The residue was purified with OH-type silica gel column chromatography (10–50% EtOAc in <i>n</i>-hexane) to give the title compound (7.9 g, 89% yield) as a pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.49 (s, 1H), 4.77 (s, 2H), 1.02–1.25 (m, 21H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 280 [M + H]<sup>+</sup>. LCMS condition: C. RT = 1.078 min.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 3-[(2-{(1<i>S</i>)-1-[(Oxan-2-yl)oxy]ethyl}-1<i>H</i>-imidazol-1-yl)methyl]-5-{[tri(propan-2-yl)silyl]ethynyl}-1,2-oxazole (<b>66</b>)</h3><div class="NLM_p last">To a stirred solution of <b>65</b> (1.4 g, 4.8 mmol) and <b>62</b> (1.1 g, 5.8 mmol) in THF (19 mL) were added TMAD (1.1 g, 6.3 mmol) and tributylphosphine (1.3 mL, 6.3 mmol) at 0 °C under a nitrogen atmosphere, and the mixture was stirred for 3.5 h at RT. The reaction mixture was quenched by water and extracted three times with EtOAc. The organic layer was dried over MgSO<sub>4</sub> and concentrated in vacuo onto ISOLUTE HM-N. The residue was purified with OH-type silica gel column chromatography (40–80% EtOAc in <i>n</i>-hexane) to afford the title compound <b>66</b> (1.5 g, 68% yield) as a yellow oil. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.85–7.08 (m, 2H), 6.16–6.35 (m, 1H), 5.29–5.50 (m, 2H), 5.06–5.19 (m, 1H), 4.46–4.75 (m, 1H), 3.81–3.95 (m, 1H), 3.42–3.56 (m, 1H), 1.05–1.85 (m, 30H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 458 [M + H]<sup>+</sup>. LCMS condition: B. RT = 1.096 min.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-Ethynyl-3-[(2-{(1<i>S</i>)-1-[(oxan-2-yl)oxy]ethyl}-1<i>H</i>-imidazol-1-yl)methyl]-1,2-oxazole (<b>67</b>)</h3><div class="NLM_p last">To a cooled (0 °C) solution of <b>66</b> (12 g, 26 mmol) in THF (0.18 L) was added TBAF (28 mL, 1 mol/L solution in THF), while the internal temperature was maintained below 5 °C. After stirring for 1 h at 0 °C, the mixture was concentrated, and the residue was diluted with EtOAc followed by washing with water and brine. The organic phase was separated, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified with OH-type silica gel column chromatography (50–100% EtOAc in n-hexane and then 0–10% MeOH in EtOAc) to give the title compound (6.5 g, 82% yield) as a brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.99–7.05 (m, 1H), 6.87 (br s, 1H), 6.25–6.40 (m, 1H), 5.31–5.51 (m, 2H), 3.81–3.98 (m, 1H), 5.06–5.15 (m, 1H), 4.46–4.75 (m, 1H), 3.61–3.65 (m, 1H), 3.42–3.55 (m, 1H), 1.70–1.84 (m, 1H), 1.37–1.66 (m, 8H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 302[M + H]<sup>+</sup>. LCMS condition: B. RT = 0.415 min.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 2,2-Dimethyl-5-[[4-[2-[3-[[2-[(1<i>S</i>)-1-tetrahydropyran-2-yloxyethyl]imidazol-1-yl]methyl]isoxazol-5-yl]ethynyl]phenoxy]methyl]-1,3-dioxan-5-amine (<b>68</b>)</h3><div class="NLM_p last">A round-bottomed flask was charged with <b>67</b> (110 mg, 0.37 mmol), <b>60</b> (110 mg, 0.3 mmol), PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (11 mg, 0.015 mmol), and CuI (5.8 mg, 0.03 mmol). The flask was evacuated and backfilled with nitrogen followed by addition of DMF (3.0 mL) and TEA (0.52 mL, 0.7 mmol). The mixture was placed into a preheated oil bath (50 °C) and stirred for 30 min. The reaction mixture was diluted with CHCl<sub>3</sub>, followed by washing with 10% aqueous ammonium hydroxide and brine. The organic phase was separated, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified with amino-type silica gel chromatography (20–100% EtOAc in <i>n</i>-hexane) to afford the title compound (90 mg, 55% yield) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.46–7.54 (m, 2H), 6.99–7.08 (m, 1H), 6.88–6.98 (m, 3H), 6.19–6.34 (m, 1H), 5.32–5.49 (m, 2H), 5.13 (quin, <i>J</i> = 6.85 Hz, 1H), 4.47–4.77 (m, 1H), 4.04 (s, 2H), 3.43–3.95 (m, 6H), 1.37–1.89 (m, 15H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 537 [M + H]<sup>+</sup>. LCMS condition: A. RT = 0.835 min.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 2-Amino-2-[(4-{[3-({2-[(1<i>S</i>)-1-hydroxyethyl]-1<i>H</i>-imidazol-1-yl}methyl)-1,2-oxazol-5-yl]ethynyl}phenoxy)methyl]propane-1,3-diol (<b>43</b>)</h3><div class="NLM_p">To a solution of <b>68</b> (0.11 g, 0.20 mmol) in MeOH (2.0 mL) and water (0.20 mL) was added p-TsOH monohydrate (91 mg, 0.48 mmol) at RT. After being stirred for 20 h, the reaction mixture was basified with saturated aqueous sodium bicarbonate at 0 °C and extracted with chloroform. The organics were combined and passed through a phase separator and concentrated in vacuo. The residue was purified with amino-type silica gel chromatography (10% MeOH in chloroform) to give the title compound (61 mg, 75% yield) as a colorless amorphous solid.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub><i>6</i></sub>) δ 7.57 (d, <i>J</i> = 9.17 Hz, 2H), 7.16 (d, <i>J</i> = 1.15 Hz, 1H), 7.03 (d, <i>J</i> = 8.79 Hz, 2H), 6.81 (d, <i>J</i> = 1.15 Hz, 1H), 6.79 (s, 1H), 5.38–5.47 (m, 3H), 4.85 (quin, <i>J</i> = 6.40 Hz, 1H), 4.61 (br s, 2H), 3.82 (s, 2H), 3.33–3.40 (m, 2H), 1.46 (d, <i>J</i> = 6.88 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub><i>6</i></sub>) δ 161.6, 160.6, 152.9, 149.7, 133.6, 126.5, 120.9, 115.2, 111.1, 107.5, 99.0, 74.1, 69.6, 63.0, 61.4, 56.4, 40.5, 21.8. HRMS (ESI<sup>+</sup>/APCI) calcd for [C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>5</sub> + H]<sup>+</sup>: 413.1819; found: 413.1800.</div><div id="sec4_20_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> General Experimental Details for Assays and Structure Determination</h4><div class="NLM_p">UDP-3-<i>O</i>-(<i>R</i>-3-hydroxydecanoyl)-<i>N</i>-acetylglucosamine was obtained from Alberta Research Council (Alberta, Canada) and stored as 100mM stock dissolved in DMSO at −80 °C.</div><div class="NLM_p">Fluorescamine was obtained from FUJIFILM Wako Pure Chemical Corporation or Sigma-Aldrich.</div><div class="NLM_p">Uridine diphosphate <i>N</i>-acetylglucosamine (UDP-GlcNAc) was stored as 200 μM stock dissolved in methanol–acetonitrile (1:1) at −20 °C.</div><div class="NLM_p last">Compounds were stored as 10 mM, 20 mM, 50 mM, or 200 mM stock in DMSO.</div></div><div id="sec4_20_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Protein Production</h4><div id="sec4_20_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> PaLpxC(1-299)C40S</h5><div class="NLM_p">Plasmids encoding P. aeruginosa LpxC(1-299) with a mutation, C40S, were obtained from ATUM, in the T7 expression vector pJ411 (ATUM).</div><div class="NLM_p">MIKQRTLKNIIRATGVGLHSGEKVYLTLKPAPVDTGIVFSRTDLDPVVEIPARAENVGETTMSTTLVKGDVKVDTVEHLLSAMAGLGIDNAYVELSASEVPIMDGSAGPFVFLIQSAGLQEQEAAKKFIRIKREVSVEEGDKRAVFVPFDGFKVSFEIDFDHPVFRGRTQQASVDFSSTSFVKEVSRARTFGFMRDIEYLRSQNLALGGSVENAIVVDENRVLNEDGLRYEDEFVKHKILDAIGDLYLLGNSLIGEFRGFKSGHALNNQLLRTLIADKDAWEVVTFEDARTAPISYMRP</div><div class="NLM_p">Plasmids were transformed into <i>E. coli</i> BL21(DE3) pLysS cells. Cultures were grown at 37 °C in Terrific Broth with Kanamycin (100 μg/mL) until the OD<sub>600</sub> was 0.7 at which time the temperature was reduced to 30 °C and expression induced by addition of isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside (IPTG) to a final concentration of 1 mM in the presence of 0.1 mM ZnCl<sub>2</sub> or ZnSO<sub>4</sub>. Cells were harvested by centrifugation after 2 h.</div><div class="NLM_p">For <sup>15</sup>N-labeled PaLpxC, the transformed cells were grown in EnpressoB, Nitrogen Free (Enpresso), supplemented with 2.5 g/L <sup>15</sup>N-(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (Goss Scientific) induced as above and harvested after 3 h.</div><div class="NLM_p last">Cells were resuspended in 3 volumes of 20 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES) (pH 7.0) and 2 mM DTT and lysed by passing three times through a Stanstead Homogenizer (Stanstead). The lysate was clarified by centrifugation and applied to CaptoQ, eluting in a gradient to 1 M NaCl. (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> was added to a final concentration of 1 M, and the LpxC was further purified by hydrophobic interaction chromatography using HiTrap Butyl HP (GE Healthcare) with a gradient of 1–0 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> and finally Superdex200pg (GE Healthcare), equilibrated in 20 mM HEPES (pH 7.4), 150 mM NaCl, and 2 mM DTT.</div></div><div id="sec4_20_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> PaLpxC(1-303-His)</h5><div class="NLM_p last">The gene fragment encoding residues 2-303 of PaLpxC was amplified using primers with NdeI and NotI sites and subcloned into pET21a vectors. The protein was expressed in <i>E. coli</i> BL21(DE3)pLysS cells by inducing with 1.0 mM IPTG and 0.2 mM ZnSO<sub>4</sub> at OD<sub>600</sub> = 0.5 for 7 h at 18 °C. The protein was purified using TALON metal affinity resin (TaKaRa) and buffer-exchanged against 40 mM HEPES (pH 8.0) using PD-10 (GE Healthcare).</div></div><div id="sec4_20_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> TSA Methods</h5><div class="NLM_p">Fluorescence was measured on an Agilent MX300P Real-Time PCR with a heating rate of 1 °C per minute and Ex/Em 492/610 nm. Final concentrations used were 5 μM PaLpxC, 5× Sypro Orange, 40 mM HEPES (pH 8.0), and 2% DMSO in a final volume of 40 μL. Protein denaturation curves were measured in the absence and presence of low nM affinity compounds <b>1</b> (PF-5081090), which were added at a 10-fold excess of the protein concentration.</div><div class="NLM_p last">When evaluating compounds, fluorescence was measured on an Applied Biosystems 7500 Fast Real-Time PCR system with a heating rate of 0.96 °C per minute (ramp rate: 1%) and the ROX (carboxy-X-rhodamine) detector. Final concentrations used were 5 μM PaLpxC(1-303-His), 1× Protein Thermal Shift Dye (Life Technologies), 40 mM HEPES (pH 8.0), 10 or 40 μM ZnCl<sub>2</sub>, and 5% DMSO in a final volume of 20 μL. Protein denaturation curves were measured in the absence and presence of compounds which were added at a more than 10-fold excess of the protein concentration.</div></div></div><div id="sec4_20_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Crystallization and Structure Determination by X-ray Crystallography</h4><div id="sec4_20_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Co-Crystallization to Establish Initial Crystals</h5><div class="NLM_p last">PaLpxC at 10.6 mg/mL was mixed with compound <b>47</b> (see SI) at 4-fold molar excess. The complex crystallized in 0.1 M Tris (pH 8.5), 30% poly(ethylene glycol) 4 K, and 0.2 M MgCl<sub>2</sub> using the sitting-drop vapor-diffusion method at 19 °C. The co-crystals took 2 weeks to grow. The compound binds in the tunnel region, making no contact with zinc.</div></div><div id="sec4_20_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Protein–Ligand Complexes Containing <b>4</b>, <b>5</b>, and <b>24</b></h5><div class="NLM_p last">Co-crystals containing <b>47</b> were soaked out and compounds 4, 5, or 24 subsequently soaked in, also overnight. The protein crystals were flash-frozen at 100 K using a cryo-protection buffer consisting of mother liquor with 20–25% glycerol added.</div></div><div id="sec4_20_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Protein–Ligand Complexes Containing Other Compounds</h5><div class="NLM_p last">Details of the conditions under which crystals were obtained by co-crystallization of individual compounds are in SI.</div></div><div id="sec4_20_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Data Collection</h5><div class="NLM_p last">Diffraction data of both the soaked and co-crystals were collected at 100 K at either Diamond Light Source (UK), Soleil Synchrotron (France), in-house on a Bruker D8 Venture TXS Generator with a Bruker Photon 100 detector, or in-house on a Micro Max 007 HF Generator (Rigaku) with an R-AXIS VII detector (Rigaku) or PILATUS 200K (Rigaku). Data processing was carried out with XDS<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>, SAINT and SADABS (Bruker AXS Inc., Madison, Wisconsin), CrystalClear (Rigaku) or CrysAlisPro (Rigaku). Data collection statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Table S2</a>.</div></div><div id="sec4_20_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> Structure Determination</h5><div class="NLM_p last">Crystallographic data was processed and structures determined and refined within the CCP4 package<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> using Sketcher (CCP4) or ProDrg<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> to build models of each compound. The structures were determined by molecular replacement using the published structure (PDBcode <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM">3UHM</a>) to calculate model phases and subsequently refined using REFMAC5.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Interactive graphical model building was carried out with COOT.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In all structures, the respective ligands were clearly defined by the initial electron density maps; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">Table S2</a> includes an image of the 2Fo-Fc map around each ligand.</div></div></div><div id="sec4_20_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> NMR Methods</h4><div class="NLM_p">All protein NMR spectra were recorded on a 600 MHz Bruker Avance III spectrometer fitted with a cryoprobe.</div><div class="NLM_p"><sup>1</sup>H–<sup>15</sup>N HSQC NMR spectra were recorded with <sup>15</sup>N-labeled PaLpxC at 250 μM in 20 mM HEPES (pH 7.4), 150 mM NaCl, and 5% D<sub>2</sub>O.</div><div class="NLM_p last">Ligand-observed fragment screening was performed with 500 μM of each compound, 10 μM PaLpxC in 20 mM HEPES (pH 7.4) and, 15 μM NaCl, in mixtures of six compounds per sample as described previously.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a></div></div><div id="sec4_20_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Computational Methods</h4><div class="NLM_p">Compounds and proteins were modeled using the Schrödinger Suite of programs (mmshare-v32018:mmshare-v3.5, Schrödinger, LLC, New York, NY). Protein structure preparation (PrepWizard), ligand preparation (Ligprep), and generation of docking grids were run with defaults. Docking was carried out using Glide, with and without constraints in the positions of the atoms chelating the zinc.</div><div class="NLM_p last">Calculations of pKa were performed with the ACD/Percepta software (ACD/Labs 2017.13, Advanced Chemistry Development, Inc) packaged in Classic mode, not using GALAS.</div></div><div id="sec4_20_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Fluorescence Polarization (FP) Assay</h4><div class="NLM_p">Compounds were screened in a 50 μL assay containing 20 nM PaLpxC or PaLpxC(1-303-His), 5 nM fluorescein probe <b>5</b>, 40 mM HEPES (pH 7.5), 150 mM NaCl, 0.02% Tween 20, with or without 25 nM ZnCl<sub>2</sub>, and 5% DMSO. PaLpxC and compounds were preincubated for 30 minutes prior to addition of the fluorescein probe, and the assay was then incubated for a further 2 h at room temperature. Plates were read on either a Biotek Synergy Neo Plate reader Ex/Em 485(20)/528(20) nm, Envision 2101 (Ex/Em 480/535 nm), or PHERAStar FSX (Ex/Em 485/535 nm) in conjunction with polarizing filters.</div><div class="NLM_p last">Inhibition values were calculated from control DMSO samples containing no compound (0% inhibition) and samples containing no enzyme (100% inhibition). Data reported in the tables is the geometric mean of at least two determinations; compound <b>69</b> or BB-78485<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was included in assays as a control with IC<sub>50</sub> values in the range 0.10–0.40 and 0.015–0.060 μM, respectively. The synthesis of <b>69</b> is described in the Supporting Information.</div></div><div id="sec4_20_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Fluorescamine-Based Functional Activity Assay</h4><div class="NLM_p">Imidazole series compounds were assayed with 3.6 nM PaLpxC(1-303-His), 20 μM UDP-3-<i>O</i>-(<i>R</i>-3-hydroxydecanoyl)-<i>N</i>-acetylglucosamine (substrate), 40 mM HEPES (pH 8.0), 25 nM ZnCl<sub>2</sub>, 0.02% Brij-35, and 5% DMSO. Assay plates were incubated for 60 min at room temperature and terminated by the addition of 1.0 mg/mL fluorescamine in 1:1 dimethylformamide/acetonitrile followed by 0.2 M sodium phosphate buffer (pH 8.0) and incubated. Plates were read on an Enspire: Ex/Em 390/495 nm.</div><div class="NLM_p">Inhibition values were calculated from control DMSO samples containing no compound (0% inhibition) and samples containing no PaLpxC(1-303-His) (100% inhibition); compound <b>69</b> or BB-78485<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was included in assays with IC<sub>50</sub> values in the range 0.30–1.2 and 0.010–0.040 μM, respectively. Data reported in tables are the geometric mean of at least two determinations.</div><div class="NLM_p last">This assay measures the extent to which LpxC catalyzes the removal of the acetyl group from the acetylglucosamine by the reaction of the resulting free amino group with fluorescamine. It is therefore not suitable for the glycine series.</div></div><div id="sec4_20_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Matrix Metalloprotease (MMP) Assays</h4><div class="NLM_p last">The MMP inhibitor profiling kit, Fluorometric RED (Enzo Life Sciences, Inc., Farmingdale, NY), was used for the human MMP inhibition assays. The inhibitory activities of compounds against MMP-2, 3, and 9 were evaluated as described in the kit protocol with some modifications. In brief the assay volume was modified to 50 μL and final enzyme concentrations of MMP-2, 3, and 9 were modified to 18.3, 21.0, and 5.67 mU/μL, respectively. Fluorescence (Ex/Em 545/576 nm) was detected using the CLARIOstar (BMG Labtech). Ilomastat (GM-6001)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> is an MMP inhibitor used as a positive control. Values reported are the geometric mean of two determinations.</div></div><div id="sec4_20_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Mass-Spectrometry-Based Functional Activity Assay</h4><div class="NLM_p">Glycine series compounds were screened with 3.6 nM PaLpxC(1-303-His), 20 μM UDP-3-<i>O</i>-(<i>R</i>-3-hydroxydecanoyl)-<i>N</i>-acetylglucosamine (substrate), 40 mM HEPES (pH 8.0), 25 nM ZnCl<sub>2</sub>, 0.02% Brij-35, and 5% DMSO. Samples were incubated for 60 min at room temperature, and the reaction was terminated by the addition of four times the volume of acetonitrile containing 25 μM UDP-GlcNAc as the internal standard. The mixture was centrifuged, and the supernatant was injected into liquid chromatography-tandem mass spectrometry (LC/MS/MS). The samples were separated with Inertsil Amide (3.0 μm, 50 mm × 2.1 mm I.D., GLScience, Japan) with a Shimadzu HPLC system. The mobile phase was 8 mM ammonium acetate containing 72% acetonitrile, and the flow rate was 0.2 mL/min. MS/MS detection of each component (<i>m</i>/<i>z</i> 734 and <i>m</i>/<i>z</i> 385) was performed using a TSQ Quantum system (Thermo Fisher Scientific) or a QTRAP-5500 system (Sciex) with the electrospray interface in negative ion detection mode.</div><div class="NLM_p last">Inhibition values were calculated from product signals of control DMSO samples containing no compound (0% inhibition) and samples containing no substrate (100% inhibition). The control compound <b>69</b> was included in assays with IC<sub>50</sub> values in the range of 0.25–1.0 μM. Data reported in tables are the geometric mean of at least two determinations.</div></div><div id="sec4_20_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Evaluation of Antibacterial Activity</h4><div class="NLM_p last">The minimum inhibitory concentration (MIC) was determined using the Clinical and Laboratory Standards Institute (CLSI) methodology, described in a CLSI document.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><i>P. aeruginosa</i> 5567 was a clinical isolate obtained in Japan. Where used, the PAβN concentration was 50 μg/mL for <i>P. aeruginosa</i> and 200 μg/mL for <i>E. coli</i>; the HSA concentration was 5% w/v.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01215" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01215?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01215</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis of compounds <b>9</b>–<b>24</b> and <b>26</b>–<b>32</b>; structural biology data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf">jm0c01215_si_001.pdf (2.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_002.csv">jm0c01215_si_002.csv (2.78 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The refined coordinates of the structures presented in this manuscript are deposited at the RCSB with the following codes: <b>6</b>: 7CIC, <b>7</b>: 7CIB, <b>8:</b> 7CIA, <b>10</b>: 7CIE, <b>14</b>: 7CI4, <b>16</b>: 7CI5, <b>22</b>: 7CI7, <b>27</b>: 7CID, <b>29</b>: 7CI6, <b>38</b>: 7CI8, and <b>43</b>: 7CI9. The authors will release the atomic coordinates and experimental data upon article publication</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01215" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00147" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00147" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yousuke Yamada</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a7dec88adec6cac6c3c6e7d3c6ced4cfc889c4c889cdd7"><span class="__cf_email__" data-cfemail="c5bcaae8bca4a8a4a1a485b1a4acb6adaaeba6aaebafb5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roderick E. Hubbard</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c6b4e8aeb3a4a4a7b4a286b0a3b4a8a7aaafb5e8a5a9ab"><span class="__cf_email__" data-cfemail="dfadf1b7aabdbdbeadbb9fa9baadb1beb3b6acf1bcb0b2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hajime Takashima</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Lee Walmsley</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fumihito Ushiyama</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yohei Matsuda</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harumi Kanazawa</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toru Yamaguchi-Sasaki</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nozomi Tanaka-Yamamoto</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junya Yamagishi</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Risa Kurimoto-Tsuruta</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuya Ogata</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Norikazu Ohtake</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hayley Angove</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa Baker</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Harris</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alba Macias</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan Robertson</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allan Surgenor</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hayato Watanabe</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Koichiro Nakano</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masashi Mima</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kunihiko Iwamoto</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsushi Okada</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iichiro Takata</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kosuke Hitaka</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akihiro Tanaka</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kiyoko Fujita</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hiroyuki Sugiyama</span> - <span class="hlFld-Affiliation affiliation">Taisho
Pharmaceutical Co., Ltd., Saitama 331-9530, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written by R.E.H., D.L.W., and Y.Y. with contributions from other authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3365-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66613" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66613" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Kanako Hatanaka, Ryo Takahashi, Maho Honda-Yamagishi, Takanori Kawaguchi, Mitsukane Yoshinaga, Takashi Yoshizumi, Shingo Sekine, Shinichi Nishimoto, Takehiko Kato, and Tomoyuki Hata of Taisho Pharmaceuticals for technical assistance; Sean McKenna and Yikang Wang of Vernalis for support in synthesis; and the staff at Diamond Light Source and Soleil synchrotrons for help with crystallographic data collection.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">LpxC</td><td class="NLM_def"><p class="first last">UDP-3-<i>O</i>-acyl-<i>N</i>-acetylglucosamine deacetylase</p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimum inhibitory concentration</p></td></tr><tr><td class="NLM_term">PAβN</td><td class="NLM_def"><p class="first last">phenylalanine-arginine β-naphthylamide</p></td></tr><tr><td class="NLM_term">UDP-GlcNAc</td><td class="NLM_def"><p class="first last">uridine diphosphate <i>N</i>-acetylglucosamine</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">retention time</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matrix metalloprotease</p></td></tr><tr><td class="NLM_term">boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">(BOC)<sub>2</sub>O</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl dicarbonate</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last"><i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetra-<i>n</i>-butylammonium fluoride</p></td></tr><tr><td class="NLM_term">THP</td><td class="NLM_def"><p class="first last">tetrahydropyranyl</p></td></tr><tr><td class="NLM_term">TMAD</td><td class="NLM_def"><p class="first last">3-(dimethylcarbamoylimino)-1,1-dimethylurea</p></td></tr><tr><td class="NLM_term">TIPS</td><td class="NLM_def"><p class="first last">triisopropylsilyl</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71175" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71175" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, U.</span></span> <span> </span><span class="NLM_article-title">The cost of antimicrobial resistance</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1038/s41579-018-0125-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fs41579-018-0125-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=30467331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=3&author=U.+Hofer&title=The+cost+of+antimicrobial+resistance&doi=10.1038%2Fs41579-018-0125-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The cost of antimicrobial resistance</span></div><div class="casAuthors">Hofer, Ursula</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new report shows that investing in the prevention of antimicrobial resistance now will have a big pay-off in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDpzM5nDqP7Vg90H21EOLACvtfcHk0lh9z1gPqnCMUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhurnE&md5=0c83ce43554a9409951e6467831919f8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41579-018-0125-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41579-018-0125-x%26sid%3Dliteratum%253Aachs%26aulast%3DHofer%26aufirst%3DU.%26atitle%3DThe%2520cost%2520of%2520antimicrobial%2520resistance%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2019%26volume%3D17%26spage%3D3%26doi%3D10.1038%2Fs41579-018-0125-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery in the resistance era</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">336</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1038/nature17042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnature17042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=26791724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=336-343&author=E.+D.+Brownauthor=G.+D.+Wright&title=Antibacterial+drug+discovery+in+the+resistance+era&doi=10.1038%2Fnature17042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery in the resistance era</span></div><div class="casAuthors">Brown, Eric D.; Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">336-343</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The looming antibiotic resistance crisis has penetrated the consciousness of clinicians, researchers, policymakers, politicians and the public at large.  The evolution and widespread distribution of antibiotic resistance elements in bacterial pathogens has made diseases that were once easily treatable deadly again.  Unfortunately, accompanying the rise in global resistance is a failure in antibacterial drug discovery.  Lessons from the history of antibiotic discovery and fresh understanding of antibiotic action and the cell biol. of microorganisms have the potential to deliver twenty-first century medicines that are able to control infection in the resistance era.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKJH_K99a5abVg90H21EOLACvtfcHk0lgrNDPZc-Azvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oisro%253D&md5=62e0454601a242db039e60d77be26406</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature17042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17042%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DAntibacterial%2520drug%2520discovery%2520in%2520the%2520resistance%2520era%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D336%26epage%3D343%26doi%3D10.1038%2Fnature17042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamek, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Credille, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeting metalloenzymes for therapeutic intervention</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1455</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=1323-1455&author=A.+Y.+Chenauthor=R.+N.+Adamekauthor=B.+L.+Dickauthor=C.+V.+Credilleauthor=C.+N.+Morrisonauthor=S.+M.+Cohen&title=Targeting+metalloenzymes+for+therapeutic+intervention&doi=10.1021%2Facs.chemrev.8b00201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Metalloenzymes for Therapeutic Intervention</span></div><div class="casAuthors">Chen, Allie Y.; Adamek, Rebecca N.; Dick, Benjamin L.; Credille, Cy V.; Morrison, Christine N.; Cohen, Seth M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1323-1455</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Metalloenzymes are central to a wide scope of essential biol. activities, including nucleic acid modification, protein degrdn., and many others.  The role of metalloenzymes in these processes also makes them central for the progression of many diseases, and as such, makes metalloenzymes attractive targets for therapeutic intervention.  Increasing awareness of the role metalloenzymes play in disease and their importance as a class of targets has amplified interest in the development of new strategies to develop inhibitors and ultimately useful drugs.  In this review, the authors provide a broad overview of several drug discovery efforts focused on metalloenzymes and attempt to map out the current landscape of high-value metalloenzyme targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ZM-fHthP3bVg90H21EOLACvtfcHk0lgrNDPZc-Azvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KjsLnP&md5=38be824f8313556bd0535507ce9e133a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00201%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DA.%2BY.%26aulast%3DAdamek%26aufirst%3DR.%2BN.%26aulast%3DDick%26aufirst%3DB.%2BL.%26aulast%3DCredille%26aufirst%3DC.%2BV.%26aulast%3DMorrison%26aufirst%3DC.%2BN.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DTargeting%2520metalloenzymes%2520for%2520therapeutic%2520intervention%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D1323%26epage%3D1455%26doi%3D10.1021%2Facs.chemrev.8b00201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, A. L.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme Lpxc</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">a025304</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a025304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1101%2Fcshperspect.a025304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=27235477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=A.+L.+Erwin&title=Antibacterial+drug+discovery+targeting+the+lipopolysaccharide+biosynthetic+enzyme+Lpxc&doi=10.1101%2Fcshperspect.a025304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme LpxC</span></div><div class="casAuthors">Erwin, Alice L.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a025304/1-a025304/15</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The enzyme LpxC (UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase) is broadly conserved across Gram-neg. bacteria and is essential for synthesis of lipid A, the membrane anchor of the lipopolysaccharides (LPSs), which are a major component of the outer membrane in nearly all Gram-neg. bacteria.  LpxC has been the focus of targetdirected antibiotic discovery projects in numerous pharmaceutical and academic groups for more than 20 years.  Despite intense effort, no LpxC inhibitor has been approved for therapeutic use, and only one has yet reached human studies.  This article will summarize the history of LpxC as a drug target and the parallel history of research on LpxC biol.  Both academic and industrial researchers have used LpxC inhibitors as tool compds., leading to increased understanding of the differing mechanisms for regulation of LPS synthesis in Escherichia coli and Pseudomonas aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-AZMwFme-97Vg90H21EOLACvtfcHk0lgrNDPZc-Azvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D&md5=a8ca24f54a9aaad079b94a41cfab62a4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a025304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a025304%26sid%3Dliteratum%253Aachs%26aulast%3DErwin%26aufirst%3DA.%2BL.%26atitle%3DAntibacterial%2520drug%2520discovery%2520targeting%2520the%2520lipopolysaccharide%2520biosynthetic%2520enzyme%2520Lpxc%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26doi%3D10.1101%2Fcshperspect.a025304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalinin, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holl, R.</span></span> <span> </span><span class="NLM_article-title">Lpxc inhibitors: A patent review (2010-2016)</span>. <i>Expert Opin. Ther. Patents</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1360282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1080%2F13543776.2017.1360282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=28742403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12htbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1227-1250&author=D.+V.+Kalininauthor=R.+Holl&title=Lpxc+inhibitors%3A+A+patent+review+%282010-2016%29&doi=10.1080%2F13543776.2017.1360282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LpxC inhibitors: a patent review (2010-2016)</span></div><div class="casAuthors">Kalinin, Dmitrii V.; Holl, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1250</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: The Zn2+-dependent deacetylase LpxC is an essential enzyme of lipid A biosynthesis in Gram-neg. bacteria and a promising target for the development of antibiotics selectively combating Gram-neg. pathogens.  Researchers from industry and academia have synthesized structurally diverse LpxC inhibitors, exhibiting different LpxC inhibitory and antibacterial activities.:.  A brief introduction into the structure and function of LpxC, showing its suitability as antibacterial target, along with the structures of several reported LpxC inhibitors, is given.  The article reviews patents (reported between 2010 and 2016) and related research publications on novel small-mol. LpxC inhibitors.  Emphasis is placed on structure-activity relationships within the reported series of LpxC inhibitors.:.  The performed anal. of patents revealed that the current search for novel LpxC inhibitors is focused on small mols., sharing common structural features like a Zn2+-chelating group as well as a highly lipophilic side-chain.  However, despite the promising preclin. data of many of the reported compds., besides the recently withdrawn clin. candidate ACHN-975, no other LpxC inhibitor has entered clin. trials.  The lack of clin. candidates might be related with undesired effects caused by the common structural elements of the LpxC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOaBFqoF4oKLVg90H21EOLACvtfcHk0lgYmkC_40vYgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12htbjL&md5=f853c58400cda075ec86f3ec20bd1a6f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1360282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1360282%26sid%3Dliteratum%253Aachs%26aulast%3DKalinin%26aufirst%3DD.%2BV.%26aulast%3DHoll%26aufirst%3DR.%26atitle%3DLpxc%2520inhibitors%253A%2520A%2520patent%2520review%2520%25282010-2016%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Patents%26date%3D2017%26volume%3D27%26spage%3D1227%26epage%3D1250%26doi%3D10.1080%2F13543776.2017.1360282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, R. E.</span></span> <span> </span><span class="NLM_article-title">Lipid a modification systems in gram-negative bacteria</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.76.010307.145803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1146%2Fannurev.biochem.76.010307.145803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=17362200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2007&pages=295-329&author=C.+R.+Raetzauthor=C.+M.+Reynoldsauthor=M.+S.+Trentauthor=R.+E.+Bishop&title=Lipid+a+modification+systems+in+gram-negative+bacteria&doi=10.1146%2Fannurev.biochem.76.010307.145803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid A modification systems in Gram-negative bacteria</span></div><div class="casAuthors">Raetz, Christian R. H.; Reynolds, C. Michael; Trent, M. Stephen; Bishop, Russell E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">295-329</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The lipid A moiety of lipopolysaccharide forms the outer monolayer of the outer membrane of most gram-neg. bacteria.  Escherichia coli lipid A is synthesized on the cytoplasmic surface of the inner membrane by a conserved pathway of nine constitutive enzymes.  Following attachment of the core oligosaccharide, nascent core-lipid A is flipped to the outer surface of the inner membrane by the ABC transporter MsbA, where the O-antigen polymer is attached.  Diverse covalent modifications of the lipid A moiety may occur during its transit from the outer surface of the inner membrane to the outer membrane.  Lipid A modification enzymes are reporters for lipopolysaccharide trafficking within the bacterial envelope.  Modification systems are variable and often regulated by environmental conditions.  Although not required for growth, the modification enzymes modulate virulence of some gram-neg. pathogens.  Heterologous expression of lipid A modification enzymes may enable the development of new vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxgMwtwJjiGrVg90H21EOLACvtfcHk0lgYmkC_40vYgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb%252FN&md5=69d24916dc1f524a813cdea71deb5822</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.76.010307.145803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.76.010307.145803%26sid%3Dliteratum%253Aachs%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DReynolds%26aufirst%3DC.%2BM.%26aulast%3DTrent%26aufirst%3DM.%2BS.%26aulast%3DBishop%26aufirst%3DR.%2BE.%26atitle%3DLipid%2520a%2520modification%2520systems%2520in%2520gram-negative%2520bacteria%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2007%26volume%3D76%26spage%3D295%26epage%3D329%26doi%3D10.1146%2Fannurev.biochem.76.010307.145803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, C.</span></span> <span> </span><span class="NLM_article-title">Lipopolysaccharide endotoxins</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.71.110601.135414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1146%2Fannurev.biochem.71.110601.135414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=12045108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD38Xos1Clt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2002&pages=635-700&author=C.+R.+H.+Raetzauthor=C.+Whitfield&title=Lipopolysaccharide+endotoxins&doi=10.1146%2Fannurev.biochem.71.110601.135414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lipopolysaccharide endotoxins</span></div><div class="casAuthors">Raetz, Christian R. H.; Whitfield, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">635-700</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Bacterial lipopolysaccharides (LPS) typically consist of a hydrophobic domain known as lipid A (or endotoxin), a nonrepeating "core" oligosaccharide, and a distal polysaccharide (or O-antigen).  Recent genomic data have facilitated study of LPS assembly in diverse Gram-neg. bacteria, many of which are human or plant pathogens, and have established the importance of lateral gene transfer in generating structural diversity of O-antigens.  Many enzymes of lipid A biosynthesis like LpxC have been validated as targets for development of new antibiotics.  Key genes for lipid A biosynthesis have unexpectedly also been found in higher plants, indicating that eukaryotic lipid A-like mols. may exist.  Most significant has been the identification of the plasma membrane protein TLR4 as the lipid A signaling receptor of animal cells.  TLR4 belongs to a family of innate immunity receptors that possess a large extracellular domain of leucine-rich repeats, a single trans-membrane segment, and a smaller cytoplasmic signaling region that engages the adaptor protein MyD88.  The expanding knowledge of TLR4 specificity and its downstream signaling pathways should provide new opportunities for blocking inflammation assocd. with infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJaZKIRBouFLVg90H21EOLACvtfcHk0lgYmkC_40vYgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xos1Clt7w%253D&md5=68019317d50df36bca36e26c623551f3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.71.110601.135414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.71.110601.135414%26sid%3Dliteratum%253Aachs%26aulast%3DRaetz%26aufirst%3DC.%2BR.%2BH.%26aulast%3DWhitfield%26aufirst%3DC.%26atitle%3DLipopolysaccharide%2520endotoxins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2002%26volume%3D71%26spage%3D635%26epage%3D700%26doi%3D10.1146%2Fannurev.biochem.71.110601.135414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swem, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Potent Lpxc inhibitors with in vitro activity against multidrug-resistant pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">e00977</span>, <span class="refDoi"> DOI: 10.1128/AAC.00977-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1128%2FAAC.00977-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=31451507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&pages=e00977&author=K.+M.+Krauseauthor=C.+M.+Haglundauthor=C.+Hebnerauthor=A.+W.+Serioauthor=G.+Leeauthor=V.+Nietoauthor=F.+Cohenauthor=T.+R.+Kaneauthor=T.+D.+Machajewskiauthor=D.+Hildebrandtauthor=C.+Pillarauthor=M.+Thwaitesauthor=D.+Hallauthor=L.+Mieselauthor=M.+Hackelauthor=A.+Burekauthor=L.+D.+Andrewsauthor=E.+Armstrongauthor=L.+Swemauthor=A.+Jubbauthor=R.+T.+Cirz&title=Potent+Lpxc+inhibitors+with+in+vitro+activity+against+multidrug-resistant+pseudomonas+aeruginosa&doi=10.1128%2FAAC.00977-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Potent LpxC inhibitors with in vitro activity against multidrug-resistant Pseudomonas aeruginosa</span></div><div class="casAuthors">Krause, Kevin M.; Haglund, Cat M.; Hebner, Christy; Serio, Alisa W.; Lee, Grace; Nieto, Vincent; Cohen, Frederick; Kane, Timothy R.; Machajewski, Timothy D.; Hildebrandt, Darrin; Pillar, Chris; Thwaites, Mary; Hall, Danielle; Miesel, Lynn; Hackel, Meredith; Burek, Amanda; Andrews, Logan D.; Armstrong, Eliana; Swem, Lee; Jubb, Adrian; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e00977-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">New drugs with novel mechanisms of resistance are desperately needed to address both community and nosocomial infections due to Gram-neg. bacteria.  One such potential target is LpxC, an essential enzyme that catalyzes the first committed step of lipid A biosynthesis.  Achaogen conducted an extensive research campaign to discover novel LpxC inhibitors with activity against Pseudomonas aeruginosa.  We report here the in vitro antibacterial activity and pharmacodynamics of ACHN-975, the only mol. from these efforts and the first ever LpxC inhibitor to be evaluated in phase 1 clin. trials.  In addn., we describe the profiles of three addnl. LpxC inhibitors that were identified as potential lead mols.  These efforts did not produce an addnl. development candidate with a sufficiently large therapeutic window and the program was subsequently terminated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpJY9gz4vxx7Vg90H21EOLACvtfcHk0lgYmkC_40vYgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCltrrP&md5=ce9c5e78cf8799e15a765ed218cae387</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.00977-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00977-19%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHebner%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DHildebrandt%26aufirst%3DD.%26aulast%3DPillar%26aufirst%3DC.%26aulast%3DThwaites%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DHackel%26aufirst%3DM.%26aulast%3DBurek%26aufirst%3DA.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DArmstrong%26aufirst%3DE.%26aulast%3DSwem%26aufirst%3DL.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DPotent%2520Lpxc%2520inhibitors%2520with%2520in%2520vitro%2520activity%2520against%2520multidrug-resistant%2520pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26spage%3De00977%26doi%3D10.1128%2FAAC.00977-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dozzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterday, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konradi, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miroshnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seroogy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhmin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlasichuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Optimization of Lpxc inhibitors for antibacterial activity and cardiovascular safety</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2Fcmdc.201900287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=31283109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1560-1572&author=F.+Cohenauthor=J.+B.+Aggenauthor=L.+D.+Andrewsauthor=Z.+Assarauthor=J.+Boggsauthor=T.+Choiauthor=P.+Dozzoauthor=A.+N.+Easterdayauthor=C.+M.+Haglundauthor=D.+J.+Hildebrandtauthor=M.+C.+Holtauthor=K.+Jolyauthor=A.+Jubbauthor=Z.+Kamalauthor=T.+R.+Kaneauthor=A.+W.+Konradiauthor=K.+M.+Krauseauthor=M.+S.+Linsellauthor=T.+D.+Machajewskiauthor=O.+Miroshnikovaauthor=H.+E.+Moserauthor=V.+Nietoauthor=T.+Phanauthor=C.+Platoauthor=A.+W.+Serioauthor=J.+Seroogyauthor=A.+Shakhminauthor=A.+J.+Steinauthor=A.+D.+Sunauthor=S.+Sviridovauthor=Z.+Wangauthor=K.+Wlasichukauthor=W.+Yangauthor=X.+Zhouauthor=H.+Zhuauthor=R.+T.+Cirz&title=Optimization+of+Lpxc+inhibitors+for+antibacterial+activity+and+cardiovascular+safety&doi=10.1002%2Fcmdc.201900287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span></div><div class="casAuthors">Cohen, Frederick; Aggen, James B.; Andrews, Logan D.; Assar, Zahra; Boggs, Jen; Choi, Taylor; Dozzo, Paola; Easterday, Ashton N.; Haglund, Cat M.; Hildebrandt, Darin J.; Holt, Melissa C.; Joly, Kristin; Jubb, Adrian; Kamal, Zeeshan; Kane, Timothy R.; Konradi, Andrei W.; Krause, Kevin M.; Linsell, Martin S.; Machajewski, Timothy D.; Miroshnikova, Olga; Moser, Heinz E.; Nieto, Vincent; Phan, Thu; Plato, Craig; Serio, Alisa W.; Seroogy, Julie; Shakhmin, Anton; Stein, Adam J.; Sun, Alex D.; Sviridov, Serguei; Wang, Zhan; Wlasichuk, Kenneth; Yang, Wen; Zhou, Xiaoming; Zhu, Hai; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1560-1572</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a Zn2+ deacetylase that is essential for the survival of most pathogenic Gram-neg. bacteria.  ACHN-975 (N-((S)-3-amino-1-(hydroxyamino)-3-methyl-1-oxobutan-2-yl)-4-(((1R,2R)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)benzamide) was the first LpxC inhibitor to reach human clin. testing and was discovered to have a dose-limiting cardiovascular toxicity of transient hypotension without compensatory tachycardia.  Herein we report the effort beyond ACHN-975 to discover LpxC inhibitors optimized for enzyme potency, antibacterial activity, pharmacokinetics, and cardiovascular safety.  Based on its overall profile, compd. 26 (LPXC-516, (S)-N-(2-(hydroxyamino)-1-(3-methoxy-1,1-dioxidothietan-3-yl)-2-oxoethyl)-4-(6-hydroxyhexa-1,3-diyn-1-yl)benzamide) was chosen for further development.  A phosphate prodrug of 26 was developed that provided a soly. of >30 mg mL-1 for parenteral administration and conversion into the active drug with a t1/2 of approx. two minutes.  Unexpectedly, and despite our optimization efforts, the prodrug of 26 still possesses a therapeutic window insufficient to support further clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhe0SbdQ2NbVg90H21EOLACvtfcHk0lj0VodVLQfHIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N&md5=b8cc87e1ebb76dfe8d330729a9d048d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900287%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DAssar%26aufirst%3DZ.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DT.%26aulast%3DDozzo%26aufirst%3DP.%26aulast%3DEasterday%26aufirst%3DA.%2BN.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHildebrandt%26aufirst%3DD.%2BJ.%26aulast%3DHolt%26aufirst%3DM.%2BC.%26aulast%3DJoly%26aufirst%3DK.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DKamal%26aufirst%3DZ.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DKonradi%26aufirst%3DA.%2BW.%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DLinsell%26aufirst%3DM.%2BS.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DMiroshnikova%26aufirst%3DO.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DPhan%26aufirst%3DT.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DSeroogy%26aufirst%3DJ.%26aulast%3DShakhmin%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DSun%26aufirst%3DA.%2BD.%26aulast%3DSviridov%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWlasichuk%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DOptimization%2520of%2520Lpxc%2520inhibitors%2520for%2520antibacterial%2520activity%2520and%2520cardiovascular%2520safety%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1560%26epage%3D1572%26doi%3D10.1002%2Fcmdc.201900287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Why hydroxamates may not be the best histone deacetylase inhibitors—what some may have forgotten or would rather forget?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2Fcmdc.201500486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=26603496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=15-21&author=S.+Shenauthor=A.+P.+Kozikowski&title=Why+hydroxamates+may+not+be+the+best+histone+deacetylase+inhibitors%E2%80%94what+some+may+have+forgotten+or+would+rather+forget%3F&doi=10.1002%2Fcmdc.201500486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Why Hydroxamates May Not Be the Best Histone Deacetylase Inhibitors-What Some May Have Forgotten or Would Rather Forget?</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-21</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Hydroxamate-based histone deacetylase inhibitors (HDACIs) have been approved as therapeutic agents by the US Food and Drug Administration for use in oncol. applications.  While the potential utility of such HDACIs in other areas of medicinal chem. is tremendous, there are significant concerns that "pan-HDAC inhibitors" may be too broadly acting and/or toxic for clin. use beyond oncol.  In addn. to the isoenzyme selectivity challenge, the potential mutagenicity of hydroxamate-contg. HDAC inhibitors represents a major hindrance in their application to other therapeutic areas.  Herein the authors report on the mutagenicity of known hydroxamates, discuss the mechanisms responsible for their genotoxicity, and review some of the current alternatives to hydroxamates.  The authors conclude that the hydroxamate group, while providing high-potency HDACIs, is not necessarily the best zinc-binding group for HDACI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplftHMFVoN6rVg90H21EOLACvtfcHk0lj0VodVLQfHIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL&md5=5183d2c0a4c86fe67bea480a66bc5dc6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500486%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DWhy%2520hydroxamates%2520may%2520not%2520be%2520the%2520best%2520histone%2520deacetylase%2520inhibitors%25E2%2580%2594what%2520some%2520may%2520have%2520forgotten%2520or%2520would%2520rather%2520forget%253F%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D15%26epage%3D21%26doi%3D10.1002%2Fcmdc.201500486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenbroucke, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span> <span> </span><span class="NLM_article-title">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd4390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnrd4390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=25376097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=904-927&author=R.+E.+Vandenbrouckeauthor=C.+Libert&title=Is+there+new+hope+for+therapeutic+matrix+metalloproteinase+inhibition%3F&doi=10.1038%2Fnrd4390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span></div><div class="casAuthors">Vandenbroucke, Roosmarijn E.; Libert, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">904-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent endopeptidases that form a family of 24 members in mammals.  Evidence of the pathol. roles of MMPs in various diseases, combined with their druggability, has made them attractive therapeutic targets.  Initial drug discovery efforts focused on the roles of MMPs in cancer progression, and more than 50 MMP inhibitors have been investigated in clin. trials in various cancers.  However, all of these trials failed.  Reasons for failure include the lack of inhibitor specificity and insufficient knowledge about the complexity of the disease biol.  MMPs are also known to be involved in several inflammatory processes, and there are new therapeutic opportunities for MMP inhibitors to treat such diseases.  In this Review, we discuss the recent advances made in understanding the role of MMPs in inflammatory diseases and the therapeutic potential of MMP inhibition in those conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpua2GBt8P5b7Vg90H21EOLACvtfcHk0lh8IYrt8shp6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I&md5=032265fe59e9f7567e4d3c7d072b3cde</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4390%26sid%3Dliteratum%253Aachs%26aulast%3DVandenbroucke%26aufirst%3DR.%2BE.%26aulast%3DLibert%26aufirst%3DC.%26atitle%3DIs%2520there%2520new%2520hope%2520for%2520therapeutic%2520matrix%2520metalloproteinase%2520inhibition%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D904%26epage%3D927%26doi%3D10.1038%2Fnrd4390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Pyridone methylsulfone hydroxamate Lpxc inhibitors for the treatment of serious gram-negative infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1670</span>, <span class="refDoi"> DOI: 10.1021/jm2014875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1662-1670&author=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=L.+M.+Priceauthor=J.+A.+Abramiteauthor=J.+Arcariauthor=R.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=S.+W.+Chungauthor=E.+M.+Collantesauthor=C.+Desbonnetauthor=M.+Doroskiauthor=J.+Dotyauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=L.+McAllisterauthor=E.+McElroyauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=R.+Oliverauthor=J.+Penzienauthor=M.+Plummerauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=A.+P.+Tomarasauthor=D.+P.+Uccelloauthor=A.+Vazauthor=D.+G.+Wishka&title=Pyridone+methylsulfone+hydroxamate+Lpxc+inhibitors+for+the+treatment+of+serious+gram-negative+infections&doi=10.1021%2Fjm2014875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridone methylsulfone hydroxamate LpxC inhibitors for the treatment of serious gram-negative infections</span></div><div class="casAuthors">Montgomery, Justin I.; Brown, Matthew F.; Reilly, Usa; Price, Loren M.; Abramite, Joseph A.; Arcari, Joel; Barham, Rose; Che, Ye; Chen, Jinshan Michael; Chung, Seung Won; Collantes, Elizabeth M.; Desbonnet, Charlene; Doroski, Matthew; Doty, Jonathan; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; McAllister, Laura; McElroy, Eric; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa; Noe, Mark C.; O'Donnell, John; Oliver, Robert; Penzien, Joseph; Plummer, Mark; Shanmugasundaram, Veerabahu; Thoma, Christy; Tomaras, Andrew P.; Uccello, Daniel P.; Vaz, Alfin; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1670</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. activity of a new series of LpxC inhibitors represented by pyridone methylsulfone hydroxamate 2a (I) is presented.  Members of this series have improved soly. and free fraction when compared to compds. in the previously described biphenyl methylsulfone hydroxamate series, and they maintain superior Gram-neg. antibacterial activity to comparator agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgNGd-bEXEabVg90H21EOLACvtfcHk0lh8IYrt8shp6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D&md5=f2410aae177b2fa24d8da72942b95564</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm2014875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014875%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcAllister%26aufirst%3DL.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DVaz%26aufirst%3DA.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPyridone%2520methylsulfone%2520hydroxamate%2520Lpxc%2520inhibitors%2520for%2520the%2520treatment%2520of%2520serious%2520gram-negative%2520infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1662%26epage%3D1670%26doi%3D10.1021%2Fjm2014875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the bacterial deacetylase Lpxc bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">34073</span>– <span class="NLM_lpage">34080</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.513028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1074%2Fjbc.M113.513028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=24108127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=34073-34080&author=G.+M.+Claytonauthor=D.+J.+Kleinauthor=K.+W.+Rickertauthor=S.+B.+Patelauthor=M.+Kornienkoauthor=J.+Zugay-Murphyauthor=J.+C.+Reidauthor=S.+Tummalaauthor=S.+Sharmaauthor=S.+B.+Singhauthor=L.+Mieselauthor=K.+J.+Lumbauthor=S.+M.+Soisson&title=Structure+of+the+bacterial+deacetylase+Lpxc+bound+to+the+nucleotide+reaction+product+reveals+mechanisms+of+oxyanion+stabilization+and+proton+transfer&doi=10.1074%2Fjbc.M113.513028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the bacterial deacetylase LpxC bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span></div><div class="casAuthors">Clayton, Gina M.; Klein, Daniel J.; Rickert, Keith W.; Patel, Sangita B.; Kornienko, Maria; Zugay-Murphy, Joan; Reid, John C.; Tummala, Srivanya; Sharma, Sujata; Singh, Sheo B.; Miesel, Lynn; Lumb, Kevin J.; Soisson, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">34073-34080</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant strains of pathogenic bacteria is an increasing threat to global health that underscores an urgent need for an expanded antibacterial armamentarium.  Gram-neg. bacteria, such as Escherichia coli, have become increasingly important clin. pathogens with limited treatment options.  This is due in part to their lipopolysaccharide (LPS) outer membrane components, which dually serve as endotoxins while also protecting Gram-neg. bacteria from antibiotic entry.  The LpxC enzyme, UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase, catalyzes the committed step of LPS biosynthesis, making LpxC a promising target for new antibacterials.  Here, the authors present the 1st structure of E. coli LpxC in complex with the deacetylation reaction product, UDP-[3-O-(R-3-hydroxymyristoyl)]glucosamine.  These studies provide valuable insight into the recognition of substrates and products by LpxC and a platform for structure-guided drug discovery of broad spectrum Gram-neg. antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm8dRnYgccKbVg90H21EOLACvtfcHk0lh8IYrt8shp6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO&md5=ac4adc79fe696089087d35a29e39f892</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.513028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.513028%26sid%3Dliteratum%253Aachs%26aulast%3DClayton%26aufirst%3DG.%2BM.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DTummala%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26atitle%3DStructure%2520of%2520the%2520bacterial%2520deacetylase%2520Lpxc%2520bound%2520to%2520the%2520nucleotide%2520reaction%2520product%2520reveals%2520mechanisms%2520of%2520oxyanion%2520stabilization%2520and%2520proton%2520transfer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D34073%26epage%3D34080%26doi%3D10.1074%2Fjbc.M113.513028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herradura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poutsiaka, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahamoon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.,  3rd</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of Lpxc inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1016%2Fj.bmcl.2017.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=28302397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksF2qtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1670-1680&author=J.+Zhangauthor=A.+Chanauthor=B.+Lippaauthor=J.+B.+Crossauthor=C.+Liuauthor=N.+Yinauthor=J.+A.+Romeroauthor=J.+Lawrenceauthor=R.+Heneyauthor=P.+Herraduraauthor=J.+Gossauthor=C.+Clarkauthor=C.+Abelauthor=Y.+Zhangauthor=K.+M.+Poutsiakaauthor=F.+Epieauthor=M.+Conradauthor=A.+Mahamoonauthor=K.+Nguyenauthor=A.+Chavanauthor=E.+Clarkauthor=T.+C.+Liauthor=R.+K.+Chengauthor=M.+Woodauthor=O.+A.+Andersenauthor=M.+Brooksauthor=J.+Kwongauthor=J.+Barkerauthor=I.+B.+Parrauthor=Y.+Guauthor=M.+D.+Ryanauthor=S.+Colemanauthor=C.+A.+Metcalf&title=Structure-based+discovery+of+Lpxc+inhibitors&doi=10.1016%2Fj.bmcl.2017.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of LpxC inhibitors</span></div><div class="casAuthors">Zhang, Jing; Chan, Audrey; Lippa, Blaise; Cross, Jason B.; Liu, Christopher; Yin, Ning; Romero, Jan Antoinette C.; Lawrence, Jonathan; Heney, Ryan; Herradura, Prudencio; Goss, Jennifer; Clark, Cynthia; Abel, Cassandra; Zhang, Yanzhi; Poutsiaka, Katherine M.; Epie, Felix; Conrad, Mary; Mahamoon, Azard; Nguyen, Kien; Chavan, Ajit; Clark, Edward; Li, Tong-chuan; Cheng, Robert K.; Wood, Michael; Andersen, Ole A.; Brooks, Mark; Kwong, Jason; Barker, John; Parr, Ian Barrie; Gu, Yugui; Ryan, M. Dominic; Coleman, Scott; Metcalf, Chester A., III</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1670-1680</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The emergence and spread of multidrug-resistant (MDR) Gram neg. bacteria presents a serious threat for public health.  Novel antimicrobials that could overcome the resistance problems are urgently needed.  UDP-3-O-(R-3-hydroxymyristol)-N-acetyl glucosamine deacetylase (LpxC) is a cytosolic zinc-based deacetylase that catalyzes the first committed step in the biosynthesis of lipid A, which is essential for the survival of Gram-neg. bacteria.  Our efforts toward the discovery of novel LpxC inhibitors are presented herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEwaZ2nN26t7Vg90H21EOLACvtfcHk0lh99U2XT939lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksF2qtr8%253D&md5=65edd36115b7b96c3e6968d3bb7e93de</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DA.%26aulast%3DLippa%26aufirst%3DB.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DRomero%26aufirst%3DJ.%2BA.%26aulast%3DLawrence%26aufirst%3DJ.%26aulast%3DHeney%26aufirst%3DR.%26aulast%3DHerradura%26aufirst%3DP.%26aulast%3DGoss%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DC.%26aulast%3DAbel%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPoutsiaka%26aufirst%3DK.%2BM.%26aulast%3DEpie%26aufirst%3DF.%26aulast%3DConrad%26aufirst%3DM.%26aulast%3DMahamoon%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DChavan%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DT.%2BC.%26aulast%3DCheng%26aufirst%3DR.%2BK.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DBrooks%26aufirst%3DM.%26aulast%3DKwong%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DI.%2BB.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DM.%2BD.%26aulast%3DColeman%26aufirst%3DS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520Lpxc%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1670%26epage%3D1680%26doi%3D10.1016%2Fj.bmcl.2017.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baurin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of libraries of molecular fragments for use in nmr screening against protein targets</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2166</span>, <span class="refDoi"> DOI: 10.1021/ci049806z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-Elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+libraries+of+molecular+fragments+for+use+in+nmr+screening+against+protein+targets&doi=10.1021%2Fci049806z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0lh99U2XT939lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-Ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520libraries%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520nmr%2520screening%2520against%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166%26doi%3D10.1021%2Fci049806z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: Analysis of output from multiple screening campaigns</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-620&author=I.+J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+Analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0lhw4AJouA8Y8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520Analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D620%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span> <span> </span><span class="NLM_article-title">The seeds approach: Integrating fragments into drug discovery</span>. <i>Curr. Top Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1568</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.2174/156802607782341109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&author=R.+E.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+J.+Drysdale&title=The+seeds+approach%3A+Integrating+fragments+into+drug+discovery&doi=10.2174%2F156802607782341109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0lhw4AJouA8Y8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520seeds%2520approach%253A%2520Integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1568%26epage%3D1581%26doi%3D10.2174%2F156802607782341109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span> <span> </span><span class="NLM_article-title">Experiences in fragment-based lead discovery</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-381274-2.00020-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1016%2FB978-0-12-381274-2.00020-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=21371604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2011&pages=509-531&author=R.+E.+Hubbardauthor=J.+B.+Murray&title=Experiences+in+fragment-based+lead+discovery&doi=10.1016%2FB978-0-12-381274-2.00020-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based lead discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Murray, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Design</span>),
    <span class="NLM_cas:pages">509-531</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  This chapter summarizes the experience at Vernalis over the past decade in developing and applying fragment-based discovery methods across a range of different targets.  The emphasis will be on the practical aspects of the different biophys. techniques (surface plasmon resonance (SPR), differential scanning fluorimetry (DSF), isothermal titrn. calorimetry, NMR, and X-ray crystallog.) that can be used to identify fragments that bind to targets and a discussion of the criteria and strategies for selecting and evolving fragments to lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbWFWP7PylG7Vg90H21EOLACvtfcHk0lhw4AJouA8Y8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D&md5=358abcb0648a3482c6be40f5b3d04ff9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-381274-2.00020-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-381274-2.00020-0%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26atitle%3DExperiences%2520in%2520fragment-based%2520lead%2520discovery%26jtitle%3DMethods%2520Enzymol.%26date%3D2011%26volume%3D493%26spage%3D509%26epage%3D531%26doi%3D10.1016%2FB978-0-12-381274-2.00020-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Credille, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamek, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure–activity relationships in metal-binding pharmacophores for influenza endonuclease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10206</span>– <span class="NLM_lpage">10217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01363</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01363" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFalur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10206-10217&author=C.+V.+Credilleauthor=B.+L.+Dickauthor=C.+N.+Morrisonauthor=R.+W.+Stokesauthor=R.+N.+Adamekauthor=N.+C.+Wuauthor=I.+A.+Wilsonauthor=S.+M.+Cohen&title=Structure%E2%80%93activity+relationships+in+metal-binding+pharmacophores+for+influenza+endonuclease&doi=10.1021%2Facs.jmedchem.8b01363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships in Metal-Binding Pharmacophores for Influenza Endonuclease</span></div><div class="casAuthors">Credille, Cy V.; Dick, Benjamin L.; Morrison, Christine N.; Stokes, Ryjul W.; Adamek, Rebecca N.; Wu, Nicholas C.; Wilson, Ian A.; Cohen, Seth M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10206-10217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloenzymes represent an important target space for drug discovery.  A limitation to the early development of metalloenzyme inhibitors has been the lack of established structure-activity relationships (SARs) for mols. that bind the metal ion cofactor(s) of a metalloenzyme.  Herein, we employed a bioinorg. perspective to develop an SAR for inhibition of the metalloenzyme influenza RNA polymerase PAN endonuclease.  The identified trends highlight the importance of the electronics of the metal-binding pharmacophore (MBP), in addn. to MBP sterics, for achieving improved inhibition and selectivity.  By optimization of the MBPs for PAN endonuclease, a class of highly active and selective fragments was developed that displays IC50 values <50 nM.  This SAR led to structurally distinct mols. that also displayed IC50 values of ∼10 nM, illustrating the utility of a metal-centric development campaign in generating highly active and selective metalloenzyme inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_SlKgVyqXmLVg90H21EOLACvtfcHk0lhw4AJouA8Y8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFalur%252FP&md5=4d4faaebd25518757c2ba909d7eb2040</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01363%26sid%3Dliteratum%253Aachs%26aulast%3DCredille%26aufirst%3DC.%2BV.%26aulast%3DDick%26aufirst%3DB.%2BL.%26aulast%3DMorrison%26aufirst%3DC.%2BN.%26aulast%3DStokes%26aufirst%3DR.%2BW.%26aulast%3DAdamek%26aufirst%3DR.%2BN.%26aulast%3DWu%26aufirst%3DN.%2BC.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DStructure%25E2%2580%2593activity%2520relationships%2520in%2520metal-binding%2520pharmacophores%2520for%2520influenza%2520endonuclease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10206%26epage%3D10217%26doi%3D10.1021%2Facs.jmedchem.8b01363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Current perspectives in fragment-based lead discovery (fbld)</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1042/EBC20170028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1042%2FEBC20170028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=29118093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=453-464&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Current+perspectives+in+fragment-based+lead+discovery+%28fbld%29&doi=10.1042%2FEBC20170028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Current perspectives in fragment-based lead discovery (FBLD)</span></div><div class="casAuthors">Lamoree Bas; Hubbard Roderick E; Hubbard Roderick E</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">453-464</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is over 20 years since the first fragment-based discovery projects were disclosed.  The methods are now mature for most 'conventional' targets in drug discovery such as enzymes (kinases and proteases) but there has also been growing success on more challenging targets, such as disruption of protein-protein interactions.  The main application is to identify tractable chemical startpoints that non-covalently modulate the activity of a biological molecule.  In this essay, we overview current practice in the methods and discuss how they have had an impact in lead discovery - generating a large number of fragment-derived compounds that are in clinical trials and two medicines treating patients.  In addition, we discuss some of the more recent applications of the methods in chemical biology - providing chemical tools to investigate biological molecules, mechanisms and systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0f6BottYdIAWfW6udTcc2eb55CeMnorLs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D&md5=35bce3c5978329c6737ffd719135d363</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1042%2FEBC20170028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170028%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DCurrent%2520perspectives%2520in%2520fragment-based%2520lead%2520discovery%2520%2528fbld%2529%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D453%26epage%3D464%26doi%3D10.1042%2FEBC20170028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: The impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+The+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0ljEhM5qzQPGkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520The%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Fragment Based Lead Discovery</span>, In  <i>The Handbook of Medicinal Chemistry: Principles and Practice</i>; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">153</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1039%2F9781782621836-00122" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=122-153&author=R.+E.+Hubbard&title=The+Handbook+of+Medicinal+Chemistry%3A+Principles+and+Practice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2F9781782621836-00122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781782621836-00122%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DFragment%2520Based%2520Lead%2520Discovery%26btitle%3DThe%2520Handbook%2520of%2520Medicinal%2520Chemistry%253A%2520Principles%2520and%2520Practice%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D122%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">The Role of Fragment-Based Discovery in Lead Finding</span>, In  <i>Fragment-Based Drug Discovery Lessons and Outlook</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span>; <span class="NLM_string-name">Jahnke, W.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2016</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">36</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2F9783527683604.ch01" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1-36&author=R.+E.+Hubbardauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-Based+Drug+Discovery+Lessons+and+Outlook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9783527683604.ch01&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527683604.ch01%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DThe%2520Role%2520of%2520Fragment-Based%2520Discovery%2520in%2520Lead%2520Finding%26btitle%3DFragment-Based%2520Drug%2520Discovery%2520Lessons%2520and%2520Outlook%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley%26date%3D2016%26spage%3D1%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coignard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijkmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. G.</span></span> <span> </span><span class="NLM_article-title">Antibacterial activities and characterization of novel inhibitors of Lpxc</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.6.1793-1799.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1128%2FAAC.46.6.1793-1799.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=12019092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFCms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=1793-1799&author=J.+M.+Clementsauthor=F.+Coignardauthor=I.+Johnsonauthor=S.+Chandlerauthor=S.+Palanauthor=A.+Wallerauthor=J.+Wijkmansauthor=M.+G.+Hunter&title=Antibacterial+activities+and+characterization+of+novel+inhibitors+of+Lpxc&doi=10.1128%2FAAC.46.6.1793-1799.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial activities and characterization of novel inhibitors of LpxC</span></div><div class="casAuthors">Clements, John M.; Coignard, Fanny; Johnson, Ian; Chandler, Stephen; Palan, Shilpa; Waller, Andrew; Wijkmans, Jac; Hunter, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1793-1799</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Lipid A is the hydrophobic anchor of lipopolysaccharide (LPS) and forms the major lipid component of the outer monolayer of the outer membrane of gram-neg. bacteria.  Lipid A is required for bacterial growth and virulence, and inhibition of its biosynthesis is lethal to bacteria.  UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a metalloenzyme that catalyzes the second step in the biosynthesis of lipid A.  Inhibitors of LpxC have previously been shown to have antibiotic activities.  We have screened a metalloenzyme inhibitor library for antibacterial activities against an Escherichia coli strain with reduced LpxC activity.  From this screen, a series of sulfonamide derivs. of the α-(R)-amino hydroxamic acids, exemplified by BB-78484 and BB-78485, have been identified as having potent inhibitory activities against LpxC in an in vitro assay.  Leads from this series showed gram-neg. selective activities against members of the Enterobacteriaceae, Serratia marcescens, Morganella morganii, Haemophilus influenzae, Moraxella catarrhalis, and Burkholderia cepacia.  BB-78484 was bactericidal against E. coli, achieving 3-log killing in 4 h at a concn. 4 times above the MIC, as would be predicted for an inhibitor of lipid A biosynthesis.  E. coli mutants with decreased susceptibility to BB-78484 were selected.  Anal. of these mutants revealed that resistance arose as a consequence of mutations in the fabZ or lpxC genes.  These data confirm the antibacterial target of BB-78484 and BB-78485 and validate LpxC as a target for gram-neg. selective antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN9-hhfLvqObVg90H21EOLACvtfcHk0liv5hVzIKd0JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFCms7k%253D&md5=c7830984a5330b363a687e23f399faa4</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.6.1793-1799.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.6.1793-1799.2002%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DJ.%2BM.%26aulast%3DCoignard%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DChandler%26aufirst%3DS.%26aulast%3DPalan%26aufirst%3DS.%26aulast%3DWaller%26aufirst%3DA.%26aulast%3DWijkmans%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DM.%2BG.%26atitle%3DAntibacterial%2520activities%2520and%2520characterization%2520of%2520novel%2520inhibitors%2520of%2520Lpxc%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D1793%26epage%3D1799%26doi%3D10.1128%2FAAC.46.6.1793-1799.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liv5hVzIKd0JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the ccp4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+ccp4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljnpP8Tt6AIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520ccp4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schüttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Prodrg: A tool for high-throughput crystallography of protein-ligand complexes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1107/S0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Sch%C3%BCttelkopfauthor=D.+M.+F.+van+Aalten&title=Prodrg%3A+A+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2FS0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1355-1363</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The small-mol. topol. generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and mol. topologies suitable for x-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compds. from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the std. libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallog. R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbflrch0bs97Vg90H21EOLACvtfcHk0ljnpP8Tt6AIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D&md5=f9bfdb14be7b04ea5c0db67feb39f72a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DProdrg%253A%2520A%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2FS0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljnpP8Tt6AIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lipXtLFoGE6Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapierre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grobelny, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casabonne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holme, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadzan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardy, R. E.</span></span> <span> </span><span class="NLM_article-title">Matrix metalloproteinase inhibitors: A structure-activity study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1021/jm970494j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970494j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADyaK1cXptFGl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=199-223&author=D.+E.+Levyauthor=F.+Lapierreauthor=W.+Liangauthor=W.+Yeauthor=C.+W.+Langeauthor=X.+Liauthor=D.+Grobelnyauthor=M.+Casabonneauthor=D.+Tyrrellauthor=K.+Holmeauthor=A.+Nadzanauthor=R.+E.+Galardy&title=Matrix+metalloproteinase+inhibitors%3A+A+structure-activity+study&doi=10.1021%2Fjm970494j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Matrix Metalloproteinase Inhibitors: A Structure Activity Study</span></div><div class="casAuthors">Levy, Daniel E.; Lapierre, France; Liang, Weisheng; Ye, Wenqing; Lange, Christopher W.; Li, Xiaoyuan; Grobelny, Damian; Casabonne, Marie; Tyrrell, David; Holme, Kevin; Nadzan, Alex; Galardy, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-223</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modifications around the dipeptide-mimetic core of hydroxamic acid based matrix metalloproteinase inhibitors I [AA = L-Trp, D-Trp, L-Trp(Me), L-3-benzothienylalanine, L-1- and -2-naphthylalanine, L-3- and -8-quinolylalanine, L-4-phenylphenylalanine, L-Phe, L-3- and -4-pyridylalanine, L-tert-leucine, L-abrine; R6 = NHMe, NH(CH2)4Me, NHCH2CH2OH, NHCH2CH2NHCO2CH2Ph, cyclopropylamino, cyclopentylamino, (S)- and (R)-1-indanylamino, (1R,2S)- and (1S,2R)-2-hydroxy-1-indanylamino, (S)-NHCHMePh, NHCH2Ph, piperonylamino, 2-, 3-, and 4-pyridylmethylamino, 2-(4-pyridyl)ethylamino, NHCH2CH2C6H4OH-4, 2-furanylmethylamino, 2-thiazolylmethylamino, 2-benzimidazolylamino, 3-(1-imidazolyl)propylamino, 3-(4-morpholinyl)propylamino; R2 = H, OH; R3 = CH2CHMe2, Bu, n-hexyl, n-octyl, OCHMe2, O(CH2)4Me] were studied.  These variations incorporated a variety of natural, unnatural, and synthetic amino acids in addn. to modifications of the P1' and P3' substituents.  The results of this study indicate the following structural requirements: (1) Two key hydrogen bonds must be present between the enzyme and potent substrates.  (2) Potent inhibitors must possess potent zinc-binding functionalities.  (3) The potential importance of the hydrophobic group at position R3 as illustrated by its ability to impart greater relative potency against stromelysin when larger hydrophobic groups are used.  (4) Requirements surrounding the nature of the amino acid appear to be more restrictive for stromelysin than for neutrophil collagenase, 72 kDa gelatinase, and 92 kDa gelatinase.  These requirements may involve planar fused-ring aryl systems and possibly hydrogen-bonding capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2kkOPfSKi9LVg90H21EOLACvtfcHk0lipXtLFoGE6Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXptFGl&md5=d1094c574641b569df5b33b962c48d52</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm970494j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970494j%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DLapierre%26aufirst%3DF.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DW.%26aulast%3DLange%26aufirst%3DC.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGrobelny%26aufirst%3DD.%26aulast%3DCasabonne%26aufirst%3DM.%26aulast%3DTyrrell%26aufirst%3DD.%26aulast%3DHolme%26aufirst%3DK.%26aulast%3DNadzan%26aufirst%3DA.%26aulast%3DGalardy%26aufirst%3DR.%2BE.%26atitle%3DMatrix%2520metalloproteinase%2520inhibitors%253A%2520A%2520structure-activity%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D199%26epage%3D223%26doi%3D10.1021%2Fjm970494j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">CLSI</span>. <span> </span><span class="NLM_article-title">Performance Standards for Antimicrobial Susceptibility
Testing</span>. In  <i>CLSI Supplement M100-S26</i>, <span class="NLM_edition">27</span>th ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, P. A.</span></span>, Ed.; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=CLSIauthor=P.+A.+Wayne&title=CLSI+Supplement+M100-S26"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPerformance%2520Standards%2520for%2520Antimicrobial%2520Susceptibility%250ATesting%26btitle%3DCLSI%2520Supplement%2520M100-S26%26aulast%3DWayne%26aufirst%3DP.%2BA.%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pei Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Jiyong Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Structure- and Ligand-Dynamics-Based Design of Novel Antibiotics Targeting Lipid A Enzymes LpxC and LpxH in Gram-Negative Bacteria. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2021,</strong> <em>54 </em>
                                    (7)
                                     , 1623-1634. <a href="https://doi.org/10.1021/acs.accounts.0c00880" title="DOI URL">https://doi.org/10.1021/acs.accounts.0c00880</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.0c00880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.0c00880%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DStructure-%252Band%252BLigand-Dynamics-Based%252BDesign%252Bof%252BNovel%252BAntibiotics%252BTargeting%252BLipid%252BA%252BEnzymes%252BLpxC%252Band%252BLpxH%252Bin%252BGram-Negative%252BBacteria%26aulast%3DZhou%26aufirst%3DPei%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24122020%26date%3D15032021%26volume%3D54%26issue%3D7%26spage%3D1623%26epage%3D1634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark  Greener</span>. </span><span class="cited-content_cbyCitation_article-title">Antibiotic advances: soil, sea and perverse disincentives. </span><span class="cited-content_cbyCitation_journal-name">Prescriber</span><span> <strong>2021,</strong> <em>32 </em>
                                    (6)
                                     , 19-23. <a href="https://doi.org/10.1002/psb.1924" title="DOI URL">https://doi.org/10.1002/psb.1924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/psb.1924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpsb.1924%26sid%3Dliteratum%253Aachs%26jtitle%3DPrescriber%26atitle%3DAntibiotic%252Badvances%25253A%252Bsoil%25252C%252Bsea%252Band%252Bperverse%252Bdisincentives%26aulast%3DGreener%26aufirst%3DMark%26date%3D2021%26date%3D2021%26volume%3D32%26issue%3D6%26spage%3D19%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fumihito  Ushiyama</span>, <span class="hlFld-ContribAuthor ">Hajime  Takashima</span>, <span class="hlFld-ContribAuthor ">Yohei  Matsuda</span>, <span class="hlFld-ContribAuthor ">Yuya  Ogata</span>, <span class="hlFld-ContribAuthor ">Naoki  Sasamoto</span>, <span class="hlFld-ContribAuthor ">Risa  Kurimoto-Tsuruta</span>, <span class="hlFld-ContribAuthor ">Kaori  Ueki</span>, <span class="hlFld-ContribAuthor ">Nozomi  Tanaka-Yamamoto</span>, <span class="hlFld-ContribAuthor ">Mayumi  Endo</span>, <span class="hlFld-ContribAuthor ">Masashi  Mima</span>, <span class="hlFld-ContribAuthor ">Kiyoko  Fujita</span>, <span class="hlFld-ContribAuthor ">Iichiro  Takata</span>, <span class="hlFld-ContribAuthor ">Satoshi  Tsuji</span>, <span class="hlFld-ContribAuthor ">Haruhiro  Yamashita</span>, <span class="hlFld-ContribAuthor ">Hirotoshi  Okumura</span>, <span class="hlFld-ContribAuthor ">Katsumasa  Otake</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Sugiyama</span>. </span><span class="cited-content_cbyCitation_article-title">Lead optimization of 2-hydroxymethyl imidazoles as non-hydroxamate LpxC inhibitors: Discovery of TP0586532. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>30 </em>, 115964. <a href="https://doi.org/10.1016/j.bmc.2020.115964" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115964</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115964%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DLead%252Boptimization%252Bof%252B2-hydroxymethyl%252Bimidazoles%252Bas%252Bnon-hydroxamate%252BLpxC%252Binhibitors%25253A%252BDiscovery%252Bof%252BTP0586532%26aulast%3DUshiyama%26aufirst%3DFumihito%26date%3D2021%26volume%3D30%26spage%3D115964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. LpxC catalyzes the first committed step of LPS biosynthesis (shown is the substrate for <i>Pseudomonas aeruginosa</i>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative LpxC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Detail of the crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UPG">5UPG</a>) of <b>1</b> (PF-5081090) binding to the active site of PaLpxC. Unless otherwise stated, in all structural figures, compounds are shown by sticks with orange carbons, the zinc ion as a van der Waals sphere, and protein amino acids (side chains only) in ball and stick. Except for (b) and 4(a), all other figures of compounds binding to LpxC are presented with the same view and convention. (a) Inhibitor binding site, with the inside of the solvent surface in gray and the outside in yellow. (b) Detail of inhibitor and protein binding to a zinc ion, showing just the hydroxamate moiety of the ligand, the NE2 atoms of the imidazole of histidines, and one oxygen of the side chain of aspartic acid. Note that <b>1</b> also binds in a second site, described as the F181 site later in this manuscript.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fragments are found that bind to different sites on LpxC. (a). Schematic of the protein structure, with F181 in ball and stick, with ligand in CPK for the crystal structures of a zinc-coordinating glycine fragment (<b>6</b>, yellow), a tunnel binder (<b>7</b>, brown), and binding to the F181 site (<b>8</b>, green). A second copy (not shown) of <b>8</b> binds in the tunnel. (b) Detail of the binding of glycine fragment (<b>6</b>) with just the main chain of M62 shown (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for the legend). (c). Representative fragment hits.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a). Detail of the crystal structure of <b>22</b> binding to the active site of PaLpxC(1-299). See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used. (b). Different view showing zinc coordination to the selected ligand and protein side-chain atoms showing CD1, NE2, and CD2 of the imidazole of histidines and the carboxylic acid groups of the glutamic and aspartic acids.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0007.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Detail of the crystal structure of <b>29</b> binding to the active site of PaLpxC. See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0008.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Detail of the crystal structure of <b>38</b> binding to the active site of PaLpxC. The carbonyl of M62 is alone; otherwise, see the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for conventions used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0009.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Detail of the crystal structure of <b>43</b> binding to the active site of PaLpxC. Clipped, semitransparent solvent accessible surface, yellow outside and gray inside. See the legend of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for other conventions used. (b) Different view showing zinc coordination to the selected ligand and protein side-chain atoms as described in the legend to <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>(b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>rac</i>-(<i>R</i>*,<i>R</i>*) Acid as the Sodium Salt (<b>53</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N,N</i>-diisopropylethylamine (DIPEA), <i>n-</i>BuLi, 2-methylsulfanylacetaldehyde, tetrahydrofuran (THF), −78 °C to RT, 30 min, chromatography EtOAc/<i>n</i>-hexane, diastereoisomer <i>rac</i>-(<i>R</i>*,<i>R</i>*) <b>49a</b>, 15%, diastereoisomer <i>rac</i>-(<i>R</i>*,<i>S</i>*) <b>49b</b>, 25%; (b) <i>m</i>-chloroperbenzoic acid, CHCl<sub>3</sub>, 0 °C to RT, 16 h, 47%; (c) hydrogen, palladium hydroxide on carbon, EtOH, RT, 16 h, quant; (d) di-<i>tert</i>-butyl dicarbonate ((BOC)<sub>2</sub>O), K<sub>2</sub>CO<sub>3</sub>, THF, EtOH, RT, 3 h, quant; (e) NaOH (10 M aq), EtOH, RT, 3 h, 73%.</p></p></figure><figure data-id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 2-Amino-3-hydroxy-4-(methanesulfonyl)butan-1-one (<b>22</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-ethynylbenzenemethanol, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, triethylamine (TEA), dimethylformamide (DMF), 50 °C, 30 min, 40%; (b) 1,4-dioxane, 4<i>N</i> HCl in 1,4-dioxane, RT, 3 h, quant; (c) compound <b>53</b>, <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HATU), DIPEA, DMF, RT, 3 h, 96%; (d) formic acid, RT, 2 h, 35%.</p></p></figure><figure data-id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0012.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>60</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-iodophenol, NaH, DMF, 100 °C, 16 h, 75%; (b) iron powder, ammonium chloride, EtOH, Water, 70 °C, 15 h, 88%.</p></p></figure><figure data-id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Imidazole Compound <b>62</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 28% aqueous ammonia, glyoxal, MeOH, RT, 16 h, quant.</p></p></figure><figure data-id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/medium/jm0c01215_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0014.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01215/20210119/images/large/jm0c01215_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01215&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethynyltri(propan-2-yl)silane, TEA, Pd(PPh<sub>3</sub>)<sub>4</sub>, CuI, DMF, 80 °C, 1 h, 69%; (b) NaBH<sub>4</sub>, EtOH, 0 °C, 4 h, 89%; (c) <b>62</b>, 3-(dimethylcarbamoylimino)-1,1-dimethylurea (TMAD), tributylphosphine, THF, RT, 2 h, 93%; (d) tetra-<i>n</i>-butylammonium fluoride (TBAF), THF, 0 °C, 1 h, 82%; (e) <b>60</b>, TEA, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, DMF, 80 °C, 0.5 h, 55%; (f) <i>p</i>-TsOH, MeOH, RT, 1 h, 87%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, U.</span></span> <span> </span><span class="NLM_article-title">The cost of antimicrobial resistance</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1038/s41579-018-0125-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fs41579-018-0125-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=30467331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=3&author=U.+Hofer&title=The+cost+of+antimicrobial+resistance&doi=10.1038%2Fs41579-018-0125-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The cost of antimicrobial resistance</span></div><div class="casAuthors">Hofer, Ursula</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A new report shows that investing in the prevention of antimicrobial resistance now will have a big pay-off in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDpzM5nDqP7Vg90H21EOLACvtfcHk0liyMzKhRqSmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhurnE&md5=0c83ce43554a9409951e6467831919f8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41579-018-0125-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41579-018-0125-x%26sid%3Dliteratum%253Aachs%26aulast%3DHofer%26aufirst%3DU.%26atitle%3DThe%2520cost%2520of%2520antimicrobial%2520resistance%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2019%26volume%3D17%26spage%3D3%26doi%3D10.1038%2Fs41579-018-0125-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery in the resistance era</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">336</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1038/nature17042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnature17042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=26791724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=336-343&author=E.+D.+Brownauthor=G.+D.+Wright&title=Antibacterial+drug+discovery+in+the+resistance+era&doi=10.1038%2Fnature17042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery in the resistance era</span></div><div class="casAuthors">Brown, Eric D.; Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">336-343</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The looming antibiotic resistance crisis has penetrated the consciousness of clinicians, researchers, policymakers, politicians and the public at large.  The evolution and widespread distribution of antibiotic resistance elements in bacterial pathogens has made diseases that were once easily treatable deadly again.  Unfortunately, accompanying the rise in global resistance is a failure in antibacterial drug discovery.  Lessons from the history of antibiotic discovery and fresh understanding of antibiotic action and the cell biol. of microorganisms have the potential to deliver twenty-first century medicines that are able to control infection in the resistance era.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKJH_K99a5abVg90H21EOLACvtfcHk0liyMzKhRqSmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oisro%253D&md5=62e0454601a242db039e60d77be26406</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature17042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17042%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DAntibacterial%2520drug%2520discovery%2520in%2520the%2520resistance%2520era%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D336%26epage%3D343%26doi%3D10.1038%2Fnature17042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamek, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Credille, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeting metalloenzymes for therapeutic intervention</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1455</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=1323-1455&author=A.+Y.+Chenauthor=R.+N.+Adamekauthor=B.+L.+Dickauthor=C.+V.+Credilleauthor=C.+N.+Morrisonauthor=S.+M.+Cohen&title=Targeting+metalloenzymes+for+therapeutic+intervention&doi=10.1021%2Facs.chemrev.8b00201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Metalloenzymes for Therapeutic Intervention</span></div><div class="casAuthors">Chen, Allie Y.; Adamek, Rebecca N.; Dick, Benjamin L.; Credille, Cy V.; Morrison, Christine N.; Cohen, Seth M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1323-1455</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Metalloenzymes are central to a wide scope of essential biol. activities, including nucleic acid modification, protein degrdn., and many others.  The role of metalloenzymes in these processes also makes them central for the progression of many diseases, and as such, makes metalloenzymes attractive targets for therapeutic intervention.  Increasing awareness of the role metalloenzymes play in disease and their importance as a class of targets has amplified interest in the development of new strategies to develop inhibitors and ultimately useful drugs.  In this review, the authors provide a broad overview of several drug discovery efforts focused on metalloenzymes and attempt to map out the current landscape of high-value metalloenzyme targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ZM-fHthP3bVg90H21EOLACvtfcHk0liyMzKhRqSmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KjsLnP&md5=38be824f8313556bd0535507ce9e133a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00201%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DA.%2BY.%26aulast%3DAdamek%26aufirst%3DR.%2BN.%26aulast%3DDick%26aufirst%3DB.%2BL.%26aulast%3DCredille%26aufirst%3DC.%2BV.%26aulast%3DMorrison%26aufirst%3DC.%2BN.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DTargeting%2520metalloenzymes%2520for%2520therapeutic%2520intervention%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D1323%26epage%3D1455%26doi%3D10.1021%2Facs.chemrev.8b00201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, A. L.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme Lpxc</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">a025304</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a025304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1101%2Fcshperspect.a025304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=27235477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=A.+L.+Erwin&title=Antibacterial+drug+discovery+targeting+the+lipopolysaccharide+biosynthetic+enzyme+Lpxc&doi=10.1101%2Fcshperspect.a025304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme LpxC</span></div><div class="casAuthors">Erwin, Alice L.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a025304/1-a025304/15</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The enzyme LpxC (UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase) is broadly conserved across Gram-neg. bacteria and is essential for synthesis of lipid A, the membrane anchor of the lipopolysaccharides (LPSs), which are a major component of the outer membrane in nearly all Gram-neg. bacteria.  LpxC has been the focus of targetdirected antibiotic discovery projects in numerous pharmaceutical and academic groups for more than 20 years.  Despite intense effort, no LpxC inhibitor has been approved for therapeutic use, and only one has yet reached human studies.  This article will summarize the history of LpxC as a drug target and the parallel history of research on LpxC biol.  Both academic and industrial researchers have used LpxC inhibitors as tool compds., leading to increased understanding of the differing mechanisms for regulation of LPS synthesis in Escherichia coli and Pseudomonas aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-AZMwFme-97Vg90H21EOLACvtfcHk0lixxGfY8-uC7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D&md5=a8ca24f54a9aaad079b94a41cfab62a4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a025304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a025304%26sid%3Dliteratum%253Aachs%26aulast%3DErwin%26aufirst%3DA.%2BL.%26atitle%3DAntibacterial%2520drug%2520discovery%2520targeting%2520the%2520lipopolysaccharide%2520biosynthetic%2520enzyme%2520Lpxc%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26doi%3D10.1101%2Fcshperspect.a025304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalinin, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holl, R.</span></span> <span> </span><span class="NLM_article-title">Lpxc inhibitors: A patent review (2010-2016)</span>. <i>Expert Opin. Ther. Patents</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1360282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1080%2F13543776.2017.1360282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=28742403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12htbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1227-1250&author=D.+V.+Kalininauthor=R.+Holl&title=Lpxc+inhibitors%3A+A+patent+review+%282010-2016%29&doi=10.1080%2F13543776.2017.1360282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LpxC inhibitors: a patent review (2010-2016)</span></div><div class="casAuthors">Kalinin, Dmitrii V.; Holl, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1250</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: The Zn2+-dependent deacetylase LpxC is an essential enzyme of lipid A biosynthesis in Gram-neg. bacteria and a promising target for the development of antibiotics selectively combating Gram-neg. pathogens.  Researchers from industry and academia have synthesized structurally diverse LpxC inhibitors, exhibiting different LpxC inhibitory and antibacterial activities.:.  A brief introduction into the structure and function of LpxC, showing its suitability as antibacterial target, along with the structures of several reported LpxC inhibitors, is given.  The article reviews patents (reported between 2010 and 2016) and related research publications on novel small-mol. LpxC inhibitors.  Emphasis is placed on structure-activity relationships within the reported series of LpxC inhibitors.:.  The performed anal. of patents revealed that the current search for novel LpxC inhibitors is focused on small mols., sharing common structural features like a Zn2+-chelating group as well as a highly lipophilic side-chain.  However, despite the promising preclin. data of many of the reported compds., besides the recently withdrawn clin. candidate ACHN-975, no other LpxC inhibitor has entered clin. trials.  The lack of clin. candidates might be related with undesired effects caused by the common structural elements of the LpxC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOaBFqoF4oKLVg90H21EOLACvtfcHk0lixxGfY8-uC7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12htbjL&md5=f853c58400cda075ec86f3ec20bd1a6f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1360282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1360282%26sid%3Dliteratum%253Aachs%26aulast%3DKalinin%26aufirst%3DD.%2BV.%26aulast%3DHoll%26aufirst%3DR.%26atitle%3DLpxc%2520inhibitors%253A%2520A%2520patent%2520review%2520%25282010-2016%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Patents%26date%3D2017%26volume%3D27%26spage%3D1227%26epage%3D1250%26doi%3D10.1080%2F13543776.2017.1360282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, R. E.</span></span> <span> </span><span class="NLM_article-title">Lipid a modification systems in gram-negative bacteria</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.76.010307.145803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1146%2Fannurev.biochem.76.010307.145803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=17362200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2007&pages=295-329&author=C.+R.+Raetzauthor=C.+M.+Reynoldsauthor=M.+S.+Trentauthor=R.+E.+Bishop&title=Lipid+a+modification+systems+in+gram-negative+bacteria&doi=10.1146%2Fannurev.biochem.76.010307.145803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid A modification systems in Gram-negative bacteria</span></div><div class="casAuthors">Raetz, Christian R. H.; Reynolds, C. Michael; Trent, M. Stephen; Bishop, Russell E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">295-329</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The lipid A moiety of lipopolysaccharide forms the outer monolayer of the outer membrane of most gram-neg. bacteria.  Escherichia coli lipid A is synthesized on the cytoplasmic surface of the inner membrane by a conserved pathway of nine constitutive enzymes.  Following attachment of the core oligosaccharide, nascent core-lipid A is flipped to the outer surface of the inner membrane by the ABC transporter MsbA, where the O-antigen polymer is attached.  Diverse covalent modifications of the lipid A moiety may occur during its transit from the outer surface of the inner membrane to the outer membrane.  Lipid A modification enzymes are reporters for lipopolysaccharide trafficking within the bacterial envelope.  Modification systems are variable and often regulated by environmental conditions.  Although not required for growth, the modification enzymes modulate virulence of some gram-neg. pathogens.  Heterologous expression of lipid A modification enzymes may enable the development of new vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxgMwtwJjiGrVg90H21EOLACvtfcHk0lixxGfY8-uC7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb%252FN&md5=69d24916dc1f524a813cdea71deb5822</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.76.010307.145803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.76.010307.145803%26sid%3Dliteratum%253Aachs%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DReynolds%26aufirst%3DC.%2BM.%26aulast%3DTrent%26aufirst%3DM.%2BS.%26aulast%3DBishop%26aufirst%3DR.%2BE.%26atitle%3DLipid%2520a%2520modification%2520systems%2520in%2520gram-negative%2520bacteria%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2007%26volume%3D76%26spage%3D295%26epage%3D329%26doi%3D10.1146%2Fannurev.biochem.76.010307.145803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, C.</span></span> <span> </span><span class="NLM_article-title">Lipopolysaccharide endotoxins</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.71.110601.135414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1146%2Fannurev.biochem.71.110601.135414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=12045108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD38Xos1Clt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2002&pages=635-700&author=C.+R.+H.+Raetzauthor=C.+Whitfield&title=Lipopolysaccharide+endotoxins&doi=10.1146%2Fannurev.biochem.71.110601.135414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lipopolysaccharide endotoxins</span></div><div class="casAuthors">Raetz, Christian R. H.; Whitfield, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">635-700</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Bacterial lipopolysaccharides (LPS) typically consist of a hydrophobic domain known as lipid A (or endotoxin), a nonrepeating "core" oligosaccharide, and a distal polysaccharide (or O-antigen).  Recent genomic data have facilitated study of LPS assembly in diverse Gram-neg. bacteria, many of which are human or plant pathogens, and have established the importance of lateral gene transfer in generating structural diversity of O-antigens.  Many enzymes of lipid A biosynthesis like LpxC have been validated as targets for development of new antibiotics.  Key genes for lipid A biosynthesis have unexpectedly also been found in higher plants, indicating that eukaryotic lipid A-like mols. may exist.  Most significant has been the identification of the plasma membrane protein TLR4 as the lipid A signaling receptor of animal cells.  TLR4 belongs to a family of innate immunity receptors that possess a large extracellular domain of leucine-rich repeats, a single trans-membrane segment, and a smaller cytoplasmic signaling region that engages the adaptor protein MyD88.  The expanding knowledge of TLR4 specificity and its downstream signaling pathways should provide new opportunities for blocking inflammation assocd. with infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJaZKIRBouFLVg90H21EOLACvtfcHk0lixxGfY8-uC7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xos1Clt7w%253D&md5=68019317d50df36bca36e26c623551f3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.71.110601.135414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.71.110601.135414%26sid%3Dliteratum%253Aachs%26aulast%3DRaetz%26aufirst%3DC.%2BR.%2BH.%26aulast%3DWhitfield%26aufirst%3DC.%26atitle%3DLipopolysaccharide%2520endotoxins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2002%26volume%3D71%26spage%3D635%26epage%3D700%26doi%3D10.1146%2Fannurev.biochem.71.110601.135414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swem, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Potent Lpxc inhibitors with in vitro activity against multidrug-resistant pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">e00977</span>, <span class="refDoi"> DOI: 10.1128/AAC.00977-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1128%2FAAC.00977-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=31451507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2019&pages=e00977&author=K.+M.+Krauseauthor=C.+M.+Haglundauthor=C.+Hebnerauthor=A.+W.+Serioauthor=G.+Leeauthor=V.+Nietoauthor=F.+Cohenauthor=T.+R.+Kaneauthor=T.+D.+Machajewskiauthor=D.+Hildebrandtauthor=C.+Pillarauthor=M.+Thwaitesauthor=D.+Hallauthor=L.+Mieselauthor=M.+Hackelauthor=A.+Burekauthor=L.+D.+Andrewsauthor=E.+Armstrongauthor=L.+Swemauthor=A.+Jubbauthor=R.+T.+Cirz&title=Potent+Lpxc+inhibitors+with+in+vitro+activity+against+multidrug-resistant+pseudomonas+aeruginosa&doi=10.1128%2FAAC.00977-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Potent LpxC inhibitors with in vitro activity against multidrug-resistant Pseudomonas aeruginosa</span></div><div class="casAuthors">Krause, Kevin M.; Haglund, Cat M.; Hebner, Christy; Serio, Alisa W.; Lee, Grace; Nieto, Vincent; Cohen, Frederick; Kane, Timothy R.; Machajewski, Timothy D.; Hildebrandt, Darrin; Pillar, Chris; Thwaites, Mary; Hall, Danielle; Miesel, Lynn; Hackel, Meredith; Burek, Amanda; Andrews, Logan D.; Armstrong, Eliana; Swem, Lee; Jubb, Adrian; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e00977-19</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">New drugs with novel mechanisms of resistance are desperately needed to address both community and nosocomial infections due to Gram-neg. bacteria.  One such potential target is LpxC, an essential enzyme that catalyzes the first committed step of lipid A biosynthesis.  Achaogen conducted an extensive research campaign to discover novel LpxC inhibitors with activity against Pseudomonas aeruginosa.  We report here the in vitro antibacterial activity and pharmacodynamics of ACHN-975, the only mol. from these efforts and the first ever LpxC inhibitor to be evaluated in phase 1 clin. trials.  In addn., we describe the profiles of three addnl. LpxC inhibitors that were identified as potential lead mols.  These efforts did not produce an addnl. development candidate with a sufficiently large therapeutic window and the program was subsequently terminated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpJY9gz4vxx7Vg90H21EOLACvtfcHk0lgpQztuEHFdGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCltrrP&md5=ce9c5e78cf8799e15a765ed218cae387</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.00977-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00977-19%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHebner%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DHildebrandt%26aufirst%3DD.%26aulast%3DPillar%26aufirst%3DC.%26aulast%3DThwaites%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DD.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DHackel%26aufirst%3DM.%26aulast%3DBurek%26aufirst%3DA.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DArmstrong%26aufirst%3DE.%26aulast%3DSwem%26aufirst%3DL.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DPotent%2520Lpxc%2520inhibitors%2520with%2520in%2520vitro%2520activity%2520against%2520multidrug-resistant%2520pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D63%26spage%3De00977%26doi%3D10.1128%2FAAC.00977-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dozzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterday, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konradi, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miroshnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seroogy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhmin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlasichuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Optimization of Lpxc inhibitors for antibacterial activity and cardiovascular safety</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2Fcmdc.201900287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=31283109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1560-1572&author=F.+Cohenauthor=J.+B.+Aggenauthor=L.+D.+Andrewsauthor=Z.+Assarauthor=J.+Boggsauthor=T.+Choiauthor=P.+Dozzoauthor=A.+N.+Easterdayauthor=C.+M.+Haglundauthor=D.+J.+Hildebrandtauthor=M.+C.+Holtauthor=K.+Jolyauthor=A.+Jubbauthor=Z.+Kamalauthor=T.+R.+Kaneauthor=A.+W.+Konradiauthor=K.+M.+Krauseauthor=M.+S.+Linsellauthor=T.+D.+Machajewskiauthor=O.+Miroshnikovaauthor=H.+E.+Moserauthor=V.+Nietoauthor=T.+Phanauthor=C.+Platoauthor=A.+W.+Serioauthor=J.+Seroogyauthor=A.+Shakhminauthor=A.+J.+Steinauthor=A.+D.+Sunauthor=S.+Sviridovauthor=Z.+Wangauthor=K.+Wlasichukauthor=W.+Yangauthor=X.+Zhouauthor=H.+Zhuauthor=R.+T.+Cirz&title=Optimization+of+Lpxc+inhibitors+for+antibacterial+activity+and+cardiovascular+safety&doi=10.1002%2Fcmdc.201900287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span></div><div class="casAuthors">Cohen, Frederick; Aggen, James B.; Andrews, Logan D.; Assar, Zahra; Boggs, Jen; Choi, Taylor; Dozzo, Paola; Easterday, Ashton N.; Haglund, Cat M.; Hildebrandt, Darin J.; Holt, Melissa C.; Joly, Kristin; Jubb, Adrian; Kamal, Zeeshan; Kane, Timothy R.; Konradi, Andrei W.; Krause, Kevin M.; Linsell, Martin S.; Machajewski, Timothy D.; Miroshnikova, Olga; Moser, Heinz E.; Nieto, Vincent; Phan, Thu; Plato, Craig; Serio, Alisa W.; Seroogy, Julie; Shakhmin, Anton; Stein, Adam J.; Sun, Alex D.; Sviridov, Serguei; Wang, Zhan; Wlasichuk, Kenneth; Yang, Wen; Zhou, Xiaoming; Zhu, Hai; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1560-1572</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a Zn2+ deacetylase that is essential for the survival of most pathogenic Gram-neg. bacteria.  ACHN-975 (N-((S)-3-amino-1-(hydroxyamino)-3-methyl-1-oxobutan-2-yl)-4-(((1R,2R)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)benzamide) was the first LpxC inhibitor to reach human clin. testing and was discovered to have a dose-limiting cardiovascular toxicity of transient hypotension without compensatory tachycardia.  Herein we report the effort beyond ACHN-975 to discover LpxC inhibitors optimized for enzyme potency, antibacterial activity, pharmacokinetics, and cardiovascular safety.  Based on its overall profile, compd. 26 (LPXC-516, (S)-N-(2-(hydroxyamino)-1-(3-methoxy-1,1-dioxidothietan-3-yl)-2-oxoethyl)-4-(6-hydroxyhexa-1,3-diyn-1-yl)benzamide) was chosen for further development.  A phosphate prodrug of 26 was developed that provided a soly. of >30 mg mL-1 for parenteral administration and conversion into the active drug with a t1/2 of approx. two minutes.  Unexpectedly, and despite our optimization efforts, the prodrug of 26 still possesses a therapeutic window insufficient to support further clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhe0SbdQ2NbVg90H21EOLACvtfcHk0lgpQztuEHFdGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N&md5=b8cc87e1ebb76dfe8d330729a9d048d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900287%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DAssar%26aufirst%3DZ.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DT.%26aulast%3DDozzo%26aufirst%3DP.%26aulast%3DEasterday%26aufirst%3DA.%2BN.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHildebrandt%26aufirst%3DD.%2BJ.%26aulast%3DHolt%26aufirst%3DM.%2BC.%26aulast%3DJoly%26aufirst%3DK.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DKamal%26aufirst%3DZ.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DKonradi%26aufirst%3DA.%2BW.%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DLinsell%26aufirst%3DM.%2BS.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DMiroshnikova%26aufirst%3DO.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DPhan%26aufirst%3DT.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DSeroogy%26aufirst%3DJ.%26aulast%3DShakhmin%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DSun%26aufirst%3DA.%2BD.%26aulast%3DSviridov%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWlasichuk%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DOptimization%2520of%2520Lpxc%2520inhibitors%2520for%2520antibacterial%2520activity%2520and%2520cardiovascular%2520safety%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1560%26epage%3D1572%26doi%3D10.1002%2Fcmdc.201900287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Why hydroxamates may not be the best histone deacetylase inhibitors—what some may have forgotten or would rather forget?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2Fcmdc.201500486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=26603496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=15-21&author=S.+Shenauthor=A.+P.+Kozikowski&title=Why+hydroxamates+may+not+be+the+best+histone+deacetylase+inhibitors%E2%80%94what+some+may+have+forgotten+or+would+rather+forget%3F&doi=10.1002%2Fcmdc.201500486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Why Hydroxamates May Not Be the Best Histone Deacetylase Inhibitors-What Some May Have Forgotten or Would Rather Forget?</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-21</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Hydroxamate-based histone deacetylase inhibitors (HDACIs) have been approved as therapeutic agents by the US Food and Drug Administration for use in oncol. applications.  While the potential utility of such HDACIs in other areas of medicinal chem. is tremendous, there are significant concerns that "pan-HDAC inhibitors" may be too broadly acting and/or toxic for clin. use beyond oncol.  In addn. to the isoenzyme selectivity challenge, the potential mutagenicity of hydroxamate-contg. HDAC inhibitors represents a major hindrance in their application to other therapeutic areas.  Herein the authors report on the mutagenicity of known hydroxamates, discuss the mechanisms responsible for their genotoxicity, and review some of the current alternatives to hydroxamates.  The authors conclude that the hydroxamate group, while providing high-potency HDACIs, is not necessarily the best zinc-binding group for HDACI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplftHMFVoN6rVg90H21EOLACvtfcHk0lhlhBpzddcwKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL&md5=5183d2c0a4c86fe67bea480a66bc5dc6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500486%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DWhy%2520hydroxamates%2520may%2520not%2520be%2520the%2520best%2520histone%2520deacetylase%2520inhibitors%25E2%2580%2594what%2520some%2520may%2520have%2520forgotten%2520or%2520would%2520rather%2520forget%253F%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D15%26epage%3D21%26doi%3D10.1002%2Fcmdc.201500486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenbroucke, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span> <span> </span><span class="NLM_article-title">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd4390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnrd4390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=25376097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=904-927&author=R.+E.+Vandenbrouckeauthor=C.+Libert&title=Is+there+new+hope+for+therapeutic+matrix+metalloproteinase+inhibition%3F&doi=10.1038%2Fnrd4390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Is there new hope for therapeutic matrix metalloproteinase inhibition?</span></div><div class="casAuthors">Vandenbroucke, Roosmarijn E.; Libert, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">904-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent endopeptidases that form a family of 24 members in mammals.  Evidence of the pathol. roles of MMPs in various diseases, combined with their druggability, has made them attractive therapeutic targets.  Initial drug discovery efforts focused on the roles of MMPs in cancer progression, and more than 50 MMP inhibitors have been investigated in clin. trials in various cancers.  However, all of these trials failed.  Reasons for failure include the lack of inhibitor specificity and insufficient knowledge about the complexity of the disease biol.  MMPs are also known to be involved in several inflammatory processes, and there are new therapeutic opportunities for MMP inhibitors to treat such diseases.  In this Review, we discuss the recent advances made in understanding the role of MMPs in inflammatory diseases and the therapeutic potential of MMP inhibition in those conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpua2GBt8P5b7Vg90H21EOLACvtfcHk0lhlhBpzddcwKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtb3I&md5=032265fe59e9f7567e4d3c7d072b3cde</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4390%26sid%3Dliteratum%253Aachs%26aulast%3DVandenbroucke%26aufirst%3DR.%2BE.%26aulast%3DLibert%26aufirst%3DC.%26atitle%3DIs%2520there%2520new%2520hope%2520for%2520therapeutic%2520matrix%2520metalloproteinase%2520inhibition%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D904%26epage%3D927%26doi%3D10.1038%2Fnrd4390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Pyridone methylsulfone hydroxamate Lpxc inhibitors for the treatment of serious gram-negative infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1670</span>, <span class="refDoi"> DOI: 10.1021/jm2014875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1662-1670&author=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=L.+M.+Priceauthor=J.+A.+Abramiteauthor=J.+Arcariauthor=R.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=S.+W.+Chungauthor=E.+M.+Collantesauthor=C.+Desbonnetauthor=M.+Doroskiauthor=J.+Dotyauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=L.+McAllisterauthor=E.+McElroyauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=R.+Oliverauthor=J.+Penzienauthor=M.+Plummerauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=A.+P.+Tomarasauthor=D.+P.+Uccelloauthor=A.+Vazauthor=D.+G.+Wishka&title=Pyridone+methylsulfone+hydroxamate+Lpxc+inhibitors+for+the+treatment+of+serious+gram-negative+infections&doi=10.1021%2Fjm2014875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridone methylsulfone hydroxamate LpxC inhibitors for the treatment of serious gram-negative infections</span></div><div class="casAuthors">Montgomery, Justin I.; Brown, Matthew F.; Reilly, Usa; Price, Loren M.; Abramite, Joseph A.; Arcari, Joel; Barham, Rose; Che, Ye; Chen, Jinshan Michael; Chung, Seung Won; Collantes, Elizabeth M.; Desbonnet, Charlene; Doroski, Matthew; Doty, Jonathan; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; McAllister, Laura; McElroy, Eric; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa; Noe, Mark C.; O'Donnell, John; Oliver, Robert; Penzien, Joseph; Plummer, Mark; Shanmugasundaram, Veerabahu; Thoma, Christy; Tomaras, Andrew P.; Uccello, Daniel P.; Vaz, Alfin; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1670</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. activity of a new series of LpxC inhibitors represented by pyridone methylsulfone hydroxamate 2a (I) is presented.  Members of this series have improved soly. and free fraction when compared to compds. in the previously described biphenyl methylsulfone hydroxamate series, and they maintain superior Gram-neg. antibacterial activity to comparator agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgNGd-bEXEabVg90H21EOLACvtfcHk0lhlhBpzddcwKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D&md5=f2410aae177b2fa24d8da72942b95564</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm2014875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014875%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcAllister%26aufirst%3DL.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DVaz%26aufirst%3DA.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPyridone%2520methylsulfone%2520hydroxamate%2520Lpxc%2520inhibitors%2520for%2520the%2520treatment%2520of%2520serious%2520gram-negative%2520infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1662%26epage%3D1670%26doi%3D10.1021%2Fjm2014875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the bacterial deacetylase Lpxc bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">34073</span>– <span class="NLM_lpage">34080</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.513028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1074%2Fjbc.M113.513028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=24108127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=34073-34080&author=G.+M.+Claytonauthor=D.+J.+Kleinauthor=K.+W.+Rickertauthor=S.+B.+Patelauthor=M.+Kornienkoauthor=J.+Zugay-Murphyauthor=J.+C.+Reidauthor=S.+Tummalaauthor=S.+Sharmaauthor=S.+B.+Singhauthor=L.+Mieselauthor=K.+J.+Lumbauthor=S.+M.+Soisson&title=Structure+of+the+bacterial+deacetylase+Lpxc+bound+to+the+nucleotide+reaction+product+reveals+mechanisms+of+oxyanion+stabilization+and+proton+transfer&doi=10.1074%2Fjbc.M113.513028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the bacterial deacetylase LpxC bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span></div><div class="casAuthors">Clayton, Gina M.; Klein, Daniel J.; Rickert, Keith W.; Patel, Sangita B.; Kornienko, Maria; Zugay-Murphy, Joan; Reid, John C.; Tummala, Srivanya; Sharma, Sujata; Singh, Sheo B.; Miesel, Lynn; Lumb, Kevin J.; Soisson, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">34073-34080</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant strains of pathogenic bacteria is an increasing threat to global health that underscores an urgent need for an expanded antibacterial armamentarium.  Gram-neg. bacteria, such as Escherichia coli, have become increasingly important clin. pathogens with limited treatment options.  This is due in part to their lipopolysaccharide (LPS) outer membrane components, which dually serve as endotoxins while also protecting Gram-neg. bacteria from antibiotic entry.  The LpxC enzyme, UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase, catalyzes the committed step of LPS biosynthesis, making LpxC a promising target for new antibacterials.  Here, the authors present the 1st structure of E. coli LpxC in complex with the deacetylation reaction product, UDP-[3-O-(R-3-hydroxymyristoyl)]glucosamine.  These studies provide valuable insight into the recognition of substrates and products by LpxC and a platform for structure-guided drug discovery of broad spectrum Gram-neg. antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm8dRnYgccKbVg90H21EOLACvtfcHk0liiU93sFbISug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO&md5=ac4adc79fe696089087d35a29e39f892</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.513028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.513028%26sid%3Dliteratum%253Aachs%26aulast%3DClayton%26aufirst%3DG.%2BM.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DTummala%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26atitle%3DStructure%2520of%2520the%2520bacterial%2520deacetylase%2520Lpxc%2520bound%2520to%2520the%2520nucleotide%2520reaction%2520product%2520reveals%2520mechanisms%2520of%2520oxyanion%2520stabilization%2520and%2520proton%2520transfer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D34073%26epage%3D34080%26doi%3D10.1074%2Fjbc.M113.513028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herradura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poutsiaka, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahamoon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.,  3rd</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of Lpxc inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1016%2Fj.bmcl.2017.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=28302397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksF2qtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1670-1680&author=J.+Zhangauthor=A.+Chanauthor=B.+Lippaauthor=J.+B.+Crossauthor=C.+Liuauthor=N.+Yinauthor=J.+A.+Romeroauthor=J.+Lawrenceauthor=R.+Heneyauthor=P.+Herraduraauthor=J.+Gossauthor=C.+Clarkauthor=C.+Abelauthor=Y.+Zhangauthor=K.+M.+Poutsiakaauthor=F.+Epieauthor=M.+Conradauthor=A.+Mahamoonauthor=K.+Nguyenauthor=A.+Chavanauthor=E.+Clarkauthor=T.+C.+Liauthor=R.+K.+Chengauthor=M.+Woodauthor=O.+A.+Andersenauthor=M.+Brooksauthor=J.+Kwongauthor=J.+Barkerauthor=I.+B.+Parrauthor=Y.+Guauthor=M.+D.+Ryanauthor=S.+Colemanauthor=C.+A.+Metcalf&title=Structure-based+discovery+of+Lpxc+inhibitors&doi=10.1016%2Fj.bmcl.2017.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of LpxC inhibitors</span></div><div class="casAuthors">Zhang, Jing; Chan, Audrey; Lippa, Blaise; Cross, Jason B.; Liu, Christopher; Yin, Ning; Romero, Jan Antoinette C.; Lawrence, Jonathan; Heney, Ryan; Herradura, Prudencio; Goss, Jennifer; Clark, Cynthia; Abel, Cassandra; Zhang, Yanzhi; Poutsiaka, Katherine M.; Epie, Felix; Conrad, Mary; Mahamoon, Azard; Nguyen, Kien; Chavan, Ajit; Clark, Edward; Li, Tong-chuan; Cheng, Robert K.; Wood, Michael; Andersen, Ole A.; Brooks, Mark; Kwong, Jason; Barker, John; Parr, Ian Barrie; Gu, Yugui; Ryan, M. Dominic; Coleman, Scott; Metcalf, Chester A., III</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1670-1680</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The emergence and spread of multidrug-resistant (MDR) Gram neg. bacteria presents a serious threat for public health.  Novel antimicrobials that could overcome the resistance problems are urgently needed.  UDP-3-O-(R-3-hydroxymyristol)-N-acetyl glucosamine deacetylase (LpxC) is a cytosolic zinc-based deacetylase that catalyzes the first committed step in the biosynthesis of lipid A, which is essential for the survival of Gram-neg. bacteria.  Our efforts toward the discovery of novel LpxC inhibitors are presented herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEwaZ2nN26t7Vg90H21EOLACvtfcHk0liiU93sFbISug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksF2qtr8%253D&md5=65edd36115b7b96c3e6968d3bb7e93de</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DA.%26aulast%3DLippa%26aufirst%3DB.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DRomero%26aufirst%3DJ.%2BA.%26aulast%3DLawrence%26aufirst%3DJ.%26aulast%3DHeney%26aufirst%3DR.%26aulast%3DHerradura%26aufirst%3DP.%26aulast%3DGoss%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DC.%26aulast%3DAbel%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPoutsiaka%26aufirst%3DK.%2BM.%26aulast%3DEpie%26aufirst%3DF.%26aulast%3DConrad%26aufirst%3DM.%26aulast%3DMahamoon%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DChavan%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DT.%2BC.%26aulast%3DCheng%26aufirst%3DR.%2BK.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DO.%2BA.%26aulast%3DBrooks%26aufirst%3DM.%26aulast%3DKwong%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DI.%2BB.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DM.%2BD.%26aulast%3DColeman%26aufirst%3DS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520Lpxc%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1670%26epage%3D1680%26doi%3D10.1016%2Fj.bmcl.2017.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baurin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of libraries of molecular fragments for use in nmr screening against protein targets</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2166</span>, <span class="refDoi"> DOI: 10.1021/ci049806z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-Elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+libraries+of+molecular+fragments+for+use+in+nmr+screening+against+protein+targets&doi=10.1021%2Fci049806z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0lh_ldSZsZ7Hng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-Ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520libraries%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520nmr%2520screening%2520against%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166%26doi%3D10.1021%2Fci049806z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: Analysis of output from multiple screening campaigns</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-620&author=I.+J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+Analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0lh_ldSZsZ7Hng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520Analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D620%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span> <span> </span><span class="NLM_article-title">The seeds approach: Integrating fragments into drug discovery</span>. <i>Curr. Top Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1568</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.2174/156802607782341109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&author=R.+E.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+J.+Drysdale&title=The+seeds+approach%3A+Integrating+fragments+into+drug+discovery&doi=10.2174%2F156802607782341109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0lh_ldSZsZ7Hng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520seeds%2520approach%253A%2520Integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1568%26epage%3D1581%26doi%3D10.2174%2F156802607782341109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span> <span> </span><span class="NLM_article-title">Experiences in fragment-based lead discovery</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-381274-2.00020-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1016%2FB978-0-12-381274-2.00020-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=21371604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2011&pages=509-531&author=R.+E.+Hubbardauthor=J.+B.+Murray&title=Experiences+in+fragment-based+lead+discovery&doi=10.1016%2FB978-0-12-381274-2.00020-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based lead discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Murray, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Design</span>),
    <span class="NLM_cas:pages">509-531</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  This chapter summarizes the experience at Vernalis over the past decade in developing and applying fragment-based discovery methods across a range of different targets.  The emphasis will be on the practical aspects of the different biophys. techniques (surface plasmon resonance (SPR), differential scanning fluorimetry (DSF), isothermal titrn. calorimetry, NMR, and X-ray crystallog.) that can be used to identify fragments that bind to targets and a discussion of the criteria and strategies for selecting and evolving fragments to lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbWFWP7PylG7Vg90H21EOLACvtfcHk0lhaCjZRSXBUHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D&md5=358abcb0648a3482c6be40f5b3d04ff9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-381274-2.00020-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-381274-2.00020-0%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26atitle%3DExperiences%2520in%2520fragment-based%2520lead%2520discovery%26jtitle%3DMethods%2520Enzymol.%26date%3D2011%26volume%3D493%26spage%3D509%26epage%3D531%26doi%3D10.1016%2FB978-0-12-381274-2.00020-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Credille, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamek, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure–activity relationships in metal-binding pharmacophores for influenza endonuclease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10206</span>– <span class="NLM_lpage">10217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01363</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01363" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFalur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10206-10217&author=C.+V.+Credilleauthor=B.+L.+Dickauthor=C.+N.+Morrisonauthor=R.+W.+Stokesauthor=R.+N.+Adamekauthor=N.+C.+Wuauthor=I.+A.+Wilsonauthor=S.+M.+Cohen&title=Structure%E2%80%93activity+relationships+in+metal-binding+pharmacophores+for+influenza+endonuclease&doi=10.1021%2Facs.jmedchem.8b01363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships in Metal-Binding Pharmacophores for Influenza Endonuclease</span></div><div class="casAuthors">Credille, Cy V.; Dick, Benjamin L.; Morrison, Christine N.; Stokes, Ryjul W.; Adamek, Rebecca N.; Wu, Nicholas C.; Wilson, Ian A.; Cohen, Seth M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10206-10217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloenzymes represent an important target space for drug discovery.  A limitation to the early development of metalloenzyme inhibitors has been the lack of established structure-activity relationships (SARs) for mols. that bind the metal ion cofactor(s) of a metalloenzyme.  Herein, we employed a bioinorg. perspective to develop an SAR for inhibition of the metalloenzyme influenza RNA polymerase PAN endonuclease.  The identified trends highlight the importance of the electronics of the metal-binding pharmacophore (MBP), in addn. to MBP sterics, for achieving improved inhibition and selectivity.  By optimization of the MBPs for PAN endonuclease, a class of highly active and selective fragments was developed that displays IC50 values <50 nM.  This SAR led to structurally distinct mols. that also displayed IC50 values of ∼10 nM, illustrating the utility of a metal-centric development campaign in generating highly active and selective metalloenzyme inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_SlKgVyqXmLVg90H21EOLACvtfcHk0lhaCjZRSXBUHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFalur%252FP&md5=4d4faaebd25518757c2ba909d7eb2040</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01363%26sid%3Dliteratum%253Aachs%26aulast%3DCredille%26aufirst%3DC.%2BV.%26aulast%3DDick%26aufirst%3DB.%2BL.%26aulast%3DMorrison%26aufirst%3DC.%2BN.%26aulast%3DStokes%26aufirst%3DR.%2BW.%26aulast%3DAdamek%26aufirst%3DR.%2BN.%26aulast%3DWu%26aufirst%3DN.%2BC.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DCohen%26aufirst%3DS.%2BM.%26atitle%3DStructure%25E2%2580%2593activity%2520relationships%2520in%2520metal-binding%2520pharmacophores%2520for%2520influenza%2520endonuclease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10206%26epage%3D10217%26doi%3D10.1021%2Facs.jmedchem.8b01363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Current perspectives in fragment-based lead discovery (fbld)</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1042/EBC20170028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1042%2FEBC20170028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=29118093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=453-464&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Current+perspectives+in+fragment-based+lead+discovery+%28fbld%29&doi=10.1042%2FEBC20170028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Current perspectives in fragment-based lead discovery (FBLD)</span></div><div class="casAuthors">Lamoree Bas; Hubbard Roderick E; Hubbard Roderick E</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">453-464</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is over 20 years since the first fragment-based discovery projects were disclosed.  The methods are now mature for most 'conventional' targets in drug discovery such as enzymes (kinases and proteases) but there has also been growing success on more challenging targets, such as disruption of protein-protein interactions.  The main application is to identify tractable chemical startpoints that non-covalently modulate the activity of a biological molecule.  In this essay, we overview current practice in the methods and discuss how they have had an impact in lead discovery - generating a large number of fragment-derived compounds that are in clinical trials and two medicines treating patients.  In addition, we discuss some of the more recent applications of the methods in chemical biology - providing chemical tools to investigate biological molecules, mechanisms and systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0f6BottYdIAWfW6udTcc2eYmWDAHoEipbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D&md5=35bce3c5978329c6737ffd719135d363</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1042%2FEBC20170028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170028%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DCurrent%2520perspectives%2520in%2520fragment-based%2520lead%2520discovery%2520%2528fbld%2529%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D453%26epage%3D464%26doi%3D10.1042%2FEBC20170028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: The impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+The+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lhaCjZRSXBUHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520The%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Fragment Based Lead Discovery</span>, In  <i>The Handbook of Medicinal Chemistry: Principles and Practice</i>; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">153</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1039%2F9781782621836-00122" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=122-153&author=R.+E.+Hubbard&title=The+Handbook+of+Medicinal+Chemistry%3A+Principles+and+Practice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2F9781782621836-00122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781782621836-00122%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DFragment%2520Based%2520Lead%2520Discovery%26btitle%3DThe%2520Handbook%2520of%2520Medicinal%2520Chemistry%253A%2520Principles%2520and%2520Practice%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D122%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">The Role of Fragment-Based Discovery in Lead Finding</span>, In  <i>Fragment-Based Drug Discovery Lessons and Outlook</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span>; <span class="NLM_string-name">Jahnke, W.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2016</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">36</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1002%2F9783527683604.ch01" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1-36&author=R.+E.+Hubbardauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-Based+Drug+Discovery+Lessons+and+Outlook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9783527683604.ch01&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527683604.ch01%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DThe%2520Role%2520of%2520Fragment-Based%2520Discovery%2520in%2520Lead%2520Finding%26btitle%3DFragment-Based%2520Drug%2520Discovery%2520Lessons%2520and%2520Outlook%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley%26date%3D2016%26spage%3D1%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coignard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijkmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. G.</span></span> <span> </span><span class="NLM_article-title">Antibacterial activities and characterization of novel inhibitors of Lpxc</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.6.1793-1799.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1128%2FAAC.46.6.1793-1799.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=12019092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFCms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=1793-1799&author=J.+M.+Clementsauthor=F.+Coignardauthor=I.+Johnsonauthor=S.+Chandlerauthor=S.+Palanauthor=A.+Wallerauthor=J.+Wijkmansauthor=M.+G.+Hunter&title=Antibacterial+activities+and+characterization+of+novel+inhibitors+of+Lpxc&doi=10.1128%2FAAC.46.6.1793-1799.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial activities and characterization of novel inhibitors of LpxC</span></div><div class="casAuthors">Clements, John M.; Coignard, Fanny; Johnson, Ian; Chandler, Stephen; Palan, Shilpa; Waller, Andrew; Wijkmans, Jac; Hunter, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1793-1799</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Lipid A is the hydrophobic anchor of lipopolysaccharide (LPS) and forms the major lipid component of the outer monolayer of the outer membrane of gram-neg. bacteria.  Lipid A is required for bacterial growth and virulence, and inhibition of its biosynthesis is lethal to bacteria.  UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a metalloenzyme that catalyzes the second step in the biosynthesis of lipid A.  Inhibitors of LpxC have previously been shown to have antibiotic activities.  We have screened a metalloenzyme inhibitor library for antibacterial activities against an Escherichia coli strain with reduced LpxC activity.  From this screen, a series of sulfonamide derivs. of the α-(R)-amino hydroxamic acids, exemplified by BB-78484 and BB-78485, have been identified as having potent inhibitory activities against LpxC in an in vitro assay.  Leads from this series showed gram-neg. selective activities against members of the Enterobacteriaceae, Serratia marcescens, Morganella morganii, Haemophilus influenzae, Moraxella catarrhalis, and Burkholderia cepacia.  BB-78484 was bactericidal against E. coli, achieving 3-log killing in 4 h at a concn. 4 times above the MIC, as would be predicted for an inhibitor of lipid A biosynthesis.  E. coli mutants with decreased susceptibility to BB-78484 were selected.  Anal. of these mutants revealed that resistance arose as a consequence of mutations in the fabZ or lpxC genes.  These data confirm the antibacterial target of BB-78484 and BB-78485 and validate LpxC as a target for gram-neg. selective antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN9-hhfLvqObVg90H21EOLACvtfcHk0likUjhWzPEmAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFCms7k%253D&md5=c7830984a5330b363a687e23f399faa4</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.6.1793-1799.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.6.1793-1799.2002%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DJ.%2BM.%26aulast%3DCoignard%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DI.%26aulast%3DChandler%26aufirst%3DS.%26aulast%3DPalan%26aufirst%3DS.%26aulast%3DWaller%26aufirst%3DA.%26aulast%3DWijkmans%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DM.%2BG.%26atitle%3DAntibacterial%2520activities%2520and%2520characterization%2520of%2520novel%2520inhibitors%2520of%2520Lpxc%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D1793%26epage%3D1799%26doi%3D10.1128%2FAAC.46.6.1793-1799.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0likUjhWzPEmAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the ccp4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+ccp4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhe0Oe7kQJgrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520ccp4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schüttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span> <span> </span><span class="NLM_article-title">Prodrg: A tool for high-throughput crystallography of protein-ligand complexes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1107/S0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Sch%C3%BCttelkopfauthor=D.+M.+F.+van+Aalten&title=Prodrg%3A+A+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2FS0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1355-1363</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The small-mol. topol. generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and mol. topologies suitable for x-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compds. from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the std. libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallog. R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbflrch0bs97Vg90H21EOLACvtfcHk0lhe0Oe7kQJgrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D&md5=f9bfdb14be7b04ea5c0db67feb39f72a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DProdrg%253A%2520A%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2FS0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhe0Oe7kQJgrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgUoUZHyeonfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapierre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grobelny, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casabonne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holme, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadzan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardy, R. E.</span></span> <span> </span><span class="NLM_article-title">Matrix metalloproteinase inhibitors: A structure-activity study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1021/jm970494j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970494j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;key=1%3ACAS%3A528%3ADyaK1cXptFGl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=199-223&author=D.+E.+Levyauthor=F.+Lapierreauthor=W.+Liangauthor=W.+Yeauthor=C.+W.+Langeauthor=X.+Liauthor=D.+Grobelnyauthor=M.+Casabonneauthor=D.+Tyrrellauthor=K.+Holmeauthor=A.+Nadzanauthor=R.+E.+Galardy&title=Matrix+metalloproteinase+inhibitors%3A+A+structure-activity+study&doi=10.1021%2Fjm970494j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Matrix Metalloproteinase Inhibitors: A Structure Activity Study</span></div><div class="casAuthors">Levy, Daniel E.; Lapierre, France; Liang, Weisheng; Ye, Wenqing; Lange, Christopher W.; Li, Xiaoyuan; Grobelny, Damian; Casabonne, Marie; Tyrrell, David; Holme, Kevin; Nadzan, Alex; Galardy, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-223</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modifications around the dipeptide-mimetic core of hydroxamic acid based matrix metalloproteinase inhibitors I [AA = L-Trp, D-Trp, L-Trp(Me), L-3-benzothienylalanine, L-1- and -2-naphthylalanine, L-3- and -8-quinolylalanine, L-4-phenylphenylalanine, L-Phe, L-3- and -4-pyridylalanine, L-tert-leucine, L-abrine; R6 = NHMe, NH(CH2)4Me, NHCH2CH2OH, NHCH2CH2NHCO2CH2Ph, cyclopropylamino, cyclopentylamino, (S)- and (R)-1-indanylamino, (1R,2S)- and (1S,2R)-2-hydroxy-1-indanylamino, (S)-NHCHMePh, NHCH2Ph, piperonylamino, 2-, 3-, and 4-pyridylmethylamino, 2-(4-pyridyl)ethylamino, NHCH2CH2C6H4OH-4, 2-furanylmethylamino, 2-thiazolylmethylamino, 2-benzimidazolylamino, 3-(1-imidazolyl)propylamino, 3-(4-morpholinyl)propylamino; R2 = H, OH; R3 = CH2CHMe2, Bu, n-hexyl, n-octyl, OCHMe2, O(CH2)4Me] were studied.  These variations incorporated a variety of natural, unnatural, and synthetic amino acids in addn. to modifications of the P1' and P3' substituents.  The results of this study indicate the following structural requirements: (1) Two key hydrogen bonds must be present between the enzyme and potent substrates.  (2) Potent inhibitors must possess potent zinc-binding functionalities.  (3) The potential importance of the hydrophobic group at position R3 as illustrated by its ability to impart greater relative potency against stromelysin when larger hydrophobic groups are used.  (4) Requirements surrounding the nature of the amino acid appear to be more restrictive for stromelysin than for neutrophil collagenase, 72 kDa gelatinase, and 92 kDa gelatinase.  These requirements may involve planar fused-ring aryl systems and possibly hydrogen-bonding capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2kkOPfSKi9LVg90H21EOLACvtfcHk0lgUoUZHyeonfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXptFGl&md5=d1094c574641b569df5b33b962c48d52</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm970494j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970494j%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DLapierre%26aufirst%3DF.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DW.%26aulast%3DLange%26aufirst%3DC.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGrobelny%26aufirst%3DD.%26aulast%3DCasabonne%26aufirst%3DM.%26aulast%3DTyrrell%26aufirst%3DD.%26aulast%3DHolme%26aufirst%3DK.%26aulast%3DNadzan%26aufirst%3DA.%26aulast%3DGalardy%26aufirst%3DR.%2BE.%26atitle%3DMatrix%2520metalloproteinase%2520inhibitors%253A%2520A%2520structure-activity%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D199%26epage%3D223%26doi%3D10.1021%2Fjm970494j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">CLSI</span>. <span> </span><span class="NLM_article-title">Performance Standards for Antimicrobial Susceptibility
Testing</span>. In  <i>CLSI Supplement M100-S26</i>, <span class="NLM_edition">27</span>th ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, P. A.</span></span>, Ed.; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=CLSIauthor=P.+A.+Wayne&title=CLSI+Supplement+M100-S26"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPerformance%2520Standards%2520for%2520Antimicrobial%2520Susceptibility%250ATesting%26btitle%3DCLSI%2520Supplement%2520M100-S26%26aulast%3DWayne%26aufirst%3DP.%2BA.%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MDT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MDT','PDB','4MDT'); return false;">PDB: 4MDT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UPG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UPG','PDB','5UPG'); return false;">PDB: 5UPG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM','PDB','3UHM'); return false;">PDB: 3UHM</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01215">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78677"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01215?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01215</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis of compounds <b>9</b>–<b>24</b> and <b>26</b>–<b>32</b>; structural biology data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_001.pdf">jm0c01215_si_001.pdf (2.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01215/suppl_file/jm0c01215_si_002.csv">jm0c01215_si_002.csv (2.78 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The refined coordinates of the structures presented in this manuscript are deposited at the RCSB with the following codes: <b>6</b>: 7CIC, <b>7</b>: 7CIB, <b>8:</b> 7CIA, <b>10</b>: 7CIE, <b>14</b>: 7CI4, <b>16</b>: 7CI5, <b>22</b>: 7CI7, <b>27</b>: 7CID, <b>29</b>: 7CI6, <b>38</b>: 7CI8, and <b>43</b>: 7CI9. The authors will release the atomic coordinates and experimental data upon article publication</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01215&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01215" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01215" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679932442e10d956","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
