
<html lang="en"     class="pb-page"  data-request-id="98649601-3a10-4cb1-873a-b37afb2816c2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.5b01311;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design and Structural Characterization of Potent and Selective Inhibitors of Phosphatidylinositol 4 Kinase IIIβ" /></meta><meta name="dc.Creator" content="Florentine U.  Rutaganira" /></meta><meta name="dc.Creator" content="Melissa L.  Fowler" /></meta><meta name="dc.Creator" content="Jacob A.  McPhail" /></meta><meta name="dc.Creator" content="Michael A.  Gelman" /></meta><meta name="dc.Creator" content="Khanh  Nguyen" /></meta><meta name="dc.Creator" content="Anming  Xiong" /></meta><meta name="dc.Creator" content="Gillian L.  Dornan" /></meta><meta name="dc.Creator" content="Brandon  Tavshanjian" /></meta><meta name="dc.Creator" content="Jeffrey S.  Glenn" /></meta><meta name="dc.Creator" content="Kevan M.  Shokat" /></meta><meta name="dc.Creator" content="John E.  Burke" /></meta><meta name="dc.Description" content="Type III phosphatidylinositol 4-kinase (PI4KIIIβ) is an essential enzyme in mediating membrane trafficking and is implicated in a variety of pathogenic processes. It is a key host factor mediating ..." /></meta><meta name="Description" content="Type III phosphatidylinositol 4-kinase (PI4KIIIβ) is an essential enzyme in mediating membrane trafficking and is implicated in a variety of pathogenic processes. It is a key host factor mediating ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 26, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01311" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01311" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01311" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01311" /></link>
        
    
    

<title>Design and Structural Characterization of Potent and Selective Inhibitors of Phosphatidylinositol 4 Kinase IIIβ | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01311" /></meta><meta property="og:title" content="Design and Structural Characterization of Potent and Selective Inhibitors of Phosphatidylinositol 4 Kinase IIIβ" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0007.jpeg" /></meta><meta property="og:description" content="Type III phosphatidylinositol 4-kinase (PI4KIIIβ) is an essential enzyme in mediating membrane trafficking and is implicated in a variety of pathogenic processes. It is a key host factor mediating replication of RNA viruses. The design of potent and specific inhibitors of this enzyme will be essential to define its cellular roles and may lead to novel antiviral therapeutics. We previously reported the PI4K inhibitor PIK93, and this compound has defined key functions of PI4KIIIβ. However, this compound showed high cross reactivity with class I and III PI3Ks. Using structure-based drug design, we have designed novel potent and selective (&gt;1000-fold over class I and class III PI3Ks) PI4KIIIβ inhibitors. These compounds showed antiviral activity against hepatitis C virus. The co-crystal structure of PI4KIIIβ bound to one of the most potent compounds reveals the molecular basis of specificity. This work will be vital in the design of novel PI4KIIIβ inhibitors, which may play significant roles as antiviral therapeutics." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01311"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01311">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01311&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01311&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01311&amp;href=/doi/10.1021/acs.jmedchem.5b01311" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1830-1839</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01293" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01331" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Structural Characterization of Potent and Selective Inhibitors of Phosphatidylinositol 4 Kinase IIIβ</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florentine+U.++Rutaganira">Florentine U. Rutaganira</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Melissa+L.++Fowler">Melissa L. Fowler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacob+A.++McPhail">Jacob A. McPhail</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+A.++Gelman">Michael A. Gelman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Khanh++Nguyen">Khanh Nguyen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anming++Xiong">Anming Xiong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gillian+L.++Dornan">Gillian L. Dornan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brandon++Tavshanjian">Brandon Tavshanjian</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+S.++Glenn">Jeffrey S. Glenn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevan+M.++Shokat">Kevan M. Shokat</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+E.++Burke">John E. Burke</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Howard Hughes Medical Institute and Department of Cellular and Molecular Pharmacology, University of California, San Francisco (UCSF), San Francisco, California 94143, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Biochemistry and Microbiology, University of Victoria, Victoria, BC V8W 2Y2, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Medicine and Department of Microbiology & Immunology, Stanford University, Palo Alto, California 94305, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Veterans Administration Medical Center, Palo Alto, California 94304, United States</span></div><div class="corresp-info"><strong>*</strong>(K.M.S.) E-mail: <a href="/cdn-cgi/l/email-protection#5c37392a3d32722f3433373d281c293f2f3a72393829"><span class="__cf_email__" data-cfemail="63080615020d4d100b0c08021723160010054d060716">[email protected]</span></a>. Tel.: 1-415-514-0472.</div><div class="corresp-info"><strong>*</strong>(J.E.B.) E-mail: <a href="/cdn-cgi/l/email-protection#670d020512150c022712110e04490406"><span class="__cf_email__" data-cfemail="97fdf2f5e2e5fcf2d7e2e1fef4b9f4f6">[email protected]</span></a>. Tel.: 1-250-721-8732.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01311&amp;href=/doi/10.1021%2Facs.jmedchem.5b01311" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1830–1839</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 17, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 August 2015</li><li><span class="item_label"><b>Published</b> online</span>26 February 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01311" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01311</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1830%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFlorentine%2BU.%2BRutaganira%252C%2BMelissa%2BL.%2BFowler%252C%2BJacob%2BA.%2BMcPhail%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D5%26contentID%3Dacs.jmedchem.5b01311%26title%3DDesign%2Band%2BStructural%2BCharacterization%2Bof%2BPotent%2Band%2BSelective%2BInhibitors%2Bof%2BPhosphatidylinositol%2B4%2BKinase%2BIII%25CE%25B2%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1839%26publicationDate%3DMarch%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01311"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2224</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">36</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01311" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Structural Characterization of Potent and Selective Inhibitors of Phosphatidylinositol 4 Kinase IIIβ&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Florentine&quot;,&quot;last_name&quot;:&quot;U. Rutaganira&quot;},{&quot;first_name&quot;:&quot;Melissa&quot;,&quot;last_name&quot;:&quot;L. Fowler&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;A. McPhail&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;A. Gelman&quot;},{&quot;first_name&quot;:&quot;Khanh&quot;,&quot;last_name&quot;:&quot;Nguyen&quot;},{&quot;first_name&quot;:&quot;Anming&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Gillian&quot;,&quot;last_name&quot;:&quot;L. Dornan&quot;},{&quot;first_name&quot;:&quot;Brandon&quot;,&quot;last_name&quot;:&quot;Tavshanjian&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;S. Glenn&quot;},{&quot;first_name&quot;:&quot;Kevan&quot;,&quot;last_name&quot;:&quot;M. Shokat&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;E. Burke&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1830-1839&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01311&quot;},&quot;abstract&quot;:&quot;Type III phosphatidylinositol 4-kinase (PI4KIIIβ) is an essential enzyme in mediating membrane trafficking and is implicated in a variety of pathogenic processes. It is a key host factor mediating replication of RNA viruses. The design of potent and specific inhibitors of this enzyme will be essential to define its cellular roles and may lead to novel antiviral therapeutics. We previously reported the PI4K inhibitor PIK93, and this compound has defined key functions of PI4KIIIβ. However, this compound showed high cross reactivity with class I and III PI3Ks. Using structure-based drug design, we have designed novel potent and selective (&gt;1000-fold over class I and class III PI3Ks) PI4KIIIβ inhibitors. These compounds showed antiviral activity against hepatitis C virus. The co-crystal structure of PI4KIIIβ bound to one of the most potent compounds reveals the molecular basis of specificity. This work will be vital in the design of novel PI4KIIIβ inhibitors, which may play significant roles as antiviral therapeu&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01311&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01311" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01311&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01311" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01311&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01311" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01311&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01311&amp;href=/doi/10.1021/acs.jmedchem.5b01311" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01311" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01311" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01311%26sid%3Dliteratum%253Aachs%26pmid%3D26885694%26genre%3Darticle%26aulast%3DRutaganira%26date%3D2016%26atitle%3DDesign%2Band%2BStructural%2BCharacterization%2Bof%2BPotent%2Band%2BSelective%2BInhibitors%2Bof%2BPhosphatidylinositol%2B4%2BKinase%2BIII%25CE%25B2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D5%26spage%3D1830%26epage%3D1839%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291137" title="Lipids">Lipids</a>,</li><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/jmcmar.2016.59.issue-5/20160310-01/jmcmar.2016.59.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Type III phosphatidylinositol 4-kinase (PI4KIIIβ) is an essential enzyme in mediating membrane trafficking and is implicated in a variety of pathogenic processes. It is a key host factor mediating replication of RNA viruses. The design of potent and specific inhibitors of this enzyme will be essential to define its cellular roles and may lead to novel antiviral therapeutics. We previously reported the PI4K inhibitor PIK93, and this compound has defined key functions of PI4KIIIβ. However, this compound showed high cross reactivity with class I and III PI3Ks. Using structure-based drug design, we have designed novel potent and selective (>1000-fold over class I and class III PI3Ks) PI4KIIIβ inhibitors. These compounds showed antiviral activity against hepatitis C virus. The co-crystal structure of PI4KIIIβ bound to one of the most potent compounds reveals the molecular basis of specificity. This work will be vital in the design of novel PI4KIIIβ inhibitors, which may play significant roles as antiviral therapeutics.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71561" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71561" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Lipid phosphoinositides are essential regulators of myriad cellular processes, including signaling, membrane trafficking, and cytokinesis.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Phosphoinositides are generated through the phosphorylation of the inositol ring of phosphatidylinositol. Phosphatidylinositol can be phosphorylated and dephosphorylated by a diverse set of enzymes, and this results in a total of seven different mono- and polyphosphorylated phosphoinositides. The lipid species phosphatidylinositol 4-phosphate (PI4P) is generated by the action of phosphatidylinositol 4 kinases (PI4Ks).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> PI4P is the main biosynthetic route for the multiply phosphorylated signaling lipids phosphatidylinositol 4,5-bisphosphate (PIP<sub>2</sub>) and phosphatidylinositol 3,4,5-trisphosphate (PIP<sub>3</sub>).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In mammals, there are four different PI4K enzymes: two type II enzymes (PI4KIIα and PI4KIIβ) and two type III enzymes (PI4KIIIα and PI4KIIIβ). PI4KIIIβ is a peripheral membrane protein that is primarily localized at the Golgi and the trans-Golgi network (TGN). This enzyme plays key roles in mediating lipid transport<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and cytokinesis,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> maintaining lysosomal identity,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and, in tandem with Rab GTPases, regulating membrane trafficking.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Interest in the development of potent small molecules of PI4KIIIβ has been driven recently by the discovery of the key role of this enzyme in mediating both viral replication<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and <i>Plasmodium</i> development.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">PI4KIIIβ is critical for mediating viral replication of a number of RNA viruses through the generation of PI4P-enriched viral replication platforms. These membranous webs enriched in PI4P play essential roles in spatially concentrating viral replication proteins and are key in intracellular viral replication. This process is essential for many human pathogenic viruses including poliovirus, coxsackieviruses, enterovirus 71, rhinovirus, and Aichi virus.<a onclick="showRef(event, 'ref8 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref8 ref10 ref11 ref12 ref13 ref14">(8, 10-14)</a> There is also evidence that PI4KIIIβ together with PI4KIIIα plays a key role in mediating viral replication of hepatitis C virus.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">Small molecule inhibitors of PI4KIIIβ are potent antiviral agents.<a onclick="showRef(event, 'ref8 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref16">(8, 15, 16)</a> We previously reported the potent PI4KIIIβ inhibitor PIK93 (compound <b>1</b>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and this compound has been used extensively to decipher the cellular roles of PI4KIIIβ<a onclick="showRef(event, 'ref4 ref18'); return false;" href="javascript:void(0);" class="ref ref4 ref18">(4, 18)</a> and its role in mediating viral replication of pathogenic RNA viruses.<a onclick="showRef(event, 'ref8 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref8 ref11 ref12 ref13 ref14">(8, 11-14)</a> Compound <b>1</b> potently inhibits PI4KIIIβ; however, it shows cross reactivity toward a number of other lipid kinases. Compound <b>1</b> has very similar IC<sub>50</sub> values for PI4KIIIβ, class III PI3 kinase (vps34), and class IB PI3Kγ (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). We have previously crystallized <b>1</b> in complex with PI4KIIIβ,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> vps34,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and PI3Kγ<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B–E).</div><div class="NLM_p last">Development of PI4KIIIβ as an effective drug target for antiviral therapeutics requires the generation of highly potent and specific inhibitors. We report the development of a set of derivatives from compound <b>1</b>, and these represent some of the most potent PI4KIIIβ inhibitors reported to date. The selectivity profile of these compounds has been determined against vps34, PI3Kδ, and PI3Kγ, with the most selective compounds being >1000-fold selective over the related PI3K family of lipid kinases. We have successfully determined the structure of PI4KIIIβ bound to one of the most potent and selective compounds, and this structure reveals the molecular basis for the increased selectivity and potency of these compounds.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37011" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37011" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design of Optimized PI4KIIIβ Inhibitors</h3><div class="NLM_p">Compound <b>1</b> is highly selective for PI4KIIIβ over PI4KIIIα; however, it is similarly potent for a number of phosphoinositide 3-kinases (PI3Ks), specifically the class I isoforms PI3Kγ (also referred to as p110γ) and PI3Kδ (also referred to as p110δ), as well as the class III PI3K vps34 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). The structures of <b>1</b> bound to vps34,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> PI3Kγ,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and PI4KIIIβ<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> revealed that within the binding pocket there were significant opportunities to modify <b>1</b> to increase both potency and selectivity for PI4KIIIβ.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structural basis for inhibition of PI4KIIIβ and PI3Ks by the inhibitor PIK93 (compound <b>1</b>). (A) Structure of compound <b>1</b>, with the ethanolamine substituent off of the sulfonamide colored blue, the chloro substituent off of the central phenyl colored green, and the acetamide substituent off of the thiazol colored red. The potency of <b>1</b> against PI4KIIIβ, PI3Kγ, and vps34 is graphed. (B) Structures of PI4KIIIβ<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D0L">4D0L</a>), vps34<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J">2X6J</a>), and PI3Kγ<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ">2CHZ</a>) bound to <b>1</b> aligned, showing the chloro substituent of <b>1</b> with the activation loop of each enzyme colored according to the legend. (C–E) Structures of PI4KIIIβ (C), PI3Kγ (D), and vps34 (E) with residues within 5 Å of the acetamide group of <b>1</b> shown as spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From examining the structures of <b>1</b> bound to each enzyme, there were three regions of the molecule that were the focus to optimize both potency and selectivity of novel PI4KIIIβ inhibitors. These consisted of modifying the substituent on the central phenyl ring (colored green, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A), the substituent off of the sulfonamide (colored blue, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A), and the acetamide moiety (colored red, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). The chloro substituent on the central phenyl ring of <b>1</b> fits into a pocket partially composed of the activation loop of both PI3Ks and PI4KIIIβ (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C). The conformation of the activation loop of PI3Kγ when bound to compound <b>1</b> is positioned closer to this substituent than it is in either vps34 or PI4KIIIβ. This suggested that modifying this group to a larger substituent might increase selectivity over class I PI3Ks. Changing this group to a bromo substituent (compound <b>2</b>) caused a slight increase in potency for PI4KIIIβ, PI3Kδ, and vps34 and a slight decrease in potency for PI3Kγ. Modifying this group to a methoxy substituent (compound <b>3</b>) caused both an increase in potency for PI4KIIIβ and a large increase in specificity over PI3Kγ, PI3Kδ (>40-fold selective for both), and vps34 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Development of novel PI4KIIIβ inhibitors. (A) Structure of compound <b>1</b> is shown, with novel substituents at the R1, R2, and R3 positions shown in tabular form and with the IC<sub>50</sub> values for each compound against PI4KIIIβ, PI3Kγ, PI3Kδ, and vps34 listed. Kinase assays were carried out in the presence of 10 μM ATP and 10 mM MgCl<sub>2</sub> (PI4KIIIβ, PI3Kδ, and PI3Kγ) or 10 mM MnCl<sub>2</sub> (vps34). IC<sub>50</sub> values measured using MnCl<sub>2</sub> are 10–20-fold higher than those with the physiologically relevant MgCl<sub>2</sub> due to increased kinase activity. Inhibitors anti-HCV antiviral activity (IC<sub>50</sub>) was assessed by <i>Gaussia</i> luciferase assay (Gluc HCV2a activity), and cell viability (CC<sub>50</sub>) was assessed by Presto Blue assay (Gluc HCV2a viability), as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. (B) Full inhibitor curves for compound <b>8</b>–<b>10</b> are shown for PI4KIIIβ, PI3Kγ, PI3Kδ, and vps34, with all measurements carried out in triplicate. For compounds <b>9</b> and <b>10</b>, the curves for the related lipid kinase PI4KIIIα is shown as well, with assays against other related lipid kinases shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">Figure S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The structures of <b>1</b> bound to PI4KIIIβ, PI3Kγ, and vps34 also suggested that modification of the acetamide group derived from the central thiazole might lead to further gains in specificity. PI4KIIIβ has a much more open pocket around the acetamide group of <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C) compared to that of both PI3Kγ (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D) and vps34 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>E), which have a number of bulky hydrophobic residues. The methyl group of <b>1</b> was replaced with either a <i>t</i>-butyl group (compound <b>4</b>) or a cyclopentyl group (compound <b>5</b>). Both compounds showed similar or slightly better potency against PI4KIIIβ; however, both showed a very large increase in selectivity over PI3Kγ (>600-fold for compound <b>4</b> and >90-fold for compound <b>5</b>), PI3Kδ, and vps34 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A).</div><div class="NLM_p">Finally, the ethanolamine attached to the sulfonamide in compound <b>1</b> was also derivatized. The ethanolamine was replaced with either a <i>p</i>-hydroxy phenol group (compound <b>6</b>) or a <i>p</i>-fluoro phenyl (compound <b>7</b>). The presence of the <i>p</i>-hydroxy phenol group caused a large increase in potency for all lipid kinases tested, with minimal differences in both potency and specificity for the <i>p</i>-fluoro phenyl substituent (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A).</div><div class="NLM_p">Combining this set of information, we generated three compound <b>1</b> derivatives (compounds <b>8</b>–<b>10</b>) that contained derivations at multiple positions of <b>1</b>. All of these compounds contained a methoxy substituent off of the central phenyl, with compounds <b>9</b> and <b>10</b> containing a <i>p</i>-hydroxy phenol off of the sulfonamide and either a cylopentyl (compound <b>9</b>) or a <i>t</i>-butyl (compounds <b>8</b> and <b>10</b>) group at the acetamide position off of the central thiazole. Both of the compounds containing the <i>p</i>-hydroxyl phenol were extremely potent against PI4KIIIβ (IC<sub>50</sub>’s of 7 and 3.6 nM for compounds <b>9</b> and <b>10</b>, respectively) and were >140- and >1000-fold selective over PI3Kγ and >20- and >200-fold selective over PI3Kδ, and they showed no inhibition of vps34 at concentrations up to 20 μM (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). Compound <b>8</b> was not as potent (IC<sub>50</sub> of 36 nM), but it showed an excellent selectivity profile, with no inhibition of both PI3Kγ and vps34 (<20% at 50 μM) and very little inhibition of PI3Kδ (<50% inhibition at 50 μM). Compound <b>10</b> is the most potent PI4KIIIβ inhibitor currently reported, with very minor off-target inhibition of PI4KIIIβ related lipid kinases. Both compounds <b>9</b> and <b>10</b> were further characterized against a panel of six additional lipid kinases (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>). Compound <b>9</b> showed weak inhibition of PI3KC2γ (IC<sub>50</sub>–1 μM), PI3Kα (∼2 μM), and PI4KIIIα (∼2.6 μM) and <50% inhibition at concentrations up to 20 μM for PI4K2α, PI4K2β, and PI3Kβ. Compound <b>10</b> showed weak inhibition of PI3KC2γ (IC<sub>50</sub>–1 μM), PI3Kα (∼10 μM), and PI4KIIIα (∼3 μM), and <20% inhibition at concentrations up to 20 μM for PI4K2α, PI4K2β, and PI3Kβ.</div><div class="NLM_p last">PI4KIIIβ has been shown to be a key host factor for the replication of the hepatitis C virus.<a onclick="showRef(event, 'ref14 ref21'); return false;" href="javascript:void(0);" class="ref ref14 ref21">(14, 21)</a> To test the potency of the compound <b>1</b> derivative inhibitors as antivirals, we carried out viral replication assays using a luciferase reporter-linked infectious HCV clone. These results (<a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">S2</a>) generally show a trend of the most potent molecules being the most effective antivirals, with increased specificity leading to decreased toxicity. One noticeable trend in the assay was that absence of the hydroxyl-phenyl group led to a large decrease in the antiviral potency of the compounds. This may be due to differences in cell permeability caused by this group. The compounds with the best combination of antiviral efficacy and lowest toxicity were <b>9</b> and <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structural Basis of Specificity for PI4KIIIβ Inhibitors</h3><div class="NLM_p">To determine the molecular basis for the potency and specificity of these novel PI4KIIIβ inhibitors, we set out to crystallize them bound to PI4KIIIβ. We have previously crystallized a truncated construct of PI4KIIIβ in complex with compound <b>1</b> and the GTPase Rab11.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This structure revealed the molecular basis of its interaction with Rab11 and also revealed the binding of compound <b>1</b> to PI4KIIIβ; however, this construct crystallized only in the presence of compound <b>1</b>. To generate crystals of PI4KIIIβ bound to compound <b>9</b>, we used a novel crystallization construct generated through the use of a hydrogen–deuterium exchange mass spectrometry-based approach.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This construct allowed us to generate crystals of PI4KIIIβ bound to GDP-loaded Rab11 in the absence of inhibitors. These crystals were used to soak compound <b>9</b> and allowed us to solve the co-crystal structure at 3.2 Å resolution. The binding mode of compound <b>9</b> was unambiguous (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">Figure S3</a>). The residues mediating contacts with compound <b>9</b> are shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A, with the shape of the inhibitor in the active site pocket shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis of inhibition of PI4KIIIβ by compound <b>9</b>. (A) Residues mediating the interaction of PI4KIIIβ with compound <b>9</b> are shown, with putative hydrogen bonds indicated by dotted lines. Figure generated using LIGPLOT.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (B) Fit of compound <b>9</b> in the PI4KIIIβ active site pocket. The kinase domain is colored with the N-lobe shown in red and the C-lobe shown in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure of PI4KIIIβ Bound to Compound <b>9</b></h3><div class="NLM_p">Compound <b>9</b> forms a crescent shape that conforms to the active site of PI4KIIIβ. This molecule makes extensive contacts with PI4KIIIβ (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). There are two putative hydrogen bonds formed between the thiazole and the acetamide of compound <b>9</b> with both the amide and carbonyl group of V598. The sulfonamide group also forms a hydrogen bond with K549. All of these hydrogen bonds are similar to those reported in the structure of <b>1</b> bound to PI4KIIIβ.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Intriguingly the <i>p</i>-hydoxy group on the N-phenol sulfonamide also makes a putative hydrogen bond with the carbonyl of G660. The presence of this <i>p</i>-hydroxyl group is important for potency because molecules lacking this group are less potent against PI4KIIIβ (compound <b>6</b> versus compound <b>7</b>). The presence of this hydroxyl group also increases potency for both PI3Kγ and vps34, suggesting that this putative hydrogen bond is most likely conserved across all three enzymes.</div><div class="NLM_p">The structure of compound <b>9</b> also reveals the molecular mechanism for how the acetamide and methoxy substituents mediate inhibitor selectivity over PI3Kγ and vps34. Aligning the structure of PI4KIIIβ bound to compound <b>9</b> with structures of compound <b>1</b> bound to PI3Kγ and vps34 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A–C) reveals a number of steric clashes that explain the lack of potency of compound <b>9</b> against the PI3K family kinases. The acetamide group clashes with residues W812 and M953 in PI3Kγ and residues Y746 and F673 in vps34. The methoxy group off of the central phenyl group clashes with the aspartic acid from the DFG motif in the activation loop (D964 in PI3Kγ and D823 in vps34).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural basis for selectivity of compound <b>9</b>. (A) Structure of compound <b>9</b> bound to PI4KIIIβ, with selected residues colored in orange and shown as spheres. (B) Model of compound <b>9</b> in the active site of PI3Kγ. The model was generated by aligning the active site of the structure of compound <b>9</b> bound to PI4KIIIβ to the structure of PI3Kγ bound to compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ">2CHZ</a>). The equivalent residues shown in panel A are colored in green and shown as spheres, with steric clashes highlighted in red. (C) Model of compound <b>9</b> in the active site of vps34. The model was generated by aligning the active site of the structure of compound <b>9</b> bound to PI4KIIIβ to the structure of vps34 bound to compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J">2X6J</a>). The equivalent residues shown in panel A are colored in cyan and shown as spheres, with steric clashes highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Phosphatidylinositol 4 kinase IIIβ plays both key physiological and pathological roles, and the design of novel potent and selective inhibitors will be essential both in deciphering the cellular functions of this enzyme and in developing potential future therapeutic agents in diseases dependent on PI4KIIIβ activity. PI4KIIIβ mediates the replication of a variety of pathogenic RNA viruses, including members of both the <i>Picornaviridae</i> and <i>Flaviviridae</i> families of virus.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These include viruses that pose significant threats to human health including SARS, MERS, hepatitis C, and polio. Along with this key role in mediating pathological conditions, PI4KIIIβ also plays key roles in a number of physiological roles including membrane trafficking,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> lipid transport,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and cytokinesis,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and the development of potent and selective molecules will be essential in deciphering the isoform-specific functions of this enzyme.</div><div class="NLM_p">Since PI4KIIIβ plays an essential role in mediating viral replication for a variety of pathogenic viruses, the development of PI4KIIIβ inhibitors has been an attractive target for pan-antiviral agents. A number of potent PI4KIIIβ inhibitors have recently been discovered,<a onclick="showRef(event, 'ref9 ref12 ref15 ref21 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref9 ref12 ref15 ref21 ref25 ref26 ref27 ref28">(9, 12, 15, 21, 25-28)</a> and experiments on a variety of RNA viruses show they are potent antiviral agents. However, a complication of this work has been the toxic effects of some of these inhibitors on host function. There are conflicting results of the lethality of PI4KIIIβ inhibitors, with some showing lethality in mice<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and others being well-tolerated.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Deciphering whether these effects are dependent on the inhibition of PI4KIIIβ or are through other off-target effects requires the generation of novel more potent and specific PI4KIIIβ inhibitors. A specific goal for the development of antivirals is to avoid off-target effects on both PI3Kγ and PI3Kδ as both of these enzymes play key roles in the immune system.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">Here, we describe a set of new compound <b>1</b> derivative inhibitors that are some of the most potent and specific PI4KIIIβ inhibitors as yet reported. Through a detailed structure-based drug design approach, we have designed derivatives that increased potency against PI4KIIIβ >5-fold while increasing selectivity >1000-fold to the related lipid kinases PI3Kγ and vps34. The most potent of these molecules (compound <b>10</b>) was screened over a panel of nine related lipid kinases and was >200-fold selective for PI4KIIIβ across all enzymes tested. Two of the most potent and selective molecules (compounds <b>9</b> and <b>10</b>) were also the most effective antiviral compounds in a cellular model of hepatitis C virus replication, with the best balance of antiviral potency and low cellular toxicity.</div><div class="NLM_p last">The structure of one of the most potent molecules bound to PI4KIIIβ was determined and revealed the molecular basis for the specificity of the derivative molecules. The pocket that accommodates the acetamide group of the inhibitors was found to be the most important region in mediating inhibitor selectivity, and the differences in the pocket accommodating this group in PI4KIIIβ compared to that in PI3Kγ and vps34 explain this observation. The acetamide group mediating specificity of PI4KIIIβ inhibitors strongly correlates with previous studies on compounds very similar to compound <b>1</b>. Addition of a bulky substituent in a similar position to the acetamide group in compound <b>1</b> had a limited role in potency, but it greatly enhanced specificity.<a onclick="showRef(event, 'ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref30">(27, 30)</a> Structural and computational models of a different class of inhibitors bound to PI4KIIIβ suggested an important role of the pocket located near this region in mediating both potency and selectivity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The structural details of this compound bound in the active site pocket will provide a future framework for the modification of these compounds to modify the inhibitors in a way that will enhance their pharmacokinetic properties but not lead to any decreases in their potency and specificity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88436" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88436" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Protein Expression</h3><div class="NLM_p last">Truncated human PI4KIIIβ (121–784 Δ249–287Δ408–507 S294A) (plasmid JM7) and full-length human Rab11a(Q70L) (plasmid pJB88) were expressed in BL21 C41 (DE3) cells. For Rab11a(Q70L) expression, cultures were grown to an OD<sub>600</sub> of 0.7 and induced with 0.5 mM IPTG for 3.5 h at 37 °C. For truncated PI4KIIIβ expression, cultures were induced overnight at 16 °C with 0.1 mM IPTG at an OD<sub>600</sub> of 0.6. Cells were harvested by centrifugation, washed with cold phosphate-buffered saline (PBS), and frozen in liquid nitrogen, and pellets were stored at −80 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Protein Purification</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> PI4KIIIβ (Truncation) Purification</h4><div class="NLM_p last">Cells were resuspended in 20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 10 mM imidazole, 5% (v/v) glycerol, 2 mM β-mercaptoethanol, and a 1:1666 dilution of a protease inhibitor cocktail (Millipore protease inhibitor cocktail set III, animal-free) and sonicated on ice for 5 min with cycles consisting of 10 s on, 10 s off. Triton X-100 was then added to a final concentration of 0.2%, and the lysate was centrifuged for 45 min at 20 000<i>g</i>. The supernatant was then filtered through a 0.45 μm filter (Celltreat Scientific Products) and was loaded onto a 5 mL HisTrap FF column (GE Healthcare) equilibrated in buffer A (20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 10 mM imidazole, 5% (v/v) glycerol, 2 mM β-mercaptoethanol). The column was washed with 20 mL of buffer A, followed by 20 mL of 6% buffer B (20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 200 mM imidazole, 5% (v/v) glycerol, 2 mM β-mercaptoethanol), and it was eluted with 100% buffer B. The His affinity tag protein was cleaved overnight at 4 °C with TEV protease. The cleaved protein was then diluted to 50 mM NaCl (using 20 mM Tris-HCl, pH 8.0, 10 mM imidazole, 5% (v/v) glycerol, 2 mM β-mercaptoethanol) and was loaded onto a 5 mL HiTrap Q HP column (GE Healthcare) equilibrated in buffer C (20 mM Tris-HCl, pH 8.0, 50 mM NaCl, 5% (v/v) glycerol, 2 mM β-mercaptoethanol). Protein was eluted with a gradient elution using buffer D (20 mM Tris-HCl, pH 8.0, 1.0 M NaCl, 5% (v/v) glycerol, 2 mM β-mercaptoethanol). Fractions containing the cleaved PI4KIIIβ were pooled and concentrated to 700 μL in an Amicon 50K centrifugal filter (Millipore). The protein was then loaded onto a HiPrep 16/60 Sephacryl S200 column equilibrated in buffer E (20 mM HEPES, pH 7.2, 150 mM NaCl, 1 mM TCEP). The cleaved PI4KIIIβ was then concentrated to approximately 15 mg/mL in an Amicon 50K centrifugal filter (Millipore), and aliquots were frozen in liquid nitrogen and stored at −80 °C.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Rab11a(Q70L) Purification</h4><div class="NLM_p last">Cells were resuspended in 20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 5% (v/v) glycerol, 2 mM β-mercaptoethanol, and a 1:1666 dilution of a protease inhibitor cocktail (Millipore protease inhibitor cocktail set III, animal-free). Cells were sonicated and centrifuged as described for the truncated PI4KIIIβ. The supernatant was filtered through a 0.45 μm filter (Celltreat Scientific Products) and incubated for 1 h with 4 mL of glutathione Sepharose 4B beads (GE Healthcare) equilibrated in buffer F (20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 5% (v/v) glycerol, 2 mM β-mercaptoethanol) followed by a 3 × 15 mL wash in buffer F. The GST tag was cleaved overnight on the beads with TEV protease. Anion-exchange chromatography was performed as outlined above for the truncated PI4KIIIβ. Cleaved Rab11a(Q70L) was then concentrated to between 5 and 15 mg/mL and nucleotide loaded by adding EDTA to 10 mM followed by 1 U of phosphatase (phosphatase, alkaline-agarose from calf intestine; Sigma P0762-100UN) per milligram of protein. Proteins were then incubated for 1.5 h. The phosphatase was removed using a 0.2 μm spin filter (Millipore); the flow-through was collected, and a 10-fold molar excess of GDP was added followed by MgCl<sub>2</sub> to a final concentration of 20 mM. Proteins were incubated for 30 min. Gel filtration was performed with cleaved GDP-loaded Rab11a(Q70L) as described above for the truncated PI4KIIIβ.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Crystallography</h3><div class="NLM_p">Crystals of PI4KIIIβ (final concentration 7.4 mg/mL) with Rab11a-GDP (final concentration 4.5 mg/mL) were obtained in 15% (w/v) PEG-4000, 100 mM sodium citrate, pH 5.6, 200 mM ammonium sulfate (protein/precipitant ratio of 3:1). Refinement plates were set by gridding PEG-4000, ammonium sulfate, and glycerol. Optimized crystals were obtained by seeding using the Hampton Research seed bead kit according to the manufacturer’s instructions using a reservoir solution of 14.8% (w/v) PEG-4000, 100 mM sodium citrate, pH 5.6, 250 mM ammonium sulfate. The best crystals were obtained in 13–15% (w/v) PEG-4000, 100 mM sodium citrate, pH 5.6, 250 mM ammonium sulfate, and 2% glycerol with a 1/1000 or 1/10000 seed solution dilution, a Rab11a-GDP final concentration of 4.51 mg/mL, and a PI4KIIIβ final concentration of 7.38 mg/mL. Crystals were frozen in liquid nitrogen using a 15% PEG-4000 (w/v), 100 mM sodium citrate, pH 5.6, 250 mM ammonium sulfate, and 25% (v/v) glycerol cryo solution.</div><div class="NLM_p">Inhibitor soaks were performed by incubating crystals with 0.5 μL of 10 μM inhibitor stocks in cryo buffer (15% PEG-4000 (w/v), 250 mM ammonium sulfate, 100 mM sodium citrate, pH 5.6, 25% glycerol (v/v)) for 30 min, followed by a 30 min incubation with 0.5 μL of 100 μM inhibitor stock in cryo buffer and a final 30 min incubation in 1 mM inhibitor stock in cryo buffer. Before the final addition, 1 μL was removed from the crystal drop and 1 μL of the 1 mM inhibitor in cryo buffer was added.</div><div class="NLM_p">Diffraction data were collected at 100 K at beamline 08ID-1 of the Canadian Macromolecular Crystallography Facility (Canadian Light Source, CLS). Data were integrated using iMosflm 7.1.1<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and scaled with AIMLESS.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Phases were initially obtained by molecular replacement (MR) using Phaser,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> with the structure of PI4KIIIβ bound to compound <b>1</b> and Rab11 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D0L">4D0L</a>) used as the search model. The final model of PI4KIIIβ bound to compound <b>9</b> in complex with Rab11 was built using iterative model building in COOT<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and refinement using Phenix<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> to <i>R</i><sub>work</sub> = 23.87 and <i>R</i><sub>free</sub> = 26.79. The binding mode of compound <b>9</b> was unambiguous, and ligand geometry was generated using the elbow subset of Phenix.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Full crystallographic statistics are shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data Collection and Refinement Statistics<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">data collection</th><th class="colsep0 rowsep0" align="center">PI4K–Rab11–GDP–compound <b>9</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.9797</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell</td><td class="colsep0 rowsep0" align="left">(48.9 103.5 188.9), (90 90 90)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total reflections</td><td class="colsep0 rowsep0" align="left">67660 (6441)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">16316 (1611)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multiplicity</td><td class="colsep0 rowsep0" align="left">4.1 (4.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">98.63 (99.44)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">8.93 (2.11)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson B-factor</td><td class="colsep0 rowsep0" align="left">82.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.1092 (0.7019)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>meas</sub></td><td class="colsep0 rowsep0" align="left">0.1249</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC1/2</td><td class="colsep0 rowsep0" align="left">0.993 (0.375)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC*</td><td class="colsep0 rowsep0" align="left">0.998 (0.738)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">47.34–3.2 (3.31–3.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections used for <i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">5%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub></td><td class="colsep0 rowsep0" align="left">23.3 (32.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">26.6 (38.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of non-hydrogen atoms</td><td class="colsep0 rowsep0" align="left">5041</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">4970</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein residues</td><td class="colsep0 rowsep0" align="left">619</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMS (bonds)</td><td class="colsep0 rowsep0" align="left">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMS (angles)</td><td class="colsep0 rowsep0" align="left">0.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran favored (%)</td><td class="colsep0 rowsep0" align="left">96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran outliers (%)</td><td class="colsep0 rowsep0" align="left">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clashscore</td><td class="colsep0 rowsep0" align="left">21.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average B-factor</td><td class="colsep0 rowsep0" align="left">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">98.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">109</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Statistics for the highest-resolution shell are shown in parentheses.</p></div></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Biochemical Assays</h3><div class="NLM_p">Lipid kinase assays were preformed using recombinant enzyme, phosphoinositides purchased from Avanti Polar Lipids, and γ<sup>32</sup>P-ATP (PerkinElmer cat. no. BLU502A001MC) in a membrane capture assay described previously.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Each inhibitor was diluted into 10% DMSO and kinase assay buffer. Upon completion of the reaction, 4 μL was spotted onto 0.2 μm nitrocellulose (Bio-Rad cat. no. 162-0112). The membrane was dried for 5 min under a heat lamp followed by 1 × 30 s and 6 × 5 min washes in 1 M NaCl/1% phosphoric acid. The membrane was dried for 20 min under a heat lamp followed by overnight exposure to a phosphor screen, and phosphorimaging followed on a Typhoon 9500. Intensities were quantified using SPOT.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Specifications for each enzyme follow.</div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> PI4KIIIβ</h4><div class="NLM_p last">Recombinant enzyme was purchased from Life Technologies (cat. no. PV5277, lot no. 943589E). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and PI4KIIIβ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 30 min. Final conditions were as follows: 20 mM Bis-Tris Propane, pH 7.5, 10 mM MgCl<sub>2</sub>, 0.075 mM Triton X-100, 0.5 mM EGTA, 1 mM DTT, 100 μM PI, 500 ng/μL BSA, 2.5 nM PI4KIIIβ, 2% DMSO, 10 μM ATP, and 1 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> PI3Kγ</h4><div class="NLM_p last">Recombinant enzyme was purchased from Life Technologies (cat. no. PV4786, lot no. 1638926A). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol-4,5-bisphosphate (PIP<sub>2</sub>, cat. no. 840046P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PIP<sub>2</sub>:DOPS:DOPC. The reaction was setup as follows (1) kinase assay buffer, PIP<sub>2</sub>:DOPS:DOPC, BSA, and PI3Kγ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 15 min. Final conditions were as follows: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.03% CHAPS, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 5 nM PI3Kγ, 80 μM PIP<sub>2</sub>, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 1 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> PI3Kδ</h4><div class="NLM_p last">Recombinant enzyme was purchased from Life Technologies (cat. no. PV6451, lot no. 1763224). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol-4,5-bisphosphate (PIP<sub>2</sub>, cat. no. 840046P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PIP<sub>2</sub>:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PIP<sub>2</sub>:DOPS:DOPC, BSA, and PI3Kδ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 20 min. Final conditions were as follows: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.03% CHAPS, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 2.5 nM PI3Kδ, 80 μM PIP<sub>2</sub>, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 2.5 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> vps34</h4><div class="NLM_p last">Recombinant enzyme was purchased from Life Technologies (cat. no. PV5126, lot no. 1555138A). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and vps34 were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 1 h. Final conditions were as follows: 50 mM HEPES, pH 7.5, 0.1% CHAPS, 2 mM MnCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 10 nM vps34, 100 μM PI, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 2.5 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> PI4KIIIα</h4><div class="NLM_p last">Recombinant enzyme was purchased from EMD Millipore (cat. no. 14-908, lot no. D9KN031N-B). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and PI4KIIIα were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 25 min. Final conditions were as follows: 20 mM Bis-Tris Propane, pH 7.5, 10 mM MgCl<sub>2</sub>, 0.075 mM Triton X-100, 0.5 mM EGTA, 1 mM DTT, 100 μM PI, 500 ng/μL BSA, 1.2 nM PI4KIIIα, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> PI4KIIα</h4><div class="NLM_p last">PI4KIIα was cloned from DNASU plasmid HsCD00003332.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Recombinant enzyme was obtained through FLAG immunoprecipitation of FLAG-PI4KIIα transfected into HEK293T cells (10 μg of FLAG-PI4KIIα transfected with Life Technologies Lipofectamine and Plus Reagent to a 15 cm dish of HEK293Ts and 75 μL elution with FLAG peptide). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and PI4KIIα were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 25 min. Final conditions were as follows: 20 mM Bis-Tris Propane, pH 7.5, 10 mM MgCl<sub>2</sub>, 0.075 mM Triton X-100, 0.5 mM EGTA, 1 mM DTT, 100 μM PI, 500 ng/μL BSA, 2.9 μL of PI4KIIα, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> PI4KIIβ</h4><div class="NLM_p last">PI4KIIβ was cloned from DNASU plasmid HsCD00001592.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Recombinant enzyme was obtained through FLAG immunoprecipitation of FLAG-PI4KIIβ transfected into HEK293T cells (10 μg of FLAG-PI4KIIβ transfected with Life Technologies Lipofectamine and Plus Reagent to a 15 cm dish of HEK293Ts and 75 μL elution with FLAG peptide). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and PI4KIIβ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 25 min. Final conditions were as follows: 20 mM Bis-Tris Propane, pH 7.5, 10 mM MgCl<sub>2</sub>, 0.075 mM Triton X-100, 0.5 mM EGTA, 1 mM DTT, 100 μM PI, 500 ng/μL BSA, 2 μL of PI4KIIβ, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> PI3Kα</h4><div class="NLM_p last">Recombinant enzyme was purchased from EMD Millipore (cat. no. 14-602, lot no. 2150294-A). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol-4,5-bisphosphate (PIP<sub>2</sub>, cat. no. 840046P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PIP<sub>2</sub>:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PIP<sub>2</sub>:DOPS:DOPC, BSA, and PI3Kα were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 15 min. Final conditions were as follows: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.03% CHAPS, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 3.3 nM PI3Kα, 80 μM PIP<sub>2</sub>, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> PI3Kβ</h4><div class="NLM_p last">Recombinant enzyme was purchased from SignalChem (cat. no. P28-10H-10, lot no. F-532-3). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol-4,5-bisphosphate (PIP<sub>2</sub>, cat. no. 840046P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PIP<sub>2</sub>:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PIP<sub>2</sub>:DOPS:DOPC, BSA, and PI3Kβ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 15 min. Final conditions were as follows: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.03% CHAPS, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 0.6 nM PI3Kβ, 80 μM PIP<sub>2</sub>, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> PI3KC2γ</h4><div class="NLM_p last">Recombinant enzyme was purchased from EMD Millipore (cat. no. 14-910, lot no. 2023057-A). <span class="smallcaps smallerCapital">l</span>-α-Phosphatidylinositol (PI, cat. no. 840024P) and DOPS:DOPC lipids (cat. no. 790595P) were sonicated in water to generate 1 mg/mL PI:DOPS:DOPC. The reaction was setup as follows: (1) kinase assay buffer, PI:DOPS:DOPC, BSA, and PI3KC2γ were combined in a total volume of 10 μL (2.5× solution); (2) 5 μL of inhibitor solution was added (5× solution) and incubated with the enzyme mixture for 15 min; and (3) 10 μL of cold ATP and γ<sup>32</sup>P-ATP was added (2.5× solution) to initiate the reaction, which ran for 15 min. Final conditions were as follows: 50 mM HEPES, pH 7.5, 100 mM NaCl, 0.03% CHAPS, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 1 nM PI3KC2γ, 100 μM PI, 500 ng/μL BSA, 2% DMSO, 10 μM ATP, and 2 μCi γ<sup>32</sup>P-ATP.</div></div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Organic Synthesis</h3><div class="NLM_p">Materials obtained commercially were reagent grade and were used without further purification. <sup>1</sup>H NMR spectra were recorded on a Varian 400 spectrometer at 400 MHz. High-resolution electron impact mass spectra were recorded on a Thermo Fisher Exactive EMR and Thermo Fisher LTQ Orbitrap Velos at the University of California–San Francisco Center for Mass Spectrometry. Reactions were monitored by thin-layer chromatography using Merck silica gel 60 F<sub>254</sub> glass plates (0.25 mm thick). Flash chromatography was conducted with Grace Reveleris flash cartridges with 40 μm silica on an Agilent 971-FP. All RP-HPLC were performed with a Waters 2545 binary gradient module equipped with an XBridge prep C18 column using H<sub>2</sub>O + 0.1% formic acid and CH<sub>3</sub>CN + 0.1% formic acid (5–95% gradient) while monitoring at 254 nm. All final compounds were >95% pure, as measured by liquid chromatography mass spectrometry (LCMS). Full compound characterization details are presented in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">Synthesis of compound <b>1</b> and derivatives was conducted similarly to methods described previously<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. To commercially available acetaphenone was added chlorosulfuric acid (11 equiv) dropwise in an ice bath, and the reaction was heated at 40 °C for 2 h. The reaction was stopped by dropwise transfer to ice water. The aqueous phase was extracted 3× with ethyl acetate, and the combined organic was dried with sodium sulfate, filtered, and concentrated <i>in vacuo</i> to give a brown oil, the crude, corresponding to sulfonyl chloride <b>11</b>. The sulfonyl chloride was dissolved in THF (1.5 M), the appropriate amino alcohol (4.5 equiv) was added, and the reaction was allowed to stir overnight at room temperature. The reaction was concentrated <i>in vacuo</i>, water was added, and the aqueous phase was extracted 3× with ethyl acetate. The combined organic phase was concentrated, and intermediate <b>12</b> was purified using silica gel flash chromatography (2–10% methanol in dichloromethane). If pure intermediate was not obtained, then the intermediate was subjected to a second purification by reverse-phase HPLC using acetonitrile/water/0.1% formic acid as the solvent system.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>1</b> and Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Intermediate <b>12</b> was dissolved in THF (1.5 M). (2-Carboxyethyl)triphenylphosphonium bromide was dissolved in THF (1 M), and intermediate <b>12</b> was added to the bromide solution dropwise at room temperature. The reaction was allowed to proceed 1 h at room temperature, the solvent was removed <i>in vacuo</i>, and the product was purified by silica gel flash chromatography (0–10% methanol in dichloromethane) to yield intermediate <b>13</b>.</div><div class="NLM_p">To ethanol-recrystallized thiourea in dry toluene (0.5 M) was added the appropriate acyl chloride, and the reaction was heated to reflux for 18 h. Toluene was removed <i>in vacuo</i>, and the reaction was diluted in ethyl acetate and filtered. The organic layer was washed 2× with water and brine, dried with sodium sulfate, and reduced <i>in vacuo</i>. The resulting acetylthiourea <b>14</b> was purified by silica gel flash chromatography (25–100% ethyl acetate in hexanes).</div><div class="NLM_p last">Intermediate <b>13</b> was dissolved in ethanol (0.17 M), and appropriate acetylthiourea <b>14</b> was added at room temperature. The reaction was heated to reflux for 30 min and then cooled to room temperature, and compound <b>1</b> or a compound <b>1</b> derivative (compounds <b>2</b>–<b>10</b>) was purified by silica gel chromatography (0–10% methanol in dichloromethane), which, if necessary, was followed by reverse-phase HPLC using acetonitrile/water/0.1% formic acid as the solvent system.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>Gaussia</i> Luciferase-Based HCV Reporter Virus and Stable Cell Line of Huh7.5 Containing Infectious HCV Reporter Virus</h3><div class="NLM_p last">A fully infectious HCVcc genotype 2a reporter virus encoding <i>Gaussia</i> luciferase between p7 and NS2 similar to that used by Maurkian et al.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> was generated to monitor HCV replication quantitatively. Briefly, <i>Gaussia</i> luciferase gene with an in-frame foot and mouth disease virus autoproteolytic 2A peptide sequence was inserted between HCV p7 and NS2 of a J6/JFH1 infectious HCV clone,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> resulting in the fully infectious HCVcc reporter virus J6/JFH1-GLuc (p7-NS2-GLucFM2A). After Huh7.5 cells were infected with the <i>Gaussia</i> luciferase-based HCV reporter virus, the cells were maintained for 3 days and then passaged every 4 days for 20 passages. A stable cell line with consistent Gaussia luciferase secretion was established.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> HCV2a Antiviral Assay</h3><div class="NLM_p">Huh7.5 cells stably infected with J6/JFH1-GLuc were maintained and passaged in DMEM (Mediatech, Manassas, VA) supplemented with 10% FBS (Omega Scientific, Tarzana, CA), 2 mM glutamine (Mediatech, Manassas, VA), nonessential amino acids (Mediatech, Manassas, VA), 100 IU/mL penicillin (Mediatech, Manassas, VA), and 100 mg/mL streptomycin (Mediatech, Manassas, VA) and were cultured in a 37 °C incubator with 5% CO<sub>2</sub> and 95% relative humidity.</div><div class="NLM_p">Ten thousand cells were plated in 96-well plates (E&K Scientific Products, Santa Clara, CA) and supplemented with serially diluted compounds 1 h after plating. After 3 days of incubation, viability was tested using the Presto Blue cell viability reagent (Life Technologies Corporation, Grand Island, NY) per the manufacturer’s protocol, and replication was measured by determining the Gaussia luciferase activity in the supernatant using the luciferase reagent (Promega Corporation, Madison, WI) according to the manufacturer’s protocol.</div><div class="NLM_p last">Viability and luminescence were read using a plate reader (Infinite 1000, Tecan Systems, San Jose, CA). Values were imported into Prism (GraphPad Software, La Jolla, CA) for graphing and calculations of IC<sub>50</sub> and CC<sub>50</sub> values.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01311">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01311" class="ext-link">10.1021/acs.jmedchem.5b01311</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Compound characterization; inhibitor assays of compounds <b>9</b> and <b>10</b> across a panel of lipid kinases; antiviral efficacy and toxicity of selected inhibitors; binding of compound <b>9</b> in the active site pocket of PI4KIIIβ (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES strings, IC<sub>50</sub> data, and CC<sub>50</sub> data for <b>1</b>–<b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf">jm5b01311_si_001.pdf (548.94 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_002.csv">jm5b01311_si_002.csv (1.23 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure factors and PDB coordinates of PI4K bound to compound <b>9</b> have been deposited at the protein databank (PDB) with the coordinates <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUQ">5EUQ</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01311" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevan M. Shokat</span> - <span class="hlFld-Affiliation affiliation">Howard
Hughes Medical Institute and Department of Cellular and Molecular
Pharmacology, University of California,
San Francisco (UCSF), San Francisco, California 94143, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e58e8093848bcb968d8a8e8491a590869683cb808190"><span class="__cf_email__" data-cfemail="8de6e8fbece3a3fee5e2e6ecf9cdf8eefeeba3e8e9f8">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John E. Burke</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Microbiology, University
of Victoria, Victoria, BC V8W 2Y2, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#80eae5e2f5f2ebe5c0f5f6e9e3aee3e1"><span class="__cf_email__" data-cfemail="fa909f988f88919fba8f8c9399d4999b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florentine U. Rutaganira</span> - <span class="hlFld-Affiliation affiliation">Howard
Hughes Medical Institute and Department of Cellular and Molecular
Pharmacology, University of California,
San Francisco (UCSF), San Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melissa L. Fowler</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Microbiology, University
of Victoria, Victoria, BC V8W 2Y2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob A. McPhail</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Microbiology, University
of Victoria, Victoria, BC V8W 2Y2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael A. Gelman</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine and Department of Microbiology
& Immunology, Stanford University, Palo Alto, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Khanh Nguyen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine and Department of Microbiology
& Immunology, Stanford University, Palo Alto, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anming Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine and Department of Microbiology
& Immunology, Stanford University, Palo Alto, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gillian L. Dornan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Microbiology, University
of Victoria, Victoria, BC V8W 2Y2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brandon Tavshanjian</span> - <span class="hlFld-Affiliation affiliation">Howard
Hughes Medical Institute and Department of Cellular and Molecular
Pharmacology, University of California,
San Francisco (UCSF), San Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey S. Glenn</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine and Department of Microbiology
& Immunology, Stanford University, Palo Alto, California 94305, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Veterans
Administration Medical Center, Palo Alto, California 94304, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.U.R. and M.L.F. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The compounds reported here are the subject of a provisional patent application by UCSF, on which F.U.R., B.T., and K.M.S. are co-inventors.<br /></br></div></li></ul></div><div class="ack" id="ACK-d86e1806-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00570" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00570" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">J.E.B. wishes to thank CIHR for support (CIHR new investigator grant and CIHR open operating grant FRN 142393). K.M.S. and J.S.G. wish to thank NIH for support (R01 AI099245 and U19 AI109622). M.A.G. wishes to thank NIH for support (K08 AI097322). We thank the staff at the Canadian Light Source (CLS) CMCF-ID beamline, where diffraction data were collected. The CLS is supported by the Natural Sciences and Engineering Research Council of Canada, the National Research Council Canada, the Canadian Institutes of Health Research, the Province of Saskatchewan, Western Economic Diversification Canada, and the University of Saskatchewan. High-resolution mass spectrometry was provided by the Bio-Organic Biomedical Mass Spectrometry Resource at UCSF (A.L. Burlingame, Director), supported by the Biomedical Technology Research Centers program of the NIH National Institute of General Medical Sciences, NIH NIGMS 8P41GM103481 and NIH 1S10OD016229, and the Howard Hughes Medical Institute. F.U.R. wishes to thank NSF for support (Grant No. 1144247).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i34" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i34"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i35" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i35"> Abbreviations Used</h2><tr><td class="NLM_term">PI4K</td><td class="NLM_def"><p class="first last">phosphatidylinositol 4 kinase</p></td></tr><tr><td class="NLM_term">PI3K</td><td class="NLM_def"><p class="first last">phosphoinositide 3 kinase</p></td></tr><tr><td class="NLM_term">PI4P</td><td class="NLM_def"><p class="first last">phosphatidylinositol 4-phosphate</p></td></tr><tr><td class="NLM_term">PIP<sub>2</sub></td><td class="NLM_def"><p class="first last">phosphatidylinositol 4,5-bisphosphate</p></td></tr><tr><td class="NLM_term">PIP<sub>3</sub></td><td class="NLM_def"><p class="first last">phosphatidylinositol 3,4,5-trisphosphate</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 43 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphoinositides: Tiny Lipids with Giant Impact on Cell Regulation</span> <span class="citation_source-journal">Physiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">1019</span><span class="NLM_x">–</span> <span class="NLM_lpage">1137</span><span class="refDoi"> DOI: 10.1152/physrev.00028.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1152%2Fphysrev.00028.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23899561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVekurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2013&pages=1019-1137&author=T.+Balla&title=Phosphoinositides%3A+Tiny+Lipids+with+Giant+Impact+on+Cell+Regulation&doi=10.1152%2Fphysrev.00028.2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoinositides: tiny lipids with giant impact on cell regulation</span></div><div class="casAuthors">Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1019-1137</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Phosphoinositides [PIs] make up only a small fraction of cellular phospholipids, yet they control almost all aspects of a cell's life and death.  These lipids gained tremendous research interest as plasma membrane signaling mols. when discovered in the 1970s and 1980s.  Research in the last 15 years has added a wide range of biol. processes regulated by PIs, turning these lipids into one of the most universal signaling entities in eukaryotic cells.  PIs control organelle biol. by regulating vesicular trafficking, but they also modulate lipid distribution and metab. via their close relationship with lipid transfer proteins.  PIs regulate ion channels, pumps, and transporters and control both endocytic and exocytic processes.  The nuclear phosphoinositides have grown from being an epiphenomenon to a research area of its own.  As expected from such pleiotropic regulators, derangements of phosphoinositide metab. are responsible for a no. of human diseases ranging from rare genetic disorders to the most common ones such as cancer, obesity, and diabetes.  Moreover, it is increasingly evident that a no. of infectious agents hijack the PI regulatory systems of host cells for their intracellular movements, replication, and assembly.  As a result, PI converting enzymes began to be noticed by pharmaceutical companies as potential therapeutic targets.  This review is an attempt to give an overview of this enormous research field focusing on major developments in diverse areas of basic science linked to cellular physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGornhXAaVsNoLVg90H21EOLACvtfcHk0lgzhvQ0ipRm9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVekurfJ&md5=98d2bcb9092cc2105e0947b49cf7a2a6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00028.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00028.2012%26sid%3Dliteratum%253Aachs%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphoinositides%253A%2520Tiny%2520Lipids%2520with%2520Giant%2520Impact%2520on%2520Cell%2520Regulation%26jtitle%3DPhysiol.%2520Rev.%26date%3D2013%26volume%3D93%26spage%3D1019%26epage%3D1137%26doi%3D10.1152%2Fphysrev.00028.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Dornan, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhail, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. E.</span><span> </span><span class="NLM_article-title">Type III Phosphatidylinositol 4 Kinases: Structure, Function, Regulation, Signalling and Involvement in Disease</span> <span class="citation_source-journal">Biochem. Soc. Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1042/BST20150219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1042%2FBST20150219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=26862213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=260-266&author=G.+L.+Dornanauthor=J.+A.+McPhailauthor=J.+E.+Burke&title=Type+III+Phosphatidylinositol+4+Kinases%3A+Structure%2C+Function%2C+Regulation%2C+Signalling+and+Involvement+in+Disease&doi=10.1042%2FBST20150219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Type III phosphatidylinositol 4 kinases: structure, function, regulation, signaling and involvement in disease</span></div><div class="casAuthors">Dornan, Gillian L.; McPhail, Jacob A.; Burke, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">260-266</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Many important cellular functions are regulated by the selective recruitment of proteins to intracellular membranes mediated by specific interactions with lipid phosphoinositides.  The enzymes that generate lipid phosphoinositides therefore must be properly positioned and regulated at their correct cellular locations.  Phosphatidylinositol 4 kinases (PI4Ks) are key lipid signaling enzymes, and they generate the lipid species, phosphatidylinositol 4-phosphate (PI4P), which plays important roles in regulating physiol. processes including membrane trafficking, cytokinesis, and organelle identity.  PI4P also acts as the substrate for the generation of the signaling phosphoinositides, phosphatidylinositol 4,5-bisphosphate (PIP2) and phosphatidylinositol 3,4,5-trisphosphate (PIP3).  PI4Ks also play crit. roles in a no. of pathol. processes including mediating replication of a no. of pathogenic RNA viruses, and in the development of the parasite responsible for malaria.  Key to the regulation of PI4Ks is their regulation by a variety of both host and viral protein-binding partners.  Here, the authors review the current understanding of the structure, regulatory interactions, and role in disease of the type III PI4Ks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nMk0eWq3O7Vg90H21EOLACvtfcHk0lgzhvQ0ipRm9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVWqsL8%253D&md5=97c4d85ac73c36a42177ea49a9a741f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1042%2FBST20150219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20150219%26sid%3Dliteratum%253Aachs%26aulast%3DDornan%26aufirst%3DG.%2BL.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26atitle%3DType%2520III%2520Phosphatidylinositol%25204%2520Kinases%253A%2520Structure%252C%2520Function%252C%2520Regulation%252C%2520Signalling%2520and%2520Involvement%2520in%2520Disease%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2016%26volume%3D44%26spage%3D260%26epage%3D266%26doi%3D10.1042%2FBST20150219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span> </span><span class="NLM_article-title">Cinderella Story: PI4P Goes From Precursor to Key Signaling Molecule</span> <span class="citation_source-journal">Crit. Rev. Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.3109/10409238.2013.853024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.3109%2F10409238.2013.853024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24219382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2014&pages=33-58&author=J.+Tanauthor=J.+A.+Brill&title=Cinderella+Story%3A+PI4P+Goes+From+Precursor+to+Key+Signaling+Molecule&doi=10.3109%2F10409238.2013.853024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cinderella story: PI4P goes from precursor to key signaling molecule</span></div><div class="casAuthors">Tan, Julie; Brill, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-58</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Phosphatidylinositol lipids are signaling mols. involved in nearly all aspects of cellular regulation.  Prodn. of phosphatidylinositol 4-phosphate (PI4P) has long been recognized as one of the first steps in generating poly-phosphatidylinositol phosphates involved in actin organization, cell migration, and signal transduction.  In addn., progress over the last decade has brought to light independent roles for PI4P in membrane trafficking and lipid homeostasis.  Here, we describe recent advances that reveal the breadth of processes regulated by PI4P, the spectrum of PI4P effectors, and the mechanisms of spatiotemporal control that coordinate crosstalk between PI4P and cellular signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOx335EArcj7Vg90H21EOLACvtfcHk0liRo-3CUbc2uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCrsLw%253D&md5=0ad4d75e67adc6f2d58b241e3e99a7d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.853024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.853024%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26atitle%3DCinderella%2520Story%253A%2520PI4P%2520Goes%2520From%2520Precursor%2520to%2520Key%2520Signaling%2520Molecule%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D49%26spage%3D33%26epage%3D58%26doi%3D10.3109%2F10409238.2013.853024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Tóth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase IIIbeta Regulates the Transport of Ceramide Between the Endoplasmic Reticulum and Golgi</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">36369</span><span class="NLM_x">–</span> <span class="NLM_lpage">36377</span><span class="refDoi"> DOI: 10.1074/jbc.M604935200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1074%2Fjbc.M604935200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=36369-36377&author=B.+T%C3%B3thauthor=A.+Ballaauthor=H.+Maauthor=Z.+A.+Knightauthor=K.+M.+Shokatauthor=T.+Balla&title=Phosphatidylinositol+4-Kinase+IIIbeta+Regulates+the+Transport+of+Ceramide+Between+the+Endoplasmic+Reticulum+and+Golgi&doi=10.1074%2Fjbc.M604935200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M604935200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M604935200%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25B3th%26aufirst%3DB.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520IIIbeta%2520Regulates%2520the%2520Transport%2520of%2520Ceramide%2520Between%2520the%2520Endoplasmic%2520Reticulum%2520and%2520Golgi%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D36369%26epage%3D36377%26doi%3D10.1074%2Fjbc.M604935200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Polevoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldbach, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbach, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span> </span><span class="NLM_article-title">Dual Roles for the Drosophila PI 4-Kinase Four Wheel Drive in Localizing Rab11 During Cytokinesis</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">858</span><span class="refDoi"> DOI: 10.1083/jcb.200908107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1083%2Fjcb.200908107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2009&pages=847-858&author=G.+Polevoyauthor=H.-C.+Weiauthor=R.+Wongauthor=Z.+Szentpeteryauthor=Y.+J.+Kimauthor=P.+Goldbachauthor=S.+K.+Steinbachauthor=T.+Ballaauthor=J.+A.+Brill&title=Dual+Roles+for+the+Drosophila+PI+4-Kinase+Four+Wheel+Drive+in+Localizing+Rab11+During+Cytokinesis&doi=10.1083%2Fjcb.200908107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200908107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200908107%26sid%3Dliteratum%253Aachs%26aulast%3DPolevoy%26aufirst%3DG.%26aulast%3DWei%26aufirst%3DH.-C.%26aulast%3DWong%26aufirst%3DR.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DGoldbach%26aufirst%3DP.%26aulast%3DSteinbach%26aufirst%3DS.%2BK.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26atitle%3DDual%2520Roles%2520for%2520the%2520Drosophila%2520PI%25204-Kinase%2520Four%2520Wheel%2520Drive%2520in%2520Localizing%2520Rab11%2520During%2520Cytokinesis%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2009%26volume%3D187%26spage%3D847%26epage%3D858%26doi%3D10.1083%2Fjcb.200908107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Sridhar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aphkhazava, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shields, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuervo, A. M.</span><span> </span><span class="NLM_article-title">The Lipid Kinase PI4KIIIβ Preserves Lysosomal Identity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/emboj.2012.341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Femboj.2012.341" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=324-339&author=S.+Sridharauthor=B.+Patelauthor=D.+Aphkhazavaauthor=F.+Macianauthor=L.+Santambrogioauthor=D.+Shieldsauthor=A.+M.+Cuervo&title=The+Lipid+Kinase+PI4KIII%CE%B2+Preserves+Lysosomal+Identity&doi=10.1038%2Femboj.2012.341"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.341%26sid%3Dliteratum%253Aachs%26aulast%3DSridhar%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DAphkhazava%26aufirst%3DD.%26aulast%3DMacian%26aufirst%3DF.%26aulast%3DSantambrogio%26aufirst%3DL.%26aulast%3DShields%26aufirst%3DD.%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26atitle%3DThe%2520Lipid%2520Kinase%2520PI4KIII%25CE%25B2%2520Preserves%2520Lysosomal%2520Identity%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D32%26spage%3D324%26epage%3D339%26doi%3D10.1038%2Femboj.2012.341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Jean, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiger, A. A.</span><span> </span><span class="NLM_article-title">Coordination Between RAB GTPase and Phosphoinositide Regulation and Functions</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">470</span><span class="refDoi"> DOI: 10.1038/nrm3379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnrm3379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22722608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFykt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=463-470&author=S.+Jeanauthor=A.+A.+Kiger&title=Coordination+Between+RAB+GTPase+and+Phosphoinositide+Regulation+and+Functions&doi=10.1038%2Fnrm3379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Coordination between RAB GTPase and phosphoinositide regulation and functions</span></div><div class="casAuthors">Jean, Steve; Kiger, Amy A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">463-470</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Membrane trafficking relies on dynamic changes in membrane identities that are detd. by the regulation of distinct RAB GTPases and phosphoinositides.  RABs and phosphoinositides both act to spatiotemporally recruit effectors of membrane remodelling, including sequential RAB and phosphoinositide activities.  New ideas on coordinated regulation of specific RABs and phosphoinositides, achieved by direct phys. and functional interactions between their regulatory enzymes, are emerging as a central mechanism to ensure precision and fidelity of membrane trafficking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwz4JkFEj6iLVg90H21EOLACvtfcHk0liRo-3CUbc2uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFykt7g%253D&md5=91e7f7bc55d0f94cf05dc5a2b4009c65</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm3379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3379%26sid%3Dliteratum%253Aachs%26aulast%3DJean%26aufirst%3DS.%26aulast%3DKiger%26aufirst%3DA.%2BA.%26atitle%3DCoordination%2520Between%2520RAB%2520GTPase%2520and%2520Phosphoinositide%2520Regulation%2520and%2520Functions%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D463%26epage%3D470%26doi%3D10.1038%2Fnrm3379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hsu, N.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilnytska, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takvorian, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Schaar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik-Basu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrenfeld, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altan-Bonnet, N.</span><span> </span><span class="NLM_article-title">Viral Reorganization of the Secretory Pathway Generates Distinct Organelles for RNA Replication</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">799</span><span class="NLM_x">–</span> <span class="NLM_lpage">811</span><span class="refDoi"> DOI: 10.1016/j.cell.2010.03.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.cell.2010.03.050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=799-811&author=N.-Y.+Hsuauthor=O.+Ilnytskaauthor=G.+Belovauthor=M.+Santianaauthor=Y.-H.+Chenauthor=P.+M.+Takvorianauthor=C.+Pauauthor=H.+van+der+Schaarauthor=N.+Kaushik-Basuauthor=T.+Ballaauthor=C.+E.+Cameronauthor=E.+Ehrenfeldauthor=F.+J.+M.+van+Kuppeveldauthor=N.+Altan-Bonnet&title=Viral+Reorganization+of+the+Secretory+Pathway+Generates+Distinct+Organelles+for+RNA+Replication&doi=10.1016%2Fj.cell.2010.03.050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.03.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.03.050%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DN.-Y.%26aulast%3DIlnytska%26aufirst%3DO.%26aulast%3DBelov%26aufirst%3DG.%26aulast%3DSantiana%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DTakvorian%26aufirst%3DP.%2BM.%26aulast%3DPau%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%26aulast%3DKaushik-Basu%26aufirst%3DN.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DCameron%26aufirst%3DC.%2BE.%26aulast%3DEhrenfeld%26aufirst%3DE.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26aulast%3DAltan-Bonnet%26aufirst%3DN.%26atitle%3DViral%2520Reorganization%2520of%2520the%2520Secretory%2520Pathway%2520Generates%2520Distinct%2520Organelles%2520for%2520RNA%2520Replication%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D799%26epage%3D811%26doi%3D10.1016%2Fj.cell.2010.03.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">McNamara, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roland, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, B. K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormack, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manary, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeeman, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dechering, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, T. R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrich, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagaring, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibanez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhen, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plouffe, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meister, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rameh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trappe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haasen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmerman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauerwein, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suwanarusk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nosten, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tully, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kocken, C. H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glynne, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodenreider, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidock, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diagana, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winzeler, E. A.</span><span> </span><span class="NLM_article-title">Targeting Plasmodium PI(4)K to Eliminate Malaria</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1038/nature12782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnature12782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24284631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFaqsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=248-253&author=C.+W.+McNamaraauthor=M.+C.+S.+Leeauthor=C.+S.+Limauthor=S.+H.+Limauthor=J.+Rolandauthor=A.+Nagleauthor=O.+Simonauthor=B.+K.+S.+Yeungauthor=A.+K.+Chatterjeeauthor=S.+L.+McCormackauthor=M.+J.+Manaryauthor=A.-M.+Zeemanauthor=K.+J.+Decheringauthor=T.+R.+S.+Kumarauthor=P.+P.+Henrichauthor=K.+Gagaringauthor=M.+Ibanezauthor=N.+Katoauthor=K.+L.+Kuhenauthor=C.+Fischliauthor=M.+Rottmannauthor=D.+M.+Plouffeauthor=B.+Bursulayaauthor=S.+Meisterauthor=L.+Ramehauthor=J.+Trappeauthor=D.+Haasenauthor=M.+Timmermanauthor=R.+W.+Sauerweinauthor=R.+Suwanaruskauthor=B.+Russellauthor=L.+Reniaauthor=F.+Nostenauthor=D.+C.+Tullyauthor=C.+H.+M.+Kockenauthor=R.+J.+Glynneauthor=C.+Bodenreiderauthor=D.+A.+Fidockauthor=T.+T.+Diaganaauthor=E.+A.+Winzeler&title=Targeting+Plasmodium+PI%284%29K+to+Eliminate+Malaria&doi=10.1038%2Fnature12782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Plasmodium PI(4)K to eliminate malaria</span></div><div class="casAuthors">McNamara, Case W.; Lee, Marcus C. S.; Lim, Chek Shik; Lim, Siau Hoi; Roland, Jason; Nagle, Advait; Simon, Oliver; Yeung, Bryan K. S.; Chatterjee, Arnab K.; McCormack, Susan L.; Manary, Micah J.; Zeeman, Anne-Marie; Dechering, Koen J.; Kumar, T. R. Santha; Henrich, Philipp P.; Gagaring, Kerstin; Ibanez, Maureen; Kato, Nobutaka; Kuhen, Kelli L.; Fischli, Christoph; Rottmann, Matthias; Plouffe, David M.; Bursulaya, Badry; Meister, Stephan; Rameh, Lucia; Trappe, Joerg; Haasen, Dorothea; Timmerman, Martijn; Sauerwein, Robert W.; Suwanarusk, Rossarin; Russell, Bruce; Renia, Laurent; Nosten, Francois; Tully, David C.; Kocken, Clemens H. M.; Glynne, Richard J.; Bodenreider, Christophe; Fidock, David A.; Diagana, Thierry T.; Winzeler, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7479</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Achieving the goal of malaria elimination will depend on targeting Plasmodium pathways essential across all life stages.  Here the authors identify a lipid kinase, phosphatidylinositol-4-OH kinase (PI(4)K), as the target of imidazopyrazines, a new antimalarial compd. class that inhibits the intracellular development of multiple Plasmodium species at each stage of infection in the vertebrate host.  Imidazopyrazines demonstrate potent preventive, therapeutic, and transmission-blocking activity in rodent malaria models, are active against blood-stage field isolates of the major human pathogens P. falciparum and P. vivax, and inhibit liver-stage hypnozoites in the simian parasite P. cynomolgi.  The authors show that imidazopyrazines exert their effect through inhibitory interaction with the ATP-binding pocket of PI(4)K, altering the intracellular distribution of phosphatidylinositol-4-phosphate.  Collectively, the authors' data define PI(4)K as a key Plasmodium vulnerability, opening up new avenues of target-based discovery to identify drugs with an ideal activity profile for the prevention, treatment and elimination of malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTOmMeWluIubVg90H21EOLACvtfcHk0li8y6XC2ENijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFaqsLzP&md5=9ed8f3e41d257bb4535b4c813abd54cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature12782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12782%26sid%3Dliteratum%253Aachs%26aulast%3DMcNamara%26aufirst%3DC.%2BW.%26aulast%3DLee%26aufirst%3DM.%2BC.%2BS.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DLim%26aufirst%3DS.%2BH.%26aulast%3DRoland%26aufirst%3DJ.%26aulast%3DNagle%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DO.%26aulast%3DYeung%26aufirst%3DB.%2BK.%2BS.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DMcCormack%26aufirst%3DS.%2BL.%26aulast%3DManary%26aufirst%3DM.%2BJ.%26aulast%3DZeeman%26aufirst%3DA.-M.%26aulast%3DDechering%26aufirst%3DK.%2BJ.%26aulast%3DKumar%26aufirst%3DT.%2BR.%2BS.%26aulast%3DHenrich%26aufirst%3DP.%2BP.%26aulast%3DGagaring%26aufirst%3DK.%26aulast%3DIbanez%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DKuhen%26aufirst%3DK.%2BL.%26aulast%3DFischli%26aufirst%3DC.%26aulast%3DRottmann%26aufirst%3DM.%26aulast%3DPlouffe%26aufirst%3DD.%2BM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMeister%26aufirst%3DS.%26aulast%3DRameh%26aufirst%3DL.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DHaasen%26aufirst%3DD.%26aulast%3DTimmerman%26aufirst%3DM.%26aulast%3DSauerwein%26aufirst%3DR.%2BW.%26aulast%3DSuwanarusk%26aufirst%3DR.%26aulast%3DRussell%26aufirst%3DB.%26aulast%3DRenia%26aufirst%3DL.%26aulast%3DNosten%26aufirst%3DF.%26aulast%3DTully%26aufirst%3DD.%2BC.%26aulast%3DKocken%26aufirst%3DC.%2BH.%2BM.%26aulast%3DGlynne%26aufirst%3DR.%2BJ.%26aulast%3DBodenreider%26aufirst%3DC.%26aulast%3DFidock%26aufirst%3DD.%2BA.%26aulast%3DDiagana%26aufirst%3DT.%2BT.%26aulast%3DWinzeler%26aufirst%3DE.%2BA.%26atitle%3DTargeting%2520Plasmodium%2520PI%25284%2529K%2520to%2520Eliminate%2520Malaria%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D248%26epage%3D253%26doi%3D10.1038%2Fnature12782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Altan-Bonnet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinases: Hostages Harnessed to Build Panviral Replication Platforms</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1016/j.tibs.2012.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.tibs.2012.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22633842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1WmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2012&pages=293-302&author=N.+Altan-Bonnetauthor=T.+Balla&title=Phosphatidylinositol+4-Kinases%3A+Hostages+Harnessed+to+Build+Panviral+Replication+Platforms&doi=10.1016%2Fj.tibs.2012.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinases: hostages harnessed to build panviral replication platforms</span></div><div class="casAuthors">Altan-Bonnet, Nihal; Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Several RNA viruses have recently been shown to hijack members of the host phosphatidylinositol (PtdIns) 4-kinase (PI4K) family of enzymes.  They use PI4K to generate membranes enriched in phosphatidylinositide 4-phosphate (PtdIns4P or PI4P) lipids, which can be used as replication platforms.  Viral replication machinery is assembled on these platforms as a supramol. complex and PtdIns4P lipids regulate viral RNA synthesis.  This article highlights these recent studies on the regulation of viral RNA synthesis by PtdIns4P lipids.  It explores the potential mechanisms by which PtdIns4P lipids can contribute to viral replication and discusses the therapeutic potential of developing antiviral mols. that target host PI4Ks as a form of panviral therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-kUkSjCTrr7Vg90H21EOLACvtfcHk0lhCpG99YlV5-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1WmtLk%253D&md5=11c0516aa3df27d16d6e945644697886</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2012.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2012.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DAltan-Bonnet%26aufirst%3DN.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphatidylinositol%25204-Kinases%253A%2520Hostages%2520Harnessed%2520to%2520Build%2520Panviral%2520Replication%2520Platforms%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2012%26volume%3D37%26spage%3D293%26epage%3D302%26doi%3D10.1016%2Fj.tibs.2012.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Mello, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguayo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cihlar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birkus, G.</span><span> </span><span class="NLM_article-title">Multiple Classes of Antiviral Agents Exhibit in Vitro Activity Against Human Rhinovirus Type C</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">–</span> <span class="NLM_lpage">1555</span><span class="refDoi"> DOI: 10.1128/AAC.01746-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.01746-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=1546-1555&author=C.+Melloauthor=E.+Aguayoauthor=M.+Rodriguezauthor=G.+Leeauthor=R.+Jordanauthor=T.+Cihlarauthor=G.+Birkus&title=Multiple+Classes+of+Antiviral+Agents+Exhibit+in+Vitro+Activity+Against+Human+Rhinovirus+Type+C&doi=10.1128%2FAAC.01746-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.01746-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01746-13%26sid%3Dliteratum%253Aachs%26aulast%3DMello%26aufirst%3DC.%26aulast%3DAguayo%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DJordan%26aufirst%3DR.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DBirkus%26aufirst%3DG.%26atitle%3DMultiple%2520Classes%2520of%2520Antiviral%2520Agents%2520Exhibit%2520in%2520Vitro%2520Activity%2520Against%2520Human%2520Rhinovirus%2520Type%2520C%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D1546%26epage%3D1555%26doi%3D10.1128%2FAAC.01746-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">van der Schaar, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Linden, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanke, K. H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strating, J. R. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pürstinger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Haan, C. A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span> </span><span class="NLM_article-title">Coxsackievirus Mutants That Can Bypass Host Factor PI4KIIIβ and the Need for High Levels of PI4P Lipids for Replication</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1576</span><span class="NLM_x">–</span> <span class="NLM_lpage">1592</span><span class="refDoi"> DOI: 10.1038/cr.2012.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fcr.2012.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22945356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1arsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1576-1592&author=H.+M.+van+der+Schaarauthor=L.+van+der+Lindenauthor=K.+H.+W.+Lankeauthor=J.+R.+P.+M.+Stratingauthor=G.+P%C3%BCrstingerauthor=E.+de+Vriesauthor=C.+A.+M.+de+Haanauthor=J.+Neytsauthor=F.+J.+M.+van+Kuppeveld&title=Coxsackievirus+Mutants+That+Can+Bypass+Host+Factor+PI4KIII%CE%B2+and+the+Need+for+High+Levels+of+PI4P+Lipids+for+Replication&doi=10.1038%2Fcr.2012.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Coxsackievirus mutants that can bypass host factor PI4KIIIβ and the need for high levels of PI4P lipids for replication</span></div><div class="casAuthors">van der Schaar, Hilde M.; van der Linden, Lonneke; Lanke, Kjerstin H. W.; Strating, Jeroen R. P. M.; Puerstinger, Gerhard; de Vries, Erik; de Haan, Cornelis A. M.; Neyts, Johan; van Kuppeveld, Frank J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1576-1592</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">RNA viruses can rapidly mutate and acquire resistance to drugs that directly target viral enzymes, which poses serious problems in a clin. context.  Therefore, there is a growing interest in the development of antiviral drugs that target host factors crit. for viral replication, since they are unlikely to mutate in response to therapy.  We recently demonstrated that phosphatidylinositol-4-kinase IIIβ (PI4KIIIβ) and its product phosphatidylinositol-4-phosphate (PI4P) are essential for replication of enteroviruses, a group of medically important RNA viruses including poliovirus (PV), coxsackievirus, rhinovirus, and enterovirus 71.  Here, we show that enviroxime and GW5074 decreased PI4P levels at the Golgi complex by directly inhibiting PI4KIIIβ.  Coxsackievirus mutants resistant to these inhibitors harbor single point mutations in the non-structural protein 3A.  These 3A mutations did not confer compd.-resistance by restoring the activity of PI4KIIIβ in the presence of the compds.  Instead, replication of the mutant viruses no longer depended on PI4KIIIβ, since their replication was insensitive to siRNA-mediated depletion of PI4KIIIβ.  The mutant viruses also did not rely on other isoforms of PI4K.  Consistently, no high level of PI4P could be detected at the replication sites induced by the mutant viruses in the presence of the compds.  Collectively, these findings indicate that through specific single point mutations in 3A, CVB3 can bypass an essential host factor and lipid for its propagation, which is a new example of RNA viruses acquiring resistance against antiviral compds., even when they directly target host factors.  Cell Research (2012) 22:1576-1592. doi:10.1038/cr.2012.129; published online 4 Sept. 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6O0gJl98m4rVg90H21EOLACvtfcHk0lhCpG99YlV5-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1arsLjF&md5=dc40ab703aba2302324cc44929f97932</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fcr.2012.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2012.129%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%2BM.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DL.%26aulast%3DLanke%26aufirst%3DK.%2BH.%2BW.%26aulast%3DStrating%26aufirst%3DJ.%2BR.%2BP.%2BM.%26aulast%3DP%25C3%25BCrstinger%26aufirst%3DG.%26aulast%3Dde%2BVries%26aufirst%3DE.%26aulast%3Dde%2BHaan%26aufirst%3DC.%2BA.%2BM.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26atitle%3DCoxsackievirus%2520Mutants%2520That%2520Can%2520Bypass%2520Host%2520Factor%2520PI4KIII%25CE%25B2%2520and%2520the%2520Need%2520for%2520High%2520Levels%2520of%2520PI4P%2520Lipids%2520for%2520Replication%26jtitle%3DCell%2520Res.%26date%3D2012%26volume%3D22%26spage%3D1576%26epage%3D1592%26doi%3D10.1038%2Fcr.2012.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Greninger, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knudsen, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betegon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burlingame, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeRisi, J. L.</span><span> </span><span class="NLM_article-title">The 3A Protein From Multiple Picornaviruses Utilizes the Golgi Adaptor Protein ACBD3 to Recruit PI4KIIIβ</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">3605</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1128/JVI.06778-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.06778-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22258260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A280%3ADC%252BC383pvFyrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=3605-3616&author=A.+L.+Greningerauthor=G.+M.+Knudsenauthor=M.+Betegonauthor=A.+L.+Burlingameauthor=J.+L.+DeRisi&title=The+3A+Protein+From+Multiple+Picornaviruses+Utilizes+the+Golgi+Adaptor+Protein+ACBD3+to+Recruit+PI4KIII%CE%B2&doi=10.1128%2FJVI.06778-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The 3A protein from multiple picornaviruses utilizes the golgi adaptor protein ACBD3 to recruit PI4KIIIβ</span></div><div class="casAuthors">Greninger Alexander L; Knudsen Giselle M; Betegon Miguel; Burlingame Alma L; Derisi Joseph L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3605-16</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The activity of phosphatidylinositol 4-kinase class III beta (PI4KIIIβ) has been shown to be required for the replication of multiple picornaviruses; however, it is unclear whether a physical association between PI4KIIIβ and the viral replication machinery exists and, if it does, whether association is necessary.  We examined the ability of the 3A protein from 18 different picornaviruses to form a complex with PI4KIIIβ by affinity purification of Strep-Tagged transiently transfected constructs followed by mass spectrometry and Western blotting for putative interacting targets.  We found that the 3A proteins of Aichi virus, bovine kobuvirus, poliovirus, coxsackievirus B3, and human rhinovirus 14 all copurify with PI4KIIIβ.  Furthermore, we found that multiple picornavirus 3A proteins copurify with the Golgi adaptor protein acyl coenzyme A (acyl-CoA) binding domain protein 3 (ACBD3/GPC60), including those from Aichi virus, bovine kobuvirus, human rhinovirus 14, poliovirus, and coxsackievirus B2, B3, and B5.  Affinity purification of ACBD3 confirmed interaction with multiple picornaviral 3A proteins and revealed the ability to bind PI4KIIIβ in the absence of 3A.  Mass-spectrometric analysis of transiently expressed Aichi virus, bovine kobuvirus, and human klassevirus 3A proteins demonstrated that the N-terminal glycines of these 3A proteins are myristoylated.  Alanine-scanning mutagenesis along the entire length of Aichi virus 3A followed by transient expression and affinity purification revealed that copurification of PI4KIIIβ could be eliminated by mutation of specific residues, with little or no effect on recruitment of ACBD3.  One mutation at the N terminus, I5A, significantly reduced copurification of both ACBD3 and PI4KIIIβ.  The dependence of Aichi virus replication on the activity of PI4KIIIβ was confirmed by both chemical and genetic inhibition.  Knockdown of ACBD3 by small interfering RNA (siRNA) also prevented replication of both Aichi virus and poliovirus.  Point mutations in 3A that eliminate PI4KIIIβ association sensitized Aichi virus to PIK93, suggesting that disruption of the 3A/ACBD3/PI4KIIIβ complex may represent a novel target for therapeutic intervention that would be complementary to the inhibition of the kinase activity itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBIfUigH8Y6oqdXsLcGTEUfW6udTcc2ebOYUw4QnNHK7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC383pvFyrtA%253D%253D&md5=f5f1e487bed0c97e42250d778c271e72</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.06778-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.06778-11%26sid%3Dliteratum%253Aachs%26aulast%3DGreninger%26aufirst%3DA.%2BL.%26aulast%3DKnudsen%26aufirst%3DG.%2BM.%26aulast%3DBetegon%26aufirst%3DM.%26aulast%3DBurlingame%26aufirst%3DA.%2BL.%26aulast%3DDeRisi%26aufirst%3DJ.%2BL.%26atitle%3DThe%25203A%2520Protein%2520From%2520Multiple%2520Picornaviruses%2520Utilizes%2520the%2520Golgi%2520Adaptor%2520Protein%2520ACBD3%2520to%2520Recruit%2520PI4KIII%25CE%25B2%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D3605%26epage%3D3616%26doi%3D10.1128%2FJVI.06778-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Borawski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibaja, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mickanin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leighton-Davies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornellataracido, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baryza, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallarico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joberty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathy, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Compton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaither, L. A.</span><span> </span><span class="NLM_article-title">Class III Phosphatidylinositol 4-Kinase Alpha and Beta Are Novel Host Factor Regulators of Hepatitis C Virus Replication</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">10058</span><span class="NLM_x">–</span> <span class="NLM_lpage">10074</span><span class="refDoi"> DOI: 10.1128/JVI.02418-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.02418-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19605471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyrurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=10058-10074&author=J.+Borawskiauthor=P.+Trokeauthor=X.+Puyangauthor=V.+Gibajaauthor=S.+Zhaoauthor=C.+Mickaninauthor=J.+Leighton-Daviesauthor=C.+J.+Wilsonauthor=V.+Myerauthor=I.+Cornellataracidoauthor=J.+Baryzaauthor=J.+Tallaricoauthor=G.+Jobertyauthor=M.+Bantscheffauthor=M.+Schirleauthor=T.+Bouwmeesterauthor=J.+E.+Mathyauthor=K.+Linauthor=T.+Comptonauthor=M.+Labowauthor=B.+Wiedmannauthor=L.+A.+Gaither&title=Class+III+Phosphatidylinositol+4-Kinase+Alpha+and+Beta+Are+Novel+Host+Factor+Regulators+of+Hepatitis+C+Virus+Replication&doi=10.1128%2FJVI.02418-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Class III phosphatidylinositol 4-kinase alpha and beta are novel host factor regulators of hepatitis C virus replication</span></div><div class="casAuthors">Borawski, Jason; Troke, Philip; Puyang, Xiaoling; Gibaja, Veronica; Zhao, ShanChaun; Mickanin, Craig; Leighton-Davies, Juliet; Wilson, Christopher J.; Myer, Vic; Taracido, Ivan Cornella; Baryza, Jeremy; Tallarico, John; Joberty, Gerard; Bantscheff, Marcus; Schirle, Markus; Bouwmeester, Tewis; Mathy, Joanna E.; Lin, Kai; Compton, Teresa; Labow, Mark; Wiedmann, Brigitte; Gaither, L. Alex</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10058-10074</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Host factor pathways are known to be essential for hepatitis C virus (HCV) infection and replication in human liver cells.  To search for novel host factor proteins required for HCV replication, we screened a subgenomic genotype 1b replicon cell line (Luc-1b) with a kinome and druggable collection of 20,779 siRNAs.  We identified and validated several enzymes required for HCV replication, including class III phosphatidylinositol 4-kinases (PI4KA and PI4KB), carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase (CAD), and mevalonate (diphospho) decarboxylase.  Knockdown of PI4KA could inhibit the replication and/or HCV RNA levels of the two subgenomic genotype 1b clones (SG-1b and Luc-1b), two subgenomic genotype 1a clones (SG-1a and Luc-1a), JFH-1 genotype 2a infectious virus (JFH1-2a), and the genomic genotype 1a (FL-1a) replicon.  In contrast, PI4KB knockdown inhibited replication and/or HCV RNA levels of Luc-1b, SG-1b, and Luc-1a replicons.  The small mol. inhibitor, PIK93, was found to block subgenomic genotype 1b (Luc-1b), subgenomic genotype 1a (Luc-1a), and genomic genotype 2a (JFH1-2a) infectious virus replication in the nanomolar range.  PIK93 was characterized by using quant. chem. proteomics and in vitro biochem. assays to demonstrate PIK93 is a bone fide PI4KA and PI4KB inhibitor.  Our data demonstrate that genetic or pharmacol. modulation of PI4KA and PI4KB inhibits multiple genotypes of HCV and represents a novel druggable class of therapeutic targets for HCV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Zjz0O33L9bVg90H21EOLACvtfcHk0liRw4YPlyxP7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyrurjN&md5=f823b4ac8a9efd864fced393cf487c94</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FJVI.02418-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02418-08%26sid%3Dliteratum%253Aachs%26aulast%3DBorawski%26aufirst%3DJ.%26aulast%3DTroke%26aufirst%3DP.%26aulast%3DPuyang%26aufirst%3DX.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLeighton-Davies%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DCornellataracido%26aufirst%3DI.%26aulast%3DBaryza%26aufirst%3DJ.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DMathy%26aufirst%3DJ.%2BE.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DCompton%26aufirst%3DT.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DWiedmann%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26atitle%3DClass%2520III%2520Phosphatidylinositol%25204-Kinase%2520Alpha%2520and%2520Beta%2520Are%2520Novel%2520Host%2520Factor%2520Regulators%2520of%2520Hepatitis%2520C%2520Virus%2520Replication%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D10058%26epage%3D10074%26doi%3D10.1128%2FJVI.02418-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">van der Schaar, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyssen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibaut, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Palma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Linden, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanke, K. H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbeken, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conrath, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Poël, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span> </span><span class="NLM_article-title">A Novel, Broad-Spectrum Inhibitor of Enterovirus Replication That Targets Host Cell Factor Phosphatidylinositol 4-Kinase IIIβ</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4971</span><span class="NLM_x">–</span> <span class="NLM_lpage">4981</span><span class="refDoi"> DOI: 10.1128/AAC.01175-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.01175-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=4971-4981&author=H.+M.+van+der+Schaarauthor=P.+Leyssenauthor=H.+J.+Thibautauthor=A.+de+Palmaauthor=L.+van+der+Lindenauthor=K.+H.+W.+Lankeauthor=C.+Lacroixauthor=E.+Verbekenauthor=K.+Conrathauthor=A.+M.+Macleodauthor=D.+R.+Mitchellauthor=N.+J.+Palmerauthor=H.+van+de+Po%C3%ABlauthor=M.+Andrewsauthor=J.+Neytsauthor=F.+J.+M.+van+Kuppeveld&title=A+Novel%2C+Broad-Spectrum+Inhibitor+of+Enterovirus+Replication+That+Targets+Host+Cell+Factor+Phosphatidylinositol+4-Kinase+III%CE%B2&doi=10.1128%2FAAC.01175-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.01175-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01175-13%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%2BM.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3Dde%2BPalma%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DL.%26aulast%3DLanke%26aufirst%3DK.%2BH.%2BW.%26aulast%3DLacroix%26aufirst%3DC.%26aulast%3DVerbeken%26aufirst%3DE.%26aulast%3DConrath%26aufirst%3DK.%26aulast%3DMacleod%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BR.%26aulast%3DPalmer%26aufirst%3DN.%2BJ.%26aulast%3Dvan%2Bde%2BPo%25C3%25ABl%26aufirst%3DH.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26atitle%3DA%2520Novel%252C%2520Broad-Spectrum%2520Inhibitor%2520of%2520Enterovirus%2520Replication%2520That%2520Targets%2520Host%2520Cell%2520Factor%2520Phosphatidylinositol%25204-Kinase%2520III%25CE%25B2%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D4971%26epage%3D4981%26doi%3D10.1128%2FAAC.01175-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Arita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kojima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okabe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimizu, H.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is a Target of Enviroxime-Like Compounds for Antipoliovirus Activity</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2364</span><span class="NLM_x">–</span> <span class="NLM_lpage">2372</span><span class="refDoi"> DOI: 10.1128/JVI.02249-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.02249-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21177810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVWltL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=2364-2372&author=M.+Aritaauthor=H.+Kojimaauthor=T.+Naganoauthor=T.+Okabeauthor=T.+Wakitaauthor=H.+Shimizu&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+a+Target+of+Enviroxime-Like+Compounds+for+Antipoliovirus+Activity&doi=10.1128%2FJVI.02249-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinase III beta is a target of enviroxime-like compounds for antipoliovirus activity</span></div><div class="casAuthors">Arita, Minetaro; Kojima, Hirotatsu; Nagano, Tetsuo; Okabe, Takayoshi; Wakita, Takaji; Shimizu, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2364-2372</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enviroxime is an antienterovirus compd. that targets viral protein 3A and/or 3AB and suppresses a step in enterovirus replication by unknown mechanism.  To date, four antienterovirus compds., i.e., GW5074, Flt3 inhibitor II, TTP-8307, and AN-12-H5, are known to have similar mutations in the 3A protein-encoding region causing resistance to enviroxime (a G5318A [3A-Ala70Thr] mutation in poliovirus [PV]) and are considered enviroxime-like compds.  Recently, antienterovirus activity of a phosphatidylinositol 4-kinase III beta (PI4KB) inhibitor, PIK93, was reported, suggesting that PI4KB is an important host factor targetable by antienterovirus compds.  In this study, we analyzed the inhibitory effects of previously identified enviroxime-like compds. (GW5074 and AN-12-H5) and a newly identified antienterovirus compd., T-00127-HEV1, on phosphoinositide (PI) kinases.  We found that T-00127-HEV1 inhibited PI4KB activity with a higher specificity for than other PI kinases, in contrast to GW5074, which had a broad specificity for PI kinases.  In contrast, AN-12-H5 showed no inhibitory effect on PI4KB activity and only moderate inhibitory effects on PI 3-kinase activity.  Small interfering RNA (siRNA) screening targeting PI kinases identified PI4KB is a target of GW5074 and T-00127-HEV1, but not of AN-12-H5, for anti-PV activity.  Interestingly, T-00127-HEV1 and GW5074 did not inhibit hepatitis C virus (HCV) replication, in contrast to a strong inhibitory effect of AN-12-H5.  These results suggested that PI4KB is an enterovirus-specific host factor required for the replication process and targeted by some enviroxime-like compds. (T-00127-HEV1 and GW5074) and that enviroxime-like compds. may have targets other than PI kinases for their antiviral effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRbO_t7G_n4rVg90H21EOLACvtfcHk0lj64AXrwErzRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVWltL%252FM&md5=10d970def4a0ab38c7ed9fc33e200d4f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FJVI.02249-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02249-10%26sid%3Dliteratum%253Aachs%26aulast%3DArita%26aufirst%3DM.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DH.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520a%2520Target%2520of%2520Enviroxime-Like%2520Compounds%2520for%2520Antipoliovirus%2520Activity%26jtitle%3DJ.%2520Virol.%26date%3D2011%26volume%3D85%26spage%3D2364%26epage%3D2372%26doi%3D10.1128%2FJVI.02249-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zunder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldenberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loewith, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span> </span><span class="NLM_article-title">A Pharmacological Map of the PI3-K Family Defines a Role for P110alpha in Insulin Signaling</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.cell.2006.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=16647110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlSkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=733-747&author=Z.+Knightauthor=B.+Gonzalezauthor=M.+Feldmanauthor=E.+Zunderauthor=D.+Goldenbergauthor=O.+Williamsauthor=R.+Loewithauthor=D.+Stokoeauthor=A.+Ballaauthor=B.+Tothauthor=T.+Ballaauthor=W.+Weissauthor=R.+Williamsauthor=K.+Shokat&title=A+Pharmacological+Map+of+the+PI3-K+Family+Defines+a+Role+for+P110alpha+in+Insulin+Signaling&doi=10.1016%2Fj.cell.2006.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A pharmacological map of the PI3-K family defines a role for p110α in insulin signaling</span></div><div class="casAuthors">Knight, Zachary A.; Gonzalez, Beatriz; Feldman, Morri E.; Zunder, Eli R.; Goldenberg, David D.; Williams, Olusegun; Loewith, Robbie; Stokoe, David; Balla, Andras; Toth, Balazs; Balla, Tamas; Weiss, William A.; Williams, Roger L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">733-747</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3-Ks) are an important emerging class of drug targets, but the unique roles of PI3-K isoforms remain poorly defined.  We describe here an approach to pharmacol. interrogate the PI3-K family.  A chem. diverse panel of PI3-K inhibitors was synthesized, and their target selectivity was biochem. enumerated, revealing cryptic homologies across targets and chemotypes.  Crystal structures of three inhibitors bound to p110γ identify a conformationally mobile region that is uniquely exploited by selective compds.  This chem. array was then used to define the PI3-K isoforms required for insulin signaling.  We find that p110α is the primary insulin-responsive PI3-K in cultured cells, whereas p110β is dispensable but sets a phenotypic threshold for p110α activity.  Compds. targeting p110α block the acute effects of insulin treatment in vivo, whereas a p110β inhibitor has no effect.  These results illustrate systematic target validation using a matrix of inhibitors that span a protein family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrozjvOeMXP3bVg90H21EOLACvtfcHk0lj64AXrwErzRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlSkt7o%253D&md5=b95a8cb433927ff9a75b33fc4ac2c8a0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DGonzalez%26aufirst%3DB.%26aulast%3DFeldman%26aufirst%3DM.%26aulast%3DZunder%26aufirst%3DE.%26aulast%3DGoldenberg%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DO.%26aulast%3DLoewith%26aufirst%3DR.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DB.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DWeiss%26aufirst%3DW.%26aulast%3DWilliams%26aufirst%3DR.%26aulast%3DShokat%26aufirst%3DK.%26atitle%3DA%2520Pharmacological%2520Map%2520of%2520the%2520PI3-K%2520Family%2520Defines%2520a%2520Role%2520for%2520P110alpha%2520in%2520Insulin%2520Signaling%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D733%26epage%3D747%26doi%3D10.1016%2Fj.cell.2006.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Várnai, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Maintenance of Hormone-Sensitive Phosphoinositide Pools in the Plasma Membrane Requires Phosphatidylinositol 4-Kinase IIIalpha</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">711</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span><span class="refDoi"> DOI: 10.1091/mbc.E07-07-0713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1091%2Fmbc.E07-07-0713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=18077555" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=711-721&author=A.+Ballaauthor=Y.+J.+Kimauthor=P.+V%C3%A1rnaiauthor=Z.+Szentpeteryauthor=Z.+Knightauthor=K.+M.+Shokatauthor=T.+Balla&title=Maintenance+of+Hormone-Sensitive+Phosphoinositide+Pools+in+the+Plasma+Membrane+Requires+Phosphatidylinositol+4-Kinase+IIIalpha&doi=10.1091%2Fmbc.E07-07-0713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E07-07-0713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E07-07-0713%26sid%3Dliteratum%253Aachs%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DV%25C3%25A1rnai%26aufirst%3DP.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DMaintenance%2520of%2520Hormone-Sensitive%2520Phosphoinositide%2520Pools%2520in%2520the%2520Plasma%2520Membrane%2520Requires%2520Phosphatidylinositol%25204-Kinase%2520IIIalpha%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2007%26volume%3D19%26spage%3D711%26epage%3D721%26doi%3D10.1091%2Fmbc.E07-07-0713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Burke, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perisic, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaughlin, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutaganira, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. L.</span><span> </span><span class="NLM_article-title">Structures of PI4KIIIβ Complexes Show Simultaneous Recruitment of Rab11 and Its Effectors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">344</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">1038</span><span class="refDoi"> DOI: 10.1126/science.1253397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1126%2Fscience.1253397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24876499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFSmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=1035-1038&author=J.+E.+Burkeauthor=A.+J.+Inglisauthor=O.+Perisicauthor=G.+R.+Massonauthor=S.+H.+McLaughlinauthor=F.+Rutaganiraauthor=K.+M.+Shokatauthor=R.+L.+Williams&title=Structures+of+PI4KIII%CE%B2+Complexes+Show+Simultaneous+Recruitment+of+Rab11+and+Its+Effectors&doi=10.1126%2Fscience.1253397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of PI4KIIIβ complexes show simultaneous recruitment of Rab11 and its effectors</span></div><div class="casAuthors">Burke, John E.; Inglis, Alison J.; Perisic, Olga; Masson, Glenn R.; McLaughlin, Stephen H.; Rutaganira, Florentine; Shokat, Kevan M.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6187</span>),
    <span class="NLM_cas:pages">1035-1038</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Phosphatidylinositol 4-kinases (PI4Ks) and small guanosine triphosphatases (GTPases) are essential for processes that require expansion and remodeling of phosphatidylinositol 4-phosphate (PI4P)-contg. membranes, including cytokinesis, intracellular development of malarial pathogens, and replication of a wide range of RNA viruses.  However, the structural basis for coordination of PI4K, GTPases, and their effectors is unknown.  Here, we describe structures of PI4Kβ (PI4KIIIβ) bound to the small GTPase Rab11a without and with the Rab11 effector protein FIP3.  The Rab11-PI4KIIIβ interface is distinct compared with known structures of Rab complexes and does not involve switch regions used by GTPase effectors.  Our data provide a mechanism for how PI4KIIIβ coordinates Rab11 and its effectors on PI4P-enriched membranes and also provide strategies for the design of specific inhibitors that could potentially target plasmodial PI4KIIIβ to combat malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplEdUeizaig7Vg90H21EOLACvtfcHk0lj64AXrwErzRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFSmu7w%253D&md5=9eed42f95ff8f3aeb4f6cef76e3ccdec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1253397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1253397%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DInglis%26aufirst%3DA.%2BJ.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DMasson%26aufirst%3DG.%2BR.%26aulast%3DMcLaughlin%26aufirst%3DS.%2BH.%26aulast%3DRutaganira%26aufirst%3DF.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26atitle%3DStructures%2520of%2520PI4KIII%25CE%25B2%2520Complexes%2520Show%2520Simultaneous%2520Recruitment%2520of%2520Rab11%2520and%2520Its%2520Effectors%26jtitle%3DScience%26date%3D2014%26volume%3D344%26spage%3D1035%26epage%3D1038%26doi%3D10.1126%2Fscience.1253397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavshanjian, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perisic, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houseman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R.</span><span> </span><span class="NLM_article-title">Shaping Development of Autophagy Inhibitors with the Structure of the Lipid Kinase Vps34</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">1638</span><span class="NLM_x">–</span> <span class="NLM_lpage">1642</span><span class="refDoi"> DOI: 10.1126/science.1184429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1126%2Fscience.1184429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=20339072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslaqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1638-1642&author=S.+Millerauthor=B.+Tavshanjianauthor=A.+Oleksyauthor=O.+Perisicauthor=B.+Housemanauthor=K.+Shokatauthor=R.+Williams&title=Shaping+Development+of+Autophagy+Inhibitors+with+the+Structure+of+the+Lipid+Kinase+Vps34&doi=10.1126%2Fscience.1184429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Shaping Development of Autophagy Inhibitors with the Structure of the Lipid Kinase Vps34</span></div><div class="casAuthors">Miller, Simon; Tavshanjian, Brandon; Oleksy, Arkadiusz; Perisic, Olga; Houseman, Benjamin T.; Shokat, Kevan M.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5973</span>),
    <span class="NLM_cas:pages">1638-1642</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3Ks) are lipid kinases with diverse roles in health and disease.  The primordial PI3K, Vps34, is present in all eukaryotes and has essential roles in autophagy, membrane trafficking, and cell signaling.  We solved the crystal structure of Vps34 at 2.9 angstrom resoln., which revealed a constricted adenine-binding pocket, suggesting the reason that specific inhibitors of this class of PI3K have proven elusive.  Both the phosphoinositide-binding loop and the carboxyl-terminal helix of Vps34 mediate catalysis on membranes and suppress futile ATPase cycles.  Vps34 appears to alternate between a closed cytosolic form and an open form on the membrane.  Structures of Vps34 complexes with a series of inhibitors reveal the reason that an autophagy inhibitor preferentially inhibits Vps34 and underpin the development of new potent and specific Vps34 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkkZ9w5NCkFLVg90H21EOLACvtfcHk0lgu93n5TmPivA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslaqs7s%253D&md5=f20c69ef94d3b80fba72cf04e625649e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.1184429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1184429%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DS.%26aulast%3DTavshanjian%26aufirst%3DB.%26aulast%3DOleksy%26aufirst%3DA.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DHouseman%26aufirst%3DB.%26aulast%3DShokat%26aufirst%3DK.%26aulast%3DWilliams%26aufirst%3DR.%26atitle%3DShaping%2520Development%2520of%2520Autophagy%2520Inhibitors%2520with%2520the%2520Structure%2520of%2520the%2520Lipid%2520Kinase%2520Vps34%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1638%26epage%3D1642%26doi%3D10.1126%2Fscience.1184429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Lamarche, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borawski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bose, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capacci-Daniel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colvin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drumm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaither, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeever, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thohan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tommasi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zabawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span> </span><span class="NLM_article-title">Anti-Hepatitis C Virus Activity and Toxicity of Type III Phosphatidylinositol-4-Kinase Beta Inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5149</span><span class="NLM_x">–</span> <span class="NLM_lpage">5156</span><span class="refDoi"> DOI: 10.1128/AAC.00946-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.00946-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22825118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKiu7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=5149-5156&author=M.+J.+Lamarcheauthor=J.+Borawskiauthor=A.+Boseauthor=C.+Capacci-Danielauthor=R.+Colvinauthor=M.+Dennehyauthor=J.+Dingauthor=M.+Doblerauthor=J.+Drummauthor=L.+A.+Gaitherauthor=J.+Gaoauthor=X.+Jiangauthor=K.+Linauthor=U.+McKeeverauthor=X.+Puyangauthor=P.+Ramanauthor=S.+Thohanauthor=R.+Tommasiauthor=K.+Wagnerauthor=X.+Xiongauthor=T.+Zabawaauthor=S.+Zhuauthor=B.+Wiedmann&title=Anti-Hepatitis+C+Virus+Activity+and+Toxicity+of+Type+III+Phosphatidylinositol-4-Kinase+Beta+Inhibitors&doi=10.1128%2FAAC.00946-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-hepatitis C virus activity and toxicity of type III phosphatidylinositol-4-kinase beta inhibitors</span></div><div class="casAuthors">LaMarche, M. J.; Borawski, J.; Bose, A.; Capacci-Daniel, C.; Colvin, R.; Dennehy, M.; Ding, J.; Dobler, M.; Drumm, J.; Gaither, L. A.; Gao, J.; Jiang, X.; Lin, K.; McKeever, U.; Puyang, X.; Raman, P.; Thohan, S.; Tommasi, R.; Wagner, K.; Xiong, X.; Zabawa, T.; Zhu, S.; Wiedmann, B.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5149-5156</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Type III phosphatidylinositol-4-kinase beta (PI4KIIIβ) was previously implicated in hepatitis C virus (HCV) replication by small interfering RNA (siRNA) depletion and was therefore proposed as a novel cellular target for the treatment of hepatitis C.  Medicinal chem. efforts identified highly selective PI4KIIIβ inhibitors that potently inhibited the replication of genotype 1a and 1b HCV replicons and genotype 2a virus in vitro.  Replicon cells required more than 5 wk to reach low levels of 3- to 5-fold resistance, suggesting a high resistance barrier to these cellular targets.  Extensive in vitro profiling of the compds. revealed a role of PI4KIIIβ in lymphocyte proliferation.  Previously proposed functions of PI4KIIIβ in insulin secretion and the regulation of several ion channels were not perturbed with these inhibitors.  Moreover, PI4KIIIβ inhibitors were not generally cytotoxic as demonstrated across hundreds of cell lines and primary cells.  However, an unexpected antiproliferative effect in lymphocytes precluded their further development for the treatment of hepatitis C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg3G6T2U7Lv7Vg90H21EOLACvtfcHk0lgu93n5TmPivA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKiu7jK&md5=fd7a50d1813d04862e5d8fa549c73da5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FAAC.00946-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00946-12%26sid%3Dliteratum%253Aachs%26aulast%3DLamarche%26aufirst%3DM.%2BJ.%26aulast%3DBorawski%26aufirst%3DJ.%26aulast%3DBose%26aufirst%3DA.%26aulast%3DCapacci-Daniel%26aufirst%3DC.%26aulast%3DColvin%26aufirst%3DR.%26aulast%3DDennehy%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DDrumm%26aufirst%3DJ.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DMcKeever%26aufirst%3DU.%26aulast%3DPuyang%26aufirst%3DX.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DK.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DZabawa%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DWiedmann%26aufirst%3DB.%26atitle%3DAnti-Hepatitis%2520C%2520Virus%2520Activity%2520and%2520Toxicity%2520of%2520Type%2520III%2520Phosphatidylinositol-4-Kinase%2520Beta%2520Inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D5149%26epage%3D5156%26doi%3D10.1128%2FAAC.00946-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Fowler, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhail, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutaganira, F. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. E.</span><span> </span><span class="NLM_article-title">Using Hydrogen Deuterium Exchange Mass Spectrometry to Engineer Optimized Constructs for Crystallization of Protein Complexes: Case Study of PI4KIIIβ with Rab11</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/pro.2879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1002%2Fpro.2879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=26756197" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+L.+Fowlerauthor=J.+A.+McPhailauthor=M.+L.+Jenkinsauthor=G.+R.+Massonauthor=F.+U.+Rutaganiraauthor=K.+M.+Shokatauthor=R.+L.+Williamsauthor=J.+E.+Burke&title=Using+Hydrogen+Deuterium+Exchange+Mass+Spectrometry+to+Engineer+Optimized+Constructs+for+Crystallization+of+Protein+Complexes%3A+Case+Study+of+PI4KIII%CE%B2+with+Rab11&doi=10.1002%2Fpro.2879"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fpro.2879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2879%26sid%3Dliteratum%253Aachs%26aulast%3DFowler%26aufirst%3DM.%2BL.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DJenkins%26aufirst%3DM.%2BL.%26aulast%3DMasson%26aufirst%3DG.%2BR.%26aulast%3DRutaganira%26aufirst%3DF.%2BU.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26atitle%3DUsing%2520Hydrogen%2520Deuterium%2520Exchange%2520Mass%2520Spectrometry%2520to%2520Engineer%2520Optimized%2520Constructs%2520for%2520Crystallization%2520of%2520Protein%2520Complexes%253A%2520Case%2520Study%2520of%2520PI4KIII%25CE%25B2%2520with%2520Rab11%26jtitle%3DProtein%2520Sci.%26date%3D2016%26doi%3D10.1002%2Fpro.2879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Jovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kean, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polevoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Two Phosphatidylinositol 4-Kinases Control Lysosomal Delivery of the Gaucher Disease Enzyme, B-Glucocerebrosidase</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1533</span><span class="NLM_x">–</span> <span class="NLM_lpage">1545</span><span class="refDoi"> DOI: 10.1091/mbc.E11-06-0553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1091%2Fmbc.E11-06-0553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22337770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFWjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=1533-1545&author=M.+Jovicauthor=M.+J.+Keanauthor=Z.+Szentpeteryauthor=G.+Polevoyauthor=A.-C.+Gingrasauthor=J.+A.+Brillauthor=T.+Balla&title=Two+Phosphatidylinositol+4-Kinases+Control+Lysosomal+Delivery+of+the+Gaucher+Disease+Enzyme%2C+B-Glucocerebrosidase&doi=10.1091%2Fmbc.E11-06-0553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Two phosphatidylinositol 4-kinases control lysosomal delivery of the Gaucher disease enzyme, β-glucocerebrosidase</span></div><div class="casAuthors">Jovic, Marko; Kean, Michelle J.; Szentpetery, Zsofia; Polevoy, Gordon; Gingras, Anne-Claude; Brill, Julie A.; Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1533-1545</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Gaucher disease is a lysosomal storage disorder caused by a defect in the degrdn. of glucosylceramide catalyzed by the lysosomal enzyme β-glucocerebrosidase (GBA).  GBA reaches lysosomes via assocn. with its receptor, lysosomal integral membrane protein type 2 (LIMP-2).  We found that distinct phosphatidylinositol 4-kinases (PI4Ks) play important roles at multiple steps in the trafficking pathway of the LIMP-2/GBA complex.  Acute depletion of phosphatidylinositol 4-phosphate in the Golgi caused accumulation of LIMP-2 in this compartment, and PI4KIIIβ was found to be responsible for controlling the exit of LIMP-2 from the Golgi.  In contrast, depletion of PI4KIIα blocked trafficking at a post-Golgi compartment, leading to accumulation of LIMP-2 in enlarged endosomal vesicles.  PI4KIIα depletion also caused secretion of missorted GBA into the medium, which was attenuated by limiting LIMP-2/GBA exit from the Golgi by PI4KIIIβ inhibitors.  These studies identified PI4KIIIβ and PI4KIIα as important regulators of lysosomal delivery of GBA, revealing a new element of control to sphingolipid homeostasis by phosphoinositides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWLQpU4eDPf7Vg90H21EOLACvtfcHk0lghxWemFw0rCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFWjsrY%253D&md5=2d56c46e7f55dda26042bb5aab8bee95</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E11-06-0553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E11-06-0553%26sid%3Dliteratum%253Aachs%26aulast%3DJovic%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DM.%2BJ.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DPolevoy%26aufirst%3DG.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DTwo%2520Phosphatidylinositol%25204-Kinases%2520Control%2520Lysosomal%2520Delivery%2520of%2520the%2520Gaucher%2520Disease%2520Enzyme%252C%2520B-Glucocerebrosidase%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2012%26volume%3D23%26spage%3D1533%26epage%3D1545%26doi%3D10.1091%2Fmbc.E11-06-0553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">D’Angelo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicinanza, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Matteis, M. A.</span><span> </span><span class="NLM_article-title">Phosphoinositides in Golgi Complex Function</span> <span class="citation_source-journal">Subcell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1007/978-94-007-3015-1_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1007%2F978-94-007-3015-1_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22374093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2012&pages=255-270&author=G.+D%E2%80%99Angeloauthor=M.+Vicinanzaauthor=C.+Wilsonauthor=M.+A.+De+Matteis&title=Phosphoinositides+in+Golgi+Complex+Function&doi=10.1007%2F978-94-007-3015-1_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoinositides in Golgi complex function</span></div><div class="casAuthors">D'Angelo, Giovanni; Vicinanza, Mariella; Wilson, Cathal; De Matteis, Maria Antonietta</div><div class="citationInfo"><span class="NLM_cas:title">Subcellular Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">Phosphoinositides II: The Diverse Biological Functions</span>),
    <span class="NLM_cas:pages">255-270</span>CODEN:
                <span class="NLM_cas:coden">SBCBAG</span>;
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The Golgi complex is a ribbon-like organelle composed of stacks of flat cisternae interconnected by tubular junctions.  It occupies a central position in the endomembrane system as proteins and lipids that are synthesized in the endoplasmic reticulum (ER) pass through the Golgi complex to undergo biosynthetic modification (mainly glycosylation) and to be sorted to their final destinations.  In addn. the Golgi complex possesses a no. of activities, apparently not directly connected with its main role in trafficking and sorting, which have been recently reviewed in Wilson et al. 2011.  In spite of the const. massive flux of material the Golgi complex maintains its identity and phosphoinositides (PIs), among other factors, play a central role in this process.  The active metab. of PIs at the Golgi is necessary for the proper functioning of the organelle both in terms of membrane trafficking/sorting and its manifold metabolic and signalling activities.  Phosphatidylinositol 4-phosphate (PtdIns4P), in particular, is responsible for the recruitment of numerous cytosolic proteins that recognize and bind PtdIns4P via specific lipid-binding domains.  In this chapter we will summarize the findings that have contributed to our current understanding of the role of PIs in the biol. of the Golgi complex in terms of the regulation of PI metab. and the functional roles and regulation of PtdIns4P effectors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWoGmyQfy8bVg90H21EOLACvtfcHk0lghxWemFw0rCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChtrzP&md5=ced1c6db948d43f03830d245c5cb15b9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-3015-1_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-94-007-3015-1_8%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DG.%26aulast%3DVicinanza%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDe%2BMatteis%26aufirst%3DM.%2BA.%26atitle%3DPhosphoinositides%2520in%2520Golgi%2520Complex%2520Function%26jtitle%3DSubcell.%2520Biochem.%26date%3D2012%26volume%3D59%26spage%3D255%26epage%3D270%26doi%3D10.1007%2F978-94-007-3015-1_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Mejdrová, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalupská, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kögler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šála, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plačková, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumlová, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hřebabecký, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Procházková, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dejmek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strunin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birkus, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertlíková-Kaiserová, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nencka, R.</span><span> </span><span class="NLM_article-title">Highly Selective Phosphatidylinositol 4-Kinase IIIβ Inhibitors and Structural Insight Into Their Mode of Action</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">3767</span><span class="NLM_x">–</span> <span class="NLM_lpage">3793</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00499</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00499" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslOit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3767-3793&author=I.+Mejdrov%C3%A1author=D.+Chalupsk%C3%A1author=M.+K%C3%B6glerauthor=M.+%C5%A0%C3%A1laauthor=P.+Pla%C4%8Dkov%C3%A1author=A.+Baumlov%C3%A1author=H.+H%C5%99ebabeck%C3%BDauthor=E.+Proch%C3%A1zkov%C3%A1author=M.+Dejmekauthor=R.+Guillonauthor=D.+Struninauthor=J.+Weberauthor=G.+Leeauthor=G.+Birkusauthor=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1author=E.+Bouraauthor=R.+Nencka&title=Highly+Selective+Phosphatidylinositol+4-Kinase+III%CE%B2+Inhibitors+and+Structural+Insight+Into+Their+Mode+of+Action&doi=10.1021%2Facs.jmedchem.5b00499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective Phosphatidylinositol 4-Kinase IIIβ Inhibitors and Structural Insight into Their Mode of Action</span></div><div class="casAuthors">Mejdrova, Ivana; Chalupska, Dominika; Kogler, Martin; Sala, Michal; Plackova, Pavla; Baumlova, Adriana; Hrebabecky, Hubert; Prochazkova, Eliska; Dejmek, Milan; Guillon, Remi; Strunin, Dmytro; Weber, Jan; Lee, Gary; Birkus, Gabriel; Mertlikova-Kaiserova, Helena; Boura, Evzen; Nencka, Radim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3767-3793</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 4-kinase IIIβ is a cellular lipid kinase pivotal to pathogenesis of various RNA viruses.  These viruses hijack the enzyme to modify the structure of intracellular membranes and use them for the construction of functional replication machinery.  Selective inhibitors of this enzyme are potential broad-spectrum antiviral agents, as inhibition of this enzyme results in the arrest of replication of PI4K IIIβ-dependent viruses.  Herein, we report a detailed study of novel selective inhibitors of PI4K IIIβ, which exert antiviral activity against a panel of single-stranded pos.-sense RNA viruses.  Our crystallog. data show that the inhibitors occupy the binding site for the adenine ring of the ATP mol. and therefore prevent the phosphorylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojwEoKXCJFnbVg90H21EOLACvtfcHk0lgQ05oOW0tn5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslOit7k%253D&md5=0f1e2c86236783ded33bfe529fafe5f1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00499%26sid%3Dliteratum%253Aachs%26aulast%3DMejdrov%25C3%25A1%26aufirst%3DI.%26aulast%3DChalupsk%25C3%25A1%26aufirst%3DD.%26aulast%3DK%25C3%25B6gler%26aufirst%3DM.%26aulast%3D%25C5%25A0%25C3%25A1la%26aufirst%3DM.%26aulast%3DPla%25C4%258Dkov%25C3%25A1%26aufirst%3DP.%26aulast%3DBaumlov%25C3%25A1%26aufirst%3DA.%26aulast%3DH%25C5%2599ebabeck%25C3%25BD%26aufirst%3DH.%26aulast%3DProch%25C3%25A1zkov%25C3%25A1%26aufirst%3DE.%26aulast%3DDejmek%26aufirst%3DM.%26aulast%3DGuillon%26aufirst%3DR.%26aulast%3DStrunin%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DBirkus%26aufirst%3DG.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26aulast%3DBoura%26aufirst%3DE.%26aulast%3DNencka%26aufirst%3DR.%26atitle%3DHighly%2520Selective%2520Phosphatidylinositol%25204-Kinase%2520III%25CE%25B2%2520Inhibitors%2520and%2520Structural%2520Insight%2520Into%2520Their%2520Mode%2520of%2520Action%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3767%26epage%3D3793%26doi%3D10.1021%2Facs.jmedchem.5b00499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Arita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philipov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galabov, A. S.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is the Target of Oxoglaucine and Pachypodol (Ro 09–0179) for Their Anti-Poliovirus Activities, and Locates at Upstream of the Target Step of Brefeldin a</span> <span class="citation_source-journal">Microbiol. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1111/1348-0421.12261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1111%2F1348-0421.12261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=25891300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGmtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=338-347&author=M.+Aritaauthor=S.+Philipovauthor=A.+S.+Galabov&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+the+Target+of+Oxoglaucine+and+Pachypodol+%28Ro+09%E2%80%930179%29+for+Their+Anti-Poliovirus+Activities%2C+and+Locates+at+Upstream+of+the+Target+Step+of+Brefeldin+a&doi=10.1111%2F1348-0421.12261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinase III beta is the target of oxoglaucine and pachypodol (Ro 09-0179) for their anti-poliovirus activities, and is located at upstream of the target step of brefeldin A</span></div><div class="casAuthors">Arita, Minetaro; Philipov, Stefan; Galabov, Angel S.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">338-347</span>CODEN:
                <span class="NLM_cas:coden">MIIMDV</span>;
        ISSN:<span class="NLM_cas:issn">0385-5600</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In recent years, phosphatidylinositol 4-kinase III beta (PI4KB) has emerged as a conserved target of anti-picornavirus compds.  PI4KB was identified as the direct target of the plant-derived anti-picornavirus compds. oxoglaucine and pachypodol (also known as Ro 09-0179).  PI4KB was also identified as the target via which pachypodol interferes with brefeldin A (BFA)-induced Golgi disassembly in non-infected cells.  Oxysterol-binding protein (OSBP) inhibitor also has interfering activity against BFA.  It seems that this interference is not essential for the anti-poliovirus (PV) activities of BFA and PI4KB/OSBP inhibitors.  BFA inhibited early to late phase PV replication (0 to 6 h postinfection) as well as PI4KB inhibitor, but with some delay compared to guanidine hydrochloride treatment.  In contrast with PI4KB/OSBP inhibitors, BFA inhibited viral nascent RNA synthesis, suggesting that BFA targets some step of viral RNA synthesis located downstream of the PI4KB/OSBP pathway in PV replication.  The authors' results suggest that PI4KB is a major target of anti-picornavirus compds. identified in vitro for their anti-picornavirus activities and for some uncharacterized biol. phenomena caused by these compds., and that BFA and PI4KB/OSBP inhibitors synergistically repress PV replication by targeting distinct steps in viral RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_6y2_9R-NK7Vg90H21EOLACvtfcHk0lgQ05oOW0tn5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGmtbjO&md5=4661d88f9c34142f0584c9f4d243e6f5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2F1348-0421.12261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1348-0421.12261%26sid%3Dliteratum%253Aachs%26aulast%3DArita%26aufirst%3DM.%26aulast%3DPhilipov%26aufirst%3DS.%26aulast%3DGalabov%26aufirst%3DA.%2BS.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520the%2520Target%2520of%2520Oxoglaucine%2520and%2520Pachypodol%2520%2528Ro%252009%25E2%2580%25930179%2529%2520for%2520Their%2520Anti-Poliovirus%2520Activities%252C%2520and%2520Locates%2520at%2520Upstream%2520of%2520the%2520Target%2520Step%2520of%2520Brefeldin%2520a%26jtitle%3DMicrobiol.%2520Immunol.%26date%3D2015%26volume%3D59%26spage%3D338%26epage%3D347%26doi%3D10.1111%2F1348-0421.12261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Waring, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulder, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemington, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKelvie, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preston, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raubo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treinies, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, J. J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, R. J.</span><span> </span><span class="NLM_article-title">Potent, Selective Small Molecule Inhibitors of Type III Phosphatidylinositol-4-Kinase A- but Not B-Inhibit the Phosphatidylinositol Signaling Cascade and Cancer Cell Proliferation</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5388</span><span class="NLM_x">–</span> <span class="NLM_lpage">5390</span><span class="refDoi"> DOI: 10.1039/C3CC48391F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1039%2FC3CC48391F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24366037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=5388-5390&author=M.+J.+Waringauthor=D.+M.+Andrewsauthor=P.+F.+Faulderauthor=V.+Flemingtonauthor=J.+C.+McKelvieauthor=S.+Mamanauthor=M.+Prestonauthor=P.+Rauboauthor=G.+R.+Robbauthor=K.+Robertsauthor=R.+Rowlinsonauthor=J.+M.+Smithauthor=M.+E.+Swarbrickauthor=I.+Treiniesauthor=J.+J.+G.+Winterauthor=R.+J.+Wood&title=Potent%2C+Selective+Small+Molecule+Inhibitors+of+Type+III+Phosphatidylinositol-4-Kinase+A-+but+Not+B-Inhibit+the+Phosphatidylinositol+Signaling+Cascade+and+Cancer+Cell+Proliferation&doi=10.1039%2FC3CC48391F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, selective small molecule inhibitors of type III phosphatidylinositol-4-kinase α- but not β-inhibit the phosphatidylinositol signaling cascade and cancer cell proliferation</span></div><div class="casAuthors">Waring, Michael J.; Andrews, David M.; Faulder, Paul F.; Flemington, Vikki; McKelvie, Jennifer C.; Maman, Sarita; Preston, Marian; Raubo, Piotr; Robb, Graeme R.; Roberts, Karen; Rowlinson, Rachel; Smith, James M.; Swarbrick, Martin E.; Treinies, Iris; Winter, Jon J. G.; Wood, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5388-5390</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of inhibitors of type III phosphatidylinositol-4-kinase were identified by high throughput screening and optimized to derive probe compds. that independently and selectively inhibit the α- and the β-isoforms with no significant activity towards related kinases in the pathway.  In a cellular environment, inhibition of the α- but not the β-subtype led to a redn. in phosphatidylinositol-4-phosphate and phosphatidylinositol-4,5-bisphosphate concn., causing inhibition of inositol-1-phosphate formation and inhibition of proliferation in a panel of cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDy9GsjuWbErVg90H21EOLACvtfcHk0lgQ05oOW0tn5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCjs7k%253D&md5=dd250429a80af1e974a28b281d801d62</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC3CC48391F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3CC48391F%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DFaulder%26aufirst%3DP.%2BF.%26aulast%3DFlemington%26aufirst%3DV.%26aulast%3DMcKelvie%26aufirst%3DJ.%2BC.%26aulast%3DMaman%26aufirst%3DS.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DJ.%2BM.%26aulast%3DSwarbrick%26aufirst%3DM.%2BE.%26aulast%3DTreinies%26aufirst%3DI.%26aulast%3DWinter%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26atitle%3DPotent%252C%2520Selective%2520Small%2520Molecule%2520Inhibitors%2520of%2520Type%2520III%2520Phosphatidylinositol-4-Kinase%2520A-%2520but%2520Not%2520B-Inhibit%2520the%2520Phosphatidylinositol%2520Signaling%2520Cascade%2520and%2520Cancer%2520Cell%2520Proliferation%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2014%26volume%3D50%26spage%3D5388%26epage%3D5390%26doi%3D10.1039%2FC3CC48391F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Spickler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laberge, M.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmeules, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellavance, É.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garneau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hucke, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyssen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaillancourt, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Décor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauthier, A.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is Essential for Replication of Human Rhinovirus and Its Inhibition Causes a Lethal Phenotype in Vivo</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">3358</span><span class="NLM_x">–</span> <span class="NLM_lpage">3368</span><span class="refDoi"> DOI: 10.1128/AAC.00303-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.00303-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=3358-3368&author=C.+Spicklerauthor=J.+Lippensauthor=M.-K.+Labergeauthor=S.+Desmeulesauthor=%C3%89.+Bellavanceauthor=M.+Garneauauthor=T.+Guoauthor=O.+Huckeauthor=P.+Leyssenauthor=J.+Neytsauthor=F.+H.+Vaillancourtauthor=A.+D%C3%A9corauthor=J.+O%E2%80%99Mearaauthor=M.+Frantiauthor=A.+Gauthier&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+Essential+for+Replication+of+Human+Rhinovirus+and+Its+Inhibition+Causes+a+Lethal+Phenotype+in+Vivo&doi=10.1128%2FAAC.00303-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FAAC.00303-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00303-13%26sid%3Dliteratum%253Aachs%26aulast%3DSpickler%26aufirst%3DC.%26aulast%3DLippens%26aufirst%3DJ.%26aulast%3DLaberge%26aufirst%3DM.-K.%26aulast%3DDesmeules%26aufirst%3DS.%26aulast%3DBellavance%26aufirst%3D%25C3%2589.%26aulast%3DGarneau%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DT.%26aulast%3DHucke%26aufirst%3DO.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DVaillancourt%26aufirst%3DF.%2BH.%26aulast%3DD%25C3%25A9cor%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DJ.%26aulast%3DFranti%26aufirst%3DM.%26aulast%3DGauthier%26aufirst%3DA.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520Essential%2520for%2520Replication%2520of%2520Human%2520Rhinovirus%2520and%2520Its%2520Inhibition%2520Causes%2520a%2520Lethal%2520Phenotype%2520in%2520Vivo%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D3358%26epage%3D3368%26doi%3D10.1128%2FAAC.00303-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Okkenhaug, K.</span><span> </span><span class="NLM_article-title">Signaling by the Phosphoinositide 3-Kinase Family in Immune Cells</span> <span class="citation_source-journal">Annu. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">675</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1146/annurev-immunol-032712-095946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1146%2Fannurev-immunol-032712-095946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23330955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFClsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=675-704&author=K.+Okkenhaug&title=Signaling+by+the+Phosphoinositide+3-Kinase+Family+in+Immune+Cells&doi=10.1146%2Fannurev-immunol-032712-095946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling by the phosphoinositide 3-kinase family in immune cells</span></div><div class="casAuthors">Okkenhaug, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">675-704</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Phosphoinositide 3-kinases (PI3Ks) control many important aspects of immune cell development, differentiation, and function.  Mammals have eight PI3K catalytic subunits that are divided into three classes based on similarities in structure and function.  Specific roles for the class I PI3Ks have been broadly investigated and are relatively well understood, as is the function of their corresponding phosphatases.  More recently, specific roles for the class II and class III PI3Ks have emerged.  Through vertebrate evolution and in parallel with the evolution of adaptive immunity, there has been a dramatic increase not only in the genes for PI3K subunits but also in genes for phosphatases that act on 3-phosphoinositides and in 3-phosphoinositide-binding proteins.  Our understanding of the PI3Ks in immunity is guided by fundamental discoveries made in simpler model organisms as well as by appreciating new adaptations of this signaling module in mammals in general and in immune cells in particular.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIM9nlUNrG1rVg90H21EOLACvtfcHk0liWS3U-k0MiSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFClsbc%253D&md5=619556fb1d852e274e17381865a248d3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032712-095946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032712-095946%26sid%3Dliteratum%253Aachs%26aulast%3DOkkenhaug%26aufirst%3DK.%26atitle%3DSignaling%2520by%2520the%2520Phosphoinositide%25203-Kinase%2520Family%2520in%2520Immune%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2013%26volume%3D31%26spage%3D675%26epage%3D704%26doi%3D10.1146%2Fannurev-immunol-032712-095946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Keaney, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sokup, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karur, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Britt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamann, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaMarche, M. J.</span><span> </span><span class="NLM_article-title">2-Alkyloxazoles as Potent and Selective PI4KIIIβ Inhibitors Demonstrating Inhibition of HCV Replication</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3718</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.bmcl.2014.07.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3714-3718&author=E.+P.+Keaneyauthor=M.+Connollyauthor=M.+Doblerauthor=R.+Karkiauthor=A.+Hondaauthor=S.+Sokupauthor=S.+Karurauthor=S.+Brittauthor=A.+Patnaikauthor=P.+Ramanauthor=L.+G.+Hamannauthor=B.+Wiedmannauthor=M.+J.+LaMarche&title=2-Alkyloxazoles+as+Potent+and+Selective+PI4KIII%CE%B2+Inhibitors+Demonstrating+Inhibition+of+HCV+Replication&doi=10.1016%2Fj.bmcl.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DKeaney%26aufirst%3DE.%2BP.%26aulast%3DConnolly%26aufirst%3DM.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DHonda%26aufirst%3DA.%26aulast%3DSokup%26aufirst%3DS.%26aulast%3DKarur%26aufirst%3DS.%26aulast%3DBritt%26aufirst%3DS.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DWiedmann%26aufirst%3DB.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3D2-Alkyloxazoles%2520as%2520Potent%2520and%2520Selective%2520PI4KIII%25CE%25B2%2520Inhibitors%2520Demonstrating%2520Inhibition%2520of%2520HCV%2520Replication%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3714%26epage%3D3718%26doi%3D10.1016%2Fj.bmcl.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">iMOSFLM: a New Graphical Interface for Diffraction-Image Processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+New+Graphical+Interface+for+Diffraction-Image+Processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0liWS3U-k0MiSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520New%2520Graphical%2520Interface%2520for%2520Diffraction-Image%2520Processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljTNLMJ7RE2Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljTNLMJ7RE2Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgQNJTTDoYYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">The Phenix Software for Automated Determination of Macromolecular Structures</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2011.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.ymeth.2011.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21821126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=94-106&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=S.+Jainauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=The+Phenix+Software+for+Automated+Determination+of+Macromolecular+Structures&doi=10.1016%2Fj.ymeth.2011.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Phenix software for automated determination of macromolecular structures</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li-Wei; Jain, Swati; Kapral, Gary J.; Grosse Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-106</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  X-ray crystallog. is a crit. tool in the study of biol. systems.  It is able to provide information that has been a prerequisite to understanding the fundamentals of life.  It is also a method that is central to the development of new therapeutics for human disease.  Significant time and effort are required to det. and optimize many macromol. structures because of the need for manual interpretation of complex numerical data, often using many different software packages, and the repeated use of interactive three-dimensional graphics.  The Phenix software package has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on automation.  This has required the development of new algorithms that minimize or eliminate subjective input in favor of built-in expert-systems knowledge, the automation of procedures that are traditionally performed by hand, and the development of a computational framework that allows a tight integration between the algorithms.  The application of automated methods is particularly appropriate in the field of structural proteomics, where high throughput is desired.  Features in Phenix for the automation of exptl. phasing with subsequent model building, mol. replacement, structure refinement and validation are described and examples given of running Phenix from both the command line and graphical user interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgjdX_7M-NcLVg90H21EOLACvtfcHk0lgQNJTTDoYYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F&md5=ba8f207cab9f3597dc0c6db885667a3f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2011.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2011.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DThe%2520Phenix%2520Software%2520for%2520Automated%2520Determination%2520of%2520Macromolecular%2520Structures%26jtitle%3DMethods%26date%3D2011%26volume%3D55%26spage%3D94%26epage%3D106%26doi%3D10.1016%2Fj.ymeth.2011.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.Refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.Refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lgQNJTTDoYYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.Refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): a Tool for Ligand Coordinate and Restraint Generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0ljoWHmnhX6OXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span> </span><span class="NLM_article-title">A Membrane Capture Assay for Lipid Kinase Activity</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2459</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1038/nprot.2007.361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnprot.2007.361" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2459-2466&author=Z.+Knightauthor=M.+Feldmanauthor=A.+Ballaauthor=T.+Ballaauthor=K.+Shokat&title=A+Membrane+Capture+Assay+for+Lipid+Kinase+Activity&doi=10.1038%2Fnprot.2007.361"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.361%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DFeldman%26aufirst%3DM.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DShokat%26aufirst%3DK.%26atitle%3DA%2520Membrane%2520Capture%2520Assay%2520for%2520Lipid%2520Kinase%2520Activity%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2459%26epage%3D2466%26doi%3D10.1038%2Fnprot.2007.361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Seiler, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surapaneni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedillo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Throop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiacco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labaer, J.</span><span> </span><span class="NLM_article-title">DNASU Plasmid and PSI:Biology-Materials Repositories: Resources to Accelerate Biological Research</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1253</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1260</span><span class="refDoi"> DOI: 10.1093/nar/gkt1060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1093%2Fnar%2Fgkt1060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24225319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1ai" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1253-D1260&author=C.+Y.+Seilerauthor=J.+G.+Parkauthor=A.+Sharmaauthor=P.+Hunterauthor=P.+Surapaneniauthor=C.+Sedilloauthor=J.+Fieldauthor=R.+Algarauthor=A.+Priceauthor=J.+Steelauthor=A.+Throopauthor=M.+Fiaccoauthor=J.+Labaer&title=DNASU+Plasmid+and+PSI%3ABiology-Materials+Repositories%3A+Resources+to+Accelerate+Biological+Research&doi=10.1093%2Fnar%2Fgkt1060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">DNASU plasmid and PSI:Biology-Materials repositories: resources to accelerate biological research</span></div><div class="casAuthors">Seiler, Catherine Y.; Park, Jin G.; Sharma, Amit; Hunter, Preston; Surapaneni, Padmini; Sedillo, Casey; Field, James; Algar, Rhys; Price, Andrea; Steel, Jason; Throop, Andrea; Fiacco, Michael; LaBaer, Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1253-D1260</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The mission of the DNASU Plasmid Repository is to accelerate research by providing high-quality, annotated plasmid samples and online plasmid resources to the research community through the curated DNASU database, website and repository http://dnasu.asu.edu or http://dnasu.org.  The collection includes plasmids from grant-funded, high-throughput cloning projects performed in our lab., plasmids from external researchers, and large collections from consortia such as the ORFeome Collaboration and the NIGMS-funded Protein Structure Initiative: Biol. PSI:Biol.  Through DNASU, researchers can search for and access detailed information about each plasmid such as the full length gene insert sequence, vector information, assocd. publications, and links to external resources that provide addnl. protein annotations and exptl. protocols.  Plasmids can be requested directly through the DNASU website.  DNASU and the PSI:Biol.-Materials Repositories were previously described in the 2010 NAR Database Issue Cormier, C.Y., Mohr, S.E., Zuo, D., Hu, Y., Rolfs, A., Kramer, J., Taycher, E., Kelley, F., Fiacco, M., Turnbull, G. et al. 2010 Protein Structure Initiative Material Repository: an open shared public resource of structural genomics plasmids for the biol. community.  Nucleic Acids Res., 38, D743-D749. In this update we will describe the plasmid collection and highlight the new features in the website redesign, including new browse/search options, plasmid annotations and a dynamic vector mapping feature that was developed in collaboration with LabGenius.  Overall, these plasmid resources continue to enable research with the goal of elucidating the role of proteins in both normal biol. processes and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6aWb1YkMc-7Vg90H21EOLACvtfcHk0ljoWHmnhX6OXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1ai&md5=423692499b7abc1b326bef66c939c722</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1060%26sid%3Dliteratum%253Aachs%26aulast%3DSeiler%26aufirst%3DC.%2BY.%26aulast%3DPark%26aufirst%3DJ.%2BG.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DHunter%26aufirst%3DP.%26aulast%3DSurapaneni%26aufirst%3DP.%26aulast%3DSedillo%26aufirst%3DC.%26aulast%3DField%26aufirst%3DJ.%26aulast%3DAlgar%26aufirst%3DR.%26aulast%3DPrice%26aufirst%3DA.%26aulast%3DSteel%26aufirst%3DJ.%26aulast%3DThroop%26aufirst%3DA.%26aulast%3DFiacco%26aufirst%3DM.%26aulast%3DLabaer%26aufirst%3DJ.%26atitle%3DDNASU%2520Plasmid%2520and%2520PSI%253ABiology-Materials%2520Repositories%253A%2520Resources%2520to%2520Accelerate%2520Biological%2520Research%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1253%26epage%3DD1260%26doi%3D10.1093%2Fnar%2Fgkt1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Park, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murthy, T. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labaer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlow, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brizuela, L.</span><span> </span><span class="NLM_article-title">Building a Human Kinase Gene Repository: Bioinformatics, Molecular Cloning, and Functional Validation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">8114</span><span class="NLM_x">–</span> <span class="NLM_lpage">8119</span><span class="refDoi"> DOI: 10.1073/pnas.0503141102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1073%2Fpnas.0503141102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8114-8119&author=J.+Parkauthor=Y.+Huauthor=T.+V.+S.+Murthyauthor=F.+Vannbergauthor=B.+Shenauthor=A.+Rolfsauthor=J.+E.+Huttiauthor=L.+C.+Cantleyauthor=J.+Labaerauthor=E.+Harlowauthor=L.+Brizuela&title=Building+a+Human+Kinase+Gene+Repository%3A+Bioinformatics%2C+Molecular+Cloning%2C+and+Functional+Validation&doi=10.1073%2Fpnas.0503141102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0503141102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0503141102%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DMurthy%26aufirst%3DT.%2BV.%2BS.%26aulast%3DVannberg%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DRolfs%26aufirst%3DA.%26aulast%3DHutti%26aufirst%3DJ.%2BE.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DLabaer%26aufirst%3DJ.%26aulast%3DHarlow%26aufirst%3DE.%26aulast%3DBrizuela%26aufirst%3DL.%26atitle%3DBuilding%2520a%2520Human%2520Kinase%2520Gene%2520Repository%253A%2520Bioinformatics%252C%2520Molecular%2520Cloning%252C%2520and%2520Functional%2520Validation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8114%26epage%3D8119%26doi%3D10.1073%2Fpnas.0503141102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Marukian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritola, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charles, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dustin, L. B.</span><span> </span><span class="NLM_article-title">Cell Culture-Produced Hepatitis C Virus Does Not Infect Peripheral Blood Mononuclear Cells</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1850</span><span class="refDoi"> DOI: 10.1002/hep.22550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1002%2Fhep.22550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1843-1850&author=S.+Marukianauthor=C.+T.+Jonesauthor=L.+Andrusauthor=M.+J.+Evansauthor=K.+D.+Ritolaauthor=E.+D.+Charlesauthor=C.+M.+Riceauthor=L.+B.+Dustin&title=Cell+Culture-Produced+Hepatitis+C+Virus+Does+Not+Infect+Peripheral+Blood+Mononuclear+Cells&doi=10.1002%2Fhep.22550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fhep.22550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.22550%26sid%3Dliteratum%253Aachs%26aulast%3DMarukian%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DC.%2BT.%26aulast%3DAndrus%26aufirst%3DL.%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DRitola%26aufirst%3DK.%2BD.%26aulast%3DCharles%26aufirst%3DE.%2BD.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DDustin%26aufirst%3DL.%2BB.%26atitle%3DCell%2520Culture-Produced%2520Hepatitis%2520C%2520Virus%2520Does%2520Not%2520Infect%2520Peripheral%2520Blood%2520Mononuclear%2520Cells%26jtitle%3DHepatology%26date%3D2008%26volume%3D48%26spage%3D1843%26epage%3D1850%26doi%3D10.1002%2Fhep.22550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Cho, N.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fram, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sklan, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elazar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koytak, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kersten, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanazawa, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glenn, J. S.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4,5-Bisphosphate Is an HCV NS5A Ligand and Mediates Replication of the Viral Genome</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">148</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span><span class="refDoi"> DOI: 10.1053/j.gastro.2014.11.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1053%2Fj.gastro.2014.11.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2015&pages=616-625&author=N.-J.+Choauthor=C.+Leeauthor=P.+S.+Pangauthor=E.+A.+Phamauthor=B.+Framauthor=K.+Nguyenauthor=A.+Xiongauthor=E.+H.+Sklanauthor=M.+Elazarauthor=E.+S.+Koytakauthor=C.+Kerstenauthor=K.+K.+Kanazawaauthor=C.+W.+Frankauthor=J.+S.+Glenn&title=Phosphatidylinositol+4%2C5-Bisphosphate+Is+an+HCV+NS5A+Ligand+and+Mediates+Replication+of+the+Viral+Genome&doi=10.1053%2Fj.gastro.2014.11.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2014.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2014.11.043%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DN.-J.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DPang%26aufirst%3DP.%2BS.%26aulast%3DPham%26aufirst%3DE.%2BA.%26aulast%3DFram%26aufirst%3DB.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DXiong%26aufirst%3DA.%26aulast%3DSklan%26aufirst%3DE.%2BH.%26aulast%3DElazar%26aufirst%3DM.%26aulast%3DKoytak%26aufirst%3DE.%2BS.%26aulast%3DKersten%26aufirst%3DC.%26aulast%3DKanazawa%26aufirst%3DK.%2BK.%26aulast%3DFrank%26aufirst%3DC.%2BW.%26aulast%3DGlenn%26aufirst%3DJ.%2BS.%26atitle%3DPhosphatidylinositol%25204%252C5-Bisphosphate%2520Is%2520an%2520HCV%2520NS5A%2520Ligand%2520and%2520Mediates%2520Replication%2520of%2520the%2520Viral%2520Genome%26jtitle%3DGastroenterology%26date%3D2015%26volume%3D148%26spage%3D616%26epage%3D625%26doi%3D10.1053%2Fj.gastro.2014.11.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swindells, M. B.</span><span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2778</span><span class="NLM_x">–</span> <span class="NLM_lpage">2786</span><span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-Protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0lhc5CrAzjmoJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-Protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 36 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Joshua A. Jackman, <span class="NLM_string-name hlFld-ContribAuthor">Nam-Joon Cho</span>. </span><span class="cited-content_cbyCitation_article-title">Supported Lipid Bilayer Formation: Beyond Vesicle Fusion. </span><span class="cited-content_cbyCitation_journal-name">Langmuir</span><span> <strong>2020,</strong> <em>36 </em>
                                    (6)
                                     , 1387-1400. <a href="https://doi.org/10.1021/acs.langmuir.9b03706" title="DOI URL">https://doi.org/10.1021/acs.langmuir.9b03706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.langmuir.9b03706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.langmuir.9b03706%26sid%3Dliteratum%253Aachs%26jtitle%3DLangmuir%26atitle%3DSupported%252BLipid%252BBilayer%252BFormation%25253A%252BBeyond%252BVesicle%252BFusion%26aulast%3DJackman%26aufirst%3DJoshua%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2020%26date%3D02122019%26date%3D25012020%26date%3D10022020%26date%3D28012020%26volume%3D36%26issue%3D6%26spage%3D1387%26epage%3D1400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">James Reuberson, Helen Horsley, Richard J. Franklin, Daniel Ford, Judi Neuss, Daniel Brookings, Qiuya Huang, Bart Vanderhoydonck, Ling-Jie Gao, Mi-Yeon Jang, Piet Herdewijn, Anant Ghawalkar, Farnaz Fallah-Arani, Adnan R. Khan, Jamie Henshall, Mark Jairaj, Sarah Malcolm, Eleanor Ward, Lindsay Shuttleworth, Yuan Lin, Shengqiao Li, Thierry Louat, Mark Waer, Jean Herman, Andrew Payne, Tom Ceska, Carl Doyle, Will Pitt, Mark Calmiano, Martin Augustin, Stefan Steinbacher, Alfred Lammens, <span class="NLM_string-name hlFld-ContribAuthor">Rodger Allen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent, Orally Bioavailable PI4KIIIβ Inhibitor (UCB9608) Able To Significantly Prolong Allogeneic Organ Engraftment in Vivo. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6705-6723. <a href="https://doi.org/10.1021/acs.jmedchem.8b00521" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00521%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%25252C%252BOrally%252BBioavailable%252BPI4KIII%2525CE%2525B2%252BInhibitor%252B%252528UCB9608%252529%252BAble%252BTo%252BSignificantly%252BProlong%252BAllogeneic%252BOrgan%252BEngraftment%252Bin%252BVivo%26aulast%3DReuberson%26aufirst%3DJames%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D03042018%26date%3D19072018%26date%3D28062018%26volume%3D61%26issue%3D15%26spage%3D6705%26epage%3D6723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew R.  Hassett</span>, <span class="hlFld-ContribAuthor ">Anna R.  Sternberg</span>, and <span class="hlFld-ContribAuthor ">Paul D.  Roepe</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibition of Human Class I vs Class III Phosphatidylinositol 3′-Kinases. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2017,</strong> <em>56 </em>
                                    (33)
                                     , 4326-4334. <a href="https://doi.org/10.1021/acs.biochem.7b00413" title="DOI URL">https://doi.org/10.1021/acs.biochem.7b00413</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.7b00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.7b00413%26sid%3Dliteratum%253Aachs%26date%3D2011%26jtitle%3DJ.%2520Med.%2520Chem.%26volume%3D54%26spage%3D1473%26epage%3D1480%26jtitle%3DBiochemistry%26atitle%3DInhibition%252Bof%252BHuman%252BClass%252BI%252Bvs%252BClass%252BIII%252BPhosphatidylinositol%252B3%2525E2%252580%2525B2-Kinases%26aulast%3DHassett%26aufirst%3DMatthew%2BR.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D02052017%26date%3D11072017%26date%3D08082017%26date%3D22082017%26date%3D18072017%26volume%3D56%26issue%3D33%26spage%3D4326%26epage%3D4334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minetaro  Arita</span>, <span class="hlFld-ContribAuthor ">Georgi  Dobrikov</span>, <span class="hlFld-ContribAuthor ">Gerhard  Pürstinger</span>, and <span class="hlFld-ContribAuthor ">Angel S.  Galabov</span>  . </span><span class="cited-content_cbyCitation_article-title">Allosteric Regulation of Phosphatidylinositol 4-Kinase III Beta by an Antipicornavirus Compound MDL-860. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2017,</strong> <em>3 </em>
                                    (8)
                                     , 585-594. <a href="https://doi.org/10.1021/acsinfecdis.7b00053" title="DOI URL">https://doi.org/10.1021/acsinfecdis.7b00053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.7b00053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.7b00053%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DAllosteric%252BRegulation%252Bof%252BPhosphatidylinositol%252B4-Kinase%252BIII%252BBeta%252Bby%252Ban%252BAntipicornavirus%252BCompound%252BMDL-860%26aulast%3DArita%26aufirst%3DMinetaro%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18042017%26date%3D28062017%26date%3D11082017%26date%3D12062017%26volume%3D3%26issue%3D8%26spage%3D585%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ivana  Mejdrová</span>, <span class="hlFld-ContribAuthor ">Dominika  Chalupská</span>, <span class="hlFld-ContribAuthor ">Pavla  Plačková</span>, <span class="hlFld-ContribAuthor ">Christin  Müller</span>, <span class="hlFld-ContribAuthor ">Michal  Šála</span>, <span class="hlFld-ContribAuthor ">Martin  Klíma</span>, <span class="hlFld-ContribAuthor ">Adriana  Baumlová</span>, <span class="hlFld-ContribAuthor ">Hubert  Hřebabecký</span>, <span class="hlFld-ContribAuthor ">Eliška  Procházková</span>, <span class="hlFld-ContribAuthor ">Milan  Dejmek</span>, <span class="hlFld-ContribAuthor ">Dmytro  Strunin</span>, <span class="hlFld-ContribAuthor ">Jan  Weber</span>, <span class="hlFld-ContribAuthor ">Gary  Lee</span>, <span class="hlFld-ContribAuthor ">Marika  Matoušová</span>, <span class="hlFld-ContribAuthor ">Helena  Mertlíková-Kaiserová</span>, <span class="hlFld-ContribAuthor ">John  Ziebuhr</span>, <span class="hlFld-ContribAuthor ">Gabriel  Birkus</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>, and <span class="hlFld-ContribAuthor ">Radim  Nencka</span>  . </span><span class="cited-content_cbyCitation_article-title">Rational Design of Novel Highly Potent and Selective Phosphatidylinositol 4-Kinase IIIβ (PI4KB) Inhibitors as Broad-Spectrum Antiviral Agents and Tools for Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (1)
                                     , 100-118. <a href="https://doi.org/10.1021/acs.jmedchem.6b01465" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01465</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01465%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Bof%252BNovel%252BHighly%252BPotent%252Band%252BSelective%252BPhosphatidylinositol%252B4-Kinase%252BIII%2525CE%2525B2%252B%252528PI4KB%252529%252BInhibitors%252Bas%252BBroad-Spectrum%252BAntiviral%252BAgents%252Band%252BTools%252Bfor%252BChemical%252BBiology%26aulast%3DMejdrov%25C3%25A1%26aufirst%3DIvana%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D05102016%26date%3D22122016%26date%3D12012017%26date%3D08122016%26volume%3D60%26issue%3D1%26spage%3D100%26epage%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jana  Humpolickova</span>, <span class="hlFld-ContribAuthor ">Ivana  Mejdrová</span>, <span class="hlFld-ContribAuthor ">Marika  Matousova</span>, <span class="hlFld-ContribAuthor ">Radim  Nencka</span>, and <span class="hlFld-ContribAuthor ">Evzen  Boura</span>  . </span><span class="cited-content_cbyCitation_article-title">Fluorescent Inhibitors as Tools To Characterize Enzymes: Case Study of the Lipid Kinase Phosphatidylinositol 4-Kinase IIIβ (PI4KB). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (1)
                                     , 119-127. <a href="https://doi.org/10.1021/acs.jmedchem.6b01466" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01466</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01466%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFluorescent%252BInhibitors%252Bas%252BTools%252BTo%252BCharacterize%252BEnzymes%25253A%252BCase%252BStudy%252Bof%252Bthe%252BLipid%252BKinase%252BPhosphatidylinositol%252B4-Kinase%252BIII%2525CE%2525B2%252B%252528PI4KB%252529%26aulast%3DHumpolickova%26aufirst%3DJana%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D10102016%26date%3D22122016%26date%3D12012017%26date%3D08122016%26volume%3D60%26issue%3D1%26spage%3D119%26epage%3D127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seyed R.  Tabaei</span>, <span class="hlFld-ContribAuthor ">Feng  Guo</span>, <span class="hlFld-ContribAuthor ">Florentine U.  Rutaganira</span>, <span class="hlFld-ContribAuthor ">Setareh  Vafaei</span>, <span class="hlFld-ContribAuthor ">Ingrid  Choong</span>, <span class="hlFld-ContribAuthor ">Kevan M.  Shokat</span>, <span class="hlFld-ContribAuthor ">Jeffrey S.  Glenn</span>, and <span class="hlFld-ContribAuthor ">Nam-Joon  Cho</span>  . </span><span class="cited-content_cbyCitation_article-title">Multistep Compositional Remodeling of Supported Lipid Membranes by Interfacially Active Phosphatidylinositol Kinases. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2016,</strong> <em>88 </em>
                                    (10)
                                     , 5042-5045. <a href="https://doi.org/10.1021/acs.analchem.6b01293" title="DOI URL">https://doi.org/10.1021/acs.analchem.6b01293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.6b01293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.6b01293%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DMultistep%252BCompositional%252BRemodeling%252Bof%252BSupported%252BLipid%252BMembranes%252Bby%252BInterfacially%252BActive%252BPhosphatidylinositol%252BKinases%26aulast%3DTabaei%26aufirst%3DSeyed%2BR.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D03042016%26date%3D27042016%26date%3D28042016%26date%3D17052016%26date%3D27042016%26volume%3D88%26issue%3D10%26spage%3D5042%26epage%3D5045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan-ping  Li</span>, <span class="hlFld-ContribAuthor ">Reyaj  Mikrani</span>, <span class="hlFld-ContribAuthor ">Yi-fan  Hu</span>, <span class="hlFld-ContribAuthor ">Mirza Muhammad  Faran Ashraf Baig</span>, <span class="hlFld-ContribAuthor ">Muhammad  Abbas</span>, <span class="hlFld-ContribAuthor ">Fahad  Akhtar</span>, <span class="hlFld-ContribAuthor ">Ming  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of phosphatidylinositol 4-kinase and its inhibitors in inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>907 </em>, 174300. <a href="https://doi.org/10.1016/j.ejphar.2021.174300" title="DOI URL">https://doi.org/10.1016/j.ejphar.2021.174300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2021.174300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2021.174300%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DResearch%252Bprogress%252Bof%252Bphosphatidylinositol%252B4-kinase%252Band%252Bits%252Binhibitors%252Bin%252Binflammatory%252Bdiseases%26aulast%3DLi%26aufirst%3DYan-ping%26date%3D2021%26volume%3D907%26spage%3D174300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erin L.  Higgins</span>, <span class="hlFld-ContribAuthor ">Julian S.  Kellner-Rogers</span>, <span class="hlFld-ContribAuthor ">Alexandra M.  Estanislau</span>, <span class="hlFld-ContribAuthor ">Alec C.  Esposito</span>, <span class="hlFld-ContribAuthor ">Nora R.  Vail</span>, <span class="hlFld-ContribAuthor ">Sterling R.  Payne</span>, <span class="hlFld-ContribAuthor ">Julia G.  Stockwell</span>, <span class="hlFld-ContribAuthor ">Scott M.  Ulrich</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and evaluation of transition-state analogs as inhibitors of the bacterial quorum sensing autoinducer synthase CepI. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>39 </em>, 127873. <a href="https://doi.org/10.1016/j.bmcl.2021.127873" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127873%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bevaluation%252Bof%252Btransition-state%252Banalogs%252Bas%252Binhibitors%252Bof%252Bthe%252Bbacterial%252Bquorum%252Bsensing%252Bautoinducer%252Bsynthase%252BCepI%26aulast%3DHiggins%26aufirst%3DErin%2BL.%26date%3D2021%26volume%3D39%26spage%3D127873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dhanusha  Yesudhas</span>, <span class="hlFld-ContribAuthor ">Ambuj  Srivastava</span>, <span class="hlFld-ContribAuthor ">M. Michael  Gromiha</span>. </span><span class="cited-content_cbyCitation_article-title">COVID-19 outbreak: history, mechanism, transmission, structural studies and therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Infection</span><span> <strong>2021,</strong> <em>49 </em>
                                    (2)
                                     , 199-213. <a href="https://doi.org/10.1007/s15010-020-01516-2" title="DOI URL">https://doi.org/10.1007/s15010-020-01516-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s15010-020-01516-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs15010-020-01516-2%26sid%3Dliteratum%253Aachs%26jtitle%3DInfection%26atitle%3DCOVID-19%252Boutbreak%25253A%252Bhistory%25252C%252Bmechanism%25252C%252Btransmission%25252C%252Bstructural%252Bstudies%252Band%252Btherapeutics%26aulast%3DYesudhas%26aufirst%3DDhanusha%26date%3D2021%26date%3D2020%26volume%3D49%26issue%3D2%26spage%3D199%26epage%3D213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie M.  Colodette</span>, <span class="hlFld-ContribAuthor ">Lucas S.  Franco</span>, <span class="hlFld-ContribAuthor ">Rodolfo C.  Maia</span>, <span class="hlFld-ContribAuthor ">Harold H.  Fokoue</span>, <span class="hlFld-ContribAuthor ">Carlos Mauricio R.  Sant’Anna</span>, <span class="hlFld-ContribAuthor ">Eliezer J.  Barreiro</span>. </span><span class="cited-content_cbyCitation_article-title">Novel phosphatidylinositol 4-kinases III beta (PI4KIIIβ) inhibitors discovered by virtual screening using free energy models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (10)
                                     , 1091-1103. <a href="https://doi.org/10.1007/s10822-020-00327-9" title="DOI URL">https://doi.org/10.1007/s10822-020-00327-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00327-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00327-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DNovel%252Bphosphatidylinositol%252B4-kinases%252BIII%252Bbeta%252B%252528PI4KIII%2525CE%2525B2%252529%252Binhibitors%252Bdiscovered%252Bby%252Bvirtual%252Bscreening%252Busing%252Bfree%252Benergy%252Bmodels%26aulast%3DColodette%26aufirst%3DNatalie%2BM.%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D10%26spage%3D1091%26epage%3D1103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinhong  Li</span>, <span class="hlFld-ContribAuthor ">Mingshu  Wang</span>, <span class="hlFld-ContribAuthor ">Anchun  Cheng</span>, <span class="hlFld-ContribAuthor ">Xingjian  Wen</span>, <span class="hlFld-ContribAuthor ">Xumin  Ou</span>, <span class="hlFld-ContribAuthor ">Sai  Mao</span>, <span class="hlFld-ContribAuthor ">Qun  Gao</span>, <span class="hlFld-ContribAuthor ">Di  Sun</span>, <span class="hlFld-ContribAuthor ">Renyong  Jia</span>, <span class="hlFld-ContribAuthor ">Qiao  Yang</span>, <span class="hlFld-ContribAuthor ">Ying  Wu</span>, <span class="hlFld-ContribAuthor ">Dekang  Zhu</span>, <span class="hlFld-ContribAuthor ">Xinxin  Zhao</span>, <span class="hlFld-ContribAuthor ">Shun  Chen</span>, <span class="hlFld-ContribAuthor ">Mafeng  Liu</span>, <span class="hlFld-ContribAuthor ">Shaqiu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yunya  Liu</span>, <span class="hlFld-ContribAuthor ">Yanling  Yu</span>, <span class="hlFld-ContribAuthor ">Ling  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Tian</span>, <span class="hlFld-ContribAuthor ">Leichang  Pan</span>, <span class="hlFld-ContribAuthor ">Xiaoyue  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Enterovirus Replication Organelles and Inhibitors of Their Formation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Microbiology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fmicb.2020.01817" title="DOI URL">https://doi.org/10.3389/fmicb.2020.01817</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmicb.2020.01817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmicb.2020.01817%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Microbiology%26atitle%3DEnterovirus%252BReplication%252BOrganelles%252Band%252BInhibitors%252Bof%252BTheir%252BFormation%26aulast%3DLi%26aufirst%3DXinhong%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaochao  Tan</span>, <span class="hlFld-ContribAuthor ">Priyam  Banerjee</span>, <span class="hlFld-ContribAuthor ">Edward A.  Pham</span>, <span class="hlFld-ContribAuthor ">Florentine U. N.  Rutaganira</span>, <span class="hlFld-ContribAuthor ">Kaustabh  Basu</span>, <span class="hlFld-ContribAuthor ">Neus  Bota-Rabassedas</span>, <span class="hlFld-ContribAuthor ">Hou-Fu  Guo</span>, <span class="hlFld-ContribAuthor ">Caitlin L.  Grzeskowiak</span>, <span class="hlFld-ContribAuthor ">Xin  Liu</span>, <span class="hlFld-ContribAuthor ">Jiang  Yu</span>, <span class="hlFld-ContribAuthor ">Lei  Shi</span>, <span class="hlFld-ContribAuthor ">David H.  Peng</span>, <span class="hlFld-ContribAuthor ">B. Leticia  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Jiaqi  Zhang</span>, <span class="hlFld-ContribAuthor ">Veronica  Zheng</span>, <span class="hlFld-ContribAuthor ">Dzifa Y.  Duose</span>, <span class="hlFld-ContribAuthor ">Luisa M.  Solis</span>, <span class="hlFld-ContribAuthor ">Barbara  Mino</span>, <span class="hlFld-ContribAuthor ">Maria Gabriela  Raso</span>, <span class="hlFld-ContribAuthor ">Carmen  Behrens</span>, <span class="hlFld-ContribAuthor ">Ignacio I.  Wistuba</span>, <span class="hlFld-ContribAuthor ">Kenneth L.  Scott</span>, <span class="hlFld-ContribAuthor ">Mark  Smith</span>, <span class="hlFld-ContribAuthor ">Khanh  Nguyen</span>, <span class="hlFld-ContribAuthor ">Grace  Lam</span>, <span class="hlFld-ContribAuthor ">Ingrid  Choong</span>, <span class="hlFld-ContribAuthor ">Abhijit  Mazumdar</span>, <span class="hlFld-ContribAuthor ">Jamal L.  Hill</span>, <span class="hlFld-ContribAuthor ">Don L.  Gibbons</span>, <span class="hlFld-ContribAuthor ">Powel H.  Brown</span>, <span class="hlFld-ContribAuthor ">William K.  Russell</span>, <span class="hlFld-ContribAuthor ">Kevan  Shokat</span>, <span class="hlFld-ContribAuthor ">Chad J.  Creighton</span>, <span class="hlFld-ContribAuthor ">Jeffrey S.  Glenn</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Kurie</span>. </span><span class="cited-content_cbyCitation_article-title">PI4KIIIβ is a therapeutic target in chromosome 1q–amplified lung adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">Science Translational Medicine</span><span> <strong>2020,</strong> <em>12 </em>
                                    (527)
                                     , eaax3772. <a href="https://doi.org/10.1126/scitranslmed.aax3772" title="DOI URL">https://doi.org/10.1126/scitranslmed.aax3772</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scitranslmed.aax3772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscitranslmed.aax3772%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Translational%2520Medicine%26atitle%3DPI4KIII%2525CE%2525B2%252Bis%252Ba%252Btherapeutic%252Btarget%252Bin%252Bchromosome%252B1q%2525E2%252580%252593amplified%252Blung%252Badenocarcinoma%26aulast%3DTan%26aufirst%3DXiaochao%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D527%26spage%3Deaax3772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacob A.  McPhail</span>, <span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging the Phosphoinositide 3-Kinase (PI3K) and Phosphatidylinositol 4-Kinase (PI4K) Family of Enzymes for Treatment of Cancer, Immune Disorders, and Viral/Parasitic Infections. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 203-222. <a href="https://doi.org/10.1007/978-3-030-50621-6_9" title="DOI URL">https://doi.org/10.1007/978-3-030-50621-6_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-50621-6_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-50621-6_9%26sid%3Dliteratum%253Aachs%26atitle%3DDrugging%252Bthe%252BPhosphoinositide%252B3-Kinase%252B%252528PI3K%252529%252Band%252BPhosphatidylinositol%252B4-Kinase%252B%252528PI4K%252529%252BFamily%252Bof%252BEnzymes%252Bfor%252BTreatment%252Bof%252BCancer%25252C%252BImmune%252BDisorders%25252C%252Band%252BViral%25252FParasitic%252BInfections%26aulast%3DMcPhail%26aufirst%3DJacob%2BA.%26date%3D2020%26date%3D2020%26spage%3D203%26epage%3D222%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDruggable%252BLipid%252BSignaling%252BPathways%26aulast%3DKihara%26aufirst%3DYasuyuki%26date%3D2020%26volume%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominika  Chalupska</span>, <span class="hlFld-ContribAuthor ">Bartosz  Różycki</span>, <span class="hlFld-ContribAuthor ">Martin  Klima</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights into Acyl‐coenzyme A binding domain containing 3 (ACBD3) protein hijacking by picornaviruses. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2019,</strong> <em>28 </em>
                                    (12)
                                     , 2073-2079. <a href="https://doi.org/10.1002/pro.3738" title="DOI URL">https://doi.org/10.1002/pro.3738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3738%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DStructural%252Binsights%252Binto%252BAcyl%2525E2%252580%252590coenzyme%252BA%252Bbinding%252Bdomain%252Bcontaining%252B3%252B%252528ACBD3%252529%252Bprotein%252Bhijacking%252Bby%252Bpicornaviruses%26aulast%3DChalupska%26aufirst%3DDominika%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D12%26spage%3D2073%26epage%3D2079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominika  Chalupska</span>, <span class="hlFld-ContribAuthor ">Bartosz  Różycki</span>, <span class="hlFld-ContribAuthor ">Jana  Humpolickova</span>, <span class="hlFld-ContribAuthor ">Lenka  Faltova</span>, <span class="hlFld-ContribAuthor ">Martin  Klima</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphatidylinositol 4-kinase IIIβ (PI4KB) forms highly flexible heterocomplexes that include ACBD3, 14-3-3, and Rab11 proteins. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-37158-6" title="DOI URL">https://doi.org/10.1038/s41598-018-37158-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-37158-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-37158-6%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DPhosphatidylinositol%252B4-kinase%252BIII%2525CE%2525B2%252B%252528PI4KB%252529%252Bforms%252Bhighly%252Bflexible%252Bheterocomplexes%252Bthat%252Binclude%252BACBD3%25252C%252B14-3-3%25252C%252Band%252BRab11%252Bproteins%26aulast%3DChalupska%26aufirst%3DDominika%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Varun  Chahal</span>, <span class="hlFld-ContribAuthor ">Sonam  Nirwan</span>, <span class="hlFld-ContribAuthor ">Rita  Kakkar</span>. </span><span class="cited-content_cbyCitation_article-title">Combined approach of homology modeling, molecular dynamics, and docking: computer-aided drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Physical Sciences Reviews</span><span> <strong>2019,</strong> <em>4 </em>
                                    (10)
                                     <a href="https://doi.org/10.1515/psr-2019-0066" title="DOI URL">https://doi.org/10.1515/psr-2019-0066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1515/psr-2019-0066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1515%2Fpsr-2019-0066%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Sciences%2520Reviews%26atitle%3DCombined%252Bapproach%252Bof%252Bhomology%252Bmodeling%25252C%252Bmolecular%252Bdynamics%25252C%252Band%252Bdocking%25253A%252Bcomputer-aided%252Bdrug%252Bdiscovery%26aulast%3DChahal%26aufirst%3DVarun%26date%3D2019%26volume%3D4%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuaizhen  Tian</span>, <span class="hlFld-ContribAuthor ">Jinzhe  Zeng</span>, <span class="hlFld-ContribAuthor ">Xiao  Liu</span>, <span class="hlFld-ContribAuthor ">Jianzhong  Chen</span>, <span class="hlFld-ContribAuthor ">John Z. H.  Zhang</span>, <span class="hlFld-ContribAuthor ">Tong  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding the selectivity of inhibitors toward PI4KIIIα and PI4KIIIβ based molecular modeling. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2019,</strong> <em>21 </em>
                                    (39)
                                     , 22103-22112. <a href="https://doi.org/10.1039/C9CP03598B" title="DOI URL">https://doi.org/10.1039/C9CP03598B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CP03598B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CP03598B%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DUnderstanding%252Bthe%252Bselectivity%252Bof%252Binhibitors%252Btoward%252BPI4KIII%2525CE%2525B1%252Band%252BPI4KIII%2525CE%2525B2%252Bbased%252Bmolecular%252Bmodeling%26aulast%3DTian%26aufirst%3DShuaizhen%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D39%26spage%3D22103%26epage%3D22112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark G.  Waugh</span>. </span><span class="cited-content_cbyCitation_article-title">The Great Escape: how phosphatidylinositol 4-kinases and PI4P promote vesicle exit from the Golgi (and drive cancer). </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2019,</strong> <em>476 </em>
                                    (16)
                                     , 2321-2346. <a href="https://doi.org/10.1042/BCJ20180622" title="DOI URL">https://doi.org/10.1042/BCJ20180622</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20180622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20180622%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DThe%252BGreat%252BEscape%25253A%252Bhow%252Bphosphatidylinositol%252B4-kinases%252Band%252BPI4P%252Bpromote%252Bvesicle%252Bexit%252Bfrom%252Bthe%252BGolgi%252B%252528and%252Bdrive%252Bcancer%252529%26aulast%3DWaugh%26aufirst%3DMark%2BG.%26date%3D2019%26date%3D2019%26volume%3D476%26issue%3D16%26spage%3D2321%26epage%3D2346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dongxue  Mao</span>, <span class="hlFld-ContribAuthor ">Guang  Lin</span>, <span class="hlFld-ContribAuthor ">Burak  Tepe</span>, <span class="hlFld-ContribAuthor ">Zhongyuan  Zuo</span>, <span class="hlFld-ContribAuthor ">Kai Li  Tan</span>, <span class="hlFld-ContribAuthor ">Mumine  Senturk</span>, <span class="hlFld-ContribAuthor ">Sheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Benjamin R  Arenkiel</span>, <span class="hlFld-ContribAuthor ">Marco  Sardiello</span>, <span class="hlFld-ContribAuthor ">Hugo J.  Bellen</span>. </span><span class="cited-content_cbyCitation_article-title">VAMP associated proteins are required for autophagic and lysosomal degradation by promoting a PtdIns4P-mediated endosomal pathway. </span><span class="cited-content_cbyCitation_journal-name">Autophagy</span><span> <strong>2019,</strong> <em>15 </em>
                                    (7)
                                     , 1214-1233. <a href="https://doi.org/10.1080/15548627.2019.1580103" title="DOI URL">https://doi.org/10.1080/15548627.2019.1580103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15548627.2019.1580103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15548627.2019.1580103%26sid%3Dliteratum%253Aachs%26jtitle%3DAutophagy%26atitle%3DVAMP%252Bassociated%252Bproteins%252Bare%252Brequired%252Bfor%252Bautophagic%252Band%252Blysosomal%252Bdegradation%252Bby%252Bpromoting%252Ba%252BPtdIns4P-mediated%252Bendosomal%252Bpathway%26aulast%3DMao%26aufirst%3DDongxue%26date%3D2019%26date%3D2019%26volume%3D15%26issue%3D7%26spage%3D1214%26epage%3D1233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahmoud A.A.  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Alaa H.M.  Abdelrahman</span>, <span class="hlFld-ContribAuthor ">Alaa M.A.  Hassan</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel Plasmodium falciparum PI4KB inhibitors as potential anti-malarial drugs: Homology modeling, molecular docking and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2019,</strong> <em>80 </em>, 79-89. <a href="https://doi.org/10.1016/j.compbiolchem.2019.03.010" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2019.03.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2019.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2019.03.010%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252BPlasmodium%252Bfalciparum%252BPI4KB%252Binhibitors%252Bas%252Bpotential%252Banti-malarial%252Bdrugs%25253A%252BHomology%252Bmodeling%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DIbrahim%26aufirst%3DMahmoud%2BA.A.%26date%3D2019%26volume%3D80%26spage%3D79%26epage%3D89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Hernández-Ortega</span>, <span class="hlFld-ContribAuthor ">Abril  Sánchez-Botet</span>, <span class="hlFld-ContribAuthor ">Eva  Quandt</span>, <span class="hlFld-ContribAuthor ">Núria  Masip</span>, <span class="hlFld-ContribAuthor ">Laura  Gasa</span>, <span class="hlFld-ContribAuthor ">Gaetano  Verde</span>, <span class="hlFld-ContribAuthor ">Javier  Jiménez</span>, <span class="hlFld-ContribAuthor ">Rebecca S.  Levin</span>, <span class="hlFld-ContribAuthor ">Florentine U.  Rutaganira</span>, <span class="hlFld-ContribAuthor ">Alma L.  Burlingame</span>, <span class="hlFld-ContribAuthor ">Don  Wolfgeher</span>, <span class="hlFld-ContribAuthor ">Mariana P. C.  Ribeiro</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Kron</span>, <span class="hlFld-ContribAuthor ">Kevan M.  Shokat</span>, <span class="hlFld-ContribAuthor ">Josep  Clotet</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphoregulation of the oncogenic protein regulator of cytokinesis 1 (PRC1) by the atypical CDK16/CCNY complex. </span><span class="cited-content_cbyCitation_journal-name">Experimental & Molecular Medicine</span><span> <strong>2019,</strong> <em>51 </em>
                                    (4)
                                     , 1-17. <a href="https://doi.org/10.1038/s12276-019-0242-2" title="DOI URL">https://doi.org/10.1038/s12276-019-0242-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s12276-019-0242-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs12276-019-0242-2%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520%2526%2520Molecular%2520Medicine%26atitle%3DPhosphoregulation%252Bof%252Bthe%252Boncogenic%252Bprotein%252Bregulator%252Bof%252Bcytokinesis%252B1%252B%252528PRC1%252529%252Bby%252Bthe%252Batypical%252BCDK16%25252FCCNY%252Bcomplex%26aulast%3DHern%25C3%25A1ndez-Ortega%26aufirst%3DSara%26date%3D2019%26date%3D2019%26volume%3D51%26issue%3D4%26spage%3D1%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paola  Genevini</span>, <span class="hlFld-ContribAuthor ">Maria Nicol  Colombo</span>, <span class="hlFld-ContribAuthor ">Rossella  Venditti</span>, <span class="hlFld-ContribAuthor ">Stefania  Marcuzzo</span>, <span class="hlFld-ContribAuthor ">Sara Francesca  Colombo</span>, <span class="hlFld-ContribAuthor ">Pia  Bernasconi</span>, <span class="hlFld-ContribAuthor ">Maria Antonietta  De Matteis</span>, <span class="hlFld-ContribAuthor ">Nica  Borgese</span>, <span class="hlFld-ContribAuthor ">Francesca  Navone</span>. </span><span class="cited-content_cbyCitation_article-title">VAPB depletion alters neuritogenesis and phosphoinositide balance in motoneuron-like cells: relevance to VAPB-linked amyotrophic lateral sclerosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cell Science</span><span> <strong>2019,</strong> <em>132 </em>
                                    (7)
                                     <a href="https://doi.org/10.1242/jcs.220061" title="DOI URL">https://doi.org/10.1242/jcs.220061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1242/jcs.220061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1242%2Fjcs.220061%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cell%2520Science%26atitle%3DVAPB%252Bdepletion%252Balters%252Bneuritogenesis%252Band%252Bphosphoinositide%252Bbalance%252Bin%252Bmotoneuron-like%252Bcells%25253A%252Brelevance%252Bto%252BVAPB-linked%252Bamyotrophic%252Blateral%252Bsclerosis%26aulast%3DGenevini%26aufirst%3DPaola%26date%3D2019%26date%3D2019%26volume%3D132%26issue%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alberto  Fernández-Oliva</span>, <span class="hlFld-ContribAuthor ">Paula  Ortega-González</span>, <span class="hlFld-ContribAuthor ">Cristina  Risco</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting host lipid flows: Exploring new antiviral and antibiotic strategies. </span><span class="cited-content_cbyCitation_journal-name">Cellular Microbiology</span><span> <strong>2019,</strong> <em>21 </em>
                                    (3)
                                     , e12996. <a href="https://doi.org/10.1111/cmi.12996" title="DOI URL">https://doi.org/10.1111/cmi.12996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cmi.12996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcmi.12996%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Microbiology%26atitle%3DTargeting%252Bhost%252Blipid%252Bflows%25253A%252BExploring%252Bnew%252Bantiviral%252Band%252Bantibiotic%252Bstrategies%26aulast%3DFern%25C3%25A1ndez-Oliva%26aufirst%3DAlberto%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D3%26spage%3De12996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">Dynamic structural biology at the protein membrane interface. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (11)
                                     , 3872-3880. <a href="https://doi.org/10.1074/jbc.AW118.003236" title="DOI URL">https://doi.org/10.1074/jbc.AW118.003236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.AW118.003236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.AW118.003236%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DDynamic%252Bstructural%252Bbiology%252Bat%252Bthe%252Bprotein%252Bmembrane%252Binterface%26aulast%3DBurke%26aufirst%3DJohn%2BE.%26date%3D2019%26volume%3D294%26issue%3D11%26spage%3D3872%26epage%3D3880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gillian L.  Dornan</span>, <span class="hlFld-ContribAuthor ">Udit  Dalwadi</span>, <span class="hlFld-ContribAuthor ">David J.  Hamelin</span>, <span class="hlFld-ContribAuthor ">Reece M.  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Calvin K.  Yip</span>, <span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Architecture, Dynamics, and Inhibition of the PI4KIIIα/TTC7/FAM126 Complex. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2018,</strong> <em>430 </em>
                                    (18)
                                     , 3129-3142. <a href="https://doi.org/10.1016/j.jmb.2018.07.020" title="DOI URL">https://doi.org/10.1016/j.jmb.2018.07.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2018.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2018.07.020%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DProbing%252Bthe%252BArchitecture%25252C%252BDynamics%25252C%252Band%252BInhibition%252Bof%252Bthe%252BPI4KIII%2525CE%2525B1%25252FTTC7%25252FFAM126%252BComplex%26aulast%3DDornan%26aufirst%3DGillian%2BL.%26date%3D2018%26volume%3D430%26issue%3D18%26spage%3D3129%26epage%3D3142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Basis for Regulation of Phosphoinositide Kinases and Their Involvement in Human Disease. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2018,</strong> <em>71 </em>
                                    (5)
                                     , 653-673. <a href="https://doi.org/10.1016/j.molcel.2018.08.005" title="DOI URL">https://doi.org/10.1016/j.molcel.2018.08.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2018.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2018.08.005%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DStructural%252BBasis%252Bfor%252BRegulation%252Bof%252BPhosphoinositide%252BKinases%252Band%252BTheir%252BInvolvement%252Bin%252BHuman%252BDisease%26aulast%3DBurke%26aufirst%3DJohn%2BE.%26date%3D2018%26volume%3D71%26issue%3D5%26spage%3D653%26epage%3D673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaofei  Liang</span>, <span class="hlFld-ContribAuthor ">Feng  Li</span>, <span class="hlFld-ContribAuthor ">Cheng  Chen</span>, <span class="hlFld-ContribAuthor ">Zongru  Jiang</span>, <span class="hlFld-ContribAuthor ">Aoli  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaochuan  Liu</span>, <span class="hlFld-ContribAuthor ">Juan  Ge</span>, <span class="hlFld-ContribAuthor ">Zhenquan  Hu</span>, <span class="hlFld-ContribAuthor ">Kailin  Yu</span>, <span class="hlFld-ContribAuthor ">Wenliang  Wang</span>, <span class="hlFld-ContribAuthor ">Fengming  Zou</span>, <span class="hlFld-ContribAuthor ">Qingwang  Liu</span>, <span class="hlFld-ContribAuthor ">Beilei  Wang</span>, <span class="hlFld-ContribAuthor ">Li  Wang</span>, <span class="hlFld-ContribAuthor ">Shanchun  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuxin  Wang</span>, <span class="hlFld-ContribAuthor ">Qingsong  Liu</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of (S)-2-amino-N-(5-(6-chloro-5-(3-methylphenylsulfonamido)pyridin-3-yl)-4-methylthiazol-2-yl)-3-methylbutanamide (CHMFL-PI3KD-317) as a potent and selective phosphoinositide 3-kinase delta (PI3Kδ) inhibitor. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>156 </em>, 831-846. <a href="https://doi.org/10.1016/j.ejmech.2018.07.036" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.07.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.07.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.07.036%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%252528S%252529-2-amino-N-%2525285-%2525286-chloro-5-%2525283-methylphenylsulfonamido%252529pyridin-3-yl%252529-4-methylthiazol-2-yl%252529-3-methylbutanamide%252B%252528CHMFL-PI3KD-317%252529%252Bas%252Ba%252Bpotent%252Band%252Bselective%252Bphosphoinositide%252B3-kinase%252Bdelta%252B%252528PI3K%2525CE%2525B4%252529%252Binhibitor%26aulast%3DLiang%26aufirst%3DXiaofei%26date%3D2018%26volume%3D156%26spage%3D831%26epage%3D846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Zappa</span>, <span class="hlFld-ContribAuthor ">Mario  Failli</span>, <span class="hlFld-ContribAuthor ">Maria Antonietta  De Matteis</span>. </span><span class="cited-content_cbyCitation_article-title">The Golgi complex in disease and therapy. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Cell Biology</span><span> <strong>2018,</strong> <em>50 </em>, 102-116. <a href="https://doi.org/10.1016/j.ceb.2018.03.005" title="DOI URL">https://doi.org/10.1016/j.ceb.2018.03.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ceb.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ceb.2018.03.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Cell%2520Biology%26atitle%3DThe%252BGolgi%252Bcomplex%252Bin%252Bdisease%252Band%252Btherapy%26aulast%3DZappa%26aufirst%3DFrancesca%26date%3D2018%26volume%3D50%26spage%3D102%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Dubankova</span>, <span class="hlFld-ContribAuthor ">Jana  Humpolickova</span>, <span class="hlFld-ContribAuthor ">Martin  Klima</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>. </span><span class="cited-content_cbyCitation_article-title">Negative charge and membrane-tethered viral 3B cooperate to recruit viral RNA dependent RNA polymerase 3D pol. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-17621-6" title="DOI URL">https://doi.org/10.1038/s41598-017-17621-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-17621-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-17621-6%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DNegative%252Bcharge%252Band%252Bmembrane-tethered%252Bviral%252B3B%252Bcooperate%252Bto%252Brecruit%252Bviral%252BRNA%252Bdependent%252BRNA%252Bpolymerase%252B3D%252Bpol%26aulast%3DDubankova%26aufirst%3DAnna%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Glenn R.  Masson</span>, <span class="hlFld-ContribAuthor ">Meredith L.  Jenkins</span>, <span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">An overview of hydrogen deuterium exchange mass spectrometry (HDX-MS) in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2017,</strong> <em>12 </em>
                                    (10)
                                     , 981-994. <a href="https://doi.org/10.1080/17460441.2017.1363734" title="DOI URL">https://doi.org/10.1080/17460441.2017.1363734</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2017.1363734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2017.1363734%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Boverview%252Bof%252Bhydrogen%252Bdeuterium%252Bexchange%252Bmass%252Bspectrometry%252B%252528HDX-MS%252529%252Bin%252Bdrug%252Bdiscovery%26aulast%3DMasson%26aufirst%3DGlenn%2BR.%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D10%26spage%3D981%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominika  Chalupska</span>, <span class="hlFld-ContribAuthor ">Andrea  Eisenreichova</span>, <span class="hlFld-ContribAuthor ">Bartosz  Różycki</span>, <span class="hlFld-ContribAuthor ">Lenka  Rezabkova</span>, <span class="hlFld-ContribAuthor ">Jana  Humpolickova</span>, <span class="hlFld-ContribAuthor ">Martin  Klima</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>. </span><span class="cited-content_cbyCitation_article-title">Structural analysis of phosphatidylinositol 4-kinase IIIβ (PI4KB) – 14-3-3 protein complex reveals internal flexibility and explains 14-3-3 mediated protection from degradation in vitro. </span><span class="cited-content_cbyCitation_journal-name">Journal of Structural Biology</span><span> <strong>2017,</strong> <em>200 </em>
                                    (1)
                                     , 36-44. <a href="https://doi.org/10.1016/j.jsb.2017.08.006" title="DOI URL">https://doi.org/10.1016/j.jsb.2017.08.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsb.2017.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsb.2017.08.006%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Structural%2520Biology%26atitle%3DStructural%252Banalysis%252Bof%252Bphosphatidylinositol%252B4-kinase%252BIII%2525CE%2525B2%252B%252528PI4KB%252529%252B%2525E2%252580%252593%252B14-3-3%252Bprotein%252Bcomplex%252Breveals%252Binternal%252Bflexibility%252Band%252Bexplains%252B14-3-3%252Bmediated%252Bprotection%252Bfrom%252Bdegradation%252Bin%252Bvitro%26aulast%3DChalupska%26aufirst%3DDominika%26date%3D2017%26volume%3D200%26issue%3D1%26spage%3D36%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Grace Q.  Gong</span>, <span class="hlFld-ContribAuthor ">Jackie D.  Kendall</span>, <span class="hlFld-ContribAuthor ">James M.J.  Dickson</span>, <span class="hlFld-ContribAuthor ">Gordon W.  Rewcastle</span>, <span class="hlFld-ContribAuthor ">Christina M.  Buchanan</span>, <span class="hlFld-ContribAuthor ">William A.  Denny</span>, <span class="hlFld-ContribAuthor ">Peter R.  Shepherd</span>, <span class="hlFld-ContribAuthor ">Jack U.  Flanagan</span>. </span><span class="cited-content_cbyCitation_article-title">Combining properties of different classes of PI3Kα inhibitors to understand the molecular features that confer selectivity. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2017,</strong> <em>474 </em>
                                    (13)
                                     , 2261-2276. <a href="https://doi.org/10.1042/BCJ20161098" title="DOI URL">https://doi.org/10.1042/BCJ20161098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20161098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20161098%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DCombining%252Bproperties%252Bof%252Bdifferent%252Bclasses%252Bof%252BPI3K%2525CE%2525B1%252Binhibitors%252Bto%252Bunderstand%252Bthe%252Bmolecular%252Bfeatures%252Bthat%252Bconfer%252Bselectivity%26aulast%3DGong%26aufirst%3DGrace%2BQ.%26date%3D2017%26date%3D2017%26volume%3D474%26issue%3D13%26spage%3D2261%26epage%3D2276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nihal  Altan-Bonnet</span>. </span><span class="cited-content_cbyCitation_article-title">Lipid Tales of Viral Replication and Transmission. </span><span class="cited-content_cbyCitation_journal-name">Trends in Cell Biology</span><span> <strong>2017,</strong> <em>27 </em>
                                    (3)
                                     , 201-213. <a href="https://doi.org/10.1016/j.tcb.2016.09.011" title="DOI URL">https://doi.org/10.1016/j.tcb.2016.09.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tcb.2016.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tcb.2016.09.011%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Cell%2520Biology%26atitle%3DLipid%252BTales%252Bof%252BViral%252BReplication%252Band%252BTransmission%26aulast%3DAltan-Bonnet%26aufirst%3DNihal%26date%3D2017%26volume%3D27%26issue%3D3%26spage%3D201%26epage%3D213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacob A.  McPhail</span>, <span class="hlFld-ContribAuthor ">Erik H.  Ottosen</span>, <span class="hlFld-ContribAuthor ">Meredith L.  Jenkins</span>, <span class="hlFld-ContribAuthor ">John E.  Burke</span>. </span><span class="cited-content_cbyCitation_article-title">The Molecular Basis of Aichi Virus 3A Protein Activation of Phosphatidylinositol 4 Kinase IIIβ, PI4KB, through ACBD3. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2017,</strong> <em>25 </em>
                                    (1)
                                     , 121-131. <a href="https://doi.org/10.1016/j.str.2016.11.016" title="DOI URL">https://doi.org/10.1016/j.str.2016.11.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2016.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2016.11.016%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DThe%252BMolecular%252BBasis%252Bof%252BAichi%252BVirus%252B3A%252BProtein%252BActivation%252Bof%252BPhosphatidylinositol%252B4%252BKinase%252BIII%2525CE%2525B2%25252C%252BPI4KB%25252C%252Bthrough%252BACBD3%26aulast%3DMcPhail%26aufirst%3DJacob%2BA.%26date%3D2017%26volume%3D25%26issue%3D1%26spage%3D121%26epage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni-Anna  Loundras</span>, <span class="hlFld-ContribAuthor ">Morgan R.  Herod</span>, <span class="hlFld-ContribAuthor ">Mark  Harris</span>, <span class="hlFld-ContribAuthor ">Nicola J.  Stonehouse</span>. </span><span class="cited-content_cbyCitation_article-title">Foot-and-mouth disease virus genome replication is unaffected by inhibition of type III phosphatidylinositol-4-kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of General Virology</span><span> <strong>2016,</strong> <em>97 </em>
                                    (9)
                                     , 2221-2230. <a href="https://doi.org/10.1099/jgv.0.000527" title="DOI URL">https://doi.org/10.1099/jgv.0.000527</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1099/jgv.0.000527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1099%2Fjgv.0.000527%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520General%2520Virology%26atitle%3DFoot-and-mouth%252Bdisease%252Bvirus%252Bgenome%252Breplication%252Bis%252Bunaffected%252Bby%252Binhibition%252Bof%252Btype%252BIII%252Bphosphatidylinositol-4-kinases%26aulast%3DLoundras%26aufirst%3DEleni-Anna%26date%3D2016%26volume%3D97%26issue%3D9%26spage%3D2221%26epage%3D2230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structural basis for inhibition of PI4KIIIβ and PI3Ks by the inhibitor PIK93 (compound <b>1</b>). (A) Structure of compound <b>1</b>, with the ethanolamine substituent off of the sulfonamide colored blue, the chloro substituent off of the central phenyl colored green, and the acetamide substituent off of the thiazol colored red. The potency of <b>1</b> against PI4KIIIβ, PI3Kγ, and vps34 is graphed. (B) Structures of PI4KIIIβ<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D0L">4D0L</a>), vps34<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J">2X6J</a>), and PI3Kγ<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ">2CHZ</a>) bound to <b>1</b> aligned, showing the chloro substituent of <b>1</b> with the activation loop of each enzyme colored according to the legend. (C–E) Structures of PI4KIIIβ (C), PI3Kγ (D), and vps34 (E) with residues within 5 Å of the acetamide group of <b>1</b> shown as spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Development of novel PI4KIIIβ inhibitors. (A) Structure of compound <b>1</b> is shown, with novel substituents at the R1, R2, and R3 positions shown in tabular form and with the IC<sub>50</sub> values for each compound against PI4KIIIβ, PI3Kγ, PI3Kδ, and vps34 listed. Kinase assays were carried out in the presence of 10 μM ATP and 10 mM MgCl<sub>2</sub> (PI4KIIIβ, PI3Kδ, and PI3Kγ) or 10 mM MnCl<sub>2</sub> (vps34). IC<sub>50</sub> values measured using MnCl<sub>2</sub> are 10–20-fold higher than those with the physiologically relevant MgCl<sub>2</sub> due to increased kinase activity. Inhibitors anti-HCV antiviral activity (IC<sub>50</sub>) was assessed by <i>Gaussia</i> luciferase assay (Gluc HCV2a activity), and cell viability (CC<sub>50</sub>) was assessed by Presto Blue assay (Gluc HCV2a viability), as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. (B) Full inhibitor curves for compound <b>8</b>–<b>10</b> are shown for PI4KIIIβ, PI3Kγ, PI3Kδ, and vps34, with all measurements carried out in triplicate. For compounds <b>9</b> and <b>10</b>, the curves for the related lipid kinase PI4KIIIα is shown as well, with assays against other related lipid kinases shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf" class="ext-link">Figure S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis of inhibition of PI4KIIIβ by compound <b>9</b>. (A) Residues mediating the interaction of PI4KIIIβ with compound <b>9</b> are shown, with putative hydrogen bonds indicated by dotted lines. Figure generated using LIGPLOT.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (B) Fit of compound <b>9</b> in the PI4KIIIβ active site pocket. The kinase domain is colored with the N-lobe shown in red and the C-lobe shown in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural basis for selectivity of compound <b>9</b>. (A) Structure of compound <b>9</b> bound to PI4KIIIβ, with selected residues colored in orange and shown as spheres. (B) Model of compound <b>9</b> in the active site of PI3Kγ. The model was generated by aligning the active site of the structure of compound <b>9</b> bound to PI4KIIIβ to the structure of PI3Kγ bound to compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ">2CHZ</a>). The equivalent residues shown in panel A are colored in green and shown as spheres, with steric clashes highlighted in red. (C) Model of compound <b>9</b> in the active site of vps34. The model was generated by aligning the active site of the structure of compound <b>9</b> bound to PI4KIIIβ to the structure of vps34 bound to compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J">2X6J</a>). The equivalent residues shown in panel A are colored in cyan and shown as spheres, with steric clashes highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/medium/jm-2015-013112_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>1</b> and Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01311/20160310/images/large/jm-2015-013112_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01311&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76657" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76657" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 43 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphoinositides: Tiny Lipids with Giant Impact on Cell Regulation</span> <span class="citation_source-journal">Physiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">1019</span><span class="NLM_x">–</span> <span class="NLM_lpage">1137</span><span class="refDoi"> DOI: 10.1152/physrev.00028.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1152%2Fphysrev.00028.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23899561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVekurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2013&pages=1019-1137&author=T.+Balla&title=Phosphoinositides%3A+Tiny+Lipids+with+Giant+Impact+on+Cell+Regulation&doi=10.1152%2Fphysrev.00028.2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoinositides: tiny lipids with giant impact on cell regulation</span></div><div class="casAuthors">Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1019-1137</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Phosphoinositides [PIs] make up only a small fraction of cellular phospholipids, yet they control almost all aspects of a cell's life and death.  These lipids gained tremendous research interest as plasma membrane signaling mols. when discovered in the 1970s and 1980s.  Research in the last 15 years has added a wide range of biol. processes regulated by PIs, turning these lipids into one of the most universal signaling entities in eukaryotic cells.  PIs control organelle biol. by regulating vesicular trafficking, but they also modulate lipid distribution and metab. via their close relationship with lipid transfer proteins.  PIs regulate ion channels, pumps, and transporters and control both endocytic and exocytic processes.  The nuclear phosphoinositides have grown from being an epiphenomenon to a research area of its own.  As expected from such pleiotropic regulators, derangements of phosphoinositide metab. are responsible for a no. of human diseases ranging from rare genetic disorders to the most common ones such as cancer, obesity, and diabetes.  Moreover, it is increasingly evident that a no. of infectious agents hijack the PI regulatory systems of host cells for their intracellular movements, replication, and assembly.  As a result, PI converting enzymes began to be noticed by pharmaceutical companies as potential therapeutic targets.  This review is an attempt to give an overview of this enormous research field focusing on major developments in diverse areas of basic science linked to cellular physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGornhXAaVsNoLVg90H21EOLACvtfcHk0lgqaCF68qAtbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVekurfJ&md5=98d2bcb9092cc2105e0947b49cf7a2a6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00028.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00028.2012%26sid%3Dliteratum%253Aachs%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphoinositides%253A%2520Tiny%2520Lipids%2520with%2520Giant%2520Impact%2520on%2520Cell%2520Regulation%26jtitle%3DPhysiol.%2520Rev.%26date%3D2013%26volume%3D93%26spage%3D1019%26epage%3D1137%26doi%3D10.1152%2Fphysrev.00028.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Dornan, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhail, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. E.</span><span> </span><span class="NLM_article-title">Type III Phosphatidylinositol 4 Kinases: Structure, Function, Regulation, Signalling and Involvement in Disease</span> <span class="citation_source-journal">Biochem. Soc. Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1042/BST20150219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1042%2FBST20150219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=26862213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=260-266&author=G.+L.+Dornanauthor=J.+A.+McPhailauthor=J.+E.+Burke&title=Type+III+Phosphatidylinositol+4+Kinases%3A+Structure%2C+Function%2C+Regulation%2C+Signalling+and+Involvement+in+Disease&doi=10.1042%2FBST20150219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Type III phosphatidylinositol 4 kinases: structure, function, regulation, signaling and involvement in disease</span></div><div class="casAuthors">Dornan, Gillian L.; McPhail, Jacob A.; Burke, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">260-266</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Many important cellular functions are regulated by the selective recruitment of proteins to intracellular membranes mediated by specific interactions with lipid phosphoinositides.  The enzymes that generate lipid phosphoinositides therefore must be properly positioned and regulated at their correct cellular locations.  Phosphatidylinositol 4 kinases (PI4Ks) are key lipid signaling enzymes, and they generate the lipid species, phosphatidylinositol 4-phosphate (PI4P), which plays important roles in regulating physiol. processes including membrane trafficking, cytokinesis, and organelle identity.  PI4P also acts as the substrate for the generation of the signaling phosphoinositides, phosphatidylinositol 4,5-bisphosphate (PIP2) and phosphatidylinositol 3,4,5-trisphosphate (PIP3).  PI4Ks also play crit. roles in a no. of pathol. processes including mediating replication of a no. of pathogenic RNA viruses, and in the development of the parasite responsible for malaria.  Key to the regulation of PI4Ks is their regulation by a variety of both host and viral protein-binding partners.  Here, the authors review the current understanding of the structure, regulatory interactions, and role in disease of the type III PI4Ks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nMk0eWq3O7Vg90H21EOLACvtfcHk0liHCOtseYM5Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVWqsL8%253D&md5=97c4d85ac73c36a42177ea49a9a741f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1042%2FBST20150219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20150219%26sid%3Dliteratum%253Aachs%26aulast%3DDornan%26aufirst%3DG.%2BL.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26atitle%3DType%2520III%2520Phosphatidylinositol%25204%2520Kinases%253A%2520Structure%252C%2520Function%252C%2520Regulation%252C%2520Signalling%2520and%2520Involvement%2520in%2520Disease%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2016%26volume%3D44%26spage%3D260%26epage%3D266%26doi%3D10.1042%2FBST20150219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span> </span><span class="NLM_article-title">Cinderella Story: PI4P Goes From Precursor to Key Signaling Molecule</span> <span class="citation_source-journal">Crit. Rev. Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.3109/10409238.2013.853024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.3109%2F10409238.2013.853024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24219382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2014&pages=33-58&author=J.+Tanauthor=J.+A.+Brill&title=Cinderella+Story%3A+PI4P+Goes+From+Precursor+to+Key+Signaling+Molecule&doi=10.3109%2F10409238.2013.853024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cinderella story: PI4P goes from precursor to key signaling molecule</span></div><div class="casAuthors">Tan, Julie; Brill, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-58</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Phosphatidylinositol lipids are signaling mols. involved in nearly all aspects of cellular regulation.  Prodn. of phosphatidylinositol 4-phosphate (PI4P) has long been recognized as one of the first steps in generating poly-phosphatidylinositol phosphates involved in actin organization, cell migration, and signal transduction.  In addn., progress over the last decade has brought to light independent roles for PI4P in membrane trafficking and lipid homeostasis.  Here, we describe recent advances that reveal the breadth of processes regulated by PI4P, the spectrum of PI4P effectors, and the mechanisms of spatiotemporal control that coordinate crosstalk between PI4P and cellular signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOx335EArcj7Vg90H21EOLACvtfcHk0liHCOtseYM5Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCrsLw%253D&md5=0ad4d75e67adc6f2d58b241e3e99a7d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.853024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.853024%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26atitle%3DCinderella%2520Story%253A%2520PI4P%2520Goes%2520From%2520Precursor%2520to%2520Key%2520Signaling%2520Molecule%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D49%26spage%3D33%26epage%3D58%26doi%3D10.3109%2F10409238.2013.853024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Tóth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase IIIbeta Regulates the Transport of Ceramide Between the Endoplasmic Reticulum and Golgi</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">36369</span><span class="NLM_x">–</span> <span class="NLM_lpage">36377</span><span class="refDoi"> DOI: 10.1074/jbc.M604935200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1074%2Fjbc.M604935200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=36369-36377&author=B.+T%C3%B3thauthor=A.+Ballaauthor=H.+Maauthor=Z.+A.+Knightauthor=K.+M.+Shokatauthor=T.+Balla&title=Phosphatidylinositol+4-Kinase+IIIbeta+Regulates+the+Transport+of+Ceramide+Between+the+Endoplasmic+Reticulum+and+Golgi&doi=10.1074%2Fjbc.M604935200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M604935200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M604935200%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25B3th%26aufirst%3DB.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520IIIbeta%2520Regulates%2520the%2520Transport%2520of%2520Ceramide%2520Between%2520the%2520Endoplasmic%2520Reticulum%2520and%2520Golgi%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D36369%26epage%3D36377%26doi%3D10.1074%2Fjbc.M604935200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Polevoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldbach, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbach, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span> </span><span class="NLM_article-title">Dual Roles for the Drosophila PI 4-Kinase Four Wheel Drive in Localizing Rab11 During Cytokinesis</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">858</span><span class="refDoi"> DOI: 10.1083/jcb.200908107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1083%2Fjcb.200908107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2009&pages=847-858&author=G.+Polevoyauthor=H.-C.+Weiauthor=R.+Wongauthor=Z.+Szentpeteryauthor=Y.+J.+Kimauthor=P.+Goldbachauthor=S.+K.+Steinbachauthor=T.+Ballaauthor=J.+A.+Brill&title=Dual+Roles+for+the+Drosophila+PI+4-Kinase+Four+Wheel+Drive+in+Localizing+Rab11+During+Cytokinesis&doi=10.1083%2Fjcb.200908107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200908107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200908107%26sid%3Dliteratum%253Aachs%26aulast%3DPolevoy%26aufirst%3DG.%26aulast%3DWei%26aufirst%3DH.-C.%26aulast%3DWong%26aufirst%3DR.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DGoldbach%26aufirst%3DP.%26aulast%3DSteinbach%26aufirst%3DS.%2BK.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26atitle%3DDual%2520Roles%2520for%2520the%2520Drosophila%2520PI%25204-Kinase%2520Four%2520Wheel%2520Drive%2520in%2520Localizing%2520Rab11%2520During%2520Cytokinesis%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2009%26volume%3D187%26spage%3D847%26epage%3D858%26doi%3D10.1083%2Fjcb.200908107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Sridhar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aphkhazava, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shields, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuervo, A. M.</span><span> </span><span class="NLM_article-title">The Lipid Kinase PI4KIIIβ Preserves Lysosomal Identity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/emboj.2012.341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Femboj.2012.341" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=324-339&author=S.+Sridharauthor=B.+Patelauthor=D.+Aphkhazavaauthor=F.+Macianauthor=L.+Santambrogioauthor=D.+Shieldsauthor=A.+M.+Cuervo&title=The+Lipid+Kinase+PI4KIII%CE%B2+Preserves+Lysosomal+Identity&doi=10.1038%2Femboj.2012.341"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.341%26sid%3Dliteratum%253Aachs%26aulast%3DSridhar%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DAphkhazava%26aufirst%3DD.%26aulast%3DMacian%26aufirst%3DF.%26aulast%3DSantambrogio%26aufirst%3DL.%26aulast%3DShields%26aufirst%3DD.%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26atitle%3DThe%2520Lipid%2520Kinase%2520PI4KIII%25CE%25B2%2520Preserves%2520Lysosomal%2520Identity%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D32%26spage%3D324%26epage%3D339%26doi%3D10.1038%2Femboj.2012.341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Jean, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiger, A. A.</span><span> </span><span class="NLM_article-title">Coordination Between RAB GTPase and Phosphoinositide Regulation and Functions</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">470</span><span class="refDoi"> DOI: 10.1038/nrm3379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnrm3379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22722608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFykt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=463-470&author=S.+Jeanauthor=A.+A.+Kiger&title=Coordination+Between+RAB+GTPase+and+Phosphoinositide+Regulation+and+Functions&doi=10.1038%2Fnrm3379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Coordination between RAB GTPase and phosphoinositide regulation and functions</span></div><div class="casAuthors">Jean, Steve; Kiger, Amy A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">463-470</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Membrane trafficking relies on dynamic changes in membrane identities that are detd. by the regulation of distinct RAB GTPases and phosphoinositides.  RABs and phosphoinositides both act to spatiotemporally recruit effectors of membrane remodelling, including sequential RAB and phosphoinositide activities.  New ideas on coordinated regulation of specific RABs and phosphoinositides, achieved by direct phys. and functional interactions between their regulatory enzymes, are emerging as a central mechanism to ensure precision and fidelity of membrane trafficking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwz4JkFEj6iLVg90H21EOLACvtfcHk0ljJVI5-rqsOSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFykt7g%253D&md5=91e7f7bc55d0f94cf05dc5a2b4009c65</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm3379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3379%26sid%3Dliteratum%253Aachs%26aulast%3DJean%26aufirst%3DS.%26aulast%3DKiger%26aufirst%3DA.%2BA.%26atitle%3DCoordination%2520Between%2520RAB%2520GTPase%2520and%2520Phosphoinositide%2520Regulation%2520and%2520Functions%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D463%26epage%3D470%26doi%3D10.1038%2Fnrm3379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hsu, N.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilnytska, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takvorian, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Schaar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik-Basu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrenfeld, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altan-Bonnet, N.</span><span> </span><span class="NLM_article-title">Viral Reorganization of the Secretory Pathway Generates Distinct Organelles for RNA Replication</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">799</span><span class="NLM_x">–</span> <span class="NLM_lpage">811</span><span class="refDoi"> DOI: 10.1016/j.cell.2010.03.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.cell.2010.03.050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=799-811&author=N.-Y.+Hsuauthor=O.+Ilnytskaauthor=G.+Belovauthor=M.+Santianaauthor=Y.-H.+Chenauthor=P.+M.+Takvorianauthor=C.+Pauauthor=H.+van+der+Schaarauthor=N.+Kaushik-Basuauthor=T.+Ballaauthor=C.+E.+Cameronauthor=E.+Ehrenfeldauthor=F.+J.+M.+van+Kuppeveldauthor=N.+Altan-Bonnet&title=Viral+Reorganization+of+the+Secretory+Pathway+Generates+Distinct+Organelles+for+RNA+Replication&doi=10.1016%2Fj.cell.2010.03.050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.03.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.03.050%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DN.-Y.%26aulast%3DIlnytska%26aufirst%3DO.%26aulast%3DBelov%26aufirst%3DG.%26aulast%3DSantiana%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DTakvorian%26aufirst%3DP.%2BM.%26aulast%3DPau%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%26aulast%3DKaushik-Basu%26aufirst%3DN.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DCameron%26aufirst%3DC.%2BE.%26aulast%3DEhrenfeld%26aufirst%3DE.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26aulast%3DAltan-Bonnet%26aufirst%3DN.%26atitle%3DViral%2520Reorganization%2520of%2520the%2520Secretory%2520Pathway%2520Generates%2520Distinct%2520Organelles%2520for%2520RNA%2520Replication%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D799%26epage%3D811%26doi%3D10.1016%2Fj.cell.2010.03.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">McNamara, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roland, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, B. K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormack, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manary, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeeman, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dechering, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, T. R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrich, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagaring, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibanez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhen, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plouffe, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meister, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rameh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trappe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haasen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmerman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauerwein, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suwanarusk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nosten, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tully, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kocken, C. H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glynne, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodenreider, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidock, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diagana, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winzeler, E. A.</span><span> </span><span class="NLM_article-title">Targeting Plasmodium PI(4)K to Eliminate Malaria</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1038/nature12782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnature12782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24284631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFaqsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=248-253&author=C.+W.+McNamaraauthor=M.+C.+S.+Leeauthor=C.+S.+Limauthor=S.+H.+Limauthor=J.+Rolandauthor=A.+Nagleauthor=O.+Simonauthor=B.+K.+S.+Yeungauthor=A.+K.+Chatterjeeauthor=S.+L.+McCormackauthor=M.+J.+Manaryauthor=A.-M.+Zeemanauthor=K.+J.+Decheringauthor=T.+R.+S.+Kumarauthor=P.+P.+Henrichauthor=K.+Gagaringauthor=M.+Ibanezauthor=N.+Katoauthor=K.+L.+Kuhenauthor=C.+Fischliauthor=M.+Rottmannauthor=D.+M.+Plouffeauthor=B.+Bursulayaauthor=S.+Meisterauthor=L.+Ramehauthor=J.+Trappeauthor=D.+Haasenauthor=M.+Timmermanauthor=R.+W.+Sauerweinauthor=R.+Suwanaruskauthor=B.+Russellauthor=L.+Reniaauthor=F.+Nostenauthor=D.+C.+Tullyauthor=C.+H.+M.+Kockenauthor=R.+J.+Glynneauthor=C.+Bodenreiderauthor=D.+A.+Fidockauthor=T.+T.+Diaganaauthor=E.+A.+Winzeler&title=Targeting+Plasmodium+PI%284%29K+to+Eliminate+Malaria&doi=10.1038%2Fnature12782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Plasmodium PI(4)K to eliminate malaria</span></div><div class="casAuthors">McNamara, Case W.; Lee, Marcus C. S.; Lim, Chek Shik; Lim, Siau Hoi; Roland, Jason; Nagle, Advait; Simon, Oliver; Yeung, Bryan K. S.; Chatterjee, Arnab K.; McCormack, Susan L.; Manary, Micah J.; Zeeman, Anne-Marie; Dechering, Koen J.; Kumar, T. R. Santha; Henrich, Philipp P.; Gagaring, Kerstin; Ibanez, Maureen; Kato, Nobutaka; Kuhen, Kelli L.; Fischli, Christoph; Rottmann, Matthias; Plouffe, David M.; Bursulaya, Badry; Meister, Stephan; Rameh, Lucia; Trappe, Joerg; Haasen, Dorothea; Timmerman, Martijn; Sauerwein, Robert W.; Suwanarusk, Rossarin; Russell, Bruce; Renia, Laurent; Nosten, Francois; Tully, David C.; Kocken, Clemens H. M.; Glynne, Richard J.; Bodenreider, Christophe; Fidock, David A.; Diagana, Thierry T.; Winzeler, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7479</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Achieving the goal of malaria elimination will depend on targeting Plasmodium pathways essential across all life stages.  Here the authors identify a lipid kinase, phosphatidylinositol-4-OH kinase (PI(4)K), as the target of imidazopyrazines, a new antimalarial compd. class that inhibits the intracellular development of multiple Plasmodium species at each stage of infection in the vertebrate host.  Imidazopyrazines demonstrate potent preventive, therapeutic, and transmission-blocking activity in rodent malaria models, are active against blood-stage field isolates of the major human pathogens P. falciparum and P. vivax, and inhibit liver-stage hypnozoites in the simian parasite P. cynomolgi.  The authors show that imidazopyrazines exert their effect through inhibitory interaction with the ATP-binding pocket of PI(4)K, altering the intracellular distribution of phosphatidylinositol-4-phosphate.  Collectively, the authors' data define PI(4)K as a key Plasmodium vulnerability, opening up new avenues of target-based discovery to identify drugs with an ideal activity profile for the prevention, treatment and elimination of malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTOmMeWluIubVg90H21EOLACvtfcHk0ljJVI5-rqsOSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFaqsLzP&md5=9ed8f3e41d257bb4535b4c813abd54cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature12782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12782%26sid%3Dliteratum%253Aachs%26aulast%3DMcNamara%26aufirst%3DC.%2BW.%26aulast%3DLee%26aufirst%3DM.%2BC.%2BS.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DLim%26aufirst%3DS.%2BH.%26aulast%3DRoland%26aufirst%3DJ.%26aulast%3DNagle%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DO.%26aulast%3DYeung%26aufirst%3DB.%2BK.%2BS.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DMcCormack%26aufirst%3DS.%2BL.%26aulast%3DManary%26aufirst%3DM.%2BJ.%26aulast%3DZeeman%26aufirst%3DA.-M.%26aulast%3DDechering%26aufirst%3DK.%2BJ.%26aulast%3DKumar%26aufirst%3DT.%2BR.%2BS.%26aulast%3DHenrich%26aufirst%3DP.%2BP.%26aulast%3DGagaring%26aufirst%3DK.%26aulast%3DIbanez%26aufirst%3DM.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DKuhen%26aufirst%3DK.%2BL.%26aulast%3DFischli%26aufirst%3DC.%26aulast%3DRottmann%26aufirst%3DM.%26aulast%3DPlouffe%26aufirst%3DD.%2BM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMeister%26aufirst%3DS.%26aulast%3DRameh%26aufirst%3DL.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DHaasen%26aufirst%3DD.%26aulast%3DTimmerman%26aufirst%3DM.%26aulast%3DSauerwein%26aufirst%3DR.%2BW.%26aulast%3DSuwanarusk%26aufirst%3DR.%26aulast%3DRussell%26aufirst%3DB.%26aulast%3DRenia%26aufirst%3DL.%26aulast%3DNosten%26aufirst%3DF.%26aulast%3DTully%26aufirst%3DD.%2BC.%26aulast%3DKocken%26aufirst%3DC.%2BH.%2BM.%26aulast%3DGlynne%26aufirst%3DR.%2BJ.%26aulast%3DBodenreider%26aufirst%3DC.%26aulast%3DFidock%26aufirst%3DD.%2BA.%26aulast%3DDiagana%26aufirst%3DT.%2BT.%26aulast%3DWinzeler%26aufirst%3DE.%2BA.%26atitle%3DTargeting%2520Plasmodium%2520PI%25284%2529K%2520to%2520Eliminate%2520Malaria%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D248%26epage%3D253%26doi%3D10.1038%2Fnature12782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Altan-Bonnet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinases: Hostages Harnessed to Build Panviral Replication Platforms</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1016/j.tibs.2012.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.tibs.2012.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22633842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1WmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2012&pages=293-302&author=N.+Altan-Bonnetauthor=T.+Balla&title=Phosphatidylinositol+4-Kinases%3A+Hostages+Harnessed+to+Build+Panviral+Replication+Platforms&doi=10.1016%2Fj.tibs.2012.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinases: hostages harnessed to build panviral replication platforms</span></div><div class="casAuthors">Altan-Bonnet, Nihal; Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Several RNA viruses have recently been shown to hijack members of the host phosphatidylinositol (PtdIns) 4-kinase (PI4K) family of enzymes.  They use PI4K to generate membranes enriched in phosphatidylinositide 4-phosphate (PtdIns4P or PI4P) lipids, which can be used as replication platforms.  Viral replication machinery is assembled on these platforms as a supramol. complex and PtdIns4P lipids regulate viral RNA synthesis.  This article highlights these recent studies on the regulation of viral RNA synthesis by PtdIns4P lipids.  It explores the potential mechanisms by which PtdIns4P lipids can contribute to viral replication and discusses the therapeutic potential of developing antiviral mols. that target host PI4Ks as a form of panviral therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-kUkSjCTrr7Vg90H21EOLACvtfcHk0li6wMEfGFokGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1WmtLk%253D&md5=11c0516aa3df27d16d6e945644697886</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2012.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2012.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DAltan-Bonnet%26aufirst%3DN.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DPhosphatidylinositol%25204-Kinases%253A%2520Hostages%2520Harnessed%2520to%2520Build%2520Panviral%2520Replication%2520Platforms%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2012%26volume%3D37%26spage%3D293%26epage%3D302%26doi%3D10.1016%2Fj.tibs.2012.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Mello, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguayo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cihlar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birkus, G.</span><span> </span><span class="NLM_article-title">Multiple Classes of Antiviral Agents Exhibit in Vitro Activity Against Human Rhinovirus Type C</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">–</span> <span class="NLM_lpage">1555</span><span class="refDoi"> DOI: 10.1128/AAC.01746-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.01746-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=1546-1555&author=C.+Melloauthor=E.+Aguayoauthor=M.+Rodriguezauthor=G.+Leeauthor=R.+Jordanauthor=T.+Cihlarauthor=G.+Birkus&title=Multiple+Classes+of+Antiviral+Agents+Exhibit+in+Vitro+Activity+Against+Human+Rhinovirus+Type+C&doi=10.1128%2FAAC.01746-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.01746-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01746-13%26sid%3Dliteratum%253Aachs%26aulast%3DMello%26aufirst%3DC.%26aulast%3DAguayo%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DJordan%26aufirst%3DR.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DBirkus%26aufirst%3DG.%26atitle%3DMultiple%2520Classes%2520of%2520Antiviral%2520Agents%2520Exhibit%2520in%2520Vitro%2520Activity%2520Against%2520Human%2520Rhinovirus%2520Type%2520C%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D1546%26epage%3D1555%26doi%3D10.1128%2FAAC.01746-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">van der Schaar, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Linden, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanke, K. H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strating, J. R. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pürstinger, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Haan, C. A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span> </span><span class="NLM_article-title">Coxsackievirus Mutants That Can Bypass Host Factor PI4KIIIβ and the Need for High Levels of PI4P Lipids for Replication</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1576</span><span class="NLM_x">–</span> <span class="NLM_lpage">1592</span><span class="refDoi"> DOI: 10.1038/cr.2012.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fcr.2012.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22945356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1arsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1576-1592&author=H.+M.+van+der+Schaarauthor=L.+van+der+Lindenauthor=K.+H.+W.+Lankeauthor=J.+R.+P.+M.+Stratingauthor=G.+P%C3%BCrstingerauthor=E.+de+Vriesauthor=C.+A.+M.+de+Haanauthor=J.+Neytsauthor=F.+J.+M.+van+Kuppeveld&title=Coxsackievirus+Mutants+That+Can+Bypass+Host+Factor+PI4KIII%CE%B2+and+the+Need+for+High+Levels+of+PI4P+Lipids+for+Replication&doi=10.1038%2Fcr.2012.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Coxsackievirus mutants that can bypass host factor PI4KIIIβ and the need for high levels of PI4P lipids for replication</span></div><div class="casAuthors">van der Schaar, Hilde M.; van der Linden, Lonneke; Lanke, Kjerstin H. W.; Strating, Jeroen R. P. M.; Puerstinger, Gerhard; de Vries, Erik; de Haan, Cornelis A. M.; Neyts, Johan; van Kuppeveld, Frank J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1576-1592</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">RNA viruses can rapidly mutate and acquire resistance to drugs that directly target viral enzymes, which poses serious problems in a clin. context.  Therefore, there is a growing interest in the development of antiviral drugs that target host factors crit. for viral replication, since they are unlikely to mutate in response to therapy.  We recently demonstrated that phosphatidylinositol-4-kinase IIIβ (PI4KIIIβ) and its product phosphatidylinositol-4-phosphate (PI4P) are essential for replication of enteroviruses, a group of medically important RNA viruses including poliovirus (PV), coxsackievirus, rhinovirus, and enterovirus 71.  Here, we show that enviroxime and GW5074 decreased PI4P levels at the Golgi complex by directly inhibiting PI4KIIIβ.  Coxsackievirus mutants resistant to these inhibitors harbor single point mutations in the non-structural protein 3A.  These 3A mutations did not confer compd.-resistance by restoring the activity of PI4KIIIβ in the presence of the compds.  Instead, replication of the mutant viruses no longer depended on PI4KIIIβ, since their replication was insensitive to siRNA-mediated depletion of PI4KIIIβ.  The mutant viruses also did not rely on other isoforms of PI4K.  Consistently, no high level of PI4P could be detected at the replication sites induced by the mutant viruses in the presence of the compds.  Collectively, these findings indicate that through specific single point mutations in 3A, CVB3 can bypass an essential host factor and lipid for its propagation, which is a new example of RNA viruses acquiring resistance against antiviral compds., even when they directly target host factors.  Cell Research (2012) 22:1576-1592. doi:10.1038/cr.2012.129; published online 4 Sept. 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6O0gJl98m4rVg90H21EOLACvtfcHk0li6wMEfGFokGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1arsLjF&md5=dc40ab703aba2302324cc44929f97932</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fcr.2012.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2012.129%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%2BM.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DL.%26aulast%3DLanke%26aufirst%3DK.%2BH.%2BW.%26aulast%3DStrating%26aufirst%3DJ.%2BR.%2BP.%2BM.%26aulast%3DP%25C3%25BCrstinger%26aufirst%3DG.%26aulast%3Dde%2BVries%26aufirst%3DE.%26aulast%3Dde%2BHaan%26aufirst%3DC.%2BA.%2BM.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26atitle%3DCoxsackievirus%2520Mutants%2520That%2520Can%2520Bypass%2520Host%2520Factor%2520PI4KIII%25CE%25B2%2520and%2520the%2520Need%2520for%2520High%2520Levels%2520of%2520PI4P%2520Lipids%2520for%2520Replication%26jtitle%3DCell%2520Res.%26date%3D2012%26volume%3D22%26spage%3D1576%26epage%3D1592%26doi%3D10.1038%2Fcr.2012.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Greninger, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knudsen, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betegon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burlingame, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeRisi, J. L.</span><span> </span><span class="NLM_article-title">The 3A Protein From Multiple Picornaviruses Utilizes the Golgi Adaptor Protein ACBD3 to Recruit PI4KIIIβ</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">3605</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1128/JVI.06778-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.06778-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22258260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A280%3ADC%252BC383pvFyrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=3605-3616&author=A.+L.+Greningerauthor=G.+M.+Knudsenauthor=M.+Betegonauthor=A.+L.+Burlingameauthor=J.+L.+DeRisi&title=The+3A+Protein+From+Multiple+Picornaviruses+Utilizes+the+Golgi+Adaptor+Protein+ACBD3+to+Recruit+PI4KIII%CE%B2&doi=10.1128%2FJVI.06778-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The 3A protein from multiple picornaviruses utilizes the golgi adaptor protein ACBD3 to recruit PI4KIIIβ</span></div><div class="casAuthors">Greninger Alexander L; Knudsen Giselle M; Betegon Miguel; Burlingame Alma L; Derisi Joseph L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3605-16</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The activity of phosphatidylinositol 4-kinase class III beta (PI4KIIIβ) has been shown to be required for the replication of multiple picornaviruses; however, it is unclear whether a physical association between PI4KIIIβ and the viral replication machinery exists and, if it does, whether association is necessary.  We examined the ability of the 3A protein from 18 different picornaviruses to form a complex with PI4KIIIβ by affinity purification of Strep-Tagged transiently transfected constructs followed by mass spectrometry and Western blotting for putative interacting targets.  We found that the 3A proteins of Aichi virus, bovine kobuvirus, poliovirus, coxsackievirus B3, and human rhinovirus 14 all copurify with PI4KIIIβ.  Furthermore, we found that multiple picornavirus 3A proteins copurify with the Golgi adaptor protein acyl coenzyme A (acyl-CoA) binding domain protein 3 (ACBD3/GPC60), including those from Aichi virus, bovine kobuvirus, human rhinovirus 14, poliovirus, and coxsackievirus B2, B3, and B5.  Affinity purification of ACBD3 confirmed interaction with multiple picornaviral 3A proteins and revealed the ability to bind PI4KIIIβ in the absence of 3A.  Mass-spectrometric analysis of transiently expressed Aichi virus, bovine kobuvirus, and human klassevirus 3A proteins demonstrated that the N-terminal glycines of these 3A proteins are myristoylated.  Alanine-scanning mutagenesis along the entire length of Aichi virus 3A followed by transient expression and affinity purification revealed that copurification of PI4KIIIβ could be eliminated by mutation of specific residues, with little or no effect on recruitment of ACBD3.  One mutation at the N terminus, I5A, significantly reduced copurification of both ACBD3 and PI4KIIIβ.  The dependence of Aichi virus replication on the activity of PI4KIIIβ was confirmed by both chemical and genetic inhibition.  Knockdown of ACBD3 by small interfering RNA (siRNA) also prevented replication of both Aichi virus and poliovirus.  Point mutations in 3A that eliminate PI4KIIIβ association sensitized Aichi virus to PIK93, suggesting that disruption of the 3A/ACBD3/PI4KIIIβ complex may represent a novel target for therapeutic intervention that would be complementary to the inhibition of the kinase activity itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBIfUigH8Y6oqdXsLcGTEUfW6udTcc2eap73pQm2XpFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC383pvFyrtA%253D%253D&md5=f5f1e487bed0c97e42250d778c271e72</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.06778-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.06778-11%26sid%3Dliteratum%253Aachs%26aulast%3DGreninger%26aufirst%3DA.%2BL.%26aulast%3DKnudsen%26aufirst%3DG.%2BM.%26aulast%3DBetegon%26aufirst%3DM.%26aulast%3DBurlingame%26aufirst%3DA.%2BL.%26aulast%3DDeRisi%26aufirst%3DJ.%2BL.%26atitle%3DThe%25203A%2520Protein%2520From%2520Multiple%2520Picornaviruses%2520Utilizes%2520the%2520Golgi%2520Adaptor%2520Protein%2520ACBD3%2520to%2520Recruit%2520PI4KIII%25CE%25B2%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D3605%26epage%3D3616%26doi%3D10.1128%2FJVI.06778-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Borawski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibaja, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mickanin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leighton-Davies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornellataracido, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baryza, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallarico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joberty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathy, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Compton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaither, L. A.</span><span> </span><span class="NLM_article-title">Class III Phosphatidylinositol 4-Kinase Alpha and Beta Are Novel Host Factor Regulators of Hepatitis C Virus Replication</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">10058</span><span class="NLM_x">–</span> <span class="NLM_lpage">10074</span><span class="refDoi"> DOI: 10.1128/JVI.02418-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.02418-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19605471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyrurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=10058-10074&author=J.+Borawskiauthor=P.+Trokeauthor=X.+Puyangauthor=V.+Gibajaauthor=S.+Zhaoauthor=C.+Mickaninauthor=J.+Leighton-Daviesauthor=C.+J.+Wilsonauthor=V.+Myerauthor=I.+Cornellataracidoauthor=J.+Baryzaauthor=J.+Tallaricoauthor=G.+Jobertyauthor=M.+Bantscheffauthor=M.+Schirleauthor=T.+Bouwmeesterauthor=J.+E.+Mathyauthor=K.+Linauthor=T.+Comptonauthor=M.+Labowauthor=B.+Wiedmannauthor=L.+A.+Gaither&title=Class+III+Phosphatidylinositol+4-Kinase+Alpha+and+Beta+Are+Novel+Host+Factor+Regulators+of+Hepatitis+C+Virus+Replication&doi=10.1128%2FJVI.02418-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Class III phosphatidylinositol 4-kinase alpha and beta are novel host factor regulators of hepatitis C virus replication</span></div><div class="casAuthors">Borawski, Jason; Troke, Philip; Puyang, Xiaoling; Gibaja, Veronica; Zhao, ShanChaun; Mickanin, Craig; Leighton-Davies, Juliet; Wilson, Christopher J.; Myer, Vic; Taracido, Ivan Cornella; Baryza, Jeremy; Tallarico, John; Joberty, Gerard; Bantscheff, Marcus; Schirle, Markus; Bouwmeester, Tewis; Mathy, Joanna E.; Lin, Kai; Compton, Teresa; Labow, Mark; Wiedmann, Brigitte; Gaither, L. Alex</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10058-10074</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Host factor pathways are known to be essential for hepatitis C virus (HCV) infection and replication in human liver cells.  To search for novel host factor proteins required for HCV replication, we screened a subgenomic genotype 1b replicon cell line (Luc-1b) with a kinome and druggable collection of 20,779 siRNAs.  We identified and validated several enzymes required for HCV replication, including class III phosphatidylinositol 4-kinases (PI4KA and PI4KB), carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase (CAD), and mevalonate (diphospho) decarboxylase.  Knockdown of PI4KA could inhibit the replication and/or HCV RNA levels of the two subgenomic genotype 1b clones (SG-1b and Luc-1b), two subgenomic genotype 1a clones (SG-1a and Luc-1a), JFH-1 genotype 2a infectious virus (JFH1-2a), and the genomic genotype 1a (FL-1a) replicon.  In contrast, PI4KB knockdown inhibited replication and/or HCV RNA levels of Luc-1b, SG-1b, and Luc-1a replicons.  The small mol. inhibitor, PIK93, was found to block subgenomic genotype 1b (Luc-1b), subgenomic genotype 1a (Luc-1a), and genomic genotype 2a (JFH1-2a) infectious virus replication in the nanomolar range.  PIK93 was characterized by using quant. chem. proteomics and in vitro biochem. assays to demonstrate PIK93 is a bone fide PI4KA and PI4KB inhibitor.  Our data demonstrate that genetic or pharmacol. modulation of PI4KA and PI4KB inhibits multiple genotypes of HCV and represents a novel druggable class of therapeutic targets for HCV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Zjz0O33L9bVg90H21EOLACvtfcHk0ljjdRDQ8--uGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyrurjN&md5=f823b4ac8a9efd864fced393cf487c94</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FJVI.02418-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02418-08%26sid%3Dliteratum%253Aachs%26aulast%3DBorawski%26aufirst%3DJ.%26aulast%3DTroke%26aufirst%3DP.%26aulast%3DPuyang%26aufirst%3DX.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLeighton-Davies%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DCornellataracido%26aufirst%3DI.%26aulast%3DBaryza%26aufirst%3DJ.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DMathy%26aufirst%3DJ.%2BE.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DCompton%26aufirst%3DT.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DWiedmann%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26atitle%3DClass%2520III%2520Phosphatidylinositol%25204-Kinase%2520Alpha%2520and%2520Beta%2520Are%2520Novel%2520Host%2520Factor%2520Regulators%2520of%2520Hepatitis%2520C%2520Virus%2520Replication%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D10058%26epage%3D10074%26doi%3D10.1128%2FJVI.02418-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">van der Schaar, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyssen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibaut, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Palma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Linden, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanke, K. H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbeken, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conrath, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Poël, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F. J. M.</span><span> </span><span class="NLM_article-title">A Novel, Broad-Spectrum Inhibitor of Enterovirus Replication That Targets Host Cell Factor Phosphatidylinositol 4-Kinase IIIβ</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4971</span><span class="NLM_x">–</span> <span class="NLM_lpage">4981</span><span class="refDoi"> DOI: 10.1128/AAC.01175-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.01175-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=4971-4981&author=H.+M.+van+der+Schaarauthor=P.+Leyssenauthor=H.+J.+Thibautauthor=A.+de+Palmaauthor=L.+van+der+Lindenauthor=K.+H.+W.+Lankeauthor=C.+Lacroixauthor=E.+Verbekenauthor=K.+Conrathauthor=A.+M.+Macleodauthor=D.+R.+Mitchellauthor=N.+J.+Palmerauthor=H.+van+de+Po%C3%ABlauthor=M.+Andrewsauthor=J.+Neytsauthor=F.+J.+M.+van+Kuppeveld&title=A+Novel%2C+Broad-Spectrum+Inhibitor+of+Enterovirus+Replication+That+Targets+Host+Cell+Factor+Phosphatidylinositol+4-Kinase+III%CE%B2&doi=10.1128%2FAAC.01175-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.01175-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01175-13%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BSchaar%26aufirst%3DH.%2BM.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3Dde%2BPalma%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DL.%26aulast%3DLanke%26aufirst%3DK.%2BH.%2BW.%26aulast%3DLacroix%26aufirst%3DC.%26aulast%3DVerbeken%26aufirst%3DE.%26aulast%3DConrath%26aufirst%3DK.%26aulast%3DMacleod%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BR.%26aulast%3DPalmer%26aufirst%3DN.%2BJ.%26aulast%3Dvan%2Bde%2BPo%25C3%25ABl%26aufirst%3DH.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26atitle%3DA%2520Novel%252C%2520Broad-Spectrum%2520Inhibitor%2520of%2520Enterovirus%2520Replication%2520That%2520Targets%2520Host%2520Cell%2520Factor%2520Phosphatidylinositol%25204-Kinase%2520III%25CE%25B2%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D4971%26epage%3D4981%26doi%3D10.1128%2FAAC.01175-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Arita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kojima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okabe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimizu, H.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is a Target of Enviroxime-Like Compounds for Antipoliovirus Activity</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2364</span><span class="NLM_x">–</span> <span class="NLM_lpage">2372</span><span class="refDoi"> DOI: 10.1128/JVI.02249-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FJVI.02249-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21177810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVWltL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=2364-2372&author=M.+Aritaauthor=H.+Kojimaauthor=T.+Naganoauthor=T.+Okabeauthor=T.+Wakitaauthor=H.+Shimizu&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+a+Target+of+Enviroxime-Like+Compounds+for+Antipoliovirus+Activity&doi=10.1128%2FJVI.02249-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinase III beta is a target of enviroxime-like compounds for antipoliovirus activity</span></div><div class="casAuthors">Arita, Minetaro; Kojima, Hirotatsu; Nagano, Tetsuo; Okabe, Takayoshi; Wakita, Takaji; Shimizu, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2364-2372</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enviroxime is an antienterovirus compd. that targets viral protein 3A and/or 3AB and suppresses a step in enterovirus replication by unknown mechanism.  To date, four antienterovirus compds., i.e., GW5074, Flt3 inhibitor II, TTP-8307, and AN-12-H5, are known to have similar mutations in the 3A protein-encoding region causing resistance to enviroxime (a G5318A [3A-Ala70Thr] mutation in poliovirus [PV]) and are considered enviroxime-like compds.  Recently, antienterovirus activity of a phosphatidylinositol 4-kinase III beta (PI4KB) inhibitor, PIK93, was reported, suggesting that PI4KB is an important host factor targetable by antienterovirus compds.  In this study, we analyzed the inhibitory effects of previously identified enviroxime-like compds. (GW5074 and AN-12-H5) and a newly identified antienterovirus compd., T-00127-HEV1, on phosphoinositide (PI) kinases.  We found that T-00127-HEV1 inhibited PI4KB activity with a higher specificity for than other PI kinases, in contrast to GW5074, which had a broad specificity for PI kinases.  In contrast, AN-12-H5 showed no inhibitory effect on PI4KB activity and only moderate inhibitory effects on PI 3-kinase activity.  Small interfering RNA (siRNA) screening targeting PI kinases identified PI4KB is a target of GW5074 and T-00127-HEV1, but not of AN-12-H5, for anti-PV activity.  Interestingly, T-00127-HEV1 and GW5074 did not inhibit hepatitis C virus (HCV) replication, in contrast to a strong inhibitory effect of AN-12-H5.  These results suggested that PI4KB is an enterovirus-specific host factor required for the replication process and targeted by some enviroxime-like compds. (T-00127-HEV1 and GW5074) and that enviroxime-like compds. may have targets other than PI kinases for their antiviral effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRbO_t7G_n4rVg90H21EOLACvtfcHk0lgfiSilw9A-0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVWltL%252FM&md5=10d970def4a0ab38c7ed9fc33e200d4f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FJVI.02249-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02249-10%26sid%3Dliteratum%253Aachs%26aulast%3DArita%26aufirst%3DM.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DH.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520a%2520Target%2520of%2520Enviroxime-Like%2520Compounds%2520for%2520Antipoliovirus%2520Activity%26jtitle%3DJ.%2520Virol.%26date%3D2011%26volume%3D85%26spage%3D2364%26epage%3D2372%26doi%3D10.1128%2FJVI.02249-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zunder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldenberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loewith, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span> </span><span class="NLM_article-title">A Pharmacological Map of the PI3-K Family Defines a Role for P110alpha in Insulin Signaling</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.cell.2006.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=16647110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlSkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=733-747&author=Z.+Knightauthor=B.+Gonzalezauthor=M.+Feldmanauthor=E.+Zunderauthor=D.+Goldenbergauthor=O.+Williamsauthor=R.+Loewithauthor=D.+Stokoeauthor=A.+Ballaauthor=B.+Tothauthor=T.+Ballaauthor=W.+Weissauthor=R.+Williamsauthor=K.+Shokat&title=A+Pharmacological+Map+of+the+PI3-K+Family+Defines+a+Role+for+P110alpha+in+Insulin+Signaling&doi=10.1016%2Fj.cell.2006.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A pharmacological map of the PI3-K family defines a role for p110α in insulin signaling</span></div><div class="casAuthors">Knight, Zachary A.; Gonzalez, Beatriz; Feldman, Morri E.; Zunder, Eli R.; Goldenberg, David D.; Williams, Olusegun; Loewith, Robbie; Stokoe, David; Balla, Andras; Toth, Balazs; Balla, Tamas; Weiss, William A.; Williams, Roger L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">733-747</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3-Ks) are an important emerging class of drug targets, but the unique roles of PI3-K isoforms remain poorly defined.  We describe here an approach to pharmacol. interrogate the PI3-K family.  A chem. diverse panel of PI3-K inhibitors was synthesized, and their target selectivity was biochem. enumerated, revealing cryptic homologies across targets and chemotypes.  Crystal structures of three inhibitors bound to p110γ identify a conformationally mobile region that is uniquely exploited by selective compds.  This chem. array was then used to define the PI3-K isoforms required for insulin signaling.  We find that p110α is the primary insulin-responsive PI3-K in cultured cells, whereas p110β is dispensable but sets a phenotypic threshold for p110α activity.  Compds. targeting p110α block the acute effects of insulin treatment in vivo, whereas a p110β inhibitor has no effect.  These results illustrate systematic target validation using a matrix of inhibitors that span a protein family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrozjvOeMXP3bVg90H21EOLACvtfcHk0lgfiSilw9A-0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlSkt7o%253D&md5=b95a8cb433927ff9a75b33fc4ac2c8a0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DGonzalez%26aufirst%3DB.%26aulast%3DFeldman%26aufirst%3DM.%26aulast%3DZunder%26aufirst%3DE.%26aulast%3DGoldenberg%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DO.%26aulast%3DLoewith%26aufirst%3DR.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DB.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DWeiss%26aufirst%3DW.%26aulast%3DWilliams%26aufirst%3DR.%26aulast%3DShokat%26aufirst%3DK.%26atitle%3DA%2520Pharmacological%2520Map%2520of%2520the%2520PI3-K%2520Family%2520Defines%2520a%2520Role%2520for%2520P110alpha%2520in%2520Insulin%2520Signaling%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D733%26epage%3D747%26doi%3D10.1016%2Fj.cell.2006.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Várnai, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Maintenance of Hormone-Sensitive Phosphoinositide Pools in the Plasma Membrane Requires Phosphatidylinositol 4-Kinase IIIalpha</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">711</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span><span class="refDoi"> DOI: 10.1091/mbc.E07-07-0713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1091%2Fmbc.E07-07-0713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=18077555" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=711-721&author=A.+Ballaauthor=Y.+J.+Kimauthor=P.+V%C3%A1rnaiauthor=Z.+Szentpeteryauthor=Z.+Knightauthor=K.+M.+Shokatauthor=T.+Balla&title=Maintenance+of+Hormone-Sensitive+Phosphoinositide+Pools+in+the+Plasma+Membrane+Requires+Phosphatidylinositol+4-Kinase+IIIalpha&doi=10.1091%2Fmbc.E07-07-0713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E07-07-0713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E07-07-0713%26sid%3Dliteratum%253Aachs%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DV%25C3%25A1rnai%26aufirst%3DP.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DMaintenance%2520of%2520Hormone-Sensitive%2520Phosphoinositide%2520Pools%2520in%2520the%2520Plasma%2520Membrane%2520Requires%2520Phosphatidylinositol%25204-Kinase%2520IIIalpha%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2007%26volume%3D19%26spage%3D711%26epage%3D721%26doi%3D10.1091%2Fmbc.E07-07-0713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Burke, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perisic, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaughlin, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutaganira, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. L.</span><span> </span><span class="NLM_article-title">Structures of PI4KIIIβ Complexes Show Simultaneous Recruitment of Rab11 and Its Effectors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">344</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">1038</span><span class="refDoi"> DOI: 10.1126/science.1253397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1126%2Fscience.1253397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24876499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFSmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=1035-1038&author=J.+E.+Burkeauthor=A.+J.+Inglisauthor=O.+Perisicauthor=G.+R.+Massonauthor=S.+H.+McLaughlinauthor=F.+Rutaganiraauthor=K.+M.+Shokatauthor=R.+L.+Williams&title=Structures+of+PI4KIII%CE%B2+Complexes+Show+Simultaneous+Recruitment+of+Rab11+and+Its+Effectors&doi=10.1126%2Fscience.1253397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of PI4KIIIβ complexes show simultaneous recruitment of Rab11 and its effectors</span></div><div class="casAuthors">Burke, John E.; Inglis, Alison J.; Perisic, Olga; Masson, Glenn R.; McLaughlin, Stephen H.; Rutaganira, Florentine; Shokat, Kevan M.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6187</span>),
    <span class="NLM_cas:pages">1035-1038</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Phosphatidylinositol 4-kinases (PI4Ks) and small guanosine triphosphatases (GTPases) are essential for processes that require expansion and remodeling of phosphatidylinositol 4-phosphate (PI4P)-contg. membranes, including cytokinesis, intracellular development of malarial pathogens, and replication of a wide range of RNA viruses.  However, the structural basis for coordination of PI4K, GTPases, and their effectors is unknown.  Here, we describe structures of PI4Kβ (PI4KIIIβ) bound to the small GTPase Rab11a without and with the Rab11 effector protein FIP3.  The Rab11-PI4KIIIβ interface is distinct compared with known structures of Rab complexes and does not involve switch regions used by GTPase effectors.  Our data provide a mechanism for how PI4KIIIβ coordinates Rab11 and its effectors on PI4P-enriched membranes and also provide strategies for the design of specific inhibitors that could potentially target plasmodial PI4KIIIβ to combat malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplEdUeizaig7Vg90H21EOLACvtfcHk0lgfiSilw9A-0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFSmu7w%253D&md5=9eed42f95ff8f3aeb4f6cef76e3ccdec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1253397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1253397%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DInglis%26aufirst%3DA.%2BJ.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DMasson%26aufirst%3DG.%2BR.%26aulast%3DMcLaughlin%26aufirst%3DS.%2BH.%26aulast%3DRutaganira%26aufirst%3DF.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26atitle%3DStructures%2520of%2520PI4KIII%25CE%25B2%2520Complexes%2520Show%2520Simultaneous%2520Recruitment%2520of%2520Rab11%2520and%2520Its%2520Effectors%26jtitle%3DScience%26date%3D2014%26volume%3D344%26spage%3D1035%26epage%3D1038%26doi%3D10.1126%2Fscience.1253397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavshanjian, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perisic, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houseman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R.</span><span> </span><span class="NLM_article-title">Shaping Development of Autophagy Inhibitors with the Structure of the Lipid Kinase Vps34</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">1638</span><span class="NLM_x">–</span> <span class="NLM_lpage">1642</span><span class="refDoi"> DOI: 10.1126/science.1184429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1126%2Fscience.1184429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=20339072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslaqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1638-1642&author=S.+Millerauthor=B.+Tavshanjianauthor=A.+Oleksyauthor=O.+Perisicauthor=B.+Housemanauthor=K.+Shokatauthor=R.+Williams&title=Shaping+Development+of+Autophagy+Inhibitors+with+the+Structure+of+the+Lipid+Kinase+Vps34&doi=10.1126%2Fscience.1184429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Shaping Development of Autophagy Inhibitors with the Structure of the Lipid Kinase Vps34</span></div><div class="casAuthors">Miller, Simon; Tavshanjian, Brandon; Oleksy, Arkadiusz; Perisic, Olga; Houseman, Benjamin T.; Shokat, Kevan M.; Williams, Roger L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5973</span>),
    <span class="NLM_cas:pages">1638-1642</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinases (PI3Ks) are lipid kinases with diverse roles in health and disease.  The primordial PI3K, Vps34, is present in all eukaryotes and has essential roles in autophagy, membrane trafficking, and cell signaling.  We solved the crystal structure of Vps34 at 2.9 angstrom resoln., which revealed a constricted adenine-binding pocket, suggesting the reason that specific inhibitors of this class of PI3K have proven elusive.  Both the phosphoinositide-binding loop and the carboxyl-terminal helix of Vps34 mediate catalysis on membranes and suppress futile ATPase cycles.  Vps34 appears to alternate between a closed cytosolic form and an open form on the membrane.  Structures of Vps34 complexes with a series of inhibitors reveal the reason that an autophagy inhibitor preferentially inhibits Vps34 and underpin the development of new potent and specific Vps34 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkkZ9w5NCkFLVg90H21EOLACvtfcHk0ljTnQjE5JR0nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslaqs7s%253D&md5=f20c69ef94d3b80fba72cf04e625649e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.1184429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1184429%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DS.%26aulast%3DTavshanjian%26aufirst%3DB.%26aulast%3DOleksy%26aufirst%3DA.%26aulast%3DPerisic%26aufirst%3DO.%26aulast%3DHouseman%26aufirst%3DB.%26aulast%3DShokat%26aufirst%3DK.%26aulast%3DWilliams%26aufirst%3DR.%26atitle%3DShaping%2520Development%2520of%2520Autophagy%2520Inhibitors%2520with%2520the%2520Structure%2520of%2520the%2520Lipid%2520Kinase%2520Vps34%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1638%26epage%3D1642%26doi%3D10.1126%2Fscience.1184429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Lamarche, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borawski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bose, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capacci-Daniel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colvin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennehy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drumm, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaither, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeever, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thohan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tommasi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zabawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span> </span><span class="NLM_article-title">Anti-Hepatitis C Virus Activity and Toxicity of Type III Phosphatidylinositol-4-Kinase Beta Inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5149</span><span class="NLM_x">–</span> <span class="NLM_lpage">5156</span><span class="refDoi"> DOI: 10.1128/AAC.00946-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.00946-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22825118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKiu7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=5149-5156&author=M.+J.+Lamarcheauthor=J.+Borawskiauthor=A.+Boseauthor=C.+Capacci-Danielauthor=R.+Colvinauthor=M.+Dennehyauthor=J.+Dingauthor=M.+Doblerauthor=J.+Drummauthor=L.+A.+Gaitherauthor=J.+Gaoauthor=X.+Jiangauthor=K.+Linauthor=U.+McKeeverauthor=X.+Puyangauthor=P.+Ramanauthor=S.+Thohanauthor=R.+Tommasiauthor=K.+Wagnerauthor=X.+Xiongauthor=T.+Zabawaauthor=S.+Zhuauthor=B.+Wiedmann&title=Anti-Hepatitis+C+Virus+Activity+and+Toxicity+of+Type+III+Phosphatidylinositol-4-Kinase+Beta+Inhibitors&doi=10.1128%2FAAC.00946-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-hepatitis C virus activity and toxicity of type III phosphatidylinositol-4-kinase beta inhibitors</span></div><div class="casAuthors">LaMarche, M. J.; Borawski, J.; Bose, A.; Capacci-Daniel, C.; Colvin, R.; Dennehy, M.; Ding, J.; Dobler, M.; Drumm, J.; Gaither, L. A.; Gao, J.; Jiang, X.; Lin, K.; McKeever, U.; Puyang, X.; Raman, P.; Thohan, S.; Tommasi, R.; Wagner, K.; Xiong, X.; Zabawa, T.; Zhu, S.; Wiedmann, B.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5149-5156</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Type III phosphatidylinositol-4-kinase beta (PI4KIIIβ) was previously implicated in hepatitis C virus (HCV) replication by small interfering RNA (siRNA) depletion and was therefore proposed as a novel cellular target for the treatment of hepatitis C.  Medicinal chem. efforts identified highly selective PI4KIIIβ inhibitors that potently inhibited the replication of genotype 1a and 1b HCV replicons and genotype 2a virus in vitro.  Replicon cells required more than 5 wk to reach low levels of 3- to 5-fold resistance, suggesting a high resistance barrier to these cellular targets.  Extensive in vitro profiling of the compds. revealed a role of PI4KIIIβ in lymphocyte proliferation.  Previously proposed functions of PI4KIIIβ in insulin secretion and the regulation of several ion channels were not perturbed with these inhibitors.  Moreover, PI4KIIIβ inhibitors were not generally cytotoxic as demonstrated across hundreds of cell lines and primary cells.  However, an unexpected antiproliferative effect in lymphocytes precluded their further development for the treatment of hepatitis C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg3G6T2U7Lv7Vg90H21EOLACvtfcHk0ljTnQjE5JR0nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKiu7jK&md5=fd7a50d1813d04862e5d8fa549c73da5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FAAC.00946-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00946-12%26sid%3Dliteratum%253Aachs%26aulast%3DLamarche%26aufirst%3DM.%2BJ.%26aulast%3DBorawski%26aufirst%3DJ.%26aulast%3DBose%26aufirst%3DA.%26aulast%3DCapacci-Daniel%26aufirst%3DC.%26aulast%3DColvin%26aufirst%3DR.%26aulast%3DDennehy%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DDrumm%26aufirst%3DJ.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DMcKeever%26aufirst%3DU.%26aulast%3DPuyang%26aufirst%3DX.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DK.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DZabawa%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DWiedmann%26aufirst%3DB.%26atitle%3DAnti-Hepatitis%2520C%2520Virus%2520Activity%2520and%2520Toxicity%2520of%2520Type%2520III%2520Phosphatidylinositol-4-Kinase%2520Beta%2520Inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D5149%26epage%3D5156%26doi%3D10.1128%2FAAC.00946-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Fowler, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhail, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masson, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutaganira, F. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. E.</span><span> </span><span class="NLM_article-title">Using Hydrogen Deuterium Exchange Mass Spectrometry to Engineer Optimized Constructs for Crystallization of Protein Complexes: Case Study of PI4KIIIβ with Rab11</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/pro.2879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1002%2Fpro.2879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=26756197" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+L.+Fowlerauthor=J.+A.+McPhailauthor=M.+L.+Jenkinsauthor=G.+R.+Massonauthor=F.+U.+Rutaganiraauthor=K.+M.+Shokatauthor=R.+L.+Williamsauthor=J.+E.+Burke&title=Using+Hydrogen+Deuterium+Exchange+Mass+Spectrometry+to+Engineer+Optimized+Constructs+for+Crystallization+of+Protein+Complexes%3A+Case+Study+of+PI4KIII%CE%B2+with+Rab11&doi=10.1002%2Fpro.2879"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fpro.2879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2879%26sid%3Dliteratum%253Aachs%26aulast%3DFowler%26aufirst%3DM.%2BL.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DJenkins%26aufirst%3DM.%2BL.%26aulast%3DMasson%26aufirst%3DG.%2BR.%26aulast%3DRutaganira%26aufirst%3DF.%2BU.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWilliams%26aufirst%3DR.%2BL.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26atitle%3DUsing%2520Hydrogen%2520Deuterium%2520Exchange%2520Mass%2520Spectrometry%2520to%2520Engineer%2520Optimized%2520Constructs%2520for%2520Crystallization%2520of%2520Protein%2520Complexes%253A%2520Case%2520Study%2520of%2520PI4KIII%25CE%25B2%2520with%2520Rab11%26jtitle%3DProtein%2520Sci.%26date%3D2016%26doi%3D10.1002%2Fpro.2879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Jovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kean, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szentpetery, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polevoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brill, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span> </span><span class="NLM_article-title">Two Phosphatidylinositol 4-Kinases Control Lysosomal Delivery of the Gaucher Disease Enzyme, B-Glucocerebrosidase</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1533</span><span class="NLM_x">–</span> <span class="NLM_lpage">1545</span><span class="refDoi"> DOI: 10.1091/mbc.E11-06-0553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1091%2Fmbc.E11-06-0553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22337770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFWjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=1533-1545&author=M.+Jovicauthor=M.+J.+Keanauthor=Z.+Szentpeteryauthor=G.+Polevoyauthor=A.-C.+Gingrasauthor=J.+A.+Brillauthor=T.+Balla&title=Two+Phosphatidylinositol+4-Kinases+Control+Lysosomal+Delivery+of+the+Gaucher+Disease+Enzyme%2C+B-Glucocerebrosidase&doi=10.1091%2Fmbc.E11-06-0553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Two phosphatidylinositol 4-kinases control lysosomal delivery of the Gaucher disease enzyme, β-glucocerebrosidase</span></div><div class="casAuthors">Jovic, Marko; Kean, Michelle J.; Szentpetery, Zsofia; Polevoy, Gordon; Gingras, Anne-Claude; Brill, Julie A.; Balla, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1533-1545</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Gaucher disease is a lysosomal storage disorder caused by a defect in the degrdn. of glucosylceramide catalyzed by the lysosomal enzyme β-glucocerebrosidase (GBA).  GBA reaches lysosomes via assocn. with its receptor, lysosomal integral membrane protein type 2 (LIMP-2).  We found that distinct phosphatidylinositol 4-kinases (PI4Ks) play important roles at multiple steps in the trafficking pathway of the LIMP-2/GBA complex.  Acute depletion of phosphatidylinositol 4-phosphate in the Golgi caused accumulation of LIMP-2 in this compartment, and PI4KIIIβ was found to be responsible for controlling the exit of LIMP-2 from the Golgi.  In contrast, depletion of PI4KIIα blocked trafficking at a post-Golgi compartment, leading to accumulation of LIMP-2 in enlarged endosomal vesicles.  PI4KIIα depletion also caused secretion of missorted GBA into the medium, which was attenuated by limiting LIMP-2/GBA exit from the Golgi by PI4KIIIβ inhibitors.  These studies identified PI4KIIIβ and PI4KIIα as important regulators of lysosomal delivery of GBA, revealing a new element of control to sphingolipid homeostasis by phosphoinositides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWLQpU4eDPf7Vg90H21EOLACvtfcHk0ljqoQ6ElujNjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFWjsrY%253D&md5=2d56c46e7f55dda26042bb5aab8bee95</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E11-06-0553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E11-06-0553%26sid%3Dliteratum%253Aachs%26aulast%3DJovic%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DM.%2BJ.%26aulast%3DSzentpetery%26aufirst%3DZ.%26aulast%3DPolevoy%26aufirst%3DG.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DBrill%26aufirst%3DJ.%2BA.%26aulast%3DBalla%26aufirst%3DT.%26atitle%3DTwo%2520Phosphatidylinositol%25204-Kinases%2520Control%2520Lysosomal%2520Delivery%2520of%2520the%2520Gaucher%2520Disease%2520Enzyme%252C%2520B-Glucocerebrosidase%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2012%26volume%3D23%26spage%3D1533%26epage%3D1545%26doi%3D10.1091%2Fmbc.E11-06-0553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">D’Angelo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicinanza, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Matteis, M. A.</span><span> </span><span class="NLM_article-title">Phosphoinositides in Golgi Complex Function</span> <span class="citation_source-journal">Subcell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1007/978-94-007-3015-1_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1007%2F978-94-007-3015-1_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22374093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2012&pages=255-270&author=G.+D%E2%80%99Angeloauthor=M.+Vicinanzaauthor=C.+Wilsonauthor=M.+A.+De+Matteis&title=Phosphoinositides+in+Golgi+Complex+Function&doi=10.1007%2F978-94-007-3015-1_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoinositides in Golgi complex function</span></div><div class="casAuthors">D'Angelo, Giovanni; Vicinanza, Mariella; Wilson, Cathal; De Matteis, Maria Antonietta</div><div class="citationInfo"><span class="NLM_cas:title">Subcellular Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">Phosphoinositides II: The Diverse Biological Functions</span>),
    <span class="NLM_cas:pages">255-270</span>CODEN:
                <span class="NLM_cas:coden">SBCBAG</span>;
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The Golgi complex is a ribbon-like organelle composed of stacks of flat cisternae interconnected by tubular junctions.  It occupies a central position in the endomembrane system as proteins and lipids that are synthesized in the endoplasmic reticulum (ER) pass through the Golgi complex to undergo biosynthetic modification (mainly glycosylation) and to be sorted to their final destinations.  In addn. the Golgi complex possesses a no. of activities, apparently not directly connected with its main role in trafficking and sorting, which have been recently reviewed in Wilson et al. 2011.  In spite of the const. massive flux of material the Golgi complex maintains its identity and phosphoinositides (PIs), among other factors, play a central role in this process.  The active metab. of PIs at the Golgi is necessary for the proper functioning of the organelle both in terms of membrane trafficking/sorting and its manifold metabolic and signalling activities.  Phosphatidylinositol 4-phosphate (PtdIns4P), in particular, is responsible for the recruitment of numerous cytosolic proteins that recognize and bind PtdIns4P via specific lipid-binding domains.  In this chapter we will summarize the findings that have contributed to our current understanding of the role of PIs in the biol. of the Golgi complex in terms of the regulation of PI metab. and the functional roles and regulation of PtdIns4P effectors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWoGmyQfy8bVg90H21EOLACvtfcHk0ljqoQ6ElujNjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChtrzP&md5=ced1c6db948d43f03830d245c5cb15b9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-3015-1_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-94-007-3015-1_8%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DG.%26aulast%3DVicinanza%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDe%2BMatteis%26aufirst%3DM.%2BA.%26atitle%3DPhosphoinositides%2520in%2520Golgi%2520Complex%2520Function%26jtitle%3DSubcell.%2520Biochem.%26date%3D2012%26volume%3D59%26spage%3D255%26epage%3D270%26doi%3D10.1007%2F978-94-007-3015-1_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Mejdrová, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalupská, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kögler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šála, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plačková, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumlová, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hřebabecký, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Procházková, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dejmek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strunin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birkus, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertlíková-Kaiserová, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nencka, R.</span><span> </span><span class="NLM_article-title">Highly Selective Phosphatidylinositol 4-Kinase IIIβ Inhibitors and Structural Insight Into Their Mode of Action</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">3767</span><span class="NLM_x">–</span> <span class="NLM_lpage">3793</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00499</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00499" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslOit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3767-3793&author=I.+Mejdrov%C3%A1author=D.+Chalupsk%C3%A1author=M.+K%C3%B6glerauthor=M.+%C5%A0%C3%A1laauthor=P.+Pla%C4%8Dkov%C3%A1author=A.+Baumlov%C3%A1author=H.+H%C5%99ebabeck%C3%BDauthor=E.+Proch%C3%A1zkov%C3%A1author=M.+Dejmekauthor=R.+Guillonauthor=D.+Struninauthor=J.+Weberauthor=G.+Leeauthor=G.+Birkusauthor=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1author=E.+Bouraauthor=R.+Nencka&title=Highly+Selective+Phosphatidylinositol+4-Kinase+III%CE%B2+Inhibitors+and+Structural+Insight+Into+Their+Mode+of+Action&doi=10.1021%2Facs.jmedchem.5b00499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective Phosphatidylinositol 4-Kinase IIIβ Inhibitors and Structural Insight into Their Mode of Action</span></div><div class="casAuthors">Mejdrova, Ivana; Chalupska, Dominika; Kogler, Martin; Sala, Michal; Plackova, Pavla; Baumlova, Adriana; Hrebabecky, Hubert; Prochazkova, Eliska; Dejmek, Milan; Guillon, Remi; Strunin, Dmytro; Weber, Jan; Lee, Gary; Birkus, Gabriel; Mertlikova-Kaiserova, Helena; Boura, Evzen; Nencka, Radim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3767-3793</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 4-kinase IIIβ is a cellular lipid kinase pivotal to pathogenesis of various RNA viruses.  These viruses hijack the enzyme to modify the structure of intracellular membranes and use them for the construction of functional replication machinery.  Selective inhibitors of this enzyme are potential broad-spectrum antiviral agents, as inhibition of this enzyme results in the arrest of replication of PI4K IIIβ-dependent viruses.  Herein, we report a detailed study of novel selective inhibitors of PI4K IIIβ, which exert antiviral activity against a panel of single-stranded pos.-sense RNA viruses.  Our crystallog. data show that the inhibitors occupy the binding site for the adenine ring of the ATP mol. and therefore prevent the phosphorylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojwEoKXCJFnbVg90H21EOLACvtfcHk0ljqoQ6ElujNjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslOit7k%253D&md5=0f1e2c86236783ded33bfe529fafe5f1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00499%26sid%3Dliteratum%253Aachs%26aulast%3DMejdrov%25C3%25A1%26aufirst%3DI.%26aulast%3DChalupsk%25C3%25A1%26aufirst%3DD.%26aulast%3DK%25C3%25B6gler%26aufirst%3DM.%26aulast%3D%25C5%25A0%25C3%25A1la%26aufirst%3DM.%26aulast%3DPla%25C4%258Dkov%25C3%25A1%26aufirst%3DP.%26aulast%3DBaumlov%25C3%25A1%26aufirst%3DA.%26aulast%3DH%25C5%2599ebabeck%25C3%25BD%26aufirst%3DH.%26aulast%3DProch%25C3%25A1zkov%25C3%25A1%26aufirst%3DE.%26aulast%3DDejmek%26aufirst%3DM.%26aulast%3DGuillon%26aufirst%3DR.%26aulast%3DStrunin%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DBirkus%26aufirst%3DG.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26aulast%3DBoura%26aufirst%3DE.%26aulast%3DNencka%26aufirst%3DR.%26atitle%3DHighly%2520Selective%2520Phosphatidylinositol%25204-Kinase%2520III%25CE%25B2%2520Inhibitors%2520and%2520Structural%2520Insight%2520Into%2520Their%2520Mode%2520of%2520Action%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3767%26epage%3D3793%26doi%3D10.1021%2Facs.jmedchem.5b00499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Arita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philipov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galabov, A. S.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is the Target of Oxoglaucine and Pachypodol (Ro 09–0179) for Their Anti-Poliovirus Activities, and Locates at Upstream of the Target Step of Brefeldin a</span> <span class="citation_source-journal">Microbiol. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1111/1348-0421.12261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1111%2F1348-0421.12261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=25891300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGmtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=338-347&author=M.+Aritaauthor=S.+Philipovauthor=A.+S.+Galabov&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+the+Target+of+Oxoglaucine+and+Pachypodol+%28Ro+09%E2%80%930179%29+for+Their+Anti-Poliovirus+Activities%2C+and+Locates+at+Upstream+of+the+Target+Step+of+Brefeldin+a&doi=10.1111%2F1348-0421.12261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphatidylinositol 4-kinase III beta is the target of oxoglaucine and pachypodol (Ro 09-0179) for their anti-poliovirus activities, and is located at upstream of the target step of brefeldin A</span></div><div class="casAuthors">Arita, Minetaro; Philipov, Stefan; Galabov, Angel S.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">338-347</span>CODEN:
                <span class="NLM_cas:coden">MIIMDV</span>;
        ISSN:<span class="NLM_cas:issn">0385-5600</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In recent years, phosphatidylinositol 4-kinase III beta (PI4KB) has emerged as a conserved target of anti-picornavirus compds.  PI4KB was identified as the direct target of the plant-derived anti-picornavirus compds. oxoglaucine and pachypodol (also known as Ro 09-0179).  PI4KB was also identified as the target via which pachypodol interferes with brefeldin A (BFA)-induced Golgi disassembly in non-infected cells.  Oxysterol-binding protein (OSBP) inhibitor also has interfering activity against BFA.  It seems that this interference is not essential for the anti-poliovirus (PV) activities of BFA and PI4KB/OSBP inhibitors.  BFA inhibited early to late phase PV replication (0 to 6 h postinfection) as well as PI4KB inhibitor, but with some delay compared to guanidine hydrochloride treatment.  In contrast with PI4KB/OSBP inhibitors, BFA inhibited viral nascent RNA synthesis, suggesting that BFA targets some step of viral RNA synthesis located downstream of the PI4KB/OSBP pathway in PV replication.  The authors' results suggest that PI4KB is a major target of anti-picornavirus compds. identified in vitro for their anti-picornavirus activities and for some uncharacterized biol. phenomena caused by these compds., and that BFA and PI4KB/OSBP inhibitors synergistically repress PV replication by targeting distinct steps in viral RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_6y2_9R-NK7Vg90H21EOLACvtfcHk0lhM6064sgGjWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGmtbjO&md5=4661d88f9c34142f0584c9f4d243e6f5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2F1348-0421.12261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1348-0421.12261%26sid%3Dliteratum%253Aachs%26aulast%3DArita%26aufirst%3DM.%26aulast%3DPhilipov%26aufirst%3DS.%26aulast%3DGalabov%26aufirst%3DA.%2BS.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520the%2520Target%2520of%2520Oxoglaucine%2520and%2520Pachypodol%2520%2528Ro%252009%25E2%2580%25930179%2529%2520for%2520Their%2520Anti-Poliovirus%2520Activities%252C%2520and%2520Locates%2520at%2520Upstream%2520of%2520the%2520Target%2520Step%2520of%2520Brefeldin%2520a%26jtitle%3DMicrobiol.%2520Immunol.%26date%3D2015%26volume%3D59%26spage%3D338%26epage%3D347%26doi%3D10.1111%2F1348-0421.12261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Waring, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulder, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemington, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKelvie, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preston, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raubo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treinies, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, J. J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, R. J.</span><span> </span><span class="NLM_article-title">Potent, Selective Small Molecule Inhibitors of Type III Phosphatidylinositol-4-Kinase A- but Not B-Inhibit the Phosphatidylinositol Signaling Cascade and Cancer Cell Proliferation</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5388</span><span class="NLM_x">–</span> <span class="NLM_lpage">5390</span><span class="refDoi"> DOI: 10.1039/C3CC48391F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1039%2FC3CC48391F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24366037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=5388-5390&author=M.+J.+Waringauthor=D.+M.+Andrewsauthor=P.+F.+Faulderauthor=V.+Flemingtonauthor=J.+C.+McKelvieauthor=S.+Mamanauthor=M.+Prestonauthor=P.+Rauboauthor=G.+R.+Robbauthor=K.+Robertsauthor=R.+Rowlinsonauthor=J.+M.+Smithauthor=M.+E.+Swarbrickauthor=I.+Treiniesauthor=J.+J.+G.+Winterauthor=R.+J.+Wood&title=Potent%2C+Selective+Small+Molecule+Inhibitors+of+Type+III+Phosphatidylinositol-4-Kinase+A-+but+Not+B-Inhibit+the+Phosphatidylinositol+Signaling+Cascade+and+Cancer+Cell+Proliferation&doi=10.1039%2FC3CC48391F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, selective small molecule inhibitors of type III phosphatidylinositol-4-kinase α- but not β-inhibit the phosphatidylinositol signaling cascade and cancer cell proliferation</span></div><div class="casAuthors">Waring, Michael J.; Andrews, David M.; Faulder, Paul F.; Flemington, Vikki; McKelvie, Jennifer C.; Maman, Sarita; Preston, Marian; Raubo, Piotr; Robb, Graeme R.; Roberts, Karen; Rowlinson, Rachel; Smith, James M.; Swarbrick, Martin E.; Treinies, Iris; Winter, Jon J. G.; Wood, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5388-5390</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of inhibitors of type III phosphatidylinositol-4-kinase were identified by high throughput screening and optimized to derive probe compds. that independently and selectively inhibit the α- and the β-isoforms with no significant activity towards related kinases in the pathway.  In a cellular environment, inhibition of the α- but not the β-subtype led to a redn. in phosphatidylinositol-4-phosphate and phosphatidylinositol-4,5-bisphosphate concn., causing inhibition of inositol-1-phosphate formation and inhibition of proliferation in a panel of cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDy9GsjuWbErVg90H21EOLACvtfcHk0lgKiAn2Lo5ikQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCjs7k%253D&md5=dd250429a80af1e974a28b281d801d62</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC3CC48391F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3CC48391F%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DFaulder%26aufirst%3DP.%2BF.%26aulast%3DFlemington%26aufirst%3DV.%26aulast%3DMcKelvie%26aufirst%3DJ.%2BC.%26aulast%3DMaman%26aufirst%3DS.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DJ.%2BM.%26aulast%3DSwarbrick%26aufirst%3DM.%2BE.%26aulast%3DTreinies%26aufirst%3DI.%26aulast%3DWinter%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26atitle%3DPotent%252C%2520Selective%2520Small%2520Molecule%2520Inhibitors%2520of%2520Type%2520III%2520Phosphatidylinositol-4-Kinase%2520A-%2520but%2520Not%2520B-Inhibit%2520the%2520Phosphatidylinositol%2520Signaling%2520Cascade%2520and%2520Cancer%2520Cell%2520Proliferation%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2014%26volume%3D50%26spage%3D5388%26epage%3D5390%26doi%3D10.1039%2FC3CC48391F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Spickler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laberge, M.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmeules, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellavance, É.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garneau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hucke, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyssen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neyts, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaillancourt, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Décor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauthier, A.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4-Kinase III Beta Is Essential for Replication of Human Rhinovirus and Its Inhibition Causes a Lethal Phenotype in Vivo</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">3358</span><span class="NLM_x">–</span> <span class="NLM_lpage">3368</span><span class="refDoi"> DOI: 10.1128/AAC.00303-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1128%2FAAC.00303-13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=3358-3368&author=C.+Spicklerauthor=J.+Lippensauthor=M.-K.+Labergeauthor=S.+Desmeulesauthor=%C3%89.+Bellavanceauthor=M.+Garneauauthor=T.+Guoauthor=O.+Huckeauthor=P.+Leyssenauthor=J.+Neytsauthor=F.+H.+Vaillancourtauthor=A.+D%C3%A9corauthor=J.+O%E2%80%99Mearaauthor=M.+Frantiauthor=A.+Gauthier&title=Phosphatidylinositol+4-Kinase+III+Beta+Is+Essential+for+Replication+of+Human+Rhinovirus+and+Its+Inhibition+Causes+a+Lethal+Phenotype+in+Vivo&doi=10.1128%2FAAC.00303-13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FAAC.00303-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00303-13%26sid%3Dliteratum%253Aachs%26aulast%3DSpickler%26aufirst%3DC.%26aulast%3DLippens%26aufirst%3DJ.%26aulast%3DLaberge%26aufirst%3DM.-K.%26aulast%3DDesmeules%26aufirst%3DS.%26aulast%3DBellavance%26aufirst%3D%25C3%2589.%26aulast%3DGarneau%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DT.%26aulast%3DHucke%26aufirst%3DO.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DVaillancourt%26aufirst%3DF.%2BH.%26aulast%3DD%25C3%25A9cor%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DJ.%26aulast%3DFranti%26aufirst%3DM.%26aulast%3DGauthier%26aufirst%3DA.%26atitle%3DPhosphatidylinositol%25204-Kinase%2520III%2520Beta%2520Is%2520Essential%2520for%2520Replication%2520of%2520Human%2520Rhinovirus%2520and%2520Its%2520Inhibition%2520Causes%2520a%2520Lethal%2520Phenotype%2520in%2520Vivo%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D3358%26epage%3D3368%26doi%3D10.1128%2FAAC.00303-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Okkenhaug, K.</span><span> </span><span class="NLM_article-title">Signaling by the Phosphoinositide 3-Kinase Family in Immune Cells</span> <span class="citation_source-journal">Annu. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">675</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1146/annurev-immunol-032712-095946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1146%2Fannurev-immunol-032712-095946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23330955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFClsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=675-704&author=K.+Okkenhaug&title=Signaling+by+the+Phosphoinositide+3-Kinase+Family+in+Immune+Cells&doi=10.1146%2Fannurev-immunol-032712-095946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling by the phosphoinositide 3-kinase family in immune cells</span></div><div class="casAuthors">Okkenhaug, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">675-704</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Phosphoinositide 3-kinases (PI3Ks) control many important aspects of immune cell development, differentiation, and function.  Mammals have eight PI3K catalytic subunits that are divided into three classes based on similarities in structure and function.  Specific roles for the class I PI3Ks have been broadly investigated and are relatively well understood, as is the function of their corresponding phosphatases.  More recently, specific roles for the class II and class III PI3Ks have emerged.  Through vertebrate evolution and in parallel with the evolution of adaptive immunity, there has been a dramatic increase not only in the genes for PI3K subunits but also in genes for phosphatases that act on 3-phosphoinositides and in 3-phosphoinositide-binding proteins.  Our understanding of the PI3Ks in immunity is guided by fundamental discoveries made in simpler model organisms as well as by appreciating new adaptations of this signaling module in mammals in general and in immune cells in particular.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIM9nlUNrG1rVg90H21EOLACvtfcHk0lgKiAn2Lo5ikQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFClsbc%253D&md5=619556fb1d852e274e17381865a248d3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032712-095946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032712-095946%26sid%3Dliteratum%253Aachs%26aulast%3DOkkenhaug%26aufirst%3DK.%26atitle%3DSignaling%2520by%2520the%2520Phosphoinositide%25203-Kinase%2520Family%2520in%2520Immune%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2013%26volume%3D31%26spage%3D675%26epage%3D704%26doi%3D10.1146%2Fannurev-immunol-032712-095946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Keaney, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sokup, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karur, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Britt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamann, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaMarche, M. J.</span><span> </span><span class="NLM_article-title">2-Alkyloxazoles as Potent and Selective PI4KIIIβ Inhibitors Demonstrating Inhibition of HCV Replication</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3718</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.bmcl.2014.07.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3714-3718&author=E.+P.+Keaneyauthor=M.+Connollyauthor=M.+Doblerauthor=R.+Karkiauthor=A.+Hondaauthor=S.+Sokupauthor=S.+Karurauthor=S.+Brittauthor=A.+Patnaikauthor=P.+Ramanauthor=L.+G.+Hamannauthor=B.+Wiedmannauthor=M.+J.+LaMarche&title=2-Alkyloxazoles+as+Potent+and+Selective+PI4KIII%CE%B2+Inhibitors+Demonstrating+Inhibition+of+HCV+Replication&doi=10.1016%2Fj.bmcl.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DKeaney%26aufirst%3DE.%2BP.%26aulast%3DConnolly%26aufirst%3DM.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DKarki%26aufirst%3DR.%26aulast%3DHonda%26aufirst%3DA.%26aulast%3DSokup%26aufirst%3DS.%26aulast%3DKarur%26aufirst%3DS.%26aulast%3DBritt%26aufirst%3DS.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DWiedmann%26aufirst%3DB.%26aulast%3DLaMarche%26aufirst%3DM.%2BJ.%26atitle%3D2-Alkyloxazoles%2520as%2520Potent%2520and%2520Selective%2520PI4KIII%25CE%25B2%2520Inhibitors%2520Demonstrating%2520Inhibition%2520of%2520HCV%2520Replication%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3714%26epage%3D3718%26doi%3D10.1016%2Fj.bmcl.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">iMOSFLM: a New Graphical Interface for Diffraction-Image Processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+New+Graphical+Interface+for+Diffraction-Image+Processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0lgQFcXygSnhJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520New%2520Graphical%2520Interface%2520for%2520Diffraction-Image%2520Processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgQFcXygSnhJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgQFcXygSnhJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0livD-wRM9l9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">The Phenix Software for Automated Determination of Macromolecular Structures</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2011.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1016%2Fj.ymeth.2011.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=21821126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=94-106&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=S.+Jainauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=The+Phenix+Software+for+Automated+Determination+of+Macromolecular+Structures&doi=10.1016%2Fj.ymeth.2011.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Phenix software for automated determination of macromolecular structures</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li-Wei; Jain, Swati; Kapral, Gary J.; Grosse Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-106</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  X-ray crystallog. is a crit. tool in the study of biol. systems.  It is able to provide information that has been a prerequisite to understanding the fundamentals of life.  It is also a method that is central to the development of new therapeutics for human disease.  Significant time and effort are required to det. and optimize many macromol. structures because of the need for manual interpretation of complex numerical data, often using many different software packages, and the repeated use of interactive three-dimensional graphics.  The Phenix software package has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on automation.  This has required the development of new algorithms that minimize or eliminate subjective input in favor of built-in expert-systems knowledge, the automation of procedures that are traditionally performed by hand, and the development of a computational framework that allows a tight integration between the algorithms.  The application of automated methods is particularly appropriate in the field of structural proteomics, where high throughput is desired.  Features in Phenix for the automation of exptl. phasing with subsequent model building, mol. replacement, structure refinement and validation are described and examples given of running Phenix from both the command line and graphical user interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgjdX_7M-NcLVg90H21EOLACvtfcHk0livD-wRM9l9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F&md5=ba8f207cab9f3597dc0c6db885667a3f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2011.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2011.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DThe%2520Phenix%2520Software%2520for%2520Automated%2520Determination%2520of%2520Macromolecular%2520Structures%26jtitle%3DMethods%26date%3D2011%26volume%3D55%26spage%3D94%26epage%3D106%26doi%3D10.1016%2Fj.ymeth.2011.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.Refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.Refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0livD-wRM9l9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.Refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): a Tool for Ligand Coordinate and Restraint Generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0li-ZlMzFtMe7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Knight, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balla, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K.</span><span> </span><span class="NLM_article-title">A Membrane Capture Assay for Lipid Kinase Activity</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2459</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1038/nprot.2007.361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1038%2Fnprot.2007.361" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2459-2466&author=Z.+Knightauthor=M.+Feldmanauthor=A.+Ballaauthor=T.+Ballaauthor=K.+Shokat&title=A+Membrane+Capture+Assay+for+Lipid+Kinase+Activity&doi=10.1038%2Fnprot.2007.361"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.361%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%26aulast%3DFeldman%26aufirst%3DM.%26aulast%3DBalla%26aufirst%3DA.%26aulast%3DBalla%26aufirst%3DT.%26aulast%3DShokat%26aufirst%3DK.%26atitle%3DA%2520Membrane%2520Capture%2520Assay%2520for%2520Lipid%2520Kinase%2520Activity%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2459%26epage%3D2466%26doi%3D10.1038%2Fnprot.2007.361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Seiler, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surapaneni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedillo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Throop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiacco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labaer, J.</span><span> </span><span class="NLM_article-title">DNASU Plasmid and PSI:Biology-Materials Repositories: Resources to Accelerate Biological Research</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1253</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1260</span><span class="refDoi"> DOI: 10.1093/nar/gkt1060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1093%2Fnar%2Fgkt1060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=24225319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1ai" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1253-D1260&author=C.+Y.+Seilerauthor=J.+G.+Parkauthor=A.+Sharmaauthor=P.+Hunterauthor=P.+Surapaneniauthor=C.+Sedilloauthor=J.+Fieldauthor=R.+Algarauthor=A.+Priceauthor=J.+Steelauthor=A.+Throopauthor=M.+Fiaccoauthor=J.+Labaer&title=DNASU+Plasmid+and+PSI%3ABiology-Materials+Repositories%3A+Resources+to+Accelerate+Biological+Research&doi=10.1093%2Fnar%2Fgkt1060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">DNASU plasmid and PSI:Biology-Materials repositories: resources to accelerate biological research</span></div><div class="casAuthors">Seiler, Catherine Y.; Park, Jin G.; Sharma, Amit; Hunter, Preston; Surapaneni, Padmini; Sedillo, Casey; Field, James; Algar, Rhys; Price, Andrea; Steel, Jason; Throop, Andrea; Fiacco, Michael; LaBaer, Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1253-D1260</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The mission of the DNASU Plasmid Repository is to accelerate research by providing high-quality, annotated plasmid samples and online plasmid resources to the research community through the curated DNASU database, website and repository http://dnasu.asu.edu or http://dnasu.org.  The collection includes plasmids from grant-funded, high-throughput cloning projects performed in our lab., plasmids from external researchers, and large collections from consortia such as the ORFeome Collaboration and the NIGMS-funded Protein Structure Initiative: Biol. PSI:Biol.  Through DNASU, researchers can search for and access detailed information about each plasmid such as the full length gene insert sequence, vector information, assocd. publications, and links to external resources that provide addnl. protein annotations and exptl. protocols.  Plasmids can be requested directly through the DNASU website.  DNASU and the PSI:Biol.-Materials Repositories were previously described in the 2010 NAR Database Issue Cormier, C.Y., Mohr, S.E., Zuo, D., Hu, Y., Rolfs, A., Kramer, J., Taycher, E., Kelley, F., Fiacco, M., Turnbull, G. et al. 2010 Protein Structure Initiative Material Repository: an open shared public resource of structural genomics plasmids for the biol. community.  Nucleic Acids Res., 38, D743-D749. In this update we will describe the plasmid collection and highlight the new features in the website redesign, including new browse/search options, plasmid annotations and a dynamic vector mapping feature that was developed in collaboration with LabGenius.  Overall, these plasmid resources continue to enable research with the goal of elucidating the role of proteins in both normal biol. processes and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6aWb1YkMc-7Vg90H21EOLACvtfcHk0li-ZlMzFtMe7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1ai&md5=423692499b7abc1b326bef66c939c722</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1060%26sid%3Dliteratum%253Aachs%26aulast%3DSeiler%26aufirst%3DC.%2BY.%26aulast%3DPark%26aufirst%3DJ.%2BG.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DHunter%26aufirst%3DP.%26aulast%3DSurapaneni%26aufirst%3DP.%26aulast%3DSedillo%26aufirst%3DC.%26aulast%3DField%26aufirst%3DJ.%26aulast%3DAlgar%26aufirst%3DR.%26aulast%3DPrice%26aufirst%3DA.%26aulast%3DSteel%26aufirst%3DJ.%26aulast%3DThroop%26aufirst%3DA.%26aulast%3DFiacco%26aufirst%3DM.%26aulast%3DLabaer%26aufirst%3DJ.%26atitle%3DDNASU%2520Plasmid%2520and%2520PSI%253ABiology-Materials%2520Repositories%253A%2520Resources%2520to%2520Accelerate%2520Biological%2520Research%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1253%26epage%3DD1260%26doi%3D10.1093%2Fnar%2Fgkt1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Park, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murthy, T. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantley, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labaer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harlow, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brizuela, L.</span><span> </span><span class="NLM_article-title">Building a Human Kinase Gene Repository: Bioinformatics, Molecular Cloning, and Functional Validation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">8114</span><span class="NLM_x">–</span> <span class="NLM_lpage">8119</span><span class="refDoi"> DOI: 10.1073/pnas.0503141102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1073%2Fpnas.0503141102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8114-8119&author=J.+Parkauthor=Y.+Huauthor=T.+V.+S.+Murthyauthor=F.+Vannbergauthor=B.+Shenauthor=A.+Rolfsauthor=J.+E.+Huttiauthor=L.+C.+Cantleyauthor=J.+Labaerauthor=E.+Harlowauthor=L.+Brizuela&title=Building+a+Human+Kinase+Gene+Repository%3A+Bioinformatics%2C+Molecular+Cloning%2C+and+Functional+Validation&doi=10.1073%2Fpnas.0503141102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0503141102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0503141102%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DMurthy%26aufirst%3DT.%2BV.%2BS.%26aulast%3DVannberg%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DRolfs%26aufirst%3DA.%26aulast%3DHutti%26aufirst%3DJ.%2BE.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DLabaer%26aufirst%3DJ.%26aulast%3DHarlow%26aufirst%3DE.%26aulast%3DBrizuela%26aufirst%3DL.%26atitle%3DBuilding%2520a%2520Human%2520Kinase%2520Gene%2520Repository%253A%2520Bioinformatics%252C%2520Molecular%2520Cloning%252C%2520and%2520Functional%2520Validation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8114%26epage%3D8119%26doi%3D10.1073%2Fpnas.0503141102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Marukian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritola, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charles, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dustin, L. B.</span><span> </span><span class="NLM_article-title">Cell Culture-Produced Hepatitis C Virus Does Not Infect Peripheral Blood Mononuclear Cells</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1850</span><span class="refDoi"> DOI: 10.1002/hep.22550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1002%2Fhep.22550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1843-1850&author=S.+Marukianauthor=C.+T.+Jonesauthor=L.+Andrusauthor=M.+J.+Evansauthor=K.+D.+Ritolaauthor=E.+D.+Charlesauthor=C.+M.+Riceauthor=L.+B.+Dustin&title=Cell+Culture-Produced+Hepatitis+C+Virus+Does+Not+Infect+Peripheral+Blood+Mononuclear+Cells&doi=10.1002%2Fhep.22550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fhep.22550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.22550%26sid%3Dliteratum%253Aachs%26aulast%3DMarukian%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DC.%2BT.%26aulast%3DAndrus%26aufirst%3DL.%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DRitola%26aufirst%3DK.%2BD.%26aulast%3DCharles%26aufirst%3DE.%2BD.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DDustin%26aufirst%3DL.%2BB.%26atitle%3DCell%2520Culture-Produced%2520Hepatitis%2520C%2520Virus%2520Does%2520Not%2520Infect%2520Peripheral%2520Blood%2520Mononuclear%2520Cells%26jtitle%3DHepatology%26date%3D2008%26volume%3D48%26spage%3D1843%26epage%3D1850%26doi%3D10.1002%2Fhep.22550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Cho, N.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fram, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sklan, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elazar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koytak, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kersten, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanazawa, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glenn, J. S.</span><span> </span><span class="NLM_article-title">Phosphatidylinositol 4,5-Bisphosphate Is an HCV NS5A Ligand and Mediates Replication of the Viral Genome</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">148</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span><span class="refDoi"> DOI: 10.1053/j.gastro.2014.11.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=10.1053%2Fj.gastro.2014.11.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2015&pages=616-625&author=N.-J.+Choauthor=C.+Leeauthor=P.+S.+Pangauthor=E.+A.+Phamauthor=B.+Framauthor=K.+Nguyenauthor=A.+Xiongauthor=E.+H.+Sklanauthor=M.+Elazarauthor=E.+S.+Koytakauthor=C.+Kerstenauthor=K.+K.+Kanazawaauthor=C.+W.+Frankauthor=J.+S.+Glenn&title=Phosphatidylinositol+4%2C5-Bisphosphate+Is+an+HCV+NS5A+Ligand+and+Mediates+Replication+of+the+Viral+Genome&doi=10.1053%2Fj.gastro.2014.11.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2014.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2014.11.043%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DN.-J.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DPang%26aufirst%3DP.%2BS.%26aulast%3DPham%26aufirst%3DE.%2BA.%26aulast%3DFram%26aufirst%3DB.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DXiong%26aufirst%3DA.%26aulast%3DSklan%26aufirst%3DE.%2BH.%26aulast%3DElazar%26aufirst%3DM.%26aulast%3DKoytak%26aufirst%3DE.%2BS.%26aulast%3DKersten%26aufirst%3DC.%26aulast%3DKanazawa%26aufirst%3DK.%2BK.%26aulast%3DFrank%26aufirst%3DC.%2BW.%26aulast%3DGlenn%26aufirst%3DJ.%2BS.%26atitle%3DPhosphatidylinositol%25204%252C5-Bisphosphate%2520Is%2520an%2520HCV%2520NS5A%2520Ligand%2520and%2520Mediates%2520Replication%2520of%2520the%2520Viral%2520Genome%26jtitle%3DGastroenterology%26date%3D2015%26volume%3D148%26spage%3D616%26epage%3D625%26doi%3D10.1053%2Fj.gastro.2014.11.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swindells, M. B.</span><span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2778</span><span class="NLM_x">–</span> <span class="NLM_lpage">2786</span><span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-Protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0lgilae9zgY45g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-Protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D0L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D0L','PDB','4D0L'); return false;">PDB: 4D0L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X6J','PDB','2X6J'); return false;">PDB: 2X6J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CHZ','PDB','2CHZ'); return false;">PDB: 2CHZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUQ','PDB','5EUQ'); return false;">PDB: 5EUQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01311">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_87658"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01311">10.1021/acs.jmedchem.5b01311</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Compound characterization; inhibitor assays of compounds <b>9</b> and <b>10</b> across a panel of lipid kinases; antiviral efficacy and toxicity of selected inhibitors; binding of compound <b>9</b> in the active site pocket of PI4KIIIβ (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES strings, IC<sub>50</sub> data, and CC<sub>50</sub> data for <b>1</b>–<b>10</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_001.pdf">jm5b01311_si_001.pdf (548.94 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01311/suppl_file/jm5b01311_si_002.csv">jm5b01311_si_002.csv (1.23 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure factors and PDB coordinates of PI4K bound to compound <b>9</b> have been deposited at the protein databank (PDB) with the coordinates <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUQ">5EUQ</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01311&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01311%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01311" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679970650c16d1db","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
