
<html lang="en"     class="pb-page"  data-request-id="5bb81522-b258-4f9c-a0d6-c536b5eb62a0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.7b01647"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of 7-Oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine Derivatives as Potent, Orally Available, and Brain-Penetrating Receptor Interacting Protein 1 (RIP1) Kinase Inhibitors: Analysis of Structure–Kinetic Relationships" /></meta><meta name="dc.Creator" content="Masato  Yoshikawa" /></meta><meta name="dc.Creator" content="Morihisa  Saitoh" /></meta><meta name="dc.Creator" content="Taisuke  Katoh" /></meta><meta name="dc.Creator" content="Tomohiro  Seki" /></meta><meta name="dc.Creator" content="Simone V.  Bigi" /></meta><meta name="dc.Creator" content="Yuji  Shimizu" /></meta><meta name="dc.Creator" content="Tsuyoshi  Ishii" /></meta><meta name="dc.Creator" content="Takuro  Okai" /></meta><meta name="dc.Creator" content="Masako  Kuno" /></meta><meta name="dc.Creator" content="Harumi  Hattori" /></meta><meta name="dc.Creator" content="Etsuro  Watanabe" /></meta><meta name="dc.Creator" content="Kumar S.  Saikatendu" /></meta><meta name="dc.Creator" content="Hua  Zou" /></meta><meta name="dc.Creator" content="Masanori  Nakakariya" /></meta><meta name="dc.Creator" content="Takayuki  Tatamiya" /></meta><meta name="dc.Creator" content="Yoshihisa  Nakada" /></meta><meta name="dc.Creator" content="Takatoshi  Yogo" /></meta><meta name="dc.Description" content="We report the discovery of 7-oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine derivatives as a novel class of receptor interacting protein 1 (RIP1) kinase inhibitors. On the basis of the overlay s..." /></meta><meta name="Description" content="We report the discovery of 7-oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine derivatives as a novel class of receptor interacting protein 1 (RIP1) kinase inhibitors. On the basis of the overlay s..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 27, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01647" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01647" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01647" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01647" /></link>
        
    
    

<title>Discovery of 7-Oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine Derivatives as Potent, Orally Available, and Brain-Penetrating Receptor Interacting Protein 1 (RIP1) Kinase Inhibitors: Analysis of Structure–Kinetic Relationships | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01647" /></meta><meta property="og:title" content="Discovery of 7-Oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine Derivatives as Potent, Orally Available, and Brain-Penetrating Receptor Interacting Protein 1 (RIP1) Kinase Inhibitors: Analysis of Structure–Kinetic Relationships" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0027.jpeg" /></meta><meta property="og:description" content="We report the discovery of 7-oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine derivatives as a novel class of receptor interacting protein 1 (RIP1) kinase inhibitors. On the basis of the overlay study between HTS hit 10 and GSK2982772 (6) in RIP1 kinase, we designed and synthesized a novel class of RIP1 kinase inhibitor 11 possessing moderate RIP1 kinase inhibitory activity and P-gp mediated efflux. The optimization of the core structure and the exploration of appropriate substituents utilizing SBDD approach led to the discovery of 22, a highly potent, orally available, and brain-penetrating RIP1 kinase inhibitor with excellent PK profiles. Compound 22 significantly suppressed necroptotic cell death both in mouse and human cells. Oral administration of 22 (10 mg/kg, bid) attenuated disease progression in the mouse experimental autoimmune encephalomyelitis (EAE) model of multiple sclerosis (MS). Moreover, analysis of structure–kinetic relationship (SKR) for our novel chemical series was also discussed." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01647"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01647">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01647&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01647&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01647&amp;href=/doi/10.1021/acs.jmedchem.7b01647" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2384-2409</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01641" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01155" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of 7-Oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine Derivatives as Potent, Orally Available, and Brain-Penetrating Receptor Interacting Protein 1 (RIP1) Kinase Inhibitors: Analysis of Structure–Kinetic Relationships</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Masato Yoshikawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masato Yoshikawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masato++Yoshikawa">Masato Yoshikawa</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7175-1173" title="Orcid link">http://orcid.org/0000-0002-7175-1173</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Morihisa Saitoh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Morihisa Saitoh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Morihisa++Saitoh">Morihisa Saitoh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Taisuke Katoh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Taisuke Katoh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Taisuke++Katoh">Taisuke Katoh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tomohiro Seki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tomohiro Seki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tomohiro++Seki">Tomohiro Seki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simone V. Bigi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simone V. Bigi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Takeda Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simone+V.++Bigi">Simone V. Bigi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuji Shimizu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuji Shimizu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuji++Shimizu">Yuji Shimizu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tsuyoshi Ishii</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tsuyoshi Ishii</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tsuyoshi++Ishii">Tsuyoshi Ishii</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Takuro Okai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takuro Okai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takuro++Okai">Takuro Okai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masako Kuno</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masako Kuno</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masako++Kuno">Masako Kuno</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Harumi Hattori</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harumi Hattori</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harumi++Hattori">Harumi Hattori</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Etsuro Watanabe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Etsuro Watanabe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Etsuro++Watanabe">Etsuro Watanabe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kumar S. Saikatendu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kumar S. Saikatendu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Takeda Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kumar+S.++Saikatendu">Kumar S. Saikatendu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hua Zou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hua Zou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Takeda Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hua++Zou">Hua Zou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masanori Nakakariya</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masanori Nakakariya</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masanori++Nakakariya">Masanori Nakakariya</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Takayuki Tatamiya</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takayuki Tatamiya</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takayuki++Tatamiya">Takayuki Tatamiya</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yoshihisa Nakada</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoshihisa Nakada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoshihisa++Nakada">Yoshihisa Nakada</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Takatoshi Yogo</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takatoshi Yogo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: (+81)-466-32-1076. Fax: (+81)-466-29-4454. E-mail: <a href="/cdn-cgi/l/email-protection#dda9bcb6bca9b2aeb5b4f3a4b2bab29da9bcb6b8b9bcf3beb2b0"><span class="__cf_email__" data-cfemail="89fde8e2e8fde6fae1e0a7f0e6eee6c9fde8e2ecede8a7eae6e4">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takatoshi++Yogo">Takatoshi Yogo</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01647&amp;href=/doi/10.1021%2Facs.jmedchem.7b01647" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2384–2409</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 27, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 November 2017</li><li><span class="item_label"><b>Published</b> online</span>27 February 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 March 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01647" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01647</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2384%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMasato%2BYoshikawa%252C%2BMorihisa%2BSaitoh%252C%2BTaisuke%2BKatoh%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D6%26contentID%3Dacs.jmedchem.7b01647%26title%3DDiscovery%2Bof%2B7-Oxo-2%252C4%252C5%252C7-tetrahydro-6H-pyrazolo%255B3%252C4-c%255Dpyridine%2BDerivatives%2Bas%2BPotent%252C%2BOrally%2BAvailable%252C%2Band%2BBrain-Penetrating%2BReceptor%2BInteracting%2BProtein%2B1%2B%2528RIP1%2529%2BKinase%2BInhibitors%253A%2BAnalysis%2Bof%2BStructure%25E2%2580%2593Kinetic%2BRelationships%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2409%26publicationDate%3DMarch%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01647"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6177</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">35</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01647" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of 7-Oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine Derivatives as Potent, Orally Available, and Brain-Penetrating Receptor Interacting Protein 1 (RIP1) Kinase Inhibitors: Analysis of Structure–Kinetic Relationships&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Masato&quot;,&quot;last_name&quot;:&quot;Yoshikawa&quot;},{&quot;first_name&quot;:&quot;Morihisa&quot;,&quot;last_name&quot;:&quot;Saitoh&quot;},{&quot;first_name&quot;:&quot;Taisuke&quot;,&quot;last_name&quot;:&quot;Katoh&quot;},{&quot;first_name&quot;:&quot;Tomohiro&quot;,&quot;last_name&quot;:&quot;Seki&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;V. Bigi&quot;},{&quot;first_name&quot;:&quot;Yuji&quot;,&quot;last_name&quot;:&quot;Shimizu&quot;},{&quot;first_name&quot;:&quot;Tsuyoshi&quot;,&quot;last_name&quot;:&quot;Ishii&quot;},{&quot;first_name&quot;:&quot;Takuro&quot;,&quot;last_name&quot;:&quot;Okai&quot;},{&quot;first_name&quot;:&quot;Masako&quot;,&quot;last_name&quot;:&quot;Kuno&quot;},{&quot;first_name&quot;:&quot;Harumi&quot;,&quot;last_name&quot;:&quot;Hattori&quot;},{&quot;first_name&quot;:&quot;Etsuro&quot;,&quot;last_name&quot;:&quot;Watanabe&quot;},{&quot;first_name&quot;:&quot;Kumar&quot;,&quot;last_name&quot;:&quot;S. Saikatendu&quot;},{&quot;first_name&quot;:&quot;Hua&quot;,&quot;last_name&quot;:&quot;Zou&quot;},{&quot;first_name&quot;:&quot;Masanori&quot;,&quot;last_name&quot;:&quot;Nakakariya&quot;},{&quot;first_name&quot;:&quot;Takayuki&quot;,&quot;last_name&quot;:&quot;Tatamiya&quot;},{&quot;first_name&quot;:&quot;Yoshihisa&quot;,&quot;last_name&quot;:&quot;Nakada&quot;},{&quot;first_name&quot;:&quot;Takatoshi&quot;,&quot;last_name&quot;:&quot;Yogo&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;2384-2409&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01647&quot;},&quot;abstract&quot;:&quot;We report the discovery of 7-oxo-2,4,5,7-tetrahydro-6H-pyrazolo[3,4-c]pyridine derivatives as a novel class of receptor interacting protein 1 (RIP1) kinase inhibitors. On the basis of the overlay study between HTS hit 10 and GSK2982772 (6) in RIP1 kinase, we designed and synthesized a novel class of RIP1 kinase inhibitor 11 possessing moderate RIP1 kinase inhibitory activity and P-gp mediated efflux. The optimization of the core structure and the exploration of appropriate substituents utilizing SBDD approach led to the discovery of 22, a highly potent, orally available, and brain-penetrating RIP1 kinase inhibitor with excellent PK profiles. Compound 22 significantly suppressed necroptotic cell death both in mouse and human cells. Oral administration of 22 (10 mg/kg, bid) attenuated disease progression in the mouse experimental autoimmune encephalomyelitis (EAE) model of multiple sclerosis (MS). Moreover, analysis of structure–kinetic relationship (SKR) for our novel chemical series was also discussed.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01647&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01647" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01647&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01647" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01647&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01647" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01647&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01647&amp;href=/doi/10.1021/acs.jmedchem.7b01647" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01647" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01647" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01647%26sid%3Dliteratum%253Aachs%26pmid%3D29485864%26genre%3Darticle%26aulast%3DYoshikawa%26date%3D2018%26atitle%3DDiscovery%2Bof%2B7-Oxo-2%252C4%252C5%252C7-tetrahydro-6H-pyrazolo%255B3%252C4-c%255Dpyridine%2BDerivatives%2Bas%2BPotent%252C%2BOrally%2BAvailable%252C%2Band%2BBrain-Penetrating%2BReceptor%2BInteracting%2BProtein%2B1%2B%2528RIP1%2529%2BKinase%2BInhibitors%253A%2BAnalysis%2Bof%2BStructure%25E2%2580%2593Kinetic%2BRelationships%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D6%26spage%3D2384%26epage%3D2409%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291264" title="Kinase inhibitors">Kinase inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/jmcmar.2018.61.issue-6/20180322/jmcmar.2018.61.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report the discovery of 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives as a novel class of receptor interacting protein 1 (RIP1) kinase inhibitors. On the basis of the overlay study between HTS hit <b>10</b> and GSK2982772 (<b>6</b>) in RIP1 kinase, we designed and synthesized a novel class of RIP1 kinase inhibitor <b>11</b> possessing moderate RIP1 kinase inhibitory activity and P-gp mediated efflux. The optimization of the core structure and the exploration of appropriate substituents utilizing SBDD approach led to the discovery of <b>22</b>, a highly potent, orally available, and brain-penetrating RIP1 kinase inhibitor with excellent PK profiles. Compound <b>22</b> significantly suppressed necroptotic cell death both in mouse and human cells. Oral administration of <b>22</b> (10 mg/kg, bid) attenuated disease progression in the mouse experimental autoimmune encephalomyelitis (EAE) model of multiple sclerosis (MS). Moreover, analysis of structure–kinetic relationship (SKR) for our novel chemical series was also discussed.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57103" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57103" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A process of programmed necrosis, termed necroptosis, has recently been recognized as an important physiological process that involves cell death, innate immunity, and inflammation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Receptor interacting protein 1 (RIP1) kinase is a crucial upstream regulator of necroptosis, and association of RIP1 kinase with a variety of pathologies, such as ischemic injury, inflammatory diseases, and neurodegenerative diseases, has been investigated.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><div class="NLM_p">Neurodegenerative diseases, such as multiple sclerosis (MS),<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Parkinson’s disease (PD),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and amyotrophic lateral sclerosis (ALS),<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> have recently been intensively studied to understand their relationship with the necroptosis pathway. Ofengeim et al. showed the involvement of RIP1 kinase and necroptosis in mediating the deleterious process in MS by using both animal models and human tissues from MS patients.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> RIP1-mediated axonal degradation has also been demonstrated in ALS. Thus, inhibitors of RIP1 kinase have the potential to provide new therapeutic opportunities to neurodegenerative diseases with significant unmet medical needs.</div><div class="NLM_p">The first published RIP1 kinase inhibitor, Nec-1 (<b>1</b>),<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> was discovered by Degterev et al. in 2005, with a subsequent disclosure of an analogue, Nec-1s (<b>2</b>),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> with improved metabolic stability. Both have been widely used to elucidate the biology of the necroptosis pathway. Over the recent years, additional series of RIP1 kinase inhibitors have been identified as illustrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> such as a type-II<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> RIP1 kinase inhibitor <b>3</b><a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and a hybrid-type RIP1 kinase inhibitor <b>4</b>,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> which was designed based on <b>2</b> and ponatinib.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Among them, benzo[<i>b</i>][1,4]oxazepin-4-one analogues,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> represented by <b>5</b>, comprise the most advanced series, with the optimized analogue <b>6</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> currently being evaluated in human clinical trials for inflammatory diseases such as ulcerative colitis, rheumatoid arthritis, and psoriasis. However, despite the attractiveness of RIP1 kinase inhibitors as a therapeutic option for neurodegenerative disorders, the chemical series reported so far are not suitable for clinical development to treat CNS related disorders because of either low oral bioavailability and/or low brain permeability. For example, the necrostatin series (<b>2</b>) has shown efficacy in animal models of MS or ALS by oral administration, but its low metabolic stability has likely hindered its clinical development. Compound <b>8</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> has been detected in mouse brains but only after intravenous injection. The brain penetration or the in vivo efficacy of the other compounds (<b>3</b>–<b>7</b>)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> in neurodegenerative diseases have not been reported to date. Thus, to develop RIP1 kinase inhibitors targeting CNS diseases, we focused on the development of a novel chemical series of RIP1 kinase inhibitors with good oral bioavailability and brain-penetrating properties for treatment of neurodegenerative diseases.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0001.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of reported RIP1 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Herein, we report the discovery of (3<i>S</i>)-3-(2-benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carbonitrile (<b>22</b>) as a potent, orally bioavailable, and brain-penetrating RIP1 kinase inhibitor. The following strategies were used to identify <b>22</b>: (i) optimization of core structures for appropriate brain penetration in light of favorable physicochemical properties such as topological polar surface area (TPSA)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and the number of hydrogen-bonding donors (HBDs), (ii) exploration of potent substituents in back pockets by utilizing structure-based drug design (SBDD), and (iii) introduction of substituents toward the solvent-accessible region to control the favorable ADMET profiles. Compound <b>22</b> exhibited potent cellular activity both in humans and mice and an excellent pharmacokinetic (PK) profile with brain exposure in mice after oral administration. Compound <b>22</b> was efficacious in experimental autoimmune encephalomyelitis (EAE),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> which is a model of human inflammatory demyelinating disease such as MS. Moreover, in the course of optimization, kinetic profiles of all synthesized compounds were evaluated to analyze structure–kinetic relationships (SKRs). We also discuss the SKR of our novel chemical series of RIP1 kinase inhibitors later in this paper.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21147" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21147" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Assay Development</h3><div class="NLM_p">The kinetics of the interaction between a ligand and its target has been recognized as an important factor for lead generation/optimization,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> and long drug–target residence time is often correlated with potent in vivo efficacy.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> In the case of RIP1 kinase inhibitors, the benzoxazepinone-based compounds have a long drug–target residence time (<i>t</i><sub>1/2</sub> = 112 min for <b>6</b>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which could translate into prolonged efficacy in cellular assays and potentially in vivo. These results led us to analyze kinetic profiles including <i>t</i><sub>1/2</sub> in the course of the optimization to identity potent and brain-penetrating RIP1 kinase inhibitors. Measurement of the binding affinity for RIP1 kinase and kinetic profiles was performed by using a time-resolved fluorescence resonance energy transfer (TR-FRET) assay and a novel fluorescent probe, 3-(3-((3-(4-amino-5-(4-(3-(2-fluoro-5-(trifluoromethyl)phenyl)ureido)phenyl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-7-yl)propyl)amino)-3-oxopropyl)-5,5-difluoro-7,9-dimethyl-5<i>H</i>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-4-ium-5-uide (<b>9</b>), described in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> (the synthesis of <b>9</b> is described in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Supporting Information</a>). By using the TR-FRET assay, a high-throughput assay format was achieved sufficiently to evaluate all synthesized compounds. The binding affinity (p<i>K</i><sub>i</sub>) was calculated as the sum of log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub>. The drug–target residence time (<i>t</i><sub>1/2</sub>) was defined as ln2/<i>k</i><sub>off</sub>. Regarding compounds that quickly dissociate from the target (<i>t</i><sub>1/2</sub> < 5 min), the kinetic profiles could not be precisely evaluated because of the detection limit of this TR-FRET assay.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0002.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of fluorescent probe <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Lead Generation</h3><div class="NLM_p">To identify good starting points for lead optimization and subsequent development, we performed high-throughput screening (HTS) against RIP1 kinase by using our compound library and identified a benzimidazole derivative <b>10</b> (p<i>K</i><sub>i</sub> = 7.20) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Reported RIP1 kinase inhibitors can be classified into three categories based on the binding modes: (i) type-II class<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> targeting “DFG-out (or DLG-out in the case of RIP1 kinase)” conformation of kinase (e.g., <b>3</b>), (ii) type-III (allosteric) class<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (e.g., <b>1</b>, benzo[<i>b</i>][1,4]oxazepin-4-one series <b>5</b> and <b>6</b>), or (iii) hybrid type <b>4</b><a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). To clarify the binding mode of <b>10</b>, an X-ray co-crystal structure of <b>10</b> with RIP1 kinase was obtained (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The X-ray co-crystal structure revealed that compound <b>10</b> had no hinge-binding interaction and occupied an allosteric lipophilic pocket at the back of the ATP binding site, making a hydrogen-bonding interaction with the amide NH of Asp156 at a distance of 2.8 Å, which indicated that the binding mode of compound <b>10</b> was type-III class similar to that of the Nec-1 and benzo[<i>b</i>][1,4]oxazepin-4-one series. The overlaid image of <b>10</b> and the benzo[<i>b</i>][1,4]oxazepin-4-one derivative <b>6</b> in RIP1 kinase showed that the compounds occupied the same space and that the benzyl group and the 5-membered heteroaromatic ring of compounds <b>6</b> and <b>10</b> were well-overlapped (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). This result led us to design and synthesize a novel class of RIP1 kinase inhibitor <b>11</b>, possessing a 4-oxo-6,7-dihydro-1<i>H</i>-imidazo[4,5-<i>c</i>]pyridine moiety as a core structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As a result, compound <b>11</b> showed moderate RIP1 kinase inhibitory activity (p<i>K</i><sub>i</sub> = 6.93). Parent compound <b>6</b> had potent RIP1 kinase inhibitory activity with excellent in vitro and in vivo profiles with highly favorable physicochemical and ADMET properties. However, <b>6</b> was reported to have low brain penetration,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> presumably due to active extrusion of drug from the brain via the efflux drug transporter P-glycoprotein (P-gp). On the other hand, compound <b>11</b> showed a better P-gp-mediated efflux (ER = 1.9) than that of <b>6</b> (ER = 3.1), likely reflecting the decreased TPSA and HBD.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> On the basis of this result, further 6,5-bicyclic core structures were investigated as replacements of the 4-oxo-6,7-dihydro-1<i>H</i>-imidazo[4,5-<i>c</i>]pyridine maintaining the favorable physicochemical parameters (TPSA < 90 Å<sup>2</sup>, HBD = 0) for brain exposure<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> and enhancing RIP1 kinase inhibitory activity.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0003.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. HTS hit compound <b>10</b>. (A) Structure of compound <b>10</b> and <i>K</i><sub>i</sub> for human RIP1 kinase. (B) Co-crystal structure of <b>10</b> with hRIP1 kinase. (C) Overlay study between compounds <b>6</b> (yellow) and <b>10</b> (white) in hRIP1 kinase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0004.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design concept for a new class of brain-penetrating RIP1 kinase inhibitors. (a) Binding affinity for human RIP1 kinase. (b) Calculated by Daylight. (c) Efflux ratio in P-gp-overexpressing cells at 1 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The results of modification of the core structure are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Triazole derivative <b>12</b>, which was synthesized from <b>6</b> and removed HBD, showed dramatically decreased P-gp-mediated efflux and enhanced binding affinity (p<i>K</i><sub>i</sub>) compared to those of <b>11</b>. Although imidazole derivative <b>13</b>, which has a nitrogen atom on a bridgehead, maintained moderate potency, <b>13</b> exhibited a higher ER than that of <b>12</b> despite reducing TPSA, presumably due to the basicity of the imidazole moiety. Pyrazole derivatives <b>14</b> and <b>15</b> also had a lower ER. These results in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> indicated that our design strategy of lowering TPSA and removing HBD was effective in reducing ER. Compound <b>14</b> showed the strongest binding affinity for RIP1 kinase and longest drug–target residence time. Accordingly, the 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine moiety of <b>14</b> (p<i>K</i><sub>i</sub> = 8.41, <i>t</i><sub>1/2</sub> = 59 min, ER = 0.7) was identified as the most suitable core structure for further optimization.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Modification of Core Structure</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0019.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0020.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Calculated by Daylight.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Efflux ratios in P-gp-overexpressing cells at 1 μM.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last"><i>t</i><sub>1/2</sub> = ln2/<i>k</i><sub>off</sub>.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Not determined.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Lead Optimization</h3><div class="NLM_p">A docking model of <b>14</b> with RIP1 kinase was developed to utilize SBDD for further optimization (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). This model showed that the benzyl group deeply occupied the allosteric hydrophobic pocket (we refer to this pocket as deep pocket 1, DP1) at the back of the ATP binding site. The DP1 was mainly constructed by the side chains of lipophilic amino acid residues such as Leu129 and Val134. According to the X-ray co-crystal structure of <b>2</b> with RIP1 kinase, a hydrogen-bonding interaction was observed between the OH of Ser161 and NH of the indole in <b>2</b> in DP1. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, a small unoccupied pocket, which is defined by Met67 and Leu70, was found around the 3-position of the core structure (referred to as deep pocket 2, DP2). The docking model also suggested that the 7- and 8-positions of the benzoxazepinone ring were oriented toward the solvent-accessible region. On the basis of these analyses, optimization of <b>14</b> was continued by (i) investigation of favorable substituents in DP1 (R<sup>1</sup>) in view of the lipophilicity, size, or hydrogen bonding with OH of Ser161, (ii) exploration of substituents in DP2 (R<sup>2</sup>) that can efficiently occupy the small lipophilic pocket, and (iii) introduction of an appropriate substituent (R<sup>3</sup>) into the 7- or 8-positions of the benzoxazepinone ring to control the favorable ADMET profiles (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0005.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Synthetic strategy based on the docking model. (A) The docking study between <b>14</b> and RIP1 kinase. The DLG motif including Asp156 in the activation loop was omitted for simplicity. (B) Image of the protein surface around DP2. (C) Schematic illustration of <b>14</b> in RIP1 kinase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">First, modification of the substituent R<sup>1</sup> was conducted. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> (the detailed SKR analysis for all synthesized compounds will be discussed (vide infra)). Introduction of a fluoro group into the <i>o</i>-position resulted in enhanced binding affinity (compound <b>16a</b>), whereas introduction into the <i>m</i>- and <i>p</i>-positions (compounds <b>16b</b>,<b>c</b>) decreased the affinity compared to that of the nonsubstituted analogue <b>14</b>. An additional fluoro group was introduced into the second <i>o</i>-position of <b>16a</b>, yielding compound <b>16d</b>, increasing the binding affinity. Replacement of the phenyl group with a cyclobutyl group resulted in reduced binding affinity. To direct a hydrogen-bonding interaction toward the OH of Ser161, substituents possessing a hydrogen-bonding acceptor (HBA), such as heteroaromatic rings and sulfone groups, were introduced. Pyridine analogues <b>16f</b> and <b>16g</b> exhibited 10-fold lower <i>K</i><sub>i</sub> values than that of <b>14</b>, whereas introduction of sulfone groups (<b>16h</b> and <b>16i</b>) dramatically decreased the binding affinity. Among all compounds with modification of substituents for R<sup>1</sup>, the 2-fluorophenyl analogue <b>16a</b> and 2,6-difluorophenyl analogue <b>16d</b> showed more potent binding affinities and longer drug–target residence times. Therefore, 2-fluorophenyl and 2,6-difluorophenyl groups were selected as favorable substituents for R<sup>1</sup> for further optimization.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Modification of Substituent R<sup>1</sup></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0021.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0022.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last"><i>t</i><sub>1/2</sub> = ln2/<i>k</i><sub>off</sub>.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p">Second, the results of introduction of the substituent R<sup>2</sup> are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The methyl analogue <b>17</b> showed slightly enhanced p<i>K</i><sub>i</sub> compared to that of the nonsubstituted derivative <b>14</b>. Introduction of a chloro group (<b>18</b>) significantly boosted the binding affinity and resulted in marked prolongation of <i>t</i><sub>1/2</sub> (<i>t</i><sub>1/2</sub> = 210 min). Compounds <b>19</b> and <b>20</b>, which had polar substituents such as cyano and carboxamide groups, also enhanced their binding affinity. These results demonstrated that filling the DP2 with not only lipophilic groups but also polar groups was remarkably effective to enhance the binding affinity. Accordingly, a chloro group was identified as the most potent and slow-dissociating substituent (p<i>K</i><sub>i</sub> = 9.39, <i>t</i><sub>1/2</sub> = 210 min).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effect of Introducing a Substituent into the 3-Position of the Core Structure</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0023.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0024.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last"><i>t</i><sub>1/2</sub> = ln2/<i>k</i><sub>off</sub>.</p></div></div><div></div></div><div class="NLM_p">We identified the favorable substituents for R<sup>1</sup> and R<sup>2</sup> to show potent binding affinity and slow dissociation (i.e., 2,6-difluorophenyl and 2-fluorophenyl groups for R<sup>1</sup> and chloro group for R<sup>2</sup>), so further optimization of the scaffold was continued to control the favorable ADMET profiles by using substituent R<sup>3</sup> toward the solvent-accessible region. Compounds <b>16a</b>, <b>16d</b>, and <b>18</b> bearing these optimal substituents showed potent p<i>K</i><sub>i</sub> and long <i>t</i><sub>1/2</sub> value, but they had an issue of poor metabolic stabilities (data for <b>18</b> in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, and data for <b>16a</b> and <b>16d</b> in a <a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Supporting Information</a>). We assumed the benzene ring of the benzoxazepinone would be metabolically labile due to the high electron density. Therefore, electron-withdrawing and polar substituents such as cyano and amide groups were introduced as R<sup>3</sup> into the 7- and 8-positions of the benzoxazepinone ring to block the assumed metabolic site, reduce the electron density of the benzoxazepinone, and decrease the total lipophilicity of the molecule. The results of introduction of the substituent R<sup>3</sup> and the combination between R<sup>1</sup> and R<sup>2</sup> are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Introduction of cyano groups into the 7- and 8-positions improved the metabolic stabilities while maintaining potent binding affinity. In particular, the 8-cyano analogue <b>22</b> showed better metabolic stability than that of the 7-cyano analogue <b>21</b>. This result suggests that a metabolic soft spot could be the 8-position of the benzoxazepinone. Amide groups were introduced into the 8-position instead of the cyano group of <b>22</b>. The amide analogues <b>23</b> and <b>24</b> showed high P-gp-mediated efflux (ER = 17 and 15, respectively) despite maintaining moderate to potent binding affinity and satisfactory metabolic stability. Compounds <b>25</b> and <b>26</b> possessing 2-fluorophenyl or 2,6-difluorophenyl groups were synthesized. Compounds <b>25</b> and <b>26</b> had moderate potency and good metabolic stability in addition to improved P-gp-mediated efflux (ER = 1.1 and 1.3, respectively). On the basis of these results, we selected compound <b>22</b> for further evaluation by considering the balance between potency, metabolic stability, and P-gp ER. We also evaluated the selectivity of compound <b>22</b> for RIP1 kinase by using a Reaction Biology Corporation kinase panel and a Eurofins Panlabs panel. As a result, compound <b>22</b> represented excellent specificity for RIP1 kinase over 406 kinases including RIP3 kinase (%inhibition: <50% at 10 μM) other than LIMK2 (>50% at 10 μM) in the Reaction Biology Corporation kinase panel. Compound <b>22</b> also showed excellent selectivity against 106 targets in the Eurofins Panlabs panel. The results of the Reaction Biology Corporation kinase panel and the Eurofins Panlabs panel are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Supporting Information</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Tables 4 and 5</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Optimization of Substituents R<sup>1</sup>, R<sup>2</sup>, and R<sup>3</sup></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0025.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0026.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last"><i>t</i><sub>1/2</sub> = ln2/<i>k</i><sub>off</sub>.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Metabolic stability in human and mouse liver microsomes.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Efflux ratios in P-gp-overexpressing cells at 1 μM.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Not tested.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Crystal Structure of Compound <b>22</b></h3><div class="NLM_p">An X-ray co-crystal structure of <b>22</b> bound to RIP1 kinase is shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. As we expected, the type III binding mode was observed. The core structure formed a hydrogen-bonding network with Asp156 among the nitrogen atom at the 1-position and the carbonyl group at the 7-position. The benzyl group fit into the allosteric hydrophobic DP1 at the back of the ATP binding site. The chloro group effectively occupied the small DP2, presumably interacting with the carbonyl group of Met67 at a distance of 3.5 Å.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> This chloro-oxygen interaction might result in extremely high affinity of <b>18</b> compared to that of the methyl analogue <b>17</b>. The cyano group on the benzoxazepinone ring was exposed toward the solvent-accessible region. No interaction was observed between <b>22</b> and Ser161, which is an autophosphorylation site. Reflecting this type III binding mode, compound <b>22</b> represented excellent selectivity against 406 kinases including RIP3 kinase (%inhibition: <50% at 10 μM) other than LIMK2 (>50% at 10 μM). These results demonstrated that compound <b>22</b> would be an excellent tool compound for in vitro and in vivo experiments.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0006.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of compound <b>22</b> (white).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> In Vitro and in Vivo Profiling of Selected Candidate</h3><div class="NLM_p">Compound <b>22</b> was tested further in an in vitro binding assay for human and mouse RIP1 kinase proteins (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Compound <b>22</b> had approximately 90 times more potent affinity for human RIP1 kinase than that for mouse RIP1 kinase in TR-FRET assay. Cellular assays were also performed by using human colorectal adenocarcinoma HT-29 cells and mouse L-cells NCTC 929 (L929). L929 necroptosis assay has been generally used to examine the necroptosis pathways in mouse cells and regarded as a well-established cellular model.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Compound <b>22</b> strongly suppressed necroptotic cell death and phosphorylation of MLKL (pMLKL), which was one of the hallmark mediators of necroptosis, in HT-29 cells (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) (nectoptosis, IC<sub>50</sub> = 2.0 nM; pMLKL, IC<sub>50</sub> = 1.3 nM) as well as L929 cells (nectoptosis, IC<sub>50</sub> = 15 nM; pMLKL, IC<sub>50</sub> = 2.7 nM). The high correlation was observed between TR-FRET and cellular assays in human, whereas there was a discrepancy between TR-FRET and cellular assays in mouse compared to human. It was not clear why there was a bigger discrepancy between TR-FRET and cellular assay of compound <b>22</b> in mouse than that in human, however, the similar trend was observed between human and mouse in the other 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and compound <b>6</b>.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In the cellular conditions, endogenous full-length enzyme of RIP1 kinase is expressed and forms a complex with a variety of proteins.<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34−36)</a> On the other hand, RIP1 kinase in the TR-FRET assay was composed of the catalytic domain of RIP1 kinase without any associated proteins. Although there is no information about the structural difference between the truncated catalytic domain of RIP1 kinase and the intracellular intact RIP1 kinase, the structural difference between truncated enzymes of human and mouse RIP1 kinase might explain the discrepancy between TR-FRET and cellular assays in mouse compared to human. Furthermore, in the case of L929 cells, TNF-α prominently causes necrosis instead of apoptosis, and this response is different from the other cell lines.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In fact, HT-29 cells require TNF-α, cIAP inhibitor, and a pan-caspase inhibitor to induce necroptosis.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Therefore, the necroptosis-inducible conditions are highly dependent on the kind of cell line. The L929 cells might be sensitive to necroptosis inducer and not require full target occupation of endogenous RIP1 kinase. Considering the excellent specificity for RIP1 kinase of <b>22</b> and the result of in vivo target engagement in mouse using cellular thermal shift assay (CETSA),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> it is indicated that cellular activities of <b>22</b> should be caused by the inhibition of RIP1 kinase. Compound <b>22</b> was progressed to an in vivo study to evaluate its efficacy through inhibition of nonprimate RIP1 kinase.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Cellular Activities in HT-29 and L929 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">p<i>K</i><sub>i</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">necroptosis IC<sub>50</sub> (nM)</th><th class="rowsep1 colsep0" colspan="2" align="center">pMLKL IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">human</th><th class="colsep0 rowsep0" align="center">mouse<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">human<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">mouse<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">human<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">mouse<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">9.04</td><td class="colsep0 rowsep0" align="center">7.09 (6.96–7.23)</td><td class="colsep0 rowsep0" align="center">2.0 (1.5–2.7)</td><td class="colsep0 rowsep0" align="center">15 (14–17)</td><td class="colsep0 rowsep0" align="center">1.3 (0.3–5.5)</td><td class="colsep0 rowsep0" align="center">2.7 (1.9–3.7)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">HT-29 cells.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">L929 cells.</p></div></div></div><div class="NLM_p">PK study of compound <b>22</b> was conducted, and the results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Compound <b>22</b> was administered to mice, and the plasma concentration was determined after oral (po, 1 mg/kg) and intravenous (iv, 0.1 mg/kg) administration. Compound <b>22</b> exhibited a desirable PK profile including high plasma exposure (AUC = 658 ng·h/mL) and a moderate plasma duration (MRT = 3.1 h).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. PK Parameters of Compound <b>22</b> in Mice<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center" char=".">PK parameters in mouse</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">CL* (mL/h/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub>** (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">MRT** (h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0–8h</sub>** (ng·h/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">794</td><td class="colsep0 rowsep0" align="char" char=".">140.1</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">658</td><td class="colsep0 rowsep0" align="char" char=".">52.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">The values are shown as means of three determinations (0.1 mg/kg, iv*, and 1.0 mg/kg, po**).</p></div></div></div><div class="NLM_p">Moreover, compound <b>22</b> showed satisfactory brain exposure (<i>C</i><sub>b,u</sub> = 0.0230 μg/g), which was equivalent to 50 nM, and a brain-to-plasma ratio (<i>K</i><sub>p,uu</sub> = 0.3) 1 h after oral administration (po, 10 mg/kg) in mice (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). We have also confirmed the target association of <b>22</b> in the mouse brain tissues by using CETSA at 1 h after oral administration of <b>22</b>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> On the basis of these results, compound <b>22</b> was selected for further evaluation in the EAE models of MS.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Concentrations in Plasma and Brain of Compound <b>22</b> in Mice<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">concn (μg/mL or μg/g)</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">time (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>p</sub><a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>b</sub><a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>p,u</sub><a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>b,u</sub><a class="ref internalNav" href="#t7fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>p</sub><a class="ref internalNav" href="#t7fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>p,uu</sub><a class="ref internalNav" href="#t7fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1.479</td><td class="colsep0 rowsep0" align="char" char=".">1.038</td><td class="colsep0 rowsep0" align="char" char=".">0.089</td><td class="colsep0 rowsep0" align="char" char=".">0.0230</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Administered orally at 10 mg/kg (<i>n</i> = 3).</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">Total plasma concentration.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Total brain concentration.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">Unbound plasma concentration.</p></div><div class="footnote" id="t7fn5"><sup><sup>e</sup></sup><p class="last">Unbound brain concentration.</p></div><div class="footnote" id="t7fn6"><sup><sup>f</sup></sup><p class="last"><i>K</i><sub>p</sub> = <i>C</i><sub>b</sub>/<i>C</i><sub>p</sub>.</p></div><div class="footnote" id="t7fn7"><sup><sup>g</sup></sup><p class="last"><i>K</i><sub>p,uu</sub> = <i>C</i><sub>b,u</sub>/<i>C</i><sub>p,u</sub>.</p></div></div></div><div class="NLM_p">Compound <b>22</b> was orally administered from day 0 to day 26, and progression of clinical signs of EAE was monitored daily. Clinical signs of EAE in vehicle-treated mice were observed from day 12 and peaked at around day 21. On the other hand, the orally administered RIP1 kinase inhibitor <b>22</b> at doses of 10 mg/kg twice a day (20 mg/kg/day) attenuated the development of clinical symptoms of EAE (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Cumulative score of EAE was significantly lower than that of the vehicle-treated group (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). These results indicated that compound <b>22</b> protected against disease progression in the EAE model. Ofengeim et al. also reported that Nec-1s (7N-1, compound <b>2</b> in this paper) blocks the disease progression in the EAE model.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Although it is difficult to compare the efficacy between <b>22</b> and <b>2</b> without head-to-head comparison in the same condition, compound <b>22</b> would be a novel class of an orally available and brain-penetrating in vivo tool for RIP1 kinase.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0007.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of compound <b>22</b> in the EAE model. RIP1 kinase inhibitor <b>22</b> (10 mg/kg) or vehicle (0.5% methylcellulose) was orally administered to mice twice a day for 26 days. (A,B) The EAE clinical score of (A) the time course and (B) the cumulative clinical score from day 0 to day 26 are shown. Data are presented as the mean ± SE (<i>n</i> = 12). **<i>p</i> < 0.05 compared to the naïve group (Dunn’s test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Structure–Kinetic Relationship</h3><div class="NLM_p">Several properties of ligands, including molecular weight, lipophilicity, and rotatable bond count, have been reported to affect kinetic parameters.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> During the optimization of 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives, molecular weight and rotatable bond count were not changed greatly and lipophilicity was the main parameter to optimize profiles such as binding activity and metabolic stability. Therefore, the SKR at R<sup>1</sup>–R<sup>3</sup> substituents were analyzed in relation with lipophilicity. We also investigated the relationship between <i>k</i><sub>on</sub> and <i>k</i><sub>off</sub> to know the value of SKR analysis compared to that of conventional SAR.</div><div class="NLM_p">In the modification of the substituent for R<sup>1</sup>, relationships between kinetic profiles (log <i>k</i><sub>on</sub>, blue dots; p<i>k</i><sub>off</sub>, red dots) and lipophilicity are summarized in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> (the detailed kinetic parameters are listed in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Supporting Information</a>). Introduction of a fluoro group into the benzene ring in DP1 affected the p<i>k</i><sub>off</sub> more than the log <i>k</i><sub>on</sub> (<b>14</b> and <b>16a</b>–<b>d</b>). Replacement of phenyl group with cyclobutyl group maintained the log <i>k</i><sub>on</sub> value and resulted in a 4-fold decrease in the <i>k</i><sub>off</sub> value (<b>14</b> and <b>16e</b>). This finding suggested that the decreased <i>k</i><sub>off</sub> value of <b>16e</b> caused the decreased binding affinity. The pyridine analogues <b>16f</b> and <b>16g</b> exhibited slightly lower log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> values than that of <b>14</b>, whereas introduction of sulfone groups (<b>16h</b> and <b>16i</b>) dramatically decreased both log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> values. Overall, the log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> values were mostly raised depending on the degree of increase in the clogP, and a particularly good correlation was observed between the log <i>k</i><sub>on</sub> and clogP.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0008.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. SKR and relationships between kinetic profiles and lipophilicity in the modification of R<sup>1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The relationship between the log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> in modification of the substituent at R<sup>1</sup> is displayed in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. No apparent correlation was observed between the log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub>. Interestingly, it was revealed that the <i>k</i><sub>on</sub> and <i>k</i><sub>off</sub> changed independently although both the log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> values roughly correlated with clogP. These results suggested that <i>k</i><sub>on</sub> and <i>k</i><sub>off</sub> can be optimized independently by measuring SKR, which would provide another medicinal chemistry strategy to obtain more potent compounds by combining the best <i>k</i><sub>on</sub> moiety with the best <i>k</i><sub>off</sub> moiety.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0009.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Relationships between log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> in modification of R<sup>1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Regarding the substituents at R<sup>2</sup> and R<sup>3</sup>, introduction of the appropriate R<sup>2</sup> substituent resulted in a longer drug–target residence time, but no notable effect on kinetic profiles was observed for substituent R<sup>3</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</div><div class="NLM_p">The cellular activity and binding parameters are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl7a" href="#tbl7a">Table <a class="ref showTableEvent internalNav" data-ID="tbl7a" href="#tbl7a">8</a></a>. On the basis of the co-crystal structure of <b>22</b>, compounds <b>14</b> and <b>22</b> should be type III inhibitors and would also be ATP competitive inhibitors as <b>3</b> and <b>6</b>.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In comparison between four ATP competitive inhibitors in <a class="ref showTableEvent internalNav" data-ID="tbl7a" href="#tbl7a">Table <a class="ref showTableEvent internalNav" data-ID="tbl7a" href="#tbl7a">8</a></a>, slower dissociating compounds exhibited more potent cellular activity. The residence time (<i>t</i><sub>1/2</sub>) was more correlated with the cellular activity than the affinity (p<i>K</i><sub>i</sub>). It suggested that in the case of RIP1 kinase inhibitors, the residence time would be more important for achieving optimal cellular activity than affinity. However, further study is needed to conclude which is more critical for the cellular activity, the affinity or the residence time, because the sample size is small in this study. By analyzing kinetic profiles as well as thermodynamic profiles, we succeeded in identifying a favorable RIP1 kinase inhibitor that showed the potent cellular activity.</div><div class="NLM_table-wrap" id="tbl7a"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Impact of <i>t</i><sub>1/2</sub> on Cellular Activity</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">HT29 necroptosis IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t7afn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t7afn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">log <i>k</i><sub>on</sub><a class="ref internalNav" href="#t7afn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">p<i>k</i><sub>off</sub><a class="ref internalNav" href="#t7afn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>1/2</sub><a class="ref internalNav" href="#t7afn2" aria-label="b">b</a>,<a class="ref internalNav" href="#t7afn3" aria-label="c">c</a> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">8.69</td><td class="colsep0 rowsep0" align="center">300 (160–580)</td><td class="colsep0 rowsep0" align="center">7.63 (7.56–7.69)</td><td class="colsep0 rowsep0" align="center">1.06 (0.99–1.15)</td><td class="colsep0 rowsep0" align="center">8.0 (6.8–9.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">8.41</td><td class="colsep0 rowsep0" align="center">18 (14–23)</td><td class="colsep0 rowsep0" align="center">6.48 (6.39–6.55)</td><td class="colsep0 rowsep0" align="center">1.93 (1.83–2.06)</td><td class="colsep0 rowsep0" align="center">59 (46–80)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">9.01</td><td class="colsep0 rowsep0" align="center">4.1 (2.7–6.1)</td><td class="colsep0 rowsep0" align="center">6.79 (6.74–6.84)</td><td class="colsep0 rowsep0" align="center">2.22 (2.14–2.31)</td><td class="colsep0 rowsep0" align="center">115 (97–143)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">9.04</td><td class="colsep0 rowsep0" align="center">2.0 (1.5–2.7)</td><td class="colsep0 rowsep0" align="center">6.55 (6.51–6.59)</td><td class="colsep0 rowsep0" align="center">2.49 (2.41–2.58)</td><td class="colsep0 rowsep0" align="center">210 (180–260)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7afn1"><div class="footnote" id="t7afn1"><sup><sup>a</sup></sup><p class="last">HT-29 cells.</p></div><div class="footnote" id="t7afn2"><sup><sup>b</sup></sup><p class="last">95% confidence intervals (given in parentheses) determined in duplicate (<i>n</i> = 2).</p></div><div class="footnote" id="t7afn3"><sup><sup>c</sup></sup><p class="last"><i>t</i><sub>1/2</sub> = ln2/<i>k</i><sub>off</sub>.</p></div></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51260" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51260" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Modifications of the core structure were performed according to <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch5" aria-label="5">5</a>. Synthesis of 4-oxo-1,4,6,7-tetrahydro-5<i>H</i>-imidazo[4,5-<i>c</i>]pyridine derivative <b>11</b> is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Commercially available <b>27</b> was reacted with methyl propiolate to afford imidazole <b>28</b>. After bromination and protection with a <i>p</i>-methoxybenzyl group (PMB), Sizuki–Miyaura coupling yielded <b>31</b>. The ethoxyvinyl group of <b>31</b> was converted to a formylmethyl group, followed by reductive amination with (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> to give <b>33</b>. Cyclization using AlMe<sub>3</sub> and subsequent deprotection of the PMB group afforded 4-oxo-1,4,6,7-tetrahydro-5<i>H</i>-imidazo[4,5-<i>c</i>]pyridine derivative <b>11</b>.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) methyl propiolate, diphenyl ether, microwave, 200 °C, 35%; (b) bromine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 56%; (c) 4-methoxybenzyl chloride, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 21%; (d) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, 75%; (e) 6 M HCl aq/THF, rt–60 °C, 98%; (f) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 65%; (g) AlMe<sub>3</sub>, toluene, 120 °C, under Ar, 50%; (h) Pd(OH)<sub>2</sub>, MeOH, 50 °C, under H<sub>2</sub>, 66%.</p></p></figure><div class="NLM_p">8-Oxo-5,6-dihydro[1,2,4]triazolo[1,5-<i>a</i>]pyrazine derivative <b>12</b> was synthesized by cyclization of <b>6</b> with 1,2-dibromoethane (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, 1,2-dibromoethane, DMF, 60 °C, 22%.</p></p></figure><div class="NLM_p">Treatment of commercially available <b>36</b> with ethyl 2-ethoxy-2-iminoacetate formed imidazole derivative <b>37</b>. After hydrolysis of the ester group and amidation, 8-oxo-5,6-dihydroimidazo[1,2-<i>a</i>]pyrazine derivative <b>13</b> was obtained by similar cyclization with <b>12</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0012.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl 2-ethoxy-2-iminoacetate, NaOAc, AcOH, 110 °C, 9.1%; (b) (1) 2 M NaOH aq, EtOH, 100 °C, (2) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, HATU, DIPEA, DMF, rt, 59%; (c) Cs<sub>2</sub>CO<sub>3</sub>, 1,2-dibromoethane, DMF, 100 °C, 3.2%.</p></p></figure><div class="NLM_p">Preparation of 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivative <b>14</b> was conducted in a manner similar to that for <b>11</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Modification of the substituent for R<sup>1</sup> was conducted by iterative synthesis or combinatorial chemistry using debenzylated compound <b>45</b>. Alkylation of <b>45</b> with the corresponding alkyl halide afforded compounds <b>16a</b>–<b>aj</b>, respectively.</div><figure id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, benzyl bromide, DMF, rt, 64%; (b) bromine, MeCN, rt, 98%; (c) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, 94%; (d) 6 M HCl aq/THF, 60 °C, 94%; (e) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 94%; (f) AlMe<sub>3</sub>, toluene, 120 °C, under Ar, 41%; (g) Pd/C, 1 M HCl aq, MeOH, rt, under H<sub>2</sub>, 82%; (h) R<sup>1</sup>CH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF or DMA, rt–80 °C, 1.4–65%.</p></p></figure><div class="NLM_p">Synthesis of 4-oxo-2,4,6,7-tetrahydro-5<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridine derivative <b>15</b> was performed along with <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Commercially available <b>46</b> was treated with hydrazine monohydrate to give pyrazole derivative <b>47</b>. After benzylation, deprotection, and oxidation, aldehyde <b>50</b> was synthesized. Aldehyde <b>50</b> was reacted with (methoxymethyl)triphenylphosphonium chloride in the presence of potassium <i>tert</i>-butoxide (<sup><i>t</i></sup>BuOK), followed by hydrolysis, reductive amination, and cyclization to give compound <b>15</b>.</div><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0014.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) DMFDMA, toluene, 65 °C, (2) NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O, AcOH, rt, 52% in 2 steps; (b) benzyl chloride, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 64%; (c) TFA, DCM, rt, 86%; (d) MnO<sub>2</sub>, DCM, 40 °C, 59%; (e) (methoxymethyl)triphenylphosphonium chloride, <sup><i>t</i></sup>BuOK, THF, rt, 53%; (f) (1) 6 M HCl aq/THF, rt, (g) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt; (h) AlMe<sub>3</sub>, toluene, 100 °C, 6% in 3 steps.</p></p></figure><div class="NLM_p">3-Substituted 7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives <b>17</b>–<b>20</b> were synthesized as shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. Treatment of commercially available <b>54</b> with benzylhydrazine hydrochloride yielded <b>55</b>, followed by conversion to compounds <b>56a</b>,<b>b</b> by using phosphoryl bromide or phosphoryl chloride. Compounds <b>18</b> and <b>60</b> were prepared in a similar way with <b>15</b>. Sizuki–Miyaura coupling of <b>60</b> provided methyl analogue <b>17</b>. The cyanation of <b>60</b> using dicyanozinc formed compound <b>19</b>, followed by hydrolysis to give <b>20</b>.</div><figure id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0015.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzylhydrazine hydrochloride, K<sub>2</sub>CO<sub>3</sub>, EtOH, 90 °C, 36%; (b) phosphoryl bromide, DMF, 1,2-dichloroethane, 90 °C, 48%; (c) phosphoryl chloride, DMF, 90 °C, 36%; (d) (methoxymethyl)triphenylphosphonium chloride, <sup><i>t</i></sup>BuOK, THF, rt, 30–42%; (e) 6 M HCl aq/THF, rt; (f) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 67–94% in 2 steps; (g) AlMe<sub>3</sub>, toluene, 100 °C, 23–82%; (h) Pd(PPh<sub>3</sub>)<sub>4</sub>, 2,4,6-trimethyl-1,3,5,2,4,6-trioxatriborinane, KOH, THF, 100 °C, under Ar, 9%; (i) dicyanozinc, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 100 °C, under Ar, 90%; (j) K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O<sub>2</sub>, DMSO, rt, 60%.</p></p></figure><div class="NLM_p">Synthesis of key intermediates <b>64a</b>,<b>b</b> and <b>70a</b>–<b>d</b> are described in <a class="ref internalNav" href="#sch7" aria-label="Schemes 7">Schemes 7</a> and <a class="ref internalNav" href="#sch8" aria-label="8">8</a>, respectively. Compounds <b>64a</b>,<b>b</b> were synthesized in a manner similar to that described for <b>43</b> in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Substituted fluoronitrobenzenes <b>65a</b>–<b>c</b> were reacted with (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-3-hydroxypropanoic acid in a presence of sodium hydride, followed by reduction of the nitro group, intramolecular amidation, and <i>N</i>-methylation to give Boc-protected <b>69a</b>–<b>c</b>. Compound <b>69c</b> was converted to compounds <b>69e</b>,<b>f</b> via deprotection of the benzyl group and amidation. (<i>S</i>)-3-Amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one <b>70a</b>–<b>d</b> were obtained by deprotection of the corresponding intermediate.</div><figure id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0016.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) R<sup>1</sup>CH<sub>2</sub>Br, NaH, THF, rt, 95–97%; (b) bromine, MeCN, rt, 88–98%; (c) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, under Ar, 84–87%; (d) 6 M HCl a./THF, rt, 68%–quant</p></p></figure><figure id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0017.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-3-hydroxypropanoic acid, NaH, DMF, rt, 22–74%; (b) Fe, NH<sub>4</sub>Cl, EtOH/water, 80 °C; (c) Zn, AcOH, 0 °C; (d) HATU, TEA, DMSO, rt, 3–26% in 2 steps; (e) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, rt, 45–75%; (f) Pd/C, THF, rt, under H<sub>2</sub>, 99%; (g) methylamine or dimethylamine, HATU, TEA, DMF, rt, 85–100%; (h) 4 M HCl in EtOAc, THF, rt, 60–98%.</p></p></figure><div class="NLM_p">Modifications of the substituent for R<sup>3</sup> were performed according to <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>. Reductive amination of aldehydes <b>58b</b> and <b>64a</b>,<b>b</b> with the corresponding amines <b>70a</b>–<b>d</b> gave compounds <b>71a</b>–<b>f</b>, respectively. After cyclization and cyanation if needed, compounds <b>21</b>–<b>26</b> were finally synthesized.</div><figure id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0018.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>70a</b>–<b>d</b>, 2-picoline boran, acetic acid, MeOH, rt, 48–100%; (b) AlMe<sub>3</sub>, toluene, 100 °C, 29–89%; (c) dicyanozinc, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 100 °C, under Ar, 62–85%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65071" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65071" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">7-Oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives were discovered as potent and brain-penetrating RIP1 kinase inhibitors. Optimization of the core structure and exploration of appropriate substituents for DP1, DP2, and solvent-accessible region by utilizing the SBDD approach led to the discovery of (3<i>S</i>)-3-(2-benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carbonitrile (<b>22</b>), a highly potent, orally active, and brain-penetrating RIP1 kinase inhibitor with an excellent PK profile. Compound <b>22</b> significantly suppressed necroptotic cell death and phosphorylation of MLKL in cellular assays using mouse cells in addition to human cells, which enabled evaluation of a RIP1 kinase inhibitor in a mouse neurodegenerative model. Oral administration of <b>22</b> at 10 mg/kg, bid, attenuated disease progression in a mouse EAE model of MS. Although confirmation of the target binding of compound <b>22</b> with CETSA in the mouse brain tissues demonstrates that our novel chemical series target RIP1 kinase signaling pathway in CNS, further studies are needed to strengthen the evidence for the target engagement in animal brain tissues. Further evaluation of compound <b>22</b> is ongoing for neurodegenerative disorders. Compound <b>22</b> is also a good CNS tool compound for RIP1 kinase to investigate therapeutic opportunities for neurodegenerative disorders such as MS, PD, and ALS, with significant unmet medical needs.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63045" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63045" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Chemistry Information</h3><div class="NLM_p">All commercially available solvents and reagents were used without further purification. Yields were not optimized. <sup>1</sup>H NMR spectra were recorded on Bruker DPX300 (300 MHz) or DPX400 (400 MHz) instruments. Chemical shifts are reported as δ values (ppm) downfield from internal TMS of the indicated organic solution. Peak multiplicities are expressed as follows: s, singlet; d, doublet; t, triplet; q, quartet; dd, doublet of doublets; td, triplet of doublets; ddd, doublet of doublet of doublets; brs, broad singlet; m, multiplet. Coupling constants (<i>J</i> values) are given in hertz (Hz). LC-MS was performed on a Waters liquid chromatography–mass spectrometer system, using a CAPCELL PAK UG-120 ODS column (2.0 mm i.d.mm × 50 mm, Shiseido Co., Ltd.) with a 5–95% gradient of MeCN in water containing 0.04% TFA and an HP-1100 (Agilent Technologies) apparatus for monitoring at 220 nm. All microwave reactions were performed in a Biotage Initiator 2.0 or 2.5 microwave synthesizer. Preparative HPLC was performed on an automated Gilson HPLC system using a YMC C-18 column (S-5 μm, 50 mm × 20 mm i.d.) with a 5–95% gradient of MeCN in water containing 0.1% TFA. Reaction progress was determined by TLC analysis on Merck Kieselgel 60 F254 plates or Fuji Silysia NH plates. Column chromatography was performed using silica gel (Merck Kieselgel 60, 70–230 mesh), basic silica gel (Chromatorex NH-DM 1020, 100–200 mesh, Fuji Silysia Chemical, Ltd.), or Purif-Pack (SI ϕ 60 μM or NH ϕ 60 μM, Fuji Silysia Chemical, Ltd.). Unless otherwise stated, the purities of synthesized compounds for biological testing were >95% determined by analytical HPLC. (The purities of <b>11</b>, <b>13</b>, <b>16b</b>, <b>16h</b>, and <b>16t</b> were 89.6%, 86.8%, 94.7%, 94.4%, and 93.9%, respectively.) Analytical HPLC was performed with a Corona charged aerosol detector (CAD), Nano quantity analyte detector (NQAD), or photo diode array detector. The column was a Capcell Pak C18AQ (50 mm × 3.0 mm I.D., Shiseido, Japan) or L-column 2 ODS (30 mm × 2.0 mm I.D., CERI, Japan) with a temperature of 50 °C and a flow rate of 0.5 mL/min. Mobile phase A and B under a neutral condition were a mixture of 50 mmol/L ammonium acetate, water, and acetonitrile (1:8:1, v/v/v) and a mixture of 50 mmol/L ammonium acetate and acetonitrile (1:9, v/v), respectively. The ratio of mobile phase B was increased linearly from 5% to 95% over 3 min and 95% over the next 1 min. Mobile phase A and B under acidic conditions were a mixture of 0.2% formic acid in 10 mmol/L ammonium formate and 0.2% formic acid in acetonitrile, respectively. The ratio of mobile phase B was increased linearly from 14% to 86% over 3 min and 86% over the next 1 min. Abbreviations of solvents are used as follows: CDCl<sub>3</sub>, deuterated chloroform; DMSO-<i>d</i><sub>6</sub>, dimethyl sulfoxide-<i>d</i><sub>6</sub>; dichloromethane, DCM; EtOAc, ethyl acetate; DMF, <i>N</i>,<i>N</i>-dimethylformamide; MeOH, methanol; THF, tetrahydrofuran; EtOH, ethanol; DMSO, dimethyl sulfoxide; DMA, <i>N</i>,<i>N</i>-dimethylactamide; AcOH, acetic acid; DMFDMA, <i>N</i>,<i>N</i>-dimethylformamide dimethylacetal; MeCN, acetonitrile; DIPEA, <i>N</i>,<i>N</i>-diisopropylethylamine; TEA, triethylamine; IPE, diisopropyl ether.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (3<i>S</i>)-3-(2-Benzyl-4-oxo-1,4,6,7-tetrahydro-5<i>H</i>-imidazo[4,5-<i>c</i>]pyridin-5-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one trifluoroacetic Acid (<b>11</b>)</h4><div class="NLM_p last">Two batches of the reaction were conducted and worked up together after the reaction. A mixture of <b>34</b> (10 mg, 0.02 mmol) and 10% Pd(OH)<sub>2</sub> on carbon (2.4 mg, 1.71 μmol) in MeOH (5 mL) was hydrogenated under balloon pressure at room temperature overnight. Pd(OH)<sub>2</sub> (7 mg) and 6 M HCl aq (1 mL) were added. After stirring at rt under H<sub>2</sub> atmosphere for 2 h, the mixture was heated to 50 °C and stirred at 30 min. The catalyst was removed by filtration, and the filtrate was concentrated under reduced pressure. A mixture of <b>34</b> (22 mg, 0.04 mmol) and 10% Pd(OH)<sub>2</sub> on carbon (16 mg, 0.01 mmol) in MeOH (5 mL), and 6 M HCl aq (1.000 mL) was hydrogenated under balloon pressure at 50 °C for 2.5 h. The catalyst was removed by filtration, and the filtrate was concentrated under reduced pressure. The residues were combined and diluted with MeOH. The solution of the mixture in MeOH was purified by preparative HPLC (L-Column 2 ODS, eluted with H<sub>2</sub>O in acetonitrile containing 0.1% TFA). The desired fractions were concentrated under reduced pressure to give <b>11</b> (21 mg, 66%) as a light-brown amorphous powder. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.01–3.17 (1H, m), 3.20–3.39 (4H, m), 3.63–3.79 (1H, m), 3.89–4.13 (2H, m), 4.29–4.46 (2H, m), 4.74 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.64 (1H, dd, <i>J</i> = 11.9, 7.7 Hz), 6.77–6.89 (1H, m), 6.95–7.21 (5H, m), 7.28–7.35 (3H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 403.2 [M + H]<sup>+</sup>. Analytical HPLC 89.6% purity.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (3<i>S</i>)-3-(2-Benzyl-8-oxo-5,6-dihydro[1,2,4]triazolo[1,5-<i>a</i>]pyrazin-7(8<i>H</i>)-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">To a solution of <b>6</b> (47 mg, 0.12 mmol) and sodium hydride (10.96 mg, 0.27 mmol) in DMF (1 mL) was added 1,2-dibromoethane (0.013 mL, 0.15 mmol) at room temperature, and the mixture was stirred 24 h at 60 °C. The reaction mixture was filtered through a short pad of silica gel eluted by EtOAc/MeOH (1/1) and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 80–100% EtOAc in hexane) to give <b>12</b> (11.10 mg, 22%) as a white amorphous solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.38 (3H, s), 3.82 (1H, ddd, <i>J</i> = 12.4, 11.0, 4.2 Hz), 4.06–4.18 (2H, m), 4.34–4.80 (5H, m), 5.82 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.14–7.40 (9H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 404.3 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (3<i>S</i>)-3-(2-Benzyl-8-oxo-5,6-dihydroimidazo[1,2-<i>a</i>]pyrazin-7(8<i>H</i>)-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>13</b>)</h4><div class="NLM_p last">To a solution of <b>38</b> (89 mg, 0.24 mmol) and 1,2-dibromoethane (commercially available, 0.072 mL, 0.83 mmol) in DMF (2 mL, 25.83 mmol) was added cesium carbonate (308 mg, 0.95 mmol) at room temperature, and the mixture was stirred for 2 h at 100 °C. To the reaction mixture was added water and extracted with EtOAc. The organic layer was washed twice with water and once with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 17–43% MeOH in EtOAc). The residue was purified by preparative HPLC (L-Column 2 ODS, eluted with H<sub>2</sub>O in acetonitrile containing 0.1% TFA). The desired fraction was neutralized with saturated NaHCO<sub>3</sub> aq and extracted with EtOAc. The organic layer was separated, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to give <b>13</b> (3.0 mg, 3.2%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.37 (3H, s), 3.74 (1H, td, <i>J</i> = 11.6, 4.0 Hz), 3.99 (2H, s), 4.06 (1H, dt, <i>J</i> = 12.6, 3.9 Hz), 4.28–4.66 (4H, m), 5.89 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 6.56 (1H, s), 7.15–7.34 (9H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 403.2 [M + H]<sup>+</sup>. Analytical HPLC 86.8% purity.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (3<i>S</i>)-3-(2-Benzyl-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was obtained (41%) as a white amorphous solid in a manner similar to that described for the synthesis of <b>34</b>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.70 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 3.10 (1H, ddd, <i>J</i> = 15.6, 10.7, 4.7 Hz), 3.38 (3H, s), 3.54 (1H, td, <i>J</i> = 11.2, 4.3 Hz), 4.22 (1H, dt, <i>J</i> = 11.9, 5.0 Hz), 4.42 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.62 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.34 (2H, s), 5.96 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.11–7.26 (7H, m), 7.29–7.39 (3H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 403.3 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.6, 35.6, 45.8, 51.2, 56.8, 74.8, 121.3, 122.6, 123.3, 125.7, 125.9, 127.2, 128.0, 128.3, 128.9, 135.8, 136.8, 142.7, 150.0, 161.4, 169.6. Anal. Calcd for C<sub>23</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>3</sub> + 0.4H<sub>2</sub>O: C, 67.43; H, 5.61; N, 13.68. Found: C, 67.54; H, 5.63; N, 13.48. Analytical HPLC 99.9% purity.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (3<i>S</i>)-3-(2-Benzyl-4-oxo-2,4,6,7-tetrahydro-5<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridin-5-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">To a solution of <b>51</b> (200 mg, 0.699 mmol) in THF (5 mL) was added 6 M HCl aq (0.47 mL, 2.80 mmol). After stirring at 25–30 °C for 14 h, the micture was diluted with water, neutralized with saturated NaHCO<sub>3</sub> aq until pH = 7, and extracted with EtOAc (twice). The combined organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give <b>52</b> as a colorless oil. To a solution of the crude of <b>52</b> and (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride (75 mg, 0.33 mmol) in MeOH (2 mL) and AcOH (0.2 mL) was added 2-picoline boran (53 mg, 0.50 mmol) at 0 °C, and the mixture was stirred at 0 °C for 0.5 h. The reaction was quenched with water (20 mL), and to the mixture was added saturated NaHCO<sub>3</sub> aq until pH = 7, then it was extracted with EtOAc (30 mL × 2). The combined organic layer was washed with brine (50 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic layer was concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with petroleum ether/EtOAc (1:1)) to give <b>53</b> as an off-white solid (LC-MS (APCI) <i>m</i>/<i>z</i> 449.3 [M + H]<sup>+</sup>). To a solution of the crude of <b>53</b> in toluene (2 mL) was added AlMe<sub>3</sub> (2 M in toluene, 0.25 mL,0.51 mmol) at 25 °C. The mixture was stirred at 100 °C under N<sub>2</sub> for 1 h. After cooling to 20 °C, the reaction mixture was quenched with water (30 mL), and it was extracted with EtOAc (30 mL × 3). The combined organic layer was washed with brine (50 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic layer was concentrated under reduced pressure to give crude product. It was purified by prep-HPLC (0.1% NH<sub>3</sub>·H<sub>2</sub>O as additive) and most of MeCN was removed under reduced pressure to give the residue and it was lyophilized to give <b>15</b> (18 mg, 6% in 3 steps) as an off-white solid. LC-MS (mobile phase: from 90% [water + 0.04% TFA] and 10% [MeCN + 0.02% TFA] to 20% [water + 0.04% TFA] and 80% [MeCN + 0.02% TFA] in 1.35 min, then under this condition for 0.9 min, finally changed to 90% [water + 0.04% TFA] and 10% [MeCN + 0.02% TFA] and under this condition for 0.75 min) purity is >99%, <i>R</i><sub>t</sub> = 1.615 min. MS Calcd: 402.2. MS Found: 403.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.75–2.85 (1H, m), 3.10–3.20 (1H, m), 3.31 (3H, s), 3.45–3.55 (1H, m), 4.15–4.25 (1H, m), 4.31 (1H, dd, <i>J</i> = 10.0, 8.4 Hz), 4.60 (1H, dd, <i>J</i> = 11.6, 10.0 Hz), 5.18 (2H, s), 5.75 (1H, dd, <i>J</i> = 11.6, 8.0 Hz), 7.05–7.20 (6H, m), 7.25–7.35 (3H, m, overlap with CDCl<sub>3</sub> signal), 7.66 (1H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 23.3, 35.5, 45.1, 50.9, 56.4, 74.9, 113.8, 122.6, 123.1, 125.8, 127.1, 127.9, 128.4, 129.0, 130.7, 135.4, 136.9, 149.9, 152.4, 162.6, 169.7. Anal. Calcd for C<sub>23</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>3</sub> + 0.5H<sub>2</sub>O: C, 67.14; H, 5.63; N, 13.62. Found: C, 67.09; H, 5.61; N, 13.44. Analytical HPLC 98.5% purity.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (3<i>S</i>)-3-(2-(2-Fluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16a</b>)</h4><div class="NLM_p last">2-Fluorobenzyl bromide (commercially available, 12 μL, 0.10 mmol) was added to a solution of <b>45</b> (26 mg, 0.08 mmol) and potassium carbonate (34.5 mg, 0.25 mmol) in DMF (1.0 mL) at room temperature. The mixture was stirred at room temperature overnight. The mixture was partitioned between EtOAc and water. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 80–100% EtOAc in hexane) to give <b>16a</b> (17.6 mg, 0.042 mmol, 50%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 421.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.77 (1H, dt, <i>J</i> = 15.9, 5.3 Hz), 3.12 (1H, ddd, <i>J</i> = 15.7, 10.4, 4.9 Hz), 3.38 (3H, s), 3.59 (1H, ddd, <i>J</i> = 12.1, 10.2, 4.5 Hz), 4.18–4.31 (1H, m), 4.40 (1H, dd, <i>J</i> = 9.8, 7.9 Hz), 4.67 (1H, dd, <i>J</i> = 11.9, 10.0 Hz), 5.66–5.88 (3H, m), 6.88–6.96 (1H, m), 6.97–7.06 (2H, m), 7.12–7.25 (5H, m), 7.42 (1H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.8, 50.1 (d, <i>J</i> = 4.4 Hz), 51.2, 74.8, 115.5 (d, <i>J</i> = 20.9 Hz), 121.3, 122.6, 123.0 (d, <i>J</i> = 14.9 Hz), 123.3, 124.7 (d, <i>J</i> = 3.9 Hz), 125.9, 126.0 (d, <i>J</i> = 1.7 Hz), 127.2, 130.4 (d, <i>J</i> = 8.3 Hz), 130.6 (d, <i>J</i> = 3.9 Hz), 136.8, 142.9, 150.0, 160.4 (d, <i>J</i> = 247 Hz), 161.3, 169.6. Anal. Calcd for C<sub>23</sub>H<sub>21</sub>FN<sub>4</sub>O<sub>3</sub> + 0.7H<sub>2</sub>O: C, 63.79; H, 5.21; N, 12.94. Found: C, 63.92; H, 5.16; N, 12.68. Analytical HPLC 96.3% purity.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (3<i>S</i>)-3-(2-(3-Fluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16b</b>)</h4><div class="NLM_p last">Compound <b>16b</b> was obtained (2% in 4 steps) as a white solid in a manner similar to that described for the synthesis of <b>14</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 421.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.65–2.80 (1H, m), 2.80–2.95 (1H, m), 3.29 (3H, s, overlap with water signal), 3.50–3.65 (1H, m), 3.90–4.05 (1H, m), 4.32 (1H, dd, <i>J</i> = 10.0, 8.0 Hz), 4.75–4.90 (1H, m), 5.38 (2H, s), 5.56 (1H, dd, <i>J</i> = 12.0, 7.6 Hz), 7.04–7.09 (2H, m), 7.10–7.20 (1H, m), 7.25–7.45 (4H, m), 7.45–7.55 (1H, m), 7.78 (1H, s). Analytical HPLC 94.7% purity.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (3<i>S</i>)-3-(2-(4-Fluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16c</b>)</h4><div class="NLM_p last">Compound <b>16c</b> was obtained (23%) as a white solid in a manner similar to that described for the synthesis of <b>14</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 421.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.61–2.73 (1H, m), 2.75–2.90 (1H, m), 3.29 (3H, s, overlap with water signal), 3.50–3.62 (1H, m), 3.92–4.01 (1H, m), 4.25–4.35 (1H, m), 4.75–4.85 (1H, m), 5.34 (2H, s), 5.56 (1H, dd, <i>J</i> = 15.6, 8.0 Hz), 7.12–7.37 (7H, m), 7.45–7.53 (1H, m), 7.75 (1H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.8, 51.2, 56.0, 74.8, 115.8 (d, <i>J</i> = 21.5 Hz), 121.4, 122.6, 123.3, 125.6, 125.9, 127.2, 129.8 (d, <i>J</i> = 8.3 Hz), 131.6 (d, <i>J</i> = 3.3 Hz), 136.8, 142.9, 150.0, 161.3, 162.6 (d, <i>J</i> = 248 Hz), 169.5. Anal. Calcd for C<sub>23</sub>H<sub>21</sub>FN<sub>4</sub>O<sub>3</sub> + 0.1H<sub>2</sub>O: C, 65.42; H, 5.06; N, 13.27. Found: C, 65.14; H, 5.10; N, 13.03. Analytical HPLC 98.2% purity.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (3<i>S</i>)-3-(2-(2,6-Difluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16d</b>)</h4><div class="NLM_p last">Compound <b>45</b> (25.0 mg, 80 μmol), potassium carbonate (22.3 mg, 160 μmol), 2,6-difluorobenzyl bromide (commercially available, 160 μmol), and DMA (1 mL) were charged to 16.5 mL test tube. After the mixture was stirred at 80 °C for 1.5 h, the salt was removed by filtration and washed with MeOH. The filtrate was purified by directly injecting into the Gilson Preparative HPLC under neutral conditions. The obtained fraction was dried by blowing away with the air at 60 °C to give <b>16d</b> (17.9 mg, 51%). LC-MS (APCI) <i>m</i>/<i>z</i> 439.1 [M + H]<sup>+</sup>. Analytical HPLC 99.7% purity.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (3<i>S</i>)-3-(2-(Cyclobutylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16e</b>)</h4><div class="NLM_p last">Compound <b>16e</b> was obtained (22%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 381.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(pyridin-3-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16f</b>)</h4><div class="NLM_p last">Compound <b>16f</b> was obtained (9.3%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 404.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.72 (1H, dt, <i>J</i> = 15.49, 4.72 Hz), 3.12 (1H, tdd, <i>J</i> = 10.39, 10.39, 5.29, 4.15 Hz), 3.38 (3H, s), 3.55 (1H, ddd, <i>J</i> = 11.99, 10.67, 4.15 Hz), 4.23 (1H, dt, <i>J</i> = 11.99, 5.15 Hz), 4.42 (1H, dd, <i>J</i> = 10.01, 8.12 Hz), 4.63 (1H, dd, <i>J</i> = 11.71, 9.82 Hz), 5.36 (2H, d, <i>J</i> = 0.76 Hz), 5.94 (1H, dd, <i>J</i> = 11.71, 8.31 Hz), 7.13–7.32 (5H, m), 7.57 (1H, dq, <i>J</i> = 7.93, 1.26 Hz), 8.56 (2H, td, <i>J</i> = 4.63, 1.70 Hz). Analytical HPLC 98.7% purity.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(pyridin-2-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16g</b>)</h4><div class="NLM_p last">Compound <b>16g</b> was obtained (11%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 404.1 [M + H]<sup>+</sup>. Analytical HPLC 96.5% purity.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (3<i>S</i>)-3-(2-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16h</b>)</h4><div class="NLM_p last">Compound <b>16h</b> was obtained (9.5%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 459.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.80–2.01 (4H, m), 2.24–2.42 (1H, m), 2.75 (1H, dt, <i>J</i> = 15.6, 4.8 Hz), 2.85–3.19 (5H, m), 3.38 (3H, s), 3.50–3.62 (1H, m), 4.05 (2H, d, <i>J</i> = 7.4 Hz), 4.24 (1H, dt, <i>J</i> = 12.0, 5.2 Hz), 4.43 (1H, dd, <i>J</i> = 9.9, 8.2 Hz), 4.64 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.93 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.10–7.34 (5H, m). Analytical HPLC 94.4% purity.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (3<i>S</i>)-3-(2-((1,1-Dioxidotetrahydrothiophen-3-yl)methyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16i</b>)</h4><div class="NLM_p last">Compound <b>16i</b> was obtained (1.4%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 445.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.86–2.02 (1H, m), 2.22–2.38 (1H, m), 2.69–2.86 (2H, m), 2.99–3.27 (5H, m), 3.39 (3H, s), 3.56 (1H, td, <i>J</i> = 11.1, 4.5 Hz), 4.19–4.34 (3H, m), 4.43 (1H, dd, <i>J</i> = 9.6, 8.5 Hz), 4.65 (1H, dd, <i>J</i> = 11.7, 9.8 Hz), 5.93 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.13–7.30 (5H, m). Analytical HPLC 67.8% purity.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (3<i>S</i>)-3-(2-(4-Chloro-2-fluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16j</b>)</h4><div class="NLM_p last">Compound <b>16j</b> was obtained (51%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 455.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.72 (1H, dt, <i>J</i> = 15.39, 4.77 Hz), 3.11 (1H, tdd, <i>J</i> = 10.34, 10.34, 5.38, 4.15 Hz), 3.38 (3H, s), 3.54 (1H, ddd, <i>J</i> = 11.80, 10.67, 4.34 Hz), 4.22 (1H, dt, <i>J</i> = 11.80, 5.24 Hz), 4.41 (1H, dd, <i>J</i> = 10.01, 8.12 Hz), 4.62 (1H, dd, <i>J</i> = 11.71, 10.20 Hz), 5.35 (2H, s), 5.93 (1H, dd, <i>J</i> = 11.71, 8.31 Hz), 7.05–7.30 (8H, m), 11.35–11.35 (1H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.8, 49.5 (d, <i>J</i> = 3.9 Hz), 51.2, 74.8, 116.4 (d, <i>J</i> = 24.8 Hz), 121.4, 121.8 (d, <i>J</i> = 14.9 Hz), 122.6, 123.3, 125.1 (d, <i>J</i> = 3.3 Hz), 125.9, 126.0 (d, <i>J</i> = 1.1 Hz), 127.2, 131.3 (d, <i>J</i> = 4.4 Hz), 135.4 (d, <i>J</i> = 9.9 Hz), 136.8, 143.2, 150.0, 160.1 (d, <i>J</i> = 250 Hz), 161.2, 169.5. Anal. Calcd for C<sub>23</sub>H<sub>20</sub>ClFN<sub>4</sub>O<sub>3</sub>: C, 60.73; H, 4.43; N, 12.32. Found: C, 60.82; H, 4.49; N, 12.20. Analytical HPLC 99.3% purity.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (3<i>S</i>)-5-Methyl-3-(2-(2-methylbenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16k</b>)</h4><div class="NLM_p last">Compound <b>16k</b> was obtained (44%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 417.1 [M + H]<sup>+</sup>. Analytical HPLC 98.8% purity.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (3<i>S</i>)-3-(2-(2-Chlorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16l</b>)</h4><div class="NLM_p last">Compound <b>16l</b> was obtained (46%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 437.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.65–2.78 (1H, m), 2.80–2.92 (1H, m), 3.30 (3H, s), 3.52–3.63 (1H, m), 3.93–4.04 (1H, m), 4.32 (1H, dd, <i>J</i> = 10.0, 7.8 Hz), 4.83 (1H, dd, <i>J</i> = 12.0, 10.3 Hz), 5.47 (2H, s), 5.57 (1H, dd, <i>J</i> = 11.9, 7.9 Hz), 6.99–7.07 (1H, m), 7.21–7.40 (5H, m), 7.45–7.54 (2H, m), 7.75 (1H, s). Analytical HPLC 100% purity.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (3<i>S</i>)-3-(2-(3-Chlorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16m</b>)</h4><div class="NLM_p last">Compound <b>16m</b> was obtained (29%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 437.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (3<i>S</i>)-3-(2-(4-Chlorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16n</b>)</h4><div class="NLM_p last">Compound <b>16n</b> was obtained (7%) as a white solid in a manner similar to that described for the synthesis of <b>14</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 437.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.65–2.75 (1H, m), 2.80–2.90 (1H, m), 3.30 (3H, s, overlap with water signal), 3.54–3.65 (1H, m), 3.90–4.00 (1H, m), 4.26–4.36 (1H, m), 4.76–4.86 (1H, m), 5.36 (2H, s), 5.52–5.62 (1H, m), 7.20–7.35 (5H, m), 7.41 (2H, d, <i>J</i> = 8.4 Hz), 7.47–7.54 (1H, m), 7.77 (1H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.8, 51.2, 56.0, 74.8, 121.4, 122.6, 123.3, 125.7, 125.9, 127.2, 129.1, 129.2, 134.3, 136.7, 143.0, 150.0, 161.3, 169.5. Anal. Calcd for C<sub>23</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>3</sub>: C, 63.23; H, 4.84; N, 12.82. Found: C, 62.83; H, 4.84; N, 12.52. Analytical HPLC 99.6% purity.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (3<i>S</i>)-5-Methyl-3-(2-(4-(methylsulfonyl)benzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16o</b>)</h4><div class="NLM_p last">Compound <b>16o</b> was obtained (10%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 481.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.75 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 3.00–3.07 (3H, m), 3.14 (1H, ddd, <i>J</i> = 15.4, 10.7, 4.5 Hz), 3.39 (3H, s), 3.57 (1H, td, <i>J</i> = 11.2, 4.3 Hz), 4.25 (1H, dt, <i>J</i> = 12.0, 5.1 Hz), 4.43 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.64 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.43 (2H, s), 5.94 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 7.14–7.30 (5H, m), 7.38 (2H, d, <i>J</i> = 8.3 Hz), 7.90 (2H, d, <i>J</i> = 8.3 Hz). Analytical HPLC 99.3% purity.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (3<i>S</i>)-5-Methyl-3-(2-(3-(methylsulfonyl)benzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16p</b>)</h4><div class="NLM_p last">Compound <b>16p</b> was obtained (34%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 481.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.75 (1H, dt, <i>J</i> = 15.7, 4.6 Hz), 3.04 (3H, s), 3.08–3.23 (1H, m), 3.39 (3H, s), 3.58 (1H, td, <i>J</i> = 11.3, 4.2 Hz), 4.25 (1H, dt, <i>J</i> = 12.1, 5.3 Hz), 4.44 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.64 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.43 (2H, d, <i>J</i> = 1.9 Hz), 5.94 (1H, dd, <i>J</i> = 11.7, 7.9 Hz), 7.13–7.30 (5H, m), 7.45–7.63 (2H, m), 7.80 (1H, s), 7.85–7.92 (1H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 44.5, 45.7, 51.3, 55.8, 74.8, 121.6, 122.6, 123.2, 125.9, 126.2, 126.3, 127.2, 130.1, 132.9, 136.7, 137.8, 141.2, 143.5, 149.9, 159.7, 161.2, 169.5. Analytical HPLC 99.1% purity.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-((6-((3<i>S</i>)-5-Methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-2-yl)methyl)benzonitrile (<b>16q</b>)</h4><div class="NLM_p last">Compound <b>16q</b> was obtained (17%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 428.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-((6-((3<i>S</i>)-5-Methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-2-yl)methyl)benzonitrile (<b>16r</b>)</h4><div class="NLM_p last">Compound <b>16r</b> was obtained (37%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 428.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-((6-((3<i>S</i>)-5-Methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-2-yl)methyl)benzonitrile (<b>16s</b>)</h4><div class="NLM_p last">Compound <b>16s</b> was obtained (21%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 428.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 3-Chloro-2-((6-((3<i>S</i>)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-2-yl)methyl)benzonitrile (<b>16t</b>)</h4><div class="NLM_p last">Compound <b>16t</b> was obtained (41%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 462.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.71 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 3.10 (1H, ddd, <i>J</i> = 15.6, 10.7, 4.7 Hz), 3.37 (3H, s), 3.53 (1H, td, <i>J</i> = 11.2, 4.3 Hz), 4.21 (1H, dt, <i>J</i> = 11.7, 5.3 Hz), 4.39 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.62 (1H, dd, <i>J</i> = 11.7, 9.8 Hz), 5.67 (2H, s), 5.92 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.12–7.26 (5H, m), 7.40–7.49 (1H, m), 7.62–7.71 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.7, 51.1, 52.2, 74.8, 115.9, 116.5, 121.0, 122.6, 123.2, 125.8, 126.1, 127.2, 130.6, 131.9, 134.9, 135.7, 136.8, 137.1, 143.3, 149.9, 161.1, 169.5. Analytical HPLC 93.9% purity.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 3-Fluoro-4-((6-((3<i>S</i>)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-2-yl)methyl)benzonitrile (<b>16u</b>)</h4><div class="NLM_p last">Compound <b>16u</b> was obtained (13%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 446.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.75 (1H, dt, <i>J</i> = 15.5, 4.9 Hz), 3.13 (1H, ddd, <i>J</i> = 15.5, 10.6, 4.5 Hz), 3.38 (3H, s), 3.50–3.64 (1H, m), 4.24 (1H, dt, <i>J</i> = 12.1, 5.3 Hz), 4.42 (1H, dd, <i>J</i> = 10.2, 8.3 Hz), 4.64 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.44 (2H, s), 5.93 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 7.12–7.27 (5H, m), 7.31 (1H, s), 7.35–7.43 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.7, 49.6 (d, <i>J</i> = 4.4 Hz), 51.3, 74.8, 113.8 (d, <i>J</i> = 9.4 Hz), 117.1 (d, <i>J</i> = 3.3 Hz), 119.2 (d, <i>J</i> = 24.8 Hz), 121.6, 122.6, 123.3, 125.9, 126.5, 127.2, 128.7 (d, <i>J</i> = 4.4 Hz), 129.1 (d, <i>J</i> = 14.9 Hz), 131.1 (d, <i>J</i> = 4.4 Hz), 136.7, 143.8, 149.9, 159.5 (d, <i>J</i> = 251 Hz), 161.0, 169.4. Anal. Calcd for C<sub>24</sub>H<sub>20</sub>FN<sub>5</sub>O<sub>3</sub>: C, 64.71; H, 4.53; N, 15.72. Found: C, 64.50; H, 4.53; N, 15.56. Analytical HPLC 99.1% purity.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (3<i>S</i>)-3-(2-(4-Methoxybenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16v</b>)</h4><div class="NLM_p last">Compound <b>16v</b> was obtained (65%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 433.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.68 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 3.02–3.18 (1H, m), 3.38 (3H, s), 3.46–3.60 (1H, m), 3.79 (3H, s), 4.15–4.28 (1H, m), 4.42 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.62 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.27 (2H, s), 5.95 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 6.81–7.01 (2H, m), 7.10 (1H, s), 7.12–7.30 (6H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.6, 45.8, 51.2, 55.3, 56.3, 74.8, 114.2, 121.2, 122.6, 123.3, 125.4, 125.8, 127.2, 127.7, 129.6, 136.8, 142.5, 150.0, 159.6, 161.5, 169.6. Analytical HPLC 98.6% purity.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (3<i>S</i>)-3-(2-(Cyclopentylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16w</b>)</h4><div class="NLM_p last">Compound <b>16w</b> was obtained (21%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 395.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (3<i>S</i>)-3-(2-(1,4-Dioxan-2-ylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16x</b>)</h4><div class="NLM_p last">Compound <b>16x</b> was obtained (22%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 413.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(tetrahydro-2<i>H</i>-pyran-3-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16y</b>)</h4><div class="NLM_p last">Compound <b>16y</b> was obtained (29%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 411.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.21–1.38 (1H, m), 1.46–1.80 (3H, m), 2.14–2.36 (1H, m), 2.73 (1H, dt, <i>J</i> = 15.49, 4.91 Hz), 3.04–3.28 (2H, m), 3.38 (3H, s), 3.42–3.63 (2H, m), 3.63–3.82 (2H, m), 3.98–4.17 (2H, m), 4.23 (1H, dt, <i>J</i> = 11.80, 5.24 Hz), 4.43 (1H, dd, <i>J</i> = 9.82, 8.31 Hz), 4.63 (1H, dd, <i>J</i> = 11.52, 10.01 Hz), 5.94 (1H, dd, <i>J</i> = 11.52, 8.12 Hz), 7.13–7.26 (5H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 19.5, 23.3, 25.7, 34.6, 35.6, 44.8, 50.2, 53.5, 67.5, 69.3, 73.8, 119.6, 121.6, 122.2, 124.8, 125.3, 126.1, 135.8, 141.9, 149.0, 160.3, 168.6. Analytical HPLC 98.6% purity.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(pyrazin-2-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16z</b>)</h4><div class="NLM_p last">Compound <b>16z</b> was obtained (15%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 405.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.76 (1H, dt, <i>J</i> = 15.5, 4.5 Hz), 3.15 (1H, ddd, <i>J</i> = 15.7, 11.1, 4.9 Hz), 3.38 (3H, s), 3.45–3.64 (1H, m), 4.24 (1H, dt, <i>J</i> = 11.9, 5.0 Hz), 4.38–4.48 (1H, m), 4.63 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.39–5.58 (2H, m), 5.94 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 7.13–7.25 (4H, m), 7.41 (1H, s), 8.48 (1H, s), 8.53 (2H, s). Analytical HPLC 99.5% purity.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (3<i>S</i>)-3-(2-(3-Furylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16aa</b>)</h4><div class="NLM_p last">Compound <b>16aa</b> was obtained (11%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 393.1 [M + H]<sup>+</sup>. Analytical HPLC 98.0% purity.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (3<i>S</i>)-5-Methyl-3-(2-((3-methyloxetan-3-yl)methyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ab</b>)</h4><div class="NLM_p last">Compound <b>16ab</b> was obtained (27%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 397.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.24 (3H, s), 2.74 (1H, dt, <i>J</i> = 15.49, 4.91 Hz), 3.04–3.20 (1H, m), 3.38 (3H, s), 3.56 (1H, ddd, <i>J</i> = 11.90, 10.58, 4.34 Hz), 4.18–4.31 (1H, m), 4.36 (4H, dd, <i>J</i> = 6.42, 3.02 Hz), 4.42 (1H, dd, <i>J</i> = 9.82, 8.31 Hz), 4.57–4.74 (3H, m), 5.93 (1H, dd, <i>J</i> = 11.71, 7.93 Hz), 7.12–7.32 (5H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.4, 21.8, 35.6, 40.6, 45.7, 51.1, 59.1, 74.8, 80.1, 80.1, 120.7, 122.6, 123.2, 125.9, 126.5, 127.2, 136.8, 143.1, 149.9, 161.3, 169.6. Anal. Calcd for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub>: C, 63.62; H, 6.10; N, 14.13. Found: C, 63.28; H, 6.07; N, 13.89. Analytical HPLC 98.6% purity.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (3<i>S</i>)-5-Methyl-3-(2-(1,3-oxazol-2-ylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ac</b>)</h4><div class="NLM_p last">Compound <b>16ac</b> was obtained (37%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 394.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (3<i>S</i>)-5-Methyl-3-(2-((1-methylcyclobutyl)methyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ad</b>)</h4><div class="NLM_p last">Compound <b>16ad</b> was obtained (18%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 395.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.07 (3H, s), 1.64–2.12 (6H, m), 2.74 (1H, dt, <i>J</i> = 15.5, 4.9 Hz), 3.05–3.21 (1H, m), 3.38 (3H, s), 3.47–3.66 (1H, m), 4.03–4.17 (2H, m), 4.23 (1H, dt, <i>J</i> = 12.0, 5.1 Hz), 4.43 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.64 (1H, dd, <i>J</i> = 11.7, 9.8 Hz), 5.96 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.10–7.25 (5H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 14.8, 20.5, 24.7, 31.1, 31.1, 35.6, 39.6, 45.8, 51.1, 62.2, 74.9, 120.5, 122.6, 123.2, 125.8, 126.1, 127.1, 136.8, 142.2, 150.0, 161.5, 169.7. Analytical HPLC 99.4% purity.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(tetrahydrofuran-3-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ae</b>)</h4><div class="NLM_p last">Compound <b>16ae</b> was obtained (15%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 397.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.56–1.71 (1H, m), 1.93–2.09 (1H, m), 2.74 (1H, dt, <i>J</i> = 15.39, 4.77 Hz), 2.89 (1H, td, <i>J</i> = 7.37, 2.64 Hz), 3.12 (1H, ddd, <i>J</i> = 15.49, 10.58, 4.91 Hz), 3.38 (3H, s), 3.50–3.62 (2H, m), 3.69–3.79 (2H, m), 3.84–3.95 (1H, m), 4.08–4.17 (2H, m), 4.24 (1H, dt, <i>J</i> = 11.99, 5.15 Hz), 4.42 (1H, dd, <i>J</i> = 9.82, 8.31 Hz), 4.64 (1H, dd, <i>J</i> = 11.71, 10.20 Hz), 5.94 (1H, dd, <i>J</i> = 11.52, 8.12 Hz), 7.13–7.26 (5H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 29.6, 35.6, 40.0, 45.8, 51.2, 55.3, 67.6, 70.8, 74.8, 120.6, 120.7, 122.6, 123.2, 125.9, 126.0, 126.0, 127.2, 136.8, 143.1, 149.9, 161.3, 169.6. Analytical HPLC 98.6% purity.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (3<i>S</i>)-3-(2-((3,3-Difluorocyclobutyl)methyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16af</b>)</h4><div class="NLM_p last">Compound <b>16af</b> was obtained (16%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 417.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(pyrimidin-2-ylmethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ag</b>)</h4><div class="NLM_p last">Compound <b>16ag</b> was obtained (4%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 405.1 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.6, 35.6, 45.8, 51.2, 58.4, 74.8, 120.1, 121.1, 122.6, 123.3, 125.9, 127.2, 127.3, 136.8, 143.3, 150.0, 157.5, 161.3, 164.9, 169.5. Anal. Calcd for C<sub>21</sub>H<sub>20</sub>N<sub>6</sub>O<sub>3</sub> + 0.6H<sub>2</sub>O: C, 60.74; H, 5.15; N, 20.24. Found: C, 60.44; H, 4.93; N, 20.24. Analytical HPLC 96.5% purity.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (3<i>S</i>)-3-(2-(3-Methoxybenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ah</b>)</h4><div class="NLM_p last">Compound <b>16ah</b> was obtained (29%) as a white solid in a manner similar to that described for the synthesis of <b>16a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 433.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.70 (1H, dt, <i>J</i> = 15.4, 4.8 Hz), 2.99–3.18 (1H, m), 3.38 (3H, s), 3.54 (1H, td, <i>J</i> = 11.3, 4.2 Hz), 3.77 (3H, s), 4.22 (1H, dt, <i>J</i> = 12.0, 5.1 Hz), 4.42 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.63 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.31 (2H, s), 5.96 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 6.73–6.88 (3H, m), 7.10–7.31 (6H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.6, 35.6, 45.8, 51.2, 55.3, 56.7, 74.8, 113.6, 113.9, 120.3, 121.3, 122.6, 123.3, 125.7, 125.9, 127.2, 129.9, 136.8, 137.2, 142.7, 150.0, 160.0, 161.4, 169.6. Anal. Calcd for C<sub>24</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub> + 0.2H<sub>2</sub>O: C, 66.10; H, 5.64; N, 12.85. Found: C, 65.97; H, 5.59; N, 12.55. Analytical HPLC 98.4% purity.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (3<i>S</i>)-3-(2-(Cyclopropylmethyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16ai</b>)</h4><div class="NLM_p last">Compound <b>16ai</b> was obtained (20%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 367.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (3<i>S</i>)-5-Methyl-3-(7-oxo-2-(2-phenylethyl)-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>16aj</b>)</h4><div class="NLM_p last">Compound <b>16aj</b> was obtained (25%) in a manner similar to that described for the synthesis of <b>16d</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 417.1 [M + H]<sup>+</sup>. Analytical HPLC 100% purity.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (3<i>S</i>)-3-(2-Benzyl-3-methyl-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">To a solution of <b>60</b> (25 mg, 0.05 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (6.00 mg, 5.19 μmol), and 2,4,6-trimethyl-1,3,5,2,4,6-trioxatriborinane (commercially available, 6.52 mg, 0.05 mmol) in THF (5 mL, 0.05 mmol) was added potassium hydroxide (1.039 mL, 0.10 mmol) and at room temperature. The mixture was stirred at 100 °C under Ar for 1 h. The mixture was stirred at room temperature under Ar overnight. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 50–70% EtOAc in hexane). The residue was purified by column chromatography (NH silica gel, eluted with 70–90%EtOAc in hexane). The residue was purified by preparative HPLC (L-Column 2 ODS, eluted with H<sub>2</sub>O in acetonitrile containing 0.1% TFA). The desired fraction was concentrated under reduced pressure and neutralized with saturated NaHCO<sub>3</sub> aq and extracted with EtOAc. The organic layer was separated, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (NH silica gel, eluted with 50–70% EtOAc in hexane). The residue was purified by column chromatography (NH silica gel, eluted with 70–90% EtOAc in hexane) to give <b>17</b> (2.00 mg, 9%). LC-MS (APCI) <i>m</i>/<i>z</i> 417.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.10 (3H, s), 2.60 (1H, dt, <i>J</i> = 15.3, 4.8 Hz), 3.01 (1H, ddd, <i>J</i> = 15.2, 10.4, 4.7 Hz), 3.38 (3H, s), 3.55 (1H, ddd, <i>J</i> = 11.8, 10.5, 4.3 Hz), 4.24 (1H, dt, <i>J</i> = 11.9, 5.2 Hz), 4.43 (1H, dd, <i>J</i> = 9.9, 8.2 Hz), 4.64 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.35 (2H, s), 5.97 (1H, dd, <i>J</i> = 11.6, 8.2 Hz), 7.07–7.36 (9H, m). Analytical HPLC 99.1% purity.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (3<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">Compound <b>18</b> was obtained (23%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 432.7 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.66 (1H, dt, <i>J</i> = 15.7, 4.6 Hz), 3.06 (1H, ddd, <i>J</i> = 15.5, 10.4, 5.1 Hz), 3.38 (3H, s), 3.56 (1H, ddd, <i>J</i> = 12.0, 10.5, 4.3 Hz), 4.25 (1H, dt, <i>J</i> = 12.0, 5.1 Hz), 4.42 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.62 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.39 (2H, s), 5.92 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.13–7.38 (9H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 19.8, 35.6, 45.4, 51.4, 53.8, 74.7, 118.1, 122.6, 123.2, 123.3, 125.9, 127.2, 127.7, 128.2, 128.7, 135.1, 136.7, 142.3, 149.9, 160.7, 169.4. Anal. Calcd for C<sub>23</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>3</sub> + 0.5H<sub>2</sub>O: C, 61.95; H, 4.97; N, 12.56. Found: C, 62.36; H, 5.35; N, 12.56. Analytical HPLC 100% purity.</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-Benzyl-6-((3<i>S</i>)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine-3-carbonitrile (<b>19</b>)</h4><div class="NLM_p last">To a solution of <b>60</b> (15 mg, 0.03 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (5 mL, 0.03 mmol) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (3.60 mg, 3.12 μmol) and dicyanozinc (7.32 mg, 0.06 mmol) at room temperature. The mixture was stirred at 100 °C under Ar for 1 h. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 50–100% EtOAc in hexane). The residue was purified by column chromatography (NH silica gel, eluted with 50–75% EtOAc in hexane) to give <b>19</b> (12.0 mg, 90%). LC-MS (APCI) <i>m</i>/<i>z</i> 428.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.86 (1H, dt, <i>J</i> = 16.2, 4.7 Hz), 3.24 (1H, ddd, <i>J</i> = 16.1, 10.6, 5.1 Hz), 3.38 (3H, s), 3.57 (1H, ddd, <i>J</i> = 12.1, 10.6, 4.2 Hz), 4.29 (1H, dt, <i>J</i> = 12.3, 5.0 Hz), 4.41 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.62 (1H, dd, <i>J</i> = 11.7, 9.8 Hz), 5.50 (2H, s), 5.88 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 7.11–7.42 (9H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.3, 35.6, 45.0, 51.4, 56.6, 74.6, 109.6, 111.7, 122.6, 123.3, 126.0, 127.3, 128.3, 129.0, 129.1, 129.3, 134.0, 136.6, 142.9, 149.9, 159.6, 169.1. Analytical HPLC 96.3% purity.</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-Benzyl-6-((3<i>S</i>)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepin-3-yl)-7-oxo-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine-3-carboxamide (<b>20</b>)</h4><div class="NLM_p last">To a solution of <b>19</b> (40 mg, 0.09 mmol) and potassium carbonate (15.52 mg, 0.11 mmol) in DMSO (3 mL) was added H<sub>2</sub>O<sub>2</sub> (0.041 mL, 0.47 mmol) at room temperature. The mixture was stirred at room temperature under air for 1 h. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 80–100% EtOAc in hexane). The obtained solids were triturated in EtOAc and collected by filtration to give <b>20</b> (25.0 mg, 60%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 446.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.79–2.92 (1H, m), 2.99–3.12 (1H, m), 3.30 (3H, s), 3.52–3.67 (1H, m), 3.94–4.08 (1H, m), 4.34 (1H, dd, <i>J</i> = 10.0, 7.7 Hz), 4.85 (1H, dd, <i>J</i> = 11.9, 10.4 Hz), 5.55 (1H, dd, <i>J</i> = 12.1, 7.9 Hz), 5.63 (2H, s), 7.12–7.20 (2H, m), 7.21–7.37 (6H, m), 7.50 (1H, dd, <i>J</i> = 7.6, 1.9 Hz), 7.70 (1H, brs), 7.79 (1H, brs). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 21.5, 35.6, 45.1, 51.3, 55.6, 74.7, 121.9, 122.6, 123.2, 126.0, 127.4, 128.0, 128.0, 128.5, 131.0, 136.3, 136.6, 141.6, 149.9, 160.6, 160.8, 169.4. Analytical HPLC 100% purity.</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (3<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-7-carbonitrile (<b>21</b>)</h4><div class="NLM_p last">Compound <b>21</b> was obtained (29%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 462.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.68 (1H, dt, <i>J</i> = 15.7, 4.8 Hz), 3.05 (1H, ddd, <i>J</i> = 15.6, 10.3, 5.1 Hz), 3.39 (3H, s), 3.48–3.61 (1H, m), 4.15–4.28 (1H, m), 4.45 (1H, dd, <i>J</i> = 9.8, 7.6 Hz), 4.73 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.40 (2H, s), 5.89 (1H, dd, <i>J</i> = 11.9, 7.7 Hz), 7.21–7.37 (6H, m), 7.48–7.58 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 19.7, 35.8, 45.3, 51.0, 53.9, 75.0, 109.7, 117.7, 118.0, 123.4, 124.1, 127.1, 127.7, 128.2, 128.8, 131.0, 135.0, 137.8, 141.9, 153.3, 160.7, 169.0. Anal. Calcd for C<sub>24</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>3</sub> + 0.2H<sub>2</sub>O: C, 61.92; H, 4.42; N, 15.04. Found: C, 61.92; H, 4.44; N, 14.79. Analytical HPLC 100% purity.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (3<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carbonitrile (<b>22</b>)</h4><div class="NLM_p last">To a solution of <b>72a</b> (1.08 g, 2.09 mmol) and dicyanozinc (0.369 g, 3.14 mmol) in DMF (20 mL) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (0.484 g, 0.42 mmol) at room temperature. The mixture was stirred at 100 °C under Ar for 2 h. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 40–55% EtOAc in hexane). The obtained oil was triturated in Et<sub>2</sub>O–IPE–EtOAc, and the formed solid was collected by filtration to give <b>22</b> (0.600 g, 1.299 mmol, 62.0%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 462.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.69 (1H, dt, <i>J</i> = 15.7, 4.8 Hz), 3.05 (1H, ddd, <i>J</i> = 15.6, 10.3, 5.1 Hz), 3.39 (3H, s), 3.51–3.61 (1H, m), 4.15–4.27 (1H, m), 4.45 (1H, dd, <i>J</i> = 10.2, 7.9 Hz), 4.69 (1H, dd, <i>J</i> = 11.9, 10.0 Hz), 5.40 (2H, s), 5.89 (1H, dd, <i>J</i> = 11.7, 7.9 Hz), 7.20–7.38 (6H, m), 7.48 (1H, d, <i>J</i> = 1.9 Hz), 7.57 (1H, dd, <i>J</i> = 8.3, 1.9 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 19.7, 35.6, 45.3, 51.2, 53.9, 74.9, 110.2, 117.6, 118.0, 123.4, 123.8, 126.7, 127.7, 128.2, 128.7, 129.8, 135.0, 141.5, 141.9, 149.9, 160.7, 169.1. Anal. Calcd for C<sub>24</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>3</sub> + 0.2H<sub>2</sub>O: C, 61.92; H, 4.42; N, 15.04. Found: C, 61.78; H, 4.69; N, 14.78. Analytical HPLC 99.4% purity.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (3<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-<i>N</i>,5-dimethyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carboxamide (<b>23</b>)</h4><div class="NLM_p last">Compound <b>23</b> was obtained (55%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 494.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.66 (1H, dt, <i>J</i> = 15.8, 4.8 Hz), 2.91–3.12 (4H, m), 3.38 (3H, s), 3.53 (1H, td, <i>J</i> = 11.1, 4.5 Hz), 4.17–4.28 (1H, m), 4.34 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.62 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.39 (2H, s), 5.86 (1H, dd, <i>J</i> = 11.9, 8.1 Hz), 6.41 (1H, d, <i>J</i> = 4.5 Hz), 7.19–7.37 (6H, m), 7.54–7.69 (2H, m). Analytical HPLC 99.0% purity.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (3<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-<i>N</i>,<i>N</i>,5-trimethyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carboxamide (<b>24</b>)</h4><div class="NLM_p last">Compound <b>24</b> was obtained (42%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 508.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.67 (1H, dt, <i>J</i> = 15.7, 4.8 Hz), 2.96–3.19 (7H, m), 3.38 (3H, s), 3.56 (1H, ddd, <i>J</i> = 11.9, 10.4, 4.5 Hz), 4.23 (1H, dt, <i>J</i> = 11.9, 5.2 Hz), 4.44 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.66 (1H, dd, <i>J</i> = 11.7, 9.8 Hz), 5.39 (2H, s), 5.93 (1H, dd, <i>J</i> = 11.7, 7.9 Hz), 7.21–7.38 (8H, m). Analytical HPLC 99.7% purity.</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (3<i>S</i>)-3-(2-(2-Fluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carbonitrile (<b>25</b>)</h4><div class="NLM_p last">To a mixture of <b>71e</b> (3.51 g, 7.03 mmol) and dicyanozinc (1.238 g, 10.54 mmol) in DMF (dry) (35 mL) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (2.031 g, 1.76 mmol) at room temperature. The mixture was stirred at 100 °C under Ar for 2.5 h. The mixture was quenched with diluted saturated NaHCO<sub>3</sub> aq at 0 °C and extracted with EtOAc. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 80–100% EtOAc in hexane) to give target compound (2.85 g) with triphenylphosphine oxide. To remove triphenylphosphine oxide from the residue magnesium chloride (0.669 g, 7.03 mmol) and toluene (10 mL) were added. The mixture was stirred at 50 °C for 30 min. The mixture was purified directly by column chromatography (silica gel, eluted with 80–100% EtOAc in hexane) to give <b>25</b> (2.57 g, 82%) as a white amorphous powder. LC-MS (APCI) <i>m</i>/<i>z</i> 446.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.73 (1H, dt, <i>J</i> = 15.5, 4.9 Hz), 3.10 (1H, ddd, <i>J</i> = 15.2, 10.3, 4.7 Hz), 3.39 (3H, s), 3.54 (1H, ddd, <i>J</i> = 11.8, 10.5, 4.2 Hz), 4.14–4.22 (1H, m), 4.44 (1H, dd, <i>J</i> = 10.2, 7.9 Hz), 4.68 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.40 (2H, s), 5.92 (1H, dd, <i>J</i> = 11.9, 8.1 Hz), 7.04–7.14 (2H, m), 7.21–7.36 (4H, m), 7.47 (1H, d, <i>J</i> = 1.9 Hz), 7.56 (1H, dd, <i>J</i> = 8.3, 1.9 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.4, 35.5, 45.7, 50.2 (d, <i>J</i> = 4.4 Hz), 51.1, 75.0, 110.1, 115.6 (d, <i>J</i> = 21.4 Hz), 117.6, 121.2, 122.8 (d, <i>J</i> = 14.3 Hz), 123.8, 124.6 (d, <i>J</i> = 3.8 Hz), 126.1, 126.7, 129.7, 130.4 (d, <i>J</i> = 8.2 Hz), 130.6 (d, <i>J</i> = 3.3 Hz), 141.6, 142.6, 149.9, 160.4 (d, <i>J</i> = 248 Hz), 161.4, 169.3. Anal. Calcd for C<sub>24</sub>H<sub>20</sub>FN<sub>5</sub>O<sub>3</sub> + 0.2H<sub>2</sub>O: C, 64.19; H, 4.58; N, 15.60. Found: C, 63.91; H, 4.67; N, 15.39. Analytical HPLC 98.9% purity.</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (3<i>S</i>)-3-(2-(2,6-Difluorobenzyl)-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-4-oxo-2,3,4,5-tetrahydro-1,5-benzoxazepine-8-carbonitrile (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was obtained (85%) as a pale-yellow amorphous solid in a manner similar to that described for the synthesis of <b>22</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 464.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.71 (1H, dt, <i>J</i> = 15.4, 4.8 Hz), 2.99–3.16 (1H, m), 3.38 (3H, s), 3.46–3.59 (1H, m), 4.12–4.21 (1H, m), 4.42 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.67 (1H, dd, <i>J</i> = 11.9, 10.0 Hz), 5.45 (2H, s), 5.91 (1H, dd, <i>J</i> = 11.9, 8.1 Hz), 6.88–7.00 (2H, m), 7.23 (1H, s), 7.28–7.41 (2H, m), 7.46 (1H, d, <i>J</i> = 1.9 Hz), 7.56 (1H, dd, <i>J</i> = 8.3, 1.9 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 20.5, 35.5, 44.0 (t, <i>J</i> = 3.9 Hz), 45.7, 51.1, 75.0, 110.1, 111.5 (m), 111.8 (m), 117.7, 121.1, 123.8, 125.8, 126.7, 129.7, 131.1 (t, <i>J</i> = 10.5 Hz), 141.6, 142.5, 149.9, 161.3, 161.5 (dd, <i>J</i> = 251, 7.2 Hz), 169.3. Analytical HPLC 97.5% purity.</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Methyl 2-Benzyl-1<i>H</i>-imidazole-5-carboxylate (<b>28</b>)</h4><div class="NLM_p last">To a solution of (<i>Z</i>)-<i>N</i>′-hydroxy-2-phenylacetimidamide (<b>27</b>) (commercially available, 1.0 g, 6.66 mmol) in MeOH (20 mL) was added methyl propiolate (0.800 mL, 8.99 mmol). The mixture was refluxed for 5 h and concentrated under reduced pressure. The residue was dissolved in diphenyl ether (10 mL) was heated 200 °C for 1 h under microwave irradiation. The mixture was diluted with EtOAc, 1 M HCl aq (10 mL), and water. The aqueous layer was neutralized with saturated NaHCO<sub>3</sub> aq and extracted with EtOAc. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residual solid was washed with EtOAc/hexane to give <b>28</b> (0.510 g, 35%) as a pale-brown solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.71 (3H × 2/3, s), 3.77 (3H × 1/3, s), 3.98 (2H, s), 7.14–7.36 (5H, m), 7.50 (1H × 1/3, d, <i>J</i> = 1.5 Hz), 7.76 (1H × 2/3, d, <i>J</i> = 2.3 Hz), 12.46 (1H × 2/3, brs), 13.01 (1H × 1/3, brs). LC-MS (APCI) <i>m</i>/<i>z</i> 217.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Methyl 2-Benzyl-5-bromo-1<i>H</i>-imidazole-4-carboxylate (<b>29</b>)</h4><div class="NLM_p last">To a mixture of <b>28</b> (1.37 g, 6.34 mmol) and potassium bicarbonate (0.761 g, 7.60 mmol) in DMF (20 mL) was added bromine (0.390 mL, 7.60 mmol) at rt. After stirring at rt for 2 h, the mixture was diluted with EtOAc, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 5–50% EtOAc in hexane) and washed with IPE to give <b>29</b> (1.04 g, 56%) as a light-brown solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.86 (3H, s), 4.12 (2H, s), 7.23–7.25 (1H, m), 7.27–7.44 (4H, m), 9.25 (1H, brs). LC-MS (APCI) <i>m</i>/<i>z</i> 295.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Methyl 2-Benzyl-5-bromo-1-(4-methoxybenzyl)-1<i>H</i>-imidazole-4-carboxylate (<b>30</b>)</h4><div class="NLM_p last">A mixture of <b>29</b> (1.04 g, 3.52 mmol), 4-methoxybenzyl chloride (0.285 mL, 2.10 mmol), and potassium carbonate (0.42 g, 3.04 mmol) in DMF (10 mL) was stirred at rt for 1 day and stirred at 50 °C for 4 h. The mixture was diluted with water and extracted with EtOAc. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 5–70% EtOAc in hexane) to give <b>30</b> (0.300 g, 21%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.79 (3H, s), 3.96 (3H, s), 4.06 (2H, s), 4.93 (2H, s), 6.78–6.89 (4H, m), 7.07–7.15 (2H, m), 7.18–7.32 (3H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 415.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (<i>E</i>)-Methyl 2-Benzyl-5-(2-ethoxyvinyl)-1-(4-methoxybenzyl)-1<i>H</i>-imidazole-4-carboxylate (<b>31</b>)</h4><div class="NLM_p last">A mixture of <b>30</b> (0.30 g, 0.72 mmol), (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (commercially available, 0.286 g, 1.44 mmol), Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub> (0.059 g, 0.07 mmol), and cesium carbonate (0.518 g, 1.59 mmol) in DME (8 mL)/water (1 mL) was stirred at 90 °C overnight under Ar atmosphere. The mixture was diluted with EtOAc, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 20–100% EtOAc in hexane) to give <b>31</b> (0.220 g, 75%) as a light-brown oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.25–1.32 (3H, m), 3.78 (3H, s), 3.87 (2H, q, <i>J</i> = 7.2 Hz), 3.93 (3H, s), 4.03 (2H, s), 4.84 (2H, s), 5.71 (1H, d, <i>J</i> = 12.8 Hz), 6.70–6.86 (4H, m), 7.07–7.26 (5H, m), 7.31 (1H, d, <i>J</i> = 12.8 Hz). LC-MS (APCI) <i>m</i>/<i>z</i> 407.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Methyl 2-Benzyl-1-(4-methoxybenzyl)-5-(2-oxoethyl)-1<i>H</i>-imidazole-4-carboxylate (<b>32</b>)</h4><div class="NLM_p last">To a solution of <b>31</b> (110 mg, 0.27 mmol) in THF (5 mL, 0.27 mmol) was added 6 M HCl aq (1 mL, 6.00 mmol) at room temperature. The mixture was stirred at room temperature for 2 h. To the mixture was added 6 M HCl aq (2 mL, 12.00 mmol). After stirring at rt for 2 h, the mixture was heated to 60 °C and stirred at 60 °C for 30 min. The mixture was neutralized with saturated NaHCO<sub>3</sub> aq, saturated with NaCl, and extracted with EtOAc. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give <b>32</b> (100 mg, 98%). This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 377.1 [M – H]<sup>+</sup>.</div></div><div id="sec5_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (<i>S</i>)-Methyl 2-Benzyl-1-(4-methoxybenzyl)-5-(2-((5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-imidazole-4-carboxylate (<b>33</b>)</h4><div class="NLM_p last">To a solution of <b>32</b> (100 mg, 0.26 mmol) and (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride (60.4 mg, 0.26 mmol) in acetic acid (0.5 mL, 0.26 mmol) and MeOH (5 mL, 0.26 mmol) was added 2-picoline boran (36.7 mg, 0.34 mmol) at room temperature. The mixture was stirred at room temperature under air for 1 h. The mixture was quenched with saturated NaHCO<sub>3</sub> aq at room temperature and extracted with EtOAc. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 30–70% EtOAc in hexane) to give <b>33</b> (95 mg, 65%) as a colorless oil. LC-MS (APCI) <i>m</i>/<i>z</i> 555.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (<i>S</i>)-3-(2-Benzyl-1-(4-methoxybenzyl)-4-oxo-6,7-dihydro-1<i>H</i>-imidazo[4,5-<i>c</i>]pyridin-5(4<i>H</i>)-yl)-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one (<b>34</b>)</h4><div class="NLM_p last">To a solution of <b>33</b> (95 mg, 0.17 mmol) in toluene (10 mL) was added trimethylaluminum in toluene (0.285 mL, 0.51 mmol) at room temperature. After stirring at 100 °C for 4 h under Ar atmosphere, the mixture was heated to 120 °C and stirred at 120 °C overnight. The mixture was quenched with water at room temperature and diluted with EtOAc. Saturated Rochelle salt aq was added to the mixture. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 50–100% EtOAc in hexane) to give <b>34</b> (45.0 mg, 50%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.53 (1H, dt, <i>J</i> = 16.1, 4.6 Hz), 2.94–3.09 (1H, m), 3.36 (3H, s), 3.58 (1H, td, <i>J</i> = 11.6, 4.7 Hz), 3.79 (3H, s), 4.09 (2H, s), 4.18–4.30 (1H, m), 4.36–4.48 (1H, m), 4.51–4.63 (1H, m), 4.69–4.89 (2H, m), 5.93 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 6.70–6.85 (4H, m), 7.10–7.31 (9H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 523.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Ethyl 4-Benzyl-1<i>H</i>-imidazole-2-carboxylate (<b>37</b>)</h4><div class="NLM_p last">To a solution of 1-amino-3-phenylpropan-2-one hydrochloride (<b>36</b>) (commercially available, 0.93 g, 5.01 mmol) and ethyl 2-ethoxy-2-iminoacetate (commercially available, 0.727 g, 5.01 mmol) in acetic acid (10 mL, 174.68 mmol) was added sodium acetate (0.740 g, 9.02 mmol) at room temperature, and the mixture was stirred for 6 h at 110 °C in a sealed tube. The reaction mixture was concentrated under reduced pressure. To the residue was added saturated NaHCO<sub>3</sub> aq and extracted with EtOAc. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 48–69% EtOAc in hexane) to give <b>37</b> (105 mg, 9.1%) as a brown oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.29 (3H, t, <i>J</i> = 7.2 Hz), 3.85 (1H, s), 3.94 (1H, s), 4.27 (2H, q, <i>J</i> = 7.2 Hz), 7.17–7.35 (6H, m), 12.98–13.34 (1H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 231.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (<i>S</i>)-4-Benzyl-<i>N</i>-(5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)-1<i>H</i>-imidazole-2-carboxamide (<b>38</b>)</h4><div class="NLM_p last">To a solution of <b>37</b> (105 mg, 0.46 mmol) in EtOH (1 mL, 17.04 mmol) was added 2 M sodium hydroxide (0.684 mL, 1.37 mmol) at room temperature, and the mixture was stirred for 30 min at 100 °C. The mixture was concentrated under reduced pressure and dissolved in DMF (2 mL, 25.83 mmol). (<i>S</i>)-3-Amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride (136 mg, 0.59 mmol), HATU (225 mg, 0.59 mmol), and DIPEA (0.238 mL, 1.37 mmol) were added to the mixture. After stirring at room temperature for 2 h, the mixture was partitioned between water and EtOAc. The organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 70–90% EtOAc in hexane) to give <b>38</b> (101 mg, 59%) as a brown amorphous powder. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.41 (3H, d, <i>J</i> = 5.7 Hz), 3.88 (1H, s), 3.95 (1H, s), 4.29 (1H, dt, <i>J</i> = 11.0, 9.4 Hz), 4.64 (1H, dt, <i>J</i> = 9.7, 7.4 Hz), 5.00 (1H, tt, <i>J</i> = 11.0, 7.6 Hz), 6.63–6.74 (0.5H, m), 6.83–6.88 (0.5H, m), 7.02–7.49 (10H, m), 7.99 (1H, t, <i>J</i> = 8.3 Hz), 10.33 (0.5H, brs), 10.54 (0.5H, brs). LC-MS (APCI) <i>m</i>/<i>z</i> 377.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Ethyl 1-Benzyl-1<i>H</i>-pyrazole-3-carboxylate (<b>40</b>)</h4><div class="NLM_p last">To a suspension of ethyl 1<i>H</i>-pyrazole-3-carboxylate (commercially available, <b>39</b>) (13.4 g, 95.62 mmol) and potassium carbonate (39.6 g, 286.86 mmol) in DMF (dry) (200 mL) was added (bromomethyl)benzene (commercially available, 12.51 mL, 105.18 mmol) at room temperature, and the mixture was stirred for 2 h at room temperature. The mixture was filtered, and then the filtrate was diluted with EtOAc and washed with water. The aqueous layer was extracted with EtOAc. The combined organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 5–50% EtOAc in hexane) to give <b>40</b> (14.10 g, 64%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.40 (3H, t, <i>J</i> = 7.0 Hz), 4.42 (2H, q, <i>J</i> = 7.2 Hz), 5.40 (2H, s), 6.82 (1H, d, <i>J</i> = 2.3 Hz), 7.21–7.26 (2H, m), 7.30–7.41 (4H, m).</div></div><div id="sec5_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> Ethyl 1-Benzyl-4-bromo-1<i>H</i>-pyrazole-3-carboxylate (<b>41</b>)</h4><div class="NLM_p last">To a solution of <b>40</b> (6.68 g, 29.01 mmol) in MeCN (120 mL) was added bromine (2.230 mL, 43.52 mmol) at rt. After stirring at rt overnight, an additional bromine (2.230 mL, 43.52 mmol) was added and the mixture was stirred at rt for 1 day. The mixture was concentrated under reduced pressure, diluted with EtOAc, and washed with 10% sodium thiosulfate aq. The aqueous layer was extracted with EtOAc/THF. The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 10–50% EtOAc in hexane) to give <b>41</b> (8.76 g, 98%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.43 (3H, t, <i>J</i> = 7.0 Hz), 4.44 (2H, q, <i>J</i> = 6.9 Hz), 5.35 (2H, s), 7.22–7.44 (6H, m). LC-MS (APCI) <i>m</i>/<i>z</i> 309.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>E</i>)-Ethyl 1-Benzyl-4-(2-ethoxyvinyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>42</b>)</h4><div class="NLM_p last">Compound <b>42</b> was obtained (94%) as a pale-yellow oil in a manner similar to that described for the synthesis of <b>31</b>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ1.31 (3H, t, <i>J</i> = 7.2 Hz), 1.42 (3H, t, <i>J</i> = 7.2 Hz), 3.86 (2H, q, <i>J</i> = 6.8 Hz), 4.42 (2H, q, <i>J</i> = 7.2 Hz), 5.33 (2H, s), 6.19 (1H, d, <i>J</i> = 13.2 Hz), 6.81 (1H, d, <i>J</i> = 13.6 Hz), 7.20–7.28 (3H, m), 7.31–7.41 (3H, m). LC-MS (ESI) <i>m</i>/<i>z</i> 301.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Ethyl 1-Benzyl-4-(2-oxoethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>43</b>)</h4><div class="NLM_p last">Compound <b>43</b> was obtained (94%) as a brown solid in a manner similar to that described for the synthesis of <b>32</b>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ1.40 (3H, t, <i>J</i> = 7.2 Hz), 3.85 (2H, d, <i>J</i> = 0.8 Hz), 4.41 (2H, q, <i>J</i> = 7.2 Hz), 5.37 (2H, s), 7.22–7.46 (6H, m), 9.66–9.78 (1H, m).</div></div><div id="sec5_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (<i>S</i>)-Ethyl 1-Benzyl-4-(2-((5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>44</b>)</h4><div class="NLM_p last">Compound <b>44</b> was obtained (94%) as a brown solid in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 449.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.37 (3H, t, <i>J</i> = 7.0 Hz), 2.43–2.53 (1H, m), 2.76–2.91 (3H, m), 3.38 (3H, s), 3.52 (1H, dd, <i>J</i> = 11.7, 7.6 Hz), 4.06 (1H, dd, <i>J</i> = 11.3, 10.2 Hz), 4.30–4.42 (3H, m), 5.32 (2H, s), 7.10–7.24 (7H, m), 7.29–7.38 (3H, m).</div></div><div id="sec5_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (<i>S</i>)-5-Methyl-3-(7-oxo-4,5-dihydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6(7<i>H</i>)-yl)-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one (<b>45</b>)</h4><div class="NLM_p last">A mixture of <b>14</b> (1.29 g, 3.21 mmol), 1 M HCl aq (7 mL, 7.00 mmol), and 10% palladium on carbon (0.341 g, 0.32 mmol) in MeOH (30 mL, 740.59 mmol) was hydrogenated under balloon pressure at room temperature overnight. The catalyst was removed by filtration, and the filtrate was concentrated under reduced pressure. The residue was dissolved in MeOH (20 mL, 493.73 mmol) and palladium on carbon (0.682 g, 0.64 mmol), and 1 M HCl aq (5 mL, 5.00 mmol) was added to the solution. The mixture was hydrogenated under balloon pressure for 4 h at room. The catalyst was removed by filtration, and the filtrate was concentrated under reduced pressure. The solid was washed with EtOAc/heptane (1/4) to give <b>45</b> (0.820 g, 82%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.84 (1H, d, <i>J</i> = 15.1 Hz), 3.16 (1H, d, <i>J</i> = 7.9 Hz), 3.39 (3H, s), 3.62 (1H, brs), 4.19–4.47 (2H, m), 4.68 (1H, t, <i>J</i> = 10.4 Hz), 5.81 (1H, dd, <i>J</i> = 11.3, 7.6 Hz), 7.11–7.26 (4H, m), 7.78 (1H, s). LC-MS (APCI) <i>m</i>/<i>z</i> 313.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Ethyl 3-(<i>tert</i>-Butoxymethyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>47</b>)</h4><div class="NLM_p last">To a solution of ethyl 4-(<i>tert</i>-butoxy)-3-oxobutanoate (<b>46</b>) (commercially available, 12.0 g, 59.3 mmol) in toluene (100 mL) was added DMF–DMA (10.6 g, 89.0 mmol). The mixture was stirred at 65 °C for 16 h. After cooling to 25 °C, the reaction mixture was concentrated under reduced pressure. To a mixture of the residue and AcOH (20 mL) was added NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O (4.55 g, 90.9 mmol). The mixture was stirred at 25–30 °C for 14 h. The reaction mixture was diluted with EtOAc (300 mL), and it was washed with saturated NaHCO<sub>3</sub> aq (200 mL), brine (200 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with petroleum ether/EtOAc (3:1) to give <b>47</b> (7.00 g, 52%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.23 (9H, s), 1.27 (3H, t, <i>J</i> = 7.2 Hz), 4.21 (2H, q, <i>J</i> = 7.2 Hz), 4.69 (2H, s), 7.80 (1H, s), 13.27 (1H, brs).</div></div><div id="sec5_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> Ethyl 1-Benzyl-3-(<i>tert</i>-butoxymethyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>48</b>)</h4><div class="NLM_p last">Two batches of the reaction were conducted and worked up together after the reaction. To a solution of <b>47</b> (3.00 g, 13.3 mmol) and benzyl chloride (1.68 g, 13.3 mmol) in MeCN (50 mL) was added potassium carbonate (11.0 g, 79.6 mmol). The mixture was stirred at 20–25 °C for 14 h. To a solution of <b>47</b> (7.00 g, 31.0 mmol) and benzyl chloride (2.74 g, 21.7 mmol) in MeCN (50 mL) was added potassium carbonate (18.0 g, 130 mmol). The mixture was stirred at 20–25 °C for 14 h. Two reaction mixtures were combined, diluted with water, and extracted with EtOAc (3 times). The combined organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 9–25% EtOAc in petroleum ether) to give <b>48</b> (9.00 g, 64%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.31 (3H, t, <i>J</i> = 6.8 Hz), 1.33 (9H, s), 4.25 (2H, q, <i>J</i> = 6.8 Hz), 4.71 (2H, s), 5.28 (2H, s), 7.20–7.30 (2H, m), 7.35–7.45 (3H, m), 7.73 (1H, m).</div></div><div id="sec5_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Ethyl 1-Benzyl-3-(hydroxymethyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>49</b>)</h4><div class="NLM_p last">To a solution of <b>48</b> (9.00 g, 28. Five mmol) in DCM (10 mL) was added TFA (5 mL). The mixture was stirred at 20–25 °C for 2 h, quenched with 1 M NaOH aq until pH = 10, diluted with water, and extracted with EtOAc (3 times). The combined organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 9–25% EtOAc in petroleum ether) to give <b>49</b> (6.40 g, 86%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.26 (3H, t, <i>J</i> = 7.2 Hz), 4.19 (2H, q, <i>J</i> = 7.2 Hz), 4.58 (2H, d, <i>J</i> = 6.0 Hz), 4.84 (1H, t, <i>J</i> = 6.0 Hz), 5.32 (2H, s), 7.25–7.40 (5H, m), 8.38 (1H, s).</div></div><div id="sec5_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> Ethyl 1-Benzyl-3-formyl-1<i>H</i>-pyrazole-4-carboxylate (<b>50</b>)</h4><div class="NLM_p last">To a solution of <b>49</b> (4.60 g, 17.7 mmol) in DCM (50 mL) was added MnO<sub>2</sub> (15.0 g, 173 mmol). The reaction was stirred at 40 °C for 14 h. The reaction was concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 9–25% EtOAc in petroleum ether) to give <b>50</b> (2.70 g, 59%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.28 (3H, t, <i>J</i> = 7.6 Hz), 4.27 (2H, q, <i>J</i> = 7.2 Hz), 5.47 (2H, s), 7.25–7.45 (5H, m), 8.63 (1H, s), 10.26 (1H, s).</div></div><div id="sec5_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> (<i>E</i>)-Ethyl 1-Benzyl-3-(2-methoxyvinyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>51</b>)</h4><div class="NLM_p last">To a mixture of (methoxymethyl)triphenylphosphonium chloride (commercially available, 15.8 g, 46.1 mmol) in THF (50 mL) was added <i>t</i>-BuOK (44.9 mL, 44.9 mmol, 1 M in THF) over 1 min at 0 °C. The mixture was stirred at 0 °C for 10 min, and to the mixture was added a solution of <b>50</b> (2.70 g, 10.5 mmol) in THF (20 mL) over 20 min. The mixture was stirred at 0 °C for 30 min, and then it was warmed to 20–25 °C for 12 h. The reaction mixture was diluted with EtOAc (200 mL), and it was washed with water (100 mL) and brine (100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 9–25% EtOAc in petroleum ether) to give <b>51</b> (1.60 g, 53%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.26 (3H, t, <i>J</i> = 7.2 Hz), 4.19 (2H, q, <i>J</i> = 7.2 Hz), 5.28 (2H, s), 6.16 (1H, d, <i>J</i> = 13.2 Hz), 7.25–7.45 (6H, m), 8.34 (1H, s).</div></div><div id="sec5_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Ethyl 1-Benzyl-5-hydroxy-1<i>H</i>-pyrazole-3-carboxylate (<b>55</b>)</h4><div class="NLM_p last">Diethyl but-2-ynedioate (<b>54</b>) (commercially available, 20.42 mL, 127.59 mmol) was added to a mixture of benzylhydrazine hydrochloride (commercially available, 20.24 g, 127.59 mmol) and potassium carbonate (26.5 g, 191.39 mmol) in EtOH (600 mL). The mixture was stirred at 90 °C overnight. After stirring at rt for 2 days, the mixture was acidified with 6 M HCl aq (70 mL) at 0 °C and extracted with EtOAc (twice). The combined organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residual solid was triturated with MeCN (50 mL), and the collected solid was washed with MeCN (30 mL) to give <b>55</b> (11.33 g, 36%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ1.25 (3H, t, <i>J</i> = 7.0 Hz), 4.21 (2H, q, <i>J</i> = 6.9 Hz), 5.16 (2H, s), 5.81 (1H, s), 7.13–7.21 (2H, m), 7.23–7.40 (3H, m), 11.57 (1H, s). LC-MS (APCI) <i>m</i>/<i>z</i> 247.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Ethyl 1-Benzyl-5-bromo-4-formyl-1<i>H</i>-pyrazole-3-carboxylate (<b>56a</b>)</h4><div class="NLM_p last">To a solution of <b>55</b> (3.60 g, 14.62 mmol) and phosphoryl tribromide (7.80 g, 27.21 mmol) in 1,2-dichloroethane (60 mL, 14.62 mmol) was added DMF (2.1 mL, 27.29 mmol) at room temperature. The mixture was stirred at 90 °C under air for 3 h. To the mixture was added phosphoryl tribromide (19.6 g, 68.37 mmol) at rt. The mixture was stirred at 90 °C under air for 19 h. To the mixture was added phosphoryl tribromide (9.64 g, 33.62 mmol) at the same temperature. The mixture was stirred at 90 °C under air for 3 h. The mixture was poured into water with ice. The organic layer was separated. The aqueous layer was extracted with EtOAc. The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure.The residue was purified by column chromatography (silica gel, eluted with 15–30% EtOAc in hexane) to give <b>56a</b> (2.340 g, 48%) as a pale-yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.44 (3H, t, <i>J</i> = 7.2 Hz), 4.49 (2H, q, <i>J</i> = 7.2 Hz), 5.51 (2H, s), 7.26–7.39 (5H, m), 10.43 (1H, s). LC-MS (APCI) <i>m</i>/<i>z</i> 337.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Ethyl 1-Benzyl-5-chloro-4-formyl-1<i>H</i>-pyrazole-3-carboxylate (<b>56b</b>)</h4><div class="NLM_p last">To a solution of <b>55</b> (10.0 g, 40.61 mmol) in DMF (12.50 mL, 162.43 mmol) was dropwisely added phosphoryl trichloride (30.3 mL, 324.86 mmol) at room temperature. The mixture was stirred at 90 °C under air for 7 h. The solvent was removed by rotary evaporation under reduced pressure. To the residue was added saturated NaHCO<sub>3</sub> aq at 0 °C and extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 20–35% EtOAc in hexane) to give <b>56b</b> (4.25 g, 36%) as a pale-yellow solid. LC-MS (APCI) <i>m</i>/<i>z</i> 293.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.44 (3H, t, <i>J</i> = 7.2 Hz), 4.49 (2H, q, <i>J</i> = 7.2 Hz), 5.46 (2H, s), 7.27–7.38 (5H, m), 10.43 (1H, s).</div></div><div id="sec5_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> (<i>E</i>)-Ethyl 1-Benzyl-5-bromo-4-(2-methoxyvinyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>57a</b>)</h4><div class="NLM_p last">Compound <b>57a</b> was obtained (30%) as a colorless oil in a manner similar to that described for the synthesis of <b>51</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 365.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.41 (3H, t, <i>J</i> = 7.2 Hz), 3.69 (3H, s), 4.42 (2H, q, <i>J</i> = 7.2 Hz), 5.47 (2H, s), 6.02 (1H, d, <i>J</i> = 13.2 Hz), 7.12–7.39 (5H, m).</div></div><div id="sec5_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> Ethyl 1-Benzyl-5-chloro-4-(2-methoxyvinyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>57b</b>)</h4><div class="NLM_p last">Compound <b>57b</b> was obtained (42%) as a colorless oil in a manner similar to that described for the synthesis of <b>51</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 321.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.35–1.46 (3H, m), 3.65–3.71 (3H, m), 4.33–4.49 (2H, m), 5.40–5.44 (2H, m), 6.03–6.18 (1H, m), 7.17–7.38 (6H, m).</div></div><div id="sec5_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Ethyl 1-Benzyl-5-bromo-4-(2-oxoethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>58a</b>)</h4><div class="NLM_p last">Compound <b>58a</b> was obtained in a manner similar to that described for the synthesis of <b>32</b>. This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 351.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> Ethyl 1-Benzyl-5-chloro-4-(2-oxoethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>58b</b>)</h4><div class="NLM_p last">Compound <b>58b</b> was obtained in a manner similar to that described for the synthesis of <b>32</b>. This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 307.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> (<i>S</i>)-Ethyl 1-Benzyl-5-bromo-4-(2-((5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>59a</b>)</h4><div class="NLM_p last">Compound <b>59a</b> was obtained (67% in 2 steps from <b>57a</b>) as a colorless oil in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 527.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.36 (3H, t, <i>J</i> = 7.0 Hz), 2.36–2.51 (1H, m), 2.66–2.91 (3H, m), 3.38 (3H, s), 3.55 (1H, dd, <i>J</i> = 11.7, 7.6 Hz), 4.00–4.17 (1H, m), 4.29–4.41 (3H, m), 5.42 (2H, s), 7.09–7.37 (9H, m).</div></div><div id="sec5_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> (<i>S</i>)-Ethyl 1-Benzyl-5-chloro-4-(2-((5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>59b</b>)</h4><div class="NLM_p last">Compound <b>59b</b> was obtained (94% in 2 steps from <b>57b</b>) as a colorless oil in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 483.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.36 (3H, t, <i>J</i> = 7.2 Hz), 2.38–2.53 (1H, m), 2.67–2.92 (3H, m), 3.38 (3H, s), 3.53 (1H, dd, <i>J</i> = 11.7, 7.6 Hz), 4.00–4.09 (1H, m), 4.30–4.41 (3H, m), 5.38 (2H, s), 7.09–7.22 (6H, m), 7.27–7.36 (3H, m).</div></div><div id="sec5_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> (3<i>S</i>)-3-(2-Benzyl-3-bromo-7-oxo-2,4,5,7-tetrahydro-6<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6-yl)-5-methyl-2,3-dihydro-1,5-benzoxazepin-4(5<i>H</i>)-one (<b>60</b>)</h4><div class="NLM_p last">Compound <b>60</b> was obtained (82%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 481.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ2.63 (1H, dt, <i>J</i> = 15.6, 4.9 Hz), 2.98–3.12 (1H, m), 3.38 (3H, s), 3.49–3.64 (1H, m), 4.20–4.32 (1H, m), 4.42 (1H, dd, <i>J</i> = 9.8, 8.3 Hz), 4.63 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 5.43 (2H, s), 5.92 (1H, dd, <i>J</i> = 11.5, 8.1 Hz), 7.13–7.35 (9H, m).</div></div><div id="sec5_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Ethyl 4-Bromo-1-(2-fluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>62a</b>)</h4><div class="NLM_p">To a solution of ethyl 1<i>H</i>-pyrazole-3-carboxylate (commercially available, 13.56 g, 96.76 mmol) in THF (140 mL) was added 60% NaH in oil (4.64 g, 116.11 mmol) at 0 °C. After being stirred at 0 °C for 20 min, 1-(bromomethyl)-2-fluorobenzene (commercially available, 12.84 mL, 106.44 mmol) was added to the reaction mixture at 0 °C. The mixture was stirred at room temperature overnight. The mixture was quenched with saturated NH<sub>4</sub>Cl aq and extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 18–39% EtOAc in hexane) to give ethyl 1-(2-fluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (22.77 g, 95%) as a pale-yellow oil. LC-MS (APCI) <i>m</i>/<i>z</i> 249.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.40 (3H, t, <i>J</i> = 7.2 Hz), 4.41 (2H, q, <i>J</i> = 7.2 Hz), 5.46 (2H, s), 6.83 (1H, d, <i>J</i> = 2.3 Hz), 7.05–7.15 (2H, m), 7.17–7.24 (1H, m), 7.28–7.37 (1H, m), 7.43 (1H, d, <i>J</i> = 1.9 Hz).</div><div class="NLM_p last">A solution of bromine (5.25 mL, 102.37 mmol) in MeCN (30 mL) was added to a solution of ethyl 1-(2-fluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (commercially available, 22.77 g, 91.72 mmol) in MeCN (200 mL) at room temperature. After being stirred at room temperature overnight, TEA (15.34 mL, 110.07 mmol) was added to the reaction mixture. After being stirred at room temperature for 5 min, bromine (2.87 mL, 56.07 mmol) was added to the reaction mixture. The mixture was quenched with saturated NaHCO<sub>3</sub> aq and extracted with EtOAc. The organic layer was separated, washed with Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> aq and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was disolved in MeCN (200 mL) and added a solution of bromine (21.2 g, 132.66 mmol) in MeCN (30 mL) at room temperature. The mixture was stirred at room temperature for 10 min. The mixture was quenched with Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> aq and extracted with EtOAc. The organic layer was separated, washed with saturated NaHCO<sub>3</sub> aq and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 20–40% EtOAc in hexane) to give <b>62a</b> (29.5 g, 98%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 327.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.42 (3H, t, <i>J</i> = 7.2 Hz), 4.43 (2H, q, <i>J</i> = 7.2 Hz), 5.42 (2H, s), 7.08–7.18 (2H, m), 7.22–7.29 (1H, m), 7.32–7.41 (1H, m), 7.48 (1H, s)</div></div><div id="sec5_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Ethyl 4-Bromo-1-(2,6-difluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>62b</b>)</h4><div class="NLM_p">Bromine (2.194 mL, 42.81 mmol) was added to a solution of ethyl 1<i>H</i>-pyrazole-3-carboxylate (<b>61</b>) (1.5 g, 10.70 mmol) in AcOH (40 mL) at room temperature. The mixture was stirred at room temperature for 6 h. The mixture was quenched with saturated Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> aq and dried up under reduced pressure. The residue was extracted with EtOAc, washed with saturated NaHCO<sub>3</sub> aq and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to give ethyl 4-bromo-1<i>H</i>-pyrazole-3-carboxylate (2.07 g, 88%) as a white solid. This product was subjected to the next reaction without further purification.</div><div class="NLM_p last">A solution of ethyl 4-bromo-1<i>H</i>-pyrazole-3-carboxylate (2.07 g, 9.45 mmol) in THF (dry) (10 mL) was added to a solution of sodium hydride, 60% in oil (0.454 g, 11.34 mmol) in THF (dry) (10 mL) at 0 °C, and then the resulting mixture was stirred for 5 min. To this bright-yellow mixture was added 2-(bromomethyl)-1,3-difluorobenzene (2.152 g, 10.40 mmol) dropwise with stirring, and then the resulting mixture was further stirred in an ice bath and was gradually warmed up to rt overnight. The mixture was quenched with saturated NH<sub>4</sub>Cl aq and extracted with EtOAc. The organic layer was separated, washed with saturated NaHCO<sub>3</sub> aq and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 15–36% EtOAc in hexane) to give <b>62b</b> (3.16 g, 97%) as a colorless solid. LC-MS (APCI) <i>m</i>/<i>z</i> 345.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.41 (3H, t, <i>J</i> = 7.2 Hz), 4.42 (2H, q, <i>J</i> = 7.2 Hz), 5.46 (2H, s), 6.98 (2H, dd, <i>J</i> = 8.5, 7.4 Hz), 7.32–7.43 (1H, m), 7.45 (1H, s).</div></div><div id="sec5_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Ethyl 4-(2-Ethoxyvinyl)-1-(2-fluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>63a</b>)</h4><div class="NLM_p last">Compound <b>63a</b> was obtained (84%) as a white solid in a manner similar to that described for the synthesis of <b>31</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 319.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.23–1.45 (6H, m), 3.82–4.00 (2H, m), 4.37–4.46 (2H, m), 5.37–5.47 (2H, m), 6.00–6.06 (0.5H, m), 6.14–6.22 (0.5H, m), 6.80–6.88 (0.5H, m), 7.04–7.24 (2.5H, m), 7.27–7.38 (1H, m), 7.93 (0.5H, s).</div></div><div id="sec5_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> (<i>Z</i>)-Ethyl 1-(2,6-Difluorobenzyl)-4-(2-ethoxyvinyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>63b</b>)</h4><div class="NLM_p last">A mixture of <b>62b</b> (2.80 g, 8.11 mmol), (<i>Z</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (commercially available, 2.89 g, 14.60 mmol), PdCl<sub>2</sub>(dppf) (0.594 g, 0.81 mmol), and cesium carbonate (5.82 g, 17.85 mmol) in DME (80 mL)/water (10.00 mL) was stirred at 90 °C overnight under Ar atmosphere. The mixture was diluted with EtOAc, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 5–50% EtOAc in hexane) to give <b>63b</b> (2.370 g, 87%) as a pale- yellow oil. LC-MS (APCI) <i>m</i>/<i>z</i> 337.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.26–1.34 (3H, m), 1.40 (3H, t, <i>J</i> = 7.2 Hz), 3.94 (2H, q, <i>J</i> = 6.9 Hz), 4.40 (2H, q, <i>J</i> = 7.2 Hz), 5.39–5.53 (2H, m), 6.01 (1H, d, <i>J</i> = 6.8 Hz), 6.18 (1H, d, <i>J</i> = 6.8 Hz), 6.88–7.04 (2H, m), 7.28–7.44 (1H, m), 7.90 (1H, s).</div></div><div id="sec5_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Ethyl 1-(2-Fluorobenzyl)-4-(2-oxoethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>64a</b>)</h4><div class="NLM_p last">To a solution of <b>63a</b> (10.33 g, 32.45 mmol) in THF (100 mL) was added 6 M HCl aq in water (50 mL, 300.00 mmol) at room temperature. The mixture was stirred at room temperature under air for 2.5 h. The mixture was quenched with saturated NaHCO<sub>3</sub> aq at room temperature and extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 29–50% EtOAc in hexane) to give <b>64a</b> (6.41 g, 68%) as a pale-yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.39 (3H, t, <i>J</i> = 7.2 Hz), 3.83–3.88 (2H, m), 4.40 (2H, q, <i>J</i> = 7.2 Hz), 5.39–5.45 (2H, m), 7.06–7.17 (2H, m), 7.20–7.29 (1H, m), 7.30–7.39 (1H, m), 7.44 (1H, s), 9.72 (1H, t, <i>J</i> = 1.3 Hz).</div></div><div id="sec5_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Ethyl 1-(2,6-Difluorobenzyl)-4-(2-oxoethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>64b</b>)</h4><div class="NLM_p last">Compound <b>64b</b> was obtained in a manner similar to that described for the synthesis of <b>32</b>. This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 309.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> (<i>S</i>)-2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(4-cyano-2-nitrophenoxy)propanoic Acid (<b>66a</b>)</h4><div class="NLM_p last">To the suspendion of 60% NaH in oil (5.12 g, 127.92 mmol) in DMF (95 mL) was added a solution of (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-3-hydroxypropanoic acid (commercially available, 12.5 g, 60.91 mmol) in DMF (30 mL) at 0 °C over 20 min. After being stirred at 0 °C for 1 h, 4-fluoro-3-nitrobenzonitrile (<b>65a</b>) (commercially available, 10.12 g, 60.91 mmol) was added to the reaction mixture at 0 °C. The mixture was stirred at 0 °C for 2 h at room temperature for over the weekend. The mixture was neutralized with iced water and 1 M HCl aq (50 mL) at 0 °C and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 30–70% EtOAc in hexane) to give <b>66a</b> (4.74 g, 22%) as a yellow gum. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.35 (9H, s), 4.30–4.60 (3H, m), 7.12–7.21 (1H, m), 7.56 (1H, d, <i>J</i> = 8.7 Hz), 8.10–8.17 (1H, m), 8.46–8.51 (1H, m), 12.99 (1H, brs).</div></div><div id="sec5_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> (<i>S</i>)-3-(5-Bromo-2-nitrophenoxy)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic Acid (<b>66b</b>)</h4><div class="NLM_p last">Compound <b>66b</b> was obtained (74%) as a yellow gum in a manner similar to that described for the synthesis of <b>66a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 402.9 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.38 (9H, s), 4.33–4.50 (3H, m), 7.09 (1H, d, <i>J</i> = 7.9 Hz), 7.35 (1H, dd, <i>J</i> = 8.7, 1.9 Hz), 7.65 (1H, d, <i>J</i> = 1.9 Hz), 7.84 (1H, d, <i>J</i> = 8.7 Hz), 13.00 (1H, brs).</div></div><div id="sec5_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> (<i>S</i>)-3-(5-((Benzyloxy)carbonyl)-2-nitrophenoxy)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic Acid (<b>66c</b>)</h4><div class="NLM_p last">Compound <b>66c</b> was obtained in a manner similar to that described for the synthesis of <b>66a</b>. This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 459.1 [M – H]<sup>+</sup>.</div></div><div id="sec5_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> (<i>S</i>)-3-(2-Amino-4-cyanophenoxy)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic Acid (<b>67a</b>)</h4><div class="NLM_p last">A mixture of <b>66a</b> (4.2 g, 11.96 mmol), ammonium chloride (2.56 g, 47.82 mmol), and iron (6.68 g, 119.55 mmol) in EtOH (80 mL, 11.96 mmol)/water (16 mL) was stirred at 80 °C for 3 h. To this mixture was added THF (100 mL), and insoluble material was removed by filtration through Celite. The filtrate was evaporated. To the filtrate was added EtOAc, and insoluble material was removed by filtration. The filtrate was evaporated to give the crude of <b>67a</b> as a brown solid. This product was subjected to the next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 320.1 [M – H]<sup>+</sup>.</div></div><div id="sec5_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> (<i>S</i>)-3-(2-Amino-5-bromophenoxy)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic Acid (<b>67b</b>)</h4><div class="NLM_p last">A mixture of <b>66b</b> (10.00 g, 24.68 mmol), ammonium chloride (5.28 g, 98.72 mmol), and iron (13.78 g, 246.79 mmol) in EtOH (100 mL, 24.68 mmol)/water (30 mL) was stirred at 80 °C for 1.5 h. Insoluble material was removed by filtration through Celite. The filtrate was evaporated. To the filtrate was added EtOAc, and insoluble material was removed by filtration. The filtrate was evaporated to give <b>67b</b> (8.40 g, 91%) as a brown solid. LC-MS (APCI) <i>m</i>/<i>z</i> 372.9 [M – H]<sup>+</sup>.</div></div><div id="sec5_1_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> (<i>S</i>)-3-(2-Amino-5-((benzyloxy)carbonyl)phenoxy)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic Acid (<b>67c</b>)</h4><div class="NLM_p last">To a solution of <b>66c</b> (14.98 g, 32.53 mmol) in AcOH (150 mL, 2622.73 mmol) was added zinc powder (21.27 g, 325.35 mmol) at 0 °C, and the mixture was stirred for 10 min at 0 °C and 2 h at rt. The reaction mixture was filtered, and the filtrate was concentrated in vauo. AcOH was removed by azeotropic distillation with toluene to give the crude of <b>67c</b>. The product was used for next reaction without further purification. LC-MS (APCI) <i>m</i>/<i>z</i> 431.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> (<i>S</i>)-<i>tert</i>-Butyl (7-Cyano-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>68a</b>)</h4><div class="NLM_p">To a mixture of <b>67a</b> (3843 mg, 11.96 mmol) and HATU (4775 mg, 12.56 mmol) in DMSO (40 mL) was added TEA (1.834 mL, 13.16 mmol) at room temperature. The mixture was stirred at room temperature overnight. The mixture was diluted with water and the formed precipitate was collected by filtration and washed with water.</div><div class="NLM_p">(1) The filtrate was extracted with EtOAc. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure.</div><div class="NLM_p">(2) The collected solid was suspended in EtOAc, washed with water, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure.</div><div class="NLM_p last">(1) and (2) were combined and purified by column chromatography (silica gel, eluted with 10–30% EtOAc in hexane) to give <b>68a</b> (835 mg, 23% in 2 steps) as a yellow solid. LC-MS (APCI) <i>m</i>/<i>z</i> 302.1 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.37 (9H, s), 4.29–4.45 (3H, m), 7.13–7.20 (1H, m), 7.25 (1H, d, <i>J</i> = 8.3 Hz), 7.48–7.60 (2H, m), 10.21 (1H, s).</div></div><div id="sec5_1_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> (<i>S</i>)-<i>tert</i>-Butyl (8-Bromo-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>68b</b>)</h4><div class="NLM_p last">TEA (3.43 mL, 24.63 mmol) was added to a mixture of <b>67b</b> (8.4 g, 22.39 mmol) and HATU (8.94 g, 23.51 mmol) in DMSO (40 mL) at room temperature. The mixture was stirred at room temperature for 1 h. The mixture was diluted with water, and precipitate was collected by filtration and washed with water 3 times. The solid was disolved in EtOAc and dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 10–30% EtOAc in hexane) to give <b>68b</b> (2.290 g, 29%) as a light-brown solid. LC-MS (APCI) <i>m</i>/<i>z</i> 355.0 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.36 (9H, s), 4.24–4.40 (3H, m), 7.03 (1H, d, <i>J</i> = 9.1 Hz), 7.11 (1H, d, <i>J</i> = 7.6 Hz), 7.29–7.36 (2H, m), 10.00 (1H, s).</div></div><div id="sec5_1_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> (<i>S</i>)-Benzyl 3-((<i>tert</i>-Butoxycarbonyl)amino)-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-8-carboxylate (<b>68c</b>)</h4><div class="NLM_p last">Compound <b>68c</b> was obtained (3% in 3 steps from <b>65c</b>) as a white solid in a manner similar to that described for the synthesis of <b>68b</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 411.1 [M – H]<sup>+</sup>.</div></div><div id="sec5_1_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> (<i>S</i>)-<i>tert</i>-Butyl (7-Cyano-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>69a</b>)</h4><div class="NLM_p last">A mixture of <b>68a</b> (830 mg, 2.74 mmol), cesium carbonate (1070 mg, 3.28 mmol), and MeI (0.180 mL, 2.87 mmol) in DMF (10 mL) was stirred at room temperature for 3 h. The mixture was quenched with water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The solid was crystallized from EtOA to give <b>69a</b> (389 mg, 45%) as a colorless solid. LC-MS (APCI) <i>m</i>/<i>z</i> 316.1 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.35 (9H, s), 3.30 (3H, s), 4.34–4.42 (3H, m), 7.21–7.25 (1H, m), 7.37 (1H, d, <i>J</i> = 8.3 Hz), 7.75 (1H, dd, <i>J</i> = 8.3, 1.9 Hz), 8.08 (1H, d, <i>J</i> = 1.9 Hz).</div></div><div id="sec5_1_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> (<i>S</i>)-<i>tert</i>-Butyl (8-Bromo-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>69b</b>)</h4><div class="NLM_p last">A mixture of <b>68b</b> (2.2 g, 6.16 mmol), cesium carbonate (3.01 g, 9.24 mmol), and MeI (0.501 mL, 8.01 mmol) in DMF (15 mL) was stirred at room temperature for 3 h. The mixture was diluted with water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 10–30% EtOAc in hexane) to give <b>69b</b> (1.520 g, 67%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 271.2 [M + H – (Boc)]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.34 (9H, s), 3.26 (3H, s), 4.26–4.43 (3H, m), 7.18 (1H, d, <i>J</i> = 7.6 Hz), 7.40–7.53 (3H, m).</div></div><div id="sec5_1_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> (<i>S</i>)-Benzyl 3-((<i>tert</i>-Butoxycarbonyl)amino)-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-8-carboxylate (<b>69c</b>)</h4><div class="NLM_p last">Compound <b>69c</b> was obtained (75%) as a white solid in a manner similar to that described for the synthesis of <b>69a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 327.0 [M + H – (Boc)]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.35–1.44 (9H, m), 3.42 (3H, s), 4.16–4.26 (1H, m), 4.53–4.69 (2H, m), 5.30–5.52 (3H, m), 7.22–7.27 (1H, m), 7.32–7.48 (5H, m), 7.82–7.86 (1H, m), 7.93 (1H, dd, <i>J</i> = 8.3, 1.9 Hz).</div></div><div id="sec5_1_100" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> (<i>S</i>)-3-((<i>tert</i>-Butoxycarbonyl)amino)-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-8-carboxylic Acid (<b>69d</b>)</h4><div class="NLM_p last">A mixture of <b>69c</b> (100 mg, 0.23 mmol) and 10% palladium on carbon (24.95 mg, 0.02 mmol) in THF (3 mL, 36.95 mmol) was hydrogenated under balloon pressure at room temperature for 2 h. The catalyst was removed by filtration, and the filtrate was concentrated under reduced pressure to give <b>69d</b> (78 mg, 99%) as white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 335.1 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.29–1.43 (9H, m), 3.31 (3H, s), 4.23–4.50 (3H, m), 7.19–7.31 (1H, m), 7.58 (1H, d, <i>J</i> = 8.3 Hz), 7.68 (1H, d, <i>J</i> = 1.9 Hz), 7.85 (1H, dd, <i>J</i> = 8.3, 1.9 Hz), 13.16 (1H, brs).</div></div><div id="sec5_1_101" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> (<i>S</i>)-<i>tert</i>-Butyl (5-Methyl-8-(methylcarbamoyl)-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>69e</b>)</h4><div class="NLM_p last">Compound <b>69e</b> was obtained (100%) as a white solid in a manner similar to that described for the synthesis of <b>68a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 250.3 [M + H – (Boc)]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.40 (9H, s), 3.02 (3H, d, <i>J</i> = 4.9 Hz), 3.41 (3H, s), 4.19 (1H, dd, <i>J</i> = 11.0, 9.1 Hz), 4.51–4.71 (2H, m), 5.48 (1H, d, <i>J</i> = 6.4 Hz), 6.18 (1H, brs), 7.21–7.27 (1H, m), 7.52 (1H, d, <i>J</i> = 1.9 Hz), 7.63 (1H, dd, <i>J</i> = 8.3, 1.9 Hz).</div></div><div id="sec5_1_102" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> (<i>S</i>)-<i>tert</i>-Butyl (8-(Dimethylcarbamoyl)-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)carbamate (<b>69f</b>)</h4><div class="NLM_p last">Compound <b>69f</b> was obtained (85%) as a white solid in a manner similar to that described for the synthesis of <b>68a</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 264.3 [M + H – (Boc)]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.40 (9H, s), 2.91–3.25 (6H, m), 3.41 (3H, s), 4.19 (1H, dd, <i>J</i> = 11.0, 9.4 Hz), 4.53–4.74 (2H, m), 5.48 (1H, d, <i>J</i> = 6.8 Hz), 7.19–7.33 (3H, m).</div></div><div id="sec5_1_103" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> (<i>S</i>)-3-Amino-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-7-carbonitrile (<b>70a</b>)</h4><div class="NLM_p last">A mixture of <b>69a</b> (510 mg, 1.61 mmol) and 4 M HCl in EtOAc (6.03 mL, 24.11 mmol) in EtOAc (10 mL)/THF (10 mL) was stirred at room temperature for 3 h. After evaporation of the solvent under reduced pressure, the residue was purified by column chromatography (NH silica gel, eluted with 20–100% EtOAc) to give starting material containing mixture (400 mg). The mixture (400 mg) was dissolved in MeOH (10 mL)/THF(12 mL) and to the mixture was added 4 M HCl in EtOAc (8 mL) and the mixture was stirred at rt for 2.5 h. To this mixture was added 4 M HCl in EtOAc (6 mL), and the mixture was stirred at rt for 2.5 h. After evaporation, the residue was was purified by column chromatography (NH silica gel, eluted with 50–100% EtOAc in hexane, 0–10% MeOH in EtOAc) to give <b>70a</b> (210 mg, 60%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 218.4 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.83 (2H, s), 3.31 (3H, s), 3.60–3.70 (1H, m), 4.06–4.17 (1H, m), 4.28–4.37 (1H, m), 7.33 (1H, d, <i>J</i> = 8.3 Hz), 7.70 (1H, dd, <i>J</i> = 8.3, 2.3 Hz), 8.01 (1H, d, <i>J</i> = 1.9 Hz).</div></div><div id="sec5_1_104" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> (<i>S</i>)-3-Amino-8-bromo-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one Hydrochloride (<b>70b</b>)</h4><div class="NLM_p last">A mixture of <b>69b</b> (1.2 g, 3.23 mmol) and 4 M HCl in EtOAc (8.08 mL, 32.33 mmol) in EtOAc (10 mL) was stirred at room temperature for 10 h. To this mixture was added 4 M HCl in EtOAc (4 mL), and the mixture was stirred at room temperature for 10 h. The precipitate was collected by filtration and washed with EtOAc to give <b>70b</b> (0.970 g, 98%) as a white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 271.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.33 (3H, s), 4.29–4.68 (3H, m), 7.45–7.59 (3H, m), 8.54 (3H, brs).</div></div><div id="sec5_1_105" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> (<i>S</i>)-3-Amino-<i>N</i>,5-dimethyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-8-carboxamide (<b>70c</b>)</h4><div class="NLM_p last">A solution of <b>69e</b> (162 mg, 0.46 mmol) in 4 M hydrogen chloride in EtOAc (2 mL, 8.00 mmol) was stirred for 2 h at rt. MeOH was added to the reaction mixture and filtered through a short pad of NH silica gel eluted with MeOH. The residue was concentrated to give <b>70c</b> (85 mg, 74%) as white solid. LC-MS (APCI) <i>m</i>/<i>z</i> 250.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.75 (2H, brs), 2.78 (3H, d, <i>J</i> = 4.5 Hz), 3.31 (3H, s), 3.60 (1H, dd, <i>J</i> = 11.5, 7.4 Hz), 4.03 (1H, dd, <i>J</i> = 11.5, 10.0 Hz), 4.29 (1H, dd, <i>J</i> = 10.0, 7.4 Hz), 7.48 (1H, d, <i>J</i> = 8.3 Hz), 7.61 (1H, d, <i>J</i> = 1.9 Hz), 7.72 (1H, dd, <i>J</i> = 8.3, 1.9 Hz), 8.48 (1H, d, <i>J</i> = 4.2 Hz).</div></div><div id="sec5_1_106" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> (<i>S</i>)-3-Amino-<i>N</i>,<i>N</i>,5-trimethyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepine-8-carboxamide (<b>70d</b>)</h4><div class="NLM_p last">Compound <b>70d</b> was obtained (71%) as a white solid in a manner similar to that described for the synthesis of <b>70c</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 264.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.73 (2H, s), 2.96 (6H, brs), 3.30 (3H, s), 3.63 (1H, dd, <i>J</i> = 11.5, 7.4 Hz), 3.94–4.09 (1H, m), 4.29 (1H, dd, <i>J</i> = 10.0, 7.4 Hz), 7.18 (1H, d, <i>J</i> = 1.9 Hz), 7.28 (1H, dd, <i>J</i> = 7.9, 1.9 Hz), 7.45 (1H, d, <i>J</i> = 7.9 Hz).</div></div><div id="sec5_1_107" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> (<i>S</i>)-Ethyl 1-Benzyl-5-chloro-4-(2-((7-cyano-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>71a</b>)</h4><div class="NLM_p last">Compound <b>71a</b> was obtained (81%) as a colorless oil in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 508.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.38 (3H, t, <i>J</i> = 7.2 Hz), 2.41–2.54 (1H, m), 2.68–2.91 (3H, m), 3.39 (3H, s), 3.57 (1H, dd, <i>J</i> = 11.7, 7.2 Hz), 4.05–4.23 (2H, m), 4.31–4.44 (3H, m), 5.39 (2H, s), 7.15–7.36 (6H, m), 7.45–7.54 (2H, m).</div></div><div id="sec5_1_108" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> (<i>S</i>)-Ethyl 1-Benzyl-4-(2-((8-bromo-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-5-chloro-1<i>H</i>-pyrazole-3-carboxylate (<b>71b</b>)</h4><div class="NLM_p last">Compound <b>71b</b> was obtained (100%) as a white solid in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 561.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.37 (3H, t, <i>J</i> = 7.2 Hz), 2.37–2.52 (1H, m), 2.71–2.92 (3H, m), 3.35 (3H, s), 3.53 (1H, dd, <i>J</i> = 11.5, 7.4 Hz), 3.99–4.09 (1H, m), 4.27–4.42 (3H, m), 5.38 (2H, s), 7.06 (1H, d, <i>J</i> = 8.3 Hz), 7.14–7.22 (2H, m), 7.24–7.37 (5H, m).</div></div><div id="sec5_1_109" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> (<i>S</i>)-Ethyl 1-Benzyl-5-chloro-4-(2-((5-methyl-8-(methylcarbamoyl)-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>71c</b>)</h4><div class="NLM_p last">Compound <b>71c</b> was obtained (48%) as a white solid in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 540.3 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.32–1.44 (3H, m), 2.38–2.56 (1H, m), 2.66–2.92 (3H, m), 2.98–3.15 (3H, m), 3.39 (3H, s), 3.45–3.65 (1H, m), 4.07–4.17 (1H, m), 4.27–4.51 (3H, m), 5.38 (2H, s), 6.11 (1H, brs), 7.09–7.38 (6H, m), 7.51 (1H, d, <i>J</i> = 1.9 Hz), 7.62 (1H, dd, <i>J</i> = 8.3, 1.9 Hz).</div></div><div id="sec5_1_110" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> (<i>S</i>)-Ethyl 1-Benzyl-5-chloro-4-(2-((8-(dimethylcarbamoyl)-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>71d</b>)</h4><div class="NLM_p last">Compound <b>71d</b> was obtained (49%) as a white solid in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 554.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.38 (3H, t, <i>J</i> = 7.2 Hz), 2.34–2.52 (1H, m), 2.64–2.90 (3H, m), 2.97–3.22 (6H, m), 3.39 (3H, s), 3.54 (1H, dd, <i>J</i> = 11.7, 7.6 Hz), 3.98–4.17 (1H, m), 4.28–4.46 (3H, m), 5.36–5.43 (2H, m), 7.10–7.41 (8H, m).</div></div><div id="sec5_1_111" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> (<i>S</i>)-Ethyl 4-(2-((8-Bromo-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1-(2-fluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>71e</b>)</h4><div class="NLM_p last">Compound <b>71e</b> was obtained (77%) as a white solid in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 545.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.37 (3H, t, <i>J</i> = 7.2 Hz), 2.44–2.57 (1H, m), 2.75–2.92 (3H, m), 3.36 (3H, s), 3.52 (1H, dd, <i>J</i> = 11.7, 7.2 Hz), 4.03–4.09 (1H, m), 4.31–4.40 (3H, m), 5.38 (2H, s), 7.04–7.20 (4H, m), 7.28–7.37 (4H, m). NH proton was not detected.</div></div><div id="sec5_1_112" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> (<i>S</i>)-Ethyl 4-(2-((8-Bromo-5-methyl-4-oxo-2,3,4,5-tetrahydrobenzo[<i>b</i>][1,4]oxazepin-3-yl)amino)ethyl)-1-(2,6-difluorobenzyl)-1<i>H</i>-pyrazole-3-carboxylate (<b>71f</b>)</h4><div class="NLM_p last">Compound <b>71f</b> was obtained (58% in 2 steps from <b>64b</b>) as a colorless oil in a manner similar to that described for the synthesis of <b>33</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 563.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.36 (3H, t, <i>J</i> = 7.0 Hz), 2.40–2.54 (1H, m), 2.71–2.92 (3H, m), 3.35 (3H, s), 3.51 (1H, dd, <i>J</i> = 11.7, 7.6 Hz), 4.00–4.10 (1H, m), 4.27–4.43 (3H, m), 5.41 (2H, s), 6.90–7.01 (2H, m), 7.07 (1H, d, <i>J</i> = 8.3 Hz), 7.27–7.40 (4H, m).</div></div><div id="sec5_1_113" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> (<i>S</i>)-3-(2-Benzyl-3-chloro-7-oxo-4,5-dihydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6(7<i>H</i>)-yl)-8-bromo-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one (<b>72a</b>)</h4><div class="NLM_p last">Compound <b>72a</b> was obtained (83%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 515.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.67 (1H, dt, <i>J</i> = 15.9, 4.7 Hz), 3.05 (1H, ddd, <i>J</i> = 15.5, 10.4, 5.1 Hz), 3.35 (3H, s), 3.54 (1H, ddd, <i>J</i> = 12.0, 10.5, 4.3 Hz), 4.23 (1H, dt, <i>J</i> = 12.1, 5.3 Hz), 4.41 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.64 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.39 (2H, s), 5.90 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.11 (1H, d, <i>J</i> = 8.3 Hz), 7.23–7.42 (7H, m).</div></div><div id="sec5_1_114" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> (<i>S</i>)-8-Bromo-3-(2-(2-fluorobenzyl)-7-oxo-4,5-dihydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6(7<i>H</i>)-yl)-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one (<b>72b</b>)</h4><div class="NLM_p last">Compound <b>72b</b> was obtained (89%) as a white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 499.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.72 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 3.10 (1H, ddd, <i>J</i> = 15.1, 10.6, 4.5 Hz), 3.35 (3H, s), 3.53 (1H, ddd, <i>J</i> = 11.8, 10.7, 4.3 Hz), 4.16–4.24 (1H, m), 4.41 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.63 (1H, dd, <i>J</i> = 11.7, 10.2 Hz), 5.40 (2H, s), 5.93 (1H, dd, <i>J</i> = 11.7, 8.3 Hz), 7.04–7.14 (3H, m), 7.20–7.26 (2H, m), 7.27–7.41 (3H, m).</div></div><div id="sec5_1_115" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> (<i>S</i>)-8-Bromo-3-(2-(2,6-difluorobenzyl)-7-oxo-4,5-dihydro-2<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-6(7<i>H</i>)-yl)-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one (<b>72c</b>)</h4><div class="NLM_p last">Compound <b>72c</b> was obtained (59%) as an off-white solid in a manner similar to that described for the synthesis of <b>34</b>. LC-MS (APCI) <i>m</i>/<i>z</i> 517.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.69 (1H, dt, <i>J</i> = 15.5, 4.7 Hz), 2.99–3.15 (1H, m), 3.34 (3H, s), 3.43–3.58 (1H, m), 4.06–4.25 (1H, m), 4.39 (1H, dd, <i>J</i> = 10.0, 8.1 Hz), 4.61 (1H, dd, <i>J</i> = 11.9, 10.0 Hz), 5.45 (2H, s), 5.92 (1H, dd, <i>J</i> = 11.7, 7.9 Hz), 6.89–6.99 (2H, m), 7.10 (1H, d, <i>J</i> = 8.3 Hz), 7.22 (1H, s), 7.27–7.42 (3H, m).</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i137" class="anchor-spacer"></div><h3 class="article-section__title" id="_i137"> TR-FRET Binding Assay</h3><div class="NLM_p last">TR-FRET binding assays were conducted using human GST-His-RIPK1(1–375)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and a fluorescent labeled probe <b>9</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">Supporting Information</a>). The human RIPK1 cDNA (NM_003804) coding 1–375 aa was cloned to have an N-terminal GST-His tag. Human GST-His-RipK1(1–375) has two protease recognition sites and results in GST-(PreScission)-His-(TEV)-human RIPK1 (1–375). GST-His-human RIPK1 (1–375) baculovirus was generated using the BaculoDirect baculovirus expression system (Invitrogen) according to manufacturer’s specifications.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i138" class="anchor-spacer"></div><h3 class="article-section__title" id="_i138"> Data Analysis</h3><div class="NLM_p last">The obtained data at 10 min and 6 h reactions were globally fitted to the following equation described by Motulsky and Mahan<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> to estimate association rates (<i>k</i><sub>3</sub>), dissociation rates (<i>k</i><sub>4</sub>), and dissociation constants (<i>K</i><sub>i</sub>):<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_m001.gif" alt="" id="_i139" /></img></span><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_m002.gif" alt="" id="_i140" /></img></span><span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_m003.gif" alt="" id="_i141" /></img></span>where <i>t</i> is the reaction time, <i>B</i><sub>max</sub> is the total amount of enzyme, <i>Y</i> is the specific binding of the probe, [L] is the concentration of the probe, [I] is the concentration of test compound, <i>k</i><sub>1</sub> and <i>k</i><sub>2</sub> are the association rate and dissociation rate of the probe, respectively, <i>k</i><sub>3</sub> and <i>k</i><sub>4</sub> are the association rate and dissociation rate of the test compound. The dissociation half-life (<i>t</i><sub>1/2</sub>) was determined as <i>t</i><sub>1/2</sub> = ln(2)/<i>k</i><sub>4</sub>, and the kinetic equilibrium dissociation constant of unlabeled compound (<i>K</i><sub>i</sub>) was determined as <i>K</i><sub>i</sub> = <i>k</i><sub>4</sub>/<i>k</i><sub>3</sub>.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i142" class="anchor-spacer"></div><h3 class="article-section__title" id="_i142"> Docking Study</h3><div class="NLM_p last">Molecular docking study was performed using Glide 7.2 (Schrodinger Inc., USA) for better understanding of the interaction of <b>14</b> with RIP1 kinase. Crystal structure of RIP1 kinase (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TX5">5TX5</a>) was obtained from the Protein Data Bank and prepared by a multistep process through Protein Preparation Wizard in Maestro 2016-3 by using OPLS3 force field. The grid for molecular docking was generated with the bound cocrystallized ligand. The ligand was submitted to the LigPrep module to generate a range of ionization states and docked into the binding site of the RIP1 kinase structure using Glide.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i143" class="anchor-spacer"></div><h3 class="article-section__title" id="_i143"> Pharmacokinetic Analysis in Mouse Cassette Dosing</h3><div class="NLM_p last">Compound <b>22</b> was administered intravenously (0.1 mg/kg) or orally (1 mg/kg, suspended in 0.5% methylcellulose aqueous solution) to nonfasted mice by cassette dosing. After administration, blood samples (10 μL) were collected and then centrifuged to obtain plasma fraction. The plasma samples were deproteinized with CH<sub>3</sub>CN containing an internal standard. After centrifugation, the compound concentrations in the supernatant were measured by LC/MS/MS.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i144" class="anchor-spacer"></div><h3 class="article-section__title" id="_i144"> Brain and Plasma Concentration in Mice</h3><div class="NLM_p last">Compound <b>22</b> was administered orally to nonfasted mice (<i>n</i> = 3) at 10 mg/kg. After sacrifice, blood and brain samples were collected at 1 h after oral administration. The blood samples were centrifuged to obtain the plasma fraction. The brain samples were homogenized in saline to obtain the brain homogenate. Plasma and brain protein binding of compound <b>22</b> was determined using the equilibrium dialysis method.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The plasma and brain homogenate samples were deproteinized with CH<sub>3</sub>CN containing an internal standard. After centrifugation, the compound concentrations in the supernatant were measured by LC/MS/MS.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i145" class="anchor-spacer"></div><h3 class="article-section__title" id="_i145"> Biological in Vitro Cell Assay</h3><div id="sec5_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> Reagents</h4><div class="NLM_p last">Z-VAD-FMK as a 20 mM stock solution in neat DMSO purchased from Promega (catalogue no. G7232). Smac mimetic (AT-406) was purchased from AdooQ BioScience (catalogue no. A11163-10). Recombinant human, mouse, and rat TNFα were purchased from Wako and R&D Systems (catalogue nos. 210-TA-100, 410-MT-050, and 510-RT-050, respectively). The following antibodies were used for simple Western (WES system): human phospho-MLKL/Ser358 (Abcam, catalogue no. ab187091). The following antibodies were used for ELISA: mouse MLKL (Millipore, MABC604), mouse phosphor-MLKL/S345 (Abcam, (catalogue no. ab196436), and rabbit IgG HRP (Jackson ImmunoResearch, 711-035-152). TMB was purchased from KPL ((catalogue no. 52-00-02).</div></div><div id="sec5_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> Protocol</h4><div class="NLM_p last">The efficacies of RIP1 kinase inhibitors were tested in vitro using human colorectal adenocarcinoma HT-29 cells and mouse L-cells NCTC 929 (L929) cells in a necroptosis assay.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> HT-29 and L929 cells were acquired from ATCC ((catalogue no. HTB-38 and CCL-1, respectively) and banked in liquid nitrogen. For HT-29 necroptosis assay, frozen HT-29 cells were thawed and diluted to 1.5 × 10<sup>5</sup> cells/mL in McCoy’s 5A medium (Invitrogen, (catalogue no. 16600108) supplemented with 10% fetal bovine serum, 100 units/mL penicillin, and 100 μg/mL streptomycin. A 20 μL aliquot of cell suspension (3000 cells) was added to each well of a 384-assay plate and were incubated for 16–20 h at 37 °C with 5% CO<sub>2</sub>. After overnight incubation, each well was added with 2.5 μL of inhibitor-containing medium and then treated with 2.5 μL of necroptosis inducer containing rat TNFα, AT-406, and zVAD-FMK to achieve a final concentration of 20 ng/mL, 1 μM, and 20 μM, respectively. After the incubation at 37 °C with 5% CO<sub>2</sub> for 16–20 h, the LDH quantity in supernatant of culture medium in each well was measured by CytoTox 96 nonradioactive cytotoxicity assay (Promega, (catalogue no. G1780) according to the manufacturer’s instructions. The Absorbance of test plate was measured by 2103 EnVision multilabel plate reader. Percentage inhibition was calculated from the signal intensity of CytoTox assay by using the following formula: % inhibition = 100 – (A – X) × 100/(A – B); A, no necroptosis inducer; B, necroptosis inducer; X, necroptosis inducer plus test inhibitor. IC<sub>50</sub> values were estimated using a four-parameter logistic curve using XLfit software (IDBS, London, UK). All of the data are shown as mean ± standard error of the mean. For mouse L929 necroptosis assay, 3 × 10<sup>4</sup> of L929 cells in 120 μL of RPMI1640 medium supplemented with 10%FBS, 100 units/mL penicillin, and 100 μg/mL streptomycin were added to each well of a 96-well assay plates and cultured overnight. The next day, the cells were treated with 15 μL of compound in the growth medium for 30 min and then stimulated with 15 μL of necroptosis stimulation mixture containing mouse TNFα and Z-VAD-FMK (final concentration was 20 ng/mL and 20 μM, respectively). After incubation for 4 h at 37 °C with 5% CO<sub>2</sub>, supernatant LDH was measured in the same way as HT-29. The IC<sub>50</sub> values were calculated from sigmoidal-dose response curves utilizing Prism 5.0 (Graphpad, San Diego, CA).</div></div><div id="sec5_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> Human pMLKL WES Assay</h4><div class="NLM_p last">For the detection of phospho-MLKL, 3 × 10<sup>5</sup> cells were seeded in 12-well dishes and cultured overnight. The next day, cells were treated with compounds for 30 min and then stimulated with TNFα/AT-406/Z-VAD-FMK mixture as descrived above. After 6 h incubation, cells were washed in ice cold PBS, lysed in 1× cell lysis buffer (Cell Signaling Technology (catalogue no. 9803)) containing 1 mM PMSF. After harvesting cells with cell scrapers, cell lysates were snap-frozen in liquid nitrogen and stored at −80 °C until following WES. For WES experiments, the cell lysates were allowed to thaw and centrifuged at 20000<i>g</i> for 15 min at 4 °C to remove insoluble cellular debris, and resulting cell lysates were assayed for protein content. Protein content was normalized to 1 mg/mL using the same lysis buffer and analyzed in duplicate with the anti-pMLKL/S358 antibodies using a WES Simple Western system (Protein Simple) according to the manufacturer’s instructions.</div></div><div id="sec5_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> Mouse pMLKL ELISA Assay</h4><div class="NLM_p last">Phospho-MLKL in L929 cells were detected by general sandwich ELISA. L929 cell lysates stimulated with necroptotosis inducer were prepared in the same manner as the case of HT29. The protein content of each cell lysates were adjusted to 200–300 μg/mL to be normalized. Then 100 μL of normalized samples were added onto 96-well ELISA plates (Nunc, catalogue no. 442404) which had been coated with 100 μL of antimouse MLKL Ab (1 μg/mL in PBS) for O/N at 4 °C and incubated for 1 h at RT. Next, plates were incubated with 100 μL of antiphospho-MLKL Ab (1 μg/mL in PBS supplemented with 1% BSA and 0.05% Tween20) for 1 h at RT. Then, plates were reacted with 100 μL of antirabbit IgG HRP followed by detecting with 100 μL of TMB. After stopping reaction by adding 100 μL of 0.5 M H<sub>2</sub>SO<sub>4</sub>, plates were read on a spectrophotometer, Viento (DS Pharma Biomedical), at A450.</div></div><div id="sec5_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> Mice</h4><div class="NLM_p last">C57BL/6J (female, 10-week-old, Charles River Japan) were used for EAE model. The mice were bred on white chip. All procedures were performed in accordance with the standards for humane care, and treatment of research animal was approved by the Takeda Institutional Animal Ethics Committee (approval no. 10111).</div></div><div id="sec5_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> Induction of Experimental Autoimmune Encephalomyelitis (EAE)</h4><div class="NLM_p last">EAE induction was performed as previously described.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Briefly, C57BL/6 mice were immunized subcutaneously in back with an emulsion containing of 200 μg of MOG35-55 peptide and 500 μg of killed <i>Mycobacterium tuberculosis</i> H37Ra suspended in Freund’s incomplete adjuvant on day 0, followed by intraperitoneal injection with 200 ng of pertussis toxin on day 0 and day 2. Clinical signs of EAE in individual mice were assessed using the following score: 0, no clinical signs; 0.5, partially limp tail; 1, paralyzed tail; 2, partially paralysis of hind limb; 3, complete paralysis of both hind limbs: 4, partial paralysis of forelimbs; 5, complete paralysis of forelimbs, death, or humane end point. The clinical score was daily observed under blind fashion except holiday after the onset.</div></div><div id="sec5_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> Prophylactic Efficacy of RIP1 Kinase Inibitor <b>22</b> in Murine EAE Model</h4><div class="NLM_p last">RIP1 kinase inhibitor <b>22</b> was suspended in 0.5% methyl cellulose (0.5% MC) and administered twice a day to the mice by oral gavage from day 0 to day 26 after MOG immunization. Vehicle-treated control mice were administered with 0.5% MC solution. Clinical symptoms of EAE were daily scored as mentioned above. The cumulative score was calculated by adding the individual score obtained from day 0 to day 26.</div></div><div id="sec5_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i153" class="anchor-spacer"></div><h4 class="article-section__title" id="_i153"> Statistical Analysis</h4><div class="NLM_p last">Data are presented as mean ± SEM of 9–12 mice/group. A statistical comparison of data was analyzed by Dunns, and <i>p</i> values lower than 0.05 were considered significant. Statistical analyses were performed using GraphPad Prism 6.01 software (GraphPad Software Inc., La Jolla, CA, USA).</div></div><div id="sec5_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> Transcellular Transport Study Using Transporter-Expression System</h4><div class="NLM_p last">Human MDR1-expressing LLC-PK1 cells were cultured with minor modification as reported previously.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The transcellular transport study was performed as reported previously.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In brief, the cells were grown for 7 days in HTS Transwell 96 well permeable support (pore size 0.4 μm, 0.143 cm<sup>2</sup> surface area) with polyethylene terephthalate membrane (Corning Life Sciences, Lowell, MA, USA) at a density of 1.125 × 10<sup>5</sup> cells/well. The cells were preincubated with M199 at 37 °C for 30 min. Subsequently, transcellular transport was initiated by the addition of M199 either to apical compartments or to the basolateral compartments containing 1 μM test compounds and terminated by the removal of each assay plate after 1 h. The aliquots in the opposite compartments were measured in a LC-MS/MS analysis. The apparent permeability (<i>P</i><sub>app</sub>) of test compounds in the receiver wells was determined, and the ER for MDR1 membrane permeability test was calculated using the following equation:<span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_m004.gif" alt="" id="_i155" /></img></span>where <i>P</i><sub>app, AtoB</sub> is the apical-to-basal passive permeability-surface area product and <i>P</i><sub>app, BtoA</sub> is the basal-to-apical passive permeability-surface area product.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i156"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01647" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44517" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44517" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01647" class="ext-link">10.1021/acs.jmedchem.7b01647</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis of fluorescent probe <b>9</b>, the detailed kinetic parameters of compounds <b>16a</b>–<b>aj</b>; metabolic stabilities of compounds <b>16a</b> and <b>16d</b>; PK graph of <b>22</b>; experimental methods for cloning, expression, and purification of RIP1 kinase, as well as crystallization, data collection, and structure solution of RIP1 kinase; results of Reaction Biology Corporation kinase panel and Eurofins Panlabs panel (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv">jm7b01647_si_001.csv (6.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_002.pdf">jm7b01647_si_002.pdf (368.26 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of the crystal structures of RIP1 kinase in complex with compounds <b>10</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C3E">6C3E</a>) and <b>22</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C4D">6C4D</a>) have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01647" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takatoshi Yogo</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1561747e74617a667d7c3b6c7a727a5561747e7071743b767a78"><span class="__cf_email__" data-cfemail="dfabbeb4beabb0acb7b6f1a6b0b8b09fabbeb4babbbef1bcb0b2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masato Yoshikawa</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7175-1173" title="Orcid link">http://orcid.org/0000-0002-7175-1173</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Morihisa Saitoh</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span>Present Address:
                        For Morihisa Saitoh: Boehringer Ingelheim Japan, Inc., Osaka, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Taisuke Katoh</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomohiro Seki</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone V. Bigi</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuji Shimizu</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tsuyoshi Ishii</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takuro Okai</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masako Kuno</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harumi Hattori</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Etsuro Watanabe</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span>Present Address:
                        For Etsuro Watanabe: UCB Japan Co., Ltd., Tokyo, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kumar S. Saikatendu</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hua Zou</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceuticals, 10410 Science Center Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masanori Nakakariya</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takayuki Tatamiya</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoshihisa Nakada</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span>Present Address:
                        For Yoshihisa Nakada: Ichimaru Pharcos Co., Ltd., Gifu, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d94e10462-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i162">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44407" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44407" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Ikuo Miyahisa for fruitful discussions and valuable suggestions to develop TR-FRET assay for kinetic analysis, Dr. Jeong-ho Oak for fruitful discussion about biology, Dr. Shigekazu Sasaki for kind support to manage CRO, Dr. Kimio Tohyama for data checking of PK profiles in plasma and brain, Dr. Masato Yoshida for fruitful discussions and valuable suggestions about synthetic strategy and research plans, and Dr. Takashi Ichikawa for valuable suggestions and substantial supports to progress the project.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">ALS</td><td class="NLM_def"><p class="first last">amyotrophic lateral sclerosis</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the plasma concentration–time curve</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">total clearance</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum plasma concentration</p></td></tr><tr><td class="NLM_term">Cb</td><td class="NLM_def"><p class="first last">total brain concentration</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>b,u</sub></td><td class="NLM_def"><p class="first last">unbound brain concentration</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>p</sub></td><td class="NLM_def"><p class="first last">total plasma concentration</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>p,u</sub></td><td class="NLM_def"><p class="first last">unbound plasma concentration</p></td></tr><tr><td class="NLM_term">DP1</td><td class="NLM_def"><p class="first last">deep pocket 1</p></td></tr><tr><td class="NLM_term">DP2</td><td class="NLM_def"><p class="first last">deep pocket 2</p></td></tr><tr><td class="NLM_term">EAE</td><td class="NLM_def"><p class="first last">experimental autoimmune encephalomyelitis</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">efflux ratio</p></td></tr><tr><td class="NLM_term"><i>F</i></td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>p</sub></td><td class="NLM_def"><p class="first last">brain-to-plasma ratio</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>p,uu</sub></td><td class="NLM_def"><p class="first last">unbound brain-to-plasma ratio</p></td></tr><tr><td class="NLM_term">HBAs</td><td class="NLM_def"><p class="first last">hydrogen-bonding acceptors</p></td></tr><tr><td class="NLM_term">HBDs</td><td class="NLM_def"><p class="first last">hydrogen-bonding donors</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">MRT</td><td class="NLM_def"><p class="first last">mean residence time</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">multiple sclerosis</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">Parkinson’s disease</p></td></tr><tr><td class="NLM_term">P-gp</td><td class="NLM_def"><p class="first last">P-glycoprotein</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">pMLKL</td><td class="NLM_def"><p class="first last">phosphorylation of MLKL</p></td></tr><tr><td class="NLM_term">RIP1</td><td class="NLM_def"><p class="first last">receptor interacting protein 1</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug design</p></td></tr><tr><td class="NLM_term">SKRs</td><td class="NLM_def"><p class="first last">structure–kinetic relationships</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i164">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26517" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26517" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beere, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span> <span> </span><span class="NLM_article-title">Necroptosis in development, inflammation and disease</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrm.2016.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27999438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFahtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=127-136&author=R.+Weinlichauthor=A.+Oberstauthor=H.+M.+Beereauthor=D.+R.+Green&title=Necroptosis+in+development%2C+inflammation+and+disease&doi=10.1038%2Fnrm.2016.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Necroptosis in development, inflammation and disease</span></div><div class="casAuthors">Weinlich, Ricardo; Oberst, Andrew; Beere, Helen M.; Green, Douglas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the early 2000s, receptor-interacting serine/threonine protein kinase 1 (RIPK1), a mol. already recognized as an important regulator of cell survival, inflammation and disease, was attributed an addnl. function: the regulation of a novel cell death pathway that came to be known as necroptosis.  Subsequently, the related kinase RIPK3 and its substrate mixed-lineage kinase domain-like protein (MLKL) were also implicated in the necroptotic pathway, and links between this pathway and apoptosis were established.  In this Timeline article, we outline the discoveries that have helped to identify the roles of RIPK1, RIPK3, MLKL and other regulators of necroptosis, and how they interact to det. cell fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry23k0aTKuWbVg90H21EOLACvtfcHk0liCiWIjaGxEEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFahtrrJ&md5=7a4b7af9a1da473d45ea833a54558d53</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.149%26sid%3Dliteratum%253Aachs%26aulast%3DWeinlich%26aufirst%3DR.%26aulast%3DOberst%26aufirst%3DA.%26aulast%3DBeere%26aufirst%3DH.%2BM.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26atitle%3DNecroptosis%2520in%2520development%252C%2520inflammation%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D127%26epage%3D136%26doi%3D10.1038%2Fnrm.2016.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Complex pathologic roles of RIPK1 and RIPK3: moving beyond necroptosis</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.tips.2016.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=28126382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt12ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=202-225&author=K.+W.+Wegnerauthor=D.+Salehauthor=A.+Degterev&title=Complex+pathologic+roles+of+RIPK1+and+RIPK3%3A+moving+beyond+necroptosis&doi=10.1016%2Fj.tips.2016.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Complex Pathologic Roles of RIPK1 and RIPK3: Moving Beyond Necroptosis</span></div><div class="casAuthors">Wegner, Kelby W.; Saleh, Danish; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-225</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A process of regulated necrosis, termed necroptosis, has been recognized as a major contributor to cell death and inflammation occurring under a wide range of pathol. settings.  The core event in necroptosis is the formation of the detergent-insol. 'necrosome' complex of homologous Ser/Thr kinases, receptor protein interacting kinase 1 (RIPK1) and receptor interacting protein kinase 3 (RIPK3), which promotes phosphorylation of a key prodeath effector, mixed lineage kinase domain-like (MLKL), by RIPK3.  Core necroptosis mediators are under multiple controls, which have been a subject of intense investigation.  Addnl., non-necroptotic functions of these factors, primarily in controlling apoptosis and inflammatory responses, have also begun to emerge.  This review will provide an overview of the current understanding of the human disease relevance of this pathway, and potential therapeutic strategies, targeting necroptosis mediators in various pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeKXMgW5Q3rLVg90H21EOLACvtfcHk0lh099GkipAzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt12ltA%253D%253D&md5=296e25764f643a458906f2a9aac1df05</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWegner%26aufirst%3DK.%2BW.%26aulast%3DSaleh%26aufirst%3DD.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DComplex%2520pathologic%2520roles%2520of%2520RIPK1%2520and%2520RIPK3%253A%2520moving%2520beyond%2520necroptosis%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26spage%3D202%26epage%3D225%26doi%3D10.1016%2Fj.tips.2016.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span> <span> </span><span class="NLM_article-title">The contribution of necroptosis in neurodegenerative diseases</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2117</span>– <span class="NLM_lpage">2126</span>, <span class="refDoi"> DOI: 10.1007/s11064-017-2249-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1007%2Fs11064-017-2249-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=28382594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1KmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=2117-2126&author=L.+Shaoauthor=S.+Yuauthor=W.+Jiauthor=H.+Liauthor=Y.+Gao&title=The+contribution+of+necroptosis+in+neurodegenerative+diseases&doi=10.1007%2Fs11064-017-2249-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Contribution of Necroptosis in Neurodegenerative Diseases</span></div><div class="casAuthors">Shao, Lifei; Yu, Shuping; Ji, Wei; Li, Haizhen; Gao, Yilu</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2117-2126</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past decades, cell apoptosis has been significantly reputed as an accidental, redundant and alternative manner of cell demise which partakes in homeostasis in the development of extensive diseases.  Nevertheless, necroptosis, another novel manner of cell death through a caspase-independent way, esp. in neurodegenerative diseases remains ambiguous.  The cognition of this form of cell demise is helpful to understand other forms of morphol. resemblance of necrosis.  Addnl., the concrete signal mechanism in the regulation of necroptosis is beneficial to the diagnosis and treatment of neurodegenerative diseases.  Recent studies have demonstrated that necroptotic inhibitor, 24(S)-Hydroxycholesterol and partial specific histone deacetylase inhibitors could alleviate pathogenetic conditions of neurodegenerative diseases via necroptosis pathway.  In this review, we summarize recent researches about mechanisms and modulation of necroptotic signaling pathways and probe into the role of programmed necroptotic cell demise in neurodegenerative diseases such as Parkinson's disease, Multiple sclerosis, Amyotrophic lateral sclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmvV7Ouen9CrVg90H21EOLACvtfcHk0lh099GkipAzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1KmsbY%253D&md5=f20a352421c3060c79f2cd960f8ad87a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs11064-017-2249-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11064-017-2249-1%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DY.%26atitle%3DThe%2520contribution%2520of%2520necroptosis%2520in%2520neurodegenerative%2520diseases%26jtitle%3DNeurochem.%2520Res.%26date%3D2017%26volume%3D42%26spage%3D2117%26epage%3D2126%26doi%3D10.1007%2Fs11064-017-2249-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWitt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakifahmetoglu-Norberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Py, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlink Lima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Activation of necroptosis in multiple sclerosis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1836</span>– <span class="NLM_lpage">1849</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.celrep.2015.02.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=25801023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1836-1849&author=D.+Ofengeimauthor=Y.+Itoauthor=A.+Najafovauthor=Y.+Zhangauthor=B.+Shanauthor=J.+P.+DeWittauthor=J.+Yeauthor=X.+Zhangauthor=A.+Changauthor=H.+Vakifahmetoglu-Norbergauthor=J.+Gengauthor=B.+Pyauthor=W.+Zhouauthor=P.+Aminauthor=J.+Berlink+Limaauthor=C.+Qiauthor=Q.+Yuauthor=B.+Trappauthor=J.+Yuan&title=Activation+of+necroptosis+in+multiple+sclerosis&doi=10.1016%2Fj.celrep.2015.02.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Necroptosis in Multiple Sclerosis</span></div><div class="casAuthors">Ofengeim, Dimitry; Ito, Yasushi; Najafov, Ayaz; Zhang, Yaoyang; Shan, Bing; DeWitt, Judy Park; Ye, Juanying; Zhang, Xumin; Chang, Ansi; Vakifahmetoglu-Norberg, Helin; Geng, Jiefei; Py, Benedicte; Zhou, Wen; Amin, Palak; Lima, Jonilson Berlink; Qi, Chunting; Yu, Qiang; Trapp, Bruce; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1836-1849</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Multiple sclerosis (MS), a common neurodegenerative disease of the CNS, is characterized by the loss of oligodendrocytes and demyelination.  Tumor necrosis factor α (TNF-α), a proinflammatory cytokine implicated in MS, can activate necroptosis, a necrotic cell death pathway regulated by RIPK1 and RIPK3 under caspase-8-deficient conditions.  Here, we demonstrate defective caspase-8 activation, as well as activation of RIPK1, RIPK3, and MLKL, the hallmark mediators of necroptosis, in the cortical lesions of human MS pathol. samples.  Furthermore, we show that MS pathol. samples are characterized by an increased insol. proteome in common with other neurodegenerative diseases such as Alzheimer's disease (AD), Parkinson's disease (PD), and Huntington's disease (HD).  Finally, we show that necroptosis mediates oligodendrocyte degeneration induced by TNF-α and that inhibition of RIPK1 protects against oligodendrocyte cell death in two animal models of MS and in culture.  Our findings demonstrate that necroptosis is involved in MS and suggest that targeting RIPK1 may represent a therapeutic strategy for MS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDaFlNf5zTmrVg90H21EOLACvtfcHk0lh099GkipAzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktrk%253D&md5=561b6327d7b454d5694df7b69c0713f1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.051%26sid%3Dliteratum%253Aachs%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DNajafov%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DB.%26aulast%3DDeWitt%26aufirst%3DJ.%2BP.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DVakifahmetoglu-Norberg%26aufirst%3DH.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DPy%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DAmin%26aufirst%3DP.%26aulast%3DBerlink%2BLima%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DTrapp%26aufirst%3DB.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DActivation%2520of%2520necroptosis%2520in%2520multiple%2520sclerosis%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1836%26epage%3D1849%26doi%3D10.1016%2Fj.celrep.2015.02.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. B.</span></span> <span> </span><span class="NLM_article-title">Necrostatin-1 protection of dopaminergic neurons</span>. <i>Neural Regener. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.4103/1673-5374.160108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4103%2F1673-5374.160108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26330837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BC287otlOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1120-1124&author=J.+R.+Wuauthor=J.+Wangauthor=S.+K.+Zhouauthor=L.+Yangauthor=J.+L.+Yinauthor=J.+P.+Caoauthor=Y.+B.+Cheng&title=Necrostatin-1+protection+of+dopaminergic+neurons&doi=10.4103%2F1673-5374.160108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Necrostatin-1 protection of dopaminergic neurons</span></div><div class="casAuthors">Wu Jing-Ru; Cao Jun-Ping; Wang Jie; Yin Jia-le; Zhou Sheng-Kui; Yang Long; Cheng Yan-Bo</div><div class="citationInfo"><span class="NLM_cas:title">Neural regeneration research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1120-4</span>
        ISSN:<span class="NLM_cas:issn">1673-5374</span>.
    </div><div class="casAbstract">Necroptosis is characterized by programmed necrotic cell death and autophagic activation and might be involved in the death process of dopaminergic neurons in Parkinson's disease.  We hypothesized that necrostatin-1 could block necroptosis and give protection to dopaminergic neurons.  There is likely to be crosstalk between necroptosis and other cell death pathways, such as apoptosis and autophagy.  PC12 cells were pretreated with necroststin-1 1 hour before exposure to 6-hydroxydopamine.  We examined cell viability, mitochondrial membrane potential and expression patterns of apoptotic and necroptotic death signaling proteins.  The results showed that the autophagy/lysosomal pathway is involved in the 6-hydroxydopamine-induced death process of PC12 cells.  Mitochondrial disability induced overactive autophagy, increased cathepsin B expression, and diminished Bcl-2 expression.  Necrostatin-1 within a certain concentration range (5-30 μM) elevated the viability of PC12 cells, stabilized mitochondrial membrane potential, inhibited excessive autophagy, reduced the expression of LC3-II and cathepsin B, and increased Bcl-2 expression.  These findings suggest that necrostatin-1 exerted a protective effect against injury on dopaminergic neurons.  Necrostatin-1 interacts with the apoptosis signaling pathway during this process.  This pathway could be a new neuroprotective and therapeutic target in Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQIKhwA2KI3uSnGiHuhFGv8fW6udTcc2eZwnf06vqQ5ybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287otlOkug%253D%253D&md5=04516fc88aaf3bddc9924a3a4db96001</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4103%2F1673-5374.160108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F1673-5374.160108%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%2BK.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DJ.%2BL.%26aulast%3DCao%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DY.%2BB.%26atitle%3DNecrostatin-1%2520protection%2520of%2520dopaminergic%2520neurons%26jtitle%3DNeural%2520Regener.%2520Res.%26date%3D2015%26volume%3D10%26spage%3D1120%26epage%3D1124%26doi%3D10.4103%2F1673-5374.160108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przedborski, S.</span></span> <span> </span><span class="NLM_article-title">Axonal degeneration: RIPK1 multitasking in ALS</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">R932</span>– <span class="NLM_lpage">R934</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2016.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.cub.2016.08.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27780064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCrsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=R932-R934&author=K.+Politiauthor=S.+Przedborski&title=Axonal+degeneration%3A+RIPK1+multitasking+in+ALS&doi=10.1016%2Fj.cub.2016.08.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Axonal Degeneration: RIPK1 Multitasking in ALS</span></div><div class="casAuthors">Politi, Kristin; Przedborski, Serge</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">R932-R934</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The research of Y. Ito et al. (Science, 2016, 353, 603-8) is reviewed with commentary and refs.  A recent study reports that microglia and oligodendrocytes promote motor neuron degeneration by inducing inflammation and necroptosis in a manner dependent on receptor-interacting kinase 1 (RIPK1).  These findings could be significant for our understanding of the neurobiol. and treatment of neurodegenerative diseases like amyotrophic lateral sclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGresQaHd93y-rVg90H21EOLACvtfcHk0liX5kM4HLmRhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCrsrzO&md5=842be679cf56ccc16b7e3bd6f397c9c6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2016.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2016.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DPrzedborski%26aufirst%3DS.%26atitle%3DAxonal%2520degeneration%253A%2520RIPK1%2520multitasking%2520in%2520ALS%26jtitle%3DCurr.%2520Biol.%26date%3D2016%26volume%3D26%26spage%3DR932%26epage%3DR934%26doi%3D10.1016%2Fj.cub.2016.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saberi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitomi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWitt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookhtiar, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouchida, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasparakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravits, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">RIPK1 mediates axonal degeneration by promoting inflammation and necroptosis in ALS</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1126/science.aaf6803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1126%2Fscience.aaf6803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27493188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=603-608&author=Y.+Itoauthor=D.+Ofengeimauthor=A.+Najafovauthor=S.+Dasauthor=S.+Saberiauthor=Y.+Liauthor=J.+Hitomiauthor=H.+Zhuauthor=H.+Chenauthor=L.+Mayoauthor=J.+Gengauthor=P.+Aminauthor=J.+P.+DeWittauthor=A.+K.+Mookhtiarauthor=M.+Florezauthor=A.+T.+Ouchidaauthor=J.+B.+Fanauthor=M.+Pasparakisauthor=M.+A.+Kelliherauthor=J.+Ravitsauthor=J.+Yuan&title=RIPK1+mediates+axonal+degeneration+by+promoting+inflammation+and+necroptosis+in+ALS&doi=10.1126%2Fscience.aaf6803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RIPK1 mediates axonal degeneration by promoting inflammation and necroptosis in ALS</span></div><div class="casAuthors">Ito, Yasushi; Ofengeim, Dimitry; Najafov, Ayaz; Das, Sudeshna; Saberi, Shahram; Li, Ying; Hitomi, Junichi; Zhu, Hong; Chen, Hongbo; Mayo, Lior; Geng, Jiefei; Amin, Palak; DeWitt, Judy Park; Mookhtiar, Adnan Kasim; Florez, Marcus; Ouchida, Amanda Tomie; Fan, Jian-bing; Pasparakis, Manolis; Kelliher, Michelle A.; Ravits, John; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6299</span>),
    <span class="NLM_cas:pages">603-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mutations in the optineurin (OPTN) gene have been implicated in both familial and sporadic amyotrophic lateral sclerosis (ALS).  However, the role of this protein in the central nervous system (CNS) and how it may contribute to ALS pathol. are unclear.  Here, we found that optineurin actively suppressed receptor-interacting kinase 1 (RIPK1)-dependent signaling by regulating its turnover.  Loss of OPTN led to progressive dysmyelination and axonal degeneration through engagement of necroptotic machinery in the CNS, including RIPK1, RIPK3, and mixed lineage kinase domain-like protein (MLKL).  Furthermore, RIPK1- and RIPK3-mediated axonal pathol. was commonly obsd. in SOD1G93A transgenic mice and pathol. samples from human ALS patients.  Thus, RIPK1 and RIPK3 play a crit. role in mediating progressive axonal degeneration.  Furthermore, inhibiting RIPK1 kinase may provide an axonal protective strategy for the treatment of ALS and other human degenerative diseases characterized by axonal degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo120pi55BWcLVg90H21EOLACvtfcHk0liX5kM4HLmRhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7jO&md5=6145b22849aa523b5dc1200dc8610c1c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf6803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf6803%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DY.%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DNajafov%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DSaberi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHitomi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMayo%26aufirst%3DL.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DAmin%26aufirst%3DP.%26aulast%3DDeWitt%26aufirst%3DJ.%2BP.%26aulast%3DMookhtiar%26aufirst%3DA.%2BK.%26aulast%3DFlorez%26aufirst%3DM.%26aulast%3DOuchida%26aufirst%3DA.%2BT.%26aulast%3DFan%26aufirst%3DJ.%2BB.%26aulast%3DPasparakis%26aufirst%3DM.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26aulast%3DRavits%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DRIPK1%2520mediates%2520axonal%2520degeneration%2520by%2520promoting%2520inflammation%2520and%2520necroptosis%2520in%2520ALS%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D603%26epage%3D608%26doi%3D10.1126%2Fscience.aaf6803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Chemical inhibitor of nonapoptotic cell death with therapeutic potential for ischemic brain injury</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1038/nchembio711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnchembio711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16408008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1OmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=112-119&author=A.+Degterevauthor=Z.+Huangauthor=M.+Boyceauthor=Y.+Liauthor=P.+Jagtapauthor=N.+Mizushimaauthor=G.+D.+Cunyauthor=T.+J.+Mitchisonauthor=M.+A.+Moskowitzauthor=J.+Yuan&title=Chemical+inhibitor+of+nonapoptotic+cell+death+with+therapeutic+potential+for+ischemic+brain+injury&doi=10.1038%2Fnchembio711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical inhibitor of nonapoptotic cell death with therapeutic potential for ischemic brain injury</span></div><div class="casAuthors">Degterev, Alexei; Huang, Zhihong; Boyce, Michael; Li, Yaqiao; Jagtap, Prakash; Mizushima, Noboru; Cuny, Gregory D.; Mitchison, Timothy J.; Moskowitz, Michael A.; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-119</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mechanism of apoptosis has been extensively characterized over the past decade, but little is known about alternative forms of regulated cell death.  Although stimulation of the Fas/TNFR receptor family triggers a canonical 'extrinsic' apoptosis pathway, the authors demonstrated that in the absence of intracellular apoptotic signaling it is capable of activating a common nonapoptotic death pathway, which the authors term necroptosis.  The authors showed that necroptosis is characterized by necrotic cell death morphol. and activation of autophagy.  The authors identified a specific and potent small-mol. inhibitor of necroptosis, necrostatin-1, which blocks a crit. step in necroptosis.  The authors demonstrated that necroptosis contributes to delayed mouse ischemic brain injury in vivo through a mechanism distinct from that of apoptosis and offers a new therapeutic target for stroke with an extended window for neuroprotection.  Our study identifies a previously undescribed basic cell-death pathway with potentially broad relevance to human pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiiBDVRJkdUbVg90H21EOLACvtfcHk0liD70rKwpsg_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1OmsLg%253D&md5=83f8c363ffb73ccc14bf11ec02577336</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio711%26sid%3Dliteratum%253Aachs%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DBoyce%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DJagtap%26aufirst%3DP.%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DChemical%2520inhibitor%2520of%2520nonapoptotic%2520cell%2520death%2520with%2520therapeutic%2520potential%2520for%2520ischemic%2520brain%2520injury%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D112%26epage%3D119%26doi%3D10.1038%2Fnchembio711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitomi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germscheid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ch’en, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korkina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Identification of RIP1 kinase as a specific cellular target of necrostatins</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1038/nchembio.83</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnchembio.83" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=18408713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvVSmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=313-321&author=A.+Degterevauthor=J.+Hitomiauthor=M.+Germscheidauthor=I.+L.+Ch%E2%80%99enauthor=O.+Korkinaauthor=X.+Tengauthor=D.+Abbottauthor=G.+D.+Cunyauthor=C.+Yuanauthor=G.+Wagnerauthor=S.+M.+Hedrickauthor=S.+A.+Gerberauthor=A.+Lugovskoyauthor=J.+Yuan&title=Identification+of+RIP1+kinase+as+a+specific+cellular+target+of+necrostatins&doi=10.1038%2Fnchembio.83"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of RIP1 kinase as a specific cellular target of necrostatins</span></div><div class="casAuthors">Degterev, Alexei; Hitomi, Junichi; Germscheid, Megan; Ch'en, Irene L.; Korkina, Olga; Teng, Xin; Abbott, Derek; Cuny, Gregory D.; Yuan, Chengye; Wagner, Gerhard; Hedrick, Stephen M.; Gerber, Scott A.; Lugovskoy, Alexey; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">313-321</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Necroptosis is a cellular mechanism of necrotic cell death induced by apoptotic stimuli in the form of death domain receptor engagement by their resp. ligands under conditions where apoptotic execution is prevented.  Although it occurs under regulated conditions, necroptotic cell death is characterized by the same morphol. features as unregulated necrotic death.  Here we report that necrostatin-1, a previously identified small-mol. inhibitor of necroptosis, is a selective allosteric inhibitor of the death domain receptor-assocd. adaptor kinase RIP1 in vitro.  We show that RIP1 is the primary cellular target responsible for the antinecroptosis activity of necrostatin-1.  In addn., we show that two other necrostatins, necrostatin-3 and necrostatin-5, also target the RIP1 kinase step in the necroptosis pathway, but through mechanisms distinct from that of necrostatin-1.  Overall, our data establish necrostatins as the first-in-class inhibitors of RIP1 kinase, the key upstream kinase involved in the activation of necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOaKMR-RMwdLVg90H21EOLACvtfcHk0liD70rKwpsg_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvVSmtL0%253D&md5=8ba3859c2b4c752da534441747a79bb5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.83&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.83%26sid%3Dliteratum%253Aachs%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DHitomi%26aufirst%3DJ.%26aulast%3DGermscheid%26aufirst%3DM.%26aulast%3DCh%25E2%2580%2599en%26aufirst%3DI.%2BL.%26aulast%3DKorkina%26aufirst%3DO.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DAbbott%26aufirst%3DD.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DHedrick%26aufirst%3DS.%2BM.%26aulast%3DGerber%26aufirst%3DS.%2BA.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520RIP1%2520kinase%2520as%2520a%2520specific%2520cellular%2520target%2520of%2520necrostatins%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D313%26epage%3D321%26doi%3D10.1038%2Fnchembio.83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duprez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootjans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cauwels, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerinckx, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DuHadaway, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roelandt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hauwermeiren, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Declercq, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callewaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prendergast, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenabeele, P.</span></span> <span> </span><span class="NLM_article-title">Necrostatin-1 analogues: critical issues on the specificity, activity and in vivo use in experimental disease models</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e437</span>, <span class="refDoi"> DOI: 10.1038/cddis.2012.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcddis.2012.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=23190609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=e437&author=N.+Takahashiauthor=L.+Duprezauthor=S.+Grootjansauthor=A.+Cauwelsauthor=W.+Nerinckxauthor=J.+B.+DuHadawayauthor=V.+Goossensauthor=R.+Roelandtauthor=F.+Van+Hauwermeirenauthor=C.+Libertauthor=W.+Declercqauthor=N.+Callewaertauthor=G.+C.+Prendergastauthor=A.+Degterevauthor=J.+Yuanauthor=P.+Vandenabeele&title=Necrostatin-1+analogues%3A+critical+issues+on+the+specificity%2C+activity+and+in+vivo+use+in+experimental+disease+models&doi=10.1038%2Fcddis.2012.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Necrostatin-1 analogues: critical issues on the specificity, activity and in vivo use in experimental disease models</span></div><div class="casAuthors">Takahashi, N.; Duprez, L.; Grootjans, S.; Cauwels, A.; Nerinckx, W.; DuHadaway, J. B.; Goossens, V.; Roelandt, R.; Van Hauwermeiren, F.; Libert, C.; Declercq, W.; Callewaert, N.; Prendergast, G. C.; Degterev, A.; Yuan, J.; Vandenabeele, P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Nov.</span>),
    <span class="NLM_cas:pages">e437, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Necrostatin-1 (Nec-1) is widely used in disease models to examine the contribution of receptor-interacting protein kinase (RIPK) 1 in cell death and inflammation.  We studied three Nec-1 analogs: Nec-1, the active inhibitor of RIPK1, Nec-1 inactive (Nec-1i), its inactive variant, and Nec-1 stable (Nec-1s), its more stable variant.  We report that Nec-1 is identical to methyl-thiohydantoin-tryptophan, an inhibitor of the potent immunomodulatory enzyme indoleamine 2,3-dioxygenase (IDO).  Both Nec-1 and Nec-1i inhibited human IDO, but Nec-1s did not, as predicted by mol. modeling.  Therefore, Nec-1s is a more specific RIPK1 inhibitor lacking the IDO-targeting effect.  Next, although Nec-1i was ∼100 × less effective than Nec-1 in inhibiting human RIPK1 kinase activity in vitro, it was only 10 times less potent than Nec-1 and Nec-1s in a mouse necroptosis assay and became even equipotent at high concns.  Along the same line, in vivo, high doses of Nec-1, Nec-1i and Nec-1s prevented tumor necrosis factor (TNF)-induced mortality equally well, excluding the use of Nec-1i as an inactive control.  Paradoxically, low doses of Nec-1 or Nec-1i, but not Nec-1s, even sensitized mice to TNF-induced mortality.  Importantly, Nec-1s did not exhibit this low dose toxicity, stressing again the preferred use of Nec-1s in vivo.  Our findings have important implications for the interpretation of Nec-1-based data in exptl. disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2yIQ4UD-yb7Vg90H21EOLACvtfcHk0liD70rKwpsg_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFSitg%253D%253D&md5=2d4f57016d88e474cbfe00dee540b8fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2012.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2012.176%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DN.%26aulast%3DDuprez%26aufirst%3DL.%26aulast%3DGrootjans%26aufirst%3DS.%26aulast%3DCauwels%26aufirst%3DA.%26aulast%3DNerinckx%26aufirst%3DW.%26aulast%3DDuHadaway%26aufirst%3DJ.%2BB.%26aulast%3DGoossens%26aufirst%3DV.%26aulast%3DRoelandt%26aufirst%3DR.%26aulast%3DVan%2BHauwermeiren%26aufirst%3DF.%26aulast%3DLibert%26aufirst%3DC.%26aulast%3DDeclercq%26aufirst%3DW.%26aulast%3DCallewaert%26aufirst%3DN.%26aulast%3DPrendergast%26aufirst%3DG.%2BC.%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DVandenabeele%26aufirst%3DP.%26atitle%3DNecrostatin-1%2520analogues%253A%2520critical%2520issues%2520on%2520the%2520specificity%252C%2520activity%2520and%2520in%2520vivo%2520use%2520in%2520experimental%2520disease%2520models%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2012%26volume%3D3%26spage%3De437%26doi%3D10.1038%2Fcddis.2012.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0ljWfPTKT4VOGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahler, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolte, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule RIP1 kinase inhibitors for the treatment of pathologies associated with necroptosis</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1021/ml400382p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400382p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1238-1243&author=P.+A.+Harrisauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=K.+M.+Kahlerauthor=R.+Lehrauthor=J.+D.+Lichauthor=R.+Nagillaauthor=R.+T.+Nolteauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=M.+C.+Schaefferauthor=A.+Smallwoodauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=Discovery+of+small+molecule+RIP1+kinase+inhibitors+for+the+treatment+of+pathologies+associated+with+necroptosis&doi=10.1021%2Fml400382p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule RIP1 Kinase Inhibitors for the Treatment of Pathologies Associated with Necroptosis</span></div><div class="casAuthors">Harris, Philip A.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Finger, Joshua N.; Hoffman, Sandra J.; Kahler, Kirsten M.; Lehr, Ruth; Lich, John D.; Nagilla, Rakesh; Nolte, Robert T.; Ouellette, Michael T.; Pao, Christina S.; Schaeffer, Michelle C.; Smallwood, Angela; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1238-1243</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Potent inhibitors of RIP1 kinase from three distinct series, 1-aminoisoquinolines, pyrrolo-[2,3-b]-pyridines, and furo-[2,3-d]-pyrimidines, all of the type II class recognizing a DLG-out inactive conformation, were identified from screening of our inhouse kinase focused sets.  An exemplar from the furo-[2,3-d]-pyrimidine series showed a dose proportional response in protection from hypothermia in a mouse model of TNFα induced lethal shock.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN6cNcMjIyjrVg90H21EOLACvtfcHk0lga4hUa8yWTbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWrtrnF&md5=818a831f7083f7f17767daf02992ba55</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml400382p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400382p%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKahler%26aufirst%3DK.%2BM.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DNolte%26aufirst%3DR.%2BT.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520RIP1%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520pathologies%2520associated%2520with%2520necroptosis%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1238%26epage%3D1243%26doi%3D10.1021%2Fml400382p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Najjar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suebsuwong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogusa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Structure guided design of potent and selective ponatinib-based hybrid inhibitors for RIPK1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1850</span>– <span class="NLM_lpage">1860</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.celrep.2015.02.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=25801024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1850-1860&author=M.+Najjarauthor=C.+Suebsuwongauthor=S.+S.+Rayauthor=R.+J.+Thapaauthor=J.+L.+Makiauthor=S.+Nogusaauthor=S.+Shahauthor=D.+Salehauthor=P.+J.+Goughauthor=J.+Bertinauthor=J.+Yuanauthor=S.+Balachandranauthor=G.+D.+Cunyauthor=A.+Degterev&title=Structure+guided+design+of+potent+and+selective+ponatinib-based+hybrid+inhibitors+for+RIPK1&doi=10.1016%2Fj.celrep.2015.02.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Guided Design of Potent and Selective Ponatinib-Based Hybrid Inhibitors for RIPK1</span></div><div class="casAuthors">Najjar, Malek; Suebsuwong, Chalada; Ray, Soumya S.; Thapa, Roshan J.; Maki, Jenny L.; Nogusa, Shoko; Shah, Saumil; Saleh, Danish; Gough, Peter J.; Bertin, John; Yuan, Junying; Balachandran, Siddharth; Cuny, Gregory D.; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1850-1860</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RIPK1 and RIPK3, two closely related RIPK family members, have emerged as important regulators of pathol. cell death and inflammation.  In the current work, we report that the Bcr-Abl inhibitor and anti-leukemia agent ponatinib is also a first-in-class dual inhibitor of RIPK1 and RIPK3.  Ponatinib potently inhibited multiple paradigms of RIPK1- and RIPK3-dependent cell death and inflammatory tumor necrosis factor alpha (TNF-α) gene transcription.  We further describe design strategies that utilize the ponatinib scaffold to develop two classes of inhibitors (CS and PN series), each with greatly improved selectivity for RIPK1.  In particular, we detail the development of PN10, a highly potent and selective "hybrid" RIPK1 inhibitor, capturing the best properties of two different allosteric RIPK1 inhibitors, ponatinib and necrostatin-1.  Finally, we show that RIPK1 inhibitors from both classes are powerful blockers of TNF-induced injury in vivo.  Altogether, these findings outline promising candidate mols. and design approaches for targeting RIPK1- and RIPK3-driven inflammatory pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUQJThkEIWBbVg90H21EOLACvtfcHk0lga4hUa8yWTbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktro%253D&md5=c13215958ddcca5a685a4d1e82a96560</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.052%26sid%3Dliteratum%253Aachs%26aulast%3DNajjar%26aufirst%3DM.%26aulast%3DSuebsuwong%26aufirst%3DC.%26aulast%3DRay%26aufirst%3DS.%2BS.%26aulast%3DThapa%26aufirst%3DR.%2BJ.%26aulast%3DMaki%26aufirst%3DJ.%2BL.%26aulast%3DNogusa%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DSaleh%26aufirst%3DD.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DBalachandran%26aufirst%3DS.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DStructure%2520guided%2520design%2520of%2520potent%2520and%2520selective%2520ponatinib-based%2520hybrid%2520inhibitors%2520for%2520RIPK1%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1850%26epage%3D1860%26doi%3D10.1016%2Fj.celrep.2015.02.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broudy, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iuliucci, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-[2-(imidazo[1,2-<i>b</i>]pyridazin-3-yl)ethynyl]-4-methyl-<i>N</i>-{4-[(4-methylpiperazin-1-yl)-methyl]-3-(trifluoromethyl)phenyl}benzamide (AP24534), a potent, orally active pan-inhibitor of breakpoint cluster region-abelson (BCR-ABL) kinase including the T315I gatekeeper mutant</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4701</span>– <span class="NLM_lpage">4719</span>, <span class="refDoi"> DOI: 10.1021/jm100395q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100395q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4701-4719&author=W.+S.+Huangauthor=C.+A.+Metcalfauthor=R.+Sundaramoorthiauthor=Y.+Wangauthor=D.+Zouauthor=R.+M.+Thomasauthor=X.+Zhuauthor=L.+Caiauthor=D.+Wenauthor=S.+Liuauthor=J.+Romeroauthor=J.+Qiauthor=I.+Chenauthor=G.+Bandaauthor=S.+P.+Lentiniauthor=S.+Dasauthor=Q.+Xuauthor=J.+Keatsauthor=F.+Wangauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+T.+Snodgrassauthor=M.+I.+Broudyauthor=K.+Russianauthor=T.+Zhouauthor=L.+Commodoreauthor=N.+I.+Narasimhanauthor=Q.+K.+Mohemmadauthor=J.+Iuliucciauthor=V.+M.+Riveraauthor=D.+C.+Dalgarnoauthor=T.+K.+Sawyerauthor=T.+Clacksonauthor=W.+C.+Shakespeare&title=Discovery+of+3-%5B2-%28imidazo%5B1%2C2-b%5Dpyridazin-3-yl%29ethynyl%5D-4-methyl-N-%7B4-%5B%284-methylpiperazin-1-yl%29-methyl%5D-3-%28trifluoromethyl%29phenyl%7Dbenzamide+%28AP24534%29%2C+a+potent%2C+orally+active+pan-inhibitor+of+breakpoint+cluster+region-abelson+%28BCR-ABL%29+kinase+including+the+T315I+gatekeeper+mutant&doi=10.1021%2Fjm100395q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-[2-(Imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-[4-((4-methylpiperazin-1-yl)methyl)-3-(trifluoromethyl)phenyl]benzamide (AP24534), a Potent, Orally Active Pan-Inhibitor of Breakpoint Cluster Region-Abelson (BCR-ABL) Kinase Including the T315I Gatekeeper Mutant</span></div><div class="casAuthors">Huang, Wei-Sheng; Metcalf, Chester A.; Sundaramoorthi, Raji; Wang, Yihan; Zou, Dong; Thomas, R. Mathew; Zhu, Xiaotian; Cai, Lisi; Wen, David; Liu, Shuangying; Romero, Jan; Qi, Jiwei; Chen, Ingrid; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Zhou, Tianjun; Commodore, Lois; Narasimhan, Narayana I.; Mohemmad, Qurish K.; Iuliucci, John; Rivera, Victor M.; Dalgarno, David C.; Sawyer, Tomi K.; Clackson, Tim; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4701-4719</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of chronic myeloid leukemia (CML) with BCR-ABL kinase inhibitors, the T315I gatekeeper mutant has emerged as resistant to all currently approved agents.  This report describes the structure-guided design of a novel series of potent pan-inhibitors of BCR-ABL, including the T315I mutation.  A key structural feature is the carbon-carbon triple bond linker which skirts the increased bulk of Ile315 side chain.  Extensive SAR studies led to the discovery of development candidate benzamide I (AP24534), which inhibited the kinase activity of both native BCR-ABL and the T315I mutant with low nM IC50s, and potently inhibited proliferation of corresponding Ba/F3-derived cell lines.  Daily oral administration of I significantly prolonged survival of mice injected i.v. with BCR-ABLT315I expressing Ba/F3 cells.  These data, coupled with a favorable ADME profile, support the potential of I to be an effective treatment for CML, including patients refractory to all currently approved therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoci2zVFUazgLVg90H21EOLACvtfcHk0lhVqcLPd9cJFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D&md5=0dcd7b4ff31164b1d9883e897b7058ff</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm100395q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100395q%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSnodgrass%26aufirst%3DJ.%2BT.%26aulast%3DBroudy%26aufirst%3DM.%2BI.%26aulast%3DRussian%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DIuliucci%26aufirst%3DJ.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%25203-%255B2-%2528imidazo%255B1%252C2-b%255Dpyridazin-3-yl%2529ethynyl%255D-4-methyl-N-%257B4-%255B%25284-methylpiperazin-1-yl%2529-methyl%255D-3-%2528trifluoromethyl%2529phenyl%257Dbenzamide%2520%2528AP24534%2529%252C%2520a%2520potent%252C%2520orally%2520active%2520pan-inhibitor%2520of%2520breakpoint%2520cluster%2520region-abelson%2520%2528BCR-ABL%2529%2520kinase%2520including%2520the%2520T315I%2520gatekeeper%2520mutant%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4701%26epage%3D4719%26doi%3D10.1021%2Fjm100395q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded library screening identifies benzo[b][1,4]oxazepin-4-ones as highly potent and monoselective receptor interacting protein 1 kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2163</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01898</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01898" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2163-2178&author=P.+A.+Harrisauthor=B.+W.+Kingauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=J.+N.+Fingerauthor=L.+C.+Gradyauthor=S.+J.+Hoffmanauthor=J.+U.+Jeongauthor=J.+Kangauthor=V.+Kasparcovaauthor=A.+S.+Lakdawalaauthor=R.+Lehrauthor=D.+E.+McNultyauthor=R.+Nagillaauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+R.+Rendinaauthor=M.+C.+Schaefferauthor=J.+D.+Summerfieldauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=A.+Zhangauthor=D.+Zhangauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=DNA-encoded+library+screening+identifies+benzo%5Bb%5D%5B1%2C4%5Doxazepin-4-ones+as+highly+potent+and+monoselective+receptor+interacting+protein+1+kinase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span></div><div class="casAuthors">Harris, Philip A.; King, Bryan W.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Finger, Joshua N.; Grady, LaShadric C.; Hoffman, Sandra J.; Jeong, Jae U.; Kang, James; Kasparcova, Viera; Lakdawala, Ami S.; Lehr, Ruth; McNulty, Dean E.; Nagilla, Rakesh; Ouellette, Michael T.; Pao, Christina S.; Rendina, Alan R.; Schaeffer, Michelle C.; Summerfield, Jennifer D.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Zhang, Aming; Zhang, Daohua; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2163-2178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent discovery of the role of receptor interacting protein 1 (RIP1) kinase in tumor necrosis factor (TNF)-mediated inflammation has led to its emergence as a highly promising target for the treatment of multiple inflammatory diseases.  The authors screened RIP1 against GSK's DNA-encoded small-mol. libraries and identified a novel highly potent benzoxazepinone inhibitor series.  The authors demonstrate that this template possesses complete monokinase selectivity for RIP1 plus unique species selectivity for primate vs. nonprimate RIP1.  The authors elucidate the conformation of RIP1 bound to this benzoxazepinone inhibitor driving its high kinase selectivity and design specific mutations in murine RIP1 to restore potency to levels similar to primate RIP1.  This series differentiates itself from known RIP1 inhibitors in combining high potency and kinase selectivity with good pharmacokinetic profiles in rodents.  The favorable developability profile of this benzoxazepinone template, as exemplified by compd. 14 (GSK'481), makes it an excellent starting point for further optimization into a RIP1 clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XvpziWGWg7Vg90H21EOLACvtfcHk0lhVqcLPd9cJFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D&md5=d6325dcaed97b7c339d1830ca041298c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01898%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSummerfield%26aufirst%3DJ.%2BD.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDNA-encoded%2520library%2520screening%2520identifies%2520benzo%255Bb%255D%255B1%252C4%255Doxazepin-4-ones%2520as%2520highly%2520potent%2520and%2520monoselective%2520receptor%2520interacting%2520protein%25201%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2163%26epage%3D2178%26doi%3D10.1021%2Facs.jmedchem.5b01898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkamper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class receptor interacting protein 1 (RIP1) kinase specific clinical candidate (GSK2982772) for the treatment of inflammatory diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenkamperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+first-in-class+receptor+interacting+protein+1+%28RIP1%29+kinase+specific+clinical+candidate+%28GSK2982772%29+for+the+treatment+of+inflammatory+diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0liBNd-1-QywKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenkamper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520receptor%2520interacting%2520protein%25201%2520%2528RIP1%2529%2520kinase%2520specific%2520clinical%2520candidate%2520%2528GSK2982772%2529%2520for%2520the%2520treatment%2520of%2520inflammatory%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a highly potent, selective, and metabolically stable inhibitor of receptor-interacting protein 1 (RIP1) for the treatment of systemic inflammatory response syndrome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">972</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaitbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=972-986&author=Y.+Renauthor=Y.+Suauthor=L.+Sunauthor=S.+Heauthor=L.+Mengauthor=D.+Liaoauthor=X.+Liuauthor=Y.+Maauthor=C.+Liuauthor=S.+Liauthor=H.+Ruanauthor=X.+Leiauthor=X.+Wangauthor=Z.+Zhang&title=Discovery+of+a+highly+potent%2C+selective%2C+and+metabolically+stable+inhibitor+of+receptor-interacting+protein+1+%28RIP1%29+for+the+treatment+of+systemic+inflammatory+response+syndrome&doi=10.1021%2Facs.jmedchem.6b01196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Potent, Selective, and Metabolically Stable Inhibitor of Receptor-Interacting Protein 1 (RIP1) for the Treatment of Systemic Inflammatory Response Syndrome</span></div><div class="casAuthors">Ren, Yan; Su, Yaning; Sun, Liming; He, Sudan; Meng, Lingjun; Liao, Daohong; Liu, Xiao; Ma, Yongfen; Liu, Chunyan; Li, Sisi; Ruan, Hanying; Lei, Xiaoguang; Wang, Xiaodong; Zhang, Zhiyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">972-986</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Based on its essential role in driving inflammation and disease pathol., cell necrosis has gradually been verified as a promising therapeutic target for treating atherosclerosis, systemic inflammatory response syndrome (SIRS), and ischemia injury, among other diseases.  Most necrosis inhibitors targeting receptor-interacting protein 1 (RIP1) still require further optimization because of weak potency or poor metabolic stability.  The authors conducted a phenotypic screen and identified a micromolar hit with novel amide structure.  Medicinal chem. efforts yielded a highly-potent, selective, and metabolically stable drug candidate - compd. 56 (RIPA-56, N-benzyl-N-hydroxy-2,2-dimethylbutanamide).  Biochem. studies and mol. docking revealed that RIP1 is the direct target of this new series of type III kinase inhibitors.  In the SIRS mice disease model, 56 efficiently reduced tumor necrosis factor alpha (TNFα)-induced mortality and multi-organ damage.  Compared to known RIP1 inhibitors, 56 is potent in both human and murine cells, is much more stable in vivo, and is efficacious in animal model studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqos-N-fJJ4vbVg90H21EOLACvtfcHk0liBNd-1-QywKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaitbfO&md5=6e7045a9646be5f9dc9559d90efe8aed</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01196%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DLiao%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRuan%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520metabolically%2520stable%2520inhibitor%2520of%2520receptor-interacting%2520protein%25201%2520%2528RIP1%2529%2520for%2520the%2520treatment%2520of%2520systemic%2520inflammatory%2520response%2520syndrome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D972%26epage%3D986%26doi%3D10.1021%2Facs.jmedchem.6b01196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of GSK′963: a structurally distinct, potent and selective inhibitor of RIP1 kinase</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">15009</span>, <span class="refDoi"> DOI: 10.1038/cddiscovery.2015.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcddiscovery.2015.9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=15009&author=S.+B.+Bergerauthor=P.+Harrisauthor=R.+Nagillaauthor=V.+Kasparcovaauthor=S.+Hoffmanauthor=B.+Swiftauthor=L.+Dareauthor=M.+Schaefferauthor=C.+Capriottiauthor=M.+Ouelletteauthor=B.+W.+Kingauthor=D.+Wisnoskiauthor=J.+Coxauthor=M.+Reillyauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=Characterization+of+GSK%E2%80%B2963%3A+a+structurally+distinct%2C+potent+and+selective+inhibitor+of+RIP1+kinase&doi=10.1038%2Fcddiscovery.2015.9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fcddiscovery.2015.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddiscovery.2015.9%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DHoffman%26aufirst%3DS.%26aulast%3DSwift%26aufirst%3DB.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DSchaeffer%26aufirst%3DM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DWisnoski%26aufirst%3DD.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DReilly%26aufirst%3DM.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DCharacterization%2520of%2520GSK%25E2%2580%25B2963%253A%2520a%2520structurally%2520distinct%252C%2520potent%2520and%2520selective%2520inhibitor%2520of%2520RIP1%2520kinase%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D15009%26doi%3D10.1038%2Fcddiscovery.2015.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3714</span>– <span class="NLM_lpage">3717</span>, <span class="refDoi"> DOI: 10.1021/jm000942e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties&doi=10.1021%2Fjm000942e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0lg1FrvpTvqBEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3717%26doi%3D10.1021%2Fjm000942e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stromnes, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goverman, J. M.</span></span> <span> </span><span class="NLM_article-title">Passive induction of experimental allergic encephalomyelitis</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1952</span>– <span class="NLM_lpage">1960</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnprot.2006.284" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1952-1960&author=I.+M.+Stromnesauthor=J.+M.+Goverman&title=Passive+induction+of+experimental+allergic+encephalomyelitis&doi=10.1038%2Fnprot.2006.284"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.284%26sid%3Dliteratum%253Aachs%26aulast%3DStromnes%26aufirst%3DI.%2BM.%26aulast%3DGoverman%26aufirst%3DJ.%2BM.%26atitle%3DPassive%2520induction%2520of%2520experimental%2520allergic%2520encephalomyelitis%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1952%26epage%3D1960%26doi%3D10.1038%2Fnprot.2006.284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lg1FrvpTvqBEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R.
A.</span></span> <span> </span><span class="NLM_article-title">The drug-target residence time model: a 10-year retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&author=R.%0AA.+Copeland&title=The+drug-target+residence+time+model%3A+a+10-year+retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0lg1FrvpTvqBEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520drug-target%2520residence%2520time%2520model%253A%2520a%252010-year%2520retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1517/17460441003677725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1517%2F17460441003677725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=22823083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=305-310&author=R.+A.+Copeland&title=The+dynamics+of+drug-target+interactions%3A+drug-target+residence+time+and+its+impact+on+efficacy+and+safety&doi=10.1517%2F17460441003677725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-310</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The extent and duration of pharmacol. action is detd. by the lifetime of drug occupancy on a mol. target.  This lifetime is defined by dynamic processes that control the rates of drug assocn. and dissocn. from the target.  Recently, the term residence time has been coined to describe exptl. measurements that can be related to the lifetime of the binary drug-target complex, and this in turn to durable, pharmacodynamic activity.  The residence time concept and its impact on drug optimization are reviewed here.  Examples are provided that demonstrate how a long residence time can improve drug efficacy in vivo.  Addnl., optimization of drug-target residence time can help to mitigate off-target mediated toxicity, hence, improving drug safety and tolerability.  Recent applications of the residence time concept to both drug discovery and development are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyt3KrIYdRjbVg90H21EOLACvtfcHk0ljxqyDznyZ9hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D&md5=aca925e00abefddf6e92d83b470d96f1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1517%2F17460441003677725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003677725%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520drug-target%2520interactions%253A%2520drug-target%2520residence%2520time%2520and%2520its%2520impact%2520on%2520efficacy%2520and%2520safety%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D305%26epage%3D310%26doi%3D10.1517%2F17460441003677725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Evaluation of enzyme inhibitors in drug discovery. A guide for medicinal chemists and pharmacologists</span>. <i>Methods Biochem. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1002/9781118540398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16350889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BD2MnmtlyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1-469&author=R.+A.+Copeland&title=Evaluation+of+enzyme+inhibitors+in+drug+discovery.+A+guide+for+medicinal+chemists+and+pharmacologists&doi=10.1002%2F9781118540398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of enzyme inhibitors in drug discovery. A guide for medicinal chemists and pharmacologists</span></div><div class="casAuthors">Copeland Robert A</div><div class="citationInfo"><span class="NLM_cas:title">Methods of biochemical analysis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-265</span>
        ISSN:<span class="NLM_cas:issn">0076-6941</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR7_ubfiC0DK2q3xNe7a7QfW6udTcc2eY9N8z2yHP0fLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnmtlyjsA%253D%253D&md5=4bb6759b5155aa6c2b6a462432afda7f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DEvaluation%2520of%2520enzyme%2520inhibitors%2520in%2520drug%2520discovery.%2520A%2520guide%2520for%2520medicinal%2520chemists%2520and%2520pharmacologists%26jtitle%3DMethods%2520Biochem.%2520Anal.%26date%3D2005%26volume%3D46%26spage%3D1%26epage%3D469%26doi%3D10.1002%2F9781118540398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Residence time of receptor-ligand complexes and its effect on biological function</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5481</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/bi8002023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8002023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5481-92&author=P.+J.+Tumminoauthor=R.+A.+Copeland&title=Residence+time+of+receptor-ligand+complexes+and+its+effect+on+biological+function&doi=10.1021%2Fbi8002023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Residence Time of Receptor-Ligand Complexes and Its Effect on Biological Function</span></div><div class="casAuthors">Tummino, Peter J.; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5481-5492</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation and duration of binary receptor-ligand complexes are fundamental to many physiol. processes.  Most often, the effectiveness of interaction between a receptor and its ligand is quantified in terms of closed system, equil. affinity measurements, such as IC50 and Kd.  In the context of in vivo biol., however, the extent and duration of responses to receptor-ligand interactions depend greatly on the time period over which the ligand is in residence on its receptor.  Here we define receptor-ligand complex residence time in quant. terms and describe its significance to biol. function.  Examples of the importance of residence time are presented for natural ligands of different receptor types.  The impact of residence time on the optimization of potential ligands as drugs for human medicine is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHshN2Bt_r9LVg90H21EOLACvtfcHk0ljxqyDznyZ9hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D&md5=57f79f69f12c6ed14e3347638381d943</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi8002023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8002023%26sid%3Dliteratum%253Aachs%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DResidence%2520time%2520of%2520receptor-ligand%2520complexes%2520and%2520its%2520effect%2520on%2520biological%2520function%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5481%26epage%3D92%26doi%3D10.1021%2Fbi8002023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Defining desirable central nervous system drug space through the alignment of molecular properties, in vitro ADME, and safety attributes</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1021/cn100007x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100007x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=420-434&author=T.+T.+Wagerauthor=R.+Y.+Chandrasekaranauthor=X.+Houauthor=M.+D.+Troutmanauthor=P.+R.+Verhoestauthor=A.+Villalobosauthor=Y.+Will&title=Defining+desirable+central+nervous+system+drug+space+through+the+alignment+of+molecular+properties%2C+in+vitro+ADME%2C+and+safety+attributes&doi=10.1021%2Fcn100007x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Defining Desirable Central Nervous System Drug Space through the Alignment of Molecular Properties, in Vitro ADME, and Safety Attributes</span></div><div class="casAuthors">Wager, Travis T.; Chandrasekaran, Ramalakshmi Y.; Hou, Xinjun; Troutman, Matthew D.; Verhoest, Patrick R.; Villalobos, Anabella; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">420-434</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of our effort to increase survival of drug candidates and to move our medicinal chem. design to higher probability space for success in the Neuroscience therapeutic area, we embarked on a detailed study of the property space for a collection of central nervous system (CNS) mols.  We carried out a thorough anal. of properties for 119 marketed CNS drugs and a set of 108 Pfizer CNS candidates.  In particular, we focused on understanding the relationships between physicochem. properties, in vitro ADME (absorption, distribution, metab., and elimination) attributes, primary pharmacol. binding efficiencies, and in vitro safety data for these two sets of compds.  This scholarship provides guidance for the design of CNS mols. in a property space with increased probability of success and may lead to the identification of druglike candidates with favorable safety profiles that can successfully test hypotheses in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEMBo2f34BrVg90H21EOLACvtfcHk0ljJ2B0FF6Camg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVaqtbk%253D&md5=85716be86290fd85e93c6b8d5621683c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcn100007x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100007x%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DChandrasekaran%26aufirst%3DR.%2BY.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DDefining%2520desirable%2520central%2520nervous%2520system%2520drug%2520space%2520through%2520the%2520alignment%2520of%2520molecular%2520properties%252C%2520in%2520vitro%2520ADME%252C%2520and%2520safety%2520attributes%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D420%26epage%3D434%26doi%3D10.1021%2Fcn100007x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootenhuis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayada, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delbressine, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploemen, J. P.</span></span> <span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10554091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&author=J.+Kelderauthor=P.+D.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+Delbressineauthor=J.+P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span></div><div class="casAuthors">Kelder, Jan; Grootenhuis, Peter D. J.; Bayada, Denis M.; Delbressine, Leon P. C.; Ploemen, Jan-Peter</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1514-1519</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose: To study oral absorption and brain penetration as a function of polar mol. surface area.  Methods: Measured brain penetration data of 45 drug mols. were investigated.  The dynamic polar surface areas were calcd. and correlated with the brain penetration data.  Also the static polar surface areas of 776 orally administered CNS drugs that have reached at least Phase II efficacy studies were calcd.  The same was done for a series of 1590 orally administered non-CNS drugs that have reached at least Phase II efficacy studies.  Results: A linear relationship between brain penetration and dynamic polar surface area (A2) was found (n = 45, R = 0.917, F1,43 = 229).  Brain penetration decreases with increasing polar surface area.  A clear difference between the distribution of the polar surface area of the 776 CNS and 1590 non-CNS drugs was found.  It was deduced that orally active drugs that are transported passively by the transcellular route should not exceed a polar surface area of about 120 A2.  They can be tailored to brain penetration by decreasing the polar surface to <60-70 A2.  This conclusion is supported by the inverse linear relationship between exptl. brain penetration data and the dynamic polar surface area of 45 drug mols.  Conclusions: The polar mol. surface area is a dominating determinant for oral absorption and brain penetration of drugs that are transported by the transcellular route.  This property should be considered in the early phase of drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmc3v7TzWM4LVg90H21EOLACvtfcHk0ljJ2B0FF6Camg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D&md5=868fe1e9491dd3b8b92b326d28ee48f9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%26aulast%3DPloemen%26aufirst%3DJ.%2BP.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pajouhesh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G. R.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemical properties of successful central nervous system drugs</span>. <i>NeuroRx</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1602/neurorx.2.4.541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1602%2Fneurorx.2.4.541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16489364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FpvVKqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=541-553&author=H.+Pajouheshauthor=G.+R.+Lenz&title=Medicinal+chemical+properties+of+successful+central+nervous+system+drugs&doi=10.1602%2Fneurorx.2.4.541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemical properties of successful central nervous system drugs</span></div><div class="casAuthors">Pajouhesh Hassan; Lenz George R</div><div class="citationInfo"><span class="NLM_cas:title">NeuroRx : the journal of the American Society for Experimental NeuroTherapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">541-53</span>
        ISSN:<span class="NLM_cas:issn">1545-5343</span>.
    </div><div class="casAbstract">Fundamental physiochemical features of CNS drugs are related to their ability to penetrate the blood-brain barrier affinity and exhibit CNS activity.  Factors relevant to the success of CNS drugs are reviewed.  CNS drugs show values of molecular weight, lipophilicity, and hydrogen bond donor and acceptor that in general have a smaller range than general therapeutics.  Pharmacokinetic properties can be manipulated by the medicinal chemist to a significant extent.  The solubility, permeability, metabolic stability, protein binding, and human ether-ago-go-related gene inhibition of CNS compounds need to be optimized simultaneously with potency, selectivity, and other biological parameters.  The balance between optimizing the physiochemical and pharmacokinetic properties to make the best compromises in properties is critical for designing new drugs likely to penetrate the blood brain barrier and affect relevant biological systems.  This review is intended as a guide to designing CNS therapeutic agents with better drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_ncsy3yuEinl8icjszYl0fW6udTcc2eZQwrGcWk0FVLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FpvVKqsg%253D%253D&md5=0815084e2d107b9243b785d0782dfc3a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1602%2Fneurorx.2.4.541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1602%252Fneurorx.2.4.541%26sid%3Dliteratum%253Aachs%26aulast%3DPajouhesh%26aufirst%3DH.%26aulast%3DLenz%26aufirst%3DG.%2BR.%26atitle%3DMedicinal%2520chemical%2520properties%2520of%2520successful%2520central%2520nervous%2520system%2520drugs%26jtitle%3DNeuroRx%26date%3D2005%26volume%3D2%26spage%3D541%26epage%3D553%26doi%3D10.1602%2Fneurorx.2.4.541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-549&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0ljJ2B0FF6Camg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D549%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirimulla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vegesna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, M.</span></span> <span> </span><span class="NLM_article-title">Halogen interactions in protein-ligand complexes: implications of halogen bonding for rational drug design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1021/ci400257k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400257k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cisr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2781-2791&author=S.+Sirimullaauthor=J.+B.+Baileyauthor=R.+Vegesnaauthor=M.+Narayan&title=Halogen+interactions+in+protein-ligand+complexes%3A+implications+of+halogen+bonding+for+rational+drug+design&doi=10.1021%2Fci400257k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Interactions in Protein-Ligand Complexes: Implications of Halogen Bonding for Rational Drug Design</span></div><div class="casAuthors">Sirimulla, Suman; Bailey, Jake B.; Vegesna, Rahulsimham; Narayan, Mahesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2791</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Halogen bonding interactions between halogenated ligands and proteins were examd. using the crystal structures deposited to date in the PDB.  The data was analyzed as a function of halogen bonding to main chain Lewis bases, viz. oxygen of backbone carbonyl and backbone amide nitrogen.  This anal. also examd. halogen bonding to side-chain Lewis bases (O, N, and S) and to the electron-rich arom. amino acids.  All interactions were restricted to van der Waals radii with resp. atoms.  The data reveals that while fluorine and chlorine have strong tendencies favoring interactions with the backbone Lewis bases at glycine, the trend is not restricted to the achiral amino acid backbone for larger halogens.  Halogen side-chain interactions are not restricted to amino acids contg. O, N, and S as Lewis bases.  Electron-rich arom. amino acids host a high frequency of halogen bonds as does Leu.  A closer examn. of the latter hydrophobic side chain reveals that the "propensity of interactions" of halogen ligands at this oily residue is an outcome of strong classical halogen bonds with Lewis bases in the vicinity.  Finally, an examn. of Θ1 (C-X···O and C-X···N) and Θ2 (X···O-Z and X···N-Z) angles reveals that very few ligands adopt classical halogen bonding angles, suggesting that steric and other factors may influence these angles.  The data is discussed in the context of ligand design for pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wcnZDXmkYrVg90H21EOLACvtfcHk0li4A1GTY49siA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cisr3O&md5=a73378681587e3ac13143452ecb78882</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci400257k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400257k%26sid%3Dliteratum%253Aachs%26aulast%3DSirimulla%26aufirst%3DS.%26aulast%3DBailey%26aufirst%3DJ.%2BB.%26aulast%3DVegesna%26aufirst%3DR.%26aulast%3DNarayan%26aufirst%3DM.%26atitle%3DHalogen%2520interactions%2520in%2520protein-ligand%2520complexes%253A%2520implications%2520of%2520halogen%2520bonding%2520for%2520rational%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2781%26epage%3D2791%26doi%3D10.1021%2Fci400257k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0li4A1GTY49siA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-alpha</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1011</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.cell.2009.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=19524512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1eiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2009&pages=1100-1011&author=S.+Heauthor=L.+Wangauthor=L.+Miaoauthor=T.+Wangauthor=F.+Duauthor=L.+Zhaoauthor=X.+Wang&title=Receptor+interacting+protein+kinase-3+determines+cellular+necrotic+response+to+TNF-alpha&doi=10.1016%2Fj.cell.2009.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-α</span></div><div class="casAuthors">He, Sudan; Wang, Lai; Miao, Lin; Wang, Tao; Du, Fenghe; Zhao, Liping; Wang, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1100-1111</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Smac mimetics induce apoptosis synergistically with TNF-α by triggering the formation of a caspase-8-activating complex contg. receptor interacting protein kinase-1 (RIPK1).  Caspase inhibitors block this form of apoptosis in many types of cells.  However, in several other cell lines, caspase inhibitors switch the apoptotic response to necrosis.  A genome wide siRNA screen revealed another member of the RIP kinase family, RIP3, to be required for necrosis.  The expression of RIP3 in different cell lines correlates with their responsiveness to necrosis induction.  The kinase activity of RIP3 is essential for necrosis execution.  Upon induction of necrosis, RIP3 is recruited to RIPK1 to form a necrosis-inducing complex.  Embryonic fibroblasts from RIP3 knockout mice are resistant to necrosis and RIP3 knockout animals are devoid of inflammation inflicted tissue damage in an acute pancreatitis model.  These data indicate RIP3 as the determinant for cellular necrosis in response to TNF-α family of death-inducing cytokines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymGY6l3d1LbVg90H21EOLACvtfcHk0li4A1GTY49siA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1eiu70%253D&md5=6397b34b896f8fe36c726f6d2ab39933</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMiao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DReceptor%2520interacting%2520protein%2520kinase-3%2520determines%2520cellular%2520necrotic%2520response%2520to%2520TNF-alpha%26jtitle%3DCell%26date%3D2009%26volume%3D137%26spage%3D1100%26epage%3D1011%26doi%3D10.1016%2Fj.cell.2009.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani-Yoshihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span> <span> </span><span class="NLM_article-title">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13000</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12513-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fs41598-017-12513-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=29026104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13000&author=T.+Ishiiauthor=T.+Okaiauthor=M.+Iwatani-Yoshiharaauthor=M.+Mochizukiauthor=S.+Unnoauthor=M.+Kunoauthor=M.+Yoshikawaauthor=S.+Shibataauthor=M.+Nakakariyaauthor=T.+Yogoauthor=T.+Kawamoto&title=CETSA+quantitatively+verifies+in+vivo+target+engagement+of+novel+RIPK1+inhibitors+in+various+biospecimens&doi=10.1038%2Fs41598-017-12513-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span></div><div class="casAuthors">Ishii Tsuyoshi; Iwatani-Yoshihara Misa; Mochizuki Manabu; Kawamoto Tomohiro; Okai Takuro; Unno Satoko; Kuno Masako; Yoshikawa Masato; Yogo Takatoshi; Shibata Sachio; Nakakariya Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13000</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The proof of target engagement (TE) is a key element for evaluating potential investment in drug development.  The cellular thermal shift assay (CETSA) is expected to facilitate direct measurement of intracellular TE at all stages of drug development.  However, there have been no reports of applying this technology to comprehensive animal and clinical studies.  This report demonstrates that CETSA can not only quantitatively evaluate the drug-TE in mouse peripheral blood, but also confirm TE in animal tissues exemplified by using the receptor interacting protein 1 kinase (RIPK1) lead compound we have developed.  Our established semi-automated system allows evaluation of the structure-activity relationship using native RIPK1 in culture cell lines, and also enables estimation of drug occupancy ratio in mouse peripheral blood mononuclear cells.  Moreover, optimized tissue homogenisation enables monitoring of the in vivo drug-TE in spleen and brain.  Our results indicate that CETSA methodology will provide an efficient tool for preclinical and clinical drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVxZ2bIR2iP3O4gDb7JggCfW6udTcc2eaMrueH__wPZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D&md5=ae576418ac1d2ab3788ea241f74f59de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12513-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12513-1%26sid%3Dliteratum%253Aachs%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DOkai%26aufirst%3DT.%26aulast%3DIwatani-Yoshihara%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DUnno%26aufirst%3DS.%26aulast%3DKuno%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DYogo%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26atitle%3DCETSA%2520quantitatively%2520verifies%2520in%2520vivo%2520target%2520engagement%2520of%2520novel%2520RIPK1%2520inhibitors%2520in%2520various%2520biospecimens%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13000%26doi%3D10.1038%2Fs41598-017-12513-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes-Earle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mocarski, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">RIP1 kinase activity is dispensable for normal development but is a key regulator of inflammation in SHARPIN-deficient mice</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">5476</span>– <span class="NLM_lpage">5480</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1400499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4049%2Fjimmunol.1400499" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2014&pages=5476-5480&author=S.+B.+Bergerauthor=V.+Kasparcovaauthor=S.+Hoffmanauthor=B.+Swiftauthor=L.+Dareauthor=M.+Schaefferauthor=C.+Capriottiauthor=M.+Cookauthor=J.+Fingerauthor=A.+Hughes-Earleauthor=P.+A.+Harrisauthor=W.+J.+Kaiserauthor=E.+S.+Mocarskiauthor=J.+Bertinauthor=P.+J.+Gough&title=RIP1+kinase+activity+is+dispensable+for+normal+development+but+is+a+key+regulator+of+inflammation+in+SHARPIN-deficient+mice&doi=10.4049%2Fjimmunol.1400499"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400499%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DHoffman%26aufirst%3DS.%26aulast%3DSwift%26aufirst%3DB.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DSchaeffer%26aufirst%3DM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DM.%26aulast%3DFinger%26aufirst%3DJ.%26aulast%3DHughes-Earle%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKaiser%26aufirst%3DW.%2BJ.%26aulast%3DMocarski%26aufirst%3DE.%2BS.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DRIP1%2520kinase%2520activity%2520is%2520dispensable%2520for%2520normal%2520development%2520but%2520is%2520a%2520key%2520regulator%2520of%2520inflammation%2520in%2520SHARPIN-deficient%2520mice%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D192%26spage%3D5476%26epage%3D5480%26doi%3D10.4049%2Fjimmunol.1400499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polykratis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermance, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roderick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasparakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span> <span> </span><span class="NLM_article-title">RIPK1 Kinase inactive mice are viable and protected from TNF-induced necroptosis in vivo</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">1539</span>– <span class="NLM_lpage">1543</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1400590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4049%2Fjimmunol.1400590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2014&pages=1539-1543&author=A.+Polykratisauthor=N.+Hermanceauthor=M.+Zelicauthor=J.+Roderickauthor=C.+Kimauthor=T.+M.+Vanauthor=T.+H.+Leeauthor=F.+K.+M.+Chanauthor=M.+Pasparakisauthor=M.+A.+Kelliher&title=RIPK1+Kinase+inactive+mice+are+viable+and+protected+from+TNF-induced+necroptosis+in+vivo&doi=10.4049%2Fjimmunol.1400590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400590%26sid%3Dliteratum%253Aachs%26aulast%3DPolykratis%26aufirst%3DA.%26aulast%3DHermance%26aufirst%3DN.%26aulast%3DZelic%26aufirst%3DM.%26aulast%3DRoderick%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DC.%26aulast%3DVan%26aufirst%3DT.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BH.%26aulast%3DChan%26aufirst%3DF.%2BK.%2BM.%26aulast%3DPasparakis%26aufirst%3DM.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26atitle%3DRIPK1%2520Kinase%2520inactive%2520mice%2520are%2520viable%2520and%2520protected%2520from%2520TNF-induced%2520necroptosis%2520in%2520vivo%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D193%26spage%3D1539%26epage%3D1543%26doi%3D10.4049%2Fjimmunol.1400590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarato, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cripps, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1038/cdd.2015.70</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcdd.2015.70" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26024392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSiurjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=76-88&author=D.+A.+Rodriguezauthor=R.+Weinlichauthor=S.+Brownauthor=C.+Guyauthor=P.+Fitzgeraldauthor=C.+P.+Dillonauthor=A.+Oberstauthor=G.+Quaratoauthor=J.+Lowauthor=J.+G.+Crippsauthor=T.+Chenauthor=D.+R.+Green&title=Characterization+of+RIPK3-mediated+phosphorylation+of+the+activation+loop+of+MLKL+during+necroptosis&doi=10.1038%2Fcdd.2015.70"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis</span></div><div class="casAuthors">Rodriguez, D. A.; Weinlich, R.; Brown, S.; Guy, C.; Fitzgerald, P.; Dillon, C. P.; Oberst, A.; Quarato, G.; Low, J.; Cripps, J. G.; Chen, T.; Green, D. R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-88</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mixed lineage kinase domain-like pseudokinase (MLKL) mediates necroptosis by translocating to the plasma membrane and inducing its rupture.  The activation of MLKL occurs in a multimol. complex (the 'necrosome'), which is comprised of MLKL, receptor-interacting serine/threonine kinase (RIPK)-3 (RIPK3) and, in some cases, RIPK1.  Within this complex, RIPK3 phosphorylates the activation loop of MLKL, promoting conformational changes and allowing the formation of MLKL oligomers, which migrate to the plasma membrane.  Previous studies suggested that RIPK3 could phosphorylate the murine MLKL activation loop at Ser345, Ser347 and Thr349.  Moreover, substitution of the Ser345 for an aspartic acid creates a constitutively active MLKL, independent of RIPK3 function.  Here we examine the role of each of these residues and found that the phosphorylation of Ser345 is crit. for RIPK3-mediated necroptosis, Ser347 has a minor accessory role and Thr349 seems to be irrelevant.  We generated a specific monoclonal antibody to detect phospho-Ser345 in murine cells.  Using this antibody, a series of MLKL mutants and a novel RIPK3 inhibitor, we demonstrate that the phosphorylation of Ser345 is not required for the interaction between RIPK3 and MLKL in the necrosome, but is essential for MLKL translocation, accumulation in the plasma membrane, and consequent necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjrS_O0spUN7Vg90H21EOLACvtfcHk0lg2MoRzWjr90A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSiurjF&md5=85f1eb0ff4ddaf746855d456f6ee6f6e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2015.70&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2015.70%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DD.%2BA.%26aulast%3DWeinlich%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DGuy%26aufirst%3DC.%26aulast%3DFitzgerald%26aufirst%3DP.%26aulast%3DDillon%26aufirst%3DC.%2BP.%26aulast%3DOberst%26aufirst%3DA.%26aulast%3DQuarato%26aufirst%3DG.%26aulast%3DLow%26aufirst%3DJ.%26aulast%3DCripps%26aufirst%3DJ.%2BG.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26atitle%3DCharacterization%2520of%2520RIPK3-mediated%2520phosphorylation%2520of%2520the%2520activation%2520loop%2520of%2520MLKL%2520during%2520necroptosis%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2016%26volume%3D23%26spage%3D76%26epage%3D88%26doi%3D10.1038%2Fcdd.2015.70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shisler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felices, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orenstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenardo, M. J.</span></span> <span> </span><span class="NLM_article-title">A role for tumor necrosis factor receptor-2 and receptor-interacting protein in programmed necrosis and antiviral responses</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">51613</span>– <span class="NLM_lpage">51621</span>, <span class="refDoi"> DOI: 10.1074/jbc.M305633200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1074%2Fjbc.M305633200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=51613-51621&author=F.+K.+Chanauthor=J.+Shislerauthor=J.+G.+Bixbyauthor=M.+Felicesauthor=L.+Zhengauthor=M.+Appelauthor=J.+Orensteinauthor=B.+Mossauthor=M.+J.+Lenardo&title=A+role+for+tumor+necrosis+factor+receptor-2+and+receptor-interacting+protein+in+programmed+necrosis+and+antiviral+responses&doi=10.1074%2Fjbc.M305633200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M305633200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M305633200%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DF.%2BK.%26aulast%3DShisler%26aufirst%3DJ.%26aulast%3DBixby%26aufirst%3DJ.%2BG.%26aulast%3DFelices%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DAppel%26aufirst%3DM.%26aulast%3DOrenstein%26aufirst%3DJ.%26aulast%3DMoss%26aufirst%3DB.%26aulast%3DLenardo%26aufirst%3DM.%2BJ.%26atitle%3DA%2520role%2520for%2520tumor%2520necrosis%2520factor%2520receptor-2%2520and%2520receptor-interacting%2520protein%2520in%2520programmed%2520necrosis%2520and%2520antiviral%2520responses%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D51613%26epage%3D51621%26doi%3D10.1074%2Fjbc.M305633200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-rate based optimization of structure-kinetic relationship - surfing the kinetic map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-rate+based+optimization+of+structure-kinetic+relationship+-+surfing+the+kinetic+map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-rate%2520based%2520optimization%2520of%2520structure-kinetic%2520relationship%2520-%2520surfing%2520the%2520kinetic%2520map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motulsky, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, L. C.</span></span> <span> </span><span class="NLM_article-title">The kinetics of competitive radioligand binding predicted by the law of mass action</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=6708928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADyaL2c7mslOmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1984&pages=1-9&author=H.+J.+Motulskyauthor=L.+C.+Mahan&title=The+kinetics+of+competitive+radioligand+binding+predicted+by+the+law+of+mass+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of competitive radioligand binding predicted by the law of mass action</span></div><div class="casAuthors">Motulsky H J; Mahan L C</div><div class="citationInfo"><span class="NLM_cas:title">Molecular pharmacology</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Although equilibrium competitive radioligand binding studies are often used to characterize hormone and neurotransmitter receptors, the kinetics of such experiments have not been extensively explored.  The interactions of the radioligand and competitor with the receptors can be described by two differential equations which can be solved to yield a single equation describing the binding of the radioligand as a function of time.  This equation has several applications: First, it can be used to simulate competitive binding reactions under defined conditions.  Second, fitting experimental data to this equation allows one to determine the association and dissociation rate constants of the competing ligand, parameters that cannot be derived from equilibrium experiments.  Furthermore, this method can be used to determine the KI of the competing drug from data acquired before equilibrium is reached.  Third, mathematical analysis of the binding equation allowed us to answer two specific questions regarding the kinetics of competitive radioligand binding: how long such an incubation takes to equilibrate, and how the IC50 varies over time.  The answers to these questions depended, to a large extent, on the relative values of the dissociation rate constants of the radioligand and competitor, which can be determined as noted above.  When the competitor dissociates from the receptors more rapidly than the radioligand, the IC50 first decreases and then increases, but never has a value less than the KI.  At low radioligand concentrations, equilibrium is reached in the same amount of time required of the radioligand to dissociate completely from the receptors as determined in an "off-rate experiment." At higher concentrations of radioligand this time is halved.  When the competitor dissociates from the receptor more slowly than does the radioligand, then the time required to equilibrate depends only on the dissociation rate constant of the competitor, and the IC50 decreases over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCXDwvoK7MbN1QT_1O5WBYfW6udTcc2eaAqLe2lF4XdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL2c7mslOmtQ%253D%253D&md5=f0c09a16d7f8947fe0d72c256e2e4914</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMotulsky%26aufirst%3DH.%2BJ.%26aulast%3DMahan%26aufirst%3DL.%2BC.%26atitle%3DThe%2520kinetics%2520of%2520competitive%2520radioligand%2520binding%2520predicted%2520by%2520the%2520law%2520of%2520mass%2520action%26jtitle%3DMol.%2520Pharmacol.%26date%3D1984%26volume%3D25%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, Y.</span></span> <span> </span><span class="NLM_article-title">In vivo biliary clearance should be predicted by intrinsic biliary clearance in sandwich-cultured hepatocytes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">602</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.042101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1124%2Fdmd.111.042101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=602-609&author=M.+Nakakariyaauthor=M.+Onoauthor=N.+Amanoauthor=T.+Moriwakiauthor=K.+Maedaauthor=Y.+Sugiyama&title=In+vivo+biliary+clearance+should+be+predicted+by+intrinsic+biliary+clearance+in+sandwich-cultured+hepatocytes&doi=10.1124%2Fdmd.111.042101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.042101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.042101%26sid%3Dliteratum%253Aachs%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DOno%26aufirst%3DM.%26aulast%3DAmano%26aufirst%3DN.%26aulast%3DMoriwaki%26aufirst%3DT.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520biliary%2520clearance%2520should%2520be%2520predicted%2520by%2520intrinsic%2520biliary%2520clearance%2520in%2520sandwich-cultured%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D602%26epage%3D609%26doi%3D10.1124%2Fdmd.111.042101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirabayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, T.</span></span> <span> </span><span class="NLM_article-title">Quantitative investigation of the impact of P-glycoprotein inhibition on drug transport across blood-brain barrier in rats</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1124/dmd.110.035774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1124%2Fdmd.110.035774" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=8-14&author=H.+Sugimotoauthor=H.+Hirabayashiauthor=Y.+Kimuraauthor=A.+Furutaauthor=N.+Amanoauthor=T.+Moriwaki&title=Quantitative+investigation+of+the+impact+of+P-glycoprotein+inhibition+on+drug+transport+across+blood-brain+barrier+in+rats&doi=10.1124%2Fdmd.110.035774"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.035774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.035774%26sid%3Dliteratum%253Aachs%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DHirabayashi%26aufirst%3DH.%26aulast%3DKimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DA.%26aulast%3DAmano%26aufirst%3DN.%26aulast%3DMoriwaki%26aufirst%3DT.%26atitle%3DQuantitative%2520investigation%2520of%2520the%2520impact%2520of%2520P-glycoprotein%2520inhibition%2520on%2520drug%2520transport%2520across%2520blood-brain%2520barrier%2520in%2520rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D8%26epage%3D14%26doi%3D10.1124%2Fdmd.110.035774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebihara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, S.</span></span> <span> </span><span class="NLM_article-title">Establishment and characterization of the transformants stably-expressing MDR1 derived from various Animal species in LLC-PK1</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1007/s11095-006-0285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1007%2Fs11095-006-0285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16779700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=1460-1472&author=T.+Takeuchiauthor=S.+Yoshitomiauthor=T.+Higuchiauthor=K.+Ikemotoauthor=S.-I.+Niwaauthor=T.+Ebiharaauthor=M.+Katohauthor=T.+Yokoiauthor=S.+Asahi&title=Establishment+and+characterization+of+the+transformants+stably-expressing+MDR1+derived+from+various+Animal+species+in+LLC-PK1&doi=10.1007%2Fs11095-006-0285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span></div><div class="casAuthors">Takeuchi, Toshiyuki; Yoshitomi, Sumie; Higuchi, Tomoaki; Ikemoto, Keiko; Niwa, Shin-Ichi; Ebihara, Takuya; Katoh, Miki; Yokoi, Tsuyoshi; Asahi, Satoru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Stable transformants expressing human multidrug resistance 1 (MDR1), monkey MDR1, canine MDR1, rat MDR1a, rat MDR1b, mouse mdr1a, and mouse mdr1b in LLC-PK1 were established to investigate species differences in P-glycoprotein (P-gp, ABCB1) mediated efflux activity.  Methods: The seven cDNAs of MDR1 from five animals were cloned, and their transformants stably expressing the series of MDR1 in LLC-PK1 were established.  Transport studies of clarithromycin, daunorubicin, digoxin, erythromycin, etoposide, paclitaxel, propranolol, quinidine, ritonavir, saquinavir, verapamil, and vinblastine were performed by using these cells, and efflux activity was compared among the species.  Results: Except for propranolol, all compds. showed efflux activity in all transformants, and were judged to be substrates of P-gp.  There were slight interspecies and interisoforms differences in the substrate recognition.  However, the efflux ratio among the series of the MDR1 stably expressing cells showed good correlation as represented between human and monkey MDR1, and poor correlation as represented between human MDR1 and mouse mdr1a, and human and canine MDR1.  Conclusions: Results in the present study indicate that all MDR1 stably expressing cells have efflux activity for various P-gp substrates, and that interspecies differences and similarities of the P-gp substrate efflux activity may exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28IPlZqwtHLVg90H21EOLACvtfcHk0ljuqK1SV-uwxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D&md5=a21044c202d7ff2eb8ab27c4623909de</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-0285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-0285-7%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DIkemoto%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DS.-I.%26aulast%3DEbihara%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DT.%26aulast%3DAsahi%26aufirst%3DS.%26atitle%3DEstablishment%2520and%2520characterization%2520of%2520the%2520transformants%2520stably-expressing%2520MDR1%2520derived%2520from%2520various%2520Animal%2520species%2520in%2520LLC-PK1%26jtitle%3DPharm.%2520Res.%26date%3D2006%26volume%3D23%26spage%3D1460%26epage%3D1472%26doi%3D10.1007%2Fs11095-006-0285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 35 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rohan R. Merchant, Simon B. Lang, Tingting Yu, Shuilian Zhao, Zhiqi Qi, Takao Suzuki, <span class="NLM_string-name hlFld-ContribAuthor">Jianming Bao</span>. </span><span class="cited-content_cbyCitation_article-title">A General One-Pot Protocol for Hindered N-Alkyl Azaheterocycles from Tertiary Carboxylic Acids. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (11)
                                     , 4180-4184. <a href="https://doi.org/10.1021/acs.orglett.0c01254" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c01254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c01254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c01254%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DA%252BGeneral%252BOne-Pot%252BProtocol%252Bfor%252BHindered%252BN-Alkyl%252BAzaheterocycles%252Bfrom%252BTertiary%252BCarboxylic%252BAcids%26aulast%3DMerchant%26aufirst%3DRohan%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08042020%26date%3D08052020%26volume%3D22%26issue%3D11%26spage%3D4180%26epage%3D4184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amy C. Hart, Lynn Abell, Junqing Guo, Michael E. Mertzman, Ramesh Padmanabha, John E. Macor, Charu Chaudhry, Hao Lu, Kevin O’Malley, Patrick J. Shaw, Carolyn Weigelt, Matthew Pokross, Kevin Kish, Kyoung S. Kim, Lyndon Cornelius, Andrew E. Douglas, Deepa Calambur, Ping Zhang, Brian Carpenter, <span class="NLM_string-name hlFld-ContribAuthor">William J. Pitts</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of RIPK3 Type II Inhibitors Using High-Throughput Mechanistic Studies in Hit Triage. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 266-271. <a href="https://doi.org/10.1021/acsmedchemlett.9b00065" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00065%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252BRIPK3%252BType%252BII%252BInhibitors%252BUsing%252BHigh-Throughput%252BMechanistic%252BStudies%252Bin%252BHit%252BTriage%26aulast%3DHart%26aufirst%3DAmy%2BC.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D19022019%26date%3D06052019%26date%3D08052019%26date%3D06052019%26volume%3D11%26issue%3D3%26spage%3D266%26epage%3D271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chunlin Zhuang, <span class="NLM_string-name hlFld-ContribAuthor">Fener Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Inhibitors of Necroptosis: Current Status and Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (4)
                                     , 1490-1510. <a href="https://doi.org/10.1021/acs.jmedchem.9b01317" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01317%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BInhibitors%252Bof%252BNecroptosis%25253A%252BCurrent%252BStatus%252Band%252BPerspectives%26aulast%3DZhuang%26aufirst%3DChunlin%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D11082019%26date%3D31102019%26date%3D17102019%26volume%3D63%26issue%3D4%26spage%3D1490%26epage%3D1510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hao Zhang, Lijuan Xu, Xia Qin, Xiaofei Chen, Hui Cong, Longmiao Hu, Long Chen, Zhenyuan Miao, Wannian Zhang, Zhenyu Cai, <span class="NLM_string-name hlFld-ContribAuthor">Chunlin Zhuang</span>. </span><span class="cited-content_cbyCitation_article-title">N-(7-Cyano-6-(4-fluoro-3-(2-(3-(trifluoromethyl)phenyl)acetamido)phenoxy)benzo[d]thiazol-2-yl)cyclopropanecarboxamide (TAK-632) Analogues as Novel Necroptosis Inhibitors by Targeting Receptor-Interacting Protein Kinase 3 (RIPK3): Synthesis, Structure–Activity Relationships, and in Vivo Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6665-6681. <a href="https://doi.org/10.1021/acs.jmedchem.9b00611" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00611</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00611%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DN-%2525287-Cyano-6-%2525284-fluoro-3-%2525282-%2525283-%252528trifluoromethyl%252529phenyl%252529acetamido%252529phenoxy%252529benzo%25255Bd%25255Dthiazol-2-yl%252529cyclopropanecarboxamide%252B%252528TAK-632%252529%252BAnalogues%252Bas%252BNovel%252BNecroptosis%252BInhibitors%252Bby%252BTargeting%252BReceptor-Interacting%252BProtein%252BKinase%252B3%252B%252528RIPK3%252529%25253A%252BSynthesis%25252C%252BStructure%2525E2%252580%252593Activity%252BRelationships%25252C%252Band%252Bin%252BVivo%252BEfficacy%26aulast%3DZhang%26aufirst%3DHao%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09042019%26date%3D28052019%26date%3D16052019%26volume%3D62%26issue%3D14%26spage%3D6665%26epage%3D6681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Philip A. Harris, Jill M. Marinis, John D. Lich, Scott B. Berger, Anirudh Chirala, Julie A. Cox, Patrick M. Eidam, Joshua N. Finger, Peter J. Gough, Jae U. Jeong, James Kang, Viera Kasparcova, Lara K. Leister, Mukesh K. Mahajan, George Miller, Rakesh Nagilla, Michael T. Ouellette, Michael A. Reilly, Alan R. Rendina, Elizabeth J. Rivera, Helen H. Sun, James H. Thorpe, Rachel D. Totoritis, Wei Wang, Dongling Wu, Daohua Zhang, John Bertin, <span class="NLM_string-name hlFld-ContribAuthor">Robert W. Marquis</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a RIP1 Kinase Inhibitor Clinical Candidate (GSK3145095) for the Treatment of Pancreatic Cancer. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (6)
                                     , 857-862. <a href="https://doi.org/10.1021/acsmedchemlett.9b00108" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00108</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00108%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252Ba%252BRIP1%252BKinase%252BInhibitor%252BClinical%252BCandidate%252B%252528GSK3145095%252529%252Bfor%252Bthe%252BTreatment%252Bof%252BPancreatic%252BCancer%26aulast%3DHarris%26aufirst%3DPhilip%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D09052019%26date%3D09052019%26date%3D09052019%26volume%3D10%26issue%3D6%26spage%3D857%26epage%3D862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Philip A. Harris, Nicolas Faucher, Nicolas George, Patrick M. Eidam, Bryan W. King, Gemma V. White, Niall A. Anderson, Deepak Bandyopadhyay, Allison M. Beal, Veronique Beneton, Scott B. Berger, Nino Campobasso, Sebastien Campos, Carol A. Capriotti, Julie A. Cox, Alain Daugan, Frederic Donche, Marie-Hélène Fouchet, Joshua N. Finger, Brad Geddes, Peter J. Gough, Pascal Grondin, Bonnie L. Hoffman, Sandra J. Hoffman, Susan E. Hutchinson, Jae U. Jeong, Emilie Jigorel, Pauline Lamoureux, Lara K. Leister, John D. Lich, Mukesh K. Mahajan, Jamel Meslamani, Julie E. Mosley, Rakesh Nagilla, Pamela M. Nassau, Sze-Ling Ng, Michael T. Ouellette, Kishore K. Pasikanti, Florent Potvain, Michael A. Reilly, Elizabeth J. Rivera, Stéphane Sautet, Michelle C. Schaeffer, Clark A. Sehon, Helen Sun, James H. Thorpe, Rachel D. Totoritis, Paris Ward, Natalie Wellaway, David D. Wisnoski, James M. Woolven, John Bertin, <span class="NLM_string-name hlFld-ContribAuthor">Robert W. Marquis</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Lead-Optimization of 4,5-Dihydropyrazoles as Mono-Kinase Selective, Orally Bioavailable and Efficacious Inhibitors of Receptor Interacting Protein 1 (RIP1) Kinase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (10)
                                     , 5096-5110. <a href="https://doi.org/10.1021/acs.jmedchem.9b00318" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00318</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00318%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BLead-Optimization%252Bof%252B4%25252C5-Dihydropyrazoles%252Bas%252BMono-Kinase%252BSelective%25252C%252BOrally%252BBioavailable%252Band%252BEfficacious%252BInhibitors%252Bof%252BReceptor%252BInteracting%252BProtein%252B1%252B%252528RIP1%252529%252BKinase%26aulast%3DHarris%26aufirst%3DPhilip%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D18022019%26date%3D02052019%26date%3D23042019%26volume%3D62%26issue%3D10%26spage%3D5096%26epage%3D5110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yueshan Li, Yu Xiong, Guo Zhang, Liting Zhang, Wei Yang, Jiao Yang, Luyi Huang, Zeen Qiao, Zhuang Miao, Guifeng Lin, Qiu Sun, Ting Niu, Lijuan Chen, Dawen Niu, Linli Li, <span class="NLM_string-name hlFld-ContribAuthor">Shengyong Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 5-(2,3-Dihydro-1H-indol-5-yl)-7H-pyrrolo[2,3-d]pyrimidin-4-amine Derivatives as a New Class of Receptor-Interacting Protein Kinase 1 (RIPK1) Inhibitors, Which Showed Potent Activity in a Tumor Metastasis Model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (24)
                                     , 11398-11414. <a href="https://doi.org/10.1021/acs.jmedchem.8b01652" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01652</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01652%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252B5-%2525282%25252C3-Dihydro-1H-indol-5-yl%252529-7H-pyrrolo%25255B2%25252C3-d%25255Dpyrimidin-4-amine%252BDerivatives%252Bas%252Ba%252BNew%252BClass%252Bof%252BReceptor-Interacting%252BProtein%252BKinase%252B1%252B%252528RIPK1%252529%252BInhibitors%25252C%252BWhich%252BShowed%252BPotent%252BActivity%252Bin%252Ba%252BTumor%252BMetastasis%252BModel%26aulast%3DLi%26aufirst%3DYueshan%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D24102018%26date%3D10122018%26date%3D27112018%26volume%3D61%26issue%3D24%26spage%3D11398%26epage%3D11414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunnian  Xia</span>, <span class="hlFld-ContribAuthor ">Zhengguang  Yao</span>, <span class="hlFld-ContribAuthor ">Lijuan  Xu</span>, <span class="hlFld-ContribAuthor ">Wannian  Zhang</span>, <span class="hlFld-ContribAuthor ">Haihu  Chen</span>, <span class="hlFld-ContribAuthor ">Chunlin  Zhuang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based bioisosterism design of thio-benzoxazepinones as novel necroptosis inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113484. <a href="https://doi.org/10.1016/j.ejmech.2021.113484" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113484%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bbioisosterism%252Bdesign%252Bof%252Bthio-benzoxazepinones%252Bas%252Bnovel%252Bnecroptosis%252Binhibitors%26aulast%3DXia%26aufirst%3DChunnian%26date%3D2021%26volume%3D220%26spage%3D113484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Narjes  Shahsavani</span>, <span class="hlFld-ContribAuthor ">Hardeep  Kataria</span>, <span class="hlFld-ContribAuthor ">Soheila  Karimi-Abdolrezaee</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and repair strategies for white matter degeneration in CNS injury and diseases. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Molecular Basis of Disease</span><span> <strong>2021,</strong> <em>1867 </em>
                                    (6)
                                     , 166117. <a href="https://doi.org/10.1016/j.bbadis.2021.166117" title="DOI URL">https://doi.org/10.1016/j.bbadis.2021.166117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbadis.2021.166117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbadis.2021.166117%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Molecular%2520Basis%2520of%2520Disease%26atitle%3DMechanisms%252Band%252Brepair%252Bstrategies%252Bfor%252Bwhite%252Bmatter%252Bdegeneration%252Bin%252BCNS%252Binjury%252Band%252Bdiseases%26aulast%3DShahsavani%26aufirst%3DNarjes%26date%3D2021%26volume%3D1867%26issue%3D6%26spage%3D166117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhennan  Fang</span>, <span class="hlFld-ContribAuthor ">Huiqiang  Wei</span>, <span class="hlFld-ContribAuthor ">Wenfeng  Gou</span>, <span class="hlFld-ContribAuthor ">Leyuan  Chen</span>, <span class="hlFld-ContribAuthor ">Changfen  Bi</span>, <span class="hlFld-ContribAuthor ">Wenbin  Hou</span>, <span class="hlFld-ContribAuthor ">Yiliang  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in small-molecule inhibitors for critical therapeutic targets of necroptosis. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (9)
                                     , 817-837. <a href="https://doi.org/10.4155/fmc-2020-0386" title="DOI URL">https://doi.org/10.4155/fmc-2020-0386</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0386%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252Bsmall-molecule%252Binhibitors%252Bfor%252Bcritical%252Btherapeutic%252Btargets%252Bof%252Bnecroptosis%26aulast%3DFang%26aufirst%3DZhennan%26date%3D2021%26volume%3D13%26issue%3D9%26spage%3D817%26epage%3D837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matija  Zelic</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Pontarelli</span>, <span class="hlFld-ContribAuthor ">Lisa  Woodworth</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhu</span>, <span class="hlFld-ContribAuthor ">Amy  Mahan</span>, <span class="hlFld-ContribAuthor ">Yi  Ren</span>, <span class="hlFld-ContribAuthor ">Michael  LaMorte</span>, <span class="hlFld-ContribAuthor ">Ross  Gruber</span>, <span class="hlFld-ContribAuthor ">Aislinn  Keane</span>, <span class="hlFld-ContribAuthor ">Pequita  Loring</span>, <span class="hlFld-ContribAuthor ">Lilu  Guo</span>, <span class="hlFld-ContribAuthor ">Tai-he  Xia</span>, <span class="hlFld-ContribAuthor ">Boyao  Zhang</span>, <span class="hlFld-ContribAuthor ">Pontus  Orning</span>, <span class="hlFld-ContribAuthor ">Egil  Lien</span>, <span class="hlFld-ContribAuthor ">Alexei  Degterev</span>, <span class="hlFld-ContribAuthor ">Timothy  Hammond</span>, <span class="hlFld-ContribAuthor ">Dimitry  Ofengeim</span>. </span><span class="cited-content_cbyCitation_article-title">RIPK1 activation mediates neuroinflammation and disease progression in multiple sclerosis. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2021,</strong> <em>35 </em>
                                    (6)
                                     , 109112. <a href="https://doi.org/10.1016/j.celrep.2021.109112" title="DOI URL">https://doi.org/10.1016/j.celrep.2021.109112</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2021.109112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2021.109112%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DRIPK1%252Bactivation%252Bmediates%252Bneuroinflammation%252Band%252Bdisease%252Bprogression%252Bin%252Bmultiple%252Bsclerosis%26aulast%3DZelic%26aufirst%3DMatija%26date%3D2021%26volume%3D35%26issue%3D6%26spage%3D109112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Dominguez</span>, <span class="hlFld-ContribAuthor ">Eugene  Varfolomeev</span>, <span class="hlFld-ContribAuthor ">Robert  Brendza</span>, <span class="hlFld-ContribAuthor ">Kim  Stark</span>, <span class="hlFld-ContribAuthor ">Joy  Tea</span>, <span class="hlFld-ContribAuthor ">Jose  Imperio</span>, <span class="hlFld-ContribAuthor ">Hai  Ngu</span>, <span class="hlFld-ContribAuthor ">Timothy  Earr</span>, <span class="hlFld-ContribAuthor ">Oded  Foreman</span>, <span class="hlFld-ContribAuthor ">Joshua D.  Webster</span>, <span class="hlFld-ContribAuthor ">Amy  Easton</span>, <span class="hlFld-ContribAuthor ">Domagoj  Vucic</span>, <span class="hlFld-ContribAuthor ">Baris  Bingol</span>. </span><span class="cited-content_cbyCitation_article-title">Genetic inactivation of RIP1 kinase does not ameliorate disease in a mouse model of ALS. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Differentiation</span><span> <strong>2021,</strong> <em>28 </em>
                                    (3)
                                     , 915-931. <a href="https://doi.org/10.1038/s41418-020-00625-7" title="DOI URL">https://doi.org/10.1038/s41418-020-00625-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41418-020-00625-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41418-020-00625-7%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Differentiation%26atitle%3DGenetic%252Binactivation%252Bof%252BRIP1%252Bkinase%252Bdoes%252Bnot%252Bameliorate%252Bdisease%252Bin%252Ba%252Bmouse%252Bmodel%252Bof%252BALS%26aulast%3DDominguez%26aufirst%3DSara%26date%3D2021%26date%3D2020%26volume%3D28%26issue%3D3%26spage%3D915%26epage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Hao  Ma</span>, <span class="hlFld-ContribAuthor ">Jiaxuan  Huang</span>, <span class="hlFld-ContribAuthor ">Zhengguang  Yao</span>, <span class="hlFld-ContribAuthor ">Jianqiang  Yu</span>, <span class="hlFld-ContribAuthor ">Wannian  Zhang</span>, <span class="hlFld-ContribAuthor ">Lichao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhibin  Wang</span>, <span class="hlFld-ContribAuthor ">Chunlin  Zhuang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of bardoxolone derivatives as novel orally active necroptosis inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>212 </em>, 113030. <a href="https://doi.org/10.1016/j.ejmech.2020.113030" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113030%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bbardoxolone%252Bderivatives%252Bas%252Bnovel%252Borally%252Bactive%252Bnecroptosis%252Binhibitors%26aulast%3DWang%26aufirst%3DYuanyuan%26date%3D2021%26volume%3D212%26spage%3D113030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philip A.  Harris</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of RIP1 kinase: a patent review (2016–present). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (2)
                                     , 137-151. <a href="https://doi.org/10.1080/13543776.2021.1854729" title="DOI URL">https://doi.org/10.1080/13543776.2021.1854729</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1854729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1854729%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DInhibitors%252Bof%252BRIP1%252Bkinase%25253A%252Ba%252Bpatent%252Breview%252B%2525282016%2525E2%252580%252593present%252529%26aulast%3DHarris%26aufirst%3DPhilip%2BA.%26date%3D2021%26date%3D2020%26volume%3D31%26issue%3D2%26spage%3D137%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory D.  Cuny</span>, <span class="hlFld-ContribAuthor ">Alexei  Degterev</span>. </span><span class="cited-content_cbyCitation_article-title">RIPK protein kinase family: Atypical lives of typical kinases. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cell & Developmental Biology</span><span> <strong>2021,</strong> <em>109 </em>, 96-105. <a href="https://doi.org/10.1016/j.semcdb.2020.06.014" title="DOI URL">https://doi.org/10.1016/j.semcdb.2020.06.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcdb.2020.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcdb.2020.06.014%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cell%2520%2526%2520Developmental%2520Biology%26atitle%3DRIPK%252Bprotein%252Bkinase%252Bfamily%25253A%252BAtypical%252Blives%252Bof%252Btypical%252Bkinases%26aulast%3DCuny%26aufirst%3DGregory%2BD.%26date%3D2021%26volume%3D109%26spage%3D96%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Doris A.  Schuetz</span>, <span class="hlFld-ContribAuthor ">Lars  Richter</span>, <span class="hlFld-ContribAuthor ">Riccardo  Martini</span>, <span class="hlFld-ContribAuthor ">Gerhard F.  Ecker</span>. </span><span class="cited-content_cbyCitation_article-title">A structure–kinetic relationship study using matched molecular pair analysis. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 1285-1294. <a href="https://doi.org/10.1039/D0MD00178C" title="DOI URL">https://doi.org/10.1039/D0MD00178C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00178C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00178C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DA%252Bstructure%2525E2%252580%252593kinetic%252Brelationship%252Bstudy%252Busing%252Bmatched%252Bmolecular%252Bpair%252Banalysis%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D11%26spage%3D1285%26epage%3D1294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed  Benchekroun</span>, <span class="hlFld-ContribAuthor ">Ludmila  Ermolenko</span>, <span class="hlFld-ContribAuthor ">Minh Quan  Tran</span>, <span class="hlFld-ContribAuthor ">Agathe  Vagneux</span>, <span class="hlFld-ContribAuthor ">Hristo  Nedev</span>, <span class="hlFld-ContribAuthor ">Claire  Delehouzé</span>, <span class="hlFld-ContribAuthor ">Mohamed  Souab</span>, <span class="hlFld-ContribAuthor ">Blandine  Baratte</span>, <span class="hlFld-ContribAuthor ">Béatrice  Josselin</span>, <span class="hlFld-ContribAuthor ">Bogdan I.  Iorga</span>, <span class="hlFld-ContribAuthor ">Sandrine  Ruchaud</span>, <span class="hlFld-ContribAuthor ">Stéphane  Bach</span>, <span class="hlFld-ContribAuthor ">Ali  Al-Mourabit</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of simplified benzazole fragments derived from the marine benzosceptrin B as necroptosis inhibitors involving the receptor interacting protein Kinase-1. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>201 </em>, 112337. <a href="https://doi.org/10.1016/j.ejmech.2020.112337" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112337</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112337%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bsimplified%252Bbenzazole%252Bfragments%252Bderived%252Bfrom%252Bthe%252Bmarine%252Bbenzosceptrin%252BB%252Bas%252Bnecroptosis%252Binhibitors%252Binvolving%252Bthe%252Breceptor%252Binteracting%252Bprotein%252BKinase-1%26aulast%3DBenchekroun%26aufirst%3DMohamed%26date%3D2020%26volume%3D201%26spage%3D112337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren  Mifflin</span>, <span class="hlFld-ContribAuthor ">Dimitry  Ofengeim</span>, <span class="hlFld-ContribAuthor ">Junying  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Receptor-interacting protein kinase 1 (RIPK1) as a therapeutic target. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2020,</strong> <em>19 </em>
                                    (8)
                                     , 553-571. <a href="https://doi.org/10.1038/s41573-020-0071-y" title="DOI URL">https://doi.org/10.1038/s41573-020-0071-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-0071-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-0071-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DReceptor-interacting%252Bprotein%252Bkinase%252B1%252B%252528RIPK1%252529%252Bas%252Ba%252Btherapeutic%252Btarget%26aulast%3DMifflin%26aufirst%3DLauren%26date%3D2020%26date%3D2020%26volume%3D19%26issue%3D8%26spage%3D553%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xia  Qin</span>, <span class="hlFld-ContribAuthor ">Longmiao  Hu</span>, <span class="hlFld-ContribAuthor ">Shen-nan  Shi</span>, <span class="hlFld-ContribAuthor ">Xiaofei  Chen</span>, <span class="hlFld-ContribAuthor ">Chunlin  Zhuang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhang</span>, <span class="hlFld-ContribAuthor ">Siriporn  Jitkaew</span>, <span class="hlFld-ContribAuthor ">Xiufeng  Pang</span>, <span class="hlFld-ContribAuthor ">Jianqiang  Yu</span>, <span class="hlFld-ContribAuthor ">Ye-xiong  Tan</span>, <span class="hlFld-ContribAuthor ">Hong-yang  Wang</span>, <span class="hlFld-ContribAuthor ">Zhenyu  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">The Bcr-Abl inhibitor GNF-7 inhibits necroptosis and ameliorates acute kidney injury by targeting RIPK1 and RIPK3 kinases. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>177 </em>, 113947. <a href="https://doi.org/10.1016/j.bcp.2020.113947" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.113947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.113947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.113947%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DThe%252BBcr-Abl%252Binhibitor%252BGNF-7%252Binhibits%252Bnecroptosis%252Band%252Bameliorates%252Bacute%252Bkidney%252Binjury%252Bby%252Btargeting%252BRIPK1%252Band%252BRIPK3%252Bkinases%26aulast%3DQin%26aufirst%3DXia%26date%3D2020%26volume%3D177%26spage%3D113947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sofie  Martens</span>, <span class="hlFld-ContribAuthor ">Sam  Hofmans</span>, <span class="hlFld-ContribAuthor ">Wim  Declercq</span>, <span class="hlFld-ContribAuthor ">Koen  Augustyns</span>, <span class="hlFld-ContribAuthor ">Peter  Vandenabeele</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors Targeting RIPK1/RIPK3: Old and New Drugs. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (3)
                                     , 209-224. <a href="https://doi.org/10.1016/j.tips.2020.01.002" title="DOI URL">https://doi.org/10.1016/j.tips.2020.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DInhibitors%252BTargeting%252BRIPK1%25252FRIPK3%25253A%252BOld%252Band%252BNew%252BDrugs%26aulast%3DMartens%26aufirst%3DSofie%26date%3D2020%26volume%3D41%26issue%3D3%26spage%3D209%26epage%3D224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janos  Sapi</span>, <span class="hlFld-ContribAuthor ">Stéphane  Gérard</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclic 5-6 Systems: Three Heteroatoms 2:1. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00021-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00021-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00021-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00021-4%26sid%3Dliteratum%253Aachs%26atitle%3DBicyclic%252B5-6%252BSystems%25253A%252BThree%252BHeteroatoms%252B2%25253A1%26aulast%3DSapi%26aufirst%3DJanos%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3Dvol.%252010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam G.  Meyer</span>, <span class="hlFld-ContribAuthor ">Alex C.  Bissember</span>, <span class="hlFld-ContribAuthor ">Christopher J.T.  Hyland</span>, <span class="hlFld-ContribAuthor ">Charlotte C.  Williams</span>, <span class="hlFld-ContribAuthor ">Monika  Szabo</span>, <span class="hlFld-ContribAuthor ">Melanie A.  Pearsall</span>, <span class="hlFld-ContribAuthor ">Isabel K.  Hyland</span>, <span class="hlFld-ContribAuthor ">Wesley J.  Olivier</span>. </span><span class="cited-content_cbyCitation_article-title">Seven-Membered Rings. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 597-647. <a href="https://doi.org/10.1016/B978-0-12-819962-6.00016-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-819962-6.00016-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819962-6.00016-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819962-6.00016-6%26sid%3Dliteratum%253Aachs%26atitle%3DSeven-Membered%252BRings%26aulast%3DMeyer%26aufirst%3DAdam%2BG.%26date%3D2020%26spage%3D597%26epage%3D647%26pub%3DElsevier%26date%3D2020%26volume%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro A.  Dionísio</span>, <span class="hlFld-ContribAuthor ">Joana D.  Amaral</span>, <span class="hlFld-ContribAuthor ">Cecília M.P.  Rodrigues</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular mechanisms of necroptosis and relevance for neurodegenerative diseases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 31-82. <a href="https://doi.org/10.1016/bs.ircmb.2019.12.006" title="DOI URL">https://doi.org/10.1016/bs.ircmb.2019.12.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.ircmb.2019.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.ircmb.2019.12.006%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252Bmechanisms%252Bof%252Bnecroptosis%252Band%252Brelevance%252Bfor%252Bneurodegenerative%252Bdiseases%26aulast%3DDion%25C3%25ADsio%26aufirst%3DPedro%2BA.%26date%3D2020%26spage%3D31%26epage%3D82%26pub%3DElsevier%26atitle%3DCell%252BDeath%252BRegulation%252BIn%252BHealth%252BAnd%252BDisease%252B-%252BPart%252BC%26date%3D2020%26volume%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solvent‐exposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jim  Küppers</span>, <span class="hlFld-ContribAuthor ">Tobias  Benkel</span>, <span class="hlFld-ContribAuthor ">Suvi  Annala</span>, <span class="hlFld-ContribAuthor ">Gregor  Schnakenburg</span>, <span class="hlFld-ContribAuthor ">Evi  Kostenis</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">BIM-46174 fragments as potential ligands of G proteins. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 1838-1843. <a href="https://doi.org/10.1039/C9MD00269C" title="DOI URL">https://doi.org/10.1039/C9MD00269C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00269C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00269C%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DBIM-46174%252Bfragments%252Bas%252Bpotential%252Bligands%252Bof%252BG%252Bproteins%26aulast%3DK%25C3%25BCppers%26aufirst%3DJim%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D1838%26epage%3D1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James E.  Kath</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Baranczak</span>. </span><span class="cited-content_cbyCitation_article-title">Target engagement approaches for pharmacological evaluation in animal models. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (63)
                                     , 9241-9250. <a href="https://doi.org/10.1039/C9CC02824B" title="DOI URL">https://doi.org/10.1039/C9CC02824B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC02824B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC02824B%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DTarget%252Bengagement%252Bapproaches%252Bfor%252Bpharmacological%252Bevaluation%252Bin%252Banimal%252Bmodels%26aulast%3DKath%26aufirst%3DJames%2BE.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D63%26spage%3D9241%26epage%3D9250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuping  Liu</span>, <span class="hlFld-ContribAuthor ">Ting  Liu</span>, <span class="hlFld-ContribAuthor ">Tiantian  Lei</span>, <span class="hlFld-ContribAuthor ">Dingding  Zhang</span>, <span class="hlFld-ContribAuthor ">Suya  Du</span>, <span class="hlFld-ContribAuthor ">Lea  Girani</span>, <span class="hlFld-ContribAuthor ">Dandan  Qi</span>, <span class="hlFld-ContribAuthor ">Chen  Lin</span>, <span class="hlFld-ContribAuthor ">Rongsheng  Tong</span>, <span class="hlFld-ContribAuthor ">Yi  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">RIP1/RIP3-regulated necroptosis as a target for multifaceted disease therapy (Review). </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Medicine</span><span> <strong>2019,</strong> <em> </em><a href="https://doi.org/10.3892/ijmm.2019.4244" title="DOI URL">https://doi.org/10.3892/ijmm.2019.4244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ijmm.2019.4244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fijmm.2019.4244%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Medicine%26atitle%3DRIP1%25252FRIP3-regulated%252Bnecroptosis%252Bas%252Ba%252Btarget%252Bfor%252Bmultifaceted%252Bdisease%252Btherapy%252B%252528Review%252529%26aulast%3DLiu%26aufirst%3DYuping%26date%3D2019%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory L.  Hamilton</span>, <span class="hlFld-ContribAuthor ">Huifen  Chen</span>, <span class="hlFld-ContribAuthor ">Gauri  Deshmukh</span>, <span class="hlFld-ContribAuthor ">Charles  Eigenbrot</span>, <span class="hlFld-ContribAuthor ">Rina  Fong</span>, <span class="hlFld-ContribAuthor ">Adam  Johnson</span>, <span class="hlFld-ContribAuthor ">Pawan Bir  Kohli</span>, <span class="hlFld-ContribAuthor ">Patrick J.  Lupardus</span>, <span class="hlFld-ContribAuthor ">Bianca M.  Liederer</span>, <span class="hlFld-ContribAuthor ">Sreemathy  Ramaswamy</span>, <span class="hlFld-ContribAuthor ">Haowei  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhaowu  Xu</span>, <span class="hlFld-ContribAuthor ">Yunliang  Zhu</span>, <span class="hlFld-ContribAuthor ">Domagoj  Vucic</span>, <span class="hlFld-ContribAuthor ">Snahel  Patel</span>. </span><span class="cited-content_cbyCitation_article-title">Potent and selective inhibitors of receptor-interacting protein kinase 1 that lack an aromatic back pocket group. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (12)
                                     , 1497-1501. <a href="https://doi.org/10.1016/j.bmcl.2019.04.014" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.04.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.04.014%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPotent%252Band%252Bselective%252Binhibitors%252Bof%252Breceptor-interacting%252Bprotein%252Bkinase%252B1%252Bthat%252Black%252Ban%252Baromatic%252Bback%252Bpocket%252Bgroup%26aulast%3DHamilton%26aufirst%3DGregory%2BL.%26date%3D2019%26volume%3D29%26issue%3D12%26spage%3D1497%26epage%3D1501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexei  Degterev</span>, <span class="hlFld-ContribAuthor ">Dimitry  Ofengeim</span>, <span class="hlFld-ContribAuthor ">Junying  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RIPK1 for the treatment of human diseases. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2019,</strong> <em>116 </em>
                                    (20)
                                     , 9714-9722. <a href="https://doi.org/10.1073/pnas.1901179116" title="DOI URL">https://doi.org/10.1073/pnas.1901179116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1901179116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1901179116%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DTargeting%252BRIPK1%252Bfor%252Bthe%252Btreatment%252Bof%252Bhuman%252Bdiseases%26aulast%3DDegterev%26aufirst%3DAlexei%26date%3D2019%26date%3D2019%26volume%3D116%26issue%3D20%26spage%3D9714%26epage%3D9722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joerg  Holenz</span>, <span class="hlFld-ContribAuthor ">Patrick  Stoy</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Lead Generation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 517-524. <a href="https://doi.org/10.1016/j.bmcl.2018.12.001" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.001%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAdvances%252Bin%252BLead%252BGeneration%26aulast%3DHolenz%26aufirst%3DJoerg%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D517%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junying  Yuan</span>, <span class="hlFld-ContribAuthor ">Palak  Amin</span>, <span class="hlFld-ContribAuthor ">Dimitry  Ofengeim</span>. </span><span class="cited-content_cbyCitation_article-title">Necroptosis and RIPK1-mediated neuroinflammation in CNS diseases. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Neuroscience</span><span> <strong>2019,</strong> <em>20 </em>
                                    (1)
                                     , 19-33. <a href="https://doi.org/10.1038/s41583-018-0093-1" title="DOI URL">https://doi.org/10.1038/s41583-018-0093-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41583-018-0093-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41583-018-0093-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Neuroscience%26atitle%3DNecroptosis%252Band%252BRIPK1-mediated%252Bneuroinflammation%252Bin%252BCNS%252Bdiseases%26aulast%3DYuan%26aufirst%3DJunying%26date%3D2019%26date%3D2018%26volume%3D20%26issue%3D1%26spage%3D19%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eun-Jung  In</span>, <span class="hlFld-ContribAuthor ">Yuno  Lee</span>, <span class="hlFld-ContribAuthor ">Sushruta  Koppula</span>, <span class="hlFld-ContribAuthor ">Tae-Yeon  Kim</span>, <span class="hlFld-ContribAuthor ">Jun-Hyuk  Han</span>, <span class="hlFld-ContribAuthor ">Kwang-Ho  Lee</span>, <span class="hlFld-ContribAuthor ">Tae-Bong  Kang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Characterization of NTB451 as a Potential Inhibitor of Necroptosis. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (11)
                                     , 2884. <a href="https://doi.org/10.3390/molecules23112884" title="DOI URL">https://doi.org/10.3390/molecules23112884</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23112884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23112884%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Band%252BCharacterization%252Bof%252BNTB451%252Bas%252Ba%252BPotential%252BInhibitor%252Bof%252BNecroptosis%26aulast%3DIn%26aufirst%3DEun-Jung%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D11%26spage%3D2884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Lu</span>, <span class="hlFld-ContribAuthor ">James N  Iuliano</span>, <span class="hlFld-ContribAuthor ">Peter J  Tonge</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–kinetic relationships that control the residence time of drug–target complexes: insights from molecular structure and dynamics. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>44 </em>, 101-109. <a href="https://doi.org/10.1016/j.cbpa.2018.06.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.06.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DStructure%2525E2%252580%252593kinetic%252Brelationships%252Bthat%252Bcontrol%252Bthe%252Bresidence%252Btime%252Bof%252Bdrug%2525E2%252580%252593target%252Bcomplexes%25253A%252Binsights%252Bfrom%252Bmolecular%252Bstructure%252Band%252Bdynamics%26aulast%3DLu%26aufirst%3DHao%26date%3D2018%26volume%3D44%26spage%3D101%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0001.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of reported RIP1 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0002.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of fluorescent probe <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0003.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. HTS hit compound <b>10</b>. (A) Structure of compound <b>10</b> and <i>K</i><sub>i</sub> for human RIP1 kinase. (B) Co-crystal structure of <b>10</b> with hRIP1 kinase. (C) Overlay study between compounds <b>6</b> (yellow) and <b>10</b> (white) in hRIP1 kinase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0004.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design concept for a new class of brain-penetrating RIP1 kinase inhibitors. (a) Binding affinity for human RIP1 kinase. (b) Calculated by Daylight. (c) Efflux ratio in P-gp-overexpressing cells at 1 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0005.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Synthetic strategy based on the docking model. (A) The docking study between <b>14</b> and RIP1 kinase. The DLG motif including Asp156 in the activation loop was omitted for simplicity. (B) Image of the protein surface around DP2. (C) Schematic illustration of <b>14</b> in RIP1 kinase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0006.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of compound <b>22</b> (white).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0007.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of compound <b>22</b> in the EAE model. RIP1 kinase inhibitor <b>22</b> (10 mg/kg) or vehicle (0.5% methylcellulose) was orally administered to mice twice a day for 26 days. (A,B) The EAE clinical score of (A) the time course and (B) the cumulative clinical score from day 0 to day 26 are shown. Data are presented as the mean ± SE (<i>n</i> = 12). **<i>p</i> < 0.05 compared to the naïve group (Dunn’s test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0008.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. SKR and relationships between kinetic profiles and lipophilicity in the modification of R<sup>1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0009.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Relationships between log <i>k</i><sub>on</sub> and p<i>k</i><sub>off</sub> in modification of R<sup>1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) methyl propiolate, diphenyl ether, microwave, 200 °C, 35%; (b) bromine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 56%; (c) 4-methoxybenzyl chloride, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 21%; (d) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, 75%; (e) 6 M HCl aq/THF, rt–60 °C, 98%; (f) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 65%; (g) AlMe<sub>3</sub>, toluene, 120 °C, under Ar, 50%; (h) Pd(OH)<sub>2</sub>, MeOH, 50 °C, under H<sub>2</sub>, 66%.</p></p></figure><figure data-id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, 1,2-dibromoethane, DMF, 60 °C, 22%.</p></p></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0012.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl 2-ethoxy-2-iminoacetate, NaOAc, AcOH, 110 °C, 9.1%; (b) (1) 2 M NaOH aq, EtOH, 100 °C, (2) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, HATU, DIPEA, DMF, rt, 59%; (c) Cs<sub>2</sub>CO<sub>3</sub>, 1,2-dibromoethane, DMF, 100 °C, 3.2%.</p></p></figure><figure data-id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, benzyl bromide, DMF, rt, 64%; (b) bromine, MeCN, rt, 98%; (c) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, 94%; (d) 6 M HCl aq/THF, 60 °C, 94%; (e) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 94%; (f) AlMe<sub>3</sub>, toluene, 120 °C, under Ar, 41%; (g) Pd/C, 1 M HCl aq, MeOH, rt, under H<sub>2</sub>, 82%; (h) R<sup>1</sup>CH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF or DMA, rt–80 °C, 1.4–65%.</p></p></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0014.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) DMFDMA, toluene, 65 °C, (2) NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O, AcOH, rt, 52% in 2 steps; (b) benzyl chloride, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 64%; (c) TFA, DCM, rt, 86%; (d) MnO<sub>2</sub>, DCM, 40 °C, 59%; (e) (methoxymethyl)triphenylphosphonium chloride, <sup><i>t</i></sup>BuOK, THF, rt, 53%; (f) (1) 6 M HCl aq/THF, rt, (g) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt; (h) AlMe<sub>3</sub>, toluene, 100 °C, 6% in 3 steps.</p></p></figure><figure data-id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0015.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzylhydrazine hydrochloride, K<sub>2</sub>CO<sub>3</sub>, EtOH, 90 °C, 36%; (b) phosphoryl bromide, DMF, 1,2-dichloroethane, 90 °C, 48%; (c) phosphoryl chloride, DMF, 90 °C, 36%; (d) (methoxymethyl)triphenylphosphonium chloride, <sup><i>t</i></sup>BuOK, THF, rt, 30–42%; (e) 6 M HCl aq/THF, rt; (f) (<i>S</i>)-3-amino-5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]oxazepin-4(5<i>H</i>)-one hydrochloride, 2-picoline boran, acetic acid, MeOH, rt, 67–94% in 2 steps; (g) AlMe<sub>3</sub>, toluene, 100 °C, 23–82%; (h) Pd(PPh<sub>3</sub>)<sub>4</sub>, 2,4,6-trimethyl-1,3,5,2,4,6-trioxatriborinane, KOH, THF, 100 °C, under Ar, 9%; (i) dicyanozinc, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 100 °C, under Ar, 90%; (j) K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O<sub>2</sub>, DMSO, rt, 60%.</p></p></figure><figure data-id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0016.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) R<sup>1</sup>CH<sub>2</sub>Br, NaH, THF, rt, 95–97%; (b) bromine, MeCN, rt, 88–98%; (c) (<i>E</i>)-2-(2-ethoxyvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DME/water, 90 °C, under Ar, 84–87%; (d) 6 M HCl a./THF, rt, 68%–quant</p></p></figure><figure data-id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0017.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-3-hydroxypropanoic acid, NaH, DMF, rt, 22–74%; (b) Fe, NH<sub>4</sub>Cl, EtOH/water, 80 °C; (c) Zn, AcOH, 0 °C; (d) HATU, TEA, DMSO, rt, 3–26% in 2 steps; (e) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, rt, 45–75%; (f) Pd/C, THF, rt, under H<sub>2</sub>, 99%; (g) methylamine or dimethylamine, HATU, TEA, DMF, rt, 85–100%; (h) 4 M HCl in EtOAc, THF, rt, 60–98%.</p></p></figure><figure data-id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/medium/jm-2017-01647y_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0018.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01647/20180315/images/large/jm-2017-01647y_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01647&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>70a</b>–<b>d</b>, 2-picoline boran, acetic acid, MeOH, rt, 48–100%; (b) AlMe<sub>3</sub>, toluene, 100 °C, 29–89%; (c) dicyanozinc, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 100 °C, under Ar, 62–85%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i164">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36703" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36703" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beere, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span> <span> </span><span class="NLM_article-title">Necroptosis in development, inflammation and disease</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrm.2016.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27999438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFahtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=127-136&author=R.+Weinlichauthor=A.+Oberstauthor=H.+M.+Beereauthor=D.+R.+Green&title=Necroptosis+in+development%2C+inflammation+and+disease&doi=10.1038%2Fnrm.2016.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Necroptosis in development, inflammation and disease</span></div><div class="casAuthors">Weinlich, Ricardo; Oberst, Andrew; Beere, Helen M.; Green, Douglas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the early 2000s, receptor-interacting serine/threonine protein kinase 1 (RIPK1), a mol. already recognized as an important regulator of cell survival, inflammation and disease, was attributed an addnl. function: the regulation of a novel cell death pathway that came to be known as necroptosis.  Subsequently, the related kinase RIPK3 and its substrate mixed-lineage kinase domain-like protein (MLKL) were also implicated in the necroptotic pathway, and links between this pathway and apoptosis were established.  In this Timeline article, we outline the discoveries that have helped to identify the roles of RIPK1, RIPK3, MLKL and other regulators of necroptosis, and how they interact to det. cell fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry23k0aTKuWbVg90H21EOLACvtfcHk0liiEYX_Rc5y5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFahtrrJ&md5=7a4b7af9a1da473d45ea833a54558d53</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.149%26sid%3Dliteratum%253Aachs%26aulast%3DWeinlich%26aufirst%3DR.%26aulast%3DOberst%26aufirst%3DA.%26aulast%3DBeere%26aufirst%3DH.%2BM.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26atitle%3DNecroptosis%2520in%2520development%252C%2520inflammation%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D127%26epage%3D136%26doi%3D10.1038%2Fnrm.2016.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Complex pathologic roles of RIPK1 and RIPK3: moving beyond necroptosis</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.tips.2016.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=28126382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt12ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=202-225&author=K.+W.+Wegnerauthor=D.+Salehauthor=A.+Degterev&title=Complex+pathologic+roles+of+RIPK1+and+RIPK3%3A+moving+beyond+necroptosis&doi=10.1016%2Fj.tips.2016.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Complex Pathologic Roles of RIPK1 and RIPK3: Moving Beyond Necroptosis</span></div><div class="casAuthors">Wegner, Kelby W.; Saleh, Danish; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-225</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A process of regulated necrosis, termed necroptosis, has been recognized as a major contributor to cell death and inflammation occurring under a wide range of pathol. settings.  The core event in necroptosis is the formation of the detergent-insol. 'necrosome' complex of homologous Ser/Thr kinases, receptor protein interacting kinase 1 (RIPK1) and receptor interacting protein kinase 3 (RIPK3), which promotes phosphorylation of a key prodeath effector, mixed lineage kinase domain-like (MLKL), by RIPK3.  Core necroptosis mediators are under multiple controls, which have been a subject of intense investigation.  Addnl., non-necroptotic functions of these factors, primarily in controlling apoptosis and inflammatory responses, have also begun to emerge.  This review will provide an overview of the current understanding of the human disease relevance of this pathway, and potential therapeutic strategies, targeting necroptosis mediators in various pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeKXMgW5Q3rLVg90H21EOLACvtfcHk0liiEYX_Rc5y5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt12ltA%253D%253D&md5=296e25764f643a458906f2a9aac1df05</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWegner%26aufirst%3DK.%2BW.%26aulast%3DSaleh%26aufirst%3DD.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DComplex%2520pathologic%2520roles%2520of%2520RIPK1%2520and%2520RIPK3%253A%2520moving%2520beyond%2520necroptosis%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26spage%3D202%26epage%3D225%26doi%3D10.1016%2Fj.tips.2016.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span> <span> </span><span class="NLM_article-title">The contribution of necroptosis in neurodegenerative diseases</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2117</span>– <span class="NLM_lpage">2126</span>, <span class="refDoi"> DOI: 10.1007/s11064-017-2249-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1007%2Fs11064-017-2249-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=28382594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1KmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=2117-2126&author=L.+Shaoauthor=S.+Yuauthor=W.+Jiauthor=H.+Liauthor=Y.+Gao&title=The+contribution+of+necroptosis+in+neurodegenerative+diseases&doi=10.1007%2Fs11064-017-2249-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Contribution of Necroptosis in Neurodegenerative Diseases</span></div><div class="casAuthors">Shao, Lifei; Yu, Shuping; Ji, Wei; Li, Haizhen; Gao, Yilu</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2117-2126</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past decades, cell apoptosis has been significantly reputed as an accidental, redundant and alternative manner of cell demise which partakes in homeostasis in the development of extensive diseases.  Nevertheless, necroptosis, another novel manner of cell death through a caspase-independent way, esp. in neurodegenerative diseases remains ambiguous.  The cognition of this form of cell demise is helpful to understand other forms of morphol. resemblance of necrosis.  Addnl., the concrete signal mechanism in the regulation of necroptosis is beneficial to the diagnosis and treatment of neurodegenerative diseases.  Recent studies have demonstrated that necroptotic inhibitor, 24(S)-Hydroxycholesterol and partial specific histone deacetylase inhibitors could alleviate pathogenetic conditions of neurodegenerative diseases via necroptosis pathway.  In this review, we summarize recent researches about mechanisms and modulation of necroptotic signaling pathways and probe into the role of programmed necroptotic cell demise in neurodegenerative diseases such as Parkinson's disease, Multiple sclerosis, Amyotrophic lateral sclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmvV7Ouen9CrVg90H21EOLACvtfcHk0ljH_BXASIMkCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1KmsbY%253D&md5=f20a352421c3060c79f2cd960f8ad87a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs11064-017-2249-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11064-017-2249-1%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DY.%26atitle%3DThe%2520contribution%2520of%2520necroptosis%2520in%2520neurodegenerative%2520diseases%26jtitle%3DNeurochem.%2520Res.%26date%3D2017%26volume%3D42%26spage%3D2117%26epage%3D2126%26doi%3D10.1007%2Fs11064-017-2249-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWitt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakifahmetoglu-Norberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Py, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlink Lima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Activation of necroptosis in multiple sclerosis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1836</span>– <span class="NLM_lpage">1849</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.celrep.2015.02.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=25801023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1836-1849&author=D.+Ofengeimauthor=Y.+Itoauthor=A.+Najafovauthor=Y.+Zhangauthor=B.+Shanauthor=J.+P.+DeWittauthor=J.+Yeauthor=X.+Zhangauthor=A.+Changauthor=H.+Vakifahmetoglu-Norbergauthor=J.+Gengauthor=B.+Pyauthor=W.+Zhouauthor=P.+Aminauthor=J.+Berlink+Limaauthor=C.+Qiauthor=Q.+Yuauthor=B.+Trappauthor=J.+Yuan&title=Activation+of+necroptosis+in+multiple+sclerosis&doi=10.1016%2Fj.celrep.2015.02.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Necroptosis in Multiple Sclerosis</span></div><div class="casAuthors">Ofengeim, Dimitry; Ito, Yasushi; Najafov, Ayaz; Zhang, Yaoyang; Shan, Bing; DeWitt, Judy Park; Ye, Juanying; Zhang, Xumin; Chang, Ansi; Vakifahmetoglu-Norberg, Helin; Geng, Jiefei; Py, Benedicte; Zhou, Wen; Amin, Palak; Lima, Jonilson Berlink; Qi, Chunting; Yu, Qiang; Trapp, Bruce; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1836-1849</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Multiple sclerosis (MS), a common neurodegenerative disease of the CNS, is characterized by the loss of oligodendrocytes and demyelination.  Tumor necrosis factor α (TNF-α), a proinflammatory cytokine implicated in MS, can activate necroptosis, a necrotic cell death pathway regulated by RIPK1 and RIPK3 under caspase-8-deficient conditions.  Here, we demonstrate defective caspase-8 activation, as well as activation of RIPK1, RIPK3, and MLKL, the hallmark mediators of necroptosis, in the cortical lesions of human MS pathol. samples.  Furthermore, we show that MS pathol. samples are characterized by an increased insol. proteome in common with other neurodegenerative diseases such as Alzheimer's disease (AD), Parkinson's disease (PD), and Huntington's disease (HD).  Finally, we show that necroptosis mediates oligodendrocyte degeneration induced by TNF-α and that inhibition of RIPK1 protects against oligodendrocyte cell death in two animal models of MS and in culture.  Our findings demonstrate that necroptosis is involved in MS and suggest that targeting RIPK1 may represent a therapeutic strategy for MS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDaFlNf5zTmrVg90H21EOLACvtfcHk0ljH_BXASIMkCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktrk%253D&md5=561b6327d7b454d5694df7b69c0713f1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.051%26sid%3Dliteratum%253Aachs%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DNajafov%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DB.%26aulast%3DDeWitt%26aufirst%3DJ.%2BP.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DVakifahmetoglu-Norberg%26aufirst%3DH.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DPy%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DAmin%26aufirst%3DP.%26aulast%3DBerlink%2BLima%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DTrapp%26aufirst%3DB.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DActivation%2520of%2520necroptosis%2520in%2520multiple%2520sclerosis%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1836%26epage%3D1849%26doi%3D10.1016%2Fj.celrep.2015.02.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. B.</span></span> <span> </span><span class="NLM_article-title">Necrostatin-1 protection of dopaminergic neurons</span>. <i>Neural Regener. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.4103/1673-5374.160108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4103%2F1673-5374.160108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26330837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BC287otlOkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1120-1124&author=J.+R.+Wuauthor=J.+Wangauthor=S.+K.+Zhouauthor=L.+Yangauthor=J.+L.+Yinauthor=J.+P.+Caoauthor=Y.+B.+Cheng&title=Necrostatin-1+protection+of+dopaminergic+neurons&doi=10.4103%2F1673-5374.160108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Necrostatin-1 protection of dopaminergic neurons</span></div><div class="casAuthors">Wu Jing-Ru; Cao Jun-Ping; Wang Jie; Yin Jia-le; Zhou Sheng-Kui; Yang Long; Cheng Yan-Bo</div><div class="citationInfo"><span class="NLM_cas:title">Neural regeneration research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1120-4</span>
        ISSN:<span class="NLM_cas:issn">1673-5374</span>.
    </div><div class="casAbstract">Necroptosis is characterized by programmed necrotic cell death and autophagic activation and might be involved in the death process of dopaminergic neurons in Parkinson's disease.  We hypothesized that necrostatin-1 could block necroptosis and give protection to dopaminergic neurons.  There is likely to be crosstalk between necroptosis and other cell death pathways, such as apoptosis and autophagy.  PC12 cells were pretreated with necroststin-1 1 hour before exposure to 6-hydroxydopamine.  We examined cell viability, mitochondrial membrane potential and expression patterns of apoptotic and necroptotic death signaling proteins.  The results showed that the autophagy/lysosomal pathway is involved in the 6-hydroxydopamine-induced death process of PC12 cells.  Mitochondrial disability induced overactive autophagy, increased cathepsin B expression, and diminished Bcl-2 expression.  Necrostatin-1 within a certain concentration range (5-30 μM) elevated the viability of PC12 cells, stabilized mitochondrial membrane potential, inhibited excessive autophagy, reduced the expression of LC3-II and cathepsin B, and increased Bcl-2 expression.  These findings suggest that necrostatin-1 exerted a protective effect against injury on dopaminergic neurons.  Necrostatin-1 interacts with the apoptosis signaling pathway during this process.  This pathway could be a new neuroprotective and therapeutic target in Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQIKhwA2KI3uSnGiHuhFGv8fW6udTcc2eae3BG1EmY-Ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287otlOkug%253D%253D&md5=04516fc88aaf3bddc9924a3a4db96001</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4103%2F1673-5374.160108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F1673-5374.160108%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%2BK.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DJ.%2BL.%26aulast%3DCao%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DY.%2BB.%26atitle%3DNecrostatin-1%2520protection%2520of%2520dopaminergic%2520neurons%26jtitle%3DNeural%2520Regener.%2520Res.%26date%3D2015%26volume%3D10%26spage%3D1120%26epage%3D1124%26doi%3D10.4103%2F1673-5374.160108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przedborski, S.</span></span> <span> </span><span class="NLM_article-title">Axonal degeneration: RIPK1 multitasking in ALS</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">R932</span>– <span class="NLM_lpage">R934</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2016.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.cub.2016.08.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27780064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCrsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=R932-R934&author=K.+Politiauthor=S.+Przedborski&title=Axonal+degeneration%3A+RIPK1+multitasking+in+ALS&doi=10.1016%2Fj.cub.2016.08.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Axonal Degeneration: RIPK1 Multitasking in ALS</span></div><div class="casAuthors">Politi, Kristin; Przedborski, Serge</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">R932-R934</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The research of Y. Ito et al. (Science, 2016, 353, 603-8) is reviewed with commentary and refs.  A recent study reports that microglia and oligodendrocytes promote motor neuron degeneration by inducing inflammation and necroptosis in a manner dependent on receptor-interacting kinase 1 (RIPK1).  These findings could be significant for our understanding of the neurobiol. and treatment of neurodegenerative diseases like amyotrophic lateral sclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGresQaHd93y-rVg90H21EOLACvtfcHk0lj5zDN_X4_UpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCrsrzO&md5=842be679cf56ccc16b7e3bd6f397c9c6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2016.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2016.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DPrzedborski%26aufirst%3DS.%26atitle%3DAxonal%2520degeneration%253A%2520RIPK1%2520multitasking%2520in%2520ALS%26jtitle%3DCurr.%2520Biol.%26date%3D2016%26volume%3D26%26spage%3DR932%26epage%3DR934%26doi%3D10.1016%2Fj.cub.2016.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saberi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitomi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWitt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookhtiar, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouchida, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasparakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravits, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">RIPK1 mediates axonal degeneration by promoting inflammation and necroptosis in ALS</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1126/science.aaf6803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1126%2Fscience.aaf6803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=27493188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=603-608&author=Y.+Itoauthor=D.+Ofengeimauthor=A.+Najafovauthor=S.+Dasauthor=S.+Saberiauthor=Y.+Liauthor=J.+Hitomiauthor=H.+Zhuauthor=H.+Chenauthor=L.+Mayoauthor=J.+Gengauthor=P.+Aminauthor=J.+P.+DeWittauthor=A.+K.+Mookhtiarauthor=M.+Florezauthor=A.+T.+Ouchidaauthor=J.+B.+Fanauthor=M.+Pasparakisauthor=M.+A.+Kelliherauthor=J.+Ravitsauthor=J.+Yuan&title=RIPK1+mediates+axonal+degeneration+by+promoting+inflammation+and+necroptosis+in+ALS&doi=10.1126%2Fscience.aaf6803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RIPK1 mediates axonal degeneration by promoting inflammation and necroptosis in ALS</span></div><div class="casAuthors">Ito, Yasushi; Ofengeim, Dimitry; Najafov, Ayaz; Das, Sudeshna; Saberi, Shahram; Li, Ying; Hitomi, Junichi; Zhu, Hong; Chen, Hongbo; Mayo, Lior; Geng, Jiefei; Amin, Palak; DeWitt, Judy Park; Mookhtiar, Adnan Kasim; Florez, Marcus; Ouchida, Amanda Tomie; Fan, Jian-bing; Pasparakis, Manolis; Kelliher, Michelle A.; Ravits, John; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6299</span>),
    <span class="NLM_cas:pages">603-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mutations in the optineurin (OPTN) gene have been implicated in both familial and sporadic amyotrophic lateral sclerosis (ALS).  However, the role of this protein in the central nervous system (CNS) and how it may contribute to ALS pathol. are unclear.  Here, we found that optineurin actively suppressed receptor-interacting kinase 1 (RIPK1)-dependent signaling by regulating its turnover.  Loss of OPTN led to progressive dysmyelination and axonal degeneration through engagement of necroptotic machinery in the CNS, including RIPK1, RIPK3, and mixed lineage kinase domain-like protein (MLKL).  Furthermore, RIPK1- and RIPK3-mediated axonal pathol. was commonly obsd. in SOD1G93A transgenic mice and pathol. samples from human ALS patients.  Thus, RIPK1 and RIPK3 play a crit. role in mediating progressive axonal degeneration.  Furthermore, inhibiting RIPK1 kinase may provide an axonal protective strategy for the treatment of ALS and other human degenerative diseases characterized by axonal degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo120pi55BWcLVg90H21EOLACvtfcHk0lhO_hkiBKF20g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7jO&md5=6145b22849aa523b5dc1200dc8610c1c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf6803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf6803%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DY.%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DNajafov%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DSaberi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHitomi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMayo%26aufirst%3DL.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DAmin%26aufirst%3DP.%26aulast%3DDeWitt%26aufirst%3DJ.%2BP.%26aulast%3DMookhtiar%26aufirst%3DA.%2BK.%26aulast%3DFlorez%26aufirst%3DM.%26aulast%3DOuchida%26aufirst%3DA.%2BT.%26aulast%3DFan%26aufirst%3DJ.%2BB.%26aulast%3DPasparakis%26aufirst%3DM.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26aulast%3DRavits%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DRIPK1%2520mediates%2520axonal%2520degeneration%2520by%2520promoting%2520inflammation%2520and%2520necroptosis%2520in%2520ALS%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D603%26epage%3D608%26doi%3D10.1126%2Fscience.aaf6803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Chemical inhibitor of nonapoptotic cell death with therapeutic potential for ischemic brain injury</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1038/nchembio711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnchembio711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16408008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1OmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=112-119&author=A.+Degterevauthor=Z.+Huangauthor=M.+Boyceauthor=Y.+Liauthor=P.+Jagtapauthor=N.+Mizushimaauthor=G.+D.+Cunyauthor=T.+J.+Mitchisonauthor=M.+A.+Moskowitzauthor=J.+Yuan&title=Chemical+inhibitor+of+nonapoptotic+cell+death+with+therapeutic+potential+for+ischemic+brain+injury&doi=10.1038%2Fnchembio711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical inhibitor of nonapoptotic cell death with therapeutic potential for ischemic brain injury</span></div><div class="casAuthors">Degterev, Alexei; Huang, Zhihong; Boyce, Michael; Li, Yaqiao; Jagtap, Prakash; Mizushima, Noboru; Cuny, Gregory D.; Mitchison, Timothy J.; Moskowitz, Michael A.; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-119</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mechanism of apoptosis has been extensively characterized over the past decade, but little is known about alternative forms of regulated cell death.  Although stimulation of the Fas/TNFR receptor family triggers a canonical 'extrinsic' apoptosis pathway, the authors demonstrated that in the absence of intracellular apoptotic signaling it is capable of activating a common nonapoptotic death pathway, which the authors term necroptosis.  The authors showed that necroptosis is characterized by necrotic cell death morphol. and activation of autophagy.  The authors identified a specific and potent small-mol. inhibitor of necroptosis, necrostatin-1, which blocks a crit. step in necroptosis.  The authors demonstrated that necroptosis contributes to delayed mouse ischemic brain injury in vivo through a mechanism distinct from that of apoptosis and offers a new therapeutic target for stroke with an extended window for neuroprotection.  Our study identifies a previously undescribed basic cell-death pathway with potentially broad relevance to human pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiiBDVRJkdUbVg90H21EOLACvtfcHk0lhO_hkiBKF20g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1OmsLg%253D&md5=83f8c363ffb73ccc14bf11ec02577336</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio711%26sid%3Dliteratum%253Aachs%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DBoyce%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DJagtap%26aufirst%3DP.%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DChemical%2520inhibitor%2520of%2520nonapoptotic%2520cell%2520death%2520with%2520therapeutic%2520potential%2520for%2520ischemic%2520brain%2520injury%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D112%26epage%3D119%26doi%3D10.1038%2Fnchembio711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitomi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germscheid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ch’en, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korkina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span> <span> </span><span class="NLM_article-title">Identification of RIP1 kinase as a specific cellular target of necrostatins</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1038/nchembio.83</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnchembio.83" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=18408713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvVSmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=313-321&author=A.+Degterevauthor=J.+Hitomiauthor=M.+Germscheidauthor=I.+L.+Ch%E2%80%99enauthor=O.+Korkinaauthor=X.+Tengauthor=D.+Abbottauthor=G.+D.+Cunyauthor=C.+Yuanauthor=G.+Wagnerauthor=S.+M.+Hedrickauthor=S.+A.+Gerberauthor=A.+Lugovskoyauthor=J.+Yuan&title=Identification+of+RIP1+kinase+as+a+specific+cellular+target+of+necrostatins&doi=10.1038%2Fnchembio.83"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of RIP1 kinase as a specific cellular target of necrostatins</span></div><div class="casAuthors">Degterev, Alexei; Hitomi, Junichi; Germscheid, Megan; Ch'en, Irene L.; Korkina, Olga; Teng, Xin; Abbott, Derek; Cuny, Gregory D.; Yuan, Chengye; Wagner, Gerhard; Hedrick, Stephen M.; Gerber, Scott A.; Lugovskoy, Alexey; Yuan, Junying</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">313-321</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Necroptosis is a cellular mechanism of necrotic cell death induced by apoptotic stimuli in the form of death domain receptor engagement by their resp. ligands under conditions where apoptotic execution is prevented.  Although it occurs under regulated conditions, necroptotic cell death is characterized by the same morphol. features as unregulated necrotic death.  Here we report that necrostatin-1, a previously identified small-mol. inhibitor of necroptosis, is a selective allosteric inhibitor of the death domain receptor-assocd. adaptor kinase RIP1 in vitro.  We show that RIP1 is the primary cellular target responsible for the antinecroptosis activity of necrostatin-1.  In addn., we show that two other necrostatins, necrostatin-3 and necrostatin-5, also target the RIP1 kinase step in the necroptosis pathway, but through mechanisms distinct from that of necrostatin-1.  Overall, our data establish necrostatins as the first-in-class inhibitors of RIP1 kinase, the key upstream kinase involved in the activation of necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOaKMR-RMwdLVg90H21EOLACvtfcHk0lhO_hkiBKF20g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvVSmtL0%253D&md5=8ba3859c2b4c752da534441747a79bb5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.83&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.83%26sid%3Dliteratum%253Aachs%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DHitomi%26aufirst%3DJ.%26aulast%3DGermscheid%26aufirst%3DM.%26aulast%3DCh%25E2%2580%2599en%26aufirst%3DI.%2BL.%26aulast%3DKorkina%26aufirst%3DO.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DAbbott%26aufirst%3DD.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DHedrick%26aufirst%3DS.%2BM.%26aulast%3DGerber%26aufirst%3DS.%2BA.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DYuan%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520RIP1%2520kinase%2520as%2520a%2520specific%2520cellular%2520target%2520of%2520necrostatins%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D313%26epage%3D321%26doi%3D10.1038%2Fnchembio.83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duprez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootjans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cauwels, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerinckx, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DuHadaway, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roelandt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hauwermeiren, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Declercq, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callewaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prendergast, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenabeele, P.</span></span> <span> </span><span class="NLM_article-title">Necrostatin-1 analogues: critical issues on the specificity, activity and in vivo use in experimental disease models</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e437</span>, <span class="refDoi"> DOI: 10.1038/cddis.2012.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcddis.2012.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=23190609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=e437&author=N.+Takahashiauthor=L.+Duprezauthor=S.+Grootjansauthor=A.+Cauwelsauthor=W.+Nerinckxauthor=J.+B.+DuHadawayauthor=V.+Goossensauthor=R.+Roelandtauthor=F.+Van+Hauwermeirenauthor=C.+Libertauthor=W.+Declercqauthor=N.+Callewaertauthor=G.+C.+Prendergastauthor=A.+Degterevauthor=J.+Yuanauthor=P.+Vandenabeele&title=Necrostatin-1+analogues%3A+critical+issues+on+the+specificity%2C+activity+and+in+vivo+use+in+experimental+disease+models&doi=10.1038%2Fcddis.2012.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Necrostatin-1 analogues: critical issues on the specificity, activity and in vivo use in experimental disease models</span></div><div class="casAuthors">Takahashi, N.; Duprez, L.; Grootjans, S.; Cauwels, A.; Nerinckx, W.; DuHadaway, J. B.; Goossens, V.; Roelandt, R.; Van Hauwermeiren, F.; Libert, C.; Declercq, W.; Callewaert, N.; Prendergast, G. C.; Degterev, A.; Yuan, J.; Vandenabeele, P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Nov.</span>),
    <span class="NLM_cas:pages">e437, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Necrostatin-1 (Nec-1) is widely used in disease models to examine the contribution of receptor-interacting protein kinase (RIPK) 1 in cell death and inflammation.  We studied three Nec-1 analogs: Nec-1, the active inhibitor of RIPK1, Nec-1 inactive (Nec-1i), its inactive variant, and Nec-1 stable (Nec-1s), its more stable variant.  We report that Nec-1 is identical to methyl-thiohydantoin-tryptophan, an inhibitor of the potent immunomodulatory enzyme indoleamine 2,3-dioxygenase (IDO).  Both Nec-1 and Nec-1i inhibited human IDO, but Nec-1s did not, as predicted by mol. modeling.  Therefore, Nec-1s is a more specific RIPK1 inhibitor lacking the IDO-targeting effect.  Next, although Nec-1i was ∼100 × less effective than Nec-1 in inhibiting human RIPK1 kinase activity in vitro, it was only 10 times less potent than Nec-1 and Nec-1s in a mouse necroptosis assay and became even equipotent at high concns.  Along the same line, in vivo, high doses of Nec-1, Nec-1i and Nec-1s prevented tumor necrosis factor (TNF)-induced mortality equally well, excluding the use of Nec-1i as an inactive control.  Paradoxically, low doses of Nec-1 or Nec-1i, but not Nec-1s, even sensitized mice to TNF-induced mortality.  Importantly, Nec-1s did not exhibit this low dose toxicity, stressing again the preferred use of Nec-1s in vivo.  Our findings have important implications for the interpretation of Nec-1-based data in exptl. disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2yIQ4UD-yb7Vg90H21EOLACvtfcHk0lhm33nHTVP_Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFSitg%253D%253D&md5=2d4f57016d88e474cbfe00dee540b8fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2012.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2012.176%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DN.%26aulast%3DDuprez%26aufirst%3DL.%26aulast%3DGrootjans%26aufirst%3DS.%26aulast%3DCauwels%26aufirst%3DA.%26aulast%3DNerinckx%26aufirst%3DW.%26aulast%3DDuHadaway%26aufirst%3DJ.%2BB.%26aulast%3DGoossens%26aufirst%3DV.%26aulast%3DRoelandt%26aufirst%3DR.%26aulast%3DVan%2BHauwermeiren%26aufirst%3DF.%26aulast%3DLibert%26aufirst%3DC.%26aulast%3DDeclercq%26aufirst%3DW.%26aulast%3DCallewaert%26aufirst%3DN.%26aulast%3DPrendergast%26aufirst%3DG.%2BC.%26aulast%3DDegterev%26aufirst%3DA.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DVandenabeele%26aufirst%3DP.%26atitle%3DNecrostatin-1%2520analogues%253A%2520critical%2520issues%2520on%2520the%2520specificity%252C%2520activity%2520and%2520in%2520vivo%2520use%2520in%2520experimental%2520disease%2520models%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2012%26volume%3D3%26spage%3De437%26doi%3D10.1038%2Fcddis.2012.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lhm33nHTVP_Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahler, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolte, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule RIP1 kinase inhibitors for the treatment of pathologies associated with necroptosis</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1021/ml400382p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400382p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1238-1243&author=P.+A.+Harrisauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=K.+M.+Kahlerauthor=R.+Lehrauthor=J.+D.+Lichauthor=R.+Nagillaauthor=R.+T.+Nolteauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=M.+C.+Schaefferauthor=A.+Smallwoodauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=Discovery+of+small+molecule+RIP1+kinase+inhibitors+for+the+treatment+of+pathologies+associated+with+necroptosis&doi=10.1021%2Fml400382p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule RIP1 Kinase Inhibitors for the Treatment of Pathologies Associated with Necroptosis</span></div><div class="casAuthors">Harris, Philip A.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Finger, Joshua N.; Hoffman, Sandra J.; Kahler, Kirsten M.; Lehr, Ruth; Lich, John D.; Nagilla, Rakesh; Nolte, Robert T.; Ouellette, Michael T.; Pao, Christina S.; Schaeffer, Michelle C.; Smallwood, Angela; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1238-1243</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Potent inhibitors of RIP1 kinase from three distinct series, 1-aminoisoquinolines, pyrrolo-[2,3-b]-pyridines, and furo-[2,3-d]-pyrimidines, all of the type II class recognizing a DLG-out inactive conformation, were identified from screening of our inhouse kinase focused sets.  An exemplar from the furo-[2,3-d]-pyrimidine series showed a dose proportional response in protection from hypothermia in a mouse model of TNFα induced lethal shock.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN6cNcMjIyjrVg90H21EOLACvtfcHk0lhm33nHTVP_Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWrtrnF&md5=818a831f7083f7f17767daf02992ba55</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml400382p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400382p%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKahler%26aufirst%3DK.%2BM.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DNolte%26aufirst%3DR.%2BT.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520RIP1%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520pathologies%2520associated%2520with%2520necroptosis%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1238%26epage%3D1243%26doi%3D10.1021%2Fml400382p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Najjar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suebsuwong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogusa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuny, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Structure guided design of potent and selective ponatinib-based hybrid inhibitors for RIPK1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1850</span>– <span class="NLM_lpage">1860</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.celrep.2015.02.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=25801024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1850-1860&author=M.+Najjarauthor=C.+Suebsuwongauthor=S.+S.+Rayauthor=R.+J.+Thapaauthor=J.+L.+Makiauthor=S.+Nogusaauthor=S.+Shahauthor=D.+Salehauthor=P.+J.+Goughauthor=J.+Bertinauthor=J.+Yuanauthor=S.+Balachandranauthor=G.+D.+Cunyauthor=A.+Degterev&title=Structure+guided+design+of+potent+and+selective+ponatinib-based+hybrid+inhibitors+for+RIPK1&doi=10.1016%2Fj.celrep.2015.02.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Guided Design of Potent and Selective Ponatinib-Based Hybrid Inhibitors for RIPK1</span></div><div class="casAuthors">Najjar, Malek; Suebsuwong, Chalada; Ray, Soumya S.; Thapa, Roshan J.; Maki, Jenny L.; Nogusa, Shoko; Shah, Saumil; Saleh, Danish; Gough, Peter J.; Bertin, John; Yuan, Junying; Balachandran, Siddharth; Cuny, Gregory D.; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1850-1860</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RIPK1 and RIPK3, two closely related RIPK family members, have emerged as important regulators of pathol. cell death and inflammation.  In the current work, we report that the Bcr-Abl inhibitor and anti-leukemia agent ponatinib is also a first-in-class dual inhibitor of RIPK1 and RIPK3.  Ponatinib potently inhibited multiple paradigms of RIPK1- and RIPK3-dependent cell death and inflammatory tumor necrosis factor alpha (TNF-α) gene transcription.  We further describe design strategies that utilize the ponatinib scaffold to develop two classes of inhibitors (CS and PN series), each with greatly improved selectivity for RIPK1.  In particular, we detail the development of PN10, a highly potent and selective "hybrid" RIPK1 inhibitor, capturing the best properties of two different allosteric RIPK1 inhibitors, ponatinib and necrostatin-1.  Finally, we show that RIPK1 inhibitors from both classes are powerful blockers of TNF-induced injury in vivo.  Altogether, these findings outline promising candidate mols. and design approaches for targeting RIPK1- and RIPK3-driven inflammatory pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUQJThkEIWBbVg90H21EOLACvtfcHk0liKRSpKKY8z6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktro%253D&md5=c13215958ddcca5a685a4d1e82a96560</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.052%26sid%3Dliteratum%253Aachs%26aulast%3DNajjar%26aufirst%3DM.%26aulast%3DSuebsuwong%26aufirst%3DC.%26aulast%3DRay%26aufirst%3DS.%2BS.%26aulast%3DThapa%26aufirst%3DR.%2BJ.%26aulast%3DMaki%26aufirst%3DJ.%2BL.%26aulast%3DNogusa%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DSaleh%26aufirst%3DD.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DBalachandran%26aufirst%3DS.%26aulast%3DCuny%26aufirst%3DG.%2BD.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DStructure%2520guided%2520design%2520of%2520potent%2520and%2520selective%2520ponatinib-based%2520hybrid%2520inhibitors%2520for%2520RIPK1%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1850%26epage%3D1860%26doi%3D10.1016%2Fj.celrep.2015.02.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broudy, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iuliucci, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-[2-(imidazo[1,2-<i>b</i>]pyridazin-3-yl)ethynyl]-4-methyl-<i>N</i>-{4-[(4-methylpiperazin-1-yl)-methyl]-3-(trifluoromethyl)phenyl}benzamide (AP24534), a potent, orally active pan-inhibitor of breakpoint cluster region-abelson (BCR-ABL) kinase including the T315I gatekeeper mutant</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4701</span>– <span class="NLM_lpage">4719</span>, <span class="refDoi"> DOI: 10.1021/jm100395q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100395q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4701-4719&author=W.+S.+Huangauthor=C.+A.+Metcalfauthor=R.+Sundaramoorthiauthor=Y.+Wangauthor=D.+Zouauthor=R.+M.+Thomasauthor=X.+Zhuauthor=L.+Caiauthor=D.+Wenauthor=S.+Liuauthor=J.+Romeroauthor=J.+Qiauthor=I.+Chenauthor=G.+Bandaauthor=S.+P.+Lentiniauthor=S.+Dasauthor=Q.+Xuauthor=J.+Keatsauthor=F.+Wangauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+T.+Snodgrassauthor=M.+I.+Broudyauthor=K.+Russianauthor=T.+Zhouauthor=L.+Commodoreauthor=N.+I.+Narasimhanauthor=Q.+K.+Mohemmadauthor=J.+Iuliucciauthor=V.+M.+Riveraauthor=D.+C.+Dalgarnoauthor=T.+K.+Sawyerauthor=T.+Clacksonauthor=W.+C.+Shakespeare&title=Discovery+of+3-%5B2-%28imidazo%5B1%2C2-b%5Dpyridazin-3-yl%29ethynyl%5D-4-methyl-N-%7B4-%5B%284-methylpiperazin-1-yl%29-methyl%5D-3-%28trifluoromethyl%29phenyl%7Dbenzamide+%28AP24534%29%2C+a+potent%2C+orally+active+pan-inhibitor+of+breakpoint+cluster+region-abelson+%28BCR-ABL%29+kinase+including+the+T315I+gatekeeper+mutant&doi=10.1021%2Fjm100395q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-[2-(Imidazo[1,2-b]pyridazin-3-yl)ethynyl]-4-methyl-N-[4-((4-methylpiperazin-1-yl)methyl)-3-(trifluoromethyl)phenyl]benzamide (AP24534), a Potent, Orally Active Pan-Inhibitor of Breakpoint Cluster Region-Abelson (BCR-ABL) Kinase Including the T315I Gatekeeper Mutant</span></div><div class="casAuthors">Huang, Wei-Sheng; Metcalf, Chester A.; Sundaramoorthi, Raji; Wang, Yihan; Zou, Dong; Thomas, R. Mathew; Zhu, Xiaotian; Cai, Lisi; Wen, David; Liu, Shuangying; Romero, Jan; Qi, Jiwei; Chen, Ingrid; Banda, Geetha; Lentini, Scott P.; Das, Sasmita; Xu, Qihong; Keats, Jeff; Wang, Frank; Wardwell, Scott; Ning, Yaoyu; Snodgrass, Joseph T.; Broudy, Marc I.; Russian, Karin; Zhou, Tianjun; Commodore, Lois; Narasimhan, Narayana I.; Mohemmad, Qurish K.; Iuliucci, John; Rivera, Victor M.; Dalgarno, David C.; Sawyer, Tomi K.; Clackson, Tim; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4701-4719</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of chronic myeloid leukemia (CML) with BCR-ABL kinase inhibitors, the T315I gatekeeper mutant has emerged as resistant to all currently approved agents.  This report describes the structure-guided design of a novel series of potent pan-inhibitors of BCR-ABL, including the T315I mutation.  A key structural feature is the carbon-carbon triple bond linker which skirts the increased bulk of Ile315 side chain.  Extensive SAR studies led to the discovery of development candidate benzamide I (AP24534), which inhibited the kinase activity of both native BCR-ABL and the T315I mutant with low nM IC50s, and potently inhibited proliferation of corresponding Ba/F3-derived cell lines.  Daily oral administration of I significantly prolonged survival of mice injected i.v. with BCR-ABLT315I expressing Ba/F3 cells.  These data, coupled with a favorable ADME profile, support the potential of I to be an effective treatment for CML, including patients refractory to all currently approved therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoci2zVFUazgLVg90H21EOLACvtfcHk0liKRSpKKY8z6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslels7s%253D&md5=0dcd7b4ff31164b1d9883e897b7058ff</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm100395q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100395q%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DBanda%26aufirst%3DG.%26aulast%3DLentini%26aufirst%3DS.%2BP.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSnodgrass%26aufirst%3DJ.%2BT.%26aulast%3DBroudy%26aufirst%3DM.%2BI.%26aulast%3DRussian%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DIuliucci%26aufirst%3DJ.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%25203-%255B2-%2528imidazo%255B1%252C2-b%255Dpyridazin-3-yl%2529ethynyl%255D-4-methyl-N-%257B4-%255B%25284-methylpiperazin-1-yl%2529-methyl%255D-3-%2528trifluoromethyl%2529phenyl%257Dbenzamide%2520%2528AP24534%2529%252C%2520a%2520potent%252C%2520orally%2520active%2520pan-inhibitor%2520of%2520breakpoint%2520cluster%2520region-abelson%2520%2528BCR-ABL%2529%2520kinase%2520including%2520the%2520T315I%2520gatekeeper%2520mutant%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4701%26epage%3D4719%26doi%3D10.1021%2Fjm100395q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded library screening identifies benzo[b][1,4]oxazepin-4-ones as highly potent and monoselective receptor interacting protein 1 kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2163</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01898</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01898" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2163-2178&author=P.+A.+Harrisauthor=B.+W.+Kingauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=J.+N.+Fingerauthor=L.+C.+Gradyauthor=S.+J.+Hoffmanauthor=J.+U.+Jeongauthor=J.+Kangauthor=V.+Kasparcovaauthor=A.+S.+Lakdawalaauthor=R.+Lehrauthor=D.+E.+McNultyauthor=R.+Nagillaauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+R.+Rendinaauthor=M.+C.+Schaefferauthor=J.+D.+Summerfieldauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=A.+Zhangauthor=D.+Zhangauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=DNA-encoded+library+screening+identifies+benzo%5Bb%5D%5B1%2C4%5Doxazepin-4-ones+as+highly+potent+and+monoselective+receptor+interacting+protein+1+kinase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span></div><div class="casAuthors">Harris, Philip A.; King, Bryan W.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Finger, Joshua N.; Grady, LaShadric C.; Hoffman, Sandra J.; Jeong, Jae U.; Kang, James; Kasparcova, Viera; Lakdawala, Ami S.; Lehr, Ruth; McNulty, Dean E.; Nagilla, Rakesh; Ouellette, Michael T.; Pao, Christina S.; Rendina, Alan R.; Schaeffer, Michelle C.; Summerfield, Jennifer D.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Zhang, Aming; Zhang, Daohua; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2163-2178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent discovery of the role of receptor interacting protein 1 (RIP1) kinase in tumor necrosis factor (TNF)-mediated inflammation has led to its emergence as a highly promising target for the treatment of multiple inflammatory diseases.  The authors screened RIP1 against GSK's DNA-encoded small-mol. libraries and identified a novel highly potent benzoxazepinone inhibitor series.  The authors demonstrate that this template possesses complete monokinase selectivity for RIP1 plus unique species selectivity for primate vs. nonprimate RIP1.  The authors elucidate the conformation of RIP1 bound to this benzoxazepinone inhibitor driving its high kinase selectivity and design specific mutations in murine RIP1 to restore potency to levels similar to primate RIP1.  This series differentiates itself from known RIP1 inhibitors in combining high potency and kinase selectivity with good pharmacokinetic profiles in rodents.  The favorable developability profile of this benzoxazepinone template, as exemplified by compd. 14 (GSK'481), makes it an excellent starting point for further optimization into a RIP1 clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XvpziWGWg7Vg90H21EOLACvtfcHk0liCgvnODB6CPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D&md5=d6325dcaed97b7c339d1830ca041298c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01898%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSummerfield%26aufirst%3DJ.%2BD.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDNA-encoded%2520library%2520screening%2520identifies%2520benzo%255Bb%255D%255B1%252C4%255Doxazepin-4-ones%2520as%2520highly%2520potent%2520and%2520monoselective%2520receptor%2520interacting%2520protein%25201%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2163%26epage%3D2178%26doi%3D10.1021%2Facs.jmedchem.5b01898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkamper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class receptor interacting protein 1 (RIP1) kinase specific clinical candidate (GSK2982772) for the treatment of inflammatory diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenkamperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+first-in-class+receptor+interacting+protein+1+%28RIP1%29+kinase+specific+clinical+candidate+%28GSK2982772%29+for+the+treatment+of+inflammatory+diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0lio7ER2coGdTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenkamper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520receptor%2520interacting%2520protein%25201%2520%2528RIP1%2529%2520kinase%2520specific%2520clinical%2520candidate%2520%2528GSK2982772%2529%2520for%2520the%2520treatment%2520of%2520inflammatory%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a highly potent, selective, and metabolically stable inhibitor of receptor-interacting protein 1 (RIP1) for the treatment of systemic inflammatory response syndrome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">972</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaitbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=972-986&author=Y.+Renauthor=Y.+Suauthor=L.+Sunauthor=S.+Heauthor=L.+Mengauthor=D.+Liaoauthor=X.+Liuauthor=Y.+Maauthor=C.+Liuauthor=S.+Liauthor=H.+Ruanauthor=X.+Leiauthor=X.+Wangauthor=Z.+Zhang&title=Discovery+of+a+highly+potent%2C+selective%2C+and+metabolically+stable+inhibitor+of+receptor-interacting+protein+1+%28RIP1%29+for+the+treatment+of+systemic+inflammatory+response+syndrome&doi=10.1021%2Facs.jmedchem.6b01196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Potent, Selective, and Metabolically Stable Inhibitor of Receptor-Interacting Protein 1 (RIP1) for the Treatment of Systemic Inflammatory Response Syndrome</span></div><div class="casAuthors">Ren, Yan; Su, Yaning; Sun, Liming; He, Sudan; Meng, Lingjun; Liao, Daohong; Liu, Xiao; Ma, Yongfen; Liu, Chunyan; Li, Sisi; Ruan, Hanying; Lei, Xiaoguang; Wang, Xiaodong; Zhang, Zhiyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">972-986</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Based on its essential role in driving inflammation and disease pathol., cell necrosis has gradually been verified as a promising therapeutic target for treating atherosclerosis, systemic inflammatory response syndrome (SIRS), and ischemia injury, among other diseases.  Most necrosis inhibitors targeting receptor-interacting protein 1 (RIP1) still require further optimization because of weak potency or poor metabolic stability.  The authors conducted a phenotypic screen and identified a micromolar hit with novel amide structure.  Medicinal chem. efforts yielded a highly-potent, selective, and metabolically stable drug candidate - compd. 56 (RIPA-56, N-benzyl-N-hydroxy-2,2-dimethylbutanamide).  Biochem. studies and mol. docking revealed that RIP1 is the direct target of this new series of type III kinase inhibitors.  In the SIRS mice disease model, 56 efficiently reduced tumor necrosis factor alpha (TNFα)-induced mortality and multi-organ damage.  Compared to known RIP1 inhibitors, 56 is potent in both human and murine cells, is much more stable in vivo, and is efficacious in animal model studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqos-N-fJJ4vbVg90H21EOLACvtfcHk0ljdOrt0qapvYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaitbfO&md5=6e7045a9646be5f9dc9559d90efe8aed</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01196%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DLiao%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRuan%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520metabolically%2520stable%2520inhibitor%2520of%2520receptor-interacting%2520protein%25201%2520%2528RIP1%2529%2520for%2520the%2520treatment%2520of%2520systemic%2520inflammatory%2520response%2520syndrome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D972%26epage%3D986%26doi%3D10.1021%2Facs.jmedchem.6b01196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of GSK′963: a structurally distinct, potent and selective inhibitor of RIP1 kinase</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">15009</span>, <span class="refDoi"> DOI: 10.1038/cddiscovery.2015.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcddiscovery.2015.9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=15009&author=S.+B.+Bergerauthor=P.+Harrisauthor=R.+Nagillaauthor=V.+Kasparcovaauthor=S.+Hoffmanauthor=B.+Swiftauthor=L.+Dareauthor=M.+Schaefferauthor=C.+Capriottiauthor=M.+Ouelletteauthor=B.+W.+Kingauthor=D.+Wisnoskiauthor=J.+Coxauthor=M.+Reillyauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=Characterization+of+GSK%E2%80%B2963%3A+a+structurally+distinct%2C+potent+and+selective+inhibitor+of+RIP1+kinase&doi=10.1038%2Fcddiscovery.2015.9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fcddiscovery.2015.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddiscovery.2015.9%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DHoffman%26aufirst%3DS.%26aulast%3DSwift%26aufirst%3DB.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DSchaeffer%26aufirst%3DM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DWisnoski%26aufirst%3DD.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DReilly%26aufirst%3DM.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DCharacterization%2520of%2520GSK%25E2%2580%25B2963%253A%2520a%2520structurally%2520distinct%252C%2520potent%2520and%2520selective%2520inhibitor%2520of%2520RIP1%2520kinase%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D15009%26doi%3D10.1038%2Fcddiscovery.2015.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span> <span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3714</span>– <span class="NLM_lpage">3717</span>, <span class="refDoi"> DOI: 10.1021/jm000942e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties&doi=10.1021%2Fjm000942e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0ljy_gAMxzwJmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3717%26doi%3D10.1021%2Fjm000942e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stromnes, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goverman, J. M.</span></span> <span> </span><span class="NLM_article-title">Passive induction of experimental allergic encephalomyelitis</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1952</span>– <span class="NLM_lpage">1960</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnprot.2006.284" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1952-1960&author=I.+M.+Stromnesauthor=J.+M.+Goverman&title=Passive+induction+of+experimental+allergic+encephalomyelitis&doi=10.1038%2Fnprot.2006.284"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.284%26sid%3Dliteratum%253Aachs%26aulast%3DStromnes%26aufirst%3DI.%2BM.%26aulast%3DGoverman%26aufirst%3DJ.%2BM.%26atitle%3DPassive%2520induction%2520of%2520experimental%2520allergic%2520encephalomyelitis%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1952%26epage%3D1960%26doi%3D10.1038%2Fnprot.2006.284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lggYS-eGGaClA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R.
A.</span></span> <span> </span><span class="NLM_article-title">The drug-target residence time model: a 10-year retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&author=R.%0AA.+Copeland&title=The+drug-target+residence+time+model%3A+a+10-year+retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0lggYS-eGGaClA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520drug-target%2520residence%2520time%2520model%253A%2520a%252010-year%2520retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1517/17460441003677725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1517%2F17460441003677725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=22823083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=305-310&author=R.+A.+Copeland&title=The+dynamics+of+drug-target+interactions%3A+drug-target+residence+time+and+its+impact+on+efficacy+and+safety&doi=10.1517%2F17460441003677725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-310</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The extent and duration of pharmacol. action is detd. by the lifetime of drug occupancy on a mol. target.  This lifetime is defined by dynamic processes that control the rates of drug assocn. and dissocn. from the target.  Recently, the term residence time has been coined to describe exptl. measurements that can be related to the lifetime of the binary drug-target complex, and this in turn to durable, pharmacodynamic activity.  The residence time concept and its impact on drug optimization are reviewed here.  Examples are provided that demonstrate how a long residence time can improve drug efficacy in vivo.  Addnl., optimization of drug-target residence time can help to mitigate off-target mediated toxicity, hence, improving drug safety and tolerability.  Recent applications of the residence time concept to both drug discovery and development are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyt3KrIYdRjbVg90H21EOLACvtfcHk0lg8PzgzgzRI7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D&md5=aca925e00abefddf6e92d83b470d96f1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1517%2F17460441003677725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003677725%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520drug-target%2520interactions%253A%2520drug-target%2520residence%2520time%2520and%2520its%2520impact%2520on%2520efficacy%2520and%2520safety%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D305%26epage%3D310%26doi%3D10.1517%2F17460441003677725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Evaluation of enzyme inhibitors in drug discovery. A guide for medicinal chemists and pharmacologists</span>. <i>Methods Biochem. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1002/9781118540398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16350889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BD2MnmtlyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1-469&author=R.+A.+Copeland&title=Evaluation+of+enzyme+inhibitors+in+drug+discovery.+A+guide+for+medicinal+chemists+and+pharmacologists&doi=10.1002%2F9781118540398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of enzyme inhibitors in drug discovery. A guide for medicinal chemists and pharmacologists</span></div><div class="casAuthors">Copeland Robert A</div><div class="citationInfo"><span class="NLM_cas:title">Methods of biochemical analysis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-265</span>
        ISSN:<span class="NLM_cas:issn">0076-6941</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR7_ubfiC0DK2q3xNe7a7QfW6udTcc2earE-PBge-Bu7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnmtlyjsA%253D%253D&md5=4bb6759b5155aa6c2b6a462432afda7f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DEvaluation%2520of%2520enzyme%2520inhibitors%2520in%2520drug%2520discovery.%2520A%2520guide%2520for%2520medicinal%2520chemists%2520and%2520pharmacologists%26jtitle%3DMethods%2520Biochem.%2520Anal.%26date%3D2005%26volume%3D46%26spage%3D1%26epage%3D469%26doi%3D10.1002%2F9781118540398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Residence time of receptor-ligand complexes and its effect on biological function</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5481</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/bi8002023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8002023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5481-92&author=P.+J.+Tumminoauthor=R.+A.+Copeland&title=Residence+time+of+receptor-ligand+complexes+and+its+effect+on+biological+function&doi=10.1021%2Fbi8002023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Residence Time of Receptor-Ligand Complexes and Its Effect on Biological Function</span></div><div class="casAuthors">Tummino, Peter J.; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5481-5492</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation and duration of binary receptor-ligand complexes are fundamental to many physiol. processes.  Most often, the effectiveness of interaction between a receptor and its ligand is quantified in terms of closed system, equil. affinity measurements, such as IC50 and Kd.  In the context of in vivo biol., however, the extent and duration of responses to receptor-ligand interactions depend greatly on the time period over which the ligand is in residence on its receptor.  Here we define receptor-ligand complex residence time in quant. terms and describe its significance to biol. function.  Examples of the importance of residence time are presented for natural ligands of different receptor types.  The impact of residence time on the optimization of potential ligands as drugs for human medicine is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHshN2Bt_r9LVg90H21EOLACvtfcHk0liFfvcr9Wr7tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D&md5=57f79f69f12c6ed14e3347638381d943</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi8002023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8002023%26sid%3Dliteratum%253Aachs%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DResidence%2520time%2520of%2520receptor-ligand%2520complexes%2520and%2520its%2520effect%2520on%2520biological%2520function%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5481%26epage%3D92%26doi%3D10.1021%2Fbi8002023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, Y.</span></span> <span> </span><span class="NLM_article-title">Defining desirable central nervous system drug space through the alignment of molecular properties, in vitro ADME, and safety attributes</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1021/cn100007x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100007x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=420-434&author=T.+T.+Wagerauthor=R.+Y.+Chandrasekaranauthor=X.+Houauthor=M.+D.+Troutmanauthor=P.+R.+Verhoestauthor=A.+Villalobosauthor=Y.+Will&title=Defining+desirable+central+nervous+system+drug+space+through+the+alignment+of+molecular+properties%2C+in+vitro+ADME%2C+and+safety+attributes&doi=10.1021%2Fcn100007x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Defining Desirable Central Nervous System Drug Space through the Alignment of Molecular Properties, in Vitro ADME, and Safety Attributes</span></div><div class="casAuthors">Wager, Travis T.; Chandrasekaran, Ramalakshmi Y.; Hou, Xinjun; Troutman, Matthew D.; Verhoest, Patrick R.; Villalobos, Anabella; Will, Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">420-434</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of our effort to increase survival of drug candidates and to move our medicinal chem. design to higher probability space for success in the Neuroscience therapeutic area, we embarked on a detailed study of the property space for a collection of central nervous system (CNS) mols.  We carried out a thorough anal. of properties for 119 marketed CNS drugs and a set of 108 Pfizer CNS candidates.  In particular, we focused on understanding the relationships between physicochem. properties, in vitro ADME (absorption, distribution, metab., and elimination) attributes, primary pharmacol. binding efficiencies, and in vitro safety data for these two sets of compds.  This scholarship provides guidance for the design of CNS mols. in a property space with increased probability of success and may lead to the identification of druglike candidates with favorable safety profiles that can successfully test hypotheses in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEMBo2f34BrVg90H21EOLACvtfcHk0liFfvcr9Wr7tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVaqtbk%253D&md5=85716be86290fd85e93c6b8d5621683c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcn100007x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100007x%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DChandrasekaran%26aufirst%3DR.%2BY.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DWill%26aufirst%3DY.%26atitle%3DDefining%2520desirable%2520central%2520nervous%2520system%2520drug%2520space%2520through%2520the%2520alignment%2520of%2520molecular%2520properties%252C%2520in%2520vitro%2520ADME%252C%2520and%2520safety%2520attributes%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D420%26epage%3D434%26doi%3D10.1021%2Fcn100007x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootenhuis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayada, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delbressine, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploemen, J. P.</span></span> <span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10554091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&author=J.+Kelderauthor=P.+D.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+Delbressineauthor=J.+P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span></div><div class="casAuthors">Kelder, Jan; Grootenhuis, Peter D. J.; Bayada, Denis M.; Delbressine, Leon P. C.; Ploemen, Jan-Peter</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1514-1519</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose: To study oral absorption and brain penetration as a function of polar mol. surface area.  Methods: Measured brain penetration data of 45 drug mols. were investigated.  The dynamic polar surface areas were calcd. and correlated with the brain penetration data.  Also the static polar surface areas of 776 orally administered CNS drugs that have reached at least Phase II efficacy studies were calcd.  The same was done for a series of 1590 orally administered non-CNS drugs that have reached at least Phase II efficacy studies.  Results: A linear relationship between brain penetration and dynamic polar surface area (A2) was found (n = 45, R = 0.917, F1,43 = 229).  Brain penetration decreases with increasing polar surface area.  A clear difference between the distribution of the polar surface area of the 776 CNS and 1590 non-CNS drugs was found.  It was deduced that orally active drugs that are transported passively by the transcellular route should not exceed a polar surface area of about 120 A2.  They can be tailored to brain penetration by decreasing the polar surface to <60-70 A2.  This conclusion is supported by the inverse linear relationship between exptl. brain penetration data and the dynamic polar surface area of 45 drug mols.  Conclusions: The polar mol. surface area is a dominating determinant for oral absorption and brain penetration of drugs that are transported by the transcellular route.  This property should be considered in the early phase of drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmc3v7TzWM4LVg90H21EOLACvtfcHk0liFfvcr9Wr7tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D&md5=868fe1e9491dd3b8b92b326d28ee48f9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%26aulast%3DPloemen%26aufirst%3DJ.%2BP.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pajouhesh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G. R.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemical properties of successful central nervous system drugs</span>. <i>NeuroRx</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1602/neurorx.2.4.541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1602%2Fneurorx.2.4.541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16489364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FpvVKqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=541-553&author=H.+Pajouheshauthor=G.+R.+Lenz&title=Medicinal+chemical+properties+of+successful+central+nervous+system+drugs&doi=10.1602%2Fneurorx.2.4.541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemical properties of successful central nervous system drugs</span></div><div class="casAuthors">Pajouhesh Hassan; Lenz George R</div><div class="citationInfo"><span class="NLM_cas:title">NeuroRx : the journal of the American Society for Experimental NeuroTherapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">541-53</span>
        ISSN:<span class="NLM_cas:issn">1545-5343</span>.
    </div><div class="casAbstract">Fundamental physiochemical features of CNS drugs are related to their ability to penetrate the blood-brain barrier affinity and exhibit CNS activity.  Factors relevant to the success of CNS drugs are reviewed.  CNS drugs show values of molecular weight, lipophilicity, and hydrogen bond donor and acceptor that in general have a smaller range than general therapeutics.  Pharmacokinetic properties can be manipulated by the medicinal chemist to a significant extent.  The solubility, permeability, metabolic stability, protein binding, and human ether-ago-go-related gene inhibition of CNS compounds need to be optimized simultaneously with potency, selectivity, and other biological parameters.  The balance between optimizing the physiochemical and pharmacokinetic properties to make the best compromises in properties is critical for designing new drugs likely to penetrate the blood brain barrier and affect relevant biological systems.  This review is intended as a guide to designing CNS therapeutic agents with better drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_ncsy3yuEinl8icjszYl0fW6udTcc2ebyCCZEOEfTlLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FpvVKqsg%253D%253D&md5=0815084e2d107b9243b785d0782dfc3a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1602%2Fneurorx.2.4.541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1602%252Fneurorx.2.4.541%26sid%3Dliteratum%253Aachs%26aulast%3DPajouhesh%26aufirst%3DH.%26aulast%3DLenz%26aufirst%3DG.%2BR.%26atitle%3DMedicinal%2520chemical%2520properties%2520of%2520successful%2520central%2520nervous%2520system%2520drugs%26jtitle%3DNeuroRx%26date%3D2005%26volume%3D2%26spage%3D541%26epage%3D553%26doi%3D10.1602%2Fneurorx.2.4.541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-549&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0liQskLdxEg-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D549%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirimulla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vegesna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, M.</span></span> <span> </span><span class="NLM_article-title">Halogen interactions in protein-ligand complexes: implications of halogen bonding for rational drug design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1021/ci400257k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400257k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cisr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2781-2791&author=S.+Sirimullaauthor=J.+B.+Baileyauthor=R.+Vegesnaauthor=M.+Narayan&title=Halogen+interactions+in+protein-ligand+complexes%3A+implications+of+halogen+bonding+for+rational+drug+design&doi=10.1021%2Fci400257k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Interactions in Protein-Ligand Complexes: Implications of Halogen Bonding for Rational Drug Design</span></div><div class="casAuthors">Sirimulla, Suman; Bailey, Jake B.; Vegesna, Rahulsimham; Narayan, Mahesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2791</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Halogen bonding interactions between halogenated ligands and proteins were examd. using the crystal structures deposited to date in the PDB.  The data was analyzed as a function of halogen bonding to main chain Lewis bases, viz. oxygen of backbone carbonyl and backbone amide nitrogen.  This anal. also examd. halogen bonding to side-chain Lewis bases (O, N, and S) and to the electron-rich arom. amino acids.  All interactions were restricted to van der Waals radii with resp. atoms.  The data reveals that while fluorine and chlorine have strong tendencies favoring interactions with the backbone Lewis bases at glycine, the trend is not restricted to the achiral amino acid backbone for larger halogens.  Halogen side-chain interactions are not restricted to amino acids contg. O, N, and S as Lewis bases.  Electron-rich arom. amino acids host a high frequency of halogen bonds as does Leu.  A closer examn. of the latter hydrophobic side chain reveals that the "propensity of interactions" of halogen ligands at this oily residue is an outcome of strong classical halogen bonds with Lewis bases in the vicinity.  Finally, an examn. of Θ1 (C-X···O and C-X···N) and Θ2 (X···O-Z and X···N-Z) angles reveals that very few ligands adopt classical halogen bonding angles, suggesting that steric and other factors may influence these angles.  The data is discussed in the context of ligand design for pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wcnZDXmkYrVg90H21EOLACvtfcHk0lglLAKMzEFwVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cisr3O&md5=a73378681587e3ac13143452ecb78882</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci400257k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400257k%26sid%3Dliteratum%253Aachs%26aulast%3DSirimulla%26aufirst%3DS.%26aulast%3DBailey%26aufirst%3DJ.%2BB.%26aulast%3DVegesna%26aufirst%3DR.%26aulast%3DNarayan%26aufirst%3DM.%26atitle%3DHalogen%2520interactions%2520in%2520protein-ligand%2520complexes%253A%2520implications%2520of%2520halogen%2520bonding%2520for%2520rational%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2781%26epage%3D2791%26doi%3D10.1021%2Fci400257k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lglLAKMzEFwVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-alpha</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1011</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.cell.2009.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=19524512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1eiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2009&pages=1100-1011&author=S.+Heauthor=L.+Wangauthor=L.+Miaoauthor=T.+Wangauthor=F.+Duauthor=L.+Zhaoauthor=X.+Wang&title=Receptor+interacting+protein+kinase-3+determines+cellular+necrotic+response+to+TNF-alpha&doi=10.1016%2Fj.cell.2009.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-α</span></div><div class="casAuthors">He, Sudan; Wang, Lai; Miao, Lin; Wang, Tao; Du, Fenghe; Zhao, Liping; Wang, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1100-1111</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Smac mimetics induce apoptosis synergistically with TNF-α by triggering the formation of a caspase-8-activating complex contg. receptor interacting protein kinase-1 (RIPK1).  Caspase inhibitors block this form of apoptosis in many types of cells.  However, in several other cell lines, caspase inhibitors switch the apoptotic response to necrosis.  A genome wide siRNA screen revealed another member of the RIP kinase family, RIP3, to be required for necrosis.  The expression of RIP3 in different cell lines correlates with their responsiveness to necrosis induction.  The kinase activity of RIP3 is essential for necrosis execution.  Upon induction of necrosis, RIP3 is recruited to RIPK1 to form a necrosis-inducing complex.  Embryonic fibroblasts from RIP3 knockout mice are resistant to necrosis and RIP3 knockout animals are devoid of inflammation inflicted tissue damage in an acute pancreatitis model.  These data indicate RIP3 as the determinant for cellular necrosis in response to TNF-α family of death-inducing cytokines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoymGY6l3d1LbVg90H21EOLACvtfcHk0li_zg_ee4jxIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1eiu70%253D&md5=6397b34b896f8fe36c726f6d2ab39933</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMiao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DReceptor%2520interacting%2520protein%2520kinase-3%2520determines%2520cellular%2520necrotic%2520response%2520to%2520TNF-alpha%26jtitle%3DCell%26date%3D2009%26volume%3D137%26spage%3D1100%26epage%3D1011%26doi%3D10.1016%2Fj.cell.2009.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani-Yoshihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span> <span> </span><span class="NLM_article-title">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13000</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12513-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fs41598-017-12513-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=29026104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=13000&author=T.+Ishiiauthor=T.+Okaiauthor=M.+Iwatani-Yoshiharaauthor=M.+Mochizukiauthor=S.+Unnoauthor=M.+Kunoauthor=M.+Yoshikawaauthor=S.+Shibataauthor=M.+Nakakariyaauthor=T.+Yogoauthor=T.+Kawamoto&title=CETSA+quantitatively+verifies+in+vivo+target+engagement+of+novel+RIPK1+inhibitors+in+various+biospecimens&doi=10.1038%2Fs41598-017-12513-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CETSA quantitatively verifies in vivo target engagement of novel RIPK1 inhibitors in various biospecimens</span></div><div class="casAuthors">Ishii Tsuyoshi; Iwatani-Yoshihara Misa; Mochizuki Manabu; Kawamoto Tomohiro; Okai Takuro; Unno Satoko; Kuno Masako; Yoshikawa Masato; Yogo Takatoshi; Shibata Sachio; Nakakariya Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13000</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The proof of target engagement (TE) is a key element for evaluating potential investment in drug development.  The cellular thermal shift assay (CETSA) is expected to facilitate direct measurement of intracellular TE at all stages of drug development.  However, there have been no reports of applying this technology to comprehensive animal and clinical studies.  This report demonstrates that CETSA can not only quantitatively evaluate the drug-TE in mouse peripheral blood, but also confirm TE in animal tissues exemplified by using the receptor interacting protein 1 kinase (RIPK1) lead compound we have developed.  Our established semi-automated system allows evaluation of the structure-activity relationship using native RIPK1 in culture cell lines, and also enables estimation of drug occupancy ratio in mouse peripheral blood mononuclear cells.  Moreover, optimized tissue homogenisation enables monitoring of the in vivo drug-TE in spleen and brain.  Our results indicate that CETSA methodology will provide an efficient tool for preclinical and clinical drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVxZ2bIR2iP3O4gDb7JggCfW6udTcc2eaIbjcPYiJrKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gtFSmsw%253D%253D&md5=ae576418ac1d2ab3788ea241f74f59de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12513-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12513-1%26sid%3Dliteratum%253Aachs%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DOkai%26aufirst%3DT.%26aulast%3DIwatani-Yoshihara%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DUnno%26aufirst%3DS.%26aulast%3DKuno%26aufirst%3DM.%26aulast%3DYoshikawa%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DYogo%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26atitle%3DCETSA%2520quantitatively%2520verifies%2520in%2520vivo%2520target%2520engagement%2520of%2520novel%2520RIPK1%2520inhibitors%2520in%2520various%2520biospecimens%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D13000%26doi%3D10.1038%2Fs41598-017-12513-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes-Earle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mocarski, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">RIP1 kinase activity is dispensable for normal development but is a key regulator of inflammation in SHARPIN-deficient mice</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">5476</span>– <span class="NLM_lpage">5480</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1400499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4049%2Fjimmunol.1400499" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2014&pages=5476-5480&author=S.+B.+Bergerauthor=V.+Kasparcovaauthor=S.+Hoffmanauthor=B.+Swiftauthor=L.+Dareauthor=M.+Schaefferauthor=C.+Capriottiauthor=M.+Cookauthor=J.+Fingerauthor=A.+Hughes-Earleauthor=P.+A.+Harrisauthor=W.+J.+Kaiserauthor=E.+S.+Mocarskiauthor=J.+Bertinauthor=P.+J.+Gough&title=RIP1+kinase+activity+is+dispensable+for+normal+development+but+is+a+key+regulator+of+inflammation+in+SHARPIN-deficient+mice&doi=10.4049%2Fjimmunol.1400499"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400499%26sid%3Dliteratum%253Aachs%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DHoffman%26aufirst%3DS.%26aulast%3DSwift%26aufirst%3DB.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DSchaeffer%26aufirst%3DM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DM.%26aulast%3DFinger%26aufirst%3DJ.%26aulast%3DHughes-Earle%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKaiser%26aufirst%3DW.%2BJ.%26aulast%3DMocarski%26aufirst%3DE.%2BS.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DRIP1%2520kinase%2520activity%2520is%2520dispensable%2520for%2520normal%2520development%2520but%2520is%2520a%2520key%2520regulator%2520of%2520inflammation%2520in%2520SHARPIN-deficient%2520mice%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D192%26spage%3D5476%26epage%3D5480%26doi%3D10.4049%2Fjimmunol.1400499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polykratis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermance, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roderick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasparakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span> <span> </span><span class="NLM_article-title">RIPK1 Kinase inactive mice are viable and protected from TNF-induced necroptosis in vivo</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">1539</span>– <span class="NLM_lpage">1543</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1400590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.4049%2Fjimmunol.1400590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2014&pages=1539-1543&author=A.+Polykratisauthor=N.+Hermanceauthor=M.+Zelicauthor=J.+Roderickauthor=C.+Kimauthor=T.+M.+Vanauthor=T.+H.+Leeauthor=F.+K.+M.+Chanauthor=M.+Pasparakisauthor=M.+A.+Kelliher&title=RIPK1+Kinase+inactive+mice+are+viable+and+protected+from+TNF-induced+necroptosis+in+vivo&doi=10.4049%2Fjimmunol.1400590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400590%26sid%3Dliteratum%253Aachs%26aulast%3DPolykratis%26aufirst%3DA.%26aulast%3DHermance%26aufirst%3DN.%26aulast%3DZelic%26aufirst%3DM.%26aulast%3DRoderick%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DC.%26aulast%3DVan%26aufirst%3DT.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BH.%26aulast%3DChan%26aufirst%3DF.%2BK.%2BM.%26aulast%3DPasparakis%26aufirst%3DM.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26atitle%3DRIPK1%2520Kinase%2520inactive%2520mice%2520are%2520viable%2520and%2520protected%2520from%2520TNF-induced%2520necroptosis%2520in%2520vivo%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D193%26spage%3D1539%26epage%3D1543%26doi%3D10.4049%2Fjimmunol.1400590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarato, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cripps, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1038/cdd.2015.70</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1038%2Fcdd.2015.70" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=26024392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSiurjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=76-88&author=D.+A.+Rodriguezauthor=R.+Weinlichauthor=S.+Brownauthor=C.+Guyauthor=P.+Fitzgeraldauthor=C.+P.+Dillonauthor=A.+Oberstauthor=G.+Quaratoauthor=J.+Lowauthor=J.+G.+Crippsauthor=T.+Chenauthor=D.+R.+Green&title=Characterization+of+RIPK3-mediated+phosphorylation+of+the+activation+loop+of+MLKL+during+necroptosis&doi=10.1038%2Fcdd.2015.70"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis</span></div><div class="casAuthors">Rodriguez, D. A.; Weinlich, R.; Brown, S.; Guy, C.; Fitzgerald, P.; Dillon, C. P.; Oberst, A.; Quarato, G.; Low, J.; Cripps, J. G.; Chen, T.; Green, D. R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-88</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mixed lineage kinase domain-like pseudokinase (MLKL) mediates necroptosis by translocating to the plasma membrane and inducing its rupture.  The activation of MLKL occurs in a multimol. complex (the 'necrosome'), which is comprised of MLKL, receptor-interacting serine/threonine kinase (RIPK)-3 (RIPK3) and, in some cases, RIPK1.  Within this complex, RIPK3 phosphorylates the activation loop of MLKL, promoting conformational changes and allowing the formation of MLKL oligomers, which migrate to the plasma membrane.  Previous studies suggested that RIPK3 could phosphorylate the murine MLKL activation loop at Ser345, Ser347 and Thr349.  Moreover, substitution of the Ser345 for an aspartic acid creates a constitutively active MLKL, independent of RIPK3 function.  Here we examine the role of each of these residues and found that the phosphorylation of Ser345 is crit. for RIPK3-mediated necroptosis, Ser347 has a minor accessory role and Thr349 seems to be irrelevant.  We generated a specific monoclonal antibody to detect phospho-Ser345 in murine cells.  Using this antibody, a series of MLKL mutants and a novel RIPK3 inhibitor, we demonstrate that the phosphorylation of Ser345 is not required for the interaction between RIPK3 and MLKL in the necrosome, but is essential for MLKL translocation, accumulation in the plasma membrane, and consequent necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjrS_O0spUN7Vg90H21EOLACvtfcHk0lgePOEgiOzWuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSiurjF&md5=85f1eb0ff4ddaf746855d456f6ee6f6e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2015.70&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2015.70%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DD.%2BA.%26aulast%3DWeinlich%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DGuy%26aufirst%3DC.%26aulast%3DFitzgerald%26aufirst%3DP.%26aulast%3DDillon%26aufirst%3DC.%2BP.%26aulast%3DOberst%26aufirst%3DA.%26aulast%3DQuarato%26aufirst%3DG.%26aulast%3DLow%26aufirst%3DJ.%26aulast%3DCripps%26aufirst%3DJ.%2BG.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26atitle%3DCharacterization%2520of%2520RIPK3-mediated%2520phosphorylation%2520of%2520the%2520activation%2520loop%2520of%2520MLKL%2520during%2520necroptosis%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2016%26volume%3D23%26spage%3D76%26epage%3D88%26doi%3D10.1038%2Fcdd.2015.70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shisler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felices, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orenstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenardo, M. J.</span></span> <span> </span><span class="NLM_article-title">A role for tumor necrosis factor receptor-2 and receptor-interacting protein in programmed necrosis and antiviral responses</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">51613</span>– <span class="NLM_lpage">51621</span>, <span class="refDoi"> DOI: 10.1074/jbc.M305633200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1074%2Fjbc.M305633200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=51613-51621&author=F.+K.+Chanauthor=J.+Shislerauthor=J.+G.+Bixbyauthor=M.+Felicesauthor=L.+Zhengauthor=M.+Appelauthor=J.+Orensteinauthor=B.+Mossauthor=M.+J.+Lenardo&title=A+role+for+tumor+necrosis+factor+receptor-2+and+receptor-interacting+protein+in+programmed+necrosis+and+antiviral+responses&doi=10.1074%2Fjbc.M305633200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M305633200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M305633200%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DF.%2BK.%26aulast%3DShisler%26aufirst%3DJ.%26aulast%3DBixby%26aufirst%3DJ.%2BG.%26aulast%3DFelices%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DAppel%26aufirst%3DM.%26aulast%3DOrenstein%26aufirst%3DJ.%26aulast%3DMoss%26aufirst%3DB.%26aulast%3DLenardo%26aufirst%3DM.%2BJ.%26atitle%3DA%2520role%2520for%2520tumor%2520necrosis%2520factor%2520receptor-2%2520and%2520receptor-interacting%2520protein%2520in%2520programmed%2520necrosis%2520and%2520antiviral%2520responses%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D51613%26epage%3D51621%26doi%3D10.1074%2Fjbc.M305633200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-rate based optimization of structure-kinetic relationship - surfing the kinetic map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-rate+based+optimization+of+structure-kinetic+relationship+-+surfing+the+kinetic+map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-rate%2520based%2520optimization%2520of%2520structure-kinetic%2520relationship%2520-%2520surfing%2520the%2520kinetic%2520map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motulsky, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, L. C.</span></span> <span> </span><span class="NLM_article-title">The kinetics of competitive radioligand binding predicted by the law of mass action</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=6708928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A280%3ADyaL2c7mslOmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1984&pages=1-9&author=H.+J.+Motulskyauthor=L.+C.+Mahan&title=The+kinetics+of+competitive+radioligand+binding+predicted+by+the+law+of+mass+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of competitive radioligand binding predicted by the law of mass action</span></div><div class="casAuthors">Motulsky H J; Mahan L C</div><div class="citationInfo"><span class="NLM_cas:title">Molecular pharmacology</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Although equilibrium competitive radioligand binding studies are often used to characterize hormone and neurotransmitter receptors, the kinetics of such experiments have not been extensively explored.  The interactions of the radioligand and competitor with the receptors can be described by two differential equations which can be solved to yield a single equation describing the binding of the radioligand as a function of time.  This equation has several applications: First, it can be used to simulate competitive binding reactions under defined conditions.  Second, fitting experimental data to this equation allows one to determine the association and dissociation rate constants of the competing ligand, parameters that cannot be derived from equilibrium experiments.  Furthermore, this method can be used to determine the KI of the competing drug from data acquired before equilibrium is reached.  Third, mathematical analysis of the binding equation allowed us to answer two specific questions regarding the kinetics of competitive radioligand binding: how long such an incubation takes to equilibrate, and how the IC50 varies over time.  The answers to these questions depended, to a large extent, on the relative values of the dissociation rate constants of the radioligand and competitor, which can be determined as noted above.  When the competitor dissociates from the receptors more rapidly than the radioligand, the IC50 first decreases and then increases, but never has a value less than the KI.  At low radioligand concentrations, equilibrium is reached in the same amount of time required of the radioligand to dissociate completely from the receptors as determined in an "off-rate experiment." At higher concentrations of radioligand this time is halved.  When the competitor dissociates from the receptor more slowly than does the radioligand, then the time required to equilibrate depends only on the dissociation rate constant of the competitor, and the IC50 decreases over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCXDwvoK7MbN1QT_1O5WBYfW6udTcc2eZaGBolUkv-TLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL2c7mslOmtQ%253D%253D&md5=f0c09a16d7f8947fe0d72c256e2e4914</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMotulsky%26aufirst%3DH.%2BJ.%26aulast%3DMahan%26aufirst%3DL.%2BC.%26atitle%3DThe%2520kinetics%2520of%2520competitive%2520radioligand%2520binding%2520predicted%2520by%2520the%2520law%2520of%2520mass%2520action%26jtitle%3DMol.%2520Pharmacol.%26date%3D1984%26volume%3D25%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakakariya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, Y.</span></span> <span> </span><span class="NLM_article-title">In vivo biliary clearance should be predicted by intrinsic biliary clearance in sandwich-cultured hepatocytes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">602</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.042101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1124%2Fdmd.111.042101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=602-609&author=M.+Nakakariyaauthor=M.+Onoauthor=N.+Amanoauthor=T.+Moriwakiauthor=K.+Maedaauthor=Y.+Sugiyama&title=In+vivo+biliary+clearance+should+be+predicted+by+intrinsic+biliary+clearance+in+sandwich-cultured+hepatocytes&doi=10.1124%2Fdmd.111.042101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.042101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.042101%26sid%3Dliteratum%253Aachs%26aulast%3DNakakariya%26aufirst%3DM.%26aulast%3DOno%26aufirst%3DM.%26aulast%3DAmano%26aufirst%3DN.%26aulast%3DMoriwaki%26aufirst%3DT.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520biliary%2520clearance%2520should%2520be%2520predicted%2520by%2520intrinsic%2520biliary%2520clearance%2520in%2520sandwich-cultured%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D602%26epage%3D609%26doi%3D10.1124%2Fdmd.111.042101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirabayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriwaki, T.</span></span> <span> </span><span class="NLM_article-title">Quantitative investigation of the impact of P-glycoprotein inhibition on drug transport across blood-brain barrier in rats</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1124/dmd.110.035774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1124%2Fdmd.110.035774" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=8-14&author=H.+Sugimotoauthor=H.+Hirabayashiauthor=Y.+Kimuraauthor=A.+Furutaauthor=N.+Amanoauthor=T.+Moriwaki&title=Quantitative+investigation+of+the+impact+of+P-glycoprotein+inhibition+on+drug+transport+across+blood-brain+barrier+in+rats&doi=10.1124%2Fdmd.110.035774"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.035774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.035774%26sid%3Dliteratum%253Aachs%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DHirabayashi%26aufirst%3DH.%26aulast%3DKimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DA.%26aulast%3DAmano%26aufirst%3DN.%26aulast%3DMoriwaki%26aufirst%3DT.%26atitle%3DQuantitative%2520investigation%2520of%2520the%2520impact%2520of%2520P-glycoprotein%2520inhibition%2520on%2520drug%2520transport%2520across%2520blood-brain%2520barrier%2520in%2520rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D8%26epage%3D14%26doi%3D10.1124%2Fdmd.110.035774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebihara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, S.</span></span> <span> </span><span class="NLM_article-title">Establishment and characterization of the transformants stably-expressing MDR1 derived from various Animal species in LLC-PK1</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1007/s11095-006-0285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=10.1007%2Fs11095-006-0285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=16779700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=1460-1472&author=T.+Takeuchiauthor=S.+Yoshitomiauthor=T.+Higuchiauthor=K.+Ikemotoauthor=S.-I.+Niwaauthor=T.+Ebiharaauthor=M.+Katohauthor=T.+Yokoiauthor=S.+Asahi&title=Establishment+and+characterization+of+the+transformants+stably-expressing+MDR1+derived+from+various+Animal+species+in+LLC-PK1&doi=10.1007%2Fs11095-006-0285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span></div><div class="casAuthors">Takeuchi, Toshiyuki; Yoshitomi, Sumie; Higuchi, Tomoaki; Ikemoto, Keiko; Niwa, Shin-Ichi; Ebihara, Takuya; Katoh, Miki; Yokoi, Tsuyoshi; Asahi, Satoru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Stable transformants expressing human multidrug resistance 1 (MDR1), monkey MDR1, canine MDR1, rat MDR1a, rat MDR1b, mouse mdr1a, and mouse mdr1b in LLC-PK1 were established to investigate species differences in P-glycoprotein (P-gp, ABCB1) mediated efflux activity.  Methods: The seven cDNAs of MDR1 from five animals were cloned, and their transformants stably expressing the series of MDR1 in LLC-PK1 were established.  Transport studies of clarithromycin, daunorubicin, digoxin, erythromycin, etoposide, paclitaxel, propranolol, quinidine, ritonavir, saquinavir, verapamil, and vinblastine were performed by using these cells, and efflux activity was compared among the species.  Results: Except for propranolol, all compds. showed efflux activity in all transformants, and were judged to be substrates of P-gp.  There were slight interspecies and interisoforms differences in the substrate recognition.  However, the efflux ratio among the series of the MDR1 stably expressing cells showed good correlation as represented between human and monkey MDR1, and poor correlation as represented between human MDR1 and mouse mdr1a, and human and canine MDR1.  Conclusions: Results in the present study indicate that all MDR1 stably expressing cells have efflux activity for various P-gp substrates, and that interspecies differences and similarities of the P-gp substrate efflux activity may exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28IPlZqwtHLVg90H21EOLACvtfcHk0ljOUihzv3zoIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D&md5=a21044c202d7ff2eb8ab27c4623909de</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-0285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-0285-7%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DIkemoto%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DS.-I.%26aulast%3DEbihara%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DT.%26aulast%3DAsahi%26aufirst%3DS.%26atitle%3DEstablishment%2520and%2520characterization%2520of%2520the%2520transformants%2520stably-expressing%2520MDR1%2520derived%2520from%2520various%2520Animal%2520species%2520in%2520LLC-PK1%26jtitle%3DPharm.%2520Res.%26date%3D2006%26volume%3D23%26spage%3D1460%26epage%3D1472%26doi%3D10.1007%2Fs11095-006-0285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TX5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TX5','PDB','5TX5'); return false;">PDB: 5TX5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C3E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C3E','PDB','6C3E'); return false;">PDB: 6C3E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C4D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C4D','PDB','6C4D'); return false;">PDB: 6C4D</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i156"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01647">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_01043"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01647">10.1021/acs.jmedchem.7b01647</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis of fluorescent probe <b>9</b>, the detailed kinetic parameters of compounds <b>16a</b>–<b>aj</b>; metabolic stabilities of compounds <b>16a</b> and <b>16d</b>; PK graph of <b>22</b>; experimental methods for cloning, expression, and purification of RIP1 kinase, as well as crystallization, data collection, and structure solution of RIP1 kinase; results of Reaction Biology Corporation kinase panel and Eurofins Panlabs panel (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_001.csv">jm7b01647_si_001.csv (6.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01647/suppl_file/jm7b01647_si_002.pdf">jm7b01647_si_002.pdf (368.26 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of the crystal structures of RIP1 kinase in complex with compounds <b>10</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C3E">6C3E</a>) and <b>22</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C4D">6C4D</a>) have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01647&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01647" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01647" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799cd2f0ac03c22","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
