
<html lang="en"     class="pb-page"  data-request-id="60a5913e-5b9f-49d2-9d6c-d71123591bcd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-2;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b01606;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Characterization of the Potent and Highly Selective (Piperidin-4-yl)pyrido[3,2-d]pyrimidine Based in Vitro Probe BAY-885 for the Kinase ERK5" /></meta><meta name="dc.Creator" content="Duy  Nguyen" /></meta><meta name="dc.Creator" content="Clara  Lemos" /></meta><meta name="dc.Creator" content="Lars  Wortmann" /></meta><meta name="dc.Creator" content="Knut  Eis" /></meta><meta name="dc.Creator" content="Simon J.  Holton" /></meta><meta name="dc.Creator" content="Ulf  Boemer" /></meta><meta name="dc.Creator" content="Dieter  Moosmayer" /></meta><meta name="dc.Creator" content="Uwe  Eberspaecher" /></meta><meta name="dc.Creator" content="Joerg  Weiske" /></meta><meta name="dc.Creator" content="Christian  Lechner" /></meta><meta name="dc.Creator" content="Stefan  Prechtl" /></meta><meta name="dc.Creator" content="Detlev  Suelzle" /></meta><meta name="dc.Creator" content="Franziska  Siegel" /></meta><meta name="dc.Creator" content="Florian  Prinz" /></meta><meta name="dc.Creator" content="Ralf  Lesche" /></meta><meta name="dc.Creator" content="Barbara  Nicke" /></meta><meta name="dc.Creator" content="Katrin  Nowak-Reppel" /></meta><meta name="dc.Creator" content="Herbert  Himmel" /></meta><meta name="dc.Creator" content="Dominik  Mumberg" /></meta><meta name="dc.Creator" content="Franz  von Nussbaum" /></meta><meta name="dc.Creator" content="Carl F.  Nising" /></meta><meta name="dc.Creator" content="Marcus  Bauser" /></meta><meta name="dc.Creator" content="Andrea  Haegebarth" /></meta><meta name="dc.Description" content="The availability of a chemical probe to study the role of a specific domain of a protein in a concentration- and time-dependent manner is of high value. Herein, we report the identification of a hi..." /></meta><meta name="Description" content="The availability of a chemical probe to study the role of a specific domain of a protein in a concentration- and time-dependent manner is of high value. Herein, we report the identification of a hi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 18, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01606" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01606" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01606" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01606" /></link>
        
    
    

<title>Discovery and Characterization of the Potent and Highly Selective (Piperidin-4-yl)pyrido[3,2-d]pyrimidine Based in Vitro Probe BAY-885 for the Kinase ERK5 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01606" /></meta><meta property="og:title" content="Discovery and Characterization of the Potent and Highly Selective (Piperidin-4-yl)pyrido[3,2-d]pyrimidine Based in Vitro Probe BAY-885 for the Kinase ERK5" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0019.jpeg" /></meta><meta property="og:description" content="The availability of a chemical probe to study the role of a specific domain of a protein in a concentration- and time-dependent manner is of high value. Herein, we report the identification of a highly potent and selective ERK5 inhibitor BAY-885 by high-throughput screening and subsequent structure-based optimization. ERK5 is a key integrator of cellular signal transduction, and it has been shown to play a role in various cellular processes such as proliferation, differentiation, apoptosis, and cell survival. We could demonstrate that inhibition of ERK5 kinase and transcriptional activity with a small molecule did not translate into antiproliferative activity in different relevant cell models, which is in contrast to the results obtained by RNAi technology." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01606"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01606">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01606&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01606&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01606&amp;href=/doi/10.1021/acs.jmedchem.8b01606" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 928-940</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01621" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01631" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Characterization of the Potent and Highly Selective (Piperidin-4-yl)pyrido[3,2-<i>d</i>]pyrimidine Based <i>in Vitro</i> Probe <b>BAY-885</b> for the Kinase ERK5</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Duy Nguyen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duy Nguyen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#aacedfd384c4cddfd3cfc4eac8cbd3cfd884c9c5c7"><span class="__cf_email__" data-cfemail="ceaabbb7e0a0a9bbb7aba08eacafb7abbce0ada1a3">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duy++Nguyen">Duy Nguyen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4534-745X" title="Orcid link">http://orcid.org/0000-0002-4534-745X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Clara Lemos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Clara Lemos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Clara++Lemos">Clara Lemos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lars Wortmann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lars Wortmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#402c2132336e372f32342d212e2e0022213925326e232f2d"><span class="__cf_email__" data-cfemail="3c505d4e4f124b534e48515d52527c5e5d45594e125f5351">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lars++Wortmann">Lars Wortmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6514-947X" title="Orcid link">http://orcid.org/0000-0001-6514-947X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Knut Eis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Knut Eis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Knut++Eis">Knut Eis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon J. Holton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon J. Holton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon+J.++Holton">Simon J. Holton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulf Boemer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulf Boemer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulf++Boemer">Ulf Boemer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dieter Moosmayer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dieter Moosmayer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dieter++Moosmayer">Dieter Moosmayer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Uwe Eberspaecher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Uwe Eberspaecher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Uwe++Eberspaecher">Uwe Eberspaecher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joerg Weiske</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joerg Weiske</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joerg++Weiske">Joerg Weiske</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Lechner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Lechner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Lechner">Christian Lechner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Prechtl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Prechtl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Prechtl">Stefan Prechtl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Detlev Suelzle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Detlev Suelzle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Detlev++Suelzle">Detlev Suelzle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Franziska Siegel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Franziska Siegel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Franziska++Siegel">Franziska Siegel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Florian Prinz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Florian Prinz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Florian++Prinz">Florian Prinz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ralf Lesche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ralf Lesche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ralf++Lesche">Ralf Lesche</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Barbara Nicke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Barbara Nicke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Barbara++Nicke">Barbara Nicke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katrin Nowak-Reppel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katrin Nowak-Reppel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katrin++Nowak-Reppel">Katrin Nowak-Reppel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Herbert Himmel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Herbert Himmel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Herbert++Himmel">Herbert Himmel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominik Mumberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominik Mumberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominik++Mumberg">Dominik Mumberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Franz von Nussbaum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Franz von Nussbaum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Franz++von+Nussbaum">Franz von Nussbaum</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carl F. Nising</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carl F. Nising</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carl+F.++Nising">Carl F. Nising</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcus Bauser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcus Bauser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcus++Bauser">Marcus Bauser</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Andrea Haegebarth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Haegebarth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research & Development, Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Haegebarth">Andrea Haegebarth</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01606&amp;href=/doi/10.1021%2Facs.jmedchem.8b01606" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 928–940</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 18, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 October 2018</li><li><span class="item_label"><b>Published</b> online</span>18 December 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 January 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01606" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01606</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D928%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDuy%2BNguyen%252C%2BClara%2BLemos%252C%2BLars%2BWortmann%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D2%26contentID%3Dacs.jmedchem.8b01606%26title%3DDiscovery%2Band%2BCharacterization%2Bof%2Bthe%2BPotent%2Band%2BHighly%2BSelective%2B%2528Piperidin-4-yl%2529pyrido%255B3%252C2-d%255Dpyrimidine%2BBased%2Bin%2BVitro%2BProbe%2BBAY-885%2Bfor%2Bthe%2BKinase%2BERK5%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D940%26publicationDate%3DJanuary%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01606"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3300</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">17</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01606" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Characterization of the Potent and Highly Selective (Piperidin-4-yl)pyrido[3,2-d]pyrimidine Based in Vitro Probe BAY-885 for the Kinase ERK5&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Duy&quot;,&quot;last_name&quot;:&quot;Nguyen&quot;},{&quot;first_name&quot;:&quot;Clara&quot;,&quot;last_name&quot;:&quot;Lemos&quot;},{&quot;first_name&quot;:&quot;Lars&quot;,&quot;last_name&quot;:&quot;Wortmann&quot;},{&quot;first_name&quot;:&quot;Knut&quot;,&quot;last_name&quot;:&quot;Eis&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;J. Holton&quot;},{&quot;first_name&quot;:&quot;Ulf&quot;,&quot;last_name&quot;:&quot;Boemer&quot;},{&quot;first_name&quot;:&quot;Dieter&quot;,&quot;last_name&quot;:&quot;Moosmayer&quot;},{&quot;first_name&quot;:&quot;Uwe&quot;,&quot;last_name&quot;:&quot;Eberspaecher&quot;},{&quot;first_name&quot;:&quot;Joerg&quot;,&quot;last_name&quot;:&quot;Weiske&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Lechner&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Prechtl&quot;},{&quot;first_name&quot;:&quot;Detlev&quot;,&quot;last_name&quot;:&quot;Suelzle&quot;},{&quot;first_name&quot;:&quot;Franziska&quot;,&quot;last_name&quot;:&quot;Siegel&quot;},{&quot;first_name&quot;:&quot;Florian&quot;,&quot;last_name&quot;:&quot;Prinz&quot;},{&quot;first_name&quot;:&quot;Ralf&quot;,&quot;last_name&quot;:&quot;Lesche&quot;},{&quot;first_name&quot;:&quot;Barbara&quot;,&quot;last_name&quot;:&quot;Nicke&quot;},{&quot;first_name&quot;:&quot;Katrin&quot;,&quot;last_name&quot;:&quot;Nowak-Reppel&quot;},{&quot;first_name&quot;:&quot;Herbert&quot;,&quot;last_name&quot;:&quot;Himmel&quot;},{&quot;first_name&quot;:&quot;Dominik&quot;,&quot;last_name&quot;:&quot;Mumberg&quot;},{&quot;first_name&quot;:&quot;Franz&quot;,&quot;last_name&quot;:&quot;von Nussbaum&quot;},{&quot;first_name&quot;:&quot;Carl&quot;,&quot;last_name&quot;:&quot;F. Nising&quot;},{&quot;first_name&quot;:&quot;Marcus&quot;,&quot;last_name&quot;:&quot;Bauser&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Haegebarth&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;928-940&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01606&quot;},&quot;abstract&quot;:&quot;The availability of a chemical probe to study the role of a specific domain of a protein in a concentration- and time-dependent manner is of high value. Herein, we report the identification of a highly potent and selective ERK5 inhibitor BAY-885 by high-throughput screening and subsequent structure-based optimization. ERK5 is a key integrator of cellular signal transduction, and it has been shown to play a role in various cellular processes such as proliferation, differentiation, apoptosis, and cell survival. We could demonstrate that inhibition of ERK5 kinase and transcriptional activity with a small molecule did not translate into antiproliferative activity in different relevant cell models, which is in contrast to the results obtained by RNAi technology.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01606&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01606" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01606&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01606" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01606&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01606" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01606&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01606&amp;href=/doi/10.1021/acs.jmedchem.8b01606" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01606" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01606" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01606%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DNguyen%26date%3D2019%26atitle%3DDiscovery%2Band%2BCharacterization%2Bof%2Bthe%2BPotent%2Band%2BHighly%2BSelective%2B%2528Piperidin-4-yl%2529pyrido%255B3%252C2-d%255Dpyrimidine%2BBased%2Bin%2BVitro%2BProbe%2BBAY-885%2Bfor%2Bthe%2BKinase%2BERK5%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D2%26spage%3D928%26epage%3D940%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290758" title="Anatomy">Anatomy</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/jmcmar.2019.62.issue-2/20190124/jmcmar.2019.62.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0019.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The availability of a chemical probe to study the role of a specific domain of a protein in a concentration- and time-dependent manner is of high value. Herein, we report the identification of a highly potent and selective ERK5 inhibitor <b>BAY-885</b> by high-throughput screening and subsequent structure-based optimization. ERK5 is a key integrator of cellular signal transduction, and it has been shown to play a role in various cellular processes such as proliferation, differentiation, apoptosis, and cell survival. We could demonstrate that inhibition of ERK5 kinase and transcriptional activity with a small molecule did not translate into antiproliferative activity in different relevant cell models, which is in contrast to the results obtained by RNAi technology.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The extracellular signal-regulated kinase 5 (ERK5, also known as big MAP kinase 1, BMK1) protein, encoded by the MAPK7 gene, is a member of the mitogen-activated protein kinase (MAPK) family. The ERK5 signaling cascade can be activated by environmental stresses, mitogens, and cytokines. These stimuli activate MEKK2 and MEKK3, which are able to phosphorylate and activate MEK5. Once activated, MEK5 phosphorylates the TEY motif in the activation loop of the ERK5 kinase domain, thereby leading to ERK5 activation (for reviews, see refs <a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a>).</div><div class="NLM_p">ERK5 is a key integrator of cellular signal transduction, and it has been shown to play a role in various cellular processes such as proliferation, differentiation, apoptosis, and cell survival. Several studies have demonstrated that silencing ERK5 with siRNA or shRNA decreases the proliferation and increases cell death in different tumor models, thereby highlighting the potential of ERK5 as a therapeutic target in cancer.<a onclick="showRef(event, 'ref1 ref2 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref4">(1,2,4)</a> Of note, many cancer types (e.g., sarcoma and hepatocellular carcinoma) display genomic ERK5 amplifications, while others exhibit constitutive activation of ERK5 (e.g., breast cancer with ErbB2 overexpression), rendering them particularly sensitive to ERK5 depletion.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5−8)</a></div><div class="NLM_p">The therapeutic potential of targeting ERK5 resulted in different attempts to develop ERK5 kinase inhibitors over the recent years. The first of such compounds, <b>XMD8–92</b>, has been extensively used and showed promising <i>in vitro</i> and <i>in vivo</i> antitumor efficacy (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> However, a recent study demonstrated that the biological activity of <b>XMD8–92</b> derived from an off-target activity on bromodomains (BRDs).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The same authors developed new ERK5 inhibitors (e.g., <b>AX15836</b>) with improved potency and selectivity, but with suboptimal pharmacokinetic properties.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Therefore, we aimed at developing a new generation of ERK5 inhibitors with an overall improved profile, including high potency and an excellent kinase selectivity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published ERK5 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> High-Throughput Screen</h3><div class="NLM_p last">For the identification of a small-molecule inhibitor of ERK5 we screened the Bayer compound collection (approximately 3.4 million compounds) using a high-throughput TR-FRET (time-resolved fluorescence energy transfer)-based kinase inhibition assay: ERK5 was incubated with the test compounds, biotinylated substrate peptide and 250 μM ATP for 60 min. The amount of phosphorylated peptide was quantified with a phosphospecific detection antibody and generic TR-FRET detection reagents. Subsequent hit-to-lead process resulted in the identification of a quinazoline cluster, including compound <b>1</b> with an IC<sub>50</sub> value of 270 nM, as the most promising lead structure series, which was selected as a starting point for further optimization.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structure–Activity Relationships and Chemical Syntheses</h3><div class="NLM_p">We were able to obtain a cocrystal structure of <b>1</b> in complex with ERK5 protein unlocking the binding mode with three key interactions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Nitrogen N1 of the quinazoline interacts with the backbone NH of the hinge amino acid Leu<sup>139</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The amide carbonyl of <b>1</b> interacts via a hydrogen bond with Asp<sup>200</sup> and the hydrophobic back pocket is occupied by the <i>para</i>-(trifluoromethoxy)benzamide moiety. It is important to note that the central piperidine ring is crucial for the S-shaped geometry of the lead structure series allowing optimal fit into the binding pocket of ERK5 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The results of this cocrystal structure of <b>1</b> and ERK5 guided further structure optimization and helped to understand the structure–activity relationships (SAR).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of compound <b>1</b> with ERK5 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM">6HKM</a>, 2.46 Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>1</b> was synthesized as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>: Following a literature procedure, anthranilic acid <b>2</b> was converted to the corresponding 4-chloro-6,7-dimethoxyquinazoline.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Subsequent Suzuki reaction, cleavage of the Boc protecting group, and hydrogenation of the double bond furnished piperidine derivative <b>4</b>. Amide formation with 4-(trifluoromethoxy)benzoic acid gave rise to compound <b>1</b>. Several analogues described in this publication were synthesized via a similar reaction sequence. For further details, please see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0006.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>1</b> as Representative Synthesis for Quinazoline Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) formamidinium acetate, 2-methoxyethanol, 100 °C, overnight, 88%; (b) SOCl<sub>2</sub> in DMF, reflux, overnight, 88%; (c) <i>tert-</i>butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate, Na<sub>2</sub>CO<sub>3</sub>, 1,1′-bis(diphenylphosphino)-ferrocenepalladium(II) chloride, 1,4-dioxane/water (5:1), 100 °C, 4 h, 74%; (d) TFA, dichloromethane, rt, 2 h, 87%; (e) H<sub>2</sub>, Pd/C, methanol, rt, overnight, 80%; (f) 4-(trifluoromethoxy)benzoic acid, HATU, NEt<sub>3</sub>, DMF, rt, overnight, 18%.</p></p></figure><div class="NLM_p">We first investigated the role of the two methoxy groups of <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Interestingly, the 7-methoxy group contributes more to the potency than the 6-methoxy group (compare <b>5</b> and <b>6</b>). In accordance with the X-ray structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), nitrogen N1 of the quinazoline core is crucial for activity (see <b>8</b>), whereas removal of N3 does not compromise activity (compare <b>6</b> and <b>7</b>). A methyl scan on the quinazoline core confirms that substitution on positions 6 and 7 are tolerated (see <b>11</b> and <b>12</b>), but substitutions on positions 2 and 8 are detrimental for activity (see <b>9</b> and <b>10</b>). Introduction of further nitrogen in the quinazoline core, particularly such in case of pyridopyrimidine <b>15</b>, results in improvement of activity due to its ability to stabilize the conformation of the piperidine ring by an additional nonclassical hydrogen bond between N5 of the quinazoline and the CH<sub>2</sub> of the piperidine (structure/data not shown).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Quinazoline Core<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0008.gif" alt="" id="gr6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0009.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div><div></div></div><div class="NLM_p">Changes affecting the unique S-shaped conformation of the piperidine in compound <b>1</b> as seen in the cocrystal structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) are detrimental for potency. For example, replacement of piperidine atoms with an sp<sup>3</sup> hybridization by atoms with an sp<sup>2</sup> hybridization leads to a dramatic conformational change and results in significantly reduced activity (see <b>16</b> and <b>17</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Also a change of the exit vectors by moving to a seven-membered ring size as in <b>18</b> results in complete loss of activity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Piperidine Core<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0010.gif" alt="" id="gr8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0011.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div><div></div></div><div class="NLM_p">As predicted from the cocrystal structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), the amide carbonyl is crucial for activity (see <b>21</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). A scan of a trifluoromethoxy group demonstrated that the substituent boosts ERK5 potency in the para position, whereas the same substituent in the meta or ortho position is detrimental for activity (compare <b>1</b>, <b>19</b>, and <b>20</b>). A broad range of different para substituents was investigated, indicating that small lipophilic para substituents are beneficial for potency (see <b>22</b>, <b>23</b>, and <b>25</b>), whereas large or polar substituents result in significantly reduced ERK5 activity (see <b>27</b> and <b>29</b>–<b>32</b>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of Benzamide Moiety (Monosubstitution)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0012.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0013.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div><div></div></div><div class="NLM_p">Next we investigated whether further interaction with ERK5 protein can be obtained by introducing additional hydrogen bond donors or acceptors on the 4-(trifluoromethoxy)benzamide moiety (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Additional ring nitrogens were tolerated but lead to a decrease of ERK5 potency (see <b>33</b> and <b>34</b> as compared to compound <b>1</b>). Surprisingly an additional amino group at C2 was able to further improve ERK5 activity (see <b>35</b>). A cocrystal of compound <b>35</b> with ERK5 (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) indicates that the amino group is engaged in a hydrogen bond network with Lys<sup>84</sup>, Asp<sup>200</sup>, and Glu<sup>102</sup> thus leading to potency improvement as compared to compound <b>6</b>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of compound <b>35</b> with ERK5 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN">6HKN</a>, 2.33 Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of Benzamide Moiety (Double Substitution)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0014.gif" alt="" id="gr12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0015.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>6</b> and <b>15</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) indicated that further investigation of 7-substituted pyrido[3,2-<i>d</i>]pyrimidines could be beneficial to further boost ERK5 activity. As seen in the cocrystal structure of <b>35</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), the methoxy group in the 7-position points toward the exit to the solvent. Therefore, we expected to be able to tune physicochemical and pharmacokinetic properties of the lead series in this position. In particular, to improve water solubility of the compounds, we investigated a set of amines in the 7-position (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). All compounds depicted in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> show good ERK5 activity with <b>39</b> being the most potent and <b>40</b> being the most soluble (data not shown).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of Substituents in the 7-Position<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0016.gif" alt="" id="gr14" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0017.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div><div></div></div><div class="NLM_p">The combination of the best residues and properties lead to the identification of <b>41</b> (<b>BAY-885</b>): a potent and selective <i>in vitro</i> ERK5 probe molecule (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The synthesis of <b>41</b> is outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. 4,7-Dichloropyrido[3,2-<i>d</i>]pyrimidine <b>43</b><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> is converted to <b>44</b> via a Suzuki reaction followed by a Buchwald amination to provide <b>45</b>. Cleavage of the Boc protecting group followed by hydrogenation of the double bond and amide formation gives rise to <b>41</b> (<b>BAY-885</b>). As a control compound for pharmacological experiments we also provide the negative ERK5 probe <b>42</b> (<b>BAY-693</b>, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). A detailed synthesis description for <b>42</b> can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Summary of chemical, physicochemical, and <i>in vitro</i> DMPK properties of <b>41</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ERK5 Negative Control Molecule <b>42</b> (<b>BAY-693</b>)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0018.gif" alt="" id="gr16" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center">ERK5 IC<sub>50</sub> [μM]</th><th class="colsep0 rowsep0" align="center">SN12C-MEF2 Luciferase reporter gene assay [μM]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b> (<b>BAY-693</b>)</td><td class="colsep0 rowsep0" align="left">6.40</td><td class="colsep0 rowsep0" align="left">11.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are arithmetic means of multiple measurements.</p></div></div></div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0007.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of ERK5 Probe Molecule <b>41</b> (<b>BAY-885</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>-butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate, Na<sub>2</sub>CO<sub>3</sub>, 1,1′-bis(diphenylphosphino)ferrocene-palladium(II) chloride, 1,4-dioxane/water (5:1), 100 °C, 4 h, 37%; (b) 1-methylpiperazine, Pd(OAc)<sub>2</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 100 °C, 14 h; (c) H<sub>2</sub>, Pd/C, methanol, rt, overnight, 20%; (d) TFA, dichloromethane, rt, 2 h, 58%; (e) 2-amino-4-(trifluoromethoxy)benzoic acid, HATU, (<i>i</i>-Pr)<sub>2</sub>NEt, DMF, rt, 2 h, 25%.</p></p></figure><div class="NLM_p last">Compound <b>41</b> (<b>BAY-885</b>) exhibits favorable physicochemical properties such as high solubility in combination with reasonable lipophilicity as measured by LogD (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Compound <b>41</b> is highly permeable in the Caco assay and shows no inhibition of activity of CYP enzymes up to 20 μM. Metabolic stability is low to moderate in rat hepatocytes and human liver microsomes, respectively. Compound <b>41</b> is chemically stable at different pH values and showed no inhibition of hERG up to 10 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Furthermore, compound <b>41</b> was tested at 1 μM concentration against 358 kinases in the commercial Eurofins kinase panel with only Fer (r), EphB3 (h), and EphA5 (h) kinases being inhibited at 62%, 58%, and 43%, respectively. For other kinases, inhibition was ≤20%, thus demonstrating that compound <b>41</b> (<b>BAY-885</b>) is a highly selective ERK5 inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf" class="ext-link">Table S2</a>). In contrast to <b>XMD8–92</b>, compound <b>41</b> showed no binding to BRD4 up to 20 μM.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Cellular Assays</h3><div class="NLM_p">The encouraging combination of favorable physicochemical properties with a highly selective profile against kinases prompted us to use <b>41</b> (<b>BAY-885</b>) to study ERK5 signaling in various cancer cell lines.</div><div class="NLM_p">In order to demonstrate that our compounds inhibit ERK5 kinase activity in cancer cell lines, we established a MEF2 reporter cell assay using the kidney cancer cell line SN12C (SN12C-MEF2-luc). MEF2 is a transcription factor directly activated by ERK5.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Therefore, ERK5 inhibition should result in a decreased luciferase signal in this cell line. In unstimulated cells, the basal luciferase signal was low, possibly due to poor activation of the ERK5 signaling pathway. To increase this signal, the cells were stimulated with EGF, which resulted in ERK5 activation and strong increase in the luciferase signal. Initial validation of the assay was performed with the available tool compounds, <b>XMD8–92</b> and <b>AX15836</b>. Both compounds had, however, strong limitations. <b>XMD8–92</b> has low kinase selectivity and is a dual ERK5/BRD4 inhibitor. <b>AX15836</b> is a potent ERK5 inhibitor, with increased selectivity.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Consistently, <b>AX15836</b> showed a strong inhibitory effect in the cellular mechanistic assay, with an IC<sub>50</sub> of 86 nM. However, at slightly higher concentrations, <b>AX15836</b> became completely inactive, potentially due to precipitation in the medium (IC<sub>90</sub> > 30 μM). Further validation of the cellular mechanistic assay was performed with our own compound series, showing a very good correlation between ERK5 kinase inhibition and cellular on-target activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlation between ERK5 kinase inhibition in a biochemical assay and inhibition of the transcriptional activity of MEF2 in SN12C cell line. Data of the entire quinazoline cluster are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The ERK5 probe <b>41</b> (<b>BAY-885</b>) showed potent ERK5 kinase and transcriptional inhibition in the SN12C-MEF2 reporter cell line (IC<sub>50</sub> = 115 nM/IC<sub>90</sub> = 691 nM) and had no effects on a reporter control cell line with constitutive luciferase expression (SN12C-CMV-luc, IC<sub>50</sub> > 30 μM), thereby ruling out potential effects as a general inhibitor of transcription or translation. To further address the potential of ERK5 as a therapeutic target in oncology, we tested the impact of compound <b>41</b> (<b>BAY-885</b>) on the proliferation of cells with ERK5 genomic amplification (SN12C, SNU-449, MFM-223)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> or with constitutively active ERK5 signaling (BT-474, SK-BR-3).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Importantly, despite its high potency, compound <b>41</b> (<b>BAY-885)</b> failed to inhibit the proliferation of all these cell lines. These results are in strong contrast with literature data suggesting that ERK5 is an oncogenic driver in tumors with dysregulated ERK5 signaling.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5−8)</a> A possible explanation lies on the different methodology used, with the mentioned studies employing RNAi technology to silence the ERK5 protein, while our results are based on ERK5 kinase inhibition with a small molecule.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33478" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33478" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Our results are in agreement with the findings of Lin et al.,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> which suggest that the kinase activity of ERK5 is dispensable for cancer cell growth, thus raising doubts as to the viability of ERK5 kinase as a therapeutic target for anticancer drug development. The availability of a potent and selective chemical probe, such as compound <b>41</b> (<b>BAY-885</b>), which was recently accepted as a donated chemical probe by the SGC,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> will significantly contribute to further understanding the biology of ERK5 signaling in cancer.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Whether inhibition of ERK5 kinase activity can be compensated by other pathways remains to be shown by future works, e.g., by combination studies using ERK1/ERK2 inhibitor with ERK5 inhibitor as suggested by Cox et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The lack of efficacy of an ERK5 kinase small molecule inhibitor contrasts with the antiproliferative effects of genetic depletion or deletion of ERK5, thus raising the question whether ERK5 can also act as a scaffolding protein. In this case, a PROTAC approach, leading to degradation of the whole ERK5 protein, would be appropriate to exploit the therapeutic potential of ERK5.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70860" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70860" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Methods</h4><div class="NLM_p">All reagents and solvents were used as purchased, unless otherwise specified. All final products were at least 95% pure, as determined by UPLC.</div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Compound Names</h5><div class="NLM_p last">ICS software was used to generate compound names.</div></div><div id="sec4_1_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Materials</h5><div class="NLM_p last">The syntheses of compound <b>1</b> and ERK5 probe molecule <b>41</b> (<b>BAY-885</b>) are described later in this section. The syntheses and analytical data for all other compounds (<b>2</b>–<b>40</b>, <b>42</b>) are described in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>. <sup>1</sup>H NMR purities were determined to be >95%. <sup>1</sup>H NMR spectra were recorded on Bruker Avance III HD spectrometers operating at 300, 400, or 500 MHz. The chemical shifts (δ) reported are given in parts per million (ppm), and the coupling constants (<i>J</i>) are in Hertz (Hz). The spin multiplicities are reported as s = singlet, br s = broad singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad. Please note that many of the final compounds bearing a piperidine core show broad signals for the piperidine NCH<sub>2</sub> and CH groups in the <sup>1</sup>H NMR (most likely due to ring inversion dynamics).</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> LC–MS Method 1</h5><div class="NLM_p last">System: Shimadzu LC–MS: UFLC 20-AD and LCMS 2020 MS detector. Column: Shim-pack XR-ODS 2.2 μm, 3.0 × 50 mm; solvent: A = water + 0.05% vol. HCOOH (99%); B = acetonitrile + 0.05% vol. HCOOH (99%).</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> LC–MS Method 2</h5><div class="NLM_p last">System: Shimadzu LC–MS: UFLC 20-AD and LCMS 2020 MS detector. Column: Shim-pack XR-ODS 2.2 μm, 3.0 × 50 mm; solvent: A = water + 0.05% vol. TFA (99%); B = acetonitrile + 0.05% vol. TFA (99%).</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> LC–MS Method 3</h5><div class="NLM_p last">System: Shimadzu LC–MS: UFLC 20-AD and LCMS 2020 MS detector. Column: Shim-pack XR-ODS 2.2 μm, 3.0 × 50 mm; solvent: A = water + 0.05% vol. NH<sub>4</sub>HCO<sub>3</sub> (99%); B = acetonitrile + 0.05% vol. NH<sub>4</sub>HCO<sub>3</sub> (99%).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 6,7-Dimethoxy-4-{1-[4-(trifluoromethoxy)benzoyl]piperidin-4-yl}quinazoline (<b>1</b>)</h4><div class="NLM_p">6,7-Dimethoxy-4-(piperidin-4-yl)quinazoline (<b>4</b>). Step 1: To a solution of 4-chloro-6,7-dimethoxyquinazoline (0.8 g, 3.6 mmol, prepared according to literature procedure<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a>) in 10 mL of 1,4-dioxane/water (v/v = 5:1) were added <i>tert</i>-butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate (1.70 g, 5.3 mmol), sodium carbonate (1.50 g, 14.2 mmol), and 1,1′- bis(diphenylphosphino)ferrocenepalladium(II) chloride (0.3 g, 0.4 mmol). The resulting mixture was stirred at 100 °C for 4 h under nitrogen atmosphere. After cooled to room temperature, water was added, and the resulting mixture was extracted with ethyl acetate. The combined organic layer was washed with water, brine, dried over anhydrous sodium sulfate, and concentrated in vacuo. The residue was purified by chromatography to give 0.98 g (74%) of <i>tert</i>-butyl-4-(6, 7-dimethoxyquinazolin-4-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate as a yellow solid. LC–MS [Method 3]: <i>R</i><sub>t</sub> = 1.33 min. MS (ESIpos): <i>m</i>/<i>z</i> = 372 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, methanol-d4) δ 8.95 (s, 1H, pyrimidine), 7.46 (s, 1H, phenyl), 7.33 (s, 1H, phenyl), 6.24 (br s, 1H, vinyl), 4.22 (br s, 2H, NCH<sub>2</sub>), 4.04 (s, 3H, OMe), 3.98 (s, 3H, OMe), 3.86–3.66 (m, 2H, NCH<sub>2</sub>), 2.70 (br d, <i>J</i> = 1.71 Hz, 2H, methylene-CH<sub>2</sub>), 1.52 (s, 9H, tBu).</div><div class="NLM_p">Step 2: <i>tert</i>-Butyl 4-(6, 7-dimethoxyquinazolin-4-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate (0.8 g, 2.2 mmol) was dissolved in 3 mL of dichloromethane. Trifluoroacetic acid (3 mL, 39 mmol) was added, and the resulting mixture was stirred at room temperature for 2 h. After evaporation in vacuo, saturated aqueous sodium carbonate was added to adjust the pH = 8. The mixture was extracted with dichloromethane, and the combined organic phase was dried over anhydrous sodium sulfate. After removal of the solvent, 0.51 g (87%) of 6,7-dimethoxy-4-(1,2,3,6-tetrahydropyridin-4-yl)quinazoline was obtained as a white solid. LC–MS [Method 2]: <i>R</i><sub>t</sub> = 0.79 min. MS (ESIpos): <i>m</i>/<i>z</i> = 272 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.93 (s, 1H, pyrimidine), 7.52 (s, 1H, phenyl), 7.29 (s, 1H, phenyl), 6.22–6.16 (m, 1H, vinyl), 4.03 (s, 3H, OMe), 3.98 (s, 3H, OMe), 3.59 (m, 2H, NCH<sub>2</sub>), 3.15 (m, 2H, NCH<sub>2</sub>), 2.62 (dt, <i>J</i> = 2.87, 5.17 Hz, 2H, methylene-CH<sub>2</sub>), NH proton not visible.</div><div class="NLM_p">Step 3: 6,7-dimethoxy-4-(1,2,3,6-tetrahydropyridin-4-yl)quinazoline (310 mg, 1.1 mmol) was dissolved in 20 mL of MeOH. After addition of 0.1 g of palladium/carbon, the resulting mixture was stirred at room temperature for overnight under a hydrogen atmosphere (3 atm). The solid was removed by filtration, and the filtrate was concentrated in vacuo to give 256.0 mg (80%) of 6,7-dimethoxy-4-(piperidin-4-yl)quinazoline <b>4</b> as a white solid. LC–MS [Method 1]: <i>R</i><sub>t</sub> = 0.67 min. MS (ESIpos): <i>m</i>/<i>z</i> = 274 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.03 (s, 1H, pyrimidine), 8.59 (br s, 1H, NH), 7.56 (s, 1H, phenyl), 7.36 (s, 1H, phenyl), 3.99 (m, 7H, 2 × OMe, CH), 3.51–3.40 (m, 2H, NCH<sub>2</sub>), 3.33–3.09 (m, 2H, NCH<sub>2</sub>), 2.19–1.91 (m, 4H, methylene-CH<sub>2</sub>).</div><div class="NLM_p last">Step 4<i>:</i> 6,7-Dimethoxy-4-{1-[4-(trifluoromethoxy)benzoyl]piperidin-4-yl}quinazoline (<b>1</b>). To a solution of 6,7-dimethoxy-4-(piperidin-4-yl)quinazoline <b>4</b> (100.0 mg, 0.4 mmol) in 6 mL of <i>N</i>,<i>N</i>-dimethylformamide were added 4-(trifluoromethoxy)benzoic acid (90.5 mg, 0.4 mmol), triethylamine (74 mg, 0.7 mmol), and HATU (166.8 mg, 0.4 mmol). The resulting mixture was stirred at room temperature for overnight. Upon completion of the reaction, 20 mL of water was added. The mixture was extracted with ethyl acetate, and the combined organic phase was dried over anhydrous sodium sulfate. The solvent was removed in vacuo, and the residue was purified by silica gel column chromatography to give 31 mg (18%) of 6,7-dimethoxy-4-{1-[4-(trifluoromethoxy)benzoyl]piperidin-4-yl}quinazoline (<b>1</b>) as a yellow solid. MS (ESIpos): <i>m</i>/<i>z</i> = 462 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.01 (s, 1H, pyrimidine), 7.68–7.52 (m, 3H, benzamide, phenyl of quinoline), 7.46 (d, <i>J</i> = 8.19 Hz, 2H, benzamide), 7.35 (s, 1H, phenyl of quinoline), 4.65 (br s, 1H, NCH<sub>2</sub>), 4.00 (m, 4H, OMe, CH), 3.98 (s, 3H, OMe), 3.70 (br s, 1H, NCH<sub>2</sub>), 3.29–3.02 (m, 2H, NCH<sub>2</sub>), 1.88 (br m, 4H, alkylene-CH<sub>2</sub>); please note that piperidine signals are very broad. <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 169.30, 167.78, 155.62, 152.80, 150.18, 148.77 (q, <i>J</i> = 1.9 Hz, 1C), 147.54, 135.56, 129.07, 121.07, 120.10 (q, <i>J</i> = 256 Hz, 1C), 118.05, 106.93, 102.12, 56.19, 56.17, 46.94, 41.75, 37.60, 31.37, 30.40. <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ = −57.07 (s, 3F).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> [2-Amino-4-(trifluoromethoxy)phenyl]{4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]piperidin-1-yl}methanone (<b>41</b>)</h4><div class="NLM_p">Step 1: Prepared in analogous fashion as described for compound <b>1</b> with 4,7-dichloropyrido[3,2-<i>d</i>]pyrimidine, <i>tert</i>-butyl 4-(7-chloropyrido[3,2-<i>d</i>]pyrimidin-4-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate (<b>44</b>) was obtained as a yellow solid (5.2 g, 37%). LC–MS [Method 2]: <i>R</i><sub>t</sub> = 2.81 min. MS (ESIpos): <i>m</i>/<i>z</i> = 347 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.28 (s, 1H, pyrimidine), 9.06 (d, <i>J</i> = 2.4 Hz, 1H, pyridine), 8.59 (d, <i>J</i> = 2.4 Hz, 1H, pyridine), 7.42 (br, 1H, vinyl), 4.21–4.12 (m, 2H, NCH<sub>2</sub>), 3.62–3.59 (m, 2H, NCH<sub>2</sub>), 2.78–2.76 (m, 2H, methylene-CH<sub>2</sub>), 1.46 (s, 9H, tBu).</div><div class="NLM_p">Step 2: To a solution of <i>tert</i>-butyl 4-(7-chloropyrido[3,2-<i>d</i>]pyrimidin-4-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate (1.2 g, 3.4 mmol) in 20 mL of toluene were added 1-methylpiperazine (507 mg, 5.1 mmol), palladium(II) acetate (76 mg, 0.3 mmol), BINAP (420 mg, 0.7 mmol), and cesium carbonate (3.3 g, 10.1 mmol). The resulting mixture was stirred at 100 °C for 14 h under nitrogen atmosphere. After cooled to room temperature, the solid was removed by filtration, and the filtrate was concentrated in vacuo. The residue was purified by silica gel column chromatography (dichloromethane/methanol = 20:1) to give 700 mg (crude) of the product as a yellow solid. Then 80 mg of crude product was repurified by <i>prep</i>-HPLC [Mobile Phase A: Water (0.1% NH<sub>4</sub>HCO<sub>3</sub>); Mobile Phase B: Acetonitrile; Gradient: 35% B to 60% B in 8 min] to give 32.1 mg of <i>tert</i>-butyl 4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]-1,2,3,6-tetrahydropyridine-1-carboxylate as a light yellow solid (<b>45</b>). LC–MS [Method 1]: <i>R</i><sub>t</sub> = 1.02 min. MS (ESIpos): <i>m</i>/<i>z</i> = 411 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d<sub>6</sub></i>): δ 9.00 (d, <i>J</i> = 2.8 Hz, 1H, pyridine), 8.96 (s, 1H, pyrimidine), 7.32 (d, <i>J</i> = 2.8 Hz, 1H, pyridine), 6.79–6.91 (m, 1H, vinyl), 5.12–4.72 (m, 2H, −NCH<sub>2</sub>), 3.89–3.82 (m, 2H, −NCH<sub>2</sub>), 3.54–3.44 (m, 4H, piperazine), 2.51–2.48 (m, 4H, piperazine, overlap with DMSO), 2.20 (s, 3H, −CH<sub>3</sub>), 1.99–2.15 (m, 2H, piperidine-CH<sub>2</sub>), 1.46 (s, 9H, tBu).</div><div class="NLM_p">Step 3: Prepared in analogous fashion as described for compound <b>1</b> with <i>tert</i>-butyl 4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]-1,2,3,6-tetrahydropyridine-1-carboxylate, <i>tert</i>-butyl 4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]piperidine-1-carboxylate was obtained as a yellow solid (20%, 110 mg). LC–MS [Method 1]: <i>R</i><sub>t</sub> = 0.81 min. MS(ESIpos): <i>m</i>/<i>z</i> = 413 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.04 (d, <i>J</i> = 2.7 Hz, 1H, pyridine), 9.03 (s, 1H, pyrimidine), 7.35 (d, <i>J</i> = 2.7 Hz, 1H, pyridine), 4.20–4.09 (m, 3H, piperidine), 3.53–3.51 (m, 4H, piperazine), 2.98–2.83 (br, 2H, NCH<sub>2</sub> piperidine), 2.24 (s, 3H, −CH<sub>3</sub>), 1.82–1.72 (m, 4H, piperidine), 1.43 (s, 9H -tBu), 4 piperazine protons not visible due to overlap with DMSO.</div><div class="NLM_p">Step 4: Prepared in analogous fashion as described for compound <b>1</b> with <i>tert</i>-butyl 4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]piperidine-1-carboxylate, 7-(4-methyl-piperazin-1-yl)-4-(piperidin-4-yl)pyrido[3,2-<i>d</i>]pyrimidine (<b>46</b>) was obtained as a yellow solid (58%, 50 mg). LC–MS [Method 1]: <i>R</i><sub>t</sub> = 0.13 min. MS (ESIpos): <i>m</i>/<i>z</i> = 313 (M + H)<sup>+</sup>.</div><div class="NLM_p last">Step 5: Prepared in analogous fashion as described for compound <b>1</b> with 7-(4-methyl-piperazin-1-yl)-4-(piperidin-4-yl)pyrido[3,2-<i>d</i>]pyrimidine to afford to afford [2-amino-4-(trifluoromethoxy)-phenyl]{4-[7-(4-methylpiperazin-1-yl)pyrido[3,2-<i>d</i>]pyrimidin-4-yl]piperidin-1-yl}methanone as a yellow solid (71 mg, 25%). LC–MS [Method 1]: <i>R</i><sub>t</sub> = 0.94 min. MS (ESIpos): <i>m</i>/<i>z</i> = 516 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>,) δ = 9.06 (d, <i>J</i> = 2.8 Hz, 1H, pyridine), 9.04 (s, 1H, pyrimidine), 7.38 (d, <i>J</i> = 2.8 Hz, 1H, pyridine), 7.14 (d, <i>J</i> = 8.3 Hz, 1H, phenyl), 6.67 (br, 1H, phenyl), 6.50 (br d, <i>J</i> = 8.4 Hz, 1H, phenyl), 5.61 (br s, 2H, −NH<sub>2</sub>), 4.34–4.25 (m, 1H, piperidine), 3.55–3.50 (m, 4H, piperazine), 3.22–3.05 (very br, 2H, NCH<sub>2</sub> piperidine), 2.49–2.47 (m, 4H, piperazine, partially overlap with DMSO), 2.24 (s, 3H, −CH<sub>3</sub>), 1.94–1.86 (br m, 4H, piperidine), 2 protons of piperidine-NCH<sub>2</sub>, expected to be around 4.1 ppm not visible. <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.84, 167.43, 155.47, 149.63, 149.39, 147.69, 147.25, 144.02, 130.57, 129.46, 120.10 (q, <i>J</i> = 256 Hz, 1C), 118.72, 112.01, 107.10, 106.49, 54.09, 46.49, 45.75, 36.80, 30.5 (br), 3 piperidine carbon resonances not visible (2× −NCH<sub>2</sub>, might overlap with piperazine at 46 ppm, and −CH<sub>2</sub>, note of broad signal at 30 ppm). <sup>19</sup>F NMR (565 MHz, DMSO-<i>d</i><sub>6</sub>) δ = −56.70 (s, 3F).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Biology</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Cell Culture</h4><div class="NLM_p last">Parental cell lines were obtained from the American Type Culture Collection (BT-474, SK-BR-3, SNU-449, NCI-H460), the NCI 60 panel (SN12C), and the European Collection of Authenticated Cell Cultures (MFM-223). They were maintained in the recommended cell culture media at 37 °C in 5% CO<sub>2</sub>.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Biochemical ERK5 Inhibition Assay</h4><div class="NLM_p">Recombinant fusion protein of N-terminal Glutathion-S-Transferase (GST) and a fragment of human ERK5 (amino acids 1–398 of accession number NP_002740.2]), expressed in <i>E. coli</i>, purified via affinity chromatography using Glutathion Sepharose and subsequently activated with His-tagged MAP2K5, was purchased from Carna Biosciences (product number 04–146) and used as kinase. As substrate for the kinase reaction biotinylated peptide biotin-Ahx-PPGDYSTTPGGTLFSTTPGGTRI (C-terminus in amide form) was used, which can be purchased, e.g., from the company Biosyntan (Berli<i>n</i>-Buch, Germany).</div><div class="NLM_p last">For the assay, 50 nL of a 100-fold concentrated solution of the test compound in DMSO was pipetted into either a black low volume 384-well microtiter plate or a black 1536-well microtiter plate (both Greiner Bio-One, Frickenhausen, Germany), 2 μL of a solution of ERK5 in aqueous assay buffer [50 mM Hepes pH 7.0, 15 mM MgCl<sub>2</sub>, 1 mM dithiothreitol, 0.5 mM EGTA, 0.05% (w/v) bovine γ-globulin (Sigma-Aldrich, # G5009), 0.005% (v/v) NP40 (AppliChem, # A2239)] were added, and the mixture was incubated for 15 min at 22 °C to allow prebinding of the test compounds to the enzyme before the start of the kinase reaction. Then the kinase reaction was started by the addition of 3 μL of a solution of adenosine-triphosphate (ATP, 417 μM → the final conc. in the 5 μL assay volume is 250 μM) and substrate (1.67 μM → final conc. in the 5 μL assay volume is 1 μM) in assay buffer, and the resulting mixture was incubated for a reaction time of 60 min at 22 °C. The concentration of ERK5 was adjusted depending of the activity of the enzyme lot and was chosen appropriate to have the assay in the linear range, a typical concentration was 0.5 μg/mL. The reaction was stopped by the addition of 3 μL of a solution of TR-FRET detection reagents (0.33 μM streptavidine-XL665 [Cisbio Bioassays, Codolet, France] and 1.67 nM anti-4E-BP1 (pT46) antibody from Invitrogen [catalogue no.700397] and 1.67 nM LANCE EU-W1024 labeled antirabbit IgG antibody [PerkinElmer, product no. AD0083]) in an aqueous EDTA-solution (83.3 mM EDTA, 0.2% (w/v) bovine serum albumin in 50 mM HEPES, pH 7.5). The resulting mixture was incubated 1 h at 22 °C to allow the formation of complex between the phosphorylated biotinylated peptide and the detection reagents. Subsequently, the amount of phosphorylated substrate was evaluated by measurement of the resonance energy transfer from the Eu-chelate to the streptavidine-XL. Therefore, the fluorescence emissions at 620 and 665 nm after excitation at 350 nm was measured in a TR-FRET reader, e.g., a Pherastar FS (BMG Labtechnologies, Offenburg, Germany) or a Viewlux (PerkinElmer). The ratio of the emissions at 665 nm and at 622 nm was taken as the measure for the amount of phosphorylated substrate. The data were normalized (enzyme reaction without inhibitor = 0% inhibition; all other assay components but no enzyme = 100% inhibition). Usually the test compounds were tested on the same microtiter plate in 11 different concentrations in the range of 20 μM to 0.07 nM (20 μM, 5.7 μM, 1.6 μM, 0.47 μM, 0.13 μM, 38 nM, 11 nM, 3.1 nM, 0.9 nM, 0.25 nM, and 0.07 nM, the dilution series prepared separately before the assay on the level of the 100-fold concentrated solutions in DMSO by serial dilutions; exact concentrations may vary depending pipettors used) in duplicate values for each concentration, and IC<sub>50</sub> values were calculated using Genedata Screener software.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Cellular Luciferase Reporter Assay</h4><div class="NLM_p">The SN12C-MEF2-luc (clone #37) reporter cell line has been generated by stably transducing SN12C cells with a MEF2-responsive transcription element upstream of a firefly luciferase gene (Qiagen/SABiosciences) and was used to determine the cellular activity of ERK5 inhibitors. In parallel, a SN12C-CMV-luc (clone #1) reporter cell line has been generated that recombinantly carries a CMV-promoter driven firefly luciferase gene, thereby constitutively expressing luciferase. The latter is used to detect false positive hits of SN12C-MEF2-luc reporter assay, being either toxic compounds, general inhibitors of the transcriptional or translational machinery, or inhibitors of luciferase activity. Generation of the poly- and selection of the monoclonal reporter cell lines was carried out at the NMI (Natural and Medical Sciences Institute) at the University of Tuebingen. These cell lines were grown in RPMI 1640 Medium without Phenol Red (Biochrom, #F1275) supplemented with 10% FCS and Glutamax. Additionally, for SN12C-CMV-luc, 0.4 μg/μL of Blasticidin (GIBCO, #R210–01) was also present in the culture medium. All cells were grown at 37 °C in a humidified atmosphere with 5% CO<sub>2</sub>. On day 1, the cells were seeded in 384-well white plates (PerkinElmer #6007680) at a density of 10,000 cells per well in 20 μL of culture medium. On day 2, the test compounds were added in serial dilutions using the HP D300 Digital Dispenser and incubated at 37 °C for 16 h. On day 3, EGF (Invitrogen, #PHG0311L) was added to every well (final concentration = 100 ng/mL), and the plates were incubated for additional 2 h at 37 °C. Then, 25 μL of ONE-Glo (Promega, #E6120) was added to each well, and the plates were incubated for 5 min at RT (shaking). The luminescence signal was read on PHERAStar (BMG Labtech).</div><div class="NLM_p last">IC<sub>50</sub>s were calculated using the DRC Master Spreadsheet (Bella software). Values obtained for cells treated with EGF and DMSO were defined as the maximum control, while values for cells treated with EGF and 10 μM of XMD8–92 (SN12C-MEF2-luc) or 1 μM Staurosporine (SN12C-CMV-luc) were defined as the minimum control (i.e., maximum inhibition).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Biochemical BRD4 BD1 and BD2 Binding Inhibition Assays</h4><div class="NLM_p last">To test inhibitor affinity toward bromodomains BD 1 and 2 of Bromodomain containing 4 (BRD4), a biochemical TR-FRET based assay was used, similarly to that described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In this assay, a fluorescence signal is produced through the binding of N-terminal His6-tagged human BRD4 BD1 or BD2 to synthetic, tetra-acetylated peptides derived from human histone 4. BD1 (amino acids 44–168) and BD2 (amino acids 333–460) proteins were expressed in <i>E. coli</i>, purified via affinity chromatography/IMAC and size exclusion chromatography. Acetylated, biotinylated peptides (peptide for BD1 interaction: SGRGK(Ac)GGK(Ac)GLGK(Ac)GGAK(Ac)RHGSGSK-Btn; peptide for BD2 interaction: SGRGK(Ac)GGK(Ac)GLGK(Ac)GGAK(Ac)RHRKVLRDNGSGSK-Btn) were purchased from Biosyntan (Berlin, Germany). The extent of inhibitor binding to BRD4, i.e., displacement of the biotinylated peptides, is monitored by the degree of signal decrease. The experiments were performed in 384-well black small-volume microtiter plates (Greiner Bio-One, Frickenhausen, Germany), where 50 nL of a 100-fold concentrated solution of a test compound (see final conc. below) in DMSO was predispensed. BRD4 BD1 (final conc. in assay 10 nM) in 2 μL of assay buffer composed of 50 mM HEPES (pH 7.5) (Applichem, Darmstadt, Germany), 50 mM NaCl (Sigma), 0.25 mM CHAPS (Sigma), and 0.05% bovine serum albumin (BSA; Sigma) was added to the test compound. After 10 min preincubation at room temperature, 3 μL of a detection solution containing biotinylated peptide for BD1 interaction (final conc. 50 nM), anti-6His-XL665 (final conc. 10 nM; Cisbio Bioassays, Codolet, France), streptavidin Eu<sup>3+</sup> chelate (final conc. 2.5 nM; W1024, PerkinElmer, Waltham, USA), and potassium fluoride (KF; final conc. 50 mM; Sigma) in assay buffer were added to the plate. After incubation for 3 h at 4 °C, the plate was measured in a PheraStar reader (BMG Labtech) using the homogeneous time-resolved fluorescence (HTRF) module (excitation, 337 nm; emission, 620 and 665 nm). The BRD4 BD2 (final conc. in assay 100 nM) assay was performed similarly, except for the following modifications: conditions assay buffer (50 mM HEPES (pH 7.5), 100 mM NaCl, 0.25 mM CHAPS, and 0.05% BSA) and detection solution (3 μL, containing biotinylated peptide for BD2 interaction (final conc. in assay 50 nM), anti-6His-XL665 (final conc. 50 nM), streptavidin Eu<sup>3+</sup> chelate (final conc. 8.6 nM), and KF (final conc. 50 mM) in assay buffer). The ratio of the emissions at 665 and 620 nm were calculated and normalized to neutral (DMSO instead of test compound = 0% binding inhibition) and inhibitor control (all assay components except BRD4 BD1/BD2 = 100% binding inhibition). The compounds were tested at 11 different concentrations in the range of 20 μM to 0.07 nM (20 μM, 5.7 μM, 1.6 μM, 0.47 μM, 0.13 μM, 38 nM, 11 nM, 3.1 nM, 0.9 nM, 0.25 nM, and 0.07 nM) in duplicates for each concentration, and IC<sub>50</sub> values were calculated using the Genedata Screener software.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> X-ray Structures of <b>1</b> and <b>35</b> in Complex with Erk5</h3><div class="NLM_p last">Protein was produced as described previously and concentrated to 12 mg/mL in 50 mM HEPES pH 6.5, 150 mM NaCl, 10% glycerol, and 2 mM DTT.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Prior to crystallization, the protein solution was supplemented with either compound <b>1</b> or compound <b>35</b> (final concentration of 1 mM) and incubated for 3 h on ice. Samples were then clarified by centrifugation (5 min, 13 000<i>g</i>, 277 K). Complex crystals were grown using the hanging-drop method by mixing 0.75 μL of protein and with 0.5 μL of reservoir solution (Compound <b>1</b>: 11% PEG 4000, 100 mM MgCl<sub>2</sub>, 160 mM sodium formate, 100 mM MES, pH 6.75, and 100 mM Tris, pH 8.5; compound <b>35</b>: 15% PEG 4000, 100 mM MgCl<sub>2</sub>, 180 mM sodium formate, 100 mM MES, pH 6.5, and 100 mM Tris, pH 8.5. Crystals were harvested and cryoprotected by brief immersion in a cryoprotection solution consisting of mother liquor supplemented with 30% glycerol and then flash frozen in liquid nitrogen. Data were collected under cryogenic conditions at the synchrotron facility at the SLS in Villigen, Switzerland. The structure was solved by molecular replacement using PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IC8">4IC8</a> as a search model. The structure was refined using REFMAC5 within the CCP4 suite.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Statistics for the final modes are given in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf" class="ext-link">Table S3</a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> PK Assays</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Caco2 Permeability Assay</h4><div class="NLM_p last">Cell culture: Caco-2 cells (purchased from DSMZ Braunschweig, Germany) were seeded at a density of 2.5 × 10<sup>5</sup> cells per well on 24-well insert plates, 0.4 μm pore size, 0.3 cm<sup>2</sup> (Costar), and grown for 13–15 days in DMEM medium supplemented with 10% fetal calf serum (FCS), 1% GlutaMAX (100×, GIBCO), 100 U/mL penicillin, 100 μg/mL streptomycin (GIBCO), and 1% nonessential amino acids (100×). Cells were maintained at 37 °C in a humidified 5% CO<sub>2</sub> atm. Medium was changed every 2–3 days. Evaluation of Caco-2 permeability in a bidirectional transport assay: The bidirectional transport assay was done in 24-well insert plates using a robotic system (Tecan). Before running the bidirectional transport assay, culture medium was replaced by transport medium (FCS-free HEPES-carbonate transport puffer, pH 7.2). For assessment of monolayer integrity, the transepithelial electrical resistance (TEER) was measured. Only monolayers with a TEER of at least 400 Ω cm<sup>2</sup> were used. Test compounds were predissolved in DMSO and added either to the apical or basolateral compartment in final concentration of 2 μM. Evaluation was done in triplicates. Before and after 2 h of incubation at 37 °C, samples were taken from both compartments and analyzed after precipitation with methanol by LC–MS/MS. The apparent permeability coefficient (Papp) was calculated both for the apical to basolateral (A → B) and the basolateral to apical (B → A) direction using following equation: Papp = (<i>V</i><sub>r</sub>/<i>P</i><sub>0</sub>)(1/<i>S</i>)(<i>P</i><sub>2</sub>/<i>t</i>) where <i>V</i><sub>r</sub> is the volume of medium in the receiver chamber, <i>P</i><sub>0</sub> is the measured peak area of the test drug in the donor chamber at <i>t</i> = 0, <i>S</i> is the surface area of the monolayer, <i>P</i><sub>2</sub> is the measured peak area of the test drug in the acceptor chamber after 2 h of incubation, and <i>t</i> is the incubation time. The efflux ratio basolateral (B) to apical (A) was calculated by dividing Papp(B-A) by Papp(A-B).</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> In Vitro Metabolic Stability in Human Liver Microsomes</h4><div class="NLM_p last">The in vitro metabolic stability of test compounds was determined by incubating them at 1 μM in a suspension of liver microsomes in 100 mM phosphate buffer, pH 7.4 (NaH<sub>2</sub>PO<sub>4</sub>·H<sub>2</sub>O + Na<sub>2</sub>HPO<sub>4</sub>·2H<sub>2</sub>O) and at a protein concentration of 0.5 mg/mL at 37 °C. The microsomes were activated by adding a cofactor mix containing 8 mM Glukose-6-Phosphat, 4 mM MgCl<sub>2</sub>, 0.5 mM NADP, and 1 IU/mL G-6-P-dehydrogenase in phosphate buffer, pH 7.4. The metabolic assay was started shortly afterward by adding the test compound to the incubation at a final volume of 1 mL. Organic solvent in the incubations was limited to ≤0.01% dimethyl sulfoxide (DMSO) and ≤1% acetonitrile. During incubation, the microsomal suspensions were continuously shaken at 580 rpm, and aliquots were taken at 2, 8, 16, 30, 45, and 60 min, to which equal volumes of cold methanol were immediately added. Samples were frozen at −20 °C overnight and subsequently centrifuged for 15 min at 3000 rpm, and the supernatant was analyzed with an Agilent 1200 HPLC-system with LC–MS/MS detection. The half-life of a test compound was determined from the concentration–time plot. From the half-life the intrinsic clearances and the hepatic in vivo blood clearance (CL) and maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the “well stirred” liver model together with the additional parameters liver blood flow, specific liver weight, and microsomal protein content. The following parameter values were used: liver blood flow 1.32 L/h/kg, specific liver weight 21 g/kg, and microsomal protein content 40 mg/g.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> In Vitro Metabolic Stability in Rat Hepatocytes</h4><div class="NLM_p last">Hepatocytes from Han/Wistar rats were isolated via a two-step perfusion method. After perfusion, the liver was carefully removed from the rat: the liver capsule was opened, and the hepatocytes were gently shaken out into a Petri dish with ice-cold Williams’ medium E (WME). The resulting cell suspension was filtered through sterile gaze in 50 mL falcon tubes and centrifuged at 50<i>g</i> for 3 min at room temperature. The cell pellet was resuspended in 30 mL of WME and centrifuged twice through a Percoll gradient at 100<i>g</i>. The hepatocytes were washed again with WME and resuspended in medium containing 5% FCS. Cell viability was determined by trypan blue exclusion. For the metabolic stability assay, liver cells were distributed in WME containing 5% FCS to glass vials at a density of 1.0 × 10<sup>6</sup> vital cells/mL. The test compound was added to a final concentration of 1 μM. During incubation, the hepatocyte suspensions were continuously shaken at 580 rpm, and aliquots were taken at 2, 8, 16, 30, 45, and 90 min, to which equal volumes of cold methanol were immediately added. Samples were frozen at −20 °C overnight, subsequently centrifuged for 15 min at 3000 rpm, and the supernatant was analyzed with an Agilent 1200 HPLC-system with LC–MS/MS detection. The half-life of a test compound was determined from the concentration–time plot. From the half-life, the intrinsic clearances, the hepatic in vivo blood clearance (CL), and the maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the “well stirred” liver model together with the additional parameters of liver blood flow, specific liver weight, and amount of liver cells in vivo and in vitro. The following parameter values were used: liver blood flow 4.2 L/h/kg, specific liver weight 32 g/kg, liver cells in vivo 1.1 × 10<sup>8</sup> cells/g liver, and liver cells in vitro 1.0 × 10<sup>6</sup>/mL.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Inhibition of CYP450 Metabolism</h4><div class="NLM_p last">The inhibitory potency of the test compounds toward cytochrome P450-dependent metabolic pathways was determined in human liver microsomes by applying individual CYP isoform-selective standard probes (CYP1A2 phenacetin, CYP2C8 amodiaquine, CYP2C9 diclofenac, CYP2D6 dextromethorphan, CYP3A4 midazolam). Reference inhibitors were included as positive controls. Incubation conditions (protein and substrate concentration, incubation time) were optimized with regard to linearity of metabolite formation. Assays were processed in 96-well plates at 37 °C by using a Genesis Workstation (Tecan, Crailsheim, Germany). After protein precipitation, the metabolite formation was quantified by LC–MS/MS analysis followed by inhibition evaluation and IC<sub>50</sub> calculation.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Safety Assays</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Automated hERG K+ Current Voltage-Clamp Assay</h4><div class="NLM_p last">The hERG K+ current assay is based on a recombinant HEK293 cell line with stable expression of the KCNH2(HERG) gene.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The cells were cultured using a humidified incubator (37 °C, 5% CO<sub>2</sub>) and a standard culture medium (MEM with Earle’s salts and <span class="smallcaps smallerCapital">l</span>-glutamine, 10% noninactivated fetal calf serum, 0.1 mmol/L nonessential amino acids, 1 mmol/L Na-pyruvate, penicillin/streptomycin (50 μg/mL each), 0.4 mg/mL Geneticin). Approximately 0.5–8 h following cell dissociation, the cells are investigated by means of the “whole-cell voltage-clamp” technique<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> in an automated 8-channel system (Patchliner; Nanion Technologies, München, Germany) with PatchControlHT software (Nanion) to control the Patchliner system and to handle data aquisition and analysis. Voltage-clamp control was provided by two EPC 10 quadro amplifiers under control of the PatchMasterPro software (both: HEKA Elektronik, Lambrecht, Germany) and with NPC-16 medium resistance (∼2 MΩ) chips (Nanion) serving as planar substrate at room temperature (22–24 °C). NPC-16 chips are filled with intra- and extracellular solution (intracellular solution (in mmol/L): NaCl 10, KCl 50, KF 60, EGTA 20, HEPES 10, pH 7.2 (KOH); extracellular solution (in mmol/L): NaCl 140, KCl 4, CaCl<sub>2</sub> 2, MgCl<sub>2</sub> 1, glucose 5, HEPES 10, pH 7.4 (NaOH)) and with cell suspension. After formation of a GΩ-seal and entering whole-cell mode (including several automated quality control steps), the cell membrane is clamped to the holding potential (−80 mV). Following an activating clamp step (+20 mV, 1000 ms), exclusively hERG-mediated inward tail currents are elicited by hyperpolarizing voltage steps from +20 mV to −120 mV (duration 500 ms); this clamp protocol is repeated every 12 s.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> After an initial stabilization phase (5–6 min), test compounds were added either as single concentration (10 μmol/L) or in ascending concentrations (0.1, 1, and 10 μmol/L; 5–6 min per concentration), followed by several washout steps. Effects of test compounds are quantified by analyzing the amplitude of the hERG-mediated inward tail currents (in percent of predrug control) as a function of test compound concentration (IgorPro Software). Mean concentration–response data were fitted with a standard sigmoidal 4-parameter logistic equation of the form: <i>Y</i> = bottom + (top – bottom)/(1 + exp((Log IC<sub>50</sub> – <i>X</i>) × Hill Slope)), where <i>Y</i> is the current inhibition (in % of predrug control), <i>X</i> is the logarithm of drug concentration, and IC<sub>50</sub> is the drug concentration producing half-maximal current inhibition, and using the following constraints: top = 100%, bottom = 0%. No curve fitting was performed in cases with an obvious lack of a concentration-dependent current inhibition and/or a too small effect size (approximately ≤20%).</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Physicochemical Assays</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Stability of Compounds in Solution (pH 10, 7, and 1 at 37 °C)</h4><div class="NLM_p">Solution stability was determined by HPLC-UV.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Five microliters of a 10 mM solution of drug in DMSO was solved in 1 mL of acetonitrile. One hundred microliters of this solution were transferred to 1 mL of the respective buffer and mixed thoroughly. Injections were made immediately after mixing for time zero injection and then again after 1, 2, and 24 h. Compounds were incubated at 37 °C.</div><div class="NLM_p last">Degradation rate (recovery in %) was calculated by relating peak areas after 1, 2, and 24 h to the time zero injection.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Aqueous Solubility of Compound–DMSO Solutions</h4><div class="NLM_p last">Aqueous solubility at pH 6.5 was determined by an orientating high throughput screening method.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Test compounds were applied as 1 mM DMSO solution. After addition of buffer pH 6.5, solutions were shaken for 24 h at room temperature. Undissolved material was separated by filtration. The compound dissolved in the supernatant was quantified by HPLC–MS/MS.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> LogD Measurement</h4><div class="NLM_p last">LogD values at pH 7.5 were recorded using an indirect method for determining hydrophobicity constants by reversed-phase high performance liquid chromatography (RP-HPLC).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> A homologous series of <i>n</i>-alkan-2-ones (C<sub>3</sub>–C<sub>16</sub>, 0.02 mol in acetonitrile) was used for calibration. Test compounds were applied as 0.67 mM DMSO stock solutions in acetonitrile/water 1:1. The lipophilicity of compounds was then assessed by comparison to the calibration curve.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01606" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01606" class="ext-link">10.1021/acs.jmedchem.8b01606</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Reference compounds used for optimization of cellular mechanistic assay; results of commercial kinase panel at Eurofins for ERK5 probe molecule <b>41 (BAY-885)</b>; crystallographic data of compounds <b>1</b> and <b>35</b>; data collection and refinement statistics; syntheses procedures for compounds <b>5</b>–<b>40</b> and <b>42</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf">jm8b01606_si_001.pdf (1.1 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_002.csv">jm8b01606_si_002.csv (2.65 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for the described crystal structures have been deposited with the Protein Data Bank (PDB). The PDB accession codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM">6HKM</a> for compound <b>1</b> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN">6HKN</a> for compound <b>35</b>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01606" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duy Nguyen</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4534-745X" title="Orcid link">http://orcid.org/0000-0002-4534-745X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ceaabbb7e0a0a9bbb7aba08eacafb7abbce0ada1a3"><span class="__cf_email__" data-cfemail="c8acbdb1e6a6afbdb1ada688aaa9b1adbae6aba7a5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lars Wortmann</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6514-947X" title="Orcid link">http://orcid.org/0000-0001-6514-947X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7c101d0e0f520b130e08111d12123c1e1d05190e521f1311"><span class="__cf_email__" data-cfemail="96faf7e4e5b8e1f9e4e2fbf7f8f8d6f4f7eff3e4b8f5f9fb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clara Lemos</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Knut Eis</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon J. Holton</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulf Boemer</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dieter Moosmayer</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Uwe Eberspaecher</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joerg Weiske</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Lechner</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Prechtl</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Detlev Suelzle</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franziska Siegel</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florian Prinz</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ralf Lesche</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Barbara Nicke</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katrin Nowak-Reppel</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Herbert Himmel</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominik Mumberg</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franz von Nussbaum</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carl F. Nising</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcus Bauser</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Haegebarth</span> - <span class="hlFld-Affiliation affiliation">Research & Development,
Pharmaceuticals, Bayer AG, 13353 Berlin, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written with contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are or have been employees and stockholders of Bayer AG.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Pharmaron Beijing Co., Ltd. for excellent support in chemistry and Proteros Biostructures GmbH for the X-ray structure determination of compounds <b>1</b> and <b>35</b>.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS USED</td></tr><tr><td class="NLM_term">BMK1</td><td class="NLM_def"><p class="first last">big MAP kinase</p></td></tr><tr><td class="NLM_term">BRDs</td><td class="NLM_def"><p class="first last">bromodomains</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain containing 4</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DRC</td><td class="NLM_def"><p class="first last">dose–response curve</p></td></tr><tr><td class="NLM_term">EGF</td><td class="NLM_def"><p class="first last">epidermal growth factor</p></td></tr><tr><td class="NLM_term">ERK1</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase 1</p></td></tr><tr><td class="NLM_term">ERK2</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase 2</p></td></tr><tr><td class="NLM_term">ERK5</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase 5</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">(4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">MAP2K5 or MEK5</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase 5</p></td></tr><tr><td class="NLM_term">MEF2</td><td class="NLM_def"><p class="first last">myocyte enhancer factor-2</p></td></tr><tr><td class="NLM_term">MEKK2</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase kinase 2</p></td></tr><tr><td class="NLM_term">MEKK3</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase kinase 3</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">RNAi</td><td class="NLM_def"><p class="first last">RNA interference</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">TEY</td><td class="NLM_def"><p class="first last">threonine-glutamic acid-tyrosine</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drew, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckman, B. S.</span></span> <span> </span><span class="NLM_article-title">MEK5/ERK5 pathway: the first fifteen years</span>. <i>Biochim. Biophys. Acta, Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1825</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2011.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.bbcan.2011.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=22020294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Cju7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1825&publication_year=2012&pages=37-48&author=B.+A.+Drewauthor=M.+E.+Burowauthor=B.+S.+Beckman&title=MEK5%2FERK5+pathway%3A+the+first+fifteen+years&doi=10.1016%2Fj.bbcan.2011.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEK5/ERK5 pathway: The first fifteen years</span></div><div class="casAuthors">Drew, Barbara A.; Burow, Matthew E.; Beckman, Barbara S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1825</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-48</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  While conventional MAP kinase pathways are one of the most highly studied signal transduction mols., less is known about the MEK5 signaling pathway.  This pathway has been shown to play a role in normal cell growth cycles, survival and differentiation.  The MEK5 pathway is also believed to mediate the effects of a no. of oncogenes.  MEK5 is the upstream activator of ERK5 in many epithelial cells.  Activation of the MEK-MAPK pathway is a frequent event in malignant tumor formation and contributes to chemoresistance and anti-apoptotic signaling.  This pathway may be involved in a no. of more aggressive, metastatic varieties of cancer due to its role in cell survival, proliferation and EMT transitioning.  Further study of this pathway may lead to new prognostic factors and new drug targets to combat more aggressive forms of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofQScETgHyILVg90H21EOLACvtfcHk0lj78v5MDwulNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Cju7%252FJ&md5=5f56fde5440678be1cf0e358daf1c6d2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2011.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2011.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDrew%26aufirst%3DB.%2BA.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26aulast%3DBeckman%26aufirst%3DB.%2BS.%26atitle%3DMEK5%252FERK5%2520pathway%253A%2520the%2520first%2520fifteen%2520years%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2012%26volume%3D1825%26spage%3D37%26epage%3D48%26doi%3D10.1016%2Fj.bbcan.2011.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavanaugh, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckman, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span> <span> </span><span class="NLM_article-title">Oncogenic signaling of MEK5-ERK5</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>392</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.canlet.2017.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=28153789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2017&pages=51-59&author=V.+T.+Hoangauthor=T.+J.+Yanauthor=J.+E.+Cavanaughauthor=P.+T.+Flahertyauthor=B.+S.+Beckmanauthor=M.+E.+Burow&title=Oncogenic+signaling+of+MEK5-ERK5&doi=10.1016%2Fj.canlet.2017.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic signaling of MEK5-ERK5</span></div><div class="casAuthors">Hoang, Van T.; Yan, Thomas J.; Cavanaugh, Jane E.; Flaherty, Patrick T.; Beckman, Barbara S.; Burow, Matthew E.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-59</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinases (MAPKs) regulate diverse cellular processes including proliferation, cell survival, differentiation, and apoptosis.  While conventional MAPK constituents have well-defined roles in oncogenesis, the MEK5 pathway has only recently emerged in cancer research.  In this review, we consider the MEK5 signaling cascade, focusing specifically on its involvement in drug resistance and regulation of aggressive cancer phenotypes.  Moreover, we explore the role of MEK5/ERK5 in tumorigenesis and metastatic progression, discussing the discrepancies in preclin. studies and assessing its viability as a therapeutic target for anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprYwxatp_EjrVg90H21EOLACvtfcHk0lj78v5MDwulNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Smt74%253D&md5=2047e484e36e06d8345829b49f0c1c4e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DHoang%26aufirst%3DV.%2BT.%26aulast%3DYan%26aufirst%3DT.%2BJ.%26aulast%3DCavanaugh%26aufirst%3DJ.%2BE.%26aulast%3DFlaherty%26aufirst%3DP.%2BT.%26aulast%3DBeckman%26aufirst%3DB.%2BS.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26atitle%3DOncogenic%2520signaling%2520of%2520MEK5-ERK5%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D392%26spage%3D51%26epage%3D59%26doi%3D10.1016%2Fj.canlet.2017.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nithianandarajah-Jones, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldring, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, M. J.</span></span> <span> </span><span class="NLM_article-title">ERK5: Structure, regulation and function</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2187</span>– <span class="NLM_lpage">2196</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2012.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.cellsig.2012.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=22800864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSns7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=2187-2196&author=G.+N.+Nithianandarajah-Jonesauthor=B.+Wilmauthor=C.+E.+Goldringauthor=J.+Mullerauthor=M.+J.+Cross&title=ERK5%3A+Structure%2C+regulation+and+function&doi=10.1016%2Fj.cellsig.2012.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">ERK5: Structure, regulation and function</span></div><div class="casAuthors">Nithianandarajah-Jones, Gopika N.; Wilm, Bettina; Goldring, Christopher E. P.; Muller, Jurgen; Cross, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2187-2196</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Extracellular signal-regulated kinase 5 (ERK5), also termed big mitogen-activated protein kinase-1 (BMK1), is the most recently identified member of the mitogen-activated protein kinase (MAPK) family and consists of an amino-terminal kinase domain, with a relatively large carboxy-terminal of unique structure and function that makes it distinct from other MAPK members.  It is ubiquitously expressed in numerous tissues and is activated by a variety of extracellular stimuli, such as cellular stresses and growth factors, to regulate processes such as cell proliferation and differentiation.  Targeted deletion of Erk5 in mice has revealed that the ERK5 signalling cascade plays a crit. role in cardiovascular development and vascular integrity.  Recent data points to a potential role in pathol. conditions such as cancer and tumor angiogenesis.  This review focuses on the physiol. and pathol. role of ERK5, the regulation of this kinase and the recent development of small mol. inhibitors of the ERK5 signalling cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqhjeO2v9aX7Vg90H21EOLACvtfcHk0lj78v5MDwulNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSns7fP&md5=ebae0228bd187490f05945d411d34ad4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2012.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2012.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DNithianandarajah-Jones%26aufirst%3DG.%2BN.%26aulast%3DWilm%26aufirst%3DB.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DCross%26aufirst%3DM.%2BJ.%26atitle%3DERK5%253A%2520Structure%252C%2520regulation%2520and%2520function%26jtitle%3DCell.%2520Signalling%26date%3D2012%26volume%3D24%26spage%3D2187%26epage%3D2196%26doi%3D10.1016%2Fj.cellsig.2012.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simoes, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borralho, P. M.</span></span> <span> </span><span class="NLM_article-title">The MEK5/ERK5 signalling pathway in cancer: A promising novel therapeutic target</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1663</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.drudis.2016.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=27320690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1654-1663&author=A.+E.+Simoesauthor=C.+M.+Rodriguesauthor=P.+M.+Borralho&title=The+MEK5%2FERK5+signalling+pathway+in+cancer%3A+A+promising+novel+therapeutic+target&doi=10.1016%2Fj.drudis.2016.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The MEK5/ERK5 signalling pathway in cancer: a promising novel therapeutic target</span></div><div class="casAuthors">Simoes, Andre E. S.; Rodrigues, Cecilia M. P.; Borralho, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1654-1663</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Conventional mitogen-activated protein kinase (MAPK) family members are among the most sought-after oncogenic effectors for the development of novel human cancer treatment strategies.  MEK5/ERK5 has been the less-studied MAPK subfamily, despite its increasingly demonstrated relevance in the growth, survival, and differentiation of normal cells.  MEK5/ERK5 signalling has already been proposed to have pivotal roles in several cancer hallmarks, and to mediate the effects of a range of oncogenes.  Accumulating evidence indicates the contribution of MEK5/ERK5 signalling to therapy resistance and the benefits of using MEK5/ERK5 inhibitory strategies in the treatment of human cancer.  Here, we explore the major known contributions of MEK5/ERK5 signalling to the onset and progression of several types of cancer, and highlight the potential clin. relevance of targeting MEK5/ERK5 pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHG6CKquuTJbVg90H21EOLACvtfcHk0ljXiNT_q0VtTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynsbzL&md5=3d73f6af2848b0f78b53cde743cac805</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DSimoes%26aufirst%3DA.%2BE.%26aulast%3DRodrigues%26aufirst%3DC.%2BM.%26aulast%3DBorralho%26aufirst%3DP.%2BM.%26atitle%3DThe%2520MEK5%252FERK5%2520signalling%2520pathway%2520in%2520cancer%253A%2520A%2520promising%2520novel%2520therapeutic%2520target%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1654%26epage%3D1663%26doi%3D10.1016%2Fj.drudis.2016.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Esparis-Ogando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Rodriguez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuste, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandiella, A.</span></span> <span> </span><span class="NLM_article-title">Erk5 participates in neuregulin signal transduction and is constitutively active in breast cancer cells overexpressing ErbB2</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1128/MCB.22.1.270-285.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1128%2FMCB.22.1.270-285.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=11739740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptlOiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=270-285&author=A.+Esparis-Ogandoauthor=E.+Diaz-Rodriguezauthor=J.+C.+Monteroauthor=L.+Yusteauthor=P.+Crespoauthor=A.+Pandiella&title=Erk5+participates+in+neuregulin+signal+transduction+and+is+constitutively+active+in+breast+cancer+cells+overexpressing+ErbB2&doi=10.1128%2FMCB.22.1.270-285.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Erk5 participates in neuregulin signal transduction and is constitutively active in breast cancer cells overexpressing ErbB2</span></div><div class="casAuthors">Esparis-Ogando, Azucena; Diaz-Rodriguez, Elena; Montero, Juan Carlos; Yuste, Laura; Crespo, Piero; Pandiella, Atanasio</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">270-285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The four receptor tyrosine kinases of the ErbB family play essential roles in several physiol. processes and have also been implicated in tumor generation and/or progression.  Activation of ErbB1/EGFR is mainly triggered by epidermal growth factor (EGF) and other related ligands, while activation of ErbB2, ErbB3, and ErbB4 receptors occurs by binding to another set of EGF-like ligands termed neuregulins (NRGs).  Here we show that the Erk5 mitogen-activated protein kinase (MAPK) pathway participates in NRG signal transduction.  In MCF7 cells, NRG activated Erk5 in a time- and dose-dependent fashion.  The action of NRG on Erk5 was dependent on the kinase activity of ErbB receptors but was independent of Ras.  Expression in MCF7 cells of a dominant neg. form of Erk5 resulted in a significant decrease in NRG-induced proliferation of MCF7 cells.  Anal. of Erk5 in several human tumor cell lines indicated that a constitutively active form of this kinase was present in the BT474 and SKBR3 cell lines, which also expressed activated forms of ErbB2, ErbB3, and ErbB4.  Treatments aimed at decreasing the activity of these receptors caused Erk5 inactivation, indicating that the active form of Erk5 present in BT474 and SKBR3 cells was due to a persistent pos. stimulus originating at the ErbB receptors.  In BT474 cells expression of the dominant neg. form of Erk5 resulted in reduced proliferation, indicating that in these cells Erk5 was also involved in the control of proliferation.  Taken together, these results suggest that Erk5 may play a role in the regulation of cell proliferation by NRG receptors and indicate that constitutively active NRG receptors may induce proliferative responses in cancer cells through this MAPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHa2TbZPveLVg90H21EOLACvtfcHk0ljXiNT_q0VtTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptlOiurY%253D&md5=dd1dd2d893f7c5cdc5cf6ee9515ecd7f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.1.270-285.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.1.270-285.2002%26sid%3Dliteratum%253Aachs%26aulast%3DEsparis-Ogando%26aufirst%3DA.%26aulast%3DDiaz-Rodriguez%26aufirst%3DE.%26aulast%3DMontero%26aufirst%3DJ.%2BC.%26aulast%3DYuste%26aufirst%3DL.%26aulast%3DCrespo%26aufirst%3DP.%26aulast%3DPandiella%26aufirst%3DA.%26atitle%3DErk5%2520participates%2520in%2520neuregulin%2520signal%2520transduction%2520and%2520is%2520constitutively%2520active%2520in%2520breast%2520cancer%2520cells%2520overexpressing%2520ErbB2%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D270%26epage%3D285%26doi%3D10.1128%2FMCB.22.1.270-285.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Q.</span></span> <span> </span><span class="NLM_article-title">Identification and validation of dysregulated MAPK7 (ERK5) as a novel oncogenic target in squamous cell lung and esophageal carcinoma</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">454</span>, <span class="refDoi"> DOI: 10.1186/s12885-015-1455-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1186%2Fs12885-015-1455-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=26040563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADC%252BC2MbgtlGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=454&author=P.+R.+Gavineauthor=M.+Wangauthor=D.+Yuauthor=E.+Huauthor=C.+Huangauthor=J.+Xiaauthor=X.+Suauthor=J.+Fanauthor=T.+Zhangauthor=Q.+Yeauthor=L.+Zhengauthor=G.+Zhuauthor=Z.+Qianauthor=Q.+Luoauthor=Y.+Y.+Houauthor=Q.+Ji&title=Identification+and+validation+of+dysregulated+MAPK7+%28ERK5%29+as+a+novel+oncogenic+target+in+squamous+cell+lung+and+esophageal+carcinoma&doi=10.1186%2Fs12885-015-1455-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and validation of dysregulated MAPK7 (ERK5) as a novel oncogenic target in squamous cell lung and esophageal carcinoma</span></div><div class="casAuthors">Gavine Paul R; Wang Mei; Yu Dehua; Hu Eva; Huang Chunlei; Xia Jenny; Su Xinying; Fan Joan; Zhang Tianwei; Ye Qingqing; Zheng Li; Zhu Guanshan; Qian Ziliang; Ji Qunsheng; Luo Qingquan; Hou Ying Yong</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">454</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  MAPK7/ERK5 (extracellular-signal-regulated kinase 5) functions within a canonical three-tiered MAPK (mitogen activated protein kinase) signaling cascade comprising MEK (MAPK/ERK kinase) 5, MEKK(MEK kinase) 2/3 and ERK5 itself.  Despite being the least well studied of the MAPK-modules, evidence supports a role for MAPK7-signaling in the pathology of several cancer types.  METHODS AND RESULTS:  Fluorescence in situ hybridization (FISH) analysis identified MAPK7 gene amplification in 4% (3/74) of non-small cell lung cancers (NSCLC) (enriched to 6% (3/49) in squamous cell carcinoma) and 2% (2/95) of squamous esophageal cancers (sqEC).  Immunohistochemical (IHC) analysis revealed a good correlation between MAPK7 gene amplification and protein expression.  MAPK7 was validated as a proliferative oncogenic driver by performing in vitro siRNA knockdown of MAPK7 in tumor cell lines.  Finally, a novel MEK5/MAPK7 co-transfected HEK293 cell line was developed and used for routine cell-based pharmacodynamic screening.  Phosphorylation antibody microarray analysis also identified novel downstream pharmacodynamic (PD) biomarkers of MAPK7 kinase inhibition in tumor cells (pMEF2A and pMEF2D).  CONCLUSIONS:  Together, these data highlight a broader role for dysregulated MAPK7 in driving tumorigenesis within niche populations of highly prevalent tumor types, and describe current efforts in establishing a robust drug discovery screening cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQObyGhx8V5ZSSm0e6ZOaemfW6udTcc2eYbfOU8JTM7a7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbgtlGktQ%253D%253D&md5=cfbc661ae8f4a71779c8588be6f1de03</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs12885-015-1455-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-015-1455-y%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DHou%26aufirst%3DY.%2BY.%26aulast%3DJi%26aufirst%3DQ.%26atitle%3DIdentification%2520and%2520validation%2520of%2520dysregulated%2520MAPK7%2520%2528ERK5%2529%2520as%2520a%2520novel%2520oncogenic%2520target%2520in%2520squamous%2520cell%2520lung%2520and%2520esophageal%2520carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D454%26doi%3D10.1186%2Fs12885-015-1455-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cason, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuschel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillegass, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacek, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mossman, B. T.</span></span> <span> </span><span class="NLM_article-title">Extracellular signal-regulated kinase 5: A potential therapeutic target for malignant mesotheliomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2071</span>– <span class="NLM_lpage">2083</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-3202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1158%2F1078-0432.CCR-12-3202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=23446998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvFyjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2071-2083&author=A.+Shuklaauthor=J.+M.+Millerauthor=C.+Casonauthor=M.+Sayanauthor=M.+B.+MacPhersonauthor=S.+L.+Beuschelauthor=J.+Hillegassauthor=P.+M.+Vacekauthor=H.+I.+Passauthor=B.+T.+Mossman&title=Extracellular+signal-regulated+kinase+5%3A+A+potential+therapeutic+target+for+malignant+mesotheliomas&doi=10.1158%2F1078-0432.CCR-12-3202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular Signal-Regulated Kinase 5: A Potential Therapeutic Target for Malignant Mesotheliomas</span></div><div class="casAuthors">Shukla, Arti; Miller, Jill M.; Cason, Christopher; Sayan, Mutlay; MacPherson, Maximilian B.; Beuschel, Stacie L.; Hillegass, Jedd; Vacek, Pamela M.; Pass, Harvey I.; Mossman, Brooke T.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2071-2083</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Malignant mesothelioma is a devastating disease with a need for new treatment strategies.  In the present study, we showed the importance of extracellular signal-regulated kinase 5 (ERK5) in malignant mesothelioma tumor growth and treatment.  Exptl. Design: ERK5 as a target for malignant mesothelioma therapy was verified using mesothelial and mesothelioma cell lines as well as by xenograft severe combined immunodeficient (SCID) mouse models.  Results: We first showed that crocidolite asbestos activated ERK5 in LP9 cells and mesothelioma cell lines exhibit constitutive activation of ERK5.  Addn. of doxorubicin resulted in further activation of ERK5 in malignant mesothelioma cells.  ERK5 silencing increased doxorubicin-induced cell death and doxorubicin retention in malignant mesothelioma cells.  In addn., shERK5 malignant mesothelioma lines exhibited both attenuated colony formation on soft agar and invasion of malignant mesothelioma cells in vitro that could be related to modulation of gene expression linked to cell proliferation, apoptosis, migration/invasion, and drug resistance as shown by microarray anal.  Most importantly, injection of shERK5 malignant mesothelioma cell lines into SCID mice showed significant redn. in tumor growth using both s.c. and i.p. models.  Assessment of selected human cytokine profiles in peritoneal lavage fluid from i.p. shERK5 and control tumor-bearing mice showed that ERK5 was crit. in regulation of various proinflammatory (RANTES/CCL5, MCP-1) and angiogenesis-related (interleukin-8, VEGF) cytokines.  Finally, use of doxorubicin and cisplatin in combination with ERK5 inhibition showed further redn. in tumor wt. and vol. in the i.p. model of tumor growth.  Conclusion: ERK5 inhibition in combination with chemotherapeutic drugs is a beneficial strategy for combination therapy in patients with malignant mesothelioma.  Clin Cancer Res; 19(8); 2071-83. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_r7UsUaX-IbVg90H21EOLACvtfcHk0lgoeDC8V385_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvFyjsLo%253D&md5=9ccd4c727165e7c0874ec0dd314cb2fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3202%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DJ.%2BM.%26aulast%3DCason%26aufirst%3DC.%26aulast%3DSayan%26aufirst%3DM.%26aulast%3DMacPherson%26aufirst%3DM.%2BB.%26aulast%3DBeuschel%26aufirst%3DS.%2BL.%26aulast%3DHillegass%26aufirst%3DJ.%26aulast%3DVacek%26aufirst%3DP.%2BM.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DMossman%26aufirst%3DB.%2BT.%26atitle%3DExtracellular%2520signal-regulated%2520kinase%25205%253A%2520A%2520potential%2520therapeutic%2520target%2520for%2520malignant%2520mesotheliomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D2071%26epage%3D2083%26doi%3D10.1158%2F1078-0432.CCR-12-3202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuyoshi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsufuji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniwaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okanoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, T.</span></span> <span> </span><span class="NLM_article-title">ERK5 is a target for gene amplification at 17p11 and promotes cell growth in hepatocellular carcinoma by regulating mitotic entry</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1002/gcc.20624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1002%2Fgcc.20624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=18973138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FhvFyjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=109-120&author=K.+Zenauthor=K.+Yasuiauthor=T.+Nakajimaauthor=Y.+Zenauthor=K.+Zenauthor=Y.+Genauthor=H.+Mitsuyoshiauthor=M.+Minamiauthor=S.+Mitsufujiauthor=S.+Tanakaauthor=Y.+Itohauthor=Y.+Nakanumaauthor=M.+Taniwakiauthor=S.+Ariiauthor=T.+Okanoueauthor=T.+Yoshikawa&title=ERK5+is+a+target+for+gene+amplification+at+17p11+and+promotes+cell+growth+in+hepatocellular+carcinoma+by+regulating+mitotic+entry&doi=10.1002%2Fgcc.20624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ERK5 is a target for gene amplification at 17p11 and promotes cell growth in hepatocellular carcinoma by regulating mitotic entry</span></div><div class="casAuthors">Zen Keika; Yasui Kohichiroh; Nakajima Tomoaki; Zen Yoh; Zen Kan; Gen Yasuyuki; Mitsuyoshi Hironori; Minami Masahito; Mitsufuji Shoji; Tanaka Shinji; Itoh Yoshito; Nakanuma Yasuni; Taniwaki Masafumi; Arii Shigeki; Okanoue Takeshi; Yoshikawa Toshikazu</div><div class="citationInfo"><span class="NLM_cas:title">Genes, chromosomes & cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Using high-density oligonucleotide microarrays, we investigated DNA copy-number aberrations in cell lines derived from hepatocellular carcinomas (HCCs) and detected a novel amplification at 17p11.  To identify the target of amplification at 17p11, we defined the extent of the amplicon and examined HCC cell lines for expression of all seven genes in the 750-kb commonly amplified region.  Mitogen-activated protein kinase (MAPK) 7, which encodes extracellular-regulated protein kinase (ERK) 5, was overexpressed in cell lines in which the gene was amplified.  An increase in MAPK7 copy number was detected in 35 of 66 primary HCC tumors.  Downregulation of MAPK7 by small interfering RNA suppressed the growth of SNU449 cells, the HCC cell line with the greatest amplification and overexpression of MAPK7.  ERK5, phosphorylated during the G2/M phases of the cell cycle, regulated entry into mitosis in SNU449 cells.  In conclusion, our results suggest that MAPK7 is likely the target of 17p11 amplification and that the ERK5 protein product of MAPK7 promotes the growth of HCC cells by regulating mitotic entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcScksBqKJIaWZI8NvIb08rbfW6udTcc2eYXlsfA1BJQg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FhvFyjsg%253D%253D&md5=a86797e020531fb31385555be1e03dd0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fgcc.20624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.20624%26sid%3Dliteratum%253Aachs%26aulast%3DZen%26aufirst%3DK.%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DNakajima%26aufirst%3DT.%26aulast%3DZen%26aufirst%3DY.%26aulast%3DZen%26aufirst%3DK.%26aulast%3DGen%26aufirst%3DY.%26aulast%3DMitsuyoshi%26aufirst%3DH.%26aulast%3DMinami%26aufirst%3DM.%26aulast%3DMitsufuji%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DNakanuma%26aufirst%3DY.%26aulast%3DTaniwaki%26aufirst%3DM.%26aulast%3DArii%26aufirst%3DS.%26aulast%3DOkanoue%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DT.%26atitle%3DERK5%2520is%2520a%2520target%2520for%2520gene%2520amplification%2520at%252017p11%2520and%2520promotes%2520cell%2520growth%2520in%2520hepatocellular%2520carcinoma%2520by%2520regulating%2520mitotic%2520entry%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2009%26volume%3D48%26spage%3D109%26epage%3D120%26doi%3D10.1002%2Fgcc.20624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a benzo[e]pyrimido-[5,4-b][1,4]diazepin-6(11H)-one as a potent and selective inhibitor of Big MAP Kinase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1021/ml100304b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100304b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=195-200&author=X.+Dengauthor=Q.+Yangauthor=N.+Kwiatkowskiauthor=T.+Simauthor=U.+McDermottauthor=J.+E.+Settlemanauthor=J.+D.+Leeauthor=N.+S.+Gray&title=Discovery+of+a+benzo%5Be%5Dpyrimido-%5B5%2C4-b%5D%5B1%2C4%5Ddiazepin-6%2811H%29-one+as+a+potent+and+selective+inhibitor+of+Big+MAP+Kinase+1&doi=10.1021%2Fml100304b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml100304b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100304b%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DSettleman%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520benzo%255Be%255Dpyrimido-%255B5%252C4-b%255D%255B1%252C4%255Ddiazepin-6%252811H%2529-one%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520Big%2520MAP%2520Kinase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D195%26epage%3D200%26doi%3D10.1021%2Fml100304b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fearns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of BMK1 suppresses tumor growth through promyelocytic leukemia protein</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.08.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.ccr.2010.08.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=20832753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=258-267&author=Q.+Yangauthor=X.+Dengauthor=B.+Luauthor=M.+Cameronauthor=C.+Fearnsauthor=M.+P.+Patricelliauthor=J.+R.+Yatesauthor=N.+S.+Grayauthor=J.+D.+Lee&title=Pharmacological+inhibition+of+BMK1+suppresses+tumor+growth+through+promyelocytic+leukemia+protein&doi=10.1016%2Fj.ccr.2010.08.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Inhibition of BMK1 Suppresses Tumor Growth through Promyelocytic Leukemia Protein</span></div><div class="casAuthors">Yang, Qing-Kai; Deng, Xian-Ming; Lu, Bing-Wen; Cameron, Michael; Fearns, Colleen; Patricelli, Matthew P.; Yates, John R., III; Gray, Nathanael S.; Lee, Jiing-Dwan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">258-267</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BMK1 is activated by mitogens and oncogenic signals and, thus, is strongly implicated in tumorigenesis.  We found that BMK1 interacted with promyelocytic leukemia protein (PML), and inhibited its tumor-suppressor function through phosphorylation.  Furthermore, activated BMK1 notably inhibited PML-dependent activation of p21.  To further investigate the BMK-mediated inhibition of the tumor suppressor activity of PML in tumor cells, we developed a small-mol. inhibitor of the kinase activity of BMK1, XMD8-92.  Inhibition of BMK1 by XMD8-92 blocked tumor cell proliferation in vitro and significantly inhibited tumor growth in vivo by 95%, demonstrating the efficacy and tolerability of BMK1-targeted cancer treatment in animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog5sdylMRjmrVg90H21EOLACvtfcHk0ljF7m720gaVkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bK&md5=45900e7ef5abcb99d22c91ac94b561d1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.08.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.08.008%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DCameron%26aufirst%3DM.%26aulast%3DFearns%26aufirst%3DC.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26atitle%3DPharmacological%2520inhibition%2520of%2520BMK1%2520suppresses%2520tumor%2520growth%2520through%2520promyelocytic%2520leukemia%2520protein%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D258%26epage%3D267%26doi%3D10.1016%2Fj.ccr.2010.08.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amantea, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidique, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarich, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, J. S.</span></span> <span> </span><span class="NLM_article-title">ERK5 kinase activity is dispensable for cellular immune response and proliferation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">11865</span>– <span class="NLM_lpage">11870</span>, <span class="refDoi"> DOI: 10.1073/pnas.1609019113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1073%2Fpnas.1609019113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=11865-11870&author=E.+C.+Linauthor=C.+M.+Amanteaauthor=T.+K.+Nomanbhoyauthor=H.+Weissigauthor=J.+Ishiyamaauthor=Y.+Huauthor=S.+Sidiqueauthor=B.+Liauthor=J.+W.+Kozarichauthor=J.+S.+Rosenblum&title=ERK5+kinase+activity+is+dispensable+for+cellular+immune+response+and+proliferation&doi=10.1073%2Fpnas.1609019113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1609019113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1609019113%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DE.%2BC.%26aulast%3DAmantea%26aufirst%3DC.%2BM.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DIshiyama%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSidique%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26aulast%3DRosenblum%26aufirst%3DJ.%2BS.%26atitle%3DERK5%2520kinase%2520activity%2520is%2520dispensable%2520for%2520cellular%2520immune%2520response%2520and%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D11865%26epage%3D11870%26doi%3D10.1073%2Fpnas.1609019113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lueth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loewe, W.</span></span> <span> </span><span class="NLM_article-title">Syntheses of 4-(indole-3-yl)quinazolines - A new class of epidermal growth factor receptor tyrosine kinase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1478</span>– <span class="NLM_lpage">1488</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2007.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.ejmech.2007.09.018" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2008&pages=1478-1488&author=A.+Luethauthor=W.+Loewe&title=Syntheses+of+4-%28indole-3-yl%29quinazolines+-+A+new+class+of+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.ejmech.2007.09.018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2007.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2007.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DLueth%26aufirst%3DA.%26aulast%3DLoewe%26aufirst%3DW.%26atitle%3DSyntheses%2520of%25204-%2528indole-3-yl%2529quinazolines%2520-%2520A%2520new%2520class%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2008%26volume%3D43%26spage%3D1478%26epage%3D1488%26doi%3D10.1016%2Fj.ejmech.2007.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortlock, A. A.</span>; <span class="NLM_string-name">Keen, N. J.</span></span> <span> </span><span class="NLM_article-title">Quinazoline Derivatives</span>. <span class="NLM_patent">EP1218357 B1</span>, April 6, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=A.+A.+Mortlock&author=N.+J.+Keen&title=Quinazoline+Derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMortlock%26aufirst%3DA.%2BA.%26atitle%3DQuinazoline%2520Derivatives%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="note"><p class="first last">For synthesis of <b>43</b>, see:</p></div><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span>; <span class="NLM_string-name">Low, J. D.</span>; <span class="NLM_string-name">Allen, J. R.</span>; <span class="NLM_string-name">Chen, J.</span>; <span class="NLM_string-name">Chen, N.</span>; <span class="NLM_string-name">Cheng, Y.</span>; <span class="NLM_string-name">Judd, T.</span>; <span class="NLM_string-name">Liu, Q.</span>; <span class="NLM_string-name">Lopez, P.</span>; <span class="NLM_string-name">Qian, W.</span>; <span class="NLM_string-name">Rumfelt, S.</span>; <span class="NLM_string-name">Rzasa, R. M.</span>; <span class="NLM_string-name">Tamayo, N. A.</span>; <span class="NLM_string-name">Xue, Q.</span>; <span class="NLM_string-name">Yang, B.</span>; <span class="NLM_string-name">Zhong, W.</span></span> <span> </span><span class="NLM_article-title">Perfluorinated 5,6-Dihydro-4H-1,3-oxazin-2-amine Compounds as Beta-Secretase Inhibitors and Methods of Use</span>. <span class="NLM_patent">WO2014/134341 A1</span>, September 4, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+E.+Minatti&author=J.+D.+Low&author=J.+R.+Allen&author=J.+Chen&author=N.+Chen&author=Y.+Cheng&author=T.+Judd&author=Q.+Liu&author=P.+Lopez&author=W.+Qian&author=S.+Rumfelt&author=R.+M.+Rzasa&author=N.+A.+Tamayo&author=Q.+Xue&author=B.+Yang&author=W.+Zhong&title=Perfluorinated+5%2C6-Dihydro-4H-1%2C3-oxazin-2-amine+Compounds+as+Beta-Secretase+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26atitle%3DPerfluorinated%25205%252C6-Dihydro-4H-1%252C3-oxazin-2-amine%2520Compounds%2520as%2520Beta-Secretase%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erazo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Babot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Asiain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrice, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espadamala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayascas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizcano, J. M.</span></span> <span> </span><span class="NLM_article-title">Canonical and kinase activity-independent mechanisms for extracellular signal-regulated kinase 5 (ERK5) nuclear translocation require dissociation of Hsp90 from the ERK5-Cdc37 complex</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1128/MCB.01246-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1128%2FMCB.01246-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=23428871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslajt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1671-1686&author=T.+Erazoauthor=A.+Morenoauthor=G.+Ruiz-Babotauthor=A.+Rodriguez-Asiainauthor=N.+A.+Morriceauthor=J.+Espadamalaauthor=J.+R.+Bayascasauthor=N.+Gomezauthor=J.+M.+Lizcano&title=Canonical+and+kinase+activity-independent+mechanisms+for+extracellular+signal-regulated+kinase+5+%28ERK5%29+nuclear+translocation+require+dissociation+of+Hsp90+from+the+ERK5-Cdc37+complex&doi=10.1128%2FMCB.01246-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical and kinase activity-independent mechanisms for extracellular signal-regulated kinase 5 (ERK5) nuclear translocation require dissociation of Hsp90 from the ERK5-Cdc37 complex</span></div><div class="casAuthors">Erazo, Tatiana; Moreno, Ana; Ruiz-Babot, Gerard; Rodriguez-Asiain, Arantza; Morrice, Nicholas A.; Espadamala, Josep; Bayascas, Jose R.; Gomez, Nestor; Lizcano, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1671-1686</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein (MAP) kinase extracellular signal-regulated kinase 5 (ERK5) plays a crucial role in cell proliferation, regulating gene transcription.  ERK5 has a unique C-terminal tail which contains a transcriptional activation domain, and activates transcription by phosphorylating transcription factors and acting itself as a transcriptional coactivator.  However, the mol. mechanisms that regulate its nucleocytoplasmic traffic are unknown.  We have used tandem affinity purifn. to identify proteins that interact with ERK5.  We show that ERK5 interacts with the Hsp90-Cdc37 chaperone in resting cells, and that inhibition of Hsp90 or Cdc37 results in ERK5 ubiquitylation and proteasomal degrdn.  Interestingly, activation of cellular ERK5 induces Hsp90 dissocn. from the ERK5-Cdc37 complex, leading to ERK5 nuclear translocation and activation of transcription, by a mechanism which requires the autophosphorylation at its C-terminal tail.  Consequently, active ERK5 is no longer sensitive to Hsp90 or Cdc37 inhibitors.  Cdc37 overexpression also induces Hsp90 dissocn. and the nuclear translocation of a kinase-inactive form of ERK5 which retains transcriptional activity.  This is the first example showing that ERK5 transcriptional activity does not require kinase activity.  Since Cdc37 cooperates with ERK5 to promote cell proliferation, Cdc37 overexpression (as happens in some cancers) might represent a new, noncanonical mechanism by which ERK5 regulates tumor proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8nP0WXDHSqrVg90H21EOLACvtfcHk0ljF7m720gaVkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslajt70%253D&md5=fd610778cfd77357213571583bf7a80b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FMCB.01246-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01246-12%26sid%3Dliteratum%253Aachs%26aulast%3DErazo%26aufirst%3DT.%26aulast%3DMoreno%26aufirst%3DA.%26aulast%3DRuiz-Babot%26aufirst%3DG.%26aulast%3DRodriguez-Asiain%26aufirst%3DA.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DEspadamala%26aufirst%3DJ.%26aulast%3DBayascas%26aufirst%3DJ.%2BR.%26aulast%3DGomez%26aufirst%3DN.%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26atitle%3DCanonical%2520and%2520kinase%2520activity-independent%2520mechanisms%2520for%2520extracellular%2520signal-regulated%2520kinase%25205%2520%2528ERK5%2529%2520nuclear%2520translocation%2520require%2520dissociation%2520of%2520Hsp90%2520from%2520the%2520ERK5-Cdc37%2520complex%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2013%26volume%3D33%26spage%3D1671%26epage%3D1686%26doi%3D10.1128%2FMCB.01246-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kravchenko, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapping, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulevitch, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span> <span> </span><span class="NLM_article-title">BMK1/ERK5 regulates serum-induced early gene expression through transcription factor MEF2C</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">7054</span>– <span class="NLM_lpage">7066</span>, <span class="refDoi"> DOI: 10.1093/emboj/16.23.7054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1093%2Femboj%2F16.23.7054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=7054-7066&author=Y.+Katoauthor=V.+V.+Kravchenkoauthor=R.+I.+Tappingauthor=J.+Hanauthor=R.+J.+Ulevitchauthor=J.+D.+Lee&title=BMK1%2FERK5+regulates+serum-induced+early+gene+expression+through+transcription+factor+MEF2C&doi=10.1093%2Femboj%2F16.23.7054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F16.23.7054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F16.23.7054%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DY.%26aulast%3DKravchenko%26aufirst%3DV.%2BV.%26aulast%3DTapping%26aufirst%3DR.%2BI.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DUlevitch%26aufirst%3DR.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26atitle%3DBMK1%252FERK5%2520regulates%2520serum-induced%2520early%2520gene%2520expression%2520through%2520transcription%2520factor%2520MEF2C%26jtitle%3DEMBO%2520J.%26date%3D1997%26volume%3D16%26spage%3D7054%26epage%3D7066%26doi%3D10.1093%2Femboj%2F16.23.7054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span> <span> </span><span class="NLM_article-title">Challenges in validating candidate therapeutic targets</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e32402</span>, <span class="refDoi"> DOI: 10.7554/eLife.32402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.7554%2FeLife.32402" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e32402&author=J.+Settlemanauthor=C.+L.+Sawyersauthor=T.+Hunter&title=Challenges+in+validating+candidate+therapeutic+targets&doi=10.7554%2FeLife.32402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.32402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.32402%26sid%3Dliteratum%253Aachs%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26aulast%3DHunter%26aufirst%3DT.%26atitle%3DChallenges%2520in%2520validating%2520candidate%2520therapeutic%2520targets%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26spage%3De32402%26doi%3D10.7554%2FeLife.32402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trésaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Novel class of potent and cellularly active inhibitors devalidates MTH1 as broad-spectrum cancer target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstr%C3%B6mauthor=C.+Silvanderauthor=L.+Tr%C3%A9sauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=B.+Bauserauthor=A.+Haegebarthauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Novel+class+of+potent+and+cellularly+active+inhibitors+devalidates+MTH1+as+broad-spectrum+cancer+target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span></div><div class="casAuthors">Ellermann, Manuel; Eheim, Ashley; Rahm, Fredrik; Viklund, Jenny; Guenther, Judith; Andersson, Martin; Ericsson, Ulrika; Forsblom, Rickard; Ginman, Tobias; Lindstroem, Johan; Silvander, Camilla; Tresaugues, Lionel; Giese, Anja; Bunse, Stefanie; Neuhaus, Roland; Weiske, Joerg; Quanz, Maria; Glasauer, Andrea; Nowak-Reppel, Katrin; Bader, Benjamin; Irlbacher, Horst; Meyer, Hanna; Queisser, Nina; Bauser, Marcus; Haegebarth, Andrea; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1986-1992</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MTH1 is a hydrolase responsible for sanitization of oxidized purine nucleoside triphosphates to prevent their incorporation into replicating DNA.  Early tool compds. published in the literature inhibited the enzymic activity of MTH1 and subsequently induced cancer cell death; however recent studies have questioned the reported link between these two events.  Therefore, it is important to validate MTH1 as a cancer dependency with high quality chem. probes.  Here, we present BAY-707, a substrate-competitive, highly potent and selective inhibitor of MTH1, chem. distinct compared to those previously published.  Despite superior cellular target engagement and pharmacokinetic properties, inhibition of MTH1 with BAY-707 resulted in a clear lack of in vitro or in vivo anticancer efficacy either in mono- or in combination therapies.  Therefore, we conclude that MTH1 is dispensable for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHT2Ivo0Gx87Vg90H21EOLACvtfcHk0li_AWcIT19evQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP&md5=69aa2247fae019cf695df8352e2ade18</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTr%25C3%25A9saugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DB.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DNovel%2520class%2520of%2520potent%2520and%2520cellularly%2520active%2520inhibitors%2520devalidates%2520MTH1%2520as%2520broad-spectrum%2520cancer%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span>Structure
of BAY-885 to be published on <a href="https://www.thesgc.org/donated-chemical-probes" class="extLink">https://www.thesgc.org/donated-chemical-probes</a>; <a href="https://innovate.bayer.com/what-we-offer/chemical-probes-for-open-science/" class="extLink">https://innovate.bayer.com/what-we-offer/chemical-probes-for-open-science/</a> (accessed Nov, 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Structure%0Aof+BAY-885+to+be+published+on+https%3A%2F%2Fwww.thesgc.org%2Fdonated-chemical-probes%3B+https%3A%2F%2Finnovate.bayer.com%2Fwhat-we-offer%2Fchemical-probes-for-open-science%2F+%28accessed+Nov%2C+15%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">A chemical biology view of bioactive small molecules and a binder-based approach to connect biology to precision medicines</span>.  <i>Isr. J. Chem.</i><span class="NLM_year" style="font-weight: bold;">2018</span><span class="refDoi"> DOI: 10.1002/ijch.201800113</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1002%2Fijch.201800113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=S.+L.+Schreiber&title=A+chemical+biology+view+of+bioactive+small+molecules+and+a+binder-based+approach+to+connect+biology+to+precision+medicines&doi=10.1002%2Fijch.201800113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fijch.201800113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijch.201800113%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520chemical%2520biology%2520view%2520of%2520bioactive%2520small%2520molecules%2520and%2520a%2520binder-based%2520approach%2520to%2520connect%2520biology%2520to%2520precision%2520medicines%26jtitle%3DIsr.%2520J.%2520Chem.%26date%3D2018%26doi%3D10.1002%2Fijch.201800113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A.</span></span> Oral presentation at AACR <span class="NLM_year">2018</span>, Chicago, Illinois, USA. <span> </span><span class="NLM_article-title">Inhibitor combinations targeting KRAS effector signaling in KRAS-mutant pancreatic cancer</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+Cox&title=Inhibitor+combinations+targeting+KRAS+effector+signaling+in+KRAS-mutant+pancreatic+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%26date%3D2018%26atitle%3DInhibitor%2520combinations%2520targeting%2520KRAS%2520effector%2520signaling%2520in%2520KRAS-mutant%2520pancreatic%2520cancer" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: An emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+An+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0li_AWcIT19evQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberspächer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmees, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span> <span> </span><span class="NLM_article-title">Affinity map of bromo-domain protein 4 (BRD4) interactions with the histone H4 tail and the small molecule inhibitor JQ1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9304</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.523019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1074%2Fjbc.M113.523019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9304-9319&author=M.+Jungauthor=M.+Philpottauthor=S.+M%C3%BCllerauthor=J.+Schulzeauthor=V.+Badockauthor=U.+Ebersp%C3%A4cherauthor=D.+Moosmayerauthor=B.+Baderauthor=N.+Schmeesauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=B.+Haendler&title=Affinity+map+of+bromo-domain+protein+4+%28BRD4%29+interactions+with+the+histone+H4+tail+and+the+small+molecule+inhibitor+JQ1&doi=10.1074%2Fjbc.M113.523019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DEbersp%25C3%25A4cher%26aufirst%3DU.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DSchmees%26aufirst%3DN.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHaendler%26aufirst%3DB.%26atitle%3DAffinity%2520map%2520of%2520bromo-domain%2520protein%25204%2520%2528BRD4%2529%2520interactions%2520with%2520the%2520histone%2520H4%2520tail%2520and%2520the%2520small%2520molecule%2520inhibitor%2520JQ1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9304%26epage%3D9319%26doi%3D10.1074%2Fjbc.M113.523019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit22a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span> <span> </span><span class="NLM_article-title">Cell-based protein stabilization assays for the detection of interactions between small-molecule inhibitors and BRD4</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1177/1087057114552398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1177%2F1087057114552398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=25266565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWrurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=180-189&author=J.+Schulzeauthor=D.+Moosmayerauthor=J.+Weiskeauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=C.+Herbstauthor=M.+Jungauthor=B.+Haendlerauthor=B.+Bader&title=Cell-based+protein+stabilization+assays+for+the+detection+of+interactions+between+small-molecule+inhibitors+and+BRD4&doi=10.1177%2F1087057114552398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-based protein stabilization assays for the detection of interactions between small-molecule inhibitors and BRD4</span></div><div class="casAuthors">Schulze, Jessica; Moosmayer, Dieter; Weiske, Joerg; Fernandez-Montalvan, Amaury; Herbst, Christopher; Jung, Marie; Haendler, Bernard; Bader, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">180-189/1-180-189/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4), a member of the bromodomain and extra-terminal (BET) protein family, acts as a central element in transcriptional elongation and plays essential roles in cell proliferation.  Inhibition of BRD4 binding to acetylated histone tails via its two bromodomains, BD1 and BD2, with small-mol. inhibitors has been shown to be a valid strategy to prevent cancer growth.  We have evaluated and established two novel assays that quantify the interaction of transfected BRD4 BD1 with chem. inhibitors inside cultured cells.  Both methods are based on the principle of ligand-induced protein stabilization by which the binding of a small-mol. inhibitor stabilizes intracellular BRD4 BD1 and protects it from proteolytic degrdn.  We demonstrate the universal character of this principle by using two orthogonal, highly sensitive detection technologies for the quantification of BRD4 BD1 levels in cellular lysates: enzyme fragment complementation and time-resolved fluorescence resonance energy transfer (TR-FRET).  Upon optimization of both assays to a miniaturized high-throughput format, the methods were validated by testing a set of small-mol. BET inhibitors and comparing the results with those from a cell-free binding assay and a biophys. thermal shift assay.  In addn., point mutations were introduced into BRD4 BD1, and the corresponding mutants were characterized in the TR-FRET stabilization assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCzgkcUeBYVbVg90H21EOLACvtfcHk0lgWI6Wrk5oLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWrurY%253D&md5=283b113f600aa455b6859ef21b7515c1</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1177%2F1087057114552398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057114552398%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHerbst%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DBader%26aufirst%3DB.%26atitle%3DCell-based%2520protein%2520stabilization%2520assays%2520for%2520the%2520detection%2520of%2520interactions%2520between%2520small-molecule%2520inhibitors%2520and%2520BRD4%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D180%26epage%3D189%26doi%3D10.1177%2F1087057114552398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogg, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel allosteric inhibitor-binding site in ERK5: Comparison with the canonical kinase ATP-binding site</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>D72</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1107/S2059798316004502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1107%2FS2059798316004502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D72&publication_year=2016&pages=682-693&author=H.+Chenauthor=J.+Tuckerauthor=X.+Wangauthor=P.+R.+Gavineauthor=C.+Phillipsauthor=M.+A.+Augustinauthor=P.+Schreinerauthor=S.+Steinbacherauthor=M.+Prestonauthor=D.+Ogg&title=Discovery+of+a+novel+allosteric+inhibitor-binding+site+in+ERK5%3A+Comparison+with+the+canonical+kinase+ATP-binding+site&doi=10.1107%2FS2059798316004502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS2059798316004502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316004502%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DAugustin%26aufirst%3DM.%2BA.%26aulast%3DSchreiner%26aufirst%3DP.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DOgg%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520allosteric%2520inhibitor-binding%2520site%2520in%2520ERK5%253A%2520Comparison%2520with%2520the%2520canonical%2520kinase%2520ATP-binding%2520site%26jtitle%3DActa%2520Crystallogr.%26date%3D2016%26volume%3DD72%26spage%3D682%26epage%3D693%26doi%3D10.1107%2FS2059798316004502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4.</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4.&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makielski, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Properties of hERG channels stably expressed in HEK293 cells studied at physiological temperature</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/S0006-3495(98)77782-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2FS0006-3495%2898%2977782-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1998&pages=230-241&author=Z.+F.+Zhouauthor=Q.+Gongauthor=B.+Yeauthor=Z.+Fanauthor=J.+C.+Makielskiauthor=G.+A.+Robertsonauthor=C.+T.+January&title=Properties+of+hERG+channels+stably+expressed+in+HEK293+cells+studied+at+physiological+temperature&doi=10.1016%2FS0006-3495%2898%2977782-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2898%2977782-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252898%252977782-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%2BF.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DYe%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DMakielski%26aufirst%3DJ.%2BC.%26aulast%3DRobertson%26aufirst%3DG.%2BA.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DProperties%2520of%2520hERG%2520channels%2520stably%2520expressed%2520in%2520HEK293%2520cells%2520studied%2520at%2520physiological%2520temperature%26jtitle%3DBiophys.%2520J.%26date%3D1998%26volume%3D74%26spage%3D230%26epage%3D241%26doi%3D10.1016%2FS0006-3495%2898%2977782-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M. C.</span>, <span class="NLM_string-name">Chandy, K. G.</span>, <span class="NLM_string-name">Grissme, R. S.</span>, <span class="NLM_string-name">Gutman, G. A.</span>, <span class="NLM_string-name">Lazdunski, M.</span>, <span class="NLM_string-name">Mckinnon, D.</span>, <span class="NLM_string-name">Pardo, L. A.</span>, <span class="NLM_string-name">Robertson, G. A.</span>, <span class="NLM_string-name">Rudy, B.</span>, <span class="NLM_string-name">Stuehmer, W.</span>, <span class="NLM_string-name">Wang, X. L.</span></span> <span> </span><span class="NLM_article-title">Voltage-gated potassium channels: Kv11.1 (hERG)</span>. IUPHAR/BPS <i>Guide to Pharmacology</i>, <a href="http://www.guidetopharmacology.org" class="extLink">http://www.guidetopharmacology.org</a> (accessed Nov, 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+C.+Sanguinetti&author=K.+G.+Chandy&author=R.+S.+Grissme&author=G.+A.+Gutman&author=M.+Lazdunski&author=D.+Mckinnon&author=L.+A.+Pardo&author=G.+A.+Robertson&author=B.+Rudy&author=W.+Stuehmer&author=X.+L.+Wang&title=Voltage-gated+potassium+channels%3A+Kv11.1+%28hERG%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26atitle%3DVoltage-gated%2520potassium%2520channels%253A%2520Kv11.1%2520%2528hERG%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamill, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigworth, F. J.</span></span> <span> </span><span class="NLM_article-title">Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches</span>. <i>Pfluegers Arch.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1007/BF00656997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1007%2FBF00656997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=6270629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADyaL38%252FjslWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1981&pages=85-100&author=O.+P.+Hamillauthor=A.+Martyauthor=E.+Neherauthor=B.+Sakmannauthor=F.+J.+Sigworth&title=Improved+patch-clamp+techniques+for+high-resolution+current+recording+from+cells+and+cell-free+membrane+patches&doi=10.1007%2FBF00656997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches</span></div><div class="casAuthors">Hamill O P; Marty A; Neher E; Sakmann B; Sigworth F J</div><div class="citationInfo"><span class="NLM_cas:title">Pflugers Archiv : European journal of physiology</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    </div><div class="casAbstract">1.  The extracellular patch clamp method, which first allowed the detection of single channel currents in biological membranes, has been further refined to enable higher current resolution, direct membrane patch potential control, and physical isolation of membrane patches. 2.  A description of a convenient method for the fabrication of patch recording pipettes is given together with procedures followed to achieve giga-seals i.e. pipette-membrane seals with resistances of 10(9) - 10(11) omega. 3.  The basic patch clamp recording circuit, and designs for improved frequency response are described along with the present limitations in recording the currents from single channels. 4.  Procedures for preparation and recording from three representative cell types are given.  Some properties of single acetylcholine-activated channels in muscle membrane are described to illustrate the improved current and time resolution achieved with giga-seals. 5.  A description is given of the various ways that patches of membrane can be physically isolated from cells.  This isolation enables the recording of single channel currents with well-defined solutions on both sides of the membrane.  Two types of isolated cell-free patch configurations can be formed: an inside-out patch with its cytoplasmic membrane face exposed to the bath solution, and an outside-out patch with its extracellular membrane face exposed to the bath solution. 6.  The application of the method for the recording of ionic currents and internal dialysis of small cells is considered.  Single channel resolution can be achieved when recording from whole cells, if the cell diameter is small (less than 20 micrometer). 7.  The wide range of cell types amenable to giga-seal formation is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTk3IMHMxuoQXy-c-E1v-fmfW6udTcc2eYQ7VkgXYd7Crntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL38%252FjslWrug%253D%253D&md5=c981295f57b3239ca3719093b059f44d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00656997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00656997%26sid%3Dliteratum%253Aachs%26aulast%3DHamill%26aufirst%3DO.%2BP.%26aulast%3DMarty%26aufirst%3DA.%26aulast%3DNeher%26aufirst%3DE.%26aulast%3DSakmann%26aufirst%3DB.%26aulast%3DSigworth%26aufirst%3DF.%2BJ.%26atitle%3DImproved%2520patch-clamp%2520techniques%2520for%2520high-resolution%2520current%2520recording%2520from%2520cells%2520and%2520cell-free%2520membrane%2520patches%26jtitle%3DPfluegers%2520Arch.%26date%3D1981%26volume%3D391%26spage%3D85%26epage%3D100%26doi%3D10.1007%2FBF00656997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, H.
M.</span></span> <span> </span><span class="NLM_article-title">Suitability of commonly used excipients for electrophysiological in-vitro safety pharmacology assessment of effects on hERG potassium current and on rabbit Purkinje fiber action potential</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">158</span>, <span class="refDoi"> DOI: 10.1016/j.vascn.2007.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.vascn.2007.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=17590357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCmtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=145-158&author=H.%0AM.+Himmel&title=Suitability+of+commonly+used+excipients+for+electrophysiological+in-vitro+safety+pharmacology+assessment+of+effects+on+hERG+potassium+current+and+on+rabbit+Purkinje+fiber+action+potential&doi=10.1016%2Fj.vascn.2007.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Suitability of commonly used excipients for electrophysiological in-vitro safety pharmacology assessment of effects on hERG potassium current and on rabbit Purkinje fiber action potential</span></div><div class="casAuthors">Himmel, Herbert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Introduction: Regulatory guidelines require investigation of the liability for delayed ventricular repolarization by new chem. entities within a broad concn. range in-vitro.  However, investigation can be limited by poor drug aq. soly., and by solvent physicochem. attributes that disrupt cell membrane integrity.  Although excipients or solubilizing agents may aid to achieve the necessary high concns., no comprehensive overview on the suitability of solvents for in-vitro electrophysiol. safety studies exists.  Methods: Excipients were tested for potential interference with the hERG (human ether-a-go-go-related gene) K+ current (whole-cell voltage-clamp, 23±2°), and the shape of rabbit Purkinje fiber action potentials (conventional glass microelectrode technique, 37±1°).  Results and discussion: Water-sol. complexation builders/carriers had little effect on hERG K+ current at up to 50 mg/mL (BSA, bovine serum albumin) and 11 mg/mL (HP-β-CD, hydroxypropyl-β-cyclodextrin; IC20, concn. of 20% inhibition).  Water-sol. org. (co)solvents inhibited hERG K+ currents (IC20, %/mM): 0.7/152, ethanol; 0.9/67, Transcutol; 1.2/154, DMSO (dimethylsulfoxide); 1.6/389, acetonitrile; 1.9/48, polyethylene glycol 400; 2.1/660, methanol.  Part of their inhibitory effect is attributed to the osmolality of extracellular solns., because hERG IC20 and extrapolated osmolality at the hERG IC20 strongly correlate.  Water-sol. nonionic solubilizers/surfactants are potent inhibitors of hERG K+ current with IC20 concns. of 0.07% (Cremophor EL) or lower (Tween 20, Tween 80: ∼ 0.001%).  Part of this inhibitory effect is attributed to their interaction with lipid membranes, because hERG inhibition occurs close to crit. micelle concns. (Cremophor, ∼0.009%; Tween 20, ∼0.007%).  Purkinje fiber action potentials are little affected by HP-β-CD at up to 2 mg/mL, while DMSO tends to shorten the action potential duration at 1%.  Conclusion: When conducting electrophysiol. in-vitro assessments of drug effects, solubilizers/surfactants (Cremophor EL, Tween 20, Tween 80) should be avoided.  Instead, water-sol. org. (co)solvents (methanol, acetonitrile, DMSO) or complexation builders/carriers (HP-β-CD, BSA) appear to be more favorable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsEAdKY44zz7Vg90H21EOLACvtfcHk0lir1fdAMXd6Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCmtLbN&md5=49c61123b2fd0c35fcae026a5d45977f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2007.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2007.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DHimmel%26aufirst%3DH.%2BM.%26atitle%3DSuitability%2520of%2520commonly%2520used%2520excipients%2520for%2520electrophysiological%2520in-vitro%2520safety%2520pharmacology%2520assessment%2520of%2520effects%2520on%2520hERG%2520potassium%2520current%2520and%2520on%2520rabbit%2520Purkinje%2520fiber%2520action%2520potential%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2007%26volume%3D56%26spage%3D145%26epage%3D158%26doi%3D10.1016%2Fj.vascn.2007.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span>, <span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Solution Stability Methods</span>. In  <i>Drug-like Properties: Concepts, Structure Design and Methods</i>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Burlington, MA</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">356</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2FB978-012369520-8.50032-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=353-356&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FB978-012369520-8.50032-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-012369520-8.50032-2%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DSolution%2520Stability%2520Methods%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%26pub%3DAcademic%2520Press%26date%3D2008%26spage%3D353%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Onofrey, T.</span>, <span class="NLM_string-name">Kazan, G.</span></span> <span> </span><span class="NLM_article-title">Performance and correlation of a 96-well high throughput screening method to determine aqueous drug solubility</span>; Millipore Corporation Application Note, <span class="NLM_year">2003</span>; Lit. No. AN1731EN00.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=T.+Onofrey&author=G.+Kazan&title=Performance+and+correlation+of+a+96-well+high+throughput+screening+method+to+determine+aqueous+drug+solubility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DOnofrey%26aufirst%3DT.%26atitle%3DPerformance%2520and%2520correlation%2520of%2520a%252096-well%2520high%2520throughput%2520screening%2520method%2520to%2520determine%2520aqueous%2520drug%2520solubility%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minick, D.J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenz, J.H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, M.A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brent, D.A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive method for determining hydrophobicity constants by reversed-phase high performance liquid chromatography</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1021/jm00118a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00118a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1923-1933&author=D.J.+Minickauthor=J.H.+Frenzauthor=M.A.+Patrickauthor=D.A.+Brent&title=A+comprehensive+method+for+determining+hydrophobicity+constants+by+reversed-phase+high+performance+liquid+chromatography&doi=10.1021%2Fjm00118a010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm00118a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00118a010%26sid%3Dliteratum%253Aachs%26aulast%3DMinick%26aufirst%3DD.J.%26aulast%3DFrenz%26aufirst%3DJ.H.%26aulast%3DPatrick%26aufirst%3DM.A.%26aulast%3DBrent%26aufirst%3DD.A.%26atitle%3DA%2520comprehensive%2520method%2520for%2520determining%2520hydrophobicity%2520constants%2520by%2520reversed-phase%2520high%2520performance%2520liquid%2520chromatography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1923%26epage%3D1933%26doi%3D10.1021%2Fjm00118a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17','cit17a'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22','cit22a'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25','cit25a'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 17 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yoseph Atilaw, Vasanthanathan Poongavanam, Caroline Svensson Nilsson, Duy Nguyen, Anja Giese, Daniel Meibom, Mate Erdelyi, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Solution Conformations Shed Light on PROTAC Cell Permeability. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 107-114. <a href="https://doi.org/10.1021/acsmedchemlett.0c00556" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00556</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00556%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSolution%252BConformations%252BShed%252BLight%252Bon%252BPROTAC%252BCell%252BPermeability%26aulast%3DAtilaw%26aufirst%3DYoseph%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16102020%26date%3D18122020%26date%3D25122020%26volume%3D12%26issue%3D1%26spage%3D107%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Guan Wang, Yuqian Zhao, Yao Liu, Dejuan Sun, Yongqi Zhen, Jie Liu, Leilei Fu, Lan Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Novel Dual-Target Inhibitor of ERK1 and ERK5 That Induces Regulated Cell Death to Overcome Compensatory Mechanism in Specific Tumor Types. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 3976-3995. <a href="https://doi.org/10.1021/acs.jmedchem.9b01896" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01896</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01896%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNovel%252BDual-Target%252BInhibitor%252Bof%252BERK1%252Band%252BERK5%252BThat%252BInduces%252BRegulated%252BCell%252BDeath%252Bto%252BOvercome%252BCompensatory%252BMechanism%252Bin%252BSpecific%252BTumor%252BTypes%26aulast%3DWang%26aufirst%3DGuan%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D14112019%26date%3D05032020%26date%3D20022020%26volume%3D63%26issue%3D8%26spage%3D3976%26epage%3D3995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julien Lefranc, Volker Klaus Schulze, Roman Christian Hillig, Hans Briem, Florian Prinz, Anne Mengel, Tobias Heinrich, Jozsef Balint, Srinivasan Rengachari, Horst Irlbacher, Detlef Stöckigt, Ulf Bömer, Benjamin Bader, Stefan Nikolaus Gradl, Carl Friedrich Nising, Franz von Nussbaum, Dominik Mumberg, Daniel Panne, <span class="NLM_string-name hlFld-ContribAuthor">Antje Margret Wengner</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of BAY-985, a Highly Selective TBK1/IKKε Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 601-612. <a href="https://doi.org/10.1021/acs.jmedchem.9b01460" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01460</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01460%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBAY-985%25252C%252Ba%252BHighly%252BSelective%252BTBK1%25252FIKK%2525CE%2525B5%252BInhibitor%26aulast%3DLefranc%26aufirst%3DJulien%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D02092019%26date%3D10012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D601%26epage%3D612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philipp E.  Hofmann</span>, <span class="hlFld-ContribAuthor ">Jannick  Meinecke</span>, <span class="hlFld-ContribAuthor ">Klaus  Harms</span>, <span class="hlFld-ContribAuthor ">Ulrich  Koert</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Functionalization of C6/C7 Substituted Pyrido[3,2‐
              d
              ]pyrimidines. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em> </em><a href="https://doi.org/10.1002/ejoc.202100593" title="DOI URL">https://doi.org/10.1002/ejoc.202100593</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100593%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Band%252BFunctionalization%252Bof%252BC6%25252FC7%252BSubstituted%252BPyrido%25255B3%25252C2%2525E2%252580%252590%252Bd%252B%25255Dpyrimidines%26aulast%3DHofmann%26aufirst%3DPhilipp%2BE.%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seungbeom  Lee</span>, <span class="hlFld-ContribAuthor ">Jisu  Kim</span>, <span class="hlFld-ContribAuthor ">Jeyun  Jo</span>, <span class="hlFld-ContribAuthor ">Jae Won  Chang</span>, <span class="hlFld-ContribAuthor ">Jaehoon  Sim</span>, <span class="hlFld-ContribAuthor ">Hwayoung  Yun</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in development of hetero-bivalent kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113318. <a href="https://doi.org/10.1016/j.ejmech.2021.113318" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113318</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113318%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bdevelopment%252Bof%252Bhetero-bivalent%252Bkinase%252Binhibitors%26aulast%3DLee%26aufirst%3DSeungbeom%26date%3D2021%26volume%3D216%26spage%3D113318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Tubita</span>, <span class="hlFld-ContribAuthor ">Ignazia  Tusa</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Rovida</span>. </span><span class="cited-content_cbyCitation_article-title">Playing the Whack-A-Mole Game: ERK5 Activation Emerges Among the Resistance Mechanisms to RAF-MEK1/2-ERK1/2- Targeted Therapy. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.647311" title="DOI URL">https://doi.org/10.3389/fcell.2021.647311</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.647311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.647311%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DPlaying%252Bthe%252BWhack-A-Mole%252BGame%25253A%252BERK5%252BActivation%252BEmerges%252BAmong%252Bthe%252BResistance%252BMechanisms%252Bto%252BRAF-MEK1%25252F2-ERK1%25252F2-%252BTargeted%252BTherapy%26aulast%3DTubita%26aufirst%3DAlessandro%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasha  Carmell</span>, <span class="hlFld-ContribAuthor ">Ola  Rominiyi</span>, <span class="hlFld-ContribAuthor ">Katie N.  Myers</span>, <span class="hlFld-ContribAuthor ">Connor  McGarrity-Cottrell</span>, <span class="hlFld-ContribAuthor ">Aurelie  Vanderlinden</span>, <span class="hlFld-ContribAuthor ">Nikita  Lad</span>, <span class="hlFld-ContribAuthor ">Eva  Perroux-David</span>, <span class="hlFld-ContribAuthor ">Sherif F.  El-Khamisy</span>, <span class="hlFld-ContribAuthor ">Malee  Fernando</span>, <span class="hlFld-ContribAuthor ">Katherine G.  Finegan</span>, <span class="hlFld-ContribAuthor ">Stephen  Brown</span>, <span class="hlFld-ContribAuthor ">Spencer J.  Collis</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Validation of ERK5 as a DNA Damage Modulating Drug Target in Glioblastoma. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (5)
                                     , 944. <a href="https://doi.org/10.3390/cancers13050944" title="DOI URL">https://doi.org/10.3390/cancers13050944</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13050944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13050944%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DIdentification%252Band%252BValidation%252Bof%252BERK5%252Bas%252Ba%252BDNA%252BDamage%252BModulating%252BDrug%252BTarget%252Bin%252BGlioblastoma%26aulast%3DCarmell%26aufirst%3DNatasha%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D5%26spage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samanth R.  Kedika</span>, <span class="hlFld-ContribAuthor ">Satya P.  Shukla</span>, <span class="hlFld-ContribAuthor ">D. Gomika  Udugamasooriya</span>. </span><span class="cited-content_cbyCitation_article-title">Design of a dual ERK5 kinase activation and autophosphorylation inhibitor to block cancer stem cell activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (23)
                                     , 127552. <a href="https://doi.org/10.1016/j.bmcl.2020.127552" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127552</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127552%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Bof%252Ba%252Bdual%252BERK5%252Bkinase%252Bactivation%252Band%252Bautophosphorylation%252Binhibitor%252Bto%252Bblock%252Bcancer%252Bstem%252Bcell%252Bactivity%26aulast%3DKedika%26aufirst%3DSamanth%2BR.%26date%3D2020%26volume%3D30%26issue%3D23%26spage%3D127552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon J.  Cook</span>, <span class="hlFld-ContribAuthor ">Julie A.  Tucker</span>, <span class="hlFld-ContribAuthor ">Pamela A.  Lochhead</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule ERK5 kinase inhibitors paradoxically activate ERK5 signalling: be careful what you wish for…. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2020,</strong> <em>48 </em>
                                    (5)
                                     , 1859-1875. <a href="https://doi.org/10.1042/BST20190338" title="DOI URL">https://doi.org/10.1042/BST20190338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20190338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20190338%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DSmall%252Bmolecule%252BERK5%252Bkinase%252Binhibitors%252Bparadoxically%252Bactivate%252BERK5%252Bsignalling%25253A%252Bbe%252Bcareful%252Bwhat%252Byou%252Bwish%252Bfor%2525E2%252580%2525A6%26aulast%3DCook%26aufirst%3DSimon%2BJ.%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D5%26spage%3D1859%26epage%3D1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cornelius  Trünkle</span>, <span class="hlFld-ContribAuthor ">Christian  Lechner</span>, <span class="hlFld-ContribAuthor ">Daniel  Korr</span>, <span class="hlFld-ContribAuthor ">Léa  Bouché</span>, <span class="hlFld-ContribAuthor ">Naomi  Barak</span>, <span class="hlFld-ContribAuthor ">Amaury  Fernández-Montalván</span>, <span class="hlFld-ContribAuthor ">Roderich D  Süssmuth</span>, <span class="hlFld-ContribAuthor ">Andreas  Reichel</span>. </span><span class="cited-content_cbyCitation_article-title">Concentration Dependence of the Unbound Partition Coefficient Kp
              uu
              and Its Application to Correct for Exposure-Related Discrepancies between Biochemical and Cellular Potency of KAT6A Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2020,</strong> <em>48 </em>
                                    (7)
                                     , 553-562. <a href="https://doi.org/10.1124/dmd.120.090563" title="DOI URL">https://doi.org/10.1124/dmd.120.090563</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.120.090563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.120.090563%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DConcentration%252BDependence%252Bof%252Bthe%252BUnbound%252BPartition%252BCoefficient%252BKp%252Buu%252Band%252BIts%252BApplication%252Bto%252BCorrect%252Bfor%252BExposure-Related%252BDiscrepancies%252Bbetween%252BBiochemical%252Band%252BCellular%252BPotency%252Bof%252BKAT6A%252BInhibitors%26aulast%3DTr%25C3%25BCnkle%26aufirst%3DCornelius%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D7%26spage%3D553%26epage%3D562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John K.  Eaton</span>, <span class="hlFld-ContribAuthor ">Laura  Furst</span>, <span class="hlFld-ContribAuthor ">Richard A.  Ruberto</span>, <span class="hlFld-ContribAuthor ">Dieter  Moosmayer</span>, <span class="hlFld-ContribAuthor ">André  Hilpmann</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Ryan</span>, <span class="hlFld-ContribAuthor ">Katja  Zimmermann</span>, <span class="hlFld-ContribAuthor ">Luke L.  Cai</span>, <span class="hlFld-ContribAuthor ">Michael  Niehues</span>, <span class="hlFld-ContribAuthor ">Volker  Badock</span>, <span class="hlFld-ContribAuthor ">Anneke  Kramm</span>, <span class="hlFld-ContribAuthor ">Sixun  Chen</span>, <span class="hlFld-ContribAuthor ">Roman C.  Hillig</span>, <span class="hlFld-ContribAuthor ">Paul A.  Clemons</span>, <span class="hlFld-ContribAuthor ">Stefan  Gradl</span>, <span class="hlFld-ContribAuthor ">Claire  Montagnon</span>, <span class="hlFld-ContribAuthor ">Kiel E.  Lazarski</span>, <span class="hlFld-ContribAuthor ">Sven  Christian</span>, <span class="hlFld-ContribAuthor ">Besnik  Bajrami</span>, <span class="hlFld-ContribAuthor ">Roland  Neuhaus</span>, <span class="hlFld-ContribAuthor ">Ashley L.  Eheim</span>, <span class="hlFld-ContribAuthor ">Vasanthi S.  Viswanathan</span>, <span class="hlFld-ContribAuthor ">Stuart L.  Schreiber</span>. </span><span class="cited-content_cbyCitation_article-title">Selective covalent targeting of GPX4 using masked nitrile-oxide electrophiles. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 497-506. <a href="https://doi.org/10.1038/s41589-020-0501-5" title="DOI URL">https://doi.org/10.1038/s41589-020-0501-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41589-020-0501-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41589-020-0501-5%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DSelective%252Bcovalent%252Btargeting%252Bof%252BGPX4%252Busing%252Bmasked%252Bnitrile-oxide%252Belectrophiles%26aulast%3DEaton%26aufirst%3DJohn%2BK.%26date%3D2020%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D497%26epage%3D506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diane M.  Pereira</span>, <span class="hlFld-ContribAuthor ">Cecília M.P.  Rodrigues</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Avenues for Cancer Treatment: The MEK5–ERK5 Signaling Pathway. </span><span class="cited-content_cbyCitation_journal-name">Trends in Molecular Medicine</span><span> <strong>2020,</strong> <em>26 </em>
                                    (4)
                                     , 394-407. <a href="https://doi.org/10.1016/j.molmed.2020.01.006" title="DOI URL">https://doi.org/10.1016/j.molmed.2020.01.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmed.2020.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmed.2020.01.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Molecular%2520Medicine%26atitle%3DTargeted%252BAvenues%252Bfor%252BCancer%252BTreatment%25253A%252BThe%252BMEK5%2525E2%252580%252593ERK5%252BSignaling%252BPathway%26aulast%3DPereira%26aufirst%3DDiane%2BM.%26date%3D2020%26volume%3D26%26issue%3D4%26spage%3D394%26epage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandra  Cristea</span>, <span class="hlFld-ContribAuthor ">Garry L.  Coles</span>, <span class="hlFld-ContribAuthor ">Daniel  Hornburg</span>, <span class="hlFld-ContribAuthor ">Maya  Gershkovitz</span>, <span class="hlFld-ContribAuthor ">Julia  Arand</span>, <span class="hlFld-ContribAuthor ">Siqi  Cao</span>, <span class="hlFld-ContribAuthor ">Triparna  Sen</span>, <span class="hlFld-ContribAuthor ">Stuart C.  Williamson</span>, <span class="hlFld-ContribAuthor ">Jun W.  Kim</span>, <span class="hlFld-ContribAuthor ">Alexandros P.  Drainas</span>, <span class="hlFld-ContribAuthor ">Andrew  He</span>, <span class="hlFld-ContribAuthor ">Laurent Le  Cam</span>, <span class="hlFld-ContribAuthor ">Lauren Averett  Byers</span>, <span class="hlFld-ContribAuthor ">Michael P.  Snyder</span>, <span class="hlFld-ContribAuthor ">Kévin  Contrepois</span>, <span class="hlFld-ContribAuthor ">Julien  Sage</span>. </span><span class="cited-content_cbyCitation_article-title">The MEK5–ERK5 Kinase Axis Controls Lipid Metabolism in Small-Cell Lung Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2020,</strong> <em>80 </em>
                                    (6)
                                     , 1293-1303. <a href="https://doi.org/10.1158/0008-5472.CAN-19-1027" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-19-1027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-19-1027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-19-1027%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DThe%252BMEK5%2525E2%252580%252593ERK5%252BKinase%252BAxis%252BControls%252BLipid%252BMetabolism%252Bin%252BSmall-Cell%252BLung%252BCancer%26aulast%3DCristea%26aufirst%3DSandra%26date%3D2020%26date%3D2020%26volume%3D80%26issue%3D6%26spage%3D1293%26epage%3D1303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Tubita</span>, <span class="hlFld-ContribAuthor ">Zoe  Lombardi</span>, <span class="hlFld-ContribAuthor ">Ignazia  Tusa</span>, <span class="hlFld-ContribAuthor ">Persio  Dello Sbarba</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Rovida</span>. </span><span class="cited-content_cbyCitation_article-title">Beyond Kinase Activity: ERK5 Nucleo-Cytoplasmic Shuttling as a Novel Target for Anticancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (3)
                                     , 938. <a href="https://doi.org/10.3390/ijms21030938" title="DOI URL">https://doi.org/10.3390/ijms21030938</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21030938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21030938%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBeyond%252BKinase%252BActivity%25253A%252BERK5%252BNucleo-Cytoplasmic%252BShuttling%252Bas%252Ba%252BNovel%252BTarget%252Bfor%252BAnticancer%252BTherapy%26aulast%3DTubita%26aufirst%3DAlessandro%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D3%26spage%3D938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie M.  Myers</span>, <span class="hlFld-ContribAuthor ">Duncan C.  Miller</span>, <span class="hlFld-ContribAuthor ">Lauren  Molyneux</span>, <span class="hlFld-ContribAuthor ">Mercedes  Arasta</span>, <span class="hlFld-ContribAuthor ">Ruth H.  Bawn</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Blackburn</span>, <span class="hlFld-ContribAuthor ">Simon J.  Cook</span>, <span class="hlFld-ContribAuthor ">Noel  Edwards</span>, <span class="hlFld-ContribAuthor ">Jane A.  Endicott</span>, <span class="hlFld-ContribAuthor ">Bernard T.  Golding</span>, <span class="hlFld-ContribAuthor ">Roger J.  Griffin</span>, <span class="hlFld-ContribAuthor ">Tim  Hammonds</span>, <span class="hlFld-ContribAuthor ">Ian R.  Hardcastle</span>, <span class="hlFld-ContribAuthor ">Suzannah J.  Harnor</span>, <span class="hlFld-ContribAuthor ">Amy B.  Heptinstall</span>, <span class="hlFld-ContribAuthor ">Pamela A.  Lochhead</span>, <span class="hlFld-ContribAuthor ">Mathew P.  Martin</span>, <span class="hlFld-ContribAuthor ">Nick C.  Martin</span>, <span class="hlFld-ContribAuthor ">David R.  Newell</span>, <span class="hlFld-ContribAuthor ">Paul J.  Owen</span>, <span class="hlFld-ContribAuthor ">Leon C.  Pang</span>, <span class="hlFld-ContribAuthor ">Tristan  Reuillon</span>, <span class="hlFld-ContribAuthor ">Laurent J.M.  Rigoreau</span>, <span class="hlFld-ContribAuthor ">Huw D.  Thomas</span>, <span class="hlFld-ContribAuthor ">Julie A.  Tucker</span>, <span class="hlFld-ContribAuthor ">Lan-Zhen  Wang</span>, <span class="hlFld-ContribAuthor ">Ai-Ching  Wong</span>, <span class="hlFld-ContribAuthor ">Martin E.M.  Noble</span>, <span class="hlFld-ContribAuthor ">Stephen R.  Wedge</span>, <span class="hlFld-ContribAuthor ">Celine  Cano</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a novel orally bioavailable ERK5 inhibitor with selectivity over p38α and BRD4. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>178 </em>, 530-543. <a href="https://doi.org/10.1016/j.ejmech.2019.05.057" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.05.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.05.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.05.057%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Ba%252Bnovel%252Borally%252Bbioavailable%252BERK5%252Binhibitor%252Bwith%252Bselectivity%252Bover%252Bp38%2525CE%2525B1%252Band%252BBRD4%26aulast%3DMyers%26aufirst%3DStephanie%2BM.%26date%3D2019%26volume%3D178%26spage%3D530%26epage%3D543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Stecca</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Rovida</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of ERK5 on the Hallmarks of Cancer. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (6)
                                     , 1426. <a href="https://doi.org/10.3390/ijms20061426" title="DOI URL">https://doi.org/10.3390/ijms20061426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20061426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20061426%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DImpact%252Bof%252BERK5%252Bon%252Bthe%252BHallmarks%252Bof%252BCancer%26aulast%3DStecca%26aufirst%3DBarbara%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D6%26spage%3D1426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0019.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published ERK5 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of compound <b>1</b> with ERK5 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM">6HKM</a>, 2.46 Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0006.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>1</b> as Representative Synthesis for Quinazoline Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) formamidinium acetate, 2-methoxyethanol, 100 °C, overnight, 88%; (b) SOCl<sub>2</sub> in DMF, reflux, overnight, 88%; (c) <i>tert-</i>butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate, Na<sub>2</sub>CO<sub>3</sub>, 1,1′-bis(diphenylphosphino)-ferrocenepalladium(II) chloride, 1,4-dioxane/water (5:1), 100 °C, 4 h, 74%; (d) TFA, dichloromethane, rt, 2 h, 87%; (e) H<sub>2</sub>, Pd/C, methanol, rt, overnight, 80%; (f) 4-(trifluoromethoxy)benzoic acid, HATU, NEt<sub>3</sub>, DMF, rt, overnight, 18%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of compound <b>35</b> with ERK5 protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN">6HKN</a>, 2.33 Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Summary of chemical, physicochemical, and <i>in vitro</i> DMPK properties of <b>41</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0007.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of ERK5 Probe Molecule <b>41</b> (<b>BAY-885</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>-butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2,3,6-tetrahydropyridine-1-carboxylate, Na<sub>2</sub>CO<sub>3</sub>, 1,1′-bis(diphenylphosphino)ferrocene-palladium(II) chloride, 1,4-dioxane/water (5:1), 100 °C, 4 h, 37%; (b) 1-methylpiperazine, Pd(OAc)<sub>2</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 100 °C, 14 h; (c) H<sub>2</sub>, Pd/C, methanol, rt, overnight, 20%; (d) TFA, dichloromethane, rt, 2 h, 58%; (e) 2-amino-4-(trifluoromethoxy)benzoic acid, HATU, (<i>i</i>-Pr)<sub>2</sub>NEt, DMF, rt, 2 h, 25%.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/medium/jm-2018-01606f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlation between ERK5 kinase inhibition in a biochemical assay and inhibition of the transcriptional activity of MEF2 in SN12C cell line. Data of the entire quinazoline cluster are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-2/acs.jmedchem.8b01606/20190212/images/large/jm-2018-01606f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01606&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drew, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckman, B. S.</span></span> <span> </span><span class="NLM_article-title">MEK5/ERK5 pathway: the first fifteen years</span>. <i>Biochim. Biophys. Acta, Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1825</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2011.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.bbcan.2011.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=22020294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Cju7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1825&publication_year=2012&pages=37-48&author=B.+A.+Drewauthor=M.+E.+Burowauthor=B.+S.+Beckman&title=MEK5%2FERK5+pathway%3A+the+first+fifteen+years&doi=10.1016%2Fj.bbcan.2011.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEK5/ERK5 pathway: The first fifteen years</span></div><div class="casAuthors">Drew, Barbara A.; Burow, Matthew E.; Beckman, Barbara S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1825</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-48</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  While conventional MAP kinase pathways are one of the most highly studied signal transduction mols., less is known about the MEK5 signaling pathway.  This pathway has been shown to play a role in normal cell growth cycles, survival and differentiation.  The MEK5 pathway is also believed to mediate the effects of a no. of oncogenes.  MEK5 is the upstream activator of ERK5 in many epithelial cells.  Activation of the MEK-MAPK pathway is a frequent event in malignant tumor formation and contributes to chemoresistance and anti-apoptotic signaling.  This pathway may be involved in a no. of more aggressive, metastatic varieties of cancer due to its role in cell survival, proliferation and EMT transitioning.  Further study of this pathway may lead to new prognostic factors and new drug targets to combat more aggressive forms of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofQScETgHyILVg90H21EOLACvtfcHk0lj9iNysCt00aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Cju7%252FJ&md5=5f56fde5440678be1cf0e358daf1c6d2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2011.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2011.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDrew%26aufirst%3DB.%2BA.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26aulast%3DBeckman%26aufirst%3DB.%2BS.%26atitle%3DMEK5%252FERK5%2520pathway%253A%2520the%2520first%2520fifteen%2520years%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2012%26volume%3D1825%26spage%3D37%26epage%3D48%26doi%3D10.1016%2Fj.bbcan.2011.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavanaugh, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckman, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span> <span> </span><span class="NLM_article-title">Oncogenic signaling of MEK5-ERK5</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>392</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.canlet.2017.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=28153789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2017&pages=51-59&author=V.+T.+Hoangauthor=T.+J.+Yanauthor=J.+E.+Cavanaughauthor=P.+T.+Flahertyauthor=B.+S.+Beckmanauthor=M.+E.+Burow&title=Oncogenic+signaling+of+MEK5-ERK5&doi=10.1016%2Fj.canlet.2017.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic signaling of MEK5-ERK5</span></div><div class="casAuthors">Hoang, Van T.; Yan, Thomas J.; Cavanaugh, Jane E.; Flaherty, Patrick T.; Beckman, Barbara S.; Burow, Matthew E.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-59</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Mitogen-activated protein kinases (MAPKs) regulate diverse cellular processes including proliferation, cell survival, differentiation, and apoptosis.  While conventional MAPK constituents have well-defined roles in oncogenesis, the MEK5 pathway has only recently emerged in cancer research.  In this review, we consider the MEK5 signaling cascade, focusing specifically on its involvement in drug resistance and regulation of aggressive cancer phenotypes.  Moreover, we explore the role of MEK5/ERK5 in tumorigenesis and metastatic progression, discussing the discrepancies in preclin. studies and assessing its viability as a therapeutic target for anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprYwxatp_EjrVg90H21EOLACvtfcHk0lj9iNysCt00aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Smt74%253D&md5=2047e484e36e06d8345829b49f0c1c4e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DHoang%26aufirst%3DV.%2BT.%26aulast%3DYan%26aufirst%3DT.%2BJ.%26aulast%3DCavanaugh%26aufirst%3DJ.%2BE.%26aulast%3DFlaherty%26aufirst%3DP.%2BT.%26aulast%3DBeckman%26aufirst%3DB.%2BS.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26atitle%3DOncogenic%2520signaling%2520of%2520MEK5-ERK5%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D392%26spage%3D51%26epage%3D59%26doi%3D10.1016%2Fj.canlet.2017.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nithianandarajah-Jones, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldring, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, M. J.</span></span> <span> </span><span class="NLM_article-title">ERK5: Structure, regulation and function</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2187</span>– <span class="NLM_lpage">2196</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2012.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.cellsig.2012.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=22800864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSns7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=2187-2196&author=G.+N.+Nithianandarajah-Jonesauthor=B.+Wilmauthor=C.+E.+Goldringauthor=J.+Mullerauthor=M.+J.+Cross&title=ERK5%3A+Structure%2C+regulation+and+function&doi=10.1016%2Fj.cellsig.2012.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">ERK5: Structure, regulation and function</span></div><div class="casAuthors">Nithianandarajah-Jones, Gopika N.; Wilm, Bettina; Goldring, Christopher E. P.; Muller, Jurgen; Cross, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2187-2196</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Extracellular signal-regulated kinase 5 (ERK5), also termed big mitogen-activated protein kinase-1 (BMK1), is the most recently identified member of the mitogen-activated protein kinase (MAPK) family and consists of an amino-terminal kinase domain, with a relatively large carboxy-terminal of unique structure and function that makes it distinct from other MAPK members.  It is ubiquitously expressed in numerous tissues and is activated by a variety of extracellular stimuli, such as cellular stresses and growth factors, to regulate processes such as cell proliferation and differentiation.  Targeted deletion of Erk5 in mice has revealed that the ERK5 signalling cascade plays a crit. role in cardiovascular development and vascular integrity.  Recent data points to a potential role in pathol. conditions such as cancer and tumor angiogenesis.  This review focuses on the physiol. and pathol. role of ERK5, the regulation of this kinase and the recent development of small mol. inhibitors of the ERK5 signalling cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqhjeO2v9aX7Vg90H21EOLACvtfcHk0lh4taRqt0kL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSns7fP&md5=ebae0228bd187490f05945d411d34ad4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2012.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2012.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DNithianandarajah-Jones%26aufirst%3DG.%2BN.%26aulast%3DWilm%26aufirst%3DB.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DCross%26aufirst%3DM.%2BJ.%26atitle%3DERK5%253A%2520Structure%252C%2520regulation%2520and%2520function%26jtitle%3DCell.%2520Signalling%26date%3D2012%26volume%3D24%26spage%3D2187%26epage%3D2196%26doi%3D10.1016%2Fj.cellsig.2012.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simoes, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borralho, P. M.</span></span> <span> </span><span class="NLM_article-title">The MEK5/ERK5 signalling pathway in cancer: A promising novel therapeutic target</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1663</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.drudis.2016.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=27320690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1654-1663&author=A.+E.+Simoesauthor=C.+M.+Rodriguesauthor=P.+M.+Borralho&title=The+MEK5%2FERK5+signalling+pathway+in+cancer%3A+A+promising+novel+therapeutic+target&doi=10.1016%2Fj.drudis.2016.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The MEK5/ERK5 signalling pathway in cancer: a promising novel therapeutic target</span></div><div class="casAuthors">Simoes, Andre E. S.; Rodrigues, Cecilia M. P.; Borralho, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1654-1663</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Conventional mitogen-activated protein kinase (MAPK) family members are among the most sought-after oncogenic effectors for the development of novel human cancer treatment strategies.  MEK5/ERK5 has been the less-studied MAPK subfamily, despite its increasingly demonstrated relevance in the growth, survival, and differentiation of normal cells.  MEK5/ERK5 signalling has already been proposed to have pivotal roles in several cancer hallmarks, and to mediate the effects of a range of oncogenes.  Accumulating evidence indicates the contribution of MEK5/ERK5 signalling to therapy resistance and the benefits of using MEK5/ERK5 inhibitory strategies in the treatment of human cancer.  Here, we explore the major known contributions of MEK5/ERK5 signalling to the onset and progression of several types of cancer, and highlight the potential clin. relevance of targeting MEK5/ERK5 pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHG6CKquuTJbVg90H21EOLACvtfcHk0lh4taRqt0kL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynsbzL&md5=3d73f6af2848b0f78b53cde743cac805</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DSimoes%26aufirst%3DA.%2BE.%26aulast%3DRodrigues%26aufirst%3DC.%2BM.%26aulast%3DBorralho%26aufirst%3DP.%2BM.%26atitle%3DThe%2520MEK5%252FERK5%2520signalling%2520pathway%2520in%2520cancer%253A%2520A%2520promising%2520novel%2520therapeutic%2520target%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1654%26epage%3D1663%26doi%3D10.1016%2Fj.drudis.2016.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Esparis-Ogando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Rodriguez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuste, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandiella, A.</span></span> <span> </span><span class="NLM_article-title">Erk5 participates in neuregulin signal transduction and is constitutively active in breast cancer cells overexpressing ErbB2</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1128/MCB.22.1.270-285.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1128%2FMCB.22.1.270-285.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=11739740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptlOiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=270-285&author=A.+Esparis-Ogandoauthor=E.+Diaz-Rodriguezauthor=J.+C.+Monteroauthor=L.+Yusteauthor=P.+Crespoauthor=A.+Pandiella&title=Erk5+participates+in+neuregulin+signal+transduction+and+is+constitutively+active+in+breast+cancer+cells+overexpressing+ErbB2&doi=10.1128%2FMCB.22.1.270-285.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Erk5 participates in neuregulin signal transduction and is constitutively active in breast cancer cells overexpressing ErbB2</span></div><div class="casAuthors">Esparis-Ogando, Azucena; Diaz-Rodriguez, Elena; Montero, Juan Carlos; Yuste, Laura; Crespo, Piero; Pandiella, Atanasio</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">270-285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The four receptor tyrosine kinases of the ErbB family play essential roles in several physiol. processes and have also been implicated in tumor generation and/or progression.  Activation of ErbB1/EGFR is mainly triggered by epidermal growth factor (EGF) and other related ligands, while activation of ErbB2, ErbB3, and ErbB4 receptors occurs by binding to another set of EGF-like ligands termed neuregulins (NRGs).  Here we show that the Erk5 mitogen-activated protein kinase (MAPK) pathway participates in NRG signal transduction.  In MCF7 cells, NRG activated Erk5 in a time- and dose-dependent fashion.  The action of NRG on Erk5 was dependent on the kinase activity of ErbB receptors but was independent of Ras.  Expression in MCF7 cells of a dominant neg. form of Erk5 resulted in a significant decrease in NRG-induced proliferation of MCF7 cells.  Anal. of Erk5 in several human tumor cell lines indicated that a constitutively active form of this kinase was present in the BT474 and SKBR3 cell lines, which also expressed activated forms of ErbB2, ErbB3, and ErbB4.  Treatments aimed at decreasing the activity of these receptors caused Erk5 inactivation, indicating that the active form of Erk5 present in BT474 and SKBR3 cells was due to a persistent pos. stimulus originating at the ErbB receptors.  In BT474 cells expression of the dominant neg. form of Erk5 resulted in reduced proliferation, indicating that in these cells Erk5 was also involved in the control of proliferation.  Taken together, these results suggest that Erk5 may play a role in the regulation of cell proliferation by NRG receptors and indicate that constitutively active NRG receptors may induce proliferative responses in cancer cells through this MAPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHa2TbZPveLVg90H21EOLACvtfcHk0lh4taRqt0kL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptlOiurY%253D&md5=dd1dd2d893f7c5cdc5cf6ee9515ecd7f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.1.270-285.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.1.270-285.2002%26sid%3Dliteratum%253Aachs%26aulast%3DEsparis-Ogando%26aufirst%3DA.%26aulast%3DDiaz-Rodriguez%26aufirst%3DE.%26aulast%3DMontero%26aufirst%3DJ.%2BC.%26aulast%3DYuste%26aufirst%3DL.%26aulast%3DCrespo%26aufirst%3DP.%26aulast%3DPandiella%26aufirst%3DA.%26atitle%3DErk5%2520participates%2520in%2520neuregulin%2520signal%2520transduction%2520and%2520is%2520constitutively%2520active%2520in%2520breast%2520cancer%2520cells%2520overexpressing%2520ErbB2%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D270%26epage%3D285%26doi%3D10.1128%2FMCB.22.1.270-285.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Q.</span></span> <span> </span><span class="NLM_article-title">Identification and validation of dysregulated MAPK7 (ERK5) as a novel oncogenic target in squamous cell lung and esophageal carcinoma</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">454</span>, <span class="refDoi"> DOI: 10.1186/s12885-015-1455-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1186%2Fs12885-015-1455-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=26040563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADC%252BC2MbgtlGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=454&author=P.+R.+Gavineauthor=M.+Wangauthor=D.+Yuauthor=E.+Huauthor=C.+Huangauthor=J.+Xiaauthor=X.+Suauthor=J.+Fanauthor=T.+Zhangauthor=Q.+Yeauthor=L.+Zhengauthor=G.+Zhuauthor=Z.+Qianauthor=Q.+Luoauthor=Y.+Y.+Houauthor=Q.+Ji&title=Identification+and+validation+of+dysregulated+MAPK7+%28ERK5%29+as+a+novel+oncogenic+target+in+squamous+cell+lung+and+esophageal+carcinoma&doi=10.1186%2Fs12885-015-1455-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and validation of dysregulated MAPK7 (ERK5) as a novel oncogenic target in squamous cell lung and esophageal carcinoma</span></div><div class="casAuthors">Gavine Paul R; Wang Mei; Yu Dehua; Hu Eva; Huang Chunlei; Xia Jenny; Su Xinying; Fan Joan; Zhang Tianwei; Ye Qingqing; Zheng Li; Zhu Guanshan; Qian Ziliang; Ji Qunsheng; Luo Qingquan; Hou Ying Yong</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">454</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  MAPK7/ERK5 (extracellular-signal-regulated kinase 5) functions within a canonical three-tiered MAPK (mitogen activated protein kinase) signaling cascade comprising MEK (MAPK/ERK kinase) 5, MEKK(MEK kinase) 2/3 and ERK5 itself.  Despite being the least well studied of the MAPK-modules, evidence supports a role for MAPK7-signaling in the pathology of several cancer types.  METHODS AND RESULTS:  Fluorescence in situ hybridization (FISH) analysis identified MAPK7 gene amplification in 4% (3/74) of non-small cell lung cancers (NSCLC) (enriched to 6% (3/49) in squamous cell carcinoma) and 2% (2/95) of squamous esophageal cancers (sqEC).  Immunohistochemical (IHC) analysis revealed a good correlation between MAPK7 gene amplification and protein expression.  MAPK7 was validated as a proliferative oncogenic driver by performing in vitro siRNA knockdown of MAPK7 in tumor cell lines.  Finally, a novel MEK5/MAPK7 co-transfected HEK293 cell line was developed and used for routine cell-based pharmacodynamic screening.  Phosphorylation antibody microarray analysis also identified novel downstream pharmacodynamic (PD) biomarkers of MAPK7 kinase inhibition in tumor cells (pMEF2A and pMEF2D).  CONCLUSIONS:  Together, these data highlight a broader role for dysregulated MAPK7 in driving tumorigenesis within niche populations of highly prevalent tumor types, and describe current efforts in establishing a robust drug discovery screening cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQObyGhx8V5ZSSm0e6ZOaemfW6udTcc2ebcnzICeJX7jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbgtlGktQ%253D%253D&md5=cfbc661ae8f4a71779c8588be6f1de03</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs12885-015-1455-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-015-1455-y%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DHou%26aufirst%3DY.%2BY.%26aulast%3DJi%26aufirst%3DQ.%26atitle%3DIdentification%2520and%2520validation%2520of%2520dysregulated%2520MAPK7%2520%2528ERK5%2529%2520as%2520a%2520novel%2520oncogenic%2520target%2520in%2520squamous%2520cell%2520lung%2520and%2520esophageal%2520carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D454%26doi%3D10.1186%2Fs12885-015-1455-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cason, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuschel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillegass, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacek, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mossman, B. T.</span></span> <span> </span><span class="NLM_article-title">Extracellular signal-regulated kinase 5: A potential therapeutic target for malignant mesotheliomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2071</span>– <span class="NLM_lpage">2083</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-3202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1158%2F1078-0432.CCR-12-3202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=23446998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvFyjsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2071-2083&author=A.+Shuklaauthor=J.+M.+Millerauthor=C.+Casonauthor=M.+Sayanauthor=M.+B.+MacPhersonauthor=S.+L.+Beuschelauthor=J.+Hillegassauthor=P.+M.+Vacekauthor=H.+I.+Passauthor=B.+T.+Mossman&title=Extracellular+signal-regulated+kinase+5%3A+A+potential+therapeutic+target+for+malignant+mesotheliomas&doi=10.1158%2F1078-0432.CCR-12-3202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular Signal-Regulated Kinase 5: A Potential Therapeutic Target for Malignant Mesotheliomas</span></div><div class="casAuthors">Shukla, Arti; Miller, Jill M.; Cason, Christopher; Sayan, Mutlay; MacPherson, Maximilian B.; Beuschel, Stacie L.; Hillegass, Jedd; Vacek, Pamela M.; Pass, Harvey I.; Mossman, Brooke T.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2071-2083</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Malignant mesothelioma is a devastating disease with a need for new treatment strategies.  In the present study, we showed the importance of extracellular signal-regulated kinase 5 (ERK5) in malignant mesothelioma tumor growth and treatment.  Exptl. Design: ERK5 as a target for malignant mesothelioma therapy was verified using mesothelial and mesothelioma cell lines as well as by xenograft severe combined immunodeficient (SCID) mouse models.  Results: We first showed that crocidolite asbestos activated ERK5 in LP9 cells and mesothelioma cell lines exhibit constitutive activation of ERK5.  Addn. of doxorubicin resulted in further activation of ERK5 in malignant mesothelioma cells.  ERK5 silencing increased doxorubicin-induced cell death and doxorubicin retention in malignant mesothelioma cells.  In addn., shERK5 malignant mesothelioma lines exhibited both attenuated colony formation on soft agar and invasion of malignant mesothelioma cells in vitro that could be related to modulation of gene expression linked to cell proliferation, apoptosis, migration/invasion, and drug resistance as shown by microarray anal.  Most importantly, injection of shERK5 malignant mesothelioma cell lines into SCID mice showed significant redn. in tumor growth using both s.c. and i.p. models.  Assessment of selected human cytokine profiles in peritoneal lavage fluid from i.p. shERK5 and control tumor-bearing mice showed that ERK5 was crit. in regulation of various proinflammatory (RANTES/CCL5, MCP-1) and angiogenesis-related (interleukin-8, VEGF) cytokines.  Finally, use of doxorubicin and cisplatin in combination with ERK5 inhibition showed further redn. in tumor wt. and vol. in the i.p. model of tumor growth.  Conclusion: ERK5 inhibition in combination with chemotherapeutic drugs is a beneficial strategy for combination therapy in patients with malignant mesothelioma.  Clin Cancer Res; 19(8); 2071-83. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_r7UsUaX-IbVg90H21EOLACvtfcHk0li2Nr_s95DS0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvFyjsLo%253D&md5=9ccd4c727165e7c0874ec0dd314cb2fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3202%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DJ.%2BM.%26aulast%3DCason%26aufirst%3DC.%26aulast%3DSayan%26aufirst%3DM.%26aulast%3DMacPherson%26aufirst%3DM.%2BB.%26aulast%3DBeuschel%26aufirst%3DS.%2BL.%26aulast%3DHillegass%26aufirst%3DJ.%26aulast%3DVacek%26aufirst%3DP.%2BM.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DMossman%26aufirst%3DB.%2BT.%26atitle%3DExtracellular%2520signal-regulated%2520kinase%25205%253A%2520A%2520potential%2520therapeutic%2520target%2520for%2520malignant%2520mesotheliomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D2071%26epage%3D2083%26doi%3D10.1158%2F1078-0432.CCR-12-3202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuyoshi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsufuji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniwaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okanoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, T.</span></span> <span> </span><span class="NLM_article-title">ERK5 is a target for gene amplification at 17p11 and promotes cell growth in hepatocellular carcinoma by regulating mitotic entry</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1002/gcc.20624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1002%2Fgcc.20624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=18973138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FhvFyjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=109-120&author=K.+Zenauthor=K.+Yasuiauthor=T.+Nakajimaauthor=Y.+Zenauthor=K.+Zenauthor=Y.+Genauthor=H.+Mitsuyoshiauthor=M.+Minamiauthor=S.+Mitsufujiauthor=S.+Tanakaauthor=Y.+Itohauthor=Y.+Nakanumaauthor=M.+Taniwakiauthor=S.+Ariiauthor=T.+Okanoueauthor=T.+Yoshikawa&title=ERK5+is+a+target+for+gene+amplification+at+17p11+and+promotes+cell+growth+in+hepatocellular+carcinoma+by+regulating+mitotic+entry&doi=10.1002%2Fgcc.20624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ERK5 is a target for gene amplification at 17p11 and promotes cell growth in hepatocellular carcinoma by regulating mitotic entry</span></div><div class="casAuthors">Zen Keika; Yasui Kohichiroh; Nakajima Tomoaki; Zen Yoh; Zen Kan; Gen Yasuyuki; Mitsuyoshi Hironori; Minami Masahito; Mitsufuji Shoji; Tanaka Shinji; Itoh Yoshito; Nakanuma Yasuni; Taniwaki Masafumi; Arii Shigeki; Okanoue Takeshi; Yoshikawa Toshikazu</div><div class="citationInfo"><span class="NLM_cas:title">Genes, chromosomes & cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Using high-density oligonucleotide microarrays, we investigated DNA copy-number aberrations in cell lines derived from hepatocellular carcinomas (HCCs) and detected a novel amplification at 17p11.  To identify the target of amplification at 17p11, we defined the extent of the amplicon and examined HCC cell lines for expression of all seven genes in the 750-kb commonly amplified region.  Mitogen-activated protein kinase (MAPK) 7, which encodes extracellular-regulated protein kinase (ERK) 5, was overexpressed in cell lines in which the gene was amplified.  An increase in MAPK7 copy number was detected in 35 of 66 primary HCC tumors.  Downregulation of MAPK7 by small interfering RNA suppressed the growth of SNU449 cells, the HCC cell line with the greatest amplification and overexpression of MAPK7.  ERK5, phosphorylated during the G2/M phases of the cell cycle, regulated entry into mitosis in SNU449 cells.  In conclusion, our results suggest that MAPK7 is likely the target of 17p11 amplification and that the ERK5 protein product of MAPK7 promotes the growth of HCC cells by regulating mitotic entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcScksBqKJIaWZI8NvIb08rbfW6udTcc2ebcnzICeJX7jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FhvFyjsg%253D%253D&md5=a86797e020531fb31385555be1e03dd0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fgcc.20624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.20624%26sid%3Dliteratum%253Aachs%26aulast%3DZen%26aufirst%3DK.%26aulast%3DYasui%26aufirst%3DK.%26aulast%3DNakajima%26aufirst%3DT.%26aulast%3DZen%26aufirst%3DY.%26aulast%3DZen%26aufirst%3DK.%26aulast%3DGen%26aufirst%3DY.%26aulast%3DMitsuyoshi%26aufirst%3DH.%26aulast%3DMinami%26aufirst%3DM.%26aulast%3DMitsufuji%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DNakanuma%26aufirst%3DY.%26aulast%3DTaniwaki%26aufirst%3DM.%26aulast%3DArii%26aufirst%3DS.%26aulast%3DOkanoue%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DT.%26atitle%3DERK5%2520is%2520a%2520target%2520for%2520gene%2520amplification%2520at%252017p11%2520and%2520promotes%2520cell%2520growth%2520in%2520hepatocellular%2520carcinoma%2520by%2520regulating%2520mitotic%2520entry%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2009%26volume%3D48%26spage%3D109%26epage%3D120%26doi%3D10.1002%2Fgcc.20624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a benzo[e]pyrimido-[5,4-b][1,4]diazepin-6(11H)-one as a potent and selective inhibitor of Big MAP Kinase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1021/ml100304b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100304b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=195-200&author=X.+Dengauthor=Q.+Yangauthor=N.+Kwiatkowskiauthor=T.+Simauthor=U.+McDermottauthor=J.+E.+Settlemanauthor=J.+D.+Leeauthor=N.+S.+Gray&title=Discovery+of+a+benzo%5Be%5Dpyrimido-%5B5%2C4-b%5D%5B1%2C4%5Ddiazepin-6%2811H%29-one+as+a+potent+and+selective+inhibitor+of+Big+MAP+Kinase+1&doi=10.1021%2Fml100304b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml100304b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100304b%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DSettleman%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520benzo%255Be%255Dpyrimido-%255B5%252C4-b%255D%255B1%252C4%255Ddiazepin-6%252811H%2529-one%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520Big%2520MAP%2520Kinase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D195%26epage%3D200%26doi%3D10.1021%2Fml100304b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fearns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of BMK1 suppresses tumor growth through promyelocytic leukemia protein</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.08.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.ccr.2010.08.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=20832753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=258-267&author=Q.+Yangauthor=X.+Dengauthor=B.+Luauthor=M.+Cameronauthor=C.+Fearnsauthor=M.+P.+Patricelliauthor=J.+R.+Yatesauthor=N.+S.+Grayauthor=J.+D.+Lee&title=Pharmacological+inhibition+of+BMK1+suppresses+tumor+growth+through+promyelocytic+leukemia+protein&doi=10.1016%2Fj.ccr.2010.08.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Inhibition of BMK1 Suppresses Tumor Growth through Promyelocytic Leukemia Protein</span></div><div class="casAuthors">Yang, Qing-Kai; Deng, Xian-Ming; Lu, Bing-Wen; Cameron, Michael; Fearns, Colleen; Patricelli, Matthew P.; Yates, John R., III; Gray, Nathanael S.; Lee, Jiing-Dwan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">258-267</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BMK1 is activated by mitogens and oncogenic signals and, thus, is strongly implicated in tumorigenesis.  We found that BMK1 interacted with promyelocytic leukemia protein (PML), and inhibited its tumor-suppressor function through phosphorylation.  Furthermore, activated BMK1 notably inhibited PML-dependent activation of p21.  To further investigate the BMK-mediated inhibition of the tumor suppressor activity of PML in tumor cells, we developed a small-mol. inhibitor of the kinase activity of BMK1, XMD8-92.  Inhibition of BMK1 by XMD8-92 blocked tumor cell proliferation in vitro and significantly inhibited tumor growth in vivo by 95%, demonstrating the efficacy and tolerability of BMK1-targeted cancer treatment in animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog5sdylMRjmrVg90H21EOLACvtfcHk0lj_AsBn_tu_Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bK&md5=45900e7ef5abcb99d22c91ac94b561d1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.08.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.08.008%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DCameron%26aufirst%3DM.%26aulast%3DFearns%26aufirst%3DC.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26atitle%3DPharmacological%2520inhibition%2520of%2520BMK1%2520suppresses%2520tumor%2520growth%2520through%2520promyelocytic%2520leukemia%2520protein%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D258%26epage%3D267%26doi%3D10.1016%2Fj.ccr.2010.08.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amantea, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidique, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarich, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, J. S.</span></span> <span> </span><span class="NLM_article-title">ERK5 kinase activity is dispensable for cellular immune response and proliferation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">11865</span>– <span class="NLM_lpage">11870</span>, <span class="refDoi"> DOI: 10.1073/pnas.1609019113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1073%2Fpnas.1609019113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=11865-11870&author=E.+C.+Linauthor=C.+M.+Amanteaauthor=T.+K.+Nomanbhoyauthor=H.+Weissigauthor=J.+Ishiyamaauthor=Y.+Huauthor=S.+Sidiqueauthor=B.+Liauthor=J.+W.+Kozarichauthor=J.+S.+Rosenblum&title=ERK5+kinase+activity+is+dispensable+for+cellular+immune+response+and+proliferation&doi=10.1073%2Fpnas.1609019113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1609019113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1609019113%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DE.%2BC.%26aulast%3DAmantea%26aufirst%3DC.%2BM.%26aulast%3DNomanbhoy%26aufirst%3DT.%2BK.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DIshiyama%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DSidique%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26aulast%3DRosenblum%26aufirst%3DJ.%2BS.%26atitle%3DERK5%2520kinase%2520activity%2520is%2520dispensable%2520for%2520cellular%2520immune%2520response%2520and%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D11865%26epage%3D11870%26doi%3D10.1073%2Fpnas.1609019113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lueth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loewe, W.</span></span> <span> </span><span class="NLM_article-title">Syntheses of 4-(indole-3-yl)quinazolines - A new class of epidermal growth factor receptor tyrosine kinase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1478</span>– <span class="NLM_lpage">1488</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2007.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.ejmech.2007.09.018" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2008&pages=1478-1488&author=A.+Luethauthor=W.+Loewe&title=Syntheses+of+4-%28indole-3-yl%29quinazolines+-+A+new+class+of+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.ejmech.2007.09.018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2007.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2007.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DLueth%26aufirst%3DA.%26aulast%3DLoewe%26aufirst%3DW.%26atitle%3DSyntheses%2520of%25204-%2528indole-3-yl%2529quinazolines%2520-%2520A%2520new%2520class%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2008%26volume%3D43%26spage%3D1478%26epage%3D1488%26doi%3D10.1016%2Fj.ejmech.2007.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortlock, A. A.</span>; <span class="NLM_string-name">Keen, N. J.</span></span> <span> </span><span class="NLM_article-title">Quinazoline Derivatives</span>. <span class="NLM_patent">EP1218357 B1</span>, April 6, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=A.+A.+Mortlock&author=N.+J.+Keen&title=Quinazoline+Derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMortlock%26aufirst%3DA.%2BA.%26atitle%3DQuinazoline%2520Derivatives%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="note"><p class="first last">For synthesis of <b>43</b>, see:</p></div><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span>; <span class="NLM_string-name">Low, J. D.</span>; <span class="NLM_string-name">Allen, J. R.</span>; <span class="NLM_string-name">Chen, J.</span>; <span class="NLM_string-name">Chen, N.</span>; <span class="NLM_string-name">Cheng, Y.</span>; <span class="NLM_string-name">Judd, T.</span>; <span class="NLM_string-name">Liu, Q.</span>; <span class="NLM_string-name">Lopez, P.</span>; <span class="NLM_string-name">Qian, W.</span>; <span class="NLM_string-name">Rumfelt, S.</span>; <span class="NLM_string-name">Rzasa, R. M.</span>; <span class="NLM_string-name">Tamayo, N. A.</span>; <span class="NLM_string-name">Xue, Q.</span>; <span class="NLM_string-name">Yang, B.</span>; <span class="NLM_string-name">Zhong, W.</span></span> <span> </span><span class="NLM_article-title">Perfluorinated 5,6-Dihydro-4H-1,3-oxazin-2-amine Compounds as Beta-Secretase Inhibitors and Methods of Use</span>. <span class="NLM_patent">WO2014/134341 A1</span>, September 4, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+E.+Minatti&author=J.+D.+Low&author=J.+R.+Allen&author=J.+Chen&author=N.+Chen&author=Y.+Cheng&author=T.+Judd&author=Q.+Liu&author=P.+Lopez&author=W.+Qian&author=S.+Rumfelt&author=R.+M.+Rzasa&author=N.+A.+Tamayo&author=Q.+Xue&author=B.+Yang&author=W.+Zhong&title=Perfluorinated+5%2C6-Dihydro-4H-1%2C3-oxazin-2-amine+Compounds+as+Beta-Secretase+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26atitle%3DPerfluorinated%25205%252C6-Dihydro-4H-1%252C3-oxazin-2-amine%2520Compounds%2520as%2520Beta-Secretase%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erazo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Babot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Asiain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrice, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espadamala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayascas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizcano, J. M.</span></span> <span> </span><span class="NLM_article-title">Canonical and kinase activity-independent mechanisms for extracellular signal-regulated kinase 5 (ERK5) nuclear translocation require dissociation of Hsp90 from the ERK5-Cdc37 complex</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1128/MCB.01246-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1128%2FMCB.01246-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=23428871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslajt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1671-1686&author=T.+Erazoauthor=A.+Morenoauthor=G.+Ruiz-Babotauthor=A.+Rodriguez-Asiainauthor=N.+A.+Morriceauthor=J.+Espadamalaauthor=J.+R.+Bayascasauthor=N.+Gomezauthor=J.+M.+Lizcano&title=Canonical+and+kinase+activity-independent+mechanisms+for+extracellular+signal-regulated+kinase+5+%28ERK5%29+nuclear+translocation+require+dissociation+of+Hsp90+from+the+ERK5-Cdc37+complex&doi=10.1128%2FMCB.01246-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical and kinase activity-independent mechanisms for extracellular signal-regulated kinase 5 (ERK5) nuclear translocation require dissociation of Hsp90 from the ERK5-Cdc37 complex</span></div><div class="casAuthors">Erazo, Tatiana; Moreno, Ana; Ruiz-Babot, Gerard; Rodriguez-Asiain, Arantza; Morrice, Nicholas A.; Espadamala, Josep; Bayascas, Jose R.; Gomez, Nestor; Lizcano, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1671-1686</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein (MAP) kinase extracellular signal-regulated kinase 5 (ERK5) plays a crucial role in cell proliferation, regulating gene transcription.  ERK5 has a unique C-terminal tail which contains a transcriptional activation domain, and activates transcription by phosphorylating transcription factors and acting itself as a transcriptional coactivator.  However, the mol. mechanisms that regulate its nucleocytoplasmic traffic are unknown.  We have used tandem affinity purifn. to identify proteins that interact with ERK5.  We show that ERK5 interacts with the Hsp90-Cdc37 chaperone in resting cells, and that inhibition of Hsp90 or Cdc37 results in ERK5 ubiquitylation and proteasomal degrdn.  Interestingly, activation of cellular ERK5 induces Hsp90 dissocn. from the ERK5-Cdc37 complex, leading to ERK5 nuclear translocation and activation of transcription, by a mechanism which requires the autophosphorylation at its C-terminal tail.  Consequently, active ERK5 is no longer sensitive to Hsp90 or Cdc37 inhibitors.  Cdc37 overexpression also induces Hsp90 dissocn. and the nuclear translocation of a kinase-inactive form of ERK5 which retains transcriptional activity.  This is the first example showing that ERK5 transcriptional activity does not require kinase activity.  Since Cdc37 cooperates with ERK5 to promote cell proliferation, Cdc37 overexpression (as happens in some cancers) might represent a new, noncanonical mechanism by which ERK5 regulates tumor proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8nP0WXDHSqrVg90H21EOLACvtfcHk0lj_AsBn_tu_Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslajt70%253D&md5=fd610778cfd77357213571583bf7a80b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FMCB.01246-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01246-12%26sid%3Dliteratum%253Aachs%26aulast%3DErazo%26aufirst%3DT.%26aulast%3DMoreno%26aufirst%3DA.%26aulast%3DRuiz-Babot%26aufirst%3DG.%26aulast%3DRodriguez-Asiain%26aufirst%3DA.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DEspadamala%26aufirst%3DJ.%26aulast%3DBayascas%26aufirst%3DJ.%2BR.%26aulast%3DGomez%26aufirst%3DN.%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26atitle%3DCanonical%2520and%2520kinase%2520activity-independent%2520mechanisms%2520for%2520extracellular%2520signal-regulated%2520kinase%25205%2520%2528ERK5%2529%2520nuclear%2520translocation%2520require%2520dissociation%2520of%2520Hsp90%2520from%2520the%2520ERK5-Cdc37%2520complex%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2013%26volume%3D33%26spage%3D1671%26epage%3D1686%26doi%3D10.1128%2FMCB.01246-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kravchenko, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapping, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulevitch, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. D.</span></span> <span> </span><span class="NLM_article-title">BMK1/ERK5 regulates serum-induced early gene expression through transcription factor MEF2C</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">7054</span>– <span class="NLM_lpage">7066</span>, <span class="refDoi"> DOI: 10.1093/emboj/16.23.7054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1093%2Femboj%2F16.23.7054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=7054-7066&author=Y.+Katoauthor=V.+V.+Kravchenkoauthor=R.+I.+Tappingauthor=J.+Hanauthor=R.+J.+Ulevitchauthor=J.+D.+Lee&title=BMK1%2FERK5+regulates+serum-induced+early+gene+expression+through+transcription+factor+MEF2C&doi=10.1093%2Femboj%2F16.23.7054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F16.23.7054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F16.23.7054%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DY.%26aulast%3DKravchenko%26aufirst%3DV.%2BV.%26aulast%3DTapping%26aufirst%3DR.%2BI.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DUlevitch%26aufirst%3DR.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BD.%26atitle%3DBMK1%252FERK5%2520regulates%2520serum-induced%2520early%2520gene%2520expression%2520through%2520transcription%2520factor%2520MEF2C%26jtitle%3DEMBO%2520J.%26date%3D1997%26volume%3D16%26spage%3D7054%26epage%3D7066%26doi%3D10.1093%2Femboj%2F16.23.7054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span> <span> </span><span class="NLM_article-title">Challenges in validating candidate therapeutic targets</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e32402</span>, <span class="refDoi"> DOI: 10.7554/eLife.32402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.7554%2FeLife.32402" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e32402&author=J.+Settlemanauthor=C.+L.+Sawyersauthor=T.+Hunter&title=Challenges+in+validating+candidate+therapeutic+targets&doi=10.7554%2FeLife.32402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.32402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.32402%26sid%3Dliteratum%253Aachs%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DSawyers%26aufirst%3DC.%2BL.%26aulast%3DHunter%26aufirst%3DT.%26atitle%3DChallenges%2520in%2520validating%2520candidate%2520therapeutic%2520targets%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26spage%3De32402%26doi%3D10.7554%2FeLife.32402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trésaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Novel class of potent and cellularly active inhibitors devalidates MTH1 as broad-spectrum cancer target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstr%C3%B6mauthor=C.+Silvanderauthor=L.+Tr%C3%A9sauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=B.+Bauserauthor=A.+Haegebarthauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Novel+class+of+potent+and+cellularly+active+inhibitors+devalidates+MTH1+as+broad-spectrum+cancer+target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span></div><div class="casAuthors">Ellermann, Manuel; Eheim, Ashley; Rahm, Fredrik; Viklund, Jenny; Guenther, Judith; Andersson, Martin; Ericsson, Ulrika; Forsblom, Rickard; Ginman, Tobias; Lindstroem, Johan; Silvander, Camilla; Tresaugues, Lionel; Giese, Anja; Bunse, Stefanie; Neuhaus, Roland; Weiske, Joerg; Quanz, Maria; Glasauer, Andrea; Nowak-Reppel, Katrin; Bader, Benjamin; Irlbacher, Horst; Meyer, Hanna; Queisser, Nina; Bauser, Marcus; Haegebarth, Andrea; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1986-1992</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MTH1 is a hydrolase responsible for sanitization of oxidized purine nucleoside triphosphates to prevent their incorporation into replicating DNA.  Early tool compds. published in the literature inhibited the enzymic activity of MTH1 and subsequently induced cancer cell death; however recent studies have questioned the reported link between these two events.  Therefore, it is important to validate MTH1 as a cancer dependency with high quality chem. probes.  Here, we present BAY-707, a substrate-competitive, highly potent and selective inhibitor of MTH1, chem. distinct compared to those previously published.  Despite superior cellular target engagement and pharmacokinetic properties, inhibition of MTH1 with BAY-707 resulted in a clear lack of in vitro or in vivo anticancer efficacy either in mono- or in combination therapies.  Therefore, we conclude that MTH1 is dispensable for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHT2Ivo0Gx87Vg90H21EOLACvtfcHk0ljI9sPuCbuBEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSqsrbP&md5=69aa2247fae019cf695df8352e2ade18</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTr%25C3%25A9saugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DB.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DNovel%2520class%2520of%2520potent%2520and%2520cellularly%2520active%2520inhibitors%2520devalidates%2520MTH1%2520as%2520broad-spectrum%2520cancer%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span>Structure
of BAY-885 to be published on <a href="https://www.thesgc.org/donated-chemical-probes" class="extLink">https://www.thesgc.org/donated-chemical-probes</a>; <a href="https://innovate.bayer.com/what-we-offer/chemical-probes-for-open-science/" class="extLink">https://innovate.bayer.com/what-we-offer/chemical-probes-for-open-science/</a> (accessed Nov, 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Structure%0Aof+BAY-885+to+be+published+on+https%3A%2F%2Fwww.thesgc.org%2Fdonated-chemical-probes%3B+https%3A%2F%2Finnovate.bayer.com%2Fwhat-we-offer%2Fchemical-probes-for-open-science%2F+%28accessed+Nov%2C+15%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">A chemical biology view of bioactive small molecules and a binder-based approach to connect biology to precision medicines</span>.  <i>Isr. J. Chem.</i><span class="NLM_year" style="font-weight: bold;">2018</span><span class="refDoi"> DOI: 10.1002/ijch.201800113</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1002%2Fijch.201800113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=S.+L.+Schreiber&title=A+chemical+biology+view+of+bioactive+small+molecules+and+a+binder-based+approach+to+connect+biology+to+precision+medicines&doi=10.1002%2Fijch.201800113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fijch.201800113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijch.201800113%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520chemical%2520biology%2520view%2520of%2520bioactive%2520small%2520molecules%2520and%2520a%2520binder-based%2520approach%2520to%2520connect%2520biology%2520to%2520precision%2520medicines%26jtitle%3DIsr.%2520J.%2520Chem.%26date%3D2018%26doi%3D10.1002%2Fijch.201800113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A.</span></span> Oral presentation at AACR <span class="NLM_year">2018</span>, Chicago, Illinois, USA. <span> </span><span class="NLM_article-title">Inhibitor combinations targeting KRAS effector signaling in KRAS-mutant pancreatic cancer</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=A.+Cox&title=Inhibitor+combinations+targeting+KRAS+effector+signaling+in+KRAS-mutant+pancreatic+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%26date%3D2018%26atitle%3DInhibitor%2520combinations%2520targeting%2520KRAS%2520effector%2520signaling%2520in%2520KRAS-mutant%2520pancreatic%2520cancer" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: An emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+An+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lg0RfGxUtdQJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberspächer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmees, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span> <span> </span><span class="NLM_article-title">Affinity map of bromo-domain protein 4 (BRD4) interactions with the histone H4 tail and the small molecule inhibitor JQ1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9304</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.523019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1074%2Fjbc.M113.523019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9304-9319&author=M.+Jungauthor=M.+Philpottauthor=S.+M%C3%BCllerauthor=J.+Schulzeauthor=V.+Badockauthor=U.+Ebersp%C3%A4cherauthor=D.+Moosmayerauthor=B.+Baderauthor=N.+Schmeesauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=B.+Haendler&title=Affinity+map+of+bromo-domain+protein+4+%28BRD4%29+interactions+with+the+histone+H4+tail+and+the+small+molecule+inhibitor+JQ1&doi=10.1074%2Fjbc.M113.523019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DEbersp%25C3%25A4cher%26aufirst%3DU.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DSchmees%26aufirst%3DN.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHaendler%26aufirst%3DB.%26atitle%3DAffinity%2520map%2520of%2520bromo-domain%2520protein%25204%2520%2528BRD4%2529%2520interactions%2520with%2520the%2520histone%2520H4%2520tail%2520and%2520the%2520small%2520molecule%2520inhibitor%2520JQ1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9304%26epage%3D9319%26doi%3D10.1074%2Fjbc.M113.523019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit22a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span> <span> </span><span class="NLM_article-title">Cell-based protein stabilization assays for the detection of interactions between small-molecule inhibitors and BRD4</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1177/1087057114552398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1177%2F1087057114552398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=25266565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWrurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=180-189&author=J.+Schulzeauthor=D.+Moosmayerauthor=J.+Weiskeauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=C.+Herbstauthor=M.+Jungauthor=B.+Haendlerauthor=B.+Bader&title=Cell-based+protein+stabilization+assays+for+the+detection+of+interactions+between+small-molecule+inhibitors+and+BRD4&doi=10.1177%2F1087057114552398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-based protein stabilization assays for the detection of interactions between small-molecule inhibitors and BRD4</span></div><div class="casAuthors">Schulze, Jessica; Moosmayer, Dieter; Weiske, Joerg; Fernandez-Montalvan, Amaury; Herbst, Christopher; Jung, Marie; Haendler, Bernard; Bader, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">180-189/1-180-189/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4), a member of the bromodomain and extra-terminal (BET) protein family, acts as a central element in transcriptional elongation and plays essential roles in cell proliferation.  Inhibition of BRD4 binding to acetylated histone tails via its two bromodomains, BD1 and BD2, with small-mol. inhibitors has been shown to be a valid strategy to prevent cancer growth.  We have evaluated and established two novel assays that quantify the interaction of transfected BRD4 BD1 with chem. inhibitors inside cultured cells.  Both methods are based on the principle of ligand-induced protein stabilization by which the binding of a small-mol. inhibitor stabilizes intracellular BRD4 BD1 and protects it from proteolytic degrdn.  We demonstrate the universal character of this principle by using two orthogonal, highly sensitive detection technologies for the quantification of BRD4 BD1 levels in cellular lysates: enzyme fragment complementation and time-resolved fluorescence resonance energy transfer (TR-FRET).  Upon optimization of both assays to a miniaturized high-throughput format, the methods were validated by testing a set of small-mol. BET inhibitors and comparing the results with those from a cell-free binding assay and a biophys. thermal shift assay.  In addn., point mutations were introduced into BRD4 BD1, and the corresponding mutants were characterized in the TR-FRET stabilization assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCzgkcUeBYVbVg90H21EOLACvtfcHk0lg0RfGxUtdQJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWrurY%253D&md5=283b113f600aa455b6859ef21b7515c1</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1177%2F1087057114552398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057114552398%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHerbst%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DBader%26aufirst%3DB.%26atitle%3DCell-based%2520protein%2520stabilization%2520assays%2520for%2520the%2520detection%2520of%2520interactions%2520between%2520small-molecule%2520inhibitors%2520and%2520BRD4%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D180%26epage%3D189%26doi%3D10.1177%2F1087057114552398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogg, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel allosteric inhibitor-binding site in ERK5: Comparison with the canonical kinase ATP-binding site</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>D72</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1107/S2059798316004502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1107%2FS2059798316004502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D72&publication_year=2016&pages=682-693&author=H.+Chenauthor=J.+Tuckerauthor=X.+Wangauthor=P.+R.+Gavineauthor=C.+Phillipsauthor=M.+A.+Augustinauthor=P.+Schreinerauthor=S.+Steinbacherauthor=M.+Prestonauthor=D.+Ogg&title=Discovery+of+a+novel+allosteric+inhibitor-binding+site+in+ERK5%3A+Comparison+with+the+canonical+kinase+ATP-binding+site&doi=10.1107%2FS2059798316004502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS2059798316004502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316004502%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DAugustin%26aufirst%3DM.%2BA.%26aulast%3DSchreiner%26aufirst%3DP.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DOgg%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520allosteric%2520inhibitor-binding%2520site%2520in%2520ERK5%253A%2520Comparison%2520with%2520the%2520canonical%2520kinase%2520ATP-binding%2520site%26jtitle%3DActa%2520Crystallogr.%26date%3D2016%26volume%3DD72%26spage%3D682%26epage%3D693%26doi%3D10.1107%2FS2059798316004502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4.</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4.&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makielski, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Properties of hERG channels stably expressed in HEK293 cells studied at physiological temperature</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/S0006-3495(98)77782-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2FS0006-3495%2898%2977782-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1998&pages=230-241&author=Z.+F.+Zhouauthor=Q.+Gongauthor=B.+Yeauthor=Z.+Fanauthor=J.+C.+Makielskiauthor=G.+A.+Robertsonauthor=C.+T.+January&title=Properties+of+hERG+channels+stably+expressed+in+HEK293+cells+studied+at+physiological+temperature&doi=10.1016%2FS0006-3495%2898%2977782-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2898%2977782-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252898%252977782-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%2BF.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DYe%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DMakielski%26aufirst%3DJ.%2BC.%26aulast%3DRobertson%26aufirst%3DG.%2BA.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DProperties%2520of%2520hERG%2520channels%2520stably%2520expressed%2520in%2520HEK293%2520cells%2520studied%2520at%2520physiological%2520temperature%26jtitle%3DBiophys.%2520J.%26date%3D1998%26volume%3D74%26spage%3D230%26epage%3D241%26doi%3D10.1016%2FS0006-3495%2898%2977782-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M. C.</span>, <span class="NLM_string-name">Chandy, K. G.</span>, <span class="NLM_string-name">Grissme, R. S.</span>, <span class="NLM_string-name">Gutman, G. A.</span>, <span class="NLM_string-name">Lazdunski, M.</span>, <span class="NLM_string-name">Mckinnon, D.</span>, <span class="NLM_string-name">Pardo, L. A.</span>, <span class="NLM_string-name">Robertson, G. A.</span>, <span class="NLM_string-name">Rudy, B.</span>, <span class="NLM_string-name">Stuehmer, W.</span>, <span class="NLM_string-name">Wang, X. L.</span></span> <span> </span><span class="NLM_article-title">Voltage-gated potassium channels: Kv11.1 (hERG)</span>. IUPHAR/BPS <i>Guide to Pharmacology</i>, <a href="http://www.guidetopharmacology.org" class="extLink">http://www.guidetopharmacology.org</a> (accessed Nov, 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+C.+Sanguinetti&author=K.+G.+Chandy&author=R.+S.+Grissme&author=G.+A.+Gutman&author=M.+Lazdunski&author=D.+Mckinnon&author=L.+A.+Pardo&author=G.+A.+Robertson&author=B.+Rudy&author=W.+Stuehmer&author=X.+L.+Wang&title=Voltage-gated+potassium+channels%3A+Kv11.1+%28hERG%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26atitle%3DVoltage-gated%2520potassium%2520channels%253A%2520Kv11.1%2520%2528hERG%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamill, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigworth, F. J.</span></span> <span> </span><span class="NLM_article-title">Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches</span>. <i>Pfluegers Arch.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1007/BF00656997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1007%2FBF00656997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=6270629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A280%3ADyaL38%252FjslWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1981&pages=85-100&author=O.+P.+Hamillauthor=A.+Martyauthor=E.+Neherauthor=B.+Sakmannauthor=F.+J.+Sigworth&title=Improved+patch-clamp+techniques+for+high-resolution+current+recording+from+cells+and+cell-free+membrane+patches&doi=10.1007%2FBF00656997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches</span></div><div class="casAuthors">Hamill O P; Marty A; Neher E; Sakmann B; Sigworth F J</div><div class="citationInfo"><span class="NLM_cas:title">Pflugers Archiv : European journal of physiology</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    </div><div class="casAbstract">1.  The extracellular patch clamp method, which first allowed the detection of single channel currents in biological membranes, has been further refined to enable higher current resolution, direct membrane patch potential control, and physical isolation of membrane patches. 2.  A description of a convenient method for the fabrication of patch recording pipettes is given together with procedures followed to achieve giga-seals i.e. pipette-membrane seals with resistances of 10(9) - 10(11) omega. 3.  The basic patch clamp recording circuit, and designs for improved frequency response are described along with the present limitations in recording the currents from single channels. 4.  Procedures for preparation and recording from three representative cell types are given.  Some properties of single acetylcholine-activated channels in muscle membrane are described to illustrate the improved current and time resolution achieved with giga-seals. 5.  A description is given of the various ways that patches of membrane can be physically isolated from cells.  This isolation enables the recording of single channel currents with well-defined solutions on both sides of the membrane.  Two types of isolated cell-free patch configurations can be formed: an inside-out patch with its cytoplasmic membrane face exposed to the bath solution, and an outside-out patch with its extracellular membrane face exposed to the bath solution. 6.  The application of the method for the recording of ionic currents and internal dialysis of small cells is considered.  Single channel resolution can be achieved when recording from whole cells, if the cell diameter is small (less than 20 micrometer). 7.  The wide range of cell types amenable to giga-seal formation is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTk3IMHMxuoQXy-c-E1v-fmfW6udTcc2eaFUmjRlC1Qxrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL38%252FjslWrug%253D%253D&md5=c981295f57b3239ca3719093b059f44d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00656997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00656997%26sid%3Dliteratum%253Aachs%26aulast%3DHamill%26aufirst%3DO.%2BP.%26aulast%3DMarty%26aufirst%3DA.%26aulast%3DNeher%26aufirst%3DE.%26aulast%3DSakmann%26aufirst%3DB.%26aulast%3DSigworth%26aufirst%3DF.%2BJ.%26atitle%3DImproved%2520patch-clamp%2520techniques%2520for%2520high-resolution%2520current%2520recording%2520from%2520cells%2520and%2520cell-free%2520membrane%2520patches%26jtitle%3DPfluegers%2520Arch.%26date%3D1981%26volume%3D391%26spage%3D85%26epage%3D100%26doi%3D10.1007%2FBF00656997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, H.
M.</span></span> <span> </span><span class="NLM_article-title">Suitability of commonly used excipients for electrophysiological in-vitro safety pharmacology assessment of effects on hERG potassium current and on rabbit Purkinje fiber action potential</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">158</span>, <span class="refDoi"> DOI: 10.1016/j.vascn.2007.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2Fj.vascn.2007.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=17590357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCmtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=145-158&author=H.%0AM.+Himmel&title=Suitability+of+commonly+used+excipients+for+electrophysiological+in-vitro+safety+pharmacology+assessment+of+effects+on+hERG+potassium+current+and+on+rabbit+Purkinje+fiber+action+potential&doi=10.1016%2Fj.vascn.2007.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Suitability of commonly used excipients for electrophysiological in-vitro safety pharmacology assessment of effects on hERG potassium current and on rabbit Purkinje fiber action potential</span></div><div class="casAuthors">Himmel, Herbert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Introduction: Regulatory guidelines require investigation of the liability for delayed ventricular repolarization by new chem. entities within a broad concn. range in-vitro.  However, investigation can be limited by poor drug aq. soly., and by solvent physicochem. attributes that disrupt cell membrane integrity.  Although excipients or solubilizing agents may aid to achieve the necessary high concns., no comprehensive overview on the suitability of solvents for in-vitro electrophysiol. safety studies exists.  Methods: Excipients were tested for potential interference with the hERG (human ether-a-go-go-related gene) K+ current (whole-cell voltage-clamp, 23±2°), and the shape of rabbit Purkinje fiber action potentials (conventional glass microelectrode technique, 37±1°).  Results and discussion: Water-sol. complexation builders/carriers had little effect on hERG K+ current at up to 50 mg/mL (BSA, bovine serum albumin) and 11 mg/mL (HP-β-CD, hydroxypropyl-β-cyclodextrin; IC20, concn. of 20% inhibition).  Water-sol. org. (co)solvents inhibited hERG K+ currents (IC20, %/mM): 0.7/152, ethanol; 0.9/67, Transcutol; 1.2/154, DMSO (dimethylsulfoxide); 1.6/389, acetonitrile; 1.9/48, polyethylene glycol 400; 2.1/660, methanol.  Part of their inhibitory effect is attributed to the osmolality of extracellular solns., because hERG IC20 and extrapolated osmolality at the hERG IC20 strongly correlate.  Water-sol. nonionic solubilizers/surfactants are potent inhibitors of hERG K+ current with IC20 concns. of 0.07% (Cremophor EL) or lower (Tween 20, Tween 80: ∼ 0.001%).  Part of this inhibitory effect is attributed to their interaction with lipid membranes, because hERG inhibition occurs close to crit. micelle concns. (Cremophor, ∼0.009%; Tween 20, ∼0.007%).  Purkinje fiber action potentials are little affected by HP-β-CD at up to 2 mg/mL, while DMSO tends to shorten the action potential duration at 1%.  Conclusion: When conducting electrophysiol. in-vitro assessments of drug effects, solubilizers/surfactants (Cremophor EL, Tween 20, Tween 80) should be avoided.  Instead, water-sol. org. (co)solvents (methanol, acetonitrile, DMSO) or complexation builders/carriers (HP-β-CD, BSA) appear to be more favorable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsEAdKY44zz7Vg90H21EOLACvtfcHk0ljFVIcSDeIhdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCmtLbN&md5=49c61123b2fd0c35fcae026a5d45977f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2007.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2007.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DHimmel%26aufirst%3DH.%2BM.%26atitle%3DSuitability%2520of%2520commonly%2520used%2520excipients%2520for%2520electrophysiological%2520in-vitro%2520safety%2520pharmacology%2520assessment%2520of%2520effects%2520on%2520hERG%2520potassium%2520current%2520and%2520on%2520rabbit%2520Purkinje%2520fiber%2520action%2520potential%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2007%26volume%3D56%26spage%3D145%26epage%3D158%26doi%3D10.1016%2Fj.vascn.2007.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span>, <span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Solution Stability Methods</span>. In  <i>Drug-like Properties: Concepts, Structure Design and Methods</i>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Burlington, MA</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">356</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;key=10.1016%2FB978-012369520-8.50032-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=353-356&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FB978-012369520-8.50032-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-012369520-8.50032-2%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DSolution%2520Stability%2520Methods%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%26pub%3DAcademic%2520Press%26date%3D2008%26spage%3D353%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Onofrey, T.</span>, <span class="NLM_string-name">Kazan, G.</span></span> <span> </span><span class="NLM_article-title">Performance and correlation of a 96-well high throughput screening method to determine aqueous drug solubility</span>; Millipore Corporation Application Note, <span class="NLM_year">2003</span>; Lit. No. AN1731EN00.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=T.+Onofrey&author=G.+Kazan&title=Performance+and+correlation+of+a+96-well+high+throughput+screening+method+to+determine+aqueous+drug+solubility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DOnofrey%26aufirst%3DT.%26atitle%3DPerformance%2520and%2520correlation%2520of%2520a%252096-well%2520high%2520throughput%2520screening%2520method%2520to%2520determine%2520aqueous%2520drug%2520solubility%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minick, D.J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenz, J.H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, M.A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brent, D.A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive method for determining hydrophobicity constants by reversed-phase high performance liquid chromatography</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1021/jm00118a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00118a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1923-1933&author=D.J.+Minickauthor=J.H.+Frenzauthor=M.A.+Patrickauthor=D.A.+Brent&title=A+comprehensive+method+for+determining+hydrophobicity+constants+by+reversed-phase+high+performance+liquid+chromatography&doi=10.1021%2Fjm00118a010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm00118a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00118a010%26sid%3Dliteratum%253Aachs%26aulast%3DMinick%26aufirst%3DD.J.%26aulast%3DFrenz%26aufirst%3DJ.H.%26aulast%3DPatrick%26aufirst%3DM.A.%26aulast%3DBrent%26aufirst%3DD.A.%26atitle%3DA%2520comprehensive%2520method%2520for%2520determining%2520hydrophobicity%2520constants%2520by%2520reversed-phase%2520high%2520performance%2520liquid%2520chromatography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1923%26epage%3D1933%26doi%3D10.1021%2Fjm00118a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM','PDB','6HKM'); return false;">PDB: 6HKM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN','PDB','6HKN'); return false;">PDB: 6HKN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IC8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IC8','PDB','4IC8'); return false;">PDB: 4IC8</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i44"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01606">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12620"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01606">10.1021/acs.jmedchem.8b01606</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Reference compounds used for optimization of cellular mechanistic assay; results of commercial kinase panel at Eurofins for ERK5 probe molecule <b>41 (BAY-885)</b>; crystallographic data of compounds <b>1</b> and <b>35</b>; data collection and refinement statistics; syntheses procedures for compounds <b>5</b>–<b>40</b> and <b>42</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_001.pdf">jm8b01606_si_001.pdf (1.1 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01606/suppl_file/jm8b01606_si_002.csv">jm8b01606_si_002.csv (2.65 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for the described crystal structures have been deposited with the Protein Data Bank (PDB). The PDB accession codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKM">6HKM</a> for compound <b>1</b> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HKN">6HKN</a> for compound <b>35</b>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01606&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-2%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01606%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01606" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997bcc9a7824eb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
