
<html lang="en"     class="pb-page"  data-request-id="7b92360d-30eb-4ece-a3d3-f707414e2953"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-13;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b00021;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Angelo  Aguilar" /></meta><meta name="dc.Creator" content="Ke  Zheng" /></meta><meta name="dc.Creator" content="Tianfeng  Xu" /></meta><meta name="dc.Creator" content="Shilin  Xu" /></meta><meta name="dc.Creator" content="Liyue  Huang" /></meta><meta name="dc.Creator" content="Ester  Fernandez-Salas" /></meta><meta name="dc.Creator" content="Liu  Liu" /></meta><meta name="dc.Creator" content="Denzil  Bernard" /></meta><meta name="dc.Creator" content="Kaitlin P.  Harvey" /></meta><meta name="dc.Creator" content="Caroline  Foster" /></meta><meta name="dc.Creator" content="Donna  McEachern" /></meta><meta name="dc.Creator" content="Jeanne  Stuckey" /></meta><meta name="dc.Creator" content="Krishnapriya  Chinnaswamy" /></meta><meta name="dc.Creator" content="James  Delproposto" /></meta><meta name="dc.Creator" content="Jeff W.  Kampf" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="Inhibition of the menin-mixed lineage leukemia (MLL) protein–protein interaction is a promising new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia). We d..." /></meta><meta name="Description" content="Inhibition of the menin-mixed lineage leukemia (MLL) protein–protein interaction is a promising new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia). We d..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 22, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00021" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00021" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00021" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00021" /></link>
        
    
    

<title>Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00021" /></meta><meta property="og:title" content="Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0020.jpeg" /></meta><meta property="og:description" content="Inhibition of the menin-mixed lineage leukemia (MLL) protein–protein interaction is a promising new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia). We describe herein our structure-based design, synthesis, and evaluation of a new class of small-molecule inhibitors of the menin-MLL interaction (hereafter called menin inhibitors). Our efforts have resulted in the discovery of highly potent menin inhibitors, as exemplified by compound 42 (M-89). M-89 binds to menin with a Kd value of 1.4 nM and effectively engages cellular menin protein at low nanomolar concentrations. M-89 inhibits cell growth in the MV4;11 and MOLM-13 leukemia cell lines carrying MLL fusion with IC50 values of 25 and 55 nM, respectively, and demonstrates &gt;100-fold selectivity over the HL-60 leukemia cell line lacking MLL fusion. The determination of a co-crystal structure of M-89 in a complex with menin provides the structural basis for their high-affinity interaction. Further optimization of M-89 may lead to a new class of therapy for the treatment of MLL leukemia." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00021"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00021">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00021&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00021&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00021&amp;href=/doi/10.1021/acs.jmedchem.9b00021" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 6015-6034</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00654" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00145" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Angelo Aguilar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angelo Aguilar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angelo++Aguilar">Angelo Aguilar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ke Zheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ke Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ke++Zheng">Ke Zheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tianfeng Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tianfeng Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tianfeng++Xu">Tianfeng Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shilin Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shilin Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shilin++Xu">Shilin Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liyue Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liyue Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liyue++Huang">Liyue Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ester Fernandez-Salas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ester Fernandez-Salas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ester++Fernandez-Salas">Ester Fernandez-Salas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liu++Liu">Liu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Denzil Bernard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Denzil Bernard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Denzil++Bernard">Denzil Bernard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kaitlin P. Harvey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaitlin P. Harvey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaitlin+P.++Harvey">Kaitlin P. Harvey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Caroline Foster</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline Foster</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline++Foster">Caroline Foster</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Donna McEachern</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donna McEachern</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donna++McEachern">Donna McEachern</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeanne Stuckey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanne Stuckey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne++Stuckey">Jeanne Stuckey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Krishnapriya Chinnaswamy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krishnapriya Chinnaswamy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krishnapriya++Chinnaswamy">Krishnapriya Chinnaswamy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James Delproposto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Delproposto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Delproposto">James Delproposto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeff W. Kampf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeff W. Kampf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeff+W.++Kampf">Jeff W. Kampf</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Departments of Internal Medicine,  Pharmacology,  Medicinal Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#3a49525b55575f545d7a4f57535952145f5e4f"><span class="__cf_email__" data-cfemail="84f7ece5ebe9e1eae3c4f1e9ede7ecaae1e0f1">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00021&amp;href=/doi/10.1021%2Facs.jmedchem.9b00021" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 6015–6034</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 22, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 January 2019</li><li><span class="item_label"><b>Published</b> online</span>22 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 July 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00021</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6015%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAngelo%2BAguilar%252C%2BKe%2BZheng%252C%2BTianfeng%2BXu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D13%26contentID%3Dacs.jmedchem.9b00021%26title%3DStructure-Based%2BDiscovery%2Bof%2BM-89%2Bas%2Ba%2BHighly%2BPotent%2BInhibitor%2Bof%2Bthe%2BMenin-Mixed%2BLineage%2BLeukemia%2B%2528Menin-MLL%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6034%26publicationDate%3DJuly%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00021"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3550</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00021" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Angelo&quot;,&quot;last_name&quot;:&quot;Aguilar&quot;},{&quot;first_name&quot;:&quot;Ke&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Tianfeng&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Shilin&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Liyue&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Ester&quot;,&quot;last_name&quot;:&quot;Fernandez-Salas&quot;},{&quot;first_name&quot;:&quot;Liu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Denzil&quot;,&quot;last_name&quot;:&quot;Bernard&quot;},{&quot;first_name&quot;:&quot;Kaitlin&quot;,&quot;last_name&quot;:&quot;P. Harvey&quot;},{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;Foster&quot;},{&quot;first_name&quot;:&quot;Donna&quot;,&quot;last_name&quot;:&quot;McEachern&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;Stuckey&quot;},{&quot;first_name&quot;:&quot;Krishnapriya&quot;,&quot;last_name&quot;:&quot;Chinnaswamy&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Delproposto&quot;},{&quot;first_name&quot;:&quot;Jeff&quot;,&quot;last_name&quot;:&quot;W. Kampf&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6015-6034&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00021&quot;},&quot;abstract&quot;:&quot;Inhibition of the menin-mixed lineage leukemia (MLL) protein–protein interaction is a promising new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia). We describe herein our structure-based design, synthesis, and evaluation of a new class of small-molecule inhibitors of the menin-MLL interaction (hereafter called menin inhibitors). Our efforts have resulted in the discovery of highly potent menin inhibitors, as exemplified by compound 42 (M-89). M-89 binds to menin with a Kd value of 1.4 nM and effectively engages cellular menin protein at low nanomolar concentrations. M-89 inhibits cell growth in the MV4;11 and MOLM-13 leukemia cell lines carrying MLL fusion with IC50 values of 25 and 55 nM, respectively, and demonstrates &gt;100-fold selectivity over the HL-60 leukemia cell line lacking MLL fusion. The determination of a co-crystal structure of M-89 in a complex with menin provides the structural basis for their high-affinity interaction. Further optimization of M-89 m&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00021&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00021" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00021&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00021" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00021&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00021" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00021&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00021&amp;href=/doi/10.1021/acs.jmedchem.9b00021" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00021" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00021" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00021%26sid%3Dliteratum%253Aachs%26pmid%3D31244110%26genre%3Darticle%26aulast%3DAguilar%26date%3D2019%26atitle%3DStructure-Based%2BDiscovery%2Bof%2BM-89%2Bas%2Ba%2BHighly%2BPotent%2BInhibitor%2Bof%2Bthe%2BMenin-Mixed%2BLineage%2BLeukemia%2B%2528Menin-MLL%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D13%26spage%3D6015%26epage%3D6034%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/jmcmar.2019.62.issue-13/20190711/jmcmar.2019.62.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of the menin-mixed lineage leukemia (MLL) protein–protein interaction is a promising new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia). We describe herein our structure-based design, synthesis, and evaluation of a new class of small-molecule inhibitors of the menin-MLL interaction (hereafter called menin inhibitors). Our efforts have resulted in the discovery of highly potent menin inhibitors, as exemplified by compound <b>42</b> (M-89). M-89 binds to menin with a <i>K</i><sub>d</sub> value of 1.4 nM and effectively engages cellular menin protein at low nanomolar concentrations. M-89 inhibits cell growth in the MV4;11 and MOLM-13 leukemia cell lines carrying MLL fusion with IC<sub>50</sub> values of 25 and 55 nM, respectively, and demonstrates >100-fold selectivity over the HL-60 leukemia cell line lacking MLL fusion. The determination of a co-crystal structure of M-89 in a complex with menin provides the structural basis for their high-affinity interaction. Further optimization of M-89 may lead to a new class of therapy for the treatment of MLL leukemia.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mixed lineage leukemia (MLL, also called MLL1 to distinguish it from MLL2-4) protein is a histone methyltransferase and specifically methylates histone H3 lysine 4 residue (H3K4). MLL gene rearrangement is found in 5–10% of acute myeloid leukemia in adults and almost 70% of acute lymphoblastic leukemia in infants.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Adult leukemia patients carrying a MLL rearrangement, or MLL leukemia, have very poor prognosis with current treatments.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a>Accordingly, there is an urgent need to develop new and effective therapies for the treatment of MLL leukemia.</div><div class="NLM_p">The most common MLL rearrangements are balanced MLL translocations, in which only one MLL allele is truncated and fused with one of over 70 partners.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Approximately 1400 amino acids from the MLL N-terminus are retained in all of the MLL fusion proteins and interact directly with the oncogenic co-factor menin.<a onclick="showRef(event, 'ref3 ref8 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref3 ref8 ref10 ref11 ref12 ref13 ref14">(3,8,10−14)</a> The menin-MLL protein–protein interaction is essential for the expression of <i>HOX</i> and <i>MEIS1</i> genes, which drive leukemogenesis in MLL leukemia.<a onclick="showRef(event, 'ref12 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref12 ref14 ref15">(12,14,15)</a> Consequently, targeting the menin-MLL protein–protein interaction has been proposed to be a promising, targeted therapeutic strategy for the treatment of MLL leukemia.</div><div class="NLM_p">Structural studies have shown that menin protein has an approximately 5000 Å<sup>3</sup> cavity, which serves as the binding site for MLL protein.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> This very large binding site suggests that targeting the menin-MLL interaction using nonpeptide small-molecules, hereafter also called menin inhibitors, could be quite challenging. However, recent efforts have led to the discovery of potent, nonpeptide small-molecule menin inhibitors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22">(18−22)</a> The most promising noncovalent menin inhibitors reported to date are <b>1</b> (MI-463), <b>2</b> (MI-503), <b>3</b> (MI-538), and <b>5</b> (MI-1481), which belong to the same chemical class of 5-(piperidin-1-ylmethyl)-1<i>H</i>-indoles. These menin inhibitors bind to menin with a <i>K</i><sub>d</sub> value of ∼10 nM and achieve sub-micromolar cellular activity in MLL leukemia cell lines and good antitumor activity in vivo against MLL leukemia tumors.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Importantly, small-molecule menin inhibitors can effectively block oncogenic transformation by MLL fusion proteins in a manner independent of the MLL fusion partner,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> suggesting a broad therapeutic potential of menin inhibitors for the treatment of MLL leukemia. These studies have provided an important preclinical proof-of-concept that small-molecule inhibitors targeting the menin-MLL protein–protein interaction may have a promising therapeutic potential for the treatment of human MLL leukemia.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previously reported reversible and irreversible small-molecule menin inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recently, we reported the discovery of M-525 as a first-in-class, highly potent, covalent menin inhibitor.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> M-525 binds to menin covalently with an IC<sub>50</sub> value of 3 nM and achieves low nanomolar cellular potencies in inhibition of cell growth in a panel of leukemia cell lines carrying MLL fusion.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> M-525 represents a promising covalent menin inhibitor for further optimization.</div><div class="NLM_p last">Despite these major progresses, the development of menin inhibitors for the treatment of MLL leukemia is still in its early stage. In the present study, we describe our structure-based design and synthesis of menin inhibitors, which has yielded a class of potent, noncovalent menin inhibitors. The most potent compound, <b>42</b> (M-89), binds to menin with a <i>K</i><sub>d</sub> value of 1.4 nM, achieves low nanomolar potency in the inhibition of cell growth in human MLL leukemia cell lines, and demonstrates >100-fold selectivity over leukemia cells lacking the MLL fusion. M-89 represents a class of promising noncovalent menin inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We employed a previously reported menin inhibitor <b>4</b> (MIV-6, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> as the starting point for our optimization efforts based on the following considerations: (1) MIV-6 has a good binding affinity to menin with a reported <i>K</i><sub>d</sub> = 85 nM; (2) MIV-6 is cell permeable and inhibits growth of the human leukemia MV4;11 cell line with MLL-AF4 fusion with a reported IC<sub>50</sub> = 2.8 μM; (3) Its available co-crystal structure in a complex with menin provides a solid foundation for effective structure-based optimization.</div><div class="NLM_p">Comparison of previously published co-crystal structures of MIV-6/menin and MLL/menin shows that MIV-6 captures the interactions of the Phe9, Pro10, and Pro13 residues in MLL with menin (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Additionally, the nitrile of the benzonitrile “tail” group in MIV-6 forms an additional hydrogen bond with the NH of Trp341 of the menin protein.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Superposition of the co-crystal structure of menin in complex with menin binding motif 1 (MBM1 in yellow lines, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6">4GQ6</a>) from MLL onto the co-crystal structure of menin in complex with MIV-6 (magenta sticks) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>). Menin protein is shown in the gray surface. (B) Our proposed four modification sites in MIV-6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Conformational constraint of a small-molecule inhibitor not only can enhance the binding affinity to its intended target protein by reducing conformational entropic costs upon binding but also can improve the binding selectivity by reducing accessible, low-energy conformational space. Therefore, we investigated the possibilities of locking the bound conformation of MIV-6 to menin. Our analysis of the co-crystal structure of MIV-6/menin suggested that the primary amino group in MIV-6 can be cyclized with its adjacent phenyl group, which led to the design of compound <b>6</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This compound (<b>6</b>) was synthesized and was found to bind to menin with a <i>K</i><sub>i</sub> = 22 nM, which is 2–3 times more potent than MIV-6 in our binding assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), supporting our conformational constraint strategy.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Optimization of MIV-6 through Cyclization of the Core, Rigidification of the Linker, and Replacement of the Nitrile Tail Group with a Substituted Sulfonyl Group<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0013.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0014.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values are average of at least two tests. <i>K</i><sub>i</sub>’s were determined from averaged IC<sub>50</sub> values.</p></div></div><div></div></div><div class="NLM_p">Encouraged by the successful design of compound <b>6</b>, we next focused on the rigidification of the flexible linker (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Since the flexible linker in the MIV-6/menin complex does not adopt an extended conformation, we investigated if other linker lengths can adopt a favorable conformation for interaction with the menin protein and give insight to the design of the rigid linker. We tested compounds with a linker one atom shorter (<b>7</b>) and a linker with one atom longer (<b>8</b>) than that in <b>6</b>. Both modifications significantly reduced the binding affinity to menin, suggesting that a linker with four atoms has the optimal length between the core-piperidine nitrogen and the para-position of the benzonitrile group. To rigidify the linker, we explored several cyclic amines, compounds <b>9</b>–<b>12</b>, that maintain approximately the four-atom linker length. Compounds <b>9</b> and <b>10</b>, containing a piperidine in the linker, are approximately 10- and 100-times less potent than <b>6</b>, whereas compound <b>11</b>, containing a pyrrolidine in the linker, is as potent as <b>6</b>. Compound <b>12</b> containing an azetidine is 2 times more potent than <b>6</b>. Hence, through cyclization of the core and rigidification of the linker, we obtained compound <b>12</b>, which is 5 times more potent than MIV-6 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">The co-crystal structure of MIV-6/menin (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) shows that the terminal nitrile group forms a hydrogen bond with the indole NH group in Trp341. Additionally, there is a well-defined hydrophobic pocket formed by Trp341, Val371, Cys329, Val367, and Ala325 adjacent to the nitrile group, which can be utilized to further enhance the binding affinity to menin. To maintain the hydrogen bonding interaction and at the same time to capture additional interactions with this well-defined hydrophobic pocket, we replaced the nitrile group in MIV-6 with a substituted sulfonyl group (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">Replacing the terminal nitrile group with a simple methylsulfonyl group resulted in <b>13</b>, which has a <i>K</i><sub>i</sub> value of 45 nM to menin and is thus 4 times less potent than <b>12</b>. However, replacing the methyl group in <b>13</b> with an ethyl group, which yielded <b>14</b>, restored the binding affinity (<i>K</i><sub>i</sub> = 10 nM). Encouraged with the strong binding affinity for <b>14</b>, we next systematically explored this site using a variety of alkyl- or aryl-substituted sulfonyl groups and obtained the results shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Replacing the methyl group in <b>13</b> with an isopropyl group led to <b>15</b>, which is 2 times more potent than <b>13</b> but 2 times less potent than <b>14</b> in binding to menin. Changing the isopropyl group in <b>15</b> to a cyclopropyl group led to <b>16</b>, which is 2–3 times more potent than <b>15</b>. Encouraged by the high binding affinity of <b>16</b>, we replaced the cyclopropyl group in <b>16</b> with a cyclobutyl (<b>17</b>), cyclopentyl (<b>18</b>), or a cyclohexyl group (<b>19</b>). Compounds <b>17</b>–<b>19</b> are all high-affinity menin inhibitors with <i>K</i><sub>i</sub> values of 11, 4.6, and 2.4 nM to menin, respectively. We next replaced the cyclohexyl group in <b>19</b> with a phenyl or a pyridinyl group, which yielded <b>20</b> and <b>21</b>, respectively. Compounds <b>20</b> and <b>21</b> bind to menin with <i>K</i><sub>i</sub> values of 5.6 and 4.6 nM, respectively.</div><div class="NLM_p">We next evaluated these menin inhibitors for their cell growth inhibitory activity in the MV4;11 cell line carrying MLL-AF4 fusion, which was shown to be sensitive to menin inhibitors.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The data are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_p">Consistent with their improved binding affinity to menin over MIV-6, a number of these new menin inhibitors are also more potent than MIV-6 in the inhibition of cell growth in the MV4;11 cell line (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). For example, compounds <b>18</b>–<b>21</b> have IC<sub>50</sub> values 0.9, 1.1, 0.5, and 1.3 μM, respectively, and are 3–7 times more potent than MIV-6. However, the most potent compound <b>20</b> in this series only achieves a sub-micromolar cellular potency, indicating the need for further optimization.</div><div class="NLM_p">Although compound <b>20</b> is more potent than <b>21</b> in the inhibition of cell growth in the MV4;11 cell line, <b>21</b> is more soluble than <b>20</b>. Accordingly, we used compound <b>21</b> as the template for further optimization.</div><div class="NLM_p">We moved the nitrogen atom in the tetrahydroisoquinoline core one atom away from the quaternary carbon center in compound <b>21</b> to render the functionalization of this nitrogen atom more synthetically feasible, which yielded compound <b>22</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compound <b>22</b> binds to menin with a <i>K</i><sub>i</sub> = 1.7 nM and is, therefore, 2–3 times more potent than <b>21</b>. However, <b>22</b> is only slightly more potent than <b>21</b> in the inhibition of cell growth in the MV4;11 cell line. To further improve the cellular potency, we introduced a variety of small hydrophobic groups as a substituent on the nitrogen atom (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Substitution of the nitrogen with a small, aliphatic group, such as methyl (<b>23</b>), ethyl (<b>24</b>), isopropyl (<b>25</b>), and cyclobutyl (<b>26</b>), resulted in compounds with high binding affinities to menin, with <i>K</i><sub>i</sub> values = 1–2 nM. Substitution of the nitrogen atom with an electron-withdrawing group, such as benzylpyridine (<b>29</b>), however, significantly reduces the binding affinity to menin. Despite the high binding affinities to menin, compounds <b>23</b>–<b>26</b> only achieve IC<sub>50</sub> values of ∼0.4–1 μM in the inhibition of cell growth in the MV4;11 cell line, suggesting that a much greater improvement in binding affinity is needed to achieve a much more potent cellular activity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationship of the N-Substituents<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0015.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0016.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values are average of at least two tests. <i>K</i><sub>i</sub> was calculated from an averaged IC<sub>50</sub> value in the binding experiments for each compound.</p></div></div><div></div></div><div class="NLM_p">In the co-crystal structures of MIV-6/menin and MLL/menin (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), there is a well-defined binding pocket (the P1 pocket) adjacent to the cyclopentyl group and formed by Cys241, Tyr276, and Met278 residues of menin, and this pocket is accessed by an alanine residue (Ala11) in MLL (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). We reasoned that the installation of an appropriate group onto the cyclopentyl ring to capture additional interactions with the residues forming the P1 pocket should greatly enhance the binding affinity. Since it was synthetically difficult to install diverse groups onto the cyclopentyl ring in both MIV-6 and <b>23</b>, we synthesized <b>31</b>, in which the quaternary amino group in the core of MIV-6 is replaced by a nitrile group, but the rigid linker in <b>23</b> is retained. Compound <b>31</b> binds to menin with a modest affinity (<i>K</i><sub>i</sub> = 0.96 μM) and is, thus, 16 times weaker than MIV-6, suggesting that the basic nitrogen in <b>23</b> and its analogues greatly enhance the binding affinity to menin as compared to the neutral nitrile group in <b>31</b>. Despite the modest binding affinity for <b>31</b>, it provided us with a synthetically more accessible template molecule with which to explore the P1 pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modifications of the cyclopentyl group of MIV-6 to access the P1 pocket, which was utilized by MBM1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6">4GQ6</a>) but not by MIV-6 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Since the P1 pocket is hydrophobic in nature, we decided to install a small hydrophobic group onto the cyclopentyl ring, with the results summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. An alkoxy group was placed on the cyclopentyl ring, adjacent to the bond linking the cyclopentyl ring to the tertiary carbon atom, and this produced two diastereomers (<b>32</b> and <b>33</b>) with relative stereochemistry. Compounds <b>32</b> and <b>33</b> bind to menin with similar affinities (<i>K</i><sub>i</sub> values = 4.9 and 6.1 μM), which are weaker than compound <b>31</b>. When the methoxy group in <b>32</b> is replaced by an ethoxy group, the resulting compound <b>34</b> has an improved binding affinity (<i>K</i><sub>i</sub> = 1.7 μM) over <b>32</b> but is still a weaker binder than <b>31</b>. When the methoxy group in <b>32</b> was replaced by an acetoxyl group, the resulting compound <b>35</b> has a <i>K</i><sub>i</sub> = 12 nM and is thus 80 times more potent than <b>31</b>. However, when the methoxy group in <b>33</b>, the diastereomer of <b>32</b>, was replaced by an acetoxyl group, the resulting compound (<b>36</b>) has a <i>K</i><sub>i</sub> value of 1.1 μM and is equipotent with <b>31</b>. The difference in binding affinities of the diastereomers <b>35</b> and <b>36</b> to menin clearly demonstrates the stereospecific nature of the binding to menin.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure–Activity Relationship of Cyclopentyl-Substituted Compounds<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0017.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0018.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are average of at least three independent experiments. <i>K</i><sub>i</sub> values were calculated from averaged IC<sub>50</sub> values.</p></div></div><div></div></div><div class="NLM_p">Encouraged by its excellent binding affinity, we performed further modifications of <b>35</b> to optimize the interactions with the P1 pocket (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Changing the acetoxyl group in <b>35</b> to a propionoxyl generated <b>37</b>, which has a <i>K</i><sub>i</sub> value of 9.7 nM, slightly more potent than <b>35</b>. However, converting the propionoxyl group in <b>37</b> to a butyroxyl group, producing <b>38</b>, results in a 30-fold loss in binding affinity to menin, consistent with the limited space in the P1 pocket.</div><div class="NLM_p">Because ester groups are typically not metabolically stable, we converted the acetoxyl group in <b>37</b> into a methyl carbamate (<b>39</b>), an acetamide (<b>40</b>), or a reverse carbamate (<b>41</b>). Compounds <b>39</b>–<b>41</b> bind to menin with <i>K</i><sub>i</sub> values of 2.7, 3.2, and 8.5 nM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), and are, therefore, high-affinity menin inhibitors.</div><div class="NLM_p">These menin inhibitors in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> were evaluated for their inhibition of cell growth in the MV4;11 cell line, giving the results in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The majority of these menin inhibitors has IC<sub>50</sub> values only in the micromolar range, but compound <b>39</b> with the highest binding affinity achieves the best IC<sub>50</sub> value of 0.3 μM in the inhibition of cell growth in the MV4;11 cell line among this series of compounds.</div><div class="NLM_p">Taken together, the data show that the substitution of a methyl carbamate in the cyclopentyl group of the modestly potent inhibitor <b>31</b>, which generates <b>39</b>, enhances the binding to menin by a factor of >100 and the cellular activity in the MV4;11 cell line by a factor of >50. These data indicate that the additional interactions with residues in the P1 pocket greatly enhance the binding affinity to menin and cellular activity.</div><div class="NLM_p">We next combined the best structural modifications in <b>23</b> and <b>39</b> and designed and synthesized compound <b>42</b> (M-89) (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Design of Compound <b>42</b> (M-89) by Combination of the Best Structural Features in Compounds <b>23</b> and <b>39</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0019.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">binding affinity to menin</th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">cell growth inhibition (IC<sub>50</sub>, μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">MV4-11 (μM)</th><th class="colsep0 rowsep0" align="center" char="±">MOLM-13 (μM)</th><th class="colsep0 rowsep0" align="center" char="±">HL-60 (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char="±">0.011 ± 0.0002</td><td class="colsep0 rowsep0" align="left">0.002</td><td class="colsep0 rowsep0" align="char" char="±">0.384 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">4.3 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">2.5 ± 0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b> (M-89)</td><td class="colsep0 rowsep0" align="char" char="±">0.005 ± 0.001</td><td class="colsep0 rowsep0" align="left"><0.001</td><td class="colsep0 rowsep0" align="char" char="±">0.025 ± 0.004</td><td class="colsep0 rowsep0" align="char" char="±">0.054 ± 0.015</td><td class="colsep0 rowsep0" align="char" char="±">10.2 ± 1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the average of at least three independent experiments. <i>K</i><sub>i</sub> values for binding affinities were calculated from average IC<sub>50</sub> values.</p></div></div></div><div class="NLM_p">In our fluorescence polarization (FP)-based binding assay, M-89 achieves an IC<sub>50</sub> value of 5 nM with an estimated <i>K</i><sub>i</sub> value <1 nM. Because the binding affinity of M-89 exceeds the lower limit in our FP-based assay, we employed a biolayer interferometry (BLI) assay to further evaluate its binding affinity. The BLI assay determined that M-89 has a <i>K</i><sub>d</sub> value of 1.4 nM, a <i>k</i><sub>off</sub> value of 2.9 × 10<sup>–4</sup> s<sup>–1</sup>, and a <i>k</i><sub>on</sub> value of 2.3 × 10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup>. In comparison, MIV-6 has a <i>K</i><sub>d</sub> value of 110 nM, a <i>k</i><sub>off</sub> value of 460 × 10<sup>–4</sup> s<sup>–1</sup>, and a <i>k</i><sub>on</sub> value of 4.1 × 10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup> in our experiments. Hence, M-89 has a high affinity to menin and an off-rate, which is 158 times slower than that of MIV-6 (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Determination of <i>K</i><sub>d</sub>, <i>k</i><sub>on</sub>, and <i>k</i><sub>off</sub> Values for Two Menin Inhibitors with a Label Free, Biolayer Interferometry Assay Using an Octet RED System</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM) to menin by FP</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>on</sub> (10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>off</sub> (10<sup>–4</sup> s<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIV-6</td><td class="colsep0 rowsep0" align="char" char="±">229 ± 29</td><td class="colsep0 rowsep0" align="char" char="±">110 ± 29</td><td class="colsep0 rowsep0" align="char" char="±">4.1 ± 1.1</td><td class="colsep0 rowsep0" align="char" char="±">460 ± 102</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">M-89</td><td class="colsep0 rowsep0" align="char" char="±">5.0 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">2.3 ± 1.1</td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 0.2</td></tr></tbody></table></div></div><div class="NLM_p">M-89 was evaluated for its ability to inhibit cell growth in the MV4;11 leukemia cell line carrying MLL-AF4 fusion and in the HL-60 leukemia cell line lacking the MLL fusion. Our data showed that M-89 potently inhibits cell growth in the MV4;11 cell line, achieving an IC<sub>50</sub> value of 25 nM. In comparison, M-89 has an IC<sub>50</sub> value of 10.2 μM in the HL-60 cell line. We further evaluated M-89 for its cell growth inhibitory activity in the MOLM-13 leukemia cell line carrying MLL-AF9 fusion and found that it achieves an IC<sub>50</sub> value of 54 nM. Thus, M-89 displays a potent cell growth inhibitory activity in the MV4;11 and MOLM-13 leukemia cell lines carrying MLL fusion and >100-fold specificity over the HL-60 cell line lacking MLL fusion.</div><div class="NLM_p">To gain structural insights into the high binding affinity of M-89 with menin, we determined their co-crystal structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A">6E1A</a>). This co-crystal structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) shows that M-89 not only maintains the key interactions observed for MIV-6 but captures additional interactions. Consistent with our design, the carbamate group on the cyclopentyl fits precisely into the available P1 pocket in menin, and the carbonyl group forms a strong hydrogen bond with the hydroxyl group of Tyr276 of menin. The sulfonyl group forms a strong hydrogen bond with the NH of Trp341, and the pyridyl group has additional hydrophobic contacts with menin. The inflexible azetidine linker constrains the molecule into a conformation ideal for effective interactions with menin. This co-crystal structure provides structural insights into the high-affinity binding of M-89 with menin and a solid structural basis for further structure-based optimization.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Co-crystal structure of M-89 bound to menin (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A">6E1A</a>) (left panel) and superposition of MIV-6 and M-89 based upon their co-crystal structures (right panel).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The cellular thermal shift assay (CETSA) is a powerful assay with which to examine cellular protein thermal stability and to determine if a small-molecule inhibitor targets a specific protein in cells.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> We employed the CETSA assay to assess if M-89 stabilizes menin protein in cells. Our CETSA data (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) show that M-89 stabilizes cellular menin protein in both MV4;11 and MOLM-13 cells in a dose-dependent manner. It significantly enhances the thermal stability of cellular menin protein at concentrations as low as 3.7 nM and reaches a maximal effect at 33–100 nM. Since the cells were treated with M-89 for only 1 h, the thermal stabilization of cellular menin by the compound is expected to be a direct effect. Our CETSA data, thus, provide clear evidence that M-89 targets cellular menin protein at low nanomolar concentrations, consistent with its excellent cellular potency in the inhibition of cell growth in leukemia cells carrying MLL fusion.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Stabilization of cellular menin protein by M-89 as assessed using the cellular thermal shift assay (CETSA) in MV4;11 cells. (B) Quantitative real-time polymerase chain reaction (qRT-PCR) analysis of the effect of M-89 on the mRNA levels of <i>Hoxa9</i> and <i>MEIS1</i> genes in MV4;11 cells. (C) Flow cytometry analysis of the effect of M-89 on apoptosis and cell differentiation in MV4;11 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The menin-MLL protein–protein interaction has been shown to play a key role in the regulation of <i>Hoxa9</i> and <i>MEIS1</i> gene expression in leukemia cells carrying an MLL fusion.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Accordingly, we evaluated M-89 for its ability to suppress the expression of <i>Hoxa9</i> and <i>MEIS1</i> in the MV4;11 cells carrying MLL-AF4 fusion and in MOLM-13 cells carrying MLL-AF9 fusion by qRT-PCR. Our data (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) showed that M-89 potently and effectively inhibits <i>Hoxa9</i> and <i>MEIS1</i> gene transcription in both cell lines, consistent with its potencies in cell growth inhibition assay.</div><div class="NLM_p">Using flow cytometry, we evaluated the ability of M-89 to induce apoptosis and cell differentiation in the MV4;11 cell line. Treatment of the MV4;11 cells with M-89 for 24 or 48 h resulted in time- and dose-dependent induction of apoptosis (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, left panel). Robust apoptosis induction was observed at concentrations as low as 100–300 nM with a 48 h treatment. Although M-89 only has a modest effect on cell differentiation with 24 h treatment, it effectively induces cell differentiation with 48 h treatment in a dose-dependent manner in the MV4;11 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, right panel).</div><div class="NLM_p">We evaluated the pharmacodynamic (PD) effect of M-89 in mice bearing MV4;11 xenograft tumors. Previous study showed that for reversible menin inhibitors, repeated administration was needed to observe their PD effect in vivo.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Therefore, mice bearing MV4;11 xenograft tumors were dosed with M-89 at 50 mg/kg daily for 3 days with intraperitoneal (IP) administration. Mice were sacrificed at different time points to harvest tumor tissues for RT-PCR analysis of the expression levels of <i>Hoxa9</i> and <i>MEIS1</i> genes, with the data provided in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. Our data showed that M-89 significantly decreases the expression of <i>Hoxa9</i> and <i>MEIS1</i> genes in the MV4;11 xenograft tumor tissue at 6, 24, and 48 h time points.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacodynamic effect of M-89 on the expression of <i>Hoxa9</i> and <i>MEIS1</i> genes in the MV4;11 xenograft tissue in mice. Mice bearing MV4;11 xenograft tumors were dosed with M-89 for 3 days and tumors were harvested at 6, 24, and 48 h after the last administration for qRT-PCR analysis of <i>Hoxa9</i> and <i>MEIS1</i> gene expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemistry</h3><div class="NLM_p">The compounds in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were synthesized, as shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. Amide coupling of <b>43</b> with <b>44</b> yielded <b>45</b> that was cyclized to produce an imine (<b>46</b>). A Grignard reaction of <b>46</b> with cyclopentyl magnesium bromide produced an intermediate (<b>47</b>) that was deprotected by catalytic hydrogenation to produce a core intermediate (<b>48</b>) with a reactive piperidine that could be used as a synthetic handle for exploration of tail groups. The tail groups were installed using either a convergent or a linear synthetic method. In the convergent method, the tail portion (<b>49</b>, <b>50</b>, or <b>51</b>) with a chloride or bromide leaving group was reacted with the intermediate (<b>48</b>) to produce compounds <b>6</b>–<b>8</b>. In the linear method, the piperidine compound (<b>48</b>) was reacted with keto- or aldehyde-<i>N</i>-Boc-protected cyclic amine linker groups in a reductive amination which was followed by acid deprotection of their corresponding Boc-amino groups to produce intermediates <b>52</b>–<b>55</b>. The final compounds <b>9</b>–<b>12</b> were obtained through a nucleophilic aromatic substitution (S<sub>N</sub>Ar) reaction between the free amino groups in compounds <b>52</b>–<b>55</b> and <i>p</i>-fluorobenzonitrile (<b>56</b>).</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6</b>–<b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) i. <b>43</b>, dichloromethane (DCM), dimethylformamide (DMF) (cat.), oxalyl chloride; ii. DCM, Et<sub>3</sub>N, <b>44</b>, 0 °C to room temperature (rt) overnight; (b) toluene, POCl<sub>3</sub>, P<sub>2</sub>O<sub>5</sub>, reflux; (c) tetrahydrofuran (THF), BF<sub>3</sub>·Et<sub>2</sub>O, cyclopentyl magnesium bromide; (d) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (e) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O (cat.), <b>49</b> or <b>50</b> or <b>51</b>, reflux; (f) DCM/AcOH (1:1), <i>N</i>-Boc-cyclic-amine-ketone or -aldehyde, NaBH(OAc)<sub>3</sub>; (g) DCM/trifluoracetic acid (TFA) (1:1) 15 min; (h) dimethyl sulfoxide (DMSO), K<sub>2</sub>CO<sub>3</sub>, <b>56</b>, 90 °C overnight.</p></p></figure><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>13</b>–<b>21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) For R = alkyl, <i>t</i>-BuONa, DMSO, R-Br, 90 °C overnight; for R = 4-bromopyridine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 90 °C overnight; (b) for <b>62</b>–<b>65</b>, DCM, <i>meta</i>-chloroperbenzoic acid (<i>m</i>CPBA), overnight; for <b>66</b> Me<sub>2</sub>CO/H<sub>2</sub>O (5:1), oxone, overnight; (c) DMSO, K<sub>2</sub>CO<sub>3</sub>, 4-F-Ph-SO<sub>2</sub>-R, 90 °C overnight.</p></p></figure><div class="NLM_p">To explore the replacement of the tail nitrile with sulfones, we used <i>p</i>-fluorophenyl sulfones (<b>67</b>–<b>75</b>) in the final S<sub>N</sub>Ar reactions to produce compounds <b>13</b>–<b>21</b>. The <i>p</i>-fluorophenyl sulfones that are not commercially available were synthesized, as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div><div class="NLM_p"><i>p-</i>Fluorothiophenol (<b>56</b>) was used to substitute alkyl bromides or in an S<sub>N</sub>Ar reaction with 4-bromopyridine to produce the sulfides (<b>62</b>–<b>66</b>). Subsequent oxidation to the sulfones (<b>67</b>–<b>71</b>) was accomplished with <i>m</i>CPBA for thioalkyls (<b>62</b>–<b>65</b>) and potassium peroxymonosulfate (Oxone) for the thiopyridine (<b>66</b>).</div><div class="NLM_p">For further exploration, we took advantage of the pyridine sulfone and nitrile tail groups. Their reactive intermediates (<b>78</b> and <b>79</b>) were synthesized (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) and used in a convergent method to install on our modified cores. To obtain these intermediates, first an S<sub>N</sub>Ar reaction of azetidin-3-ylmethanol to either <b>71</b> or <b>56</b> produced alcohols (<b>76</b> and <b>77</b>) that were converted to their corresponding mesylates <b>78</b> and <b>79</b>.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Intermediates <b>78</b> and <b>79</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMSO, K<sub>2</sub>CO<sub>3</sub>, azetidin-3-ylmethanol·HCl, 80 °C, overnight; (b) DCM, Et<sub>3</sub>N, MsCl, 0 °C to rt, 30 min.</p></p></figure><div class="NLM_p">Compounds in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> were synthesized according to the route shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Condensation of <b>80</b> and <b>81</b> followed by the reduction of the double bond in compound <b>82</b> yielded the intermediate (<b>83</b>). Deprotonation of the hydrogen next to the nitrile group generated a nucleophilic carbon that was reacted with cyclopentyl bromide to yield <b>84</b>. The nitrile in <b>84</b> was reduced in two steps. The treatment with diisobutylaluminium hydride (DIBALH) in toluene yielded the imine that was sufficiently stable to be isolated. This imine was treated with NaBH<sub>4</sub> to obtain the amine (<b>85</b>) that was then converted to a methyl carbamate (<b>86</b>). An intramolecular Friedel Crafts reaction produced the dihydroisoquinolin-1(2<i>H</i>)-one (<b>87</b>), which was efficiently reduced with the soluble aluminum reagent Red-Al to produce the tetrahydroisoquinoline (<b>88</b>). Boc protection of <b>88</b> produced <b>89</b>, which was debenzylated by catalytic hydrogenation, and the resulting piperidine was reacted with <b>78</b> in an second-order nucleophilic substitution (S<sub>N</sub>2) reaction to produce an intermediate (<b>90</b>). Acidic removal of the Boc yielded <b>22</b>, whose free nitrogen was then substituted by either an S<sub>N</sub>2 reaction or by reductive amination to produce the final compounds (<b>23</b>–<b>30</b>).</div><figure id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>22</b>–<b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeOH, NaOMe, reflux; (b) MeOH, NaBH<sub>4</sub>; (c) i. THF, lithium bis(trimethylsilyl)amide (LiHMDS), −78 °C 30 min; ii. cyclopentyl bromide, −78 °C to rt; (d) toluene, DIBALH, 30 min, rt—work-up with aqueous (aq) NaOH; (e) MeOH, NaBH<sub>4</sub>; (f) DCM, Et<sub>3</sub>N, ClCO<sub>2</sub>Me, 0 °C, 1 h; (g) polyphosphoric acid (PPA), 150 °C, 1 h; (h) toluene, Red-Al, 30 min, rt; (i) DCM, Et<sub>3</sub>N, Boc<sub>2</sub>O, 0 °C to rt; (j) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (k) CH<sub>3</sub>CN, <b>78</b>, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, reflux; (l) DCM/CF<sub>3</sub>CO<sub>2</sub>H, 15 min; (m) CH<sub>3</sub>CN, alkyl-Br, KI·H<sub>2</sub>O, reflux or DCM/AcOH (1:1), alkyl aldehyde or ketone, NaBH(OAc)<sub>3</sub>.</p></p></figure><div class="NLM_p">Functionalization of the cyclopentyl group extending from a core structure that has a diversely substituted quaternary center was necessary and would result in the formation of three stereogenic centers. To simplify the situation, we decided to explore this modification with intermediate <b>83</b>, which has an easily generated nucleophilic carbon atom and is an early intermediate in the synthesis of tetrahydroisoquinolines. Reacting the carbanion of compound <b>83</b> with either a cyclopentene-epoxide (<b>93</b>) or cyclopentene-<i>N</i>-Boc-aziridine (<b>94</b>) produced a (1:1) diastereomeric mixture of intermediates <b>95a</b> and <b>95b</b> or <b>96a</b> and <b>96b</b>, respectively. Acid removal of Boc from <b>96a</b> followed by the reaction with acetic anhydride or methyl chloroformate produced <b>97a</b> and <b>98a</b>, respectively. The removal of the benzyl protecting group on the piperidine followed by S<sub>N</sub>2 reaction with the tail intermediate <b>79</b> produced target molecules <b>31</b> (from <b>84b</b>), <b>40</b>, and <b>41</b> and the intermediates <b>102a</b> and <b>102b</b> with a hydroxyl that could be substituted, as a handle. Compounds <b>32</b>–<b>39</b> were produced by the substitution of the hydroxyl group. Consistently, the diastereomer with the same relative stereochemistry as the intermediate (<b>95a</b>) was the more potent, and its stereochemistry was confirmed by the single-crystal structure of the compound (<b>103</b>), which was obtained from the diastereomeric compound (<b>95a</b>) (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>31</b>–<b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeOH, Et<sub>3</sub>N, Boc<sub>2</sub>O; (b) THF, PPh<sub>3</sub>, diisopropylazodicarboxylate, −78 °C to rt overnight; (c) <b>83</b>, THF, LiHMDS, −78 °C for 30 min then <b>93</b>, −78 °C to rt overnight; (d) <b>83</b>, 18-crown-6, toluene, potassium bis(trimethylsilyl)amide, −78 °C for 30 min then <b>94</b>, −78 °C to rt overnight; (e) DCM/TFA, 30 min; (f) DCM, Et<sub>3</sub>N, Ac<sub>2</sub>O or ClCO<sub>2</sub>Me; (g) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (h) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, <b>79</b>, reflux; (i) NaH, DMF, alkyl iodide; (j) DCM, Et<sub>3</sub>N, alkyl anhydride or MeN═C═O; (k) DCM, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), Et<sub>3</sub>N, 4-Br-PhCO<sub>2</sub>H, 0 °C to rt.</p></p></figure><div class="NLM_p">After determining the appropriate substituents and stereochemistry of the cyclopentyl group, this modification was applied to the more active tetrahydroisoquinoline core. First, the hydroxyl group of the active diastereomer (<b>95a</b>) was benzylated, and this was followed by the reduction of the nitrile group using DIBAL to produce an amine (<b>105</b>), which was converted to a methyl carbamate to facilitate the Pictet–Spengler cyclization forming the tetrahydroisoquinoline (<b>107</b>) whose methyl carbamate was reduced with the soluble aluminum hydride reagent Red-Al to produce the intermediate core compound (<b>108</b>). Upon hydrogenation, both benzyl protecting groups were removed, and the piperidine ring in the product (<b>109</b>) was regioselectively reacted with <b>78</b> in an S<sub>N</sub>2 reaction to produce compound <b>110</b>. The reaction of <b>110</b> with methyl isocyanate produced the final compound <b>42</b> (M-89) (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0012.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of M-89<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF/toluene (1:1), NaH, BnBr, Bu<sub>4</sub>NI; (b) toluene, DIBALH; (c) DCM, Et<sub>3</sub>N, ClCO<sub>2</sub>Me, 0 °C to rt; (d) AcOH, CF<sub>3</sub>CO<sub>2</sub>H, paraformaldehyde; (e) toluene, Red-Al; (f) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (g) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, <b>78</b>, reflux; (h) DCM, NaHCO<sub>3</sub>, Et<sub>3</sub>N, MeN═C═O, rt.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18305" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18305" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Starting from a previously reported menin inhibitor (MIV-6), we have performed extensive structure-based optimization to dramatically improve its binding affinity, cellular potency, and selectivity. Through the systematic optimization of four different sites in the molecule, we have successfully obtained M-89 as a high-affinity menin inhibitor. M-89 has a <i>K</i><sub>d</sub> value of 1.4 nM to menin and is >50 times more potent than MIV-6. M-89 achieves IC<sub>50</sub> values of 25 and 54 nM, respectively, in the MV4;11 and MOLM-13 cell lines carrying MLL fusion and is >100 times more potent than MIV-6. Significantly, M-89 demonstrates >100-fold cellular selectivity in its inhibition of cell growth in the MV4;11 and MOLM-13 leukemia cell lines over the HL-60 leukemia cell line lacking MLL fusion. M-89 stabilizes cellular menin protein and effectively suppresses the expression of <i>Hoxa9</i> and <i>MEIS1</i> genes at low nanomolar concentrations in both MV4;11 and MOLM-13 leukemia cell lines. M-89 is also effective in the induction of apoptosis and cell differentiation in the MV4;11 cell line. Our pharmacodynamic experiment in mice bearing MV4;11 xenograft tumors showed that M-89 effectively downregulates the expression of <i>MEIS1</i> and <i>Hoxa9</i> genes in the MV4;11 xenograft tumor tissue. The determination of the co-crystal structure of M-89 in a complex with menin provides a solid structural basis for its high-affinity binding and for further structure-based optimization.</div><div class="NLM_p last">Taken together, our study has led to the discovery of M-89 as a highly potent and specific menin inhibitor. M-89 represents a promising lead compound for further optimization toward the development of a menin inhibitor for the treatment of MLL leukemia.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Information</h3><div class="NLM_p">Unless otherwise stated, all commercial reagents were used as supplied without further purification, and all reactions were performed under a nitrogen atmosphere in a dry solvent under anhydrous conditions. NMR spectra were obtained on a Bruker 400 Ascend spectrometer at a <sup>1</sup>H frequency of 400 MHz and a <sup>13</sup>C frequency of 100 MHz. Chemical shifts (δ) are reported in parts per million (ppm) relative to an internal standard. The final products were purified on a preparative high-performance liquid chromatography (HPLC) column (Waters 2545, Quaternary Gradient Module) with a SunFire Prep C18 OBD 5 μm 50 × 100 mm<sup>2</sup> reverse phase column. The mobile phase was a gradient of solvent A (H<sub>2</sub>O with 0.1% TFA) and solvent B (MeCN with 0.1% TFA) at a flow rate of 60 mL/min and 1%/min increase of solvent B. All final compounds have purity ≥95% as determined by Waters ACQUITY ultra-performance liquid chromatograph (UPLC) using reverse phase column (SunFire, C18, 5 μm, 4.6 × 150 mm<sup>2</sup>) and a solvent gradient of A (H<sub>2</sub>O with 0.1% of TFA) and solvent B (CH<sub>3</sub>CN with 0.1% of TFA). Electrospray ionization (ESI) mass spectral (MS) analysis was performed on a Thermo Scientific LCQ Fleet mass spectrometer.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 4-(3-(4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)propoxy)-benzonitrile (<b>6</b>)</h4><div class="NLM_p last">Compound <b>49</b> (75 mg, 0.384 mmol), K<sub>2</sub>CO<sub>3</sub> (88 mg, 0.640 mmol), and KI (catalytic) were added to a solution of the intermediate <b>48</b> (91 mg, 0.320 mmol) in MeCN (2 mL). The mixture was stirred at 80 °C overnight and then extracted with EtOAc, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated. The residue was purified with preparative HPLC to give the title compound (80 mg, 56%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.57 (d, <i>J</i> = 8.4 Hz, 2H), 7.36–7.29 (m, 2H), 7.23–7.15 (m, 2H), 6.90 (d, <i>J</i> = 8.5 Hz, 2H), 4.10 (s, 2H), 3.84–3.71 (m, 1H), 3.67–3.52 (m, 2H), 3.49–3.36 (m, 1H), 3.27–2.93 (m, 4H), 2.90–2.72 (m, 3H), 2.58–2.43 (m, 1H), 2.36–2.17 (m, 4H), 2.06–1.84 (m, 2H), 1.77–1.45 (m, 4H), 1.37–1.12 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 444.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 4-(2-(4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)ethoxy)benzonitrile (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared from compounds <b>48</b> and <b>50</b> according to the procedure used to produce compound <b>6</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.69 (d, <i>J</i> = 8.9 Hz, 2H), 7.47–7.42 (m, 1H), 7.41–7.28 (m, 3H), 7.13 (d, <i>J</i> = 9.0 Hz, 2H), 4.44 (t, <i>J</i> = 4.8 Hz, 2H), 3.82–3.65 (m, 2H), 3.63–3.51 (m, 3H), 3.51–3.41 (m, 1H), 3.23–3.05 (m, 4H), 2.81 (p, <i>J</i> = 9.7 Hz, 1H), 2.67–2.53 (m, 1H), 2.24 (d, <i>J</i> = 14.1 Hz, 1H), 2.06–1.90 (m, 2H), 1.90–1.51 (m, 7H), 1.52–1.37 (m, 1H), 1.32–1.17 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 430.26 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-(4-(4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)butoxy)-benzonitrile (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared from compounds <b>48</b> and <b>51</b> according to the procedure used to make compound <b>6</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.65 (d, <i>J</i> = 8.9 Hz, 2H), 7.47–7.41 (m, 1H), 7.40–7.29 (m, 3H), 7.05 (d, <i>J</i> = 8.9 Hz, 2H), 4.11 (t, <i>J</i> = 5.5 Hz, 2H), 3.73–3.59 (m, 2H), 3.59–3.40 (m, 2H), 3.21–3.06 (m, 4H), 3.06–2.94 (m, 2H), 2.87–2.73 (m, 1H), 2.58 (t, <i>J</i> = 12.0 Hz, 1H), 2.23 (d, <i>J</i> = 14.1 Hz, 1H), 2.01–1.79 (m, 7H), 1.79–1.51 (m, 6H), 1.51–1.37 (m, 1H), 1.32–1.15 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 458.30 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-(4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)-[1,4′-bipiperidin]-1′-yl)benzonitrile (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared according to the procedure used to make compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.53 (d, <i>J</i> = 9.0 Hz, 2H), 7.47–7.41 (m, 1H), 7.40–7.29 (m, 3H), 7.04 (d, <i>J</i> = 9.0 Hz, 2H), 4.11 (d, <i>J</i> = 13.2 Hz, 2H), 3.70–3.51 (m, 3H), 3.51–3.39 (m, 2H), 3.17–3.04 (m, 4H), 2.93 (t, <i>J</i> = 12.7 Hz, 2H), 2.86–2.74 (m, 1H), 2.60 (t, <i>J</i> = 12.0 Hz, 1H), 2.27 (d, <i>J</i> = 14.0 Hz, 1H), 2.16 (d, <i>J</i> = 12.1 Hz, 2H), 2.02–1.51 (m, 11H), 1.52–1.39 (m, 1H), 1.28–1.16 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 469.31 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-(3-((4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)methyl)piperidin-1-yl)benzonitrile (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was prepared according to the procedure used to make compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54–7.41 (m, 3H), 7.42–7.27 (m, 3H), 7.04 (dd, <i>J</i> = 9.2, 2.3 Hz, 2H), 3.96 (d, <i>J</i> = 12.9 Hz, 1H), 3.84–3.58 (m, 3H), 3.57–3.43 (m, 2H), 3.20–2.86 (m, 7H), 2.75 (dd, <i>J</i> = 12.6, 9.8 Hz, 2H), 2.65 (t, <i>J</i> = 12.6 Hz, 1H), 2.30–2.01 (m, 3H), 2.01–1.87 (m, 3H), 1.87–1.42 (m, 9H), 1.42–1.27 (m, 1H), 1.24–1.07 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 483.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-(3-((4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)methyl)-pyrrolidin-1-yl)benzonitrile (<b>11</b>)</h4><div class="NLM_p last">Compound <b>11</b> was prepared according to the procedure used to make compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50–7.43 (m, 3H), 7.36 (tt, <i>J</i> = 12.0, 7.0 Hz, 3H), 6.62 (d, <i>J</i> = 8.7 Hz, 2H), 3.85–3.70 (m, 2H), 3.66 (t, <i>J</i> = 8.8 Hz, 1H), 3.58–3.44 (m, 3H), 3.42–3.34 (m, 1H), 3.26 (d, <i>J</i> = 6.8 Hz, 2H), 3.19–2.97 (m, 5H), 2.89–2.72 (m, 2H), 2.69–2.55 (m, 1H), 2.37–2.20 (m, 2H), 2.12–1.73 (m, 6H), 1.73–1.40 (m, 5H), 1.27–1.14 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 469.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-(3-((4-(1-Cyclopentyl-1,2,3,4-tetrahydroisoquinolin-1-yl)piperidin-1-yl)methyl)azetidin-1-yl)benzonitrile (<b>12</b>)</h4><div class="NLM_p last"><i>p</i>-Fluorobenzonitrile (<b>56</b>) (18 mg, 0.148 mmol) was added to a solution of compound <b>55</b> (26 mg, 0.074 mmol) and K<sub>2</sub>CO<sub>3</sub> (41 mg, 0.295 mmol) in DMSO (2 mL), then stirred and heated to 80 °C. After being stirred overnight, the reaction was quenched with TFA (0.5 mL), diluted with 3:1 MeOH/H<sub>2</sub>O, and purified by preparative HPLC. The pure fractions were combined, concentrated, rediluted with H<sub>2</sub>O, frozen, and lyophilized to give <b>12</b> (14 mg, 42%) as a white powder. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.48 (d, <i>J</i> = 7.3 Hz, 2H), 7.45 (d, <i>J</i> = 9.7 Hz, 1H), 7.41–7.28 (m, 3H), 6.47 (d, <i>J</i> = 7.5 Hz, 2H), 4.17 (t, <i>J</i> = 8.0 Hz, 2H), 3.80–3.71 (m, 2H), 3.63 (d, <i>J</i> = 11.7 Hz, 1H), 3.60–3.40 (m, 7H), 3.12–2.95 (m, 3H), 2.87–2.71 (m, 1H), 2.65–2.50 (m, 1H), 2.23 (d, <i>J</i> = 14.9 Hz, 1H), 2.01–1.87 (m, 2H), 1.86–1.40 (m, 8H), 1.22 (s, 1H). ESI-MS <i>m</i>/<i>z</i> 455.83 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-Cyclopentyl-1-(1-((1-(4-(methylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>13</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>72</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.71 (d, <i>J</i> = 8.8 Hz, 2H), 7.48–7.41 (m, 1H), 7.41–7.28 (m, 3H), 6.52 (d, <i>J</i> = 8.8 Hz, 2H), 4.19 (t, <i>J</i> = 8.0 Hz, 2H), 3.80–3.74 (m, 2H), 3.64 (d, <i>J</i> = 11.7 Hz, 1H), 3.61–3.42 (m, 6H), 3.15–3.04 (m, 4H), 3.02 (s, 3H), 2.86–2.71 (m, 1H), 2.62 (t, <i>J</i> = 12.2 Hz, 1H), 2.25 (d, <i>J</i> = 14.2 Hz, 1H), 2.03–1.89 (m, 2H), 1.86–1.39 (m, 8H), 1.28–1.12 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 508.83 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1-Cyclopentyl-1-(1-((1-(4-(ethylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>14</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>73</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.66 (d, <i>J</i> = 8.2 Hz, 2H), 7.44 (d, <i>J</i> = 6.8 Hz, 1H), 7.41–7.29 (m, 3H), 6.52 (d, <i>J</i> = 8.4 Hz, 2H), 4.19 (t, <i>J</i> = 8.0 Hz, 2H), 3.78 (t, <i>J</i> = 6.5 Hz, 2H), 3.65 (d, <i>J</i> = 12.0 Hz, 1H), 3.61–3.42 (m, 5H), 3.19–2.97 (m, 6H), 2.78 (dt, <i>J</i> = 18.1, 9.6 Hz, 1H), 2.64 (t, <i>J</i> = 12.1 Hz, 1H), 2.25 (d, <i>J</i> = 14.0 Hz, 1H), 2.02–1.89 (m, 2H), 1.86–1.72 (m, 3H), 1.73–1.42 (m, 5H), 1.36–1.08 (m, 5H). ESI-MS <i>m</i>/<i>z</i> 522.50 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 1-Cyclopentyl-1-(1-((1-(4-(isopropylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>15</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>74</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.63 (d, <i>J</i> = 8.8 Hz, 2H), 7.48–7.42 (m, 1H), 7.41–7.28 (m, 3H), 6.53 (d, <i>J</i> = 8.8 Hz, 2H), 4.23–4.16 (m, 2H), 3.78 (ddd, <i>J</i> = 8.0, 5.6, 2.2 Hz, 2H), 3.65 (d, <i>J</i> = 12.3 Hz, 1H), 3.61–3.40 (m, 5H), 3.29–3.24 (m, 1H), 3.24–3.14 (m, 1H), 3.13–2.98 (m, 4H), 2.79 (p, <i>J</i> = 9.6 Hz, 1H), 2.64 (d, <i>J</i> = 12.2 Hz, 1H), 2.25 (d, <i>J</i> = 14.0 Hz, 1H), 2.02–1.89 (m, 2H), 1.87–1.40 (m, 8H), 1.27–1.14 (m, 7H). ESI-MS <i>m</i>/<i>z</i> 536.67 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1-Cyclopentyl-1-(1-((1-(4-(cyclopropylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>16</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>67</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.66 (d, <i>J</i> = 7.5 Hz, 2H), 7.45 (d, <i>J</i> = 7.4 Hz, 1H), 7.41–7.27 (m, 3H), 6.52 (d, <i>J</i> = 7.8 Hz, 2H), 4.19 (t, <i>J</i> = 7.9 Hz, 2H), 3.82–3.73 (m, 2H), 3.64 (d, <i>J</i> = 12.0 Hz, 1H), 3.60–3.41 (m, 5H), 3.18–2.94 (m, 4H), 2.86–2.71 (m, 1H), 2.69–2.50 (m, 2H), 2.25 (d, <i>J</i> = 14.7 Hz, 1H), 2.04–1.88 (m, 2H), 1.87–1.40 (m, 8H), 1.32–1.09 (m, 4H), 1.06–0.90 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 534.50 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(1-((1-(4-(Cyclobutylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1-cyclopentyl-1,2,3,4-tetrahydroisoquinoline (<b>17</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>68</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.63 (d, <i>J</i> = 8.8 Hz, 2H), 7.48–7.42 (m, 1H), 7.41–7.29 (m, 3H), 6.52 (d, <i>J</i> = 8.8 Hz, 2H), 4.18 (t, <i>J</i> = 7.9 Hz, 2H), 3.88 (pd, <i>J</i> = 8.2, 0.8 Hz, 1H), 3.77 (ddd, <i>J</i> = 8.0, 5.6, 2.2 Hz, 2H), 3.65 (d, <i>J</i> = 12.1 Hz, 1H), 3.61–3.40 (m, 5H), 3.29–3.20 (m, 1H), 3.15–2.97 (m, 4H), 2.79 (p, <i>J</i> = 9.8 Hz, 1H), 2.63 (t, <i>J</i> = 12.8 Hz, 1H), 2.48–2.33 (m, 2H), 2.25 (d, <i>J</i> = 13.9 Hz, 1H), 2.22–2.08 (m, 2H), 2.04–1.85 (m, 4H), 1.85–1.40 (m, 8H), 1.28–1.13 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 548.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1-Cyclopentyl-1-(1-((1-(4-(cyclopentylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>18</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>69</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.65 (d, <i>J</i> = 8.7 Hz, 2H), 7.47–7.41 (m, 1H), 7.40–7.27 (m, 3H), 6.52 (d, <i>J</i> = 8.8 Hz, 2H), 4.18 (t, <i>J</i> = 8.0 Hz, 2H), 3.82–3.73 (m, 2H), 3.64 (d, <i>J</i> = 11.8 Hz, 1H), 3.60–3.40 (m, 6H), 3.17–2.96 (m, 4H), 2.79 (dt, <i>J</i> = 17.7, 8.0 Hz, 1H), 2.70–2.54 (m, 1H), 2.25 (d, <i>J</i> = 14.2 Hz, 1H), 2.03–1.40 (m, 19H), 1.31–1.10 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 562.92 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-(1-((1-(4-(Cyclohexylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1-cyclopentyl-1,2,3,4-tetrahydroisoquinoline (<b>19</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>70</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.61 (d, <i>J</i> = 7.4 Hz, 2H), 7.45 (d, <i>J</i> = 6.5 Hz, 1H), 7.41–7.29 (m, 3H), 6.52 (d, <i>J</i> = 7.6 Hz, 2H), 4.19 (t, <i>J</i> = 8.0 Hz, 2H), 3.78 (t, <i>J</i> = 6.7 Hz, 2H), 3.65 (d, <i>J</i> = 11.9 Hz, 1H), 3.60–3.44 (m, 5H), 3.29–3.22 (m, 1H), 3.16–3.01 (m, 4H), 2.98–2.87 (m, 1H), 2.84–2.73 (m, 1H), 2.70–2.58 (m, 1H), 2.25 (d, <i>J</i> = 14.8 Hz, 1H), 2.06–1.91 (m, 4H), 1.89–1.73 (m, 5H), 1.72–1.60 (m, 4H), 1.58–1.43 (m, 2H), 1.38–1.19 (m, 5H), 1.19–1.05 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 576.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-Cyclopentyl-1-(1-((1-(4-(phenylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>20</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>75</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.90–7.83 (m, 2H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.61–7.49 (m, 3H), 7.47–7.40 (m, 1H), 7.40–7.27 (m, 3H), 6.47 (d, <i>J</i> = 8.9 Hz, 2H), 4.14 (t, <i>J</i> = 8.0 Hz, 2H), 3.73 (ddd, <i>J</i> = 8.0, 5.5, 2.3 Hz, 2H), 3.62 (d, <i>J</i> = 12.2 Hz, 1H), 3.59–3.36 (m, 5H), 3.24 (p, <i>J</i> = 6.6 Hz, 1H), 3.17–2.95 (m, 4H), 2.77 (p, <i>J</i> = 9.6 Hz, 1H), 2.68–2.54 (m, 1H), 2.24 (d, <i>J</i> = 14.0 Hz, 1H), 2.02–1.88 (m, 2H), 1.86–1.39 (m, 8H), 1.27–1.09 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 570.50 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-Cyclopentyl-1-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>21</b>)</h4><div class="NLM_p last">Starting with intermediates <b>55</b> and <b>71</b>, the target molecule was prepared according to the procedure described for compound <b>12</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 5.2 Hz, 2H), 7.88–7.79 (m, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.48–7.40 (m, 1H), 7.42–7.27 (m, 3H), 6.51 (d, <i>J</i> = 8.9 Hz, 1H), 4.19 (t, <i>J</i> = 7.9 Hz, 2H), 3.77 (t, <i>J</i> = 6.7 Hz, 2H), 3.67–3.41 (m, 7H), 3.17–2.97 (m, 4H), 2.89–2.72 (m, 1H), 2.68–2.50 (m, 1H), 2.24 (d, <i>J</i> = 14.5 Hz, 1H), 2.04–1.86 (m, 2H), 1.88–1.37 (m, 8H), 1.35–1.16 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 571.67 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-Cyclopentyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>22</b>)</h4><div class="NLM_p last">Compound <b>90</b> (20 mg, 0.0298 mmol) was dissolved in TFA (1 mL). After 30 min, the TFA was removed in vacuo, and the crude was purified by preparative HPLC to yield <b>22</b>–TFA salt (12 mg) as a yellow solid. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.76 (s, 2H), 7.83 (d, <i>J</i> = 5.1 Hz, 2H), 7.76 (d, <i>J</i> = 8.6 Hz, 2H), 7.55 (d, <i>J</i> = 8.0 Hz, 1H), 7.47–7.37 (m, 1H), 7.38–7.22 (m, 2H), 6.50 (d, <i>J</i> = 8.6 Hz, 2H), 4.33–4.22 (m, 2H), 4.23–4.11 (m, 2H), 3.76 (t, <i>J</i> = 6.8 Hz, 2H), 3.66–3.37 (m, 7H), 3.09–2.90 (m, 2H), 2.64–2.47 (m, 1H), 2.34–2.19 (m, 1H), 2.14 (d, <i>J</i> = 14.0 Hz, 1H), 1.89–1.70 (m, 3H), 1.72–1.44 (m, 6H), 1.36–1.21 (m, 1H), 1.18–1.00 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 571.58 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-Cyclopentyl-2-methyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)-piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>23</b>)</h4><div class="NLM_p last">Crude <b>22</b> (0.0298 mmol) was dissolved in MeCN (1 mL), followed by the addition of K<sub>2</sub>CO<sub>3</sub> (16 mg, 0.119 mmol) and MeI (8.4 mg, 0.0596 mmol) at rt. After 1 h, the reaction was filtered through celite, concentrated, and purified by preparative HPLC to produce <b>23</b>–TFA salt. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 6.3 Hz, 2H), 7.82 (d, <i>J</i> = 6.2 Hz, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.60–7.55 (m, 1H), 7.46 (t, <i>J</i> = 7.5 Hz, 1H), 7.36 (td, <i>J</i> = 7.4, 1.1 Hz, 1H), 7.31–7.25 (m, 1H), 6.50 (d, <i>J</i> = 8.8 Hz, 2H), 4.46–4.23 (m, 2H), 4.18 (t, <i>J</i> = 8.1 Hz, 2H), 3.76 (s, 2H), 3.68–3.46 (m, 3H), 3.46–3.37 (m, 2H), 3.29–3.18 (m, 1H), 3.15 (s, 3H), 3.07–2.90 (m, 2H), 2.52–2.29 (m, 1H), 2.22–2.01 (m, 2H), 1.95–1.41 (m, 10H), 1.40–1.20 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 585.26 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Cyclopentyl-2-ethyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)-piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>24</b>)</h4><div class="NLM_p last">Acetaldehyde (3.9 mg, 0.0894 mmol) was added to a solution of crude <b>22</b> (0.0298 mmol) in DCM/AcOH (1:1, 2 mL). After 10 min, NaBH(OAc)<sub>3</sub> (38 mg, 0.179 mmol) was slowly added to the reaction. After standing overnight, the reaction was slowly quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to produce a crude product that was purified by preparative HPLC to yield <b>24</b>–TFA salt. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.80–8.72 (m, 2H), 7.82 (d, <i>J</i> = 6.2 Hz, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.61–7.55 (m, 1H), 7.51–7.43 (m, 1H), 7.35 (ddd, <i>J</i> = 16.2, 7.9, 6.6 Hz, 2H), 6.51 (d, <i>J</i> = 8.9 Hz, 2H), 4.51–4.11 (m, 4H), 3.90–3.60 (m, 4H), 3.58–3.37 (m, 6H), 3.26–3.15 (m, 1H), 3.10–2.91 (m, 2H), 2.87–2.66 (m, 1H), 2.59–2.26 (m, 2H), 2.16 (d, <i>J</i> = 14.0 Hz, 1H), 2.10–1.92 (m, 1H), 1.92–1.80 (m, 1H), 1.80–1.52 (m, 6H), 1.48 (t, <i>J</i> = 7.3 Hz, 3H), 1.36–1.16 (m, 1H), 1.16–0.99 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 599.67 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Cyclopentyl-2-isopropyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)-piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>25</b>)</h4><div class="NLM_p last">Compound <b>25</b> was prepared from crude <b>22</b> and Me<sub>2</sub>CO according to the procedure described for compound <b>24</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (s, 2H), 7.90–7.79 (m, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.56 (d, <i>J</i> = 8.0 Hz, 1H), 7.46 (td, <i>J</i> = 8.0, 7.5, 1.8 Hz, 1H), 7.40–7.25 (m, 2H), 6.49 (d, <i>J</i> = 8.7 Hz, 2H), 4.45–4.25 (m, 2H), 4.25–4.05 (m, 2H), 3.87–3.60 (m, 5H), 3.56–3.38 (m, 4H), 3.09–2.89 (m, 2H), 2.84–2.64 (m, 1H), 2.64–2.45 (m, 1H), 2.45–2.24 (m, 1H), 2.24–2.03 (m, 2H), 1.93–1.78 (m, 2H), 1.78–1.59 (m, 6H), 1.51 (d, <i>J</i> = 6.6 Hz, 6H), 1.38–1.08 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 613.58 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-Cyclobutyl-4-cyclopentyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)-piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared from crude <b>22</b> and cyclobutanone according to the procedure described for compound <b>24</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 5.9 Hz, 2H), 7.82 (d, <i>J</i> = 6.1 Hz, 2H), 7.76 (d, <i>J</i> = 8.8 Hz, 2H), 7.57 (d, <i>J</i> = 8.0 Hz, 1H), 7.47 (t, <i>J</i> = 7.5 Hz, 1H), 7.36 (q, <i>J</i> = 7.8 Hz, 2H), 6.50 (d, <i>J</i> = 8.8 Hz, 2H), 4.45–4.26 (m, 1H), 4.17 (t, <i>J</i> = 7.8 Hz, 2H), 4.04–3.89 (m, 1H), 3.83–3.68 (m, 2H), 3.68–3.36 (m, 5H), 3.24–3.15 (m, 1H), 3.09–2.92 (m, 2H), 2.88–2.73 (m, 1H), 2.65–2.19 (m, 6H), 2.15 (d, <i>J</i> = 15.8 Hz, 1H), 2.03–1.80 (m, 4H), 1.80–1.37 (m, 8H), 1.34–1.17 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 625.58 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-Cyclopentyl-2-(cyclopropylmethyl)-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared from crude <b>22</b> and (bromomethyl)cyclopropane according to the procedure described for compound <b>23</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 5.6 Hz, 2H), 7.82 (d, <i>J</i> = 6.1 Hz, 2H), 7.76 (d, <i>J</i> = 8.9 Hz, 2H), 7.58 (d, <i>J</i> = 7.8 Hz, 1H), 7.47 (t, <i>J</i> = 7.4 Hz, 1H), 7.37 (t, <i>J</i> = 7.4 Hz, 1H), 7.32 (d, <i>J</i> = 6.7 Hz, 1H), 6.50 (d, <i>J</i> = 8.9 Hz, 2H), 4.62–4.24 (m, 2H), 4.17 (t, <i>J</i> = 8.3 Hz, 2H), 4.07–3.85 (m, 1H), 3.83–3.69 (m, 2H), 3.58–3.37 (m, 5H), 3.10–2.89 (m, 2H), 2.88–2.66 (m, 1H), 2.60–2.25 (m, 2H), 2.17 (d, <i>J</i> = 14.0 Hz, 1H), 1.94–1.36 (m, 10H), 1.36–1.17 (m, 3H), 1.16–0.97 (m, 1H), 0.94–0.70 (m, 2H), 0.59–0.43 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 625.75 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-Cyclopentyl-2-(oxetan-3-ylmethyl)-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)-methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>28</b>)</h4><div class="NLM_p last">Compound <b>28</b> was prepared from crude <b>22</b> and oxetane-3-carbaldehyde according to the procedure described for compound <b>24</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.76 (d, <i>J</i> = 5.8 Hz, 2H), 7.84 (d, <i>J</i> = 6.1 Hz, 2H), 7.76 (d, <i>J</i> = 8.8 Hz, 2H), 7.60–7.49 (m, 1H), 7.45 (t, <i>J</i> = 7.5 Hz, 1H), 7.36 (t, <i>J</i> = 7.4 Hz, 1H), 7.30 (d, <i>J</i> = 7.4 Hz, 1H), 6.50 (d, <i>J</i> = 8.8 Hz, 2H), 4.18 (t, <i>J</i> = 8.1 Hz, 2H), 3.88–3.70 (m, 5H), 3.70–3.54 (m, 5H), 3.54–3.37 (m, 4H), 3.09–2.88 (m, 2H), 2.68–2.42 (m, 2H), 2.41–2.23 (m, 1H), 2.17 (d, <i>J</i> = 14.3 Hz, 1H), 1.93–1.75 (m, 3H), 1.75–1.21 (m, 10H), 1.12–0.93 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 641.93 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-Cyclopentyl-2-(pyridin-4-ylmethyl)-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>29</b>)</h4><div class="NLM_p last">Compound <b>29</b> was prepared from crude <b>22</b> and isonicotinaldehyde according to the procedure described for compound <b>24</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.78 (dd, <i>J</i> = 12.1, 3.8 Hz, 4H), 8.07 (d, <i>J</i> = 5.7 Hz, 2H), 7.84 (d, <i>J</i> = 4.7 Hz, 2H), 7.76 (d, <i>J</i> = 8.7 Hz, 2H), 7.41 (d, <i>J</i> = 7.6 Hz, 1H), 7.24 (t, <i>J</i> = 7.1 Hz, 1H), 7.14 (t, <i>J</i> = 7.4 Hz, 1H), 6.99 (d, <i>J</i> = 7.2 Hz, 1H), 6.50 (d, <i>J</i> = 8.8 Hz, 2H), 4.26–4.06 (m, 3H), 3.93 (d, <i>J</i> = 15.6 Hz, 1H), 3.84–3.70 (m, 2H), 3.68–3.37 (m, 7H), 3.10–2.95 (m, 2H), 2.90 (t, <i>J</i> = 12.7 Hz, 1H), 2.78 (d, <i>J</i> = 12.0 Hz, 1H), 2.58–2.44 (m, 1H), 2.40 (d, <i>J</i> = 13.8 Hz, 1H), 2.25–2.10 (m, 1H), 2.02–1.80 (m, 2H), 1.73 (d, <i>J</i> = 14.3 Hz, 1H), 1.66–1.38 (m, 6H), 1.38–1.27 (m, 1H), 1.27–1.12 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 662.58 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-(2-(4-Cyclopentyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)ethyl)morpholine (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared from crude <b>22</b> and 4-(2-bromoethyl)morpholine according to the procedure described for compound <b>23</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 5.4 Hz, 2H), 7.82 (d, <i>J</i> = 6.1 Hz, 2H), 7.76 (d, <i>J</i> = 8.8 Hz, 2H), 7.41 (d, <i>J</i> = 7.7 Hz, 1H), 7.25 (t, <i>J</i> = 7.6 Hz, 1H), 7.18 (t, <i>J</i> = 7.7 Hz, 1H), 7.12 (d, <i>J</i> = 6.7 Hz, 1H), 6.50 (d, <i>J</i> = 8.9 Hz, 2H), 4.18 (t, <i>J</i> = 7.9 Hz, 2H), 3.98–3.85 (m, 4H), 3.83 (s, 1H), 3.75 (dd, <i>J</i> = 9.6, 4.0 Hz, 2H), 3.62–3.38 (m, 9H), 3.27–3.17 (m, 1H), 3.16–2.81 (m, 6H), 2.70 (d, <i>J</i> = 12.3 Hz, 1H), 2.48 (dt, <i>J</i> = 17.6, 8.7 Hz, 1H), 2.32 (d, <i>J</i> = 14.4 Hz, 1H), 2.23–2.10 (m, 1H), 2.01–1.85 (m, 1H), 1.85–1.73 (m, 1H), 1.67 (d, <i>J</i> = 13.9 Hz, 1H), 1.63–1.37 (m, 7H), 1.37–1.24 (m, 1H), 1.24–1.09 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 684.50 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>rac</i>-4-(3-((4-(Cyano(cyclopentyl)(phenyl)methyl)piperidin-1-yl)methyl)azetidin-1-yl)benzonitrile (<b>31</b>)</h4><div class="NLM_p last">Compound <b>31</b> was synthesized using the method described for compound <b>102a</b> from the intermediates <b>84</b> and <b>79</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50–7.42 (m, 6H), 7.40–7.36 (m, 1H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.16–4.12 (m, 2H), 3.72–3.69 (m, 2H), 3.54 (t, <i>J</i> = 14.1 Hz, 2H), 3.41 (d, <i>J</i> = 7.2 Hz, 2H), 3.24–3.16 (m, 1H), 3.12–2.98 (m, 2H), 2.97–2.89 (m, 1H), 2.42–2.36 (m, 1H), 2.28 (d, <i>J</i> = 14.3 Hz, 1H), 2.08–2.00 (m, 2H), 1.76–1.67 (m 2H), 1.65–1.55 (m, 4H), 1.53–1.38 (m, 2H), 1.29–1.16 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 439.42 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>rac</i>-4-(3-((4-((<i>S</i>)-Cyano((1<i>R</i>,2<i>S</i>)-2-methoxycyclopentyl)(phenyl)methyl)piperidin-1-yl)-methyl)azetidin-1-yl)benzonitrile (<b>32</b>)</h4><div class="NLM_p last">NaH (60% dispersion in mineral oil, 5.3 mg, 0.132 mmol) at 0 °C was added to a solution of the intermediate <b>102a</b> (30 mg, 0.066 mmol) in DMF (1 mL). After stirring for 10 min, MeI (9.4 mg, 0.066 mmol) was added. The reaction mixture was stirred for additional 3 h, then was quenched with H<sub>2</sub>O. The mixture was purified directly by reverse phase preparative HPLC to give the title compound as a TFA salt (14 mg, 38%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.53 (d, <i>J</i> = 7.2 Hz, 2H), 7.49–7.44 (m, 4H), 7.43–7.39 (m, 1H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.17–4.12 (m, 2H), 3.74–3.70 (m, 2H), 3.58 (d, <i>J</i> = 12.2 Hz, 1H), 3.51 (d, 12.5 Hz, 1H), 3.42 (d, <i>J</i> = 7.1 Hz, 2H), 3.39–3.35 (m, 1H), 3.25–3.15 (m, 1H), 3.11–3.00 (m, 2H), 2.98 (s, 3H), 2.89–2.83 (m, 1H), 2.50–2.44 (m, 1H), 2.39–2.36 (m, 1H), 2.14–2.08 (m, 1H), 1.96 (d, <i>J</i> = 14.4 Hz, 1H), 1.78–1.72 (m, 1H), 1.71–1.65 (m, 2H), 1.62–1.45 (m, 4H); ESI-MS <i>m</i>/<i>z</i> 469.41 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>rac</i>-4-(3-((4-((<i>R</i>)-Cyano((1<i>R</i>,2<i>S</i>)-2-methoxycyclopentyl)(phenyl)methyl)piperidin-1-yl)-methyl)azetidin-1-yl)benzonitrile (<b>33</b>)</h4><div class="NLM_p last">Compound <b>33</b> was synthesized from the intermediate <b>95b</b> using the method described for compound <b>32</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54–7.42 (m, 6H), 7.41–7.36 (m, 1H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.16–4.11 (m, 2H), 3.93–3.88 (m, 1H), 3.72–3.69 (m, 2H), 3.58–3.51 (m, 2H), 3.41 (d, <i>J</i> = 7.2 Hz, 2H), 3.39 (s, 3H), 3.23–3.09 (m, 2H), 2.99–2.90 (m, 2H), 2.49–2.43 (m, 1H), 2.30 (d, <i>J</i> = 14.4 Hz, 1H), 2.21 (d, <i>J</i> = 14.1 Hz, 1H), 1.89–1.76 (m, 2H), 1.75–1.55 (m, 3H), 1.49–1.34 (m, 2H), 1.29–1.16 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 469.41 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>rac</i>-4-(3-((4-((<i>S</i>)-Cyano((1<i>R</i>,2<i>S</i>)-2-ethoxycyclopentyl)(phenyl)methyl)piperidin-1-yl)-methyl)azetidin-1-yl)benzonitrile (<b>34</b>)</h4><div class="NLM_p last">Compound <b>34</b> was obtained according to the procedure described for compound <b>32</b> using EtI. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54 (d, <i>J</i> = 7.3 Hz, 2H), 7.49–7.38 (m, 5H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.17–4.12 (m, 2H), 3.73–3.70 (m, 2H), 3.60–3.47 (m, 3H), 3.42 (d, <i>J</i> = 7.1 Hz, 2H), 3.36–3.32 (m, 1H), 3.23–3.16 (m, 1H), 3.09–3.00 (m, 2H), 2.90–2.81 (m, 2H), 2.48–2.35 (m, 2H), 2.15–2.09 (m, 1H), 1.98 (d, <i>J</i> = 14.4 Hz, 1H), 1.74–1.67 (m, 3H), 1.65–1.56 (m, 1H), 1.54–1.42 (m, 3H), 0.95 (t, <i>J</i> = 7.0 Hz, 3H); ESI-MS <i>m</i>/<i>z</i> 483.42 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)(phenyl)methyl)cyclopentyl Acetate (<b>35</b>)</h4><div class="NLM_p last">Acetic anhydride (9 mg, 0.088 mmol) was added at 0 °C to a solution of the intermediate <b>102a</b> (20 mg, 0.044 mmol) and Et<sub>3</sub>N (13 mg, 0.131 mmol) in DCM (2 mL). The reaction mixture was stirred for 1 h at rt. Then, the reaction was concentrated under vacuum. The residue was purified directly by reverse phase preparative HPLC to give the title compound as a TFA salt (13 mg, 50%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.36 (m, 7H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.91–4.88 (m, 1H), 4.16–4.12 (m, 2H), 3.72–3.69 (m, 2H), 3.55 (t, <i>J</i> = 11.6 Hz, 2H), 3.41 (d, <i>J</i> = 7.1 Hz, 2H), 3.24–3.13 (m, 2H), 3.10–3.01 (m, 2H), 2.41 (t, <i>J</i> = 12.3 Hz, 1H), 2.28–2.25 (m, 2H), 2.06 (d, <i>J</i> = 14.3 Hz, 1H), 1.87–1.77 (m, 3H), 1.76 (s, 3H), 1.73–1.62 (m, 2H), 1.59–1.48 (m, 1H), 1.45–1.34 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 497.42 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>R</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)-(phenyl)methyl)cyclopentyl Acetate (<b>36</b>)</h4><div class="NLM_p last">Compound <b>36</b> was synthesized from the intermediate <b>95b</b> using the method described for compound <b>35</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50–7.44 (m, 6H), 7.42–7.38 (m, 1H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 5.21–5.18 (m, 1H), 4.16–4.11 (m, 2H), 3.73–3.69 (m, 2H), 3.56 (t, <i>J</i> = 14.0 Hz, 2H), 3.42 (d, <i>J</i> = 7.1 Hz, 2H), 3.23–3.18 (m, 1H), 3.16–3.09 (m, 2H), 2.96 (t, <i>J</i> = 12.4 Hz, 1H), 2.54 (t, <i>J</i> = 12.1 Hz, 1H), 2.31–2.27 (m, 1H), 2.11 (s, 3H), 2.04 (d, <i>J</i> = 14.3 Hz, 1H), 1.99–1.91 (m, 1H), 1.90–1.82 (m, 1H), 1.76–1.68 (m, 2H), 1.67–1.58 (m, 1H), 1.56–1.41 (m, 2H), 1.33–1.22 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 497.38 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)-(phenyl)methyl)cyclopentyl Propionate (<b>37</b>)</h4><div class="NLM_p last">Compound <b>37</b> was obtained according to the procedure described for compound <b>35</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.36 (m, 7H), 7.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.93–4.88 (m, 1H), 4.16–4.12 (m, 2H), 3.71 (t, <i>J</i> = 6.7 Hz, 2H), 3.54 (t, <i>J</i> = 9.8 Hz, 2H), 3.41 (d, <i>J</i> = 7.1 Hz, 2H), 3.22–3.13 (m, 2H), 3.09–3.01 (m, 2H), 2.40 (t, <i>J</i> = 12.3 Hz, 1H), 2.27–2.23 (m, 2H), 2.15–1.96 (m, 3H), 1.88–1.74 (m, 3H), 1.73–1.61 (m, 2H), 1.58–1.47 (m, 1H), 1.43–1.29 (m, 1H), 0.93 (t, <i>J</i> = 7.5 Hz, 3H); ESI-MS <i>m</i>/<i>z</i> 511.38 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)-(phenyl)methyl)cyclopentyl Butyrate (<b>38</b>)</h4><div class="NLM_p last">Compound <b>38</b> was obtained according to the procedure described for compound <b>35</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.42 (m, 6H), 7.40–7.36 (m, 1H), 6,46 (d, <i>J</i> = 8.8 Hz, 2H), 4.93–4.88 (m, 1H), 4.16–4.12 (m, 2H), 3.71 (t, <i>J</i> = 5.9 Hz, 2H), 3.54 (t, <i>J</i> = 9.9 Hz, 2H), 3.41 (d, <i>J</i> = 7.1 Hz, 2H), 3.23–3.12 (m, 2H), 3.09–3.00 (m, 2H), 2.39 (t, <i>J</i> = 12.4 Hz, 1H), 2.27–2.24 (m, 2H), 2.11–1.94 (m, 3H), 1.88–1.61 (m, 5H), 1.57–1.50 (m, 1H), 1.49–1.40 (m, 2H), 1.39–1.29 (m, 1H), 0.84 (t, <i>J</i> = 7.4 Hz, 3H); ESI-MS <i>m</i>/<i>z</i> 525.37 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)-(phenyl)methyl)cyclopentyl Methyl Carbamate (<b>39</b>)</h4><div class="NLM_p last">Methyl isocyanate (8 mg, 0.145 mmol) was added at 0 °C to a solution of the intermediates <b>102a</b> (22 mg, 0.048 mmol) and Et<sub>3</sub>N (15 mg, 0.145 mmol) in DCM (2 mL). The reaction mixture was stirred overnight at rt. Then, the reaction was concentrated under vacuum. The residue was purified directly by reverse phase preparative HPLC to give the title compound as a salt of TFA (13 mg, 34%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.37 (m, 7H), 6.46 (d, <i>J</i> = 8.8 Hz, 2H), 4.88–4.86 (m, 1H), 4.17–4.12 (m, 2H), 3.73–3.70 (m, 2H), 3.58 (d, <i>J</i> = 12.2 Hz, 1H), 3.52 (d, <i>J</i> = 12.3 Hz, 1H), 3.43 (d, <i>J</i> = 7.1 Hz, 2H), 3.23–3.17 (m, 1H), 3.08–3.01 (m, 3H), 2.56 (s, 3H), 2.45 (t, <i>J</i> = 12.0 Hz, 1H), 2.32 (d, <i>J</i> = 14.3 Hz, 1H), 2.22–2.16 (m, 1H), 1.97 (d, <i>J</i> = 14.1 Hz, 1H), 1.74–1.63 (m, 4H), 1.60–1.47 (m, 3H); ESI-MS <i>m</i>/<i>z</i> 512.40 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>rac</i>-<i>N</i>-((1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)-(phenyl)methyl)cyclopentyl)acetamide (<b>40</b>)</h4><div class="NLM_p last">Compound <b>40</b> was synthesized using the method described for compound <b>102a</b> from the intermediates <b>97a</b> and <b>79</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54 (d, <i>J</i> = 7.7 Hz, 2H), 7.51–7.36 (m, 5H), 6.48 (d, <i>J</i> = 8.8 Hz, 2H), 4.26–4.12 (m, 3H), 3.77–3.68 (m, 2H), 3.59 (d, <i>J</i> = 12.4 Hz, 1H), 3.53 (d, <i>J</i> = 12.2 Hz, 1H), 3.43 (d, <i>J</i> = 7.1 Hz, 2H), 3.26–3.14 (m, 1H), 3.04 (t, <i>J</i> = 13.1 Hz, 2H), 2.84 (q, <i>J</i> = 7.7 Hz, 1H), 2.54 (t, <i>J</i> = 12.2 Hz, 1H), 2.25 (d, <i>J</i> = 14.7 Hz, 1H), 2.20–2.10 (m, 1H), 1.99–1.90 (m, 1H), 1.79–1.67 (m, 3H), 1.66 (s, 3H), 1.65–1.41 (m, 4H); ESI-MS <i>m</i>/<i>z</i> 496.47 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>rac</i>-Methyl ((1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-Cyano(1-((1-(4-cyanophenyl)azetidin-3-yl)methyl)piperidin-4-yl)(phenyl)methyl)cyclopentyl)carbamate (<b>41</b>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></h4><div class="NLM_p last">Compound <b>41</b> was synthesized using the method described for compound <b>102a</b> starting from the intermediate <b>101a</b> and <b>79</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.52 (d, <i>J</i> = 7.2 Hz, 2H), 7.47 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.36 (m, 3H), 6.47 (<i>d</i>, <i>J</i> = 8.8 Hz, 2H), 4.17–4.12 (m, 2H), 3.93–3.88 (m, 1H), 3.73–3.70 (m, 2H), 3.60–3.51 (m, 2H), 3.44 (s, 3H), 3.42 (s, 2H), 3.23–3.17 (m, 1H), 3.07–2.99 (m, 2H), 2.89–2.83 (m, 1H), 2.50 (t, <i>J</i> = 10.2 Hz, 1H), 2.28 (d, <i>J</i> = 14.6 Hz, 1H), 2.16–2.10 (m, 1H), 1.94 (d, <i>J</i> = 14.3 Hz, 1H), 1.79–1.44 (m, 7H); ESI-MS <i>m</i>/<i>z</i> 512.42 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-2-Methyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinolin-4-yl)cyclopentyl Methyl Carbamate <b>42</b> (M-89)</h4><div class="NLM_p last">Methyl isocyanate (0.3 mmol) was added to a solution of compound <b>110</b> (30 mg, 0.05 mmol) and NEt<sub>3</sub> (28 μL, 0.2 mmol) in DCM (1 mL), then stirred at rt for 4 h. The reaction was diluted with 3:1 MeOH/H<sub>2</sub>O (10% TFA) and purified by preparative HPLC. The pure fractions were combined, concentrated, diluted with H<sub>2</sub>O, frozen, and lyophilized to give <b>42</b> (M-89) as a yellow powder. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 5.6 Hz, 2H), 7.83 (d, <i>J</i> = 5.5 Hz, 2H), 7.75 (d, <i>J</i> = 8.6 Hz, 2H), 7.54 (d, <i>J</i> = 7.8 Hz, 1H), 7.41 (t, <i>J</i> = 7.7 Hz, 1H), 7.34 (t, <i>J</i> = 7.4 Hz, 1H), 7.24 (d, <i>J</i> = 7.4 Hz, 1H), 6.48 (d, <i>J</i> = 8.7 Hz, 2H), 5.17–5.04 (m, 1H), 4.44–4.25 (m, 3H), 4.15 (t, <i>J</i> = 8.2 Hz, 2H), 3.77–3.67 (m, 2H), 3.55 (d, <i>J</i> = 12.2 Hz, 1H), 3.38 (d, <i>J</i> = 7.3 Hz, 2H), 3.24–3.14 (m, 4H), 3.05–2.84 (m, 4H), 2.28 (s, 3H), 2.14–1.99 (m, 4H), 1.92–1.55 (m, 8H); ESI-MS <i>m</i>/<i>z</i> 658.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 1-Benzyl-<i>N</i>-phenethylpiperidine-4-carboxamide (<b>45</b>)</h4><div class="NLM_p last">DMF (one drop) was added to a suspension of 1-benzylpiperidine-4-carboxylic acid (15 g, 68.4 mmol) in DCM (100 mL), and this was followed by the dropwise addition of oxalyl chloride (7 mL, 82 mmol). The mixture was stirred for 4 h, then concentrated under vacuum, affording an acid chloride, which was rediluted with DCM (100 mL). Triethylamine (23.8 mL, 171 mmol) was added to the mixture, followed by 2-phenylethan-1-amine (8.29 g, 68.4 mmol) at 0 °C. The reaction mixture was stirred at rt overnight. The organic phase was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated. The crude product was purified by recrystallization in DCM to obtain the title compound (14.9 g, 68%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34–7.20 (m, 8H), 7.19–7.14 (m, 2H), 5.45 (s, 1H), 3.57–3.40 (m, 4H), 2.90 (dt, <i>J</i> = 11.2, 2.5 Hz, 2H), 2.80 (t, <i>J</i> = 6.8 Hz, 2H), 2.08–1.88 (m, 3H), 1.83–1.61 (m, 4H); ESI-MS <i>m</i>/<i>z</i> 323.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-(1-Benzylpiperidin-4-yl)-3,4-dihydroisoquinoline (<b>46</b>)</h4><div class="NLM_p last">Phosphoryl chloride (3.3 mL, 35.4 mmol) and phosphorus pentoxide (3.35 g, 23.6 mmol) were added to a solution of the intermediate <b>45</b> (3.82 g, 11.9 mmol) in toluene (15 mL). The reaction mixture was stirred under reflux overnight, then the mixture was quenched and basified with saturated NaHCO<sub>3</sub>, extracted with DCM, and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated to give the title compound (3.5 g, 97%), which was used in the next step without further purification; ESI-MS <i>m</i>/<i>z</i> 305.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 1-(1-Benzylpiperidin-4-yl)-1-cyclopentyl-1,2,3,4-tetrahydroisoquinoline (<b>47</b>)</h4><div class="NLM_p last">Boron trifluoride diethyl etherate (0.6 mL) was added to a solution of the intermediate <b>46</b> (0.5 g, 1.64 mmol) at 0 °C under a N<sub>2</sub> atmosphere. After the mixture was stirred for 5 min, a 2 M solution of cyclopentyl magnesium bromide in Et<sub>2</sub>O (3.3 mL, 6.6 mmol) was added into the mixture dropwise at 0 °C. The reaction mixture was stirred overnight, then warmed slowly to rt. The reaction was quenched with saturated aqueous NH<sub>4</sub>Cl, extracted with DCM, dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated. The crude product was purified by preparative HPLC to give the title compound as its TFA salt (740 mg, 60%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.48–7.35 (m, 5H), 7.32–7.28 (m, 2H), 7.21–7.13 (m, 2H), 4.11 (dd, <i>J</i> = 25.6, 12.9 Hz, 2H), 3.65–3.37 (m, 4H), 3.16–2.91 (m, 2H), 2.86–2.75 (m, 2H), 2.69 (t, <i>J</i> = 11.5 Hz, 1H), 2.40 (d, <i>J</i> = 11.7 Hz, 1H), 2.27–2.12 (m, 2H), 1.96–1.80 (m, 2H), 1.75–1.57 (m, 4H), 1.46 (dd, <i>J</i> = 33.9, 3.1 Hz, 2H), 1.37–1.25 (m, 1H), 1.22–1.09 (m, 1H); ESI-MS <i>m</i>/<i>z</i> 375.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-Cyclopentyl-1-(piperidin-4-yl)-1,2,3,4-tetrahydroisoquinoline (<b>48</b>)</h4><div class="NLM_p last">Compound <b>47</b> (588 mg, 1.24 mmol) was dissolved in MeOH (5 mL), and the solution was vacuumed briefly and then put under a N<sub>2</sub> atmosphere. This was repeated three times, then Pd/C (10% w/w) (150 mg) was quickly added to the solution, which was again vacuumed and put under a N<sub>2</sub> atmosphere. The solution was briefly vacuumed to remove the N<sub>2</sub> atmosphere and then put under H<sub>2</sub> atmosphere. This was repeated three times. After 30 min, the reaction was filtered through celite and concentrated to give a crude product (<b>48</b>) that was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO) δ 7.43–7.38 (m, 1H), 7.34–7.26 (m, 3H), 3.49–3.41 (m, 1H), 3.30 (t, <i>J</i> = 11.6 Hz, 3H), 3.00 (t, <i>J</i> = 5.9 Hz, 2H), 2.96–2.89 (m, 1H), 2.86–2.74 (m, 2H), 2.41–2.30 (m, 1H), 1.98 (d, <i>J</i> = 13.1 Hz, 1H), 1.89–1.79 (m, 1H), 1.70–1.39 (m, 7H), 1.35–1.18 (m, 3H); ESI-MS <i>m</i>/<i>z</i> 285.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-(1-(Azetidin-3-ylmethyl)piperidin-4-yl)-1-cyclopentyl-1,2,3,4-tetrahydroisoquinoline (<b>55</b>)</h4><div class="NLM_p last">1-Boc-azetidine-3-carboxaldehyde (920 mg, 4.96 mmol) was added to a solution of crude <b>48</b> (1.24 mmol) in DCM/AcOH (1:1, 6 mL) and stirred. After 10 min, NaBH(OAc)<sub>3</sub> (2.10 g, 9.92 mmol) was slowly added to the reaction. After standing overnight, the reaction was slowly quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to produce crude Boc-protected-<b>55</b>. To remove the Boc protecting group, the crude product was dissolved in TFA and stirred. After 10 min, the TFA was removed in vacuo, the crude product was purified by reverse phase preparative HPLC, and the pure product was lyophilized to give <b>55</b>-TFA salt (569 mg) as white solid. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.47–7.39 (m, 1H), 7.39–7.27 (m, 3H), 4.27–4.13 (m, 2H), 4.08–3.94 (m, 2H), 3.62 (d, <i>J</i> = 12.4 Hz, 1H), 3.57–3.37 (m, 6H), 3.14–2.93 (m, 4H), 2.80–2.58 (m, 2H), 2.24 (d, <i>J</i> = 14.3 Hz, 1H), 2.05–1.40 (m, 10H), 1.22–1.07 (m, 1H).</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-(Cyclopropylsulfonyl)-<i>p</i>-fluorobenzene (<b>67</b>)</h4><div class="NLM_p last">Bromocyclopropane (2.06 mL, 24.75 mmol) was added to a solution of <i>p</i>-fluorobenzenethiol (3.0 g, 23.41 mmol) and sodium <i>tert</i>-butoxide (3.15 g, 37.77 mmol) in 60 mL of DMSO, and the reaction was heated to 80 °C. After reacting overnight, the system was cooled, quenched with saturated NH<sub>4</sub>Cl, and extracted with Et<sub>2</sub>O. The combined organic layers were washed twice with saturated NaHCO<sub>3</sub>, once with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to produce crude <b>62</b> (2.20 g) that was used without further purification. <i>m-</i>Chloroperbenzoic acid (7.34 g, 32.74 mmol, of 77 wt %) was added to a solution, at 0 °C, of crude <b>62</b> in DCM (50 mL). After standing overnight at rt, the reaction was quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, and purified by column chromatography to give <b>67</b> (2.11 g, 45%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.96–7.88 (m, 2H), 7.28–7.19 (m, 2H), 2.45 (tt, <i>J</i> = 7.9, 4.8 Hz, 1H), 1.39–1.31 (m, 2H), 1.08–1.01 (m, 2H).</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-(Cyclobutylsulfonyl)-<i>p</i>-fluorobenzene (<b>68</b>)</h4><div class="NLM_p last">Compound <b>68</b> was prepared according to the procedure used to make compound <b>67</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91–7.87 (m, 2H), 7.26–7.21(m, 2H), 3.83–3.75 (m, 1H), 2.61–2.51 (m, 2H), 2.25–2.15 (m, 2H), 2.05–1.94 (m, 2H).</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(Cyclopentylsulfonyl)-<i>p</i>-fluorobenzene (<b>69</b>)</h4><div class="NLM_p last">Compound <b>69</b> was prepared according to the procedure used to make compound <b>67</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98–7.87 (m, 2H), 7.29–7.19 (m, 2H), 3.47 (tt, <i>J</i> = 8.8, 7.2 Hz, 1H), 2.12–1.98 (m, 2H), 1.95–1.71 (m, 4H), 1.67–1.54 (m, 2H).</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(Cyclohexylsulfonyl)-<i>p</i>-fluorobenzene (<b>70</b>)</h4><div class="NLM_p last">Compound <b>70</b> was prepared according to the procedure used to make compound <b>67</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90–7.86 (m, 2H), 7.26–7.22 (m, 2H), 2.93–2.85 (m, 1H), 2.08–2.05 (m, 2H), 1.89–1.85 (m, 2H), 1.69–1.66 (m, 1H), 1.44–1.34 (m, 2H), 1.28–1.11 (m, 3H).</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 4-((<i>p</i>-Fluorophenyl)sulfonyl)pyridine (<b>71</b>)</h4><div class="NLM_p last">4-Bromopyridine·HCl (4.02 g, 20.68 mmol) was added to a solution of <i>p</i>-fluorobenzenethiol (2.41 g, 18.80 mmol) and K<sub>2</sub>CO<sub>3</sub> (7.78 g, 56.4 mmol) in DMSO (20 mL), and the reaction was maintained at 110 °C overnight. Then, the reaction was cooled, quenched with saturated NH<sub>4</sub>Cl, and extracted with EtOAc. The combined organic layers were washed twice with saturated NaHCO<sub>3</sub>, once with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to produce crude <b>66</b> (4.01 g, quantitative yield) that was used without further purification. Oxone monopersulfate (15.03 g, 48.90 mmol) was added to a solution of crude compound <b>66</b> in Me<sub>2</sub>CO/H<sub>2</sub>O (5:1, 84 mL). After standing overnight, the reaction was quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, and purified by column chromatography to give <b>71</b> (quantitative yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.88–8.81 (m, 2H), 8.03–7.95 (m, 2H), 7.78–7.73 (m, 2H), 7.28–7.21 (m, 2H).</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (1-(4-(Pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl Methanesulfonate (<b>78</b>)</h4><div class="NLM_p last">Compound <b>78</b> was prepared according to the procedure described for compound <b>79</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.77 (d, <i>J</i> = 6.0 Hz, 2H), 7.75 (d, <i>J</i> = 8.9 Hz, 2H), 7.70 (d, <i>J</i> = 6.1 Hz, 2H), 6.40 (d, <i>J</i> = 8.9 Hz, 2H), 4.43 (d, <i>J</i> = 6.6 Hz, 2H), 4.10 (t, <i>J</i> = 8.2 Hz, 2H), 3.81 (dd, <i>J</i> = 8.2, 5.0 Hz, 2H), 3.19 (dddd, <i>J</i> = 11.5, 7.9, 6.6, 2.7 Hz, 1H), 3.04 (s, 3H).</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (1-(4-Cyanophenyl)azetidin-3-yl)methyl Methanesulfonate (<b>79</b>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></h4><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (8.8 g, 63.7 mmol) was added to a solution of <i>p</i>-fluorobenzonitrile (2.6 g, 21.2 mmol) and azetidin-3-ylmethanol hydrochloride (3.4 g, 27.6 mmol) in DMSO. The mixture was stirred at 80 °C overnight. Then, the reaction mixture was cooled to rt, poured into ice, and extracted twice with EtOAc. The combined organic solution was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent was evaporated in vacuo. The residue was dissolved in DCM (100 mL), and Et<sub>3</sub>N (8.9 mL, 63.6 mmol) and methanesulfonyl chloride (2.5 mL, 31.8 mmol) were added dropwise at 0 °C. The reaction mixture was allowed to warm to rt and stirred for 5 h. Then, the reaction mixture was quenched with saturated NaHCO<sub>3</sub>, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under vacuum. The residue was purified by flash chromatography to obtain the title compound as a white solid (2.5 g, 44%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.39 (d, <i>J</i> = 8.8 Hz, 2H), 6.31 (d, <i>J</i> = 8.8 Hz, 2H), 4.38 (d, <i>J</i> = 6.7 Hz, 2H), 4.02 (t, <i>J</i> = 8.2 Hz, 2H), 3.74–3.70 (m, 2H), 3.16–3.08 (m, 1H), 2.99 (s, 3H); ESI-MS <i>m</i>/<i>z</i> 267.11 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 2-(1-Benzylpiperidin-4-ylidene)-2-phenylacetonitrile (<b>82</b>)</h4><div class="NLM_p last">Sodium methoxide (25 wt % in MeOH) (46.8 mL, 205 mmol) was added to a solution of 1-benzylpiperidin-4-one (32.3 g, 171 mmol) and 2-phenylacetonitrile (20 g, 171 mmol) in anhydrous MeOH (200 mL) under N<sub>2</sub>, and the mixture was stirred under reflux overnight. Then, the reaction mixture was cooled to rt and poured into ice (200 g). The resulting mixture was extracted with EtOAc. The separated organic layer was dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent was evaporated in vacuum to yield the title compound (48 g, 95%); ESI-MS <i>m</i>/<i>z</i> 289.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 2-(1-Benzylpiperidin-4-yl)-2-phenylacetonitrile (<b>83</b>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></h4><div class="NLM_p last">Sodium borohydride (12.6 g, 333 mmol) was added to a solution of intermediate <b>82</b> (48 g, 166 mmol) in MeOH (100 ml). The mixture was stirred at rt overnight. Then, a mixture of H<sub>2</sub>O and ice (200 ml) was added; a light yellow precipitate was formed and collected by filtration. The residue was washed with H<sub>2</sub>O and dried in vacuum to yield a light yellow product (38 g, 79%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.31–7.25 (m, 3H), 7.24–7.20 (m, 5H), 7.20–7.17 (m, 2H), 3.52 (d, <i>J</i> = 7.6 Hz, 1H), 3.43 (s, 2H), 2.90–2.81 (m, 2H), 1.92–1.79 (m, 3H), 1.74–1.64 (m, 1H), 1.52–1.34 (m, 3H); ESI-MS <i>m</i>/<i>z</i> 291.19 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 2-(1-Benzylpiperidin-4-yl)-2-cyclopentyl-2-phenylacetonitrile (<b>84</b>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></h4><div class="NLM_p last">LiHMDS (1 M in THF, 20.66 mL, 20.66 mmol) was added dropwise at −78 °C to a solution of <b>83</b> (3 g, 10.33 mmol) in dry THF (100 mL) and stirred. After 30 min at −78 °C, cyclopentyl bromide (3.32 mL, 30.99 mmol) was added dropwise, and the reaction was allowed to slowly warm to rt. After overnight at rt, the reaction was quenched with saturated NH<sub>4</sub>Cl, extracted with EtOAc, concentrated, and purified by column chromatography to produce 3.64 g of compound <b>84</b> as an oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.31–7.24 (m, 4H), 7.23–7.15 (m, 6H), 3.39 (s, 2H), 2.88–2.83 (m, 2H), 2.71–2.62 (m, 1H), 1.96–1.78 (m, 5H), 1.70–1.64 (m, 1H), 1.58–1.49 (m, 4H), 1.46–1.41 (m, 2H), 1.26–1.11 (m, 3H); ESI-MS <i>m</i>/<i>z</i> 359.32 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-(1-Benzylpiperidin-4-yl)-2-cyclopentyl-2-phenylethan-1-amine (<b>85</b>)</h4><div class="NLM_p last">DIBALH (25 wt % in toluene, 29 mL, 51.72 mmol) was added dropwise to a solution of <b>84</b> (3.0 g, 8.37 mmol) in toluene (60 mL) and stirred at rt. After 1 h, the reaction was quenched by the dropwise addition of 2 M NaOH, and the aqueous was extracted with EtOAc and concentrated. The crude imine was redissolved in MeOH, NaBH<sub>4</sub> (786 mg, 20.68 mmol) was slowly added, and the reaction was stirred. After overnight, the reaction was quenched with H<sub>2</sub>O, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered through celite, and concentrated to produce crude compound <b>85</b> (3.15 g) that was used in the next step without further purification.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 4-(1-Benzylpiperidin-4-yl)-4-cyclopentyl-1,2,3,4-tetrahydroisoquinoline (<b>88</b>)</h4><div class="NLM_p last">Methyl chloroformate (0.600 mL, 7.76 mmol) was added to a solution, at 0 °C, of crude <b>85</b> (crude, 5.17 mmol) and Et<sub>3</sub>N (1.4 mL, 10.34 mmol) in DCM (50 mL) and stirred. After 30 min at 0 °C, the reaction was stirred at rt. After 30 min at rt, the reaction was quenched with H<sub>2</sub>O and brine, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated to give 1.77 g of the crude methyl carbamate (<b>86</b>) that was used without further purification. Polyphosphoric acid (PPA) (20 mL) was added to crude <b>86</b>, and the reaction heated to 150 °C. After 1.5 h, UPLC indicated that the reaction was complete, and it was cooled to a moderate temperature, diluted with H<sub>2</sub>O, slowly quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give crude <b>87</b> (1.82 g) that was used without further purification. Red-Al (3.2 M in toluene, 7.3 mL) was added dropwise to a solution, at rt, of crude <b>87</b> (1.82 g, 4.69 mmol) in toluene (30 mL) and stirred. After 30 min, the reaction was quenched by the dropwise addition of 2 M NaOH, and the aqueous solution was extracted with EtOAc and concentrated. The crude <b>88</b> was purified by reverse phase preparative HPLC, and the pure compound was lyophilized to produce <b>88</b>-TFA salt as a white powder. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.60–7.20 (m, 9H), 4.33–4.16 (m, 3H), 3.58–3.37 (m, 4H), 3.11–2.88 (m, 2H), 2.62–2.45 (m, 1H), 2.29–2.17 (m, 1H), 2.13 (d, <i>J</i> = 14.1 Hz, 1H), 1.91–1.41 (m, 9H), 1.36–1.17 (m, 2H), 1.10 (p, <i>J</i> = 9.0 Hz, 1H).</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>tert</i>-Butyl 4-(1-Benzylpiperidin-4-yl)-4-cyclopentyl-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (<b>89</b>)</h4><div class="NLM_p last">At 0 °C, Et<sub>3</sub>N (446 μL, 3.21 mmol) followed by Boc<sub>2</sub>O (350 mg, 1.60 mmol) were added to a stirring solution of <b>88</b> (300 mg, 0.802 mmol) in DCM (5 mL), then allowed to warm to rt. After 1 h, the solvent was removed in vacuo then purified by flash column chromatography to produce 288 mg of <b>89</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.41–7.25 (m, 6H), 7.23–7.08 (m, 3H), 4.65–4.33 (m, 2H), 3.74–3.52 (m, 4H), 3.16–2.88 (m, 2H), 2.52–2.35 (m, 1H), 2.34–1.82 (m, 4H), 1.73–1.56 (m, 2H), 1.50 (s, 9H), 1.48–1.35 (m, 6H), 1.35–1.20 (m, 2H), 1.06–0.92 (m, 1H).</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>tert</i>-Butyl 4-Cyclopentyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)methyl)-piperidin-4-yl)-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (<b>90</b>)</h4><div class="NLM_p last">Compound <b>89</b> (288 mg, 0.608 mmol) was dissolved in MeOH (5 mL), and the solution was vacuumed briefly then put under a N<sub>2</sub> atmosphere. This was repeated three times, then Pd/C (10% w/w, 200 mg) was quickly added to the solution that was vacuumed and put under N<sub>2</sub> atmosphere. The solution was briefly vacuumed to remove the N<sub>2</sub> atmosphere then put under a H<sub>2</sub> atmosphere, this was repeated three times. After 30 min, the reaction was filtered through celite and concentrated to give crude debenzylated-<b>89</b> that was redissolved in MeCN (10 mL). K<sub>2</sub>CO<sub>3</sub> (420 mg, 3.04 mmol), <b>78</b> (301 mg, 0.790 mmol), KI (cat.) were added to the crude solution which was then refluxed. After overnight reflux, the reaction was cooled, filtered through celite, concentrated, and purified by flash column chromatography to produce <b>90</b> (176 mg, 43%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.74 (dd, <i>J</i> = 4.6, 1.6 Hz, 2H), 7.80 (dd, <i>J</i> = 4.5, 1.6 Hz, 2H), 7.72 (d, <i>J</i> = 8.9 Hz, 2H), 7.40–7.29 (m, 1H), 7.25–7.07 (m, 3H), 6.46 (d, <i>J</i> = 8.9 Hz, 2H), 4.63–4.35 (m, 2H), 4.10 (t, <i>J</i> = 8.1 Hz, 2H), 3.70–3.55 (m, 4H), 3.17–2.93 (m, 3H), 2.87–2.69 (m, 2H), 2.55–2.38 (m, 1H), 2.36–2.06 (m, 2H), 2.06–1.84 (m, 2H), 1.76–1.57 (m, 2H), 1.50 (s, 9H), 1.48–1.36 (m, 6H), 1.36–1.17 (m, 2H), 1.08–0.93 (m, 1H).</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>tert</i>-Butyl ((1<i>S</i>,2<i>S</i>)-2-Hydroxycyclopentyl)carbamate (<b>92</b>)</h4><div class="NLM_p last">The title compound was prepared, as reported previously.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.66 (s, 1H), 4.00–3.96 (m, 2H), 3.66–3.59 (m, 1H), 2.12–2.06 (m, 1H), 2.05–1.98 (m, 1H), 1.81–1.74 (m, 1H), 1.70–1.63 (m, 2H), 1.45 (s, 9H), 1.38–1.31 (m, 1H).</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>tert</i>-Butyl 6-Azabicyclo[3.1.0]hexane-6-carboxylate (<b>94</b>)</h4><div class="NLM_p last">The title compound was prepared, as reported previously.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.9 (s, 2H), 2.11–2.05 (m, 2H), 1.66–1.56 (m, 3H), 1.46 (s, 9H), 1.30–1.13 (m, 1H).</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>rac</i>-(<i>S</i>)-2-(1-Benzylpiperidin-4-yl)-2-((1<i>R</i>,2<i>S</i>)-2-hydroxycyclopentyl)-2-phenylacetonitrile (<b>95a</b>) and <i>rac</i>-(<i>R</i>)-2-(1-Benzylpiperidin-4-yl)-2-((1<i>R</i>,2<i>S</i>)-2-hydroxycyclopentyl)-2-phenylacetonitrile (<b>95b</b>)</h4><div class="NLM_p last">To a solution of 2-(1-benzylpiperidin-4-yl)-2-phenylacetonitrile (<b>83</b>) (1.4 g, 4.8 mmol) in dry THF (20 mL) was added dropwise LiHMDS (1 M in THF, 14.3 mL, 14.3 mmol) at 0 °C under N<sub>2</sub>. After stirring for 30 min, cyclopentene oxide (1.7 mL, 19.0 mmol) was added dropwise, and the reaction mixture was allowed to warm slowly to rt. After stirring overnight, the reaction was quenched with saturated NH<sub>4</sub>Cl and concentrated to remove the THF. The resulting mixture was extracted with DCM twice, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under vacuum. The residue was purified by flash chromatography to obtain a diasteromeric mixture (<b>95a</b>/<b>95b</b>). The mixture was further separated by reverse phase preparative HPLC and basified with saturated NaHCO<sub>3</sub> to give the title compounds <b>95a</b> (0.56 g, 32%) and <b>95b</b> (0.5 g, 28%) as colorless oils. Data for <b>95a</b>: <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50–7.47 (m, 2H), 7.42–7.38 (m, 2H), 7.36–7.32 (m, 1H), 7.30–7.21 (m, 5H), 3.84–3.80 (m, 1H), 3.45 (s, 2H), 2.95–2.86 (m, 2H), 2.83–2.77 (m, 1H), 2.14–2.02 (m, 4H), 2.00–1.93 (m, 1H), 1.78–1.57 (m, 5H), 1.54–1.44 (m, 1H), 1.31–1.19 (m, 3H); ESI-MS <i>m</i>/<i>z</i> 375.33 [M + H]<sup>+</sup>; Data for <b>95b</b>: <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.43–7.37 (m, 4H), 7.34–7.23 (m, 6H), 4.27–4.23 (m, 1H), 3.50 (s, 2H), 2.98–2.94 (m, 1H), 2.91–2.87 (m, 1H), 2.83–2.77 (m, 1H), 2.30–2.22 (m, 1H), 2.18–2.11 (m, 1H), 2.09–2.01 (m, 2H), 1.90–1.79 (m, 2H), 1.75–1.61 (m, 3H), 1.60–1.49 (m, 1H), 1.30–1.11 (m, 4H).; ESI-MS <i>m</i>/<i>z</i> 375.33 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Mixture of <i>rac</i>-<i>tert</i>-Butyl ((1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-(1-Benzylpiperidin-4-yl)(cyano)(phenyl)methyl)cyclopentyl)carbamate (<b>96a</b>) and <i>rac</i>-<i>tert</i>-Butyl ((1<i>S</i>,2<i>R</i>)-2-((<i>R</i>)-(1-Benzylpiperidin-4-yl)(cyano)(phenyl)methyl)cyclopentyl)carbamate (<b>96b</b>)</h4><div class="NLM_p last">A mixture of <b>96a</b> and <b>96b</b> was obtained, as described in our previous publication.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Preparative HPLC separated the diastereomers, and their data are as follows. <b>96a</b>: <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.55–7.33 (m, 10H), 4.25 (s, 2H), 3.89–3.80 (m, 1H), 3.52 (d, <i>J</i> = 12.6 Hz, 1H), 3.45 (d, <i>J</i> = 12.8 Hz, 1H), 3.06–2.94 (m, 2H), 2.85–2.75 (m, 1H), 2.50 (t, <i>J</i> = 10.8 Hz, 1H), 2.28 (d, <i>J</i> = 14.3 Hz, 1H), 2.15–2.03 (m, 1H), 1.89 (d, <i>J</i> = 14.4 Hz, 1H), 1.72–1.40 (m, 7H), 1.32 (s, 9H). <b>96b</b>: <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.41 (dd, <i>J</i> = 19.2, 4.0 Hz, 10H), 4.19–3.98 (m, 3H), 3.45–3.34 (m, 2H), 2.95–2.80 (m, 2H), 2.58–2.45 (m, 1H), 2.21 (d, <i>J</i> = 13.8 Hz, 1H), 2.00–1.84 (m, 2H), 1.82–1.71 (m, 1H), 1.69–1.52 (m, 3H), 1.46 (s, 9H), 1.42–1.15 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 474.50 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>rac</i>-<i>N</i>-((1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-(1-Benzylpiperidin-4-yl)(cyano)(phenyl)methyl) cyclopentyl)-acetamide (<b>97a</b>)</h4><div class="NLM_p last">Compound <b>96a</b> (20 mg, 0.042 mmol) was dissolved and stirred in TFA. After 15 min, the TFA was removed in vacuo and the crude compound was redissolved in DCM (3 mL), cooled to 0 °C, then Et<sub>3</sub>N and acetic anhydride were added, and the mixture was stirred. After 30 min at rt, the solvent was removed, and the crude product was purified by preparative HPLC to produce <b>97a</b> as the TFA salt (8 mg, 47%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56–7.31 (m, 10H), 4.24 (s, 2H), 4.17 (q, <i>J</i> = 6.7 Hz, 1H), 3.51 (d, <i>J</i> = 13.7 Hz, 1H), 3.44 (d, <i>J</i> = 12.7 Hz, 1H), 3.03 (td, <i>J</i> = 12.9, 3.0 Hz, 2H), 2.81 (q, <i>J</i> = 7.9 Hz, 1H), 2.51 (tt, <i>J</i> = 12.2, 9.0, 3.4 Hz, 1H), 2.27–2.18 (m, 1H), 2.18–2.08 (m, 1H), 1.96–1.86 (m, 1H), 1.81–1.65 (m, 3H), 1.64 (s, 3H), 1.61–1.36 (m, 4H).</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>rac</i>-Methyl ((1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-(1-Benzylpiperidin-4-yl)(cyano)(phenyl)methyl)-cyclopentyl)-carbamate (<b>98a</b>)</h4><div class="NLM_p last">Starting with diastereomer <b>96a</b>, the title compound was obtained, as previously reported.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.34 (m, 10H), 4.24 (s, 2H), 3.91–3.86 (m, 1H), 3.53–3.46 (m, 2H), 3.43 (s, 3H), 3.06–2.96 (m, 2H), 2.86–2.80 (m, 1H), 2.47 (t, <i>J</i> = 12.1 Hz, 1H), 2.26 (d, <i>J</i> = 14.4 Hz, 1H), 2.13–2.07 (m, 1H), 1.90 (d, <i>J</i> = 14.4 Hz, 1H), 1.79–1.72 (m, 1H), 1.70–1.60 (m, 2H), 1.58–1.40 (m, 4H); ESI-MS <i>m</i>/<i>z</i> 432.53 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>rac</i>-(<i>S</i>)-2-((1<i>R</i>,2<i>S</i>)-2-Hydroxycyclopentyl)-2-phenyl-2-(piperidin-4-yl)acetonitrile (<b>99a</b>)</h4><div class="NLM_p last">Compound <b>99a</b> was obtained according to the same procedure used for intermediate <b>48</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51–7.48 (m, 2H), 7.43–7.39 (m, 2H), 7.36–7.32 (m, 1H), 3.86–3.82 (m, 1H), 3.06–2.98 (m, 2H), 2.85–2.80 (m, 1H), 2.65–2.58 (m, 1H), 2.57–2.50 (m, 1H), 2.28–2.20 (m, 1H), 2.13–2.03 (m, 2H), 1.80–1.71 (m, 1H), 1.69–1.56 (m, 4H), 1.54–1.47 (m, 1H), 1.20–1.08 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 285.23 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>rac</i>-4-(3-((4-((<i>S</i>)-Cyano((1<i>R</i>,2<i>S</i>)-2-hydroxycyclopentyl)(phenyl)methyl)piperidin-1-yl)methyl)azetidin-1-yl)benzonitrile (<b>102a</b>)</h4><div class="NLM_p last"><b>102a</b> was obtained using <b>99a</b> and <b>79</b> according to the procedure used to obtain compound <b>5</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.51 (d, <i>J</i> = 7.2 Hz, 2H), 7.45–7.40 (m, 4H), 7.38–7.33 (m, 1H), 6.41 (d, <i>J</i> = 8.9 Hz, 2H), 4.07–4.03 (m, 2H), 3.85–3.82 (m, 1H), 3.60–3.56 (m, 2H), 3.00–2.93 (m, 3H), 2.85–2.79 (m, 1H), 2.65 (d, <i>J</i> = 7.2 Hz, 2H), 2.18–2.01 (m, 5H), 1.81–1.58 (m, 5H), 1.56–1.46 (m, 1H), 1.33–1.22 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 455.39 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>rac</i>-(<i>S</i>)-2-(1-(4-Bromobenzoyl)piperidin-4-yl)-2-((1<i>R</i>,2<i>S</i>)-2-hydroxycyclopentyl)-2-phenyl-acetonitrile (<b>103</b>)</h4><div class="NLM_p last">Et<sub>3</sub>N (0.25 mL, 1.41 mmol) and HATU (320 mg, 0.844 mmol) were added at 0 °C to a solution of intermediate <b>99a</b> (200 mg, 0.703 mmol) and <i>p</i>-bromobenzoic acid (169 mg, 0.844 mmol) in DCM. The reaction mixture was stirred at rt for 3 h. Then, the mixture was extracted with DCM, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under vacuum. The residue was purified by flash chromatography to obtain the title compound as a white solid (210 mg, 64%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.59 (d, <i>J</i> = 8.3 Hz, 2H), 7.50 (d, <i>J</i> = 7.5 Hz, 2H), 7.43 (t, <i>J</i> = 7.1 Hz, 2H), 7.38–7.35 (m, 1H), 7.22 (d, <i>J</i> = 8.5 Hz, 2H), 4.69–4.60 (m, 1H), 3.87–3.84 (m, 1H), 3.75–3.64 (m, 1H), 3.17–3.09 (m, 1H), 2.86–2.76 (m, 2H), 2.54–2.47 (m, 1H), 2.27–2.08 (m, 2H), 1.80–1.43 (m, 6H), 1.18–1.13 (m, 2H); ESI-MS <i>m</i>/<i>z</i> 467.27, 469.28 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>rac</i>-(<i>S</i>)-2-((1<i>R</i>,2<i>S</i>)-2-(Benzyloxy)cyclopentyl)-2-(1-benzylpiperidin-4-yl)-2-phenylacetonitrile (<b>104</b>)</h4><div class="NLM_p last">NaH (65 wt % in oil, 263 mg, 6.59 mmol) was added to a solution of <b>95a</b> (821 mg, 2.20 mmol), and tetrabutylammonium iodide (8 mg, 0.220 mmol) dissolved in dry THF/PhCH<sub>3</sub> (1:1, 10 mL) at 0 °C and stirred. After 30 min at 0 °C, benzyl bromide (0.286 mL, 2.63 mmol) was added dropwise, and the reaction was allowed to warm to rt. After overnight at rt, the reaction was quenched with saturated NH<sub>4</sub>Cl, extracted with EtOAc, concentrated, and purified by column chromatography to produce <b>104</b> (643 mg). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44 (d, <i>J</i> = 7.8 Hz, 2H), 7.37–7.17 (m, 11H), 7.08 (dd, <i>J</i> = 7.4, 2.1 Hz, 2H), 4.20 (d, <i>J</i> = 11.0 Hz, 1H), 3.79 (d, <i>J</i> = 10.9 Hz, 1H), 3.58 (ddd, <i>J</i> = 5.8, 3.7, 2.1 Hz, 1H), 3.48–3.35 (m, 2H), 2.97–2.81 (m, 3H), 2.09–2.00 (m, 1H), 2.00–1.91 (m, 2H), 1.89–1.80 (m, 3H), 1.79–1.57 (m, 5H), 1.31–1.07 (m, 2H).</div></div><div id="sec4_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>rac</i>-(<i>S</i>)-2-((1<i>R</i>,2<i>S</i>)-2-(Benzyloxy)cyclopentyl)-2-(1-benzylpiperidin-4-yl)-2-phenylethan-1-amine (<b>105</b>)</h4><div class="NLM_p last">DIBALH (3.92 mL, 6.89 mmol) was added dropwise to a solution of <b>104</b> (640 mg, 1.38 mmol) in toluene (6 mL) and stirred at rt. After 1 h, the reaction was quenched by the dropwise addition of 2 M NaOH, and the aqueous solution was extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered through celite, and concentrated to produce the crude compound <b>105</b> (691 mg) that was used in the next step without further purification.</div></div><div id="sec4_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> <i>rac</i>-(<i>S</i>)-4-((1<i>R</i>,2<i>S</i>)-2-(Benzyloxy)cyclopentyl)-4-(1-benzylpiperidin-4-yl)-2-methyl-1,2,3,4-tetrahydroisoquinoline (<b>108</b>)</h4><div class="NLM_p">Methyl chloroformate (0.166 mL, 2.14 mmol) was added at 0 °C to a solution of crude <b>105</b> (501 mg, 1.07 mmol) and Et<sub>3</sub>N (0.595 mL, 4.28 mmol) in DCM (10 mL) and stirred. After 30 min at 0 °C, the reaction was stirred at rt. After 30 min at rt, the reaction was quenched with H<sub>2</sub>O and brine, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give crude <b>106</b> (561 mg) that was used without further purification.</div><div class="NLM_p">Crude <b>106</b> (561 mg) was dissolved in AcOH (5 mL), followed by the addition of paraformaldehyde (3 equiv) and concentrated TFA (2 mL) at rt. After overnight at rt, the reaction was slowly quenched with saturated NaHCO<sub>3</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give crude <b>107</b> (572 mg) that was used without further purification.</div><div class="NLM_p last">Red-Al (3.2 M in toluene, 3 equiv) was added dropwise to a solution, at rt, of crude <b>107</b> in toluene (15 mL) and stirred. After 30 min, the reaction was quenched by the dropwise addition of 2 M NaOH, and the aqueous solution was extracted with EtOAc and concentrated. The crude compound <b>108</b> was purified by reverse phase preparative HPLC, and the pure compound was lyophilized to produce <b>108</b>–TFA salt as a white powder. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.64–7.10 (m, 14H), 4.71–4.48 (m, 1H), 4.48–4.26 (m, 3H), 4.28–4.05 (m, 3H), 3.90–3.58 (m, 1H), 3.40 (d, <i>J</i> = 12.3 Hz, 2H), 3.22–2.67 (m, 5H), 2.66–2.49 (m, 1H), 2.48–2.38 (m, 1H), 2.37–2.08 (m, 1H), 2.05–1.06 (m, 10H).</div></div><div id="sec4_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-2-Methyl-4-(piperidin-4-yl)-1,2,3,4-tetrahydroisoquinolin-4-yl)-cyclopentan-1-ol (<b>109</b>)</h4><div class="NLM_p last">Compound <b>108</b> (110 mg, 0.223 mmol) was dissolved in MeOH (10 mL), and the solution was vacuumed briefly then put under a N<sub>2</sub> atmosphere, this was repeated three times. Pd/C (10% w/w, 100 mg) was quickly added to the solution, which was then vacuumed and put under a N<sub>2</sub> atmosphere. The solution was briefly vacuumed to remove the N<sub>2</sub> atmosphere, then put under a H<sub>2</sub> atmosphere, this was repeated three times. After 4 h, the reaction was filtered through celite and concentrated to give crude <b>109</b> (76 mg) that was used without further purification.</div></div><div id="sec4_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>rac</i>-(1<i>S</i>,2<i>R</i>)-2-((<i>S</i>)-2-Methyl-4-(1-((1-(4-(pyridin-4-ylsulfonyl)phenyl)azetidin-3-yl)-methyl)piperidin-4-yl)-1,2,3,4-tetrahydroisoquinolin-4-yl)cyclopentan-1-ol (<b>110</b>)</h4><div class="NLM_p last">Compound <b>78</b> (44 mg, 0.115 mmol), K<sub>2</sub>CO<sub>3</sub> (31 mg, 0.228 mmol), and KI (cat.) were added to a solution of the intermediate <b>109</b> (18 mg, 0.057 mmol) in MeCN (3 mL). The mixture was refluxed overnight. Then, the mixture was filtered through celite, concentrated, and purified with preparative HPLC to give <b>110</b>–TFA (29 mg) salt as a white solid.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Fluorescence Polarization (FP)-Based Binding Assay</h3><div class="NLM_p">Binding affinities of menin inhibitors to menin protein were determined using our established fluorescence polarization (FP) binding assay described previously.<a onclick="showRef(event, 'ref25 ref28'); return false;" href="javascript:void(0);" class="ref ref25 ref28">(25,28)</a> The FP competitive binding assay was developed and optimized using a novel FAM-labeled fluorescent probe. Equilibrium dissociation constant (<i>K</i><sub>d</sub>) value of FAM-probe to menin protein was determined from protein saturation experiments by monitoring the total fluorescence polarization of mixtures composed with the fluorescent probe at a fixed concentration and the protein with increasing concentrations up to full saturation. FP values in millipolarization units (mP) were measured using the Infinite M-1000 plate reader (Tecan US, Research Triangle Park, NC) in Microfluor 1 96-well, black, v-bottom plates (Thermo Scientific, Waltham, MA) at an excitation wavelength of 485 nm and an emission wavelength of 530 nm. <i>K</i><sub>d</sub> value of FAM-probe, which was calculated by fitting the sigmoidal dose-dependent FP increases as a function of protein concentrations using the GraphPad Prism 6.0 software (GraphPad Software, San Diego, CA), is determined as 1.4 nM.</div><div class="NLM_p last">The IC<sub>50</sub> and <i>K</i><sub>i</sub> values of compounds were determined in a competitive binding experiment, with a final concentration of the menin protein at 4 nM and a final probe concentration at 4 nM. Negative controls containing protein/probe complex only (equivalent to 0% inhibition) and positive controls containing only free probes (equivalent to 100% inhibition) were included in each assay plate. FP values were measured following the procedure described above. IC<sub>50</sub> values were determined by nonlinear regression fitting of the competition curves.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Biolayer Interferometry (BLI) Binding Assay</h3><div class="NLM_p">Biotinylation of purified recombinant menin protein was accomplished by using the Thermo EZ-Link long-chain biotinylation reagent. Menin protein and biotinylation reagent were mixed with 1:1 molar ratio in phosphate-buffered saline (PBS) at 4 °C. This reaction was incubated at 4 °C for 2h. The reaction mixture was then dialyzed using Fishersci 10K MWCO dialysis cassettes to remove the unreacted biotinylation reagent.</div><div class="NLM_p last">BLI binding assays were performed in 96-well microplates at room temperature with continuous 1000 rpm shaking using the Octet Red 96 system (ForteBio, Menlo Park, CA). PBS with 0.1% bovine serum albumin, 0.01% Tween-20, and 2% DMSO were used as the assay buffer. Biotinylated menin protein was tethered on super streptavidin (SSA) biosensors (ForteBio) by dipping sensors into 200 μL per well 10 μg/mL protein solutions. Average saturation response level of 8–10 nm was achieved in 20 min. The measurement processes were all under computer control. Program procedures were established as follows: For the initial step, biosensors were washed in assay buffer for 60 sec to form a baseline; the biosensors labeled with biotin-menin were exposed to 100 nM compounds for association and were monitored for 1200 s, and then, the biosensors were moved back into assay buffer to disassociate for another 1800 s. Data were fit globally and generated automatically by the Octet User software (version 9.0; ForteBio).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Cell Growth Inhibition Assay</h3><div class="NLM_p">The human acute leukemia MV4;11 cell line and HL-60 were purchased from the American Type Culture Collection, and the human acute leukemia MOLM-13 cell line was purchased from the DSMZ German cell bank. In all experiments, cultured human leukemia cells were used within 2 months of thawing fresh vials. Cells were cultured in Roswell Park Memorial Institute 1640 media (MOLM-13) or Iscove’s modified Dulbecco’s media (MV4;11 and HL-60) supplemented with 10% fetal bovine serum and 1% penicillin, streptomycin at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub> in air.</div><div class="NLM_p last">In the cell growth experiments, cells were seeded in 96-well cell culture plates at a density of 10 000–20 000 cells/well in 200 μL of culture medium containing serial dilution of testing compounds. After 4 days of treatments, cell growth was measured by a lactate dehydrogenase-based WST-8 assay (Dojindo Molecular Technologies) using a Tecan Infinite M-1000 multimode microplate reader (Tecan US, Morrisville, NC). The WST-8 reagent was added to each well, and cells were incubated for an additional 1–2 h and read at 450 nm. The readings were normalized to the vehicle-treated cells, and the IC<sub>50</sub> was calculated by nonlinear regression analysis using the GraphPad Prism 6 software.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Cellular Thermal Shift Assay</h3><div class="NLM_p last">MV4;11 or MOLM-13 cells were treated with M-89 ranging from 3.7 to 300 nM for 1 h in six-well plates at a density of 5 × 10<sup>6</sup> cells per well. Centrifugation followed by washing with PBS produced cell pellets that were resuspended in 100 μL of PBS containing Halt protease inhibitors, then transferred to PCR tubes, and heated to 45 °C for 3 min. Two freeze–thaw cycles in liquid nitrogen achieved cell lysis, and the lysates were clarified by centrifugation at 15 000 rpm. In clean tubes, the transferred supernatants were mixed with loading buffer, heated, and proteins were separated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis. Membranes were blotted with an antimenin antibody (CST, cat. No. 6891).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Real-Time PCR</h3><div class="NLM_p last">MV4;11 or MOLM-13 cells were treated for 48 h with vehicle or M-89 at 30, 100, and 300 nM concentrations for MV4;11 cells or 100 nM, 300 nM, and 1 μM for MOLM-13 cells. According to the manufacturer’s protocol, total RNA was isolated from cells using the RNeasy kit (QIAGEN). A High Capacity cDNA Reverse Transcription Kit (Applied Biosystems) was used to generate the cDNA. Using TacMan gene expression assays (Applied Biosystems) and primers specific for each gene real-time PCR amplifications of <i>Hoxa9</i> (Applied Biosystems, Hs00365956_m1), <i>MEIS1</i> (Applied Biosystems, Hs00180020_m1) and GAPDH (Applied Biosystems, Hs02786624_g1) genes were carried out. Using the comparative cycle threshold (Ct) method, the relative quantification of each gene transcript was calculated and the results were presented as relative expression to vehicle treatment after normalization to the internal control of GAPDH.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Cell Apoptosis and Differentiation Analysis by Flow Cytometry</h3><div class="NLM_p last">Flow cytometry was used to analyze effects of M-89 on apoptosis (annexin V and propidium iodide staining) or cell differentiation using CD11b expression as an indicator. Cells were treated with M-89 at the indicated concentrations for 24 or 48 h, collected, stained with an apoptosis assay kit (Roche cat. No. 11 988 549 001) or a phycoerythrin-anti-mouse/human CD11b antibody (BioLegend cat. No 101208), and analyzed by flow cytometry.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> In Vivo Pharmacodynamic Assay</h3><div class="NLM_p">All animal experiments were performed under the guidelines of the University of Michigan Committee for Use and Care of Animals using an approved animal protocol. Female immunodeficient (severe combined immunodeficient) mice obtained from Charles River were injected subcutaneously in the right flank with 5 million MV4;11 cells in a 5 mg/mL solution of Matrigel. When tumors reached 100 mm<sup>3</sup>, mice were dosed by intraperitoneal injection with either vehicle or M-89 at 50 mg/kg daily for 3 days. Tumors were harvested at 6, 24, or 48 h after the third dose of M-89 with three animals per time point. Tumors were immediately frozen in liquid nitrogen, ground into a fine powder, placed on dry ice, and stored at −80°C for analysis. Expression levels of <i>MEIS1</i> and <i>Hoxa9</i> were analyzed by RT-PCR, as described above.</div><div class="NLM_p last">Statistical analysis and graph creation were performed in the GraphPad Prism version 7 for Windows (GraphPad Software, La Jolla California), using <i>t</i>-test (nonparametric test) to calculate significant levels between treated groups and control groups. ** (<i>p</i> < 0.01).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Determination of the Co-crystal Structure of M-89 with Recombinant Human Menin Protein</h3><div class="NLM_p last">Menin (residues 2-610 containing a deletion from 460-519) was purified, as previously described.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For crystallization, menin (25 mg/mL in 20 mM Tris 8.0, 150 mM NaCl and 5 mM dithiothreitol) was incubated for 1 h at 4 °C in a 1:1 molar ratio with M-89. Crystals grew at 4 °Ç in sitting drop vapor diffusion experiments over wells containing 2.0 M NaCl, 90.9 M bis(2-hydroxyethyl)amino–tris(hydroxymethyl)methane (Bis–Tris) pH 6.5, 0.182 M MgCl<sub>2</sub>, and 9 mM Pr acetate. Crystals were annealed and cryoprotected through serial soaking in solutions containing 1, 2, 3, 4, and 5 M sodium formate in 0.1 M Bis–Tris pH 6.5, 2.2 M NaCl, 0.2 M MgCl<sub>2</sub>, and12 mM Pr acetate. Diffraction data were collected on at the Advanced Photon Source LS-CAT 21-ID-G beamline at Argonne National Laboratory with a wavelength of 0.9786 Å and processed with HKL2000.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The structure of menin bound to M-89 was solved by molecular replacement (Molrep)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> using the apo menin structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U84">3U84</a>) as the search model. The menin/M-89 co-crystal structure was produced via iterative rounds of electron density fitting and structural refinement using Coot<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and Buster,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> respectively. The coordinates and restraint files for the ligand were created from SMILES in Grade.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The initial Fo–Fc electron density map showed the presence of M-89 bound in the active site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf" class="ext-link">Figure S2</a>). The following regions were disordered in the structure: 71-73, 386-401, 528-547, and 582-610. Data collection and structural refinement statistics are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf" class="ext-link">Table S2</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00021" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18386" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18386" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00021" class="ext-link">10.1021/acs.jmedchem.9b00021</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Single-crystal structure of compound <b>103</b> and a table of crystal data and structure refinement of <b>103</b>; electron density maps of the co-crystal structure of M-89 complexed with menin and a table of crystallographic data collection and refinement statistics for the menin/M-89 co-crystal (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">CSV file containing SMILES, binding and cellular data for final target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf">jm9b00021_si_001.pdf (975.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_002.csv">jm9b00021_si_002.csv (4.38 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for M-89 complexed with menin has been deposited with the Protein Data Bank with the PDB ID code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A">6E1A</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00021" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86506" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86506" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    Pharmacology, 
    Medicinal Chemistry, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81f2e9e0eeece4efe6c1f4ece8e2e9afe4e5f4"><span class="__cf_email__" data-cfemail="fe8d969f91939b9099be8b93979d96d09b9a8b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angelo Aguilar</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ke Zheng</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianfeng Xu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shilin Xu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liyue Huang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ester Fernandez-Salas</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Pathology, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liu Liu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denzil Bernard</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaitlin P. Harvey</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline Foster</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donna McEachern</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Departments of Internal Medicine, 
    , , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne Stuckey</span> - <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krishnapriya Chinnaswamy</span> - <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Delproposto</span> - <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeff W. Kampf</span> - <span class="hlFld-Affiliation affiliation">Chemistry, , , , , and , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.A., K.Z., T.X., S.X., and L.H. contributed equally to this work.</p></li><li><span class="author-information-subsection-header">Funding</span><p>We are grateful for the financial support from the Prostate Cancer Foundation (to S.W.), the National Cancer Institute, NIH (R01CA208267 to S.W.), and the University of Michigan Comprehensive Cancer Center support grant from the National Cancer Institute, NIH (P30 CA046592). Use of the Advanced Photon Source, an Office of Science User Facility operated for the U.S. Department of Energy (DOE) Office of Science by Argonne National Laboratory, was supported by the U.S. DOE under Contract No. DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor (Grant 085P1000817).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The University of Michigan has filed a patent application on these menin inhibitors reported in this study, which has been licensed to Medsyn Biopharma, LLC. S. Wang, A. Aguilar, K. Zheng, T. Xu, S. Xu, L. Huang, E. Fernandez-Salas, D. Bernard, and J. Stuckey are co-inventors on the patent application, and receive royalties on the patent from the University of Michigan. S. Wang is a co-founder of Medsyn Biopharma. The University of Michigan and S. Wang own stock in Medsyn Biopharma.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Bis–Tris</td><td class="NLM_def"><p class="first last">bis(2-hydroxyethyl)amino–tris(hydroxymethyl)methane</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">biolayer interferometry</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">ESI-MS</td><td class="NLM_def"><p class="first last">electrospray ionization mass spectrometry</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">LiHMDS</td><td class="NLM_def"><p class="first last">lithium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed lineage leukemia</p></td></tr><tr><td class="NLM_term">Oxone</td><td class="NLM_def"><p class="first last">potassium peroxymonosulfate</p></td></tr><tr><td class="NLM_term">PPA</td><td class="NLM_def"><p class="first last">polyphosphoric acid</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultra-performance liquid chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">The role of MLL in hematopoiesis and leukemia</span>. <i>Curr. Opin. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1097/00062752-200207000-00004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1097%2F00062752-200207000-00004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=12042701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A280%3ADC%252BD38zgtFSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=282-287&author=P.+Ernstauthor=J.+Wangauthor=S.+J.+Korsmeyer&title=The+role+of+MLL+in+hematopoiesis+and+leukemia&doi=10.1097%2F00062752-200207000-00004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of MLL in hematopoiesis and leukemia</span></div><div class="casAuthors">Ernst Patricia; Wang Jing; Korsmeyer Stanley J</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in hematology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-7</span>
        ISSN:<span class="NLM_cas:issn">1065-6251</span>.
    </div><div class="casAbstract">The MLL gene, also called HRX or ALL-1, was originally identified as a recurrent chromosomal translocation in particular subtypes of acute lymphocytic leukemia (ALL) and acute myelogenous leukemia (AML).  Reciprocal rearrangements of the MLL gene are most common in infant ALL and secondary AML.  Because of the unique association with infant leukemia and the intriguingly immature and mixed lineage phenotype of leukemic cells, the authors speculate that the wild-type MLL gene plays an important role early in the development of the hematopoietic system.  This article reviews recent progress in understanding the function of the wild-type MLL protein, with particular consideration of potential functions within the developing hematopoietic system.  Murine gain- and loss-of-function models have provided clues to the normal functions of MLL and altered functions of oncogenic MLL fusion proteins.  Biochemical and genetic approaches using other model organisms have also elucidated mechanisms by which these functions are achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFpKjX-b8FPVUIs6wFK5xJfW6udTcc2eYynH4G0h3ODrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38zgtFSgtg%253D%253D&md5=2ac2403992ed9904cd89efd1abcbc470</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1097%2F00062752-200207000-00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00062752-200207000-00004%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520role%2520of%2520MLL%2520in%2520hematopoiesis%2520and%2520leukemia%26jtitle%3DCurr.%2520Opin.%2520Hematol.%26date%3D2002%26volume%3D9%26spage%3D282%26epage%3D287%26doi%3D10.1097%2F00062752-200207000-00004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of leukemogenesis by MLL fusion proteins</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2141.2010.08459.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1111%2Fj.1365-2141.2010.08459.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=21118195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=141-154&author=R.+Marschalek&title=Mechanisms+of+leukemogenesis+by+MLL+fusion+proteins&doi=10.1111%2Fj.1365-2141.2010.08459.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of leukemogenesis by MLL fusion proteins</span></div><div class="casAuthors">Marschalek, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-154</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Infant acute leukemia is characterised by specific genetic rearrangements and a rapid onset of disease shortly after birth.  The vast majority of these cases bear rearranged MLL alleles.  However, many facets of MLL-rearranged leukemia are largely unknown.  Basically, there exists a fundamental and evolutionary conserved relationship between the family of MLL/Trithorax proteins and the regulation of HOX gene clusters.  Therefore, direct MLL fusion proteins are per se able to deregulate HOX genes, except when reciprocal MLL fusion proteins come into play.  This reviews discusses (i) the current situation in MLL-rearranged leukemia, (ii) the mol. and genetic tools to functionally investigate the many different MLL fusions, (iii) the latency of disease development, (iv) a novel cancer mechanism that has been recently uncovered when different MLL fusion protein complexes were characterized, (v) mutated signalling pathways in MLL-rearranged leukemia and (vi) presents new ideas on how a given MLL fusion protein may modulate existing signalling pathways in leukemic cells.  The hypothesis is posed that the many different fusion partners of MLL are critically distinct entities for which specific inhibitors should be identified in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeSBrUEMt1JLVg90H21EOLACvtfcHk0lgJK8ajuWzHRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D&md5=f0330065cb216625a242a2508d5d1c93</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2010.08459.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2010.08459.x%26sid%3Dliteratum%253Aachs%26aulast%3DMarschalek%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520leukemogenesis%2520by%2520MLL%2520fusion%2520proteins%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2011%26volume%3D152%26spage%3D141%26epage%3D154%26doi%3D10.1111%2Fj.1365-2141.2010.08459.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span> <span> </span><span class="NLM_article-title">MLL: How complex does it get?</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1002/jcb.20430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fjcb.20430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15779005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2005&pages=234-242&author=R.+Popovicauthor=N.+J.+Zeleznik-Le&title=MLL%3A+How+complex+does+it+get%3F&doi=10.1002%2Fjcb.20430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: How complex does it get?</span></div><div class="casAuthors">Popovic, Relja; Zeleznik-Le, Nancy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">234-242</span>CODEN:
                <span class="NLM_cas:coden">JCEBD5</span>;
        ISSN:<span class="NLM_cas:issn">0730-2312</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The mixed lineage leukemia (MLL) gene encodes a very large nuclear protein homologous to Drosophila trithorax (trx).  MLL is required for the proper maintenance of HOX gene expression during development and hematopoiesis.  The exact regulatory mechanism of HOX gene expression by MLL is poorly understood, but it is believed that MLL functions at the level of chromatin organization.  MLL was identified as a common target of chromosomal translocations assocd. with human acute leukemias.  About 50 different MLL fusion partners have been isolated to date, and while similarities exist between groups of partners, there exists no unifying property shared by all the partners.  MLL gene rearrangements are found in leukemias with both lymphoid and myeloid phenotypes and are often assocd. with infant and secondary leukemias.  The immature phenotype of the leukemic blasts suggests an important role for MLL in the early stages of hematopoietic development.  Mll homozygous mutant mice are embryonic lethal and exhibit deficiencies in yolk sac hematopoiesis.  Recently, two different MLL-contg. protein complexes have been isolated.  These and other gain- and loss-of-function expts. have provided insight into normal MLL function and altered functions of MLL fusion proteins.  This article reviews the progress made toward understanding the function of the wild-type MLL protein.  While many advances in understanding this multifaceted protein have been made since its discovery, many challenging questions remain to be answered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1zjCdPpVE_7Vg90H21EOLACvtfcHk0lgJK8ajuWzHRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFKjs7o%253D&md5=78d94e50c640366d568bb0876a91d916</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fjcb.20430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.20430%26sid%3Dliteratum%253Aachs%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26atitle%3DMLL%253A%2520How%2520complex%2520does%2520it%2520get%253F%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2005%26volume%3D95%26spage%3D234%26epage%3D242%26doi%3D10.1002%2Fjcb.20430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">When epigenetics kills: MLL fusion proteins in leukemia</span>. <i>Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1002/hon.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fhon.739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16118769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlamsrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=1-9&author=R.+K.+Slany&title=When+epigenetics+kills%3A+MLL+fusion+proteins+in+leukemia&doi=10.1002%2Fhon.739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">When epigenetics kills: MLL fusion proteins in leukemia</span></div><div class="casAuthors">Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Hematological Oncology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">HAONDL</span>;
        ISSN:<span class="NLM_cas:issn">0278-0232</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review.  Chromosomal aberrations that affect the MLL (Mixed Lineage Leukemia) gene at the locus 11q23 are assocd. with an aggressive subtype of leukemia.  These alterations create MLL fusion derivs. with an active transforming potential.  This review summarizes recent advances in our knowledge about normal and malignant MLL proteins with special emphasis on epigenetic processes affected by these mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon2L1pw0c0HLVg90H21EOLACvtfcHk0lh2SJHh6jdslA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlamsrrP&md5=ebe7728f67a381c6108f7f9125bc2796</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fhon.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhon.739%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DWhen%2520epigenetics%2520kills%253A%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DHematol.%2520Oncol.%26date%3D2005%26volume%3D23%26spage%3D1%26epage%3D9%26doi%3D10.1002%2Fhon.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomizawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinukawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isoyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horibe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakahata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, E.</span></span> <span> </span><span class="NLM_article-title">Outcome of risk-based therapy for infant acute lymphoblastic leukemia with or without an MLL gene rearrangement, with emphasis on late effects: a final report of two consecutive studies, MLL96 and MLL98, of the Japan infant leukemia study group</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2258</span>– <span class="NLM_lpage">2263</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2404903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fsj.leu.2404903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17690691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1WgsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2258-2263&author=D.+Tomizawaauthor=K.+Kohauthor=T.+Satoauthor=N.+Kinukawaauthor=A.+Morimotoauthor=K.+Isoyamaauthor=Y.+Kosakaauthor=T.+Odaauthor=M.+Odaauthor=Y.+Hayashiauthor=M.+Eguchiauthor=K.+Horibeauthor=T.+Nakahataauthor=S.+Mizutaniauthor=E.+Ishii&title=Outcome+of+risk-based+therapy+for+infant+acute+lymphoblastic+leukemia+with+or+without+an+MLL+gene+rearrangement%2C+with+emphasis+on+late+effects%3A+a+final+report+of+two+consecutive+studies%2C+MLL96+and+MLL98%2C+of+the+Japan+infant+leukemia+study+group&doi=10.1038%2Fsj.leu.2404903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Outcome of risk-based therapy for infant acute lymphoblastic leukemia with or without an MLL gene rearrangement, with emphasis on late effects: a final report of two consecutive studies, MLL96 and MLL98, of the Japan Infant Leukemia Study Group</span></div><div class="casAuthors">Tomizawa, D.; Koh, K.; Sato, T.; Kinukawa, N.; Morimoto, A.; Isoyama, K.; Kosaka, Y.; Oda, T.; Oda, M.; Hayashi, Y.; Eguchi, M.; Horibe, K.; Nakahata, T.; Mizutani, S.; Ishii, E.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2258-2263</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We evaluated the efficacy of a treatment strategy in which infants with acute lymphoblastic leukemia (ALL) were stratified by their MLL gene status and then assigned to different risk-based therapies.  A total of 102 patients were registered on two consecutive multicenter trials, designated MLL96 and MLL98, between 1995 and 2001.  Those with a rearranged MLL gene (MLL-R, n=80) were assigned to receive intensive chemotherapy followed by hematopoietic stem cell transplantation (HSCT), while those with germline MLL (MLL-G, n=22) were treated with chemotherapy alone.  The 5-yr event-free survival (EFS) rate for all 102 infants was 50.9% (95% confidence interval, 41.0-60.8%).  The most prominent late effect was growth impairment, obsd. in 58.9% of all evaluable patients in the MLL-R group.  This plan of risk-based therapy appears to have improved the overall prognosis for infants with ALL, compared with previously reported results.  However, over half the events in patients with MLL rearrangement occurred before the instigation of HSCT, and that HSCT-related toxic events comprised 36.3% (8/22) of post-transplantation events, suggesting that further stratification within the MLL-R group and the development of more effective early-phase intensification chemotherapy will be needed before the full potential of this strategy is realized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI1I1wHxFGWLVg90H21EOLACvtfcHk0lh2SJHh6jdslA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1WgsbnI&md5=4f513655b925c27a4aaffd9c8e236cdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2404903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2404903%26sid%3Dliteratum%253Aachs%26aulast%3DTomizawa%26aufirst%3DD.%26aulast%3DKoh%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DKinukawa%26aufirst%3DN.%26aulast%3DMorimoto%26aufirst%3DA.%26aulast%3DIsoyama%26aufirst%3DK.%26aulast%3DKosaka%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DT.%26aulast%3DOda%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DEguchi%26aufirst%3DM.%26aulast%3DHoribe%26aufirst%3DK.%26aulast%3DNakahata%26aufirst%3DT.%26aulast%3DMizutani%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DE.%26atitle%3DOutcome%2520of%2520risk-based%2520therapy%2520for%2520infant%2520acute%2520lymphoblastic%2520leukemia%2520with%2520or%2520without%2520an%2520MLL%2520gene%2520rearrangement%252C%2520with%2520emphasis%2520on%2520late%2520effects%253A%2520a%2520final%2520report%2520of%2520two%2520consecutive%2520studies%252C%2520MLL96%2520and%2520MLL98%252C%2520of%2520the%2520Japan%2520infant%2520leukemia%2520study%2520group%26jtitle%3DLeukemia%26date%3D2007%26volume%3D21%26spage%3D2258%26epage%3D2263%26doi%3D10.1038%2Fsj.leu.2404903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panetta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo-Coco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Poeta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venditti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurillo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Principe, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauriello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anemona, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palombo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadori, S.</span></span> <span> </span><span class="NLM_article-title">Chromosomal aberration of the 11q23 locus in acute leukemia and frequency of MLL gene translocation: results in 378 adult patients</span>. <i>Am. J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1309/RX27R8GJQM330C22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1309%2FRX27R8GJQM330C22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15323147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvF2htLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2004&pages=298-306&author=M.+C.+Coxauthor=P.+Panettaauthor=F.+Lo-Cocoauthor=G.+Del+Poetaauthor=A.+Vendittiauthor=L.+Maurilloauthor=M.+I.+Del+Principeauthor=A.+Maurielloauthor=L.+Anemonaauthor=A.+Brunoauthor=C.+Mazzoneauthor=P.+Palomboauthor=S.+Amadori&title=Chromosomal+aberration+of+the+11q23+locus+in+acute+leukemia+and+frequency+of+MLL+gene+translocation%3A+results+in+378+adult+patients&doi=10.1309%2FRX27R8GJQM330C22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chromosomal aberration of the 11q23 locus in acute leukemia and frequency of MLL gene translocation: Results in 378 adult patients</span></div><div class="casAuthors">Cox, M. Christina; Panetta, Paola; Lo-Coco, Francesco; del Poeta, Giovanni; Venditti, Adriano; Maurillo, Luca; del Principe, M. Ilaria; Mauriello, Alessandro; Anemona, Lucia; Bruno, Antonio; Mazzone, Carla; Palombo, Paolo; Amadori, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-306</span>CODEN:
                <span class="NLM_cas:coden">AJCPAI</span>;
        ISSN:<span class="NLM_cas:issn">0002-9173</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Pathologists</span>)
        </div><div class="casAbstract">Structural abnormality of the 11q23 band (11q23+) bearing the MLL gene translocation (MLL+) is a recurrent chromosome change obsd. in 3% to 7% of acute lymphoblastic leukemias and in 3% to 4% of acute myeloblastic leukemias.  The resoln. of conventional cytogenetics (CC) in detecting 11q23 rearrangement is limited when the translocative partner has a telomeric location; furthermore, CC can barely discriminate between true 11q23+/MLL+ and rearrangements clustering within the 11q22∼25 region without MLL involvement (MLL-).  We characterized a series of 378 consecutive patients with adult acute leukemia by CC, fluorescence in situ hybridization (FISH), and multiplex karyotyping (M-FISH) anal.  Our aim was to define the frequency of cryptic MLL+ cases and the frequency of MLL+ within 11q22∼25+ cases.  As expected, FISH was more sensitive than CC in detecting MLL+ cases, but rather unexpectedly, 9 (45%) of 20 patients with 11q22∼25+ were MLL-.  A better characterization of 11q22∼25+/MLL- leukemias is relevant for the identification of new, recurrent translocations.  Moreover, these cases should be readily distinguishable from 11q23+/MLL+ cases.  We recommend that karyotypic anal. always be complemented by mol. or FISH methods to unravel MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz-09gQjw1BLVg90H21EOLACvtfcHk0lgRBVg6uyvqpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvF2htLo%253D&md5=eff8f0b8bd29e6b7dedb7b428ce15bbf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1309%2FRX27R8GJQM330C22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1309%252FRX27R8GJQM330C22%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DM.%2BC.%26aulast%3DPanetta%26aufirst%3DP.%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DDel%2BPoeta%26aufirst%3DG.%26aulast%3DVenditti%26aufirst%3DA.%26aulast%3DMaurillo%26aufirst%3DL.%26aulast%3DDel%2BPrincipe%26aufirst%3DM.%2BI.%26aulast%3DMauriello%26aufirst%3DA.%26aulast%3DAnemona%26aufirst%3DL.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DMazzone%26aufirst%3DC.%26aulast%3DPalombo%26aufirst%3DP.%26aulast%3DAmadori%26aufirst%3DS.%26atitle%3DChromosomal%2520aberration%2520of%2520the%252011q23%2520locus%2520in%2520acute%2520leukemia%2520and%2520frequency%2520of%2520MLL%2520gene%2520translocation%253A%2520results%2520in%2520378%2520adult%2520patients%26jtitle%3DAm.%2520J.%2520Clin.%2520Pathol.%26date%3D2004%26volume%3D122%26spage%3D298%26epage%3D306%26doi%3D10.1309%2FRX27R8GJQM330C22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimartino, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Mll rearrangements in haematological malignancies: lessons from clinical and biological studies</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2141.1999.01439.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1046%2Fj.1365-2141.1999.01439.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10468849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1999&pages=614-626&author=J.+F.+Dimartinoauthor=M.+L.+Cleary&title=Mll+rearrangements+in+haematological+malignancies%3A+lessons+from+clinical+and+biological+studies&doi=10.1046%2Fj.1365-2141.1999.01439.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MLL rearrangements in haematological malignancies: Lessons from clinical and biological studies</span></div><div class="casAuthors">DiMartino, Jorge F.; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 96 refs., covering the MLL family of 11q23 chromosomal rearrangements, assocn. between 11q23 rearrangements and leukemia phenotype, MLL rearrangements in the absence of leukemia, prognostic implications of 11q23 rearrangements, etiol. of MLL rearrangements, MLL rearrangements leading to leukemia with short latency, structure and probable function of the MLL protein, and possible mechanisms of leukemogenesis by MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOiR8FH8-xQbVg90H21EOLACvtfcHk0lgRBVg6uyvqpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D&md5=845845da43dc70b59322261edb88bb6e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1999.01439.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1999.01439.x%26sid%3Dliteratum%253Aachs%26aulast%3DDimartino%26aufirst%3DJ.%2BF.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMll%2520rearrangements%2520in%2520haematological%2520malignancies%253A%2520lessons%2520from%2520clinical%2520and%2520biological%2520studies%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D1999%26volume%3D106%26spage%3D614%26epage%3D626%26doi%3D10.1046%2Fj.1365-2141.1999.01439.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">The molecular biology of mixed lineage leukemia</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">984</span>– <span class="NLM_lpage">993</span>, <span class="refDoi"> DOI: 10.3324/haematol.2008.002436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.3324%2Fhaematol.2008.002436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=19535349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=984-993&author=R.+K.+Slany&title=The+molecular+biology+of+mixed+lineage+leukemia&doi=10.3324%2Fhaematol.2008.002436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular biology of mixed lineage leukemia</span></div><div class="casAuthors">Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">984-993</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">0390-6078</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia is a very aggressive blood cancer that predominantly occurs in pediatric patients.  In contrast to other types of childhood acute leukemias, mixed lineage leukemia presents with a dismal prognosis and despite the availability of advanced treatment methods cure rates have stagnated over the last years.  Mixed lineage leukemia is characterized by the presence of MLL fusion proteins that are the result of chromosomal translocations affecting the MLL gene at 11q23.  These events juxtapose the amino-terminus of the histone methyltransferase MLL with a variety of different fusion partners that destroy normal histone methyltransferase function of MLL and replace it by heterologous functions contributed by the fusion partner.  The resulting chimeras are transcriptional regulators that take control of targets normally controlled by MLL with the clustered HOX homeobox genes as prominent examples.  Recent studies suggested that MLL fusion partners activate transcription by two different mechanisms.  Some of these proteins are themselves chromatin modifiers that introduce histone acetylation whereas other fusion partners can recruit histone methyltransferases.  In particular, histone H3 specific methylation at lysine 79 catalyzed by DOT1L has been recognized as a hallmark of chromatin activated by MLL fusion proteins.  Interestingly several frequent MLL fusion partners seem to coordinate DOT1L activity with a protein complex that stimulates the elongation phase of transcription by phosphorylating the carboxy-terminal repeat domain of RNA polymerase II.  The discovery of these novel enzymic activities that are essentially involved in MLL fusion protein function presents potential new targets for a rational drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpudla5d7qUgrVg90H21EOLACvtfcHk0lhGxVgKxDlpKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGitrrK&md5=103284cb0a582febe6e14e8d12afc4f1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2008.002436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2008.002436%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520molecular%2520biology%2520of%2520mixed%2520lineage%2520leukemia%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D984%26epage%3D993%26doi%3D10.3324%2Fhaematol.2008.002436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span> <span> </span><span class="NLM_article-title">Disordered epigenetic regulation in MLL-related leukemia</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1007/s12185-012-1180-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1007%2Fs12185-012-1180-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=23054645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFamtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2012&pages=428-437&author=Y.+Zhangauthor=A.+Chenauthor=X.+M.+Yanauthor=G.+Huang&title=Disordered+epigenetic+regulation+in+MLL-related+leukemia&doi=10.1007%2Fs12185-012-1180-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered epigenetic regulation in MLL-related leukemia</span></div><div class="casAuthors">Zhang, Yue; Chen, Aili; Yan, Xiao-Mei; Huang, Gang</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Hematology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">428-437</span>CODEN:
                <span class="NLM_cas:coden">IJHEEY</span>;
        ISSN:<span class="NLM_cas:issn">0925-5710</span>.
    
            (<span class="NLM_cas:orgname">Springer Japan</span>)
        </div><div class="casAbstract">A review.  Leukemias bearing rearrangements of chromosome 11q23 are of particular interest due to their unique clin. and biol. characteristics. 11q23 abnormalities occur in up to 70 % of infant leukemias, and about 10 % of adult acute myelogenous leukemias (AML).  Two major rearrangements of the MLL gene are found in MLL-related leukemia.  The most common of these is balanced translocations in which the N-terminal portion of MLL is fused to the C-terminus of the translocation partner.  To date, nearly 100 different chromosome bands have been described in rearrangements involving MLL, and more than 70 known fusion partners of MLL have been cloned and characterized at the mol. level.  Another major aberration of the MLL gene creates a repeat within the N-terminal MLL resulting in an internal partial tandem duplication (PTD).  As a consequence, an extra amino-terminus is added in-frame to full-length MLL, resulting in leukemogenic MLL-PTD.  MLL-PTD occurs predominantly in myeloid dysplasia syndromes, secondary AML (s-AML), and de novo AML.  The presence of an MLL rearrangement generally confers a poor prognosis.  MLL fusions and MLL-PTD are transcriptional regulators that take control of targets normally controlled by MLL, with the clustered HOX homeobox genes as prominent examples.  Several epigenetic regulators that modify DNA or histones have been implicated in MLL fusion driven leukemogenesis, including DNA methylation, histone acetylation, and histone methylation.  Recently, the histone methyltransferase DOT1L, the bromodomain and extra-terminal (BET) family member BRD4, and the MLL-interacting protein Menin have emerged as important mediators of MLL fusion-mediated leukemic transformation.  The clin. development of targeted inhibitors of these epigenetic regulators has heralded promise for the treatment of MLL fusion leukemia.  Although the biol. function and mol. mechanism for MLL-PTD remains largely unknown, based on the primary protein structure of MLL-PTD and the knowledge gained so far from MLL fusions, newly developed inhibitors of epigenetic regulators could potentially also prove effective in the treatment of MLL-PTD related leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCNNdsRmcXjLVg90H21EOLACvtfcHk0lhGxVgKxDlpKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFamtbbK&md5=9b8ad8df35b4a098f34e343800cc4d0c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs12185-012-1180-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-012-1180-0%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DX.%2BM.%26aulast%3DHuang%26aufirst%3DG.%26atitle%3DDisordered%2520epigenetic%2520regulation%2520in%2520MLL-related%2520leukemia%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2012%26volume%3D96%26spage%3D428%26epage%3D437%26doi%3D10.1007%2Fs12185-012-1180-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">MLL: a histone methyltransferase disrupted in leukemia</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2004.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.molmed.2004.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15464450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+a+histone+methyltransferase+disrupted+in+leukemia&doi=10.1016%2Fj.molmed.2004.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: a histone methyltransferase disrupted in leukemia</span></div><div class="casAuthors">Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">500-507</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rearrangements of the MLL gene, which is located at chromosome 11q23, are assocd. with aggressive acute leukemias in both children and adults.  MLL regulates Hox gene expression through direct promoter binding and histone modification.  MLL rearrangements occurring in leukemia include MLL fusion genes, partial tandem duplications of MLL and MLL amplification.  MLL fusions and amplification upregulate Hox expression, apparently resulting in a block of hematopoietic differentiation.  Future therapies for MLL-assocd. leukemia might involve blocking Hox gene upregulation by using fusion proteins or inhibiting the activity of Hox proteins themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2pQrErnZf7Vg90H21EOLACvtfcHk0lhGxVgKxDlpKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D&md5=ae9d59e5e59cc017fd0fcf479a4a618d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2004.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2004.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520a%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507%26doi%3D10.1016%2Fj.molmed.2004.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0ljGmUlkI3hLJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caslini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Osta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of MLL amino terminal sequences with menin is required for transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7283</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1158%2F0008-5472.CAN-06-2369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17671196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVemtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=7275-7283&author=C.+Casliniauthor=Z.+Yangauthor=M.+El-Ostaauthor=T.+A.+Milneauthor=R.+K.+Slanyauthor=J.+L.+Hess&title=Interaction+of+MLL+amino+terminal+sequences+with+menin+is+required+for+transformation&doi=10.1158%2F0008-5472.CAN-06-2369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of MLL Amino Terminal Sequences with Menin Is Required for Transformation</span></div><div class="casAuthors">Caslini, Corrado; Yang, Zhaohai; El-Osta, Mohamad; Milne, Thomas A.; Slany, Robert K.; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7275-7283</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Rearrangements of the mixed lineage leukemia gene MLL are assocd. with aggressive lymphoid and myeloid leukemias.  The resulting MLL fusion proteins enforce high-level expression of HOX genes and the HOX cofactor MEIS1, which is pivotal for leukemogenesis.  Both wild-type MLL and MLL fusion proteins interact with the tumor suppressor menin and with the Hoxa9 locus in vivo.  Here, we show that MLL sequences between amino acids 5 and 44 are required for interaction with menin and for the transformation of hematopoietic progenitors.  Blocking the MLL-menin interaction by the expression of a dominant neg. inhibitor composed of amino terminal MLL sequences down-regulates Meis1 expression and inhibits cell proliferation, suggesting that targeting this interaction may be an effective therapeutic strategy for leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4C8hcS0mav7Vg90H21EOLACvtfcHk0ljGmUlkI3hLJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVemtrc%253D&md5=df8f9ffb41c080aae65fe9722cdf11ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2369%26sid%3Dliteratum%253Aachs%26aulast%3DCaslini%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DEl-Osta%26aufirst%3DM.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520MLL%2520amino%2520terminal%2520sequences%2520with%2520menin%2520is%2520required%2520for%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D7275%26epage%3D7283%26doi%3D10.1158%2F0008-5472.CAN-06-2369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ccr.2008.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=18598942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=36-46&author=A.+Yokoyamaauthor=M.+L.+Cleary&title=Menin+critically+links+MLL+proteins+with+LEDGF+on+cancer-associated+target+genes&doi=10.1016%2Fj.ccr.2008.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span></div><div class="casAuthors">Yokoyama, Akihiko; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-46</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Menin displays the unique ability to either promote oncogenic function in the hematopoietic lineage or suppress tumorigenesis in the endocrine lineage; however, its mol. mechanism of action has not been defined.  We demonstrate here that these discordant functions are unified by menin's ability to serve as a mol. adaptor that phys. links the MLL (mixed-lineage leukemia) histone methyltransferase with LEDGF (lens epithelium-derived growth factor), a chromatin-assocd. protein previously implicated in leukemia, autoimmunity, and HIV-1 pathogenesis.  LEDGF is required for both MLL-dependent transcription and leukemic transformation.  Conversely, a subset of menin mutations in multiple endocrine neoplasia type 1 patients abrogate interaction with LEDGF while preserving MLL interaction but nevertheless compromise MLL/menin-dependent functions.  Thus, LEDGF critically assocs. with MLL and menin at the nexus of transcriptional pathways that are recurrently targeted in diverse diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjaPq2DDJ-sLVg90H21EOLACvtfcHk0livneTmthRCMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekt7Y%253D&md5=3eb5242d529ec6c8222908a6116547f7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMenin%2520critically%2520links%2520MLL%2520proteins%2520with%2520LEDGF%2520on%2520cancer-associated%2520target%2520genes%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26spage%3D36%26epage%3D46%26doi%3D10.1016%2Fj.ccr.2008.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16239140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span></div><div class="casAuthors">Yokoyama, Akihiko; Somervaille, Tim C. P.; Smith, Kevin S.; Rozenblatt-Rosen, Orit; Meyerson, Matthew; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed-Lineage Leukemia (MLL) protein is a histone methyltransferase that is mutated in clin. and biol. distinctive subsets of acute leukemia.  MLL normally assocs. with a cohort of highly conserved cofactors to form a macromol. complex that includes menin, a product of the MEN1 tumor suppressor gene, which is mutated in heritable and sporadic endocrine tumors.  We demonstrate here that oncogenic MLL fusion proteins retain an ability to stably assoc. with menin through a high-affinity, amino-terminal, conserved binding motif and that this interaction is required for the initiation of MLL-mediated leukemogenesis.  Furthermore, menin is essential for maintenance of MLL-assocd. but not other oncogene induced myeloid transformation.  Acute genetic ablation of menin reverses aberrant Hox gene expression mediated by MLL-menin promoter-assocd. complexes, and specifically abrogates the differentiation arrest and oncogenic properties of MLL-transformed leukemic blasts.  These results demonstrate that a human oncoprotein is critically dependent on direct phys. interaction with a tumor suppressor protein for its oncogenic activity, validate a potential target for mol. therapy, and suggest central roles for menin in altered epigenetic functions underlying the pathogenesis of hematopoietic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MXmqsWp8BrVg90H21EOLACvtfcHk0livneTmthRCMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP&md5=d582f614f98d25ebaf2282e2915fd4b6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeshan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pear, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, X.</span></span> <span> </span><span class="NLM_article-title">The tumor suppressor menin regulates hematopoiesis and myeloid transformation by influencing Hox gene expression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1023</span>, <span class="refDoi"> DOI: 10.1073/pnas.0510347103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1073%2Fpnas.0510347103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16415155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=1018-1023&author=Y.+X.+Chenauthor=J.+Yanauthor=K.+Keeshanauthor=A.+T.+Tubbsauthor=H.+Wangauthor=A.+Silvaauthor=E.+J.+Brownauthor=J.+L.+Hessauthor=W.+S.+Pearauthor=X.+Hua&title=The+tumor+suppressor+menin+regulates+hematopoiesis+and+myeloid+transformation+by+influencing+Hox+gene+expression&doi=10.1073%2Fpnas.0510347103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The tumor suppressor menin regulates hematopoiesis and myeloid transformation by influencing Hox gene expression</span></div><div class="casAuthors">Chen, Ya-Xiong; Yan, Jizhou; Keeshan, Karen; Tubbs, Anthony T.; Wang, Haoren; Silva, Albert; Brown, Eric J.; Hess, Jay L.; Pear, Warren S.; Hua, Xianxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1018-1023</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Menin is the product of the tumor suppressor gene Men1 that is mutated in the inherited tumor syndrome multiple endocrine neoplasia type 1 (MEN1).  Menin has been shown to interact with SET-1 domain-contg. histone 3 lysine 4 (H3K4) methyltransferases including mixed lineage leukemia proteins to regulate homeobox (Hox) gene expression in vitro.  Using conditional Men1 knockout mice, we have investigated the requirement for menin in hematopoiesis and myeloid transformation.  Men1 excision causes redn. of Hoxa9 expression, colony formation by hematopoietic progenitors, and the peripheral white blood cell count.  Menin directly activates Hoxa9 expression, at least in part, by binding to the Hoxa9 locus, facilitating methylation of H3K4, and recruiting the methylated H3K4 binding protein chd1 to the locus.  Consistent with signaling downstream of menin, ectopic expression of both Hoxa9 and Meis1 rescues colony formation defects in Men1-excised bone marrow.  Moreover, Men1 excision also suppresses proliferation of leukemogenic mixed lineage leukemia-AF9 fusion-protein-transformed myeloid cells and Hoxa9 expression.  These studies uncover an important role for menin in both normal hematopoiesis and myeloid transformation and provide a mechanistic understanding of menin's function in these processes that may be used for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrba146rYH-WrVg90H21EOLACvtfcHk0livneTmthRCMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKgtb4%253D&md5=c6f88d7cab1d4e755ff2e27ecc88bf83</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0510347103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0510347103%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DKeeshan%26aufirst%3DK.%26aulast%3DTubbs%26aufirst%3DA.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DSilva%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DE.%2BJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DPear%26aufirst%3DW.%2BS.%26aulast%3DHua%26aufirst%3DX.%26atitle%3DThe%2520tumor%2520suppressor%2520menin%2520regulates%2520hematopoiesis%2520and%2520myeloid%2520transformation%2520by%2520influencing%2520Hox%2520gene%2520expression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D1018%26epage%3D1023%26doi%3D10.1073%2Fpnas.0510347103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the menin-MLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin-MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0lgKPh44CCmjeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin-MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurung, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veniaminova, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">The same pocket in menin binds both MLL and JUND but has opposite effects on transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1038/nature10806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnature10806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22327296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XitFalt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=542-546&author=J.+Huangauthor=B.+Gurungauthor=B.+Wanauthor=S.+Matkarauthor=N.+A.+Veniaminovaauthor=K.+Wanauthor=J.+L.+Merchantauthor=X.+Huaauthor=M.+Lei&title=The+same+pocket+in+menin+binds+both+MLL+and+JUND+but+has+opposite+effects+on+transcription&doi=10.1038%2Fnature10806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The same pocket in menin binds both MLL and JUND but has opposite effects on transcription</span></div><div class="casAuthors">Huang, Jing; Gurung, Buddha; Wan, Bingbing; Matkar, Smita; Veniaminova, Natalia A.; Wan, Ke; Merchant, Juanita L.; Hua, Xianxin; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Menin is a tumor suppressor protein whose loss or inactivation causes multiple endocrine neoplasia 1 (MEN1), a hereditary autosomal dominant tumor syndrome that is characterized by tumorigenesis in multiple endocrine organs.  Menin interacts with many proteins and is involved in a variety of cellular processes.  Menin binds the JUN family transcription factor JUND and inhibits its transcriptional activity.  Several MEN1 missense mutations disrupt the menin-JUND interaction, suggesting a correlation between the tumor-suppressor function of menin and its suppression of JUND-activated transcription.  Menin also interacts with mixed lineage leukemia protein 1 (MLL1), a histone H3 lysine 4 methyltransferase, and functions as an oncogenic cofactor to upregulate gene transcription and promote MLL1-fusion-protein-induced leukemogenesis.  A recent report on the tethering of MLL1 to chromatin binding factor lens epithelium-derived growth factor (LEDGF) by menin indicates that menin is a mol. adaptor coordinating the functions of multiple proteins.  Despite its importance, how menin interacts with many distinct partners and regulates their functions remains poorly understood.  Here we present the crystal structures of human menin in its free form and in complexes with MLL1 or with JUND, or with an MLL1-LEDGF heterodimer.  These structures show that menin contains a deep pocket that binds short peptides of MLL1 or JUND in the same manner, but that it can have opposite effects on transcription.  The menin-JUND interaction blocks JUN N-terminal kinase (JNK)-mediated JUND phosphorylation and suppresses JUND-induced transcription.  In contrast, menin promotes gene transcription by binding the transcription activator MLL1 through the peptide pocket while still interacting with the chromatin-anchoring protein LEDGF at a distinct surface formed by both menin and MLL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCm7SaZ0LyW7Vg90H21EOLACvtfcHk0lgKPh44CCmjeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitFalt7k%253D&md5=95c952303b917ed0c5e2d475926941d7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature10806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10806%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DGurung%26aufirst%3DB.%26aulast%3DWan%26aufirst%3DB.%26aulast%3DMatkar%26aufirst%3DS.%26aulast%3DVeniaminova%26aufirst%3DN.%2BA.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DMerchant%26aufirst%3DJ.%2BL.%26aulast%3DHua%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DThe%2520same%2520pocket%2520in%2520menin%2520binds%2520both%2520MLL%2520and%2520JUND%2520but%2520has%2520opposite%2520effects%2520on%2520transcription%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D542%26epage%3D546%26doi%3D10.1038%2Fnature10806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+B.+Shiauthor=M.+J.+Muraiauthor=S.+H.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+menin-MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0ljkHENLBuKSjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%2BB.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%2BH.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520menin-MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0ljkHENLBuKSjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Small-Molecule Inhibitors of the Menin-Mixed Lineage Leukemia (MLL) Interaction Closely Mimic a Natural Protein-Protein Interaction</span></div><div class="casAuthors">He, Shihan; Senter, Timothy J.; Pollock, Jonathan; Han, Changho; Upadhyay, Sunil Kumar; Purohit, Trupta; Gogliotti, Rocco D.; Lindsley, Craig W.; Cierpicki, Tomasz; Stauffer, Shaun R.; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1543-1556</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction (PPI) between menin and mixed lineage leukemia (MLL) plays a crit. role in acute leukemias, and inhibition of this interaction represents a new potential therapeutic strategy for MLL leukemias.  We report development of a novel class of small-mol. inhibitors of the menin-MLL interaction, the hydroxy- and aminomethylpiperidine compds., which originated from HTS of ∼288000 small mols.  We detd. menin-inhibitor co-crystal structures and found that these compds. closely mimic all key interactions of MLL with menin.  Extensive crystallog. studies combined with structure-based design were applied for optimization of these compds., resulting in MIV-6R, which inhibits the menin-MLL interaction with IC50 = 56 nM.  Treatment with MIV-6 demonstrated strong and selective effects in MLL leukemia cells, validating specific mechanism of action.  Our studies provide novel and attractive scaffold as a new potential therapeutic approach for MLL leukemias and demonstrate an example of PPI amenable to inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC4yHqMSn87Vg90H21EOLACvtfcHk0litJxYncyaahA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D&md5=a580d1b2ea3f657c4abb31678f39548c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and mixed lineage leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+mixed+lineage+leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Property Focused Structure-Based Optimization of Small Molecule Inhibitors of the Protein-Protein Interaction between Menin and Mixed Lineage Leukemia (MLL)</span></div><div class="casAuthors">Borkin, Dmitry; Pollock, Jonathan; Kempinska, Katarzyna; Purohit, Trupta; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Miao, Hongzhi; Shukla, Shirish; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of potent small mol. inhibitors of protein-protein interactions with optimized druglike properties represents a challenging task in lead optimization process.  Here, we report synthesis and structure-based optimization of new thienopyrimidine class of compds., which block the protein-protein interaction between menin and MLL fusion proteins that plays an important role in acute leukemias with MLL translocations.  We performed simultaneous optimization of both activity and druglike properties through systematic exploration of substituents introduced to the indole ring of lead compd. 1 (MI-136) to identify compds. suitable for in vivo studies in mice.  This work resulted in the identification of compd. 27 (MI-538), which showed significantly increased activity, selectivity, polarity, and pharmacokinetic profile over 1 and demonstrated a pronounced effect in a mouse model of MLL leukemia.  This study, which reports detailed structure-activity and structure-property relationships for the menin-MLL inhibitors, demonstrates challenges in optimizing inhibitors of protein-protein interactions for potential therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDD5t7XKKz_LVg90H21EOLACvtfcHk0litJxYncyaahA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D&md5=f89b6eda72671f133a38181b3b23632d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linhares, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Complexity of blocking bivalent protein-protein interactions: Development of a highly potent inhibitor of the menin-mixed-lineage leukemia interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4832</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4832-4850&author=D.+Borkinauthor=S.+Klossowskiauthor=J.+Pollockauthor=H.+Miaoauthor=B.+M.+Linharesauthor=K.+Kempinskaauthor=Z.+Jinauthor=T.+Purohitauthor=B.+Wenauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Complexity+of+blocking+bivalent+protein-protein+interactions%3A+Development+of+a+highly+potent+inhibitor+of+the+menin-mixed-lineage+leukemia+interaction&doi=10.1021%2Facs.jmedchem.8b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Complexity of Blocking Bivalent Protein-Protein Interactions: Development of a Highly Potent Inhibitor of the Menin-Mixed-Lineage Leukemia Interaction</span></div><div class="casAuthors">Borkin, Dmitry; Klossowski, Szymon; Pollock, Jonathan; Miao, Hongzhi; Linhares, Brian M.; Kempinska, Katarzyna; Jin, Zhuang; Purohit, Trupta; Wen, Bo; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4832-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors.  Here, we report the development of a new generation of menin-MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compds.  This work resulted in compd. I (MI-1481), which showed very potent inhibition of the menin-MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin-MLL1 inhibitor reported to date.  The crystal structure of the menin-I complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of I.  Compd. I also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models.  Thus, I is a valuable menin-MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_eephZ2UHnLVg90H21EOLACvtfcHk0lhbxfKySdw_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D&md5=8d12fa816af3a5aa191694d4bfe48802</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00071%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DLinhares%26aufirst%3DB.%2BM.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DComplexity%2520of%2520blocking%2520bivalent%2520protein-protein%2520interactions%253A%2520Development%2520of%2520a%2520highly%2520potent%2520inhibitor%2520of%2520the%2520menin-mixed-lineage%2520leukemia%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4832%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=25817203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Blocks Progression of MLL Leukemia In Vivo</span></div><div class="casAuthors">Borkin, Dmitry; He, Shihan; Miao, Hongzhi; Kempinska, Katarzyna; Pollock, Jonathan; Chase, Jennifer; Purohit, Trupta; Malik, Bhavna; Zhao, Ting; Wang, Jingya; Wen, Bo; Zong, Hongliang; Jones, Morgan; Danet-Desnoyers, Gwenn; Guzman, Monica L.; Talpaz, Moshe; Bixby, Dale L.; Sun, Duxin; Hess, Jay L.; Muntean, Andrew G.; Maillard, Ivan; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-602</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal translocations affecting mixed lineage leukemia gene (MLL) result in acute leukemias resistant to therapy.  The leukemogenic activity of MLL fusion proteins is dependent on their interaction with menin, providing basis for therapeutic intervention.  Here we report the development of highly potent and orally bioavailable small-mol. inhibitors of the menin-MLL interaction, MI-463 and MI-503, and show their profound effects in MLL leukemia cells and substantial survival benefit in mouse models of MLL leukemia.  Finally, we demonstrate the efficacy of these compds. in primary samples derived from MLL leukemia patients.  Overall, we demonstrate that pharmacol. inhibition of the menin-MLL interaction represents an effective treatment for MLL leukemias in vivo and provide advanced mol. scaffold for clin. lead identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc2hESfSmR8bVg90H21EOLACvtfcHk0lhbxfKySdw_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D&md5=1369b1556d139e9006c3fa7d1add49a0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors block oncogenic transformation by MLL-fusion proteins in a fusion partner-independent manner</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1038/leu.2015.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fleu.2015.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=26084867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wqsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=508-513&author=S.+Heauthor=B.+Malikauthor=D.+Borkinauthor=H.+Miaoauthor=S.+Shuklaauthor=K.+Kempinskaauthor=T.+Purohitauthor=J.+Wangauthor=L.+Chenauthor=B.+Parkinauthor=S.+N.+Malekauthor=G.+Danet-Desnoyersauthor=A.+G.+Munteanauthor=T.+Cierpickiauthor=J.+Grembecka&title=Menin-MLL+inhibitors+block+oncogenic+transformation+by+MLL-fusion+proteins+in+a+fusion+partner-independent+manner&doi=10.1038%2Fleu.2015.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors block oncogenic transformation by MLL-fusion proteins in a fusion partner-independent manner</span></div><div class="casAuthors">He, S.; Malik, B.; Borkin, D.; Miao, H.; Shukla, S.; Kempinska, K.; Purohit, T.; Wang, J.; Chen, L.; Parkin, B.; Malek, S. N.; Danet-Desnoyers, G.; Muntean, A. G.; Cierpicki, T.; Grembecka, J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">508-513</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genome-wide anal. of gene expression revealed that MI-2-2 reverses MLL-rearranged gene signature and induced expression of genes assocd. with differentiation in cells harboring various MLL-fusion proteins.  Taken together, the menin MLL inhibitor is capable of reversing oncogenic activity of different MLL fusions according to a mechanism that is independent of the fusion partner.  Such activity remains to be in agreement with menin binding motif retained in all MLL-fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5a_fPNgF5cLVg90H21EOLACvtfcHk0lhbxfKySdw_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wqsr7L&md5=2a46a4568d58ddb2fde5727ab348c710</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.144%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DParkin%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%2BN.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DMenin-MLL%2520inhibitors%2520block%2520oncogenic%2520transformation%2520by%2520MLL-fusion%2520proteins%2520in%2520a%2520fusion%2520partner-independent%2520manner%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D508%26epage%3D513%26doi%3D10.1038%2Fleu.2015.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the Menin-MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fernandez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+Menin-MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0ljBGrkFSn5Dbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520Menin-MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+H.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0ljBGrkFSn5Dbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%2BH.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez%0AMolina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0liobcLBWWnbvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of high-affinity macrocyclic peptidomimetics to block the menin-mixed lineage leukemia 1 (MLL1) protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1021/jm3015298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3015298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCqs7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1113-1123&author=H.+Zhouauthor=L.+Liuauthor=J.+Huangauthor=D.+Bernardauthor=H.+Karatasauthor=A.+Navarroauthor=M.+Leiauthor=S.+Wang&title=Structure-based+design+of+high-affinity+macrocyclic+peptidomimetics+to+block+the+menin-mixed+lineage+leukemia+1+%28MLL1%29+protein-protein+interaction&doi=10.1021%2Fjm3015298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of High-Affinity Macrocyclic Peptidomimetics to Block the Menin-Mixed Lineage Leukemia 1 (MLL1) Protein-Protein Interaction</span></div><div class="casAuthors">Zhou, Haibin; Liu, Liu; Huang, Jing; Bernard, Denzil; Karatas, Hacer; Navarro, Alexandro; Lei, Ming; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1113-1123</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Menin is an essential oncogenic cofactor for mixed lineage leukemia 1 (MLL1)-mediated leukemogenesis through its direct interaction with MLL1.  Targeting the menin-MLL1 protein-protein interaction represents a promising strategy to block MLL1-mediated leukemogenesis.  Employing a structure-based approach and starting from a linear MLL1 octapeptide, we have designed a class of potent macrocyclic peptidomimetic inhibitors of the menin-MLL1 interaction.  The most potent macrocyclic peptidomimetic (MCP-1), 34 (I), binds to menin with a Ki value of 4.7 nM and is >600 times more potent than the corresponding acyclic peptide.  Compd. 34 is also less peptide-like and has a lower mol. wt. than the initial MLL1 peptide.  Therefore, compd. 34 serves as a promising lead structure for the design of potent and cell-permeable inhibitors of the menin-MLL1 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMcRYRG7I0y7Vg90H21EOLACvtfcHk0liobcLBWWnbvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCqs7rE&md5=df004832c2fad317611851268a3d574e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm3015298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3015298%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DNavarro%26aufirst%3DA.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520high-affinity%2520macrocyclic%2520peptidomimetics%2520to%2520block%2520the%2520menin-mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1113%26epage%3D1123%26doi%3D10.1021%2Fjm3015298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0liobcLBWWnbvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liobcLBWWnbvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lil04dvEAB0Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lil04dvEAB0Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.11.2; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DRoversi%26aufirst%3DP.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Meng Zhang, Angelo Aguilar, Shilin Xu, Liyue Huang, Krishnapriya Chinnaswamy, Taryn Sleger, Bo Wang, Stefan Gross, Brandon N. Nicolay, Sebastien Ronseaux, Kaitlin Harvey, Yu Wang, Donna McEachern, Paul D. Kirchhoff, Zhaomin Liu, Jeanne Stuckey, Adriana E. Tron, Tao Liu, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of M-1121 as an Orally Active Covalent Inhibitor of Menin-MLL Interaction Capable of Achieving Complete and Long-Lasting Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10333-10349. <a href="https://doi.org/10.1021/acs.jmedchem.1c00789" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00789</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00789%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BM-1121%252Bas%252Ban%252BOrally%252BActive%252BCovalent%252BInhibitor%252Bof%252BMenin-MLL%252BInteraction%252BCapable%252Bof%252BAchieving%252BComplete%252Band%252BLong-Lasting%252BTumor%252BRegression%26aulast%3DZhang%26aufirst%3DMeng%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D30042021%26date%3D01072021%26volume%3D64%26issue%3D14%26spage%3D10333%26epage%3D10349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shilin Xu, Angelo Aguilar, Liyue Huang, Tianfeng Xu, Ke Zheng, Donna McEachern, Sally Przybranowski, Caroline Foster, Kaitlin Zawacki, Zhaomin Liu, Krishnapriya Chinnaswamy, Jeanne Stuckey, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of M-808 as a Highly Potent, Covalent, Small-Molecule Inhibitor of the Menin–MLL Interaction with Strong In Vivo Antitumor Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4997-5010. <a href="https://doi.org/10.1021/acs.jmedchem.0c00547" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00547</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00547%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BM-808%252Bas%252Ba%252BHighly%252BPotent%25252C%252BCovalent%25252C%252BSmall-Molecule%252BInhibitor%252Bof%252Bthe%252BMenin%2525E2%252580%252593MLL%252BInteraction%252Bwith%252BStrong%252BIn%252BVivo%252BAntitumor%252BActivity%26aulast%3DXu%26aufirst%3DShilin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03042020%26date%3D27042020%26volume%3D63%26issue%3D9%26spage%3D4997%26epage%3D5010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, function and inhibition of critical protein–protein interactions involving mixed lineage leukemia 1 and its fusion oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01057-7" title="DOI URL">https://doi.org/10.1186/s13045-021-01057-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01057-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01057-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DStructure%25252C%252Bfunction%252Band%252Binhibition%252Bof%252Bcritical%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Binvolving%252Bmixed%252Blineage%252Bleukemia%252B1%252Band%252Bits%252Bfusion%252Boncoproteins%26aulast%3DLi%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Myojeong  Kim</span>, <span class="hlFld-ContribAuthor ">Euna  You</span>, <span class="hlFld-ContribAuthor ">Seongjin  Park</span>, <span class="hlFld-ContribAuthor ">Sungwoo  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Divergent reactivity of sulfinates with pyridinium salts based on one-
              versus
              two-electron pathways. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (19)
                                     , 6629-6637. <a href="https://doi.org/10.1039/D1SC00776A" title="DOI URL">https://doi.org/10.1039/D1SC00776A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1SC00776A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1SC00776A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DDivergent%252Breactivity%252Bof%252Bsulfinates%252Bwith%252Bpyridinium%252Bsalts%252Bbased%252Bon%252Bone-%252Bversus%252Btwo-electron%252Bpathways%26aulast%3DKim%26aufirst%3DMyojeong%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D19%26spage%3D6629%26epage%3D6637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ezgi  Ozyerli‐Goknar</span>, <span class="hlFld-ContribAuthor ">Sheikh  Nizamuddin</span>, <span class="hlFld-ContribAuthor ">H. T. Marc  Timmers</span>. </span><span class="cited-content_cbyCitation_article-title">A Box of Chemistry to Inhibit the MEN1 Tumor Suppressor Gene Promoting Leukemia. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (9)
                                     , 1391-1402. <a href="https://doi.org/10.1002/cmdc.202000972" title="DOI URL">https://doi.org/10.1002/cmdc.202000972</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000972%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BBox%252Bof%252BChemistry%252Bto%252BInhibit%252Bthe%252BMEN1%252BTumor%252BSuppressor%252BGene%252BPromoting%252BLeukemia%26aulast%3DOzyerli%25E2%2580%2590Goknar%26aufirst%3DEzgi%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D9%26spage%3D1391%26epage%3D1402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siska  Van Belle</span>, <span class="hlFld-ContribAuthor ">Sara  El Ashkar</span>, <span class="hlFld-ContribAuthor ">Kateřina  Čermáková</span>, <span class="hlFld-ContribAuthor ">Filip  Matthijssens</span>, <span class="hlFld-ContribAuthor ">Steven  Goossens</span>, <span class="hlFld-ContribAuthor ">Alessandro  Canella</span>, <span class="hlFld-ContribAuthor ">Courtney H.  Hodges</span>, <span class="hlFld-ContribAuthor ">Frauke  Christ</span>, <span class="hlFld-ContribAuthor ">Jan  De Rijck</span>, <span class="hlFld-ContribAuthor ">Pieter  Van Vlierberghe</span>, <span class="hlFld-ContribAuthor ">Václav  Veverka</span>, <span class="hlFld-ContribAuthor ">Zeger  Debyser</span>. </span><span class="cited-content_cbyCitation_article-title">Unlike its Paralog LEDGF/p75, HRP-2 Is Dispensable for MLL-R Leukemogenesis but Important for Leukemic Cell Survival. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2021,</strong> <em>10 </em>
                                    (1)
                                     , 192. <a href="https://doi.org/10.3390/cells10010192" title="DOI URL">https://doi.org/10.3390/cells10010192</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells10010192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells10010192%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DUnlike%252Bits%252BParalog%252BLEDGF%25252Fp75%25252C%252BHRP-2%252BIs%252BDispensable%252Bfor%252BMLL-R%252BLeukemogenesis%252Bbut%252BImportant%252Bfor%252BLeukemic%252BCell%252BSurvival%26aulast%3DVan%2BBelle%26aufirst%3DSiska%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D1%26spage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sha-Sha  Cheng</span>, <span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">The design and development of covalent protein-protein interaction inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00850-0" title="DOI URL">https://doi.org/10.1186/s13045-020-00850-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00850-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00850-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DThe%252Bdesign%252Band%252Bdevelopment%252Bof%252Bcovalent%252Bprotein-protein%252Binteraction%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DCheng%26aufirst%3DSha-Sha%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>204 </em>, 107406. <a href="https://doi.org/10.1016/j.pharmthera.2019.107406" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.107406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bepigenetic%252Bmachinery%25253A%252BEmerging%252Bnovel%252Ballosteric%252Binhibitors%26aulast%3DYe%26aufirst%3DFei%26date%3D2019%26volume%3D204%26spage%3D107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previously reported reversible and irreversible small-molecule menin inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Superposition of the co-crystal structure of menin in complex with menin binding motif 1 (MBM1 in yellow lines, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6">4GQ6</a>) from MLL onto the co-crystal structure of menin in complex with MIV-6 (magenta sticks) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>). Menin protein is shown in the gray surface. (B) Our proposed four modification sites in MIV-6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modifications of the cyclopentyl group of MIV-6 to access the P1 pocket, which was utilized by MBM1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6">4GQ6</a>) but not by MIV-6 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Co-crystal structure of M-89 bound to menin (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A">6E1A</a>) (left panel) and superposition of MIV-6 and M-89 based upon their co-crystal structures (right panel).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Stabilization of cellular menin protein by M-89 as assessed using the cellular thermal shift assay (CETSA) in MV4;11 cells. (B) Quantitative real-time polymerase chain reaction (qRT-PCR) analysis of the effect of M-89 on the mRNA levels of <i>Hoxa9</i> and <i>MEIS1</i> genes in MV4;11 cells. (C) Flow cytometry analysis of the effect of M-89 on apoptosis and cell differentiation in MV4;11 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacodynamic effect of M-89 on the expression of <i>Hoxa9</i> and <i>MEIS1</i> genes in the MV4;11 xenograft tissue in mice. Mice bearing MV4;11 xenograft tumors were dosed with M-89 for 3 days and tumors were harvested at 6, 24, and 48 h after the last administration for qRT-PCR analysis of <i>Hoxa9</i> and <i>MEIS1</i> gene expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6</b>–<b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) i. <b>43</b>, dichloromethane (DCM), dimethylformamide (DMF) (cat.), oxalyl chloride; ii. DCM, Et<sub>3</sub>N, <b>44</b>, 0 °C to room temperature (rt) overnight; (b) toluene, POCl<sub>3</sub>, P<sub>2</sub>O<sub>5</sub>, reflux; (c) tetrahydrofuran (THF), BF<sub>3</sub>·Et<sub>2</sub>O, cyclopentyl magnesium bromide; (d) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (e) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O (cat.), <b>49</b> or <b>50</b> or <b>51</b>, reflux; (f) DCM/AcOH (1:1), <i>N</i>-Boc-cyclic-amine-ketone or -aldehyde, NaBH(OAc)<sub>3</sub>; (g) DCM/trifluoracetic acid (TFA) (1:1) 15 min; (h) dimethyl sulfoxide (DMSO), K<sub>2</sub>CO<sub>3</sub>, <b>56</b>, 90 °C overnight.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>13</b>–<b>21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) For R = alkyl, <i>t</i>-BuONa, DMSO, R-Br, 90 °C overnight; for R = 4-bromopyridine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 90 °C overnight; (b) for <b>62</b>–<b>65</b>, DCM, <i>meta</i>-chloroperbenzoic acid (<i>m</i>CPBA), overnight; for <b>66</b> Me<sub>2</sub>CO/H<sub>2</sub>O (5:1), oxone, overnight; (c) DMSO, K<sub>2</sub>CO<sub>3</sub>, 4-F-Ph-SO<sub>2</sub>-R, 90 °C overnight.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Intermediates <b>78</b> and <b>79</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMSO, K<sub>2</sub>CO<sub>3</sub>, azetidin-3-ylmethanol·HCl, 80 °C, overnight; (b) DCM, Et<sub>3</sub>N, MsCl, 0 °C to rt, 30 min.</p></p></figure><figure data-id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>22</b>–<b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeOH, NaOMe, reflux; (b) MeOH, NaBH<sub>4</sub>; (c) i. THF, lithium bis(trimethylsilyl)amide (LiHMDS), −78 °C 30 min; ii. cyclopentyl bromide, −78 °C to rt; (d) toluene, DIBALH, 30 min, rt—work-up with aqueous (aq) NaOH; (e) MeOH, NaBH<sub>4</sub>; (f) DCM, Et<sub>3</sub>N, ClCO<sub>2</sub>Me, 0 °C, 1 h; (g) polyphosphoric acid (PPA), 150 °C, 1 h; (h) toluene, Red-Al, 30 min, rt; (i) DCM, Et<sub>3</sub>N, Boc<sub>2</sub>O, 0 °C to rt; (j) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (k) CH<sub>3</sub>CN, <b>78</b>, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, reflux; (l) DCM/CF<sub>3</sub>CO<sub>2</sub>H, 15 min; (m) CH<sub>3</sub>CN, alkyl-Br, KI·H<sub>2</sub>O, reflux or DCM/AcOH (1:1), alkyl aldehyde or ketone, NaBH(OAc)<sub>3</sub>.</p></p></figure><figure data-id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>31</b>–<b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeOH, Et<sub>3</sub>N, Boc<sub>2</sub>O; (b) THF, PPh<sub>3</sub>, diisopropylazodicarboxylate, −78 °C to rt overnight; (c) <b>83</b>, THF, LiHMDS, −78 °C for 30 min then <b>93</b>, −78 °C to rt overnight; (d) <b>83</b>, 18-crown-6, toluene, potassium bis(trimethylsilyl)amide, −78 °C for 30 min then <b>94</b>, −78 °C to rt overnight; (e) DCM/TFA, 30 min; (f) DCM, Et<sub>3</sub>N, Ac<sub>2</sub>O or ClCO<sub>2</sub>Me; (g) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (h) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, <b>79</b>, reflux; (i) NaH, DMF, alkyl iodide; (j) DCM, Et<sub>3</sub>N, alkyl anhydride or MeN═C═O; (k) DCM, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), Et<sub>3</sub>N, 4-Br-PhCO<sub>2</sub>H, 0 °C to rt.</p></p></figure><figure data-id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/medium/jm-2019-00021j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0012.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of M-89<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-13/acs.jmedchem.9b00021/20190705/images/large/jm-2019-00021j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00021&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF/toluene (1:1), NaH, BnBr, Bu<sub>4</sub>NI; (b) toluene, DIBALH; (c) DCM, Et<sub>3</sub>N, ClCO<sub>2</sub>Me, 0 °C to rt; (d) AcOH, CF<sub>3</sub>CO<sub>2</sub>H, paraformaldehyde; (e) toluene, Red-Al; (f) MeOH, Pd–C (cat.), H<sub>2</sub> (1 atm); (g) CH<sub>3</sub>CN, K<sub>2</sub>CO<sub>3</sub>, KI·H<sub>2</sub>O, <b>78</b>, reflux; (h) DCM, NaHCO<sub>3</sub>, Et<sub>3</sub>N, MeN═C═O, rt.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">The role of MLL in hematopoiesis and leukemia</span>. <i>Curr. Opin. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1097/00062752-200207000-00004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1097%2F00062752-200207000-00004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=12042701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A280%3ADC%252BD38zgtFSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=282-287&author=P.+Ernstauthor=J.+Wangauthor=S.+J.+Korsmeyer&title=The+role+of+MLL+in+hematopoiesis+and+leukemia&doi=10.1097%2F00062752-200207000-00004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of MLL in hematopoiesis and leukemia</span></div><div class="casAuthors">Ernst Patricia; Wang Jing; Korsmeyer Stanley J</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in hematology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-7</span>
        ISSN:<span class="NLM_cas:issn">1065-6251</span>.
    </div><div class="casAbstract">The MLL gene, also called HRX or ALL-1, was originally identified as a recurrent chromosomal translocation in particular subtypes of acute lymphocytic leukemia (ALL) and acute myelogenous leukemia (AML).  Reciprocal rearrangements of the MLL gene are most common in infant ALL and secondary AML.  Because of the unique association with infant leukemia and the intriguingly immature and mixed lineage phenotype of leukemic cells, the authors speculate that the wild-type MLL gene plays an important role early in the development of the hematopoietic system.  This article reviews recent progress in understanding the function of the wild-type MLL protein, with particular consideration of potential functions within the developing hematopoietic system.  Murine gain- and loss-of-function models have provided clues to the normal functions of MLL and altered functions of oncogenic MLL fusion proteins.  Biochemical and genetic approaches using other model organisms have also elucidated mechanisms by which these functions are achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFpKjX-b8FPVUIs6wFK5xJfW6udTcc2eYFKRJPeseJRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38zgtFSgtg%253D%253D&md5=2ac2403992ed9904cd89efd1abcbc470</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1097%2F00062752-200207000-00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00062752-200207000-00004%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520role%2520of%2520MLL%2520in%2520hematopoiesis%2520and%2520leukemia%26jtitle%3DCurr.%2520Opin.%2520Hematol.%26date%3D2002%26volume%3D9%26spage%3D282%26epage%3D287%26doi%3D10.1097%2F00062752-200207000-00004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of leukemogenesis by MLL fusion proteins</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2141.2010.08459.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1111%2Fj.1365-2141.2010.08459.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=21118195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=141-154&author=R.+Marschalek&title=Mechanisms+of+leukemogenesis+by+MLL+fusion+proteins&doi=10.1111%2Fj.1365-2141.2010.08459.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of leukemogenesis by MLL fusion proteins</span></div><div class="casAuthors">Marschalek, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-154</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Infant acute leukemia is characterised by specific genetic rearrangements and a rapid onset of disease shortly after birth.  The vast majority of these cases bear rearranged MLL alleles.  However, many facets of MLL-rearranged leukemia are largely unknown.  Basically, there exists a fundamental and evolutionary conserved relationship between the family of MLL/Trithorax proteins and the regulation of HOX gene clusters.  Therefore, direct MLL fusion proteins are per se able to deregulate HOX genes, except when reciprocal MLL fusion proteins come into play.  This reviews discusses (i) the current situation in MLL-rearranged leukemia, (ii) the mol. and genetic tools to functionally investigate the many different MLL fusions, (iii) the latency of disease development, (iv) a novel cancer mechanism that has been recently uncovered when different MLL fusion protein complexes were characterized, (v) mutated signalling pathways in MLL-rearranged leukemia and (vi) presents new ideas on how a given MLL fusion protein may modulate existing signalling pathways in leukemic cells.  The hypothesis is posed that the many different fusion partners of MLL are critically distinct entities for which specific inhibitors should be identified in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeSBrUEMt1JLVg90H21EOLACvtfcHk0ljRra2z_OfpVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D&md5=f0330065cb216625a242a2508d5d1c93</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2010.08459.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2010.08459.x%26sid%3Dliteratum%253Aachs%26aulast%3DMarschalek%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520leukemogenesis%2520by%2520MLL%2520fusion%2520proteins%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2011%26volume%3D152%26spage%3D141%26epage%3D154%26doi%3D10.1111%2Fj.1365-2141.2010.08459.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span> <span> </span><span class="NLM_article-title">MLL: How complex does it get?</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1002/jcb.20430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fjcb.20430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15779005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2005&pages=234-242&author=R.+Popovicauthor=N.+J.+Zeleznik-Le&title=MLL%3A+How+complex+does+it+get%3F&doi=10.1002%2Fjcb.20430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: How complex does it get?</span></div><div class="casAuthors">Popovic, Relja; Zeleznik-Le, Nancy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">234-242</span>CODEN:
                <span class="NLM_cas:coden">JCEBD5</span>;
        ISSN:<span class="NLM_cas:issn">0730-2312</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The mixed lineage leukemia (MLL) gene encodes a very large nuclear protein homologous to Drosophila trithorax (trx).  MLL is required for the proper maintenance of HOX gene expression during development and hematopoiesis.  The exact regulatory mechanism of HOX gene expression by MLL is poorly understood, but it is believed that MLL functions at the level of chromatin organization.  MLL was identified as a common target of chromosomal translocations assocd. with human acute leukemias.  About 50 different MLL fusion partners have been isolated to date, and while similarities exist between groups of partners, there exists no unifying property shared by all the partners.  MLL gene rearrangements are found in leukemias with both lymphoid and myeloid phenotypes and are often assocd. with infant and secondary leukemias.  The immature phenotype of the leukemic blasts suggests an important role for MLL in the early stages of hematopoietic development.  Mll homozygous mutant mice are embryonic lethal and exhibit deficiencies in yolk sac hematopoiesis.  Recently, two different MLL-contg. protein complexes have been isolated.  These and other gain- and loss-of-function expts. have provided insight into normal MLL function and altered functions of MLL fusion proteins.  This article reviews the progress made toward understanding the function of the wild-type MLL protein.  While many advances in understanding this multifaceted protein have been made since its discovery, many challenging questions remain to be answered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1zjCdPpVE_7Vg90H21EOLACvtfcHk0ljRra2z_OfpVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFKjs7o%253D&md5=78d94e50c640366d568bb0876a91d916</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fjcb.20430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.20430%26sid%3Dliteratum%253Aachs%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26atitle%3DMLL%253A%2520How%2520complex%2520does%2520it%2520get%253F%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2005%26volume%3D95%26spage%3D234%26epage%3D242%26doi%3D10.1002%2Fjcb.20430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">When epigenetics kills: MLL fusion proteins in leukemia</span>. <i>Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1002/hon.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fhon.739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16118769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlamsrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=1-9&author=R.+K.+Slany&title=When+epigenetics+kills%3A+MLL+fusion+proteins+in+leukemia&doi=10.1002%2Fhon.739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">When epigenetics kills: MLL fusion proteins in leukemia</span></div><div class="casAuthors">Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Hematological Oncology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">HAONDL</span>;
        ISSN:<span class="NLM_cas:issn">0278-0232</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review.  Chromosomal aberrations that affect the MLL (Mixed Lineage Leukemia) gene at the locus 11q23 are assocd. with an aggressive subtype of leukemia.  These alterations create MLL fusion derivs. with an active transforming potential.  This review summarizes recent advances in our knowledge about normal and malignant MLL proteins with special emphasis on epigenetic processes affected by these mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon2L1pw0c0HLVg90H21EOLACvtfcHk0lhGOiGjlM70hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlamsrrP&md5=ebe7728f67a381c6108f7f9125bc2796</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fhon.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhon.739%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DWhen%2520epigenetics%2520kills%253A%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DHematol.%2520Oncol.%26date%3D2005%26volume%3D23%26spage%3D1%26epage%3D9%26doi%3D10.1002%2Fhon.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomizawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinukawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isoyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horibe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakahata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, E.</span></span> <span> </span><span class="NLM_article-title">Outcome of risk-based therapy for infant acute lymphoblastic leukemia with or without an MLL gene rearrangement, with emphasis on late effects: a final report of two consecutive studies, MLL96 and MLL98, of the Japan infant leukemia study group</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2258</span>– <span class="NLM_lpage">2263</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2404903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fsj.leu.2404903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17690691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1WgsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2258-2263&author=D.+Tomizawaauthor=K.+Kohauthor=T.+Satoauthor=N.+Kinukawaauthor=A.+Morimotoauthor=K.+Isoyamaauthor=Y.+Kosakaauthor=T.+Odaauthor=M.+Odaauthor=Y.+Hayashiauthor=M.+Eguchiauthor=K.+Horibeauthor=T.+Nakahataauthor=S.+Mizutaniauthor=E.+Ishii&title=Outcome+of+risk-based+therapy+for+infant+acute+lymphoblastic+leukemia+with+or+without+an+MLL+gene+rearrangement%2C+with+emphasis+on+late+effects%3A+a+final+report+of+two+consecutive+studies%2C+MLL96+and+MLL98%2C+of+the+Japan+infant+leukemia+study+group&doi=10.1038%2Fsj.leu.2404903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Outcome of risk-based therapy for infant acute lymphoblastic leukemia with or without an MLL gene rearrangement, with emphasis on late effects: a final report of two consecutive studies, MLL96 and MLL98, of the Japan Infant Leukemia Study Group</span></div><div class="casAuthors">Tomizawa, D.; Koh, K.; Sato, T.; Kinukawa, N.; Morimoto, A.; Isoyama, K.; Kosaka, Y.; Oda, T.; Oda, M.; Hayashi, Y.; Eguchi, M.; Horibe, K.; Nakahata, T.; Mizutani, S.; Ishii, E.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2258-2263</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We evaluated the efficacy of a treatment strategy in which infants with acute lymphoblastic leukemia (ALL) were stratified by their MLL gene status and then assigned to different risk-based therapies.  A total of 102 patients were registered on two consecutive multicenter trials, designated MLL96 and MLL98, between 1995 and 2001.  Those with a rearranged MLL gene (MLL-R, n=80) were assigned to receive intensive chemotherapy followed by hematopoietic stem cell transplantation (HSCT), while those with germline MLL (MLL-G, n=22) were treated with chemotherapy alone.  The 5-yr event-free survival (EFS) rate for all 102 infants was 50.9% (95% confidence interval, 41.0-60.8%).  The most prominent late effect was growth impairment, obsd. in 58.9% of all evaluable patients in the MLL-R group.  This plan of risk-based therapy appears to have improved the overall prognosis for infants with ALL, compared with previously reported results.  However, over half the events in patients with MLL rearrangement occurred before the instigation of HSCT, and that HSCT-related toxic events comprised 36.3% (8/22) of post-transplantation events, suggesting that further stratification within the MLL-R group and the development of more effective early-phase intensification chemotherapy will be needed before the full potential of this strategy is realized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI1I1wHxFGWLVg90H21EOLACvtfcHk0lhGOiGjlM70hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1WgsbnI&md5=4f513655b925c27a4aaffd9c8e236cdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2404903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2404903%26sid%3Dliteratum%253Aachs%26aulast%3DTomizawa%26aufirst%3DD.%26aulast%3DKoh%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DKinukawa%26aufirst%3DN.%26aulast%3DMorimoto%26aufirst%3DA.%26aulast%3DIsoyama%26aufirst%3DK.%26aulast%3DKosaka%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DT.%26aulast%3DOda%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DEguchi%26aufirst%3DM.%26aulast%3DHoribe%26aufirst%3DK.%26aulast%3DNakahata%26aufirst%3DT.%26aulast%3DMizutani%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DE.%26atitle%3DOutcome%2520of%2520risk-based%2520therapy%2520for%2520infant%2520acute%2520lymphoblastic%2520leukemia%2520with%2520or%2520without%2520an%2520MLL%2520gene%2520rearrangement%252C%2520with%2520emphasis%2520on%2520late%2520effects%253A%2520a%2520final%2520report%2520of%2520two%2520consecutive%2520studies%252C%2520MLL96%2520and%2520MLL98%252C%2520of%2520the%2520Japan%2520infant%2520leukemia%2520study%2520group%26jtitle%3DLeukemia%26date%3D2007%26volume%3D21%26spage%3D2258%26epage%3D2263%26doi%3D10.1038%2Fsj.leu.2404903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panetta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo-Coco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Poeta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venditti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurillo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Principe, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauriello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anemona, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palombo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadori, S.</span></span> <span> </span><span class="NLM_article-title">Chromosomal aberration of the 11q23 locus in acute leukemia and frequency of MLL gene translocation: results in 378 adult patients</span>. <i>Am. J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1309/RX27R8GJQM330C22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1309%2FRX27R8GJQM330C22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15323147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvF2htLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2004&pages=298-306&author=M.+C.+Coxauthor=P.+Panettaauthor=F.+Lo-Cocoauthor=G.+Del+Poetaauthor=A.+Vendittiauthor=L.+Maurilloauthor=M.+I.+Del+Principeauthor=A.+Maurielloauthor=L.+Anemonaauthor=A.+Brunoauthor=C.+Mazzoneauthor=P.+Palomboauthor=S.+Amadori&title=Chromosomal+aberration+of+the+11q23+locus+in+acute+leukemia+and+frequency+of+MLL+gene+translocation%3A+results+in+378+adult+patients&doi=10.1309%2FRX27R8GJQM330C22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chromosomal aberration of the 11q23 locus in acute leukemia and frequency of MLL gene translocation: Results in 378 adult patients</span></div><div class="casAuthors">Cox, M. Christina; Panetta, Paola; Lo-Coco, Francesco; del Poeta, Giovanni; Venditti, Adriano; Maurillo, Luca; del Principe, M. Ilaria; Mauriello, Alessandro; Anemona, Lucia; Bruno, Antonio; Mazzone, Carla; Palombo, Paolo; Amadori, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-306</span>CODEN:
                <span class="NLM_cas:coden">AJCPAI</span>;
        ISSN:<span class="NLM_cas:issn">0002-9173</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Pathologists</span>)
        </div><div class="casAbstract">Structural abnormality of the 11q23 band (11q23+) bearing the MLL gene translocation (MLL+) is a recurrent chromosome change obsd. in 3% to 7% of acute lymphoblastic leukemias and in 3% to 4% of acute myeloblastic leukemias.  The resoln. of conventional cytogenetics (CC) in detecting 11q23 rearrangement is limited when the translocative partner has a telomeric location; furthermore, CC can barely discriminate between true 11q23+/MLL+ and rearrangements clustering within the 11q22∼25 region without MLL involvement (MLL-).  We characterized a series of 378 consecutive patients with adult acute leukemia by CC, fluorescence in situ hybridization (FISH), and multiplex karyotyping (M-FISH) anal.  Our aim was to define the frequency of cryptic MLL+ cases and the frequency of MLL+ within 11q22∼25+ cases.  As expected, FISH was more sensitive than CC in detecting MLL+ cases, but rather unexpectedly, 9 (45%) of 20 patients with 11q22∼25+ were MLL-.  A better characterization of 11q22∼25+/MLL- leukemias is relevant for the identification of new, recurrent translocations.  Moreover, these cases should be readily distinguishable from 11q23+/MLL+ cases.  We recommend that karyotypic anal. always be complemented by mol. or FISH methods to unravel MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz-09gQjw1BLVg90H21EOLACvtfcHk0lgtndk0VROGxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvF2htLo%253D&md5=eff8f0b8bd29e6b7dedb7b428ce15bbf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1309%2FRX27R8GJQM330C22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1309%252FRX27R8GJQM330C22%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DM.%2BC.%26aulast%3DPanetta%26aufirst%3DP.%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DDel%2BPoeta%26aufirst%3DG.%26aulast%3DVenditti%26aufirst%3DA.%26aulast%3DMaurillo%26aufirst%3DL.%26aulast%3DDel%2BPrincipe%26aufirst%3DM.%2BI.%26aulast%3DMauriello%26aufirst%3DA.%26aulast%3DAnemona%26aufirst%3DL.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DMazzone%26aufirst%3DC.%26aulast%3DPalombo%26aufirst%3DP.%26aulast%3DAmadori%26aufirst%3DS.%26atitle%3DChromosomal%2520aberration%2520of%2520the%252011q23%2520locus%2520in%2520acute%2520leukemia%2520and%2520frequency%2520of%2520MLL%2520gene%2520translocation%253A%2520results%2520in%2520378%2520adult%2520patients%26jtitle%3DAm.%2520J.%2520Clin.%2520Pathol.%26date%3D2004%26volume%3D122%26spage%3D298%26epage%3D306%26doi%3D10.1309%2FRX27R8GJQM330C22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimartino, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Mll rearrangements in haematological malignancies: lessons from clinical and biological studies</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2141.1999.01439.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1046%2Fj.1365-2141.1999.01439.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10468849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1999&pages=614-626&author=J.+F.+Dimartinoauthor=M.+L.+Cleary&title=Mll+rearrangements+in+haematological+malignancies%3A+lessons+from+clinical+and+biological+studies&doi=10.1046%2Fj.1365-2141.1999.01439.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MLL rearrangements in haematological malignancies: Lessons from clinical and biological studies</span></div><div class="casAuthors">DiMartino, Jorge F.; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 96 refs., covering the MLL family of 11q23 chromosomal rearrangements, assocn. between 11q23 rearrangements and leukemia phenotype, MLL rearrangements in the absence of leukemia, prognostic implications of 11q23 rearrangements, etiol. of MLL rearrangements, MLL rearrangements leading to leukemia with short latency, structure and probable function of the MLL protein, and possible mechanisms of leukemogenesis by MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOiR8FH8-xQbVg90H21EOLACvtfcHk0lgtndk0VROGxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D&md5=845845da43dc70b59322261edb88bb6e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1999.01439.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1999.01439.x%26sid%3Dliteratum%253Aachs%26aulast%3DDimartino%26aufirst%3DJ.%2BF.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMll%2520rearrangements%2520in%2520haematological%2520malignancies%253A%2520lessons%2520from%2520clinical%2520and%2520biological%2520studies%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D1999%26volume%3D106%26spage%3D614%26epage%3D626%26doi%3D10.1046%2Fj.1365-2141.1999.01439.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">The molecular biology of mixed lineage leukemia</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">984</span>– <span class="NLM_lpage">993</span>, <span class="refDoi"> DOI: 10.3324/haematol.2008.002436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.3324%2Fhaematol.2008.002436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=19535349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=984-993&author=R.+K.+Slany&title=The+molecular+biology+of+mixed+lineage+leukemia&doi=10.3324%2Fhaematol.2008.002436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular biology of mixed lineage leukemia</span></div><div class="casAuthors">Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">984-993</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">0390-6078</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia is a very aggressive blood cancer that predominantly occurs in pediatric patients.  In contrast to other types of childhood acute leukemias, mixed lineage leukemia presents with a dismal prognosis and despite the availability of advanced treatment methods cure rates have stagnated over the last years.  Mixed lineage leukemia is characterized by the presence of MLL fusion proteins that are the result of chromosomal translocations affecting the MLL gene at 11q23.  These events juxtapose the amino-terminus of the histone methyltransferase MLL with a variety of different fusion partners that destroy normal histone methyltransferase function of MLL and replace it by heterologous functions contributed by the fusion partner.  The resulting chimeras are transcriptional regulators that take control of targets normally controlled by MLL with the clustered HOX homeobox genes as prominent examples.  Recent studies suggested that MLL fusion partners activate transcription by two different mechanisms.  Some of these proteins are themselves chromatin modifiers that introduce histone acetylation whereas other fusion partners can recruit histone methyltransferases.  In particular, histone H3 specific methylation at lysine 79 catalyzed by DOT1L has been recognized as a hallmark of chromatin activated by MLL fusion proteins.  Interestingly several frequent MLL fusion partners seem to coordinate DOT1L activity with a protein complex that stimulates the elongation phase of transcription by phosphorylating the carboxy-terminal repeat domain of RNA polymerase II.  The discovery of these novel enzymic activities that are essentially involved in MLL fusion protein function presents potential new targets for a rational drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpudla5d7qUgrVg90H21EOLACvtfcHk0lgv83hBMwsZvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGitrrK&md5=103284cb0a582febe6e14e8d12afc4f1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2008.002436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2008.002436%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520molecular%2520biology%2520of%2520mixed%2520lineage%2520leukemia%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D984%26epage%3D993%26doi%3D10.3324%2Fhaematol.2008.002436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span> <span> </span><span class="NLM_article-title">Disordered epigenetic regulation in MLL-related leukemia</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1007/s12185-012-1180-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1007%2Fs12185-012-1180-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=23054645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFamtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2012&pages=428-437&author=Y.+Zhangauthor=A.+Chenauthor=X.+M.+Yanauthor=G.+Huang&title=Disordered+epigenetic+regulation+in+MLL-related+leukemia&doi=10.1007%2Fs12185-012-1180-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered epigenetic regulation in MLL-related leukemia</span></div><div class="casAuthors">Zhang, Yue; Chen, Aili; Yan, Xiao-Mei; Huang, Gang</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Hematology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">428-437</span>CODEN:
                <span class="NLM_cas:coden">IJHEEY</span>;
        ISSN:<span class="NLM_cas:issn">0925-5710</span>.
    
            (<span class="NLM_cas:orgname">Springer Japan</span>)
        </div><div class="casAbstract">A review.  Leukemias bearing rearrangements of chromosome 11q23 are of particular interest due to their unique clin. and biol. characteristics. 11q23 abnormalities occur in up to 70 % of infant leukemias, and about 10 % of adult acute myelogenous leukemias (AML).  Two major rearrangements of the MLL gene are found in MLL-related leukemia.  The most common of these is balanced translocations in which the N-terminal portion of MLL is fused to the C-terminus of the translocation partner.  To date, nearly 100 different chromosome bands have been described in rearrangements involving MLL, and more than 70 known fusion partners of MLL have been cloned and characterized at the mol. level.  Another major aberration of the MLL gene creates a repeat within the N-terminal MLL resulting in an internal partial tandem duplication (PTD).  As a consequence, an extra amino-terminus is added in-frame to full-length MLL, resulting in leukemogenic MLL-PTD.  MLL-PTD occurs predominantly in myeloid dysplasia syndromes, secondary AML (s-AML), and de novo AML.  The presence of an MLL rearrangement generally confers a poor prognosis.  MLL fusions and MLL-PTD are transcriptional regulators that take control of targets normally controlled by MLL, with the clustered HOX homeobox genes as prominent examples.  Several epigenetic regulators that modify DNA or histones have been implicated in MLL fusion driven leukemogenesis, including DNA methylation, histone acetylation, and histone methylation.  Recently, the histone methyltransferase DOT1L, the bromodomain and extra-terminal (BET) family member BRD4, and the MLL-interacting protein Menin have emerged as important mediators of MLL fusion-mediated leukemic transformation.  The clin. development of targeted inhibitors of these epigenetic regulators has heralded promise for the treatment of MLL fusion leukemia.  Although the biol. function and mol. mechanism for MLL-PTD remains largely unknown, based on the primary protein structure of MLL-PTD and the knowledge gained so far from MLL fusions, newly developed inhibitors of epigenetic regulators could potentially also prove effective in the treatment of MLL-PTD related leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCNNdsRmcXjLVg90H21EOLACvtfcHk0lgv83hBMwsZvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFamtbbK&md5=9b8ad8df35b4a098f34e343800cc4d0c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs12185-012-1180-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-012-1180-0%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DX.%2BM.%26aulast%3DHuang%26aufirst%3DG.%26atitle%3DDisordered%2520epigenetic%2520regulation%2520in%2520MLL-related%2520leukemia%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2012%26volume%3D96%26spage%3D428%26epage%3D437%26doi%3D10.1007%2Fs12185-012-1180-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">MLL: a histone methyltransferase disrupted in leukemia</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2004.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.molmed.2004.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15464450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+a+histone+methyltransferase+disrupted+in+leukemia&doi=10.1016%2Fj.molmed.2004.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: a histone methyltransferase disrupted in leukemia</span></div><div class="casAuthors">Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">500-507</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rearrangements of the MLL gene, which is located at chromosome 11q23, are assocd. with aggressive acute leukemias in both children and adults.  MLL regulates Hox gene expression through direct promoter binding and histone modification.  MLL rearrangements occurring in leukemia include MLL fusion genes, partial tandem duplications of MLL and MLL amplification.  MLL fusions and amplification upregulate Hox expression, apparently resulting in a block of hematopoietic differentiation.  Future therapies for MLL-assocd. leukemia might involve blocking Hox gene upregulation by using fusion proteins or inhibiting the activity of Hox proteins themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2pQrErnZf7Vg90H21EOLACvtfcHk0lhTZ06ezQD6GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D&md5=ae9d59e5e59cc017fd0fcf479a4a618d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2004.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2004.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520a%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507%26doi%3D10.1016%2Fj.molmed.2004.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0lhTZ06ezQD6GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caslini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Osta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of MLL amino terminal sequences with menin is required for transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7283</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1158%2F0008-5472.CAN-06-2369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=17671196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVemtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=7275-7283&author=C.+Casliniauthor=Z.+Yangauthor=M.+El-Ostaauthor=T.+A.+Milneauthor=R.+K.+Slanyauthor=J.+L.+Hess&title=Interaction+of+MLL+amino+terminal+sequences+with+menin+is+required+for+transformation&doi=10.1158%2F0008-5472.CAN-06-2369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of MLL Amino Terminal Sequences with Menin Is Required for Transformation</span></div><div class="casAuthors">Caslini, Corrado; Yang, Zhaohai; El-Osta, Mohamad; Milne, Thomas A.; Slany, Robert K.; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7275-7283</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Rearrangements of the mixed lineage leukemia gene MLL are assocd. with aggressive lymphoid and myeloid leukemias.  The resulting MLL fusion proteins enforce high-level expression of HOX genes and the HOX cofactor MEIS1, which is pivotal for leukemogenesis.  Both wild-type MLL and MLL fusion proteins interact with the tumor suppressor menin and with the Hoxa9 locus in vivo.  Here, we show that MLL sequences between amino acids 5 and 44 are required for interaction with menin and for the transformation of hematopoietic progenitors.  Blocking the MLL-menin interaction by the expression of a dominant neg. inhibitor composed of amino terminal MLL sequences down-regulates Meis1 expression and inhibits cell proliferation, suggesting that targeting this interaction may be an effective therapeutic strategy for leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4C8hcS0mav7Vg90H21EOLACvtfcHk0lhTZ06ezQD6GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVemtrc%253D&md5=df8f9ffb41c080aae65fe9722cdf11ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2369%26sid%3Dliteratum%253Aachs%26aulast%3DCaslini%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DEl-Osta%26aufirst%3DM.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520MLL%2520amino%2520terminal%2520sequences%2520with%2520menin%2520is%2520required%2520for%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D7275%26epage%3D7283%26doi%3D10.1158%2F0008-5472.CAN-06-2369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ccr.2008.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=18598942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=36-46&author=A.+Yokoyamaauthor=M.+L.+Cleary&title=Menin+critically+links+MLL+proteins+with+LEDGF+on+cancer-associated+target+genes&doi=10.1016%2Fj.ccr.2008.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span></div><div class="casAuthors">Yokoyama, Akihiko; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-46</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Menin displays the unique ability to either promote oncogenic function in the hematopoietic lineage or suppress tumorigenesis in the endocrine lineage; however, its mol. mechanism of action has not been defined.  We demonstrate here that these discordant functions are unified by menin's ability to serve as a mol. adaptor that phys. links the MLL (mixed-lineage leukemia) histone methyltransferase with LEDGF (lens epithelium-derived growth factor), a chromatin-assocd. protein previously implicated in leukemia, autoimmunity, and HIV-1 pathogenesis.  LEDGF is required for both MLL-dependent transcription and leukemic transformation.  Conversely, a subset of menin mutations in multiple endocrine neoplasia type 1 patients abrogate interaction with LEDGF while preserving MLL interaction but nevertheless compromise MLL/menin-dependent functions.  Thus, LEDGF critically assocs. with MLL and menin at the nexus of transcriptional pathways that are recurrently targeted in diverse diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjaPq2DDJ-sLVg90H21EOLACvtfcHk0lhg_NA2WapIUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekt7Y%253D&md5=3eb5242d529ec6c8222908a6116547f7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMenin%2520critically%2520links%2520MLL%2520proteins%2520with%2520LEDGF%2520on%2520cancer-associated%2520target%2520genes%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26spage%3D36%26epage%3D46%26doi%3D10.1016%2Fj.ccr.2008.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16239140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span></div><div class="casAuthors">Yokoyama, Akihiko; Somervaille, Tim C. P.; Smith, Kevin S.; Rozenblatt-Rosen, Orit; Meyerson, Matthew; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed-Lineage Leukemia (MLL) protein is a histone methyltransferase that is mutated in clin. and biol. distinctive subsets of acute leukemia.  MLL normally assocs. with a cohort of highly conserved cofactors to form a macromol. complex that includes menin, a product of the MEN1 tumor suppressor gene, which is mutated in heritable and sporadic endocrine tumors.  We demonstrate here that oncogenic MLL fusion proteins retain an ability to stably assoc. with menin through a high-affinity, amino-terminal, conserved binding motif and that this interaction is required for the initiation of MLL-mediated leukemogenesis.  Furthermore, menin is essential for maintenance of MLL-assocd. but not other oncogene induced myeloid transformation.  Acute genetic ablation of menin reverses aberrant Hox gene expression mediated by MLL-menin promoter-assocd. complexes, and specifically abrogates the differentiation arrest and oncogenic properties of MLL-transformed leukemic blasts.  These results demonstrate that a human oncoprotein is critically dependent on direct phys. interaction with a tumor suppressor protein for its oncogenic activity, validate a potential target for mol. therapy, and suggest central roles for menin in altered epigenetic functions underlying the pathogenesis of hematopoietic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MXmqsWp8BrVg90H21EOLACvtfcHk0lhg_NA2WapIUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP&md5=d582f614f98d25ebaf2282e2915fd4b6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeshan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pear, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, X.</span></span> <span> </span><span class="NLM_article-title">The tumor suppressor menin regulates hematopoiesis and myeloid transformation by influencing Hox gene expression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1023</span>, <span class="refDoi"> DOI: 10.1073/pnas.0510347103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1073%2Fpnas.0510347103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=16415155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=1018-1023&author=Y.+X.+Chenauthor=J.+Yanauthor=K.+Keeshanauthor=A.+T.+Tubbsauthor=H.+Wangauthor=A.+Silvaauthor=E.+J.+Brownauthor=J.+L.+Hessauthor=W.+S.+Pearauthor=X.+Hua&title=The+tumor+suppressor+menin+regulates+hematopoiesis+and+myeloid+transformation+by+influencing+Hox+gene+expression&doi=10.1073%2Fpnas.0510347103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The tumor suppressor menin regulates hematopoiesis and myeloid transformation by influencing Hox gene expression</span></div><div class="casAuthors">Chen, Ya-Xiong; Yan, Jizhou; Keeshan, Karen; Tubbs, Anthony T.; Wang, Haoren; Silva, Albert; Brown, Eric J.; Hess, Jay L.; Pear, Warren S.; Hua, Xianxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1018-1023</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Menin is the product of the tumor suppressor gene Men1 that is mutated in the inherited tumor syndrome multiple endocrine neoplasia type 1 (MEN1).  Menin has been shown to interact with SET-1 domain-contg. histone 3 lysine 4 (H3K4) methyltransferases including mixed lineage leukemia proteins to regulate homeobox (Hox) gene expression in vitro.  Using conditional Men1 knockout mice, we have investigated the requirement for menin in hematopoiesis and myeloid transformation.  Men1 excision causes redn. of Hoxa9 expression, colony formation by hematopoietic progenitors, and the peripheral white blood cell count.  Menin directly activates Hoxa9 expression, at least in part, by binding to the Hoxa9 locus, facilitating methylation of H3K4, and recruiting the methylated H3K4 binding protein chd1 to the locus.  Consistent with signaling downstream of menin, ectopic expression of both Hoxa9 and Meis1 rescues colony formation defects in Men1-excised bone marrow.  Moreover, Men1 excision also suppresses proliferation of leukemogenic mixed lineage leukemia-AF9 fusion-protein-transformed myeloid cells and Hoxa9 expression.  These studies uncover an important role for menin in both normal hematopoiesis and myeloid transformation and provide a mechanistic understanding of menin's function in these processes that may be used for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrba146rYH-WrVg90H21EOLACvtfcHk0lg39l1WJj5RUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKgtb4%253D&md5=c6f88d7cab1d4e755ff2e27ecc88bf83</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0510347103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0510347103%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DKeeshan%26aufirst%3DK.%26aulast%3DTubbs%26aufirst%3DA.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DSilva%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DE.%2BJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DPear%26aufirst%3DW.%2BS.%26aulast%3DHua%26aufirst%3DX.%26atitle%3DThe%2520tumor%2520suppressor%2520menin%2520regulates%2520hematopoiesis%2520and%2520myeloid%2520transformation%2520by%2520influencing%2520Hox%2520gene%2520expression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D1018%26epage%3D1023%26doi%3D10.1073%2Fpnas.0510347103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the menin-MLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin-MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0lg39l1WJj5RUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin-MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurung, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veniaminova, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">The same pocket in menin binds both MLL and JUND but has opposite effects on transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1038/nature10806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnature10806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22327296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XitFalt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=542-546&author=J.+Huangauthor=B.+Gurungauthor=B.+Wanauthor=S.+Matkarauthor=N.+A.+Veniaminovaauthor=K.+Wanauthor=J.+L.+Merchantauthor=X.+Huaauthor=M.+Lei&title=The+same+pocket+in+menin+binds+both+MLL+and+JUND+but+has+opposite+effects+on+transcription&doi=10.1038%2Fnature10806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The same pocket in menin binds both MLL and JUND but has opposite effects on transcription</span></div><div class="casAuthors">Huang, Jing; Gurung, Buddha; Wan, Bingbing; Matkar, Smita; Veniaminova, Natalia A.; Wan, Ke; Merchant, Juanita L.; Hua, Xianxin; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Menin is a tumor suppressor protein whose loss or inactivation causes multiple endocrine neoplasia 1 (MEN1), a hereditary autosomal dominant tumor syndrome that is characterized by tumorigenesis in multiple endocrine organs.  Menin interacts with many proteins and is involved in a variety of cellular processes.  Menin binds the JUN family transcription factor JUND and inhibits its transcriptional activity.  Several MEN1 missense mutations disrupt the menin-JUND interaction, suggesting a correlation between the tumor-suppressor function of menin and its suppression of JUND-activated transcription.  Menin also interacts with mixed lineage leukemia protein 1 (MLL1), a histone H3 lysine 4 methyltransferase, and functions as an oncogenic cofactor to upregulate gene transcription and promote MLL1-fusion-protein-induced leukemogenesis.  A recent report on the tethering of MLL1 to chromatin binding factor lens epithelium-derived growth factor (LEDGF) by menin indicates that menin is a mol. adaptor coordinating the functions of multiple proteins.  Despite its importance, how menin interacts with many distinct partners and regulates their functions remains poorly understood.  Here we present the crystal structures of human menin in its free form and in complexes with MLL1 or with JUND, or with an MLL1-LEDGF heterodimer.  These structures show that menin contains a deep pocket that binds short peptides of MLL1 or JUND in the same manner, but that it can have opposite effects on transcription.  The menin-JUND interaction blocks JUN N-terminal kinase (JNK)-mediated JUND phosphorylation and suppresses JUND-induced transcription.  In contrast, menin promotes gene transcription by binding the transcription activator MLL1 through the peptide pocket while still interacting with the chromatin-anchoring protein LEDGF at a distinct surface formed by both menin and MLL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCm7SaZ0LyW7Vg90H21EOLACvtfcHk0lg39l1WJj5RUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitFalt7k%253D&md5=95c952303b917ed0c5e2d475926941d7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature10806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10806%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DGurung%26aufirst%3DB.%26aulast%3DWan%26aufirst%3DB.%26aulast%3DMatkar%26aufirst%3DS.%26aulast%3DVeniaminova%26aufirst%3DN.%2BA.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DMerchant%26aufirst%3DJ.%2BL.%26aulast%3DHua%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DThe%2520same%2520pocket%2520in%2520menin%2520binds%2520both%2520MLL%2520and%2520JUND%2520but%2520has%2520opposite%2520effects%2520on%2520transcription%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D542%26epage%3D546%26doi%3D10.1038%2Fnature10806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+B.+Shiauthor=M.+J.+Muraiauthor=S.+H.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+menin-MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0lis50PoZx-_Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%2BB.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%2BH.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520menin-MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0lis50PoZx-_Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Small-Molecule Inhibitors of the Menin-Mixed Lineage Leukemia (MLL) Interaction Closely Mimic a Natural Protein-Protein Interaction</span></div><div class="casAuthors">He, Shihan; Senter, Timothy J.; Pollock, Jonathan; Han, Changho; Upadhyay, Sunil Kumar; Purohit, Trupta; Gogliotti, Rocco D.; Lindsley, Craig W.; Cierpicki, Tomasz; Stauffer, Shaun R.; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1543-1556</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction (PPI) between menin and mixed lineage leukemia (MLL) plays a crit. role in acute leukemias, and inhibition of this interaction represents a new potential therapeutic strategy for MLL leukemias.  We report development of a novel class of small-mol. inhibitors of the menin-MLL interaction, the hydroxy- and aminomethylpiperidine compds., which originated from HTS of ∼288000 small mols.  We detd. menin-inhibitor co-crystal structures and found that these compds. closely mimic all key interactions of MLL with menin.  Extensive crystallog. studies combined with structure-based design were applied for optimization of these compds., resulting in MIV-6R, which inhibits the menin-MLL interaction with IC50 = 56 nM.  Treatment with MIV-6 demonstrated strong and selective effects in MLL leukemia cells, validating specific mechanism of action.  Our studies provide novel and attractive scaffold as a new potential therapeutic approach for MLL leukemias and demonstrate an example of PPI amenable to inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC4yHqMSn87Vg90H21EOLACvtfcHk0lis50PoZx-_Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D&md5=a580d1b2ea3f657c4abb31678f39548c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and mixed lineage leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+mixed+lineage+leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Property Focused Structure-Based Optimization of Small Molecule Inhibitors of the Protein-Protein Interaction between Menin and Mixed Lineage Leukemia (MLL)</span></div><div class="casAuthors">Borkin, Dmitry; Pollock, Jonathan; Kempinska, Katarzyna; Purohit, Trupta; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Miao, Hongzhi; Shukla, Shirish; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of potent small mol. inhibitors of protein-protein interactions with optimized druglike properties represents a challenging task in lead optimization process.  Here, we report synthesis and structure-based optimization of new thienopyrimidine class of compds., which block the protein-protein interaction between menin and MLL fusion proteins that plays an important role in acute leukemias with MLL translocations.  We performed simultaneous optimization of both activity and druglike properties through systematic exploration of substituents introduced to the indole ring of lead compd. 1 (MI-136) to identify compds. suitable for in vivo studies in mice.  This work resulted in the identification of compd. 27 (MI-538), which showed significantly increased activity, selectivity, polarity, and pharmacokinetic profile over 1 and demonstrated a pronounced effect in a mouse model of MLL leukemia.  This study, which reports detailed structure-activity and structure-property relationships for the menin-MLL inhibitors, demonstrates challenges in optimizing inhibitors of protein-protein interactions for potential therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDD5t7XKKz_LVg90H21EOLACvtfcHk0lh5yYsXR5QvIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D&md5=f89b6eda72671f133a38181b3b23632d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linhares, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Complexity of blocking bivalent protein-protein interactions: Development of a highly potent inhibitor of the menin-mixed-lineage leukemia interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4832</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4832-4850&author=D.+Borkinauthor=S.+Klossowskiauthor=J.+Pollockauthor=H.+Miaoauthor=B.+M.+Linharesauthor=K.+Kempinskaauthor=Z.+Jinauthor=T.+Purohitauthor=B.+Wenauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Complexity+of+blocking+bivalent+protein-protein+interactions%3A+Development+of+a+highly+potent+inhibitor+of+the+menin-mixed-lineage+leukemia+interaction&doi=10.1021%2Facs.jmedchem.8b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Complexity of Blocking Bivalent Protein-Protein Interactions: Development of a Highly Potent Inhibitor of the Menin-Mixed-Lineage Leukemia Interaction</span></div><div class="casAuthors">Borkin, Dmitry; Klossowski, Szymon; Pollock, Jonathan; Miao, Hongzhi; Linhares, Brian M.; Kempinska, Katarzyna; Jin, Zhuang; Purohit, Trupta; Wen, Bo; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4832-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors.  Here, we report the development of a new generation of menin-MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compds.  This work resulted in compd. I (MI-1481), which showed very potent inhibition of the menin-MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin-MLL1 inhibitor reported to date.  The crystal structure of the menin-I complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of I.  Compd. I also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models.  Thus, I is a valuable menin-MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_eephZ2UHnLVg90H21EOLACvtfcHk0lh5yYsXR5QvIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D&md5=8d12fa816af3a5aa191694d4bfe48802</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00071%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DLinhares%26aufirst%3DB.%2BM.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DComplexity%2520of%2520blocking%2520bivalent%2520protein-protein%2520interactions%253A%2520Development%2520of%2520a%2520highly%2520potent%2520inhibitor%2520of%2520the%2520menin-mixed-lineage%2520leukemia%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4832%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=25817203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Blocks Progression of MLL Leukemia In Vivo</span></div><div class="casAuthors">Borkin, Dmitry; He, Shihan; Miao, Hongzhi; Kempinska, Katarzyna; Pollock, Jonathan; Chase, Jennifer; Purohit, Trupta; Malik, Bhavna; Zhao, Ting; Wang, Jingya; Wen, Bo; Zong, Hongliang; Jones, Morgan; Danet-Desnoyers, Gwenn; Guzman, Monica L.; Talpaz, Moshe; Bixby, Dale L.; Sun, Duxin; Hess, Jay L.; Muntean, Andrew G.; Maillard, Ivan; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-602</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal translocations affecting mixed lineage leukemia gene (MLL) result in acute leukemias resistant to therapy.  The leukemogenic activity of MLL fusion proteins is dependent on their interaction with menin, providing basis for therapeutic intervention.  Here we report the development of highly potent and orally bioavailable small-mol. inhibitors of the menin-MLL interaction, MI-463 and MI-503, and show their profound effects in MLL leukemia cells and substantial survival benefit in mouse models of MLL leukemia.  Finally, we demonstrate the efficacy of these compds. in primary samples derived from MLL leukemia patients.  Overall, we demonstrate that pharmacol. inhibition of the menin-MLL interaction represents an effective treatment for MLL leukemias in vivo and provide advanced mol. scaffold for clin. lead identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc2hESfSmR8bVg90H21EOLACvtfcHk0lh5yYsXR5QvIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D&md5=1369b1556d139e9006c3fa7d1add49a0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors block oncogenic transformation by MLL-fusion proteins in a fusion partner-independent manner</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1038/leu.2015.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fleu.2015.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=26084867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wqsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=508-513&author=S.+Heauthor=B.+Malikauthor=D.+Borkinauthor=H.+Miaoauthor=S.+Shuklaauthor=K.+Kempinskaauthor=T.+Purohitauthor=J.+Wangauthor=L.+Chenauthor=B.+Parkinauthor=S.+N.+Malekauthor=G.+Danet-Desnoyersauthor=A.+G.+Munteanauthor=T.+Cierpickiauthor=J.+Grembecka&title=Menin-MLL+inhibitors+block+oncogenic+transformation+by+MLL-fusion+proteins+in+a+fusion+partner-independent+manner&doi=10.1038%2Fleu.2015.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors block oncogenic transformation by MLL-fusion proteins in a fusion partner-independent manner</span></div><div class="casAuthors">He, S.; Malik, B.; Borkin, D.; Miao, H.; Shukla, S.; Kempinska, K.; Purohit, T.; Wang, J.; Chen, L.; Parkin, B.; Malek, S. N.; Danet-Desnoyers, G.; Muntean, A. G.; Cierpicki, T.; Grembecka, J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">508-513</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genome-wide anal. of gene expression revealed that MI-2-2 reverses MLL-rearranged gene signature and induced expression of genes assocd. with differentiation in cells harboring various MLL-fusion proteins.  Taken together, the menin MLL inhibitor is capable of reversing oncogenic activity of different MLL fusions according to a mechanism that is independent of the fusion partner.  Such activity remains to be in agreement with menin binding motif retained in all MLL-fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5a_fPNgF5cLVg90H21EOLACvtfcHk0lgMYawKC-P9yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wqsr7L&md5=2a46a4568d58ddb2fde5727ab348c710</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.144%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DParkin%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%2BN.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DMenin-MLL%2520inhibitors%2520block%2520oncogenic%2520transformation%2520by%2520MLL-fusion%2520proteins%2520in%2520a%2520fusion%2520partner-independent%2520manner%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D508%26epage%3D513%26doi%3D10.1038%2Fleu.2015.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the Menin-MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fernandez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+Menin-MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0lhSF976t1sJww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520Menin-MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+H.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lhSF976t1sJww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%2BH.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez%0AMolina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0lhSF976t1sJww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of high-affinity macrocyclic peptidomimetics to block the menin-mixed lineage leukemia 1 (MLL1) protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1021/jm3015298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3015298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCqs7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1113-1123&author=H.+Zhouauthor=L.+Liuauthor=J.+Huangauthor=D.+Bernardauthor=H.+Karatasauthor=A.+Navarroauthor=M.+Leiauthor=S.+Wang&title=Structure-based+design+of+high-affinity+macrocyclic+peptidomimetics+to+block+the+menin-mixed+lineage+leukemia+1+%28MLL1%29+protein-protein+interaction&doi=10.1021%2Fjm3015298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of High-Affinity Macrocyclic Peptidomimetics to Block the Menin-Mixed Lineage Leukemia 1 (MLL1) Protein-Protein Interaction</span></div><div class="casAuthors">Zhou, Haibin; Liu, Liu; Huang, Jing; Bernard, Denzil; Karatas, Hacer; Navarro, Alexandro; Lei, Ming; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1113-1123</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Menin is an essential oncogenic cofactor for mixed lineage leukemia 1 (MLL1)-mediated leukemogenesis through its direct interaction with MLL1.  Targeting the menin-MLL1 protein-protein interaction represents a promising strategy to block MLL1-mediated leukemogenesis.  Employing a structure-based approach and starting from a linear MLL1 octapeptide, we have designed a class of potent macrocyclic peptidomimetic inhibitors of the menin-MLL1 interaction.  The most potent macrocyclic peptidomimetic (MCP-1), 34 (I), binds to menin with a Ki value of 4.7 nM and is >600 times more potent than the corresponding acyclic peptide.  Compd. 34 is also less peptide-like and has a lower mol. wt. than the initial MLL1 peptide.  Therefore, compd. 34 serves as a promising lead structure for the design of potent and cell-permeable inhibitors of the menin-MLL1 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMcRYRG7I0y7Vg90H21EOLACvtfcHk0lhh2r53LzOokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCqs7rE&md5=df004832c2fad317611851268a3d574e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm3015298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3015298%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DNavarro%26aufirst%3DA.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520high-affinity%2520macrocyclic%2520peptidomimetics%2520to%2520block%2520the%2520menin-mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1113%26epage%3D1123%26doi%3D10.1021%2Fjm3015298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lhh2r53LzOokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhh2r53LzOokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lhGVM6qtSvFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhGVM6qtSvFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.11.2; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DRoversi%26aufirst%3DP.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ6','PDB','4GQ6'); return false;">PDB: 4GQ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8','PDB','4OG8'); return false;">PDB: 4OG8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A','PDB','6E1A'); return false;">PDB: 6E1A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U84" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U84','PDB','3U84'); return false;">PDB: 3U84</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i97"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00021">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_00785"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00021">10.1021/acs.jmedchem.9b00021</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Single-crystal structure of compound <b>103</b> and a table of crystal data and structure refinement of <b>103</b>; electron density maps of the co-crystal structure of M-89 complexed with menin and a table of crystallographic data collection and refinement statistics for the menin/M-89 co-crystal (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf">PDF</a>)</p></li><li><p class="inline">CSV file containing SMILES, binding and cellular data for final target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_001.pdf">jm9b00021_si_001.pdf (975.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00021/suppl_file/jm9b00021_si_002.csv">jm9b00021_si_002.csv (4.38 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for M-89 complexed with menin has been deposited with the Protein Data Bank with the PDB ID code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E1A">6E1A</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00021&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-13%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00021%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00021" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bee53fc83d1e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
