
<html lang="en"     class="pb-page"  data-request-id="fb188c38-457d-413c-9daa-0e9d42f7f0b0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-2;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.6b01781"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="A “Click Chemistry Platform” for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation" /></meta><meta name="dc.Creator" content="Ryan P.  Wurz" /></meta><meta name="dc.Creator" content="Ken  Dellamaggiore" /></meta><meta name="dc.Creator" content="Hannah  Dou" /></meta><meta name="dc.Creator" content="Noelle  Javier" /></meta><meta name="dc.Creator" content="Mei-Chu  Lo" /></meta><meta name="dc.Creator" content="John D.  McCarter" /></meta><meta name="dc.Creator" content="Dane  Mohl" /></meta><meta name="dc.Creator" content="Christine  Sastri" /></meta><meta name="dc.Creator" content="J. Russell  Lipford" /></meta><meta name="dc.Creator" content="Victor J.  Cee" /></meta><meta name="dc.Description" content="Proteolysis targeting chimeras (PROTACs) are bispecific molecules containing a target protein binder and an ubiquitin ligase binder connected by a linker. By recruiting an ubiquitin ligase to a tar..." /></meta><meta name="Description" content="Proteolysis targeting chimeras (PROTACs) are bispecific molecules containing a target protein binder and an ubiquitin ligase binder connected by a linker. By recruiting an ubiquitin ligase to a tar..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 17, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01781" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01781" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01781" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01781" /></link>
        
    
    

<title>A “Click Chemistry Platform” for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01781" /></meta><meta property="og:title" content="A “Click Chemistry Platform” for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0012.jpeg" /></meta><meta property="og:description" content="Proteolysis targeting chimeras (PROTACs) are bispecific molecules containing a target protein binder and an ubiquitin ligase binder connected by a linker. By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degradation of the target protein. The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker composition, all of which have been difficult to address in a systematic way. Herein, we describe a “click chemistry” approach for the synthesis of PROTACs. We demonstrate the utility of this approach with the bromodomain and extraterminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel–Lindau (VHL) proteins. An AlphaScreen proximity assay was used to determine the ability of PROTACs to form the ternary ligase–PROTAC–target protein complex and a MSD assay to measure cellular degradation of the target protein promoted by PROTACs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01781"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01781">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01781&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01781&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01781&amp;href=/doi/10.1021/acs.jmedchem.6b01781" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 453-461</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01272" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01816" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A “Click Chemistry Platform” for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ryan+P.++Wurz">Ryan P. Wurz</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1413-5208" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ken++Dellamaggiore">Ken Dellamaggiore</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hannah++Dou">Hannah Dou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Noelle++Javier">Noelle Javier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mei-Chu++Lo">Mei-Chu Lo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+D.++McCarter">John D. McCarter</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dane++Mohl">Dane Mohl</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Sastri">Christine Sastri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Russell++Lipford">J. Russell Lipford</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Victor+J.++Cee">Victor J. Cee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Departments of Therapeutic Discovery−Medicinal Chemistry, <sup>‡</sup>Oncology Research and <sup>§</sup>Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Therapeutic Discovery−Discovery Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 805-313-5400. E-mail: <a href="/cdn-cgi/l/email-protection#394b4e4c4b437958545e5c57175a5654"><span class="__cf_email__" data-cfemail="41333634333b01202c26242f6f222e2c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01781&amp;href=/doi/10.1021%2Facs.jmedchem.6b01781" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 453–461</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 5, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 December 2016</li><li><span class="item_label"><b>Published</b> online</span>17 April 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01781" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01781</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D453%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRyan%2BP.%2BWurz%252C%2BKen%2BDellamaggiore%252C%2BHannah%2BDou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D2%26contentID%3Dacs.jmedchem.6b01781%26title%3DA%2B%25E2%2580%259CClick%2BChemistry%2BPlatform%25E2%2580%259D%2Bfor%2Bthe%2BRapid%2BSynthesis%2Bof%2BBispecific%2BMolecules%2Bfor%2BInducing%2BProtein%2BDegradation%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D461%26publicationDate%3DJanuary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01781"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">12241</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">64</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01781" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A “Click Chemistry Platform” for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ryan&quot;,&quot;last_name&quot;:&quot;P. Wurz&quot;},{&quot;first_name&quot;:&quot;Ken&quot;,&quot;last_name&quot;:&quot;Dellamaggiore&quot;},{&quot;first_name&quot;:&quot;Hannah&quot;,&quot;last_name&quot;:&quot;Dou&quot;},{&quot;first_name&quot;:&quot;Noelle&quot;,&quot;last_name&quot;:&quot;Javier&quot;},{&quot;first_name&quot;:&quot;Mei-Chu&quot;,&quot;last_name&quot;:&quot;Lo&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;D. McCarter&quot;},{&quot;first_name&quot;:&quot;Dane&quot;,&quot;last_name&quot;:&quot;Mohl&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Sastri&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Russell Lipford&quot;},{&quot;first_name&quot;:&quot;Victor&quot;,&quot;last_name&quot;:&quot;J. Cee&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;453-461&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01781&quot;},&quot;abstract&quot;:&quot;Proteolysis targeting chimeras (PROTACs) are bispecific molecules containing a target protein binder and an ubiquitin ligase binder connected by a linker. By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degradation of the target protein. The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker composition, all of which have been difficult to address in a systematic way. Herein, we describe a “click chemistry” approach for the synthesis of PROTACs. We demonstrate the utility of this approach with the bromodomain and extraterminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel–Lindau (VHL) proteins. An AlphaScreen proximity assay was used to determine the ability of PROTACs to form the ternary ligase–PROTAC–target protein complex and a MSD assay to measure cellular degradation of the target protein promoted by PROTACs.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01781&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01781" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01781&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01781" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01781&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01781" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01781&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01781&amp;href=/doi/10.1021/acs.jmedchem.6b01781" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01781" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01781" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01781%26sid%3Dliteratum%253Aachs%26pmid%3D28378579%26genre%3Darticle%26aulast%3DWurz%26date%3D2018%26atitle%3DA%2B%25E2%2580%259CClick%2BChemistry%2BPlatform%25E2%2580%259D%2Bfor%2Bthe%2BRapid%2BSynthesis%2Bof%2BBispecific%2BMolecules%2Bfor%2BInducing%2BProtein%2BDegradation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D2%26spage%3D453%26epage%3D461%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/jmcmar.2018.61.issue-2/20180125/jmcmar.2018.61.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Proteolysis targeting chimeras (PROTACs) are bispecific molecules containing a target protein binder and an ubiquitin ligase binder connected by a linker. By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degradation of the target protein. The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker composition, all of which have been difficult to address in a systematic way. Herein, we describe a “click chemistry” approach for the synthesis of PROTACs. We demonstrate the utility of this approach with the bromodomain and extraterminal domain-4 (BRD4) ligand JQ-1 (<b>3</b>) and ligase binders targeting cereblon (CRBN) and Von Hippel–Lindau (VHL) proteins. An AlphaScreen proximity assay was used to determine the ability of PROTACs to form the ternary ligase–PROTAC–target protein complex and a MSD assay to measure cellular degradation of the target protein promoted by PROTACs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/61/2"><issue-title>Inducing Protein Degradation as a Therapeutic Strategy</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The development of bispecific molecules or proteolysis targeting chimeras (PROTACs)<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> as potential drugs capable of recruiting target proteins to the cellular machinery for elimination has created alternative options to tackle traditionally “difficult to drug” proteins. This approach “hijacks” the function of a ligase to transfer ubiquitin subunits to the target protein, thereby tagging it for recognition by the proteasome and coding its destruction.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> A variety of protein types have been successfully degraded using this approach, including transcription factors,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> kinases<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5-8)</a> and nuclear epigenetic readers<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8-11)</a> using a variety of ligases,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> including Von Hippel–Lindau (VHL),<a onclick="showRef(event, 'ref4 ref5 ref9 ref13'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref9 ref13">(4, 5, 9, 13)</a> mouse double minute 2 homologue (MDM2),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> F-box/WD repeat-containing protein (β-TrCP),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> cereblon,<a onclick="showRef(event, 'ref6 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref6 ref8 ref9 ref10">(6, 8-10)</a> and inhibitor of apoptosis protein (IAP).<a onclick="showRef(event, 'ref4 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref6 ref7">(4, 6, 7)</a></div><div class="NLM_p">Arguably one of the more potent examples of PROTACs in the literature to date involves the degradation of the BET bromodomain protein BRD4. Of the PROTACs employing BRD4 ligand JQ-1 (<b>3</b>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> MZ-1 (<b>1</b>),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and dBET1 (<b>2</b>)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> have successfully used both VHL and cereblon E3 ligase ligands, respectively, for BRD4 protein degradation and serve as excellent benchmarks for the development of novel PROTACs (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PROTACs for the degradation of BET bromodomain BRD4 protein derived from ligand <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A comprehensive survey of the targeted proteolysis literature for commonality of linker composition and length yielded little insight into the ideal linker and suggests a degree of empiricism in PROTAC design. Furthermore, the ligase ligand may have a dramatic influence on activity of the PROTAC<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> depending on the cell line because E3 ligase expression levels may vary. Given these complexities to PROTAC design and that a comprehensive study of ligase ligand/linker length activity relationship with a BRD4 targeting PROTAC has not been forthcoming in the literature, we wanted to advance our own understanding of the SAR of these bispecific molecules.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> To facilitate the discovery process for new PROTACs, a reliable linking strategy was sought to couple ligase ligand-containing motifs and target protein ligands in a parallel manner, which led us to the Huisgen 1,3-dipolar cycloaddition reaction. Arguably, this bond forming reaction is one of the most reliable. This copper-catalyzed reaction couples azides and alkynes forming a new triazole ring and is commonly referred to as “click chemistry” due to its ease of use.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> This reaction is typically high yielding, requiring stoichiometric quantities of each component and boasts excellent functional group compatibility under mild reaction conditions. This approach would allow for the parallel synthesis of libraries of PROTACs provided the necessary azides and alkynes could be prepared. To test this approach, ligands for VHL and cereblon were modified with terminal alkynes with linkers containing varying ethylene glycol units. A ligand for the target protein would have to be modified with an azide at a suitable position on the molecule so as to minimally influence affinity of the ligand for its target protein and serve as the azide input for the “click reaction” (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. General strategy using “click chemistry” for parallel PROTAC synthesis. POI = protein of interest.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For a proof-of-concept for this “click chemistry” platform, we decided to focus on the synthesis of PROTACs targeting the BET bromodomain, BRD4. <b>3</b> was used as the targeting ligand and converted into an amide containing a pendant azide necessary for the “click reaction”. This was accomplished by TFA-mediated deprotection of the <i>tert</i>-butyl ester of <b>3</b> followed by amide formation with 2-azidoethanamine, furnishing azide <b>4</b> in high yield (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>).<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0001.gif" alt="" id="GRAPHIC-d49e383-autogenerated" /></img></span></div><div class="NLM_p">For the ligase portion of the “click reaction”, ligands for cereblon and VHL ligases were modified with terminal alkyne motifs containing ethylene glycol units of varying lengths. Thalidomide modified with an ethereal linkage at the 4-position has resulted in potent PROTACs in previous literature reports, such as <b>2</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and served as a useful starting point for this investigation. These ligands for cereblon could be synthesized in an expeditious manner from 4-hydroxyphthalic anhydride (<b>5</b>) following treatment with 3-amino-piperidine-2,6-dione (<b>6</b>) to afford 4-hydroxy-thalidomide (<b>7</b>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Alkylation of the phenol with either propargyl bromide or propargyl-tosylates containing varying ethylene glycol units afforded the requisite library of cereblon ligase ligands (<b>8a</b>–<b>e</b>, <b>IMiD</b><b>0</b>–<b>4</b>) necessary for the “click reaction” (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a><span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0002.gif" alt="" id="GRAPHIC-d49e427-autogenerated" /></img></span></div><div class="NLM_p">In a similar manner, a library of VHL ligands modified with terminal alkynes containing varying ethylene glycol units were prepared. To access this library, VHL-amine (<b>9</b>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> was treated with the corresponding propargyl-glycolic-acid chlorides to generate the library of VHL ligands (<b>10a</b>–<b>e</b>, <b>VHL</b><b>0–4</b>) for the “click reaction” (<a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>).<span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0003.gif" alt="" id="GRAPHIC-d49e455-autogenerated" /></img></span></div><div class="NLM_p">With both azide and alkyne components in hand, we then subjected them to typical conditions for a click reaction (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). This involved CuSO<sub>4</sub> as a catalyst, sodium ascorbate for catalyst regeneration, and 0.1 M in THF with 3–4 drops of water to aid in solubilizing the copper catalyst. The reactions were typically very clean and often went to completion within 3 h, resulting in 55–90% isolated yields for the 10-membered PROTAC library (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. PROTAC Synthesis Using the “Click Reaction”<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">PROTAC</th><th class="colsep0 rowsep0" align="center" char=".">yield (%)</th><th class="colsep0 rowsep0" align="center">PROTAC</th><th class="colsep0 rowsep0" align="center" char=".">yield (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="char" char=".">79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="char" char=".">60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="left"><b>12c</b></td><td class="colsep0 rowsep0" align="char" char=".">72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="char" char=".">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11e</b></td><td class="colsep0 rowsep0" align="char" char=".">69</td><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="char" char=".">55</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Isolated yields following purification by silica gel chromatography.</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26451" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26451" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To evaluate the activity of the 10 new PROTACs, a cell-free proximity assay utilizing AlphaScreen technology (amplified luminescent proximity homogeneous assay, PerkinElmer, Waltham, MA)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> was developed to determine if the bispecific molecules could induce ternary complex formation between the E3 ligase, PROTAC, and BRD4 protein. The proximity assay uses Glutathione Donor beads to bind the glutathione-<i>S</i>-transferase (GST)-tagged BRD4 protein and Nickel Chelate AlphaScreen Acceptor beads to capture the poly-His-tagged E3 ligase and reports the luminescence arising from proximity of E3 ligase-bound acceptor beads and BRD4-bound donor beads through ternary complex formation. To determine cellular degradation of BRD4, we used the Meso Scale Diagnostics (MSD, Rockville, MD) platform and a commercial detection antibody to quantify BRD4 content in cellular lysates. The PROTACs’ activity was assessed in NCI-H661 cells<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> at a 4 h time point. The benchmark BRD4 PROTACs <b>1</b> and <b>2</b> were used as positive controls for these studies and for points of comparison (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). The data generated from these two assays represent the first systematic study of the effect of linker length on the activity of the PROTACs for the cereblon and VHL ligases in a given series.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Proximity Assay Results and BRD4 Degradation in a NCI-H661 Cell Line for Cereblon/BRD4 and VHL/BRD4 PROTACs</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">proximity assay<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> POC<sub>max</sub></th><th class="colsep0 rowsep0" align="center">protein degradation<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> H661 cells (4 h) DC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cLogP<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">solubility PBS/FaSIF/0.01 N HCl (μM)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="center">0<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">>5</td><td class="colsep0 rowsep0" align="char" char=".">4.82</td><td class="colsep0 rowsep0" align="center">146/434/281</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="center">100<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">0.14</td><td class="colsep0 rowsep0" align="char" char=".">2.49</td><td class="colsep0 rowsep0" align="center">60/97/171</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="center">116<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">0.49</td><td class="colsep0 rowsep0" align="char" char=".">2.87</td><td class="colsep0 rowsep0" align="center">31/191/171</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="center">36<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">>5</td><td class="colsep0 rowsep0" align="char" char=".">2.74</td><td class="colsep0 rowsep0" align="center">111/247/242</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="center">118<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">>5</td><td class="colsep0 rowsep0" align="char" char=".">2.60</td><td class="colsep0 rowsep0" align="center">76/237/237</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="center">113<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">0.45</td><td class="colsep0 rowsep0" align="char" char=".">2.46</td><td class="colsep0 rowsep0" align="center">130/216/310</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11e</b></td><td class="colsep0 rowsep0" align="center">148<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">0.20</td><td class="colsep0 rowsep0" align="char" char=".">2.33</td><td class="colsep0 rowsep0" align="center">158/273/317</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="center">100<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.008</td><td class="colsep0 rowsep0" align="char" char=".">4.90</td><td class="colsep0 rowsep0" align="center">97/164/450</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="center">46<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.059</td><td class="colsep0 rowsep0" align="char" char=".">4.29</td><td class="colsep0 rowsep0" align="center">0/90/314</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="center">22<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.083</td><td class="colsep0 rowsep0" align="char" char=".">4.17</td><td class="colsep0 rowsep0" align="center">38/178/343</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12c</b></td><td class="colsep0 rowsep0" align="center">11<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.24</td><td class="colsep0 rowsep0" align="char" char=".">4.04</td><td class="colsep0 rowsep0" align="center">71/183/293</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="center">15<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.46</td><td class="colsep0 rowsep0" align="char" char=".">3.90</td><td class="colsep0 rowsep0" align="center">168/294/394</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="center">3<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.63</td><td class="colsep0 rowsep0" align="char" char=".">3.77</td><td class="colsep0 rowsep0" align="center">206/313/500</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Data represents an average of at least 2 separate determinations. DC<sub>50</sub>, concentration of a PROTAC at which the cellular protein content is reduced by half.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Log <i>P</i> values were calculated via a Pipeline Pilot Web Port interface (Biovia, Inc.) to the Daylight software suite (version 4.81, Daylight Chemical Information Systems, Inc.).</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last"><i>n</i> = 1 for pH solubility data, PBS = phosphate-buffered saline, FaSIF = fasted state simulated intestinal fluid.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">Benchmark <b>2</b> was assigned PO<i>C</i><sub>max</sub> = 100 in the proximity assay for IMiD containing PROTACs.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">Benchmark <b>1</b> was assigned PO<i>C</i><sub>max</sub> = 100 in the proximity assay for VHL containing PROTACs.</p></div></div></div><div class="NLM_p">In the cereblon/PROTAC/BRD4 proximity assay, the IMiD-derived PROTACs showed a bell-shaped dose–response curve (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) consistent with the three-body binding equilibria in which excess bridging molecules out-compete ternary complex formation.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> We used the maximal normalized signal (POC<sub>max</sub>) to compare the ability of PROTACs to induce ternary complex formation because POC<sub>max</sub> is directly related to the maximal amount of ternary complex that can be formed and it has better resolution than other parameters such as the concentration of the half-maximal response. <b>2</b> was used as the positive control, and its POC<sub>max</sub> was set to be 100. Most of the IMiD-derived PROTACs were able to induce ternary complex formation to a similar extent as the positive control with the exception of <b>11b</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). In the MSD assay, the PROTAC with the longest linker studied (<b>11e</b>) was found to be the most active, resulting in a DC<sub>50</sub> value of 0.20 μM which was very similar to <b>2</b> with a DC<sub>50</sub> value of 0.14 μM (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The poor cellular activity of <b>11b</b> and <b>11c</b> relative to shorter (<b>11a</b>) and longer (<b>11d</b>, <b>11e</b>) linker lengths was an unexpected result, and this highlights the importance of investigating linker length in the optimization of PROTACs. Forming a ternary cereblon/PROTAC/BRD4 complex is a prerequisite for the subsequent ubiquitin transfer and proteasomal degradation of BRD4, but it is not the only factor that determines the level of cellular protein degradation. Other factors such as optimal geometries may play a key role in the kinetics of ubiquitin transfer and therefore cause a loose correlation between POC<sub>max</sub> and DC<sub>50</sub> from the two assays as observed for these CRBN/BRD4 PROTACs. The cLogP values and solubilities of the IMiD PROTACs (<b>11a</b>–<b>e</b>) prepared using click chemistry were similar to the benchmark IMiD PROTAC <b>2</b>, and this deviation in physical properties presumably had little impact on their activity versus the benchmark.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Proximity assay data for JQ-1-IMiD PROTACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MSD assay data for JQ-1-IMiD PROTACs in a NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For the VHL-derived PROTACs, <b>1</b> was used as the positive control (POC<sub>max</sub> = 100) for the proximity assay, and the PROTACs resulting from the “click ligation” showed moderate-to-low complex formation relative to <b>1</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). In this case, the presence of the triazole motif in the linker resulted in a small decline in solubility and a small decrease in cLogP when compared to the benchmark VHL PROTAC (<b>1</b>). Nevertheless, the activity of some of the VHL/BRD4 PROTACs were measurable in the BRD4 protein assay, with low double-digit nanomolar activities favoring the PROTACs with the shortest linker lengths (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). These PROTACs were still ca. 10-fold less active than the benchmark <b>1</b>, suggesting that the triazole moiety negatively impacts the activity because <b>12a</b> and <b>12b</b> have roughly the same linker length as <b>1</b>. This was not observed with the CRBN/BRD4 PROTACs. Therefore, the platform approach in this case was successful in identifying VHL/BRD4 as a suitable protein/ligase pair and establishing that shorter linker lengths produce more potent PROTACs, but additional linker composition SAR may still be necessary to achieve maximum potency in a series. There was a modest correlation between the POC<sub>max</sub> in the proximity assay and the DC<sub>50</sub> in the BRD4 MSD assay. The VHL-derived PROTACs showed much greater differences in ternary complex formation as measured by POC<sub>max</sub> than their IMiD-derived counterparts. Experiments involving in vitro reconstitution of ligase complexes and BRD4 will be necessary to fully understand these observations.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proximity assay data for JQ-1-VHL PROTACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MSD assay data for JQ-1-VHL (<b>12a</b>–<b>e</b>) PROTACs in a NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To confirm that the BRD4 protein disappearance was driven by ternary complex formation between cereblon/PROTAC/BRD4 protein and was proteasome-mediated, a series of control experiments were performed with the NCI-H661 cell line. Dosing of this cell line alone with the PROTAC <b>11e</b> at 1 μM resulted in a >90% reduction in BRD4 protein after 4 h versus a DMSO control, whereas cotreatment with 10 μM <b>3</b>, 10 μM thalidomide, or a proteasome inhibitor (20 μM MG132)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> all blocked the <b>11e</b>-induced degradation of BRD4 consistent with ternary complex formation and that loss of BRD4 protein is proteasome mediated. We also show that 10 μM of the azide/alkyne components individually or codosed (<b>4</b> and <b>8e</b>) resulted in no appreciable degradation of the BRD4 protein, indicating that the nonlinked components of the PROTAC are not sufficient for degradation (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). A similar set of experiments was run to confirm that the BRD4 protein disappearance was driven by ternary complex formation between VHL/PROTAC/BRD4 protein and was also proteasome-mediated (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). In this case, cotreatment with VHL ligand (<b>13</b>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> resulted in blockage of <b>12b</b>-induced degradation of BRD4.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Control experiments: (A) <b>11e</b> PROTAC in the NCI-H661 cell line (4 h time point); (B) <b>12b</b> PROTAC in NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In conclusion, we have described the use of a “click chemistry” platform to prepare a 10-membered library of PROTACs using both cereblon and VHL ligase ligands and evaluated these molecules in their ability to form ternary complexes between the ligase/PROTAC/BRD4 protein as well as their activity in the targeted degradation of BRD4 protein in two lung carcinoma cell lines. Their mechanism of activity is supported by a number of control experiments that suggest the necessity for ternary complex formation and that the process is proteasome mediated. These examples build upon the earlier study by Lebraud and co-workers demonstrating the utility of click chemistry for the synthesis of cell penetrant PROTACs.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p last">This report represents the first comprehensive study of linker length/activity relationship as well as the relationship between protein degradation and the ability to form strong ternary complexes as measured by a proximity assay. Conceptually, this platform represents a powerful new tool to access libraries of PROTACs as its principles can be easily applied to other ligases and target proteins, the results of which will be reported in due course.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biology</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> BRD4 MSD Assay</h4><div class="NLM_p last">NCI-H661 (ATCC, 2.5 × 10<sup>4</sup> cells/well) or NCI-H838 (ATCC, 4.0 × 10<sup>4</sup> cells/well) were seeded in 96-well flat-bottom plates (Corning 3904). Cells were incubated overnight at 37 °C then dosed for 4 h with compound dose responses. Cells were lysed in MSD lysis buffer (R60TX-2) containing protease inhibitors (Roche 04 693 116001) and phosphatase inhibitors (Roche 04 906 837001). BRD4 protein levels were determined by MSD following the standard bind, spot-coating MSD protocol (<a href="https://www.mesoscale.com/~/media/files/technical%20notes/assay%20development%20plates%20uncoated.pdf" class="extLink">https://www.mesoscale.com/~/media/files/technical%20notes/assay%20development%20plates%20uncoated.pdf</a>). BRD4 capture antibody (Cell Signaling no. 13440) was diluted 1:50 into 0.03% Triton X-100 in PBS. BRD4 detection antibody (Atlas no. AMAb90841) was diluted 1:500 into 0.6% blocker A (MSD R93BA-4) in 1× wash buffer.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> CRBN-DDB1 Protein Expression</h4><div class="NLM_p last">Two expression vectors were constructed with either full-length human CRBN with a thrombin-cleavable N-terminal histidine tag or full-length DDB1 cloned behind the polyhedron promotor of pORB1 (Orbigin INC). Virus stocks of each were prepared in Sf9 insect cells using the BestBac 2.0 cotransfection kit according to the manufacturer’s instructions (Expression Systems LLC) and titered by Expression Systems. Recombinant heterodimer was prepared using cotransfection of HisCRBN and DDB1 in 10 L WAVE bioreactor culture of Sf9 cells at 2E6/mL. Cells were infected using HisCRBN and DDB1 virus stocks at an MOI of 1:4, respectively. Zinc acetate was added to the culture to a final concentration of 50 μM. The culture was harvested after 48 h, and cells were pelleted at 4000<i>g</i>. The pelleted cells were resuspended in 50 mM Tris pH 7.4, 500 mM NaCl, 7.5 mM imidazole, 10% glycerol with Roche Complete EDTA-free proteinase cocktail and lysed by microfluidizer. The lysate was clarified by centrifugation at 10000<i>g</i> and filtration through a 1 μm serum filter (Pall). The lysate was bound at 5 mL/min to a HisTrap excel 5 mL column. The column was washed with 80 CVs of resuspension buffer and eluted with 10 CVs of an elution buffer of 50 mM Tris pH 7.4, 500 mM NaCl, 500 mM imidazole, and 10% glycerol. Elution fractions containing heterodimer were concentrated and purified by sizing on a HiLoad 26/600 Superdex 200 pg in a buffer of PBS 1 mM DTT. HisCRBN DDB1 eluted as a single peak and yielded approximately 1 mg per liter of culture.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> CRBN-DDB1/BRD4 and VHL/BRD4 Proximity Assays</h4><div class="NLM_p last">To detect ternary complex formation induced by PROTACs, we used AlphaScreen technology to measure luminescence arising from proximity of CRBN- or VHL-bound acceptor beads and BRD4-bound donor beads. The proximity assay was performed in a 384-well white OptiPlate (PerkinElmer) in a total volume of 30 μL. For CRBN-DDB1/BRD4 proximity assay, the reaction mixture contained 60 nM His-tagged CRBN-DDB1, 60 nM GST-tagged BRD4 (49–170) (Sigma-Aldrich), 20 μg/mL Nickel Chelate AlphaScreen Acceptor, and Glutathione Donor beads (PerkinElmer) and serially diluted test compounds in the binding buffer of 50 mM HEPES, pH 7.4, 200 mM NaCl, 1 mM TCEP, and 0.1% BSA. 10 μL of 3× test compounds (in 3% DMSO) were preincubated with 10 μL of 3× CRBN-DDB1/BRD4 mixture at rt for 60 min. Then 10 μL of 3× Nickel Chelate AlphaScreen Acceptor and Glutathione Donor beads were added and assay plates were further incubated at rt for 60 min prior to luminescence detection on an Envision 2102 plate reader (PerkinElmer). VHL/BRD4 proximity assay was performed under the same assay conditions but replacing His-tagged CRBN-DDB1 with His-tagged VHL(1–154) (Abcam). For data analysis, the maximal luminescence signal was normalized to POC<sub>max</sub> (maximal percent of control) using <a class="ref internalNav" href="#eq4" aria-label="eq 4">eq 4</a>:<span class="NLM_disp-formula" id="eq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/jm-2016-017816_m001.gif" alt="" /></img><span class="labelSpan">(4)</span></span>where <i>S</i><sub>t</sub> is the maximal luminescence signal of test compound, S<sub>c+</sub> is the maximal luminescence signal of positive control, and S<sub>c–</sub> is the average luminescence signal of DMSO control.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry</h3><div class="NLM_p">Unless otherwise noted, all reagents were commercially available and used as received. All final compounds possessed purity ≥95% as determined by high performance liquid chromatography (HPLC). The HPLC method used the following conditions: HALO-C18 column (3 mm × 50 mm, 2.7 μm) at 40 °C with a 2.0 mL/min flow rate; solvent A of 0.1% TFA in water, solvent B of 0.1% TFA in MeCN; 5–95% gradient A:B over 1.5 min. Flow from the UV detector was split (50:50) to the MS detector, which was configured with APIES as ionizable source. <sup>1</sup>H NMR spectra were recorded on a 400 MHz Bruker NMR spectrometer at ambient temperature. Data are reported as follows: chemical shift (ppm, δ units) from an internal standard, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad), coupling constant (Hz), and integration. <sup>13</sup>C NMR spectra were recorded on a 600 MHz Bruker NMR spectrometer at ambient temperature (32768 scans). Silica gel chromatography was performed using an ISCO Combiflash RF medium pressure liquid chromatography system. Benchmark BRD4 PROTACs <b>1</b>,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and <b>2</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and VHL-amine (<b>9</b>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> were synthesized according to previously reported literature protocols. The IMiD containing PROTACs were isolated as a mixture of diastereomers which included racemic mixtures at the IMiD stereocenter with enantio-pure (<i>S</i>)-configuration at the JQ-1 stereocenter.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> (<i>S</i>)-<i>N</i>-(2-Azidoethyl)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamide (<b>4</b>)</h4><div class="NLM_p">Step 1. (<i>S</i>)-2-(4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetic acid compound with 2,2,2-trifluoroacetic acid (1:1) prepared according to Ciulli and co-workers.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (<i>S</i>)-<i>tert</i>-Butyl 2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetate (<a href="http://Selleckchem.com" class="extLink">Selleckchem.com</a>, catalogue no. S7110, 897 mg, 1.96 mmol) was treated with DCM (6 mL) and TFA (4 mL, 51.9 mmol) and allowed to stir at rt for 2 h. LC-MS analysis of the reaction mixture indicated complete <i>tert</i>-butyl ester deprotection <i>m</i>/<i>z</i> (ESI, +ve) 401.0 (M + H)<sup>+</sup>. The reaction mixture was concentrated to dryness, affording crude (<i>S</i>)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetic acid compound with 2,2,2-trifluoroacetic acid (1:1) (1.01 g, 1.96 mmol, 100% yield) as a viscous yellow tar (assuming quantitative yield of the mono(TFA) salt). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 3.49 (2 H, dd, <i>J</i> = 16.6, 6.8 Hz), 3.37 (1 H, dd, <i>J</i> = 16.6, 7.4 Hz), 2.65–2.70 (3 H, m), 2.14 (1 H, s). <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm −74.94 (1 F, s), assume mono(TFA) salt. <i>m</i>/<i>z</i> (ESI, +ve) 401.1 (M + H)<sup>+</sup>.</div><div class="NLM_p last">Step 2. (<i>S</i>)-2-(4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetic acid compound with 2,2,2-trifluoroacetic acid (1:1) (1.01 g, 1.96 mmol) was treated with 2-azidoethanamine hydrochloride (Enamine Chemicals, 289 mg, 2.36 mmol), DMF (10 mL), and Hunig’s base (1.72 mL, 9.82 mmol) followed by HATU (933 mg, 2.45 mmol) in one portion. The reaction mixture was then allowed to stir at rt for 30 min. LC-MS analysis of the reaction mixture indicated ca. 50% conversion to the desired product <i>m</i>/<i>z</i> (ESI, +ve) 469.1 (M + H)<sup>+</sup>. The reaction mixture was treated with additional Hunig’s base (1.0 mL), and after another 30 min, LC-MS analysis indicated clean conversion to the desired product. The reaction mixture was treated with a saturated solution of NaHCO<sub>3</sub> and extracted with EtOAc (2 × 50 mL), washed with brine (3 × 25 mL), and dried over MgSO<sub>4</sub>, filtered, and concentrated. The crude residue was purified on an ISCO Combiflash RF (40 g Grace Reveleris column, using a gradient of 0–60% [20% MeOH in DCM] in DCM (product eluted with ca. 30–38% [20% MeOH in DCM]/DCM)), affording (<i>S</i>)-<i>N</i>-(2-azidoethyl)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamide (<b>4</b>, 931 mg, 1.99 mmol, > 99% yield) as a light-yellow foam after drying in a vacuum oven at 45 °C overnight. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.45 (1 H, br t, <i>J</i> = 5.2 Hz), 7.42–7.50 (4 H, m), 4.51 (1 H, dd, <i>J</i> = 8.2, 5.9 Hz), 3.33–3.44 (3 H, m), 3.17–3.29 (3 H, m), 2.59 (3 H, s), 2.41 (3 H, s), 1.62 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 170.4, 165.1, 163.5, 162.8, 155.6, 150.3, 137.2, 135.7, 132.7, 131.2, 130.6, 130.3, 130.1, 128.9, 54.3, 50.5, 38.72, 38.65, 38.1, 36.3, 31.2, 28.3, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 469.1 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 2-(2,6-Dioxopiperidin-3-yl)-4-(prop-2-yn-1-yloxy)isoindoline-1,3-dione (<b>8a</b>)</h4><div class="NLM_p last">Compound <b>7</b> (10 g, 36.5 mmol), propargyl bromide (6.1 g, 51.1 mmol), and Na<sub>2</sub>CO<sub>3</sub> (5.8 g, 54.7 mmol) in DMF (80 mL) was stirred at 60 °C for 24 h. The solvent was evaporated, water (150 mL) was added, and then extracted with EtOAc (3 × 200 mL). The combined organic layers were washed with brine (100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give crude product which was purified by reverse phase column chromatography (RP-18, H<sub>2</sub>O/MeOH = 1:1) to furnish the title compound (<b>8a</b>, 6.0 g, 19.2 mmol, 53% yield) as a tan solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.06 (1 H, br s), 7.86 (1 H, t, <i>J</i> = 7.9 Hz), 7.57 (1 H, d, <i>J</i> = 8.1 Hz), 7.51 (1 H, d, <i>J</i> = 7.1 Hz), 5.05–5.13 (3 H, m), 3.69 (1 H, t, <i>J</i> = 2.2 Hz), 2.82–2.95 (1 H, m), 2.52–2.63 (2 H, m), 1.99–2.10 (1 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.4, 167.2, 165.6, 154.8, 137.3, 133.8, 120.7, 117.3, 116.5, 79.9, 78.7, 56.9, 49.3, 31.4, 22.4. <i>m</i>/<i>z</i> (ESI, +ve) 313.0 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Representative Procedure for the Synthesis of IMiD Alkynes (<b>8b</b>–<b>e</b>)</h4><div id="sec5_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> 2-(2,6-Dioxopiperidin-3-yl)-4-(2-(prop-2-yn-1-yloxy)ethoxy)isoindoline-1,3-dione (<b>8b</b>)</h5><div class="NLM_p last">A mixture of 2-(prop-2-yn-1-yloxy)ethyl 4-methylbenzenesulfonate (8.9 g, 35 mmol), <b>7</b> (8 g, 29 mmol) and Na<sub>2</sub>CO<sub>3</sub> (4.6 g, 44 mmol) in DMF (100 mL) was stirred at 80 °C for 42 h. The mixture was added to water (300 mL) and extracted with DCM (3 × 300 mL), and the combined organic layers were washed with brine (200 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give crude product. The residue was purified by reverse phase HPLC (0.1% TFA in CH<sub>3</sub>CN/water) to furnish the title compound (<b>8b</b>, 6.7 g, 18.4 mmol, 63% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.10 (1 H, s), 7.82 (1 H, t, <i>J</i> = 7.9 Hz), 7.53 (1 H, d, <i>J</i> = 8.7 Hz), 7.46 (1 H, d, <i>J</i> = 7.0 Hz), 5.09 (1 H, dd, <i>J</i> = 12.9, 5.4 Hz), 4.32–4.43 (2 H, m), 4.28 (2 H, d, <i>J</i> = 2.3 Hz), 3.79–3.88 (2 H, m), 3.38–3.51 (1 H, m), 2.82–2.94 (1 H, m), 2.52–2.63 (2 H, m), 1.99–2.07 (1 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.4, 167.3, 165.8, 156.2, 137.5, 133.7, 120.4, 116.8, 115.9, 80.7, 77.8, 69.1, 67.8, 58.4, 49.2, 31.4, 22.5. <i>m</i>/<i>z</i> (ESI, +ve) 357.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> 2-(2,6-Dioxopiperidin-3-yl)-4-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)isoindoline-1,3-dione (<b>8c</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of IMiD alkynes, affording the title compound (<b>8c</b>, 6.7 g, 16 mmol, 55% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.09 (1 H, s), 7.81 (1 H, t, <i>J</i> = 7.9 Hz), 7.53 (1 H, d, <i>J</i> = 8.5 Hz), 7.46 (1 H, d, <i>J</i> = 7.0 Hz), 5.08 (1 H, dd, <i>J</i> = 12.9, 5.4 Hz), 4.30–4.40 (2 H, m), 4.09–4.19 (2 H, m), 3.76–3.85 (2 H, m), 3.63–3.73 (2 H, m), 3.53–3.61 (2 H, m), 3.36–3.45 (1 H, m), 2.80–2.94 (1 H, m), 2.52–2.63 (2 H, m), 1.98–2.08 (1 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.4, 167.3, 165.8, 156.3, 137.5, 133.7, 120.6, 116.8, 115.9, 80.8, 77.6, 70.4, 69.4, 69.2, 69.1, 58.0, 49.2, 31.4, 22.5. <i>m</i>/<i>z</i> (ESI, +ve) 401.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> 2-(2,6-Dioxopiperidin-3-yl)-4-(2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)ethoxy)isoindoline-1,3-dione (<b>8d</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of IMiD alkynes, affording the title compound (<b>8d</b>, 5.4 g, 12 mmol, 41% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.09 (1 H, s), 7.81 (1 H, t, <i>J</i> = 7.9 Hz), 7.53 (1 H, d, <i>J</i> = 8.5 Hz), 7.46 (1 H, d, <i>J</i> = 7.3 Hz), 5.08 (1 H, dd, <i>J</i> = 12.9, 5.4 Hz), 4.29–4.40 (2 H, m), 4.12 (2 H, d, <i>J</i> = 2.3 Hz), 3.73–3.88 (2 H, m), 3.58–3.71 (2 H, m), 3.52–3.55 (6 H, m), 3.40 (1 H, t, <i>J</i> = 2.3 Hz), 3.32 (2 H, s), 2.82–2.95 (1 H, m), 2.52–2.63 (2 H, m), 1.98–2.08 (1 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.4, 167.3, 165.8, 156.3, 137.5, 133.7, 120.6, 116.8, 115.9, 80.8, 77.5, 70.6, 70.3, 70.0, 69.4, 69.2, 69.0, 58.0, 49.2, 31.4, 22.5. <i>m</i>/<i>z</i> (ESI, +ve) 445.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> 4-(3,6,9,12-Tetraoxapentadec-14-yn-1-yloxy)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>8e</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of IMiD alkynes, affording the title compound (<b>8e</b>, 6.8 g, 13.3 mmol, 45% yield) as a pale-yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.09 (1 H, s), 7.81 (1 H, t, <i>J</i> = 7.9 Hz), 7.53 (1 H, d, <i>J</i> = 8.5 Hz), 7.46 (1 H, d, <i>J</i> = 7.3 Hz), 5.08 (1 H, dd, <i>J</i> = 12.6, 5.4 Hz), 4.28–4.41 (3 H, m), 4.08–4.17 (3 H, m), 3.75–3.86 (3 H, m), 3.62–3.67 (3 H, m), 3.54–3.57 (2 H, m), 3.39–3.44 (1 H, m), 2.82–2.94 (1 H, m), 2.52–2.63 (2 H, m), 1.98–2.07 (1 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.4, 167.3, 165.7, 156.3, 137.5, 133.7, 120.6, 116.8, 115.9, 80.8, 77.6, 70.6, 70.3, 70.2, 70.0, 69.4, 69.2, 69.0, 58.0, 49.2, 31.4, 22.5. <i>m</i>/<i>z</i> (ESI, +ve) 489.2 (M + H)<sup>+</sup>.</div></div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Representative Procedure for the Synthesis of VHL-alkynes (<b>10a</b>–<b>e</b>)</h4><div id="sec5_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-Dimethyl-2-(2-(prop-2-yn-1-yloxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10a</b>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></h5><div class="NLM_p last">To a solution of 2-(prop-2-yn-1-yloxy)acetic acid (684 mg, 6.0 mmol, 1.0 equiv) in DCM (50 mL) was added (COCl)<sub>2</sub> (1.90 g, 15.0 mmol, 1.31 mL, 2.5 equiv) followed by DMF (438 mg, 6 mmol, 0.5 mL, 1.0 equiv) dropwise. After addition, the mixture was stirred at 15 °C for 1 h, then the mixture was concentrated under vacuum. The crude acid chloride was diluted with THF (50 mL), <b>9</b> (2.8 g, 6.0 mmol, 1.0 equiv) and TEA (3.04 g, 30 mmol, 4.16 mL, 5.0 equiv), and then the mixture was stirred at 40 °C for 12 h. The mixture was quenched with water (80 mL) and extracted with EtOAc (2 × 80 mL), and the combined organic phase was washed with brine (100 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under vacuum. The residue was purified by prep-HPLC affording the title compound (<b>10a</b>, 1.40 g, 2.64 mmol, 44% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.03 (1 H, s), 8.64 (1 H, t, <i>J</i> = 5.9 Hz), 7.47–7.54 (1 H, m), 7.45 (4 H, s), 5.20 (1 H, br s), 4.60 (1 H, d, <i>J</i> = 9.5 Hz), 4.44–4.52 (1 H, m), 4.37–4.44 (2 H, m), 4.25–4.35 (4 H, m), 3.97–4.12 (2 H, m), 3.61–3.75 (2 H, m), 3.56 (1 H, t, <i>J</i> = 2.3 Hz), 2.10 (1 H, br s), 1.95 (1 H, s), 0.93–1.03 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 169.6, 168.4, 151.9, 148.2, 139.9, 131.6, 130.2, 129.4, 129.2, 127.9, 80.0, 78.4, 69.4, 68.5, 59.2, 58.4, 57.1, 56.3, 42.2, 38.4, 36.2, 26.8, 26.7, 16.4. <i>m</i>/<i>z</i> (ESI, +ve) 527.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-Dimethyl-2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10b</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of VHL-alkynes, affording the title compound (<b>10b</b>, 2.0 g, 40% yield) as a light-yellow syrup. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.98 (1 H, s), 8.60 (1 H, t, <i>J</i> = 6.1 Hz), 7.48 (1 H, br d, <i>J</i> = 9.5 Hz), 7.40 (4 H, s), 5.15 (1 H, d, <i>J</i> = 3.5 Hz), 4.57 (1 H, d, <i>J</i> = 9.5 Hz), 4.39–4.47 (2 H, m), 4.37 (1 H, br d, <i>J</i> = 13.7 Hz), 4.12–4.30 (4 H, m), 3.96 (2 H, s), 3.56–3.70 (7 H, m), 3.36–3.44 (1 H, m), 1.90 (1 H, br s), 0.88–1.00 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.3, 169.6, 169.1, 151.9, 148.2, 139.9, 131.6, 130.2, 129.4, 129.2, 128.6, 127.9, 80.6, 77.73, 77.70, 70.7, 70.5, 70.0, 69.4, 68.7, 59.2, 58.0, 57.0, 56.2, 42.2, 40.9, 38.4, 36.2, 26.8, 26.7, 16.4, 16.4. <i>m</i>/<i>z</i> (ESI, +ve) 571.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-4-oxo-6,9,12-trioxa-3-azapentadec-14-yn-1-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10c</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of VHL-alkynes, affording the title compound (<b>10c</b>, 1.0 g, 40% yield) as a light-yellow syrup. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.03 (1 H, s), 8.64 (1 H, br t, <i>J</i> = 6.0 Hz), 7.41–7.51 (6 H, m), 5.19 (1 H, d, <i>J</i> = 3.5 Hz), 4.61 (1 H, d, <i>J</i> = 9.5 Hz), 4.36–4.52 (4 H, m), 4.24–4.36 (2 H, m), 4.12–4.22 (3 H, m), 4.01 (2 H, s), 3.56–3.74 (13 H, m), 3.40–3.52 (3 H, m), 2.65 (1 H, br s), 2.57–2.60 (21 H, m), 2.03–2.25 (1 H, m), 1.95 (1 H, ddd, <i>J</i> = 12.9, 8.7, 4.6 Hz), 0.93–1.04 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 169.6, 169.1, 151.9, 148.2, 139.9, 131.6, 130.2, 129.4, 129.2, 127.9, 80.8, 77.6, 70.9, 70.1, 70.0, 69.4, 69.01, 68.99, 59.2, 58.0, 57.0, 56.2, 42.2, 38.4, 36.2, 26.8, 26.8, 26.7, 16.4, 16.4. <i>m</i>/<i>z</i> (ESI, +ve) 615.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-4-oxo-6,9,12,15-tetraoxa-3-azaoctadec-17-yn-1-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10d</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of VHL-alkynes, affording the title compound (<b>10d</b>, 3.6 g, 32% yield) as a light-yellow syrup. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.98 (1 H, s), 8.59 (1 H, t, <i>J</i> = 6.1 Hz), 7.40 (5 H, s), 5.15 (1 H, d, <i>J</i> = 3.5 Hz), 4.56 (1 H, d, <i>J</i> = 9.5 Hz), 4.39–4.47 (2 H, m), 4.32–4.39 (2 H, m), 4.19–4.31 (2 H, m), 4.10–4.15 (3 H, m), 3.96 (2 H, s), 3.62–3.69 (3 H, m), 3.52–3.62 (11 H, m), 3.38–3.42 (2 H, m), 2.53–2.55 (35 H, m), 2.42–2.47 (4 H, m), 2.05 (1 H, br dd, <i>J</i> = 12.5, 7.8 Hz), 1.90 (1 H, ddd, <i>J</i> = 12.8, 8.7, 4.5 Hz), 0.89–0.99 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 169.6, 169.1, 151.9, 148.2, 139.9, 131.6, 130.2, 129.2, 128.6, 127.9, 80.8, 77.6, 70.9, 70.1, 70.03, 70.00, 69.4, 69.01, 68.99, 59.2, 58.0, 57.0, 56.2, 42.2, 38.4, 36.2, 26.8, 26.7, 16.4. <i>m</i>/<i>z</i> (ESI, +ve) 637.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-4-oxo-6,9,12,15,18-pentaoxa-3-azahenicos-20-yn-1-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10e</b>)</h5><div class="NLM_p last">Prepared according to the representative procedure for the synthesis of VHL-alkynes, affording the title compound (<b>10e</b>, 6.0 g, 39% yield) as a light-yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.98 (1 H, s), 8.59 (1 H, br t, <i>J</i> = 5.9 Hz), 7.34–7.46 (6 H, m), 5.15 (1 H, d, <i>J</i> = 3.3 Hz), 4.56 (1 H, d, <i>J</i> = 9.7 Hz), 4.40–4.47 (2 H, m), 4.31–4.39 (2 H, m), 4.19–4.29 (2 H, m), 4.13 (3 H, d, <i>J</i> = 2.3 Hz), 3.96 (2 H, s), 3.57–3.69 (7 H, m), 3.44–3.57 (18 H, m), 3.38–3.42 (2 H, m), 2.60 (1 H, br s), 2.52–2.55 (21 H, m), 2.42–2.46 (4 H, m), 2.01–2.11 (1 H, m), 1.90 (1 H, ddd, <i>J</i> = 12.9, 8.8, 4.5 Hz), 0.88–0.99 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 169.6, 169.1, 151.9, 148.2, 139.9, 131.6, 130.2, 129.4, 129.2, 128.6, 127.9, 80.8, 77.6, 70.9, 70.3, 70.2, 70.1, 69.4, 69.0, 59.2, 58.0, 57.0, 56.2, 42.2, 40.9, 38.4, 36.2, 26.8, 26.8, 26.7, 16.4. <i>m</i>/<i>z</i> (ESI, +ve) 703.4 (M + H)<sup>+</sup>.</div></div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Representative Procedure for the “Click Ligation”</h4><div class="NLM_p last">In a 3 mL vial was weighed (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-dimethyl-2-(2-(prop-2-yn-1-yloxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10a</b>, 51 mg, 0.097 mmol), (+)-sodium l-ascorbate (3.8 mg, 0.019 mmol), copper(II) sulfate powder (3.1 mg, 0.019 mmol), and (<i>S</i>)-<i>N</i>-(2-azidoethyl)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamide (<b>4</b>, 45.4 mg, 0.097 mmol). The reaction mixture was treated with THF (1 mL, ca. 0.1M) and 3–4 drops of water, and then the headspace of the vial was purged briefly with argon and stirred at rt for 16 h. LC-MS analysis of the crude reaction mixture indicated clean conversion to the desired product <i>m</i>/<i>z</i> (ESI, +ve) 995.4 (M + H)<sup>+</sup>. The reaction mixture was transferred to a 5 g silica gel sample loader using a pipet and purified on an ISCO Combiflash RF (12 g Redisep Gold column, using a gradient of 0–100% [20% MeOH in DCM] in DCM (product eluted with ca. 85–95% [20% MeOH in DCM]/DCM)), affording the desired (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-((1-(2-(2-((<i>S</i>)-4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12a</b>, 76.1 mg, 0.076 mmol, 79% yield) as a white amorphous solid after drying overnight in a vacuum oven at 45 °C.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)-<i>N</i>-(2-(4-(((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethyl)acetamide (<b>11a</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording the title compound (<b>11a</b>, 79.6 mg, 0.10 mmol, 90% yield) as a white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.08 (1 H, s), 8.44 (1 H, br t, <i>J</i> = 5.4 Hz), 8.31 (1 H, s), 7.72–7.84 (2 H, m), 7.40–7.50 (5 H, m), 5.75 (1 H, s), 5.42 (2 H, s), 5.06 (1 H, dd, <i>J</i> = 12.7, 5.5 Hz), 4.44–4.52 (3 H, m), 3.44–3.64 (2 H, m), 3.22 (2 H, br d, <i>J</i> = 7.0 Hz), 2.80–2.93 (1 H, m), 2.58 (3 H, s), 2.40 (3 H, s), 1.96–2.05 (1 H, m), 1.62 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.2, 170.6, 170.4, 167.2, 165.7, 163.6, 155.8, 155.6, 142.3, 137.4, 137.2, 135.7, 133.8, 132.7, 131.2, 130.7, 130.4, 130.1, 128.9, 125.8, 120.9, 117.0, 116.1, 62.7, 55.4, 54.2, 49.4, 49.2, 38.0, 31.4, 31.2, 22.5, 14.6, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 781.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)-<i>N</i>-(2-(4-((2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethyl)acetamide (<b>11b</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording the title compound (<b>11b</b>, 89.9 mg, 0.11 mmol, 67% yield) as a light-yellow amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.11 (1 H, s), 8.40–8.46 (1 H, m), 8.20 (1 H, s), 7.79 (1 H, t, <i>J</i> = 7.5 Hz), 7.40–7.52 (5 H, m), 5.09 (1 H, dd, <i>J</i> = 12.8, 5.4 Hz), 4.66 (2 H, s), 4.41–4.50 (3 H, m), 4.29–4.38 (2 H, m), 3.80–3.88 (2 H, m), 3.54–3.62 (1 H, m), 3.22 (2 H, br d, <i>J</i> = 7.2 Hz), 2.81–2.92 (1 H, m), 2.58 (3 H, s), 2.39 (3 H, s), 1.97–2.05 (1 H, m), 1.61 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.2, 170.59, 170.57, 170.4, 167.3, 165.8, 163.6, 156.3, 137.5, 137.2, 135.7, 133.71, 133.69, 132.7, 131.22, 131.21, 130.7, 130.4, 130.3, 130.0, 128.9, 124.7, 120.4, 116.8, 115.9, 100.0, 69.2, 68.1, 64.3, 55.4, 54.2, 49.4, 49.3, 49.2, 49.1, 38.0, 31.4, 22.5, 14.5, 13.1, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 825.2 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)-<i>N</i>-(2-(4-((2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethyl)acetamide (<b>11c</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording the title compound (<b>11c</b>, 80.0 mg, 0.092 mmol, 70% yield) as a light-yellow amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.10 (1 H, s), 8.44 (1 H, br t, <i>J</i> = 5.6 Hz), 8.12 (1 H, s), 7.79 (1 H, t, <i>J</i> = 7.8 Hz), 7.39–7.53 (6 H, m), 5.08 (1 H, dd, <i>J</i> = 12.7, 5.3 Hz), 4.40–4.54 (5 H, m), 4.27–4.35 (2 H, m), 3.73–3.84 (2 H, m), 3.61–3.68 (2 H, m), 3.49–3.60 (4 H, m), 3.22 (2 H, br d, <i>J</i> = 7.0 Hz), 2.81–2.93 (1 H, m), 2.59 (3 H, s), 2.40 (3 H, s), 1.96–2.06 (1 H, m), 1.62 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.6, 170.4, 167.3, 165.8, 163.6, 156.3, 155.6, 144.4, 137.4, 137.2, 135.7, 133.7, 132.7, 131.2, 130.7, 130.4, 130.0, 128.9, 124.7, 120.5, 116.8, 115.9, 70.5, 69.5, 69.3, 69.2, 64.0, 55.4, 54.2, 49.26, 49.23, 49.1, 38.0, 31.4, 22.5, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 869.4 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)-<i>N</i>-(2-(4-((2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethyl)acetamide (<b>11d</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording the title compound (<b>11d,</b> 83.3 mg, 0.091 mmol, 73% yield) as an off-white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.09 (1 H, s), 8.43 (1 H, br t, <i>J</i> = 5.7 Hz), 8.11 (1 H, s), 7.79 (1 H, t, <i>J</i> = 7.8 Hz), 7.37–7.55 (6 H, m), 5.08 (1 H, dd, <i>J</i> = 12.8, 5.4 Hz), 4.38–4.53 (5 H, m), 4.31–4.37 (2 H, m), 3.72–3.82 (2 H, m), 3.60–3.65 (2 H, m), 3.44–3.58 (8 H, m), 3.16–3.27 (3 H, m), 2.75–2.93 (1 H, m), 2.53–2.62 (5 H, m), 2.38–2.43 (3 H, m), 1.96–2.06 (1 H, m), 1.62 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.6, 170.4, 167.3, 165.7, 163.6, 156.3, 155.5, 150.4, 144.3, 137.4, 137.2, 135.7, 133.7, 132.7, 131.2, 130.7, 130.4, 130.0, 128.9, 124.7, 120.5, 116.8, 115.9, 105.9, 91.8, 88.4, 75.3, 73.8, 70.6, 70.3, 70.2, 69.4, 69.3, 69.2, 64.0, 55.4, 54.2, 49.3, 49.2, 38.0, 31.4, 31.2, 22.5, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 913.4 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)-<i>N</i>-(2-(4-(13-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)-2,5,8,11-tetraoxatridecyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethyl)acetamide (<b>11e</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording the title compound (<b>11e</b>, 76.2 mg, 0.080 mmol, 69% yield) as an off-white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.10 (1 H, s), 8.44 (1 H, br t, <i>J</i> = 5.5 Hz), 8.12 (1 H, s), 7.80 (1 H, t, <i>J</i> = 7.9 Hz), 7.41–7.55 (6 H, m), 5.09 (1 H, dd, <i>J</i> = 12.8, 5.4 Hz), 4.32–4.54 (7 H, m), 4.09 (1 H, br d, <i>J</i> = 5.1 Hz), 3.72–3.84 (2 H, m), 3.57–3.69 (3 H, m), 3.47–3.57 (10 H, m), 3.23 (2 H, br d, <i>J</i> = 7.0 Hz), 3.18 (3 H, d, <i>J</i> = 3.7 Hz), 2.83–2.95 (1 H, m), 2.60 (3 H, s), 2.41 (3 H, s), 1.98–2.08 (1 H, m), 1.63 (3 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 170.6, 170.4, 167.3, 165.7, 163.6, 156.3, 155.5, 150.3, 144.3, 137.4, 137.2, 135.7, 133.7, 132.7, 131.2, 130.7, 130.4, 130.0, 128.9, 124.7, 120.5, 116.8, 115.9, 91.8, 88.4, 75.3, 73.8, 70.6, 70.2, 69.4, 69.3, 69.2, 64.0, 54.2, 49.3, 49.2, 49.1, 38.0, 31.4, 22.5, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 957.4 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-((1-(2-(2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12a</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording (<b>12a</b>, 76.1 mg, 0.076 mmol, 79% yield) as a white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.97 (1 H, s), 8.58 (1 H, t, <i>J</i> = 5.9 Hz), 8.45 (1 H, t, <i>J</i> = 5.6 Hz), 8.20 (1 H, s), 7.47 (1 H, s), 7.48 (2 H, d, <i>J</i> = 7.9 Hz), 7.35–7.45 (7 H, m), 5.15 (1 H, d, <i>J</i> = 3.5 Hz), 4.63 (2 H, s), 4.56 (1 H, d, <i>J</i> = 9.6 Hz), 4.50 (1 H, t, <i>J</i> = 7.1 Hz), 4.31–4.47 (5 H, m), 4.20–4.31 (1 H, m), 4.00 (2 H, d, <i>J</i> = 4.7 Hz), 3.53–3.69 (4 H, m), 3.21–3.27 (2 H, m), 2.55–2.63 (3 H, m), 2.43–2.46 (3 H, m), 2.41 (3 H, s), 1.98–2.10 (1 H, m), 1.90 (1 H, td, <i>J</i> = 8.7, 4.4 Hz), 1.62 (3 H, s), 0.87–0.97 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 170.6, 169.6, 168.7, 163.6, 155.5, 151.9, 150.3, 148.2, 143.5, 139.9, 137.2, 135.7, 132.7, 131.6, 131.2, 130.7, 130.4, 130.2, 130.0, 129.4, 129.2, 128.9, 127.9, 125.1, 69.4, 69.0, 64.2, 59.2, 57.1, 56.2, 54.2, 49.3, 42.2, 38.4, 38.0, 36.2, 26.7, 26.7, 16.4, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 995.3 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(2-((1-(2-(2-((<i>S</i>)-4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12b</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording (<b>12b</b>, 61.5 mg, 0.059 mmol, 60% yield) as an off-white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.95–8.98 (1 H, m), 8.59 (1 H, t, <i>J</i> = 6.0 Hz), 8.43 (1 H, br t, <i>J</i> = 5.7 Hz), 8.14 (1 H, s), 7.45–7.52 (3 H, m), 7.33–7.45 (6 H, m), 5.15 (1 H, d, <i>J</i> = 3.5 Hz), 4.52–4.61 (3 H, m), 4.38–4.52 (4 H, m), 4.36 (2 H, br d, <i>J</i> = 6.1 Hz), 4.20–4.32 (1 H, m), 3.95 (2 H, s), 3.47–3.70 (8 H, m), 3.19–3.27 (2 H, m), 3.17 (1 H, d, <i>J</i> = 5.1 Hz), 2.59 (3 H, s), 2.42 (3 H, s), 2.40 (3 H, s), 1.98–2.10 (1 H, m), 1.90 (1 H, ddd, <i>J</i> = 12.9, 8.7, 4.5 Hz), 1.62 (3 H, s), 0.87–0.99 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 171.8, 170.6, 170.0, 169.7, 169.1, 168.2, 163.6, 155.5, 152.0, 151.9, 150.3, 148.2, 144.2, 139.9, 137.2, 135.7, 132.7, 131.6, 131.2, 130.7, 130.5, 130.4, 130.2, 130.0, 129.4, 129.2, 128.9, 128.6, 127.9, 124.8, 124.7, 109.0, 70.8, 70.7, 70.1, 70.1, 69.9, 69.3, 69.03, 68.97, 67.4, 64.1, 64.0, 59.4, 59.2, 57.0, 56.2, 56.0, 55.4, 54.2, 49.3, 49.1, 38.4, 38.0, 36.6, 36.2, 26.8, 26.7, 26.7, 22.7, 18.4, 16.43, 16.38, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 1039.4 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>S,</i>4<i>R</i>)-1-((<i>S</i>)-12-(<i>tert</i>-Butyl)-1-(1-(2-(2-((<i>S</i>)-4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)-10-oxo-2,5,8-trioxa-11-azatridecan-13-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12c</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording (<b>12c</b>, 87.6 mg, 0.081 mmol, 72% yield) as an off-white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.94–9.00 (1 H, m), 8.59 (1 H, t, <i>J</i> = 6.0 Hz), 8.43 (1 H, t, <i>J</i> = 5.7 Hz), 8.07–8.14 (1 H, m), 7.46–7.51 (2 H, m), 7.30–7.45 (7 H, m), 5.14 (1 H, d, <i>J</i> = 3.5 Hz), 4.56 (1 H, d, <i>J</i> = 9.8 Hz), 4.47–4.53 (3 H, m), 4.39–4.47 (4 H, m), 4.36 (2 H, br d, <i>J</i> = 6.3 Hz), 4.25 (1 H, br dd, <i>J</i> = 15.7, 5.8 Hz), 4.09 (3 H, q, <i>J</i> = 5.3 Hz), 3.96 (2 H, s), 3.64–3.70 (1 H, m), 3.53–3.63 (11 H, m), 3.23 (2 H, br d, <i>J</i> = 7.0 Hz), 3.17 (9 H, d, <i>J</i> = 5.3 Hz), 2.59 (3 H, s), 2.43 (3 H, s), 2.41 (3 H, s), 2.04 (1 H, br d, <i>J</i> = 8.4 Hz), 1.85–1.98 (1 H, m), 1.62 (3 H, s), 1.25 (2 H, br d, <i>J</i> = 8.2 Hz), 0.93 (9 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 170.6, 169.6, 169.1, 163.6, 155.5, 151.9, 150.3, 148.2, 144.3, 139.9, 137.2, 135.7, 132.7, 131.6, 131.2, 130.7, 130.4, 130.2, 130.0, 129.4, 129.2, 128.9, 128.6, 128.0, 124.7, 100.0, 70.9, 70.2, 70.08, 70.07, 69.4, 69.2, 64.0, 59.2, 57.0, 56.2, 54.2, 49.3, 49.1, 46.2, 42.2, 38.4, 38.0, 36.6, 36.2, 26.7, 26.7, 16.44, 16.39, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 1083.3 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (2<i>S,</i>4<i>R</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(1-(2-(2-((<i>S</i>)-4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)-13-oxo-2,5,8,11-tetraoxa-14-azahexadecan-16-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12d</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method, affording (<b>12d</b>, 61.8 mg, 0.055 mmol, 68% yield) as a pale-green amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.97 (1 H, s), 8.58 (1 H, t, <i>J</i> = 5.9 Hz), 8.43 (1 H, t, <i>J</i> = 5.6 Hz), 8.11 (1 H, s), 7.45–7.51 (2 H, m), 7.34–7.44 (7 H, m), 5.14 (1 H, d, <i>J</i> = 3.5 Hz), 4.55 (1 H, d, <i>J</i> = 9.6 Hz), 4.48–4.52 (3 H, m), 4.38–4.47 (4 H, m), 4.36 (2 H, br d, <i>J</i> = 6.5 Hz), 4.20–4.31 (1 H, m), 4.08 (2 H, q, <i>J</i> = 5.3 Hz), 3.95 (2 H, s), 3.63–3.69 (1 H, m), 3.57–3.63 (3 H, m), 3.45–3.57 (13 H, m), 3.22 (2 H, d, <i>J</i> = 7.2 Hz), 3.16 (7 H, d, <i>J</i> = 5.3 Hz), 2.59 (3 H, s), 2.43 (3 H, s), 2.40 (3 H, s), 2.03 (1 H, br d, <i>J</i> = 8.2 Hz), 1.84–1.94 (1 H, m), 1.62 (3 H, s), 0.93 (9 H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 173.3, 172.2, 170.6, 169.6, 169.1, 163.6, 155.6, 151.9, 148.2, 144.3, 139.9, 139.4, 137.2, 135.7, 132.7, 131.6, 131.2, 130.7, 130.3, 130.2, 130.0, 129.4, 129.2, 128.9, 128.6, 127.9, 124.7, 70.9, 70.29, 70.26, 70.14, 70.06, 69.36, 69.35, 64.0, 59.2, 57.0, 56.2, 54.2, 49.3, 49.1, 38.4, 38.0, 36.2, 26.7, 26.6, 16.43, 16.39, 14.5, 13.2, 11.8. <i>m</i>/<i>z</i> (ESI, +ve) 1127.4 (M + H)<sup>+</sup>.</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>S,</i>4<i>R</i>)-1-((<i>S</i>)-18-(<i>tert</i>-Butyl)-1-(1-(2-(2-((<i>S</i>)-4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)-16-oxo-2,5,8,11,14-pentaoxa-17-azanonadecan-19-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12e</b>)</h4><div class="NLM_p last">Prepared according to the general “click ligation” method and isolated (<b>12e</b>, 61.6 mg, 0.053 mmol, 55% yield) as an off-white amorphous solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.97 (1 H, s), 8.59 (1 H, t, <i>J</i> = 6.0 Hz), 8.43 (1 H, t, <i>J</i> = 5.8 Hz), 8.12 (1 H, s), 7.44–7.49 (2 H, m), 7.37–7.44 (7 H, m), 5.14 (1 H, d, <i>J</i> = 3.5 Hz), 4.56 (1 H, d, <i>J</i> = 9.6 Hz), 4.32–4.53 (6 H, m), 4.20–4.29 (1 H, m), 3.96 (2 H, s), 3.63–3.70 (1 H, m), 3.48–3.63 (4 H, m), 3.23 (2 H, d, <i>J</i> = 7.0 Hz), 2.59 (3 H, s), 2.42–2.45 (3 H, m), 2.41 (3 H, s), 2.02–2.10 (2 H, m), 1.86–1.95 (1 H, m), 1.62 (3 H, s), 0.91–0.98 (9 H, m). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 172.2, 170.6, 169.6, 169.1, 163.6, 155.5, 151.9, 150.3, 148.2, 144.3, 139.9, 137.2, 135.7, 132.7, 131.6, 131.2, 130.7, 130.3, 130.2, 130.0, 129.4, 129.2, 128.9, 128.6, 127.9, 124.7, 70.8, 70.3, 70.24, 70.23, 70.21, 70.13, 70.05, 70.01, 69.4, 64.0, 59.2, 57.0, 56.2, 55.4, 54.2, 49.3, 38.4, 38.0, 36.6, 36.2, 31.2, 26.8, 26.6, 16.4, 13.2. <i>m</i>/<i>z</i> (ESI, +ve) 1172.2 (M + H)<sup>+</sup>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01781">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80735" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80735" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01781" class="ext-link">10.1021/acs.jmedchem.6b01781</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra are provided for all of the novel compounds reported (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf">jm6b01781_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_002.csv">jm6b01781_si_002.csv (1.99 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01781" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ryan P. Wurz</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1413-5208" title="Orcid link">http://orcid.org/0000-0003-1413-5208</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f6848183848cb6979b919398d895999b"><span class="__cf_email__" data-cfemail="addfdad8dfd7edccc0cac8c383cec2c0">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ken Dellamaggiore</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hannah Dou</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Noelle Javier</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mei-Chu Lo</span> - <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John D. McCarter</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dane Mohl</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Sastri</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Russell Lipford</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victor J. Cee</span> - <span class="hlFld-Affiliation affiliation">†Departments
of Therapeutic Discovery−Medicinal Chemistry, ‡Oncology Research and §Discovery Technologies, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Therapeutic Discovery−Discovery
Technologies, Amgen Inc., 1120 Veterans Boulevard, South San Francisco, California 94080, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d49e2999-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13588" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13588" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Chris Wilde for NMR studies, Bill Romanow and Steve Thibault for DDB1/cereblon protein expression and purification and Sasha Kamb, Philip Tagari, Margaret Chu-Moyer and Chris Fotsch for their support of this program.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i40" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i40"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i41" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i41"> Abbreviations Used</h2><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">member of the BET bromodomain family</p></td></tr><tr><td class="NLM_term">DC<sub>50</sub></td><td class="NLM_def"><p class="first last">concentration of a PROTAC at which the cellular protein content is reduced by half</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">IMiD</td><td class="NLM_def"><p class="first last">ligand for cereblon E3 ligase</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimeras</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau protein</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Small-molecule PROTACS: New approaches to protein degradation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1966</span><span class="NLM_x">–</span> <span class="NLM_lpage">1973</span><span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+New+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0liV0IjeE2nzgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520New%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhu487fU1ntFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Berndsen, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">New insights into ubiquitin E3 ligase mechanism</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span><span class="refDoi"> DOI: 10.1038/nsmb.2780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnsmb.2780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=24699078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1aisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=301-307&author=C.+E.+Berndsenauthor=C.+Wolberger&title=New+insights+into+ubiquitin+E3+ligase+mechanism&doi=10.1038%2Fnsmb.2780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into ubiquitin E3 ligase mechanism</span></div><div class="casAuthors">Berndsen, Christopher E.; Wolberger, Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-307</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  E3 ligases carry out the final step in the ubiquitination cascade, catalyzing transfer of ubiquitin from an E2 enzyme to form a covalent bond with a substrate lysine.  Three distinct classes of E3 ligases have been identified that stimulate transfer of ubiquitin and ubiquitin-like proteins through either a direct or an indirect mechanism.  Only recently have the catalytic mechanisms of E3 ligases begun to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovYmD2sLo2RrVg90H21EOLACvtfcHk0lgN6XQU2MTNiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1aisrk%253D&md5=fd44f6d69f85067b70322ca304aed507</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2780%26sid%3Dliteratum%253Aachs%26aulast%3DBerndsen%26aufirst%3DC.%2BE.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DNew%2520insights%2520into%2520ubiquitin%2520E3%2520ligase%2520mechanism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D301%26epage%3D307%26doi%3D10.1038%2Fnsmb.2780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Okuhira, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimaki-Mogami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">Development of hybrid small molecules that induce degradation of estrogen receptor-alpha and necrotic cell death in breast cancer cells</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">1492</span><span class="NLM_x">–</span> <span class="NLM_lpage">1498</span><span class="refDoi"> DOI: 10.1111/cas.12272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1111%2Fcas.12272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=23992566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ylu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=1492-1498&author=K.+Okuhiraauthor=Y.+Demizuauthor=T.+Hattoriauthor=N.+Ohokaauthor=N.+Shibataauthor=T.+Nishimaki-Mogamiauthor=H.+Okudaauthor=M.+Kuriharaauthor=M.+Naito&title=Development+of+hybrid+small+molecules+that+induce+degradation+of+estrogen+receptor-alpha+and+necrotic+cell+death+in+breast+cancer+cells&doi=10.1111%2Fcas.12272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Development of hybrid small molecules that induce degradation of estrogen receptor-alpha and necrotic cell death in breast cancer cells</span></div><div class="casAuthors">Okuhira, Keiichiro; Demizu, Yosuke; Hattori, Takayuki; Ohoka, Nobumichi; Shibata, Norihito; Nishimaki-Mogami, Tomoko; Okuda, Haruhiro; Kurihara, Masaaki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1492-1498</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Manipulation of protein stability with small mols. has a great potential for both basic research and clin. therapy.  Recently, we have developed a series of hybrid small mols. named SNIPER (Specific and Non-genetic IAP-dependent Protein ERaser) that induces degrdn. of target proteins via ubiquitin-proteasome system.  Here we report the activities of SNIPER(ER) that targets estrogen receptor alpha (ERα) for degrdn.  SNIPER(ER) induced degrdn. of ERα and inhibited estrogen-dependent expression of pS2 gene in an estrogen-dependent breast cancer cell line MCF-7.  A proteasome inhibitor MG132 and siRNA-mediated downregulation of cIAP1 abrogated the SNIPER(ER)-induced ERα degrdn., suggesting that the ERα is degraded by proteasome subsequent to cIAP1-mediated ubiquitylation.  Intriguingly, after the ERα degrdn., the SNIPER(ER)-treated MCF-7 cells undergo rapid cell death.  Detailed anal. indicated that SNIPER(ER) caused necrotic cell death accompanied by a release of HMGB1, a marker of necrosis, from the cells.  Following the ERα degrdn., reactive oxygen species (ROS) was produced in the SNIPER(ER)-treated MCF-7 cells, and an anti-oxidant N-acetylcysteine inhibited the necrotic cell death.  These results indicate that SNIPER(ER) induces ERα degrdn., ROS prodn. and necrotic cell death, implying a therapeutic potential of SNIPER(ER) as a lead for the treatment of ERα-pos. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpckyKeS1C4ZrVg90H21EOLACvtfcHk0lgN6XQU2MTNiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ylu7fM&md5=dc35aee47ea4cd9554a663eb077137b5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcas.12272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12272%26sid%3Dliteratum%253Aachs%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNishimaki-Mogami%26aufirst%3DT.%26aulast%3DOkuda%26aufirst%3DH.%26aulast%3DKurihara%26aufirst%3DM.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520hybrid%2520small%2520molecules%2520that%2520induce%2520degradation%2520of%2520estrogen%2520receptor-alpha%2520and%2520necrotic%2520cell%2520death%2520in%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Sci.%26date%3D2013%26volume%3D104%26spage%3D1492%26epage%3D1498%26doi%3D10.1111%2Fcas.12272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lghPbQFH-e4gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerschmied, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salami, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaime-Figueroa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic Bcr-Abl</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">810</span><span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+Bcr-Abl&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0ljKmolHBXd3QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520Bcr-Abl%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span> </span><span class="NLM_article-title">Development of Bcr-Abl degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">4865</span><span class="NLM_x">–</span> <span class="NLM_lpage">4869</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+Bcr-Abl+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0liL0nu4b13Osw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520Bcr-Abl%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lebraud, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span> <span class="citation_source-journal">ACS Cent. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0lji3lGA3pPliw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lgc0VPEeATxqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Winter, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Drug development. Phthalimide conjugation as a strategy for in vivo target protein degradation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">1376</span><span class="NLM_x">–</span> <span class="NLM_lpage">1381</span><span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Drug+development.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0liyWPt0yKvXSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDrug%2520development.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1770</span><span class="NLM_x">–</span> <span class="NLM_lpage">1777</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhe_UNlmKirGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><div class="note"><p class="first last">For a review on E3 ligases see:</p></div><span class="NLM_contrib-group">Bulatov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Targeting Cullin–RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1042/BJ20141450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1042%2FBJ20141450" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=365-386&author=E.+Bulatovauthor=A.+Ciulli&title=Targeting+Cullin%E2%80%93RING+E3+ubiquitin+ligases+for+drug+discovery%3A+structure%2C+assembly+and+small-molecule+modulation&doi=10.1042%2FBJ20141450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20141450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141450%26sid%3Dliteratum%253Aachs%26aulast%3DBulatov%26aufirst%3DE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520Cullin%25E2%2580%2593RING%2520E3%2520ubiquitin%2520ligases%2520for%2520drug%2520discovery%253A%2520structure%252C%2520assembly%2520and%2520small-molecule%2520modulation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D365%26epage%3D386%26doi%3D10.1042%2FBJ20141450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><div class="note"><p class="first last">For a leading reference into the optimization of VHL see:</p></div><span class="NLM_contrib-group">Galdeano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaffidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Molle, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birced, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the Von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8657</span><span class="NLM_x">–</span> <span class="NLM_lpage">8663</span><span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+Von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lh2kIltuYRJyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Schneekloth, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pucheault, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Targeted intracellular protein degradation induced by a small molecule: en route to chemical proteomics</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">5904</span><span class="NLM_x">–</span> <span class="NLM_lpage">5908</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2008.07.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=18752944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5904-5908&author=A.+R.+Schneeklothauthor=M.+Pucheaultauthor=H.+S.+Taeauthor=C.+M.+Crews&title=Targeted+intracellular+protein+degradation+induced+by+a+small+molecule%3A+en+route+to+chemical+proteomics&doi=10.1016%2Fj.bmcl.2008.07.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted intracellular protein degradation induced by a small molecule: En route to chemical proteomics</span></div><div class="casAuthors">Schneekloth, Ashley R.; Pucheault, Mathieu; Tae, Hyun Seop; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5904-5908</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have developed a heterobifunctional all-small mol. PROTAC (PROteolysis TArgeting Chimera) capable of inducing proteasomal degrdn. of the androgen receptor.  This cell permeable PROTAC consists of a non-steroidal androgen receptor ligand (SARM) and the MDM2 ligand known as nutlin, connected by a PEG-based linker.  The SARM-nutlin PROTAC recruits the androgen receptor to MDM2, which functions as an E3 ubiquitin ligase.  This leads to the ubiquitination of the androgen receptor, and its subsequent degrdn. by the proteasome.  Upon treatment of HeLa cells with 10 μM PROTAC for 7 h, we were able to observe a decrease in androgen receptor levels.  This degrdn. is proteasome dependent, as it is mitigated in cells pre-treated with 10 μM epoxomicin, a specific proteasome inhibitor.  These results have implications for the potential study and treatment of various cancers with increased androgen receptor levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZh1309EM47LVg90H21EOLACvtfcHk0lh2kIltuYRJyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7rL&md5=49f32203efc4b2f6f2cbc900df398a1a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.114%26sid%3Dliteratum%253Aachs%26aulast%3DSchneekloth%26aufirst%3DA.%2BR.%26aulast%3DPucheault%26aufirst%3DM.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520intracellular%2520protein%2520degradation%2520induced%2520by%2520a%2520small%2520molecule%253A%2520en%2520route%2520to%2520chemical%2520proteomics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5904%26epage%3D5908%26doi%3D10.1016%2Fj.bmcl.2008.07.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sakamoto, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumagai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshaies, R. J.</span><span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">8554</span><span class="NLM_x">–</span> <span class="NLM_lpage">8559</span><span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lic3Qlybm1wlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhjLXVFRcLKgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><div class="note"><p class="first last">For a recent report on the synthesis of a phthalimide ligand library, see:</p></div><span class="NLM_contrib-group">Lohbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. K.</span><span> </span><span class="NLM_article-title">Practical synthesis of a phthalimide-based cereblon ligand to enable PROTAC development</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5260</span><span class="NLM_x">–</span> <span class="NLM_lpage">5262</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2016.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27687673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5260-5262&author=J.+Lohbeckauthor=A.+K.+Miller&title=Practical+synthesis+of+a+phthalimide-based+cereblon+ligand+to+enable+PROTAC+development&doi=10.1016%2Fj.bmcl.2016.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Practical synthesis of a phthalimide-based Cereblon ligand to enable PROTAC development</span></div><div class="casAuthors">Lohbeck, Jasmin; Miller, Aubry K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5260-5262</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of small mols. to regulate cellular levels of specific proteins is poised to become a powerful technique in the coming years.  Crit. to the success of any project using such an approach will be the ability to synthesize libraries of candidate small mols. for testing in cellular systems.  Herein, the authors describe a practical synthesis of a phthalimide-based scaffold, which can be easily diversified to make Cereblon-targeting PROTACs.  The authors demonstrate the effectiveness of this approach by synthesizing a 'PROTAC toolbox' of four amines which can be coupled to inhibitors in a straightforward manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo01AK2Qjj97Vg90H21EOLACvtfcHk0lggP1ZrOIwYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI&md5=864f4c671ef2c8d43191fc6a6e22b3b2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLohbeck%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%2BK.%26atitle%3DPractical%2520synthesis%2520of%2520a%2520phthalimide-based%2520cereblon%2520ligand%2520to%2520enable%2520PROTAC%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5260%26epage%3D5262%26doi%3D10.1016%2Fj.bmcl.2016.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><div class="note"><p class="first last">For a recent review on azide-alkyne cycloaddition-click chemistry, see:</p></div><span class="NLM_contrib-group">Singh, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koley, S.</span><span> </span><span class="NLM_article-title">Advances of azide-alkyne cycloaddition-click chemistry over the recent decade</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">5257</span><span class="NLM_x">–</span> <span class="NLM_lpage">5283</span><span class="refDoi"> DOI: 10.1016/j.tet.2016.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.tet.2016.07.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtF2hsbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=5257-5283&author=M.+S.+Singhauthor=S.+Chowdhuryauthor=S.+Koley&title=Advances+of+azide-alkyne+cycloaddition-click+chemistry+over+the+recent+decade&doi=10.1016%2Fj.tet.2016.07.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Advances of azide-alkyne cycloaddition-click chemistry over the recent decade</span></div><div class="casAuthors">Singh, Maya Shankar; Chowdhury, Sushobhan; Koley, Suvajit</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5257-5283</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  This review presents an overview of the open literature on the development of azide-dipolarophile 1,3-dipolar cycloaddn. chem. over the recent years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9L_G6zGkGkrVg90H21EOLACvtfcHk0lhtFXkRWmooKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtF2hsbvN&md5=af0463fba34ef4fe7d7239ad8dc90256</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DM.%2BS.%26aulast%3DChowdhury%26aufirst%3DS.%26aulast%3DKoley%26aufirst%3DS.%26atitle%3DAdvances%2520of%2520azide-alkyne%2520cycloaddition-click%2520chemistry%2520over%2520the%2520recent%2520decade%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D5257%26epage%3D5283%26doi%3D10.1016%2Fj.tet.2016.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><div class="note"><p class="first last">For a recent report on the use of click chemistry to prepare analogues of thalidomide, see:</p></div><span class="NLM_contrib-group">Ronnebaum, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzzio, F. A.</span><span> </span><span class="NLM_article-title">Synthesis of 1,2,3-triazole “click” analogues of thalidomide</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">6136</span><span class="NLM_x">–</span> <span class="NLM_lpage">6141</span><span class="refDoi"> DOI: 10.1016/j.tet.2016.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.tet.2016.07.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=6136-6141&author=J.+M.+Ronnebaumauthor=F.+A.+Luzzio&title=Synthesis+of+1%2C2%2C3-triazole+%E2%80%9Cclick%E2%80%9D+analogues+of+thalidomide&doi=10.1016%2Fj.tet.2016.07.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DRonnebaum%26aufirst%3DJ.%2BM.%26aulast%3DLuzzio%26aufirst%3DF.%2BA.%26atitle%3DSynthesis%2520of%25201%252C2%252C3-triazole%2520%25E2%2580%259Cclick%25E2%2580%259D%2520analogues%2520of%2520thalidomide%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D6136%26epage%3D6141%26doi%3D10.1016%2Fj.tet.2016.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><div class="note"><p class="first last">The thalidomide motif was used as a racemate because it is known that its center racemizes in vivo, see for example:</p></div><span class="NLM_contrib-group">Tian, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span> </span><span class="NLM_article-title">Enantiomerization mechanism of thalidomide and the role of water and hydroxide ions</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">14305</span><span class="NLM_x">–</span> <span class="NLM_lpage">14313</span><span class="refDoi"> DOI: 10.1002/chem.201202651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fchem.201202651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=23065668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rtr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=14305-14313&author=C.+Tianauthor=P.+Xiuauthor=Y.+Mengauthor=W.+Zhaoauthor=Z.+Wangauthor=R.+Zhou&title=Enantiomerization+mechanism+of+thalidomide+and+the+role+of+water+and+hydroxide+ions&doi=10.1002%2Fchem.201202651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Enantiomerization Mechanism of Thalidomide and the Role of Water and Hydroxide Ions</span></div><div class="casAuthors">Tian, Chuanjin; Xiu, Peng; Meng, Yan; Zhao, Wenyan; Wang, Zhigang; Zhou, Ruhong</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">14305-14313, S14305/1-S14305/24</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The significance of the mol. chirality of drugs has been widely recognized due to the thalidomide tragedy.  Most of the new drugs reaching the market today are single enantiomers, rather than racemic mixts.  However, many optically pure drugs, including thalidomide, undergo enantiomerization in vivo, thus negating the single enantiomers' benefits or inducing unexpected effects.  A detailed at. level understanding of chiral conversion, which is still largely lacking, is thus crit. for drug development.  Herein, we use first-principle d. function theory (DFT) to explore the mechanism of enantiomerization of thalidomide.  We have identified the two most plausible interconversion pathways for isolated thalidomide: (1) proton transfer from the chiral carbon center to an adjacent carbonyl oxygen atom, followed by isomerization and rotation of the glutarimide ring (before the proton hops back to the chiral carbon atom); and (2) a pathway that is the same as "1", but with the isomerization of the glutarimide ring occurring ahead of the initial proton transfer reaction.  There are two remarkable energy barriers, 73.29 and 23.59 kcal mol-1, corresponding to the proton transfer and the rotation of the glutarimide ring, resp.  Furthermore, we found that water effectively catalyzes the interconversion by facilitating the proton transfer with the highest energy barrier falling to approx. 30 kcal mol-1, which, to our knowledge, is the first time that this important role of water in chiral conversion has been demonstrated.  Finally, we show that the hydroxide ion can further lower the enantiomerization energy barrier to approx. 24 kcal mol-1 by facilitating proton abstraction, which agrees well with recent exptl. data under basic conditions.  Our current findings highlight the importance of water and hydroxide ions in the enantiomerization of thalidomide and also provide new insights into the mechanism of enantiomerization at an at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtc9xvH0R_Z7Vg90H21EOLACvtfcHk0lhtFXkRWmooKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rtr%252FK&md5=a169947388b8463fc11dcdb1e546c0e3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fchem.201202651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201202651%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DC.%26aulast%3DXiu%26aufirst%3DP.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DR.%26atitle%3DEnantiomerization%2520mechanism%2520of%2520thalidomide%2520and%2520the%2520role%2520of%2520water%2520and%2520hydroxide%2520ions%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2012%26volume%3D18%26spage%3D14305%26epage%3D14313%26doi%3D10.1002%2Fchem.201202651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a> for details of the synthesis of this library.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>For more information, see: <a href="http://www.perkinelmer.com" class="extLink">http://www.perkinelmer.com</a> (accessed February 8,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information%2C+see%3A+http%3A%2F%2Fwww.perkinelmer.com+%28accessed+February+8%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="note"><p class="first last">The NCI-H661 cell line is a human large cell lung carcinoma cell line and found to have excellent levels of activity for both VHL and cereblon ligases and is also adherent making this cell line ideal for the MSD assay format.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="note"><p class="first last">Similar data was also generated for a NCI-H838 cell line and this data is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Douglass, E. F.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, D. A.</span><span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6092</span><span class="NLM_x">–</span> <span class="NLM_lpage">6099</span><span class="refDoi"> DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0lhtFXkRWmooKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="note"><p class="first last">These PROTACs also showed similar activities in the NCI-H838 cell line (nonsmall cell lung carcinoma) see: <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Table 1</a> in the Supporting Information.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Meiners, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stangl, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloetzel, P.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, E.</span><span> </span><span class="NLM_article-title">Inhibition of proteasome activity induces concerted expression of proteasome genes and de novo formation of mammalian proteasomes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">21517</span><span class="NLM_x">–</span> <span class="NLM_lpage">21525</span><span class="refDoi"> DOI: 10.1074/jbc.M301032200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1074%2Fjbc.M301032200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=12676932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVemsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=21517-21525&author=S.+Meinersauthor=D.+Heykenauthor=A.+Wellerauthor=A.+Ludwigauthor=K.+Stanglauthor=P.-M.+Kloetzelauthor=E.+Kr%C3%BCger&title=Inhibition+of+proteasome+activity+induces+concerted+expression+of+proteasome+genes+and+de+novo+formation+of+mammalian+proteasomes&doi=10.1074%2Fjbc.M301032200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Proteasome Activity Induces Concerted Expression of Proteasome Genes and de Novo Formation of Mammalian Proteasomes</span></div><div class="casAuthors">Meiners, Silke; Heyken, Dirk; Weller, Andrea; Ludwig, Antje; Stangl, Karl; Kloetzel, Peter-M.; Krueger, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">21517-21525</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The 26 S proteasome is a high mol. mass proteinase complex that is built by at least 32 different protein subunits.  Such protease complexes in bacteria and yeast are systems that undergo a highly sophisticated network of gene expression regulation.  However, regulation of mammalian proteasome gene expression has been neglected so far as a possible control mechanism for the amt. of proteasomes in the cell.  Here, we show that treatment of cells with proteasome inhibitors and the concomitant impairment of proteasomal enzyme activity induce a transient and concerted up-regulation of all mammalian 26 S proteasome subunit mRNAs.  Proteasome inhibition in combination with inhibition of transcription revealed that the obsd. up-regulation is mediated by coordinated transcriptional activation of the proteasome genes and not by post-transcriptional events.  Our expts. also demonstrate that inhibitor-induced proteasome gene activation results in enhanced de novo protein synthesis of all subunits and in increased de novo formation of proteasomes.  This phenomenon is accompanied by enhanced expression of the proteasome maturation factor POMP.  Thus, our expts. present the first evidence that the amt. of proteasomes in mammalia is regulated at the transcriptional level and that there exists an autoregulatory feedback mechanism that allows the compensation of reduced proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbyNxj9qhUXLVg90H21EOLACvtfcHk0lh-oXjkyukL5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVemsrk%253D&md5=2dba4af89533fe1720299bef5653e451</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301032200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301032200%26sid%3Dliteratum%253Aachs%26aulast%3DMeiners%26aufirst%3DS.%26aulast%3DHeyken%26aufirst%3DD.%26aulast%3DWeller%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DA.%26aulast%3DStangl%26aufirst%3DK.%26aulast%3DKloetzel%26aufirst%3DP.-M.%26aulast%3DKr%25C3%25BCger%26aufirst%3DE.%26atitle%3DInhibition%2520of%2520proteasome%2520activity%2520induces%2520concerted%2520expression%2520of%2520proteasome%2520genes%2520and%2520de%2520novo%2520formation%2520of%2520mammalian%2520proteasomes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D21517%26epage%3D21525%26doi%3D10.1074%2Fjbc.M301032200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a> for the comprehensive experimental procedures and for NMR spectra.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':[],'ref22':['cit22'],'ref23':[],'ref24':[],'ref25':['cit25'],'ref26':[],'ref27':['cit27'],'ref28':[]}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 64 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Troy A. Bemis, James J. La Clair, <span class="NLM_string-name hlFld-ContribAuthor">Michael D. Burkart</span>. </span><span class="cited-content_cbyCitation_article-title">Unraveling the Role of Linker Design in Proteolysis Targeting Chimeras. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8042-8052. <a href="https://doi.org/10.1021/acs.jmedchem.1c00482" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00482%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUnraveling%252Bthe%252BRole%252Bof%252BLinker%252BDesign%252Bin%252BProteolysis%252BTargeting%252BChimeras%26aulast%3DBemis%26aufirst%3DTroy%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16032021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8042%26epage%3D8052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jeremy S. Disch, Jennifer M. Duffy, Esther C. Y. Lee, Diana Gikunju, Betty Chan, Benjamin Levin, Michael I. Monteiro, Sarah A. Talcott, Anthony C. Lau, Fei Zhou, Anton Kozhushnyan, Neil E. Westlund, Patrick B. Mullins, Yan Yu, Moritz von Rechenberg, Junyi Zhang, Yelena A. Arnautova, Yanbin Liu, Ying Zhang, Andrew J. McRiner, Anthony D. Keefe, Anna Kohlmann, Matthew A. Clark, John W. Cuozzo, Christelle Huguet, <span class="NLM_string-name hlFld-ContribAuthor">Shilpi Arora</span>. </span><span class="cited-content_cbyCitation_article-title">Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 5049-5066. <a href="https://doi.org/10.1021/acs.jmedchem.1c00127" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00127%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBispecific%252BEstrogen%252BReceptor%252B%2525CE%2525B1%252BDegraders%252BIncorporating%252BNovel%252BBinders%252BIdentified%252BUsing%252BDNA-Encoded%252BChemical%252BLibrary%252BScreening%26aulast%3DDisch%26aufirst%3DJeremy%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D22012021%26date%3D12042021%26volume%3D64%26issue%3D8%26spage%3D5049%26epage%3D5066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jun Young Hong, Hui Jing, Ian Robert Price, Ji Cao, Jessica Jingyi Bai, <span class="NLM_string-name hlFld-ContribAuthor">Hening Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Simultaneous Inhibition of SIRT2 Deacetylase and Defatty-Acylase Activities via a PROTAC Strategy. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 2305-2311. <a href="https://doi.org/10.1021/acsmedchemlett.0c00423" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00423</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00423%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSimultaneous%252BInhibition%252Bof%252BSIRT2%252BDeacetylase%252Band%252BDefatty-Acylase%252BActivities%252Bvia%252Ba%252BPROTAC%252BStrategy%26aulast%3DHong%26aufirst%3DJun%2BYoung%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02082020%26date%3D21092020%26date%3D28092020%26date%3D21092020%26volume%3D11%26issue%3D11%26spage%3D2305%26epage%3D2311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Laura Goracci, Jenny Desantis, Aurora Valeri, Beatrice Castellani, Michela Eleuteri, <span class="NLM_string-name hlFld-ContribAuthor">Gabriele Cruciani</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding the Metabolism of Proteolysis Targeting Chimeras (PROTACs): The Next Step toward Pharmaceutical Applications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (20)
                                     , 11615-11638. <a href="https://doi.org/10.1021/acs.jmedchem.0c00793" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00793%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUnderstanding%252Bthe%252BMetabolism%252Bof%252BProteolysis%252BTargeting%252BChimeras%252B%252528PROTACs%252529%25253A%252BThe%252BNext%252BStep%252Btoward%252BPharmaceutical%252BApplications%26aulast%3DGoracci%26aufirst%3DLaura%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D07102020%26volume%3D63%26issue%3D20%26spage%3D11615%26epage%3D11638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sachini U. Siriwardena, Dhanushka N. P. Munkanatta Godage, Veronika M. Shoba, Sophia Lai, Mengchao Shi, Peng Wu, Santosh K. Chaudhary, Stuart L. Schreiber, <span class="NLM_string-name hlFld-ContribAuthor">Amit Choudhary</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphorylation-Inducing Chimeric Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (33)
                                     , 14052-14057. <a href="https://doi.org/10.1021/jacs.0c05537" title="DOI URL">https://doi.org/10.1021/jacs.0c05537</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c05537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c05537%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DPhosphorylation-Inducing%252BChimeric%252BSmall%252BMolecules%26aulast%3DSiriwardena%26aufirst%3DSachini%2BU.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20052020%26date%3D04082020%26volume%3D142%26issue%3D33%26spage%3D14052%26epage%3D14057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mingliang Wang, Jianfeng Lu, Mi Wang, Chao-Yie Yang, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7510-7528. <a href="https://doi.org/10.1021/acs.jmedchem.0c00471" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00471%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSHP2-D26%252Bas%252Ba%252BFirst%25252C%252BPotent%25252C%252Band%252BEffective%252BPROTAC%252BDegrader%252Bof%252BSHP2%252BProtein%26aulast%3DWang%26aufirst%3DMingliang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20032020%26date%3D21052020%26volume%3D63%26issue%3D14%26spage%3D7510%26epage%3D7528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brett L. Roberts, Zhi-Xiong Ma, Ang Gao, Eric D. Leisten, Dan Yin, Wei Xu, <span class="NLM_string-name hlFld-ContribAuthor">Weiping Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (6)
                                     , 1487-1496. <a href="https://doi.org/10.1021/acschembio.0c00140" title="DOI URL">https://doi.org/10.1021/acschembio.0c00140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00140%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DTwo-Stage%252BStrategy%252Bfor%252BDevelopment%252Bof%252BProteolysis%252BTargeting%252BChimeras%252Band%252Bits%252BApplication%252Bfor%252BEstrogen%252BReceptor%252BDegraders%26aulast%3DRoberts%26aufirst%3DBrett%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26022020%26date%3D07042020%26date%3D16042020%26date%3D07042020%26volume%3D15%26issue%3D6%26spage%3D1487%26epage%3D1496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xi Xu, Zijian Kuang, Jie Han, Ying Meng, Lei Li, Hongyu Luan, Pengfei Xu, Jubo Wang, Cheng Luo, Hong Ding, Zhiyu Li, <span class="NLM_string-name hlFld-ContribAuthor">Jinlei Bian</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Characterization of a Fluorescent Probe for GLS1 and the Application for High-Throughput Screening of Allosteric Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9642-9657. <a href="https://doi.org/10.1021/acs.jmedchem.9b01035" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01035%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Band%252BCharacterization%252Bof%252Ba%252BFluorescent%252BProbe%252Bfor%252BGLS1%252Band%252Bthe%252BApplication%252Bfor%252BHigh-Throughput%252BScreening%252Bof%252BAllosteric%252BInhibitors%26aulast%3DXu%26aufirst%3DXi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03072019%26date%3D24102019%26date%3D11102019%26volume%3D62%26issue%3D21%26spage%3D9642%26epage%3D9657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Junting Zhou, Xueyang Jiang, Siyu He, Hongli Jiang, Feng Feng, Wenyuan Liu, Wei Qu, <span class="NLM_string-name hlFld-ContribAuthor">Haopeng Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design of Multitarget-Directed Ligands: Strategies and Emerging Paradigms. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 8881-8914. <a href="https://doi.org/10.1021/acs.jmedchem.9b00017" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00017%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Bof%252BMultitarget-Directed%252BLigands%25253A%252BStrategies%252Band%252BEmerging%252BParadigms%26aulast%3DZhou%26aufirst%3DJunting%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04012019%26date%3D28052019%26date%3D13052019%26volume%3D62%26issue%3D20%26spage%3D8881%26epage%3D8914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shang Su, Zimo Yang, Hongying Gao, Haiyan Yang, Songbiao Zhu, Zixuan An, Juanjuan Wang, Qing Li, Sarat Chandarlapaty, Haiteng Deng, Wei Wu, <span class="NLM_string-name hlFld-ContribAuthor">Yu Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Potent and Preferential Degradation of CDK6 via Proteolysis Targeting Chimera Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (16)
                                     , 7575-7582. <a href="https://doi.org/10.1021/acs.jmedchem.9b00871" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00871</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00871%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%252Band%252BPreferential%252BDegradation%252Bof%252BCDK6%252Bvia%252BProteolysis%252BTargeting%252BChimera%252BDegraders%26aulast%3DSu%26aufirst%3DShang%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29052019%26date%3D02082019%26date%3D22072019%26volume%3D62%26issue%3D16%26spage%3D7575%26epage%3D7582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">James
W. Papatzimas, Evgueni Gorobets, Ranjan Maity, Mir Ishruna Muniyat, Justin L. MacCallum, Paola Neri, Nizar J. Bahlis, <span class="NLM_string-name hlFld-ContribAuthor">Darren J. Derksen</span>. </span><span class="cited-content_cbyCitation_article-title">From Inhibition to Degradation: Targeting the Antiapoptotic Protein Myeloid Cell Leukemia 1 (MCL1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (11)
                                     , 5522-5540. <a href="https://doi.org/10.1021/acs.jmedchem.9b00455" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00455</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00455%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFrom%252BInhibition%252Bto%252BDegradation%25253A%252BTargeting%252Bthe%252BAntiapoptotic%252BProtein%252BMyeloid%252BCell%252BLeukemia%252B1%252B%252528MCL1%252529%26aulast%3DPapatzimas%26aufirst%3DJames%2BW.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D04062019%26date%3D22052019%26volume%3D62%26issue%3D11%26spage%3D5522%26epage%3D5540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kristin
M. Riching, Sarah Mahan, Cesear R. Corona, Mark McDougall, James D. Vasta, Matthew B. Robers, Marjeta Urh, <span class="NLM_string-name hlFld-ContribAuthor">Danette L. Daniels</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Live-Cell Kinetic Degradation and Mechanistic Profiling of PROTAC Mode of Action. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (9)
                                     , 2758-2770. <a href="https://doi.org/10.1021/acschembio.8b00692" title="DOI URL">https://doi.org/10.1021/acschembio.8b00692</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00692%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DQuantitative%252BLive-Cell%252BKinetic%252BDegradation%252Band%252BMechanistic%252BProfiling%252Bof%252BPROTAC%252BMode%252Bof%252BAction%26aulast%3DRiching%26aufirst%3DKristin%2BM.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D25072018%26date%3D23082018%26date%3D30082018%26date%3D23082018%26volume%3D13%26issue%3D9%26spage%3D2758%26epage%3D2770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian Steinebach, Stefanie Lindner, Namrata D. Udeshi, Deepak C. Mani, Hannes Kehm, Simon Köpff, Steven A. Carr, Michael Gütschow, <span class="NLM_string-name hlFld-ContribAuthor">Jan Krönke</span>. </span><span class="cited-content_cbyCitation_article-title">Homo-PROTACs for the Chemical Knockdown of Cereblon. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (9)
                                     , 2771-2782. <a href="https://doi.org/10.1021/acschembio.8b00693" title="DOI URL">https://doi.org/10.1021/acschembio.8b00693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00693%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DHomo-PROTACs%252Bfor%252Bthe%252BChemical%252BKnockdown%252Bof%252BCereblon%26aulast%3DSteinebach%26aufirst%3DChristian%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D25072018%26date%3D17082018%26date%3D05092018%26date%3D17082018%26volume%3D13%26issue%3D9%26spage%3D2771%26epage%3D2782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kwok-Ho  Chan</span>, <span class="hlFld-ContribAuthor ">Michael  Zengerle</span>, <span class="hlFld-ContribAuthor ">Andrea  Testa</span>, and <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>  . </span><span class="cited-content_cbyCitation_article-title">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (2)
                                     , 504-513. <a href="https://doi.org/10.1021/acs.jmedchem.6b01912" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01912</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImpact%252Bof%252BTarget%252BWarhead%252Band%252BLinkage%252BVector%252Bon%252BInducing%252BProtein%252BDegradation%25253A%252BComparison%252Bof%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BDegraders%252BDerived%252Bfrom%252BTriazolodiazepine%252B%252528JQ1%252529%252Band%252BTetrahydroquinoline%252B%252528I-BET726%252529%252BBET%252BInhibitor%252BScaffolds%26aulast%3DChan%26aufirst%3DKwok-Ho%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D04012017%26date%3D22062017%26date%3D25012018%26date%3D08062017%26volume%3D61%26issue%3D2%26spage%3D504%26epage%3D513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro  Soares</span>, <span class="hlFld-ContribAuthor ">Morgan S.  Gadd</span>, <span class="hlFld-ContribAuthor ">Julianty  Frost</span>, <span class="hlFld-ContribAuthor ">Carles  Galdeano</span>, <span class="hlFld-ContribAuthor ">Lucy  Ellis</span>, <span class="hlFld-ContribAuthor ">Ola  Epemolu</span>, <span class="hlFld-ContribAuthor ">Sonia  Rocha</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Read</span>, and <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>  . </span><span class="cited-content_cbyCitation_article-title">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel–Lindau (VHL) E3 Ubiquitin Ligase: Structure–Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (2)
                                     , 599-618. <a href="https://doi.org/10.1021/acs.jmedchem.7b00675" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00675</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGroup-Based%252BOptimization%252Bof%252BPotent%252Band%252BCell-Active%252BInhibitors%252Bof%252Bthe%252Bvon%252BHippel%2525E2%252580%252593Lindau%252B%252528VHL%252529%252BE3%252BUbiquitin%252BLigase%25253A%252BStructure%2525E2%252580%252593Activity%252BRelationships%252BLeading%252Bto%252Bthe%252BChemical%252BProbe%252B%2525282S%25252C4R%252529-1-%252528%252528S%252529-2-%2525281-Cyanocyclopropanecarboxamido%252529-3%25252C3-dimethylbutanoyl%252529-4-hydroxy-N-%2525284-%2525284-methylthiazol-5-yl%252529benzyl%252529pyrrolidine-2-carboxamide%252B%252528VH298%252529%26aulast%3DSoares%26aufirst%3DPedro%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D06052017%26date%3D18092017%26date%3D25012018%26date%3D30082017%26volume%3D61%26issue%3D2%26spage%3D599%26epage%3D618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Craig M.  Crews</span> (<span class="NLM_role">Guest Editor</span>). </span><span class="cited-content_cbyCitation_article-title">Inducing Protein Degradation as a Therapeutic Strategy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (2)
                                     , 403-404. <a href="https://doi.org/10.1021/acs.jmedchem.7b01333" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01333</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01333%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInducing%252BProtein%252BDegradation%252Bas%252Ba%252BTherapeutic%252BStrategy%26aulast%3DCrews%26aufirst%3DCraig%2BM.%26date%3D2018%26date%3D2017%26date%3D13092017%26date%3D22112017%26date%3D25012018%26volume%3D61%26issue%3D2%26spage%3D403%26epage%3D404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span> and <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>  . </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Modulation of Protein Homeostasis. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2017,</strong> <em>117 </em>
                                    (17)
                                     , 11269-11301. <a href="https://doi.org/10.1021/acs.chemrev.7b00077" title="DOI URL">https://doi.org/10.1021/acs.chemrev.7b00077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DSmall-Molecule%252BModulation%252Bof%252BProtein%252BHomeostasis%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D02022017%26date%3D04082017%26date%3D13092017%26volume%3D117%26issue%3D17%26spage%3D11269%26epage%3D11301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shusuke  Tomoshige</span>, <span class="hlFld-ContribAuthor ">Minoru  Ishikawa</span>. </span><span class="cited-content_cbyCitation_article-title">In vivo synthetic chemistry of proteolysis targeting chimeras (PROTACs). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>41 </em>, 116221. <a href="https://doi.org/10.1016/j.bmc.2021.116221" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116221%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DIn%252Bvivo%252Bsynthetic%252Bchemistry%252Bof%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%26aulast%3DTomoshige%26aufirst%3DShusuke%26date%3D2021%26volume%3D41%26spage%3D116221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian-Jia  Liang</span>, <span class="hlFld-ContribAuthor ">Hang  Xie</span>, <span class="hlFld-ContribAuthor ">Rui-Hua  Yang</span>, <span class="hlFld-ContribAuthor ">Ni  Wang</span>, <span class="hlFld-ContribAuthor ">Zi-Jun  Zheng</span>, <span class="hlFld-ContribAuthor ">Chen  Zhou</span>, <span class="hlFld-ContribAuthor ">Ya-Lei  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Li-Hong  Shan</span>, <span class="hlFld-ContribAuthor ">Yu  Ke</span>. </span><span class="cited-content_cbyCitation_article-title">Designed, synthesized and biological evaluation of proteolysis targeting chimeras (PROTACs) as AR degraders for prostate cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>120 </em>, 116331. <a href="https://doi.org/10.1016/j.bmc.2021.116331" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116331</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116331%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesigned%25252C%252Bsynthesized%252Band%252Bbiological%252Bevaluation%252Bof%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bas%252BAR%252Bdegraders%252Bfor%252Bprostate%252Bcancer%252Btreatment%26aulast%3DLiang%26aufirst%3DJian-Jia%26date%3D2021%26volume%3D120%26spage%3D116331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlotta  Cecchini</span>, <span class="hlFld-ContribAuthor ">Sara  Pannilunghi</span>, <span class="hlFld-ContribAuthor ">Sébastien  Tardy</span>, <span class="hlFld-ContribAuthor ">Leonardo  Scapozza</span>. </span><span class="cited-content_cbyCitation_article-title">From Conception to Development: Investigating PROTACs Features for Improved Cell Permeability and Successful Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.672267" title="DOI URL">https://doi.org/10.3389/fchem.2021.672267</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.672267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.672267%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFrom%252BConception%252Bto%252BDevelopment%25253A%252BInvestigating%252BPROTACs%252BFeatures%252Bfor%252BImproved%252BCell%252BPermeability%252Band%252BSuccessful%252BProtein%252BDegradation%26aulast%3DCecchini%26aufirst%3DCarlotta%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen F.  Vieux</span>, <span class="hlFld-ContribAuthor ">Roman V.  Agafonov</span>, <span class="hlFld-ContribAuthor ">Lydia  Emerson</span>, <span class="hlFld-ContribAuthor ">Marta  Isasa</span>, <span class="hlFld-ContribAuthor ">Richard W.  Deibler</span>, <span class="hlFld-ContribAuthor ">Jeffrey R.  Simard</span>, <span class="hlFld-ContribAuthor ">David  Cocozziello</span>, <span class="hlFld-ContribAuthor ">Brendon  Ladd</span>, <span class="hlFld-ContribAuthor ">Linda  Lee</span>, <span class="hlFld-ContribAuthor ">Heng  Li</span>, <span class="hlFld-ContribAuthor ">Stephen  Archer</span>, <span class="hlFld-ContribAuthor ">Mark  Fitzgerald</span>, <span class="hlFld-ContribAuthor ">Ryan  Michael</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Nasveschuk</span>, <span class="hlFld-ContribAuthor ">Eunice S.  Park</span>, <span class="hlFld-ContribAuthor ">Gunther  Kern</span>, <span class="hlFld-ContribAuthor ">David A.  Proia</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Phillips</span>, <span class="hlFld-ContribAuthor ">Stewart L.  Fisher</span>. </span><span class="cited-content_cbyCitation_article-title">A Method for Determining the Kinetics of Small-Molecule-Induced Ubiquitination. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 547-559. <a href="https://doi.org/10.1177/24725552211000673" title="DOI URL">https://doi.org/10.1177/24725552211000673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211000673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211000673%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DA%252BMethod%252Bfor%252BDetermining%252Bthe%252BKinetics%252Bof%252BSmall-Molecule-Induced%252BUbiquitination%26aulast%3DVieux%26aufirst%3DEllen%2BF.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D547%26epage%3D559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan K.  Brownsey</span>, <span class="hlFld-ContribAuthor ">Ben C.  Rowley</span>, <span class="hlFld-ContribAuthor ">Evgueni  Gorobets</span>, <span class="hlFld-ContribAuthor ">Benjamin S.  Gelfand</span>, <span class="hlFld-ContribAuthor ">Darren J.  Derksen</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid synthesis of pomalidomide-conjugates for the development of protein degrader libraries. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (12)
                                     , 4519-4525. <a href="https://doi.org/10.1039/D0SC05442A" title="DOI URL">https://doi.org/10.1039/D0SC05442A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05442A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05442A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DRapid%252Bsynthesis%252Bof%252Bpomalidomide-conjugates%252Bfor%252Bthe%252Bdevelopment%252Bof%252Bprotein%252Bdegrader%252Blibraries%26aulast%3DBrownsey%26aufirst%3DDuncan%2BK.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D12%26spage%3D4519%26epage%3D4525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne A.  Nagel</span>, <span class="hlFld-ContribAuthor ">Adrian  Britschgi</span>, <span class="hlFld-ContribAuthor ">Antonio  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">From Degraders to Molecular Glues: New Ways of Breaking Down Disease‐Associated Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 47-85. <a href="https://doi.org/10.1002/9783527826872.ch2" title="DOI URL">https://doi.org/10.1002/9783527826872.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527826872.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527826872.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BDegraders%252Bto%252BMolecular%252BGlues%25253A%252BNew%252BWays%252Bof%252BBreaking%252BDown%252BDisease%2525E2%252580%252590Associated%252BProteins%26aulast%3DNagel%26aufirst%3DYvonne%2BA.%26date%3D2021%26date%3D2021%26spage%3D47%26epage%3D85%26pub%3DWiley%26atitle%3DSuccessful%252BDrug%252BDiscovery%26aulast%3DFischer%26aufirst%3DJ%25C3%25A1nos%26date%3D2021%26date%3D2021%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoyi  Yan</span>, <span class="hlFld-ContribAuthor ">Xinxin  Zhong</span>, <span class="hlFld-ContribAuthor ">Lin  Yue</span>, <span class="hlFld-ContribAuthor ">Chunlan  Pu</span>, <span class="hlFld-ContribAuthor ">Huifang  Shan</span>, <span class="hlFld-ContribAuthor ">Suke  Lan</span>, <span class="hlFld-ContribAuthor ">Meng  Zhou</span>, <span class="hlFld-ContribAuthor ">Xueyan  Hou</span>, <span class="hlFld-ContribAuthor ">Jie  Yang</span>, <span class="hlFld-ContribAuthor ">Rui  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a PROTAC targeting ALK with in vivo activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>212 </em>, 113150. <a href="https://doi.org/10.1016/j.ejmech.2020.113150" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113150%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPROTAC%252Btargeting%252BALK%252Bwith%252Bin%2525C2%2525A0vivo%252Bactivity%26aulast%3DYan%26aufirst%3DGuoyi%26date%3D2021%26volume%3D212%26spage%3D113150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Troy A.  Bemis</span>, <span class="hlFld-ContribAuthor ">James J.  La Clair</span>, <span class="hlFld-ContribAuthor ">Michael D.  Burkart</span>. </span><span class="cited-content_cbyCitation_article-title">Traceless Staudinger ligation enabled parallel synthesis of proteolysis targeting chimera linker variants. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2021,</strong> <em>57 </em>
                                    (8)
                                     , 1026-1029. <a href="https://doi.org/10.1039/D0CC05395C" title="DOI URL">https://doi.org/10.1039/D0CC05395C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CC05395C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CC05395C%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DTraceless%252BStaudinger%252Bligation%252Benabled%252Bparallel%252Bsynthesis%252Bof%252Bproteolysis%252Btargeting%252Bchimera%252Blinker%252Bvariants%26aulast%3DBemis%26aufirst%3DTroy%2BA.%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D8%26spage%3D1026%26epage%3D1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haixia  Liu</span>, <span class="hlFld-ContribAuthor ">Renhong  Sun</span>, <span class="hlFld-ContribAuthor ">Chaowei  Ren</span>, <span class="hlFld-ContribAuthor ">Xing  Qiu</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Biao  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Construction of an IMiD-based azide library as a kit for PROTAC research. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (1)
                                     , 166-170. <a href="https://doi.org/10.1039/D0OB02120B" title="DOI URL">https://doi.org/10.1039/D0OB02120B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB02120B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB02120B%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DConstruction%252Bof%252Ban%252BIMiD-based%252Bazide%252Blibrary%252Bas%252Ba%252Bkit%252Bfor%252BPROTAC%252Bresearch%26aulast%3DLiu%26aufirst%3DHaixia%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D1%26spage%3D166%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryosuke  Shirasaki</span>, <span class="hlFld-ContribAuthor ">Geoffrey M.  Matthews</span>, <span class="hlFld-ContribAuthor ">Sara  Gandolfi</span>, <span class="hlFld-ContribAuthor ">Ricardo  de Matos Simoes</span>, <span class="hlFld-ContribAuthor ">Dennis L.  Buckley</span>, <span class="hlFld-ContribAuthor ">Joseline  Raja Vora</span>, <span class="hlFld-ContribAuthor ">Quinlan L.  Sievers</span>, <span class="hlFld-ContribAuthor ">Johanna B.  Brüggenthies</span>, <span class="hlFld-ContribAuthor ">Olga  Dashevsky</span>, <span class="hlFld-ContribAuthor ">Haley  Poarch</span>, <span class="hlFld-ContribAuthor ">Huihui  Tang</span>, <span class="hlFld-ContribAuthor ">Megan A.  Bariteau</span>, <span class="hlFld-ContribAuthor ">Michal  Sheffer</span>, <span class="hlFld-ContribAuthor ">Yiguo  Hu</span>, <span class="hlFld-ContribAuthor ">Sondra L.  Downey-Kopyscinski</span>, <span class="hlFld-ContribAuthor ">Paul J.  Hengeveld</span>, <span class="hlFld-ContribAuthor ">Brian J.  Glassner</span>, <span class="hlFld-ContribAuthor ">Eugen  Dhimolea</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Ott</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Nicholas P.  Kwiatkowski</span>, <span class="hlFld-ContribAuthor ">Jacob P.  Laubach</span>, <span class="hlFld-ContribAuthor ">Robert L.  Schlossman</span>, <span class="hlFld-ContribAuthor ">Paul G.  Richardson</span>, <span class="hlFld-ContribAuthor ">Aedin C.  Culhane</span>, <span class="hlFld-ContribAuthor ">Richard W.J.  Groen</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Francisca  Vazquez</span>, <span class="hlFld-ContribAuthor ">Aviad  Tsherniak</span>, <span class="hlFld-ContribAuthor ">William C.  Hahn</span>, <span class="hlFld-ContribAuthor ">Joan  Levy</span>, <span class="hlFld-ContribAuthor ">Daniel  Auclair</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Licht</span>, <span class="hlFld-ContribAuthor ">Jonathan J.  Keats</span>, <span class="hlFld-ContribAuthor ">Lawrence H.  Boise</span>, <span class="hlFld-ContribAuthor ">Benjamin L.  Ebert</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>, <span class="hlFld-ContribAuthor ">Constantine S.  Mitsiades</span>. </span><span class="cited-content_cbyCitation_article-title">Functional Genomics Identify Distinct and Overlapping Genes Mediating Resistance to Different Classes of Heterobifunctional Degraders of Oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2021,</strong> <em>34 </em>
                                    (1)
                                     , 108532. <a href="https://doi.org/10.1016/j.celrep.2020.108532" title="DOI URL">https://doi.org/10.1016/j.celrep.2020.108532</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2020.108532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2020.108532%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DFunctional%252BGenomics%252BIdentify%252BDistinct%252Band%252BOverlapping%252BGenes%252BMediating%252BResistance%252Bto%252BDifferent%252BClasses%252Bof%252BHeterobifunctional%252BDegraders%252Bof%252BOncoproteins%26aulast%3DShirasaki%26aufirst%3DRyosuke%26date%3D2021%26volume%3D34%26issue%3D1%26spage%3D108532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Ma</span>, <span class="hlFld-ContribAuthor ">Xiao-Xiao  Han</span>, <span class="hlFld-ContribAuthor ">Xiao-Ming  Yang</span>, <span class="hlFld-ContribAuthor ">Song-Lin  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera technology: a novel strategy for treating diseases of the central nervous system. </span><span class="cited-content_cbyCitation_journal-name">Neural Regeneration Research</span><span> <strong>2021,</strong> <em>16 </em>
                                    (10)
                                     , 1944. <a href="https://doi.org/10.4103/1673-5374.308075" title="DOI URL">https://doi.org/10.4103/1673-5374.308075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4103/1673-5374.308075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4103%2F1673-5374.308075%26sid%3Dliteratum%253Aachs%26jtitle%3DNeural%2520Regeneration%2520Research%26atitle%3DProteolysis%252Btargeting%252Bchimera%252Btechnology%25253A%252Ba%252Bnovel%252Bstrategy%252Bfor%252Btreating%252Bdiseases%252Bof%252Bthe%252Bcentral%252Bnervous%252Bsystem%26aulast%3DMa%26aufirst%3DKe%26date%3D2021%26volume%3D16%26issue%3D10%26spage%3D1944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marta  Serafini</span>, <span class="hlFld-ContribAuthor ">Tracey  Pirali</span>, <span class="hlFld-ContribAuthor ">Gian Cesare  Tron</span>. </span><span class="cited-content_cbyCitation_article-title">Click 1,2,3-triazoles in drug discovery and development: From the flask to the clinic?. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 101-148. <a href="https://doi.org/10.1016/bs.aihch.2020.10.001" title="DOI URL">https://doi.org/10.1016/bs.aihch.2020.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.aihch.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.aihch.2020.10.001%26sid%3Dliteratum%253Aachs%26atitle%3DClick%252B1%25252C2%25252C3-triazoles%252Bin%252Bdrug%252Bdiscovery%252Band%252Bdevelopment%25253A%252BFrom%252Bthe%252Bflask%252Bto%252Bthe%252Bclinic%25253F%26aulast%3DSerafini%26aufirst%3DMarta%26date%3D2021%26spage%3D101%26epage%3D148%26pub%3DElsevier%26atitle%3DApplications%252Bof%252BHeterocycles%252Bin%252Bthe%252BDesign%252Bof%252BDrugs%252Band%252BAgricultural%252BProducts%26date%3D2021%26volume%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Beichen  Hu</span>, <span class="hlFld-ContribAuthor ">Yirong  Zhou</span>, <span class="hlFld-ContribAuthor ">Dejuan  Sun</span>, <span class="hlFld-ContribAuthor ">Yueying  Yang</span>, <span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Xingzhou  Li</span>, <span class="hlFld-ContribAuthor ">Hua  Li</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: New method to degrade transcription regulating proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112698. <a href="https://doi.org/10.1016/j.ejmech.2020.112698" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112698</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112698%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPROTACs%25253A%252BNew%252Bmethod%252Bto%252Bdegrade%252Btranscription%252Bregulating%252Bproteins%26aulast%3DHu%26aufirst%3DBeichen%26date%3D2020%26volume%3D207%26spage%3D112698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112750. <a href="https://doi.org/10.1016/j.ejmech.2020.112750" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bepigenetic%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528Epi-PROTACs%252529%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26volume%3D207%26spage%3D112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00885-3" title="DOI URL">https://doi.org/10.1186/s13045-020-00885-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00885-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00885-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DProteolysis-targeting%252Bchimera%252B%252528PROTAC%252529%252Bfor%252Btargeted%252Bprotein%252Bdegradation%252Band%252Bcancer%252Btherapy%26aulast%3DLi%26aufirst%3DXin%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>, <span class="hlFld-ContribAuthor ">Guoshun  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting estrogen receptor α for degradation with PROTACs: A promising approach to overcome endocrine resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112689. <a href="https://doi.org/10.1016/j.ejmech.2020.112689" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112689%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bestrogen%252Breceptor%252B%2525CE%2525B1%252Bfor%252Bdegradation%252Bwith%252BPROTACs%25253A%252BA%252Bpromising%252Bapproach%252Bto%252Bovercome%252Bendocrine%252Bresistance%26aulast%3DLin%26aufirst%3DXin%26date%3D2020%26volume%3D206%26spage%3D112689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert I.  Troup</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>, <span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>. </span><span class="cited-content_cbyCitation_article-title">Current strategies for the design of PROTAC linkers: a critical review. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (5)
                                     <a href="https://doi.org/10.37349/etat.2020.00018" title="DOI URL">https://doi.org/10.37349/etat.2020.00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00018%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DCurrent%252Bstrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BPROTAC%252Blinkers%25253A%252Ba%252Bcritical%252Breview%26aulast%3DTroup%26aufirst%3DRobert%2BI.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Craig  Donoghue</span>, <span class="hlFld-ContribAuthor ">Monica  Cubillos-Rojas</span>, <span class="hlFld-ContribAuthor ">Nuria  Gutierrez-Prat</span>, <span class="hlFld-ContribAuthor ">Carolina  Sanchez-Zarzalejo</span>, <span class="hlFld-ContribAuthor ">Xavier  Verdaguer</span>, <span class="hlFld-ContribAuthor ">Antoni  Riera</span>, <span class="hlFld-ContribAuthor ">Angel R.  Nebreda</span>. </span><span class="cited-content_cbyCitation_article-title">Optimal linker length for small molecule PROTACs that selectively target p38α and p38β for degradation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>201 </em>, 112451. <a href="https://doi.org/10.1016/j.ejmech.2020.112451" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112451</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112451%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimal%252Blinker%252Blength%252Bfor%252Bsmall%252Bmolecule%252BPROTACs%252Bthat%252Bselectively%252Btarget%252Bp38%2525CE%2525B1%252Band%252Bp38%2525CE%2525B2%252Bfor%252Bdegradation%26aulast%3DDonoghue%26aufirst%3DCraig%26date%3D2020%26volume%3D201%26spage%3D112451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anja  Vogelmann</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) for epigenetics research. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 8-16. <a href="https://doi.org/10.1016/j.cbpa.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bfor%252Bepigenetics%252Bresearch%26aulast%3DVogelmann%26aufirst%3DAnja%26date%3D2020%26volume%3D57%26spage%3D8%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dongwen  Lv</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Assays and technologies for developing proteolysis targeting chimera degraders. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>18 </em><a href="https://doi.org/10.4155/fmc-2020-0073" title="DOI URL">https://doi.org/10.4155/fmc-2020-0073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0073%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAssays%252Band%252Btechnologies%252Bfor%252Bdeveloping%252Bproteolysis%252Btargeting%252Bchimera%252Bdegraders%26aulast%3DLiu%26aufirst%3DXingui%26date%3D2020%26date%3D2020%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dinesh  Thummuri</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Wanyi  Hu</span>, <span class="hlFld-ContribAuthor ">Peiyi  Zhang</span>, <span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of PROTAC BCL-XL degraders as potent anticancer agents with low on-target platelet toxicity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112186. <a href="https://doi.org/10.1016/j.ejmech.2020.112186" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112186%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPROTAC%252BBCL-XL%252Bdegraders%252Bas%252Bpotent%252Banticancer%252Bagents%252Bwith%252Blow%252Bon-target%252Bplatelet%252Btoxicity%26aulast%3DZhang%26aufirst%3DXuan%26date%3D2020%26volume%3D192%26spage%3D112186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Zhou</span>, <span class="hlFld-ContribAuthor ">Wenming  Chen</span>, <span class="hlFld-ContribAuthor ">Chenyang  Cao</span>, <span class="hlFld-ContribAuthor ">Yonghui  Shi</span>, <span class="hlFld-ContribAuthor ">Wenchong  Ye</span>, <span class="hlFld-ContribAuthor ">Jiliang  Hu</span>, <span class="hlFld-ContribAuthor ">LingLi  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of α-naphthoflavone chimera derivatives able to eliminate cytochrome P450 (CYP)1B1-mediated drug resistance via targeted CYP1B1 degradation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>189 </em>, 112028. <a href="https://doi.org/10.1016/j.ejmech.2019.112028" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.112028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.112028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.112028%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252B%2525CE%2525B1-naphthoflavone%252Bchimera%252Bderivatives%252Bable%252Bto%252Beliminate%252Bcytochrome%252BP450%252B%252528CYP%2525291B1-mediated%252Bdrug%252Bresistance%252Bvia%252Btargeted%252BCYP1B1%252Bdegradation%26aulast%3DZhou%26aufirst%3DLi%26date%3D2020%26volume%3D189%26spage%3D112028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Wang</span>, <span class="hlFld-ContribAuthor ">Xueyang  Jiang</span>, <span class="hlFld-ContribAuthor ">Feng  Feng</span>, <span class="hlFld-ContribAuthor ">Wenyuan  Liu</span>, <span class="hlFld-ContribAuthor ">Haopeng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation of proteins by PROTACs and other strategies. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (2)
                                     , 207-238. <a href="https://doi.org/10.1016/j.apsb.2019.08.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2019.08.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2019.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2019.08.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDegradation%252Bof%252Bproteins%252Bby%252BPROTACs%252Band%252Bother%252Bstrategies%26aulast%3DWang%26aufirst%3DYang%26date%3D2020%26volume%3D10%26issue%3D2%26spage%3D207%26epage%3D238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiuyao  Huang</span>, <span class="hlFld-ContribAuthor ">Yan  Zhong</span>, <span class="hlFld-ContribAuthor ">Hui  Dong</span>, <span class="hlFld-ContribAuthor ">Qiyao  Zheng</span>, <span class="hlFld-ContribAuthor ">Shuo  Shi</span>, <span class="hlFld-ContribAuthor ">Kai  Zhu</span>, <span class="hlFld-ContribAuthor ">Xinming  Qu</span>, <span class="hlFld-ContribAuthor ">Wenhao  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaolei  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuanxiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Revisiting signal transducer and activator of transcription 3 (STAT3) as an anticancer target and its inhibitor discovery: Where are we and where should we go?. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111922. <a href="https://doi.org/10.1016/j.ejmech.2019.111922" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111922</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111922%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRevisiting%252Bsignal%252Btransducer%252Band%252Bactivator%252Bof%252Btranscription%252B3%252B%252528STAT3%252529%252Bas%252Ban%252Banticancer%252Btarget%252Band%252Bits%252Binhibitor%252Bdiscovery%25253A%252BWhere%252Bare%252Bwe%252Band%252Bwhere%252Bshould%252Bwe%252Bgo%25253F%26aulast%3DHuang%26aufirst%3DQiuyao%26date%3D2020%26volume%3D187%26spage%3D111922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Noritaka  Abe</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Chemistry of Azaazulenes and Related Compounds. </span><span class="cited-content_cbyCitation_journal-name">HETEROCYCLES</span><span> <strong>2020,</strong> <em>101 </em>
                                    (1)
                                     , 33. <a href="https://doi.org/10.3987/REV-19-SR(F)1" title="DOI URL">https://doi.org/10.3987/REV-19-SR(F)1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3987/REV-19-SR(F)1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3987%2FREV-19-SR%28F%291%26sid%3Dliteratum%253Aachs%26jtitle%3DHETEROCYCLES%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BChemistry%252Bof%252BAzaazulenes%252Band%252BRelated%252BCompounds%26aulast%3DAbe%26aufirst%3DNoritaka%26date%3D2020%26volume%3D101%26issue%3D1%26spage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solvent‐exposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li-Wen  Xia</span>, <span class="hlFld-ContribAuthor ">Meng-Yu  Ba</span>, <span class="hlFld-ContribAuthor ">Wei  Liu</span>, <span class="hlFld-ContribAuthor ">Weyland  Cheng</span>, <span class="hlFld-ContribAuthor ">Chao-Ping  Hu</span>, <span class="hlFld-ContribAuthor ">Qing  Zhao</span>, <span class="hlFld-ContribAuthor ">Yong-Fang  Yao</span>, <span class="hlFld-ContribAuthor ">Mo-Ran  Sun</span>, <span class="hlFld-ContribAuthor ">Yong-Tao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Triazol: a privileged scaffold for proteolysis targeting chimeras. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (22)
                                     , 2919-2973. <a href="https://doi.org/10.4155/fmc-2019-0159" title="DOI URL">https://doi.org/10.4155/fmc-2019-0159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0159%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTriazol%25253A%252Ba%252Bprivileged%252Bscaffold%252Bfor%252Bproteolysis%252Btargeting%252Bchimeras%26aulast%3DXia%26aufirst%3DLi-Wen%26date%3D2019%26volume%3D11%26issue%3D22%26spage%3D2919%26epage%3D2973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Beibei  Wang</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Chixiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Progress on small-molecule proteolysis-targeting chimeras. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (20)
                                     , 2715-2734. <a href="https://doi.org/10.4155/fmc-2019-0161" title="DOI URL">https://doi.org/10.4155/fmc-2019-0161</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0161%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Bon%252Bsmall-molecule%252Bproteolysis-targeting%252Bchimeras%26aulast%3DHuang%26aufirst%3DWenhai%26date%3D2019%26volume%3D11%26issue%3D20%26spage%3D2715%26epage%3D2734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karthigayan  Shanmugasundaram</span>, <span class="hlFld-ContribAuthor ">Peng  Shao</span>, <span class="hlFld-ContribAuthor ">Han  Chen</span>, <span class="hlFld-ContribAuthor ">Bismarck  Campos</span>, <span class="hlFld-ContribAuthor ">Stanton F.  McHardy</span>, <span class="hlFld-ContribAuthor ">Tuoping  Luo</span>, <span class="hlFld-ContribAuthor ">Hai  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">A modular PROTAC design for target destruction using a degradation signal based on a single amino acid. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (41)
                                     , 15172-15175. <a href="https://doi.org/10.1074/jbc.AC119.010790" title="DOI URL">https://doi.org/10.1074/jbc.AC119.010790</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.AC119.010790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.AC119.010790%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DA%252Bmodular%252BPROTAC%252Bdesign%252Bfor%252Btarget%252Bdestruction%252Busing%252Ba%252Bdegradation%252Bsignal%252Bbased%252Bon%252Ba%252Bsingle%252Bamino%252Bacid%26aulast%3DShanmugasundaram%26aufirst%3DKarthigayan%26date%3D2019%26volume%3D294%26issue%3D41%26spage%3D15172%26epage%3D15175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Xia  Hao</span>, <span class="hlFld-ContribAuthor ">Lanlan  Jing</span>, <span class="hlFld-ContribAuthor ">Gaochan  Wu</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Recent applications of click chemistry in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 779-789. <a href="https://doi.org/10.1080/17460441.2019.1614910" title="DOI URL">https://doi.org/10.1080/17460441.2019.1614910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1614910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1614910%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DRecent%252Bapplications%252Bof%252Bclick%252Bchemistry%252Bin%252Bdrug%252Bdiscovery%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D8%26spage%3D779%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea G.  Cochran</span>, <span class="hlFld-ContribAuthor ">Andrew R.  Conery</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sims</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: a new target class for drug development. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (8)
                                     , 609-628. <a href="https://doi.org/10.1038/s41573-019-0030-7" title="DOI URL">https://doi.org/10.1038/s41573-019-0030-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DBromodomains%25253A%252Ba%252Bnew%252Btarget%252Bclass%252Bfor%2525C2%2525A0drug%252Bdevelopment%26aulast%3DCochran%26aufirst%3DAndrea%2BG.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D609%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Scott D.  Edmondson</span>, <span class="hlFld-ContribAuthor ">Bin  Yang</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) in ‘beyond rule-of-five’ chemical space: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (13)
                                     , 1555-1564. <a href="https://doi.org/10.1016/j.bmcl.2019.04.030" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.04.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bin%252B%2525E2%252580%252598beyond%252Brule-of-five%2525E2%252580%252599%252Bchemical%252Bspace%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DEdmondson%26aufirst%3DScott%2BD.%26date%3D2019%26volume%3D29%26issue%3D13%26spage%3D1555%26epage%3D1564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Izidor  Sosič</span>, <span class="hlFld-ContribAuthor ">Stefanie  Lindner</span>, <span class="hlFld-ContribAuthor ">Aleša  Bricelj</span>, <span class="hlFld-ContribAuthor ">Franziska  Kohl</span>, <span class="hlFld-ContribAuthor ">Yuen Lam Dora  Ng</span>, <span class="hlFld-ContribAuthor ">Marius  Monschke</span>, <span class="hlFld-ContribAuthor ">Karl G.  Wagner</span>, <span class="hlFld-ContribAuthor ">Jan  Krönke</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">A MedChem toolbox for cereblon-directed PROTACs. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (6)
                                     , 1037-1041. <a href="https://doi.org/10.1039/C9MD00185A" title="DOI URL">https://doi.org/10.1039/C9MD00185A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00185A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00185A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DA%252BMedChem%252Btoolbox%252Bfor%252Bcereblon-directed%252BPROTACs%26aulast%3DSteinebach%26aufirst%3DChristian%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D6%26spage%3D1037%26epage%3D1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanyuan  Li</span>, <span class="hlFld-ContribAuthor ">Silong  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhiye  Hu</span>, <span class="hlFld-ContribAuthor ">Yuan  Xiao</span>, <span class="hlFld-ContribAuthor ">Jian  Huang</span>, <span class="hlFld-ContribAuthor ">Chune  Dong</span>, <span class="hlFld-ContribAuthor ">Shengtang  Huang</span>, <span class="hlFld-ContribAuthor ">Hai-Bing  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the PROTAC degron candidates: OBHSA with different side chains as novel selective estrogen receptor degraders (SERDs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>172 </em>, 48-61. <a href="https://doi.org/10.1016/j.ejmech.2019.03.058" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.03.058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.03.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.03.058%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploring%252Bthe%252BPROTAC%252Bdegron%252Bcandidates%25253A%252BOBHSA%252Bwith%252Bdifferent%252Bside%252Bchains%252Bas%252Bnovel%252Bselective%252Bestrogen%252Breceptor%252Bdegraders%252B%252528SERDs%252529%26aulast%3DLi%26aufirst%3DYuanyuan%26date%3D2019%26volume%3D172%26spage%3D48%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haixiang  Pei</span>, <span class="hlFld-ContribAuthor ">Yangrui  Peng</span>, <span class="hlFld-ContribAuthor ">Qiuhua  Zhao</span>, <span class="hlFld-ContribAuthor ">Yihua  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule PROTACs: an emerging technology for targeted therapy in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (30)
                                     , 16967-16976. <a href="https://doi.org/10.1039/C9RA03423D" title="DOI URL">https://doi.org/10.1039/C9RA03423D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA03423D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA03423D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSmall%252Bmolecule%252BPROTACs%25253A%252Ban%252Bemerging%252Btechnology%252Bfor%252Btargeted%252Btherapy%252Bin%252Bdrug%252Bdiscovery%26aulast%3DPei%26aufirst%3DHaixiang%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D30%26spage%3D16967%26epage%3D16976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Krajcovicova</span>, <span class="hlFld-ContribAuthor ">R.  Jorda</span>, <span class="hlFld-ContribAuthor ">D.  Hendrychova</span>, <span class="hlFld-ContribAuthor ">V.  Krystof</span>, <span class="hlFld-ContribAuthor ">M.  Soural</span>. </span><span class="cited-content_cbyCitation_article-title">Solid-phase synthesis for thalidomide-based proteolysis-targeting chimeras (PROTAC). </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (7)
                                     , 929-932. <a href="https://doi.org/10.1039/C8CC08716D" title="DOI URL">https://doi.org/10.1039/C8CC08716D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CC08716D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CC08716D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DSolid-phase%252Bsynthesis%252Bfor%252Bthalidomide-based%252Bproteolysis-targeting%252Bchimeras%252B%252528PROTAC%252529%26aulast%3DKrajcovicova%26aufirst%3DS.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D7%26spage%3D929%26epage%3D932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yutian  Zou</span>, <span class="hlFld-ContribAuthor ">Danhui  Ma</span>, <span class="hlFld-ContribAuthor ">Yinyin  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">The PROTAC technology in drug development. </span><span class="cited-content_cbyCitation_journal-name">Cell Biochemistry and Function</span><span> <strong>2019,</strong> <em>37 </em>
                                    (1)
                                     , 21-30. <a href="https://doi.org/10.1002/cbf.3369" title="DOI URL">https://doi.org/10.1002/cbf.3369</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbf.3369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbf.3369%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Biochemistry%2520and%2520Function%26atitle%3DThe%252BPROTAC%252Btechnology%252Bin%252Bdrug%252Bdevelopment%26aulast%3DZou%26aufirst%3DYutian%26date%3D2019%26date%3D2019%26volume%3D37%26issue%3D1%26spage%3D21%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frederick A.  Luzzio</span>. </span><span class="cited-content_cbyCitation_article-title">Thalidomide and Analogues. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 367-429. <a href="https://doi.org/10.1016/B978-0-12-815675-9.00010-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-815675-9.00010-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-815675-9.00010-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-815675-9.00010-2%26sid%3Dliteratum%253Aachs%26atitle%3DThalidomide%252Band%252BAnalogues%26aulast%3DLuzzio%26aufirst%3DFrederick%2BA.%26date%3D2019%26spage%3D367%26epage%3D429%26pub%3DElsevier%26atitle%3DImides%26date%3D2019%26volume%3D278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcel  Scheepstra</span>, <span class="hlFld-ContribAuthor ">Koen F.W.  Hekking</span>, <span class="hlFld-ContribAuthor ">Luc  van Hijfte</span>, <span class="hlFld-ContribAuthor ">Rutger H.A.  Folmer</span>. </span><span class="cited-content_cbyCitation_article-title">Bivalent Ligands for Protein Degradation in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2019,</strong> <em>17 </em>, 160-176. <a href="https://doi.org/10.1016/j.csbj.2019.01.006" title="DOI URL">https://doi.org/10.1016/j.csbj.2019.01.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2019.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2019.01.006%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DBivalent%252BLigands%252Bfor%252BProtein%252BDegradation%252Bin%252BDrug%252BDiscovery%26aulast%3DScheepstra%26aufirst%3DMarcel%26date%3D2019%26volume%3D17%26spage%3D160%26epage%3D176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Protein Degradation Approach and its Application to Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1681-1700. <a href="https://doi.org/10.1002/tcr.201800032" title="DOI URL">https://doi.org/10.1002/tcr.201800032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800032%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DChemical%252BProtein%252BDegradation%252BApproach%252Band%252Bits%252BApplication%252Bto%252BEpigenetic%252BTargets%26aulast%3DItoh%26aufirst%3DYukihiro%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1681%26epage%3D1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barry  Boland</span>, <span class="hlFld-ContribAuthor ">Wai Haung  Yu</span>, <span class="hlFld-ContribAuthor ">Olga  Corti</span>, <span class="hlFld-ContribAuthor ">Bertrand  Mollereau</span>, <span class="hlFld-ContribAuthor ">Alexandre  Henriques</span>, <span class="hlFld-ContribAuthor ">Erwan  Bezard</span>, <span class="hlFld-ContribAuthor ">Greg M.  Pastores</span>, <span class="hlFld-ContribAuthor ">David C.  Rubinsztein</span>, <span class="hlFld-ContribAuthor ">Ralph A.  Nixon</span>, <span class="hlFld-ContribAuthor ">Michael R.  Duchen</span>, <span class="hlFld-ContribAuthor ">Giovanna R.  Mallucci</span>, <span class="hlFld-ContribAuthor ">Guido  Kroemer</span>, <span class="hlFld-ContribAuthor ">Beth  Levine</span>, <span class="hlFld-ContribAuthor ">Eeva-Liisa  Eskelinen</span>, <span class="hlFld-ContribAuthor ">Fanny  Mochel</span>, <span class="hlFld-ContribAuthor ">Michael  Spedding</span>, <span class="hlFld-ContribAuthor ">Caroline  Louis</span>, <span class="hlFld-ContribAuthor ">Olivier R.  Martin</span>, <span class="hlFld-ContribAuthor ">Mark J.  Millan</span>. </span><span class="cited-content_cbyCitation_article-title">Promoting the clearance of neurotoxic proteins in neurodegenerative disorders of ageing. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (9)
                                     , 660-688. <a href="https://doi.org/10.1038/nrd.2018.109" title="DOI URL">https://doi.org/10.1038/nrd.2018.109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2018.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2018.109%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DPromoting%252Bthe%252Bclearance%252Bof%252Bneurotoxic%252Bproteins%252Bin%252Bneurodegenerative%252Bdisorders%252Bof%252Bageing%26aulast%3DBoland%26aufirst%3DBarry%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D9%26spage%3D660%26epage%3D688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikhail M.  Savitski</span>, <span class="hlFld-ContribAuthor ">Nico  Zinn</span>, <span class="hlFld-ContribAuthor ">Maria  Faelth-Savitski</span>, <span class="hlFld-ContribAuthor ">Daniel  Poeckel</span>, <span class="hlFld-ContribAuthor ">Stephan  Gade</span>, <span class="hlFld-ContribAuthor ">Isabelle  Becher</span>, <span class="hlFld-ContribAuthor ">Marcel  Muelbaier</span>, <span class="hlFld-ContribAuthor ">Anne J.  Wagner</span>, <span class="hlFld-ContribAuthor ">Katrin  Strohmer</span>, <span class="hlFld-ContribAuthor ">Thilo  Werner</span>, <span class="hlFld-ContribAuthor ">Stephanie  Melchert</span>, <span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Anna  Rutkowska</span>, <span class="hlFld-ContribAuthor ">Johanna  Vappiani</span>, <span class="hlFld-ContribAuthor ">Holger  Franken</span>, <span class="hlFld-ContribAuthor ">Michael  Steidel</span>, <span class="hlFld-ContribAuthor ">Gavain M.  Sweetman</span>, <span class="hlFld-ContribAuthor ">Omer  Gilan</span>, <span class="hlFld-ContribAuthor ">Enid Y.N.  Lam</span>, <span class="hlFld-ContribAuthor ">Mark A.  Dawson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>, <span class="hlFld-ContribAuthor ">Giovanna  Bergamini</span>, <span class="hlFld-ContribAuthor ">Marcus  Bantscheff</span>. </span><span class="cited-content_cbyCitation_article-title">Multiplexed Proteome Dynamics Profiling Reveals Mechanisms Controlling Protein Homeostasis. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2018,</strong> <em>173 </em>
                                    (1)
                                     , 260-274.e25. <a href="https://doi.org/10.1016/j.cell.2018.02.030" title="DOI URL">https://doi.org/10.1016/j.cell.2018.02.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2018.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2018.02.030%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DMultiplexed%252BProteome%252BDynamics%252BProfiling%252BReveals%252BMechanisms%252BControlling%252BProtein%252BHomeostasis%26aulast%3DSavitski%26aufirst%3DMikhail%2BM.%26date%3D2018%26volume%3D173%26issue%3D1%26spage%3D260%26epage%3D274.e25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, , <span class="hlFld-ContribAuthor ">Scott J.  Hughes</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders. </span><span class="cited-content_cbyCitation_journal-name">Essays in Biochemistry</span><span> <strong>2017,</strong> <em>61 </em>
                                    (5)
                                     , 505-516. <a href="https://doi.org/10.1042/EBC20170041" title="DOI URL">https://doi.org/10.1042/EBC20170041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/EBC20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FEBC20170041%26sid%3Dliteratum%253Aachs%26jtitle%3DEssays%2520in%2520Biochemistry%26atitle%3DMolecular%252Brecognition%252Bof%252Bternary%252Bcomplexes%25253A%252Ba%252Bnew%252Bdimension%252Bin%252Bthe%252Bstructure-guided%252Bdesign%252Bof%252Bchemical%252Bdegraders%26aulast%3Dvan%2BMontfort%26aufirst%3DRob%2BL.M.%26date%3D2017%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D505%26epage%3D516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Favour A.  Akinjiyan</span>, <span class="hlFld-ContribAuthor ">Seth  Carbonneau</span>, <span class="hlFld-ContribAuthor ">Nathan T.  Ross</span>. </span><span class="cited-content_cbyCitation_article-title">Lead discovery and chemical biology approaches targeting the ubiquitin proteasome system. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (20)
                                     , 4589-4596. <a href="https://doi.org/10.1016/j.bmcl.2017.08.058" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.08.058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.08.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.08.058%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLead%252Bdiscovery%252Band%252Bchemical%252Bbiology%252Bapproaches%252Btargeting%252Bthe%252Bubiquitin%252Bproteasome%252Bsystem%26aulast%3DAkinjiyan%26aufirst%3DFavour%2BA.%26date%3D2017%26volume%3D27%26issue%3D20%26spage%3D4589%26epage%3D4596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline M.  Robb</span>, <span class="hlFld-ContribAuthor ">Jacob I.  Contreras</span>, <span class="hlFld-ContribAuthor ">Smit  Kour</span>, <span class="hlFld-ContribAuthor ">Margaret A.  Taylor</span>, <span class="hlFld-ContribAuthor ">Mohammad  Abid</span>, <span class="hlFld-ContribAuthor ">Yogesh A.  Sonawane</span>, <span class="hlFld-ContribAuthor ">Muhammad  Zahid</span>, <span class="hlFld-ContribAuthor ">Daryl J.  Murry</span>, <span class="hlFld-ContribAuthor ">Amarnath  Natarajan</span>, <span class="hlFld-ContribAuthor ">Sandeep  Rana</span>. </span><span class="cited-content_cbyCitation_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC). </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2017,</strong> <em>53 </em>
                                    (54)
                                     , 7577-7580. <a href="https://doi.org/10.1039/C7CC03879H" title="DOI URL">https://doi.org/10.1039/C7CC03879H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7CC03879H%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DChemically%252Binduced%252Bdegradation%252Bof%252BCDK9%252Bby%252Ba%252Bproteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%26aulast%3DRobb%26aufirst%3DCaroline%2BM.%26date%3D2017%26date%3D2017%26volume%3D53%26issue%3D54%26spage%3D7577%26epage%3D7580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PROTACs for the degradation of BET bromodomain BRD4 protein derived from ligand <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. General strategy using “click chemistry” for parallel PROTAC synthesis. POI = protein of interest.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Proximity assay data for JQ-1-IMiD PROTACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MSD assay data for JQ-1-IMiD PROTACs in a NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proximity assay data for JQ-1-VHL PROTACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MSD assay data for JQ-1-VHL (<b>12a</b>–<b>e</b>) PROTACs in a NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/medium/jm-2016-017816_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Control experiments: (A) <b>11e</b> PROTAC in the NCI-H661 cell line (4 h time point); (B) <b>12b</b> PROTAC in NCI-H661 cell line (4 h time point).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-2/acs.jmedchem.6b01781/20180119/images/large/jm-2016-017816_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01781&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50487" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50487" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Small-molecule PROTACS: New approaches to protein degradation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1966</span><span class="NLM_x">–</span> <span class="NLM_lpage">1973</span><span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+New+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lhI3oME1nDVTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520New%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljdIU-JAK-BQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Berndsen, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">New insights into ubiquitin E3 ligase mechanism</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span><span class="refDoi"> DOI: 10.1038/nsmb.2780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnsmb.2780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=24699078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1aisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=301-307&author=C.+E.+Berndsenauthor=C.+Wolberger&title=New+insights+into+ubiquitin+E3+ligase+mechanism&doi=10.1038%2Fnsmb.2780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">New insights into ubiquitin E3 ligase mechanism</span></div><div class="casAuthors">Berndsen, Christopher E.; Wolberger, Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-307</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  E3 ligases carry out the final step in the ubiquitination cascade, catalyzing transfer of ubiquitin from an E2 enzyme to form a covalent bond with a substrate lysine.  Three distinct classes of E3 ligases have been identified that stimulate transfer of ubiquitin and ubiquitin-like proteins through either a direct or an indirect mechanism.  Only recently have the catalytic mechanisms of E3 ligases begun to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovYmD2sLo2RrVg90H21EOLACvtfcHk0ljdIU-JAK-BQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1aisrk%253D&md5=fd44f6d69f85067b70322ca304aed507</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2780%26sid%3Dliteratum%253Aachs%26aulast%3DBerndsen%26aufirst%3DC.%2BE.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DNew%2520insights%2520into%2520ubiquitin%2520E3%2520ligase%2520mechanism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D301%26epage%3D307%26doi%3D10.1038%2Fnsmb.2780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Okuhira, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimaki-Mogami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">Development of hybrid small molecules that induce degradation of estrogen receptor-alpha and necrotic cell death in breast cancer cells</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">1492</span><span class="NLM_x">–</span> <span class="NLM_lpage">1498</span><span class="refDoi"> DOI: 10.1111/cas.12272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1111%2Fcas.12272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=23992566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ylu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=1492-1498&author=K.+Okuhiraauthor=Y.+Demizuauthor=T.+Hattoriauthor=N.+Ohokaauthor=N.+Shibataauthor=T.+Nishimaki-Mogamiauthor=H.+Okudaauthor=M.+Kuriharaauthor=M.+Naito&title=Development+of+hybrid+small+molecules+that+induce+degradation+of+estrogen+receptor-alpha+and+necrotic+cell+death+in+breast+cancer+cells&doi=10.1111%2Fcas.12272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Development of hybrid small molecules that induce degradation of estrogen receptor-alpha and necrotic cell death in breast cancer cells</span></div><div class="casAuthors">Okuhira, Keiichiro; Demizu, Yosuke; Hattori, Takayuki; Ohoka, Nobumichi; Shibata, Norihito; Nishimaki-Mogami, Tomoko; Okuda, Haruhiro; Kurihara, Masaaki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1492-1498</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Manipulation of protein stability with small mols. has a great potential for both basic research and clin. therapy.  Recently, we have developed a series of hybrid small mols. named SNIPER (Specific and Non-genetic IAP-dependent Protein ERaser) that induces degrdn. of target proteins via ubiquitin-proteasome system.  Here we report the activities of SNIPER(ER) that targets estrogen receptor alpha (ERα) for degrdn.  SNIPER(ER) induced degrdn. of ERα and inhibited estrogen-dependent expression of pS2 gene in an estrogen-dependent breast cancer cell line MCF-7.  A proteasome inhibitor MG132 and siRNA-mediated downregulation of cIAP1 abrogated the SNIPER(ER)-induced ERα degrdn., suggesting that the ERα is degraded by proteasome subsequent to cIAP1-mediated ubiquitylation.  Intriguingly, after the ERα degrdn., the SNIPER(ER)-treated MCF-7 cells undergo rapid cell death.  Detailed anal. indicated that SNIPER(ER) caused necrotic cell death accompanied by a release of HMGB1, a marker of necrosis, from the cells.  Following the ERα degrdn., reactive oxygen species (ROS) was produced in the SNIPER(ER)-treated MCF-7 cells, and an anti-oxidant N-acetylcysteine inhibited the necrotic cell death.  These results indicate that SNIPER(ER) induces ERα degrdn., ROS prodn. and necrotic cell death, implying a therapeutic potential of SNIPER(ER) as a lead for the treatment of ERα-pos. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpckyKeS1C4ZrVg90H21EOLACvtfcHk0ljdIU-JAK-BQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ylu7fM&md5=dc35aee47ea4cd9554a663eb077137b5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcas.12272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12272%26sid%3Dliteratum%253Aachs%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNishimaki-Mogami%26aufirst%3DT.%26aulast%3DOkuda%26aufirst%3DH.%26aulast%3DKurihara%26aufirst%3DM.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520hybrid%2520small%2520molecules%2520that%2520induce%2520degradation%2520of%2520estrogen%2520receptor-alpha%2520and%2520necrotic%2520cell%2520death%2520in%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Sci.%26date%3D2013%26volume%3D104%26spage%3D1492%26epage%3D1498%26doi%3D10.1111%2Fcas.12272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lin0ltvJ95yIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerschmied, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salami, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaime-Figueroa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic Bcr-Abl</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">810</span><span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+Bcr-Abl&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0liSW96oZeOJkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520Bcr-Abl%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span> </span><span class="NLM_article-title">Development of Bcr-Abl degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">4865</span><span class="NLM_x">–</span> <span class="NLM_lpage">4869</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+Bcr-Abl+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0li56jrTwB380Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520Bcr-Abl%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lebraud, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span> <span class="citation_source-journal">ACS Cent. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">927</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0li56jrTwB380Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljYzA_COahHPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Winter, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Drug development. Phthalimide conjugation as a strategy for in vivo target protein degradation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">1376</span><span class="NLM_x">–</span> <span class="NLM_lpage">1381</span><span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Drug+development.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljYzA_COahHPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDrug%2520development.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1770</span><span class="NLM_x">–</span> <span class="NLM_lpage">1777</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0ljYzA_COahHPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><div class="note"><p class="first last">For a review on E3 ligases see:</p></div><span class="NLM_contrib-group">Bulatov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Targeting Cullin–RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1042/BJ20141450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1042%2FBJ20141450" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=365-386&author=E.+Bulatovauthor=A.+Ciulli&title=Targeting+Cullin%E2%80%93RING+E3+ubiquitin+ligases+for+drug+discovery%3A+structure%2C+assembly+and+small-molecule+modulation&doi=10.1042%2FBJ20141450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20141450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141450%26sid%3Dliteratum%253Aachs%26aulast%3DBulatov%26aufirst%3DE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520Cullin%25E2%2580%2593RING%2520E3%2520ubiquitin%2520ligases%2520for%2520drug%2520discovery%253A%2520structure%252C%2520assembly%2520and%2520small-molecule%2520modulation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D365%26epage%3D386%26doi%3D10.1042%2FBJ20141450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><div class="note"><p class="first last">For a leading reference into the optimization of VHL see:</p></div><span class="NLM_contrib-group">Galdeano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaffidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Molle, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birced, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the Von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8657</span><span class="NLM_x">–</span> <span class="NLM_lpage">8663</span><span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+Von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhEO2yxdFToTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Schneekloth, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pucheault, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Targeted intracellular protein degradation induced by a small molecule: en route to chemical proteomics</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">5904</span><span class="NLM_x">–</span> <span class="NLM_lpage">5908</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2008.07.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=18752944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5904-5908&author=A.+R.+Schneeklothauthor=M.+Pucheaultauthor=H.+S.+Taeauthor=C.+M.+Crews&title=Targeted+intracellular+protein+degradation+induced+by+a+small+molecule%3A+en+route+to+chemical+proteomics&doi=10.1016%2Fj.bmcl.2008.07.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted intracellular protein degradation induced by a small molecule: En route to chemical proteomics</span></div><div class="casAuthors">Schneekloth, Ashley R.; Pucheault, Mathieu; Tae, Hyun Seop; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5904-5908</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have developed a heterobifunctional all-small mol. PROTAC (PROteolysis TArgeting Chimera) capable of inducing proteasomal degrdn. of the androgen receptor.  This cell permeable PROTAC consists of a non-steroidal androgen receptor ligand (SARM) and the MDM2 ligand known as nutlin, connected by a PEG-based linker.  The SARM-nutlin PROTAC recruits the androgen receptor to MDM2, which functions as an E3 ubiquitin ligase.  This leads to the ubiquitination of the androgen receptor, and its subsequent degrdn. by the proteasome.  Upon treatment of HeLa cells with 10 μM PROTAC for 7 h, we were able to observe a decrease in androgen receptor levels.  This degrdn. is proteasome dependent, as it is mitigated in cells pre-treated with 10 μM epoxomicin, a specific proteasome inhibitor.  These results have implications for the potential study and treatment of various cancers with increased androgen receptor levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZh1309EM47LVg90H21EOLACvtfcHk0lhEO2yxdFToTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7rL&md5=49f32203efc4b2f6f2cbc900df398a1a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.114%26sid%3Dliteratum%253Aachs%26aulast%3DSchneekloth%26aufirst%3DA.%2BR.%26aulast%3DPucheault%26aufirst%3DM.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520intracellular%2520protein%2520degradation%2520induced%2520by%2520a%2520small%2520molecule%253A%2520en%2520route%2520to%2520chemical%2520proteomics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5904%26epage%3D5908%26doi%3D10.1016%2Fj.bmcl.2008.07.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sakamoto, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumagai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshaies, R. J.</span><span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">8554</span><span class="NLM_x">–</span> <span class="NLM_lpage">8559</span><span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lhEO2yxdFToTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljyM4TrclykUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><div class="note"><p class="first last">For a recent report on the synthesis of a phthalimide ligand library, see:</p></div><span class="NLM_contrib-group">Lohbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. K.</span><span> </span><span class="NLM_article-title">Practical synthesis of a phthalimide-based cereblon ligand to enable PROTAC development</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5260</span><span class="NLM_x">–</span> <span class="NLM_lpage">5262</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.bmcl.2016.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=27687673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5260-5262&author=J.+Lohbeckauthor=A.+K.+Miller&title=Practical+synthesis+of+a+phthalimide-based+cereblon+ligand+to+enable+PROTAC+development&doi=10.1016%2Fj.bmcl.2016.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Practical synthesis of a phthalimide-based Cereblon ligand to enable PROTAC development</span></div><div class="casAuthors">Lohbeck, Jasmin; Miller, Aubry K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5260-5262</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of small mols. to regulate cellular levels of specific proteins is poised to become a powerful technique in the coming years.  Crit. to the success of any project using such an approach will be the ability to synthesize libraries of candidate small mols. for testing in cellular systems.  Herein, the authors describe a practical synthesis of a phthalimide-based scaffold, which can be easily diversified to make Cereblon-targeting PROTACs.  The authors demonstrate the effectiveness of this approach by synthesizing a 'PROTAC toolbox' of four amines which can be coupled to inhibitors in a straightforward manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo01AK2Qjj97Vg90H21EOLACvtfcHk0ljyM4TrclykUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI&md5=864f4c671ef2c8d43191fc6a6e22b3b2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLohbeck%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%2BK.%26atitle%3DPractical%2520synthesis%2520of%2520a%2520phthalimide-based%2520cereblon%2520ligand%2520to%2520enable%2520PROTAC%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5260%26epage%3D5262%26doi%3D10.1016%2Fj.bmcl.2016.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><div class="note"><p class="first last">For a recent review on azide-alkyne cycloaddition-click chemistry, see:</p></div><span class="NLM_contrib-group">Singh, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koley, S.</span><span> </span><span class="NLM_article-title">Advances of azide-alkyne cycloaddition-click chemistry over the recent decade</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">5257</span><span class="NLM_x">–</span> <span class="NLM_lpage">5283</span><span class="refDoi"> DOI: 10.1016/j.tet.2016.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.tet.2016.07.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtF2hsbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=5257-5283&author=M.+S.+Singhauthor=S.+Chowdhuryauthor=S.+Koley&title=Advances+of+azide-alkyne+cycloaddition-click+chemistry+over+the+recent+decade&doi=10.1016%2Fj.tet.2016.07.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Advances of azide-alkyne cycloaddition-click chemistry over the recent decade</span></div><div class="casAuthors">Singh, Maya Shankar; Chowdhury, Sushobhan; Koley, Suvajit</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5257-5283</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  This review presents an overview of the open literature on the development of azide-dipolarophile 1,3-dipolar cycloaddn. chem. over the recent years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9L_G6zGkGkrVg90H21EOLACvtfcHk0ljyM4TrclykUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtF2hsbvN&md5=af0463fba34ef4fe7d7239ad8dc90256</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DM.%2BS.%26aulast%3DChowdhury%26aufirst%3DS.%26aulast%3DKoley%26aufirst%3DS.%26atitle%3DAdvances%2520of%2520azide-alkyne%2520cycloaddition-click%2520chemistry%2520over%2520the%2520recent%2520decade%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D5257%26epage%3D5283%26doi%3D10.1016%2Fj.tet.2016.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><div class="note"><p class="first last">For a recent report on the use of click chemistry to prepare analogues of thalidomide, see:</p></div><span class="NLM_contrib-group">Ronnebaum, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzzio, F. A.</span><span> </span><span class="NLM_article-title">Synthesis of 1,2,3-triazole “click” analogues of thalidomide</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">6136</span><span class="NLM_x">–</span> <span class="NLM_lpage">6141</span><span class="refDoi"> DOI: 10.1016/j.tet.2016.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1016%2Fj.tet.2016.07.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=6136-6141&author=J.+M.+Ronnebaumauthor=F.+A.+Luzzio&title=Synthesis+of+1%2C2%2C3-triazole+%E2%80%9Cclick%E2%80%9D+analogues+of+thalidomide&doi=10.1016%2Fj.tet.2016.07.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DRonnebaum%26aufirst%3DJ.%2BM.%26aulast%3DLuzzio%26aufirst%3DF.%2BA.%26atitle%3DSynthesis%2520of%25201%252C2%252C3-triazole%2520%25E2%2580%259Cclick%25E2%2580%259D%2520analogues%2520of%2520thalidomide%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D6136%26epage%3D6141%26doi%3D10.1016%2Fj.tet.2016.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><div class="note"><p class="first last">The thalidomide motif was used as a racemate because it is known that its center racemizes in vivo, see for example:</p></div><span class="NLM_contrib-group">Tian, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span> </span><span class="NLM_article-title">Enantiomerization mechanism of thalidomide and the role of water and hydroxide ions</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">14305</span><span class="NLM_x">–</span> <span class="NLM_lpage">14313</span><span class="refDoi"> DOI: 10.1002/chem.201202651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1002%2Fchem.201202651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=23065668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rtr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=14305-14313&author=C.+Tianauthor=P.+Xiuauthor=Y.+Mengauthor=W.+Zhaoauthor=Z.+Wangauthor=R.+Zhou&title=Enantiomerization+mechanism+of+thalidomide+and+the+role+of+water+and+hydroxide+ions&doi=10.1002%2Fchem.201202651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Enantiomerization Mechanism of Thalidomide and the Role of Water and Hydroxide Ions</span></div><div class="casAuthors">Tian, Chuanjin; Xiu, Peng; Meng, Yan; Zhao, Wenyan; Wang, Zhigang; Zhou, Ruhong</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">14305-14313, S14305/1-S14305/24</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The significance of the mol. chirality of drugs has been widely recognized due to the thalidomide tragedy.  Most of the new drugs reaching the market today are single enantiomers, rather than racemic mixts.  However, many optically pure drugs, including thalidomide, undergo enantiomerization in vivo, thus negating the single enantiomers' benefits or inducing unexpected effects.  A detailed at. level understanding of chiral conversion, which is still largely lacking, is thus crit. for drug development.  Herein, we use first-principle d. function theory (DFT) to explore the mechanism of enantiomerization of thalidomide.  We have identified the two most plausible interconversion pathways for isolated thalidomide: (1) proton transfer from the chiral carbon center to an adjacent carbonyl oxygen atom, followed by isomerization and rotation of the glutarimide ring (before the proton hops back to the chiral carbon atom); and (2) a pathway that is the same as "1", but with the isomerization of the glutarimide ring occurring ahead of the initial proton transfer reaction.  There are two remarkable energy barriers, 73.29 and 23.59 kcal mol-1, corresponding to the proton transfer and the rotation of the glutarimide ring, resp.  Furthermore, we found that water effectively catalyzes the interconversion by facilitating the proton transfer with the highest energy barrier falling to approx. 30 kcal mol-1, which, to our knowledge, is the first time that this important role of water in chiral conversion has been demonstrated.  Finally, we show that the hydroxide ion can further lower the enantiomerization energy barrier to approx. 24 kcal mol-1 by facilitating proton abstraction, which agrees well with recent exptl. data under basic conditions.  Our current findings highlight the importance of water and hydroxide ions in the enantiomerization of thalidomide and also provide new insights into the mechanism of enantiomerization at an at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtc9xvH0R_Z7Vg90H21EOLACvtfcHk0lh3cYwTsDiy7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rtr%252FK&md5=a169947388b8463fc11dcdb1e546c0e3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fchem.201202651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201202651%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DC.%26aulast%3DXiu%26aufirst%3DP.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DR.%26atitle%3DEnantiomerization%2520mechanism%2520of%2520thalidomide%2520and%2520the%2520role%2520of%2520water%2520and%2520hydroxide%2520ions%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2012%26volume%3D18%26spage%3D14305%26epage%3D14313%26doi%3D10.1002%2Fchem.201202651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a> for details of the synthesis of this library.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>For more information, see: <a href="http://www.perkinelmer.com" class="extLink">http://www.perkinelmer.com</a> (accessed February 8,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information%2C+see%3A+http%3A%2F%2Fwww.perkinelmer.com+%28accessed+February+8%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="note"><p class="first last">The NCI-H661 cell line is a human large cell lung carcinoma cell line and found to have excellent levels of activity for both VHL and cereblon ligases and is also adherent making this cell line ideal for the MSD assay format.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="note"><p class="first last">Similar data was also generated for a NCI-H838 cell line and this data is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Douglass, E. F.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, D. A.</span><span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6092</span><span class="NLM_x">–</span> <span class="NLM_lpage">6099</span><span class="refDoi"> DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0lh3cYwTsDiy7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="note"><p class="first last">These PROTACs also showed similar activities in the NCI-H838 cell line (nonsmall cell lung carcinoma) see: <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Table 1</a> in the Supporting Information.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Meiners, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stangl, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloetzel, P.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, E.</span><span> </span><span class="NLM_article-title">Inhibition of proteasome activity induces concerted expression of proteasome genes and de novo formation of mammalian proteasomes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">21517</span><span class="NLM_x">–</span> <span class="NLM_lpage">21525</span><span class="refDoi"> DOI: 10.1074/jbc.M301032200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=10.1074%2Fjbc.M301032200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=12676932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVemsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=21517-21525&author=S.+Meinersauthor=D.+Heykenauthor=A.+Wellerauthor=A.+Ludwigauthor=K.+Stanglauthor=P.-M.+Kloetzelauthor=E.+Kr%C3%BCger&title=Inhibition+of+proteasome+activity+induces+concerted+expression+of+proteasome+genes+and+de+novo+formation+of+mammalian+proteasomes&doi=10.1074%2Fjbc.M301032200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Proteasome Activity Induces Concerted Expression of Proteasome Genes and de Novo Formation of Mammalian Proteasomes</span></div><div class="casAuthors">Meiners, Silke; Heyken, Dirk; Weller, Andrea; Ludwig, Antje; Stangl, Karl; Kloetzel, Peter-M.; Krueger, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">21517-21525</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The 26 S proteasome is a high mol. mass proteinase complex that is built by at least 32 different protein subunits.  Such protease complexes in bacteria and yeast are systems that undergo a highly sophisticated network of gene expression regulation.  However, regulation of mammalian proteasome gene expression has been neglected so far as a possible control mechanism for the amt. of proteasomes in the cell.  Here, we show that treatment of cells with proteasome inhibitors and the concomitant impairment of proteasomal enzyme activity induce a transient and concerted up-regulation of all mammalian 26 S proteasome subunit mRNAs.  Proteasome inhibition in combination with inhibition of transcription revealed that the obsd. up-regulation is mediated by coordinated transcriptional activation of the proteasome genes and not by post-transcriptional events.  Our expts. also demonstrate that inhibitor-induced proteasome gene activation results in enhanced de novo protein synthesis of all subunits and in increased de novo formation of proteasomes.  This phenomenon is accompanied by enhanced expression of the proteasome maturation factor POMP.  Thus, our expts. present the first evidence that the amt. of proteasomes in mammalia is regulated at the transcriptional level and that there exists an autoregulatory feedback mechanism that allows the compensation of reduced proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbyNxj9qhUXLVg90H21EOLACvtfcHk0ljMOlOmWcwtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVemsrk%253D&md5=2dba4af89533fe1720299bef5653e451</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301032200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301032200%26sid%3Dliteratum%253Aachs%26aulast%3DMeiners%26aufirst%3DS.%26aulast%3DHeyken%26aufirst%3DD.%26aulast%3DWeller%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DA.%26aulast%3DStangl%26aufirst%3DK.%26aulast%3DKloetzel%26aufirst%3DP.-M.%26aulast%3DKr%25C3%25BCger%26aufirst%3DE.%26atitle%3DInhibition%2520of%2520proteasome%2520activity%2520induces%2520concerted%2520expression%2520of%2520proteasome%2520genes%2520and%2520de%2520novo%2520formation%2520of%2520mammalian%2520proteasomes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D21517%26epage%3D21525%26doi%3D10.1074%2Fjbc.M301032200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf" class="ext-link">Supporting Information</a> for the comprehensive experimental procedures and for NMR spectra.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01781">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34015"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01781">10.1021/acs.jmedchem.6b01781</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra are provided for all of the novel compounds reported (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_001.pdf">jm6b01781_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01781/suppl_file/jm6b01781_si_002.csv">jm6b01781_si_002.csv (1.99 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-2%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01781" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01781" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799859c1fe11964","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
