
<html lang="en"     class="pb-page"  data-request-id="8f27acd1-0847-4ca1-aa28-274b4d9f5a8e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b00220;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Ligands and Receptors with Broad Binding Capabilities Have Common Structural Characteristics: An Antibiotic Design Perspective" /></meta><meta name="dc.Creator" content="György  Abrusán" /></meta><meta name="dc.Creator" content="Joseph A.  Marsh" /></meta><meta name="dc.Description" content="The spread of antibiotic resistance is one of the most serious global public-health problems. Here we show that a particular class of homomers with binding sites spanning multiple protein chains is..." /></meta><meta name="Description" content="The spread of antibiotic resistance is one of the most serious global public-health problems. Here we show that a particular class of homomers with binding sites spanning multiple protein chains is..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 12, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00220" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00220" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00220" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00220" /></link>
        
    
    

<title>Ligands and Receptors with Broad Binding Capabilities Have Common Structural Characteristics: An Antibiotic Design Perspective | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00220" /></meta><meta property="og:title" content="Ligands and Receptors with Broad Binding Capabilities Have Common Structural Characteristics: An Antibiotic Design Perspective" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0011.jpeg" /></meta><meta property="og:description" content="The spread of antibiotic resistance is one of the most serious global public-health problems. Here we show that a particular class of homomers with binding sites spanning multiple protein chains is particularly suitable for targeting by broad-spectrum antibacterial agents because due to the slow evolutionary change of such binding pockets, ligands of such homomers are much more likely to bind their homologs than ligands of monomers, or homomers with a single-chain binding site. Additionally, using de novo ligand design and deep learning, we show that the chemical compounds that can bind several different receptors have common structural characteristics and that halogens and fragments similar to the building blocks existing antimicrobials are overrepresented in them. Finally, we show that binding multiple receptors selects for flexible compounds, which are less likely to accumulate in Gram-negative bacteria; thus there is trade-off between reducing the emergence of resistance by multitargeting and broad-spectrum antibacterial activity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00220"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00220">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00220&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00220&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00220&amp;href=/doi/10.1021/acs.jmedchem.9b00220" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9357-9374</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01453" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b00359" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.9b01944"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Ligands and Receptors with Broad Binding Capabilities Have Common Structural Characteristics: An Antibiotic Design Perspective</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">György Abrusán</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">György Abrusán</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Crewe Road, Edinburgh EH4 2XU, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +44 131 651 8500. E-mail: <a href="/cdn-cgi/l/email-protection#95f2ecfae7f2ecbbf4f7e7e0e6f4fbd5fcf2f8f8bbf0f1bbf4f6bbe0fe"><span class="__cf_email__" data-cfemail="cdaab4a2bfaab4e3acafbfb8beaca38da4aaa0a0e3a8a9e3acaee3b8a6">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gy%C3%B6rgy++Abrus%C3%A1n">György Abrusán</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4375-1552" title="Orcid link">http://orcid.org/0000-0003-4375-1552</a></div></div></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor">Joseph A. Marsh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph A. Marsh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, Crewe Road, Edinburgh EH4 2XU, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+A.++Marsh">Joseph A. Marsh</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00220&amp;href=/doi/10.1021%2Facs.jmedchem.9b00220" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9357–9374</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 12, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 February 2019</li><li><span class="item_label"><b>Published</b> online</span>12 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00220" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00220</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00220"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2843</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00220" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Ligands and Receptors with Broad Binding Capabilities Have Common Structural Characteristics: An Antibiotic Design Perspective&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;György&quot;,&quot;last_name&quot;:&quot;Abrusán&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;A. Marsh&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9357-9374&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00220&quot;},&quot;abstract&quot;:&quot;The spread of antibiotic resistance is one of the most serious global public-health problems. Here we show that a particular class of homomers with binding sites spanning multiple protein chains is particularly suitable for targeting by broad-spectrum antibacterial agents because due to the slow evolutionary change of such binding pockets, ligands of such homomers are much more likely to bind their homologs than ligands of monomers, or homomers with a single-chain binding site. Additionally, using de novo ligand design and deep learning, we show that the chemical compounds that can bind several different receptors have common structural characteristics and that halogens and fragments similar to the building blocks existing antimicrobials are overrepresented in them. Finally, we show that binding multiple receptors selects for flexible compounds, which are less likely to accumulate in Gram-negative bacteria; thus there is trade-off between reducing the emergence of resistance by multitargeting and broad-spectr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00220&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00220" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00220&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00220" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00220&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00220" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00220&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00220&amp;href=/doi/10.1021/acs.jmedchem.9b00220" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00220" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00220" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00220%26sid%3Dliteratum%253Aachs%26pmid%3D31188598%26genre%3Darticle%26aulast%3DAbrus%25C3%25A1n%26date%3D2019%26atitle%3DLigands%2Band%2BReceptors%2Bwith%2BBroad%2BBinding%2BCapabilities%2BHave%2BCommon%2BStructural%2BCharacteristics%253A%2BAn%2BAntibiotic%2BDesign%2BPerspective%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9357%26epage%3D9374%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The spread of antibiotic resistance is one of the most serious global public-health problems. Here we show that a particular class of homomers with binding sites spanning multiple protein chains is particularly suitable for targeting by broad-spectrum antibacterial agents because due to the slow evolutionary change of such binding pockets, ligands of such homomers are much more likely to bind their homologs than ligands of monomers, or homomers with a single-chain binding site. Additionally, using de novo ligand design and deep learning, we show that the chemical compounds that can bind several different receptors have common structural characteristics and that halogens and fragments similar to the building blocks existing antimicrobials are overrepresented in them. Finally, we show that binding multiple receptors selects for flexible compounds, which are less likely to accumulate in Gram-negative bacteria; thus there is trade-off between reducing the emergence of resistance by multitargeting and broad-spectrum antibacterial activity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43831" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43831" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Currently the world faces an emerging antibiotic resistance crisis because the rate of developing new antibiotics has not caught up with the pace of the spread of antibiotic resistance.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> This is caused by several factors: antibiotic resistance is an ancient evolutionary phenomenon and is unavoidable,<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> while the small number of novel antibiotics entering the market might be partly caused by the limitations of existing compound libraries used in the pharmaceutical industry and the possible lack of unexplored, “low-hanging-fruit” drug classes<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> (but see also ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a>). Socioeconomic factors also contribute, including the irresponsible use of antibiotics promoting resistance and the relatively low profitability of novel antimicrobials, which, exactly to prevent the emergence of resistance, are likely to be used as last-resort drugs rather than first-line medications. Since antibiotic resistance can emerge quickly, even in laboratory settings,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> developing drugs that reduce the likelihood of resistance is a central goal of the field.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> Resistance can emerge due to several factors, like changes in the proteins targeted by the antibiotic, changes in the rate of removal or uptake of the antibiotic, or changes in the degradation rate of the antibiotic. However, the analysis of currently available antibiotics indicates that most successful antibiotics or antibiotic classes bind several protein targets, e.g., β-lactam antibiotics, fluoroquinolones (or target substrates rather than enzymes, e.g., vancomycin<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a>), while resistance emerges much more quickly for antibiotics that target only a single protein (e.g., sulfonamides, trimetophrim), and such drugs are used mostly in combination with other drugs.<a onclick="showRef(event, 'ref7 ref11'); return false;" href="javascript:void(0);" class="ref ref7 ref11">(7,11)</a> The most likely cause of this phenomenon is that in the case of single target drugs, a few mutations at a single binding site can be sufficient to make the drug ineffective, whereas for multitarget drugs, several binding sites have to be mutated to achieve resistance.</div><div class="NLM_p">As a consequence, the strategies that have been employed to slow down the emergence of resistance typically rely on targeting several proteins simultaneously, by either a single drug or “cocktails” of drugs. The central goal is obviously to find novel drug classes, but an alternative and very promising strategy is to create hybrid molecules that contain the core pharmacophores of several existing drugs, connected by a linker.<a onclick="showRef(event, 'ref7 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref12 ref13 ref14">(7,12−14)</a> For several difficult to treat infections like <i>Helicobacter pylori</i> or <i>Mycobacterium tuberculosis</i> (but also for pathogens like HIV or <i>Plasmodium</i>), combination therapy is already the only effective treatment, and multitarget drugs are currently being developed.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Additionally, it has been shown that the emergence of resistance for one antibiotic frequently influences (and sometimes increases) the sensitivity to other antibiotics,<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20">(16−20)</a> suggesting that, aside from combination therapy, developing drugs that target the “right” protein combinations may significantly reduce the speed at which resistance develops.</div><div class="NLM_p">A theoretical problem in designing multitarget antibiotics is that the proteins targeted by them may not be similarly druggable in different species. First, species that are distantly related (i.e., Gram positive and Gram negative bacteria) may not share all the proteins targeted by the drug, even if they are essential. Second, and more importantly, the binding sites of homologous proteins in distantly related species may not be structurally identical, and thus the drug might not be able to bind efficiently all of them, even if they are present. These problems are expected to scale exponentially with the number of targeted proteins, and as a consequence, drugs that are multitarget in one species might effectively be single target in another one and therefore not be “resistance resistant”. Therefore, selecting targets that have similar binding sites and functions across the broadest possible spectrum of bacterial species is of great importance to the selection of antibiotic targets.</div><div class="NLM_p">Recently, we have found that the structure of ligand binding sites has profound consequences for the evolution of protein function and structural divergence of binding pockets, particularly in proteins that form homomers–protein complexes made up of multiple copies of the same polypeptide subunit.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In prokaryotes, homomers are by far the most common type of protein complexes, with >50% of bacterial proteins of known structure falling into this category.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Binding sites that are formed by multiple protein chains show much higher structural conservation than sites that are formed by the residues of individual protein chains, and have more similar ligands than binding sites of monomers, or homomers with a single chain binding site.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Since similar binding pockets generally bind similar ligands,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> proteins where distant homologs have similar binding pockets are potentially good candidates for broad-spectrum antibiotic targets. In this paper, using in silico experiments, we show that ligands and de novo designed ligands of homomers with multichain binding sites (MBS, see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) are more likely to bind their homologs than ligands of monomers or homomers with a single-chain binding site (SBS, see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). This shows that considering the quaternary structure of proteins and the structures of their ligand-binding sites can aid the selection of protein targets for new broad-spectrum antibiotics. In addition, using de novo ligand design and methods based on deep learning, we also test whether the chemical compounds that can bind several different receptors have common structural characteristics. We show that fragments similar to the building blocks of existing antibiotics are overrepresented in them and that there is likely to be a trade-off between preventing the emergence of drug resistance and broad spectrum activity. We expect these findings to be useful in selecting targets of novel leads and in generating targeted fragment libraries for antibiotic design.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of homomers with multichain binding sites (MBS), single-chain binding sites (SBS), and an overview of binding site identification in homologs. (A) Structure of nitroreductase ydjA from <i>E. coli</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3bm1">3bm1</a>). The dimer structure has two multichain binding sites, both “sandwiched” between the two chains of the complex. The ligand (flavin-mononucleotide) is displayed in red, and ligand binding residues are in yellow. (B) Structure FabH protein from <i>E. coli</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ebl">1ebl</a>). The dimer has two binding sites, both restricted to a single chain. The ligand (coenzyme A) is displayed in red, and ligand binding residues are in yellow. (C) Structure of SiaP protein from <i>Haemophilus influenzae</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wyk">2wyk</a>). The protein is a monomer, and has a single binding site, with its ligand (<i>N</i>-glyconeuraminic acid) displayed in red. (D) We searched for similar binding sites in homologous proteins with ProBiS and defined the region of binding sites for grid building and docking through local structural alignments. The structural alignment shows the <i>H. influenzae</i> SiaP protein binding site superposed with the binding site (identified by ProBis) of the homologous c4-dicarboxylate-binding protein of <i>Pseudomonas aeruginosa</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4nf0">4nf0</a>); the red box indicates the region of the binding sites. Note that the alignment optimized the superposition of the binding sites and not the global protein structures. (E) Once the binding site has been identified in c4-dicarboxylate-binding protein, the ligand of <i>H. influenzae</i> SiaP protein was docked into it, and the binding energies (i.e., grid score) in the two structures were compared.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23752" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23752" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Ligands of MBS Homomers Bind Their Homologs Significantly Better than Ligands of SBS Homomers or Monomers</h3><div class="NLM_p">In the first step of the analysis we identified bacterial proteins that are likely to be suitable targets for antibiotics. Using BLAST and the prokaryotic proteins present in the Protein Data Bank (PDB), we compiled a data set of 687 pairs of homologous bacterial proteins. We only used proteins that form homomers or monomers. The proteins of the homologous pairs were selected according to the following criteria: (1) none of the proteins have a homolog in the human genome with BLAST <i>e</i>-value <10<sup>–3</sup>; (2) the pairs have homologous regions, with BLAST <i>e</i>-value <10<sup>–5</sup>; (3) at least one protein of the pair is essential; (4) the sequence similarity between them is less than 40% because above 40% protein structures are very similar (see ref <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> for examples); (5) both proteins have a biologically relevant ligand, i.e., present in the BioLiP database;<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and (6) their quaternary structure and binding site type (MBS vs SBS) are similar. Next, in each PDB structure of the protein pairs, the ligand-binding site of every small molecule ligand was identified (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), and we identified the best binding site candidate in the structures of their homologs with ProBiS<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Once the location of the putative binding site was determined in the homolog, we docked the ligand with DOCK<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> to the original binding site and the binding site in the homolog and compared their estimated binding energies (i.e., grid score, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). Only those pockets (and proteins) were used where the performance of DOCK was acceptable; i.e., it could reproduce the position of the original ligand in the top 10 scoring poses (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> for details). In addition, the binding sites of metals and cofactors were excluded from the analysis because metals typically have very small binding sites, while the binding pockets of cofactors are structurally so conserved that they are likely to be similar in bacterial and mammalian proteomes.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This procedure resulted in a set of 1007 binding pocket pairs (78 in MBS homomers, 212 in SBS homomers, and 717 in monomers) between 1464 different binding pockets of 262 different proteins (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S1</a> for a list of pocket pairs). Note that the pocket pairs contain redundancies; the number of nonredundant pockets (excluding pockets with metals and cofactors) is 524 in the original PDB receptors and 940 in their homologs.</div><div class="NLM_p">The binding pocket searches with ProBiS indicate that this data set, despite its much smaller size, shows a qualitatively similar pattern of binding pocket similarity as the entire PDB in our previous study:<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> the frequency of MBS homomer pairs that have no highly similar (ProBiS <i>Z</i> score >2) binding pockets is much smaller than in SBS homomers or monomers (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, <i>p</i> = 0.029 and <i>p</i> = 0.0001, respectively; tests of proportions). Docking of the ligands into their original binding site and the best binding site of their homologs shows a similar trend (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The normalized grid score [(score<sub>original</sub> – score<sub>homolog</sub>)/score<sub>original</sub>] indicates that for MBS homomers, the binding of the ligand in their homolog is nearly as strong as in the original binding site, while in SBS homomers and especially monomers, ligands bind much more weakly at the homologous biding sites (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, <i>p</i> = 0.013 between MBs and SBS homomers, and <i>p</i> ≪ 0.001 for other comparisons, Dunn’s test [Kruskal–Wallis rank sum test]). The performance of DOCK was highly dependent on the size of the ligand (and number of rotatable bonds), as reported previously<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C), and performed best on ligands with 10–30 heavy atoms. The estimated binding energy of ligands (i.e., DOCK grid score, which estimates interaction strength based on a simplified force field) generally follows a linear trend in all three quaternary structure types, with the ligands of monomers having many more clashes in their homologs than homomers (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D–F). Statistical analyses with ANCOVA indicate a similar trend as the normalized grid scores: there is a marginally significant difference between MBS homomers and their homologs (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E), and the difference is largest between monomers and their homologs (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). Taken together, these findings indicate that ligands of MBS homomers bind much more strongly in their homologs than ligands of SBS homomers or monomers.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligands of homomers are more likely to bind their homologs than the ligands of monomers. (A) Frequency of homologous protein pairs without similar ligand binding sites, (ProBiS <i>Z</i> score of >2). See also ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> for general trends in ligand binding site evolution. MBS homomers are much more likely to have an identical binding site in their homologs than SBS homomers (<i>p</i> = 0.029, test of proportions) or monomers (<i>p</i> = 0.00011, test of proportions). (B) The difference between the estimated ligand binding energies (grid score) in the original binding sites and homologous binding sites shows that homomers, and particularly MBS homomers, are much more likely to bind the ligands of their homologs than monomers. Normalized grid score difference was calculated as (score<sub>original</sub> – score<sub>homolog</sub>)/score<sub>original</sub>. All three possible comparisons are significant (<i>p</i> = 0.013 for MBS vs SBS homomer; <i>p</i> ≪ 0.005 for MBS/SBS homomer vs monomers; Kruskal–Wallis rank sum tests). (C) Pose reproduction success rate of DOCK for ligands of different sizes. The high flexibility of ligands above 40 heavy atoms in this particular data set results in a relatively low (∼50%) success rate for large ligands. (D–F) The relationships between ligand size and binding energy (grid score) follow a linear trend in the original ligand binding sites and the binding sites of homologs. The difference between the original binding sites and the binding sites of homologs is lowest in MBS homomers and largest in monomers (ANCOVA).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Characteristics of Antibiotics and de Novo Ligands</h3><div class="NLM_p">We next determined whether de novo designed ligands show similar trends as the previous analysis of the original ligands of the PDB structures. We used de novo DOCK<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> to design novel ligands in each binding site where the pose reproduction of the original ligand was successful, excluding cofactor and metal binding sites and also sites where the original ligand had only one or no rotatable bonds (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> for details), with a maximum molecular weight cutoff of 550 Da. This resulted in 453 binding sites where de novo ligands were built (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S3</a> for the full list). From all de novo ligands that were generated in each receptor we selected the 50 with the lowest grid scores (i.e., the 50 best binders) and used only these in the further analyses, altogether 22 650. Note that de novo ligands were not designed in receptors identified through homology. We also downloaded the SMILES strings of FDA approved antibiotics present in DrugBank<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> from the ZINC15 database<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> to compare the properties of de novo ligands with real antimicrobial agents. Antibiotics with molecular weight of >600 Da were not included for two reasons. First, only a relatively small number of compounds fall into this size category (glycopeptides [e.g., vancomycin], lipopeptides [e.g., daptomycin], or macrolides), and these are typically active only against Gram positive bacteria.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Additionally, these compounds generally have complex, cyclic structures, and designing functional ligands of this size and complexity is beyond the capabilities of the current de novo design tools. The list of antibiotics included is available in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S2</a>.</div><div class="NLM_p">The comparison of molecular weight distributions of antibiotics and de novo ligands shows that the de novo ligands of MBS homomers are somewhat biased toward larger molecular weights than antibiotics, which have a peak between 400 and 500 Da (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B), while monomers (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) and SBS homomers (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S1A</a>) are characterized by more uniform distributions. Next, the 50 best scoring de novo ligands of each receptor were re-docked into their original binding site and to the corresponding binding site in their homologous proteins, and we determined the fraction of ligands that have an approximately similar molecular weight to the original ligand (min. 85%), and bind in the homologous binding sites similarly as in the original site (min 85% of grid score). The comparison of de novo ligands in the different quaternary structures indicates that in homomers, and particularly MBS homomers, the fraction of ligands that bind similarly in homologs and in the original binding site is much higher than in monomers, particularly above 300 Da (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S1B</a>; ∗∗, <i>p</i> ≪ 0.005; tests of proportions). The difference is primarily caused by the higher similarity of binding sites and not by the qualitatively different performance of de novo DOCK in proteins with different binding sites (although it may somewhat contribute to the pattern in SBS homomers) because the fraction of strongly binding ligands (grid score of de novo ligand is min 85% of the original ligand) is comparable in all quaternary structure types (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S2</a>). Surprisingly, SBS homomers perform somewhat better than MBS homomers or monomers (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S2</a>), although the difference is less than 20% for most molecular weight bins.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. General properties of antibiotics and de novo ligands. For each receptor, the 50 best scoring ligands were included. (A) Molecular weight distribution of Drugbank antibiotics, below 600 Da. (B) Molecular weight distributions of de novo ligands in MBS homomers and monomers. The de novo ligands of MBS homomers are significantly larger than of monomers, while the ligands of SBS homomers (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S1</a>) do not show the same trend. (C) The frequency of ligands that bind similarly in homologs and their original binding site is significantly higher among MBS homomers than monomers above 300 Da (∗∗, <i>p</i> ≪ 0.005, tests of proportions). (D) The log <i>P</i> values of antibiotics and de novo ligands. For most antibiotics log <i>P</i> is below 3, but for a significant number of de novo ligands it is above 3, particularly above 300 Da. (E) The druglikeness of antibiotics and de novo ligands follows the same trend and declines above molecular weight of 300. (F) Below 300 Da the synthetic accessibility of de novo ligands is considerably worse than of real antibiotics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next we examined whether there are consistent qualitative differences between antibiotics and de novo ligands for the three different quaternary structure types. We used a chemical variational autoencoder (VAE), a recently developed method based on deep learning,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> to estimate the synthetic accessibility<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (SAS), quantitative estimate of druglikeness<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (QED), and octanol–water partition coefficient (log <i>P</i>) values for antibiotics and each de novo ligand using their SMILES (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D–F). The solubility in organic solvents (log <i>P</i> values) of de novo ligands gradually increase with molecular weight and, for ligands with molecular weight above 300 Da, can substantially exceed the values observed in antibiotics, which are generally below 3 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). The quantitative estimate of druglikeness (QED, higher is better) is a more modern estimate of druglikeness than Lipinski’s rule of five and is a combination of eight different molecular descriptors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Antibiotics and de novo ligands show a qualitatively similar, nonlinear pattern, with the most druglike compounds having molecular weight of 200–400 Da (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). However, even the compounds with molecular weight above 400 Da have reasonably good (>0.5) QED estimates, and since antibiotics and natural compounds are known to not follow the same trend in druglikeness as synthetic drugs,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the druglikeness of de novo ligands should be generally seen as satisfactory. The largest difference between antibiotics and de novo ligands is in their synthetic accessibility (SAS, lower is better), which estimates the ease of synthesis of chemical compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Antibiotics show a clearly increasing trend with molecular weight, with SAS being below 3 for small compounds (easy synthesis) and gradually increasing to ∼6 (more difficult synthesis) above 300 Da. In contrast, de novo ligands show almost no change with molecular weight, with high variability in every weight class (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Moreover, their SAS below 300 Da is much higher than of real antibiotics. This is in agreement with previous reports that the synthetic accessibility of de novo designed ligands can be low. However, it should be noted that most natural products also fall in the range of SAS = 3–6, and thus the values we observe are not prohibitively high.</div><div class="NLM_p last">Finally, using the VAE, we tested whether there are consistent differences in the chemical composition of de novo ligands of MBS homomers, SBS homomers, and monomers. The VAE is fundamentally a pair of neural networks that “encode” (convert to) and “decode” (convert back) discrete chemical compounds into a continuous, high dimensional space called “latent space”, where they are represented as a numerical vector. The vector representation offers several powerful operations on chemical compounds, like generating entirely novel molecules, interpolating between molecules, or optimizing existing molecules<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (see also further analyses below). Additionally, since the latent space is structured, where similar compounds are located close to each other, it can also be used to visualize whether there are any structural clusters in the de novo ligands. We transformed each de novo ligand into a vector in the latent space using the encoder module of the VAE. The latent space has 196 dimensions, and to reduce its dimensionality, we used Barnes–Hut t-SNE<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> for clustering. This transforms the position of chemical compounds in a high dimensional space into 2D, which is suitable for visualization. The 2D maps of de novo ligands show that monomers and MBS homomers have different distributions in the chemical space, and SBS homomers show an intermediate pattern (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S3</a>). However, much of the variation is simply due to differences in size (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), and aside from these coarse grained differences, no distinct, quaternary structure specific clusters could be identified: in the areas of the plot where all quaternary structure types are present, their distribution is homogeneous (e.g., red circle, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S3</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> The Best Ranking de Novo Ligands Show Similar Binding Patterns as the Original Ligands of the Receptors</h3><div class="NLM_p">Next we tested whether the de novo ligands show similar patterns in binding the homologous binding sites as the original ligands of the receptors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). We found that this is the case for the best ranking ligands, although the pattern is less pronounced than for the original ligands (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) most likely due to the higher flexibility of de novo ligands. For the 10 best scoring de novo ligands of every receptor, the normalized grid scores are significantly different in all three possible comparisons (<i>p</i> ≪ 0.001 for all three comparisons, Dunn’s test, [Kruskal–Wallis rank sums test], <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The frequency of clashes (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) is significantly higher in monomers than in MBS and SBS homomers (<i>p</i> < 0.001 in both cases, tests of proportions), but there is no difference between MBS and SBS homomers (<i>p</i> = 0.369, test of proportions). The grid scores of de novo ligands show a similar linear scaling with their size, in their original and homologous binding sites, with somewhat weaker binding in homologs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C–E). For the lower ranking ligands, we found that the difference between ligands of MBS and SBS homomers is not consistent; however, de novo ligands of monomers have consistently worse (higher) scores in the binding sites of homologs than MBS or SBS homomers (not shown).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. The best scoring de novo ligands show qualitatively similar, although weaker trends than the original ligands of the receptors. (A) Difference between the normalized grid score of the 10 best ligands of each receptor. All three possible comparisons are significant (<i>p</i> < 0.0001 Kruskal–Wallis rank sum tests). (B) The frequency of clashes is significantly higher in monomers than in homomers (<i>p</i> < 0.001 for both MBS and SBS homomers, tests of proportions), but there is no difference between MBS and SBS homomers homomers (<i>p</i> = 0.43). (C–E) Relationships between ligand size and estimated binding energy (grid score) of the best scoring ligand of each receptor in the original ligand binding site and the binding sites of homologs. Similar to the original ligands of the receptor, the relationship between size and grid score is linear, and the difference between the original binding sites and the binding sites of homologs is smallest in MBS homomers and largest in monomers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Characteristics of de Novo Ligands That Can Target Several Proteins</h3><div class="NLM_p">Next, we examined whether de novo ligands that are likely to bind several proteins have common structural characteristics. We performed the analysis in two steps. First, since de novo DOCK uses a fixed fragment library to build the ligands, we performed a fragment enrichment analysis of the de novo ligands that have structurally similar ligands in multiple receptors. Second, we optimized/evolved these ligands using the VAE to test whether structural motifs that were absent in the fragment library of de novo DOCK emerge during the optimization.</div><div class="NLM_p">From the de novo ligands (using the best 50 in each receptor), we selected ligands predicted to target several proteins with the following procedure. (1) First we converted each de novo ligand into its vector representation in the latent space of the VAE and calculated all possible (256 million) distances between the 22 650 de novo ligands in the latent space, using their vector representation <named-content content-type="anchor" rid="fig4" type="simple"></named-content>(see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). (2) Next we selected the pairs of de novo ligands where their distance in the latent space is less than 13, and both de novo ligands have lower (better) grid score than the original ligand of the binding site. The distance in the latent space correlates with the structural similarity of compounds; however, it also depends on the size of the ligand: in the case of ligands of 300–400 Da, the distance 13 roughly corresponds to Tanimoto similarity 0.7 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B,C), while for larger ligands, larger distances have to be used for the same structural similarity (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S4</a>). We chose the distance cutoff 13, because the druglikeness of de novo ligands in our data set is highest below 400 Da; see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E. (3) From the ligand pairs, we selected the ligands that have a low (<13) scoring pair in at least two more bacterial species than the query ligand. (4) We discarded ligands with properties that are substantially different from real antibiotics: molecular weight of <300 Da, log <i>P</i> > 3, and where SAS > (molecular weight/100) + 1. These steps reduced the total pool of ligands to only 56 (1 duplicate ligand), which are expected to be druglike, have structurally similar ligands in the receptors of minimum three different species, and are likely to be easy to synthesize (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplmentary Data</a> for the list of ligands and their structures).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Outline of the structural comparisons using the VAE vector representation. (A) De novo ligands were converted to vector, which represents the parameters of a statistical distribution and defines the location of the compound in the latent space. The Euclidean distance between the vectors was used to measure the structural similarity between all ∼256 million possible pairs of compounds. (B) The frequency of pairs with distance below 13 is negatively correlated with their molecular weight. (C) For molecules above 300 Da the distance cutoff 13 results in an approximate Tanimoto similarity of 0.7 (most of them are in the range of 300–400 Da).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The statistical variability of such a small set of ligands is expected to be much larger than for the whole data set of 22 650 molecules. Therefore, to estimate the uncertainty associated with our analysis, we repeated the entire de novo design process and generated a second, fully independent set of de novo ligands using the same methods. In this second set, de novo ligands were built in 452 binding sites (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S3</a>). This resulted in an additional, independent set (set 2) of 73 putatively antibiotic-like ligands (with three duplicate ligands; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a>).</div><div class="NLM_p">Several of the selected de novo ligands are reminiscent of compounds and drugs with known antimicrobial activity, particularly of quinolones, quinazolinones, oxadiazoles, and morpholine antifungals (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S5A</a>; note the top left compound of <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, which resembles a hybrid between a quinolone and an oxazolidinone antibiotic, and the bottom left compound, which is similar to morpholine antifungal).<named-content content-type="anchor" rid="fig5" type="simple"></named-content> Quinazolinones and oxadiazoles represent two new groups of antibacterials that were shown to be effective against methicillin and vancomycin resistant <i>Staphylococcus aureus</i>.<a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42 ref43 ref44">(38−44)</a> From the 380 fragments of the fragment library, 11 (set 1) and 12 (set 2) are significantly enriched in the compounds (<i>p</i> < 0.05, tests of proportions), and there are overlaps between the two sets of fragments (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B–D; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S5B–D</a>). The most common fragment is the carboxyl group (side chain 23), and in both sets halogen-containing fragments are enriched. Additionally, in set 1 the oxadiazole linker (lnk.36), and the 4-morpholinyl side chain (sid.68) that is the core pharmacophore of several ergosterol synthesis inhibiting antifungals (amorolfine, fenpropimorph, tridemorph),<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45−47)</a> is enriched, while in set 2, a quinolone-like linker (quinazolinone, lnk.308, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S5C</a>) is enriched. However, although being significant in only one of the sets, quinolone-like (quinazolinone) and morpholine groups are common in both sets (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a>; note that only quinazolinone and not quinolone fragments was present in the fragment library).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Properties of the selected de novo ligands in set 1. (A) Examples of interesting de novo ligands. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a> for the full list. Several of them are reminiscent of quinolones, quinazolinones, oxadiazoles, and morpholine antifungals. Oxazolidinone, quinazolinone, oxadiazole, and morholine groups are highlighted with red, and the DOCK anchor fragments are highlighted with blue. A particularly interesting case (top left) contains a quinazolinone linker and an oxazolidinone side chain (oxazolidinones are among the newest antibiotics, e.g., linezolid, tedizolid,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> used to treat vancomycin resistant Gram-positive bacteria). The compound at the top center is reminiscent of an oxadiazole antibiotic, while the bottom left compound resembles morpholine antifungals. (B) Word cloud of the significantly enriched fragments (<i>p</i> < 0.05, tests of proportions; sid. = side chain, a fragment with only one connection; lnk. = linker, a fragment with two connections; scf. = scaffold, a fragment with three connections). The size of the symbols corresponds to the abundance of the fragments. (C) Enrichment of the fragments. Enrichment was calculated as frequency in set 1/frequency in all de novo ligands with similar size and properties to set 1. (D) 2D structures of the significantly enriched fragments. Several fragments with halogen groups (Cl/Br) are present among them and also a morpholine side chain (sid.68).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next we tested whether the selected ligands originate from binding sites of proteins with similar functions to the proteins that are targeted by the antibiotics they are similar to. Quinolones target the DNA binding gyrase and topoisomerase IV,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> quinazolinones and oxadiazoles target penicillin binding protein 2a,<a onclick="showRef(event, 'ref38 ref39 ref44'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref44">(38,39,44)</a> and morpholine antifungals inhibit ergosterol synthesis.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The de novo ligands of the two sets originate from the binding sites of 23 and 25 proteins (set 1 and set 2, respectively, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S4 and S5</a>). Neither their quaternary structure (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S6A</a>) nor their gene ontology (GO) terms show significant enrichment compared to the full set of <named-content content-type="anchor" rid="fig6" type="simple"></named-content>proteins we used. The frequency of cellular component GO terms indicates that they are present in several different cellular components, including the cytoplasm, cell wall, or periplasmic space (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S6B</a>). The frequency of molecular function GO terms show that most of them have enzymatic functions that are not related to the functions of quinolone, quinazolinone, oxadiazole, or morpholine antimicrobials, although many of them utilize NADP cofactors and ATP in catalysis (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C,D; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S6C</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Tables S4 and S5</a>). While we excluded cofactor-binding sites from our analysis, the binding sites of substrates and cofactors frequently form a continuum, and the structural constraints imposed by nucleic acid containing cofactors or ATP could contribute to the similarity of several de novo ligands to quinolones.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Gene ontology analysis of the receptors of selected ligands. (A) The quaternary structure composition is not significantly different from the total set of proteins (<i>p</i> = 0.42), although the frequency of MBS homomers is lower, as a result of the requirement to have similar structures in at least three species. (B) Graph of cellular component terms associated with the proteins. Note that the color-coding of terms (intensity of red) corresponds to the frequency of terms and not statistical significance. The proteins are present in most cellular components, including cell wall, periplasmic space, plasma membrane, and cytoplasm. (C, D) Graphs of molecular function terms associated with the proteins. The two highest-level terms are binding (C) and catalytic activity (D), with nucleoside/nucleotide binding and oxidoreductase activity being the most common terms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Optimization and Evolution of de Novo Ligands Using AI</h3><div class="NLM_p">The main limitation of using fragment libraries by de novo design tools is that the extent of chemical space that can be explored by the de novo ligands is limited by the fragment library. To overcome this, we further optimized the selected sets of de novo ligands (56 and 73 compounds) using the VAE (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). We applied the following, so-called “hill climbing” protocol for ligand optimization. (1) First we docked the selected ligands to all the receptors where a de novo ligand with distance in the latent space less than 13 was found and selected the three receptors, each from a different species, where the grid score is the lowest. (2) Next we encoded the ligand and sampled its neighborhood in the latent space for structurally related molecules (see <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S7</a>). We used several different <i>Z</i>-distance cutoffs (see ref <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>), from 0.5 to 50, which correspond to increasingly larger added noise levels to the encoded molecule. (3) The neighbors were converted to 3D structures and were docked to the three previously selected receptors. If at least two neighbors had lower (better) grid scores than the encoded molecule, the best performing neighbor was selected, and the cycle was repeated until no improvement was detected (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). This process generally resulted in lower grid scores (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S8</a> for an example), and it also takes into account the similarity of properties like log <i>P</i>, QED, or SAS during the optimization of the ligand. (4) In the final step, using the ligands of the entire optimization process (including the neighbors that were not selected for further optimization), we selected the ligand with the best average grid score that also satisfied the same criteria as the original de novo ligand: log <i>P</i> < 3, SAS < (molecular weight/100) + 1, and molecular weight of >300.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Optimization and evolution of ligands with AI. (A) Outline of the method. First, the selected de novo ligands were encoded with the VAE, and the latent space was sampled in their vicinity to search for related structures. Next the, samples were “decoded” into chemical structures and were docked into three different receptors, and the best performing (lowest scoring) molecule was used to repeat the cycle until there was no more improvement in the estimated binding energies (grid score). See also <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S8</a>. (B) The optimization resulted in a comparable improvement in binding energies (grid score), irrespective of the <i>Z</i> distance cutoff used. (C) The median Tanimoto similarity of optimized ligands and de novo ligands is 0.7. (D, E) The frequency of four- and three-membered rings in the optimized ligands is significantly higher than in the original de novo ligands and the random expectation (∗, <i>p</i> < 0.05; ∗∗, <i>p</i> < 0.005, randomization tests). Distance 0 indicates the original de novo ligands, red horizontal bars indicate the observed frequency of four- or three-membered rings, while black horizontal bars indicate their expected frequency. Violin plots show the frequency distribution of 10 000 random replicates. Outliers above 20% were not plotted for clarity. Note that in the case of four-member rings, the expected frequency is higher than their frequency among the de novo ligands (<i>Z</i> = 0) because they are more common in the VAE samples. (F) Example of the emergence of a β-lactam-like ring in a de novo ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><named-content content-type="anchor" rid="fig7" type="simple"></named-content>The majority of unoptimized de novo ligands could bind all three receptors without clashes (78% and 75%, in set 1 and set 2, respectively), and 98 and 95% of them could bind at least two. Nevertheless, the optimization resulted in a clear improvement of grid scores in both sets of molecules (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S10A</a>). While the final optimized molecules frequently differed, there was little difference in the magnitude of the grid score improvement when different noise levels (<i>Z</i>-distances) were used. This is due to several processes: first the characteristics of the VAE sampling play a role, as the structural diversity of the sampled SMILES changes little with the magnitude of the selected noise (<i>Z</i>-distance) level (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S9</a>). With larger noise levels, the probability of a obtaining a valid SMILES string decreases, and in consequence most decoded ligands still originate from the neighborhood of the input even when the added noise (<i>Z</i>-distance) is large. (Note that the VAE is inherently stochastic, and decoding the same vector several times, without any added noise, results in variability in the returned SMILES.) Second, the ligands can probably be improved only to a limited degree, and the different runs reached different (and comparable) local optima. Similarly, the Tanimoto similarity score of the final optimized molecules with the original de novo ligand depends little on the <i>Z</i>-distance used and is approximately ∼0.75 for all <i>Z</i> cutoffs (<named-content content-type="anchor" rid="fig8" type="simple"></named-content><a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S10B</a>).</div><div class="NLM_p">The optimized ligands show structural differences compared to the original de novo ligands. Quinolone-like and aromatic rings were generally not modified substantially by the optimization, while morpholine groups (and nonaromatic rings) were more variable and were frequently modified or substituted (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data Sets</a>). Currently it is unclear to what degree these differences are caused by the nonhomogeneous nature or other characteristics of the latent space, as chemical generative methods are still under rapid development<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> (see also Segler et al.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> for a different, recurrent neural network based approach). We tested the optimized (and de novo) ligands for toxicity and the presence of pan-assay interference compounds (PAINS)<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> with FAF-Drugs4<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and the rd_filters tool, using the Glaxo filter.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> We found that none of the de novo ligands and only a small fraction of optimized ligands (∼2%; 4 in set 1 and 7 in set 2, using all <i>Z</i>-distances) contain PAINS fragments. The fraction of putatively toxic (“rejected”) compounds is, 6% and 9%, while the fraction of compounds with low-risk structural alerts is 50–60% both in de novo and in optimized ligands in the two sets using FAF-Drugs4 (note that low-risk structural alerts are frequent in existing drugs). The Glaxo filters flag ∼16% of the de novo and ∼20–21% of the optimized compounds as problematic in both sets (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S6</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a>). However, most likely both tools underestimate the frequency of toxic or unstable fragments.</div><div class="NLM_p">Generally the optimized ligands are characterized by a significantly higher frequency of three- and four-membered rings than the original de novo ligands (red bars, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D,E, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S10C,D</a>). This pattern can be the result of two separate processes. First, in the molecules returned by the VAE sampling, the frequency of small rings can be higher than in the original de novo ligands due to the characteristics of the training set of the VAE. Second, such fragments can be enriched due to the structural constraints imposed by binding in several receptors. We separated these two processes by Monte Carlo simulations. We determined the presence of three- and four- membered rings in the molecules returned by the VAE sampling for each de novo ligand with RDKit. Next, we resampled the returned molecules 10 000 times to determine the random expectation for the frequency of three- and four-membered rings in the optimized ligands. The results show that the frequency of three- and four-membered rings in the final, optimized ligands is significantly higher (randomization test, see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>) than the random expectation for most <i>Z</i>-distances, despite the fact that in the case of four-membered rings the expected frequency is considerably higher than their frequency in de novo ligands (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D,E, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S10C,D</a>). This indicates that the necessity to bind multiple receptors selects for small rings, and their enrichment is not a simple byproduct of the sampling characteristics of the VAE.</div><div class="NLM_p last">Neither the fragment library of DOCK nor the compounds of the VAE were specifically tailored for antibiotic design. The fragment library of DOCK was extracted from 13 million ZINC compounds by selecting the most common ones,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> while the VAE was built from 250K randomly selected ZINC molecules.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Using the optimized compounds to update the de novo fragment library and repeating the entire de novo design and VAE optimization process several times may be useful in evolving fragment libraries tailored for specific tasks from a generic one (both for computational or experimental screening<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a>). Using DOCK, we extracted the fragments from the optimized ligands and filtered out those already present in the input de novo ligands with Open Babel and also those lacking a carbon atom. This resulted in a very high diversity of fragments: altogether we identified 352 new fragments in the two sets (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S11</a>), which do, however, contain several fragments that are toxic, reactive, or unstable, like epoxide, aziridine, or cyclobutadiene-like rings (and others). Note that different fragments can contain the same substructure if the number and location of their attachment points (−Xx) are different. However, the majority of them are present in only in a single structure, and selecting the ones that are present in at least two structures in the same VAE batches (and excluding putatively toxic ones) results in a much smaller set of 28 fragments (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>), of which four contain halogens and seven contain four-membered rings, that are almost completely absent in the fragment library of DOCK. Several of them contain reactive double bonds; thus in practice their saturated versions should be used. The effect of such (e.g., azetidine-like) four-membered rings on toxicity is less clear; azetidine-based inhibitors of herpesvirus proteases are known and being developed,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and azetidines have been suggested for use as peptidomimetics in medicinal chemistry.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> However, exactly due to its incorporation into proteins (as a substitute of proline), azetidine-2-carboxylic acid is known to be toxic.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Trade-Off between Preventing Resistance and Accumulation in Gram-Negative Bacteria</h3><div class="NLM_p">One of the key difficulties in developing broad spectrum antibiotics that are active against both Gram-positive and Gram-negative bacteria is due to the outer membrane and efficient efflux pumps in the latter, which prevent the accumulation of antibiotics. In general, accumulating compounds are expected to be small (<600 Da), but currently there are no established rules that would enable the design of compounds with a high likelihood off accumulation in Gram-negative bacteria. Recently it has been suggested that a few key parameters, like the number of rotatable bonds, the globularity of the molecule, and the presence of ionizable nitrogen (particularly primary amines) might be important for accumulation and passing through the porins of the outer membrane.<a onclick="showRef(event, 'ref59 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref59 ref60 ref61">(59−61)</a> It is unclear how general these rules are, as they were derived from a relatively small set of accumulating molecules; broad-spectrum β-lactam antibiotics (but also polymyxins, macrolides) typically have many more rotatable bonds than recommended, and neither fluoroquinolones nor broad-spectrum β-lactams are particularly enriched in ionizable primary amines (in fact, frequently lack them). However, some compounds active only against Gram-positive bacteria could be successfully converted to broad-spectrum antibiotics using these rules,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> and recent measurements of accumulation in Gram-negative bacteria<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> also provide some support for them, which indicates that they are nevertheless a step in the right direction.</div><div class="NLM_p">To test whether optimization to bind multiple receptors might influence the properties necessary for accumulation in Gram-negative bacteria, we examined whether it influences the number of rotatable bonds and globularity of the designed compounds (rotatable bonds were calculated ignoring hydrogens; thus methyl or amino groups were not assigned as rotatable; see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>). We separated the possible biases of the VAE from the effect of optimization by calculating the number of rotatable bonds (and globularity) in all the compounds returned by the VAE when de novo ligands were used as the input (expectation) and compared it with the number of rotatable bonds (and globularity) in the final optimized ligands. We find that in both sets 1 and 2, optimization resulted in a highly significant increase in the number of rotatable bonds in all VAE optimized batches (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>) and that there is a weak but significant correlation between the change in rotatable bonds and improvement in grid score (note that in the case of set 1 it is significant [<i>p</i> = 7.88 × 10<sup>–4</sup>] only after removing the outliers with a rotatable bond change of −2). The globularity of compounds (measured with the plane of best fit [PBF] algorithm;<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>) does not show a similar consistent change (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S12</a>), although in some VAE batches of set 2, we do observe a slightly increased globularity compared to de novo ligands, mostly due to biases of the VAE (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S12C</a>). The frequency of compounds with primary amines is relatively high in the de novo ligands (23% and 13% in set 1 and set 2, respectively), and somewhat lower in the optimized ligands (not shown). However, this is most likely due to the low frequency of primary amines in the compounds used to train the VAE (2.7%) rather than the result of binding multiple targets.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Fragments of the optimized ligands, which are present in at least two compounds in any of the VAE batches (excluding fragments that are likely to be toxic). Several halogenated compounds and four-membered rings are present among them. Since the four-membered rings frequently contain reactive double bonds, in practice their saturated versions should be used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Ligands optimized for binding multiple receptors have more rotatable bonds than the original de novo ligands. (A) Number of rotatable bonds in de novo and VAE optimized ligands of set 1. Dark red represents the original de novo (<i>Z</i> = 0) or final optimized ligands, and blue indicates the expected number of rotatable bonds in the compounds returned by the VAE, without selection for multitarget binding. This was calculated as the averages of all ligands returned by the VAE when the de novo ligands were used as the input. In all VAE batches the number of rotatable bonds is significantly higher than in the expectation or de novo ligands (∗∗, <i>p</i> < 0.005, Wilcoxon tests on the differences from de novo ligands). (B) Relationship between the change in rotatable bonds and grid score in set 1, using the pooled data of all VAE batches. The correlation is not significant due to a few outliers with rotatable bond change of −2; excluding them results in significant correlation (<i>p</i> = 7.88 × 10<sup>–4</sup>). (C) Number of rotatable bonds in de novo and VAE optimized ligands of set 2. The color coding is the same as on panel A. Similar to set 1, in all VAE batches the number of rotatable bonds is significantly higher than in the expectation or de novo ligands (∗∗, <i>p</i> < 0.005, Wilcoxon tests on the differences from de novo ligands). (D) In set 2, the correlation between the change in rotatable bonds and grid score is highly significant (<i>p</i> = 7.57 × 10<sup>–10</sup>), indicating that the more flexible is the molecule, the better it can bind multiple receptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taken together these results indicate that binding multiple receptors selects for flexible compounds with more rotatable bonds. Thus, preventing the evolution of resistance by multitargeting is likely to have the side effect that compounds capable of binding several receptors efficiently are at the same time less likely to accumulate in Gram-negative bacteria. Since high flexibility is the result of the necessity to adapt to binding sites with somewhat different topologies, one possible way to overcome this trade-off is to target binding sites with high structural similarity; thus MBS homomers may be preferable targets over other SBS homomers or monomers also for this reason.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Drug discovery is an extremely lengthy and costly process: on average, developing a new drug takes ∼14 years and costs an estimated 1–2 billion USD,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> more than the cost of sending a spacecraft to an asteroid.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> One of the earliest, and most critical steps in the drug discovery process is the identification of a druggable target protein, for which a ligand that modifies its function and has a therapeutic effect can be designed. Selecting the right drug target is critical, as poor target protein selection is one of the main causes of the low (∼10%) success rate of drug candidates entering the clinical phase.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Our results indicate that considering the quaternary structure of proteins can help in the selection of drug targets where the goal is targeting several different pathogen species. Ligands and de novo ligands of homomers, particularly MBS homomers, are much more likely to bind the binding sites of their equally diverged homologs than monomers (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). Since high conservation is always the consequence of strong constraints on function, the higher structural conservation of multichain binding sites in homomers<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> also indicates that such sites are less likely to accumulate mutations than the binding sites of SBS homomers or monomers. Finally, their more conserved binding sites reduce the need for flexibility and might improve the chances of accumulation in Gram-negative bacteria.</div><div class="NLM_p">Successful antiretroviral drugs offer insights into this hypothesis, as the very high evolutionary rates of drug targets is a particularly severe problem in the treatment of retroviral infections like HIV. Interestingly, several antiretroviral drugs bind residues from more than one protein chain in their targets. HIV protease is an MBS homomer, and drugs targeting it bind both chains of its binding site.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Besides binding the active site, some HIV protease inhibitors (darunavir and tipranavir) also inhibit the dimerization of the protease, which has been suggested as a factor in the slow evolution resistance for these compounds.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Most HIV reverse transcriptase (RT) inhibitors are nucleoside analogs, although some of the non-nucleoside analog inhibitors like rilpivirine and etravirine bind residues from both chains of the RT heterodimer.<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67,68)</a> Drugs targeting HIV integrase typically bind the catalytic site, which interacts with DNA; however compounds not targeting the catalytic site bind allosteric pockets with residues from multiple chains.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Interestingly, one HIV integrase inhibitor, elvitegravir, is also characterized by a fluoroquinolone-like structure.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a></div><div class="NLM_p">Taken together, our results indicate that the high binding site similarity of MBS homomers makes them promising targets for broad spectrum antibacterial agents. Moreover, their slow structural change during evolution<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> suggests that targeting MBS homomers might also slow down the evolution of resistance, due to the high functional constraints on such sites. Targeting multichain binding sites might also result in the inhibition of protein complex formation itself, which is likely to have a significant effect on the evolution of resistance. Finally, our recent results show that allostery is much more frequent among MBS complexes than among SBS complexes, particularly in the case of homomers.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> This indicates that MBS homomers also offer more targetable pockets than SBS homomers for drug development, and more diverse mechanisms can be exploited for developing novel inhibitors. Although our work focused primarily on homomers, we also note that a “trivial” way of achieving multitargeting is to target multichain binding sites of heteromeric protein complexes that are formed by multiple distinct polypeptide subunits. In humans, such sites are characterized by the highest frequency of pathogenic mutations in all quaternary structure types, indicating strong purifying selection.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, MBS heteromers are generally not characterized by more conserved binding sites<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> or much higher frequency of allostery than monomers.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></div><div class="NLM_p">The analysis of the structural properties of de novo ligands with similar de novo structures in several receptors indicates that such ligands do have common structural characteristics (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S5</a>): they are enriched in halogen-containing, quinolone-like (quinazolinone), oxadiazole, and morpholine-like groups. Such fragments are characteristic in current broad-spectrum fluoroquinolone antibiotics (also the HIV integrase inhibitor elvitegravir), quinazolinone and oxadiazole antibacterials, and morpholine antifungals, suggesting that morpholine-based inhibitors could also be developed for bacteria. Additionally, the high frequency of quinazolinone fragments in our data sets suggests that quinazolinones are promising compounds for further development.</div><div class="NLM_p">Halogens, particularly fluorine, are routinely used in medicinal chemistry for optimization and to improve ligand–target interactions.<a onclick="showRef(event, 'ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref72 ref73">(72,73)</a> However, the fact that halogen groups primarily interact with the carbonyl oxygens of the protein backbone and not with the side chains<a onclick="showRef(event, 'ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref72 ref73 ref74">(72−74)</a> makes them particularly suitable for use in “resistance-resistant” drugs,<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> as point mutations affect primarily the side chains of amino acids and change the protein backbone much less. Ligand–backbone interactions are already used in HIV reverse trascriptase inhibitors<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> and the protease inhibitor darunavir<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> to slow down the emergence of resistance (although in the latter it is not a halogen that interacts with the backbone). Additionally, our results suggest that the high overall frequency of halogens in drugs (∼40% <a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a>) might be partly the result of interacting with several targets, even in the case of drugs that were originally designed to target only a single protein.</div><div class="NLM_p">Most of the selected de novo compounds could bind several receptors without modifications, and they could be further improved by optimization with the VAE (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S10</a>). In the majority of the cases the optimization resulted in compounds that are not dramatically different from the original de novo ligands, with Tanimoto similarity of ∼0.7–0.75. However, in a fraction of the cases, it resulted in substantially different compounds (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data Sets</a>). A characteristic structural difference in the optimized ligands is the appearance of small, three- and four-membered (sometimes β-lactam-like; see <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>F) rings, of which the latter are effectively absent in the original de novo ligands.</div><div class="NLM_p">While the selected compounds contain many novel topologies, most of the interesting enriched fragments (quinolones/quinazolinones, oxadiazoles, morpholines, four-member rings [β-lactams/monobactams], organohalogens) are already used in the core pharmacophores of existing antimicrobials. This indicates that our approach has the potential to identify fragments relevant for antibiotic design but also that the chemical space of broad-spectrum antibiotics is not unlimited and, unfortunately, that the perception of the industry that many of the “low hanging fruit” antibiotic classes might have already been discovered<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> is not unfounded. From the strategies that help to preserve our current antibiotics, combination therapy is likely to be a successful strategy<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> as it effectively implements multitargeting, while recent work suggests that antibiotic cycling and mixing may not be very effective in practice.<a onclick="showRef(event, 'ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref76 ref77">(76,77)</a> We did not customize the fragment library used by de novo DOCK toward antibiotics, and also the VAE was based on a random selection of compounds<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> without any specific tailoring for antibiotic design. Thus, the emergence of compounds enriched in antibiotic-like fragments is not the result of such biases and indicates that the combination of fragment library based tools like DOCK, and deep-learning based tools like the VAE, that can explore the chemical space in ways fragment library based tools cannot is a powerful combination and can also be used to customize fragment libraries for a specific task. Its main current limitation is that some of the resulting compounds can be toxic, difficult to synthesize, or unstable (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Table S6</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a>), and that occasionally the combination of certain fragments can result in compounds with protonation states that are not relevant at physiologically relevant pH.</div><div class="NLM_p last">Another limitation of our analysis is that the protein targets that resulted in the ligand sets used for the enrichment analysis and evolution using AI represent a relatively small set of molecular functions (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S6</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Tables S4 and S5</a>). Currently only a small fraction of the essential proteins of clinically relevant bacteria have a 3D structure deposited in the PDB, and even the available structures frequently cover only fragments or individual domains of them. Thus, our analysis was by necessity limited by the available structures, and a large, possibly global effort to determine the structures of the “essentialome” in the clinically most problematic microbes (e.g., ESKAPE) could have a major impact on designing new antibiotics with fundamentally novel structures.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06113" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06113" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Selection of Bacterial Proteins</h3><div class="NLM_p last">We selected the bacterial proteins for the analysis as follows. First, using the OGEE,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> CEG,<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> and DEG<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> essential gene databases, we identified the known essential prokaryotic genes in the PDB. Next, using BLAST with an <i>e</i>-value cutoff of 10<sup>–3</sup>, we removed those proteins from the data set that have a homolog in the human genome, the ones that form heteromeric protein complexes, and ones that have no structure with a small molecule ligand in the BioLiP database.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Finally, using BLAST with an <i>e</i>-value of 10<sup>–5</sup> cutoff, we identified pairs of homologous bacterial proteins in the PDB where (1) the sequence similarity is below 40%, (2) at least one member of the pair is essential, (3) none of them have a homolog in the human proteome (<i>e</i>-value of 10<sup>–3</sup>), (4) their structures overlap in the alignment, thus their structures also contain homologous regions, (5) they have similar quaternary structure, and (6) the type of ligand binding (MBS or SBS) is similar. The list of protein pairs and the PDB codes used in the analysis is available in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">Supporting Information Table S1</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Docking, Preparation of Receptors and Ligands, and de Novo Design of Ligands</h3><div class="NLM_p">We used DOCK 6.8<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and its utility tools for binding site and ligand preparation, and docking. The first biological assembly was used for all PDB entries. Since some DOCK utility tools are unable to process large proteins and complexes, before docking we identified the residues of the receptor within 10 Å of the ligand with ProBiS, discarded all other residues of the structure, and built the grids using this substructure of the receptor. This did not have any measurable effect on the performance or accuracy of DOCK (the grids were built within 5 Å of the ligand) and enabled us to process large complexes. Receptor proteins and ligands were prepared with a standard procedure: in the receptor, incomplete side chains were completed, hydrogens were added, residues with low occupancy were removed, and Amber charges were added with the dock-prep tool of Chimera.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> Ligands were converted to mol2 format, and hydrogens and Amber charges were added with Chimera. In a small number of cases where Chimera was unable to process the ligand, we added hydrogens and (Gasteiger) charges with OpenBabel.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> To estimate the pose reproduction success rates, we first minimized the ligand through rigid docking, next docked it to its receptor with the FLX (flexible) algorithm.<a onclick="showRef(event, 'ref28 ref83'); return false;" href="javascript:void(0);" class="ref ref28 ref83">(28,83)</a> The parameters were similar as in ref <a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a>, the main parameters being max_orients = 1000, pruning_max_orients = 1000, pruning_clustering_cutoff = 100 (see <a href="http://ringo.ams.sunysb.edu/downloads/SB2010/FLX.in" class="ext-link">http://ringo.ams.sunysb.edu/downloads/SB2010/FLX.in</a> for the original parameters). We used only those receptors in further analyses where at least one of the first 10 clusterheads had RMSD < 2 Å with the original position of the ligand. We used this relaxed strategy because in the case of large ligands with several rotatable bonds, it is common that the core of the ligand is in the correct position, but certain side chains are not, pushing their RMSD above 2 Å. Furthermore, ligands in the PDB are not necessarily in a location that is the energetically most favorable, and in such cases a ”successful” pose reproduction is actually an error. Further docking was performed with the FLX protocol, both for the original binding site of the ligand and for the homologous binding site.</div><div class="NLM_p last">De novo ligands were built with the de novo algorithm of DOCK (a prerelease version of 6.9),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and we used the fragment library distributed with it, which was built using 13 million compounds of the ZINC database.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Only those binding sites were used where pose reproduction was successful, and binding sites of cofactors or metals were not used. We followed the following procedure for ligand generation: (1) First, we identified the number of rotatable bonds of the original ligand of the receptor with DOCK. (2) Next, we decomposed the original ligand of the receptor to fragments and identified the largest one. (3) From the fragment library we randomly selected a fragment with ±1 heavy atoms as the largest fragment of the original ligand for anchor. (For ligands where the largest fragment had 9 or more heavy atoms, fragments with 9–12 heavy atoms were used). (4) Next, the number of rotatable bonds of the ligand of the original binding site was determined with DOCK, and we set the number of growth layers as the number of rotatable bonds/2. Only binding sites of ligands with two or more rotatable bonds were processed. We used the graph sampling method, and the maximum allowed molecular weight of the de novo ligands was set to 550. (5) We ran 30 independent replicates for each receptor (i.e., we used 30 different anchors), and from the output of the 30 independent runs we selected a total of 50 de novo ligands with the best grid score. Overall this procedure resulted in de novo ligands of comparable size and complexity as the original ligand.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Ligand Characterization</h3><div class="NLM_p last">The characteristics of the 50 best de novo ligands of each receptor were calculated with the chemical variational autoencoder (VAE) developed by the Aspuru-Guzik lab,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> using the autoencoder distributed with the tool itself (“zinc_properties”). We used the VAE to calculate log <i>P</i>, synthetic accessibility score (SAS),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and quantitative druglikeness (QED)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> for each de novo ligand, and also a vector corresponding to their representation in the latent space (with 196 dimensions). Molecular weight was calculated with the obprop tool of the Open Babel suite. Clustering of the latent space representation of de novo ligands (for visualization) was performed with Barnes–Hut t-SNE,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> with two dimensions and perplexity 50.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Optimization and Evolution of de Novo Ligands with Docking and AI</h3><div class="NLM_p last">The selected de novo ligands were further optimized and evolved with a strategy that used DOCK 6.8 and the VAE (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). First, from the list of binding sites that had a de novo ligand with a distance less than 13, we selected the binding site from the three different species where the selected de novo ligand could be docked with the best grid score. Next, each selected de novo ligand was converted to a SMILES string with RDKit, and we used the VAE to sample the latent space for structurally related chemicals, using <i>Z</i> cutoffs of 0.5, 1, 3.125, 6.25, 12.5, 25 and 50, taking 30 000 samples. This usually returned 10–100 unique chemical structures. If the number of returned chemicals was higher than 50, we randomly selected 50 of them. The structures returned by the VAE were docked to the three receptors, and the one with the best average score (if its score was an improvement in at least two of the receptors) was chosen and used in the next round of sampling/docking. This cycle was repeated as long as there was an improvement in the grid score of the new ligands.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Monte Carlo Simulations (Randomization Test)</h3><div class="NLM_p last">We performed Monte Carlo simulations to test whether small rings are enriched in the optimized molecules due to the constraints of binding in multiple receptors. First, in the SMILES returned in the first step of the optimization (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S8</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">FigureS8_FullResult.txt</a> for an example) we determined for every atom whether it is part of a three-membered and four-membered ring using RDKit, and the number of such rings. Next we took 10 000 samples using all de novo ligands by selecting one SMILES randomly from the returned molecules of every de novo ligand and calculated the frequency of three- and four-membered ring in every random sample. Finally, we estimated whether the observed frequency of three- and four-membered rings is significantly higher than the expectation using the formula <i>p</i> = (<i>n</i> + 1)/(<i>N</i> + 1), where <i>N</i> is the total number of samples (10 000) and <i>n</i> is the number of samples with equal or higher frequency of small rings than their observed frequency. The expected frequency of small rings was determined as the median of random samples.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Globularity and Rotatable Bond Estimation</h3><div class="NLM_p last">Globularity was measured with the plane of best fit (PBF) method.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> SMILES of de novo and VAE ligands were first converted to 3D structures with Open Babel using the --gen3d flag, which uses the MMFF94 force field. PBF scores were calculated on hydrogen-free structures,<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> with the pbf function of RDKit. A fraction of compounds (∼3% in the optimized ligands, 5–10% in the raw ligands returned by the VAE) were excluded, due to the inability of RDKit to process their 3D structure. The number of rotatable bonds in each ligand was calculated with the obprop tool of the Open Babel suite, ignoring hydrogens; thus methyl or amino groups are not counted as rotatable bonds.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Visualization and Statistics</h3><div class="NLM_p last">All statistical tests were performed with in-house Perl scripts and R and were corrected for multiple testing with the Benjamini–Hochberg method.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Protein structures were visualized with Chimera (version 1.11.2), chemical structures with Open Babel 2.3.1.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00220" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00220" class="ext-link">10.1021/acs.jmedchem.9b00220</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Six tables of data on receptors, antibiotics, de novo sites, proteins, and PAINS (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Two data sets (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">ZIP</a>)</p></li><li><p class="inline">Twelve figures showing comparison of ligands, performance, chemical characteristics, weight–frequency relationships, gene ontology analysis results, sampled molecules, optimization steps, and globularity results (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx">jm9b00220_si_001.xlsx (164.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip">jm9b00220_si_002.zip (6.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf">jm9b00220_si_003.pdf (3.39 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00220" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31113" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31113" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">György Abrusán</span> - <span class="hlFld-Affiliation affiliation">MRC Human Genetics Unit, Institute
of Genetics and Molecular Medicine, University
of Edinburgh, Crewe Road, Edinburgh EH4 2XU, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4375-1552" title="Orcid link">http://orcid.org/0000-0003-4375-1552</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5b3c2234293c22753a39292e283a351b323c3636753e3f753a38752e30"><span class="__cf_email__" data-cfemail="3f5846504d5846115e5d4d4a4c5e517f56585252115a5b115e5c114a54">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph A. Marsh</span> - <span class="hlFld-Affiliation affiliation">MRC Human Genetics Unit, Institute
of Genetics and Molecular Medicine, University
of Edinburgh, Crewe Road, Edinburgh EH4 2XU, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.A. conceived the project, performed the analyses, and wrote the first version of the manuscript. J.A.M. contributed to the analysis and corrected the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58517" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58517" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">György Abrusán</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=BIO-d7e1314-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>György Abrusán</b> obtained his M.Sc. in Biology in 1997 from the University of Debrecen (Hungary) and his Ph.D. in Hydrobiology from Warsaw University (Poland) in 2003. Currently he is a Research Fellow at the University of Edinburgh. His research interests include comparative genomics and computational structural biology, and focus on the factors governing the evolution of quaternary structure of proteins, and its dependence on interactions with ligands.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Joseph A. Marsh</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=BIO-d7e1319-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Joseph A. Marsh</b> obtained his B.Sc. in Microbiology and Immunology from the University of British Columbia in 2003 and his Ph.D. in Biochemistry from the University of Toronto in 2010. This was followed by postdoctoral research at the MRC Laboratory of Molecular Biology and the EMBL European Bioinformatics Institute in Cambridge. He moved to the MRC Human Genetics Unit, University of Edinburgh, in 2014, where he is now a Reader. His research uses computational methods to study the structure, assembly, and evolution of protein complexes and to investigate the implications of this for understanding human disease.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11628" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11628" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Lauren Prentis, Stephen Telehany, Scott Brozell, and Robert Rizzo for help with de novo DOCK and for providing the prerelease version of the source code of DOCK 6.9. We thank Neil Clark and Asier Unciti-Broceta for critical reading of the manuscript. We acknowledge the use of the Eddie3 computing cluster of the University of Edinburgh and thank the support of John Ireland and Steve Thorn from Research Computing. This work was supported by the Medical Research Council (Career Development Award MR/M02122X/1 for J.A.M.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">GO</td><td class="NLM_def"><p class="first last">gene ontology</p></td></tr><tr><td class="NLM_term">MBS</td><td class="NLM_def"><p class="first last">multichain binding site</p></td></tr><tr><td class="NLM_term">PBF</td><td class="NLM_def"><p class="first last">plane of best fit</p></td></tr><tr><td class="NLM_term">QED</td><td class="NLM_def"><p class="first last">quantitative estimate of druglikeness</p></td></tr><tr><td class="NLM_term">SAS</td><td class="NLM_def"><p class="first last">synthetic accessibility</p></td></tr><tr><td class="NLM_term">SBS</td><td class="NLM_def"><p class="first last">single-chain binding site</p></td></tr><tr><td class="NLM_term">VAE</td><td class="NLM_def"><p class="first last">variational autoencoder</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 84 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowater, L.</span></span> <i>The Microbes Fight Back: Antibiotic Resistance</i>, <span class="NLM_edition">Gld</span> ed.; <span class="NLM_publisher-name">Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Bowater&title=The+Microbes+Fight+Back%3A+Antibiotic+Resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBowater%26aufirst%3DL.%26btitle%3DThe%2520Microbes%2520Fight%2520Back%253A%2520Antibiotic%2520Resistance%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D.</span></span> <span> </span><span class="NLM_article-title">Origins and Evolution of Antibiotic Resistance</span>. <i>Microbiol Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1128/MMBR.00016-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1128%2FMMBR.00016-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20805405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSqurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2010&pages=417-433&issue=3&author=J.+Daviesauthor=D.+Davies&title=Origins+and+Evolution+of+Antibiotic+Resistance&doi=10.1128%2FMMBR.00016-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Origins and evolution of antibiotic resistance</span></div><div class="casAuthors">Davies, Julian; Davies, Dorothy</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-433</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Antibiotics have always been considered one of the wonder discoveries of the 20th century.  This is true, but the real wonder is the rise of antibiotic resistance in hospitals, communities, and the environment concomitant with their use.  The extraordinary genetic capacities of microbes have benefitted from man's overuse of antibiotics to exploit every source of resistance genes and every means of horizontal gene transmission to develop multiple mechanisms of resistance for each and every antibiotic introduced into practice clin., agriculturally, or otherwise.  This review presents the salient aspects of antibiotic resistance development over the past half-century, with the oft-restated conclusion that it is time to act.  To achieve complete restitution of therapeutic applications of antibiotics, there is a need for more information on the role of environmental microbiomes in the rise of antibiotic resistance.  In particular, creative approaches to the discovery of novel antibiotics and their expedited and controlled introduction to therapy are obligatory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6-EM4T8NgWLVg90H21EOLACvtfcHk0liSSWg4TDLG0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSqurnO&md5=64214f42f2ae91c1ba94dbe765b7cd0c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMMBR.00016-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.00016-10%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DD.%26atitle%3DOrigins%2520and%2520Evolution%2520of%2520Antibiotic%2520Resistance%26jtitle%3DMicrobiol%2520Mol.%2520Biol.%2520Rev.%26date%3D2010%26volume%3D74%26issue%3D3%26spage%3D417%26epage%3D433%26doi%3D10.1128%2FMMBR.00016-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span> <span> </span><span class="NLM_article-title">Drugs for Bad Bugs: Confronting the Challenges of Antibacterial Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/nrd2201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&issue=1&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+L.+Pompliano&title=Drugs+for+Bad+Bugs%3A+Confronting+the+Challenges+of+Antibacterial+Discovery&doi=10.1038%2Fnrd2201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0ljY9-EBfYXCFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26atitle%3DDrugs%2520for%2520Bad%2520Bugs%253A%2520Confronting%2520the%2520Challenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D1%26spage%3D29%26epage%3D40%26doi%3D10.1038%2Fnrd2201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Retrospective Analysis of Natural Products Provides Insights for Future Discovery Trends</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5601</span>– <span class="NLM_lpage">5606</span>, <span class="refDoi"> DOI: 10.1073/pnas.1614680114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.1614680114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28461474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=5601-5606&issue=22&author=C.+R.+Pyeauthor=M.+J.+Bertinauthor=R.+S.+Lokeyauthor=W.+H.+Gerwickauthor=R.+G.+Linington&title=Retrospective+Analysis+of+Natural+Products+Provides+Insights+for+Future+Discovery+Trends&doi=10.1073%2Fpnas.1614680114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Retrospective analysis of natural products provides insights for future discovery trends</span></div><div class="casAuthors">Pye, Cameron R.; Bertin, Matthew J.; Lokey, R. Scott; Gerwick, William H.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5601-5606</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding of the capacity of the natural world to produce secondary metabolites is important to a broad range of fields, including drug discovery, ecol., biosynthesis, and chem. biol., among others.  Both the abs. no. and the rate of discovery of natural products have increased significantly in recent years.  However, there is a perception and concern that the fundamental novelty of these discoveries is decreasing relative to previously known natural products.  This study presents a quant. examn. of the field from the perspective of both no. of compds. and compd. novelty using a dataset of all published microbial and marine-derived natural products.  This anal. aimed to explore a no. of key questions, such as how the rate of discovery of new natural products has changed over the past decades, how the av. natural product structural novelty has changed as a function of time, whether exploring novel taxonomic space affords an advantage in terms of novel compd. discovery, and whether it is possible to est. how close we are to having described all of the chem. space covered by natural products.  Our analyses demonstrate that most natural products being published today bear structural similarity to previously published compds., and that the range of scaffolds readily accessible from nature is limited.  However, the anal. also shows that the field continues to discover appreciable nos. of natural products with no structural precedent.  Together, these results suggest that the development of innovative discovery methods will continue to yield compds. with unique structural and biol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3OY35vWvx_bVg90H21EOLACvtfcHk0ljY9-EBfYXCFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D&md5=44da23318031b09694c94b0bfdf1d718</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614680114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614680114%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DBertin%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DGerwick%26aufirst%3DW.%2BH.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DRetrospective%2520Analysis%2520of%2520Natural%2520Products%2520Provides%2520Insights%2520for%2520Future%2520Discovery%2520Trends%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D22%26spage%3D5601%26epage%3D5606%26doi%3D10.1073%2Fpnas.1614680114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skinnider, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarvey, N. A.</span></span> <span> </span><span class="NLM_article-title">Statistical Reanalysis of Natural Products Reveals Increasing Chemical Diversity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">E6271</span>– <span class="NLM_lpage">E6272</span>, <span class="refDoi"> DOI: 10.1073/pnas.1708560114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.1708560114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28710332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ggt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6271-E6272&issue=31&author=M.+A.+Skinniderauthor=N.+A.+Magarvey&title=Statistical+Reanalysis+of+Natural+Products+Reveals+Increasing+Chemical+Diversity&doi=10.1073%2Fpnas.1708560114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical reanalysis of natural products reveals increasing chemical diversity</span></div><div class="casAuthors">Skinnider, Michael A.; Magarvey, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">E6271-E6272</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4qQ4_ZBc7RLVg90H21EOLACvtfcHk0ljY9-EBfYXCFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ggt77O&md5=12ed6c205f7df965a331f4f48b3b245c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1708560114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1708560114%26sid%3Dliteratum%253Aachs%26aulast%3DSkinnider%26aufirst%3DM.%2BA.%26aulast%3DMagarvey%26aufirst%3DN.%2BA.%26atitle%3DStatistical%2520Reanalysis%2520of%2520Natural%2520Products%2520Reveals%2520Increasing%2520Chemical%2520Diversity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D31%26spage%3DE6271%26epage%3DE6272%26doi%3D10.1073%2Fpnas.1708560114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toprak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veres, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Evolutionary Paths to Antibiotic Resistance under Dynamically Sustained Drug Selection</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/ng.1034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fng.1034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1eisbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=101-105&issue=1&author=E.+Toprakauthor=A.+Veresauthor=J.-B.+Michelauthor=R.+Chaitauthor=D.+L.+Hartlauthor=R.+Kishony&title=Evolutionary+Paths+to+Antibiotic+Resistance+under+Dynamically+Sustained+Drug+Selection&doi=10.1038%2Fng.1034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary paths to antibiotic resistance under dynamically sustained drug selection</span></div><div class="casAuthors">Toprak, Erdal; Veres, Adrian; Michel, Jean-Baptiste; Chait, Remy; Hartl, Daniel L.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic resistance can evolve through the sequential accumulation of multiple mutations.  To study such gradual evolution, we developed a selection device, the 'morbidostat', that continuously monitors bacterial growth and dynamically regulates drug concns., such that the evolving population is constantly challenged.  We analyzed the evolution of resistance in Escherichia coli under selection with single drugs, including chloramphenicol, doxycycline and trimethoprim.  Over a period of ∼20 days, resistance levels increased dramatically, with parallel populations showing similar phenotypic trajectories.  Whole-genome sequencing of the evolved strains identified mutations both specific to resistance to a particular drug and shared in resistance to multiple drugs.  Chloramphenicol and doxycycline resistance evolved smoothly through diverse combinations of mutations in genes involved in translation, transcription and transport.  In contrast, trimethoprim resistance evolved in a stepwise manner, through mutations restricted to the gene encoding the enzyme dihydrofolate reductase (DHFR).  Sequencing of DHFR over the time course of the expt. showed that parallel populations evolved similar mutations and acquired them in a similar order.  Whole-genome sequencing data from the morbidostat-evolved strains have been deposited in the NCBI Sequence Read Archive (SRA046097).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU4Gv27INPlrVg90H21EOLACvtfcHk0li8pCfdE0eKYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1eisbbN&md5=f48ae03dee0860f699533b44f249fcca</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fng.1034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.1034%26sid%3Dliteratum%253Aachs%26aulast%3DToprak%26aufirst%3DE.%26aulast%3DVeres%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DJ.-B.%26aulast%3DChait%26aufirst%3DR.%26aulast%3DHartl%26aufirst%3DD.%2BL.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DEvolutionary%2520Paths%2520to%2520Antibiotic%2520Resistance%2520under%2520Dynamically%2520Sustained%2520Drug%2520Selection%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26issue%3D1%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fng.1034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Multi-Targeting by Monotherapeutic Antibacterials</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/nrd2202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnrd2202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17159922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=41-55&issue=1&author=L.+L.+Silver&title=Multi-Targeting+by+Monotherapeutic+Antibacterials&doi=10.1038%2Fnrd2202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-targeting by monotherapeutic antibacterials</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibacterial discovery research has been driven, medically, com. and intellectually, by the need for new therapeutics that are not subject to the resistance mechanisms that have evolved to combat previous generations of antibacterial agents.  This need has often been equated with the identification and exploitation of novel targets.  But efforts towards discovery and development of inhibitors of novel targets have proved frustrating.  It might be that the 'good old targets' are qual. different from the crop of all possible novel targets.  What has been learned from existing targets that can be applied to the quest for new antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpItQysyGWT8LVg90H21EOLACvtfcHk0li8pCfdE0eKYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfN&md5=49d78749fc31f5f0546b859e17b31c47</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2202%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DMulti-Targeting%2520by%2520Monotherapeutic%2520Antibacterials%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D1%26spage%3D41%26epage%3D55%26doi%3D10.1038%2Fnrd2202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span> <span> </span><span class="NLM_article-title">Resistance-Resistant Antibiotics</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.tips.2014.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25458541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=664-674&issue=12&author=E.+Oldfieldauthor=X.+Feng&title=Resistance-Resistant+Antibiotics&doi=10.1016%2Fj.tips.2014.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance-resistant antibiotics</span></div><div class="casAuthors">Oldfield, Eric; Feng, Xinxin</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">664-674</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">New antibiotics are needed because drug resistance is increasing while the introduction of new antibiotics is decreasing.  We discuss here six possible approaches to develop 'resistance-resistant' antibiotics.  First, multitarget inhibitors in which a single compd. inhibits more than one target may be easier to develop than conventional combination therapies with two new drugs.  Second, inhibiting multiple targets in the same metabolic pathway is expected to be an effective strategy owing to synergy.  Third, discovering multiple-target inhibitors should be possible by using sequential virtual screening.  Fourth, repurposing existing drugs can lead to combinations of multitarget therapeutics.  Fifth, targets need not be proteins.  Sixth, inhibiting virulence factor formation and boosting innate immunity may also lead to decreased susceptibility to resistance.  Although it is not possible to eliminate resistance, the approaches reviewed here offer several possibilities for reducing the effects of mutations and, in some cases, suggest that sensitivity to existing antibiotics may be restored in otherwise drug-resistant organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoFcN1H0q81rVg90H21EOLACvtfcHk0li8pCfdE0eKYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrzP&md5=1fd8f53ba0e90abbc83a6616db24568a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DOldfield%26aufirst%3DE.%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DResistance-Resistant%2520Antibiotics%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26issue%3D12%26spage%3D664%26epage%3D674%26doi%3D10.1016%2Fj.tips.2014.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">What Is an “ideal” Antibiotic? Discovery Challenges and Path Forward</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bcp.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28087253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=63-73&author=S.+B.+Singhauthor=K.+Youngauthor=L.+L.+Silver&title=What+Is+an+%E2%80%9Cideal%E2%80%9D+Antibiotic%3F+Discovery+Challenges+and+Path+Forward&doi=10.1016%2Fj.bcp.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">What is an "ideal" antibiotic? Discovery challenges and path forward</span></div><div class="casAuthors">Singh, Sheo B.; Young, Katherine; Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  An ideal antibiotic is an antibacterial agent that kills or inhibits the growth of all harmful bacteria in a host, regardless of site of infection without affecting beneficial gut microbes (gut flora) or causing undue toxicity to the host.  Sadly, no such antibiotics exist.  What exist are many effective Gram-pos. antibacterial agents as well as broad-spectrum agents that provide treatment of certain Gram-neg. bacteria but not holistic treatment of all bacteria.  However effectiveness of all antibacterial agents is being rapidly eroded due to resistance.  This viewpoint provides an overview of today's antibiotics, challenges and potential path forward of discovery and development of new (ideal) antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1-YaZx5VrMLVg90H21EOLACvtfcHk0li8pCfdE0eKYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChsLw%253D&md5=6cb221ed31dc85f205e2e4c28194a822</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DYoung%26aufirst%3DK.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DWhat%2520Is%2520an%2520%25E2%2580%259Cideal%25E2%2580%259D%2520Antibiotic%253F%2520Discovery%2520Challenges%2520and%2520Path%2520Forward%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D133%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.bcp.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samaddar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, J.</span></span> <span> </span><span class="NLM_article-title">Battle against Vancomycin-Resistant Bacteria: Recent Developments in Chemical Strategies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3184</span>– <span class="NLM_lpage">3205</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01093</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFagurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3184-3205&issue=7&author=G.+Dhandaauthor=P.+Sarkarauthor=S.+Samaddarauthor=J.+Haldar&title=Battle+against+Vancomycin-Resistant+Bacteria%3A+Recent+Developments+in+Chemical+Strategies&doi=10.1021%2Facs.jmedchem.8b01093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Battle against Vancomycin-Resistant Bacteria: Recent Developments in Chemical Strategies</span></div><div class="casAuthors">Dhanda, Geetika; Sarkar, Paramita; Samaddar, Sandip; Haldar, Jayanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3184-3205</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Vancomycin, a natural glycopeptide antibiotic, was used as the antibiotic of last resort for the treatment of multidrug-resistant Gram-pos. bacterial infections.  However, almost 30 years after its use, resistance to vancomycin was first reported in 1986 in France.  This became a major health concern, and alternative treatment strategies were urgently needed.  New classes of mols., including semisynthetic antibacterial compds. and newer generations of the previously used antibiotics, were developed.  Semisynthetic derivs. of vancomycin with enhanced binding affinity, membrane disruption ability, and lipid binding properties have exhibited promising results against both Gram-pos. and Gram-neg. bacteria.  Various successful approaches developed to overcome the acquired resistance in Gram-pos. bacteria, intrinsic resistance in Gram-neg. bacteria, and other forms of noninherited resistance to vancomycin have been discussed in this Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8kPOprosWa7Vg90H21EOLACvtfcHk0lgXNbm4Jh7a-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFagurvK&md5=b08a3a82c9f61c5e31af8d17af0f9d76</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01093%26sid%3Dliteratum%253Aachs%26aulast%3DDhanda%26aufirst%3DG.%26aulast%3DSarkar%26aufirst%3DP.%26aulast%3DSamaddar%26aufirst%3DS.%26aulast%3DHaldar%26aufirst%3DJ.%26atitle%3DBattle%2520against%2520Vancomycin-Resistant%2520Bacteria%253A%2520Recent%2520Developments%2520in%2520Chemical%2520Strategies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D7%26spage%3D3184%26epage%3D3205%26doi%3D10.1021%2Facs.jmedchem.8b01093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brötz-Oesterhelt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, N. A.</span></span> <span> </span><span class="NLM_article-title">How Many Modes of Action Should an Antibiotic Have?</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2008.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.coph.2008.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18621146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BD1cjhsFygsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=564-573&issue=5&author=H.+Br%C3%B6tz-Oesterheltauthor=N.+A.+Brunner&title=How+Many+Modes+of+Action+Should+an+Antibiotic+Have%3F&doi=10.1016%2Fj.coph.2008.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">How many modes of action should an antibiotic have?</span></div><div class="casAuthors">Brotz-Oesterhelt Heike; Brunner Nina A</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">564-73</span>
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    </div><div class="casAbstract">All antibiotics that have been successfully employed for decades as monotherapeutics in the treatment of bacterial infections rely on mechanisms of bacterial growth inhibition which are by far more complex than inhibition of a single enzyme.  Such successful antibiotics have in common that they address several targets in parallel and/or that their targets are encoded by multiple genes.  Such multiplicity of targets and of target genes has the advantage that the emergence of spontaneous target-related resistance is a comparatively slow process.  Recently registered antibiotics and novel antibiotics in development are discussed in the light of this promising concept of antibacterial polypharmacology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzIF5GvxYY_0HdcurJClFcfW6udTcc2eZYtMHTJ6-INrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjhsFygsA%253D%253D&md5=820b32bfce47dd05c0a37bf310a7884e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2008.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2008.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25B6tz-Oesterhelt%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DN.%2BA.%26atitle%3DHow%2520Many%2520Modes%2520of%2520Action%2520Should%2520an%2520Antibiotic%2520Have%253F%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2008%26volume%3D8%26issue%3D5%26spage%3D564%26epage%3D573%26doi%3D10.1016%2Fj.coph.2008.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.-S.</span></span> <span> </span><span class="NLM_article-title">4-Quinolone Hybrids and Their Antibacterial Activities</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.ejmech.2017.09.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29031077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GltrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=335-345&author=Y.-Q.+Huauthor=S.+Zhangauthor=Z.+Xuauthor=Z.-S.+Lvauthor=M.-L.+Liuauthor=L.-S.+Feng&title=4-Quinolone+Hybrids+and+Their+Antibacterial+Activities&doi=10.1016%2Fj.ejmech.2017.09.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">4-Quinolone hybrids and their antibacterial activities</span></div><div class="casAuthors">Hu, Yuan-Qiang; Zhang, Shu; Xu, Zhi; Lv, Zao-Sheng; Liu, Ming-Liang; Feng, Lian-Shun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">335-345</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The emergence and wide-spread of drug-resistant bacteria including multi-drug resistant and pan-drug resistant pathogens which are assocd. with considerable mortality, represent a significant global health threat.  4-Quinolones which are exemplified by fluoroquinolones are the second largest chemotherapy agents used in clin. practice for the treatment of various bacterial infections.  However, the resistance of bacteria to 4-quinolones develops rapidly and spreads widely throughout the world due to the long-term, inappropriate use and even abuse.  To overcome the resistance and improve the potency, several strategies have been developed.  Amongst them, mol. hybridization, which is based on the incorporation of two or more pharmacophores into a single mol. with a flexible linker, is one of the most practical approaches.  This review aims to summarize the recent advances made towards the discovery of 4-quinolone hybrids as potential antibacterial agents as well as their structure-activity relationship (SAR).  The enriched SAR may pave the way for the further rational development of 4-quinolone hybrids with excellent potency against both drug-susceptible and drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiel6w3j-bZLVg90H21EOLACvtfcHk0lgXNbm4Jh7a-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GltrbM&md5=0af2128ec93ddeb3ceddc7d3fd8d2b8e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.050%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.-Q.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLv%26aufirst%3DZ.-S.%26aulast%3DLiu%26aufirst%3DM.-L.%26aulast%3DFeng%26aufirst%3DL.-S.%26atitle%3D4-Quinolone%2520Hybrids%2520and%2520Their%2520Antibacterial%2520Activities%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D141%26spage%3D335%26epage%3D345%26doi%3D10.1016%2Fj.ejmech.2017.09.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Specklin, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigwalt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, H. H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of Oxazolidinone-Quinolone Hybrids</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2319</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(03)00083-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2FS0968-0896%2803%2900083-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=12713843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFWhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=2313-2319&issue=10&author=C.+Hubschwerlenauthor=J.-L.+Specklinauthor=C.+Sigwaltauthor=S.+Schroederauthor=H.+H.+Locher&title=Design%2C+Synthesis+and+Biological+Evaluation+of+Oxazolidinone-Quinolone+Hybrids&doi=10.1016%2FS0968-0896%2803%2900083-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of oxazolidinone-quinolone hybrids</span></div><div class="casAuthors">Hubschwerlen, Christian; Specklin, Jean-Luc; Sigwalt, Christine; Schroeder, Susanne; Locher, Hans H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2313-2319</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Oxazolidinone-quinolone hybrids that combine the pharmacophores of a quinolone and an oxazolidinone were synthesized and shown to be active against a variety of resistant and susceptible Gram-pos. and fastidious Gram-neg. organisms.  The best compds. in this series overcome all types of resistance in relevant clin. Gram-pos. pathogens.  The nature of the spacer greatly influences the antibacterial activity.  The dual mode of action could be demonstrated for compds. having a piperazinyl spacer.  Antibacterial activity was higher at acidic pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEhQdn0-mi_7Vg90H21EOLACvtfcHk0ljAmMJ9EuNAqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFWhsLY%253D&md5=6caee544ae6e443865da04345538241b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2803%2900083-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252803%252900083-X%26sid%3Dliteratum%253Aachs%26aulast%3DHubschwerlen%26aufirst%3DC.%26aulast%3DSpecklin%26aufirst%3DJ.-L.%26aulast%3DSigwalt%26aufirst%3DC.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Oxazolidinone-Quinolone%2520Hybrids%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26issue%3D10%26spage%3D2313%26epage%3D2319%26doi%3D10.1016%2FS0968-0896%2803%2900083-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Domalaon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idowu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhanel, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, F.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Hybrids: The Next Generation of Agents and Adjuvants against Gram-Negative Pathogens?</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">2</span>), <span class="NLM_elocation-id">e00077-17</span> <span class="refDoi"> DOI: 10.1128/CMR.00077-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1128%2FCMR.00077-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29540434" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2018&issue=2&author=R.+Domalaonauthor=T.+Idowuauthor=G.+G.+Zhanelauthor=F.+Schweizer&title=Antibiotic+Hybrids%3A+The+Next+Generation+of+Agents+and+Adjuvants+against+Gram-Negative+Pathogens%3F&doi=10.1128%2FCMR.00077-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FCMR.00077-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00077-17%26sid%3Dliteratum%253Aachs%26aulast%3DDomalaon%26aufirst%3DR.%26aulast%3DIdowu%26aufirst%3DT.%26aulast%3DZhanel%26aufirst%3DG.%2BG.%26aulast%3DSchweizer%26aufirst%3DF.%26atitle%3DAntibiotic%2520Hybrids%253A%2520The%2520Next%2520Generation%2520of%2520Agents%2520and%2520Adjuvants%2520against%2520Gram-Negative%2520Pathogens%253F%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2018%26volume%3D31%26issue%3D2%26doi%3D10.1128%2FCMR.00077-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurig-Briccio, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lechartier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">No, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cintra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molohon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlean, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas-Junior, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennis, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crick, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span> <span> </span><span class="NLM_article-title">Multitarget Drug Discovery for Tuberculosis and Other Infectious Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3126</span>– <span class="NLM_lpage">3139</span>, <span class="refDoi"> DOI: 10.1021/jm500131s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500131s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVeqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3126-3139&issue=7&author=K.+Liauthor=L.+A.+Schurig-Briccioauthor=X.+Fengauthor=A.+Upadhyayauthor=V.+Pujariauthor=B.+Lechartierauthor=F.+L.+Fontesauthor=H.+Yangauthor=G.+Raoauthor=W.+Zhuauthor=A.+Gulatiauthor=J.+H.+Noauthor=G.+Cintraauthor=S.+Bogueauthor=Y.-L.+Liuauthor=K.+Molohonauthor=P.+Orleanauthor=D.+A.+Mitchellauthor=L.+Freitas-Juniorauthor=F.+Renauthor=H.+Sunauthor=T.+Jiangauthor=Y.+Liauthor=R.-T.+Guoauthor=S.+T.+Coleauthor=R.+B.+Gennisauthor=D.+C.+Crickauthor=E.+Oldfield&title=Multitarget+Drug+Discovery+for+Tuberculosis+and+Other+Infectious+Diseases&doi=10.1021%2Fjm500131s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget Drug Discovery for Tuberculosis and Other Infectious Diseases</span></div><div class="casAuthors">Li, Kai; Schurig-Briccio, Lici A.; Feng, Xinxin; Upadhyay, Ashutosh; Pujari, Venugopal; Lechartier, Benoit; Fontes, Fabio L.; Yang, Hongliang; Rao, Guodong; Zhu, Wei; Gulati, Anmol; No, Joo Hwan; Cintra, Giovana; Bogue, Shannon; Liu, Yi-Liang; Molohon, Katie; Orlean, Peter; Mitchell, Douglas A.; Freitas-Junior, Lucio; Ren, Feifei; Sun, Hong; Jiang, Tong; Li, Yujie; Guo, Rey-Ting; Cole, Stewart T.; Gennis, Robert B.; Crick, Dean C.; Oldfield, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3126-3139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery of a series of new drug leads that have potent activity against Mycobacterium tuberculosis as well as against other bacteria, fungi, and a malaria parasite.  The compds. are analogs of the new tuberculosis (TB) drug SQ109, which has been reported to act by inhibiting a transporter called MmpL3, involved in cell wall biosynthesis.  The authors show that SQ109 and the new compds. also target enzymes involved in menaquinone biosynthesis and electron transport, inhibiting respiration and ATP biosynthesis, and are uncouplers, collapsing the pH gradient and membrane potential used to power transporters.  The result of such multitarget inhibition is potent inhibition of TB cell growth, as well as very low rates of spontaneous drug resistance.  Several targets are absent in humans but are present in other bacteria, as well as in malaria parasites, whose growth is also inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFazutTcGaBLVg90H21EOLACvtfcHk0ljAmMJ9EuNAqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVeqtLY%253D&md5=85fb16a07423e1473d64e0c41707c3db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm500131s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500131s%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DK.%26aulast%3DSchurig-Briccio%26aufirst%3DL.%2BA.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DUpadhyay%26aufirst%3DA.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DLechartier%26aufirst%3DB.%26aulast%3DFontes%26aufirst%3DF.%2BL.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DGulati%26aufirst%3DA.%26aulast%3DNo%26aufirst%3DJ.%2BH.%26aulast%3DCintra%26aufirst%3DG.%26aulast%3DBogue%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.-L.%26aulast%3DMolohon%26aufirst%3DK.%26aulast%3DOrlean%26aufirst%3DP.%26aulast%3DMitchell%26aufirst%3DD.%2BA.%26aulast%3DFreitas-Junior%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DR.-T.%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DGennis%26aufirst%3DR.%2BB.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DOldfield%26aufirst%3DE.%26atitle%3DMultitarget%2520Drug%2520Discovery%2520for%2520Tuberculosis%2520and%2520Other%2520Infectious%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D7%26spage%3D3126%26epage%3D3139%26doi%3D10.1021%2Fjm500131s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lázár, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horváth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrtyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busa Fekete, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méhi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csörgő, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pósfai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekete, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szappanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kégl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span> <span> </span><span class="NLM_article-title">Bacterial Evolution of Antibiotic Hypersensitivity</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">700</span>, <span class="refDoi"> DOI: 10.1038/msb.2013.57</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fmsb.2013.57" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24169403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=700&issue=1&author=V.+L%C3%A1z%C3%A1rauthor=G.+P.+Singhauthor=R.+Spohnauthor=I.+Nagyauthor=B.+Horv%C3%A1thauthor=M.+Hrtyanauthor=R.+Busa+Feketeauthor=B.+Bogosauthor=O.+M%C3%A9hiauthor=B.+Cs%C3%B6rg%C5%91author=G.+P%C3%B3sfaiauthor=G.+Feketeauthor=B.+Szappanosauthor=B.+K%C3%A9glauthor=B.+Pappauthor=C.+P%C3%A1l&title=Bacterial+Evolution+of+Antibiotic+Hypersensitivity&doi=10.1038%2Fmsb.2013.57"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial evolution of antibiotic hypersensitivity</span></div><div class="casAuthors">Lazar, Viktoria; Pal Singh, Gajinder; Spohn, Reka; Nagy, Istvan; Horvath, Balazs; Hrtyan, Monika; Busa-Fekete, Robert; Bogos, Balazs; Mehi, Orsolya; Csorgo, Balint; Posfai, Gyorgy; Fekete, Gergely; Szappanos, Balazs; Kegl, Balazs; Papp, Balazs; Pal, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">700</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The evolution of resistance to a single antibiotic is frequently accompanied by increased resistance to multiple other antimicrobial agents.  In sharp contrast, very little is known about the frequency and mechanisms underlying collateral sensitivity.  In this case, genetic adaptation under antibiotic stress yields enhanced sensitivity to other antibiotics.  Using large-scale lab. evolutionary expts. with Escherichia coli, we demonstrate that collateral sensitivity occurs frequently during the evolution of antibiotic resistance.  Specifically, populations adapted to aminoglycosides have an esp. low fitness in the presence of several other antibiotics.  Whole-genome sequencing of lab.-evolved strains revealed multiple mechanisms underlying aminoglycoside resistance, including a redn. in the proton-motive force (PMF) across the inner membrane.  We propose that as a side effect, these mutations diminish the activity of PMF-dependent major efflux pumps (including the AcrAB transporter), leading to hypersensitivity to several other antibiotics.  More generally, our work offers an insight into the mechanisms that drive the evolution of neg. trade-offs under antibiotic selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoVjK_fOBE87Vg90H21EOLACvtfcHk0ljAmMJ9EuNAqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqsbbK&md5=73a0b2f41cf6bbedd43f67056d558cea</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2013.57&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2013.57%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25A1z%25C3%25A1r%26aufirst%3DV.%26aulast%3DSingh%26aufirst%3DG.%2BP.%26aulast%3DSpohn%26aufirst%3DR.%26aulast%3DNagy%26aufirst%3DI.%26aulast%3DHorv%25C3%25A1th%26aufirst%3DB.%26aulast%3DHrtyan%26aufirst%3DM.%26aulast%3DBusa%2BFekete%26aufirst%3DR.%26aulast%3DBogos%26aufirst%3DB.%26aulast%3DM%25C3%25A9hi%26aufirst%3DO.%26aulast%3DCs%25C3%25B6rg%25C5%2591%26aufirst%3DB.%26aulast%3DP%25C3%25B3sfai%26aufirst%3DG.%26aulast%3DFekete%26aufirst%3DG.%26aulast%3DSzappanos%26aufirst%3DB.%26aulast%3DK%25C3%25A9gl%26aufirst%3DB.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26atitle%3DBacterial%2520Evolution%2520of%2520Antibiotic%2520Hypersensitivity%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D1%26spage%3D700%26doi%3D10.1038%2Fmsb.2013.57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lázár, V.</span></span> <span> </span><span class="NLM_article-title">Collateral Sensitivity of Antibiotic-Resistant Microbes</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2015.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.tim.2015.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25818802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFehu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=401-407&issue=7&author=C.+P%C3%A1lauthor=B.+Pappauthor=V.+L%C3%A1z%C3%A1r&title=Collateral+Sensitivity+of+Antibiotic-Resistant+Microbes&doi=10.1016%2Fj.tim.2015.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Collateral sensitivity of antibiotic-resistant microbes</span></div><div class="casAuthors">Pal, Csaba; Papp, Balazs; Lazar, Viktoria</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  Understanding how evolution of microbial resistance towards a given antibiotic influences susceptibility to other drugs is a challenge of profound importance.  By combining lab. evolution, genome sequencing, and functional analyses, recent works have charted the map of evolutionary trade-offs between antibiotics and have explored the underlying mol. mechanisms.  Strikingly, mutations that caused multidrug resistance in bacteria simultaneously enhanced sensitivity to many other unrelated drugs (collateral sensitivity).  Here, we explore how this emerging research sheds new light on resistance mechanisms and the way it could be exploited for the development of alternative antimicrobial strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBgzATZDim-7Vg90H21EOLACvtfcHk0ljAyNQZFnmAzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFehu7o%253D&md5=e89a69905d95738673f331e6baa312d9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2015.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2015.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DL%25C3%25A1z%25C3%25A1r%26aufirst%3DV.%26atitle%3DCollateral%2520Sensitivity%2520of%2520Antibiotic-Resistant%2520Microbes%26jtitle%3DTrends%2520Microbiol.%26date%3D2015%26volume%3D23%26issue%3D7%26spage%3D401%26epage%3D407%26doi%3D10.1016%2Fj.tim.2015.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horinouchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furusawa, C.</span></span> <span> </span><span class="NLM_article-title">Prediction of Antibiotic Resistance by Gene Expression Profiles</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5792</span>, <span class="refDoi"> DOI: 10.1038/ncomms6792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fncomms6792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25517437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFWqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5792&author=S.+Suzukiauthor=T.+Horinouchiauthor=C.+Furusawa&title=Prediction+of+Antibiotic+Resistance+by+Gene+Expression+Profiles&doi=10.1038%2Fncomms6792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of antibiotic resistance by gene expression profiles</span></div><div class="casAuthors">Suzuki, Shingo; Horinouchi, Takaaki; Furusawa, Chikara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5792</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although many mutations contributing to antibiotic resistance have been identified, the relationship between the mutations and the related phenotypic changes responsible for the resistance has yet to be fully elucidated.  To better characterize phenotype-genotype mapping for drug resistance, here we analyze phenotypic and genotypic changes of antibiotic-resistant Escherichia coli strains obtained by lab. evolution.  We demonstrate that the resistances can be quant. predicted by the expression changes of a small no. of genes.  Several candidate mutations contributing to the resistances are identified, while phenotype-genotype mapping is suggested to be complex and includes various mutations that cause similar phenotypic changes.  The integration of transcriptome and genome data enables us to ext. essential phenotypic changes for drug resistances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM0giqmwlUxrVg90H21EOLACvtfcHk0ljAyNQZFnmAzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFWqsbk%253D&md5=4f2d72dc34ec9f323200aa3cab85f586</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fncomms6792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6792%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DHorinouchi%26aufirst%3DT.%26aulast%3DFurusawa%26aufirst%3DC.%26atitle%3DPrediction%2520of%2520Antibiotic%2520Resistance%2520by%2520Gene%2520Expression%2520Profiles%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5792%26doi%3D10.1038%2Fncomms6792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Interactions That Select against Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>446</i></span> (<span class="NLM_issue">7136</span>),  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1038/nature05685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature05685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17410176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjslOlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=446&publication_year=2007&pages=668-671&issue=7136&author=R.+Chaitauthor=A.+Craneyauthor=R.+Kishony&title=Antibiotic+Interactions+That+Select+against+Resistance&doi=10.1038%2Fnature05685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic interactions that select against resistance</span></div><div class="casAuthors">Chait, Remy; Craney, Allison; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">446</span>
        (<span class="NLM_cas:issue">7136</span>),
    <span class="NLM_cas:pages">668-671</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Multidrug combinations are increasingly important in combating the spread of antibiotic-resistance in bacterial pathogens.  On a broader scale, such combinations are also important in understanding microbial ecol. and evolution.  Although the effects of multidrug combinations on bacterial growth have been studied extensively, relatively little is known about their impact on the differential selection between sensitive and resistant bacterial populations.  Normally, the presence of a drug confers an advantage on its resistant mutants in competition with the sensitive wild-type population.  Here the authors show, by a direct competition assay between doxycycline-resistant and doxycycline-sensitive Escherichia coli, that this differential selection can be inverted in a hyper-antagonistic class of drug combinations.  Used in such a combination, a drug can render the combined treatment selective against the drug's own resistance allele.  Further, this inversion of selection seems largely insensitive to the underlying resistance mechanism and occurs, at sublethal concns., while maintaining inhibition of the wild type.  These seemingly paradoxical results can be rationalized in terms of a simple geometric argument.  These findings demonstrate a previously unappreciated feature of the fitness landscape for the evolution of resistance and point to a trade-off between the effect of drug interactions on abs. potency and the relative competitive selection that they impose on emerging resistant populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAA0PXHRu6TrVg90H21EOLACvtfcHk0ljAyNQZFnmAzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjslOlsbo%253D&md5=d4f9acd88af10905e4e423efc4e31916</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature05685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05685%26sid%3Dliteratum%253Aachs%26aulast%3DChait%26aufirst%3DR.%26aulast%3DCraney%26aufirst%3DA.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DAntibiotic%2520Interactions%2520That%2520Select%2520against%2520Resistance%26jtitle%3DNature%26date%3D2007%26volume%3D446%26issue%3D7136%26spage%3D668%26epage%3D671%26doi%3D10.1038%2Fnature05685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Drug Interactions Modulate the Potential for Evolution of Resistance</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span> (<span class="NLM_issue">39</span>),  <span class="NLM_fpage">14918</span>– <span class="NLM_lpage">14923</span>, <span class="refDoi"> DOI: 10.1073/pnas.0800944105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0800944105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18815368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1aqur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=14918-14923&issue=39&author=J.-B.+Michelauthor=P.+J.+Yehauthor=R.+Chaitauthor=R.+C.+Moelleringauthor=R.+Kishony&title=Drug+Interactions+Modulate+the+Potential+for+Evolution+of+Resistance&doi=10.1073%2Fpnas.0800944105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Drug interactions modulate the potential for evolution of resistance</span></div><div class="casAuthors">Michel, Jean-Baptiste; Yeh, Pamela J.; Chait, Remy; Moellering, Robert C., Jr.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">14918-14923</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Antimicrobial treatments increasingly rely on multidrug combinations, in part because of the emergence and spread of antibiotic resistance.  The continued effectiveness of combination treatments depends crucially on the frequency with which multidrug resistance arises.  Yet, it is unknown how this propensity for resistance depends on cross-resistance and on epistatic interactions-ranging from synergy to antagonism-between the drugs.  Here, we analyzed how interactions between pairs of drugs affect the spontaneous emergence of resistance in the medically important pathogen Staphylococcus aureus.  Resistance is selected for within a window of drug concns. high enough to inhibit wild-type growth but low enough for some resistant mutants to grow.  Introducing an exptl. method for high-throughput colony imaging, we counted resistant colonies arising across a two-dimensional matrix of drug concns. for each of three drug pairs.  Our data show that these different drug combinations have significantly different impacts on the size of the window of drug concns. where resistance is selected for.  We framed these results in a math. model in which the frequencies of resistance to single drugs, cross-resistance, and epistasis combine to det. the propensity for multidrug resistance.  The theory suggests that drug pairs which interact synergistically, preferred for their immediate efficacy, may in fact favor the future evolution of resistance.  This framework reveals the central role of drug epistasis in the evolution of resistance and points to new strategies for combating the emergence of drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJqt7bUFSpcbVg90H21EOLACvtfcHk0lhdnW5bGErPsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1aqur%252FO&md5=69c1441ff4e1fec2180c5e281da19551</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0800944105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0800944105%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.-B.%26aulast%3DYeh%26aufirst%3DP.%2BJ.%26aulast%3DChait%26aufirst%3DR.%26aulast%3DMoellering%26aufirst%3DR.%2BC.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DDrug%2520Interactions%2520Modulate%2520the%2520Potential%2520for%2520Evolution%2520of%2520Resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D39%26spage%3D14918%26epage%3D14923%26doi%3D10.1073%2Fpnas.0800944105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Site Structure Influences the Evolution of Protein Complex Function and Topology</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3265</span>– <span class="NLM_lpage">3276</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.02.085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.celrep.2018.02.085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29562182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsV2ntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=3265-3276&issue=12&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Ligand+Binding+Site+Structure+Influences+the+Evolution+of+Protein+Complex+Function+and+Topology&doi=10.1016%2Fj.celrep.2018.02.085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Site Structure Influences the Evolution of Protein Complex Function and Topology</span></div><div class="casAuthors">Abrusan, Gyorgy; Marsh, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3265-3276</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">It has been suggested that the evolution of protein complexes is significantly influenced by stochastic, non-adaptive processes.  Using ligand binding as a proxy of function, we show that the structure of ligand-binding sites significantly influences the evolution of protein complexes.  We show that homomers with multi-chain binding sites (MBSs) evolve new functions slower than monomers or other homomers, and those binding cofactors and metals have more conserved quaternary structure than other homomers.  Moreover, the ligands and ligand-binding pockets of homologous MBS homomers are more similar than monomers and other homomers.  Our results suggest strong evolutionary selection for quaternary structure in cofactor-binding MBS homomers, whereas neutral processes are more important in complexes with single-chain binding sites.  They also have pharmacol. implications, suggesting that complexes with single-chain binding sites are better targets for selective drugs, whereas MBS homomers are good candidates for broad-spectrum antibiotic and multitarget drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogMHthIv-ZirVg90H21EOLACvtfcHk0lhdnW5bGErPsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsV2ntLg%253D&md5=40f5a6898caa52e8b1af62d96fb180ee</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.02.085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.02.085%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520Binding%2520Site%2520Structure%2520Influences%2520the%2520Evolution%2520of%2520Protein%2520Complex%2520Function%2520and%2520Topology%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D22%26issue%3D12%26spage%3D3265%26epage%3D3276%26doi%3D10.1016%2Fj.celrep.2018.02.085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span> <span> </span><span class="NLM_article-title">The Evolution of Multimeric Protein Assemblages</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1366</span>, <span class="refDoi"> DOI: 10.1093/molbev/msr300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsr300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22144639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1yhu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=1353-1366&issue=5&author=M.+Lynch&title=The+Evolution+of+Multimeric+Protein+Assemblages&doi=10.1093%2Fmolbev%2Fmsr300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Evolution of Multimeric Protein Assemblages</span></div><div class="casAuthors">Lynch, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology and Evolution</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1353-1366</span>CODEN:
                <span class="NLM_cas:coden">MBEVEO</span>;
        ISSN:<span class="NLM_cas:issn">0737-4038</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the mechanisms by which complex cellular features evolve constitute one of the great unsolved problems of evolutionary biol., it is clear that the emergence of new protein-protein interactions, often accompanied by the diversification of duplicate genes, is involved.  Using information on the levels of protein multimerization in major phylogenetic groups as a guide to the patterns that must be explained and relying on results from population-genetic theory to define the relative plausibility of alternative evolutionary pathways, a framework for understanding the evolution of dimers is developed.  The resultant theory demonstrates that the likelihoods of alternative pathways for the emergence of protein complexes depend strongly on the effective population size.  Nonetheless, it is equally clear that further advancements in this area will require comparative studies on the fitness consequences of alternative monomeric and dimeric proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHo6eOvzfwdbVg90H21EOLACvtfcHk0lhdnW5bGErPsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1yhu7c%253D&md5=debb2b74024d78175fa82eecad1f8f03</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsr300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsr300%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DM.%26atitle%3DThe%2520Evolution%2520of%2520Multimeric%2520Protein%2520Assemblages%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2012%26volume%3D29%26issue%3D5%26spage%3D1353%26epage%3D1366%26doi%3D10.1093%2Fmolbev%2Fmsr300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klabunde, T.</span></span> <span> </span><span class="NLM_article-title">Chemogenomic Approaches to Drug Discovery: Similar Receptors Bind Similar Ligands</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0707308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&issue=1&author=T.+Klabunde&title=Chemogenomic+Approaches+to+Drug+Discovery%3A+Similar+Receptors+Bind+Similar+Ligands&doi=10.1038%2Fsj.bjp.0707308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0lhdnW5bGErPsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520Approaches%2520to%2520Drug%2520Discovery%253A%2520Similar%2520Receptors%2520Bind%2520Similar%2520Ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26issue%3D1%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fsj.bjp.0707308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Alpha Helices Are More Robust to Mutations than Beta Strands</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>), <span class="NLM_elocation-id">e1005242</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1371%2Fjournal.pcbi.1005242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27935949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1ansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&issue=12&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Alpha+Helices+Are+More+Robust+to+Mutations+than+Beta+Strands&doi=10.1371%2Fjournal.pcbi.1005242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha helices are more robust to mutations than beta strands</span></div><div class="casAuthors">Abrusan, Gyoergy; Marsh, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1005242/1-e1005242/16</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The rapidly increasing amt. of data on human genetic variation has resulted in a growing demand to identify pathogenic mutations computationally, as their exptl. validation is currently beyond reach.  Here we show that alpha helixes and beta strands differ significantly in their ability to tolerate mutations: helixes can accumulate more mutations than strands without change, due to the higher nos. of inter-residue contacts in helixes.  This results in two patterns: a) the same no. of mutations causes less structural change in helixes than in strands; b) helixes diverge more rapidly in sequence than strands within the same domains.  Addnl., both helixes and strands are significantly more robust than coils.  Based on this observation we show that human missense mutations that change secondary structure are more likely to be pathogenic than those that do not.  Moreover, inclusion of predicted secondary structure changes shows significant utility for improving upon state-of-theart pathogenicity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqavBv_HbqhX7Vg90H21EOLACvtfcHk0ljD0RZ4A6csdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1ansbs%253D&md5=5f114109a3737aa9075370a938cc2c47</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005242%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DAlpha%2520Helices%2520Are%2520More%2520Robust%2520to%2520Mutations%2520than%2520Beta%2520Strands%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26doi%3D10.1371%2Fjournal.pcbi.1005242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">BioLiP: A Semi-Manually Curated Database for Biologically Relevant Ligand-protein Interactions</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1096</span>– <span class="NLM_lpage">D1103</span>, <span class="refDoi"> DOI: 10.1093/nar/gks966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgks966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=23087378" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=D1096-D1103&issue=D1&author=J.+Yangauthor=A.+Royauthor=Y.+Zhang&title=BioLiP%3A+A+Semi-Manually+Curated+Database+for+Biologically+Relevant+Ligand-protein+Interactions&doi=10.1093%2Fnar%2Fgks966"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks966%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DBioLiP%253A%2520A%2520Semi-Manually%2520Curated%2520Database%2520for%2520Biologically%2520Relevant%2520Ligand-protein%2520Interactions%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D41%26issue%3DD1%26spage%3DD1096%26epage%3DD1103%26doi%3D10.1093%2Fnar%2Fgks966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Algorithm for Detection of Structurally Similar Protein Binding Sites by Local Structural Alignment</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1168</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&issue=9&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Algorithm+for+Detection+of+Structurally+Similar+Protein+Binding+Sites+by+Local+Structural+Alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0ljD0RZ4A6csdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Algorithm%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%2520by%2520Local%2520Structural%2520Alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D9%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Tools (algorithm, Database, and Web Servers) for Predicting and Modeling of Biologically Interesting Proteins</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2017.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.pbiomolbio.2017.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28212856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2017&pages=24-32&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Tools+%28algorithm%2C+Database%2C+and+Web+Servers%29+for+Predicting+and+Modeling+of+Biologically+Interesting+Proteins&doi=10.1016%2Fj.pbiomolbio.2017.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS tools (algorithm, database, and web servers) for predicting and modeling of biologically interesting proteins</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-32</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">ProBiS (Protein Binding Sites) Tools consist of algorithm, database, and web servers for prediction of binding sites and protein ligands based on the detection of structurally similar binding sites in the Protein Data Bank.  In this article, we review the operations that ProBiS Tools perform, provide comments on the evolution of the tools, and give some implementation details.  We review some of its applications to biol. interesting proteins.  ProBiS Tools are freely available at http://probis.cmm.ki.si and http://probis.nih.gov.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmqisRzdGSrVg90H21EOLACvtfcHk0ljD0RZ4A6csdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemtLo%253D&md5=2a34bc81fb14bce843f917f496383ee5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2017.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2017.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Tools%2520%2528algorithm%252C%2520Database%252C%2520and%2520Web%2520Servers%2529%2520for%2520Predicting%2520and%2520Modeling%2520of%2520Biologically%2520Interesting%2520Proteins%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D128%26spage%3D24%26epage%3D32%26doi%3D10.1016%2Fj.pbiomolbio.2017.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">DOCK 6: Impact of New Features and Current Docking Performance</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1156</span>, <span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&issue=15&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+New+Features+and+Current+Docking+Performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0lgbtZZw4aGLCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520New%2520Features%2520and%2520Current%2520Docking%2520Performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26issue%3D15%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of DOCK 6 as a Pose Generation and Database Enrichment Tool</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">773</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9565-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1007%2Fs10822-012-9565-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22569593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=749-773&issue=6&author=S.+R.+Brozellauthor=S.+Mukherjeeauthor=T.+E.+Baliusauthor=D.+R.+Roeauthor=D.+A.+Caseauthor=R.+C.+Rizzo&title=Evaluation+of+DOCK+6+as+a+Pose+Generation+and+Database+Enrichment+Tool&doi=10.1007%2Fs10822-012-9565-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of DOCK 6 as a pose generation and database enrichment tool</span></div><div class="casAuthors">Brozell, Scott R.; Mukherjee, Sudipto; Balius, Trent E.; Roe, Daniel R.; Case, David A.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">749-773</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In conjunction with the recent American Chem. Society symposium titled "Docking and Scoring: A Review of Docking Programs" the performance of the DOCK6 program was evaluated through (1) pose reprodn. and (2) database enrichment calcns. on a common set of organizer-specified systems and datasets (ASTEX, DUD, WOMBAT).  Representative baseline grid score results averaged over five docking runs yield a relatively high pose identification success rate of 72.5 % (symmetry cor. rmsd) and sampling rate of 91.9 % for the multi site ASTEX set (N = 147) using organizer-supplied structures.  Numerous addnl. docking expts. showed that ligand starting conditions, symmetry, multiple binding sites, clustering, and receptor prepn. protocols all affect success.  Encouragingly, in some cases, use of more sophisticated scoring and sampling methods yielded results which were comparable (Amber score ligand movable protocol) or exceeded (LMOD score) analogous baseline grid-score results.  The anal. highlights the potential benefit and challenges assocd. with including receptor flexibility and indicates that different scoring functions have system dependent strengths and weaknesses.  Enrichment studies with the DUD database prepd. using the SB2010 prepn. protocol and native ligand pairings yielded individual area under the curve (AUC) values derived from receiver operating characteristic curve anal. ranging from 0.29 (bad enrichment) to 0.96 (good enrichment) with an av. value of 0.60 (27/38 have AUC ≥ 0.5).  Strong early enrichment was also obsd. in the critically important 1.0-2.0 % region.  Somewhat surprisingly, an alternative receptor prepn. protocol yielded comparable results.  As expected, semi-random pairings yielded poorer enrichments, in particular, for unrelated receptors.  Overall, the breadth and no. of expts. performed provide a useful snapshot of current capabilities of DOCK6 as well as starting points to guide future development efforts to further improve sampling and scoring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYqYQ0o4s9srVg90H21EOLACvtfcHk0lgbtZZw4aGLCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nP&md5=6888c35b532535b2a8ac5f556140878d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9565-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9565-y%26sid%3Dliteratum%253Aachs%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DEvaluation%2520of%2520DOCK%25206%2520as%2520a%2520Pose%2520Generation%2520and%2520Database%2520Enrichment%2520Tool%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26issue%3D6%26spage%3D749%26epage%3D773%26doi%3D10.1007%2Fs10822-012-9565-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fochtman, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Customizable de Novo Design Strategies for DOCK: Application to HIVgp41 and Other Therapeutic Targets</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">2641</span>– <span class="NLM_lpage">2663</span>, <span class="refDoi"> DOI: 10.1002/jcc.25052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.25052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28940386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFClu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=2641-2663&issue=30&author=W.+J.+Allenauthor=B.+C.+Fochtmanauthor=T.+E.+Baliusauthor=R.+C.+Rizzo&title=Customizable+de+Novo+Design+Strategies+for+DOCK%3A+Application+to+HIVgp41+and+Other+Therapeutic+Targets&doi=10.1002%2Fjcc.25052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Customizable de novo design strategies for DOCK: Application to HIVgp41 and other therapeutic targets</span></div><div class="casAuthors">Allen, William J.; Fochtman, Brian C.; Balius, Trent E.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">2641-2663</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">De novo design can be used to explore vast areas of chem. space in computational lead discovery.  As a complement to virtual screening, from-scratch construction of mols. is not limited to compds. in pre-existing vendor catalogs.  Here, we present an iterative fragment growth method, integrated into the program DOCK, in which new mols. are built using rules for allowable connections based on known mols.  The method leverages DOCK's advanced scoring and pruning approaches and users can define very specific criteria in terms of properties or features to customize growth toward a particular region of chem. space.  The code was validated using three increasingly difficult classes of calcns.: (1) Rebuilding known X-ray ligands taken from 663 complexes using only their component parts (focused libraries), (2) construction of new ligands in 57 drug target sites using a library derived from ∼13M drug-like compds. (generic libraries), and (3) application to a challenging protein-protein interface on the viral drug target HIVgp41.  The computational testing confirms that the de novo DOCK routines are robust and working as envisioned, and the compelling results highlight the potential utility for designing new mols. against a wide variety of important protein targets. © 2017 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMx6MoUXyTU7Vg90H21EOLACvtfcHk0lhOFbUiuxOHTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFClu7fE&md5=807e8526b3c314e6c02e9f91e16ea5e3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fjcc.25052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.25052%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DFochtman%26aufirst%3DB.%2BC.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DCustomizable%2520de%2520Novo%2520Design%2520Strategies%2520for%2520DOCK%253A%2520Application%2520to%2520HIVgp41%2520and%2520Other%2520Therapeutic%2520Targets%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2017%26volume%3D38%26issue%3D30%26spage%3D2641%26epage%3D2663%26doi%3D10.1002%2Fjcc.25052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: A Major Update to the DrugBank Database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&issue=D1&author=D.%0AS.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+A+Major+Update+to+the+DrugBank+Database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lhOFbUiuxOHTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520A%2520Major%2520Update%2520to%2520the%2520DrugBank%2520Database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">ZINC 15 - Ligand Discovery for Everyone</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2324</span>– <span class="NLM_lpage">2337</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&issue=11&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+-+Ligand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lhOFbUiuxOHTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520-%2520Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D11%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical Properties of Antibacterial Compounds: Implications for Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&issue=10&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+Properties+of+Antibacterial+Compounds%3A+Implications+for+Drug+Discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0lgkll-dKSIwHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520Properties%2520of%2520Antibacterial%2520Compounds%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D10%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Bombarelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvenaud, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Lobato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Lengeling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheberla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera-Iparraguirre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirzel, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span> <span> </span><span class="NLM_article-title">Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntlWquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=268-276&author=R.+G%C3%B3mez-Bombarelliauthor=J.+N.+Weiauthor=D.+Duvenaudauthor=J.+M.+Hern%C3%A1ndez-Lobatoauthor=B.+S%C3%A1nchez-Lengelingauthor=D.+Sheberlaauthor=J.+Aguilera-Iparraguirreauthor=T.+D.+Hirzelauthor=R.+P.+Adamsauthor=A.+Aspuru-Guzik&title=Automatic+Chemical+Design+Using+a+Data-Driven+Continuous+Representation+of+Molecules&doi=10.1021%2Facscentsci.7b00572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules</span></div><div class="casAuthors">Gomez-Bombarelli, Rafael; Wei, Jennifer N.; Duvenaud, David; Hernandez-Lobato, Jose Miguel; Sanchez-Lengeling, Benjamin; Sheberla, Dennis; Aguilera-Iparraguirre, Jorge; Hirzel, Timothy D.; Adams, Ryan P.; Aspuru-Guzik, Alan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">268-276</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a method to convert discrete representations of mols. to and from a multidimensional continuous representation.  This model allows us to generate new mols. for efficient exploration and optimization through open-ended spaces of chem. compds.  A deep neural network was trained on hundreds of thousands of existing chem. structures to construct three coupled functions: an encoder, a decoder, and a predictor.  The encoder converts the discrete representation of a mol. into a real-valued continuous vector, and the decoder converts these continuous vectors back to discrete mol. representations.  The predictor ests. chem. properties from the latent continuous vector representation of the mol.  Continuous representations of mols. allow us to automatically generate novel chem. structures by performing simple operations in the latent space, such as decoding random vectors, perturbing known chem. structures, or interpolating between mols.  Continuous representations also allow the use of powerful gradient-based optimization to efficiently guide the search for optimized functional compds.  We demonstrate our method in the domain of drug-like mols. and also in a set of mols. with fewer that nine heavy atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXN9RuDQ2cp7Vg90H21EOLACvtfcHk0lgkll-dKSIwHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntlWquw%253D%253D&md5=322d9ff569fc9c8831e91d915104d985</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00572%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B3mez-Bombarelli%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DJ.%2BN.%26aulast%3DDuvenaud%26aufirst%3DD.%26aulast%3DHern%25C3%25A1ndez-Lobato%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1nchez-Lengeling%26aufirst%3DB.%26aulast%3DSheberla%26aufirst%3DD.%26aulast%3DAguilera-Iparraguirre%26aufirst%3DJ.%26aulast%3DHirzel%26aufirst%3DT.%2BD.%26aulast%3DAdams%26aufirst%3DR.%2BP.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26atitle%3DAutomatic%2520Chemical%2520Design%2520Using%2520a%2520Data-Driven%2520Continuous%2520Representation%2520of%2520Molecules%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26spage%3D268%26epage%3D276%26doi%3D10.1021%2Facscentsci.7b00572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Estimation of Synthetic Accessibility Score of Drug-like Molecules Based on Molecular Complexity and Fragment Contributions</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">8</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-1-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1758-2946-1-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=8&author=P.+Ertlauthor=A.+Schuffenhauer&title=Estimation+of+Synthetic+Accessibility+Score+of+Drug-like+Molecules+Based+on+Molecular+Complexity+and+Fragment+Contributions&doi=10.1186%2F1758-2946-1-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span></div><div class="casAuthors">Ertl Peter; Schuffenhauer Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A method to estimate ease of synthesis (synthetic accessibility) of drug-like molecules is needed in many areas of the drug discovery process.  The development and validation of such a method that is able to characterize molecule synthetic accessibility as a score between 1 (easy to make) and 10 (very difficult to make) is described in this article.  RESULTS:  The method for estimation of the synthetic accessibility score (SAscore) described here is based on a combination of fragment contributions and a complexity penalty.  Fragment contributions have been calculated based on the analysis of one million representative molecules from PubChem and therefore one can say that they capture historical synthetic knowledge stored in this database.  The molecular complexity score takes into account the presence of non-standard structural features, such as large rings, non-standard ring fusions, stereocomplexity and molecule size.  The method has been validated by comparing calculated SAscores with ease of synthesis as estimated by experienced medicinal chemists for a set of 40 molecules.  The agreement between calculated and manually estimated synthetic accessibility is very good with r2 = 0.89.  CONCLUSION:  A novel method to estimate synthetic accessibility of molecules has been developed.  This method uses historical synthetic knowledge obtained by analyzing information from millions of already synthesized chemicals and considers also molecule complexity.  The method is sufficiently fast and provides results consistent with estimation of ease of synthesis by experienced medicinal chemists.  The calculated SAscore may be used to support various drug discovery processes where a large number of molecules needs to be ranked based on their synthetic accessibility, for example when purchasing samples for screening, selecting hits from high-throughput screening for follow-up, or ranking molecules generated by various de novo design approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQW_MJ6K-6QK9K0qsdY6Nc1fW6udTcc2eYohNU56A6Pw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D&md5=9ffc064a9d3952a583532411cb0f0673</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-8%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DEstimation%2520of%2520Synthetic%2520Accessibility%2520Score%2520of%2520Drug-like%2520Molecules%2520Based%2520on%2520Molecular%2520Complexity%2520and%2520Fragment%2520Contributions%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-1-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muresan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Quantifying the Chemical Beauty of Drugs</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">90</span>, <span class="refDoi"> DOI: 10.1038/nchem.1243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnchem.1243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22270643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=90&issue=2&author=G.+R.+Bickertonauthor=G.+V.+Paoliniauthor=J.+Besnardauthor=S.+Muresanauthor=A.+L.+Hopkins&title=Quantifying+the+Chemical+Beauty+of+Drugs&doi=10.1038%2Fnchem.1243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chemical beauty of drugs</span></div><div class="casAuthors">Bickerton, G. Richard; Paolini, Gaia V.; Besnard, Jeremy; Muresan, Sorel; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">90-98</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-likeness is a key consideration when selecting compds. during the early stages of drug discovery.  However, evaluation of drug-likeness in abs. terms does not reflect adequately the whole spectrum of compd. quality.  More worryingly, widely used rules may inadvertently foster undesirable mol. property inflation as they permit the encroachment of rule-compliant compds. towards their boundaries.  We propose a measure of drug-likeness based on the concept of desirability called the quant. est. of drug-likeness (QED).  The empirical rationale of QED reflects the underlying distribution of mol. properties.  QED is intuitive, transparent, straightforward to implement in many practical settings and allows compds. to be ranked by their relative merit.  We extended the utility of QED by applying it to the problem of mol. target druggability assessment by prioritizing a large set of published bioactive compds.  The measure may also capture the abstr. notion of aesthetics in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCb7vrfeUodrVg90H21EOLACvtfcHk0lgkll-dKSIwHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D&md5=7ab125d4ab381924f965d072695f7432</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1243%26sid%3Dliteratum%253Aachs%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DQuantifying%2520the%2520Chemical%2520Beauty%2520of%2520Drugs%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26issue%3D2%26spage%3D90%26doi%3D10.1038%2Fnchem.1243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
der Maaten, L.</span></span> <span> </span><span class="NLM_article-title">Accelerating T-SNE Using Tree-Based Algorithms</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3221</span>– <span class="NLM_lpage">3245</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3221-3245&author=L.+Van%0Ader+Maaten&title=Accelerating+T-SNE+Using+Tree-Based+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BMaaten%26aufirst%3DL.%26atitle%3DAccelerating%2520T-SNE%2520Using%2520Tree-Based%2520Algorithms%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D3221%26epage%3D3245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasiri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otero, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermoso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Antibiotic (E)-3-(3-Carboxyphenyl)-2-(4-Cyanostyryl)quinazolin-4(3H)-One</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1738</span>– <span class="NLM_lpage">1741</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b00056</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b00056" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12lsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=1738-1741&issue=5&author=R.+Bouleyauthor=M.+Kumarasiriauthor=Z.+Pengauthor=L.+H.+Oteroauthor=W.+Songauthor=M.+A.+Suckowauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=E.+Lastochkinauthor=N.+T.+Antunesauthor=H.+Piauthor=S.+Vakulenkoauthor=J.+A.+Hermosoauthor=M.+Changauthor=S.+Mobashery&title=Discovery+of+Antibiotic+%28E%29-3-%283-Carboxyphenyl%29-2-%284-Cyanostyryl%29quinazolin-4%283H%29-One&doi=10.1021%2Fjacs.5b00056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Antibiotic (E)-3-(3-Carboxyphenyl)-2-(4-cyanostyryl)quinazolin-4(3H)-one</span></div><div class="casAuthors">Bouley, Renee; Kumarasiri, Malika; Peng, Zhihong; Otero, Lisandro H.; Song, Wei; Suckow, Mark A.; Schroeder, Valerie A.; Wolter, William R.; Lastochkin, Elena; Antunes, Nuno T.; Pi, Hualiang; Vakulenko, Sergei; Hermoso, Juan A.; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1738-1741</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the face of the clin. challenge posed by resistant bacteria, the present needs for novel classes of antibiotics are genuine.  In silico docking and screening, followed by chem. synthesis of a library of quinazolinones, led to the discovery of (E)-3-(3-carboxyphenyl)-2-(4-cyanostyryl)quinazolin-4(3H)-one (compd. 2) as an antibiotic effective in vivo against methicillin-resistant Staphylococcus aureus (MRSA).  This antibiotic impairs cell-wall biosynthesis as documented by functional assays, showing binding of 2 to penicillin-binding protein (PBP) 2a.  We document that the antibiotic also inhibits PBP1 of S. aureus, indicating a broad targeting of structurally similar PBPs by this antibiotic.  This class of antibiotics holds promise in fighting MRSA infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7oFjd7fxCHrVg90H21EOLACvtfcHk0ljmBsxB_9CDkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12lsb8%253D&md5=c03bc141d73ee85c387fb4be6499a945</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b00056%26sid%3Dliteratum%253Aachs%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DKumarasiri%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DOtero%26aufirst%3DL.%2BH.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DHermoso%26aufirst%3DJ.%2BA.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Antibiotic%2520%2528E%2529-3-%25283-Carboxyphenyl%2529-2-%25284-Cyanostyryl%2529quinazolin-4%25283H%2529-One%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26issue%3D5%26spage%3D1738%26epage%3D1741%26doi%3D10.1021%2Fjacs.5b00056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for the 4(3H)-Quinazolinone Antibacterials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5011</span>– <span class="NLM_lpage">5021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00372</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00372" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtF2lsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5011-5021&issue=10&author=R.+Bouleyauthor=D.+Dingauthor=Z.+Pengauthor=M.+Bastianauthor=E.+Lastochkinauthor=W.+Songauthor=M.+A.+Suckowauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=S.+Mobasheryauthor=M.+Chang&title=Structure-Activity+Relationship+for+the+4%283H%29-Quinazolinone+Antibacterials&doi=10.1021%2Facs.jmedchem.6b00372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for the 4(3H)-Quinazolinone Antibacterials</span></div><div class="casAuthors">Bouley, Renee; Ding, Derong; Peng, Zhihong; Bastian, Maria; Lastochkin, Elena; Song, Wei; Suckow, Mark A.; Schroeder, Valerie A.; Wolter, William R.; Mobashery, Shahriar; Chang, Mayland</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5011-5021</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently reported on the discovery of a novel antibacterial (2) with a 4(3H)-quinazolinone core.  This discovery was made by in silico screening of 1.2 million compds. for binding to a penicillin-binding protein and the subsequent demonstration of antibacterial activity against Staphylococcus aureus.  The first structure-activity relationship for this antibacterial scaffold is explored in this report with evaluation of 77 variants of the structural class.  Eleven promising compds. were further evaluated for in vitro toxicity, pharmacokinetics, and efficacy in a mouse peritonitis model of infection, which led to the discovery of compd. 27.  This new quinazolinone has potent activity against methicillin-resistant (MRSA) strains, low clearance, oral bioavailability and shows efficacy in a mouse neutropenic thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoES54kEeDf6bVg90H21EOLACvtfcHk0ljmBsxB_9CDkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtF2lsr4%253D&md5=9287f7924141b19a517b3ec04d723eff</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00372%26sid%3Dliteratum%253Aachs%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DBastian%26aufirst%3DM.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DMobashery%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520the%25204%25283H%2529-Quinazolinone%2520Antibacterials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D10%26spage%3D5011%26epage%3D5021%26doi%3D10.1021%2Facs.jmedchem.6b00372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gour, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malasala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borra, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of 1,2,3-Triazole Linked 4(3H)-Quinazolinones as Potent Antibacterial Agents against Multidrug-Resistant Staphylococcus Aureus</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1056</span>– <span class="NLM_lpage">1067</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.ejmech.2018.08.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=30176536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OmsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=1056-1067&author=S.+Gatadiauthor=J.+Gourauthor=M.+Shuklaauthor=G.+Kaulauthor=S.+Dasauthor=A.+Dasguptaauthor=S.+Malasalaauthor=R.+S.+Borraauthor=Y.+V.+Madhaviauthor=S.+Chopraauthor=S.+Nanduri&title=Synthesis+of+1%2C2%2C3-Triazole+Linked+4%283H%29-Quinazolinones+as+Potent+Antibacterial+Agents+against+Multidrug-Resistant+Staphylococcus+Aureus&doi=10.1016%2Fj.ejmech.2018.08.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of 1,2,3-triazole linked 4(3H)-Quinazolinones as potent antibacterial agents against multidrug-resistant Staphylococcus aureus</span></div><div class="casAuthors">Gatadi, Srikanth; Gour, Jitendra; Shukla, Manjulika; Kaul, Grace; Das, Swetarka; Dasgupta, Arunava; Malasala, Satyaveni; Borra, Ramya Sri; Madhavi, Y. V.; Chopra, Sidharth; Nanduri, Srinivas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1056-1067</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Methicillin and vancomycin resistant Staphylococcus aureus infections are an emerging global health concern leading to increasing morbidity and mortality.  Continuous increase in drug resistance has underlined the need for discovery and development of new antibacterial agents acting via novel mechanisms to overcome this pressing issue.  In this context, a no. of 1,2,3-triazole linked 4(3H)-quinazolinone derivs. were designed and synthesized as potent antibacterial agents.  When evaluated against ESKAP pathogen panel, a wide variety of substituted compds. exhibited significantly selective inhibitory activities towards Staphylococcus aureus (MIC = 0.5-4 μg/mL).  To understand and confirm the specificity of these compds., the compds. I and II were tested against E. coli and A. baumannii in combination with sub-lethal concns. of Polymyxin B nonapeptide (PMBN) and were found to be inactive.  This clearly indicated that these compds. possess specific and potent activity towards S. aureus and are inactive against gram-neg. pathogens.  Encouragingly, the compds. were also found to be non toxic to Vero cells and displayed favorable selectivity index (SI = 40 to 80).  Furthermore, I and II were found to possess potent inhibitory activity when tested against multidrug resistant S. aureus including strains resistant to vancomycin (MIC values 0.5-32 μg/mL), indicating that the compds. are able to escape current drug-resistance mechanisms.  With the potent anti-bacterial activity exhibited the new series of 1,2,3-triazole linked 4(3H)-quinazolinones have emerged as promising candidates for treating multidrug resistant Staphylococcus aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDsXYciQOosbVg90H21EOLACvtfcHk0lik18oR9WjV4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OmsLrO&md5=10d0b1f98545762afd4df0dbabd25e41</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.070%26sid%3Dliteratum%253Aachs%26aulast%3DGatadi%26aufirst%3DS.%26aulast%3DGour%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DKaul%26aufirst%3DG.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DMalasala%26aufirst%3DS.%26aulast%3DBorra%26aufirst%3DR.%2BS.%26aulast%3DMadhavi%26aufirst%3DY.%2BV.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DS.%26atitle%3DSynthesis%2520of%25201%252C2%252C3-Triazole%2520Linked%25204%25283H%2529-Quinazolinones%2520as%2520Potent%2520Antibacterial%2520Agents%2520against%2520Multidrug-Resistant%2520Staphylococcus%2520Aureus%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D1056%26epage%3D1067%26doi%3D10.1016%2Fj.ejmech.2018.08.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gour, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akunuri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of New 3-Phenylquinazolin-4(3H)-One Derivatives as Potent Antibacterial Agents Effective against Methicillin- and Vancomycin-Resistant Staphylococcus Aureus (MRSA and VRSA)</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bioorg.2018.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=30130650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCgs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=175-183&author=S.+Gatadiauthor=J.+Gourauthor=G.+Kaulauthor=M.+Shuklaauthor=A.+Dasguptaauthor=R.+Akunuriauthor=R.+Tripathiauthor=Y.+V.+Madhaviauthor=S.+Chopraauthor=S.+Nanduri&title=Synthesis+of+New+3-Phenylquinazolin-4%283H%29-One+Derivatives+as+Potent+Antibacterial+Agents+Effective+against+Methicillin-+and+Vancomycin-Resistant+Staphylococcus+Aureus+%28MRSA+and+VRSA%29&doi=10.1016%2Fj.bioorg.2018.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of new 3-phenylquinazolin-4(3H)-one derivatives as potent antibacterial agents effective against methicillin-resistant and vancomycin-resistant Staphylococcus aureus (MRSA and VRSA)</span></div><div class="casAuthors">Gatadi, Srikanth; Gour, Jitendra; Kaul, Grace; Shukla, Manjulika; Dasgupta, Arunava; Akunuri, Ravikumar; Tripathi, Richa; Madhavi, Y. V.; Chopra, Sidharth; Nanduri, Srinivas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-183</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Occurrence of infections due to the drug resistant Staphylococcus aureus is on rise necessitating the need for rapid development of new antibacterial agents.  In our present work, a series of new 3-phenylquinazolin-4(3H)-one derivs. were designed, synthesized and evaluated for their antibacterial activity against ESKAP (E. coli, S. aureus, K. pneumoniae, A. baumannii, P. aeruginosa) pathogen panel and pathogenic mycobacterial strains.  The study revealed that compds. 4a, 4c, 4e, 4f, 4g, 4i, 4o and 4p exhibited selective and potent inhibitory activity against Staphylococcus aureus with MIC values in the range of 0.125-8 μg/mL.  Further, the compds. 4c, 4e and 4g were found to be non toxic to Vero cells (CC50 = >10->100 μg/mL) and exhibited favorable selectivity index (SI = 40->200).  The compds. 4c, 4e and 4g also showed potent inhibitory activity against various MDR-S. aureus including VRSA.  The promising results obtained indicated the potential use of the above series of compds. as promising antibacterial agents for the treatment of multidrug resistant Staphylococcus aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2pnk1A-dXrVg90H21EOLACvtfcHk0lik18oR9WjV4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCgs7zL&md5=a21ae7970485b6ce84110f8fa3f0bc64</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DGatadi%26aufirst%3DS.%26aulast%3DGour%26aufirst%3DJ.%26aulast%3DKaul%26aufirst%3DG.%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DAkunuri%26aufirst%3DR.%26aulast%3DTripathi%26aufirst%3DR.%26aulast%3DMadhavi%26aufirst%3DY.%2BV.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520New%25203-Phenylquinazolin-4%25283H%2529-One%2520Derivatives%2520as%2520Potent%2520Antibacterial%2520Agents%2520Effective%2520against%2520Methicillin-%2520and%2520Vancomycin-Resistant%2520Staphylococcus%2520Aureus%2520%2528MRSA%2520and%2520VRSA%2529%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D175%26epage%3D183%26doi%3D10.1016%2Fj.bioorg.2018.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Daniel, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testero, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spink, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llarrull, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasiri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espahbodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenwalter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a New Class of Non-Β-Lactam Inhibitors of Penicillin-Binding Proteins with Gram-Positive Antibacterial Activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3664</span>– <span class="NLM_lpage">3672</span>, <span class="refDoi"> DOI: 10.1021/ja500053x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja500053x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVWhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=3664-3672&issue=9&author=P.+I.+O%E2%80%99Danielauthor=Z.+Pengauthor=H.+Piauthor=S.+A.+Testeroauthor=D.+Dingauthor=E.+Spinkauthor=E.+Leemansauthor=M.+A.+Boudreauauthor=T.+Yamaguchiauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=L.+I.+Llarrullauthor=W.+Songauthor=E.+Lastochkinauthor=M.+Kumarasiriauthor=N.+T.+Antunesauthor=M.+Espahbodiauthor=K.+Lichtenwalterauthor=M.+A.+Suckowauthor=S.+Vakulenkoauthor=S.+Mobasheryauthor=M.+Chang&title=Discovery+of+a+New+Class+of+Non-%CE%92-Lactam+Inhibitors+of+Penicillin-Binding+Proteins+with+Gram-Positive+Antibacterial+Activity&doi=10.1021%2Fja500053x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Class of Non-β-lactam Inhibitors of Penicillin-Binding Proteins with Gram-Positive Antibacterial Activity</span></div><div class="casAuthors">O'Daniel, Peter I.; Peng, Zhihong; Pi, Hualiang; Testero, Sebastian A.; Ding, Derong; Spink, Edward; Leemans, Erika; Boudreau, Marc A.; Yamaguchi, Takao; Schroeder, Valerie A.; Wolter, William R.; Llarrull, Leticia I.; Song, Wei; Lastochkin, Elena; Kumarasiri, Malika; Antunes, Nuno T.; Espahbodi, Mana; Lichtenwalter, Katerina; Suckow, Mark A.; Vakulenko, Sergei; Mobashery, Shahriar; Chang, Mayland</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3664-3672</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Infections caused by hard-to-treat methicillin-resistant Staphylococcus aureus (MRSA) are a serious global public-health concern, as MRSA has become broadly resistant to many classes of antibiotics.  We disclose herein the discovery of a new class of non-β-lactam antibiotics, the oxadiazoles, which inhibit penicillin-binding protein 2a (PBP2a) of MRSA.  The oxadiazoles show bactericidal activity against vancomycin- and linezolid-resistant MRSA and other Gram-pos. bacterial strains, in vivo efficacy in a mouse model of infection, and have 100% oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDVcdd66dic7Vg90H21EOLACvtfcHk0lik18oR9WjV4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVWhs70%253D&md5=e8fb52aeb853be0a2d8cfd55d67a39f8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja500053x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja500053x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Daniel%26aufirst%3DP.%2BI.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DTestero%26aufirst%3DS.%2BA.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DSpink%26aufirst%3DE.%26aulast%3DLeemans%26aufirst%3DE.%26aulast%3DBoudreau%26aufirst%3DM.%2BA.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DLlarrull%26aufirst%3DL.%2BI.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DKumarasiri%26aufirst%3DM.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DEspahbodi%26aufirst%3DM.%26aulast%3DLichtenwalter%26aufirst%3DK.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DMobashery%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520New%2520Class%2520of%2520Non-%25CE%2592-Lactam%2520Inhibitors%2520of%2520Penicillin-Binding%2520Proteins%2520with%2520Gram-Positive%2520Antibacterial%2520Activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D9%26spage%3D3664%26epage%3D3672%26doi%3D10.1021%2Fja500053x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spink, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenwalter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Daniel, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testero, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for the Oxadiazole Class of Antibiotics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1380</span>– <span class="NLM_lpage">1389</span>, <span class="refDoi"> DOI: 10.1021/jm501661f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501661f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVOlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1380-1389&issue=3&author=E.+Spinkauthor=D.+Dingauthor=Z.+Pengauthor=M.+A.+Boudreauauthor=E.+Leemansauthor=E.+Lastochkinauthor=W.+Songauthor=K.+Lichtenwalterauthor=P.+I.+O%E2%80%99Danielauthor=S.+A.+Testeroauthor=H.+Piauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=N.+T.+Antunesauthor=M.+A.+Suckowauthor=S.+Vakulenkoauthor=M.+Changauthor=S.+Mobashery&title=Structure-Activity+Relationship+for+the+Oxadiazole+Class+of+Antibiotics&doi=10.1021%2Fjm501661f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for the Oxadiazole Class of Antibiotics</span></div><div class="casAuthors">Spink, Edward; Ding, Derong; Peng, Zhihong; Boudreau, Marc A.; Leemans, Erika; Lastochkin, Elena; Song, Wei; Lichtenwalter, Katerina; O'Daniel, Peter I.; Testero, Sebastian A.; Pi, Hualiang; Schroeder, Valerie A.; Wolter, William R.; Antunes, Nuno T.; Suckow, Mark A.; Vakulenko, Sergei; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1380-1389</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-activity relationship (SAR) for the newly discovered oxadiazole class of antibiotics is described with evaluation of 120 derivs. of the lead structure.  This class of antibiotics was discovered by in silico docking and scoring against the crystal structure of a penicillin-binding protein.  They impair cell-wall biosynthesis and exhibit activities against the Gram-pos. bacterium Staphylococcus aureus, including methicillin-resistant S. aureus (MRSA) and vancomycin-resistant and linezolid-resistant S. aureus.  Compd. I was efficacious in a mouse model of MRSA infection, exhibiting a long half-life, a high vol. of distribution, and low clearance.  This antibiotic is bactericidal and is orally bioavailable in mice.  This class of antibiotics holds great promise in recourse against infections by MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7oqinjSM5EbVg90H21EOLACvtfcHk0lgemsmTl9RGng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVOlsg%253D%253D&md5=197ec5fb19de5b9f56e83d0979ebb77a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm501661f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501661f%26sid%3Dliteratum%253Aachs%26aulast%3DSpink%26aufirst%3DE.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DBoudreau%26aufirst%3DM.%2BA.%26aulast%3DLeemans%26aufirst%3DE.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLichtenwalter%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Daniel%26aufirst%3DP.%2BI.%26aulast%3DTestero%26aufirst%3DS.%2BA.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520the%2520Oxadiazole%2520Class%2520of%2520Antibiotics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D3%26spage%3D1380%26epage%3D1389%26doi%3D10.1021%2Fjm501661f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janardhanan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">The Oxadiazole Antibacterials</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.mib.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27239942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=13-17&author=J.+Janardhananauthor=M.+Changauthor=S.+Mobashery&title=The+Oxadiazole+Antibacterials&doi=10.1016%2Fj.mib.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The oxadiazole antibacterials</span></div><div class="casAuthors">Janardhanan, Jeshina; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The oxadiazoles are a class of antibacterials discovered by in silico docking and scoring of compds. against the X-ray structure of a penicillin-binding protein.  These antibacterials exhibit activity against Gram-pos. bacteria, including against methicillin-resistant Staphylococcus aureus (MRSA) and vancomycin-resistant Enterococcus (VRE).  They show in vivo efficacy in murine models of peritonitis/sepsis and neutropenic thigh MRSA infection.  They are bactericidal and orally bioavailable.  The oxadiazoles show promise in treatment of MRSA infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGkNPBwBHezbVg90H21EOLACvtfcHk0lgemsmTl9RGng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWlur0%253D&md5=b0a236214aa4f89149d7197e43c8d4e7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DJanardhanan%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DThe%2520Oxadiazole%2520Antibacterials%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2016%26volume%3D33%26spage%3D13%26epage%3D17%26doi%3D10.1016%2Fj.mib.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrio, J. L.</span></span> <span> </span><span class="NLM_article-title">Antifungals</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bcp.2016.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27884742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCgtrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=86-96&author=S.+Campoyauthor=J.+L.+Adrio&title=Antifungals&doi=10.1016%2Fj.bcp.2016.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungals</span></div><div class="casAuthors">Campoy, Sonia; Adrio, Jose L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86-96</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The need for new antifungal agents is undeniable.  Current therapeutic choices for the treatment of invasive fungal infections are limited to three classes of drugs.  Most used antifungal agents are not completely effective due to the development of resistance, host toxicity and undesirable side effects that limit their use in medical practice.  Invasive fungal infections have significantly increased over the last decades and the mortality rates remain unacceptably high.  More threatening, new resistance patterns have been obsd. including simultaneous resistance to different antifungal classes.  In the last years, deeper insights into the mol. mechanisms for fungal resistance and virulence have yielded some new potential targets for antifungal therapeutics.  Chem. genomics-based screenings, high throughput screenings of natural products and repurposing of approved drugs are some of the approaches being followed for the discovery of new antifungal mols.  However, despite the emerging need for effective antifungal agents, the current pipeline contains only a few promising mols., with novel modes of action, in early clin. development stages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplq5ZsJFOZS7Vg90H21EOLACvtfcHk0lgemsmTl9RGng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCgtrfI&md5=1d84f62c68cf48c04e498746156a90fe</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCampoy%26aufirst%3DS.%26aulast%3DAdrio%26aufirst%3DJ.%2BL.%26atitle%3DAntifungals%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D133%26spage%3D86%26epage%3D96%26doi%3D10.1016%2Fj.bcp.2016.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, E. I.</span></span> <span> </span><span class="NLM_article-title">Morpholine Antifungals and Their Mode of Action</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">788</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1042/bst0190788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1042%2Fbst0190788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1783217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADyaK3MXkvVClu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=788-793&issue=3&author=E.+I.+Mercer&title=Morpholine+Antifungals+and+Their+Mode+of+Action&doi=10.1042%2Fbst0190788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Morpholine antifungals and their mode of action</span></div><div class="casAuthors">Mercer, E. Ian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">788-93</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    </div><div class="casAbstract">A review with 30 refs., discussing the effects of morpholines on the growth of fungi, higher plants, and protozoa.  It details the primary biochem. effects of these compds. and elucidates the mechanism of inhibition of sterol-synthesizing enzymes, the consequences of sterol precursor accumulation and/or ergosterol deprivation in fungi, as well as the consequences of accumulation of 9β,19-cyclopropyl sterols in higher plants and protozoa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8INR0FrE8-rVg90H21EOLACvtfcHk0lgqwEKsXososw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXkvVClu74%253D&md5=facd28fad52aac9160b1901006a04545</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1042%2Fbst0190788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbst0190788%26sid%3Dliteratum%253Aachs%26aulast%3DMercer%26aufirst%3DE.%2BI.%26atitle%3DMorpholine%2520Antifungals%2520and%2520Their%2520Mode%2520of%2520Action%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D1991%26volume%3D19%26issue%3D3%26spage%3D788%26epage%3D793%26doi%3D10.1042%2Fbst0190788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, R.</span>; <span class="NLM_string-name">Shah, A.
H.</span>; <span class="NLM_string-name">Rawal, M.
K.</span></span> <span> </span><span class="NLM_article-title">Antifungals: Mechanism of Action and Drug Resistance</span>. In  <i>Yeast Membrane Transport</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, J.</span>, <span class="NLM_string-name">Sychrová, H.</span>, <span class="NLM_string-name">Kschischo, M.</span></span>, Eds.; Advances in Experimental Medicine and Biology; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham, Switzerland</span>, <span class="NLM_year">2016</span>; pp  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">349</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1007%2F978-3-319-25304-6_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=327-349&author=R.+Prasad&author=A.%0AH.+Shah&author=M.%0AK.+Rawalauthor=J.+Ramos&author=H.+Sychrov%C3%A1&author=M.+Kschischo&title=Yeast+Membrane+Transport"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-25304-6_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-25304-6_14%26sid%3Dliteratum%253Aachs%26aulast%3DPrasad%26aufirst%3DR.%26atitle%3DAntifungals%253A%2520Mechanism%2520of%2520Action%2520and%2520Drug%2520Resistance%26btitle%3DYeast%2520Membrane%2520Transport%26aulast%3DRamos%26aufirst%3DJ.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2016%26spage%3D327%26epage%3D349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldred, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osheroff, N.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Quinolone Action and Resistance</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1574</span>, <span class="refDoi"> DOI: 10.1021/bi5000564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5000564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=1565-1574&issue=10&author=K.+J.+Aldredauthor=R.+J.+Kernsauthor=N.+Osheroff&title=Mechanism+of+Quinolone+Action+and+Resistance&doi=10.1021%2Fbi5000564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of quinolone action and resistance</span></div><div class="casAuthors">Aldred, Katie J.; Kerns, Robert J.; Osheroff, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1565-1574</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quinolones are one of the most commonly prescribed classes of antibacterials in the world and are used to treat a variety of bacterial infections in humans.  Because of the wide use (and overuse) of these drugs, the no. of quinolone-resistant bacterial strains has been growing steadily since the 1990s.  As is the case with other antibacterial agents, the rise in quinolone resistance threatens the clin. utility of this important drug class.  Quinolones act by converting their targets, gyrase and topoisomerase IV, into toxic enzymes that fragment the bacterial chromosome.  This review describes the development of the quinolones as antibacterials, the structure and function of gyrase and topoisomerase IV, and the mechanistic basis for quinolone action against their enzyme targets.  It will then discuss the following three mechanisms that decrease the sensitivity of bacterial cells to quinolones.  Target-mediated resistance is the most common and clin. significant form of resistance.  It is caused by specific mutations in gyrase and topoisomerase IV that weaken interactions between quinolones and these enzymes.  Plasmid-mediated resistance results from extrachromosomal elements that encode proteins that disrupt quinolone-enzyme interactions, alter drug metab., or increase quinolone efflux.  Chromosome-mediated resistance results from the underexpression of porins or the overexpression of cellular efflux pumps, both of which decrease cellular concns. of quinolones.  Finally, this review discusses recent advancements in our understanding of how quinolones interact with gyrase and topoisomerase IV and how mutations in these enzymes cause resistance.  These last findings suggest approaches to designing new drugs that display improved activity against resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro57dziddbmLVg90H21EOLACvtfcHk0lgqwEKsXososw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymtrk%253D&md5=38134014d5acd1d5c807095eaba34e3b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi5000564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5000564%26sid%3Dliteratum%253Aachs%26aulast%3DAldred%26aufirst%3DK.%2BJ.%26aulast%3DKerns%26aufirst%3DR.%2BJ.%26aulast%3DOsheroff%26aufirst%3DN.%26atitle%3DMechanism%2520of%2520Quinolone%2520Action%2520and%2520Resistance%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26issue%3D10%26spage%3D1565%26epage%3D1574%26doi%3D10.1021%2Fbi5000564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polykovskiy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhebrak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Lengeling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golovanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatanov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyaev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurbanov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artamonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aladinskiy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadurin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhavoronkov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models</span>. <i>ArXiv</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1811.12823</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1811.12823&author=D.+Polykovskiyauthor=A.+Zhebrakauthor=B.+Sanchez-Lengelingauthor=S.+Golovanovauthor=O.+Tatanovauthor=S.+Belyaevauthor=R.+Kurbanovauthor=A.+Artamonovauthor=V.+Aladinskiyauthor=M.+Veselovauthor=A.+Kadurinauthor=S.+Nikolenkoauthor=A.+Aspuru-Guzikauthor=A.+Zhavoronkov&title=Molecular+Sets+%28MOSES%29%3A+A+Benchmarking+Platform+for+Molecular+Generation+Models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPolykovskiy%26aufirst%3DD.%26aulast%3DZhebrak%26aufirst%3DA.%26aulast%3DSanchez-Lengeling%26aufirst%3DB.%26aulast%3DGolovanov%26aufirst%3DS.%26aulast%3DTatanov%26aufirst%3DO.%26aulast%3DBelyaev%26aufirst%3DS.%26aulast%3DKurbanov%26aufirst%3DR.%26aulast%3DArtamonov%26aufirst%3DA.%26aulast%3DAladinskiy%26aufirst%3DV.%26aulast%3DVeselov%26aufirst%3DM.%26aulast%3DKadurin%26aufirst%3DA.%26aulast%3DNikolenko%26aufirst%3DS.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DZhavoronkov%26aufirst%3DA.%26atitle%3DMolecular%2520Sets%2520%2528MOSES%2529%253A%2520A%2520Benchmarking%2520Platform%2520for%2520Molecular%2520Generation%2520Models%26jtitle%3DArXiv%26date%3D2018%26spage%3D1811.12823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiscato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaucher, A. C.</span></span> <span> </span><span class="NLM_article-title">GuacaMol: Benchmarking Models for de Novo Molecular Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1096</span>– <span class="NLM_lpage">1108</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVWrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1096-1108&issue=3&author=N.+Brownauthor=M.+Fiscatoauthor=M.+H.+S.+Seglerauthor=A.+C.+Vaucher&title=GuacaMol%3A+Benchmarking+Models+for+de+Novo+Molecular+Design&doi=10.1021%2Facs.jcim.8b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">GuacaMol: Benchmarking Models for de Novo Molecular Design</span></div><div class="casAuthors">Brown, Nathan; Fiscato, Marco; Segler, Marwin H. S.; Vaucher, Alain C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1096-1108</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">De novo design seeks to generate mols. with required property profiles by virtual design-make-test cycles.  With the emergence of deep learning and neural generative models in many application areas, models for mol. design based on neural networks appeared recently and show promising results.  However, the new models have not been profiled on consistent tasks, and comparative studies to well-established algorithms have only seldom been performed.  To standardize the assessment of both classical and neural models for de novo mol. design, we propose an evaluation framework, GuacaMol, based on a suite of standardized benchmarks.  The benchmark tasks encompass measuring the fidelity of the models to reproduce the property distribution of the training sets, the ability to generate novel mols., the exploration and exploitation of chem. space, and a variety of single and multiobjective optimization tasks.  The benchmarking open-source Python code and a leaderboard can be found on https://benevolent.ai/guacamol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosXrGIhvFoCrVg90H21EOLACvtfcHk0lgqwEKsXososw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVWrsbY%253D&md5=d3fb616b81a4b146cf77950a1c92e4d1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00839%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DFiscato%26aufirst%3DM.%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DVaucher%26aufirst%3DA.%2BC.%26atitle%3DGuacaMol%253A%2520Benchmarking%2520Models%2520for%2520de%2520Novo%2520Molecular%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D3%26spage%3D1096%26epage%3D1108%26doi%3D10.1021%2Facs.jcim.8b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrchan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, M. P.</span></span> <span> </span><span class="NLM_article-title">Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00512</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00512" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCjsLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=120-131&issue=1&author=M.+H.+S.+Seglerauthor=T.+Kogejauthor=C.+Tyrchanauthor=M.+P.+Waller&title=Generating+Focused+Molecule+Libraries+for+Drug+Discovery+with+Recurrent+Neural+Networks&doi=10.1021%2Facscentsci.7b00512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks</span></div><div class="casAuthors">Segler, Marwin H. S.; Kogej, Thierry; Tyrchan, Christian; Waller, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">120-131</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In de novo drug design, computational strategies are used to generate novel mols. with good affinity to the desired biol. target.  In this work, we show that recurrent neural networks can be trained as generative models for mol. structures, similar to statistical language models in natural language processing.  We demonstrate that the properties of the generated mols. correlate very well with the properties of the mols. used to train the model.  In order to enrich libraries with mols. active toward a given biol. target, we propose to fine-tune the model with small sets of mols., which are known to be active against that target.  Against Staphylococcus aureus, the model reproduced 14% of 6051 hold-out test mols. that medicinal chemists designed, whereas against Plasmodium falciparum (Malaria), it reproduced 28% of 1240 test mols.  When coupled with a scoring function, our model can perform the complete de novo drug design cycle to generate large sets of novel mols. for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwhyDKGtV8eLVg90H21EOLACvtfcHk0lgqqYclAb8MkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCjsLfP&md5=708f40422c7a911c629525ce5b66088b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00512%26sid%3Dliteratum%253Aachs%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DWaller%26aufirst%3DM.%2BP.%26atitle%3DGenerating%2520Focused%2520Molecule%2520Libraries%2520for%2520Drug%2520Discovery%2520with%2520Recurrent%2520Neural%2520Networks%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26issue%3D1%26spage%3D120%26epage%3D131%26doi%3D10.1021%2Facscentsci.7b00512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgqqYclAb8MkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017—Utility and Limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&issue=1&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+Year+Itch%3A+Pan-Assay+Interference+Compounds+%28PAINS%29+in+2017%E2%80%94Utility+and+Limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0lgqqYclAb8MkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520Year%2520Itch%253A%2520Pan-Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520in%25202017%25E2%2580%2594Utility%2520and%2520Limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D1%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">FAF-Drugs3: A Web Server for Compound Property Calculation and Chemical Library Design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W200</span>– <span class="NLM_lpage">W207</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkv353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25883137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W200-W207&issue=W1&author=D.+Lagorceauthor=O.+Sperandioauthor=J.+B.+Baellauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=FAF-Drugs3%3A+A+Web+Server+for+Compound+Property+Calculation+and+Chemical+Library+Design&doi=10.1093%2Fnar%2Fgkv353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">FAF-Drugs3: a web server for compound property calculation and chemical library design</span></div><div class="casAuthors">Lagorce, David; Sperandio, Olivier; Baell, Jonathan B.; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W200-W207</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Drug attrition late in preclin. or clin. development is a serious economic problem in the field of drug discovery.  These problems can be linked, in part, to the quality of the compd. collections used during the hit generation stage and to the selection of compds. undergoing optimization.  Here, we present FAF-Drugs3, a web server that can be used for drug discovery and chem. biol. projects to help in prepg. compd. libraries and to assist decision-making during the hit selection/lead optimization phase.  Since it was first described in 2006, FAF-Drugs has been significantly modified.  The tool now applies an enhanced structure curation procedure, can filter or analyze mols. with user-defined or eight predefined physicochem. filters as well as with several simple ADMET (absorption, distribution, metab., excretion and toxicity) rules.  In addn., compds. can be filtered using an updated list of 154 hand-curated structural alerts while Pan Assay Interference compds. (PAINS) and other, generally unwanted groups are also investigated.  FAF-Drugs3 offers access to user-friendly html result pages and the possibility to download all computed data.  The server requires as input an SDF file of the compds.; it is open to all users and can be accessed without registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxYZ-hCUhlmrVg90H21EOLACvtfcHk0lgqqYclAb8MkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM&md5=5cf8888c489b90f392117ab065986214</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv353%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DFAF-Drugs3%253A%2520A%2520Web%2520Server%2520for%2520Compound%2520Property%2520Calculation%2520and%2520Chemical%2520Library%2520Design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW200%26epage%3DW207%26doi%3D10.1093%2Fnar%2Fgkv353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, P.</span></span> A Script to Run Structural Alerts Using the RDKit and ChEMBL. <span class="NLM_year">2018</span>. <a href="https://github.com/PatWalters/rd_filters" class="extLink">https://github.com/PatWalters/rd_filters</a> (accessed Apr 14, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Walters%2C+P.+A+Script+to+Run+Structural+Alerts+Using+the+RDKit+and+ChEMBL.+2018.+https%3A%2F%2Fgithub.com%2FPatWalters%2Frd_filters+%28accessed+Apr+14%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DP.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Using Fragment-Based Approaches to Discover New Antibiotics</span>. <i>Slas Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1177/2472555218773034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1177%2F2472555218773034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29923463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=495-510&issue=6&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Using+Fragment-Based+Approaches+to+Discover+New+Antibiotics&doi=10.1177%2F2472555218773034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Using Fragment-Based Approaches to Discover New Antibiotics</span></div><div class="casAuthors">Lamoree, Bas; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">495-510</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Fragment-based lead discovery has emerged over the past two decades as a successful approach to generate novel lead candidates in drug discovery programs.  The two main advantages over conventional high-throughput screening (HTS) are more efficient sampling of chem. space and tighter control over the physicochem. properties of the lead candidates.  Antibiotics are a class of drugs with particularly strict property requirements for efficacy and safety.  The development of novel antibiotics has slowed down so much that resistance has now evolved against every available antibiotic drug.  Here we give an overview of fragment-based approaches in screening and lead discovery projects for new antibiotics.  We discuss several successful hit-to-lead development examples.  Finally, we highlight the current challenges and opportunities for fragment-based lead discovery toward new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUbNlZsC5x6bVg90H21EOLACvtfcHk0lgaqq3niaJQjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrjK&md5=fccb6116d20d6b6091c78dba82c70819</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F2472555218773034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218773034%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DUsing%2520Fragment-Based%2520Approaches%2520to%2520Discover%2520New%2520Antibiotics%26jtitle%3DSlas%2520Discovery%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D495%26epage%3D510%26doi%3D10.1177%2F2472555218773034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gable, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Current and Potential Treatments for Ubiquitous but Neglected Herpesvirus Infections</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11412</span>, <span class="refDoi"> DOI: 10.1021/cr500255e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500255e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=11382-11412&issue=22&author=J.+E.+Gableauthor=T.+M.+Ackerauthor=C.+S.+Craik&title=Current+and+Potential+Treatments+for+Ubiquitous+but+Neglected+Herpesvirus+Infections&doi=10.1021%2Fcr500255e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Current and Potential Treatments for Ubiquitous but Neglected Herpesvirus Infections</span></div><div class="casAuthors">Gable, Jonathan E.; Acker, Timothy M.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">11382-11412</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxtq3MNOVUBLVg90H21EOLACvtfcHk0lgaqq3niaJQjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLrN&md5=2663dc411d94ff3ef539a738e2e4eb17</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fcr500255e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500255e%26sid%3Dliteratum%253Aachs%26aulast%3DGable%26aufirst%3DJ.%2BE.%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DCurrent%2520and%2520Potential%2520Treatments%2520for%2520Ubiquitous%2520but%2520Neglected%2520Herpesvirus%2520Infections%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26issue%3D22%26spage%3D11382%26epage%3D11412%26doi%3D10.1021%2Fcr500255e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glawar, A. F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkinson, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butters, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleet, G. W. J.</span></span> <span> </span><span class="NLM_article-title">3-Hydroxyazetidine Carboxylic Acids: Non-Proteinogenic Amino Acids for Medicinal Chemists</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fcmdc.201200541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=23468173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVOhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=658-666&issue=4&author=A.+F.+G.+Glawarauthor=S.+F.+Jenkinsonauthor=A.+L.+Thompsonauthor=S.+Nakagawaauthor=A.+Katoauthor=T.+D.+Buttersauthor=G.+W.+J.+Fleet&title=3-Hydroxyazetidine+Carboxylic+Acids%3A+Non-Proteinogenic+Amino+Acids+for+Medicinal+Chemists&doi=10.1002%2Fcmdc.201200541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">3-Hydroxyazetidine Carboxylic Acids: Non-Proteinogenic Amino Acids for Medicinal Chemists</span></div><div class="casAuthors">Glawar, Andreas F. G.; Jenkinson, Sarah F.; Thompson, Amber L.; Nakagawa, Shinpei; Kato, Atsushi; Butters, Terry D.; Fleet, George W. J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-666</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The formation from D-glucose of both enantiomers of 2,4-dideoxy-2,4-iminoribonic acid is the first chem. synthesis of unprotected 3-hydroxyazetidine carboxylic acids.  The long-term stability of 3-hydroxyazetidine amides is established at acidic and neutral pH and implies their value as non-proteinogenic amino acid components of peptides, providing medicinal chemists with a new class of peptide isosteres.  The structure of N,3-O-dibenzyl-2,4-dideoxy-2,4-imino-D-ribonic acid was established by x-ray crystallog. anal.  An N-methylazetidine amide deriv. is a specific inhibitor of β-hexosaminidases at the micromolar level, and is only the second example of potent inhibition of any glycosidase by an amide of a sugar amino acid related to an iminosugar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6WH8Z235zDbVg90H21EOLACvtfcHk0lgaqq3niaJQjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVOhsrg%253D&md5=e0e792ca1419766d93f9f96270e7a9f4</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200541%26sid%3Dliteratum%253Aachs%26aulast%3DGlawar%26aufirst%3DA.%2BF.%2BG.%26aulast%3DJenkinson%26aufirst%3DS.%2BF.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DNakagawa%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DButters%26aufirst%3DT.%2BD.%26aulast%3DFleet%26aufirst%3DG.%2BW.%2BJ.%26atitle%3D3-Hydroxyazetidine%2520Carboxylic%2520Acids%253A%2520Non-Proteinogenic%2520Amino%2520Acids%2520for%2520Medicinal%2520Chemists%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26issue%3D4%26spage%3D658%26epage%3D666%26doi%3D10.1002%2Fcmdc.201200541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span> (<span class="NLM_issue">7654</span>),  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&issue=7654&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lh1U0NhXJKT8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26issue%3D7654%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takrouri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahalingam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucchi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tear, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuning, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span> <span> </span><span class="NLM_article-title">Compound Design Guidelines for Evading the Efflux and Permeation Barriers of Escherichia Coli with the Oxazolidinone Class of Antibacterials: Test Case for a General Approach to Improving Whole Cell Gram-Negative Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">5310</span>– <span class="NLM_lpage">5321</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bmcl.2017.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29102393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyht7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5310-5321&issue=23&author=A.+Spauldingauthor=K.+Takrouriauthor=P.+Mahalingamauthor=D.+C.+Clearyauthor=H.+D.+Cooperauthor=P.+Zucchiauthor=W.+Tearauthor=B.+Kolevaauthor=P.+J.+Beuningauthor=E.+B.+Hirschauthor=J.+B.+Aggen&title=Compound+Design+Guidelines+for+Evading+the+Efflux+and+Permeation+Barriers+of+Escherichia+Coli+with+the+Oxazolidinone+Class+of+Antibacterials%3A+Test+Case+for+a+General+Approach+to+Improving+Whole+Cell+Gram-Negative+Activity&doi=10.1016%2Fj.bmcl.2017.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Compound design guidelines for evading the efflux and permeation barriers of Escherichia coli with the oxazolidinone class of antibacterials: Test case for a general approach to improving whole cell Gram-negative activity</span></div><div class="casAuthors">Spaulding, Andrew; Takrouri, Khuloud; Mahalingam, Pornachandran; Cleary, Dillon C.; Cooper, Harold D.; Zucchi, Paola; Tear, Westley; Koleva, Bilyana; Beuning, Penny J.; Hirsch, Elizabeth B.; Aggen, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5310-5321</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Previously the authors reported the results from an effort to improve Gram-neg. antibacterial activity in the oxazolidinone class of antibiotics via a systematic medicinal chem. campaign focused entirely on C-ring modifications.  In that series the authors set about testing if the efflux and permeation barriers intrinsic to the outer membrane of Escherichia coli could be rationally overcome by designing analogs to reside in specific property limits assocd. with Gram-neg. activity: (i) low MW (<400), (ii) high polarity (clogD7.4 <1), and (iii) zwitterionic character at pH 7.4.  Indeed, the authors obsd. that only analogs residing within these limits were able to overcome these barriers.  Herein the authors report the results from a parallel effort where the authors explored structural changes throughout all three rings in the scaffold for the same purpose.  Compds. were tested against a diagnostic MIC panel of Escherichia coli and Staphylococcus aureus strains to det. the impact of combining structural modifications in overcoming the OM barriers and in bridging the potency gap between the species.  The results demonstrated that distributing the charge-carrying moieties across two rings was also beneficial for avoidance of the outer membrane barriers.  Importantly, anal. of the structure-permeation relationship (SPR) obtained from this and the prior study indicated that in addn. to MW, polarity, and zwitterionic character, having ≤4 rotatable bonds is also assocd. with evasion of the OM barriers.  These combined results provide the medicinal chemist with a framework and strategy for overcoming the OM barriers in GNB in antibacterial drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxD-MAU_7WfbVg90H21EOLACvtfcHk0lh1U0NhXJKT8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyht7%252FM&md5=fc70a1d802e4ec625f8aa21b0ab02f02</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DSpaulding%26aufirst%3DA.%26aulast%3DTakrouri%26aufirst%3DK.%26aulast%3DMahalingam%26aufirst%3DP.%26aulast%3DCleary%26aufirst%3DD.%2BC.%26aulast%3DCooper%26aufirst%3DH.%2BD.%26aulast%3DZucchi%26aufirst%3DP.%26aulast%3DTear%26aufirst%3DW.%26aulast%3DKoleva%26aufirst%3DB.%26aulast%3DBeuning%26aufirst%3DP.%2BJ.%26aulast%3DHirsch%26aufirst%3DE.%2BB.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26atitle%3DCompound%2520Design%2520Guidelines%2520for%2520Evading%2520the%2520Efflux%2520and%2520Permeation%2520Barriers%2520of%2520Escherichia%2520Coli%2520with%2520the%2520Oxazolidinone%2520Class%2520of%2520Antibacterials%253A%2520Test%2520Case%2520for%2520a%2520General%2520Approach%2520to%2520Improving%2520Whole%2520Cell%2520Gram-Negative%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26issue%3D23%26spage%3D5310%26epage%3D5321%26doi%3D10.1016%2Fj.bmcl.2017.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Converting Gram-Positive-Only Compounds into Broad-Spectrum Antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&issue=1&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+Challenge+of+Converting+Gram-Positive-Only+Compounds+into+Broad-Spectrum+Antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0lh1U0NhXJKT8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520Challenge%2520of%2520Converting%2520Gram-Positive-Only%2520Compounds%2520into%2520Broad-Spectrum%2520Antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26issue%3D1%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acosta-Gutiérrez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathania, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodrenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterhalter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavenger, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naismith, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Berg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, M.</span></span> <span> </span><span class="NLM_article-title">Getting Drugs into Gram-Negative Bacteria: Rational Rules for Permeation through General Porins</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=1487-1498&issue=10&author=S.+Acosta-Guti%C3%A9rrezauthor=L.+Ferraraauthor=M.+Pathaniaauthor=M.+Masiauthor=J.+Wangauthor=I.+Bodrenkoauthor=M.+Zahnauthor=M.+Winterhalterauthor=R.+A.+Stavengerauthor=J.-M.+Pag%C3%A8sauthor=J.+H.+Naismithauthor=B.+van+den%0ABergauthor=M.+G.+P.+Pageauthor=M.+Ceccarelli&title=Getting+Drugs+into+Gram-Negative+Bacteria%3A+Rational+Rules+for+Permeation+through+General+Porins&doi=10.1021%2Facsinfecdis.8b00108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Getting Drugs into Gram-Negative Bacteria: Rational Rules for Permeation through General Porins</span></div><div class="casAuthors">Acosta-Gutierrez, Silvia; Ferrara, Luana; Pathania, Monisha; Masi, Muriel; Wang, Jiajun; Bodrenko, Igor; Zahn, Michael; Winterhalter, Mathias; Stavenger, Robert A.; Pages, Jean-Marie; Naismith, James H.; van den Berg, Bert; Page, Malcolm G. P.; Ceccarelli, Matteo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1487-1498</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small, hydrophilic mols., including most important antibiotics in clin. use, cross the Gram-neg. outer membrane through the water-filled channels provided by porins.  We have detd. the X-ray crystal structures of the principal general porins from three species of Enterobacteriaceae, namely Enterobacter aerogenes, Enterobacter cloacae, and Klebsiella pneumoniae, and detd. their antibiotic permeabilities as well as those of the orthologues from Escherichia coli.  Starting from the structure of the porins and mols., we propose a phys. mechanism underlying transport and condense it in a computationally efficient scoring function.  The scoring function shows good agreement with in vitro penetration data and will enable the screening of virtual databases to identify mols. with optimal permeability through porins and help to guide the optimization of antibiotics with poor permeation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppq33eJ0gJ1bVg90H21EOLACvtfcHk0lg6tkIyOZrEEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrfM&md5=d66864f6de3193581e4663ef7c436baf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00108%26sid%3Dliteratum%253Aachs%26aulast%3DAcosta-Guti%25C3%25A9rrez%26aufirst%3DS.%26aulast%3DFerrara%26aufirst%3DL.%26aulast%3DPathania%26aufirst%3DM.%26aulast%3DMasi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBodrenko%26aufirst%3DI.%26aulast%3DZahn%26aufirst%3DM.%26aulast%3DWinterhalter%26aufirst%3DM.%26aulast%3DStavenger%26aufirst%3DR.%2BA.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26aulast%3DNaismith%26aufirst%3DJ.%2BH.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DB.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DCeccarelli%26aufirst%3DM.%26atitle%3DGetting%2520Drugs%2520into%2520Gram-Negative%2520Bacteria%253A%2520Rational%2520Rules%2520for%2520Permeation%2520through%2520General%2520Porins%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26issue%3D10%26spage%3D1487%26epage%3D1498%26doi%3D10.1021%2Facsinfecdis.8b00108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Firth, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span> <span> </span><span class="NLM_article-title">Plane of Best Fit: A Novel Method to Characterize the Three-Dimensionality of Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2516</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/ci300293f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300293f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2516-2525&issue=10&author=N.+C.+Firthauthor=N.+Brownauthor=J.+Blagg&title=Plane+of+Best+Fit%3A+A+Novel+Method+to+Characterize+the+Three-Dimensionality+of+Molecules&doi=10.1021%2Fci300293f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Plane of Best Fit: A Novel Method to Characterize the Three-Dimensionality of Molecules</span></div><div class="casAuthors">Firth, Nicholas C.; Brown, Nathan; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2516-2525</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a computational method, plane of best fit (PBF), to quantify and characterize the 3D character of mols.  This method is rapid and amenable to anal. of large diverse data sets.  We compare PBF with alternative literature methods used to assess 3D character and apply the method to diverse data sets of fragment-like, drug-like, and natural product compd. libraries.  We show that exemplar fragment libraries underexploit the potential of 3D character in fragment-like chem. space and that drug-like mols. in the libraries examd. are predominantly 2D in character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszhZRQVuP0LVg90H21EOLACvtfcHk0lg6tkIyOZrEEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rt7zP&md5=747bb9c5fb38eb34af4e1047f83633e8</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci300293f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300293f%26sid%3Dliteratum%253Aachs%26aulast%3DFirth%26aufirst%3DN.%2BC.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3DPlane%2520of%2520Best%2520Fit%253A%2520A%2520Novel%2520Method%2520to%2520Characterize%2520the%2520Three-Dimensionality%2520of%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D10%26spage%3D2516%26epage%3D2525%26doi%3D10.1021%2Fci300293f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span> <span> </span><span class="NLM_article-title">Advancing the Drug Discovery and Development Process</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">35</span>),  <span class="NLM_fpage">9128</span>– <span class="NLM_lpage">9140</span>, <span class="refDoi"> DOI: 10.1002/anie.201404761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fanie.201404761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGmu77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=9128-9140&issue=35&author=K.+C.+Nicolaou&title=Advancing+the+Drug+Discovery+and+Development+Process&doi=10.1002%2Fanie.201404761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing the Drug Discovery and Development Process</span></div><div class="casAuthors">Nicolaou, K. C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9128-9140</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current state of affairs in the drug discovery and development process is briefly summarized and then ways to take advantage of the ever-increasing fundamental knowledge and tech. knowhow in chem. and biol. and related disciplines are discussed.  The primary motivation of this Essay is to celebrate the great achievements of chem., biol., and medicine and to inform and inspire students and academics to enter the field of drug discovery and development while, at the same time, continue to advance the fundamentals of their disciplines.  It is also meant to encourage and catalyze multidisciplinary partnerships between academia and industry as scientists attempt to merge their often complementary interests and expertise to achieve new improvements and breakthroughs in their resp. fields, and the common goal of applying them to the discovery and invention of new and better medicines, esp. in areas of unmet needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFwL5Am5_V7Vg90H21EOLACvtfcHk0lg6tkIyOZrEEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGmu77O&md5=1348675f7bdf8e35a0ae11881ba11750</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fanie.201404761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201404761%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26atitle%3DAdvancing%2520the%2520Drug%2520Discovery%2520and%2520Development%2520Process%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26issue%3D35%26spage%3D9128%26epage%3D9140%26doi%3D10.1002%2Fanie.201404761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span>OSIRIS-REx.
Wikipedia, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OSIRIS-REx.%0AWikipedia%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&issue=11&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+Progress+in+the+Development+of+HIV-1+Protease+Inhibitors+for+the+Treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0lg6tkIyOZrEEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520HIV-1%2520Protease%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenkel, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewi, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shatkin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">High-Resolution Structures of HIV-1 Reverse transcriptase/TMC278 Complexes: Strategic Flexibility Explains Potency against Resistance Mutations</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1466</span>– <span class="NLM_lpage">1471</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711209105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0711209105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18230722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1466-1471&issue=5&author=K.+Dasauthor=J.+D.+Baumanauthor=A.+D.+Clarkauthor=Y.+V.+Frenkelauthor=P.+J.+Lewiauthor=A.+J.+Shatkinauthor=S.+H.+Hughesauthor=E.+Arnold&title=High-Resolution+Structures+of+HIV-1+Reverse+transcriptase%2FTMC278+Complexes%3A+Strategic+Flexibility+Explains+Potency+against+Resistance+Mutations&doi=10.1073%2Fpnas.0711209105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: strategic flexibility explains potency against resistance mutations</span></div><div class="casAuthors">Das, Kalyan; Bauman, Joseph D.; Clark, Arthur D.; Frenkel, Yulia V.; Lewi, Paul J.; Shatkin, Aaron J.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1466-1471</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TMC278 is a diarylpyrimidine (DAPY) nonnucleoside reverse transcriptase inhibitor (NNRTI) that is highly effective in treating wild-type and drug-resistant HIV-1 infections in clin. trials at relatively low doses (∼ 25-75 mg/day).  We have detd. the structure of wild-type HIV-1 RT complexed with TMC278 at 1.8 Å resoln., using an RT crystal form engineered by systematic RT mutagenesis.  This high-resoln. structure reveals that the cyanovinyl group of TMC278 is positioned in a hydrophobic tunnel connecting the NNRTI-binding pocket to the nucleic acid-binding cleft.  The crystal structures of TMC278 in complexes with the double mutant K103N/Y181C (2.1 Å) and L100I/K103N HIV-1 RTs (2.9 Å) demonstrated that TMC278 adapts to bind mutant RTs.  In the K103N/Y181C RT/TMC278 structure, loss of the arom. ring interaction caused by the Y181C mutation is counterbalanced by interactions between the cyanovinyl group of TMC278 and the arom. side chain of Y183, which is facilitated by an ∼ 1.5 Å shift of the conserved Y183MDD motif.  In the L1001/K103N RT/ TMC278 structure, the binding mode of TMC278 is significantly altered so that the drug conforms to changes in the binding pocket primarily caused by the L1001 mutation.  The flexible binding pocket acts as a mol. "shrink wrap" that makes a shape complementary to the optimized TMC278 in wild-type and drug-resistant forms of HIV-1 RT.  The crystal structures provide a better understanding of how the flexibility of an inhibitor can compensate for drug-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRbleFGfFk7Vg90H21EOLACvtfcHk0lhqbQI--F4KHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D&md5=5f2ea5e4d88e9822ceecee636e1e4bd1</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711209105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711209105%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DFrenkel%26aufirst%3DY.%2BV.%26aulast%3DLewi%26aufirst%3DP.%2BJ.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHigh-Resolution%2520Structures%2520of%2520HIV-1%2520Reverse%2520transcriptase%252FTMC278%2520Complexes%253A%2520Strategic%2520Flexibility%2520Explains%2520Potency%2520against%2520Resistance%2520Mutations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D5%26spage%3D1466%26epage%3D1471%26doi%3D10.1073%2Fpnas.0711209105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansdon, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendza, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of HIV-1 Reverse Transcriptase with Etravirine (TMC125) and Rilpivirine (TMC278): Implications for Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4295</span>– <span class="NLM_lpage">4299</span>, <span class="refDoi"> DOI: 10.1021/jm1002233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1002233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFeqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4295-4299&issue=10&author=E.+B.+Lansdonauthor=K.+M.+Brendzaauthor=M.+Hungauthor=R.+Wangauthor=S.+Mukundauthor=D.+Jinauthor=G.+Birkusauthor=N.+Kuttyauthor=X.+Liu&title=Crystal+Structures+of+HIV-1+Reverse+Transcriptase+with+Etravirine+%28TMC125%29+and+Rilpivirine+%28TMC278%29%3A+Implications+for+Drug+Design&doi=10.1021%2Fjm1002233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of HIV-1 Reverse Transcriptase with Etravirine (TMC125) and Rilpivirine (TMC278): Implications for Drug Design</span></div><div class="casAuthors">Lansdon, Eric B.; Brendza, Katherine M.; Hung, Magdeleine; Wang, Ruth; Mukund, Susmith; Jin, Debi; Birkus, Gabriel; Kutty, Nilima; Liu, Xiaohong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4295-4299</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diarylpyrimidine (DAPY) non-nucleoside reverse transcriptase inhibitors (NNRTIs) have inherent flexibility, helping to maintain activity against a wide range of resistance mutations.  Crystal structures were detd. with wild-type and K103N HIV-1 reverse transcriptase with etravirine (TMC125) and rilpivirine (TMC278).  These structures reveal a similar binding mode for TMC125 and TMC278, whether bound to wild-type or K103N RT.  Comparison to previously published structures reveals differences in binding modes for TMC125 and differences in protein conformation for TMC278.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNZlZQF054rVg90H21EOLACvtfcHk0lhqbQI--F4KHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFeqsLs%253D&md5=d99b8d74a488eb94da35246c6f141240</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm1002233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1002233%26sid%3Dliteratum%253Aachs%26aulast%3DLansdon%26aufirst%3DE.%2BB.%26aulast%3DBrendza%26aufirst%3DK.%2BM.%26aulast%3DHung%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DMukund%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DBirkus%26aufirst%3DG.%26aulast%3DKutty%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DCrystal%2520Structures%2520of%2520HIV-1%2520Reverse%2520Transcriptase%2520with%2520Etravirine%2520%2528TMC125%2529%2520and%2520Rilpivirine%2520%2528TMC278%2529%253A%2520Implications%2520for%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D10%26spage%3D4295%26epage%3D4299%26doi%3D10.1021%2Fjm1002233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fader, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenfant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilodeau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brochu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulombe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garneau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordingley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bös, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupart, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faucher, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoakim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsantrizos, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of BI 224436, a Noncatalytic Site Integrase Inhibitor (NCINI) of HIV-1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/ml500002n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500002n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=422-427&issue=4&author=L.+D.+Faderauthor=E.+Malenfantauthor=M.+Parisienauthor=R.+Carsonauthor=F.+Bilodeauauthor=S.+Landryauthor=M.+Pesantauthor=C.+Brochuauthor=S.+Morinauthor=C.+Chabotauthor=T.+Halmosauthor=Y.+Bousquetauthor=M.+D.+Baileyauthor=S.+H.+Kawaiauthor=R.+Coulombeauthor=S.+LaPlanteauthor=A.+Jakalianauthor=P.+K.+Bhardwajauthor=D.+Wernicauthor=P.+Schroederauthor=M.+Amadauthor=P.+Edwardsauthor=M.+Garneauauthor=J.+Duanauthor=M.+Cordingleyauthor=R.+Bethellauthor=S.+W.+Masonauthor=M.+B%C3%B6sauthor=P.+Bonneauauthor=M.-A.+Poupartauthor=A.-M.+Faucherauthor=B.+Simoneauauthor=C.+Fenwickauthor=C.+Yoakimauthor=Y.+Tsantrizos&title=Discovery+of+BI+224436%2C+a+Noncatalytic+Site+Integrase+Inhibitor+%28NCINI%29+of+HIV-1&doi=10.1021%2Fml500002n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of BI 224436, a Noncatalytic Site Integrase Inhibitor (NCINI) of HIV-1</span></div><div class="casAuthors">Fader, Lee D.; Malenfant, Eric; Parisien, Mathieu; Carson, Rebekah; Bilodeau, Francois; Landry, Serge; Pesant, Marc; Brochu, Christian; Morin, Sebastien; Chabot, Catherine; Halmos, Ted; Bousquet, Yves; Bailey, Murray D.; Kawai, Stephen H.; Coulombe, Rene; LaPlante, Steven; Jakalian, Araz; Bhardwaj, Punit K.; Wernic, Dominik; Schroeder, Patricia; Amad, Ma'an; Edwards, Paul; Garneau, Michel; Duan, Jianmin; Cordingley, Michael; Bethell, Richard; Mason, Stephen W.; Bos, Michael; Bonneau, Pierre; Poupart, Marc-Andre; Faucher, Anne-Marie; Simoneau, Bruno; Fenwick, Craig; Yoakim, Christiane; Tsantrizos, Youla</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">422-427</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assay recapitulating the 3' processing activity of HIV-1 integrase (IN) was used to screen the Boehringer Ingelheim compd. collection.  Hit-to-lead and lead optimization beginning with compd. I established the importance of the C3 and C4 substituent to antiviral potency against viruses with different aa124/aa125 variants of IN.  The importance of the C7 position on the serum shifted potency was established.  Introduction of a quinoline substituent at the C4 position provided a balance of potency and metabolic stability.  Combination of these findings ultimately led to the discovery of compd. II (BI 224436), the first NCINI to advance into a phase Ia clin. trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrE57FQpskKLVg90H21EOLACvtfcHk0ljiHLTVNvXtBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitb8%253D&md5=6df7ddab6dadf569d00d4314a773532c</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fml500002n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500002n%26sid%3Dliteratum%253Aachs%26aulast%3DFader%26aufirst%3DL.%2BD.%26aulast%3DMalenfant%26aufirst%3DE.%26aulast%3DParisien%26aufirst%3DM.%26aulast%3DCarson%26aufirst%3DR.%26aulast%3DBilodeau%26aufirst%3DF.%26aulast%3DLandry%26aufirst%3DS.%26aulast%3DPesant%26aufirst%3DM.%26aulast%3DBrochu%26aufirst%3DC.%26aulast%3DMorin%26aufirst%3DS.%26aulast%3DChabot%26aufirst%3DC.%26aulast%3DHalmos%26aufirst%3DT.%26aulast%3DBousquet%26aufirst%3DY.%26aulast%3DBailey%26aufirst%3DM.%2BD.%26aulast%3DKawai%26aufirst%3DS.%2BH.%26aulast%3DCoulombe%26aufirst%3DR.%26aulast%3DLaPlante%26aufirst%3DS.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBhardwaj%26aufirst%3DP.%2BK.%26aulast%3DWernic%26aufirst%3DD.%26aulast%3DSchroeder%26aufirst%3DP.%26aulast%3DAmad%26aufirst%3DM.%26aulast%3DEdwards%26aufirst%3DP.%26aulast%3DGarneau%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DCordingley%26aufirst%3DM.%26aulast%3DBethell%26aufirst%3DR.%26aulast%3DMason%26aufirst%3DS.%2BW.%26aulast%3DB%25C3%25B6s%26aufirst%3DM.%26aulast%3DBonneau%26aufirst%3DP.%26aulast%3DPoupart%26aufirst%3DM.-A.%26aulast%3DFaucher%26aufirst%3DA.-M.%26aulast%3DSimoneau%26aufirst%3DB.%26aulast%3DFenwick%26aufirst%3DC.%26aulast%3DYoakim%26aufirst%3DC.%26aulast%3DTsantrizos%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520BI%2520224436%252C%2520a%2520Noncatalytic%2520Site%2520Integrase%2520Inhibitor%2520%2528NCINI%2529%2520of%2520HIV-1%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D4%26spage%3D422%26epage%3D427%26doi%3D10.1021%2Fml500002n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valkov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherepanov, P.</span></span> <span> </span><span class="NLM_article-title">Retroviral Intasome Assembly and Inhibition of DNA Strand Transfer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span> (<span class="NLM_issue">7286</span>),  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1038/nature08784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature08784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20118915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=232-236&issue=7286&author=S.+Hareauthor=S.+S.+Guptaauthor=E.+Valkovauthor=A.+Engelmanauthor=P.+Cherepanov&title=Retroviral+Intasome+Assembly+and+Inhibition+of+DNA+Strand+Transfer&doi=10.1038%2Fnature08784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral intasome assembly and inhibition of DNA strand transfer</span></div><div class="casAuthors">Hare, Stephen; Gupta, Saumya Shree; Valkov, Eugene; Engelman, Alan; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7286</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Integrase is an essential retroviral enzyme that binds both termini of linear viral DNA and inserts them into a host cell chromosome.  The structure of full-length retroviral integrase, either sep. or in complex with DNA, has been lacking.  Furthermore, although clin. useful inhibitors of HIV integrase have been developed, their mechanism of action remains speculative.  Here the authors present a crystal structure of full-length integrase from the prototype foamy virus in complex with its cognate DNA.  The structure shows the organization of the retroviral intasome comprising an integrase tetramer tightly assocd. with a pair of viral DNA ends.  All three canonical integrase structural domains are involved in extensive protein-DNA and protein-protein interactions.  The binding of strand-transfer inhibitors displaces the reactive viral DNA end from the active site, disarming the viral nucleoprotein complex.  These findings define the structural basis of retroviral DNA integration, and will allow modeling of the HIV-1 intasome to aid in the development of antiretroviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdEDTFdvZMbVg90H21EOLACvtfcHk0ljiHLTVNvXtBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D&md5=9551e74b6bc33494855651c3b157d753</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature08784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08784%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DS.%2BS.%26aulast%3DValkov%26aufirst%3DE.%26aulast%3DEngelman%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520Intasome%2520Assembly%2520and%2520Inhibition%2520of%2520DNA%2520Strand%2520Transfer%26jtitle%3DNature%26date%3D2010%26volume%3D464%26issue%3D7286%26spage%3D232%26epage%3D236%26doi%3D10.1038%2Fnature08784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Site Structure Shapes Allosteric Signal Transduction and the Evolution of Allostery in Protein Complexes</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1093/molbev/msz093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsz093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=31004156" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Ligand+Binding+Site+Structure+Shapes+Allosteric+Signal+Transduction+and+the+Evolution+of+Allostery+in+Protein+Complexes&doi=10.1093%2Fmolbev%2Fmsz093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsz093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsz093%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520Binding%2520Site%2520Structure%2520Shapes%2520Allosteric%2520Signal%2520Transduction%2520and%2520the%2520Evolution%2520of%2520Allostery%2520in%2520Protein%2520Complexes%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2019%26doi%3D10.1093%2Fmolbev%2Fmsz093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metrangolo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilati, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priimagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terraneo, G.</span></span> <span> </span><span class="NLM_article-title">The Halogen Bond</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2601</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&issue=4&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+Halogen+Bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0ljiHLTVNvXtBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520Halogen%2520Bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26issue%3D4%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Halogen Bonding for Rational Drug Design and New Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">383</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.678829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1517%2F17460441.2012.678829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22462734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlCqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=375-383&issue=5&author=Y.+Luauthor=Y.+Liuauthor=Z.+Xuauthor=H.+Liauthor=H.+Liuauthor=W.+Zhu&title=Halogen+Bonding+for+Rational+Drug+Design+and+New+Drug+Discovery&doi=10.1517%2F17460441.2012.678829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonding for rational drug design and new drug discovery</span></div><div class="casAuthors">Lu, Yunxiang; Liu, Yingtao; Xu, Zhijian; Li, Haiying; Liu, Honglai; Zhu, Weiliang</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">375-383</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: A large no. of drugs and drug candidates in clin. development contain halogen substituents.  For a long time, only the steric and lipophilic contributions of halogens were considered when trying to exploit their effects on ligand binding.  However, the ability of halogens to form stabilizing interactions, such as halogen bonding, hydrogen bonding and multipolar interactions, in biomol. systems was revealed recently.  Halogen bonding, the non-covalent interaction in which covalently bound halogens interact with Lewis bases, has now been utilized in the context of rational drug design.Areas covered: The purpose of this review is to show how halogen bonding could be used in drug design, and in particular, to stimulate researchers to apply halogen bonding in lead optimization.  This review article covers the recent advances relevant to halogen bonding in drug discovery and biol. design over the past decade, including database survey of this interaction in protein-ligand complexes, mol. mech. investigations of halogen bonding in drug discovery and applications of this interaction in the development of halogenated ligands as inhibitors and drugs for protein kinases, serine protease factor Xa, HIV reverse transcriptase and so on.Expert opinion: Halogen bonding should intentionally be used as a powerful tool, comparable with hydrogen bonding, to enhance the binding affinity and also influence the binding selectivity.  Rational design of new and potent inhibitors against therapeutic targets through halogen bonding continues to be an exciting area, which will be further elucidated with the combination of various exptl. techniques and theor. calcns. in the forthcoming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO8AUipLTwmbVg90H21EOLACvtfcHk0lh9_5D8LxIA3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlCqsrc%253D&md5=f263970ae5ef3706b0148f0c675b70aa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.678829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.678829%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DHalogen%2520Bonding%2520for%2520Rational%2520Drug%2520Design%2520and%2520New%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26issue%3D5%26spage%3D375%26epage%3D383%26doi%3D10.1517%2F17460441.2012.678829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auffinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hays, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P. S.</span></span> <span> </span><span class="NLM_article-title">Halogen Bonds in Biological Molecules</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">16789</span>– <span class="NLM_lpage">16794</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407607101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0407607101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=15557000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16789-16794&issue=48&author=P.+Auffingerauthor=F.+A.+Haysauthor=E.+Westhofauthor=P.+S.+Ho&title=Halogen+Bonds+in+Biological+Molecules&doi=10.1073%2Fpnas.0407607101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonds in biological molecules</span></div><div class="casAuthors">Auffinger, Pascal; Hays, Franklin A.; Westhof, Eric; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16789-16794</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Short oxygen-halogen interactions have been known in org. chem. since the 1950s and recently have been exploited in the design of supramol. assemblies.  The present survey of protein and nucleic acid structures reveals similar halogen bonds as potentially stabilizing inter- and intramol. interactions that can affect ligand binding and mol. folding.  A halogen bond in biomols. can be defined as a short C-X···O-Y interaction (C-X is a carbon-bonded chlorine, bromine, or iodine, and O-Y is a carbonyl, hydroxyl, charged carboxylate, or phosphate group), where the X···O distance is less than or equal to the sums of the resp. van der Waals radii (3.27 Å for Cl···O, 3.37 Å for Br···O, and 3.50 Å for I···O) and can conform to the geometry seen in small mols., with the C-X···O angle ≈165° (consistent with a strong directional polarization of the halogen) and the X···O-Y angle ≈120°.  Alternative geometries can be imposed by the more complex environment found in biomols., depending on which of the two types of donor systems are involved in the interaction: (i) the lone pair electrons of oxygen (and, to a lesser extent, nitrogen and sulfur) atoms or (ii) the delocalized π-electrons of peptide bonds or carboxylate or amide groups.  Thus, the specific geometry and diversity of the interacting partners of halogen bonds offer new and versatile tools for the design of ligands as drugs and materials in nanotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPMdZU91iT77Vg90H21EOLACvtfcHk0lh9_5D8LxIA3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P&md5=dfcce25d4e336c66294f6ab8ce412154</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407607101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407607101%26sid%3Dliteratum%253Aachs%26aulast%3DAuffinger%26aufirst%3DP.%26aulast%3DHays%26aufirst%3DF.%2BA.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DHo%26aufirst%3DP.%2BS.%26atitle%3DHalogen%2520Bonds%2520in%2520Biological%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D48%26spage%3D16789%26epage%3D16794%26doi%3D10.1073%2Fpnas.0407607101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjahad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oumouch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemont, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coupa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csoka, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andries, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grierson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures for HIV-1 Reverse Transcriptase in Complexes with Three Pyridinone Derivatives: A New Class of Non-Nucleoside Inhibitors Effective against a Broad Range of Drug-Resistant Strains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">7582</span>– <span class="NLM_lpage">7591</span>, <span class="refDoi"> DOI: 10.1021/jm0500323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0500323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7582-7591&issue=24&author=D.+M.+Himmelauthor=K.+Dasauthor=A.+D.+Clarkauthor=S.+H.+Hughesauthor=A.+Benjahadauthor=S.+Oumouchauthor=J.+Guillemontauthor=S.+Coupaauthor=A.+Ponceletauthor=I.+Csokaauthor=C.+Meyerauthor=K.+Andriesauthor=C.+H.+Nguyenauthor=D.+S.+Griersonauthor=E.+Arnold&title=Crystal+Structures+for+HIV-1+Reverse+Transcriptase+in+Complexes+with+Three+Pyridinone+Derivatives%3A+A+New+Class+of+Non-Nucleoside+Inhibitors+Effective+against+a+Broad+Range+of+Drug-Resistant+Strains&doi=10.1021%2Fjm0500323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures for HIV-1 Reverse Transcriptase in Complexes with Three Pyridinone Derivatives: A New Class of Non-Nucleoside Inhibitors Effective against a Broad Range of Drug-Resistant Strains</span></div><div class="casAuthors">Himmel, Daniel M.; Das, Kalyan; Clark, Arthur D., Jr.; Hughes, Stephen H.; Benjahad, Abdellah; Oumouch, Said; Guillemont, Jerome; Coupa, Sophie; Poncelet, Alain; Csoka, Imre; Meyer, Christophe; Andries, Koen; Nguyen, Chi Hung; Grierson, David S.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7582-7591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of AIDS, the efficacy of all drugs, including nonnucleoside inhibitors (NNRTIs) of HIV-1 reverse transcriptase (RT), has been limited by the rapid appearance of drug-resistant viruses.  Lys103Asn, Tyr181Cys, and Tyr188Leu are some of the most common RT mutations that cause resistance to NNRTIs in the clinic.  The authors report x-ray crystal structures for RT complexed with three different pyridinone derivs., R157208, R165481, and R221239, at 2.95, 2.9, and 2.43 Å resoln., resp.  All three ligands exhibit nanomolar or subnanomolar inhibitory activity against wild-type RT, but varying activities against drug-resistant mutants.  R165481 and R221239 differ from most NNRTIs in that binding does not involve significant contacts with Tyr181.  These compds. strongly inhibit wild-type HIV-1 RT and drug-resistant variants, including Tyr181Cys and Lys103Asn RT.  These properties result in part from an iodine atom on the pyridinone ring of both inhibitors that interacts with the main-chain carbonyl oxygen of Tyr188.  An acrylonitrile substituent on R165481 substantially improves the activity of the compd. against wild-type RT (and several mutants) and provides a way to generate novel inhibitors that could interact with conserved elements of HIV-1 RT at the polymerase catalytic site.  In R221239, there is a flexible linker to a furan ring that permits interactions with Val106, Phe227, and Pro236.  These contacts appear to enhance the inhibitory activity of R221239 against the HIV-1 strains that carry the Val106Ala, Tyr188Leu, and Phe227Cys mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJHWVjop0BXLVg90H21EOLACvtfcHk0lh9_5D8LxIA3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlur3J&md5=6c88c3dc1cf348274d35f1b402e9aae6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjm0500323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0500323%26sid%3Dliteratum%253Aachs%26aulast%3DHimmel%26aufirst%3DD.%2BM.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBenjahad%26aufirst%3DA.%26aulast%3DOumouch%26aufirst%3DS.%26aulast%3DGuillemont%26aufirst%3DJ.%26aulast%3DCoupa%26aufirst%3DS.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DCsoka%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DAndries%26aufirst%3DK.%26aulast%3DNguyen%26aufirst%3DC.%2BH.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DCrystal%2520Structures%2520for%2520HIV-1%2520Reverse%2520Transcriptase%2520in%2520Complexes%2520with%2520Three%2520Pyridinone%2520Derivatives%253A%2520A%2520New%2520Class%2520of%2520Non-Nucleoside%2520Inhibitors%2520Effective%2520against%2520a%2520Broad%2520Range%2520of%2520Drug-Resistant%2520Strains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26issue%3D24%26spage%3D7582%26epage%3D7591%26doi%3D10.1021%2Fjm0500323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Duijn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbrugghe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorens, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spöhr, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schedler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annane, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen Van, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misset, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jereb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seme, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šifrer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomiç, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estevez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eijkemans, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, I.</span></span> <span> </span><span class="NLM_article-title">The Effects of Antibiotic Cycling and Mixing on Antibiotic Resistance in Intensive Care Units: A Cluster-Randomised Crossover Trial</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30056-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2FS1473-3099%2818%2930056-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29396000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BC1MvmvV2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=401-409&issue=4&author=P.+J.+Van%0ADuijnauthor=W.+Verbruggheauthor=P.+G.+Jorensauthor=F.+Sp%C3%B6hrauthor=D.+Schedlerauthor=M.+Dejaauthor=A.+Rothbartauthor=D.+Annaneauthor=C.+Lawrenceauthor=J.-C.+Nguyen+Vanauthor=B.+Missetauthor=M.+Jerebauthor=K.+Semeauthor=F.+%C5%A0ifrerauthor=V.+Tomi%C3%A7author=F.+Estevezauthor=J.+Carneiroauthor=S.+Harbarthauthor=M.+J.+C.+Eijkemansauthor=M.+Bontenauthor=H.+Goossensauthor=S.+Malhotra-Kumarauthor=C.+Lammensauthor=J.+Vilaauthor=I.+Roca&title=The+Effects+of+Antibiotic+Cycling+and+Mixing+on+Antibiotic+Resistance+in+Intensive+Care+Units%3A+A+Cluster-Randomised+Crossover+Trial&doi=10.1016%2FS1473-3099%2818%2930056-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The effects of antibiotic cycling and mixing on antibiotic resistance in intensive care units: a cluster-randomised crossover trial</span></div><div class="casAuthors">van Duijn Pleun Joppe; Verbrugghe Walter; Jorens Philippe Germaine; Spohr Fabian; Schedler Dirk; Deja Maria; Rothbart Andreas; Annane Djillali; Lawrence Christine; Nguyen Van Jean-Claude; Misset Benoit; Jereb Matjaz; Seme Katja; Sifrer Franc; Tomic Viktorija; Estevez Francisco; Carneiro Jandira; Harbarth Stephan; Eijkemans Marinus Johannes Cornelis; Bonten Marc</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-409</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Whether antibiotic rotation strategies reduce prevalence of antibiotic-resistant, Gram-negative bacteria in intensive care units (ICUs) has not been accurately established.  We aimed to assess whether cycling of antibiotics compared with a mixing strategy (changing antibiotic to an alternative class for each consecutive patient) would reduce the prevalence of antibiotic-resistant, Gram-negative bacteria in European intensive care units (ICUs).  METHODS:  In a cluster-randomised crossover study, we randomly assigned ICUs to use one of three antibiotic groups (third-generation or fourth-generation cephalosporins, piperacillin-tazobactam, and carbapenems) as preferred empirical treatment during 6-week periods (cycling) or to change preference after every consecutively treated patient (mixing).  Computer-based randomisation of intervention and rotated antibiotic sequence was done centrally.  Cycling or mixing was applied for 9 months; then, following a washout period, the alternative strategy was implemented.  We defined antibiotic-resistant, Gram-negative bacteria as Enterobacteriaceae with extended-spectrum β-lactamase production or piperacillin-tazobactam resistance, and Acinetobacter spp and Pseudomonas aeruginosa with piperacillin-tazobactam or carbapenem resistance.  Data were collected for all admissions during the study.  The primary endpoint was average, unit-wide, monthly point prevalence of antibiotic-resistant, Gram-negative bacteria in respiratory and perineal swabs with adjustment for potential confounders.  This trial is registered with ClinicalTrials.gov, number NCT01293071.  FINDINGS:  Eight ICUs (from Belgium, France, Germany, Portugal, and Slovenia) were randomly assigned and patients enrolled from June 27, 2011, to Feb 16, 2014. 4069 patients were admitted during the cycling periods in total and 4707 were admitted during the mixing periods.  Of these, 745 patients during cycling and 853 patients during mixing were present during the monthly point-prevalence surveys, and were included in the main analysis.  Mean prevalence of the composite primary endpoint was 23% (168/745) during cycling and 22% (184/853) during mixing (p=0·64), yielding an adjusted incidence rate ratio during mixing of 1·039 (95% CI 0·837-1·291; p=0·73).  There was no difference in all-cause in-ICU mortality between intervention periods.  INTERPRETATION:  Antibiotic cycling does not reduce the prevalence of carriage of antibiotic-resistant, Gram-negative bacteria in patients admitted to the ICU.  FUNDING:  European Union Seventh Framework Programme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWfgjEsV9PxiCEBeYuoi7AfW6udTcc2eYR-zSOZ-3r5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvmvV2ksQ%253D%253D&md5=b09911c1f08c5a6d97b01bb102a28916</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930056-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930056-2%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDuijn%26aufirst%3DP.%2BJ.%26aulast%3DVerbrugghe%26aufirst%3DW.%26aulast%3DJorens%26aufirst%3DP.%2BG.%26aulast%3DSp%25C3%25B6hr%26aufirst%3DF.%26aulast%3DSchedler%26aufirst%3DD.%26aulast%3DDeja%26aufirst%3DM.%26aulast%3DRothbart%26aufirst%3DA.%26aulast%3DAnnane%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DC.%26aulast%3DNguyen%2BVan%26aufirst%3DJ.-C.%26aulast%3DMisset%26aufirst%3DB.%26aulast%3DJereb%26aufirst%3DM.%26aulast%3DSeme%26aufirst%3DK.%26aulast%3D%25C5%25A0ifrer%26aufirst%3DF.%26aulast%3DTomi%25C3%25A7%26aufirst%3DV.%26aulast%3DEstevez%26aufirst%3DF.%26aulast%3DCarneiro%26aufirst%3DJ.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DEijkemans%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DBonten%26aufirst%3DM.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DLammens%26aufirst%3DC.%26aulast%3DVila%26aufirst%3DJ.%26aulast%3DRoca%26aufirst%3DI.%26atitle%3DThe%2520Effects%2520of%2520Antibiotic%2520Cycling%2520and%2520Mixing%2520on%2520Antibiotic%2520Resistance%2520in%2520Intensive%2520Care%2520Units%253A%2520A%2520Cluster-Randomised%2520Crossover%2520Trial%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26issue%3D4%26spage%3D401%26epage%3D409%26doi%3D10.1016%2FS1473-3099%2818%2930056-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beardmore, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peña-Miller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iredell, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Cycling and Antibiotic Mixing: Which One Best Mitigates Antibiotic Resistance?</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">802</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1093/molbev/msw292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsw292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28096304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFOmt7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2017&pages=802-817&issue=4&author=R.+E.+Beardmoreauthor=R.+Pe%C3%B1a-Millerauthor=F.+Goriauthor=J.+Iredell&title=Antibiotic+Cycling+and+Antibiotic+Mixing%3A+Which+One+Best+Mitigates+Antibiotic+Resistance%3F&doi=10.1093%2Fmolbev%2Fmsw292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic cycling and antibiotic mixing: which one best mitigates antibiotic resistance?</span></div><div class="casAuthors">Beardmore, Robert Eric; Pena-Miller, Rafael; Gori, Fabio; Iredell, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology and Evolution</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">802-817</span>CODEN:
                <span class="NLM_cas:coden">MBEVEO</span>;
        ISSN:<span class="NLM_cas:issn">1537-1719</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Can we exploit our burgeoning understanding of mol. evolution to slow the progress of drug resistance.  One role of an infection clinician is exactly that: to foresee trajectories to resistance during antibiotic treatment and to hinder that evolutionary course.  But can this be done at a hospital-wide scale.  Clinicians and theoreticians tried to when they proposed two conflicting behavioral strategies that are expected to curb resistance evolution in the clinic, these are known as "antibiotic cycling" and "antibiotic mixing." However, the accumulated data from clin. trials, now approaching 4 million patient days of treatment, is too variable for cycling or mixing to be deemed successful.  The former implements the restriction and prioritization of different antibiotics at different times in hospitals in a manner said to "cycle" between them.  In antibiotic mixing, appropriate antibiotics are allocated to patients but randomly.  Mixing results in no correlation, in time or across patients, in the drugs used for treatment which is why theorists saw this as an optimal behavioral strategy.  So while cycling and mixing were proposed as ways of controlling evolution, we show there is good reason why clin. datasets cannot choose between them: by re-examg. the theor. literature we show prior support for the theor. optimality of mixing was misplaced.  Our anal. is consistent with a pattern emerging in data: neither cycling or mixing is a priori better than the other at mitigating selection for antibiotic resistance in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKctTVU3331rVg90H21EOLACvtfcHk0lj3OaMXUcqr-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFOmt7%252FM&md5=c6e039f95e2d03c21eee22b16f2daad0</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsw292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsw292%26sid%3Dliteratum%253Aachs%26aulast%3DBeardmore%26aufirst%3DR.%2BE.%26aulast%3DPe%25C3%25B1a-Miller%26aufirst%3DR.%26aulast%3DGori%26aufirst%3DF.%26aulast%3DIredell%26aufirst%3DJ.%26atitle%3DAntibiotic%2520Cycling%2520and%2520Antibiotic%2520Mixing%253A%2520Which%2520One%2520Best%2520Mitigates%2520Antibiotic%2520Resistance%253F%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2017%26volume%3D34%26issue%3D4%26spage%3D802%26epage%3D817%26doi%3D10.1093%2Fmolbev%2Fmsw292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">OGEE v2: An Update of the Online Gene Essentiality Database with Special Focus on Differentially Essential Genes in Human Cancer Cell Lines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D940</span>– <span class="NLM_lpage">D944</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkw1013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27799467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D940-D944&issue=D1&author=W.-H.+Chenauthor=G.+Luauthor=X.+Chenauthor=X.-M.+Zhaoauthor=P.+Bork&title=OGEE+v2%3A+An+Update+of+the+Online+Gene+Essentiality+Database+with+Special+Focus+on+Differentially+Essential+Genes+in+Human+Cancer+Cell+Lines&doi=10.1093%2Fnar%2Fgkw1013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">GEE v2: an update of the online gene essentiality database with special focus on differentially essential genes in human cancer cell lines</span></div><div class="casAuthors">Chen, Wei-Hua; Lu, Guanting; Chen, Xiao; Zhao, Xing-Ming; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D940-D944</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  OGEE is an Online GEne Essentiality database.  To enhance our understanding of the essentiality of genes, in OGEE we collected exptl. tested essential and non-essential genes, as well as assocd. gene properties known to contribute to gene essentiality.  We focus on large-scale expts., and complement our data with text-mining results.  We organized tested genes into data sets according to their sources, and tagged those with variable essentiality statuses across data sets as conditionally essential genes, intending to highlight the complex interplay between gene functions and environments/exptl. perturbations.  Developments since the last public release include increased nos. of species and gene essentiality data sets, inclusion of non-coding essential sequences and genes with intermediate essentiality statuses.  In addn., we included 16 essentiality data sets from cancer cell lines, corresponding to 9 human cancers; with OGEE, users can easily explore the shared and differentially essential genes within and between cancer types.  These genes, esp. those derived from cell lines that are similar to tumor samples, could reveal the oncogenic drivers, paralogous gene expression pattern and chromosomal structure of the corresponding cancer types, and can be further screened to identify targets for cancer therapy and/or new drug development.  OGEE is freely available at http:// ogee.medgenius.info.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm3bqO4MARdrVg90H21EOLACvtfcHk0lj3OaMXUcqr-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhsLc%253D&md5=0df15fc6e5627f11a3c418b654d6e4b0</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1013%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.-H.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DX.-M.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DOGEE%2520v2%253A%2520An%2520Update%2520of%2520the%2520Online%2520Gene%2520Essentiality%2520Database%2520with%2520Special%2520Focus%2520on%2520Differentially%2520Essential%2520Genes%2520in%2520Human%2520Cancer%2520Cell%2520Lines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD940%26epage%3DD944%26doi%3D10.1093%2Fnar%2Fgkw1013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.-B.</span></span> <span> </span><span class="NLM_article-title">CEG: A Database of Essential Gene Clusters</span>. <i>BMC Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">769</span>, <span class="refDoi"> DOI: 10.1186/1471-2164-14-769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1471-2164-14-769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24209780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKnu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=769&author=Y.-N.+Yeauthor=Z.-G.+Huaauthor=J.+Huangauthor=N.+Raoauthor=F.-B.+Guo&title=CEG%3A+A+Database+of+Essential+Gene+Clusters&doi=10.1186%2F1471-2164-14-769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">CEG: a database of essential gene clusters</span></div><div class="casAuthors">Ye, Yuan-Nong; Hua, Zhi-Gang; Huang, Jian; Rao, Nini; Guo, Feng-Biao</div><div class="citationInfo"><span class="NLM_cas:title">BMC Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769</span>CODEN:
                <span class="NLM_cas:coden">BGMEET</span>;
        ISSN:<span class="NLM_cas:issn">1471-2164</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Essential genes are indispensable for the survival of living entities.  They are the cornerstones of synthetic biol., and are potential candidate targets for antimicrobial and vaccine design.  Description: Here we describe the Cluster of Essential Genes (CEG) database, which contains clusters of orthologous essential genes.  Based on the size of a cluster, users can easily decide whether an essential gene is conserved in multiple bacterial species or is species-specific.  It contains the similarity value of every essential gene cluster against human proteins or genes.  The CEG_Match tool is based on the CEG database, and was developed for prediction of essential genes according to function.  The database is available at online.  Conclusions: Properties contained in the CEG database, such as cluster size, and the similarity of essential gene clusters against human proteins or genes, are very important for evolutionary research and drug design.  An advantage of CEG is that it clusters essential genes based on function, and therefore decreases false pos. results when predicting essential genes in comparison with using the similarity alignment method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZhiPu_sGPJLVg90H21EOLACvtfcHk0lgmI4GOpL7_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKnu7zL&md5=f3680dd9347640fc61682e7f9204cb72</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1186%2F1471-2164-14-769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2164-14-769%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.-N.%26aulast%3DHua%26aufirst%3DZ.-G.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DRao%26aufirst%3DN.%26aulast%3DGuo%26aufirst%3DF.-B.%26atitle%3DCEG%253A%2520A%2520Database%2520of%2520Essential%2520Gene%2520Clusters%26jtitle%3DBMC%2520Genomics%26date%3D2013%26volume%3D14%26spage%3D769%26doi%3D10.1186%2F1471-2164-14-769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">DEG 10, an Update of the Database of Essential Genes That Includes Both Protein-Coding Genes and Noncoding Genomic Elements</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D574</span>– <span class="NLM_lpage">D580</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkt1131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24243843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1am" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D574-D580&issue=D1&author=H.+Luoauthor=Y.+Linauthor=F.+Gaoauthor=C.-T.+Zhangauthor=R.+Zhang&title=DEG+10%2C+an+Update+of+the+Database+of+Essential+Genes+That+Includes+Both+Protein-Coding+Genes+and+Noncoding+Genomic+Elements&doi=10.1093%2Fnar%2Fgkt1131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements</span></div><div class="casAuthors">Luo, Hao; Lin, Yan; Gao, Feng; Zhang, Chun-Ting; Zhang, Ren</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D574-D580</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The combination of high-d. transposon-mediated mutagenesis and high-throughput sequencing has led to significant advancements in research on essential genes, resulting in a dramatic increase in the no. of identified prokaryotic essential genes under diverse conditions and a revised essential-gene concept that includes all essential genomic elements, rather than focusing on protein-coding genes only.  DEG 10, a new release of the Database of Essential Genes (available at http://www.essentialgene.org), has been developed to accommodate these quant. and qual. advancements.  In addn. to increasing the no. of bacterial and archaeal essential genes detd. by genome-wide gene essentiality screens, DEG 10 also harbors essential noncoding RNAs, promoters, regulatory sequences and replication origins.  These essential genomic elements are detd. not only in vitro, but also in vivo, under diverse conditions including those for survival, pathogenesis and antibiotic resistance.  We have developed customizable BLAST tools that allow users to perform species- and expt.-specific BLAST searches for a single gene, a list of genes, annotated or unannotated genomes.  Therefore, DEG 10 includes essential genomic elements under different conditions in three domains of life, with customizable BLAST tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxIBwiW86_xLVg90H21EOLACvtfcHk0lgmI4GOpL7_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1am&md5=f7ddd7a094ab324d43dedb4c54c8c138</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1131%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DC.-T.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DDEG%252010%252C%2520an%2520Update%2520of%2520the%2520Database%2520of%2520Essential%2520Genes%2520That%2520Includes%2520Both%2520Protein-Coding%2520Genes%2520and%2520Noncoding%2520Genomic%2520Elements%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD574%26epage%3DD580%26doi%3D10.1093%2Fnar%2Fgkt1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&issue=13&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94A+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lioak4k6c3ZSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26issue%3D13%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Boyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Open Babel: An Open Chemical Toolbox</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+Open+Chemical+Toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0lioak4k6c3ZSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520Open%2520Chemical%2520Toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Docking Validation Resources: Protein Family and Ligand Flexibility Experiments</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">2000</span>, <span class="refDoi"> DOI: 10.1021/ci1001982</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1001982" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGktLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1986-2000&issue=11&author=S.+Mukherjeeauthor=T.+E.+Baliusauthor=R.+C.+Rizzo&title=Docking+Validation+Resources%3A+Protein+Family+and+Ligand+Flexibility+Experiments&doi=10.1021%2Fci1001982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Validation Resources: Protein Family and Ligand Flexibility Experiments</span></div><div class="casAuthors">Mukherjee, Sudipto; Balius, Trent E.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1986-2000</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A database consisting of 780 ligand-receptor complexes, termed SB2010, has been derived from the Protein Databank to evaluate the accuracy of docking protocols for regenerating bound ligand conformations.  The goal is to provide easily accessible community resources for development of improved procedures to aid virtual screening for ligands with a wide range of flexibilities.  Three core expts. using the program DOCK, which employ rigid (RGD), fixed anchor (FAD), and flexible (FLX) protocols, were used to gauge performance by several different metrics: (1) global results, (2) ligand flexibility, (3) protein family, and (4) cross-docking.  Global spectrum plots of successes and failures vs rmsd reveal well-defined inflection regions, which suggest the commonly used 2 Å criteria is a reasonable choice for defining success.  Across all 780 systems, success tracks with the relative difficulty of the calcns.: RGD (82.3%) > FAD (78.1%) > FLX (63.8%).  In general, failures due to scoring strongly outweigh those due to sampling.  Subsets of SB2010 grouped by ligand flexibility (7-or-less, 8-to-15, and 15-plus rotatable bonds) reveal that success degrades linearly for FAD and FLX protocols, in contrast to RGD, which remains const.  Despite the challenges assocd. with FLX anchor orientation and on-the-fly flexible growth, success rates for the 7-or-less (74.5%) and, in particular, the 8-to-15 (55.2%) subset are encouraging.  Poorer results for the very flexible 15-plus set (39.3%) indicate substantial room for improvement.  Family-based success appears largely independent of ligand flexibility, suggesting a strong dependence on the binding site environment.  For example, zinc-contg. proteins are generally problematic, despite moderately flexible ligands.  Finally, representative cross-docking examples, for carbonic anhydrase, thermolysin, and neuraminidase families, show the utility of family-based anal. for rapid identification of particularly good or bad docking trends, and the type of failures involved (scoring/sampling), which will likely be of interest to researchers making specific receptor choices for virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp2rTY0qXg77Vg90H21EOLACvtfcHk0lioak4k6c3ZSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGktLzL&md5=ed08ece5b658b52a83361a2ffb92ca83</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci1001982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1001982%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDocking%2520Validation%2520Resources%253A%2520Protein%2520Family%2520and%2520Ligand%2520Flexibility%2520Experiments%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D11%26spage%3D1986%26epage%3D2000%26doi%3D10.1021%2Fci1001982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benjamini, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, Y.</span></span> <span> </span><span class="NLM_article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>. <i>J. R. Stat. Soc. Ser. B Methodol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1111/j.2517-6161.1995.tb02031.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&issue=1&author=Y.+Benjaminiauthor=Y.+Hochberg&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing&doi=10.1111%2Fj.2517-6161.1995.tb02031.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.2517-6161.1995.tb02031.x%26sid%3Dliteratum%253Aachs%26aulast%3DBenjamini%26aufirst%3DY.%26aulast%3DHochberg%26aufirst%3DY.%26atitle%3DControlling%2520the%2520False%2520Discovery%2520Rate%253A%2520A%2520Practical%2520and%2520Powerful%2520Approach%2520to%2520Multiple%2520Testing%26jtitle%3DJ.%2520R.%2520Stat.%2520Soc.%2520Ser.%2520B%2520Methodol.%26date%3D1995%26volume%3D57%26issue%3D1%26spage%3D289%26epage%3D300%26doi%3D10.1111%2Fj.2517-6161.1995.tb02031.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Jiang</span>, <span class="hlFld-ContribAuthor ">Wei-Rong  Cui</span>, <span class="hlFld-ContribAuthor ">Ru-Ping  Liang</span>, <span class="hlFld-ContribAuthor ">Jian-Ding  Qiu</span>. </span><span class="cited-content_cbyCitation_article-title">Zwitterionic surface charge regulation in ionic covalent organic nanosheets: Synergistic adsorption of fluoroquinolone antibiotics. </span><span class="cited-content_cbyCitation_journal-name">Chemical Engineering Journal</span><span> <strong>2021,</strong> <em>417 </em>, 128034. <a href="https://doi.org/10.1016/j.cej.2020.128034" title="DOI URL">https://doi.org/10.1016/j.cej.2020.128034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cej.2020.128034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cej.2020.128034%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Engineering%2520Journal%26atitle%3DZwitterionic%252Bsurface%252Bcharge%252Bregulation%252Bin%252Bionic%252Bcovalent%252Borganic%252Bnanosheets%25253A%252BSynergistic%252Badsorption%252Bof%252Bfluoroquinolone%252Bantibiotics%26aulast%3DJiang%26aufirst%3DWei%26date%3D2021%26volume%3D417%26spage%3D128034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Jiang</span>, <span class="hlFld-ContribAuthor ">Wei-Rong  Cui</span>, <span class="hlFld-ContribAuthor ">Ru-Ping  Liang</span>, <span class="hlFld-ContribAuthor ">Jian-Ding  Qiu</span>. </span><span class="cited-content_cbyCitation_article-title">Difunctional covalent organic framework hybrid material for synergistic adsorption and selective removal of fluoroquinolone antibiotics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hazardous Materials</span><span> <strong>2021,</strong> <em>413 </em>, 125302. <a href="https://doi.org/10.1016/j.jhazmat.2021.125302" title="DOI URL">https://doi.org/10.1016/j.jhazmat.2021.125302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jhazmat.2021.125302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jhazmat.2021.125302%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hazardous%2520Materials%26atitle%3DDifunctional%252Bcovalent%252Borganic%252Bframework%252Bhybrid%252Bmaterial%252Bfor%252Bsynergistic%252Badsorption%252Band%252Bselective%252Bremoval%252Bof%252Bfluoroquinolone%252Bantibiotics%26aulast%3DJiang%26aufirst%3DWei%26date%3D2021%26volume%3D413%26spage%3D125302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  Bremner</span>. </span><span class="cited-content_cbyCitation_article-title">Single Molecule Non-cleavable Multiply Active Antibacterials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 51-119. <a href="https://doi.org/10.1007/978-981-16-0999-2_3" title="DOI URL">https://doi.org/10.1007/978-981-16-0999-2_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-16-0999-2_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-16-0999-2_3%26sid%3Dliteratum%253Aachs%26atitle%3DSingle%252BMolecule%252BNon-cleavable%252BMultiply%252BActive%252BAntibacterials%26aulast%3DBremner%26aufirst%3DJohn%26date%3D2021%26date%3D2021%26spage%3D51%26epage%3D119%26pub%3DSpringer%2520Singapore%26atitle%3DMultiple%252BAction-Based%252BDesign%252BApproaches%252Bto%252BAntibacterials%26aulast%3DBremner%26aufirst%3DJohn%26date%3D2021%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natesh  Singh</span>, <span class="hlFld-ContribAuthor ">Bruno O.  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Resources and computational strategies to advance small molecule SARS-CoV-2 discovery: Lessons from the pandemic and preparing for future health crises. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 2537-2548. <a href="https://doi.org/10.1016/j.csbj.2021.04.059" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.04.059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.04.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.04.059%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DResources%252Band%252Bcomputational%252Bstrategies%252Bto%252Badvance%252Bsmall%252Bmolecule%252BSARS-CoV-2%252Bdiscovery%25253A%252BLessons%252Bfrom%252Bthe%252Bpandemic%252Band%252Bpreparing%252Bfor%252Bfuture%252Bhealth%252Bcrises%26aulast%3DSingh%26aufirst%3DNatesh%26date%3D2021%26volume%3D19%26spage%3D2537%26epage%3D2548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minmin  Liu</span>, <span class="hlFld-ContribAuthor ">Zhiying  Zhao</span>, <span class="hlFld-ContribAuthor ">Wenzheng  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Citric acid modified wood membranes for efficient adsorption of tetracycline: Effect of alkali pretreatment concentration and adsorption mechanism. </span><span class="cited-content_cbyCitation_journal-name">Chemical Engineering Journal</span><span> <strong>2020,</strong> <em>393 </em>, 124748. <a href="https://doi.org/10.1016/j.cej.2020.124748" title="DOI URL">https://doi.org/10.1016/j.cej.2020.124748</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cej.2020.124748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cej.2020.124748%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Engineering%2520Journal%26atitle%3DCitric%252Bacid%252Bmodified%252Bwood%252Bmembranes%252Bfor%252Befficient%252Badsorption%252Bof%252Btetracycline%25253A%252BEffect%252Bof%252Balkali%252Bpretreatment%252Bconcentration%252Band%252Badsorption%252Bmechanism%26aulast%3DLiu%26aufirst%3DMinmin%26date%3D2020%26volume%3D393%26spage%3D124748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of homomers with multichain binding sites (MBS), single-chain binding sites (SBS), and an overview of binding site identification in homologs. (A) Structure of nitroreductase ydjA from <i>E. coli</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3bm1">3bm1</a>). The dimer structure has two multichain binding sites, both “sandwiched” between the two chains of the complex. The ligand (flavin-mononucleotide) is displayed in red, and ligand binding residues are in yellow. (B) Structure FabH protein from <i>E. coli</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ebl">1ebl</a>). The dimer has two binding sites, both restricted to a single chain. The ligand (coenzyme A) is displayed in red, and ligand binding residues are in yellow. (C) Structure of SiaP protein from <i>Haemophilus influenzae</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wyk">2wyk</a>). The protein is a monomer, and has a single binding site, with its ligand (<i>N</i>-glyconeuraminic acid) displayed in red. (D) We searched for similar binding sites in homologous proteins with ProBiS and defined the region of binding sites for grid building and docking through local structural alignments. The structural alignment shows the <i>H. influenzae</i> SiaP protein binding site superposed with the binding site (identified by ProBis) of the homologous c4-dicarboxylate-binding protein of <i>Pseudomonas aeruginosa</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4nf0">4nf0</a>); the red box indicates the region of the binding sites. Note that the alignment optimized the superposition of the binding sites and not the global protein structures. (E) Once the binding site has been identified in c4-dicarboxylate-binding protein, the ligand of <i>H. influenzae</i> SiaP protein was docked into it, and the binding energies (i.e., grid score) in the two structures were compared.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligands of homomers are more likely to bind their homologs than the ligands of monomers. (A) Frequency of homologous protein pairs without similar ligand binding sites, (ProBiS <i>Z</i> score of >2). See also ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> for general trends in ligand binding site evolution. MBS homomers are much more likely to have an identical binding site in their homologs than SBS homomers (<i>p</i> = 0.029, test of proportions) or monomers (<i>p</i> = 0.00011, test of proportions). (B) The difference between the estimated ligand binding energies (grid score) in the original binding sites and homologous binding sites shows that homomers, and particularly MBS homomers, are much more likely to bind the ligands of their homologs than monomers. Normalized grid score difference was calculated as (score<sub>original</sub> – score<sub>homolog</sub>)/score<sub>original</sub>. All three possible comparisons are significant (<i>p</i> = 0.013 for MBS vs SBS homomer; <i>p</i> ≪ 0.005 for MBS/SBS homomer vs monomers; Kruskal–Wallis rank sum tests). (C) Pose reproduction success rate of DOCK for ligands of different sizes. The high flexibility of ligands above 40 heavy atoms in this particular data set results in a relatively low (∼50%) success rate for large ligands. (D–F) The relationships between ligand size and binding energy (grid score) follow a linear trend in the original ligand binding sites and the binding sites of homologs. The difference between the original binding sites and the binding sites of homologs is lowest in MBS homomers and largest in monomers (ANCOVA).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. General properties of antibiotics and de novo ligands. For each receptor, the 50 best scoring ligands were included. (A) Molecular weight distribution of Drugbank antibiotics, below 600 Da. (B) Molecular weight distributions of de novo ligands in MBS homomers and monomers. The de novo ligands of MBS homomers are significantly larger than of monomers, while the ligands of SBS homomers (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S1</a>) do not show the same trend. (C) The frequency of ligands that bind similarly in homologs and their original binding site is significantly higher among MBS homomers than monomers above 300 Da (∗∗, <i>p</i> ≪ 0.005, tests of proportions). (D) The log <i>P</i> values of antibiotics and de novo ligands. For most antibiotics log <i>P</i> is below 3, but for a significant number of de novo ligands it is above 3, particularly above 300 Da. (E) The druglikeness of antibiotics and de novo ligands follows the same trend and declines above molecular weight of 300. (F) Below 300 Da the synthetic accessibility of de novo ligands is considerably worse than of real antibiotics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. The best scoring de novo ligands show qualitatively similar, although weaker trends than the original ligands of the receptors. (A) Difference between the normalized grid score of the 10 best ligands of each receptor. All three possible comparisons are significant (<i>p</i> < 0.0001 Kruskal–Wallis rank sum tests). (B) The frequency of clashes is significantly higher in monomers than in homomers (<i>p</i> < 0.001 for both MBS and SBS homomers, tests of proportions), but there is no difference between MBS and SBS homomers homomers (<i>p</i> = 0.43). (C–E) Relationships between ligand size and estimated binding energy (grid score) of the best scoring ligand of each receptor in the original ligand binding site and the binding sites of homologs. Similar to the original ligands of the receptor, the relationship between size and grid score is linear, and the difference between the original binding sites and the binding sites of homologs is smallest in MBS homomers and largest in monomers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Outline of the structural comparisons using the VAE vector representation. (A) De novo ligands were converted to vector, which represents the parameters of a statistical distribution and defines the location of the compound in the latent space. The Euclidean distance between the vectors was used to measure the structural similarity between all ∼256 million possible pairs of compounds. (B) The frequency of pairs with distance below 13 is negatively correlated with their molecular weight. (C) For molecules above 300 Da the distance cutoff 13 results in an approximate Tanimoto similarity of 0.7 (most of them are in the range of 300–400 Da).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Properties of the selected de novo ligands in set 1. (A) Examples of interesting de novo ligands. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip" class="ext-link">Supplementary Data</a> for the full list. Several of them are reminiscent of quinolones, quinazolinones, oxadiazoles, and morpholine antifungals. Oxazolidinone, quinazolinone, oxadiazole, and morholine groups are highlighted with red, and the DOCK anchor fragments are highlighted with blue. A particularly interesting case (top left) contains a quinazolinone linker and an oxazolidinone side chain (oxazolidinones are among the newest antibiotics, e.g., linezolid, tedizolid,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> used to treat vancomycin resistant Gram-positive bacteria). The compound at the top center is reminiscent of an oxadiazole antibiotic, while the bottom left compound resembles morpholine antifungals. (B) Word cloud of the significantly enriched fragments (<i>p</i> < 0.05, tests of proportions; sid. = side chain, a fragment with only one connection; lnk. = linker, a fragment with two connections; scf. = scaffold, a fragment with three connections). The size of the symbols corresponds to the abundance of the fragments. (C) Enrichment of the fragments. Enrichment was calculated as frequency in set 1/frequency in all de novo ligands with similar size and properties to set 1. (D) 2D structures of the significantly enriched fragments. Several fragments with halogen groups (Cl/Br) are present among them and also a morpholine side chain (sid.68).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Gene ontology analysis of the receptors of selected ligands. (A) The quaternary structure composition is not significantly different from the total set of proteins (<i>p</i> = 0.42), although the frequency of MBS homomers is lower, as a result of the requirement to have similar structures in at least three species. (B) Graph of cellular component terms associated with the proteins. Note that the color-coding of terms (intensity of red) corresponds to the frequency of terms and not statistical significance. The proteins are present in most cellular components, including cell wall, periplasmic space, plasma membrane, and cytoplasm. (C, D) Graphs of molecular function terms associated with the proteins. The two highest-level terms are binding (C) and catalytic activity (D), with nucleoside/nucleotide binding and oxidoreductase activity being the most common terms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Optimization and evolution of ligands with AI. (A) Outline of the method. First, the selected de novo ligands were encoded with the VAE, and the latent space was sampled in their vicinity to search for related structures. Next the, samples were “decoded” into chemical structures and were docked into three different receptors, and the best performing (lowest scoring) molecule was used to repeat the cycle until there was no more improvement in the estimated binding energies (grid score). See also <a href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf" class="ext-link">Figure S8</a>. (B) The optimization resulted in a comparable improvement in binding energies (grid score), irrespective of the <i>Z</i> distance cutoff used. (C) The median Tanimoto similarity of optimized ligands and de novo ligands is 0.7. (D, E) The frequency of four- and three-membered rings in the optimized ligands is significantly higher than in the original de novo ligands and the random expectation (∗, <i>p</i> < 0.05; ∗∗, <i>p</i> < 0.005, randomization tests). Distance 0 indicates the original de novo ligands, red horizontal bars indicate the observed frequency of four- or three-membered rings, while black horizontal bars indicate their expected frequency. Violin plots show the frequency distribution of 10 000 random replicates. Outliers above 20% were not plotted for clarity. Note that in the case of four-member rings, the expected frequency is higher than their frequency among the de novo ligands (<i>Z</i> = 0) because they are more common in the VAE samples. (F) Example of the emergence of a β-lactam-like ring in a de novo ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Fragments of the optimized ligands, which are present in at least two compounds in any of the VAE batches (excluding fragments that are likely to be toxic). Several halogenated compounds and four-membered rings are present among them. Since the four-membered rings frequently contain reactive double bonds, in practice their saturated versions should be used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/medium/jm-2019-00220a_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Ligands optimized for binding multiple receptors have more rotatable bonds than the original de novo ligands. (A) Number of rotatable bonds in de novo and VAE optimized ligands of set 1. Dark red represents the original de novo (<i>Z</i> = 0) or final optimized ligands, and blue indicates the expected number of rotatable bonds in the compounds returned by the VAE, without selection for multitarget binding. This was calculated as the averages of all ligands returned by the VAE when the de novo ligands were used as the input. In all VAE batches the number of rotatable bonds is significantly higher than in the expectation or de novo ligands (∗∗, <i>p</i> < 0.005, Wilcoxon tests on the differences from de novo ligands). (B) Relationship between the change in rotatable bonds and grid score in set 1, using the pooled data of all VAE batches. The correlation is not significant due to a few outliers with rotatable bond change of −2; excluding them results in significant correlation (<i>p</i> = 7.88 × 10<sup>–4</sup>). (C) Number of rotatable bonds in de novo and VAE optimized ligands of set 2. The color coding is the same as on panel A. Similar to set 1, in all VAE batches the number of rotatable bonds is significantly higher than in the expectation or de novo ligands (∗∗, <i>p</i> < 0.005, Wilcoxon tests on the differences from de novo ligands). (D) In set 2, the correlation between the change in rotatable bonds and grid score is highly significant (<i>p</i> = 7.57 × 10<sup>–10</sup>), indicating that the more flexible is the molecule, the better it can bind multiple receptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00220/20191108/images/large/jm-2019-00220a_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00220&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33455" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33455" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 84 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowater, L.</span></span> <i>The Microbes Fight Back: Antibiotic Resistance</i>, <span class="NLM_edition">Gld</span> ed.; <span class="NLM_publisher-name">Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Bowater&title=The+Microbes+Fight+Back%3A+Antibiotic+Resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBowater%26aufirst%3DL.%26btitle%3DThe%2520Microbes%2520Fight%2520Back%253A%2520Antibiotic%2520Resistance%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D.</span></span> <span> </span><span class="NLM_article-title">Origins and Evolution of Antibiotic Resistance</span>. <i>Microbiol Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1128/MMBR.00016-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1128%2FMMBR.00016-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20805405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSqurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2010&pages=417-433&issue=3&author=J.+Daviesauthor=D.+Davies&title=Origins+and+Evolution+of+Antibiotic+Resistance&doi=10.1128%2FMMBR.00016-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Origins and evolution of antibiotic resistance</span></div><div class="casAuthors">Davies, Julian; Davies, Dorothy</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-433</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Antibiotics have always been considered one of the wonder discoveries of the 20th century.  This is true, but the real wonder is the rise of antibiotic resistance in hospitals, communities, and the environment concomitant with their use.  The extraordinary genetic capacities of microbes have benefitted from man's overuse of antibiotics to exploit every source of resistance genes and every means of horizontal gene transmission to develop multiple mechanisms of resistance for each and every antibiotic introduced into practice clin., agriculturally, or otherwise.  This review presents the salient aspects of antibiotic resistance development over the past half-century, with the oft-restated conclusion that it is time to act.  To achieve complete restitution of therapeutic applications of antibiotics, there is a need for more information on the role of environmental microbiomes in the rise of antibiotic resistance.  In particular, creative approaches to the discovery of novel antibiotics and their expedited and controlled introduction to therapy are obligatory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6-EM4T8NgWLVg90H21EOLACvtfcHk0ljHwwagCQohQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSqurnO&md5=64214f42f2ae91c1ba94dbe765b7cd0c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMMBR.00016-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.00016-10%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DD.%26atitle%3DOrigins%2520and%2520Evolution%2520of%2520Antibiotic%2520Resistance%26jtitle%3DMicrobiol%2520Mol.%2520Biol.%2520Rev.%26date%3D2010%26volume%3D74%26issue%3D3%26spage%3D417%26epage%3D433%26doi%3D10.1128%2FMMBR.00016-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span> <span> </span><span class="NLM_article-title">Drugs for Bad Bugs: Confronting the Challenges of Antibacterial Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/nrd2201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&issue=1&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+L.+Pompliano&title=Drugs+for+Bad+Bugs%3A+Confronting+the+Challenges+of+Antibacterial+Discovery&doi=10.1038%2Fnrd2201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0ljHwwagCQohQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26atitle%3DDrugs%2520for%2520Bad%2520Bugs%253A%2520Confronting%2520the%2520Challenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D1%26spage%3D29%26epage%3D40%26doi%3D10.1038%2Fnrd2201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Retrospective Analysis of Natural Products Provides Insights for Future Discovery Trends</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5601</span>– <span class="NLM_lpage">5606</span>, <span class="refDoi"> DOI: 10.1073/pnas.1614680114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.1614680114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28461474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=5601-5606&issue=22&author=C.+R.+Pyeauthor=M.+J.+Bertinauthor=R.+S.+Lokeyauthor=W.+H.+Gerwickauthor=R.+G.+Linington&title=Retrospective+Analysis+of+Natural+Products+Provides+Insights+for+Future+Discovery+Trends&doi=10.1073%2Fpnas.1614680114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Retrospective analysis of natural products provides insights for future discovery trends</span></div><div class="casAuthors">Pye, Cameron R.; Bertin, Matthew J.; Lokey, R. Scott; Gerwick, William H.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5601-5606</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding of the capacity of the natural world to produce secondary metabolites is important to a broad range of fields, including drug discovery, ecol., biosynthesis, and chem. biol., among others.  Both the abs. no. and the rate of discovery of natural products have increased significantly in recent years.  However, there is a perception and concern that the fundamental novelty of these discoveries is decreasing relative to previously known natural products.  This study presents a quant. examn. of the field from the perspective of both no. of compds. and compd. novelty using a dataset of all published microbial and marine-derived natural products.  This anal. aimed to explore a no. of key questions, such as how the rate of discovery of new natural products has changed over the past decades, how the av. natural product structural novelty has changed as a function of time, whether exploring novel taxonomic space affords an advantage in terms of novel compd. discovery, and whether it is possible to est. how close we are to having described all of the chem. space covered by natural products.  Our analyses demonstrate that most natural products being published today bear structural similarity to previously published compds., and that the range of scaffolds readily accessible from nature is limited.  However, the anal. also shows that the field continues to discover appreciable nos. of natural products with no structural precedent.  Together, these results suggest that the development of innovative discovery methods will continue to yield compds. with unique structural and biol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3OY35vWvx_bVg90H21EOLACvtfcHk0ljHwwagCQohQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D&md5=44da23318031b09694c94b0bfdf1d718</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614680114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614680114%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DBertin%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DGerwick%26aufirst%3DW.%2BH.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DRetrospective%2520Analysis%2520of%2520Natural%2520Products%2520Provides%2520Insights%2520for%2520Future%2520Discovery%2520Trends%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D22%26spage%3D5601%26epage%3D5606%26doi%3D10.1073%2Fpnas.1614680114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skinnider, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarvey, N. A.</span></span> <span> </span><span class="NLM_article-title">Statistical Reanalysis of Natural Products Reveals Increasing Chemical Diversity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">E6271</span>– <span class="NLM_lpage">E6272</span>, <span class="refDoi"> DOI: 10.1073/pnas.1708560114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.1708560114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28710332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ggt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6271-E6272&issue=31&author=M.+A.+Skinniderauthor=N.+A.+Magarvey&title=Statistical+Reanalysis+of+Natural+Products+Reveals+Increasing+Chemical+Diversity&doi=10.1073%2Fpnas.1708560114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical reanalysis of natural products reveals increasing chemical diversity</span></div><div class="casAuthors">Skinnider, Michael A.; Magarvey, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">E6271-E6272</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4qQ4_ZBc7RLVg90H21EOLACvtfcHk0liUi2PN382-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ggt77O&md5=12ed6c205f7df965a331f4f48b3b245c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1708560114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1708560114%26sid%3Dliteratum%253Aachs%26aulast%3DSkinnider%26aufirst%3DM.%2BA.%26aulast%3DMagarvey%26aufirst%3DN.%2BA.%26atitle%3DStatistical%2520Reanalysis%2520of%2520Natural%2520Products%2520Reveals%2520Increasing%2520Chemical%2520Diversity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D31%26spage%3DE6271%26epage%3DE6272%26doi%3D10.1073%2Fpnas.1708560114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toprak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veres, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Evolutionary Paths to Antibiotic Resistance under Dynamically Sustained Drug Selection</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/ng.1034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fng.1034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1eisbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=101-105&issue=1&author=E.+Toprakauthor=A.+Veresauthor=J.-B.+Michelauthor=R.+Chaitauthor=D.+L.+Hartlauthor=R.+Kishony&title=Evolutionary+Paths+to+Antibiotic+Resistance+under+Dynamically+Sustained+Drug+Selection&doi=10.1038%2Fng.1034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary paths to antibiotic resistance under dynamically sustained drug selection</span></div><div class="casAuthors">Toprak, Erdal; Veres, Adrian; Michel, Jean-Baptiste; Chait, Remy; Hartl, Daniel L.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-105</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic resistance can evolve through the sequential accumulation of multiple mutations.  To study such gradual evolution, we developed a selection device, the 'morbidostat', that continuously monitors bacterial growth and dynamically regulates drug concns., such that the evolving population is constantly challenged.  We analyzed the evolution of resistance in Escherichia coli under selection with single drugs, including chloramphenicol, doxycycline and trimethoprim.  Over a period of ∼20 days, resistance levels increased dramatically, with parallel populations showing similar phenotypic trajectories.  Whole-genome sequencing of the evolved strains identified mutations both specific to resistance to a particular drug and shared in resistance to multiple drugs.  Chloramphenicol and doxycycline resistance evolved smoothly through diverse combinations of mutations in genes involved in translation, transcription and transport.  In contrast, trimethoprim resistance evolved in a stepwise manner, through mutations restricted to the gene encoding the enzyme dihydrofolate reductase (DHFR).  Sequencing of DHFR over the time course of the expt. showed that parallel populations evolved similar mutations and acquired them in a similar order.  Whole-genome sequencing data from the morbidostat-evolved strains have been deposited in the NCBI Sequence Read Archive (SRA046097).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU4Gv27INPlrVg90H21EOLACvtfcHk0liUi2PN382-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1eisbbN&md5=f48ae03dee0860f699533b44f249fcca</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fng.1034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.1034%26sid%3Dliteratum%253Aachs%26aulast%3DToprak%26aufirst%3DE.%26aulast%3DVeres%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DJ.-B.%26aulast%3DChait%26aufirst%3DR.%26aulast%3DHartl%26aufirst%3DD.%2BL.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DEvolutionary%2520Paths%2520to%2520Antibiotic%2520Resistance%2520under%2520Dynamically%2520Sustained%2520Drug%2520Selection%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26issue%3D1%26spage%3D101%26epage%3D105%26doi%3D10.1038%2Fng.1034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Multi-Targeting by Monotherapeutic Antibacterials</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/nrd2202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnrd2202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17159922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=41-55&issue=1&author=L.+L.+Silver&title=Multi-Targeting+by+Monotherapeutic+Antibacterials&doi=10.1038%2Fnrd2202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-targeting by monotherapeutic antibacterials</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibacterial discovery research has been driven, medically, com. and intellectually, by the need for new therapeutics that are not subject to the resistance mechanisms that have evolved to combat previous generations of antibacterial agents.  This need has often been equated with the identification and exploitation of novel targets.  But efforts towards discovery and development of inhibitors of novel targets have proved frustrating.  It might be that the 'good old targets' are qual. different from the crop of all possible novel targets.  What has been learned from existing targets that can be applied to the quest for new antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpItQysyGWT8LVg90H21EOLACvtfcHk0liUi2PN382-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfN&md5=49d78749fc31f5f0546b859e17b31c47</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2202%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DMulti-Targeting%2520by%2520Monotherapeutic%2520Antibacterials%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D1%26spage%3D41%26epage%3D55%26doi%3D10.1038%2Fnrd2202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span> <span> </span><span class="NLM_article-title">Resistance-Resistant Antibiotics</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.tips.2014.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25458541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=664-674&issue=12&author=E.+Oldfieldauthor=X.+Feng&title=Resistance-Resistant+Antibiotics&doi=10.1016%2Fj.tips.2014.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance-resistant antibiotics</span></div><div class="casAuthors">Oldfield, Eric; Feng, Xinxin</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">664-674</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">New antibiotics are needed because drug resistance is increasing while the introduction of new antibiotics is decreasing.  We discuss here six possible approaches to develop 'resistance-resistant' antibiotics.  First, multitarget inhibitors in which a single compd. inhibits more than one target may be easier to develop than conventional combination therapies with two new drugs.  Second, inhibiting multiple targets in the same metabolic pathway is expected to be an effective strategy owing to synergy.  Third, discovering multiple-target inhibitors should be possible by using sequential virtual screening.  Fourth, repurposing existing drugs can lead to combinations of multitarget therapeutics.  Fifth, targets need not be proteins.  Sixth, inhibiting virulence factor formation and boosting innate immunity may also lead to decreased susceptibility to resistance.  Although it is not possible to eliminate resistance, the approaches reviewed here offer several possibilities for reducing the effects of mutations and, in some cases, suggest that sensitivity to existing antibiotics may be restored in otherwise drug-resistant organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoFcN1H0q81rVg90H21EOLACvtfcHk0liUi2PN382-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrzP&md5=1fd8f53ba0e90abbc83a6616db24568a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DOldfield%26aufirst%3DE.%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DResistance-Resistant%2520Antibiotics%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26issue%3D12%26spage%3D664%26epage%3D674%26doi%3D10.1016%2Fj.tips.2014.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">What Is an “ideal” Antibiotic? Discovery Challenges and Path Forward</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bcp.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28087253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=63-73&author=S.+B.+Singhauthor=K.+Youngauthor=L.+L.+Silver&title=What+Is+an+%E2%80%9Cideal%E2%80%9D+Antibiotic%3F+Discovery+Challenges+and+Path+Forward&doi=10.1016%2Fj.bcp.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">What is an "ideal" antibiotic? Discovery challenges and path forward</span></div><div class="casAuthors">Singh, Sheo B.; Young, Katherine; Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  An ideal antibiotic is an antibacterial agent that kills or inhibits the growth of all harmful bacteria in a host, regardless of site of infection without affecting beneficial gut microbes (gut flora) or causing undue toxicity to the host.  Sadly, no such antibiotics exist.  What exist are many effective Gram-pos. antibacterial agents as well as broad-spectrum agents that provide treatment of certain Gram-neg. bacteria but not holistic treatment of all bacteria.  However effectiveness of all antibacterial agents is being rapidly eroded due to resistance.  This viewpoint provides an overview of today's antibiotics, challenges and potential path forward of discovery and development of new (ideal) antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1-YaZx5VrMLVg90H21EOLACvtfcHk0lhbXPMQ5fr7hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChsLw%253D&md5=6cb221ed31dc85f205e2e4c28194a822</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DYoung%26aufirst%3DK.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DWhat%2520Is%2520an%2520%25E2%2580%259Cideal%25E2%2580%259D%2520Antibiotic%253F%2520Discovery%2520Challenges%2520and%2520Path%2520Forward%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D133%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.bcp.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samaddar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, J.</span></span> <span> </span><span class="NLM_article-title">Battle against Vancomycin-Resistant Bacteria: Recent Developments in Chemical Strategies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3184</span>– <span class="NLM_lpage">3205</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01093</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFagurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3184-3205&issue=7&author=G.+Dhandaauthor=P.+Sarkarauthor=S.+Samaddarauthor=J.+Haldar&title=Battle+against+Vancomycin-Resistant+Bacteria%3A+Recent+Developments+in+Chemical+Strategies&doi=10.1021%2Facs.jmedchem.8b01093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Battle against Vancomycin-Resistant Bacteria: Recent Developments in Chemical Strategies</span></div><div class="casAuthors">Dhanda, Geetika; Sarkar, Paramita; Samaddar, Sandip; Haldar, Jayanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3184-3205</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Vancomycin, a natural glycopeptide antibiotic, was used as the antibiotic of last resort for the treatment of multidrug-resistant Gram-pos. bacterial infections.  However, almost 30 years after its use, resistance to vancomycin was first reported in 1986 in France.  This became a major health concern, and alternative treatment strategies were urgently needed.  New classes of mols., including semisynthetic antibacterial compds. and newer generations of the previously used antibiotics, were developed.  Semisynthetic derivs. of vancomycin with enhanced binding affinity, membrane disruption ability, and lipid binding properties have exhibited promising results against both Gram-pos. and Gram-neg. bacteria.  Various successful approaches developed to overcome the acquired resistance in Gram-pos. bacteria, intrinsic resistance in Gram-neg. bacteria, and other forms of noninherited resistance to vancomycin have been discussed in this Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8kPOprosWa7Vg90H21EOLACvtfcHk0lhbXPMQ5fr7hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFagurvK&md5=b08a3a82c9f61c5e31af8d17af0f9d76</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01093%26sid%3Dliteratum%253Aachs%26aulast%3DDhanda%26aufirst%3DG.%26aulast%3DSarkar%26aufirst%3DP.%26aulast%3DSamaddar%26aufirst%3DS.%26aulast%3DHaldar%26aufirst%3DJ.%26atitle%3DBattle%2520against%2520Vancomycin-Resistant%2520Bacteria%253A%2520Recent%2520Developments%2520in%2520Chemical%2520Strategies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D7%26spage%3D3184%26epage%3D3205%26doi%3D10.1021%2Facs.jmedchem.8b01093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brötz-Oesterhelt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, N. A.</span></span> <span> </span><span class="NLM_article-title">How Many Modes of Action Should an Antibiotic Have?</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2008.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.coph.2008.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18621146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BD1cjhsFygsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=564-573&issue=5&author=H.+Br%C3%B6tz-Oesterheltauthor=N.+A.+Brunner&title=How+Many+Modes+of+Action+Should+an+Antibiotic+Have%3F&doi=10.1016%2Fj.coph.2008.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">How many modes of action should an antibiotic have?</span></div><div class="casAuthors">Brotz-Oesterhelt Heike; Brunner Nina A</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">564-73</span>
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    </div><div class="casAbstract">All antibiotics that have been successfully employed for decades as monotherapeutics in the treatment of bacterial infections rely on mechanisms of bacterial growth inhibition which are by far more complex than inhibition of a single enzyme.  Such successful antibiotics have in common that they address several targets in parallel and/or that their targets are encoded by multiple genes.  Such multiplicity of targets and of target genes has the advantage that the emergence of spontaneous target-related resistance is a comparatively slow process.  Recently registered antibiotics and novel antibiotics in development are discussed in the light of this promising concept of antibacterial polypharmacology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzIF5GvxYY_0HdcurJClFcfW6udTcc2eaNavrU-pSgZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjhsFygsA%253D%253D&md5=820b32bfce47dd05c0a37bf310a7884e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2008.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2008.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25B6tz-Oesterhelt%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DN.%2BA.%26atitle%3DHow%2520Many%2520Modes%2520of%2520Action%2520Should%2520an%2520Antibiotic%2520Have%253F%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2008%26volume%3D8%26issue%3D5%26spage%3D564%26epage%3D573%26doi%3D10.1016%2Fj.coph.2008.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.-S.</span></span> <span> </span><span class="NLM_article-title">4-Quinolone Hybrids and Their Antibacterial Activities</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.ejmech.2017.09.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29031077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GltrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=335-345&author=Y.-Q.+Huauthor=S.+Zhangauthor=Z.+Xuauthor=Z.-S.+Lvauthor=M.-L.+Liuauthor=L.-S.+Feng&title=4-Quinolone+Hybrids+and+Their+Antibacterial+Activities&doi=10.1016%2Fj.ejmech.2017.09.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">4-Quinolone hybrids and their antibacterial activities</span></div><div class="casAuthors">Hu, Yuan-Qiang; Zhang, Shu; Xu, Zhi; Lv, Zao-Sheng; Liu, Ming-Liang; Feng, Lian-Shun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">335-345</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The emergence and wide-spread of drug-resistant bacteria including multi-drug resistant and pan-drug resistant pathogens which are assocd. with considerable mortality, represent a significant global health threat.  4-Quinolones which are exemplified by fluoroquinolones are the second largest chemotherapy agents used in clin. practice for the treatment of various bacterial infections.  However, the resistance of bacteria to 4-quinolones develops rapidly and spreads widely throughout the world due to the long-term, inappropriate use and even abuse.  To overcome the resistance and improve the potency, several strategies have been developed.  Amongst them, mol. hybridization, which is based on the incorporation of two or more pharmacophores into a single mol. with a flexible linker, is one of the most practical approaches.  This review aims to summarize the recent advances made towards the discovery of 4-quinolone hybrids as potential antibacterial agents as well as their structure-activity relationship (SAR).  The enriched SAR may pave the way for the further rational development of 4-quinolone hybrids with excellent potency against both drug-susceptible and drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiel6w3j-bZLVg90H21EOLACvtfcHk0lj-onfnzn9ExQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GltrbM&md5=0af2128ec93ddeb3ceddc7d3fd8d2b8e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.050%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.-Q.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLv%26aufirst%3DZ.-S.%26aulast%3DLiu%26aufirst%3DM.-L.%26aulast%3DFeng%26aufirst%3DL.-S.%26atitle%3D4-Quinolone%2520Hybrids%2520and%2520Their%2520Antibacterial%2520Activities%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D141%26spage%3D335%26epage%3D345%26doi%3D10.1016%2Fj.ejmech.2017.09.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Specklin, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigwalt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, H. H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of Oxazolidinone-Quinolone Hybrids</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2319</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(03)00083-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2FS0968-0896%2803%2900083-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=12713843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFWhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=2313-2319&issue=10&author=C.+Hubschwerlenauthor=J.-L.+Specklinauthor=C.+Sigwaltauthor=S.+Schroederauthor=H.+H.+Locher&title=Design%2C+Synthesis+and+Biological+Evaluation+of+Oxazolidinone-Quinolone+Hybrids&doi=10.1016%2FS0968-0896%2803%2900083-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of oxazolidinone-quinolone hybrids</span></div><div class="casAuthors">Hubschwerlen, Christian; Specklin, Jean-Luc; Sigwalt, Christine; Schroeder, Susanne; Locher, Hans H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2313-2319</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Oxazolidinone-quinolone hybrids that combine the pharmacophores of a quinolone and an oxazolidinone were synthesized and shown to be active against a variety of resistant and susceptible Gram-pos. and fastidious Gram-neg. organisms.  The best compds. in this series overcome all types of resistance in relevant clin. Gram-pos. pathogens.  The nature of the spacer greatly influences the antibacterial activity.  The dual mode of action could be demonstrated for compds. having a piperazinyl spacer.  Antibacterial activity was higher at acidic pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEhQdn0-mi_7Vg90H21EOLACvtfcHk0lj-onfnzn9ExQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFWhsLY%253D&md5=6caee544ae6e443865da04345538241b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2803%2900083-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252803%252900083-X%26sid%3Dliteratum%253Aachs%26aulast%3DHubschwerlen%26aufirst%3DC.%26aulast%3DSpecklin%26aufirst%3DJ.-L.%26aulast%3DSigwalt%26aufirst%3DC.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Oxazolidinone-Quinolone%2520Hybrids%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26issue%3D10%26spage%3D2313%26epage%3D2319%26doi%3D10.1016%2FS0968-0896%2803%2900083-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Domalaon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idowu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhanel, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, F.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Hybrids: The Next Generation of Agents and Adjuvants against Gram-Negative Pathogens?</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">2</span>), <span class="NLM_elocation-id">e00077-17</span> <span class="refDoi"> DOI: 10.1128/CMR.00077-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1128%2FCMR.00077-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29540434" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2018&issue=2&author=R.+Domalaonauthor=T.+Idowuauthor=G.+G.+Zhanelauthor=F.+Schweizer&title=Antibiotic+Hybrids%3A+The+Next+Generation+of+Agents+and+Adjuvants+against+Gram-Negative+Pathogens%3F&doi=10.1128%2FCMR.00077-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FCMR.00077-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00077-17%26sid%3Dliteratum%253Aachs%26aulast%3DDomalaon%26aufirst%3DR.%26aulast%3DIdowu%26aufirst%3DT.%26aulast%3DZhanel%26aufirst%3DG.%2BG.%26aulast%3DSchweizer%26aufirst%3DF.%26atitle%3DAntibiotic%2520Hybrids%253A%2520The%2520Next%2520Generation%2520of%2520Agents%2520and%2520Adjuvants%2520against%2520Gram-Negative%2520Pathogens%253F%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2018%26volume%3D31%26issue%3D2%26doi%3D10.1128%2FCMR.00077-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schurig-Briccio, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lechartier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">No, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cintra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molohon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlean, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas-Junior, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennis, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crick, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span> <span> </span><span class="NLM_article-title">Multitarget Drug Discovery for Tuberculosis and Other Infectious Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3126</span>– <span class="NLM_lpage">3139</span>, <span class="refDoi"> DOI: 10.1021/jm500131s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500131s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVeqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3126-3139&issue=7&author=K.+Liauthor=L.+A.+Schurig-Briccioauthor=X.+Fengauthor=A.+Upadhyayauthor=V.+Pujariauthor=B.+Lechartierauthor=F.+L.+Fontesauthor=H.+Yangauthor=G.+Raoauthor=W.+Zhuauthor=A.+Gulatiauthor=J.+H.+Noauthor=G.+Cintraauthor=S.+Bogueauthor=Y.-L.+Liuauthor=K.+Molohonauthor=P.+Orleanauthor=D.+A.+Mitchellauthor=L.+Freitas-Juniorauthor=F.+Renauthor=H.+Sunauthor=T.+Jiangauthor=Y.+Liauthor=R.-T.+Guoauthor=S.+T.+Coleauthor=R.+B.+Gennisauthor=D.+C.+Crickauthor=E.+Oldfield&title=Multitarget+Drug+Discovery+for+Tuberculosis+and+Other+Infectious+Diseases&doi=10.1021%2Fjm500131s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget Drug Discovery for Tuberculosis and Other Infectious Diseases</span></div><div class="casAuthors">Li, Kai; Schurig-Briccio, Lici A.; Feng, Xinxin; Upadhyay, Ashutosh; Pujari, Venugopal; Lechartier, Benoit; Fontes, Fabio L.; Yang, Hongliang; Rao, Guodong; Zhu, Wei; Gulati, Anmol; No, Joo Hwan; Cintra, Giovana; Bogue, Shannon; Liu, Yi-Liang; Molohon, Katie; Orlean, Peter; Mitchell, Douglas A.; Freitas-Junior, Lucio; Ren, Feifei; Sun, Hong; Jiang, Tong; Li, Yujie; Guo, Rey-Ting; Cole, Stewart T.; Gennis, Robert B.; Crick, Dean C.; Oldfield, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3126-3139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery of a series of new drug leads that have potent activity against Mycobacterium tuberculosis as well as against other bacteria, fungi, and a malaria parasite.  The compds. are analogs of the new tuberculosis (TB) drug SQ109, which has been reported to act by inhibiting a transporter called MmpL3, involved in cell wall biosynthesis.  The authors show that SQ109 and the new compds. also target enzymes involved in menaquinone biosynthesis and electron transport, inhibiting respiration and ATP biosynthesis, and are uncouplers, collapsing the pH gradient and membrane potential used to power transporters.  The result of such multitarget inhibition is potent inhibition of TB cell growth, as well as very low rates of spontaneous drug resistance.  Several targets are absent in humans but are present in other bacteria, as well as in malaria parasites, whose growth is also inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFazutTcGaBLVg90H21EOLACvtfcHk0lj-onfnzn9ExQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVeqtLY%253D&md5=85fb16a07423e1473d64e0c41707c3db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm500131s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500131s%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DK.%26aulast%3DSchurig-Briccio%26aufirst%3DL.%2BA.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DUpadhyay%26aufirst%3DA.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DLechartier%26aufirst%3DB.%26aulast%3DFontes%26aufirst%3DF.%2BL.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DGulati%26aufirst%3DA.%26aulast%3DNo%26aufirst%3DJ.%2BH.%26aulast%3DCintra%26aufirst%3DG.%26aulast%3DBogue%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.-L.%26aulast%3DMolohon%26aufirst%3DK.%26aulast%3DOrlean%26aufirst%3DP.%26aulast%3DMitchell%26aufirst%3DD.%2BA.%26aulast%3DFreitas-Junior%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DR.-T.%26aulast%3DCole%26aufirst%3DS.%2BT.%26aulast%3DGennis%26aufirst%3DR.%2BB.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DOldfield%26aufirst%3DE.%26atitle%3DMultitarget%2520Drug%2520Discovery%2520for%2520Tuberculosis%2520and%2520Other%2520Infectious%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D7%26spage%3D3126%26epage%3D3139%26doi%3D10.1021%2Fjm500131s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lázár, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horváth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrtyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busa Fekete, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méhi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csörgő, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pósfai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekete, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szappanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kégl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span> <span> </span><span class="NLM_article-title">Bacterial Evolution of Antibiotic Hypersensitivity</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">700</span>, <span class="refDoi"> DOI: 10.1038/msb.2013.57</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fmsb.2013.57" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24169403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=700&issue=1&author=V.+L%C3%A1z%C3%A1rauthor=G.+P.+Singhauthor=R.+Spohnauthor=I.+Nagyauthor=B.+Horv%C3%A1thauthor=M.+Hrtyanauthor=R.+Busa+Feketeauthor=B.+Bogosauthor=O.+M%C3%A9hiauthor=B.+Cs%C3%B6rg%C5%91author=G.+P%C3%B3sfaiauthor=G.+Feketeauthor=B.+Szappanosauthor=B.+K%C3%A9glauthor=B.+Pappauthor=C.+P%C3%A1l&title=Bacterial+Evolution+of+Antibiotic+Hypersensitivity&doi=10.1038%2Fmsb.2013.57"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial evolution of antibiotic hypersensitivity</span></div><div class="casAuthors">Lazar, Viktoria; Pal Singh, Gajinder; Spohn, Reka; Nagy, Istvan; Horvath, Balazs; Hrtyan, Monika; Busa-Fekete, Robert; Bogos, Balazs; Mehi, Orsolya; Csorgo, Balint; Posfai, Gyorgy; Fekete, Gergely; Szappanos, Balazs; Kegl, Balazs; Papp, Balazs; Pal, Csaba</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">700</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The evolution of resistance to a single antibiotic is frequently accompanied by increased resistance to multiple other antimicrobial agents.  In sharp contrast, very little is known about the frequency and mechanisms underlying collateral sensitivity.  In this case, genetic adaptation under antibiotic stress yields enhanced sensitivity to other antibiotics.  Using large-scale lab. evolutionary expts. with Escherichia coli, we demonstrate that collateral sensitivity occurs frequently during the evolution of antibiotic resistance.  Specifically, populations adapted to aminoglycosides have an esp. low fitness in the presence of several other antibiotics.  Whole-genome sequencing of lab.-evolved strains revealed multiple mechanisms underlying aminoglycoside resistance, including a redn. in the proton-motive force (PMF) across the inner membrane.  We propose that as a side effect, these mutations diminish the activity of PMF-dependent major efflux pumps (including the AcrAB transporter), leading to hypersensitivity to several other antibiotics.  More generally, our work offers an insight into the mechanisms that drive the evolution of neg. trade-offs under antibiotic selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoVjK_fOBE87Vg90H21EOLACvtfcHk0lieXTQ_iSPZTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqsbbK&md5=73a0b2f41cf6bbedd43f67056d558cea</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2013.57&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2013.57%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25A1z%25C3%25A1r%26aufirst%3DV.%26aulast%3DSingh%26aufirst%3DG.%2BP.%26aulast%3DSpohn%26aufirst%3DR.%26aulast%3DNagy%26aufirst%3DI.%26aulast%3DHorv%25C3%25A1th%26aufirst%3DB.%26aulast%3DHrtyan%26aufirst%3DM.%26aulast%3DBusa%2BFekete%26aufirst%3DR.%26aulast%3DBogos%26aufirst%3DB.%26aulast%3DM%25C3%25A9hi%26aufirst%3DO.%26aulast%3DCs%25C3%25B6rg%25C5%2591%26aufirst%3DB.%26aulast%3DP%25C3%25B3sfai%26aufirst%3DG.%26aulast%3DFekete%26aufirst%3DG.%26aulast%3DSzappanos%26aufirst%3DB.%26aulast%3DK%25C3%25A9gl%26aufirst%3DB.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26atitle%3DBacterial%2520Evolution%2520of%2520Antibiotic%2520Hypersensitivity%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D1%26spage%3D700%26doi%3D10.1038%2Fmsb.2013.57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lázár, V.</span></span> <span> </span><span class="NLM_article-title">Collateral Sensitivity of Antibiotic-Resistant Microbes</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2015.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.tim.2015.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25818802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFehu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=401-407&issue=7&author=C.+P%C3%A1lauthor=B.+Pappauthor=V.+L%C3%A1z%C3%A1r&title=Collateral+Sensitivity+of+Antibiotic-Resistant+Microbes&doi=10.1016%2Fj.tim.2015.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Collateral sensitivity of antibiotic-resistant microbes</span></div><div class="casAuthors">Pal, Csaba; Papp, Balazs; Lazar, Viktoria</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  Understanding how evolution of microbial resistance towards a given antibiotic influences susceptibility to other drugs is a challenge of profound importance.  By combining lab. evolution, genome sequencing, and functional analyses, recent works have charted the map of evolutionary trade-offs between antibiotics and have explored the underlying mol. mechanisms.  Strikingly, mutations that caused multidrug resistance in bacteria simultaneously enhanced sensitivity to many other unrelated drugs (collateral sensitivity).  Here, we explore how this emerging research sheds new light on resistance mechanisms and the way it could be exploited for the development of alternative antimicrobial strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBgzATZDim-7Vg90H21EOLACvtfcHk0lieXTQ_iSPZTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFehu7o%253D&md5=e89a69905d95738673f331e6baa312d9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2015.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2015.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DL%25C3%25A1z%25C3%25A1r%26aufirst%3DV.%26atitle%3DCollateral%2520Sensitivity%2520of%2520Antibiotic-Resistant%2520Microbes%26jtitle%3DTrends%2520Microbiol.%26date%3D2015%26volume%3D23%26issue%3D7%26spage%3D401%26epage%3D407%26doi%3D10.1016%2Fj.tim.2015.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horinouchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furusawa, C.</span></span> <span> </span><span class="NLM_article-title">Prediction of Antibiotic Resistance by Gene Expression Profiles</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5792</span>, <span class="refDoi"> DOI: 10.1038/ncomms6792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fncomms6792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25517437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFWqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5792&author=S.+Suzukiauthor=T.+Horinouchiauthor=C.+Furusawa&title=Prediction+of+Antibiotic+Resistance+by+Gene+Expression+Profiles&doi=10.1038%2Fncomms6792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of antibiotic resistance by gene expression profiles</span></div><div class="casAuthors">Suzuki, Shingo; Horinouchi, Takaaki; Furusawa, Chikara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5792</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although many mutations contributing to antibiotic resistance have been identified, the relationship between the mutations and the related phenotypic changes responsible for the resistance has yet to be fully elucidated.  To better characterize phenotype-genotype mapping for drug resistance, here we analyze phenotypic and genotypic changes of antibiotic-resistant Escherichia coli strains obtained by lab. evolution.  We demonstrate that the resistances can be quant. predicted by the expression changes of a small no. of genes.  Several candidate mutations contributing to the resistances are identified, while phenotype-genotype mapping is suggested to be complex and includes various mutations that cause similar phenotypic changes.  The integration of transcriptome and genome data enables us to ext. essential phenotypic changes for drug resistances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM0giqmwlUxrVg90H21EOLACvtfcHk0lieXTQ_iSPZTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFWqsbk%253D&md5=4f2d72dc34ec9f323200aa3cab85f586</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fncomms6792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6792%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DHorinouchi%26aufirst%3DT.%26aulast%3DFurusawa%26aufirst%3DC.%26atitle%3DPrediction%2520of%2520Antibiotic%2520Resistance%2520by%2520Gene%2520Expression%2520Profiles%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D5792%26doi%3D10.1038%2Fncomms6792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Interactions That Select against Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>446</i></span> (<span class="NLM_issue">7136</span>),  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1038/nature05685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature05685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17410176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjslOlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=446&publication_year=2007&pages=668-671&issue=7136&author=R.+Chaitauthor=A.+Craneyauthor=R.+Kishony&title=Antibiotic+Interactions+That+Select+against+Resistance&doi=10.1038%2Fnature05685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic interactions that select against resistance</span></div><div class="casAuthors">Chait, Remy; Craney, Allison; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">446</span>
        (<span class="NLM_cas:issue">7136</span>),
    <span class="NLM_cas:pages">668-671</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Multidrug combinations are increasingly important in combating the spread of antibiotic-resistance in bacterial pathogens.  On a broader scale, such combinations are also important in understanding microbial ecol. and evolution.  Although the effects of multidrug combinations on bacterial growth have been studied extensively, relatively little is known about their impact on the differential selection between sensitive and resistant bacterial populations.  Normally, the presence of a drug confers an advantage on its resistant mutants in competition with the sensitive wild-type population.  Here the authors show, by a direct competition assay between doxycycline-resistant and doxycycline-sensitive Escherichia coli, that this differential selection can be inverted in a hyper-antagonistic class of drug combinations.  Used in such a combination, a drug can render the combined treatment selective against the drug's own resistance allele.  Further, this inversion of selection seems largely insensitive to the underlying resistance mechanism and occurs, at sublethal concns., while maintaining inhibition of the wild type.  These seemingly paradoxical results can be rationalized in terms of a simple geometric argument.  These findings demonstrate a previously unappreciated feature of the fitness landscape for the evolution of resistance and point to a trade-off between the effect of drug interactions on abs. potency and the relative competitive selection that they impose on emerging resistant populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAA0PXHRu6TrVg90H21EOLACvtfcHk0lgyTfV2n0nNDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjslOlsbo%253D&md5=d4f9acd88af10905e4e423efc4e31916</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature05685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05685%26sid%3Dliteratum%253Aachs%26aulast%3DChait%26aufirst%3DR.%26aulast%3DCraney%26aufirst%3DA.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DAntibiotic%2520Interactions%2520That%2520Select%2520against%2520Resistance%26jtitle%3DNature%26date%3D2007%26volume%3D446%26issue%3D7136%26spage%3D668%26epage%3D671%26doi%3D10.1038%2Fnature05685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Drug Interactions Modulate the Potential for Evolution of Resistance</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span> (<span class="NLM_issue">39</span>),  <span class="NLM_fpage">14918</span>– <span class="NLM_lpage">14923</span>, <span class="refDoi"> DOI: 10.1073/pnas.0800944105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0800944105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18815368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1aqur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=14918-14923&issue=39&author=J.-B.+Michelauthor=P.+J.+Yehauthor=R.+Chaitauthor=R.+C.+Moelleringauthor=R.+Kishony&title=Drug+Interactions+Modulate+the+Potential+for+Evolution+of+Resistance&doi=10.1073%2Fpnas.0800944105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Drug interactions modulate the potential for evolution of resistance</span></div><div class="casAuthors">Michel, Jean-Baptiste; Yeh, Pamela J.; Chait, Remy; Moellering, Robert C., Jr.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">14918-14923</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Antimicrobial treatments increasingly rely on multidrug combinations, in part because of the emergence and spread of antibiotic resistance.  The continued effectiveness of combination treatments depends crucially on the frequency with which multidrug resistance arises.  Yet, it is unknown how this propensity for resistance depends on cross-resistance and on epistatic interactions-ranging from synergy to antagonism-between the drugs.  Here, we analyzed how interactions between pairs of drugs affect the spontaneous emergence of resistance in the medically important pathogen Staphylococcus aureus.  Resistance is selected for within a window of drug concns. high enough to inhibit wild-type growth but low enough for some resistant mutants to grow.  Introducing an exptl. method for high-throughput colony imaging, we counted resistant colonies arising across a two-dimensional matrix of drug concns. for each of three drug pairs.  Our data show that these different drug combinations have significantly different impacts on the size of the window of drug concns. where resistance is selected for.  We framed these results in a math. model in which the frequencies of resistance to single drugs, cross-resistance, and epistasis combine to det. the propensity for multidrug resistance.  The theory suggests that drug pairs which interact synergistically, preferred for their immediate efficacy, may in fact favor the future evolution of resistance.  This framework reveals the central role of drug epistasis in the evolution of resistance and points to new strategies for combating the emergence of drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJqt7bUFSpcbVg90H21EOLACvtfcHk0lgyTfV2n0nNDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1aqur%252FO&md5=69c1441ff4e1fec2180c5e281da19551</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0800944105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0800944105%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.-B.%26aulast%3DYeh%26aufirst%3DP.%2BJ.%26aulast%3DChait%26aufirst%3DR.%26aulast%3DMoellering%26aufirst%3DR.%2BC.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DDrug%2520Interactions%2520Modulate%2520the%2520Potential%2520for%2520Evolution%2520of%2520Resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D39%26spage%3D14918%26epage%3D14923%26doi%3D10.1073%2Fpnas.0800944105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Site Structure Influences the Evolution of Protein Complex Function and Topology</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3265</span>– <span class="NLM_lpage">3276</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.02.085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.celrep.2018.02.085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29562182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlsV2ntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=3265-3276&issue=12&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Ligand+Binding+Site+Structure+Influences+the+Evolution+of+Protein+Complex+Function+and+Topology&doi=10.1016%2Fj.celrep.2018.02.085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Site Structure Influences the Evolution of Protein Complex Function and Topology</span></div><div class="casAuthors">Abrusan, Gyorgy; Marsh, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3265-3276</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">It has been suggested that the evolution of protein complexes is significantly influenced by stochastic, non-adaptive processes.  Using ligand binding as a proxy of function, we show that the structure of ligand-binding sites significantly influences the evolution of protein complexes.  We show that homomers with multi-chain binding sites (MBSs) evolve new functions slower than monomers or other homomers, and those binding cofactors and metals have more conserved quaternary structure than other homomers.  Moreover, the ligands and ligand-binding pockets of homologous MBS homomers are more similar than monomers and other homomers.  Our results suggest strong evolutionary selection for quaternary structure in cofactor-binding MBS homomers, whereas neutral processes are more important in complexes with single-chain binding sites.  They also have pharmacol. implications, suggesting that complexes with single-chain binding sites are better targets for selective drugs, whereas MBS homomers are good candidates for broad-spectrum antibiotic and multitarget drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogMHthIv-ZirVg90H21EOLACvtfcHk0lgyTfV2n0nNDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlsV2ntLg%253D&md5=40f5a6898caa52e8b1af62d96fb180ee</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.02.085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.02.085%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520Binding%2520Site%2520Structure%2520Influences%2520the%2520Evolution%2520of%2520Protein%2520Complex%2520Function%2520and%2520Topology%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D22%26issue%3D12%26spage%3D3265%26epage%3D3276%26doi%3D10.1016%2Fj.celrep.2018.02.085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span> <span> </span><span class="NLM_article-title">The Evolution of Multimeric Protein Assemblages</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1366</span>, <span class="refDoi"> DOI: 10.1093/molbev/msr300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsr300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22144639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1yhu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=1353-1366&issue=5&author=M.+Lynch&title=The+Evolution+of+Multimeric+Protein+Assemblages&doi=10.1093%2Fmolbev%2Fmsr300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Evolution of Multimeric Protein Assemblages</span></div><div class="casAuthors">Lynch, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology and Evolution</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1353-1366</span>CODEN:
                <span class="NLM_cas:coden">MBEVEO</span>;
        ISSN:<span class="NLM_cas:issn">0737-4038</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the mechanisms by which complex cellular features evolve constitute one of the great unsolved problems of evolutionary biol., it is clear that the emergence of new protein-protein interactions, often accompanied by the diversification of duplicate genes, is involved.  Using information on the levels of protein multimerization in major phylogenetic groups as a guide to the patterns that must be explained and relying on results from population-genetic theory to define the relative plausibility of alternative evolutionary pathways, a framework for understanding the evolution of dimers is developed.  The resultant theory demonstrates that the likelihoods of alternative pathways for the emergence of protein complexes depend strongly on the effective population size.  Nonetheless, it is equally clear that further advancements in this area will require comparative studies on the fitness consequences of alternative monomeric and dimeric proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHo6eOvzfwdbVg90H21EOLACvtfcHk0lgyTfV2n0nNDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1yhu7c%253D&md5=debb2b74024d78175fa82eecad1f8f03</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsr300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsr300%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DM.%26atitle%3DThe%2520Evolution%2520of%2520Multimeric%2520Protein%2520Assemblages%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2012%26volume%3D29%26issue%3D5%26spage%3D1353%26epage%3D1366%26doi%3D10.1093%2Fmolbev%2Fmsr300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klabunde, T.</span></span> <span> </span><span class="NLM_article-title">Chemogenomic Approaches to Drug Discovery: Similar Receptors Bind Similar Ligands</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0707308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&issue=1&author=T.+Klabunde&title=Chemogenomic+Approaches+to+Drug+Discovery%3A+Similar+Receptors+Bind+Similar+Ligands&doi=10.1038%2Fsj.bjp.0707308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0limEiNLE660Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520Approaches%2520to%2520Drug%2520Discovery%253A%2520Similar%2520Receptors%2520Bind%2520Similar%2520Ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26issue%3D1%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fsj.bjp.0707308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Alpha Helices Are More Robust to Mutations than Beta Strands</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>), <span class="NLM_elocation-id">e1005242</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1371%2Fjournal.pcbi.1005242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27935949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1ansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&issue=12&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Alpha+Helices+Are+More+Robust+to+Mutations+than+Beta+Strands&doi=10.1371%2Fjournal.pcbi.1005242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha helices are more robust to mutations than beta strands</span></div><div class="casAuthors">Abrusan, Gyoergy; Marsh, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1005242/1-e1005242/16</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The rapidly increasing amt. of data on human genetic variation has resulted in a growing demand to identify pathogenic mutations computationally, as their exptl. validation is currently beyond reach.  Here we show that alpha helixes and beta strands differ significantly in their ability to tolerate mutations: helixes can accumulate more mutations than strands without change, due to the higher nos. of inter-residue contacts in helixes.  This results in two patterns: a) the same no. of mutations causes less structural change in helixes than in strands; b) helixes diverge more rapidly in sequence than strands within the same domains.  Addnl., both helixes and strands are significantly more robust than coils.  Based on this observation we show that human missense mutations that change secondary structure are more likely to be pathogenic than those that do not.  Moreover, inclusion of predicted secondary structure changes shows significant utility for improving upon state-of-theart pathogenicity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqavBv_HbqhX7Vg90H21EOLACvtfcHk0limEiNLE660Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1ansbs%253D&md5=5f114109a3737aa9075370a938cc2c47</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005242%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DAlpha%2520Helices%2520Are%2520More%2520Robust%2520to%2520Mutations%2520than%2520Beta%2520Strands%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26doi%3D10.1371%2Fjournal.pcbi.1005242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">BioLiP: A Semi-Manually Curated Database for Biologically Relevant Ligand-protein Interactions</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1096</span>– <span class="NLM_lpage">D1103</span>, <span class="refDoi"> DOI: 10.1093/nar/gks966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgks966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=23087378" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=D1096-D1103&issue=D1&author=J.+Yangauthor=A.+Royauthor=Y.+Zhang&title=BioLiP%3A+A+Semi-Manually+Curated+Database+for+Biologically+Relevant+Ligand-protein+Interactions&doi=10.1093%2Fnar%2Fgks966"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks966%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DBioLiP%253A%2520A%2520Semi-Manually%2520Curated%2520Database%2520for%2520Biologically%2520Relevant%2520Ligand-protein%2520Interactions%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D41%26issue%3DD1%26spage%3DD1096%26epage%3DD1103%26doi%3D10.1093%2Fnar%2Fgks966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Algorithm for Detection of Structurally Similar Protein Binding Sites by Local Structural Alignment</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1168</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&issue=9&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Algorithm+for+Detection+of+Structurally+Similar+Protein+Binding+Sites+by+Local+Structural+Alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0limEiNLE660Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Algorithm%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%2520by%2520Local%2520Structural%2520Alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D9%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Tools (algorithm, Database, and Web Servers) for Predicting and Modeling of Biologically Interesting Proteins</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2017.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.pbiomolbio.2017.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28212856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2017&pages=24-32&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Tools+%28algorithm%2C+Database%2C+and+Web+Servers%29+for+Predicting+and+Modeling+of+Biologically+Interesting+Proteins&doi=10.1016%2Fj.pbiomolbio.2017.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS tools (algorithm, database, and web servers) for predicting and modeling of biologically interesting proteins</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-32</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">ProBiS (Protein Binding Sites) Tools consist of algorithm, database, and web servers for prediction of binding sites and protein ligands based on the detection of structurally similar binding sites in the Protein Data Bank.  In this article, we review the operations that ProBiS Tools perform, provide comments on the evolution of the tools, and give some implementation details.  We review some of its applications to biol. interesting proteins.  ProBiS Tools are freely available at http://probis.cmm.ki.si and http://probis.nih.gov.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmqisRzdGSrVg90H21EOLACvtfcHk0lgruzWDjxkXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemtLo%253D&md5=2a34bc81fb14bce843f917f496383ee5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2017.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2017.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Tools%2520%2528algorithm%252C%2520Database%252C%2520and%2520Web%2520Servers%2529%2520for%2520Predicting%2520and%2520Modeling%2520of%2520Biologically%2520Interesting%2520Proteins%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D128%26spage%3D24%26epage%3D32%26doi%3D10.1016%2Fj.pbiomolbio.2017.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">DOCK 6: Impact of New Features and Current Docking Performance</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1156</span>, <span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&issue=15&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+New+Features+and+Current+Docking+Performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0lgruzWDjxkXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520New%2520Features%2520and%2520Current%2520Docking%2520Performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26issue%3D15%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of DOCK 6 as a Pose Generation and Database Enrichment Tool</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">773</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9565-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1007%2Fs10822-012-9565-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22569593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=749-773&issue=6&author=S.+R.+Brozellauthor=S.+Mukherjeeauthor=T.+E.+Baliusauthor=D.+R.+Roeauthor=D.+A.+Caseauthor=R.+C.+Rizzo&title=Evaluation+of+DOCK+6+as+a+Pose+Generation+and+Database+Enrichment+Tool&doi=10.1007%2Fs10822-012-9565-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of DOCK 6 as a pose generation and database enrichment tool</span></div><div class="casAuthors">Brozell, Scott R.; Mukherjee, Sudipto; Balius, Trent E.; Roe, Daniel R.; Case, David A.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">749-773</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In conjunction with the recent American Chem. Society symposium titled "Docking and Scoring: A Review of Docking Programs" the performance of the DOCK6 program was evaluated through (1) pose reprodn. and (2) database enrichment calcns. on a common set of organizer-specified systems and datasets (ASTEX, DUD, WOMBAT).  Representative baseline grid score results averaged over five docking runs yield a relatively high pose identification success rate of 72.5 % (symmetry cor. rmsd) and sampling rate of 91.9 % for the multi site ASTEX set (N = 147) using organizer-supplied structures.  Numerous addnl. docking expts. showed that ligand starting conditions, symmetry, multiple binding sites, clustering, and receptor prepn. protocols all affect success.  Encouragingly, in some cases, use of more sophisticated scoring and sampling methods yielded results which were comparable (Amber score ligand movable protocol) or exceeded (LMOD score) analogous baseline grid-score results.  The anal. highlights the potential benefit and challenges assocd. with including receptor flexibility and indicates that different scoring functions have system dependent strengths and weaknesses.  Enrichment studies with the DUD database prepd. using the SB2010 prepn. protocol and native ligand pairings yielded individual area under the curve (AUC) values derived from receiver operating characteristic curve anal. ranging from 0.29 (bad enrichment) to 0.96 (good enrichment) with an av. value of 0.60 (27/38 have AUC ≥ 0.5).  Strong early enrichment was also obsd. in the critically important 1.0-2.0 % region.  Somewhat surprisingly, an alternative receptor prepn. protocol yielded comparable results.  As expected, semi-random pairings yielded poorer enrichments, in particular, for unrelated receptors.  Overall, the breadth and no. of expts. performed provide a useful snapshot of current capabilities of DOCK6 as well as starting points to guide future development efforts to further improve sampling and scoring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYqYQ0o4s9srVg90H21EOLACvtfcHk0lgruzWDjxkXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nP&md5=6888c35b532535b2a8ac5f556140878d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9565-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9565-y%26sid%3Dliteratum%253Aachs%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DEvaluation%2520of%2520DOCK%25206%2520as%2520a%2520Pose%2520Generation%2520and%2520Database%2520Enrichment%2520Tool%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26issue%3D6%26spage%3D749%26epage%3D773%26doi%3D10.1007%2Fs10822-012-9565-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fochtman, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Customizable de Novo Design Strategies for DOCK: Application to HIVgp41 and Other Therapeutic Targets</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">2641</span>– <span class="NLM_lpage">2663</span>, <span class="refDoi"> DOI: 10.1002/jcc.25052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.25052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28940386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFClu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=2641-2663&issue=30&author=W.+J.+Allenauthor=B.+C.+Fochtmanauthor=T.+E.+Baliusauthor=R.+C.+Rizzo&title=Customizable+de+Novo+Design+Strategies+for+DOCK%3A+Application+to+HIVgp41+and+Other+Therapeutic+Targets&doi=10.1002%2Fjcc.25052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Customizable de novo design strategies for DOCK: Application to HIVgp41 and other therapeutic targets</span></div><div class="casAuthors">Allen, William J.; Fochtman, Brian C.; Balius, Trent E.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">2641-2663</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">De novo design can be used to explore vast areas of chem. space in computational lead discovery.  As a complement to virtual screening, from-scratch construction of mols. is not limited to compds. in pre-existing vendor catalogs.  Here, we present an iterative fragment growth method, integrated into the program DOCK, in which new mols. are built using rules for allowable connections based on known mols.  The method leverages DOCK's advanced scoring and pruning approaches and users can define very specific criteria in terms of properties or features to customize growth toward a particular region of chem. space.  The code was validated using three increasingly difficult classes of calcns.: (1) Rebuilding known X-ray ligands taken from 663 complexes using only their component parts (focused libraries), (2) construction of new ligands in 57 drug target sites using a library derived from ∼13M drug-like compds. (generic libraries), and (3) application to a challenging protein-protein interface on the viral drug target HIVgp41.  The computational testing confirms that the de novo DOCK routines are robust and working as envisioned, and the compelling results highlight the potential utility for designing new mols. against a wide variety of important protein targets. © 2017 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMx6MoUXyTU7Vg90H21EOLACvtfcHk0lgruzWDjxkXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFClu7fE&md5=807e8526b3c314e6c02e9f91e16ea5e3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fjcc.25052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.25052%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DFochtman%26aufirst%3DB.%2BC.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DCustomizable%2520de%2520Novo%2520Design%2520Strategies%2520for%2520DOCK%253A%2520Application%2520to%2520HIVgp41%2520and%2520Other%2520Therapeutic%2520Targets%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2017%26volume%3D38%26issue%3D30%26spage%3D2641%26epage%3D2663%26doi%3D10.1002%2Fjcc.25052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: A Major Update to the DrugBank Database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&issue=D1&author=D.%0AS.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+A+Major+Update+to+the+DrugBank+Database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lj5WJguQRI9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520A%2520Major%2520Update%2520to%2520the%2520DrugBank%2520Database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">ZINC 15 - Ligand Discovery for Everyone</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2324</span>– <span class="NLM_lpage">2337</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&issue=11&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+-+Ligand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lj5WJguQRI9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520-%2520Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D11%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical Properties of Antibacterial Compounds: Implications for Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&issue=10&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+Properties+of+Antibacterial+Compounds%3A+Implications+for+Drug+Discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0lj5WJguQRI9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520Properties%2520of%2520Antibacterial%2520Compounds%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D10%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Bombarelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvenaud, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Lobato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Lengeling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheberla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera-Iparraguirre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirzel, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span> <span> </span><span class="NLM_article-title">Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntlWquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=268-276&author=R.+G%C3%B3mez-Bombarelliauthor=J.+N.+Weiauthor=D.+Duvenaudauthor=J.+M.+Hern%C3%A1ndez-Lobatoauthor=B.+S%C3%A1nchez-Lengelingauthor=D.+Sheberlaauthor=J.+Aguilera-Iparraguirreauthor=T.+D.+Hirzelauthor=R.+P.+Adamsauthor=A.+Aspuru-Guzik&title=Automatic+Chemical+Design+Using+a+Data-Driven+Continuous+Representation+of+Molecules&doi=10.1021%2Facscentsci.7b00572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules</span></div><div class="casAuthors">Gomez-Bombarelli, Rafael; Wei, Jennifer N.; Duvenaud, David; Hernandez-Lobato, Jose Miguel; Sanchez-Lengeling, Benjamin; Sheberla, Dennis; Aguilera-Iparraguirre, Jorge; Hirzel, Timothy D.; Adams, Ryan P.; Aspuru-Guzik, Alan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">268-276</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a method to convert discrete representations of mols. to and from a multidimensional continuous representation.  This model allows us to generate new mols. for efficient exploration and optimization through open-ended spaces of chem. compds.  A deep neural network was trained on hundreds of thousands of existing chem. structures to construct three coupled functions: an encoder, a decoder, and a predictor.  The encoder converts the discrete representation of a mol. into a real-valued continuous vector, and the decoder converts these continuous vectors back to discrete mol. representations.  The predictor ests. chem. properties from the latent continuous vector representation of the mol.  Continuous representations of mols. allow us to automatically generate novel chem. structures by performing simple operations in the latent space, such as decoding random vectors, perturbing known chem. structures, or interpolating between mols.  Continuous representations also allow the use of powerful gradient-based optimization to efficiently guide the search for optimized functional compds.  We demonstrate our method in the domain of drug-like mols. and also in a set of mols. with fewer that nine heavy atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXN9RuDQ2cp7Vg90H21EOLACvtfcHk0lggKRxOXK4JAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntlWquw%253D%253D&md5=322d9ff569fc9c8831e91d915104d985</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00572%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B3mez-Bombarelli%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DJ.%2BN.%26aulast%3DDuvenaud%26aufirst%3DD.%26aulast%3DHern%25C3%25A1ndez-Lobato%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1nchez-Lengeling%26aufirst%3DB.%26aulast%3DSheberla%26aufirst%3DD.%26aulast%3DAguilera-Iparraguirre%26aufirst%3DJ.%26aulast%3DHirzel%26aufirst%3DT.%2BD.%26aulast%3DAdams%26aufirst%3DR.%2BP.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26atitle%3DAutomatic%2520Chemical%2520Design%2520Using%2520a%2520Data-Driven%2520Continuous%2520Representation%2520of%2520Molecules%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26spage%3D268%26epage%3D276%26doi%3D10.1021%2Facscentsci.7b00572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">Estimation of Synthetic Accessibility Score of Drug-like Molecules Based on Molecular Complexity and Fragment Contributions</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">8</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-1-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1758-2946-1-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=8&author=P.+Ertlauthor=A.+Schuffenhauer&title=Estimation+of+Synthetic+Accessibility+Score+of+Drug-like+Molecules+Based+on+Molecular+Complexity+and+Fragment+Contributions&doi=10.1186%2F1758-2946-1-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span></div><div class="casAuthors">Ertl Peter; Schuffenhauer Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A method to estimate ease of synthesis (synthetic accessibility) of drug-like molecules is needed in many areas of the drug discovery process.  The development and validation of such a method that is able to characterize molecule synthetic accessibility as a score between 1 (easy to make) and 10 (very difficult to make) is described in this article.  RESULTS:  The method for estimation of the synthetic accessibility score (SAscore) described here is based on a combination of fragment contributions and a complexity penalty.  Fragment contributions have been calculated based on the analysis of one million representative molecules from PubChem and therefore one can say that they capture historical synthetic knowledge stored in this database.  The molecular complexity score takes into account the presence of non-standard structural features, such as large rings, non-standard ring fusions, stereocomplexity and molecule size.  The method has been validated by comparing calculated SAscores with ease of synthesis as estimated by experienced medicinal chemists for a set of 40 molecules.  The agreement between calculated and manually estimated synthetic accessibility is very good with r2 = 0.89.  CONCLUSION:  A novel method to estimate synthetic accessibility of molecules has been developed.  This method uses historical synthetic knowledge obtained by analyzing information from millions of already synthesized chemicals and considers also molecule complexity.  The method is sufficiently fast and provides results consistent with estimation of ease of synthesis by experienced medicinal chemists.  The calculated SAscore may be used to support various drug discovery processes where a large number of molecules needs to be ranked based on their synthetic accessibility, for example when purchasing samples for screening, selecting hits from high-throughput screening for follow-up, or ranking molecules generated by various de novo design approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQW_MJ6K-6QK9K0qsdY6Nc1fW6udTcc2eaHPZAJhY7BAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D&md5=9ffc064a9d3952a583532411cb0f0673</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-8%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DEstimation%2520of%2520Synthetic%2520Accessibility%2520Score%2520of%2520Drug-like%2520Molecules%2520Based%2520on%2520Molecular%2520Complexity%2520and%2520Fragment%2520Contributions%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-1-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muresan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Quantifying the Chemical Beauty of Drugs</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">90</span>, <span class="refDoi"> DOI: 10.1038/nchem.1243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnchem.1243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22270643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=90&issue=2&author=G.+R.+Bickertonauthor=G.+V.+Paoliniauthor=J.+Besnardauthor=S.+Muresanauthor=A.+L.+Hopkins&title=Quantifying+the+Chemical+Beauty+of+Drugs&doi=10.1038%2Fnchem.1243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chemical beauty of drugs</span></div><div class="casAuthors">Bickerton, G. Richard; Paolini, Gaia V.; Besnard, Jeremy; Muresan, Sorel; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">90-98</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drug-likeness is a key consideration when selecting compds. during the early stages of drug discovery.  However, evaluation of drug-likeness in abs. terms does not reflect adequately the whole spectrum of compd. quality.  More worryingly, widely used rules may inadvertently foster undesirable mol. property inflation as they permit the encroachment of rule-compliant compds. towards their boundaries.  We propose a measure of drug-likeness based on the concept of desirability called the quant. est. of drug-likeness (QED).  The empirical rationale of QED reflects the underlying distribution of mol. properties.  QED is intuitive, transparent, straightforward to implement in many practical settings and allows compds. to be ranked by their relative merit.  We extended the utility of QED by applying it to the problem of mol. target druggability assessment by prioritizing a large set of published bioactive compds.  The measure may also capture the abstr. notion of aesthetics in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCb7vrfeUodrVg90H21EOLACvtfcHk0lggKRxOXK4JAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D&md5=7ab125d4ab381924f965d072695f7432</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1243%26sid%3Dliteratum%253Aachs%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DMuresan%26aufirst%3DS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DQuantifying%2520the%2520Chemical%2520Beauty%2520of%2520Drugs%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26issue%3D2%26spage%3D90%26doi%3D10.1038%2Fnchem.1243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
der Maaten, L.</span></span> <span> </span><span class="NLM_article-title">Accelerating T-SNE Using Tree-Based Algorithms</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3221</span>– <span class="NLM_lpage">3245</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3221-3245&author=L.+Van%0Ader+Maaten&title=Accelerating+T-SNE+Using+Tree-Based+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BMaaten%26aufirst%3DL.%26atitle%3DAccelerating%2520T-SNE%2520Using%2520Tree-Based%2520Algorithms%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D3221%26epage%3D3245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasiri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otero, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermoso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Antibiotic (E)-3-(3-Carboxyphenyl)-2-(4-Cyanostyryl)quinazolin-4(3H)-One</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1738</span>– <span class="NLM_lpage">1741</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b00056</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b00056" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12lsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=1738-1741&issue=5&author=R.+Bouleyauthor=M.+Kumarasiriauthor=Z.+Pengauthor=L.+H.+Oteroauthor=W.+Songauthor=M.+A.+Suckowauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=E.+Lastochkinauthor=N.+T.+Antunesauthor=H.+Piauthor=S.+Vakulenkoauthor=J.+A.+Hermosoauthor=M.+Changauthor=S.+Mobashery&title=Discovery+of+Antibiotic+%28E%29-3-%283-Carboxyphenyl%29-2-%284-Cyanostyryl%29quinazolin-4%283H%29-One&doi=10.1021%2Fjacs.5b00056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Antibiotic (E)-3-(3-Carboxyphenyl)-2-(4-cyanostyryl)quinazolin-4(3H)-one</span></div><div class="casAuthors">Bouley, Renee; Kumarasiri, Malika; Peng, Zhihong; Otero, Lisandro H.; Song, Wei; Suckow, Mark A.; Schroeder, Valerie A.; Wolter, William R.; Lastochkin, Elena; Antunes, Nuno T.; Pi, Hualiang; Vakulenko, Sergei; Hermoso, Juan A.; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1738-1741</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the face of the clin. challenge posed by resistant bacteria, the present needs for novel classes of antibiotics are genuine.  In silico docking and screening, followed by chem. synthesis of a library of quinazolinones, led to the discovery of (E)-3-(3-carboxyphenyl)-2-(4-cyanostyryl)quinazolin-4(3H)-one (compd. 2) as an antibiotic effective in vivo against methicillin-resistant Staphylococcus aureus (MRSA).  This antibiotic impairs cell-wall biosynthesis as documented by functional assays, showing binding of 2 to penicillin-binding protein (PBP) 2a.  We document that the antibiotic also inhibits PBP1 of S. aureus, indicating a broad targeting of structurally similar PBPs by this antibiotic.  This class of antibiotics holds promise in fighting MRSA infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7oFjd7fxCHrVg90H21EOLACvtfcHk0liOu2T7ok349A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12lsb8%253D&md5=c03bc141d73ee85c387fb4be6499a945</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b00056%26sid%3Dliteratum%253Aachs%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DKumarasiri%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DOtero%26aufirst%3DL.%2BH.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DHermoso%26aufirst%3DJ.%2BA.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Antibiotic%2520%2528E%2529-3-%25283-Carboxyphenyl%2529-2-%25284-Cyanostyryl%2529quinazolin-4%25283H%2529-One%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26issue%3D5%26spage%3D1738%26epage%3D1741%26doi%3D10.1021%2Fjacs.5b00056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for the 4(3H)-Quinazolinone Antibacterials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5011</span>– <span class="NLM_lpage">5021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00372</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00372" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtF2lsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5011-5021&issue=10&author=R.+Bouleyauthor=D.+Dingauthor=Z.+Pengauthor=M.+Bastianauthor=E.+Lastochkinauthor=W.+Songauthor=M.+A.+Suckowauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=S.+Mobasheryauthor=M.+Chang&title=Structure-Activity+Relationship+for+the+4%283H%29-Quinazolinone+Antibacterials&doi=10.1021%2Facs.jmedchem.6b00372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for the 4(3H)-Quinazolinone Antibacterials</span></div><div class="casAuthors">Bouley, Renee; Ding, Derong; Peng, Zhihong; Bastian, Maria; Lastochkin, Elena; Song, Wei; Suckow, Mark A.; Schroeder, Valerie A.; Wolter, William R.; Mobashery, Shahriar; Chang, Mayland</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5011-5021</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently reported on the discovery of a novel antibacterial (2) with a 4(3H)-quinazolinone core.  This discovery was made by in silico screening of 1.2 million compds. for binding to a penicillin-binding protein and the subsequent demonstration of antibacterial activity against Staphylococcus aureus.  The first structure-activity relationship for this antibacterial scaffold is explored in this report with evaluation of 77 variants of the structural class.  Eleven promising compds. were further evaluated for in vitro toxicity, pharmacokinetics, and efficacy in a mouse peritonitis model of infection, which led to the discovery of compd. 27.  This new quinazolinone has potent activity against methicillin-resistant (MRSA) strains, low clearance, oral bioavailability and shows efficacy in a mouse neutropenic thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoES54kEeDf6bVg90H21EOLACvtfcHk0liOu2T7ok349A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtF2lsr4%253D&md5=9287f7924141b19a517b3ec04d723eff</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00372%26sid%3Dliteratum%253Aachs%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DBastian%26aufirst%3DM.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DMobashery%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520the%25204%25283H%2529-Quinazolinone%2520Antibacterials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D10%26spage%3D5011%26epage%3D5021%26doi%3D10.1021%2Facs.jmedchem.6b00372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gour, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malasala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borra, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of 1,2,3-Triazole Linked 4(3H)-Quinazolinones as Potent Antibacterial Agents against Multidrug-Resistant Staphylococcus Aureus</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1056</span>– <span class="NLM_lpage">1067</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.ejmech.2018.08.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=30176536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OmsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=1056-1067&author=S.+Gatadiauthor=J.+Gourauthor=M.+Shuklaauthor=G.+Kaulauthor=S.+Dasauthor=A.+Dasguptaauthor=S.+Malasalaauthor=R.+S.+Borraauthor=Y.+V.+Madhaviauthor=S.+Chopraauthor=S.+Nanduri&title=Synthesis+of+1%2C2%2C3-Triazole+Linked+4%283H%29-Quinazolinones+as+Potent+Antibacterial+Agents+against+Multidrug-Resistant+Staphylococcus+Aureus&doi=10.1016%2Fj.ejmech.2018.08.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of 1,2,3-triazole linked 4(3H)-Quinazolinones as potent antibacterial agents against multidrug-resistant Staphylococcus aureus</span></div><div class="casAuthors">Gatadi, Srikanth; Gour, Jitendra; Shukla, Manjulika; Kaul, Grace; Das, Swetarka; Dasgupta, Arunava; Malasala, Satyaveni; Borra, Ramya Sri; Madhavi, Y. V.; Chopra, Sidharth; Nanduri, Srinivas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1056-1067</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Methicillin and vancomycin resistant Staphylococcus aureus infections are an emerging global health concern leading to increasing morbidity and mortality.  Continuous increase in drug resistance has underlined the need for discovery and development of new antibacterial agents acting via novel mechanisms to overcome this pressing issue.  In this context, a no. of 1,2,3-triazole linked 4(3H)-quinazolinone derivs. were designed and synthesized as potent antibacterial agents.  When evaluated against ESKAP pathogen panel, a wide variety of substituted compds. exhibited significantly selective inhibitory activities towards Staphylococcus aureus (MIC = 0.5-4 μg/mL).  To understand and confirm the specificity of these compds., the compds. I and II were tested against E. coli and A. baumannii in combination with sub-lethal concns. of Polymyxin B nonapeptide (PMBN) and were found to be inactive.  This clearly indicated that these compds. possess specific and potent activity towards S. aureus and are inactive against gram-neg. pathogens.  Encouragingly, the compds. were also found to be non toxic to Vero cells and displayed favorable selectivity index (SI = 40 to 80).  Furthermore, I and II were found to possess potent inhibitory activity when tested against multidrug resistant S. aureus including strains resistant to vancomycin (MIC values 0.5-32 μg/mL), indicating that the compds. are able to escape current drug-resistance mechanisms.  With the potent anti-bacterial activity exhibited the new series of 1,2,3-triazole linked 4(3H)-quinazolinones have emerged as promising candidates for treating multidrug resistant Staphylococcus aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDsXYciQOosbVg90H21EOLACvtfcHk0liOu2T7ok349A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OmsLrO&md5=10d0b1f98545762afd4df0dbabd25e41</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.070%26sid%3Dliteratum%253Aachs%26aulast%3DGatadi%26aufirst%3DS.%26aulast%3DGour%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DKaul%26aufirst%3DG.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DMalasala%26aufirst%3DS.%26aulast%3DBorra%26aufirst%3DR.%2BS.%26aulast%3DMadhavi%26aufirst%3DY.%2BV.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DS.%26atitle%3DSynthesis%2520of%25201%252C2%252C3-Triazole%2520Linked%25204%25283H%2529-Quinazolinones%2520as%2520Potent%2520Antibacterial%2520Agents%2520against%2520Multidrug-Resistant%2520Staphylococcus%2520Aureus%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D1056%26epage%3D1067%26doi%3D10.1016%2Fj.ejmech.2018.08.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gour, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akunuri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of New 3-Phenylquinazolin-4(3H)-One Derivatives as Potent Antibacterial Agents Effective against Methicillin- and Vancomycin-Resistant Staphylococcus Aureus (MRSA and VRSA)</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bioorg.2018.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=30130650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCgs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=175-183&author=S.+Gatadiauthor=J.+Gourauthor=G.+Kaulauthor=M.+Shuklaauthor=A.+Dasguptaauthor=R.+Akunuriauthor=R.+Tripathiauthor=Y.+V.+Madhaviauthor=S.+Chopraauthor=S.+Nanduri&title=Synthesis+of+New+3-Phenylquinazolin-4%283H%29-One+Derivatives+as+Potent+Antibacterial+Agents+Effective+against+Methicillin-+and+Vancomycin-Resistant+Staphylococcus+Aureus+%28MRSA+and+VRSA%29&doi=10.1016%2Fj.bioorg.2018.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of new 3-phenylquinazolin-4(3H)-one derivatives as potent antibacterial agents effective against methicillin-resistant and vancomycin-resistant Staphylococcus aureus (MRSA and VRSA)</span></div><div class="casAuthors">Gatadi, Srikanth; Gour, Jitendra; Kaul, Grace; Shukla, Manjulika; Dasgupta, Arunava; Akunuri, Ravikumar; Tripathi, Richa; Madhavi, Y. V.; Chopra, Sidharth; Nanduri, Srinivas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-183</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Occurrence of infections due to the drug resistant Staphylococcus aureus is on rise necessitating the need for rapid development of new antibacterial agents.  In our present work, a series of new 3-phenylquinazolin-4(3H)-one derivs. were designed, synthesized and evaluated for their antibacterial activity against ESKAP (E. coli, S. aureus, K. pneumoniae, A. baumannii, P. aeruginosa) pathogen panel and pathogenic mycobacterial strains.  The study revealed that compds. 4a, 4c, 4e, 4f, 4g, 4i, 4o and 4p exhibited selective and potent inhibitory activity against Staphylococcus aureus with MIC values in the range of 0.125-8 μg/mL.  Further, the compds. 4c, 4e and 4g were found to be non toxic to Vero cells (CC50 = >10->100 μg/mL) and exhibited favorable selectivity index (SI = 40->200).  The compds. 4c, 4e and 4g also showed potent inhibitory activity against various MDR-S. aureus including VRSA.  The promising results obtained indicated the potential use of the above series of compds. as promising antibacterial agents for the treatment of multidrug resistant Staphylococcus aureus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2pnk1A-dXrVg90H21EOLACvtfcHk0lj6sk6mxK-MaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCgs7zL&md5=a21ae7970485b6ce84110f8fa3f0bc64</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DGatadi%26aufirst%3DS.%26aulast%3DGour%26aufirst%3DJ.%26aulast%3DKaul%26aufirst%3DG.%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DAkunuri%26aufirst%3DR.%26aulast%3DTripathi%26aufirst%3DR.%26aulast%3DMadhavi%26aufirst%3DY.%2BV.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520New%25203-Phenylquinazolin-4%25283H%2529-One%2520Derivatives%2520as%2520Potent%2520Antibacterial%2520Agents%2520Effective%2520against%2520Methicillin-%2520and%2520Vancomycin-Resistant%2520Staphylococcus%2520Aureus%2520%2528MRSA%2520and%2520VRSA%2529%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D175%26epage%3D183%26doi%3D10.1016%2Fj.bioorg.2018.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Daniel, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testero, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spink, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llarrull, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasiri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espahbodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenwalter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a New Class of Non-Β-Lactam Inhibitors of Penicillin-Binding Proteins with Gram-Positive Antibacterial Activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3664</span>– <span class="NLM_lpage">3672</span>, <span class="refDoi"> DOI: 10.1021/ja500053x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja500053x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVWhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=3664-3672&issue=9&author=P.+I.+O%E2%80%99Danielauthor=Z.+Pengauthor=H.+Piauthor=S.+A.+Testeroauthor=D.+Dingauthor=E.+Spinkauthor=E.+Leemansauthor=M.+A.+Boudreauauthor=T.+Yamaguchiauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=L.+I.+Llarrullauthor=W.+Songauthor=E.+Lastochkinauthor=M.+Kumarasiriauthor=N.+T.+Antunesauthor=M.+Espahbodiauthor=K.+Lichtenwalterauthor=M.+A.+Suckowauthor=S.+Vakulenkoauthor=S.+Mobasheryauthor=M.+Chang&title=Discovery+of+a+New+Class+of+Non-%CE%92-Lactam+Inhibitors+of+Penicillin-Binding+Proteins+with+Gram-Positive+Antibacterial+Activity&doi=10.1021%2Fja500053x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Class of Non-β-lactam Inhibitors of Penicillin-Binding Proteins with Gram-Positive Antibacterial Activity</span></div><div class="casAuthors">O'Daniel, Peter I.; Peng, Zhihong; Pi, Hualiang; Testero, Sebastian A.; Ding, Derong; Spink, Edward; Leemans, Erika; Boudreau, Marc A.; Yamaguchi, Takao; Schroeder, Valerie A.; Wolter, William R.; Llarrull, Leticia I.; Song, Wei; Lastochkin, Elena; Kumarasiri, Malika; Antunes, Nuno T.; Espahbodi, Mana; Lichtenwalter, Katerina; Suckow, Mark A.; Vakulenko, Sergei; Mobashery, Shahriar; Chang, Mayland</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3664-3672</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Infections caused by hard-to-treat methicillin-resistant Staphylococcus aureus (MRSA) are a serious global public-health concern, as MRSA has become broadly resistant to many classes of antibiotics.  We disclose herein the discovery of a new class of non-β-lactam antibiotics, the oxadiazoles, which inhibit penicillin-binding protein 2a (PBP2a) of MRSA.  The oxadiazoles show bactericidal activity against vancomycin- and linezolid-resistant MRSA and other Gram-pos. bacterial strains, in vivo efficacy in a mouse model of infection, and have 100% oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDVcdd66dic7Vg90H21EOLACvtfcHk0lj6sk6mxK-MaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVWhs70%253D&md5=e8fb52aeb853be0a2d8cfd55d67a39f8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja500053x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja500053x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Daniel%26aufirst%3DP.%2BI.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DTestero%26aufirst%3DS.%2BA.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DSpink%26aufirst%3DE.%26aulast%3DLeemans%26aufirst%3DE.%26aulast%3DBoudreau%26aufirst%3DM.%2BA.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DLlarrull%26aufirst%3DL.%2BI.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DKumarasiri%26aufirst%3DM.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DEspahbodi%26aufirst%3DM.%26aulast%3DLichtenwalter%26aufirst%3DK.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DMobashery%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520New%2520Class%2520of%2520Non-%25CE%2592-Lactam%2520Inhibitors%2520of%2520Penicillin-Binding%2520Proteins%2520with%2520Gram-Positive%2520Antibacterial%2520Activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D9%26spage%3D3664%26epage%3D3672%26doi%3D10.1021%2Fja500053x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spink, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastochkin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtenwalter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Daniel, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testero, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckow, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakulenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for the Oxadiazole Class of Antibiotics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1380</span>– <span class="NLM_lpage">1389</span>, <span class="refDoi"> DOI: 10.1021/jm501661f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501661f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVOlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1380-1389&issue=3&author=E.+Spinkauthor=D.+Dingauthor=Z.+Pengauthor=M.+A.+Boudreauauthor=E.+Leemansauthor=E.+Lastochkinauthor=W.+Songauthor=K.+Lichtenwalterauthor=P.+I.+O%E2%80%99Danielauthor=S.+A.+Testeroauthor=H.+Piauthor=V.+A.+Schroederauthor=W.+R.+Wolterauthor=N.+T.+Antunesauthor=M.+A.+Suckowauthor=S.+Vakulenkoauthor=M.+Changauthor=S.+Mobashery&title=Structure-Activity+Relationship+for+the+Oxadiazole+Class+of+Antibiotics&doi=10.1021%2Fjm501661f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for the Oxadiazole Class of Antibiotics</span></div><div class="casAuthors">Spink, Edward; Ding, Derong; Peng, Zhihong; Boudreau, Marc A.; Leemans, Erika; Lastochkin, Elena; Song, Wei; Lichtenwalter, Katerina; O'Daniel, Peter I.; Testero, Sebastian A.; Pi, Hualiang; Schroeder, Valerie A.; Wolter, William R.; Antunes, Nuno T.; Suckow, Mark A.; Vakulenko, Sergei; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1380-1389</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-activity relationship (SAR) for the newly discovered oxadiazole class of antibiotics is described with evaluation of 120 derivs. of the lead structure.  This class of antibiotics was discovered by in silico docking and scoring against the crystal structure of a penicillin-binding protein.  They impair cell-wall biosynthesis and exhibit activities against the Gram-pos. bacterium Staphylococcus aureus, including methicillin-resistant S. aureus (MRSA) and vancomycin-resistant and linezolid-resistant S. aureus.  Compd. I was efficacious in a mouse model of MRSA infection, exhibiting a long half-life, a high vol. of distribution, and low clearance.  This antibiotic is bactericidal and is orally bioavailable in mice.  This class of antibiotics holds great promise in recourse against infections by MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7oqinjSM5EbVg90H21EOLACvtfcHk0ljvBro1rNKkcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVOlsg%253D%253D&md5=197ec5fb19de5b9f56e83d0979ebb77a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm501661f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501661f%26sid%3Dliteratum%253Aachs%26aulast%3DSpink%26aufirst%3DE.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DBoudreau%26aufirst%3DM.%2BA.%26aulast%3DLeemans%26aufirst%3DE.%26aulast%3DLastochkin%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLichtenwalter%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Daniel%26aufirst%3DP.%2BI.%26aulast%3DTestero%26aufirst%3DS.%2BA.%26aulast%3DPi%26aufirst%3DH.%26aulast%3DSchroeder%26aufirst%3DV.%2BA.%26aulast%3DWolter%26aufirst%3DW.%2BR.%26aulast%3DAntunes%26aufirst%3DN.%2BT.%26aulast%3DSuckow%26aufirst%3DM.%2BA.%26aulast%3DVakulenko%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520the%2520Oxadiazole%2520Class%2520of%2520Antibiotics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D3%26spage%3D1380%26epage%3D1389%26doi%3D10.1021%2Fjm501661f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janardhanan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobashery, S.</span></span> <span> </span><span class="NLM_article-title">The Oxadiazole Antibacterials</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.mib.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27239942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=13-17&author=J.+Janardhananauthor=M.+Changauthor=S.+Mobashery&title=The+Oxadiazole+Antibacterials&doi=10.1016%2Fj.mib.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The oxadiazole antibacterials</span></div><div class="casAuthors">Janardhanan, Jeshina; Chang, Mayland; Mobashery, Shahriar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The oxadiazoles are a class of antibacterials discovered by in silico docking and scoring of compds. against the X-ray structure of a penicillin-binding protein.  These antibacterials exhibit activity against Gram-pos. bacteria, including against methicillin-resistant Staphylococcus aureus (MRSA) and vancomycin-resistant Enterococcus (VRE).  They show in vivo efficacy in murine models of peritonitis/sepsis and neutropenic thigh MRSA infection.  They are bactericidal and orally bioavailable.  The oxadiazoles show promise in treatment of MRSA infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGkNPBwBHezbVg90H21EOLACvtfcHk0ljvBro1rNKkcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWlur0%253D&md5=b0a236214aa4f89149d7197e43c8d4e7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DJanardhanan%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DMobashery%26aufirst%3DS.%26atitle%3DThe%2520Oxadiazole%2520Antibacterials%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2016%26volume%3D33%26spage%3D13%26epage%3D17%26doi%3D10.1016%2Fj.mib.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrio, J. L.</span></span> <span> </span><span class="NLM_article-title">Antifungals</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bcp.2016.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27884742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCgtrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=86-96&author=S.+Campoyauthor=J.+L.+Adrio&title=Antifungals&doi=10.1016%2Fj.bcp.2016.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungals</span></div><div class="casAuthors">Campoy, Sonia; Adrio, Jose L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86-96</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The need for new antifungal agents is undeniable.  Current therapeutic choices for the treatment of invasive fungal infections are limited to three classes of drugs.  Most used antifungal agents are not completely effective due to the development of resistance, host toxicity and undesirable side effects that limit their use in medical practice.  Invasive fungal infections have significantly increased over the last decades and the mortality rates remain unacceptably high.  More threatening, new resistance patterns have been obsd. including simultaneous resistance to different antifungal classes.  In the last years, deeper insights into the mol. mechanisms for fungal resistance and virulence have yielded some new potential targets for antifungal therapeutics.  Chem. genomics-based screenings, high throughput screenings of natural products and repurposing of approved drugs are some of the approaches being followed for the discovery of new antifungal mols.  However, despite the emerging need for effective antifungal agents, the current pipeline contains only a few promising mols., with novel modes of action, in early clin. development stages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplq5ZsJFOZS7Vg90H21EOLACvtfcHk0ljvBro1rNKkcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCgtrfI&md5=1d84f62c68cf48c04e498746156a90fe</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCampoy%26aufirst%3DS.%26aulast%3DAdrio%26aufirst%3DJ.%2BL.%26atitle%3DAntifungals%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D133%26spage%3D86%26epage%3D96%26doi%3D10.1016%2Fj.bcp.2016.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, E. I.</span></span> <span> </span><span class="NLM_article-title">Morpholine Antifungals and Their Mode of Action</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">788</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1042/bst0190788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1042%2Fbst0190788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1783217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADyaK3MXkvVClu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=788-793&issue=3&author=E.+I.+Mercer&title=Morpholine+Antifungals+and+Their+Mode+of+Action&doi=10.1042%2Fbst0190788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Morpholine antifungals and their mode of action</span></div><div class="casAuthors">Mercer, E. Ian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">788-93</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    </div><div class="casAbstract">A review with 30 refs., discussing the effects of morpholines on the growth of fungi, higher plants, and protozoa.  It details the primary biochem. effects of these compds. and elucidates the mechanism of inhibition of sterol-synthesizing enzymes, the consequences of sterol precursor accumulation and/or ergosterol deprivation in fungi, as well as the consequences of accumulation of 9β,19-cyclopropyl sterols in higher plants and protozoa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8INR0FrE8-rVg90H21EOLACvtfcHk0ljvBro1rNKkcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXkvVClu74%253D&md5=facd28fad52aac9160b1901006a04545</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1042%2Fbst0190788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbst0190788%26sid%3Dliteratum%253Aachs%26aulast%3DMercer%26aufirst%3DE.%2BI.%26atitle%3DMorpholine%2520Antifungals%2520and%2520Their%2520Mode%2520of%2520Action%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D1991%26volume%3D19%26issue%3D3%26spage%3D788%26epage%3D793%26doi%3D10.1042%2Fbst0190788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, R.</span>; <span class="NLM_string-name">Shah, A.
H.</span>; <span class="NLM_string-name">Rawal, M.
K.</span></span> <span> </span><span class="NLM_article-title">Antifungals: Mechanism of Action and Drug Resistance</span>. In  <i>Yeast Membrane Transport</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, J.</span>, <span class="NLM_string-name">Sychrová, H.</span>, <span class="NLM_string-name">Kschischo, M.</span></span>, Eds.; Advances in Experimental Medicine and Biology; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham, Switzerland</span>, <span class="NLM_year">2016</span>; pp  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">349</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1007%2F978-3-319-25304-6_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=327-349&author=R.+Prasad&author=A.%0AH.+Shah&author=M.%0AK.+Rawalauthor=J.+Ramos&author=H.+Sychrov%C3%A1&author=M.+Kschischo&title=Yeast+Membrane+Transport"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-25304-6_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-25304-6_14%26sid%3Dliteratum%253Aachs%26aulast%3DPrasad%26aufirst%3DR.%26atitle%3DAntifungals%253A%2520Mechanism%2520of%2520Action%2520and%2520Drug%2520Resistance%26btitle%3DYeast%2520Membrane%2520Transport%26aulast%3DRamos%26aufirst%3DJ.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2016%26spage%3D327%26epage%3D349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldred, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osheroff, N.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Quinolone Action and Resistance</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1574</span>, <span class="refDoi"> DOI: 10.1021/bi5000564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5000564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=1565-1574&issue=10&author=K.+J.+Aldredauthor=R.+J.+Kernsauthor=N.+Osheroff&title=Mechanism+of+Quinolone+Action+and+Resistance&doi=10.1021%2Fbi5000564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of quinolone action and resistance</span></div><div class="casAuthors">Aldred, Katie J.; Kerns, Robert J.; Osheroff, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1565-1574</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quinolones are one of the most commonly prescribed classes of antibacterials in the world and are used to treat a variety of bacterial infections in humans.  Because of the wide use (and overuse) of these drugs, the no. of quinolone-resistant bacterial strains has been growing steadily since the 1990s.  As is the case with other antibacterial agents, the rise in quinolone resistance threatens the clin. utility of this important drug class.  Quinolones act by converting their targets, gyrase and topoisomerase IV, into toxic enzymes that fragment the bacterial chromosome.  This review describes the development of the quinolones as antibacterials, the structure and function of gyrase and topoisomerase IV, and the mechanistic basis for quinolone action against their enzyme targets.  It will then discuss the following three mechanisms that decrease the sensitivity of bacterial cells to quinolones.  Target-mediated resistance is the most common and clin. significant form of resistance.  It is caused by specific mutations in gyrase and topoisomerase IV that weaken interactions between quinolones and these enzymes.  Plasmid-mediated resistance results from extrachromosomal elements that encode proteins that disrupt quinolone-enzyme interactions, alter drug metab., or increase quinolone efflux.  Chromosome-mediated resistance results from the underexpression of porins or the overexpression of cellular efflux pumps, both of which decrease cellular concns. of quinolones.  Finally, this review discusses recent advancements in our understanding of how quinolones interact with gyrase and topoisomerase IV and how mutations in these enzymes cause resistance.  These last findings suggest approaches to designing new drugs that display improved activity against resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro57dziddbmLVg90H21EOLACvtfcHk0lgjAmoeIjGpYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymtrk%253D&md5=38134014d5acd1d5c807095eaba34e3b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi5000564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5000564%26sid%3Dliteratum%253Aachs%26aulast%3DAldred%26aufirst%3DK.%2BJ.%26aulast%3DKerns%26aufirst%3DR.%2BJ.%26aulast%3DOsheroff%26aufirst%3DN.%26atitle%3DMechanism%2520of%2520Quinolone%2520Action%2520and%2520Resistance%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26issue%3D10%26spage%3D1565%26epage%3D1574%26doi%3D10.1021%2Fbi5000564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polykovskiy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhebrak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Lengeling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golovanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatanov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyaev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurbanov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artamonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aladinskiy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadurin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhavoronkov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models</span>. <i>ArXiv</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">1811.12823</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=1811.12823&author=D.+Polykovskiyauthor=A.+Zhebrakauthor=B.+Sanchez-Lengelingauthor=S.+Golovanovauthor=O.+Tatanovauthor=S.+Belyaevauthor=R.+Kurbanovauthor=A.+Artamonovauthor=V.+Aladinskiyauthor=M.+Veselovauthor=A.+Kadurinauthor=S.+Nikolenkoauthor=A.+Aspuru-Guzikauthor=A.+Zhavoronkov&title=Molecular+Sets+%28MOSES%29%3A+A+Benchmarking+Platform+for+Molecular+Generation+Models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPolykovskiy%26aufirst%3DD.%26aulast%3DZhebrak%26aufirst%3DA.%26aulast%3DSanchez-Lengeling%26aufirst%3DB.%26aulast%3DGolovanov%26aufirst%3DS.%26aulast%3DTatanov%26aufirst%3DO.%26aulast%3DBelyaev%26aufirst%3DS.%26aulast%3DKurbanov%26aufirst%3DR.%26aulast%3DArtamonov%26aufirst%3DA.%26aulast%3DAladinskiy%26aufirst%3DV.%26aulast%3DVeselov%26aufirst%3DM.%26aulast%3DKadurin%26aufirst%3DA.%26aulast%3DNikolenko%26aufirst%3DS.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DZhavoronkov%26aufirst%3DA.%26atitle%3DMolecular%2520Sets%2520%2528MOSES%2529%253A%2520A%2520Benchmarking%2520Platform%2520for%2520Molecular%2520Generation%2520Models%26jtitle%3DArXiv%26date%3D2018%26spage%3D1811.12823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiscato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaucher, A. C.</span></span> <span> </span><span class="NLM_article-title">GuacaMol: Benchmarking Models for de Novo Molecular Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1096</span>– <span class="NLM_lpage">1108</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVWrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1096-1108&issue=3&author=N.+Brownauthor=M.+Fiscatoauthor=M.+H.+S.+Seglerauthor=A.+C.+Vaucher&title=GuacaMol%3A+Benchmarking+Models+for+de+Novo+Molecular+Design&doi=10.1021%2Facs.jcim.8b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">GuacaMol: Benchmarking Models for de Novo Molecular Design</span></div><div class="casAuthors">Brown, Nathan; Fiscato, Marco; Segler, Marwin H. S.; Vaucher, Alain C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1096-1108</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">De novo design seeks to generate mols. with required property profiles by virtual design-make-test cycles.  With the emergence of deep learning and neural generative models in many application areas, models for mol. design based on neural networks appeared recently and show promising results.  However, the new models have not been profiled on consistent tasks, and comparative studies to well-established algorithms have only seldom been performed.  To standardize the assessment of both classical and neural models for de novo mol. design, we propose an evaluation framework, GuacaMol, based on a suite of standardized benchmarks.  The benchmark tasks encompass measuring the fidelity of the models to reproduce the property distribution of the training sets, the ability to generate novel mols., the exploration and exploitation of chem. space, and a variety of single and multiobjective optimization tasks.  The benchmarking open-source Python code and a leaderboard can be found on https://benevolent.ai/guacamol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosXrGIhvFoCrVg90H21EOLACvtfcHk0lgjAmoeIjGpYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVWrsbY%253D&md5=d3fb616b81a4b146cf77950a1c92e4d1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00839%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DFiscato%26aufirst%3DM.%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DVaucher%26aufirst%3DA.%2BC.%26atitle%3DGuacaMol%253A%2520Benchmarking%2520Models%2520for%2520de%2520Novo%2520Molecular%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D3%26spage%3D1096%26epage%3D1108%26doi%3D10.1021%2Facs.jcim.8b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrchan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, M. P.</span></span> <span> </span><span class="NLM_article-title">Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00512</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00512" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCjsLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=120-131&issue=1&author=M.+H.+S.+Seglerauthor=T.+Kogejauthor=C.+Tyrchanauthor=M.+P.+Waller&title=Generating+Focused+Molecule+Libraries+for+Drug+Discovery+with+Recurrent+Neural+Networks&doi=10.1021%2Facscentsci.7b00512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Generating Focused Molecule Libraries for Drug Discovery with Recurrent Neural Networks</span></div><div class="casAuthors">Segler, Marwin H. S.; Kogej, Thierry; Tyrchan, Christian; Waller, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">120-131</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In de novo drug design, computational strategies are used to generate novel mols. with good affinity to the desired biol. target.  In this work, we show that recurrent neural networks can be trained as generative models for mol. structures, similar to statistical language models in natural language processing.  We demonstrate that the properties of the generated mols. correlate very well with the properties of the mols. used to train the model.  In order to enrich libraries with mols. active toward a given biol. target, we propose to fine-tune the model with small sets of mols., which are known to be active against that target.  Against Staphylococcus aureus, the model reproduced 14% of 6051 hold-out test mols. that medicinal chemists designed, whereas against Plasmodium falciparum (Malaria), it reproduced 28% of 1240 test mols.  When coupled with a scoring function, our model can perform the complete de novo drug design cycle to generate large sets of novel mols. for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwhyDKGtV8eLVg90H21EOLACvtfcHk0lgjAmoeIjGpYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCjsLfP&md5=708f40422c7a911c629525ce5b66088b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00512%26sid%3Dliteratum%253Aachs%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DWaller%26aufirst%3DM.%2BP.%26atitle%3DGenerating%2520Focused%2520Molecule%2520Libraries%2520for%2520Drug%2520Discovery%2520with%2520Recurrent%2520Neural%2520Networks%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26issue%3D1%26spage%3D120%26epage%3D131%26doi%3D10.1021%2Facscentsci.7b00512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljsmiZyKW5Zdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017—Utility and Limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&issue=1&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+Year+Itch%3A+Pan-Assay+Interference+Compounds+%28PAINS%29+in+2017%E2%80%94Utility+and+Limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0ljsmiZyKW5Zdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520Year%2520Itch%253A%2520Pan-Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520in%25202017%25E2%2580%2594Utility%2520and%2520Limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D1%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">FAF-Drugs3: A Web Server for Compound Property Calculation and Chemical Library Design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W200</span>– <span class="NLM_lpage">W207</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkv353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=25883137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W200-W207&issue=W1&author=D.+Lagorceauthor=O.+Sperandioauthor=J.+B.+Baellauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=FAF-Drugs3%3A+A+Web+Server+for+Compound+Property+Calculation+and+Chemical+Library+Design&doi=10.1093%2Fnar%2Fgkv353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">FAF-Drugs3: a web server for compound property calculation and chemical library design</span></div><div class="casAuthors">Lagorce, David; Sperandio, Olivier; Baell, Jonathan B.; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W200-W207</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Drug attrition late in preclin. or clin. development is a serious economic problem in the field of drug discovery.  These problems can be linked, in part, to the quality of the compd. collections used during the hit generation stage and to the selection of compds. undergoing optimization.  Here, we present FAF-Drugs3, a web server that can be used for drug discovery and chem. biol. projects to help in prepg. compd. libraries and to assist decision-making during the hit selection/lead optimization phase.  Since it was first described in 2006, FAF-Drugs has been significantly modified.  The tool now applies an enhanced structure curation procedure, can filter or analyze mols. with user-defined or eight predefined physicochem. filters as well as with several simple ADMET (absorption, distribution, metab., excretion and toxicity) rules.  In addn., compds. can be filtered using an updated list of 154 hand-curated structural alerts while Pan Assay Interference compds. (PAINS) and other, generally unwanted groups are also investigated.  FAF-Drugs3 offers access to user-friendly html result pages and the possibility to download all computed data.  The server requires as input an SDF file of the compds.; it is open to all users and can be accessed without registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxYZ-hCUhlmrVg90H21EOLACvtfcHk0ljsmiZyKW5Zdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM&md5=5cf8888c489b90f392117ab065986214</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv353%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DFAF-Drugs3%253A%2520A%2520Web%2520Server%2520for%2520Compound%2520Property%2520Calculation%2520and%2520Chemical%2520Library%2520Design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW200%26epage%3DW207%26doi%3D10.1093%2Fnar%2Fgkv353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, P.</span></span> A Script to Run Structural Alerts Using the RDKit and ChEMBL. <span class="NLM_year">2018</span>. <a href="https://github.com/PatWalters/rd_filters" class="extLink">https://github.com/PatWalters/rd_filters</a> (accessed Apr 14, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Walters%2C+P.+A+Script+to+Run+Structural+Alerts+Using+the+RDKit+and+ChEMBL.+2018.+https%3A%2F%2Fgithub.com%2FPatWalters%2Frd_filters+%28accessed+Apr+14%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DP.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Using Fragment-Based Approaches to Discover New Antibiotics</span>. <i>Slas Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1177/2472555218773034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1177%2F2472555218773034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29923463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=495-510&issue=6&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Using+Fragment-Based+Approaches+to+Discover+New+Antibiotics&doi=10.1177%2F2472555218773034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Using Fragment-Based Approaches to Discover New Antibiotics</span></div><div class="casAuthors">Lamoree, Bas; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">495-510</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Fragment-based lead discovery has emerged over the past two decades as a successful approach to generate novel lead candidates in drug discovery programs.  The two main advantages over conventional high-throughput screening (HTS) are more efficient sampling of chem. space and tighter control over the physicochem. properties of the lead candidates.  Antibiotics are a class of drugs with particularly strict property requirements for efficacy and safety.  The development of novel antibiotics has slowed down so much that resistance has now evolved against every available antibiotic drug.  Here we give an overview of fragment-based approaches in screening and lead discovery projects for new antibiotics.  We discuss several successful hit-to-lead development examples.  Finally, we highlight the current challenges and opportunities for fragment-based lead discovery toward new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUbNlZsC5x6bVg90H21EOLACvtfcHk0lgte5XBcpH_Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrjK&md5=fccb6116d20d6b6091c78dba82c70819</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F2472555218773034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218773034%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DUsing%2520Fragment-Based%2520Approaches%2520to%2520Discover%2520New%2520Antibiotics%26jtitle%3DSlas%2520Discovery%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D495%26epage%3D510%26doi%3D10.1177%2F2472555218773034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gable, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Current and Potential Treatments for Ubiquitous but Neglected Herpesvirus Infections</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11412</span>, <span class="refDoi"> DOI: 10.1021/cr500255e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500255e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=11382-11412&issue=22&author=J.+E.+Gableauthor=T.+M.+Ackerauthor=C.+S.+Craik&title=Current+and+Potential+Treatments+for+Ubiquitous+but+Neglected+Herpesvirus+Infections&doi=10.1021%2Fcr500255e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Current and Potential Treatments for Ubiquitous but Neglected Herpesvirus Infections</span></div><div class="casAuthors">Gable, Jonathan E.; Acker, Timothy M.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">11382-11412</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxtq3MNOVUBLVg90H21EOLACvtfcHk0lgte5XBcpH_Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLrN&md5=2663dc411d94ff3ef539a738e2e4eb17</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fcr500255e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500255e%26sid%3Dliteratum%253Aachs%26aulast%3DGable%26aufirst%3DJ.%2BE.%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DCurrent%2520and%2520Potential%2520Treatments%2520for%2520Ubiquitous%2520but%2520Neglected%2520Herpesvirus%2520Infections%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26issue%3D22%26spage%3D11382%26epage%3D11412%26doi%3D10.1021%2Fcr500255e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glawar, A. F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkinson, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butters, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleet, G. W. J.</span></span> <span> </span><span class="NLM_article-title">3-Hydroxyazetidine Carboxylic Acids: Non-Proteinogenic Amino Acids for Medicinal Chemists</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fcmdc.201200541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=23468173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVOhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=658-666&issue=4&author=A.+F.+G.+Glawarauthor=S.+F.+Jenkinsonauthor=A.+L.+Thompsonauthor=S.+Nakagawaauthor=A.+Katoauthor=T.+D.+Buttersauthor=G.+W.+J.+Fleet&title=3-Hydroxyazetidine+Carboxylic+Acids%3A+Non-Proteinogenic+Amino+Acids+for+Medicinal+Chemists&doi=10.1002%2Fcmdc.201200541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">3-Hydroxyazetidine Carboxylic Acids: Non-Proteinogenic Amino Acids for Medicinal Chemists</span></div><div class="casAuthors">Glawar, Andreas F. G.; Jenkinson, Sarah F.; Thompson, Amber L.; Nakagawa, Shinpei; Kato, Atsushi; Butters, Terry D.; Fleet, George W. J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-666</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The formation from D-glucose of both enantiomers of 2,4-dideoxy-2,4-iminoribonic acid is the first chem. synthesis of unprotected 3-hydroxyazetidine carboxylic acids.  The long-term stability of 3-hydroxyazetidine amides is established at acidic and neutral pH and implies their value as non-proteinogenic amino acid components of peptides, providing medicinal chemists with a new class of peptide isosteres.  The structure of N,3-O-dibenzyl-2,4-dideoxy-2,4-imino-D-ribonic acid was established by x-ray crystallog. anal.  An N-methylazetidine amide deriv. is a specific inhibitor of β-hexosaminidases at the micromolar level, and is only the second example of potent inhibition of any glycosidase by an amide of a sugar amino acid related to an iminosugar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6WH8Z235zDbVg90H21EOLACvtfcHk0lgte5XBcpH_Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVOhsrg%253D&md5=e0e792ca1419766d93f9f96270e7a9f4</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200541%26sid%3Dliteratum%253Aachs%26aulast%3DGlawar%26aufirst%3DA.%2BF.%2BG.%26aulast%3DJenkinson%26aufirst%3DS.%2BF.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DNakagawa%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DButters%26aufirst%3DT.%2BD.%26aulast%3DFleet%26aufirst%3DG.%2BW.%2BJ.%26atitle%3D3-Hydroxyazetidine%2520Carboxylic%2520Acids%253A%2520Non-Proteinogenic%2520Amino%2520Acids%2520for%2520Medicinal%2520Chemists%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26issue%3D4%26spage%3D658%26epage%3D666%26doi%3D10.1002%2Fcmdc.201200541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span> (<span class="NLM_issue">7654</span>),  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&issue=7654&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lgHW5FFjXZooQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26issue%3D7654%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takrouri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahalingam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucchi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tear, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuning, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span> <span> </span><span class="NLM_article-title">Compound Design Guidelines for Evading the Efflux and Permeation Barriers of Escherichia Coli with the Oxazolidinone Class of Antibacterials: Test Case for a General Approach to Improving Whole Cell Gram-Negative Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">5310</span>– <span class="NLM_lpage">5321</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2Fj.bmcl.2017.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29102393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyht7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5310-5321&issue=23&author=A.+Spauldingauthor=K.+Takrouriauthor=P.+Mahalingamauthor=D.+C.+Clearyauthor=H.+D.+Cooperauthor=P.+Zucchiauthor=W.+Tearauthor=B.+Kolevaauthor=P.+J.+Beuningauthor=E.+B.+Hirschauthor=J.+B.+Aggen&title=Compound+Design+Guidelines+for+Evading+the+Efflux+and+Permeation+Barriers+of+Escherichia+Coli+with+the+Oxazolidinone+Class+of+Antibacterials%3A+Test+Case+for+a+General+Approach+to+Improving+Whole+Cell+Gram-Negative+Activity&doi=10.1016%2Fj.bmcl.2017.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Compound design guidelines for evading the efflux and permeation barriers of Escherichia coli with the oxazolidinone class of antibacterials: Test case for a general approach to improving whole cell Gram-negative activity</span></div><div class="casAuthors">Spaulding, Andrew; Takrouri, Khuloud; Mahalingam, Pornachandran; Cleary, Dillon C.; Cooper, Harold D.; Zucchi, Paola; Tear, Westley; Koleva, Bilyana; Beuning, Penny J.; Hirsch, Elizabeth B.; Aggen, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5310-5321</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Previously the authors reported the results from an effort to improve Gram-neg. antibacterial activity in the oxazolidinone class of antibiotics via a systematic medicinal chem. campaign focused entirely on C-ring modifications.  In that series the authors set about testing if the efflux and permeation barriers intrinsic to the outer membrane of Escherichia coli could be rationally overcome by designing analogs to reside in specific property limits assocd. with Gram-neg. activity: (i) low MW (<400), (ii) high polarity (clogD7.4 <1), and (iii) zwitterionic character at pH 7.4.  Indeed, the authors obsd. that only analogs residing within these limits were able to overcome these barriers.  Herein the authors report the results from a parallel effort where the authors explored structural changes throughout all three rings in the scaffold for the same purpose.  Compds. were tested against a diagnostic MIC panel of Escherichia coli and Staphylococcus aureus strains to det. the impact of combining structural modifications in overcoming the OM barriers and in bridging the potency gap between the species.  The results demonstrated that distributing the charge-carrying moieties across two rings was also beneficial for avoidance of the outer membrane barriers.  Importantly, anal. of the structure-permeation relationship (SPR) obtained from this and the prior study indicated that in addn. to MW, polarity, and zwitterionic character, having ≤4 rotatable bonds is also assocd. with evasion of the OM barriers.  These combined results provide the medicinal chemist with a framework and strategy for overcoming the OM barriers in GNB in antibacterial drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxD-MAU_7WfbVg90H21EOLACvtfcHk0lgHW5FFjXZooQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyht7%252FM&md5=fc70a1d802e4ec625f8aa21b0ab02f02</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DSpaulding%26aufirst%3DA.%26aulast%3DTakrouri%26aufirst%3DK.%26aulast%3DMahalingam%26aufirst%3DP.%26aulast%3DCleary%26aufirst%3DD.%2BC.%26aulast%3DCooper%26aufirst%3DH.%2BD.%26aulast%3DZucchi%26aufirst%3DP.%26aulast%3DTear%26aufirst%3DW.%26aulast%3DKoleva%26aufirst%3DB.%26aulast%3DBeuning%26aufirst%3DP.%2BJ.%26aulast%3DHirsch%26aufirst%3DE.%2BB.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26atitle%3DCompound%2520Design%2520Guidelines%2520for%2520Evading%2520the%2520Efflux%2520and%2520Permeation%2520Barriers%2520of%2520Escherichia%2520Coli%2520with%2520the%2520Oxazolidinone%2520Class%2520of%2520Antibacterials%253A%2520Test%2520Case%2520for%2520a%2520General%2520Approach%2520to%2520Improving%2520Whole%2520Cell%2520Gram-Negative%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26issue%3D23%26spage%3D5310%26epage%3D5321%26doi%3D10.1016%2Fj.bmcl.2017.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Converting Gram-Positive-Only Compounds into Broad-Spectrum Antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&issue=1&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+Challenge+of+Converting+Gram-Positive-Only+Compounds+into+Broad-Spectrum+Antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0lgHW5FFjXZooQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520Challenge%2520of%2520Converting%2520Gram-Positive-Only%2520Compounds%2520into%2520Broad-Spectrum%2520Antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26issue%3D1%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acosta-Gutiérrez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathania, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodrenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterhalter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavenger, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naismith, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Berg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, M.</span></span> <span> </span><span class="NLM_article-title">Getting Drugs into Gram-Negative Bacteria: Rational Rules for Permeation through General Porins</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=1487-1498&issue=10&author=S.+Acosta-Guti%C3%A9rrezauthor=L.+Ferraraauthor=M.+Pathaniaauthor=M.+Masiauthor=J.+Wangauthor=I.+Bodrenkoauthor=M.+Zahnauthor=M.+Winterhalterauthor=R.+A.+Stavengerauthor=J.-M.+Pag%C3%A8sauthor=J.+H.+Naismithauthor=B.+van+den%0ABergauthor=M.+G.+P.+Pageauthor=M.+Ceccarelli&title=Getting+Drugs+into+Gram-Negative+Bacteria%3A+Rational+Rules+for+Permeation+through+General+Porins&doi=10.1021%2Facsinfecdis.8b00108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Getting Drugs into Gram-Negative Bacteria: Rational Rules for Permeation through General Porins</span></div><div class="casAuthors">Acosta-Gutierrez, Silvia; Ferrara, Luana; Pathania, Monisha; Masi, Muriel; Wang, Jiajun; Bodrenko, Igor; Zahn, Michael; Winterhalter, Mathias; Stavenger, Robert A.; Pages, Jean-Marie; Naismith, James H.; van den Berg, Bert; Page, Malcolm G. P.; Ceccarelli, Matteo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1487-1498</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small, hydrophilic mols., including most important antibiotics in clin. use, cross the Gram-neg. outer membrane through the water-filled channels provided by porins.  We have detd. the X-ray crystal structures of the principal general porins from three species of Enterobacteriaceae, namely Enterobacter aerogenes, Enterobacter cloacae, and Klebsiella pneumoniae, and detd. their antibiotic permeabilities as well as those of the orthologues from Escherichia coli.  Starting from the structure of the porins and mols., we propose a phys. mechanism underlying transport and condense it in a computationally efficient scoring function.  The scoring function shows good agreement with in vitro penetration data and will enable the screening of virtual databases to identify mols. with optimal permeability through porins and help to guide the optimization of antibiotics with poor permeation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppq33eJ0gJ1bVg90H21EOLACvtfcHk0lgHW5FFjXZooQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrfM&md5=d66864f6de3193581e4663ef7c436baf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00108%26sid%3Dliteratum%253Aachs%26aulast%3DAcosta-Guti%25C3%25A9rrez%26aufirst%3DS.%26aulast%3DFerrara%26aufirst%3DL.%26aulast%3DPathania%26aufirst%3DM.%26aulast%3DMasi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBodrenko%26aufirst%3DI.%26aulast%3DZahn%26aufirst%3DM.%26aulast%3DWinterhalter%26aufirst%3DM.%26aulast%3DStavenger%26aufirst%3DR.%2BA.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26aulast%3DNaismith%26aufirst%3DJ.%2BH.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DB.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DCeccarelli%26aufirst%3DM.%26atitle%3DGetting%2520Drugs%2520into%2520Gram-Negative%2520Bacteria%253A%2520Rational%2520Rules%2520for%2520Permeation%2520through%2520General%2520Porins%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26issue%3D10%26spage%3D1487%26epage%3D1498%26doi%3D10.1021%2Facsinfecdis.8b00108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Firth, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span> <span> </span><span class="NLM_article-title">Plane of Best Fit: A Novel Method to Characterize the Three-Dimensionality of Molecules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2516</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/ci300293f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300293f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2516-2525&issue=10&author=N.+C.+Firthauthor=N.+Brownauthor=J.+Blagg&title=Plane+of+Best+Fit%3A+A+Novel+Method+to+Characterize+the+Three-Dimensionality+of+Molecules&doi=10.1021%2Fci300293f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Plane of Best Fit: A Novel Method to Characterize the Three-Dimensionality of Molecules</span></div><div class="casAuthors">Firth, Nicholas C.; Brown, Nathan; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2516-2525</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a computational method, plane of best fit (PBF), to quantify and characterize the 3D character of mols.  This method is rapid and amenable to anal. of large diverse data sets.  We compare PBF with alternative literature methods used to assess 3D character and apply the method to diverse data sets of fragment-like, drug-like, and natural product compd. libraries.  We show that exemplar fragment libraries underexploit the potential of 3D character in fragment-like chem. space and that drug-like mols. in the libraries examd. are predominantly 2D in character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszhZRQVuP0LVg90H21EOLACvtfcHk0ljFCfsef18ekA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rt7zP&md5=747bb9c5fb38eb34af4e1047f83633e8</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci300293f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300293f%26sid%3Dliteratum%253Aachs%26aulast%3DFirth%26aufirst%3DN.%2BC.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3DPlane%2520of%2520Best%2520Fit%253A%2520A%2520Novel%2520Method%2520to%2520Characterize%2520the%2520Three-Dimensionality%2520of%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D10%26spage%3D2516%26epage%3D2525%26doi%3D10.1021%2Fci300293f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span> <span> </span><span class="NLM_article-title">Advancing the Drug Discovery and Development Process</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">35</span>),  <span class="NLM_fpage">9128</span>– <span class="NLM_lpage">9140</span>, <span class="refDoi"> DOI: 10.1002/anie.201404761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fanie.201404761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGmu77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=9128-9140&issue=35&author=K.+C.+Nicolaou&title=Advancing+the+Drug+Discovery+and+Development+Process&doi=10.1002%2Fanie.201404761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing the Drug Discovery and Development Process</span></div><div class="casAuthors">Nicolaou, K. C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9128-9140</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current state of affairs in the drug discovery and development process is briefly summarized and then ways to take advantage of the ever-increasing fundamental knowledge and tech. knowhow in chem. and biol. and related disciplines are discussed.  The primary motivation of this Essay is to celebrate the great achievements of chem., biol., and medicine and to inform and inspire students and academics to enter the field of drug discovery and development while, at the same time, continue to advance the fundamentals of their disciplines.  It is also meant to encourage and catalyze multidisciplinary partnerships between academia and industry as scientists attempt to merge their often complementary interests and expertise to achieve new improvements and breakthroughs in their resp. fields, and the common goal of applying them to the discovery and invention of new and better medicines, esp. in areas of unmet needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFwL5Am5_V7Vg90H21EOLACvtfcHk0ljFCfsef18ekA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGmu77O&md5=1348675f7bdf8e35a0ae11881ba11750</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fanie.201404761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201404761%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26atitle%3DAdvancing%2520the%2520Drug%2520Discovery%2520and%2520Development%2520Process%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26issue%3D35%26spage%3D9128%26epage%3D9140%26doi%3D10.1002%2Fanie.201404761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span>OSIRIS-REx.
Wikipedia, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OSIRIS-REx.%0AWikipedia%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&issue=11&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+Progress+in+the+Development+of+HIV-1+Protease+Inhibitors+for+the+Treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0ljFCfsef18ekA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520HIV-1%2520Protease%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenkel, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewi, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shatkin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">High-Resolution Structures of HIV-1 Reverse transcriptase/TMC278 Complexes: Strategic Flexibility Explains Potency against Resistance Mutations</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1466</span>– <span class="NLM_lpage">1471</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711209105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0711209105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=18230722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1466-1471&issue=5&author=K.+Dasauthor=J.+D.+Baumanauthor=A.+D.+Clarkauthor=Y.+V.+Frenkelauthor=P.+J.+Lewiauthor=A.+J.+Shatkinauthor=S.+H.+Hughesauthor=E.+Arnold&title=High-Resolution+Structures+of+HIV-1+Reverse+transcriptase%2FTMC278+Complexes%3A+Strategic+Flexibility+Explains+Potency+against+Resistance+Mutations&doi=10.1073%2Fpnas.0711209105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: strategic flexibility explains potency against resistance mutations</span></div><div class="casAuthors">Das, Kalyan; Bauman, Joseph D.; Clark, Arthur D.; Frenkel, Yulia V.; Lewi, Paul J.; Shatkin, Aaron J.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1466-1471</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TMC278 is a diarylpyrimidine (DAPY) nonnucleoside reverse transcriptase inhibitor (NNRTI) that is highly effective in treating wild-type and drug-resistant HIV-1 infections in clin. trials at relatively low doses (∼ 25-75 mg/day).  We have detd. the structure of wild-type HIV-1 RT complexed with TMC278 at 1.8 Å resoln., using an RT crystal form engineered by systematic RT mutagenesis.  This high-resoln. structure reveals that the cyanovinyl group of TMC278 is positioned in a hydrophobic tunnel connecting the NNRTI-binding pocket to the nucleic acid-binding cleft.  The crystal structures of TMC278 in complexes with the double mutant K103N/Y181C (2.1 Å) and L100I/K103N HIV-1 RTs (2.9 Å) demonstrated that TMC278 adapts to bind mutant RTs.  In the K103N/Y181C RT/TMC278 structure, loss of the arom. ring interaction caused by the Y181C mutation is counterbalanced by interactions between the cyanovinyl group of TMC278 and the arom. side chain of Y183, which is facilitated by an ∼ 1.5 Å shift of the conserved Y183MDD motif.  In the L1001/K103N RT/ TMC278 structure, the binding mode of TMC278 is significantly altered so that the drug conforms to changes in the binding pocket primarily caused by the L1001 mutation.  The flexible binding pocket acts as a mol. "shrink wrap" that makes a shape complementary to the optimized TMC278 in wild-type and drug-resistant forms of HIV-1 RT.  The crystal structures provide a better understanding of how the flexibility of an inhibitor can compensate for drug-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRbleFGfFk7Vg90H21EOLACvtfcHk0ljfe9NlsZgKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D&md5=5f2ea5e4d88e9822ceecee636e1e4bd1</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711209105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711209105%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DFrenkel%26aufirst%3DY.%2BV.%26aulast%3DLewi%26aufirst%3DP.%2BJ.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHigh-Resolution%2520Structures%2520of%2520HIV-1%2520Reverse%2520transcriptase%252FTMC278%2520Complexes%253A%2520Strategic%2520Flexibility%2520Explains%2520Potency%2520against%2520Resistance%2520Mutations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D5%26spage%3D1466%26epage%3D1471%26doi%3D10.1073%2Fpnas.0711209105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansdon, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brendza, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of HIV-1 Reverse Transcriptase with Etravirine (TMC125) and Rilpivirine (TMC278): Implications for Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4295</span>– <span class="NLM_lpage">4299</span>, <span class="refDoi"> DOI: 10.1021/jm1002233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1002233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFeqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4295-4299&issue=10&author=E.+B.+Lansdonauthor=K.+M.+Brendzaauthor=M.+Hungauthor=R.+Wangauthor=S.+Mukundauthor=D.+Jinauthor=G.+Birkusauthor=N.+Kuttyauthor=X.+Liu&title=Crystal+Structures+of+HIV-1+Reverse+Transcriptase+with+Etravirine+%28TMC125%29+and+Rilpivirine+%28TMC278%29%3A+Implications+for+Drug+Design&doi=10.1021%2Fjm1002233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of HIV-1 Reverse Transcriptase with Etravirine (TMC125) and Rilpivirine (TMC278): Implications for Drug Design</span></div><div class="casAuthors">Lansdon, Eric B.; Brendza, Katherine M.; Hung, Magdeleine; Wang, Ruth; Mukund, Susmith; Jin, Debi; Birkus, Gabriel; Kutty, Nilima; Liu, Xiaohong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4295-4299</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diarylpyrimidine (DAPY) non-nucleoside reverse transcriptase inhibitors (NNRTIs) have inherent flexibility, helping to maintain activity against a wide range of resistance mutations.  Crystal structures were detd. with wild-type and K103N HIV-1 reverse transcriptase with etravirine (TMC125) and rilpivirine (TMC278).  These structures reveal a similar binding mode for TMC125 and TMC278, whether bound to wild-type or K103N RT.  Comparison to previously published structures reveals differences in binding modes for TMC125 and differences in protein conformation for TMC278.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNZlZQF054rVg90H21EOLACvtfcHk0ljfe9NlsZgKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFeqsLs%253D&md5=d99b8d74a488eb94da35246c6f141240</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm1002233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1002233%26sid%3Dliteratum%253Aachs%26aulast%3DLansdon%26aufirst%3DE.%2BB.%26aulast%3DBrendza%26aufirst%3DK.%2BM.%26aulast%3DHung%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DMukund%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DBirkus%26aufirst%3DG.%26aulast%3DKutty%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DCrystal%2520Structures%2520of%2520HIV-1%2520Reverse%2520Transcriptase%2520with%2520Etravirine%2520%2528TMC125%2529%2520and%2520Rilpivirine%2520%2528TMC278%2529%253A%2520Implications%2520for%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D10%26spage%3D4295%26epage%3D4299%26doi%3D10.1021%2Fjm1002233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fader, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenfant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilodeau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brochu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulombe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garneau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordingley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bös, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupart, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faucher, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoakim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsantrizos, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of BI 224436, a Noncatalytic Site Integrase Inhibitor (NCINI) of HIV-1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/ml500002n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500002n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=422-427&issue=4&author=L.+D.+Faderauthor=E.+Malenfantauthor=M.+Parisienauthor=R.+Carsonauthor=F.+Bilodeauauthor=S.+Landryauthor=M.+Pesantauthor=C.+Brochuauthor=S.+Morinauthor=C.+Chabotauthor=T.+Halmosauthor=Y.+Bousquetauthor=M.+D.+Baileyauthor=S.+H.+Kawaiauthor=R.+Coulombeauthor=S.+LaPlanteauthor=A.+Jakalianauthor=P.+K.+Bhardwajauthor=D.+Wernicauthor=P.+Schroederauthor=M.+Amadauthor=P.+Edwardsauthor=M.+Garneauauthor=J.+Duanauthor=M.+Cordingleyauthor=R.+Bethellauthor=S.+W.+Masonauthor=M.+B%C3%B6sauthor=P.+Bonneauauthor=M.-A.+Poupartauthor=A.-M.+Faucherauthor=B.+Simoneauauthor=C.+Fenwickauthor=C.+Yoakimauthor=Y.+Tsantrizos&title=Discovery+of+BI+224436%2C+a+Noncatalytic+Site+Integrase+Inhibitor+%28NCINI%29+of+HIV-1&doi=10.1021%2Fml500002n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of BI 224436, a Noncatalytic Site Integrase Inhibitor (NCINI) of HIV-1</span></div><div class="casAuthors">Fader, Lee D.; Malenfant, Eric; Parisien, Mathieu; Carson, Rebekah; Bilodeau, Francois; Landry, Serge; Pesant, Marc; Brochu, Christian; Morin, Sebastien; Chabot, Catherine; Halmos, Ted; Bousquet, Yves; Bailey, Murray D.; Kawai, Stephen H.; Coulombe, Rene; LaPlante, Steven; Jakalian, Araz; Bhardwaj, Punit K.; Wernic, Dominik; Schroeder, Patricia; Amad, Ma'an; Edwards, Paul; Garneau, Michel; Duan, Jianmin; Cordingley, Michael; Bethell, Richard; Mason, Stephen W.; Bos, Michael; Bonneau, Pierre; Poupart, Marc-Andre; Faucher, Anne-Marie; Simoneau, Bruno; Fenwick, Craig; Yoakim, Christiane; Tsantrizos, Youla</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">422-427</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assay recapitulating the 3' processing activity of HIV-1 integrase (IN) was used to screen the Boehringer Ingelheim compd. collection.  Hit-to-lead and lead optimization beginning with compd. I established the importance of the C3 and C4 substituent to antiviral potency against viruses with different aa124/aa125 variants of IN.  The importance of the C7 position on the serum shifted potency was established.  Introduction of a quinoline substituent at the C4 position provided a balance of potency and metabolic stability.  Combination of these findings ultimately led to the discovery of compd. II (BI 224436), the first NCINI to advance into a phase Ia clin. trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrE57FQpskKLVg90H21EOLACvtfcHk0lgi_DEN_JK0fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitb8%253D&md5=6df7ddab6dadf569d00d4314a773532c</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fml500002n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500002n%26sid%3Dliteratum%253Aachs%26aulast%3DFader%26aufirst%3DL.%2BD.%26aulast%3DMalenfant%26aufirst%3DE.%26aulast%3DParisien%26aufirst%3DM.%26aulast%3DCarson%26aufirst%3DR.%26aulast%3DBilodeau%26aufirst%3DF.%26aulast%3DLandry%26aufirst%3DS.%26aulast%3DPesant%26aufirst%3DM.%26aulast%3DBrochu%26aufirst%3DC.%26aulast%3DMorin%26aufirst%3DS.%26aulast%3DChabot%26aufirst%3DC.%26aulast%3DHalmos%26aufirst%3DT.%26aulast%3DBousquet%26aufirst%3DY.%26aulast%3DBailey%26aufirst%3DM.%2BD.%26aulast%3DKawai%26aufirst%3DS.%2BH.%26aulast%3DCoulombe%26aufirst%3DR.%26aulast%3DLaPlante%26aufirst%3DS.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBhardwaj%26aufirst%3DP.%2BK.%26aulast%3DWernic%26aufirst%3DD.%26aulast%3DSchroeder%26aufirst%3DP.%26aulast%3DAmad%26aufirst%3DM.%26aulast%3DEdwards%26aufirst%3DP.%26aulast%3DGarneau%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DCordingley%26aufirst%3DM.%26aulast%3DBethell%26aufirst%3DR.%26aulast%3DMason%26aufirst%3DS.%2BW.%26aulast%3DB%25C3%25B6s%26aufirst%3DM.%26aulast%3DBonneau%26aufirst%3DP.%26aulast%3DPoupart%26aufirst%3DM.-A.%26aulast%3DFaucher%26aufirst%3DA.-M.%26aulast%3DSimoneau%26aufirst%3DB.%26aulast%3DFenwick%26aufirst%3DC.%26aulast%3DYoakim%26aufirst%3DC.%26aulast%3DTsantrizos%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520BI%2520224436%252C%2520a%2520Noncatalytic%2520Site%2520Integrase%2520Inhibitor%2520%2528NCINI%2529%2520of%2520HIV-1%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D4%26spage%3D422%26epage%3D427%26doi%3D10.1021%2Fml500002n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valkov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherepanov, P.</span></span> <span> </span><span class="NLM_article-title">Retroviral Intasome Assembly and Inhibition of DNA Strand Transfer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span> (<span class="NLM_issue">7286</span>),  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1038/nature08784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1038%2Fnature08784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=20118915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=232-236&issue=7286&author=S.+Hareauthor=S.+S.+Guptaauthor=E.+Valkovauthor=A.+Engelmanauthor=P.+Cherepanov&title=Retroviral+Intasome+Assembly+and+Inhibition+of+DNA+Strand+Transfer&doi=10.1038%2Fnature08784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral intasome assembly and inhibition of DNA strand transfer</span></div><div class="casAuthors">Hare, Stephen; Gupta, Saumya Shree; Valkov, Eugene; Engelman, Alan; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7286</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Integrase is an essential retroviral enzyme that binds both termini of linear viral DNA and inserts them into a host cell chromosome.  The structure of full-length retroviral integrase, either sep. or in complex with DNA, has been lacking.  Furthermore, although clin. useful inhibitors of HIV integrase have been developed, their mechanism of action remains speculative.  Here the authors present a crystal structure of full-length integrase from the prototype foamy virus in complex with its cognate DNA.  The structure shows the organization of the retroviral intasome comprising an integrase tetramer tightly assocd. with a pair of viral DNA ends.  All three canonical integrase structural domains are involved in extensive protein-DNA and protein-protein interactions.  The binding of strand-transfer inhibitors displaces the reactive viral DNA end from the active site, disarming the viral nucleoprotein complex.  These findings define the structural basis of retroviral DNA integration, and will allow modeling of the HIV-1 intasome to aid in the development of antiretroviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdEDTFdvZMbVg90H21EOLACvtfcHk0lgi_DEN_JK0fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D&md5=9551e74b6bc33494855651c3b157d753</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature08784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08784%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DS.%2BS.%26aulast%3DValkov%26aufirst%3DE.%26aulast%3DEngelman%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520Intasome%2520Assembly%2520and%2520Inhibition%2520of%2520DNA%2520Strand%2520Transfer%26jtitle%3DNature%26date%3D2010%26volume%3D464%26issue%3D7286%26spage%3D232%26epage%3D236%26doi%3D10.1038%2Fnature08784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrusán, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Site Structure Shapes Allosteric Signal Transduction and the Evolution of Allostery in Protein Complexes</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1093/molbev/msz093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsz093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=31004156" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=G.+Abrus%C3%A1nauthor=J.+A.+Marsh&title=Ligand+Binding+Site+Structure+Shapes+Allosteric+Signal+Transduction+and+the+Evolution+of+Allostery+in+Protein+Complexes&doi=10.1093%2Fmolbev%2Fmsz093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsz093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsz093%26sid%3Dliteratum%253Aachs%26aulast%3DAbrus%25C3%25A1n%26aufirst%3DG.%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520Binding%2520Site%2520Structure%2520Shapes%2520Allosteric%2520Signal%2520Transduction%2520and%2520the%2520Evolution%2520of%2520Allostery%2520in%2520Protein%2520Complexes%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2019%26doi%3D10.1093%2Fmolbev%2Fmsz093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metrangolo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilati, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priimagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terraneo, G.</span></span> <span> </span><span class="NLM_article-title">The Halogen Bond</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2601</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&issue=4&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+Halogen+Bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0lgi_DEN_JK0fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520Halogen%2520Bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26issue%3D4%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Halogen Bonding for Rational Drug Design and New Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">383</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.678829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1517%2F17460441.2012.678829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=22462734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlCqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=375-383&issue=5&author=Y.+Luauthor=Y.+Liuauthor=Z.+Xuauthor=H.+Liauthor=H.+Liuauthor=W.+Zhu&title=Halogen+Bonding+for+Rational+Drug+Design+and+New+Drug+Discovery&doi=10.1517%2F17460441.2012.678829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonding for rational drug design and new drug discovery</span></div><div class="casAuthors">Lu, Yunxiang; Liu, Yingtao; Xu, Zhijian; Li, Haiying; Liu, Honglai; Zhu, Weiliang</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">375-383</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: A large no. of drugs and drug candidates in clin. development contain halogen substituents.  For a long time, only the steric and lipophilic contributions of halogens were considered when trying to exploit their effects on ligand binding.  However, the ability of halogens to form stabilizing interactions, such as halogen bonding, hydrogen bonding and multipolar interactions, in biomol. systems was revealed recently.  Halogen bonding, the non-covalent interaction in which covalently bound halogens interact with Lewis bases, has now been utilized in the context of rational drug design.Areas covered: The purpose of this review is to show how halogen bonding could be used in drug design, and in particular, to stimulate researchers to apply halogen bonding in lead optimization.  This review article covers the recent advances relevant to halogen bonding in drug discovery and biol. design over the past decade, including database survey of this interaction in protein-ligand complexes, mol. mech. investigations of halogen bonding in drug discovery and applications of this interaction in the development of halogenated ligands as inhibitors and drugs for protein kinases, serine protease factor Xa, HIV reverse transcriptase and so on.Expert opinion: Halogen bonding should intentionally be used as a powerful tool, comparable with hydrogen bonding, to enhance the binding affinity and also influence the binding selectivity.  Rational design of new and potent inhibitors against therapeutic targets through halogen bonding continues to be an exciting area, which will be further elucidated with the combination of various exptl. techniques and theor. calcns. in the forthcoming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO8AUipLTwmbVg90H21EOLACvtfcHk0li3sBWeT3RO_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlCqsrc%253D&md5=f263970ae5ef3706b0148f0c675b70aa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.678829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.678829%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DHalogen%2520Bonding%2520for%2520Rational%2520Drug%2520Design%2520and%2520New%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26issue%3D5%26spage%3D375%26epage%3D383%26doi%3D10.1517%2F17460441.2012.678829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auffinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hays, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P. S.</span></span> <span> </span><span class="NLM_article-title">Halogen Bonds in Biological Molecules</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">16789</span>– <span class="NLM_lpage">16794</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407607101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1073%2Fpnas.0407607101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=15557000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16789-16794&issue=48&author=P.+Auffingerauthor=F.+A.+Haysauthor=E.+Westhofauthor=P.+S.+Ho&title=Halogen+Bonds+in+Biological+Molecules&doi=10.1073%2Fpnas.0407607101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonds in biological molecules</span></div><div class="casAuthors">Auffinger, Pascal; Hays, Franklin A.; Westhof, Eric; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16789-16794</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Short oxygen-halogen interactions have been known in org. chem. since the 1950s and recently have been exploited in the design of supramol. assemblies.  The present survey of protein and nucleic acid structures reveals similar halogen bonds as potentially stabilizing inter- and intramol. interactions that can affect ligand binding and mol. folding.  A halogen bond in biomols. can be defined as a short C-X···O-Y interaction (C-X is a carbon-bonded chlorine, bromine, or iodine, and O-Y is a carbonyl, hydroxyl, charged carboxylate, or phosphate group), where the X···O distance is less than or equal to the sums of the resp. van der Waals radii (3.27 Å for Cl···O, 3.37 Å for Br···O, and 3.50 Å for I···O) and can conform to the geometry seen in small mols., with the C-X···O angle ≈165° (consistent with a strong directional polarization of the halogen) and the X···O-Y angle ≈120°.  Alternative geometries can be imposed by the more complex environment found in biomols., depending on which of the two types of donor systems are involved in the interaction: (i) the lone pair electrons of oxygen (and, to a lesser extent, nitrogen and sulfur) atoms or (ii) the delocalized π-electrons of peptide bonds or carboxylate or amide groups.  Thus, the specific geometry and diversity of the interacting partners of halogen bonds offer new and versatile tools for the design of ligands as drugs and materials in nanotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPMdZU91iT77Vg90H21EOLACvtfcHk0li3sBWeT3RO_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P&md5=dfcce25d4e336c66294f6ab8ce412154</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407607101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407607101%26sid%3Dliteratum%253Aachs%26aulast%3DAuffinger%26aufirst%3DP.%26aulast%3DHays%26aufirst%3DF.%2BA.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DHo%26aufirst%3DP.%2BS.%26atitle%3DHalogen%2520Bonds%2520in%2520Biological%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D48%26spage%3D16789%26epage%3D16794%26doi%3D10.1073%2Fpnas.0407607101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjahad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oumouch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemont, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coupa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csoka, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andries, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grierson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures for HIV-1 Reverse Transcriptase in Complexes with Three Pyridinone Derivatives: A New Class of Non-Nucleoside Inhibitors Effective against a Broad Range of Drug-Resistant Strains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">7582</span>– <span class="NLM_lpage">7591</span>, <span class="refDoi"> DOI: 10.1021/jm0500323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0500323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7582-7591&issue=24&author=D.+M.+Himmelauthor=K.+Dasauthor=A.+D.+Clarkauthor=S.+H.+Hughesauthor=A.+Benjahadauthor=S.+Oumouchauthor=J.+Guillemontauthor=S.+Coupaauthor=A.+Ponceletauthor=I.+Csokaauthor=C.+Meyerauthor=K.+Andriesauthor=C.+H.+Nguyenauthor=D.+S.+Griersonauthor=E.+Arnold&title=Crystal+Structures+for+HIV-1+Reverse+Transcriptase+in+Complexes+with+Three+Pyridinone+Derivatives%3A+A+New+Class+of+Non-Nucleoside+Inhibitors+Effective+against+a+Broad+Range+of+Drug-Resistant+Strains&doi=10.1021%2Fjm0500323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures for HIV-1 Reverse Transcriptase in Complexes with Three Pyridinone Derivatives: A New Class of Non-Nucleoside Inhibitors Effective against a Broad Range of Drug-Resistant Strains</span></div><div class="casAuthors">Himmel, Daniel M.; Das, Kalyan; Clark, Arthur D., Jr.; Hughes, Stephen H.; Benjahad, Abdellah; Oumouch, Said; Guillemont, Jerome; Coupa, Sophie; Poncelet, Alain; Csoka, Imre; Meyer, Christophe; Andries, Koen; Nguyen, Chi Hung; Grierson, David S.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7582-7591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of AIDS, the efficacy of all drugs, including nonnucleoside inhibitors (NNRTIs) of HIV-1 reverse transcriptase (RT), has been limited by the rapid appearance of drug-resistant viruses.  Lys103Asn, Tyr181Cys, and Tyr188Leu are some of the most common RT mutations that cause resistance to NNRTIs in the clinic.  The authors report x-ray crystal structures for RT complexed with three different pyridinone derivs., R157208, R165481, and R221239, at 2.95, 2.9, and 2.43 Å resoln., resp.  All three ligands exhibit nanomolar or subnanomolar inhibitory activity against wild-type RT, but varying activities against drug-resistant mutants.  R165481 and R221239 differ from most NNRTIs in that binding does not involve significant contacts with Tyr181.  These compds. strongly inhibit wild-type HIV-1 RT and drug-resistant variants, including Tyr181Cys and Lys103Asn RT.  These properties result in part from an iodine atom on the pyridinone ring of both inhibitors that interacts with the main-chain carbonyl oxygen of Tyr188.  An acrylonitrile substituent on R165481 substantially improves the activity of the compd. against wild-type RT (and several mutants) and provides a way to generate novel inhibitors that could interact with conserved elements of HIV-1 RT at the polymerase catalytic site.  In R221239, there is a flexible linker to a furan ring that permits interactions with Val106, Phe227, and Pro236.  These contacts appear to enhance the inhibitory activity of R221239 against the HIV-1 strains that carry the Val106Ala, Tyr188Leu, and Phe227Cys mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJHWVjop0BXLVg90H21EOLACvtfcHk0li3sBWeT3RO_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlur3J&md5=6c88c3dc1cf348274d35f1b402e9aae6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjm0500323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0500323%26sid%3Dliteratum%253Aachs%26aulast%3DHimmel%26aufirst%3DD.%2BM.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBenjahad%26aufirst%3DA.%26aulast%3DOumouch%26aufirst%3DS.%26aulast%3DGuillemont%26aufirst%3DJ.%26aulast%3DCoupa%26aufirst%3DS.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DCsoka%26aufirst%3DI.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DAndries%26aufirst%3DK.%26aulast%3DNguyen%26aufirst%3DC.%2BH.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DCrystal%2520Structures%2520for%2520HIV-1%2520Reverse%2520Transcriptase%2520in%2520Complexes%2520with%2520Three%2520Pyridinone%2520Derivatives%253A%2520A%2520New%2520Class%2520of%2520Non-Nucleoside%2520Inhibitors%2520Effective%2520against%2520a%2520Broad%2520Range%2520of%2520Drug-Resistant%2520Strains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26issue%3D24%26spage%3D7582%26epage%3D7591%26doi%3D10.1021%2Fjm0500323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Duijn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbrugghe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorens, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spöhr, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schedler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annane, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen Van, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misset, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jereb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seme, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šifrer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomiç, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estevez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eijkemans, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, I.</span></span> <span> </span><span class="NLM_article-title">The Effects of Antibiotic Cycling and Mixing on Antibiotic Resistance in Intensive Care Units: A Cluster-Randomised Crossover Trial</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30056-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1016%2FS1473-3099%2818%2930056-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=29396000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A280%3ADC%252BC1MvmvV2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=401-409&issue=4&author=P.+J.+Van%0ADuijnauthor=W.+Verbruggheauthor=P.+G.+Jorensauthor=F.+Sp%C3%B6hrauthor=D.+Schedlerauthor=M.+Dejaauthor=A.+Rothbartauthor=D.+Annaneauthor=C.+Lawrenceauthor=J.-C.+Nguyen+Vanauthor=B.+Missetauthor=M.+Jerebauthor=K.+Semeauthor=F.+%C5%A0ifrerauthor=V.+Tomi%C3%A7author=F.+Estevezauthor=J.+Carneiroauthor=S.+Harbarthauthor=M.+J.+C.+Eijkemansauthor=M.+Bontenauthor=H.+Goossensauthor=S.+Malhotra-Kumarauthor=C.+Lammensauthor=J.+Vilaauthor=I.+Roca&title=The+Effects+of+Antibiotic+Cycling+and+Mixing+on+Antibiotic+Resistance+in+Intensive+Care+Units%3A+A+Cluster-Randomised+Crossover+Trial&doi=10.1016%2FS1473-3099%2818%2930056-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The effects of antibiotic cycling and mixing on antibiotic resistance in intensive care units: a cluster-randomised crossover trial</span></div><div class="casAuthors">van Duijn Pleun Joppe; Verbrugghe Walter; Jorens Philippe Germaine; Spohr Fabian; Schedler Dirk; Deja Maria; Rothbart Andreas; Annane Djillali; Lawrence Christine; Nguyen Van Jean-Claude; Misset Benoit; Jereb Matjaz; Seme Katja; Sifrer Franc; Tomic Viktorija; Estevez Francisco; Carneiro Jandira; Harbarth Stephan; Eijkemans Marinus Johannes Cornelis; Bonten Marc</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-409</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Whether antibiotic rotation strategies reduce prevalence of antibiotic-resistant, Gram-negative bacteria in intensive care units (ICUs) has not been accurately established.  We aimed to assess whether cycling of antibiotics compared with a mixing strategy (changing antibiotic to an alternative class for each consecutive patient) would reduce the prevalence of antibiotic-resistant, Gram-negative bacteria in European intensive care units (ICUs).  METHODS:  In a cluster-randomised crossover study, we randomly assigned ICUs to use one of three antibiotic groups (third-generation or fourth-generation cephalosporins, piperacillin-tazobactam, and carbapenems) as preferred empirical treatment during 6-week periods (cycling) or to change preference after every consecutively treated patient (mixing).  Computer-based randomisation of intervention and rotated antibiotic sequence was done centrally.  Cycling or mixing was applied for 9 months; then, following a washout period, the alternative strategy was implemented.  We defined antibiotic-resistant, Gram-negative bacteria as Enterobacteriaceae with extended-spectrum β-lactamase production or piperacillin-tazobactam resistance, and Acinetobacter spp and Pseudomonas aeruginosa with piperacillin-tazobactam or carbapenem resistance.  Data were collected for all admissions during the study.  The primary endpoint was average, unit-wide, monthly point prevalence of antibiotic-resistant, Gram-negative bacteria in respiratory and perineal swabs with adjustment for potential confounders.  This trial is registered with ClinicalTrials.gov, number NCT01293071.  FINDINGS:  Eight ICUs (from Belgium, France, Germany, Portugal, and Slovenia) were randomly assigned and patients enrolled from June 27, 2011, to Feb 16, 2014. 4069 patients were admitted during the cycling periods in total and 4707 were admitted during the mixing periods.  Of these, 745 patients during cycling and 853 patients during mixing were present during the monthly point-prevalence surveys, and were included in the main analysis.  Mean prevalence of the composite primary endpoint was 23% (168/745) during cycling and 22% (184/853) during mixing (p=0·64), yielding an adjusted incidence rate ratio during mixing of 1·039 (95% CI 0·837-1·291; p=0·73).  There was no difference in all-cause in-ICU mortality between intervention periods.  INTERPRETATION:  Antibiotic cycling does not reduce the prevalence of carriage of antibiotic-resistant, Gram-negative bacteria in patients admitted to the ICU.  FUNDING:  European Union Seventh Framework Programme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWfgjEsV9PxiCEBeYuoi7AfW6udTcc2eZPKpISb8poA7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvmvV2ksQ%253D%253D&md5=b09911c1f08c5a6d97b01bb102a28916</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930056-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930056-2%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDuijn%26aufirst%3DP.%2BJ.%26aulast%3DVerbrugghe%26aufirst%3DW.%26aulast%3DJorens%26aufirst%3DP.%2BG.%26aulast%3DSp%25C3%25B6hr%26aufirst%3DF.%26aulast%3DSchedler%26aufirst%3DD.%26aulast%3DDeja%26aufirst%3DM.%26aulast%3DRothbart%26aufirst%3DA.%26aulast%3DAnnane%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DC.%26aulast%3DNguyen%2BVan%26aufirst%3DJ.-C.%26aulast%3DMisset%26aufirst%3DB.%26aulast%3DJereb%26aufirst%3DM.%26aulast%3DSeme%26aufirst%3DK.%26aulast%3D%25C5%25A0ifrer%26aufirst%3DF.%26aulast%3DTomi%25C3%25A7%26aufirst%3DV.%26aulast%3DEstevez%26aufirst%3DF.%26aulast%3DCarneiro%26aufirst%3DJ.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DEijkemans%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DBonten%26aufirst%3DM.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DLammens%26aufirst%3DC.%26aulast%3DVila%26aufirst%3DJ.%26aulast%3DRoca%26aufirst%3DI.%26atitle%3DThe%2520Effects%2520of%2520Antibiotic%2520Cycling%2520and%2520Mixing%2520on%2520Antibiotic%2520Resistance%2520in%2520Intensive%2520Care%2520Units%253A%2520A%2520Cluster-Randomised%2520Crossover%2520Trial%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26issue%3D4%26spage%3D401%26epage%3D409%26doi%3D10.1016%2FS1473-3099%2818%2930056-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beardmore, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peña-Miller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iredell, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Cycling and Antibiotic Mixing: Which One Best Mitigates Antibiotic Resistance?</span>. <i>Mol. Biol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">802</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1093/molbev/msw292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fmolbev%2Fmsw292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=28096304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFOmt7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2017&pages=802-817&issue=4&author=R.+E.+Beardmoreauthor=R.+Pe%C3%B1a-Millerauthor=F.+Goriauthor=J.+Iredell&title=Antibiotic+Cycling+and+Antibiotic+Mixing%3A+Which+One+Best+Mitigates+Antibiotic+Resistance%3F&doi=10.1093%2Fmolbev%2Fmsw292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic cycling and antibiotic mixing: which one best mitigates antibiotic resistance?</span></div><div class="casAuthors">Beardmore, Robert Eric; Pena-Miller, Rafael; Gori, Fabio; Iredell, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology and Evolution</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">802-817</span>CODEN:
                <span class="NLM_cas:coden">MBEVEO</span>;
        ISSN:<span class="NLM_cas:issn">1537-1719</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Can we exploit our burgeoning understanding of mol. evolution to slow the progress of drug resistance.  One role of an infection clinician is exactly that: to foresee trajectories to resistance during antibiotic treatment and to hinder that evolutionary course.  But can this be done at a hospital-wide scale.  Clinicians and theoreticians tried to when they proposed two conflicting behavioral strategies that are expected to curb resistance evolution in the clinic, these are known as "antibiotic cycling" and "antibiotic mixing." However, the accumulated data from clin. trials, now approaching 4 million patient days of treatment, is too variable for cycling or mixing to be deemed successful.  The former implements the restriction and prioritization of different antibiotics at different times in hospitals in a manner said to "cycle" between them.  In antibiotic mixing, appropriate antibiotics are allocated to patients but randomly.  Mixing results in no correlation, in time or across patients, in the drugs used for treatment which is why theorists saw this as an optimal behavioral strategy.  So while cycling and mixing were proposed as ways of controlling evolution, we show there is good reason why clin. datasets cannot choose between them: by re-examg. the theor. literature we show prior support for the theor. optimality of mixing was misplaced.  Our anal. is consistent with a pattern emerging in data: neither cycling or mixing is a priori better than the other at mitigating selection for antibiotic resistance in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKctTVU3331rVg90H21EOLACvtfcHk0lix_nnrctpIGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFOmt7%252FM&md5=c6e039f95e2d03c21eee22b16f2daad0</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1093%2Fmolbev%2Fmsw292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fmolbev%252Fmsw292%26sid%3Dliteratum%253Aachs%26aulast%3DBeardmore%26aufirst%3DR.%2BE.%26aulast%3DPe%25C3%25B1a-Miller%26aufirst%3DR.%26aulast%3DGori%26aufirst%3DF.%26aulast%3DIredell%26aufirst%3DJ.%26atitle%3DAntibiotic%2520Cycling%2520and%2520Antibiotic%2520Mixing%253A%2520Which%2520One%2520Best%2520Mitigates%2520Antibiotic%2520Resistance%253F%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2017%26volume%3D34%26issue%3D4%26spage%3D802%26epage%3D817%26doi%3D10.1093%2Fmolbev%2Fmsw292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">OGEE v2: An Update of the Online Gene Essentiality Database with Special Focus on Differentially Essential Genes in Human Cancer Cell Lines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D940</span>– <span class="NLM_lpage">D944</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkw1013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=27799467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D940-D944&issue=D1&author=W.-H.+Chenauthor=G.+Luauthor=X.+Chenauthor=X.-M.+Zhaoauthor=P.+Bork&title=OGEE+v2%3A+An+Update+of+the+Online+Gene+Essentiality+Database+with+Special+Focus+on+Differentially+Essential+Genes+in+Human+Cancer+Cell+Lines&doi=10.1093%2Fnar%2Fgkw1013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">GEE v2: an update of the online gene essentiality database with special focus on differentially essential genes in human cancer cell lines</span></div><div class="casAuthors">Chen, Wei-Hua; Lu, Guanting; Chen, Xiao; Zhao, Xing-Ming; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D940-D944</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  OGEE is an Online GEne Essentiality database.  To enhance our understanding of the essentiality of genes, in OGEE we collected exptl. tested essential and non-essential genes, as well as assocd. gene properties known to contribute to gene essentiality.  We focus on large-scale expts., and complement our data with text-mining results.  We organized tested genes into data sets according to their sources, and tagged those with variable essentiality statuses across data sets as conditionally essential genes, intending to highlight the complex interplay between gene functions and environments/exptl. perturbations.  Developments since the last public release include increased nos. of species and gene essentiality data sets, inclusion of non-coding essential sequences and genes with intermediate essentiality statuses.  In addn., we included 16 essentiality data sets from cancer cell lines, corresponding to 9 human cancers; with OGEE, users can easily explore the shared and differentially essential genes within and between cancer types.  These genes, esp. those derived from cell lines that are similar to tumor samples, could reveal the oncogenic drivers, paralogous gene expression pattern and chromosomal structure of the corresponding cancer types, and can be further screened to identify targets for cancer therapy and/or new drug development.  OGEE is freely available at http:// ogee.medgenius.info.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm3bqO4MARdrVg90H21EOLACvtfcHk0lix_nnrctpIGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhsLc%253D&md5=0df15fc6e5627f11a3c418b654d6e4b0</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1013%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.-H.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DX.-M.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DOGEE%2520v2%253A%2520An%2520Update%2520of%2520the%2520Online%2520Gene%2520Essentiality%2520Database%2520with%2520Special%2520Focus%2520on%2520Differentially%2520Essential%2520Genes%2520in%2520Human%2520Cancer%2520Cell%2520Lines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD940%26epage%3DD944%26doi%3D10.1093%2Fnar%2Fgkw1013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.-B.</span></span> <span> </span><span class="NLM_article-title">CEG: A Database of Essential Gene Clusters</span>. <i>BMC Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">769</span>, <span class="refDoi"> DOI: 10.1186/1471-2164-14-769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1471-2164-14-769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24209780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKnu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=769&author=Y.-N.+Yeauthor=Z.-G.+Huaauthor=J.+Huangauthor=N.+Raoauthor=F.-B.+Guo&title=CEG%3A+A+Database+of+Essential+Gene+Clusters&doi=10.1186%2F1471-2164-14-769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">CEG: a database of essential gene clusters</span></div><div class="casAuthors">Ye, Yuan-Nong; Hua, Zhi-Gang; Huang, Jian; Rao, Nini; Guo, Feng-Biao</div><div class="citationInfo"><span class="NLM_cas:title">BMC Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769</span>CODEN:
                <span class="NLM_cas:coden">BGMEET</span>;
        ISSN:<span class="NLM_cas:issn">1471-2164</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Essential genes are indispensable for the survival of living entities.  They are the cornerstones of synthetic biol., and are potential candidate targets for antimicrobial and vaccine design.  Description: Here we describe the Cluster of Essential Genes (CEG) database, which contains clusters of orthologous essential genes.  Based on the size of a cluster, users can easily decide whether an essential gene is conserved in multiple bacterial species or is species-specific.  It contains the similarity value of every essential gene cluster against human proteins or genes.  The CEG_Match tool is based on the CEG database, and was developed for prediction of essential genes according to function.  The database is available at online.  Conclusions: Properties contained in the CEG database, such as cluster size, and the similarity of essential gene clusters against human proteins or genes, are very important for evolutionary research and drug design.  An advantage of CEG is that it clusters essential genes based on function, and therefore decreases false pos. results when predicting essential genes in comparison with using the similarity alignment method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZhiPu_sGPJLVg90H21EOLACvtfcHk0lhzOgdI2O7pYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKnu7zL&md5=f3680dd9347640fc61682e7f9204cb72</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1186%2F1471-2164-14-769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2164-14-769%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.-N.%26aulast%3DHua%26aufirst%3DZ.-G.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DRao%26aufirst%3DN.%26aulast%3DGuo%26aufirst%3DF.-B.%26atitle%3DCEG%253A%2520A%2520Database%2520of%2520Essential%2520Gene%2520Clusters%26jtitle%3DBMC%2520Genomics%26date%3D2013%26volume%3D14%26spage%3D769%26doi%3D10.1186%2F1471-2164-14-769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">DEG 10, an Update of the Database of Essential Genes That Includes Both Protein-Coding Genes and Noncoding Genomic Elements</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D574</span>– <span class="NLM_lpage">D580</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1093%2Fnar%2Fgkt1131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=24243843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1am" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D574-D580&issue=D1&author=H.+Luoauthor=Y.+Linauthor=F.+Gaoauthor=C.-T.+Zhangauthor=R.+Zhang&title=DEG+10%2C+an+Update+of+the+Database+of+Essential+Genes+That+Includes+Both+Protein-Coding+Genes+and+Noncoding+Genomic+Elements&doi=10.1093%2Fnar%2Fgkt1131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements</span></div><div class="casAuthors">Luo, Hao; Lin, Yan; Gao, Feng; Zhang, Chun-Ting; Zhang, Ren</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D574-D580</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The combination of high-d. transposon-mediated mutagenesis and high-throughput sequencing has led to significant advancements in research on essential genes, resulting in a dramatic increase in the no. of identified prokaryotic essential genes under diverse conditions and a revised essential-gene concept that includes all essential genomic elements, rather than focusing on protein-coding genes only.  DEG 10, a new release of the Database of Essential Genes (available at http://www.essentialgene.org), has been developed to accommodate these quant. and qual. advancements.  In addn. to increasing the no. of bacterial and archaeal essential genes detd. by genome-wide gene essentiality screens, DEG 10 also harbors essential noncoding RNAs, promoters, regulatory sequences and replication origins.  These essential genomic elements are detd. not only in vitro, but also in vivo, under diverse conditions including those for survival, pathogenesis and antibiotic resistance.  We have developed customizable BLAST tools that allow users to perform species- and expt.-specific BLAST searches for a single gene, a list of genes, annotated or unannotated genomes.  Therefore, DEG 10 includes essential genomic elements under different conditions in three domains of life, with customizable BLAST tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxIBwiW86_xLVg90H21EOLACvtfcHk0lhzOgdI2O7pYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1am&md5=f7ddd7a094ab324d43dedb4c54c8c138</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1131%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DC.-T.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DDEG%252010%252C%2520an%2520Update%2520of%2520the%2520Database%2520of%2520Essential%2520Genes%2520That%2520Includes%2520Both%2520Protein-Coding%2520Genes%2520and%2520Noncoding%2520Genomic%2520Elements%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD574%26epage%3DD580%26doi%3D10.1093%2Fnar%2Fgkt1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&issue=13&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94A+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lhzOgdI2O7pYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26issue%3D13%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Boyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Open Babel: An Open Chemical Toolbox</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+Open+Chemical+Toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0lhzOgdI2O7pYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520Open%2520Chemical%2520Toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">Docking Validation Resources: Protein Family and Ligand Flexibility Experiments</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">2000</span>, <span class="refDoi"> DOI: 10.1021/ci1001982</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1001982" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGktLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1986-2000&issue=11&author=S.+Mukherjeeauthor=T.+E.+Baliusauthor=R.+C.+Rizzo&title=Docking+Validation+Resources%3A+Protein+Family+and+Ligand+Flexibility+Experiments&doi=10.1021%2Fci1001982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Validation Resources: Protein Family and Ligand Flexibility Experiments</span></div><div class="casAuthors">Mukherjee, Sudipto; Balius, Trent E.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1986-2000</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A database consisting of 780 ligand-receptor complexes, termed SB2010, has been derived from the Protein Databank to evaluate the accuracy of docking protocols for regenerating bound ligand conformations.  The goal is to provide easily accessible community resources for development of improved procedures to aid virtual screening for ligands with a wide range of flexibilities.  Three core expts. using the program DOCK, which employ rigid (RGD), fixed anchor (FAD), and flexible (FLX) protocols, were used to gauge performance by several different metrics: (1) global results, (2) ligand flexibility, (3) protein family, and (4) cross-docking.  Global spectrum plots of successes and failures vs rmsd reveal well-defined inflection regions, which suggest the commonly used 2 Å criteria is a reasonable choice for defining success.  Across all 780 systems, success tracks with the relative difficulty of the calcns.: RGD (82.3%) > FAD (78.1%) > FLX (63.8%).  In general, failures due to scoring strongly outweigh those due to sampling.  Subsets of SB2010 grouped by ligand flexibility (7-or-less, 8-to-15, and 15-plus rotatable bonds) reveal that success degrades linearly for FAD and FLX protocols, in contrast to RGD, which remains const.  Despite the challenges assocd. with FLX anchor orientation and on-the-fly flexible growth, success rates for the 7-or-less (74.5%) and, in particular, the 8-to-15 (55.2%) subset are encouraging.  Poorer results for the very flexible 15-plus set (39.3%) indicate substantial room for improvement.  Family-based success appears largely independent of ligand flexibility, suggesting a strong dependence on the binding site environment.  For example, zinc-contg. proteins are generally problematic, despite moderately flexible ligands.  Finally, representative cross-docking examples, for carbonic anhydrase, thermolysin, and neuraminidase families, show the utility of family-based anal. for rapid identification of particularly good or bad docking trends, and the type of failures involved (scoring/sampling), which will likely be of interest to researchers making specific receptor choices for virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp2rTY0qXg77Vg90H21EOLACvtfcHk0li0hhktzi7M0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGktLzL&md5=ed08ece5b658b52a83361a2ffb92ca83</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci1001982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1001982%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDocking%2520Validation%2520Resources%253A%2520Protein%2520Family%2520and%2520Ligand%2520Flexibility%2520Experiments%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D11%26spage%3D1986%26epage%3D2000%26doi%3D10.1021%2Fci1001982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benjamini, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochberg, Y.</span></span> <span> </span><span class="NLM_article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>. <i>J. R. Stat. Soc. Ser. B Methodol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1111/j.2517-6161.1995.tb02031.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&issue=1&author=Y.+Benjaminiauthor=Y.+Hochberg&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing&doi=10.1111%2Fj.2517-6161.1995.tb02031.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.2517-6161.1995.tb02031.x%26sid%3Dliteratum%253Aachs%26aulast%3DBenjamini%26aufirst%3DY.%26aulast%3DHochberg%26aufirst%3DY.%26atitle%3DControlling%2520the%2520False%2520Discovery%2520Rate%253A%2520A%2520Practical%2520and%2520Powerful%2520Approach%2520to%2520Multiple%2520Testing%26jtitle%3DJ.%2520R.%2520Stat.%2520Soc.%2520Ser.%2520B%2520Methodol.%26date%3D1995%26volume%3D57%26issue%3D1%26spage%3D289%26epage%3D300%26doi%3D10.1111%2Fj.2517-6161.1995.tb02031.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3bm1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3bm1','PDB','3bm1'); return false;">PDB: 3bm1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ebl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ebl','PDB','1ebl'); return false;">PDB: 1ebl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wyk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wyk','PDB','2wyk'); return false;">PDB: 2wyk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4nf0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4nf0','PDB','4nf0'); return false;">PDB: 4nf0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i19"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00220">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65750"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00220">10.1021/acs.jmedchem.9b00220</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Six tables of data on receptors, antibiotics, de novo sites, proteins, and PAINS (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Two data sets (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip">ZIP</a>)</p></li><li><p class="inline">Twelve figures showing comparison of ligands, performance, chemical characteristics, weight–frequency relationships, gene ontology analysis results, sampled molecules, optimization steps, and globularity results (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_001.xlsx">jm9b00220_si_001.xlsx (164.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_002.zip">jm9b00220_si_002.zip (6.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00220/suppl_file/jm9b00220_si_003.pdf">jm9b00220_si_003.pdf (3.39 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00220&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00220%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00220" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679998c0bf513de3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
