
<html lang="en"     class="pb-page"  data-request-id="e12cc780-d192-41d4-8006-e52cf9cc8041"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2013.56.issue-4;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/jm301856j"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Development of Bacteriostatic DNA Aptamers for Salmonella" /></meta><meta name="dc.Creator" content="Olga S.  Kolovskaya" /></meta><meta name="dc.Creator" content="Anna G.  Savitskaya" /></meta><meta name="dc.Creator" content="Tatiana N.  Zamay" /></meta><meta name="dc.Creator" content="Irina T.  Reshetneva" /></meta><meta name="dc.Creator" content="Galina S.  Zamay" /></meta><meta name="dc.Creator" content="Evgeny N.  Erkaev" /></meta><meta name="dc.Creator" content="Xiaoyan  Wang" /></meta><meta name="dc.Creator" content="Mohamed  Wehbe" /></meta><meta name="dc.Creator" content="Alla B.  Salmina" /></meta><meta name="dc.Creator" content="Olga V.  Perianova" /></meta><meta name="dc.Creator" content="Olga A.  Zubkova" /></meta><meta name="dc.Creator" content="Ekaterina A.  Spivak" /></meta><meta name="dc.Creator" content="Vasily S.  Mezko" /></meta><meta name="dc.Creator" content="Yury E.  Glazyrin" /></meta><meta name="dc.Creator" content="Nadezhda M.  Titova" /></meta><meta name="dc.Creator" content="Maxim V.  Berezovski" /></meta><meta name="dc.Creator" content="Anna S.  Zamay" /></meta><meta name="dc.Description" content="Salmonella is one of the most dangerous and common food-borne pathogens. The overuse of antibiotics for disease prevention has led to the development of multidrug resistant Salmonella. Now, more th..." /></meta><meta name="Description" content="Salmonella is one of the most dangerous and common food-borne pathogens. The overuse of antibiotics for disease prevention has led to the development of multidrug resistant Salmonella. Now, more th..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 19, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm301856j" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm301856j" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm301856j" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm301856j" /></link>
        
    
    

<title>Development of Bacteriostatic DNA Aptamers for Salmonella | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm301856j" /></meta><meta property="og:title" content="Development of Bacteriostatic DNA Aptamers for Salmonella" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0008.jpeg" /></meta><meta property="og:description" content="Salmonella is one of the most dangerous and common food-borne pathogens. The overuse of antibiotics for disease prevention has led to the development of multidrug resistant Salmonella. Now, more than ever, there is a need for new antimicrobial drugs to combat these resistant bacteria. Aptamers have grown in popularity since their discovery, and their properties make them attractive candidates for therapeutic use. In this work, we describe the selection of highly specific DNA aptamers to S. enteritidis and S. typhimurium. To evolve species-specific aptamers, twelve rounds of selection to live S. enteritidis and S. typhimurium were performed, alternating with a negative selection against a mixture of related pathogens. Studies have shown that synthetic pools combined from individual aptamers have the capacity to inhibit growth of S. enteritidis and S. typhimurium in bacterial cultures; this was the result of a decrease in their membrane potential." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm301856j"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm301856j">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm301856j&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm301856j&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm301856j&amp;href=/doi/10.1021/jm301856j" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1564-1572</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm301788a" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm3013882" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of Bacteriostatic DNA Aptamers for Salmonella</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Olga+S.++Kolovskaya">Olga S. Kolovskaya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna+G.++Savitskaya">Anna G. Savitskaya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tatiana+N.++Zamay">Tatiana N. Zamay</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Irina+T.++Reshetneva">Irina T. Reshetneva</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Galina+S.++Zamay">Galina S. Zamay</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Evgeny+N.++Erkaev">Evgeny N. Erkaev</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyan++Wang">Xiaoyan Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mohamed++Wehbe">Mohamed Wehbe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alla+B.++Salmina">Alla B. Salmina</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Olga+V.++Perianova">Olga V. Perianova</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Olga+A.++Zubkova">Olga A. Zubkova</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ekaterina+A.++Spivak">Ekaterina A. Spivak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vasily+S.++Mezko">Vasily S. Mezko</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yury+E.++Glazyrin">Yury E. Glazyrin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nadezhda+M.++Titova">Nadezhda M. Titova</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maxim+V.++Berezovski">Maxim V. Berezovski</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna+S.++Zamay">Anna S. Zamay</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Research Institute of Molecular Medicine and Pathobiochemistry, <sup>§</sup>Department of Biological Chemistry, and <sup>∥</sup>Department of Microbiology, Krasnoyarsk State Medical University, 1 Partizana Zheleznyaka str., Krasnoyarsk 660022, Russia</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario K1N 6N5, Canada</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">#</span> <span class="aff-text">Department of Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk 660041, Russia</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#f1909f9f908b909c9088b188909f959489df8384"><span class="__cf_email__" data-cfemail="12737c7c7368737f736b526b737c76776a3c6067">[email protected]</span></a>, <a href="/cdn-cgi/l/email-protection#3455474e5559554d745359555d581a575b59"><span class="__cf_email__" data-cfemail="d8b9aba2b9b5b9a198bfb5b9b1b4f6bbb7b5">[email protected]</span></a>. Phone: +7-903-923-84-02.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm301856j&amp;href=/doi/10.1021%2Fjm301856j" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1564–1572</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 7, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 September 2012</li><li><span class="item_label"><b>Published</b> online</span>19 February 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 February 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm301856j" title="DOI URL">https://doi.org/10.1021/jm301856j</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1564%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DOlga%2BS.%2BKolovskaya%252C%2BAnna%2BG.%2BSavitskaya%252C%2BTatiana%2BN.%2BZamay%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D4%26contentID%3Djm301856j%26title%3DDevelopment%2Bof%2BBacteriostatic%2BDNA%2BAptamers%2Bfor%2BSalmonella%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1572%26publicationDate%3DFebruary%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm301856j"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1783</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">54</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm301856j" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of Bacteriostatic DNA Aptamers for Salmonella&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Olga&quot;,&quot;last_name&quot;:&quot;S. Kolovskaya&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;G. Savitskaya&quot;},{&quot;first_name&quot;:&quot;Tatiana&quot;,&quot;last_name&quot;:&quot;N. Zamay&quot;},{&quot;first_name&quot;:&quot;Irina&quot;,&quot;last_name&quot;:&quot;T. Reshetneva&quot;},{&quot;first_name&quot;:&quot;Galina&quot;,&quot;last_name&quot;:&quot;S. Zamay&quot;},{&quot;first_name&quot;:&quot;Evgeny&quot;,&quot;last_name&quot;:&quot;N. Erkaev&quot;},{&quot;first_name&quot;:&quot;Xiaoyan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Mohamed&quot;,&quot;last_name&quot;:&quot;Wehbe&quot;},{&quot;first_name&quot;:&quot;Alla&quot;,&quot;last_name&quot;:&quot;B. Salmina&quot;},{&quot;first_name&quot;:&quot;Olga&quot;,&quot;last_name&quot;:&quot;V. Perianova&quot;},{&quot;first_name&quot;:&quot;Olga&quot;,&quot;last_name&quot;:&quot;A. Zubkova&quot;},{&quot;first_name&quot;:&quot;Ekaterina&quot;,&quot;last_name&quot;:&quot;A. Spivak&quot;},{&quot;first_name&quot;:&quot;Vasily&quot;,&quot;last_name&quot;:&quot;S. Mezko&quot;},{&quot;first_name&quot;:&quot;Yury&quot;,&quot;last_name&quot;:&quot;E. Glazyrin&quot;},{&quot;first_name&quot;:&quot;Nadezhda&quot;,&quot;last_name&quot;:&quot;M. Titova&quot;},{&quot;first_name&quot;:&quot;Maxim&quot;,&quot;last_name&quot;:&quot;V. Berezovski&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;S. Zamay&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;1564-1572&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm301856j&quot;},&quot;abstract&quot;:&quot;Salmonella is one of the most dangerous and common food-borne pathogens. The overuse of antibiotics for disease prevention has led to the development of multidrug resistant Salmonella. Now, more than ever, there is a need for new antimicrobial drugs to combat these resistant bacteria. Aptamers have grown in popularity since their discovery, and their properties make them attractive candidates for therapeutic use. In this work, we describe the selection of highly specific DNA aptamers to S. enteritidis and S. typhimurium. To evolve species-specific aptamers, twelve rounds of selection to live S. enteritidis and S. typhimurium were performed, alternating with a negative selection against a mixture of related pathogens. Studies have shown that synthetic pools combined from individual aptamers have the capacity to inhibit growth of S. enteritidis and S. typhimurium in bacterial cultures; this was the result of a decrease in their membrane potential.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301856j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301856j" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301856j&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301856j" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301856j&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301856j" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm301856j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301856j&amp;href=/doi/10.1021/jm301856j" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm301856j" /></input><a href="/doi/pdf/10.1021/jm301856j" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm301856j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm301856j%26sid%3Dliteratum%253Aachs%26pmid%3D23387511%26genre%3Darticle%26aulast%3DKolovskaya%26date%3D2013%26atitle%3DDevelopment%2Bof%2BBacteriostatic%2BDNA%2BAptamers%2Bfor%2BSalmonella%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D4%26spage%3D1564%26epage%3D1572%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jmcmar.2013.56.issue-4/production/jmcmar.2013.56.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Salmonella</i> is one of the most dangerous and common food-borne pathogens. The overuse of antibiotics for disease prevention has led to the development of multidrug resistant <i>Salmonella</i>. Now, more than ever, there is a need for new antimicrobial drugs to combat these resistant bacteria. Aptamers have grown in popularity since their discovery, and their properties make them attractive candidates for therapeutic use. In this work, we describe the selection of highly specific DNA aptamers to <i>S. enteritidis</i> and <i>S. typhimurium</i>. To evolve species-specific aptamers, twelve rounds of selection to live <i>S. enteritidis</i> and <i>S. typhimurium</i> were performed, alternating with a negative selection against a mixture of related pathogens. Studies have shown that synthetic pools combined from individual aptamers have the capacity to inhibit growth of <i>S. enteritidis</i> and <i>S. typhimurium</i> in bacterial cultures; this was the result of a decrease in their membrane potential.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Salmonella are a leading cause of food-borne illnesses and are considered to be one of the most dangerous pathogens to humans. Ingestion causes intoxication through an infectious process that often ends with typhoid fever, paratyphoid fever, gastroenteritis, liver cirrhosis, septic arthritis, meningitis, osteomyelitis, and pneumonia.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Currently, the most widespread <i>Salmonella</i> serovars belong to the <i>S. enterica</i> subspecies, specifically <i>S. typhimurium</i> and <i>S. enteritidis</i>. In humans, <i>S. typhimurium</i> causes diarrhea, abdominal cramps, vomiting, and nausea, which generally last up to 7 days. In the last 20 years, <i>S. enteritidis</i> has become the single most common cause of food poisoning. <i>S. enteritidis</i> is particularly adept at infecting chicken flocks without causing visible disease and spreading from hen to hen rapidly. Unfortunately, in immunocompromised persons, that is the elderly, young, or people with depressed immune systems, <i>Salmonella</i> infections are often fatal if not treated with antibiotics. In recent years, increasing antimicrobial resistance in <i>Salmonella enterica</i> has become a serious clinical problem worldwide. The majority of the Salmonella species have acquired multidrug resistance (MDR) to ampicillin, chloramphenicol, streptomycin, sulphonamides, and tetracyclines.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Therefore, the search for new antibiotics is of utmost importance as MDR strains continue to appear.</div><div class="NLM_p">Aptamers alone or coupled with antibiotics show prominent neutralizing properties to pathogenic bacteria such as <i>Bacillus cereus</i>,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a><i>Bacillus anthracis</i>,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a><i>Mycobacterium spp</i>.,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and multidrug resistant <i>Enterococcus spp</i>.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Aptamers are single-stranded DNA or RNA oligonucleotides selected from large random-sequence nucleic acid libraries for their selective affinity to different molecular targets. These molecules (5–25 kDa) have an identified primary sequence, and they can be synthesized by chemical or enzymatic procedures. They fold into well-defined three-dimensional structures<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> and show high affinity and specificity for their targets. Aptamers are often viewed as synthetic affinity probes with applications ranging from target detection to drug discovery and delivery. They can be chemically synthesized in high purity at low cost and are easily modified with different dyes and labels. Aptamers are thermostable and can be raised against toxic substances and single or multiple targets in complex mixtures (live cells,<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> cell lysates,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and viruses<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13-15)</a>). Lastly, aptamers are nonimmunogenic and nontoxic, thereby allowing for their application as drugs.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This is of considerable importance when discussing their potential use as antibiotic drugs. Indeed, one RNA aptamer (Macugen) directed against vascular endothelial growth factor was approved by the FDA,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and eight new aptamers are in clinical trials.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Few aptamers have been selected against different bacterial pathogens,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> including <i>Salmonella.</i><a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> In the study performed by the R. Joshi research group,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> aptamers for the outer membrane proteins isolated from <i>S. typhimurium</i> lysate were bound to magnetic beads and used to capture <i>S. typhimurium</i> in whole carcass chicken rinse samples, followed by detection using quantitative real-time RT-PCR.</div><div class="NLM_p last">In this study, we describe the selection of aptamers with high affinity and specificity for two types of live <i>Salmonella</i> serovars: <i>S. enteritidis</i> and <i>S. typhimurium</i> with different resistance to antibiotics. <i>S. typhimurium</i> was resistant to ampicillin, streptomycin, chloramphenicol, and amikacin, and <i>S. enteritidis</i> was resistant to tetracycline. Some of the selected aptamer clones showed a bacteriostatic effect against these <i>Salmonella</i> serotypes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44702" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Selection of Aptamers to Live <i>S. enteritidis</i> and <i>S. typhimurium</i></h3><div class="NLM_p">Aptamers for live <i>Salmonella</i> were selected based on a cell-SELEX (systematic evolution of ligands by exponential enrichment) scheme: an <i>in vitro</i> selection technique of nucleic acid binders to live cells, bacteria, and viruses from a random ssDNA library.<a onclick="showRef(event, 'ref13 ref14 ref23'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref23">(13, 14, 23)</a> A total of twelve rounds of selection against <i>S. enteritidis</i> were performed at 19–24 °C using the protocol schematically presented in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. The first round was positive selection to <i>S. enteritidis</i> involving 4 steps: (1) incubation of <i>S. enteritidis</i> with the ssDNA library or an enriched pool, (2) washing of unbound ssDNA, (3) extraction of bound ssDNA by heat denaturation, and (4) amplification by symmetric and asymmetric PCR to produce the enriched ssDNA aptamer pool. The second and each subsequent round were started from negative selection and consisted of 6 major steps: (1) incubation with a “negative” bacteria mixture: <i>S. typhimurium</i>, heat treated <i>S. enteritidis</i> and <i>S. typhimurium</i>, live <i>Staphylococcus aureus</i>, <i>Escherichia coli</i>,<i> Pseudomonas aeruginosa</i>, and <i>Citrobacter freundi</i>; (2) centrifugation and collection of a supernatant with the unbound DNA; (3) incubation of the supernatant with “positive” <i>S. enteritidis</i> cells; (4) washing of unbound ssDNA; (5) extraction of bound ssDNA; and (6) DNA amplification by symmetric and asymmetric PCR. The incorporation of <i>S. typhimurium</i> and other bacterial cells to the negative step of the selection ensured high specificity of aptamers to <i>S. enteritidis</i> serovar only. We also added salmon sperm DNA (0.1 mg L<sup>–1</sup>) as masking DNA in the positive selection step starting from the sixth round to eliminate nonspecific DNA binding and increase the stringency of aptamer selection. Aptamers to live <i>S. typhimurium</i> were selected similarly as described above. However, we used <i>S. typhimurium</i> in the positive steps and <i>S. enteritidis</i> in the negative steps.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Aptamer selection scheme for <i>S. enteritidis</i> and <i>S. typhimurium</i>. The first round included positive selection to <i>S. enteritidis</i> or <i>S. typhimurium</i>. The second and each subsequent round were started from negative selection and consisted of the following steps: incubation with bacteria mixture, <i>S. typhimurium</i> or <i>S. enteritidis</i>, heat treated Salmonellas, <i>S. aureus</i>, <i>E. coli</i>, <i>P. aeruginosa</i>, and <i>C. freundi</i>; collection of unbound sequences; incubation of collected ssDNA with <i>S. enteritidis</i> or <i>S. typhimurium</i>; washing of unbound ssDNA from <i>Salmonella</i>; extraction of bound ssDNA sequences; and ssDNA amplification by symmetric and asymmetric PCR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To analyze aptamer binding, fluorescently labeled aptamer pools were incubated with <i>Salmonella</i> and analyzed by flow cytometry. Intact bacteria were gated from cell debris and Dulbecco phosphate buffer saline (DPBS) particles as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. Relative binding analysis of DNA pools revealed that the affinity to bacteria started to increase after the fourth round and reached a maximum in the seventh round (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). It is interesting to note that the best aptamer pools, SE7 and ST7, demonstrate their maximum binding affinity to <i>S. enteritidis</i> and <i>S. typhimurium</i> cells, respectively, right after the introduction of masking DNA in the selection protocol. Dissociation constants (<i>K</i><sub>d</sub>) of SE7 and ST7 pools were measured in titration experiments using different concentrations of Alexa-488 labeled pools with a fixed amount of 50 × 10<sup>3</sup> colony forming units (CFU) of live <i>Salmonella</i> (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure S1). Apparent <i>K</i><sub>d</sub>’s of 7 nM and 25 nM were calculated for SE7 and ST7 pools, respectively, and they show high selectivity and good specificity. They did not bind to heat denatured <i>Salmonella</i> (95 °C for 15 min), intact <i>S. aureus</i>, <i>E. coli</i>, <i>P. aeruginosa</i>, and <i>C. freundi</i> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). The original ssDNA library did not show any significant binding to <i>Salmonella</i>. The aptamer pools SE7 and ST7 were cloned in <i>E. coli</i> and sequenced. A total of 23 unique DNA sequences for <i>S. enteritidis</i> and 22 for <i>S. typhimurium</i> (25 nM each) were evaluated by flow cytometry for their affinity to <i>Salmonella</i> (5 × 10<sup>4</sup> CFU). Ten full or truncated (without one or both primer binding sites that did not significantly influence estimated secondary structure and stability) aptamer clones for <i>S. enteritidis</i> and nine aptamer clones (full or truncated) for <i>S. typhimurium</i> were selected because of their high affinities. They were chemically synthesized and chosen for further antibacterial experiments. Sequences of these clones and the estimated secondary structures are presented in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>. All truncated aptamers show better binding abilities to salmonella cells, indicating that primer sites do not participate in binding recognition. This is likely the result of decreased negative charges and steric hindrance of the aptamer caused by the removal of the primer binding portion.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Flow cytometry binding analysis of the Alexa-488 labeled ssDNA to salmonellas. (A) Two-dimensional plot of side and forward scattering of buffer particles. (B) Gating for salmonella. (C, D) Histograms of ssDNA library and 7th pool binding to <i>S. enteritidis</i> and <i>S. typhimurium</i> for intact bacteria (red), ssDNA library with salmonella (green), and the pool with highest affinity (blue). #1 is the gate for Alexa-488 positive bacteria, and #2 is the gate for Alexa-488 negative bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Flow cytometry analysis of selected aptamer pools’ affinity to <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (B). 50 × 10<sup>3</sup> CFU was preincubated with 0.1 mg mL<sup>–1</sup> of salmon sperm DNA for 15 min at 19–24 °C. Afterward, bacteria were incubated with the 100 nM Alexa-488-labeled aptamer pool in 500 μL DPBS buffer for 15 min at 25 °C. The percentage of bacteria cells with bound ssDNA was analyzed on a flow cytometer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Some of the Best Sequences of Clones Isolated from the Seventh Pool for <i>S. enteritidis</i> and the Seventh Pool for <i>S. typhimurium</i><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0010.gif" alt="" id="GRAPHIC-d200e704-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">Clone name and estimated secondary structure are given as found by using OligoAnalyzer 3.1.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Antibacterial Effect of Aptamers</h3><div class="NLM_p">To evaluate the bactericidal properties of aptamers, we used the method proposed by A. Miles, S. Misra, and J. Irwin<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (also called “surface viable counting”) and determined the number of CFUs in the <i>Salmonella</i> suspension. The antibacterial effect of aptamers was estimated by the number of <i>Salmonella</i> colonies grown in Petri dishes after a 30-min incubation of the bacteria in DPBS with 1 μM of aptamer clones, pools, or ssDNA library and compared with the bacteria in DPBS (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). All experiments were done in triplicate. The results demonstrate the bacteria growth-suppressive effect of aptamer clones and pools. Apparent K<sub>d</sub>’s are presented in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. The inhibition caused was 43 ± 4% (SE-20, 60nt), and 62 ± 12% (ST-12, 60nt) in <i>S. enteritidis</i> and <i>S. typhimurium</i>,respectively. A pool of all synthetic clones inhibits growth of bacteria with highest efficacy: up to 73 ± 15% and 75 ± 15% for <i>S. enteritidis</i> and <i>S. typhimurium</i>,respectively. The ssDNA library did not significantly influence the colony growth. Interestingly, a few aptamer clones and a synthetic 80 nt oligonucleotide with a scrambled sequence slightly increased the growth of <i>Salmonella</i> colonies compared with the control experiment. Dose–response curves for the aptamers with bacteriostatic effect are presented in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> (Figure S2).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of aptamer pools on colony formation on Petri dishes with LB agar. (A) Bacterial culturability test after incubation of salmonella with aptamers. Aptamer pools, clones, or ssDNA library (1 μM) in DPBS were incubated with <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min. Bacteria were diluted, plated on Petri dishes with LB agar, and incubated for 18 h at 37 °C in a dry incubator (repeated in triplicate). The effect was estimated by counting the number of colonies. (B) Intact <i>S. typhimurium</i>; (C) <i>S. typhimurium</i> plated after a 30-min incubation with 1 μM of synthetic aptamer ST-12; (D) Intact <i>S. enteritidis</i>; (E) <i>S. enteritidis</i> plated after a 30-min incubation with 1 μM of synthetic aptamer SE-20. In order to get the same number of bacteria in the beginning of incubation, B and C, and D and E, were initially diluted from the same aliquot of <i>S. enteritidis</i> or <i>S. typhimurium</i>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antibacterial effect of aptamers on <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (B) showed by culturability and viability Rhodamine 123 tests. The symbols in the figure indicate the following: *, Pool SE combined from clones SE-6, SE-20, and SE-22; **, Random sequence: CTCCTCTGACTGTAACCACGGATTGTCAGCGCGCCGCCGGATCAAGCATTTCGACAATTCGCATAGGTAGTCCAGAAGCC; ***, Pool ST combined from clones ST-2, ST-12, ST-20, and ST-33; <sup>1</sup><i>Growth suppression.</i> 1 μM of ssDNA samples in DPBS were incubated with 10<sup>3</sup> CFU mL<sup>–1</sup> of <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min. Afterward, the mixture was diluted and plated on Petri dishes with LB Agar and incubated for 18 h at 37 °C in a dry incubator (done in triplicate). The effect was estimated by counting the number of colonies (<i>N</i>). The number of colonies (<i>N</i><sub>c</sub>) in control plates with intact <i>S. enteritidis</i> or <i>S. typhimurium</i> (incubated in DPBS without ssDNA) was considered as 100%. Growth suppression was calculated as 100 – (<i>N</i> × 100/<i>N</i><sub>c</sub>). <sup>2</sup><i>Cell wall electrochemical gradient decrease.</i> Samples of  1 μM ssDNA in DPBS were incubated with 5 × 10<sup>3</sup> CFU mL<sup>–1</sup><i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min and 18 h. EGTA at the final concentration 1 mM was added to samples to permeabilize the bacterial outer membrane; bacteria were then loaded with Rh123 and analyzed on a flow cytometer. The samples were loaded with Rh123 (2 μg/mL final concentration) for 15 min at 19–24 °C and analyzed. The median of the histogram was taken as a relative value of electrochemical gradient (EG). Samples with intact <i>S. typhimurium</i> or <i>S. enteritidis</i> incubated in DPBS without ssDNA and loaded with Rh123 (EGc) were considered as positive controls (100% viability). The electrochemical gradient decrease was calculated as EGc – EG. For example, EG for <i>S. typhimurium</i> was 28.9 relative units, and EG for heat treated <i>S. typhimurium</i> was 15.9 relative units; therefore, the EG decrease (EGc – EG) was 13 relative units.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, we tested <i>Salmonella</i> viability after aptamer treatment using Rhodamine 123 (Rh123) staining. Rh123 was used for the indication of the metabolic or vital state of the <i>S. typhimurium</i> cell population.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The dye enters the cells in a membrane potential-dependent fashion—the higher the membrane potential, the higher Rh123 staining. The popular bacterial viability dyes propidium iodide (PI) and SYTO<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> do not work well in the presence of aptamers due to nonspecific binding. The majority of aptamer clones and pools caused strong depolarization of bacterial cell wall after a 30-min incubation at 19–24 °C (Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a><b>)</b>. The ssDNA library and scrambled 80 nt sequence did not influence the bacterial cell wall potential. Incubation for an additional 18 h of <i>S. enteritidis</i> with aptamers SE-3, SE-11, SE-20, and SE-22 and SE pool resulted in irreversible cell wall depolarization. The same occurred in a case of 18-h incubation of ST-12, ST-33, and pool ST with <i>S. typhimurium</i>. For the clones ST-1 and ST-20, the membrane potential of <i>S. typhimurium</i> was recovered after an 18-h incubation (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Bacteria viability test based on Rhodamine 123 uptake. Samples of 1 μM ssDNA in DPBS were incubated with 5 × 10<sup>3</sup> CFU mL<sup>–1</sup> of <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min and 18 h. EGTA was added to samples in order to permeabilize the bacterial outer membrane; samples were then loaded with Rh123 for 15 min at 19–24 °C and measured on a flow cytometer. Samples with live intact bacteria and heat treated salmonella (10 min at 95 °C) in DPBS loaded with Rh123 were taken as a positive and a negative control. Flow cytometric analysis of Rh123 uptake by <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (D): intact salmonella (blue), unstained bacteria (red), heat treated bacteria (green), initial ssDNA library (purple), <i>S. enteritidis</i> (black) or <i>S. typhimurium</i> (orange) incubated for 30 min accordingly with pool SE or pool ST and aptamer clones SE-20 or ST-12.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74178" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74178" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work we describe the selection of highly specific DNA aptamers to <i>S. enteritidis</i> and <i>S. typhimurium via</i> Cell-SELEX technique. To evolve the species-specific aptamers, twelve rounds of selection to live <i>S. enteritidis</i> and <i>S. typhimurium</i> alternating with the negative selection against a mixture of related pathogens and heat denatured <i>Salmonella</i>, were performed. The incorporation of <i>S. typhimurium</i> or <i>S. enteritidis</i> to the negative step of selection is crucial for evolving highly selective aptamers to specific <i>S. enteritidis</i> or <i>S. typhimurium</i> serovars. To eliminate nonspecific DNA binding and increase the stringency of the aptamer selection, masking DNA was used in the positive selection. Incorporating different controls to increase selection strigency resulted with aptamers that showed strong affinity, with dissociation constants in the nanomolar range and high specificity to their respective salmonella serovar. A few truncated aptamers show better binding abilities after the removal of the primer sites, which are not expected to participate in binding recognition. Truncated 60 nt aptamer clones SE-20 and ST-12 had dissociation constants in the nanomolar range. Shorter ssDNA has less negative charge than the longer sequence. The cell wall of salmonella species is also negatively charged. We hypothesize that the truncation of primer binding sites that is not involved in binding makes aptamers less negative and improves their binding. Here we show a new functionality of aptamers where they behave as antibiotics against MDR <i>S. enteritidis</i> and <i>S. typhimurium</i>, as seen with clones SE-20 and ST-12, which have promising results for aptamer based antibiotics. They have inhibited growth of bacteria. However, the pool of all synthetic clones had the highest bacterial growth suppression. Certain aptamer clones and a scrambled sequence (80 nt) slightly stimulated the growth of <i>Salmonella</i> colonies compared to the control experiment. No correlation was found between the affinity and the biological effect of aptamers. Therefore, it could be assumed that aptamers with opposite bacterial effects have different binding sites. The competition experiments proved this for clones SE-3 and SE-6, SE-3 and SE-20, ST-6 and ST-12, revealing that aptamers with different biological effects did not replace each other from the bacterial cell wall (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S3">Supporting Information Figure S3</a>). But also, it appeared that aptamers ST-6 and ST-33, with opposite effects on bacteria, could share the same binding site, because the first one was displaced by the second one from the <i>S. typhimurium</i> surface. Sequences of aptamers with similar effects did not have any consensus motives (compared using Program BLASTN 2.2.27+). Meaningful nucleotide motifs, relevant aptamer targets, and molecular mechanisms of their interaction remain to be determined.</div><div class="NLM_p">The majority of aptamer clones and pools with bacteriostatic properties also caused strong depolarization of bacterial cell wall after a 30-min incubation at 19–24 °C. The ssDNA library and scrambled 80 nt sequence did not influence the bacterial cell wall potential. An additional 18-h incubation of <i>S. enteritidis</i> with aptamers SE-3, SE-11, SE-20, and SE-22 and the pool SE resulted in irreversible cell wall depolarization. A similar effect was observed with the 18-h incubation of aptamers ST-12 and ST-33 and pool ST with <i>S. typhimurium</i>. For clones ST-1 and ST-20, the membrane potential of <i>S. typhimurium</i> was recovered after an 18-h incubation. This could happen because MDR <i>S. typhimurium</i> is less sensitive than <i>S. enteritidis</i>. Thus, clones SE-6, SE-20, and SE-22 and pool SE, as well as clones ST-2, ST-12, ST-20, ST-33, and pool ST, have strong bacteriostatic effects on <i>S. enteritidis</i> and <i>S. typhimurium</i>, respectively. They cause the bacterial membrane depolarization and suppress the formation of colonies on agar plates. Molecular mechanisms of aptamer-induced bacteria membrane depolarization remain to be determined.</div><div class="NLM_p last">In summary, clones SE-6 and SE-20 and aptamer pool SE (the correspondent EC50s are 500, 400, and 650 nM), ST-12, ST-33, and pool ST (the correspondent EC50s are 250, 800, and 750 nM) have good antibacterial potential against <i>S. enteritidis</i> and <i>S. typhimurium</i>, respectively (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S2">Supporting Information Figure S2</a>), due to induction of cell membrane depolarization. Clones SE-3, SE-11, SE-34, ST-1, and ST-6, which did not show antibacterial properties, can still be used as affinity probes in flow cytometry or biosensors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21210" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21210" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Selection of aptamers was performed using a modified cell-SELEX technology by alternating positive and negative rounds of selection.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18-21)</a> Aptamers were selected against clinical strains of <i>S. enteritidis</i> and MDR <i>S. typhimurium</i> isolated in the Krasnoyarsk region (kindly provided by the Federal State Institution of Health “Center for Hygiene and Epidemiology of the Krasnoyarsk Territory”). For negative rounds of selection against <i>S. enteritidis</i>, the bacterial mixture contained <i>S. typhimurium</i>, <i>Staphylococcus aureus</i> (The American Type Culture Collection (ATCC) 25923), <i>Escherichia coli</i> (ATCC 25922), <i>Pseudomonas aeruginosa</i> (ATCC 27853), and the clinical strain of <i>Citrobacter freundi</i> (kindly provided by the Federal State Institution of Health “Center for Hygiene and Epidemiology of the Krasnoyarsk Territory”).</div><div class="NLM_p">The selection was started with a naı̈ve ssDNA library (Integrated DNA Technologies, USA) which consisted of a randomized region of 40 nucleotides (N40) flanked by two constant primer-hybridization sites, 5′-CTC CTC TGA CTG TAA CCA CG N40 GC ATA GGT AGT CCA GAA GCC-3′. Before each round of selection and binding experiments, the ssDNA library and aptamer pools were denatured by heating for 5 min at 95 °C in Dulbecco’s phosphate buffered saline (DPBS, Sigma-Aldrich, USA) and then renatured on ice for 10 min. For each round of positive selection, 3 × 10<sup>6</sup> CFU of the target salmonella was washed and resuspended in DPBS prior to use. The aptamer selection scheme for <i>S. enteritidis</i> and <i>S. typhimurium</i> is presented in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. For the first round of selection, negative selection steps were omitted and thus ST or SE were incubated (separately) with 500 μL of DPBS containing 1 μM (0.1 nmol or 6 × 10<sup>13</sup> sequences) ssDNA library for 30 min at 25 °C and then centrifuged at 3500<i>g</i> for 10 min at 25 °C, to remove unbound aptamers, followed by rinsing twice with DPBS; the pellet was resuspended in 95 μL of 10 mM Tris-HCl buffer containing 10 mM EDTA, pH 7.4 (TE) (Sigma-Aldrich, USA), and heated for 10 min at 95 °C to release the aptamers bound to the bacteria. After the denaturing step, bacterial debris was removed by centrifugation at 14000<i>g</i> for 15 min at 4 °C and the supernatant (containing the aptamers) was collected and stored at −20 °C. Subsequently, salmonella binding aptamers were amplified using symmetric and asymmetric PCR. For the symmetric PCR, 5 μL of the aptamer pool in TE was mixed with 45 μL of symmetric PCR Master Mix, containing the following: 1X PCR buffer, 2.5 mM MgCl<sub>2</sub>, 0.025 U μL<sup>–1</sup> KAPA2G HotStart Robust polymerase (KAPABiosystems, USA), 220 μM dNTPs, 300 nM forward primer (5′- CTC CTC TGA CTG TAA CCA CG-3′), and 300 nM reverse primer (5′- GGC TTC TGG ACT ACC TAT GC-3′) (Syntol, Russia). Afterward, asymmetric PCR was performed where 5 μL of the symmetric PCR product in TE buffer was mixed with 45 μL of the asymmetric PCR Master Mix containing the following: 1X PCR buffer, 2.5 mM MgCl<sub>2</sub>, 0.025 U μL<sup>–1</sup> KAPA2G HotStart Robust polymerase, 220 μM dNTPs, 1 μM forward Alexa-488 primer (5′-Alexa488- CTC CTC TGA CTG TAA CCA CG-3′), and 50 nM reverse primer (5′- GGC TTC TGG ACT ACC TAT GC-3′). Amplification was performed using the following PCR program: preheating for 2 min at 95 °C, 15 cycles of 30 s at 95 °C, 15 s at 56.3 °C, and 15 s at 72 °C. The concentration of the PCR product was estimated using 3% agar electrophoresis. Fluorescence ssDNA labeled with Alexa-488 was analyzed in the gel-documenting system GBOX/EF2-E. Evolved aptamer pools product was stored at −20 °C. For the subsequent rounds of selection, 100 nM of fluorescently labeled ssDNA aptamers was used for the negative selection, where 3 × 10<sup>6</sup> CFU of bacteria mixture containing <i>Staphylococcus aureus</i>, <i>Escherichia coli</i>, <i>Pseudomonas aeruginosa</i>, and a clinical stain of <i>Citrobacter freundi</i> in DPBS was used along with the opposite Salmonella serovar that we were selecting for. The bacteria mixture was incubated for 30 min at 25 °C in 500 μL of DPBS with 100 nM ssDNA aptamer candidates evolved from the previous round and then centrifuged at 3500<i>g</i> for 10 min at 25 °C to collect unbound sequences. Extracted aptamers were used for the next step of the positive selection (described above). The evolved aptamer pool was stored at −20 °C and used for the next round of selection following the aforementioned procedure, starting from the negative step. To remove nonspecific binding aptamers from the pool, another additional step was applied: starting from round 6 during positive selection, salmonella species were preincubated with masking DNA from salmon sperm.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Cloning and Sequencing of the Aptamer Pool</h3><div class="NLM_p last">Evolved aptamer pools were cloned to obtain unique aptamer sequences. The dsDNA pool was extracted from gel using the DNA gel extraction kit Axy prep (AxyGen Biosystems, USA) in order to purify the DNA from the PCR mixture. Cloning was performed according to the supplied protocol using a M13mp18 Perfectly Blunt Cloning Kit (Novagen, Germany). White colonies were collected into separate tubes, left to grow overnight in 1 mL of SOC medium (Cellgro, USA) at 37 °C, and then analyzed by PCR and agarose gel electrophoresis to ensure that the clones have the insert in the plasmid. For each PCR reaction, 2 μL of the cell suspension was mixed with 18 μL of the PCR Master Mix containing the following: 1X PCR buffer B, 2.5 mM MgCl<sub>2</sub> (Mallinckrodt Baker, Inc., USA), 0.025 U μL<sup>–1</sup> KAPA 2G Robust Hot Start DNA polymerase (Kapa Biosystems, USA), 220 μM dNTPs, 300 nM forward FAM primer, and 300 nM reverse primer. Amplification was performed using a PCR program (preheating for 5 min at 95 °C, 25–30 cycles of 30 s at 95 °C, 15 s at 56.3 °C, and 15 s at 72 °C). Plasmids were isolated using a GeneJET plasmid miniprep kit (Fermentas, USA), according to the supplied protocol. The part of the plasmid with the insert was amplified using M13 forward primer (5′-GTA AAA CGA CGG CCA GT-3′) and M13r everse primer (5′- AGC GGA TAA CAA TTT CAC ACA GG-3′), according to the following protocol. For each PCR reaction, 5 μL of the plasmid (∼10 ng μL<sup>–1</sup>) diluted in water was mixed with 45 μL of PCR Master Mix containing the following: 1X PCR buffer A, 2.5 mM MgCl<sub>2</sub>, 0.025 U μL<sup>–1</sup> KAPA 2G Robust Hot Start DNA polymerase, 220 μM dNTPs, 300 nM forward M13 primer, and 300 nM reverse M13 primer. Amplification was performed using a PCR program (preheating for 2 min at 95 °C, 25 cycles of 30 s at 95 °C, 15 s at 60 °C, and 15 s at 72 °C). Finally, the unpurified PCR product was sequenced at McGill University and Génome Québec Innovation Centre, Canada. Synthetic aptamer clones with and without a FAM label were purchased at Integrated DNA Technologies, USA.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Flow Cytometric Binding Analysis of Pools and Clones</h3><div class="NLM_p last">The affinities of the evolved aptamer pools against <i>S. enteritidis</i> and <i>S. typhimurium</i> were determined using an FC-500 flow cytometer (Beckman Coulter Inc., USA). Before incubating with aptamers, each sample contained 50 × 10<sup>3</sup> CFU <i>S. enteritidis</i> or <i>S. typhimurium</i> and was preincubated with 0.1 mg mL<sup>–1</sup> of salmon sperm DNA as a masking nucleic acid to suppress nonspecific binding for 15 min at 19–24 °C. Afterward, bacteria were incubated with 100 nM Alexa-488-labeled aptamer pool in 500 μL of DPBS buffer for 30 min at 25 °C. Control experiments were performed using the fluorescently labeled ssDNA library in DPBS. Subsequently, each sample was washed once to remove unbound DNA and then resuspended in 0.5 mL of DPBS and measured using flow cytometry for 30 000 events. The gate for intact <i>S. enteritidis</i> and <i>S. typhimurium</i> in DPBS was taken as background fluorescence. The fluorescence above the background was gated and considered to be the real fluorescence of the Alexa-488 labeled aptamers bound with bacteria (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Data analysis was performed using Kaluza 1.1 software.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Evaluation of the Antibacterial Effect of Aptamers on <i>S. enteritidis</i> and <i>S. typhimurium</i> by Colony Forming Unit Assay and Culturability Test</h3><div class="NLM_p last">A bacterial culture in LB nutrient broth (Sigma-Aldrich, USA) of <i>S. enteritidis</i> and <i>S. typhimurium</i> (10<sup>3</sup> CFU mL<sup>–1</sup>) was incubated with aptamer pools, clones, or the ssDNA library to a final concentration of 1 μM. In order to get the same number of bacteria in all samples, incubation samples were diluted from the same aliquot <i>S. enteritidis</i> or <i>S. typhimurium</i>, respectively. Control samples contained intact <i>S. enteritidis</i> or <i>S. typhimurium</i> incubated in DPBS without ssDNA. All samples were incubated for 30 min at 19–24 °C and then diluted and plated on Petri dishes with LB Agar (Sigma-Aldrich, USA) in triplicate; this was followed by a 24-h incubation at 37 °C to allow bacterial growth. The colonies were then counted to compare their growth. The number of colonies (<i>N</i><sub>c</sub>) in control plates was considered as 100%. Growth suppression was calculated as 100 – (<i>N</i> × 100/<i>N</i><sub>c</sub>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Bacteria Viability Estimated by Rhodamine 123</h3><div class="NLM_p last">In addition, the antibacterial effect of aptamers was also evaluated by Rhodamine 123 (Sigma-Aldrich, USA) penetration into bacteria, which allows for predicting the viability of bacteria because of their ability to accumulate this dye as a function of membrane potential. In live bacteria the membrane potential allows Rh123 to enter the cell, resulting in bacteria fluorescence in the green spectral region. Cell death leads to a decrease in membrane potential and reduction of input dye, thereby reducing the fluorescence intensity of bacteria. The fluorescent intensity of bacteria loaded with Rh123 was evaluated by flow cytometry (Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). Samples of ssDNA (1 μM each) in DPBS were incubated with 5 × 10<sup>3</sup> CFU mL<sup>–1</sup><i>S. enteritidis</i> or <i>S. typhimurium</i> for 30 min and 18 h. Afterward, ethylene glycol-bis(2-aminoethylether)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetraacetic acid (EGTA) (Sigma-Aldrich, USA) was added to samples to a final concentration of 1 mM before addition of dyes in order to permeabilize the bacterial outer membrane. Samples were loaded with Rh123 (2 μg mL<sup>–1</sup> final concentration) for 15 min at 19–24 °C and measured on a flow cytometer.<sup>30</sup> The resulting histogram was analyzed, and the median was used as a relative value of electrochemical gradient (EG). Samples with intact <i>S. enteritidis</i> or <i>S. typhimurium</i> were incubated in DPBS without ssDNA and loaded with Rh123 (EGc) as a positive control (100% viability). The electrochemical gradient of heat treated <i>Salmonella</i> (10 min at 95 °C) was taken as negative control. The electrochemical gradient decrease was calculated as EGc – EG.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12"><a href="/doi/suppl/10.1021/jm301856j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57522" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57522" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Dose–response curves for the aptamers with bacteriostatic effect; example of <i>K</i><sub>d</sub> determination; competition experiments between aptamer clones. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm301856j/suppl_file/jm301856j_si_001.pdf">jm301856j_si_001.pdf (1.0 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm301856j" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24246" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24246" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna S. Zamay</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b0d1deded1cad1ddd1c9f0c9d1ded4d5c89ec2c590d1c3cad1ddd1c9f0d7ddd1d9dc9ed3dfdd"><span class="__cf_email__" data-cfemail="89e8e7e7e8f3e8e4e8f0c9f0e8e7edecf1a7fbfc">[email protected]</span> <span class="__cf_email__" data-cfemail="c4a5b7bea5a9a5bd84a3a9a5ada8eaa7aba9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olga S. Kolovskaya</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna G. Savitskaya</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tatiana N. Zamay</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irina T. Reshetneva</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Galina S. Zamay</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Evgeny N. Erkaev</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyan Wang</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohamed Wehbe</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alla B. Salmina</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olga V. Perianova</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olga A. Zubkova</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ekaterina A. Spivak</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vasily S. Mezko</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yury E. Glazyrin</span> - <span class="hlFld-Affiliation affiliation">†Research
Institute of Molecular Medicine and Pathobiochemistry, §Department of Biological
Chemistry, and ∥Department of Microbiology, Krasnoyarsk
State Medical University, 1 Partizana Zheleznyaka str.,
Krasnoyarsk 660022, Russia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nadezhda M. Titova</span> - <span class="hlFld-Affiliation affiliation">Department of
Medical Biology, Siberian Federal University, 79 Svobodny, Krasnoyarsk
660041, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maxim V. Berezovski</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Ottawa, 10 Marie Curie, Ottawa, Ontario
K1N 6N5, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Olga S. Kolovskaya and Anna G. Savitskaya contributed equally to this work. The manuscript was written with contributions from all authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d200e1283-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84613" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84613" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Ministry Healthcare of the Russian Federation. The authors thank Professor Ivan P. Artyukhov, Dr. Maria Trusova, and Professor Andrey A. Savchenko for supporting the work, as well as Ms. Darija Muharemagic and Ms. Ana Gargaun for editing. Bacteria isolates were kindly provided by the Federal State Institution of Health “Center for Hygiene and Epidemiology of the Krasnoyarsk Territory”.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i15" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i15"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i16" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i16"> Abbreviations Used</h2><tr><td class="NLM_term">SELEX</td><td class="NLM_def"><p class="first last">systematic evolution of ligands by exponential enrichment</p></td></tr><tr><td class="NLM_term">DPBS</td><td class="NLM_def"><p class="first last">Dulbecco phosphate buffer saline</p></td></tr><tr><td class="NLM_term">CFU</td><td class="NLM_def"><p class="first last">colony forming units</p></td></tr><tr><td class="NLM_term">Rh123</td><td class="NLM_def"><p class="first last">Rhodamine 123</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug resistance</p></td></tr><tr><td class="NLM_term">ATCC</td><td class="NLM_def"><p class="first last">American Type Culture Collection</p></td></tr><tr><td class="NLM_term">EG</td><td class="NLM_def"><p class="first last">electrochemical gradient</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Pegues, D. A.; Ohl, M. E.; Miller, S. I.</span><span> </span><span class="NLM_article-title">Salmonella species, including Salmonella typhi</span>. In  <span class="citation_source-book">Principles and Practice of Infectious Diseases</span>, <span class="NLM_edition">6</span>th ed.; <span class="NLM_contrib-group">Mandell, G. L., Bennet, J. E., Dolin, R.</span>, Eds.; <span class="NLM_publisher-name">Churchill Livingstone</span>: <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">2636</span><span class="NLM_x">–</span> <span class="NLM_lpage">2654</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=2636-2654&author=D.+A.+Pegues&author=M.+E.+Ohl&author=S.+I.+Millerauthor=G.+L.+Mandell&author=J.+E.+Bennet&author=R.+Dolin&title=Principles+and+Practice+of+Infectious+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPegues%26aufirst%3DD.%2BA.%26atitle%3DSalmonella%2520species%252C%2520including%2520Salmonella%2520typhi%26btitle%3DPrinciples%2520and%2520Practice%2520of%2520Infectious%2520Diseases%26aulast%3DMandell%26aufirst%3DG.%2BL.%26pub%3DChurchill%2520Livingstone%26pub%3DElsevier%26date%3D2005%26spage%3D2636%26epage%3D2654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Helms, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ethelberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molbak, K.</span><span> </span><span class="NLM_article-title">International Salmonella typhimurium DT104 infections, 1992–2001</span> <span class="citation_source-journal">Emerging Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">867</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.3201%2Feid1106.041017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=15963280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A280%3ADC%252BD2MzhsFGqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=859-867&author=M.+Helmsauthor=S.+Ethelbergauthor=K.+Molbak&title=International+Salmonella+typhimurium+DT104+infections%2C+1992%E2%80%932001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">International Salmonella Typhimurium DT104 infections, 1992-2001</span></div><div class="casAuthors">Helms Morten; Ethelberg Steen; Molbak Kare</div><div class="citationInfo"><span class="NLM_cas:title">Emerging infectious diseases</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-67</span>
        ISSN:<span class="NLM_cas:issn">1080-6040</span>.
    </div><div class="casAbstract">The incidence of multidrug-resistant (MDR) Salmonella Typhimurium infections in humans, and in particular MDR definitive phage type 104 (DT104), has increased substantially in many countries in the last 2 decades, often associated with increased illness.  To examine the magnitude of this problem, a survey was conducted among countries with available antimicrobial resistance or phage typing surveillance data.  A total of 29, primarily industrialized, countries participated in the survey, which covered the years 1992-2001.  Overall, the incidence of MDR S.  Typhimurium and DT104 increased continuously during this period, although the problem affected primarily Europe and North America.  The increase appeared to have peaked in the United Kingdom but not in other countries.  Also, the incidence of quinolone-resistant S.  Typhimurium was increasing.  This survey implies that MDR S.  Typhimurium constitutes an increasing public health problem in large parts of the world and emphasizes the importance of surveillance and control programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8Hcwa-B2JQfIW7hznt0zpfW6udTcc2eaWHUnZfxUQ_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzhsFGqsw%253D%253D&md5=c0d32f308e43a01d8cac443356583d85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3201%2Feid1106.041017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3201%252Feid1106.041017%26sid%3Dliteratum%253Aachs%26aulast%3DHelms%26aufirst%3DM.%26aulast%3DEthelberg%26aufirst%3DS.%26aulast%3DMolbak%26aufirst%3DK.%26atitle%3DInternational%2520Salmonella%2520typhimurium%2520DT104%2520infections%252C%25201992%25E2%2580%25932001%26jtitle%3DEmerging%2520Infect.%2520Dis.%26date%3D2005%26volume%3D11%26spage%3D859%26epage%3D867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Pui, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyaletchumi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noor Hidayah, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farinazleen, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheah, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, R.</span><span> </span><span class="NLM_article-title">Salmonella: A foodborne pathogen</span> <span class="citation_source-journal">Int. Food Res. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">473</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=465-473&author=C.+F.+Puiauthor=W.+C.+Wongauthor=L.+C.+Chaiauthor=R.+Tunungauthor=P.+Jeyaletchumiauthor=M.+S.+Noor+Hidayahauthor=A.+Ubongauthor=M.+G.+Farinazleenauthor=Y.+K.+Cheahauthor=R.+Son&title=Salmonella%3A+A+foodborne+pathogen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPui%26aufirst%3DC.%2BF.%26aulast%3DWong%26aufirst%3DW.%2BC.%26aulast%3DChai%26aufirst%3DL.%2BC.%26aulast%3DTunung%26aufirst%3DR.%26aulast%3DJeyaletchumi%26aufirst%3DP.%26aulast%3DNoor%2BHidayah%26aufirst%3DM.%2BS.%26aulast%3DUbong%26aufirst%3DA.%26aulast%3DFarinazleen%26aufirst%3DM.%2BG.%26aulast%3DCheah%26aufirst%3DY.%2BK.%26aulast%3DSon%26aufirst%3DR.%26atitle%3DSalmonella%253A%2520A%2520foodborne%2520pathogen%26jtitle%3DInt.%2520Food%2520Res.%2520J.%26date%3D2011%26volume%3D18%26spage%3D465%26epage%3D473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Schlesinger, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahousse, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.-K.</span><span> </span><span class="NLM_article-title">Metallo-β-lactamases and aptamer-based inhibition</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=419-428&author=S.+R.+Schlesingerauthor=M.+J.+Lahousseauthor=T.+O.+Fosterauthor=S.-K.+Kim&title=Metallo-%CE%B2-lactamases+and+aptamer-based+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchlesinger%26aufirst%3DS.%2BR.%26aulast%3DLahousse%26aufirst%3DM.%2BJ.%26aulast%3DFoster%26aufirst%3DT.%2BO.%26aulast%3DKim%26aufirst%3DS.-K.%26atitle%3DMetallo-%25CE%25B2-lactamases%2520and%2520aptamer-based%2520inhibition%26jtitle%3DPharmaceuticals%26date%3D2011%26volume%3D4%26spage%3D419%26epage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Vivekananda, J.; Keil, J. L.</span>US20036569630,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vivekananda%2C+J.%3B+Keil%2C+J.+L.US20036569630%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVivekananda%26aufirst%3DJ.%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Xiaolian, Z.; Fan, C.</span>CN101109013,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Xiaolian%2C+Z.%3B+Fan%2C+C.CN101109013%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DXiaolian%26aufirst%3DZ.%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Morrissey, D.; Haeberli, P.</span>WO03106476,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Morrissey%2C+D.%3B+Haeberli%2C+P.WO03106476%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMorrissey%26aufirst%3DD.%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ellington, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szostak, J.</span><span> </span><span class="NLM_article-title">In vitro selection of RNA molecules that bind specific ligands</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1038%2F346818a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=1697402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=1990&pages=818-822&author=A.+Ellingtonauthor=J.+Szostak&title=In+vitro+selection+of+RNA+molecules+that+bind+specific+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro selection of RNA molecules that bind specific ligands</span></div><div class="casAuthors">Ellington, Andrew D.; Szostak, Jack W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6287</span>),
    <span class="NLM_cas:pages">818-22</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Subpopulations of RNA mols. that bind specifically to a variety of org. dyes have been isolated from a population of random sequence RNA mols.  Roughly one in 1010 random sequence RNA mols. folds in such a way as to create a specific binding site for small ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTZFjRL1GKQ7Vg90H21EOLACvtfcHk0lhpqlghjCVTzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGgsw%253D%253D&md5=522ad1b4f0284ee88173b02035f9e9c5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F346818a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F346818a0%26sid%3Dliteratum%253Aachs%26aulast%3DEllington%26aufirst%3DA.%26aulast%3DSzostak%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520selection%2520of%2520RNA%2520molecules%2520that%2520bind%2520specific%2520ligands%26jtitle%3DNature%26date%3D1990%26volume%3D346%26spage%3D818%26epage%3D822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Tuerk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, L.</span><span> </span><span class="NLM_article-title">Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">249</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">510</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1126%2Fscience.2200121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=2200121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaK3cXlt1OltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1990&pages=505-510&author=C.+Tuerkauthor=L.+Gold&title=Systematic+evolution+of+ligands+by+exponential+enrichment%3A+RNA+ligands+to+bacteriophage+T4+DNA+polymerase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic evolution of ligands by exponential enrichment:  RNA ligands to bacteriophage T4 DNA polymerase</span></div><div class="casAuthors">Tuerk, Craig; Gold, Larry</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">4968</span>),
    <span class="NLM_cas:pages">505-10</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A novel method was developed for rapidly selecting preferred binding sequences from a population of random sequences.  The method is designated systematic evolution of ligands by exponential enrichment (SELEX).  High-affinity nucleic acid ligands for a protein were isolated by a procedure that depends on alternate cycles of ligand selection from pools of variant sequences and amplification of the bound species.  Multiple rounds exponentially enrich the population for the highest affinity species that can be clonally isolated and characterized.  In particular one eight-base region of an RNA that interacts with the T4 DNA polymerase was chosen and randomized.  Two different sequences were selected by this procedure from the calcd. pool of 65,536 species.  One is the wild-type sequence found in the bacteriophage mRNA; one is varied from wild type at four positions.  The binding consts. of these two RNA's to T4 DNA polymerase are equiv.  These protocols with minimal modification can yield high-affinity ligands for any protein that binds nucleic acids as part of its function; high affinity ligands could conceivably be developed for any target mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTzwgV60C9LVg90H21EOLACvtfcHk0lgs6V7MZ3D_rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlt1OltLY%253D&md5=a40a28db9df4a5da4c1b1652a34aa715</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.2200121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2200121%26sid%3Dliteratum%253Aachs%26aulast%3DTuerk%26aufirst%3DC.%26aulast%3DGold%26aufirst%3DL.%26atitle%3DSystematic%2520evolution%2520of%2520ligands%2520by%2520exponential%2520enrichment%253A%2520RNA%2520ligands%2520to%2520bacteriophage%2520T4%2520DNA%2520polymerase%26jtitle%3DScience%26date%3D1990%26volume%3D249%26spage%3D505%26epage%3D510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Berezovski, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lechmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musheev, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, S. N.</span><span> </span><span class="NLM_article-title">Aptamer-facilitated biomarker discovery (AptaBiD)</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9137</span><span class="NLM_x">–</span> <span class="NLM_lpage">9143</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja801951p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9137-9143&author=M.+V.+Berezovskiauthor=M.+Lechmannauthor=M.+U.+Musheevauthor=T.+W.+Makauthor=S.+N.+Krylov&title=Aptamer-facilitated+biomarker+discovery+%28AptaBiD%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja801951p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja801951p%26sid%3Dliteratum%253Aachs%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26aulast%3DLechmann%26aufirst%3DM.%26aulast%3DMusheev%26aufirst%3DM.%2BU.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DKrylov%26aufirst%3DS.%2BN.%26atitle%3DAptamer-facilitated%2520biomarker%2520discovery%2520%2528AptaBiD%2529%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9137%26epage%3D9143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Shangguan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardona, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witek, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, W.</span><span> </span><span class="NLM_article-title">Identification of liver cancer-specific aptamers using whole live cells</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac701962v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=721-728&author=D.+Shangguanauthor=L.+Mengauthor=Z.+C.+Caoauthor=Z.+Xiaoauthor=X.+Fangauthor=Y.+Liauthor=D.+Cardonaauthor=R.+P.+Witekauthor=C.+Liuauthor=W.+Tan&title=Identification+of+liver+cancer-specific+aptamers+using+whole+live+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fac701962v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac701962v%26sid%3Dliteratum%253Aachs%26aulast%3DShangguan%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DZ.%2BC.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCardona%26aufirst%3DD.%26aulast%3DWitek%26aufirst%3DR.%2BP.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520liver%2520cancer-specific%2520aptamers%2520using%2520whole%2520live%2520cells%26jtitle%3DAnal.%2520Chem.%26date%3D2008%26volume%3D80%26spage%3D721%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Javaherian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musheev, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanoatov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, S. N.</span><span> </span><span class="NLM_article-title">Selection of aptamers for a protein target in cell lysate and their application to protein purification</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e62</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=e62&issue=8&author=S.+Javaherianauthor=M.+U.+Musheevauthor=M.+Kanoatovauthor=M.+V.+Berezovskiauthor=S.+N.+Krylov&title=Selection+of+aptamers+for+a+protein+target+in+cell+lysate+and+their+application+to+protein+purification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJavaherian%26aufirst%3DS.%26aulast%3DMusheev%26aufirst%3DM.%2BU.%26aulast%3DKanoatov%26aufirst%3DM.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26aulast%3DKrylov%26aufirst%3DS.%2BN.%26atitle%3DSelection%2520of%2520aptamers%2520for%2520a%2520protein%2520target%2520in%2520cell%2520lysate%2520and%2520their%2520application%2520to%2520protein%2520purification%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3D8%26spage%3De62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">electrochemical sensing of aptamer-facilitated virus immunoshielding</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">–</span> <span class="NLM_lpage">1686</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac202978r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=1677-1686&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=electrochemical+sensing+of+aptamer-facilitated+virus+immunoshielding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fac202978r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac202978r%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3Delectrochemical%2520sensing%2520of%2520aptamer-facilitated%2520virus%2520immunoshielding%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D1677%26epage%3D1686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">Aptamer-based viability impedimetric sensor for viruses</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1816</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac203412m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=1813-1816&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=Aptamer-based+viability+impedimetric+sensor+for+viruses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fac203412m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac203412m%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3DAptamer-based%2520viability%2520impedimetric%2520sensor%2520for%2520viruses%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D1813%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">Electrochemical differentiation of epitope-specific aptamers</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">2548</span><span class="NLM_x">–</span> <span class="NLM_lpage">2556</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac300047c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=2548-2556&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=Electrochemical+differentiation+of+epitope-specific+aptamers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fac300047c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac300047c%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3DElectrochemical%2520differentiation%2520of%2520epitope-specific%2520aptamers%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D2548%26epage%3D2556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Ulrich, H.</span><span> </span><span class="NLM_article-title">RNA aptamers: from basic science towards therapy</span> <span class="citation_source-journal">Handb. Exp. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">173</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1007%2F3-540-27262-3_15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=16594622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVWit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2006&pages=305-326&author=H.+Ulrich&title=RNA+aptamers%3A+from+basic+science+towards+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RNA aptamers: from basic science towards therapy</span></div><div class="casAuthors">Ulrich, H.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">RNA towards Medicine</span>),
    <span class="NLM_cas:pages">305-326</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The SELEX technique (systematic evolution of ligands by exponential enrichment) provides a powerful tool for the in vitro selection of nucleic acid ligands (aptamers) from combinatorial oligonucleotide libraries against a target mol.  In the beginning of the technique's use, RNA mols. were identified that bind to proteins that naturally interact with nucleic acids or to small org. mols.  In the following years, the use of the SELEX technique was extended to isolate oligonucleotide ligands (aptamers) for a wide range of proteins of importance for therapy and diagnostics, such as growth factors and cell surface antigens.  These oligonucleotides bind their targets with similar affinities and specificities as antibodies do.  The in vitro selection of oligonucleotides with enzymic activity, denominated aptazymes, allows the direct transduction of mol. recognition to catalysis.  Recently, the use of in vitro selection methods to isolate protein inhibitors was extended to complex targets, such as membrane-bound receptors, and even entire cells.  RNA aptamers were also expressed in living cells.  These aptamers, also called intramers, can be used to dissect intracellular signal transduction pathways.  The utility of RNA aptamers for in vivo expts., as well as for diagnostic and therapeutic purposes, is considerably enhanced by chem. modifications, such as substitutions of the 2'-OH groups of the ribose backbone to provide resistance against enzymic degrdn. in biol. fluids.  In an alternative approach, Spiegelmers are identified through in vitro selection of an unmodified D-RNA mol. against a mirror-image (i.e. a D-peptide) of a selection target, followed by synthesis of the unnatural nuclease-resistant L-configuration of the RNA aptamer that recognizes the natural configuration of its selection target (i.e. a L-peptide).  Recently, nuclease-resistant inhibitory RNA aptamers were developed against a great variety of targets implicated in disease.  Some results have already been obtained in animal models and in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGDYZXLANvbVg90H21EOLACvtfcHk0liJInpsTVrVQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVWit7Y%253D&md5=65f1b55cf095291c2417356172eca00b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F3-540-27262-3_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F3-540-27262-3_15%26sid%3Dliteratum%253Aachs%26aulast%3DUlrich%26aufirst%3DH.%26atitle%3DRNA%2520aptamers%253A%2520from%2520basic%2520science%2520towards%2520therapy%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2006%26volume%3D173%26spage%3D305%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Vavvas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Amico, D. J.</span><span> </span><span class="NLM_article-title">Pegaptanib (Macugen): treating neovascular age-related macular degeneration and current role in clinical practice</span> <span class="citation_source-journal">Ophthalmol. Clin. North. Am.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=16935210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A280%3ADC%252BD28rht1yqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=353-360&author=D.+Vavvasauthor=D.+J.+D%E2%80%99Amico&title=Pegaptanib+%28Macugen%29%3A+treating+neovascular+age-related+macular+degeneration+and+current+role+in+clinical+practice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pegaptanib (Macugen): treating neovascular age-related macular degeneration and current role in clinical practice</span></div><div class="casAuthors">Vavvas Demetrios; D'Amico Donald J</div><div class="citationInfo"><span class="NLM_cas:title">Ophthalmology clinics of North America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-60</span>
        ISSN:<span class="NLM_cas:issn">0896-1549</span>.
    </div><div class="casAbstract">Repeated Macugen intravitreal injections, well tolerated and safe when performed according to protocol, is the first successful pharmacotherapy for wet AMD.  Macugen has already had a significant impact on neovascular AMD management, in regard to both individual patients with AMD lesions that would become amenable to treatment, and in its effects on visual function and its preservation in the aging United States population.  Although results and delivery method are not optimal, the positive outcomes indicate the beginning, and not the limit, of pharmacotherapy for AMD.  The benefits of Macugen therapy for AMD strongly outweigh the risks.  It is pointed out by some that the overall magnitude of the efficacy results is very similar to the PDT trials and question why there is so much enthusiasm about a treatment that entails intravitreal injections every 6 weeks instead of PDT every 12 weeks.  In all fairness, PDT is not equally efficacious across the lesion subtypes and sizes, whereas Macugen has effect with all lesion subtypes.  In addition, these studies widen the armamentarium and open up the possibility of combination therapy in attacking neovascularization through multiple ways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfiy0myGSrxT0O18WR3iAofW6udTcc2eboGemdLwHNQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28rht1yqtA%253D%253D&md5=3b129dd4b19ffa835c49a6af932f612c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVavvas%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Amico%26aufirst%3DD.%2BJ.%26atitle%3DPegaptanib%2520%2528Macugen%2529%253A%2520treating%2520neovascular%2520age-related%2520macular%2520degeneration%2520and%2520current%2520role%2520in%2520clinical%2520practice%26jtitle%3DOphthalmol.%2520Clin.%2520North.%2520Am.%26date%3D2006%26volume%3D19%26spage%3D353%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Keefe, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellington, A.</span><span> </span><span class="NLM_article-title">Aptamers as therapeutics</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=538-550&author=A.+D.+Keefeauthor=S.+Paiauthor=A.+Ellington&title=Aptamers+as+therapeutics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DPai%26aufirst%3DS.%26aulast%3DEllington%26aufirst%3DA.%26atitle%3DAptamers%2520as%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D9%26spage%3D538%26epage%3D550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hamula, C. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, X. C.</span><span> </span><span class="NLM_article-title">Selection of aptamers against live bacterial cells</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">7812</span><span class="NLM_x">–</span> <span class="NLM_lpage">7819</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac801272s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=7812-7819&issue=20&author=C.+L.+A.+Hamulaauthor=H.+Zhangauthor=L.+L.+Guanauthor=X.-F.+Liauthor=X.+C.+Le&title=Selection+of+aptamers+against+live+bacterial+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fac801272s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac801272s%26sid%3Dliteratum%253Aachs%26aulast%3DHamula%26aufirst%3DC.%2BL.%2BA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DL.%2BL.%26aulast%3DLi%26aufirst%3DX.-F.%26aulast%3DLe%26aufirst%3DX.%2BC.%26atitle%3DSelection%2520of%2520aptamers%2520against%2520live%2520bacterial%2520cells%26jtitle%3DAnal.%2520Chem.%26date%3D2008%26volume%3D80%26issue%3D20%26spage%3D7812%26epage%3D7819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hamula, C. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-F.</span><span> </span><span class="NLM_article-title">DNA Aptamers binding to multiple prevalent M-types of streptococcus pyogenes</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">3640</span><span class="NLM_x">–</span> <span class="NLM_lpage">3647</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac200575e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2011&pages=3640-3647&issue=10&author=C.+L.+A.+Hamulaauthor=X.+C.+Leauthor=X.-F.+Li&title=DNA+Aptamers+binding+to+multiple+prevalent+M-types+of+streptococcus+pyogenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fac200575e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac200575e%26sid%3Dliteratum%253Aachs%26aulast%3DHamula%26aufirst%3DC.%2BL.%2BA.%26aulast%3DLe%26aufirst%3DX.%2BC.%26aulast%3DLi%26aufirst%3DX.-F.%26atitle%3DDNA%2520Aptamers%2520binding%2520to%2520multiple%2520prevalent%2520M-types%2520of%2520streptococcus%2520pyogenes%26jtitle%3DAnal.%2520Chem.%26date%3D2011%26volume%3D83%26issue%3D10%26spage%3D3640%26epage%3D3647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Joshi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janagama, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dwivedi, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S. T. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaykus, L.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schefers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreevatsan, S.</span><span> </span><span class="NLM_article-title">Selection, characterization, and application of DNA aptamers for the capture and detection of Salmonella enterica serovars</span> <span class="citation_source-journal">Mol. Cell. Probes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1016%2Fj.mcp.2008.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=19049862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1artrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=20-28&author=R.+Joshiauthor=H.+Janagamaauthor=H.+P.+Dwivediauthor=S.+T.+M.+A.+Kumarauthor=L.-A.+Jaykusauthor=J.+Schefersauthor=S.+Sreevatsan&title=Selection%2C+characterization%2C+and+application+of+DNA+aptamers+for+the+capture+and+detection+of+Salmonella+enterica+serovars"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selection, characterization, and application of DNA aptamers for the capture and detection of Salmonella enterica serovars</span></div><div class="casAuthors">Joshi, Raghavendra; Janagama, Harish; Dwivedi, Hari P.; Senthil Kumar, T. M. A.; Jaykus, Lee-Ann; Schefers, Jeremy; Sreevatsan, Srinand</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Probes</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-28</span>CODEN:
                <span class="NLM_cas:coden">MCPRE6</span>;
        ISSN:<span class="NLM_cas:issn">0890-8508</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sensitive and specific pre-anal. sample processing methods are needed to enhance our ability to detect and quantify food borne pathogens from complex food and environmental samples.  In this study, DNA aptamers were selected and evaluated for the capture and detection of Salmonella enterica Typhimurium.  A total of 66 candidate sequences were enriched against S. Typhimurium outer membrane proteins (OMPs) with counter-selection against Escherichia coli OMPs and lipopolysaccharides (LPS).  Specificity of the selected aptamers was evaluated by gel-shift anal. against S. Typhimurium OMP.  Five Salmonella-specific aptamer candidates were selected for further characterization.  A diln.-to-extinction capture protocol using pure cultures of S. Typhimurium further narrowed the field to two candidates (aptamers 33 and 45) which showed low-end detection limits of 10-40 CFU.  DNase protection assays applied to these aptamers confirmed sequence-specific binding to S. Typhimurium OMP prepns., while South-Western blot anal. combined with mass spectrometry identified putative membrane proteins as targets for aptamer binding.  Aptamer 33 was bound to magnetic beads and used for the capture of S. Typhimurium seeded into whole carcass chicken rinse samples, followed by detection using quant. real-time RT-PCR.  In a pull-down assay format, detection limits were 101-102 CFU S. Typhimurium/9 mL rinsate, while in a recirculation format, detection limits were 102-103 CFU/25 mL rinsate.  Reproducible detection at <101 S. typhimurium CFU/g was also achieved in spike-and-recovery expts. using bovine feces.  The pull-down anal. using aptamer 33 was validated on 3 naturally infected chicken litter samples confirming their applicability in the field.  This study demonstrates the applicability of Salmonella specific aptamers for pre-anal. sample processing as applied to food and environmental sample matrixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_O5V8LIZefrVg90H21EOLACvtfcHk0lgekvgj92ol3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1artrs%253D&md5=c5356fbcd4ecb7965209741e550478d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.mcp.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mcp.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DR.%26aulast%3DJanagama%26aufirst%3DH.%26aulast%3DDwivedi%26aufirst%3DH.%2BP.%26aulast%3DKumar%26aufirst%3DS.%2BT.%2BM.%2BA.%26aulast%3DJaykus%26aufirst%3DL.-A.%26aulast%3DSchefers%26aufirst%3DJ.%26aulast%3DSreevatsan%26aufirst%3DS.%26atitle%3DSelection%252C%2520characterization%252C%2520and%2520application%2520of%2520DNA%2520aptamers%2520for%2520the%2520capture%2520and%2520detection%2520of%2520Salmonella%2520enterica%2520serovars%26jtitle%3DMol.%2520Cell.%2520Probes%26date%3D2009%26volume%3D23%26spage%3D20%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Fubin, Wang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming-Sheng, Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian-Ming, Hu</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bing Xia Qin, Pan</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao-Lian, Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong-Yan, Wu</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan-Wen, He</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jianguo, Wu.</span><span> </span><span class="NLM_article-title">Aptamers that preferentially bind type IVB pili and inhibit human monocytic-cell invasion by Salmonella enterica serovar typhi</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4052</span><span class="NLM_x">–</span> <span class="NLM_lpage">4060</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=4052-4060&issue=10&author=Wang+Fubinauthor=Zhang+Ming-Shengauthor=Hu+Jian-Mingauthor=Pan+Bing+Xia+Qinauthor=Zhang+Xiao-Lianauthor=Wu+Hong-Yanauthor=He+Pan-Wenauthor=Wu.+Jianguo&title=Aptamers+that+preferentially+bind+type+IVB+pili+and+inhibit+human+monocytic-cell+invasion+by+Salmonella+enterica+serovar+typhi"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFubin%26aufirst%3DWang%26aulast%3DMing-Sheng%26aufirst%3DZhang%26aulast%3DJian-Ming%26aufirst%3DHu%26aulast%3DBing%2BXia%2BQin%26aufirst%3DPan%26aulast%3DXiao-Lian%26aufirst%3DZhang%26aulast%3DHong-Yan%26aufirst%3DWu%26aulast%3DPan-Wen%26aufirst%3DHe%26aulast%3DJianguo%26aufirst%3DWu.%26atitle%3DAptamers%2520that%2520preferentially%2520bind%2520type%2520IVB%2520pili%2520and%2520inhibit%2520human%2520monocytic-cell%2520invasion%2520by%2520Salmonella%2520enterica%2520serovar%2520typhi%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26issue%3D10%26spage%3D4052%26epage%3D4060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Sefah, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shangguan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Donoghue, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, W.</span><span> </span><span class="NLM_article-title">Development of DNA aptamers using Cell-SELEX</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1169</span><span class="NLM_x">–</span> <span class="NLM_lpage">1185</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1038%2Fnprot.2010.66" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=20539292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFSkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1169-1185&author=K.+Sefahauthor=D.+Shangguanauthor=X.+Xiongauthor=M.+B.+O%E2%80%99Donoghueauthor=W.+Tan&title=Development+of+DNA+aptamers+using+Cell-SELEX"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Development of DNA aptamers using Cell-SELEX</span></div><div class="casAuthors">Sefah, Kwame; Shangguan, Dihua; Xiong, Xiangling; O'Donoghue, Meghan B.; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1185</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past two decades, high-affinity nucleic acid aptamers have been developed for a wide variety of pure mols. and complex systems such as live cells.  Conceptually, aptamers are developed by an evolutionary process, whereby, as selection progresses, sequences with a certain conformation capable of binding to the target of interest emerge and dominate the pool.  This protocol, cell-SELEX (systematic evolution of ligands by exponential enrichment), is a method that can generate DNA aptamers that can bind specifically to a cell type of interest.  Commonly, a cancer cell line is used as the target to generate aptamers that can differentiate that cell type from other cancers or normal cells.  A single-stranded DNA (ssDNA) library pool is incubated with the target cells.  Nonbinding sequences are washed off and bound sequences are recovered from the cells by heating cell-DNA complexes at 95 °C, followed by centrifugation.  The recovered pool is incubated with the control cell line to filter out the sequences that bind to common mols. on both the target and the control, leading to the enrichment of specific binders to the target.  Binding sequences are amplified by PCR using fluorescein isothiocyanate-labeled sense and biotin-labeled antisense primers.  This is followed by removal of antisense strands to generate an ssDNA pool for subsequent rounds of selection.  The enrichment of the selected pools is monitored by flow cytometry binding assays, with selected pools having increased fluorescence compared with the unselected DNA library.  The procedure, from design of oligonucleotides to enrichment of the selected pools, takes ∼ 3 mo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtFzU80-TsLbVg90H21EOLACvtfcHk0lhaSigit2o2ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFSkt74%253D&md5=396e01ca2ce0530ace68b12bbf8975a4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2010.66&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2010.66%26sid%3Dliteratum%253Aachs%26aulast%3DSefah%26aufirst%3DK.%26aulast%3DShangguan%26aufirst%3DD.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DM.%2BB.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520DNA%2520aptamers%2520using%2520Cell-SELEX%26jtitle%3DNat.%2520Protoc.%26date%3D2010%26volume%3D5%26spage%3D1169%26epage%3D1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Miles, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misra, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. O.</span><span> </span><span class="NLM_article-title">The estimation of the bactericidal power of the blood</span> <span class="citation_source-journal">J. Hyg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1938</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">732</span><span class="NLM_x">–</span> <span class="NLM_lpage">749</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1017%2FS002217240001158X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=20475467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaA1MXhtFGnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1938&pages=732-749&issue=6&author=A.+A.+Milesauthor=S.+S.+Misraauthor=J.+O.+Irwin&title=The+estimation+of+the+bactericidal+power+of+the+blood"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The estimation of the bactericidal power of the blood</span></div><div class="casAuthors">Miles, A. A.; Misra, S. S.; Irwin, J. O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hygiene</span>
        (<span class="NLM_cas:date">1938</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">732-49</span>CODEN:
                <span class="NLM_cas:coden">JOHYAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1724</span>.
    </div><div class="casAbstract">The survival rate (p) of a measured inoculum of Staphylococcus aureus in a standard vol. of defibrinated blood is a reliable quant. measure of the bactericidal power of blood.  The immunological significance of p has not been investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtsaEeWDTKkrVg90H21EOLACvtfcHk0lhaSigit2o2ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA1MXhtFGnug%253D%253D&md5=1c91fe942e3e37d0be154c340ae862a5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1017%2FS002217240001158X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS002217240001158X%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DA.%2BA.%26aulast%3DMisra%26aufirst%3DS.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BO.%26atitle%3DThe%2520estimation%2520of%2520the%2520bactericidal%2520power%2520of%2520the%2520blood%26jtitle%3DJ.%2520Hyg.%26date%3D1938%26volume%3D38%26issue%3D6%26spage%3D732%26epage%3D749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Lopez-Amoros, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vives-Rego, J.</span><span> </span><span class="NLM_article-title">Flow cytometric assessment of Escherichia coli and Salmonella typhimurium starvation-survival in seawater using rhodamine 123, propidium iodide, and oxonol</span> <span class="citation_source-journal">Appl. Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2521</span><span class="NLM_x">–</span> <span class="NLM_lpage">2526</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1995&pages=2521-2526&issue=7&author=R.+Lopez-Amorosauthor=J.+Comasauthor=J.+Vives-Rego&title=Flow+cytometric+assessment+of+Escherichia+coli+and+Salmonella+typhimurium+starvation-survival+in+seawater+using+rhodamine+123%2C+propidium+iodide%2C+and+oxonol"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Amoros%26aufirst%3DR.%26aulast%3DComas%26aufirst%3DJ.%26aulast%3DVives-Rego%26aufirst%3DJ.%26atitle%3DFlow%2520cytometric%2520assessment%2520of%2520Escherichia%2520coli%2520and%2520Salmonella%2520typhimurium%2520starvation-survival%2520in%2520seawater%2520using%2520rhodamine%2520123%252C%2520propidium%2520iodide%252C%2520and%2520oxonol%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D1995%26volume%3D61%26issue%3D7%26spage%3D2521%26epage%3D2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kennedy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, U. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, M. G.</span><span> </span><span class="NLM_article-title">Responses of Escherichia coli, Listeria monocytogenes, and Staphylococcus aureus to simulated food processing treatments, determined using Fluorescence-Activated Cell Sorting and plate counting</span> <span class="citation_source-journal">Appl. Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">4657</span><span class="NLM_x">–</span> <span class="NLM_lpage">4668</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=4657-4668&issue=13&author=D.+Kennedyauthor=U.+P.+Croninauthor=M.+G.+Wilkinson&title=Responses+of+Escherichia+coli%2C+Listeria+monocytogenes%2C+and+Staphylococcus+aureus+to+simulated+food+processing+treatments%2C+determined+using+Fluorescence-Activated+Cell+Sorting+and+plate+counting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DD.%26aulast%3DCronin%26aufirst%3DU.%2BP.%26aulast%3DWilkinson%26aufirst%3DM.%2BG.%26atitle%3DResponses%2520of%2520Escherichia%2520coli%252C%2520Listeria%2520monocytogenes%252C%2520and%2520Staphylococcus%2520aureus%2520to%2520simulated%2520food%2520processing%2520treatments%252C%2520determined%2520using%2520Fluorescence-Activated%2520Cell%2520Sorting%2520and%2520plate%2520counting%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26issue%3D13%26spage%3D4657%26epage%3D4668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 54 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piyanan  Chuesiang</span>, <span class="hlFld-ContribAuthor ">Victor  Ryu</span>, <span class="hlFld-ContribAuthor ">Ubonrat  Siripatrawan</span>, <span class="hlFld-ContribAuthor ">Lynne  McLandsborough</span>, <span class="hlFld-ContribAuthor ">Lili  He</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of factors that impact the label-free surface-enhanced Raman scattering (SERS) for the detection and discrimination of Salmonella Enteritidis. </span><span class="cited-content_cbyCitation_journal-name">LWT</span><span> <strong>2021,</strong> <em>150 </em>, 111962. <a href="https://doi.org/10.1016/j.lwt.2021.111962" title="DOI URL">https://doi.org/10.1016/j.lwt.2021.111962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lwt.2021.111962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lwt.2021.111962%26sid%3Dliteratum%253Aachs%26jtitle%3DLWT%26atitle%3DInvestigation%252Bof%252Bfactors%252Bthat%252Bimpact%252Bthe%252Blabel-free%252Bsurface-enhanced%252BRaman%252Bscattering%252B%252528SERS%252529%252Bfor%252Bthe%252Bdetection%252Band%252Bdiscrimination%252Bof%252BSalmonella%252BEnteritidis%26aulast%3DChuesiang%26aufirst%3DPiyanan%26date%3D2021%26volume%3D150%26spage%3D111962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chan-Jin  Kim</span>, <span class="hlFld-ContribAuthor ">Zhangyong  Si</span>, <span class="hlFld-ContribAuthor ">Sheethal  Reghu</span>, <span class="hlFld-ContribAuthor ">Zhong  Guo</span>, <span class="hlFld-ContribAuthor ">Kaixi  Zhang</span>, <span class="hlFld-ContribAuthor ">Jianghua  Li</span>, <span class="hlFld-ContribAuthor ">Mary B.  Chan-Park</span>. </span><span class="cited-content_cbyCitation_article-title">DNA-derived nanostructures selectively capture gram-positive bacteria. </span><span class="cited-content_cbyCitation_journal-name">Drug Delivery and Translational Research</span><span> <strong>2021,</strong> <em>11 </em>
                                    (4)
                                     , 1438-1450. <a href="https://doi.org/10.1007/s13346-021-00975-w" title="DOI URL">https://doi.org/10.1007/s13346-021-00975-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13346-021-00975-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13346-021-00975-w%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Delivery%2520and%2520Translational%2520Research%26atitle%3DDNA-derived%252Bnanostructures%252Bselectively%252Bcapture%252Bgram-positive%252Bbacteria%26aulast%3DKim%26aufirst%3DChan-Jin%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D4%26spage%3D1438%26epage%3D1450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fatemeh  Hemmati</span>, <span class="hlFld-ContribAuthor ">Mohammad Ahangarzadeh  Rezaee</span>, <span class="hlFld-ContribAuthor ">Saba  Ebrahimzadeh</span>, <span class="hlFld-ContribAuthor ">Leila  Yousefi</span>, <span class="hlFld-ContribAuthor ">Roghayeh  Nouri</span>, <span class="hlFld-ContribAuthor ">Hossein Samadi  Kafil</span>, <span class="hlFld-ContribAuthor ">Pourya  Gholizadeh</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Strategies to Combat Bacterial Biofilms. </span><span class="cited-content_cbyCitation_journal-name">Molecular Biotechnology</span><span> <strong>2021,</strong> <em>63 </em>
                                    (7)
                                     , 569-586. <a href="https://doi.org/10.1007/s12033-021-00325-8" title="DOI URL">https://doi.org/10.1007/s12033-021-00325-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12033-021-00325-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12033-021-00325-8%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Biotechnology%26atitle%3DNovel%252BStrategies%252Bto%252BCombat%252BBacterial%252BBiofilms%26aulast%3DHemmati%26aufirst%3DFatemeh%26date%3D2021%26date%3D2021%26volume%3D63%26issue%3D7%26spage%3D569%26epage%3D586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piyanan  Chuesiang</span>, <span class="hlFld-ContribAuthor ">Victor  Ryu</span>, <span class="hlFld-ContribAuthor ">Ubonrat  Siripatrawan</span>, <span class="hlFld-ContribAuthor ">Lili  He</span>, <span class="hlFld-ContribAuthor ">Lynne  McLandsborough</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamer-based surface enhanced Raman spectroscopy (SERS) for the rapid detection of Salmonella Enteritidis contaminated in ground beef. </span><span class="cited-content_cbyCitation_journal-name">LWT</span><span> <strong>2021,</strong> <em>93 </em>, 111937. <a href="https://doi.org/10.1016/j.lwt.2021.111937" title="DOI URL">https://doi.org/10.1016/j.lwt.2021.111937</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lwt.2021.111937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lwt.2021.111937%26sid%3Dliteratum%253Aachs%26jtitle%3DLWT%26atitle%3DAptamer-based%252Bsurface%252Benhanced%252BRaman%252Bspectroscopy%252B%252528SERS%252529%252Bfor%252Bthe%252Brapid%252Bdetection%252Bof%252BSalmonella%252BEnteritidis%252Bcontaminated%252Bin%252Bground%252Bbeef%26aulast%3DChuesiang%26aufirst%3DPiyanan%26date%3D2021%26volume%3D93%26spage%3D111937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fang  Yu</span>, <span class="hlFld-ContribAuthor ">Jing  Chen</span>, <span class="hlFld-ContribAuthor ">Zecheng  Wang</span>, <span class="hlFld-ContribAuthor ">Huixin  Yang</span>, <span class="hlFld-ContribAuthor ">Hui  Li</span>, <span class="hlFld-ContribAuthor ">Wenchao  Jia</span>, <span class="hlFld-ContribAuthor ">Shuyuan  Xue</span>, <span class="hlFld-ContribAuthor ">Hexin  Xie</span>, <span class="hlFld-ContribAuthor ">Danke  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Screening aptamers for serine β-lactamase-expressing bacteria with Precision-SELEX. </span><span class="cited-content_cbyCitation_journal-name">Talanta</span><span> <strong>2021,</strong> <em>224 </em>, 121750. <a href="https://doi.org/10.1016/j.talanta.2020.121750" title="DOI URL">https://doi.org/10.1016/j.talanta.2020.121750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.talanta.2020.121750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.talanta.2020.121750%26sid%3Dliteratum%253Aachs%26jtitle%3DTalanta%26atitle%3DScreening%252Baptamers%252Bfor%252Bserine%252B%2525CE%2525B2-lactamase-expressing%252Bbacteria%252Bwith%252BPrecision-SELEX%26aulast%3DYu%26aufirst%3DFang%26date%3D2021%26volume%3D224%26spage%3D121750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Monali  Mukherjee</span>, <span class="hlFld-ContribAuthor ">Rajendra Kumar Reddy  Gajjala</span>, <span class="hlFld-ContribAuthor ">Pravin Savata  Gade</span>, <span class="hlFld-ContribAuthor ">Praveena  Bhatt</span>. </span><span class="cited-content_cbyCitation_article-title">Aptasensors: Paradigm Shift for Detection of Food Toxins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 712-730. <a href="https://doi.org/10.1016/B978-0-08-100596-5.22997-1" title="DOI URL">https://doi.org/10.1016/B978-0-08-100596-5.22997-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100596-5.22997-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100596-5.22997-1%26sid%3Dliteratum%253Aachs%26atitle%3DAptasensors%25253A%252BParadigm%252BShift%252Bfor%252BDetection%252Bof%252BFood%252BToxins%26aulast%3DMukherjee%26aufirst%3DMonali%26date%3D2021%26spage%3D712%26epage%3D730%26pub%3DElsevier%26atitle%3DInnovative%252BFood%252BProcessing%252BTechnologies%26date%3D2021%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinjin  Shen</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuanyue  Shan</span>, <span class="hlFld-ContribAuthor ">Huahua  Yue</span>, <span class="hlFld-ContribAuthor ">Ru  Huang</span>, <span class="hlFld-ContribAuthor ">Jiaming  Hu</span>, <span class="hlFld-ContribAuthor ">Da  Xing</span>. </span><span class="cited-content_cbyCitation_article-title">Sensitive detection of a bacterial pathogen using allosteric probe-initiated catalysis and CRISPR-Cas13a amplification reaction. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-019-14135-9" title="DOI URL">https://doi.org/10.1038/s41467-019-14135-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-019-14135-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-019-14135-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DSensitive%252Bdetection%252Bof%252Ba%252Bbacterial%252Bpathogen%252Busing%252Ballosteric%252Bprobe-initiated%252Bcatalysis%252Band%252BCRISPR-Cas13a%252Bamplification%252Breaction%26aulast%3DShen%26aufirst%3DJinjin%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shima  Afrasiabi</span>, <span class="hlFld-ContribAuthor ">Maryam  Pourhajibagher</span>, <span class="hlFld-ContribAuthor ">Reza  Raoofian</span>, <span class="hlFld-ContribAuthor ">Maryam  Tabarzad</span>, <span class="hlFld-ContribAuthor ">Abbas  Bahador</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic applications of nucleic acid aptamers in microbial infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2020,</strong> <em>27 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-019-0611-0" title="DOI URL">https://doi.org/10.1186/s12929-019-0611-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-019-0611-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-019-0611-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DTherapeutic%252Bapplications%252Bof%252Bnucleic%252Bacid%252Baptamers%252Bin%252Bmicrobial%252Binfections%26aulast%3DAfrasiabi%26aufirst%3DShima%26date%3D2020%26date%3D2020%26volume%3D27%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dennis  Kubiczek</span>, <span class="hlFld-ContribAuthor ">Heinz  Raber</span>, <span class="hlFld-ContribAuthor ">Nicholas  Bodenberger</span>, <span class="hlFld-ContribAuthor ">Thomas  Oswald</span>, <span class="hlFld-ContribAuthor ">Melis  Sahan</span>, <span class="hlFld-ContribAuthor ">Daniel  Mayer</span>, <span class="hlFld-ContribAuthor ">Sebastian  Wiese</span>, <span class="hlFld-ContribAuthor ">Steffen  Stenger</span>, <span class="hlFld-ContribAuthor ">Tanja  Weil</span>, <span class="hlFld-ContribAuthor ">Frank  Rosenau</span>. </span><span class="cited-content_cbyCitation_article-title">The Diversity of a Polyclonal FluCell‐SELEX Library Outperforms Individual Aptamers as Emerging Diagnostic Tools for the Identification of Carbapenem Resistant
              Pseudomonas aeruginosa. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (64)
                                     , 14536-14545. <a href="https://doi.org/10.1002/chem.202000213" title="DOI URL">https://doi.org/10.1002/chem.202000213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202000213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202000213%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DThe%252BDiversity%252Bof%252Ba%252BPolyclonal%252BFluCell%2525E2%252580%252590SELEX%252BLibrary%252BOutperforms%252BIndividual%252BAptamers%252Bas%252BEmerging%252BDiagnostic%252BTools%252Bfor%252Bthe%252BIdentification%252Bof%252BCarbapenem%252BResistant%252BPseudomonas%252Baeruginosa%26aulast%3DKubiczek%26aufirst%3DDennis%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D64%26spage%3D14536%26epage%3D14545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stery Brenda  Oroh</span>, <span class="hlFld-ContribAuthor ">Apon Zaenal  Mustopa</span>, <span class="hlFld-ContribAuthor ">Sri  Budiarti</span>, <span class="hlFld-ContribAuthor ">Bugi Ratno  Budiarto</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of enteropathogenic Escherichia coli biofilm formation by DNA aptamer. </span><span class="cited-content_cbyCitation_journal-name">Molecular Biology Reports</span><span> <strong>2020,</strong> <em>47 </em>
                                    (10)
                                     , 7567-7573. <a href="https://doi.org/10.1007/s11033-020-05822-8" title="DOI URL">https://doi.org/10.1007/s11033-020-05822-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11033-020-05822-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11033-020-05822-8%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Biology%2520Reports%26atitle%3DInhibition%252Bof%252Benteropathogenic%252BEscherichia%252Bcoli%252Bbiofilm%252Bformation%252Bby%252BDNA%252Baptamer%26aulast%3DOroh%26aufirst%3DStery%2BBrenda%26date%3D2020%26date%3D2020%26volume%3D47%26issue%3D10%26spage%3D7567%26epage%3D7573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Mengxiao  Song</span>, <span class="hlFld-ContribAuthor ">Linqin  Dou</span>, <span class="hlFld-ContribAuthor ">Yuling  Xiao</span>, <span class="hlFld-ContribAuthor ">Kaiju  Li</span>, <span class="hlFld-ContribAuthor ">Guangzhen  Shen</span>, <span class="hlFld-ContribAuthor ">Binwu  Ying</span>, <span class="hlFld-ContribAuthor ">Jia  Geng</span>, <span class="hlFld-ContribAuthor ">Depo  Yang</span>, <span class="hlFld-ContribAuthor ">Zhengzhi  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a fluorescent DNA nanomachine for ultrasensitive detection of Salmonella enteritidis without labeling and enzymes. </span><span class="cited-content_cbyCitation_journal-name">Microchimica Acta</span><span> <strong>2020,</strong> <em>187 </em>
                                    (7)
                                     <a href="https://doi.org/10.1007/s00604-020-04334-3" title="DOI URL">https://doi.org/10.1007/s00604-020-04334-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00604-020-04334-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00604-020-04334-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrochimica%2520Acta%26atitle%3DDevelopment%252Bof%252Ba%2525C2%2525A0fluorescent%252BDNA%252Bnanomachine%252Bfor%252Bultrasensitive%252Bdetection%252Bof%252BSalmonella%252Benteritidis%252Bwithout%252Blabeling%252Band%252Benzymes%26aulast%3DZhang%26aufirst%3DPeng%26date%3D2020%26date%3D2020%26volume%3D187%26issue%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fatima  Shatila</span>, <span class="hlFld-ContribAuthor ">H. Tansel  Yalçın</span>, <span class="hlFld-ContribAuthor ">Canan  Özyurt</span>, <span class="hlFld-ContribAuthor ">Serap  Evran</span>, <span class="hlFld-ContribAuthor ">Büşra  Çakır</span>, <span class="hlFld-ContribAuthor ">İhsan  Yaşa</span>, <span class="hlFld-ContribAuthor ">Ayşe  Nalbantsoy</span>. </span><span class="cited-content_cbyCitation_article-title">Single-stranded DNA (ssDNA) Aptamer targeting SipA protein inhibits Salmonella Enteritidis invasion of intestinal epithelial cells. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2020,</strong> <em>148 </em>, 518-524. <a href="https://doi.org/10.1016/j.ijbiomac.2020.01.132" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2020.01.132</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2020.01.132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2020.01.132%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DSingle-stranded%252BDNA%252B%252528ssDNA%252529%252BAptamer%252Btargeting%252BSipA%252Bprotein%252Binhibits%252BSalmonella%252BEnteritidis%252Binvasion%252Bof%252Bintestinal%252Bepithelial%252Bcells%26aulast%3DShatila%26aufirst%3DFatima%26date%3D2020%26volume%3D148%26spage%3D518%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer  Soundy</span>, <span class="hlFld-ContribAuthor ">Darren  Day</span>. </span><span class="cited-content_cbyCitation_article-title">Delivery of antibacterial silver nanoclusters to Pseudomonas aeruginosa using species-specific DNA aptamers. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medical Microbiology</span><span> <strong>2020,</strong> <em>69 </em>
                                    (4)
                                     , 640-652. <a href="https://doi.org/10.1099/jmm.0.001174" title="DOI URL">https://doi.org/10.1099/jmm.0.001174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1099/jmm.0.001174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1099%2Fjmm.0.001174%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medical%2520Microbiology%26atitle%3DDelivery%252Bof%252Bantibacterial%252Bsilver%252Bnanoclusters%252Bto%252BPseudomonas%252Baeruginosa%252Busing%252Bspecies-specific%252BDNA%252Baptamers%26aulast%3DSoundy%26aufirst%3DJennifer%26date%3D2020%26volume%3D69%26issue%3D4%26spage%3D640%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Galina S.  Zamay</span>, <span class="hlFld-ContribAuthor ">Tatiana N.  Zamay</span>, <span class="hlFld-ContribAuthor ">Kirill A.  Lukyanenko</span>, <span class="hlFld-ContribAuthor ">Anna S.  Kichkailo</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamers Increase Biocompatibility and Reduce the Toxicity of Magnetic Nanoparticles Used in Biomedicine. </span><span class="cited-content_cbyCitation_journal-name">Biomedicines</span><span> <strong>2020,</strong> <em>8 </em>
                                    (3)
                                     , 59. <a href="https://doi.org/10.3390/biomedicines8030059" title="DOI URL">https://doi.org/10.3390/biomedicines8030059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biomedicines8030059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiomedicines8030059%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicines%26atitle%3DAptamers%252BIncrease%252BBiocompatibility%252Band%252BReduce%252Bthe%252BToxicity%252Bof%252BMagnetic%252BNanoparticles%252BUsed%252Bin%252BBiomedicine%26aulast%3DZamay%26aufirst%3DGalina%2BS.%26date%3D2020%26date%3D2020%26volume%3D8%26issue%3D3%26spage%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dawei  Li</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Bei  Lv</span>. </span><span class="cited-content_cbyCitation_article-title">Antibacterial Therapeutic Agents Composed of Functional Biological Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemistry</span><span> <strong>2020,</strong> <em>2020 </em>, 1-13. <a href="https://doi.org/10.1155/2020/6578579" title="DOI URL">https://doi.org/10.1155/2020/6578579</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2020/6578579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2020%2F6578579%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemistry%26atitle%3DAntibacterial%252BTherapeutic%252BAgents%252BComposed%252Bof%252BFunctional%252BBiological%252BMolecules%26aulast%3DLi%26aufirst%3DDawei%26date%3D2020%26volume%3D2020%26spage%3D1%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bhoomika  Yadav</span>, <span class="hlFld-ContribAuthor ">Balasubramanian  Sellamuthu</span>, <span class="hlFld-ContribAuthor ">R.D.  Tyagi</span>. </span><span class="cited-content_cbyCitation_article-title">Riboswitches and aptamers: potential future targets to control drug-resistant bacteria. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 421-462. <a href="https://doi.org/10.1016/B978-0-12-819722-6.00012-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-819722-6.00012-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819722-6.00012-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819722-6.00012-2%26sid%3Dliteratum%253Aachs%26atitle%3DRiboswitches%252Band%252Baptamers%25253A%252Bpotential%252Bfuture%252Btargets%252Bto%252Bcontrol%252Bdrug-resistant%252Bbacteria%26aulast%3DYadav%26aufirst%3DBhoomika%26date%3D2020%26spage%3D421%26epage%3D462%26pub%3DElsevier%26atitle%3DCurrent%252BDevelopments%252Bin%252BBiotechnology%252Band%252BBioengineering%26date%3D2020%26volume%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Víctor  Díaz-García</span>, <span class="hlFld-ContribAuthor ">Gerardo  Retamal-Morales</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamer and Riboswitches: A Novel Tool for the Need of New Antimicrobial Active Compounds. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 231-247. <a href="https://doi.org/10.1016/B978-0-12-820054-4.00013-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-820054-4.00013-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820054-4.00013-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820054-4.00013-6%26sid%3Dliteratum%253Aachs%26atitle%3DAptamer%252Band%252BRiboswitches%25253A%252BA%252BNovel%252BTool%252Bfor%252Bthe%252BNeed%252Bof%252BNew%252BAntimicrobial%252BActive%252BCompounds%26aulast%3DD%25C3%25ADaz-Garc%25C3%25ADa%26aufirst%3DV%25C3%25ADctor%26date%3D2020%26spage%3D231%26epage%3D247%26pub%3DElsevier%26atitle%3DAntibiotic%252BMaterials%252Bin%252BHealthcare%26date%3D2020%26volume%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olga S.  Kolovskaya</span>, <span class="hlFld-ContribAuthor ">Tatiana N.  Zamay</span>, <span class="hlFld-ContribAuthor ">Galina S.  Zamay</span>, <span class="hlFld-ContribAuthor ">Vasily A.  Babkin</span>, <span class="hlFld-ContribAuthor ">Elena N.  Medvedeva</span>, <span class="hlFld-ContribAuthor ">Nadezhda A.  Neverova</span>, <span class="hlFld-ContribAuthor ">Andrey K.  Kirichenko</span>, <span class="hlFld-ContribAuthor ">Sergey S.  Zamay</span>, <span class="hlFld-ContribAuthor ">Ivan N.  Lapin</span>, <span class="hlFld-ContribAuthor ">Evgeny V.  Morozov</span>, <span class="hlFld-ContribAuthor ">Alexey E.  Sokolov</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Narodov</span>, <span class="hlFld-ContribAuthor ">Dmitri G.  Fedorov</span>, <span class="hlFld-ContribAuthor ">Felix N.  Tomilin</span>, <span class="hlFld-ContribAuthor ">Vladimir N.  Zabluda</span>, <span class="hlFld-ContribAuthor ">Yulia  Alekhina</span>, <span class="hlFld-ContribAuthor ">Kirill A.  Lukyanenko</span>, <span class="hlFld-ContribAuthor ">Yury E.  Glazyrin</span>, <span class="hlFld-ContribAuthor ">Valery A.  Svetlichnyi</span>, <span class="hlFld-ContribAuthor ">Maxim V.  Berezovski</span>, <span class="hlFld-ContribAuthor ">Anna S.  Kichkailo</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamer-Conjugated Superparamagnetic Ferroarabinogalactan Nanoparticles for Targeted Magnetodynamic Therapy of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     , 216. <a href="https://doi.org/10.3390/cancers12010216" title="DOI URL">https://doi.org/10.3390/cancers12010216</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12010216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12010216%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DAptamer-Conjugated%252BSuperparamagnetic%252BFerroarabinogalactan%252BNanoparticles%252Bfor%252BTargeted%252BMagnetodynamic%252BTherapy%252Bof%252BCancer%26aulast%3DKolovskaya%26aufirst%3DOlga%2BS.%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1%26spage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gajapati Y N  Varma</span>, <span class="hlFld-ContribAuthor ">Githavani  Kummari</span>, <span class="hlFld-ContribAuthor ">Pradip  Paik</span>, <span class="hlFld-ContribAuthor ">Arunasree M  Kalle</span>. </span><span class="cited-content_cbyCitation_article-title">Celecoxib potentiates antibiotic uptake by altering membrane potential and permeability in Staphylococcus aureus. </span><span class="cited-content_cbyCitation_journal-name">Journal of Antimicrobial Chemotherapy</span><span> <strong>2019,</strong> <em>74 </em>
                                    (12)
                                     , 3462-3472. <a href="https://doi.org/10.1093/jac/dkz391" title="DOI URL">https://doi.org/10.1093/jac/dkz391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/jac/dkz391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fjac%2Fdkz391%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Antimicrobial%2520Chemotherapy%26atitle%3DCelecoxib%252Bpotentiates%252Bantibiotic%252Buptake%252Bby%252Baltering%252Bmembrane%252Bpotential%252Band%252Bpermeability%252Bin%252BStaphylococcus%252Baureus%26aulast%3DVarma%26aufirst%3DGajapati%2BY%2BN%26date%3D2019%26date%3D2019%26volume%3D74%26issue%3D12%26spage%3D3462%26epage%3D3472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyue  Gao</span>, <span class="hlFld-ContribAuthor ">Lili  He</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a filtration-based SERS mapping platform for specific screening of Salmonella enterica serovar Enteritidis. </span><span class="cited-content_cbyCitation_journal-name">Analytical and Bioanalytical Chemistry</span><span> <strong>2019,</strong> <em>411 </em>
                                    (29)
                                     , 7899-7906. <a href="https://doi.org/10.1007/s00216-019-02204-3" title="DOI URL">https://doi.org/10.1007/s00216-019-02204-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00216-019-02204-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00216-019-02204-3%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520and%2520Bioanalytical%2520Chemistry%26atitle%3DDevelopment%252Bof%252Ba%252Bfiltration-based%252BSERS%252Bmapping%252Bplatform%252Bfor%252Bspecific%252Bscreening%252Bof%252BSalmonella%252Benterica%252Bserovar%252BEnteritidis%26aulast%3DGao%26aufirst%3DSiyue%26date%3D2019%26date%3D2019%26volume%3D411%26issue%3D29%26spage%3D7899%26epage%3D7906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng-Ying  Xia</span>, <span class="hlFld-ContribAuthor ">Yu  Xie</span>, <span class="hlFld-ContribAuthor ">Chen-Hao  Yu</span>, <span class="hlFld-ContribAuthor ">Ge-Yun  Chen</span>, <span class="hlFld-ContribAuthor ">Yuan-Hong  Li</span>, <span class="hlFld-ContribAuthor ">Ting  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Graphene-based nanomaterials: the promising active agents for antibiotics-independent antibacterial applications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Controlled Release</span><span> <strong>2019,</strong> <em>307 </em>, 16-31. <a href="https://doi.org/10.1016/j.jconrel.2019.06.011" title="DOI URL">https://doi.org/10.1016/j.jconrel.2019.06.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jconrel.2019.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jconrel.2019.06.011%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Controlled%2520Release%26atitle%3DGraphene-based%252Bnanomaterials%25253A%252Bthe%252Bpromising%252Bactive%252Bagents%252Bfor%252Bantibiotics-independent%252Bantibacterial%252Bapplications%26aulast%3DXia%26aufirst%3DMeng-Ying%26date%3D2019%26volume%3D307%26spage%3D16%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Biyao  Mao</span>, <span class="hlFld-ContribAuthor ">Lijuan  Cheng</span>, <span class="hlFld-ContribAuthor ">Sheng  Wang</span>, <span class="hlFld-ContribAuthor ">Jiaqi  Zhou</span>, <span class="hlFld-ContribAuthor ">Le  Deng</span>. </span><span class="cited-content_cbyCitation_article-title">Combat biofilm by bacteriostatic aptamer‐functionalized graphene oxide. </span><span class="cited-content_cbyCitation_journal-name">Biotechnology and Applied Biochemistry</span><span> <strong>2018,</strong> <em>65 </em>
                                    (3)
                                     , 355-361. <a href="https://doi.org/10.1002/bab.1631" title="DOI URL">https://doi.org/10.1002/bab.1631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bab.1631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbab.1631%26sid%3Dliteratum%253Aachs%26jtitle%3DBiotechnology%2520and%2520Applied%2520Biochemistry%26atitle%3DCombat%252Bbiofilm%252Bby%252Bbacteriostatic%252Baptamer%2525E2%252580%252590functionalized%252Bgraphene%252Boxide%26aulast%3DMao%26aufirst%3DBiyao%26date%3D2018%26date%3D2018%26volume%3D65%26issue%3D3%26spage%3D355%26epage%3D361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raja  Chinnappan</span>, <span class="hlFld-ContribAuthor ">Saleh  AlAmer</span>, <span class="hlFld-ContribAuthor ">Shimaa  Eissa</span>, <span class="hlFld-ContribAuthor ">Anas Abdel  Rahamn</span>, <span class="hlFld-ContribAuthor ">Khalid M.  Abu Salah</span>, <span class="hlFld-ContribAuthor ">Mohammed  Zourob</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorometric graphene oxide-based detection of Salmonella enteritis using a truncated DNA aptamer. </span><span class="cited-content_cbyCitation_journal-name">Microchimica Acta</span><span> <strong>2018,</strong> <em>185 </em>
                                    (1)
                                     <a href="https://doi.org/10.1007/s00604-017-2601-9" title="DOI URL">https://doi.org/10.1007/s00604-017-2601-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00604-017-2601-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00604-017-2601-9%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrochimica%2520Acta%26atitle%3DFluorometric%252Bgraphene%252Boxide-based%252Bdetection%252Bof%252BSalmonella%252Benteritis%252Busing%252Ba%252Btruncated%252BDNA%252Baptamer%26aulast%3DChinnappan%26aufirst%3DRaja%26date%3D2018%26date%3D2017%26volume%3D185%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meral  Yüce</span>, <span class="hlFld-ContribAuthor ">Hasan  Kurt</span>, <span class="hlFld-ContribAuthor ">Babar  Hussain</span>, <span class="hlFld-ContribAuthor ">Hikmet  Budak</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Evolution of Ligands by Exponential Enrichment for Aptamer Selection. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 211-243. <a href="https://doi.org/10.1016/B978-0-323-50878-0.00008-2" title="DOI URL">https://doi.org/10.1016/B978-0-323-50878-0.00008-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-50878-0.00008-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-50878-0.00008-2%26sid%3Dliteratum%253Aachs%26atitle%3DSystematic%252BEvolution%252Bof%252BLigands%252Bby%252BExponential%252BEnrichment%252Bfor%252BAptamer%252BSelection%26aulast%3DY%25C3%25BCce%26aufirst%3DMeral%26date%3D2018%26spage%3D211%26epage%3D243%26pub%3DElsevier%26atitle%3DBiomedical%252BApplications%252Bof%252BFunctionalized%252BNanomaterials%26date%3D2018%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaofan  Yu</span>, <span class="hlFld-ContribAuthor ">Fang  Chen</span>, <span class="hlFld-ContribAuthor ">Ronghui  Wang</span>, <span class="hlFld-ContribAuthor ">Yanbin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Whole-bacterium SELEX of DNA aptamers for rapid detection of E.coli O157:H7 using a QCM sensor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biotechnology</span><span> <strong>2018,</strong> <em>266 </em>, 39-49. <a href="https://doi.org/10.1016/j.jbiotec.2017.12.011" title="DOI URL">https://doi.org/10.1016/j.jbiotec.2017.12.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbiotec.2017.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbiotec.2017.12.011%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biotechnology%26atitle%3DWhole-bacterium%252BSELEX%252Bof%252BDNA%252Baptamers%252Bfor%252Brapid%252Bdetection%252Bof%252BE.coli%252BO157%25253AH7%252Busing%252Ba%252BQCM%252Bsensor%26aulast%3DYu%26aufirst%3DXiaofan%26date%3D2018%26volume%3D266%26spage%3D39%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Torsten  Schüling</span>, <span class="hlFld-ContribAuthor ">Alina  Eilers</span>, <span class="hlFld-ContribAuthor ">Thomas  Scheper</span>, <span class="hlFld-ContribAuthor ">Johanna  Walter</span>, . </span><span class="cited-content_cbyCitation_article-title">Aptamer-based lateral flow assays. </span><span class="cited-content_cbyCitation_journal-name">AIMS Bioengineering</span><span> <strong>2018,</strong> <em>5 </em>
                                    (2)
                                     , 78-102. <a href="https://doi.org/10.3934/bioeng.2018.2.78" title="DOI URL">https://doi.org/10.3934/bioeng.2018.2.78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3934/bioeng.2018.2.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3934%2Fbioeng.2018.2.78%26sid%3Dliteratum%253Aachs%26jtitle%3DAIMS%2520Bioengineering%26atitle%3DAptamer-based%252Blateral%252Bflow%252Bassays%26aulast%3DSch%25C3%25BCling%26aufirst%3DTorsten%26date%3D2018%26volume%3D5%26issue%3D2%26spage%3D78%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ceren  Bayraç</span>, <span class="hlFld-ContribAuthor ">Füsun  Eyidoğan</span>, <span class="hlFld-ContribAuthor ">Hüseyin  Avni Öktem</span>. </span><span class="cited-content_cbyCitation_article-title">DNA aptamer-based colorimetric detection platform for Salmonella Enteritidis. </span><span class="cited-content_cbyCitation_journal-name">Biosensors and Bioelectronics</span><span> <strong>2017,</strong> <em>98 </em>, 22-28. <a href="https://doi.org/10.1016/j.bios.2017.06.029" title="DOI URL">https://doi.org/10.1016/j.bios.2017.06.029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bios.2017.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bios.2017.06.029%26sid%3Dliteratum%253Aachs%26jtitle%3DBiosensors%2520and%2520Bioelectronics%26atitle%3DDNA%252Baptamer-based%252Bcolorimetric%252Bdetection%252Bplatform%252Bfor%252BSalmonella%252BEnteritidis%26aulast%3DBayra%25C3%25A7%26aufirst%3DCeren%26date%3D2017%26volume%3D98%26spage%3D22%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer  Soundy</span>, <span class="hlFld-ContribAuthor ">Darren  Day</span>, . </span><span class="cited-content_cbyCitation_article-title">Selection of DNA aptamers specific for live Pseudomonas aeruginosa. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (9)
                                     , e0185385. <a href="https://doi.org/10.1371/journal.pone.0185385" title="DOI URL">https://doi.org/10.1371/journal.pone.0185385</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0185385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0185385%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DSelection%252Bof%252BDNA%252Baptamers%252Bspecific%252Bfor%252Blive%252BPseudomonas%252Baeruginosa%26aulast%3DSoundy%26aufirst%3DJennifer%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3De0185385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hedieh  Malekzad</span>, <span class="hlFld-ContribAuthor ">Abolghasem  Jouyban</span>, <span class="hlFld-ContribAuthor ">Mohammad  Hasanzadeh</span>, <span class="hlFld-ContribAuthor ">Nasrin  Shadjou</span>, <span class="hlFld-ContribAuthor ">Miguel  de la Guardia</span>. </span><span class="cited-content_cbyCitation_article-title">Ensuring food safety using aptamer based assays: Electroanalytical approach. </span><span class="cited-content_cbyCitation_journal-name">TrAC Trends in Analytical Chemistry</span><span> <strong>2017,</strong> <em>94 </em>, 77-94. <a href="https://doi.org/10.1016/j.trac.2017.07.001" title="DOI URL">https://doi.org/10.1016/j.trac.2017.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.trac.2017.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.trac.2017.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTrAC%2520Trends%2520in%2520Analytical%2520Chemistry%26atitle%3DEnsuring%252Bfood%252Bsafety%252Busing%252Baptamer%252Bbased%252Bassays%25253A%252BElectroanalytical%252Bapproach%26aulast%3DMalekzad%26aufirst%3DHedieh%26date%3D2017%26volume%3D94%26spage%3D77%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sang-Choon  Lee</span>, <span class="hlFld-ContribAuthor ">Min-Seo  Kim</span>, <span class="hlFld-ContribAuthor ">Ki-Chun  Yoo</span>, <span class="hlFld-ContribAuthor ">Na-Reum  Ha</span>, <span class="hlFld-ContribAuthor ">Ji-Young  Moon</span>, <span class="hlFld-ContribAuthor ">Su-Jae  Lee</span>, <span class="hlFld-ContribAuthor ">Moon-Young  Yoon</span>. </span><span class="cited-content_cbyCitation_article-title">Sensitive fluorescent imaging of Salmonella enteritidis and Salmonella typhimurium using a polyvalent directed peptide polymer. </span><span class="cited-content_cbyCitation_journal-name">Microchimica Acta</span><span> <strong>2017,</strong> <em>184 </em>
                                    (8)
                                     , 2611-2620. <a href="https://doi.org/10.1007/s00604-017-2240-1" title="DOI URL">https://doi.org/10.1007/s00604-017-2240-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00604-017-2240-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00604-017-2240-1%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrochimica%2520Acta%26atitle%3DSensitive%252Bfluorescent%252Bimaging%252Bof%252BSalmonella%252Benteritidis%252Band%252BSalmonella%252Btyphimurium%252Busing%252Ba%252Bpolyvalent%252Bdirected%252Bpeptide%252Bpolymer%26aulast%3DLee%26aufirst%3DSang-Choon%26date%3D2017%26date%3D2017%26volume%3D184%26issue%3D8%26spage%3D2611%26epage%3D2620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheng  Lijuan</span>, <span class="hlFld-ContribAuthor ">Yan  Xing</span>, <span class="hlFld-ContribAuthor ">Wu  Minxi</span>, <span class="hlFld-ContribAuthor ">Li  Wenkai</span>, <span class="hlFld-ContribAuthor ">Deng  Le</span>. </span><span class="cited-content_cbyCitation_article-title">Development of an aptamer-ampicillin conjugate for treating biofilms. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2017,</strong> <em>483 </em>
                                    (2)
                                     , 847-854. <a href="https://doi.org/10.1016/j.bbrc.2017.01.016" title="DOI URL">https://doi.org/10.1016/j.bbrc.2017.01.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2017.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2017.01.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DDevelopment%252Bof%252Ban%252Baptamer-ampicillin%252Bconjugate%252Bfor%252Btreating%252Bbiofilms%26aulast%3DLijuan%26aufirst%3DCheng%26date%3D2017%26volume%3D483%26issue%3D2%26spage%3D847%26epage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong-Seok  Kim</span>, <span class="hlFld-ContribAuthor ">Young-Ji  Kim</span>, <span class="hlFld-ContribAuthor ">Jung-Whan  Chon</span>, <span class="hlFld-ContribAuthor ">Dong-Hyeon  Kim</span>, <span class="hlFld-ContribAuthor ">Jin-Hyeok  Yim</span>, <span class="hlFld-ContribAuthor ">Hyunsook  Kim</span>, <span class="hlFld-ContribAuthor ">Kun-Ho  Seo</span>. </span><span class="cited-content_cbyCitation_article-title">Two-stage label-free aptasensing platform for rapid detection of Cronobacter sakazakii in powdered infant formula. </span><span class="cited-content_cbyCitation_journal-name">Sensors and Actuators B: Chemical</span><span> <strong>2017,</strong> <em>239 </em>, 94-99. <a href="https://doi.org/10.1016/j.snb.2016.07.173" title="DOI URL">https://doi.org/10.1016/j.snb.2016.07.173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.snb.2016.07.173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.snb.2016.07.173%26sid%3Dliteratum%253Aachs%26jtitle%3DSensors%2520and%2520Actuators%2520B%253A%2520Chemical%26atitle%3DTwo-stage%252Blabel-free%252Baptasensing%252Bplatform%252Bfor%252Brapid%252Bdetection%252Bof%252BCronobacter%252Bsakazakii%252Bin%252Bpowdered%252Binfant%252Bformula%26aulast%3DKim%26aufirst%3DHong-Seok%26date%3D2017%26volume%3D239%26spage%3D94%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Hui  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaocheng  Li</span>, <span class="hlFld-ContribAuthor ">Suzhen  Ma</span>, <span class="hlFld-ContribAuthor ">Shuai  Men</span>, <span class="hlFld-ContribAuthor ">Heng  Wei</span>, <span class="hlFld-ContribAuthor ">Jingjing  Cui</span>, <span class="hlFld-ContribAuthor ">Hongning  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">A label-free fluorescent direct detection of live Salmonella typhimurium using cascade triple trigger sequences-regenerated strand displacement amplification and hairpin template-generated-scaffolded silver nanoclusters. </span><span class="cited-content_cbyCitation_journal-name">Biosensors and Bioelectronics</span><span> <strong>2017,</strong> <em>87 </em>, 1044-1049. <a href="https://doi.org/10.1016/j.bios.2016.09.037" title="DOI URL">https://doi.org/10.1016/j.bios.2016.09.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bios.2016.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bios.2016.09.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBiosensors%2520and%2520Bioelectronics%26atitle%3DA%252Blabel-free%252Bfluorescent%252Bdirect%252Bdetection%252Bof%252Blive%252BSalmonella%252Btyphimurium%252Busing%252Bcascade%252Btriple%252Btrigger%252Bsequences-regenerated%252Bstrand%252Bdisplacement%252Bamplification%252Band%252Bhairpin%252Btemplate-generated-scaffolded%252Bsilver%252Bnanoclusters%26aulast%3DZhang%26aufirst%3DPeng%26date%3D2017%26volume%3D87%26spage%3D1044%26epage%3D1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marleen  Renders</span>, <span class="hlFld-ContribAuthor ">Emily  Miller</span>, <span class="hlFld-ContribAuthor ">Curtis H.  Lam</span>, <span class="hlFld-ContribAuthor ">David M.  Perrin</span>. </span><span class="cited-content_cbyCitation_article-title">Whole cell-SELEX of aptamers with a tyrosine-like side chain against live bacteria. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (9)
                                     , 1980-1989. <a href="https://doi.org/10.1039/C6OB02451C" title="DOI URL">https://doi.org/10.1039/C6OB02451C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6OB02451C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6OB02451C%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DWhole%252Bcell-SELEX%252Bof%252Baptamers%252Bwith%252Ba%252Btyrosine-like%252Bside%252Bchain%252Bagainst%252Blive%252Bbacteria%26aulast%3DRenders%26aufirst%3DMarleen%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D9%26spage%3D1980%26epage%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jayeon  Lee</span>, <span class="hlFld-ContribAuthor ">Juyeon  Jung</span>, <span class="hlFld-ContribAuthor ">Chang Soo  Lee</span>, <span class="hlFld-ContribAuthor ">Tai Hwan  Ha</span>. </span><span class="cited-content_cbyCitation_article-title">Design and optimization of an ultra-sensitive hairpin DNA aptasensor for Salmonella detection. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2017,</strong> <em>7 </em>
                                    (56)
                                     , 34933-34938. <a href="https://doi.org/10.1039/C7RA06000A" title="DOI URL">https://doi.org/10.1039/C7RA06000A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA06000A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA06000A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDesign%252Band%252Boptimization%252Bof%252Ban%252Bultra-sensitive%252Bhairpin%252BDNA%252Baptasensor%252Bfor%252BSalmonella%252Bdetection%26aulast%3DLee%26aufirst%3DJayeon%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D56%26spage%3D34933%26epage%3D34938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N. A.  Zeninskaya</span>, <span class="hlFld-ContribAuthor ">A. V.  Kolesnikov</span>, <span class="hlFld-ContribAuthor ">A. K.  Ryabko</span>, <span class="hlFld-ContribAuthor ">I. G.  Shemyakin</span>, <span class="hlFld-ContribAuthor ">I. A.  Dyatlov</span>, <span class="hlFld-ContribAuthor ">A. V.  Kozyr</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamers in the Treatment of Bacterial Infections: Problems and Prospects. </span><span class="cited-content_cbyCitation_journal-name">Annals of the Russian academy of medical sciences</span><span> <strong>2016,</strong> <em>71 </em>
                                    (5)
                                     , 350-358. <a href="https://doi.org/10.15690/vramn591" title="DOI URL">https://doi.org/10.15690/vramn591</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15690/vramn591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15690%2Fvramn591%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnals%2520of%2520the%2520Russian%2520academy%2520of%2520medical%2520sciences%26atitle%3DAptamers%252Bin%252Bthe%252BTreatment%252Bof%252BBacterial%252BInfections%25253A%252BProblems%252Band%252BProspects%26aulast%3DZeninskaya%26aufirst%3DN.%2BA.%26date%3D2016%26date%3D2016%26volume%3D71%26issue%3D5%26spage%3D350%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Davydova</span>, <span class="hlFld-ContribAuthor ">Maria  Vorobjeva</span>, <span class="hlFld-ContribAuthor ">Dmitrii  Pyshnyi</span>, <span class="hlFld-ContribAuthor ">Sidney  Altman</span>, <span class="hlFld-ContribAuthor ">Valentin  Vlassov</span>, <span class="hlFld-ContribAuthor ">Alya  Venyaminova</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamers against pathogenic microorganisms. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Microbiology</span><span> <strong>2016,</strong> <em>42 </em>
                                    (6)
                                     , 847-865. <a href="https://doi.org/10.3109/1040841X.2015.1070115" title="DOI URL">https://doi.org/10.3109/1040841X.2015.1070115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/1040841X.2015.1070115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F1040841X.2015.1070115%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Microbiology%26atitle%3DAptamers%252Bagainst%252Bpathogenic%252Bmicroorganisms%26aulast%3DDavydova%26aufirst%3DAnna%26date%3D2016%26date%3D2015%26volume%3D42%26issue%3D6%26spage%3D847%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Hui  Liu</span>, <span class="hlFld-ContribAuthor ">Suzhen  Ma</span>, <span class="hlFld-ContribAuthor ">Shuai  Men</span>, <span class="hlFld-ContribAuthor ">Qingzhou  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Yang</span>, <span class="hlFld-ContribAuthor ">Hongning  Wang</span>, <span class="hlFld-ContribAuthor ">Anyun  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">A label-free ultrasensitive fluorescence detection of viable Salmonella enteritidis using enzyme-induced cascade two-stage toehold strand-displacement-driven assembly of G-quadruplex DNA. </span><span class="cited-content_cbyCitation_journal-name">Biosensors and Bioelectronics</span><span> <strong>2016,</strong> <em>80 </em>, 538-542. <a href="https://doi.org/10.1016/j.bios.2016.02.031" title="DOI URL">https://doi.org/10.1016/j.bios.2016.02.031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bios.2016.02.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bios.2016.02.031%26sid%3Dliteratum%253Aachs%26jtitle%3DBiosensors%2520and%2520Bioelectronics%26atitle%3DA%252Blabel-free%252Bultrasensitive%252Bfluorescence%252Bdetection%252Bof%252Bviable%252BSalmonella%252Benteritidis%252Busing%252Benzyme-induced%252Bcascade%252Btwo-stage%252Btoehold%252Bstrand-displacement-driven%252Bassembly%252Bof%252BG-quadruplex%252BDNA%26aulast%3DZhang%26aufirst%3DPeng%26date%3D2016%26volume%3D80%26spage%3D538%26epage%3D542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soledad  Marton</span>, <span class="hlFld-ContribAuthor ">Fernanda  Cleto</span>, <span class="hlFld-ContribAuthor ">Marco Aurélio  Krieger</span>, <span class="hlFld-ContribAuthor ">Josiane  Cardoso</span>, . </span><span class="cited-content_cbyCitation_article-title">Isolation of an Aptamer that Binds Specifically to E. coli. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2016,</strong> <em>11 </em>
                                    (4)
                                     , e0153637. <a href="https://doi.org/10.1371/journal.pone.0153637" title="DOI URL">https://doi.org/10.1371/journal.pone.0153637</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0153637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0153637%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIsolation%252Bof%252Ban%252BAptamer%252Bthat%252BBinds%252BSpecifically%252Bto%252BE.%252Bcoli%26aulast%3DMarton%26aufirst%3DSoledad%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D4%26spage%3De0153637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">G. S.  Zamay</span>, <span class="hlFld-ContribAuthor ">I. V.  Belyanina</span>, <span class="hlFld-ContribAuthor ">A. S.  Zamay</span>, <span class="hlFld-ContribAuthor ">M. A.  Komarova</span>, <span class="hlFld-ContribAuthor ">A. V.  Krat</span>, <span class="hlFld-ContribAuthor ">E. N.  Eremina</span>, <span class="hlFld-ContribAuthor ">R. A.  Zukov</span>, <span class="hlFld-ContribAuthor ">A. E.  Sokolov</span>, <span class="hlFld-ContribAuthor ">T. N.  Zamay</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of DNA aptamers for breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry (Moscow) Supplement Series B: Biomedical Chemistry</span><span> <strong>2016,</strong> <em>10 </em>
                                    (2)
                                     , 158-164. <a href="https://doi.org/10.1134/S1990750816020128" title="DOI URL">https://doi.org/10.1134/S1990750816020128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1990750816020128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1990750816020128%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520%2528Moscow%2529%2520Supplement%2520Series%2520B%253A%2520Biomedical%2520Chemistry%26atitle%3DSelection%252Bof%252BDNA%252Baptamers%252Bfor%252Bbreast%252Bcancer%26aulast%3DZamay%26aufirst%3DG.%2BS.%26date%3D2016%26date%3D2016%26volume%3D10%26issue%3D2%26spage%3D158%26epage%3D164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wentao  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamers and Their Application. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 219-244. <a href="https://doi.org/10.1007/978-981-10-1618-9_11" title="DOI URL">https://doi.org/10.1007/978-981-10-1618-9_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-10-1618-9_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-10-1618-9_11%26sid%3Dliteratum%253Aachs%26atitle%3DAptamers%252Band%252BTheir%252BApplication%26aulast%3DXu%26aufirst%3DWentao%26date%3D2016%26date%3D2016%26spage%3D219%26epage%3D244%26pub%3DSpringer%2520Singapore%26atitle%3DFunctional%252BNucleic%252BAcids%252BDetection%252Bin%252BFood%252BSafety%26aulast%3DXu%26aufirst%3DWentao%26date%3D2016%26volume%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Na-Reum  Ha</span>, <span class="hlFld-ContribAuthor ">Sang-Choon  Lee</span>, <span class="hlFld-ContribAuthor ">Jae-Wook  Hyun</span>, <span class="hlFld-ContribAuthor ">Moon-Young  Yoon</span>. </span><span class="cited-content_cbyCitation_article-title">Development of inhibitory ssDNA aptamers for the FtsZ cell division protein from citrus canker phytopathogen. </span><span class="cited-content_cbyCitation_journal-name">Process Biochemistry</span><span> <strong>2016,</strong> <em>51 </em>
                                    (1)
                                     , 24-33. <a href="https://doi.org/10.1016/j.procbio.2015.11.008" title="DOI URL">https://doi.org/10.1016/j.procbio.2015.11.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.procbio.2015.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.procbio.2015.11.008%26sid%3Dliteratum%253Aachs%26jtitle%3DProcess%2520Biochemistry%26atitle%3DDevelopment%252Bof%252Binhibitory%252BssDNA%252Baptamers%252Bfor%252Bthe%252BFtsZ%252Bcell%252Bdivision%252Bprotein%252Bfrom%252Bcitrus%252Bcanker%252Bphytopathogen%26aulast%3DHa%26aufirst%3DNa-Reum%26date%3D2016%26volume%3D51%26issue%3D1%26spage%3D24%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ghamar Soltan  Dorraj</span>, <span class="hlFld-ContribAuthor ">Mohammad Javad  Rassaee</span>, <span class="hlFld-ContribAuthor ">Ali Mohammad  Latifi</span>, <span class="hlFld-ContribAuthor ">Bahram  Pishgoo</span>, <span class="hlFld-ContribAuthor ">Mahmood  Tavallaei</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of DNA aptamers against Human Cardiac Troponin I for colorimetric sensor based dot blot application. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biotechnology</span><span> <strong>2015,</strong> <em>208 </em>, 80-86. <a href="https://doi.org/10.1016/j.jbiotec.2015.05.002" title="DOI URL">https://doi.org/10.1016/j.jbiotec.2015.05.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbiotec.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbiotec.2015.05.002%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biotechnology%26atitle%3DSelection%252Bof%252BDNA%252Baptamers%252Bagainst%252BHuman%252BCardiac%252BTroponin%252BI%252Bfor%252Bcolorimetric%252Bsensor%252Bbased%252Bdot%252Bblot%252Bapplication%26aulast%3DDorraj%26aufirst%3DGhamar%2BSoltan%26date%3D2015%26volume%3D208%26spage%3D80%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  Bruno</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting the Uncertain Future of Aptamer-Based Diagnostics and Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2015,</strong> <em>20 </em>
                                    (4)
                                     , 6866-6887. <a href="https://doi.org/10.3390/molecules20046866" title="DOI URL">https://doi.org/10.3390/molecules20046866</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules20046866&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules20046866%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPredicting%252Bthe%252BUncertain%252BFuture%252Bof%252BAptamer-Based%252BDiagnostics%252Band%252BTherapeutics%26aulast%3DBruno%26aufirst%3DJohn%26date%3D2015%26date%3D2015%26volume%3D20%26issue%3D4%26spage%3D6866%26epage%3D6887" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongming  Sang</span>, <span class="hlFld-ContribAuthor ">Frank  Blecha</span>. </span><span class="cited-content_cbyCitation_article-title">Alternatives to Antibiotics in Animal Agriculture:  An Ecoimmunological View. </span><span class="cited-content_cbyCitation_journal-name">Pathogens</span><span> <strong>2015,</strong> <em>4 </em>
                                    (1)
                                     , 1-19. <a href="https://doi.org/10.3390/pathogens4010001" title="DOI URL">https://doi.org/10.3390/pathogens4010001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/pathogens4010001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fpathogens4010001%26sid%3Dliteratum%253Aachs%26jtitle%3DPathogens%26atitle%3DAlternatives%252Bto%252BAntibiotics%252Bin%252BAnimal%252BAgriculture%25253A%252BAn%252BEcoimmunological%252BView%26aulast%3DSang%26aufirst%3DYongming%26date%3D2015%26date%3D2014%26volume%3D4%26issue%3D1%26spage%3D1%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darija  Muharemagic</span>, <span class="hlFld-ContribAuthor ">Maxim V.  Berezovski</span>. </span><span class="cited-content_cbyCitation_article-title">Aptamers in Oncotherapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 107-121. <a href="https://doi.org/10.1007/978-3-319-17305-4_5" title="DOI URL">https://doi.org/10.1007/978-3-319-17305-4_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-17305-4_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-17305-4_5%26sid%3Dliteratum%253Aachs%26atitle%3DAptamers%252Bin%252BOncotherapy%26aulast%3DMuharemagic%26aufirst%3DDarija%26date%3D2015%26spage%3D107%26epage%3D121%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DRNA%252Band%252BDNA%252BDiagnostics%26aulast%3DErdmann%26aufirst%3DVolker%2BA.%26date%3D2015%26volume%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ka Lok  Hong</span>, <span class="hlFld-ContribAuthor ">Letha J.  Sooter</span>. </span><span class="cited-content_cbyCitation_article-title">Single-Stranded DNA Aptamers against Pathogens and Toxins: Identification and Biosensing Applications. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2015,</strong> <em>2015 </em>, 1-31. <a href="https://doi.org/10.1155/2015/419318" title="DOI URL">https://doi.org/10.1155/2015/419318</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2015/419318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2015%2F419318%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DSingle-Stranded%252BDNA%252BAptamers%252Bagainst%252BPathogens%252Band%252BToxins%25253A%252BIdentification%252Band%252BBiosensing%252BApplications%26aulast%3DHong%26aufirst%3DKa%2BLok%26date%3D2015%26volume%3D2015%26spage%3D1%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renni  Deng</span>, <span class="hlFld-ContribAuthor ">Li  Wang</span>, <span class="hlFld-ContribAuthor ">Guangzhao  Yi</span>, <span class="hlFld-ContribAuthor ">Erhui  Hua</span>, <span class="hlFld-ContribAuthor ">Guoming  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Target-induced aptamer release strategy based on electrochemical detection of staphylococcal enterotoxin B using GNPs-ZrO2-Chits film. </span><span class="cited-content_cbyCitation_journal-name">Colloids and Surfaces B: Biointerfaces</span><span> <strong>2014,</strong> <em>120 </em>, 1-7. <a href="https://doi.org/10.1016/j.colsurfb.2014.04.028" title="DOI URL">https://doi.org/10.1016/j.colsurfb.2014.04.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.colsurfb.2014.04.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.colsurfb.2014.04.028%26sid%3Dliteratum%253Aachs%26jtitle%3DColloids%2520and%2520Surfaces%2520B%253A%2520Biointerfaces%26atitle%3DTarget-induced%252Baptamer%252Brelease%252Bstrategy%252Bbased%252Bon%252Belectrochemical%252Bdetection%252Bof%252Bstaphylococcal%252Benterotoxin%252BB%252Busing%252BGNPs-ZrO2-Chits%252Bfilm%26aulast%3DDeng%26aufirst%3DRenni%26date%3D2014%26volume%3D120%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhiyuan  Fang</span>, <span class="hlFld-ContribAuthor ">Wei  Wu</span>, <span class="hlFld-ContribAuthor ">Xuewen  Lu</span>, <span class="hlFld-ContribAuthor ">Lingwen  Zeng</span>. </span><span class="cited-content_cbyCitation_article-title">Lateral flow biosensor for DNA extraction-free detection of salmonella based on aptamer mediated strand displacement amplification. </span><span class="cited-content_cbyCitation_journal-name">Biosensors and Bioelectronics</span><span> <strong>2014,</strong> <em>56 </em>, 192-197. <a href="https://doi.org/10.1016/j.bios.2014.01.015" title="DOI URL">https://doi.org/10.1016/j.bios.2014.01.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bios.2014.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bios.2014.01.015%26sid%3Dliteratum%253Aachs%26jtitle%3DBiosensors%2520and%2520Bioelectronics%26atitle%3DLateral%252Bflow%252Bbiosensor%252Bfor%252BDNA%252Bextraction-free%252Bdetection%252Bof%252Bsalmonella%252Bbased%252Bon%252Baptamer%252Bmediated%252Bstrand%252Bdisplacement%252Bamplification%26aulast%3DFang%26aufirst%3DZhiyuan%26date%3D2014%26volume%3D56%26spage%3D192%26epage%3D197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tatyana N.  Zamay</span>, <span class="hlFld-ContribAuthor ">Olga S.  Kolovskaya</span>, <span class="hlFld-ContribAuthor ">Yury E.  Glazyrin</span>, <span class="hlFld-ContribAuthor ">Galina S.  Zamay</span>, <span class="hlFld-ContribAuthor ">Svetlana A.  Kuznetsova</span>, <span class="hlFld-ContribAuthor ">Ekaterina A.  Spivak</span>, <span class="hlFld-ContribAuthor ">Mohamed  Wehbe</span>, <span class="hlFld-ContribAuthor ">Anna G.  Savitskaya</span>, <span class="hlFld-ContribAuthor ">Olga A.  Zubkova</span>, <span class="hlFld-ContribAuthor ">Anastasia  Kadkina</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Wang</span>, <span class="hlFld-ContribAuthor ">Darija  Muharemagic</span>, <span class="hlFld-ContribAuthor ">Anna  Dubynina</span>, <span class="hlFld-ContribAuthor ">Yuliya  Sheina</span>, <span class="hlFld-ContribAuthor ">Alla B.  Salmina</span>, <span class="hlFld-ContribAuthor ">Maxim V.  Berezovski</span>, <span class="hlFld-ContribAuthor ">Anna S.  Zamay</span>. </span><span class="cited-content_cbyCitation_article-title">DNA-Aptamer Targeting Vimentin for Tumor Therapy
              In Vivo. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acid Therapeutics</span><span> <strong>2014,</strong> <em>24 </em>
                                    (2)
                                     , 160-170. <a href="https://doi.org/10.1089/nat.2013.0471" title="DOI URL">https://doi.org/10.1089/nat.2013.0471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/nat.2013.0471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fnat.2013.0471%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acid%2520Therapeutics%26atitle%3DDNA-Aptamer%252BTargeting%252BVimentin%252Bfor%252BTumor%252BTherapy%252BIn%252BVivo%26aulast%3DZamay%26aufirst%3DTatyana%2BN.%26date%3D2014%26volume%3D24%26issue%3D2%26spage%3D160%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jihea  Moon</span>, <span class="hlFld-ContribAuthor ">Giyoung  Kim</span>, <span class="hlFld-ContribAuthor ">Saetbyeol  Park</span>. </span><span class="cited-content_cbyCitation_article-title">Development of ssDNA aptamers for the capture and detection of Salmonella typhimurium. </span><span class="cited-content_cbyCitation_journal-name">Anal. Methods</span><span> <strong>2014,</strong> <em>6 </em>
                                    (18)
                                     , 7442-7448. <a href="https://doi.org/10.1039/C4AY01035C" title="DOI URL">https://doi.org/10.1039/C4AY01035C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4AY01035C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4AY01035C%26sid%3Dliteratum%253Aachs%26jtitle%3DAnal.%2520Methods%26atitle%3DDevelopment%252Bof%252BssDNA%252Baptamers%252Bfor%252Bthe%252Bcapture%252Band%252Bdetection%252Bof%252BSalmonella%252Btyphimurium%26aulast%3DMoon%26aufirst%3DJihea%26date%3D2014%26date%3D2014%26volume%3D6%26issue%3D18%26spage%3D7442%26epage%3D7448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Wu</span>, <span class="hlFld-ContribAuthor ">Zhiyuan  Fang</span>, <span class="hlFld-ContribAuthor ">Shiming  Zhao</span>, <span class="hlFld-ContribAuthor ">Xuewen  Lu</span>, <span class="hlFld-ContribAuthor ">Luxin  Yu</span>, <span class="hlFld-ContribAuthor ">Ting  Mei</span>, <span class="hlFld-ContribAuthor ">Lingwen  Zeng</span>. </span><span class="cited-content_cbyCitation_article-title">A simple aptamer biosensor for Salmonellae enteritidis based on fluorescence-switch signaling graphene oxide. </span><span class="cited-content_cbyCitation_journal-name">RSC Adv.</span><span> <strong>2014,</strong> <em>4 </em>
                                    (42)
                                     , 22009-22012. <a href="https://doi.org/10.1039/C4RA01901F" title="DOI URL">https://doi.org/10.1039/C4RA01901F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4RA01901F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4RA01901F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Adv.%26atitle%3DA%252Bsimple%252Baptamer%252Bbiosensor%252Bfor%252BSalmonellae%252Benteritidis%252Bbased%252Bon%252Bfluorescence-switch%252Bsignaling%252Bgraphene%252Boxide%26aulast%3DWu%26aufirst%3DWei%26date%3D2014%26date%3D2014%26volume%3D4%26issue%3D42%26spage%3D22009%26epage%3D22012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thao T.  Le</span>, <span class="hlFld-ContribAuthor ">Orada  Chumphukam</span>, <span class="hlFld-ContribAuthor ">Anthony E. G.  Cass</span>. </span><span class="cited-content_cbyCitation_article-title">Determination of minimal sequence for binding of an aptamer. A comparison of truncation and hybridization inhibition methods. </span><span class="cited-content_cbyCitation_journal-name">RSC Adv.</span><span> <strong>2014,</strong> <em>4 </em>
                                    (88)
                                     , 47227-47233. <a href="https://doi.org/10.1039/C4RA08243E" title="DOI URL">https://doi.org/10.1039/C4RA08243E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4RA08243E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4RA08243E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Adv.%26atitle%3DDetermination%252Bof%252Bminimal%252Bsequence%252Bfor%252Bbinding%252Bof%252Ban%252Baptamer.%252BA%252Bcomparison%252Bof%252Btruncation%252Band%252Bhybridization%252Binhibition%252Bmethods%26aulast%3DLe%26aufirst%3DThao%2BT.%26date%3D2014%26date%3D2014%26volume%3D4%26issue%3D88%26spage%3D47227%26epage%3D47233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">O. S.  Kolovskaya</span>, <span class="hlFld-ContribAuthor ">T. N.  Zamay</span>, <span class="hlFld-ContribAuthor ">A. S.  Zamay</span>, <span class="hlFld-ContribAuthor ">Y. E.  Glazyrin</span>, <span class="hlFld-ContribAuthor ">E. A.  Spivak</span>, <span class="hlFld-ContribAuthor ">O. A.  Zubkova</span>, <span class="hlFld-ContribAuthor ">A. V.  Kadkina</span>, <span class="hlFld-ContribAuthor ">E. N.  Erkaev</span>, <span class="hlFld-ContribAuthor ">G. S.  Zamay</span>, <span class="hlFld-ContribAuthor ">A. G.  Savitskaya</span>, <span class="hlFld-ContribAuthor ">L. V.  Trufanova</span>, <span class="hlFld-ContribAuthor ">L. L.  Petrova</span>, <span class="hlFld-ContribAuthor ">M. V.  Berezovski</span>. </span><span class="cited-content_cbyCitation_article-title">DNA-aptamer/protein interaction as a cause of apoptosis and arrest of proliferation in Ehrlich ascites adenocarcinoma cells. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry (Moscow) Supplement Series A: Membrane and Cell Biology</span><span> <strong>2014,</strong> <em>8 </em>
                                    (1)
                                     , 60-72. <a href="https://doi.org/10.1134/S1990747813050061" title="DOI URL">https://doi.org/10.1134/S1990747813050061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1990747813050061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1990747813050061%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520%2528Moscow%2529%2520Supplement%2520Series%2520A%253A%2520Membrane%2520and%2520Cell%2520Biology%26atitle%3DDNA-aptamer%25252Fprotein%252Binteraction%252Bas%252Ba%252Bcause%252Bof%252Bapoptosis%252Band%252Barrest%252Bof%252Bproliferation%252Bin%252BEhrlich%252Bascites%252Badenocarcinoma%252Bcells%26aulast%3DKolovskaya%26aufirst%3DO.%2BS.%26date%3D2014%26date%3D2014%26volume%3D8%26issue%3D1%26spage%3D60%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Aptamer selection scheme for <i>S. enteritidis</i> and <i>S. typhimurium</i>. The first round included positive selection to <i>S. enteritidis</i> or <i>S. typhimurium</i>. The second and each subsequent round were started from negative selection and consisted of the following steps: incubation with bacteria mixture, <i>S. typhimurium</i> or <i>S. enteritidis</i>, heat treated Salmonellas, <i>S. aureus</i>, <i>E. coli</i>, <i>P. aeruginosa</i>, and <i>C. freundi</i>; collection of unbound sequences; incubation of collected ssDNA with <i>S. enteritidis</i> or <i>S. typhimurium</i>; washing of unbound ssDNA from <i>Salmonella</i>; extraction of bound ssDNA sequences; and ssDNA amplification by symmetric and asymmetric PCR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Flow cytometry binding analysis of the Alexa-488 labeled ssDNA to salmonellas. (A) Two-dimensional plot of side and forward scattering of buffer particles. (B) Gating for salmonella. (C, D) Histograms of ssDNA library and 7th pool binding to <i>S. enteritidis</i> and <i>S. typhimurium</i> for intact bacteria (red), ssDNA library with salmonella (green), and the pool with highest affinity (blue). #1 is the gate for Alexa-488 positive bacteria, and #2 is the gate for Alexa-488 negative bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Flow cytometry analysis of selected aptamer pools’ affinity to <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (B). 50 × 10<sup>3</sup> CFU was preincubated with 0.1 mg mL<sup>–1</sup> of salmon sperm DNA for 15 min at 19–24 °C. Afterward, bacteria were incubated with the 100 nM Alexa-488-labeled aptamer pool in 500 μL DPBS buffer for 15 min at 25 °C. The percentage of bacteria cells with bound ssDNA was analyzed on a flow cytometer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of aptamer pools on colony formation on Petri dishes with LB agar. (A) Bacterial culturability test after incubation of salmonella with aptamers. Aptamer pools, clones, or ssDNA library (1 μM) in DPBS were incubated with <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min. Bacteria were diluted, plated on Petri dishes with LB agar, and incubated for 18 h at 37 °C in a dry incubator (repeated in triplicate). The effect was estimated by counting the number of colonies. (B) Intact <i>S. typhimurium</i>; (C) <i>S. typhimurium</i> plated after a 30-min incubation with 1 μM of synthetic aptamer ST-12; (D) Intact <i>S. enteritidis</i>; (E) <i>S. enteritidis</i> plated after a 30-min incubation with 1 μM of synthetic aptamer SE-20. In order to get the same number of bacteria in the beginning of incubation, B and C, and D and E, were initially diluted from the same aliquot of <i>S. enteritidis</i> or <i>S. typhimurium</i>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Antibacterial effect of aptamers on <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (B) showed by culturability and viability Rhodamine 123 tests. The symbols in the figure indicate the following: *, Pool SE combined from clones SE-6, SE-20, and SE-22; **, Random sequence: CTCCTCTGACTGTAACCACGGATTGTCAGCGCGCCGCCGGATCAAGCATTTCGACAATTCGCATAGGTAGTCCAGAAGCC; ***, Pool ST combined from clones ST-2, ST-12, ST-20, and ST-33; <sup>1</sup><i>Growth suppression.</i> 1 μM of ssDNA samples in DPBS were incubated with 10<sup>3</sup> CFU mL<sup>–1</sup> of <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min. Afterward, the mixture was diluted and plated on Petri dishes with LB Agar and incubated for 18 h at 37 °C in a dry incubator (done in triplicate). The effect was estimated by counting the number of colonies (<i>N</i>). The number of colonies (<i>N</i><sub>c</sub>) in control plates with intact <i>S. enteritidis</i> or <i>S. typhimurium</i> (incubated in DPBS without ssDNA) was considered as 100%. Growth suppression was calculated as 100 – (<i>N</i> × 100/<i>N</i><sub>c</sub>). <sup>2</sup><i>Cell wall electrochemical gradient decrease.</i> Samples of  1 μM ssDNA in DPBS were incubated with 5 × 10<sup>3</sup> CFU mL<sup>–1</sup><i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min and 18 h. EGTA at the final concentration 1 mM was added to samples to permeabilize the bacterial outer membrane; bacteria were then loaded with Rh123 and analyzed on a flow cytometer. The samples were loaded with Rh123 (2 μg/mL final concentration) for 15 min at 19–24 °C and analyzed. The median of the histogram was taken as a relative value of electrochemical gradient (EG). Samples with intact <i>S. typhimurium</i> or <i>S. enteritidis</i> incubated in DPBS without ssDNA and loaded with Rh123 (EGc) were considered as positive controls (100% viability). The electrochemical gradient decrease was calculated as EGc – EG. For example, EG for <i>S. typhimurium</i> was 28.9 relative units, and EG for heat treated <i>S. typhimurium</i> was 15.9 relative units; therefore, the EG decrease (EGc – EG) was 13 relative units.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/medium/jm-2012-01856j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Bacteria viability test based on Rhodamine 123 uptake. Samples of 1 μM ssDNA in DPBS were incubated with 5 × 10<sup>3</sup> CFU mL<sup>–1</sup> of <i>S. typhimurium</i> or <i>S. enteritidis</i> for 30 min and 18 h. EGTA was added to samples in order to permeabilize the bacterial outer membrane; samples were then loaded with Rh123 for 15 min at 19–24 °C and measured on a flow cytometer. Samples with live intact bacteria and heat treated salmonella (10 min at 95 °C) in DPBS loaded with Rh123 were taken as a positive and a negative control. Flow cytometric analysis of Rh123 uptake by <i>S. enteritidis</i> (A) and <i>S. typhimurium</i> (D): intact salmonella (blue), unstained bacteria (red), heat treated bacteria (green), initial ssDNA library (purple), <i>S. enteritidis</i> (black) or <i>S. typhimurium</i> (orange) incubated for 30 min accordingly with pool SE or pool ST and aptamer clones SE-20 or ST-12.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-4/jm301856j/production/images/large/jm-2012-01856j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301856j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68188" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68188" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Pegues, D. A.; Ohl, M. E.; Miller, S. I.</span><span> </span><span class="NLM_article-title">Salmonella species, including Salmonella typhi</span>. In  <span class="citation_source-book">Principles and Practice of Infectious Diseases</span>, <span class="NLM_edition">6</span>th ed.; <span class="NLM_contrib-group">Mandell, G. L., Bennet, J. E., Dolin, R.</span>, Eds.; <span class="NLM_publisher-name">Churchill Livingstone</span>: <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">2636</span><span class="NLM_x">–</span> <span class="NLM_lpage">2654</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=2636-2654&author=D.+A.+Pegues&author=M.+E.+Ohl&author=S.+I.+Millerauthor=G.+L.+Mandell&author=J.+E.+Bennet&author=R.+Dolin&title=Principles+and+Practice+of+Infectious+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPegues%26aufirst%3DD.%2BA.%26atitle%3DSalmonella%2520species%252C%2520including%2520Salmonella%2520typhi%26btitle%3DPrinciples%2520and%2520Practice%2520of%2520Infectious%2520Diseases%26aulast%3DMandell%26aufirst%3DG.%2BL.%26pub%3DChurchill%2520Livingstone%26pub%3DElsevier%26date%3D2005%26spage%3D2636%26epage%3D2654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Helms, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ethelberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molbak, K.</span><span> </span><span class="NLM_article-title">International Salmonella typhimurium DT104 infections, 1992–2001</span> <span class="citation_source-journal">Emerging Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">867</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.3201%2Feid1106.041017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=15963280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A280%3ADC%252BD2MzhsFGqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=859-867&author=M.+Helmsauthor=S.+Ethelbergauthor=K.+Molbak&title=International+Salmonella+typhimurium+DT104+infections%2C+1992%E2%80%932001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">International Salmonella Typhimurium DT104 infections, 1992-2001</span></div><div class="casAuthors">Helms Morten; Ethelberg Steen; Molbak Kare</div><div class="citationInfo"><span class="NLM_cas:title">Emerging infectious diseases</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-67</span>
        ISSN:<span class="NLM_cas:issn">1080-6040</span>.
    </div><div class="casAbstract">The incidence of multidrug-resistant (MDR) Salmonella Typhimurium infections in humans, and in particular MDR definitive phage type 104 (DT104), has increased substantially in many countries in the last 2 decades, often associated with increased illness.  To examine the magnitude of this problem, a survey was conducted among countries with available antimicrobial resistance or phage typing surveillance data.  A total of 29, primarily industrialized, countries participated in the survey, which covered the years 1992-2001.  Overall, the incidence of MDR S.  Typhimurium and DT104 increased continuously during this period, although the problem affected primarily Europe and North America.  The increase appeared to have peaked in the United Kingdom but not in other countries.  Also, the incidence of quinolone-resistant S.  Typhimurium was increasing.  This survey implies that MDR S.  Typhimurium constitutes an increasing public health problem in large parts of the world and emphasizes the importance of surveillance and control programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8Hcwa-B2JQfIW7hznt0zpfW6udTcc2eZzrm7HmW_F4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzhsFGqsw%253D%253D&md5=c0d32f308e43a01d8cac443356583d85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3201%2Feid1106.041017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3201%252Feid1106.041017%26sid%3Dliteratum%253Aachs%26aulast%3DHelms%26aufirst%3DM.%26aulast%3DEthelberg%26aufirst%3DS.%26aulast%3DMolbak%26aufirst%3DK.%26atitle%3DInternational%2520Salmonella%2520typhimurium%2520DT104%2520infections%252C%25201992%25E2%2580%25932001%26jtitle%3DEmerging%2520Infect.%2520Dis.%26date%3D2005%26volume%3D11%26spage%3D859%26epage%3D867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Pui, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyaletchumi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noor Hidayah, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farinazleen, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheah, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, R.</span><span> </span><span class="NLM_article-title">Salmonella: A foodborne pathogen</span> <span class="citation_source-journal">Int. Food Res. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">473</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=465-473&author=C.+F.+Puiauthor=W.+C.+Wongauthor=L.+C.+Chaiauthor=R.+Tunungauthor=P.+Jeyaletchumiauthor=M.+S.+Noor+Hidayahauthor=A.+Ubongauthor=M.+G.+Farinazleenauthor=Y.+K.+Cheahauthor=R.+Son&title=Salmonella%3A+A+foodborne+pathogen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPui%26aufirst%3DC.%2BF.%26aulast%3DWong%26aufirst%3DW.%2BC.%26aulast%3DChai%26aufirst%3DL.%2BC.%26aulast%3DTunung%26aufirst%3DR.%26aulast%3DJeyaletchumi%26aufirst%3DP.%26aulast%3DNoor%2BHidayah%26aufirst%3DM.%2BS.%26aulast%3DUbong%26aufirst%3DA.%26aulast%3DFarinazleen%26aufirst%3DM.%2BG.%26aulast%3DCheah%26aufirst%3DY.%2BK.%26aulast%3DSon%26aufirst%3DR.%26atitle%3DSalmonella%253A%2520A%2520foodborne%2520pathogen%26jtitle%3DInt.%2520Food%2520Res.%2520J.%26date%3D2011%26volume%3D18%26spage%3D465%26epage%3D473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Schlesinger, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahousse, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.-K.</span><span> </span><span class="NLM_article-title">Metallo-β-lactamases and aptamer-based inhibition</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=419-428&author=S.+R.+Schlesingerauthor=M.+J.+Lahousseauthor=T.+O.+Fosterauthor=S.-K.+Kim&title=Metallo-%CE%B2-lactamases+and+aptamer-based+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchlesinger%26aufirst%3DS.%2BR.%26aulast%3DLahousse%26aufirst%3DM.%2BJ.%26aulast%3DFoster%26aufirst%3DT.%2BO.%26aulast%3DKim%26aufirst%3DS.-K.%26atitle%3DMetallo-%25CE%25B2-lactamases%2520and%2520aptamer-based%2520inhibition%26jtitle%3DPharmaceuticals%26date%3D2011%26volume%3D4%26spage%3D419%26epage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Vivekananda, J.; Keil, J. L.</span>US20036569630,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Vivekananda%2C+J.%3B+Keil%2C+J.+L.US20036569630%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVivekananda%26aufirst%3DJ.%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Xiaolian, Z.; Fan, C.</span>CN101109013,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Xiaolian%2C+Z.%3B+Fan%2C+C.CN101109013%2C+2008."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DXiaolian%26aufirst%3DZ.%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Morrissey, D.; Haeberli, P.</span>WO03106476,<span class="NLM_x"> </span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Morrissey%2C+D.%3B+Haeberli%2C+P.WO03106476%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMorrissey%26aufirst%3DD.%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ellington, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szostak, J.</span><span> </span><span class="NLM_article-title">In vitro selection of RNA molecules that bind specific ligands</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1038%2F346818a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=1697402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=1990&pages=818-822&author=A.+Ellingtonauthor=J.+Szostak&title=In+vitro+selection+of+RNA+molecules+that+bind+specific+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro selection of RNA molecules that bind specific ligands</span></div><div class="casAuthors">Ellington, Andrew D.; Szostak, Jack W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6287</span>),
    <span class="NLM_cas:pages">818-22</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Subpopulations of RNA mols. that bind specifically to a variety of org. dyes have been isolated from a population of random sequence RNA mols.  Roughly one in 1010 random sequence RNA mols. folds in such a way as to create a specific binding site for small ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTZFjRL1GKQ7Vg90H21EOLACvtfcHk0livD-vi1zG02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGgsw%253D%253D&md5=522ad1b4f0284ee88173b02035f9e9c5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F346818a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F346818a0%26sid%3Dliteratum%253Aachs%26aulast%3DEllington%26aufirst%3DA.%26aulast%3DSzostak%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520selection%2520of%2520RNA%2520molecules%2520that%2520bind%2520specific%2520ligands%26jtitle%3DNature%26date%3D1990%26volume%3D346%26spage%3D818%26epage%3D822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Tuerk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, L.</span><span> </span><span class="NLM_article-title">Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">249</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">510</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1126%2Fscience.2200121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=2200121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaK3cXlt1OltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1990&pages=505-510&author=C.+Tuerkauthor=L.+Gold&title=Systematic+evolution+of+ligands+by+exponential+enrichment%3A+RNA+ligands+to+bacteriophage+T4+DNA+polymerase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic evolution of ligands by exponential enrichment:  RNA ligands to bacteriophage T4 DNA polymerase</span></div><div class="casAuthors">Tuerk, Craig; Gold, Larry</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">4968</span>),
    <span class="NLM_cas:pages">505-10</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A novel method was developed for rapidly selecting preferred binding sequences from a population of random sequences.  The method is designated systematic evolution of ligands by exponential enrichment (SELEX).  High-affinity nucleic acid ligands for a protein were isolated by a procedure that depends on alternate cycles of ligand selection from pools of variant sequences and amplification of the bound species.  Multiple rounds exponentially enrich the population for the highest affinity species that can be clonally isolated and characterized.  In particular one eight-base region of an RNA that interacts with the T4 DNA polymerase was chosen and randomized.  Two different sequences were selected by this procedure from the calcd. pool of 65,536 species.  One is the wild-type sequence found in the bacteriophage mRNA; one is varied from wild type at four positions.  The binding consts. of these two RNA's to T4 DNA polymerase are equiv.  These protocols with minimal modification can yield high-affinity ligands for any protein that binds nucleic acids as part of its function; high affinity ligands could conceivably be developed for any target mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTzwgV60C9LVg90H21EOLACvtfcHk0liQd1-THsCAmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlt1OltLY%253D&md5=a40a28db9df4a5da4c1b1652a34aa715</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.2200121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2200121%26sid%3Dliteratum%253Aachs%26aulast%3DTuerk%26aufirst%3DC.%26aulast%3DGold%26aufirst%3DL.%26atitle%3DSystematic%2520evolution%2520of%2520ligands%2520by%2520exponential%2520enrichment%253A%2520RNA%2520ligands%2520to%2520bacteriophage%2520T4%2520DNA%2520polymerase%26jtitle%3DScience%26date%3D1990%26volume%3D249%26spage%3D505%26epage%3D510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Berezovski, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lechmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musheev, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, S. N.</span><span> </span><span class="NLM_article-title">Aptamer-facilitated biomarker discovery (AptaBiD)</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9137</span><span class="NLM_x">–</span> <span class="NLM_lpage">9143</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja801951p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9137-9143&author=M.+V.+Berezovskiauthor=M.+Lechmannauthor=M.+U.+Musheevauthor=T.+W.+Makauthor=S.+N.+Krylov&title=Aptamer-facilitated+biomarker+discovery+%28AptaBiD%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja801951p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja801951p%26sid%3Dliteratum%253Aachs%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26aulast%3DLechmann%26aufirst%3DM.%26aulast%3DMusheev%26aufirst%3DM.%2BU.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DKrylov%26aufirst%3DS.%2BN.%26atitle%3DAptamer-facilitated%2520biomarker%2520discovery%2520%2528AptaBiD%2529%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9137%26epage%3D9143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Shangguan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardona, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witek, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, W.</span><span> </span><span class="NLM_article-title">Identification of liver cancer-specific aptamers using whole live cells</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac701962v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=721-728&author=D.+Shangguanauthor=L.+Mengauthor=Z.+C.+Caoauthor=Z.+Xiaoauthor=X.+Fangauthor=Y.+Liauthor=D.+Cardonaauthor=R.+P.+Witekauthor=C.+Liuauthor=W.+Tan&title=Identification+of+liver+cancer-specific+aptamers+using+whole+live+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fac701962v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac701962v%26sid%3Dliteratum%253Aachs%26aulast%3DShangguan%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DZ.%2BC.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCardona%26aufirst%3DD.%26aulast%3DWitek%26aufirst%3DR.%2BP.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520liver%2520cancer-specific%2520aptamers%2520using%2520whole%2520live%2520cells%26jtitle%3DAnal.%2520Chem.%26date%3D2008%26volume%3D80%26spage%3D721%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Javaherian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musheev, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanoatov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, S. N.</span><span> </span><span class="NLM_article-title">Selection of aptamers for a protein target in cell lysate and their application to protein purification</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e62</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=e62&issue=8&author=S.+Javaherianauthor=M.+U.+Musheevauthor=M.+Kanoatovauthor=M.+V.+Berezovskiauthor=S.+N.+Krylov&title=Selection+of+aptamers+for+a+protein+target+in+cell+lysate+and+their+application+to+protein+purification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJavaherian%26aufirst%3DS.%26aulast%3DMusheev%26aufirst%3DM.%2BU.%26aulast%3DKanoatov%26aufirst%3DM.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26aulast%3DKrylov%26aufirst%3DS.%2BN.%26atitle%3DSelection%2520of%2520aptamers%2520for%2520a%2520protein%2520target%2520in%2520cell%2520lysate%2520and%2520their%2520application%2520to%2520protein%2520purification%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3D8%26spage%3De62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">electrochemical sensing of aptamer-facilitated virus immunoshielding</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">–</span> <span class="NLM_lpage">1686</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac202978r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=1677-1686&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=electrochemical+sensing+of+aptamer-facilitated+virus+immunoshielding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fac202978r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac202978r%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3Delectrochemical%2520sensing%2520of%2520aptamer-facilitated%2520virus%2520immunoshielding%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D1677%26epage%3D1686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">Aptamer-based viability impedimetric sensor for viruses</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1816</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac203412m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=1813-1816&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=Aptamer-based+viability+impedimetric+sensor+for+viruses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fac203412m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac203412m%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3DAptamer-based%2520viability%2520impedimetric%2520sensor%2520for%2520viruses%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D1813%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamay, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muharemagic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chechik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berezovski, M. V.</span><span> </span><span class="NLM_article-title">Electrochemical differentiation of epitope-specific aptamers</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">2548</span><span class="NLM_x">–</span> <span class="NLM_lpage">2556</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac300047c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=2548-2556&author=M.+Labibauthor=A.+S.+Zamayauthor=D.+Muharemagicauthor=A.+Chechikauthor=J.+C.+Bellauthor=M.+V.+Berezovski&title=Electrochemical+differentiation+of+epitope-specific+aptamers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fac300047c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac300047c%26sid%3Dliteratum%253Aachs%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DZamay%26aufirst%3DA.%2BS.%26aulast%3DMuharemagic%26aufirst%3DD.%26aulast%3DChechik%26aufirst%3DA.%26aulast%3DBell%26aufirst%3DJ.%2BC.%26aulast%3DBerezovski%26aufirst%3DM.%2BV.%26atitle%3DElectrochemical%2520differentiation%2520of%2520epitope-specific%2520aptamers%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D2548%26epage%3D2556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Ulrich, H.</span><span> </span><span class="NLM_article-title">RNA aptamers: from basic science towards therapy</span> <span class="citation_source-journal">Handb. Exp. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">173</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1007%2F3-540-27262-3_15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=16594622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVWit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2006&pages=305-326&author=H.+Ulrich&title=RNA+aptamers%3A+from+basic+science+towards+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RNA aptamers: from basic science towards therapy</span></div><div class="casAuthors">Ulrich, H.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">RNA towards Medicine</span>),
    <span class="NLM_cas:pages">305-326</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The SELEX technique (systematic evolution of ligands by exponential enrichment) provides a powerful tool for the in vitro selection of nucleic acid ligands (aptamers) from combinatorial oligonucleotide libraries against a target mol.  In the beginning of the technique's use, RNA mols. were identified that bind to proteins that naturally interact with nucleic acids or to small org. mols.  In the following years, the use of the SELEX technique was extended to isolate oligonucleotide ligands (aptamers) for a wide range of proteins of importance for therapy and diagnostics, such as growth factors and cell surface antigens.  These oligonucleotides bind their targets with similar affinities and specificities as antibodies do.  The in vitro selection of oligonucleotides with enzymic activity, denominated aptazymes, allows the direct transduction of mol. recognition to catalysis.  Recently, the use of in vitro selection methods to isolate protein inhibitors was extended to complex targets, such as membrane-bound receptors, and even entire cells.  RNA aptamers were also expressed in living cells.  These aptamers, also called intramers, can be used to dissect intracellular signal transduction pathways.  The utility of RNA aptamers for in vivo expts., as well as for diagnostic and therapeutic purposes, is considerably enhanced by chem. modifications, such as substitutions of the 2'-OH groups of the ribose backbone to provide resistance against enzymic degrdn. in biol. fluids.  In an alternative approach, Spiegelmers are identified through in vitro selection of an unmodified D-RNA mol. against a mirror-image (i.e. a D-peptide) of a selection target, followed by synthesis of the unnatural nuclease-resistant L-configuration of the RNA aptamer that recognizes the natural configuration of its selection target (i.e. a L-peptide).  Recently, nuclease-resistant inhibitory RNA aptamers were developed against a great variety of targets implicated in disease.  Some results have already been obtained in animal models and in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGDYZXLANvbVg90H21EOLACvtfcHk0lgDWtVgF7Hy8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVWit7Y%253D&md5=65f1b55cf095291c2417356172eca00b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F3-540-27262-3_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F3-540-27262-3_15%26sid%3Dliteratum%253Aachs%26aulast%3DUlrich%26aufirst%3DH.%26atitle%3DRNA%2520aptamers%253A%2520from%2520basic%2520science%2520towards%2520therapy%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2006%26volume%3D173%26spage%3D305%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Vavvas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Amico, D. J.</span><span> </span><span class="NLM_article-title">Pegaptanib (Macugen): treating neovascular age-related macular degeneration and current role in clinical practice</span> <span class="citation_source-journal">Ophthalmol. Clin. North. Am.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=16935210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A280%3ADC%252BD28rht1yqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=353-360&author=D.+Vavvasauthor=D.+J.+D%E2%80%99Amico&title=Pegaptanib+%28Macugen%29%3A+treating+neovascular+age-related+macular+degeneration+and+current+role+in+clinical+practice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pegaptanib (Macugen): treating neovascular age-related macular degeneration and current role in clinical practice</span></div><div class="casAuthors">Vavvas Demetrios; D'Amico Donald J</div><div class="citationInfo"><span class="NLM_cas:title">Ophthalmology clinics of North America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-60</span>
        ISSN:<span class="NLM_cas:issn">0896-1549</span>.
    </div><div class="casAbstract">Repeated Macugen intravitreal injections, well tolerated and safe when performed according to protocol, is the first successful pharmacotherapy for wet AMD.  Macugen has already had a significant impact on neovascular AMD management, in regard to both individual patients with AMD lesions that would become amenable to treatment, and in its effects on visual function and its preservation in the aging United States population.  Although results and delivery method are not optimal, the positive outcomes indicate the beginning, and not the limit, of pharmacotherapy for AMD.  The benefits of Macugen therapy for AMD strongly outweigh the risks.  It is pointed out by some that the overall magnitude of the efficacy results is very similar to the PDT trials and question why there is so much enthusiasm about a treatment that entails intravitreal injections every 6 weeks instead of PDT every 12 weeks.  In all fairness, PDT is not equally efficacious across the lesion subtypes and sizes, whereas Macugen has effect with all lesion subtypes.  In addition, these studies widen the armamentarium and open up the possibility of combination therapy in attacking neovascularization through multiple ways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfiy0myGSrxT0O18WR3iAofW6udTcc2ea_xkjFfljiX7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28rht1yqtA%253D%253D&md5=3b129dd4b19ffa835c49a6af932f612c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVavvas%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Amico%26aufirst%3DD.%2BJ.%26atitle%3DPegaptanib%2520%2528Macugen%2529%253A%2520treating%2520neovascular%2520age-related%2520macular%2520degeneration%2520and%2520current%2520role%2520in%2520clinical%2520practice%26jtitle%3DOphthalmol.%2520Clin.%2520North.%2520Am.%26date%3D2006%26volume%3D19%26spage%3D353%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Keefe, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellington, A.</span><span> </span><span class="NLM_article-title">Aptamers as therapeutics</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=538-550&author=A.+D.+Keefeauthor=S.+Paiauthor=A.+Ellington&title=Aptamers+as+therapeutics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DPai%26aufirst%3DS.%26aulast%3DEllington%26aufirst%3DA.%26atitle%3DAptamers%2520as%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D9%26spage%3D538%26epage%3D550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hamula, C. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, X. C.</span><span> </span><span class="NLM_article-title">Selection of aptamers against live bacterial cells</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">7812</span><span class="NLM_x">–</span> <span class="NLM_lpage">7819</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac801272s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=7812-7819&issue=20&author=C.+L.+A.+Hamulaauthor=H.+Zhangauthor=L.+L.+Guanauthor=X.-F.+Liauthor=X.+C.+Le&title=Selection+of+aptamers+against+live+bacterial+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fac801272s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac801272s%26sid%3Dliteratum%253Aachs%26aulast%3DHamula%26aufirst%3DC.%2BL.%2BA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DL.%2BL.%26aulast%3DLi%26aufirst%3DX.-F.%26aulast%3DLe%26aufirst%3DX.%2BC.%26atitle%3DSelection%2520of%2520aptamers%2520against%2520live%2520bacterial%2520cells%26jtitle%3DAnal.%2520Chem.%26date%3D2008%26volume%3D80%26issue%3D20%26spage%3D7812%26epage%3D7819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hamula, C. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-F.</span><span> </span><span class="NLM_article-title">DNA Aptamers binding to multiple prevalent M-types of streptococcus pyogenes</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">3640</span><span class="NLM_x">–</span> <span class="NLM_lpage">3647</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac200575e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2011&pages=3640-3647&issue=10&author=C.+L.+A.+Hamulaauthor=X.+C.+Leauthor=X.-F.+Li&title=DNA+Aptamers+binding+to+multiple+prevalent+M-types+of+streptococcus+pyogenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fac200575e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac200575e%26sid%3Dliteratum%253Aachs%26aulast%3DHamula%26aufirst%3DC.%2BL.%2BA.%26aulast%3DLe%26aufirst%3DX.%2BC.%26aulast%3DLi%26aufirst%3DX.-F.%26atitle%3DDNA%2520Aptamers%2520binding%2520to%2520multiple%2520prevalent%2520M-types%2520of%2520streptococcus%2520pyogenes%26jtitle%3DAnal.%2520Chem.%26date%3D2011%26volume%3D83%26issue%3D10%26spage%3D3640%26epage%3D3647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Joshi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janagama, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dwivedi, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S. T. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaykus, L.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schefers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreevatsan, S.</span><span> </span><span class="NLM_article-title">Selection, characterization, and application of DNA aptamers for the capture and detection of Salmonella enterica serovars</span> <span class="citation_source-journal">Mol. Cell. Probes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1016%2Fj.mcp.2008.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=19049862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1artrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=20-28&author=R.+Joshiauthor=H.+Janagamaauthor=H.+P.+Dwivediauthor=S.+T.+M.+A.+Kumarauthor=L.-A.+Jaykusauthor=J.+Schefersauthor=S.+Sreevatsan&title=Selection%2C+characterization%2C+and+application+of+DNA+aptamers+for+the+capture+and+detection+of+Salmonella+enterica+serovars"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selection, characterization, and application of DNA aptamers for the capture and detection of Salmonella enterica serovars</span></div><div class="casAuthors">Joshi, Raghavendra; Janagama, Harish; Dwivedi, Hari P.; Senthil Kumar, T. M. A.; Jaykus, Lee-Ann; Schefers, Jeremy; Sreevatsan, Srinand</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Probes</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-28</span>CODEN:
                <span class="NLM_cas:coden">MCPRE6</span>;
        ISSN:<span class="NLM_cas:issn">0890-8508</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sensitive and specific pre-anal. sample processing methods are needed to enhance our ability to detect and quantify food borne pathogens from complex food and environmental samples.  In this study, DNA aptamers were selected and evaluated for the capture and detection of Salmonella enterica Typhimurium.  A total of 66 candidate sequences were enriched against S. Typhimurium outer membrane proteins (OMPs) with counter-selection against Escherichia coli OMPs and lipopolysaccharides (LPS).  Specificity of the selected aptamers was evaluated by gel-shift anal. against S. Typhimurium OMP.  Five Salmonella-specific aptamer candidates were selected for further characterization.  A diln.-to-extinction capture protocol using pure cultures of S. Typhimurium further narrowed the field to two candidates (aptamers 33 and 45) which showed low-end detection limits of 10-40 CFU.  DNase protection assays applied to these aptamers confirmed sequence-specific binding to S. Typhimurium OMP prepns., while South-Western blot anal. combined with mass spectrometry identified putative membrane proteins as targets for aptamer binding.  Aptamer 33 was bound to magnetic beads and used for the capture of S. Typhimurium seeded into whole carcass chicken rinse samples, followed by detection using quant. real-time RT-PCR.  In a pull-down assay format, detection limits were 101-102 CFU S. Typhimurium/9 mL rinsate, while in a recirculation format, detection limits were 102-103 CFU/25 mL rinsate.  Reproducible detection at <101 S. typhimurium CFU/g was also achieved in spike-and-recovery expts. using bovine feces.  The pull-down anal. using aptamer 33 was validated on 3 naturally infected chicken litter samples confirming their applicability in the field.  This study demonstrates the applicability of Salmonella specific aptamers for pre-anal. sample processing as applied to food and environmental sample matrixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_O5V8LIZefrVg90H21EOLACvtfcHk0lhD6znZXnNd7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1artrs%253D&md5=c5356fbcd4ecb7965209741e550478d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.mcp.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mcp.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DR.%26aulast%3DJanagama%26aufirst%3DH.%26aulast%3DDwivedi%26aufirst%3DH.%2BP.%26aulast%3DKumar%26aufirst%3DS.%2BT.%2BM.%2BA.%26aulast%3DJaykus%26aufirst%3DL.-A.%26aulast%3DSchefers%26aufirst%3DJ.%26aulast%3DSreevatsan%26aufirst%3DS.%26atitle%3DSelection%252C%2520characterization%252C%2520and%2520application%2520of%2520DNA%2520aptamers%2520for%2520the%2520capture%2520and%2520detection%2520of%2520Salmonella%2520enterica%2520serovars%26jtitle%3DMol.%2520Cell.%2520Probes%26date%3D2009%26volume%3D23%26spage%3D20%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Fubin, Wang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming-Sheng, Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian-Ming, Hu</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bing Xia Qin, Pan</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao-Lian, Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong-Yan, Wu</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan-Wen, He</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jianguo, Wu.</span><span> </span><span class="NLM_article-title">Aptamers that preferentially bind type IVB pili and inhibit human monocytic-cell invasion by Salmonella enterica serovar typhi</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">4052</span><span class="NLM_x">–</span> <span class="NLM_lpage">4060</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=4052-4060&issue=10&author=Wang+Fubinauthor=Zhang+Ming-Shengauthor=Hu+Jian-Mingauthor=Pan+Bing+Xia+Qinauthor=Zhang+Xiao-Lianauthor=Wu+Hong-Yanauthor=He+Pan-Wenauthor=Wu.+Jianguo&title=Aptamers+that+preferentially+bind+type+IVB+pili+and+inhibit+human+monocytic-cell+invasion+by+Salmonella+enterica+serovar+typhi"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFubin%26aufirst%3DWang%26aulast%3DMing-Sheng%26aufirst%3DZhang%26aulast%3DJian-Ming%26aufirst%3DHu%26aulast%3DBing%2BXia%2BQin%26aufirst%3DPan%26aulast%3DXiao-Lian%26aufirst%3DZhang%26aulast%3DHong-Yan%26aufirst%3DWu%26aulast%3DPan-Wen%26aufirst%3DHe%26aulast%3DJianguo%26aufirst%3DWu.%26atitle%3DAptamers%2520that%2520preferentially%2520bind%2520type%2520IVB%2520pili%2520and%2520inhibit%2520human%2520monocytic-cell%2520invasion%2520by%2520Salmonella%2520enterica%2520serovar%2520typhi%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26issue%3D10%26spage%3D4052%26epage%3D4060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Sefah, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shangguan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Donoghue, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, W.</span><span> </span><span class="NLM_article-title">Development of DNA aptamers using Cell-SELEX</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1169</span><span class="NLM_x">–</span> <span class="NLM_lpage">1185</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1038%2Fnprot.2010.66" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=20539292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFSkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1169-1185&author=K.+Sefahauthor=D.+Shangguanauthor=X.+Xiongauthor=M.+B.+O%E2%80%99Donoghueauthor=W.+Tan&title=Development+of+DNA+aptamers+using+Cell-SELEX"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Development of DNA aptamers using Cell-SELEX</span></div><div class="casAuthors">Sefah, Kwame; Shangguan, Dihua; Xiong, Xiangling; O'Donoghue, Meghan B.; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1185</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past two decades, high-affinity nucleic acid aptamers have been developed for a wide variety of pure mols. and complex systems such as live cells.  Conceptually, aptamers are developed by an evolutionary process, whereby, as selection progresses, sequences with a certain conformation capable of binding to the target of interest emerge and dominate the pool.  This protocol, cell-SELEX (systematic evolution of ligands by exponential enrichment), is a method that can generate DNA aptamers that can bind specifically to a cell type of interest.  Commonly, a cancer cell line is used as the target to generate aptamers that can differentiate that cell type from other cancers or normal cells.  A single-stranded DNA (ssDNA) library pool is incubated with the target cells.  Nonbinding sequences are washed off and bound sequences are recovered from the cells by heating cell-DNA complexes at 95 °C, followed by centrifugation.  The recovered pool is incubated with the control cell line to filter out the sequences that bind to common mols. on both the target and the control, leading to the enrichment of specific binders to the target.  Binding sequences are amplified by PCR using fluorescein isothiocyanate-labeled sense and biotin-labeled antisense primers.  This is followed by removal of antisense strands to generate an ssDNA pool for subsequent rounds of selection.  The enrichment of the selected pools is monitored by flow cytometry binding assays, with selected pools having increased fluorescence compared with the unselected DNA library.  The procedure, from design of oligonucleotides to enrichment of the selected pools, takes ∼ 3 mo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtFzU80-TsLbVg90H21EOLACvtfcHk0lhD6znZXnNd7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFSkt74%253D&md5=396e01ca2ce0530ace68b12bbf8975a4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2010.66&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2010.66%26sid%3Dliteratum%253Aachs%26aulast%3DSefah%26aufirst%3DK.%26aulast%3DShangguan%26aufirst%3DD.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DM.%2BB.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520DNA%2520aptamers%2520using%2520Cell-SELEX%26jtitle%3DNat.%2520Protoc.%26date%3D2010%26volume%3D5%26spage%3D1169%26epage%3D1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Miles, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misra, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. O.</span><span> </span><span class="NLM_article-title">The estimation of the bactericidal power of the blood</span> <span class="citation_source-journal">J. Hyg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1938</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">732</span><span class="NLM_x">–</span> <span class="NLM_lpage">749</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm301856j&amp;key=10.1017%2FS002217240001158X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm301856j&amp;key=20475467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm301856j&amp;key=1%3ACAS%3A528%3ADyaA1MXhtFGnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1938&pages=732-749&issue=6&author=A.+A.+Milesauthor=S.+S.+Misraauthor=J.+O.+Irwin&title=The+estimation+of+the+bactericidal+power+of+the+blood"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The estimation of the bactericidal power of the blood</span></div><div class="casAuthors">Miles, A. A.; Misra, S. S.; Irwin, J. O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hygiene</span>
        (<span class="NLM_cas:date">1938</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">732-49</span>CODEN:
                <span class="NLM_cas:coden">JOHYAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1724</span>.
    </div><div class="casAbstract">The survival rate (p) of a measured inoculum of Staphylococcus aureus in a standard vol. of defibrinated blood is a reliable quant. measure of the bactericidal power of blood.  The immunological significance of p has not been investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtsaEeWDTKkrVg90H21EOLACvtfcHk0lignACVs1KNOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA1MXhtFGnug%253D%253D&md5=1c91fe942e3e37d0be154c340ae862a5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1017%2FS002217240001158X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS002217240001158X%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DA.%2BA.%26aulast%3DMisra%26aufirst%3DS.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BO.%26atitle%3DThe%2520estimation%2520of%2520the%2520bactericidal%2520power%2520of%2520the%2520blood%26jtitle%3DJ.%2520Hyg.%26date%3D1938%26volume%3D38%26issue%3D6%26spage%3D732%26epage%3D749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Lopez-Amoros, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vives-Rego, J.</span><span> </span><span class="NLM_article-title">Flow cytometric assessment of Escherichia coli and Salmonella typhimurium starvation-survival in seawater using rhodamine 123, propidium iodide, and oxonol</span> <span class="citation_source-journal">Appl. Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2521</span><span class="NLM_x">–</span> <span class="NLM_lpage">2526</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1995&pages=2521-2526&issue=7&author=R.+Lopez-Amorosauthor=J.+Comasauthor=J.+Vives-Rego&title=Flow+cytometric+assessment+of+Escherichia+coli+and+Salmonella+typhimurium+starvation-survival+in+seawater+using+rhodamine+123%2C+propidium+iodide%2C+and+oxonol"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Amoros%26aufirst%3DR.%26aulast%3DComas%26aufirst%3DJ.%26aulast%3DVives-Rego%26aufirst%3DJ.%26atitle%3DFlow%2520cytometric%2520assessment%2520of%2520Escherichia%2520coli%2520and%2520Salmonella%2520typhimurium%2520starvation-survival%2520in%2520seawater%2520using%2520rhodamine%2520123%252C%2520propidium%2520iodide%252C%2520and%2520oxonol%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D1995%26volume%3D61%26issue%3D7%26spage%3D2521%26epage%3D2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kennedy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, U. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, M. G.</span><span> </span><span class="NLM_article-title">Responses of Escherichia coli, Listeria monocytogenes, and Staphylococcus aureus to simulated food processing treatments, determined using Fluorescence-Activated Cell Sorting and plate counting</span> <span class="citation_source-journal">Appl. Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">4657</span><span class="NLM_x">–</span> <span class="NLM_lpage">4668</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=4657-4668&issue=13&author=D.+Kennedyauthor=U.+P.+Croninauthor=M.+G.+Wilkinson&title=Responses+of+Escherichia+coli%2C+Listeria+monocytogenes%2C+and+Staphylococcus+aureus+to+simulated+food+processing+treatments%2C+determined+using+Fluorescence-Activated+Cell+Sorting+and+plate+counting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DD.%26aulast%3DCronin%26aufirst%3DU.%2BP.%26aulast%3DWilkinson%26aufirst%3DM.%2BG.%26atitle%3DResponses%2520of%2520Escherichia%2520coli%252C%2520Listeria%2520monocytogenes%252C%2520and%2520Staphylococcus%2520aureus%2520to%2520simulated%2520food%2520processing%2520treatments%252C%2520determined%2520using%2520Fluorescence-Activated%2520Cell%2520Sorting%2520and%2520plate%2520counting%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26issue%3D13%26spage%3D4657%26epage%3D4668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i12"><a href="/doi/suppl/10.1021/jm301856j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_15224"></div></div></div></div></div><hr /></hr><p class="last">Dose–response curves for the aptamers with bacteriostatic effect; example of <i>K</i><sub>d</sub> determination; competition experiments between aptamer clones. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm301856j/suppl_file/jm301856j_si_001.pdf">jm301856j_si_001.pdf (1.0 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm301856j&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-4%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Fjm301856j" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm301856j" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679960983f793bec","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
