
<html lang="en"     class="pb-page"  data-request-id="7fbd666a-fd66-4f3e-9db1-f724343f6be6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-24;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01411;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Jonathan D.  Macdonald" /></meta><meta name="dc.Creator" content="Selena  Chacón Simon" /></meta><meta name="dc.Creator" content="Changho  Han" /></meta><meta name="dc.Creator" content="Feng  Wang" /></meta><meta name="dc.Creator" content="J. Grace  Shaw" /></meta><meta name="dc.Creator" content="Jennifer E.  Howes" /></meta><meta name="dc.Creator" content="Jiqing  Sai" /></meta><meta name="dc.Creator" content="Joannes P.  Yuh" /></meta><meta name="dc.Creator" content="Demarco  Camper" /></meta><meta name="dc.Creator" content="Bethany M.  Alicie" /></meta><meta name="dc.Creator" content="Joseph  Alvarado" /></meta><meta name="dc.Creator" content="Sameer  Nikhar" /></meta><meta name="dc.Creator" content="William  Payne" /></meta><meta name="dc.Creator" content="Erin R.  Aho" /></meta><meta name="dc.Creator" content="Joshua A.  Bauer" /></meta><meta name="dc.Creator" content="Bin  Zhao" /></meta><meta name="dc.Creator" content="Jason  Phan" /></meta><meta name="dc.Creator" content="Lance R.  Thomas" /></meta><meta name="dc.Creator" content="Olivia W.  Rossanese" /></meta><meta name="dc.Creator" content="William P.  Tansey" /></meta><meta name="dc.Creator" content="Alex G.  Waterson" /></meta><meta name="dc.Creator" content="Shaun R.  Stauffer" /></meta><meta name="dc.Creator" content="Stephen W.  Fesik" /></meta><meta name="dc.Description" content="The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery. Here, we present a new strategy toward the inhibition of MYC vi..." /></meta><meta name="Description" content="The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery. Here, we present a new strategy toward the inhibition of MYC vi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 14, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01411" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01411" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01411" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01411" /></link>
        
    
    

<title>Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01411" /></meta><meta property="og:title" content="Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0006.jpeg" /></meta><meta property="og:description" content="The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery. Here, we present a new strategy toward the inhibition of MYC via the disruption of the protein–protein interaction between MYC and its chromatin cofactor WD Repeat-Containing Protein 5. Blocking the association of these proteins is hypothesized to disrupt the localization of MYC to chromatin, thus disrupting the ability of MYC to sustain tumorigenesis. Utilizing a high-throughput screening campaign and subsequent structure-guided design, we identify small-molecule inhibitors of this interaction with potent in vitro binding affinity and report structurally related negative controls that can be used to study the effect of this disruption. Our work suggests that disruption of this protein–protein interaction may provide a path toward an effective approach for the treatment of multiple tumors and anticipate that the molecules disclosed can be used as starting points for future efforts toward compounds with improved drug-like properties." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01411"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01411">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01411&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01411&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01411&amp;href=/doi/10.1021/acs.jmedchem.9b01411" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11232-11259</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01393" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01489" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jonathan D. Macdonald</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan D. Macdonald</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+D.++Macdonald">Jonathan D. Macdonald</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6360-2768" title="Orcid link">http://orcid.org/0000-0002-6360-2768</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Selena Chacón Simon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Selena Chacón Simon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Selena++Chac%C3%B3n+Simon">Selena Chacón Simon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Changho Han</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Changho Han</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Changho++Han">Changho Han</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Feng Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Feng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Wang">Feng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4322-2127" title="Orcid link">http://orcid.org/0000-0002-4322-2127</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">J. Grace Shaw</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Grace Shaw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Grace++Shaw">J. Grace Shaw</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer E. Howes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer E. Howes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+E.++Howes">Jennifer E. Howes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiqing Sai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiqing Sai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiqing++Sai">Jiqing Sai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joannes P. Yuh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joannes P. Yuh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joannes+P.++Yuh">Joannes P. Yuh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Demarco Camper</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Demarco Camper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Demarco++Camper">Demarco Camper</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bethany M. Alicie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bethany M. Alicie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bethany+M.++Alicie">Bethany M. Alicie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph Alvarado</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph Alvarado</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph++Alvarado">Joseph Alvarado</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sameer Nikhar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sameer Nikhar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sameer++Nikhar">Sameer Nikhar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William Payne</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William Payne</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++Payne">William Payne</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Erin R. Aho</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Erin R. Aho</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Erin+R.++Aho">Erin R. Aho</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joshua A. Bauer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joshua A. Bauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joshua+A.++Bauer">Joshua A. Bauer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bin Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Zhao">Bin Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jason Phan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jason Phan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Phan">Jason Phan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lance R. Thomas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lance R. Thomas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lance+R.++Thomas">Lance R. Thomas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olivia W. Rossanese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olivia W. Rossanese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olivia+W.++Rossanese">Olivia W. Rossanese</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William P. Tansey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William P. Tansey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+P.++Tansey">William P. Tansey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex G. Waterson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex G. Waterson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex+G.++Waterson">Alex G. Waterson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0153-5723" title="Orcid link">http://orcid.org/0000-0002-0153-5723</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shaun R. Stauffer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaun R. Stauffer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaun+R.++Stauffer">Shaun R. Stauffer</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Stephen W. Fesik</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen W. Fesik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry,  Department of Pharmacology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ec9f98899c848982c28a899f8587ac9a8d8288899e8e858098c2898899"><span class="__cf_email__" data-cfemail="592a2d3c29313c37773f3c2a3032192f38373d3c2b3b30352d773c3d2c">[email protected]</span></a>. Tel.: +1 (615) 322-6303.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+W.++Fesik">Stephen W. Fesik</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5957-6192" title="Orcid link">http://orcid.org/0000-0001-5957-6192</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01411&amp;href=/doi/10.1021%2Facs.jmedchem.9b01411" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11232–11259</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 14, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 August 2019</li><li><span class="item_label"><b>Published</b> online</span>14 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01411" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01411</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11232%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJonathan%2BD.%2BMacdonald%252C%2BSelena%2BChac%25C3%25B3n%2BSimon%252C%2BChangho%2BHan%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D24%26contentID%3Dacs.jmedchem.9b01411%26title%3DDiscovery%2Band%2BOptimization%2Bof%2BSalicylic%2BAcid-Derived%2BSulfonamide%2BInhibitors%2Bof%2Bthe%2BWD%2BRepeat-Containing%2BProtein%2B5%25E2%2580%2593MYC%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11259%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01411"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2108</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01411" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;D. Macdonald&quot;},{&quot;first_name&quot;:&quot;Selena&quot;,&quot;last_name&quot;:&quot;Chacón Simon&quot;},{&quot;first_name&quot;:&quot;Changho&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Grace Shaw&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;E. Howes&quot;},{&quot;first_name&quot;:&quot;Jiqing&quot;,&quot;last_name&quot;:&quot;Sai&quot;},{&quot;first_name&quot;:&quot;Joannes&quot;,&quot;last_name&quot;:&quot;P. Yuh&quot;},{&quot;first_name&quot;:&quot;Demarco&quot;,&quot;last_name&quot;:&quot;Camper&quot;},{&quot;first_name&quot;:&quot;Bethany&quot;,&quot;last_name&quot;:&quot;M. Alicie&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;Alvarado&quot;},{&quot;first_name&quot;:&quot;Sameer&quot;,&quot;last_name&quot;:&quot;Nikhar&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;Payne&quot;},{&quot;first_name&quot;:&quot;Erin&quot;,&quot;last_name&quot;:&quot;R. Aho&quot;},{&quot;first_name&quot;:&quot;Joshua&quot;,&quot;last_name&quot;:&quot;A. Bauer&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Phan&quot;},{&quot;first_name&quot;:&quot;Lance&quot;,&quot;last_name&quot;:&quot;R. Thomas&quot;},{&quot;first_name&quot;:&quot;Olivia&quot;,&quot;last_name&quot;:&quot;W. Rossanese&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;P. Tansey&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;G. Waterson&quot;},{&quot;first_name&quot;:&quot;Shaun&quot;,&quot;last_name&quot;:&quot;R. Stauffer&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;W. Fesik&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;11232-11259&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01411&quot;},&quot;abstract&quot;:&quot;The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery. Here, we present a new strategy toward the inhibition of MYC via the disruption of the protein–protein interaction between MYC and its chromatin cofactor WD Repeat-Containing Protein 5. Blocking the association of these proteins is hypothesized to disrupt the localization of MYC to chromatin, thus disrupting the ability of MYC to sustain tumorigenesis. Utilizing a high-throughput screening campaign and subsequent structure-guided design, we identify small-molecule inhibitors of this interaction with potent in vitro binding affinity and report structurally related negative controls that can be used to study the effect of this disruption. Our work suggests that disruption of this protein–protein interaction may provide a path toward an effective approach for the treatment of multiple tumors and anticipate that the molecules disclosed can be used as starting points for future effo&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01411&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01411" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01411&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01411" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01411&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01411" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01411&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01411&amp;href=/doi/10.1021/acs.jmedchem.9b01411" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01411" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01411" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01411%26sid%3Dliteratum%253Aachs%26pmid%3D31724864%26genre%3Darticle%26aulast%3DMacdonald%26date%3D2019%26atitle%3DDiscovery%2Band%2BOptimization%2Bof%2BSalicylic%2BAcid-Derived%2BSulfonamide%2BInhibitors%2Bof%2Bthe%2BWD%2BRepeat-Containing%2BProtein%2B5%25E2%2580%2593MYC%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D24%26spage%3D11232%26epage%3D11259%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=290716" title="Sulfones">Sulfones</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/jmcmar.2019.62.issue-24/20191226/jmcmar.2019.62.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The treatment of tumors driven by overexpression or amplification of MYC oncogenes remains a significant challenge in drug discovery. Here, we present a new strategy toward the inhibition of MYC via the disruption of the protein–protein interaction between MYC and its chromatin cofactor WD Repeat-Containing Protein 5. Blocking the association of these proteins is hypothesized to disrupt the localization of MYC to chromatin, thus disrupting the ability of MYC to sustain tumorigenesis. Utilizing a high-throughput screening campaign and subsequent structure-guided design, we identify small-molecule inhibitors of this interaction with potent in vitro binding affinity and report structurally related negative controls that can be used to study the effect of this disruption. Our work suggests that disruption of this protein–protein interaction may provide a path toward an effective approach for the treatment of multiple tumors and anticipate that the molecules disclosed can be used as starting points for future efforts toward compounds with improved drug-like properties.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48813" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48813" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">MYC oncogenes encode a family of related transcription factors (C-, L-, and N-MYC) that are overexpressed in a significant number of malignancies.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The oncogenic effects of these MYC proteins are caused by their action as sequence-specific DNA binding transcription factors that drive the expression of genes required for cell growth, proliferation, metabolism, genome instability, and apoptosis.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> MYC proteins are intrinsically disordered, with a minimal secondary structure; as such, they lack traditional binding sites for direct inhibition and have been considered to be “undruggable”.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Despite significant effort and investment toward the identification of compounds that can directly affect MYC function, few successes have been reported to date.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> New approaches to MYC-targeted therapies may focus on the inhibition of the protein–protein interactions (PPIs) of MYC with critical partner proteins.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> For example, much work on the MYC-MAX heterodimer has been reported,<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12">(8−12)</a> and the direct interactions of MYC with other proteins, such as Aurora A, have recently garnered interest.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> We have previously reported on the identification of WD repeat-containing protein 5 (WDR5) as a crucial partner in the facilitated recruitment of MYC to chromatin,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> thus making it a critical cofactor for MYC-driven tumorigenesis. Further, we have defined the WDR5–MYC interaction site as a potentially tractable target for small molecule inhibition of MYC-driven tumors.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a></div><div class="NLM_p">WD repeat proteins are a ubiquitous family of scaffolding proteins, containing β-propeller domains that form donut-shaped structures, which participate in many multiprotein complexes.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> In yeast, WD repeats are reported to participate in more interactions than any other protein domain.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The direct targeting of these multiprotein complexes makes the WD repeat family an interesting, although clinically unvalidated, target class in drug discovery.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> WDR5 is one such regulatory scaffolding protein that has reported involvement in protein complexes related to the chromatin structure and epigenetic machinery.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The protein has two major binding sites, previously referred to as the <u class="uu">W</u>DR5 <u class="uu">B</u>inding <u class="uu">M</u>otif (WBM) and <u class="uu">W</u>DR5 <u class="uu">In</u>teraction Motif (WIN) sites (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B, respectively), which have each been demonstrated to be key interfaces for its role in complex formation.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0020.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overlay of WDR5 bound to MYC and MLL peptides (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R">4Y7R</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6">3EG6</a>, respectively). (A) Visualization of the MYC MbIIIb peptide bound to a cleft of WDR5, referred to as the “WBM” binding site. (B) MLL1 peptide binds on the reverse face of WDR5 with an arginine residue deep into the “WIN” site binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We previously described that the association of WDR5 with MYC occurs at the WBM site and confirmed by coimmunoprecipitation that WDR5 binds to the central portion of MYC via the evolutionarily conserved “MYC box” MbIIIb motif. The binding surface is a shallow, hydrophobic cleft containing multiple basic residues, which binds to a 10-amino acid chain containing an “EEIDVV” motif that is conserved in all extant MYC proteins. This same binding cleft has been shown to also bind other proteins; those reported to date include RBBP5 in the MLL1/SET complex<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> and KANSL2, a protein critical for the histone acetyltransferase (HAT) activity of the NSL complex.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> These proteins (MYC, RBBP5, and KANSL2) contain a highly similar binding motif (IDVV, LDVV, and VDVT, respectively) that engage WDR5 in a structurally homologous way.</div><div class="NLM_p">On the opposite face of the protein is the WIN binding site, which is well studied and contains a deep binding pocket that binds mixed lineage leukemia 1 (MLL1) via a conserved arginine-containing motif known as the WIN motif.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> The ability of MLL1 to act as a histone methyltransferase (HMT) and catalytically methylate histone H3 lysine 4 (H3K4) has been demonstrated to be reliant on the formation of a four-protein complex, including <u class="uu">W</u>DR5, <u class="uu">R</u>BBP5, <u class="uu">A</u>SH2L, and <u class="uu">D</u>PY30 (WRAD complex).<a onclick="showRef(event, 'ref26 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref26 ref29 ref30">(26,29,30)</a> Multiple small molecules that disrupt this complex by binding to WDR5 at the WIN binding site have now been published, including both peptidomimetics<a onclick="showRef(event, 'ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33">(31−33)</a> and small-molecule inhibitors.<a onclick="showRef(event, 'ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36 ref37 ref38">(34−38)</a></div><div class="NLM_p">Due to the tertiary structure of the protein, targeting the WIN site is expected to be much more straightforward than discovering potent small molecules against the WBM site. It has been demonstrated that targeting the central cavity of WD repeat proteins leads to inhibitors that have significantly higher potency than compounds that occupy cavities on the outer surfaces,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> highlighting the challenge presented in targeting this MYC binding site. For example, CDC20 and Cdc4 are WD repeat proteins for which inhibitors are reported that bind at such “external” sites with affinities of >10 and 6.2 μM, respectively.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> Interestingly, the Cdc4 inhibitor, SCF-12, is reported to occlude substrate binding into the central cavity through a long-range allosteric effect.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> To date, however, there are no reports of a small-molecule probe that directly binds to the WBM cleft used by MYC and RBBP5.</div><div class="NLM_p last">Here, we describe the discovery of the small molecules that bind to the WBM binding cleft. Utilizing structure-based design approaches based upon our previously published structure of WDR5, we have improved the affinity of the screening hits. Further, we characterize the utility of the lead compounds for their ability to disrupt binding of WDR5 to MYC and RBBP5 and demonstrate that they are capable of functional inhibition of the WRAD complex.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35066" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35066" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification and Validation of Hits</h3><div class="NLM_p">A high-throughput screen (HTS) was performed using a competition fluorescence polarization (FP)-based assay to assess interruption of the binding of an FITC-labeled peptide probe derived from the MYC MbIII protein sequence to WDR5 (a.a. 23–334).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Compounds from the Vanderbilt Discovery and VICB collections (∼250,000 total) were screened at 50 μM, and 410 preliminary “hit” compounds that demonstrated greater than 15% inhibition of the interaction were identified (a hit rate of 0.16%). After duplicate screening, 110 compounds were confirmed as hits; each of these was subsequently evaluated using a 10-point dose–response protocol to calculate binding affinity from fresh DMSO compound dilutions. Seventy-six compounds afforded a measurable <i>K</i><sub>d</sub> value and were taken forward as MYC site hits (0.03% confirmed hit rate).</div><div class="NLM_p">As the HTS was based upon a competition assay, we also validated the hits in an orthogonal biophysical assay, collecting SOFAST <sup>1</sup>H–<sup>15</sup>N heteronuclear multiple-quantum coherence (HMQC) spectra of WDR5 to determine direct binding of the compounds. Peak shifts that correspond to binding at the WBM binding site were identified by obtaining an HMQC spectrum of uniformly <sup>15</sup>N-labeled WDR5 with the unlabeled MYC-derived peptide. Identification of this chemical shift perturbation (CSP) pattern allowed us to monitor the compound binding to this site by NMR. Screening hits that did not cause any CSP to the NMR spectra of WDR5 were removed from consideration for follow-up chemistry. Validated hits induced either peak shift (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) or peak broadening (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). These findings suggest that different HTS hits presumably exhibited an intermediate and fast exchange based upon the NMR chemical shift timescale. These exchange regimes suggest <i>K</i><sub>d</sub> values in the single-digit micromolar (intermediate exchange) or high micromolar (fast exchange) range consistent with the primary FP assay, ranging from 8 to 175 μM. Multiple confirmed hits contain a biaryl sulfonamide motif, with many of the examples also containing an acidic motif that may interact with the charged residues that surround the MYC binding site of WDR5 (exemplar hits shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0019.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example <sup>1</sup>H–<sup>15</sup>N SOFAST-HMQC spectra showing the observable effect of compounds that (A) induce peak shift and (B) induce peak broadening. Apo-WDR5 (blue); Hit-WDR5 (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative hits that bind to WDR5 at the WBM site. <i>K</i><sub>d</sub> values obtained by a fluorescence polarization assay utilizing an FITC-labeled peptide based upon the MbIIIb sequence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> X-Ray Cocrystal Structure of WDR5 Bound with a WBM Site Hit</h3><div class="NLM_p">We obtained an X-ray cocrystal structure of the leading hit, <b>1</b>, bound to WDR5 by cocrystallization. The overlay of this structure with those of published peptides confirmed that the small molecule bound into the WBM site that mediates the binding of WDR5 to MYC and RBBP5 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The chlorine atom on the aniline ring is well positioned to mimic the hydrophobic side chain of the isoleucine of the IDVV motif, while the methoxy group provides a potential vector for compound growth. Key interactions responsible for compound binding include (1) a hydrogen bond from an oxygen of the sulfonamide to the backbone NH of ASN225 and (2) a halogen bonding interaction from the aromatic bromine to the carbonyl of TRP273 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). These interactions appear to cause a shift of the ASN225 containing loop toward the ligand.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0018.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overlay of hit compound <b>1</b> with the structure of bound MYC-derived MbIIIb peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R">4Y7R</a>), demonstrating the frameshift of the loop enabling retainment of the H-bond interaction with the backbone; WDR5–<b>1</b> complex (green) and WDR5–peptide complex (gray). (B) Zoom of the binding site of the cocrystal structure of WDR5 and <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6U6W">6U6W</a>). Shown as dashes are the halogen bonding interaction between the bromine atom and Trp273 and the hydrogen bond between the sulfonamide oxygen and Asn225; key amino acid residues are shown as sticks and are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Early SAR Studies</h3><div class="NLM_p">After confirmation of the hit compounds, the focus of our initial synthetic efforts was to investigate the two halogen substituents, the aniline ring chlorine and the sulfonyl ring bromide. The bromine atom forms a key halogen bonding interaction in the X-ray structure. Indeed, when this bromine atom is removed (<b>5a</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), all binding affinity for WDR5 is lost, while replacement with either a chlorine or an iodide both affords a compound with reduced <i>K</i><sub>d</sub> (<b>5b</b> and <b>5c</b>). Interestingly, removing any one of the four substituents that decorate the biaryl sulfonamide substructure was proved to be highly deleterious (<b>5d</b> and <b>5e</b>), demonstrating that this substitution pattern shows good shape complementarity for this binding surface. Similarly, we observed a consistent, significant improvement in affinity when switching from a methoxy R3 group to a phenol (<b>5f</b>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Early SAR of HTS Hit, <b>1</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0009.gif" alt="" id="gr14" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0004.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data displayed is the average of at least three independent replicates ± standard deviation.</p></div></div><div></div></div><div class="NLM_p">Early in the project, we observed compounds that were able to bind below the lower limit of detection of the HTS FP assay that used the FITC-labeled MYC peptide (to <i>K</i><sub>d</sub> < 2.5 μM). The data for hits shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> corresponds to this peptide-based probe used throughout the screening campaign. We later designed and synthesized fluorescein-labeled small-molecule probes with higher affinity for WDR5 compared to the peptide and thus offered lower limits of detection in the FP assay and correspondingly our interpretation of Structure Activity Relationship (SAR). For clarity, data shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<named-content content-type="anchor" rid="tbl2" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> utilize only a single generation of small-molecule fluorescein probe (see the structure in Supplemental Figure S1); hence, the difference in the <i>K</i><sub>d</sub> value of <b>1</b> is reported in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, from the high-throughput screening campaign, and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of Salicylic Acid Derivatives, <b>6</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0008.gif" alt="" id="gr16" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0002.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data displayed is the average of at least three independent replicates ± standard deviation.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Compound contains a <i>para</i>-methoxy bromide on the sulfonyl ring.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and Biochemical Data for Salicylamide-Derived WDR5 WBM Site Inhibitors<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0007.gif" alt="" id="gr18" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0003.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Data displayed is the average of at least three independent replicates ± standard deviation.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Data is the average of two independent replicates.</p></div></div><div></div></div><div class="NLM_p">On the aniline ring, a chlorine atom occupies the same region as the isoleucine of the native IDVV motif, and as such, the chlorine was substituted for a range of hydrophobic groups that were hypothesized to more optimally occupy this region of the protein. Replacement of the halogen with phenyl was unfavorable and resulted in a loss of binding affinity (<b>5g</b>), while the <i>tert-</i>butyl analogue (<b>5h</b>) has similar affinity to the parent chlorine. X-ray crystallography demonstrated that larger groups tended to sit in the same space in the binding pocket but doing so forces the rest of the molecule to shift upward (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A); smaller aliphatic groups complement this pocket much better (<b>5i</b> and <b>5j</b>). Pentafluorosulfanyl groups have been reported as metabolically inert <i>tert-</i>butyl analogues,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and their use herein affords the most potent example of this phenolic class, <b>5k</b>, with ∼5-fold improved binding affinity in comparison to the trifluoromethyl cyclopropane, <b>5l</b>.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0017.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-Ray cocrystallography of WDR5 bound to WBM site inhibitors. (A) Overlay of the X-Ray cocrystal structure of <b>5g</b> (green; PDB: 6U80) and <b>1</b> (gray). There are few major changes in the protein orientation, but <b>5g</b> showed diminished binding affinity compared to <b>1</b>, suggesting that the upward frameshift affords an unfavorable position of the small-molecule ligand. (B) Structure of <b>6b</b> bound to WDR5 (PDB: 6U8B). Key residues are shown as sticks, highlighting hydrogen and halogen bonding interactions between the ligand and protein. (C) Structure of <b>5n</b> bound to WDR5, demonstrating the orientation of the nitrile toward the protein (PDB: 6U8O). (D) Structure of <b>6g</b> bound to WDR5 (PDB: 6U8L). (E) Structure of <b>7k</b> bound to WDR5 (PDB: 6U5Y). (F) Structure of <b>7u</b> bound to WDR5 (PDB: 6U5M).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The cyano-spirocyclic examples (<b>5m</b>–<b>5p</b>) were initially designed to act as a handle for further growth of the compounds along the edge of this pocket, with the hydrocarbon chain hypothesized, using molecular docking studies, to complement the hydrophobic region. However, we observed by X-ray crystallography that the cyano moiety of <b>5n</b> is instead oriented toward the protein, albeit with no obvious directional interaction (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The three- and five-membered rings, for example, <b>5m</b> and <b>5o</b>, have slightly diminished affinity in comparison to the four- and six-membered rings. We hypothesize that the shape of C3- and C5-carbocycles produces a less favorable orientation of the nitrile, although we do not have crystallographic evidence of such an altered binding pose.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Optimization to Salicylic Acid Motif</h3><div class="NLM_p">The X-ray cocrystal structure of <b>1</b> allowed us to employ structure-based design for the development of further optimized compounds, although a significant challenge to this effort relates to the shallow nature of this binding interface, which contains few areas that may traditionally be referred to as a “pocket”.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> As well as the early optimization of the hydrophobic moiety in the “ILE” region of the peptide binding site, a further breakthrough came from the addition of a carboxylic acid directed toward Q289, affording salicylic acid derivative <b>6b</b> (<i>K</i><sub>d</sub> = 0.43 μM, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This engagement of Q289 afforded an increase in binding affinity compared to <b>1</b>, and the positive interaction was confirmed by X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Similarly, a significant potency increase was obtained by the addition of a chlorine <i>ortho</i> to the phenol of  the sulfonyl ring (<b>6d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), gaining additional hydrophobic contacts with WDR5 in the interface occupied by a valine side chain of MYC and RBBP5 (IDVV).</div><div class="NLM_p last">The cyclopropyl derivatives (<b>6e</b> and <b>6f</b>) demonstrated an improvement versus the parent chloro-containing compound, and the addition of an <i>ortho</i>-fluoro group afforded a further increase (<b>6g</b>). This enhanced affinity could be explained by the good shape complementarity of this compound to the binding site as demonstrated crystallographically (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). In line with the SAR trends of <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the hydrophobic chloro-replacement of trifluoromethoxy, pentafluorosulfanyl, and cyano-spirocycle derivatives afforded low nanomolar potency with improvements of 5- to 15-fold upon the addition of the acid. Generally, we observed a significant decrease in binding affinity when removing the phenol of the salicylic acid moiety, <b>6a</b>, <b>6i</b>, and <b>6p</b>; however, this decrease was not observed for the −SF<sub>5</sub> analogues (<b>6l</b>, <b>6m</b>, <b>6n</b>, and <b>6o</b>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Salicylic Acid Replacement</h3><div class="NLM_p">With this series of salicylic acid-based compounds, we were able to reach low nanomolar levels of potency. However, we suspected inherent issues related to permeability and plasma protein binding due to the presence of the carboxylic acid, limiting their use in the whole cell mechanism of action (MOA) studies. To avoid these problems, we explored the replacement of the carboxylic acid with a methylamide (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Within this salicylamide series, a reduction in potency compared to the direct acid analogues was observed; however, by combining optimal SAR observations in the ILE262 region, we were able to identify analogues, which retained binding affinities below 100 nM. The hydrophobic ring of the cyanocyclobutyl moiety (<b>7k</b>) adopts a similar binding pose as the cyclopropane of <b>6g</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D), bolstering our hypothesis of improved affinity when designing analogues that grow from the <i>ipso</i>-carbon. Interestingly, while in the salicylic acid series, the cyclohexyl derivatives <b>6s</b> and <b>6t</b> demonstrated high binding affinity, and this trend did not hold for the amide derivatives <b>7l</b> and <b>7m</b>.</div><div class="NLM_p last">During this exercise, we explored alternate amide groups, including <b>7n</b> and <b>7o</b>, but, with the exception of <b>7n</b>, we observed very flat SAR when exploring different amine partners (data not shown). We hypothesize that the amide group is directed toward solvent and does not make positive interactions with the protein, which led to the choice of this vector for attaching the small-molecule FITC probes used for the assay (structure of probe shown in Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Figure S1</a>). In a separate salicylate replacement effort, we were able to demonstrate that the acid/amide moiety could be exchanged with a sulfone; for example, methyl sulfone-containing compounds <b>7p</b>–<b>7z</b> were synthesized. Their design was driven in part to enhance physicochemical properties, which might overcome some of the limitations observed in the salicylic acid subseries. Incorporation of the best-in-class pieces led to compounds such as <b>7w</b> and <b>7z</b> that bind with high potency to WDR5 and are comparable to analogues in the acid and amide series. Indeed, examples lacking the aniline-ring phenol were superior to the corresponding amide (compare <b>7b</b> vs <b>7q</b> and <b>7e</b> vs <b>7t</b>). Cocrystallization of WDR5 with <b>7x</b> demonstrates that the sulfone also engages Q289 via a hydrogen bond (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). Similar to the amides, the methyl group is directed toward the bulk solvent offering a potential vector for future derivatization to tune the physicochemical properties.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Phenol Substitutions</h3><div class="NLM_p">While we were able to discover small molecules that bind to WDR5 with excellent affinity, considering the shallow nature of the binding site, we recognize that these molecules retain functionality, such as phenols, which would likely occlude their development. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> already detail the synthesis of selected compounds with the aniline-ring phenol removed. Separately, we explored several strategies to remove or replace the <i>para</i>-bromophenol of the sulfonyl ring. A number of phenol and bromine replacement strategies failed (not shown); however, we were able to show that using optimized aniline-ring groups “capping” the phenol as a methoxy (e.g., <b>8</b> and <b>9</b>) can lead to inhibitors with submicromolar affinity in selected cases (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Further, combining the best-in-class aniline groups to generate 6-bromoquinoline-8-sulfonamides afforded nanomolar analogues that do not contain the <i>para</i>-bromophenol motif. In this class, we observe that the sulfone and acid derivatives retain promising affinity in comparison to their direct <i>ortho</i>-chlorophenol analogues. Interestingly, the reduction in affinity observed when replacing the bromophenol appears more pronounced for the methylamides (compare <b>7j</b>, <b>7k</b>, vs <b>9</b>, <b>13</b>), while the sulfone examples <b>8</b>, <b>10</b>, and <b>11</b>, remain similar to <b>7x</b>–<b>7z</b>, respectively. Notably, <b>15</b> is an exception.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selected compounds featuring removal or replacement of the sulfonyl ring phenol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Pharmaceutical Properties</h3><div class="NLM_p">We explored the tier 1 drug metabolism and pharmacokinetic profile of a number of compounds within this class (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). These data demonstrate that all tested analogues have good solubility, and the amide and sulfone series have significantly improved Madin-Darby Canine Kidney (MDCK) permeability in comparison to their salicylic acid counterparts. There was no evidence of P-gp mediated efflux with the tested analogues. A limitation of this compound class is the severe plasma protein binding exhibited by all tested compounds; indeed, salicylic acid itself is known to be a substrate for human serum albumin.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Efforts to overcome this hurdle to develop an improved chemical probe are ongoing.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmaceutical Property Profile of Selected Compounds<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">solubility (μM)</th><th class="colsep0 rowsep0" align="center" char=".">MDCK A-B <i>P</i><sub>app</sub> (10<sup>6</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">MDCK B-A <i>P</i><sub>app</sub> (10<sup>6</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i><sub>u</sub> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5f</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">36.4</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0026</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5n</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">27.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5k</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">14.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.61</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.33</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">16.7</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td><td class="colsep0 rowsep0" align="char" char="."><0.0004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char="."><0.0002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7j</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">45.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7k</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">14.0</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7n</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">58.4</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7p</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">4.79</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7q</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">3.89</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.0007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7t</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">16.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7w</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">18.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><0.0003</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">All experiments performed at Q<sup>2</sup> Solutions Ltd.</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> WDR5 Stabilization by Thermal Shift</h3><div class="NLM_p">To further demonstrate effective, on-target binding of our inhibitors to WDR5, we used differential scanning fluorimetry (DSF) to examine a subset of compounds (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). For the salicylic acid subseries (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A), we observe that the most potent examples by fluorescence polarization assay (FPA) (<b>6o</b>, <b>6r</b>, and <b>6t</b>) also impart the largest Δ<i>T</i><sub>m</sub> and thereby stabilize WDR5. A negative control compound, <b>6-NC</b>, contains a regioisomeric bromine designed such that the key halogen bonding bromine atom is moved around the phenyl ring, causing a significant decrease in potency (see Supplemental Figure S1). This is a direct regioisomer of <b>6q</b>, and in this assay, <b>6-NC</b> has a minimal effect (Δ<i>T</i><sub>m</sub> = 0.42 °C). When comparing a compound subset that all contain the spirocyclobutane moiety (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B), the addition of the carboxylic acid (<b>6q</b> vs <b>5n</b>) offers a significant increase in binding affinity by FPA, but only a marginal improvement in thermal shift is observed. However, the additional hydrophobic interaction brought by the addition of the chlorine on the sulfonyl ring affords compounds with an increased Δ<i>T</i><sub>m</sub> (<b>6r</b> and <b>7k</b>). <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C highlights that the binding of <b>7k</b> demonstrates a concentration-dependent response. A summary of the Δ<i>T</i><sub>m</sub> values is displayed in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D, and the effect of further tested examples is shown in Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Table S1</a>. We have also confirmed that the binding of <b>7k</b> in the absence of WDR5 protein does not have any effect on the background fluorescence at any of these concentrations (Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Figure S2</a>), and that these compounds do not display significant binding at the “WIN” binding site (Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0016.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of compounds by thermal shift, all compounds measured at 10 μM in triplicate, and average values are displayed; RFU = relative fluorescence unit. (A) Comparison of compounds in the salicylic acid subseries. (B) Comparison of compounds that all contain the spirocyclobutane substructure. (C) Ten-point dilution of compound <b>7k</b> demonstrates the dose-dependent effect on WDR5 stabilization. (D) Measured Δ<i>T</i><sub>m</sub> values for compounds displayed in panels A–C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Confirmation of Target Engagement in Cell Lysates</h3><div class="NLM_p">To assess whether compounds disrupt the association between WDR5 and interacting partners at the WBM site (MYC and RBBP5), we immunoprecipitated WDR5 from cell lysates treated with compounds. We first chose a representative selection of compounds from the acid and amide subseries with a range of binding affinities to WDR5 as measured by FPA. Initially, we treated cell lysates with 50 μM compound and observed varying degrees of disruption between both WDR5-c-MYC and WDR5-RBBP5 as compared to both the vehicle control and the representative negative control compounds that bind with markedly lower affinity to WDR5 from both acid and amide subseries (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A,B). As anticipated, the degree of disruption between WDR5-c-MYC and WDR5-RBBP5 was similar for each compound. Furthermore, the degree of disruption also corresponded to the compound’s respective binding affinity to WDR5, where compounds with a higher affinity to WDR5 disrupted the WBM site protein interactions to a greater degree, further highlighting the efficacy of the acid compound subseries. Similarly, exemplar compounds from the sulfone series as well as those with phenol replacements can also disrupt binding of c-MYC to WDR5 in cellular lysates (Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Figure S3</a>), in a manner that is associated with the affinity of their binding to WDR5.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0015.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound target engagement in cell lysates. Levels of c-MYC and RBBP5 proteins in association with WDR5 were assessed after treatment with compound in lysates from WDR5-FLAG and c-MYC-HA co-overexpressing HEK293 cells. A fraction of the lysate before addition of compound is shown as a control lane. An α-FLAG immunoprecipitation was used to isolate WDR5, which is not detectable in the input due to the excess of WDR5 in the immunoprecipitation (IP) fraction. c-MYC and RBBP5 were blotted in independent experiments. Data are representative of minimum two independent biological repeats. α-FLAG immunoprecipitation from lysates treated with 50 μM compound from (A) acid subseries and (B) amide subseries. (C) Dose–response treatment of compounds <b>6r</b> and <b>7k</b> from 6.25 to 50 μM, as compared to 50 μM treatment with respective inactive controls <b>6-NC</b> and <b>7-NC</b> and the vehicle control (VC). Control cells express c-MYC-HA and a GFP-FLAG construct. (D–F) Quantification of two independent experiments, normalized to vehicle control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Further, to assess the effects of lower compound concentrations on the disruption of the WDR5-c-MYC and WDR5-RBBP5 associations, we also conducted a dose–response treatment with compounds <b>6r</b> and <b>7k</b>, alongside their corresponding negative controls <b>6-NC</b> and <b>7-NC</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). We observed a dose-dependent disruption of the association between both WDR5-c-MYC and WDR5-RBBP5, as compared to the vehicle control and to the respective negative control compounds from each subseries. The dose dependency was most clearly demonstrated in response to <b>7k</b> treatment when assessing the association between WDR5 and RBBP5. The improved efficacy of acid compound <b>6r</b> was also evident as a marked disruption between both WDR5-c-MYC and WDR5-RBBP5 was clearly observed even at the lowest concentration tested.</div><div class="NLM_p last">Overall, these compounds demonstrate biochemical binding affinity caused by the displacement of an FITC-labeled probe from recombinant, purified WDR5 and binding to WDR5 in cell lysates by thermal shift and coimmunoprecipitation. We are also able to demonstrate a functional effect of this binding as these compounds inhibit the biochemical histone methyltransferase (HMT) activity of MLL1 in the full WRAD complex (Supplemental <a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">Table S2</a>). The results of this assay suggest that compounds disrupting the WDR5–RBBP5 interface of the multiprotein WRAD complex may offer an alternative strategy for the targeting of tumors driven by abrogation of MLL proteins, avoiding the WIN binding site. Interestingly, we observe that the potency of functional inhibition in the HMT assay is much closer to the biochemical binding affinity of inhibitors at the WBM site versus previously disclosed inhibitors of WIN site from both our group and others.<a onclick="showRef(event, 'ref32 ref33 ref38'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref38">(32,33,38)</a> The inherent weaker affinity of the WDR5–RBBP5 interaction may present a significantly lower barrier for a small-molecule disruptor to overcome in comparison to that of the WDR5–MLL interface.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p">All compounds were synthesized in modular fashion using a sulfonamide coupling to join elaborated anilines and sulfonyl chlorides. This coupling used pyridine as a base, which favors N- versus O-linked reaction. For examples <b>1</b>, <b>5a</b>–<b>5e</b>, <b>5g</b>, and <b>5h</b>, both coupling partners were commercially available, allowing for rapid initial SAR exploration (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A), while the more elaborate aniline pieces often required multistep synthesis. Chlorosulfonation of the 4-bromophenol derivatives afforded corresponding sulfonyl chlorides, <b>18a</b> and <b>18b</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B). Examples <b>5j</b> and <b>5k</b> both originated from corresponding phenols, which were nitrated and reduced to the required aniline intermediates <b>20</b>. The Suzuki-Miyaura coupling using cyclopropylboronic acid was reliant on protection to form intermediate <b>23</b>. The phenol was benzyl protected, and the aniline masked as the corresponding nitro group. After cross-coupling, the subsequent hydrogenation cleanly affords the anilinophenol, <b>23</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>D). The trifluoromethylcyclopropane example was synthesized using methodology from Merck,<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48−50)</a> cross-coupling first to the CF<sub>3</sub>-styrene, <b>21</b>, and cyclopropanation using a sulfonium ylide as a diazomethane equivalent (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>E). As previously described, deprotection by hydrogenation affords the desired intermediate <b>26</b>.</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0014.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Sulfonamides and the Requisite Aniline Building Blocks to Compounds <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) pyridine, DCM, r.t.; (b) HSO<sub>3</sub>Cl, 0 °C to r.t.; (c) HNO<sub>3</sub>·H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C to r.t.; (d) H<sub>2</sub>, Pd/C, MeOH, r.t.; (e) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (f) benzyl bromide, K<sub>2</sub>CO<sub>3</sub>, MeCN, Δ; (g) cyclopropylboronic acid, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>·HBF<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene, 80 °C; (h) 3,3,3-trifluoroprop-1-ene, Pd(dppf)Cl<sub>2</sub>·DCM, K<sub>2</sub>CO<sub>3</sub>, dioxane, water, 70 °C; (i) Ph<sub>2</sub>MeSBF<sub>4</sub>, LiHMDS, THF, −78 °C.</p></p></figure><div class="NLM_p">The cyano-spirocycles were all synthesized by cyclization of O-protected intermediates <b>27</b> with the corresponding dibromoalkane using sodium hydride in DMSO, and the common bromide intermediates <b>28</b> were used to furnish all of the required aniline intermediates (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Toward products <b>5m</b>–<b>5p</b>, the bromide was displaced with <i>tert</i>-butyl carbamate to furnish the desired aniline, <b>29a</b>–<b>29d</b>, after deprotection. Intermediates <b>28</b> could instead be converted to the respective phenyl ester using palladium-mediated carbonylation to generate <b>30</b> or to the methyl sulfone by palladium-mediated coupling with sodium methanethiolate and then oxidation with potassium peroxymonosulfate to <b>32</b>. The synthesis of products <b>6q</b>, <b>6r</b>, and <b>12</b> was completed by heating the phenyl ester, <b>31</b>, with lithium hydroxide. The salicylamide formation to <b>7j</b>–<b>7o</b>, <b>9</b>, and <b>13</b> proceeded by heating the ester directly with the corresponding amine; such amide formations did not require prior hydrolysis.</div><figure id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0013.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Cyano-Spirocycle Products Utilizing Common Bromide Intermediate, <b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dialkylbromide, NaH, DMSO, r.t.; (b) <i>tert</i>-butyl carbamate, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C; (c) BBr<sub>3</sub>, DCM, −78 °C; (d) TFA, DCM, r.t.; (e) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (f) phenyl formate, phenol, Pd(OAc)<sub>2</sub>, P(<i><sup>t</sup></i>Bu)<sub>3</sub>·HBF<sub>4</sub>, NEt<sub>3</sub>, toluene; (g) HNO<sub>3</sub>, TBAB, DCE/H<sub>2</sub>O, 0–23 °C; (h) H<sub>2</sub>, Pd/C, MeOH, r.t.; (i) 2 M LiOH, THF, r.t.; (j) amine, DIPEA, THF, Δ; (k) NaSMe, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, NEt<sub>3</sub>, THF, Δ; (l) oxone, EtOH, H<sub>2</sub>O, r.t.; (m) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C.</p></p></figure><div class="NLM_p">Pentafluorosulfanyl analogues were synthesized using a similar chemical approach as previously used for the cyano-spirocycles (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The −SF<sub>5</sub> group in all instances was incorporated in the commercially available intermediates. <b>34</b> underwent the same palladium-mediated carbonylation and demethylation to <b>35</b>, but the nitration of the phenyl ester was unsuccessful despite a condition screen. However, nitration of the benzoic acid and subsequent esterification to the methyl ester afforded access to intermediate <b>36</b> in good yield (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>A). The corresponding sulfones were introduced via palladium cross-coupling of the aryl bromide with dimethyl disulfide, oxidation to <b>38</b>, nitration with nitric acid, followed by hydrogenation furnished <b>39</b>.</div><figure id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Pentafluorosulfonyl Products from Commercial Bromide Intermediate, <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenyl formate, phenol, Pd(OAc)<sub>2</sub>, P(<i><sup>t</sup></i>Bu<sub>3</sub>)·HBF<sub>4</sub>, Et<sub>3</sub>N, MeCN, 90 °C; (b) BBr<sub>3</sub>, DCM, −78 °C; (c) LiOH, THF, 65 °C; (d) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C; (e) H<sub>2</sub>SO<sub>4</sub>, MeOH, 65 °C; (f) H<sub>2</sub>, Pd/C, MeOH, r.t.; (g) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (h) amine, THF, 65 °C; (i) Br<sub>2</sub>, FeCl<sub>3</sub>, AcOH, 0 °C; (j) S<sub>2</sub>(CH<sub>3</sub>)<sub>2</sub>, Cu, pyridine, 115 °C; (k) oxone, EtOH, H<sub>2</sub>O, r.t.</p></p></figure><div class="NLM_p">The synthesis of the densely substituted analogues <b>6g</b>, <b>6h</b>, <b>7c</b>, and <b>7d</b> is outlined in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The readily available acid <b>40</b> was esterified, and the subsequent bromination was found to be largely selective <i>para</i>- to the phenol. The inseparable, minor dibrominated byproduct was separable after subsequent nitration to <b>43</b>. As described above, protection of the phenol was required for cross-coupling of cyclopropylboronic acid, and a similar sequence of hydrogenation, sulfonamide coupling, and hydrolysis or amidation affords the screened compounds.</div><figure id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0011.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>6g</b>, <b>6h</b>, <b>7c</b>, and <b>7d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TMS diazomethane, THF, MeOH, 0 °C; (b) HBF<sub>4</sub>·Et<sub>2</sub>O, NBS, MeCN, −10 °C; (c) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C; (d) BnBr, K<sub>2</sub>CO<sub>3</sub>, MeCN, Δ; (e) cyclopropylboronic acid, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>·HBF<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene, 80 °C; (f) H<sub>2</sub>, Pd/C, MeOH, r.t.; (g) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (h) LiOH, THF, 65 °C; (i). H<sub>2</sub>NMe, THF, 65 °C.</p></p></figure><div class="NLM_p">The synthesis of analogues that lack a phenol on the aniline ring is shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The sulfone intermediates are all synthesized by palladium-mediated cross-coupling with sodium methanesulfinate (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>A–C). Toward <b>7t</b> and <b>7u</b>, a Curtis rearrangement of the acid intermediate was utilized to generate aniline <b>49</b>. Spirocyclobutanes were synthesized from toluene-derived starting materials, which were brominated, and the nitrile was introduced using trimethylsilyl cyanide (TMSCN). Cyclization with 1,3-dihalopropanes affords intermediates <b>51</b> and <b>53</b>, which were used for sulfonamide formation.</div><figure id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0010.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. (A–C) Synthesis of Sulfone Analogues <b>7p</b>, <b>7q</b>, <b>7t</b>, <b>7u</b>, and <b>7x</b> and (D) Spirocyclobutane Analogues <b>6p</b>, <b>7h</b>, and <b>7i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeSO<sub>2</sub>Na, CuI, <span class="smallcaps smallerCapital">l</span>-proline, NaOH, DMSO, 80 °C; (b) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (c) 2 M LiOH, THF, 65 °C; (d) DPPA, <i>t</i>-BuOH, DME, 100 °C; (e) TFA, DCM, r.t.; (f) Boc<sub>2</sub>O, MeOH, r.t.; (g) NBS, AIBN, CCl<sub>4</sub>, 80 °C; (h) TMSCN, K<sub>2</sub>CO<sub>3</sub>, MeCN, 0–50 °C; (i) 1,3-dibromopropane, NaH, DMSO, r.t.; (j) HCl, dioxane, 50 °C; (k) 1,3-diiodopropane, KO<i><sup>t</sup></i>Bu, THF, r.t.; (l) <i>tert</i>-butyl carbamate, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C; (m) H<sub>2</sub>NMe, THF, 65 °C.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">WD repeat proteins are an emerging class of targets for a range of therapeutic areas, although there has been, to date, no clinical validation of these proteins with approved drugs. Inhibitors of WDR5 are being sought as highly attractive options for the treatment of multiple cancer types. To date, there have been several inhibitors published that bind to WDR5 at the “WIN” binding site, but no inhibitors of the “WBM” site have been disclosed. An inhibitor of this site directly prevents WDR5 from being able to bind to MYC, RBBP5, and KANSL2 among others and thus could be of therapeutic interest in cancers driven by oncogenic effects of these proteins. Further, this approach may offer an alternative pathway to affect the HMT activity of MLL1 while not directly preventing the binding of other protein partners into the MLL1 binding region.</div><div class="NLM_p">We performed a high-throughput screen of the Vanderbilt library and obtained a series of biaryl sulfonamide hits that were unambiguously confirmed as inhibitors of this binding site by NMR and X-ray crystallography. Using rational design, we optimized hit compound <b>1</b>, yielding multiple analogues binding at less than 30 nM. To our knowledge, the data reported herein represents the first published examples of a small-molecule inhibitor binding to an external cleft of a WD repeat protein with such levels of affinity. Binding to the protein was confirmed by secondary assays, including NMR and DSF. Biochemical activity at this cleft was demonstrated in a secondary HMT assay, whereby the small molecule was able to successfully disrupt the WRAD multiprotein complex and thus the methyltransferase function of MLL1. These compounds bind on the opposite face of WDR5 to the MLL1 interaction. We observe no major changes in the protein X-ray structure that would suggest an allosteric modification in the MLL1 binding pocket, yet these compounds are still able to prevent the arginine methylation of this complex. Intriguingly, the difference in activity observed between the binding and HMT assays is significantly less than that previously reported for direct WIN site inhibitors.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Further, some SAR correlations are evident with subseries of compounds, even in this limited dataset, prompting significant interest and further studies within our laboratories.</div><div class="NLM_p last">These inhibitors were demonstrated to be highly soluble, and permeability was significantly improved by switching from a salicylic acid to salicylamide or methyl sulfone motif, all of which are able to form a H-bond interaction with GLN289. Compounds from this series are able to disrupt the interaction of WDR5 with both c-MYC and RBBP5 by coimmunoprecipitation in cellular lysates in a manner that is correlated with binding affinity. However, the low free fraction shown by most analogues limits their utility in whole cells. Ongoing efforts seek to improve the drug-like character of these compounds to discover molecules that can be used as probes of the interruption of the WBM site.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Protein Expression and Purification</h3><div class="NLM_p">Truncated WDR5 (a.a. 22–334) was cloned into a pET vector with a 6× His-SUMO tag fused at the N-terminus. The plasmid WDR5 was transformed into <i>Escherichia coli</i> BL21(DE3) cells. The overnight culture was used to start a 10 L fermentation (BioFlo 415, New Brunswick Scientific) grown at 37 °C. For NMR samples, uniformly <sup>15</sup>N-labeled protein was produced in a minimal M9 medium, where <sup>15</sup>NH<sub>4</sub>Cl (Cambridge Isotope Laboratories) and <span class="smallcaps smallerCapital">d</span>-glucose were used as sole nitrogen and carbon sources. When the cell density reached OD600 = 2.5, the temperature was lowered to 30 °C. The protein was expressed overnight with 1 mM isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactoside (IPTG). MYC peptide (DEEEIDVVSVE) was ordered (GenScript) as HPLC purified synthetic polypeptide. It was dissolved in DMSO for further use.</div><div class="NLM_p last">Cell pellets were dissolved in lysis buffer (1× PBS and 300 mM NaCl, 20 mM imidazole, 5 mM BME, and 10% glycerol) and broken by homogenization (APV-2000, APV). The lysate was cleared by centrifugation and filtration and then applied to an affinity column (140 mL, ProBond, Invitrogen). Bound protein was eluted by an imidazole gradient. The His-SUMO tag was removed by SUMO protease cleavage during dialysis and the subsequent subtractive second nickel column. WDR5 protein was then purified by size-exclusion chromatography (HiLoad 26/60, Superdex 75, GE Healthcare) using NMR or crystallization buffer.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> HTS Screening</h3><div class="NLM_p last">The previously described MYC peptide<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> was labeled with FITC and used as the probe for FPA assays. The probe was ordered from Genscript. WDR5 protein (5 μM) and probe (5 μM) were used in the HTS, and the buffer condition was 1× PBS and 300 mM NaCl, pH 6.0, 0.5 mM TCEP, 0.1% CHAP, and 5% DMSO. Vanderbilt Discovery Collection (VDC) and VICB collection compounds (∼250,000) were tested at 50 μM concentration, and the compounds was put into 781,386-well plates with necessary positive and negative controls in each plate. After adding all reagents, the plates were shaken for ∼2 min and incubated for 60 min before the first plate to be read on a plate reader (Biotek). The reading settings of the plate reader were 50 flashes, low lamp energy, and 7.75 read height. All the plates were screened, and those plates (totaling 17) with <i>Z</i>′ less than 0.3 were repeated. Compounds that showed >15% inhibition were selected for the confirmation screen. Confirmed compounds then were used for a 10-point dose–response study.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Hit Validation by NMR</h3><div class="NLM_p last">NMR samples contained 2 mg mL<sup>–1</sup> (∼67 μM) <sup>15</sup>N-labeled WDR5 in 25 mM phosphate buffer, pH = 6.0, 100 mM NaCl, and 1 mM DTT. HTS hits were used at 100 μM. Nuclear magnetic resonance screening was conducted using a Bruker Avance III 600 MHz NMR spectrometer equipped with a 5 mm single-axis <i>z</i>-gradient cryoprobe and a Bruker SampleJet sample changer. Two-dimensional, gradient-enhanced <sup>1</sup>H–<sup>15</sup>N heteronuclear multiple-quantum coherence (SOFAST-HMQC)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> spectra were collected at 25 °C and used to track chemical shift perturbation upon compound binding. Spectra were processed and visualized using TopSpin (Bruker BioSpin).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Protein Crystallization, Data Collection, and Structure Refinement</h3><div class="NLM_p">WDR5 was concentrated to 10 mg mL<sup>–1</sup> (∼300 μM) in the buffer of 20 mM HEPES, pH 7.0, 250 mM NaCl, and 5 mM DTT. Cocrystals were obtained at 18 °C using the hanging drop method. The crystallization condition was 0.1 M Bis-Tris, pH 6.0, 0.2 M ammonium acetate, and 28 to 32% PEG3350. Soaking was also applied to some compounds using crystals of WDR5 that contain an MLL site binder. Crystals were flash frozen in liquid nitrogen directly.</div><div class="NLM_p last">Diffraction data were collected on the Life Sciences Collaborative Access Team (LS-CAT) 21-ID-D and 21-ID-G beamlines at the Advanced Photon Source (APS), Argonne National Laboratory. Data were indexed, integrated, and scaled with HKL2000.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Molecular replacement was achieved with Phaser44 as implemented in CCP4.45<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> using a previously determined WDR5 structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6">3EG6</a>). Refinement of the structural models was conducted with PHENIX<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and included rounds of manual model building in COOT.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> All structure images were prepared with PyMOL.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Cloning and Plasmids</h3><div class="NLM_p last">Full-length human WDR5 with an N-terminal FLAG tag was cloned into the <i>Ngo</i>MIV and <i>Sal</i>I sites as a <i>Ngo</i>MIV/<i>Xho</i>I fragment. pBabe-GFP was constructed by PCR amplifying the GFP fragment from pEGFP-C2 (Clontech) and cloning the product into the <i>Eco</i>RI and <i>Bam</i>HI sites of pBabe-Puro. Full-length human c-MYC with a C-terminal double HA tag was cloned into the <i>Bam</i>HI and <i>Eco</i>RI sites of pBabe-IGH.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> HEK293 cells stably expressing MYC2HA were made by retroviral transduction followed by selection in hygromycin (50 μg mL<sup>–1</sup>). The mixed population was then infected with pBabe-Puro expressing GFP or WDR5 with selection in puromycin (1 μg mL<sup>–1</sup>). For retroviral transductions, HEK293T cells were transfected with the appropriate pBabe vector, the pCL10A packaging vector, and pMax-GFP to estimate transfection efficiency. Viral supernatant was collected and used to infect HEK293 class over 3 days.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Cell Culture</h3><div class="NLM_p last">HEK293 cells were maintained in DMEM supplemented with 10% FBS. Hygromycin B (50 μg mL<sup>–1</sup>) and puromycin (100 ng mL<sup>–1</sup>) were added to media to maintain plasmid expression. Both cell lines were tested and confirmed negative of mycoplasma using the VenorGeM PCR test kit (Sigma-Aldrich). After thawing from liquid nitrogen, cells were passaged at least twice before use in experiments and passaged for a maximum of 25 times.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> WDR5 Immunoprecipitation</h3><div class="NLM_p last">HEK293 cells were harvested, and lysates were prepared on ice in lysis buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X100, and supplemented with protease and phosphatase inhibitors). Equal amounts of protein lysate were subjected to immunoprecipitation with M2 agarose overnight (for WDR5-c-MYC) or for 2 h with magnetic M2 beads (for WDR5-RBBP5) at 4 °C. Immune complexes were recovered, washed in lysis buffer, and resolved by SDS-PAGE. Immunoblotting was performed using the indicated primary antibodies and incubated with labeled secondary antibodies, and membranes were scanned using an Odyssey imager (LiCor).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Antibodies</h3><div class="NLM_p last">The following primary antibodies were used for this study: α-c-MYC (#5605), α-RBBP5 (#13171), α-WDR5 (#13105), and α-FLAG (#8146) all purchased from Cell Signaling.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Pharmaceutical Property Determinations</h3><div class="NLM_p last">All experiments performed at Q<sup>2</sup> Solutions Ltd. (<a href="https://www.q2labsolutions.com" class="extLink">https://www.q2labsolutions.com</a>) using commercially available, standard assay formats. Kinetic solubility was determined using method QUI-SOL-001. MDCK permeability was determined using method QUI-PERM-003. Protein binding assessment was performed using method QUI-PERM-002.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> General Chemistry</h3><div class="NLM_p last">All chemical reagents and reaction solvents were purchased from commercial suppliers and used as received. Hydrogenation reactions are performed using an atmospheric balloon. Normal-phase flash silica gel-based column chromatography is performed using ready-to-connect cartridges from ISCO, on irregular silica gel, a particle size of 15–40 μM using a Teledyne ISCO CombiFlash Rf system. Preparative reversed-phase HPLC was performed on a Gilson instrument equipped with a Phenomenex Kinetex C18 column, using gradients of MeCN in H<sub>2</sub>O and 0.1% TFA. Compounds that are obtained as a TFA salt after purification were afforded as the free base, by dissolving the salt in EtOAc and washing with sat. aq. K<sub>2</sub>CO<sub>3</sub>. Proton nuclear magnetic resonance (<sup>1</sup>H NMR) spectra were recorded at either 400 or 600 MHz on a Bruker spectrometer, as stated. For <sup>1</sup>H NMR spectra, chemical shifts are reported in parts per million (ppm) and relative to residual nondeuterated solvent signals. Coupling constants are reported in hertz (Hz). The following abbreviations (or a combination, thereof) are used to describe splitting patterns: s, singlet; d, doublet; t, triplet; q, quartet; p, pentet; m, multiplet; and br, broad. All compounds were of 95% purity or higher, unless otherwise noted, as measured by analytical reversed-phase HPLC. Analytical HPLC was performed on an Agilent 1200 series system with UV detection at 214 and 254 nm, along with evaporative light scattering detection (ELSD). Low-resolution mass spectra were obtained on an Agilent 6140 mass spectrometer with electrospray ionization (ESI). LC–MS experiments were performed with the following parameters: Phenomenex Kinetex 2.6 μm XB-C18 100 Å, LC column 50 × 2.1 mm, a gradient of 5–95% MeCN in H<sub>2</sub>O, and 0.1% TFA, over a 1.2or 2.0 min gradient. HRMS experiments were conducted at the University of Notre Dame Mass Spectrometry and Proteomics Facility.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> General Procedure A: Sulfonamide Coupling</h3><div class="NLM_p last">The corresponding aniline (1 equiv) and sulfonyl chloride (1.2–1.5 equiv) were combined in CH<sub>2</sub>Cl<sub>2</sub> (at [0.2 M]) at r.t., and pyridine (3 equiv) was added; the mixture was stirred for 1–16 h. The mixture was concentrated in vacuo and purified by ISCO flash chromatography, unless stated.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Procedure B: Hydrogenation</h3><div class="NLM_p last">Corresponding intermediate (1 equiv) was dissolved in MeOH, and EtOAc was added to aid dissolution where required. Pd/C (5% C by wt, 10 mol%) was added, and the mixture was stirred under a H<sub>2</sub> atmosphere for 16 h. The mixture was filtered through celite, concentrated, and purified by ISCO flash chromatography if required.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> General Procedure C: Nitration</h3><div class="NLM_p last">Corresponding intermediate (1 equiv) was dissolved in DCM (at [0.4 M]) and cooled to 0 °C in an ice/water bath. To this was added a premixed solution of HNO<sub>3</sub> (concd, 1.5 equiv) and H<sub>2</sub>SO<sub>4</sub> (concd, 1.5 equiv) dropwise. The mixture was stirred for 1–16 h, allowed to warm to r.t., poured over ice, and then extracted with DCM. The crude material was purified by ISCO flash chromatography.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> General Procedure D: BBr<sub>3</sub> Mediated Demethylation</h3><div class="NLM_p last">The intermediate (1 equiv) was dissolved in anhydrous DCM (at [0.4 M]) and cooled to −78 °C in an acetone/dry ice bath under an inert atmosphere. To this was added BBr<sub>3</sub> ([1.0 M in DCM], 2 equiv) dropwise, and the mixture was stirred for 1 h at −78 °C and then allowed to warm to r.t. The mixture was poured over an ice/water slurry, extracted with EtOAc, and washed with water and brine. The crude product was purified by ISCO flash chromatography.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> General Procedure E: Ester Hydrolysis</h3><div class="NLM_p last">The ester intermediate (1 equiv) was stirred with LiOH (2 M, 5 equiv) in THF (at [0.2 M]) for 1–16 h at 65 °C, unless stated at r.t. The mixture was acidified with hydrochloric acid, extracted with EtOAc, and washed with brine. The crude material was purified by preparative HPLC.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> General Procedure F: Methylamide Synthesis</h3><div class="NLM_p last">To the salicylate ester intermediate (1 equiv) was added methylamine ([2.0 M in THF], 10 equiv), and the mixture was heated at 65 °C for 1–16 h. The mixture was concentrated in vacuo and purified by preparative HPLC.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> General Procedure G: Benzyl Protection</h3><div class="NLM_p last">The intermediate phenol (1 equiv), benzyl bromide (1.05 equiv), and potassium carbonate (1.1 equiv) were combined in acetonitrile and heated to reflux for 18 h. Upon cooling, the mixture was concentrated in vacuo, redissolved in EtOAc, and washed with water and brine. The crude material was purified by ISCO flash chromatography.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-Bromo-<i>N</i>-(5-chloro-2-hydroxyphenyl)-2-methoxybenzenesulfonamide (<b>1</b>)</h3><div class="NLM_p last">2-Amino-4-chlorophenol (144 mg, 1.0 mmol) was reacted with 5-bromo-2-methoxybenzenesulfonyl chloride, and <b>16a</b> (428 mg, 1.5 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (283 mg, 0.72 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.83 (d, <i>J =</i> 2.5 Hz, 1H), 7.66 (dd, <i>J =</i> 8.9, 2.5 Hz, 1H), 7.31 (d, <i>J =</i> 2.5 Hz, 1H), 7.08 (d, <i>J =</i> 8.9 Hz, 1H), 6.88 (dd, <i>J =</i> 8.6, 2.6 Hz, 1H), 6.67 (d, <i>J =</i> 8.6 Hz, 1H), 3.91 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.44 min, <i>m/z</i> = 393.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>12</sub>BrClNO<sub>4</sub>S, 393.9353; found, 393.9344.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-(5-Chloro-2-hydroxyphenyl)-2-methoxybenzenesulfonamide (<b>5a</b>)</h3><div class="NLM_p last">2-Amino-4-chlorophenol (14 mg, 0.10 mmol) was reacted with 2-methoxybenzenesulfonyl chloride (31 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (25 mg, 0.08 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.79 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.57 (ddd, <i>J</i> = 8.7, 7.3, 1.7 Hz, 1H), 7.08 (d, <i>J</i> = 8.4 Hz, 1H), 7.03 (t, <i>J</i> = 7.6 Hz, 1H), 6.99 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 6.86–6.79 (m, 3H), 6.58 (s, 1H), 4.07 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.54 min, <i>m/z</i> = 313.9 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 5-Chloro-<i>N</i>-(5-chloro-2-hydroxyphenyl)-2-methoxybenzenesulfonamide (<b>5b</b>)</h3><div class="NLM_p last">2-Amino-4-chlorophenol (14 mg, 0.1 mmol) was reacted with 5-chloro-2-methoxybenzenesulfonyl chloride (36 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (21 mg, 0.06 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.50 (s, 1H), 7.47 (dd, <i>J</i> = 6.6, 2.6 Hz, 2H), 7.34 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.11 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.04 (s, 1H), 6.89 (d, <i>J</i> = 8.8 Hz, 1H), 6.70 (d, <i>J</i> = 8.8 Hz, 1H), 3.65 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.03 min, <i>m/z</i> = 377.3 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i>-(5-Chloro-2-hydroxyphenyl)-5-iodo-2-methoxybenzenesulfonamide (<b>5c</b>)</h3><div class="NLM_p last">2-Amino-4-chlorophenol (14 mg, 0.1 mmol) was reacted with 5-iodo-2-methoxybenzenesulfonyl chloride (50 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a pale brown solid (5 mg, 0.01 mmol, 11% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.98 (d, <i>J =</i> 2.3 Hz, 1H), 7.82 (dd, <i>J =</i> 8.8, 2.3 Hz, 1H), 7.27 (d, <i>J =</i> 2.6 Hz, 1H), 6.95 (d, <i>J =</i> 8.8 Hz, 1H), 6.86 (dd, <i>J =</i> 8.6, 2.6 Hz, 1H), 6.67 (d, <i>J =</i> 8.6 Hz, 1H), 3.90 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.03 min, <i>m/z</i> = 440.1 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 3-Bromo-<i>N</i>-(5-chloro-2-hydroxyphenyl)benzenesulfonamide (<b>5d</b>)</h3><div class="NLM_p last">2-Amino-4-chlorophenol (17 mg, 0.12 mmol) was reacted with 3-bromobenzenesulfonyl chloride (46 mg, 0.18 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (22 mg, 0.72 mmol, 51% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.92 (t, <i>J =</i> 1.8 Hz, 1H), 7.75–7.66 (m, 2H), 7.37 (t, <i>J =</i> 8.0 Hz, 1H), 7.33 (d, <i>J =</i> 2.6 Hz, 1H), 6.95 (dd, <i>J =</i> 8.7, 2.6 Hz, 1H), 6.66 (d, <i>J =</i> 8.7 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.50 min, <i>m/z</i> = 363.6 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 5-Bromo-<i>N</i>-(2-hydroxyphenyl)-2-methoxybenzenesulfonamide (<b>5e</b>)</h3><div class="NLM_p last">2-Aminophenol (17 mg, 0.16 mmol) was reacted with <b>16a</b> (68 mg, 0.24 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (22 mg, 0.06 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.88 (d, <i>J =</i> 2.5 Hz, 1H), 7.63 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.07 (ddd, <i>J =</i> 8.9, 7.4, 1.6 Hz, 1H), 6.95 (d, <i>J =</i> 8.9 Hz, 1H), 6.91 (dd, <i>J =</i> 8.0, 1.4 Hz, 1H), 6.86 (dd, <i>J =</i> 8.0, 1.6 Hz, 1H), 6.83 (s, 1H), 6.73 (td, <i>J =</i> 7.7, 1.4 Hz, 1H), 6.39 (br s, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.41 min, <i>m/z</i> = 359.3 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 5-Bromo-<i>N</i>-(5-chloro-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5f</b>)</h3><div id="sec4_24_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Step A: 5-Bromo-2-hydroxybenzenesulfonyl Chloride (<b>18a</b>)</h4><div class="NLM_p last">4-Bromophenol (<b>17a</b>; 1.73 g, 10 mmol) was added portion wise to ice-cold chlorosulfonic acid (4.77 mL, 70 mmol), and the mixture was stirred for 16 h and warmed to r.t. The mixture was carefully poured over a slurry of ice, DCM, and brine and extracted with DCM. Purification by flash chromatography afforded the title compound as a pale brown oily solid (1.50 g, 5.53 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.95 (d, <i>J =</i> 2.4 Hz, 1H), 7.71 (dd, <i>J =</i> 8.9, 2.4 Hz, 1H), 7.04 (d, <i>J =</i> 8.9 Hz, 1H).</div></div><div id="sec4_24_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Step B: 5-Bromo-<i>N</i>-(5-chloro-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide</h4><div class="NLM_p last">2-Amino-4-chlorophenol (14 mg, 0.1 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (21 mg, 0.06 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.66 (d, <i>J</i> = 2.5 Hz, 1H), 7.53 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.08 (d, <i>J =</i> 8.2 Hz, 2H), 6.88 (d, <i>J =</i> 8.8 Hz, 1H), 6.80 (d, <i>J =</i> 8.2 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.53 min, <i>m/z</i> = 377.8, 379.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-Bromo-<i>N</i>-(4-hydroxy-[1,1′-biphenyl]-3-yl)-2-methoxybenzenesulfonamide (<b>5g</b>)</h3><div class="NLM_p last">3-Amino-[1,1′-biphenyl]-4-ol (24.3 mg, 0.13 mmol) was reacted with <b>16a</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a colorless solid (28 mg, 0.06 mmol, 49% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.98 (t, <i>J =</i> 1.9 Hz, 1H), 7.57 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.40–7.35 (m, 5H), 7.31–7.27 (m, 1H), 7.18 (dd, <i>J =</i> 8.4, 2.2 Hz, 1H), 6.88 (t, <i>J =</i> 7.7 Hz, 2H), 3.96 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.644 min, <i>m/z</i> = 435.3 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 5-Bromo-<i>N</i>-(5-(<i>tert</i>-butyl)-2-hydroxyphenyl)-2-methoxybenzenesulfonamide (<b>5h</b>)</h3><div class="NLM_p last">2-Amino-4-(<i>tert</i>-butyl)phenol (22 mg, 0.13 mmol) was reacted with <b>16a</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a colorless solid (26 mg, 0.06 mmol, 48% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.86 (d, <i>J =</i> 2.5 Hz, 1H), 7.62 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.07 (dd, <i>J =</i> 8.6, 2.4 Hz, 1H), 6.95 (d, <i>J =</i> 8.8 Hz, 1H), 6.84 (d, <i>J =</i> 8.5 Hz, 1H), 6.76 (d, <i>J =</i> 2.5 Hz, 2H), 6.27 (s, 1H), 4.05 (s, 3H), 1.10 (s, 9H). LC–MS: <i>t</i><sub>R</sub> = 1.70 min, <i>m/z</i> = 415.3 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 5-Bromo-<i>N</i>-(5-cyclopropyl-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5i</b>)</h3><div id="sec4_27_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Step A: 1-(Benzyloxy)-4-bromo-2-nitrobenzene (<b>22</b>)</h4><div class="NLM_p last">2-Nitro-4-bromophenol (<b>21</b>; 10.9 g, 50 mmol) was reacted according to <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure G: Benzyl Protection">General Procedure G: Benzyl Protection</a> to afford <b>22</b> as a yellow solid (14.2 g, 46.1 mmol, 92% yield). LC–MS: <i>t</i><sub>R</sub> = 1.25 min, <i>m/z</i> = 307.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_27_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Step B: 1-(Benzyloxy)-4-cyclopropyl-2-nitrobenzene</h4><div class="NLM_p last">1-(Benzyloxy)-4-bromo-2-nitrobenzene (<b>22</b>; 1.54 g, 5.0 mmol), cyclopropylboronic acid (515 mg, 6.0 mmol), Pd(OAc)<sub>2</sub> (56 mg, 0.25 mmol), PCy<sub>3</sub>·HBF<sub>4</sub> (184 mg, 0.50 mmol), and K<sub>3</sub>PO<sub>4</sub> (2.65 g, 12.5 mmol) were taken in toluene (25 mL) and water (5 mL) and stirred at 110 °C in a sealed tube for 16 h. The cooled mixture was filtered through celite, diluted with EtOAc, and washed with water. Purification by flash chromatography afforded a yellow solid (1.22 g, 4.53 mmol, 91% yield). LC–MS: <i>t</i><sub>R</sub> = 1.27 min, does not ionize by ESI.</div></div><div id="sec4_27_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Step C: 2-Amino-4-cyclopropylphenol (<b>23</b>)</h4><div class="NLM_p last">1-(Benzyloxy)-4-cyclopropyl-2-nitrobenzene (539 mg, 2.0 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford <b>23</b> as a colorless solid (262 mg, 1.75 mmol, 88% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 6.59 (d, <i>J =</i> 8.2 Hz, 1H), 6.50 (d, <i>J =</i> 2.2 Hz, 1H), 6.36 (dd, <i>J =</i> 8.2, 2.2 Hz, 1H), 1.80–1.70 (m, 1H), 0.87–0.78 (m, 2H), 0.58–0.49 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.00 min, <i>m/z</i> = 150.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_27_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Step D: 5-Bromo-<i>N</i>-(5-cyclopropyl-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5i</b>)</h4><div class="NLM_p last">2-Amino-4-cyclopropylphenol (<b>23</b>; 15 mg, 0.1 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound as a colorless solid (17 mg, 0.04 mmol, 44% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.40 (s, 1H), 7.61 (d, <i>J =</i> 2.5 Hz, 1H), 7.52 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.90 (dd, <i>J =</i> 8.4, 2.2 Hz, 1H), 6.86 (d, <i>J =</i> 8.8 Hz, 1H), 6.77 (d, <i>J =</i> 8.4 Hz, 1H), 6.60 (d, <i>J =</i> 2.2 Hz, 1H), 6.56 (s, 1H), 5.53 (s, 1H), 1.80–1.68 (m, 1H), 0.91–0.81 (m, 2H), 0.46 (dt, <i>J =</i> 6.6, 4.7 Hz, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.56 min, <i>m/z</i> = 383.8, 385.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(trifluoromethoxy)phenyl)benzenesulfonamide (<b>5j</b>)</h3><div id="sec4_28_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Step A: 2-Amino-4-(trifluoromethoxy)phenol (<b>20a</b>)</h4><div class="NLM_p last">2-Nitro-4-(trifluoromethoxy)phenol (900 mg, 4.03 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>, affording a crude brown solid (693 mg, 3.59 mmol, 89% yield), which was taken forward without purification. LC–MS <i>t</i><sub>R</sub> = 0.83 min, <i>m/z</i> = 194.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_28_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Step B. 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(trifluoromethoxy)phenyl)benzenesulfonamide (<b>5j</b>)</h4><div class="NLM_p last">2-amino-4-(trifluoromethoxy)phenol (19 mg, 0.1 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (4 mg, 0.009 mmol, 9% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.80 (d, <i>J =</i> 2.4 Hz, 1H), 7.58 (d, <i>J =</i> 2.2 Hz, 1H), 7.51 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.43 (d, <i>J =</i> 2.3 Hz, 1H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −60.3. LC–MS: <i>t</i><sub>R</sub> = 1.57 min, <i>m/z</i> = 429.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>5k</b>)</h3><div id="sec4_29_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Step A: 2-Nitro-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">4-Hydroxyphenyl sulfur pentafluoride (1.1 g, 5 mmol) was dissolved in DCM (25 mL) and cooled to 0 °C before the addition of nitronium tetrafluoroborate (863 mg, 6.5 mmol) and stirred for 18 h. The mixture was neutralized with sat. aq. NaHCO<sub>3</sub> and extracted with DCM to afford a crude yellow oil (479 mg, 1.81 mmol, 36% yield), which was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.78 (s, 1H), 8.57 (d, <i>J =</i> 2.7 Hz, 1H), 7.96 (dd, <i>J =</i> 9.2, 2.7 Hz, 1H), 7.27 (d, <i>J =</i> 9.2 Hz, 1H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 82.43 (d, <i>J =</i> 151 Hz, 1F), 63.76 (d, <i>J =</i> 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.03 min, does not ionize by ESI.</div></div><div id="sec4_29_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Step B: 2-Amino-4-(pentafluorosulfanyl)phenol (<b>20b</b>)</h4><div class="NLM_p last">2-Nitro-4-(pentafluorosulfanyl)phenol (479 mg, 1.81 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>, affording a brown gum (99 mg, 0.42 mmol, 23% yield). LC–MS <i>t</i><sub>R</sub> = 0.55 min, <i>m/z</i> = 236.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_29_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Step C: 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>5k</b>)</h4><div class="NLM_p last">2-Amino-4-(pentafluorosulfanyl)phenol, <b>20b</b> (24 mg, 0.1 mmol), was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording <b>5k</b> as a colorless solid (12 mg, 0.026 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.63 (d, <i>J =</i> 2.6 Hz, 1H), 7.59–7.54 (m, 1H), 7.57–7.52 (m, 1H), 7.31 (d, <i>J =</i> 2.6 Hz, 1H), 6.96 (d, <i>J =</i> 9.0 Hz, 1H), 6.89 (d, <i>J =</i> 9.0 Hz, 1H), 6.70 (br s, 2H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 84.3 (p, <i>J =</i> 151 Hz, 1F), 64.0 (d, <i>J =</i> 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.66 min,  <i>m/z</i> = 471.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(1-(trifluoromethyl)cyclopropyl)phenyl)benzenesulfonamide (<b>5l</b>)</h3><div id="sec4_30_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Step A: 2-(4-(Benzyloxy)-3-nitrophenyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane</h4><div class="NLM_p last">4-Hydroxy-3-nitrophenylboronic acid (pinacol ester, <b>24</b>; 980 mg, 3.7 mmol) was reacted following <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure G: Benzyl Protection">General Procedure G: Benzyl Protection</a>. Purification by flash chromatography afforded a yellow solid (1215 mg, 3.4 mmol, 92% yield). MS (ESI) <i>m/z</i> = 373.1 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_30_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Step B: 1-(Benzyloxy)-2-nitro-4-(3,3,3-trifluoroprop-1-en-2-yl)Benzene (<b>25</b>)</h4><div class="NLM_p last">2-(4-(Benzyloxy)-3-nitrophenyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (710 mg, 2.0 mmol), 2-bromo-3,3,3-trifluoroprop-1-ene (830 μL, 8.0 mmol), Pd(dppf)Cl<sub>2</sub>·DCM (163 mg, 0.2 mmol), and K<sub>2</sub>CO<sub>3</sub> (553 mg, 4.0 mmol) were taken in dioxane/water (10:1, 10 mL) and heated to 70 °C for 3 h under an inert atmosphere. The cooled mixture was diluted with EtOAc and washed with water and brine. Purification by flash chromatography afforded a straw-colored oil (433 mg, 1.34 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.96 (d, <i>J =</i> 2.4 Hz, 1H), 7.61–7.55 (m, 1H), 7.48–7.32 (m, 5H), 7.13 (d, <i>J =</i> 8.8 Hz, 1H), 6.01 (d, <i>J =</i> 1.6 Hz, 1H), 5.80 (d, <i>J =</i> 1.6 Hz, 1H), 5.28 (s, 2H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −65.1. LC–MS: <i>t</i><sub>R</sub> = 1.26 min, does not ionize by ESI.</div></div><div id="sec4_30_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Step C: 1-(Benzyloxy)-2-nitro-4-(1-(trifluoromethyl)cyclopropyl)benzene</h4><div class="NLM_p last">To a mixture of 1-(benzyloxy)-2-nitro-4-(3,3,3-trifluoroprop-1-en-2-yl)benzene (65 mg, 0.20 mmol) and Ph<sub>2</sub>MeSBF<sub>4</sub> (115 mg, 0.40 mmol) in anhydrous THF at −78 °C was added LiHMDS [1.0 M in THF] (0.8 mL, 0.8 mmol). The mixture was stirred for 12 h, warmed to r.t., diluted with EtOAc, and then washed with water. Purification by flash chromatography afforded a pale yellow gum (32 mg, 0.09 mmol, 47% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.96 (d, <i>J =</i> 2.3 Hz, 1H), 7.61 (dd, <i>J =</i> 8.7, 2.3 Hz, 1H), 7.51–7.35 (m, 5H), 7.11 (d, <i>J =</i> 8.7 Hz, 1H), 5.27 (s, 2H), 1.45–1.39 (m, 2H), 1.09–1.01 (m, 1H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −70.3. LC–MS: <i>t</i><sub>R</sub> = 1.26 min, does not ionize by ESI.</div></div><div id="sec4_30_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Step D: 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-5-(1-(trifluoromethyl)cyclopropyl)phenyl)benzenesulfonamide (<b>5l</b>)</h4><div class="NLM_p last">1-(Benzyloxy)-2-nitro-4-(1-(trifluoromethyl)cyclopropyl)benzene (32 mg, 0.09 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>, affording a crude colorless solid (20 mg, 0.09 mmol) that was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (13 mg, 0.029 mmol, 31% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.72 (d, <i>J =</i> 2.5 Hz, 1H), 7.49 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.36 (d, <i>J =</i> 2.2 Hz, 1H), 6.99 (dd, <i>J =</i> 8.3, 2.2 Hz, 1H), 6.83 (d, <i>J =</i> 8.8 Hz, 1H), 6.70 (d, <i>J =</i> 8.3 Hz, 1H), 1.30–1.20 (m, 2H), 0.93–0.87 (m, 2H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −71.9. LC–MS: <i>t</i><sub>R</sub> = 1.67 min, <i>m/z</i> = 451.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 5-Bromo-<i>N</i>-(5-(1-cyanocyclopropyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5m</b>)</h3><div id="sec4_31_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Step A: 1-(4-Methoxy-3-nitrophenyl)cyclopropane-1-carbonitrile (<b>28a</b>)</h4><div class="NLM_p last">1-(4-Methoxyphenyl)cyclopropanecarbonitrile (<b>27</b>; 1 mL, 6.3 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a>, affording a yellow solid (1178 mg, 5.40 mmol, 86% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.69 (d, <i>J =</i> 2.5 Hz, 1H), 7.60 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.09 (d, <i>J =</i> 8.8 Hz, 1H), 3.97 (s, 3H), 1.83–1.71 (m, 2H), 1.45–1.36 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 0.87 min, does not ionize by ESI.</div></div><div id="sec4_31_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Step B: 1-(4-Hydroxy-3-nitrophenyl)cyclopropane-1-carbonitrile</h4><div class="NLM_p last">1-(4-Methoxy-3-nitrophenyl)cyclopropane-1-carbonitrile (218 mg, 1.0 mmol) was reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford a cream solid (161 mg, 0.79 mmol, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.54 (s, 1H), 8.01 (d, <i>J =</i> 2.5 Hz, 1H), 7.60 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.19 (d, <i>J =</i> 8.8 Hz, 1H), 1.87–1.71 (m, 2H), 1.48–1.32 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 0.86 min, does not ionize by ESI.</div></div><div id="sec4_31_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Step C: 1-(3-Amino-4-hydroxyphenyl)cyclopropane-1-carbonitrile (<b>29a</b>)</h4><div class="NLM_p last">1-(4-Hydroxy-3-nitrophenyl)cyclopropane-1-carbonitrile (102 mg, 0.5 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>, affording a cream solid (74 mg, 0.43 mmol, 85% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 6.73 (d, <i>J =</i> 2.2 Hz, 1H), 6.67 (d, <i>J =</i> 8.2 Hz, 1H), 6.57 (dd, <i>J =</i> 8.2, 2.2 Hz, 1H), 4.72 (s, 1H), 3.71 (s, 2H), 1.65–1.58 (m, 2H), 1.32–1.27 (m, 2H). ESI <i>m/z</i> = 175.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_31_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Step D: 5-Bromo-<i>N</i>-(5-(1-cyanocyclopropyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5m</b>)</h4><div class="NLM_p last">1-(3-Amino-4-hydroxyphenyl)cyclopropane-1-carbonitrile (27 mg, 0.16 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (47 mg, 0.12 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.51 (s, 1H), 7.61 (d, <i>J =</i> 2.4 Hz, 1H), 7.53 (dd, <i>J =</i> 8.9, 2.4 Hz, 1H), 7.07 (dd, <i>J =</i> 8.5, 2.4 Hz, 1H), 6.92–6.85 (m, 3H), 6.80 (s, 1H), 1.67–1.61 (m, 2H), 1.26–1.20 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.42 min, <i>m/z</i> = 408.9, 410.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 5-Bromo-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5n</b>)</h3><div id="sec4_32_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Step A: 2-(4-Hydroxy-3-nitrophenyl)acetonitrile</h4><div class="NLM_p last">4-Hydroxybenzylacetontrile (2.66 g, 20 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford a yellow-orange solid (3.42 g, 19.2 mmol, 96% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.57 (s, 1H), 8.10 (d, <i>J =</i> 2.4 Hz, 1H), 7.57 (dd, <i>J =</i> 8.7, 2.4 Hz, 1H), 7.22 (d, <i>J =</i> 8.7 Hz, 1H), 3.76 (s, 2H).</div></div><div id="sec4_32_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Step B: 2-(4-(Benzyloxy)-3-nitrophenyl)acetonitrile</h4><div class="NLM_p last">2-(4-Hydroxy-3-nitrophenyl)acetonitrile (1.78 g, 10 mmol) was reacted following <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure G: Benzyl Protection">General Procedure G: Benzyl Protection</a> to afford a yellow-orange solid (2.31 g, 8.6 mmol, 86% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.82 (d, <i>J =</i> 2.4 Hz, 1H), 7.49 (dd, <i>J =</i> 8.7, 2.4 Hz, 1H), 7.47–7.30 (m, 6H), 7.15 (d, <i>J =</i> 8.7 Hz, 1H), 5.26 (s, 2H), 3.74 (s, 2H).</div></div><div id="sec4_32_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Step C: 1-(4-(Benzyloxy)-3-nitrophenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">To a solution of 2-(4-(benzyloxy)-3-nitrophenyl)acetonitrile (134 mg, 0.5 mmol) and 1,3-dibromopropane (61 μL, 0.6 mmol) in DMSO (5 mL) was added carefully NaH (60% in mineral oil, 60 mg, 1.5 mmol), and the mixture was stirred for 16 h. The mixture was diluted with EtOAc/Et<sub>2</sub>O (1:1, 50 mL) and washed with water (3 × 100 mL) and brine (100 mL). Purification by flash chromatography afforded a yellow-brown oil (80 mg, 0.26 mmol, 52% yield). ESI <i>m/z</i> = 309.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_32_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Step D: 1-(3-Amino-4-hydroxyphenyl)cyclobutane-1-carbonitrile (<b>29b</b>)</h4><div class="NLM_p last">1-(4-(Benzyloxy)-3-nitrophenyl)cyclobutane-1-carbonitrile (80 mg, 0.26 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford a colorless oil (26 mg, 0.14 mmol, 53% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 6.84 (d, <i>J =</i> 2.3 Hz, 1H), 6.73 (d, <i>J =</i> 8.2 Hz, 1H), 6.66 (dd, <i>J =</i> 8.2, 2.3 Hz, 1H), 2.77–2.66 (m, 2H), 2.66–2.56 (m, 2H), 2.42–2.28 (m, 1H), 2.13–2.00 (m, 1H). ESI<i>m/z</i> = 189.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_32_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Step E: 5-Bromo-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5n</b>)</h4><div class="NLM_p last">1-(3-Amino-4-hydroxyphenyl)cyclobutane-1-carbonitrile (<b>29b</b>; 26 mg, 0.14 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (22.5 mg, 0.05 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.65 (d, <i>J =</i> 2.4 Hz, 1H), 7.56 (dd, <i>J =</i> 8.8, 2.4 Hz, 1H), 7.22 (dd, <i>J =</i> 8.5, 2.4 Hz, 1H), 6.98 (d, <i>J =</i> 2.4 Hz, 1H), 6.95 (d, <i>J =</i> 8.5 Hz, 1H), 6.91 (d, <i>J =</i> 8.8 Hz, 1H), 6.71 (s, 1H), 2.82–2.71 (m, 2H), 2.52–2.33 (m, 2H), 2.08–2.01 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.60 min, <i>m/z</i> = 439.8, 441.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 5-Bromo-<i>N</i>-(5-(1-cyanocyclopentyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5o</b>)</h3><div id="sec4_33_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Step A: 1-(4-(Benzyloxy)-3-nitrophenyl)cyclopentane-1-carbonitrile</h4><div class="NLM_p last">To a solution of 2-(4-(benzyloxy)-3-nitrophenyl)acetonitrile (134 mg, 0.5 mmol) and 1,4-dibromobutane (72 μL, 0.6 mmol) in DMSO (5 mL) was added carefully NaH (60% in mineral oil, 60 mg, 1.5 mmol), and the mixture was stirred for 16 h. The mixture was diluted with EtOAc/Et<sub>2</sub>O (1:1, 50 mL) and washed with water (3 × 100 mL) and brine (100 mL). Purification by flash chromatography afforded a yellow-brown oil (103 mg, 0.32 mmol, 64% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.89 (d, <i>J =</i> 2.5 Hz, 1H), 7.62 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.48–7.31 (m, 5H), 7.13 (d, <i>J =</i> 8.8 Hz, 1H), 5.26 (s, 2H), 2.55–2.44 (m, 2H), 2.14–1.88 (m, 6H). LC–MS: <i>t</i><sub>R</sub> = 1.96 min, <i>m/z</i> = 323.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_33_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> Step B: 1-(3-Amino-4-hydroxyphenyl)cyclopentane-1-carbonitrile (<b>29c</b>)</h4><div class="NLM_p last">1-(4-(Benzyloxy)-3-nitrophenyl)cyclopentane-1-carbonitrile (103 mg, 0.32 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford a colorless oil (50 mg, 0.25 mmol, 78% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 6.86 (t, <i>J =</i> 1.4 Hz, 1H), 6.68 (d, <i>J =</i> 1.4 Hz, 2H), 2.44–2.29 (m, 2H), 2.13–2.00 (m, 2H), 2.00–1.89 (m, 4H). ESI <i>m/z</i> = 203.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_33_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Step C: 5-Bromo-<i>N</i>-(5-(1-cyanocyclopentyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5o</b>)</h4><div class="NLM_p last">1-(3-Amino-4-hydroxyphenyl)cyclopentane-1-carbonitrile (<b>29c</b>; 50 mg, 0.25 mmol) was reacted with <b>18a</b> (41 mg, 0.15 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (12 mg, 0.04 mmol, 11% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.61 (d, <i>J =</i> 2.5 Hz, 1H), 7.50 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.17 (dd, <i>J =</i> 8.5, 2.3 Hz, 1H), 6.98 (d, <i>J =</i> 2.3 Hz, 2H), 6.90 (d, <i>J =</i> 5.2 Hz, 1H), 6.87 (d, <i>J =</i> 5.2 Hz, 1H), 2.40–2.26 (m, 2H), 2.01–1.81 (m, 6H). LC–MS: <i>t</i><sub>R</sub> = 1.65 min, <i>m/z</i> = 453.8, 455.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> 5-Bromo-<i>N</i>-(5-(1-cyanocyclohexyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5p</b>)</h3><div id="sec4_34_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Step A: 1-(4-(Benzyloxy)-3-nitrophenyl)cyclohexane-1-carbonitrile</h4><div class="NLM_p last">To a solution of 2-(4-(benzyloxy)-3-nitrophenyl)acetonitrile (134 mg, 0.5 mmol) and 1,5-dibromopentane (82 μL, 0.6 mmol) in DMSO (5 mL) was added carefully NaH (60% in mineral oil, 60 mg, 1.5 mmol), and the mixture was stirred for 16 h. The mixture was diluted with EtOAc/Et<sub>2</sub>O (1:1, 50 mL) and washed with water (3 × 100 mL) and brine (100 mL). Purification by flash chromatography afforded a yellow-brown oil (91 mg, 0.27 mmol, 54% yield). LC–MS: <i>t</i><sub>R</sub> = 1.98 min, <i>m/z</i> = 337.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_34_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Step B: 1-(3-Amino-4-hydroxyphenyl)cyclohexane-1-carbonitrile (<b>29d</b>)</h4><div class="NLM_p last">1-(4-(Benzyloxy)-3-nitrophenyl)cyclohexane-1-carbonitrile (91 mg, 0.27 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford a colorless oil (39 mg, 0.18 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 6.91 (d, <i>J =</i> 1.9 Hz, 1H), 6.72 (d, <i>J =</i> 1.9 Hz, 1H), 6.71 (s, 1H), 2.12–2.02 (m, 3H), 1.92–1.69 (m, 6H), 1.43–1.26 (m, 1H). ESI <i>m/z</i> = 217.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_34_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Step C: 5-Bromo-<i>N</i>-(5-(1-cyanocyclohexyl)-2-hydroxyphenyl)-2-hydroxybenzenesulfonamide (<b>5p</b>)</h4><div class="NLM_p last">1-(3-Amino-4-hydroxyphenyl)cyclohexane-1-carbonitrile (<b>29d</b>; 39 mg, 0.18 mmol) was reacted with <b>18a</b> (54 mg, 0.20 mmol) following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (24 mg, 0.05 mmol, 30% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.79 (d, <i>J =</i> 2.5 Hz, 1H), 7.51 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.37 (d, <i>J =</i> 2.4 Hz, 1H), 7.07 (dd, <i>J =</i> 8.5, 2.4 Hz, 1H), 6.86 (d, <i>J =</i> 8.8 Hz, 1H), 6.77 (d, <i>J =</i> 8.5 Hz, 1H), 1.98 (dt, <i>J =</i> 12.8, 1.9 Hz, 2H), 1.91–1.63 (m, 9H), 1.43–1.27 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.72 min, <i>m/z</i> = 467.8, 469.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-chlorobenzoic Acid (<b>6a</b>)</h3><div class="NLM_p last">Methyl 3-amino-5-chlorobenzoate (143 mg, 0.5 mmol) was reacted with <b>18a</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, which without purification was hydrolyzed following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a>, to afford the title compound as a colorless solid (86 mg, 0.21 mmol, 42% over 2 steps). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.83 (d, <i>J =</i> 2.5 Hz, 1H), 7.65 (d, <i>J =</i> 2.6 Hz, 1H), 7.55 (d, <i>J =</i> 2.6 Hz, 1H), 7.53 (dd, <i>J =</i> 8.7, 2.5 Hz, 1H), 7.42 (d, <i>J =</i> 2.6 Hz, 1H), 6.87 (d, <i>J =</i> 8.7 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.31 min, <i>m/z</i> = 427.8, 429.7 [M + Na]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 3-((5-Bromo-2-methoxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6b</b>)</h3><div id="sec4_36_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> Step A: Methyl 3-((5-Bromo-2-methoxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-chloro-2-hydroxybenzoate (202 mg, 1.0 mmol) was reacted with <b>16a</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (370 mg, 0.82 mmol, 82% yield). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 7.81–7.76 (m, 2H), 7.55 (d, <i>J =</i> 2.7 Hz, 1H), 7.49 (d, <i>J =</i> 2.6 Hz, 1H), 7.19 (dt, <i>J =</i> 8.6, 1.0 Hz, 1H), 3.88 (s, 3H), 3.76 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.77 min, <i>m/z</i> = 450, 452 [M + H]<sup>+</sup>.</div></div><div id="sec4_36_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Step B: 3-((5-Bromo-2-methoxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6b</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-methoxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate (370 mg, 0.82 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (260 mg, 0.59 mmol, 72% yield). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 9.52 (s, 1H), 7.80–7.75 (m, 2H), 7.52 (d, <i>J =</i> 2.7 Hz, 1H), 7.43 (d, <i>J =</i> 2.7 Hz, 1H), 7.18 (d, <i>J =</i> 9.6 Hz, 1H), 3.77 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.55 min, <i>m/z</i> = 436.0, 438.0 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>14</sub>H<sub>11</sub>BrClNNaO<sub>6</sub>S, 457.9071; found, 457.9075.</div></div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6c</b>)</h3><div id="sec4_37_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Step A: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-chloro-2-hydroxybenzoate (605 mg, 3.0 mmol) was reacted with <b>18a</b> following General Procedure A: Sulfonamide Coupling to afford a colorless solid (918 mg, 2.1 mmol, 70% yield). LC–MS: <i>t</i><sub>R</sub> = 1.15 min, <i>m/z</i> = 437.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_37_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Step B: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6c</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate (918 mg, 2.10 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (425 mg, 1.0 mmol, 48% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.81 (d, <i>J =</i> 2.5 Hz, 1H), 7.62 (d, <i>J =</i> 2.6 Hz, 1H), 7.55 (d, <i>J =</i> 2.6 Hz, 1H), 7.53 (dd, <i>J =</i> 8.7, 2.5 Hz, 1H), 6.87 (d, <i>J =</i> 8.7 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.53 min, <i>m/z</i> = 421.7, 423.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>13</sub>H<sub>9</sub>BrClNNaO<sub>6</sub>S, 443.8915; found, 443.8909.</div></div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6d</b>)</h3><div id="sec4_38_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> Step A: 5-Bromo-3-chloro-2-hydroxybenzenesulfonyl Chloride (<b>18b</b>)</h4><div class="NLM_p last">Chlorosulfonic acid (4.66 mL, 70 mmol) was cooled to 0 °C, and 2-chloro-4-bromophenol (2.08 g, 10 mmol) was added portion wise and stirred for 24 h. The mixture was carefully poured over a slurry of ice, DCM, and brine and extracted with DCM. Purification by flash chromatography afforded the title compound as a colorless solid (2.19 g, 7.15 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.92 (d, <i>J =</i> 2.4 Hz, 1H), 7.86 (d, <i>J =</i> 2.4 Hz, 1H).</div></div><div id="sec4_38_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Step B: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-chloro-2-hydroxybenzoate (192 mg, 0.95 mmol) was reacted with 5-bromo-3-chloro-2-hydroxybenzenesulfonyl chloride (<b>18b)</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (374 mg, 0.91 mmol, 91% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.03 (s, 1H), 7.71–7.68 (m, 2H), 7.64 (d, <i>J =</i> 2.5 Hz, 1H), 7.62 (d, <i>J =</i> 2.5 Hz, 1H), 3.95 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.22 min, <i>m/z</i> = 471.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_38_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> Step C: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic Acid (<b>6d</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate (298 mg, 0.63 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (231 mg, 0.51 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.76 (s, 2H), 7.63 (d, <i>J =</i> 2.5 Hz, 1H), 7.61 (d, <i>J =</i> 2.6 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.68 min, <i>m/z</i> = 457.6 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoic Acid (<b>6e</b>)</h3><div id="sec4_39_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> Step A: Methyl 5-Bromo-2-hydroxy-3-nitrobenzoate</h4><div class="NLM_p last">Methyl 2-hydroxy-4-bromobenzoate (9.24 g, 40 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford the nitrated product as a crude yellow solid (10.3 g, 37.3 mmol, 93% yield), which was used without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.27 (d, <i>J =</i> 2.5 Hz, 1H), 8.24 (d, <i>J =</i> 2.5 Hz, 1H), 4.03 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.02 min, <i>m/z</i> = 276, 278 [M + H]<sup>+</sup>.</div></div><div id="sec4_39_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> Step B: Methyl 2-(Benzyloxy)-5-bromo-3-nitrobenzoate</h4><div class="NLM_p last">Methyl 5-bromo-2-hydroxy-3-nitrobenzoate (8.9 g, 32.2 mmol) was reacted following <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure G: Benzyl Protection">General Procedure G: Benzyl Protection</a>. Purification by flash column chromatography afforded a yellow solid (10.6 g, 28.8 mmol, 89% yield).</div></div><div id="sec4_39_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Step C: Methyl 2-(Benzyloxy)-5-cyclopropyl-3-nitrobenzoate</h4><div class="NLM_p last">Methyl 2-(benzyloxy)-5-bromo-3-nitrobenzoate (1.1 g, 3.0 mmol), cyclopropylboronic acid (515 mg, 6 mmol), Pd(OAc)<sub>2</sub> (33.7 mg, 0.15 mmol), PCy<sub>3</sub>·HBF<sub>4</sub> (110 mg, 0.30 mmol), and K<sub>3</sub>PO<sub>4</sub> (1.59 g, 7.5 mmol) were taken in toluene/H<sub>2</sub>O (10:1, 15 mL) and heated to 80 °C for 16 h. The cooled mixture was diluted with EtOAc and washed with water and brine. Purification by flash chromatography afforded a brown oil (895 mg, 2.73 mmol, 91% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.77 (d, <i>J =</i> 2.4 Hz, 1H), 7.64 (d, <i>J =</i> 2.5 Hz, 1H), 7.52–7.47 (m, 2H), 7.46–7.35 (m, 3H), 3.91 (s, 3H), 2.03–1.94 (m, 1H), 1.13–1.07 (m, 2H), 0.83–0.74 (m, 2H).</div></div><div id="sec4_39_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Step D: Methyl 3-Amino-5-cyclopropyl-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 2-(benzyloxy)-5-cyclopropyl-3-nitrobenzoate (895 mg, 2.73 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford an off-white solid (495 mg, 2.39 mmol, 88% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.78 (s, 1H), 7.15 (d, <i>J =</i> 2.1 Hz, 1H), 6.94 (d, <i>J =</i> 2.1 Hz, 1H), 3.96 (s, 3H), 1.83 (tt, <i>J =</i> 8.4, 5.1 Hz, 1H), 0.96–0.86 (m, 2H), 0.68–0.59 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.28 min, <i>m/z</i> = 208.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_39_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Step E: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-cyclopropyl-2-hydroxybenzoate (207 mg, 1.0 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a cream solid (395 mg, 0.89 mmol, 89% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.76 (d, <i>J =</i> 2.5 Hz, 1H), 7.51 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.36 (d, <i>J =</i> 2.3 Hz, 1H), 7.31 (d, <i>J =</i> 2.3 Hz, 1H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H), 3.93 (s, 3H), 1.84 (tt, <i>J =</i> 8.4, 5.1 Hz, 1H), 0.98–0.90 (m, 2H), 0.54 (dt, <i>J =</i> 6.5, 4.7 Hz, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.67 min, <i>m/z</i> = 442.1, 444.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_39_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Step F: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoic Acid (<b>6e</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoate (44 mg, 0.10 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (33 mg, 0.078 mmol, 78% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 11.25 (s, 1H), 8.85 (s, 1H), 7.68 (d, <i>J =</i> 2.6 Hz, 1H), 7.59 (dd, <i>J =</i> 8.7, 2.6 Hz, 1H), 7.30 (d, <i>J =</i> 2.3 Hz, 1H), 7.06 (d, <i>J =</i> 2.3 Hz, 1H), 6.94 (d, <i>J =</i> 8.8 Hz, 1H), 1.84 (tt, <i>J =</i> 8.4, 5.1 Hz, 1H), 0.97–0.77 (m, 2H), 0.51–0.37 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.51 min, <i>m/z</i> = 427.8, 429.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>15</sub>BrNO<sub>6</sub>S, 427.9798; found, 427.9796.</div></div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoic Acid (<b>6f</b>)</h3><div id="sec4_40_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Step A: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-cyclopropyl-2-hydroxybenzoate (41 mg, 0.20 mmol) and <b>18b</b> were combined following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (76 mg, 0.16 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 10.62 (s, 1H), 9.42 (s, 1H), 7.93 (d, <i>J =</i> 2.5 Hz, 1H), 7.67 (d, <i>J =</i> 2.4 Hz, 1H), 7.34 (d, <i>J =</i> 2.3 Hz, 1H), 7.09 (d, <i>J =</i> 2.3 Hz, 1H), 3.88 (s, 3H), 1.89 (ddt, <i>J =</i> 13.5, 8.5, 4.3 Hz, 1H), 1.03–0.76 (m, 2H), 0.61–0.32 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.76 min, <i>m/z</i> = 475.8, 477.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_40_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoic Acid (<b>6f</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-2-hydroxybenzoate (24 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (19 mg, 82% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.61 (dd, <i>J =</i> 2.4, 1.7 Hz, 1H), 7.53 (dd, <i>J =</i> 10.1, 2.4 Hz, 1H), 7.39 (d, <i>J =</i> 2.1 Hz, 1H), 7.30 (d, <i>J =</i> 2.3 Hz, 1H), 1.85 (tt, <i>J =</i> 8.4, 5.1 Hz, 1H), 0.98–0.90 (m, 2H), 0.59–0.52 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.53 min, <i>m/z</i> = 462.8, 464.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoic Acid (<b>6g</b>)</h3><div id="sec4_41_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> Step A: Methyl 2-Fluoro-6-hydroxybenzoate (<b>41</b>)</h4><div class="NLM_p last">2-Fluoro-6-hydroxybenzoic acid (<b>40</b>; 5.0 g, 32 mmol) was dissolved in THF (160 mL) and MeOH (32 mL) and cooled to 0 °C. TMS diazomethane [2.0 M in hexanes] (17.6 mL, 35.2 mmol) was added over 20 min, and the mixture was stirred for 3 h at 0 °C. The mixture was concentrated, redissolved in EtOAc, and washed with water and brine. Purification by flash chromatography afforded a colorless liquid (3.8 g, 22.3 mmol, 70% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.26 (s, 1H), 7.40–7.36 (m, 1H), 6.79 (br d, 1H), 6.63–6.58 (m, 1H), 3.99 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −105.1. LC–MS: <i>t</i><sub>R</sub> = 1.33 min, <i>m/z</i> = 171.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_41_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Step B: Methyl 3-Bromo-2-fluoro-6-hydroxybenzoate (<b>42</b>)</h4><div class="NLM_p last"><b>41</b> (3.8 g, 22.3 mmol) was dissolved in MeCN (144 mL) and cooled to −10 °C in a salt/ice bath. HBF<sub>4</sub>·Et<sub>2</sub>O (3.37 mL, 24.6 mmol) was added, followed by <i>N</i>-bromosuccinimide (4.8 g, 26.8 mmol) portion wise, and then stirred for 1 h at −10 °C. The mixture was concentrated in vacuo, redissolved in EtOAc, washed with sat. aq. Na<sub>2</sub>S<sub>2</sub>O<sub>8</sub>, sat. aq. NaCl, and then dried (MgSO<sub>4</sub>). Purification by flash chromatography affords a cream solid (2.49 g, 10 mmol, 45% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.26 (s, 1H), 7.58 (dd, <i>J</i> = 9.1, 7.5 Hz, 1H), 6.76 (dt, <i>J</i> = 7.5, 0.9 Hz), 4.01 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −86.7. LC–MS: <i>t</i><sub>R</sub> = 1.59 min, <i>m/z</i> = 249, 251 [M + H]<sup>+</sup>. Note: The material contains dibrominated byproduct, which is inseparable by chromatography, but is readily removed after Step C.</div></div><div id="sec4_41_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> Step C: Methyl 3-Bromo-2-fluoro-6-hydroxy-5-nitrobenzoate (<b>43</b>)</h4><div class="NLM_p last"><b>42</b> (2.49 g, 10 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford a pale yellow solid (1.92 mg, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.44 (d, <i>J =</i> 7.0 Hz, 1H), 4.03 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −88.1. LC–MS: <i>t</i><sub>R</sub> = 1.55 min, <i>m/z</i> = 294, 296 [M + H]<sup>+</sup>.</div></div><div id="sec4_41_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Step D: Methyl 2-(Benzyloxy)-5-bromo-6-fluoro-3-nitrobenzoate (<b>44</b>)</h4><div class="NLM_p last"><b>43</b> (588 mg, 2.0 mmol) was reacted following <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure G: Benzyl Protection">General Procedure G: Benzyl Protection</a> to afford a pale yellow solid (615 mg, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.25 (d, <i>J =</i> 7.0 Hz, 1H), 7.42–7.38 (m, 5H), 5.16 (s, 2H), 3.88 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −86.8. LC–MS: <i>t</i><sub>R</sub> = 1.75 min, <i>m/z</i> = 406, 408 [M + Na]<sup>+</sup>.</div></div><div id="sec4_41_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Step E: Methyl 2-(Benzyloxy)-5-cyclopropyl-6-fluoro-3-nitrobenzoate</h4><div class="NLM_p last">Methyl 2-(benzyloxy)-5-bromo-6-fluoro-3-nitrobenzoate (103 mg, 1.2 mmol), cyclopropylboronic acid (155 mg, 1.8 mmol), Pd(OAc)<sub>2</sub> (13.5 mg, 0.06 mmol), PCy<sub>3</sub>·HBF<sub>4</sub> (44.2 mg, 0.12 mmol), and K<sub>3</sub>PO<sub>4</sub> (637 mg, 3.0 mmol) were taken in toluene/H<sub>2</sub>O (10:1, 6 mL) and heated to 80 °C for 16 h. The cooled mixture was diluted with EtOAc and washed with water and brine. Purification by flash chromatography afforded a cream solid (150 mg, 0.43 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.59 (d, <i>J =</i> 7.6 Hz, 1H), 7.45–7.37 (m, 5H), 5.12 (s, 2H), 3.88 (s, 3H), 2.09–2.06 (m, 1H), 1.11–1.07 (m, 2H), 0.81–0.77 (m, 2H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −108.7. LC–MS: <i>t</i><sub>R</sub> = 1.85 min, <i>m/z</i> = 363.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_41_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> Step F: Methyl 3-Amino-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (<b>45</b>)</h4><div class="NLM_p last">Methyl 2-(benzyloxy)-5-cyclopropyl-6-fluoro-3-nitrobenzoate (145 mg, 0.42 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford an off-white solid (66 mg, 70% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.17 (s, 1H), 6.46 (d, <i>J =</i> 7.1 Hz, 1H), 4.01 (s, 3H), 2.12–1.92 (m, 1H), 1.07–0.82 (m, 2H), 0.70–0.50 (m, 2H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −125.0. LC–MS: <i>t</i><sub>R</sub> = 1.04 min, <i>m/z</i> = 226.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_41_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Step G: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate</h4><div class="NLM_p last"><b>45</b> (45 mg, 0.2 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a pale brown solid (69 mg, 0.15 mmol, 75% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 11.34 (s, 1H), 10.13 (s, 1H), 9.00 (s, 1H), 7.61 (dd, <i>J =</i> 8.7, 2.6 Hz, 1H), 7.57 (d, <i>J =</i> 2.5 Hz, 1H), 6.97 (d, <i>J =</i> 8.7 Hz, 1H), 6.62 (d, <i>J =</i> 8.0 Hz, 1H), 3.82 (s, 3H), 1.90–1.78 (m, 1H), 0.95–0.77 (m, 2H), 0.39–0.24 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.82 min, <i>m/z</i> = 460, 462 [M + H]<sup>+</sup>.</div></div><div id="sec4_41_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Step H: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoic Acid (<b>6g</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (23 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_17" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (9 mg, 40% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.71 (d, <i>J =</i> 2.5 Hz, 1H), 7.53 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.13 (d, <i>J =</i> 7.4 Hz, 1H), 6.87 (d, <i>J =</i> 8.8 Hz, 1H), 3.96 (s, 3H), 2.01–1.93 (m, 1H), 0.99–0.91 (m, 2H), 0.59–0.51 (m, 2H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −116.6. LC–MS: <i>t</i><sub>R</sub> = 1.53 min, <i>m/z</i> = 467.7, 469.7 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoic Acid (<b>6h</b>)</h3><div id="sec4_42_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> Step A: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate</h4><div class="NLM_p last">Methyl 3-amino-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (45 mg, 0.2 mmol) and <b>18b</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a pale brown solid (69 mg, 0.15 mmol, 75% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.73 (d, <i>J =</i> 2.4 Hz, 1H), 7.63 (d, <i>J =</i> 2.4 Hz, 1H), 7.14 (d, <i>J =</i> 7.4 Hz, 1H), 3.97 (s, 3H), 2.03–1.94 (m, 1H), 1.01–0.93 (m, 2H), 0.63–0.55 (m, 2H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −115.5. LC–MS: <i>t</i><sub>R</sub> = 1.94 min, <i>m/z</i> = 494.2, 496.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_42_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoic Acid (<b>6h</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (23 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (10 mg, 42% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 7.94 (d, <i>J =</i> 2.5 Hz, 1H), 7.61 (d, <i>J =</i> 2.5 Hz, 1H), 6.78 (d, <i>J =</i> 7.7 Hz, 1H), 1.89 (td, <i>J =</i> 8.4, 4.2 Hz, 1H), 1.00–0.83 (m, 2H), 0.48–0.37 (m, 2H). <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>F</sub> −116.3. LC–MS: <i>t</i><sub>R</sub> = 1.74 min, <i>m/z</i> = 477.7, 479.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>16</sub>H<sub>12</sub>BrClFNNaO<sub>6</sub>S, 501.9133; found, 501.9121.</div></div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(trifluoromethoxy)benzoic Acid (<b>6i</b>)</h3><div id="sec4_43_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> Step A: Methyl 3-Bromo-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">3-Bromo-5-(trifluoromethoxy)benzoic acid (2.0 g, 7.0 mmol) was dissolved in THF (35 mL) and MeOH (7 mL) and cooled to 0 °C. TMS diazomethane [2.0 M in Et<sub>2</sub>O] (3.85 mL, 7.7 mmol) was added dropwise, and then the mixture was stirred for 16 h. The mixture was concentrated and purified by flash chromatography to afford a colorless solid (1.82 g, 6.09 mmol, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.12 (t, <i>J =</i> 1.6 Hz, 1H), 7.83 (t, <i>J =</i> 2.4 Hz, 1H), 7.57 (t, <i>J =</i> 1.6 Hz, 1H), 3.95 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −57.98. ESI <i>m/z</i> = 300.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_43_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> Step B: Methyl 3-Amino-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 3-bromo-5-(trifluoromethoxy)benzoate (1.79 g, 5.98 mmol), <i>tert</i>-butyl carbamate (841 mg, 7.18 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (110 mg, 0.12 mmol), xantphos (208 mg, 0.36 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (5.85 g, 17.9 mmol) were combined in toluene (24 mL) in a sealed tube and stirred at 90 °C for 16 h. The mixture was filtered through celite and concentrated, and the residue was redissolved in DCM (50 mL). TFA (10 mL) was added, and the mixture was stirred at r.t. for 2 h. The mixture was washed with sat. aq. Na<sub>2</sub>CO<sub>3</sub>, concentrated, and purified by flash chromatography to afford a brown solid (1.18 g, 5.02 mmol, 84% yield). ESI <i>m/z</i> = 236.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_43_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> Step C: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 3-amino-5-(trifluoromethoxy)benzoate (118 mg, 0.5 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a pale brown solid (179 mg, 0.38 mmol, 76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.72–7.69 (m, 1H), 7.68 (d, <i>J =</i> 2.5 Hz, 1H), 7.64 (t, <i>J =</i> 1.7 Hz, 1H), 7.52 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.27 (s, 1H), 6.89 (d, <i>J =</i> 8.9 Hz, 1H), 3.93 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −58.0. LC–MS: <i>t</i><sub>R</sub> = 1.11 min, <i>m/z</i> = 486.9, 488.9 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_43_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> Step D: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(trifluoromethoxy)benzoic Acid (<b>6i</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(trifluoromethoxy)benzoate (91 mg, 0.19 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (64 mg, 0.14 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.86 (d, <i>J =</i> 2.5 Hz, 1H), 7.79 (dd, <i>J =</i> 2.1, 1.4 Hz, 1H), 7.53–7.49 (m, 2H), 7.35 (dq, <i>J =</i> 2.1, 1.1 Hz, 1H), 6.85 (d, <i>J =</i> 8.7 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −59.7. LC–MS: <i>t</i><sub>R</sub> = 1.34 min, <i>m/z</i> = 477.7, 479.7 [M + Na]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i127" class="anchor-spacer"></div><h3 class="article-section__title" id="_i127"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoic Acid (<b>6j</b>)</h3><div id="sec4_44_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> Step A: Methyl 2-Hydroxy-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">2-Hydroxy-5-(trifluoromethoxy)benzoic acid (2.0 g, 9.00 mmol) was dissolved in DCM (30 mL) and MeOH (30 mL) and cooled to 0 °C. To the flask was added EDCI (2.59 g, 13.5 mmol), followed by DMAP (220.0 mg, 1.80 mmol, 0.2 equiv). The reaction was stirred for 16 h and warmed to r.t.. The mixture was washed with water and dried (Na<sub>2</sub>SO<sub>4</sub>). Purification by flash chromatography afforded the title compound (1.35 g, 5.74 mmol, 63% yield). LC–MS: <i>t</i><sub>R</sub> = 1.62 min, <i>m/z</i> = 237.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_44_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> Step B. Methyl 2-Hydroxy-3-nitro-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 2-hydroxy-5-(trifluoromethoxy)benzoate (1.36 g, 5.74 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a>. Purification by flash chromatography afforded the title compound (1.61 g, quant.). LC–MS: <i>t</i><sub>R</sub> = 1.51 min, <i>m/z</i> = 252.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_44_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> Step C: Methyl 3-Amino-2-hydroxy-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 2-hydroxy-3-nitro-5-(trifluoromethoxy)benzoate (1.61 g, 5.74 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>. The crude product was used for the next step without further purification (1.40 g, 5.59 mmol, 97% over 2 steps). LC–MS: <i>t</i><sub>R</sub> = 1.42 min, <i>m/z</i> = 251.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_44_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> Step D: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 3-amino-2-hydroxy-5-(trifluoromethoxy)benzoate (1.26 g, 5.0 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a pale brown solid (1.02 g, 2.2 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.10 (s, 1H), 8.68 (s, 1H), 7.68 (d, <i>J =</i> 2.4 Hz, 1H), 7.64–7.60 (m, 1H), 7.51–7.46 (m, 3H), 6.88 (d, <i>J =</i> 8.9 Hz, 1H), 3.96 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> −58.6. LC–MS: <i>t</i><sub>R</sub> = 1.20 min, <i>m/z</i> = 485.8, 487.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_44_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> Step E: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoic Acid (<b>6j</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoate (1.02 g, 2.09 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (575 mg, 1.22 mmol, 58% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.82 (d, <i>J =</i> 2.5 Hz, 1H), 7.58–7.55 (m, 1H), 7.54 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.45–7.41 (m, 1H), 6.86 (d, <i>J =</i> 8.8 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −60.3. LC–MS: <i>t</i><sub>R</sub> = 1.57 min, <i>m/z</i> = 471.8, 473.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoic Acid (<b>6k</b>)</h3><div id="sec4_45_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> Step A: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoate</h4><div class="NLM_p last">Methyl 3-amino-2-hydroxy-5-(trifluoromethoxy)benzoate (53.4 mg, 0.21 mmol) was reacted with <b>18b</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a solid (26.4 mg, 31% yield). LC–MS: <i>t</i><sub>R</sub> = 1.80 min, <i>m/z</i> = 521.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_45_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> Step B. 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoic Acid (<b>6k</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(trifluoromethoxy)benzoate (10.5 mg, 0.02 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (7.0 mg, 68% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.74 (q, <i>J =</i> 2.4 Hz, 2H), 7.54 (d, <i>J =</i> 2.3 Hz, 1H), 7.47 (d, <i>J =</i> 2.2 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ −60.3. LC–MS: <i>t</i><sub>R</sub> = 1.66 min, <i>m/z</i> = 507.6 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i136" class="anchor-spacer"></div><h3 class="article-section__title" id="_i136"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoic Acid (<b>6l</b>)</h3><div id="sec4_46_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> Step A: Methyl 3-Bromo-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">3-Bromo-5-(pentafluorosulfanyl)benzoic acid (1.05 g, 3.21 mmol) was dissolved in THF (16 mL) and MeOH (3.2 mL) and cooled to 0 °C. TMS diazomethane [2.0 M in Et<sub>2</sub>O] (1.77 mL, 3.53 mmol) was added dropwise, and then the mixture was stirred for 16 h. The mixture was concentrated and purified by flash chromatography to afford a colorless solid (1.02 mg, 2.97 mmol, 93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.36 (dd, <i>J =</i> 2.1, 1.3 Hz, 1H), 8.33 (t, <i>J =</i> 1.6 Hz, 1H), 8.07 (t, <i>J =</i> 2.0 Hz, 1H), 3.98 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 81.67 (d, <i>J =</i> 151 Hz, 1F), 62.94 (d, <i>J =</i> 151 Hz, 4F).</div></div><div id="sec4_46_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> Step B: Methyl 3-Amino-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-bromo-5-(pentafluorosulfanyl)benzoate (890 mg, 2.61 mmol), <i>tert</i>-butyl carbamate (367 mg, 3.13 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (48 mg, 0.05 mmol), xantphos (91 mg, 0.16 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (2.55 g, 7.83 mmol) were combined in toluene (10 mL) in a sealed tube and stirred at 90 °C for 16 h. The mixture was filtered through celite and concentrated, and the residue was redissolved in DCM (25 mL). TFA (5 mL) was added, and the mixture was stirred at r.t. for 2 h. The mixture was washed with sat. aq. Na<sub>2</sub>CO<sub>3</sub>, concentrated, and purified by flash chromatography to afford a brown solid (585 mg, 2.11 mmol, 81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.82–7.77 (m, 1H), 7.49–7.47 (m, 1H), 7.22 (t, <i>J =</i> 2.1 Hz, 1H), 3.95 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 83.9 (p, <i>J =</i> 150 Hz, 1F), 62.47 (d, <i>J =</i> 150 Hz, 4F).</div></div><div id="sec4_46_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> Step C: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-amino-5-(pentafluorosulfanyl)benzoate (139 mg, 0.50 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (184 mg, 0.36 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.69 (d, <i>J =</i> 5.0 Hz, 2H), 8.26 (t, <i>J =</i> 1.7 Hz, 1H), 7.92 (br s, 1H), 7.74–7.69 (m, 2H), 7.56 (dd, <i>J =</i> 8.8, 2.4 Hz, 1H), 6.93 (d, <i>J =</i> 8.8 Hz, 1H), 3.98 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 81.6 (p, <i>J =</i> 151 Hz, 1F), 62.7 (d, <i>J =</i> 151 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.14 min, <i>m/z</i> = 528.8, 530.9 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_46_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> Step D: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoic Acid (<b>6l</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoate (68 mg, 0.13 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (8 mg, 0.016 mmol, 12% yield). <sup>1</sup>H NMR (600 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 8.02–7.98 (m, 2H), 7.87–7.83 (m, 2H), 7.51 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.84 (d, <i>J =</i> 8.7 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 81.4 (p, <i>J =</i> 149 Hz, 1F), 60.9 (d, <i>J =</i> 149 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.40 min, <i>m/z</i> = 519.7, 521.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i141" class="anchor-spacer"></div><h3 class="article-section__title" id="_i141"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoic Acid (<b>6m</b>)</h3><div id="sec4_47_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> Step A: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-amino-5-(pentafluorosulfanyl)benzoate (139 mg, 0.50 mmol) and <b>18b</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a colorless solid (131 mg, 0.24 mmol, 48% yield). LC–MS: <i>t</i><sub>R</sub> = 1.67 min, <i>m/z</i> = 569.5 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_47_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoic Acid (<b>6m</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoate (72 mg, 0.13 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (18 mg, 0.034 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 8.08–8.04 (t, <i>J =</i> 1.9 Hz, 1H), 8.02 (t, <i>J =</i> 1.9 Hz, 1H), 7.87 (d, <i>J =</i> 2.4 Hz, 1H), 7.84 (t, <i>J =</i> 1.9 Hz, 1H), 7.75 (d, <i>J =</i> 2.4 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 81.4 (p, <i>J =</i> 149 Hz, 1F), 60.98 (d, <i>J =</i> 149 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.10 min, <i>m/z</i> = 533.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i144" class="anchor-spacer"></div><h3 class="article-section__title" id="_i144"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>6n</b>)</h3><div id="sec4_48_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> Step A: Phenyl 2-Methoxy-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p">(3-Bromo-4-methoxyphenyl)pentafluorosulfane (<b>34</b>; 939 mg, 3.0 mmol), Pd(OAc)<sub>2</sub> (34 mg, 0.15 mmol), P(<i><sup>t</sup></i>Bu)<sub>3</sub>·HBF<sub>4</sub> (174 mg, 0.60 mmol), and phenol (282 mg, 3.0 mmol) were taken in MeCN (13.7 mL) in a 40 mL reaction vial, and phenyl formate (500 μL) and Et<sub>3</sub>N (1.25 mL, 9.0 mmol) were added. The reaction was run in five parallel reactions. The sealed vials were heated to 90 °C for 18 h combined after filtration through celite, concentrated, redissolved in EtOAc (100 mL), and then washed with H<sub>2</sub>O (100 mL). Purification by flash chromatography afforded a pale brown liquid (4.36 g, 12.3 mmol, 82% yield).</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.41 (d, <i>J</i> = 2.9 Hz, 1H), 7.93 (dd, <i>J</i> = 9.2, 2.9 Hz, 1H), 7.48–7.40 (m, 2H), 7.31–7.26 (m, 2H), 7.07 (d, <i>J</i> = 9.2 Hz, 1H), 6.93 (tt, <i>J</i> = 7.3, 1.1 Hz, 1H), 4.00 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 84.52 (d, <i>J</i> = 150 Hz, 1F), 64.07 (d, <i>J</i> = 150 Hz, 4F). MS (ESI) <i>m/z</i> = 355.2.</div></div><div id="sec4_48_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> Step B: Phenyl 2-Hydroxy-5-(pentafluorosulfanyl)benzoate (<b>35</b>)</h4><div class="NLM_p last">Phenyl 2-methoxy-5-(pentafluorosulfanyl)benzoate (2.80 g) was reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford a colorless solid (1.81 g, 5.31 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.88 (s, 1H), 8.50 (d, <i>J</i> = 2.8 Hz, 1H), 7.92 (dd, <i>J</i> = 9.2, 2.8 Hz, 1H), 7.53–7.45 (m, 2H), 7.40–7.31 (m, 1H), 7.29–7.20 (m, 3H), 7.11 (d, <i>J</i> = 9.2 Hz, 1H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ 84.58 (d, <i>J</i> = 151 Hz, 1F), 63.90 (d, <i>J</i> = 151 Hz, 4F). MS (ESI) <i>m/z</i> = 341.3.</div></div><div id="sec4_48_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> Step C: 2-Hydroxy-5-(pentafluorosulfanyl)benzoic Acid</h4><div class="NLM_p last">Phenyl 2-hydroxy-5-(pentafluorosulfanyl)benzoate (<b>35</b>; 1.81 g, 5.32 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford the title compound (1.15 g, 4.35 mmol, 82% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 8.27 (d, <i>J</i> = 2.9 Hz, 1H), 7.91 (dd, <i>J</i> = 9.2, 2.9 Hz, 1H), 7.07 (d, <i>J</i> = 9.2 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 83.68 (d, <i>J</i> = 149 Hz, 1F), 62.35 (d, <i>J</i> = 149 Hz, 4F).</div></div><div id="sec4_48_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> Step D: 2-Hydroxy-3-nitro-5-(pentafluorosulfanyl)benzoic Acid</h4><div class="NLM_p last">2-Hydroxy-5-(pentafluorosulfanyl)benzoic acid (300 mg, 1.14 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford the title compound (159 mg, 0.51 mmol, 45% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 8.60 (d, <i>J</i> = 2.8 Hz, 4H), 8.53 (d, <i>J</i> = 2.8 Hz, 4H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 80.9 (p, <i>J</i> = 149 Hz), 62.4 (d, <i>J</i> = 149 Hz).</div></div><div id="sec4_48_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> Step E: Methyl 2-Hydroxy-3-nitro-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">To a mixture containing 2-hydroxy-3-nitro-5-(pentafluorosulfanyl)benzoic acid (60 mg, 0.19 mmol) in MeOH (1 mL), two drops of H<sub>2</sub>SO<sub>4</sub> were added. The reaction mixture was heated to reflux and stirred overnight. The cooled mixture was concentrated, and the residue was dissolved in DCM and washed with water. The organic phase was concentrated to afford the crude title compound, which was used without purification (60 mg, 0.19 mmol, 96% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.55 (d, <i>J</i> = 2.9 Hz, 1H), 8.51 (d, <i>J</i> = 2.9 Hz, 1H), 4.09 (s, 3H).</div></div><div id="sec4_48_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> Step F: Methyl 3-Amino-2-hydroxy-5-(pentafluorosulfanyl)benzoate (<b>36</b>)</h4><div class="NLM_p last">Methyl 2-hydroxy-3-nitro-5-(pentafluorosulfanyl)benzoate (223 mg, 0.69 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford the title compound (170 mg, 0.58 mmol, 84% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 11.26 (s, 1H), 7.64 (d, <i>J</i> = 2.6 Hz, 1H), 7.20 (d, <i>J</i> = 2.6 Hz, 1H), 3.98 (br s, H). <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>): δ 85.67 (p, <i>J</i> = 151 Hz, 1F), 63.71 (d, <i>J</i> = 150 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.06 min, <i>m/z</i> = 294.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_48_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> Step G: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last"><b>36</b> (50 mg, 0.17 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a colorless solid (54 mg, 0.10 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.06 (d, <i>J</i> = 2.7 Hz, 1H), 8.03 (d, <i>J</i> = 2.7 Hz, 1H), 7.70 (d, <i>J</i> = 2.5 Hz, 1H), 7.48 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.86 (d, <i>J</i> = 8.8 Hz, 1H), 3.99 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 83.5 (p, <i>J</i> = 151 Hz, 1F), 63.7 (d, <i>J</i> = 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.24 min, <i>m/z</i> = 527.9, 529.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_48_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> Step H: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>6n</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (27 mg, 0.046 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (6 mg, 0.12 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 8.02 (d, <i>J</i> = 2.7 Hz, 1H), 7.97 (d, <i>J</i> = 2.7 Hz, 1H), 7.81 (d, <i>J</i> = 2.5 Hz, 1H), 7.52 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.85 (d, <i>J</i> = 8.8 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 83.0 (p, <i>J</i> = 149 Hz, 1F), 62.2 (d, <i>J</i> = 149 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.07 min, <i>m/z</i> = 537.3 [M + Na]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i153" class="anchor-spacer"></div><h3 class="article-section__title" id="_i153"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>6o</b>)</h3><div id="sec4_49_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> Step A: Methyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-amino-2-hydroxy-5-(pentafluorosulfanyl)benzoate (50 mg, 0.17 mmol) and <b>18b</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (21 mg, 0.04 mmol, 22% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.72–8.67 (m, 1H), 8.11 (d, <i>J</i> = 2.6 Hz, 1H), 8.02 (d, <i>J</i> = 2.6 Hz, 1H), 7.74 (d, <i>J</i> = 2.4 Hz, 1H), 7.63 (d, <i>J</i> = 2.4 Hz, 1H), 3.99 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 83.3 (p, <i>J =</i> 151 Hz, 1F), 63.7 (d, <i>J</i> = 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.19 min, <i>m/z</i> = 562.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_49_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>6o</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (20 mg, 0.04 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford the title compound as a colorless solid (8 mg, 0.015 mmol, 41% yield). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ<sub>H</sub> 8.12 (d, <i>J</i> = 2.7 Hz, 1H), 8.09 (d, <i>J</i> = 2.7 Hz, 1H), 7.86 (d, <i>J</i> = 2.4 Hz, 1H), 7.81 (d, <i>J</i> = 2.4 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ 82.94 (p, <i>J</i> = 149 Hz), 62.21 (d, <i>J</i> = 149 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.111 min, <i>m/z</i> = 548.1, 550.1 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i156" class="anchor-spacer"></div><h3 class="article-section__title" id="_i156"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)benzoic Acid (<b>6p</b>)</h3><div id="sec4_50_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i157" class="anchor-spacer"></div><h4 class="article-section__title" id="_i157"> Step A: Methyl 3-Bromo-5-(cyanomethyl)benzoate</h4><div class="NLM_p last">To a solution of methyl 3-bromo-5-methylbenzoate, <b>52</b> (10.0 g, 44 mmol), in CHCl<sub>3</sub> (200 mL) was added NBS (8.6 g, 48 mmol) followed by AIBN (730 mg, 4.4 mmol), and the mixture was heated to reflux overnight. Upon cooling, the mixture was filtered through celite and concentrated under reduced pressure. To the crude intermediate (6 g, 19.5 mmol) in THF (100 mL) was added TMSCN (1.93 g 19.5 mmol) and stirred for 30 min, and then TBAF (1.0 M in THF, 19.5 mL) was added. The mixture was stirred at r.t. for 16 h, concentrated, and then purified by flash chromatography to afford the title compound (2.0 g, 7.9 mmol, 18% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>Η</sub> 8.17 (s, 1H), 7.96 (s, 1H), 7.73 (s, 1H), 3.97 (s, 3H), 3.82 (s, 2H).</div></div><div id="sec4_50_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i158" class="anchor-spacer"></div><h4 class="article-section__title" id="_i158"> Step B: Methyl 3-Bromo-5-(1-cyanocyclobutyl)benzoate</h4><div class="NLM_p last">To a solution of methyl 3-bromo-5-(cyanomethyl)benzoate (500 mg, 2 mmol) and 1,3-diiodopropane (1.18 g, 4 mmol) in THF (30 mL) was added <i>t</i>-BuOK (448 mg, 4 mmol), and the mixture was stirred at r.t. for 16 h. The mixture was diluted with EtOAc and washed with sat. aq. NH<sub>4</sub>Cl, concentrated, and purified by flash chromatography to afford the title compound (353 mg, 1.2 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>Η</sub> 8.13 (t, <i>J =</i> 1.7 Hz, 1H), 8.01 (t, <i>J =</i> 1.7 Hz, 1H), 7.74 (t, <i>J =</i> 1.7 Hz, 1H), 3.95 (s, 3H), 2.82–2.91 (m, 2H), 2.64 (td, <i>J =</i> 9.4, 12.0 Hz, 2H), 2.40–2.55 (m, 1H), 2.12 (td, <i>J =</i> 4.6, 7.0 Hz, 1H).</div></div><div id="sec4_50_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i159" class="anchor-spacer"></div><h4 class="article-section__title" id="_i159"> Step C: Methyl 3-Amino-5-(1-cyanocyclobutyl)benzoate (<b>53</b>)</h4><div class="NLM_p last">To a solution of methyl 3-bromo-5-(1-cyanocyclobutyl)benzoate (1.1 g, 3.7 mmol), <i>tert</i>-butyl carbamate (500 mg, 4.3 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (2.33 g, 7.2 mmol) in DMF (20 mL) were added Pd<sub>2</sub>(dba)<sub>3</sub> (171 mg, 0.19 mmol) and xantphos (220 mg, 0.38 mmol), and the mixture was heated to 150 °C for 16 h. The cooled mixture was filtered through celite and concentrated. The residue was dissolved in DCM (20 mL), and TFA (5 mL) was added. The mixture was stirred for 1 h, washed with sat. aq. NaHCO<sub>3</sub>, and then concentrated. Purification by flash chromatography afforded the title compound (300 mg, 1.3 mmol, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>Η</sub> 7.45 (s, 1H), 7.26–7.35 (m, 1H), 6.89 (t, <i>J =</i> 1.9 Hz, 1H), 3.91 (s, 3H), 2.74–2.93 (m, 2H), 2.55–2.69 (m, 2H), 2.42 (td, <i>J =</i> 8.8, 11.5 Hz, 1H), 1.94–2.13 (m, 1H). MS (ESI) <i>m/z</i> = 231.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_50_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i160" class="anchor-spacer"></div><h4 class="article-section__title" id="_i160"> Step D: Methyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)benzoate</h4><div class="NLM_p last">Methyl 3-amino-5-(1-cyanocyclobutyl)benzoate (<b>53</b>; 230 mg, 1.0 mmol) and <b>18a</b> were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>. Purification by flash chromatography afforded a colorless solid (435 mg, 0.94 mmol, 94% yield). MS (ESI) <i>m/z</i> = 484.0 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_50_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i161" class="anchor-spacer"></div><h4 class="article-section__title" id="_i161"> Step E: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)benzoic Acid (<b>6p</b>)</h4><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)benzoate (435 mg, 0.94 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> (at r.t.) to afford a colorless solid (297 mg, 0.66 mmol, 70% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.85 (d, <i>J =</i> 2.5 Hz, 1H), 7.79 (t, <i>J =</i> 1.8 Hz, 1H), 7.76 (d, <i>J =</i> 1.8 Hz, 1H), 7.50 (t, <i>J =</i> 1.8 Hz, 1H), 7.47 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.85 (d, <i>J =</i> 8.8 Hz, 1H), 2.77 (ddt, <i>J =</i> 12.4, 8.8, 3.3 Hz, 2H), 2.61–2.49 (m, 2H), 2.38 (dp, <i>J =</i> 11.7, 8.8 Hz, 1H), 2.07 (dtt, <i>J =</i> 11.7, 9.2, 4.5 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.44 min, <i>m/z</i> = 467.8, 469.8 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i162" class="anchor-spacer"></div><h3 class="article-section__title" id="_i162"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>6q</b>)</h3><div id="sec4_51_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i163" class="anchor-spacer"></div><h4 class="article-section__title" id="_i163"> Step A: 1-(3-Bromo-4-methoxyphenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">2-(3-Bromo-4-methoxyphenyl)acetonitrile (4.52 g, 20 mmol) and 1,3-dibromopropane (2.23 mL, 22 mmol) were dissolved in DMSO (100 mL), and to this mixture was added NaH (60% dispersion in mineral oil, 2.0 g, 50 mmol) portion wise. The solution was stirred for 16 h at r.t., diluted with EtOAc/Et<sub>2</sub>O (200 mL, 1:1), and then washed with water (3 × 400 mL). The combined aqueous layers were back-extracted with further EtOAc (200 mL). The combined organics were washed with sat. aq. NaCl, concentrated, and purified by ISCO flash column chromatography (120 g, 0–25% EtOAc in hexanes) to afford a colorless oil (3.65 g, 13.7 mmol, 68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.61 (d, <i>J =</i> 2.4 Hz, 1H), 7.34 (dd, <i>J =</i> 8.6, 2.4 Hz, 1H), 6.93 (d, <i>J =</i> 8.6 Hz, 1H), 3.93 (s, 3H), 2.88–2.78 (m, 2H), 2.66–2.54 (m, 2H), 2.44 (dt, <i>J =</i> 11.6, 8.7 Hz, 1H), 2.16–2.04 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.10 min, no mass observed.</div></div><div id="sec4_51_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i164" class="anchor-spacer"></div><h4 class="article-section__title" id="_i164"> Step B: Phenyl 5-(1-Cyanocyclobutyl)-2-methoxybenzoate</h4><div class="NLM_p last">1-(3-Bromo-4-methoxyphenyl)cyclobutane-1-carbonitrile (531 μL, 3.43 mmol), Pd(OAc)<sub>2</sub> (39 mg, 0.17 mmol), P(<i><sup>t</sup></i>Bu<sub>4</sub>)·HBF<sub>4</sub> (199 mg, 0.69 mmol), and phenol (323 mg, 3.43 mmol) were taken up in anhydrous MeCN (13.7 mL) in a sealed tube. Phenyl formate (687 μL, 6.86 mmol) and NEt<sub>3</sub> (1.43 mL, 10.3 mmol) were added, and the mixture was heated to 90 °C for 18 h. This reaction was performed in quadruplicate. Upon cooling, the four reactions were combined, filtered through celite, and concentrated. The residue was dissolved in EtOAc and washed with water. Purification by flash chromatography afforded a colorless solid (3.72 g, 12.1 mmol, 88% yield).<sup>1</sup>H NMR (CDCl<sub>3</sub>): δ<sub>H</sub> 8.04 (d, <i>J =</i> 2.6 Hz, 1H), 7.61 (dd, <i>J =</i> 8.7, 2.6 Hz, 1H), 7.49–7.42 (m, 2H), 7.33–7.23 (m, 3H), 7.09 (d, <i>J =</i> 8.7 Hz, 1H), 3.98 (s, 3H), 2.93–2.83 (m, 2H), 2.66 (dt, <i>J =</i> 11.8, 9.0 Hz, 2H), 2.55–2.41 (m, 1H), 2.19–2.06 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.71 min, <i>m/z</i> = 308.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_51_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i165" class="anchor-spacer"></div><h4 class="article-section__title" id="_i165"> Step C: Phenyl 5-(1-Cyanocyclobutyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 5-(1-cyanocyclobutyl)-2-methoxybenzoate (3.72 g, 12.1 mmol) was reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford a colorless oil (3.47 g, 11.84 mmol, 98% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.55 (s, 1H), 8.09 (d, <i>J =</i> 2.5 Hz, 1H), 7.59 (dd, <i>J =</i> 8.7, 2.5 Hz, 1H), 7.52–7.43 (m, 2H), 7.34 (d, <i>J =</i> 7.4 Hz, 1H), 7.25–7.20 (m, 2H), 7.09 (d, <i>J =</i> 8.7 Hz, 1H), 2.91–2.81 (m, 2H), 2.72–2.59 (m, 2H), 2.53–2.38 (m, 1H), 2.16–2.05 (m, 1H). LC–MS (Method A): <i>t</i><sub>R</sub> = 1.20 min, <i>m/z</i> = 294.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_51_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i166" class="anchor-spacer"></div><h4 class="article-section__title" id="_i166"> Step D: Phenyl 5-(1-Cyanocyclobutyl)-2-hydroxy-3-nitrobenzoate</h4><div class="NLM_p last">Phenyl 5-(1-cyanocyclobutyl)-2-hydroxybenzoate (2.93 g, 10 mmol) was dissolved in DCE/H<sub>2</sub>O (1:1, 20 mL) and cooled to 0 °C in an ice/water bath. Tetrabutylammonium bromide (161 mg, 0.50 mmol) was added, followed by HNO<sub>3</sub> (concd, 1.3 mL, 20 mmol). The mixture was stirred vigorously at 60 °C for 16 h, cooled, diluted with CH<sub>2</sub>Cl<sub>2</sub>, and then washed with water. Purification by flash chromatography afforded a pale yellow solid (2.71 g, 8.0 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.42 (d, <i>J =</i> 2.5 Hz, 1H), 8.31 (d, <i>J =</i> 2.6 Hz, 1H), 7.55–7.48 (m, 2H), 7.43–7.35 (m, 1H), 7.32–7.24 (m, 3H), 3.00–2.89 (m, 2H), 2.76–2.63 (m, 2H), 2.61–2.49 (m, 1H), 2.25–2.11 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.18 min, <i>m/z</i> = 339.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_51_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i167" class="anchor-spacer"></div><h4 class="article-section__title" id="_i167"> Step E: Phenyl 3-Amino-5-(1-cyanocyclobutyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 5-(1-cyanocyclobutyl)-2-hydroxy-3-nitrobenzoate (2.71 g, 8.0 mmol) was reacted according to <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>. Purification by flash chromatography afforded a cream solid (2.15 g, 7.0 mmol, 87% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.70 (d, <i>J =</i> 0.7 Hz, 1H), 7.52–7.46 (m, 2H), 7.48 (d, <i>J =</i> 2.3 Hz, 1H), 7.38–7.32 (m, 1H), 7.27–7.22 (m, 2H), 6.98 (dd, <i>J =</i> 2.3, 0.7 Hz, 1H), 2.90–2.77 (m, 2H), 2.71–2.58 (m, 2H), 2.54–2.39 (m, 1H), 2.16–2.07 (m, 1H). LC–MS (Method B): <i>t</i><sub>R</sub> = 1.08 min, <i>m/z</i> = 309.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_51_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i168" class="anchor-spacer"></div><h4 class="article-section__title" id="_i168"> Step F: Phenyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 3-amino-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (308 mg, 1.0 mmol) and <b>18a</b> were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonation Coupling">General Procedure A: Sulfonation Coupling</a>. Purification by flash chromatography afforded a colorless solid (393 mg, 0.72 mmol, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.63 (dt, <i>J =</i> 4.4, 1.7 Hz, 2H), 7.90 (d, <i>J =</i> 2.3 Hz, 1H), 7.82 (d, <i>J =</i> 2.3 Hz, 1H), 7.77 (d, <i>J =</i> 2.5 Hz, 1H), 7.49–7.45 (m, 2H), 7.38–7.35 (m, 2H), 7.23–7.17 (m, 2H), 6.88 (d, <i>J =</i> 8.8 Hz, 1H), 2.94–2.84 (m, 3H), 2.68–2.56 (m, 2H), 2.55–2.43 (m, 1H), 2.21–2.09 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.86 min, <i>m/z</i> = 559.8, 561.7 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_51_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i169" class="anchor-spacer"></div><h4 class="article-section__title" id="_i169"> Step G: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>6q</b>)</h4><div class="NLM_p last">Phenyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (108 mg, 0.20 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> (at r.t. instead of 65 °C) to afford a colorless solid (61 mg, 0.16 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.85 (d, <i>J =</i> 2.5 Hz, 1H), 7.68 (d, <i>J =</i> 2.4 Hz, 1H), 7.64 (d, <i>J =</i> 2.4 Hz, 1H), 7.53 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.86 (d, <i>J =</i> 8.8 Hz, 1H), 2.82–2.72 (m, 2H), 2.60–2.49 (m, 2H), 2.46–2.32 (m, 1H), 2.16–2.03 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.44 min, <i>m/z</i> = 466.8, 468.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>BrN<sub>2</sub>NaO<sub>6</sub>S, 488.9726; found, 488.9746.</div></div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i170" class="anchor-spacer"></div><h3 class="article-section__title" id="_i170"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>6r</b>)</h3><div id="sec4_52_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i171" class="anchor-spacer"></div><h4 class="article-section__title" id="_i171"> Step A: Phenyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 3-amino-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (154 mg, 0.50 mmol) and <b>18b</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a brown solid (208 mg, 72% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.79 (d, <i>J =</i> 2.5 Hz, 1H), 7.78 (d, <i>J =</i> 2.5 Hz, 1H), 7.75 (d, <i>J =</i> 2.4 Hz, 1H), 7.72 (d, <i>J =</i> 2.5 Hz, 1H), 2.05 (dd, <i>J =</i> 8.3, 3.5 Hz, 2H), 1.99–1.87 (m, 2H), 1.89–1.78 (m, 1H), 1.80–1.69 (m, 3H), 1.46–1.31 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.97 min, <i>m/z</i> = 595.7, 597.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_52_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i172" class="anchor-spacer"></div><h4 class="article-section__title" id="_i172"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>6r</b>)</h4><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (58 mg, 0.10 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> at room temperature to afford a colorless solid (12 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.76 (d, <i>J =</i> 2.4 Hz, 1H), 7.73 (d, <i>J =</i> 2.4 Hz, 1H), 7.69 (d, <i>J =</i> 2.4 Hz, 1H), 7.67 (d, <i>J =</i> 2.4 Hz, 1H), 2.82–2.72 (m, 2H), 2.62–2.51 (m, 2H), 2.46–2.32 (m, 1H), 2.14–2.03 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.56 min, <i>m/z</i> = 500.7, 502.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>14</sub>BrClN<sub>2</sub>NaO<sub>6</sub>S, 522.9337; found, 522.9358.</div></div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i173" class="anchor-spacer"></div><h3 class="article-section__title" id="_i173"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoic Acid (<b>6s</b>)</h3><div id="sec4_53_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i174" class="anchor-spacer"></div><h4 class="article-section__title" id="_i174"> Step A: 1-(3-Bromo-4-methoxyphenyl)cyclohexane-1-carbonitrile</h4><div class="NLM_p last">2-(3-Bromo-4-methoxyphenyl)acetonitrile (4.52 g, 20 mmol) and 1,5-dibromopentane (3.0 mL, 22 mmol) were dissolved in DMSO (100 mL), and to this mixture was added NaH (60% dispersion in mineral oil, 2.0 g, 50 mmol) portion wise. The solution was stirred for 16 h at r.t., diluted with EtOAc/Et<sub>2</sub>O (200 mL, 1:1), and then washed with water (3 × 400 mL). The combined aqueous layers were back-extracted with further EtOAc (200 mL). The combined organics were washed with sat. aq. NaCl, concentrated, and purified by ISCO flash column chromatography (120 g, 0–25% EtOAc in hexanes) to afford a colorless solid (4.35 g, 14.8 mmol, 74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.62 (d, <i>J =</i> 2.5 Hz, 1H), 7.42 (dd, <i>J =</i> 8.6, 2.5 Hz, 1H), 6.90 (d, <i>J =</i> 8.6 Hz, 1H), 3.90 (s, 3H), 2.18–2.10 (m, 2H), 1.91–1.79 (m, 6H), 1.78–1.65 (m, 2H). LC–MS (Method B): t<sub>R</sub> = 1.10 min, no mass observed.</div></div><div id="sec4_53_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i175" class="anchor-spacer"></div><h4 class="article-section__title" id="_i175"> Step B: Phenyl 5-(1-Cyanocyclohexyl)-2-methoxybenzoate</h4><div class="NLM_p last">1-(3-Bromo-4-methoxyphenyl)cyclohexane-1-carbonitrile (883 mg, 3.0 mmol), Pd(OAc)<sub>2</sub> (39 mg, 0.17 mmol), P(<i><sup>t</sup></i>Bu<sub>4</sub>)·HBF<sub>4</sub> (199 mg, 0.69 mmol), and phenol (323 mg, 3.43 mmol) were taken up in anhydrous MeCN (13.7 mL) in a sealed tube. Phenyl formate (687 μL, 6.86 mmol) and NEt<sub>3</sub> (1.43 mL, 10.3 mmol) were added, and the mixture was heated to 90 °C for 18 h. This reaction was performed in quadruplicate. Upon cooling, the four reactions were combined, filtered through celite, and concentrated. The residue was dissolved in EtOAc and washed with water. Purification by flash chromatography afforded a colorless solid (3.53 g, 10.5 mmol, 88% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>): δ<sub>H</sub> 8.07 (d, <i>J =</i> 2.7 Hz, 1H), 7.73 (dd, <i>J =</i> 8.8, 2.7 Hz, 1H), 7.49–7.40 (m, 2H), 7.33–7.22 (m, 3H), 7.08 (d, <i>J =</i> 8.8 Hz, 1H), 3.97 (s, 3H), 2.22 (d, <i>J =</i> 11.6 Hz, 2H), 1.98–1.74 (m, 8H). LC–MS (Method A): <i>t</i><sub>R</sub> = 1.86 min, <i>m/z</i> = 336.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_53_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i176" class="anchor-spacer"></div><h4 class="article-section__title" id="_i176"> Step C: Phenyl 5-(1-Cyanocyclohexyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 5-(1-cyanocyclohexyl)-2-methoxybenzoate (3.52 g, 10.5 mmol) was reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford a colorless oil (3.03 g, 9.49 mmol, 90% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>): δ<sub>H</sub> 10.55 (s, 1H), 8.19 (d, <i>J =</i> 2.6 Hz, 1H), 7.70 (dd, <i>J =</i> 8.8, 2.6 Hz, 1H), 7.54–7.45 (m, 2H), 7.40–7.33 (m, 1H), 7.28–7.22 (m, 2H), 7.10 (d, <i>J =</i> 8.9 Hz, 1H), 2.23 (d, <i>J =</i> 12.2 Hz, 2H), 2.03–1.72 (m, 8H). LC–MS (Method A): <i>t</i><sub>R</sub> = 1.75 min, <i>m/z</i> = 322.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_53_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i177" class="anchor-spacer"></div><h4 class="article-section__title" id="_i177"> Step D: Phenyl 3-Amino-5-(1-cyanocyclohexyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 5-(1-cyanocyclohexyl)-2-hydroxybenzoate (3.03 g, 9.5 mmol) was dissolved in DCE/H<sub>2</sub>O (1:1, 20 mL) and cooled to 0 °C in an ice/water bath. Tetrabutylammonium bromide (153 mg, 0.47 mmol) was added, followed by HNO<sub>3</sub> (concd, 1.2 mL, 19.0 mmol). The reaction was stirred vigorously at 60 °C for 18 h, cooled, diluted with CH<sub>2</sub>Cl<sub>2</sub>, and then washed with water. The crude material was carried forward without purification for hydrogenation, following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a>. Upon purification, a colorless oil was obtained (1.65 g, 4.9 mmol, 52% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>): δ<sub>H</sub> 10.55 (s, 1H), 8.19 (d, <i>J =</i> 2.6 Hz, 1H), 7.70 (dd, <i>J =</i> 8.8, 2.6 Hz, 1H), 7.52–7.46 (m, 2H), 7.39–7.32 (m, 1H), 7.28–7.22 (m, 2H), 7.10 (d, <i>J =</i> 8.8 Hz, 1H), 2.23 (d, <i>J =</i> 12.2 Hz, 3H), 1.99–1.72 (m, 8H). LC–MS (Method B): <i>t</i><sub>R</sub> = 1.28 min, <i>m/z</i> = 337.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_53_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i178" class="anchor-spacer"></div><h4 class="article-section__title" id="_i178"> Step E: Phenyl 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 3-amino-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (308 mg, 0.91 mmol) and <b>18a</b> (313 mg, 1.1 mmol) were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>. Purification by flash chromatography afforded a colorless solid (443 mg, 0.78 mmol, 85% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.85 (d, <i>J =</i> 2.5 Hz, 1H), 7.73 (d, <i>J =</i> 2.4 Hz, 1H), 7.70 (d, <i>J =</i> 2.4 Hz, 1H), 7.52 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H), 2.08–1.87 (m, 4H), 1.88–1.77 (m, 1H), 1.74 (td, <i>J =</i> 12.9, 10.5, 6.3 Hz, 4H), 1.46–1.31 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.29 min, <i>m/z</i> = 588.3, 590.3 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_53_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i179" class="anchor-spacer"></div><h4 class="article-section__title" id="_i179"> Step F: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoic Acid (<b>6s</b>)</h4><div class="NLM_p last">Phenyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (29 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (17 mg, 69% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.85 (d, <i>J =</i> 2.5 Hz, 1H), 7.73 (d, <i>J =</i> 2.4 Hz, 1H), 7.70 (d, <i>J =</i> 2.4 Hz, 1H), 7.52 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H), 2.08–1.87 (m, 4H), 1.88–1.77 (m, 1H), 1.74 (td, <i>J =</i> 12.9, 10.5, 6.3 Hz, 4H), 1.46–1.31 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.65 min, <i>m/z</i> = 494.8, 496.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i180" class="anchor-spacer"></div><h3 class="article-section__title" id="_i180"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoic Acid (<b>6t</b>)</h3><div id="sec4_54_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i181" class="anchor-spacer"></div><h4 class="article-section__title" id="_i181"> Step A: Phenyl 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 3-amino-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (308 mg, 0.91 mmol) and <b>18b</b> (336 mg, 1.1 mmol) were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>. Purification by flash chromatography afforded a colorless solid (455 mg, 0.75 mmol, 82% yield). LC–MS (Method B): <i>t</i><sub>R</sub> = 1.39 min, <i>m/z</i> = 622.3 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_54_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i182" class="anchor-spacer"></div><h4 class="article-section__title" id="_i182"> Step B: 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoic Acid (<b>6t</b>)</h4><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (30 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford a colorless solid (17 mg, 64% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.79 (d, <i>J =</i> 2.5 Hz, 1H), 7.78 (d, <i>J =</i> 2.5 Hz, 1H), 7.75 (d, <i>J =</i> 2.4 Hz, 1H), 7.72 (d, <i>J =</i> 2.5 Hz, 1H), 2.05 (dd, <i>J =</i> 8.3, 3.5 Hz, 2H), 1.99–1.87 (m, 2H), 1.89–1.78 (m, 1H), 1.80–1.69 (m, 3H), 1.46–1.31 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.74 min, <i>m/z</i> = 530.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>BrClN<sub>2</sub>NaO<sub>6</sub>S, 550.9650; found, 550.9681.</div></div></div><div id="sec4_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i183" class="anchor-spacer"></div><h3 class="article-section__title" id="_i183"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxy-<i>N</i>-methylbenzamide (<b>7a</b>)</h3><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoate (109 mg, 0.25 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (89 mg, 82% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 7.80 (d, <i>J =</i> 2.5 Hz, 1H), 7.54 (d, <i>J =</i> 2.4 Hz, 1H), 7.53 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.47 (d, <i>J =</i> 2.4 Hz, 1H), 6.87 (d, <i>J =</i> 8.8 Hz, 1H), 2.88 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.57 min, <i>m/z</i> = 434.8, 436.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + Na]<sup>+</sup> calcd for C<sub>14</sub>H<sub>12</sub>BrClN<sub>2</sub>NaO<sub>5</sub>S, 456.9231; found, 456.9247.</div></div><div id="sec4_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i184" class="anchor-spacer"></div><h3 class="article-section__title" id="_i184"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxy-<i>N</i>-methylbenzamide (<b>7b</b>)</h3><div class="NLM_p last">3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-chloro-2-hydroxybenzoic acid (23 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (5 mg, 21% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 10.94 (br s, 1H), 7.72 (dd, <i>J =</i> 9.2, 2.4 Hz, 1H), 7.70 (d, <i>J =</i> 2.4 Hz, 1H), 7.65 (d, <i>J =</i> 2.4 Hz, 1H), 7.63 (dd, <i>J =</i> 2.4, 1.2 Hz, 2H), 7.47 (s, 1H), 7.12 (d, <i>J =</i> 2.4 Hz, 1H), 6.27 (br s, 1H), 3.00 (d, <i>J =</i> 4.9 Hz, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.69 min, <i>m/z</i> = 470.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i185" class="anchor-spacer"></div><h3 class="article-section__title" id="_i185"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxy-<i>N</i>-methylbenzamide (<b>7c</b>)</h3><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (23 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (21 mg, 91% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 11.27 (s, 1H), 8.88 (s, 1H), 8.44–8.35 (m, 1H), 7.60 (d, <i>J =</i> 8.0 Hz, 2H), 6.95 (d, <i>J =</i> 8.3 Hz, 1H), 6.75 (d, <i>J =</i> 8.1 Hz, 1H), 2.79 (d, <i>J =</i> 4.2 Hz, 3H), 1.94–1.79 (m, 1H), 0.95–0.81 (m, 2H), 0.45–0.34 (m, 2H). <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>F</sub> −120.4. LC–MS: <i>t</i><sub>R</sub> = 1.69 min, <i>m/z</i> = 458.8, 460.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i186" class="anchor-spacer"></div><h3 class="article-section__title" id="_i186"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxy-<i>N</i>-methylbenzamide (<b>7d</b>)</h3><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-cyclopropyl-6-fluoro-2-hydroxybenzoate (49 mg, 0.1 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (36 mg, 73% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.73 (d, <i>J =</i> 2.4 Hz, 1H), 7.63 (d, <i>J =</i> 2.4 Hz, 1H), 7.09 (d, <i>J =</i> 8.1 Hz, 1H), 2.93 (s, 3H), 2.21–1.86 (m, 1H), 1.05–0.87 (m, 2H), 0.70–0.51 (m, 2H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> −121.7. LC–MS: <i>t</i><sub>R</sub> = 1.83 min, <i>m/z</i> = 492.7, 494.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i187" class="anchor-spacer"></div><h3 class="article-section__title" id="_i187"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-<i>N</i>-methyl-5-(pentafluorosulfanyl)benzamide (<b>7e</b>)</h3><div class="NLM_p last">3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(pentafluorosulfanyl)benzoic acid (<b>6l</b>; 25 mg, 0.05 mmol) was dissolved in DCM (0.5 mL), and DIPEA (26 μL, 0.15 mmol) was added. The mixture was cooled to 0 °C before the addition of HATU (29 mg, 0.075 mmol). The mixture was stirred for 30 min, and then methylamine (2.0 M in THF, 125 μL, 0.25 mmol) was added and stirred for 16 h. The mixture was diluted with DCM, washed with water, and purified by prep HPLC to afford a colorless solid (6 mg, 0.01, 23% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.99 (dt, <i>J</i> = 5.8, 1.5 Hz, 2H), 7.85 (dd, <i>J</i> = 2.4, 1.3 Hz, 2H), 7.50 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.83 (d, <i>J</i> = 8.7 Hz, 1H), 3.92 (s, 3H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 81.2 (p, <i>J</i> = 149 Hz, 1F), 61.0 (d, <i>J</i> = 149 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.33 min, <i>m/z</i> = 510.7, 512.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i188" class="anchor-spacer"></div><h3 class="article-section__title" id="_i188"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-<i>N</i>-methyl-5-(pentafluorosulfanyl)benzamide (<b>7f</b>)</h3><div class="NLM_p last">Methyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (26 mg, 0.05 mmol) was reacted following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (7 mg, 0.013 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.34 (s, 1H), 8.05 (s, 2H), 7.68 (d, <i>J</i> = 2.4 Hz, 1H), 7.50 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.28 (s, 1H), 6.87 (d, <i>J</i> = 8.8 Hz, 1H), 4.00 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 83.3 (d, <i>J</i> = 150 Hz), 63.7 (d, <i>J</i> = 150 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.89 min, <i>m/z</i> = 527.9, 529.9 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i189" class="anchor-spacer"></div><h3 class="article-section__title" id="_i189"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-<i>N</i>-methyl-5-(pentafluorosulfanyl)benzamide (<b>7g</b>)</h3><div class="NLM_p last">Methyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (18 mg, 0.03 mmol) was reacted following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford the title compound as a colorless solid (3 mg, 0.005 mmol, 17% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.04 (d, <i>J</i> = 2.4 Hz, 1H), 7.72 (d, <i>J</i> = 2.4 Hz, 1H), 7.64 (d, <i>J</i> = 2.3 Hz, 1H), 7.53 (d, <i>J</i> = 2.3 Hz, 1H), 6.42 (br s, 1H), 3.04 (d, <i>J</i> = 4.9 Hz, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 84.0 (p, <i>J</i> = 150 Hz), 64.1 (d, <i>J</i> = 150 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.16 min, <i>m/z</i> = 561.1, 563.1 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i190" class="anchor-spacer"></div><h3 class="article-section__title" id="_i190"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide (<b>7h</b>)</h3><div id="sec4_62_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i191" class="anchor-spacer"></div><h4 class="article-section__title" id="_i191"> Step A: 3-Amino-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide</h4><div class="NLM_p last"><b>53</b> (230 mg, 1.0 mmol) was taken in a solution of methylamine ([2.0 M in THF], 2 mL) and heated to 65 °C for 3 h. The mixture was concentrated to afford the title compound (202 mg, 0.88 mmol, 88% yield), which was used without purification. MS (ESI) <i>m/z</i> = 230.1.</div></div><div id="sec4_62_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i192" class="anchor-spacer"></div><h4 class="article-section__title" id="_i192"> Step B: 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide (<b>7h</b>)</h4><div class="NLM_p last">3-Amino-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide (23 mg, 0.1 mmol) and <b>18a</b> (33 mg, 0.12 mmol) were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>. Purification by reversed-phase HPLC afforded a colorless solid (12 mg, 0.026 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 8.47 (s, 1H), 7.76 (d, <i>J</i> = 2.5 Hz, 1H), 7.60–7.42 (m, 3H), 7.28 (s, 1H), 6.88 (d, <i>J</i> = 8.8 Hz, 1H), 3.18 (s, 3H), 2.76 (d, <i>J</i> = 4.5 Hz, 3H), 2.74–2.65 (m, 2H), 2.38–2.21 (m, 2H), 2.04–1.92 (m, 2H). LC–MS: <i>t</i><sub>R</sub> = 1.41 min, <i>m/z</i> = 463.8, 465.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i193" class="anchor-spacer"></div><h3 class="article-section__title" id="_i193"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide (<b>7i</b>)</h3><div class="NLM_p last">3-Amino-5-(1-cyanocyclobutyl)-<i>N</i>-methylbenzamide (23 mg, 0.1 mmol) and <b>18b</b> (37 mg, 0.12 mmol) were reacted according to <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>. Purification by reversed-phase HPLC afforded a colorless solid (17 mg, 0.034 mmol, 34% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 8.50 (d, <i>J</i> = 4.9 Hz, 1H), 7.91 (d, <i>J</i> = 2.5 Hz, 1H), 7.79 (d, <i>J</i> = 2.5 Hz, 1H), 7.55 (t, <i>J</i> = 1.8 Hz, 1H), 7.53 (d, <i>J</i> = 1.6 Hz, 1H), 7.28 (t, <i>J</i> = 1.9 Hz, 1H), 2.76 (d, <i>J</i> = 4.5 Hz, 3H), 2.77–2.65 (m, 2H), 2.37–2.22 (m, 1H), 1.99 (dtt, <i>J</i> = 9.1, 7.0, 4.5 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.50 min, <i>m/z</i> = 497.8, 499.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i194" class="anchor-spacer"></div><h3 class="article-section__title" id="_i194"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>7j</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (31 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (18 mg, 75% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.82 (d, <i>J =</i> 2.5 Hz, 1H), 7.56–7.47 (m, 3H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H), 2.91 (s, 3H), 2.81–2.71 (m, 2H), 2.65–2.51 (m, 2H), 2.38 (dt, <i>J =</i> 11.5, 8.6 Hz, 1H), 2.12–2.02 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.54 min, <i>m/z</i> = 479.8, 481.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>BrN<sub>3</sub>O<sub>5</sub>S, 480.0223; found, 480.0239.</div></div><div id="sec4_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i195" class="anchor-spacer"></div><h3 class="article-section__title" id="_i195"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>7k</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (29 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (13 mg, 54% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 11.07 (br s, 1H), 9.39 (br s, 2H), 9.12 (d, <i>J =</i> 4.6 Hz, 1H), 7.93 (d, <i>J =</i> 2.5 Hz, 1H), 7.71 (d, <i>J =</i> 2.1 Hz, 1H), 7.71 (d, <i>J =</i> 2.5 Hz, 1H), 7.35 (d, <i>J =</i> 2.1 Hz, 1H), 2.83 (d, <i>J =</i> 4.6 Hz, 3H), 2.75–2.65 (m, 2H), 2.58–2.46 (m, 2H), 2.25 (dt, <i>J =</i> 11.3, 8.6 Hz, 1H), 2.05–1.88 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.66 min, <i>m/z</i> = 513.7, 515.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>BrClN<sub>3</sub>O<sub>5</sub>S, 513.9834; found, 513.9848.</div></div><div id="sec4_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i196" class="anchor-spacer"></div><h3 class="article-section__title" id="_i196"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>7l</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (29 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (16 mg, 63% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 11.30 (s, 1H), 9.09 (s, 1H), 8.88 (s, 1H), 7.71 (d, <i>J =</i> 2.6 Hz, 1H), 7.65 (d, <i>J =</i> 2.1 Hz, 1H), 7.58 (dd, <i>J =</i> 8.7, 2.1 Hz, 1H), 7.50–7.45 (m, 3H), 6.91 (d, <i>J =</i> 8.7 Hz, 1H), 2.81 (d, <i>J =</i> 4.3 Hz, 3H), 1.99 (d, <i>J =</i> 11.7 Hz, 2H), 1.90–1.51 (m, 8H). LC–MS: <i>t</i><sub>R</sub> = 1.72 min, <i>m/z</i> = 507.8, 509.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i197" class="anchor-spacer"></div><h3 class="article-section__title" id="_i197"> 3-((5-Bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>7m</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclohexyl)-2-hydroxybenzoate (30 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (21 mg, 77% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ<sub>H</sub> 9.09 (d, <i>J =</i> 4.4 Hz, 1H), 7.92 (d, <i>J =</i> 2.3 Hz, 1H), 7.72 (d, <i>J =</i> 2.4 Hz, 1H), 7.70 (d, <i>J =</i> 2.4 Hz, 1H), 7.46 (d, <i>J =</i> 2.2 Hz, 1H), 2.82 (d, <i>J =</i> 4.4 Hz, 3H), 2.02 (d, <i>J =</i> 12.7 Hz, 2H), 1.88–1.51 (m, 7H), 1.26 (d, <i>J =</i> 12.7 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.84 min, <i>m/z</i> = 543.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%. HRMS (ESI/TOF) [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>BrClN<sub>3</sub>O<sub>5</sub>S, 542.0147; found, 542.0161.</div></div><div id="sec4_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i198" class="anchor-spacer"></div><h3 class="article-section__title" id="_i198"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-(1<i>H</i>-1,2,4-triazol-3-yl)benzamide (<b>7n</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (27 mg, 0.05 mmol), 3-amino-1,2,4-triazole (21 mg), and NEt<sub>3</sub> (35 μL, 0.25 mmol) were heated to 90 °C in dioxane (1 mL) for 16 h. The mixture was concentrated and purified by prep HPLC to afford a colorless solid (15 mg, 56% yield) after neutralization with sat. aq. K<sub>2</sub>CO<sub>3</sub>. LC–MS: <i>t</i><sub>R</sub> = 1.41 min, <i>m/z</i> = 532.8, 534.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i199" class="anchor-spacer"></div><h3 class="article-section__title" id="_i199"> 3-((5-Bromo-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-(2-methoxyethyl)benzamide (<b>7o</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-3-chloro-2-hydroxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (27 mg, 0.05 mmol), (2-methoxyethyl)ethylamine (22 μL, 0.25 mmol), and NEt<sub>3</sub> (35 μL, 0.25 mmol) were heated to 90 °C in dioxane (1 mL) for 16 h. The mixture was concentrated and purified by prep HPLC to afford a colorless solid (16 mg, 61% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.80 (d, <i>J =</i> 2.5 Hz, 1H), 7.54 (d, <i>J =</i> 2.2 Hz, 1H), 7.53 (d, <i>J =</i> 2.2 Hz, 1H), 7.50 (dd, <i>J =</i> 8.7, 2.5 Hz, 1H), 6.84 (d, <i>J =</i> 8.8 Hz, 1H), 3.55 (d, <i>J =</i> 1.4 Hz, 4H), 3.36 (s, 3H), 2.81–2.70 (m, 2H), 2.62–2.51 (m, 2H), 2.44–2.30 (m, 1H), 2.11–1.97 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.57 min, <i>m/z</i> = 523.8, 525.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i200" class="anchor-spacer"></div><h3 class="article-section__title" id="_i200"> 5-Bromo-<i>N</i>-(3-chloro-5-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7p</b>)</h3><div id="sec4_70_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i201" class="anchor-spacer"></div><h4 class="article-section__title" id="_i201"> Step A: 3-Chloro-5-(methylsulfonyl)aniline</h4><div class="NLM_p last">1-Chloro-3-iodo-5-nitrobenzene (400 mg, 1.41 mmol), sodium methanesulfinate (174 mg, 1.7 mmol), CuI (27 mg, 0.14 mmol), <span class="smallcaps smallerCapital">l</span>-proline (32 mg, 0.28 mmol), and NaOH (11 mg, 0.28 mmol) were taken in DMSO (3 mL) under an inert atmosphere and heated to 85 °C for 16 h. Upon cooling, the mixture was diluted with H<sub>2</sub>O and extracted with DCM. The organic phase was dried and concentrated to afford a colorless solid, which without purification was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford a yellow solid (108 mg, 0.53 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.23 (s, 1H), 7.08 (s, 1H), 6.86 (s, 1H), 4.11 (s, 2H), 3.04 (s, 3H).</div></div><div id="sec4_70_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i202" class="anchor-spacer"></div><h4 class="article-section__title" id="_i202"> Step B: 5-Bromo-<i>N</i>-(3-chloro-5-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7p</b>)</h4><div class="NLM_p last">3-Chloro-5-(methylsulfonyl)aniline (20 mg, 0.10 mmol) was reacted with <b>18a</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (20 mg 0.046 mmol, 48% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.88 (d, <i>J</i> = 2.5, 1H), 7.65 (t, <i>J</i> = 1.7 Hz, 1H), 7.58 (t, <i>J</i> = 1.7 Hz, 1H), 7.52 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.47 (t, <i>J</i> = 1.7 Hz, 1H), 6.85 (d, <i>J</i> = 8.8 Hz, 1H), 3.06 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.63 min, <i>m/z</i> = 458.7 [M + NH4]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i203" class="anchor-spacer"></div><h3 class="article-section__title" id="_i203"> 5-Bromo-3-chloro-<i>N</i>-(3-chloro-5-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7q</b>)</h3><div class="NLM_p last">3-Chloro-5-(methylsulfonyl)aniline (50 mg, 0.24 mmol) was reacted with <b>18b</b> following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (38 mg 0.085 mmol, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.79 (t, <i>J</i> = 1.9 Hz, 1H), 7.70 (d, <i>J</i> = 2.2 Hz, 2H), 7.52 (d, <i>J</i> = 1.9 Hz, 2H), 3.05 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.07 min, <i>m/z</i> = 492.8 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i204" class="anchor-spacer"></div><h3 class="article-section__title" id="_i204"> 5-Bromo-<i>N</i>-(5-chloro-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7r</b>)</h3><div id="sec4_72_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i205" class="anchor-spacer"></div><h4 class="article-section__title" id="_i205"> Step A: 4-Chloro-2-(methylthio)-6-nitrophenol</h4><div class="NLM_p last">To a solution of 2-bromo-4-chloro-6-nitrophenol (1000 mg, 3.96 mmol) in pyridine (10 mL) were added Cu (500 mg, 7.9 mmol) and dimethyl disulfide (713 μL, 7.9 mmol). The mixture was heated to 110 °C for 16 h. The cooled mixture was filtered through celite, suspended in 3 M HCl, extracted with DCM, and concentrated. Purification by flash chromatography afforded the title compound (630 mg, 2.87 mmol, 72% yield). MS (ESI) <i>m/z</i> = 221.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_72_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i206" class="anchor-spacer"></div><h4 class="article-section__title" id="_i206"> Step B: 4-Chloro-2-(methylsulfonyl)-6-nitrophenol</h4><div class="NLM_p last">4-Chloro-2-(methylthio)-6-nitrophenol (1024 mg, 4.66 mmol) was dissolved in EtOH/H<sub>2</sub>O (1:1, 15 mL), and oxone (2.86 g, 9.32 mmol) was added. The mixture was stirred at r.t. for 24 h. The mixture was diluted with water, extracted with DCM, and dried (MgSO<sub>4</sub>) to afford a yellow solid (1155 mg, 4.58 equiv, 98% yield), which was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 11.40 (s, 1H), 8.39 (d, <i>J</i> = 2.7 Hz, 1H), 8.31 (d, <i>J</i> = 2.7 Hz, 1H), 7.26 (s, 1H), 3.34 (s, 3H).</div></div><div id="sec4_72_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i207" class="anchor-spacer"></div><h4 class="article-section__title" id="_i207"> Step C: 2-Amino-4-chloro-6-(methylsulfonyl)phenol</h4><div class="NLM_p last">4-Chloro-2-(methylsulfonyl)-6-nitrophenol (68 mg, 0.20 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford the title compound (60 mg, 0.2 mmol, 98% yield), which was taken forward without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.00 (d, <i>J</i> = 2.3 Hz, 1H), 6.86 (d, <i>J</i> = 2.4 Hz, 1H), 3.12 (s, 3H). MS (ESI) <i>m/z</i> =222.1, 224.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_72_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i208" class="anchor-spacer"></div><h4 class="article-section__title" id="_i208"> Step D: 5-Bromo-<i>N</i>-(5-chloro-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7r</b>)</h4><div class="NLM_p last">Amino-4-chloro-6-(methylsulfonyl)phenol (21 mg, 0.09 mmol) and 5-bromo-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (11 mg, 0.024 mmol, 26% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.13 (s, 1H), 8.33 (s, 1H), 7.72 (d, <i>J</i> = 2.5 Hz, 1H), 7.63 (d, <i>J</i> = 2.4 Hz, 1H), 7.54 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 7.48 (d, <i>J</i> = 2.5 Hz, 1H), 7.18 (s, 1H), 6.91 (d, <i>J</i> = 8.9 Hz, 1H), 3.08 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.71 min, <i>m/z</i> = 475.1 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i209" class="anchor-spacer"></div><h3 class="article-section__title" id="_i209"> 5-Bromo-3-chloro-<i>N</i>-(5-chloro-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7s</b>)</h3><div class="NLM_p last">Amino-4-chloro-6-(methylsulfonyl)phenol (21 mg, 0.09 mmol) and 5-bromo-3-chloro-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording a colorless solid (12.2 mg, 0.024 mmol, 28% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.76 (d, <i>J</i> = 2.5 Hz, 1H), 7.74 (d, <i>J</i> = 2.4 Hz, 1H), 7.69 (d, <i>J</i> = 2.4 Hz, 1H), 7.45 (d, <i>J</i> = 2.5 Hz, 1H), 3.11 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.06 min, <i>m/z</i> = 509.1, 511.1 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i210" class="anchor-spacer"></div><h3 class="article-section__title" id="_i210"> 5-Bromo-2-hydroxy-<i>N</i>-(3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7t</b>)</h3><div id="sec4_74_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i211" class="anchor-spacer"></div><h4 class="article-section__title" id="_i211"> Step A: Methyl 3-(methylsulfonyl)-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-bromo-5-(pentafluorosulfanyl)benzoate (600 mg, 1.91 mmol), sodium methanesulfinate (146 mg, 1.4 mmol), CuI (23 mg, 0.12 mmol), <span class="smallcaps smallerCapital">l</span>-proline (27 mg, 0.24 mmol), and NaOH (10 mg, 0.24 mmol) were taken in DMSO (2.5 mL) under an inert atmosphere and heated to 85 °C for 16 h. Upon cooling, the mixture was diluted with H<sub>2</sub>O and extracted with DCM. The organic phase was dried and concentrated to afford a colorless solid (164.1 mg, 0.48 mmol, 25% yield), which was taken forward without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): δ<sub>H</sub> 8.74 (d, <i>J</i> = 1.7 Hz, 1H), 8.68 (t, <i>J</i> = 1.7 Hz, 1H), 8.50 (t, <i>J</i> = 1.7 Hz, 1H), 4.02 (s, 3H), 3.15 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 80.4 (t, <i>J</i> = 151 Hz), 63.0 (d, <i>J</i> = 151 Hz). LC–MS: <i>t</i><sub>R</sub> = 0.98 min, <i>m/z</i> = 341.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_74_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i212" class="anchor-spacer"></div><h4 class="article-section__title" id="_i212"> Step B: 3-(Methylsulfonyl)-5-(pentafluorosulfanyl)benzoic Acid</h4><div class="NLM_p last">Methyl 3-(methylsulfonyl)-5-(pentafluorosulfanyl)benzoate (162 mg, 0.54 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General procedure E: Ester Hydrolysis">General procedure E: Ester Hydrolysis</a> to afford a colorless solid (117.7 mg, 0.36 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.79 (s, 1H), 8.73 (s, 1H), 8.56 (s, 1H), 3.17 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 80.2 (d, <i>J</i> = 151 Hz, 1F), 63.0 (d, <i>J</i> = 151 Hz, 4F).</div></div><div id="sec4_74_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i213" class="anchor-spacer"></div><h4 class="article-section__title" id="_i213"> Step C: 3-(Methylsulfonyl)-5-(pentafluorosulfanyl)aniline</h4><div class="NLM_p last">3-(Methylsulfonyl)-5-(pentafluorosulfanyl)benzoic acid (117 mg, 0.36 mmol), diphenylphosphoryl azide (119 mg, 0.43 mmol), and <i>tert</i>-butanol (172 μL, 1.8 mmol) were taken in DME (1.8 mL) and heated to 100 °C for 16 h. The cooled mixture was diluted with EtOAc, washed with sat. aq. NH<sub>4</sub>Cl, sat. aq. NaHCO<sub>3</sub>, water, and then concentrated. The residue was dissolved in DCM (5 mL), and TFA (1 mL) was added. The mixture was stirred for 1 h at r.t. The mixture was concentrated and purified by flash chromatography to afford a colorless solid (51.2 mg, 0.17 mmol, 48% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.61 (t, <i>J</i> = 1.8 Hz, 1H), 7.32 (t, <i>J</i> = 1.8 Hz, 1H), 7.25 (t, <i>J</i> = 1.8 Hz, 1H), 4.29 (s, 2H), 3.07 (s, 3H).<sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ 79.4 (p, <i>J</i> = 151 Hz, 1F), 59.53 (d, <i>J</i> = 150.5 Hz, 4F).</div></div><div id="sec4_74_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i214" class="anchor-spacer"></div><h4 class="article-section__title" id="_i214"> Step D: 5-Bromo-2-hydroxy-<i>N</i>-(3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7t</b>)</h4><div class="NLM_p last">3-(Methylsulfonyl)-5-(pentafluorosulfanyl)aniline (20 mg, 0.07 mmol) and <b>18a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (8.4 mg, 0.016 mmol, 23% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.08 (s, 1H), 7.82 (s, 1H), 7.77 (s, 1H), 7.73 (d, <i>J</i> = 2.5 Hz, 1H), 7.65 (s, 1H), 7.54 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.90 (d, <i>J</i> = 8.8 Hz, 1H), 3.08 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 80.5 (d, <i>J</i> = 151 Hz), 62.9 (d, <i>J</i> = 151 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.128 min, <i>m/z</i> = 548.8 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i215" class="anchor-spacer"></div><h3 class="article-section__title" id="_i215"> 5-Bromo-3-chloro-2-hydroxy-<i>N</i>-(3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7u</b>)</h3><div class="NLM_p last">3-(Methylsulfonyl)-5-(pentafluorosulfanyl)aniline (20 mg, 0.07 mmol) and <b>18b</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford a colorless solid (7 mg, 0.013 mmol, 19% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.08 (s, 1H), 7.82 (s, 1H), 7.77 (s, 1H), 7.73 (d, <i>J</i> = 2.5 Hz, 1H), 7.65 (s, 1H), 6.90 (d, <i>J</i> = 8.8 Hz, 1H), 3.08 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 80.5 (d, <i>J</i> = 151 Hz, 1F), 62.9 (d, <i>J</i> = 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.201 min, <i>m/z</i> = 584.7 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i216" class="anchor-spacer"></div><h3 class="article-section__title" id="_i216"> 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7v</b>)</h3><div id="sec4_76_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i217" class="anchor-spacer"></div><h4 class="article-section__title" id="_i217"> Step A: 2-Bromo-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">To 4-(pentafluorosulfanyl)phenol (800 mg, 3.63 mmol) in acetic acid (4 mL) was added FeCl<sub>3</sub> (59 mg, 0.36 mmol). The mixture was cooled to 0 °C, and then Br<sub>2</sub> (812 mg, 5.45 mmol) was added. The mixture was allowed to reach room temperature and stirred for 2 h. Then, the solvent was removed under reduced pressure, and the crude material was purified by flash chromatography to afford the title compound (439 mg, 1.5 mmol, 41% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.90 (d, <i>J</i> = 2.6 Hz, 1H), 7.63 (dd, <i>J</i> = 9.0, 2.6 Hz, 1H), 7.06 (d, <i>J</i> = 9.0 Hz, 1H).</div></div><div id="sec4_76_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i218" class="anchor-spacer"></div><h4 class="article-section__title" id="_i218"> Step B: 2-(Methylthio)-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">To 2-bromo-4-(pentafluorosulfanyl)phenol (439 mg, 1.47 mmol) in pyridine (4 mL) were added Cu (500 mg, 7.9 mmol) and dimethyl disulfide (713 μL, 7.9 mmol). The mixture was heated to 110 °C for 16 h. The cooled mixture was filtered through celite, suspended in 3 M HCl, extracted with DCM, and concentrated. Purification by flash chromatography afforded the title compound (84 mg, 0.32 mmol, 32% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.89 (d, <i>J</i> = 2.7 Hz, 1H), 7.63 (dd, <i>J</i> = 9.0, 2.7 Hz, 1H), 7.05–6.97 (m, 2H), 2.37 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 85.5 (d, <i>J</i> = 151 Hz), 64.3 (d, <i>J</i> = 151 Hz). MS (ESI) <i>m/z</i> = 266.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_76_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i219" class="anchor-spacer"></div><h4 class="article-section__title" id="_i219"> Step C: 2-(Methylsulfonyl)-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">2-(Methylthio)-4-(pentafluorosulfanyl)phenol (84 mg, 0.32 mmol) was dissolved in EtOH/H<sub>2</sub>O (1:1, 1 mL), and oxone (197 g, 9.32 mmol) was added. The mixture was stirred at r.t. for 24 h. The mixture was diluted with water, extracted with DCM, and dried (MgSO<sub>4</sub>) to afford the title compound (44 mg, 0.25 mmol, 78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.25 (s, 1H), 8.11 (d, <i>J</i> = 2.7 Hz, 1H), 7.91 (dd, <i>J</i> = 9.2, 2.7 Hz, 1H), 7.13 (d, <i>J</i> = 9.2 Hz, 1H), 3.18 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 83.2 (d, <i>J</i> = 151 Hz), 64.0 (d, <i>J</i> = 151 Hz).</div></div><div id="sec4_76_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i220" class="anchor-spacer"></div><h4 class="article-section__title" id="_i220"> Step D: 2-(Methylsulfonyl)-6-nitro-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">2-(Methylsulfonyl)-4-(pentafluorosulfanyl)phenol (74 mg, 0.25 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford a yellow solid (68 mg, 0.20 mmol, 80% yield).<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.81 (s, 1H), 8.70 (s, 1H), 3.38 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 80.0 (p, <i>J</i> = 152 Hz), 63.9 (d, <i>J</i> = 152 Hz).</div></div><div id="sec4_76_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i221" class="anchor-spacer"></div><h4 class="article-section__title" id="_i221"> Step E: 2-Amino-6-(methylsulfonyl)-4-(pentafluorosulfanyl)phenol</h4><div class="NLM_p last">2-(Methylsulfonyl)-6-nitro-4-(pentafluorosulfanyl)phenol (235 mg, 0.68 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford the title compound as a pale brown solid (214 mg, 0.68 mmol, quant.), which was taken forward without purification. MS (ESI) <i>m/z</i> = 314.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_76_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i222" class="anchor-spacer"></div><h4 class="article-section__title" id="_i222"> Step F: 5-Bromo-2-hydroxy-<i>N</i>-(2-hydroxy-3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7v</b>)</h4><div class="NLM_p last">2-Amino-6-(methylsulfonyl)-4-(pentafluorosulfanyl)phenol (40 mg, 0.13 mmol) and 5-bromo-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (24 mg, 0.044 mmol, 34% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.03 (d, <i>J</i> = 2.8 Hz, 1H), 7.75 (d, <i>J</i> = 2.5 Hz, 1H), 7.67 (d, <i>J</i> = 2.8 Hz, 1H), 7.62 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.97 (d, <i>J</i> = 8.8 Hz, 1H), 3.31 (s, 3H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 79.1 (p, <i>J</i> = 149 Hz), 59.4 (d, <i>J</i> = 149 Hz). LC–MS: <i>t</i><sub>R</sub> = 1.01 min, <i>m/z</i> = 547.9, 549.9 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i223" class="anchor-spacer"></div><h3 class="article-section__title" id="_i223"> 5-Bromo-3-chloro-2-hydroxy-<i>N</i>-(2-hydroxy-3-(methylsulfonyl)-5-(pentafluorosulfanyl)phenyl)benzenesulfonamide (<b>7w</b>)</h3><div class="NLM_p last">2-Amino-6-(methylsulfonyl)-4-(pentafluorosulfanyl)phenol (21 mg, 0.09 mmol) and 5-bromo-3-chloro-2-hydroxybenzenesulfonyl chloride was reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (32 mg, 0.055 mmol, 42% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.16 (d, <i>J</i> = 2.5 Hz, 1H), 7.85 (d, <i>J</i> = 2.4 Hz, 1H), 7.78 (d, <i>J</i> = 2.4 Hz, 1H), 7.71 (d, <i>J</i> = 2.5 Hz, 1H), 3.17 (s, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 79.0 (p, <i>J</i> = 151 Hz, 1F), 60.7 (d, <i>J</i> = 151 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.01 min, <i>m/z</i> = 566.3, 569.9 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_78" class="NLM_sec NLM_sec_level_2"><div id="ac_i224" class="anchor-spacer"></div><h3 class="article-section__title" id="_i224"> 5-Bromo-3-chloro-<i>N</i>-(3-(1-cyanocyclobutyl)-5-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7x</b>)</h3><div id="sec4_78_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i225" class="anchor-spacer"></div><h4 class="article-section__title" id="_i225"> Step A: 3-Methyl-5-(methylsulfonyl)aniline</h4><div class="NLM_p last">3-Bromo-5-methylaniline (2.0 g, 10.75 mmol), sodium methanesulfinate (1.32 g, 12.9 mmol), CuI (205 mg, 1.08 mmol), <span class="smallcaps smallerCapital">l</span>-proline (248 mg, 2.15 mmol), and NaOH (86 mg, 0.24 mmol) were taken in DMSO (22 mL) under an inert atmosphere and heated to 85 °C for 16 h. Upon cooling, the mixture was diluted with H<sub>2</sub>O and extracted with DCM. The organic phase was dried and concentrated to afford the title compound (1.2 g, 6.46 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.10 (d, <i>J</i> = 1.6 Hz, 1H), 7.01 (d, <i>J</i> = 2.1 Hz, 1H), 6.71 (dd, <i>J</i> = 2.1, 1.6 Hz, 1H), 3.90 (s, 2H), 3.01 (s, 3H), 2.36–2.31 (m, 3H). MS (ESI) <i>m/z</i> = 371.4 [2M + H]<sup>+</sup>.</div></div><div id="sec4_78_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i226" class="anchor-spacer"></div><h4 class="article-section__title" id="_i226"> Step B: <i>tert</i>-Butyl (3-Methyl-5-(methylsulfonyl)phenyl) Carbamate</h4><div class="NLM_p last">A mixture containing 3-methyl-5-(methylsulfonyl)aniline (1196 mg, 6.46 mmol) and di-<i>tert</i>-butyl carbonate (1671 mg, 7.66 mmol) in MeOH (22 mL) was allowed to stir at r.t. for 16 h. The solvent was removed under reduced pressure, and the crude material was purified by flash chromatography to afford the title compound (826 mg, 2.89 mmol, 45% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.71 (t, <i>J</i> = 1.9 Hz, 1H), 7.56 (s, 1H), 7.39 (d, <i>J</i> = 1.9 Hz, 1H), 6.86 (s, 1H), 3.03 (s, 3H), 2.39 (s, 3H), 1.51 (s, 9H). MS (ESI) <i>m/z</i> = 303.2 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_78_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i227" class="anchor-spacer"></div><h4 class="article-section__title" id="_i227"> Step C: <i>tert</i>-Butyl (3-(Bromomethyl-5-(methylsulfonyl)phenyl) Carbamate</h4><div class="NLM_p last"><i>tert</i>-Butyl (3-methyl-5-(methylsulfonyl)phenyl) carbamate (826 mg, 2.89 mmol) and NBS (618 mg, 3.47 mmol) were taken in CCl<sub>4</sub> (25 mL), and the solution was degassed with Ar for 10 min. To this mixture was added AIBN (119 mg, 724 mmol), and the reaction mixture was heated to 80 °C for 16 h. The cooled reaction mixture was filtered. The filtrate was washed first with HCl [1 M] and then with a saturated solution of NaHCO<sub>3</sub>. The organic phase was dried using a phase separator, and the solvent was removed under reduced pressure. The crude material was purified by flash chromatography to give the compound (961 mg, 2.64 mmol, 91% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.84 (d, <i>J</i> = 1.8 Hz, 1H), 7.60 (s, 1H), 6.88 (s, 1H), 4.47 (s, 2H), 3.06 (s, 3H), 1.52 (s, 9H). MS (ESI) <i>m/z</i> = 381.3, 383.2 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_78_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i228" class="anchor-spacer"></div><h4 class="article-section__title" id="_i228"> Step D: <i>tert</i>-Butyl (3-(Cyanomethyl-5-(methylsulfonyl)phenyl) Carbamate</h4><div class="NLM_p last"><i>tert</i>-Butyl (3-(bromomethyl-5-(methylsulfonyl)phenyl) carbamate (961 mg, 2.64 mmol) was dissolved in MeCN (5.0 mL) and cooled to 0 °C. K<sub>2</sub>CO<sub>3</sub> (438 mg, 3.17 mmol) and TMSCN (0.420 mL, 3.17 mmol) were added and heated to 50 °C for 16 h. The cooled mixture was diluted with NaOH [3 M] and extracted with EtOAc. The crude material was purified by flash chromatography to afford the title compound (441 mg, 1.42 mmol, 49% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.89 (t, <i>J</i> = 1.8 Hz, 1H), 7.77 (s, 1H), 7.55 (t, <i>J</i> = 1.8 Hz, 1H), 6.82 (s, 1H), 3.81 (s, 2H), 3.07 (s, 3H), 1.53 (s, 9H). MS (ESI) <i>m/z</i> = 328.3 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_78_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i229" class="anchor-spacer"></div><h4 class="article-section__title" id="_i229"> Step E: <i>tert</i>-Butyl (3-(1-Cyanocyclobutyl)-5-(methylsulfonyl)phenyl) Carbamate</h4><div class="NLM_p last"><i>tert</i>-Butyl (3-(cyanomethyl-5-(methylsulfonyl)phenyl) carbamate (421 mg, 1.36 mmol) and 1,3-dibromopropane (151 μL, 1.49 mmol) were dissolved in DMSO (7 mL), and NaH (60% dispersion in mineral oil, 140 mg, 3.39 mmol) was added with caution. The solution was stirred for 16 h at r.t., diluted with EtOAc/Et<sub>2</sub>O (100 mL, 1:1), and then washed with water (3 × 100 mL). The combined aqueous layers were back-extracted with further EtOAc (100 mL). The combined organics were washed with sat. aq. NaCl, concentrated, and purified by flash column chromatography to afford a colorless oil (311 mg, 0.887 mmol, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.94 (s, 1H), 7.80 (s, 1H), 7.56 (s, 1H), 7.30 (s, 1H), 3.06 (s, 3H), 2.87–2.76 (m, 2H), 2.69–2.56 (m, 2H), 2.43 (dt, <i>J</i> = 11.9, 8.8 Hz, 1H), 2.13–2.04 (m, 1H), 1.50 (s, 9H). LC–MS: <i>t</i><sub>R</sub> = 0.99 min, <i>m/z</i> = 368.3 [M + NH<sub>4</sub>]<sup>+</sup><sub>.</sub></div></div><div id="sec4_78_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i230" class="anchor-spacer"></div><h4 class="article-section__title" id="_i230"> Step F: 1-(3-Amino-5-(methylsulfonyl)phenyl)cyclobutene-1-carbonitrile Hydrochloride</h4><div class="NLM_p last"><i>tert</i>-Butyl (3-(1-cyanocyclobutyl)-5-(methylsulfonyl)phenyl) carbamate (311 mg, 0.887 mmol) was dissolved in a solution of HCl (4 M in dioxane, 5 mL) and heated to 50 °C for 1 h. The solvent was removed under reduced pressure to afford the title compound (253 mg, 0.88 mmol, quant.), which was taken forward without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.14 (s, 1H), 7.96 (s, 1H), 7.82 (s, 1H), 3.09 (s, 3H), 2.79 (s, 2H), 2.61 (d, <i>J</i> = 11.4 Hz, 2H), 2.44–2.34 (m, 1H), 2.07 (s, 1H). LC–MS: <i>t</i><sub>R</sub> = 0.59 min, <i>m/z</i> = 251.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_78_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i231" class="anchor-spacer"></div><h4 class="article-section__title" id="_i231"> Step G: 5-Bromo-3-chloro-<i>N</i>-(3-(1-cyanocyclobutyl)-5-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide</h4><div class="NLM_p last">1-(3-Amino-5-(methylsulfonyl)phenyl)cyclobutene-1-carbonitrile hydrochloride (83 mg, 0.29 mmol) and 5-bromo-3-chloro-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (55 mg, 0.11 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.90 (br s, 1H), 7.78 (d, <i>J</i> = 2.4 Hz, 1H), 7.73 (d, <i>J</i> = 1.8 Hz, 1H), 7.67 (d, <i>J</i> = 2.4 Hz, 1H), 7.62 (d, <i>J</i> = 1.8 Hz, 1H), 7.53 (t, <i>J</i> = 1.8 Hz, 1H), 3.08 (s, 3H), 2.92–2.81 (m, 2H), 2.64–2.52 (m, 2H), 2.52–2.42 (m, 1H), 2.16–2.07 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.81 min, <i>m/z</i> = 518.9, 521.9 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_79" class="NLM_sec NLM_sec_level_2"><div id="ac_i232" class="anchor-spacer"></div><h3 class="article-section__title" id="_i232"> 5-Bromo-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7y</b>)</h3><div id="sec4_79_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i233" class="anchor-spacer"></div><h4 class="article-section__title" id="_i233"> Step A: 1-(4-Methoxy-3-(methylthio)phenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">1-(3-Bromo-4-methoxyphenyl)cyclobutane-1-carbonitrile (200 mg, 0.75 mmol), sodium methanethiolate (55 mg, 0.79 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (17 mg, 0.02 mmol), and xantphos (22 mg 0.04 mmol) were combined in a vial purged with argon. THF (3.75 mL) and Et<sub>3</sub>N (0.13 mL, 0.94 mmol) were added, and the mixture was heated to reflux for 18 h. The cooled mixture was filtered, concentrated, and purified by flash chromatography to afford a yellow oil (174 mg, 0.746 mmol, 99% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.21–7.13 (m, 2H), 6.87–6.80 (m, 1H), 3.90 (s, 3H), 2.88–2.75 (m, 2H), 2.59 (dt, <i>J</i> = 9.8, 8.0 Hz, 2H), 2.45 (s, 3H), 2.43–2.36 (m, 1H), 2.12–2.00 (m, 1H). MS (ESI) <i>m/z</i> = 234.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_79_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i234" class="anchor-spacer"></div><h4 class="article-section__title" id="_i234"> Step B: 1-(4-Methoxy-3-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">1-(4-Methoxy-3-(methylthio)phenyl)cyclobutane-1-carbonitrile (174 mg, 0.671 mmol) was dissolved in EtOH/H<sub>2</sub>O (1:1, 1.5 mL), and oxone (413 mg, 1.34 mmol) was added. The mixture was stirred at r.t. for 24 h. The mixture was diluted with water, extracted with DCM, and dried (MgSO<sub>4</sub>) to afford a pale yellow solid (188 mg, 0.58 mmol, 87% yield), which was taken forward without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.03 (d, <i>J</i> = 2.5 Hz, 1H), 7.67 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 7.10 (d, <i>J</i> = 8.6 Hz, 1H), 2.91–2.81 (m, 1H), 2.68–2.58 (m, 2H), 2.53–2.40 (m, 1H), 2.17–2.07 (m, 1H).</div></div><div id="sec4_79_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i235" class="anchor-spacer"></div><h4 class="article-section__title" id="_i235"> Step C: 1-(4-Hydroxy-3-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">1-(4-Methoxy-3-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile (71 mg, 0.27 mmol) was reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford an off-white solid (68 mg, 0.27 mmol, quant.). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 8.86 (s, 1H), 7.68 (d, <i>J</i> = 2.5 Hz, 1H), 7.58 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.10 (d, <i>J</i> = 8.7 Hz, 1H), 3.15 (s, 3H), 2.91–2.77 (m, 2H), 2.69–2.54 (m, 2H), 2.54–2.39 (m, 1H), 2.17–2.02 (m, 1H).</div></div><div id="sec4_79_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i236" class="anchor-spacer"></div><h4 class="article-section__title" id="_i236"> Step D: 1-(4-Hydroxy-3-(methylsulfonyl)-5-nitrophenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">1-(4-Hydroxy-3-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile (170 mg, 0.68 mmol) was reacted following <a class="ref internalNav" href="#sec4_13" aria-label="General Procedure C: Nitration">General Procedure C: Nitration</a> to afford the title compound (145 mg, 0.49 mmol, 84% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.41 (d, <i>J</i> = 2.5 Hz, 1H), 8.33 (dt, <i>J</i> = 2.2, 1.0 Hz, 1H), 3.34 (s, 3H), 2.95–2.84 (m, 2H), 2.69–2.57 (m, 2H), 2.57–2.43 (m, 1H), 2.19–2.06 (m, 1H). MS (ESI) <i>m/z</i> = 314.3 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_79_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i237" class="anchor-spacer"></div><h4 class="article-section__title" id="_i237"> Step E: 1-(3-Amino-4-hydroxy-5-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile</h4><div class="NLM_p last">1-(4-Hydroxy-3-(methylsulfonyl)-5-nitrophenyl)cyclobutane-1-carbonitrile (145 mg, 0.489 mmol) was reacted following <a class="ref internalNav" href="#sec4_12" aria-label="General Procedure B: Hydrogenation">General Procedure B: Hydrogenation</a> to afford the title compound (122 mg, 0.458 mmol, 94% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.01 (d, <i>J</i> = 2.2 Hz, 1H), 6.93 (d, <i>J</i> = 2.2 Hz, 1H), 3.13 (s, 3H), 2.83–2.72 (m, 2H), 2.62–2.50 (m, 2H), 2.41 (dt, <i>J</i> = 11.5, 8.8 Hz, 1H), 2.11–1.98 (m, 1H). MS (ESI) <i>m/z</i> = 267.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_79_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i238" class="anchor-spacer"></div><h4 class="article-section__title" id="_i238"> Step F: 5-Bromo-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7y</b>)</h4><div class="NLM_p last">1-(3-Amino-4-hydroxy-5-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile (11 mg, 0.04 mmol) and 5-bromo-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (3 mg, 0.006 mmol, 15% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 7.76 (d, <i>J =</i> 2.5 Hz, 1H), 7.68 (d, <i>J =</i> 2.4 Hz, 1H), 7.60 (dd, <i>J =</i> 8.8, 2.5 Hz, 1H), 7.38 (d, <i>J =</i> 2.4 Hz, 1H), 6.96 (d, <i>J =</i> 8.8 Hz, 1H), 3.27 (s, 3H), 2.80–2.65 (m, 2H), 2.56–2.24 (m, 3H), 2.15–1.94 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.44 min, <i>m/z</i> = 501.0, 503.0 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_80" class="NLM_sec NLM_sec_level_2"><div id="ac_i239" class="anchor-spacer"></div><h3 class="article-section__title" id="_i239"> 5-Bromo-3-chloro-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxy-3-(methylsulfonyl)phenyl)-2-hydroxybenzenesulfonamide (<b>7z</b>)</h3><div class="NLM_p last">1-(3-Amino-4-hydroxy-5-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile (15 mg, 0.056 mmol) and 5-bromo-3-chloro-2-hydroxybenzenesulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound (8 mg, 0.015 mmol, 27% yield). <sup>1</sup>H NMR (400 MHz,acetonitrile-<i>d</i><sub>3</sub>): δ<sub>H</sub> 7.81 (d, <i>J</i> = 2.4 Hz, 1H), 7.69 (d, <i>J</i> = 2.4 Hz, 1H), 7.61 (d, <i>J</i> = 2.4 Hz, 1H), 7.46 (d, <i>J</i> = 2.4 Hz, 1H), 3.16 (s, 3H), 2.77–2.70 (m, 2H), 2.53–2.45 (m, 2H), 2.34 (dq, <i>J</i> = 11.5, 8.6 Hz, 1H), 2.06–1.97 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.02 min, <i>m/z</i> = 552.2, 553.2 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_81" class="NLM_sec NLM_sec_level_2"><div id="ac_i240" class="anchor-spacer"></div><h3 class="article-section__title" id="_i240"> 5-Bromo-<i>N</i>-(5-chloro-2-hydroxy-3-(methylsulfonyl)phenyl)-2-methoxybenzenesulfonamide (<b>8</b>)</h3><div class="NLM_p last">Amino-4-chloro-6-(methylsulfonyl)phenol (50 mg, 0.23 mmol) and <b>16a</b> were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (3 mg, 0.006 mmol, 3% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.24 (s, 1H), 8.00 (d, <i>J</i> = 2.5 Hz, 1H), 7.82 (d, <i>J</i> = 2.5 Hz, 1H), 7.68 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.34 (d, <i>J</i> = 2.4 Hz, 1H), 6.89 (d, <i>J</i> = 8.8 Hz, 1H), 3.96 (s, 3H), 3.11 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.055 min, <i>m/z</i> = 489.2 [M + NH<sub>4</sub>]<sup>+</sup>. Purity (AUC), ≥ 95%.</div></div><div id="sec4_82" class="NLM_sec NLM_sec_level_2"><div id="ac_i241" class="anchor-spacer"></div><h3 class="article-section__title" id="_i241"> 3-((5-Bromo-2-methoxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>9</b>)</h3><div class="NLM_p last">Phenyl 3-((5-bromo-2-methoxyphenyl)sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (28 mg, 0.05 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (14 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 7.88 (d, <i>J =</i> 2.5 Hz, 1H), 7.65 (dd, <i>J =</i> 8.9, 2.5 Hz, 1H), 7.56 (d, <i>J =</i> 2.2 Hz, 1H), 7.47 (d, <i>J =</i> 2.2 Hz, 1H), 7.06 (d, <i>J =</i> 8.9 Hz, 1H), 3.90 (s, 3H), 2.89 (s, 3H), 2.81–2.70 (m, 2H), 2.62–2.50 (m, 2H), 2.38 (dq, <i>J =</i> 11.5, 8.5 Hz, 1H), 2.12–1.98 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.74 min, <i>m/z</i> = 493.8, 495.8 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_83" class="NLM_sec NLM_sec_level_2"><div id="ac_i242" class="anchor-spacer"></div><h3 class="article-section__title" id="_i242"> 6-Bromo-<i>N</i>-(5-chloro-2-hydroxy-3-(methylsulfonyl)phenyl)quinoline-8-sulfonamide (<b>10</b>)</h3><div class="NLM_p last">Amino-4-chloro-6-(methylsulfonyl)phenol (10 mg, 0.45 mmol) and 6-bromoquinoline-8-sulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a>, affording the title compound (4 mg, 0.008 mmol, 18% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.10 (dd, <i>J</i> = 4.3, 1.7 Hz, 1H), 8.48 (d, <i>J</i> = 2.2 Hz, 1H), 8.23 (d, <i>J</i> = 2.2 Hz, 1H), 8.20 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 7.93 (d, <i>J</i> = 2.5 Hz, 1H), 7.62 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 7.32 (d, <i>J</i> = 2.5 Hz, 1H), 3.05 (s, 3H). LC–MS: <i>t</i><sub>R</sub> = 1.038 min, <i>m/z</i> = 491.2, 493.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_84" class="NLM_sec NLM_sec_level_2"><div id="ac_i243" class="anchor-spacer"></div><h3 class="article-section__title" id="_i243"> 6-Bromo-<i>N</i>-(5-(1-cyanocyclobutyl)-2-hydroxy-3-(methylsulfonyl)phenyl)quinoline-8-sulfonamide (<b>11</b>)</h3><div class="NLM_p last">1-(3-Amino-4-hydroxy-5-(methylsulfonyl)phenyl)cyclobutane-1-carbonitrile (21 mg, 0.079 mmol) and 6-bromoquinoline-8-sulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound (16 mg, 0.030 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.11 (dd, <i>J</i> = 4.4, 1.6 Hz, 1H), 8.46 (d, <i>J</i> = 2.1 Hz, 1H), 8.25–8.17 (m, 2H), 7.93 (d, <i>J</i> = 2.3 Hz, 1H), 7.62 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 7.37 (d, <i>J</i> = 2.3 Hz, 1H), 3.06 (s, 3H), 2.82 (ddd, <i>J</i> = 11.9, 8.5, 4.1 Hz, 2H), 2.59–2.51 (m, 2H), 2.44 (dq, <i>J</i> = 11.5, 8.7, 8.1 Hz, 1H), 2.14–2.04 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.02 min, <i>m/z</i> = 552.2, 553.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_85" class="NLM_sec NLM_sec_level_2"><div id="ac_i244" class="anchor-spacer"></div><h3 class="article-section__title" id="_i244"> 3-((6-Bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>12</b>)</h3><div id="sec4_85_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i245" class="anchor-spacer"></div><h4 class="article-section__title" id="_i245"> Step A: Phenyl 3-((6-Bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate</h4><div class="NLM_p last">Phenyl 3-amino-5-(1-cyanocyclobutyl)-2-hydroxybenzoate 31 mg, 0.10 mmol) and 6-bromoquinoline-8-sulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_14" aria-label="General Procedure D: Mediated Demethylation">General Procedure D: Mediated Demethylation</a> to afford a colorless solid (22 mg, 0.04 mmol, 38% yield). LC–MS: <i>t</i><sub>R</sub> = 1.18 min, <i>m/z</i> = 578.3, 580.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_85_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i246" class="anchor-spacer"></div><h4 class="article-section__title" id="_i246"> Step B: 3-((6-Bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoic Acid (<b>12</b>)</h4><div class="NLM_p last">Phenyl 3-((6-bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (71 mg, 0.12 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford the title compound (40 mg, 0.08 mmol, 65% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 9.07 (dd, <i>J</i> = 4.3, 1.7 Hz, 1H), 8.41–8.38 (m, 2H), 8.34 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 7.79 (d, <i>J</i> = 2.4 Hz, 1H), 7.65 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 7.56 (d, <i>J</i> = 2.4 Hz, 1H), 2.79–2.69 (m, 2H), 2.59–2.48 (m, 2H), 2.44–2.30 (m, 1H), 2.06 (ddt, <i>J</i> = 9.0, 6.9, 4.5 Hz, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.04 min, <i>m/z</i> = 502.3, 504.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_86" class="NLM_sec NLM_sec_level_2"><div id="ac_i247" class="anchor-spacer"></div><h3 class="article-section__title" id="_i247"> 3-((6-Bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxy-<i>N</i>-methylbenzamide (<b>13</b>)</h3><div class="NLM_p last">Phenyl 3-((6-bromoquinoline)-8-sulfonamido)-5-(1-cyanocyclobutyl)-2-hydroxybenzoate (22 mg, 0.04 mmol) was reacted with methylamine following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford a colorless solid (7 mg, 36% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 9.09 (dd, <i>J =</i> 4.3, 1.7 Hz, 1H), 8.42 (d, <i>J =</i> 2.2 Hz, 1H), 8.40 (d, <i>J =</i> 2.2 Hz, 1H), 8.37 (dd, <i>J =</i> 8.4, 1.8 Hz, 1H), 7.70–7.66 (m, 1H), 7.68 (d, <i>J =</i> 2.3 Hz, 1H), 7.42 (d, <i>J =</i> 2.2 Hz, 1H), 2.82 (s, 3H), 2.79–2.71 (m, 2H), 2.63–2.52 (m, 1H), 2.43–2.32 (m, 1H), 2.11–2.00 (m, 1H). LC–MS: <i>t</i><sub>R</sub> = 1.19 min, <i>m/z</i> = 514.8, 516.7 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div><div id="sec4_87" class="NLM_sec NLM_sec_level_2"><div id="ac_i248" class="anchor-spacer"></div><h3 class="article-section__title" id="_i248"> 3-((6-Bromoquinoline)-8-sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>14</b>)</h3><div id="sec4_87_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i249" class="anchor-spacer"></div><h4 class="article-section__title" id="_i249"> Step A: Methyl 3-((6-Bromoquinoline)-8-sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate</h4><div class="NLM_p last">Methyl 3-amino-2-hydroxy-5-(pentafluorosulfanyl)benzoate (35 mg, 0.12 mmol) and 6-bromoquinoline-8-sulfonyl chloride were reacted following <a class="ref internalNav" href="#sec4_11" aria-label="General Procedure A: Sulfonamide Coupling">General Procedure A: Sulfonamide Coupling</a> to afford the title compound as a colorless solid (28 mg, 0.05 mmol, 42% yield). MS (ESI) <i>m/z</i> = 563.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_87_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i250" class="anchor-spacer"></div><h4 class="article-section__title" id="_i250"> Step B: 3-((6-Bromoquinoline)-8-sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoic Acid (<b>14</b>)</h4><div class="NLM_p last">Methyl 3-((6-bromoquinoline)-8-sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (14 mg, 0.022 mmol) was reacted following <a class="ref internalNav" href="#sec4_15" aria-label="General Procedure E: Ester Hydrolysis">General Procedure E: Ester Hydrolysis</a> to afford the title compound (8 mg, 0.015 mmol, 66% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>H</sub> 9.04 (dd, <i>J</i> = 4.3, 1.7 Hz, 1H), 8.45–8.39 (m, 2H), 8.37 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 8.15 (d, <i>J</i> = 2.8 Hz, 1H), 7.92 (d, <i>J</i> = 2.7 Hz, 1H), 7.67 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H). <sup>19</sup>F NMR (376 MHz, MeOH-<i>d</i><sub>4</sub>): δ<sub>F</sub> 83.0 (p, <i>J</i> = 150 Hz, 1F), 62.1 (d, <i>J</i> = 150 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.13 min, <i>m/z</i> = 549.1, 551.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div id="sec4_88" class="NLM_sec NLM_sec_level_2"><div id="ac_i251" class="anchor-spacer"></div><h3 class="article-section__title" id="_i251"> 3-((6-Bromoquinoline)-8-sulfonamido)-2-hydroxy-<i>N</i>-methyl-5-(pentafluorosulfanyl)benzamide (<b>15</b>)</h3><div class="NLM_p last">Phenyl 3-((6-bromoquinoline)-8-sulfonamido)-2-hydroxy-5-(pentafluorosulfanyl)benzoate (14 mg, 0.022 mmol) was reacted following <a class="ref internalNav" href="#sec4_16" aria-label="General Procedure F: Methylamide Synthesis">General Procedure F: Methylamide Synthesis</a> to afford the title compound (9 mg, 0.015 mmol, 69% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ<sub>H</sub> 9.11 (dd, <i>J</i> = 4.4, 1.7 Hz, 1H), 8.43 (d, <i>J</i> = 2.2 Hz, 1H), 8.22 (d, <i>J</i> = 2.4 Hz, 1H), 8.20–8.12 (m, 2H), 7.58 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 7.41 (d, <i>J</i> = 2.5 Hz, 1H), 6.41–6.36 (m, 1H), 2.94 (d, <i>J</i> = 4.8 Hz, 3H). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>): δ<sub>F</sub> 84.6 (p, <i>J</i> = 150 Hz, 1F), 64.1 (d, <i>J</i> = 150 Hz, 4F). LC–MS: <i>t</i><sub>R</sub> = 1.19 min, <i>m/z</i> = 563.2 [M + H]<sup>+</sup>. Purity (AUC), ≥95%.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i252"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01411" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02206" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02206" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01411?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01411</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structure of FPA probe and negative controls; additional thermal shift data; additional coimmunoprecipitation experiments; histone methyl transferase assay data; X-ray data collection and refinement statistics; experimental details for the synthesis of probe and controls (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf">jm9b01411_si_001.pdf (810.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_002.csv">jm9b01411_si_002.csv (6.28 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01411" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen W. Fesik</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, 
    Department of
Pharmacology, 
    , , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5957-6192" title="Orcid link">http://orcid.org/0000-0001-5957-6192</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#295a5d4c59414c47074f4c5a4042695f48474d4c5b4b40455d074c4d5c"><span class="__cf_email__" data-cfemail="4f3c3b2a3f272a2161292a3c26240f392e212b2a3d2d26233b612a2b3a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan D. Macdonald</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Cleveland Clinic Center for Therapeutics Discovery, Cleveland, Ohio 44195, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6360-2768" title="Orcid link">http://orcid.org/0000-0002-6360-2768</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Selena Chacón Simon</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Changho Han</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Vanderbilt Center for Neuroscience Drug Discovery, Franklin, Tennessee 37067, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Novartis Institutes for Biomedical Research, Emeryville, California 94608, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4322-2127" title="Orcid link">http://orcid.org/0000-0002-4322-2127</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Grace Shaw</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer E. Howes</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        PhoreMost Ltd. Cambridge, CB22 3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiqing Sai</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joannes P. Yuh</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Bluerock Therapeutics, Toronto, ON, M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Demarco Camper</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Bristol-Meyers Squibb, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bethany M. Alicie</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph Alvarado</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Cleveland Clinic Center for Therapeutics Discovery, Cleveland, Ohio 44195, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sameer Nikhar</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William Payne</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erin R. Aho</span> - <span class="hlFld-Affiliation affiliation">Department of Cell and Developmental Biology, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joshua A. Bauer</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Zhao</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Phan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lance R. Thomas</span> - <span class="hlFld-Affiliation affiliation">Department of Cell and Developmental Biology, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olivia W. Rossanese</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        The Institute of Cancer Research, London, SM2 5NG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William P. Tansey</span> - <span class="hlFld-Affiliation affiliation">Department of Cell and Developmental Biology, , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex G. Waterson</span> - <span class="hlFld-Affiliation affiliation">Department of
Pharmacology, 
    , , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0153-5723" title="Orcid link">http://orcid.org/0000-0002-0153-5723</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaun R. Stauffer</span> - <span class="hlFld-Affiliation affiliation">Department of
Pharmacology, 
    , , and , Vanderbilt University School
of Medicine, Nashville, Tennessee 37232, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span>Present Address:
                        Cleveland Clinic Center for Therapeutics Discovery, Cleveland, Ohio 44195, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Atom coordinates and structure factors for WDR5-ligand complexes can be accessed in the PDB via the following accession codes: <b>1</b>, 6U6W; <b>5g</b>, 6U80; <b>5n</b>, 6U8O; <b>6b</b>, 6U8B; <b>6g</b>, 6U8L; <b>7k</b>, 6U5Y; <b>7t</b>, 6U5M. Authors will release the atomic coordinates upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i261">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34072" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34072" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Alexey Kuznetsov and Caden Schlund for their contributions to this series of compounds during this project. The authors thank our co-workers at the High-Throughput Screening Core facility at Vanderbilt University, TN, for compound management services and use of their instrumentation and the Vanderbilt University Biomolecular NMR Facility for instrumentation. This facility receives support from an NIH SIG grant (1S-10RR025677-01) and Vanderbilt University matching funds. Use of the Advanced Photon Source, an Office of Science User Facility operated for the U.S. Department of Energy Office of Science by the Argonne National Laboratory, was supported by the U.S. Department of Energy (contract no. DE-AC02-06CH11357). Use of the Life Sciences Collaborative Access Team Sector 21 was supported by the Michigan Economic Development Corporation and Michigan Technology Tri-Corridor grant (grant no. 085P1000817). This work was supported by a grant from the NIH/NCI (CA200709 to W.P.T). This work was also supported by grants from the Robert J. Kleberg, Jr. and Helen C. Kleberg Foundation (to W.P.T. and S.W.F.), the TJ Martell Foundation (to W.P.T. and S.W.F.), the Edward P. Evans Foundation (to W.P.T.), Alex’s Lemonade Stand Foundation (to W.P.T.), and St. Baldrick’s Foundation (to W.P.T.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">WDR5</td><td class="NLM_def"><p class="first last">WD repeat-containing protein 5</p></td></tr><tr><td class="NLM_term">WBM</td><td class="NLM_def"><p class="first last">WDR5 binding motif</p></td></tr><tr><td class="NLM_term">WIN</td><td class="NLM_def"><p class="first last">WDR5 interaction site</p></td></tr><tr><td class="NLM_term">RBBP5</td><td class="NLM_def"><p class="first last">retinoblastoma-binding protein 5</p></td></tr><tr><td class="NLM_term">KANSL2</td><td class="NLM_def"><p class="first last">KAT8 regulatory NSL complex subunit 2</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed lineage leukemia</p></td></tr><tr><td class="NLM_term">HMT</td><td class="NLM_def"><p class="first last">histone methyl transferase</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">FPA</td><td class="NLM_def"><p class="first last">fluorescence polarization assay</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">RFU</td><td class="NLM_def"><p class="first last">relative fluorescence unit</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i263">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67155" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67155" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groudine, M.</span></span> <span> </span><span class="NLM_article-title">Control of C-MYC Regulation in Normal and Neoplastic Cells</span>. <i>Adv. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/S0065-230X(08)60476-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0065-230X%2808%2960476-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=2028839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK3MXltlarsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=1-48&author=C.+A.+Spencerauthor=M.+Groudine&title=Control+of+C-MYC+Regulation+in+Normal+and+Neoplastic+Cells&doi=10.1016%2FS0065-230X%2808%2960476-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Control of c-myc regulation in normal and neoplastic cells</span></div><div class="casAuthors">Spencer, Charlotte A.; Groudine, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-48</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    </div><div class="casAbstract">A review with ∼200 refs. of normal c-myc gene regulation, and abnormal c-myc gene regulation in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX_UXmyVSX1LVg90H21EOLACvtfcHk0liwqN6jfGJvGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltlarsLc%253D&md5=34146e22b8744855132783e1f96d3d3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0065-230X%2808%2960476-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-230X%252808%252960476-5%26sid%3Dliteratum%253Aachs%26aulast%3DSpencer%26aufirst%3DC.%2BA.%26aulast%3DGroudine%26aufirst%3DM.%26atitle%3DControl%2520of%2520C-MYC%2520Regulation%2520in%2520Normal%2520and%2520Neoplastic%2520Cells%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D1991%26volume%3D56%26spage%3D1%26epage%3D48%26doi%3D10.1016%2FS0065-230X%2808%2960476-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mermel, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychaudhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urashima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mc Henry, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinchback, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haery, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yecies, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demichelis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rustgi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonescu, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">True, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">The Landscape of Somatic Copy-Number Alteration across Human Cancers</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1038/nature08822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature08822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20164920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFemsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=899-905&author=R.+Beroukhimauthor=C.+H.+Mermelauthor=D.+Porterauthor=G.+Weiauthor=S.+Raychaudhuriauthor=J.+Donovanauthor=J.+Barretinaauthor=J.+S.+Boehmauthor=J.+Dobsonauthor=M.+Urashimaauthor=K.+T.+Mc+Henryauthor=R.+M.+Pinchbackauthor=A.+H.+Ligonauthor=Y.-J.+Choauthor=L.+Haeryauthor=H.+Greulichauthor=M.+Reichauthor=W.+Wincklerauthor=M.+S.+Lawrenceauthor=B.+A.+Weirauthor=K.+E.+Tanakaauthor=D.+Y.+Chiangauthor=A.+J.+Bassauthor=A.+Looauthor=C.+Hoffmanauthor=J.+Prensnerauthor=T.+Liefeldauthor=Q.+Gaoauthor=D.+Yeciesauthor=S.+Signorettiauthor=E.+Maherauthor=F.+J.+Kayeauthor=H.+Sasakiauthor=J.+E.+Tepperauthor=J.+A.+Fletcherauthor=J.+Taberneroauthor=J.+Baselgaauthor=M.-S.+Tsaoauthor=F.+Demichelisauthor=M.+A.+Rubinauthor=P.+A.+Janneauthor=M.+J.+Dalyauthor=C.+Nuceraauthor=R.+L.+Levineauthor=B.+L.+Ebertauthor=S.+Gabrielauthor=A.+K.+Rustgiauthor=C.+R.+Antonescuauthor=M.+Ladanyiauthor=A.+Letaiauthor=L.+A.+Garrawayauthor=M.+Lodaauthor=D.+G.+Beerauthor=L.+D.+Trueauthor=A.+Okamotoauthor=S.+L.+Pomeroyauthor=S.+Singerauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=G.+Getzauthor=W.+R.+Sellersauthor=M.+Meyerson&title=The+Landscape+of+Somatic+Copy-Number+Alteration+across+Human+Cancers&doi=10.1038%2Fnature08822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The landscape of somatic copy-number alteration across human cancers</span></div><div class="casAuthors">Beroukhim, Rameen; Mermel, Craig H.; Porter, Dale; Wei, Guo; Raychaudhuri, Soumya; Donovan, Jerry; Barretina, Jordi; Boehm, Jesse S.; Dobson, Jennifer; Urashima, Mitsuyoshi; Mc Henry, Kevin T.; Pinchback, Reid M.; Ligon, Azra H.; Cho, Yoon-Jae; Haery, Leila; Greulich, Heidi; Reich, Michael; Winckler, Wendy; Lawrence, Michael S.; Weir, Barbara A.; Tanaka, Kumiko E.; Chiang, Derek Y.; Bass, Adam J.; Loo, Alice; Hoffman, Carter; Prensner, John; Liefeld, Ted; Gao, Qing; Yecies, Derek; Signoretti, Sabina; Maher, Elizabeth; Kaye, Frederic J.; Sasaki, Hidefumi; Tepper, Joel E.; Fletcher, Jonathan A.; Tabernero, Josep; Baselga, Jose; Tsao, Ming-Sound; Demichelis, Francesca; Rubin, Mark A.; Janne, Pasi A.; Daly, Mark J.; Nucera, Carmelo; Levine, Ross L.; Ebert, Benjamin L.; Gabriel, Stacey; Rustgi, Anil K.; Antonescu, Cristina R.; Ladanyi, Marc; Letai, Anthony; Garraway, Levi A.; Loda, Massimo; Beer, David G.; True, Lawrence D.; Okamoto, Aikou; Pomeroy, Scott L.; Singer, Samuel; Golub, Todd R.; Lander, Eric S.; Getz, Gad; Sellers, William R.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7283</span>),
    <span class="NLM_cas:pages">899-905</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A powerful way to discover key genes with causal roles in oncogenesis is to identify genomic regions that undergo frequent alteration in human cancers.  Here we present high-resoln. analyses of somatic copy-no. alterations (SCNAs) from 3131 cancer specimens, belonging largely to 26 histol. types.  We identify 158 regions of focal SCNA that are altered at significant frequency across several cancer types, of which 122 cannot be explained by the presence of a known cancer target gene located within these regions.  Several gene families are enriched among these regions of focal SCNA, including the BCL2 family of apoptosis regulators and the NF-κB pathway.  We show that cancer cells contg. amplifications surrounding the MCL1 and BCL2L1 anti-apoptotic genes depend on the expression of these genes for survival.  Finally, we demonstrate that a large majority of SCNAs identified in individual cancer types are present in several cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCxYRlqR0wXLVg90H21EOLACvtfcHk0liwqN6jfGJvGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFemsr0%253D&md5=be6d81f5449cba96dfb4659106f706b0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature08822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08822%26sid%3Dliteratum%253Aachs%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DMermel%26aufirst%3DC.%2BH.%26aulast%3DPorter%26aufirst%3DD.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DRaychaudhuri%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DDobson%26aufirst%3DJ.%26aulast%3DUrashima%26aufirst%3DM.%26aulast%3DMc%2BHenry%26aufirst%3DK.%2BT.%26aulast%3DPinchback%26aufirst%3DR.%2BM.%26aulast%3DLigon%26aufirst%3DA.%2BH.%26aulast%3DCho%26aufirst%3DY.-J.%26aulast%3DHaery%26aufirst%3DL.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DTanaka%26aufirst%3DK.%2BE.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26aulast%3DBass%26aufirst%3DA.%2BJ.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DHoffman%26aufirst%3DC.%26aulast%3DPrensner%26aufirst%3DJ.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DYecies%26aufirst%3DD.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DMaher%26aufirst%3DE.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DTepper%26aufirst%3DJ.%2BE.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DTsao%26aufirst%3DM.-S.%26aulast%3DDemichelis%26aufirst%3DF.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DNucera%26aufirst%3DC.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DRustgi%26aufirst%3DA.%2BK.%26aulast%3DAntonescu%26aufirst%3DC.%2BR.%26aulast%3DLadanyi%26aufirst%3DM.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DLoda%26aufirst%3DM.%26aulast%3DBeer%26aufirst%3DD.%2BG.%26aulast%3DTrue%26aufirst%3DL.%2BD.%26aulast%3DOkamoto%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DSinger%26aufirst%3DS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DThe%2520Landscape%2520of%2520Somatic%2520Copy-Number%2520Alteration%2520across%2520Human%2520Cancers%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D899%26epage%3D905%26doi%3D10.1038%2Fnature08822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Mammalian MYC Proteins and Cancer</span>. <i>New J. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1155/2014/757534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1155%2F2014%2F757534" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=1-27&author=W.+P.+Tansey&title=Mammalian+MYC+Proteins+and+Cancer&doi=10.1155%2F2014%2F757534"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1155%2F2014%2F757534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F757534%26sid%3Dliteratum%253Aachs%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DMammalian%2520MYC%2520Proteins%2520and%2520Cancer%26jtitle%3DNew%2520J.%2520Sci.%26date%3D2014%26volume%3D2014%26spage%3D1%26epage%3D27%26doi%3D10.1155%2F2014%2F757534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the “undruggable” Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%9Cundruggable%E2%80%9D+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0lhQIvDyVQZY0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%259Cundruggable%25E2%2580%259D%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Strategies to Inhibit MYC</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">a014266</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a014266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1101%2Fcshperspect.a014266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25274755" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=a014266&author=M.+R.+McKeownauthor=J.+E.+Bradner&title=Therapeutic+Strategies+to+Inhibit+MYC&doi=10.1101%2Fcshperspect.a014266"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a014266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a014266%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DTherapeutic%2520Strategies%2520to%2520Inhibit%2520MYC%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2014%26volume%3D4%26spage%3Da014266%26doi%3D10.1101%2Fcshperspect.a014266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Strategies to Inhibit MYC and Their Clinical Applicability</span>. <i>Front. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.3389/fcell.2017.00010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3389%2Ffcell.2017.00010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28280720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADC%252BC1czkvFamtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=10&author=J.+R.+Whitfieldauthor=M.-E.+Beaulieuauthor=L.+Soucek&title=Strategies+to+Inhibit+MYC+and+Their+Clinical+Applicability&doi=10.3389%2Ffcell.2017.00010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to Inhibit Myc and Their Clinical Applicability</span></div><div class="casAuthors">Whitfield Jonathan R; Beaulieu Marie-Eve; Soucek Laura</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in cell and developmental biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn">2296-634X</span>.
    </div><div class="casAbstract">Myc is an oncogene deregulated in most-perhaps all-human cancers.  Each Myc family member, c-, L-, and N-Myc, has been connected to tumor progression and maintenance.  Myc is recognized as a "most wanted" target for cancer therapy, but has for many years been considered undruggable, mainly due to its nuclear localization, lack of a defined ligand binding site, and physiological function essential to the maintenance of normal tissues.  The challenge of identifying a pharmacophore capable of overcoming these hurdles is reflected in the current absence of a clinically-viable Myc inhibitor.  The first attempts to inhibit Myc used antisense technology some three decades ago, followed by small molecule inhibitors discovered through "classical" compound library screens.  Notable breakthroughs proving the feasibility of systemic Myc inhibition were made with the Myc dominant negative mutant Omomyc, showing both the great promise in targeting this infamous oncogene for cancer treatment as well as allaying fears about the deleterious side effects that Myc inhibition might have on normal proliferating tissues.  During this time many other strategies have appeared in an attempt to drug the undruggable, including direct and indirect targeting, knockdown, protein/protein and DNA interaction inhibitors, and translation and expression regulation.  The inhibitors range from traditional small molecules to natural chemicals, to RNA and antisense, to peptides and miniproteins.  Here, we briefly describe the many approaches taken so far, with a particular focus on their potential clinical applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQn7rW6dLAkzqx48Rj5GnGofW6udTcc2eZKagYByXYhb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czkvFamtw%253D%253D&md5=5e65b897b6bec6b93de5fbf22a05b226</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffcell.2017.00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffcell.2017.00010%26sid%3Dliteratum%253Aachs%26aulast%3DWhitfield%26aufirst%3DJ.%2BR.%26aulast%3DBeaulieu%26aufirst%3DM.-E.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DStrategies%2520to%2520Inhibit%2520MYC%2520and%2520Their%2520Clinical%2520Applicability%26jtitle%3DFront.%2520Cell%2520Dev.%2520Biol.%26date%3D2017%26volume%3D5%26spage%3D10%26doi%3D10.3389%2Ffcell.2017.00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalkat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resetca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lourenco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiah, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunnerhagen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Done, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raught, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">MYC Protein Interactome Profiling Reveals Functionally Distinct Regions That Cooperate to Drive Tumorigenesis</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">836</span>– <span class="NLM_lpage">848.e7</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.09.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2018.09.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=30415952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCgsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2018&pages=836-848.e7&author=M.+Kalkatauthor=D.+Resetcaauthor=C.+Lourencoauthor=P.-K.+Chanauthor=Y.+Weiauthor=Y.-J.+Shiahauthor=N.+Vitkinauthor=Y.+Tongauthor=M.+Sunnerhagenauthor=S.+J.+Doneauthor=P.+C.+Boutrosauthor=B.+Raughtauthor=L.+Z.+Penn&title=MYC+Protein+Interactome+Profiling+Reveals+Functionally+Distinct+Regions+That+Cooperate+to+Drive+Tumorigenesis&doi=10.1016%2Fj.molcel.2018.09.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MYC Protein Interactome Profiling Reveals Functionally Distinct Regions that Cooperate to Drive Tumorigenesis</span></div><div class="casAuthors">Kalkat, Manpreet; Resetca, Diana; Lourenco, Corey; Chan, Pak-Kei; Wei, Yong; Shiah, Yu-Jia; Vitkin, Natasha; Tong, Yufeng; Sunnerhagen, Maria; Done, Susan J.; Boutros, Paul C.; Raught, Brian; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">836-848.e7</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Transforming members of the MYC family (MYC, MYCL1, and MYCN) encode transcription factors contg. six highly conserved regions, termed MYC homol. boxes (MBs).  By conducting proteomic profiling of the MB interactomes, we demonstrate that half of the MYC interactors require one or more MBs for binding.  Comprehensive phenotypic analyses reveal that two MBs, MB0 and MBII, are universally required for transformation.  MBII mediates interactions with acetyltransferase-contg. complexes, enabling histone acetylation, and is essential for MYC-dependent tumor initiation.  By contrast, MB0 mediates interactions with transcription elongation factors via direct binding to the general transcription factor TFIIF.  MB0 is dispensable for tumor initiation but is a major accelerator of tumor growth.  Notably, the full transforming activity of MYC can be restored by co-expression of the non-transforming MB0 and MBII deletion proteins, indicating that these two regions confer sep. mol. functions, both of which are required for oncogenic MYC activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYfeSlTYyXB7Vg90H21EOLACvtfcHk0lhzoIdr8b8KEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCgsbrJ&md5=119154face5629c3133fe3a6eeea1989</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.09.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.09.031%26sid%3Dliteratum%253Aachs%26aulast%3DKalkat%26aufirst%3DM.%26aulast%3DResetca%26aufirst%3DD.%26aulast%3DLourenco%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DP.-K.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DShiah%26aufirst%3DY.-J.%26aulast%3DVitkin%26aufirst%3DN.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DSunnerhagen%26aufirst%3DM.%26aulast%3DDone%26aufirst%3DS.%2BJ.%26aulast%3DBoutros%26aufirst%3DP.%2BC.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DMYC%2520Protein%2520Interactome%2520Profiling%2520Reveals%2520Functionally%2520Distinct%2520Regions%2520That%2520Cooperate%2520to%2520Drive%2520Tumorigenesis%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D72%26spage%3D836%26epage%3D848.e7%26doi%3D10.1016%2Fj.molcel.2018.09.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desharnais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonderegger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maslyar, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Antagonists of MYC/Max Dimerization Inhibit MYC-Induced Transformation of Chicken Embryo Fibroblasts</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">3830</span>– <span class="NLM_lpage">3835</span>, <span class="refDoi"> DOI: 10.1073/pnas.062036999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.062036999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=11891322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1KltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=3830-3835&author=T.+Bergauthor=S.+B.+Cohenauthor=J.+Desharnaisauthor=C.+Sondereggerauthor=D.+J.+Maslyarauthor=J.+Goldbergauthor=D.+L.+Bogerauthor=P.+K.+Vogt&title=Small-Molecule+Antagonists+of+MYC%2FMax+Dimerization+Inhibit+MYC-Induced+Transformation+of+Chicken+Embryo+Fibroblasts&doi=10.1073%2Fpnas.062036999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule antagonists of Myc/max dimerization inhibit Myc-induced transformation of chicken embryo fibroblasts</span></div><div class="casAuthors">Berg, Thorsten; Cohen, Steven B.; Desharnais, Joel; Sonderegger, Corinna; Maslyar, Daniel J.; Goldberg, Joel; Boger, Dale L.; Vogt, Peter K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3830-3835</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myc is a transcriptional regulator of the basic helix-loop-helix leucine zipper protein family.  It has strong oncogenic potential, mutated or virally transduced forms of Myc induce lymphoid tumors in animals, and deregulated expression of Myc is assocd. with numerous types of human cancers.  For its oncogenic activity, Myc must dimerize with the ubiquitously expressed basic helix-loop-helix leucine zipper protein Max.  This requirement for dimerization may allow control of Myc activity with small mols. that interfere with Myc/Max dimerization.  We have measured Myc/Max dimerization with fluorescence resonance energy transfer and have screened combinatorial chem. libraries for inhibitors of dimerization.  Candidate inhibitors were isolated from a peptidomimetics library.  Inhibition of Myc/Max interaction was validated by ELISA and electrophoretic mobility-shift assay.  Two of the candidate inhibitors also interfere with Myc-induced oncogenic transformation in chicken embryo fibroblast cultures.  Our work provides proof of principle for the identification of small mol. inhibitors of protein-protein interactions by using high-throughput screens of combinatorial chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkAaKA-IEYw7Vg90H21EOLACvtfcHk0lhzoIdr8b8KEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1KltL0%253D&md5=7712f526695657c3a31fdbd722f10b40</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062036999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062036999%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DS.%2BB.%26aulast%3DDesharnais%26aufirst%3DJ.%26aulast%3DSonderegger%26aufirst%3DC.%26aulast%3DMaslyar%26aufirst%3DD.%2BJ.%26aulast%3DGoldberg%26aufirst%3DJ.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26atitle%3DSmall-Molecule%2520Antagonists%2520of%2520MYC%252FMax%2520Dimerization%2520Inhibit%2520MYC-Induced%2520Transformation%2520of%2520Chicken%2520Embryo%2520Fibroblasts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D3830%26epage%3D3835%26doi%3D10.1073%2Fpnas.062036999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giap, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Low Molecular Weight Inhibitors of MYC–Max Interaction and Function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6159</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1206641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fsj.onc.1206641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=13679853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntlWhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=6151-6159&author=X.+Yinauthor=C.+Giapauthor=J.+S.+Lazoauthor=E.+V.+Prochownik&title=Low+Molecular+Weight+Inhibitors+of+MYC%E2%80%93Max+Interaction+and+Function&doi=10.1038%2Fsj.onc.1206641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Low molecular weight inhibitors of Myc-Max interaction and function</span></div><div class="casAuthors">Yin, Xiaoying; Giap, Christine; Lazo, John S.; Prochownik, Edward V.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">6151-6159</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">C-Myc is helix-loop-helix-leucine zipper (HLH-ZIP) oncoprotein that is frequently deregulated in human cancers.  To bind DNA, regulate target gene expression, and function in a biol. context, c-Myc must dimerize with another HLH-ZIP protein, Max.  A large no. of c-Myc target genes have been identified, and many of the encoded proteins are transforming.  Such functional redundancy, however, complicates therapeutic strategies aimed at inhibiting any single target gene product.  Given this consideration, we have instead attempted to identify ways by which c-Myc itself could be effectively disabled.  We have used a yeast two-hybrid approach to identify low-mol.-wt. compds. that inhibit c-Myc-Max assocn.  All of the compds. prevented transactivation by c-Myc-Max heterodimers, inhibited cell cycle progression, and prevented the in vitro growth of fibroblasts in a c-Myc-dependent manner.  Several of the compds. also inhibited tumor growth in vivo.  These results show that the yeast two-hybrid screen is useful for identifying compds. that can be exploited in mammalian cells.  More specifically, they provide a means by which structural analogs, based upon these first-generation Myc-Max inhibitors, can be developed to enhance antitumor efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowgLbI0hqQcLVg90H21EOLACvtfcHk0lhzoIdr8b8KEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntlWhtLs%253D&md5=3a418e091182370b30b5008aafc47c96</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1206641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1206641%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DX.%26aulast%3DGiap%26aufirst%3DC.%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DLow%2520Molecular%2520Weight%2520Inhibitors%2520of%2520MYC%25E2%2580%2593Max%2520Interaction%2520and%2520Function%26jtitle%3DOncogene%26date%3D2003%26volume%3D22%26spage%3D6151%26epage%3D6159%26doi%3D10.1038%2Fsj.onc.1206641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabato, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Disruption of MYC-Max Heterodimerization with Improved Cell-Penetrating Analogs of the Small Molecule 10074-G5</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.18632%2Foncotarget.1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=23801058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADC%252BC3sjlsV2ksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=936-947&author=H.+Wangauthor=J.+Chauhanauthor=A.+Huauthor=K.+Pendletonauthor=J.+L.+Yapauthor=P.+E.+Sabatoauthor=J.+W.+Jonesauthor=M.+Perriauthor=J.+Yuauthor=E.+Cioneauthor=M.+A.+Kaneauthor=S.+Fletcherauthor=E.+V.+Prochownik&title=Disruption+of+MYC-Max+Heterodimerization+with+Improved+Cell-Penetrating+Analogs+of+the+Small+Molecule+10074-G5&doi=10.18632%2Foncotarget.1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of Myc-Max heterodimerization with improved cell-penetrating analogs of the small molecule 10074-G5</span></div><div class="casAuthors">Wang Huabo; Chauhan Jay; Hu Angela; Pendleton Kelsey; Yap Jeremy L; Sabato Philip E; Jones Jace W; Perri Mariarita; Yu Jianshi; Cione Erika; Kane Maureen A; Fletcher Steven; Prochownik Edward V</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">936-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The c-Myc (Myc) oncoprotein is a high-value therapeutic target given that it is deregulated in multiple types of cancer.  However, potent small molecule inhibitors of Myc have been difficult to identify, particularly those whose mechanism relies on blocking the association between Myc and its obligate heterodimerization partner, Max.  We have recently reported a structure-activity relationship study of one such small molecule, 10074-G5, and generated an analog, JY-3-094, with significantly improved ability to prevent or disrupt the association between recombinant Myc and Max proteins.  However, JY-3094 penetrates cells poorly.  Here, we show that esterification of a critical para-carboxylic acid function of JY-3-094 by various blocking groups significantly improves cellular uptake although it impairs the ability to disrupt Myc-Max association in vitro.  These pro-drugs are highly concentrated within cells where JY-3-094 is then generated by the action of esterases.  However, the pro-drugs are also variably susceptible to extracellular esterases, which can deplete extracellular reservoirs.  Furthermore, while JY-3-094 is retained by cells for long periods of time, much of it is compartmentalized within the cytoplasm in a form that appears to be less available to interact with Myc.  Our results suggest that persistently high extracellular levels of pro-drug, without excessive susceptibility to extracellular esterases, are critical to establishing and maintaining intracellular levels of JY-3-094 that are sufficient to provide for long-term inhibition of Myc-Max association.  Analogs of JY-3-094 appear to represent promising small molecule Myc inhibitors that warrant further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUpo3PsQLURaqcneEtCi92fW6udTcc2eahyC_xhBcDj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjlsV2ksA%253D%253D&md5=044730ff03610d8981f13a77f0edb10a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1108%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DChauhan%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DA.%26aulast%3DPendleton%26aufirst%3DK.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DSabato%26aufirst%3DP.%2BE.%26aulast%3DJones%26aufirst%3DJ.%2BW.%26aulast%3DPerri%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DCione%26aufirst%3DE.%26aulast%3DKane%26aufirst%3DM.%2BA.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DDisruption%2520of%2520MYC-Max%2520Heterodimerization%2520with%2520Improved%2520Cell-Penetrating%2520Analogs%2520of%2520the%2520Small%2520Molecule%252010074-G5%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D936%26epage%3D947%26doi%3D10.18632%2Foncotarget.1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of C-MYC/Max Dimerization and DNA Binding by Small Molecules</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">751</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.chembiol.2006.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=16873022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD28XntlGis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=745-751&author=A.+Kiesslingauthor=B.+Sperlauthor=A.+Hollisauthor=D.+Eickauthor=T.+Berg&title=Selective+Inhibition+of+C-MYC%2FMax+Dimerization+and+DNA+Binding+by+Small+Molecules&doi=10.1016%2Fj.chembiol.2006.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of c-Myc/Max Dimerization and DNA Binding by Small Molecules</span></div><div class="casAuthors">Kiessling, Anke; Sperl, Bianca; Hollis, Angela; Eick, Dirk; Berg, Thorsten</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">745-751</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: bZip and bHLHZip protein family members comprise a large fraction of eukaryotic transcription factors and need to bind DNA in order to exert most of their fundamental biol. roles.  Their binding to DNA requires homo- or heterodimerization via α-helical domains, which generally do not contain obvious binding sites for small mols.  We have identified two small mols., dubbed Mycro1 and Mycro2, which inhibit the protein-protein interactions between the bHLHZip proteins c-Myc and Max.  Mycros are the first inhibitors of c-Myc/Max dimerization, which have been demonstrated to inhibit DNA binding of c-Myc with preference over other dimeric transcription factors in vitro.  Mycros inhibit c-Myc-dependent proliferation, gene transcription, and oncogenic transformation in the low micromolar concn. range.  Our data support the idea that dimeric transcription factors can be druggable even in the absence of obvious small-mol. binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwi1DGz-5I0bVg90H21EOLACvtfcHk0lgKUEux6bHS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntlGis7Y%253D&md5=e90e63fc10e677504f217b922d6c1d04</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DKiessling%26aufirst%3DA.%26aulast%3DSperl%26aufirst%3DB.%26aulast%3DHollis%26aufirst%3DA.%26aulast%3DEick%26aufirst%3DD.%26aulast%3DBerg%26aufirst%3DT.%26atitle%3DSelective%2520Inhibition%2520of%2520C-MYC%252FMax%2520Dimerization%2520and%2520DNA%2520Binding%2520by%2520Small%2520Molecules%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D745%26epage%3D751%26doi%3D10.1016%2Fj.chembiol.2006.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baksh, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bister, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janda, K. D.</span></span> <span> </span><span class="NLM_article-title">Inhibitor of MYC Identified in a Kröhnke Pyridine Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12556</span>– <span class="NLM_lpage">12561</span>, <span class="refDoi"> DOI: 10.1073/pnas.1319488111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.1319488111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25114221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12556-12561&author=J.+R.+Hartauthor=A.+L.+Garnerauthor=J.+Yuauthor=Y.+Itoauthor=M.+Sunauthor=L.+Uenoauthor=J.-K.+Rheeauthor=M.+M.+Bakshauthor=E.+Stefanauthor=M.+Hartlauthor=K.+Bisterauthor=P.+K.+Vogtauthor=K.+D.+Janda&title=Inhibitor+of+MYC+Identified+in+a+Kr%C3%B6hnke+Pyridine+Library&doi=10.1073%2Fpnas.1319488111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor of MYC identified in a Krohnke pyridine library</span></div><div class="casAuthors">Hart, Jonathan R.; Garner, Amanda L.; Yu, Jing; Ito, Yoshihiro; Sun, Minghao; Ueno, Lynn; Rhee, Jin-Kyu; Baksh, Michael M.; Stefan, Eduard; Hartl, Markus; Bister, Klaus; Vogt, Peter K.; Janda, Kim D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12556-12561</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a fluorescence polarization screen for the MYC-MAX interaction, we have identified a novel small-mol. inhibitor of MYC, KJ-Pyr-9, from a Krohnke pyridine library.  The Kd of KJ-Pyr-9 for MYC in vitro is 6.5 ± 1.0 nM, as detd. by backscattering interferometry; KJ-Pyr-9 also interferes with MYC-MAX complex formation in the cell, as shown in a protein fragment complementation assay.  KJ-Pyr-9 specifically inhibits MYC-induced oncogenic transformation in cell culture; it has no or only weak effects on the oncogenic activity of several unrelated oncoproteins.  KJ-Pyr-9 preferentially interferes with the proliferation of MYC-overexpressing human and avian cells and specifically reduces the MYC-driven transcriptional signature.  In vivo, KJ-Pyr-9 effectively blocks the growth of a xenotransplant of MYC-amplified human cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNNR9yUaQgSLVg90H21EOLACvtfcHk0lgKUEux6bHS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nF&md5=97a65b219a5b98e1df5b2c264dfc2723</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1319488111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1319488111%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DJ.%2BR.%26aulast%3DGarner%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DM.%26aulast%3DUeno%26aufirst%3DL.%26aulast%3DRhee%26aufirst%3DJ.-K.%26aulast%3DBaksh%26aufirst%3DM.%2BM.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DHartl%26aufirst%3DM.%26aulast%3DBister%26aufirst%3DK.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26aulast%3DJanda%26aufirst%3DK.%2BD.%26atitle%3DInhibitor%2520of%2520MYC%2520Identified%2520in%2520a%2520Kr%25C3%25B6hnke%2520Pyridine%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12556%26epage%3D12561%26doi%3D10.1073%2Fpnas.1319488111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerowitz, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekritz, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonds, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthay, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span> <span> </span><span class="NLM_article-title">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.ccr.2014.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25175806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=414-427&author=W.+C.+Gustafsonauthor=J.+G.+Meyerowitzauthor=E.+A.+Nekritzauthor=J.+Chenauthor=C.+Benesauthor=E.+Charronauthor=E.+F.+Simondsauthor=R.+Seegerauthor=K.+K.+Matthayauthor=N.+T.+Hertzauthor=M.+Eilersauthor=K.+M.+Shokatauthor=W.+A.+Weiss&title=Drugging+MYCN+through+an+Allosteric+Transition+in+Aurora+Kinase+A&doi=10.1016%2Fj.ccr.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span></div><div class="casAuthors">Gustafson, William Clay; Meyerowitz, Justin Gabriel; Nekritz, Erin A.; Chen, Justin; Benes, Cyril; Charron, Elise; Simonds, Erin F.; Seeger, Robert; Matthay, Katherine K.; Hertz, Nicholas T.; Eilers, Martin; Shokat, Kevan M.; Weiss, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">414-427</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC proteins are major drivers of cancer, yet are considered undruggable because their DNA binding domains are composed of two extended alpha helixes with no apparent surfaces for small-mol. binding.  Proteolytic degrdn. of MYCN protein is regulated in part by a kinase-independent function of Aurora A.  We describe a class of inhibitors that disrupts the native conformation of Aurora A and drives the degrdn. of MYCN protein across MYCN-driven cancers.  Comparison of cocrystal structures with structure-activity relationships across multiple inhibitors and chemotypes, coupled with mechanistic studies and biochem. assays, delineates an Aurora A conformation-specific effect on proteolytic degrdn. of MYCN, rather than simple nanomolar-level inhibition of Aurora A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8At2LNPE-ubVg90H21EOLACvtfcHk0lhcsSJsFCxIYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ&md5=20944894851fe5466422c149a0a0b266</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DW.%2BC.%26aulast%3DMeyerowitz%26aufirst%3DJ.%2BG.%26aulast%3DNekritz%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%26aulast%3DCharron%26aufirst%3DE.%26aulast%3DSimonds%26aufirst%3DE.%2BF.%26aulast%3DSeeger%26aufirst%3DR.%26aulast%3DMatthay%26aufirst%3DK.%2BK.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26atitle%3DDrugging%2520MYCN%2520through%2520an%2520Allosteric%2520Transition%2520in%2520Aurora%2520Kinase%2520A%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D26%26spage%3D414%26epage%3D427%26doi%3D10.1016%2Fj.ccr.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carstensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of N-MYC Binding by Aurora-A and Its Destabilization by Kinase Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13726</span>– <span class="NLM_lpage">13731</span>, <span class="refDoi"> DOI: 10.1073/pnas.1610626113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.1610626113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27837025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCiur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=13726-13731&author=M.+W.+Richardsauthor=S.+G.+Burgessauthor=E.+Poonauthor=A.+Carstensenauthor=M.+Eilersauthor=L.+Cheslerauthor=R.+Bayliss&title=Structural+Basis+of+N-MYC+Binding+by+Aurora-A+and+Its+Destabilization+by+Kinase+Inhibitors&doi=10.1073%2Fpnas.1610626113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of N-Myc binding by Aurora-A and its destabilization by kinase inhibitors</span></div><div class="casAuthors">Richards, Mark W.; Burgess, Selena G.; Poon, Evon; Carstensen, Anne; Eilers, Martin; Chesler, Louis; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">13726-13731</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myc family proteins promote cancer by inducing widespread changes in gene expression.  Their rapid turnover by the ubiquitin-proteasome pathway is regulated through phosphorylation of Myc Box I and ubiquitination by E3 ubiquitin ligase SCFFbxW7.  However, N-Myc protein (the product of the MYCN oncogene) is stabilized in neuroblastoma by the protein kinase Aurora-A in a manner that is sensitive to certain Aurora-A-selective inhibitors.  Here, the authors identified a direct interaction between the catalytic domain of Aurora-A kinase and a site flanking Myc Box I that also binds SCFFbxW7.  The authors detd. the crystal structure of the complex between Aurora-A kinase and this region of N-Myc to 1.72-Å resoln.  The structure indicated that the conformation of Aurora-A induced by compds. such as alisertib and CD532 was not compatible with the binding of N-Myc, explaining the activity of these compds. in neuroblastoma cells and providing a rational basis for the design of cancer therapeutics optimized for destabilization of the complex.  The authors also proposed a model for the stabilization mechanism in which binding to Aurora-A alters how N-Myc interacts with SCFFbxW7 to disfavor the generation of Lys-48-linked polyubiquitin chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAK31pqP6M4bVg90H21EOLACvtfcHk0lhcsSJsFCxIYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCiur7I&md5=0aa2b960303f040d9685fb16fb8ea980</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1610626113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1610626113%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DPoon%26aufirst%3DE.%26aulast%3DCarstensen%26aufirst%3DA.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DStructural%2520Basis%2520of%2520N-MYC%2520Binding%2520by%2520Aurora-A%2520and%2520Its%2520Destabilization%2520by%2520Kinase%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D13726%26epage%3D13731%26doi%3D10.1073%2Fpnas.1610626113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span> <span> </span><span class="NLM_article-title">A Moving Target: Structure and Disorder in Pursuit of MYC Inhibitors</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1042/BST20160328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1042%2FBST20160328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28620032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWjsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=709-717&author=R.+Baylissauthor=S.+G.+Burgessauthor=E.+Leenauthor=M.+W.+Richards&title=A+Moving+Target%3A+Structure+and+Disorder+in+Pursuit+of+MYC+Inhibitors&doi=10.1042%2FBST20160328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A moving target: structure and disorder in pursuit of Myc inhibitors</span></div><div class="casAuthors">Bayliss, Richard; Burgess, Selena G.; Leen, Eoin; Richards, Mark W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">709-717</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The Myc proteins comprise a family of ubiquitous regulators of gene expression implicated in over half of all human cancers.  They interact with a large no. of other proteins, such as transcription factors, chromatin-modifying enzymes and kinases.  Remarkably, few of these interactions have been characterized structurally.  This is at least in part due to the intrinsically disordered nature of Myc proteins, which adopt a defined conformation only in the presence of binding partners.  Owing to this behavior, crystallog. studies on Myc proteins have been limited to short fragments in complex with other proteins.  Most recently, we detd. the crystal structure of Aurora-A kinase domain bound to a 28-amino acid fragment of the N-Myc transactivation domain.  The structure reveals an α-helical segment within N-Myc capped by two tryptophan residues that recognize the surface of Aurora-A.  The kinase domain acts as a mol. scaffold, independently of its catalytic activity, upon which this region of N-Myc becomes ordered.  The binding site for N-Myc on Aurora-A is disrupted by certain ATP-competitive inhibitors, such as MLN8237 (alisertib) and CD532, and explains how these kinase inhibitors are able to disrupt the protein-protein interaction to affect Myc destabilization.  Structural studies on this and other Myc complexes will lead to the design of protein-protein interaction inhibitors as chem. tools to dissect the complex pathways of Myc regulation and function, which may be developed into Myc inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv3uTLeXzAGbVg90H21EOLACvtfcHk0lhcsSJsFCxIYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWjsL%252FN&md5=c4a9d8dd32f6725c365203289cfdc018</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1042%2FBST20160328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20160328%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DLeen%26aufirst%3DE.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26atitle%3DA%2520Moving%2520Target%253A%2520Structure%2520and%2520Disorder%2520in%2520Pursuit%2520of%2520MYC%2520Inhibitors%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2017%26volume%3D45%26spage%3D709%26epage%3D717%26doi%3D10.1042%2FBST20160328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieb, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">440</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25818646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=440-452&author=L.+R.+Thomasauthor=Q.+Wangauthor=B.+C.+Griebauthor=J.+Phanauthor=A.+M.+Foshageauthor=Q.+Sunauthor=E.+T.+Olejniczakauthor=T.+Clarkauthor=S.+Deyauthor=S.+Loreyauthor=B.+Alicieauthor=G.+C.+Howardauthor=B.+Cawthonauthor=K.+C.+Essauthor=C.+M.+Eischenauthor=Z.+Zhaoauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Interaction+with+WDR5+Promotes+Target+Gene+Recognition+and+Tumorigenesis+by+MYC&doi=10.1016%2Fj.molcel.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span></div><div class="casAuthors">Thomas, Lance R.; Wang, Qingguo; Grieb, Brian C.; Phan, Jason; Foshage, Audra M.; Sun, Qi; Olejniczak, Edward T.; Clark, Travis; Dey, Soumyadeep; Lorey, Shelly; Alicie, Bethany; Howard, Gregory C.; Cawthon, Bryan; Ess, Kevin C.; Eischen, Christine M.; Zhao, Zhongming; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">440-452</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC is an oncoprotein transcription factor that is overexpressed in the majority of malignancies.  The oncogenic potential of MYC stems from its ability to bind regulatory sequences in thousands of target genes, which depends on interaction of MYC with its obligate partner, MAX.  Here, we show that broad assocn. of MYC with chromatin also depends on interaction with the WD40-repeat protein WDR5.  MYC binds WDR5 via an evolutionarily conserved "MYC box IIIb" motif that engages a shallow, hydrophobic cleft on the surface of WDR5.  Structure-guided mutations in MYC that disrupt interaction with WDR5 attenuate binding of MYC at ∼80% of its chromosomal locations and disable its ability to promote induced pluripotent stem cell formation and drive tumorigenesis.  Our data reveal WDR5 as a key determinant for MYC recruitment to chromatin and uncover a tractable target for the discovery of anticancer therapies against MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfifOnQkKDHrVg90H21EOLACvtfcHk0lhcsSJsFCxIYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D&md5=329876ab3db1a16fb58bd5de7f1de7b9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DGrieb%26aufirst%3DB.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DLorey%26aufirst%3DS.%26aulast%3DAlicie%26aufirst%3DB.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DCawthon%26aufirst%3DB.%26aulast%3DEss%26aufirst%3DK.%2BC.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DInteraction%2520with%2520WDR5%2520Promotes%2520Target%2520Gene%2520Recognition%2520and%2520Tumorigenesis%2520by%2520MYC%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D440%26epage%3D452%26doi%3D10.1016%2Fj.molcel.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">The MYC-WDR5 Nexus and Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">4012</span>– <span class="NLM_lpage">4015</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-1216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1158%2F0008-5472.CAN-15-1216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26383167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyrtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=4012-4015&author=L.+R.+Thomasauthor=A.+M.+Foshageauthor=A.+M.+Weissmillerauthor=W.+P.+Tansey&title=The+MYC-WDR5+Nexus+and+Cancer&doi=10.1158%2F0008-5472.CAN-15-1216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The MYC-WDR5 Nexus and Cancer</span></div><div class="casAuthors">Thomas, Lance R.; Foshage, Audra M.; Weissmiller, April M.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4012-4015</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The MYC oncogenes encode a family of transcription factors that feature prominently in cancer.  MYC proteins are overexpressed or deregulated in a majority of malignancies and drive tumorigenesis by inducing widespread transcriptional reprogramming that promotes cell proliferation, metab., and genomic instability.  The ability of MYC to regulate transcription depends on its dimerization with MAX, which creates a DNA-binding domain that recognizes specific sequences in the regulatory elements of MYC target genes.  Recently, we discovered that recognition of target genes by MYC also depends on its interaction with WDR5, a WD40-repeat protein that exists as part of several chromatin-regulatory complexes.  Here, we discuss how interaction of MYC with WDR5 could create an avidity-based chromatin recognition mechanism that allows MYC to select its target genes in response to both genetic and epigenetic determinants.  We rationalize how the MYC-WDR5 interaction provides plasticity in target gene selection by MYC and speculate on the biochem. and genomic contexts in which this interaction occurs.  Finally, we discuss how properties of the MYC-WDR5 interface make it an attractive point for discovery of small-mol. inhibitors of MYC function in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB2HGlzfYzWLVg90H21EOLACvtfcHk0lhi9bYry676MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyrtbvM&md5=7caa91f77ba27f4b8224f32d1c66ea9e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-1216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-1216%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DThe%2520MYC-WDR5%2520Nexus%2520and%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D4012%26epage%3D4015%26doi%3D10.1158%2F0008-5472.CAN-15-1216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaitatzes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neer, E. J.</span></span> <span> </span><span class="NLM_article-title">The WD Repeat: A Common Architecture for Diverse Functions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(99)01384-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0968-0004%2899%2901384-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10322433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK1MXkslSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1999&pages=181-185&author=T.+F.+Smithauthor=C.+Gaitatzesauthor=K.+Saxenaauthor=E.+J.+Neer&title=The+WD+Repeat%3A+A+Common+Architecture+for+Diverse+Functions&doi=10.1016%2FS0968-0004%2899%2901384-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The WD repeat: a common architecture for diverse functions</span></div><div class="casAuthors">Smith, Temple F.; Gaitatzes, Chrysanthe; Saxena, Kumkum; Neer, Eva J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">181-185</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 40 refs.  Our knowledge of the large family of proteins that contain the WD repeat continues to accumulate.  The WD-repeat proteins are found in all eukaryotes and are implicated in a wide variety of crucial functions.  The soln. of the three-dimensional structure of one WD-repeat protein and the assumption that the structure will be common to all members of this family has allowed subfamilies of WD-repeat proteins to be defined on the basis of probable surface similarity.  Proteins that have very similar surfaces are likely to have common binding partners and similar functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5gp-sO0-v-bVg90H21EOLACvtfcHk0lhi9bYry676MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkslSnsbc%253D&md5=c48cb0bacb96cc575b0629fbb7322d26</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2899%2901384-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252899%252901384-5%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DT.%2BF.%26aulast%3DGaitatzes%26aufirst%3DC.%26aulast%3DSaxena%26aufirst%3DK.%26aulast%3DNeer%26aufirst%3DE.%2BJ.%26atitle%3DThe%2520WD%2520Repeat%253A%2520A%2520Common%2520Architecture%2520for%2520Diverse%2520Functions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1999%26volume%3D24%26spage%3D181%26epage%3D185%26doi%3D10.1016%2FS0968-0004%2899%2901384-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of WD40 Domain Proteins</span>. <i>Protein Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1007/s13238-011-1018-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1007%2Fs13238-011-1018-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21468892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt12iur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=202-214&author=C.+Xuauthor=J.+Min&title=Structure+and+Function+of+WD40+Domain+Proteins&doi=10.1007%2Fs13238-011-1018-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of WD40 domain proteins</span></div><div class="casAuthors">Xu, Chao; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-214</span>CODEN:
                <span class="NLM_cas:coden">PCREFB</span>;
        ISSN:<span class="NLM_cas:issn">1674-800X</span>.
    
            (<span class="NLM_cas:orgname">Higher Education Press</span>)
        </div><div class="casAbstract">A review.  The WD40 domain exhibits a β-propeller architecture, often comprising seven blades.  The WD40 domain is one of the most abundant domains and also among the top interacting domains in eukaryotic genomes.  In this review, we will discuss the identification, definition and architecture of the WD40 domains.  WD40 domain proteins are involved in a large variety of cellular processes, in which WD40 domains function as a protein-protein or protein-DNA interaction platform.  WD40 domain mediates mol. recognition events mainly through the smaller top surface, but also through the bottom surface and sides.  So far, no WD40 domain has been found to display enzymic activity.  We will also discuss the different binding modes exhibited by the large versatile family of WD40 domain proteins.  In the last part of this review, we will discuss how post-translational modifications are recognized by WD40 domain proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPx0CEOy9VnbVg90H21EOLACvtfcHk0lhi9bYry676MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt12iur0%253D&md5=478e87d9f93f17afb65b710052174d37</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13238-011-1018-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13238-011-1018-1%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Function%2520of%2520WD40%2520Domain%2520Proteins%26jtitle%3DProtein%2520Cell%26date%3D2011%26volume%3D2%26spage%3D202%26epage%3D214%26doi%3D10.1007%2Fs13238-011-1018-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stirnimann, C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsalaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. W.</span></span> <span> </span><span class="NLM_article-title">WD40 Proteins Propel Cellular Networks</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.tibs.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20451393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=565-574&author=C.+U.+Stirnimannauthor=E.+Petsalakiauthor=R.+B.+Russellauthor=C.+W.+M%C3%BCller&title=WD40+Proteins+Propel+Cellular+Networks&doi=10.1016%2Fj.tibs.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">WD40 proteins propel cellular networks</span></div><div class="casAuthors">Stirnimann, Christian U.; Petsalaki, Evangelia; Russell, Robert B.; Mueller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">565-574</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent findings indicate that WD40 domains play central roles in biol. processes by acting as hubs in cellular networks; however, they have been studied less intensely than other common domains, such as the kinase, PDZ or SH3 domains.  As suggested by various interactome studies, they are among the most promiscuous interactors.  Structural studies suggest that this property stems from their ability, as scaffolds, to interact with diverse proteins, peptides or nucleic acids using multiple surfaces or modes of interaction.  A general scaffolding role is supported by the fact that no WD40 domain has been found with intrinsic enzymic activity despite often being part of large mol. machines.  We discuss the WD40 domain distributions in protein networks and structures of WD40-contg. assemblies to demonstrate their versatility in mediating crit. cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruw7yoJ7_J-LVg90H21EOLACvtfcHk0lgyX6aJNPOs8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7%252FO&md5=47c47c118b1c85f3d1726a873283ddb2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DStirnimann%26aufirst%3DC.%2BU.%26aulast%3DPetsalaki%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DR.%2BB.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BW.%26atitle%3DWD40%2520Proteins%2520Propel%2520Cellular%2520Networks%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D565%26epage%3D574%26doi%3D10.1016%2Fj.tibs.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Disease Association and Druggability of WD40 Repeat Proteins</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3766</span>– <span class="NLM_lpage">3773</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.7b00451</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.7b00451" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKht7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=3766-3773&author=R.+Songauthor=Z.-D.+Wangauthor=M.+Schapira&title=Disease+Association+and+Druggability+of+WD40+Repeat+Proteins&doi=10.1021%2Facs.jproteome.7b00451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Disease Association and Druggability of WD40 Repeat Proteins</span></div><div class="casAuthors">Song, Richard; Wang, Zhong-Duo; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3766-3773</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD40 repeat (WDR) domains are protein interaction scaffolds that represent one of the largest protein families in human, and a first WDR inhibitor-an allosteric antagonist of polycomb repressive complex 2-just entered the clinic.  A systematic anal. of the CORUM database of protein complexes shows that WDR is the most represented domain in transcriptional regulation and one of the most prevalent in the ubiquitin proteasome system, two pathways of high relevance to drug discovery.  Parsing the literature and the vulnerability of cancer cell lines to CRISPR knockout indicates that WDR proteins are targets of interest in oncol. and other disease areas.  A quant. anal. of WDR structures reveals that druggable binding pockets can be found on multiple surfaces of these multifaceted protein interaction platforms.  These data support the development of chem. probes to further interrogate WDR proteins as an emerging therapeutic target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-4QqKGfC1KrVg90H21EOLACvtfcHk0lgyX6aJNPOs8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKht7fM&md5=368abb89a0ecbe30c057fb2521b7ee8c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.7b00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.7b00451%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.-D.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDisease%2520Association%2520and%2520Druggability%2520of%2520WD40%2520Repeat%2520Proteins%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2017%26volume%3D16%26spage%3D3766%26epage%3D3773%26doi%3D10.1021%2Facs.jproteome.7b00451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guarnaccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W.</span></span> <span> </span><span class="NLM_article-title">Moonlighting with WDR5: A Cellular Multitasker</span>. <i>J. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.3390/jcm7020021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3390%2Fjcm7020021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=21&author=A.+Guarnacciaauthor=W.+Tansey&title=Moonlighting+with+WDR5%3A+A+Cellular+Multitasker&doi=10.3390%2Fjcm7020021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Moonlighting with WDR5: a cellular multitasker</span></div><div class="casAuthors">Guarnaccia, Alissa Dupuy; Tansey, William Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">21/1-21/17</span>CODEN:
                <span class="NLM_cas:coden">JCMOHK</span>;
        ISSN:<span class="NLM_cas:issn">2077-0383</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">WDR5 is a highly conserved WD40 repeat-contg. protein that is essential for proper regulation of multiple cellular processes.  WDR5 is best characterized as a core scaffolding component of histone methyltransferase complexes, but emerging evidence demonstrates that it does much more, ranging from expanded functions in the nucleus through to controlling the integrity of cell division.  The purpose of this review is to describe the current mol. understandings of WDR5, discuss how it participates in diverse cellular processes, and highlight drug discovery efforts around WDR5 that may form the basis of new anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-aPToDu3vgbVg90H21EOLACvtfcHk0lgyX6aJNPOs8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGjt7w%253D&md5=fba93df831fa1fd74d7f57d0205bd2d8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fjcm7020021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fjcm7020021%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnaccia%26aufirst%3DA.%26aulast%3DTansey%26aufirst%3DW.%26atitle%3DMoonlighting%2520with%2520WDR5%253A%2520A%2520Cellular%2520Multitasker%26jtitle%3DJ.%2520Clin.%2520Med.%26date%3D2018%26volume%3D7%26spage%3D21%26doi%3D10.3390%2Fjcm7020021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">A Conserved Arginine-Containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32162</span>– <span class="NLM_lpage">32175</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+Conserved+Arginine-Containing+Motif+Crucial+for+the+Assembly+and+Enzymatic+Activity+of+the+Mixed+Lineage+Leukemia+Protein-1+Core+Complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0lici-Vc1GmUew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520Conserved%2520Arginine-Containing%2520Motif%2520Crucial%2520for%2520the%2520Assembly%2520and%2520Enzymatic%2520Activity%2520of%2520the%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32158</span>– <span class="NLM_lpage">32161</span>, <span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+Bound+to+Mixed+Lineage+Leukemia+Protein-1+Peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lici-Vc1GmUew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520Bound%2520to%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">WDR5 Interacts with Mixed Lineage Leukemia (MLL) Protein via the Histone H3-Binding Pocket</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">35258</span>– <span class="NLM_lpage">35264</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806900200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M806900200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18840606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGqt73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=35258-35264&author=J.-J.+Songauthor=R.+E.+Kingston&title=WDR5+Interacts+with+Mixed+Lineage+Leukemia+%28MLL%29+Protein+via+the+Histone+H3-Binding+Pocket&doi=10.1074%2Fjbc.M806900200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Interacts with Mixed Lineage Leukemia (MLL) Protein via the Histone H3-binding Pocket</span></div><div class="casAuthors">Song, Ji-Joon; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">35258-35264</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">WDR5 is a component of the mixed lineage leukemia (MLL) complex, which methylates lysine 4 of histone H3, and was identified as a methylated Lys-4 histone H3-binding protein.  Here, we present a crystal structure of WDR5 bound to an MLL peptide.  Surprisingly, we find that WDR5 utilizes the same pocket shown to bind histone H3 for this MLL interaction.  Furthermore, the WDR5-MLL interaction is disrupted preferentially by mono- and di-methylated Lys-4 histone H3 over unmodified and tri-methylated Lys-4 histone H3.  These data implicate a delicate interplay between the effector, WDR5, the catalytic subunit, MLL, and the substrate, histone H3, of the MLL complex.  We suggest that the activity of the MLL complex might be regulated through this interplay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKATFDTs7dELVg90H21EOLACvtfcHk0lici-Vc1GmUew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGqt73J&md5=3207cca861ba0e6491a888b8586494ae</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806900200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806900200%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.-J.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DWDR5%2520Interacts%2520with%2520Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%2520Protein%2520via%2520the%2520Histone%2520H3-Binding%2520Pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D35258%26epage%3D35264%26doi%3D10.1074%2Fjbc.M806900200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odho, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Requirement of Additional Factors for MLL1 SET Domain Activity and Recognition of Epigenetic Marks</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2008.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=19187761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=181-191&author=S.+M.+Southallauthor=P.+S.+Wongauthor=Z.+Odhoauthor=S.+M.+Roeauthor=J.+R.+Wilson&title=Structural+Basis+for+the+Requirement+of+Additional+Factors+for+MLL1+SET+Domain+Activity+and+Recognition+of+Epigenetic+Marks&doi=10.1016%2Fj.molcel.2008.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the requirement of additional factors for MLL1 SET domain activity and recognition of epigenetic marks</span></div><div class="casAuthors">Southall, Stacey M.; Wong, Poon-Sheng; Odho, Zain; Roe, S. Mark; Wilson, Jon R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mixed-lineage leukemia protein MLL1 is a transcriptional regulator with an essential role in early development and hematopoiesis.  The biol. function of MLL1 is mediated by the histone H3K4 methyltransferase activity of the carboxyl-terminal SET domain.  We have detd. the crystal structure of the MLL1 SET domain in complex with cofactor product AdoHcy and a histone H3 peptide.  This structure indicates that, in order to form a well-ordered active site, a highly variable but essential component of the SET domain must be repositioned.  To test this idea, we compared the effect of the addn. of MLL complex members on methyltransferase activity and show that both RbBP5 and Ash2L but not Wdr5 stimulate activity.  Addnl., we have detd. the effect of posttranslational modifications on histone H3 residues downstream and upstream from the target lysine and provide a structural explanation for why H3T3 phosphorylation and H3K9 acetylation regulate activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TVBdtfM5BLVg90H21EOLACvtfcHk0lici-Vc1GmUew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVylsbg%253D&md5=b16a9354ef72ad0138c891c7fe698fcc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DWong%26aufirst%3DP.%2BS.%26aulast%3DOdho%26aufirst%3DZ.%26aulast%3DRoe%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Requirement%2520of%2520Additional%2520Factors%2520for%2520MLL1%2520SET%2520Domain%2520Activity%2520and%2520Recognition%2520of%2520Epigenetic%2520Marks%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D181%26epage%3D191%26doi%3D10.1016%2Fj.molcel.2008.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odho, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of a Novel WDR5-Binding Site That Recruits RbBP5 through a Conserved Motif to Enhance Methylation of Histone H3 Lysine 4 by Mixed Lineage Leukemia Protein-1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">32967</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.159921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M110.159921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20716525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=32967-32976&author=Z.+Odhoauthor=S.+M.+Southallauthor=J.+R.+Wilson&title=Characterization+of+a+Novel+WDR5-Binding+Site+That+Recruits+RbBP5+through+a+Conserved+Motif+to+Enhance+Methylation+of+Histone+H3+Lysine+4+by+Mixed+Lineage+Leukemia+Protein-1&doi=10.1074%2Fjbc.M110.159921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel WDR5-binding site that recruits RbBP5 through a conserved motif to enhance methylation of histone H3 lysine 4 by mixed lineage leukemia protein-1</span></div><div class="casAuthors">Odho, Zain; Southall, Stacey M.; Wilson, Jon R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">32967-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone modification is well established as a fundamental mechanism driving the regulation of transcription, replication, and DNA repair through the control of chromatin structure.  Likewise, it is apparent that incorrect targeting of histone modifications contributes to misregulated gene expression and hence to developmental disorders and diseases of genomic instability such as cancer.  The KMT2 family of SET domain methyltransferases, typified by mixed lineage leukemia protein-1 (MLL1), is responsible for histone H3 lysine 4 methylation, a marker of active genes.  To ensure that this modification is correctly targeted, a multiprotein complex assocs. with the methyltransferase and directs activity.  We have identified a novel interaction site on the core complex protein WD repeat protein-5 (WDR5), and we mapped the complementary site on its partner retinoblastoma-binding protein-5 (RbBP5).  We have characterized this interaction by x-ray crystallog. and show how it is fundamental to the assembly of the complex and to the regulation of methyltransferase activity.  We show which region of RbBP5 contributes directly to mixed lineage leukemia activation, and we combine our structural and biochem. data to produce a model to show how WDR5 and RbBP5 act cooperatively to stimulate activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSuXfVNduMSrVg90H21EOLACvtfcHk0lhBU39ANBScQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12nu73E&md5=fd5c6418120415b74f200381097e4c1e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.159921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.159921%26sid%3Dliteratum%253Aachs%26aulast%3DOdho%26aufirst%3DZ.%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26atitle%3DCharacterization%2520of%2520a%2520Novel%2520WDR5-Binding%2520Site%2520That%2520Recruits%2520RbBP5%2520through%2520a%2520Conserved%2520Motif%2520to%2520Enhance%2520Methylation%2520of%2520Histone%2520H3%2520Lysine%25204%2520by%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D32967%26epage%3D32976%26doi%3D10.1074%2Fjbc.M110.159921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dias, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brettschneider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadlec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, A.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of the KANSL1/WDR5/ KANSL2 Complex Reveals That WDR5 Is Required for Efficient Assembly and Chromatin Targeting of the NSL Complex</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">929</span>– <span class="NLM_lpage">942</span>, <span class="refDoi"> DOI: 10.1101/gad.240200.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1101%2Fgad.240200.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24788516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot12is7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=929-942&author=J.+Diasauthor=N.+Van+Nguyenauthor=P.+Georgievauthor=A.+Gaubauthor=J.+Brettschneiderauthor=S.+Cusackauthor=J.+Kadlecauthor=A.+Akhtar&title=Structural+Analysis+of+the+KANSL1%2FWDR5%2F+KANSL2+Complex+Reveals+That+WDR5+Is+Required+for+Efficient+Assembly+and+Chromatin+Targeting+of+the+NSL+Complex&doi=10.1101%2Fgad.240200.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of the KANSL1/WDR5/KANSL2 complex reveals that WDR5 is required for efficient assembly and chromatin targeting of the NSL complex</span></div><div class="casAuthors">Dias, Jorge; Nguyen, Nhuong Van; Georgiev, Plamen; Gaub, Aline; Brettschneider, Janine; Cusack, Stephen; Kadlec, Jan; Akhtar, Asifa</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">929-942</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The subunits of the nonspecific lethal (NSL) complex, which include the histone acetyltransferase MOF (males absent on the first), play important roles in various cellular functions, including transcription regulation and stem cell identity maintenance and reprogramming, and are frequently misregulated in disease.  Here, the authors provide the first biochem. and structural insights into the mol. architecture of this large multiprotein assembly.  The authors identified several direct interactions within the complex and show that KANSL1 acts as a scaffold protein interacting with four other subunits, including WDR5, which in turn binds KANSL2.  Structural anal. of the KANSL1/WDR5/KANSL2 subcomplex reveals how WDR5 is recruited into the NSL complex via conserved linear motifs of KANSL1 and KANSL2.  Using structure-based KANSL1 mutants in transgenic flies, the authors show that the KANSL1-WDR5 interaction is required for proper assembly, efficient recruitment of the NSL complex to target promoters, and fly viability.  The authors' data clearly show that the interactions of WDR5 with the MOF-contg. NSL complex and MLL/COMPASS histone methyltransferase complexes are mutually exclusive.  The authors propose that rather than being a shared subunit, WDR5 plays an important role in assembling distinct histone-modifying complexes with different epigenetic regulatory roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7W3dLwq_MvrVg90H21EOLACvtfcHk0lhBU39ANBScQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot12is7s%253D&md5=6719d3b97de11fd9fc5f147d2f2a6021</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2Fgad.240200.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.240200.114%26sid%3Dliteratum%253Aachs%26aulast%3DDias%26aufirst%3DJ.%26aulast%3DVan%2BNguyen%26aufirst%3DN.%26aulast%3DGeorgiev%26aufirst%3DP.%26aulast%3DGaub%26aufirst%3DA.%26aulast%3DBrettschneider%26aufirst%3DJ.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DKadlec%26aufirst%3DJ.%26aulast%3DAkhtar%26aufirst%3DA.%26atitle%3DStructural%2520Analysis%2520of%2520the%2520KANSL1%252FWDR5%252F%2520KANSL2%2520Complex%2520Reveals%2520That%2520WDR5%2520Is%2520Required%2520for%2520Efficient%2520Assembly%2520and%2520Chromatin%2520Targeting%2520of%2520the%2520NSL%2520Complex%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D929%26epage%3D942%26doi%3D10.1101%2Fgad.240200.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanouette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groulx, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span> <span> </span><span class="NLM_article-title">Structural and Biochemical Insights into MLL1 Core Complex Assembly</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/j.str.2010.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.str.2010.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21220120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFCktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=101-108&author=V.+Avdicauthor=P.+Zhangauthor=S.+Lanouetteauthor=A.+Groulxauthor=V.+Tremblayauthor=J.+Brunzelleauthor=J.+F.+Couture&title=Structural+and+Biochemical+Insights+into+MLL1+Core+Complex+Assembly&doi=10.1016%2Fj.str.2010.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical insights into MLL1 core complex assembly</span></div><div class="casAuthors">Avdic, Vanja; Zhang, Pamela; Lanouette, Sylvain; Groulx, Adam; Tremblay, Veronique; Brunzelle, Joseph; Couture, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone H3 Lys-4 methylation is predominantly catalyzed by a family of methyltransferases whose enzymic activity depends on their interaction with a three-subunit complex composed of WDR5, RbBP5, and Ash2L.  Here, we report that a segment of 50 residues of RbBP5 bridges the Ash2L C-terminal domain to WDR5.  The crystal structure of WDR5 in ternary complex with RbBP5 and MLL1 reveals that both proteins binds peptide-binding clefts located on opposite sides of WDR5's β-propeller domain.  RbBP5 engages in several hydrogen bonds and van der Waals contacts within a V-shaped cleft formed by the junction of two blades on WDR5.  Mutational analyses of both the WDR5 V-shaped cleft and RbBP5 residues reveal that the interactions between RbBP5 and WDR5 are important for the stimulation of MLL1 methyltransferase activity.  Overall, this study provides the structural basis underlying the formation of the WDR5-RbBP5 subcomplex and further highlight the crucial role of WDR5 in scaffolding the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWM9chG9JoC7Vg90H21EOLACvtfcHk0lhBU39ANBScQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFCktg%253D%253D&md5=e0539eaf4179e60d808b4cda8454f187</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2010.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2010.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DAvdic%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLanouette%26aufirst%3DS.%26aulast%3DGroulx%26aufirst%3DA.%26aulast%3DTremblay%26aufirst%3DV.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26atitle%3DStructural%2520and%2520Biochemical%2520Insights%2520into%2520MLL1%2520Core%2520Complex%2520Assembly%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D101%26epage%3D108%26doi%3D10.1016%2Fj.str.2010.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Activity Regulation of MLL Family Methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>530</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1038/nature16952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature16952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26886794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOqur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2016&pages=447-452&author=Y.+Liauthor=J.+Hanauthor=Y.+Zhangauthor=F.+Caoauthor=Z.+Liuauthor=S.+Liauthor=J.+Wuauthor=C.+Huauthor=Y.+Wangauthor=J.+Shuaiauthor=J.+Chenauthor=L.+Caoauthor=D.+Liauthor=P.+Shiauthor=C.+Tianauthor=J.+Zhangauthor=Y.+Douauthor=G.+Liauthor=Y.+Chenauthor=M.+Lei&title=Structural+Basis+for+Activity+Regulation+of+MLL+Family+Methyltransferases&doi=10.1038%2Fnature16952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for activity regulation of MLL family methyltransferases</span></div><div class="casAuthors">Li, Yanjing; Han, Jianming; Zhang, Yuebin; Cao, Fang; Liu, Zhijun; Li, Shuai; Wu, Jian; Hu, Chunyi; Wang, Yan; Shuai, Jin; Chen, Juan; Cao, Liaoran; Li, Dangsheng; Shi, Pan; Tian, Changlin; Zhang, Jian; Dou, Yali; Li, Guohui; Chen, Yong; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">7591</span>),
    <span class="NLM_cas:pages">447-452</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mixed lineage leukemia (MLL) family of proteins (including MLL1-MLL4, SET1A and SET1B) specifically methylate histone 3 Lys4, and have pivotal roles in the transcriptional regulation of genes involved in haematopoiesis and development.  The methyltransferase activity of MLL1, by itself severely compromised, is stimulated by the three conserved factors WDR5, RBBP5 and ASH2L, which are shared by all MLL family complexes.  However, the mol. mechanism of how these factors regulate the activity of MLL proteins still remains poorly understood.  Here we show that a minimized human RBBP5-ASH2L heterodimer is the structural unit that interacts with and activates all MLL family histone methyltransferases.  Our structural, biochem. and computational analyses reveal a two-step activation mechanism of MLL family proteins.  These findings provide unprecedented insights into the common theme and functional plasticity in complex assembly and activity regulation of MLL family methyltransferases, and also suggest a universal regulation mechanism for most histone methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwlhi89SDUJbVg90H21EOLACvtfcHk0liueBcu8fhI_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOqur4%253D&md5=77fb7f13196f990a0c8918a59652216b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature16952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16952%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShuai%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DTian%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Activity%2520Regulation%2520of%2520MLL%2520Family%2520Methyltransferases%26jtitle%3DNature%26date%3D2016%26volume%3D530%26spage%3D447%26epage%3D452%26doi%3D10.1038%2Fnature16952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">High-Affinity, Small-Molecule Peptidomimetic Inhibitors of MLL1/WDR5 Protein–Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOlsbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-Affinity%2C+Small-Molecule+Peptidomimetic+Inhibitors+of+MLL1%2FWDR5+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity, Small-Molecule Peptidomimetic Inhibitors of MLL1/WDR5 Protein-Protein Interaction</span></div><div class="casAuthors">Karatas, Hacer; Townsend, Elizabeth C.; Cao, Fang; Chen, Yong; Bernard, Denzil; Liu, Liu; Lei, Ming; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">669-682</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mixed lineage leukemia 1 (MLL1) is a histone H3 lysine 4 (H3K4) methyltransferase, and targeting the MLL1 enzymic activity has been proposed as a novel therapeutic strategy for the treatment of acute leukemia harboring MLL1 fusion proteins.  The MLL1/WDR5 protein-protein interaction is essential for MLL1 enzymic activity.  In the present study, we designed a large no. of peptidomimetics to target the MLL1/WDR5 interaction based upon -CO-ARA-NH-, the min. binding motif derived from MLL1.  Our study led to the design of high-affinity peptidomimetics, which bind to WDR5 with Ki < 1 nM and function as potent antagonists of MLL1 activity in a fully reconstituted in vitro H3K4 methyltransferase assay.  Detn. of co-crystal structures of two potent peptidomimetics in complex with WDR5 establishes their structural basis for high-affinity binding to WDR5.  Evaluation of one such peptidomimetic, MM-102, in bone marrow cells transduced with MLL1-AF9 fusion construct shows that the compd. effectively decreases the expression of HoxA9 and Meis-1, two crit. MLL1 target genes in MLL1 fusion protein mediated leukemogenesis.  MM-102 also specifically inhibits cell growth and induces apoptosis in leukemia cells harboring MLL1 fusion proteins.  Our study provides the first proof-of-concept for the design of small-mol. inhibitors of the WDR5/MLL1 protein-protein interaction as a novel therapeutic approach for acute leukemia harboring MLL1 fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpkwxpAigvkbVg90H21EOLACvtfcHk0liueBcu8fhI_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOlsbvO&md5=29f9e4a57635bbb71f6ea65f82a3405c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-Affinity%252C%2520Small-Molecule%2520Peptidomimetic%2520Inhibitors%2520of%2520MLL1%252FWDR5%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4818</span>– <span class="NLM_lpage">4839</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1CjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4818-4839&author=H.+Karatasauthor=Y.+Liauthor=L.+Liuauthor=J.+Jiauthor=S.+Leeauthor=Y.+Chenauthor=J.+Yangauthor=L.+Huangauthor=D.+Bernardauthor=J.+Xuauthor=E.+C.+Townsendauthor=F.+Caoauthor=X.+Ranauthor=X.+Liauthor=B.+Wenauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=Discovery+of+a+Highly+Potent%2C+Cell-Permeable+Macrocyclic+Peptidomimetic+%28MM-589%29+Targeting+the+WD+Repeat+Domain+5+Protein+%28WDR5%29%E2%80%93Mixed+Lineage+Leukemia+%28MLL%29+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Facs.jmedchem.6b01796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)-Mixed Lineage Leukemia (MLL) Protein-Protein Interaction</span></div><div class="casAuthors">Karatas, Hacer; Li, Yangbing; Liu, Liu; Ji, Jiao; Lee, Shirley; Chen, Yong; Yang, Jiuling; Huang, Liyue; Bernard, Denzil; Xu, Jing; Townsend, Elizabeth C.; Cao, Fang; Ran, Xu; Li, Xiaoqin; Wen, Bo; Sun, Duxin; Stuckey, Jeanne A.; Lei, Ming; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4818-4839</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the design, synthesis and evaluation of macrocyclic peptidomimetics that bind to WDR5 and block the WDR5-MLL protein-protein interaction.  Compd. (I) (R1 = Me; n = 1) (MM-589) binds to WDR5 with an IC50 value of 0.90 nM (Ki value <1 nM), and inhibits the MLL H3K4 methyltransferase (HMT) activity with an IC50 value of 12.7 nM.  Compd. I (R1 = Me; n = 1) potently and selectively inhibits cell growth in human leukemia cell lines harboring MLL translocations, and is >40 times better than the previously reported compd. MM-401.  Co-crystal structures of I (R1 = H, n = 5) and (R1 = Me; n = 1) complexed with WDR5 provide structural basis for their high affinity binding to WDR5.  Addnl., we have developed and optimized a new, AlphaLISA-based MLL HMT functional assay to facilitate the functional evaluation of these designed compds.  Compd. I (R1 = Me; n = 1) represents the most potent inhibitor of the WDR5-MLL interaction reported to date and further optimization of I (R1 = Me; n = 1) may yield a new therapy for acute leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrugN-nacR_FLVg90H21EOLACvtfcHk0liueBcu8fhI_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1CjsL8%253D&md5=598654db06fde01433ef5158aa94a946</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01796%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Highly%2520Potent%252C%2520Cell-Permeable%2520Macrocyclic%2520Peptidomimetic%2520%2528MM-589%2529%2520Targeting%2520the%2520WD%2520Repeat%2520Domain%25205%2520Protein%2520%2528WDR5%2529%25E2%2580%2593Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4818%26epage%3D4839%26doi%3D10.1021%2Facs.jmedchem.6b01796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouillette, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-261&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+Methyltransferase+Activity+in+Mixed-Lineage+Leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0liawMxtWq0FDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520Methyltransferase%2520Activity%2520in%2520Mixed-Lineage%2520Leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D261%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giambruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avellino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of the WDR5-MLL Interaction in C/EBPα N-Terminal Leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+Targeting+of+the+WDR5-MLL+Interaction+in+C%2FEBP%CE%B1+N-Terminal+Leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0liawMxtWq0FDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520Targeting%2520of%2520the%2520WDR5-MLL%2520Interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-Terminal%2520Leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velázquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolshan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-awar, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-2496&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Vel%C3%A1zquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-awar&title=Structure-Based+Optimization+of+a+Small+Molecule+Antagonist+of+the+Interaction+Between+WD+Repeat-Containing+Protein+5+%28WDR5%29+and+Mixed-Lineage+Leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0ljcdm4LkIVr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Vel%25C3%25A1zquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-awar%26aufirst%3DR.%26atitle%3DStructure-Based%2520Optimization%2520of%2520a%2520Small%2520Molecule%2520Antagonist%2520of%2520the%2520Interaction%2520Between%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529%2520and%2520Mixed-Lineage%2520Leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D2496%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Small Molecular Blockers of Mixed Lineage Leukemia 1 (MLL1)-WDR5 Interaction Inhibit MLL1 Complex H3K4 Methyltransferase Activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.-D.+Liauthor=W.-L.+Chenauthor=Z.-H.+Wangauthor=Y.-Y.+Xieauthor=X.-L.+Xuauthor=Z.-Y.+Jiangauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=X.-K.+Guo&title=High-Affinity+Small+Molecular+Blockers+of+Mixed+Lineage+Leukemia+1+%28MLL1%29-WDR5+Interaction+Inhibit+MLL1+Complex+H3K4+Methyltransferase+Activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0ljcdm4LkIVr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DZ.-H.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DGuo%26aufirst%3DX.-K.%26atitle%3DHigh-Affinity%2520Small%2520Molecular%2520Blockers%2520of%2520Mixed%2520Lineage%2520Leukemia%25201%2520%2528MLL1%2529-WDR5%2520Interaction%2520Inhibit%2520MLL1%2520Complex%2520H3K4%2520Methyltransferase%2520Activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent 2-Aryl-6,7-Dihydro-5H-Pyrrolo[1,2-a]Imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5623</span>– <span class="NLM_lpage">5642</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5623-5642&author=F.+Wangauthor=K.+O.+Jeonauthor=J.+M.+Salovichauthor=J.+D.+Macdonaldauthor=J.+Alvaradoauthor=R.+D.+Gogliottiauthor=J.+Phanauthor=E.+T.+Olejniczakauthor=Q.+Sunauthor=S.+Wangauthor=D.+Camperauthor=J.+P.+Yuhauthor=J.+G.+Shawauthor=J.+Saiauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+of+Potent+2-Aryl-6%2C7-Dihydro-5H-Pyrrolo%5B1%2C2-a%5DImidazoles+as+WDR5-WIN-Site+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.8b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Wang, Feng; Jeon, Kyu Ok; Salovich, James M.; Macdonald, Jonathan D.; Alvarado, Joseph; Gogliotti, Rocco D.; Phan, Jason; Olejniczak, Edward T.; Sun, Qi; Wang, Shidong; Camper, DeMarco; Yuh, Joannes P.; Shaw, J. Grace; Sai, Jiqing; Rossanese, Olivia W.; Tansey, William P.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5623-5642</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted pyrroloimidazoles such as I [R = 3,4-Cl2C6H3CH2, 3-MeO-4-MeC6H3, 2-Me-4-pyridinyl, 3,5-(MeO)2-4-MeC6H2] were prepd. as inhibitors of the chromatin-regulatory scaffold protein WDR5 WDR5-interaction (WIN) site as lead compds. for development of antitumor agents.  NMR-based screening of a large fragment library identified several chem. distinct hit series that bind to the WIN site within WDR5.  Using a structure-based design approach, pyrroloimidazoles were found with dissocn. consts. < 10 nM for WDR5 and micromolar cellular activity against an AML-leukemia cell line.  The structures of I (R = 3-MeO-4-MeC6H3) and of four other compds. bound to a fragment of WDR5 were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSZaiiuG6OrVg90H21EOLACvtfcHk0ljcdm4LkIVr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN&md5=2322c6458b2a060005bf5250a46f5cbd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJeon%26aufirst%3DK.%2BO.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Potent%25202-Aryl-6%252C7-Dihydro-5H-Pyrrolo%255B1%252C2-a%255DImidazoles%2520as%2520WDR5-WIN-Site%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5623%26epage%3D5642%26doi%3D10.1021%2Facs.jmedchem.8b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heibert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2916</span>– <span class="NLM_lpage">2928.e13</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2019.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.celrep.2019.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=30865883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=2916-2928.e13&author=E.+R.+Ahoauthor=J.+Wangauthor=R.+D.+Gogliottiauthor=G.+C.+Howardauthor=J.+Phanauthor=P.+Acharyaauthor=J.+D.+Macdonaldauthor=K.+Chengauthor=S.+L.+Loreyauthor=B.+Luauthor=S.+Wenzelauthor=A.+M.+Foshageauthor=J.+Alvaradoauthor=F.+Wangauthor=J.+G.+Shawauthor=B.+Zhaoauthor=A.+M.+Weissmillerauthor=L.+R.+Thomasauthor=C.+R.+Vakocauthor=M.+D.+Hallauthor=S.+W.+Heibertauthor=Q.+Liuauthor=S.+R.+Staufferauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Displacement+of+WDR5+from+Chromatin+by+a+WIN+Site+Inhibitor+with+Picomolar+Affinity&doi=10.1016%2Fj.celrep.2019.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span></div><div class="casAuthors">Aho, Erin R.; Wang, Jing; Gogliotti, Rocco D.; Howard, Gregory C.; Phan, Jason; Acharya, Pankaj; Macdonald, Jonathan D.; Cheng, Ken; Lorey, Shelly L.; Lu, Bin; Wenzel, Sabine; Foshage, Audra M.; Alvarado, Joseph; Wang, Feng; Shaw, J. Grace; Zhao, Bin; Weissmiller, April M.; Thomas, Lance R.; Vakoc, Christopher R.; Hall, Matthew D.; Hiebert, Scott W.; Liu, Qi; Stauffer, Shaun R.; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2916-2928.e13</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The chromatin-assocd. protein WDR5 is a promising target for pharmacol. inhibition in cancer.  Drug discovery efforts center on the blockade of the "WIN site" of WDR5, a well-defined pocket that is amenable to small mol. inhibition.  Various cancer contexts have been proposed to be targets for WIN site inhibitors, but a lack of understanding of WDR5 target genes and of the primary effects of WIN site inhibitors hampers their utility.  Here, by the discovery of potent WIN site inhibitors, we demonstrate that the WIN site links WDR5 to chromatin at a small cohort of loci, including a specific subset of ribosome protein genes.  WIN site inhibitors rapidly displace WDR5 from chromatin and decrease the expression of assocd. genes, causing translational inhibition, nucleolar stress, and p53 induction.  Our studies define a mode by which WDR5 engages chromatin and forecast that WIN site blockade could have utility against multiple cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprdgAaHB9WfLVg90H21EOLACvtfcHk0lgThyK4QIREtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D&md5=56e64b66f0d04ff5c36e104e54594bbd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2019.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2019.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DWenzel%26aufirst%3DS.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DHeibert%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DDisplacement%2520of%2520WDR5%2520from%2520Chromatin%2520by%2520a%2520WIN%2520Site%2520Inhibitor%2520with%2520Picomolar%2520Affinity%26jtitle%3DCell%2520Rep.%26date%3D2019%26volume%3D26%26spage%3D2916%26epage%3D2928.e13%26doi%3D10.1016%2Fj.celrep.2019.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">WD40 Repeat Domain Proteins: A Novel Target Class?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnrd.2017.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=29026209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cmt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=773-786&author=M.+Schapiraauthor=M.+Tyersauthor=M.+Torrentauthor=C.+H.+Arrowsmith&title=WD40+Repeat+Domain+Proteins%3A+A+Novel+Target+Class%3F&doi=10.1038%2Fnrd.2017.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">WD40 repeat domain proteins: a novel target class?</span></div><div class="casAuthors">Schapira, Matthieu; Tyers, Mike; Torrent, Maricel; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">773-786</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Antagonism of protein-protein interactions (PPIs) with small mols. is becoming more feasible as a therapeutic approach.  Successful PPI inhibitors tend to target proteins contg. deep peptide-binding grooves or pockets rather than the more common large, flat protein interaction surfaces.  Here, we review one of the most abundant PPI domains in the human proteome, the WD40 repeat (WDR) domain, which has a central peptide-binding pocket and is a member of the β-propeller domain-contg. protein family.  Recently, two WDR domain-contg. proteins, WDR5 and EED, as well as other β-propeller domains have been successfully targeted by potent, specific, cell-active, drug-like chem. probes.  Could WDR domains be a novel target class for drug discovery.  Although the research is at an early stage and therefore not clin. validated, cautious optimism is justified, as WDR domain-contg. proteins are involved in multiple disease-assocd. pathways.  The druggability and structural diversity of WDR domain binding pockets suggest that understanding how to target this prevalent domain class will open up areas of disease biol. that have so far resisted drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMccOym4Ml7Vg90H21EOLACvtfcHk0lgThyK4QIREtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cmt7rM&md5=6f99bbb97e411c9b405a981d7791a8be</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.179%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DWD40%2520Repeat%2520Domain%2520Proteins%253A%2520A%2520Novel%2520Target%2520Class%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D773%26epage%3D786%26doi%3D10.1038%2Fnrd.2017.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orlicky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neduva, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elowe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span> <span> </span><span class="NLM_article-title">An Allosteric Inhibitor of Substrate Recognition by the SCFCdc4 Ubiquitin Ligase</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nbt.1646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnbt.1646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20581844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=733-737&author=S.+Orlickyauthor=X.+Tangauthor=V.+Neduvaauthor=N.+Eloweauthor=E.+D.+Brownauthor=F.+Sicheriauthor=M.+Tyers&title=An+Allosteric+Inhibitor+of+Substrate+Recognition+by+the+SCFCdc4+Ubiquitin+Ligase&doi=10.1038%2Fnbt.1646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric inhibitor of substrate recognition by the SCFCdc4 ubiquitin ligase</span></div><div class="casAuthors">Orlicky, Stephen; Tang, Xiaojing; Neduva, Victor; Elowe, Nadine; Brown, Eric D.; Sicheri, Frank; Tyers, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specificity of SCF ubiquitin ligase-mediated protein degrdn. is detd. by F-box proteins.  We identified a biplanar dicarboxylic acid compd., called SCF-I2, as an inhibitor of substrate recognition by the yeast F-box protein Cdc4 using a fluorescence polarization screen to monitor the displacement of a fluorescein-labeled phosphodegron peptide.  SCF-I2 inhibits the binding and ubiquitination of full-length phosphorylated substrates by SCFCdc4.  A co-crystal structure reveals that SCF-I2 inserts itself between the β-strands of blades 5 and 6 of the WD40 propeller domain of Cdc4 at a site that is 25 Å away from the substrate binding site.  Long-range transmission of SCF-I2 interactions distorts the substrate binding pocket and impedes recognition of key determinants in the Cdc4 phosphodegron.  Mutation of the SCF-I2 binding site abrogates its inhibitory effect and explains specificity in the allosteric inhibition mechanism.  Mammalian WD40 domain proteins may exhibit similar allosteric responsiveness and hence represent an extensive class of druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptnWKaMi1kdLVg90H21EOLACvtfcHk0lgThyK4QIREtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D&md5=23197bd195811ad8503efb5d43457ccf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1646%26sid%3Dliteratum%253Aachs%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DNeduva%26aufirst%3DV.%26aulast%3DElowe%26aufirst%3DN.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTyers%26aufirst%3DM.%26atitle%3DAn%2520Allosteric%2520Inhibitor%2520of%2520Substrate%2520Recognition%2520by%2520the%2520SCFCdc4%2520Ubiquitin%2520Ligase%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnbt.1646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sackton, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sackton, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meaders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaff, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigoillot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span> <span> </span><span class="NLM_article-title">Synergistic Blockade of Mitotic Exit by Two Chemical Inhibitors of the APC/C</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/nature13660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature13660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25156254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCnu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=646-649&author=K.+L.+Sacktonauthor=N.+Dimovaauthor=X.+Zengauthor=W.+Tianauthor=M.+Zhangauthor=T.+B.+Sacktonauthor=J.+Meadersauthor=K.+L.+Pfaffauthor=F.+Sigoillotauthor=H.+Yuauthor=X.+Luoauthor=R.+W.+King&title=Synergistic+Blockade+of+Mitotic+Exit+by+Two+Chemical+Inhibitors+of+the+APC%2FC&doi=10.1038%2Fnature13660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic blockade of mitotic exit by two chemical inhibitors of the APC/C</span></div><div class="casAuthors">Sackton, Katharine L.; Dimova, Nevena; Zeng, Xing; Tian, Wei; Zhang, Mengmeng; Sackton, Timothy B.; Meaders, Johnathan; Pfaff, Kathleen L.; Sigoillot, Frederic; Yu, Hongtao; Luo, Xuelian; King, Randall W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">514</span>
        (<span class="NLM_cas:issue">7524</span>),
    <span class="NLM_cas:pages">646-649</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein machines are multi-subunit protein complexes that orchestrate highly regulated biochem. tasks.  An example is the anaphase-promoting complex/cyclosome (APC/C), a 13-subunit ubiquitin ligase that initiates the metaphase-anaphase transition and mitotic exit by targeting proteins such as securin and cyclin B1 for ubiquitin-dependent destruction by the proteasome.  Because blocking mitotic exit is an effective approach for inducing tumor cell death, the APC/C represents a potential novel target for cancer therapy.  APC/C activation in mitosis requires binding of Cdc20 (ref. 5), which forms a co-receptor with the APC/C to recognize substrates contg. a destruction box (D-box).  Here we demonstrate that we can synergistically inhibit APC/C-dependent proteolysis and mitotic exit by simultaneously disrupting two protein-protein interactions within the APC/C-Cdc20-substrate ternary complex.  We identify a small mol., called apcin (APC inhibitor), which binds to Cdc20 and competitively inhibits the ubiquitylation of D-box-contg. substrates.  Anal. of the crystal structure of the apcin-Cdc20 complex suggests that apcin occupies the D-box-binding pocket on the side face of the WD40-domain.  The ability of apcin to block mitotic exit is synergistically amplified by co-addn. of tosyl-L-arginine Me ester, a small mol. that blocks the APC/C-Cdc20 interaction.  This work suggests that simultaneous disruption of multiple, weak protein-protein interactions is an effective approach for inactivating a protein machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_wm8ws0xnLVg90H21EOLACvtfcHk0lideArHA4l15Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCnu7nL&md5=f11be7f3403b7fa4144d167af925a442</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature13660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13660%26sid%3Dliteratum%253Aachs%26aulast%3DSackton%26aufirst%3DK.%2BL.%26aulast%3DDimova%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DSackton%26aufirst%3DT.%2BB.%26aulast%3DMeaders%26aufirst%3DJ.%26aulast%3DPfaff%26aufirst%3DK.%2BL.%26aulast%3DSigoillot%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DKing%26aufirst%3DR.%2BW.%26atitle%3DSynergistic%2520Blockade%2520of%2520Mitotic%2520Exit%2520by%2520Two%2520Chemical%2520Inhibitors%2520of%2520the%2520APC%252FC%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3D646%26epage%3D649%26doi%3D10.1038%2Fnature13660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altomonte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanda, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic Chemistry and Biological Activity of Pentafluorosulphanyl (SF<sub>5</sub>) Organic Molecules</span>. <i>J. Fluor. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.jfluchem.2012.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.jfluchem.2012.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2012&pages=57-93&author=S.+Altomonteauthor=M.+Zanda&title=Synthetic+Chemistry+and+Biological+Activity+of+Pentafluorosulphanyl+%28SF5%29+Organic+Molecules&doi=10.1016%2Fj.jfluchem.2012.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic chemistry and biological activity of pentafluorosulphanyl (SF5) organic molecules</span></div><div class="casAuthors">Altomonte, Stefano; Zanda, Matteo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorine Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-93</span>CODEN:
                <span class="NLM_cas:coden">JFLCAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-1139</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pentafluorosulphanyl group (SF5) is a highly stable chem. function which has been attracting a great deal of interest owing to its peculiar chem., structural, physicochem. and biol. properties.  Progress in the area of SF5-compds. has been somewhat hindered by the lack of straightforward lab-scale synthetic methods for introducing the SF5-group into org. mols.  However, recent synthetic progress, the availability of some SF5-building blocks from com. suppliers and the discovery of interesting properties of SF5-substituted mols. in materials science, biol. and drug discovery are giving new momentum to research in this fascinating area of fluorine chem.  Synthesis, reactivity and biol. properties of SF5-substituted org. mols. are herein reviewed with an emphasis on the work published after year 2000.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3nL1S9vWsi7Vg90H21EOLACvtfcHk0lideArHA4l15Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktLzF&md5=cc717bde9781e01cf6d7e5d48e479154</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jfluchem.2012.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jfluchem.2012.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DAltomonte%26aufirst%3DS.%26aulast%3DZanda%26aufirst%3DM.%26atitle%3DSynthetic%2520Chemistry%2520and%2520Biological%2520Activity%2520of%2520Pentafluorosulphanyl%2520%2528SF5%2529%2520Organic%2520Molecules%26jtitle%3DJ.%2520Fluor.%2520Chem.%26date%3D2012%26volume%3D143%26spage%3D57%26epage%3D93%26doi%3D10.1016%2Fj.jfluchem.2012.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnes-Seeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsis, P.</span></span> <span> </span><span class="NLM_article-title">Metabolically Stable Tert-Butyl Replacement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/ml400045j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400045j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=514-516&author=D.+Barnes-Seemanauthor=M.+Jainauthor=L.+Bellauthor=S.+Ferreiraauthor=S.+Cohenauthor=X.-H.+Chenauthor=J.+Aminauthor=B.+Snodgrassauthor=P.+Hatsis&title=Metabolically+Stable+Tert-Butyl+Replacement&doi=10.1021%2Fml400045j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolically Stable tert-Butyl Replacement</span></div><div class="casAuthors">Barnes-Seeman, David; Jain, Monish; Bell, Leslie; Ferreira, Suzie; Cohen, Scott; Chen, Xiao-Hui; Amin, Jakal; Snodgrass, Brad; Hatsis, Panos</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">514-516</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Susceptibility to metab. is a common issue with the tert-Bu group on compds. of medicinal interest.  The authors demonstrate an approach of removing all the fully Sp3 C-Hs from a tert-Bu group: replacing some C-Hs with C-Fs and increasing the s-character of the remaining C-Hs.  This approach gave a trifluoromethylcyclopropyl group, which increased metabolic stability.  Trifluoromethylcyclopropyl-contg. analogs had consistently higher metabolic stability in vitro and in vivo compared to their tert-butyl-contg. counterparts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao2I5ukFsTrVg90H21EOLACvtfcHk0lgLiCeSYrQPZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D&md5=156572485ea3a48f5e1b6ea7409f1f61</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fml400045j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400045j%26sid%3Dliteratum%253Aachs%26aulast%3DBarnes-Seeman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DL.%26aulast%3DFerreira%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.-H.%26aulast%3DAmin%26aufirst%3DJ.%26aulast%3DSnodgrass%26aufirst%3DB.%26aulast%3DHatsis%26aufirst%3DP.%26atitle%3DMetabolically%2520Stable%2520Tert-Butyl%2520Replacement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D514%26epage%3D516%26doi%3D10.1021%2Fml400045j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haigh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, B.</span></span> <span> </span><span class="NLM_article-title">Molecular Shape and Medicinal Chemistry: A Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3862</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.1021/jm900818s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900818s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3862-3886&author=A.+Nichollsauthor=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=A.+C.+Goodauthor=G.+Warrenauthor=M.+Mathieuauthor=S.+W.+Muchmoreauthor=S.+P.+Brownauthor=J.+A.+Grantauthor=J.+A.+Haighauthor=N.+Nevinsauthor=A.+N.+Jainauthor=B.+Kelley&title=Molecular+Shape+and+Medicinal+Chemistry%3A+A+Perspective&doi=10.1021%2Fjm900818s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape and Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Nicholls, Anthony; McGaughey, Georgia B.; Sheridan, Robert P.; Good, Andrew C.; Warren, Gregory; Mathieu, Magali; Muchmore, Steven W.; Brown, Scott P.; Grant, J. Andrew; Haigh, James A.; Nevins, Neysa; Jain, Ajay N.; Kelley, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3862-3886</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review article with 111 refs. summarized perspectives of mol. shape and medicinal chem. in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqtbXOEoSmy7Vg90H21EOLACvtfcHk0lgLiCeSYrQPZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D&md5=85664344e13872527a3dfb2296d34864</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm900818s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900818s%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DWarren%26aufirst%3DG.%26aulast%3DMathieu%26aufirst%3DM.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DHaigh%26aufirst%3DJ.%2BA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DKelley%26aufirst%3DB.%26atitle%3DMolecular%2520Shape%2520and%2520Medicinal%2520Chemistry%253A%2520A%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3862%26epage%3D3886%26doi%3D10.1021%2Fjm900818s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routh, J. I.</span></span> <span> </span><span class="NLM_article-title">The Binding of Salicylic Acid and Acetylsalicylic Acid to Human Serum Albumin</span>. <i>Clin. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">98</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/S0009-9120(73)80018-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0009-9120%2873%2980018-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=4728967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADyaE3s3kvVSgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1973&pages=98-105&author=S.+E.+Kramerauthor=J.+I.+Routh&title=The+Binding+of+Salicylic+Acid+and+Acetylsalicylic+Acid+to+Human+Serum+Albumin&doi=10.1016%2FS0009-9120%2873%2980018-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The binding of salicylic acid and acetylsalicylic acid to human serum albumin</span></div><div class="casAuthors">Kramer E; Routh J I</div><div class="citationInfo"><span class="NLM_cas:title">Clinical biochemistry</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">98-105</span>
        ISSN:<span class="NLM_cas:issn">0009-9120</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBgqevziwH9AxyojvcjibhfW6udTcc2ebSx1Z231f5vrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3s3kvVSgtQ%253D%253D&md5=e894f2b559aea3efc490cd810af141ae</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0009-9120%2873%2980018-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-9120%252873%252980018-9%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DS.%2BE.%26aulast%3DRouth%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520Binding%2520of%2520Salicylic%2520Acid%2520and%2520Acetylsalicylic%2520Acid%2520to%2520Human%2520Serum%2520Albumin%26jtitle%3DClin.%2520Biochem.%26date%3D1973%26volume%3D6%26spage%3D98%26epage%3D105%26doi%3D10.1016%2FS0009-9120%2873%2980018-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span> <span> </span><span class="NLM_article-title">Interactive Association of Drugs Binding to Human Serum Albumin</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3580</span>– <span class="NLM_lpage">3595</span>, <span class="refDoi"> DOI: 10.3390/ijms15033580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3390%2Fijms15033580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24583848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWhtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3580-3595&author=F.+Yangauthor=Y.+Zhangauthor=H.+Liang&title=Interactive+Association+of+Drugs+Binding+to+Human+Serum+Albumin&doi=10.3390%2Fijms15033580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive association of drugs binding to human serum albumin</span></div><div class="casAuthors">Yang, Feng; Zhang, Yao; Liang, Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3580-3595</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Human serum albumin (HSA) is an abundant plasma protein, which attracts great interest in the pharmaceutical industry since it can bind a remarkable variety of drugs impacting their delivery and efficacy and ultimately altering the drug's pharmacokinetic and pharmacodynamic properties.  Addnl., HSA is widely used in clin. settings as a drug delivery system due to its potential for improving targeting while decreasing the side effects of drugs.  It is thus of great importance from the viewpoint of pharmaceutical sciences to clarify the structure, function, and properties of HSA-drug complexes.  This review will succinctly outline the properties of binding site of drugs in IIA subdomain within the structure of HSA.  We will also give an overview on the binding characterization of interactive assocn. of drugs to human serum albumin that may potentially lead to significant clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopBtedaqwZgLVg90H21EOLACvtfcHk0lgxanRLI2vEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWhtbnP&md5=c50411459367ad9576b28a2934da0d0f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fijms15033580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms15033580%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26atitle%3DInteractive%2520Association%2520of%2520Drugs%2520Binding%2520to%2520Human%2520Serum%2520Albumin%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D15%26spage%3D3580%26epage%3D3595%26doi%3D10.3390%2Fijms15033580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">24242</span>– <span class="NLM_lpage">24256</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.014498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M109.014498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=19556245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24242-24256&author=A.+Patelauthor=V.+Dharmarajanauthor=V.+E.+Voughtauthor=M.+S.+Cosgrove&title=On+the+Mechanism+of+Multiple+Lysine+Methylation+by+the+Human+Mixed+Lineage+Leukemia+Protein-1+%28MLL1%29+Core+Complex&doi=10.1074%2Fjbc.M109.014498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Vought, Valarie E.; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">24242-24256</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription in eukaryotic genomes depends on enzymes that regulate the degree of histone H3 lysine 4 (H3K4) methylation.  The mixed lineage leukemia protein-1 (MLL1) is a member of the SET1 family of H3K4 methyltransferases and is frequently rearranged in acute leukemias.  Despite sequence comparisons that predict that SET1 family enzymes should only monomethylate their substrates, mono-, di-, and trimethylation of H3K4 has been attributed to SET1 family complexes in vivo and in vitro.  To better understand this paradox, we have biochem. reconstituted and characterized a five-component 200-kDa MLL1 core complex contg. human MLL1, WDR5, RbBP5, Ash2L, and DPY-30.  We demonstrate that the isolated MLL1 SET domain is a slow monomethyltransferase and that tyrosine 3942 of MLL1 prevents di- and trimethylation of H3K4.  In contrast, a complex contg. the MLL1 SET domain, WDR5, RbBP5, Ash2L, and DPY-30, displays a marked ∼600-fold increase in enzymic activity but only to the di-Me form of H3K4.  Single turnover kinetic expts. reveal that the reaction leading to H3K4 dimethylation involves the transient accumulation of a monomethylated species, suggesting that the MLL1 core complex uses a non-processive mechanism to catalyze multiple lysine methylation.  We have also discovered that the non-SET domain components of the MLL1 core complex possess a previously unrecognized methyltransferase activity that catalyzes H3K4 dimethylation within the MLL1 core complex.  Our results suggest that the mechanism of multiple lysine methylation by the MLL1 core complex involves the sequential addn. of two Me groups at two distinct active sites within the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZXmcwVcpRrVg90H21EOLACvtfcHk0lgxanRLI2vEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF&md5=4289813be10179ac818b8d93fe6ebc6a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.014498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.014498%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DOn%2520the%2520Mechanism%2520of%2520Multiple%2520Lysine%2520Methylation%2520by%2520the%2520Human%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520%2528MLL1%2529%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24242%26epage%3D24256%26doi%3D10.1074%2Fjbc.M109.014498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lapointe, B. T.</span>; <span class="NLM_string-name">Fuller, P. H.</span>; <span class="NLM_string-name">Gunaydin, H.</span>; <span class="NLM_string-name">Liu, K.</span>; <span class="NLM_string-name">Sciammetta, N.</span>; <span class="NLM_string-name">Trotter, B. W.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Barr, K. J.</span>; <span class="NLM_string-name">MacLean, J. K. F.</span>; <span class="NLM_string-name">Molinari, D. F.</span>; <span class="NLM_string-name">Simov, V.</span>; <span class="NLM_string-name">Davis, C. E.</span></span> <span> </span><span class="NLM_article-title">Substituted Pyrazole Compounds as RORgammaT Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2016/130818</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+T.+Lapointe&author=P.+H.+Fuller&author=H.+Gunaydin&author=K.+Liu&author=N.+Sciammetta&author=B.+W.+Trotter&author=H.+Zhang&author=K.+J.+Barr&author=J.+K.+F.+MacLean&author=D.+F.+Molinari&author=V.+Simov&author=C.+E.+Davis&title=Substituted+Pyrazole+Compounds+as+RORgammaT+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLapointe%26aufirst%3DB.%2BT.%26atitle%3DSubstituted%2520Pyrazole%2520Compounds%2520as%2520RORgammaT%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Barr, K. J.</span>; <span class="NLM_string-name">Lapointe, B. T.</span>; <span class="NLM_string-name">Gunaydin, H.</span>; <span class="NLM_string-name">Liu, K.</span>; <span class="NLM_string-name">Trotter, B. W.</span>; <span class="NLM_string-name">Davis, C. E.</span>; <span class="NLM_string-name">Brodowski, M.</span>; <span class="NLM_string-name">Burns, D. J.</span>; <span class="NLM_string-name">Ashraf, S.</span>; <span class="NLM_string-name">Argentieri, S. R.</span></span> <span> </span><span class="NLM_article-title">Heteroaryl Substituted Benzoic Acids as RORgammaT Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2017/075185</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=H.+Zhang&author=K.+J.+Barr&author=B.+T.+Lapointe&author=H.+Gunaydin&author=K.+Liu&author=B.+W.+Trotter&author=C.+E.+Davis&author=M.+Brodowski&author=D.+J.+Burns&author=S.+Ashraf&author=S.+R.+Argentieri&title=Heteroaryl+Substituted+Benzoic+Acids+as+RORgammaT+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DHeteroaryl%2520Substituted%2520Benzoic%2520Acids%2520as%2520RORgammaT%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn-Robitaille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissarie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinier, A.</span></span> <span> </span><span class="NLM_article-title">Mild and Diazo-Free Synthesis of Trifluoromethyl-Cyclopropanes Using Sulfonium Ylides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2265</span>– <span class="NLM_lpage">2268</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b00557</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b00557" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFWku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=2265-2268&author=P.+Cyrauthor=J.+Flynn-Robitailleauthor=P.+Boissarieauthor=A.+Marinier&title=Mild+and+Diazo-Free+Synthesis+of+Trifluoromethyl-Cyclopropanes+Using+Sulfonium+Ylides&doi=10.1021%2Facs.orglett.9b00557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mild and Diazo-Free Synthesis of Trifluoromethyl-Cyclopropanes Using Sulfonium Ylides</span></div><div class="casAuthors">Cyr, Patrick; Flynn-Robitaille, Joel; Boissarie, Patrick; Marinier, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2265-2268</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of several 1,1-disubstituted trifluoromethyl-cyclopropanes (TFCPs), known as tert-Bu bioisosteres, was achieved from the reaction between trifluoromethylalkenes and unstabilized sulfonium ylides in yields of ≤97%.  This method offers practical access to this cyclopropyl moiety of pharmacol. interest, employing a com. available reagent at low temps.  The synthesis of cyclopropanes bearing other electron-withdrawing groups as well as trisubstituted TFCPs was also accomplished.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTjdgU11yVd7Vg90H21EOLACvtfcHk0lgxanRLI2vEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFWku70%253D&md5=ab2d8364af72c5a1ea65b22659d61007</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b00557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b00557%26sid%3Dliteratum%253Aachs%26aulast%3DCyr%26aufirst%3DP.%26aulast%3DFlynn-Robitaille%26aufirst%3DJ.%26aulast%3DBoissarie%26aufirst%3DP.%26aulast%3DMarinier%26aufirst%3DA.%26atitle%3DMild%2520and%2520Diazo-Free%2520Synthesis%2520of%2520Trifluoromethyl-Cyclopropanes%2520Using%2520Sulfonium%2520Ylides%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D2265%26epage%3D2268%26doi%3D10.1021%2Facs.orglett.9b00557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schanda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupče, E̅.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brutscher, B.</span></span> <span> </span><span class="NLM_article-title">SOFAST-HMQC Experiments for Recording Two-Dimensional Deteronuclear Correlation Spectra of Proteins within a Few Seconds</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1007/s10858-005-4425-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1007%2Fs10858-005-4425-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=16341750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=199-211&author=P.+Schandaauthor=E%CC%85.+Kup%C4%8Deauthor=B.+Brutscher&title=SOFAST-HMQC+Experiments+for+Recording+Two-Dimensional+Deteronuclear+Correlation+Spectra+of+Proteins+within+a+Few+Seconds&doi=10.1007%2Fs10858-005-4425-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">SOFAST-HMQC Experiments for Recording Two-dimensional Heteronuclear Correlation Spectra of Proteins within a Few Seconds</span></div><div class="casAuthors">Schanda, Paul; Kupce, Eriks; Brutscher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-211</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast multidimensional NMR with a time resoln. of a few seconds provides a new tool for high throughput screening and site-resolved real-time studies of kinetic mol. processes by NMR.  Recently we have demonstrated the feasibility to record protein 1H-15N correlation spectra in a few seconds of acquisition time using a new SOFAST-HMQC expt.  Here, we investigate in detail the performance of SOFAST-HMQC to record 1H-15N and 1H-13C correlation spectra of proteins of different size and at different magnetic field strengths.  Compared to std. 1H-15N correlation expts. SOFAST-HMQC provides a significant gain in sensitivity, esp. for fast repetition rates.  Guidelines are provided on how to set up SOFAST-HMQC expts. for a given protein sample.  In addn., an alternative pulse scheme, IPAP-SOFAST-HMQC is presented that allows application on NMR spectrometers equipped with cryogenic probes, and fast measurement of one-bond 1H-13C and 1H-15N scalar and residual dipolar coupling consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoE4Mp0wosbVg90H21EOLACvtfcHk0lgH9mvSvjpGhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbF&md5=6864fb3383e78dabae806e6869a470e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10858-005-4425-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-005-4425-x%26sid%3Dliteratum%253Aachs%26aulast%3DSchanda%26aufirst%3DP.%26aulast%3DKup%25C4%258De%26aufirst%3DE%25CC%2585.%26aulast%3DBrutscher%26aufirst%3DB.%26atitle%3DSOFAST-HMQC%2520Experiments%2520for%2520Recording%2520Two-Dimensional%2520Deteronuclear%2520Correlation%2520Spectra%2520of%2520Proteins%2520within%2520a%2520Few%2520Seconds%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2005%26volume%3D33%26spage%3D199%26epage%3D211%26doi%3D10.1007%2Fs10858-005-4425-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljfeoroUuo1xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+Mccoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljfeoroUuo1xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: Building New Software for Automated Crystallographic Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+Mccoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+New+Software+for+Automated+Crystallographic+Structure+Determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0ljfeoroUuo1xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520New%2520Software%2520for%2520Automated%2520Crystallographic%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljRqknOzyH2bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Schrodinger LLC</span> <i>The PyMOL Molecular Graphics System</i>, <i>Version
1.8.</i> <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schrodinger+LLC&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Weilin Chen, Xin Chen, Dongdong Li, Jianrui Zhou, Zhengyu Jiang, Qidong You, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoke Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of DDO-2213 as a Potent and Orally Bioavailable Inhibitor of the WDR5–Mixed Lineage Leukemia 1 Protein–Protein Interaction for the Treatment of MLL Fusion Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8221-8245. <a href="https://doi.org/10.1021/acs.jmedchem.1c00091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDDO-2213%252Bas%252Ba%252BPotent%252Band%252BOrally%252BBioavailable%252BInhibitor%252Bof%252Bthe%252BWDR5%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B1%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bfor%252Bthe%252BTreatment%252Bof%252BMLL%252BFusion%252BLeukemia%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8221%26epage%3D8245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anja Dölle, Bikash Adhikari, Andreas Krämer, Janik Weckesser, Nicola Berner, Lena-Marie Berger, Mathias Diebold, Magdalena M. Szewczyk, Dalia Barsyte-Lovejoy, Cheryl H. Arrowsmith, Jakob Gebel, Frank Löhr, Volker Dötsch, Martin Eilers, Stephanie Heinzlmeir, Bernhard Kuster, Christoph Sotriffer, Elmar Wolf, <span class="NLM_string-name hlFld-ContribAuthor">Stefan Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00146%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BEvaluation%252Bof%252BWD-Repeat-Containing%252BProtein%252B5%252B%252528WDR5%252529%252BDegraders%26aulast%3DD%25C3%25B6lle%26aufirst%3DAnja%26date%3D2021%26date%3D2021%26date%3D25012021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Selena Chacón Simon, Feng Wang, Lance R. Thomas, Jason Phan, Bin Zhao, Edward T. Olejniczak, Jonathan D. Macdonald, J. Grace Shaw, Caden Schlund, William Payne, Joy Creighton, Shaun R. Stauffer, Alex G. Waterson, William P. Tansey, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of WD Repeat-Containing Protein 5 (WDR5)–MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 4315-4333. <a href="https://doi.org/10.1021/acs.jmedchem.0c00224" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00224%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BWD%252BRepeat-Containing%252BProtein%252B5%252B%252528WDR5%252529%2525E2%252580%252593MYC%252BInhibitors%252BUsing%252BFragment-Based%252BMethods%252Band%252BStructure-Based%252BDesign%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DSelena%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06022020%26date%3D09042020%26date%3D30032020%26volume%3D63%26issue%3D8%26spage%3D4315%26epage%3D4333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Corey  Lourenco</span>, <span class="hlFld-ContribAuthor ">Diana  Resetca</span>, <span class="hlFld-ContribAuthor ">Cornelia  Redel</span>, <span class="hlFld-ContribAuthor ">Peter  Lin</span>, <span class="hlFld-ContribAuthor ">Alannah S.  MacDonald</span>, <span class="hlFld-ContribAuthor ">Roberto  Ciaccio</span>, <span class="hlFld-ContribAuthor ">Tristan M. G.  Kenney</span>, <span class="hlFld-ContribAuthor ">Yong  Wei</span>, <span class="hlFld-ContribAuthor ">David W.  Andrews</span>, <span class="hlFld-ContribAuthor ">Maria  Sunnerhagen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Brian  Raught</span>, <span class="hlFld-ContribAuthor ">Linda Z.  Penn</span>. </span><span class="cited-content_cbyCitation_article-title">MYC protein interactors in gene transcription and cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.1038/s41568-021-00367-9" title="DOI URL">https://doi.org/10.1038/s41568-021-00367-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00367-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00367-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DMYC%252Bprotein%252Binteractors%252Bin%252Bgene%252Btranscription%252Band%252Bcancer%26aulast%3DLourenco%26aufirst%3DCorey%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiawen  Cao</span>, <span class="hlFld-ContribAuthor ">Tiantian  Fan</span>, <span class="hlFld-ContribAuthor ">Yanlian  Li</span>, <span class="hlFld-ContribAuthor ">Zhiyan  Du</span>, <span class="hlFld-ContribAuthor ">Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Jingkang  Shen</span>, <span class="hlFld-ContribAuthor ">Xun  Huang</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Danyan  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Phage-Display Based Discovery and Characterization of Peptide Ligands against WDR5. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (5)
                                     , 1225. <a href="https://doi.org/10.3390/molecules26051225" title="DOI URL">https://doi.org/10.3390/molecules26051225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26051225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26051225%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPhage-Display%252BBased%252BDiscovery%252Band%252BCharacterization%252Bof%252BPeptide%252BLigands%252Bagainst%252BWDR5%26aulast%3DCao%26aufirst%3DJiawen%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D5%26spage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lance R.  Thomas</span>, <span class="hlFld-ContribAuthor ">Clare M.  Adams</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Fesik</span>, <span class="hlFld-ContribAuthor ">Christine M.  Eischen</span>, <span class="hlFld-ContribAuthor ">William P.  Tansey</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting MYC through WDR5. </span><span class="cited-content_cbyCitation_journal-name">Molecular & Cellular Oncology</span><span> <strong>2020,</strong> <em>7 </em>
                                    (2)
                                     , 1709388. <a href="https://doi.org/10.1080/23723556.2019.1709388" title="DOI URL">https://doi.org/10.1080/23723556.2019.1709388</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/23723556.2019.1709388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F23723556.2019.1709388%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520%2526%2520Cellular%2520Oncology%26atitle%3DTargeting%252BMYC%252Bthrough%252BWDR5%26aulast%3DThomas%26aufirst%3DLance%2BR.%26date%3D2020%26date%3D2020%26volume%3D7%26issue%3D2%26spage%3D1709388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0020.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overlay of WDR5 bound to MYC and MLL peptides (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R">4Y7R</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6">3EG6</a>, respectively). (A) Visualization of the MYC MbIIIb peptide bound to a cleft of WDR5, referred to as the “WBM” binding site. (B) MLL1 peptide binds on the reverse face of WDR5 with an arginine residue deep into the “WIN” site binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0019.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example <sup>1</sup>H–<sup>15</sup>N SOFAST-HMQC spectra showing the observable effect of compounds that (A) induce peak shift and (B) induce peak broadening. Apo-WDR5 (blue); Hit-WDR5 (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative hits that bind to WDR5 at the WBM site. <i>K</i><sub>d</sub> values obtained by a fluorescence polarization assay utilizing an FITC-labeled peptide based upon the MbIIIb sequence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0018.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overlay of hit compound <b>1</b> with the structure of bound MYC-derived MbIIIb peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R">4Y7R</a>), demonstrating the frameshift of the loop enabling retainment of the H-bond interaction with the backbone; WDR5–<b>1</b> complex (green) and WDR5–peptide complex (gray). (B) Zoom of the binding site of the cocrystal structure of WDR5 and <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6U6W">6U6W</a>). Shown as dashes are the halogen bonding interaction between the bromine atom and Trp273 and the hydrogen bond between the sulfonamide oxygen and Asn225; key amino acid residues are shown as sticks and are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0017.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-Ray cocrystallography of WDR5 bound to WBM site inhibitors. (A) Overlay of the X-Ray cocrystal structure of <b>5g</b> (green; PDB: 6U80) and <b>1</b> (gray). There are few major changes in the protein orientation, but <b>5g</b> showed diminished binding affinity compared to <b>1</b>, suggesting that the upward frameshift affords an unfavorable position of the small-molecule ligand. (B) Structure of <b>6b</b> bound to WDR5 (PDB: 6U8B). Key residues are shown as sticks, highlighting hydrogen and halogen bonding interactions between the ligand and protein. (C) Structure of <b>5n</b> bound to WDR5, demonstrating the orientation of the nitrile toward the protein (PDB: 6U8O). (D) Structure of <b>6g</b> bound to WDR5 (PDB: 6U8L). (E) Structure of <b>7k</b> bound to WDR5 (PDB: 6U5Y). (F) Structure of <b>7u</b> bound to WDR5 (PDB: 6U5M).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selected compounds featuring removal or replacement of the sulfonyl ring phenol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0016.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of compounds by thermal shift, all compounds measured at 10 μM in triplicate, and average values are displayed; RFU = relative fluorescence unit. (A) Comparison of compounds in the salicylic acid subseries. (B) Comparison of compounds that all contain the spirocyclobutane substructure. (C) Ten-point dilution of compound <b>7k</b> demonstrates the dose-dependent effect on WDR5 stabilization. (D) Measured Δ<i>T</i><sub>m</sub> values for compounds displayed in panels A–C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0015.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound target engagement in cell lysates. Levels of c-MYC and RBBP5 proteins in association with WDR5 were assessed after treatment with compound in lysates from WDR5-FLAG and c-MYC-HA co-overexpressing HEK293 cells. A fraction of the lysate before addition of compound is shown as a control lane. An α-FLAG immunoprecipitation was used to isolate WDR5, which is not detectable in the input due to the excess of WDR5 in the immunoprecipitation (IP) fraction. c-MYC and RBBP5 were blotted in independent experiments. Data are representative of minimum two independent biological repeats. α-FLAG immunoprecipitation from lysates treated with 50 μM compound from (A) acid subseries and (B) amide subseries. (C) Dose–response treatment of compounds <b>6r</b> and <b>7k</b> from 6.25 to 50 μM, as compared to 50 μM treatment with respective inactive controls <b>6-NC</b> and <b>7-NC</b> and the vehicle control (VC). Control cells express c-MYC-HA and a GFP-FLAG construct. (D–F) Quantification of two independent experiments, normalized to vehicle control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0014.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Sulfonamides and the Requisite Aniline Building Blocks to Compounds <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) pyridine, DCM, r.t.; (b) HSO<sub>3</sub>Cl, 0 °C to r.t.; (c) HNO<sub>3</sub>·H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C to r.t.; (d) H<sub>2</sub>, Pd/C, MeOH, r.t.; (e) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (f) benzyl bromide, K<sub>2</sub>CO<sub>3</sub>, MeCN, Δ; (g) cyclopropylboronic acid, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>·HBF<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene, 80 °C; (h) 3,3,3-trifluoroprop-1-ene, Pd(dppf)Cl<sub>2</sub>·DCM, K<sub>2</sub>CO<sub>3</sub>, dioxane, water, 70 °C; (i) Ph<sub>2</sub>MeSBF<sub>4</sub>, LiHMDS, THF, −78 °C.</p></p></figure><figure data-id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0013.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Cyano-Spirocycle Products Utilizing Common Bromide Intermediate, <b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dialkylbromide, NaH, DMSO, r.t.; (b) <i>tert</i>-butyl carbamate, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C; (c) BBr<sub>3</sub>, DCM, −78 °C; (d) TFA, DCM, r.t.; (e) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (f) phenyl formate, phenol, Pd(OAc)<sub>2</sub>, P(<i><sup>t</sup></i>Bu)<sub>3</sub>·HBF<sub>4</sub>, NEt<sub>3</sub>, toluene; (g) HNO<sub>3</sub>, TBAB, DCE/H<sub>2</sub>O, 0–23 °C; (h) H<sub>2</sub>, Pd/C, MeOH, r.t.; (i) 2 M LiOH, THF, r.t.; (j) amine, DIPEA, THF, Δ; (k) NaSMe, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, NEt<sub>3</sub>, THF, Δ; (l) oxone, EtOH, H<sub>2</sub>O, r.t.; (m) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C.</p></p></figure><figure data-id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Pentafluorosulfonyl Products from Commercial Bromide Intermediate, <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenyl formate, phenol, Pd(OAc)<sub>2</sub>, P(<i><sup>t</sup></i>Bu<sub>3</sub>)·HBF<sub>4</sub>, Et<sub>3</sub>N, MeCN, 90 °C; (b) BBr<sub>3</sub>, DCM, −78 °C; (c) LiOH, THF, 65 °C; (d) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C; (e) H<sub>2</sub>SO<sub>4</sub>, MeOH, 65 °C; (f) H<sub>2</sub>, Pd/C, MeOH, r.t.; (g) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (h) amine, THF, 65 °C; (i) Br<sub>2</sub>, FeCl<sub>3</sub>, AcOH, 0 °C; (j) S<sub>2</sub>(CH<sub>3</sub>)<sub>2</sub>, Cu, pyridine, 115 °C; (k) oxone, EtOH, H<sub>2</sub>O, r.t.</p></p></figure><figure data-id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0011.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>6g</b>, <b>6h</b>, <b>7c</b>, and <b>7d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TMS diazomethane, THF, MeOH, 0 °C; (b) HBF<sub>4</sub>·Et<sub>2</sub>O, NBS, MeCN, −10 °C; (c) HNO<sub>3</sub>, H<sub>2</sub>SO<sub>4</sub>, DCM, 0 °C; (d) BnBr, K<sub>2</sub>CO<sub>3</sub>, MeCN, Δ; (e) cyclopropylboronic acid, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>·HBF<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene, 80 °C; (f) H<sub>2</sub>, Pd/C, MeOH, r.t.; (g) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (h) LiOH, THF, 65 °C; (i). H<sub>2</sub>NMe, THF, 65 °C.</p></p></figure><figure data-id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/medium/jm9b01411_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0010.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. (A–C) Synthesis of Sulfone Analogues <b>7p</b>, <b>7q</b>, <b>7t</b>, <b>7u</b>, and <b>7x</b> and (D) Spirocyclobutane Analogues <b>6p</b>, <b>7h</b>, and <b>7i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01411/20191219/images/large/jm9b01411_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01411&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeSO<sub>2</sub>Na, CuI, <span class="smallcaps smallerCapital">l</span>-proline, NaOH, DMSO, 80 °C; (b) ArSO<sub>2</sub>Cl, pyridine, DCM, r.t.; (c) 2 M LiOH, THF, 65 °C; (d) DPPA, <i>t</i>-BuOH, DME, 100 °C; (e) TFA, DCM, r.t.; (f) Boc<sub>2</sub>O, MeOH, r.t.; (g) NBS, AIBN, CCl<sub>4</sub>, 80 °C; (h) TMSCN, K<sub>2</sub>CO<sub>3</sub>, MeCN, 0–50 °C; (i) 1,3-dibromopropane, NaH, DMSO, r.t.; (j) HCl, dioxane, 50 °C; (k) 1,3-diiodopropane, KO<i><sup>t</sup></i>Bu, THF, r.t.; (l) <i>tert</i>-butyl carbamate, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C; (m) H<sub>2</sub>NMe, THF, 65 °C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i263">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57278" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57278" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groudine, M.</span></span> <span> </span><span class="NLM_article-title">Control of C-MYC Regulation in Normal and Neoplastic Cells</span>. <i>Adv. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/S0065-230X(08)60476-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0065-230X%2808%2960476-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=2028839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK3MXltlarsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=1-48&author=C.+A.+Spencerauthor=M.+Groudine&title=Control+of+C-MYC+Regulation+in+Normal+and+Neoplastic+Cells&doi=10.1016%2FS0065-230X%2808%2960476-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Control of c-myc regulation in normal and neoplastic cells</span></div><div class="casAuthors">Spencer, Charlotte A.; Groudine, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-48</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    </div><div class="casAbstract">A review with ∼200 refs. of normal c-myc gene regulation, and abnormal c-myc gene regulation in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX_UXmyVSX1LVg90H21EOLACvtfcHk0liAISMg_ikbxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltlarsLc%253D&md5=34146e22b8744855132783e1f96d3d3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0065-230X%2808%2960476-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-230X%252808%252960476-5%26sid%3Dliteratum%253Aachs%26aulast%3DSpencer%26aufirst%3DC.%2BA.%26aulast%3DGroudine%26aufirst%3DM.%26atitle%3DControl%2520of%2520C-MYC%2520Regulation%2520in%2520Normal%2520and%2520Neoplastic%2520Cells%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D1991%26volume%3D56%26spage%3D1%26epage%3D48%26doi%3D10.1016%2FS0065-230X%2808%2960476-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mermel, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychaudhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urashima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mc Henry, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinchback, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haery, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yecies, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demichelis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rustgi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonescu, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">True, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">The Landscape of Somatic Copy-Number Alteration across Human Cancers</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1038/nature08822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature08822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20164920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFemsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=899-905&author=R.+Beroukhimauthor=C.+H.+Mermelauthor=D.+Porterauthor=G.+Weiauthor=S.+Raychaudhuriauthor=J.+Donovanauthor=J.+Barretinaauthor=J.+S.+Boehmauthor=J.+Dobsonauthor=M.+Urashimaauthor=K.+T.+Mc+Henryauthor=R.+M.+Pinchbackauthor=A.+H.+Ligonauthor=Y.-J.+Choauthor=L.+Haeryauthor=H.+Greulichauthor=M.+Reichauthor=W.+Wincklerauthor=M.+S.+Lawrenceauthor=B.+A.+Weirauthor=K.+E.+Tanakaauthor=D.+Y.+Chiangauthor=A.+J.+Bassauthor=A.+Looauthor=C.+Hoffmanauthor=J.+Prensnerauthor=T.+Liefeldauthor=Q.+Gaoauthor=D.+Yeciesauthor=S.+Signorettiauthor=E.+Maherauthor=F.+J.+Kayeauthor=H.+Sasakiauthor=J.+E.+Tepperauthor=J.+A.+Fletcherauthor=J.+Taberneroauthor=J.+Baselgaauthor=M.-S.+Tsaoauthor=F.+Demichelisauthor=M.+A.+Rubinauthor=P.+A.+Janneauthor=M.+J.+Dalyauthor=C.+Nuceraauthor=R.+L.+Levineauthor=B.+L.+Ebertauthor=S.+Gabrielauthor=A.+K.+Rustgiauthor=C.+R.+Antonescuauthor=M.+Ladanyiauthor=A.+Letaiauthor=L.+A.+Garrawayauthor=M.+Lodaauthor=D.+G.+Beerauthor=L.+D.+Trueauthor=A.+Okamotoauthor=S.+L.+Pomeroyauthor=S.+Singerauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=G.+Getzauthor=W.+R.+Sellersauthor=M.+Meyerson&title=The+Landscape+of+Somatic+Copy-Number+Alteration+across+Human+Cancers&doi=10.1038%2Fnature08822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The landscape of somatic copy-number alteration across human cancers</span></div><div class="casAuthors">Beroukhim, Rameen; Mermel, Craig H.; Porter, Dale; Wei, Guo; Raychaudhuri, Soumya; Donovan, Jerry; Barretina, Jordi; Boehm, Jesse S.; Dobson, Jennifer; Urashima, Mitsuyoshi; Mc Henry, Kevin T.; Pinchback, Reid M.; Ligon, Azra H.; Cho, Yoon-Jae; Haery, Leila; Greulich, Heidi; Reich, Michael; Winckler, Wendy; Lawrence, Michael S.; Weir, Barbara A.; Tanaka, Kumiko E.; Chiang, Derek Y.; Bass, Adam J.; Loo, Alice; Hoffman, Carter; Prensner, John; Liefeld, Ted; Gao, Qing; Yecies, Derek; Signoretti, Sabina; Maher, Elizabeth; Kaye, Frederic J.; Sasaki, Hidefumi; Tepper, Joel E.; Fletcher, Jonathan A.; Tabernero, Josep; Baselga, Jose; Tsao, Ming-Sound; Demichelis, Francesca; Rubin, Mark A.; Janne, Pasi A.; Daly, Mark J.; Nucera, Carmelo; Levine, Ross L.; Ebert, Benjamin L.; Gabriel, Stacey; Rustgi, Anil K.; Antonescu, Cristina R.; Ladanyi, Marc; Letai, Anthony; Garraway, Levi A.; Loda, Massimo; Beer, David G.; True, Lawrence D.; Okamoto, Aikou; Pomeroy, Scott L.; Singer, Samuel; Golub, Todd R.; Lander, Eric S.; Getz, Gad; Sellers, William R.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7283</span>),
    <span class="NLM_cas:pages">899-905</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A powerful way to discover key genes with causal roles in oncogenesis is to identify genomic regions that undergo frequent alteration in human cancers.  Here we present high-resoln. analyses of somatic copy-no. alterations (SCNAs) from 3131 cancer specimens, belonging largely to 26 histol. types.  We identify 158 regions of focal SCNA that are altered at significant frequency across several cancer types, of which 122 cannot be explained by the presence of a known cancer target gene located within these regions.  Several gene families are enriched among these regions of focal SCNA, including the BCL2 family of apoptosis regulators and the NF-κB pathway.  We show that cancer cells contg. amplifications surrounding the MCL1 and BCL2L1 anti-apoptotic genes depend on the expression of these genes for survival.  Finally, we demonstrate that a large majority of SCNAs identified in individual cancer types are present in several cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCxYRlqR0wXLVg90H21EOLACvtfcHk0ljg21IRwtWcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFemsr0%253D&md5=be6d81f5449cba96dfb4659106f706b0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature08822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08822%26sid%3Dliteratum%253Aachs%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DMermel%26aufirst%3DC.%2BH.%26aulast%3DPorter%26aufirst%3DD.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DRaychaudhuri%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DDobson%26aufirst%3DJ.%26aulast%3DUrashima%26aufirst%3DM.%26aulast%3DMc%2BHenry%26aufirst%3DK.%2BT.%26aulast%3DPinchback%26aufirst%3DR.%2BM.%26aulast%3DLigon%26aufirst%3DA.%2BH.%26aulast%3DCho%26aufirst%3DY.-J.%26aulast%3DHaery%26aufirst%3DL.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DTanaka%26aufirst%3DK.%2BE.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26aulast%3DBass%26aufirst%3DA.%2BJ.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DHoffman%26aufirst%3DC.%26aulast%3DPrensner%26aufirst%3DJ.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DYecies%26aufirst%3DD.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DMaher%26aufirst%3DE.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DTepper%26aufirst%3DJ.%2BE.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DTsao%26aufirst%3DM.-S.%26aulast%3DDemichelis%26aufirst%3DF.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DDaly%26aufirst%3DM.%2BJ.%26aulast%3DNucera%26aufirst%3DC.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DRustgi%26aufirst%3DA.%2BK.%26aulast%3DAntonescu%26aufirst%3DC.%2BR.%26aulast%3DLadanyi%26aufirst%3DM.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DLoda%26aufirst%3DM.%26aulast%3DBeer%26aufirst%3DD.%2BG.%26aulast%3DTrue%26aufirst%3DL.%2BD.%26aulast%3DOkamoto%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DSinger%26aufirst%3DS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DThe%2520Landscape%2520of%2520Somatic%2520Copy-Number%2520Alteration%2520across%2520Human%2520Cancers%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D899%26epage%3D905%26doi%3D10.1038%2Fnature08822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Mammalian MYC Proteins and Cancer</span>. <i>New J. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1155/2014/757534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1155%2F2014%2F757534" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=1-27&author=W.+P.+Tansey&title=Mammalian+MYC+Proteins+and+Cancer&doi=10.1155%2F2014%2F757534"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1155%2F2014%2F757534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F757534%26sid%3Dliteratum%253Aachs%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DMammalian%2520MYC%2520Proteins%2520and%2520Cancer%26jtitle%3DNew%2520J.%2520Sci.%26date%3D2014%26volume%3D2014%26spage%3D1%26epage%3D27%26doi%3D10.1155%2F2014%2F757534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the “undruggable” Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%9Cundruggable%E2%80%9D+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0lhJv_6ERMQ9LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%259Cundruggable%25E2%2580%259D%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Strategies to Inhibit MYC</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">a014266</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a014266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1101%2Fcshperspect.a014266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25274755" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=a014266&author=M.+R.+McKeownauthor=J.+E.+Bradner&title=Therapeutic+Strategies+to+Inhibit+MYC&doi=10.1101%2Fcshperspect.a014266"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a014266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a014266%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DTherapeutic%2520Strategies%2520to%2520Inhibit%2520MYC%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2014%26volume%3D4%26spage%3Da014266%26doi%3D10.1101%2Fcshperspect.a014266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitfield, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Strategies to Inhibit MYC and Their Clinical Applicability</span>. <i>Front. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.3389/fcell.2017.00010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3389%2Ffcell.2017.00010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28280720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADC%252BC1czkvFamtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=10&author=J.+R.+Whitfieldauthor=M.-E.+Beaulieuauthor=L.+Soucek&title=Strategies+to+Inhibit+MYC+and+Their+Clinical+Applicability&doi=10.3389%2Ffcell.2017.00010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to Inhibit Myc and Their Clinical Applicability</span></div><div class="casAuthors">Whitfield Jonathan R; Beaulieu Marie-Eve; Soucek Laura</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in cell and developmental biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn">2296-634X</span>.
    </div><div class="casAbstract">Myc is an oncogene deregulated in most-perhaps all-human cancers.  Each Myc family member, c-, L-, and N-Myc, has been connected to tumor progression and maintenance.  Myc is recognized as a "most wanted" target for cancer therapy, but has for many years been considered undruggable, mainly due to its nuclear localization, lack of a defined ligand binding site, and physiological function essential to the maintenance of normal tissues.  The challenge of identifying a pharmacophore capable of overcoming these hurdles is reflected in the current absence of a clinically-viable Myc inhibitor.  The first attempts to inhibit Myc used antisense technology some three decades ago, followed by small molecule inhibitors discovered through "classical" compound library screens.  Notable breakthroughs proving the feasibility of systemic Myc inhibition were made with the Myc dominant negative mutant Omomyc, showing both the great promise in targeting this infamous oncogene for cancer treatment as well as allaying fears about the deleterious side effects that Myc inhibition might have on normal proliferating tissues.  During this time many other strategies have appeared in an attempt to drug the undruggable, including direct and indirect targeting, knockdown, protein/protein and DNA interaction inhibitors, and translation and expression regulation.  The inhibitors range from traditional small molecules to natural chemicals, to RNA and antisense, to peptides and miniproteins.  Here, we briefly describe the many approaches taken so far, with a particular focus on their potential clinical applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQn7rW6dLAkzqx48Rj5GnGofW6udTcc2easZCQm-6kWmrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czkvFamtw%253D%253D&md5=5e65b897b6bec6b93de5fbf22a05b226</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffcell.2017.00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffcell.2017.00010%26sid%3Dliteratum%253Aachs%26aulast%3DWhitfield%26aufirst%3DJ.%2BR.%26aulast%3DBeaulieu%26aufirst%3DM.-E.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DStrategies%2520to%2520Inhibit%2520MYC%2520and%2520Their%2520Clinical%2520Applicability%26jtitle%3DFront.%2520Cell%2520Dev.%2520Biol.%26date%3D2017%26volume%3D5%26spage%3D10%26doi%3D10.3389%2Ffcell.2017.00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalkat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resetca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lourenco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiah, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunnerhagen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Done, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raught, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">MYC Protein Interactome Profiling Reveals Functionally Distinct Regions That Cooperate to Drive Tumorigenesis</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">836</span>– <span class="NLM_lpage">848.e7</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.09.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2018.09.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=30415952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCgsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2018&pages=836-848.e7&author=M.+Kalkatauthor=D.+Resetcaauthor=C.+Lourencoauthor=P.-K.+Chanauthor=Y.+Weiauthor=Y.-J.+Shiahauthor=N.+Vitkinauthor=Y.+Tongauthor=M.+Sunnerhagenauthor=S.+J.+Doneauthor=P.+C.+Boutrosauthor=B.+Raughtauthor=L.+Z.+Penn&title=MYC+Protein+Interactome+Profiling+Reveals+Functionally+Distinct+Regions+That+Cooperate+to+Drive+Tumorigenesis&doi=10.1016%2Fj.molcel.2018.09.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MYC Protein Interactome Profiling Reveals Functionally Distinct Regions that Cooperate to Drive Tumorigenesis</span></div><div class="casAuthors">Kalkat, Manpreet; Resetca, Diana; Lourenco, Corey; Chan, Pak-Kei; Wei, Yong; Shiah, Yu-Jia; Vitkin, Natasha; Tong, Yufeng; Sunnerhagen, Maria; Done, Susan J.; Boutros, Paul C.; Raught, Brian; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">836-848.e7</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Transforming members of the MYC family (MYC, MYCL1, and MYCN) encode transcription factors contg. six highly conserved regions, termed MYC homol. boxes (MBs).  By conducting proteomic profiling of the MB interactomes, we demonstrate that half of the MYC interactors require one or more MBs for binding.  Comprehensive phenotypic analyses reveal that two MBs, MB0 and MBII, are universally required for transformation.  MBII mediates interactions with acetyltransferase-contg. complexes, enabling histone acetylation, and is essential for MYC-dependent tumor initiation.  By contrast, MB0 mediates interactions with transcription elongation factors via direct binding to the general transcription factor TFIIF.  MB0 is dispensable for tumor initiation but is a major accelerator of tumor growth.  Notably, the full transforming activity of MYC can be restored by co-expression of the non-transforming MB0 and MBII deletion proteins, indicating that these two regions confer sep. mol. functions, both of which are required for oncogenic MYC activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYfeSlTYyXB7Vg90H21EOLACvtfcHk0lhJv_6ERMQ9LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCgsbrJ&md5=119154face5629c3133fe3a6eeea1989</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.09.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.09.031%26sid%3Dliteratum%253Aachs%26aulast%3DKalkat%26aufirst%3DM.%26aulast%3DResetca%26aufirst%3DD.%26aulast%3DLourenco%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DP.-K.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DShiah%26aufirst%3DY.-J.%26aulast%3DVitkin%26aufirst%3DN.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DSunnerhagen%26aufirst%3DM.%26aulast%3DDone%26aufirst%3DS.%2BJ.%26aulast%3DBoutros%26aufirst%3DP.%2BC.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DMYC%2520Protein%2520Interactome%2520Profiling%2520Reveals%2520Functionally%2520Distinct%2520Regions%2520That%2520Cooperate%2520to%2520Drive%2520Tumorigenesis%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D72%26spage%3D836%26epage%3D848.e7%26doi%3D10.1016%2Fj.molcel.2018.09.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desharnais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonderegger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maslyar, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Antagonists of MYC/Max Dimerization Inhibit MYC-Induced Transformation of Chicken Embryo Fibroblasts</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">3830</span>– <span class="NLM_lpage">3835</span>, <span class="refDoi"> DOI: 10.1073/pnas.062036999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.062036999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=11891322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1KltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=3830-3835&author=T.+Bergauthor=S.+B.+Cohenauthor=J.+Desharnaisauthor=C.+Sondereggerauthor=D.+J.+Maslyarauthor=J.+Goldbergauthor=D.+L.+Bogerauthor=P.+K.+Vogt&title=Small-Molecule+Antagonists+of+MYC%2FMax+Dimerization+Inhibit+MYC-Induced+Transformation+of+Chicken+Embryo+Fibroblasts&doi=10.1073%2Fpnas.062036999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule antagonists of Myc/max dimerization inhibit Myc-induced transformation of chicken embryo fibroblasts</span></div><div class="casAuthors">Berg, Thorsten; Cohen, Steven B.; Desharnais, Joel; Sonderegger, Corinna; Maslyar, Daniel J.; Goldberg, Joel; Boger, Dale L.; Vogt, Peter K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3830-3835</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myc is a transcriptional regulator of the basic helix-loop-helix leucine zipper protein family.  It has strong oncogenic potential, mutated or virally transduced forms of Myc induce lymphoid tumors in animals, and deregulated expression of Myc is assocd. with numerous types of human cancers.  For its oncogenic activity, Myc must dimerize with the ubiquitously expressed basic helix-loop-helix leucine zipper protein Max.  This requirement for dimerization may allow control of Myc activity with small mols. that interfere with Myc/Max dimerization.  We have measured Myc/Max dimerization with fluorescence resonance energy transfer and have screened combinatorial chem. libraries for inhibitors of dimerization.  Candidate inhibitors were isolated from a peptidomimetics library.  Inhibition of Myc/Max interaction was validated by ELISA and electrophoretic mobility-shift assay.  Two of the candidate inhibitors also interfere with Myc-induced oncogenic transformation in chicken embryo fibroblast cultures.  Our work provides proof of principle for the identification of small mol. inhibitors of protein-protein interactions by using high-throughput screens of combinatorial chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkAaKA-IEYw7Vg90H21EOLACvtfcHk0lgb7nWSZbzWEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1KltL0%253D&md5=7712f526695657c3a31fdbd722f10b40</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062036999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062036999%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DS.%2BB.%26aulast%3DDesharnais%26aufirst%3DJ.%26aulast%3DSonderegger%26aufirst%3DC.%26aulast%3DMaslyar%26aufirst%3DD.%2BJ.%26aulast%3DGoldberg%26aufirst%3DJ.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26atitle%3DSmall-Molecule%2520Antagonists%2520of%2520MYC%252FMax%2520Dimerization%2520Inhibit%2520MYC-Induced%2520Transformation%2520of%2520Chicken%2520Embryo%2520Fibroblasts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D3830%26epage%3D3835%26doi%3D10.1073%2Fpnas.062036999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giap, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Low Molecular Weight Inhibitors of MYC–Max Interaction and Function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6159</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1206641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fsj.onc.1206641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=13679853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntlWhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=6151-6159&author=X.+Yinauthor=C.+Giapauthor=J.+S.+Lazoauthor=E.+V.+Prochownik&title=Low+Molecular+Weight+Inhibitors+of+MYC%E2%80%93Max+Interaction+and+Function&doi=10.1038%2Fsj.onc.1206641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Low molecular weight inhibitors of Myc-Max interaction and function</span></div><div class="casAuthors">Yin, Xiaoying; Giap, Christine; Lazo, John S.; Prochownik, Edward V.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">6151-6159</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">C-Myc is helix-loop-helix-leucine zipper (HLH-ZIP) oncoprotein that is frequently deregulated in human cancers.  To bind DNA, regulate target gene expression, and function in a biol. context, c-Myc must dimerize with another HLH-ZIP protein, Max.  A large no. of c-Myc target genes have been identified, and many of the encoded proteins are transforming.  Such functional redundancy, however, complicates therapeutic strategies aimed at inhibiting any single target gene product.  Given this consideration, we have instead attempted to identify ways by which c-Myc itself could be effectively disabled.  We have used a yeast two-hybrid approach to identify low-mol.-wt. compds. that inhibit c-Myc-Max assocn.  All of the compds. prevented transactivation by c-Myc-Max heterodimers, inhibited cell cycle progression, and prevented the in vitro growth of fibroblasts in a c-Myc-dependent manner.  Several of the compds. also inhibited tumor growth in vivo.  These results show that the yeast two-hybrid screen is useful for identifying compds. that can be exploited in mammalian cells.  More specifically, they provide a means by which structural analogs, based upon these first-generation Myc-Max inhibitors, can be developed to enhance antitumor efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowgLbI0hqQcLVg90H21EOLACvtfcHk0lgb7nWSZbzWEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntlWhtLs%253D&md5=3a418e091182370b30b5008aafc47c96</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1206641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1206641%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DX.%26aulast%3DGiap%26aufirst%3DC.%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DLow%2520Molecular%2520Weight%2520Inhibitors%2520of%2520MYC%25E2%2580%2593Max%2520Interaction%2520and%2520Function%26jtitle%3DOncogene%26date%3D2003%26volume%3D22%26spage%3D6151%26epage%3D6159%26doi%3D10.1038%2Fsj.onc.1206641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabato, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Disruption of MYC-Max Heterodimerization with Improved Cell-Penetrating Analogs of the Small Molecule 10074-G5</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.18632%2Foncotarget.1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=23801058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADC%252BC3sjlsV2ksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=936-947&author=H.+Wangauthor=J.+Chauhanauthor=A.+Huauthor=K.+Pendletonauthor=J.+L.+Yapauthor=P.+E.+Sabatoauthor=J.+W.+Jonesauthor=M.+Perriauthor=J.+Yuauthor=E.+Cioneauthor=M.+A.+Kaneauthor=S.+Fletcherauthor=E.+V.+Prochownik&title=Disruption+of+MYC-Max+Heterodimerization+with+Improved+Cell-Penetrating+Analogs+of+the+Small+Molecule+10074-G5&doi=10.18632%2Foncotarget.1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of Myc-Max heterodimerization with improved cell-penetrating analogs of the small molecule 10074-G5</span></div><div class="casAuthors">Wang Huabo; Chauhan Jay; Hu Angela; Pendleton Kelsey; Yap Jeremy L; Sabato Philip E; Jones Jace W; Perri Mariarita; Yu Jianshi; Cione Erika; Kane Maureen A; Fletcher Steven; Prochownik Edward V</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">936-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The c-Myc (Myc) oncoprotein is a high-value therapeutic target given that it is deregulated in multiple types of cancer.  However, potent small molecule inhibitors of Myc have been difficult to identify, particularly those whose mechanism relies on blocking the association between Myc and its obligate heterodimerization partner, Max.  We have recently reported a structure-activity relationship study of one such small molecule, 10074-G5, and generated an analog, JY-3-094, with significantly improved ability to prevent or disrupt the association between recombinant Myc and Max proteins.  However, JY-3094 penetrates cells poorly.  Here, we show that esterification of a critical para-carboxylic acid function of JY-3-094 by various blocking groups significantly improves cellular uptake although it impairs the ability to disrupt Myc-Max association in vitro.  These pro-drugs are highly concentrated within cells where JY-3-094 is then generated by the action of esterases.  However, the pro-drugs are also variably susceptible to extracellular esterases, which can deplete extracellular reservoirs.  Furthermore, while JY-3-094 is retained by cells for long periods of time, much of it is compartmentalized within the cytoplasm in a form that appears to be less available to interact with Myc.  Our results suggest that persistently high extracellular levels of pro-drug, without excessive susceptibility to extracellular esterases, are critical to establishing and maintaining intracellular levels of JY-3-094 that are sufficient to provide for long-term inhibition of Myc-Max association.  Analogs of JY-3-094 appear to represent promising small molecule Myc inhibitors that warrant further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUpo3PsQLURaqcneEtCi92fW6udTcc2ebjG-OdSn-xD7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjlsV2ksA%253D%253D&md5=044730ff03610d8981f13a77f0edb10a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1108%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DChauhan%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DA.%26aulast%3DPendleton%26aufirst%3DK.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DSabato%26aufirst%3DP.%2BE.%26aulast%3DJones%26aufirst%3DJ.%2BW.%26aulast%3DPerri%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DCione%26aufirst%3DE.%26aulast%3DKane%26aufirst%3DM.%2BA.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DDisruption%2520of%2520MYC-Max%2520Heterodimerization%2520with%2520Improved%2520Cell-Penetrating%2520Analogs%2520of%2520the%2520Small%2520Molecule%252010074-G5%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D936%26epage%3D947%26doi%3D10.18632%2Foncotarget.1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of C-MYC/Max Dimerization and DNA Binding by Small Molecules</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">751</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.chembiol.2006.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=16873022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD28XntlGis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=745-751&author=A.+Kiesslingauthor=B.+Sperlauthor=A.+Hollisauthor=D.+Eickauthor=T.+Berg&title=Selective+Inhibition+of+C-MYC%2FMax+Dimerization+and+DNA+Binding+by+Small+Molecules&doi=10.1016%2Fj.chembiol.2006.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of c-Myc/Max Dimerization and DNA Binding by Small Molecules</span></div><div class="casAuthors">Kiessling, Anke; Sperl, Bianca; Hollis, Angela; Eick, Dirk; Berg, Thorsten</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">745-751</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: bZip and bHLHZip protein family members comprise a large fraction of eukaryotic transcription factors and need to bind DNA in order to exert most of their fundamental biol. roles.  Their binding to DNA requires homo- or heterodimerization via α-helical domains, which generally do not contain obvious binding sites for small mols.  We have identified two small mols., dubbed Mycro1 and Mycro2, which inhibit the protein-protein interactions between the bHLHZip proteins c-Myc and Max.  Mycros are the first inhibitors of c-Myc/Max dimerization, which have been demonstrated to inhibit DNA binding of c-Myc with preference over other dimeric transcription factors in vitro.  Mycros inhibit c-Myc-dependent proliferation, gene transcription, and oncogenic transformation in the low micromolar concn. range.  Our data support the idea that dimeric transcription factors can be druggable even in the absence of obvious small-mol. binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwi1DGz-5I0bVg90H21EOLACvtfcHk0lgb7nWSZbzWEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntlGis7Y%253D&md5=e90e63fc10e677504f217b922d6c1d04</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DKiessling%26aufirst%3DA.%26aulast%3DSperl%26aufirst%3DB.%26aulast%3DHollis%26aufirst%3DA.%26aulast%3DEick%26aufirst%3DD.%26aulast%3DBerg%26aufirst%3DT.%26atitle%3DSelective%2520Inhibition%2520of%2520C-MYC%252FMax%2520Dimerization%2520and%2520DNA%2520Binding%2520by%2520Small%2520Molecules%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D745%26epage%3D751%26doi%3D10.1016%2Fj.chembiol.2006.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baksh, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bister, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janda, K. D.</span></span> <span> </span><span class="NLM_article-title">Inhibitor of MYC Identified in a Kröhnke Pyridine Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12556</span>– <span class="NLM_lpage">12561</span>, <span class="refDoi"> DOI: 10.1073/pnas.1319488111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.1319488111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25114221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12556-12561&author=J.+R.+Hartauthor=A.+L.+Garnerauthor=J.+Yuauthor=Y.+Itoauthor=M.+Sunauthor=L.+Uenoauthor=J.-K.+Rheeauthor=M.+M.+Bakshauthor=E.+Stefanauthor=M.+Hartlauthor=K.+Bisterauthor=P.+K.+Vogtauthor=K.+D.+Janda&title=Inhibitor+of+MYC+Identified+in+a+Kr%C3%B6hnke+Pyridine+Library&doi=10.1073%2Fpnas.1319488111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor of MYC identified in a Krohnke pyridine library</span></div><div class="casAuthors">Hart, Jonathan R.; Garner, Amanda L.; Yu, Jing; Ito, Yoshihiro; Sun, Minghao; Ueno, Lynn; Rhee, Jin-Kyu; Baksh, Michael M.; Stefan, Eduard; Hartl, Markus; Bister, Klaus; Vogt, Peter K.; Janda, Kim D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12556-12561</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a fluorescence polarization screen for the MYC-MAX interaction, we have identified a novel small-mol. inhibitor of MYC, KJ-Pyr-9, from a Krohnke pyridine library.  The Kd of KJ-Pyr-9 for MYC in vitro is 6.5 ± 1.0 nM, as detd. by backscattering interferometry; KJ-Pyr-9 also interferes with MYC-MAX complex formation in the cell, as shown in a protein fragment complementation assay.  KJ-Pyr-9 specifically inhibits MYC-induced oncogenic transformation in cell culture; it has no or only weak effects on the oncogenic activity of several unrelated oncoproteins.  KJ-Pyr-9 preferentially interferes with the proliferation of MYC-overexpressing human and avian cells and specifically reduces the MYC-driven transcriptional signature.  In vivo, KJ-Pyr-9 effectively blocks the growth of a xenotransplant of MYC-amplified human cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNNR9yUaQgSLVg90H21EOLACvtfcHk0lhA2cN20dxU_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nF&md5=97a65b219a5b98e1df5b2c264dfc2723</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1319488111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1319488111%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DJ.%2BR.%26aulast%3DGarner%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DM.%26aulast%3DUeno%26aufirst%3DL.%26aulast%3DRhee%26aufirst%3DJ.-K.%26aulast%3DBaksh%26aufirst%3DM.%2BM.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DHartl%26aufirst%3DM.%26aulast%3DBister%26aufirst%3DK.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26aulast%3DJanda%26aufirst%3DK.%2BD.%26atitle%3DInhibitor%2520of%2520MYC%2520Identified%2520in%2520a%2520Kr%25C3%25B6hnke%2520Pyridine%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12556%26epage%3D12561%26doi%3D10.1073%2Fpnas.1319488111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerowitz, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekritz, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonds, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthay, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span> <span> </span><span class="NLM_article-title">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.ccr.2014.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25175806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=414-427&author=W.+C.+Gustafsonauthor=J.+G.+Meyerowitzauthor=E.+A.+Nekritzauthor=J.+Chenauthor=C.+Benesauthor=E.+Charronauthor=E.+F.+Simondsauthor=R.+Seegerauthor=K.+K.+Matthayauthor=N.+T.+Hertzauthor=M.+Eilersauthor=K.+M.+Shokatauthor=W.+A.+Weiss&title=Drugging+MYCN+through+an+Allosteric+Transition+in+Aurora+Kinase+A&doi=10.1016%2Fj.ccr.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span></div><div class="casAuthors">Gustafson, William Clay; Meyerowitz, Justin Gabriel; Nekritz, Erin A.; Chen, Justin; Benes, Cyril; Charron, Elise; Simonds, Erin F.; Seeger, Robert; Matthay, Katherine K.; Hertz, Nicholas T.; Eilers, Martin; Shokat, Kevan M.; Weiss, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">414-427</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC proteins are major drivers of cancer, yet are considered undruggable because their DNA binding domains are composed of two extended alpha helixes with no apparent surfaces for small-mol. binding.  Proteolytic degrdn. of MYCN protein is regulated in part by a kinase-independent function of Aurora A.  We describe a class of inhibitors that disrupts the native conformation of Aurora A and drives the degrdn. of MYCN protein across MYCN-driven cancers.  Comparison of cocrystal structures with structure-activity relationships across multiple inhibitors and chemotypes, coupled with mechanistic studies and biochem. assays, delineates an Aurora A conformation-specific effect on proteolytic degrdn. of MYCN, rather than simple nanomolar-level inhibition of Aurora A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8At2LNPE-ubVg90H21EOLACvtfcHk0lhA2cN20dxU_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ&md5=20944894851fe5466422c149a0a0b266</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DW.%2BC.%26aulast%3DMeyerowitz%26aufirst%3DJ.%2BG.%26aulast%3DNekritz%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%26aulast%3DCharron%26aufirst%3DE.%26aulast%3DSimonds%26aufirst%3DE.%2BF.%26aulast%3DSeeger%26aufirst%3DR.%26aulast%3DMatthay%26aufirst%3DK.%2BK.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26atitle%3DDrugging%2520MYCN%2520through%2520an%2520Allosteric%2520Transition%2520in%2520Aurora%2520Kinase%2520A%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D26%26spage%3D414%26epage%3D427%26doi%3D10.1016%2Fj.ccr.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carstensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of N-MYC Binding by Aurora-A and Its Destabilization by Kinase Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13726</span>– <span class="NLM_lpage">13731</span>, <span class="refDoi"> DOI: 10.1073/pnas.1610626113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1073%2Fpnas.1610626113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27837025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCiur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=13726-13731&author=M.+W.+Richardsauthor=S.+G.+Burgessauthor=E.+Poonauthor=A.+Carstensenauthor=M.+Eilersauthor=L.+Cheslerauthor=R.+Bayliss&title=Structural+Basis+of+N-MYC+Binding+by+Aurora-A+and+Its+Destabilization+by+Kinase+Inhibitors&doi=10.1073%2Fpnas.1610626113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of N-Myc binding by Aurora-A and its destabilization by kinase inhibitors</span></div><div class="casAuthors">Richards, Mark W.; Burgess, Selena G.; Poon, Evon; Carstensen, Anne; Eilers, Martin; Chesler, Louis; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">13726-13731</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myc family proteins promote cancer by inducing widespread changes in gene expression.  Their rapid turnover by the ubiquitin-proteasome pathway is regulated through phosphorylation of Myc Box I and ubiquitination by E3 ubiquitin ligase SCFFbxW7.  However, N-Myc protein (the product of the MYCN oncogene) is stabilized in neuroblastoma by the protein kinase Aurora-A in a manner that is sensitive to certain Aurora-A-selective inhibitors.  Here, the authors identified a direct interaction between the catalytic domain of Aurora-A kinase and a site flanking Myc Box I that also binds SCFFbxW7.  The authors detd. the crystal structure of the complex between Aurora-A kinase and this region of N-Myc to 1.72-Å resoln.  The structure indicated that the conformation of Aurora-A induced by compds. such as alisertib and CD532 was not compatible with the binding of N-Myc, explaining the activity of these compds. in neuroblastoma cells and providing a rational basis for the design of cancer therapeutics optimized for destabilization of the complex.  The authors also proposed a model for the stabilization mechanism in which binding to Aurora-A alters how N-Myc interacts with SCFFbxW7 to disfavor the generation of Lys-48-linked polyubiquitin chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAK31pqP6M4bVg90H21EOLACvtfcHk0lhA2cN20dxU_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCiur7I&md5=0aa2b960303f040d9685fb16fb8ea980</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1610626113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1610626113%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DPoon%26aufirst%3DE.%26aulast%3DCarstensen%26aufirst%3DA.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DStructural%2520Basis%2520of%2520N-MYC%2520Binding%2520by%2520Aurora-A%2520and%2520Its%2520Destabilization%2520by%2520Kinase%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D13726%26epage%3D13731%26doi%3D10.1073%2Fpnas.1610626113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span> <span> </span><span class="NLM_article-title">A Moving Target: Structure and Disorder in Pursuit of MYC Inhibitors</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1042/BST20160328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1042%2FBST20160328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=28620032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWjsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=709-717&author=R.+Baylissauthor=S.+G.+Burgessauthor=E.+Leenauthor=M.+W.+Richards&title=A+Moving+Target%3A+Structure+and+Disorder+in+Pursuit+of+MYC+Inhibitors&doi=10.1042%2FBST20160328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A moving target: structure and disorder in pursuit of Myc inhibitors</span></div><div class="casAuthors">Bayliss, Richard; Burgess, Selena G.; Leen, Eoin; Richards, Mark W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">709-717</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The Myc proteins comprise a family of ubiquitous regulators of gene expression implicated in over half of all human cancers.  They interact with a large no. of other proteins, such as transcription factors, chromatin-modifying enzymes and kinases.  Remarkably, few of these interactions have been characterized structurally.  This is at least in part due to the intrinsically disordered nature of Myc proteins, which adopt a defined conformation only in the presence of binding partners.  Owing to this behavior, crystallog. studies on Myc proteins have been limited to short fragments in complex with other proteins.  Most recently, we detd. the crystal structure of Aurora-A kinase domain bound to a 28-amino acid fragment of the N-Myc transactivation domain.  The structure reveals an α-helical segment within N-Myc capped by two tryptophan residues that recognize the surface of Aurora-A.  The kinase domain acts as a mol. scaffold, independently of its catalytic activity, upon which this region of N-Myc becomes ordered.  The binding site for N-Myc on Aurora-A is disrupted by certain ATP-competitive inhibitors, such as MLN8237 (alisertib) and CD532, and explains how these kinase inhibitors are able to disrupt the protein-protein interaction to affect Myc destabilization.  Structural studies on this and other Myc complexes will lead to the design of protein-protein interaction inhibitors as chem. tools to dissect the complex pathways of Myc regulation and function, which may be developed into Myc inhibitors for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv3uTLeXzAGbVg90H21EOLACvtfcHk0lhrWUpZV1FtlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWjsL%252FN&md5=c4a9d8dd32f6725c365203289cfdc018</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1042%2FBST20160328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20160328%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DLeen%26aufirst%3DE.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26atitle%3DA%2520Moving%2520Target%253A%2520Structure%2520and%2520Disorder%2520in%2520Pursuit%2520of%2520MYC%2520Inhibitors%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2017%26volume%3D45%26spage%3D709%26epage%3D717%26doi%3D10.1042%2FBST20160328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieb, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">440</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25818646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=440-452&author=L.+R.+Thomasauthor=Q.+Wangauthor=B.+C.+Griebauthor=J.+Phanauthor=A.+M.+Foshageauthor=Q.+Sunauthor=E.+T.+Olejniczakauthor=T.+Clarkauthor=S.+Deyauthor=S.+Loreyauthor=B.+Alicieauthor=G.+C.+Howardauthor=B.+Cawthonauthor=K.+C.+Essauthor=C.+M.+Eischenauthor=Z.+Zhaoauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Interaction+with+WDR5+Promotes+Target+Gene+Recognition+and+Tumorigenesis+by+MYC&doi=10.1016%2Fj.molcel.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with WDR5 Promotes Target Gene Recognition and Tumorigenesis by MYC</span></div><div class="casAuthors">Thomas, Lance R.; Wang, Qingguo; Grieb, Brian C.; Phan, Jason; Foshage, Audra M.; Sun, Qi; Olejniczak, Edward T.; Clark, Travis; Dey, Soumyadeep; Lorey, Shelly; Alicie, Bethany; Howard, Gregory C.; Cawthon, Bryan; Ess, Kevin C.; Eischen, Christine M.; Zhao, Zhongming; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">440-452</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC is an oncoprotein transcription factor that is overexpressed in the majority of malignancies.  The oncogenic potential of MYC stems from its ability to bind regulatory sequences in thousands of target genes, which depends on interaction of MYC with its obligate partner, MAX.  Here, we show that broad assocn. of MYC with chromatin also depends on interaction with the WD40-repeat protein WDR5.  MYC binds WDR5 via an evolutionarily conserved "MYC box IIIb" motif that engages a shallow, hydrophobic cleft on the surface of WDR5.  Structure-guided mutations in MYC that disrupt interaction with WDR5 attenuate binding of MYC at ∼80% of its chromosomal locations and disable its ability to promote induced pluripotent stem cell formation and drive tumorigenesis.  Our data reveal WDR5 as a key determinant for MYC recruitment to chromatin and uncover a tractable target for the discovery of anticancer therapies against MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfifOnQkKDHrVg90H21EOLACvtfcHk0lhrWUpZV1FtlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOisrk%253D&md5=329876ab3db1a16fb58bd5de7f1de7b9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DGrieb%26aufirst%3DB.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DLorey%26aufirst%3DS.%26aulast%3DAlicie%26aufirst%3DB.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DCawthon%26aufirst%3DB.%26aulast%3DEss%26aufirst%3DK.%2BC.%26aulast%3DEischen%26aufirst%3DC.%2BM.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DInteraction%2520with%2520WDR5%2520Promotes%2520Target%2520Gene%2520Recognition%2520and%2520Tumorigenesis%2520by%2520MYC%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D440%26epage%3D452%26doi%3D10.1016%2Fj.molcel.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">The MYC-WDR5 Nexus and Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">4012</span>– <span class="NLM_lpage">4015</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-1216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1158%2F0008-5472.CAN-15-1216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26383167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyrtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=4012-4015&author=L.+R.+Thomasauthor=A.+M.+Foshageauthor=A.+M.+Weissmillerauthor=W.+P.+Tansey&title=The+MYC-WDR5+Nexus+and+Cancer&doi=10.1158%2F0008-5472.CAN-15-1216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The MYC-WDR5 Nexus and Cancer</span></div><div class="casAuthors">Thomas, Lance R.; Foshage, Audra M.; Weissmiller, April M.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4012-4015</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The MYC oncogenes encode a family of transcription factors that feature prominently in cancer.  MYC proteins are overexpressed or deregulated in a majority of malignancies and drive tumorigenesis by inducing widespread transcriptional reprogramming that promotes cell proliferation, metab., and genomic instability.  The ability of MYC to regulate transcription depends on its dimerization with MAX, which creates a DNA-binding domain that recognizes specific sequences in the regulatory elements of MYC target genes.  Recently, we discovered that recognition of target genes by MYC also depends on its interaction with WDR5, a WD40-repeat protein that exists as part of several chromatin-regulatory complexes.  Here, we discuss how interaction of MYC with WDR5 could create an avidity-based chromatin recognition mechanism that allows MYC to select its target genes in response to both genetic and epigenetic determinants.  We rationalize how the MYC-WDR5 interaction provides plasticity in target gene selection by MYC and speculate on the biochem. and genomic contexts in which this interaction occurs.  Finally, we discuss how properties of the MYC-WDR5 interface make it an attractive point for discovery of small-mol. inhibitors of MYC function in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB2HGlzfYzWLVg90H21EOLACvtfcHk0lgHGC_h4jbNjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyrtbvM&md5=7caa91f77ba27f4b8224f32d1c66ea9e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-1216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-1216%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DThe%2520MYC-WDR5%2520Nexus%2520and%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D4012%26epage%3D4015%26doi%3D10.1158%2F0008-5472.CAN-15-1216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaitatzes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neer, E. J.</span></span> <span> </span><span class="NLM_article-title">The WD Repeat: A Common Architecture for Diverse Functions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(99)01384-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0968-0004%2899%2901384-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10322433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK1MXkslSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1999&pages=181-185&author=T.+F.+Smithauthor=C.+Gaitatzesauthor=K.+Saxenaauthor=E.+J.+Neer&title=The+WD+Repeat%3A+A+Common+Architecture+for+Diverse+Functions&doi=10.1016%2FS0968-0004%2899%2901384-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The WD repeat: a common architecture for diverse functions</span></div><div class="casAuthors">Smith, Temple F.; Gaitatzes, Chrysanthe; Saxena, Kumkum; Neer, Eva J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">181-185</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 40 refs.  Our knowledge of the large family of proteins that contain the WD repeat continues to accumulate.  The WD-repeat proteins are found in all eukaryotes and are implicated in a wide variety of crucial functions.  The soln. of the three-dimensional structure of one WD-repeat protein and the assumption that the structure will be common to all members of this family has allowed subfamilies of WD-repeat proteins to be defined on the basis of probable surface similarity.  Proteins that have very similar surfaces are likely to have common binding partners and similar functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5gp-sO0-v-bVg90H21EOLACvtfcHk0lgHGC_h4jbNjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkslSnsbc%253D&md5=c48cb0bacb96cc575b0629fbb7322d26</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2899%2901384-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252899%252901384-5%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DT.%2BF.%26aulast%3DGaitatzes%26aufirst%3DC.%26aulast%3DSaxena%26aufirst%3DK.%26aulast%3DNeer%26aufirst%3DE.%2BJ.%26atitle%3DThe%2520WD%2520Repeat%253A%2520A%2520Common%2520Architecture%2520for%2520Diverse%2520Functions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1999%26volume%3D24%26spage%3D181%26epage%3D185%26doi%3D10.1016%2FS0968-0004%2899%2901384-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of WD40 Domain Proteins</span>. <i>Protein Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1007/s13238-011-1018-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1007%2Fs13238-011-1018-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21468892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt12iur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=202-214&author=C.+Xuauthor=J.+Min&title=Structure+and+Function+of+WD40+Domain+Proteins&doi=10.1007%2Fs13238-011-1018-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of WD40 domain proteins</span></div><div class="casAuthors">Xu, Chao; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-214</span>CODEN:
                <span class="NLM_cas:coden">PCREFB</span>;
        ISSN:<span class="NLM_cas:issn">1674-800X</span>.
    
            (<span class="NLM_cas:orgname">Higher Education Press</span>)
        </div><div class="casAbstract">A review.  The WD40 domain exhibits a β-propeller architecture, often comprising seven blades.  The WD40 domain is one of the most abundant domains and also among the top interacting domains in eukaryotic genomes.  In this review, we will discuss the identification, definition and architecture of the WD40 domains.  WD40 domain proteins are involved in a large variety of cellular processes, in which WD40 domains function as a protein-protein or protein-DNA interaction platform.  WD40 domain mediates mol. recognition events mainly through the smaller top surface, but also through the bottom surface and sides.  So far, no WD40 domain has been found to display enzymic activity.  We will also discuss the different binding modes exhibited by the large versatile family of WD40 domain proteins.  In the last part of this review, we will discuss how post-translational modifications are recognized by WD40 domain proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPx0CEOy9VnbVg90H21EOLACvtfcHk0lgHGC_h4jbNjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt12iur0%253D&md5=478e87d9f93f17afb65b710052174d37</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13238-011-1018-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13238-011-1018-1%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Function%2520of%2520WD40%2520Domain%2520Proteins%26jtitle%3DProtein%2520Cell%26date%3D2011%26volume%3D2%26spage%3D202%26epage%3D214%26doi%3D10.1007%2Fs13238-011-1018-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stirnimann, C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsalaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. W.</span></span> <span> </span><span class="NLM_article-title">WD40 Proteins Propel Cellular Networks</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.tibs.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20451393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=565-574&author=C.+U.+Stirnimannauthor=E.+Petsalakiauthor=R.+B.+Russellauthor=C.+W.+M%C3%BCller&title=WD40+Proteins+Propel+Cellular+Networks&doi=10.1016%2Fj.tibs.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">WD40 proteins propel cellular networks</span></div><div class="casAuthors">Stirnimann, Christian U.; Petsalaki, Evangelia; Russell, Robert B.; Mueller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">565-574</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent findings indicate that WD40 domains play central roles in biol. processes by acting as hubs in cellular networks; however, they have been studied less intensely than other common domains, such as the kinase, PDZ or SH3 domains.  As suggested by various interactome studies, they are among the most promiscuous interactors.  Structural studies suggest that this property stems from their ability, as scaffolds, to interact with diverse proteins, peptides or nucleic acids using multiple surfaces or modes of interaction.  A general scaffolding role is supported by the fact that no WD40 domain has been found with intrinsic enzymic activity despite often being part of large mol. machines.  We discuss the WD40 domain distributions in protein networks and structures of WD40-contg. assemblies to demonstrate their versatility in mediating crit. cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruw7yoJ7_J-LVg90H21EOLACvtfcHk0lgHGC_h4jbNjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7%252FO&md5=47c47c118b1c85f3d1726a873283ddb2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DStirnimann%26aufirst%3DC.%2BU.%26aulast%3DPetsalaki%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DR.%2BB.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BW.%26atitle%3DWD40%2520Proteins%2520Propel%2520Cellular%2520Networks%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D565%26epage%3D574%26doi%3D10.1016%2Fj.tibs.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Disease Association and Druggability of WD40 Repeat Proteins</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3766</span>– <span class="NLM_lpage">3773</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.7b00451</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.7b00451" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKht7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=3766-3773&author=R.+Songauthor=Z.-D.+Wangauthor=M.+Schapira&title=Disease+Association+and+Druggability+of+WD40+Repeat+Proteins&doi=10.1021%2Facs.jproteome.7b00451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Disease Association and Druggability of WD40 Repeat Proteins</span></div><div class="casAuthors">Song, Richard; Wang, Zhong-Duo; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3766-3773</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD40 repeat (WDR) domains are protein interaction scaffolds that represent one of the largest protein families in human, and a first WDR inhibitor-an allosteric antagonist of polycomb repressive complex 2-just entered the clinic.  A systematic anal. of the CORUM database of protein complexes shows that WDR is the most represented domain in transcriptional regulation and one of the most prevalent in the ubiquitin proteasome system, two pathways of high relevance to drug discovery.  Parsing the literature and the vulnerability of cancer cell lines to CRISPR knockout indicates that WDR proteins are targets of interest in oncol. and other disease areas.  A quant. anal. of WDR structures reveals that druggable binding pockets can be found on multiple surfaces of these multifaceted protein interaction platforms.  These data support the development of chem. probes to further interrogate WDR proteins as an emerging therapeutic target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-4QqKGfC1KrVg90H21EOLACvtfcHk0ljyNxr-gvGYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKht7fM&md5=368abb89a0ecbe30c057fb2521b7ee8c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.7b00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.7b00451%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.-D.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDisease%2520Association%2520and%2520Druggability%2520of%2520WD40%2520Repeat%2520Proteins%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2017%26volume%3D16%26spage%3D3766%26epage%3D3773%26doi%3D10.1021%2Facs.jproteome.7b00451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guarnaccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W.</span></span> <span> </span><span class="NLM_article-title">Moonlighting with WDR5: A Cellular Multitasker</span>. <i>J. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.3390/jcm7020021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3390%2Fjcm7020021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=21&author=A.+Guarnacciaauthor=W.+Tansey&title=Moonlighting+with+WDR5%3A+A+Cellular+Multitasker&doi=10.3390%2Fjcm7020021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Moonlighting with WDR5: a cellular multitasker</span></div><div class="casAuthors">Guarnaccia, Alissa Dupuy; Tansey, William Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">21/1-21/17</span>CODEN:
                <span class="NLM_cas:coden">JCMOHK</span>;
        ISSN:<span class="NLM_cas:issn">2077-0383</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">WDR5 is a highly conserved WD40 repeat-contg. protein that is essential for proper regulation of multiple cellular processes.  WDR5 is best characterized as a core scaffolding component of histone methyltransferase complexes, but emerging evidence demonstrates that it does much more, ranging from expanded functions in the nucleus through to controlling the integrity of cell division.  The purpose of this review is to describe the current mol. understandings of WDR5, discuss how it participates in diverse cellular processes, and highlight drug discovery efforts around WDR5 that may form the basis of new anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-aPToDu3vgbVg90H21EOLACvtfcHk0ljyNxr-gvGYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGjt7w%253D&md5=fba93df831fa1fd74d7f57d0205bd2d8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fjcm7020021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fjcm7020021%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnaccia%26aufirst%3DA.%26aulast%3DTansey%26aufirst%3DW.%26atitle%3DMoonlighting%2520with%2520WDR5%253A%2520A%2520Cellular%2520Multitasker%26jtitle%3DJ.%2520Clin.%2520Med.%26date%3D2018%26volume%3D7%26spage%3D21%26doi%3D10.3390%2Fjcm7020021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">A Conserved Arginine-Containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32162</span>– <span class="NLM_lpage">32175</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+Conserved+Arginine-Containing+Motif+Crucial+for+the+Assembly+and+Enzymatic+Activity+of+the+Mixed+Lineage+Leukemia+Protein-1+Core+Complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0ljyNxr-gvGYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520Conserved%2520Arginine-Containing%2520Motif%2520Crucial%2520for%2520the%2520Assembly%2520and%2520Enzymatic%2520Activity%2520of%2520the%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">32158</span>– <span class="NLM_lpage">32161</span>, <span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+Bound+to+Mixed+Lineage+Leukemia+Protein-1+Peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lggIUXlDW3dAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520Bound%2520to%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520Peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">WDR5 Interacts with Mixed Lineage Leukemia (MLL) Protein via the Histone H3-Binding Pocket</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">35258</span>– <span class="NLM_lpage">35264</span>, <span class="refDoi"> DOI: 10.1074/jbc.M806900200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M806900200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=18840606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGqt73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=35258-35264&author=J.-J.+Songauthor=R.+E.+Kingston&title=WDR5+Interacts+with+Mixed+Lineage+Leukemia+%28MLL%29+Protein+via+the+Histone+H3-Binding+Pocket&doi=10.1074%2Fjbc.M806900200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Interacts with Mixed Lineage Leukemia (MLL) Protein via the Histone H3-binding Pocket</span></div><div class="casAuthors">Song, Ji-Joon; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">35258-35264</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">WDR5 is a component of the mixed lineage leukemia (MLL) complex, which methylates lysine 4 of histone H3, and was identified as a methylated Lys-4 histone H3-binding protein.  Here, we present a crystal structure of WDR5 bound to an MLL peptide.  Surprisingly, we find that WDR5 utilizes the same pocket shown to bind histone H3 for this MLL interaction.  Furthermore, the WDR5-MLL interaction is disrupted preferentially by mono- and di-methylated Lys-4 histone H3 over unmodified and tri-methylated Lys-4 histone H3.  These data implicate a delicate interplay between the effector, WDR5, the catalytic subunit, MLL, and the substrate, histone H3, of the MLL complex.  We suggest that the activity of the MLL complex might be regulated through this interplay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKATFDTs7dELVg90H21EOLACvtfcHk0lggIUXlDW3dAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGqt73J&md5=3207cca861ba0e6491a888b8586494ae</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806900200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806900200%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.-J.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DWDR5%2520Interacts%2520with%2520Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%2520Protein%2520via%2520the%2520Histone%2520H3-Binding%2520Pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D35258%26epage%3D35264%26doi%3D10.1074%2Fjbc.M806900200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odho, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Requirement of Additional Factors for MLL1 SET Domain Activity and Recognition of Epigenetic Marks</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2008.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=19187761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=181-191&author=S.+M.+Southallauthor=P.+S.+Wongauthor=Z.+Odhoauthor=S.+M.+Roeauthor=J.+R.+Wilson&title=Structural+Basis+for+the+Requirement+of+Additional+Factors+for+MLL1+SET+Domain+Activity+and+Recognition+of+Epigenetic+Marks&doi=10.1016%2Fj.molcel.2008.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the requirement of additional factors for MLL1 SET domain activity and recognition of epigenetic marks</span></div><div class="casAuthors">Southall, Stacey M.; Wong, Poon-Sheng; Odho, Zain; Roe, S. Mark; Wilson, Jon R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mixed-lineage leukemia protein MLL1 is a transcriptional regulator with an essential role in early development and hematopoiesis.  The biol. function of MLL1 is mediated by the histone H3K4 methyltransferase activity of the carboxyl-terminal SET domain.  We have detd. the crystal structure of the MLL1 SET domain in complex with cofactor product AdoHcy and a histone H3 peptide.  This structure indicates that, in order to form a well-ordered active site, a highly variable but essential component of the SET domain must be repositioned.  To test this idea, we compared the effect of the addn. of MLL complex members on methyltransferase activity and show that both RbBP5 and Ash2L but not Wdr5 stimulate activity.  Addnl., we have detd. the effect of posttranslational modifications on histone H3 residues downstream and upstream from the target lysine and provide a structural explanation for why H3T3 phosphorylation and H3K9 acetylation regulate activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TVBdtfM5BLVg90H21EOLACvtfcHk0lggIUXlDW3dAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVylsbg%253D&md5=b16a9354ef72ad0138c891c7fe698fcc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DWong%26aufirst%3DP.%2BS.%26aulast%3DOdho%26aufirst%3DZ.%26aulast%3DRoe%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Requirement%2520of%2520Additional%2520Factors%2520for%2520MLL1%2520SET%2520Domain%2520Activity%2520and%2520Recognition%2520of%2520Epigenetic%2520Marks%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D181%26epage%3D191%26doi%3D10.1016%2Fj.molcel.2008.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odho, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of a Novel WDR5-Binding Site That Recruits RbBP5 through a Conserved Motif to Enhance Methylation of Histone H3 Lysine 4 by Mixed Lineage Leukemia Protein-1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">32967</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.159921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M110.159921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20716525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=32967-32976&author=Z.+Odhoauthor=S.+M.+Southallauthor=J.+R.+Wilson&title=Characterization+of+a+Novel+WDR5-Binding+Site+That+Recruits+RbBP5+through+a+Conserved+Motif+to+Enhance+Methylation+of+Histone+H3+Lysine+4+by+Mixed+Lineage+Leukemia+Protein-1&doi=10.1074%2Fjbc.M110.159921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel WDR5-binding site that recruits RbBP5 through a conserved motif to enhance methylation of histone H3 lysine 4 by mixed lineage leukemia protein-1</span></div><div class="casAuthors">Odho, Zain; Southall, Stacey M.; Wilson, Jon R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">32967-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone modification is well established as a fundamental mechanism driving the regulation of transcription, replication, and DNA repair through the control of chromatin structure.  Likewise, it is apparent that incorrect targeting of histone modifications contributes to misregulated gene expression and hence to developmental disorders and diseases of genomic instability such as cancer.  The KMT2 family of SET domain methyltransferases, typified by mixed lineage leukemia protein-1 (MLL1), is responsible for histone H3 lysine 4 methylation, a marker of active genes.  To ensure that this modification is correctly targeted, a multiprotein complex assocs. with the methyltransferase and directs activity.  We have identified a novel interaction site on the core complex protein WD repeat protein-5 (WDR5), and we mapped the complementary site on its partner retinoblastoma-binding protein-5 (RbBP5).  We have characterized this interaction by x-ray crystallog. and show how it is fundamental to the assembly of the complex and to the regulation of methyltransferase activity.  We show which region of RbBP5 contributes directly to mixed lineage leukemia activation, and we combine our structural and biochem. data to produce a model to show how WDR5 and RbBP5 act cooperatively to stimulate activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSuXfVNduMSrVg90H21EOLACvtfcHk0li7C2nKhNhbdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12nu73E&md5=fd5c6418120415b74f200381097e4c1e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.159921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.159921%26sid%3Dliteratum%253Aachs%26aulast%3DOdho%26aufirst%3DZ.%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26atitle%3DCharacterization%2520of%2520a%2520Novel%2520WDR5-Binding%2520Site%2520That%2520Recruits%2520RbBP5%2520through%2520a%2520Conserved%2520Motif%2520to%2520Enhance%2520Methylation%2520of%2520Histone%2520H3%2520Lysine%25204%2520by%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D32967%26epage%3D32976%26doi%3D10.1074%2Fjbc.M110.159921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dias, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brettschneider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadlec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, A.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of the KANSL1/WDR5/ KANSL2 Complex Reveals That WDR5 Is Required for Efficient Assembly and Chromatin Targeting of the NSL Complex</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">929</span>– <span class="NLM_lpage">942</span>, <span class="refDoi"> DOI: 10.1101/gad.240200.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1101%2Fgad.240200.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24788516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot12is7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=929-942&author=J.+Diasauthor=N.+Van+Nguyenauthor=P.+Georgievauthor=A.+Gaubauthor=J.+Brettschneiderauthor=S.+Cusackauthor=J.+Kadlecauthor=A.+Akhtar&title=Structural+Analysis+of+the+KANSL1%2FWDR5%2F+KANSL2+Complex+Reveals+That+WDR5+Is+Required+for+Efficient+Assembly+and+Chromatin+Targeting+of+the+NSL+Complex&doi=10.1101%2Fgad.240200.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of the KANSL1/WDR5/KANSL2 complex reveals that WDR5 is required for efficient assembly and chromatin targeting of the NSL complex</span></div><div class="casAuthors">Dias, Jorge; Nguyen, Nhuong Van; Georgiev, Plamen; Gaub, Aline; Brettschneider, Janine; Cusack, Stephen; Kadlec, Jan; Akhtar, Asifa</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">929-942</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The subunits of the nonspecific lethal (NSL) complex, which include the histone acetyltransferase MOF (males absent on the first), play important roles in various cellular functions, including transcription regulation and stem cell identity maintenance and reprogramming, and are frequently misregulated in disease.  Here, the authors provide the first biochem. and structural insights into the mol. architecture of this large multiprotein assembly.  The authors identified several direct interactions within the complex and show that KANSL1 acts as a scaffold protein interacting with four other subunits, including WDR5, which in turn binds KANSL2.  Structural anal. of the KANSL1/WDR5/KANSL2 subcomplex reveals how WDR5 is recruited into the NSL complex via conserved linear motifs of KANSL1 and KANSL2.  Using structure-based KANSL1 mutants in transgenic flies, the authors show that the KANSL1-WDR5 interaction is required for proper assembly, efficient recruitment of the NSL complex to target promoters, and fly viability.  The authors' data clearly show that the interactions of WDR5 with the MOF-contg. NSL complex and MLL/COMPASS histone methyltransferase complexes are mutually exclusive.  The authors propose that rather than being a shared subunit, WDR5 plays an important role in assembling distinct histone-modifying complexes with different epigenetic regulatory roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7W3dLwq_MvrVg90H21EOLACvtfcHk0li7C2nKhNhbdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot12is7s%253D&md5=6719d3b97de11fd9fc5f147d2f2a6021</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2Fgad.240200.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.240200.114%26sid%3Dliteratum%253Aachs%26aulast%3DDias%26aufirst%3DJ.%26aulast%3DVan%2BNguyen%26aufirst%3DN.%26aulast%3DGeorgiev%26aufirst%3DP.%26aulast%3DGaub%26aufirst%3DA.%26aulast%3DBrettschneider%26aufirst%3DJ.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DKadlec%26aufirst%3DJ.%26aulast%3DAkhtar%26aufirst%3DA.%26atitle%3DStructural%2520Analysis%2520of%2520the%2520KANSL1%252FWDR5%252F%2520KANSL2%2520Complex%2520Reveals%2520That%2520WDR5%2520Is%2520Required%2520for%2520Efficient%2520Assembly%2520and%2520Chromatin%2520Targeting%2520of%2520the%2520NSL%2520Complex%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D929%26epage%3D942%26doi%3D10.1101%2Fgad.240200.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanouette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groulx, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span> <span> </span><span class="NLM_article-title">Structural and Biochemical Insights into MLL1 Core Complex Assembly</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/j.str.2010.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.str.2010.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21220120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFCktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=101-108&author=V.+Avdicauthor=P.+Zhangauthor=S.+Lanouetteauthor=A.+Groulxauthor=V.+Tremblayauthor=J.+Brunzelleauthor=J.+F.+Couture&title=Structural+and+Biochemical+Insights+into+MLL1+Core+Complex+Assembly&doi=10.1016%2Fj.str.2010.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical insights into MLL1 core complex assembly</span></div><div class="casAuthors">Avdic, Vanja; Zhang, Pamela; Lanouette, Sylvain; Groulx, Adam; Tremblay, Veronique; Brunzelle, Joseph; Couture, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone H3 Lys-4 methylation is predominantly catalyzed by a family of methyltransferases whose enzymic activity depends on their interaction with a three-subunit complex composed of WDR5, RbBP5, and Ash2L.  Here, we report that a segment of 50 residues of RbBP5 bridges the Ash2L C-terminal domain to WDR5.  The crystal structure of WDR5 in ternary complex with RbBP5 and MLL1 reveals that both proteins binds peptide-binding clefts located on opposite sides of WDR5's β-propeller domain.  RbBP5 engages in several hydrogen bonds and van der Waals contacts within a V-shaped cleft formed by the junction of two blades on WDR5.  Mutational analyses of both the WDR5 V-shaped cleft and RbBP5 residues reveal that the interactions between RbBP5 and WDR5 are important for the stimulation of MLL1 methyltransferase activity.  Overall, this study provides the structural basis underlying the formation of the WDR5-RbBP5 subcomplex and further highlight the crucial role of WDR5 in scaffolding the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWM9chG9JoC7Vg90H21EOLACvtfcHk0li7C2nKhNhbdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFCktg%253D%253D&md5=e0539eaf4179e60d808b4cda8454f187</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2010.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2010.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DAvdic%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLanouette%26aufirst%3DS.%26aulast%3DGroulx%26aufirst%3DA.%26aulast%3DTremblay%26aufirst%3DV.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26atitle%3DStructural%2520and%2520Biochemical%2520Insights%2520into%2520MLL1%2520Core%2520Complex%2520Assembly%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D101%26epage%3D108%26doi%3D10.1016%2Fj.str.2010.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Activity Regulation of MLL Family Methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>530</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1038/nature16952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature16952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26886794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOqur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2016&pages=447-452&author=Y.+Liauthor=J.+Hanauthor=Y.+Zhangauthor=F.+Caoauthor=Z.+Liuauthor=S.+Liauthor=J.+Wuauthor=C.+Huauthor=Y.+Wangauthor=J.+Shuaiauthor=J.+Chenauthor=L.+Caoauthor=D.+Liauthor=P.+Shiauthor=C.+Tianauthor=J.+Zhangauthor=Y.+Douauthor=G.+Liauthor=Y.+Chenauthor=M.+Lei&title=Structural+Basis+for+Activity+Regulation+of+MLL+Family+Methyltransferases&doi=10.1038%2Fnature16952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for activity regulation of MLL family methyltransferases</span></div><div class="casAuthors">Li, Yanjing; Han, Jianming; Zhang, Yuebin; Cao, Fang; Liu, Zhijun; Li, Shuai; Wu, Jian; Hu, Chunyi; Wang, Yan; Shuai, Jin; Chen, Juan; Cao, Liaoran; Li, Dangsheng; Shi, Pan; Tian, Changlin; Zhang, Jian; Dou, Yali; Li, Guohui; Chen, Yong; Lei, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">7591</span>),
    <span class="NLM_cas:pages">447-452</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mixed lineage leukemia (MLL) family of proteins (including MLL1-MLL4, SET1A and SET1B) specifically methylate histone 3 Lys4, and have pivotal roles in the transcriptional regulation of genes involved in haematopoiesis and development.  The methyltransferase activity of MLL1, by itself severely compromised, is stimulated by the three conserved factors WDR5, RBBP5 and ASH2L, which are shared by all MLL family complexes.  However, the mol. mechanism of how these factors regulate the activity of MLL proteins still remains poorly understood.  Here we show that a minimized human RBBP5-ASH2L heterodimer is the structural unit that interacts with and activates all MLL family histone methyltransferases.  Our structural, biochem. and computational analyses reveal a two-step activation mechanism of MLL family proteins.  These findings provide unprecedented insights into the common theme and functional plasticity in complex assembly and activity regulation of MLL family methyltransferases, and also suggest a universal regulation mechanism for most histone methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwlhi89SDUJbVg90H21EOLACvtfcHk0li7C2nKhNhbdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOqur4%253D&md5=77fb7f13196f990a0c8918a59652216b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature16952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16952%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShuai%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DTian%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Activity%2520Regulation%2520of%2520MLL%2520Family%2520Methyltransferases%26jtitle%3DNature%26date%3D2016%26volume%3D530%26spage%3D447%26epage%3D452%26doi%3D10.1038%2Fnature16952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">High-Affinity, Small-Molecule Peptidomimetic Inhibitors of MLL1/WDR5 Protein–Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOlsbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-Affinity%2C+Small-Molecule+Peptidomimetic+Inhibitors+of+MLL1%2FWDR5+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity, Small-Molecule Peptidomimetic Inhibitors of MLL1/WDR5 Protein-Protein Interaction</span></div><div class="casAuthors">Karatas, Hacer; Townsend, Elizabeth C.; Cao, Fang; Chen, Yong; Bernard, Denzil; Liu, Liu; Lei, Ming; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">669-682</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mixed lineage leukemia 1 (MLL1) is a histone H3 lysine 4 (H3K4) methyltransferase, and targeting the MLL1 enzymic activity has been proposed as a novel therapeutic strategy for the treatment of acute leukemia harboring MLL1 fusion proteins.  The MLL1/WDR5 protein-protein interaction is essential for MLL1 enzymic activity.  In the present study, we designed a large no. of peptidomimetics to target the MLL1/WDR5 interaction based upon -CO-ARA-NH-, the min. binding motif derived from MLL1.  Our study led to the design of high-affinity peptidomimetics, which bind to WDR5 with Ki < 1 nM and function as potent antagonists of MLL1 activity in a fully reconstituted in vitro H3K4 methyltransferase assay.  Detn. of co-crystal structures of two potent peptidomimetics in complex with WDR5 establishes their structural basis for high-affinity binding to WDR5.  Evaluation of one such peptidomimetic, MM-102, in bone marrow cells transduced with MLL1-AF9 fusion construct shows that the compd. effectively decreases the expression of HoxA9 and Meis-1, two crit. MLL1 target genes in MLL1 fusion protein mediated leukemogenesis.  MM-102 also specifically inhibits cell growth and induces apoptosis in leukemia cells harboring MLL1 fusion proteins.  Our study provides the first proof-of-concept for the design of small-mol. inhibitors of the WDR5/MLL1 protein-protein interaction as a novel therapeutic approach for acute leukemia harboring MLL1 fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpkwxpAigvkbVg90H21EOLACvtfcHk0lgIwTBHGxvSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOlsbvO&md5=29f9e4a57635bbb71f6ea65f82a3405c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-Affinity%252C%2520Small-Molecule%2520Peptidomimetic%2520Inhibitors%2520of%2520MLL1%252FWDR5%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4818</span>– <span class="NLM_lpage">4839</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1CjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4818-4839&author=H.+Karatasauthor=Y.+Liauthor=L.+Liuauthor=J.+Jiauthor=S.+Leeauthor=Y.+Chenauthor=J.+Yangauthor=L.+Huangauthor=D.+Bernardauthor=J.+Xuauthor=E.+C.+Townsendauthor=F.+Caoauthor=X.+Ranauthor=X.+Liauthor=B.+Wenauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=Discovery+of+a+Highly+Potent%2C+Cell-Permeable+Macrocyclic+Peptidomimetic+%28MM-589%29+Targeting+the+WD+Repeat+Domain+5+Protein+%28WDR5%29%E2%80%93Mixed+Lineage+Leukemia+%28MLL%29+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Facs.jmedchem.6b01796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)-Mixed Lineage Leukemia (MLL) Protein-Protein Interaction</span></div><div class="casAuthors">Karatas, Hacer; Li, Yangbing; Liu, Liu; Ji, Jiao; Lee, Shirley; Chen, Yong; Yang, Jiuling; Huang, Liyue; Bernard, Denzil; Xu, Jing; Townsend, Elizabeth C.; Cao, Fang; Ran, Xu; Li, Xiaoqin; Wen, Bo; Sun, Duxin; Stuckey, Jeanne A.; Lei, Ming; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4818-4839</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the design, synthesis and evaluation of macrocyclic peptidomimetics that bind to WDR5 and block the WDR5-MLL protein-protein interaction.  Compd. (I) (R1 = Me; n = 1) (MM-589) binds to WDR5 with an IC50 value of 0.90 nM (Ki value <1 nM), and inhibits the MLL H3K4 methyltransferase (HMT) activity with an IC50 value of 12.7 nM.  Compd. I (R1 = Me; n = 1) potently and selectively inhibits cell growth in human leukemia cell lines harboring MLL translocations, and is >40 times better than the previously reported compd. MM-401.  Co-crystal structures of I (R1 = H, n = 5) and (R1 = Me; n = 1) complexed with WDR5 provide structural basis for their high affinity binding to WDR5.  Addnl., we have developed and optimized a new, AlphaLISA-based MLL HMT functional assay to facilitate the functional evaluation of these designed compds.  Compd. I (R1 = Me; n = 1) represents the most potent inhibitor of the WDR5-MLL interaction reported to date and further optimization of I (R1 = Me; n = 1) may yield a new therapy for acute leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrugN-nacR_FLVg90H21EOLACvtfcHk0lgIwTBHGxvSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1CjsL8%253D&md5=598654db06fde01433ef5158aa94a946</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01796%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Highly%2520Potent%252C%2520Cell-Permeable%2520Macrocyclic%2520Peptidomimetic%2520%2528MM-589%2529%2520Targeting%2520the%2520WD%2520Repeat%2520Domain%25205%2520Protein%2520%2528WDR5%2529%25E2%2580%2593Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4818%26epage%3D4839%26doi%3D10.1021%2Facs.jmedchem.6b01796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouillette, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-261&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+Methyltransferase+Activity+in+Mixed-Lineage+Leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0ljvib0iKimMyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520Methyltransferase%2520Activity%2520in%2520Mixed-Lineage%2520Leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D261%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giambruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grover, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avellino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of the WDR5-MLL Interaction in C/EBPα N-Terminal Leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+Targeting+of+the+WDR5-MLL+Interaction+in+C%2FEBP%CE%B1+N-Terminal+Leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0ljvib0iKimMyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520Targeting%2520of%2520the%2520WDR5-MLL%2520Interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-Terminal%2520Leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velázquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolshan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-awar, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-2496&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Vel%C3%A1zquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-awar&title=Structure-Based+Optimization+of+a+Small+Molecule+Antagonist+of+the+Interaction+Between+WD+Repeat-Containing+Protein+5+%28WDR5%29+and+Mixed-Lineage+Leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0ljSolwADcEPag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Vel%25C3%25A1zquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-awar%26aufirst%3DR.%26atitle%3DStructure-Based%2520Optimization%2520of%2520a%2520Small%2520Molecule%2520Antagonist%2520of%2520the%2520Interaction%2520Between%2520WD%2520Repeat-Containing%2520Protein%25205%2520%2528WDR5%2529%2520and%2520Mixed-Lineage%2520Leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D2496%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Small Molecular Blockers of Mixed Lineage Leukemia 1 (MLL1)-WDR5 Interaction Inhibit MLL1 Complex H3K4 Methyltransferase Activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.-D.+Liauthor=W.-L.+Chenauthor=Z.-H.+Wangauthor=Y.-Y.+Xieauthor=X.-L.+Xuauthor=Z.-Y.+Jiangauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=X.-K.+Guo&title=High-Affinity+Small+Molecular+Blockers+of+Mixed+Lineage+Leukemia+1+%28MLL1%29-WDR5+Interaction+Inhibit+MLL1+Complex+H3K4+Methyltransferase+Activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0ljSolwADcEPag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.-D.%26aulast%3DChen%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DZ.-H.%26aulast%3DXie%26aufirst%3DY.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DGuo%26aufirst%3DX.-K.%26atitle%3DHigh-Affinity%2520Small%2520Molecular%2520Blockers%2520of%2520Mixed%2520Lineage%2520Leukemia%25201%2520%2528MLL1%2529-WDR5%2520Interaction%2520Inhibit%2520MLL1%2520Complex%2520H3K4%2520Methyltransferase%2520Activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuh, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent 2-Aryl-6,7-Dihydro-5H-Pyrrolo[1,2-a]Imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5623</span>– <span class="NLM_lpage">5642</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00375</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00375" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5623-5642&author=F.+Wangauthor=K.+O.+Jeonauthor=J.+M.+Salovichauthor=J.+D.+Macdonaldauthor=J.+Alvaradoauthor=R.+D.+Gogliottiauthor=J.+Phanauthor=E.+T.+Olejniczakauthor=Q.+Sunauthor=S.+Wangauthor=D.+Camperauthor=J.+P.+Yuhauthor=J.+G.+Shawauthor=J.+Saiauthor=O.+W.+Rossaneseauthor=W.+P.+Tanseyauthor=S.+R.+Staufferauthor=S.+W.+Fesik&title=Discovery+of+Potent+2-Aryl-6%2C7-Dihydro-5H-Pyrrolo%5B1%2C2-a%5DImidazoles+as+WDR5-WIN-Site+Inhibitors+Using+Fragment-Based+Methods+and+Structure-Based+Design&doi=10.1021%2Facs.jmedchem.8b00375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Wang, Feng; Jeon, Kyu Ok; Salovich, James M.; Macdonald, Jonathan D.; Alvarado, Joseph; Gogliotti, Rocco D.; Phan, Jason; Olejniczak, Edward T.; Sun, Qi; Wang, Shidong; Camper, DeMarco; Yuh, Joannes P.; Shaw, J. Grace; Sai, Jiqing; Rossanese, Olivia W.; Tansey, William P.; Stauffer, Shaun R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5623-5642</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted pyrroloimidazoles such as I [R = 3,4-Cl2C6H3CH2, 3-MeO-4-MeC6H3, 2-Me-4-pyridinyl, 3,5-(MeO)2-4-MeC6H2] were prepd. as inhibitors of the chromatin-regulatory scaffold protein WDR5 WDR5-interaction (WIN) site as lead compds. for development of antitumor agents.  NMR-based screening of a large fragment library identified several chem. distinct hit series that bind to the WIN site within WDR5.  Using a structure-based design approach, pyrroloimidazoles were found with dissocn. consts. < 10 nM for WDR5 and micromolar cellular activity against an AML-leukemia cell line.  The structures of I (R = 3-MeO-4-MeC6H3) and of four other compds. bound to a fragment of WDR5 were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSZaiiuG6OrVg90H21EOLACvtfcHk0lh_uBhvS84z-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrsLjN&md5=2322c6458b2a060005bf5250a46f5cbd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJeon%26aufirst%3DK.%2BO.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DYuh%26aufirst%3DJ.%2BP.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Potent%25202-Aryl-6%252C7-Dihydro-5H-Pyrrolo%255B1%252C2-a%255DImidazoles%2520as%2520WDR5-WIN-Site%2520Inhibitors%2520Using%2520Fragment-Based%2520Methods%2520and%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5623%26epage%3D5642%26doi%3D10.1021%2Facs.jmedchem.8b00375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aho, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foshage, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissmiller, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heibert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tansey, W. P.</span></span> <span> </span><span class="NLM_article-title">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2916</span>– <span class="NLM_lpage">2928.e13</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2019.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.celrep.2019.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=30865883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=2916-2928.e13&author=E.+R.+Ahoauthor=J.+Wangauthor=R.+D.+Gogliottiauthor=G.+C.+Howardauthor=J.+Phanauthor=P.+Acharyaauthor=J.+D.+Macdonaldauthor=K.+Chengauthor=S.+L.+Loreyauthor=B.+Luauthor=S.+Wenzelauthor=A.+M.+Foshageauthor=J.+Alvaradoauthor=F.+Wangauthor=J.+G.+Shawauthor=B.+Zhaoauthor=A.+M.+Weissmillerauthor=L.+R.+Thomasauthor=C.+R.+Vakocauthor=M.+D.+Hallauthor=S.+W.+Heibertauthor=Q.+Liuauthor=S.+R.+Staufferauthor=S.+W.+Fesikauthor=W.+P.+Tansey&title=Displacement+of+WDR5+from+Chromatin+by+a+WIN+Site+Inhibitor+with+Picomolar+Affinity&doi=10.1016%2Fj.celrep.2019.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity</span></div><div class="casAuthors">Aho, Erin R.; Wang, Jing; Gogliotti, Rocco D.; Howard, Gregory C.; Phan, Jason; Acharya, Pankaj; Macdonald, Jonathan D.; Cheng, Ken; Lorey, Shelly L.; Lu, Bin; Wenzel, Sabine; Foshage, Audra M.; Alvarado, Joseph; Wang, Feng; Shaw, J. Grace; Zhao, Bin; Weissmiller, April M.; Thomas, Lance R.; Vakoc, Christopher R.; Hall, Matthew D.; Hiebert, Scott W.; Liu, Qi; Stauffer, Shaun R.; Fesik, Stephen W.; Tansey, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2916-2928.e13</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The chromatin-assocd. protein WDR5 is a promising target for pharmacol. inhibition in cancer.  Drug discovery efforts center on the blockade of the "WIN site" of WDR5, a well-defined pocket that is amenable to small mol. inhibition.  Various cancer contexts have been proposed to be targets for WIN site inhibitors, but a lack of understanding of WDR5 target genes and of the primary effects of WIN site inhibitors hampers their utility.  Here, by the discovery of potent WIN site inhibitors, we demonstrate that the WIN site links WDR5 to chromatin at a small cohort of loci, including a specific subset of ribosome protein genes.  WIN site inhibitors rapidly displace WDR5 from chromatin and decrease the expression of assocd. genes, causing translational inhibition, nucleolar stress, and p53 induction.  Our studies define a mode by which WDR5 engages chromatin and forecast that WIN site blockade could have utility against multiple cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprdgAaHB9WfLVg90H21EOLACvtfcHk0lh_uBhvS84z-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVelu70%253D&md5=56e64b66f0d04ff5c36e104e54594bbd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2019.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2019.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DAho%26aufirst%3DE.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DHoward%26aufirst%3DG.%2BC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DAcharya%26aufirst%3DP.%26aulast%3DMacdonald%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DLorey%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DWenzel%26aufirst%3DS.%26aulast%3DFoshage%26aufirst%3DA.%2BM.%26aulast%3DAlvarado%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DShaw%26aufirst%3DJ.%2BG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWeissmiller%26aufirst%3DA.%2BM.%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DHeibert%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DTansey%26aufirst%3DW.%2BP.%26atitle%3DDisplacement%2520of%2520WDR5%2520from%2520Chromatin%2520by%2520a%2520WIN%2520Site%2520Inhibitor%2520with%2520Picomolar%2520Affinity%26jtitle%3DCell%2520Rep.%26date%3D2019%26volume%3D26%26spage%3D2916%26epage%3D2928.e13%26doi%3D10.1016%2Fj.celrep.2019.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">WD40 Repeat Domain Proteins: A Novel Target Class?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnrd.2017.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=29026209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cmt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=773-786&author=M.+Schapiraauthor=M.+Tyersauthor=M.+Torrentauthor=C.+H.+Arrowsmith&title=WD40+Repeat+Domain+Proteins%3A+A+Novel+Target+Class%3F&doi=10.1038%2Fnrd.2017.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">WD40 repeat domain proteins: a novel target class?</span></div><div class="casAuthors">Schapira, Matthieu; Tyers, Mike; Torrent, Maricel; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">773-786</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Antagonism of protein-protein interactions (PPIs) with small mols. is becoming more feasible as a therapeutic approach.  Successful PPI inhibitors tend to target proteins contg. deep peptide-binding grooves or pockets rather than the more common large, flat protein interaction surfaces.  Here, we review one of the most abundant PPI domains in the human proteome, the WD40 repeat (WDR) domain, which has a central peptide-binding pocket and is a member of the β-propeller domain-contg. protein family.  Recently, two WDR domain-contg. proteins, WDR5 and EED, as well as other β-propeller domains have been successfully targeted by potent, specific, cell-active, drug-like chem. probes.  Could WDR domains be a novel target class for drug discovery.  Although the research is at an early stage and therefore not clin. validated, cautious optimism is justified, as WDR domain-contg. proteins are involved in multiple disease-assocd. pathways.  The druggability and structural diversity of WDR domain binding pockets suggest that understanding how to target this prevalent domain class will open up areas of disease biol. that have so far resisted drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMccOym4Ml7Vg90H21EOLACvtfcHk0lhAWanx7q4gSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cmt7rM&md5=6f99bbb97e411c9b405a981d7791a8be</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.179%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DWD40%2520Repeat%2520Domain%2520Proteins%253A%2520A%2520Novel%2520Target%2520Class%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D773%26epage%3D786%26doi%3D10.1038%2Fnrd.2017.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orlicky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neduva, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elowe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span> <span> </span><span class="NLM_article-title">An Allosteric Inhibitor of Substrate Recognition by the SCFCdc4 Ubiquitin Ligase</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nbt.1646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnbt.1646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=20581844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=733-737&author=S.+Orlickyauthor=X.+Tangauthor=V.+Neduvaauthor=N.+Eloweauthor=E.+D.+Brownauthor=F.+Sicheriauthor=M.+Tyers&title=An+Allosteric+Inhibitor+of+Substrate+Recognition+by+the+SCFCdc4+Ubiquitin+Ligase&doi=10.1038%2Fnbt.1646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric inhibitor of substrate recognition by the SCFCdc4 ubiquitin ligase</span></div><div class="casAuthors">Orlicky, Stephen; Tang, Xiaojing; Neduva, Victor; Elowe, Nadine; Brown, Eric D.; Sicheri, Frank; Tyers, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specificity of SCF ubiquitin ligase-mediated protein degrdn. is detd. by F-box proteins.  We identified a biplanar dicarboxylic acid compd., called SCF-I2, as an inhibitor of substrate recognition by the yeast F-box protein Cdc4 using a fluorescence polarization screen to monitor the displacement of a fluorescein-labeled phosphodegron peptide.  SCF-I2 inhibits the binding and ubiquitination of full-length phosphorylated substrates by SCFCdc4.  A co-crystal structure reveals that SCF-I2 inserts itself between the β-strands of blades 5 and 6 of the WD40 propeller domain of Cdc4 at a site that is 25 Å away from the substrate binding site.  Long-range transmission of SCF-I2 interactions distorts the substrate binding pocket and impedes recognition of key determinants in the Cdc4 phosphodegron.  Mutation of the SCF-I2 binding site abrogates its inhibitory effect and explains specificity in the allosteric inhibition mechanism.  Mammalian WD40 domain proteins may exhibit similar allosteric responsiveness and hence represent an extensive class of druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptnWKaMi1kdLVg90H21EOLACvtfcHk0lhAWanx7q4gSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D&md5=23197bd195811ad8503efb5d43457ccf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1646%26sid%3Dliteratum%253Aachs%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DNeduva%26aufirst%3DV.%26aulast%3DElowe%26aufirst%3DN.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTyers%26aufirst%3DM.%26atitle%3DAn%2520Allosteric%2520Inhibitor%2520of%2520Substrate%2520Recognition%2520by%2520the%2520SCFCdc4%2520Ubiquitin%2520Ligase%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnbt.1646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sackton, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sackton, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meaders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaff, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigoillot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span> <span> </span><span class="NLM_article-title">Synergistic Blockade of Mitotic Exit by Two Chemical Inhibitors of the APC/C</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/nature13660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1038%2Fnature13660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=25156254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCnu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=646-649&author=K.+L.+Sacktonauthor=N.+Dimovaauthor=X.+Zengauthor=W.+Tianauthor=M.+Zhangauthor=T.+B.+Sacktonauthor=J.+Meadersauthor=K.+L.+Pfaffauthor=F.+Sigoillotauthor=H.+Yuauthor=X.+Luoauthor=R.+W.+King&title=Synergistic+Blockade+of+Mitotic+Exit+by+Two+Chemical+Inhibitors+of+the+APC%2FC&doi=10.1038%2Fnature13660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic blockade of mitotic exit by two chemical inhibitors of the APC/C</span></div><div class="casAuthors">Sackton, Katharine L.; Dimova, Nevena; Zeng, Xing; Tian, Wei; Zhang, Mengmeng; Sackton, Timothy B.; Meaders, Johnathan; Pfaff, Kathleen L.; Sigoillot, Frederic; Yu, Hongtao; Luo, Xuelian; King, Randall W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">514</span>
        (<span class="NLM_cas:issue">7524</span>),
    <span class="NLM_cas:pages">646-649</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein machines are multi-subunit protein complexes that orchestrate highly regulated biochem. tasks.  An example is the anaphase-promoting complex/cyclosome (APC/C), a 13-subunit ubiquitin ligase that initiates the metaphase-anaphase transition and mitotic exit by targeting proteins such as securin and cyclin B1 for ubiquitin-dependent destruction by the proteasome.  Because blocking mitotic exit is an effective approach for inducing tumor cell death, the APC/C represents a potential novel target for cancer therapy.  APC/C activation in mitosis requires binding of Cdc20 (ref. 5), which forms a co-receptor with the APC/C to recognize substrates contg. a destruction box (D-box).  Here we demonstrate that we can synergistically inhibit APC/C-dependent proteolysis and mitotic exit by simultaneously disrupting two protein-protein interactions within the APC/C-Cdc20-substrate ternary complex.  We identify a small mol., called apcin (APC inhibitor), which binds to Cdc20 and competitively inhibits the ubiquitylation of D-box-contg. substrates.  Anal. of the crystal structure of the apcin-Cdc20 complex suggests that apcin occupies the D-box-binding pocket on the side face of the WD40-domain.  The ability of apcin to block mitotic exit is synergistically amplified by co-addn. of tosyl-L-arginine Me ester, a small mol. that blocks the APC/C-Cdc20 interaction.  This work suggests that simultaneous disruption of multiple, weak protein-protein interactions is an effective approach for inactivating a protein machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_wm8ws0xnLVg90H21EOLACvtfcHk0lgec2cXCMbqZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCnu7nL&md5=f11be7f3403b7fa4144d167af925a442</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature13660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13660%26sid%3Dliteratum%253Aachs%26aulast%3DSackton%26aufirst%3DK.%2BL.%26aulast%3DDimova%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DSackton%26aufirst%3DT.%2BB.%26aulast%3DMeaders%26aufirst%3DJ.%26aulast%3DPfaff%26aufirst%3DK.%2BL.%26aulast%3DSigoillot%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DKing%26aufirst%3DR.%2BW.%26atitle%3DSynergistic%2520Blockade%2520of%2520Mitotic%2520Exit%2520by%2520Two%2520Chemical%2520Inhibitors%2520of%2520the%2520APC%252FC%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3D646%26epage%3D649%26doi%3D10.1038%2Fnature13660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altomonte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanda, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic Chemistry and Biological Activity of Pentafluorosulphanyl (SF<sub>5</sub>) Organic Molecules</span>. <i>J. Fluor. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.jfluchem.2012.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2Fj.jfluchem.2012.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2012&pages=57-93&author=S.+Altomonteauthor=M.+Zanda&title=Synthetic+Chemistry+and+Biological+Activity+of+Pentafluorosulphanyl+%28SF5%29+Organic+Molecules&doi=10.1016%2Fj.jfluchem.2012.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic chemistry and biological activity of pentafluorosulphanyl (SF5) organic molecules</span></div><div class="casAuthors">Altomonte, Stefano; Zanda, Matteo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorine Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-93</span>CODEN:
                <span class="NLM_cas:coden">JFLCAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-1139</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pentafluorosulphanyl group (SF5) is a highly stable chem. function which has been attracting a great deal of interest owing to its peculiar chem., structural, physicochem. and biol. properties.  Progress in the area of SF5-compds. has been somewhat hindered by the lack of straightforward lab-scale synthetic methods for introducing the SF5-group into org. mols.  However, recent synthetic progress, the availability of some SF5-building blocks from com. suppliers and the discovery of interesting properties of SF5-substituted mols. in materials science, biol. and drug discovery are giving new momentum to research in this fascinating area of fluorine chem.  Synthesis, reactivity and biol. properties of SF5-substituted org. mols. are herein reviewed with an emphasis on the work published after year 2000.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3nL1S9vWsi7Vg90H21EOLACvtfcHk0lgec2cXCMbqZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktLzF&md5=cc717bde9781e01cf6d7e5d48e479154</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jfluchem.2012.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jfluchem.2012.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DAltomonte%26aufirst%3DS.%26aulast%3DZanda%26aufirst%3DM.%26atitle%3DSynthetic%2520Chemistry%2520and%2520Biological%2520Activity%2520of%2520Pentafluorosulphanyl%2520%2528SF5%2529%2520Organic%2520Molecules%26jtitle%3DJ.%2520Fluor.%2520Chem.%26date%3D2012%26volume%3D143%26spage%3D57%26epage%3D93%26doi%3D10.1016%2Fj.jfluchem.2012.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnes-Seeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsis, P.</span></span> <span> </span><span class="NLM_article-title">Metabolically Stable Tert-Butyl Replacement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/ml400045j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400045j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=514-516&author=D.+Barnes-Seemanauthor=M.+Jainauthor=L.+Bellauthor=S.+Ferreiraauthor=S.+Cohenauthor=X.-H.+Chenauthor=J.+Aminauthor=B.+Snodgrassauthor=P.+Hatsis&title=Metabolically+Stable+Tert-Butyl+Replacement&doi=10.1021%2Fml400045j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolically Stable tert-Butyl Replacement</span></div><div class="casAuthors">Barnes-Seeman, David; Jain, Monish; Bell, Leslie; Ferreira, Suzie; Cohen, Scott; Chen, Xiao-Hui; Amin, Jakal; Snodgrass, Brad; Hatsis, Panos</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">514-516</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Susceptibility to metab. is a common issue with the tert-Bu group on compds. of medicinal interest.  The authors demonstrate an approach of removing all the fully Sp3 C-Hs from a tert-Bu group: replacing some C-Hs with C-Fs and increasing the s-character of the remaining C-Hs.  This approach gave a trifluoromethylcyclopropyl group, which increased metabolic stability.  Trifluoromethylcyclopropyl-contg. analogs had consistently higher metabolic stability in vitro and in vivo compared to their tert-butyl-contg. counterparts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao2I5ukFsTrVg90H21EOLACvtfcHk0ljY7t_tlkoEBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D&md5=156572485ea3a48f5e1b6ea7409f1f61</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fml400045j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400045j%26sid%3Dliteratum%253Aachs%26aulast%3DBarnes-Seeman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DL.%26aulast%3DFerreira%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.-H.%26aulast%3DAmin%26aufirst%3DJ.%26aulast%3DSnodgrass%26aufirst%3DB.%26aulast%3DHatsis%26aufirst%3DP.%26atitle%3DMetabolically%2520Stable%2520Tert-Butyl%2520Replacement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D514%26epage%3D516%26doi%3D10.1021%2Fml400045j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haigh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, B.</span></span> <span> </span><span class="NLM_article-title">Molecular Shape and Medicinal Chemistry: A Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3862</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.1021/jm900818s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900818s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3862-3886&author=A.+Nichollsauthor=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=A.+C.+Goodauthor=G.+Warrenauthor=M.+Mathieuauthor=S.+W.+Muchmoreauthor=S.+P.+Brownauthor=J.+A.+Grantauthor=J.+A.+Haighauthor=N.+Nevinsauthor=A.+N.+Jainauthor=B.+Kelley&title=Molecular+Shape+and+Medicinal+Chemistry%3A+A+Perspective&doi=10.1021%2Fjm900818s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape and Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Nicholls, Anthony; McGaughey, Georgia B.; Sheridan, Robert P.; Good, Andrew C.; Warren, Gregory; Mathieu, Magali; Muchmore, Steven W.; Brown, Scott P.; Grant, J. Andrew; Haigh, James A.; Nevins, Neysa; Jain, Ajay N.; Kelley, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3862-3886</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review article with 111 refs. summarized perspectives of mol. shape and medicinal chem. in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqtbXOEoSmy7Vg90H21EOLACvtfcHk0ljY7t_tlkoEBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D&md5=85664344e13872527a3dfb2296d34864</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm900818s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900818s%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DWarren%26aufirst%3DG.%26aulast%3DMathieu%26aufirst%3DM.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DHaigh%26aufirst%3DJ.%2BA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DKelley%26aufirst%3DB.%26atitle%3DMolecular%2520Shape%2520and%2520Medicinal%2520Chemistry%253A%2520A%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3862%26epage%3D3886%26doi%3D10.1021%2Fjm900818s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routh, J. I.</span></span> <span> </span><span class="NLM_article-title">The Binding of Salicylic Acid and Acetylsalicylic Acid to Human Serum Albumin</span>. <i>Clin. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">98</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/S0009-9120(73)80018-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0009-9120%2873%2980018-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=4728967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A280%3ADyaE3s3kvVSgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1973&pages=98-105&author=S.+E.+Kramerauthor=J.+I.+Routh&title=The+Binding+of+Salicylic+Acid+and+Acetylsalicylic+Acid+to+Human+Serum+Albumin&doi=10.1016%2FS0009-9120%2873%2980018-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The binding of salicylic acid and acetylsalicylic acid to human serum albumin</span></div><div class="casAuthors">Kramer E; Routh J I</div><div class="citationInfo"><span class="NLM_cas:title">Clinical biochemistry</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">98-105</span>
        ISSN:<span class="NLM_cas:issn">0009-9120</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBgqevziwH9AxyojvcjibhfW6udTcc2eZF3dPkLJRQLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3s3kvVSgtQ%253D%253D&md5=e894f2b559aea3efc490cd810af141ae</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0009-9120%2873%2980018-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-9120%252873%252980018-9%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DS.%2BE.%26aulast%3DRouth%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520Binding%2520of%2520Salicylic%2520Acid%2520and%2520Acetylsalicylic%2520Acid%2520to%2520Human%2520Serum%2520Albumin%26jtitle%3DClin.%2520Biochem.%26date%3D1973%26volume%3D6%26spage%3D98%26epage%3D105%26doi%3D10.1016%2FS0009-9120%2873%2980018-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span> <span> </span><span class="NLM_article-title">Interactive Association of Drugs Binding to Human Serum Albumin</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3580</span>– <span class="NLM_lpage">3595</span>, <span class="refDoi"> DOI: 10.3390/ijms15033580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.3390%2Fijms15033580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=24583848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWhtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3580-3595&author=F.+Yangauthor=Y.+Zhangauthor=H.+Liang&title=Interactive+Association+of+Drugs+Binding+to+Human+Serum+Albumin&doi=10.3390%2Fijms15033580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive association of drugs binding to human serum albumin</span></div><div class="casAuthors">Yang, Feng; Zhang, Yao; Liang, Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3580-3595</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Human serum albumin (HSA) is an abundant plasma protein, which attracts great interest in the pharmaceutical industry since it can bind a remarkable variety of drugs impacting their delivery and efficacy and ultimately altering the drug's pharmacokinetic and pharmacodynamic properties.  Addnl., HSA is widely used in clin. settings as a drug delivery system due to its potential for improving targeting while decreasing the side effects of drugs.  It is thus of great importance from the viewpoint of pharmaceutical sciences to clarify the structure, function, and properties of HSA-drug complexes.  This review will succinctly outline the properties of binding site of drugs in IIA subdomain within the structure of HSA.  We will also give an overview on the binding characterization of interactive assocn. of drugs to human serum albumin that may potentially lead to significant clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopBtedaqwZgLVg90H21EOLACvtfcHk0liBdIGN48p08g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWhtbnP&md5=c50411459367ad9576b28a2934da0d0f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fijms15033580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms15033580%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26atitle%3DInteractive%2520Association%2520of%2520Drugs%2520Binding%2520to%2520Human%2520Serum%2520Albumin%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D15%26spage%3D3580%26epage%3D3595%26doi%3D10.3390%2Fijms15033580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmarajan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vought, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span> <span> </span><span class="NLM_article-title">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">24242</span>– <span class="NLM_lpage">24256</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.014498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1074%2Fjbc.M109.014498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=19556245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24242-24256&author=A.+Patelauthor=V.+Dharmarajanauthor=V.+E.+Voughtauthor=M.+S.+Cosgrove&title=On+the+Mechanism+of+Multiple+Lysine+Methylation+by+the+Human+Mixed+Lineage+Leukemia+Protein-1+%28MLL1%29+Core+Complex&doi=10.1074%2Fjbc.M109.014498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Vought, Valarie E.; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">24242-24256</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription in eukaryotic genomes depends on enzymes that regulate the degree of histone H3 lysine 4 (H3K4) methylation.  The mixed lineage leukemia protein-1 (MLL1) is a member of the SET1 family of H3K4 methyltransferases and is frequently rearranged in acute leukemias.  Despite sequence comparisons that predict that SET1 family enzymes should only monomethylate their substrates, mono-, di-, and trimethylation of H3K4 has been attributed to SET1 family complexes in vivo and in vitro.  To better understand this paradox, we have biochem. reconstituted and characterized a five-component 200-kDa MLL1 core complex contg. human MLL1, WDR5, RbBP5, Ash2L, and DPY-30.  We demonstrate that the isolated MLL1 SET domain is a slow monomethyltransferase and that tyrosine 3942 of MLL1 prevents di- and trimethylation of H3K4.  In contrast, a complex contg. the MLL1 SET domain, WDR5, RbBP5, Ash2L, and DPY-30, displays a marked ∼600-fold increase in enzymic activity but only to the di-Me form of H3K4.  Single turnover kinetic expts. reveal that the reaction leading to H3K4 dimethylation involves the transient accumulation of a monomethylated species, suggesting that the MLL1 core complex uses a non-processive mechanism to catalyze multiple lysine methylation.  We have also discovered that the non-SET domain components of the MLL1 core complex possess a previously unrecognized methyltransferase activity that catalyzes H3K4 dimethylation within the MLL1 core complex.  Our results suggest that the mechanism of multiple lysine methylation by the MLL1 core complex involves the sequential addn. of two Me groups at two distinct active sites within the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZXmcwVcpRrVg90H21EOLACvtfcHk0liBdIGN48p08g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF&md5=4289813be10179ac818b8d93fe6ebc6a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.014498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.014498%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DOn%2520the%2520Mechanism%2520of%2520Multiple%2520Lysine%2520Methylation%2520by%2520the%2520Human%2520Mixed%2520Lineage%2520Leukemia%2520Protein-1%2520%2528MLL1%2529%2520Core%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24242%26epage%3D24256%26doi%3D10.1074%2Fjbc.M109.014498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lapointe, B. T.</span>; <span class="NLM_string-name">Fuller, P. H.</span>; <span class="NLM_string-name">Gunaydin, H.</span>; <span class="NLM_string-name">Liu, K.</span>; <span class="NLM_string-name">Sciammetta, N.</span>; <span class="NLM_string-name">Trotter, B. W.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Barr, K. J.</span>; <span class="NLM_string-name">MacLean, J. K. F.</span>; <span class="NLM_string-name">Molinari, D. F.</span>; <span class="NLM_string-name">Simov, V.</span>; <span class="NLM_string-name">Davis, C. E.</span></span> <span> </span><span class="NLM_article-title">Substituted Pyrazole Compounds as RORgammaT Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2016/130818</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+T.+Lapointe&author=P.+H.+Fuller&author=H.+Gunaydin&author=K.+Liu&author=N.+Sciammetta&author=B.+W.+Trotter&author=H.+Zhang&author=K.+J.+Barr&author=J.+K.+F.+MacLean&author=D.+F.+Molinari&author=V.+Simov&author=C.+E.+Davis&title=Substituted+Pyrazole+Compounds+as+RORgammaT+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLapointe%26aufirst%3DB.%2BT.%26atitle%3DSubstituted%2520Pyrazole%2520Compounds%2520as%2520RORgammaT%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Barr, K. J.</span>; <span class="NLM_string-name">Lapointe, B. T.</span>; <span class="NLM_string-name">Gunaydin, H.</span>; <span class="NLM_string-name">Liu, K.</span>; <span class="NLM_string-name">Trotter, B. W.</span>; <span class="NLM_string-name">Davis, C. E.</span>; <span class="NLM_string-name">Brodowski, M.</span>; <span class="NLM_string-name">Burns, D. J.</span>; <span class="NLM_string-name">Ashraf, S.</span>; <span class="NLM_string-name">Argentieri, S. R.</span></span> <span> </span><span class="NLM_article-title">Heteroaryl Substituted Benzoic Acids as RORgammaT Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2017/075185</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=H.+Zhang&author=K.+J.+Barr&author=B.+T.+Lapointe&author=H.+Gunaydin&author=K.+Liu&author=B.+W.+Trotter&author=C.+E.+Davis&author=M.+Brodowski&author=D.+J.+Burns&author=S.+Ashraf&author=S.+R.+Argentieri&title=Heteroaryl+Substituted+Benzoic+Acids+as+RORgammaT+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DHeteroaryl%2520Substituted%2520Benzoic%2520Acids%2520as%2520RORgammaT%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn-Robitaille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissarie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinier, A.</span></span> <span> </span><span class="NLM_article-title">Mild and Diazo-Free Synthesis of Trifluoromethyl-Cyclopropanes Using Sulfonium Ylides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2265</span>– <span class="NLM_lpage">2268</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b00557</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b00557" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFWku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=2265-2268&author=P.+Cyrauthor=J.+Flynn-Robitailleauthor=P.+Boissarieauthor=A.+Marinier&title=Mild+and+Diazo-Free+Synthesis+of+Trifluoromethyl-Cyclopropanes+Using+Sulfonium+Ylides&doi=10.1021%2Facs.orglett.9b00557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mild and Diazo-Free Synthesis of Trifluoromethyl-Cyclopropanes Using Sulfonium Ylides</span></div><div class="casAuthors">Cyr, Patrick; Flynn-Robitaille, Joel; Boissarie, Patrick; Marinier, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2265-2268</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of several 1,1-disubstituted trifluoromethyl-cyclopropanes (TFCPs), known as tert-Bu bioisosteres, was achieved from the reaction between trifluoromethylalkenes and unstabilized sulfonium ylides in yields of ≤97%.  This method offers practical access to this cyclopropyl moiety of pharmacol. interest, employing a com. available reagent at low temps.  The synthesis of cyclopropanes bearing other electron-withdrawing groups as well as trisubstituted TFCPs was also accomplished.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTjdgU11yVd7Vg90H21EOLACvtfcHk0lhGJ-hzl5ferQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFWku70%253D&md5=ab2d8364af72c5a1ea65b22659d61007</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b00557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b00557%26sid%3Dliteratum%253Aachs%26aulast%3DCyr%26aufirst%3DP.%26aulast%3DFlynn-Robitaille%26aufirst%3DJ.%26aulast%3DBoissarie%26aufirst%3DP.%26aulast%3DMarinier%26aufirst%3DA.%26atitle%3DMild%2520and%2520Diazo-Free%2520Synthesis%2520of%2520Trifluoromethyl-Cyclopropanes%2520Using%2520Sulfonium%2520Ylides%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D2265%26epage%3D2268%26doi%3D10.1021%2Facs.orglett.9b00557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schanda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupče, E̅.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brutscher, B.</span></span> <span> </span><span class="NLM_article-title">SOFAST-HMQC Experiments for Recording Two-Dimensional Deteronuclear Correlation Spectra of Proteins within a Few Seconds</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1007/s10858-005-4425-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1007%2Fs10858-005-4425-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=16341750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=199-211&author=P.+Schandaauthor=E%CC%85.+Kup%C4%8Deauthor=B.+Brutscher&title=SOFAST-HMQC+Experiments+for+Recording+Two-Dimensional+Deteronuclear+Correlation+Spectra+of+Proteins+within+a+Few+Seconds&doi=10.1007%2Fs10858-005-4425-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">SOFAST-HMQC Experiments for Recording Two-dimensional Heteronuclear Correlation Spectra of Proteins within a Few Seconds</span></div><div class="casAuthors">Schanda, Paul; Kupce, Eriks; Brutscher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-211</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast multidimensional NMR with a time resoln. of a few seconds provides a new tool for high throughput screening and site-resolved real-time studies of kinetic mol. processes by NMR.  Recently we have demonstrated the feasibility to record protein 1H-15N correlation spectra in a few seconds of acquisition time using a new SOFAST-HMQC expt.  Here, we investigate in detail the performance of SOFAST-HMQC to record 1H-15N and 1H-13C correlation spectra of proteins of different size and at different magnetic field strengths.  Compared to std. 1H-15N correlation expts. SOFAST-HMQC provides a significant gain in sensitivity, esp. for fast repetition rates.  Guidelines are provided on how to set up SOFAST-HMQC expts. for a given protein sample.  In addn., an alternative pulse scheme, IPAP-SOFAST-HMQC is presented that allows application on NMR spectrometers equipped with cryogenic probes, and fast measurement of one-bond 1H-13C and 1H-15N scalar and residual dipolar coupling consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoE4Mp0wosbVg90H21EOLACvtfcHk0lhGJ-hzl5ferQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbF&md5=6864fb3383e78dabae806e6869a470e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10858-005-4425-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-005-4425-x%26sid%3Dliteratum%253Aachs%26aulast%3DSchanda%26aufirst%3DP.%26aulast%3DKup%25C4%258De%26aufirst%3DE%25CC%2585.%26aulast%3DBrutscher%26aufirst%3DB.%26atitle%3DSOFAST-HMQC%2520Experiments%2520for%2520Recording%2520Two-Dimensional%2520Deteronuclear%2520Correlation%2520Spectra%2520of%2520Proteins%2520within%2520a%2520Few%2520Seconds%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2005%26volume%3D33%26spage%3D199%26epage%3D211%26doi%3D10.1007%2Fs10858-005-4425-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liiIeYhOHh9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+Mccoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liiIeYhOHh9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: Building New Software for Automated Crystallographic Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+Mccoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+New+Software+for+Automated+Crystallographic+Structure+Determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0liiIeYhOHh9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520New%2520Software%2520for%2520Automated%2520Crystallographic%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh5-EPsmmpIcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Schrodinger LLC</span> <i>The PyMOL Molecular Graphics System</i>, <i>Version
1.8.</i> <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schrodinger+LLC&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y7R','PDB','4Y7R'); return false;">PDB: 4Y7R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6','PDB','3EG6'); return false;">PDB: 3EG6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6U6W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6U6W','PDB','6U6W'); return false;">PDB: 6U6W</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i252"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01411">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_74686"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01411?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01411</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structure of FPA probe and negative controls; additional thermal shift data; additional coimmunoprecipitation experiments; histone methyl transferase assay data; X-ray data collection and refinement statistics; experimental details for the synthesis of probe and controls (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_001.pdf">jm9b01411_si_001.pdf (810.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01411/suppl_file/jm9b01411_si_002.csv">jm9b01411_si_002.csv (6.28 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01411&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01411%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01411" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679952d32a6c3dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
