
<html lang="en"     class="pb-page"  data-request-id="17d09489-2e2e-44cb-95f0-b6d549727a63"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.8b00168;requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Modification of Heterocyclic Antagonists of the P2Y14 Receptor" /></meta><meta name="dc.Creator" content="Jinha  Yu" /></meta><meta name="dc.Creator" content="Antonella  Ciancetta" /></meta><meta name="dc.Creator" content="Steven  Dudas" /></meta><meta name="dc.Creator" content="Sierra  Duca" /></meta><meta name="dc.Creator" content="Justine  Lottermoser" /></meta><meta name="dc.Creator" content="Kenneth A.  Jacobson" /></meta><meta name="dc.Description" content="The P2Y14 receptor (P2Y14R) mediates inflammatory activity by activating neutrophil motility, but few classes of antagonists are known. We have explored the structure–activity relationship of a 3-(..." /></meta><meta name="Description" content="The P2Y14 receptor (P2Y14R) mediates inflammatory activity by activating neutrophil motility, but few classes of antagonists are known. We have explored the structure–activity relationship of a 3-(..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 16, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00168" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="This article not subject to U.S. Copyright. Published 2018 by the American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00168" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00168" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00168" /></link>
        
    
    

<title>Structure-Guided Modification of Heterocyclic Antagonists of the P2Y14 Receptor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00168" /></meta><meta property="og:title" content="Structure-Guided Modification of Heterocyclic Antagonists of the P2Y14 Receptor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0016.jpeg" /></meta><meta property="og:description" content="The P2Y14 receptor (P2Y14R) mediates inflammatory activity by activating neutrophil motility, but few classes of antagonists are known. We have explored the structure–activity relationship of a 3-(4-phenyl-1H-1,2,3-triazol-1-yl)-5-(aryl)benzoic acid antagonist scaffold, assisted by docking and molecular dynamics (MD) simulation at a P2Y14R homology model. A computational pipeline using the High Throughput MD Python environment guided the analogue design. Selection of candidates was based upon ligand–protein shape and complementarity and the persistence of ligand–protein interactions over time. Predictions of a favorable substitution of a 5-phenyl group with thiophene and an insertion of a three-methylene spacer between the 5-aromatic and alkyl amino moieties were largely consistent with empirical results. The substitution of a key carboxylate group on the core phenyl ring with tetrazole or truncation of the 5-aryl group reduced affinity. The most potent antagonists, using a fluorescent assay, were a primary 3-aminopropyl congener 20 (MRS4458) and phenyl p-carboxamide 30 (MRS4478)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00168"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00168">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00168&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00168&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00168&amp;href=/doi/10.1021/acs.jmedchem.8b00168" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 4860-4882</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00140" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00270" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Modification of Heterocyclic Antagonists of the P2Y<sub>14</sub> Receptor</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jinha Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinha Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinha++Yu">Jinha Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antonella Ciancetta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antonella Ciancetta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antonella++Ciancetta">Antonella Ciancetta</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7612-2050" title="Orcid link">http://orcid.org/0000-0002-7612-2050</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steven Dudas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven Dudas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven++Dudas">Steven Dudas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sierra Duca</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sierra Duca</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sierra++Duca">Sierra Duca</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justine Lottermoser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justine Lottermoser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justine++Lottermoser">Justine Lottermoser</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Kenneth A. Jacobson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth A. Jacobson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Recognition Section, Laboratory of Bioorganic Chemistry, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Address: Molecular Recognition Section, Bldg. 8A, Rm. B1A-19, NIH, NIDDK, LBC, Bethesda, MD 20892-0810. Tel: 301-496-9024. Fax: 301-480-8422. E-mail: <a href="/cdn-cgi/l/email-protection#bcd7d9d2d2d9c8d4d6fcd2d5d8d8d792d2d5d492dbd3ca"><span class="__cf_email__" data-cfemail="dcb7b9b2b2b9a8b4b69cb2b5b8b8b7f2b2b5b4f2bbb3aa">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+A.++Jacobson">Kenneth A. Jacobson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8104-1493" title="Orcid link">http://orcid.org/0000-0001-8104-1493</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00168&amp;href=/doi/10.1021%2Facs.jmedchem.8b00168" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 4860–4882</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 16, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 February 2018</li><li><span class="item_label"><b>Published</b> online</span>16 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 June 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00168" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00168</a></div><div class="article_header-article-copyright"><strong>This article not subject to U.S. Copyright. Published 2018 by the American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4860%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJinha%2BYu%252C%2BAntonella%2BCiancetta%252C%2BSteven%2BDudas%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D11%26contentID%3Dacs.jmedchem.8b00168%26title%3DStructure-Guided%2BModification%2Bof%2BHeterocyclic%2BAntagonists%2Bof%2Bthe%2BP2Y14%2BReceptor%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4882%26publicationDate%3DJune%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00168"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1867</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">20</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00168" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Modification of Heterocyclic Antagonists of the P2Y14 Receptor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jinha&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Antonella&quot;,&quot;last_name&quot;:&quot;Ciancetta&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;Dudas&quot;},{&quot;first_name&quot;:&quot;Sierra&quot;,&quot;last_name&quot;:&quot;Duca&quot;},{&quot;first_name&quot;:&quot;Justine&quot;,&quot;last_name&quot;:&quot;Lottermoser&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;A. Jacobson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;4860-4882&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00168&quot;},&quot;abstract&quot;:&quot;The P2Y14 receptor (P2Y14R) mediates inflammatory activity by activating neutrophil motility, but few classes of antagonists are known. We have explored the structure–activity relationship of a 3-(4-phenyl-1H-1,2,3-triazol-1-yl)-5-(aryl)benzoic acid antagonist scaffold, assisted by docking and molecular dynamics (MD) simulation at a P2Y14R homology model. A computational pipeline using the High Throughput MD Python environment guided the analogue design. Selection of candidates was based upon ligand–protein shape and complementarity and the persistence of ligand–protein interactions over time. Predictions of a favorable substitution of a 5-phenyl group with thiophene and an insertion of a three-methylene spacer between the 5-aromatic and alkyl amino moieties were largely consistent with empirical results. The substitution of a key carboxylate group on the core phenyl ring with tetrazole or truncation of the 5-aryl group reduced affinity. The most potent antagonists, using a fluorescent assay, were a primary 3&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00168&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00168" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00168&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00168" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00168&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00168" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00168&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00168&amp;href=/doi/10.1021/acs.jmedchem.8b00168" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00168" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00168" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00168%26sid%3Dliteratum%253Aachs%26pmid%3D29767967%26genre%3Darticle%26aulast%3DYu%26date%3D2018%26atitle%3DStructure-Guided%2BModification%2Bof%2BHeterocyclic%2BAntagonists%2Bof%2Bthe%2BP2Y14%2BReceptor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D11%26spage%3D4860%26epage%3D4882%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (8)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=290716" title="Sulfones">Sulfones</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/jmcmar.2018.61.issue-11/20180614/jmcmar.2018.61.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The P2Y<sub>14</sub> receptor (P2Y<sub>14</sub>R) mediates inflammatory activity by activating neutrophil motility, but few classes of antagonists are known. We have explored the structure–activity relationship of a 3-(4-phenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(aryl)benzoic acid antagonist scaffold, assisted by docking and molecular dynamics (MD) simulation at a P2Y<sub>14</sub>R homology model. A computational pipeline using the High Throughput MD Python environment guided the analogue design. Selection of candidates was based upon ligand–protein shape and complementarity and the persistence of ligand–protein interactions over time. Predictions of a favorable substitution of a 5-phenyl group with thiophene and an insertion of a three-methylene spacer between the 5-aromatic and alkyl amino moieties were largely consistent with empirical results. The substitution of a key carboxylate group on the core phenyl ring with tetrazole or truncation of the 5-aryl group reduced affinity. The most potent antagonists, using a fluorescent assay, were a primary 3-aminopropyl congener <b>20</b> (MRS4458) and phenyl <i>p</i>-carboxamide <b>30</b> (MRS4478).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Extracellular nucleotides released by tissue and organs during stress or injury activate a class of cell-surface receptors (P2Rs) to boost the innate and adaptive immune responses.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> This mechanism acts as a time-dependent component of the signaling purinome, along with the anti-inflammatory adenosine receptors (ARs, also termed P1 receptors), to protect the organism in various challenged circumstances. The P2Y<sub>14</sub> receptor (P2Y<sub>14</sub>R) responds to endogenous agonists uridine-5′-diphosphoglucose and uridine-5′-diphosphate to mediate inflammatory activity, in part by activating human neutrophil motility.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Mouse neutrophils do not express the P2Y<sub>14</sub>R,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and the recruitment of neutrophils in the mouse lung induced by LPS is mediated by a platelet P2Y<sub>14</sub>R, which has no effect on hemostasis.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_p">Structurally, the P2Y<sub>14</sub>R belongs to the δ-branch of rhodopsin-like G protein-coupled receptors (GPCRs) in a subfamily of three P2YRs that are preferentially coupled to inhibition of adenylate cyclase through guanine nucleotide inhibitory (G<sub>i</sub>) protein. P2Y<sub>14</sub>R and other P2YR subtypes are also associated with proinflammatory effects, and their antagonists are desired for their anti-inflammatory activity.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14">(7−14)</a> Selective P2Y<sub>14</sub>R antagonists are sought as potential agents for treating asthma, sterile inflammation of the kidney, and possibly diabetes and neurodegeneration.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12">(7−12)</a> However, only a few classes of antagonists are known, so there is a clear need for more diverse competitive P2Y<sub>14</sub>R antagonists.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Selective P2Y<sub>14</sub>R antagonists endowed with good bioavailability properties are not available.</div><div class="NLM_p">As the three-dimensional structure of the P2Y<sub>14</sub>R has not yet been elucidated, X-ray crystallographic structures of the closely related P2Y<sub>12</sub>R have been used to predict the binding mode of various competitive agonist and antagonist ligands.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> The sequence homology between human (h) P2Y<sub>12</sub>R and P2Y<sub>14</sub>R is ∼48%, and key amino acids for coordinating the bound nucleotides are common to both receptors. Thus, homology modeling has predicted detailed interactions of P2Y<sub>14</sub>R agonists and antagonists in a manner consistent with their structure–activity relationship (SAR).<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a></div><div class="NLM_p">Antagonists of the P2Y<sub>14</sub>R were first reported by Black and colleagues,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and of the two classes reported, naphthoic acids and pyrido[4,3-<i>d</i>]pyrimidines, only the former appeared to be competitive antagonists. We have explored both the pharmacology of a reported high affinity antagonist in that series, 4-(4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl)-2-naphthoic acid (PPTN, <b>1</b>, <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) and the SAR of related heterocyclic scaffolds.<a onclick="showRef(event, 'ref17 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref20">(17,20)</a> Compound <b>1</b> was shown to be highly specific for interaction with the P2Y<sub>14</sub>R with high affinity (<i>K</i><sub>i</sub> = 0.4 nM in a functional assay).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, it was poorly bioavailable, and its amphiphilic and hydrophobic character leads to low solubility in either aqueous or apolar solvents. A prodrug of <b>1</b> containing a dimethylaminocarboxymethyl ester improved the utility of this antagonist in vivo.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Recently, we reported that a 3-(4-phenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(phenyl)benzoic acid scaffold was suitable for the design of P2Y<sub>14</sub>R antagonists. The combination of a core benzene ring and triazole (shown in blue in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) was predicted by modeling to be a bioisosteric replacement for the naphthalene moiety. Here, we also used a structure-based approach that included docking and molecular dynamics (MD) simulation at a hP2Y<sub>14</sub>R homology model. This study extends the SAR reported by Junker et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with replacement of specific features attached to the core benzene ring of the scaffold: the 5-aryl group and the carboxylate group.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Progression of the Structural Modifications of Naphthoic Acid Derivative <b>1</b> To the Present Set of Derivatives <b>2b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The major focus was the introduction of diverse heteroaromatic rings in place of the bridging phenyl ring shown in green. The <i>p</i>-trifluoromethylphenyl moiety shown in red and the core benzene ring and adjacent triazole shown in blue were not modified in this study. Compound <b>3a</b> is the fluorescent antagonist containing structure <b>1</b> as a pharmacophore that was used in the flow cytometric assay. Group X is a carboxylic acid or a bioisosteric replacement.</p></p></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20708" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20708" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Molecular Modeling and Compound Design</h3><div class="NLM_p">We based our computational approach to the design of 1-substituted 3-(4-phenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(aryl)benzene derivatives as new P2Y<sub>14</sub>R antagonists on the assumption that this chemical series binds at the orthosteric site of the receptor, located at the upper portion of the transmembrane (TM) domain facing the extracellular environment. Confidence in this assumption has accrued from previously published studies in which the predicted detailed interactions of P2Y<sub>14</sub>R agonists and antagonists were largely consistent with their SAR.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> We used a homology model of the human (h) P2Y<sub>14</sub>R based on the high-resolution X-ray structure of an agonist-bound hP2Y<sub>12</sub>R (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PXZ">4PXZ</a>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which was previously refined using MD simulation,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and set up a computational pipeline to guide the design of the new scaffold and prioritize the synthesis of the newly designed analogues. Similar to our previous approach, the pipeline consisted of docking of potential candidates at the hP2Y<sub>14</sub>R homology model followed by molecular dynamics (MD) refinement (30 ns of MD simulation run in triplicate for each structure) of a few selected hits in the membrane-embedded solvated environment (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). At variance with the previously reported strategy,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> in this study we accelerated the pipeline by docking the compounds at the rigid receptor and automating the MD refinement phase by exploiting the recently developed High Throughput MD (HTMD) module,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> a programmable environment based on Python that enables performing an automatic system setup, equilibration, and production by running a single script.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Computational pipeline applied for the design of new derivatives as hP2Y<sub>14</sub>R antagonists. A library of compounds was designed according to the general scheme reported in <a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>. The compounds were docked at a hP2Y<sub>14</sub>R homology model (Glide, SP scoring function). Hit selection was performed by visual inspection followed, for a few selected candidates, by membrane molecular dynamics (MD) refinement (30 ns, run in triplicate, CHARMM36/CGenFF, POPC lipid model). The MD refinement phase was sped up using High Throughput MD (HTMD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The design strategy was to retain the optimal, hydrophobic 4-(4-trifluoromethyl)-phenyl moiety (red in <a class="ref internalNav" href="#cht1" aria-label="Charts 1">Charts 1</a> and <a class="ref internalNav" href="#cht2" aria-label="2">2</a>) that was present in the original lead compound <b>1</b> and modify other parts of the scaffold. This trifluoromethyl-phenyl moiety was found to be optimal among 64 differently substituted 4-phenyl derivatives<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and predicted to be deeply buried in a narrow, hydrophobic region of the orthosteric hP2Y<sub>14</sub>R binding site. As a result, the 4-(4-trifluoromethyl)-phenyl analogue <b>2a</b> from Junker et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was only 6-fold less potent than <b>1</b> in a fluorescence binding assay using flow cytometry in hP2Y<sub>14</sub>R-expressing Chinese hamster ovary (CHO) cells.<a onclick="showRef(event, 'ref17 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref20">(17,20)</a> The adjacent triazole moiety, introduced with the previous set of P2Y<sub>14</sub>R antagonists, was also retained in the current set (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) due to its ability to be synthesized from readily available intermediates through a click ([2 + 3]cycloaddition of azide and alkyne) reaction. On another region of this molecular series, a primary or secondary alkylamine was included, similar to the piperidine moiety in <b>1</b> and <b>2a</b>. This moiety is predicted to interact with the P2Y<sub>14</sub>R extracellular loop (EL) region, which is more flexible than the deeper binding region surrounding the minimal pharmacophore. Although our SAR exploration of <b>1</b> predicted that a charged amino group is not essential,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> we preserved the amino function in various analogues for interaction with polar groups present in this region of the receptor. The expectation from molecular modeling that the amino moiety could be substituted with various extended or tethered bulky groups enabled the introduction of the high affinity fluorescent antagonist ligand <b>3a</b> used in the flow cytometry assay.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We, therefore, devoted an effort to more clearly define possible interactions of this ligand series with the ELs using molecular modeling. An intermediate structure of the fluorescent probe <b>3a</b> that terminated in a primary amino group, containing the triazole ring, but not the amide-linked fluorophore, i.e., <b>3d</b>, was built in place starting from the docking pose of <b>1</b> in the hP2Y<sub>14</sub>R homology model. Then, the region around the ligand (10 Å) was subjected to Monte Carlo sampling (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Supporting Information</a>). In the resulting refined complex (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S1</a>), the primary amino group of <b>3d</b> extended toward the extracellular side and established additional H-bond and salt-bridge interactions with the backbone and side chain of Glu166 (EL2).</div><figure id="cht2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Detailed General Approach for the Design of New Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The expected orientation of the scaffold with respect to the hP2Y<sub>14</sub>R is shown in a symbolic representation. Diverse 5- and 6-membered heteroaromatic rings linking the core (cyan) to the group facing the EC side of the receptor (yellow) were considered, although not all of these analogues were synthesized. Various linking groups or linkers (magenta), including amide, sulfonamide, and methylene, were inserted to provide conformational diversity in the terminal alkylamino moiety. The terminal group on the spacer moiety was an H-bond donor/acceptor group, an amino, a protected amino, or a hydroxyl group. The spacer group consisted of either a straight chain alkyl moiety or a six-membered ring containing the secondary amine.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures, Affinity, and cLogP Values of hP2Y<sub>14</sub>R Antagonists<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0012.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0013.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0014.gif" alt="" id="fx20" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0015.gif" alt="" id="fx21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> inhibition of hP2Y<sub>14</sub>R antagonist binding, determined using flow cytometry of whole hP2Y<sub>14</sub>R-CHO cells in the presence of a fixed concentration (20 nM) of <b>3a</b> (mean ± SEM, <i>n</i> = 3–6).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Structure shown in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Tested as the triethylammonium salt.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Tested as the sodium salt.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Data from Junker et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">cLogP calculated using ALOGPS 2.1 program (<a href="http://www.vcclab.org/lab/alogps/" class="extLink">www.vcclab.org/lab/alogps/</a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></div><div></div></div><div class="NLM_p">The major focus in the present set of P2Y<sub>14</sub>R antagonist analogues is, therefore, the aryl ring (green, in <a class="ref internalNav" href="#cht1" aria-label="Charts 1">Charts 1</a> and <a class="ref internalNav" href="#cht2" aria-label="2">2</a>) bridging the polar amino chain to the rest of the molecule. In compounds <b>1</b> and <b>2a</b>, this aryl group consists of a <i>p</i>-substituted phenyl ring attached at the 5 position of the core benzene ring. We therefore designed compounds bearing heterocyclic 5- and 6-membered rings in place of the <i>p</i>-substituted phenyl ring that could be linked via a sulfonyl chloride intermediate to the amino function as shown in the general scaffold (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>, linker = SO<sub>2</sub>NH). The selection of promising candidates was performed mainly by visual inspection according to the ligand–protein shape and complementarity and, for a few selected derivatives, the persistency of ligand–protein interactions over time. As part of this inspection, we selected docking poses that maintained the expected interaction pattern for the hydrophobic 4-(4-trifluoromethyl)-phenyl (common to chemical series <b>1</b> and <b>2a</b>) and the carboxylic acid moieties rather than relying on the ranking by docking score (listed in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S1</a>). In particular, the key polar interactions of the carboxylate group identified previously are electrostatic pairing with the side chains of Lys77<sup>2.60</sup> (Ballesteros–Weinstein numbering<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>) and Lys277<sup>7.35</sup>, and H-bonding with the phenol of Tyr102<sup>3.33</sup>. Furthermore, the terminal secondary amino group in <b>1</b> and <b>2a</b> interacted with the hP2Y<sub>14</sub>R EL region through H-bonding to the helical backbone carbonyl group of Gly80<sup>2.63</sup>. Notably, as discussed below, the computational predictions, based on this interaction pattern in qualitative agreement with the observed binding data, were able to identify specific geometric and electrostatic requirements for receptor binding.</div><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Selection of the Aromatic Ring</h4><div class="NLM_p">Among the computationally tested analogues containing 5-membered heteroaromatic rings such as thiophene, pyrrole, furan, and thiazole (green box in <a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>), the thiophene ring was predicted to better fit a narrow pocket connecting the orthosteric binding site to the EL environment. As depicted in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, along with the above-mentioned interactions, the thiophene ring of <b>13</b> enabled an additional π–cation interaction with Arg274<sup>7.32</sup>. Therefore, <b>13</b> was selected as the first target heterocycle to synthesize in the new series. Among the computationally tested analogues containing 6-membered aromatic rings such as benzene and different isomers of pyridine and pyrimidine (see green box in <a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>), the compounds were ranked as follows: benzene and pyridine were predicted to better fit the receptor than pyrimidine. Among the sulfonamide derivatives, the highest ranked 6-membered rings were predicted to establish the additional π–cation interaction with Arg274<sup>7.35</sup> (e.g., benzene-containing compound <b>16</b> in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). However, as the scaffolds of the analogues bearing a thiophene and a benzene ring superimposed well in the binding site, it was challenging to predict which of the two rings would lead to a more potent compound.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Upper panels: Selection of the aromatic ring. Docking pose of (A) thiophene derivative <b>13</b> (cyan carbon atoms) and (B) benzene derivative <b>16</b> (salmon carbon atoms) at the hP2Y<sub>14</sub>R. Both the thiophene and the benzene rings establish an additional π–cation interaction with Arg274<sup>7.32</sup> (dark green dashed lines). Lower panel: Selection of the optimal spacer between the aromatic and amine functions. Docking poses of derivatives bearing a spacer of three methylene groups and a (C) terminal primary amine (compound <b>7</b>, light blue carbon atoms) or (D) a hydroxyl group (dark green carbon atoms) at the hP2Y<sub>14</sub>R. Side chains of residues important for ligand recognition are reported as sticks (gray carbon atoms). H-bonds, salt bridges, and π–π stacking interactions are pictured as orange, magenta, and cyan dashed lines, respectively. Nonpolar hydrogen atoms are omitted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Selection of the Optimal Spacer between the Aromatic and Amine Functions</h4><div class="NLM_p last">Proceeding with the selected 5-membered heteroaromatic ring, i.e., thiophene, we evaluated the length of the spacer connecting the aromatic moiety to the amine function via a sulfonamide linker. In this stage, we also tested the possibility of replacing the amino group (positively charged at physiological pH and therefore giving rise to Zwitterionic compounds) with another hydrogen bond donor/acceptor moiety, such as the hydroxyl group (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S1</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C, a spacer consisting of three methylene groups was predicted to yield a derivative, compound <b>7</b>, amenable to establish additional H-bond interactions with residues residing at the extracellular tip of TM2. Moreover, the docking pose predicted for the charged amino group resulted in a better geometry and a more favorable pattern of H-bond interactions with respect to the corresponding derivative (not synthesized) bearing a hydroxyl moiety (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D).</div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Selection of the Optimal Spacer Geometry</h4><div class="NLM_p">An alternative binding pose of <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S2A</a>) suggested the possibility of rigidifying the flexible alkyl spacer by enclosing it in a 6-membered ring. We therefore docked several piperazine analogues bearing linkers of different geometries such as sulfonamide (<b>12</b>), amide (<b>22</b>), and methylene (<b>18</b>). The docking poses suggested that both the amide (compound <b>22</b>, <a class="ref internalNav" href="#notes-2" aria-label="Figure S2C">Figure S2C</a>) and the sulfonamide (compound <b>12</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S2B</a>) would allow the optimal orientation of the functional group, thus retaining the H-bond pattern with residues located in the extracellular tip of TM2. However, a methylene spacer (compound <b>18</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S2D</a>) was predicted to place the piperazine group in an unfavorable orientation with respect to the above-mentioned H-bond network. However, the bioassay result (see below) contradicted this prediction, suggesting that some degree of flexibility either in the linker or in the spacer is required. We then evaluated the differences between the sulfonamide and amide linkers in the acyclic series by comparing the two corresponding derivatives bearing three methylene groups, previously predicted as the optimal spacer (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, upper panel). The superimposition of the docking poses of the compounds bearing the sulfonamide group (<b>7</b>) and the amide (<b>20</b>) did not predict significant changes in the H-bond pattern around the primary amino function (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). We therefore subjected the two selected hits to 30 ns of MD simulation (run in triplicate, replicate analyses in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S2</a>) and observed the stability of the ligand–protein interactions over time. As shown by the visualization of the trajectories (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi" class="ext-link">Video S1</a>), both ligands were anchored in the binding site by a stable interaction pattern around the carboxylate moiety consisting of a H-bond with the Tyr102<sup>3.33</sup> side chain and two salt bridges with Lys77<sup>2.60</sup> and Lys277<sup>7.35</sup> for the <b>7</b>-hP2Y<sub>14</sub>R complex and with Lys77<sup>2.60</sup> and Arg253<sup>6.55</sup> for the <b>20</b>–hP2Y<sub>14</sub>R complex. The ligands were further stabilized by constant π–π stacking interactions connecting the 4-(4-trifluoromethyl)-phenyl moiety and the triazole ring to the His184<sup>5.36</sup> and Tyr102<sup>3.33</sup> side chains, respectively. With respect to the H-bond pattern anchoring the primary amine to the extracellular (EC) tip of TM2, only the H-bond with the Gly80<sup>2.63</sup> side chain was preserved during the MD simulations. Analogously, the π–cation interaction with Arg274<sup>7.35</sup> was not stable over time, as the residue engaged in either a salt bridge with the Asp81<sup>2.64</sup> side chain or a H-bond with the Gln16 (N-Term) side chain (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi" class="ext-link">Video S1</a>). No striking differences were noticed in the simulations of the two complexes, except a higher rigidity of the amine moiety in the amide compared with that in the sulfonamide-linked compound. This rigidity anchored the ligand in a slightly deeper position by allowing the carboxylate moiety to establish an additional salt bridge interaction with Arg253<sup>6.55</sup> though requiring higher conformational rearrangement by the protein (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi" class="ext-link">Video S1</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Indeed, as reported in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S2</a>, the <b>20</b>–hP2Y<sub>14</sub>R complex returned higher average protein alpha-carbon atom (Cα) RMSD values with respect to the <b>7</b>-hP2Y<sub>14</sub>R complex. A comparison between the ligand–protein complexes characterized by the lowest ligand–protein interaction energy (IE<sub>min</sub>) extracted from the selected trajectories (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D) highlighted the presence of an additional salt bridge for the amide derivative (<b>20</b>). Although it was challenging to unambiguously identify which of the two compounds would lead to a more potent derivative, knowledge gained from a previous SAR study<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and the MD results described above provided a hypothesis. Specifically, moieties displaying a higher conformational rigidity, a planar geometry, and an aromatic character would fit better within the narrow pocket connecting the orthosteric binding site with the extracellular (EC) environment delimited by the conserved disulfide bridge between Cys94<sup>3.25</sup> and Cys172 (EL2), on one side, and TM2, on the other side (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selection of the optimal linker geometry: alkyl sulfonamide vs amide. Upper panel: docking analysis. (A) Superimposition of docking poses of sulfonamide derivative <b>7</b> (light blue carbon atoms) and amide derivative <b>20</b> (purple carbon atom) representing the starting geometry for subsequent MD simulations. (B) Detail of the shape (highlighted with a solid yellow line) of the narrow pocket surrounding the thiophene ring delimited by the conserved disulfide bridge and the EC tip of TM2 of the hP2Y<sub>14</sub>R. Lower panel: lowest interaction energy (IE) structure extracted from the selected trajectory of 30 ns of MD simulation of the (C) <b>7</b>–hP2Y<sub>14</sub>R and (D) <b>20</b>–hP2Y<sub>14</sub>R complexes. Side chains of residues important for ligand recognition are reported as sticks (gray carbon atoms). H-bonds and salt bridges are pictured as orange and magenta dashed lines, respectively. Nonpolar hydrogen atoms and π–π stacking interactions are omitted. The binding site surface is color-coded by residue property (blue, positively charged; red, negatively charged; green, hydrophobic; cyan, polar). The interaction pattern around the carboxylate moiety is highlighted with a yellow circle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Replacement of the Carboxylate Group</h4><div class="NLM_p last">The 3-carboxylate group, in combination with an amino group on one side of the molecule and a hydrophobic portion on the other, contributed to limited aqueous solubility of some of the analogues. Therefore, we sought to substitute the carboxylate with known bioisosteres,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> beginning with a tetrazole moiety, which might also improve the bioavailability. Docking predictions suggested that the replacement of the carboxylate group with a bulkier tetrazole ring would require the scaffold to turn to fit in the narrow binding site while maintaining at least two out of the three key interactions with charged residues in TM7 and TM2 (data not shown).</div></div><div id="sec2_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Retrospective Analysis</h4><div class="NLM_p">Modeling analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_006.avi" class="ext-link">Video S2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S3</a>) of a subsequent potent analogue (primary carboxamide <b>30</b>, see below) suggested that the scaffold might adopt a different orientation in the hP2Y<sub>14</sub>R binding site stabilized by a specific H-bond interaction between the ligand’s amide moiety and the side chain of the nonconserved Asn90<sup>3.21</sup> allowing the ligand to interact with residues located in the EC tip of TM3 and EL1. As this EC region considerably differs among P2YRs receptors, it seems that derivative <b>30</b> might be a good lead compound for the design of more selective antagonists.</div><div class="NLM_p last">In an effort to explain the difference in the affinity of the new derivatives with respect to the parent compounds, we performed a MD analysis of the <b>2a</b>–hP2Y<sub>14</sub>R complex. The initial structure was extracted from a previously published MD simulation<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and subjected to the same HTMD protocol described above (see replica analysis in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S1</a>). The interaction pattern that emerged from the trajectory visualizations (see selected run in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_007.avi" class="ext-link">Video S3</a>) was consistent with the previously published MD analysis: the ligand was anchored in the binding site by its carboxylate moiety establishing salt bridges with Lys77<sup>2.60</sup> and Lys277<sup>7.35</sup> side chains and H-bonds with Tyr102<sup>3.33</sup> side chain. The 4-(4-trifluoromethyl)-phenyl moiety and the triazole ring established stable π–π stacking interactions with Tyr102<sup>3.33</sup> and His184<sup>5.39</sup>, respectively. The ligand was positioned slightly higher in the binding site with respect to the initial structure, with the piperidine ring pointing toward TM3 and anchored to the Asn90<sup>3.21</sup> side chain through a H-bond. The only noticeable difference between the MD analyses of the <b>2a</b>-hP2Y<sub>14</sub>R complex and the complexes with new derivatives is a higher conformational plasticity around the bond connecting the <i>p</i>-substituted phenyl (<b>30</b>) or the thiophene ring (<b>7</b> and <b>20</b>) to the core benzene ring. It is likely that the H-bond anchoring the piperidine ring of <b>2a</b> to TM3 restricts the conformation to ensure an optimal fit of the <i>p</i>-substituted phenyl in the narrow pocket connecting the orthosteric binding site with the EC environment. Consistent with this hypothesis, the amide derivative (<b>20</b>) displayed a higher rigidity in the linker, i.e., the region in the immediate proximity of the core benzene ring, to contribute to increased potency relative to the sulfonamide (<b>7</b>) derivative.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemical Synthesis</h3><div class="NLM_p">The synthesis of the 3-(4-phenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(aryl)benzene derivatives was performed as shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>. <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> shows the preparation by click chemistry, involving reaction of azide <b>40</b> and 4-(trifluoromethyl)phenylethylene, of a common intermediate <b>41</b> containing a methyl ester as a protected carboxylate group. A dioxaborolane group served as the preactivated attachment point for an additional aryl ring through a Suzuki reaction. The carboxylic group of <b>38</b> was first protected as the methyl ester <b>39</b>, followed by conversion of the amine functionality into an azide <b>40</b>, which was reacted with 4-(trifluoromethyl)phenylethylene in the presence of copper sulfate to afford triazole <b>41</b>. Dioxaborolane <b>42</b>, which was the substrate for the Suzuki reaction, was synthesized by coupling aryl bromide <b>41</b> with bis(pinacolato)diboron (B<sub>2</sub>pin<sub>2</sub>) under a basic condition. Two of these synthetic intermediates, i.e., <b>41</b> and <b>42</b>, were deprotected to yield truncated analogues <b>37b</b> and <b>37a</b>, respectively, for biological evaluation. Thus, the dioxoborolane group of <b>42</b> was converted to a hydroxyl group under basic conditions to obtain <b>37a</b>.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, CH<sub>3</sub>OH, rt, 15 h, 98%; (b) PTSA, NaNO<sub>2</sub> then NaN<sub>3</sub>, H<sub>2</sub>O, CH<sub>3</sub>CN, rt, 15 h, 63%; (c) 4-ethynyltrifluorotoluene, sodium ascorbate, CuSO<sub>4</sub>, THF, H<sub>2</sub>O, rt, 1 h, 84%; (d) LiOH, MeOH, 60 °C, 2 h, 76%; (e) B<sub>2</sub>pin<sub>2</sub>, KOAc, PdCl<sub>2</sub>(dppf), dioxane, 70 °C, 15 h, 87%; (f) (i) 10% NaOH, H<sub>2</sub>O<sub>2</sub>, THF, 0 °C, 0.5 h; (ii) KOH, MeOH, water, 50 °C, 5 h, 86%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows the preparation of a series of aryl bromides <b>55</b>–<b>78</b> having either a carboxamide or sulfonamide as the terminal group or as a linker to an extended aminoalkyl (or hydroxyalkyl) chain. The amide and sulfonamide groups were intended for increasing aqueous solubility. For the synthesis of lead compound <b>2a</b>, <i>tert</i>-butyl 4-(4-bromophenyl)piperidine-1-carboxylate (<b>55</b>) was prepared from compound <b>43</b>, which was synthesized according to a previously reported procedure.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> To achieve the synthesis of various aryl bromides <b>56</b>–<b>78</b>, three different strategies were employed. Thiophene and benzenesulfonyl chlorides were converted to sulfonamides (<b>56</b>–<b>63</b>) in the presence of triethylamine. Compound <b>64</b> was obtained by reductive amination of 5-bromothiophene-2-carbaldehyde (<b>46</b>).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The carboxamides <b>65</b>–<b>78</b> were obtained with two different methods; compounds <b>65</b>–<b>73</b> were synthesized from the carboxylic acids <b>47</b>–<b>53</b> under HATU coupling conditions, while 4-bromobenzoyl chloride (<b>54</b>) was reacted with an appropriate amine to afford compounds <b>74</b>–<b>78</b>. <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> shows the preparation of the condensed compounds through a Suzuki reaction with dioxaborolane <b>42</b> and aryl bromides <b>55</b>–<b>78</b> to afford compounds <b>79</b>–<b>102</b>, which were then deprotected by one of three methods to yield compounds <b>2a</b> and <b>4</b>–<b>34</b>. In most cases, the hydrolysis of the methyl ester was accomplished using potassium hydroxide, followed by removal of a <i>N</i>-Boc group, when present on R, with trifluoroacetic acid. However, compounds <b>25</b> and <b>26</b> were obtained by deprotection using AlCl<sub>3</sub> and Me<sub>2</sub>S because these two compounds were unstable in basic environments.</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, DCM, 0 °C, 0.5 h, 70%; (b) amine, Et<sub>3</sub>N, DCM, rt, 78–99%; (c) <i>N</i>-Boc piperazine, Et<sub>3</sub>N, NaBH(OAc)<sub>3</sub>, DCE, rt, 15 h, 90%; (d) amine, HATU, DIPEA, DMF, rt, 14–88%; (e) amine, Et<sub>3</sub>N, CH<sub>3</sub>CN, 0 °C, 73–99%.</p></p></figure><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) PdCl<sub>2</sub>(dppf), 2 M Na<sub>2</sub>CO<sub>3</sub>, DME, 50 °C, 25–73%; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub> (KOAc for <b>79</b>), DMF, H<sub>2</sub>O, 11–43%; (c) KOH, MeOH, H<sub>2</sub>O, 50–70 °C, 30–99%; (d) TFA, rt, 34–92%; (e) AlCl<sub>3</sub>, Me<sub>2</sub>S, rt, 15 h, 29–61%. The methyl esters of <b>79</b>–<b>102</b> were hydrolyzed using KOH to give compound <b>4</b>, <b>6</b>, <b>8</b>, <b>10</b>, <b>11</b>, <b>13</b>, <b>14</b>, <b>16</b>, <b>17</b>, <b>19</b>, <b>21</b>, and <b>27</b>–<b>34</b>. The <i>N</i>-Boc groups of compounds <b>103</b>, <b>4</b>, <b>6</b>, <b>8</b>, <b>11</b>, <b>14</b>, <b>17</b>, <b>19</b>, and <b>21</b> were removed with TFA to give compounds <b>2a</b>, <b>5</b>, <b>7</b>, <b>9</b>, <b>12</b>, <b>15</b>, <b>18</b>, <b>20</b>, and <b>22</b>, respectively. Compounds <b>23</b> and <b>25</b>–<b>26</b> were obtained using AlCl<sub>3</sub> and Me<sub>2</sub>S.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> shows the preparation of tetrazole analogues <b>35</b>, <b>36a</b>–<b>c</b>, and <b>37c</b> of compounds <b>2a</b>, <b>20</b>, <b>30</b>, <b>34</b>, and <b>37b</b> through benzonitrile precursors <b>108</b>–<b>112</b>. The carboxylic acid was converted into a carboxamide group and then into a nitrile group with trifluoroacetic anhydride. The conversion of the nitriles <b>108</b>–<b>112</b> to tetrazoles <b>113</b>, <b>114</b>, <b>36c</b>, <b>36b</b>, and <b>37c</b>, respectively, was accomplished using NaN<sub>3</sub> in the presence of catalytic CuSO<sub>4</sub>·5H<sub>2</sub>O.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Removing the <i>N</i>-Boc protecting group of <b>113</b>–<b>114</b> gave tetrazole derivatives <b>35</b> and <b>36a</b>.</div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0010.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NH<sub>4</sub>Cl, HATU, DIPEA, DMF, rt, 1 h, 80–99%; (b) (CF<sub>3</sub>CO)<sub>2</sub>O, Et<sub>3</sub>N, DCM, 0 °C to rt, 1 h, 64–76%; (c) TMSOTf, anisole, μW, 140 °C, 1 h, 80%; (d) CuSO<sub>4</sub>·5H<sub>2</sub>O, NaN<sub>3</sub>, DMSO, 140 °C, 6 h, 69–99%; (e) TFA, rt, 10 min; 44–74%.</p></p></figure><div class="NLM_p">As previously described, the fluorescent antagonist <b>3a</b> was prepared from the primary amino intermediate <b>3d</b>. As a model compound to examine the effect of distal changes on the affinity in this chain-extended series, we prepared the terminal <i>N</i>-acetyl derivative <b>3e</b> as described in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Dioxaborolane <b>119</b> was synthesized according to literature procedures reported.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Suzuki reaction of <b>119</b> with compound <b>55</b> yielded <b>120</b>, followed by removal of the ethyl ester and <i>N</i>-Boc protecting groups to provide <b>1</b>. This route to prototypical P2Y<sub>14</sub>R antagonist <b>1</b> was improved over the previous route<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> by avoiding catalytic reduction as the final step. The triazole linker leading to the fluorescent analogue was obtained following the previously reported procedure.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The ethyl ester of <b>123</b> was hydrolyzed under basic condition to afford <i>N</i>-Boc derivative <b>3c</b>, and then the free amine derivative <b>3d</b> was obtained by acid treatment. The free amine <b>3d</b> was masked with an acetyl group to give the acetyl derivative <b>3e</b>.</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0011.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, rt, 3 h, 80%; (b) (i) MsCl, Et<sub>3</sub>N, DCM, 0 °C, 5 min; (ii) NaN<sub>3</sub>, NBu<sub>4</sub>Br, H<sub>2</sub>O, 100 °C, 15 h, 59%; (c) KOH, MeOH, water, 50 °C, 10 h; (d) (i) Tf<sub>2</sub>O, pyr, −78 °C to rt, 1 h; (ii) B<sub>2</sub>pin<sub>2</sub>, PdCl<sub>2</sub>(dppf), KOAc, dioxane, 85 °C, 4 h, 61%; (e) <i>tert</i>-butyl 4-(4-bromophenyl)piperidine-1-carboxylate <b>55</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 5 h, 59%; (f) TFA/THF = 1:1, rt, 1 h, 93%; (g) 6-bromohexyne-1, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 15 h, 70%; (h) <b>117</b>, CuSO<sub>4</sub>·5H<sub>2</sub>O, sodium ascorbate, DCM/<i>t</i>BuOH/H<sub>2</sub>O = 1:1:1, rt, 15 h, 58%; (i) LiOH, MeOH, 80 °C, 15 h, 76%; (j) TFA, rt, 30 min, 78%; (k) Ac<sub>2</sub>O, pyr, rt, 1 h, 81%.</p></p></figure><div class="NLM_p last">The truncated derivative <b>37d</b> for biological evaluation was obtained by deprotection of synthetic intermediate <b>118</b>. The synthetic procedures for truncated derivatives <b>37</b> are described in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Biological Characterization</h3><div class="NLM_p">The flow cytometric assay for measuring affinity at the hP2Y<sub>14</sub>R expressed in CHO cells was based on inhibition of cell surface binding of fluorescent antagonist <b>3a</b>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The procedure was further adapted here to a more efficient 96-well format to increase the throughput. The IC<sub>50</sub> values determined for prototypical P2Y<sub>14</sub>R antagonist <b>1</b> using the two method variations were nearly identical (8.0 ± 0.4 nM using the 96-well format compared to 6.0 ± 0.1 nM reported previously).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Representative fluorescent binding inhibition curves are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fluorescent assay of specific hP2Y<sub>14</sub>R binding of selected 3-(4-trifluorophenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(aryl) derivatives. AlexaFluor488-labeled tracer <b>3a</b> binding was determined by flow cytometry using P2Y<sub>14</sub>R-expressing CHO cells. (A) 5-Thienyl <b>12</b> and <b>20</b>, 5-furyl <b>24</b>, and primary 4-carboxamidophenyl <b>30</b> derivatives. (B) Primary 4-substituted carboxamides with 5-pyridyl <b>27</b> and <b>28</b> and 5-pyrimidyl <b>29</b> aromatic groups. (C) <i>N</i>-Alkyl elaboration of a potent 4-carboxamidophenyl congener <b>30</b>: Me, <b>31</b>; Et, <b>32</b>; <i>n</i>-Pr, <b>33</b>; <i>tert</i>-Bu, <b>34</b>. The IC<sub>50</sub> values are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The most potent free amino derivative in the 2,5-disubstituted thiophene series was <b>20</b> (IC<sub>50</sub> 169 nM, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), which had a three-methylene spacer adjacent to a terminal primary amine and was anchored on the thiophene moiety through an amide linker. In the series of linear amino derivatives, carboxamides <b>19</b> and <b>20</b> were 4-fold more potent than the corresponding sulfonamides <b>6</b> and <b>7</b>. However, in the series of cyclic amines, thienyl amide <b>22</b> was 2-fold less potent than thienyl sulfonamide <b>12</b>. The effect of the linear alkyl chain length and the terminal functionality was probed in the sulfonamide series. The optimal chain length among the primary amines was found to be three methylenes in <b>7</b> (IC<sub>50</sub> 756 nM), with the primary amino analogues containing two and four methylenes, <b>5</b> and <b>9</b>, respectively, being 3-fold less potent in P2Y<sub>14</sub>R binding. When isolated, the Boc intermediates used for preparation of the free amines were tested for P2Y<sub>14</sub>R affinity. In each case of sulfonamide and amide derivatives with linear amine moieties, the free amine derivative, i.e., <b>5</b>, <b>7</b>, <b>9</b>, and <b>20</b>, was more potent than the corresponding Boc-protected amine. For example, the 3-aminopropyl congener <b>7</b> was 12-fold more potent than its Boc derivative <b>6</b> (IC<sub>50</sub> 9.33 μM), but the difference was less pronounced for amines <b>5</b> and <b>9</b>. Similarly, for cyclic amines, the Boc group reduced potency, e.g., piperazine sulfonamide <b>15</b> (IC<sub>50</sub> 1.64 μM) compared to its inactive Boc derivative <b>14</b> (IC<sub>50</sub> > 50 μM). However, none of the piperazine free amine derivatives surpassed micromolar IC<sub>50</sub> values. Methylpiperazine <b>18</b> was the most potent in the group consisting of piperazine sulfonamide <b>12</b>, methylpiperazine <b>18</b>, and piperazine carboxamide <b>22</b>, while its Boc derivative <b>17</b> was 34-fold weaker. A primary alcohol derivative <b>10</b> was 4-fold less potent than the corresponding primary aminobutyl congener <b>9</b>.</div><div class="NLM_p">In a comparison of aryl groups, primary phenyl <i>p</i>-sulfonamide <b>16</b> was 10-fold more potent than the corresponding 2,5-disubstituted thienyl sulfonamide <b>13</b>. However, <i>p</i>-phenyl sulfonamido-piperazine derivative <b>15</b> was similar in potency to the corresponding thienyl derivative <b>12</b>. In the series of linear amino derivatives, 2,5-disubstituted thiophene carboxamides, e.g., Boc-protected <b>19</b> and free amine <b>20</b>, were more potent than the corresponding thiophene sulfonamides <b>6</b> and <b>7</b>, respectively. The phenyl primary sulfonamide <b>16</b> and phenyl primary carboxamide <b>30</b> displayed a sub-micromolar affinity (IC<sub>50</sub> = 608 and 269 nM, respectively) and could serve as leads for additional analogues. Unlike most analogues in the initial set, the compounds are not Zwitterionic. It is recognized that pharmaceutical drugs include fewer Zwitterionic compounds than acidic, basic, or neutral compounds because of generally low oral bioavailability.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">The affinities of isomers of the most potent thiophene derivative <b>20</b> were compared. The alternate substitution patterns of thiophene, e.g., primary 3-aminopropyl derivatives <b>25</b> and <b>26</b>, were less potent than the 2,5-disubstituted lead compound. Similarly, sulfur replacement with oxygen in furan <b>23</b> reduced affinity 39-fold. The furan primary carboxamide <b>24</b> was 10-fold less potent than the corresponding <i>p</i>-phenyl substituted carboxamide <b>30</b>. The corresponding thiophene 5-carboxamide related to <b>20</b> was not prepared.</div><div class="NLM_p">The introduction of nitrogen atoms in the <i>p</i>-substituted phenyl ring of <b>30</b> reduced the potency by 2- to 5-fold in <b>27</b>–<b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Compounds <b>31</b>–<b>34</b> are homologated <i>N</i>-alkyl amides based on potent antagonist <b>30</b>. The <i>N</i>-methyl <b>31</b> and ethyl <b>32</b> analogues were roughly equipotent to the parent carboxamide, but the <i>N</i>-propyl analogue <b>33</b> was less potent (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The <i>N</i>-<i>tert</i>-Bu analogue <b>34</b> was 15-fold less potent than the primary carboxamide <b>30</b>.</div><div class="NLM_p">Carboxylate bioisosteres were considered,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> although this group was deemed essential in previous studies of P2Y<sub>14</sub>R antagonists. The substitution of the carboxylate of <b>2a</b><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with a tetrazole moiety in <b>35</b> reduced the affinity by 30-fold. However, the difference between carboxylate and tetrazole isostere was less in comparing <b>20</b> and <b>36a</b> (7-fold). In comparing potent carboxamide analogues <b>30</b> and <b>34</b> and their tetrazole derivatives <b>36b</b> and <b>36c</b>, the affinity was either reduced 9-fold or 3-fold, respectively. The truncated derivatives <b>37</b> were greatly reduced in potency in the triazole series, but less so in the naphthalene series (<b>37d</b>) with an IC<sub>50</sub> value of 3.2 μM. Thus, carboxylic acid <b>37d</b> is a relatively simple, novel molecule with moderate affinity at the P2Y<sub>14</sub>R.</div><div class="NLM_p">The aryl ring (green in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) contribution to the overall affinity was explored with compound <b>37a</b> in which that ring is replaced with a hydroxyl group. Compared to phenylcarboxamide <b>30</b>, compound <b>37a</b> was 60-fold weaker in its interaction with the P2Y<sub>14</sub>R. Thus, this aryl ring provides a major contribution to the P2Y<sub>14</sub>R binding.</div><div class="NLM_p last">To further explore the interaction of long chains, including amino functionalized chains with the outer P2Y<sub>14</sub>R regions, compounds <b>3c</b> (alkyne), <b>3c</b> (Boc-protected), <b>3d</b> (primary amine), and <b>3e</b> (acetamido derivative), all related to the naphthalene-containing fluorescent probe <b>3a</b>, were prepared and tested. The order of affinity in the fluorescent P2Y<sub>14</sub>R assay was <b>3d</b><i>></i><b>3e</b><i>></i><b>3c</b><i>></i><b>3b</b>, and all three congeners were more potent than the most potent triazole derivatives prepared in the present study, i.e., <b>20</b> and <b>30</b>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Off-Target and ADME-tox Testing</h3><div class="NLM_p">Off-target interactions of selected antagonists (<b>1</b>, <b>2a</b>, <b>3d</b>, <b>3e</b>, <b>16</b>, <b>20</b>, <b>30</b>, and <b>37d</b>) were determined at 45 receptors, transporters, and ion channels by the Psychoactive Drug Screening Program (PDSP).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The previously reported<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a><i>K</i><sub>i</sub> values (μM) of <b>1</b> were 6.79 (D<sub>3</sub> dopamine receptor) and 2.75 (δ-opioid receptor); of <b>2a</b>, α<sub>2C</sub>-adrenergic, 1.32; H<sub>1</sub>-histamine, 0.17. Compounds <b>16</b>, <b>20</b>, and <b>30</b> also bound to the δ-opioid receptor with <i>K</i><sub>i</sub> values (μM) of 3.26 ± 0.64, 0.941, and 1.47, respectively. Carboxamide <b>30</b> did not bind appreciably to any other sites in the comprehensive PDSP screen, and truncated naphthalene derivative <b>37d</b> bound only to NET (5.8 μM). Therefore, this compound series is generally not promiscuous, i.e., PAINS compounds.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">Curiously, the long-chain primary amino derivative <b>3d</b> displayed extensive off-target interactions with biogenic amine receptors. The <i>K</i><sub>i</sub> values (μM) of <b>3d</b> that were below 10 μM were 5HT<sub>1A</sub>, 5.65; 5HT<sub>2A</sub>, 8.4; α<sub>1A</sub>, 1.77; α<sub>1B</sub>, 1.11 ± 0.37; α<sub>1D</sub>, 1.27; α<sub>2A</sub>, 0.028; α<sub>2B</sub>, 0.984; D<sub>1</sub>, 7.60; D<sub>2</sub>, 8.21; D<sub>3</sub>, 1.60; D<sub>5</sub>, 1.07; H<sub>2</sub>, 0.365. Also, interactions of <b>3d</b> with the dopamine transporter (DAT, 1.07 μM), norepinephrine transporter (NET, 1.87 μM), and sigma<sub>2</sub> receptor (1.2 μM) were observed. We hypothesize that the terminal amino group interacts with the common amine-binding residue in biogenic amine receptors (D2.50) because the off-target activities at α-adrenergic and dopamine receptors were greatly reduced in the terminal <i>N</i>-acetyl derivative <b>3e</b>. Thus, the nine interactions below 10 μM of <b>3c</b> were (<i>K</i><sub>i</sub>, μM) 5HT<sub>2B</sub>, 2.78; 5HT<sub>6</sub>, 7.9; ; α<sub>2A</sub>, 5.28; β<sub>3</sub>, 2.36 ± 0.48; H<sub>2</sub>, 0.829; H<sub>3</sub>, 2.77; KOR, 4.06; TSPO, 5.60; SERT, 2.08. However, compound <b>20</b> was also a primary amine, but it did not display extensive off-target interactions with biogenic amine receptors.</div><div class="NLM_p">cLogP (calculated logarithm of partition coefficient between <i>n</i>-octanol and water), a good predictor of solubility, is shown for each analogue in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Based on the range of cLogP values, the carboxylate analogues <b>4</b>–<b>34</b> are expected to have more favorable (lower) hydrophobicity (0.40–4.90) than compound <b>1</b> (6.18) and its <i>N</i>-alkylated congeners <b>3b</b>–<b>3e</b> (4.04–7.54). cLogS and cLogD at pH 7.4 of <b>30</b> were calculated to be −3.24 and 1.51, respectively, indicating predicted moderate aqueous solubility when ionized at physiological pH (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S4</a>). However, <b>20</b> as a Zwitterion has increased solubility predicted at pH extremes compared to physiological pH, with cLogS and cLogD at pH 7.4 of −6.5 and 1.9.</div><div class="NLM_p">The tetrazole derivatives <b>35</b>, <b>36</b> and <b>37c</b> displayed intermediate cLogP values (2.97–4.40). The effect, based on cLogD and cLogS (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Figure S4</a>), of replacing a carboxylate group with tetrazole was generally to reduce predicted solubility at pH 7.4 (ΔcLogD): for <b>2a</b> and <b>35</b> (−0.3); for <b>20</b> and <b>36a</b> (−0.3); and for <b>30</b> and <b>36b</b> (−1.3).</div><div class="NLM_p">Predicted pharmacokinetic parameters for some of the more potent compounds <b>7</b>, <b>16</b>, <b>20</b>, <b>25</b>, <b>26</b>, <b>29</b>–<b>32</b>, and <b>35</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S3</a>)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> indicated satisfactory intestinal absorption (63–100%) and unbound fraction (18–27%), with no AMES toxicity, renal OGT substrate, or PGP I inhibition properties. However, for all compounds, PGP substrate properties were predicted. Compounds <b>7</b>, <b>16</b>, <b>20</b>, <b>25</b>, <b>26</b>, <b>32</b>, and <b>35</b> were predicted as CYP 3A4 substrates.</div><div class="NLM_p last">The pharmacokinetics of a representative non-Zwitterionic analogue <b>30</b> was studied in the male Sprague–Dawley rat following oral administration (1–10 mg/kg) indicated a lack of bioavailability. Consistent with this finding, <b>30</b> lacked Caco-2 cell permeability (P<sub>app</sub>, apical to basal, was 0.0 × 10<sup>–6</sup> cm/s). Its plasma half-life when administered i.v. (0.5 mg/kg) was short (0.20 ± 0.14 h, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">Table S4</a>). Other preclinical assays were performed as reported.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Compound <b>30</b> did not inhibit CYP450 enzymes (IC<sub>50</sub> ≥ 30 μM at 1A2, 2C9, 2C19, 2D6, and 3A4 isoforms) and was stable in simulated gastric and intestinal fluids (100% remaining after 240 min) and in plasma of three species (94–100% remaining at 120 min; human, rat, and mouse). Compound <b>30</b> was not cytotoxic to Hep-G2 cells (CC<sub>50</sub> > 30 μM). Its aqueous solubility was 138 ± 4 μg/mL, and the IC<sub>50</sub> at hERG was 0.166 μM.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14710" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14710" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have expanded the SAR of heterocyclic P2Y<sub>14</sub>R antagonists, guided by computational methods. Although oral bioavailability was lacking in the one compound tested (<b>30</b>), it displayed high aqueous solubility and stability in simulated body fluids. We now know that <b>30</b> has a hERG liability, but this can likely be overcome with additional structural modification.</div><div class="NLM_p">In the previous modeling analysis,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> the required aryl carboxylate of this chemical series interacted with a pair of Lys residues in TMs 2 and 7 that are specific to the G<sub>i</sub>-coupled subfamily of P2YRs. Furthermore, a Tyr residue, which forms a putative π interaction with the naphthalene ring of <b>1</b>, was predicted to shift to the vicinity of the triazole ring in its bioisosteric equivalent in <b>2a</b>, to form a stabilizing π−π interaction.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This interaction was not observed for a synthesized bioisosteric alkyne derivative, which was suggested as an alternative scaffold, consistent with the greater affinity of the triazole-containing scaffold compared to the alkyne. We have, therefore, conserved the CF<sub>3</sub>-phenyl ring of <b>2a</b> and the triazole moiety in the present set of analogues and concentrated on modification of the terminal phenyl-piperidine rings of <b>1</b> and <b>2a</b>. In addition, we tried to find a bioisosteric replacement for the carboxylate group on the central ring.</div><div class="NLM_p">Thus, a modeling pipeline based on a hP2Y<sub>14</sub>R homology model guided analogue design. Many but not all predictions based on this approach were confirmed experimentally. In particular, we were able to identify (i) the bioisostere replacement of the <i>p</i>-substituted phenyl ring with other 5- and 6-membered rings, (ii) suitable linker and spacer connecting the aromatic moiety to the amine function, and, to some degree, (iii) the optimal spacer length and geometry between the linker and the amine moiety. With respect to the latter aspect, the experimental data suggested that some degree of flexibility either in the linker or in the spacer is required. In particular, if the amine was enclosed in a rigidified spacer, e.g., a piperazine ring, more flexible linkers yielded more active compounds (<b>12</b> and <b>18</b> more active than <b>22</b>). Conversely, when the amine was attached to a flexible alkyl spacer, the more rigidified the linker, the more potent the corresponding compound (e.g., <b>7</b> vs <b>20</b>).</div><div class="NLM_p">In conclusion, we have used a computational pipeline to predict the P2Y<sub>14</sub>R interactions of 5-aryl ring substituents and functional groups, which were largely consistent with empirical results. The substitution of a key carboxylate group on the core phenyl ring with tetrazole or truncation of the 5-aryl group reduced affinity. The most potent antagonists, using a fluorescent assay, were a thienyl-linked primary 3-aminopropyl congener <b>20</b> and a phenyl <i>p</i>-carboxamide <b>30</b>.</div><div class="NLM_p last">The activity of a P2Y<sub>14</sub>R antagonist in reducing neutrophil motility and inflammation, a predictor of utility of selective P2Y<sub>14</sub>R antagonists in inflammatory diseases, has been shown. Future studies will focus on the use of these leads to discovery of more drug-like molecules for application to animal models.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Molecular Modeling</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Protein Preparation</h4><div class="NLM_p last">In this study, we used a previously published<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> hP2Y<sub>14</sub>R homology model based on the high-resolution X-ray structure of an agonist-bound hP2Y<sub>12</sub>R (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PXZ">4PXZ</a>)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> that was further refined using molecular dynamics (MD) simulation. The structure extracted from the MD trajectory was prepared for docking using the Protein Preparation Wizard tool implemented in the Schrödinger suite<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> by removing water molecules and minimizing protein side chains with OPLS3 (convergence criterion: heavy atoms RMSD < 0.30 Å). The protonation state of histidine residues was determined according to H-bond patterns analysis. In particular, His184/217/280 were protonated on the N<sup>δ</sup>, and His264 was considered doubly protonated.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Docking</h4><div class="NLM_p last">Ligand structures were built using the Builder tool implemented in the Schrödinger suite<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and minimized using the OPLS3 force field. Molecular docking was performed with Glide,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> by selecting the barycenter of PPTN in the MD-refined hP2Y14 homology model as the grid center (inner box: 10 × 10 × 10 Å; outer box extended by 30 Å in each direction from the inner box). Docking was performed considering the protein binding site residues rigid by using the standard precision (SP) scoring function and generating five poses for each ligand. Selection of hits were performed by visual inspection of optimal ligand–protein interactions rather than on the basis of docking scores. In particular, for each ligand we selected the docking poses establishing H-bond and salt-bridge interactions with the Tyr102<sup>3.33</sup>, Lys77<sup>2.60</sup>, and Lys277<sup>7.35</sup> side chains.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Molecular Dynamics</h4><div class="NLM_p last">MD system setup, equilibration, and production were performed with the HTMD<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> module (Acellera, Barcelona Spain, version 1.5.4). The ligand–protein complexes were embedded into an 80 × 80 Å 1-palmitoyl-2-oleoyl-<i>sn</i>-glycero-3-phosphocholine (POPC) membrane leaflet generated through the VMD Membrane Plugin tool.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Overlapping lipids (within 0.6 Å) were removed upon protein insertion, and the systems were solvated with TIP3P<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> water and neutralized by Na<sup>+</sup>/Cl<sup>–</sup> counterions (final concentration 0.154 M). MD simulations were carried out with the ACEMD engine (Acellera, version 2016.10.27)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> using the CHARMM36<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a>/CGenFF(3.0.1)<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> force fields for lipid and protein and ligand atoms, respectively, and periodic boundaries conditions. Ligand parameters were retrieved from the ParamChem service (<a href="https://cgenff.paramchem.org" class="extLink">https://cgenff.paramchem.org</a>, accessed 07/2017, version 1.0.0) with no further optimization. The systems were equilibrated through a 5000-step minimization followed by 40 ns of MD simulation in the NPT ensemble by applying initial constraints (0.8 for the ligand atoms, 0.85 for α carbon atoms, and 0.4 for the other protein atoms) that were linearly reduced after 20 ns. During the equilibration procedure, the temperature was maintained at 310 K using a Langevin thermostat (damping constant = 1 ps<sup>–1</sup>), and the pressure was maintained at 1 atm using a Berendensen barostat. Bond lengths involving hydrogen atoms were constrained using the M-SHAKE<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> algorithm. The equilibrated systems were subjected to 30 ns of unrestrained MD simulations run in triplicate for each ligand–protein complex (NVT ensemble, time step = 2 fs, damping constant = 0.1 ps<sup>–1</sup>). Long-range Coulomb interactions were handled using the particle mesh Ewald summation method (PME)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> with grid size rounded to the approximate integer value of cell wall dimensions. A nonbonded cutoff distance of 9 Å with a switching distance of 7.5 Å was used. All simulations were run on three NVIDIA GeForce GTX (two 980Ti and one 1080).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> MD Trajectory Analysis</h4><div class="NLM_p last">MD trajectory analysis was performed with an in-house script exploiting the NAMD 2.10<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a><i>mdenergy</i> function and the RMSD trajectory tool (RSMDTT) implemented in VMD.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> All simulations were run in triplicate and selection of representative trajectories and of lowest interaction energy (IE) ligand–protein complexes were based upon the total ligand–protein interaction energy (IE<sub>tot</sub>) expressed as the sum of van der Waals (IE<sub>vdW</sub>) and electrostatic (IE<sub>ele</sub>) contribution as previously described.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemical Synthesis</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Reagents and Instrumentation</h4><div class="NLM_p">All reagents and solvents were purchased from Sigma-Aldrich (St. Louis, MO), Ark Pharm, Inc. (Libertyville, IL; 6-bromonicotinic acid, 5-bromopicolinic acid and 5-bromopyrazine-2-carboxylic acid), and Enamine LLC (Cincinnati, OH; 5-bromopyrazine-2-carboxlic acid). <sup>1</sup>H NMR spectra were obtained with a Bruker 400 spectrometer using CDCl<sub>3</sub>, CD<sub>3</sub>OD, and DMSO-<i>d</i><sub>6</sub> as solvents. Chemical shifts are expressed in δ values (ppm) with tetramethylsilane (δ 0.00) for CDCl<sub>3</sub> and water (δ 3.30) for CD<sub>3</sub>OD. NMR spectra were collected with a Bruker AV spectrometer equipped with a z-gradient [<sup>1</sup>H,<sup>13</sup>C,<sup>15</sup>N]-cryoprobe. TLC analysis was carried out on glass sheets precoated with silica gel F254 (0.2 mm) from Sigma-Aldrich. The purity of final compounds was checked using a Hewlett–Packard 1100 HPLC equipped with a Zorbax SB-Aq 5 μm analytical column (50 × 4.6 mm; Agilent Technologies Inc., Palo Alto, CA). Mobile phase: linear gradient solvent system, 5 mM tetrabutylammonium dihydrogen phosphate–CH<sub>3</sub>CN from 100:0 to 0:100 in 15 min; the flow rate was 0.5 mL/min. Peaks were detected by UV absorption with a diode array detector at 230, 254, and 280 nm. All derivatives tested for biological activity showed >95% purity by HPLC analysis (detection at 254 nm). Low-resolution mass spectrometry was performed with a JEOL SX102 spectrometer with 6 kV Xe atoms following desorption from a glycerol matrix or on an Agilent LC/MS 1100 MSD, with a Waters (Milford, MA) Atlantis C18 column. High-resolution mass spectroscopic (HRMS) measurements were performed on a proteomics optimized Q-TOF-2 (MicromassWaters) using external calibration with polyalanine, unless noted. Observed mass accuracies are those expected based on known instrument performance as well as trends in masses of standard compounds observed at intervals during the series of measurements. Reported masses are observed masses uncorrected for this time-dependent drift in mass accuracy. tPSA (total polar surface area) was calculated using ChemDraw Professional (PerkinElmer, Boston, MA, v. 16.0). cLogP was calculated as reported.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Simulated pharmacokinetic parameters were calculated as reported.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Compound <b>3b</b> was prepared as reported.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> 4-(4-(3-Carboxy-6-(4-(trifluoromethyl)phenyl)naphthalen-1-yl)phenyl)piperidin-1-ium Chloride (<b>1</b>)</h5><div class="NLM_p last">The mixture of compound <b>121</b> (55 mg, 0.109 mmol) in methanol (2 mL) and water (1 mL) was added potassium hydroxide (30 mg, 0.545 mmol), and then this reaction mixture was stirred at 70 °C for 15 h. After being neutralized with 1 M HCl, a white precipitate was collected by filtration, washed with water, and dried under vacuum to afford compound <b>1</b> as a white solid (40 mg, 77%).</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> 4-(4-(1-(4-(1-(6-((<i>tert</i>-Butoxycarbonyl)amino)hexyl)-1<i>H</i>-1,2,3-triazol-4-yl)butyl)piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoic Acid (<b>3c</b>)</h5><div class="NLM_p last">A mixture of compound <b>123</b> (30 mg, 36.3 μmol) and lithium hydroxide (9 mg, 0.363 mmol) in methanol (5 mL) was stirred at 80 °C for 15 h. After cooling at room temperature, this mixture was neutralized with 1 N HCl until pH was 5–6. The slightly acidic mixture was evaporated under reduced pressure and purified by silica gel column chromatography (dichloromethane/methanol = 5:1) to afford compound <b>3c</b> (22 mg, 76%) as a white solid; HPLC purity 95% (<i>R</i><sub>t</sub> = 13.99 min); <sup>1</sup>H NMR (400 MHz, CD<sub><b>3</b></sub>OD) δ 8.62 (s, 1H), 8.37 (s, 1H), 8.04 (s, 1H), 7.95 (d, <i>J</i> = 8.04 Hz, 2H), 7.90 (d, <i>J</i> = 8.80 Hz, 1H), 7.82 (s, 1H), 7.78–7.74 (m, 3H), 7.49 (d, <i>J</i> = 7.44 Hz, 2H), 7.35 (d, <i>J</i> = 7.72 Hz, 2H), 4.36 (t, <i>J</i> = 7.08 Hz, 2H), 3.65 (d, <i>J</i> = 11.6 Hz, 2H), 3.19 (t, <i>J</i> = 7.68 Hz, 2H), 3.08–2.97 (m, 4H), 2.94–2.91 (m, 1H), 2.83 (t, <i>J</i> = 7.06 Hz, 2H), 2.09 (s, 4H), 1.90–1.80 (m, 6H), 1.43 (s, 9H), 1.33–1.31 (m, 6H); MS (ESI, <i>m</i>/<i>z</i>) 798.4 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>46</sub>H<sub>55</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> 798.4206, found 798.4217 [M + 1]<sup>+</sup>.</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> 4-(4-(1-(4-(1-(6-Aminohexyl)-1<i>H</i>-1,2,3-triazol-4-yl)butyl)piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoic Acid (<b>3d</b>)</h5><div class="NLM_p last">A mixture of compound <b>3c</b> (22 mg, 27.6 μmol) in trifluoroacetic acid (0.2 mL) was stirred at room temperature for 30 min. The solvent was evaporated with methanol under reduced pressure. The residue was purified by semipreparative HPLC (0.1% TFA–water/acetonitrile = 20:80 to 80:20 in 40 min) to afford compound <b>3d</b> (15 mg, 78%) as a white solid; <i>R</i><sub>t</sub> = 32.7 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 11.84 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) 8.78 (s, 1H), 8.46 (s, 1H), 8.04–7.99 (m, 4H), 7.96–7.93 (m, 1H), 7.84–7.78 (m, 3H), 7.55 (d, <i>J</i> = 8.24 Hz, 2H), 7.50 (d, <i>J</i> = 8.24 Hz, 2H), 4.42 (t, <i>J</i> = 7.04 Hz, 2H), 3.76 (d, <i>J</i> = 11.96 Hz, 2H), 3.29–3.23 (m, 4H), 3.21–3.14 (m, 2H), 3.11–3.05 (m, 1H), 2.93 (t, <i>J</i> = 7.56 Hz, 2H), 2.83 (t, <i>J</i> = 7.24 Hz, 2H), 2.28 (d, <i>J</i> = 14.96 Hz, 2H), 2.15–2.05 (m, 1H), 2.00–1.82 (m, 5H), 1.71–1.64 (m, 2H), 1.50–1.37 (m, 4H); MS (ESI, <i>m</i>/<i>z</i>) 698.4 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>41</sub>H<sub>47</sub>N<sub>5</sub>O<sub>2</sub>F<sub>3</sub> 698.3682, found 698.3689 [M + 1]<sup>+</sup>.</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 4-(4-(1-(4-(1-(6-Acetamidohexyl)-1<i>H</i>-1,2,3-triazol-4-yl)butyl)piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoic Acid (<b>3e</b>)</h5><div class="NLM_p last">To a solution of compound <b>3e</b> (2.24 mg, 3.21 μmol) in pyridine (0.3 mL) was added acetic anhydride (10 μL), and then this reaction mixture was stirred at room temperature for 1 h. After all volatiles were evaporated under reduced pressure, the residue purified by semipreparative HPLC (10 mM triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min) to afford compound <b>3e</b> (1.9 mg, 81%) as a white solid; <i>R</i><sub>t</sub> = 34.2 min; HPLC purity 97% (<i>R</i><sub>t</sub> = 13.07 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) 8.59 (s, 1H), 8.37 (s, 1H), 8.04 (s, 1H), 8.01–7.98 (m, 3H), 7.84–7.80 (m, 4H), 7.50 (d, <i>J</i> = 7.96 Hz, 2H), 7.43 (d, <i>J</i> = 7.96 Hz, 2H), 4.39 (t, <i>J</i> = 7.06 Hz, 2H), 3.50–3.43 (m, 2H), 3.15 (t, <i>J</i> = 7.02 Hz, 2H), 2.88–2.81 (m, 4H), 2.74–2.68 (m, 1H), 2.13–1.99 (m, 2H), 1.96–1.89 (m, 7H), 1.78 (m, 4H), 1.54–1.47 (m, 2H), 1.43–1.27 (m, 6H); MS (ESI, <i>m</i>/<i>z</i>) 740.4 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>43</sub>H<sub>49</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub> 740.3788, found 740.3779 [M + 1]<sup>+</sup>.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedure for Carboxylic Acid Derivative Deprotection Reactions</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Method A</h4><div class="NLM_p last">A mixture of the condensed compound (<b>41</b>, <b>79</b>–<b>90</b>, <b>92</b>, <b>95</b>–<b>102</b>, <b>118</b>, <b>121</b>, <b>123</b>; 1 equiv, 5–10 mg) and potassium hydroxide (5 equiv) in a methanol (1 mL)/water (0.5 mL) mixture was stirred at 50 °C for 15 h. This mixture was neutralized with 1 N HCl until pH 5–6. The slightly acidic mixture was evaporated under reduced pressure, and the product was purified by silica gel column chromatography (dichloromethane/methanol/acetic acid = 95:5:0.1) or semipreparative HPLC (10 mM triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min) to afford the product as a white solid. The salt form obtained was determined using NMR.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Method B</h4><div class="NLM_p last">A solution of the carboxylic derivative (<b>4</b>, <b>6</b>, <b>8</b>, <b>11</b>, <b>14</b>, <b>17</b>, <b>19</b>, <b>21</b>, <b>113</b>, <b>114</b>; 1 equiv, 5 mM) in trifluoroacetic acid was stirred at room temperature for 10 min. The solvent, with methanol added, was evaporated under reduced pressure. The residue was purified by semipreparative HPLC (10 mM triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min) to afford the compound as a white solid.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Method C</h4><div class="NLM_p">For compounds that decomposed using Method A or B, the condensed compound (<b>91</b>, <b>93</b>, <b>94</b>; 1 equiv, 5 mM) was dissolved in dimethyl sulfide and treated with aluminum chloride (15 equiv), and the reaction mixture was stirred at room temperature for 15 h. The resulting suspension was filtered through a bed of silica gel, which was washed with dichloromethane/methanol/acetic acid (100:20:1) to separate residual product from the aluminum salts. The combined solution was evaporated to dryness under reduced pressure. The residue was purified by semipreparative HPLC (triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min) to afford the product as a white solid.</div><div id="sec4_3_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 3-(5-(<i>N</i>-(2-((<i>tert</i>-Butoxycarbonyl)amino)ethyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>4</b>)</h5><div class="NLM_p last">Method A: Yield 72%; HPLC purity 99% (<i>R</i><sub>t</sub> = 13.46 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.75 (s, 1H), 8.54 (s, 1H), 8.52 (s, 1H), 8.29 (s, 1H), 8.21 (d, <i>J</i> = 8.4 Hz, 2H), 8.06 (broad s, 1H), 7.91 (d, <i>J</i> = 8.4 Hz, 2H), 7.88 (d, <i>J</i> = 4.0 Hz, 1H), 7.67 (d, <i>J</i> = 4.0 Hz, 1H), 6.84 (broad s, 1H), 3.04–3.02 (m, 2H), 2.95–2.93 (m, 2H), 1.35 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 538.1 [M + 1-Boc]<sup>+</sup>, 582.0 [M + 1 – isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>19</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 538.0831, found 538.0841 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_3_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 3-(5-(<i>N</i>-(2-Aminoethyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>5</b>)</h5><div class="NLM_p last">Method B: Yield 34%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 27.6 min; HPLC purity 95% (<i>R</i><sub>t</sub> = 10.40 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.56 (s, 1H), 8.42 (s, 1H), 8.19 (s, 1H), 8.16 (s, 1H), 8.10 (d, <i>J</i> = 8.0 Hz, 2H), 7.81 (d, <i>J</i> = 8.0 Hz, 2H), 7.72 (d, <i>J</i> = 4.0 Hz, 1H), 7.69 (d, <i>J</i> = 4.0 Hz, 1H), 6.59 (broad s, 2H; N<i>H</i><sub>2</sub>), 2.97 (t, <i>J</i> = 6.4 Hz, 4H); MS (ESI, <i>m</i>/<i>z</i>) 538.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>19</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 538.0831, found 538.0837 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 3-(5-(<i>N</i>-(3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>6</b>)</h5><div class="NLM_p last">Method A: Yield 45%; HPLC purity 98% (<i>R</i><sub>t</sub> = 14.00 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.26 (s, 1H), 8.51 (s, 1H), 8.42–8.41 (m, 2H), 8.19 (d, <i>J</i> = 8.12 Hz, 2H), 7.81 (d, <i>J</i> = 8.28 Hz, 2H), 7.68 (d, <i>J</i> = 3.92 Hz, 1H), 7.65 (d, <i>J</i> = 3.96 Hz, 1H), 3.10 (t, <i>J</i> = 6.62 Hz, 2H), 3.05 (t, <i>J</i> = 7.08 Hz, 2H), 1.70 (t, <i>J</i> = 6.90 Hz, 2H), 1.41 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 552.1 [M + 1 – Boc]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 552.0987, found 552.0979 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_3_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 3-(5-(<i>N</i>-(3-Aminopropyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>7</b>)</h5><div class="NLM_p last">Method B: Yield 73%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 30.9 min; HPLC purity 98% (<i>R</i><sub>t</sub> = 10.47 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.49 (s, 1H), 8.37 (s, 1H), 8.06–8.04 (m, 3H), 7.78 (s, 1H), 7.75 (d, <i>J</i> = 8.4 Hz, 2H), 7.60 (s, 2H), 3.12 (t, <i>J</i> = 6.8 Hz, 2H), 2.87 (t, <i>J</i> = 6.8 Hz, 2H), 1.80–1.76 (m, 2H); MS (ESI, <i>m</i>/<i>z</i>) 552.0 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 552.0987, found 552.0991 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 3-(5-(<i>N</i>-(4-((<i>tert</i>-Butoxycarbonyl)amino)butyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>8</b>)</h5><div class="NLM_p last">Method A: Yield 71%; HPLC purity 98% (<i>R</i><sub>t</sub> = 14.07 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.23 (s, 1H), 8.54 (s, 1H), 8.41 (s, 1H), 8.38 (s, 1H), 8.15 (d, <i>J</i> = 7.2 Hz, 2H), 7.78 (d, <i>J</i> = 7.6 Hz, 2H), 7.63 (s, 2H; thiophene), 3.05–3.02 (m, 4H), 1.54 (m, 4H), 1.41 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 664.1 [M – 1]<sup>−</sup>, 778.1 [M – H + TFA]<sup>−</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>29</sub>H<sub>29</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S<sub>2</sub> 664.1511, found 664.1507 [M – H]<sup>−</sup>.</div></div><div id="sec4_3_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 3-(5-(<i>N</i>-(4-Aminobutyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>9</b>)</h5><div class="NLM_p last">Method B: Yield 40%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 28.5 min; HPLC purity 95% (<i>R</i><sub>t</sub> = 10.88 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.62 (s, 1H), 8.40 (s, 1H), 8.23 (s, 1H), 8.22 (s, 1H), 8.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.88 (d, <i>J</i> = 8.4 Hz, 2H), 7.72 (d, <i>J</i> = 4.0 Hz, 1H), 7.63 (d, <i>J</i> = 4.0 Hz, 1H), 2.92 (s, 2H), 2.67 (s, 2H), 1.49 (s, 4H); MS (ESI, <i>m</i>/<i>z</i>) 566.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> S<sub>2</sub> 566.1144, found 566.1145 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 3-(5-(<i>N</i>-(4-Hydroxybutyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>10</b>)</h5><div class="NLM_p last">Method A: Yield 30%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 25.1 min; HPLC purity 97% (<i>R</i><sub>t</sub> = 12.13 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (s, 1H), 8.49 (s, 1H), 8.46 (s, 1H), 8.27 (s, 1H), 8.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.95 (t, <i>J</i> = 5.8 Hz, 1H; N<i>H</i>), 7.91 (d, <i>J</i> = 8.0 Hz, 2H), 7.83 (d, <i>J</i> = 4.0 Hz, 1H), 7.65 (d, <i>J</i> = 4.0 Hz, 1H), 4.42 (broad s, 1H; N<i>H</i>), 3.36 (merged with water peak), 2.92 (q, <i>J</i> = 6.6 Hz, 2H), 1.49–1.42 (m, 4H); MS (ESI, <i>m</i>/<i>z</i>) 567.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>22</sub>N<sub>4</sub>O<sub>5</sub>F<sub>3</sub>S<sub>2</sub> 567.0984, found 567.0981 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 3-(5-((4-(<i>tert</i>-Butoxycarbonyl)piperazin-1-yl)sulfonyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>11</b>)</h5><div class="NLM_p last">Method A: Yield 93%; HPLC purity 98% (<i>R</i><sub>t</sub> = 15.22 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.21 (s, 1H), 8.51 (s, 1H), 8.39 (s, 1H), 8.33 (s, 1H), 8.11 (d, <i>J</i> = 7.2 Hz, 2H), 7.76 (d, <i>J</i> = 8.0 Hz, 2H), 7.70 (s, 1H), 7.63 (s, 1H), 3.58 (s, 4H), 3.11 (s, 4H), 1.42 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 564.1 [M + 1-Boc]<sup>+</sup>, 608.1 [M + 1-isobutyl]<sup>+</sup>, 664.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>29</sub>H<sub>29</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S 664.1511, found 664.1523 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 3-(5-(Piperazin-1-ylsulfonyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>12</b>)</h5><div class="NLM_p last">Method B: Yield 54%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 29.3 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 12.22 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.38 (s, 1H), 8.34 (s, 1H), 8.03 (s, 1H), 7.97 (d, <i>J</i> = 8.0 Hz, 2H), 7.92 (s, 1H), 7.74 (d, <i>J</i> = 4.0 Hz, 1H), 7.72 (d, <i>J</i> = 4.0 Hz, 1H), 7.68 (d, <i>J</i> = 8.0 Hz, 2H), 3.18 (s, 4H), 3.11 (s, 4H); MS (ESI, <i>m</i>/<i>z</i>) 564.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S 564.0987, found 564.0980 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 3-(5-Sulfamoylthiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>13</b>)</h5><div class="NLM_p last">Method A: Yield 64%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 29.8 min; HPLC purity 99% (<i>R</i><sub>t</sub> = 12.71 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.62 (s, 1H), 8.42 (s, 1H), 8.30 (s, 1H), 8.24 (s, 1H), 8.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.89 (d, <i>J</i> = 8.0 Hz, 2H), 7.82 (s, 2H; N<i>H</i><sub>2</sub>), 7.70 (d, <i>J</i> = 4.0 Hz, 1H), 7.62 (d, <i>J</i> = 4.0 Hz, 1H), 2.99–2.97 (m, 6H; Et<sub>3</sub>N salt), 1.14 (t, <i>J</i> = 7.4 Hz, 9H; Et<sub>3</sub>N salt); MS (ESI, <i>m</i>/<i>z</i>) 495.0 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>20</sub>H<sub>14</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 495.0409, found 495.0402 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 4′-((4-(<i>tert</i>-Butoxycarbonyl)piperazin-1-yl)sulfonyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>14</b>)</h5><div class="NLM_p last">Method A: Yield 82%; HPLC purity 99% (<i>R</i><sub>t</sub> = 14.54 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.26 (s, 1H), 8.60 (s, 1H), 8.47–8.46 (m, 2H), 8.18 (d, <i>J</i> = 8.12 Hz, 2H), 8.06 (d, <i>J</i> = 8.44 Hz, 2H), 7.95 (d, <i>J</i> = 8.44 Hz, 2H), 7.81 (d, <i>J</i> = 8.24 Hz, 2H), 3.56–3.53 (m, 4H), 3.05 (t, <i>J</i> = 4.88 Hz, 4H), 1.42 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 558.2 [M + 1-Boc]<sup>+</sup>, 602.1 [M + 1-isobutyl]<sup>+</sup>, 658.3 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>23</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S 602.1321, found 602.1326 [M + 1-isobutyl]<sup>+</sup>.</div></div><div id="sec4_3_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 4′-(Piperazin-1-ylsulfonyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>15</b>)</h5><div class="NLM_p last">Method B: Yield 69%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 28.8 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 11.79 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.63 (s, 1H), 8.50 (s, 1H), 8.31 (s, 1H), 8.25 (s, 1H), 8.12 (d, <i>J</i> = 8.28 Hz, 2H), 8.07 (d, <i>J</i> = 8.20 Hz, 2H), 7.91 (d, <i>J</i> = 8.40 Hz, 2H), 7.81 (d, <i>J</i> = 8.20 Hz, 2H), 3.16 (s, 8H); MS (ESI, <i>m</i>/<i>z</i>) 558.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>26</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S 558.1423, found 558.1422 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 4′-Sulfamoyl-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>16</b>)</h5><div class="NLM_p last">Method A: Yield 99%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 28.4 min; HPLC purity 99% (<i>R</i><sub>t</sub> = 12.77 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.72 (s, 1H), 8.51 (s, 1H), 8.40 (s, 1H), 8.34 (s, 1H), 8.21 (d, <i>J</i> = 8.08 Hz, 2H), 8.05 (d, <i>J</i> = 8.48 Hz, 2H), 7.98 (d, <i>J</i> = 8.48 Hz, 2H), 7.91 (d, <i>J</i> = 8.28 Hz, 2H), 7.47 (s, 2H; N<i>H</i><sub>2</sub>), 2.78 (broad s, 6H; Et<sub>3</sub>N salt), 1.07 (t, <i>J</i> = 6.94 Hz, 9H; Et<sub>3</sub>N salt); MS (ESI, <i>m</i>/<i>z</i>) 489.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>16</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub>S 489.0844, found 489.0842 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 3-(5-((4-(<i>tert</i>-Butoxycarbonyl)piperazin-1-yl)methyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>17</b>)</h5><div class="NLM_p last">Method A: Yield 45%; HPLC purity 95% (<i>R</i><sub>t</sub> = 15.25 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.26 (s, 1H), 8.44 (s, 1H), 8.39 (t, <i>J</i> = 1.78 Hz, 1H), 8.35 (s, 1H), 8.18 (d, <i>J</i> = 8.16 Hz, 2H), 7.81 (d, <i>J</i> = 8.24 Hz, 2H), 7.54 (d, <i>J</i> = 3.64 Hz, 1H), 7.09 (d, <i>J</i> = 3.64 Hz, 1H), 3.89 (s, 2H), 3.51–3.50 (m, 4H), 2.60 (t, <i>J</i> = 4.76 Hz, 4H), 1.47 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 514.2 [M + 1 – Boc]<sup>+</sup>, 558.1 [M + 1-isobutyl]<sup>+</sup>, 614.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>30</sub>H<sub>31</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S 614.2049, found 614.2042 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 3-(5-(Piperazin-1-ylmethyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>18</b>)</h5><div class="NLM_p last">Method B: Yield 25%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 27.2 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 11.70 min); <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O+DMSO-<i>d</i><sub>6</sub>) δ 8.23 (s, 1H), 8.19–8.17 (m, 4H), 8.09 (s, 1H), 7.86 (d, <i>J</i> = 8.28 Hz, 2H), 7.49 (d, <i>J</i> = 3.52 Hz, 1H), 7.00 (d, <i>J</i> = 3.44 Hz, 1H), 3.66 (s, 2H; merged with H<sub>2</sub>O), 2.70 (s, 4H), 2.38 (s, merged with DMSO); MS (ESI, <i>m</i>/<i>z</i>) 514.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> 514.1525, found 514.1697 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 3-(5-((3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)carbamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>19</b>)</h5><div class="NLM_p last">Method A: Yield 93%; HPLC purity 95% (<i>R</i><sub>t</sub> = 13.69 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.27 (s, 1H), 8.53 (s, 1H), 8.44 (s, 2H), 8.19 (d, <i>J</i> = 8.0 Hz, 2H), 7.82 (d, <i>J</i> = 8.0 Hz, 2H), 7.74 (d, <i>J</i> = 4.0 Hz, 1H), 7.67 (d, <i>J</i> = 4.0 Hz, 1H), 3.45–3.42 (m, 2H), 3.18–3.15 (m, 2H), 1.81–1.78 (m, 2H), 1.49 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 516.2 [M + 1 – Boc]<sup>+</sup>, 638.2 [M + Na]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>29</sub>H<sub>28</sub>N<sub>5</sub>O<sub>5</sub>F<sub>3</sub>SNa 638.1661, found 638.1653 [M + Na]<sup>+</sup>.</div></div><div id="sec4_3_3_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 3-(5-((3-Aminopropyl)carbamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>20</b>)</h5><div class="NLM_p last">Method B: Yield 92%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 26.7 min; HPLC purity 95% (<i>R</i><sub>t</sub> = 10.31 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.64 (s, 1H), 8.88 (broad s, 0.5H, C(O)-N<i>H</i>), 8.42 (s, 1H), 8.31 (s, 1H), 8.28 (s, 1H), 8.19 (d, <i>J</i> = 8.0 Hz, 2H), 7.95 (d, <i>J</i> = 4.0 Hz, 1H), 7.88 (d, <i>J</i> = 8.4 Hz, 2H), 7.74 (d, <i>J</i> = 3.6 Hz, 1H), 3.47 (merged with water peak), 2.93 (t, <i>J</i> = 7.4 Hz, 2H), 1.88–1.86 (m, 2H); <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.4, 161.2, 146.6, 146.4, 140.3, 137.1, 134.8, 134.2, 129.5, 128.9, 128.6, 126.2, 126.5, 126.3, 125.6, 123.8, 121.6, 120.6, 117.2, 40.9, 40.5, 37.2; MS (ESI, <i>m</i>/<i>z</i>) 516.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>S 516.1317, found 516.1324 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 3-(5-(4-(<i>tert</i>-Butoxycarbonyl)piperazine-1-carbonyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>21</b>)</h5><div class="NLM_p last">Method A: Yield 72%; HPLC purity 98% (<i>R</i><sub>t</sub> = 14.47 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.21 (s, 1H), 8.43 (s, 1H), 8.42 (s, 1H), 8.33 (s, 1H), 8.19 (d, <i>J</i> = 8.08 Hz, 2H), 7.81 (d, <i>J</i> = 8.24 Hz, 2H), 7.64 (d, <i>J</i> = 3.88 Hz, 1H), 7.50 (d, <i>J</i> = 3.88 Hz, 1H), 3.82 (s, 4H), 3.57 (s, 4H), 1.50 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 572.1 [M + 1-isobutyl]<sup>+</sup>, 628.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>26</sub>H<sub>21</sub>N<sub>5</sub>O<sub>5</sub>F<sub>3</sub>S 572.1216, found 572.1213 [M + 1-isobutyl]<sup>+</sup>.</div></div><div id="sec4_3_3_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 3-(5-(Piperazine-1-carbonyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>22</b>)</h5><div class="NLM_p last">Method B: Yield 71%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 25.9 min; HPLC purity 97% (<i>R</i><sub>t</sub> = 10.72 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.72 (s, 1H), 8.44 (s, 1H), 8.41 (s, 1H), 8.25 (s, 1H), 8.21 (d, <i>J</i> = 8.12 Hz, 2H), 7.90 (d, <i>J</i> = 8.28 Hz, 2H), 7.75 (d, <i>J</i> = 3.84 Hz, 1H), 7.51 (d, <i>J</i> = 3.88 Hz, 1H), 3.69 (s, 4H), 2.88 (s, 4H); MS (ESI, <i>m</i>/<i>z</i>) 528.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>S 528.1317, found 528.1322 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 3-(5-((3-Aminopropyl)carbamoyl)furan-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>23</b>)</h5><div class="NLM_p last">Method C: Yield 35%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 26.7 min; HPLC purity 95% (<i>R</i><sub>t</sub> = 10.19 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.60 (s, 1H), 8.76 (broad s, 1H), 8.42 (s, 1H), 8.36 (s, 1H), 8.32 (s, 1H), 8.23 (d, <i>J</i> = 8.12 Hz, 2H), 7.88 (d, <i>J</i> = 8.36 Hz, 2H), 7.23 (d, <i>J</i> = 3.52 Hz, 1H), 7.21 (d, <i>J</i> = 3.56 Hz, 1H), 3.36 (merged with water peak), 2.71–2.67 (m, 2H), 1.70–1.65 (m, 2H); MS (ESI, <i>m</i>/<i>z</i>) 500.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> 500.1546, found 500.1544 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 3-(5-Carbamoylfuran-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>24</b>)</h5><div class="NLM_p last">Method A: Yield 30%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 28.9 min; HPLC purity 97% (<i>R</i><sub>t</sub> = 12.47 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.63 (s, 1H), 8.59 (s, 1H), 8.47 (s, 1H), 8.40 (s, 1H), 8.21 (d, <i>J</i> = 8.16 Hz, 2H), 8.14 (broad s, 0.5 H; N<i>H</i>), 7.90 (d, <i>J</i> = 8.32 Hz, 2H), 7.55 (broad s, 0.5 H; N<i>H</i>), 7.35 (d, <i>J</i> = 3.48 Hz, 1H), 7.24 (d, <i>J</i> = 3.57 Hz, 1H), 6.67 (broad s, 0.5H; N<i>H</i>), 2.94 (m, 6H; Et<sub>3</sub>N), 1.12 (t, <i>J</i> = 7.20 Hz, 9H; Et<sub>3</sub>N); MS (ESI, <i>m</i>/<i>z</i>) 443.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>21</sub>H<sub>14</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub> 443.0967, found 443.0964 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 3-(4-((3-Aminopropyl)carbamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>25</b>)</h5><div class="NLM_p last">Method C: Yield 61%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 27.3 min; HPLC purity 98% (<i>R</i><sub>t</sub> = 10.23 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 8.68 (broad s, 1H; N<i>H</i>), 8.33 (s, 1H), 8.22 (s, 1H), 8.17 (d, <i>J</i> = 8.40 Hz, 2H), 8.16 (s, 1H), 8.13 (s, 1H), 8.07 (s, 1H), 7.84 (d, <i>J</i> = 8.16 Hz, 2H), 3.36 (merged with water peak), 2.90 (t, <i>J</i> = 6.86 Hz, 2H), 1.84 (t, <i>J</i> = 7.64 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 516.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>S 516.1317, found 516.1311 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 3-(5-((3-Aminopropyl)carbamoyl)thiophen-3-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>26</b>)</h5><div class="NLM_p last">Method C: Yield 31%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 26.8 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 10.41 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.53 (s, 1H), 9.08 (broad s, 1H; N<i>H</i>), 8.40 (s, 1H), 8.34 (s, 1H), 8.31 (s, 1H), 8.20 (s, 2H), 8.15 (d, <i>J</i> = 8.16 Hz, 2H), 7.83 (d, <i>J</i> = 8.32 Hz, 2H), 3.36 (merged with water peak), 2.93 (t, <i>J</i> = 7.28 Hz, 2H), 1.91–1.87 (m, 2H); MS (ESI, <i>m</i>/<i>z</i>) 516.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>S 516.1317, found 516.1320 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 3-(6-Carbamoylpyridin-3-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>27</b>)</h5><div class="NLM_p last">Method A: Yield 52%; HPLC purity 96% (<i>R</i><sub>t</sub> = 12.16 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.21 (s, 1H), 9.19 (d, <i>J</i> = 1.68 Hz, 1H), 8.81 (s, 2H), 8.60 (s, 1H), 8.40 (dd, <i>J</i> = 2.20, 8.28 Hz, 1H), 8.20–8.16 (m, 3H), 7.80 (d, <i>J</i> = 8.24 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 454.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>, 454.1127, found 454.1130 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 3-(6-Carbamoylpyridin-3-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>28</b>)</h5><div class="NLM_p last">Method A: Yield 44%; HPLC purity 95% (<i>R</i><sub>t</sub> = 12.81 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.24 (s, 1H), 9.10 (d, <i>J</i> = 1.88 Hz, 1H), 8.55 (s, 1H), 8.47 (s, 1H), 8.41 (d, <i>J</i> = 2.36 Hz, 1H), 8.40 (s, 1H), 8.27 (d, <i>J</i> = 8.08 Hz, 1H), 8.19 (d, <i>J</i> = 8.20 Hz, 2H), 7.81 (d, <i>J</i> = 8.20 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 454.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub> 454.1127, found 454.1124 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 3-(5-Carbamoylpyrazin-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoic Acid (<b>29</b>)</h5><div class="NLM_p last">Method A: Yield 45%; HPLC purity 95% (<i>R</i><sub>t</sub> = 12.45 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.38 (s, 2H), 9.27 (s, 1H), 8.92 (s, 2H), 8.66 (s, 1H), 8.19 (d, <i>J</i> = 7.60 Hz, 2H), 7.81 (d, <i>J</i> = 7.44 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 455.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>21</sub>H<sub>14</sub>N<sub>6</sub>O<sub>3</sub>F<sub>3</sub>, 455.1079, found 455.1074 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 4′-Carbamoyl-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>30</b>)</h5><div class="NLM_p last">Method A: Yield 74%; Semipreparative HPLC: <i>R</i><sub>t</sub> = 26.4 min; HPLC purity 98% (<i>R</i><sub>t</sub> = 12.66 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.22 (s, 1H), 8.48 (s, 1H), 8.44 (s, 1H), 8.34 (s, 1H), 8.19 (d, <i>J</i> = 8.04 Hz, 2H), 8.05 (d, <i>J</i> = 8.24 Hz, 2H), 7.93 (d, <i>J</i> = 8.64 Hz, 2H), 7.81 (d, <i>J</i> = 7.88 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 453.1 [M + 1]<sup>+</sup>; <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 138.0, 137.3, 146.4, 144.7, 142.4, 140.3, 137.0, 134.9, 134.0, 128.8, 128.6, 128.0, 127.2, 126.5, 126.3, 125.9, 121.7, 120.4, 118.6; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>16</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 453.1175, found 453.1183 [M + 1]<sup>+</sup>. UV absorption (50% AcCN in PBS): λ<sub>max</sub> = 259 nm, ε = 3.0 × 10<sup>4</sup>.</div></div><div id="sec4_3_3_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 4′-(Methylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>31</b>)</h5><div class="NLM_p last">Method A: Yield 74%; HPLC purity 99% (<i>R</i><sub>t</sub> = 12.97 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.25 (s, 1H), 8.55 (s, 1H), 8.42 (s, 2H), 8.15 (d, <i>J</i> = 8.04 Hz, 2H), 7.98 (d, <i>J</i> = 8.12 Hz, 2H), 7.88 (d, <i>J</i> = 8.16 Hz, 2H), 7.79 (d, <i>J</i> = 8.12 Hz, 2H), 2.97 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 467.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 467.1331, found 467.1324 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 4′-(Ethylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>32</b>)</h5><div class="NLM_p last">Method A: Yield 74%; HPLC purity 99% (<i>R</i><sub>t</sub> = 13.27 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.24 (s, 1H), 8.55 (s, 1H), 8.43 (s, 1H), 8.41 (s, 1H), 8.16 (d, <i>J</i> = 7.76 Hz, 2H), 7.98 (d, <i>J</i> = 7.88 Hz, 2H), 7.97 (d, <i>J</i> = 7.96 Hz, 2H), 7.79 (d, <i>J</i> = 8.04 Hz, 2H), 3.46 (q, <i>J</i> = 7.30 Hz, 2H), 1.28 (t, <i>J</i> = 7.26 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 481.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 481.1488, found 481.1482 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4′-(Propylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>33</b>)</h5><div class="NLM_p last">Method A: Yield 59%; HPLC purity 99% (<i>R</i><sub>t</sub> = 13.62 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.25 (s, 1H), 8.55 (s, 1H), 8.43 (s, 2H), 8.16 (d, <i>J</i> = 7.96 Hz, 2H), 7.98 (d, <i>J</i> = 8.08 Hz, 2H), 7.89 (d, <i>J</i> = 8.04 Hz, 2H), 7.79 (d, <i>J</i> = 8.16 Hz, 2H), 3.39 (t, <i>J</i> = 7.06 Hz, 2H), 1.74–1.65 (m, 2H), 1.02 (t, <i>J</i> = 7.40 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 495.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>26</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 495.1644, found 495.1642 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 4′-(<i>tert</i>-Butylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>34</b>)</h5><div class="NLM_p last">Method A: Yield 68%; HPLC purity 98% (<i>R</i><sub>t</sub> = 14.03 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.19 (s, 1H), 8.48 (s, 1H), 8.36 (s, 1H), 8.34 (s, 1H), 8.09 (d, <i>J</i> = 8.00 Hz, 2H), 7.88 (d, <i>J</i> = 8.16 Hz, 2H), 7.80 (d, <i>J</i> = 8.00 Hz, 2H), 7.74 (d, <i>J</i> = 8.12 Hz, 2H), 1.51 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 509.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 509.1801, found 509.1802 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 4-(3′-(1<i>H</i>-Tetrazol-5-yl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)piperidine (<b>35</b>)</h5><div class="NLM_p last">A mixture of compound <b>113</b> in trifluoroacetic acid (0.1 mL) was stirred at room temperature for 10 min. The mixture was diluted with methanol and neutralized with triethylamine. After all volatiles were evaporated under reduced pressure, the residue was purified by HPLC (10 mM triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min) to afford compound <b>35</b> (4.3 mg, 74%) as a white solid; Semipreparative HPLC: <i>R</i><sub>t</sub> = 26.5 min; HPLC purity 96% (<i>R</i><sub>t</sub> = 11.25 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (s, 1H), 8.55 (s, 1H), 8.39 (s, 1H), 8.23 (d, <i>J</i> = 8.08 Hz, 2H), 8.13 (t, <i>J</i> = 1.78 Hz, 1H), 7.90 (d, <i>J</i> = 8.24 Hz, 2H), 7.84 (d, <i>J</i> = 8.24 Hz, 2H), 7.43 (d, <i>J</i> = 8.24 Hz, 2H), 3.42 (d, <i>J</i> = 12.92 Hz, 2H), 3.08–3.02 (m, 2H), 2.99–2.93 (m, 1H), 2.02 (d, <i>J</i> = 13.08 Hz, 2H), 1.91–1.80 (m, 2H); MS (ESI, <i>m</i>/<i>z</i>) 517.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>24</sub>N<sub>8</sub>F<sub>3</sub> 517.2076, found 517.2079 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 5-(3-(1<i>H</i>-Tetrazol-5-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)-<i>N</i>-(3-aminopropyl)thiophene-2-carboxamide (<b>36a</b>)</h5><div class="NLM_p last">Compound <b>114</b> (8 mg, 12.5 μmol) was converted to compound <b>36a</b> (3 mg, 44%) as a white solid, using a similar procedure to that used in the preparation of compound <b>35</b>; Semipreparative HPLC: <i>R</i><sub>t</sub> = 28.1 min; HPLC purity 95% (<i>R</i><sub>t</sub> = 10.24 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.27 (s, 1H), 8.58 (s, 2H), 8.32 (s, 1H) 8.21 (d, <i>J</i> = 8.00 Hz, 2H), 7.82 (d, <i>J</i> = 8.24 Hz, 2H), 7.76 (d, <i>J</i> = 3.88 Hz, 1H), 7.70 (d, <i>J</i> = 3.88 Hz, 1H), 3.44 (t, <i>J</i> = 6.84 Hz, 2H), 3.17 (t, <i>J</i> = 6.80 Hz, 2H), 1.79 (t, <i>J</i> = 6.62 Hz, 2H), 1.47 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 540.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>21</sub>N<sub>9</sub>OF<sub>3</sub>S 540.1542, found 540.1547 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 3′-(1<i>H</i>-Tetrazol-5-yl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-carboxamide (<b>36b</b>)</h5><div class="NLM_p last">Compound <b>111</b> (5.5 mg, 11.5 μmol) was converted to compound <b>36b</b> (5.8 mg, 99%) as a white solid, using a similar procedure to that used in the preparation of compound <b>113</b>; Purified by silica gel column chromatography (dichloromethane/methanol = 4:1); HPLC purity 96% (<i>R</i><sub>t</sub> = 11.57 min); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.27 (s, 1H), 8.62 (s, 1H), 8.57 (s, 1H), 8.32 (s, 1H), 8.20 (d, <i>J</i> = 8.16 Hz, 2H), 8.07 (d, <i>J</i> = 8.40 Hz, 2H), 7.98 (d, <i>J</i> = 8.40 Hz, 2H), 7.82 (d, <i>J</i> = 8.28 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 477.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>16</sub>N<sub>8</sub>OF<sub>3</sub> 477.1399, found 477.1391 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>N</i>-(<i>tert</i>-Butyl)-3′-(1<i>H</i>-tetrazol-5-yl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-carboxamide (<b>36c</b>)</h5><div class="NLM_p last">Compound <b>110</b> (10 mg, 0.020 mmol) was converted to compound <b>36c</b> (10 mg, 92%) as a white solid, using a similar procedure to that used in the preparation of compound <b>113</b>; Purified by silica gel column chromatography (dichloromethane/methanol = 5:1); HPLC purity 98% (<i>R</i><sub>t</sub> = 12.73 min); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 8.33 (s, 1H), 8.22 (d, <i>J</i> = 7.88 Hz, 3H), 8.00 (m, 4H), 7.20–7.90 (m, 4H), 1.47 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 533.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>24</sub>N<sub>8</sub>OF<sub>3</sub> 533.2025, found 533.2030 [M + 1]<sup>+</sup>.</div></div><div id="sec4_3_3_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> <i>tert</i>-Butyl 4-(3′-(methoxycarbonyl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)piperidine-1-carboxylate (<b>79</b>)</h5><div class="NLM_p last">The mixture of compound <b>42</b> (74 mg, 0.156 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (11 mg, 9.36 μmol), and potassium carbonate (64 mg, 0.462 mmol) in dimethylformamide (2 mL) was purged with nitrogen gas for 15 min, and then compound <b>55</b> (64 mg, 0.188 mmol) was added to the mixture. The mixture was stirred at 80 °C for 2 h, and then allowed to be cooled at room temperature. This mixture was partitioned with diethyl ether (10 mL) and water (20 mL). The aqueous layer was extracted with diethyl ether (5 mL × 2), and then the combined organic layer was washed with brine (3 mL), dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl actate = 4:1) to afford compound <b>79</b> (42 mg, 42%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.43 (s, 1H), 8.38 (t, <i>J</i> = 1.54 Hz, 1H), 8.35 (s, 1H), 8.34 (s, 1H), 8.07 (d, <i>J</i> = 8.04 Hz, 2H), 7.75 (d, <i>J</i> = 8.20 Hz, 2H), 7.67 (d, <i>J</i> = 8.28 Hz, 2H), 7.37 (d, <i>J</i> = 8.24 Hz, 2H), 4.31 (s, 2H), 4.03 (s, 3H), 2.86 (d, <i>J</i> = 12.22 Hz, 2H), 2.78–2.72 (m, 1H), 1.89 (d, <i>J</i> = 12.68 Hz, 2H), 1.75–1.64 (m, 2H), 1.52 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 607.3 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>33</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub> 607.2532, found 607.2538 [M + 1]<sup>+</sup>.</div></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> General Procedure (Suzuki Reaction) for <b>80</b>–<b>102</b></h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Method A</h4><div class="NLM_p last">A mixture of compound <b>42</b> (1 equiv), aryl bromide (1.2 equiv), and PdCl<sub>2</sub>(dppf) (0.1 equiv) in dimethoxyethane/2 M Na<sub>2</sub>CO<sub>3</sub> aqueous solution (10:1) was stirred at 50 °C. After cooling at room temperature, this mixture was partitioned with ethyl acetate and water. The aqueous layer was extracted with ethyl acetate twice, and then the combined organic layer was washed with brine, dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl acetate) to afford the condensed compounds as white solids.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Method B</h4><div class="NLM_p">A mixture of compound <b>42</b> (1 equiv) and aryl bromide (1.2 equiv) in DMF/H<sub>2</sub>O (20:1; 20 mM) was purged with nitrogen gas for 30 min, and then Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 equiv) and Na<sub>2</sub>CO<sub>3</sub> (3 equiv) were added to the mixture, which was stirred for 1 h at 40–120 °C. The aqueous layer was extracted with ethyl acetate twice, and then the combined organic layer was washed with brine, dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl acetate) to afford the condensed compounds as white solids.</div><div id="sec4_4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> Methyl 3-(5-(<i>N</i>-(2-((<i>tert</i>-Butoxycarbonyl)amino)ethyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>80</b>)</h5><div class="NLM_p last">Method A; Yield 44%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.46 (s, 1H), 8.37 (t, <i>J</i> = 1.2 Hz, 1H), 8.35 (t, <i>J</i> = 2.0 Hz, 1H), 8.34 (t, <i>J</i> = 1.4 Hz, 1H), 8.07 (d, <i>J</i> = 8.0 Hz, 2H), 7.76 (d, <i>J</i> = 8.4 Hz, 2H), 7.64 (d, <i>J</i> = 4.0 Hz, 1H), 7.47 (d, <i>J</i> = 3.6 Hz, 1H), 5.64 (broad s, 1H), 4.93 (broad s, 1H), 4.05 (s, 3H), 3.33 (t, <i>J</i> = 5.8 Hz, 2H), 3.27 (t, <i>J</i> = 5.6 Hz, 2H), 1.45 (s, 9H).</div></div><div id="sec4_4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> Methyl 3-(5-(<i>N</i>-(3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>81</b>)</h5><div class="NLM_p last">Method A; Yield 44%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.37 (s, 1H), 8.35 (s, 1H), 8.34 (s, 1H), 8.08 (d, <i>J</i> = 8.0 Hz, 2H), 7.76 (d, <i>J</i> = 8.0 Hz, 2H), 7.63 (d, <i>J</i> = 3.6 Hz, 1H), 7.46 (d, <i>J</i> = 4.0 Hz, 1H), 6.03 (broad s, 1H), 4.71 (broad s, 1H), 4.05 (s, 3H), 3.26 (q, <i>J</i> = 6.2 Hz, 2H), 3.16 (q, <i>J</i> = 6.2 Hz, 2H), 1.73–1.70 (m, 2H), 1.41 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 566.1 [M + 1-Boc]<sup>+</sup>, 666.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>29</sub>H<sub>31</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S 666.1668, found 666.1676 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> Methyl 3-(5-(<i>N</i>-(4-((<i>tert</i>-butoxycarbonyl)amino)butyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>82</b>)</h5><div class="NLM_p last">Method A; Yield 62%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.38 (s, 1H), 8.37 (s, 1H), 8.35 (s, 1H), 8.08 (d, <i>J</i> = 8.0 Hz, 2H), 7.76 (d, <i>J</i> = 7.6 Hz, 2H), 7.64 (d, <i>J</i> = 3.2 Hz, 1H), 7.47 (d, <i>J</i> = 3.2 Hz, 1H), 5.01 (broad s, 1H), 4.58 (broad s, 1H), 4.09 (s, 3H), 3.16–3.14 (m, 4H), 1.63 (m, merged with water peak), 1.45 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 792.0 [M-1+TFA]<sup>−</sup>, 678.2 [M – 1]<sup>−</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>30</sub>H<sub>31</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S<sub>2</sub> 678.1668, found 678.1666 [M – H]<sup>−</sup>.</div></div><div id="sec4_4_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> Methyl 3-(5-(<i>N</i>-(4-Hydroxybutyl)sulfamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>83</b>)</h5><div class="NLM_p last">Method A; Yield 65%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.30 (s, 1H), 8.57 (s, 1H), 8.51 (s, 1H), 8.39 (s, 1H), 8.17 (d, <i>J</i> = 8.08 Hz, 2H), 7.81 (d, <i>J</i> = 8.24 Hz, 2H), 7.70 (d, <i>J</i> = 3.92 Hz, 1H), 7.65 (d, <i>J</i> = 3.96 Hz, 1H), 4.04 (s, 3H), 3.56 (t, <i>J</i> = 7.84 Hz, 2H), 3.06 (t, <i>J</i> = 6.58 Hz, 2H), 1.63–1.57 (m, 4H); MS (ESI, <i>m</i>/<i>z</i>) 581.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>5</sub>F<sub>3</sub>S<sub>2</sub> 581.1140, found 581.1143 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> <i>tert</i>-Butyl 4-((5-(3-(Methoxycarbonyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophen-2-yl)sulfonyl)piperazine-1-carboxylate (<b>84</b>)</h5><div class="NLM_p last">Method A; Yield 63%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.43 (s, 1H), 8.39–8.38 (m, 2H), 8.37 (s, 1H), 8.09 (d, <i>J</i> = 8.4 Hz, 2H), 7.77 (d, <i>J</i> = 8.4 Hz, 2H), 7.59 (d, <i>J</i> = 3.6 Hz, 1H), 7.54 (d, <i>J</i> = 3.6 Hz, 1H), 4.06 (s, 3H), 3.60 (t, <i>J</i> = 4.8 Hz, 4H), 3.14 (t, <i>J</i> = 4.8 Hz, 4H), 1.45 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 622.1 [M + 1-isobutyl]<sup>+</sup>, 678.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>30</sub>H<sub>31</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S 678.1668, found 678.1671 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> Methyl 3-(5-Sulfamoylthiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>85</b>)</h5><div class="NLM_p last">Method A; Yield 40%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.43 (s, 1H), 8.37 (s, 2H), 8.36 (s, 1H), 8.08 (d, <i>J</i> = 8.0 Hz, 2H), 7.77 (d, <i>J</i> = 8.0 Hz, 2H), 7.72 (d, <i>J</i> = 4.0 Hz, 1H), 7.47 (d, <i>J</i> = 4.0 Hz, 1H), 5.11 (s, 2H), 4.05 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 509.0 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>21</sub>H<sub>16</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub>S<sub>2</sub> 509.0565, found 509.0565 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> <i>tert</i>-Butyl 4-((3′-(Methoxycarbonyl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)sulfonyl)piperazine-1-carboxylate (<b>86</b>)</h5><div class="NLM_p last">Method A; Yield 40%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.45–8.44 (m, 1H), 8.42–8.40 (m, 2H), 8.09 (d, <i>J</i> = 8.04 Hz, 2H), 7.92 (d, <i>J</i> = 8.60 Hz, 2H), 7.88 (d, <i>J</i> = 8.60 Hz, 2H), 7.77 (d, <i>J</i> = 8.16 Hz, 2H), 4.06 (s, 3H), 3.56 (t, <i>J</i> = 4.90 Hz, 4H), 3.07 (t, <i>J</i> = 4.70 Hz, 4H), 1.43 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 616.1 [M + 1-isobutyl]<sup>+</sup>, 672.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>32</sub>H<sub>33</sub>N<sub>5</sub>O<sub>6</sub>F<sub>3</sub>S 672.2104, found 672.2098 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> Methyl 4′-Sulfamoyl-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>87</b>)</h5><div class="NLM_p last">Method A; Yield 44%; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.31 (s, 1H), 8.63 (s, 1H), 8.52 (s, 1H), 8.45 (s, 1H), 8.17 (d, <i>J</i> = 7.92 Hz, 2H), 8.08 (d, <i>J</i> = 8.32 Hz, 2H), 8.00 (d, <i>J</i> = 8.36 Hz, 2H), 7.80 (d, <i>J</i> = 8.08 Hz, 2H), 4.04 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 503.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>18</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub>S 503.1001, found 503.0998 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> <i>tert</i>-Butyl 4-((5-(3-(Methoxycarbonyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophen-2-yl)methyl)piperazine-1-carboxylate (<b>88</b>)</h5><div class="NLM_p last">Method A; Yield 61%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.41 (s, 1H), 8.35 (t, <i>J</i> = 1.46 Hz, 1H), 8.31 (t, <i>J</i> = 1.86 Hz, 1H), 8.26 (m, 1H), 8.08 (d, <i>J</i> = 8.08 Hz, 2H), 7.76 (d, <i>J</i> = 8.24 Hz, 2H), 7.38 (d, <i>J</i> = 3.60 Hz, 1H), 6.98 (s, 1H), 4.03 (s, 3H), 3.79–3.63 (m, 2H), 3.51 (s, 4H), 2.53 (s, 4H), 1.48 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 572.2 [M + 1-isobutyl]<sup>+</sup>, 628.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>31</sub>H<sub>33</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub>S 628.2205, found 628.2200 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> Methyl 3-(5-((3-((<i>tert</i>-butoxycarbonyl)amino)propyl)carbamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl) phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>89</b>)</h5><div class="NLM_p last">Method A; Yield 48%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.42 (s, 1H), 8.40 (s, 1H), 8.37 (s, 1H), 8.36 (s, 1H), 8.09 (d, <i>J</i> = 8.0 Hz, 2H), 7.77 (d, <i>J</i> = 8.0 Hz, 2H), 7.64 (s, 1H), 7.54 (s, 1H), 4.87 (s, 0.5H; N<i>H</i>), 4.04 (s, 3H), 3.54 (s, 2H), 3.32 (s, 2H), 1.76 (s, 2H), 1.50 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 530.1 [M + 1-Boc]<sup>+</sup>, 574.1 [M + 1-isobutyl]<sup>+</sup>, 652.2 [M + Na]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>30</sub>H<sub>30</sub>N<sub>5</sub>O<sub>5</sub>F<sub>3</sub>SNa 652.1817, found 652.1818 [M + Na]<sup>+</sup>.</div></div><div id="sec4_4_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>tert</i>-Butyl 4-(5-(3-(Methoxycarbonyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophene-2-carbonyl)piperazine-1-carboxylate (<b>90</b>)</h5><div class="NLM_p last">Method A; Yield 67%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.43 (s, 1H), 8.37 (s, 2H), 8.34 (s, 1H), 8.08 (d, <i>J</i> = 8.04 Hz, 2H), 7.76 (d, <i>J</i> = 8.00 Hz, 2H), 7.46 (d, <i>J</i> = 3.56 Hz, 1H), 7.36 (d, <i>J</i> = 3.56 Hz, 1H), 4.05 (s, 3H), 3.79 (t, <i>J</i> = 4.84 Hz, 4H), 3.56 (t, <i>J</i> = 5.00 Hz, 4H), 1.51 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 586.1 [M + 1-isobutyl]<sup>+</sup>, 642.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>31</sub>H<sub>31</sub>N<sub>5</sub>O<sub>5</sub>F<sub>3</sub>S 642.1998, found 642.2000 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Methyl 3-(5-((3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)carbamoyl)furan-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>91</b>)</h5><div class="NLM_p last">Method A; Yield 25%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.53 (s, 2H), 8.44 (s, 1H), 8.42 (s, 1H), 8.06 (d, <i>J</i> = 8.08 Hz, 2H), 7.75 (d, <i>J</i> = 8.16 Hz, 2H), 7.48 (broad s, 1H), 7.25 (d, <i>J</i> = 3.48 Hz, 1H), 7.00 (d, <i>J</i> = 3.48 Hz, 1H), 4.04 (s, 3H), 3.59–3.56 (m, 2H), 3.36–3.33 (m, 2H), 1.82–1.76 (m, 2H), 1.40 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 514.1 [M + 1-Boc]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> 514.1702 found 514.1703 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_4_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> Methyl 3-(5-Carbamoylfuran-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>92</b>)</h5><div class="NLM_p last">Method B; Yield 57%; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.24 (s, 1H), 8.64 (s, 1H), 8.56 (s, 1H), 8.53 (s, 1H), 8.15 (d, <i>J</i> = 8.12 Hz, 2H), 7.80 (d, <i>J</i> = 8.20 Hz, 2H), 7.69–7.54 (m, POPh<sub>3</sub>), 7.29 (d, <i>J</i> = 3.64 Hz, 1H), 7.23 (d, <i>J</i> = 3.64 Hz, 1H), 4.03 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 457.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>16</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub> 457.1124 found 457.1125 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> Methyl 3-(4-((3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)carbamoyl)thiophen-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>93</b>)</h5><div class="NLM_p last">Method A; Yield 73%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (s, 1H), 8.38 (s, 1H), 8.34 (s, 2H), 8.08 (d, <i>J</i> = 7.00 Hz, 2H), 8.00 (s, 1H), 7.95 (s, 1H), 7.76 (d, <i>J</i> = 6.96 Hz, 2H), 4.04 (s, 3H), 3.55 (s, 2H), 3.32 (s, 2H), 1.76 (s, 2H), 1.49 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 530.1 [M + 1-Boc]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>23</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub>S 530.1474, found 530.1476 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_4_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> Methyl 3-(5-((3-((<i>tert</i>-Butoxycarbonyl)amino)propyl)carbamoyl)thiophen-3-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>94</b>)</h5><div class="NLM_p last">Method A; Yield 38%; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.26 (s, 1H), 8.60 (s, 1H), 8.49 (s, 1H), 8.40 (s, 1H), 8.18 (d, <i>J</i> = 8.24 Hz, 3H), 7.92 (s, 1H), 7.81 (d, <i>J</i> = 8.08 Hz, 2H), 4.03 (s, 3H), 3.46–3.40 (m, 2H), 3.17–3.14 (m, 2H), 1.80–1.75 (m, 2H), 1.44 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 530.1 [M + 1-Boc]<sup>+</sup>, 652.2 [M + Na]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>30</sub>H<sub>30</sub>N<sub>5</sub>O<sub>5</sub>F<sub>3</sub>SNa 652.1817, found 652.1825 [M + Na]<sup>+</sup>.</div></div><div id="sec4_4_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> Methyl 3-(5-Carbamoylpyridin-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>95</b>)</h5><div class="NLM_p last">Method B; Yield 26%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.19 (s, 1H), 8.89 (s, 1H), 8.79 (s, 1H), 8.59 (s, 1H), 8.52 (s, 1H), 8.36 (dd, <i>J</i> = 1.68, 8.28 Hz, 1H), 8.09 (d, <i>J</i> = 8.08 Hz, 2H), 8.05 (d, <i>J</i> = 8.36 Hz, 1H), 7.76 (d, <i>J</i> = 8.16 Hz, 2H), 7.71–7.47 (m, POPh<sub>3</sub>), 4.06 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 468.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>17</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub> 468.1283, found 468.1282 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> Methyl 3-(6-Carbamoylpyridin-3-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>96</b>)</h5><div class="NLM_p last">Method B; Yield 11%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.95 (s, 1H), 8.46 (s, 2H), 8.45 (s, 1H), 8.42 (s, 1H), 8.39 (d, <i>J</i> = 7.84 Hz, 1H), 8.21 (dd, <i>J</i> = 1.84, 8.08 Hz, 1H), 8.09 (d, <i>J</i> = 8.08 Hz, 2H), 7.77 (d, <i>J</i> = 8.28 Hz, 2H), 7.73–7.49 (m, POPh<sub>3</sub>), 4.07 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 468.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>17</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub> 468.1283, found 468.1275 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Methyl 3-(5-Carbamoylpyrazin-2-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzoate (<b>97</b>)</h5><div class="NLM_p last">Method B; Yield 33%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.43 (s, 1H), 9.25 (s, 1H), 8.92 (s, 2H), 8.86 (s, 1H), 8.67 (s, 1H), 8.08 (d, <i>J</i> = 8.00 Hz, 2H), 7.74 (d, <i>J</i> = 8.04 Hz, 2H), 7.75–7.46 (m, POPh<sub>3</sub>), 4.05 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 469.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>22</sub>H<sub>16</sub>N<sub>6</sub>O<sub>3</sub>F<sub>3</sub> 469.1236, found 469.1232 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Methyl 4′-Carbamoyl-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>98</b>)</h5><div class="NLM_p last">Method A; Yield 61%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.41 (m, 3H), 8.09 (d, <i>J</i> = 8.04 Hz, 2H), 8.00 (d, <i>J</i> = 8.24 Hz, 2H), 7.82 (d, <i>J</i> = 8.20 Hz, 2H), 7.77 (d, <i>J</i> = 8.16 Hz, 2H), 4.05 (s, 3H); MS (ESI, <i>m</i>/<i>z</i>) 467.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 467.1331, found 467.1332 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Methyl 4′-(Methylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>99</b>)</h5><div class="NLM_p last">Method B; Yield 33%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.38 (s, 2H), 8.36 (d, <i>J</i> = 1.60 Hz, 1H), 8.07 (d, <i>J</i> = 8.12 Hz, 2H), 7.92 (d, <i>J</i> = 8.20 Hz, 2H), 7.77–7.74 (m, 4H), 7.70–7.46 (m, POPh<sub>3</sub>), 6.36 (d, <i>J</i> = 4.24 Hz, 1H; N<i>H</i>), 4.04 (s, 3H), 3.08 (d, <i>J</i> = 4.80 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 481.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 481.1488, found 481.1487 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> Methyl 4′-(Ethylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>100</b>)</h5><div class="NLM_p last">Method B; Yield 31%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.45 (s, 1H), 8.38 (s, 2H), 8.37 (s, 1H), 8.07 (d, <i>J</i> = 8.08 Hz, 2H), 7.92 (d, <i>J</i> = 8.12 Hz, 2H), 7.77 (d, <i>J</i> = 7.28 Hz, 2H), 7.75 (d, <i>J</i> = 7.40 Hz, 2H), 6.25 (s, 1H), 4.04 (s, 3H), 3.60–3.53 (m, 2H), 1.32 (t, <i>J</i> = 7.24 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 495.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>26</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 495.1644, found 495.1650 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Methyl 4′-(Propylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>101</b>)</h5><div class="NLM_p last">Method B; Yield 43%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (s, 1H), 8.37 (s, 2H), 8.35 (d, <i>J</i> = 1.56 Hz, 1H), 8.06 (d, <i>J</i> = 8.12 Hz, 2H), 7.91 (d, <i>J</i> = 8.20 Hz, 2H), 7.76 (d, <i>J</i> = 4.36 Hz, 2H), 7.74 (d, <i>J</i> = 4.32 Hz, 2H), 6.31 (t, <i>J</i> = 5.24 Hz, 1H), 4.04 (s, 3H), 3.48 (q, <i>J</i> = 6.69 Hz, 2H), 1.75–1.63 (m, 2H), 1.04 (t, <i>J</i> = 7.38 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 509.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 509.1801, found 509.1795 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> Methyl 4′-(<i>tert</i>-Butylcarbamoyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylate (<b>102</b>)</h5><div class="NLM_p last">Method B; Yield 45%; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (s, 1H), 8.39–8.37 (m, 3H), 8.08 (d, <i>J</i> = 7.88 Hz, 2H), 7.88 (d, <i>J</i> = 8.16 Hz, 2H), 7.77–7.75 (m, 4H), 4.04 (s, 3H), 1.26 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 523.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>28</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>F<sub>3</sub> 523.1957, found 523.1967 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 4′-(1-(<i>tert</i>-Butoxycarbonyl)piperidin-4-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>103</b>)</h5><div class="NLM_p last">To a solution of <b>79</b> (52 mg, 0.086 mmol) in methanol (2 mL) was added potassium hydroxide (24 mg, 0.429 mmol), and then this reaction mixture was stirred at 50 °C for 15 h. After solvent was evaporated under reduced pressure, the residue was purified by silica gel column chromatography (dichloromethane/methanol = 10:1) to afford as a white solid <b>103</b> (47 mg, 93%); <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.22 (s, 1H), 8.48 (s, 1H), 8.35 (s, 2H), 8.14 (d, <i>J</i> = 7.76 Hz, 2H), 7.78 (d, <i>J</i> = 7.92 Hz, 2H), 7.71 (d, <i>J</i> = 7.72 Hz, 2H), 7.39 (d, <i>J</i> = 7.60 Hz, 2H), 4.25 (d, <i>J</i> = 12.8 Hz, 2H), 2.91 (s, 2H), 2.80 (t, <i>J</i> = 11.78 Hz, 1H), 1.88 (d, <i>J</i> = 12.2 Hz, 2H), 1.69–1.60 (m, 2H), 1.51 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 537.1 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>28</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub>F<sub>3</sub> 537.1750, found 537.1760 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>tert</i>-Butyl 4-(3′-Carbamoyl-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)piperidine-1-carboxylate (<b>104</b>)</h5><div class="NLM_p last">To a solution of compound <b>103</b> (47 mg, 0.079 mmol) in dimethylformamide (3 mL) were added NH<sub>4</sub>Cl (8.5 mg, 0.159 mmol), HATU (45 mg, 0.119 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (20 mg, 28 μL, 0.159 mmol), and then this reaction mixture was stirred at room temperature for 1 h. This mixture was partitioned with ethyl acetate (6 mL) and water (3 mL). The aqueous layer was extracted with ethyl acetate (5 mL × 2), and then the combined organic layer was washed with brine (3 mL), dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl actate = 1:1) to afford compound <b>104</b> (48 mg, 99%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (s, 1H), 8.27 (s, 1H), 8.23 (s, 1H), 8.11 (s, 1H), 8.07 (d, <i>J</i> = 8.04 Hz, 2H), 7.76 (d, <i>J</i> = 8.16 Hz, 2H), 7.66 (d, <i>J</i> = 8.24 Hz, 2H), 7.38 (d, <i>J</i> = 8.20 Hz, 2H), 4.31 (d, <i>J</i> = 13.68 Hz, 2H), 2.89–2.81 (m, 2H), 2.80–2.73 (m, 1H), 1.89 (d, <i>J</i> = 12.00 Hz, 2H), 1.67 (merged with water peak), 1.51 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 536.1 [M + 1-isopropyl]<sup>+</sup>, 592.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>28</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>F<sub>3</sub> 536.1909, found 536.1911 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> <i>tert</i>-Butyl (3-(5-(3-Carbamoyl-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophene-2-carboxamido)propyl)carbamate (<b>105</b>)</h5><div class="NLM_p last">Compound <b>19</b> (55 mg, 0.089 mmol) was converted to compound <b>105</b> (44 mg, 80%) as a white solid, using a similar procedure to that used in the preparation of compound <b>104</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.46 (s, 1H), 8.29 (s, 1H), 8.23 (s, 1H), 8.13 (s, 1H), 8.08 (d, <i>J</i> = 8.00 Hz, 2H), 7.77 (d, <i>J</i> = 8.28 Hz, 2H), 7.63 (d, <i>J</i> = 3.88 Hz, 1H), 7.49 (d, <i>J</i> = 3.84 Hz, 1H), 3.53 (dd, <i>J</i> = 6.2, 12.04 Hz, 2H), 3.33–3.30 (m, 2H), 1.76–1.73 (m, 2H), 1.50 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 515.1 [M + 1-Boc]<sup>+</sup>, 559.1 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>22</sub>N<sub>6</sub>O<sub>4</sub>F<sub>3</sub>S 559.1375, found 559.1383 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> <i>N</i>4′-(<i>tert</i>-Butyl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-3,4′-icarboxamide (<b>106</b>)</h5><div class="NLM_p last">Compound <b>34</b> (19 mg, 37.3 μmol) was converted to compound <b>106</b> (16 mg, 86%) as a white solid, using a similar procedure to that used in the preparation of compound <b>104</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.46 (s, 1H), 8.27 (s, 2H), 8.13 (s, 1H), 8.06 (d, <i>J</i> = 7.32 Hz, 2H), 7.85 (d, <i>J</i> = 6.96 Hz, 2H), 7.75 (d, <i>J</i> = 8.44 Hz, 2H), 7.73 (d, <i>J</i> = 8.40 Hz, 2H), 1.53 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 508.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub>F<sub>3</sub> 508.1960, found 508.1962 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 3-Bromo-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzamide (<b>107</b>)</h5><div class="NLM_p last">Compound <b>37b</b> (82 mg, 0.199 mmol) was converted to compound <b>107</b> (83 mg, 100%) as a white solid, using a similar procedure to that used in the preparation of compound <b>104</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.43 (s, 1H), 8.38 (s, 0.5H; N<i>H</i>), 8.33–8.32 (m, 0.5H; N<i>H</i>), 8.28 (s, 1H), 8.25 (s, 1H), 8.06–8.04 (m, 3H), 7.75 (d, <i>J</i> = 8.20 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 411.0, 413.0 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>16</sub>H<sub>11</sub>N<sub>4</sub>OF<sub>3</sub><sup>79</sup>Br 411.0068, found 411.0067 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> <i>tert</i>-Butyl 4-(3′-Cyano-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)piperidine-1-carboxylate (<b>108</b>)</h5><div class="NLM_p last">To a solution of compound <b>104</b> (41 mg, 0.069 mmol) in dichloromethane (2 mL) were added trifluoroacetic anhydride (97 mg, 64 μL, 0.462 mmol) and triethylamine (50 mg, 69 μL, 0.494 mmol) at 0 °C, and then this reaction mixture was stirred at room temperature for 1 h. This mixture was partitioned with dichloromethane (6 mL) and water (3 mL). The aqueous layer was extracted with dichloromethane (5 mL × 2), and the organic layer was washed with brine (3 mL), dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl actate = 4:1) to afford compound <b>108</b> (30 mg, 76%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.38 (s, 1H), 8.31 (t, <i>J</i> = 1.84 Hz, 1H), 8.07 (d, <i>J</i> = 8.08 Hz, 2H), 8.06–8.04 (m, 1H), 7.96 (t, <i>J</i> = 1.42 Hz, 1H), 7.77 (d, <i>J</i> = 8.20 Hz, 2H), 7.62 (d, <i>J</i> = 8.28 Hz, 2H), 7.39 (d, <i>J</i> = 8.20 Hz, 2H), 4.31 (d, <i>J</i> = 12.84 Hz, 2H), 2.89–2.83 (m, 2H), 2.79–2.72 (m, 1H), 1.89 (d, <i>J</i> = 12.04 Hz, 2H), 1.75–1.65 (m, 2H), 1.52 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 518.1 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>28</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub>F<sub>3</sub> 518.1804, found 518.1801 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> <i>tert</i>-Butyl (3-(5-(3-Cyano-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophene-2-carboxamido)propyl)carbamate (<b>109</b>)</h5><div class="NLM_p last">Compound <b>105</b> (40 mg, 0.065 mmol) was converted to compound <b>109</b> (28 mg, 72%) as a white solid, using a similar procedure to that used in the preparation of compound <b>108</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.39 (s, 1H), 8.31 (t, <i>J</i> = 3.44 Hz, 1H), 8.08 (d, <i>J</i> = 8.20 Hz, 2H), 8.07–8.06 (m, 1H), 7.99 (s, 1H), 7.77 (d, <i>J</i> = 8.20 Hz, 2H), 7.65 (d, <i>J</i> = 3.76 Hz, 1H), 7.49 (d, <i>J</i> = 3.88 Hz, 1H), 4.85 (s, 1H; N<i>H</i>), 3.53 (dd, <i>J</i> = 6.28, 11.96 Hz, 2H), 3.32 (dd, <i>J</i> = 6.34, 12.06 Hz, 2H), 1.75 (t, <i>J</i> = 5.34 Hz, 2H), 1.51 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 497.2 [M + 1-Boc]<sup>+</sup>, 541.1 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>24</sub>H<sub>20</sub>N<sub>6</sub>OF<sub>3</sub>S 497.1371, found 497.1375 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_4_2_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> <i>N</i>-(<i>tert</i>-Butyl)-3′-cyano-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-carboxamide (<b>110</b>)</h5><div class="NLM_p last">Compound <b>106</b> (16 mg, 31.5 μmol) was converted to compound <b>110</b> (11 mg, 71%) as a white solid, using a similar procedure to that used in the preparation of compound <b>108</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.41 (s, 1H), 8.35 (s, 1H), 8.11 (s, 1H), 8.07 (d, <i>J</i> = 8.08 Hz, 2H), 7.99 (s, 1H), 7.91 (d, <i>J</i> = 8.28 Hz, 2H), 7.77 (d, <i>J</i> = 8.24 Hz, 2H), 7.72 (d, <i>J</i> = 8.24 Hz, 2H), 1.54 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 490.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>27</sub>H<sub>23</sub>N<sub>5</sub>OF<sub>3</sub> 490.1855, found 490.1860 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 3′-Cyano-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-carboxamide (<b>111</b>)</h5><div class="NLM_p last">To a solution of compound <b>110</b> (7.8 mg, 15.9 μmol) in anisole (1 mL) was added trimethylsilyl trifluoromethanesulfonate (6 μL, 31.9 μmol), and this reaction mixture was stirred at 140 °C for 1 h in microwave. The reaction mixture was filtered through silica gel and washed with hexane to remove anisole. The desired product was eluted with ethyl acetate and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (dichloromethane/methanol = 30:1) to afford compound <b>111</b> (5.5 mg, 80%) as a white solid; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.32 (s, 1H), 8.61 (t, <i>J</i> = 1.72 Hz, 1H), 8.43 (s, 1H), 8.26 (s, 1H), 8.18 (d, <i>J</i> = 8.36 Hz, 2H), 8.07 (d, <i>J</i> = 8.36 Hz, 2H), 7.94 (d, <i>J</i> = 8.36 Hz, 2H), 7.82 (d, <i>J</i> = 8.28 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 434.1 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>23</sub>H<sub>15</sub>N<sub>5</sub>OF<sub>3</sub> 434.1229, found 434.1230 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 3-Bromo-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzonitrile (<b>112</b>)</h5><div class="NLM_p last">Compound <b>107</b> (83 mg, 0.199 mmol) was converted to compound <b>112</b> (51 mg, 64%) as a white solid, using a similar procedure to that used in the preparation of compound <b>108</b>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.34 (s, 1H), 8.30 (t, <i>J</i> = 1.82 Hz, 1H), 8.11 (t, <i>J</i> = 1.56 Hz, 1H), 8.05 (d, <i>J</i> = 8.08 Hz, 2H), 7.91 (t, <i>J</i> = 1.40 Hz, 1H), 7.76 (d, <i>J</i> = 8.20 Hz, 2H); MS (ESI, <i>m</i>/<i>z</i>) 393.0, 395.0 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>16</sub>H<sub>9</sub>N<sub>4</sub>F<sub>3</sub><sup>79</sup>Br 392.9963, found 392.9957 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> <i>tert</i>-Butyl 4-(3′-(1<i>H</i>-Tetrazol-5-yl)-5′-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)-[1,1′-biphenyl]-4-yl)piperidine-1-carboxylate (<b>113</b>)</h5><div class="NLM_p last">To a solution of compound <b>108</b> (13 mg, 22.7 μmol) in dimethyl sulfoxide (1 mL) were added sodium azide (3.2 mg, 49.8 μmol) and copper sulfate hydrate (1.4 mg, 5.6 μmol), and then this reaction mixture was heated at 140 °C for 6 h. After the solvent was removed, the residue was purified by silica gel column chromatography (dichloromethane/methanol = 20:1) to afford compound <b>113</b> (10 mg, 69%) as a white solid; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.26 (s, 1H), 8.59 (s, 1H), 8.47 (s, 1H), 8.31 (s, 1H), 8.18 (d, <i>J</i> = 7.00 Hz, 2H), 7.82–7.80 (m, 4H), 7.44–7.43 (m, 2H), 4.27–4.24 (m, 2H), 2.92–2.83 (m, 3H), 1.92–1.88 (m, 2H), 1.77–1.62 (m, 2H), 1.59 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 561.2 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>28</sub>H<sub>24</sub>N<sub>8</sub>O<sub>2</sub>F<sub>3</sub> 561.1974, found 561.1969 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> <i>tert</i>-Butyl (3-(5-(3-(1<i>H</i>-Tetrazol-5-yl)-5-(4-(4-(trifluoromethyl)phenyl)-1<i>H</i>-1,2,3-triazol-1-yl)phenyl)thiophene-2-carboxamido)propyl)carbamate (<b>114</b>)</h5><div class="NLM_p last">Compound <b>109</b> (24 mg, 0.065 mmol) was converted to compound <b>114</b> (20 mg, 78%) as a white solid, using a similar procedure to that used in the preparation of compound <b>113</b>; Semipreparative HPLC (10 mM triethylammonium acetate buffer/acetonitrile = 20:80 to 80:20 in 40 min): <i>R</i><sub>t</sub> = 35.9 min; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 9.27 (s, 1H), 8.58 (s, 2H), 8.32 (s, 1H) 8.21 (d, <i>J</i> = 8.00 Hz, 2H), 7.82 (d, <i>J</i> = 8.24 Hz, 2H), 7.76 (d, <i>J</i> = 3.88 Hz, 1H), 7.70 (d, <i>J</i> = 3.88 Hz, 1H), 3.44 (t, <i>J</i> = 6.84 Hz, 2H), 3.17 (t, <i>J</i> = 6.80 Hz, 2H), 1.79 (t, <i>J</i> = 6.62 Hz, 2H), 1.47 (s, 9H); MS (ESI, <i>m</i>/<i>z</i>) 540.1 [M + 1-Boc]<sup>+</sup>, 584.2 [M + 1-isopropyl]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>25</sub>H<sub>21</sub>N<sub>9</sub>O<sub>3</sub>F<sub>3</sub>S 584.1440, found 584.1435 [M + 1-isopropyl]<sup>+</sup>.</div></div><div id="sec4_4_2_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> <i>tert</i>-Butyl (6-Hydroxyhexyl)carbamate (<b>116</b>)</h5><div class="NLM_p last">To a solution of 6-amino-1-hexanol (<b>115</b>, 200 mg, 1.71 mmol) in dichloromethane (2.5 mL) was added dropwise a solution of di-<i>tert</i>-butyl dicarbonate (410 mg, 1.88 mmol) in dichloromethane (2.5 mL). After stirring for 30 min, the reaction mixture was partitioned with dichloromethane (10 mL) and water (5 mL). The aqueous layer was extracted with dichloromethane (5 mL × 2), and then the organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure to afford compound <b>116</b> (296 mg, 80%) as a white solid; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.75 (s, 1H; N<i>H</i>), 4.32 (t, <i>J</i> = 5.16 Hz, 1H; O<i>H</i>), 3.37 (q, <i>J</i> = 5.24 Hz, 2H), 2.88 (q, <i>J</i> = 6.76 Hz, 2H), 1.43–1.30 (s, 13H), 1.29–1.21 (m, 4H).</div></div><div id="sec4_4_2_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> <i>tert</i>-Butyl (6-Azidohexyl)carbamate (<b>117</b>)</h5><div class="NLM_p last">To a solution of compound <b>116</b> (150 mg, 0.69 mmol) in toluene (5 mL) were added triethylamine (96 μL, 70 mg, 0.69 mmol) and methanesulfonyl chloride (53 μL, 70 mg, 0.69 mmol) at 0 °C. After the mixture stirred at 0 °C for 30 min, tetrabutylammonium bromide (20 mg, 0.069 mmol) and a solution of sodium azide (269 mg, 4.14 mmol) in water (2.5 mL) were added to the reaction mixture, which was stirred at 100 °C for 15 h. The mixture was partitioned with diethyl ether (10 mL) and water (5 mL), and the organic layer was washed with brine (3 mL), dried over MgSO<sub>4</sub>, filtered, and evaporated to afford compound <b>117</b> (99 mg, 59%) as a yellow syrup; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.52 (s, 1H; N<i>H</i>), 3.28 (t, <i>J</i> = 6.88 Hz, 2H), 3.13 (q, <i>J</i> = 6.12 Hz, 2H), 1.66–1.58 (m, 2H), 1.54–1.46 (m, 2H), 1.46 (s, 9H), 1.43–1.35 (m, 4H); MS (ESI, <i>m</i>/<i>z</i>) 243.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>11</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub> 243.1821, found 243.1821 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> <i>tert</i>-Butyl 4-(4-(3-(Ethoxycarbonyl)-6-(4-(trifluoromethyl)phenyl)naphthalen-1-yl)phenyl)piperidine-1-carboxylate (<b>120</b>)</h5><div class="NLM_p last">A mixture of compound <b>119</b> (150 mg, 0.319 mmol), which was synthesized according to literature procedures reported,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and compound <b>55</b> (123 mg, 0.361 mmol) in dimethylformamide (5 mL) was purged with nitrogen gas for 30 min, and then Pd(PPh<sub>3</sub>)<sub>4</sub> (22 mg, 0.019 mmol) and potassium carbonate (132 mg, 0.957 mmol) were added to the mixture, which was stirred at 80 °C for 2 h. The reaction mixture was partitioned ethyl acetate (20 mL) and water (10 mL), and then the aqueous layer was extracted with ethyl acetate (10 mL × 2). The combined organic layer was washed with brine (5 mL), dried (MgSO<sub>4</sub>), filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography (hexane/ethyl actate = 10:1) to afford compound <b>120</b> (113 mg, 59%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.70 (s, 1H), 8.25 (d, <i>J</i> = 1.68 Hz, 1H), 8.09–8.07 (m, 2H), 7.86 (d, <i>J</i> = 8.24 Hz, 2H), 7.81–7.77 (m, 3H), 7.50 (d, <i>J</i> = 8.12, 2H), 7.38 (d, <i>J</i> = 8.08 Hz, 2H), 4.48 (q, <i>J</i> = 7.12 Hz, 2H), 4.32 (d, <i>J</i> = 13.4 Hz, 2H), 2.92–2.86 (m, 2H), 2.82–2.76 (m, 1H), 1.96 (d, <i>J</i> = 12.4 Hz, 2H), 1.80–1.68 (m, 2H), 1.52 (s, 9H), 1.47 (t, <i>J</i> = 7.12 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 548.2 [M + 1-isopropyl]<sup>+</sup>, 504.2 [M + 1-Boc]<sup>+</sup>.</div></div><div id="sec4_4_2_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> Ethyl 4-(4-(Piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoate (<b>121</b>)</h5><div class="NLM_p last">A mixture of compound <b>120</b> (160 mg, 0.265 mmol) in trifluoroacetic acid (2 mL) and tetrahydrofuran (2 mL) was stirred at room temperature for 1 h. The solvent was evaporated with methanol under reduced pressure. The residue was purified by silica gel column chromatography (dichloromethane/methanol = 10:1) to afford compound <b>121</b> (153 mg, 93%; TFA salt form) as a white solid; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.76 (s, 1H), 8.46 (d, <i>J</i> = 1.64 Hz, 1H), 8.04–8.01 (m, 3H), 7.98–7.94 (m, 2H), 7.82 (d, <i>J</i> = 8.12 Hz, 2H), 7.56–7.49 (m, 4H), 4.47 (q, <i>J</i> = 7.12 Hz, 2H), 3.58 (d, <i>J</i> = 12.32 Hz, 2H), 3.26–3.20 (m, 2H), 3.15–3.05 (m, 1H), 2.22 (d, <i>J</i> = 13.24 Hz, 2H), 2.09–1.98 (m, 2H), 1.47 (t, <i>J</i> = 7.12 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 504.2 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>31</sub>H<sub>29</sub>NO<sub>2</sub>F<sub>3</sub> 504.2150, found 504.2157 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> Ethyl 4-(4-(1-(Hex-5-yn-1-yl)piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoate (<b>122</b>)</h5><div class="NLM_p last">Potassium carbonate (68 mg, 0.24 mmol) was added to a solution of compound <b>121</b> (100 mg, 0.16 mmol) in dry <i>N</i>,<i>N</i>-dimethylformamide (5 mL), and the reaction mixture was stirred for 20 min. 6-Bromohexyne-1 (0.12 mL, 0.40 mmol) was subsequently added, and the reaction mixture was first stirred at room temperature for 2 h and then at 50 °C for 2.5 h. The reaction mixture was partitioned with saturated NaHCO<sub>3</sub> solution (5 mL) and ethyl acetate (10 mL). The aqueous layer was extracted with ethyl acetate (5 mL × 2), and then the combine organic layer was washed with brine (5 mL), dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The residue was purified by silica column chromatography (hexane/ethyl acetate = 6:1) to afford compound <b>122</b> (66 mg, 70%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.70 (s, 1H), 8.24 (d, <i>J</i> = 1.80 Hz, 1H), 8.05 (d, <i>J</i> = 1.60 Hz, 1H), 8.02 (d, <i>J</i> = 8.84 Hz, 1H), 7.84 (d, <i>J</i> = 8.16 Hz, 2H), 7.81–7.76 (m, 3H), 7.52 (d, <i>J</i> = 8.20 Hz, 2H), 7.47 (d, <i>J</i> = 8.20 Hz, 2H), 4.48 (q, <i>J</i> = 7.08 Hz, 2H), 3.77 (d, <i>J</i> = 10.36 Hz, 2H), 3.10–3.04 (m, 2H), 2.92–2.79 (m, 4H), 2.36–2.32 (m, 2H), 2.20–2.14 (m, 4H), 2.03 (t, <i>J</i> = 2.62 Hz, 1H), 1.72–1.65 (m, 2H), 1.47 (t, <i>J</i> = 7.12 Hz, 3H); MS (ESI, <i>m</i>/<i>z</i>) 584.3 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>37</sub>H<sub>37</sub>NO<sub>2</sub>F<sub>3</sub> 584.2776, found 584.2783 [M + 1]<sup>+</sup>.</div></div><div id="sec4_4_2_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> Ethyl 4-(4-(1-(4-(1-(6-((<i>tert</i>-Butoxycarbonyl)amino)hexyl)-1<i>H</i>-1,2,3-triazol-4-yl)butyl)piperidin-4-yl)phenyl)-7-(4-(trifluoromethyl)phenyl)-2-naphthoate (<b>123</b>)</h5><div class="NLM_p last">To a solution of compound <b>122</b> (40 mg, 68.5 μmol) in CH<sub>2</sub>Cl<sub>2</sub>/<i>tert</i>-BuOH:H<sub>2</sub>O (1:1:1; 2 mL), compound <b>117</b> (50 mg, 0.203 mmol) was added, followed by copper(II) sulfate pentahydrate (2.4 mg, 10.3 μmol) and sodium ascorbate (6.4 mg, 30.9 μmol). The reaction mixture was stirred at room temperature for 24 h. The solvents were removed under vacuum, and the residue was rinsed with 37% ammonia solution (5 mL) and extracted with ethyl acetate (10 mL × 2). The combined organic layer was washed with brine (5 mL), dried over MgSO<sub>4</sub>, filtered, and the evaporated under reduced pressure. The residue was purified by silica gel column chromatography (dichloromethane/methanol = 20:1) to afford compound <b>123</b> (33 mg, 58%) as a white solid; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.70 (s, 1H), 8.24 (d, <i>J</i> = 1.68 Hz, 1H), 8.06 (dd, <i>J</i> = 3.58, 5.18 Hz, 2H), 7.84 (d, <i>J</i> = 8.20 Hz, 2H), 7.81–7.76 (m, 3H), 7.49 (d, <i>J</i> = 8.12 Hz, 2H), 7.43 (d, <i>J</i> = 8.12 Hz, 2H), 7.38 (s, 1H), 4.56 (s, 1H), 4.47 (q, <i>J</i> = 7.12 Hz, 2H), 4.34 (t, <i>J</i> = 7.18 Hz, 2H), 3.39 (s, 1H), 3.14–3.09 (m, 2H), 2.83–2.75 (m, 4H), 2.53–2.15 (m, 2H), 2.06–2.04 (m, 2H), 1.94–1.81 (m, 6H), 1.49–1.46 (m, 16H), 1.38–1.36 (m, 6H); MS (ESI, <i>m</i>/<i>z</i>) 826.4 [M + 1]<sup>+</sup>; ESI-HRMS calcd. <i>m</i>/<i>z</i> for C<sub>48</sub>H<sub>59</sub>N<sub>5</sub>O<sub>4</sub>F<sub>3</sub> 826.4519 found 826.4528 [M + 1]<sup>+</sup>.</div></div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> Pharmacological Assays</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Cell Culture</h4><div class="NLM_p last">Chinese hamster ovary cells stably expressing the hP2Y<sub>14</sub>-R (CHO-hP2Y<sub>14</sub>R) were grown in Dulbecco’s modified Eagle’s medium (DMEM)/Ham’s F12 (F12) 1:1 supplemented with 10% FBS, 100 units/mL penicillin, 100 mg/mL streptomycin, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and 0.500 mg/mL G418 Sulfate (Geneticin). Cells were maintained in a humidified atmosphere and sterile incubation conditions held at 37 °C and 5% CO<sub>2</sub>(g).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Competitive Assay</h4><div class="NLM_p">Competitive fluorescent assays were performed on a BD FACSCalibur flow cytometer in conjunction with the software BD Bioscience PlateManager and CellQuest. All cell culture growth and assays for this procedure were conducted on flat-bottom 96-well plates. CHO-hP2Y<sub>14</sub>R cells were grown to approximately 80–90% confluency prior to assays. The 96-well plate format enabled four compounds to be analyzed in triplicate per run. All unlabeled ligand compounds are stored as 5 mM stock solutions in dimethyl sulfoxide (DMSO). Serial dilutions of each compound were prepared in complete medium. Cells were initially incubated with unlabeled compounds for 30 min at 37 °C and 5% CO<sub>2</sub>(g). Cells were then incubated with the fluorescent labeled (AlexaFluor 488) ligand MRS4174 for 30 min at a final concentration of 20 nM. After three consecutive washes in sterile 1× Dulbecco’s Phosphate Buffered Saline (DPBS) minus Ca<sup>2+</sup>/Mg<sup>2+</sup>, cells were detached from the plate using Corning Cellstripper to reduce damaging the hP2Y<sub>14</sub>R protein. Final cell suspensions for flow cytometry was in DPBS minus Ca<sup>2+</sup>/Mg<sup>2+</sup>.</div><div class="NLM_p last">IC<sub>50</sub> values were determined from the gathered data with the program GraphPad Prism version 7.0.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Calculation of Pharmacokinetic Properties</h3><div class="NLM_p">cLogP was calculated using <a href="http://biosig.unimelb.edu.au/pkcsm/prediction" class="extLink">http://biosig.unimelb.edu.au/pkcsm/prediction</a>.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> cLogS and cLogD were calculated as a function of pH using <a href="https://disco.chemaxon.com/apps/demos/" class="extLink">https://disco.chemaxon.com/apps/demos/</a>. Pharmacokinetic properties were calculated using <a href="http://biosig.unimelb.edu.au/pkcsm/prediction" class="extLink">http://biosig.unimelb.edu.au/pkcsm/prediction</a>, as described by Pires et al.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p last">The in vivo pharmacokinetics and in vitro preclinical testing of compound <b>30</b> was measured by GVK Biosciences Private Limited, Hyderabad, India using a protocol we previously described.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The vehicle used was 15% DMSO (in which the compound was initially dissolved, containing 10 equiv of aq. NaHCO<sub>3</sub>), 15% Kolliphor ELP (Sigma 30906, CAS 61791–12–6), and 70% PBS, at a max. concentration of 2 mg/mL for the highest p.o. dose of 10 mg/kg.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i117"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00168" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23610" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23610" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00168" class="ext-link">10.1021/acs.jmedchem.8b00168</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis of compounds <b>55</b>–<b>78</b>, off-target binding inhibition curves, pharmacological data, molecular modeling data, including study of the interactions with ELs; selection of the optimal spacer geometry; lowest IE structure; structure and docking scores of designed compounds; replicate analysis of MD simulations; predicted pharmacokinetic parameters; and predicted LogD and LogS (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">NMR and mass spectra of triazole derivatives (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Three-dimensional coordinates of hP2Y<sub>14</sub>R in complex with <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_003.pdb" class="ext-link">PDB</a>) and <b>20</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Videos of MD simulations of hP2Y<sub>14</sub>R complexes with <b>9</b> compared to <b>20</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi" class="ext-link">AVI</a>) and with <b>30</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_006.avi" class="ext-link">AVI</a>) and with <b>2a</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_007.avi" class="ext-link">AVI</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_008.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf">jm8b00168_si_001.pdf (1.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_002.pdf">jm8b00168_si_002.pdf (3.45 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_003.pdb">jm8b00168_si_003.pdb (186.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_004.pdb">jm8b00168_si_004.pdb (186.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi">jm8b00168_si_005.avi (27.87 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_006.avi">jm8b00168_si_006.avi (11.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_007.avi">jm8b00168_si_007.avi (17.56 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_008.csv">jm8b00168_si_008.csv (4.37 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00168" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54441" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54441" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth A. Jacobson</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8104-1493" title="Orcid link">http://orcid.org/0000-0001-8104-1493</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b7dcd2d9d9d2c3dfddf7d9ded3d3dc99d9dedf99d0d8c1"><span class="__cf_email__" data-cfemail="4229272c2c27362a28022c2b2626296c2c2b2a6c252d34">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinha Yu</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonella Ciancetta</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7612-2050" title="Orcid link">http://orcid.org/0000-0002-7612-2050</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven Dudas</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sierra Duca</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justine Lottermoser</span> - <span class="hlFld-Affiliation affiliation">Molecular Recognition Section,
Laboratory of Bioorganic Chemistry, National Institute of Diabetes
and Digestive and Kidney Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e9340-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i119">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75613" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75613" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge support from the NIDDK Intramural Research Program (ZIA DK031116-29). We thank Bryan L. Roth, National Institute of Mental Health’s Psychoactive Drug Screening Program (Univ. North Carolina at Chapel Hill, Contract # HHSN-271-2008-00025-C) for screening data. We thank Rolf Swenson and Zhen-Dan Shi (Imaging Probe Development Center, NHLBI, HHSN261200800001E) for reagent synthesis and Young-Hwan Jung (NIDDK) for NMR spectra.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AF488</td><td class="NLM_def"><p class="first last">Alexa Fluor 488</p></td></tr><tr><td class="NLM_term">CHO</td><td class="NLM_def"><p class="first last">Chinese hamster ovary</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DPPF</td><td class="NLM_def"><p class="first last">1,1′-bis(diphenylphosphino)ferrocene</p></td></tr><tr><td class="NLM_term">ECL</td><td class="NLM_def"><p class="first last">extracellular loop</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-ethyl-<i>N</i>′-dimethylaminopropylcarbodiimide</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">IE</td><td class="NLM_def"><p class="first last">interaction energy</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">PDSP</td><td class="NLM_def"><p class="first last">Psychoactive Drug Screening Program</p></td></tr><tr><td class="NLM_term">PLC</td><td class="NLM_def"><p class="first last">phospholipase C</p></td></tr><tr><td class="NLM_term">PPTN</td><td class="NLM_def"><p class="first last">4-(4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl)-2-naphthoic acid</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">UDPG</td><td class="NLM_def"><p class="first last">uridine-5′-diphosphoglucose</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TM</td><td class="NLM_def"><p class="first last">transmembrane helix</p></td></tr><tr><td class="NLM_term">TSTU</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyl-O-(<i>N</i>-succinimidyl)-uronium tetrafluoroborate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i121">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnstock, G.</span></span> <span> </span><span class="NLM_article-title">Purinergic signalling: Therapeutic developments</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">661</span>, <span class="refDoi"> DOI: 10.3389/fphar.2017.00661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.3389%2Ffphar.2017.00661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=28993732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFegs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=661&author=G.+Burnstock&title=Purinergic+signalling%3A+Therapeutic+developments&doi=10.3389%2Ffphar.2017.00661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Purinergic signalling: therapeutic developments</span></div><div class="casAuthors">Burnstock, Geoffrey</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">661/1-661/55</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Purinergic signalling, i.e., the role of nucleotides as extracellular signalling mols., was proposed in 1972.  However, this concept was not well accepted until the early 1990-s when receptor subtypes for purines and pyrimidines were cloned and characterised, which includes four subtypes of the P1 (adenosine) receptor, seven subtypes of P2X ion channel receptors and 8 subtypes of the P2Y G protein-coupled receptor.  Early studies were largely concerned with the physiol., pharmacol. and biochem. of purinergic signalling.  More recently, the focus has been on the pathophysiol. and therapeutic potential.  There was early recognition of the use of P1 receptor agonists for the treatment of supraventricular tachycardia and A2A receptor antagonists are promising for the treatment of Parkinson-s disease.  Clopidogrel, a P2Y12 antagonist, is widely used for the treatment of thrombosis and stroke, blocking P2Y12 receptor-mediated platelet aggregation.  Diquafosol, a long acting P2Y2 receptor agonist, is being used for the treatment of dry eye.  P2X3 receptor antagonists have been developed that are orally bioavailable and stable in vivo and are currently in clin. trials for the treatment of chronic cough, bladder incontinence, visceral pain and hypertension.  Antagonists to P2X7 receptors are being investigated for the treatment of inflammatory disorders, including neurodegenerative diseases.  Other investigations are in progress for the use of purinergic agents for the treatment of osteoporosis, myocardial infarction, irritable bowel syndrome, epilepsy, atherosclerosis, depression, autism, diabetes, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9VdnZ8zYY7Vg90H21EOLACvtfcHk0lhuFTdUyZ-2gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFegs7fK&md5=b28408418adacae80284bcf51c778b3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2017.00661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2017.00661%26sid%3Dliteratum%253Aachs%26aulast%3DBurnstock%26aufirst%3DG.%26atitle%3DPurinergic%2520signalling%253A%2520Therapeutic%2520developments%26jtitle%3DFront.%2520Pharmacol.%26date%3D2017%26volume%3D8%26spage%3D661%26doi%3D10.3389%2Ffphar.2017.00661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cekic, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linden, J.</span></span> <span> </span><span class="NLM_article-title">Purinergic regulation of the immune system</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nri.2016.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnri.2016.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=26922909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtlKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=177-192&author=C.+Cekicauthor=J.+Linden&title=Purinergic+regulation+of+the+immune+system&doi=10.1038%2Fnri.2016.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Purinergic regulation of the immune system</span></div><div class="casAuthors">Cekic, Caglar; Linden, Joel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-192</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cellular stress or apoptosis triggers the release of ATP, ADP and other nucleotides into the extracellular space.  Extracellular nucleotides function as autocrine and paracrine signalling mols. by activating cell-surface P2 purinergic receptors that elicit pro-inflammatory immune responses.  Over time, extracellular nucleotides are metabolized to adenosine, leading to reduced P2 signalling and increased signalling through anti-inflammatory adenosine (P1 purinergic) receptors.  Here, we review how local purinergic signalling changes over time during tissue responses to injury or disease, and we discuss the potential of targeting purinergic signalling pathways for the immunotherapeutic treatment of ischemia, organ transplantation, autoimmunity or cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod6c3DxezmqbVg90H21EOLACvtfcHk0lhuFTdUyZ-2gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtlKmurg%253D&md5=930862a0d6cfd16d6c3cdefc45094d0d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.4%26sid%3Dliteratum%253Aachs%26aulast%3DCekic%26aufirst%3DC.%26aulast%3DLinden%26aufirst%3DJ.%26atitle%3DPurinergic%2520regulation%2520of%2520the%2520immune%2520system%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D177%26epage%3D192%26doi%3D10.1038%2Fnri.2016.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbracchio, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnstock, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeynaems, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gachet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisman, G. A.</span></span> <span> </span><span class="NLM_article-title">International Union of Pharmacology LVIII: update on the P2Y G protein-coupled nucleotide receptors: from molecular mechanisms and pathophysiology to therapy</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1124/pr.58.3.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fpr.58.3.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=16968944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVOhtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=281-341&author=M.+P.+Abbracchioauthor=G.+Burnstockauthor=J.+M.+Boeynaemsauthor=E.+A.+Barnardauthor=J.+L.+Boyerauthor=C.+Kennedyauthor=M.+Fumagalliauthor=B.+F.+Kingauthor=C.+Gachetauthor=K.+A.+Jacobsonauthor=G.+A.+Weisman&title=International+Union+of+Pharmacology+LVIII%3A+update+on+the+P2Y+G+protein-coupled+nucleotide+receptors%3A+from+molecular+mechanisms+and+pathophysiology+to+therapy&doi=10.1124%2Fpr.58.3.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">International Union of Pharmacology LVIII: update on the P2Y G protein-coupled nucleotide receptors: from molecular mechanisms and pathophysiology to therapy</span></div><div class="casAuthors">Abbracchio, Maria P.; Burnstock, Geoffrey; Boeynaems, Jean-Marie; Barnard, Eric A.; Boyer, Jose L.; Kennedy, Charles; Knight, Gillian E.; Fumagalli, Marta; Gachet, Christian; Jacobson, Kenneth A.; Weisman, Gary A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-341</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  There have been many advances in our knowledge about different aspects of P2Y receptor signaling since the last review published by our International Union of Pharmacol. subcommittee.  More receptor subtypes have been cloned and characterized and most orphan receptors deorphanized, so that it is now possible to provide a basis for a future subdivision of P2Y receptor subtypes.  More is known about the functional elements of the P2Y receptor mols. and the signaling pathways involved, including interactions with ion channels.  There have been substantial developments in the design of selective agonists and antagonists to some of the P2Y receptor subtypes.  There are new findings about the mechanisms underlying nucleotide release and ectoenzymic nucleotide breakdown.  Interactions between P2Y receptors and receptors to other signaling mols. have been explored as well as P2Y-mediated control of gene transcription.  The distribution and roles of P2Y receptor subtypes in many different cell types are better understood and P2Y receptor-related compds. are being explored for therapeutic purposes.  These and other advances are discussed in the present review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-m5Cf-JzaLVg90H21EOLACvtfcHk0lgpsgOtVUuZqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVOhtLbF&md5=b302ee34857a5f45d740e4f2e962ed42</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1124%2Fpr.58.3.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.58.3.3%26sid%3Dliteratum%253Aachs%26aulast%3DAbbracchio%26aufirst%3DM.%2BP.%26aulast%3DBurnstock%26aufirst%3DG.%26aulast%3DBoeynaems%26aufirst%3DJ.%2BM.%26aulast%3DBarnard%26aufirst%3DE.%2BA.%26aulast%3DBoyer%26aufirst%3DJ.%2BL.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DFumagalli%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DB.%2BF.%26aulast%3DGachet%26aufirst%3DC.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DWeisman%26aufirst%3DG.%2BA.%26atitle%3DInternational%2520Union%2520of%2520Pharmacology%2520LVIII%253A%2520update%2520on%2520the%2520P2Y%2520G%2520protein-coupled%2520nucleotide%2520receptors%253A%2520from%2520molecular%2520mechanisms%2520and%2520pathophysiology%2520to%2520therapy%26jtitle%3DPharmacol.%2520Rev.%26date%3D2006%26volume%3D58%26spage%3D281%26epage%3D341%26doi%3D10.1124%2Fpr.58.3.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span> <span> </span><span class="NLM_article-title">Signalling and pharmacological properties of the P2Y<sub>14</sub> receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1124/mol.115.098756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fmol.115.098756" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=151-160&author=E.+R.+Lazarowskiauthor=T.+K.+Harden&title=Signalling+and+pharmacological+properties+of+the+P2Y14+receptor&doi=10.1124%2Fmol.115.098756"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1124%2Fmol.115.098756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.115.098756%26sid%3Dliteratum%253Aachs%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26atitle%3DSignalling%2520and%2520pharmacological%2520properties%2520of%2520the%2520P2Y14%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D151%26epage%3D160%26doi%3D10.1124%2Fmol.115.098756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinckwich-Besancon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Mata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span> <span> </span><span class="NLM_article-title">The UDP-sugar-sensing P2Y<sub>14</sub> receptor promotes Rho-mediated signaling and chemotaxis in human neutrophils</span>. <i>Am. J. Physiol. - Cell Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">C490</span>– <span class="NLM_lpage">C498</span>, <span class="refDoi"> DOI: 10.1152/ajpcell.00138.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1152%2Fajpcell.00138.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22673622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2012&pages=C490-C498&author=J.+I.+Sesmaauthor=S.+M.+Kredaauthor=N.+Steinckwich-Besanconauthor=H.+Dangauthor=R.+Garc%C3%ADa-Mataauthor=T.+K.+Hardenauthor=E.+R.+Lazarowski&title=The+UDP-sugar-sensing+P2Y14+receptor+promotes+Rho-mediated+signaling+and+chemotaxis+in+human+neutrophils&doi=10.1152%2Fajpcell.00138.2012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1152%2Fajpcell.00138.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpcell.00138.2012%26sid%3Dliteratum%253Aachs%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DKreda%26aufirst%3DS.%2BM.%26aulast%3DSteinckwich-Besancon%26aufirst%3DN.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DGarc%25C3%25ADa-Mata%26aufirst%3DR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26atitle%3DThe%2520UDP-sugar-sensing%2520P2Y14%2520receptor%2520promotes%2520Rho-mediated%2520signaling%2520and%2520chemotaxis%2520in%2520human%2520neutrophils%26jtitle%3DAm.%2520J.%2520Physiol.%2520-%2520Cell%2520Physiol.%26date%3D2012%26volume%3D303%26spage%3DC490%26epage%3DC498%26doi%3D10.1152%2Fajpcell.00138.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayasekara, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span> <span> </span><span class="NLM_article-title">A selective high-affinity antagonist of the P2Y<sub>14</sub> receptor inhibits UDP-glucose–stimulated chemotaxis of human neutrophils</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1124/mol.113.085654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fmol.113.085654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=41-49&author=M.+O.+Barrettauthor=J.+I.+Sesmaauthor=C.+B.+Ballauthor=P.+S.+Jayasekaraauthor=K.+A.+Jacobsonauthor=E.+R.+Lazarowskiauthor=T.+K.+Harden&title=A+selective+high-affinity+antagonist+of+the+P2Y14+receptor+inhibits+UDP-glucose%E2%80%93stimulated+chemotaxis+of+human+neutrophils&doi=10.1124%2Fmol.113.085654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.085654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.085654%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DM.%2BO.%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DBall%26aufirst%3DC.%2BB.%26aulast%3DJayasekara%26aufirst%3DP.%2BS.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26atitle%3DA%2520selective%2520high-affinity%2520antagonist%2520of%2520the%2520P2Y14%2520receptor%2520inhibits%2520UDP-glucose%25E2%2580%2593stimulated%2520chemotaxis%2520of%2520human%2520neutrophils%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D41%26epage%3D49%26doi%3D10.1124%2Fmol.113.085654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">UDP-glucose acting at P2Y<sub>14</sub> receptors is a mediator of mast cell degranulation</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2009.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bcp.2009.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=19896471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=873-879&author=Z.-G.+Gaoauthor=Y.+Dingauthor=K.+A.+Jacobson&title=UDP-glucose+acting+at+P2Y14+receptors+is+a+mediator+of+mast+cell+degranulation&doi=10.1016%2Fj.bcp.2009.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-glucose acting at P2Y14 receptors is a mediator of mast cell degranulation</span></div><div class="casAuthors">Gao, Zhan-Guo; Ding, Yi; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">873-879</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">UDP-glucose (UDPG), a glycosyl donor in the biosynthesis of carbohydrates, is an endogenous agonist of the G protein-coupled P2Y14 receptor.  RBL-2H3 mast cells endogenously express a P2Y14 receptor at which UDPG mediates degranulation as indicated by β-hexosaminidase (HEX) release.  Both UDPG and a more potent, selective 2-thio-modified UDPG analog, MRS2690 (diphosphoric acid 1-α--glucopyranosyl ester 2-[(2-thio)uridin-5''-yl] ester), caused a substantial calcium transient in RBL-2H3 cells, which was blocked by pertussis toxin, indicating the presence of the Gi-coupled P2Y14 receptor, supported also by quant. detection of abundant mRNA.  Expression of the closely related P2Y6 receptor was over 100 times lower than the P2Y14 receptor, and the P2Y6 agonist 3-phenacyl-UDP was inactive in RBL-2H3 cells.  P2Y14 receptor agonists also induced [35S]GTPγS binding to RBL-2H3 cell membranes, and phosphorylation of ERK1/2, P38 and JNK.  UDPG and MRS2690 concn.-dependently enhanced HEX release with EC50 values of 1150 and 103 nM, resp.  The enhancement was completely blocked by pertussis toxin and significantly diminished by P2Y14 receptor-specific siRNA.  Thus, mast cells express an endogenous P2Y14 receptor, which mediates Gi-dependent degranulation and is therefore a potential novel therapeutic target for allergic conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0MRGdKvsSbbVg90H21EOLACvtfcHk0lgpsgOtVUuZqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFGjsg%253D%253D&md5=cda30b06aac8a61c30aa405076bcb7df</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2009.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2009.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DUDP-glucose%2520acting%2520at%2520P2Y14%2520receptors%2520is%2520a%2520mediator%2520of%2520mast%2520cell%2520degranulation%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D873%26epage%3D879%26doi%3D10.1016%2Fj.bcp.2009.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azroyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez-Retamozo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittet, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breton, S.</span></span> <span> </span><span class="NLM_article-title">Renal intercalated cells sense and mediate sterile inflammation via the P2Y<sub>14</sub> receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e0121419</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0121419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1371%2Fjournal.pone.0121419" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121419&issue=3&author=A.+Azroyanauthor=V.+Cortez-Retamozoauthor=R.+Bouleyauthor=R.+Libermanauthor=Y.+C.+Ruanauthor=E.+Kiselevauthor=K.+A.+Jacobsonauthor=M.+J.+Pittetauthor=D.+Brownauthor=S.+Breton&title=Renal+intercalated+cells+sense+and+mediate+sterile+inflammation+via+the+P2Y14+receptor&doi=10.1371%2Fjournal.pone.0121419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121419%26sid%3Dliteratum%253Aachs%26aulast%3DAzroyan%26aufirst%3DA.%26aulast%3DCortez-Retamozo%26aufirst%3DV.%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DLiberman%26aufirst%3DR.%26aulast%3DRuan%26aufirst%3DY.%2BC.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DPittet%26aufirst%3DM.%2BJ.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBreton%26aufirst%3DS.%26atitle%3DRenal%2520intercalated%2520cells%2520sense%2520and%2520mediate%2520sterile%2520inflammation%2520via%2520the%2520P2Y14%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D3%26spage%3De0121419%26doi%3D10.1371%2Fjournal.pone.0121419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morinaga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukdar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olefsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. J.</span></span> <span> </span><span class="NLM_article-title">GPR105 Ablation prevents inflammation and improves insulin sensitivity in mice with diet-induced obesity</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">1992</span>– <span class="NLM_lpage">1999</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1103207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.4049%2Fjimmunol.1103207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22778393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFaqsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2012&pages=1992-1999&author=J.+Xuauthor=H.+Morinagaauthor=D.+Ohauthor=P.+Liauthor=A.+Chenauthor=S.+Talukdarauthor=E.+Lazarowskiauthor=J.+M.+Olefskyauthor=J.+J.+Kim&title=GPR105+Ablation+prevents+inflammation+and+improves+insulin+sensitivity+in+mice+with+diet-induced+obesity&doi=10.4049%2Fjimmunol.1103207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">GPR105 Ablation Prevents Inflammation and Improves Insulin Sensitivity in Mice with Diet-Induced Obesity</span></div><div class="casAuthors">Xu, Jianfeng; Morinaga, Hidetaka; Oh, Dayoung; Li, Pingping; Chen, Ai; Talukdar, Saswata; Mamane, Yaeel; Mancini, Joseph A.; Nawrocki, Andrea R.; Lazarowski, Eduardo; Olefsky, Jerrold M.; Kim, Jane J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1992-1999</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">GPR105, a G protein-coupled receptor for UDP-glucose, is highly expressed in several human tissues and participates in the innate immune response.  Because inflammation has been implicated as a key initial trigger for type 2 diabetes, we hypothesized that GPR105 (official gene name: P2RY14) might play a role in the initiation of inflammation and insulin resistance in obesity.  To this end, we investigated glucose metab. in GPR105 knockout (KO) and wild-type (WT) mice fed a high-fat diet (HFD).  We also examd. whether GPR105 regulates macrophage recruitment to liver or adipose tissues by in vivo monocyte tracking and in vitro chemotaxis expts., followed by transplantation of bone marrow from either KO or WT donors to WT recipients.  Our data show that genetic deletion of GPR105 confers protection against HFD-induced insulin resistance, with reduced macrophage infiltration and inflammation in liver, and increased insulin-stimulated Akt phosphorylation in liver, muscle, and adipose tissue.  By tracking monocytes from either KO or WT donors, we found that fewer KO monocytes were recruited to the liver of WT recipients.  Furthermore, we obsd. that uridine 5-diphosphoglucose enhanced the in vitro migration of bone marrow-derived macrophages from WT but not KO mice, and that plasma uridine 5-diphosphoglucose levels were significantly higher in obese vs. lean mice.  Finally, we confirmed that insulin sensitivity improved in HFD mice with a myeloid cell-specific deletion of GPR105.  These studies indicate that GPR105 ablation mitigates HFD-induced insulin resistance by inhibiting macrophage recruitment and tissue inflammation.  Hence GPR105 provides a novel link between innate immunity and metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE2qDmFMoeT7Vg90H21EOLACvtfcHk0li8hmF-qcQ0dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFaqsLzM&md5=3c495773f43348d3731f85ee3e207072</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1103207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1103207%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMorinaga%26aufirst%3DH.%26aulast%3DOh%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DTalukdar%26aufirst%3DS.%26aulast%3DLazarowski%26aufirst%3DE.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BJ.%26atitle%3DGPR105%2520Ablation%2520prevents%2520inflammation%2520and%2520improves%2520insulin%2520sensitivity%2520in%2520mice%2520with%2520diet-induced%2520obesity%26jtitle%3DJ.%2520Immunol.%26date%3D2012%26volume%3D189%26spage%3D1992%26epage%3D1999%26doi%3D10.4049%2Fjimmunol.1103207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasu-Tada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujishita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, S.</span></span> <span> </span><span class="NLM_article-title">Secretion of matrix metalloproteinase-9 from astrocytes by inhibition of tonic P2Y<sub>14</sub>-receptor-mediated signal(s)</span>. <i>Cell. Mol. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1007/s10571-012-9869-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs10571-012-9869-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22872320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFSnsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=47-58&author=M.+Kinoshitaauthor=K.+Nasu-Tadaauthor=K.+Fujishitaauthor=K.+Satoauthor=S.+Koizumi&title=Secretion+of+matrix+metalloproteinase-9+from+astrocytes+by+inhibition+of+tonic+P2Y14-receptor-mediated+signal%28s%29&doi=10.1007%2Fs10571-012-9869-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Secretion of Matrix Metalloproteinase-9 from Astrocytes by Inhibition of Tonic P2Y14-Receptor-Mediated Signal(s)</span></div><div class="casAuthors">Kinoshita, Manao; Nasu-Tada, Kaoru; Fujishita, Kayoko; Sato, Kaoru; Koizumi, Schuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Neurobiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">CMNEDI</span>;
        ISSN:<span class="NLM_cas:issn">0272-4340</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Glial cells have various important roles in regulation of brain functions.  For such events, extracellular nucleotides/P2 receptors have central roles.  Although there have been huge amt. of literature about activation of P2 receptors and glial functions, little is known about what happens in glia or the brain if glial P2 receptor is inhibited.  Here we show that the inhibition of P2 receptors in astrocytes, the most abundant glial cells and cause a constitutive release of nucleotides, resulted in secretion of metalloproteinase-9 (MMP-9), a metal-dependent endopeptidase that degrades extracellular matrix mols. and is important in regulation of brain remodeling.  When cultured astrocytes were treated with apyrase (ecto-nucleotidase), reactive blue 2 (P2 receptor antagonist), and pertussis toxin, they secreted MMP-9, suggesting that Gi-coupled P2Y receptor-mediated signals constitutively suppress the prodn. of MMP-9.  Among Gi-coupled P2Y receptors, we found that an inhibition of P2Y14 receptor, a receptor for nucleotide-sugars such as UDP-glucose, is responsible for the prodn. of MMP-9 by pharmacol. and mol. biochem. anal.  As for the mechanisms, the inhibition of P2Y14 receptors resulted in the release of tumor necrosis factor (TNF)-α which then acted on astrocytes to induce MMP-9.  Taken together, our results suggest that the constitutive releases of nucleotide-sugars in astrocytes should play an important role in maintaining the normal status of the cell, through Gi-coupled P2Y14 receptors, and when the signal is removed, the cells start to release TNF-α, which then acts on astrocytes in a feedback fashion to boost MMP-9 synthesis and secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8KyPG_sOekLVg90H21EOLACvtfcHk0li8hmF-qcQ0dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFSnsLjJ&md5=c0c76a48491f9c006e522875c9a39c71</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10571-012-9869-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10571-012-9869-4%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DM.%26aulast%3DNasu-Tada%26aufirst%3DK.%26aulast%3DFujishita%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DKoizumi%26aufirst%3DS.%26atitle%3DSecretion%2520of%2520matrix%2520metalloproteinase-9%2520from%2520astrocytes%2520by%2520inhibition%2520of%2520tonic%2520P2Y14-receptor-mediated%2520signal%2528s%2529%26jtitle%3DCell.%2520Mol.%2520Neurobiol.%26date%3D2013%26volume%3D33%26spage%3D47%26epage%3D58%26doi%3D10.1007%2Fs10571-012-9869-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanamoto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okubo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, K.</span></span> <span> </span><span class="NLM_article-title">Multiple P2Y subtypes in spinal microglia are involved in neuropathic pain after peripheral nerve injury</span>. <i>Glia</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1539</span>, <span class="refDoi"> DOI: 10.1002/glia.22373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2Fglia.22373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22736439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A280%3ADC%252BC38jltFenug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=1529-1539&author=K.+Kobayashiauthor=H.+Yamanakaauthor=F.+Yanamotoauthor=M.+Okuboauthor=K.+Noguchi&title=Multiple+P2Y+subtypes+in+spinal+microglia+are+involved+in+neuropathic+pain+after+peripheral+nerve+injury&doi=10.1002%2Fglia.22373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple P2Y subtypes in spinal microglia are involved in neuropathic pain after peripheral nerve injury</span></div><div class="casAuthors">Kobayashi Kimiko; Yamanaka Hiroki; Yanamoto Fujio; Okubo Masamichi; Noguchi Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Glia</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1529-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A prominent signaling pathway in the development of neuropathic pain involves ATP acting on microglial purinergic receptors.  Among the P2Y metabotropic receptors, we reported before that the P2Y12 receptor is upregulated in microglia following nerve injury and involved in the phosphorylation of p38 MAPK, and in the development of pain behavior.  In this study, we examined the expression of P2Y6, P2Y13, and P2Y14 receptors in the spinal cord and whether these receptors are involved in the pathogenesis of neuropathic pain following peripheral nerve injury.  We found that spared nerve injury induced a dramatic increase of not only P2Y12, but also P2Y6, 13, and 14 receptor mRNA expression in spinal microglia.  The increase continued for at least 2 weeks after injury.  To determine whether p38 MAPK can induce the expression of P2Y receptors, we administered intrathecally the p38 MAPK inhibitor SB203580 and found that it significantly suppressed P2Y6, P2Y13, and P2Y14 but not P2Y12 mRNAs.  Intrathecal injection of the specific P2Y6 antagonist MRS2578, specific P2Y13 antagonist MRS2211 or P2Y14 antisense LNA, attenuated mechanical pain hypersensitivity.  The mixture of three antagonists for P2Y6, 12, and 13 showed a longer suppressive effect on pain behavior than the individual treatments.  Our data demonstrate that ATP and other nucleotides may stimulate activated microglia with the upregulation of P2Y6, P2Y12, P2Y13, and P2Y14 receptors following nerve injury and these receptors are involved in the development of neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGSxmgq31PIrHHuJhbyicZfW6udTcc2eb7PpkoP3OscLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jltFenug%253D%253D&md5=3a19972be03492267641a89519b8e5bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fglia.22373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fglia.22373%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DYamanaka%26aufirst%3DH.%26aulast%3DYanamoto%26aufirst%3DF.%26aulast%3DOkubo%26aufirst%3DM.%26aulast%3DNoguchi%26aufirst%3DK.%26atitle%3DMultiple%2520P2Y%2520subtypes%2520in%2520spinal%2520microglia%2520are%2520involved%2520in%2520neuropathic%2520pain%2520after%2520peripheral%2520nerve%2520injury%26jtitle%3DGlia%26date%3D2012%26volume%3D60%26spage%3D1529%26epage%3D1539%26doi%3D10.1002%2Fglia.22373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzer, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livraghi-Butrico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexis, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span> <span> </span><span class="NLM_article-title">UDP-glucose promotes neutrophil recruitment in the lung</span>. <i>Purinergic Signalling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1007/s11302-016-9524-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs11302-016-9524-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=27421735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyntrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=627-635&author=J.+I.+Sesmaauthor=C.+D.+Weitzerauthor=A.+Livraghi-Butricoauthor=H.+Dangauthor=S.+Donaldsonauthor=N.+E.+Alexisauthor=K.+A.+Jacobsonauthor=T.+K.+Hardenauthor=E.+R.+Lazarowski&title=UDP-glucose+promotes+neutrophil+recruitment+in+the+lung&doi=10.1007%2Fs11302-016-9524-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-glucose promotes neutrophil recruitment in the lung</span></div><div class="casAuthors">Sesma, Juliana I.; Weitzer, Clarissa D.; Livraghi-Butrico, Alessandra; Dang, Hong; Donaldson, Scott; Alexis, Neil E.; Jacobson, Kenneth A.; Harden, T. Kendall; Lazarowski, Eduardo R.</div><div class="citationInfo"><span class="NLM_cas:title">Purinergic Signalling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">627-635</span>CODEN:
                <span class="NLM_cas:coden">PSUIA9</span>;
        ISSN:<span class="NLM_cas:issn">1573-9538</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In addn. to their role in glycosylation reactions, UDP-sugars are released from cells and activate widely distributed cell surface P2Y14 receptors (P2Y14R).  However, the physiol./pathophysiol. consequences of UDP-sugar release are incompletely defined.  Here, we report that UDP-glucose levels are abnormally elevated in lung secretions from patients with cystic fibrosis (CF) as well as in a mouse model of CF-like disease, the βENaC transgenic (Tg) mouse.  Instillation of UDP-glucose into wild-type mouse tracheas resulted in enhanced neutrophil lung recruitment, and this effect was nearly abolished when UDP-glucose was co-instilled with the P2Y14R antagonist PPTN [4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl-2-naphthoic acid].  Importantly, administration of PPTN to βENaC-Tg mice reduced neutrophil lung inflammation.  These results suggest that UDP-glucose released into the airways acts as a local mediator of neutrophil inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcO6QglBl4zrVg90H21EOLACvtfcHk0lijDZYIoHpyGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyntrrI&md5=0d123a5581a97af280656d05a47166e5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs11302-016-9524-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11302-016-9524-5%26sid%3Dliteratum%253Aachs%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DWeitzer%26aufirst%3DC.%2BD.%26aulast%3DLivraghi-Butrico%26aufirst%3DA.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DDonaldson%26aufirst%3DS.%26aulast%3DAlexis%26aufirst%3DN.%2BE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26atitle%3DUDP-glucose%2520promotes%2520neutrophil%2520recruitment%2520in%2520the%2520lung%26jtitle%3DPurinergic%2520Signalling%26date%3D2016%26volume%3D12%26spage%3D627%26epage%3D635%26doi%3D10.1007%2Fs11302-016-9524-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peikert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ünal, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willecke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinöhl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Zur Mühlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirlik, A.</span></span> <span> </span><span class="NLM_article-title">Extracellular ATP induces vascular inflammation and atherosclerosis via purinergic receptor Y<sub>2</sub> in mice</span>. <i>Arterioscler., Thromb., Vasc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1586</span>, <span class="refDoi"> DOI: 10.1161/ATVBAHA.115.307397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1161%2FATVBAHA.115.307397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=27339459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ekt7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=1577-1586&author=P.+Stachonauthor=S.+Geisauthor=A.+Peikertauthor=A.+Heidenreichauthor=N.+A.+Michelauthor=F.+%C3%9Cnalauthor=N.+Hoppeauthor=B.+Dufnerauthor=L.+Schulteauthor=T.+Marchiniauthor=S.+Cickoauthor=K.+Ayataauthor=A.+Zechauthor=D.+Wolfauthor=I.+Hilgendorfauthor=F.+Willeckeauthor=J.+Rein%C3%B6hlauthor=C.+von+Zur+M%C3%BChlenauthor=C.+Bodeauthor=M.+Idzkoauthor=A.+Zirlik&title=Extracellular+ATP+induces+vascular+inflammation+and+atherosclerosis+via+purinergic+receptor+Y2+in+mice&doi=10.1161%2FATVBAHA.115.307397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular ATP Induces Vascular Inflammation and Atherosclerosis via Purinergic Receptor Y2 in Mice</span></div><div class="casAuthors">Stachon, Peter; Geis, Serjosha; Peikert, Alexander; Heidenreich, Adrian; Michel, Nathaly Anto; Uenal, Fatih; Hoppe, Natalie; Dufner, Bianca; Schulte, Lisa; Marchini, Timoteo; Cicko, Sanja; Ayata, Korcan; Zech, Andreas; Wolf, Dennis; Hilgendorf, Ingo; Willecke, Florian; Reinoehl, Jochen; von zur Muehlen, Constantin; Bode, Christoph; Idzko, Marco; Zirlik, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1577-1586</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">OBJECTIVE: A solid body of evidence supports a role of extracellular ATP and its P2 receptors in innate and adaptive immunity.  It promotes inflammation as a danger signal in various chronic inflammatory diseases.  Thus, we hypothesize contribution of extracellular ATP and its receptor P2Y2 in vascular inflammation and atherosclerosis.  APPROACH AND RESULTS-: Extracellular ATP induced leukocyte rolling, adhesion, and migration in vivo as assessed by intravital microscopy and in sterile peritonitis.  To test the role of extracellular ATP in atherosclerosis, ATP or saline as control was injected i.p. 3× a week in low-d. lipoprotein receptor mice consuming high cholesterol diet.  Atherosclerosis significantly increased after 16 wk in ATP-treated mice (n=13; control group, 0.26 mm2; ATP group, 0.33 mm2; P=0.01).  To gain into the role of ATP-receptor P2Y2 in ATP-induced leukocyte recruitment, ATP was administered systemically in P2Y2-deficient or P2Y2-competent mice.  In P2Y2-deficient mice, the ATP-induced leukocyte adhesion was significantly reduced as assessed by intravital microscopy.  P2Y2 expression in atherosclerosis was measured by real-time polymerase chain reaction and immunohistochem. and demonstrates an increased expression mainly caused by influx of P2Y2-expressing macrophages.  To investigate the functional role of P2Y2 in atherogenesis, P2Y2-deficient low-d. lipoprotein receptor mice consumed high cholesterol diet.  After 16 wk, P2Y2-deficient mice showed significantly reduced atherosclerotic lesions with decreased macrophages compared with P2Y2-competent mice (n=11; aortic arch: control group, 0.25 mm; P2Y2-deficient, 0.14 mm2; P=0.04).  Mechanistically, atherosclerotic lesions from P2Y2-deficient mice expressed less vascular cell adhesion mol.-1 and intercellular adhesion mol.-1 RNA.  CONCLUSIONS: We show that extracellular ATP induces vascular inflammation and atherosclerosis via activation of P2Y2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs1FzbdbOyOrVg90H21EOLACvtfcHk0lijDZYIoHpyGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ekt7zJ&md5=e1bdf319afb42f45a8348d91bdf1c08a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1161%2FATVBAHA.115.307397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FATVBAHA.115.307397%26sid%3Dliteratum%253Aachs%26aulast%3DStachon%26aufirst%3DP.%26aulast%3DGeis%26aufirst%3DS.%26aulast%3DPeikert%26aufirst%3DA.%26aulast%3DHeidenreich%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DN.%2BA.%26aulast%3D%25C3%259Cnal%26aufirst%3DF.%26aulast%3DHoppe%26aufirst%3DN.%26aulast%3DDufner%26aufirst%3DB.%26aulast%3DSchulte%26aufirst%3DL.%26aulast%3DMarchini%26aufirst%3DT.%26aulast%3DCicko%26aufirst%3DS.%26aulast%3DAyata%26aufirst%3DK.%26aulast%3DZech%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DD.%26aulast%3DHilgendorf%26aufirst%3DI.%26aulast%3DWillecke%26aufirst%3DF.%26aulast%3DRein%25C3%25B6hl%26aufirst%3DJ.%26aulast%3Dvon%2BZur%2BM%25C3%25BChlen%26aufirst%3DC.%26aulast%3DBode%26aufirst%3DC.%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DZirlik%26aufirst%3DA.%26atitle%3DExtracellular%2520ATP%2520induces%2520vascular%2520inflammation%2520and%2520atherosclerosis%2520via%2520purinergic%2520receptor%2520Y2%2520in%2520mice%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2016%26volume%3D36%26spage%3D1577%26epage%3D1586%26doi%3D10.1161%2FATVBAHA.115.307397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eltzschig, H. K.</span></span> <span> </span><span class="NLM_article-title">Nucleotide signalling during inflammation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nature13085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=24828189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=310-317&author=M.+Idzkoauthor=D.+Ferrariauthor=H.+K.+Eltzschig&title=Nucleotide+signalling+during+inflammation&doi=10.1038%2Fnature13085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide signalling during inflammation</span></div><div class="casAuthors">Idzko, Marco; Ferrari, Davide; Eltzschig, Holger K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7500</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inflammatory conditions are assocd. with the extracellular release of nucleotides, particularly ATP.  In the extracellular compartment, ATP predominantly functions as a signalling mol. through the activation of purinergic P2 receptors.  Metabotropic P2Y receptors are G-protein-coupled, whereas ionotropic P2X receptors are ATP-gated ion channels.  Here we discuss how signalling events through P2 receptors alter the outcomes of inflammatory or infectious diseases.  Recent studies implicate a role for P2X/P2Y signalling in mounting appropriate inflammatory responses crit. for host defense against invading pathogens or tumors.  Conversely, P2X/P2Y signalling can promote chronic inflammation during ischemia and reperfusion injury, inflammatory bowel disease or acute and chronic diseases of the lungs.  Although nucleotide signalling has been used clin. in patients before, research indicates an expanding field of opportunities for specifically targeting individual P2 receptors for the treatment of inflammatory or infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQJVgswMEhUrVg90H21EOLACvtfcHk0lgm1StQqfzapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyqtLc%253D&md5=676480f2bfbe98592d603f9064d011b7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13085%26sid%3Dliteratum%253Aachs%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DFerrari%26aufirst%3DD.%26aulast%3DEltzschig%26aufirst%3DH.%2BK.%26atitle%3DNucleotide%2520signalling%2520during%2520inflammation%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D310%26epage%3D317%26doi%3D10.1038%2Fnature13085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Structure of the human P2Y<sub>12</sub> receptor in complex with an antithrombotic drug</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1038/nature13083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=24670650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlyisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=115-118&author=K.+Zhangauthor=J.+Zhangauthor=Z.+G.+Gaoauthor=D.+Zhangauthor=L.+Zhuauthor=G.+W.+Hanauthor=S.+M.+Mossauthor=S.+Paolettaauthor=E.+Kiselevauthor=W.+Luauthor=G.+Fenaltiauthor=W.+Zhangauthor=C.+E.+M%C3%BCllerauthor=H.+Yangauthor=V.+Cherezovauthor=V.+Katritchauthor=G.+W.+Hanauthor=K.+A.+Jacobsonauthor=R.+C.+Stevensauthor=B.+Wuauthor=Q.+Zhao&title=Structure+of+the+human+P2Y12+receptor+in+complex+with+an+antithrombotic+drug&doi=10.1038%2Fnature13083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human P2Y12 receptor in complex with an antithrombotic drug</span></div><div class="casAuthors">Zhang, Kaihua; Zhang, Jin; Gao, Zhan-Guo; Zhang, Dandan; Zhu, Lan; Han, Gye Won; Moss, Steven M.; Paoletta, Silvia; Kiselev, Evgeny; Lu, Weizhen; Fenalti, Gustavo; Zhang, Wenru; Mueller, Christa E.; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Katritch, Vsevolod; Jacobson, Kenneth A.; Stevens, Raymond C.; Wu, Beili; Zhao, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7498</span>),
    <span class="NLM_cas:pages">115-118</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P2Y receptors (P2YRs), a family of purinergic G-protein-coupled receptors (GPCRs), are activated by extracellular nucleotides.  There are a total of eight distinct functional P2YRs expressed in human, which are subdivided into P2Y1-like receptors and P2Y12-like receptors.  Their ligands are generally charged mols. with relatively low bioavailability and stability in vivo, which limits the authors' understanding of this receptor family.  P2Y12R regulates platelet activation and thrombus formation, and several antithrombotic drugs targeting P2Y12R-including the prodrugs clopidogrel (Plavix) and prasugrel (Effient) that are metabolized and bind covalently, and the nucleoside analog ticagrelor (Brilinta) that acts directly on the receptor-have been approved for the prevention of stroke and myocardial infarction.  However, limitations of these drugs (for example, a very long half-life of clopidogrel action and a characteristic adverse effect profile of ticagrelor) suggest that there is an unfulfilled medical need for developing a new generation of P2Y12R inhibitors.  Here the authors report the 2.6 Å resoln. crystal structure of human P2Y12R in complex with a non-nucleotide reversible antagonist, AZD1283.  The structure reveals a distinct straight conformation of helix V, which sets P2Y12R apart from all other known class A GPCR structures.  With AZD1283 bound, the highly conserved disulfide bridge in GPCRs between helix III and extracellular loop 2 is not obsd. and appears to be dynamic.  Along with the details of the AZD1283-binding site, anal. of the extracellular interface reveals an adjacent ligand-binding region and suggests that both pockets could be required for dinucleotide binding.  The structure provides essential insights for the development of improved P2Y12R ligands and allosteric modulators as drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOY06VN4D76bVg90H21EOLACvtfcHk0lgm1StQqfzapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlyisrg%253D&md5=539bb91325c547737bc98baf59f3ff26</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature13083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13083%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DStructure%2520of%2520the%2520human%2520P2Y12%2520receptor%2520in%2520complex%2520with%2520an%2520antithrombotic%2520drug%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D115%26epage%3D118%26doi%3D10.1038%2Fnature13083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Agonist-bound structure of the human P2Y<sub>12</sub>R receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/nature13288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13288" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=119-122&author=K.+Zhangauthor=J.+Zhangauthor=Z.+G.+Gaoauthor=S.+Paolettaauthor=D.+Zhangauthor=G.+W.+Hanauthor=T.+Liauthor=L.+Maauthor=W.+Zhangauthor=C.+E.+M%C3%BCllerauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=V.+Katritchauthor=K.+A.+Jacobsonauthor=R.+C.+Stevensauthor=B.+Wuauthor=Q.+Zhao&title=Agonist-bound+structure+of+the+human+P2Y12R+receptor&doi=10.1038%2Fnature13288"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature13288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13288%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DAgonist-bound%2520structure%2520of%2520the%2520human%2520P2Y12R%2520receptor%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D119%26epage%3D122%26doi%3D10.1038%2Fnature13288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzer, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Exploring a 2-naphthoic acid template for the structure-based design of P2Y<sub>14</sub> receptor antagonist molecular probes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2833</span>– <span class="NLM_lpage">2842</span>, <span class="refDoi"> DOI: 10.1021/cb500614p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500614p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslajs73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2833-2842&author=E.+Kiselevauthor=M.+Barrettauthor=V.+Katritchauthor=S.+Paolettaauthor=C.+D.+Weitzerauthor=E.+Hammesauthor=A.+L.+Yinauthor=Q.+Zhaoauthor=R.+C.+Stevensauthor=T.+K.+Hardenauthor=K.+A.+Jacobson&title=Exploring+a+2-naphthoic+acid+template+for+the+structure-based+design+of+P2Y14+receptor+antagonist+molecular+probes&doi=10.1021%2Fcb500614p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring a 2-Naphthoic Acid Template for the Structure-Based Design of P2Y14 Receptor Antagonist Molecular Probes</span></div><div class="casAuthors">Kiselev, Evgeny; Barrett, Matthew O.; Katritch, Vsevolod; Paoletta, Silvia; Weitzer, Clarissa D.; Brown, Kyle A.; Hammes, Eva; Yin, Andrew L.; Zhao, Qiang; Stevens, Raymond C.; Harden, T. Kendall; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2833-2842</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The P2Y14 receptor (P2Y14R), one of eight P2Y G protein-coupled receptors (GPCR), is involved in inflammatory, endocrine, and hypoxic processes and is an attractive pharmaceutical target.  The goal of this research is to develop high-affinity P2Y14R fluorescent probes based on the potent and highly selective antagonist 4-(4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl)-2-naphthoic acid (6, PPTN).  A model of hP2Y14R based on recent hP2Y12R X-ray structures together with simulated antagonist docking suggested that the piperidine ring is suitable for fluorophore conjugation while preserving affinity.  Chain-elongated alkynyl or amino derivs. of 6 for click or amide coupling were synthesized, and their antagonist activities were measured in hP2Y14R-expressing CHO cells.  Moreover, a new Alexa Fluor 488 (AF488) contg. deriv. 30 (MRS4174, Ki = 80 pM) exhibited exceptionally high affinity, as compared to 13 nM for the alkyne precursor 22.  A flow cytometry assay employing 30 as a fluorescent probe was used to quantify specific binding to P2Y14R.  Known P2Y receptor ligands inhibited binding of 30 with properties consistent with their previously established receptor selectivities and affinities.  These results illustrate that potency in this series of 2-naphthoic acid derivs. can be preserved by chain functionalization, leading to highly potent fluorescent mol. probes for P2Y14R.  Such conjugates will be useful tools in expanding the SAR of this receptor, which still lacks chem. diversity in its collective ligands.  This approach demonstrates the predictive power of GPCR homol. modeling and the relevance of newly detd. X-ray structures to GPCR medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwJ36T_9HjUrVg90H21EOLACvtfcHk0lh0md8mWZucjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslajs73K&md5=9dca289cc6fcf9c1418b49f811e691c0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fcb500614p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500614p%26sid%3Dliteratum%253Aachs%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DM.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DWeitzer%26aufirst%3DC.%2BD.%26aulast%3DHammes%26aufirst%3DE.%26aulast%3DYin%26aufirst%3DA.%2BL.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DExploring%2520a%25202-naphthoic%2520acid%2520template%2520for%2520the%2520structure-based%2520design%2520of%2520P2Y14%2520receptor%2520antagonist%2520molecular%2520probes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2833%26epage%3D2842%26doi%3D10.1021%2Fcb500614p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of the human P2Y<sub>14</sub> receptor: A template for structure-based design of selective agonist ligands</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4056</span>– <span class="NLM_lpage">4064</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmc.2015.03.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4056-4064&author=K.+Trujilloauthor=S.+Paolettaauthor=E.+Kiselevauthor=K.+A.+Jacobson&title=Molecular+modeling+of+the+human+P2Y14+receptor%3A+A+template+for+structure-based+design+of+selective+agonist+ligands&doi=10.1016%2Fj.bmc.2015.03.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DK.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DMolecular%2520modeling%2520of%2520the%2520human%2520P2Y14%2520receptor%253A%2520A%2520template%2520for%2520structure-based%2520design%2520of%2520selective%2520agonist%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4056%26epage%3D4064%26doi%3D10.1016%2Fj.bmc.2015.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschênes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareau, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hénault, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyjazie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kargman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavallée, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levesque, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulrooney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thérien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tranmer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, W. C.</span></span> <span> </span><span class="NLM_article-title">The identification of 4,7-disubstituted naphthoic acid derivatives as UDP-competitive antagonists of P2Y<sub>14</sub></span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2836</span>– <span class="NLM_lpage">2839</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.03.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmcl.2011.03.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=21507640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVCmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=2836-2839&author=J.+Y.+Gauthierauthor=M.+Belleyauthor=D.+Desch%C3%AAnesauthor=J.-F.+Fournierauthor=S.+Gagn%C3%A9author=Y.+Gareauauthor=M.+Hamelauthor=M.+H%C3%A9naultauthor=H.+Hyjazieauthor=S.+Kargmanauthor=G.+Lavall%C3%A9eauthor=J.+F.+Levesqueauthor=L.+Liauthor=Y.+Mamaneauthor=J.+Manciniauthor=N.+Morinauthor=E.+Mulrooneyauthor=J.+Robichaudauthor=M.+Th%C3%A9rienauthor=G.+Tranmerauthor=Z.+Wangauthor=J.+Wuauthor=W.+C.+Black&title=The+identification+of+4%2C7-disubstituted+naphthoic+acid+derivatives+as+UDP-competitive+antagonists+of+P2Y14&doi=10.1016%2Fj.bmcl.2011.03.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The identification of 4,7-disubstituted naphthoic acid derivatives as UDP-competitive antagonists of P2Y14</span></div><div class="casAuthors">Gauthier, Jacques Yves; Belley, Michel; Deschenes, Denis; Fournier, Jean-Francois; Gagne, Sebastien; Gareau, Yves; Hamel, Martine; Henault, Martin; Hyjazie, Huda; Kargman, Stacia; Lavallee, Genevieve; Levesque, Jean-Francois; Li, Lianhai; Mamane, Yael; Mancini, Joseph; Morin, Nicolas; Mulrooney, Erin; Robichaud, Joel; Therien, Michel; Tranmer, Geoffrey; Wang, Zhaoyin; Wu, Jin; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2836-2839</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A weak, UDP-competitive antagonist of the pyrimidinergic receptor P2RY14 with a naphthoic acid core was identified through high-throughput screening.  Optimization provided compds. with improved potency but poor pharmacokinetics.  Acylglucuronidation was detd. to be the major route of metab.  Increasing the electron-withdrawing nature of the substituents markedly reduced glucuronidation and improved the pharmacokinetic profile.  Addnl. optimization led to the identification of compd. 38 which is an 8 nM UDP-competitive antagonist of P2Y14 with a good pharmacokinetic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjL3gtO4Wk4LVg90H21EOLACvtfcHk0lh0md8mWZucjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVCmtrg%253D&md5=7a5b0a1ae77102f81e3ae8eb4afadccf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.03.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.03.081%26sid%3Dliteratum%253Aachs%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DBelley%26aufirst%3DM.%26aulast%3DDesch%25C3%25AAnes%26aufirst%3DD.%26aulast%3DFournier%26aufirst%3DJ.-F.%26aulast%3DGagn%25C3%25A9%26aufirst%3DS.%26aulast%3DGareau%26aufirst%3DY.%26aulast%3DHamel%26aufirst%3DM.%26aulast%3DH%25C3%25A9nault%26aufirst%3DM.%26aulast%3DHyjazie%26aufirst%3DH.%26aulast%3DKargman%26aufirst%3DS.%26aulast%3DLavall%25C3%25A9e%26aufirst%3DG.%26aulast%3DLevesque%26aufirst%3DJ.%2BF.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMamane%26aufirst%3DY.%26aulast%3DMancini%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DN.%26aulast%3DMulrooney%26aufirst%3DE.%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DTh%25C3%25A9rien%26aufirst%3DM.%26aulast%3DTranmer%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DThe%2520identification%2520of%25204%252C7-disubstituted%2520naphthoic%2520acid%2520derivatives%2520as%2520UDP-competitive%2520antagonists%2520of%2520P2Y14%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D2836%26epage%3D2839%26doi%3D10.1016%2Fj.bmcl.2011.03.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciancetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martiriggiano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mtchedlidze, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birdwell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 3-(4-aryl-1H-1,2,3-triazol-1-yl)-biphenyl derivatives as P2Y<sub>14</sub> receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6149</span>– <span class="NLM_lpage">6168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVWkurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6149-6168&author=A.+Junkerauthor=R.+Balasubramanianauthor=A.+Ciancettaauthor=E.+Uliassiauthor=E.+Kiselevauthor=C.+Martiriggianoauthor=K.+Trujilloauthor=G.+Mtchedlidzeauthor=L.+Birdwellauthor=K.+A.+Brownauthor=T.+K.+Hardenauthor=K.+A.+Jacobson&title=Structure-based+design+of+3-%284-aryl-1H-1%2C2%2C3-triazol-1-yl%29-biphenyl+derivatives+as+P2Y14+receptor+antagonists&doi=10.1021%2Facs.jmedchem.6b00044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 3-(4-Aryl-1H-1,2,3-triazol-1-yl)-Biphenyl Derivatives as P2Y14 Receptor Antagonists</span></div><div class="casAuthors">Junker, Anna; Balasubramanian, Ramachandran; Ciancetta, Antonella; Uliassi, Elisa; Kiselev, Evgeny; Martiriggiano, Chiara; Trujillo, Kevin; Mtchedlidze, Giorgi; Birdwell, Leah; Brown, Kyle A.; Harden, T. Kendall; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6149-6168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">UDP and UDP-glucose activate the P2Y14 receptor (P2Y14R) to modulate processes related to inflammation, diabetes, and asthma.  A computational pipeline suggested alternatives to naphthalene of a previously reported P2Y14R antagonist (3, PPTN) using docking and mol. dynamics simulations on a hP2Y14R homol. model based on P2Y12R structures.  By reevaluating the binding of 3 to P2Y14R computationally, two alternatives, i.e., alkynyl and triazolyl derivs., were identified.  Improved synthesis of fluorescent antagonist 4 enabled affinity quantification (IC50s, nM) using flow cytometry of P2Y14R-expressing CHO cells. p-F3C-phenyl-triazole 65 (32) was more potent than a corresponding alkyne 11.  Thus, addnl. triazolyl derivs. were prepd., as guided by docking simulations, with nonpolar aryl substituents favored.  Although triazoles were less potent than 3 (6), simpler synthesis facilitated further structural optimization.  Addnl., relative P2Y14R affinities agreed with predicted binding of alkynyl and triazole analogs.  These triazoles, designed through a structure-based approach, can be assessed in disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5CKuzhmXmxrVg90H21EOLACvtfcHk0lg6RbldsOuNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVWkurzE&md5=02287b12815a85d188e5f224735f302a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00044%26sid%3Dliteratum%253Aachs%26aulast%3DJunker%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DR.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DMartiriggiano%26aufirst%3DC.%26aulast%3DTrujillo%26aufirst%3DK.%26aulast%3DMtchedlidze%26aufirst%3DG.%26aulast%3DBirdwell%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DK.%2BA.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520design%2520of%25203-%25284-aryl-1H-1%252C2%252C3-triazol-1-yl%2529-biphenyl%2520derivatives%2520as%2520P2Y14%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6149%26epage%3D6168%26doi%3D10.1021%2Facs.jmedchem.6b00044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lg6RbldsOuNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">Integrated methods for the construction of three dimensional models and computational probing of structure function relations in G protein-coupled receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/S1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=Integrated+methods+for+the+construction+of+three+dimensional+models+and+computational+probing+of+structure+function+relations+in+G+protein-coupled+receptors&doi=10.1016%2FS1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0lg6RbldsOuNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DIntegrated%2520methods%2520for%2520the%2520construction%2520of%2520three%2520dimensional%2520models%2520and%2520computational%2520probing%2520of%2520structure%2520function%2520relations%2520in%2520G%2520protein-coupled%2520receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2FS1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span> <span> </span><span class="NLM_article-title">Acidic and basic drugs in medicinal chemistry: A perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9701</span>– <span class="NLM_lpage">9717</span>, <span class="refDoi"> DOI: 10.1021/jm501000a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501000a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOnt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9701-9717&author=P.+S.+Charifsonauthor=W.+P.+Walters&title=Acidic+and+basic+drugs+in+medicinal+chemistry%3A+A+perspective&doi=10.1021%2Fjm501000a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic and Basic Drugs in Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Charifson, Paul S.; Walters, W. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9701-9717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The acid/base properties of a mol. are among the most fundamental for drug action.  However, they are often overlooked in a prospective design manner unless it has been established that a certain ionization state (e.g., quaternary base or presence of a carboxylic acid) appears to be required for activity.  In medicinal chem. optimization programs it is relatively common to attenuate basicity to circumvent undesired effects such as lack of biol. selectivity or safety risks such as hERG or phospholipidosis.  However, teams may not prospectively explore a range of carefully chosen compd. pKa values as part of an overall chem. strategy or design hypothesis.  This review summarizes the potential advantages and disadvantages of both acidic and basic drugs and provides some new analyses based on recently available public data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDorb0LfJ8PrVg90H21EOLACvtfcHk0lg6RbldsOuNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOnt7jN&md5=51a3cb91be42ddd1a784afe737c64c60</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm501000a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501000a%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26atitle%3DAcidic%2520and%2520basic%2520drugs%2520in%2520medicinal%2520chemistry%253A%2520A%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9701%26epage%3D9717%26doi%3D10.1021%2Fjm501000a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lassalas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasfargeas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayendran, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span> <span> </span><span class="NLM_article-title">Structure property relationships of carboxylic acid isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3203</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3183-3203&author=P.+Lassalasauthor=B.+Gayauthor=C.+Lasfargeasauthor=M.+J.+Jamesauthor=V.+Tranauthor=K.+G.+Vijayendranauthor=K.+R.+Brundenauthor=M.+C.+Kozlowskiauthor=C.+J.+Thomasauthor=A.+B.+Smithauthor=D.+M.+Hurynauthor=C.+Ballatore&title=Structure+property+relationships+of+carboxylic+acid+isosteres&doi=10.1021%2Facs.jmedchem.5b01963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Property Relationships of Carboxylic Acid Isosteres</span></div><div class="casAuthors">Lassalas, Pierrik; Gay, Bryant; Lasfargeas, Caroline; James, Michael J.; Tran, Van; Vijayendran, Krishna G.; Brunden, Kurt R.; Kozlowski, Marisa C.; Thomas, Craig J.; Smith, Amos B.; Huryn, Donna M.; Ballatore, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3183-3203</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The replacement of a carboxylic acid with a surrogate structure, or (bio)-isostere, is a classical strategy in medicinal chem.  The general underlying principle is that by maintaining the features of the carboxylic acid crit. for biol. activity, but appropriately modifying the physicochem. properties, improved analogs may result.  In this context, a systematic assessment of the physicochem. properties of carboxylic acid isosteres would be desirable to enable more informed decisions of potential replacements to be used for analog design.  Herein we report the structure-property relationships (SPR) of 35 phenylpropionic acid derivs., in which the carboxylic acid moiety is replaced with a series of known isosteres.  The data set generated provides an assessment of the relative impact on the physicochem. properties that these replacements may have compared to the carboxylic acid analog.  As such, this study presents a framework for how to rationally apply isosteric replacements of the carboxylic acid functional group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIF98i2ITNrLVg90H21EOLACvtfcHk0lgs-tYeBvcwyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D&md5=34490fc72fdb321d8c9751bef1bf79c5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01963%26sid%3Dliteratum%253Aachs%26aulast%3DLassalas%26aufirst%3DP.%26aulast%3DGay%26aufirst%3DB.%26aulast%3DLasfargeas%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DVijayendran%26aufirst%3DK.%2BG.%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DKozlowski%26aufirst%3DM.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26aulast%3DBallatore%26aufirst%3DC.%26atitle%3DStructure%2520property%2520relationships%2520of%2520carboxylic%2520acid%2520isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3183%26epage%3D3203%26doi%3D10.1021%2Facs.jmedchem.5b01963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgs-tYeBvcwyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span> <span> </span><span class="NLM_article-title">Drug-like antagonists of P2Y receptors—from lead identification to drug development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9981</span>– <span class="NLM_lpage">10005</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9981-10005&author=S.+Conroyauthor=N.+Kindonauthor=B.+Kellamauthor=M.+J.+Stocks&title=Drug-like+antagonists+of+P2Y+receptors%E2%80%94from+lead+identification+to+drug+development&doi=10.1021%2Facs.jmedchem.5b01972"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01972%26sid%3Dliteratum%253Aachs%26aulast%3DConroy%26aufirst%3DS.%26aulast%3DKindon%26aufirst%3DN.%26aulast%3DKellam%26aufirst%3DB.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26atitle%3DDrug-like%2520antagonists%2520of%2520P2Y%2520receptors%25E2%2580%2594from%2520lead%2520identification%2520to%2520drug%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9981%26epage%3D10005%26doi%3D10.1021%2Facs.jmedchem.5b01972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hénault, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kargman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levesque, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulrooney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, W. C.</span></span> <span> </span><span class="NLM_article-title">Applying the pro-drug approach to afford highly bioavailable antagonists of P2Y<sub>14</sub></span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4366</span>– <span class="NLM_lpage">4368</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.12.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmcl.2010.12.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=21689930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFWrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4366-4368&author=J.+Robichaudauthor=J.+F.+Fournierauthor=S.+Gagn%C3%A9author=J.+Y.+Gauthierauthor=M.+Hamelauthor=Y.+Hanauthor=M.+H%C3%A9naultauthor=S.+Kargmanauthor=J.-F.+Levesqueauthor=Y.+Mamaneauthor=J.+Manciniauthor=N.+Morinauthor=E.+Mulrooneyauthor=J.+Wuauthor=W.+C.+Black&title=Applying+the+pro-drug+approach+to+afford+highly+bioavailable+antagonists+of+P2Y14&doi=10.1016%2Fj.bmcl.2010.12.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Applying the pro-drug approach to afford highly bioavailable antagonists of P2Y14</span></div><div class="casAuthors">Robichaud, Joel; Fournier, Jean-Francois; Gagne, Sebastien; Gauthier, Jacques Yves; Hamel, Martine; Han, Yongxin; Henault, Martin; Kargman, Stacia; Levesque, Jean-Francois; Mamane, Yael; Mancini, Joseph; Morin, Nicolas; Mulrooney, Erin; Wu, Jin; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4366-4368</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Our series of competitive antagonists against the G-protein coupled receptor P2Y14 were found to be highly shifted in the presence of serum (>99% protein bound).  A binding assay using 2% human serum albumin (HSA) was developed to guide further SAR studies and led to the identification of the zwitterion 2 (I), which is substantially less shifted (18-fold) than our previous lead compd. 1 (323-fold).  However, as the bioavailability of 2 was low, a library of ester pro-drugs was prepd. (7a-7j) and assessed in vitro.  The most interesting candidates were then profiled in vivo and led to the identification of the pro-drug 7j, which possesses a substantially improved pharmacokinetic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwKD0Yxb4pOLVg90H21EOLACvtfcHk0lg8lr6pwk-uKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFWrtLw%253D&md5=d993d60779b583f0ca7ff466387a8dbc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.12.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.12.113%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DFournier%26aufirst%3DJ.%2BF.%26aulast%3DGagn%25C3%25A9%26aufirst%3DS.%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DHamel%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DH%25C3%25A9nault%26aufirst%3DM.%26aulast%3DKargman%26aufirst%3DS.%26aulast%3DLevesque%26aufirst%3DJ.-F.%26aulast%3DMamane%26aufirst%3DY.%26aulast%3DMancini%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DN.%26aulast%3DMulrooney%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DApplying%2520the%2520pro-drug%2520approach%2520to%2520afford%2520highly%2520bioavailable%2520antagonists%2520of%2520P2Y14%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4366%26epage%3D4368%26doi%3D10.1016%2Fj.bmcl.2010.12.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0lg8lr6pwk-uKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span> <i>LigPrep</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span> <i>Glide</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noé, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">HTMD: High-throughput molecular dynamics for molecular discovery</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1845</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b00049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12hu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1845-1852&author=S.+Doerrauthor=M.+J.+Harveyauthor=F.+No%C3%A9author=G.+De+Fabritiis&title=HTMD%3A+High-throughput+molecular+dynamics+for+molecular+discovery&doi=10.1021%2Facs.jctc.6b00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">HTMD: High-Throughput Molecular Dynamics for Molecular Discovery</span></div><div class="casAuthors">Doerr, S.; Harvey, M. J.; Noe, Frank; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1845-1852</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in mol. simulations have allowed scientists to investigate slower biol. processes than ever before.  Together with these advances came an explosion of data that has transformed a traditionally computing-bound into a data-bound problem.  Here, we present HTMD, a programmable, extensible platform written in Python that aims to solve the data generation and anal. problem as well as increase reproducibility by providing a complete workspace for simulation-based discovery.  So far, HTMD includes system building for CHARMM and AMBER force fields, projection methods, clustering, mol. simulation prodn., adaptive sampling, an Amazon cloud interface, Markov state models, and visualization.  As a result, a single, short HTMD script can lead from a PDB structure to useful quantities such as relaxation time scales, equil. populations, metastable conformations, and kinetic rates.  In this paper, we focus on the adaptive sampling and Markov state modeling features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEgXtPB9K5VLVg90H21EOLACvtfcHk0lg8lr6pwk-uKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12hu7o%253D&md5=c7ce0d9a709642ad5e28e202655a9a9d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00049%26sid%3Dliteratum%253Aachs%26aulast%3DDoerr%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DNo%25C3%25A9%26aufirst%3DF.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DHTMD%253A%2520High-throughput%2520molecular%2520dynamics%2520for%2520molecular%2520discovery%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D1845%26epage%3D1852%26doi%3D10.1021%2Facs.jctc.6b00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">VMD - Visual molecular dynamics</span>. <i>J. Mol. Graphics</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD+-+Visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lg8lr6pwk-uKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%2520-%2520Visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lj_u1-Gmr583w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giupponi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">ACEMD: Accelerated molecular dynamics simulations in the microseconds timescale</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1632</span>– <span class="NLM_lpage">1639</span>, <span class="refDoi"> DOI: 10.1021/ct9000685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9000685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=1632-1639&author=M.+Harveyauthor=G.+Giupponiauthor=G.+De+Fabritiis&title=ACEMD%3A+Accelerated+molecular+dynamics+simulations+in+the+microseconds+timescale&doi=10.1021%2Fct9000685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ACEMD: Accelerating Biomolecular Dynamics in the Microsecond Time Scale</span></div><div class="casAuthors">Harvey, M. J.; Giupponi, G.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1632-1639</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The high arithmetic performance and intrinsic parallelism of recent graphical processing units (GPUs) can offer a technol. edge for mol. dynamics simulations.  ACEMD is a prodn.-class biomol. dynamics (MD) engine supporting CHARMM and AMBER force fields.  Designed specifically for GPUs it is able to achieve supercomputing scale performance of 40 ns/day for all-atom protein systems with over 23,000 atoms.  The authors provide a validation and performance evaluation of the code and run a microsecond-long trajectory for an all-atom mol. system in explicit TIP3P water on a single workstation computer equipped with just 3 GPUs.  The authors believe that microsecond time scale mol. dynamics on cost-effective hardware will have important methodol. and scientific implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryhvUDSt2mxbVg90H21EOLACvtfcHk0lj_u1-Gmr583w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rsLk%253D&md5=99a7226c62aa210bada97ee61a0c254f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fct9000685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9000685%26sid%3Dliteratum%253Aachs%26aulast%3DHarvey%26aufirst%3DM.%26aulast%3DGiupponi%26aufirst%3DG.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DACEMD%253A%2520Accelerated%2520molecular%2520dynamics%2520simulations%2520in%2520the%2520microseconds%2520timescale%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D1632%26epage%3D1639%26doi%3D10.1021%2Fct9000685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Best, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Optimization of the additive CHARMM all-atom protein force field targeting improved sampling of the backbone phi, psi and side-chain chi1 and chi2 dihedral angles</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3273</span>, <span class="refDoi"> DOI: 10.1021/ct300400x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300400x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3257-3273&author=R.+B.+Bestauthor=X.+Zhuauthor=J.+Shimauthor=P.+E.+M.+Lopesauthor=J.+Mittalauthor=M.+Feigauthor=A.+D.+MacKerell&title=Optimization+of+the+additive+CHARMM+all-atom+protein+force+field+targeting+improved+sampling+of+the+backbone+phi%2C+psi+and+side-chain+chi1+and+chi2+dihedral+angles&doi=10.1021%2Fct300400x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone .vphi., ψ and Side-Chain χ1 and χ2 Dihedral Angles</span></div><div class="casAuthors">Best, Robert B.; Zhu, Xiao; Shim, Jihyun; Lopes, Pedro E. M.; Mittal, Jeetain; Feig, Michael; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3257-3273</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the quality of the current CHARMM22/CMAP additive force field for proteins has been demonstrated in a large no. of applications, limitations in the model with respect to the equil. between the sampling of helical and extended conformations in folding simulations have been noted.  To overcome this, as well as make other improvements in the model, we present a combination of refinements that should result in enhanced accuracy in simulations of proteins.  The common (non-Gly, -Pro) backbone CMAP potential has been refined against exptl. soln. NMR data for weakly structured peptides, resulting in a rebalancing of the energies of the α-helix and extended regions of the Ramachandran map, correcting the α-helical bias of CHARMM22/CMAP.  The Gly and Pro CMAPs have been refitted to more accurate quantum-mech. energy surfaces.  Side-chain torsion parameters have been optimized by fitting to backbone-dependent quantum-mech. energy surfaces, followed by addnl. empirical optimization targeting NMR scalar couplings for unfolded proteins.  A comprehensive validation of the revised force field was then performed against a collection of exptl. data: (i) comparison of simulations of eight proteins in their crystal environments with crystal structures; (ii) comparison with backbone scalar couplings for weakly structured peptides; (iii) comparison with NMR residual dipolar couplings and scalar couplings for both backbone and side-chains in folded proteins; (iv) equil. folding of mini-proteins.  The results indicate that the revised CHARMM 36 parameters represent an improved model for modeling and simulation studies of proteins, including studies of protein folding, assembly, and functionally relevant conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnwGJMAMFvLVg90H21EOLACvtfcHk0lj_u1-Gmr583w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP&md5=9a48a0c5770fb1e887c3bb34d45b1354</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fct300400x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300400x%26sid%3Dliteratum%253Aachs%26aulast%3DBest%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DP.%2BE.%2BM.%26aulast%3DMittal%26aufirst%3DJ.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DOptimization%2520of%2520the%2520additive%2520CHARMM%2520all-atom%2520protein%2520force%2520field%2520targeting%2520improved%2520sampling%2520of%2520the%2520backbone%2520phi%252C%2520psi%2520and%2520side-chain%2520chi1%2520and%2520chi2%2520dihedral%2520angles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3257%26epage%3D3273%26doi%3D10.1021%2Fct300400x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klauda, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freites, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondragon-Ramirez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Update of the CHARMM all-atom additive force field for lipids: validation on six lipid types</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7830</span>– <span class="NLM_lpage">7843</span>, <span class="refDoi"> DOI: 10.1021/jp101759q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp101759q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=7830-7843&author=J.+B.+Klaudaauthor=R.+M.+Venableauthor=J.+A.+Freitesauthor=J.+W.+O%E2%80%99Connorauthor=D.+J.+Tobiasauthor=C.+Mondragon-Ramirezauthor=I.+Vorobyovauthor=A.+D.+MacKerellauthor=R.+W.+Pastor&title=Update+of+the+CHARMM+all-atom+additive+force+field+for+lipids%3A+validation+on+six+lipid+types&doi=10.1021%2Fjp101759q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span></div><div class="casAuthors">Klauda, Jeffery B.; Venable, Richard M.; Freites, J. Alfredo; O'Connor, Joseph W.; Tobias, Douglas J.; Mondragon-Ramirez, Carlos; Vorobyov, Igor; MacKerell, Alexander D., Jr.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7830-7843</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A significant modification to the additive all-atom CHARMM lipid force field (FF) is developed and applied to phospholipid bilayers with both choline and ethanolamine contg. head groups and with both satd. and unsatd. aliph. chains.  Motivated by the current CHARMM lipid FF (C27 and C27r) systematically yielding values of the surface area per lipid that are smaller than exptl. ests. and gel-like structures of bilayers well above the gel transition temp., selected torsional, Lennard-Jones and partial at. charge parameters were modified by targeting both quantum mech. (QM) and exptl. data.  QM calcns. ranging from high-level ab initio calcns. on small mols. to semiempirical QM studies on a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer in combination with exptl. thermodn. data were used as target data for parameter optimization.  These changes were tested with simulations of pure bilayers at high hydration of the following six lipids: DPPC, 1,2-dimyristoyl-sn-phosphatidylcholine (DMPC), 1,2-dilauroyl-sn-phosphatidylcholine (DLPC), 1-palmitoyl-2-oleoyl-sn-phosphatidylcholine (POPC), 1,2-dioleoyl-sn-phosphatidylcholine (DOPC), and 1-palmitoyl-2-oleoyl-sn-phosphatidylethanolamine (POPE); simulations of a low hydration DOPC bilayer were also performed.  Agreement with exptl. surface area is on av. within 2%, and the d. profiles agree well with neutron and x-ray diffraction expts.  NMR deuterium order parameters (SCD) are well predicted with the new FF, including proper splitting of the SCD for the aliph. carbon adjacent to the carbonyl for DPPC, POPE, and POPC bilayers.  The area compressibility modulus and frequency dependence of 13C NMR relaxation rates of DPPC and the water distribution of low hydration DOPC bilayers also agree well with expt.  Accordingly, the presented lipid FF, referred to as C36, allows for mol. dynamics simulations to be run in the tensionless ensemble (NPT), and is anticipated to be of utility for simulations of pure lipid systems as well as heterogeneous systems including membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTVhlLR7mEbVg90H21EOLACvtfcHk0lgLH3tjupdXZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D&md5=04c4d3aa1a59740190994d7df80cbb71</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjp101759q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp101759q%26sid%3Dliteratum%253Aachs%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DFreites%26aufirst%3DJ.%2BA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BW.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DMondragon-Ramirez%26aufirst%3DC.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DUpdate%2520of%2520the%2520CHARMM%2520all-atom%2520additive%2520force%2520field%2520for%2520lipids%253A%2520validation%2520on%2520six%2520lipid%2520types%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D7830%26epage%3D7843%26doi%3D10.1021%2Fjp101759q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) I: bond perception and atom typing</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3144</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0lgLH3tjupdXZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3155</span>– <span class="NLM_lpage">3168</span>, <span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0lgLH3tjupdXZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kräutler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hünenberger, P.
H.</span></span> <span> </span><span class="NLM_article-title">A Fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1002/1096-987X(20010415)22:5<501::AID-JCC1021>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisVSntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=501-508&author=V.+Kr%C3%A4utlerauthor=W.+F.+Van+Gunsterenauthor=P.%0AH.+H%C3%BCnenberger&title=A+Fast+SHAKE+algorithm+to+solve+distance+constraint+equations+for+small+molecules+in+molecular+dynamics+simulations&doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span></div><div class="casAuthors">Krautler, Vincent; Van Gunsteren, Wilfred F.; Hunenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-508</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A common method for the application of distance constraints in mol. simulations employing Cartesian coordinates is the SHAKE procedure for detg. the Lagrange multipliers regarding the constraints.  This method relies on the linearization and decoupling of the equations governing the at. coordinate resetting corresponding to each constraint in a mol., and is thus iterative.  In the present study, we consider an alternative method, M-SHAKE, which solves the coupled equations simultaneously by matrix inversion.  The performances of the two methods are compared in simulations of the pure solvents water, DMSO, and chloroform.  It is concluded that M-SHAKE is significantly faster than SHAKE when either (1) the mols. contain few distance constraints (solvent), or (2) when a high level of accuracy is required in the application of the constraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO7vyqEu_pjrVg90H21EOLACvtfcHk0ljcneZwgbByBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisVSntbY%253D&md5=20e32455e867f10fb61713f86147d96b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1096-987X%252820010415%252922%253A5%253C501%253A%253AAID-JCC1021%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4utler%26aufirst%3DV.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DA%2520Fast%2520SHAKE%2520algorithm%2520to%2520solve%2520distance%2520constraint%2520equations%2520for%2520small%2520molecules%2520in%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D501%26epage%3D508%26doi%3D10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G. A.</span></span> <span> </span><span class="NLM_article-title">Smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+A.+Pedersen&title=Smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0ljcneZwgbByBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%2BA.%26atitle%3DSmooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skeel, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kale, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Scalable molecular dynamics with NAMD</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/jcc.20289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2Fjcc.20289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=16222654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1781-1802&author=J.+C.+Phillipsauthor=R.+Braunauthor=W.+Wangauthor=J.+Gumbartauthor=E.+Tajkhorshidauthor=E.+Villaauthor=C.+Chipotauthor=R.+D.+Skeelauthor=L.+Kaleauthor=K.+Schulten&title=Scalable+molecular+dynamics+with+NAMD&doi=10.1002%2Fjcc.20289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable molecular dynamics with NAMD</span></div><div class="casAuthors">Phillips, James C.; Braun, Rosemary; Wang, Wei; Gumbart, James; Tajkhorshid, Emad; Villa, Elizabeth; Chipot, Christophe; Skeel, Robert D.; Kale, Laxmikant; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1781-1802</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">NAMD is a parallel mol. dynamics code designed for high-performance simulation of large biomol. systems.  NAMD scales to hundreds of processors on high-end parallel platforms, as well as tens of processors on low-cost commodity clusters, and also runs on individual desktop and laptop computers.  NAMD works with AMBER and CHARMM potential functions, parameters, and file formats.  This article, directed to novices as well as experts, first introduces concepts and methods used in the NAMD program, describing the classical mol. dynamics force field, equations of motion, and integration methods along with the efficient electrostatics evaluation algorithms employed and temp. and pressure controls used.  Features for steering the simulation across barriers and for calcg. both alchem. and conformational free energy differences are presented.  The motivations for and a roadmap to the internal design of NAMD, implemented in C++ and based on Charm++ parallel objects, are outlined.  The factors affecting the serial and parallel performance of a simulation are discussed.  Finally, typical NAMD use is illustrated with representative applications to a small, a medium, and a large biomol. system, highlighting particular features of NAMD, for example, the Tcl scripting language.  The article also provides a list of the key features of NAMD and discusses the benefits of combining NAMD with the mol. graphics/sequence anal. software VMD and the grid computing/collab. software BioCoRE.  NAMD is distributed free of charge with source code at www.ks.uiuc.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fq_yG00T1rVg90H21EOLACvtfcHk0ljcneZwgbByBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ&md5=189051128443b547f4300a1b8fb0e034</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20289%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DJ.%2BC.%26aulast%3DBraun%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGumbart%26aufirst%3DJ.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DChipot%26aufirst%3DC.%26aulast%3DSkeel%26aufirst%3DR.%2BD.%26aulast%3DKale%26aufirst%3DL.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DScalable%2520molecular%2520dynamics%2520with%2520NAMD%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1781%26epage%3D1802%26doi%3D10.1002%2Fjcc.20289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tosh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciancetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based scaffold repurposing for GPCRs: Transformation of adenosine derivatives into selective 5HT<sub>2B</sub>/5HT<sub>2C</sub> serotonin receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">11006</span>– <span class="NLM_lpage">11026</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=11006-11026&author=D.+K.+Toshauthor=A.+Ciancettaauthor=E.+Warnickauthor=S.+Craneauthor=Z.-G.+Gaoauthor=K.+A.+Jacobson&title=Structure-based+scaffold+repurposing+for+GPCRs%3A+Transformation+of+adenosine+derivatives+into+selective+5HT2B%2F5HT2C+serotonin+receptor+antagonists&doi=10.1021%2Facs.jmedchem.6b01183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01183%26sid%3Dliteratum%253Aachs%26aulast%3DTosh%26aufirst%3DD.%2BK.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DWarnick%26aufirst%3DE.%26aulast%3DCrane%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520scaffold%2520repurposing%2520for%2520GPCRs%253A%2520Transformation%2520of%2520adenosine%2520derivatives%2520into%2520selective%25205HT2B%252F5HT2C%2520serotonin%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D11006%26epage%3D11026%26doi%3D10.1021%2Facs.jmedchem.6b01183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Innocenti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K-M J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solanki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mas-Droux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westlake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Design of potent and selective hybrid inhibitors of the mitotic kinase Nek2: Structure–activity relationship, structural biology, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3228</span>– <span class="NLM_lpage">3241</span>, <span class="refDoi"> DOI: 10.1021/jm201683b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201683b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Wlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3228-3241&author=P.+Innocentiauthor=K-M+J.+Cheungauthor=S.+Solankiauthor=C.+Mas-Drouxauthor=F.+Rowanauthor=S.+Yeohauthor=K.+Boxallauthor=M.+Westlakeauthor=L.+Pickardauthor=T.+Hardyauthor=J.+E.+Baxterauthor=G.+W.+Aherneauthor=R.+Baylissauthor=A.+M.+Fryauthor=S.+Hoelder&title=Design+of+potent+and+selective+hybrid+inhibitors+of+the+mitotic+kinase+Nek2%3A+Structure%E2%80%93activity+relationship%2C+structural+biology%2C+and+cellular+activity&doi=10.1021%2Fjm201683b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Potent and Selective Hybrid Inhibitors of the Mitotic Kinase Nek2: Structure-Activity Relationship, Structural Biology, and Cellular Activity</span></div><div class="casAuthors">Innocenti, Paolo; Cheung, Kwai-Ming J.; Solanki, Savade; Mas-Droux, Corine; Rowan, Fiona; Yeoh, Sharon; Boxall, Kathy; Westlake, Maura; Pickard, Lisa; Hardy, Tara; Baxter, Joanne E.; Aherne, G. Wynne; Bayliss, Richard; Fry, Andrew M.; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3228-3241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein a series of Nek2 inhibitors based on an aminopyridine scaffold.  These compds. have been designed by combining key elements of two previously discovered chem. series.  Structure based design led to aminopyridine (R)-21 (I), a potent and selective inhibitor able to modulate Nek2 activity in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4618QDhLevLVg90H21EOLACvtfcHk0liDXxtnRI6HNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Wlu78%253D&md5=e777b9862be94a8e4dc22f6af6c85094</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm201683b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201683b%26sid%3Dliteratum%253Aachs%26aulast%3DInnocenti%26aufirst%3DP.%26aulast%3DCheung%26aufirst%3DK-M%2BJ.%26aulast%3DSolanki%26aufirst%3DS.%26aulast%3DMas-Droux%26aufirst%3DC.%26aulast%3DRowan%26aufirst%3DF.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DPickard%26aufirst%3DL.%26aulast%3DHardy%26aufirst%3DT.%26aulast%3DBaxter%26aufirst%3DJ.%2BE.%26aulast%3DAherne%26aufirst%3DG.%2BW.%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesign%2520of%2520potent%2520and%2520selective%2520hybrid%2520inhibitors%2520of%2520the%2520mitotic%2520kinase%2520Nek2%253A%2520Structure%25E2%2580%2593activity%2520relationship%252C%2520structural%2520biology%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3228%26epage%3D3241%26doi%3D10.1021%2Fjm201683b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akhlaghinia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezazadeh, S.</span></span> <span> </span><span class="NLM_article-title">A novel approach for the synthesis of 5-substituted-1<i>H</i>-tetrazoles</span>. <i>J. Braz. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2197</span>– <span class="NLM_lpage">2203</span>, <span class="refDoi"> DOI: 10.1590/S0103-50532013005000005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1590%2FS0103-50532013005000005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsFanu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=2197-2203&author=B.+Akhlaghiniaauthor=S.+Rezazadeh&title=A+novel+approach+for+the+synthesis+of+5-substituted-1H-tetrazoles&doi=10.1590%2FS0103-50532013005000005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A novel approach for the synthesis of 5-substituted-1H-tetrazoles</span></div><div class="casAuthors">Akhlaghinia, Batool; Rezazadeh, Soodabeh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Brazilian Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2197-2203</span>CODEN:
                <span class="NLM_cas:coden">JOCSET</span>;
        ISSN:<span class="NLM_cas:issn">0103-5053</span>.
    
            (<span class="NLM_cas:orgname">Sociedade Brasileira de Quimica</span>)
        </div><div class="casAbstract">A series of 5-substituted-1H-tetrazoles (RCN4H), e.g., I, have been synthesized by cycloaddn. reaction of different aryl and alkyl nitriles with sodium azide in DMSO using CuSO4·5H2O as catalyst.  A wide variety of aryl nitriles underwent [3 + 2] cycloaddn. to afford tetrazoles under mild reaction conditions in good to excellent yields.  The catalyst used is readily available and environmentally friendly.  Short reaction times, good to excellent yields, safe process and simple workup make this method an attractive and useful contribution to present org. synthesis of 5-substituted-1H-tetrazoles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Ulfd7WXuHbVg90H21EOLACvtfcHk0liDXxtnRI6HNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsFanu7c%253D&md5=6bdb2eabce7a4a0b2070cd72511cc355</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1590%2FS0103-50532013005000005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1590%252FS0103-50532013005000005%26sid%3Dliteratum%253Aachs%26aulast%3DAkhlaghinia%26aufirst%3DB.%26aulast%3DRezazadeh%26aufirst%3DS.%26atitle%3DA%2520novel%2520approach%2520for%2520the%2520synthesis%2520of%25205-substituted-1H-tetrazoles%26jtitle%3DJ.%2520Braz.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D23%26spage%3D2197%26epage%3D2203%26doi%3D10.1590%2FS0103-50532013005000005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amison, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shaughnessy, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofoedu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitchford, S. C.</span></span> <span> </span><span class="NLM_article-title">Lipopolysaccharide (LPS) induced pulmonary neutrophil recruitment and platelet activation is mediated via the P2Y<sub>1</sub> and P2Y<sub>14</sub> receptors in mice</span>. <i>Pulm. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.pupt.2017.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.pupt.2017.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=28487256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsFems74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=62-68&author=R.+T.+Amisonauthor=S.+Arnoldauthor=B.+G.+O%E2%80%99Shaughnessyauthor=S.+J.+Clearyauthor=J.+Ofoeduauthor=M.+Idzkoauthor=C.+P.+Pageauthor=S.+C.+Pitchford&title=Lipopolysaccharide+%28LPS%29+induced+pulmonary+neutrophil+recruitment+and+platelet+activation+is+mediated+via+the+P2Y1+and+P2Y14+receptors+in+mice&doi=10.1016%2Fj.pupt.2017.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Lipopolysaccharide (LPS) induced pulmonary neutrophil recruitment and platelet activation is mediated via the P2Y1 and P2Y14 receptors in mice</span></div><div class="casAuthors">Amison, R. T.; Arnold, S.; O'Shaughnessy, B. G.; Cleary, S. J.; Ofoedu, J.; Idzko, M.; Page, C. P.; Pitchford, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Pulmonary Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62-68</span>CODEN:
                <span class="NLM_cas:coden">PPTHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1094-5539</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Platelet activation occurs during host defense and in various inflammatory disorders.  In animal models of infection and inflammation, exptl. depletion of platelets leads to significantly reduced leukocyte recruitment and impaired clearance of pathogens from the lung.  It is now appreciated that purinergic receptor activation is required for leukocyte activation, motility and adhesion, and platelet interactions with leukocytes can be modulated by purinergic stimulation of platelets.  Here, we have investigated the role of platelet P2Y1, P2Y12, P2Y14, and P2X1 receptors on leukocyte recruitment and chemotaxis.  Mice were administered either vehicle controls or selective P2Y1, P2Y12, P2Y14, or P2X1 antagonists i.v. before intranasal administration of lipopolysaccharide (LPS) to investigate the effect of these drugs on pulmonary leukocyte recruitment, peripheral platelet counts, bleeding times, and ex vivo platelet aggregation.  Sep., platelets were incubated with P2Y1, P2Y12, P2X1 antagonists, or P2Y14 agonists to assess effects on platelet-induced neutrophil chemotaxis in vitro.  Pulmonary neutrophil recruitment induced by intranasal LPS administration was inhibited in mice administered either with P2Y1 or P2Y14 antagonists, but not with P2Y12 or P2X1 antagonists.  Furthermore, the administration of either a P2Y1 or a P2Y14 antagonist reversed the incidence of peripheral thrombocytopaenia assocd. with LPS exposure.  Bleeding times were significantly increased in mice administered P2Y1, P2Y12, or P2X1 antagonists, while ex vivo platelet aggregation to ADP was significantly reduced.  These haemostatic responses remained unaltered following antagonism of P2Y14.  In vitro chemotaxis assays revealed direct antagonism of platelet P2Y1, but not P2Y12 or P2X1 receptors suppressed platelet-dependent neutrophil motility towards Macrophage derived chemokine (MDC, CCL22).  Furthermore, the stimulation of platelets with selective P2Y14 agonists (UDP-glucose, MRS2690) resulted in significant platelet-dependent neutrophil chemotaxis.  These results reveal a role for P2Y1 and P2Y14 activation of platelets following exposure to LPS, while haemostatic indexes were unaffected by inhibition of platelet function with the P2Y14 antagonist in response to LPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ3K3utG7seLVg90H21EOLACvtfcHk0liDXxtnRI6HNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsFems74%253D&md5=823d73929bad49a537d073b62d7daec0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.pupt.2017.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pupt.2017.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DAmison%26aufirst%3DR.%2BT.%26aulast%3DArnold%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DB.%2BG.%26aulast%3DCleary%26aufirst%3DS.%2BJ.%26aulast%3DOfoedu%26aufirst%3DJ.%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DPage%26aufirst%3DC.%2BP.%26aulast%3DPitchford%26aufirst%3DS.%2BC.%26atitle%3DLipopolysaccharide%2520%2528LPS%2529%2520induced%2520pulmonary%2520neutrophil%2520recruitment%2520and%2520platelet%2520activation%2520is%2520mediated%2520via%2520the%2520P2Y1%2520and%2520P2Y14%2520receptors%2520in%2520mice%26jtitle%3DPulm.%2520Pharmacol.%2520Ther.%26date%3D2017%26volume%3D45%26spage%3D62%26epage%3D68%26doi%3D10.1016%2Fj.pupt.2017.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanchuk, V. Y.</span></span> <span> </span><span class="NLM_article-title">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1136</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci025515j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025515j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1136-1145&author=I.+V.+Tetkoauthor=V.+Y.+Tanchuk&title=Application+of+associative+neural+networks+for+prediction+of+lipophilicity+in+ALOGPS+2.1+program&doi=10.1021%2Fci025515j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span></div><div class="casAuthors">Tetko, Igor V.; Tanchuk, Vsevolod Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1136-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article provides a systematic study of several important parameters of the Associative Neural Network (ASNN), such as the no. of networks in the ensemble, distance measures, neighbor functions, selection of smoothing parameters, and strategies for the user-training feature of the algorithm.  The performance of the different methods is assessed with several training/test sets used to predict lipophilicity of chem. compds.  The Spearman rank-order correlation coeff. and Parzen-window regression methods provide the best performance of the algorithm.  If addnl. user data is available, an improved prediction of lipophilicity of chems. up to 2-5 times can be calcd. when the appropriate smoothing parameters for the neural network are selected.  The detected best combinations of parameters and strategies are implemented in the ALOGPS 2.1 program that is publicly available at http://www.vcclab.org/lab/alogps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Nz8kkyB1NbVg90H21EOLACvtfcHk0liaMloEJALi2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D&md5=45d412977a5171d4e8b03ada28c3632c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci025515j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025515j%26sid%3Dliteratum%253Aachs%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DTanchuk%26aufirst%3DV.%2BY.%26atitle%3DApplication%2520of%2520associative%2520neural%2520networks%2520for%2520prediction%2520of%2520lipophilicity%2520in%2520ALOGPS%25202.1%2520program%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1136%26epage%3D1145%26doi%3D10.1021%2Fci025515j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascher, D. B.</span></span> <span> </span><span class="NLM_article-title">pkCSM: Predicting small-molecule pharmacokinetic and toxicity properties using graph-based signatures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4066</span>– <span class="NLM_lpage">4072</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00104</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00104" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlams7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4066-4072&author=D.+E.+Piresauthor=T.+L.+Blundellauthor=D.+B.+Ascher&title=pkCSM%3A+Predicting+small-molecule+pharmacokinetic+and+toxicity+properties+using+graph-based+signatures&doi=10.1021%2Facs.jmedchem.5b00104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">pkCSM: Predicting Small-Molecule Pharmacokinetic and Toxicity Properties Using Graph-Based Signatures</span></div><div class="casAuthors">Pires, Douglas E. V.; Blundell, Tom L.; Ascher, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4066-4072</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug development has a high attrition rate, with poor pharmacokinetic and safety properties a significant hurdle.  Computational approaches may help minimize these risks.  The authors have developed a novel approach (pkCSM) which uses graph-based signatures to develop predictive models of central ADMET properties for drug development.  PkCSM performs as well or better than current methods.  A freely accessible web server (http://structure.bioc.cam.ac.uk/pkcsm), which retains no information submitted to it, provides an integrated platform to rapidly evaluate pharmacokinetic and toxicity properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMqi4e2w88abVg90H21EOLACvtfcHk0liaMloEJALi2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlams7w%253D&md5=2910009eb9e62f72822615eb049df123</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00104%26sid%3Dliteratum%253Aachs%26aulast%3DPires%26aufirst%3DD.%2BE.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAscher%26aufirst%3DD.%2BB.%26atitle%3DpkCSM%253A%2520Predicting%2520small-molecule%2520pharmacokinetic%2520and%2520toxicity%2520properties%2520using%2520graph-based%2520signatures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4066%26epage%3D4072%26doi%3D10.1021%2Facs.jmedchem.5b00104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 20 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Young-Hwan Jung, Veronica Salmaso, Zhiwei Wen, John M. Bennett, Ngan B. Phung, David I. Lieberman, Varun Gopinatth, John C. R. Randle, Zhoumou Chen, Daniela Salvemini, Tadeusz P. Karcz, Donald N. Cook, <span class="NLM_string-name hlFld-ContribAuthor">Kenneth A. Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationship of Heterocyclic P2Y14 Receptor Antagonists: Removal of the Zwitterionic Character with Piperidine Bioisosteres. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 5099-5122. <a href="https://doi.org/10.1021/acs.jmedchem.1c00164" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00164%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationship%252Bof%252BHeterocyclic%252BP2Y14%252BReceptor%252BAntagonists%25253A%252BRemoval%252Bof%252Bthe%252BZwitterionic%252BCharacter%252Bwith%252BPiperidine%252BBioisosteres%26aulast%3DJung%26aufirst%3DYoung-Hwan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D28012021%26date%3D31032021%26volume%3D64%26issue%3D8%26spage%3D5099%26epage%3D5122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Young-Hwan Jung, Jinha Yu, Zhiwei Wen, Veronica Salmaso, Tadeusz P. Karcz, Ngan B. Phung, Zhoumou Chen, Sierra Duca, John M. Bennett, Steven Dudas, Daniela Salvemini, Zhan-Guo Gao, Donald N. Cook, <span class="NLM_string-name hlFld-ContribAuthor">Kenneth A. Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of Alternative Scaffolds for P2Y14 Receptor Antagonists Containing a Biaryl Core. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9563-9589. <a href="https://doi.org/10.1021/acs.jmedchem.0c00745" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00745</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00745%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploration%252Bof%252BAlternative%252BScaffolds%252Bfor%252BP2Y14%252BReceptor%252BAntagonists%252BContaining%252Ba%252BBiaryl%252BCore%26aulast%3DJung%26aufirst%3DYoung-Hwan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03052020%26date%3D06082020%26volume%3D63%26issue%3D17%26spage%3D9563%26epage%3D9589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fatma Mufti, Young-Hwan Jung, Luigino Antonio Giancotti, Jinha Yu, Zhoumou Chen, Ngan B. Phung, Kenneth A. Jacobson, <span class="NLM_string-name hlFld-ContribAuthor">Daniela Salvemini</span>. </span><span class="cited-content_cbyCitation_article-title">P2Y14 Receptor Antagonists Reverse Chronic Neuropathic Pain in a Mouse Model. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1281-1286. <a href="https://doi.org/10.1021/acsmedchemlett.0c00115" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00115%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DP2Y14%252BReceptor%252BAntagonists%252BReverse%252BChronic%252BNeuropathic%252BPain%252Bin%252Ba%252BMouse%252BModel%26aulast%3DMufti%26aufirst%3DFatma%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08032020%26date%3D28042020%26date%3D30042020%26volume%3D11%26issue%3D6%26spage%3D1281%26epage%3D1286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuyang  Wang</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">P2Y14R for inflammation therapy: is it a promising target?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>10 </em><a href="https://doi.org/10.4155/fmc-2021-0084" title="DOI URL">https://doi.org/10.4155/fmc-2021-0084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0084%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DP2Y14R%252Bfor%252Binflammation%252Btherapy%25253A%252Bis%252Bit%252Ba%252Bpromising%252Btarget%25253F%26aulast%3DWang%26aufirst%3DYuyang%26date%3D2021%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jas  Kalayan</span>, <span class="hlFld-ContribAuthor ">Robin A.  Curtis</span>, <span class="hlFld-ContribAuthor ">Jim  Warwicker</span>, <span class="hlFld-ContribAuthor ">Richard H.  Henchman</span>. </span><span class="cited-content_cbyCitation_article-title">Thermodynamic Origin of Differential Excipient-Lysozyme Interactions. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.3389/fmolb.2021.689400" title="DOI URL">https://doi.org/10.3389/fmolb.2021.689400</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2021.689400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2021.689400%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DThermodynamic%252BOrigin%252Bof%252BDifferential%252BExcipient-Lysozyme%252BInteractions%26aulast%3DKalayan%26aufirst%3DJas%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ivar  von Kügelgen</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular pharmacology of P2Y receptor subtypes. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2021,</strong> <em>187 </em>, 114361. <a href="https://doi.org/10.1016/j.bcp.2020.114361" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114361</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114361%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DMolecular%252Bpharmacology%252Bof%252BP2Y%252Breceptor%252Bsubtypes%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DIvar%26date%3D2021%26volume%3D187%26spage%3D114361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Lu</span>, <span class="hlFld-ContribAuthor ">Yilin  Wang</span>, <span class="hlFld-ContribAuthor ">Chunxiao  Liu</span>, <span class="hlFld-ContribAuthor ">Zhenguo  Zhang</span>, <span class="hlFld-ContribAuthor ">Baiyang  Li</span>, <span class="hlFld-ContribAuthor ">Zibo  Meng</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>, <span class="hlFld-ContribAuthor ">Qinghua  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and evaluation of 3-amide-5-aryl benzoic acid derivatives as novel P2Y14R antagonists with potential high efficiency against acute gouty arthritis. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113313. <a href="https://doi.org/10.1016/j.ejmech.2021.113313" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113313</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113313%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252B3-amide-5-aryl%252Bbenzoic%252Bacid%252Bderivatives%252Bas%252Bnovel%252BP2Y14R%252Bantagonists%252Bwith%252Bpotential%252Bhigh%252Befficiency%252Bagainst%252Bacute%252Bgouty%252Barthritis%26aulast%3DLu%26aufirst%3DRan%26date%3D2021%26volume%3D216%26spage%3D113313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victor M.  Matias-Barrios</span>, <span class="hlFld-ContribAuthor ">Mariia  Radaeva</span>, <span class="hlFld-ContribAuthor ">Yi  Song</span>, <span class="hlFld-ContribAuthor ">Zaccary  Alperstein</span>, <span class="hlFld-ContribAuthor ">Ahn R.  Lee</span>, <span class="hlFld-ContribAuthor ">Veronika  Schmitt</span>, <span class="hlFld-ContribAuthor ">Joseph  Lee</span>, <span class="hlFld-ContribAuthor ">Fuqiang  Ban</span>, <span class="hlFld-ContribAuthor ">Ning  Xie</span>, <span class="hlFld-ContribAuthor ">Jianfei  Qi</span>, <span class="hlFld-ContribAuthor ">Nada  Lallous</span>, <span class="hlFld-ContribAuthor ">Martin E.  Gleave</span>, <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>, <span class="hlFld-ContribAuthor ">Xuesen  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of New Catalytic Topoisomerase II Inhibitors for Anticancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2021,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.633142" title="DOI URL">https://doi.org/10.3389/fonc.2020.633142</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.633142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.633142%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DDiscovery%252Bof%252BNew%252BCatalytic%252BTopoisomerase%252BII%252BInhibitors%252Bfor%252BAnticancer%252BTherapeutics%26aulast%3DMatias-Barrios%26aufirst%3DVictor%2BM.%26date%3D2021%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Salmaso</span>, <span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">Purinergic Signaling: Impact of GPCR Structures on Rational Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (21)
                                     , 1958-1973. <a href="https://doi.org/10.1002/cmdc.202000465" title="DOI URL">https://doi.org/10.1002/cmdc.202000465</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000465%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DPurinergic%252BSignaling%25253A%252BImpact%252Bof%252BGPCR%252BStructures%252Bon%252BRational%252BDrug%252BDesign%26aulast%3DSalmaso%26aufirst%3DVeronica%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D21%26spage%3D1958%26epage%3D1973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>, <span class="hlFld-ContribAuthor ">Esmerilda G.  Delicado</span>, <span class="hlFld-ContribAuthor ">Christian  Gachet</span>, <span class="hlFld-ContribAuthor ">Charles  Kennedy</span>, <span class="hlFld-ContribAuthor ">Ivar  Kügelgen</span>, <span class="hlFld-ContribAuthor ">Beibei  Li</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Miras‐Portugal</span>, <span class="hlFld-ContribAuthor ">Ivana  Novak</span>, <span class="hlFld-ContribAuthor ">Torsten  Schöneberg</span>, <span class="hlFld-ContribAuthor ">Raquel  Perez‐Sen</span>, <span class="hlFld-ContribAuthor ">Doreen  Thor</span>, <span class="hlFld-ContribAuthor ">Beili  Wu</span>, <span class="hlFld-ContribAuthor ">Zhenlin  Yang</span>, <span class="hlFld-ContribAuthor ">Christa E.  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Update of P2Y receptor pharmacology: IUPHAR Review 27. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2020,</strong> <em>177 </em>
                                    (11)
                                     , 2413-2433. <a href="https://doi.org/10.1111/bph.15005" title="DOI URL">https://doi.org/10.1111/bph.15005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.15005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.15005%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DUpdate%252Bof%252BP2Y%252Breceptor%252Bpharmacology%25253A%252BIUPHAR%252BReview%252B27%26aulast%3DJacobson%26aufirst%3DKenneth%2BA.%26date%3D2020%26date%3D2020%26volume%3D177%26issue%3D11%26spage%3D2413%26epage%3D2433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Salmaso</span>, <span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Drug Design for Purinergic GPCRs: Overview on Molecular Dynamics Applied to Adenosine and P2Y Receptors. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (6)
                                     , 812. <a href="https://doi.org/10.3390/biom10060812" title="DOI URL">https://doi.org/10.3390/biom10060812</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10060812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10060812%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DIn%252BSilico%252BDrug%252BDesign%252Bfor%252BPurinergic%252BGPCRs%25253A%252BOverview%252Bon%252BMolecular%252BDynamics%252BApplied%252Bto%252BAdenosine%252Band%252BP2Y%252BReceptors%26aulast%3DSalmaso%26aufirst%3DVeronica%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D6%26spage%3D812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Chunxiao  Liu</span>, <span class="hlFld-ContribAuthor ">Hanwen  Li</span>, <span class="hlFld-ContribAuthor ">Sheng  Tian</span>, <span class="hlFld-ContribAuthor ">Yingxian  Liu</span>, <span class="hlFld-ContribAuthor ">Nanxi  Wang</span>, <span class="hlFld-ContribAuthor ">Duanyang  Yan</span>, <span class="hlFld-ContribAuthor ">Huanqiu  Li</span>, <span class="hlFld-ContribAuthor ">Qinghua  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel and potent P2Y14R antagonists via structure-based virtual screening for the treatment of acute gouty arthritis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Advanced Research</span><span> <strong>2020,</strong> <em>23 </em>, 133-142. <a href="https://doi.org/10.1016/j.jare.2020.02.007" title="DOI URL">https://doi.org/10.1016/j.jare.2020.02.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jare.2020.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jare.2020.02.007%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Advanced%2520Research%26atitle%3DDiscovery%252Bof%252Bnovel%252Band%252Bpotent%252BP2Y14R%252Bantagonists%252Bvia%252Bstructure-based%252Bvirtual%252Bscreening%252Bfor%252Bthe%252Btreatment%252Bof%252Bacute%252Bgouty%252Barthritis%26aulast%3DWang%26aufirst%3DWeiwei%26date%3D2020%26volume%3D23%26spage%3D133%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuo  Yuan</span>, <span class="hlFld-ContribAuthor ">Sixi  Wang</span>, <span class="hlFld-ContribAuthor ">Min  Zhao</span>, <span class="hlFld-ContribAuthor ">Danqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinjie  Chen</span>, <span class="hlFld-ContribAuthor ">Jian-Xin  Li</span>, <span class="hlFld-ContribAuthor ">Jingya  Zhang</span>, <span class="hlFld-ContribAuthor ">Yihui  Song</span>, <span class="hlFld-ContribAuthor ">Jinyi  Wang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Brønsted acid-promoted ‘on–water’ C(sp3)-H functionalization for the synthesis of isoindolinone/[1,2,4]triazolo[1,5-a]pyrimidine derivatives targeting the SKP2-CKS1 interaction. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (2)
                                     , 349-352. <a href="https://doi.org/10.1016/j.cclet.2019.07.019" title="DOI URL">https://doi.org/10.1016/j.cclet.2019.07.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2019.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2019.07.019%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DBr%2525C3%2525B8nsted%252Bacid-promoted%252B%2525E2%252580%252598on%2525E2%252580%252593water%2525E2%252580%252599%252BC%252528sp3%252529-H%252Bfunctionalization%252Bfor%252Bthe%252Bsynthesis%252Bof%252Bisoindolinone%25252F%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine%252Bderivatives%252Btargeting%252Bthe%252BSKP2-CKS1%252Binteraction%26aulast%3DYuan%26aufirst%3DShuo%26date%3D2020%26volume%3D31%26issue%3D2%26spage%3D349%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daisuke  Ichinari</span>, <span class="hlFld-ContribAuthor ">Yosuke  Ashikari</span>, <span class="hlFld-ContribAuthor ">Kyoko  Mandai</span>, <span class="hlFld-ContribAuthor ">Yoko  Aizawa</span>, <span class="hlFld-ContribAuthor ">Jun‐ichi  Yoshida</span>, <span class="hlFld-ContribAuthor ">Aiichiro  Nagaki</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel Approach to Functionalization of Aryl Azides through the Generation and Reaction of Organolithium Species Bearing Masked Azides in Flow Microreactors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (4)
                                     , 1567-1571. <a href="https://doi.org/10.1002/anie.201912419" title="DOI URL">https://doi.org/10.1002/anie.201912419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201912419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201912419%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DA%252BNovel%252BApproach%252Bto%252BFunctionalization%252Bof%252BAryl%252BAzides%252Bthrough%252Bthe%252BGeneration%252Band%252BReaction%252Bof%252BOrganolithium%252BSpecies%252BBearing%252BMasked%252BAzides%252Bin%252BFlow%252BMicroreactors%26aulast%3DIchinari%26aufirst%3DDaisuke%26date%3D2020%26date%3D2019%26volume%3D59%26issue%3D4%26spage%3D1567%26epage%3D1571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daisuke  Ichinari</span>, <span class="hlFld-ContribAuthor ">Yosuke  Ashikari</span>, <span class="hlFld-ContribAuthor ">Kyoko  Mandai</span>, <span class="hlFld-ContribAuthor ">Yoko  Aizawa</span>, <span class="hlFld-ContribAuthor ">Jun‐ichi  Yoshida</span>, <span class="hlFld-ContribAuthor ">Aiichiro  Nagaki</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel Approach to Functionalization of Aryl Azides through the Generation and Reaction of Organolithium Species Bearing Masked Azides in Flow Microreactors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (4)
                                     , 1583-1587. <a href="https://doi.org/10.1002/ange.201912419" title="DOI URL">https://doi.org/10.1002/ange.201912419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201912419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201912419%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DA%252BNovel%252BApproach%252Bto%252BFunctionalization%252Bof%252BAryl%252BAzides%252Bthrough%252Bthe%252BGeneration%252Band%252BReaction%252Bof%252BOrganolithium%252BSpecies%252BBearing%252BMasked%252BAzides%252Bin%252BFlow%252BMicroreactors%26aulast%3DIchinari%26aufirst%3DDaisuke%26date%3D2020%26date%3D2019%26volume%3D132%26issue%3D4%26spage%3D1583%26epage%3D1587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenguo  Zhang</span>, <span class="hlFld-ContribAuthor ">Kun  Hao</span>, <span class="hlFld-ContribAuthor ">Hanwen  Li</span>, <span class="hlFld-ContribAuthor ">Ran  Lu</span>, <span class="hlFld-ContribAuthor ">Chunxiao  Liu</span>, <span class="hlFld-ContribAuthor ">Mengze  Zhou</span>, <span class="hlFld-ContribAuthor ">Baiyang  Li</span>, <span class="hlFld-ContribAuthor ">Zibo  Meng</span>, <span class="hlFld-ContribAuthor ">Qinghua  Hu</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and anti-inflammatory evaluation of 3-amide benzoic acid derivatives as novel P2Y14 receptor antagonists. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>181 </em>, 111564. <a href="https://doi.org/10.1016/j.ejmech.2019.111564" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111564</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111564%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Banti-inflammatory%252Bevaluation%252Bof%252B3-amide%252Bbenzoic%252Bacid%252Bderivatives%252Bas%252Bnovel%252BP2Y14%252Breceptor%252Bantagonists%26aulast%3DZhang%26aufirst%3DZhenguo%26date%3D2019%26volume%3D181%26spage%3D111564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ivar  von Kügelgen</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacology of P2Y receptors. </span><span class="cited-content_cbyCitation_journal-name">Brain Research Bulletin</span><span> <strong>2019,</strong> <em>151 </em>, 12-24. <a href="https://doi.org/10.1016/j.brainresbull.2019.03.010" title="DOI URL">https://doi.org/10.1016/j.brainresbull.2019.03.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.brainresbull.2019.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.brainresbull.2019.03.010%26sid%3Dliteratum%253Aachs%26jtitle%3DBrain%2520Research%2520Bulletin%26atitle%3DPharmacology%252Bof%252BP2Y%252Breceptors%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DIvar%26date%3D2019%26volume%3D151%26spage%3D12%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Khurshed  Bozorov</span>, <span class="hlFld-ContribAuthor ">Jiangyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Haji A.  Aisa</span>. </span><span class="cited-content_cbyCitation_article-title">1,2,3-Triazole-containing hybrids as leads in medicinal chemistry: A recent overview. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (16)
                                     , 3511-3531. <a href="https://doi.org/10.1016/j.bmc.2019.07.005" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.07.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.07.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3D1%25252C2%25252C3-Triazole-containing%252Bhybrids%252Bas%252Bleads%252Bin%252Bmedicinal%252Bchemistry%25253A%252BA%252Brecent%252Boverview%26aulast%3DBozorov%26aufirst%3DKhurshed%26date%3D2019%26volume%3D27%26issue%3D16%26spage%3D3511%26epage%3D3531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Lu</span>, <span class="hlFld-ContribAuthor ">Zhenguo  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on the discovery of P2Y14 receptor antagonists. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>175 </em>, 34-39. <a href="https://doi.org/10.1016/j.ejmech.2019.04.068" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.04.068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.04.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.04.068%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bon%252Bthe%252Bdiscovery%252Bof%252BP2Y14%252Breceptor%252Bantagonists%26aulast%3DLu%26aufirst%3DRan%26date%3D2019%26volume%3D175%26spage%3D34%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Salmaso</span>, <span class="hlFld-ContribAuthor ">Stefano  Moro</span>. </span><span class="cited-content_cbyCitation_article-title">Bridging Molecular Docking to Molecular Dynamics in Exploring Ligand-Protein Recognition Process: An Overview. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00923" title="DOI URL">https://doi.org/10.3389/fphar.2018.00923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00923%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DBridging%252BMolecular%252BDocking%252Bto%252BMolecular%252BDynamics%252Bin%252BExploring%252BLigand-Protein%252BRecognition%252BProcess%25253A%252BAn%252BOverview%26aulast%3DSalmaso%26aufirst%3DVeronica%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Progression of the Structural Modifications of Naphthoic Acid Derivative <b>1</b> To the Present Set of Derivatives <b>2b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The major focus was the introduction of diverse heteroaromatic rings in place of the bridging phenyl ring shown in green. The <i>p</i>-trifluoromethylphenyl moiety shown in red and the core benzene ring and adjacent triazole shown in blue were not modified in this study. Compound <b>3a</b> is the fluorescent antagonist containing structure <b>1</b> as a pharmacophore that was used in the flow cytometric assay. Group X is a carboxylic acid or a bioisosteric replacement.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Computational pipeline applied for the design of new derivatives as hP2Y<sub>14</sub>R antagonists. A library of compounds was designed according to the general scheme reported in <a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>. The compounds were docked at a hP2Y<sub>14</sub>R homology model (Glide, SP scoring function). Hit selection was performed by visual inspection followed, for a few selected candidates, by membrane molecular dynamics (MD) refinement (30 ns, run in triplicate, CHARMM36/CGenFF, POPC lipid model). The MD refinement phase was sped up using High Throughput MD (HTMD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Detailed General Approach for the Design of New Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>The expected orientation of the scaffold with respect to the hP2Y<sub>14</sub>R is shown in a symbolic representation. Diverse 5- and 6-membered heteroaromatic rings linking the core (cyan) to the group facing the EC side of the receptor (yellow) were considered, although not all of these analogues were synthesized. Various linking groups or linkers (magenta), including amide, sulfonamide, and methylene, were inserted to provide conformational diversity in the terminal alkylamino moiety. The terminal group on the spacer moiety was an H-bond donor/acceptor group, an amino, a protected amino, or a hydroxyl group. The spacer group consisted of either a straight chain alkyl moiety or a six-membered ring containing the secondary amine.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Upper panels: Selection of the aromatic ring. Docking pose of (A) thiophene derivative <b>13</b> (cyan carbon atoms) and (B) benzene derivative <b>16</b> (salmon carbon atoms) at the hP2Y<sub>14</sub>R. Both the thiophene and the benzene rings establish an additional π–cation interaction with Arg274<sup>7.32</sup> (dark green dashed lines). Lower panel: Selection of the optimal spacer between the aromatic and amine functions. Docking poses of derivatives bearing a spacer of three methylene groups and a (C) terminal primary amine (compound <b>7</b>, light blue carbon atoms) or (D) a hydroxyl group (dark green carbon atoms) at the hP2Y<sub>14</sub>R. Side chains of residues important for ligand recognition are reported as sticks (gray carbon atoms). H-bonds, salt bridges, and π–π stacking interactions are pictured as orange, magenta, and cyan dashed lines, respectively. Nonpolar hydrogen atoms are omitted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selection of the optimal linker geometry: alkyl sulfonamide vs amide. Upper panel: docking analysis. (A) Superimposition of docking poses of sulfonamide derivative <b>7</b> (light blue carbon atoms) and amide derivative <b>20</b> (purple carbon atom) representing the starting geometry for subsequent MD simulations. (B) Detail of the shape (highlighted with a solid yellow line) of the narrow pocket surrounding the thiophene ring delimited by the conserved disulfide bridge and the EC tip of TM2 of the hP2Y<sub>14</sub>R. Lower panel: lowest interaction energy (IE) structure extracted from the selected trajectory of 30 ns of MD simulation of the (C) <b>7</b>–hP2Y<sub>14</sub>R and (D) <b>20</b>–hP2Y<sub>14</sub>R complexes. Side chains of residues important for ligand recognition are reported as sticks (gray carbon atoms). H-bonds and salt bridges are pictured as orange and magenta dashed lines, respectively. Nonpolar hydrogen atoms and π–π stacking interactions are omitted. The binding site surface is color-coded by residue property (blue, positively charged; red, negatively charged; green, hydrophobic; cyan, polar). The interaction pattern around the carboxylate moiety is highlighted with a yellow circle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, CH<sub>3</sub>OH, rt, 15 h, 98%; (b) PTSA, NaNO<sub>2</sub> then NaN<sub>3</sub>, H<sub>2</sub>O, CH<sub>3</sub>CN, rt, 15 h, 63%; (c) 4-ethynyltrifluorotoluene, sodium ascorbate, CuSO<sub>4</sub>, THF, H<sub>2</sub>O, rt, 1 h, 84%; (d) LiOH, MeOH, 60 °C, 2 h, 76%; (e) B<sub>2</sub>pin<sub>2</sub>, KOAc, PdCl<sub>2</sub>(dppf), dioxane, 70 °C, 15 h, 87%; (f) (i) 10% NaOH, H<sub>2</sub>O<sub>2</sub>, THF, 0 °C, 0.5 h; (ii) KOH, MeOH, water, 50 °C, 5 h, 86%.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, DCM, 0 °C, 0.5 h, 70%; (b) amine, Et<sub>3</sub>N, DCM, rt, 78–99%; (c) <i>N</i>-Boc piperazine, Et<sub>3</sub>N, NaBH(OAc)<sub>3</sub>, DCE, rt, 15 h, 90%; (d) amine, HATU, DIPEA, DMF, rt, 14–88%; (e) amine, Et<sub>3</sub>N, CH<sub>3</sub>CN, 0 °C, 73–99%.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) PdCl<sub>2</sub>(dppf), 2 M Na<sub>2</sub>CO<sub>3</sub>, DME, 50 °C, 25–73%; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub> (KOAc for <b>79</b>), DMF, H<sub>2</sub>O, 11–43%; (c) KOH, MeOH, H<sub>2</sub>O, 50–70 °C, 30–99%; (d) TFA, rt, 34–92%; (e) AlCl<sub>3</sub>, Me<sub>2</sub>S, rt, 15 h, 29–61%. The methyl esters of <b>79</b>–<b>102</b> were hydrolyzed using KOH to give compound <b>4</b>, <b>6</b>, <b>8</b>, <b>10</b>, <b>11</b>, <b>13</b>, <b>14</b>, <b>16</b>, <b>17</b>, <b>19</b>, <b>21</b>, and <b>27</b>–<b>34</b>. The <i>N</i>-Boc groups of compounds <b>103</b>, <b>4</b>, <b>6</b>, <b>8</b>, <b>11</b>, <b>14</b>, <b>17</b>, <b>19</b>, and <b>21</b> were removed with TFA to give compounds <b>2a</b>, <b>5</b>, <b>7</b>, <b>9</b>, <b>12</b>, <b>15</b>, <b>18</b>, <b>20</b>, and <b>22</b>, respectively. Compounds <b>23</b> and <b>25</b>–<b>26</b> were obtained using AlCl<sub>3</sub> and Me<sub>2</sub>S.</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0010.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NH<sub>4</sub>Cl, HATU, DIPEA, DMF, rt, 1 h, 80–99%; (b) (CF<sub>3</sub>CO)<sub>2</sub>O, Et<sub>3</sub>N, DCM, 0 °C to rt, 1 h, 64–76%; (c) TMSOTf, anisole, μW, 140 °C, 1 h, 80%; (d) CuSO<sub>4</sub>·5H<sub>2</sub>O, NaN<sub>3</sub>, DMSO, 140 °C, 6 h, 69–99%; (e) TFA, rt, 10 min; 44–74%.</p></p></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0011.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, rt, 3 h, 80%; (b) (i) MsCl, Et<sub>3</sub>N, DCM, 0 °C, 5 min; (ii) NaN<sub>3</sub>, NBu<sub>4</sub>Br, H<sub>2</sub>O, 100 °C, 15 h, 59%; (c) KOH, MeOH, water, 50 °C, 10 h; (d) (i) Tf<sub>2</sub>O, pyr, −78 °C to rt, 1 h; (ii) B<sub>2</sub>pin<sub>2</sub>, PdCl<sub>2</sub>(dppf), KOAc, dioxane, 85 °C, 4 h, 61%; (e) <i>tert</i>-butyl 4-(4-bromophenyl)piperidine-1-carboxylate <b>55</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 5 h, 59%; (f) TFA/THF = 1:1, rt, 1 h, 93%; (g) 6-bromohexyne-1, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 15 h, 70%; (h) <b>117</b>, CuSO<sub>4</sub>·5H<sub>2</sub>O, sodium ascorbate, DCM/<i>t</i>BuOH/H<sub>2</sub>O = 1:1:1, rt, 15 h, 58%; (i) LiOH, MeOH, 80 °C, 15 h, 76%; (j) TFA, rt, 30 min, 78%; (k) Ac<sub>2</sub>O, pyr, rt, 1 h, 81%.</p></p></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/medium/jm-2018-00168h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fluorescent assay of specific hP2Y<sub>14</sub>R binding of selected 3-(4-trifluorophenyl-1<i>H</i>-1,2,3-triazol-1-yl)-5-(aryl) derivatives. AlexaFluor488-labeled tracer <b>3a</b> binding was determined by flow cytometry using P2Y<sub>14</sub>R-expressing CHO cells. (A) 5-Thienyl <b>12</b> and <b>20</b>, 5-furyl <b>24</b>, and primary 4-carboxamidophenyl <b>30</b> derivatives. (B) Primary 4-substituted carboxamides with 5-pyridyl <b>27</b> and <b>28</b> and 5-pyrimidyl <b>29</b> aromatic groups. (C) <i>N</i>-Alkyl elaboration of a potent 4-carboxamidophenyl congener <b>30</b>: Me, <b>31</b>; Et, <b>32</b>; <i>n</i>-Pr, <b>33</b>; <i>tert</i>-Bu, <b>34</b>. The IC<sub>50</sub> values are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00168/20180608/images/large/jm-2018-00168h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00168&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i121">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnstock, G.</span></span> <span> </span><span class="NLM_article-title">Purinergic signalling: Therapeutic developments</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">661</span>, <span class="refDoi"> DOI: 10.3389/fphar.2017.00661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.3389%2Ffphar.2017.00661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=28993732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFegs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=661&author=G.+Burnstock&title=Purinergic+signalling%3A+Therapeutic+developments&doi=10.3389%2Ffphar.2017.00661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Purinergic signalling: therapeutic developments</span></div><div class="casAuthors">Burnstock, Geoffrey</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">661/1-661/55</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Purinergic signalling, i.e., the role of nucleotides as extracellular signalling mols., was proposed in 1972.  However, this concept was not well accepted until the early 1990-s when receptor subtypes for purines and pyrimidines were cloned and characterised, which includes four subtypes of the P1 (adenosine) receptor, seven subtypes of P2X ion channel receptors and 8 subtypes of the P2Y G protein-coupled receptor.  Early studies were largely concerned with the physiol., pharmacol. and biochem. of purinergic signalling.  More recently, the focus has been on the pathophysiol. and therapeutic potential.  There was early recognition of the use of P1 receptor agonists for the treatment of supraventricular tachycardia and A2A receptor antagonists are promising for the treatment of Parkinson-s disease.  Clopidogrel, a P2Y12 antagonist, is widely used for the treatment of thrombosis and stroke, blocking P2Y12 receptor-mediated platelet aggregation.  Diquafosol, a long acting P2Y2 receptor agonist, is being used for the treatment of dry eye.  P2X3 receptor antagonists have been developed that are orally bioavailable and stable in vivo and are currently in clin. trials for the treatment of chronic cough, bladder incontinence, visceral pain and hypertension.  Antagonists to P2X7 receptors are being investigated for the treatment of inflammatory disorders, including neurodegenerative diseases.  Other investigations are in progress for the use of purinergic agents for the treatment of osteoporosis, myocardial infarction, irritable bowel syndrome, epilepsy, atherosclerosis, depression, autism, diabetes, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9VdnZ8zYY7Vg90H21EOLACvtfcHk0libH2BwFat6RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFegs7fK&md5=b28408418adacae80284bcf51c778b3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2017.00661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2017.00661%26sid%3Dliteratum%253Aachs%26aulast%3DBurnstock%26aufirst%3DG.%26atitle%3DPurinergic%2520signalling%253A%2520Therapeutic%2520developments%26jtitle%3DFront.%2520Pharmacol.%26date%3D2017%26volume%3D8%26spage%3D661%26doi%3D10.3389%2Ffphar.2017.00661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cekic, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linden, J.</span></span> <span> </span><span class="NLM_article-title">Purinergic regulation of the immune system</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nri.2016.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnri.2016.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=26922909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtlKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=177-192&author=C.+Cekicauthor=J.+Linden&title=Purinergic+regulation+of+the+immune+system&doi=10.1038%2Fnri.2016.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Purinergic regulation of the immune system</span></div><div class="casAuthors">Cekic, Caglar; Linden, Joel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-192</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cellular stress or apoptosis triggers the release of ATP, ADP and other nucleotides into the extracellular space.  Extracellular nucleotides function as autocrine and paracrine signalling mols. by activating cell-surface P2 purinergic receptors that elicit pro-inflammatory immune responses.  Over time, extracellular nucleotides are metabolized to adenosine, leading to reduced P2 signalling and increased signalling through anti-inflammatory adenosine (P1 purinergic) receptors.  Here, we review how local purinergic signalling changes over time during tissue responses to injury or disease, and we discuss the potential of targeting purinergic signalling pathways for the immunotherapeutic treatment of ischemia, organ transplantation, autoimmunity or cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod6c3DxezmqbVg90H21EOLACvtfcHk0ljm4MUbJjb8sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtlKmurg%253D&md5=930862a0d6cfd16d6c3cdefc45094d0d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.4%26sid%3Dliteratum%253Aachs%26aulast%3DCekic%26aufirst%3DC.%26aulast%3DLinden%26aufirst%3DJ.%26atitle%3DPurinergic%2520regulation%2520of%2520the%2520immune%2520system%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D177%26epage%3D192%26doi%3D10.1038%2Fnri.2016.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbracchio, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnstock, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeynaems, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gachet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisman, G. A.</span></span> <span> </span><span class="NLM_article-title">International Union of Pharmacology LVIII: update on the P2Y G protein-coupled nucleotide receptors: from molecular mechanisms and pathophysiology to therapy</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1124/pr.58.3.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fpr.58.3.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=16968944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVOhtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=281-341&author=M.+P.+Abbracchioauthor=G.+Burnstockauthor=J.+M.+Boeynaemsauthor=E.+A.+Barnardauthor=J.+L.+Boyerauthor=C.+Kennedyauthor=M.+Fumagalliauthor=B.+F.+Kingauthor=C.+Gachetauthor=K.+A.+Jacobsonauthor=G.+A.+Weisman&title=International+Union+of+Pharmacology+LVIII%3A+update+on+the+P2Y+G+protein-coupled+nucleotide+receptors%3A+from+molecular+mechanisms+and+pathophysiology+to+therapy&doi=10.1124%2Fpr.58.3.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">International Union of Pharmacology LVIII: update on the P2Y G protein-coupled nucleotide receptors: from molecular mechanisms and pathophysiology to therapy</span></div><div class="casAuthors">Abbracchio, Maria P.; Burnstock, Geoffrey; Boeynaems, Jean-Marie; Barnard, Eric A.; Boyer, Jose L.; Kennedy, Charles; Knight, Gillian E.; Fumagalli, Marta; Gachet, Christian; Jacobson, Kenneth A.; Weisman, Gary A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-341</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  There have been many advances in our knowledge about different aspects of P2Y receptor signaling since the last review published by our International Union of Pharmacol. subcommittee.  More receptor subtypes have been cloned and characterized and most orphan receptors deorphanized, so that it is now possible to provide a basis for a future subdivision of P2Y receptor subtypes.  More is known about the functional elements of the P2Y receptor mols. and the signaling pathways involved, including interactions with ion channels.  There have been substantial developments in the design of selective agonists and antagonists to some of the P2Y receptor subtypes.  There are new findings about the mechanisms underlying nucleotide release and ectoenzymic nucleotide breakdown.  Interactions between P2Y receptors and receptors to other signaling mols. have been explored as well as P2Y-mediated control of gene transcription.  The distribution and roles of P2Y receptor subtypes in many different cell types are better understood and P2Y receptor-related compds. are being explored for therapeutic purposes.  These and other advances are discussed in the present review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-m5Cf-JzaLVg90H21EOLACvtfcHk0ljm4MUbJjb8sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVOhtLbF&md5=b302ee34857a5f45d740e4f2e962ed42</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1124%2Fpr.58.3.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.58.3.3%26sid%3Dliteratum%253Aachs%26aulast%3DAbbracchio%26aufirst%3DM.%2BP.%26aulast%3DBurnstock%26aufirst%3DG.%26aulast%3DBoeynaems%26aufirst%3DJ.%2BM.%26aulast%3DBarnard%26aufirst%3DE.%2BA.%26aulast%3DBoyer%26aufirst%3DJ.%2BL.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DFumagalli%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DB.%2BF.%26aulast%3DGachet%26aufirst%3DC.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DWeisman%26aufirst%3DG.%2BA.%26atitle%3DInternational%2520Union%2520of%2520Pharmacology%2520LVIII%253A%2520update%2520on%2520the%2520P2Y%2520G%2520protein-coupled%2520nucleotide%2520receptors%253A%2520from%2520molecular%2520mechanisms%2520and%2520pathophysiology%2520to%2520therapy%26jtitle%3DPharmacol.%2520Rev.%26date%3D2006%26volume%3D58%26spage%3D281%26epage%3D341%26doi%3D10.1124%2Fpr.58.3.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span> <span> </span><span class="NLM_article-title">Signalling and pharmacological properties of the P2Y<sub>14</sub> receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1124/mol.115.098756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fmol.115.098756" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=151-160&author=E.+R.+Lazarowskiauthor=T.+K.+Harden&title=Signalling+and+pharmacological+properties+of+the+P2Y14+receptor&doi=10.1124%2Fmol.115.098756"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1124%2Fmol.115.098756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.115.098756%26sid%3Dliteratum%253Aachs%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26atitle%3DSignalling%2520and%2520pharmacological%2520properties%2520of%2520the%2520P2Y14%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D151%26epage%3D160%26doi%3D10.1124%2Fmol.115.098756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinckwich-Besancon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Mata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span> <span> </span><span class="NLM_article-title">The UDP-sugar-sensing P2Y<sub>14</sub> receptor promotes Rho-mediated signaling and chemotaxis in human neutrophils</span>. <i>Am. J. Physiol. - Cell Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">C490</span>– <span class="NLM_lpage">C498</span>, <span class="refDoi"> DOI: 10.1152/ajpcell.00138.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1152%2Fajpcell.00138.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22673622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2012&pages=C490-C498&author=J.+I.+Sesmaauthor=S.+M.+Kredaauthor=N.+Steinckwich-Besanconauthor=H.+Dangauthor=R.+Garc%C3%ADa-Mataauthor=T.+K.+Hardenauthor=E.+R.+Lazarowski&title=The+UDP-sugar-sensing+P2Y14+receptor+promotes+Rho-mediated+signaling+and+chemotaxis+in+human+neutrophils&doi=10.1152%2Fajpcell.00138.2012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1152%2Fajpcell.00138.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpcell.00138.2012%26sid%3Dliteratum%253Aachs%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DKreda%26aufirst%3DS.%2BM.%26aulast%3DSteinckwich-Besancon%26aufirst%3DN.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DGarc%25C3%25ADa-Mata%26aufirst%3DR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26atitle%3DThe%2520UDP-sugar-sensing%2520P2Y14%2520receptor%2520promotes%2520Rho-mediated%2520signaling%2520and%2520chemotaxis%2520in%2520human%2520neutrophils%26jtitle%3DAm.%2520J.%2520Physiol.%2520-%2520Cell%2520Physiol.%26date%3D2012%26volume%3D303%26spage%3DC490%26epage%3DC498%26doi%3D10.1152%2Fajpcell.00138.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayasekara, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span> <span> </span><span class="NLM_article-title">A selective high-affinity antagonist of the P2Y<sub>14</sub> receptor inhibits UDP-glucose–stimulated chemotaxis of human neutrophils</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1124/mol.113.085654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1124%2Fmol.113.085654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=41-49&author=M.+O.+Barrettauthor=J.+I.+Sesmaauthor=C.+B.+Ballauthor=P.+S.+Jayasekaraauthor=K.+A.+Jacobsonauthor=E.+R.+Lazarowskiauthor=T.+K.+Harden&title=A+selective+high-affinity+antagonist+of+the+P2Y14+receptor+inhibits+UDP-glucose%E2%80%93stimulated+chemotaxis+of+human+neutrophils&doi=10.1124%2Fmol.113.085654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.085654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.085654%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DM.%2BO.%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DBall%26aufirst%3DC.%2BB.%26aulast%3DJayasekara%26aufirst%3DP.%2BS.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26atitle%3DA%2520selective%2520high-affinity%2520antagonist%2520of%2520the%2520P2Y14%2520receptor%2520inhibits%2520UDP-glucose%25E2%2580%2593stimulated%2520chemotaxis%2520of%2520human%2520neutrophils%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D41%26epage%3D49%26doi%3D10.1124%2Fmol.113.085654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">UDP-glucose acting at P2Y<sub>14</sub> receptors is a mediator of mast cell degranulation</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2009.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bcp.2009.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=19896471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=873-879&author=Z.-G.+Gaoauthor=Y.+Dingauthor=K.+A.+Jacobson&title=UDP-glucose+acting+at+P2Y14+receptors+is+a+mediator+of+mast+cell+degranulation&doi=10.1016%2Fj.bcp.2009.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-glucose acting at P2Y14 receptors is a mediator of mast cell degranulation</span></div><div class="casAuthors">Gao, Zhan-Guo; Ding, Yi; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">873-879</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">UDP-glucose (UDPG), a glycosyl donor in the biosynthesis of carbohydrates, is an endogenous agonist of the G protein-coupled P2Y14 receptor.  RBL-2H3 mast cells endogenously express a P2Y14 receptor at which UDPG mediates degranulation as indicated by β-hexosaminidase (HEX) release.  Both UDPG and a more potent, selective 2-thio-modified UDPG analog, MRS2690 (diphosphoric acid 1-α--glucopyranosyl ester 2-[(2-thio)uridin-5''-yl] ester), caused a substantial calcium transient in RBL-2H3 cells, which was blocked by pertussis toxin, indicating the presence of the Gi-coupled P2Y14 receptor, supported also by quant. detection of abundant mRNA.  Expression of the closely related P2Y6 receptor was over 100 times lower than the P2Y14 receptor, and the P2Y6 agonist 3-phenacyl-UDP was inactive in RBL-2H3 cells.  P2Y14 receptor agonists also induced [35S]GTPγS binding to RBL-2H3 cell membranes, and phosphorylation of ERK1/2, P38 and JNK.  UDPG and MRS2690 concn.-dependently enhanced HEX release with EC50 values of 1150 and 103 nM, resp.  The enhancement was completely blocked by pertussis toxin and significantly diminished by P2Y14 receptor-specific siRNA.  Thus, mast cells express an endogenous P2Y14 receptor, which mediates Gi-dependent degranulation and is therefore a potential novel therapeutic target for allergic conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0MRGdKvsSbbVg90H21EOLACvtfcHk0lgw0NiCAH5KBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFGjsg%253D%253D&md5=cda30b06aac8a61c30aa405076bcb7df</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2009.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2009.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DUDP-glucose%2520acting%2520at%2520P2Y14%2520receptors%2520is%2520a%2520mediator%2520of%2520mast%2520cell%2520degranulation%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D873%26epage%3D879%26doi%3D10.1016%2Fj.bcp.2009.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azroyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez-Retamozo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittet, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breton, S.</span></span> <span> </span><span class="NLM_article-title">Renal intercalated cells sense and mediate sterile inflammation via the P2Y<sub>14</sub> receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e0121419</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0121419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1371%2Fjournal.pone.0121419" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121419&issue=3&author=A.+Azroyanauthor=V.+Cortez-Retamozoauthor=R.+Bouleyauthor=R.+Libermanauthor=Y.+C.+Ruanauthor=E.+Kiselevauthor=K.+A.+Jacobsonauthor=M.+J.+Pittetauthor=D.+Brownauthor=S.+Breton&title=Renal+intercalated+cells+sense+and+mediate+sterile+inflammation+via+the+P2Y14+receptor&doi=10.1371%2Fjournal.pone.0121419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121419%26sid%3Dliteratum%253Aachs%26aulast%3DAzroyan%26aufirst%3DA.%26aulast%3DCortez-Retamozo%26aufirst%3DV.%26aulast%3DBouley%26aufirst%3DR.%26aulast%3DLiberman%26aufirst%3DR.%26aulast%3DRuan%26aufirst%3DY.%2BC.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DPittet%26aufirst%3DM.%2BJ.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBreton%26aufirst%3DS.%26atitle%3DRenal%2520intercalated%2520cells%2520sense%2520and%2520mediate%2520sterile%2520inflammation%2520via%2520the%2520P2Y14%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D3%26spage%3De0121419%26doi%3D10.1371%2Fjournal.pone.0121419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morinaga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukdar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olefsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. J.</span></span> <span> </span><span class="NLM_article-title">GPR105 Ablation prevents inflammation and improves insulin sensitivity in mice with diet-induced obesity</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">1992</span>– <span class="NLM_lpage">1999</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1103207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.4049%2Fjimmunol.1103207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22778393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFaqsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2012&pages=1992-1999&author=J.+Xuauthor=H.+Morinagaauthor=D.+Ohauthor=P.+Liauthor=A.+Chenauthor=S.+Talukdarauthor=E.+Lazarowskiauthor=J.+M.+Olefskyauthor=J.+J.+Kim&title=GPR105+Ablation+prevents+inflammation+and+improves+insulin+sensitivity+in+mice+with+diet-induced+obesity&doi=10.4049%2Fjimmunol.1103207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">GPR105 Ablation Prevents Inflammation and Improves Insulin Sensitivity in Mice with Diet-Induced Obesity</span></div><div class="casAuthors">Xu, Jianfeng; Morinaga, Hidetaka; Oh, Dayoung; Li, Pingping; Chen, Ai; Talukdar, Saswata; Mamane, Yaeel; Mancini, Joseph A.; Nawrocki, Andrea R.; Lazarowski, Eduardo; Olefsky, Jerrold M.; Kim, Jane J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1992-1999</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">GPR105, a G protein-coupled receptor for UDP-glucose, is highly expressed in several human tissues and participates in the innate immune response.  Because inflammation has been implicated as a key initial trigger for type 2 diabetes, we hypothesized that GPR105 (official gene name: P2RY14) might play a role in the initiation of inflammation and insulin resistance in obesity.  To this end, we investigated glucose metab. in GPR105 knockout (KO) and wild-type (WT) mice fed a high-fat diet (HFD).  We also examd. whether GPR105 regulates macrophage recruitment to liver or adipose tissues by in vivo monocyte tracking and in vitro chemotaxis expts., followed by transplantation of bone marrow from either KO or WT donors to WT recipients.  Our data show that genetic deletion of GPR105 confers protection against HFD-induced insulin resistance, with reduced macrophage infiltration and inflammation in liver, and increased insulin-stimulated Akt phosphorylation in liver, muscle, and adipose tissue.  By tracking monocytes from either KO or WT donors, we found that fewer KO monocytes were recruited to the liver of WT recipients.  Furthermore, we obsd. that uridine 5-diphosphoglucose enhanced the in vitro migration of bone marrow-derived macrophages from WT but not KO mice, and that plasma uridine 5-diphosphoglucose levels were significantly higher in obese vs. lean mice.  Finally, we confirmed that insulin sensitivity improved in HFD mice with a myeloid cell-specific deletion of GPR105.  These studies indicate that GPR105 ablation mitigates HFD-induced insulin resistance by inhibiting macrophage recruitment and tissue inflammation.  Hence GPR105 provides a novel link between innate immunity and metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE2qDmFMoeT7Vg90H21EOLACvtfcHk0lgC70-_pvBN3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFaqsLzM&md5=3c495773f43348d3731f85ee3e207072</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1103207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1103207%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMorinaga%26aufirst%3DH.%26aulast%3DOh%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DTalukdar%26aufirst%3DS.%26aulast%3DLazarowski%26aufirst%3DE.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BJ.%26atitle%3DGPR105%2520Ablation%2520prevents%2520inflammation%2520and%2520improves%2520insulin%2520sensitivity%2520in%2520mice%2520with%2520diet-induced%2520obesity%26jtitle%3DJ.%2520Immunol.%26date%3D2012%26volume%3D189%26spage%3D1992%26epage%3D1999%26doi%3D10.4049%2Fjimmunol.1103207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasu-Tada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujishita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, S.</span></span> <span> </span><span class="NLM_article-title">Secretion of matrix metalloproteinase-9 from astrocytes by inhibition of tonic P2Y<sub>14</sub>-receptor-mediated signal(s)</span>. <i>Cell. Mol. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1007/s10571-012-9869-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs10571-012-9869-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22872320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFSnsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=47-58&author=M.+Kinoshitaauthor=K.+Nasu-Tadaauthor=K.+Fujishitaauthor=K.+Satoauthor=S.+Koizumi&title=Secretion+of+matrix+metalloproteinase-9+from+astrocytes+by+inhibition+of+tonic+P2Y14-receptor-mediated+signal%28s%29&doi=10.1007%2Fs10571-012-9869-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Secretion of Matrix Metalloproteinase-9 from Astrocytes by Inhibition of Tonic P2Y14-Receptor-Mediated Signal(s)</span></div><div class="casAuthors">Kinoshita, Manao; Nasu-Tada, Kaoru; Fujishita, Kayoko; Sato, Kaoru; Koizumi, Schuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Neurobiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">CMNEDI</span>;
        ISSN:<span class="NLM_cas:issn">0272-4340</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Glial cells have various important roles in regulation of brain functions.  For such events, extracellular nucleotides/P2 receptors have central roles.  Although there have been huge amt. of literature about activation of P2 receptors and glial functions, little is known about what happens in glia or the brain if glial P2 receptor is inhibited.  Here we show that the inhibition of P2 receptors in astrocytes, the most abundant glial cells and cause a constitutive release of nucleotides, resulted in secretion of metalloproteinase-9 (MMP-9), a metal-dependent endopeptidase that degrades extracellular matrix mols. and is important in regulation of brain remodeling.  When cultured astrocytes were treated with apyrase (ecto-nucleotidase), reactive blue 2 (P2 receptor antagonist), and pertussis toxin, they secreted MMP-9, suggesting that Gi-coupled P2Y receptor-mediated signals constitutively suppress the prodn. of MMP-9.  Among Gi-coupled P2Y receptors, we found that an inhibition of P2Y14 receptor, a receptor for nucleotide-sugars such as UDP-glucose, is responsible for the prodn. of MMP-9 by pharmacol. and mol. biochem. anal.  As for the mechanisms, the inhibition of P2Y14 receptors resulted in the release of tumor necrosis factor (TNF)-α which then acted on astrocytes to induce MMP-9.  Taken together, our results suggest that the constitutive releases of nucleotide-sugars in astrocytes should play an important role in maintaining the normal status of the cell, through Gi-coupled P2Y14 receptors, and when the signal is removed, the cells start to release TNF-α, which then acts on astrocytes in a feedback fashion to boost MMP-9 synthesis and secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8KyPG_sOekLVg90H21EOLACvtfcHk0ljr9qQHXtoVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFSnsLjJ&md5=c0c76a48491f9c006e522875c9a39c71</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10571-012-9869-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10571-012-9869-4%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DM.%26aulast%3DNasu-Tada%26aufirst%3DK.%26aulast%3DFujishita%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DKoizumi%26aufirst%3DS.%26atitle%3DSecretion%2520of%2520matrix%2520metalloproteinase-9%2520from%2520astrocytes%2520by%2520inhibition%2520of%2520tonic%2520P2Y14-receptor-mediated%2520signal%2528s%2529%26jtitle%3DCell.%2520Mol.%2520Neurobiol.%26date%3D2013%26volume%3D33%26spage%3D47%26epage%3D58%26doi%3D10.1007%2Fs10571-012-9869-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanamoto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okubo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, K.</span></span> <span> </span><span class="NLM_article-title">Multiple P2Y subtypes in spinal microglia are involved in neuropathic pain after peripheral nerve injury</span>. <i>Glia</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1539</span>, <span class="refDoi"> DOI: 10.1002/glia.22373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2Fglia.22373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=22736439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A280%3ADC%252BC38jltFenug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=1529-1539&author=K.+Kobayashiauthor=H.+Yamanakaauthor=F.+Yanamotoauthor=M.+Okuboauthor=K.+Noguchi&title=Multiple+P2Y+subtypes+in+spinal+microglia+are+involved+in+neuropathic+pain+after+peripheral+nerve+injury&doi=10.1002%2Fglia.22373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple P2Y subtypes in spinal microglia are involved in neuropathic pain after peripheral nerve injury</span></div><div class="casAuthors">Kobayashi Kimiko; Yamanaka Hiroki; Yanamoto Fujio; Okubo Masamichi; Noguchi Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Glia</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1529-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A prominent signaling pathway in the development of neuropathic pain involves ATP acting on microglial purinergic receptors.  Among the P2Y metabotropic receptors, we reported before that the P2Y12 receptor is upregulated in microglia following nerve injury and involved in the phosphorylation of p38 MAPK, and in the development of pain behavior.  In this study, we examined the expression of P2Y6, P2Y13, and P2Y14 receptors in the spinal cord and whether these receptors are involved in the pathogenesis of neuropathic pain following peripheral nerve injury.  We found that spared nerve injury induced a dramatic increase of not only P2Y12, but also P2Y6, 13, and 14 receptor mRNA expression in spinal microglia.  The increase continued for at least 2 weeks after injury.  To determine whether p38 MAPK can induce the expression of P2Y receptors, we administered intrathecally the p38 MAPK inhibitor SB203580 and found that it significantly suppressed P2Y6, P2Y13, and P2Y14 but not P2Y12 mRNAs.  Intrathecal injection of the specific P2Y6 antagonist MRS2578, specific P2Y13 antagonist MRS2211 or P2Y14 antisense LNA, attenuated mechanical pain hypersensitivity.  The mixture of three antagonists for P2Y6, 12, and 13 showed a longer suppressive effect on pain behavior than the individual treatments.  Our data demonstrate that ATP and other nucleotides may stimulate activated microglia with the upregulation of P2Y6, P2Y12, P2Y13, and P2Y14 receptors following nerve injury and these receptors are involved in the development of neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGSxmgq31PIrHHuJhbyicZfW6udTcc2eY-FRSiiOYczrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jltFenug%253D%253D&md5=3a19972be03492267641a89519b8e5bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fglia.22373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fglia.22373%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DYamanaka%26aufirst%3DH.%26aulast%3DYanamoto%26aufirst%3DF.%26aulast%3DOkubo%26aufirst%3DM.%26aulast%3DNoguchi%26aufirst%3DK.%26atitle%3DMultiple%2520P2Y%2520subtypes%2520in%2520spinal%2520microglia%2520are%2520involved%2520in%2520neuropathic%2520pain%2520after%2520peripheral%2520nerve%2520injury%26jtitle%3DGlia%26date%3D2012%26volume%3D60%26spage%3D1529%26epage%3D1539%26doi%3D10.1002%2Fglia.22373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sesma, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzer, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livraghi-Butrico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexis, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarowski, E. R.</span></span> <span> </span><span class="NLM_article-title">UDP-glucose promotes neutrophil recruitment in the lung</span>. <i>Purinergic Signalling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1007/s11302-016-9524-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs11302-016-9524-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=27421735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyntrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=627-635&author=J.+I.+Sesmaauthor=C.+D.+Weitzerauthor=A.+Livraghi-Butricoauthor=H.+Dangauthor=S.+Donaldsonauthor=N.+E.+Alexisauthor=K.+A.+Jacobsonauthor=T.+K.+Hardenauthor=E.+R.+Lazarowski&title=UDP-glucose+promotes+neutrophil+recruitment+in+the+lung&doi=10.1007%2Fs11302-016-9524-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-glucose promotes neutrophil recruitment in the lung</span></div><div class="casAuthors">Sesma, Juliana I.; Weitzer, Clarissa D.; Livraghi-Butrico, Alessandra; Dang, Hong; Donaldson, Scott; Alexis, Neil E.; Jacobson, Kenneth A.; Harden, T. Kendall; Lazarowski, Eduardo R.</div><div class="citationInfo"><span class="NLM_cas:title">Purinergic Signalling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">627-635</span>CODEN:
                <span class="NLM_cas:coden">PSUIA9</span>;
        ISSN:<span class="NLM_cas:issn">1573-9538</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In addn. to their role in glycosylation reactions, UDP-sugars are released from cells and activate widely distributed cell surface P2Y14 receptors (P2Y14R).  However, the physiol./pathophysiol. consequences of UDP-sugar release are incompletely defined.  Here, we report that UDP-glucose levels are abnormally elevated in lung secretions from patients with cystic fibrosis (CF) as well as in a mouse model of CF-like disease, the βENaC transgenic (Tg) mouse.  Instillation of UDP-glucose into wild-type mouse tracheas resulted in enhanced neutrophil lung recruitment, and this effect was nearly abolished when UDP-glucose was co-instilled with the P2Y14R antagonist PPTN [4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl-2-naphthoic acid].  Importantly, administration of PPTN to βENaC-Tg mice reduced neutrophil lung inflammation.  These results suggest that UDP-glucose released into the airways acts as a local mediator of neutrophil inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcO6QglBl4zrVg90H21EOLACvtfcHk0ljr9qQHXtoVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyntrrI&md5=0d123a5581a97af280656d05a47166e5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs11302-016-9524-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11302-016-9524-5%26sid%3Dliteratum%253Aachs%26aulast%3DSesma%26aufirst%3DJ.%2BI.%26aulast%3DWeitzer%26aufirst%3DC.%2BD.%26aulast%3DLivraghi-Butrico%26aufirst%3DA.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DDonaldson%26aufirst%3DS.%26aulast%3DAlexis%26aufirst%3DN.%2BE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DLazarowski%26aufirst%3DE.%2BR.%26atitle%3DUDP-glucose%2520promotes%2520neutrophil%2520recruitment%2520in%2520the%2520lung%26jtitle%3DPurinergic%2520Signalling%26date%3D2016%26volume%3D12%26spage%3D627%26epage%3D635%26doi%3D10.1007%2Fs11302-016-9524-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peikert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ünal, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willecke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinöhl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Zur Mühlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirlik, A.</span></span> <span> </span><span class="NLM_article-title">Extracellular ATP induces vascular inflammation and atherosclerosis via purinergic receptor Y<sub>2</sub> in mice</span>. <i>Arterioscler., Thromb., Vasc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1586</span>, <span class="refDoi"> DOI: 10.1161/ATVBAHA.115.307397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1161%2FATVBAHA.115.307397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=27339459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ekt7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=1577-1586&author=P.+Stachonauthor=S.+Geisauthor=A.+Peikertauthor=A.+Heidenreichauthor=N.+A.+Michelauthor=F.+%C3%9Cnalauthor=N.+Hoppeauthor=B.+Dufnerauthor=L.+Schulteauthor=T.+Marchiniauthor=S.+Cickoauthor=K.+Ayataauthor=A.+Zechauthor=D.+Wolfauthor=I.+Hilgendorfauthor=F.+Willeckeauthor=J.+Rein%C3%B6hlauthor=C.+von+Zur+M%C3%BChlenauthor=C.+Bodeauthor=M.+Idzkoauthor=A.+Zirlik&title=Extracellular+ATP+induces+vascular+inflammation+and+atherosclerosis+via+purinergic+receptor+Y2+in+mice&doi=10.1161%2FATVBAHA.115.307397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular ATP Induces Vascular Inflammation and Atherosclerosis via Purinergic Receptor Y2 in Mice</span></div><div class="casAuthors">Stachon, Peter; Geis, Serjosha; Peikert, Alexander; Heidenreich, Adrian; Michel, Nathaly Anto; Uenal, Fatih; Hoppe, Natalie; Dufner, Bianca; Schulte, Lisa; Marchini, Timoteo; Cicko, Sanja; Ayata, Korcan; Zech, Andreas; Wolf, Dennis; Hilgendorf, Ingo; Willecke, Florian; Reinoehl, Jochen; von zur Muehlen, Constantin; Bode, Christoph; Idzko, Marco; Zirlik, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1577-1586</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">OBJECTIVE: A solid body of evidence supports a role of extracellular ATP and its P2 receptors in innate and adaptive immunity.  It promotes inflammation as a danger signal in various chronic inflammatory diseases.  Thus, we hypothesize contribution of extracellular ATP and its receptor P2Y2 in vascular inflammation and atherosclerosis.  APPROACH AND RESULTS-: Extracellular ATP induced leukocyte rolling, adhesion, and migration in vivo as assessed by intravital microscopy and in sterile peritonitis.  To test the role of extracellular ATP in atherosclerosis, ATP or saline as control was injected i.p. 3× a week in low-d. lipoprotein receptor mice consuming high cholesterol diet.  Atherosclerosis significantly increased after 16 wk in ATP-treated mice (n=13; control group, 0.26 mm2; ATP group, 0.33 mm2; P=0.01).  To gain into the role of ATP-receptor P2Y2 in ATP-induced leukocyte recruitment, ATP was administered systemically in P2Y2-deficient or P2Y2-competent mice.  In P2Y2-deficient mice, the ATP-induced leukocyte adhesion was significantly reduced as assessed by intravital microscopy.  P2Y2 expression in atherosclerosis was measured by real-time polymerase chain reaction and immunohistochem. and demonstrates an increased expression mainly caused by influx of P2Y2-expressing macrophages.  To investigate the functional role of P2Y2 in atherogenesis, P2Y2-deficient low-d. lipoprotein receptor mice consumed high cholesterol diet.  After 16 wk, P2Y2-deficient mice showed significantly reduced atherosclerotic lesions with decreased macrophages compared with P2Y2-competent mice (n=11; aortic arch: control group, 0.25 mm; P2Y2-deficient, 0.14 mm2; P=0.04).  Mechanistically, atherosclerotic lesions from P2Y2-deficient mice expressed less vascular cell adhesion mol.-1 and intercellular adhesion mol.-1 RNA.  CONCLUSIONS: We show that extracellular ATP induces vascular inflammation and atherosclerosis via activation of P2Y2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs1FzbdbOyOrVg90H21EOLACvtfcHk0ljr9qQHXtoVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ekt7zJ&md5=e1bdf319afb42f45a8348d91bdf1c08a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1161%2FATVBAHA.115.307397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FATVBAHA.115.307397%26sid%3Dliteratum%253Aachs%26aulast%3DStachon%26aufirst%3DP.%26aulast%3DGeis%26aufirst%3DS.%26aulast%3DPeikert%26aufirst%3DA.%26aulast%3DHeidenreich%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DN.%2BA.%26aulast%3D%25C3%259Cnal%26aufirst%3DF.%26aulast%3DHoppe%26aufirst%3DN.%26aulast%3DDufner%26aufirst%3DB.%26aulast%3DSchulte%26aufirst%3DL.%26aulast%3DMarchini%26aufirst%3DT.%26aulast%3DCicko%26aufirst%3DS.%26aulast%3DAyata%26aufirst%3DK.%26aulast%3DZech%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DD.%26aulast%3DHilgendorf%26aufirst%3DI.%26aulast%3DWillecke%26aufirst%3DF.%26aulast%3DRein%25C3%25B6hl%26aufirst%3DJ.%26aulast%3Dvon%2BZur%2BM%25C3%25BChlen%26aufirst%3DC.%26aulast%3DBode%26aufirst%3DC.%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DZirlik%26aufirst%3DA.%26atitle%3DExtracellular%2520ATP%2520induces%2520vascular%2520inflammation%2520and%2520atherosclerosis%2520via%2520purinergic%2520receptor%2520Y2%2520in%2520mice%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2016%26volume%3D36%26spage%3D1577%26epage%3D1586%26doi%3D10.1161%2FATVBAHA.115.307397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eltzschig, H. K.</span></span> <span> </span><span class="NLM_article-title">Nucleotide signalling during inflammation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nature13085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=24828189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=310-317&author=M.+Idzkoauthor=D.+Ferrariauthor=H.+K.+Eltzschig&title=Nucleotide+signalling+during+inflammation&doi=10.1038%2Fnature13085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide signalling during inflammation</span></div><div class="casAuthors">Idzko, Marco; Ferrari, Davide; Eltzschig, Holger K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7500</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inflammatory conditions are assocd. with the extracellular release of nucleotides, particularly ATP.  In the extracellular compartment, ATP predominantly functions as a signalling mol. through the activation of purinergic P2 receptors.  Metabotropic P2Y receptors are G-protein-coupled, whereas ionotropic P2X receptors are ATP-gated ion channels.  Here we discuss how signalling events through P2 receptors alter the outcomes of inflammatory or infectious diseases.  Recent studies implicate a role for P2X/P2Y signalling in mounting appropriate inflammatory responses crit. for host defense against invading pathogens or tumors.  Conversely, P2X/P2Y signalling can promote chronic inflammation during ischemia and reperfusion injury, inflammatory bowel disease or acute and chronic diseases of the lungs.  Although nucleotide signalling has been used clin. in patients before, research indicates an expanding field of opportunities for specifically targeting individual P2 receptors for the treatment of inflammatory or infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQJVgswMEhUrVg90H21EOLACvtfcHk0lgFZmsYHhul_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyqtLc%253D&md5=676480f2bfbe98592d603f9064d011b7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13085%26sid%3Dliteratum%253Aachs%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DFerrari%26aufirst%3DD.%26aulast%3DEltzschig%26aufirst%3DH.%2BK.%26atitle%3DNucleotide%2520signalling%2520during%2520inflammation%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D310%26epage%3D317%26doi%3D10.1038%2Fnature13085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Structure of the human P2Y<sub>12</sub> receptor in complex with an antithrombotic drug</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1038/nature13083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=24670650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlyisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=115-118&author=K.+Zhangauthor=J.+Zhangauthor=Z.+G.+Gaoauthor=D.+Zhangauthor=L.+Zhuauthor=G.+W.+Hanauthor=S.+M.+Mossauthor=S.+Paolettaauthor=E.+Kiselevauthor=W.+Luauthor=G.+Fenaltiauthor=W.+Zhangauthor=C.+E.+M%C3%BCllerauthor=H.+Yangauthor=V.+Cherezovauthor=V.+Katritchauthor=G.+W.+Hanauthor=K.+A.+Jacobsonauthor=R.+C.+Stevensauthor=B.+Wuauthor=Q.+Zhao&title=Structure+of+the+human+P2Y12+receptor+in+complex+with+an+antithrombotic+drug&doi=10.1038%2Fnature13083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human P2Y12 receptor in complex with an antithrombotic drug</span></div><div class="casAuthors">Zhang, Kaihua; Zhang, Jin; Gao, Zhan-Guo; Zhang, Dandan; Zhu, Lan; Han, Gye Won; Moss, Steven M.; Paoletta, Silvia; Kiselev, Evgeny; Lu, Weizhen; Fenalti, Gustavo; Zhang, Wenru; Mueller, Christa E.; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Katritch, Vsevolod; Jacobson, Kenneth A.; Stevens, Raymond C.; Wu, Beili; Zhao, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7498</span>),
    <span class="NLM_cas:pages">115-118</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P2Y receptors (P2YRs), a family of purinergic G-protein-coupled receptors (GPCRs), are activated by extracellular nucleotides.  There are a total of eight distinct functional P2YRs expressed in human, which are subdivided into P2Y1-like receptors and P2Y12-like receptors.  Their ligands are generally charged mols. with relatively low bioavailability and stability in vivo, which limits the authors' understanding of this receptor family.  P2Y12R regulates platelet activation and thrombus formation, and several antithrombotic drugs targeting P2Y12R-including the prodrugs clopidogrel (Plavix) and prasugrel (Effient) that are metabolized and bind covalently, and the nucleoside analog ticagrelor (Brilinta) that acts directly on the receptor-have been approved for the prevention of stroke and myocardial infarction.  However, limitations of these drugs (for example, a very long half-life of clopidogrel action and a characteristic adverse effect profile of ticagrelor) suggest that there is an unfulfilled medical need for developing a new generation of P2Y12R inhibitors.  Here the authors report the 2.6 Å resoln. crystal structure of human P2Y12R in complex with a non-nucleotide reversible antagonist, AZD1283.  The structure reveals a distinct straight conformation of helix V, which sets P2Y12R apart from all other known class A GPCR structures.  With AZD1283 bound, the highly conserved disulfide bridge in GPCRs between helix III and extracellular loop 2 is not obsd. and appears to be dynamic.  Along with the details of the AZD1283-binding site, anal. of the extracellular interface reveals an adjacent ligand-binding region and suggests that both pockets could be required for dinucleotide binding.  The structure provides essential insights for the development of improved P2Y12R ligands and allosteric modulators as drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOY06VN4D76bVg90H21EOLACvtfcHk0lgFZmsYHhul_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlyisrg%253D&md5=539bb91325c547737bc98baf59f3ff26</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature13083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13083%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DStructure%2520of%2520the%2520human%2520P2Y12%2520receptor%2520in%2520complex%2520with%2520an%2520antithrombotic%2520drug%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D115%26epage%3D118%26doi%3D10.1038%2Fnature13083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Agonist-bound structure of the human P2Y<sub>12</sub>R receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>509</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/nature13288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature13288" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=119-122&author=K.+Zhangauthor=J.+Zhangauthor=Z.+G.+Gaoauthor=S.+Paolettaauthor=D.+Zhangauthor=G.+W.+Hanauthor=T.+Liauthor=L.+Maauthor=W.+Zhangauthor=C.+E.+M%C3%BCllerauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=V.+Katritchauthor=K.+A.+Jacobsonauthor=R.+C.+Stevensauthor=B.+Wuauthor=Q.+Zhao&title=Agonist-bound+structure+of+the+human+P2Y12R+receptor&doi=10.1038%2Fnature13288"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature13288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13288%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DAgonist-bound%2520structure%2520of%2520the%2520human%2520P2Y12R%2520receptor%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D119%26epage%3D122%26doi%3D10.1038%2Fnature13288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzer, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Exploring a 2-naphthoic acid template for the structure-based design of P2Y<sub>14</sub> receptor antagonist molecular probes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2833</span>– <span class="NLM_lpage">2842</span>, <span class="refDoi"> DOI: 10.1021/cb500614p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500614p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslajs73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2833-2842&author=E.+Kiselevauthor=M.+Barrettauthor=V.+Katritchauthor=S.+Paolettaauthor=C.+D.+Weitzerauthor=E.+Hammesauthor=A.+L.+Yinauthor=Q.+Zhaoauthor=R.+C.+Stevensauthor=T.+K.+Hardenauthor=K.+A.+Jacobson&title=Exploring+a+2-naphthoic+acid+template+for+the+structure-based+design+of+P2Y14+receptor+antagonist+molecular+probes&doi=10.1021%2Fcb500614p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring a 2-Naphthoic Acid Template for the Structure-Based Design of P2Y14 Receptor Antagonist Molecular Probes</span></div><div class="casAuthors">Kiselev, Evgeny; Barrett, Matthew O.; Katritch, Vsevolod; Paoletta, Silvia; Weitzer, Clarissa D.; Brown, Kyle A.; Hammes, Eva; Yin, Andrew L.; Zhao, Qiang; Stevens, Raymond C.; Harden, T. Kendall; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2833-2842</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The P2Y14 receptor (P2Y14R), one of eight P2Y G protein-coupled receptors (GPCR), is involved in inflammatory, endocrine, and hypoxic processes and is an attractive pharmaceutical target.  The goal of this research is to develop high-affinity P2Y14R fluorescent probes based on the potent and highly selective antagonist 4-(4-(piperidin-4-yl)-phenyl)-7-(4-(trifluoromethyl)-phenyl)-2-naphthoic acid (6, PPTN).  A model of hP2Y14R based on recent hP2Y12R X-ray structures together with simulated antagonist docking suggested that the piperidine ring is suitable for fluorophore conjugation while preserving affinity.  Chain-elongated alkynyl or amino derivs. of 6 for click or amide coupling were synthesized, and their antagonist activities were measured in hP2Y14R-expressing CHO cells.  Moreover, a new Alexa Fluor 488 (AF488) contg. deriv. 30 (MRS4174, Ki = 80 pM) exhibited exceptionally high affinity, as compared to 13 nM for the alkyne precursor 22.  A flow cytometry assay employing 30 as a fluorescent probe was used to quantify specific binding to P2Y14R.  Known P2Y receptor ligands inhibited binding of 30 with properties consistent with their previously established receptor selectivities and affinities.  These results illustrate that potency in this series of 2-naphthoic acid derivs. can be preserved by chain functionalization, leading to highly potent fluorescent mol. probes for P2Y14R.  Such conjugates will be useful tools in expanding the SAR of this receptor, which still lacks chem. diversity in its collective ligands.  This approach demonstrates the predictive power of GPCR homol. modeling and the relevance of newly detd. X-ray structures to GPCR medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwJ36T_9HjUrVg90H21EOLACvtfcHk0lgy3Aith-FjQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslajs73K&md5=9dca289cc6fcf9c1418b49f811e691c0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fcb500614p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500614p%26sid%3Dliteratum%253Aachs%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DM.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DWeitzer%26aufirst%3DC.%2BD.%26aulast%3DHammes%26aufirst%3DE.%26aulast%3DYin%26aufirst%3DA.%2BL.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DExploring%2520a%25202-naphthoic%2520acid%2520template%2520for%2520the%2520structure-based%2520design%2520of%2520P2Y14%2520receptor%2520antagonist%2520molecular%2520probes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2833%26epage%3D2842%26doi%3D10.1021%2Fcb500614p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of the human P2Y<sub>14</sub> receptor: A template for structure-based design of selective agonist ligands</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4056</span>– <span class="NLM_lpage">4064</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmc.2015.03.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4056-4064&author=K.+Trujilloauthor=S.+Paolettaauthor=E.+Kiselevauthor=K.+A.+Jacobson&title=Molecular+modeling+of+the+human+P2Y14+receptor%3A+A+template+for+structure-based+design+of+selective+agonist+ligands&doi=10.1016%2Fj.bmc.2015.03.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DK.%26aulast%3DPaoletta%26aufirst%3DS.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DMolecular%2520modeling%2520of%2520the%2520human%2520P2Y14%2520receptor%253A%2520A%2520template%2520for%2520structure-based%2520design%2520of%2520selective%2520agonist%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4056%26epage%3D4064%26doi%3D10.1016%2Fj.bmc.2015.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschênes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareau, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hénault, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyjazie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kargman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavallée, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levesque, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulrooney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thérien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tranmer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, W. C.</span></span> <span> </span><span class="NLM_article-title">The identification of 4,7-disubstituted naphthoic acid derivatives as UDP-competitive antagonists of P2Y<sub>14</sub></span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2836</span>– <span class="NLM_lpage">2839</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.03.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmcl.2011.03.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=21507640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVCmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=2836-2839&author=J.+Y.+Gauthierauthor=M.+Belleyauthor=D.+Desch%C3%AAnesauthor=J.-F.+Fournierauthor=S.+Gagn%C3%A9author=Y.+Gareauauthor=M.+Hamelauthor=M.+H%C3%A9naultauthor=H.+Hyjazieauthor=S.+Kargmanauthor=G.+Lavall%C3%A9eauthor=J.+F.+Levesqueauthor=L.+Liauthor=Y.+Mamaneauthor=J.+Manciniauthor=N.+Morinauthor=E.+Mulrooneyauthor=J.+Robichaudauthor=M.+Th%C3%A9rienauthor=G.+Tranmerauthor=Z.+Wangauthor=J.+Wuauthor=W.+C.+Black&title=The+identification+of+4%2C7-disubstituted+naphthoic+acid+derivatives+as+UDP-competitive+antagonists+of+P2Y14&doi=10.1016%2Fj.bmcl.2011.03.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The identification of 4,7-disubstituted naphthoic acid derivatives as UDP-competitive antagonists of P2Y14</span></div><div class="casAuthors">Gauthier, Jacques Yves; Belley, Michel; Deschenes, Denis; Fournier, Jean-Francois; Gagne, Sebastien; Gareau, Yves; Hamel, Martine; Henault, Martin; Hyjazie, Huda; Kargman, Stacia; Lavallee, Genevieve; Levesque, Jean-Francois; Li, Lianhai; Mamane, Yael; Mancini, Joseph; Morin, Nicolas; Mulrooney, Erin; Robichaud, Joel; Therien, Michel; Tranmer, Geoffrey; Wang, Zhaoyin; Wu, Jin; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2836-2839</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A weak, UDP-competitive antagonist of the pyrimidinergic receptor P2RY14 with a naphthoic acid core was identified through high-throughput screening.  Optimization provided compds. with improved potency but poor pharmacokinetics.  Acylglucuronidation was detd. to be the major route of metab.  Increasing the electron-withdrawing nature of the substituents markedly reduced glucuronidation and improved the pharmacokinetic profile.  Addnl. optimization led to the identification of compd. 38 which is an 8 nM UDP-competitive antagonist of P2Y14 with a good pharmacokinetic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjL3gtO4Wk4LVg90H21EOLACvtfcHk0lgy3Aith-FjQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVCmtrg%253D&md5=7a5b0a1ae77102f81e3ae8eb4afadccf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.03.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.03.081%26sid%3Dliteratum%253Aachs%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DBelley%26aufirst%3DM.%26aulast%3DDesch%25C3%25AAnes%26aufirst%3DD.%26aulast%3DFournier%26aufirst%3DJ.-F.%26aulast%3DGagn%25C3%25A9%26aufirst%3DS.%26aulast%3DGareau%26aufirst%3DY.%26aulast%3DHamel%26aufirst%3DM.%26aulast%3DH%25C3%25A9nault%26aufirst%3DM.%26aulast%3DHyjazie%26aufirst%3DH.%26aulast%3DKargman%26aufirst%3DS.%26aulast%3DLavall%25C3%25A9e%26aufirst%3DG.%26aulast%3DLevesque%26aufirst%3DJ.%2BF.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMamane%26aufirst%3DY.%26aulast%3DMancini%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DN.%26aulast%3DMulrooney%26aufirst%3DE.%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DTh%25C3%25A9rien%26aufirst%3DM.%26aulast%3DTranmer%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DThe%2520identification%2520of%25204%252C7-disubstituted%2520naphthoic%2520acid%2520derivatives%2520as%2520UDP-competitive%2520antagonists%2520of%2520P2Y14%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D2836%26epage%3D2839%26doi%3D10.1016%2Fj.bmcl.2011.03.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciancetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martiriggiano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mtchedlidze, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birdwell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 3-(4-aryl-1H-1,2,3-triazol-1-yl)-biphenyl derivatives as P2Y<sub>14</sub> receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6149</span>– <span class="NLM_lpage">6168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVWkurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6149-6168&author=A.+Junkerauthor=R.+Balasubramanianauthor=A.+Ciancettaauthor=E.+Uliassiauthor=E.+Kiselevauthor=C.+Martiriggianoauthor=K.+Trujilloauthor=G.+Mtchedlidzeauthor=L.+Birdwellauthor=K.+A.+Brownauthor=T.+K.+Hardenauthor=K.+A.+Jacobson&title=Structure-based+design+of+3-%284-aryl-1H-1%2C2%2C3-triazol-1-yl%29-biphenyl+derivatives+as+P2Y14+receptor+antagonists&doi=10.1021%2Facs.jmedchem.6b00044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 3-(4-Aryl-1H-1,2,3-triazol-1-yl)-Biphenyl Derivatives as P2Y14 Receptor Antagonists</span></div><div class="casAuthors">Junker, Anna; Balasubramanian, Ramachandran; Ciancetta, Antonella; Uliassi, Elisa; Kiselev, Evgeny; Martiriggiano, Chiara; Trujillo, Kevin; Mtchedlidze, Giorgi; Birdwell, Leah; Brown, Kyle A.; Harden, T. Kendall; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6149-6168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">UDP and UDP-glucose activate the P2Y14 receptor (P2Y14R) to modulate processes related to inflammation, diabetes, and asthma.  A computational pipeline suggested alternatives to naphthalene of a previously reported P2Y14R antagonist (3, PPTN) using docking and mol. dynamics simulations on a hP2Y14R homol. model based on P2Y12R structures.  By reevaluating the binding of 3 to P2Y14R computationally, two alternatives, i.e., alkynyl and triazolyl derivs., were identified.  Improved synthesis of fluorescent antagonist 4 enabled affinity quantification (IC50s, nM) using flow cytometry of P2Y14R-expressing CHO cells. p-F3C-phenyl-triazole 65 (32) was more potent than a corresponding alkyne 11.  Thus, addnl. triazolyl derivs. were prepd., as guided by docking simulations, with nonpolar aryl substituents favored.  Although triazoles were less potent than 3 (6), simpler synthesis facilitated further structural optimization.  Addnl., relative P2Y14R affinities agreed with predicted binding of alkynyl and triazole analogs.  These triazoles, designed through a structure-based approach, can be assessed in disease models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5CKuzhmXmxrVg90H21EOLACvtfcHk0lig8sYLwhVZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVWkurzE&md5=02287b12815a85d188e5f224735f302a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00044%26sid%3Dliteratum%253Aachs%26aulast%3DJunker%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DR.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DMartiriggiano%26aufirst%3DC.%26aulast%3DTrujillo%26aufirst%3DK.%26aulast%3DMtchedlidze%26aufirst%3DG.%26aulast%3DBirdwell%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DK.%2BA.%26aulast%3DHarden%26aufirst%3DT.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520design%2520of%25203-%25284-aryl-1H-1%252C2%252C3-triazol-1-yl%2529-biphenyl%2520derivatives%2520as%2520P2Y14%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6149%26epage%3D6168%26doi%3D10.1021%2Facs.jmedchem.6b00044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lig8sYLwhVZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">Integrated methods for the construction of three dimensional models and computational probing of structure function relations in G protein-coupled receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/S1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=Integrated+methods+for+the+construction+of+three+dimensional+models+and+computational+probing+of+structure+function+relations+in+G+protein-coupled+receptors&doi=10.1016%2FS1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0lig8sYLwhVZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DIntegrated%2520methods%2520for%2520the%2520construction%2520of%2520three%2520dimensional%2520models%2520and%2520computational%2520probing%2520of%2520structure%2520function%2520relations%2520in%2520G%2520protein-coupled%2520receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2FS1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span> <span> </span><span class="NLM_article-title">Acidic and basic drugs in medicinal chemistry: A perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9701</span>– <span class="NLM_lpage">9717</span>, <span class="refDoi"> DOI: 10.1021/jm501000a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501000a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOnt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9701-9717&author=P.+S.+Charifsonauthor=W.+P.+Walters&title=Acidic+and+basic+drugs+in+medicinal+chemistry%3A+A+perspective&doi=10.1021%2Fjm501000a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic and Basic Drugs in Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Charifson, Paul S.; Walters, W. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9701-9717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The acid/base properties of a mol. are among the most fundamental for drug action.  However, they are often overlooked in a prospective design manner unless it has been established that a certain ionization state (e.g., quaternary base or presence of a carboxylic acid) appears to be required for activity.  In medicinal chem. optimization programs it is relatively common to attenuate basicity to circumvent undesired effects such as lack of biol. selectivity or safety risks such as hERG or phospholipidosis.  However, teams may not prospectively explore a range of carefully chosen compd. pKa values as part of an overall chem. strategy or design hypothesis.  This review summarizes the potential advantages and disadvantages of both acidic and basic drugs and provides some new analyses based on recently available public data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDorb0LfJ8PrVg90H21EOLACvtfcHk0lhDju_dvNwItQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOnt7jN&md5=51a3cb91be42ddd1a784afe737c64c60</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm501000a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501000a%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26atitle%3DAcidic%2520and%2520basic%2520drugs%2520in%2520medicinal%2520chemistry%253A%2520A%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9701%26epage%3D9717%26doi%3D10.1021%2Fjm501000a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lassalas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasfargeas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayendran, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span> <span> </span><span class="NLM_article-title">Structure property relationships of carboxylic acid isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3203</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3183-3203&author=P.+Lassalasauthor=B.+Gayauthor=C.+Lasfargeasauthor=M.+J.+Jamesauthor=V.+Tranauthor=K.+G.+Vijayendranauthor=K.+R.+Brundenauthor=M.+C.+Kozlowskiauthor=C.+J.+Thomasauthor=A.+B.+Smithauthor=D.+M.+Hurynauthor=C.+Ballatore&title=Structure+property+relationships+of+carboxylic+acid+isosteres&doi=10.1021%2Facs.jmedchem.5b01963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Property Relationships of Carboxylic Acid Isosteres</span></div><div class="casAuthors">Lassalas, Pierrik; Gay, Bryant; Lasfargeas, Caroline; James, Michael J.; Tran, Van; Vijayendran, Krishna G.; Brunden, Kurt R.; Kozlowski, Marisa C.; Thomas, Craig J.; Smith, Amos B.; Huryn, Donna M.; Ballatore, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3183-3203</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The replacement of a carboxylic acid with a surrogate structure, or (bio)-isostere, is a classical strategy in medicinal chem.  The general underlying principle is that by maintaining the features of the carboxylic acid crit. for biol. activity, but appropriately modifying the physicochem. properties, improved analogs may result.  In this context, a systematic assessment of the physicochem. properties of carboxylic acid isosteres would be desirable to enable more informed decisions of potential replacements to be used for analog design.  Herein we report the structure-property relationships (SPR) of 35 phenylpropionic acid derivs., in which the carboxylic acid moiety is replaced with a series of known isosteres.  The data set generated provides an assessment of the relative impact on the physicochem. properties that these replacements may have compared to the carboxylic acid analog.  As such, this study presents a framework for how to rationally apply isosteric replacements of the carboxylic acid functional group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIF98i2ITNrLVg90H21EOLACvtfcHk0lhDju_dvNwItQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFartbY%253D&md5=34490fc72fdb321d8c9751bef1bf79c5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01963%26sid%3Dliteratum%253Aachs%26aulast%3DLassalas%26aufirst%3DP.%26aulast%3DGay%26aufirst%3DB.%26aulast%3DLasfargeas%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DVijayendran%26aufirst%3DK.%2BG.%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DKozlowski%26aufirst%3DM.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26aulast%3DBallatore%26aufirst%3DC.%26atitle%3DStructure%2520property%2520relationships%2520of%2520carboxylic%2520acid%2520isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3183%26epage%3D3203%26doi%3D10.1021%2Facs.jmedchem.5b01963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhDju_dvNwItQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span> <span> </span><span class="NLM_article-title">Drug-like antagonists of P2Y receptors—from lead identification to drug development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9981</span>– <span class="NLM_lpage">10005</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9981-10005&author=S.+Conroyauthor=N.+Kindonauthor=B.+Kellamauthor=M.+J.+Stocks&title=Drug-like+antagonists+of+P2Y+receptors%E2%80%94from+lead+identification+to+drug+development&doi=10.1021%2Facs.jmedchem.5b01972"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01972%26sid%3Dliteratum%253Aachs%26aulast%3DConroy%26aufirst%3DS.%26aulast%3DKindon%26aufirst%3DN.%26aulast%3DKellam%26aufirst%3DB.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26atitle%3DDrug-like%2520antagonists%2520of%2520P2Y%2520receptors%25E2%2580%2594from%2520lead%2520identification%2520to%2520drug%2520development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9981%26epage%3D10005%26doi%3D10.1021%2Facs.jmedchem.5b01972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hénault, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kargman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levesque, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulrooney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, W. C.</span></span> <span> </span><span class="NLM_article-title">Applying the pro-drug approach to afford highly bioavailable antagonists of P2Y<sub>14</sub></span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4366</span>– <span class="NLM_lpage">4368</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.12.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.bmcl.2010.12.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=21689930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFWrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4366-4368&author=J.+Robichaudauthor=J.+F.+Fournierauthor=S.+Gagn%C3%A9author=J.+Y.+Gauthierauthor=M.+Hamelauthor=Y.+Hanauthor=M.+H%C3%A9naultauthor=S.+Kargmanauthor=J.-F.+Levesqueauthor=Y.+Mamaneauthor=J.+Manciniauthor=N.+Morinauthor=E.+Mulrooneyauthor=J.+Wuauthor=W.+C.+Black&title=Applying+the+pro-drug+approach+to+afford+highly+bioavailable+antagonists+of+P2Y14&doi=10.1016%2Fj.bmcl.2010.12.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Applying the pro-drug approach to afford highly bioavailable antagonists of P2Y14</span></div><div class="casAuthors">Robichaud, Joel; Fournier, Jean-Francois; Gagne, Sebastien; Gauthier, Jacques Yves; Hamel, Martine; Han, Yongxin; Henault, Martin; Kargman, Stacia; Levesque, Jean-Francois; Mamane, Yael; Mancini, Joseph; Morin, Nicolas; Mulrooney, Erin; Wu, Jin; Black, W. Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4366-4368</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Our series of competitive antagonists against the G-protein coupled receptor P2Y14 were found to be highly shifted in the presence of serum (>99% protein bound).  A binding assay using 2% human serum albumin (HSA) was developed to guide further SAR studies and led to the identification of the zwitterion 2 (I), which is substantially less shifted (18-fold) than our previous lead compd. 1 (323-fold).  However, as the bioavailability of 2 was low, a library of ester pro-drugs was prepd. (7a-7j) and assessed in vitro.  The most interesting candidates were then profiled in vivo and led to the identification of the pro-drug 7j, which possesses a substantially improved pharmacokinetic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwKD0Yxb4pOLVg90H21EOLACvtfcHk0livbfrWomukVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFWrtLw%253D&md5=d993d60779b583f0ca7ff466387a8dbc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.12.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.12.113%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaud%26aufirst%3DJ.%26aulast%3DFournier%26aufirst%3DJ.%2BF.%26aulast%3DGagn%25C3%25A9%26aufirst%3DS.%26aulast%3DGauthier%26aufirst%3DJ.%2BY.%26aulast%3DHamel%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DH%25C3%25A9nault%26aufirst%3DM.%26aulast%3DKargman%26aufirst%3DS.%26aulast%3DLevesque%26aufirst%3DJ.-F.%26aulast%3DMamane%26aufirst%3DY.%26aulast%3DMancini%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DN.%26aulast%3DMulrooney%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBlack%26aufirst%3DW.%2BC.%26atitle%3DApplying%2520the%2520pro-drug%2520approach%2520to%2520afford%2520highly%2520bioavailable%2520antagonists%2520of%2520P2Y14%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4366%26epage%3D4368%26doi%3D10.1016%2Fj.bmcl.2010.12.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0livbfrWomukVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span> <i>LigPrep</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span> <i>Glide</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noé, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">HTMD: High-throughput molecular dynamics for molecular discovery</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1845</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b00049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12hu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1845-1852&author=S.+Doerrauthor=M.+J.+Harveyauthor=F.+No%C3%A9author=G.+De+Fabritiis&title=HTMD%3A+High-throughput+molecular+dynamics+for+molecular+discovery&doi=10.1021%2Facs.jctc.6b00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">HTMD: High-Throughput Molecular Dynamics for Molecular Discovery</span></div><div class="casAuthors">Doerr, S.; Harvey, M. J.; Noe, Frank; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1845-1852</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in mol. simulations have allowed scientists to investigate slower biol. processes than ever before.  Together with these advances came an explosion of data that has transformed a traditionally computing-bound into a data-bound problem.  Here, we present HTMD, a programmable, extensible platform written in Python that aims to solve the data generation and anal. problem as well as increase reproducibility by providing a complete workspace for simulation-based discovery.  So far, HTMD includes system building for CHARMM and AMBER force fields, projection methods, clustering, mol. simulation prodn., adaptive sampling, an Amazon cloud interface, Markov state models, and visualization.  As a result, a single, short HTMD script can lead from a PDB structure to useful quantities such as relaxation time scales, equil. populations, metastable conformations, and kinetic rates.  In this paper, we focus on the adaptive sampling and Markov state modeling features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEgXtPB9K5VLVg90H21EOLACvtfcHk0livbfrWomukVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12hu7o%253D&md5=c7ce0d9a709642ad5e28e202655a9a9d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00049%26sid%3Dliteratum%253Aachs%26aulast%3DDoerr%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DM.%2BJ.%26aulast%3DNo%25C3%25A9%26aufirst%3DF.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DHTMD%253A%2520High-throughput%2520molecular%2520dynamics%2520for%2520molecular%2520discovery%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D1845%26epage%3D1852%26doi%3D10.1021%2Facs.jctc.6b00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">VMD - Visual molecular dynamics</span>. <i>J. Mol. Graphics</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD+-+Visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0liy4rNQHq4DMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%2520-%2520Visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0liy4rNQHq4DMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giupponi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">ACEMD: Accelerated molecular dynamics simulations in the microseconds timescale</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1632</span>– <span class="NLM_lpage">1639</span>, <span class="refDoi"> DOI: 10.1021/ct9000685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9000685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=1632-1639&author=M.+Harveyauthor=G.+Giupponiauthor=G.+De+Fabritiis&title=ACEMD%3A+Accelerated+molecular+dynamics+simulations+in+the+microseconds+timescale&doi=10.1021%2Fct9000685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ACEMD: Accelerating Biomolecular Dynamics in the Microsecond Time Scale</span></div><div class="casAuthors">Harvey, M. J.; Giupponi, G.; De Fabritiis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1632-1639</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The high arithmetic performance and intrinsic parallelism of recent graphical processing units (GPUs) can offer a technol. edge for mol. dynamics simulations.  ACEMD is a prodn.-class biomol. dynamics (MD) engine supporting CHARMM and AMBER force fields.  Designed specifically for GPUs it is able to achieve supercomputing scale performance of 40 ns/day for all-atom protein systems with over 23,000 atoms.  The authors provide a validation and performance evaluation of the code and run a microsecond-long trajectory for an all-atom mol. system in explicit TIP3P water on a single workstation computer equipped with just 3 GPUs.  The authors believe that microsecond time scale mol. dynamics on cost-effective hardware will have important methodol. and scientific implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryhvUDSt2mxbVg90H21EOLACvtfcHk0liy4rNQHq4DMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rsLk%253D&md5=99a7226c62aa210bada97ee61a0c254f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fct9000685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9000685%26sid%3Dliteratum%253Aachs%26aulast%3DHarvey%26aufirst%3DM.%26aulast%3DGiupponi%26aufirst%3DG.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DACEMD%253A%2520Accelerated%2520molecular%2520dynamics%2520simulations%2520in%2520the%2520microseconds%2520timescale%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D1632%26epage%3D1639%26doi%3D10.1021%2Fct9000685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Best, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Optimization of the additive CHARMM all-atom protein force field targeting improved sampling of the backbone phi, psi and side-chain chi1 and chi2 dihedral angles</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3273</span>, <span class="refDoi"> DOI: 10.1021/ct300400x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300400x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3257-3273&author=R.+B.+Bestauthor=X.+Zhuauthor=J.+Shimauthor=P.+E.+M.+Lopesauthor=J.+Mittalauthor=M.+Feigauthor=A.+D.+MacKerell&title=Optimization+of+the+additive+CHARMM+all-atom+protein+force+field+targeting+improved+sampling+of+the+backbone+phi%2C+psi+and+side-chain+chi1+and+chi2+dihedral+angles&doi=10.1021%2Fct300400x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone .vphi., ψ and Side-Chain χ1 and χ2 Dihedral Angles</span></div><div class="casAuthors">Best, Robert B.; Zhu, Xiao; Shim, Jihyun; Lopes, Pedro E. M.; Mittal, Jeetain; Feig, Michael; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3257-3273</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the quality of the current CHARMM22/CMAP additive force field for proteins has been demonstrated in a large no. of applications, limitations in the model with respect to the equil. between the sampling of helical and extended conformations in folding simulations have been noted.  To overcome this, as well as make other improvements in the model, we present a combination of refinements that should result in enhanced accuracy in simulations of proteins.  The common (non-Gly, -Pro) backbone CMAP potential has been refined against exptl. soln. NMR data for weakly structured peptides, resulting in a rebalancing of the energies of the α-helix and extended regions of the Ramachandran map, correcting the α-helical bias of CHARMM22/CMAP.  The Gly and Pro CMAPs have been refitted to more accurate quantum-mech. energy surfaces.  Side-chain torsion parameters have been optimized by fitting to backbone-dependent quantum-mech. energy surfaces, followed by addnl. empirical optimization targeting NMR scalar couplings for unfolded proteins.  A comprehensive validation of the revised force field was then performed against a collection of exptl. data: (i) comparison of simulations of eight proteins in their crystal environments with crystal structures; (ii) comparison with backbone scalar couplings for weakly structured peptides; (iii) comparison with NMR residual dipolar couplings and scalar couplings for both backbone and side-chains in folded proteins; (iv) equil. folding of mini-proteins.  The results indicate that the revised CHARMM 36 parameters represent an improved model for modeling and simulation studies of proteins, including studies of protein folding, assembly, and functionally relevant conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnwGJMAMFvLVg90H21EOLACvtfcHk0liy4rNQHq4DMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP&md5=9a48a0c5770fb1e887c3bb34d45b1354</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fct300400x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300400x%26sid%3Dliteratum%253Aachs%26aulast%3DBest%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DP.%2BE.%2BM.%26aulast%3DMittal%26aufirst%3DJ.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DOptimization%2520of%2520the%2520additive%2520CHARMM%2520all-atom%2520protein%2520force%2520field%2520targeting%2520improved%2520sampling%2520of%2520the%2520backbone%2520phi%252C%2520psi%2520and%2520side-chain%2520chi1%2520and%2520chi2%2520dihedral%2520angles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3257%26epage%3D3273%26doi%3D10.1021%2Fct300400x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klauda, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freites, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondragon-Ramirez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Update of the CHARMM all-atom additive force field for lipids: validation on six lipid types</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7830</span>– <span class="NLM_lpage">7843</span>, <span class="refDoi"> DOI: 10.1021/jp101759q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp101759q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=7830-7843&author=J.+B.+Klaudaauthor=R.+M.+Venableauthor=J.+A.+Freitesauthor=J.+W.+O%E2%80%99Connorauthor=D.+J.+Tobiasauthor=C.+Mondragon-Ramirezauthor=I.+Vorobyovauthor=A.+D.+MacKerellauthor=R.+W.+Pastor&title=Update+of+the+CHARMM+all-atom+additive+force+field+for+lipids%3A+validation+on+six+lipid+types&doi=10.1021%2Fjp101759q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span></div><div class="casAuthors">Klauda, Jeffery B.; Venable, Richard M.; Freites, J. Alfredo; O'Connor, Joseph W.; Tobias, Douglas J.; Mondragon-Ramirez, Carlos; Vorobyov, Igor; MacKerell, Alexander D., Jr.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7830-7843</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A significant modification to the additive all-atom CHARMM lipid force field (FF) is developed and applied to phospholipid bilayers with both choline and ethanolamine contg. head groups and with both satd. and unsatd. aliph. chains.  Motivated by the current CHARMM lipid FF (C27 and C27r) systematically yielding values of the surface area per lipid that are smaller than exptl. ests. and gel-like structures of bilayers well above the gel transition temp., selected torsional, Lennard-Jones and partial at. charge parameters were modified by targeting both quantum mech. (QM) and exptl. data.  QM calcns. ranging from high-level ab initio calcns. on small mols. to semiempirical QM studies on a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer in combination with exptl. thermodn. data were used as target data for parameter optimization.  These changes were tested with simulations of pure bilayers at high hydration of the following six lipids: DPPC, 1,2-dimyristoyl-sn-phosphatidylcholine (DMPC), 1,2-dilauroyl-sn-phosphatidylcholine (DLPC), 1-palmitoyl-2-oleoyl-sn-phosphatidylcholine (POPC), 1,2-dioleoyl-sn-phosphatidylcholine (DOPC), and 1-palmitoyl-2-oleoyl-sn-phosphatidylethanolamine (POPE); simulations of a low hydration DOPC bilayer were also performed.  Agreement with exptl. surface area is on av. within 2%, and the d. profiles agree well with neutron and x-ray diffraction expts.  NMR deuterium order parameters (SCD) are well predicted with the new FF, including proper splitting of the SCD for the aliph. carbon adjacent to the carbonyl for DPPC, POPE, and POPC bilayers.  The area compressibility modulus and frequency dependence of 13C NMR relaxation rates of DPPC and the water distribution of low hydration DOPC bilayers also agree well with expt.  Accordingly, the presented lipid FF, referred to as C36, allows for mol. dynamics simulations to be run in the tensionless ensemble (NPT), and is anticipated to be of utility for simulations of pure lipid systems as well as heterogeneous systems including membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTVhlLR7mEbVg90H21EOLACvtfcHk0lh0nsX_jftUsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D&md5=04c4d3aa1a59740190994d7df80cbb71</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjp101759q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp101759q%26sid%3Dliteratum%253Aachs%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DFreites%26aufirst%3DJ.%2BA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BW.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DMondragon-Ramirez%26aufirst%3DC.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DUpdate%2520of%2520the%2520CHARMM%2520all-atom%2520additive%2520force%2520field%2520for%2520lipids%253A%2520validation%2520on%2520six%2520lipid%2520types%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D7830%26epage%3D7843%26doi%3D10.1021%2Fjp101759q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) I: bond perception and atom typing</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3144</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0lh0nsX_jftUsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3155</span>– <span class="NLM_lpage">3168</span>, <span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0lh0nsX_jftUsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kräutler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hünenberger, P.
H.</span></span> <span> </span><span class="NLM_article-title">A Fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1002/1096-987X(20010415)22:5<501::AID-JCC1021>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisVSntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=501-508&author=V.+Kr%C3%A4utlerauthor=W.+F.+Van+Gunsterenauthor=P.%0AH.+H%C3%BCnenberger&title=A+Fast+SHAKE+algorithm+to+solve+distance+constraint+equations+for+small+molecules+in+molecular+dynamics+simulations&doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span></div><div class="casAuthors">Krautler, Vincent; Van Gunsteren, Wilfred F.; Hunenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">501-508</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A common method for the application of distance constraints in mol. simulations employing Cartesian coordinates is the SHAKE procedure for detg. the Lagrange multipliers regarding the constraints.  This method relies on the linearization and decoupling of the equations governing the at. coordinate resetting corresponding to each constraint in a mol., and is thus iterative.  In the present study, we consider an alternative method, M-SHAKE, which solves the coupled equations simultaneously by matrix inversion.  The performances of the two methods are compared in simulations of the pure solvents water, DMSO, and chloroform.  It is concluded that M-SHAKE is significantly faster than SHAKE when either (1) the mols. contain few distance constraints (solvent), or (2) when a high level of accuracy is required in the application of the constraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO7vyqEu_pjrVg90H21EOLACvtfcHk0ljJBEnoBz4LaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisVSntbY%253D&md5=20e32455e867f10fb61713f86147d96b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1096-987X%252820010415%252922%253A5%253C501%253A%253AAID-JCC1021%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4utler%26aufirst%3DV.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DA%2520Fast%2520SHAKE%2520algorithm%2520to%2520solve%2520distance%2520constraint%2520equations%2520for%2520small%2520molecules%2520in%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D501%26epage%3D508%26doi%3D10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G. A.</span></span> <span> </span><span class="NLM_article-title">Smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+A.+Pedersen&title=Smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0ljJBEnoBz4LaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%2BA.%26atitle%3DSmooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skeel, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kale, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Scalable molecular dynamics with NAMD</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/jcc.20289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1002%2Fjcc.20289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=16222654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1781-1802&author=J.+C.+Phillipsauthor=R.+Braunauthor=W.+Wangauthor=J.+Gumbartauthor=E.+Tajkhorshidauthor=E.+Villaauthor=C.+Chipotauthor=R.+D.+Skeelauthor=L.+Kaleauthor=K.+Schulten&title=Scalable+molecular+dynamics+with+NAMD&doi=10.1002%2Fjcc.20289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable molecular dynamics with NAMD</span></div><div class="casAuthors">Phillips, James C.; Braun, Rosemary; Wang, Wei; Gumbart, James; Tajkhorshid, Emad; Villa, Elizabeth; Chipot, Christophe; Skeel, Robert D.; Kale, Laxmikant; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1781-1802</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">NAMD is a parallel mol. dynamics code designed for high-performance simulation of large biomol. systems.  NAMD scales to hundreds of processors on high-end parallel platforms, as well as tens of processors on low-cost commodity clusters, and also runs on individual desktop and laptop computers.  NAMD works with AMBER and CHARMM potential functions, parameters, and file formats.  This article, directed to novices as well as experts, first introduces concepts and methods used in the NAMD program, describing the classical mol. dynamics force field, equations of motion, and integration methods along with the efficient electrostatics evaluation algorithms employed and temp. and pressure controls used.  Features for steering the simulation across barriers and for calcg. both alchem. and conformational free energy differences are presented.  The motivations for and a roadmap to the internal design of NAMD, implemented in C++ and based on Charm++ parallel objects, are outlined.  The factors affecting the serial and parallel performance of a simulation are discussed.  Finally, typical NAMD use is illustrated with representative applications to a small, a medium, and a large biomol. system, highlighting particular features of NAMD, for example, the Tcl scripting language.  The article also provides a list of the key features of NAMD and discusses the benefits of combining NAMD with the mol. graphics/sequence anal. software VMD and the grid computing/collab. software BioCoRE.  NAMD is distributed free of charge with source code at www.ks.uiuc.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fq_yG00T1rVg90H21EOLACvtfcHk0ljJBEnoBz4LaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ&md5=189051128443b547f4300a1b8fb0e034</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20289%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DJ.%2BC.%26aulast%3DBraun%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGumbart%26aufirst%3DJ.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DChipot%26aufirst%3DC.%26aulast%3DSkeel%26aufirst%3DR.%2BD.%26aulast%3DKale%26aufirst%3DL.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DScalable%2520molecular%2520dynamics%2520with%2520NAMD%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1781%26epage%3D1802%26doi%3D10.1002%2Fjcc.20289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tosh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciancetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based scaffold repurposing for GPCRs: Transformation of adenosine derivatives into selective 5HT<sub>2B</sub>/5HT<sub>2C</sub> serotonin receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">11006</span>– <span class="NLM_lpage">11026</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=11006-11026&author=D.+K.+Toshauthor=A.+Ciancettaauthor=E.+Warnickauthor=S.+Craneauthor=Z.-G.+Gaoauthor=K.+A.+Jacobson&title=Structure-based+scaffold+repurposing+for+GPCRs%3A+Transformation+of+adenosine+derivatives+into+selective+5HT2B%2F5HT2C+serotonin+receptor+antagonists&doi=10.1021%2Facs.jmedchem.6b01183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01183%26sid%3Dliteratum%253Aachs%26aulast%3DTosh%26aufirst%3DD.%2BK.%26aulast%3DCiancetta%26aufirst%3DA.%26aulast%3DWarnick%26aufirst%3DE.%26aulast%3DCrane%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520scaffold%2520repurposing%2520for%2520GPCRs%253A%2520Transformation%2520of%2520adenosine%2520derivatives%2520into%2520selective%25205HT2B%252F5HT2C%2520serotonin%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D11006%26epage%3D11026%26doi%3D10.1021%2Facs.jmedchem.6b01183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Innocenti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K-M J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solanki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mas-Droux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westlake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Design of potent and selective hybrid inhibitors of the mitotic kinase Nek2: Structure–activity relationship, structural biology, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3228</span>– <span class="NLM_lpage">3241</span>, <span class="refDoi"> DOI: 10.1021/jm201683b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201683b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Wlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3228-3241&author=P.+Innocentiauthor=K-M+J.+Cheungauthor=S.+Solankiauthor=C.+Mas-Drouxauthor=F.+Rowanauthor=S.+Yeohauthor=K.+Boxallauthor=M.+Westlakeauthor=L.+Pickardauthor=T.+Hardyauthor=J.+E.+Baxterauthor=G.+W.+Aherneauthor=R.+Baylissauthor=A.+M.+Fryauthor=S.+Hoelder&title=Design+of+potent+and+selective+hybrid+inhibitors+of+the+mitotic+kinase+Nek2%3A+Structure%E2%80%93activity+relationship%2C+structural+biology%2C+and+cellular+activity&doi=10.1021%2Fjm201683b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Potent and Selective Hybrid Inhibitors of the Mitotic Kinase Nek2: Structure-Activity Relationship, Structural Biology, and Cellular Activity</span></div><div class="casAuthors">Innocenti, Paolo; Cheung, Kwai-Ming J.; Solanki, Savade; Mas-Droux, Corine; Rowan, Fiona; Yeoh, Sharon; Boxall, Kathy; Westlake, Maura; Pickard, Lisa; Hardy, Tara; Baxter, Joanne E.; Aherne, G. Wynne; Bayliss, Richard; Fry, Andrew M.; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3228-3241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein a series of Nek2 inhibitors based on an aminopyridine scaffold.  These compds. have been designed by combining key elements of two previously discovered chem. series.  Structure based design led to aminopyridine (R)-21 (I), a potent and selective inhibitor able to modulate Nek2 activity in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4618QDhLevLVg90H21EOLACvtfcHk0lh7BrBjhm-NwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Wlu78%253D&md5=e777b9862be94a8e4dc22f6af6c85094</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm201683b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201683b%26sid%3Dliteratum%253Aachs%26aulast%3DInnocenti%26aufirst%3DP.%26aulast%3DCheung%26aufirst%3DK-M%2BJ.%26aulast%3DSolanki%26aufirst%3DS.%26aulast%3DMas-Droux%26aufirst%3DC.%26aulast%3DRowan%26aufirst%3DF.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DPickard%26aufirst%3DL.%26aulast%3DHardy%26aufirst%3DT.%26aulast%3DBaxter%26aufirst%3DJ.%2BE.%26aulast%3DAherne%26aufirst%3DG.%2BW.%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesign%2520of%2520potent%2520and%2520selective%2520hybrid%2520inhibitors%2520of%2520the%2520mitotic%2520kinase%2520Nek2%253A%2520Structure%25E2%2580%2593activity%2520relationship%252C%2520structural%2520biology%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3228%26epage%3D3241%26doi%3D10.1021%2Fjm201683b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akhlaghinia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezazadeh, S.</span></span> <span> </span><span class="NLM_article-title">A novel approach for the synthesis of 5-substituted-1<i>H</i>-tetrazoles</span>. <i>J. Braz. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2197</span>– <span class="NLM_lpage">2203</span>, <span class="refDoi"> DOI: 10.1590/S0103-50532013005000005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1590%2FS0103-50532013005000005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsFanu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=2197-2203&author=B.+Akhlaghiniaauthor=S.+Rezazadeh&title=A+novel+approach+for+the+synthesis+of+5-substituted-1H-tetrazoles&doi=10.1590%2FS0103-50532013005000005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A novel approach for the synthesis of 5-substituted-1H-tetrazoles</span></div><div class="casAuthors">Akhlaghinia, Batool; Rezazadeh, Soodabeh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Brazilian Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2197-2203</span>CODEN:
                <span class="NLM_cas:coden">JOCSET</span>;
        ISSN:<span class="NLM_cas:issn">0103-5053</span>.
    
            (<span class="NLM_cas:orgname">Sociedade Brasileira de Quimica</span>)
        </div><div class="casAbstract">A series of 5-substituted-1H-tetrazoles (RCN4H), e.g., I, have been synthesized by cycloaddn. reaction of different aryl and alkyl nitriles with sodium azide in DMSO using CuSO4·5H2O as catalyst.  A wide variety of aryl nitriles underwent [3 + 2] cycloaddn. to afford tetrazoles under mild reaction conditions in good to excellent yields.  The catalyst used is readily available and environmentally friendly.  Short reaction times, good to excellent yields, safe process and simple workup make this method an attractive and useful contribution to present org. synthesis of 5-substituted-1H-tetrazoles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Ulfd7WXuHbVg90H21EOLACvtfcHk0lh7BrBjhm-NwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsFanu7c%253D&md5=6bdb2eabce7a4a0b2070cd72511cc355</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1590%2FS0103-50532013005000005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1590%252FS0103-50532013005000005%26sid%3Dliteratum%253Aachs%26aulast%3DAkhlaghinia%26aufirst%3DB.%26aulast%3DRezazadeh%26aufirst%3DS.%26atitle%3DA%2520novel%2520approach%2520for%2520the%2520synthesis%2520of%25205-substituted-1H-tetrazoles%26jtitle%3DJ.%2520Braz.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D23%26spage%3D2197%26epage%3D2203%26doi%3D10.1590%2FS0103-50532013005000005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amison, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shaughnessy, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ofoedu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idzko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitchford, S. C.</span></span> <span> </span><span class="NLM_article-title">Lipopolysaccharide (LPS) induced pulmonary neutrophil recruitment and platelet activation is mediated via the P2Y<sub>1</sub> and P2Y<sub>14</sub> receptors in mice</span>. <i>Pulm. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.pupt.2017.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=10.1016%2Fj.pupt.2017.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=28487256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsFems74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=62-68&author=R.+T.+Amisonauthor=S.+Arnoldauthor=B.+G.+O%E2%80%99Shaughnessyauthor=S.+J.+Clearyauthor=J.+Ofoeduauthor=M.+Idzkoauthor=C.+P.+Pageauthor=S.+C.+Pitchford&title=Lipopolysaccharide+%28LPS%29+induced+pulmonary+neutrophil+recruitment+and+platelet+activation+is+mediated+via+the+P2Y1+and+P2Y14+receptors+in+mice&doi=10.1016%2Fj.pupt.2017.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Lipopolysaccharide (LPS) induced pulmonary neutrophil recruitment and platelet activation is mediated via the P2Y1 and P2Y14 receptors in mice</span></div><div class="casAuthors">Amison, R. T.; Arnold, S.; O'Shaughnessy, B. G.; Cleary, S. J.; Ofoedu, J.; Idzko, M.; Page, C. P.; Pitchford, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Pulmonary Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62-68</span>CODEN:
                <span class="NLM_cas:coden">PPTHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1094-5539</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Platelet activation occurs during host defense and in various inflammatory disorders.  In animal models of infection and inflammation, exptl. depletion of platelets leads to significantly reduced leukocyte recruitment and impaired clearance of pathogens from the lung.  It is now appreciated that purinergic receptor activation is required for leukocyte activation, motility and adhesion, and platelet interactions with leukocytes can be modulated by purinergic stimulation of platelets.  Here, we have investigated the role of platelet P2Y1, P2Y12, P2Y14, and P2X1 receptors on leukocyte recruitment and chemotaxis.  Mice were administered either vehicle controls or selective P2Y1, P2Y12, P2Y14, or P2X1 antagonists i.v. before intranasal administration of lipopolysaccharide (LPS) to investigate the effect of these drugs on pulmonary leukocyte recruitment, peripheral platelet counts, bleeding times, and ex vivo platelet aggregation.  Sep., platelets were incubated with P2Y1, P2Y12, P2X1 antagonists, or P2Y14 agonists to assess effects on platelet-induced neutrophil chemotaxis in vitro.  Pulmonary neutrophil recruitment induced by intranasal LPS administration was inhibited in mice administered either with P2Y1 or P2Y14 antagonists, but not with P2Y12 or P2X1 antagonists.  Furthermore, the administration of either a P2Y1 or a P2Y14 antagonist reversed the incidence of peripheral thrombocytopaenia assocd. with LPS exposure.  Bleeding times were significantly increased in mice administered P2Y1, P2Y12, or P2X1 antagonists, while ex vivo platelet aggregation to ADP was significantly reduced.  These haemostatic responses remained unaltered following antagonism of P2Y14.  In vitro chemotaxis assays revealed direct antagonism of platelet P2Y1, but not P2Y12 or P2X1 receptors suppressed platelet-dependent neutrophil motility towards Macrophage derived chemokine (MDC, CCL22).  Furthermore, the stimulation of platelets with selective P2Y14 agonists (UDP-glucose, MRS2690) resulted in significant platelet-dependent neutrophil chemotaxis.  These results reveal a role for P2Y1 and P2Y14 activation of platelets following exposure to LPS, while haemostatic indexes were unaffected by inhibition of platelet function with the P2Y14 antagonist in response to LPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ3K3utG7seLVg90H21EOLACvtfcHk0lh7BrBjhm-NwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsFems74%253D&md5=823d73929bad49a537d073b62d7daec0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.pupt.2017.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pupt.2017.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DAmison%26aufirst%3DR.%2BT.%26aulast%3DArnold%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DB.%2BG.%26aulast%3DCleary%26aufirst%3DS.%2BJ.%26aulast%3DOfoedu%26aufirst%3DJ.%26aulast%3DIdzko%26aufirst%3DM.%26aulast%3DPage%26aufirst%3DC.%2BP.%26aulast%3DPitchford%26aufirst%3DS.%2BC.%26atitle%3DLipopolysaccharide%2520%2528LPS%2529%2520induced%2520pulmonary%2520neutrophil%2520recruitment%2520and%2520platelet%2520activation%2520is%2520mediated%2520via%2520the%2520P2Y1%2520and%2520P2Y14%2520receptors%2520in%2520mice%26jtitle%3DPulm.%2520Pharmacol.%2520Ther.%26date%3D2017%26volume%3D45%26spage%3D62%26epage%3D68%26doi%3D10.1016%2Fj.pupt.2017.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanchuk, V. Y.</span></span> <span> </span><span class="NLM_article-title">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1136</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci025515j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025515j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1136-1145&author=I.+V.+Tetkoauthor=V.+Y.+Tanchuk&title=Application+of+associative+neural+networks+for+prediction+of+lipophilicity+in+ALOGPS+2.1+program&doi=10.1021%2Fci025515j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span></div><div class="casAuthors">Tetko, Igor V.; Tanchuk, Vsevolod Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1136-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article provides a systematic study of several important parameters of the Associative Neural Network (ASNN), such as the no. of networks in the ensemble, distance measures, neighbor functions, selection of smoothing parameters, and strategies for the user-training feature of the algorithm.  The performance of the different methods is assessed with several training/test sets used to predict lipophilicity of chem. compds.  The Spearman rank-order correlation coeff. and Parzen-window regression methods provide the best performance of the algorithm.  If addnl. user data is available, an improved prediction of lipophilicity of chems. up to 2-5 times can be calcd. when the appropriate smoothing parameters for the neural network are selected.  The detected best combinations of parameters and strategies are implemented in the ALOGPS 2.1 program that is publicly available at http://www.vcclab.org/lab/alogps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Nz8kkyB1NbVg90H21EOLACvtfcHk0lgaIifCa7EVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D&md5=45d412977a5171d4e8b03ada28c3632c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci025515j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025515j%26sid%3Dliteratum%253Aachs%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DTanchuk%26aufirst%3DV.%2BY.%26atitle%3DApplication%2520of%2520associative%2520neural%2520networks%2520for%2520prediction%2520of%2520lipophilicity%2520in%2520ALOGPS%25202.1%2520program%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1136%26epage%3D1145%26doi%3D10.1021%2Fci025515j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascher, D. B.</span></span> <span> </span><span class="NLM_article-title">pkCSM: Predicting small-molecule pharmacokinetic and toxicity properties using graph-based signatures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4066</span>– <span class="NLM_lpage">4072</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00104</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00104" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlams7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4066-4072&author=D.+E.+Piresauthor=T.+L.+Blundellauthor=D.+B.+Ascher&title=pkCSM%3A+Predicting+small-molecule+pharmacokinetic+and+toxicity+properties+using+graph-based+signatures&doi=10.1021%2Facs.jmedchem.5b00104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">pkCSM: Predicting Small-Molecule Pharmacokinetic and Toxicity Properties Using Graph-Based Signatures</span></div><div class="casAuthors">Pires, Douglas E. V.; Blundell, Tom L.; Ascher, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4066-4072</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug development has a high attrition rate, with poor pharmacokinetic and safety properties a significant hurdle.  Computational approaches may help minimize these risks.  The authors have developed a novel approach (pkCSM) which uses graph-based signatures to develop predictive models of central ADMET properties for drug development.  PkCSM performs as well or better than current methods.  A freely accessible web server (http://structure.bioc.cam.ac.uk/pkcsm), which retains no information submitted to it, provides an integrated platform to rapidly evaluate pharmacokinetic and toxicity properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMqi4e2w88abVg90H21EOLACvtfcHk0lgaIifCa7EVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlams7w%253D&md5=2910009eb9e62f72822615eb049df123</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00104%26sid%3Dliteratum%253Aachs%26aulast%3DPires%26aufirst%3DD.%2BE.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAscher%26aufirst%3DD.%2BB.%26atitle%3DpkCSM%253A%2520Predicting%2520small-molecule%2520pharmacokinetic%2520and%2520toxicity%2520properties%2520using%2520graph-based%2520signatures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4066%26epage%3D4072%26doi%3D10.1021%2Facs.jmedchem.5b00104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PXZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PXZ','PDB','4PXZ'); return false;">PDB: 4PXZ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i117"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_75625"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00168">10.1021/acs.jmedchem.8b00168</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis of compounds <b>55</b>–<b>78</b>, off-target binding inhibition curves, pharmacological data, molecular modeling data, including study of the interactions with ELs; selection of the optimal spacer geometry; lowest IE structure; structure and docking scores of designed compounds; replicate analysis of MD simulations; predicted pharmacokinetic parameters; and predicted LogD and LogS (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf">PDF</a>)</p></li><li><p class="inline">NMR and mass spectra of triazole derivatives (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Three-dimensional coordinates of hP2Y<sub>14</sub>R in complex with <b>7</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_003.pdb">PDB</a>) and <b>20</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Videos of MD simulations of hP2Y<sub>14</sub>R complexes with <b>9</b> compared to <b>20</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi">AVI</a>) and with <b>30</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_006.avi">AVI</a>) and with <b>2a</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_007.avi">AVI</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_008.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_001.pdf">jm8b00168_si_001.pdf (1.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_002.pdf">jm8b00168_si_002.pdf (3.45 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_003.pdb">jm8b00168_si_003.pdb (186.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_004.pdb">jm8b00168_si_004.pdb (186.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_005.avi">jm8b00168_si_005.avi (27.87 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_006.avi">jm8b00168_si_006.avi (11.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_007.avi">jm8b00168_si_007.avi (17.56 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00168/suppl_file/jm8b00168_si_008.csv">jm8b00168_si_008.csv (4.37 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00168&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00168%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00168" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67995fb21ea83d00","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
